<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/5/1475" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/5/1475.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/5/1475.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/5/1475.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/5/1475/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/5/1475.atom" />
<meta name="HW.pisa" content="ggg;7/5/1475" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Extensive Copy Number Variation in Fermentation-Related Genes Among Saccharomyces cerevisiae Wine Strains" />
<meta name="DC.Identifier" content="10.1534/g3.117.040105" />
<meta name="DC.Date" content="2017-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Steenwyk and Rokas. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Due to the importance of Saccharomyces cerevisiae in wine-making, the genomic variation of wine yeast strains has been extensively studied. One of the major insights stemming from these studies is that wine yeast strains harbor low levels of genetic diversity in the form of single nucleotide polymorphisms (SNPs). Genomic structural variants, such as copy number (CN) variants, are another major type of variation segregating in natural populations. To test whether genetic diversity in CN variation is also low across wine yeast strains, we examined genome-wide levels of CN variation in 132 whole-genome sequences of S. cerevisiae wine strains. We found an average of 97.8 CN variable regions (CNVRs) affecting ∼4% of the genome per strain. Using two different measures of CN diversity, we found that gene families involved in fermentation-related processes such as copper resistance ( CUP ), flocculation ( FLO ), and glucose metabolism ( HXT ), as well as the SNO gene family whose members are expressed before or during the diauxic shift, showed substantial CN diversity across the 132 strains examined. Importantly, these same gene families have been shown, through comparative transcriptomic and functional assays, to be associated with adaptation to the wine fermentation environment. Our results suggest that CN variation is a substantial contributor to the genomic diversity of wine yeast strains, and identify several candidate loci whose levels of CN variation may affect the adaptation and performance of wine yeast strains during fermentation." />
<meta name="DC.Contributor" content="Jacob Steenwyk" />
<meta name="DC.Contributor" content="Antonis Rokas" />
<meta name="article:published_time" content="2017-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Extensive Copy Number Variation in Fermentation-Related Genes Among Saccharomyces cerevisiae Wine Strains" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Due to the importance of &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; in wine-making, the genomic variation of wine yeast strains has been extensively studied. One of the major insights stemming from these studies is that wine yeast strains harbor low levels of genetic diversity in the form of single nucleotide polymorphisms (SNPs). Genomic structural variants, such as copy number (CN) variants, are another major type of variation segregating in natural populations. To test whether genetic diversity in CN variation is also low across wine yeast strains, we examined genome-wide levels of CN variation in 132 whole-genome sequences of &lt;i&gt;S. cerevisiae&lt;/i&gt; wine strains. We found an average of 97.8 CN variable regions (CNVRs) affecting ∼4% of the genome per strain. Using two different measures of CN diversity, we found that gene families involved in fermentation-related processes such as copper resistance (&lt;i&gt;CUP&lt;/i&gt;), flocculation (&lt;i&gt;FLO&lt;/i&gt;), and glucose metabolism (&lt;i&gt;HXT&lt;/i&gt;), as well as the &lt;i&gt;SNO&lt;/i&gt; gene family whose members are expressed before or during the diauxic shift, showed substantial CN diversity across the 132 strains examined. Importantly, these same gene families have been shown, through comparative transcriptomic and functional assays, to be associated with adaptation to the wine fermentation environment. Our results suggest that CN variation is a substantial contributor to the genomic diversity of wine yeast strains, and identify several candidate loci whose levels of CN variation may affect the adaptation and performance of wine yeast strains during fermentation.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/05/01" />
<meta name="citation_mjid" content="ggg;7/5/1475" />
<meta name="citation_id" content="7/5/1475" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/5/1475" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/5/1475.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/5/1475.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/5/1475.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.040105" />
<meta name="citation_pmid" content="28292787" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1475" />
<meta name="citation_lastpage" content="1485" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jacob Steenwyk" />
<meta name="citation_author_institution" content="Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8436-595X" />
<meta name="citation_author" content="Antonis Rokas" />
<meta name="citation_author_institution" content="Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235" />
<meta name="citation_author_email" content="antonis.rokas@vanderbilt.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7248-6551" />
<meta name="citation_reference" content="citation_journal_title=Environ. Mol. Mutagen.;citation_author=M. F. Arlt;citation_author=S. Rajendran;citation_author=S. R. Birkeland;citation_author=T. E. Wilson;citation_author=T. W. Glover;citation_title=Copy number variants are produced in response to low-dose ionizing radiation in cultured cells.;citation_pages=103-113;citation_volume=55;citation_year=2014;citation_pmid=24327335;citation_doi=10.1002/em.21840" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=E. Axelsson;citation_author=A. Ratnakumar;citation_author=M.-L. Arendt;citation_author=K. Maqbool;citation_author=M. T. Webster;citation_title=The genomic signature of dog domestication reveals adaptation to a starch-rich diet.;citation_pages=360-364;citation_volume=495;citation_year=2013;citation_pmid=23354050;citation_doi=10.1038/nature11837" />
<meta name="citation_reference" content="citation_journal_title=Journal of computational biology :  a journal of computational molecular cell biology;citation_journal_abbrev=J Comput Biol;citation_author=A. Bankevich;citation_author=S. Nurk;citation_author=D. Antipov;citation_author=A. A. Gurevich;citation_author=M. Dvorkin;citation_title=SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing.;citation_pages=455-477;citation_volume=19;citation_year=2012;citation_issue=5;citation_pmid=22506599;citation_doi=10.1089/cmb.2012.0021" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. B. Barton;citation_author=Y. Su;citation_author=J. Lamb;citation_author=D. Barber;citation_author=D. B. Kaback;citation_title=A Function for Subtelomeric DNA in Saccharomyces cerevisiae;citation_pages=929-934;citation_volume=165;citation_year=2003;citation_issue=2;citation_pmid=14573499" />
<meta name="citation_reference" content="citation_journal_title=Journal of Applied Microbiology (Print);citation_journal_abbrev=Journal of Applied Microbiology (Print);citation_author=E. Bartra;citation_author=M. Casado;citation_author=D. Carro;citation_author=C. Campamà;citation_author=B. Piña;citation_title=Differential expression of thiamine biosynthetic genes in yeast strains with high and low production of hydrogen sulfide during wine fermentation.;citation_pages=272-281;citation_volume=109;citation_year=2010;citation_issue=1;citation_pmid=20059614" />
<meta name="citation_reference" content="citation_journal_title=J. Ferment. Bioeng.;citation_author=M. Bataillon;citation_author=A. Rico;citation_author=J. M. Sablayrolles;citation_author=J. M. Salmon;citation_author=P. Barre;citation_title=Early thiamin assimilation by yeasts under enological conditions: impact on alcoholic fermentation kinetics.;citation_pages=145-150;citation_volume=82;citation_year=1996;citation_doi=10.1016/0922-338X(96)85037-9" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. M. Bickhart;citation_author=Y. Hou;citation_author=S. G. Schroeder;citation_author=C. Alkan;citation_author=M. F. Cardone;citation_title=Copy number variation of individual cattle genomes using next-generation sequencing.;citation_volume=22;citation_year=2012;citation_pmid=22300768;citation_doi=10.1101/gr.133967.111" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=J. A. Birchler;citation_author=R. A. Veitia;citation_title=The gene balance hypothesis: implications for gene regulation, quantitative traits and evolution.;citation_pages=54-62;citation_volume=186;citation_year=2010;citation_issue=1;citation_pmid=19925558;citation_doi=10.1111/j.1469-8137.2009.03087.x" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. A. Birchler;citation_author=R. A. Veitia;citation_title=Gene balance hypothesis: connecting issues of dosage sensitivity across biological disciplines.;citation_volume=109;citation_year=2012;citation_pmid=22908297;citation_doi=10.1073/pnas.1207726109" />
<meta name="citation_reference" content="citation_journal_title=American Journal of Enology and Viticulture;citation_journal_abbrev=Am. J. Enol. Vitic.;citation_author=L. F. Bisson;citation_title=Stuck and Sluggish Fermentations;citation_pages=107-119;citation_volume=50;citation_year=1999;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Enol. Vitic.;citation_author=L. F. Bisson;citation_title=Geographic origin and diversity of wine strains of Saccharomyces.;citation_volume=63;citation_year=2012;citation_doi=10.5344/ajev.2012.11083" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=V. Boeva;citation_author=A. Zinovyev;citation_author=K. Bleakley;citation_author=J. P. Vert;citation_author=I. Janoueix-Lerosey;citation_title=Control-free calling of copy number alterations in deep-sequencing data using GC-content normalization.;citation_volume=27;citation_year=2011;citation_pmid=21081509;citation_doi=10.1093/bioinformatics/btq635" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=V. Boeva;citation_author=T. Popova;citation_author=K. Bleakley;citation_author=P. Chiche;citation_author=J. Cappo;citation_title=Control-FREEC: a tool for assessing copy number and allelic content using next-generation sequencing data.;citation_volume=28;citation_year=2012;citation_pmid=22155870;citation_doi=10.1093/bioinformatics/btr670" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data.;citation_volume=30;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=A. R. Borneman;citation_author=A. H. Forgan;citation_author=I. S. Pretorius;citation_author=P. J. Chambers;citation_title=Comparative genome analysis of a Saccharomyces cerevisiae wine strain.;citation_volume=8;citation_year=2008;citation_pmid=18778279;citation_doi=10.1111/j.1567-1364.2008.00434.x" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=A. R. Borneman;citation_author=A. H. Forgan;citation_author=R. Kolouchova;citation_author=J. A. Fraser;citation_author=S. A. Schmidt;citation_title=Whole genome comparison reveals high levels of inbreeding and strain redundancy across the spectrum of commercial wine trains of Saccharomyces cerevisiae.;citation_pages=957-971;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=C. Brion;citation_author=C. Ambroset;citation_author=P. Delobel;citation_author=I. Sanchez;citation_author=B. Blondin;citation_title=Deciphering regulatory variation of THI genes in alcoholic fermentation indicate an impact of Thi3p on PDC1 expression.;citation_pages=1085;citation_volume=15;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=C. A. Brown;citation_author=A. W. Murray;citation_author=K. J. Verstrepen;citation_title=Rapid expansion and functional divergence of subtelomeric gene families in yeasts.;citation_pages=895-903;citation_volume=20;citation_year=2010;citation_issue=10;citation_pmid=20471265;citation_doi=10.1016/j.cub.2010.04.027" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=C. J. Brown;citation_author=K. M. Todd;citation_author=R. F. Rosenzweig;citation_title=Multiple duplications of yeast hexose transport genes in response to selection in a glucose-limited environment;citation_pages=931-942;citation_volume=15;citation_year=1998;citation_issue=8;citation_pmid=9718721;citation_doi=10.1093/oxfordjournals.molbev.a026009" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Carbon;citation_author=A. Ireland;citation_author=C. J. Mungall;citation_author=S. Shu;citation_author=B. Marshall;citation_title=AmiGO: online access to ontology and annotation data.;citation_volume=25;citation_year=2009;citation_pmid=19033274;citation_doi=10.1093/bioinformatics/btn615" />
<meta name="citation_reference" content="citation_journal_title=J. Mol. Evol.;citation_author=D. Cavalieri;citation_author=P. E. McGovern;citation_author=D. L. Hartl;citation_author=R. Mortimer;citation_author=M. Polsinelli;citation_title=Evidence for S. cerevisiae fermentation in ancient wine.;citation_pages=S226-S232;citation_volume=57;citation_year=2003;citation_pmid=15008419;citation_doi=10.1007/s00239-003-0031-2" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=G. A. Cromie;citation_author=K. E. Hyma;citation_author=C. L. Ludlow;citation_author=C. Garmendia-Torres;citation_author=T. L. Gilbert;citation_title=Genomic sequence diversity and population structure of Saccharomyces cerevisiae assessed by RAD-seq.;citation_pages=2163-2171;citation_volume=3;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Cytogenetic and genome research;citation_journal_abbrev=Cytogenet Genome Res;citation_author=G. Cutler;citation_author=P. D. Kassner;citation_title=Copy number variation in the mouse genome: implications for the mouse as a model organism for human disease.;citation_pages=297-306;citation_volume=123;citation_year=2008;citation_issue=1-4;citation_pmid=19287168;citation_doi=10.1159/000184721" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. Duan;citation_author=J.-G. Zhang;citation_author=H.-W. Deng;citation_author=Y.-P. Wang;citation_title=Comparative studies of copy number variation detection methods for next-generation sequencing technologies.;citation_pages=e59128;citation_volume=8;citation_year=2013;citation_pmid=23527109;citation_doi=10.1371/journal.pone.0059128" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. J. Dunham;citation_author=H. Badrane;citation_author=T. Ferea;citation_author=J. Adams;citation_author=P. O. Brown;citation_title=Characteristic genome rearrangements in experimental evolution of Saccharomyces cerevisiae.;citation_volume=99;citation_year=2002;citation_pmid=12446845;citation_doi=10.1073/pnas.242624799" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. Dunn;citation_author=C. Richter;citation_author=D. J. Kvitek;citation_author=T. Pugh;citation_author=G. Sherlock;citation_title=Analysis of the Saccharomyces cerevisiae pan-genome reveals a pool of copy number variants distributed in diverse yeast strains from differing industrial environments.;citation_volume=22;citation_year=2012;citation_pmid=22369888;citation_doi=10.1101/gr.130310.111" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=R. A. Farrer;citation_author=D. A. Henk;citation_author=T. W. Garner;citation_author=F. Balloux;citation_author=D. C. Woodhams;citation_title=Chromosomal copy number variation, selection and uneven rates of recombination reveal cryptic genome diversity linked to pathogenicity.;citation_pages=e1003703;citation_volume=9;citation_year=2013;citation_pmid=23966879;citation_doi=10.1371/journal.pgen.1003703" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Fidalgo;citation_author=R. R. Barrales;citation_author=J. I. Ibeas;citation_author=J. Jimenez;citation_title=Adaptive evolution by mutations in the FLO11 gene.;citation_volume=103;citation_year=2006;citation_pmid=16844788;citation_doi=10.1073/pnas.0601713103" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. L. Freeman;citation_author=G. H. Perry;citation_author=L. Feuk;citation_author=R. Redon;citation_author=S. A. McCarroll;citation_title=Copy number variation: new insights in genome diversity.;citation_volume=16;citation_year=2006;citation_pmid=16809666;citation_doi=10.1101/gr.3677206" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=B. Gallone;citation_author=J. Steensels;citation_author=T. Prahl;citation_author=L. Soriaga;citation_author=V. Saels;citation_title=Domestication and divergence of Saccharomyces cerevisiae beer yeasts.;citation_pages=1397-1410.e16;citation_volume=166;citation_year=2016;citation_pmid=27610566;citation_doi=10.1016/j.cell.2016.08.020" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_title=The Gene Ontology (GO) database and informatics resource;citation_pages=D258-261;citation_volume=32;citation_year=2004;citation_issue=90001;citation_pmid=14681407;citation_doi=10.1093/nar/gkh036" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Genet. Dev.;citation_author=J. G. Gibbons;citation_author=D. C. Rinker;citation_title=The genomics of microbial domestication in the fermented food environment.;citation_pages=1-8;citation_volume=35;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=J. G. Gibbons;citation_author=L. Salichos;citation_author=J. C. Slot;citation_author=D. C. Rinker;citation_author=K. L. McGary;citation_title=The evolutionary imprint of domestication on genome variation and function of the filamentous fungus Aspergillus oryzae.;citation_pages=1403-1409;citation_volume=22;citation_year=2012;citation_issue=15;citation_pmid=22795693;citation_doi=10.1016/j.cub.2012.05.033" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. G. Gibbons;citation_author=A. T. Branco;citation_author=S. A. Godinho;citation_author=S. Yu;citation_author=B. Lemos;citation_title=Concerted copy number variation balances ribosomal DNA dosage in human and mouse genomes.;citation_volume=112;citation_year=2015;citation_pmid=25583482;citation_doi=10.1073/pnas.1416878112" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=M. Gonçalves;citation_author=A. Pontes;citation_author=P. Almeida;citation_author=R. Barbosa;citation_author=M. Serra;citation_title=Distinct domestication trajectories in top-fermenting beer yeasts and wine yeasts.;citation_pages=2750-2761;citation_volume=26;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=P. Govender;citation_author=J. L. Domingo;citation_author=M. C. Bester;citation_author=I. S. Pretorius;citation_author=F. F. Bauer;citation_title=Controlled expression of the dominant flocculation genes FLO1, FLO5, and FLO11 in Saccharomyces cerevisiae.;citation_volume=74;citation_year=2008;citation_pmid=18708514;citation_doi=10.1128/AEM.00394-08" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=D. Gresham;citation_author=M. M. Desai;citation_author=C. M. Tucker;citation_author=H. T. Jenq;citation_author=D. A. Pai;citation_title=The repertoire and dynamics of evolutionary adaptations to controlled nutrient-limited environments in yeast.;citation_pages=e1000303-e1000303;citation_volume=4;citation_year=2008;citation_issue=12;citation_pmid=19079573;citation_doi=10.1371/journal.pgen.1000303" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.;citation_author=C. N. Henrichsen;citation_author=E. Chaignat;citation_author=A. Reymond;citation_title=Copy number variants, diseases and gene expression.;citation_volume=18;citation_year=2009;citation_pmid=19297395;citation_doi=10.1093/hmg/ddp011" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. E. Hillenmeyer;citation_author=E. Fung;citation_author=J. Wildenhain;citation_author=S. E. Pierce;citation_author=S. Hoon;citation_title=The Chemical Genomic Portrait of Yeast: Uncovering a Phenotype for All Genes;citation_pages=362-365;citation_volume=320;citation_year=2008;citation_issue=5874;citation_pmid=18420932;citation_doi=10.1126/science.1150021" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=G. Hu;citation_author=J. Wang;citation_author=J. Choi;citation_author=W. H. Jung;citation_author=I. Liu;citation_title=Variation in chromosome copy number influences the virulence of Cryptococcus neoformans and occurs in isolates from AIDS patients.;citation_pages=526-526;citation_volume=12;citation_year=2011;citation_pmid=22032296;citation_doi=10.1186/1471-2164-12-526" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=A. J. Hunter;citation_author=B. Jin;citation_author=J. M. Kelly;citation_title=Independent duplications of Î±-amylase in different strains of Aspergillus oryzae.;citation_pages=438-444;citation_volume=48;citation_year=2011;citation_issue=4;citation_pmid=21277379;citation_doi=10.1016/j.fgb.2011.01.006" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=D. H. Huson;citation_title=SplitsTree: analyzing and visualizing evolutionary data;citation_pages=68-73;citation_volume=14;citation_year=1998;citation_issue=1;citation_pmid=9520503;citation_doi=10.1093/bioinformatics/14.1.68" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=K. E. Hyma;citation_author=S. M. Saerens;citation_author=K. J. Verstrepen;citation_author=J. C. Fay;citation_title=Divergence in wine characteristics produced by wild and domesticated strains of Saccharomyces cerevisiae.;citation_volume=11;citation_year=2011;citation_pmid=22093681;citation_doi=10.1111/j.1567-1364.2011.00746.x" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=H. Kaessmann;citation_author=N. Vinckenbosch;citation_author=M. Long;citation_title=RNA-based gene duplication: mechanistic and evolutionary insights.;citation_pages=19-31;citation_volume=10;citation_year=2009;citation_issue=1;citation_pmid=19030023;citation_doi=10.1038/nrg2487" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=K. C. Kao;citation_author=G. Sherlock;citation_title=Molecular characterization of clonal interference during adaptive evolution in asexual populations of Saccharomyces cerevisiae.;citation_pages=1499-1504;citation_volume=40;citation_year=2008;citation_issue=12;citation_pmid=19029899;citation_doi=10.1038/ng.280" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=T. P. Keith;citation_author=P. Green;citation_author=S. T. Reeders;citation_author=V. A. Brown;citation_author=P. Phipps;citation_title=Genetic linkage map of 46 DNA markers on human chromosome 16.;citation_pages=5754-5758;citation_volume=87;citation_year=1990;citation_issue=15;citation_pmid=2377614;citation_doi=10.1073/pnas.87.15.5754" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. C. Koboldt;citation_author=K. Chen;citation_author=T. Wylie;citation_author=D. E. Larson;citation_author=M. D. McLellan;citation_title=VarScan: variant detection in massively parallel sequencing of individual and pooled samples.;citation_volume=25;citation_year=2009;citation_pmid=19542151;citation_doi=10.1093/bioinformatics/btp373" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. C. Koboldt;citation_author=Q. Zhang;citation_author=D. E. Larson;citation_author=D. Shen;citation_author=M. D. McLellan;citation_title=VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing.;citation_volume=22;citation_year=2012;citation_pmid=22300766;citation_doi=10.1101/gr.129684.111" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=M. Li;citation_author=B. J. Petteys;citation_author=J. M. McClure;citation_author=V. Valsakumar;citation_author=S. Bekiranov;citation_title=Thiamine biosynthesis in Saccharomyces cerevisiae is regulated by the NAD+-dependent histone deacetylase Hst1.;citation_volume=30;citation_year=2010;citation_pmid=20439498;citation_doi=10.1128/MCB.01590-09" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Z. Lin;citation_author=W. H. Li;citation_title=Expansion of hexose transporter genes was associated with the evolution of aerobic fermentation in yeasts.;citation_volume=28;citation_year=2011;citation_pmid=20660490;citation_doi=10.1093/molbev/msq184" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Liti;citation_author=D. M. Carter;citation_author=A. M. Moses;citation_author=J. Warringer;citation_author=L. Parts;citation_title=Population genomics of domestic and wild yeasts.;citation_pages=337-341;citation_volume=458;citation_year=2009;citation_issue=7236;citation_pmid=19212322;citation_doi=10.1038/nature07743" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=W. S. Lo;citation_author=A. M. Dranginis;citation_title=FLO11, a yeast gene related to the STA genes, encodes a novel cell surface flocculin.;citation_pages=7144-7151;citation_volume=178;citation_year=1996;citation_issue=24;citation_pmid=8955395;citation_doi=10.1128/jb.178.24.7144-7151.1996" />
<meta name="citation_reference" content="citation_journal_title=Trends in molecular medicine;citation_journal_abbrev=Trends Mol Med;citation_author=A. Lupetti;citation_author=R. Danesi;citation_author=M. Campa;citation_author=M. Del Tacca;citation_author=S. Kelly;citation_title=Molecular basis of resistance to azole antifungals.;citation_pages=76-81;citation_volume=8;citation_year=2002;citation_issue=2;citation_pmid=11815273;citation_doi=10.1016/S1471-4914(02)02280-3" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. R. Lupski;citation_author=P. Stankiewicz;citation_title=Genomic disorders: molecular mechanisms for rearrangements and conveyed phenotypes.;citation_pages=e49-e49;citation_volume=1;citation_year=2005;citation_issue=6;citation_pmid=16444292;citation_doi=10.1371/journal.pgen.0010049" />
<meta name="citation_reference" content="citation_title=The NCBI Handbook;citation_title=The BLAST sequence analysis tool,;citation_author=T. Madden;citation_pages=1-17;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=V. D. Marks;citation_author=S. J. Ho Sui;citation_author=D. Erasmus;citation_author=G. K. Van Der Merwe;citation_author=J. Brumm;citation_title=Dynamics of the yeast transcriptome during wine fermentation reveals a novel fermentation stress response.;citation_volume=8;citation_year=2008;citation_pmid=18215224;citation_doi=10.1111/j.1567-1364.2007.00338.x" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_title=Diversity and adaptive evolution of Saccharomyces wine yeast: a review.;citation_author=S. Marsit;citation_author=S. Dequin;citation_volume=15;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Nat. Protoc.;citation_author=H. Mi;citation_author=A. Muruganujan;citation_author=J. T. Casagrande;citation_author=P. D. Thomas;citation_title=Large-scale gene function analysis with the PANTHER classification system.;citation_pages=1551-1566;citation_volume=8;citation_year=2013;citation_pmid=23868073;citation_doi=10.1038/nprot.2013.092" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=C. A. Michels;citation_author=E. Read;citation_author=K. Nat;citation_author=M. J. Charron;citation_title=The telomere-associated MAL3 locus of Saccharomyces is a tandem array of repeated genes.;citation_pages=655-665;citation_volume=8;citation_year=1992;citation_issue=8;citation_pmid=1441745;citation_doi=10.1002/yea.320080809" />
<meta name="citation_reference" content="citation_journal_title=Applied Microbiology and Biotechnology;citation_journal_abbrev=Applied Microbiology and Biotechnology;citation_author=A. M. Molina;citation_author=J. H. Swiegers;citation_author=C. Varela;citation_author=I. S. Pretorius;citation_author=E. Agosin;citation_title=Influence of wine fermentation temperature on the synthesis of yeast-derived volatile aroma compounds.;citation_pages=675-687;citation_volume=77;citation_year=2007;citation_issue=3;citation_pmid=17938912" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=R. K. Mortimer;citation_title=Evolution and Variation of the Yeast (Saccharomyces) Genome;citation_pages=403-409;citation_volume=10;citation_year=2000;citation_issue=4;citation_pmid=10779481;citation_doi=10.1101/gr.10.4.403" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Novo;citation_author=F. Bigey;citation_author=E. Beyne;citation_author=V. Galeote;citation_author=F. Gavory;citation_title=Eukaryote-to-eukaryote gene transfer events revealed by the genome sequence of the wine yeast Saccharomyces cerevisiae EC1118.;citation_volume=106;citation_year=2009;citation_pmid=19805302;citation_doi=10.1073/pnas.0904673106" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.;citation_author=L. D. Orozco;citation_author=S. J. Cokus;citation_author=A. Ghazalpour;citation_author=L. Ingram-Drake;citation_author=S. Wang;citation_title=Copy number variation influences gene expression and metabolic traits in mice.;citation_volume=18;citation_year=2009;citation_pmid=19648292;citation_doi=10.1093/hmg/ddp360" />
<meta name="citation_reference" content="citation_journal_title=Journal of Antimicrobial Chemotherapy;citation_journal_abbrev=J Antimicrob Chemother;citation_author=N. Osherov;citation_author=D. P. Kontoyiannis;citation_author=A. Romans;citation_author=G. S. May;citation_title=Resistance to itraconazole in Aspergillus nidulans and Aspergillus fumigatus is conferred by extra copies of the A. nidulans P-450 14{{alpha}}-demethylase gene, pdmA;citation_pages=75-81;citation_volume=48;citation_year=2001;citation_issue=1;citation_pmid=11418514;citation_doi=10.1093/jac/48.1.75" />
<meta name="citation_reference" content="citation_journal_title=American Journal of Enology and Viticulture;citation_journal_abbrev=Am. J. Enol. Vitic.;citation_author=C. Ough;citation_author=M. Davenport;citation_author=K. Joseph;citation_title=Effects of Certain Vitamins on Growth and Fermentation Rate of Several Commercial Active Dry Wine Yeasts;citation_pages=208-213;citation_volume=40;citation_year=1989;citation_issue=3" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=P. A. Padilla;citation_author=E. K. Fuge;citation_author=M. E. Crawford;citation_author=A. Errett;citation_author=M. Werner-Washburne;citation_title=The Highly Conserved, Coregulated SNO and SNZ Gene Families in Saccharomyces cerevisiae Respond to Nutrient Limitation;citation_pages=5718-5726;citation_volume=180;citation_year=1998;citation_issue=21;citation_pmid=9791124" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=C. Payen;citation_author=A. B. Sunshine;citation_author=G. T. Ong;citation_author=J. L. Pogachar;citation_author=W. Zhao;citation_title=High-throughput identification of adaptive mutations in experimentally evolved yeast populations.;citation_pages=e1006339;citation_volume=12;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=G. H. Perry;citation_author=N. J. Dominy;citation_author=K. G. Claw;citation_author=A. S. Lee;citation_author=H. Fiegler;citation_title=Diet and the evolution of human amylase gene copy number variation.;citation_pages=1256-1260;citation_volume=39;citation_year=2007;citation_issue=10;citation_pmid=17828263;citation_doi=10.1038/ng2123" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=Z. Pezer;citation_author=B. Harr;citation_author=M. Teschke;citation_author=H. Babiker;citation_author=D. Tautz;citation_title=Divergence patterns of genic copy number variation in natural populations of the house mouse (Mus musculus domesticus) reveal three conserved genes with major population-specific expansions.;citation_volume=25;citation_year=2015;citation_pmid=26149421;citation_doi=10.1101/gr.187187.114" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=I. S. Pretorius;citation_title=Tailoring wine yeast for the new millennium: novel approaches to the ancient art of winemaking.;citation_pages=675-729;citation_volume=16;citation_year=2000;citation_issue=8;citation_pmid=10861899;citation_doi=10.1002/1097-0061(20000615)16:8&lt;675::AID-YEA585&gt;3.0.CO;2-B" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing.;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=C. B. Ramsook;citation_author=C. Tan;citation_author=M. C. Garcia;citation_author=R. Fung;citation_author=G. Soybelman;citation_title=Yeast cell adhesion molecules have functional amyloid-forming sequences.;citation_volume=9;citation_year=2010;citation_pmid=20038605;citation_doi=10.1128/EC.00068-09" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=E. Reifenberger;citation_author=K. Freidel;citation_author=M. Ciriacy;citation_title=Identification of novel HXT genes in Saccharomyces cerevisiae reveals the impact of individual hexose transporters on glycolytic flux.;citation_pages=157-167;citation_volume=16;citation_year=1995;citation_issue=1;citation_pmid=7651133;citation_doi=10.1111/j.1365-2958.1995.tb02400.x" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=N. Risch;citation_title=Linkage strategies for genetically complex traits. II. The power of affected relative pairs.;citation_pages=229-241;citation_volume=46;citation_year=1990;citation_issue=2;citation_pmid=2301393" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=S. Rodríguez-Navarro;citation_author=B. Llorente;citation_author=M. T. Rodríguez-Manzaneque;citation_author=A. Ramne;citation_author=G. Uber;citation_title=Functional analysis of yeast gene families involved in metabolism of vitamins B1 and B6.;citation_pages=1261-1276;citation_volume=19;citation_year=2002;citation_issue=14;citation_pmid=12271461;citation_doi=10.1002/yea.916" />
<meta name="citation_reference" content="citation_journal_title=Infection and Immunity;citation_journal_abbrev=Infect. Immun.;citation_author=M. Sanguinetti;citation_author=B. Posteraro;citation_author=M. La Sorda;citation_author=R. Torelli;citation_author=B. Fiori;citation_title=Role of AFR1, an ABC Transporter-Encoding Gene, in the In Vivo Response to Fluconazole and Virulence of Cryptococcus neoformans;citation_pages=1352-1359;citation_volume=74;citation_year=2006;citation_issue=2;citation_pmid=16428784;citation_doi=10.1128/IAI.74.2.1352-1359.2006" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Schacherer;citation_author=J. A. Shapiro;citation_author=D. M. Ruderfer;citation_author=L. Kruglyak;citation_title=Comprehensive polymorphism survey elucidates population structure of Saccharomyces cerevisiae.;citation_pages=342-345;citation_volume=458;citation_year=2009;citation_issue=7236;citation_pmid=19212320;citation_doi=10.1038/nature07670" />
<meta name="citation_reference" content="citation_journal_title=Applied and Environmental Microbiology;citation_journal_abbrev=Appl. Environ. Microbiol.;citation_author=T. Seki;citation_author=E. H. Choi;citation_author=D. Ryu;citation_title=Construction of Killer Wine Yeast Strain;citation_pages=1211-1215;citation_volume=49;citation_year=1985;citation_issue=5;citation_pmid=16346794" />
<meta name="citation_reference" content="citation_journal_title=Comptes rendus biologies;citation_journal_abbrev=C R Biol;citation_author=D. Sicard;citation_author=J. L. Legras;citation_title=Bread, beer and wine: yeast domestication in the Saccharomyces sensu stricto complex.;citation_pages=229-236;citation_volume=334;citation_year=2011;citation_issue=3;citation_pmid=21377618;citation_doi=10.1016/j.crvi.2010.12.016" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=D. Sims;citation_author=I. Sudbery;citation_author=N. E. Ilott;citation_author=A. Heger;citation_author=C. P. Ponting;citation_title=Sequencing depth and coverage: key considerations in genomic analyses.;citation_pages=121-132;citation_volume=15;citation_year=2014;citation_pmid=24434847;citation_doi=10.1038/nrg3642" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Pathog;citation_author=E. Sionov;citation_author=H. Lee;citation_author=Y. C. Chang;citation_author=K. J. Kwon-Chung;citation_title=Cryptococcus neoformans overcomes stress of azole drugs by formation of disomy in specific multiple chromosomes.;citation_pages=e1000848-e1000848;citation_volume=6;citation_year=2010;citation_issue=4;citation_pmid=20368972;citation_doi=10.1371/journal.ppat.1000848" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. A. Skelly;citation_author=G. E. Merrihew;citation_author=M. Riffle;citation_author=C. F. Connelly;citation_author=E. O. Kerr;citation_title=Integrative phenomics reveals insight into the structure of phenotypic diversity in budding yeast.;citation_volume=23;citation_year=2013;citation_pmid=23720455;citation_doi=10.1101/gr.155762.113" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. S. C. Smith;citation_author=E. C. L. Chin;citation_author=H. Shu;citation_author=O. S. Smith;citation_author=S. J. Wall;citation_title=An evaluation of the utility of SSR loci as molecular markers in maize (Zea mays L.): comparisons with data from RFLPS and pedigree.;citation_pages=163-173;citation_volume=95;citation_year=1997;citation_doi=10.1007/s001220050544" />
<meta name="citation_reference" content="citation_journal_abbrev=J Biosci Bioeng;citation_author=B. U. Stambuk;citation_author=A. S. Batista;citation_author=P. S. De Araujo;citation_title=Kinetics of active sucrose transport in Saccharomyces cerevisiae.;citation_pages=212-214;citation_volume=89;citation_year=2000;citation_issue=2;citation_pmid=16232731;citation_doi=10.1016/S1389-1723(00)88742-3" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. L. Steenwyk;citation_author=J. S. Soghigian;citation_author=J. R. Perfect;citation_author=J. G. Gibbons;citation_title=Copy number variation contributes to cryptic genetic variation in outbreak lineages of Cryptococcus gattii from the North American Pacific Northwest.;citation_pages=700;citation_volume=17;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. H. Sudmant;citation_author=J. O. Kitzman;citation_author=F. Antonacci;citation_author=C. Alkan;citation_author=M. Malig;citation_title=Diversity of Human Copy Number Variation and Multicopy Genes;citation_pages=641-646;citation_volume=330;citation_year=2010;citation_issue=6004;citation_pmid=21030649;citation_doi=10.1126/science.1197005" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=P. H. Sudmant;citation_author=S. Mallick;citation_author=B. J. Nelson;citation_author=F. Hormozdiari;citation_author=N. Krumm;citation_title=Global diversity, population stratification, and selection of human copy-number variation.;citation_volume=349;citation_year=2015;citation_pmid=26249230;citation_doi=10.1126/science.aab3761" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=M. A. Teste;citation_author=J. Marie François;citation_author=J. L. Parrou;citation_title=Characterization of a new multigene family encoding isomaltases in the yeast Saccharomyces cerevisiae, the IMA family.;citation_volume=285;citation_year=2010;citation_pmid=20562106;citation_doi=10.1074/jbc.M110.145946" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=P. D. Thomas;citation_author=M. J. Campbell;citation_author=A. Kejariwal;citation_author=H. Mi;citation_author=B. Karlak;citation_title=PANTHER: A Library of Protein Families and Subfamilies Indexed by Function;citation_pages=2129-2141;citation_volume=13;citation_year=2003;citation_issue=9;citation_pmid=12952881;citation_doi=10.1101/gr.772403" />
<meta name="citation_reference" content="citation_journal_title=J. Chem. Inf. Model.;citation_author=M.M.-A. Victoria;citation_author=P. M. Carmen;citation_title=Wine chemistry and biochemistry.;citation_pages=1689-1699;citation_volume=53;citation_year=2013;citation_pmid=23800267;citation_doi=10.1021/ci400128m" />
<meta name="citation_reference" content="citation_journal_title=Applied Microbiology and Biotechnology;citation_journal_abbrev=Applied Microbiology and Biotechnology;citation_author=A. Vilela-Moura;citation_author=D. Schuller;citation_author=A. Mendes-Faia;citation_author=M. Côrte-Real;citation_title=Reduction of volatile acidity of wines by selected yeast strains.;citation_pages=881-890;citation_volume=80;citation_year=2008;citation_issue=5;citation_pmid=18677471;citation_doi=10.1007/s00253-008-1616-x" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Soc. Inf.;citation_author=D. Wallace;citation_title=The Mann-Whitney test.;citation_pages=93;citation_volume=55;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. Warringer;citation_author=E. Zörgö;citation_author=F. A. Cubillos;citation_author=A. Zia;citation_author=A. Gjuvsland;citation_title=Trait variation in yeast is defined by population history.;citation_pages=e1002111-e1002111;citation_volume=7;citation_year=2011;citation_issue=6;citation_pmid=21698134;citation_doi=10.1371/journal.pgen.1002111" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_journal_abbrev=Microbiology;citation_author=R. Wightman;citation_author=P. A. Meacock;citation_title=The THI5 gene family of Saccharomyces cerevisiae: distribution of homologues among the hemiascomycetes and functional redundancy in the aerobic biosynthesis of thiamin from pyridoxine;citation_pages=1447-1460;citation_volume=149;citation_year=2003;citation_issue=6;citation_pmid=12777485;citation_doi=10.1099/mic.0.26194-0" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. A. Winzeler;citation_author=D. D. Shoemaker;citation_author=A. Astromoff;citation_author=H. Liang;citation_author=K. Anderson;citation_title=Functional Characterization of the S. cerevisiae Genome by Gene Deletion and Parallel Analysis;citation_pages=901-906;citation_volume=285;citation_year=1999;citation_issue=5429;citation_pmid=10436161;citation_doi=10.1126/science.285.5429.901" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=H. Xu;citation_author=J. D. Boeke;citation_title=High-frequency deletion between homologous sequences during retrotransposition of Ty elements in Saccharomyces cerevisiae.;citation_pages=8553-8557;citation_volume=84;citation_year=1987;citation_issue=23;citation_pmid=2825195;citation_doi=10.1073/pnas.84.23.8553" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genomics and human genetics;citation_journal_abbrev=Annu Rev Genomics Hum Genet;citation_author=F. Zhang;citation_author=W. Gu;citation_author=M. E. Hurles;citation_author=J. R. Lupski;citation_title=Copy number variation in human health, disease, and evolution.;citation_pages=451-481;citation_volume=10;citation_year=2009;citation_pmid=19715442;citation_doi=10.1146/annurev.genom.9.081307.164217" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Extensive Copy Number Variation in Fermentation-Related Genes Among Saccharomyces cerevisiae Wine Strains" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
<meta name="twitter:description" content="Due to the importance of Saccharomyces cerevisiae in wine-making, the genomic variation of wine yeast strains has been extensively studied. One of the major insights stemming from these studies is that wine yeast strains harbor low levels of genetic diversity in the form of single nucleotide polymorphisms (SNPs). Genomic structural variants, such as copy number (CN) variants, are another major type of variation segregating in natural populations. To test whether genetic diversity in CN variation is also low across wine yeast strains, we examined genome-wide levels of CN variation in 132 whole-genome sequences of S. cerevisiae wine strains. We found an average of 97.8 CN variable regions (CNVRs) affecting ∼4% of the genome per strain. Using two different measures of CN diversity, we found that gene families involved in fermentation-related processes such as copper resistance ( CUP ), flocculation ( FLO ), and glucose metabolism ( HXT ), as well as the SNO gene family whose members are expressed before or during the diauxic shift, showed substantial CN diversity across the 132 strains examined. Importantly, these same gene families have been shown, through comparative transcriptomic and functional assays, to be associated with adaptation to the wine fermentation environment. Our results suggest that CN variation is a substantial contributor to the genomic diversity of wine yeast strains, and identify several candidate loci whose levels of CN variation may affect the adaptation and performance of wine yeast strains during fermentation." />
<meta name="og-title" property="og:title" content="Extensive Copy Number Variation in Fermentation-Related Genes Among Saccharomyces cerevisiae Wine Strains" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/5/1475" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Due to the importance of Saccharomyces cerevisiae in wine-making, the genomic variation of wine yeast strains has been extensively studied. One of the major insights stemming from these studies is that wine yeast strains harbor low levels of genetic diversity in the form of single nucleotide polymorphisms (SNPs). Genomic structural variants, such as copy number (CN) variants, are another major type of variation segregating in natural populations. To test whether genetic diversity in CN variation is also low across wine yeast strains, we examined genome-wide levels of CN variation in 132 whole-genome sequences of S. cerevisiae wine strains. We found an average of 97.8 CN variable regions (CNVRs) affecting ∼4% of the genome per strain. Using two different measures of CN diversity, we found that gene families involved in fermentation-related processes such as copper resistance ( CUP ), flocculation ( FLO ), and glucose metabolism ( HXT ), as well as the SNO gene family whose members are expressed before or during the diauxic shift, showed substantial CN diversity across the 132 strains examined. Importantly, these same gene families have been shown, through comparative transcriptomic and functional assays, to be associated with adaptation to the wine fermentation environment. Our results suggest that CN variation is a substantial contributor to the genomic diversity of wine yeast strains, and identify several candidate loci whose levels of CN variation may affect the adaptation and performance of wine yeast strains during fermentation." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
    <title>Extensive Copy Number Variation in Fermentation-Related Genes Among Saccharomyces cerevisiae Wine Strains | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"mMeQxUxXr8qKIkCcQd4XF04OFvzFoehl3DQEmrGLmQE","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"484258","apath":"\/ggg\/7\/5\/1475.atom","pisa":"ggg;7\/5\/1475","ac":{"\/ggg\/7\/5\/1475.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/5\/1475.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/5\/1475"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/5\/1475"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/5\/1475":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022MdcuCb4LS2DNdzWuh57CZDOEvsqry+uE9AyFRBvRvm\\\\\\\/MATGtRU1QwBDTlM4tGWCmTlZHN3lwyNn\\\\\\\/8w7oabAdUCVvUIoVKifPebTyrdkNA\\\\\\\/\\\\\\\/vWqCnQe5ilXpXneeuiTN5f7+Y4EsOLY8E5YcmtxAJSb4K8giQaBD9BD4MkZeap4ZUaFkPZ0vtZKYrIbXCQNhedfHg193KfjO7XilcH0pQ+ElNpcksmWOEIGf9675skNQBVzHcBSPn2OoRTnWGCCRhgtq5pwl53Oz8rOs1z5sRwMvgcFGbt+ceRfy68iZqfPr5YuJbrJujlrSQ8cMjnZfjpcxTwFN4whP0xkijchuPI5f21s9KvGouBG+DMuyx0goAavd9xBa4lQXy4yaypyScbL8vC6IVA8vTxk1tUea9WueLo3dQAma7KpWdUsQT2U+x9tFx1yImYm7NJLvD6gMrBWpQsBSB45L8PwBEHDf5F0o7TwfJtszoM7mkmjVjrPsj8T60v6ivlN4MH0XGxEbFdy8WrGirVVN5zDePUR99r+ViDjhWms8WFAA7po+rqPWCo3ZgfylP9hrGMUgdhJ8C8ESuVMNOGdmDUPSOhIhAczXXe07a0TpfYcqRUMBrDIl1sLz\\\\\\\/JaMIrZ\\\\\\\/IOzfaCEQkg62sadSnpA4dW8TQzVbe2jFZox0LdIbZPYWqqwzjlk3GDWRKV55FBxG7j8DGnAJHsvY66E5P4yZ0zi2LaJylqGcRZSt5IzXLwfUwQ7mTNyJvU2QabvGy0EWlooMveVqEDeEgDPO8V0pV0GV8ySYiQFiHsC36UOOYmjtFktpCfUp1OUXEYtqJt7GP9Ih27ZaXiKDN8KQ5GZY8ccZX+Ui7ZBrTy98H2BH5Puw31nVDB4GPBM5Kku5ucuuuJUbFztJZ0OuPmW\\\\\\\/t1x8+MmAlPaot5inYCa+\\\\\\\/XvHrvunOuipZC6r39aQagydbgwUDDF4SfXaUAJLlBQwv7vPtXqlnbGYcCyz4szv0AN4pZC6UVifgymS9EEke8mWj9NDb66lfu1JJaJ+IY89hGCcLdudMGRndJg3ApdUt7Fm7u00jhCQ+7lSMfYiSg0\\\\\\\/v0LyRoPBicu\\\\\\\/clez169leTHTFuogpp3HNy1zsDOIpNWbw9H5RCjjhtwKIKuIPT56CQnEL3gIR9kb1kxePK6O8a4haL0Ti8wb4ERHidaCNGLhaxBv4iy0yNrU\\\\\\\/qF1TtxCMCHOYry\\\\\\\/C+Kv8HMq5JzVELH1RCT3YHA32AG5i9aufWcR+axkumOlpbtmjD0nVLJQU8PWaUDhUt7P4\\\u0022,\\\u0022iv\\\u0022:\\\u0022pz1NiU0XwwfaYJtchZePHQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220915e2250f27c1f0e5480cd37bed9992154dc72306d80e3d37eebaff34d6e313\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/bdqKTEqM5Mf736jDv5SNwJ4WNzcnAX8SFwMb6UEvyaYjuEVtW9nG+xcKeCsLRAG0762fvaq8N9vh2G+bSzqIZRJGLJLEfXYov6fR+kSK5009nWPyIirhXVbXyiDfjM3Y34N8p3bbvN5LM252SmK3VfdxKweLFqBB1DJz+uhQSpqDcvjngpwPLi6F7qfBl930DxWEIcxFMxAEM6HUzHFK2HGWM2NaOgDADbjtEtxYTefg8WxJJhXUDpSY3IMauXV5qGj1JY8FcRX3mbgJ3JM8CffTz+N7qbUfWMWR68E554kTOczRHttBbyka5RT34QyCyKt\\\\\\\/il1P\\\\\\\/9i0RosG892NUSe8XO2aErDijzq0M6\\\\\\\/z7JboBMKpUjnckonEUiaTonxkp8h9lBhSeX1StgB4HLZMcxSae23\\\\\\\/v6jz12f40RgMYhGLflaDBphQhk3R3Z4m4ioaOt\\\\\\\/AO85EozWVCTeuBn3DElYLRlw9u0YPan6qcSrOvn6sP24yIr42ekBPpggH8UkVtEnPLyodaa\\\\\\\/rOdPVMEMFuhcWvfKr1JtCnlejCECmIGQLnmN2RB28\\\\\\\/WT450CkTs84a5vv8sVePJVQQoJ2vmveZOrp9sqmxaszBjKT3LWI5e1Z0c1s\\\\\\\/SY8yWkv7+itYErKq9U7EL0\\\\\\\/jweRbanZVqHW\\\\\\\/y1+VDHYWwYfrIZUjDtT6pT8i+z0jLXS0XwIdQTTMsE0ym1+X\\\\\\\/S3x5VFeDgr6JusVnl4IrEW5qpPjWLlW209aRuLLZ3CPpCrNyTCtx0Xf26XL70x\\\\\\\/ysIqQc0N53flCqbzU9TOk5zOOYRjF8g48jz79XLcM\\\\\\\/b6aiqsVBRNI8qO6IRZDR+Iwlj4YOvCrhw2lI2N1gcfSPXVuyIZwegRgaPpEp8MaVE+ZpaTuoSH4I6dTnQ08ViFl5Ropfw02jWS\\\\\\\/MJ94wjs+UxRWLP0A3sEYw7ONDE6IPH4YkQVYABafH3slsKfyUHrb98a\\\\\\\/RtMFpnfNqWvQK2du8+JszCXNM1j97952tdJA7LNeka8JXCn3uE9rTJqgNZsA4DRW41ZFkrBkboUtdlKhd1VjBRfbZUttyhMlc1LLFIAHDbl2yZJMdVFrrxMblMBSOLRVdV6dDVEdzKZGfDQMoigldo0M2no7BdOJ\\\\\\\/7sR3vEEpK7MjlIc9faJrNYyhZrQh3bze3umflzsBRzS8Qp4UCjUSgjSnUOGTWX+jktL+O1KTvr1aG\\\\\\\/pJqkeH4UKHkD9GnNqOjbit3wQZX4u8mwTcAvXu8t8JFkRXz\\\\\\\/ZhaHeVGH93UYrTrPbri5z+W0kvf6Kn6beo8a3qyCWnNn0IKzUUmdTAlJKg8HjYfRVuibI1omOHo4Dr9ACr4mho7vL2tqQJTyq2un98Rd\\\\\\\/+2tLVtJcNcO0CthZnX\\\\\\\/YfNF+CW+y1GtbczO3bllp8GSIGsYilVxXAlEKrnd\\\\\\\/7gV6RyzVZUg9dsCn9uZ3zgZRdJA2MWOmpdCFUxnNI73nW9wyxghH2a+JrDVeVEjnhdcAfMxLkuYLSZnimDragH3YRhoWaH6MllUdb8gAz8fq3Zt6FV9Dh4AySCO2N\\\\\\\/ZJNY\\\\\\\/We3RAn3zwjhAHjjbC4hmRIlG6bf7wq5\\\\\\\/Bmwjcky5+kw3YeSEnrFYZcfyXYSEQhTTguH1hnjAPxLHRVavUj4WKnouS4xRSXWl8qW5al\\\\\\\/bj0qrUyNBIfjWMtNEQKrHFkAVO8PHr+d0AuNI9zxieV364FcP0XoqR6ZoEAYntI6CWGpXjOZ+vtoa5I3QQPqQ4URzqOJXCx1X9uwGsRhxrvn6yBszb83hz5EANUGAhfTmtPhnJV7JDb9BkIc0vq\\\\\\\/m2X3MhQeLOxmnXC3XAaW0i\\\\\\\/qIln6HXAcAa77FZGNAUghlWNs0W0Vdl6oRQFNlxkSQB7M5\\\\\\\/2pq74YFDn1ZB4oLLY9Bfypo7xq5O14QQfRgu9QbRoglI41mjM6Rf6\\\\\\\/ngsl7kQ+6Zj1C+3RE4SwLbOqArJ9G6HAnj0cCrVh9PGLfU4RwarwOtqzeMdXeujDluHFgzNAhdrUvduMbEzE7QWXxEwHlVt7CFMAJge2Og+0uX5JxMhIpuz9O+pR4zWDE6erTu6Mg7m822dRoHPIyHIB6ET1+uHTAy75dF1f6KN7McbbKeL4U1XJrlAzPtW\\\\\\\/gWY6A6\\\\\\\/cg5yF+SL8sAv\\\\\\\/pmVzq11OGYh+BnOEnLru0N4NgHui6DawsSbjDuXJ0\\\\\\\/Oktjd8vCYW45DsbtgLKOGhhnaE4F\\\\\\\/t5ta+kRLh6LARGC4EvBO64IKO3vqsSPoFTvesSskYbTchp+PpZkO5h09RO1GG2BRYwMi2z6ddSiOW14pXgBm0eGDCw9N9muYQE80\\\\\\\/r5+1Xfr9FNOGruS7BIJlIkSz\\\\\\\/gNlt8zXJuf4n4wiiad2sn7oTh4cUzBU\\\\\\\/bYSPD+nQHiYSucs6Htr6OGgiL0wm2D9\\\\\\\/BjwVsSb1p9FsIeTPXA7dK+gMTxeObm\\\\\\\/sxYzrIN3QvMnbw1dKIEO1\\\\\\\/PxTCOVHXDCogj3\\\\\\\/fSSWbiWu6K3+oOqkblY2iTltxcBZCK+KIjOQqDgfWeN6IIKuNMOP+qoY7e6cyzTAM3TULUvPEjLVPLwoCz\\\\\\\/34gE+iDbA7mScKY+1vlvrn61dUjnZ\\\\\\\/CW2gC7AWANQiPzg1iQDcH\\\\\\\/P3B5jl7qjUH1EvrcWZBhP8QJVklel0BcRFJ2dMa8oMQAc78k4CMFh4T+Yzalh7EOGKa+CgP1SkPZ7foeodGEZlHg1FO+4z7RyvvD45bxlygk0RjN8UI0eMRvdQeO7pydi501MHF7tUKZHUjQEXGBLq1ZQj1MjTM3DF121PB+FzAXmeSWGjRkcQfd317XgdZ+cdTcGq\\\\\\\/AHFEHl2kY1ZEAnj2RB0sJyzArS1+knez28FhPsrHmTc\\\\\\\/vraUe+Zrz0zQ2SA7SaxvsercdsP\\\\\\\/Q8d939sveNSrDi7a7u1iAiizZet6ty4NYAbfJ+moFfjmnBp5K5CwWUExFw+ZGfV9fPEttfgzWcqwX8586Cqz9T\\\\\\\/C0zZLJC76kXJDcoof6TC+jqnHfzZ9lOQmZXaT5aEFiN668oE\\\\\\\/4z04YyWpru6CaXHNyXTUr\\\\\\\/sUR\\\\\\\/0lBlZ9ZXiXOWf9HpF62ce8asFC0Vx\\\\\\\/zInabEtAYEE542P+e7tE0q8awhT2+CEG3\\\\\\\/vSoFpDT0l5en+IY4+3iW4UaqGBFM5vbB7aLrGF69D7ly1d2hQ4epVlrcWhUYlnri8K1GlCfVkl8LbD5QOXc1oySFnEzLzcDvyoOCLpRHyoHMYaq4fQ3CnPM8PBKku\\\\\\\/QDsCfQEoti\\\\\\\/i2Nt\\\\\\\/ZCrYYkuOQNar7nnSTo75nzogOQOyFt4JXdokrDibC5tHYT3Y\\\\\\\/Qx2qZzRfEJmxTOcksGS0\\\\\\\/Sjh7zJZ\\\\\\\/ZkD6uX+vW7jiSXgWhSyFJStzRXJswcQPFdK+PxYyJP9jl4u2S4sguh30DvzlOCPmontmWNn7oJnMxQV4KTlpOElfUI8JFCKdMG1\\\\\\\/qysZE4BQg79uCWyPtz6oRBoOpCcemdHstBxgTLQe3DkH+BXX\\\\\\\/RPKCbF1m+9hVGqD7sOgppNBdh+s\\\\\\\/N4SOE+i3WevO0zAt1toW44wdka278ZyBZaxF6671ZbxpotlQWUxLxPfGVV5ueAg5jibkE8R0iR9BRbqG5EzlH0CigDTiHsRGG2ZmWHZTmGMQ+P26vtlxX0kddhlaM95mC2Xm+VNZ9UOb8ojtrNUNwewDvBSZ1CSMZJFY7NX01XMXNtr9sUrD7dWq2\\\\\\\/ouU38sq9xsjmf\\\\\\\/nQ8almvduywVc4+7J1ud7og4Mtwn+Z2VbBSA9YnmMX2ro1XeHamuZIp9T6ja4vCSloOddF7LCC5FRYL\\\\\\\/\\\\\\\/\\\\\\\/a1gP7McZYlHM768wbeQkJNV6gt+Dk3M42YPdFeRcYddoq2KooX3CgUgFYPxjpStBFSmMjoaH38\\\\\\\/t7E6jrYPOryL26O5\\\\\\\/Debx5SL8MxyeU\\\\\\\/wdbj883Ath5SX66\\\\\\\/7V+QGIHHYiW5o9FVmBfXY+ASvvJXkiDpoFK8dzBW0MjA9fznK5qEI+6+xk1X7pcW4oy5nqbT+wB3zoJWv0WJ1cpY0hd6tLTtAeTwGR3Q15xmbY9tnGJXT77D\\\\\\\/s3ebi+7lNfCmddPZdc\\\\\\\/t85gXOQ9t4c3+pCB9UpyYWLaeHFmYvDnk84IltfLMWCzfEziVeqnILR7ZJd\\\\\\\/nR1DQB9gNESsltQon1M7oS5F96zymaeQYdhyLbGssOtzis8qxUkT+sLnsvZk7WSEjbFfmJWHO4K1I930Fy2b+zc2mLQSafiA4eDLymWorA8vPeaBoEGacEszc6HzfAKGObbVVPQcZVSY2IuUqkjcTIVH3s4ONRTjCOTXMia35P4Vr0b8U95Yibp4GZPbSjLyxWO6jNFMBVcF7cAQAFOblWK\\\\\\\/Zll\\\\\\\/A0Exr2nx1JNIUvBql4IryXaZ7dkkOuRRkxfx7yxR7jW1TShhnbBJh3pd115McVGa4S7UAJikUG3qe3lltgwvKr8vCCwaT2DXygRqeh2MYk69SpcImhxgNoax1bEEC9WWiRHcU2qnmD+kGVmDCGGLzSbpDhV1pxHAzGKPELEdJ4Vq4sSKyC+oP\\\\\\\/ebFdB92DhrBDTIDY0s7aWAAgjd7k6cw\\\\\\\/YKtHixcM3NYMomFbpJeZaLftkOAzRqY1LzSwU3wYHko4Iz7YJVy70n3IxxSBb\\\\\\\/5iZuyOuAL74EhPZfVocPxGktJ8Vsb6pFx8HAaAEV8yOXmF2T1eF0cT+ChpMbX3GIb6UTTFuLSdq+ojjDBPRIZP+zdJlo3zmlH6Ah52HzV9K\\\\\\\/xaljMaVSEhn\\\\\\\/RI68VGiPHu8z82QNUseN\\\\\\\/LGO2u3ZDOUs7aCC3iFHZV+cXIkZpli\\\\\\\/p7gIfyMVEBmEEjFRrV9xUHXUUEs59Mj8YPjV\\\\\\\/TZceYg49+IZdHwX9VAo0ZNCO0kqS8PV\\\\\\\/yWgnJAlM8kY6E22eDfadRPKsbK8PCjGpcQiH8YF3ho8FiFYbBSFgXyPShQkJ4Vgw0H3hdX9GH7fN+hPVZMdsJ\\\\\\\/w6KwNdSe8YbJXKtW6D2V2i06CFG\\\\\\\/Oa\\\\\\\/iqxwr\\\\\\\/QwQsSGrx25ugZhX4vNkAJZixQA6L0Yfqc7wc7O2+N2D9+HBJxJVuE9MjKObhmKwIQHUeV0hUXesmI32mXxibRgdgTRWTao34tDiDur7ELi1WXyOKLRrTI+veFOV4IVq51oUoXpGuLcR\\\\\\\/zgrNg3PBWvErrzKq+rWbegsHvMLIr7SYByZbgJSa7WN6o2czdz7XaB\\\\\\\/6jmqlcOVJJku6ttTZlhUSkskEm1ijBNOH9DuN2ZQ4I1T5cH4V7+1x7rCZPRlOr\\\\\\\/5Bwjz2hby89icP8jsqXdYGeBCywWrRqIZyJxacIkPuU3Hkr7g1fVxpdJhEEWpeTOOPJZKZ1JA9p\\\\\\\/EyOCe0A2fOH3bnJruSTfQVDkoRfzMOGv8NQv3tqiuEy\\\\\\\/hKfMXx9kcNdez7JPgDv1VvVlSf0k0bp3GflqdTb6XAC\\\\\\\/AuYU5qmNjijv459ToYByTF87TmmJyt+3MeNDa5jHSXdWslMc\\\\\\\/8v+7zG\\\\\\\/eSd8jWddvU8N65VHwJnNBjkySY5nXpN2zw8kTzk9op+W6AqTz7uKAoRBygQBpjPqjNA++SedL+lnyd+AAglIaQqWxpXOsX9Z+o1AbiviF+o9cgEFKCueY9fgOj8hQxWSZvp7olEwnE8ovkH0FP7SHXgjqdhaE17UNd7RtWpuaHOcRLyB18td8dGh2u42CkW3Rmc92xj5ItD1xmJw6ml9WqcrZqgsOv2VKNp4QYkmSvXsFdvW5+uK39d4UqgDfQpg9QCoRXdP3VTSbDAGCMQVQDP4X8Vl03QVwplk+o6++lJkOYJU9mlgyUY1M0C9CnoSVancBm3dUf3mCvFs4o4kyITS\\\\\\\/lIx476RCX2cc40oVzq+2jcyCIfc1ukQWb87GSaw3MOYSUjPH\\\\\\\/Tl+dEcs7fym3tpyZmq5TTkhYAKfc+KeSsm\\\\\\\/sZTcOSV+mDysHmrH2jS2AML1ccQMBlr\\\\\\\/gA1sxm5IxXWN1BOOErA3kiyxqdGsitvlujJ9Eo5BTGBnY2VW5xsLlmWqUszB5vxmA\\\\\\\/tuZT8lE1hofWUBS8r1lhpjD\\\\\\\/8kF4SrxaVR3M+w\\\\\\\/SEFm0MuSbyNZ5hUQjd4OqPeLfZyTIt4xOSqY4KAWCkonvuIHTJ0gbKColgrBEgvhRbDzM\\\\\\\/6J1Ek\\\\\\\/Vo0E7H95T6QueOG+qOgeg78ILV0QsxsV1RY+aW2NUL0LEZbZLHhFCh\\\\\\\/C0cp3dI8n1E7rjY6pWdECiX9Tl6mGFCmZE972xctjQGQAkaV\\\\\\\/+US70dIlkVRYaEj6urprfPrpbwKzlt94jWuzKKsrKDMiYTENXWl5esabMINhrq+EFrxPueq8zrDe37glzbhanexEhqjxNI9m61ABe3iKLIe4y3VpsKIH0dI2cxYBe8HSswafJhqG3ua\\\\\\\/dV9EVhYk7xvvk+e2hB5AeafIa78yOndunh15Q3fMlda8DV2gLZH4pFFFwLElvyaLJQQ86t\\\\\\\/Qw6UuCCMoSyrS7qipGWAzbt602VCwibhdg0TccAdw4Fh2p84SIm16xaLX35oHj352VC7b7h4cMiIxIToYy+y5g1MjNC8WL+7e\\\\\\\/kE+rrU\\\\\\\/4EgbhMagwaTwj3QTTQRW6txqeeyxBTZGHxBF8HyFwFU\\\\\\\/blt5mXHgwHUeyO6t7inRG72GOicTVOe4qUNUTM2G+JLRrWJWqZbg044w+7J3VL7Sl0lda58tES1w\\\\\\\/wKhE3PjGnZiQhYJn3Q2RltDn608Nff0ZYvihJGKJC3PF8983F2EysKkIrV6QfvhX\\\\\\\/wd2tdr0CUQqCEHmdRQoK\\\\\\\/Iwnbv7nu4tNar8uAYkS+2E+0pLWObK1aBcaI\\\\\\\/WRV6H\\\\\\\/Gg5YPqwI8BuhQbI2kCVpxThG3u5LojxIh0r7K4SseSXaE36fmhYl+F9Z+9AxNsiXTg8wOK08gmqCSwT1f2yi+\\\\\\\/C+rW5RyNjrOgp4PpjoNqPmodn+WepyHUPrERtGnyjMVgvkzAhUllR0HFLftzHmklE+SFUORgOQFyzpWNISuG4yFKLdv3+davoaMCmdJrA1oPGTpr2WFM9Ay9C\\\\\\\/zL3hD3KAktegwftFLPHOkSkkgdI6sNkLOigUZ4xImIb7tB2z0QN79kuIppk5eQrntSL2wj2\\\\\\\/MRy8D084Ni1J9\\\\\\\/eIF9Kj+9S++daoNt75ugjmwETzFV+4hQSDKvM+u8oZ+hdw3xqhKuiyL9Sredqm0eNDL4G0MuZLQPApwZ4Q\\\\\\\/2uxSQVkGnqruTE\\\\\\\/k5Ov4vd6oUL1vswOhdM\\\\\\\/i9Q1zGbqQxs+4c13eNo6yspZupGMcrM\\\\\\\/EgcbiHEoKruDetE2sECiI5mxTWD1hWebo2Tdb7\\\\\\\/ygkpY\\\\\\\/5M5UX5cMQllSSEvbNqg9Xf++R1b022+1vw61FIMEOFLM5edK2iBIV4U7Jnaom5ZjHTHHjxB0FzRw0aeY5AVbx8\\\\\\\/jEuTU\\\\\\\/6TcefsQZMyfyMXOr8xkrANUczimZPubCNkbMZvB754o50MvV2QcqsjAQpRbDm90iyL7no3z+Xei4hUvp7yVNtI8THJvdgMyzrvNRZQ+OdjXo7Dpyr8jI1bbcOj\\\\\\\/LF0r1yod+RmyTjJcpRKUR4A7zYH+paiuHVC\\\\\\\/OnpXaPfYrxfEfSv8jzz+Fuk1\\\\\\\/y4v9vE5fy7zr5\\\\\\\/hvbf1it40YpTPyWv6wwT6bMiOmJuPKclAg4CWFgHGps2nqtePeDZpRu29kXIgezwRZ5sPmYIKJt5fBM20HKb0yL8jET3sVN9srn9zo\\\\\\\/ik3t3aZP9GYfMzIbLLYtWqGt4usjLu9M4cnTBZ25yJ\\\\\\\/Y4XQYtzJfW4wikJx28E8mh6MXZTliYVWooqVHrLWTsRiA1xA\\\\\\\/vqzxBBxkn0jch6dwkaPjKYNy11iPtOr\\\\\\\/H4xrrXMgUYKKNqEXA4TiczdDZVg3vAumqr2r79JydcB9CmDRHg5baLShP2xYekWuUJTBpXxo5L7+wvVTk9jJewb+6VxmPFZN\\\\\\\/r5pNXKLEaiJm8tBwLaIwPLmU8aHCGUhNa\\\\\\\/7HHMgZjNe6ZqpwfFACdOb+hoKVtA0UlotKBcm4j1d9OoyIWInfEkkUngDt5UcYaGrsL9WeNDZBc9vqb9P9xz4qlvh9e5n0RJ0pXD8Ty1OvQa3dv+tyT1vGWf2bd1rGUPYb\\\\\\\/bDVhnaFJrMJdXqOIrC+qbVMauChLgi9GnewcTGUJBXfytRaprSVIPhuLRJ4G1NVUuROagAaN9df3\\\\\\\/8qSWOT0pS6ohjY4Xus1PnFqsCUDnDYfQskR04bY13dnfAJGJrYRmz3VgL2uqWYuPm21dDpt1Nz0BBFPjpuYVIZOV\\\\\\\/xDqHJWocidvg0fNtm1hVW80t2yPKRZoqhJ3PNE\\\\\\\/9DuN4XoN4OEc+va24V\\\\\\\/1vSWaokbpV+8JXjM7i6Epz\\\\\\\/6O7+Va+lhP664UHz4fwyQhXNeh0t2ALmPrSyDG88bc8uEiY3GEy8zf+AmZ7ENA0CaVoi6JgPhAKZE1KrLFVeX9RZ4hKft8zQ21+2b43aH\\\\\\\/pKifQMBpmbhoUbX2VagMIRwkGHMBLgaENklwSVHOHsyoTBp9KuKCCF2Nw4jrKZb5LdeW50pBlr0cJ04pY0dBhLr9MlSb6Ujr8PBTPhl0vugOMssnwwPOUFaZ0qxoscicKUMVz0\\\\\\\/KcpNBsNMGH6I1a1Cd+rp4OAcDTfDVSPUBnvrbSYr9IaCr3wG7lefozRNtBRdSRmk+hm0cmsLNc5HPpCxgOQfItrD26Af\\\\\\\/P+FCoQXjViG2hqO0Eu4xSiR1d0i48jq7c+xkoL4yROaOCjVxazVxkU2Vx9uACnL2mS77Kc+VkVhlsvxAUk0LWpumgv4SkmOM0hm6QTd71QogYbANU3ay2Zfmf+5hO0nXvzCPI37R9FeIfr9A6CRJPjXeJ1TLTjir7sxH9vttsmigH500YtwtKfe+wqCES\\\\\\\/fvEiwqaXO+YJe66eeGTVKTpgC6gh1NjJIW6ldcd7MO+BmSI8SJbCPjr0adLlWu4UX1nf8h8LFM8PWDxQiwnGeQvSlripJ2eWhpsVgA7OkWBX87hCMXxzlDxCyHKd2rlhDUGhCmvtfEIxWfJlx06bShGcWFzGFL9kXDctkB\\\\\\\/iW6Y9pANOrkSrrKRKl5AYAzWRN7ASEF61ARxOgM+WvEvNVPIQ5ET\\\\\\\/hn1w5B2uITGq78uPJ6MBR\\\\\\\/OlKL2fZM\\\\\\\/wzjYzCxzv48oxCxemkWRPoxULkvmM5DTswEbRO9b\\\\\\\/9DmYrkg0P2raBzFd6VO53pos81e1HtEsmIu3k1cKKprTGc5fCqwHiDzXhKhZyP+5E9oKf1khtl0I2GnePQJ8LS8QYYBDXApfdfWa2zKrpt7OLOdVcSvzqQ+STFs7EwD2D2IlOHz6kudbHL3PM7gSIgneUutfKnZCmabM1bSxX3nygWQcFcOceKN83WHqFe6eMGVHlI7VbHEhagO7nHw2T0IsB2FXKGlSfDJzbXMsnb1gqAqPglG\\\\\\\/6lYq2o1VAI+VQo91SsjbcYuG5P6\\\\\\\/65l5zvs02Zv9kxhkYfmmwjn44ALWoKJKoRpc0zYr9N8vGIWRUNSyRerHqgiC6ULcmvHV3IdaxFf7+tnL1MD9TT6cBVIDZEa1NWWXHevce6HLhUSzgQFHNGcgOh3AFfsAJ4OjNnwkYVfkn8MOenVzCmlj0JXYVHi2ksBk6t\\\\\\\/\\\\\\\/exgvRrknzmZeZcVqgsmMa3JB7sV1JqQ7zJqfMM2uDKIEgNKF9T45HZUJ9++wlMA4sTBkjfLPfdH+3SgkghmMfVeAxu1CijBMNlMPDYKX3AE0IW4y2oqAF8DEnPfLK0t4aYf6SWOkrGeJ4K3+YIwfHg8uunSmcqXTyAAyjx7klRJOEXEEiPEgrlsJW9Nl3TZVqCWqjW4cDwaJQfVTeA0ymaBLKM4zbpCkld3\\\\\\\/DuFqFmdneSwMXRaAYIuG3tm1pOQ7t+TKNUSogA0hCrFDw1gwkHf+BbnFyAdiRt6aMHgAEmBSrYxNyceMcRpaUxxomqip8OlGmkJ3h9uj2581j4I89faqlzIDVVNg138XXqVnOB7\\\\\\\/vSBFTmAP9PvzWQiXvpDuKQO32InzG3o1o\\\\\\\/BY\\\\\\\/TjD\\\\\\\/MvLZracFP0th1fIJoD19GGuT2lDuoYva\\\\\\\/4PSXu0Hv8Cs8oeVZmhYR0CppEeoThns1mnZp9zDq5mSRZNPlJJ\\\\\\\/UXhYqyLNZNQlavytBBUcP56oIWNqaTxl0qQMAkymQS4WnH8JbsH1Yc2QkG9597gyNJi4H1DWEbnQxKffgkbJwlHhIh3duwXQyMuAtLAPvLxRnJVOjYBl+VRY820seaHhVhH7C1cP8LG3A+S81TV5Su3NgsZqdMdn7Qm\\\\\\\/bly1wc2iF+tDrB6N1Ta90V80YV5cVTXjPvafHlLiPH13P8ywTpfCSQdL9iV2Jkx5qZl5CQZkcIVD5dQs\\\\\\\/6DXIjzw8+7xqPgTjTy9wBXv5qLgPiT9ImGYkmJkkN+dNYxYDTGeV77q2nIqZMf9vCVJqW0tVMAVNhTrTcUjT+usaZ1NWD+mPfVmiuDHZqN2PqLNJ\\\\\\\/5FFL8CmeriT4tps8ONR9pktxujplo0WQIlsULBWx5wU3gcIo6d4u9+nXt2fl6G+5cD0JAMtaSVqaBCBUzAU0sI+kQVjMYsyADNex2bTq\\\\\\\/G9VBRLWmL0WSscvgtairmz6XJiZQOMY1W9YI8q6IJHSRjXTszvlOB1OkRThEvasPtbFWrp7HKLl2vAJS3ENd3eufXH97A8YriYHFtNlEW1+kXxFFrxKbrL71GQGAons0nYnHS+1Bnap3Gv4beV6zYPZE2Ke1eng8v1opOtow8y9pyVT5QA+rguigmkP0z78IigkBi2m6gFTz\\\\\\\/2xxanitMdGBvuajsOT4KUAc8ph2BBpulGpxbVZ0zY3oCBTZWMRixU1VwMX9gBS02JSObhB\\\\\\\/UWO7PdBChUjKgwdKAQXzBlzvqGVDxjy4zxpEm1ey4tYJoaDmQYFzmCMjgMzejyibRgiZEv68CNbXfFIqeT2pDCsU8QXUv+0vLlEB+KN0CqmAWculNrUn2mSqPknAY18WRDoaECw9s0lEOG5sXLaQhiNkQ9cOoJ+ppoOTd3bOPboJoU6Qa4bwz\\\\\\\/dWVn+fG6YrtLmMg01ziFw+VhCww1BoKnxtmsTCb7itHoZopr4YQhVMfLq3gXbFYsOcFLoT15G13e7f+5D61sWhVIQoxm65hKsUC1yE+pOgvwlFWjnQXkrMOYHHvHwPiOe\\\\\\\/o9SupfPqpsiDWnP3wC2jsMCQ32RtZjhL34BZc1T3qHc1jcQ2cUhYWWKR0Jloshf8xcEZHbsaSHoEIwl8uzLg9Atj4FbgXXKu6fDiA4gFrVG7iTmVNClTo9kMNFeF\\\\\\\/NDkplQuYHL3Q+key5M+LRMosgKxSxGdydgVP9IYyk5ogBEvmxf\\\\\\\/cetARwrkf\\\\\\\/NLAfmRbFLPrB8YaZqRUYd4f7H0HzXrvPlpTsHzZPAIpIjlno1+ZrAL58Qeao5cCn68Eso2WkL\\\\\\\/kdIqimf8ovzhDpmhUw8vSEXY7mtZbnF6\\\\\\\/mYvPs1+DmgZ\\\\\\\/Qy67N4UsAC1JbkvpB5Cj8GIiGqPD7U2bN2fNuq1SHaSX4wn12temNGGcqkAQoEm\\\\\\\/h5g66i6dR3cHXD4uFS3XmlO2KntUoP34dyqM7\\\\\\\/rCnsrhEhiCO5PqVyNLycYxtjaWRLo9HNdUVhi+aMI5mr\\\\\\\/DebpasO7EKEBVxtNQJS8BLj5+SURy6d9jZdZajqVMQNtCCZxFWASsgK\\\\\\\/4BFdTl+U0Dzb1N4nOnguRTu97eyoL4Jse0cOsKmeYTDFeOVN4XPdeaDQXjNIg9EmIOFpYagDQ8dEqmHCiIPBBYNy4U\\\\\\\/crpnR6pxW0K4MJEZo2LtIptCcLqsqSqo6S1oM9wmax3eJ0Z5qQ6Vqap074IXjaDw\\\\\\\/3YXgC7UQF5mloK04KM\\\\\\\/PUQG+vvAFz2RcIob9mY089KEChjeA68q+iQv\\\\\\\/BXuqUCAmKauvr7vAyvXYwnThF8zh1vYRO\\\\\\\/0E5A0PRI2umBOcCb\\\\\\\/1nHmC82YYrMga\\\\\\\/DlPxTqMQp3bXz3caMk50mp2N4O4qC3oe4muqP3U8q2tR9UixNWfPbm0SC4pYHK37cv+SGLdyoFdQr0CNoSbCnEheJQjMugkalQYU4e1vaMhswMSSsr3M7uLS6mgoujNkdQCoENELjnC0GSwaAY1klDQRBAJF6M+lal3j6xtPKNdUxaNbVsk0HukcCCoSZWiSXSkDEd5uoHnG2Pz9c0NRr0WSBKIVLf4ffdTSgOjicCofA7FTCoJz9XEOb\\\\\\\/3\\\\\\\/uGBcasUeUdIbQLfzceBxwGLI+icgU4NUOFUTj+wGIoqrYj+jTFIIM1rmKruFlIkn47TH9sXrizBb+vbtpbTL4TteBKu3Yhj7arwak0H9WnZJXJuzYHUE6DARAgigZX9xkxzLNdinTaeqHWLCgoLWTOGq2LYPxZ1VoyOUL919V8n1VRa4xjqymytJN8wizpPLDN67lkSTmkpWrqnQ3Ar8v3B6nG6Gom2v\\\\\\\/P3WgkRQ30OWzrbx950tzWg8hIid26iuXdzD6iKPAKEZSyTs7nNOY\\\\\\\/poqoiL7sztP06zDhJYvWlaWpLGnDNKOLq6HSC6\\\\\\\/1BQ+m\\\\\\\/2YPQjd+yCahqCZUCFDJxNR+ox6jIHArwBAZD86KRi1GX1Ogw9x4t3RpWG\\\\\\\/fFNcDzys7dFQo\\\\\\\/\\\\\\\/\\\\\\\/8EyjwmIjxRpgvhcrjoAXs9tpQkDZyzEqQ9ERxwzd39NrVlgIO45WY6nBXNLwokQ\\\\\\\/NsS+GUF\\\\\\\/vg8yIUu1zke188y5hq+Vmo40ZtWsf8QXgaj\\\\\\\/+UgUk8pojBLcOLMQ9bxA8sxCdCri3OuhNmoSK4KSXxLAzn5Ikoex3HH1Vcxzz8iVVmRF+pQfpMbtJLyjsJcroXSMZu8kbA\\\\\\\/+KUzEthBYCHMqYQvlkPS1O9sB\\\\\\\/PZ1X7lbpiH59uh9dBl2ng2GC\\\\\\\/FWRl544iBEd9sc2gVAk0JmwW8Qw3FYsUyHzMaj1GOAkxeITgI58RUfp9eYB94rkYVrHDGDcPInZdmZysy4qO01jNzXNPZdUbJclZlt\\\\\\\/oioEqaml6vUF6aHFAOZC2ImlgJ4HavlnRnmw+\\\\\\\/Ben7p7Juyw8aU5IaQTcgV52epK0EXcgVjr7o6\\\\\\\/ePFraAqqisQNs+G31Kqa+30Zp30JnUTf3IGdlnBXDJLjZF6ioJHI31oNQJVY9eRI2E6AxTav6XkK3CZcyJAppHNXm44UU61GalyO63QiUFbxNKmtm\\\\\\\/mOYEnYo70S0S+PplrFIXJ6Ii1SYtwXR44ibWYwV9fBiOJcFDnotLBvUtEVYONDYTWIcJXOyPL\\\\\\\/7H2GHHu9kFaKMmsd1v8xYSc9jqUojpjeenWG6RbSwpHaQsIKluY2WsuuWMxVVy6hFzuD7OMxTPr3Uz6OGbZ\\\\\\\/GHwGXZRIih3JXql7rcFIn2xigr7HLICwHBA0WUJZn+mnLDfeY+zvJ\\\\\\\/g0FM\\\\\\\/xKZVSYvlewL7Xw5fsoQ0phnl+V\\\\\\\/A4mvjgPzM0bO3lACPG66JVdahnuPGZC2al9ve2ujdmcdaoZcwN9aZCd4KJr8DLWCScfqCs8WM33RMOouewYCB7g85KFV4cq8b8ubXGUv6jQO4+NmoZIDasr6PTw+\\\\\\\/4TEotpEXLbU1ISX5uJ+87nqizu4gnQdFreB4S7cjdan2rbUCd1FV4TOfds+4NGE+PlTjtsJ5FRcGPvQyGM8lmn7XzPa24eKe5dzkIX\\\\\\\/FQDlYMUbLzacvV3L8Dgv\\\\\\\/gpKrhNtSoKMKPw\\\\\\\/uDk6HkYJE1s9KWooDrnC6Zg5DMgkNZpWeIlBU93VMlBLZlQvsY6iyfenrsRJUPHX+FLqMBWlWShCbN4usAI1akFpnK28qAKU3zHaRZG+VsVmvDZutiEob1sBqRcPj\\\\\\\/ZxhHyn2xwpnIgTS5BMtI0Q024saW+avCHLzqrvrcM9KM7uRYReXyPrDlaODBt3DBfEfd\\\\\\\/gi0bhwzqotzHP9Nl9H0\\\\\\\/+E2XL8mh1ex7piWx2phO6I6ZL+6+XDniUIVeLbszmaTFA0HSit3AMFpKPp7nCAxqHKBiTJU26DvTxQ4yDNDKOtcp4yCHsPHqwPf7XL7bPoxRits\\\\\\\/ZTYMAWVwa0Y5XxgMXDaXVuiq6x8UFgbZe4FUau\\\\\\\/e+CnVyOwY84dJrO5ZBZwjfyvL2k\\\\\\\/b7NBNgZgqfW6wEPNwMHuivwi4lL\\\\\\\/sApIubcxQBNELtMQzPWbj48P1jyXihgNeWd8zS+B4obsuImBIB0MkDP7tudT2dpURilrnRQ+n+M2bnEobY0khTOWB8HqDObAZhW2OcLA9cGejh2BhRX1S\\\\\\\/rKvYDh3SlAeIije\\\\\\\/02y\\\\\\\/HsvffY2QaYKpb7lqtyiSwBaTbifrpDXrZfAJAQOxEMsBJIZhDAIKU5xUhceakPsC9WJ106WilhyXqgeThc48IBboutzObCPEGKsVx0rUBuHul7LDUjiCQZe0frfAWB0JDGGmrpGsuZudgM0rxdaXuR8b2np1tHlzDt+eXPKPpN9mTQwH9ST7cSyDPjF0TcuDwIDAwUYG\\\\\\\/QETLhA4YSwMM8C7dxTKCcNc\\\\\\\/zkyZpiJt3WRnRhydrxdR0hzzuPRHgystAxKnQUIqJqEtc7rrQOGJaDL+TxSbcuezMU2TLsQctVm+Ttsv9k7v4NDjsRt7R42Gmwkwly1uRZsqv\\\\\\\/juigdYLn33g68ZoMZrNWrAoQrK+TrRYJW6P9IYLCt9I9y0IogTzV1o4m3rYRdDvBdeo2YR+L8rW8EklNhoq9WWJMUV9Ir60Z0GDrARAZpgeP41kN0Jt8DUEdYp66N+zLUhE0QgsrFhLTy1F80ugPmP9tTWEfg3s3DnlMUzenUV+5N\\\\\\\/nBn4KlilLRbNtemgNU5gHOvmnZP9dfRg5lh0Zjqo7Jc8+Gb7o5M0h3fDgylnBzC9b190xNdqN+u8\\\\\\\/WIXhNUl74CeO\\\\\\\/G4M4+u0aoasGnZklhlsNKSQH6ixs56ZD1kx6n\\\\\\\/8pIrbqRaaba2eOFHefveus4iB8BTPTfb6XMvPSFRh7lOkrsnLCu2FuYkYslK5mOUrq8+IZju2rYJKJew5bgMwbvxd2P+HhfHZ4CNLiyq8uRUSZGLTrHjStPe5cBJpGug52UGs7A92rffFRooaFCMz3o\\\\\\\/VIHiiYdP+PnWWkSQK5sYbnhDcUo00hLTVTbf+c+o0PlDt9DdVU5riP\\\\\\\/2r6ZyOsBLWS8GxsP9mhTvOCDHL3Qol0XHGMMESTgwgxL7QzTDlg9eS4fOu48uUJqeI5k4XyDRxkewwbuwyT+uh\\\\\\\/e2kyZAcOPoUzgeb0wGaWKVwZ0ml3c6wksiuZCAEg5TRQ\\\\\\\/ZLydKopy4RFGW34olUb6kIXhhEuNBys0lf5o\\\\\\\/A5zl4JKVw1oa4ZHymoT8FbPqEEW0HpfOX5bepV3fgn7C4TZp9Bs66NQTdJpPvyfleWPig2UahftFnAnTu4T5fofgtlgolClbWucFtpsCuHpESONXCe\\\\\\\/5hDi7+cJYzBbc6PC10CYhHAQfcFXK6NEn3Hqa1Ob8cNko\\\\\\\/ak6bJH9+8A\\\\\\\/9DUPXx6UDkQttTH2kb3bH8TYpYeQm8yWyN+SR8eaTTShBUGIlrBEOkuw7x+95L3M6485z\\\\\\\/Cqe+O6u8W3\\\\\\\/GRPRLSlcwd2PBQzPj5ZWOGEDCXy9i6AVouCikMoz4NFYNTMnAawheYDBaW51w1p\\\\\\\/rA9xAIc63PEO5h9AT\\\\\\\/1I6fE0ak5gtr3DnTp4KpYsJMzApkhu7ttDI+3Rmts2Se9uQpa8ERSlgEh2L9NF3Iy93\\\\\\\/s9BT1F8YIbL9BKN1hjvOE\\\\\\\/fLfD5pp5+YL2olX+NttbZ6SCYtTGH+pNjodF7yjFZZpP7ngivW6UIidpjQiSos9vmhCf4FkoU7IkiT4v0fHEuM3cMvx7GDTscOcMvwd2SKCOajqkh15gHX455MCJlGVdBVca\\\\\\\/VIW3mKvZn4mWUYTbsmz1sH4nT5Ze6Sv7SrN4MusYlwKImjhRNeC5KNbdUryu367yfZojnP8WitEJhxyevKrlf4BP1tETangEg9SQmo5dH+2xlbtQjp\\\\\\\/sFQn8Gd5zOLAs0LF7JcaZyeXB+uMDMhKPJYrgXyeXgNBOiWuxrQcHjH8P4WnGPqbNdxJO5icFywTrKAuoDObg\\\\\\\/h6fqSK0hCg74\\\\\\\/HyJgOsB89OTfy3eSHoo6uaLxzjqUS4+NXnIOMBnQ0v0GPdAgFkIp3Zv6jJD3spabj8hDfxYpnw1wkaOja7I+\\\\\\\/Rwzd947\\\\\\\/EKFQbcduxNIitIoydftOPVwVdBUCc9Hk4I\\\\\\\/nhepxylpi75qmHHc6NUV1M5Gb6IqpUGryD0\\\\\\\/\\\\\\\/FmlLSVdgvB7Ji1UW1ubjPaSw+QWruqLFcR9Fp5r5NYlCcTwN1VBOtVsr0pcr0BmBFAnnAZv3XXNC\\\\\\\/s47bY0r\\\\\\\/XlhsTRcQp7TLZ++aYxX7mVZDlFQDY4PDbnwRIroGAhuLB76kcLlqbCpC\\\\\\\/oyr1xTIioIWRBKUEAn\\\\\\\/IXd7o8lUCpE2raA5LruVcLy9ZeC5jHIQTW10uc\\\\\\\/8cyOONmdWNIayxZX\\\\\\\/Yfsb5MYm36gQfpNtGguKBfauKase8bDYwcftLq5FjsvOEvA3theuA7B4UtViXkFu8eb3wwGZKWKgd04O9n1ss2dkT3D11+0yjwWpjt2B17iGAHr3yToHoetsV0yejLOUAyMEsAC38rdO1XQ+lKCVj8P6Orn4bO4LDrLpSel2wtIDZCYVHX6KWZZX\\\\\\\/gNVXPPLWiwbxYk3ElAiiYdp8QLEudJIuHLiblgACasMUf7+EUQ6C9y8vd1OEV3j05TWkE9OBDyKZLyw8v9PWQgXCDTQ9c1yhVLuihaZfvJZZEsJpgVx+9I195\\\\\\\/FcJ1wHyjkvlB5YTMSldL3SlLgkWFSXTVMQBu\\\\\\\/aAPsMffL2KhYwspvVs97y2uqpmYsHP6nXKSDnUQKOW2iEiBPLgeMH4fiMMaGx2qFZ+z3r68ZvJxW\\\\\\\/zpXajs0EI6kM9YkqiTRBshla9\\\\\\\/oDlE2QmsGZUoQqGYT1unZ48Qcg5H1ImjoYGBFaK6COeNmtWzcmh3j\\\\\\\/dvYhCyH7CJysWVIOVbzKoKenYCYrNhHP2e\\\\\\\/tPoKcuyP\\\\\\\/bhrOmmGl0H3Ke6KDbUMkcvc11eZObTrdWBNHSn1CaV4c10nG6UEbiN5ZVU0kgoW3odHENPdqMHe7TQ71\\\\\\\/Nqc+uXWjm7hRC0w4CABJx1Z6oVwtGrUBCQSkofeqLjSxI4+00IVBVueUQypHq2JUEYFXtlABG+GNVFIXVck\\\\\\\/wQDHXbFV\\\\\\\/IzP4qOQPsjYgKLy1W7K76knRhic\\\\\\\/AFBL+GHXY6Njkf2UBlq1HvlF38XEMDYQwTmW44hzKewi3y50lRWyvQcsTdmUZYMuM170iA1OL7594FkPDrerU3zy8oe771I3bS0gyznolBUyAECWshQtf8u+kEeNUSLkzIU6y+3XqqXGyqwiYESHmjqyI8a0qerDY0N\\\\\\\/xUI2ELShmRxLmp+CLcd0E+TyvhvXxi7sPlenNYUgRac6Hq3Mgvlw0mnvnterk4YW7wt6UOdcNTeYSBoX8T0DLhnpubgWjFJIZrsND8MqdWAziuZF5pxBHT9JGc7GUrvL3rjg5VIbzClseyqLFy0i3GpEYJu04zwtmOS\\\\\\\/8bDI95kvOszhxfce4MLBNGnRHiH0Njg\\\\\\\/U5O8m8HLw9VVQ33ezIhwYiTr24VFKRr36+FdgDAE2\\\\\\\/8caa6m7XH7xAT\\\\\\\/kaZInMrY0p1T5TL9JYs2jmeHb2BBBIcd0nZU2UBNx3wHfhzd3b3M7Zui+9XphsYdWB5RWa7mK3bw8X4volmyNvJI3Mwo58HPiu90YodFh8TBrzLsnfI2LP9ebVQGw92m4yWb9zV+nIxAZpaL4L\\\\\\\/whxy5Wzj33XkdD\\\\\\\/TTkY+SPs+rK7JBgxRCTgxVnIpJ5fDLkJCWKA32ePgfYdv9p75erJVDZefIlT4BA788HJvzXsWES47hy5EufHaoqTWSU7ToVGDWp5WNaU6xzh1GC+SLgAiMNmdFF9ZPD6OtmQJTtH\\\\\\\/9o+m9aupgwU6\\\\\\\/MesR6tL\\\\\\\/zT2Vxx3kT+IwxfaEE9hy206brAwiLe5LC6Ey+At5C7OPHSu3IVJGyG90zqObQEb8\\\\\\\/8iwziuLXJGoBPdvudKLdF7vj9BidXxq+QlxGIsFuUBA79FPqE2hC5iDS91e2GD5ZBgLcoX5Q\\\\\\\/cU\\\\\\\/B72\\\\\\\/F3so2Z2hWrZ8CDZhi5QHGy3HY5F5veb59inGcH2AKcdXe8kR7HS7nICG7difaZgxccsjgM+9ddHlSZjV06CSlY6zVR2mv4mIlpurxt7rqtXy1ZmnVqx3k+tz+I3bHMs38RJtBOPd24bM5jynRFGApEiMFYnjZqPjLMz4DvGEOtNWiIVZPLZidqSmtglE\\\\\\\/HYv1bBf82IiJCZG1alnY3P1z4qMhNvtLwLsQfxyI9JQ\\\\\\\/UsUX\\\\\\\/BfQmQObWYZyNXWKULWJFVgKEyVlFkdAq7vAh4XhoxUZGL5Om4MtAnpXn7iQ5VkRQ2tQwyDBteNth9r3KGMiPjwZr4PzHiiAU94iOKxfyTBzM7vmVZb+pbQAjHl+jtZAsPL+s0T+ep8NON4REnENDLN\\\\\\\/w\\\\\\\/Vv9+GrF4h+Q\\\\\\\/Ut6mXEcIR6CU5OSoi+k9NUm\\\\\\\/L6cMRK1AV0y1CjQchdCj\\\\\\\/3HBczZJKVYx6HLkZpqmGrVZMJre+IZalzJHPcpwsVmJNxy++7TaiyfozGvJxtIL4joC1YHAHHEO047TSJx3f2SzFHtLFbV9VsNAnWXm1hlL74UpDzrVJiOQpxPxnt\\\\\\\/8L14YCTQqw5uNvdEn2FabkyoboAOLKNhmPFZnf4UJ2Yf0fqjJvUliz2HE4Ki\\\\\\\/\\\\\\\/N4S+lNk74mg\\\\\\\/gJDCIWc9amIXfeiH3HNhM5u6nLrwK81s6mR7ZvcvuElZrjaM1Odf2ggXLTo\\\\\\\/yVMYAmx7i33sS+XgTNE5MJp6teUElk05LbpbzQy0jJDg4IK9fem2tApcVmrYxzPytyc1ES+Bvrq+cGzobWSkD0bcJt7hIKX3X0coUe4utid9rBjd+teGsUqoJTlbTnZYNzZnBcRbHvbe3ksQHs\\\\\\\/hqUt73cS1\\\\\\\/2q5nUWvnmu28A1+mgQ82mby5tdHX4BGmhmY\\\\\\\/m6xkKX0Q9gHNNvDGedQZo+I+JfiEuHsPqmlX+VjS0WZ7ZIQ2h3\\\\\\\/UhNPWiZN21fZA9v+cobtWiBEOCeMxPplMpmHjQXSaRno9wIXXfV3CM4fTlD6qBZp9xR+Z17SmbbiQOi23xAdTCugE3LdXC+JAdSB2xnF0eCiFAe3NRjDjzFPKUHh7+XHdH3nUxv8iiYiXR6s2YRdc2njc3l6y8EJvJNNNL5Jz8DCgtGxipVyvB0s74JQqjxSMWGgxWYueSZ4C0wr7mPLZ6CvUoj+gNWMIzhQgTJGzjxFdD76alLXs1szIAATNRXSgWVR8cFLdF3NgK90\\\\\\\/wU8MFY+H5jqHAmh0tSYMf2Rz+gGxYF6Jn5I6vDXhFARSNPhYjryGhdXbRJvwhZ+f\\\\\\\/Kt79WGa6YqiCpUecKi596qezddj25DaGG+l1iMJXgsX6kjRI6Rkk9OnR7qE8CrzpHIwfdCezdOoyfBs6eflqE+I\\\\\\\/O1b0tVQnaWn\\\\\\\/2RQEhmreBOscfabVOosNirOkLkR6wLipx70YMiMTMDe5bEtFy57dTNtBXzbuGcVUyTN4MpTR149j4lyPPMdvsMymC1fj3YE1hPPbNbwhTWEd2sbi9U8rIh0QtT021aMxANv0Q0KNhNTvcN+La1W7EPxmaAR2Zs6v5PQqwu07w\\\\\\\/c7YBfDYyimHz\\\\\\\/hPf2fm0hyJo4mz055IU1BdhoAbCEoq643ruLNdocNrNO5tjvVx4etCyOhDSQPPs68XP5zRbdAq6Av1VoN9khr9Wm\\\\\\\/VZGxfSsZ72x4Eyvp3xK6Z1lPgwA1urnzzqodEUtAT+bFkWOKvQOVC5bTimTD+6c3VHw9mt2p9rmJ35R\\\\\\\/yaDCyhsV0qylnPiTT40ngXa6w0D0+6dRgAAtrR9C+tMPe3zDxFbu6cEvbaww+AaYeNczEtw\\\\\\\/4i3\\\\\\\/fsqqVS\\\\\\\/c4mWJVyFYU9C9xbJmLbLlOOEVIIWP4R+wItAAOeWryDM692\\\\\\\/d+e7vLu257dnawn\\\\\\\/uw93QGRrWxDYdwLtpY\\\\\\\/WMP1xVRoZZ4\\\\\\\/hZ1wsaFgMhgKAamYvBeVmULsSu5D\\\\\\\/t0IKj4NF750rUALGFugGDiCeoY9G45a5CGxmIGmV1FvKkVG0j7izN+p0uyZIOq+GYf6xVQP8vERJ6U1SS9ag95jwX6g1Xpd82MikEbntsT7DNLcGmq5c330HtvKNHVqv3DZATHL3J+7mcF2Hs39A4mGRWS5lzXkuhfKeNcepgsLw2tPCiRUzPis8Fu\\\\\\\/7jMWLpJg\\\\\\\/nfmx2BYK67v5nHIz+mDoMX\\\\\\\/kVYRV6N4r3P\\\\\\\/KrVHFLchO9\\\\\\\/H90mSRGZE70P6zclc0+m++1iaPHy1FJ70SxFzakNqWrmcSzL0ABSCavyV6BRBLYDHW8lS93fiPDipIqRlSRKT7RdOEJFhPtTxwJkyPHF3jzpMowTzls9nb39Km\\\\\\\/r8VVsjwKyNI1aIcKLfycFFxCTLXTuOyng1wZ9yBtNIlCPzSrvLtXQd7DETyxJZd\\\\\\\/rRxRdW1IQuAUxH3UAo0wMSXwTHQYfoP4t4\\\\\\\/bxhAb1Xdvk1w5t+5eYjQjxpFBCZayyL9+VV6vCiozEece3tbff20kkHUs3EoPo2YUAO2vYbLJ1IlAS2x13oLS4kovELsX1jDeNcQp9AD9j+q6LFkDliyzRlTT1E4tSDm3uXTnbl2NASNsvB\\\\\\\/+Jx6n839QdP9ocWGFdf5AGCdFssf\\\\\\\/JiBh8b\\\\\\\/9ixFUjNHgXyAtcmfwp85oBwCiduzsKLdPgDMGpV\\\\\\\/um8vS0mmFF4G3hQ2BtooAohbaSTEv54UUmkK56E1Q3h3RI8nIw8nJlXx82ITpyJt5Mbm9ZCGU6IxuzZuGCuewTPnDBcnRIZs7512or\\\\\\\/QSfMjWoCHsND4lZkTZUnECHiCXukQ\\\\\\\/o9JqGSP1pOP7Yc1AXCqwRIdhJu\\\\\\\/NgOfhSyktDuWZW5KkRwckefXGmJCtVogzKwTKuNxFAEQWn6QLdn\\\\\\\/1WSB6NQ4w9mUWu9kBBc0HvJUwL3ksV0i6TUCr8VJn0tcoN+o9u2O3aaRW7IbIpMeTyvrGPlpN09xXlv\\\\\\\/YyIDlUkBE8PRt+kt6DLd3XvbohrPl\\\\\\\/f7pOw4rN8lzXBjOOUsJRvrUQG4F7nzeqY486CU+XeJdaP4bye97PCGZpdUdo8Kphz54oAY+t0CwCYQxVSlSHZ+jCII7dn2Nxtao\\\\\\\/o6ie8V2oMsqUMfTiv5CxZ42Uvbh0u1I5cg2HdD\\\\\\\/JwLudy4gI0MAfGpZkFThdsFIfKEIqCyB0pUh\\\\\\\/ArGBJbUCDkfk7NC2qn\\\\\\\/\\\\\\\/8v97rSI49ienugRvKzcXZLNEKpLXBAaJojgRsGpjQM47jBdzm8z5yEJmQwx2IvtwphqZEQkOtQLF4k4Jk6ENy3Q55PN49r03CjgdO1UQQRQU7JZ9TwGhuZFaSnlkSmfjHyCywYkzav168VNdRI1AEmNOB9XtK482DZ4oQYlaqBDqUAL6Y\\\\\\\/TGmohxSBhUOSjaR1H7Pu9B5ONrmaf50I4LlnhV94UmIlZ5BZ2MdbCpSbHtynnzziDxypAzNT917g81XGnsZrpxrOeCwslFlHpbitB0yJ97Y4MsjeDoQkaQ7bxxo9UKcpOnx6GVpdPApDheUYstqy6eBRVoTJhBsJ5qEojt0pW9oT7qPn67XT+qxxTdbZ+rK7gkAB3TD4ruQtQktXMMmX\\\\\\\/bSXtO8UZk1PcHZowielSbBTuQ6mmHvZo\\\\\\\/xOGFMJuWRQWOrMBaAU0lsEjxJ41DgT2Ql5JpQNoC9hGKcQ0EEIXmkWxd8mECoHA6WgEy6HUmAFlMlTlNzvLkPuaKSRpagN1hR5KFVcb4iYII8UDstyZ1wzQwsJ\\\\\\\/Un6aUdDS5Av+QcnAwFnyQLiBl0NTWHtGc0TYtHfCpODG3iA6oNtvHa8vymBsUNp0KlUko7jLT19Q1xZsbMEER+FmZAzdRl4nWj50VF8E\\\\\\\/fJaLfxtEBiLjJWmlQHkb4O6pBaJ3MkylX4YEb1IddFyV3ih8c+OMAwRj6QMpPfI96ddPqm+xJtmbGREYhiL5C8P5yvvlcqSIi0XSWN9WoNWTZJNFt6H7C6zLk3x+VS70JwYrBYoiNK9Vsh9rwr\\\\\\\/EP4gwTrSvl818cuYlr3PZ\\\\\\\/J6jE73OclNCSUZAZwiFQ4bS1\\\\\\\/\\\\\\\/CqACzqG5C6dl6+YNlNhl0st4rNWbh2nTzQoz1xHxFVN1q7+RyFiYk3X+ee9Jd5Buxeu9UOKjsWF5U1eWDNMGqH6+cL9r83jVDtTLZEkjs2\\\\\\\/BP9YzpXiQVsmnj4vNkJ9AzgkNViK4T2NyopZMRf6m0gJmaDOvPsZRr+3ZzZkzKbrqaZBvvEuwMDwzonqNrVxZJhuTs4JutGC9JAEQwe1OtS\\\\\\\/08tcy8\\\\\\\/ezjpWmNIIL4psPZAM2wRIVhV+WZufo\\\\\\\/yfLaxnOw1o\\\\\\\/Tm60yopxw12i\\\\\\\/nqbzsFhbYIKCC\\\\\\\/5JYSXVwraw5XJ\\\\\\\/ti8ZiXTw\\\\\\\/r9niTd0EdjQqr+qqxEj2mt4g3OX5KX8qBJNbOF+he6dI2TO9a+RQLsc2dmQucVuxmJpe9zhRlteE\\\\\\\/Bzw8IMaydZp8ElzildZrv\\\\\\\/0WvoEKBeOjaWlvLO3LloADPvrRzxTX0+sMeOg8wYrTczHAvHTFRrXm6FVzbDrx59D63sC\\\\\\\/0tcJ5nKtaJJcZ\\\\\\\/lsrgfsiTXHdSqnxy5VFaxRaDGBwa\\\\\\\/Jj4+dZON\\\\\\\/LzQ20xKrtnfkR+fAafzYO3Gt\\\\\\\/4LxOYwBOV1WNvZLfRDi8vdvYkXehbrwBeY7a9XvauvsGMhOD2URd4pQELL4u00j6Aa1yd2QvRLBwR8nfG2HS05ioXgXj9CMR6JM8ZdjAge0P6xB1TDkhiaHVvObY4IrC+aIG7ILtl8na8m3pPcFujEND7UrAHxnjYkg5\\\\\\\/fg4Q0OOqnE5z29q\\\\\\\/3JOK6RoXSXmBkSL0CiaHNnF1IGswWl5sIRL+xqmV+Yn1e7dGI+wGgNvTsiKLNezuKlqCIDNoWLhxKMbW5yIxSFwvvpXaXEd4JATxRxGpkEphywog170uVM3WbTtt8e8BA1ZHMiPBZcCqoFUpks4mQtwIYsuaebtADFD6Nwive2bHRStC+Y9eAxetciYMhsV7jOTw8ItZ3Jj7yMobrYJWDcnDwvNRXvKsOTxWzlbVvJvXd4QguRBKpuW5e7DAHuvFhHVrkxen8InXjKLcAWJMtVfT\\\\\\\/Eg+mYX++xlFrgfdyaEU37Dt9043U2IEOnanHx3g1KWvLyXs\\\\\\\/Jk4DFf3h4urpaHGv6PsD\\\\\\\/CtGOhCuPxi2aWKCGpru+C\\\\\\\/7M+1cLk9fm2Sw\\\\\\\/+1uMESrCT\\\\\\\/Dhoe0TduEz6PmBlteeju+VqBFZtSOiyaAo6Ke31yYeUXESSTWPJ6YR+t9Ui\\\\\\\/VsChjhGzbM3pvwQo\\\\\\\/\\\\\\\/bvbfwEHk2XK9H5aD4F3+IedhMhHei5vhwH44AbvplQu5eXMcEYQ\\\\\\\/XCLdc3aIeoondIMBjfjCN+GDpVaXE8vwyRpWLhWeUXlPRd692CA4MLk9KwxLeWhYd0a7+y8BP0AaJIayngMg3LLTw6u4UKWskwyqvTHI9rFVw7Hd1I3rslw+D5OO8U192ejQYTRs\\\\\\\/Sez7uL9wWKDrbCpc7R9nPtA8xBk3PwkJVzFs9UJPsP2DbUORxcKrNbRKiefQBFr+okqHctJM6FMJLmxchwfVolP5aHqGTS30BahhTzkjrRwSjZb9sQkSMwrCCrj7aINmpmpS8Ud8SnMaGYpPvUHxdilmuIPKRIKcx7Rtyor4OOmE0iB5zh8BhC7crRm4iQKXz6GH2kXMp60TsY92bRwTqX5EDZ9Tw16wbwsSAx9vY4AoV6QSgWZDTruELxLwKDCWjE4WVM1kpg9uovsKV\\\\\\\/Oh2ZqLlZdVavVFon8YUO6+kYON7di+vCzzyUhchQas3YiO9O0WsGsiQC2E2mrvHUDEb6GlZLjlcezBeWBCAizQLKu+hI2ypxAWWswx3T3I\\\\\\\/DxXlINeUJjs93qTMIcCXn9VZP33\\\\\\\/pPgBtVCwaBSNb485cIHfyd9fvbfR1AEpouN4hwBq51y4saaWwCQEQJ9XeGY9U21aW9Bo5A2IM6JrDBydorffZaNZsnVlQBW6Zj4282RMBvz+z9uoYghDIxTJd1cyiODVYoty8zg0LmRU+qTQJgeCw87yMyfD3Zpu+OlVyuMY3J\\\\\\\/h205+blfYhrua90RmEIrrrmiLfg24Qm7w\\\\\\\/kvUum5Vvq4Qycgn4c4RUQZGSvKUfPWLFkQqZAbvzcTLk6WPY8ntMBRYr0Dp6ToO+Y5kPbLc5J+H2uD2KfdPBsHekLAYIQTZFqsZhSiRPzbvAT3Em8ZVk45T8Nv0ZH4W04YciaYVy1bU3Ooi3\\\\\\\/SG\\\\\\\/6vw1y3IoxGU\\\\\\\/b18upcmGQ1gURdCPMyV87mdgQfYfnhBh179Wph+w5xnjVDl1OyODH3wRpjD7sexTCsECvvUBm86KHezIKWhZOdDanZYf0986i6ZB9r8Fhcz2+V5Zngfh88IP9DMCGbZCR3LUtRn362VBG0Kug+j78Xl9fszhEA0y+DlSxisGas5ErJ5P8ITdc7OsDfKQbOnT3DTGn7W380U+nbdW2if5i00X2BOu+MvsSXDWHXucguDPrrA+Tn3QytIQosVSvHZ9FTAy+oSNAXivJ4Ziy4BAGk\\\\\\\/4j1+bB+9Pk3FFV69QKVa+9BPqo9mAdnsdLEK\\\\\\\/1iqRhQ3QfEcv6\\\\\\\/EAo0SG3R2C6cj05sC0fxIaQ9rSz8Hz98Y2XjTkT3wkBkFyscRYq\\\\\\\/n7Kh5O4Xb8R8AAmI5KgpStGlfmdx4bb2j6xdz2Pr\\\\\\\/V3kP03bdHk8K73voR1R2siLbz2wQqfauf74oNif016be1tc9O71IoLFLPM2n09WRnYQf9ptNVfMvB5IMhBCBmzjywuNyEfUfIk7FHbJ3I5JK893hzuDWKlXtVkD2BIrjv8CX7fJHeoJkiuh5dBySCaIoiajbFnoohOYu\\\\\\\/FqeW9c\\\\\\\/CTYWnuhiGsCMInHOs45sVT7hTQlD+x\\\\\\\/9EpBt5JRNBpf7JXOzTMnNZyh8m7b8o434eDX8xcfoUyuLenGLcY8sZhner2A1UcbFo2D+dTtBNvOGROQyqGc4bB98fcf48aheFiDoVa6lGydYW45qf7ARTjoQ4Tg5j0AuxNB5PbHnsjVbGfvRh3mUnwdizwKfvYHSyNIYIrz5DAXbAJXZrcASFgojEKqVKhWr55YO3j3+Zhhwhdu3UZI4PaL7ZVjevuSb4uC0eNXQ+x9ltF6p+POrJsLpPQkf8TbNgDaK4RIAeu8bcK9Z0DjEjOwPKEQT8hitLHd\\\\\\\/yDAoXVut+ECsUAf84oJ6Q7fcwUFUr2hcLUFJ0Mo0Pwm68eXoVRNi0WuvO7n0tDsH7+dyxdVimVjWhPdjzMTFzXTHWpouU3YURNMOPvB9x8Kezt92ckyoD7bbPzx7XWmdEFAKhwBN+qVGZRjEokb3P0cO7BFuyagyJfjp\\\\\\\/4POz5jgtGlYxS7l1W+zn\\\\\\\/4NNMhw1VRu17ExAiBloKda9RoH9MPZUHYqzmGWmTO8B\\\\\\\/eVAatyGtRq3Qlwc2ButUfpJ4FzC8jw0wenwGhApob3FGq4xTjxEgF3MVG4SPh\\\\\\\/fRY0WatY\\\\\\\/db5nkB2GyRYS3FpaeZkBm9nIm1diBok4SIvoWfyi3tS12td28btYzpWHCoXEszARXzdTg2VME2XBTrv\\\\\\\/GEqtZFyiOFfELK5sIWHdHyyEjITW7QUVTJQR64gZu5MVGzL0hriFq9AmmYnXtEDFyp4piM9AKZDE69FN4SydOOFCrSnnumQ+ZpiSFia8NOB0+qIbIvxg8cZpU9qMWtWrGXjOkBll\\\\\\\/lh+xqR00o20LSjWPYyUfIDOfL620ofwEXm69EAAMRMcy3Pa3Xat7mIu6KMTzzMgHyJRLeTgNgyIwMLIFqaCFG8vgdFIUlYOec+sY1f6OTuKt6GuETbqORuJ\\\\\\\/bl6a3Oc4rCq2Xhp+33dh5NfnyEd1pi0vCb\\\\\\\/u\\\\\\\/\\\\\\\/nJVAs\\\\\\\/cCG5tVrsPdcUOR3Hlpqr3cgoV+pE1JrclOTBIdSi2Y9S+5rd6Kx8i8VJDXNQFxKY2MOVyZtu2z1ggHVm18Y2IU2UG4NvoLhzBbFEltPH3Bmcxlv0rmda8y90vi0XzntxHx8cywDBwP6\\\\\\\/hV8UKFovoCLxTKtpAa2YhZaXwPy7LjeNwoJ5Pr0kFTnc8cxJ1YsawZ\\\\\\\/PD8SGK9mNyHkaer7J4aEX5oDbHIzVcFaqGMzYMiYBAiIou7dltkBmxTPWEO1nAg8A0qxBSALBRk6w7MmdXRpcGfuo4RkCSGIP1Kga1\\\\\\\/z2ysoWnjCYHRxOQNkXRW0j0+tsQ\\\\\\\/eCpPsMMVrKAxhRsjXw\\\\\\\/ZpForClScwI\\\\\\\/SXxpco0UKCSJ49MnI5EIwh2OXk7osr2bZ\\\\\\\/D9FhM6E3Xk0iBzSMtlkqU9W50ZbCGJHpEsAETDsISDcyRJebE1WPHbt84qF3J2zcjR\\\\\\\/0neK5b8qStGG4FX5kKpb3pRweGjZUVsmMBJQdNkVK0kxgkmJMD\\\\\\\/jbS+Rmw2XVJgiQ1DSceWuxHc2TAD2Z2+TFAwAN06BFMsEqgE\\\\\\\/epLZdb0eOUXw38ArT+kDoOwmmoOB+5HM\\\\\\\/IjDRGTj7CM2tjEAF8sV+Sb+jlb3WyLSsohwcKc069NLXocyC6OwIbdg0Yp8UpTE5OrxCJGHKKOiTNzbP2bM3E9nEHRAxXPqtd9aZPjA51NDxN5sAE\\\\\\\/+RANyd4CCGuqIu\\\\\\\/N\\\\\\\/Gvhrp0mBCQlz\\\\\\\/\\\\\\\/BE9ZFyCheYtON1nuc0og1QIVPNBbpHwS9UC1VwacKodNC8TBQzJ5ebWDKuEw4TwnF6ICi9T4SUDSYUgvdX7B9UigY5JIQMtUOeC26gf1Ilm6dOSpQ22TtPeFh28XC2hNXMC7bG6\\\\\\\/QQ7WGRMhqiSg3sC5lPkSH\\\\\\\/EK77t3z+AFS4tXzaONTaZhbN5514h4A+yuekKNYMvXxx7zaNcXwqHWlU+cR83Etcw+tYTPQrmsckaV1r5ChYkXx628dhuSMn4\\\\\\\/C56gKIdrbFQsETS2+SSPOXeAJYSBPEXnw\\\\\\\/HdzmrvefNTVXm4wbmoIjL5IkeMLvDTPhACYDSpkLDGul99zFDO39gV+dIxq+f0LUjcLXNgTb9MKg\\\\\\\/jLwQbnubDTXHjOtk+DX4gOcazm8yu34DG4M7LD9LEz5xYF8oUR2nC4vIRILtYiyv6k+F3Bj92xLd8Vtt7rnUJjm\\\\\\\/prl5V3XDBIBTDbcHnCfReYd6e+JQumZmsaCBrm8zo4ojfoLq9U\\\\\\\/BOGHv+giPW0o2uemSSijWMYxqS1VUEuzUdyhbmZJwVHX6qMxRi09cE73Ypsshc1XyAucEVT3VS2Ho0dqvEHDbOJOhbCS29tc2B20WEsaHY3hjNSQxub+Pu532Lg2dJQkyPyzVbUXHIvAh35Wsz06hbAzjuhHSS9AQY4PItNJ3Bn2rkIPg\\\\\\\/RBqL9BYOv7cTDA39\\\\\\\/Ey5Ebd5c\\\\\\\/7ZKIYhCj4LBHdPthlHIHWNCjoSpLd4BfMfGXI7Fh8A8Q7na2GHgKa00eGnUOpGwwdiR7sTVskMgdC2XzjExExbJViq3ozXSoYtf1locnDIzkqFRnKjRWeQPkFhoxXN1GYLDHm1i1xckBTksO258MZB6kcdiUfMsro+6gVedPXuJzkanBI945nW+jzXzg9SG0L\\\\\\\/yDWrqNEUgklF+pAaf4dQQALm+szU5b+riJcH4ZtvKizSZwRGl\\\\\\\/nxBiYKKXeoQ1\\\\\\\/6S0I60aUsSo1kpUNYw61nHO8WFwWHtFj1mMqP0tMb24ouKFSkQGFidq+E2SZkuP5+U+MNOZgpSC+Dm1p9Ne2r+DOk5TeGTCdI84ODb4xBkGH\\\\\\\/Cpafffqgr\\\\\\\/oMEgva6BWHL+pH2f74v6aAyzFDBKpNFmCcVXWa5PAH8KxF\\\\\\\/kkLZE6smN\\\\\\\/gZFzLB2vKK3JV37yrKiFugRZBfdnvaGnE0SvUkV6xegSyIshj864Kj3DPFGx8HifiEvNgmwGJm9T4H3jrvaPGqVkLcdFE\\\\\\\/zpzFtbqLXjejrHYo\\\\\\\/MbbTg6DzRXuw1LHq5Oz66PwVi3t0s9oN25OauUcMh9aIxAkFD3PvPCYgTP+5+OvGWxEYCvxn+tBicsrC+pqc63YK\\\\\\\/\\\\\\\/A0juNuIXvJa8HQI7uUVLfgKZrr4grC97rFuMYZZtoWyn\\\\\\\/f+ncY2FU5YuySi8Up+0uyYxmyw3xJkgzR4QUKOevAbQJ2W5Q7cA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/l\\\\\\\/dZzUXC3a8cOm3UUvC6w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002255cac3143741aed9dba08df2c378c4fdb5fb006dcac65ab92513a3450b2afe70\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022x0tZAPIWL7SC4SyUWyH0wJ8ahaSRv54QWp5GTJrhIwSZwCnF65cAb0+9FajQL73ssw+0pe4gkEC+SubcBMIvqnpObu7Pou7DE8FCmyTyR\\\\\\\/LLn+dcgtrS8A3fKBcSsDnPBTzsPHh1yk31HKZEk2ULqszSTBhzppfKG8v0gr2Fwhzg8\\\\\\\/m3XJhXxBFjoYbo5W1QLN4L0WUgOTVu4efSH6vqP0R3YAn4984vY9EFrWV1P5Jdt2lUXbylTlTpItLxk05F00vie1eN++BBnxYobDpjkUV4t5FBKFtMovyf6O2\\\\\\\/61EXz\\\\\\\/A+Z+OVkHuIYkCVE1GG+RexmkiCnWNQegIGW58e4BkAEoiebYKqAQ7IuiXUlK0EJaLCMy0ca6hbs2mfM3y3xYZQDuaLaxFxbib8D0DErYo12b1TLeg9ls92TCz5V3L+jjkYzM87PK3l3GiOkFP3zVrAl6NV56aq7Atarc5Yu+GJp2bhv6SO3K3Lv+rYq0TYHjceLJoNPLj3F5iS0xTZ45S+qAVXy+rkPqDBYcMZ9nUBv73wx2KFCoW+84BVte70+KqPBBUDHIWw2vTrUAk8TttA5vGp5PV0HMxXHfQ\\\\\\\/Gwe8VcnEWZvHzmsno4c1ULEv8BcAz94+jnZ76q28fbsckv\\\\\\\/dnOfSShr2Nw+nZMEULNpV2oQsNLhj96K+HVx9AfhHfskliWlB39wXOm+Rkm4ByQ59bZyhYO+SQElT6KPmYC6olfcpBkj66LCYaJX58FZUPIhSfto68voKvCmAbjX4pihVT1kpKvimWGqSG3rp\\\\\\\/bXmKXdHOHzVLoerfX2AhRNjGMoG2FJpYmt4wphk3qOMncPQnG95K+cTdXmH6VJpAL\\\\\\\/BEaHFT8vS0+PXQcKNOAkhx7m+Viy6CR8x5PnNQfP\\\\\\\/6Up+zvRJjBy+Noq4GXAPw9t\\\\\\\/MnG6DWd+aMai7OjFfiAM5Zve+Z+qOiA5H21tA5gLGDBjwxk5yAHZC+JpDVl0E2przY\\\\\\\/iDezVYSlYxVZtPhZ+yymVLoRDEq0LMJjn6Xx3M57pBSxFuzHzWwbpzXbU+ELk9ks+2ifoqLkaCoL+iN9KKod3Y9gEkd51lTnT\\\\\\\/9oo+CIMcVVT9ePeeBJyNGsATbUANHBWOFle7q6e7n5wzxXvI0W3O3qqTY8ZMQ1cVxC4bttpYyi8ycYpG1tZKE26sg4DbFhSR\\\\\\\/CEoURy1fD5850nBhdPM074fwRb4Q66PONFEwgD+l3rMzseKHPY+rsxjbzwlHGcA8T9F4Zy\\\\\\\/a7rjdeGiTSUbfRoo74\\\\\\\/N88GDarMhbbBEieR5y7LoWEWiV1\\\\\\\/KIvZOuGIO2Vgppna4fP6WO4Uuuz9ImSn0AFxj5Rak3WUzNaekVC3Kn7GbJOSQuQGfFz+x3i9xzXFkA4HGYUMDAY6OX6x\\\\\\\/5gCcWW5fImaXYeGWkCazPkEgJ0Zu8CwEa31+0bxXmr6tuus71HQSu0TxeD8L\\\\\\\/MWrr+hLgwb6QGPmSVkmrZvpRGmtqqN\\\\\\\/St2GETMbubt0ynIIRXVY5AP5cl9oS\\\\\\\/INESd223cStFbszzrXAVOKMTPUJ6EIXFeMXoegCsRGwdvHEFoiIvUMSJFUaLm0poWNC92vY6qaG\\\\\\\/02RVA4cD0n1Ah2aqx6f9A\\\\\\\/pq4IsCgurHwclzJi4eVNXeYUmdwpPQsusvEpK9TCDpFsmaigppcKs3IJzj89RXEwfdxDhhSarr77YTksOhJXVKSU5ef5SIPyyVhb54OpOX9tZnOwSYvjPjVx6x14d0gUyfEaukSJcrsENU0EZhIH9Hbp78920CLAgYeLVZsg1qOFR1ntuHyouEwAUlT1IGe5nCyH7z6M5P8UIvJ6SWOirpWGGzW+lwtkde52P1jq\\\\\\\/gODJFLv1RhTW2YZFRyn1raY+sd4ZBWDe\\\\\\\/m+iNyNefZS0\\\\\\\/FlDQOAJypzo2jId5ApS8e0XZknQCkN4dnkrige1yrmKD1zCBa22WhCGcwF27nfd8SD5f6p0tixMLI7godAwRwZew08nGSIzgH2oLKbRHiMTjgpKxVQA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022hCZ3lsJaN13EOQa3IgRbPw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022dee19678996b5f6462438e59e00867f7294f6accd244697fe9bc541d907f25e4\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-484258 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1475" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-imFAX8jSLfeTSzDt-3lX_WY4J7cjIVxY4g7M-VqPtZI" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1475" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0yFNXnoap8MnIRAAhJqH8EY5z65RCiYaXYFpGdovCKU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/5/1463" title="Schizosaccharomyces pombe MutSα and MutLα Maintain Stability of Tetra-Nucleotide Repeats and Msh3 of Hepta-Nucleotide Repeats" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/5/1487" title="Paternal Induction of Hybrid Dysgenesis in Drosophila melanogaster Is Weakly Correlated with Both P-Element and hobo Element Dosage" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484258" id="node484258" data-pisa="ggg;7/5/1475" data-pisa-master="ggg;g3.117.040105" data-apath="/ggg/7/5/1475.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Extensive Copy Number Variation in Fermentation-Related Genes Among <em>Saccharomyces cerevisiae</em> Wine Strains</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8436-595X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jacob Steenwyk</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-7248-6551" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Antonis Rokas</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1475-1485;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.040105 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484258" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jacob Steenwyk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacob%2BSteenwyk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steenwyk%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJacob%2BSteenwyk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8436-595X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jacob Steenwyk</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Antonis Rokas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonis%2BRokas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rokas%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAntonis%2BRokas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7248-6551" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Antonis Rokas</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:antonis.rokas@vanderbilt.edu" class="" data-icon-position="" data-hide-link-title="0">antonis.rokas@vanderbilt.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/5/1475" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:484258" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:484258/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1475.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:484258" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:484258/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1475.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:484258" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:484258/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1475.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:484258" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:484258/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/5/1475.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:484258" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:484258/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Due to the importance of <em>Saccharomyces cerevisiae</em> in wine-making, the genomic variation of wine yeast strains has been extensively studied. One of the major insights stemming from these studies is that wine yeast strains harbor low levels of genetic diversity in the form of single nucleotide polymorphisms (SNPs). Genomic structural variants, such as copy number (CN) variants, are another major type of variation segregating in natural populations. To test whether genetic diversity in CN variation is also low across wine yeast strains, we examined genome-wide levels of CN variation in 132 whole-genome sequences of <em>S. cerevisiae</em> wine strains. We found an average of 97.8 CN variable regions (CNVRs) affecting ∼4% of the genome per strain. Using two different measures of CN diversity, we found that gene families involved in fermentation-related processes such as copper resistance (<em>CUP</em>), flocculation (<em>FLO</em>), and glucose metabolism (<em>HXT</em>), as well as the <em>SNO</em> gene family whose members are expressed before or during the diauxic shift, showed substantial CN diversity across the 132 strains examined. Importantly, these same gene families have been shown, through comparative transcriptomic and functional assays, to be associated with adaptation to the wine fermentation environment. Our results suggest that CN variation is a substantial contributor to the genomic diversity of wine yeast strains, and identify several candidate loci whose levels of CN variation may affect the adaptation and performance of wine yeast strains during fermentation.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/domestication" class="hw-term hw-article-keyword hw-article-keyword-domestication" rel="nofollow">domestication</a></li><li class="kwd"><a href="/keyword/genomics" class="hw-term hw-article-keyword hw-article-keyword-genomics" rel="nofollow">genomics</a></li><li class="kwd"><a href="/keyword/structural-variation" class="hw-term hw-article-keyword hw-article-keyword-structural-variation" rel="nofollow">structural variation</a></li><li class="kwd"><a href="/keyword/carbohydrate-metabolism" class="hw-term hw-article-keyword hw-article-keyword-carbohydrate-metabolism" rel="nofollow">carbohydrate metabolism</a></li><li class="kwd"><a href="/keyword/alcoholic-fermentation" class="hw-term hw-article-keyword hw-article-keyword-alcoholic-fermentation" rel="nofollow">alcoholic fermentation</a></li></ul><p id="p-3"><em>Saccharomyces cerevisiae</em>, commonly known as baker’s or brewer’s yeast, has been utilized by humans for the production of fermented beverages since at least 1350 B.C.E. but may go as far back as the Neolithic period 7000 yr ago (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Mortimer 2000</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Cavalieri <em>et al.</em> 2003</a>). Phylogenetic analyses and archaeological evidence suggest wine strains originated from Mesopotamia (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Bisson 2012</a>), and were domesticated in a single event around the same time as the domestication of grapes (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Schacherer <em>et al.</em> 2009</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Sicard and Legras 2011</a>). Further phylogenetic, population structure and identity-by-state analyses of single nucleotide polymorphism (SNP) data reveal close affinity and low genetic diversity among wine yeast strains across the globe, consistent with a domestication-driven population bottleneck (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Liti <em>et al.</em> 2009</a>; <a id="xref-ref-81-2" class="xref-bibr" href="#ref-81">Schacherer <em>et al.</em> 2009</a>; <a id="xref-ref-83-2" class="xref-bibr" href="#ref-83">Sicard and Legras 2011</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Cromie <em>et al.</em> 2013</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Borneman <em>et al.</em> 2016</a>). These low levels of genetic diversity have led some to suggest that further wine strain development should be focused on introducing new variation into wine yeasts rather than exploiting their standing variation (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Borneman <em>et al.</em> 2016</a>).</p><p id="p-4">Many wine strains have characteristic variants that have presumably been favored in the wine-making environment (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Marsit and Dequin 2015</a>). For example, adaptive point mutations, deletions, and rearrangements in the promoter and coding sequence of <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">FLO11</a></em> contribute to flocculation and floating, thereby increasing yeast cells’ ability to obtain oxygen in the hypoxic environment of liquid fermentations (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Fidalgo <em>et al.</em> 2006</a>). Similarly, duplications of <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> are strongly associated with resistance to copper (<a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">Warringer <em>et al.</em> 2011</a>), which, at high concentrations, can cause stuck fermentations, and <em><a href="http://www.yeastgenome.org/locus/S000001836/overview">THI5</a></em>, a gene involved in thiamine metabolism whose expression is associated with an undesirable rotten-egg sensory perception in wine, is absent or downregulated among wine strains and their derivatives (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bartra <em>et al.</em> 2010</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Brion <em>et al.</em> 2014</a>). As these examples illustrate, the mutations underlying these, as well as many other, presumably adaptive, traits are not only SNPs, but also genomic structural variants, such as duplications, insertions, inversions, and translocations (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Pretorius 2000</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Marsit and Dequin 2015</a>).</p><p id="p-5">Copy number (CN) variants, a class of structural variants defined as duplicated or deleted loci ranging from 50 bp to whole chromosomes (<a id="xref-ref-102-1" class="xref-bibr" href="#ref-102">Zhang <em>et al.</em> 2009</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Arlt <em>et al.</em> 2014</a>), have received considerable attention due to their widespread occurrence (<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Sudmant <em>et al.</em> 2010</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bickhart <em>et al.</em> 2012</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Axelsson <em>et al.</em> 2013</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Pezer <em>et al.</em> 2015</a>) as well as their influence on gene expression and phenotypic diversity (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Freeman <em>et al.</em> 2006</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Henrichsen <em>et al.</em> 2009</a>). Mechanisms of CN variant evolution include nonallelic homologous recombination (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Lupski and Stankiewicz 2005</a>) and retrotransposition (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Kaessmann <em>et al.</em> 2009</a>). CN variants are well studied in various mammals, including humans (<em>Homo sapiens</em>; <a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Sudmant <em>et al.</em> 2015</a>), cattle (<em>Bos taurus</em>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Bickhart <em>et al.</em> 2012</a>), the house mouse (<em>Mus musculus</em>; <a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Pezer <em>et al.</em> 2015</a>), and the domestic dog (<em>Canis lupus familiaris</em>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Axelsson <em>et al.</em> 2013</a>), where they are important contributors to genetic and phenotypic diversity.</p><p id="p-6">Relatively few studies have investigated whole-genome CN profiles in fungi (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Hu <em>et al.</em> 2011</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Farrer <em>et al.</em> 2013</a>; <a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Steenwyk <em>et al.</em> 2016</a>). For example, the observed CN variation of chromosome 1 in the human pathogen <em>Cryptococcus neoformans</em> results in the duplications of <em>ERG11</em>, a lanosterol-14-α-demethylase and target of the triazole antifungal drug fluconazole (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Lupetti <em>et al.</em> 2002</a>), and <em>AFR1</em>, an ATP binding cassette (ABC) transporter (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Sanguinetti <em>et al.</em> 2006</a>), leading to increased fluconazole resistance (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Sionov <em>et al.</em> 2010</a>). Similarly, resistance to itraconazole, a triazole antifungal drug, is attributed to the duplication of cytochrome P-450-depdendent C-14 lanosterol α-demethylase (<em>pdmA</em>)—a gene whose product is essential for ergosterol biosynthesis—in the human pathogen <em>Aspergillus fumigatus</em> (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Osherov <em>et al.</em> 2001</a>). Finally, in the animal pathogen <em>Batrachochytrium dendrobatidis</em>, the duplication of Supercontig V is associated with increased fitness in the presence of resistance to an antimicrobial peptide, although the underlying genetic elements involved remain elusive (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Farrer <em>et al.</em> 2013</a>).</p><p id="p-7">Similarly understudied is the contribution of CN variation to fungal domestication (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Gibbons and Rinker 2015</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Gallone <em>et al.</em> 2016</a>). Notable examples of gene duplication being associated with microbial domestication include those of α-amylase in <em>Aspergillus oryzae</em>, which is instrumental in starch saccharification during the production of sake (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Hunter <em>et al.</em> 2011</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Gibbons <em>et al.</em> 2012</a>), and of the <em><a href="http://www.yeastgenome.org/locus/S000029680/overview">MAL1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000029683/overview">MAL3</a></em> loci in beer-associated strains of <em>S. cerevisiae</em> that metabolize maltose—the most abundant sugar in the beer wort (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Gallone <em>et al.</em> 2016</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Gonçalves <em>et al.</em> 2016</a>). Beer strains of <em>S. cerevisiae</em> often contain additional duplicated genes associated with maltose metabolism, including <em><a href="http://www.yeastgenome.org/locus/S000002406/overview">MPH2</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000003921/overview">MPH3</a></em>, two maltose permeases, and the putative maltose-responsive transcription factor, <em><a href="http://www.yeastgenome.org/locus/S000006400/overview">YPR196W</a></em> (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Gonçalves <em>et al.</em> 2016</a>). Adaptive gene duplication in <em>S. cerevisiae</em> has also been detected in experimentally evolved populations (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Dunham <em>et al.</em> 2002</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Gresham <em>et al.</em> 2008</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Dunn <em>et al.</em> 2012</a>). Specifically, duplication of the locus containing the high affinity glucose transporters <em><a href="http://www.yeastgenome.org/locus/S000002751/overview">HXT6</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000002750/overview">HXT7</a></em> has been observed in adaptively evolved asexual strains (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Kao and Sherlock 2008</a>), as well as in populations grown in a glucose-limited environment (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Brown <em>et al.</em> 1998</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Dunham <em>et al.</em> 2002</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Gresham <em>et al.</em> 2008</a>). Altogether, these studies suggest that CN variation is a significant contributor to <em>S. cerevisiae</em> evolution and adaptation.</p><p id="p-8">To determine the contribution of CN variation to genome evolution in wine strains of <em>S. cerevisiae</em>, we characterized patterns of CN variation across the genomes of 132 wine strains, and determined the functional impact of CN variable genes in environments reflective of wine-making. Our results suggest that there is substantial CN variation among wine yeast strains, including in gene families (such as <em>CUP</em>, <em>FLO</em>, <em>HXT</em>, and <em>MAL</em>) known to be associated with adaptation in the fermentation environment. More generally, it raises the hypothesis that CN variation is an important contributor to adaptation during microbial domestication.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Data mining, quality control, and mapping</h3><p id="p-9">Raw sequence data for 132 <em>S. cerevisiae</em> wine strains were obtained from three studies (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Borneman <em>et al.</em> 2016</a>, 127 strains, Bioproject ID: PRJNA303109; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Dunn <em>et al.</em> 2012</a>, two strains, Bioproject ID: SRA049752; <a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Skelly <em>et al.</em> 2013</a>, three strains, Bioproject ID: PRJNA186707) (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS1.xlsx">File S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FigureS1.xlsx">Figure S1</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a>). Altogether, these 132 strains represent a diverse set of commercial and noncommercial isolates from the “wine” yeast clade (<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Borneman <em>et al.</em> 2016</a>).</p><p id="p-10">Sequence reads were quality-trimmed using Trimmomatic, version 0.36 (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Bolger <em>et al.</em> 2014</a>), with the following parameters and values: leading:10, trailing:10, slidingwindow:4:20, minlen:50. Reads were then mapped to the genome sequence of the <em>S. cerevisiae</em> strain S288c (annotation release: R64.2.1; <a href="http://www.yeastgenome.org/">http://www.yeastgenome.org/</a>) using Bowtie <span class="sc">2</span>, version 1.1.2 (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Langmead and Salzberg 2012</a>), with the “sensitive” parameter on. For each sample, mapped reads were converted to the bam format, sorted and merged using SAMtools, version 1.3.1. Sample depth of coverage was obtained using the SAMtools depth function (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Li <em>et al.</em> 2009</a>).</p></div><div id="sec-3" class="subsection"><h3>CN variant identification</h3><p id="p-11">To facilitate the identification of single nucleotide polymorphisms (SNPs), we first generated mpileup files for each strain using SAMtools, version 1.3.1 (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Li <em>et al.</em> 2009</a>). Using the mpileup files as input to VarScan, version 2.3.9 (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Koboldt <em>et al.</em> 2009</a>, <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">2012</a>), we next identified all statistically significant SNPs (Fisher’s Exact test; <em>P</em> <em>&lt;</em> 0.05) present in the 132 strains that had a read frequency of at least 0.75 and minimum coverage of 8<span class="sc">×</span>. This step enabled us to identify 149,782 SNPs. By considering only SNPs that harbored a minor allele frequency of at least 10%, we retained 43,370 SNPs. These SNPs were used to confirm the evolutionary relationships among the strains using Neighbor-Net phylogenetic network analyses in SplitsTree, version 4.14.1 (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Huson 1998</a>) as well as the previously reported low levels of SNP diversity (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FigureS2.docx">Figure S2</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a>, and <a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">Borneman <em>et al.</em> 2016</a>).</p><p id="p-12">To detect and quantify CN variants we used Control-FREEC, version 9.1 (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Boeva <em>et al.</em> 2011</a>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">2012</a>), which we chose because of its low false positive rate, and high true positive rate (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Duan <em>et al.</em> 2013</a>). Importantly, the average depth of coverage or read depth of the 132 strains was 30.1 ± 14.7<span class="sc">×</span> (minimum: 13.0<span class="sc">×,</span> maximum: 104.5<span class="sc">×;</span> <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FigureS3.xlsx">Figure S3</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a><span class="sc">)</span>, which is considered sufficient for robust CNV calling (<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Sims <em>et al.</em> 2014</a>).</p><p id="p-13">Control-FREEC uses LOESS modeling for GC-bias correction and a LASSO-based algorithm for segmentation. Implemented Control-FREEC parameters included window = 250, minExpectedGC = 0.35, maxExpectedGC = 0.55, and telocentromeric = 7000. To identify statistically significant CN variable loci (<em>P</em> &lt; 0.05), we used the Wilcoxon Rank Sum test. The same Control-FREEC parameters, but with a window size of 25 bp, were used to examine CN variation within the intragenic Serine/Threonine-rich sequences of <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">FLO11</a></em> (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Lo and Dranginis 1996</a>). Bedtools, version 2.25 (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Quinlan and Hall 2010</a>) was used to identify duplicated or deleted genic loci (<em>i.e.</em>, CN variable loci) that overlapped with genes by at least one nucleotide. The CN of each gene (genic CN) was then calculated as the average CN of the 250 bp windows that overlapped with the gene’s location coordinates in the genome. The same method was used to determine nongenic CN for loci that did not overlap with genes (<em>i.e.</em>, nongenic CN variable loci). To identify statistically significant differences between CN variable loci that were duplicated <em>vs.</em> those that were deleted, we employed the Mann-Whitney <em>U</em>-test (Wilcoxon rank-sum test) with continuity correction (<a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Wallace 2004</a>).</p></div><div id="sec-4" class="subsection"><h3>Diversity in CN variation and GO enrichment</h3><p id="p-14">To identify CN diverse loci across strains, we used two different measures. The first measure calculates the statistical variance (<em>s</em><sup>2</sup>) for each locus where CN variants were identified in one or more strains; <em>s</em><sup>2</sup> values were subsequently log<sub>10</sub> normalized. Log<sub>10</sub>(<em>s</em><sup>2</sup>) accounts for diversity in raw CN values, but not for diversity in CN allele frequencies. Thus, we also employed a second measure based on the Polymorphic Index Content (PIC) algorithm, which has previously been used to identify informative microsatellite markers for linkage analyses by taking into account both the number of alleles present and their frequencies (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Keith <em>et al.</em> 1990</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Risch 1990</a>). PIC has also been used to quantify population-level diversity of simple sequence repeat loci and restriction fragment length polymorphisms in maize (<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Smith <em>et al.</em> 1997</a>). PIC values were calculated for each locus harboring at least one CN variant based on the following formula:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1475/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1475/embed/mml-math-1.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1475/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1475/embed/mml-math-2.gif"/></noscript></span></span> is the squared frequency of <em>a</em> to <em>z</em> CN values (<a id="xref-ref-87-2" class="xref-bibr" href="#ref-87">Smith <em>et al.</em> 1997</a>). PIC values may range from 0 (no CN diversity) to 1 (all CN alleles are unique). Both log<sub>10</sub>(<em>s</em><sup>2</sup>) and PIC are measurements of CN diversity between strains, and do not account for any sequence variation between copies of the same locus.</p><p id="p-15">To create a list of loci exhibiting high CN diversity for downstream analyses, we retained only those loci that fell within the 50th percentile of log<sub>10</sub>(<em>s</em><sup>2</sup>) values (min = −2.12, median = −1.02, and max = 2.40), or the 50th percentile of PIC values (min = 0.02, median = 0.14, and max = 0.96).</p><p id="p-16">Genes overlapping with loci exhibiting high CN diversity were used for Gene ontology (GO) enrichment analysis with AmiGO2, version 2.4.24 (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Carbon <em>et al.</em> 2009</a>) using the PANTHER Overrepresentation Test (release 20160715) with default settings. This test uses the PANTHER GO database, version 11.0 (<a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Thomas <em>et al.</em> 2003</a>; release date July 15, 2016), which is directly imported from the GO Ontology database, version 1.2 (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Gene Ontology Consortium2004</a>; release date 27 October, 2016), a reference gene list from <em>S. cerevisiae</em>, and a Mann-Whitney <em>U</em>-test (Wilcoxon rank-sum test) with Bonferroni multi-test corrected <em>P</em>-values to identify over- and under-represented GO terms (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Mi <em>et al.</em> 2013</a>). Statistical analyses and figures were created using pheatmap, version 1.0.8 (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Kolde 2012</a>), gplots, version 3.0.1, ggplot2 (<a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">Wickham 2009</a>), or standard functions in R, version 3.2.2 (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">R Development Core Team 2011</a>).</p></div><div id="sec-5" class="subsection"><h3>Identifying loci absent in the reference strain</h3><p id="p-17">To identify loci absent from the reference strain but present in other strains, we assembled unmapped reads from the 20 strains with the lowest percentage of mapped reads. The percentage of mapped reads was determined using SAMtools (<a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Li <em>et al.</em> 2009</a>)<span class="sc">;</span> its average across strains was 96% (min = 70.5% and max = 99%; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FigureS4.docx">Figure S4</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a>). Unmapped reads from the 20 strains with the lowest percentage of mapped reads were assembled using SPAdes, version 3.8.1 (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bankevich <em>et al.</em> 2012</a>). The identity of scaffolds longer than the average length of a <em>S. cerevisiae</em>’<em>s</em> gene (∼1400 bp) was determined using blastx from NCBI’s BLAST, version 2.3.0 (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Madden 2013</a>) against a local copy of the GenBank nonredundant protein database (downloaded on January 5, 2017).</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-18">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Descriptive statistics of CN variation</h3><p id="p-19">To examine CN variation across wine yeasts, we generated whole genome CN profiles for 132 strains (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS1.xlsx">File S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FigureS5.xlsx">Figure S5</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a>). Across all strains, we identified a total of 2820 CNVRs that overlapped with 2061 genes and spanned 3.7 Mb. The size distribution of CNVRs was skewed toward CN variants that were &lt;1 kb in length (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FigureS6A.docx">Figure S6A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/TableS1.docx">Table S1</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a>). Most CNVRs stemming from duplications were &lt;1 kb in size, whereas most CNVRs stemming from deletions were between 5 and 7 kb (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FigureS7.docx">Figure S7</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a>). These deletions were flanked by Ty transposable elements, which have been shown to frequently induce deletions (<a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">Xu and Boeke 1987</a>). Strains had an average of 97.8 ± 9.5 CNVRs (median = 86) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FigureS6B.docx">Figure S6B</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a>) that affected an average of 4.3 ± 0.1% of the genome (median = 4.1%) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FigureS6C.xlsx">Figure S6C</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1475/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Size distribution and location of CN variable loci. (A) The fraction of CNVRs (y-axis) for a given size range. Most CNVRs are &#x2264;1000&#xA0;bp. (B, C) Deleted genic (B) and nongenic (C) CNVRs are more prevalent than duplicated ones (P&#xA0;" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1670005938" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Size distribution and location of CN variable loci. (A) The fraction of CNVRs (&lt;em&gt;y&lt;/em&gt;-axis) for a given size range. Most CNVRs are &#x2264;1000&#xA0;bp. (B, C) Deleted genic (B) and nongenic (C) CNVRs are more prevalent than duplicated ones (&lt;em&gt;P&lt;/em&gt;&#xA0;&lt;&#xA0;0.01 for both comparisons). Note that 26.23% of genic duplications occurred in multiples of three. (D) Location of CN variable loci across the 16 yeast chromosomes. The small, black squares on either side of each chromosome denote centromere location. Chromosomes are oriented with the start of the chromosome on the bottom and the end on top. Loci (blue bars) and genes (orange bars) harboring high log&lt;sub&gt;10&lt;/sub&gt;(&lt;em&gt;s&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt;) or PIC values are shown. (E) A total of 684 of the 1502 CN diverse loci and 243 of the 363 CN diverse genes reside in subtelomeric regions of the yeast genome; in contrast, very few are found in pericentromeric regions (28 loci and three genes).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1475/F1.medium.gif" width="440" height="328"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/5/1475/F1.medium.gif" width="440" height="328"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1475/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1475/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484344" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-20" class="first-child">Size distribution and location of CN variable loci. (A) The fraction of CNVRs (<em>y</em>-axis) for a given size range. Most CNVRs are ≤1000 bp. (B, C) Deleted genic (B) and nongenic (C) CNVRs are more prevalent than duplicated ones (<em>P</em> &lt; 0.01 for both comparisons). Note that 26.23% of genic duplications occurred in multiples of three. (D) Location of CN variable loci across the 16 yeast chromosomes. The small, black squares on either side of each chromosome denote centromere location. Chromosomes are oriented with the start of the chromosome on the bottom and the end on top. Loci (blue bars) and genes (orange bars) harboring high log<sub>10</sub>(<em>s</em><sup>2</sup>) or PIC values are shown. (E) A total of 684 of the 1502 CN diverse loci and 243 of the 363 CN diverse genes reside in subtelomeric regions of the yeast genome; in contrast, very few are found in pericentromeric regions (28 loci and three genes).</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">Due to the known influence of CN variable genes (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Henrichsen <em>et al.</em> 2009</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Orozco <em>et al.</em> 2009</a>), we next quantified the number of genic and nongenic CNVRs (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1, B and C</a>). We found statistically significant differences in the number of duplicated and deleted loci that are genic or nongenic (Mann-Whitney <em>U</em>-test; <em>P</em> &lt; 0.01 for both genic and nongenic comparisons), revealing that there were significantly more deleted genic and nongenic CNVRs than duplicated ones.</p></div><div id="sec-9" class="subsection"><h3>CN diversity in subtelomeres</h3><p id="p-22">To identify loci that exhibited high CN diversity, we retained only those loci that fell within the 50th percentile of at least one of our two different measures [log<sub>10</sub>(<em>s</em><sup>2</sup>) and PIC] across the 132 strains. These loci covered ∼6% of the yeast genome. The distributions of the two measures (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FigureS8.docx">Figure S8</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a>) were similar, with 1326 loci (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FigureS8C.docx">Figure S8C</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a>) and 291 genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FigureS8D.docx">Figure S8D</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a>) identified in the top 50% of CN diverse genes by both measures.</p><p id="p-23">In addition, the log<sub>10</sub>(<em>s</em><sup>2</sup>) measure identified an additional 85 loci and 54 genes in its set of top 50% genes, and PIC an additional 85 loci and 18 genes. In total, our analyses identified 1502 loci and 363 genes showing high CN diversity. Among the genes harboring the highest log<sub>10</sub>(<em>s</em><sup>2</sup>) and PIC values were <em><a href="http://www.yeastgenome.org/locus/S000028561/overview">YLR154C-G</a></em> [PIC = 0.96; log<sub>10</sub>(<em>s</em><sup>2</sup>) = 2.16], <em><a href="http://www.yeastgenome.org/locus/S000028675/overview">YLR154W-A</a></em> [PIC = 0.96; log<sub>10</sub>(<em>s</em><sup>2</sup>) = 2.16], <em><a href="http://www.yeastgenome.org/locus/S000028563/overview">YLR154W-B</a></em> [PIC = 0.96; log<sub>10</sub>(<em>s</em><sup>2</sup>) = 2.16], <em><a href="http://www.yeastgenome.org/locus/S000028422/overview">YLR154W-C</a></em> [PIC = 0.96; log<sub>10</sub>(<em>s</em><sup>2</sup>) = 2.16], <em><a href="http://www.yeastgenome.org/locus/S000028676/overview">YLR154W-E</a></em> [PIC = 0.96; log<sub>10</sub>(<em>s</em><sup>2</sup>) = 2.16], <em><a href="http://www.yeastgenome.org/locus/S000028843/overview">YLR154W-F</a></em> [PIC = 0.96; log<sub>10</sub>(<em>s</em><sup>2</sup>) = 2.16] and <em><a href="http://www.yeastgenome.org/locus/S000028562/overview">YLR154C-H</a></em> [PIC = 0.93; log<sub>10</sub>(<em>s</em><sup>2</sup>) = 2.40]; these genes are all encoded within the 25S rDNA or 35S rDNA locus. The rDNA locus is known to be highly CN diverse (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Gibbons <em>et al.</em> 2015</a>), thereby demonstrating the utility and efficacy of our CN calling protocol as well as our two measures of CN diversity. We next generated CN diversity maps for all 16 <em>S. cerevisiae</em> chromosomes (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1D</a>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FigureS9.docx">Figure S9</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a>). CN diversity was higher in loci and genes located in subtelomeres (defined as the 25 kb of DNA immediately adjacent to the chromosome ends; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Barton <em>et al.</em> 2003</a>). Specifically, 684/1502 (45.5%) of CN diverse loci and 243/363 (66.9%) CN diverse genes were located in the subtelomeric regions. Conducting the same analysis using an alternative definition of subtelomere (defined as the DNA between the chromosome’s end to the first essential gene; <a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">Winzeler <em>et al.</em> 1999</a>) showed similar results. Specifically, 721/1502 (48%) of CN diverse loci and 233/363 (64.2%) of CN diverse genes were located in the subtelomeric regions.</p></div><div id="sec-10" class="subsection"><h3>GO enrichment of CN diverse genes</h3><p id="p-24">To determine the functional categories over- and under-represented in the 363 genes showing high CN diversity, we performed GO enrichment analysis. The majority of enriched GO terms were associated with metabolic functions such as α<span class="sc">-glucosidase activity</span> (<em>P &lt;</em> 0.01) and <span class="sc">carbohydrate transporter activity</span> (<em>P</em> <em>&lt;</em> 0.01) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS3.docx">File S3</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS1.xlsx">File S1</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1475/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="GO enriched terms from high CN diverse genes. Molecular function (black), biological process (gray), and cellular component (white) GO categories are represented by circles, and are enriched among the 363 genes that overlap with CN diverse loci. Enriched terms are primarily related to metabolic function, such as &#x3B1;-glucosidase activity (P&#xA0;" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1670005938" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;GO enriched terms from high CN diverse genes. Molecular function (black), biological process (gray), and cellular component (white) GO categories are represented by circles, and are enriched among the 363 genes that overlap with CN diverse loci. Enriched terms are primarily related to metabolic function, such as &#x3B1;-glucosidase activity (&lt;em&gt;P&lt;/em&gt;&#xA0;&lt;em&gt;&lt;&lt;/em&gt;&#xA0;0.01), carbohydrate transporter activity (&lt;em&gt;P&lt;/em&gt;&#xA0;&lt;em&gt;&lt;&lt;/em&gt;&#xA0;0.01) and flocculation (&lt;em&gt;P&lt;/em&gt;&#xA0;&lt;&#xA0;0.01).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1475/F2.medium.gif" width="440" height="178"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/5/1475/F2.medium.gif" width="440" height="178"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1475/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1475/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484340" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-25" class="first-child">GO enriched terms from high CN diverse genes. Molecular function (black), biological process (gray), and cellular component (white) GO categories are represented by circles, and are enriched among the 363 genes that overlap with CN diverse loci. Enriched terms are primarily related to metabolic function, such as α<span class="sc">-glucosidase activity</span> (<em>P</em> <em>&lt;</em> 0.01), <span class="sc">carbohydrate transporter activity</span> (<em>P</em> <em>&lt;</em> 0.01) and <span class="sc">flocculation</span> (<em>P</em> &lt; 0.01).</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">Genes associated with these GO terms include <em><a href="http://www.yeastgenome.org/locus/S000001424/overview">SUC2</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000001424/overview">YIL162W</a></em>, involved in hydrolyzing sucrose), all six members from the <em>MAL</em> gene family (involved in the fermentation of maltose and other carbohydrates), and all five members of the <em>IMA</em> gene family (involved in isomaltose, sucrose, and turanose metabolism). Other enriched categories were associated with multi-cellular processes such as the <span class="sc">flocculation</span> (<em>P</em> <em>&lt;</em> 0.01) and <span class="sc">aggregation of unicellular organisms</span> (<em>P</em> = 0.03). All members of the <em>FLO</em> gene family (involved in flocculation) and <em><a href="http://www.yeastgenome.org/locus/S000001256/overview">YHR213W</a></em> (a flocculin-like gene) were associated with these GO enriched terms.</p><p id="p-27">Contrary to over-represented GO terms, under-represented terms were associated with genes whose protein products are part of the interactome or protein–protein interactions such as <span class="sc">protein complex</span> (<em>P</em> &lt; 0.01), <span class="sc">macromolecular complex assembly</span> (<em>P</em> = 0.03), <span class="sc">transferase complex</span> (<em>P</em> &lt; 0.01), and <span class="sc">ribonucleoprotein complex biogenesis</span> (<em>P</em> = 0.04). Our finding of under-represented GO terms being associated with multi-unit protein complexes supports the gene balance hypothesis, which states that the stoichiometry of genes contributing to multi-subunit complexes must be maintained to conserve kinetics and assembly properties (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Birchler and Veitia 2010</a>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">2012</a>). Thus, genes associated with multi-unit protein complexes are unlikely to exhibit CN variation.</p></div><div id="sec-11" class="subsection"><h3>Genic CN diversity</h3><p id="p-28">To further understand the structure of CN variation in highly diverse CN genes, we first calculated the absolute CN of 23 genes associated with GO enriched terms related to wine fermentation processes (<em>e.g.</em>, metabolic functions; <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS3.docx">File S3</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS1.xlsx">File S1</a>) as well as 57 genes with the highest PIC or log<sub>10</sub>(<em>s</em><sup>2</sup>) values (File S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS1.xlsx">File S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FigureS10.docx">Figure S10</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a>; 69 total unique genes). Among these 69 genes, gene CN ranged from 0 to 92; both the highest CN diversity and absolute CN values were observed in segments of the rDNA locus (mentioned above).</p><p id="p-29">Importantly, 35 of the 69 genes have also been reported to have functional roles in fermentation-related processes. The remaining 34 genes with high scores for CN diversity, to our knowledge, have no known association with wine fermentation (<em>e.g.</em>, <em><a href="http://www.yeastgenome.org/locus/S000002141/overview">YAL064W-B</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000006062/overview">FRK1</a>/<a href="http://www.yeastgenome.org/locus/S000006062/overview">YPL141C</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000002445/overview">ENA5</a>/<a href="http://www.yeastgenome.org/locus/S000002445/overview">YDR038C</a></em> among others). All genes with high CN diversity scores are listed in File S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS1.xlsx">File S1</a> and summarized in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FigureS10.docx">Figure S10</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a>. For the purposes of this manuscript, we chose to focus on genes related to fermentation. For example, the CNs of <em><a href="http://www.yeastgenome.org/locus/S000000701/overview">PAU3</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000000701/overview">YCR104W</a></em>), a gene active during alcoholic fermentation, and its gene neighbor <em><a href="http://www.yeastgenome.org/locus/S000000702/overview">ADH7</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000000702/overview">YCR105W</a></em>), an alcohol dehydrogenase, both varied between zero and three. Similarly, the absolute CN of the locus containing both <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1-1</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000001095/overview">YHR053C</a></em>; PIC = 0.868) and its paralog <em><a href="http://www.yeastgenome.org/locus/S000001097/overview">CUP1-2</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000001097/overview">YHR055C</a></em>; PIC = 0.879) ranged from 0 and 14 (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS4.docx">File S4</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS1.xlsx">File S1</a>), with 90 strains (68.2%) showing duplications (<em>i.e.</em>, a CN &gt; 1), and another 11 strains (8.3%) a deletion (<em>i.e.</em>, a CN of 0). Interestingly, multiple copies of <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> confer copper resistance to wine strains of <em>S. cerevisiae</em>, with CN variation at this locus thought to be associated with domestication (<a id="xref-ref-97-2" class="xref-bibr" href="#ref-97">Warringer <em>et al.</em> 2011</a>; <a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Marsit and Dequin 2015</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1475/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="CN variation of genes and gene families. Heat map of the CN profiles the CUP, THI, SNO, MAL, IMA, and HXT gene families; rows correspond to genes and columns to strains. Blue-colored cells correspond to deletions, black-colored cells to no CN variation and red-to-purple-colored cells to duplications (ranging from 2 to 14). Dots on the right side of the figure represent the proportion of individual strains that harbor CN variation in that gene&#x2014;the larger the dot, the greater the proportion of the strains that is CN variable for that gene." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1670005938" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;CN variation of genes and gene families. Heat map of the CN profiles the &lt;em&gt;CUP&lt;/em&gt;, &lt;em&gt;THI&lt;/em&gt;, &lt;em&gt;SNO&lt;/em&gt;, &lt;em&gt;MAL&lt;/em&gt;, &lt;em&gt;IMA&lt;/em&gt;, and &lt;em&gt;HXT&lt;/em&gt; gene families; rows correspond to genes and columns to strains. Blue-colored cells correspond to deletions, black-colored cells to no CN variation and red-to-purple-colored cells to duplications (ranging from 2 to 14). Dots on the right side of the figure represent the proportion of individual strains that harbor CN variation in that gene&#x2014;the larger the dot, the greater the proportion of the strains that is CN variable for that gene.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1475/F3.medium.gif" width="440" height="305"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/5/1475/F3.medium.gif" width="440" height="305"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1475/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1475/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484315" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-30" class="first-child">CN variation of genes and gene families. Heat map of the CN profiles the <em>CUP</em>, <em>THI</em>, <em>SNO</em>, <em>MAL</em>, <em>IMA</em>, and <em>HXT</em> gene families; rows correspond to genes and columns to strains. Blue-colored cells correspond to deletions, black-colored cells to no CN variation and red-to-purple-colored cells to duplications (ranging from 2 to 14). Dots on the right side of the figure represent the proportion of individual strains that harbor CN variation in that gene—the larger the dot, the greater the proportion of the strains that is CN variable for that gene.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">The expression of <em>SNO</em> family members is induced just prior to, or after, the diauxic shift as a response to nutrient limitation, and is associated with vitamin B acquisition (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Padilla <em>et al.</em> 1998</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Rodríguez-Navarro <em>et al.</em> 2002</a>). We found that <em><a href="http://www.yeastgenome.org/locus/S000005278/overview">SNO2</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000005278/overview">YNL334C</a></em>) and <em><a href="http://www.yeastgenome.org/locus/S000001834/overview">SNO3</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000001834/overview">YFL060C</a></em>) were among the 363 genes with highest CN diversity. <em><a href="http://www.yeastgenome.org/locus/S000005278/overview">SNO2</a></em> was duplicated in 14 strains (10.6%) and deleted in nine strains (6.8%), while <em><a href="http://www.yeastgenome.org/locus/S000001834/overview">SNO3</a></em> was deleted in 117 strains (88.6%) (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). The other two members of the SNO gene family, <em><a href="http://www.yeastgenome.org/locus/S000004701/overview">SNO1</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000004701/overview">YMR095C</a></em>) and <em><a href="http://www.yeastgenome.org/locus/S000004941/overview">SNO4</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000004941/overview">YMR322C</a></em>), both showed a CN of 1 in all strains.</p><p id="p-32">Another gene family whose members show high CN diversity is the <em>THI</em> gene family, which is responsible for thiamine metabolism, and is activated at the end of the growth phase during fermentation (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Brion <em>et al.</em> 2014</a>). Specifically, <em><a href="http://www.yeastgenome.org/locus/S000002403/overview">THI13</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000002403/overview">YDL244W</a></em>; PIC = 0.759) was among the 57 genes with the highest CN diversity (File S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS1.xlsx">File S1</a>), and <em><a href="http://www.yeastgenome.org/locus/S000001836/overview">THI5</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000001836/overview">YFL058W</a></em>) and <em><a href="http://www.yeastgenome.org/locus/S000005276/overview">THI12</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000005276/overview">YNL332W</a></em>) among the 363 most CN diverse genes (File S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS1.xlsx">File S1</a>). <em><a href="http://www.yeastgenome.org/locus/S000002403/overview">THI13</a></em> was duplicated in 82 strains (62.1%) and deleted in two strains (1.5%) (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). In contrast, <em><a href="http://www.yeastgenome.org/locus/S000001836/overview">THI5</a></em> was deleted in 121 strains (91.67%), whereas <em><a href="http://www.yeastgenome.org/locus/S000005276/overview">THI12</a></em> was deleted in 23 strains (17.42%), and duplicated in only three strains (2.27%). Lastly, the CN of the last <em>THI</em> gene family member, <em><a href="http://www.yeastgenome.org/locus/S000003917/overview">THI11</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000003917/overview">YJR156C</a></em>), did not exhibit CN variation.</p><p id="p-33">In addition to the high CN diversity observed in all six members of the <em><a href="http://www.yeastgenome.org/locus/S000029680/overview">MAL1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000029683/overview">MAL3</a></em> loci responsible for maltose metabolism and growth on sucrose (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Stambuk <em>et al.</em> 2000</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Gallone <em>et al.</em> 2016</a>), <em><a href="http://www.yeastgenome.org/locus/S000003520/overview">MAL13</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000003520/overview">YGR288W</a></em>; PIC = 0.53) was among the 57 genes with the highest CN diversity (File S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS1.xlsx">File S1</a>). Evaluation of the absolute CN of all <em><a href="http://www.yeastgenome.org/locus/S000029680/overview">MAL1</a></em> locus genes (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>) showed that <em><a href="http://www.yeastgenome.org/locus/S000003521/overview">MAL11</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003524/overview">MAL12</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000003524/overview">YGR292W</a></em>), and <em><a href="http://www.yeastgenome.org/locus/S000003520/overview">MAL13</a></em> were deleted in 65 (49.2%), 86 (65.2%), and 61 strains (46.2%), respectively. In contrast, the <em><a href="http://www.yeastgenome.org/locus/S000029683/overview">MAL3</a></em> locus genes <em><a href="http://www.yeastgenome.org/locus/S000000502/overview">MAL31</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000000502/overview">YBR298C</a></em>), <em><a href="http://www.yeastgenome.org/locus/S000000503/overview">MAL32</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000000503/overview">YBR299W</a></em>), and <em><a href="http://www.yeastgenome.org/locus/S000000501/overview">MAL33</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000000501/overview">YBR297W</a></em>) were duplicated in 100 (75.8%), 99 (75%), and 98 strains (74.2%), respectively. Interestingly, we did not observe any deletions in any of the <em><a href="http://www.yeastgenome.org/locus/S000029683/overview">MAL3</a></em> locus genes across the 132 strains. When considering all members of the <em>MAL</em> gene family, we found that the 132 strains differed widely in their degree to which the locus had undergone expansion or contraction (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FigureS11.docx">Figure S11</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a>).</p><p id="p-34">All members of the <em>IMA</em> gene family, composed of genes aiding in sugar fermentation (<a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Teste <em>et al.</em> 2010</a>), were among the 363 genes with high CN diversity (File S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS1.xlsx">File S1</a>) and <em><a href="http://www.yeastgenome.org/locus/S000003519/overview">IMA1</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000003519/overview">YGR287C</a></em>; PIC = 0.87) was among the top 57 genes with the highest CN diversity (File S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS1.xlsx">File S1</a>). <em><a href="http://www.yeastgenome.org/locus/S000003519/overview">IMA1</a></em> was deleted in 54 strains (40.9%) and duplicated in 50 strains (37.9%) (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>). Although many duplications or deletions did not span the entirety of <em><a href="http://www.yeastgenome.org/locus/S000003519/overview">IMA1</a></em>, there were four strains that harbored high CNs between four and six. These same four strains also had similar and unique duplications of <em><a href="http://www.yeastgenome.org/locus/S000003521/overview">MAL11</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000003520/overview">MAL13</a></em>, suggesting that <em><a href="http://www.yeastgenome.org/locus/S000003519/overview">IMA1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003521/overview">MAL11</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000003520/overview">MAL13</a></em>, which are adjacent to each other in the genome, may have been duplicated as one locus. The other isomaltases (<em>IMA2-5</em>; <em><a href="http://www.yeastgenome.org/locus/S000005517/overview">YOL157C</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001434/overview">YIL172C</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003757/overview">YJL221C</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000003752/overview">YJL216C</a></em>) were deleted in at least 11 strains (8.3%) and at most 55 strains (41.7%). No duplications in <em>IMA2</em>-5 were detected, and only rarely in <em><a href="http://www.yeastgenome.org/locus/S000001434/overview">IMA3</a></em> (five strains, 3.8%). Altogether, the 132 strains exhibited both expansions and contractions of the <em>IMA</em> gene family (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FigureS11.docx">Figure S11</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a>).</p><p id="p-35">We identified seven members of the <em>HXT</em> gene family (<em><a href="http://www.yeastgenome.org/locus/S000002751/overview">HXT6</a></em>/<em><a href="http://www.yeastgenome.org/locus/S000002751/overview">YDR343C</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002750/overview">HXT7</a></em>/<em><a href="http://www.yeastgenome.org/locus/S000002750/overview">YDR342C</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003755/overview">HXT9</a>/</em><em><a href="http://www.yeastgenome.org/locus/S000003755/overview">YJL219W</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005516/overview">HXT11</a>/</em><em><a href="http://www.yeastgenome.org/locus/S000005516/overview">YOL156W</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000795/overview">HXT13</a>/</em><em><a href="http://www.yeastgenome.org/locus/S000000795/overview">YEL069C</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002404/overview">HXT15</a></em>/<em><a href="http://www.yeastgenome.org/locus/S000002404/overview">YDL245C</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000005355/overview">HXT17</a></em>/<em><a href="http://www.yeastgenome.org/locus/S000005355/overview">YNR072W</a></em>), which is involved in sugar transport, that were among the 363 CN diverse genes (File S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS1.xlsx">File S1</a>). Members of the <em>HXT</em> gene family were duplicated, deleted, or had mosaic absolute CN values across the 132 strains. For example, <em><a href="http://www.yeastgenome.org/locus/S000002751/overview">HXT6</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000002750/overview">HXT7</a></em> were primarily duplicated in 25 (18.9%) and 22 strains (16.7%), respectively, while only three strains (2.3%) had deletions in either gene (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>). <em><a href="http://www.yeastgenome.org/locus/S000003755/overview">HXT9</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005516/overview">HXT11</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000002404/overview">HXT15</a></em> were deleted in 32 (24.2%), 57 (43.2%), and 53 strains (40.2%), respectively, while no strains had duplications. Finally, <em><a href="http://www.yeastgenome.org/locus/S000000795/overview">HXT13</a></em> was duplicated in 12 strains (9.1%) and deleted in 17 strains (12.9%), and <em><a href="http://www.yeastgenome.org/locus/S000005355/overview">HXT17</a></em> was duplicated in 37 strains (28%) and deleted in nine strains (6.8%).</p><p id="p-36">As expansions in the <em>HXT</em> gene family are positively correlated with aerobic fermentation in <em>S. paradoxus</em> and <em>S. cerevisiae</em> (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Lin and Li 2011</a>), we also examined the absolute CN of all other 10 members (<em><a href="http://www.yeastgenome.org/locus/S000004071/overview">GAL2</a>/</em><em><a href="http://www.yeastgenome.org/locus/S000004071/overview">YLR081W</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001136/overview">HXT1</a>/</em><em><a href="http://www.yeastgenome.org/locus/S000001136/overview">YHR094C</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000004613/overview">HXT2</a>/</em><em><a href="http://www.yeastgenome.org/locus/S000004613/overview">YMR011W</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001134/overview">HXT4</a>/</em><em><a href="http://www.yeastgenome.org/locus/S000001134/overview">YHR092C</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001138/overview">HXT5</a>/</em><em><a href="http://www.yeastgenome.org/locus/S000001138/overview">YHR096C</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003750/overview">HXT8</a>/</em><em><a href="http://www.yeastgenome.org/locus/S000003750/overview">YJL214W</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001883/overview">HXT10</a>/</em><em><a href="http://www.yeastgenome.org/locus/S000001883/overview">YFL011W</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003919/overview">HXT16</a>/</em><em><a href="http://www.yeastgenome.org/locus/S000003919/overview">YJR158W</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002297/overview">RGT2</a>/</em><em><a href="http://www.yeastgenome.org/locus/S000002297/overview">YDL138W</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000002353/overview">SNF3</a>/</em><em><a href="http://www.yeastgenome.org/locus/S000002353/overview">YDL194W</a></em>) of the <em>HXT</em> gene family (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3</a>). Interestingly, all remaining 10 members of the <em>HXT</em> gene family were not CN variable. Altogether, examination of the <em>HXT</em> family CN diversity patterns across the 132 strains suggests that wine yeast strains typically exhibit minor contractions (<em>i.e.</em>, <em>HXT</em> gene deletions exceed those of duplications) relative to the S288c reference strain (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FigureS11.docx">Figure S11</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a>).</p><p id="p-37">All five members of the <em>FLO</em> gene family, which is responsible for flocculation (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Govender <em>et al.</em> 2008</a>), a trait shown to aid in the escape of oxygen limited environments during liquid fermentation (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Fidalgo <em>et al.</em> 2006</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Govender <em>et al.</em> 2008</a>), were found to be among the 363 most CN diverse genes. Furthermore, <em><a href="http://www.yeastgenome.org/locus/S000001254/overview">FLO5</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000001254/overview">YHR211W</a></em>; PIC = 0.82) and <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">FLO11</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000001458/overview">YIR019C</a></em>; PIC = 0.88) were among the 57 genes with the highest CN diversity (File S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS1.xlsx">File S1</a>). Due to the importance of site-directed CN variation in <em>FLO</em> family genes (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Fidalgo <em>et al.</em> 2006</a>), we modified our representation of CN variation to display intragenic CN variation using a 250 bp window (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FigureS12.docx">Figure S12</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a>). <em><a href="http://www.yeastgenome.org/locus/S000001254/overview">FLO5</a></em> was partially duplicated in 57 strains (43.2%), partially deleted in 47 strains (35.6%), and 115 strains (87.1%) had at least one region of the gene unaffected by CN variation. Duplications and deletions were primarily observed in the Threonine-rich region or Serine/Threonine-rich region located in the center or end of the <em><a href="http://www.yeastgenome.org/locus/S000001254/overview">FLO5</a></em> gene, respectively. To better resolve intragenic CN variation of <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">FLO11</a></em>, whose repeat unit is shorter than that of <em><a href="http://www.yeastgenome.org/locus/S000001254/overview">FLO5</a></em>, we recalled CN variants with a smaller window size of 25 bp and re-evaluated CN variation (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FigureS13.docx">Figure S13</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a>). Using this window size, we found extensive duplications in 97 strains (73.5%) between gene coordinates 250–350 bp. Furthermore, duplications were observed in the hydrophobic Serine/Threonine-rich regions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FigureS13.docx">Figure S13</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a>), which are associated with the flocculation phenotype (<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Fidalgo <em>et al.</em> 2006</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Ramsook <em>et al.</em> 2010</a>).</p><p id="p-38">In contrast to <em><a href="http://www.yeastgenome.org/locus/S000001254/overview">FLO5</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">FLO11</a></em>, other members of the <em>FLO</em> gene family did not exhibit intragenic CN variation. For example, CN variation in <em><a href="http://www.yeastgenome.org/locus/S000000084/overview">FLO1</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000000084/overview">YAR050W</a></em>) and <em><a href="http://www.yeastgenome.org/locus/S000000059/overview">FLO9</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000000059/overview">YAL063C</a></em>) typically spanned most or all of the sequence of each gene. Specifically, 125 strains (99.2%) had deletions spanning ≥80% of the gene in <em><a href="http://www.yeastgenome.org/locus/S000000084/overview">FLO1</a></em>, and only two strains (1.5%) had the entirety of the gene intact. <em><a href="http://www.yeastgenome.org/locus/S000000059/overview">FLO9</a></em> had deletions in 99 strains (75%) that spanned ≥75% of the gene, 11 strains (8.3%) that had a partial deletion spanning &lt;75% of the gene, whereas one strain (0.8%) had a CN of two, and the remaining 21 strains (15.9%) had a CN of one. In contrast, <em><a href="http://www.yeastgenome.org/locus/S000001810/overview">FLO10</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000001810/overview">YKR102W</a></em>) showed limited CN variation. Specifically, 108 strains (81.8%) had no CN variation while six strains (4.5%) had deletions spanning the entirety of the gene. No duplications spanned the entirety of the gene but partial duplications were observed in 17 strains (12.9%) and were located in, or just before, the Serine/Threonine-rich region.</p></div><div id="sec-12" class="subsection"><h3>Functional implications CN variable genes</h3><p id="p-39">To determine the functional impact of deleted CN variable genes, we examined the relative growth of deleted CN variable genes (denoted with the Δ symbol) relative to the wild-type (WT) <em>S. cerevisiae</em> strain S288c across 418 conditions using the <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Hillenmeyer <em>et al.</em> (2008)</a> data (File S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS1.xlsx">File S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FigureS14.docx">Figure S14</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a>). To determine the impact of duplicated genes, we examined growth fitness of the WT strain with low (∼2–3 gene copies) or high (∼8–24 gene copies) plasmid CN, where each plasmid contained a single gene of interest from previously published data, relative to WT (File S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS1.xlsx">File S1</a>, Figure S15 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a>, and <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Payen <em>et al.</em> 2016</a>).</p><p id="p-40">Among deleted genes, 42/69 genes for which data exist showed negative and positive fitness effects in at least one tested condition in the S288c genetic background. Furthermore, we found that 12/42 genes that are commonly deleted among wine strains typically resulted in a fitness gain in conditions that resembled the fermentation environment. These conditions include growth at 23° and at 25°, temperatures within the 15–28° range in which wine is fermented (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Molina <em>et al.</em> 2007</a>), and growth in minimal media, which is commonly used to understand fermentation-related processes (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Seki <em>et al.</em> 1985</a>; <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Govender <em>et al.</em> 2008</a>; <a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Vilela-Moura <em>et al.</em> 2008</a>).</p><p id="p-41">When examining fitness effects when grown at 23° or at 25° for five or 15 generations for the 12 commonly deleted genes, we observed at least one deletion that resulted in a fitness gain or loss for each condition. However, we observed extensive deletions in the <em><a href="http://www.yeastgenome.org/locus/S000029680/overview">MAL1</a></em> locus (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3</a>), and therefore prioritized reporting the fitness impact of deletions in <em><a href="http://www.yeastgenome.org/locus/S000003521/overview">MAL11</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003524/overview">MAL12</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000003520/overview">MAL13</a></em>. Δ<em><a href="http://www.yeastgenome.org/locus/S000003521/overview">MAL11</a></em> resulted in a fitness gain for growth at 23° and 25° for 5 (0.45<span class="sc">×</span> and 0.27<span class="sc">×,</span> respectively) and 15 generations (0.20<span class="sc">×</span> and 0.52<span class="sc">×,</span> respectively). Δ<em><a href="http://www.yeastgenome.org/locus/S000003524/overview">MAL12</a></em> resulted in a gain of fitness at only 25° after 15 generations (0.46<span class="sc">×</span>) and in a loss of fitness ranging from −0.36<span class="sc">×</span> to −1.29<span class="sc">×</span> in the other temperature conditions. Similarly, Δ<em><a href="http://www.yeastgenome.org/locus/S000003520/overview">MAL13</a></em> resulted in fitness gains and losses dependent on the number of generations. For example, when grown for 15 generations at 25° a fitness gain of 0.50<span class="sc">×</span> was observed, while a fitness loss of −0.82<span class="sc">×</span> was observed at 23°.</p><p id="p-42">We next determined the fitness effect of deleted genes in minimal media after 0, 5, and 10 generations. Similar patterns of complex fitness gain and loss were observed as for the other conditions. For example, Δ<em><a href="http://www.yeastgenome.org/locus/S000005276/overview">THI12</a></em> resulted in a loss of fitness of −4.13<span class="sc">×</span> and −1.97<span class="sc">×</span> after 0 and 5 generations, but a fitness gain of 0.63<span class="sc">×</span> after 10 generations. In contrast, other genes resulted in positive fitness effects. For example, Δ<em><a href="http://www.yeastgenome.org/locus/S000003524/overview">MAL12</a></em> resulted in a fitness gain of 7.25<span class="sc">×</span> and 10.41<span class="sc">×</span> for 0 generations and 10 generations.</p><p id="p-43">Among duplicated genes, we focused on growth in glucose- and phosphate-limited conditions because glucose becomes scarce toward the end of fermentation prior to the diauxic shift and phosphate limitation is thought to contribute to stuck fermentations (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bisson 1999</a>; <a id="xref-ref-60-4" class="xref-bibr" href="#ref-60">Marsit and Dequin 2015</a>). Among the 35 of the 69 genes where data were available, 14 genes had duplications among the 132 strains.</p><p id="p-44">When examining the fitness effects of duplicated genes in a glucose-limited environment in the S288c background, we found that fitness effects were small in magnitude, and dependent on condition and plasmid CN (File S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS1.xlsx">File S1</a>). For example, <em><a href="http://www.yeastgenome.org/locus/S000000503/overview">MAL32</a></em> low CN increased growth fitness by 0.02<span class="sc">×</span> but decreased fitness by −0.01<span class="sc">×</span> at a high CN (Figure S15 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a>). Interestingly, the most prevalent CN for <em><a href="http://www.yeastgenome.org/locus/S000000503/overview">MAL32</a></em> across the 132 strains was two (96 strains, 72.7%), with only three strains showing a CN of three and none a higher CN. Another gene found at low CN in 37 strains (28%) was <em><a href="http://www.yeastgenome.org/locus/S000005355/overview">HXT17</a></em>. Low plasmid CN in a glucose-limited conditions resulted in a fitness gain of 0.06<span class="sc">×</span>. In contrast, <em><a href="http://www.yeastgenome.org/locus/S000003520/overview">MAL13</a></em> low or high plasmid CN resulted in a negative growth fitness of −0.02<span class="sc">×</span> and −0.01<span class="sc">×,</span> respectively. Interestingly, <em><a href="http://www.yeastgenome.org/locus/S000003520/overview">MAL13</a></em> duplication is observed in only four strains (3%), and deletions are observed in 61 strains (46.2%).</p><p id="p-45">Similar to the glucose-limited condition, we found fitness was dependent on high or low plasmid CN in the phosphate-limited condition. For example, <em><a href="http://www.yeastgenome.org/locus/S000000502/overview">MAL31</a></em>, a gene present at low CN in 100 strains, had a fitness gain of 0.04<span class="sc">×</span> at high plasmid CN, but low plasmid CN resulted in a fitness loss of −0.02<span class="sc">×.</span> In contrast, <em><a href="http://www.yeastgenome.org/locus/S000000503/overview">MAL32</a></em>, which was present at low CN in 99 strains, had a small fitness gain of 0.002× at low plasmid CN, and a fitness loss at a high plasmid CN of −0.02×. A total of six genes resulted in a disadvantageous growth effect when present at low CN, such as <em><a href="http://www.yeastgenome.org/locus/S000004784/overview">DDR48</a></em>, which resulted in a fitness loss of −0.04<span class="sc">×.</span> Altogether, our results suggest that the deleted and duplicated CN variable genes we observe (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>) modulate cellular processes that result in advantageous fitness effects in conditions that resemble the fermentation environment.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1475/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Model summary of CN variable genes in wine yeast strains and their cellular functions. Genes that are deleted among wine strains are blue, whereas those that are duplicated are in red. Genes that were observed to be both duplicated and deleted (IMA1, IMA3, HXT13, and CUP1-1) are purple. Disaccharides are in thick-lined boxes, monosaccharides in thin-lined boxes, and alcohols are unboxed." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1670005938" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Model summary of CN variable genes in wine yeast strains and their cellular functions. Genes that are deleted among wine strains are blue, whereas those that are duplicated are in red. Genes that were observed to be both duplicated and deleted (&lt;em&gt;IMA1&lt;/em&gt;, &lt;em&gt;IMA3&lt;/em&gt;, &lt;em&gt;HXT13&lt;/em&gt;, and &lt;em&gt;CUP1-1&lt;/em&gt;) are purple. Disaccharides are in thick-lined boxes, monosaccharides in thin-lined boxes, and alcohols are unboxed.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1475/F4.medium.gif" width="440" height="367"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/5/1475/F4.medium.gif" width="440" height="367"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1475/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1475/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484401" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-46" class="first-child">Model summary of CN variable genes in wine yeast strains and their cellular functions. Genes that are deleted among wine strains are blue, whereas those that are duplicated are in red. Genes that were observed to be both duplicated and deleted (<em>IMA1</em>, <em>IMA3</em>, <em>HXT13</em>, and <em>CUP1-1</em>) are purple. Disaccharides are in thick-lined boxes, monosaccharides in thin-lined boxes, and alcohols are unboxed.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Identifying loci absent from CN variation analysis</h3><p id="p-47">The present study was able to capture loci represented in the WT/S288c laboratory strain. To identify loci absent from the reference strain, we assembled unmapped reads for 20 strains with the lowest percentage of reads mapped and determined their identity (see <em>Materials and Methods</em>; Figure S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a>). Across the 20 strains, we identified 429 loci absent from S288c but present in other sequenced <em>S. cerevisiae</em> strains. These loci had an average length of 6.9 kb and an average coverage of 107.2<span class="sc">×</span>. The 20 loci with the highest bitscore alongside with the number of strains containing the locus are shown in Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a>. All but two of these loci were present only in one of the 20 strains we examined. The two exceptions were the EC1118_1N26_0012p locus, which we found in 8/20 strains, which originates from horizontal gene transfer from <em>Zygosaccharomyces rouxii</em> to the commercial EC1118 wine strain of <em>S. cerevisiae</em> (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Novo <em>et al.</em> 2009</a>); and the EC1118_1O4_6656p locus, which we found in 7/20 strains. This locus was also originally found in the EC1118 strain (<a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Novo <em>et al.</em> 2009</a>), and contains a gene similar to a conserved hypothetical protein found in <em>S. cerevisiae</em> strain AWRI1631 (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Borneman <em>et al.</em> 2008</a>).</p></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><p id="p-48">CN variant loci are known to contribute to genomic and phenotypic diversity (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Perry <em>et al.</em> 2007</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Cutler and Kassner 2008</a>; <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Orozco <em>et al.</em> 2009</a>). However, the extent of CN variation in wine strains of <em>S. cerevisiae</em>, and its impact on phenotypic variation remains less understood. Our examination of structural variation in 132 yeast strains representative of the “wine clade” showed that CN variants are a significant contributor to the genomic diversity of wine strains of <em>S. cerevisiae</em>. Importantly, CN variant loci overlap with diverse genes and gene families functionally related to the fermentation environment such as <em>CUP</em>, <em>FLO</em>, <em>THI</em>, <em>MAL</em>, <em>IMA</em>, and <em>HXT</em> (summarized in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>).</p><p id="p-49">The characteristics of CN variation in wine yeast (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a>, and Figure S6 and Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a>) were found to be similar to those of the recently described beer yeast lineage (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Gallone <em>et al.</em> 2016</a>). For example, both lineages exhibited a similar size range of CNVRs (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1A</a>, and Figure S6 and Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a>), as well as a higher prevalence of CNVRs in the subtelomeric regions (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1D</a>). However, wine strains had a smaller fraction of their genome affected by CN variation (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FigureS6.docx">Figure S6</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1/FileS2.docx">File S2</a>) than beer strains (<a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Gallone <em>et al.</em> 2016</a>).</p><p id="p-50">Wine yeast strains are thought to be partially domesticated due to the seasonal nature of wine-making, which allows for outcrossing with wild populations (<a id="xref-ref-60-5" class="xref-bibr" href="#ref-60">Marsit and Dequin 2015</a>; <a id="xref-ref-30-6" class="xref-bibr" href="#ref-30">Gallone <em>et al.</em> 2016</a>; <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Gonçalves <em>et al.</em> 2016</a>). One human-driven signature of domestication is thought to be the duplication of the <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus, because multiple copies confer copper resistance and copper sulfates have been used to combat powdery mildews in vineyards since the early 1800s (<a id="xref-ref-97-3" class="xref-bibr" href="#ref-97">Warringer <em>et al.</em> 2011</a>; <a id="xref-ref-60-6" class="xref-bibr" href="#ref-60">Marsit and Dequin 2015</a>). Consistent with this “partial domestication” view (<a id="xref-ref-60-7" class="xref-bibr" href="#ref-60">Marsit and Dequin 2015</a>; <a id="xref-ref-30-7" class="xref-bibr" href="#ref-30">Gallone <em>et al.</em> 2016</a>; <a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Gonçalves <em>et al.</em> 2016</a>), many wine strains were not CN variable for <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1-1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000001097/overview">CUP1-2</a></em>, or had one or both genes deleted (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3</a>).</p><p id="p-51">An alternative, albeit not necessarily conflicting, hypothesis is that wine yeasts underwent domestication for specific but diverse wine flavor profiles (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Hyma <em>et al.</em> 2011</a>). Consistent with this view is the deletion (in &gt;90% of the strains) of the <em><a href="http://www.yeastgenome.org/locus/S000001836/overview">THI5</a></em> gene (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3</a>), whose activity is known to produce an undesirable rotten-egg sensory perception via higher SH<sub>2</sub> production and is associated with sluggish fermentations (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bartra <em>et al.</em> 2010</a>). In contrast to wine strains, duplications of <em><a href="http://www.yeastgenome.org/locus/S000001836/overview">THI5</a></em> have been observed across the <em>Saccharomyces</em> genus, including in several strains of <em>S. cerevisiae</em> (CBS1171, two copies; S288c, four copies; EM93, five copies), <em>S. paradoxus</em> (five copies), and the lager brewing yeast hybrid <em>S. pastorianus</em> (syn. <em>S. carlsbergensis</em>; 2+ copies) (<a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">Wightman and Meacock 2003</a>). In contrast, <em><a href="http://www.yeastgenome.org/locus/S000002403/overview">THI13</a></em>, which is duplicated in 62.1% of strains, shows an increase in its expression of 6- to 100-fold in <em>S. cerevisiae</em> when grown on medium containing low concentrations of thiamine, allowing for the compensation of low thiamine levels (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Li <em>et al.</em> 2010</a>). Low levels of thiamine in wine fermentation have been associated with stuck or slow fermentations (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Ough <em>et al.</em> 1989</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bataillon <em>et al.</em> 1996</a>). Similar to <em><a href="http://www.yeastgenome.org/locus/S000001836/overview">THI5</a></em> deletions, <em><a href="http://www.yeastgenome.org/locus/S000002403/overview">THI13</a></em> duplications may have also been driven by human activity due to the advantageous effect of increased expression within the fermentation environment.</p><p id="p-52">Two other gene families subject to CN variation were the <em>MAL</em> and <em>HXT</em> gene families. The S288c strain that we used as a reference contained two <em>MAL</em> loci (<em><a href="http://www.yeastgenome.org/locus/S000029680/overview">MAL1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000029683/overview">MAL3</a></em>), each containing three genes—a maltose permease (<em>MALx1</em>), a maltase (<em>MALx2</em>), and an MAL <em>trans</em>-activator (<em>MALx3</em>)—and located near the ends of different chromosomes (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Michels <em>et al.</em> 1992</a>). <em><a href="http://www.yeastgenome.org/locus/S000029680/overview">MAL1</a></em> has been observed to be duplicated in beer strains of <em>S. cerevisiae</em> (<a id="xref-ref-30-8" class="xref-bibr" href="#ref-30">Gallone <em>et al.</em> 2016</a>; <a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">Gonçalves <em>et al.</em> 2016</a>), while wine strains primarily lack this locus (<a id="xref-fig-3-11" class="xref-fig" href="#F3">Figure 3</a>; <a id="xref-ref-35-6" class="xref-bibr" href="#ref-35">Gonçalves <em>et al.</em> 2016</a>). In contrast to the deletion of the <em><a href="http://www.yeastgenome.org/locus/S000029680/overview">MAL1</a></em> locus, <em><a href="http://www.yeastgenome.org/locus/S000029683/overview">MAL3</a></em> duplication in wine yeasts (<a id="xref-fig-3-12" class="xref-fig" href="#F3">Figure 3</a>; <a id="xref-ref-35-7" class="xref-bibr" href="#ref-35">Gonçalves <em>et al.</em> 2016</a>) is surprising because maltose is absent from the grape must (<a id="xref-ref-30-9" class="xref-bibr" href="#ref-30">Gallone <em>et al.</em> 2016</a>). However, knockout studies have demonstrated <em><a href="http://www.yeastgenome.org/locus/S000000503/overview">MAL32</a></em> is necessary for growth on turanose, maltotriose, and sucrose (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Brown <em>et al.</em> 2010</a>), which are present in small quantities in wines (<a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Victoria and Carmen 2013</a>). Due to the prominent duplication of <em><a href="http://www.yeastgenome.org/locus/S000029683/overview">MAL3</a></em>, in particular the enzymatic genes <em><a href="http://www.yeastgenome.org/locus/S000000502/overview">MAL31</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000000503/overview">MAL32</a></em>, we speculate that the <em><a href="http://www.yeastgenome.org/locus/S000029683/overview">MAL3</a></em> locus may be utilized to obtain sugars less prevalent in the wine environment, or serve other purposes.</p><p id="p-53">The <em>HXT</em> gene family in the S288c strain that we used as a reference contains 16 <em>HXT</em> paralogs, <em><a href="http://www.yeastgenome.org/locus/S000004071/overview">GAL2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002353/overview">SNF3</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000002297/overview">RGT2</a></em>. The expansion of the HXT gene family is positively correlated with aerobic fermentation in <em>S. paradoxus</em> and <em>S. cerevisiae</em> (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Lin and Li 2011</a>). <em><a href="http://www.yeastgenome.org/locus/S000002751/overview">HXT6</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000002750/overview">HXT7</a></em> are high-affinity glucose transporters expressed at low glucose levels, and repressed at high glucose levels (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Reifenberger <em>et al.</em> 1995</a>). In contrast to the recently described Asia (Sake), Britain (Beer) and Mosaic lineages (<a id="xref-ref-30-10" class="xref-bibr" href="#ref-30">Gallone <em>et al.</em> 2016</a>), we detected duplications in the <em><a href="http://www.yeastgenome.org/locus/S000002751/overview">HXT6</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000002750/overview">HXT7</a></em> genes in wine yeasts (<a id="xref-fig-3-13" class="xref-fig" href="#F3">Figure 3</a>). This may confer an advantage toward the end of fermentation and before the diauxic shift when glucose becomes a scarce resource. Evidence potentially supporting this hypothesis is that <em><a href="http://www.yeastgenome.org/locus/S000002751/overview">HXT6</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000002750/overview">HXT7</a></em> are upregulated by 9.8- and 5.6-fold, respectively, through wine fermentation in the <em>S. cerevisiae</em> strain Vin13 (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Marks <em>et al.</em> 2008</a>). Furthermore, <em><a href="http://www.yeastgenome.org/locus/S000002751/overview">HXT6</a></em> or <em><a href="http://www.yeastgenome.org/locus/S000002750/overview">HXT7</a></em> is found to be duplicated in experimentally evolved populations in glucose-limited environments (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Dunham <em>et al.</em> 2002</a>; <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Gresham <em>et al.</em> 2008</a>; <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Dunn <em>et al.</em> 2012</a>).</p><p id="p-54">In summary, these results, together with recent studies of CN variation in beer yeast strains (<a id="xref-ref-30-11" class="xref-bibr" href="#ref-30">Gallone <em>et al.</em> 2016</a>; <a id="xref-ref-35-8" class="xref-bibr" href="#ref-35">Gonçalves <em>et al.</em> 2016</a>), suggest that this type of variation significantly contributes to the genomic diversity of domesticated yeast strains. Furthermore, as most studies of CN variation, including ours, use reference strains, they are likely conservative in estimating the amount of CN variation present in populations. This caveat notwithstanding, examination of publically available data regarding the functional impact of duplicated or deleted genes (again in the context provided by the reference strain’s genetic background) suggests that CN variation in several, but not all, of the wine yeast genes confer fitness advantages in conditions that resemble the fermentation environment. Our results raise the questions of the extent to which CN variation contributes to fungal, and more generally microbial, domestication as well as whether the importance of CN variants in natural yeast populations, including those of other <em>Saccharomyces</em> yeasts, is on par to their importance in domestication environments.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-55">We thank members of the Rokas laboratory for helpful discussions and advice. This work was conducted in part using the resources of the Advanced Computing Center for Research and Education at Vanderbilt University. J.S. was supported by the Graduate Program in Biological Sciences at Vanderbilt University. This work was supported in part by the National Science Foundation (DEB-1442113 to A.R.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-56">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040105/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-57"><em>Communicating editor: A. M. Dudley</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-02-02"><span class="received-label">Received </span>February 2, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-03-08"><span class="accepted-label">Accepted </span>March 8, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Steenwyk and Rokas</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.1" data-doi="10.1002/em.21840"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arlt</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rajendran</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birkeland</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glover</span>  <span class="cit-name-given-names">T. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Copy number variants are produced in response to low-dose ionizing radiation in cultured cells.</span> <abbr class="cit-jnl-abbrev">Environ. Mol. Mutagen.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnviron.%2BMol.%2BMutagen.%26rft.volume%253D55%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fem.21840%26rft_id%253Dinfo%253Apmid%252F24327335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/em.21840&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24327335&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.2" data-doi="10.1038/nature11837"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Axelsson</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ratnakumar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arendt</span>  <span class="cit-name-given-names">M.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maqbool</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Webster</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The genomic signature of dog domestication reveals adaptation to a starch-rich diet.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">495</span>: <span class="cit-fpage">360</span>–<span class="cit-lpage">364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D495%26rft.spage%253D360%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11837%26rft_id%253Dinfo%253Apmid%252F23354050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11837&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23354050&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316650500041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.3" data-doi="10.1089/cmb.2012.0021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bankevich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nurk</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antipov</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gurevich</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dvorkin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing.</span> <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">455</span>–<span class="cit-lpage">477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DBankevich%26rft.auinit1%253DA.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D455%26rft.epage%253D477%26rft.atitle%253DSPAdes%253A%2Ba%2Bnew%2Bgenome%2Bassembly%2Balgorithm%2Band%2Bits%2Bapplications%2Bto%2Bsingle-cell%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2012.0021%26rft_id%253Dinfo%253Apmid%252F22506599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.2012.0021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22506599&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamb</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barber</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaback</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A function for subtelomeric DNA in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">929</span>–<span class="cit-lpage">934</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBarton%26rft.auinit1%253DA.%2BB.%26rft.volume%253D165%26rft.issue%253D2%26rft.spage%253D929%26rft.epage%253D934%26rft.atitle%253DA%2BFunction%2Bfor%2BSubtelomeric%2BDNA%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Apmid%252F14573499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY1LzIvOTI5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bartra</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casado</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carro</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campamà</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piña</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Differential expression of thiamine biosynthetic genes in yeast strains with high and low production of hydrogen sulfide during wine fermentation.</span> <abbr class="cit-jnl-abbrev">J. Appl. Microbiol.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">272</span>–<span class="cit-lpage">281</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BApplied%2BMicrobiology%2B%2528Print%2529%26rft.stitle%253DJournal%2Bof%2BApplied%2BMicrobiology%2B%2528Print%2529%26rft.aulast%253DBartra%26rft.auinit1%253DE.%26rft.volume%253D109%26rft.issue%253D1%26rft.spage%253D272%26rft.epage%253D281%26rft.atitle%253DDifferential%2Bexpression%2Bof%2Bthiamine%2Bbiosynthetic%2Bgenes%2Bin%2Byeast%2Bstrains%2Bwith%2Bhigh%2Band%2Blow%2Bproduction%2Bof%2Bhydrogen%2Bsulfide%2Bduring%2Bwine%2Bfermentation.%26rft_id%253Dinfo%253Apmid%252F20059614%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20059614&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278674300026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.6" data-doi="10.1016/0922-338X(96)85037-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bataillon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rico</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sablayrolles</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salmon</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barre</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Early thiamin assimilation by yeasts under enological conditions: impact on alcoholic fermentation kinetics.</span> <abbr class="cit-jnl-abbrev">J. Ferment. Bioeng.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">145</span>–<span class="cit-lpage">150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BFerment.%2BBioeng.%26rft.volume%253D82%26rft.spage%253D145%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0922-338X%252896%252985037-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0922-338X(96)85037-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.7" data-doi="10.1101/gr.133967.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bickhart</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schroeder</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alkan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardone</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Copy number variation of individual cattle genomes using next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">778</span>–<span class="cit-lpage">790</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.133967.111%26rft_id%253Dinfo%253Apmid%252F22300768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzQvNzc4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.8" data-doi="10.1111/j.1469-8137.2009.03087.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Birchler</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veitia</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The gene balance hypothesis: implications for gene regulation, quantitative traits and evolution.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">54</span>–<span class="cit-lpage">62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DBirchler%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D186%26rft.issue%253D1%26rft.spage%253D54%26rft.epage%253D62%26rft.atitle%253DThe%2Bgene%2Bbalance%2Bhypothesis%253A%2Bimplications%2Bfor%2Bgene%2Bregulation%252C%2Bquantitative%2Btraits%2Band%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2009.03087.x%26rft_id%253Dinfo%253Apmid%252F19925558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2009.03087.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19925558&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275184700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.9" data-doi="10.1073/pnas.1207726109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Birchler</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veitia</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Gene balance hypothesis: connecting issues of dosage sensitivity across biological disciplines.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">14746</span>–<span class="cit-lpage">14753</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1207726109%26rft_id%253Dinfo%253Apmid%252F22908297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzM3LzE0NzQ2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bisson</span>  <span class="cit-name-given-names">L. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Stuck and sluggish fermentations.</span> <abbr class="cit-jnl-abbrev">Am. J. Enol. Vitic.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">107</span>–<span class="cit-lpage">119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BEnology%2Band%2BViticulture%26rft.stitle%253DAm.%2BJ.%2BEnol.%2BVitic.%26rft.aulast%253DBisson%26rft.auinit1%253DL.%2BF.%26rft.volume%253D50%26rft.issue%253D1%26rft.spage%253D107%26rft.epage%253D119%26rft.atitle%253DStuck%2Band%2BSluggish%2BFermentations%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiYWpldiI7czo1OiJyZXNpZCI7czo4OiI1MC8xLzEwNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.11" data-doi="10.5344/ajev.2012.11083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bisson</span>  <span class="cit-name-given-names">L. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Geographic origin and diversity of wine strains of <em>Saccharomyces</em>.</span> <abbr class="cit-jnl-abbrev">Am. J. Enol. Vitic.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">165</span>–<span class="cit-lpage">176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BEnol.%2BVitic.%26rft_id%253Dinfo%253Adoi%252F10.5344%252Fajev.2012.11083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiYWpldiI7czo1OiJyZXNpZCI7czo4OiI2My8yLzE2NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.12" data-doi="10.1093/bioinformatics/btq635"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boeva</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zinovyev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bleakley</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vert</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janoueix-Lerosey</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Control-free calling of copy number alterations in deep-sequencing data using GC-content normalization.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">268</span>–<span class="cit-lpage">269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq635%26rft_id%253Dinfo%253Apmid%252F21081509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNy8yLzI2OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.13" data-doi="10.1093/bioinformatics/btr670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boeva</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Popova</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bleakley</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiche</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cappo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Control-FREEC: a tool for assessing copy number and allelic content using next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr670%26rft_id%253Dinfo%253Apmid%252F22155870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyOC8zLzQyMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.14" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usadel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMzAvMTUvMjExNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.15" data-doi="10.1111/j.1567-1364.2008.00434.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borneman</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forgan</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pretorius</span>  <span class="cit-name-given-names">I. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chambers</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Comparative genome analysis of a <em>Saccharomyces cerevisiae</em> wine strain.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1185</span>–<span class="cit-lpage">1195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2008.00434.x%26rft_id%253Dinfo%253Apmid%252F18778279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjg6IjgvNy8xMTg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borneman</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forgan</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolouchova</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Whole genome comparison reveals high levels of inbreeding and strain redundancy across the spectrum of commercial wine trains of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">957</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D6%26rft.spage%253D957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brion</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ambroset</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delobel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blondin</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Deciphering regulatory variation of THI genes in alcoholic fermentation indicate an impact of Thi3p on PDC1 expression.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1085</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D1085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.18" data-doi="10.1016/j.cub.2010.04.027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verstrepen</span>  <span class="cit-name-given-names">K. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Rapid expansion and functional divergence of subtelomeric gene families in yeasts.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">895</span>–<span class="cit-lpage">903</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DBrown%26rft.auinit1%253DC.%2BA.%26rft.volume%253D20%26rft.issue%253D10%26rft.spage%253D895%26rft.epage%253D903%26rft.atitle%253DRapid%2Bexpansion%2Band%2Bfunctional%2Bdivergence%2Bof%2Bsubtelomeric%2Bgene%2Bfamilies%2Bin%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2010.04.027%26rft_id%253Dinfo%253Apmid%252F20471265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2010.04.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20471265&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278113300021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.19" data-doi="10.1093/oxfordjournals.molbev.a026009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todd</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenzweig</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Multiple duplications of yeast hexose transport genes in response to selection in a glucose-limited environment.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">931</span>–<span class="cit-lpage">942</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DBrown%26rft.auinit1%253DC.%26rft.volume%253D15%26rft.issue%253D8%26rft.spage%253D931%26rft.epage%253D942%26rft.atitle%253DMultiple%2Bduplications%2Bof%2Byeast%2Bhexose%2Btransport%2Bgenes%2Bin%2Bresponse%2Bto%2Bselection%2Bin%2Ba%2Bglucose-limited%2Benvironment%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a026009%26rft_id%253Dinfo%253Apmid%252F9718721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjE1LzgvOTMxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.20" data-doi="10.1093/bioinformatics/btn615"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carbon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ireland</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mungall</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">AmiGO: online access to ontology and annotation data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">288</span>–<span class="cit-lpage">289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn615%26rft_id%253Dinfo%253Apmid%252F19033274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNS8yLzI4OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.21" data-doi="10.1007/s00239-003-0031-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cavalieri</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGovern</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mortimer</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Polsinelli</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Evidence for <em>S. cerevisiae</em> fermentation in ancient wine.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">S226</span>–<span class="cit-lpage">S232</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BEvol.%26rft.volume%253D57%26rft.spage%253DS226%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-003-0031-2%26rft_id%253Dinfo%253Apmid%252F15008419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-003-0031-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15008419&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186396000023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cromie</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyma</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ludlow</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garmendia-Torres</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic sequence diversity and population structure of <em>Saccharomyces cerevisiae</em> assessed by RAD-seq.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">2163</span>–<span class="cit-lpage">2171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D3%26rft.spage%253D2163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.23" data-doi="10.1159/000184721"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cutler</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kassner</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Copy number variation in the mouse genome: implications for the mouse as a model organism for human disease.</span> <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">297</span>–<span class="cit-lpage">306</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetic%2Band%2Bgenome%2Bresearch%26rft.stitle%253DCytogenet%2BGenome%2BRes%26rft.aulast%253DCutler%26rft.auinit1%253DG.%26rft.volume%253D123%26rft.issue%253D1-4%26rft.spage%253D297%26rft.epage%253D306%26rft.atitle%253DCopy%2Bnumber%2Bvariation%2Bin%2Bthe%2Bmouse%2Bgenome%253A%2Bimplications%2Bfor%2Bthe%2Bmouse%2Bas%2Ba%2Bmodel%2Borganism%2Bfor%2Bhuman%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000184721%26rft_id%253Dinfo%253Apmid%252F19287168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000184721&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19287168&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.24" data-doi="10.1371/journal.pone.0059128"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.-G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">H.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.-P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comparative studies of copy number variation detection methods for next-generation sequencing technologies.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e59128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De59128%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0059128%26rft_id%253Dinfo%253Apmid%252F23527109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0059128&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23527109&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.25" data-doi="10.1073/pnas.242624799"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Badrane</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferea</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Characteristic genome rearrangements in experimental evolution of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">16144</span>–<span class="cit-lpage">16149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.242624799%26rft_id%253Dinfo%253Apmid%252F12446845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTkvMjUvMTYxNDQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.26" data-doi="10.1101/gr.130310.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dunn</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richter</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kvitek</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pugh</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherlock</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Analysis of the <em>Saccharomyces cerevisiae</em> pan-genome reveals a pool of copy number variants distributed in diverse yeast strains from differing industrial environments.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">908</span>–<span class="cit-lpage">924</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.130310.111%26rft_id%253Dinfo%253Apmid%252F22369888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzUvOTA4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.27" data-doi="10.1371/journal.pgen.1003703"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Farrer</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henk</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garner</span>  <span class="cit-name-given-names">T. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balloux</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woodhams</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Chromosomal copy number variation, selection and uneven rates of recombination reveal cryptic genome diversity linked to pathogenicity.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003703</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003703%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003703%26rft_id%253Dinfo%253Apmid%252F23966879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003703&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23966879&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.28" data-doi="10.1073/pnas.0601713103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fidalgo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrales</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ibeas</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jimenez</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Adaptive evolution by mutations in the FLO11 gene.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">11228</span>–<span class="cit-lpage">11233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0601713103%26rft_id%253Dinfo%253Apmid%252F16844788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzMwLzExMjI4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.29" data-doi="10.1101/gr.3677206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Freeman</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perry</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feuk</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Redon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarroll</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Copy number variation: new insights in genome diversity.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">949</span>–<span class="cit-lpage">961</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3677206%26rft_id%253Dinfo%253Apmid%252F16809666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE2LzgvOTQ5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.30" data-doi="10.1016/j.cell.2016.08.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gallone</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steensels</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prahl</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soriaga</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saels</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Domestication and divergence of <em>Saccharomyces cerevisiae</em> beer yeasts.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">1397</span>–<span class="cit-lpage">1410.e16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D166%26rft.spage%253D1397%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.08.020%26rft_id%253Dinfo%253Apmid%252F27610566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.08.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27610566&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.31" data-doi="10.1093/nar/gkh036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Gene Ontology Consortium</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The gene ontology (GO) database and informatics resource.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">D258</span>–<span class="cit-lpage">D261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DGene%2BOntology%2BConsortium%26rft.auinit1%253D%2B%26rft.volume%253D32%26rft.issue%253D90001%26rft.spage%253DD258%26rft.epage%253D261%26rft.atitle%253DThe%2BGene%2BOntology%2B%2528GO%2529%2Bdatabase%2Band%2Binformatics%2Bresource%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh036%26rft_id%253Dinfo%253Apmid%252F14681407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzMi9zdXBwbF8xL0QyNTgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gibbons</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rinker</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The genomics of microbial domestication in the fermented food environment.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BGenet.%2BDev.%26rft.volume%253D35%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.33" data-doi="10.1016/j.cub.2012.05.033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gibbons</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salichos</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slot</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rinker</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGary</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The evolutionary imprint of domestication on genome variation and function of the filamentous fungus <em>Aspergillus oryzae</em>.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1403</span>–<span class="cit-lpage">1409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DGibbons%26rft.auinit1%253DJ.%2BG.%26rft.volume%253D22%26rft.issue%253D15%26rft.spage%253D1403%26rft.epage%253D1409%26rft.atitle%253DThe%2Bevolutionary%2Bimprint%2Bof%2Bdomestication%2Bon%2Bgenome%2Bvariation%2Band%2Bfunction%2Bof%2Bthe%2Bfilamentous%2Bfungus%2BAspergillus%2Boryzae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2012.05.033%26rft_id%253Dinfo%253Apmid%252F22795693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2012.05.033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22795693&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.34" data-doi="10.1073/pnas.1416878112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gibbons</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Branco</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Godinho</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemos</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Concerted copy number variation balances ribosomal DNA dosage in human and mouse genomes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">2485</span>–<span class="cit-lpage">2490</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1416878112%26rft_id%253Dinfo%253Apmid%252F25583482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTEyLzgvMjQ4NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonçalves</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pontes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Almeida</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbosa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serra</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Distinct domestication trajectories in top-fermenting beer yeasts and wine yeasts.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2750</span>–<span class="cit-lpage">2761</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D26%26rft.spage%253D2750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.36" data-doi="10.1128/AEM.00394-08"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Govender</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Domingo</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bester</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pretorius</span>  <span class="cit-name-given-names">I. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">F. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Controlled expression of the dominant flocculation genes FLO1, FLO5, and FLO11 in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">6041</span>–<span class="cit-lpage">6052</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.00394-08%26rft_id%253Dinfo%253Apmid%252F18708514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI3NC8xOS82MDQxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.37" data-doi="10.1371/journal.pgen.1000303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gresham</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desai</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenq</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pai</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The repertoire and dynamics of evolutionary adaptations to controlled nutrient-limited environments in yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGresham%26rft.auinit1%253DD.%26rft.volume%253D4%26rft.issue%253D12%26rft.spage%253De1000303%26rft.epage%253De1000303%26rft.atitle%253DThe%2Brepertoire%2Band%2Bdynamics%2Bof%2Bevolutionary%2Badaptations%2Bto%2Bcontrolled%2Bnutrient-limited%2Benvironments%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000303%26rft_id%253Dinfo%253Apmid%252F19079573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000303&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19079573&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.38" data-doi="10.1093/hmg/ddp011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henrichsen</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaignat</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reymond</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Copy number variants, diseases and gene expression.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">R1</span>–<span class="cit-lpage">R8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddp011%26rft_id%253Dinfo%253Apmid%252F19297395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaG1nIjtzOjU6InJlc2lkIjtzOjg6IjE4L1IxL1IxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.39" data-doi="10.1126/science.1150021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hillenmeyer</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fung</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wildenhain</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pierce</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The chemical genomic portrait of yeast: uncovering a phenotype for all genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">320</span>: <span class="cit-fpage">362</span>–<span class="cit-lpage">365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHillenmeyer%26rft.auinit1%253DM.%2BE.%26rft.volume%253D320%26rft.issue%253D5874%26rft.spage%253D362%26rft.epage%253D365%26rft.atitle%253DThe%2BChemical%2BGenomic%2BPortrait%2Bof%2BYeast%253A%2BUncovering%2Ba%2BPhenotype%2Bfor%2BAll%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1150021%26rft_id%253Dinfo%253Apmid%252F18420932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjAvNTg3NC8zNjIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.40" data-doi="10.1186/1471-2164-12-526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Variation in chromosome copy number influences the virulence of <em>Cryptococcus</em> <em>neoformans</em> and occurs in isolates from AIDS patients.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">526</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DHu%26rft.auinit1%253DG.%26rft.volume%253D12%26rft.spage%253D526%26rft.epage%253D526%26rft.atitle%253DVariation%2Bin%2Bchromosome%2Bcopy%2Bnumber%2Binfluences%2Bthe%2Bvirulence%2Bof%2BCryptococcus%2Bneoformans%2Band%2Boccurs%2Bin%2Bisolates%2Bfrom%2BAIDS%2Bpatients.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-526%26rft_id%253Dinfo%253Apmid%252F22032296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-526&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22032296&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.41" data-doi="10.1016/j.fgb.2011.01.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Independent duplications of alpha-amylase in different strains of <em>Aspergillus oryzae</em>.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">438</span>–<span class="cit-lpage">444</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DHunter%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D48%26rft.issue%253D4%26rft.spage%253D438%26rft.epage%253D444%26rft.atitle%253DIndependent%2Bduplications%2Bof%2B%25CE%25B1-amylase%2Bin%2Bdifferent%2Bstrains%2Bof%2BAspergillus%2Boryzae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2011.01.006%26rft_id%253Dinfo%253Apmid%252F21277379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2011.01.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21277379&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.42" data-doi="10.1093/bioinformatics/14.1.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huson</span>  <span class="cit-name-given-names">D. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">SplitsTree: analyzing and visualizing evolutionary data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">68</span>–<span class="cit-lpage">73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DHuson%26rft.auinit1%253DD.%2BH.%26rft.volume%253D14%26rft.issue%253D1%26rft.spage%253D68%26rft.epage%253D73%26rft.atitle%253DSplitsTree%253A%2Banalyzing%2Band%2Bvisualizing%2Bevolutionary%2Bdata%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F14.1.68%26rft_id%253Dinfo%253Apmid%252F9520503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo3OiIxNC8xLzY4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.43" data-doi="10.1111/j.1567-1364.2011.00746.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hyma</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saerens</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verstrepen</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Divergence in wine characteristics produced by wild and domesticated strains of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">540</span>–<span class="cit-lpage">551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2011.00746.x%26rft_id%253Dinfo%253Apmid%252F22093681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjg6IjExLzcvNTQwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.44" data-doi="10.1038/nrg2487"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaessmann</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vinckenbosch</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">RNA-based gene duplication: mechanistic and evolutionary insights.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DKaessmann%26rft.auinit1%253DH.%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253D19%26rft.epage%253D31%26rft.atitle%253DRNA-based%2Bgene%2Bduplication%253A%2Bmechanistic%2Band%2Bevolutionary%2Binsights.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2487%26rft_id%253Dinfo%253Apmid%252F19030023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2487&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19030023&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261866500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.45" data-doi="10.1038/ng.280"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherlock</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Molecular characterization of clonal interference during adaptive evolution in asexual populations of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1499</span>–<span class="cit-lpage">1504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKao%26rft.auinit1%253DK.%2BC.%26rft.volume%253D40%26rft.issue%253D12%26rft.spage%253D1499%26rft.epage%253D1504%26rft.atitle%253DMolecular%2Bcharacterization%2Bof%2Bclonal%2Binterference%2Bduring%2Badaptive%2Bevolution%2Bin%2Basexual%2Bpopulations%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.280%26rft_id%253Dinfo%253Apmid%252F19029899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.280&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19029899&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261215900030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.46" data-doi="10.1073/pnas.87.15.5754"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keith</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reeders</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">V. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phipps</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Genetic linkage map of 46 DNA markers on human chromosome 16.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">5754</span>–<span class="cit-lpage">5758</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DKeith%26rft.auinit1%253DT.%2BP.%26rft.volume%253D87%26rft.issue%253D15%26rft.spage%253D5754%26rft.epage%253D5758%26rft.atitle%253DGenetic%2Blinkage%2Bmap%2Bof%2B46%2BDNA%2Bmarkers%2Bon%2Bhuman%2Bchromosome%2B16.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.87.15.5754%26rft_id%253Dinfo%253Apmid%252F2377614%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODcvMTUvNTc1NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.47" data-doi="10.1093/bioinformatics/btp373"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koboldt</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wylie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLellan</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">VarScan: variant detection in massively parallel sequencing of individual and pooled samples.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2283</span>–<span class="cit-lpage">2285</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp373%26rft_id%253Dinfo%253Apmid%252F19542151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTcvMjI4MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.48" data-doi="10.1101/gr.129684.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koboldt</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLellan</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">568</span>–<span class="cit-lpage">576</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.129684.111%26rft_id%253Dinfo%253Apmid%252F22300766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzMvNTY4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-web" id="cit-7.5.1475.49"><div class="cit-metadata unstructured">Kolde, R., 2012 Package ‘pheatmap’. Bioconductor 1–6. Available at: <a href="https://CRAN.R-project.org/package=pheatmap">https://CRAN.R-project.org/package=pheatmap</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.50" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.51" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.52" data-doi="10.1128/MCB.01590-09"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petteys</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McClure</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valsakumar</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bekiranov</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Thiamine biosynthesis in <em>Saccharomyces cerevisiae</em> is regulated by the NAD+-dependent histone deacetylase Hst1.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">3329</span>–<span class="cit-lpage">3341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.01590-09%26rft_id%253Dinfo%253Apmid%252F20439498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIzMC8xMy8zMzI5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.53" data-doi="10.1093/molbev/msq184"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Expansion of hexose transporter genes was associated with the evolution of aerobic fermentation in yeasts.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">142</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq184%26rft_id%253Dinfo%253Apmid%252F20660490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI4LzEvMTMxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.54" data-doi="10.1038/nature07743"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moses</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Population genomics of domestic and wild yeasts.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLiti%26rft.auinit1%253DG.%26rft.volume%253D458%26rft.issue%253D7236%26rft.spage%253D337%26rft.epage%253D341%26rft.atitle%253DPopulation%2Bgenomics%2Bof%2Bdomestic%2Band%2Bwild%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07743%26rft_id%253Dinfo%253Apmid%252F19212322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07743&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212322&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264285600041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.55" data-doi="10.1128/jb.178.24.7144-7151.1996"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lo</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dranginis</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">FLO11, a yeast gene related to the STA genes, encodes a novel cell surface flocculin.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">7144</span>–<span class="cit-lpage">7151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DLo%26rft.auinit1%253DW.%2BS.%26rft.volume%253D178%26rft.issue%253D24%26rft.spage%253D7144%26rft.epage%253D7151%26rft.atitle%253DFLO11%252C%2Ba%2Byeast%2Bgene%2Brelated%2Bto%2Bthe%2BSTA%2Bgenes%252C%2Bencodes%2Ba%2Bnovel%2Bcell%2Bsurface%2Bflocculin.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.178.24.7144-7151.1996%26rft_id%253Dinfo%253Apmid%252F8955395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE3OC8yNC83MTQ0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.56" data-doi="10.1016/S1471-4914(02)02280-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lupetti</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danesi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Del Tacca</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Molecular basis of resistance to azole antifungals.</span> <abbr class="cit-jnl-abbrev">Trends Mol. Med.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">76</span>–<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bmolecular%2Bmedicine%26rft.stitle%253DTrends%2BMol%2BMed%26rft.aulast%253DLupetti%26rft.auinit1%253DA.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D76%26rft.epage%253D81%26rft.atitle%253DMolecular%2Bbasis%2Bof%2Bresistance%2Bto%2Bazole%2Bantifungals.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1471-4914%252802%252902280-3%26rft_id%253Dinfo%253Apmid%252F11815273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1471-4914(02)02280-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11815273&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173769700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.57" data-doi="10.1371/journal.pgen.0010049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lupski</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stankiewicz</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genomic disorders: molecular mechanisms for rearrangements and conveyed phenotypes.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">e49</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLupski%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D1%26rft.issue%253D6%26rft.spage%253De49%26rft.epage%253De49%26rft.atitle%253DGenomic%2Bdisorders%253A%2Bmolecular%2Bmechanisms%2Bfor%2Brearrangements%2Band%2Bconveyed%2Bphenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0010049%26rft_id%253Dinfo%253Apmid%252F16444292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0010049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16444292&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-book" id="cit-7.5.1475.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Madden</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The BLAST sequence analysis tool,</span> pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">17</span> in <span class="cit-source"><em>The NCBI Handbook</em></span>. <span class="cit-edition">Ed. 2</span>. <span class="cit-publ-name">National Center for Biotechnology Information</span>, <span class="cit-publ-loc">Bethesda, MD</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.59" data-doi="10.1111/j.1567-1364.2007.00338.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marks</span>  <span class="cit-name-given-names">V. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ho Sui</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erasmus</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Der Merwe</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brumm</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Dynamics of the yeast transcriptome during wine fermentation reveals a novel fermentation stress response.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">35</span>–<span class="cit-lpage">52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2007.00338.x%26rft_id%253Dinfo%253Apmid%252F18215224%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjY6IjgvMS8zNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marsit</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dequin</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Diversity and adaptive evolution of <em>Saccharomyces</em> wine yeast: a review.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">15</span>: fov067.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.61" data-doi="10.1038/nprot.2013.092"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muruganujan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casagrande</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Large-scale gene function analysis with the PANTHER classification system.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1551</span>–<span class="cit-lpage">1566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BProtoc.%26rft.volume%253D8%26rft.spage%253D1551%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2013.092%26rft_id%253Dinfo%253Apmid%252F23868073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2013.092&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23868073&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.62" data-doi="10.1002/yea.320080809"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michels</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Read</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nat</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charron</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">The telomere-associated MAL3 locus of <em>Saccharomyces</em> is a tandem array of repeated genes.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">655</span>–<span class="cit-lpage">665</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DMichels%26rft.auinit1%253DC.%2BA.%26rft.volume%253D8%26rft.issue%253D8%26rft.spage%253D655%26rft.epage%253D665%26rft.atitle%253DThe%2Btelomere-associated%2BMAL3%2Blocus%2Bof%2BSaccharomyces%2Bis%2Ba%2Btandem%2Barray%2Bof%2Brepeated%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.320080809%26rft_id%253Dinfo%253Apmid%252F1441745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.320080809&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1441745&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JJ99100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Molina</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swiegers</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varela</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pretorius</span>  <span class="cit-name-given-names">I. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agosin</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Influence of wine fermentation temperature on the synthesis of yeast-derived volatile aroma compounds.</span> <abbr class="cit-jnl-abbrev">Appl. Microbiol. Biotechnol.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">675</span>–<span class="cit-lpage">687</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2BMicrobiology%2Band%2BBiotechnology%26rft.stitle%253DApplied%2BMicrobiology%2Band%2BBiotechnology%26rft.aulast%253DMolina%26rft.auinit1%253DA.%2BM.%26rft.volume%253D77%26rft.issue%253D3%26rft.spage%253D675%26rft.epage%253D687%26rft.atitle%253DInfluence%2Bof%2Bwine%2Bfermentation%2Btemperature%2Bon%2Bthe%2Bsynthesis%2Bof%2Byeast-derived%2Bvolatile%2Baroma%2Bcompounds.%26rft_id%253Dinfo%253Apmid%252F17938912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17938912&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250878500020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.64" data-doi="10.1101/gr.10.4.403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mortimer</span>  <span class="cit-name-given-names">R. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Evolution and variation of the yeast (<em>Saccharomyces</em>) genome.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DMortimer%26rft.auinit1%253DR.%2BK.%26rft.volume%253D10%26rft.issue%253D4%26rft.spage%253D403%26rft.epage%253D409%26rft.atitle%253DEvolution%2Band%2BVariation%2Bof%2Bthe%2BYeast%2B%2528Saccharomyces%2529%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.10.4.403%26rft_id%253Dinfo%253Apmid%252F10779481%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEwLzQvNDAzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.65" data-doi="10.1073/pnas.0904673106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Novo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bigey</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beyne</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galeote</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gavory</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Eukaryote-to-eukaryote gene transfer events revealed by the genome sequence of the wine yeast <em>Saccharomyces cerevisiae</em> EC1118.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">16333</span>–<span class="cit-lpage">16338</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0904673106%26rft_id%253Dinfo%253Apmid%252F19805302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzM4LzE2MzMzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.66" data-doi="10.1093/hmg/ddp360"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orozco</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cokus</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghazalpour</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ingram-Drake</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Copy number variation influences gene expression and metabolic traits in mice.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">4118</span>–<span class="cit-lpage">4129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddp360%26rft_id%253Dinfo%253Apmid%252F19648292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaG1nIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8yMS80MTE4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.67" data-doi="10.1093/jac/48.1.75"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Osherov</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kontoyiannis</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romans</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Resistance to itraconazole in <em>Aspergillus nidulans</em> and <em>Aspergillus fumigatus</em> is conferred by extra copies of the <em>A. nidulans</em> P-450 14alpha-demethylase gene, pdmA.</span> <abbr class="cit-jnl-abbrev">J. Antimicrob. Chemother.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">75</span>–<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAntimicrobial%2BChemotherapy%26rft.stitle%253DJ%2BAntimicrob%2BChemother%26rft.aulast%253DOsherov%26rft.auinit1%253DN.%26rft.volume%253D48%26rft.issue%253D1%26rft.spage%253D75%26rft.epage%253D81%26rft.atitle%253DResistance%2Bto%2Bitraconazole%2Bin%2BAspergillus%2Bnidulans%2Band%2BAspergillus%2Bfumigatus%2Bis%2Bconferred%2Bby%2Bextra%2Bcopies%2Bof%2Bthe%2BA.%2Bnidulans%2BP-450%2B14%257B%257Balpha%257D%257D-demethylase%2Bgene%252C%2BpdmA%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjac%252F48.1.75%26rft_id%253Dinfo%253Apmid%252F11418514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamFjIjtzOjU6InJlc2lkIjtzOjc6IjQ4LzEvNzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ough</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davenport</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joseph</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Effects of certain vitamins on growth and fermentation rate of several commercial active dry wine yeasts.</span> <abbr class="cit-jnl-abbrev">Am. J. Enol. Vitic.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">208</span>–<span class="cit-lpage">213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BEnology%2Band%2BViticulture%26rft.stitle%253DAm.%2BJ.%2BEnol.%2BVitic.%26rft.aulast%253DOugh%26rft.auinit1%253DC.%2BS.%26rft.volume%253D40%26rft.issue%253D3%26rft.spage%253D208%26rft.epage%253D213%26rft.atitle%253DEffects%2Bof%2BCertain%2BVitamins%2Bon%2BGrowth%2Band%2BFermentation%2BRate%2Bof%2BSeveral%2BCommercial%2BActive%2BDry%2BWine%2BYeasts%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiYWpldiI7czo1OiJyZXNpZCI7czo4OiI0MC8zLzIwOCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Padilla</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuge</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crawford</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Errett</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Werner-Washburne</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The highly conserved, coregulated SNO and SNZ gene families in <em>Saccharomyces cerevisiae</em> respond to nutrient limitation.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">5718</span>–<span class="cit-lpage">5726</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DPadilla%26rft.auinit1%253DP.%2BA.%26rft.volume%253D180%26rft.issue%253D21%26rft.spage%253D5718%26rft.epage%253D5726%26rft.atitle%253DThe%2BHighly%2BConserved%252C%2BCoregulated%2BSNO%2Band%2BSNZ%2BGene%2BFamilies%2Bin%2BSaccharomyces%2Bcerevisiae%2BRespond%2Bto%2BNutrient%2BLimitation%26rft_id%253Dinfo%253Apmid%252F9791124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE4MC8yMS81NzE4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Payen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sunshine</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ong</span>  <span class="cit-name-given-names">G. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pogachar</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">High-throughput identification of adaptive mutations in experimentally evolved yeast populations.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1006339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1006339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.71" data-doi="10.1038/ng2123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perry</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dominy</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Claw</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fiegler</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Diet and the evolution of human amylase gene copy number variation.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1256</span>–<span class="cit-lpage">1260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DPerry%26rft.auinit1%253DG.%2BH.%26rft.volume%253D39%26rft.issue%253D10%26rft.spage%253D1256%26rft.epage%253D1260%26rft.atitle%253DDiet%2Band%2Bthe%2Bevolution%2Bof%2Bhuman%2Bamylase%2Bgene%2Bcopy%2Bnumber%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2123%26rft_id%253Dinfo%253Apmid%252F17828263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17828263&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249737400022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.72" data-doi="10.1101/gr.187187.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pezer</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harr</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teschke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babiker</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tautz</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Divergence patterns of genic copy number variation in natural populations of the house mouse (<em>Mus musculus domesticus</em>) reveal three conserved genes with major population-specific expansions.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1114</span>–<span class="cit-lpage">1124</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.187187.114%26rft_id%253Dinfo%253Apmid%252F26149421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI1LzgvMTExNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.73" data-doi="10.1002/1097-0061(20000615)16:8&lt;675::AID-YEA585&gt;3.0.CO;2-B"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pretorius</span>  <span class="cit-name-given-names">I. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Tailoring wine yeast for the new millennium: novel approaches to the ancient art of winemaking.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">675</span>–<span class="cit-lpage">729</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DPretorius%26rft.auinit1%253DI.%2BS.%26rft.volume%253D16%26rft.issue%253D8%26rft.spage%253D675%26rft.epage%253D729%26rft.atitle%253DTailoring%2Bwine%2Byeast%2Bfor%2Bthe%2Bnew%2Bmillennium%253A%2Bnovel%2Bapproaches%2Bto%2Bthe%2Bancient%2Bart%2Bof%2Bwinemaking.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F1097-0061%252820000615%252916%253A8%253C675%253A%253AAID-YEA585%253E3.0.CO%253B2-B%26rft_id%253Dinfo%253Apmid%252F10861899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/1097-0061(20000615)16:8&lt;675::AID-YEA585&gt;3.0.CO;2-B&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10861899&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087677600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.74" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi82Lzg0MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-book" id="cit-7.5.1475.75"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Development Core Team</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-source"><em>R: A Language and Environment for Statistical Computing</em>.</span> <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.76" data-doi="10.1128/EC.00068-09"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramsook</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fung</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soybelman</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Yeast cell adhesion molecules have functional amyloid-forming sequences.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">393</span>–<span class="cit-lpage">404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00068-09%26rft_id%253Dinfo%253Apmid%252F20038605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiI5LzMvMzkzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.77" data-doi="10.1111/j.1365-2958.1995.tb02400.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reifenberger</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freidel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ciriacy</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Identification of novel HXT genes in <em>Saccharomyces cerevisiae</em> reveals the impact of individual hexose transporters on glycolytic flux.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">157</span>–<span class="cit-lpage">167</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DReifenberger%26rft.auinit1%253DE.%26rft.volume%253D16%26rft.issue%253D1%26rft.spage%253D157%26rft.epage%253D167%26rft.atitle%253DIdentification%2Bof%2Bnovel%2BHXT%2Bgenes%2Bin%2BSaccharomyces%2Bcerevisiae%2Breveals%2Bthe%2Bimpact%2Bof%2Bindividual%2Bhexose%2Btransporters%2Bon%2Bglycolytic%2Bflux.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.1995.tb02400.x%26rft_id%253Dinfo%253Apmid%252F7651133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.1995.tb02400.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7651133&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QU56400015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.78"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Risch</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Linkage strategies for genetically complex traits. II. The power of affected relative pairs.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">229</span>–<span class="cit-lpage">241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DRisch%26rft.auinit1%253DN.%26rft.volume%253D46%26rft.issue%253D2%26rft.spage%253D229%26rft.epage%253D241%26rft.atitle%253DLinkage%2Bstrategies%2Bfor%2Bgenetically%2Bcomplex%2Btraits.%2BII.%2BThe%2Bpower%2Bof%2Baffected%2Brelative%2Bpairs.%26rft_id%253Dinfo%253Apmid%252F2301393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2301393&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990CM93800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.79" data-doi="10.1002/yea.916"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rodríguez-Navarro</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Llorente</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodríguez-Manzaneque</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramne</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uber</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Functional analysis of yeast gene families involved in metabolism of vitamins B1 and B6.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1261</span>–<span class="cit-lpage">1276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DRodr%25C3%25ADguez-Navarro%26rft.auinit1%253DS.%26rft.volume%253D19%26rft.issue%253D14%26rft.spage%253D1261%26rft.epage%253D1276%26rft.atitle%253DFunctional%2Banalysis%2Bof%2Byeast%2Bgene%2Bfamilies%2Binvolved%2Bin%2Bmetabolism%2Bof%2Bvitamins%2BB1%2Band%2BB6.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.916%26rft_id%253Dinfo%253Apmid%252F12271461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.916&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12271461&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178623000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.80" data-doi="10.1128/IAI.74.2.1352-1359.2006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanguinetti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Posteraro</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">La Sorda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torelli</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fiori</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Role of <em>AFR1</em>, an ABC transporter-encoding gene, in the in vivo response to fluconazole and virulence of <em>Cryptococcus neoformans</em>.</span> <abbr class="cit-jnl-abbrev">Infect. Immun.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">1352</span>–<span class="cit-lpage">1359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%2Band%2BImmunity%26rft.stitle%253DInfect.%2BImmun.%26rft.aulast%253DSanguinetti%26rft.auinit1%253DM.%26rft.volume%253D74%26rft.issue%253D2%26rft.spage%253D1352%26rft.epage%253D1359%26rft.atitle%253DRole%2Bof%2BAFR1%252C%2Ban%2BABC%2BTransporter-Encoding%2BGene%252C%2Bin%2Bthe%2BIn%2BVivo%2BResponse%2Bto%2BFluconazole%2Band%2BVirulence%2Bof%2BCryptococcus%2Bneoformans%26rft_id%253Dinfo%253Adoi%252F10.1128%252FIAI.74.2.1352-1359.2006%26rft_id%253Dinfo%253Apmid%252F16428784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaWFpIjtzOjU6InJlc2lkIjtzOjk6Ijc0LzIvMTM1MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.81" data-doi="10.1038/nature07670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schacherer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruderfer</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Comprehensive polymorphism survey elucidates population structure of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>: <span class="cit-fpage">342</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSchacherer%26rft.auinit1%253DJ.%26rft.volume%253D458%26rft.issue%253D7236%26rft.spage%253D342%26rft.epage%253D345%26rft.atitle%253DComprehensive%2Bpolymorphism%2Bsurvey%2Belucidates%2Bpopulation%2Bstructure%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07670%26rft_id%253Dinfo%253Apmid%252F19212320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212320&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264285600042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.82"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ryu</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Construction of killer wine yeast strain.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1211</span>–<span class="cit-lpage">1215</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft.stitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft.aulast%253DSeki%26rft.auinit1%253DT.%26rft.volume%253D49%26rft.issue%253D5%26rft.spage%253D1211%26rft.epage%253D1215%26rft.atitle%253DConstruction%2Bof%2BKiller%2BWine%2BYeast%2BStrain%26rft_id%253Dinfo%253Apmid%252F16346794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjk6IjQ5LzUvMTIxMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.83" data-doi="10.1016/j.crvi.2010.12.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sicard</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legras</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Bread, beer and wine: yeast domestication in the <em>Saccharomyces</em> sensu stricto complex.</span> <abbr class="cit-jnl-abbrev">C. R. Biol.</abbr> <span class="cit-vol">334</span>: <span class="cit-fpage">229</span>–<span class="cit-lpage">236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComptes%2Brendus%2Bbiologies%26rft.stitle%253DC%2BR%2BBiol%26rft.aulast%253DSicard%26rft.auinit1%253DD.%26rft.volume%253D334%26rft.issue%253D3%26rft.spage%253D229%26rft.epage%253D236%26rft.atitle%253DBread%252C%2Bbeer%2Band%2Bwine%253A%2Byeast%2Bdomestication%2Bin%2Bthe%2BSaccharomyces%2Bsensu%2Bstricto%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.crvi.2010.12.016%26rft_id%253Dinfo%253Apmid%252F21377618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.crvi.2010.12.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21377618&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.84" data-doi="10.1038/nrg3642"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sims</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sudbery</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ilott</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ponting</span>  <span class="cit-name-given-names">C. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Sequencing depth and coverage: key considerations in genomic analyses.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">121</span>–<span class="cit-lpage">132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D121%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3642%26rft_id%253Dinfo%253Apmid%252F24434847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3642&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24434847&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.85" data-doi="10.1371/journal.ppat.1000848"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sionov</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwon-Chung</span>  <span class="cit-name-given-names">K. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title"><em>Cryptococcus neoformans</em> overcomes stress of azole drugs by formation of disomy in specific multiple chromosomes.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000848</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DSionov%26rft.auinit1%253DE.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253De1000848%26rft.epage%253De1000848%26rft.atitle%253DCryptococcus%2Bneoformans%2Bovercomes%2Bstress%2Bof%2Bazole%2Bdrugs%2Bby%2Bformation%2Bof%2Bdisomy%2Bin%2Bspecific%2Bmultiple%2Bchromosomes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1000848%26rft_id%253Dinfo%253Apmid%252F20368972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1000848&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20368972&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.86" data-doi="10.1101/gr.155762.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Skelly</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merrihew</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riffle</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connelly</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerr</span>  <span class="cit-name-given-names">E. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Integrative phenomics reveals insight into the structure of phenotypic diversity in budding yeast.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1496</span>–<span class="cit-lpage">1504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.155762.113%26rft_id%253Dinfo%253Apmid%252F23720455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIzLzkvMTQ5NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.87" data-doi="10.1007/s001220050544"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J. S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">E. C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">O. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">An evaluation of the utility of SSR loci as molecular markers in maize (<em>Zea mays</em> L.): comparisons with data from RFLPS and pedigree.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">173</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D95%26rft.spage%253D163%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050544%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050544&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997XR13100019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.88" data-doi="10.1016/S1389-1723(00)88742-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stambuk</span>  <span class="cit-name-given-names">B. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batista</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Araujo</span>  <span class="cit-name-given-names">P. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Kinetics of active sucrose transport in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">J. Biosci. Bioeng.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">212</span>–<span class="cit-lpage">214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BBiosci%2BBioeng%26rft.aulast%253DStambuk%26rft.auinit1%253DB.%2BU.%26rft.volume%253D89%26rft.issue%253D2%26rft.spage%253D212%26rft.epage%253D214%26rft.atitle%253DKinetics%2Bof%2Bactive%2Bsucrose%2Btransport%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1389-1723%252800%252988742-3%26rft_id%253Dinfo%253Apmid%252F16232731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1389-1723(00)88742-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16232731&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.89"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steenwyk</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soghigian</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perfect</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibbons</span>  <span class="cit-name-given-names">J. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Copy number variation contributes to cryptic genetic variation in outbreak lineages of <em>Cryptococcus gattii</em> from the North American Pacific Northwest.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">700</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.90" data-doi="10.1126/science.1197005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sudmant</span>  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kitzman</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antonacci</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alkan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malig</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Diversity of human copy number variation and multicopy genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">330</span>: <span class="cit-fpage">641</span>–<span class="cit-lpage">646</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSudmant%26rft.auinit1%253DP.%2BH.%26rft.volume%253D330%26rft.issue%253D6004%26rft.spage%253D641%26rft.epage%253D646%26rft.atitle%253DDiversity%2Bof%2BHuman%2BCopy%2BNumber%2BVariation%2Band%2BMulticopy%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1197005%26rft_id%253Dinfo%253Apmid%252F21030649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzAvNjAwNC82NDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.91" data-doi="10.1126/science.aab3761"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sudmant</span>  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mallick</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hormozdiari</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krumm</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Global diversity, population stratification, and selection of human copy-number variation.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">349</span>: <span class="cit-fpage">aab3761</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aab3761%26rft_id%253Dinfo%253Apmid%252F26249230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNDkvNjI1My9hYWIzNzYxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.92" data-doi="10.1074/jbc.M110.145946"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teste</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marie François</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parrou</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Characterization of a new multigene family encoding isomaltases in the yeast <em>Saccharomyces cerevisiae</em>, the <em>IMA</em> family.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">285</span>: <span class="cit-fpage">26815</span>–<span class="cit-lpage">26824</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M110.145946%26rft_id%253Dinfo%253Apmid%252F20562106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODUvMzUvMjY4MTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.93" data-doi="10.1101/gr.772403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kejariwal</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karlak</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">PANTHER: a library of protein families and subfamilies indexed by function.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2129</span>–<span class="cit-lpage">2141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DThomas%26rft.auinit1%253DP.%2BD.%26rft.volume%253D13%26rft.issue%253D9%26rft.spage%253D2129%26rft.epage%253D2141%26rft.atitle%253DPANTHER%253A%2BA%2BLibrary%2Bof%2BProtein%2BFamilies%2Band%2BSubfamilies%2BIndexed%2Bby%2BFunction%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.772403%26rft_id%253Dinfo%253Apmid%252F12952881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjEzLzkvMjEyOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.94" data-doi="10.1021/ci400128m"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Victoria</span>  <span class="cit-name-given-names">M.M.-A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carmen</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Wine chemistry and biochemistry.</span> <abbr class="cit-jnl-abbrev">J. Chem. Inf. Model.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">1689</span>–<span class="cit-lpage">1699</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BChem.%2BInf.%2BModel.%26rft.volume%253D53%26rft.spage%253D1689%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fci400128m%26rft_id%253Dinfo%253Apmid%252F23800267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ci400128m&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23800267&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.95" data-doi="10.1007/s00253-008-1616-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vilela-Moura</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schuller</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mendes-Faia</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Côrte-Real</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Reduction of volatile acidity of wines by selected yeast strains.</span> <abbr class="cit-jnl-abbrev">Appl. Microbiol. Biotechnol.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">881</span>–<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2BMicrobiology%2Band%2BBiotechnology%26rft.stitle%253DApplied%2BMicrobiology%2Band%2BBiotechnology%26rft.aulast%253DVilela-Moura%26rft.auinit1%253DA.%26rft.volume%253D80%26rft.issue%253D5%26rft.spage%253D881%26rft.epage%253D890%26rft.atitle%253DReduction%2Bof%2Bvolatile%2Bacidity%2Bof%2Bwines%2Bby%2Bselected%2Byeast%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00253-008-1616-x%26rft_id%253Dinfo%253Apmid%252F18677471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00253-008-1616-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18677471&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259270900015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.96"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wallace</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The Mann-Whitney test.</span> <abbr class="cit-jnl-abbrev">J. Am. Soc. Inf.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BSoc.%2BInf.%26rft.volume%253D55%26rft.spage%253D93%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference  in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.97" data-doi="10.1371/journal.pgen.1002111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zörgö</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zia</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gjuvsland</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Trait variation in yeast is defined by population history.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWarringer%26rft.auinit1%253DJ.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253De1002111%26rft.epage%253De1002111%26rft.atitle%253DTrait%2Bvariation%2Bin%2Byeast%2Bis%2Bdefined%2Bby%2Bpopulation%2Bhistory.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002111%26rft_id%253Dinfo%253Apmid%252F21698134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21698134&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference  in text" id="ref-98">↵</a><div class="cit ref-cit ref-web" id="cit-7.5.1475.98"><div class="cit-metadata unstructured">Wickham, H., 2009 <em>ggplot2</em>. Available at: <a href="http://ggplot2.org">http://ggplot2.org</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference  in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.99" data-doi="10.1099/mic.0.26194-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wightman</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meacock</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The THI5 gene family of <em>Saccharomyces cerevisiae</em>: distribution of homologues among the hemiascomycetes and functional redundancy in the aerobic biosynthesis of thiamin from pyridoxine.</span> <abbr class="cit-jnl-abbrev">Microbiology</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">1447</span>–<span class="cit-lpage">1460</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft.stitle%253DMicrobiology%26rft.aulast%253DWightman%26rft.auinit1%253DR.%26rft.volume%253D149%26rft.issue%253D6%26rft.spage%253D1447%26rft.epage%253D1460%26rft.atitle%253DThe%2BTHI5%2Bgene%2Bfamily%2Bof%2BSaccharomyces%2Bcerevisiae%253A%2Bdistribution%2Bof%2Bhomologues%2Bamong%2Bthe%2Bhemiascomycetes%2Band%2Bfunctional%2Bredundancy%2Bin%2Bthe%2Baerobic%2Bbiosynthesis%2Bof%2Bthiamin%2Bfrom%2Bpyridoxine%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fmic.0.26194-0%26rft_id%253Dinfo%253Apmid%252F12777485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/mic.0.26194-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12777485&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183621600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference  in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.100" data-doi="10.1126/science.285.5429.901"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winzeler</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Astromoff</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Functional characterization of the <em>S. cerevisiae</em> genome by gene deletion and parallel analysis.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">285</span>: <span class="cit-fpage">901</span>–<span class="cit-lpage">906</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWinzeler%26rft.auinit1%253DE.%2BA.%26rft.volume%253D285%26rft.issue%253D5429%26rft.spage%253D901%26rft.epage%253D906%26rft.atitle%253DFunctional%2BCharacterization%2Bof%2Bthe%2BS.%2Bcerevisiae%2BGenome%2Bby%2BGene%2BDeletion%2Band%2BParallel%2BAnalysis%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.285.5429.901%26rft_id%253Dinfo%253Apmid%252F10436161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODUvNTQyOS85MDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference  in text" id="ref-101">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.101" data-doi="10.1073/pnas.84.23.8553"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boeke</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">High-frequency deletion between homologous sequences during retrotransposition of Ty elements in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">8553</span>–<span class="cit-lpage">8557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DXu%26rft.auinit1%253DH.%26rft.volume%253D84%26rft.issue%253D23%26rft.spage%253D8553%26rft.epage%253D8557%26rft.atitle%253DHigh-frequency%2Bdeletion%2Bbetween%2Bhomologous%2Bsequences%2Bduring%2Bretrotransposition%2Bof%2BTy%2Belements%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.84.23.8553%26rft_id%253Dinfo%253Apmid%252F2825195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODQvMjMvODU1MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-102-1" title="View reference  in text" id="ref-102">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1475.102" data-doi="10.1146/annurev.genom.9.081307.164217"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurles</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lupski</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Copy number variation in human health, disease, and evolution.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">481</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenomics%2BHum%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DF.%26rft.volume%253D10%26rft.spage%253D451%26rft.epage%253D481%26rft.atitle%253DCopy%2Bnumber%2Bvariation%2Bin%2Bhuman%2Bhealth%252C%2Bdisease%252C%2Band%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genom.9.081307.164217%26rft_id%253Dinfo%253Apmid%252F19715442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genom.9.081307.164217&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19715442&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270313200021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/5/1475.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/5/1463" title="Schizosaccharomyces pombe MutSα and MutLα Maintain Stability of Tetra-Nucleotide Repeats and Msh3 of Hepta-Nucleotide Repeats" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/5/1487" title="Paternal Induction of Hybrid Dysgenesis in Drosophila melanogaster Is Weakly Correlated with Both P-Element and hobo Element Dosage" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/5"><span>Volume 7 Issue 5, May 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/5.cover-source.jpg?itok=VWLvGstM" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1475" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-hByaK1zza_kvN81-Wt7AmBdRtQJcIUXTvMlcPrudVaY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/5/1475" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/5/1475" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/5/1475" class="active" data-icon-position="" data-hide-link-title="0">Extensive Copy Number Variation in Fermentation-Related Genes Among Saccharomyces cerevisiae Wine Strains</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/484258" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-9f0hg83UccoEqXjYAQ5o6GgstCyQ5_5Q8df5arTSo2A" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/5/1475.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/5/1475" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-cEKUUPXkWglh1JipXo1K2blCkhl5td_f-IlMzPtU_Fs" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/5/1475" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/5/1475.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484258" id="node484258--2" data-pisa="ggg;7/5/1475" data-pisa-master="ggg;g3.117.040105" data-apath="/ggg/7/5/1475.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Extensive Copy Number Variation in Fermentation-Related Genes Among <em>Saccharomyces cerevisiae</em> Wine Strains</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8436-595X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jacob Steenwyk</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-7248-6551" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Antonis Rokas</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1475-1485;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.040105 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484258--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jacob Steenwyk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacob%2BSteenwyk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steenwyk%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJacob%2BSteenwyk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8436-595X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jacob Steenwyk</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Antonis Rokas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonis%2BRokas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rokas%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAntonis%2BRokas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7248-6551" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Antonis Rokas</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:antonis.rokas@vanderbilt.edu" class="" data-icon-position="" data-hide-link-title="0">antonis.rokas@vanderbilt.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/5/1475" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/5/1475&amp;title=Extensive%20Copy%20Number%20Variation%20in%20Fermentation-Related%20Genes%20Among%20Saccharomyces%20cerevisiae%20Wine%20Strains" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/5/1475&amp;title=Extensive%20Copy%20Number%20Variation%20in%20Fermentation-Related%20Genes%20Among%20Saccharomyces%20cerevisiae%20Wine%20Strains" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/5/1475&amp;title=Extensive%20Copy%20Number%20Variation%20in%20Fermentation-Related%20Genes%20Among%20Saccharomyces%20cerevisiae%20Wine%20Strains" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/5/1475&amp;text=Extensive%20Copy%20Number%20Variation%20in%20Fermentation-Related%20Genes%20Among%20Saccharomyces%20cerevisiae%20Wine%20Strains" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/5/1475&amp;title=Extensive%20Copy%20Number%20Variation%20in%20Fermentation-Related%20Genes%20Among%20Saccharomyces%20cerevisiae%20Wine%20Strains" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/5/1475&amp;t=Extensive%20Copy%20Number%20Variation%20in%20Fermentation-Related%20Genes%20Among%20Saccharomyces%20cerevisiae%20Wine%20Strains" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/5/1475&amp;title=Extensive%20Copy%20Number%20Variation%20in%20Fermentation-Related%20Genes%20Among%20Saccharomyces%20cerevisiae%20Wine%20Strains" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/5/1475&amp;title=Extensive%20Copy%20Number%20Variation%20in%20Fermentation-Related%20Genes%20Among%20Saccharomyces%20cerevisiae%20Wine%20Strains" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/484258/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484258" id="node484258--3" data-pisa="ggg;7/5/1475" data-pisa-master="ggg;g3.117.040105" data-apath="/ggg/7/5/1475.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Extensive Copy Number Variation in Fermentation-Related Genes Among <em>Saccharomyces cerevisiae</em> Wine Strains</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8436-595X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jacob Steenwyk</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-7248-6551" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Antonis Rokas</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1475-1485;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.040105 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484258--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jacob Steenwyk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacob%2BSteenwyk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steenwyk%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJacob%2BSteenwyk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8436-595X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jacob Steenwyk</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Antonis Rokas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonis%2BRokas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rokas%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAntonis%2BRokas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7248-6551" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Antonis Rokas</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:antonis.rokas@vanderbilt.edu" class="" data-icon-position="" data-hide-link-title="0">antonis.rokas@vanderbilt.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1475&amp;rft.epage=1485&amp;rft.atitle=Extensive%20Copy%20Number%20Variation%20in%20Fermentation-Related%20Genes%20Among%20Saccharomyces%20cerevisiae%20Wine%20Strains&amp;rft.volume=7&amp;rft.issue=5&amp;rft.date=2017-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Steenwyk%2C+Jacob&amp;rft.au=Rokas%2C+Antonis"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/484258/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/484258/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/484258/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/484258/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/484258/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/484258/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/484258/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/484258/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/484258/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/484258/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/484258/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/484258/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/5/1475" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/5/1475.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/5/1475.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/5/1475.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/5/1475.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
