<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/5/1393" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/5/1393.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/5/1393.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/5/1393.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/5/1393/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/5/1393.atom" />
<meta name="HW.pisa" content="ggg;7/5/1393" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genotype Calling from Population-Genomic Sequencing Data" />
<meta name="DC.Identifier" content="10.1534/g3.117.039008" />
<meta name="DC.Date" content="2017-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Maruki and Lynch. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genotype calling plays important roles in population-genomic studies, which have been greatly accelerated by sequencing technologies. To take full advantage of the resultant information, we have developed maximum-likelihood (ML) methods for calling genotypes from high-throughput sequencing data. As the statistical uncertainties associated with sequencing data depend on depths of coverage, we have developed two types of genotype callers. One approach is appropriate for low-coverage sequencing data, and incorporates population-level information on genotype frequencies and error rates pre-estimated by an ML method. Performance evaluation using computer simulations and human data shows that the proposed framework yields less biased estimates of allele frequencies and more accurate genotype calls than current widely used methods. Another type of genotype caller applies to high-coverage sequencing data, requires no prior genotype-frequency estimates, and makes no assumption on the number of alleles at a polymorphic site. Using computer simulations, we determine the depth of coverage necessary to accurately characterize polymorphisms using this second method. We applied the proposed method to high-coverage (mean 18×) sequencing data of 83 clones from a population of Daphnia pulex . The results show that the proposed method enables conservative and reasonably powerful detection of polymorphisms with arbitrary numbers of alleles. We have extended the proposed method to the analysis of genomic data for polyploid organisms, showing that calling accurate polyploid genotypes requires much higher coverage than diploid genotypes." />
<meta name="DC.Contributor" content="Takahiro Maruki" />
<meta name="DC.Contributor" content="Michael Lynch" />
<meta name="article:published_time" content="2017-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genotype Calling from Population-Genomic Sequencing Data" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genotype calling plays important roles in population-genomic studies, which have been greatly accelerated by sequencing technologies. To take full advantage of the resultant information, we have developed maximum-likelihood (ML) methods for calling genotypes from high-throughput sequencing data. As the statistical uncertainties associated with sequencing data depend on depths of coverage, we have developed two types of genotype callers. One approach is appropriate for low-coverage sequencing data, and incorporates population-level information on genotype frequencies and error rates pre-estimated by an ML method. Performance evaluation using computer simulations and human data shows that the proposed framework yields less biased estimates of allele frequencies and more accurate genotype calls than current widely used methods. Another type of genotype caller applies to high-coverage sequencing data, requires no prior genotype-frequency estimates, and makes no assumption on the number of alleles at a polymorphic site. Using computer simulations, we determine the depth of coverage necessary to accurately characterize polymorphisms using this second method. We applied the proposed method to high-coverage (mean 18×) sequencing data of 83 clones from a population of &lt;i&gt;Daphnia pulex&lt;/i&gt;. The results show that the proposed method enables conservative and reasonably powerful detection of polymorphisms with arbitrary numbers of alleles. We have extended the proposed method to the analysis of genomic data for polyploid organisms, showing that calling accurate polyploid genotypes requires much higher coverage than diploid genotypes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/05/01" />
<meta name="citation_mjid" content="ggg;7/5/1393" />
<meta name="citation_id" content="7/5/1393" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/5/1393" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/5/1393.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/5/1393.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/5/1393.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.039008" />
<meta name="citation_pmid" content="28108551" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1393" />
<meta name="citation_lastpage" content="1404" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Takahiro Maruki" />
<meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author_email" content="tmaruki@indiana.edu" />
<meta name="citation_author" content="Michael Lynch" />
<meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=J. Aars;citation_author=J. F. Dallas;citation_author=S. B. Piertney;citation_author=F. Marshall;citation_author=J. L. Gow;citation_title=Widespread gene flow and high genetic variability in populations of water voles Arvicola terrestris in patchy habitats.;citation_pages=1455-1466;citation_volume=15;citation_year=2006;citation_issue=6;citation_pmid=16629803;citation_doi=10.1111/j.1365-294X.2006.02889.x" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=F. L. Black;citation_author=F. M. Salzano;citation_title=Evidence for heterosis in the HLA system.;citation_pages=894-899;citation_volume=33;citation_year=1981;citation_issue=6;citation_pmid=7325154" />
<meta name="citation_reference" content="citation_journal_title=Annual review of entomology;citation_journal_abbrev=Annu Rev Entomol;citation_author=W. C. Black;citation_author=C. F. Baer;citation_author=M. F. Antolin;citation_author=N. M. DuTeau;citation_title=POPULATION GENOMICS: Genome-Wide Sampling of Insect Populations;citation_pages=441-469;citation_volume=46;citation_year=2001;citation_issue=1;citation_pmid=11112176;citation_doi=10.1146/annurev.ento.46.1.441" />
<meta name="citation_reference" content="citation_journal_title=Theor. Popul. Biol.;citation_author=A. H. D. Brown;citation_title=Enzyme polymorphism in plant-populations.;citation_pages=1-42;citation_volume=15;citation_year=1979;citation_doi=10.1016/0040-5809(79)90025-X" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=B. L. Browning;citation_author=S. R. Browning;citation_title=Genotype imputation with millions of reference samples.;citation_pages=116-126;citation_volume=98;citation_year=2016;citation_pmid=26748515;citation_doi=10.1016/j.ajhg.2015.11.020" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. R. Browning;citation_author=B. L. Browning;citation_title=Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.;citation_pages=1084-1097;citation_volume=81;citation_year=2007;citation_issue=5;citation_pmid=17924348;citation_doi=10.1086/521987" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=C. A. Buerkle;citation_author=Z. Gompert;citation_title=Population genomics based on low coverage sequencing: how low should we go?;citation_pages=3028-3035;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12105" />
<meta name="citation_reference" content="citation_journal_title=Ann. Hum. Genet.;citation_author=M. Cao;citation_author=J. Shi;citation_author=J. Wang;citation_author=J. Hong;citation_author=B. Cui;citation_title=Analysis of human triallelic SNPs by next-generation sequencing.;citation_pages=275-281;citation_volume=79;citation_year=2015;citation_pmid=25907303" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. Catchen;citation_author=P. A. Hohenlohe;citation_author=S. Bassham;citation_author=A. Amores;citation_author=W. A. Cresko;citation_title=Stacks: an analysis tool set for population genomics.;citation_pages=3124-3140;citation_volume=22;citation_year=2013;citation_pmid=23701397;citation_doi=10.1111/mec.12354" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=J. M. Catchen;citation_author=A. Amores;citation_author=P. Hohenlohe;citation_author=W. Cresko;citation_author=J. H. Postlethwait;citation_title=Stacks: building and genotyping loci de novo from short-read sequences.;citation_pages=171-182;citation_volume=1;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=C. C. Cockerham;citation_author=B. S. Weir;citation_title=Digenic descent measures for finite populations.;citation_pages=121-147;citation_volume=30;citation_year=1977;citation_doi=10.1017/S0016672300017547" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=F. Delmotte;citation_author=N. Leterme;citation_author=J. P. Gauthier;citation_author=C. Rispe;citation_author=J. C. Simon;citation_title=Genetic architecture of sexual and asexual populations of the aphid Rhopalosiphum padi based on allozyme and microsatellite markers.;citation_pages=711-723;citation_volume=11;citation_year=2002;citation_issue=4;citation_pmid=11972759;citation_doi=10.1046/j.1365-294X.2002.01478.x" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. A. DePristo;citation_author=E. Banks;citation_author=R. Poplin;citation_author=K. V. Garimella;citation_author=J. R. Maguire;citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data.;citation_pages=491-498;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21478889;citation_doi=10.1038/ng.806" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=A. G. Ferreira;citation_author=W. Amos;citation_title=Inbreeding depression and multiple regions showing heterozygote advantage in Drosophila melanogaster exposed to stress.;citation_pages=3885-3893;citation_volume=15;citation_year=2006;citation_issue=13;citation_pmid=17054490;citation_doi=10.1111/j.1365-294X.2006.03093.x" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=D. W. Foltz;citation_author=J. L. Hoogland;citation_title=Genetic-evidence of outbreeding in the black-tailed prairie dog (Cynomys-Ludovicianus).;citation_pages=273-281;citation_volume=37;citation_year=1983;citation_doi=10.2307/2408336" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=Y. B. Fu;citation_title=Genetic diversity analysis of highly incomplete SNP genotype data with imputations: an empirical assessment.;citation_pages=891-900;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=A. Auton;citation_author=L. D. Brooks;citation_author=R. M. Durbin;citation_author=E. P. Garrison;citation_title=A global reference for human genetic variation.;citation_pages=68-74;citation_volume=526;citation_year=2015;citation_pmid=26432245;citation_doi=10.1038/nature15393" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=T. C. Glenn;citation_title=Field guide to next-generation DNA sequencers.;citation_pages=759-769;citation_volume=11;citation_year=2011;citation_pmid=21592312;citation_doi=10.1111/j.1755-0998.2011.03024.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=E. Han;citation_author=J. S. Sinsheimer;citation_author=J. Novembre;citation_title=Characterizing bias in population genetic inferences from low-coverage sequencing data.;citation_volume=31;citation_year=2014;citation_pmid=24288159;citation_doi=10.1093/molbev/mst229" />
<meta name="citation_reference" content="citation_journal_title=Biol. Rev. Camb. Philos. Soc.;citation_author=P. D. N. Hebert;citation_title=Population biology of Daphnia (Crustacea, Daphnidae).;citation_pages=387-426;citation_volume=53;citation_year=1978;citation_doi=10.1111/j.1469-185X.1978.tb00860.x" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_journal_abbrev=Genetica;citation_author=P. W. Hedrick;citation_title=Balancing selection and MHC.;citation_pages=207-214;citation_volume=104;citation_year=1998;citation_issue=3;citation_pmid=10386384;citation_doi=10.1023/A:1026494212540" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Hodgkinson;citation_author=A. Eyre-Walker;citation_title=Human triallelic sites: evidence for a new mutational mechanism?;citation_volume=184;citation_year=2010;citation_pmid=19884308;citation_doi=10.1534/genetics.109.110510" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. A. Hohenlohe;citation_author=S. Bassham;citation_author=P. D. Etter;citation_author=N. Stiffler;citation_author=E. A. Johnson;citation_title=Population genomics of parallel adaptation in threespine stickleback using sequenced RAD tags.;citation_pages=e1000862-e1000862;citation_volume=6;citation_year=2010;citation_issue=2;citation_pmid=20195501;citation_doi=10.1371/journal.pgen.1000862" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. R. Hudson;citation_author=N. L. Kaplan;citation_title=STATISTICAL PROPERTIES OF THE NUMBER OF RECOMBINATION EVENTS IN THE HISTORY OF A SAMPLE OF DNA SEQUENCES;citation_pages=147-164;citation_volume=111;citation_year=1985;citation_issue=1;citation_pmid=4029609" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. Hunt;citation_author=T. Kikuchi;citation_author=M. Sanders;citation_author=C. Newbold;citation_author=M. Berriman;citation_title=REAPR: a universal tool for genome assembly evaluation.;citation_pages=R47;citation_volume=14;citation_year=2013;citation_pmid=23710727;citation_doi=10.1186/gb-2013-14-5-r47" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. A. Frazer;citation_author=D. G. Ballinger;citation_author=D. R. Cox;citation_author=D. A. Hinds;citation_title=A second generation human haplotype map of over 3.1 million SNPs.;citation_pages=851-861;citation_volume=449;citation_year=2007;citation_issue=7164;citation_pmid=17943122;citation_doi=10.1038/nature06258" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. M. Altshuler;citation_author=R. A. Gibbs;citation_author=L. Peltonen;citation_author=D. M. Altshuler;citation_title=Integrating common and rare genetic variation in diverse human populations.;citation_pages=52-58;citation_volume=467;citation_year=2010;citation_issue=7311;citation_pmid=20811451;citation_doi=10.1038/nature09298" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. A. Jenkins;citation_author=J. W. Mueller;citation_author=Y. S. Song;citation_title=General triallelic frequency spectrum under demographic models with variable population size.;citation_volume=196;citation_year=2014;citation_pmid=24214345;citation_doi=10.1534/genetics.113.158584" />
<meta name="citation_reference" content="citation_journal_title=Cytogenetic and genome research;citation_journal_abbrev=Cytogenet Genome Res;citation_author=J. Jurka;citation_author=V. V. Kapitonov;citation_author=A. Pavlicek;citation_author=P. Klonowski;citation_author=O. Kohany;citation_title=Repbase Update, a database of eukaryotic repetitive elements.;citation_pages=462-467;citation_volume=110;citation_year=2005;citation_issue=1-4;citation_pmid=16093699;citation_doi=10.1159/000084979" />
<meta name="citation_reference" content="citation_title=The Advanced Theory of Statistics;citation_author=M. Kendall;citation_author=A. Stuart;citation_volume=2;citation_year=1979" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=S. Y. Kim;citation_author=K. E. Lohmueller;citation_author=A. Albrechtsen;citation_author=Y. Li;citation_author=T. Korneliussen;citation_title=Estimation of allele frequency and association mapping using next-generation sequencing data.;citation_pages=231-231;citation_volume=12;citation_year=2011;citation_pmid=21663684;citation_doi=10.1186/1471-2105-12-231" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=T. S. Korneliussen;citation_author=A. Albrechtsen;citation_author=R. Nielsen;citation_title=ANGSD: analysis of next generation sequencing data.;citation_pages=356;citation_volume=15;citation_year=2014;citation_pmid=25420514;citation_doi=10.1186/s12859-014-0356-4" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_title=A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data.;citation_volume=27;citation_year=2011;citation_pmid=21903627;citation_doi=10.1093/bioinformatics/btr509" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=H. Li;citation_author=J. Ruan;citation_author=R. Durbin;citation_title=Mapping short DNA sequencing reads and calling variants using mapping quality scores.;citation_volume=18;citation_year=2008;citation_pmid=18714091;citation_doi=10.1101/gr.078212.108" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and samtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=R. Li;citation_author=Y. Li;citation_author=X. Fang;citation_author=H. Yang;citation_author=J. Wang;citation_title=SNP detection for massively parallel whole-genome resequencing.;citation_volume=19;citation_year=2009;citation_pmid=19420381;citation_doi=10.1101/gr.088013.108" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=Y. Li;citation_author=C. J. Willer;citation_author=J. Ding;citation_author=P. Scheet;citation_author=G. R. Abecasis;citation_title=MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes.;citation_pages=816-834;citation_volume=34;citation_year=2010;citation_issue=8;citation_pmid=21058334;citation_doi=10.1002/gepi.20533" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Lynch;citation_title=Estimation of allele frequencies from high-coverage genome-sequencing projects.;citation_volume=182;citation_year=2009;citation_pmid=19293142;citation_doi=10.1534/genetics.109.100479" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Lynch;citation_author=R. Gutenkunst;citation_author=M. S. Ackerman;citation_author=K. Spitze;citation_author=Z. Ye;citation_title=Population genomics of Daphnia pulex.;citation_pages=315-332;citation_volume=206;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=T. Markow;citation_author=P. W. Hedrick;citation_author=K. Zuerlein;citation_author=J. Danilovs;citation_author=J. Martin;citation_title=HLA polymorphism in the Havasupai: evidence for balancing selection.;citation_pages=943-952;citation_volume=53;citation_year=1993;citation_issue=4;citation_pmid=8213824" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=E. R. Martin;citation_author=D. D. Kinnamon;citation_author=M. A. Schmidt;citation_author=E. H. Powell;citation_author=S. Zuchner;citation_title=SeqEM: an adaptive genotype-calling approach for next-generation sequencing studies.;citation_volume=26;citation_year=2010;citation_pmid=20861027;citation_doi=10.1093/bioinformatics/btq526" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. Maruki;citation_author=M. Lynch;citation_title=Genome-wide estimation of linkage disequilibrium from population-level high-throughput sequencing data.;citation_volume=197;citation_year=2014;citation_pmid=24875187;citation_doi=10.1534/genetics.114.165514" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. Maruki;citation_author=M. Lynch;citation_title=Genotype-frequency estimation from high-throughput sequencing data.;citation_volume=201;citation_year=2015;citation_pmid=26224735;citation_doi=10.1534/genetics.115.179077" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_journal_abbrev=Genetica;citation_author=D. J. Melnick;citation_title=The genetic consequences of primate social organization: a review of macaques, baboons and vervet monkeys.;citation_pages=117-135;citation_volume=73;citation_year=1987;citation_issue=1-2;citation_pmid=3505885;citation_doi=10.1007/BF00055255" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. R. Nelson;citation_author=D. Wegmann;citation_author=M. G. Ehm;citation_author=D. Kessner;citation_author=P. St Jean;citation_title=An abundance of rare functional variants in 202 drug target genes sequenced in 14,002 people.;citation_volume=337;citation_year=2012;citation_pmid=22604722;citation_doi=10.1126/science.1217876" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. Nielsen;citation_author=T. Korneliussen;citation_author=A. Albrechtsen;citation_author=Y. Li;citation_author=J. Wang;citation_title=SNP calling, genotype calling, and sample allele frequency estimation from New-Generation Sequencing data.;citation_pages=e37558-e37558;citation_volume=7;citation_year=2012;citation_issue=7;citation_pmid=22911679;citation_doi=10.1371/journal.pone.0037558" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. E. Pool;citation_author=I. Hellmann;citation_author=J. D. Jensen;citation_author=R. Nielsen;citation_title=Population genetic inference from genomic sequence variation.;citation_volume=20;citation_year=2010;citation_pmid=20067940;citation_doi=10.1101/gr.079509.108" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. A. Quail;citation_author=M. Smith;citation_author=P. Coupland;citation_author=T. D. Otto;citation_author=S. R. Harris;citation_title=A tale of three next generation sequencing platforms: comparison of Ion Torrent, Pacific Biosciences and Illumina MiSeq sequencers.;citation_pages=341-341;citation_volume=13;citation_year=2012;citation_pmid=22827831;citation_doi=10.1186/1471-2164-13-341" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=P. Scheet;citation_author=M. Stephens;citation_title=A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase.;citation_pages=629-644;citation_volume=78;citation_year=2006;citation_issue=4;citation_pmid=16532393;citation_doi=10.1086/502802" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. L. Speir;citation_author=A. S. Zweig;citation_author=K. R. Rosenbloom;citation_author=B. J. Raney;citation_author=B. Paten;citation_title=The UCSC genome browser database: 2016 update.;citation_volume=44;citation_year=2016;citation_pmid=26590259;citation_doi=10.1093/nar/gkv1275" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=J. F. Storz;citation_author=H. R. Bhat;citation_author=T. H. Kunz;citation_title=Genetic consequences of polygyny and social structure in an Indian fruit bat, Cynopterus sphinx. II. Variance in male mating success and effective population size.;citation_pages=1224-1232;citation_volume=55;citation_year=2001;citation_issue=6;citation_pmid=11475058;citation_doi=10.1554/0014-3820(2001)055[1224:GCOPAS]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=C. L. Tarr;citation_author=S. Conant;citation_author=R. C. Fleischer;citation_title=Founder events and variation at microsatellite loci in an insular passerine bird, the Laysan finch (Telespiza cantans).;citation_pages=719-731;citation_volume=7;citation_year=1998;citation_doi=10.1046/j.1365-294x.1998.00385.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of evolutionary biology;citation_journal_abbrev=J Evol Biol;citation_author=C. Tollenaere;citation_author=J. Bryja;citation_author=M. Galan;citation_author=P. Cadet;citation_author=J. Deter;citation_title=Multiple parasites mediate balancing selection at two MHC class II genes in the fossorial water vole: insights from multivariate analyses and population genetics.;citation_pages=1307-1320;citation_volume=21;citation_year=2008;citation_issue=5;citation_pmid=18624885;citation_doi=10.1111/j.1420-9101.2008.01563.x" />
<meta name="citation_reference" content="citation_journal_title=Curr. Protoc. Bioinformatics;citation_author=G. A. Van der Auwera;citation_author=M. O. Carneiro;citation_author=C. Hartl;citation_author=R. Poplin;citation_author=G. Del Angel;citation_title=From FastQ data to high confidence variant calls: the Genome Analysis Toolkit best practices pipeline.;citation_pages=11101-1.10.33;citation_volume=11;citation_year=2013;citation_pmid=25431634" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=F. G. Vieira;citation_author=M. Fumagalli;citation_author=A. Albrechtsen;citation_author=R. Nielsen;citation_title=Estimating inbreeding coefficients from NGS data: impact on genotype calling and allele frequency estimation.;citation_volume=23;citation_year=2013;citation_pmid=23950147;citation_doi=10.1101/gr.157388.113" />
<meta name="citation_reference" content="citation_title=Genetic Data Analysis II: Methods for Discrete Population Genetic Data;citation_author=B. S. Weir;citation_year=1996" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genotype Calling from Population-Genomic Sequencing Data" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
<meta name="twitter:description" content="Genotype calling plays important roles in population-genomic studies, which have been greatly accelerated by sequencing technologies. To take full advantage of the resultant information, we have developed maximum-likelihood (ML) methods for calling genotypes from high-throughput sequencing data. As the statistical uncertainties associated with sequencing data depend on depths of coverage, we have developed two types of genotype callers. One approach is appropriate for low-coverage sequencing data, and incorporates population-level information on genotype frequencies and error rates pre-estimated by an ML method. Performance evaluation using computer simulations and human data shows that the proposed framework yields less biased estimates of allele frequencies and more accurate genotype calls than current widely used methods. Another type of genotype caller applies to high-coverage sequencing data, requires no prior genotype-frequency estimates, and makes no assumption on the number of alleles at a polymorphic site. Using computer simulations, we determine the depth of coverage necessary to accurately characterize polymorphisms using this second method. We applied the proposed method to high-coverage (mean 18×) sequencing data of 83 clones from a population of Daphnia pulex . The results show that the proposed method enables conservative and reasonably powerful detection of polymorphisms with arbitrary numbers of alleles. We have extended the proposed method to the analysis of genomic data for polyploid organisms, showing that calling accurate polyploid genotypes requires much higher coverage than diploid genotypes." />
<meta name="og-title" property="og:title" content="Genotype Calling from Population-Genomic Sequencing Data" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/5/1393" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genotype calling plays important roles in population-genomic studies, which have been greatly accelerated by sequencing technologies. To take full advantage of the resultant information, we have developed maximum-likelihood (ML) methods for calling genotypes from high-throughput sequencing data. As the statistical uncertainties associated with sequencing data depend on depths of coverage, we have developed two types of genotype callers. One approach is appropriate for low-coverage sequencing data, and incorporates population-level information on genotype frequencies and error rates pre-estimated by an ML method. Performance evaluation using computer simulations and human data shows that the proposed framework yields less biased estimates of allele frequencies and more accurate genotype calls than current widely used methods. Another type of genotype caller applies to high-coverage sequencing data, requires no prior genotype-frequency estimates, and makes no assumption on the number of alleles at a polymorphic site. Using computer simulations, we determine the depth of coverage necessary to accurately characterize polymorphisms using this second method. We applied the proposed method to high-coverage (mean 18×) sequencing data of 83 clones from a population of Daphnia pulex . The results show that the proposed method enables conservative and reasonably powerful detection of polymorphisms with arbitrary numbers of alleles. We have extended the proposed method to the analysis of genomic data for polyploid organisms, showing that calling accurate polyploid genotypes requires much higher coverage than diploid genotypes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
    <title>Genotype Calling from Population-Genomic Sequencing Data | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"pcCDAPp1qHynrKMP5hE_cAJdBHuaDxEqlwLdtmfJ9ik","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"484252","apath":"\/ggg\/7\/5\/1393.atom","pisa":"ggg;7\/5\/1393","ac":{"\/ggg\/7\/5\/1393.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/5\/1393.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/5\/1393"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/5\/1393"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/5\/1393.abstract":true,"\/content\/7\/5\/1393":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022hjPVXD4\\\\\\\/S+gYFaurrBKW3\\\\\\\/1npO5HVi\\\\\\\/v1N+8pdOZ2DWaWnZwpMph8pNFqhKVJPpe1jAL\\\\\\\/d6yYkzyrKdAvfIPwT1Upebhpur\\\\\\\/CJMhjd1Nvf1Kj\\\\\\\/seJX6CrzL1PkvErR4+LkxeMwuCaf8XCBxTksDmi3odDvTay\\\\\\\/S\\\\\\\/7qYd4T7vpc2SqI2O2loZ4n5N1u0WAfbG+JsJjRezI69sgcN89m\\\\\\\/k\\\\\\\/6lqcTs3UVJVrdah39NpL24rzy0in4OJ\\\\\\\/bSKvd7srPmeybeEVElPs2nVheC7ZKRWPtPtvQ7tBbTvudse6cmeXIoiYW8y9e1pw+KAjFqGMhLO1GxkBrhFGCUd\\\\\\\/A8i4fEAZdLcPqU3xJTPd5aDQXpSkCdYJLxgLsoei8tS1fkpLrqCRIGuLHj+xWmIovcAgj5gT5uU6Po54dBWS9ObNrjOqyIAJanDAWO+pExpfWAPSEwOGLOxMVLOUYpW7cQYnr9IwltG9qJcDMeDAU+y9ikAzhiBU6UXB65p7CS7utwnPgXV\\\\\\\/cKsGut7R+frG1H3w+I6E559fauy9N\\\\\\\/Rw\\\\\\\/sKBmZXwzL5k6caK9e4lypOetHF94KzWfeJ+vpJwt6psWkeAIV5ubuRzbaKuMAZNKoZfvF688BQ5aZw6\\\\\\\/Qg6VsxgKj00LBmMh\\\\\\\/5jQTOwT0IDhkEfa3RzV84aY\\\\\\\/9MaPaulhuC\\\\\\\/nO9C88jOOmUJpqeGR+zcqLfy8CI5V+l1psdyavPkkycwlGmX5PonFL\\\\\\\/Pd3XhvarVnS4spDSVpU\\\\\\\/8+uNdHcn50tif1WOTs02ogiK7C5WWxMu9G1gx8zL\\\\\\\/BQY33AJsb6aQ3vb43dzOouHF7pR5ch46cN+pHdr\\\\\\\/Rk0Gao8Tw3yXrySn7GW\\\\\\\/q7cRlSIK88YTcL1DAIMBSGW2Pg1pcFTjlTepWzI\\\\\\\/j1khQYuzL\\\\\\\/RrCmaTPK7ecGJ1LkvTHYBrSO0T3aDo4vsYxh24wom4l7pmTy68rVvcLGkh8+IV9EuY8TJI5XwdCyrsJNCMcLleIQRgaa8jCyhU\\\\\\\/EwAvX5aenUsODWfJvEiE\\\\\\\/WG0G3eJI5Trq90Z5L3A7xnvCc18BdsbP45\\\\\\\/XK95bhasnd6JkhA3XQDh0yC+ph7uFrENuDiWz9LcEZDIWcz3Oy26k8llN+VGmSGsVbJnoQBCITDufOFrL\\\\\\\/ATz7nfSNLd1GsySZpZzusintLDN6bZoI96R6XGiY2uZZhH4F3Hat2hnrVxK732xnTo71Qv8AX9xZxNx9kwm4hJbNsfWFIF4blE3kwTruEXBhO4AXYE2KR91IyyYu7nTSQcwmpwcP99U\\\u0022,\\\u0022iv\\\u0022:\\\u0022jz8gUmWhmpzlB14oZLiOpQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ee6896b9c3ff3ead9fe7cf2ab019e6dfa7fd4ec6452bbda54f1a4dfd97d3f124\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022E17LiHUXhIkrPVBYYMGfgAOX1zKHLwaZnxmOleviWOQ2dF5Mxpq8Sned4QEf0nob60shTwCbndG1B4gq\\\\\\\/IMpwgUP9sjWyLy9GT8L7zTaHsR5BsaXgIqCFYsc+ZQz1fsJPuha6VQrsIfjrcSWHoZeaX1HglIhAI8azfFOfBLCqTy43Q6gvp3bh6uzvoiyHvhUZ7+74Ah2FkKFh9JZwwwvTFDlKbsIPTrNrUCOrRKcGlulZ5klbP\\\\\\\/FezqoQsVXmroVpjsiBAg0S5Vpk4+N1zZ9CjyQ7yyxmfgF0vmpDnR\\\\\\\/K+kEVgE1v28bZFfHWAZpPransh9eq86P96+HsB73cHGDsglkiQoNlROurxVeA6usY3VNXt6upBVsyad3E++ZMocRMbsUcaWwTKHvSED9crJrAAAMqok5TfJ+Oc8XjLMai0ZkWkMF8+rX0EviTrmKiMiGE+dLtzcyBr3GrT8zSzi1+HKWd+YP8Z1hLuDF+MWMUKZUEU7RgrXCVmY\\\\\\\/bGk8P58ZYwwAW4BRtlm+7X1P88ytuEfK9i8QW13mk7bED5rpVD9XZ7FNDrnAf6CxZkbnH0\\\\\\\/X+h+iPT6s1zfSZv4eF9JU7LHkqQRuLORZwB7fkD1Ea9kMjhpUTXRLopzAiDzZaY0DoEyHQ4oxA3tovp2PvPT6JEIp3DEFsIEXhHFPsA\\\\\\\/XqtxU8uqMCLsOSbdP6fjIDlN6NZVvKcjKc7ylLXaDi6liDo54Ftsgit0kZbr56y6z9tl6AE35XmitYAU6VslFHYEpZTPagcOcSxGnUW5kZCSSYhkioKcnmjCsbu5hu7abBoKf3Xs1ZUNb9gAp2Fc7VKlaAzXSnfN\\\\\\\/io9EOiDHLpxtifG9w1gkCrp85kDprnk7P2HLlowR5J2Xhcm5GJeDdA8oviyUwPeIPBkYvWERd7L3OQZUfje05QWwUu6QTsG4efasT13Oj9CMOrbM\\\\\\\/74iIWVhzVDFmnOjfxn5RZ8tvblt3ajv556DO7Haur+wPxcubImiRtHb\\\\\\\/Y8nqyjp0znZ\\\\\\\/PKEa5htjmK\\\\\\\/ILV66WW4Ykrs+w1JrXcayeMlZutzqa5tFkWRUMQVn+acT4HELY42l1wh\\\\\\\/Oc1vhdrLSRNbaaZzfDU6DfbTGaCpWFf3IdaOfr7Q+RtHX+xVIa\\\\\\\/YmB11VWEYYY2rE\\\\\\\/V8NHbfIgSH85CsmY\\\\\\\/IEcSiT544W3HKWauWyzMY3YBPAeVQIwlbLT0KsITj4kLigLHKc7IJJvMoLdDMFK+tNrRmTEXG3mOYf5YT8DGi4Z8N6EzTG4XL\\\\\\\/jbRixNeHmM1WIscHCrwHAZQAWu4nKsur1nmB0EmAcc70dGk5wsgdhkoiYq\\\\\\\/zIParVVJ9XYVBRC0syd1BA464GEQBUR+XGI+ouVzOVxX1n+rHCX0ssPcEHwWEwDXb0hu7vtxhh7YIlEhPI549nTuOWMQPIRs\\\\\\\/dl73gprShI68N83+ee8p3YFYcG4Bm9rkL\\\\\\\/5RyM9Q22ssRFI3sEXXJMxwUmlwbMnT+mEsZIhK3GGsAeCoF8gKdLPA6aHZ+CA7yVTgKb+BJqNDvA6epgPzzPirsjzak7pVmKhDYQRwWfiAyxJyidLLgIaZua\\\\\\\/lV8x2BAURYz5wgtATM18JqTSoL+myGFIyHI4YL\\\\\\\/efyn+ipleWFgqF8uH6SEePLNawMCiFDIzwNN+inchTij9vjXPwSgksgwEPRneLyWxoZxlCQhR6JWv4sI6JWKYW6Mr43wTq81OuL7COjKjjkGiFa\\\\\\\/eJRDdsb\\\\\\\/nDTBx\\\\\\\/sIZbVmkN5a8KYbbphcfeB7vOGS0tbz7cEbzLbrJeRE+JMGArc+c\\\\\\\/An1oZLfKkF3LFnFdD\\\\\\\/27sNVCZoIktclsPdftJdothbPhCb\\\\\\\/ocX\\\\\\\/OPU66abIlQSKaFa2UPFHQIRNDCKBt7tFHmVGC22u9EnfzcZPZ9stdmGl8c\\\\\\\/eZjzAGhp9UgeaPwAtupEevbHxhtTYAYutkJTEn1R40Y1z6ywfLjiG+QARoAU8YMPo+YpZG4G+ByC\\\\\\\/0pFPVD2DAGVjLI2TLRdGDGKd0pfC5IPvSuRwQf33bRSgT\\\\\\\/U5E86PPA2K5hU9TNkV4asSje1w7FKALJGoCeZOZdqrz484cy6hVWAWGCDvvqhzKK0NkjROJyt9kMT2LkeM3L23zjJBu8uUtq+U5AG32v6hZGZe9hKsRjQdEk\\\\\\\/\\\\\\\/YglEypj+ZYw0Shvezw92SHgXNBBW5U9IKfIvo6nllnAdM2UVd2OyFpVtuFZ892QoiyZx06ctEVjcJRKDAKKpzByBwvZbtEuRM0K0S0tNftDEJkNA8CNYT3bCsQw8tTJZjvZSNlOx6Cub1AKENgCjCjYEUBZBJyrReMy+sGD4HYvkvTirHNQJdMPywGCcQI91szHGfXRT3STCl6lygiDEMWtTFmL2tavbd1hk7agfhzhaWxi155\\\\\\\/xTBGjBM48eN+6wQ8Ui\\\\\\\/c4sk5Vc3ZJ23csOnxI33tO8raoCbqVS0bRdC3EK8DjUchDS91yEEhltoZrhzp3r5xMozPKO9Z1FT5FGx3UP04sxtO\\\\\\\/uKTico1rcZef4X4kKlEtlVX5scIYcqRZssdB\\\\\\\/JnLEr+GeQ5esUvw7vsgaEFaIab3mIeC+O7kLaVCgN\\\\\\\/etA5UD\\\\\\\/ypG+iUjUbBB2zI1ZdyPQMKLa9VQG9py\\\\\\\/BCHyW0PkTaiP6hNCMIhi1PaVCPceS9miWHmchcQwGuvbf0Q5bff05jXOQ\\\\\\\/18mWLd5vEmV4m9A4hpDW0acfXBihIvUvSlza5vNMEKYmowzpvUt69KMaHQjYSRBYTEMKxJIEF+1BAufsUR\\\\\\\/1eTtBs3ym\\\\\\\/lx+Ld4p02v4kxhzlqBkbBDY9vuE+6MWxX09s5\\\\\\\/J21JZuDdHIzU4VZvgsteIu1jgvY4yl0bgeEMpdv6t0RMdhAIAjPg1MlEmzpn09+rWTm8AwE8JRb1W\\\\\\\/ryRyeY06iPnKTVtQriA9ZCyE3TdjFEagOlBLcJHf8ecNogwR5Op\\\\\\\/ovcIerx0NOrJl2MonHaGoB8Z72bLtXQtmKzq6Nvpf6bvI2qdB2i7RSdczEvlmUzcdnNECrVnmG7bJVJJCkPgqYbQa3DaS+XQHKqLlUMIea03wsW2PxA+UmDBmvLyxfHGQ783vGnqBzQXf7Xx72LIxS8tFAJ6Q7UVE8kLWjQEGgX7yD1J1O6shD1Z2yWJMpw8pZgnT+ESpbEuh29MWUSiOdC+d302VD3VQoAF4p7jJk1qeHlvSsVLX1OEaHaSrxH5uNIonvze51UlPBHZ4TSLUIpPgiK6CMO1YTRyLjlZ5677nWXLqj3ilolyho2dEJORr+NhIkvZaXCh48IVGxWSZsSjnlGjmEuxkHtpa2kgLyVIuIxH4KxdVQo0sLni5tZkl64pnMhmpWYizUuDwJO0JYVmRMd1utUuNqrEwdliMxkGpnq09+En9J8+LR5gaS3qSSbJw44pb48wQD9\\\\\\\/OS2WPn0kAUW6dmIWtLKNDkZjwzKzUa951Tl2jOAl1sR9cjPPd\\\\\\\/OPb\\\\\\\/C7IcEHM5kTJGNuPLWZqbmgrM+D0fnxHgaLL4XmbiSciEp9cQr84XTE49X6OUnDZODlvFcARp9YzK9tft6D+5w+eIgEjhI4dvBi8pdF+6fQ9+P9RDpb56pkMXp6Qzze5+r5D7Jpoq5csn1Tosj1PSJc+fFtsHvZKkYTpNNK41oF1qqJSOGlDpL0Jay0wYo+Ficd+xn7nnHTZy8HbuKZSjDl9tohUvhyswRTio8bXner705h+SDlEgtNf2n7FiKI8tTWefuqh0McK\\\\\\\/fOQnWdFoP81FFD50Ues\\\\\\\/McoG5fd0pBejKqj1IttyVBVjsXo9h5dUQSs8Yu7f7OxTaqJunKwsZlkPIJ2KwgZm8rQhB\\\\\\\/1UmOrwTRw1OG5wrCSXRTBKf1nlmUeV3eYFoFNXvD9+Gid\\\\\\\/AHmlfZAQZbvd\\\\\\\/0gVZHPiW\\\\\\\/E4f8adoxrw9EiyevCy+bJcoelNsPQ7R0H24YeXOHCADXucAWuKEkr43UBJhRhwBMetJl1qgzeGIRkaGePYNUyyzmzGDLk96VWPjePVQga3DItbPPCllAow1TDyMUctFUM4fAJrUDIpGq\\\\\\\/fx7N26mez6NC3YkDdSwHrU1\\\\\\\/GFsK0mHa0kl\\\\\\\/uffVbltgAU86EVFZBSF0jKHU\\\\\\\/gAR+O++w+c9wBPSgxSIjIshVBh5Q+PvBDkXYWvP4TzITBSAtNAycejI7Uyj5fVUUAWvCSW0jL8M+\\\\\\\/jw3iNSgO5GkK53AjJghzhKT4lFR5QUmF8adLTJ8yobLoK+tKv4AT7BlT95AQmtmA6J1ml+Lz\\\\\\\/Y3f2wdaPTbBhmV8ezgu1lMVa\\\\\\\/vK7gSFnQbR7G9sj26QT0g\\\\\\\/J75IMY6SimldYIzdRJsUGsX2F2DX4FPoP0hq2sqeZbTaaVqNfk9xnjkGc7Q+9U9XpIpIMD2z8TWrpbV6\\\\\\\/fxliV62rE8vf5Qz6LEMYF8XxRfYN9Y0Zu6DNz5uhUtkLx6tDeR6gwhbxJcR\\\\\\\/bsGbzW\\\\\\\/7wsmw8NQeEVkcws3jRfjeuurYgmUyv8w7udn5mZ4HToIb49nES+dVFMgP77aydHZczYDoBMMFxp3mwD4LYwFX\\\\\\\/21bSTr9w\\\\\\\/w7A9gfpoHfTu4hiCOqeNir8QsvA8Pgvghj6DU+8F+SBVtK3c0IyHao3NH41W3B\\\\\\\/jRuq4P+obcNA2OsqqwKoSA+S+ql\\\\\\\/MD9VM\\\\\\\/BXmjgD6DCE3KK8BR8rXNQkP28TFy+6PkA8NkcSoat5Qoh3mrSlpZYxbYZYQKNG\\\\\\\/gUyFOezkJyqoFtqUyYhbJs1ILE8fn2ofU16bA+FcEdNhcTQLpyYm8BFJtf0gl\\\\\\\/W4fnaWlFYEmWjIh6fjsVAs3prlm7GGEo075Z9iig0fv+Z6Gnexj+HYXT+jo27aoSxNZb8V8YGXIbDnLZoGNf4h5zo9fDgD1oTex3\\\\\\\/9NqqfwxilEllFwh4b5OMrsf+q\\\\\\\/EOmrDPOyIb6EqkferX3jdvyuZK8c16iu26Skib9ltC8hJgZQ5WVBOa4e4D17gjX1jP\\\\\\\/GVtgWCpi1BX2HWHTtLJWqcbZneya8zuSYGygZMeLoqvmfv5jle9bm5LaGw839reIDL3EpSAHAT+djlK7c43KXbCWTPazmZDSwYDgbgwv+bPPEE8gMUC\\\\\\\/s4zMgF2MQaO2WiJDo0uByUaOk6rL8kCRqWBsGOB6T7b9P4bHcOMGzcX3Hi2RSFG2xkpSm\\\\\\\/XlANQzv9uAJIbv7D8Eyn6um6gZYgwRPihjCAluI+p3xtmY5KP9aHn1qCf0XNdnRpyZVwFWYPUFNDKAsa1afjjUGUUrLdNxvfkVZ704M5qfu8TZk\\\\\\\/+jklPihj\\\\\\\/bk2BTEa1X2OKzK8j\\\\\\\/esw7lYuDYk3R9P\\\\\\\/Aa+clfoX8lyW5KH2BNNdHr6sQrNCxOKFkQWSf7uuGPAuvaImrGW1GAA9vRLTtF\\\\\\\/sArTu3OaYJjn6GLkKd0CaAD0+nWPvTWE8+7EBPIDYeMJju4YcH0pysokpc7dTJfTyU\\\\\\\/pxZTXoSby7TVG775DiQi6DtuNKARNVOmbetmmA6cQDtFedsStqzOrAfb3AtP+M0F\\\\\\\/XVk0VYiOy\\\\\\\/Z+8mGBc64dnPlPI39ds4OyuZZLDDgT7OMf3UCAUcvPyV8SyhkxwsUFyHC4oT\\\\\\\/YUc6q8iMqrShQbBUmQcNM7BtZ\\\\\\\/G1Eam2VQaCXdl0qghoKFbIDOAUZB8\\\\\\\/vdbjBFjsLXitInSmLf9jiJ4IOlukc1HgdpQ6BDJGZqB\\\\\\\/fwKqqZjHj7Wft4\\\\\\\/venXGz7LpWMqtP4j38hyUU8eZpWrUPy20KJ5VhJtI\\\\\\\/egCRp4rhTTjI3EaDtfumqdLg7VmfnHY+apUC5cHvWJWrex1JyLlBiFrUtxGixNmt4lyoWcZoYRrJW252+M8lKKmtmHaIgOtpkMg6yJBPLVK2DCcWOrBDCNNhMpkjogb5yY9p0SlmrhPrINR\\\\\\\/uVzAUIgnbefjv4duUNdI\\\\\\\/wNrFJmGGNXF9lgLSjP+ftiAJ\\\\\\\/mBvCQqgI\\\\\\\/h5twnICMpvtQH\\\\\\\/Kg3oTQOjfK\\\\\\\/ZGkAGXl9y7PNgC6nKIkCNF0CiapA3MkQBGrwvieYBwdDV5CgnOrJvvulJaHY8WLSRIVoIgCXtm4otdzNhmDNzYPJzcsuwtjA+yp+6RtozshOXh0yCEQgIz+xyIROE6CzS0goaDA282tylpq6fChdu43PI8HZ8MbLF+VDV7yPjx7mSV4VHAqqOEwX66UuMTBRDmqw\\\\\\\/9pf8IZ\\\\\\\/n+tXcip0r9i2Ji7LuD74kurBvb9KDo\\\\\\\/+KOXnlDRDZ+1Wq7D1fae3Rr9uhUeekj3Mc2dzUp5cQ1eWp4MKxT8C5s5bGKJ4wiEKhhJy52TKDm29qvO0YseYqbpGySBK6y9M3tUloYAJ4nf7Mt2JYjYW7MWQRxmYUr7JG3f8BQk2W9roRHEAnj9t9CwvtcBsxhl6EbmvEPkKuxUNfNxrQw1MlQtYlXIOW1S9uA9TF+DDvIUBhylE4VlPmUo0JMA0NP2aJzrqBK++SvE1nBO9eGBlpR8Y1uXnCIB14UJbWIvqb2HOxAjv6g7HmsQVBO2RWCtnfH\\\\\\\/e\\\\\\\/FQs6d8pLVTKBqvg5AmBL0cqcd\\\\\\\/yWTUv+NweM6LqmBczeKX+fQJBxwUz2i4zr+p9w2LvTRrV9QYYVbWTjBr4l7yxmWsfHacGdwM+ebtykGUdOEaE+khaGOHZQv3ORC58ErhiBC1zH8kl3By5+\\\\\\\/ACVezO\\\\\\\/zMRZj2S9INlZsGQzTPO+qKcrVmHHtRK\\\\\\\/0qEwYpBJjSr2ZiaHjnismXAGlvb7CjWvIpvk8r3VXRuLAPsn1hoW2yx0AcxMnT+lHBR5w6+hJGgzRe1gPnXOCD0vitQP1td9y4VRkwkQFFEAoWDnL5IgQqfNk09ppiCpBUslYGsgk8M27cAb7YNVSRrOK0ZiZSSKF9X3P4OF8oHqKk4rUEWnCPhNFT7zq912+5glJ35jZofRqNxGWgPww0w+348G57+QNAop8i100pYHmSwrkRicaNvnGF38STSdfQjw3a+nSrBKuVVeVistGkykfHq0yzEDCL5WQZn38ZlbSsqsKTIMdxgpRjFOnxas2dn2kxnj6mVxlOKuyWTKqIzMN7kN7rRGXMUTc7QRK9LCJrvbyskgOVVGDpWuE3eJb45rqf3xK2\\\\\\\/zaaL3Wq2P8fzRmL6EOv9cTHQ+bPg+PGLzv1bCrCDaU0WKV8EZFrqhnb1r3CgXvnvkW9pF\\\\\\\/MB2n0\\\\\\\/VEG5uO0\\\\\\\/EPxwUvAENrWMJT3+Uv\\\\\\\/JVpWP0mE7Ov53mMFRB38YtNWZ640LaBsoe9QQDblHKH+xdYtKl\\\\\\\/yscwFOZPBRCzGEU\\\\\\\/aGgDsWiXVCbOZtn8TdKrPL2XgeblbBkH6cF97gs3LhpTiptRj2fGlc6IqVN6+R0XBLL2Tw5MxW5rg4PA7GqOjvOp7Iyw80yEr1821Lgequx\\\\\\\/wgpUkoeVJLVenJfVq9Pps\\\\\\\/Rd3CdDINcGyjIoIxdTsjCv+1TSfucsAaqkWC+svfAtMWUiMVONny\\\\\\\/1IYp02BTT2I3hqmwG1XFEQbZVOoNvkWVkLC+UdovZNuGZQo\\\\\\\/so6G75bS\\\\\\\/SVrN+QdzO6nRAX9Nss80D+yTBz7JMWj1L5J6Bj6XgTar20vhzSpFyElw6XuJtQHOTNn5RWG7OWzN72EVbvZ\\\\\\\/ijpfOjx7W5vei00QQecW4eMK24gYESe8XVpoOTILa7q2EW3guHOknNtg1MvDa9sABnDepEfWP\\\\\\\/N75ed7bwXoYPF2ZoZtEe4Co8fw93OLKqpO7\\\\\\\/Yxl7n6i4TGbhj93c4dUazpus0CdK09zL2ZB7TeMtwkhVuIbUFqeEZJnzrasqT4wBz22fQ0WlCPj6c7JmTP8mWX66OHrygw+dsCca3atvlGrCljUe\\\\\\\/vYM7aEGP3UNKxE0qwnNskHwMmt03dUS+oZsN4\\\\\\\/yJRG5Ok+HH59ubJwgUbMXUBwNhmtNksJf0MrMzck3eP0ywkZZGu4PUe26yqA7ZOOwpZhTZ6SxEBlQVuhqmkU4vJ3P+qKGu7z91UzWTekFmXnSs1FWKrZ5U1Qkg9cUlfzp4u9YVY1I+HSHzUGZwXp\\\\\\\/7R57WwiNcK4+uNKXZxWm\\\\\\\/1bU6xj58cxtLAOn9DYE8DeGArgfY9khzz0vKeJVOd3z\\\\\\\/wkWRVXy0UUHqtxnuaDJ+I00dPRGq3\\\\\\\/EJ2zdPgzmrEHgq8W536p0C8wN1EWK\\\\\\\/Ox1e9KDucSbLHW0seQlNLNqshoFSlw7j4iU7FZcv+UyJe0BPB+ihTzueY8KPA8uw875IMbo3Gntz3P\\\\\\\/8pjQeZg4UUgW0bE1Ob5kIFqd\\\\\\\/LZ00iT5ZLeTDJzKd1bB\\\\\\\/erNKtcwCcAWaSBk8iUeO0i5dE\\\\\\\/v0ZLQKu3QUBYUvSYpBUO5mCnYxJ6obz2d9HwdifPYhhZEQAN7wgcfT6XN5bNcTMgDvhd19XnMw3I1VwteFUBVssUTnVS3Na3uB1vzoP7ReuxHIVxRIy57Y8+3cNnU0rxA74davcVH3AMUO3UKROPBXSeeMDssldLyQGyuqpT0hm1+ZR16EMv73jUQdrevXMdGYxb0HWhKvxDRnbrGKIGShHM7UbJv8bCqmL3BiJLRPtn9+KvMTYGuOZd7CmDiF3hlyRkD87AQAtK23j0aXpWUR1BopO\\\\\\\/EYSr2LPF8LVtSDH4ydnGTNkXjdcRa0aSElJo+9RZElwoF4dl+l9TdFy\\\\\\\/0y5JBIlpj1SDQaRo0JT0B3EwMCTEbjTu2cUjVNtoeihKyazXDC9kmbmSviGzXEqrDzfR+V0\\\\\\\/2\\\\\\\/5xBuYqenQOusb\\\\\\\/SFIx22YYhkwzDbzyCmdW4JZZm6PQyP8LYdFMhwWNIXm0Q2IsvsdWwkdjR2MpFfVyjrMV83z6UWLASoFHhJPX\\\\\\\/HCbuW37NIq3AS4x69gjDXAkfEIMEr4PrSrKWzbNvj6h1nqdZvpTLP+2sWWNkSHcnpcz0FRMyQenf2dYwiP2II4Z2dVecu90hByCqpVg6gDpkJqs0UGbR1XNH4scWtHUXBchwmAJ9xidRpsirR9dq95TaH5aJpNsTN1AEL0NwKCnqDN33dXMbS6Hz+M4DW3WvsRi5Q4+dVIDBDUReCFMU5kIJyJAvWCSgoae+aoJJLKt9a9jWcFcrQbAl4DjecYHg69qDbTHaUVHFN\\\\\\\/AO1qHrmvklCD0MA7xYyu2CrQD7GFv1xzqmh5Gdam3gO+sAnRrpf6nxa5jtEahUXLh\\\\\\\/XEIz86iFmw7hvgjkdsM58dMURdDlI2tF1MWdkDsnqwDhpBtk3HooCD0josnVaK3iPZoNJmY4tLd1KAdR+kisrZ20zJ+ADd2SJ9ZYmr\\\\\\\/6RdV3yhSQis0SDlF2EZfAvTE5tgz76qFjlvGBJy6zYVyXHZaQhu1CdQQPqP7dPIsQxuW+G9+orN6CED7TzcFvhUVty8EdHxF5kZIRUcOwyXNDDlQMk1JGDH0YxvV4mddcWcw25v40K4SVNehTtT9hCoNRopihbVt9A+NOBb\\\\\\\/faqLBg7x3fEFEwdzMrqWjvDo8VU6ZdcR7eDIdalaBNq5cd85BQ8FrZUDJku92yEZtZnkPtABuARmlVgbCHorpoxJ8RPNQsNDl6LoUnjwA56ICEKZUVt9\\\\\\\/wfWo7SRVpEfau8wzHCcBvUF\\\\\\\/urbq1fsl1HhR38s2CZhudcEu+RJGB1f6auohHyYzQQwmn7X13U2ENNvgXKabQDkLcq0JdrEPxhlc5c64lLBJUPArotSFTvNQLLeV0T9FPkWQpT+Na7TIJ2tNAD4Z1+Gr0BaDmaiSrI7ZPqkABiu1SuKKPQXy0+nQQPzhOdxR5vAWGaOm6dppyX2QgUv9X5nEy9m2cExjgtaFpd5oV8R1YeLafhdedBB6m7iE\\\\\\\/LdwTg\\\\\\\/665ZIKrv2AuZ548tdJ40xPQ8IGhX8CwEftOT0FAyWSm5Fca6dAMe7M16DZSM64yRw3eMfRBnCTBDHbWJpsIQnv\\\\\\\/vjjXAoyD+wHJPL5rIcZvU4X5TB6Otp75gyR8o8i7PM\\\\\\\/iJ0Qxs1JairDBwg6RISdx1m2U27j7D3yuM+5wNjbEA1XJGDanuDtOgfTcxfcCDNNnSWg1JoLbVHyXw46mXyUaiuWnkvsmBNaexlnuAsS96iebHmQ32IvAKhzX9h6qqEo8GM2g3FRmiK0MOW\\\\\\\/w\\\\\\\/E0S4IjbSgYbOGa7gvP\\\\\\\/mzo+dWa2Du\\\\\\\/lTGJ+rjwgMf0AdHR+\\\\\\\/ZmA1VgFXR3TnnD1DoPsKNn6fKJtQkxuG+sehFXg+lO+3rF3ZrrR1x8WF6dQyhGIxuk3CReW2yWrOVBj5xylbZXLwKiaQLiqCn2jl6lj9yETAl1x2\\\\\\\/u+WFG9QAfpFjzLcq96MeamPkvGLY358cNl9bDGw3USvflCMVB4nJ3zVLiK+NBJw9LFHhi3w6pftXwmYjZuVLl+geM\\\\\\\/KpZ4rgWVxg0AUpOFNjgfd7aL8rUBoerLsXw4yeBHH7CMBQRvk\\\\\\\/+Rzuzgw3EnnJN0pmn4zGsGk3HV8Y7B2GZM\\\\\\\/swkii5AdeX6lDNfQJ7cuxEte1ndvOzrXXEA7Qf5zC+LjonwlMOj2Hdu0+xMuq+jcABdwFy3NwdX7NmmTxlFD7vNS6lgN\\\\\\\/80clIj9y5xG1uw6nvyowE9UcYU7FTRosO\\\\\\\/TTPlEHQQwHF8iXrH5Jm87bGvq2qb5a4We7N18UbaG\\\\\\\/LJ91sHGmwWySLU23yxKo3K99U0WVpC\\\\\\\/vWHFjN8b9S29k6QsE5W9gOJbzS1bkyOh1qOyd+tMsgMVd41O7\\\\\\\/xRwlbjxkeIhibzOSaEf2pi2VN7x5acRr5FZcJ1qnrfO8gIfCec\\\\\\\/3SaWBOx\\\\\\\/uBRLPnViUYNYKzYlLIwU7Rc8I0PgpiJvhQnof5iMlDpRaOa7SRsD3Y0n4U86Cv5gwv1ROi7tA2fnQ7hx3hcASLZAC5gJ10Y1CqHNluXe6zzD83+TXxT3ABD\\\\\\\/s6eAfS967YmlDpy7F3+SR6ZyPlkgZuJtaD6b1IMJ0tpEbtaKhxcoPeXI9kkEMEfrIEpV+fSp7Kn6DORR3pAZfJFsN0oZ7Wez165XjrgUU3YqJvCrVEeg5rrNbm3CEo+pTjnQW7hDqv2wcSZMv6tSh6KSRawDvTpWgwPd2MqRaoGRmJuRw+y6gCBPT1rlznzFeN4D9hxJbYJ2WmfEE9dk69h5zfQNrOLBrXkfFQhPpe+pMRpwGGBrgFxcNVKMQoWWMzcZ2JN1F7rPZyCeuR8H4WcvyEQ8bOTnucJZGBRhpVT+cApR91Sq5mU3CVeLJB+k3GageRpT91DL2Rthru0pAr27CNkdmeDcJxiLQbXDou\\\\\\\/jYM+6MfUuLwg15jiouIDDhCQKbGxmYOKlD1UzM\\\\\\\/7QblG6Qgv3jxD547Ys2dl8Hjn3eiCvoWLJY+fvt9jrO6akUgoLd0XgocoIlG+64DD3Tew+Kupf\\\\\\\/JSmR\\\\\\\/oArslQvFor3yh5Jmlvj+\\\\\\\/FycZSsCQ6IVM6hYtrWzdKoRgGnYLTxEIGmDtDrSA7ECFw1yB12TkZigZ78mneq\\\\\\\/Xzdtai0jSZJGKm4ic8ABxnpFKt\\\\\\\/eUJ9WBUBsCfAgrYwozGaOcGRhv8sMsAgw4itS6GDp18I1BIkUoVg8RgLiB\\\\\\\/UBYDav0q8+X65HoocsUc9paCFIBWSIXX4Uvwo9ECO9FJ9I5NPd7aUM5FfA8apxt3uTlMvkZpmwOjVznghOhOnKWi6Lz\\\\\\\/coENdgyeCNfqDnF5BrVKSVT384TnKME4m3v+u4jq+llfwdpvZdWby2GV6i7Hng303kGCGq5NGF6z9Ya6HThkP+diOVsr1E1TOrDF5rnWcdLx5fHS3lbYJGdKTSbccYSbyTZhX+XmHuT8kGQphbl++aTTiwLOAN+K14yYD8VsjMbt38yasSP7F3H5XggKQVfcgDaxiLrBqG\\\\\\\/yWTuCjbVcP3uJgKYSsNDJYbK7BOkFxgUAtsHNoN3cnEHs1agi7rCIT1s6+Auzpc5tXUsuvwncreHPALR6zAUFOkrZY32z8NXXOolWfSvtTbBsbX4cU3x\\\\\\\/PL2Y5au5SWR9odKr\\\\\\\/bZMV1luZHrHTwC9ni2DiOmWaByGEZPEuhgILK81xJwlsEuj27H1wiJpQoQfdcgW+p7usN+Hrzclx\\\\\\\/3x4IRB5Nrjo25i1k\\\\\\\/ISJFWbhizYPWV1sISLVqJIk\\\\\\\/b47pnYl27mAcTGGGRmeV4H9Pl\\\\\\\/wYvn4AdU+UWqClXzDIHsARBNYmC07B+tvZQ+ox+q3bho+nMYyFVFwUCOH7byJAHQmH+cZD4Fo9XAcIeOIzg17OMI+wMutZedYbqEbkvFdHhCG3QuAO\\\\\\\/Zo8Wcp\\\\\\\/JsB9\\\\\\\/ZljdSv+pZ0ohgFtjpu05LHxiYi\\\\\\\/FRDpOyX9j9QSEQaBb0WpHgV3SED7BHfEl6ItFBWc+fZ+2nWruVtEHGApcoAAYN3SACR1HVkgGZ00mfk7r52sR2TpEJLBaXb4CEVw+Rp\\\\\\\/snQgCzZAGRMKxD3iQ+kE8kYJ8hlj2U3EozQLMbBSJeyI9MtXldgbCd5KPK7dfwLpXJqDylU3JZrgLjSUP5ucQkg6TbHy\\\\\\\/z6x23K94QmhBScJqqPzu4Dl9EOblPnYrf7pGZQLs8v5wrNn34tnB+i4yg9sct9dOM0Mr7r5D2bPlcXbSY4JVzVachp77JyA5OFgpoMjw+5alfeap+5lzpCE6YWk5D1gSkNQtiQuABwcE7a0JQkgUtMuSh6F6k9Klw2EDFQZRkedGZ3f42JGqa17bXsbtZwADoZ\\\\\\\/+BVqGDTtRFR8E5UFqvr8Wne7jlQt6jwiAkJJ5\\\\\\\/5Hm4CL0o3oWp6XNwcqtAYpgTWDZtEyPRPq2C9RpQMVaLkQyQhWo7+VJ0knN8kszQqpxKTG7lvNit+Gj\\\\\\\/\\\\\\\/UPfCA9LB7hEGJWA40RsSSZVdOSkF1Ig2MLv0cBKXIzt2CheH5mwxNHYov28jtJ3XzRd5LVXcLM9yjzQqSNSDehT08Z7OSQc5vN5kxTKCovzsIt4YxXtjgx2gioOkqAwqmW3GcZzM704tbXH0vyB+MHm+WH9BOQRhglVKoC7Lwkdmnb4gImyVJ0lJfB+gN4Ho8Y4vhP0Tg49btNQrWjcz20SKHr46mw6E8gNF8werIR2HC\\\\\\\/wGcwYinE+fTMDkMwOOUiHberQgYZwyWNJOgC+srql0M9xjVHvjjQ06aOdOIgJFH2B\\\\\\\/QqsE3pIZqL0Mf5eKNC1ADinQkycaeBgZVD6SMitVFu3OHYyAniq5l27D+z0g6dOpG7bDRaWMop6Z\\\\\\\/1QoTa7EFzK6Z7Zob+X8UOQQOunANQsjCDHLXzIqhKHkBAksNy7E\\\\\\\/F5WRETcDLOvgrp60Ihok\\\\\\\/TUR\\\\\\\/HCuL04vLyskrH2w3KPnAsFDoYhdtjgx1V2TfMtYdonIT90kxKymiOZLE85UURjdV3dBf8ujqeDY+vglow73tx1mkpcCvLeicQBJTrD\\\\\\\/qKvhLMgnOs2NYLV4xByDOMUjJEDIWf\\\\\\\/+iLfrqtLSdK9P7WwSVzH9iRAKbDwqllatt6CP3XXMOBsVzU3rEbit\\\\\\\/AlbHZht7cwRD7EQwKEZVR9NkLIVPOczSIzTwaQJHfF87LgNjeF1PIhsCIbOlx5LirGN6oCZhIcidXXAns\\\\\\\/3taVoLLvlqU4qWP2+LG8gx7Op069r\\\\\\\/JWTjpv\\\\\\\/a+AhhwWQTyUzNmVhJWzlRiCQ1NwcTfrhXB9nnAFnXf2w0Y1bzSMFV30BokWoippMI1Kro0E2oUxOIVEuGYaYLWOGCMBayXSRAKRfXFXr0t4xkfGLKuvPxuXgZaC2F9j\\\\\\\/MRLCmadrFphVgMXjVrX2uAt1\\\\\\\/Q8P\\\\\\\/JyG3sN1mme4nGPDT7QwFLrVjyUaZZwmoeZiLX8cNufWqyvjoNFYMrS1Z8OtH70kNnN1\\\\\\\/gPXQprIWAe5DR23hkv2zZ5C\\\\\\\/QsI4ggoaVoWqYtDe+RltRWzr6jyKGrFu96FB4U9CXU66GIhNxCacxGZp1PZJVh5kvAUNcYyLGzX1HLJg+ZNVEYaAWtwzo3k9O34ARxwzVKB8cuxggcXODRI8CKtQeHG\\\\\\\/bVmZ1k15UD1WSyf9rraaAyh6sDViTh8vRJ46q8hd24MOZiqwBI\\\\\\\/nLqYTgwjlCtGf7UYf3jddhWWDK3TiUUb1XSysW1YRissXYVKAdbXLPEPwk+yPDzbE3cAMV6s\\\\\\\/zH9h8LNsQ7+en5ie5BiS4TEnW\\\\\\\/7SipADtKFRxeAIoDLWu+XlP6G3Av0Omw0qJ7rYNtg4ycyw2ful2\\\\\\\/Mj96FPOa1KEnz+OLD+cBunsT+b4xoWgGQp9\\\\\\\/owazsgs2ZjE5ExrTqCGE3ucsLYiodVAU87bpX5oVavQkagC1BQqNZt52+zieGb05+\\\\\\\/j+ngHTTwvHhK+fwZQLaDGyVUtAwr+kCyfjI4MkGcI7PrxQu0FlZUfCgfzouPx919H691bfKv3Xpyf\\\\\\\/Ao0oaPbO9oqFKQaYUKdeIWyGDARZ4lU\\\\\\\/8OsFPJADG43xNQN90UJXQdjibgb9z3qYt6R9N1dS5JKm9garLoX8qyb42eXB5EupPguAl6\\\\\\\/Nai0n0oc\\\\\\\/lWxPRr9zBFHdZdIV42DbtI\\\\\\\/MEJpCCjiEF9\\\\\\\/eMD\\\\\\\/1hO\\\\\\\/FPYYFseBRHGcoV2u\\\\\\\/bLAwlS1\\\\\\\/eZhLlkOkpbfQGIiNwM+iJ9e5I0mgjWgzW80WQ7TImKB00aJLtA4pX9jAJGwHjdcP+7029d+Msd4zkUeiGUo4vHv4FdLu73+qw2YTcCSYP9GDF3DMxxmISG1KgWhewtjccjkqlQSpdFNVO7ZlR8PUQPIwCyQA4ujAQIQu7wfT4b4omtWyE0jHg1RhMG6IwLPqa4SXwQQ3a03KfcxTZ83MolyG92eWcJVo5dPo7oqbh3jkj8hrAumLbwMY0hYW1+2qLBPgTGzUJyU2265kWm9Z91A505e2+M0E\\\\\\\/KOygL+zlqr8HPtKgNlQqrxx7QrLgjMAwFPJnDF3CW3PorbbPdWC\\\\\\\/PrAmPtjxZ3eLNfeUXaJmxmLtXO5CvWbiar1jEHMYNRKOJI\\\\\\\/s5xLxN4HnmrD61vBItXprQ9116tK+lEfg\\\\\\\/CyDdlNTJsf\\\\\\\/Pe1HuLuAaZnbSkPEeKokaPF5LFEAKCDOFQ8Cx7tF\\\\\\\/+om8qvYNpR1eyu5QRZUEDyn3c8NhSBKFfFKTHc0lljI9g75RiSq+rxXzytAt5SCeINPR2DplFo7Zus1vccRSIzVEZCvMG0IF28+sxU01d\\\\\\\/0VG4Z97MR70KpCZEc+P6HSmWrQjjlhk0V6QOjvXaLyBTxQZuERyKx\\\\\\\/cSuw4EVwngoXaOjlGgKpGe\\\\\\\/4B2hN7jvPhGZoafBIy3uPGj9b4JJYdLiT4ZWhO80C4isPy7malKWK\\\\\\\/\\\\\\\/faGV3Vxkat80lCDouB7irF9dImiyRSEGXexgJi+wyZXRlYYgC\\\\\\\/aaul6uPotaBGj7JNzb+feTTzkgQmSGARrEPX2xR+H5CwTJf3UQ6wG7Ho2tgZ31qF66fdj9PKC2pAxlaG4fRJWTt6CGtLsZ8s5LRKJLNuHfW8D0Z57fRBxJET5HaDGt8z983ADcT\\\\\\\/lXzawaOLHb4pMM2H138WvOk312SpQb4Tyx+AIMTzdHdQUqqrEILXcKR6WYRrEx9bygTUnFSm5FAvuaCXBZ2m6YQcjRK5uxuqUDnWs5Vz8qQToJH+q6kcKELijiqaXf8xDQxW9mV2vVRm6jdH4LC47Q47D7rkqSe48sMr1Z4F8B+\\\\\\\/2rbAOX+Noi5QEyTyL8v54RTrg6xc47adKVJbZBgcU7gcL99dUyzel1vUck0o11Zt35S9qw42oxSvDGnAHk2ildK5rYl9cyahMyauvp8j8VFdHyIdE+RPABZACvDi4xDKfFsgEFwX68hXhw\\\\\\\/9LnSeHB9K4doIojUmM\\\\\\\/iL4sNZoiZf2lBmwKPpXrSJjZMCLf5mOx9kq3YBfZwK6r\\\\\\\/Uiu7CGcE7R4WqwpEpvq3St5xOJHDl6gchWI5ODUDjo3TtT6GU+Fuqy6ruoT5EiRUeFmFVxNeKj7hW95pOKgkBOVichhskNCiJirFMbjIDLXMRG22v2XGKd4ktHhhV2Imvur7HFk+MYzq4rUzWoKdto0ZyTiQnnMVhAAPsfhzgKSZWe17jAp6tKwkvVLt1RLcGfH5YSxemXcVbFaSrmC\\\\\\\/6w2R0btZpXeMjksWd6VTtVZsYYYPvLHVXLTg0998t+ghVBM8gMdovPp\\\\\\\/pf47m7+Y1vFWNHAKwZy3pbWUYFBy7SfGviMdsLRrcaf3kF1k\\\\\\\/l8VsIG47A3u3r3Vj6U9GfhcbK4RPr1pYg+H0JWyxq2uAM5lDyhXwMW\\\\\\\/j+oRJP4FJmEdp8XCth4VC9URelshEX9ecBJBLl7+UNRxhtPLHalDAJX35U6S8IcXm685jaGr\\\\\\\/roSEPh743K9ZkmCX7VFK2VCE5hRdH9GGxyKT5Lne++hiN8u0mcPkyLI7niO1Jgp8ojyl8CTFm5b3dW3kb4YqfXfSpERXONOKzwBoIgdysOlHGBOTo5eE6t+RZUQ30eHJqNsi7h9Qv6vbwGz7vDzF1o4V4zEt+w3eI2rp3NLbryxIRoPlcdLi7vzgr1pJEO+d5b1M4np2qOxS4DsuyKYYQGRWrzN\\\\\\\/R3\\\\\\\/bQn5OX0u3Hgv\\\\\\\/Ntp4ahnk8NhMLxDRjuEaed1XzGnhhUT4GrZcLRROYce+x1OTRicRfpi3eMlo77JQMx7FND7wEJUSSngTUT6PYQn4kuW9ek98pWZa5bXUKA7Yskawa\\\\\\\/\\\\\\\/v09BzWr1cWDz6an5Xtuhe+7TctonXteVeUFBhGxT\\\\\\\/rv\\\\\\\/Mk4\\\\\\\/STlW8IAiUDDrlfLUMxpa6vn57ZTogn4LgjNdrBBK0+LML7I6FdYtGhNMdUI5Ib6QLwyr5DggrwMQ1e2+uALh5l99FK+1kSjM5gNAplsqEDiHAtPH3koU1ot+f7rQuNfEaHF084COLG1bU88syao0UvISj2\\\\\\\/Y2GVvXE2zOBaLWOTk6Z1Mp6aRo3F65V2KPpK4FcF3K4GiAycwpEaFsY9Q6927o4WF+yNwy4Ee+0PXkCV0uSEtGm61sg6Z25I59zai6PpfKvQU89aorzbgRDX7lQVxQhdZcMO9kVNOxL1XHCoSDlpY3J1Rc7ixgy969M+xVkH1yThevXEvB\\\\\\\/4Pq+eBIyU5Xzho1LmjICs1ukR\\\\\\\/8GaZLhxiFTVOVukOovQsL1qKZxB21rXtzFZLmuRYnWe1gaf93a78ZEoPfR8jwejQKLy24If9lzSd38V095AiWib4KJPuAzBTSV5U3lgKRT7EHaE8sX6Cas6qugUngkcm0zricOF3y5jF3N6f\\\\\\\/CzmeaciFIRb3VbPzQ87fHNNnJsm2+uqA8B5RousCYEuRaWZ9sF5oGWzfVMGpeYCy2V6o+WKngYT9yLcMVYcCkaKv2ZijIAgQWtqevvquyAx6DE2n+H8tw5x\\\\\\\/53zUdllwIIpWzgHbpFPB+w\\\\\\\/J2NC256QpDQBC024M0ULpbaLDHLvL1RCr80JGdu6Ttjy6LVPCtx+Hv9+ncsjLI3scGoqjU7buL9u0JP7GcLVowMN4LPUCx3DKeIXmDuLMOZmvZxHA9u6MHl6uwuME171xN3sUSnBLLNFpbKSfdBqttx2cvHOcndyxp4XqI7s\\\\\\\/OI9Sn3ieduCACCOxW6NXEvDMhJ3+XAXZl9Oz\\\\\\\/PjOntxIVW3Fx50mpy+2clUbkExq255SgvqP3cxiC9hLu+5NTOXjVoPRGaIRCBUM44U7HCSwiA4Lp7FnuXnbBlDuxGnKFcg6+DlSE6q2FhG640xKAVsbQP1lyKKGBdX9xV\\\\\\\/HhVKTk5Eao8jIlkxr+1\\\\\\\/dBN0NgiQ5tESFtcHRBtF5dMmK1B62kedQGLoauD1XuRkachulpA7k2ePyYDXeKAOglz+wiZS45suKKuCDlyjBsaPNLyLfLT\\\\\\\/bgKhSrXMaGvWBZCOo9hG1BJEY2IaR2Dp+hvMgTTsZ9qU3mFHLyo6hTZpSCY\\\\\\\/Wvxxx47XVHS\\\\\\\/rP34BLVOMW0Y3E5M8nRYmQWva\\\\\\\/ijvSkvk9KiwFvPzFHcfJv4WYBlV6qbtv6yhiX5VEkat\\\\\\\/tGQDnL0oziVQmeU7cfe3gLMy1MKwBHCR6LynP2aHgaQkA0I05XDjn264zlHDuv1s\\\\\\\/bR2RRKrGH13JB8BzbwSiCUbKxaDqYN4Op+SReHdYL9dDruUTRrVOxv447U6tgMTqnkc9G6R1BKLqz6DTyObH0f7sPTUY8\\\\\\\/X5akBJ1TLL9tul+6vBk3VTcQqxX+FD2WmLA\\\\\\\/HMM4byBROoOxX8dG95PKpSBFPLc4+QQSGOALP2rLJjn\\\\\\\/49yEjNLqhtpY0BsfK5mjofwOB4kLyQ3xHey7Y4+oIih\\\\\\\/kdizfzyc\\\\\\\/sk9KBv1+osKQGwefYDO1+q1wwbCCfoEuHOUNiuRUXIsJUh\\\\\\\/XcoAGnl\\\\\\\/IEMiZq+jTxtLN7ZRbNNcWPexfquoX8L7cusuVPrgdhbNaYSdOSAJYQqrrqkjAuc\\\\\\\/lx3+bfdF9PZel+zw37v2PYoJ4RPhE\\\\\\\/BaRPzsuemdUdtc80kR0BgfLSRUsfkMBARWv3gFjvRf5zQxg6iyRatNJZslUaW\\\\\\\/tPNpaS\\\\\\\/2qkPBazIy91SILWsQXi6MN2WaVByiS9Ibu2WYNkbCchz3urjjBQxb1OYb0w9iKo61q+qKUFz66oftnXTf7sV1RZ6ZxbkHRWh3yOuNVRslfZQKwovpommFeMtfXglMVkJsbDL5eU5fSG\\\\\\\/oOBa3ujN90V6dxmhdr2inK0o41E2ZfcsXI\\\\\\\/1dKVTrbCcacnoLXv7lXUKqBi5qOS9CDDQlu0B1jZvG5qC0y\\\\\\\/E15q3n3HqcaVr0\\\\\\\/RU\\\\\\\/r5sbYQPY9tIIQbNyN5q51IuFgBWjYBUi\\\\\\\/7Y71ShrePNgK8iZNxEZuFqN6qwHZ4Lyg6NKHTCVUrnb\\\\\\\/IDmQXCETfWp891hVeeiJZRkhrHhwkVOphl3EvkVjjdk3pNwPh0HKeTDxpGcv6xw\\\\\\\/LTVQmF30JzYJE8vTTcbYD4iZ1lUPUIRw0u\\\\\\\/rBceN1w0zNX2iUwtMm\\\\\\\/Y98qca5bnJhWYtGdL5j46X78brfne1wXStK14LmTDegZkHNjHoU\\\\\\\/9pZGEpi2IiaxBOgduHfAI2zYuUrJ1OfgU0UeNX1f7NToISxCjmoaNKkuHZOmDl9ma8TSh4DmMNhP9lUIwou6aQW5voiXtfZqliJRhOA93qc3cwcZsTs\\\\\\\/FQFQEfsKdYmq7He0t30t+GQApZm0gTohToORyXPK0ZaA+WUm1ZZ0f9DBgHQrGrdDuXm3CLWLoQyV56EoLuomyMq2FtkkX+lSOukG8SM\\\\\\\/ZIEzXQ7lJMXODX4Xf8IDPyA2Oaiq4aUjJrSLBNAvdpDKTb5ymukiPvn8aNUk2+8A9MJ3AMF0wvG8LGx8hMQqT3Dm065sOPdCR9zcPTyBTbUgan0HAbWh24vTOOl0bqzgcZQsep57yaQWTjHir2Crqb7hIRlxN7IDLVcxrhz0E8s3L3KgAyHGuQ39Q8srNyxEQa8mk0Jo5W4W4Vg38GW47TZ7\\\\\\\/p1KsLWepq74JsvAAaTu+WGJOWpMtwYtZj0X8r4DllhAvPe8wKm7RDye7bOQnuwiiN\\\\\\\/+vijDNpcZEPP3wp4szUM76d16lVpK42Zll9MLdrTVSHAfnrfNVEIL1XDt+gvz\\\\\\\/V+XC29PheD94qgjqsXVd5ikPrEm87ONqi1sq+tawe4mW2MOxO1bbQer7UOT+Ng9QjOs1n1tSKlEQl71qmlWVNCP\\\\\\\/uJIadV9Es8ZaOKJdOtlgKwePPZvymOKqKCSw6\\\\\\\/E9v+qvZavhLfx2sSY9qnYCoaiG25zFcouHpPdp8bJtKgHtGcX0toVjh4NiX5SWBGRfrgk0KoiDwg2361qopBQX6VX2uk93+Dr60GtLzBym9pluCCQmHdX6Mu9VptfXRPAh70jVT+KnwRD2gzFahqPVQJJYLGBqZ7UGDuHyRNRhtU52Z4bB3pNeAOlwpUb5oCgpt1nSBwoQGLbT9J95iURzT3Fsyc02DWPLgl0+KImmx2SZJrTvc3AffLntt8YlOuEIzU\\\\\\\/Gx83OzLZXCF8CvliiHW2Ro4HGKc6I82UsHUaXM8Xw2eGHX6rzNjJI75ohF79fY+Psviea6V4ajqW9Bl6ZyZmx4NNZByGp8k9a9L0t+aJreQetD7hPbRCKFzjFL0BVfNosxo21XcYOfxF1e6G6LsYXq+BzWitzwfeqS35UHlGXKrs6LMaiTURioyf9fhOS+iDK9jnbL1x6x6DXF7dihGk34uuMYkAIj9T0qV1ihROMhRp\\\\\\\/zLQCGkopqIp1XbarRiwIWhd+SV8YqIj1WQkz6ahhPKV8AhoERqGh8g9ZPEDHsjVPF3lAytg\\\\\\\/3hRrA2r+G8E6vuAE1MnciefIRXcXspwP7MezW6k\\\\\\\/OdBNbzFqexxFWtA07I7FYQgT1\\\\\\\/SqT5TypaeMlSDqL3olN7NtqYGGPAiP\\\\\\\/k6vfKkTsFChkTGG\\\\\\\/Mg0COtFaQHo2Lf7J5EW57N0M\\\\\\\/AjmPC601zjZ1rMhrDzyDAxhXcocp2kb1sQ6r5GufpVzI6EXeMab9iAIw3vg5l7N+vd+xNTs6EQeoW5Tb0w55GeKWUeVDvV1DVIqmYpRfBcAaJiu9sO1Xzlj0siBpLN4zrX4xfWaTTuYkcGJ7Qr26phDHo5e725SZJjDOtIkDdKj2KkvQ6DEng6XE236GNx6bz6oa3i747dL6+DG96CFUcDYAtm7wA5yrQKd\\\\\\\/SmMk62Un14+oM3bcffhR9TFU08KoI8Er5NCsGo1Nvk+8QFu4JwhQBYpmR23mA4VrKvXiU0qiMM6OLvLbwIGvJPq2SvZWBq8qL+L5MO3D8KrmL4HMl9sD9CLHIL8RiSPIX32bK1ucrLcIxF4OD0fM7pQl9DfBmT9tgOSky7czAn3o\\\\\\\/lyYSvsVtbm4nXE5PYxd01+L8xi1xTmP7UJJGzGCFxCWc+JC9Djew5WxZw\\\\\\\/xsT4FNpcHtF2PEoZfE1qmIln1I2eza4W7BLBjK6+EkV+WEMO8yxSoP9ykrs2Hj4yIES7wJsC5cbzs6f\\\\\\\/SJHAguj+jEmnHkSaVCgU0lPtXRc2KOtxJwe3MmG1\\\\\\\/qCfFSjYF0t7EVKDB1E\\\\\\\/B53CMqzEaxYPNn2\\\\\\\/fuaOzu8sdVF+c63uJR2DgECLvAVndf4B6cA4q5uXrpbuE0K0pcmeO4wdh5q3vVWjimnfOSa+D9qW+1Ee5tUvSYFkKgtMDZPi1IL0Lk5BdN8Urf0yMPlF8KDlU7amQ8sWwl\\\\\\\/ZKx5IvpmiMvGEe5GVgZh4ZoN0jTooYiBoR+fWtlBd2tCQx3gW5Qf+bnPjaEtWJzqJtp59ojW679dunyOacvfsjzkRKtGkDEtjbF8+f1E7rPM08cR7WLi3oCX3jBYJ3EkdZoMOR7HMKUMtMAc2ebNydVHjGTioc4jQl64BVb9LiiQg5wgG96aGpYNx65OpuNX+JRmxe+Z\\\\\\\/pTENyKha5HjdNXqGI9GDKlfBIHSq8OVnqgCEtAHmypcpqj\\\\\\\/NO9zVzgmUfWmqbEUzru1Afbb2IKhLfpp8vumv9EvaiZePMDvwfCfRD5LVI9VymD4DMFMkDh932JJQcLdIuPv3hLE2YcQAISmC4xmFjnQN8NKFeocsDh7XXcFvCQs0c+nWQQyzXsZJQC\\\\\\\/kiUPeS+GxsoCTwpMzTrWWrJCpxPEigehCMuKnvbEykfuWPKMY7Tbsd2TOZm0RE9v5Sw7mAuh0SxOjSQ54heq5nnK6DXOWUlGoRH956s4xbcWS6wKTeRL5tNVVJS1pTxWxMlLlX3tCdaOTzYwEazdUDybtrlyi5cdWPjytDY30pxwt\\\\\\\/5yUQlhheGp3dfF9BxGJppwBgAd+VYhdEFajlE22KWkMqQL6MlpKVw4Ga7o0kralq+naX3EWYsC3lHXeAJWCaP1DmJOVrahB39oiUPAiKYzmUqsTBH6JOlvWllJk1yagtTmxux2urpP33tptSBpvQdEu6BIV+8gIL72NPdxMXr+NbdAhJRYeyI7JrQqZidpRjYNOn4cb6YHWwYrvJRwPQPn0Mk+ArNn2j8NRou5ixPabhBTJ9sfsXqd2jpll\\\\\\\/hg1Zw2qckAAPe8sIEH3LkGXH+SGXsaIMyoKbW+Sppok+C2LpSQZ16nOlImAvyhkpB7pEqYPjboRh3BLvqMXxnELQzMhBWMKGw0WhuqaKO0INd9Ny1MT0W4RKq10Xc3Ef3qSZ\\\\\\\/ipug9w8CnmpY5Xpt\\\\\\\/6\\\\\\\/zRgrLbW68+G793DKH\\\\\\\/S2d1XNnOk506KvWUm3Ur25d057N\\\\\\\/7wjbxGR3TDGe7Q8yHL6FEIEqdncFfcza5sSD2JyYv+926QC9qfk9oMiPN\\\\\\\/QvZExw5AuIB8ED8EtJ0cSxR+IO1PWoc5wELIfsfWqLHzHyMzlLGgp27UbRA3xW38gHhNGh4nA0P11mHhOEaoGZIK4PuKZ2STxN9ka2PsPRVeOW8c+gP9y+dhbC3fQpoJQ1zF49WTWKlBzXDm5iua4rlqo\\\\\\\/bPgU1GsLCr8q1JfkTBS28dzLh1MEMOT16KTvkUK9Ox7eZu70GLPEcmJfxvQ\\\\\\\/TRQpmX0e3uffenFEOcTiXHKSe72mvdaHXIMtK4xGU+\\\\\\\/u4ZiKkxv3A7GUB89wMimTuc+VX8Us30pwL3XfMIKHYg0pvmljZYbS64rELFIyoynwKm+D3c\\\\\\\/+QVFdynfxaF5T0P4pkNvqQaS+\\\\\\\/gNZSCgJRqcq92wzgJ6EcRY1aGfOFCmItUIh6Eqbgf\\\\\\\/OCtd+wdinz5pWpcHoUdKO8NB3HP0JGg6\\\\\\\/Kpx93TJm51hz6V6y4Ku6qZmNMsxje8Mm4LmW9wNIaqh6He\\\\\\\/TgQ8hSufZjJ5HuABvvUIpuLntjD\\\\\\\/TDgM1jDKwnYjLdDSLft+YW0JMcuNaQ0hqDI4qX\\\\\\\/i20LAiIywFWPShGkkw1ZlC6M7bdTsBfBFXbJPmvOZtIiYT0QdlOWwn2\\\\\\\/c3kB0vlgT9SRhhVF1Ctt9woJgaFSgGD+0zVSgG9SOQ266jYTn\\\\\\\/qL5n0w3l7syVja6ud7xj7FTaZgqT2SK\\\\\\\/fo7GvZoz6mMXYrUxf1PIlGc7qQmxwY6WqRH+Rh0EhqgXnCPFGLre44WcbUxKxpap1oqbCbAOnhZtUhtUCQBSFJOREZ79XKNSNdVKZ08HybCt1B+H0QikMiUxHOr01TANgTlZJWfBy0WiQsKHbV6YYVJJJBr9\\\\\\\/+iNXrzvfCpJ8tG4rrzCaAWzcNojMPejtOowVloW7zaIAdrtyfu3u5zifBMu78ofIQYn4NOghsw3SqZhhrTEW+VH2+baS2DSgW0\\\\\\\/jIhSsok5MBBOmUcJ9ryGq61Y\\\\\\\/RNEMpX2gU\\\\\\\/lqBl1IMe2Vkt84897X230oXyCBi8HiE8HyaItCT+nEfwf1r+HNk9529zudgufRo4Mvf9StmvlhW+4Pa8zKaPCUKGMwSI5TqP0o6tPsJdtDAhSXiS2\\\\\\\/GNLg1HMnkdMqidut55itmxEWYRcZ+U29bHEnKjnahTunnccmRWs9lTtOH4BgbnESMFjeqzvu9DlXWtBJeFCvsMROGf0M8DIJlktPD5TUu+eXHHj6Fo7kCZ0pyz4A6TjW4Pav8QRhcJUV04t62NRRyDU7zAFHLva4e8uBiCVC\\\\\\\/1J22jmDIuZj98\\\\\\\/++kiDsdZ0Y+qyGCYzl+2UAF8zPTYZyXLQvHe1rFYPPtHTHl+Dv9y\\\\\\\/pMv4tlCZIGJcwOLaiQVzGvJjWm\\\\\\\/M\\\\\\\/9eaVqMEXIq2eODxPBe+piJYW3Xad5WqijGAC\\\\\\\/wZA9986sTpI0HKBTcF1lJ4Y7mHM7DGzJAZzf7NT0ni9EriWn+7B4kn3KX5ImpXEovoUlA5bgcV69HSX0zzzft4UBjL\\\\\\\/JLMYXVKX91oNC20vx5hgz485aUjRLsb9WQjZe0b5gzKQACQWDQxPv9ron\\\\\\\/L\\\\\\\/sr3ZTtOAG1uNUZ5pAqopMtanz5\\\\\\\/bhgeLq3FNJsaCZ32XaZFv7yHnyuoFVQB0hu9jzSo+yKd0KRwrpH6ArwvF5\\\\\\\/RiET216rFc4ssVgMK6OEiPzZQF\\\\\\\/Zrjid80d06nNjfEalmvPHi+pzH\\\\\\\/JmmbjMSW\\\\\\\/NxqYq8ooRllY9xh3IrIcBZlJVM6zT3aQ77EAjmSZvYo6K10EcAhfQzjxug6sWF0Ua5M2PjwUIUQy4bIhEALkZE7U0BLvmzW42+PBDakPrsh8mXmVupRLo94csKwtPvO0OGtYF5eXiPaH5PMSGDX1BBldBZJFaOrlFUI5a8vsqqAf+oEtt6xddaebV59pOt2sYvtqatWjOWnnc+LLyYX6Uqbb0xuHLX50T8bpKXfhnCaAYpfe2\\\\\\\/u0mVH6S9TmTsB8G1C+mcvhn999v7G35UkM9wtHlESONuHzWKj\\\\\\\/YGBMiN9mr9w7uVwXohIeX7lYqNG9Fubs\\\\\\\/gxsPK0UXYIB3pGTQXkR7n\\\\\\\/nhXkjLOdKzE\\\\\\\/1Wth+97etwhq3qZqcDzzEeWtNHsHOGG0fjbyJm4rr5I0R9EQT2tycR7XZtl73iK\\\\\\\/baMq9+QmB08sPEvjJXxDH6raCvlGNUj5uGKlKBactzVaGUH8G\\\\\\\/cxdoltrGT++\\\\\\\/tktpY8+gFbEa41xSY9oF63FqJq0t8\\\\\\\/JirBupJ7\\\\\\\/LO4S5JZN6yQdg5bqyxbpX\\\\\\\/8yDJ8wE\\\\\\\/l+2rDdVMAw\\\\\\\/1SVMALkBuZ7NfloTBdTavTCumRi5A2rD8NKHulaAvLBa2AvjoqrxXI31U3lFznwi9Rew\\\\\\\/+S8oyzRR0H6\\\\\\\/7eXdOoy2OQ2Tw86FqmwcV3cu3tJhx5anvaUu51XD7MTr6XSS9wnGCKqPTFzZtSvge+FolJszbF7NGWocI3B77scddP731ZIMfSVqhA2FQaCXL8t3TXfhZTXQzQToKx1k5Uqr0wp+OsRTS4s4K6k21nt3fP406uH\\\\\\\/cR5iiSK9qiACy5DqF4c93KB49PzEgEja0na5n22eCfF0LLa4fpNPNBFroEu7+j6HMLMVZxGnpbr+PN4SuLQROBvw9qvlMSSt7TrH\\\\\\\/S3CmrGPeHTwZHi+yevIWCVWPN88ACekJ\\\\\\\/nDvNtmfaJ+EoRsqwYfwOauUawhhLfqkiGjEgfI9gSGeRE14r+1eY8\\\\\\\/Boy2UjdFfitpiOATdBwlZxiHkApcciBdaAoc8adPTIKqF6WwQXtZANCxzoc6h2szFl7ToULDnOvrF4T3J9sp0OmbTZ\\\\\\\/n36uycNHvTPM3poNy19OqszRnFYquDfqaZsiyOl3NcEJcYR\\\\\\\/oSSMSirtq7NSN\\\\\\\/RZJ1ZFpwgRjPKdhILOEF9UFMjO1mQW6lEHm3Afkrfd33PT28gO4FvZTi+\\\\\\\/3d1TEYI21U\\\\\\\/tR3bGrlUG2tsrM70GerJGuh68zaqmXnQLfcWebuFHNgq2fnZNRYuEauEQuiTXtfJ7BExL9oW+NAB2Is748pKTIAnBj3YUNXgvQjwM6EoSKwU6\\\\\\\/V9t4W5T\\\\\\\/8r4bPAoJ8\\\\\\\/iYUMyj68RBVSPePIQmh8nseWx8qqfpUGyHVDkUy95mfDV8NtbBL8ghL3kpgxKnU6YyHwimRXvcSMn+F9tfNAieGd6ekkXtvBH4gMq9mVbnbjYCpgeox18oxwJptJT6PrCtVyW\\\\\\\/6MKwVV9IFZOSksuVjF0h8Zd6u1nsougY5zgRw1xWOyMYjDg+drFghVBoKEvfuCg8iuEfsjwVnZvfNCPQ4dAKR8ho3cj3c8MmIROeTwYWRAVen6pIj+Ba+E5IOFR1Du0HDWyKH3HMg7jl2TZsfjHbN5h7GRoy6\\\\\\\/PBRuomHOJ1C4ZGSp\\\\\\\/c+3GM2sTNReZpaPscf6r3XmwS9Ptsfy3mik7ac9aCO7OQ6NyhkOqgm7hbsl\\\\\\\/6J9wXvNQKr27BKbMDQ1bN9jcFMTWxLMc8HtF8qarii8jIdAQ\\\\\\\/W4+iUgXMY7UoFeF6VM0qP4+pqYiLDmj9p3d6bFJjCUJ1s7lnynmkVNyyowWe6vxesQ7cpf8QLGP25UYX8vET12TIQpQ8uO9IKVxa6hHcLoOH+hLjD4X5m4IO6Td7MWGZRYtLsP+24eHZxiZeoT4Whv50e4HcX1TLoXQaIOvSq2RSZe5DncBbp9\\\\\\\/wBhXfcDlmpTYimlKdeDRF0F6qZxgwcNhvCJUCfgCMdONSmB+cJNEfUcoGF36NZH08+6SweS0BZsbVOh5E6jGjMnPOanPiShu6ITT9ISlOvGCjFXde3d7XiNiL3A3jG3+kU9ZEf4o3F1MtV\\\\\\\/5+T80YG4GoUTMPNbgwHAvyKUBRDVwBuukuZu+CXCzex36CmUnrxHSdyviE+oo2KMBdrJml96lzhdITGMWNYhDtw2JWlJANO3YMvzOunKdkCCif5VurfmVA9zDY7HCfB0Ds1xJNrxxeWUZu1XdjWMMAwC55hvlvlCvU3WbROMfVZRCIJgVyeWuVhbzBjLZ6bxBDst5hPHf9DKhKieCfCbMKA2ojg2hs4hDim08YskOsZyoytRS+hM9jAmbBcveMsycnyyYAniutAb2MvIgiyeqkCr4MuA512jbAA6+GYxReKf+JEyE\\\\\\\/lNX+8ZtAdiwO9An+LyoTL2byPpFoYLc0qUPeWLK6+DSAgx4sg5y\\\\\\\/q8amQ+chM0BWLYLQLHKWTDFdHEZadDeWd4XNJBODlBm+HZ5l3Y\\\\\\\/ABnna1lqKUByCk77Lj1MFG\\\\\\\/fjfe4wvwpUPKX\\\\\\\/2ZhAa0+Rqgw8sb+lSnGGDiRz8LSwPaX0bW2dTjRWb6sI4PP66BgIaaQn04Pq0BB3WCEY6ZeMoEmPHheeHio96\\\\\\\/yOjEo0mgJM+eNAopILcbGMpOUjk89oRTpyh\\\\\\\/ABRc3y3hJ1CyS\\\\\\\/tVZ7j\\\\\\\/P\\\\\\\/xZWPFy6er2xoWmqj6p9Y6jFglfE2Vi4qV+In5e\\\\\\\/0XN4obqmWlZddFnL545oyLCZWphZxS6hznraGH\\\\\\\/jg9KEaQPTEVOL1ZnLh\\\\\\\/CoySQrFfP4Ld3W0pn2dO3IE99jiGK9Gk7t1ohJB44cj5CkWBRTOovYfqkcaz36cAN3j0Pdgx\\\\\\\/TVP7cqolH5YNbOHhhmuekDJO78q2INIE335P7XEypp4nG8f7EQZg1MxAVy2q8UEVQo8TlwSLSUscnx9H8oPYV4Dgg4EGFf5ekPaO0haE2TX7rygfKWqh6qiV66T8Sr1hveWOFKGF1pwoXfdnxDGUyQF6o1x7XCwket8eCIbWIfXMDiVs3qdpGC7jCloqfE4lMp368YTaNWH9BW+njhCuo\\\\\\\/cOiySxAk5XU7XquLiLU\\\\\\\/Nxs5hzpbWj3evPBxY8y+xBnx35ogcIoE7kYzKPBUyogFPjppCPEr2TuZ3Qz2jZKEgIObZn5acFd\\\\\\\/eiAFYhyXxAQ6nDKR6lu7vkLoyIaSp\\\\\\\/ggvLd2Tk0z0KQxE+i2DmXjLy1tQ04Lky6nJcnxHd4oRZt2pGN6SSnd\\\\\\\/YI1rOdQDgTU27yYHW+fhVFgNuxv+vRCfrlcGaLJ0j+nFw8miXpBM6dZ4n+5QO\\\\\\\/Q\\\\\\\/fdQtGswJf1r2JbQKMvstdHynR50p1Oc6fiZVTvOUy+x6dv6YCDEx3x9a00dPjzXTHOYot0bmc\\\\\\\/hkZoImEoc6y7jGtlETyOuFE\\\\\\\/\\\\\\\/f1cfxhHLhSlvcd6HrUlrD1EixKkcu3kpB5pFK0g+XJJO3dzzVyd3uF0zxPkzwIBfRVpbCkhmD3DaF0U\\\\\\\/KM54FHhYW4sw8yc804Eis3QMvVZco7WyohYCV2z+5TmdHfKc1SUGP9l+zduwe5XP8P6Vu836\\\\\\\/9e28eKWNtaQHqA5takFuQjveELCVoWgaT1sN+ZmCybY1FgPPt0nNA7Xt4fvjUyQP0E7Izk1sHdIikX8nxwZOye64x3PXCPAtFpV6CwzcuZFK3BwZ7yPACzR5yM8YIg2XwIALSDr09j\\\\\\\/XwIw53lsREL3BEL+cgZ3dzaOFrtji+caMmYm98v7XnC5o5RstqQBrUTJUkJ6EFAyS0GlvvkxvLXyzyLdKrya+V8iGqjiwkYMVGrTWlETK72Yt7n+y0lB8i0XvqAd\\\\\\\/vVBfbiX2m0nfBQfOUK5KC9NoriPDpiOYF+gSqtcUX3qaJDSFZJ2V\\\\\\\/ZT+xVAbgSKPFTvR0KHb8yYKuBUNONxICC580YtGnl7171TUnUVBMvymHnim0jvMgsbGBBvgDkiog3poc7lVWUV7V87fcznuu96hmKCDxGnG3wuIIbGgeXcZcEqSn57ckfaA2S\\\\\\\/9I80BDOnvYu4TDZyLTkkg8VZswuFx39ZJxy0CkNdfgE4U72q4yZ6r9bjz\\\\\\\/8hIBqOn0EAxGCWcbowb3zgUW\\\\\\\/iLWwSvkrWjZP\\\\\\\/4IdWrK+6MgamEDxuXxIjpu22H318MSV6Cqp+zACh8UuucXCZVUdVVDf0WwQTeBaw0ed0UIEc2PqL+Oiuja0k2Mp8hoeEQHdf0h+rf9caKCjefIG5ML2PBVqv4rf2Y+QNgNMu3EFEI4qlfGzowDuSa\\\\\\\/mURWGI03\\\\\\\/op8As5z6ZaqIQEoFQtcIm0KrJ3\\\u0022,\\\u0022iv\\\u0022:\\\u0022Q6\\\\\\\/3FG38\\\\\\\/+xAW1TC+sb8ZQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002245a5eb86b3648c1f153e888bfd7a2413ff79418d453a32276d74960f891968af\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022eB4lNbaUoHuETD9fTyZvxT9dN5+s+u4EbUc0Vw2N\\\\\\\/FexHJ4IiHhNbrWMawxsUZOKNMf4Sbv1EWBQqrYLncW2xWI7gq2WQnNuH1chjsG+bUZ\\\\\\\/Hdao+a96xTEHjUQVYIQqEPzbUoUYR3idNMpSMAChfb7n6zR\\\\\\\/5Sn8+w9CzB\\\\\\\/BTfJURlD+ypihnmmHutPa2K2sHQjwYLd\\\\\\\/fF4PTSyiw2GVAD4bMeg3Fj9mffp1I+pw7ToT1Jajkz3yeAj7FF1FqDQOWKK+zedNg8duTp9z0qvpVt7u6A0xZwH9Jat5Js5dAKANLPl9JrfCd+\\\\\\\/tKVLvyimTgj09lWUxkLpTIsmBqElAiHJis6hnyOqEI1cK7n+sWeSD+hWTdIEddnFIzj3iAz6AiuB\\\\\\\/zO8wfoIfdPz9VR5K7VlUrHvFZLgmrNUUFRihXmdSZU2fXItVExGh5emg2odgl7B+tLrnoZRCNwN22baMVnKasjC1SB+zFY4MWEimqSJ5Arr72z2PdkDJ6urN9XFF99vjnM0Vk6+Kygm1D3TgTThWn5lZzoCsj95TVjqRJ5MzQ0yQC4Yb7U2fObI90PM0OQ+gHSawgmp4DDa+0YBT4YboGI1T8aG6w6nCxfHHxgojshyhNW0TZ4BCAE88x3f4m27QnYR3G39jYv3i24etuEHKVygMVpDCjjtV3PTgQ9Dnx8BHk5CZafkFGSvTDzaMgBZGDbCo1EdKe2uHej9Dn\\\\\\\/0bdkQ1BkT5TIG\\\\\\\/B1CRs1W145+QPUAk8xvRGkhqXcnEvlCnwZ\\\\\\\/Gt\\\\\\\/4P5ZZ3Ypv1qysGYuInoBZJuwKi8H2RWQaN0RHRe4r3TNHS9+p7rnbdyPfaV3\\\\\\\/eSUQg34l5tHoBSitrUzhSuBWBFRxIQnGkSbs4GebXsWvSocYSknYiBqCsGD\\\\\\\/BT5KVBjF+\\\\\\\/ECVJ3vzsD9zwc3+cDEDk4wzWqzUGQddJtNGv5x5NmoTX3IPMW\\\\\\\/G\\\\\\\/i6lUjMBRWR76g7HiohuapbCOvYyJAbp\\\\\\\/TEnNyaW1yL\\\\\\\/bz4DopqEwcWZZMOH8qKZVxgrkEIG3NcYUnGjNKg4vvKMFKn2l9wk0lriYUtz4IyteGdSnWo3DIhXDjdw1uKEKBI1ydiEmV2tjwWXvkPDQ4ABc5VGxK4s6hAt+FJ6I6tC9\\\\\\\/nFYBXmH7fb+cnDsZuE8wZm8wVDvhPWfxIPEOFjHXCD5WOdcXIPPuvyRATYkSRNlwyW2O5h5G5HZDvFQX50+Ww3Rc4QfbqlBXo4YCpXm1bwY3VCOu1h09PGGqql4LWXZkhXrRH0Cej52TSwtqeVVXwVdk+tLjZOT5p41HTdF3UDWrcNGUXWBjGBBrh0XyyCSgEfEEvpTHbJuUabHFmN0ZV9wfmp866mx0e5Rm3EjHGxIG0HsQtqXNj0VhbBzfPx5BX2yF1afs2Immn45cm8ccLqW7uITNHidbd7+RSRh7yfgsfCoN9BS9mHgpFRLj0YtZZcN955vGXx1YnOTD+xed8o7NlCrQS2zfH45t\\\\\\\/mhcHMCFMn9oi+MtPRQoWtsqBfMldllj60f8uFVRrFcdijJch4DZhDjGHEfe6zTylPItKX+elJ5XS1Zxh6P4glVo1isqqUFglsOvXHn4L39GciKlUYMr9Tcrlr\\\\\\\/Ve3sO1Kk8yyMX\\\\\\\/Qk5Io\\\\\\\/iE1U64Drs3e20HagGpflCANSBKQMZBgbgg2Bo6VSkz5X9p83Hifc94CSBcj1LqP\\\\\\\/neqTMRextQBWhXJ7FMuTcVV55zXv5I4E2AfkG6ITmdODSn6PnVzJefp6A2bG7T0GtxLTX76KqnMwCMgECzJEenRLvAouxsNi3VZax9a+pc2RKdBGoiuosiFzpcr9SbiZG+9KvMajioY\\\\\\\/SH7otxe8LiD6wb04KmqQcvOVET4fvxagTmYySHNd5vxZDPsD5VlRVNgKnfAnjSwS5T4CwsXAaabcwkdzV5IhVkqkoMQkGKajYGXUcQo6etFq7Q6xmEtjsk\\\\\\\/SBmkDCc52xWyf3TdQO3wlURjahYysktQsMKsrksE2WsTb2nFceoAPtCXhsRYsR9l1dm9P8E=\\\u0022,\\\u0022iv\\\u0022:\\\u0022soq0qauTzkgQfXRSw6YyXw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ef59e18b7aba1f46c4f324d67d9674f842c910d7bdb16fbbce8b66763c2d154c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-484252 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1393" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4iKolxEge0kmAhLk9aaRBPI-37lMTQaianYl_YpiIFk" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1393" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-wZkdugglL3V1aVKH6uoy-Vuj8wg7hMe6PeQTVWhSCRo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/5/1385" title="Accurate Classification of Protein Subcellular Localization from High-Throughput Microscopy Images Using Deep Learning" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/5/1405" title="A New Reference Genome Assembly for the Microcrustacean Daphnia pulex" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484252" id="node484252" data-pisa="ggg;7/5/1393" data-pisa-master="ggg;g3.117.039008" data-apath="/ggg/7/5/1393.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genotype Calling from Population-Genomic Sequencing Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Takahiro Maruki</span> and <span class="highwire-citation-author" data-delta="1">Michael Lynch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1393-1404;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.039008 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484252" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Takahiro Maruki </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Takahiro%2BMaruki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maruki%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATakahiro%2BMaruki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tmaruki@indiana.edu" class="" data-icon-position="" data-hide-link-title="0">tmaruki@indiana.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michael Lynch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BLynch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lynch%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BLynch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/5/1393" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:484252" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:484252/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1393.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:484252" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:484252/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1393.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:484252" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:484252/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1393.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:484252" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:484252/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/5/1393.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:484252" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:484252/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genotype calling plays important roles in population-genomic studies, which have been greatly accelerated by sequencing technologies. To take full advantage of the resultant information, we have developed maximum-likelihood (ML) methods for calling genotypes from high-throughput sequencing data. As the statistical uncertainties associated with sequencing data depend on depths of coverage, we have developed two types of genotype callers. One approach is appropriate for low-coverage sequencing data, and incorporates population-level information on genotype frequencies and error rates pre-estimated by an ML method. Performance evaluation using computer simulations and human data shows that the proposed framework yields less biased estimates of allele frequencies and more accurate genotype calls than current widely used methods. Another type of genotype caller applies to high-coverage sequencing data, requires no prior genotype-frequency estimates, and makes no assumption on the number of alleles at a polymorphic site. Using computer simulations, we determine the depth of coverage necessary to accurately characterize polymorphisms using this second method. We applied the proposed method to high-coverage (mean 18×) sequencing data of 83 clones from a population of <em>Daphnia pulex</em>. The results show that the proposed method enables conservative and reasonably powerful detection of polymorphisms with arbitrary numbers of alleles. We have extended the proposed method to the analysis of genomic data for polyploid organisms, showing that calling accurate polyploid genotypes requires much higher coverage than diploid genotypes.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genotype-call" class="hw-term hw-article-keyword hw-article-keyword-genotype-call" rel="nofollow">genotype call</a></li><li class="kwd"><a href="/keyword/polymorphism" class="hw-term hw-article-keyword hw-article-keyword-polymorphism" rel="nofollow">polymorphism</a></li><li class="kwd"><a href="/keyword/population-genomics" class="hw-term hw-article-keyword hw-article-keyword-population-genomics" rel="nofollow">population genomics</a></li></ul><p id="p-3">When we carry out population-genomic analyses, identifying individual genotypes is often necessary. For example, in order to identify putative loci associated with a phenotype in genome-wide association studies, calling genotypes is necessary to find which allele each individual carries at each locus. Moreover, calling individual genotypes is a first step in some population-genetic analyses. For example, many statistical methods for haplotype phasing (<em>e.g.</em>, <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Scheet and Stephens 2006</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Browning and Browning 2007</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Li <em>et al.</em> 2010</a>) start from genotype calls at each SNP site. In addition, when accurate genotypes are called at each SNP site, traditional statistical methods, including the four-gamete test (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hudson and Kaplan 1985</a>) and composite disequilibrium measures (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cockerham and Weir 1977</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Weir 1996</a>), can be used to examine the pattern of linkage disequilibrium.</p><p id="p-4">Despite the advantages, some difficulties are associated with high-throughput sequencing technologies. One of the main difficulties is the high sequencing error rates, which typically range from 0.001 to 0.01 per read per site with commonly used sequencing platforms (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Glenn 2011</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Quail <em>et al.</em> 2012</a>). Second, because sequencing occurs randomly among sites, individuals, and chromosomes in diploid organisms, depths of coverage are variable at all levels. As a result, when depths of coverage are low, there are often missing data, which introduces biases in subsequent population-genetic analyses unless they are statistically accounted for (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Pool <em>et al.</em> 2010</a>).</p><p id="p-5">To call genotypes from high-throughput sequencing data, many statistical methods have been recently developed (<em>e.g.</em>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Li <em>et al.</em> 2008</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">2009b</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hohenlohe <em>et al.</em> 2010</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Martin <em>et al.</em> 2010</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">McKenna <em>et al.</em> 2010</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Catchen <em>et al.</em> 2011</a>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">2013</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">DePristo <em>et al.</em> 2011</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Li 2011</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Nielsen <em>et al.</em> 2012</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Vieira <em>et al.</em> 2013</a>). The performance of the widely used genotype callers in population-genomic analyses is not well understood, especially when the population deviates from the Hardy–Weinberg equilibrium (HWE). Recent studies (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kim <em>et al.</em> 2011</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Han <em>et al.</em> 2014</a>) found that allele frequencies estimated directly from the sequence reads are unbiased, whereas those estimated via genotype calling are biased when depths of coverage are low. These studies assumed a population in HWE. <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Vieira <em>et al.</em> (2013)</a> recently showed that the performance of genotype calling can be improved by first estimating inbreeding coefficients from the sequencing data, and then calling genotypes incorporating the information on estimated inbreeding coefficients. Unfortunately, their method is applicable only when the inbreeding coefficients are non-negative (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Maruki and Lynch 2015</a>), and does not always take full advantage of the population-level information. Negative inbreeding coefficients are common in some organisms, including asexual aphids (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Delmotte <em>et al.</em> 2002</a>), <em>Daphnia</em> in permanent ponds/lakes (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hebert 1978</a>), fruit bats (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Storz <em>et al.</em> 2001</a>), partially inbreeding plant species (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Brown 1979</a>), Laysan finches (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Tarr <em>et al.</em> 1998</a>), prairie dogs (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Foltz and Hoogland 1983</a>), rhesus monkeys (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Melnick 1987</a>), and water voles (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aars <em>et al.</em> 2006</a>). Furthermore, some regions under balancing selection may contain excess heterozygotes and therefore show negative inbreeding coefficients (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Black and Salzano 1981</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Markow <em>et al.</em> 1993</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hedrick 1998</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Black <em>et al.</em> 2001</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Ferreira and Amos 2006</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Tollenaere <em>et al.</em> 2008</a>).</p><p id="p-6">In this study, we develop a maximum-likelihood (ML) method for calling genotypes from high-throughput sequencing data that incorporates the prior information from a genotype-frequency estimator (GFE) (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Maruki and Lynch 2015</a>). We examine the performance of the proposed method using computer simulations under different genetic conditions, including those where HWE is violated, and compare the performance with that of other widely used methods. The results show that our method yields more accurate genotype calls with low or moderately high depths of coverage than the current widely used methods, which is supported by analysis of human data. In addition, we develop another ML method for calling genotypes from high-coverage sequencing data, which relaxes the assumption of biallelic polymorphisms made in many existing methods.</p><p id="p-7">We also examine the necessary depth of coverage for identifying triallelic sites and accurate genotype calling with the proposed method, using computer simulations. Taking the results of the performance evaluation into account, we apply the proposed method to high-throughput sequencing data of 83 clones from a population of the microcrustacean <em>Daphnia pulex</em>, which have reasonably high depths of coverage (mean 18× per site per individual) (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Lynch <em>et al.</em> 2016</a>). The results show that the proposed method enables accurate and rapid identification of polymorphic sites with arbitrary numbers of alleles. Furthermore, we show that the proposed method can be applied to analyses of polyploid data.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><p id="p-8">We develop ML methods for calling individual genotypes from high-throughput sequencing data. We develop two types of methods for calling genotypes, one for low-coverage sequencing data and the other for high-coverage sequencing data, because the degree of uncertainty associated with sequencing data depends on depths of coverage. In both types of methods, we statistically test the significance of polymorphisms. Also, we do not call the genotype of an individual when more than one genotype has equivalent likelihood values for the observed data.</p><div id="sec-2" class="subsection"><h3>Genotype calling from low-coverage sequencing data</h3><p id="p-9">When depths of coverage are low, only one of the two parental chromosomes might be sampled, and sequence errors can resemble true variants. In such cases, the statistical uncertainties of individual sequence data can be high, thus calling accurate genotypes is not easy. However, when sequence data for multiple individuals from a population are available, the accuracy of the called genotypes can be improved by incorporating the population-level information on genotype frequencies and error rates into the genotype-calling process using Bayes’ theorem [see <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Martin <em>et al.</em> (2010)</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Nielsen <em>et al.</em> (2012)</a>; <a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Vieira <em>et al.</em> (2013)</a> for similar methods using the expectation-maximization algorithm for estimating priors].</p><p id="p-10">Here, the genotype of an individual at a single site is called from the nucleotide read quartet (counts of A, C, G, and T) of high-throughput sequencing data at the site by an ML method. This is achieved by maximizing the likelihood of the observed data as a function of the genotype of the individual <em>g</em>. The two most abundant nucleotide reads in the population sample are considered to be candidates for alleles at the site.</p><p id="p-11">Given the genotype of the individual, <em>g</em> = 1 (major homozygote, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-1.gif"/></noscript></span></span>), 2 (heterozygote, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-2.gif"/></noscript></span></span>), or 3 (minor homozygote, <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-3.gif"/></noscript></span></span>), and sequencing error rate per read per site <em>ε</em>, the log-likelihood of the observed site-specific read quartet consisting of the observed counts of the most abundant (major) nucleotide read <em>M</em> (<em>e.g.</em>, C) (<span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-4.gif"/></noscript></span></span>), second most abundant (minor) nucleotide read <em>m</em> (<em>e.g.</em>, T) (<span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-5.gif"/></noscript></span></span>), and other nucleotide reads (<em>e.g.</em>, in this case A and G) (<span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-6.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-7.gif"/></noscript></span></span>) in the population sample, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-8.gif"/></noscript></span></span> is given by the following multinomial distribution formula:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-9.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-10.gif"/></noscript></span></span> (the depth of coverage). <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-11.gif"/></noscript></span></span> is a probability of observed nucleotide read <em>M</em> with genotype <em>g</em>. It is a function of <em>ε</em> and is given by summing conditional probabilities of the observed nucleotide read, given the true nucleotide on the sequenced chromosome chosen from the pair (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The other <em>P</em> terms are similarly defined. For example, the probability of nucleotide read <em>M</em> with genotype 2 (<span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-12.gif"/></noscript></span></span>), <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-13.gif"/></noscript></span></span> is <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-14.gif"/></noscript></span></span> assuming the error occurs at an equal rate from the true nucleotide to one of the other three nucleotides. Because the multinomial coefficient in Equation 1 is constant regardless of the parameter values, for computational efficiency, it is ignored as follows:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-27.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>When the genotype frequencies and error rate at the site are estimated from the population sample of nucleotide reads, they can be incorporated into Equation 2 as Bayes’ priors. We previously developed an ML method to estimate the site-specific genotype frequencies and error rate from the nucleotide read quartets (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Maruki and Lynch 2015</a>). This method yields essentially unbiased genotype-frequency estimates even with moderate depths of coverage, which we use as the Bayes’ priors necessary for improving the accuracy of genotype calls. Specifically, given the estimates of the genotype frequencies <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-28.gif"/></noscript></span></span> (frequency of major homozygotes), <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-29.gif"/></noscript></span></span> (frequency of heterozygotes), and <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-30.gif"/></noscript></span></span> (frequency of minor homozygotes), and error rate (<span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-31.gif"/></noscript></span></span>) predetermined by the ML method of <a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Maruki and Lynch (2015)</a>, the log-likelihood of the observed data are given as follows:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-32.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where the ML error-rate estimate <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-33.gif"/></noscript></span></span> is substituted into the <em>P</em> terms. The inferred (called) genotype of the individual is the genotype that maximizes Equation 3. To avoid calling genotypes at false polymorphic sites, we only call genotypes at significantly polymorphic sites, which are identified beforehand using the likelihood-ratio test by the population-level genotype-frequency estimator (GFE) (<a id="xref-ref-44-5" class="xref-bibr" href="#ref-44">Maruki and Lynch 2015</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484298/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484298/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Probability of an observed nucleotide read as a function of the individual genotype <em>g</em> and error rate <em>ε</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Genotype calling from high-coverage sequencing data</h3><p id="p-13">When depths of coverage are high, both parental chromosomes are sequenced with high probability, and nucleotide reads derived from true variants are reliably abundant compared with those due to sequence errors. In such cases, if the confounding effects of binomial sampling of parental chromosomes and sequence errors are statistically accounted for, accurate and rapid genotype calls may be made from sequence data on each individual separately, relaxing the assumption of biallelic polymorphisms made in the method for low-coverage sequencing data and eliminating the need for prior population-level estimates of genotype frequencies.</p><p id="p-14">Given the nucleotide read quartet of an individual at a site, we call the genotype of the individual by finding the genotype that maximizes the likelihood of the observed data. For example, if the nucleotide read quartet of an individual contains only A, C, and G, we examine the likelihoods of six candidate genotypes (AA, AC, AG, CC, CG, and GG). Suppose, for example, that the nucleotide read quartet of an individual contains nonzero counts for all four nucleotides. Letting <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-34.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-35.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-36.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-37.gif"/></noscript></span></span> denote the counts of A, C, G, and T, respectively, the log-likelihood for genotype AA is<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-38.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where <em>n</em> is the sum of the nucleotide read counts (depth of coverage) and <em>ε</em> is the sequence error rate per read per site. This equation is a multinomial distribution formula, where the constant multinomial coefficient is ignored for computational efficiency, as in Equation 2. The subsequent likelihood functions are derived and shown in a similar way. By taking the derivative of Equation 4 with respect to <em>ε</em> and equating it to zero, <em>ε</em> is analytically estimated as<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-39.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>and this estimate is substituted into Equation 4 to find the likelihood for genotype AA. As another example, the log-likelihood for genotype AC is<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-40.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where <em>ε</em> is similarly estimated as<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-41.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>and again, this estimate is substituted into Equation 6 to find the likelihood for genotype AC. The likelihoods for the other eight candidate genotypes are similarly calculated.</p></div><div id="sec-4" class="subsection"><h3>Statistical tests of called genotypes in the high-coverage genotype caller</h3><p id="p-15">Because of high sequencing error rates, genotypes called by the high-coverage genotype caller (HGC) might sometimes falsely suggest polymorphisms. To minimize analyzing false polymorphisms and misidentifying sites containing three or four alleles, we examine statistical significance of called genotypes by likelihood-ratio tests (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kendall and Stuart 1979</a>). Specifically, we examine the statistical significance of called genotypes with respect to the genotype homozygous for the most abundant nucleotide in the population sample <em>M</em> (<span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-42.gif"/></noscript></span></span>).</p><p id="p-16">Under the null hypothesis of monomorphism, the site is fixed for <em>M</em>. Under the alternative hypothesis of polymorphism, at least one individual has a genotype different from <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-43.gif"/></noscript></span></span> Therefore, we reject the null hypothesis of population monomorphism if the likelihood of at least one non-<span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-44.gif"/></noscript></span></span> called genotype is significantly greater than that of the <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-45.gif"/></noscript></span></span> genotype. Specifically, letting <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-46.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-47.gif"/></noscript></span></span> denote the log-likelihoods of the observed data of an individual for the <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-48.gif"/></noscript></span></span> genotype and that for a non-<span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-49.gif"/></noscript></span></span> called genotype, the likelihood-ratio test statistic (<em>LRT</em>) for the individual is<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-50.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>This test statistic is expected to be asymptotically <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-51.gif"/></noscript></span></span>-distributed with one degree of freedom. We reject the null hypothesis of population monomorphism when <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-52.gif"/></noscript></span></span> is significant for at least one of the non-<span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-53.gif"/></noscript></span></span> called genotypes at a user-specified level.</p><p id="p-17">We estimate the number of alleles by examining the nucleotides contained in significant genotypes. To minimize misidentification of sites containing three or four alleles, when the significant genotype is heterozygous, we compare the likelihood of the called genotype to that of the genotype homozygous for the more abundant nucleotide in the individual, and consider the genotype heterozygous only if the former is significantly greater than the latter at the specified level by the likelihood-ratio test with one degree of freedom. Otherwise, the genotype is considered homozygous for estimating the number of alleles.</p></div><div id="sec-5" class="subsection"><h3>Genotype calling from triploid sequencing data</h3><p id="p-18">The HGC explained above can be extended to triploid data. Specifically, assuming that sequencing occurs randomly among the three chromosomes, and equal error rates occur from the true nucleotide to one of the other three, the probability of an observed nucleotide read as a function of the genotype of an individual is found in a way analogous to that for diploid data (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/TableS1.docx">Table S1</a>). Then, the likelihood of the observed nucleotide read quartet of an individual can be formulated. As with diploid data, we choose the genotype that maximizes the likelihood to call the genotype of the individual. The proposed method can be applied to low-coverage sequencing data, although high coverage is needed to call accurate genotypes.</p><p id="p-19">As an example, suppose that the nucleotide read quartet of an individual contains nonzero counts for all four nucleotides. Letting <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-54.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-55.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-56.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-57.gif"/></noscript></span></span> denote the counts of A, C, G, and T, respectively, the log-likelihood for genotype AAA is<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-58.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span>where <em>n</em> is the sum of the nucleotide read counts (depth of coverage), and <em>ε</em> is the sequence error rate per read per site. By taking the derivative of Equation 9 with respect to <em>ε</em> and equating it to zero, <em>ε</em> is analytically estimated as<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-59.gif"/></noscript></span><span class="disp-formula-label">(10)</span></span>and this estimate is substituted into Equation 9 to find the likelihood for genotype AAA. As another example, the log-likelihood for genotype ACC is<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-60.gif"/></noscript></span><span class="disp-formula-label">(11)</span></span>where <em>ε</em> is estimated as<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-61.gif"/></noscript></span><span class="disp-formula-label">(12)</span></span>and again, this estimate is substituted into Equation 11 to find the likelihood for genotype ACC. As a final example, the log-likelihood for genotype ACG is<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-62.gif"/></noscript></span><span class="disp-formula-label">(13)</span></span>where <em>ε</em> is estimated as<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-63.gif"/></noscript></span><span class="disp-formula-label">(14)</span></span>and again, this estimate is substituted into Equation 13 to find the likelihood for genotype ACG. To avoid analyzing false polymorphisms due to sequencing errors, the statistical significance of called genotypes is examined by likelihood-ratio tests analogous to those with diploid data, and the number of alleles is based on significant genotypes.</p></div><div id="sec-6" class="subsection"><h3>Genotype calling from tetraploid sequencing data</h3><p id="p-20">Because tetraploid species are common in plants and some animals, we also formulate likelihood functions for a tetraploid genotype caller. Assuming that sequencing occurs randomly among the four chromosomes, and equal error rates occur from the true nucleotide to one of the other three, the probability of an observed nucleotide read as a function of the genotype of an individual is similarly found (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/TableS2.docx">Table S2</a>). As with triploid data, we find the genotype that maximizes the likelihood of the observed nucleotide read quartet of an individual to call the genotype of the individual.</p><p id="p-21">As an example, suppose that the nucleotide read quartet of an individual contains nonzero counts for all four nucleotides. Letting <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-64.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-65.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-66.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-67.gif"/></noscript></span></span> denote the counts of A, C, G, and T, respectively, the log-likelihood for genotype AAAA is<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-68.gif"/></noscript></span><span class="disp-formula-label">(15)</span></span>where <em>n</em> is the sum of the nucleotide read counts (depth of coverage) and <em>ε</em> is the sequence error rate per read per site. By taking the derivative of Equation 15 with respect to <em>ε</em> and equating it to zero, <em>ε</em> is analytically estimated as<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-69.gif"/></noscript></span><span class="disp-formula-label">(16)</span></span>and this estimate is substituted into Equation 15 to find the likelihood for genotype AAAA. As another example, the log-likelihood for genotype ACCC is<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-70.gif"/></noscript></span><span class="disp-formula-label">(17)</span></span>where <em>ε</em> is estimated as<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-71.gif"/></noscript></span><span class="disp-formula-label">(18)</span></span>and again, this estimate is substituted into Equation 17 to find the likelihood for genotype ACCC. As another example, the log-likelihood for genotype AACC is<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-72.gif"/></noscript></span><span class="disp-formula-label">(19)</span></span>where <em>ε</em> is estimated as<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-73.gif"/></noscript></span><span class="disp-formula-label">(20)</span></span>and again, this estimate is substituted into Equation 19 to find the likelihood for genotype AACC. As a final example, the log-likelihood for genotype AACG is<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-74.gif"/></noscript></span><span class="disp-formula-label">(21)</span></span>where <em>ε</em> is estimated as<span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-75.gif"/></noscript></span><span class="disp-formula-label">(22)</span></span>and again, this estimate is substituted into Equation 21 to find the likelihood for genotype AACG. Again, we report the number of alleles at a site based on significant genotypes to avoid analyzing false polymorphisms due to sequencing errors.</p></div><div id="sec-7" class="subsection"><h3>Generation of diploid nucleotide read data at biallelic sites using computer simulations</h3><p id="p-22">To examine the performance of the genotype callers at biallelic sites, we generated nucleotide read data for <em>N</em> diploid individuals by computer simulations and called individual genotypes from the simulated data. In the simulations, the probability of sampling an individual with a particular genotype was equal to its relative frequency in the population. The population frequencies of major and minor homozygotes were specified by <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-76.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-77.gif"/></noscript></span></span> respectively. To compare the called genotypes with the true genotypes, we recorded the true genotype of each individual. The depths of coverage were assumed to be Poisson-distributed with mean <em>μ</em> among the individuals, and were specified as<span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-78.gif"/></noscript></span><span class="disp-formula-label">(23)</span></span>where <em>X</em> is a particular value of the coverage for an individual and <em>c</em> is a probability mass function of <em>X</em>. The nucleotide reads from each individual were randomly chosen from its genotype allowing for errors. Sequence errors were randomly introduced at rate <em>ε</em> per read per site from the true nucleotide to one of the other three nucleotides.</p></div><div id="sec-8" class="subsection"><h3>Comparison of the Bayesian genotype caller with other widely used methods</h3><p id="p-23">To compare the performance of our Bayesian genotype caller (BGC) with that of other widely used methods for calling genotypes, we called individual genotypes using GATK (version 3.4-0) (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">McKenna <em>et al.</em> 2010</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">DePristo <em>et al.</em> 2011</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Van der Auwera <em>et al.</em> 2013</a>) and Samtools (version 0.1.19) (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Li 2011</a>) from the same simulated nucleotide read data generated by the method described above, and compared the accuracy of the genotypes called by different methods. Although both GATK and Samtools use Bayesian genotype-frequency priors, GATK uses the same priors for all sites and Samtools assumes HWE, and therefore their approaches differ from ours. In addition to these genotype-calling methods, we also compared our performance with that of the corresponding genotype-calling function in ANGSD (version 0.911) (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Korneliussen <em>et al.</em> 2014</a>), which is also designed especially for low-coverage sequencing data. To make the generated data applicable to all methods, including ours, we made BAM files of sequence read data mapped to a simulated reference sequence.</p><p id="p-24">First, we generated a simulated reference sequence consisting of random nucleotides. Each of a total of 10,000 biallelic sites was surrounded by 50 fixed nucleotides on both sides, so that sequence reads of 101 bp are uniquely mapped to the reference sequence. We outputted the 101-bp sequence reads in the FASTA format with nucleotide reads at biallelic sites characterized in the manner described above. The sequence reads were mapped to the reference sequence using Novoalign (version 3.00.02) (<a href="http://www.novocraft.com">www.novocraft.com</a>). We converted SAM files of mapped reads to BAM files using Samtools (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Li <em>et al.</em> 2009a</a>).</p><p id="p-25">To call genotypes with GATK, we made a dictionary file of the reference sequence and added read groups to the BAM files using Picard (version 1.134) (<a href="http://broadinstitute.github.io/picard">http://broadinstitute.github.io/picard</a>). Then, we sorted and indexed the BAM files using Samtools. We generated individual GVCF files, which contain information of called genotypes, from the BAM files using HaplotypeCaller, which individually calls genotypes, assigning an arbitrarily chosen uniform base quality score (30) to every base. Changing the base quality score, however, from 30 to 20, did not noticeably influence the results (results not shown). Then, we called genotypes from the GVCF files using Genotype_GVCFs, which refines the called genotypes using the population-level information. We called genotypes with Samtools from the BAM files using mpileup and bcftools to use the population-level information. To call genotypes with ANGSD, we first converted the BAM files to SAM files using Samtools. Then, we assigned the same uniform base quality score of 30 to every base, using a custom Perl script. The resulting SAM files with base quality scores were converted back to BAM files as input for ANGSD, using Samtools. We used a Bayesian genotype-calling method corresponding to ours in ANGSD, setting the value of the -doPost option at one, using Samtools genotype likelihoods, and setting the statistical significance for calling SNPs at the 5% level. Unlike other methods, our method does not rely just on read quality, and estimates error rates from sequence data themselves. This unique feature of our method is important because errors can result from other factors including those introduced during sample preparation.</p><p id="p-26">The ML estimates of genotype frequencies and error rates, necessary for calling genotypes with BGC, were obtained using GFE (available from <a href="https://github.com/Takahiro-Maruki/Package-GFE">https://github.com/Takahiro-Maruki/Package-GFE</a>; an updated version with a function to prepare the input file of the BGC and its documentation are available as <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/FileS1.zip">File S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/FileS2.pdf">File S2</a>) (<a id="xref-ref-44-6" class="xref-bibr" href="#ref-44">Maruki and Lynch 2015</a>). The individual pro files of nucleotide read quartets were generated using sam2pro (version 0.6) (<a href="http://guanine.evolbio.mpg.de/mlRho/sam2pro_0.6.tgz">http://guanine.evolbio.mpg.de/mlRho/sam2pro_0.6.tgz</a>) from mpileup files generated from the BAM files by Samtools.</p><p id="p-27">In addition to examining the performance of genotype calling, we compared the allele-frequency estimates by GFE to those by other methods. The allele-frequency estimates via genotype calling by GATK and Samtools were calculated from the VCF files using VCFtools (version 0.1.11) (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Danecek <em>et al.</em> 2011</a>). The allele-frequency estimates by ANGSD were estimated directly from sequence reads by the method of <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Kim <em>et al.</em> (2011)</a>.</p><p id="p-28">To examine how different genotype-calling methods perform when applied to real sequencing data, we applied them to low-coverage (mean 7.6× per site per individual) sequencing data of the phase 3 1000 Genomes project (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">1000 Genomes Project Consortium 2015</a>) on chromosome 11 in the CEU population. To assess the accuracy of genotype calls by different methods, we compared the called genotypes with the corresponding genotypes in phase II and III data of the International HapMap project (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">International HapMap Consortium 2007</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">International HapMap 3 Consortium 2010</a>). Specifically, we downloaded the BAM files of the phase 3 1000 Genomes sequencing data in 99 CEU individuals on chromosome 11 from <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase3/data/">ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase3/data/</a>. We also downloaded the FASTA file of the reference sequence (human_g1k_v37.fasta) from <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/reference/">ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/reference/</a>. We downloaded the corresponding genotypes in the HapMap phase II and III data from <a href="ftp://ftp.ncbi.nlm.nih.gov/hapmap/genotypes/2010-08_phaseII+III/forward/">ftp.ncbi.nlm.nih.gov/hapmap/genotypes/2010-08_phaseII+III/forward/</a>. We found that 94 out of the 99 CEU individuals had both 1000 Genomes and HapMap data, and therefore used the 94 individuals with the necessary data in the subsequent analyses.</p><p id="p-29">To prepare high-quality input data for calling genotypes, we marked duplicate reads and locally realigned sequences around indels in the BAM files using GATK (version 3.4-0) (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">McKenna <em>et al.</em> 2010</a>; <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">DePristo <em>et al.</em> 2011</a>) and Picard (version 1.107) (<a href="http://broadinstitute.github.io/picard">http://broadinstitute.github.io/picard</a>), following GATK best practices (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Van der Auwera <em>et al.</em> 2013</a>). In addition, we clipped overlapping read pairs in the BAM files using BamUtil (version 1.0.13) (<a href="http://genome.sph.umich.edu/wiki/BamUtil">http://genome.sph.umich.edu/wiki/BamUtil</a>). We made mpileup files of the 94 individuals from the processed BAM files using Samtools. The pro files of nucleotide read quartets were made from the mpileup files using sam2pro (version 0.8) (<a href="http://guanine.evolbio.mpg.de/mlRho/sam2pro_0.8.tgz">http://guanine.evolbio.mpg.de/mlRho/sam2pro_0.8.tgz</a>). The file of nucleotide read quartets of 94 individuals necessary for the proposed method was made from the pro files using GFE (<a id="xref-ref-44-7" class="xref-bibr" href="#ref-44">Maruki and Lynch 2015</a>).</p><p id="p-30">To call genotypes with GATK, we first sorted and indexed the BAM files using Samtools. Next, we generated individual GVCF files from the BAM files using HaplotypeCaller. Then, we called genotypes from the GVCF files using Genotype_GVCFs. We called genotypes with Samtools from the BAM files using mpileup and bcftools using the population-level information. To call genotypes with ANGSD from the BAM files, we used a Bayesian genotype-calling method corresponding to ours in ANGSD setting the value of the -doPost option at one, using Samtools genotype likelihoods and setting the statistical significance for calling SNPs at the 5% level. In addition to these genotype-calling methods, we examined how the popular genotype-imputation method performs by imputing missing genotypes in GATK calls using Beagle (version 4.1) (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Browning and Browning 2016</a>). Because preexisting high-quality reference panels (genotypes/haplotypes) do not exist in the vast majority of the organisms, except a few model organisms such as humans and flies, we imputed the genotypes without a reference panel. Here, we imputed genotypes using genotype likelihoods instead of called genotypes to take the uncertainty of genotype calls into account.</p><p id="p-31">We calculated the correct-call rate among individuals as a fraction of individuals with genotype calls identical to HapMap genotypes, assuming that HapMap genotypes are correct. To examine the accuracy of actually called genotypes, we also calculated the correct-call rate among called genotypes, where the fraction is calculated among called genotypes. We converted the NCBI build 36 coordinates in the HapMap data to the GRCh37 coordinates in the 1000 Genomes data using the UCSC liftOver (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Speir <em>et al.</em> 2016</a>) to make the coordinates in the two data sets consistent with each other. To minimize the confounding effect of mismapping, we excluded sites involved in putative repetitive regions identified by RepeatMasker (<a href="http://www.repeatmasker.org/">http://www.repeatmasker.org/</a>) and those with population coverage (sum of the coverage over the individuals) less than half the mean or greater than one and a half means from the analyses. We downloaded the repeat-masked reference sequence from the Ensembl website (<a href="ftp://ftp.ensembl.org/pub/release-75/fasta/homo_sapiens/dna/">ftp.ensembl.org/pub/release-75/fasta/homo_sapiens/dna/</a>) to identify sites involved in putative repetitive regions.</p></div><div id="sec-9" class="subsection"><h3>Performance evaluation of the HGC as a function of coverage</h3><p id="p-32">Because of the random sampling of parental chromosomes, only one of the two chromosomes might be sampled when the depth of coverage for an individual is low. Therefore, the ability of a genotype caller to correctly infer individual genotypes is mainly limited by its ability to call heterozygotes when the depth of coverage is low. In addition, when the depth of coverage is low, homozygotes can appear heterozygous due to sequencing errors. Therefore, examining the effect of coverage on the accuracy of called genotypes is important for finding the optimal sequencing strategy for genotype ascertainment.</p><p id="p-33">To examine how much coverage is needed to accurately call genotypes using the HGC, we examined its rates for correctly calling homozygotes and heterozygotes as functions of the depth of coverage. Specifically, we generated nucleotide read data from an individual having a homozygous or heterozygous genotype with a fixed depth of coverage and error rate <em>ε</em>, and called the genotype of the individual with HGC. We repeated this process for 10,000 simulation replications and calculated the correct-call rate as a fraction of simulation replications where the genotype was correctly called by HGC.</p></div><div id="sec-10" class="subsection"><h3>Performance evaluation of genotype callers at triallelic sites using computer simulations</h3><p id="p-34">To examine the performance of the HGC for identifying sites with more than two alleles and calling genotypes in population samples, we generated BAM files of sequence read data in a way similar to that at biallelic sites, where nucleotide read data at each of a total of 10,000 triallelic sites for <em>N</em> diploid individuals were generated, according to their genotype frequencies, introducing errors at rate <em>ε</em>. Here, for simplicity, we specified the population genotype frequencies by assigning population allele frequencies <em>p</em>, <em>q</em>, and <em>r</em> to the most abundant, second most abundant, and the rarest allele, respectively, and assuming HWE, although our method makes no assumption on the mating system. As GATK and Samtools are both capable of calling genotypes at triallelic sites, we applied them to the BAM files using HaplotypeCaller and GenotypeGVCF, and mpileup and bcftools with the -m option, respectively, and compared their correct-call rates to that of HGC. In addition, we applied our BGC to the BAM files and examined the corresponding correct-call rate to find the effect of incorrectly assuming at most two alleles on the accuracy of genotype calls at triallelic sites.</p></div><div id="sec-11" class="subsection"><h3>Application of the HGC to empirical data</h3><p id="p-35">To examine the performance of the HGC with real data, we applied it to high-throughput Illumina sequencing data of 83 <em>D. pulex</em> clones from Kickapond (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Lynch <em>et al.</em> 2016</a>). We mapped the sequencing data to the PA42 reference genome (version 3.0) (Z. Ye, S. Xu, K. Spitze, J. Asselman, X. Jiang, M. E. Pfrender, and M. Lynch, unpublished results) using Novoalign (version 3.02.11) (<a href="http://www.novocraft.com">www.novocraft.com</a>) with the “-r None” option to prevent it from mapping a read if it matched more than one location. We converted the SAM files of mapped sequencing data to BAM files using Samtools (version 0.1.18) (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Li <em>et al.</em> 2009a</a>). Then, we marked duplicate reads and locally realigned sequences around indels using GATK (version 3.4-0) (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">McKenna <em>et al.</em> 2010</a>; <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">DePristo <em>et al.</em> 2011</a>) and Picard (version 1.107) (<a href="http://broadinstitute.github.io/picard">http://broadinstitute.github.io/picard</a>), following GATK best practices (<a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Van der Auwera <em>et al.</em> 2013</a>). In addition, we clipped overlapping read pairs using BamUtil (version 1.0.13) (<a href="http://genome.sph.umich.edu/wiki/BamUtil">http://genome.sph.umich.edu/wiki/BamUtil</a>). We made mpileup files of the 83 clones from the processed BAM files using Samtools. The pro files of nucleotide read quartets were made from the mpileup files using sam2pro (version 0.8) (<a href="http://guanine.evolbio.mpg.de/mlRho/sam2pro_0.8.tgz">http://guanine.evolbio.mpg.de/mlRho/sam2pro_0.8.tgz</a>). The input file of nucleotide read quartets of 83 clones was made from the pro files using GFE (<a id="xref-ref-44-8" class="xref-bibr" href="#ref-44">Maruki and Lynch 2015</a>). To avoid analyzing misassembled regions, we excluded regions considered to be misassembled by REAPR (version 1.134) (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hunt <em>et al.</em> 2013</a>) from our analyses. We set the minimum coverage required to call a genotype of an individual at six. To minimize analyzing sites with data on a small number of individuals or those with mismapped reads, we only analyzed sites with the population coverage (sum of the coverage over the individuals) at least half the mean and at most 1.5× of the mean. Furthermore, we excluded sites involved in putative repetitive regions identified by RepeatMasker (version 4.0.5) (<a href="http://www.repeatmasker.org/">http://www.repeatmasker.org/</a>) with the RepeatMasker library (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Jurka <em>et al.</em> 2005</a>) made on August 7, 2015. In addition, to further reduce analyzing sites with mismapped reads, we excluded sites with mean error-rate estimates among clones of &gt;0.01.</p></div><div id="sec-12" class="subsection"><h3>Performance evaluation of the triploid genotype caller as a function of coverage</h3><p id="p-36">To examine the effect of coverage on the accuracy of genotypes called by the triploid genotype caller (TRI), we examined the correct-call rate as a function of the depth of coverage in a way similar to that for diploid data. Here, we examined the correct-call rate for three different types of genotypes (homozygotes, heterozygotes containing two different nucleotides, and heterozygotes containing three different nucleotides) among a total of 10,000 simulation replications.</p></div><div id="sec-13" class="subsection"><h3>Comparison of the TRI with GATK</h3><p id="p-37">To compare the performance of the TRI to that of other widely used methods, we applied our TRI and GATK (version 3.4-0) (<a id="xref-ref-45-5" class="xref-bibr" href="#ref-45">McKenna <em>et al.</em> 2010</a>; <a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">DePristo <em>et al.</em> 2011</a>) to BAM files of simulated sequence read data generated in a way similar to that for diploid data, and compared the correct-call rates. Here, we generated fixed-coverage read data from an individual with a particular genotype 10,000 times. To call triploid genotypes with GATK, we set the ploidy of HaplotypeCaller at three.</p></div><div id="sec-14" class="subsection"><h3>Data availability</h3><p id="p-38">Source codes, written in C++, implementing the proposed methods and their documentation are available as supporting information (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/FileS1.zip">File S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/FileS2.pdf">File S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/FileS3.pdf">File S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/FileS4.pdf">File S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/FileS5.zip">File S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/FileS6.pdf">File S6</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/FileS7.zip">File S7</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/FileS8.pdf">File S8</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/FileS9.zip">File S9</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/FileS10.pdf">File S10</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/FileS11.zip">File S11</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/FileS12.pdf">File S12</a>).</p></div></div><div class="section results" id="sec-15"><h2 class="">Results</h2><p id="p-39">The performance of the BGC when applied to low-coverage sequencing data at biallelic sites was evaluated with simulated data and compared with the corresponding performance of GATK, Samtools, and ANGSD. In addition, the allele-frequency estimates by the GFE were compared with those via genotype calling by GATK and Samtools and those by a method of <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Kim <em>et al.</em> (2011)</a>, as implemented in ANGSD. To examine the performance under the worst situation, the error rate was set at 0.01, which is typically the upper bound with commonly used sequencing platforms. We evaluated the performance under three different genetic conditions, where the inbreeding coefficient <em>f</em> was zero (HWE), minimized, or maximized given minor-allele frequency <em>q</em>. When <em>f</em> is minimized, the frequencies of major homozygotes and minor homozygotes, <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-79.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-80.gif"/></noscript></span></span> are <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-81.gif"/></noscript></span></span> and 0, respectively. When <em>f</em> is maximized, <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-82.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-83.gif"/></noscript></span></span> are <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-84.gif"/></noscript></span></span> and <em>q</em>, respectively.</p><p id="p-40">When the depth of coverage is low (mean 3×), the allele-frequency estimates by GFE are unbiased, whereas those via genotype calling are biased under all examined conditions (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The allele-frequency estimates by ANGSD are similar to ours, although they are slightly biased when <em>f</em> is minimized or maximized. The allele-frequency estimates via genotype calling by GATK are more biased than those by Samtools when <em>f</em> is zero or minimized. On the other hand, when <em>f</em> is maximized, the allele-frequency estimates via genotype calling by Samtools are more biased than those by GATK. The correct-call rate among individuals by the proposed method (BGC) is higher than that by GATK, and lower than that by Samtools and ANGSD when <em>f</em> is zero or minimized. When <em>f</em> is maximized, the correct-call rate by BGC is the highest. The highest correct-call rate among individuals by Samtools and ANGSD when <em>f</em> is zero or minimized is mainly because both methods always call individual genotypes regardless of the depth of coverage, even when there is no read. In fact, the correct-call rate among called genotypes by BGC is the highest under the majority of the examined conditions.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484329/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484329/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Comparison of the allele-frequency estimates and called genotypes by different methods with low depths of coverage</span><div class="sb-div caption-clear"></div></div></div><p id="p-42">When the depth of coverage is moderately high (mean 10×), the allele-frequency estimates by all methods are similar to each other and nearly unbiased under all examined conditions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/TableS3.docx">Table S3</a>). Consistent with this, the correct-call rates by all methods are high under all examined conditions. However, the correct-call rate by BGC is the highest under all examined conditions. We confirmed that we correctly generated the simulated data by examining the realized parameter values in the population samples (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/TableS4.docx">Table S4</a>).</p><p id="p-43">Our conclusions on the performance of the genotype-calling methods based on computer simulations are further supported by qualitatively similar results of the corresponding performance evaluation using human data (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Similar to the simulation-based results of the performance evaluation when the population is in HWE, the correct-call rate among individuals of BGC was lower than that by ANGSD and Samtools and higher than that by GATK. Again, this is mainly because the former two methods call individual genotypes even when there is no read, and the correct-call rate among called genotypes by BGC is the highest. The correct-call rate among individuals of the Beagle imputation of genotypes called by GATK was higher than that by GATK and lower than that of the other methods. Interestingly, the correct-call rate among called genotypes of the Beagle imputation of genotypes called by GATK was lower than that by GATK, indicating that genotype imputation does not necessarily improve the accuracy of genotype calls, at least when preexisting high-quality reference panels are not available. The observation here is consistent with a recent finding that population-genetic parameters estimated via genotype imputation from low-coverage sequencing data are biased (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Fu 2014</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484392/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484392/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Comparison of the performance of the genotype-calling methods with human data</span><div class="sb-div caption-clear"></div></div></div><p id="p-45">To examine when the depth of coverage is high enough to accurately call individual genotypes using the HGC, we examined the correct-call rate as a function of coverage when the true genotype is homozygous or heterozygous. Overall, the rate for correctly calling homozygotes is high (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). It decreases with increased coverage when the coverage is five or less, and approaches one when the coverage is over 5. The overall rate for correctly calling heterozygotes increases with increased coverage, although it somewhat decreases when the coverage increases from five to six (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). This decrease is consistent with the sudden increase in the rate for correctly calling homozygotes when the coverage increases from five to six. These patterns are observed because when two nucleotides have nonzero read counts, one of which has just one read, in the quartet, both of them are considered to be from an allele without error when the coverage is five or less, and the one with a single read count is considered to be due to a sequence error when the coverage is over five by HGC, as the likelihood that the single read is due to a sequence error becomes greater than that without error. In addition to the valley of the correct-call rate at coverage equal to six, there is another small valley at coverage equal to 11. This is because when two nucleotides have nonzero read counts, one of which has just two reads, in the quartet, both of them are considered to be from an allele without error when the coverage is &lt;11, and the one with double reads is considered to be due to sequence errors when the coverage is ≥11 by HGC, as the likelihood that such reads are due to sequencing errors becomes greater than that without error.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1393/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correct-call rate of the high-coverage genotype caller as a function of the depth of coverage. (A) Correct-call rate when the true genotype of the individual is homozygous. (B) Correct-call rate when the true genotype of the individual is heterozygous. Error rate &#x3B5; = 0.01. Results are based on a total of 10,000 simulation replications for each parameter set." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-855133894" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Correct-call rate of the high-coverage genotype caller as a function of the depth of coverage. (A) Correct-call rate when the true genotype of the individual is homozygous. (B) Correct-call rate when the true genotype of the individual is heterozygous. Error rate &lt;em&gt;&#x3B5;&lt;/em&gt; = 0.01. Results are based on a total of 10,000 simulation replications for each parameter set.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1393/F1.medium.gif" width="440" height="214"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/5/1393/F1.medium.gif" width="440" height="214"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1393/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1393/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484409" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-46" class="first-child">Correct-call rate of the high-coverage genotype caller as a function of the depth of coverage. (A) Correct-call rate when the true genotype of the individual is homozygous. (B) Correct-call rate when the true genotype of the individual is heterozygous. Error rate <em>ε</em> = 0.01. Results are based on a total of 10,000 simulation replications for each parameter set.</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">To compare the performance of HGC in calling diploid genotypes at triallelic sites with that of other existing methods, we compared the correct-call rate of HGC to that of GATK and Samtools (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). In addition, to examine the effect of incorrectly assuming biallelic polymorphisms on the accuracy of genotype calls at triallelic sites, we applied our BGC to the same data and examined its correct-call rate. The correct-call rate of HGC is higher than that by Samtools and slightly lower than that by GATK. Compared with genotype calls by Samtools, genotype calls by HGC and GATK quickly become more accurate with higher depths of coverage. The correct-call rate of BGC is the lowest. This is because BGC assumes at most two alleles, as many other methods including ANGSD do, and fails to correctly call genotypes containing the rarest allele. We confirmed that we correctly generated the simulated data by examining the realized parameter values in the population samples (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/TableS5.docx">Table S5</a>).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484378/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484378/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Performance of different genotype callers for calling diploid genotypes from nucleotide data at triallelic sites</span><div class="sb-div caption-clear"></div></div></div><p id="p-49">To evaluate the power of HGC for detecting polymorphisms, we examined the false-positive and -negative rates as functions of the mean depth of coverage <em>μ</em>. The false-positive rate of detecting polymorphisms is low when <em>μ</em> is moderately high (10), and is essentially zero with <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/embed/mml-math-88.gif"/></noscript></span></span> (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). The false-negative rate of detecting polymorphisms is reasonably low when <em>μ</em> is moderately high (10), and approaches the theoretical minimum possible value, which is the probability that only one of the two alleles is sampled in a finite sample, when <em>μ</em> is 30 (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). We note that the high false-negative rates with low minor-allele frequencies are mainly due to the finite sample size (100), which limits sampling rare alleles, and they remain high even with infinite coverage with any method. The power for detecting three alleles is essentially the same as that for detecting polymorphisms; the false-positive rate is low (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>) and false-negative rate is reasonably low (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2D</a>). These results indicate that HGC accurately describes polymorphisms with arbitrary numbers of alleles when the depth of coverage is high.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1393/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Power analysis of the high-coverage genotype caller. (A) False-positive rate of polymorphism detection as a function of the mean depth of coverage. (B) False-negative rate of detecting polymorphisms at biallelic sites as a function of the minor-allele frequency. The solid curve shows the theoretical minimum value as the probability that just one of the alleles is sampled in a finite sample of size N given by  where q is the minor-allele frequency. (C) False-positive rate of detecting triallelic sites as a function of the mean depth of coverage with q = 0.1 (i.e., the site is biallelic). (D) False-negative rate of inferring triallelic sites as a function of the frequency of the rarest allele. The frequency of the second most abundant allele is 0.2. The statistical significance of the likelihood-ratio tests is set at the 5% level in all panels. N = 100, error rate &#x3B5; = 0.01, inbreeding coefficient f = 0 (Hardy&#x2013;Weinberg equilibrium). Results are based on a total of 10,000 simulation replications for each parameter set." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-855133894" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Power analysis of the high-coverage genotype caller. (A) False-positive rate of polymorphism detection as a function of the mean depth of coverage. (B) False-negative rate of detecting polymorphisms at biallelic sites as a function of the minor-allele frequency. The solid curve shows the theoretical minimum value as the probability that just one of the alleles is sampled in a finite sample of size &lt;em&gt;N&lt;/em&gt; given by &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-89&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; where &lt;em&gt;q&lt;/em&gt; is the minor-allele frequency. (C) False-positive rate of detecting triallelic sites as a function of the mean depth of coverage with &lt;em&gt;q&lt;/em&gt; = 0.1 (&lt;em&gt;i.e.&lt;/em&gt;, the site is biallelic). (D) False-negative rate of inferring triallelic sites as a function of the frequency of the rarest allele. The frequency of the second most abundant allele is 0.2. The statistical significance of the likelihood-ratio tests is set at the 5% level in all panels. &lt;em&gt;N&lt;/em&gt; = 100, error rate &lt;em&gt;&#x3B5;&lt;/em&gt; = 0.01, inbreeding coefficient &lt;em&gt;f&lt;/em&gt; = 0 (Hardy&#x2013;Weinberg equilibrium). Results are based on a total of 10,000 simulation replications for each parameter set.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1393/F2.medium.gif" width="440" height="410"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/5/1393/F2.medium.gif" width="440" height="410"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1393/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1393/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484277" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-50" class="first-child">Power analysis of the high-coverage genotype caller. (A) False-positive rate of polymorphism detection as a function of the mean depth of coverage. (B) False-negative rate of detecting polymorphisms at biallelic sites as a function of the minor-allele frequency. The solid curve shows the theoretical minimum value as the probability that just one of the alleles is sampled in a finite sample of size <em>N</em> given by <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/F2/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1393/F2/embed/mml-math-89.gif"/></noscript></span></span> where <em>q</em> is the minor-allele frequency. (C) False-positive rate of detecting triallelic sites as a function of the mean depth of coverage with <em>q</em> = 0.1 (<em>i.e.</em>, the site is biallelic). (D) False-negative rate of inferring triallelic sites as a function of the frequency of the rarest allele. The frequency of the second most abundant allele is 0.2. The statistical significance of the likelihood-ratio tests is set at the 5% level in all panels. <em>N</em> = 100, error rate <em>ε</em> = 0.01, inbreeding coefficient <em>f</em> = 0 (Hardy–Weinberg equilibrium). Results are based on a total of 10,000 simulation replications for each parameter set.</p><div class="sb-div caption-clear"></div></div></div><p id="p-51">We applied the HGC to high-throughput sequencing data of 83 <em>D. pulex</em> clones from Kickapond to identify sites containing more than two alleles. We set the minimum coverage required to call a genotype of an individual at six so that the rate of falsely calling homozygotes as heterozygotes is low (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>). We identified a total of 4,403,303 significantly polymorphic sites at the 5% level from the sequence data filtered by multiple procedures to minimize analyzing misassembled regions and sites with mismapped reads. The vast majority (97.07%) of these were considered to contain two alleles. However, 127,167 (2.88%) and 2030 (0.05%) sites were considered to contain three and four alleles, respectively.</p><p id="p-52">The performance evaluation using computer simulations shows that calling accurate triploid genotypes requires much higher coverage than calling diploid genotypes (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). In particular, high coverage is needed to correctly call heterozygotes (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3, B and C</a>). This is because higher coverage is required to sequence all three chromosomes rather than just two chromosomes. Furthermore, it is difficult to distinguish the two alternative heterozygotes containing two alleles (<em>e.g.</em>, AAC <em>vs.</em> ACC) unless the coverage is very high. Overall, the correct-call rate increases with increased coverage for all three types of genotypes, although there are a few local exceptions. The valley of the rate for correctly calling homozygotes at coverage equal to eight (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>) is observed because when two nucleotides have nonzero read counts, one of which has just one read, in the quartet, both of them are considered to be from an allele without error when the coverage is eight or less, and the one with a single read is considered to be due to a sequence error when the coverage is over eight by the TRI, as the likelihood that the single read is due to a sequence error becomes greater than that without error. The sudden decrease in the rate for correctly calling heterozygotes containing three different nucleotides when coverage increases from six to seven (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>) is observed because when three nucleotides have nonzero read counts, one of which has just one read, in the quartet, all of them are considered to be from an allele without error when the coverage is six or less, and the one with a single read is considered to be due to a sequence error when the coverage is over six by the TRI, as the likelihood that the single read is due to a sequence error becomes greater than that without error. These observations highlight the inherent difficulty in calling polyploid genotypes not only for our method.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1393/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correct-call rate of the triploid genotype caller as a function of the depth of coverage. (A) Correct-call rate when the true genotype of the individual is homozygous. (B) Correct-call rate when the true genotype of the individual is heterozygous containing two different nucleotides. (C) Correct-call rate when the true genotype of the individual is heterozygous containing three different nucleotides. Error rate &#x3B5; = 0.01. Results are based on a total of 10,000 simulation replications for each parameter set." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-855133894" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Correct-call rate of the triploid genotype caller as a function of the depth of coverage. (A) Correct-call rate when the true genotype of the individual is homozygous. (B) Correct-call rate when the true genotype of the individual is heterozygous containing two different nucleotides. (C) Correct-call rate when the true genotype of the individual is heterozygous containing three different nucleotides. Error rate &lt;em&gt;&#x3B5;&lt;/em&gt; = 0.01. Results are based on a total of 10,000 simulation replications for each parameter set.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1393/F3.medium.gif" width="440" height="420"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/5/1393/F3.medium.gif" width="440" height="420"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1393/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1393/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484331" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-53" class="first-child">Correct-call rate of the triploid genotype caller as a function of the depth of coverage. (A) Correct-call rate when the true genotype of the individual is homozygous. (B) Correct-call rate when the true genotype of the individual is heterozygous containing two different nucleotides. (C) Correct-call rate when the true genotype of the individual is heterozygous containing three different nucleotides. Error rate <em>ε</em> = 0.01. Results are based on a total of 10,000 simulation replications for each parameter set.</p><div class="sb-div caption-clear"></div></div></div><p id="p-54">Comparison of the performance of our TRI to that of GATK shows that TRI yields more accurate called genotypes when the true genotype is a homozygote or heterozygote with two different nucleotides, whereas GATK yields more accurate called genotypes when the true genotype is a heterozygote with three different nucleotides (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/TableS6.docx">Table S6</a>). Because the former two genotypes are generally far more abundant than the latter in natural populations, our method is likely to yield more accurate genotype calls than GATK when applied to real data. We also note that our method is more efficient than GATK in terms of both memory usage and computation time. TRI takes 2.749 sec whereas GATK takes 397.708 sec under the same computing environment to analyze simulated data with fixed coverage of 30 at 1,010,000 sites in an individual. This ∼145-fold difference in computation time becomes huge when analyzing organisms with a large genome size.</p><p id="p-55">Because it takes additional time to prepare the input file of TRI from the BAM file, make a dictionary file of the reference, and add read groups to the BAM file to use GATK, we also compared the computation time between TRI and GATK taking these additional times into account. The additional times taken for the above analysis are 22.374 and 8.863 sec for TRI and GATK, respectively, making the total time 25.123 and 406.571 sec for TRI and GATK, respectively. Therefore, the total time taken for GATK is ∼16 times greater than that for TRI.</p></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-56">Given the rapid emergence of the field of population genomics, there is a need to systematically examine the performance of genotype callers. Recent studies (<em>e.g.</em>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lynch 2009</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Buerkle and Gompert 2013</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Maruki and Lynch 2014</a>) found that sequencing as many individuals as possible with low depths of coverage is the optimal strategy for estimating population-level parameters without calling genotypes with limited budgets. However, when depths of coverage are low, statistical uncertainty of individual sequence data are high and calling genotypes is difficult. Therefore, when inferring accurate genotypes is critical for a study, it becomes necessary to sequence a more limited number of individuals with higher depths of coverage.</p><p id="p-57">In this study, we investigated how estimating genotype frequencies and error rates from a population sample and incorporating the population-level information into genotype-calling processes improves the performance of genotype calling, especially when the population deviates from HWE. We also examined the situation when individual coverage is high enough to accurately call genotypes without prior population-level estimates. To promote the use of our methods, we freely provide C++ programs implementing our genotype callers, along with their documentation (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/FileS3.pdf">File S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/FileS4.pdf">File S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/FileS5.zip">File S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/FileS6.pdf">File S6</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/FileS7.zip">File S7</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/FileS8.pdf">File S8</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/FileS9.zip">File S9</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/FileS10.pdf">File S10</a>).</p><p id="p-58">Consistent with previous results (<a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">Kim <em>et al.</em> 2011</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Han <em>et al.</em> 2014</a>), we find that allele-frequency estimates via genotype calling are biased, whereas those directly estimated from sequence read data are unbiased when depths of coverage are low. Given the importance of unbiased estimates of allele frequencies for subsequent population-genetic analyses, this reinforces the importance of estimating allele frequencies directly from sequence reads with low depths of coverage, as in <a id="xref-ref-44-10" class="xref-bibr" href="#ref-44">Maruki and Lynch (2015)</a>. Our BGC takes advantage of prior population-level estimates of genotype frequencies and error rates to improve the accuracy of genotype calls in a population with an arbitrary mating system and internal population structure. The higher accuracy of called genotypes by BGC than obtained with currently widely used genotype callers in the majority of examined cases shows that our approach improves the ability to call genotypes while also providing unbiased estimates of allele frequencies.</p><p id="p-59">Our performance evaluation of the HGC revealed that the rate of correctly calling homozygotes can be high if the minimum coverage required for calling genotypes is set at six or greater. By raising the minimum coverage cutoff to eight or greater, the rate for correctly calling heterozygotes can also be reasonably high. The power analyses of HGC for detecting polymorphisms and triallelic sites also indicate that the method is conservative and reasonably sensitive. The comparison of the performance of HGC to that of other existing methods showed that our method yields more accurate or as accurate diploid genotype calls at sites with more than two alleles.</p><p id="p-60">The application of HGC to high-throughput sequencing data of 83 <em>Daphnia</em> clones from a population indicates that a non-negligible fraction of polymorphisms is triallelic and even tetra-allelic in <em>D. pulex</em>. There is growing evidence that triallelic polymorphisms exist in the human genome (<em>e.g.</em>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hodgkinson and Eyre-Walker 2010</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Nelson <em>et al.</em> 2012</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cao <em>et al.</em> 2015</a>). In particular, a recent study sequencing many samples with high depths of coverage by <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Nelson <em>et al.</em> (2012)</a> found that the fraction of triallelic sites is much higher than that previously expected. Furthermore, a recent study (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Jenkins <em>et al.</em> 2014</a>) found triallelic polymorphisms may be more informative for demographic inferences than biallelic polymorphisms. Therefore, it is important to relax the assumption of biallelic polymorphisms. Our method provides an efficient, flexible, and statistically rigorous framework for identifying polymorphic sites containing arbitrary numbers of alleles.</p><p id="p-61">Because of its efficiency and flexibility, our method can be extended to analyses of population-genomic data from polyploid organisms. Our performance evaluation of the TRI revealed that much higher coverage is needed for correctly calling triploid genotypes than in the case of diploids. This conclusion is extended to genotype calling of organisms with higher ploidy, as the difficulties we found become greater. These results will help researchers to design sequencing strategies for population-genomic analyses of polyploid organisms.</p><p id="p-62">As guidance on proper usage of the proposed four different genotype-calling methods, we provided a summary of the proposed methods (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). BGC and HGC are both intended for calling diploid genotypes. Users should first apply HGC to population-genomic data of diploid organisms to identify sites with more than two alleles. Then, users should apply BGC to the data at biallelic sites, as BGC yields more accurate genotype calls than HGC at biallelic sites, unless the depth of coverage is extremely high. To facilitate this procedure, we provide a C++ program and documentation for setting the coverage of all individuals at zero at sites with more than two alleles as identified by HGC (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/FileS11.zip">File S11</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1/FileS12.pdf">File S12</a>). TRI and TET are extensions of HGC to triploid and tetraploid data, respectively. They enable highly efficient genotype calling at sites with arbitrary numbers of alleles. If future studies enable the population-level estimation of the genotype frequencies and error rates in diploid data at sites with arbitrary numbers of alleles and triploid and tetraploid data, we can, in principle, improve the accuracy of genotype calls from these data, although it will be very computationally expensive to estimate frequencies of many genotypes in these cases.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484291/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484291/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Summary of the proposed methods</span><div class="sb-div caption-clear"></div></div></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-63">We thank two anonymous reviewers, whose comments improved our manuscript. This work was supported by National Science Foundation grant DEB-1257806 and National Institutes of Health grant NIH-NIGMS R01-GM101672. It was also supported by the National Center for Genome Analysis Support, funded by National Science Foundation grant DBI-1458641 to Indiana University, and Indiana University Research Technology’s computational resources.</p><p id="p-64"><em>Note added in proof:</em> See Ye <em>et al</em>. 2017 (pp. <a href="http://www.g3journal.org/content/7/5/1405.full">1405</a>) in this issue and Ackerman <em>et al.</em> 2017 (pp. <a href="http://www.genetics.org/content/206/1/105.full">105</a>) and Lynch <em>et al</em>. 2017 (pp. <a href="http://www.genetics.org/content/206/1/315.full">315</a>) in the GENETICS May issue for related work.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-65">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039008/-/DC1</a>.</p></li><li class="fn" id="fn-6"><p id="p-66"><em>Communicating editor: S. I. Wright</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-09-02"><span class="received-label">Received </span>September 2, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-01-06"><span class="accepted-label">Accepted </span>January 6, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Maruki and Lynch</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.1" data-doi="10.1111/j.1365-294X.2006.02889.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aars</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dallas</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piertney</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gow</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Widespread gene flow and high genetic variability in populations of water voles Arvicola terrestris in patchy habitats.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1455</span>–<span class="cit-lpage">1466</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DAars%26rft.auinit1%253DJ.%26rft.volume%253D15%26rft.issue%253D6%26rft.spage%253D1455%26rft.epage%253D1466%26rft.atitle%253DWidespread%2Bgene%2Bflow%2Band%2Bhigh%2Bgenetic%2Bvariability%2Bin%2Bpopulations%2Bof%2Bwater%2Bvoles%2BArvicola%2Bterrestris%2Bin%2Bpatchy%2Bhabitats.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2006.02889.x%26rft_id%253Dinfo%253Apmid%252F16629803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2006.02889.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16629803&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Black</span>  <span class="cit-name-given-names">F. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzano</span>  <span class="cit-name-given-names">F. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Evidence for heterosis in the HLA system.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">894</span>–<span class="cit-lpage">899</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBlack%26rft.auinit1%253DF.%2BL.%26rft.volume%253D33%26rft.issue%253D6%26rft.spage%253D894%26rft.epage%253D899%26rft.atitle%253DEvidence%2Bfor%2Bheterosis%2Bin%2Bthe%2BHLA%2Bsystem.%26rft_id%253Dinfo%253Apmid%252F7325154%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=7325154&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1981MV19900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.3" data-doi="10.1146/annurev.ento.46.1.441"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Black</span>  <span class="cit-name-given-names">W. C.</span>  <span class="cit-name-suffix">IV</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baer</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antolin</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DuTeau</span>  <span class="cit-name-given-names">N. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Population genomics: genome-wide sampling of insect populations.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Entomol.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">441</span>–<span class="cit-lpage">469</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bentomology%26rft.stitle%253DAnnu%2BRev%2BEntomol%26rft.aulast%253DBlack%2BIV%26rft.auinit1%253DW.%2BC.%26rft.volume%253D46%26rft.issue%253D1%26rft.spage%253D441%26rft.epage%253D469%26rft.atitle%253DPOPULATION%2BGENOMICS%253A%2BGenome-Wide%2BSampling%2Bof%2BInsect%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ento.46.1.441%26rft_id%253Dinfo%253Apmid%252F11112176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ento.46.1.441&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11112176&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167877300016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.4" data-doi="10.1016/0040-5809(79)90025-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">A. H. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Enzyme polymorphism in plant-populations.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BPopul.%2BBiol.%26rft.volume%253D15%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252879%252990025-X%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(79)90025-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1979GU66000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.5" data-doi="10.1016/j.ajhg.2015.11.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genotype imputation with millions of reference samples.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">116</span>–<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D98%26rft.spage%253D116%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2015.11.020%26rft_id%253Dinfo%253Apmid%252F26748515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2015.11.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26748515&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.6" data-doi="10.1086/521987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">1084</span>–<span class="cit-lpage">1097</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DS.%2BR.%26rft.volume%253D81%26rft.issue%253D5%26rft.spage%253D1084%26rft.epage%253D1097%26rft.atitle%253DRapid%2Band%2Baccurate%2Bhaplotype%2Bphasing%2Band%2Bmissing-data%2Binference%2Bfor%2Bwhole-genome%2Bassociation%2Bstudies%2Bby%2Buse%2Bof%2Blocalized%2Bhaplotype%2Bclustering.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F521987%26rft_id%253Dinfo%253Apmid%252F17924348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/521987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17924348&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250480900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.7" data-doi="10.1111/mec.12105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buerkle</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gompert</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Population genomics based on low coverage sequencing: how low should we go?</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3028</span>–<span class="cit-lpage">3035</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3028%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12105&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Analysis of human triallelic SNPs by next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Ann. Hum. Genet.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">275</span>–<span class="cit-lpage">281</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BHum.%2BGenet.%26rft.volume%253D79%26rft.spage%253D275%26rft_id%253Dinfo%253Apmid%252F25907303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25907303&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.9" data-doi="10.1111/mec.12354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassham</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Stacks: an analysis tool set for population genomics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3124</span>–<span class="cit-lpage">3140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3124%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12354%26rft_id%253Dinfo%253Apmid%252F23701397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23701397&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319675300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Postlethwait</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Stacks: building and genotyping loci <em>de novo</em> from short-read sequences.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D1%26rft.spage%253D171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.11" data-doi="10.1017/S0016672300017547"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cockerham</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Digenic descent measures for finite populations.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">121</span>–<span class="cit-lpage">147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D30%26rft.spage%253D121%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300017547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300017547&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1977EA96700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.12" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMTUvMjE1NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xMzkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.13" data-doi="10.1046/j.1365-294X.2002.01478.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Delmotte</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leterme</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gauthier</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rispe</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genetic architecture of sexual and asexual populations of the aphid Rhopalosiphum padi based on allozyme and microsatellite markers.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">711</span>–<span class="cit-lpage">723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DDelmotte%26rft.auinit1%253DF.%26rft.volume%253D11%26rft.issue%253D4%26rft.spage%253D711%26rft.epage%253D723%26rft.atitle%253DGenetic%2Barchitecture%2Bof%2Bsexual%2Band%2Basexual%2Bpopulations%2Bof%2Bthe%2Baphid%2BRhopalosiphum%2Bpadi%2Bbased%2Bon%2Ballozyme%2Band%2Bmicrosatellite%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-294X.2002.01478.x%26rft_id%253Dinfo%253Apmid%252F11972759%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-294X.2002.01478.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11972759&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175250300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.14" data-doi="10.1038/ng.806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garimella</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478889&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.15" data-doi="10.1111/j.1365-294X.2006.03093.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amos</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Inbreeding depression and multiple regions showing heterozygote advantage in Drosophila melanogaster exposed to stress.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">3885</span>–<span class="cit-lpage">3893</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DFerreira%26rft.auinit1%253DA.%2BG.%26rft.volume%253D15%26rft.issue%253D13%26rft.spage%253D3885%26rft.epage%253D3893%26rft.atitle%253DInbreeding%2Bdepression%2Band%2Bmultiple%2Bregions%2Bshowing%2Bheterozygote%2Badvantage%2Bin%2BDrosophila%2Bmelanogaster%2Bexposed%2Bto%2Bstress.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2006.03093.x%26rft_id%253Dinfo%253Apmid%252F17054490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2006.03093.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17054490&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.16" data-doi="10.2307/2408336"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foltz</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoogland</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Genetic-evidence of outbreeding in the black-tailed prairie dog (Cynomys-Ludovicianus).</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">281</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D37%26rft.spage%253D273%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2408336%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2408336&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1983QJ41700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic diversity analysis of highly incomplete SNP genotype data with imputations: an empirical assessment.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">891</span>–<span class="cit-lpage">900</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D4%26rft.spage%253D891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.18" data-doi="10.1038/nature15393"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">1000 Genomes Project Consortium</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrison</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A global reference for human genetic variation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">526</span>: <span class="cit-fpage">68</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D526%26rft.spage%253D68%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature15393%26rft_id%253Dinfo%253Apmid%252F26432245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature15393&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26432245&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.19" data-doi="10.1111/j.1755-0998.2011.03024.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glenn</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Field guide to next-generation DNA sequencers.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">759</span>–<span class="cit-lpage">769</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D11%26rft.spage%253D759%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2011.03024.x%26rft_id%253Dinfo%253Apmid%252F21592312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2011.03024.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21592312&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.20" data-doi="10.1093/molbev/mst229"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinsheimer</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Characterizing bias in population genetic inferences from low-coverage sequencing data.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">723</span>–<span class="cit-lpage">735</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst229%26rft_id%253Dinfo%253Apmid%252F24288159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMxLzMvNzIzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzEzOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.21" data-doi="10.1111/j.1469-185X.1978.tb00860.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hebert</span>  <span class="cit-name-given-names">P. D. N.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">Population biology of Daphnia (Crustacea, Daphnidae).</span> <abbr class="cit-jnl-abbrev">Biol. Rev. Camb. Philos. Soc.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">426</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BRev.%2BCamb.%2BPhilos.%2BSoc.%26rft.volume%253D53%26rft.spage%253D387%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-185X.1978.tb00860.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-185X.1978.tb00860.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.22" data-doi="10.1023/A:1026494212540"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hedrick</span>  <span class="cit-name-given-names">P. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Balancing selection and MHC.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.stitle%253DGenetica%26rft.aulast%253DHedrick%26rft.auinit1%253DP.%2BW.%26rft.volume%253D104%26rft.issue%253D3%26rft.spage%253D207%26rft.epage%253D214%26rft.atitle%253DBalancing%2Bselection%2Band%2BMHC.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1026494212540%26rft_id%253Dinfo%253Apmid%252F10386384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1026494212540&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10386384&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000080245100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.23" data-doi="10.1534/genetics.109.110510"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hodgkinson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Human triallelic sites: evidence for a new mutational mechanism?</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">184</span>: <span class="cit-fpage">233</span>–<span class="cit-lpage">241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.110510%26rft_id%253Dinfo%253Apmid%252F19884308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg0LzEvMjMzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzEzOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.24" data-doi="10.1371/journal.pgen.1000862"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassham</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stiffler</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Population genomics of parallel adaptation in threespine stickleback using sequenced RAD tags.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000862</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHohenlohe%26rft.auinit1%253DP.%2BA.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253De1000862%26rft.epage%253De1000862%26rft.atitle%253DPopulation%2Bgenomics%2Bof%2Bparallel%2Badaptation%2Bin%2Bthreespine%2Bstickleback%2Busing%2Bsequenced%2BRAD%2Btags.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000862%26rft_id%253Dinfo%253Apmid%252F20195501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000862&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20195501&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>  <span class="cit-name-given-names">N. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Statistical properties of the number of recombination events in the history of a sample of DNA sequences.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">147</span>–<span class="cit-lpage">164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHudson%26rft.auinit1%253DR.%2BR.%26rft.volume%253D111%26rft.issue%253D1%26rft.spage%253D147%26rft.epage%253D164%26rft.atitle%253DSTATISTICAL%2BPROPERTIES%2BOF%2BTHE%2BNUMBER%2BOF%2BRECOMBINATION%2BEVENTS%2BIN%2BTHE%2BHISTORY%2BOF%2BA%2BSAMPLE%2BOF%2BDNA%2BSEQUENCES%26rft_id%253Dinfo%253Apmid%252F4029609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTExLzEvMTQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzEzOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.26" data-doi="10.1186/gb-2013-14-5-r47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kikuchi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanders</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newbold</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berriman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">REAPR: a universal tool for genome assembly evaluation.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR47%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-5-r47%26rft_id%253Dinfo%253Apmid%252F23710727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-5-r47&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23710727&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.27" data-doi="10.1038/nature06258"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">International HapMap Consortium</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frazer</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ballinger</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinds</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A second generation human haplotype map of over 3.1 million SNPs.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">449</span>: <span class="cit-fpage">851</span>–<span class="cit-lpage">861</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DFrazer%26rft.auinit1%253DK.%2BA.%26rft.volume%253D449%26rft.issue%253D7164%26rft.spage%253D851%26rft.epage%253D861%26rft.atitle%253DA%2Bsecond%2Bgeneration%2Bhuman%2Bhaplotype%2Bmap%2Bof%2Bover%2B3.1%2Bmillion%2BSNPs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06258%26rft_id%253Dinfo%253Apmid%252F17943122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06258&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17943122&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250230600036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.28" data-doi="10.1038/nature09298"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">International HapMap 3 Consortium</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altshuler</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibbs</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peltonen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altshuler</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Integrating common and rare genetic variation in diverse human populations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">52</span>–<span class="cit-lpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAltshuler%26rft.auinit1%253DD.%2BM.%26rft.volume%253D467%26rft.issue%253D7311%26rft.spage%253D52%26rft.epage%253D58%26rft.atitle%253DIntegrating%2Bcommon%2Band%2Brare%2Bgenetic%2Bvariation%2Bin%2Bdiverse%2Bhuman%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09298%26rft_id%253Dinfo%253Apmid%252F20811451%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09298&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20811451&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281461200033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.29" data-doi="10.1534/genetics.113.158584"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">General triallelic frequency spectrum under demographic models with variable population size.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">311</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.158584%26rft_id%253Dinfo%253Apmid%252F24214345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzEvMjk1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzEzOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.30" data-doi="10.1159/000084979"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jurka</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kapitonov</span>  <span class="cit-name-given-names">V. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavlicek</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klonowski</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohany</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Repbase update, a database of eukaryotic repetitive elements.</span> <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">462</span>–<span class="cit-lpage">467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetic%2Band%2Bgenome%2Bresearch%26rft.stitle%253DCytogenet%2BGenome%2BRes%26rft.aulast%253DJurka%26rft.auinit1%253DJ.%26rft.volume%253D110%26rft.issue%253D1-4%26rft.spage%253D462%26rft.epage%253D467%26rft.atitle%253DRepbase%2BUpdate%252C%2Ba%2Bdatabase%2Bof%2Beukaryotic%2Brepetitive%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000084979%26rft_id%253Dinfo%253Apmid%252F16093699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000084979&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16093699&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231064600047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-book" id="cit-7.5.1393.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kendall</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stuart</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-source"><em>The Advanced Theory of Statistics</em></span>, Vol. <span class="cit-vol">2</span>, <span class="cit-edition">Ed. 4</span>. <span class="cit-publ-name">Charles Griffin &amp; Co.</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.32" data-doi="10.1186/1471-2105-12-231"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohmueller</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korneliussen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Estimation of allele frequency and association mapping using next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DKim%26rft.auinit1%253DS.%2BY.%26rft.volume%253D12%26rft.spage%253D231%26rft.epage%253D231%26rft.atitle%253DEstimation%2Bof%2Ballele%2Bfrequency%2Band%2Bassociation%2Bmapping%2Busing%2Bnext-generation%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-231%26rft_id%253Dinfo%253Apmid%252F21663684%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-231&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21663684&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.33" data-doi="10.1186/s12859-014-0356-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korneliussen</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">ANGSD: analysis of next generation sequencing data.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">356</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D15%26rft.spage%253D356%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-014-0356-4%26rft_id%253Dinfo%253Apmid%252F25420514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-014-0356-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25420514&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.34" data-doi="10.1093/bioinformatics/btr509"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2987</span>–<span class="cit-lpage">2993</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr509%26rft_id%253Dinfo%253Apmid%252F21903627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMjEvMjk4NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xMzkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.35" data-doi="10.1101/gr.078212.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mapping short DNA sequencing reads and calling variants using mapping quality scores.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1851</span>–<span class="cit-lpage">1858</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.078212.108%26rft_id%253Dinfo%253Apmid%252F18714091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMS8xODUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzEzOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.36" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">a</span> <span class="cit-article-title">The sequence alignment/map format and samtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xMzkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.37" data-doi="10.1101/gr.088013.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">b</span> <span class="cit-article-title">SNP detection for massively parallel whole-genome resequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1124</span>–<span class="cit-lpage">1132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.088013.108%26rft_id%253Dinfo%253Apmid%252F19420381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzYvMTEyNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xMzkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.38" data-doi="10.1002/gepi.20533"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willer</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheet</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">816</span>–<span class="cit-lpage">834</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DLi%26rft.auinit1%253DY.%26rft.volume%253D34%26rft.issue%253D8%26rft.spage%253D816%26rft.epage%253D834%26rft.atitle%253DMaCH%253A%2Busing%2Bsequence%2Band%2Bgenotype%2Bdata%2Bto%2Bestimate%2Bhaplotypes%2Band%2Bunobserved%2Bgenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20533%26rft_id%253Dinfo%253Apmid%252F21058334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20533&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21058334&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284719100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.39" data-doi="10.1534/genetics.109.100479"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Estimation of allele frequencies from high-coverage genome-sequencing projects.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.100479%26rft_id%253Dinfo%253Apmid%252F19293142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMjk1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzEzOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gutenkunst</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ackerman</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spitze</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Population genomics of <em>Daphnia pulex</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">332</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D206%26rft.spage%253D315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Markow</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hedrick</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zuerlein</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danilovs</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">HLA polymorphism in the Havasupai: evidence for balancing selection.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">943</span>–<span class="cit-lpage">952</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DMarkow%26rft.auinit1%253DT.%26rft.volume%253D53%26rft.issue%253D4%26rft.spage%253D943%26rft.epage%253D952%26rft.atitle%253DHLA%2Bpolymorphism%2Bin%2Bthe%2BHavasupai%253A%2Bevidence%2Bfor%2Bbalancing%2Bselection.%26rft_id%253Dinfo%253Apmid%252F8213824%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8213824&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LY62700020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.42" data-doi="10.1093/bioinformatics/btq526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kinnamon</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zuchner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">SeqEM: an adaptive genotype-calling approach for next-generation sequencing studies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2803</span>–<span class="cit-lpage">2810</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq526%26rft_id%253Dinfo%253Apmid%252F20861027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMjIvMjgwMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xMzkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.43" data-doi="10.1534/genetics.114.165514"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maruki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide estimation of linkage disequilibrium from population-level high-throughput sequencing data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">1303</span>–<span class="cit-lpage">1313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.165514%26rft_id%253Dinfo%253Apmid%252F24875187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Ny80LzEzMDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTM5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.44" data-doi="10.1534/genetics.115.179077"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maruki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genotype-frequency estimation from high-throughput sequencing data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">473</span>–<span class="cit-lpage">486</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.179077%26rft_id%253Dinfo%253Apmid%252F26224735%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAxLzIvNDczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzEzOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.45" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xMzkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.46" data-doi="10.1007/BF00055255"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Melnick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">The genetic consequences of primate social organization: a review of macaques, baboons and vervet monkeys.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">117</span>–<span class="cit-lpage">135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.stitle%253DGenetica%26rft.aulast%253DMelnick%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D73%26rft.issue%253D1-2%26rft.spage%253D117%26rft.epage%253D135%26rft.atitle%253DThe%2Bgenetic%2Bconsequences%2Bof%2Bprimate%2Bsocial%2Borganization%253A%2Ba%2Breview%2Bof%2Bmacaques%252C%2Bbaboons%2Band%2Bvervet%2Bmonkeys.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00055255%26rft_id%253Dinfo%253Apmid%252F3505885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00055255&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3505885&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987K049300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.47" data-doi="10.1126/science.1217876"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegmann</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehm</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kessner</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">St Jean</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An abundance of rare functional variants in 202 drug target genes sequenced in 14,002 people.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">100</span>–<span class="cit-lpage">104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1217876%26rft_id%253Dinfo%253Apmid%252F22604722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzcvNjA5MC8xMDAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTM5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.48" data-doi="10.1371/journal.pone.0037558"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korneliussen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SNP calling, genotype calling, and sample allele frequency estimation from new-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e37558</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DNielsen%26rft.auinit1%253DR.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De37558%26rft.epage%253De37558%26rft.atitle%253DSNP%2Bcalling%252C%2Bgenotype%2Bcalling%252C%2Band%2Bsample%2Ballele%2Bfrequency%2Bestimation%2Bfrom%2BNew-Generation%2BSequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0037558%26rft_id%253Dinfo%253Apmid%252F22911679%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0037558&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22911679&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.49" data-doi="10.1101/gr.079509.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pool</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hellmann</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Population genetic inference from genomic sequence variation.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.079509.108%26rft_id%253Dinfo%253Apmid%252F20067940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzMvMjkxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzEzOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.50" data-doi="10.1186/1471-2164-13-341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quail</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coupland</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A tale of three next generation sequencing platforms: comparison of Ion Torrent, Pacific Biosciences and Illumina MiSeq sequencers.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DQuail%26rft.auinit1%253DM.%26rft.volume%253D13%26rft.spage%253D341%26rft.epage%253D341%26rft.atitle%253DA%2Btale%2Bof%2Bthree%2Bnext%2Bgeneration%2Bsequencing%2Bplatforms%253A%2Bcomparison%2Bof%2BIon%2BTorrent%252C%2BPacific%2BBiosciences%2Band%2BIllumina%2BMiSeq%2Bsequencers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-341%26rft_id%253Dinfo%253Apmid%252F22827831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-341&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22827831&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.51" data-doi="10.1086/502802"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scheet</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">629</span>–<span class="cit-lpage">644</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DScheet%26rft.auinit1%253DP.%26rft.volume%253D78%26rft.issue%253D4%26rft.spage%253D629%26rft.epage%253D644%26rft.atitle%253DA%2Bfast%2Band%2Bflexible%2Bstatistical%2Bmodel%2Bfor%2Blarge-scale%2Bpopulation%2Bgenotype%2Bdata%253A%2Bapplications%2Bto%2Binferring%2Bmissing%2Bgenotypes%2Band%2Bhaplotypic%2Bphase.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F502802%26rft_id%253Dinfo%253Apmid%252F16532393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/502802&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16532393&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236755900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.52" data-doi="10.1093/nar/gkv1275"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Speir</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zweig</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenbloom</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raney</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paten</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The UCSC genome browser database: 2016 update.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">D717</span>–<span class="cit-lpage">D725</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1275%26rft_id%253Dinfo%253Apmid%252F26590259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0NC9EMS9ENzE3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzEzOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.53" data-doi="10.1554/0014-3820(2001)055[1224:GCOPAS]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storz</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhat</span>  <span class="cit-name-given-names">H. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunz</span>  <span class="cit-name-given-names">T. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Genetic consequences of polygyny and social structure in an Indian fruit bat, Cynopterus sphinx. II. Variance in male mating success and effective population size.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1224</span>–<span class="cit-lpage">1232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DStorz%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D55%26rft.issue%253D6%26rft.spage%253D1224%26rft.epage%253D1232%26rft.atitle%253DGenetic%2Bconsequences%2Bof%2Bpolygyny%2Band%2Bsocial%2Bstructure%2Bin%2Ban%2BIndian%2Bfruit%2Bbat%252C%2BCynopterus%2Bsphinx.%2BII.%2BVariance%2Bin%2Bmale%2Bmating%2Bsuccess%2Band%2Beffective%2Bpopulation%2Bsize.%26rft_id%253Dinfo%253Adoi%252F10.1554%252F0014-3820%25282001%2529055%255B1224%253AGCOPAS%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F11475058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/0014-3820(2001)055[1224:GCOPAS]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11475058&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169903000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.54" data-doi="10.1046/j.1365-294x.1998.00385.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tarr</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conant</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fleischer</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Founder events and variation at microsatellite loci in an insular passerine bird, the Laysan finch (Telespiza cantans).</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">719</span>–<span class="cit-lpage">731</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D7%26rft.spage%253D719%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-294x.1998.00385.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-294x.1998.00385.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000074236300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.55" data-doi="10.1111/j.1420-9101.2008.01563.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tollenaere</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryja</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cadet</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deter</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Multiple parasites mediate balancing selection at two MHC class II genes in the fossorial water vole: insights from multivariate analyses and population genetics.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1307</span>–<span class="cit-lpage">1320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.stitle%253DJ%2BEvol%2BBiol%26rft.aulast%253DTollenaere%26rft.auinit1%253DC.%26rft.volume%253D21%26rft.issue%253D5%26rft.spage%253D1307%26rft.epage%253D1320%26rft.atitle%253DMultiple%2Bparasites%2Bmediate%2Bbalancing%2Bselection%2Bat%2Btwo%2BMHC%2Bclass%2BII%2Bgenes%2Bin%2Bthe%2Bfossorial%2Bwater%2Bvole%253A%2Binsights%2Bfrom%2Bmultivariate%2Banalyses%2Band%2Bpopulation%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1420-9101.2008.01563.x%26rft_id%253Dinfo%253Apmid%252F18624885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1420-9101.2008.01563.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18624885&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258379600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van der Auwera</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carneiro</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Del Angel</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">From FastQ data to high confidence variant calls: the Genome Analysis Toolkit best practices pipeline.</span> <abbr class="cit-jnl-abbrev">Curr. Protoc. Bioinformatics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">11.10.1</span>–<span class="cit-lpage">1.10.33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BProtoc.%2BBioinformatics%26rft.volume%253D11%26rft.spage%253D11101%26rft_id%253Dinfo%253Apmid%252F25431634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25431634&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1393.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1393.57" data-doi="10.1101/gr.157388.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vieira</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fumagalli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Estimating inbreeding coefficients from NGS data: impact on genotype calling and allele frequency estimation.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1852</span>–<span class="cit-lpage">1861</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.157388.113%26rft_id%253Dinfo%253Apmid%252F23950147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMy8xMS8xODUyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzEzOTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-book" id="cit-7.5.1393.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source"><em>Genetic Data Analysis II: Methods for Discrete Population Genetic Data</em></span>. <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/5/1393.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/5/1385" title="Accurate Classification of Protein Subcellular Localization from High-Throughput Microscopy Images Using Deep Learning" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/5/1405" title="A New Reference Genome Assembly for the Microcrustacean Daphnia pulex" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/5"><span>Volume 7 Issue 5, May 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/5.cover-source.jpg?itok=VWLvGstM" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1393.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-2g-pEpubZbPVWbqFh2hVpYCctywbZNr0bliQubfI9sY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/5/1393" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/5/1393" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/5/1393" class="active" data-icon-position="" data-hide-link-title="0">Genotype Calling from Population-Genomic Sequencing Data</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/484252" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-mgAx4T7DMLThnz0TTery1RRGP5vmGF9Babx91YT3DUY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/5/1393.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/5/1393" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Xlh_pFbtJifV2j8wyNjPKHwKwfHp1tq_1auO2CIdQd0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/5/1393" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/5/1393.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484252" id="node484252--2" data-pisa="ggg;7/5/1393" data-pisa-master="ggg;g3.117.039008" data-apath="/ggg/7/5/1393.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genotype Calling from Population-Genomic Sequencing Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Takahiro Maruki</span> and <span class="highwire-citation-author" data-delta="1">Michael Lynch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1393-1404;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.039008 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484252--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Takahiro Maruki </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Takahiro%2BMaruki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maruki%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATakahiro%2BMaruki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tmaruki@indiana.edu" class="" data-icon-position="" data-hide-link-title="0">tmaruki@indiana.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michael Lynch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BLynch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lynch%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BLynch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/5/1393" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/5/1393&amp;title=Genotype%20Calling%20from%20Population-Genomic%20Sequencing%20Data" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/5/1393&amp;title=Genotype%20Calling%20from%20Population-Genomic%20Sequencing%20Data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/5/1393&amp;title=Genotype%20Calling%20from%20Population-Genomic%20Sequencing%20Data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/5/1393&amp;text=Genotype%20Calling%20from%20Population-Genomic%20Sequencing%20Data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/5/1393&amp;title=Genotype%20Calling%20from%20Population-Genomic%20Sequencing%20Data" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/5/1393&amp;t=Genotype%20Calling%20from%20Population-Genomic%20Sequencing%20Data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/5/1393&amp;title=Genotype%20Calling%20from%20Population-Genomic%20Sequencing%20Data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/5/1393&amp;title=Genotype%20Calling%20from%20Population-Genomic%20Sequencing%20Data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/484252/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484252" id="node484252--3" data-pisa="ggg;7/5/1393" data-pisa-master="ggg;g3.117.039008" data-apath="/ggg/7/5/1393.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genotype Calling from Population-Genomic Sequencing Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Takahiro Maruki</span> and <span class="highwire-citation-author" data-delta="1">Michael Lynch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1393-1404;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.039008 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484252--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Takahiro Maruki </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Takahiro%2BMaruki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maruki%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATakahiro%2BMaruki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tmaruki@indiana.edu" class="" data-icon-position="" data-hide-link-title="0">tmaruki@indiana.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michael Lynch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BLynch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lynch%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BLynch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1393&amp;rft.epage=1404&amp;rft.atitle=Genotype%20Calling%20from%20Population-Genomic%20Sequencing%20Data&amp;rft.volume=7&amp;rft.issue=5&amp;rft.date=2017-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Maruki%2C+Takahiro&amp;rft.au=Lynch%2C+Michael"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/484252/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/484252/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/484252/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/484252/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/484252/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/484252/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/484252/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/484252/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/484252/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/484252/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/484252/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/484252/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/5/1393" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/5/1393.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/5/1393.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/5/1393.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/5/1393.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
