<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/5/1385" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/5/1385.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/5/1385.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/5/1385.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/5/1385.atom" />
<meta name="HW.pisa" content="ggg;7/5/1385" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Accurate Classification of Protein Subcellular Localization from High-Throughput Microscopy Images Using Deep Learning" />
<meta name="DC.Identifier" content="10.1534/g3.116.033654" />
<meta name="DC.Date" content="2017-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Parnamaa and Parts. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="High-throughput microscopy of many single cells generates high-dimensional data that are far from straightforward to analyze. One important problem is automatically detecting the cellular compartment where a fluorescently-tagged protein resides, a task relatively simple for an experienced human, but difficult to automate on a computer. Here, we train an 11-layer neural network on data from mapping thousands of yeast proteins, achieving per cell localization classification accuracy of 91%, and per protein accuracy of 99% on held-out images. We confirm that low-level network features correspond to basic image characteristics, while deeper layers separate localization classes. Using this network as a feature calculator, we train standard classifiers that assign proteins to previously unseen compartments after observing only a small number of training examples. Our results are the most accurate subcellular localization classifications to date, and demonstrate the usefulness of deep learning for high-throughput microscopy." />
<meta name="DC.Contributor" content="Tanel Pärnamaa" />
<meta name="DC.Contributor" content="Leopold Parts" />
<meta name="article:published_time" content="2017-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Accurate Classification of Protein Subcellular Localization from High-Throughput Microscopy Images Using Deep Learning" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;High-throughput microscopy of many single cells generates high-dimensional data that are far from straightforward to analyze. One important problem is automatically detecting the cellular compartment where a fluorescently-tagged protein resides, a task relatively simple for an experienced human, but difficult to automate on a computer. Here, we train an 11-layer neural network on data from mapping thousands of yeast proteins, achieving per cell localization classification accuracy of 91%, and per protein accuracy of 99% on held-out images. We confirm that low-level network features correspond to basic image characteristics, while deeper layers separate localization classes. Using this network as a feature calculator, we train standard classifiers that assign proteins to previously unseen compartments after observing only a small number of training examples. Our results are the most accurate subcellular localization classifications to date, and demonstrate the usefulness of deep learning for high-throughput microscopy.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/05/01" />
<meta name="citation_mjid" content="ggg;7/5/1385" />
<meta name="citation_id" content="7/5/1385" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/5/1385" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/5/1385.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/5/1385.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/5/1385.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.033654" />
<meta name="citation_pmid" content="28391243" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1385" />
<meta name="citation_lastpage" content="1392" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Tanel Pärnamaa" />
<meta name="citation_author_institution" content="Institute of Computer Science, University of Tartu, 50409, Estonia" />
<meta name="citation_author" content="Leopold Parts" />
<meta name="citation_author_institution" content="Institute of Computer Science, University of Tartu, 50409, Estonia" />
<meta name="citation_author_institution" content="Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire CB10 1SA, United Kingdom" />
<meta name="citation_author_email" content="leopold.parts@sanger.ac.uk" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=F. W. Albert;citation_author=S. Treusch;citation_author=A. H. Shockley;citation_author=J. S. Bloom;citation_author=L. Kruglyak;citation_title=Genetics of single-cell protein abundance variation in large yeast populations.;citation_pages=494-497;citation_volume=506;citation_year=2014;citation_issue=7489;citation_pmid=24402228;citation_doi=10.1038/nature12904" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=B. Alipanahi;citation_author=A. Delong;citation_author=M. T. Weirauch;citation_author=B. J. Frey;citation_title=Predicting the sequence specificities of DNA- and RNA-binding proteins by deep learning.;citation_pages=831-838;citation_volume=33;citation_year=2015;citation_pmid=26213851;citation_doi=10.1038/nbt.3300" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=C. Angermueller;citation_author=T. Parnamaa;citation_author=L. Parts;citation_author=O. Stegle;citation_title=Deep learning for computational biology;citation_pages=878-878;citation_volume=12;citation_year=2016;citation_issue=7;citation_pmid=27474269;citation_doi=10.15252/msb.20156651" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=M. V. Boland;citation_author=R. F. Murphy;citation_title=A neural network classifier capable of recognizing the patterns  of all major subcellular structures in fluorescence microscope  images of HeLa cells;citation_pages=1213-1223;citation_volume=17;citation_year=2001;citation_issue=12;citation_pmid=11751230;citation_doi=10.1093/bioinformatics/17.12.1213" />
<meta name="citation_reference" content="citation_journal_title=Cytometry;citation_journal_abbrev=Cytometry;citation_author=M. V. Boland;citation_author=M. K. Markey;citation_author=R. F. Murphy;citation_title=Automated recognition of patterns characteristic of subcellular structures in fluorescence microscopy images.;citation_pages=366-375;citation_volume=33;citation_year=1998;citation_issue=3;citation_pmid=9822349;citation_doi=10.1002/(SICI)1097-0320(19981101)33:3&lt;366::AID-CYTO12&gt;3.0.CO;2-R" />
<meta name="citation_reference" content="citation_journal_title=Curr. Protoc. Mol. Biol.;citation_author=M. A. Bray;citation_author=M. S. Vokes;citation_author=A. E. Carpenter;citation_title=Using CellProfiler for automatic identification and measurement of biological objects in images.;citation_pages=14171-14.17.13;citation_volume=109;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Mach. Learn.;citation_author=L. Breiman;citation_title=Random forests.;citation_pages=5-32;citation_volume=45;citation_year=2001;citation_issue=1;citation_doi=10.1023/A:1010933404324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=S. C. Chen;citation_author=T. Zhao;citation_author=G. J. Gordon;citation_author=R. F. Murphy;citation_title=Automated image analysis of protein localization in budding yeast;citation_pages=i66-i71;citation_volume=23;citation_year=2007;citation_issue=13;citation_pmid=17646347;citation_doi=10.1093/bioinformatics/btm206" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=Y. T. Chong;citation_author=J. L. Koh;citation_author=H. Friesen;citation_author=S. K. Duffy;citation_author=M. J. Cox;citation_title=Yeast proteome dynamics from single cell imaging and automated analysis.;citation_pages=1413-1424;citation_volume=161;citation_year=2015;citation_issue=6;citation_pmid=26046442;citation_doi=10.1016/j.cell.2015.04.051" />
<meta name="citation_reference" content="citation_title=Medical Image Computing and Computer-Assisted Intervention–MICCAI 2013;citation_title=Mitosis detection in breast cancer histology images with deep neural networks;citation_author=D. C. Cireşan;citation_author=A. Giusti;citation_author=L. M. Gambardella;citation_author=J. Schmidhuber;citation_pages=411-418;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Educational and Psychological Measurement;citation_journal_abbrev=Educational and Psychological Measurement;citation_author=J. Cohen;citation_title=A Coefficient of Agreement for Nominal Scales;citation_pages=37-46;citation_volume=20;citation_year=1960;citation_issue=1;citation_doi=10.1177/001316446002000104" />
<meta name="citation_reference" content="citation_journal_title=BioTechniques;citation_journal_abbrev=Biotechniques;citation_author=T. J. Collins;citation_title=High-content screening.;citation_pages=25- 29;citation_volume=43;citation_year=2007;citation_issue=1;citation_pmid=17695642;citation_doi=10.2144/000112517" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=C. Conrad;citation_author=H. Erfle;citation_author=P. Warnat;citation_author=N. Daigle;citation_author=T. Lorch;citation_title=Automatic Identification of Subcellular Phenotypes on Human Cell Arrays;citation_pages=1130-1136;citation_volume=14;citation_year=2004;citation_issue=6;citation_pmid=15173118;citation_doi=10.1101/gr.2383804" />
<meta name="citation_reference" content="citation_journal_title=Traffic (Copenhagen, Denmark);citation_journal_abbrev=Traffic;citation_author=A. Danckaert;citation_author=E. Gonzalez-Couto;citation_author=L. Bollondi;citation_author=N. Thompson;citation_author=B. Hayes;citation_title=Automated recognition of intracellular organelles in confocal microscope images.;citation_pages=66-73;citation_volume=3;citation_year=2002;citation_issue=1;citation_pmid=11872144" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=S. Farkash-Amar;citation_author=A. Zimmer;citation_author=E. Eden;citation_author=A. Cohen;citation_author=N. Geva-Zatorsky;citation_title=Noise genetics: inferring protein function by correlating phenotype with protein levels and localization in individual human cells.;citation_pages=e1004176;citation_volume=10;citation_year=2014;citation_issue=3" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=E. Glory;citation_author=R. F. Murphy;citation_title=Automated subcellular location determination and high-throughput microscopy.;citation_pages=7-16;citation_volume=12;citation_year=2007;citation_issue=1;citation_pmid=17199037;citation_doi=10.1016/j.devcel.2006.12.007" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=L. F. Handfield;citation_author=B. Strome;citation_author=Y. T. Chong;citation_author=A. M. Moses;citation_title=Local statistics allow quantification of cell-to-cell variability from high-throughput microscope images.;citation_volume=31;citation_year=2015;citation_issue=6;citation_pmid=25398614;citation_doi=10.1093/bioinformatics/btu759" />
<meta name="citation_reference" content="citation_journal_title=Proc. IEEE;citation_author=R. M. Haralick;citation_title=Statistical and structural approaches to texture.;citation_pages=786-804;citation_volume=67;citation_year=1979;citation_issue=5" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=W. K. Huh;citation_author=J. V. Falvo;citation_author=L. C. Gerke;citation_author=A. S. Carroll;citation_author=R. W. Howson;citation_title=Global analysis of protein localization in budding yeast.;citation_pages=686-691;citation_volume=425;citation_year=2003;citation_issue=6959;citation_pmid=14562095;citation_doi=10.1038/nature02026" />
<meta name="citation_reference" content="citation_journal_title=Pattern Recognit.;citation_author=A. K. Jain;citation_author=N. K. Ratha;citation_author=S. Lakshmanan;citation_title=Object detection using Gabor filters.;citation_pages=295-309;citation_volume=30;citation_year=1997;citation_issue=2;citation_doi=10.1016/S0031-3203(96)00068-4" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=L. Kamentsky;citation_author=T. R. Jones;citation_author=A. Fraser;citation_author=M. A. Bray;citation_author=D. J. Logan;citation_title=Improved structure, function and compatibility for CellProfiler: modular high-throughput image analysis software.;citation_volume=27;citation_year=2011;citation_issue=8;citation_pmid=21349861;citation_doi=10.1093/bioinformatics/btr095" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. R. Kelley;citation_author=J. Snoek;citation_author=J. Rinn;citation_title=Basset: learning the regulatory code of the accessible genome with deep convolutional neural networks.;citation_volume=26;citation_year=2016;citation_pmid=27197224;citation_doi=10.1101/gr.200535.115" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=J. L. Koh;citation_author=Y. T. Chong;citation_author=H. Friesen;citation_author=A. Moses;citation_author=C. Boone;citation_title=CYCLoPs: a comprehensive database constructed from automated analysis of protein abundance and subcellular localization patterns in Saccharomyces cerevisiae.;citation_pages=1223-1232;citation_volume=5;citation_year=2015;citation_issue=6" />
<meta name="citation_reference" content="citation_journal_title=Mol. Syst. Biol.;citation_title=Automated analysis of high-content microscopy data with deep learning.;citation_author=O.Z. Kraus;citation_author=B.T. Grys;citation_author=J. Ba;citation_author=Y. Chong;citation_author=B.J. Frey;citation_author=C. Boone;citation_author=B.J. Andrews;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=BioTechniques;citation_journal_abbrev=Biotechniques;citation_author=M. R. Lamprecht;citation_author=D. M. Sabatini;citation_author=A. E. Carpenter;citation_title=CellProfiler: free, versatile software for automated biological image analysis.;citation_pages=71-75;citation_volume=42;citation_year=2007;citation_issue=1;citation_pmid=17269487;citation_doi=10.2144/000112257" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=Y. LeCun;citation_author=Y. Bengio;citation_author=G. Hinton;citation_title=Deep learning.;citation_pages=436-444;citation_volume=521;citation_year=2015;citation_pmid=26017442;citation_doi=10.1038/nature14539" />
<meta name="citation_reference" content="citation_journal_title=R News;citation_author=A. Liaw;citation_author=M. Wiener;citation_title=Classification and regression by randomForest.;citation_pages=18-22;citation_volume=2;citation_year=2002;citation_issue=3" />
<meta name="citation_reference" content="citation_journal_title=J. VLSI Signal Process. Syst. Signal Image Video Technol.;citation_author=R. F. Murphy;citation_author=M. Velliste;citation_author=G. Porreca;citation_title=Robust numerical features for description and classification of subcellular location patterns in fluorescence microscope images.;citation_pages=311-321;citation_volume=35;citation_year=2003;citation_issue=3;citation_doi=10.1023/B:VLSI.0000003028.71666.44" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. Ohya;citation_author=J. Sese;citation_author=M. Yukawa;citation_author=F. Sano;citation_author=Y. Nakatani;citation_title=High-dimensional and large-scale phenotyping of yeast mutants.;citation_volume=102;citation_year=2005;citation_issue=52;citation_pmid=16365294;citation_doi=10.1073/pnas.0509436102" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=L. Parts;citation_author=Y. C. Liu;citation_author=M. M. Tekkedil;citation_author=L. M. Steinmetz;citation_author=A. A. Caudy;citation_title=Heritability and genetic basis of protein level variation in an outbred population.;citation_volume=24;citation_year=2014;citation_issue=8;citation_pmid=24823668;citation_doi=10.1101/gr.170506.113" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. Pau;citation_author=F. Fuchs;citation_author=O. Sklyar;citation_author=M. Boutros;citation_author=W. Huber;citation_title=EBImage—an R package for image processing with applications to cellular phenotypes.;citation_volume=26;citation_year=2010;citation_issue=7;citation_pmid=20338898;citation_doi=10.1093/bioinformatics/btq046" />
<meta name="citation_reference" content="citation_journal_title=J. Mach. Learn. Res.;citation_author=F. Pedregosa;citation_author=G. Varoquaux;citation_author=A. Gramfort;citation_author=V. Michel;citation_author=B. Thirion;citation_title=Scikit-learn: machine learning in Python.;citation_pages=2825-2830;citation_volume=12;citation_year=2011;citation_issue=Oct" />
<meta name="citation_reference" content="citation_journal_title=Cell Syst.;citation_author=L. Rampasek;citation_author=A. Goldenberg;citation_title=TensorFlow: biology’s gateway to deep learning?;citation_pages=12-14;citation_volume=2;citation_year=2016;citation_pmid=27136685" />
<meta name="citation_reference" content="citation_title=Medical Image Computing and Computer-Assisted Intervention–MICCAI 2015;citation_title=U-Net: convolutional networks for biomedical image segmentation;citation_author=O. Ronneberger;citation_author=P. Fischer;citation_author=T. Brox;citation_pages=234-241;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Neural Netw.;citation_author=J. Schmidhuber;citation_title=Deep learning in neural networks: an overview.;citation_pages=85-117;citation_volume=61;citation_year=2015;citation_pmid=25462637;citation_doi=10.1016/j.neunet.2014.09.003" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=L. Shamir;citation_author=J. D. Delaney;citation_author=N. Orlov;citation_author=D. M. Eckley;citation_author=I. G. Goldberg;citation_title=Pattern recognition software and techniques for biological image analysis.;citation_pages=e1000974-e1000974;citation_volume=6;citation_year=2010;citation_issue=11;citation_pmid=21124870;citation_doi=10.1371/journal.pcbi.1000974" />
<meta name="citation_reference" content="citation_journal_title=Pac. Symp. Biocomput.;citation_author=J. Tan;citation_author=M. Ung;citation_author=C. Cheng;citation_author=C. S. Greene;citation_title=Unsupervised feature construction and knowledge extraction from genome-wide assays of breast cancer with denoising autoencoders.;citation_pages=132-143;citation_volume=20;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=J. Mac. Learn. Res.;citation_author=L. Van der Maaten;citation_author=G. Hinton;citation_title=Visualizing data using t-SNE.;citation_pages=2579-2605;citation_volume=9;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=F. J. Vizeacoumar;citation_author=N. van Dyk;citation_author=F. S. Vizeacoumar;citation_author=V. Cheung;citation_author=J. Li;citation_title=Integrating high-throughput genetic interaction mapping and high-content screening to explore yeast spindle morphogenesis.;citation_volume=188;citation_year=2010;citation_issue=1;citation_pmid=20065090;citation_doi=10.1083/jcb.200909013" />
<meta name="citation_reference" content="citation_journal_title=Physica;citation_author=F. von Zernike;citation_title=Beugungstheorie des schneidenver-fahrens und seiner verbesserten form, der phasenkontrastmethode.;citation_pages=689-704;citation_volume=1;citation_year=1934;citation_issue=7–12;citation_doi=10.1016/S0031-8914(34)80259-5" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=O. Wagih;citation_author=L. Parts;citation_title=gitter: a robust and accurate method for quantification of colony sizes from plate images.;citation_pages=547-552;citation_volume=4;citation_year=2014;citation_issue=3" />
<meta name="citation_reference" content="citation_title=Nucleic Acids Res.;citation_author=O. Wagih;citation_author=M. Usaj;citation_author=A. Baryshnikova;citation_author=B. VanderSluis;citation_author=E. Kuzmin;citation_title=SGAtools: one-stop analysis and visualization of array-based genetic interaction screens.;citation_volume=41;citation_year=2013;citation_issue=Web Server issue;citation_pmid=23677617;citation_doi=10.1093/nar/gkt400" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=J. Zhou;citation_author=O. G. Troyanskaya;citation_title=Predicting effects of noncoding variants with deep learning-based sequence model.;citation_pages=931-934;citation_volume=12;citation_year=2015;citation_pmid=26301843;citation_doi=10.1038/nmeth.3547" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Accurate Classification of Protein Subcellular Localization from High-Throughput Microscopy Images Using Deep Learning" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
<meta name="twitter:description" content="High-throughput microscopy of many single cells generates high-dimensional data that are far from straightforward to analyze. One important problem is automatically detecting the cellular compartment where a fluorescently-tagged protein resides, a task relatively simple for an experienced human, but difficult to automate on a computer. Here, we train an 11-layer neural network on data from mapping thousands of yeast proteins, achieving per cell localization classification accuracy of 91%, and per protein accuracy of 99% on held-out images. We confirm that low-level network features correspond to basic image characteristics, while deeper layers separate localization classes. Using this network as a feature calculator, we train standard classifiers that assign proteins to previously unseen compartments after observing only a small number of training examples. Our results are the most accurate subcellular localization classifications to date, and demonstrate the usefulness of deep learning for high-throughput microscopy." />
<meta name="og-title" property="og:title" content="Accurate Classification of Protein Subcellular Localization from High-Throughput Microscopy Images Using Deep Learning" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/5/1385" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="High-throughput microscopy of many single cells generates high-dimensional data that are far from straightforward to analyze. One important problem is automatically detecting the cellular compartment where a fluorescently-tagged protein resides, a task relatively simple for an experienced human, but difficult to automate on a computer. Here, we train an 11-layer neural network on data from mapping thousands of yeast proteins, achieving per cell localization classification accuracy of 91%, and per protein accuracy of 99% on held-out images. We confirm that low-level network features correspond to basic image characteristics, while deeper layers separate localization classes. Using this network as a feature calculator, we train standard classifiers that assign proteins to previously unseen compartments after observing only a small number of training examples. Our results are the most accurate subcellular localization classifications to date, and demonstrate the usefulness of deep learning for high-throughput microscopy." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
    <title>Accurate Classification of Protein Subcellular Localization from High-Throughput Microscopy Images Using Deep Learning | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"uPkUehoWIvUsavHIbFM0EQ_0AEoTbLg7BdcXu-u2sbk","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"484241","apath":"\/ggg\/7\/5\/1385.atom","pisa":"ggg;7\/5\/1385","ac":{"\/ggg\/7\/5\/1385.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/5\/1385.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/5\/1385"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/5\/1385"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/5\/1385.abstract":true,"\/content\/7\/5\/1385":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jB\\\\\\\/xXePKja+4JU6R5rbE2uDCkx08GzVmBLl9R7oU61QfEJ8HOPB1lqNt22VvG2Z88aQyjq6SUWS9Ucs4IYB8Bxd4wgRWSI7faguvjyoxHeAhbzXaaVjG5SBWEA715ykes4FEN6exAfZjtPopZXC8ivS4jQ0zE5+dhJw3OOhAwuPY\\\\\\\/QP\\\\\\\/2Jn2PMp5PqHcTEFMFKj0BL+uwWzAVcrELMix\\\\\\\/SM7rNFN1AKamIyWxJqhdaf42VM\\\\\\\/DYwyDn5KOJ\\\\\\\/fDkxLaf9PM0kR2U5vPmBl7+AdkSG6Bhw+LhkcONDB\\\\\\\/KmowDfkT0kiPFrcDOa43tm8UeVZehP\\\\\\\/gxz8EMjM6iwicx5izc2IfgSVITfOq0SkdjX+wcsRrq56Zkt\\\\\\\/okHQgXPEm+DpOGbXTAP0pljnMDjF4hKqI6GvM5GQZAYH9ivehHycyhK9w1obMzzDuRUJoOR1doSqEkUFakZLnfwkjqGTLXZOVHMOHQVlOw9CpB\\\\\\\/bZLO7PE5Q7Eam7Z9L9T2Feid6DzNbtrp+tIIPVolko72YZprIW5YUH164GxLFZkQL0kaonpIN2tjWCLjbMEoherK0PGNV7iZtlGLMS8t1WHEmGtRq694\\\\\\\/yXLYr0lrVTbIuB\\\\\\\/ZhMhoRPfnzFyfa60wYw9Vz8VWpGvgoWHNCjcRyv7hLzcz+acGkvd\\\\\\\/GsgFpXQv7JUiRP9yvHWO5LmK+B5VxeaArCZT6fOf7sKDd3rvTgfPISLCwFch8dItWiyRcffhL2CcN5pxvqY5hWuSX3+vs0uHFFOwUczx1A5y2T+viMJdyyj4eq+m1sw4kGf2kkoU31JeA9qyEjZKdsdK18Cc3bQcf3dVguemHDdWHMApm9v+GjUnUCOpxW2t0t3qaTkD\\\\\\\/8RzmOG1oMct50x3Gh\\\\\\\/PxGqT9kaWUfebyrSPtgJIUjVeSMcXl1Ti9LEQpxCyDOLWA16q5\\\\\\\/pSZ3FnlFsMUgtuTHv6ODtYFS2VMEj8FuAK5ppUkFLG2AfLy959wBa6cso2YSAPfv7jKeMw5waE\\\\\\\/Dz9ttcIz0CDJ9zgaTcDrOnkIcAtRBek9SASyiqYrygvHfJu9MizVDhoptNwjVM3FMbs8ODx6IiTqILUQDqycsOe+czF19f1Q2hRG69kmHXh7D6GoH4A8lgt5PZXD49ldEG1Ojc7Vxb1JpydBzCkwDtS1lg8Uje14T0S5bZdEB9qfkWB0WIeZ5iHnmm8G1K9MaIPLo22Z+niditVWeDMGEW\\\\\\\/wodvRO\\\\\\\/cJ8BKv7bNVmKteweXiXZW9nvRgnDJ4UEYAwG2yQ2TDGnY\\\u0022,\\\u0022iv\\\u0022:\\\u00221Ekr8A6TewJ1sH+ofbYeZg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002224189ad7576352eabc19b0bb9659b93b0cb53c4d286777e1b51a5962fc554dff\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022NjrsVAfajEctCdCnzwvDFsYfjVjvOsOxh041wiis9CEtBma3+Z1s13wB3w+B\\\\\\\/DThLl9Z6Rl7vRW0sScpOhQVF1yL4stej02mLmdxQ8K4eDapSX7uSWBBhW0euhVa\\\\\\\/+GV9RaAXdcHQb+aohDqTBehgs7pdX6veVOfFpwP1YHjO1Yh9Dj5aIq32gEjL6VxxkhdqyYHa32nHfx4YT6ph8E\\\\\\\/\\\\\\\/ZE\\\\\\\/8AeWi6K3Mmd9qwUkUDvneeGvHATiW5BJGTPcHkzXuISSZB639NmVrOtQPRUm\\\\\\\/Pxc3+G7NDHjqIPxL9s2tEWeU9Ah6tbwcjLkIlUPVmjFbo8Cj9fb8lryFTdQeBvGSxYxRmU9kGh8kTxsUOl3XZNijTuPrx9bB982NzHt0apEWRrXV8QBGYovosvRqbYFop\\\\\\\/+7db0xr1hsABKG6\\\\\\\/DDtPNvnM+D96jy\\\\\\\/XNi2bwOInrIO0KAzrtTIvwcHGiJmHIBS8oOGhEWp5BugM1\\\\\\\/Gyj0Gqa\\\\\\\/3hLzNQo3xR2TclD3EMN1ps\\\\\\\/byjqq8J\\\\\\\/c40uuCzqPlfZM0iWwDuBhDI+BeoZFoWY7DD4+M3JQjrm9vc6rGCo8oHdRR8iB4xCEhzljLcvCv8\\\\\\\/csE\\\\\\\/XcD5cHH7SySCBUNRqzxXoRU\\\\\\\/zbr+dRYDJWvgfxVp+LNXcIlOTSW346iFdbJub1QegbrEoVKA+ZJo8\\\\\\\/8W7IZHHAAL5MKUJ5IxCXqiNNPlxO4e7Bf9mutmXulV90Rtmq2uy9+hyenREQkahMsOcbPjF9XD5a0Uy4+GwtjXQzj3vQe4Dw\\\\\\\/eK+AvAcamDHbxS4I\\\\\\\/rzeWvHVWtNE\\\\\\\/Li4o4C\\\\\\\/uF0L96SoVCmnphUdA\\\\\\\/ZDazt\\\\\\\/CcCZ2Xg0066agII3hnFik0I\\\\\\\/QVpmt0f6w48RSU7FbPQWq4BGHHUM\\\\\\\/tQQlKayAjj3pc3vlr1hNTu0UQ2rvZTWhBXQGQvCr3bdqFFEsfvGoHa7KDHDLbe7H0ztYMeIQLSsjBcivb8qVF6eMGkV2yW8FKfRaLF8uiQJ++iblE0YhptMj8JXNONoeCC34BkEbyra2MULRR14JF1G6sRrsDvM20ISGMoAAEts6WDMQNUs2OSsRPB0JbPKrkHntY\\\\\\\/IsYJrJO1dRmpvMa6DF2yRHhBYdGpBlVpdfgONuSp+tuRh1jq1MtC8xYYBINruVuf2AARGeDOpK1f6StV3VoGjmpf7ImyrSGFYyLGhSsg+DQxjUHmbdpLHxXFfCogBdMBDqc0d6vsjFyW1pIRCic+RnADFpPtquXEOFf8y7A0BOBfTB5E7DfKavkT7uZAR\\\\\\\/igoicdukTIHFto7tyaHOAmC4vk7awyEl\\\\\\\/WOwhLhxjpXIjGWv31pdztcza67+n7WvgyV\\\\\\\/veqkkz7+pvTPJ6EIjPmtpVNGEFi1x4Dk18SYrqrM7a+un82sird\\\\\\\/JEwghNuzciPv\\\\\\\/KVc5gAZMbLNK+OfYgUKl1J2X3bx742QBVJPfh4nrT95V3oiSuHBMJkwAprfDzue27A\\\\\\\/AV9HBWYK1fgKA\\\\\\\/4cWxTKueCAxn1QHcpAMPAm2ftUW5WqiGdw3BbWE9oqGACjwWkdAMh3rKLFkt7kQxCDqLaxGC0g2pMWrXqZVgJT7ZaS+zmEOfVG8ZVJHwCyA8Z2ws0o+XE00OZ7VRuXNB22Cn\\\\\\\/TbD++6Pi+IiwfWAf3rSVgeSKLOMWjAeFTNsg7+5BhcQs+byujFsGVhTrx1+YRodEk5+w2DE3Y\\\\\\\/6AZrpGyFYhYZZ8NiFIxnf0Z6QTVTv2TdjAQC3SwGiKwqQZG\\\\\\\/D+lUblqMOLfp4IDmwmMy0JiLeTYG\\\\\\\/GIwq3j4+67iJeKWuoWj153VIbAmG6+y4yaWYst3qvkT6UKy5V4VNsjqdZhmVcCD6RkzZPgaKBweKolra5kBJ+lIaG5zIUQqgnb7dMoDIhAiluxRXdxu4kvA7fcDUybmtLJzwZIMqc33YFX26xxlyaI6I\\\\\\\/wXHIinO4apFtWa0Y7lRfthbPDydRUt6ytMyP8qrtcrg6nU9gjVSVNp4yB6LOirnfjfbP\\\\\\\/bkWv4\\\\\\\/Vg50Q1UtMC94vrSVpjvl+VxweQibI3t\\\\\\\/AECNOBbpvRNfIwBERW6NlbzhIpxr9bUjFlhKRNB3HFeDkw31eBAbBrZ\\\\\\\/0tjJFHZhlW\\\\\\\/Knxa6Z+r8MDWQvbEua+jo99cmVPJoenI5\\\\\\\/gR6Ej\\\\\\\/6voWD6pHGGclJjgT23ywCd8US4OPSXL3dHE+jEpAVG3PB9f\\\\\\\/acx\\\\\\\/8GoruS4dh9n1U3biNXUGKMzWoeuLwMyLI0vptxpCo70wOBmqdGAkP51mte3V44fbzWk1vFZ4De+uz2wg4ziO3rhXpzYL2V\\\\\\\/0IlTeyYTQQ+fQlr93jpYr65ol9koPZ\\\\\\\/jAA+9WmApNRXaDTbdxdNDNDeqrt6mv7XNhe5\\\\\\\/oftkb+HfBYsc7xqHQRFGQ2tpVVNSpZPc84r0XGd\\\\\\\/YJBI\\\\\\\/L0thkho2OoJu7ub6tjkDx+mXKSGCMyVJOE2WK4+f\\\\\\\/HhDbqwGOumkL6GxzAmLrMGSNPWTKWEdbXrvaUOwfXnSA3fuFmTsk9V0Y8FUYSFoeZYQvKJNdpj3V5OhQvBywQtdMPNRh30mKVcXbXG\\\\\\\/SF8Mol20eFyZJXHbji+7TyuqurwMQCmSWJzlm10pqWXuGNd3YnZHxC3F2fq+3t5k0nsOZfhPb46vdDD3euWbds59n9B052aY4kDCuh0esPBalF0Ex3TtMDftEXJDKvbSmxlGKMo5pp5GjyIXmQ34XrYAJH6dcwH\\\\\\\/GXAyOQGY\\\\\\\/rSxK7\\\\\\\/nd1o3p2yXNzZVTv0t8CKoCFZnwjhtGohXP0Wuzya+M0fuDM7CyGqQVvMXfjs17B+WRlqsb5oGJKGQhGTlxo6CjuHO0h6Db\\\\\\\/EhX4yUfcFH+A7F3VyAiGNw21xxsdSvLwTjiXYrGNEBEDkezvfK2WIuLkFxe0uwLNXQzxI51Jio1NRdM3IYb+ulWU\\\\\\\/55tPqfy50AN\\\\\\\/q6mbZH6bqUgulBMis6Bjy31SxpOyqUOOlskHSSU+N7QtGUjf+nkGAxmnE6TqkfgbmaEJb7nBA5Z8AgoOXkKKp6r2MFFZNfUhcDudCxL0POK8JBuzRu1fsTuDCiIe0h3z9KKZuGh9k9vb6M0KzyL+2rdpmDWCHaGcozdSlKU7TTSSymu7GR+cdjzhvuEXYIpfvc\\\\\\\/TR1m96DrXZdFioq6oqrc7NC\\\\\\\/+IAQTOyOQ4B\\\\\\\/SEmH+uoxtvrQoRWMfHQ8559azAoJOx8\\\\\\\/sans7A3At5EFlUiHDJggVxhWEJZcqtTyJBxAd\\\\\\\/WWEGD2K7NPb7h7cGLUtAnQAMeWGNhE6JMxvVg+TXzppoQxu+Mx01hCPlh1eEGxfVOvb8ejbao64GIDBPfSG2dSNXEFAJvwrfdnA42jb\\\\\\\/NqT400mL6WIUJt5cpj+gjJYDdl4S1yLJpyYno8j80VkHBwlcP6drUWIFOuB0j8+nGfdXGvlYMIMVycYlxzRCYwEx53egDWrjOLQJDsuFQAeTbYZ\\\\\\\/zeyf60xmxdSlsjUFbLWsBziujWoP1bTPbcWlu8FjNNLN8ZT4wstwQHqiyIvXeALUL2yKezP6rJHG5gdX+BsT0u25RyCxlGKRMQcFxCoYH9VlJ9Pt2LrZx21XRbNZO2qINm8c86GBI\\\\\\\/EfS1xcLqy+TYGiSf19IQ00XbK0pCrGzBsvGmttJ3QbNFxihY\\\\\\\/LVzvvtz7dyVCv\\\\\\\/ZR81F7GxS47GEYdWr\\\\\\\/Snu661AX2Rg3pEoiHmO6KFmAF3XJtpsVprQm6P5T\\\\\\\/3AR9CRzmS0abqjMtmTVtioNXPDOhCO16erqtY42rFX0tfL69OQQic5rpNQU6R4ouEZ\\\\\\\/cgPiwdUMEbsN6tTLjoY3xIlnOvduT03qtEkH9s9NDzwmVu0l1AZQRJUb\\\\\\\/jsgq1hUYEbu3bkdhI49Ry2zfFVivblDyGeYm\\\\\\\/Sl94aa1TzO\\\\\\\/KeGzvuMuNryuq3UCAYeZI4kryf7C9Y2i7ff4wo9pCZIPbkLl2MbTXaSF0RhgljFMEsE\\\\\\\/oBjD9tys3fMmO\\\\\\\/gN90FXcr+wrD0GU0rfAHnR1xDS0WbwCeXNWYPkOkYfgcYT5oksuZC4kWlF9\\\\\\\/TK4H0zwlC5ZtH16YdIvWgwax6CHin5M3PN5YVyH912rbYL+sHaE6G+ShibmFw2t13MP\\\\\\\/m47CAwHsVBUrdrk1ElryXQK90W7Ryghy0MN4IBQDBgh83HOvI9BuasgEa0FSkpB2pDZLLDP0V1z3r120XndQlP8ne35fDR0BdM0RIZx+jIPizn4Jwxi\\\\\\\/qD2beUuRlAg9PS7LIyCfBJ9TsWJjt2OPgz0wW7NgegRB42Qyw85qBh4PWCDDk9pSKFb4+dl3SPHO7idZ3tK1T1FxIIGqRxu1OBRBvawKbZnAdmZ5BNKOQd0NrVtOuWa0io1bFaJhaTgIhZ5XVVLPlks5R94zzKjMKv2HM9zWirizGhaLPAZYLtptpOsEsJDlsSu7t0+eHTi\\\\\\\/+n248dyI3erdWetD2ag7tbzB3L7D+Lf7apoE0st4ZHY\\\\\\\/JahgI5QkyrRVsR4CRIYvXDCnZt7oVQFc+\\\\\\\/tpqtiazxyyKhybv3nyo8lMBCn+H9JHq\\\\\\\/Z7SZ8wX0RffF5pWBnA0\\\\\\\/pXXA6qEILxiy23UWQA5E1sUGsiNriqKjvs\\\\\\\/sV1ciZchoUaYDHQfafvYGV9WqOEsxDODJCmMzxCy+jg04gK5Dz3bo4XjPzIjoDyMeZdAoerNmFENabddH3pL2Na+K5BOFoJW5Og4eRIMccz4RHmRveVMGrjgkUxleHSX\\\\\\\/RysPqW7wxLJL\\\\\\\/CZvCmJMQDrbhZhbNSkqBXpkQuOWisbKCcUajOuq2lZnFXkSpgGyYwHZ7Oy4gSxXBy0KkuJoEHTG5zBdDlTITR0+s7t+fIrxNU2vkGr9Tj87MzXpkpJwZaBKtWZ9k6s3vpYsqsXT0HHfzcbBWv7F9wwKZUZtktBSuB0DzjfpQEwNtZVDM9RTAWj\\\\\\\/rTXiGxtv6o+vbwbvK05ztuxc9ULWhIZ32oP8JGPKUumfbwvbGSivgbm4vMYT12oN3jsf+xnCgyqCm534J90wJWi4zu07T60WWZ2g0T9dsNHCzPPDiYzqg8IomCc22OS6UZiM3U6oC3UbSlK6mSTafxgdYWXd8cCzG46AMkt7Q2AWfunyE1\\\\\\\/UXNxTHKDiBqwrrZz5UCaArHdlnDYnY3XCloXHw+YhQzujgJb7gzbfA5BVPw019MzL04Y7j4Xa0YaWb8xq2o78ETyPeFygcLrzHaLBfHB2VBLYKjb2LlfMDKPnbjC4wtAhbQpFFy43\\\\\\\/YEa4wkGqAsJzeH5\\\\\\\/9N2p0SzLNhVpR5CvC1NT0aVuYLM5CnVbNRPNPG9E7Tx1xMNKrAlgC7hUN772KsRS\\\\\\\/bL5d+UV7VArcxDQBNhkMN+L6xnMrocAXBS72jyPNWDk+rVKFJj89VtMcW0JIiFi7wANTjpgsM6KWBXJ2uT32s\\\\\\\/QBtVkoKU1jYgQsms81ymORBjDw57PamNNy4aLvVaPfHWZ6DVNwRLRT5ToZSPQfIXYQU06oyuk69wLMcMK+HJYy0SzBvgowl\\\\\\\/ybRkZuBNnhNhX5X9sNDGJ5v9EQu2lXMUlVIdNTTyP\\\\\\\/ZJojMKpw4R8l3ueb3ipUED5wZJ5cdIMtoz5gJGoaHEKddIcJiJ1nIiGgxHXuSNWEtKSvd7cnQgbd4KtiFEFX0Gg8mPwRnxDJMClPnGBTYBo3B\\\\\\\/0\\\\\\\/itdOSd8jWI1IH+IH8H4l2PmYWWzLCfwgGGBKZEA6PfsJUGNIpdTJJXP+7w0iRN8v3FxoEY7jqvnSeWh7L8D1egXT4d+83BVFFsRya6GzeVQ+YrUIT5pHiS2dWh4h9LQYwrRm8NA1Um0mPAY04oFt9H1Mbq+LOEY0Ahmdo9CcCp89gmet+0sVIUE+UrcMJr4L+0ps6reUecGnVAYurASjtcmjX8PaHuJg+TqQXsLuV+mOiHACPWS22gtWvL8NJWFLnYvNrz4xlWFl3iYVJxxo25+Xf186WbV\\\\\\\/rudfak1zgrG3mdJVPbHE54hszjUPWg1yq84jxP+\\\\\\\/3r9Ngm++LGtVmQB9Crjd2GnAK8SYspChVzFSMKdkoI6+BHlSpDTZpRlyVuYk23gvUq+WSPhPqnhs\\\\\\\/vY5cx3d57CtIW8Hphx6K2Zaz2QJ5rAH7nOcA74MqwouxwtwYsXj3Zx8Qw9JPuGC8odLWz950eSUNc5\\\\\\\/3+jcjyYJRHHGpfgcCVrBMpWf8WTeLXFhmFBN\\\\\\\/xdXFgldtf9IhX9e5AhhaiwLkSzODm80TxZCZ3xghvMBgs65BDfn86k\\\\\\\/Z+y+I05o5FpUfpZYvPAkDPC5x3Lw6NOrjv+RebLY\\\\\\\/DOilr9Wi6bxalYdn6uLsVS3OFzhBvbkVuv793WQ8yMJmCegSaoZxXNpSJSJutPb89hBo\\\\\\\/UUZBXqfRsgKQ9Y5+2YTgtJO9aC63WIaQuXnbusMo6r9pJsYjoO7F8rxRkLn12tTCCOMzznQCj2GUW9TdUARQhkR7NXYH68RBOw3koqIkq2nDUMGBjevFo+YMQKM8JwTmnP+vlOV7v1QDdrGbKYIpeScEuYaoYtsS\\\\\\\/R27nNRGH8zkSwIJqfxl\\\\\\\/stqF0rQI6J4PSEreLbL2S8JTVNV+vQsC4nu\\\\\\\/eqQQT4yn6LbU0SAQeeWztMiYARfkp1Fg4FbuhuDVKOVdLf9BrQj6+bpn1jTH1mYWDGkE+AKEyQT+HZdDZ9nt1Gb7qEeE9j6augbP3lI4fWHMjMkLchTb0f8ybgZ9nFS7dt\\\\\\\/qCkWWkHW9JCQhsV7EhKc7ViZfRfl4xdYvKYy6tdcHL+IiVDN1Rw5SqSqi1g7l4Ph\\\\\\\/ENvu7qbDKhiNOnP0jqW2zLLj4yK6Sv927R15kG2VYQtV+bjBwkPbKS3F4KyjyPIs4\\\\\\\/2LbGeLgGH0UpX7lpNEhezRrOF82XLotsCza2VpanHeS8R9MlMXYSTBMKauQEf8k0quX2\\\\\\\/F1TC5pO+qhrq3KasdgoxouS08\\\\\\\/6zdi+8zYQET9tmq82OdryW0B1Fwqn1NylH9iQMvf0Y1dFSHcI3zG+moSV18bBNAuyYTwJE72y7zbOK1hP4\\\\\\\/376mkCZ7Ip3XuCOYk8eDmSSZ8wbippgTWGV+vxcwfg6McsRAGC5tSP0AnUQlvc1CQyg\\\\\\\/DjVwt+JVd1wAbU61qQDiI9GwQyDHvASGSU4a\\\\\\\/pVkKsPGTgQGUvBCaToUDKfgxDpZFDAfdBJfru1zHK7zKRMChluirCQRmljRM8Vu6k1r9hbRhrKzQOrkWCWQTpIyagn3OjKUB6gIemRoSZIk6eYfRFw7ZOldN+KGWEbAv2MAFGxMOpv3d86sB5wpEd3HfyR0Zil3Dgie6jwxojkfG2eBdzS0TtSlzVTTz7ff0vHJ3ck6RlnC3nidY\\\\\\\/L6jmI1tQwxO8RvH3srtq2R3kKiiHNXsi78gb\\\\\\\/Bjf9xOgQ\\\\\\\/xEj48lGOcKqAT2uXdgaSm4IRzNg1abhr6G2RecGuVLuHbenc6a93CbOy\\\\\\\/w+TlClJHTqNqfgPKXat6Bpl0RvwKYWx+r27hJh2TtA3yWPERlucsKZKJoUFN1gNOTg1DGT+uebeFc4qDl6\\\\\\\/tMAEc3+469pyQjKaOq8km86fPDWtW8v2gBNPcInxgnN1uFknwJ7N0CO\\\\\\\/4tVg\\\\\\\/s5ZdoVC0wxnIxPG76Nvuy70QVO+Nk7E6GX92a\\\\\\\/jmaVAMmyw9JCsncydKMUMzuBk9Wnzaz+9AnLwTDNnC+lL5gDK1X7MwnxtDmgyhjk02vIfImtDkArXBN5H0nfnCqHdbepvTj3aUaZvqWpYMsT7IQFXQiG\\\\\\\/JAb2vB8Gwq0ptY8FPL9x29RgB+yW1\\\\\\\/Ylbd7PhJr2LiOL7xw\\\\\\\/OU0JoAK6cTIrchUvCca+NBGw6H\\\\\\\/unls5rMfvsdN\\\\\\\/rBfNnUsn1G4c6Uhe5IgJDfokol+7KLXMXGlkfuxC7tev8pAQfaDybLqffsTCklyhbABVVwwM2epoMU2v2qXZMci+HDWyzY+igJh+eeU6eJa0K\\\\\\\/ebFnk8PMvjHESiE0U7arTOSab7NyqEPh0gb400ewsedWzJW7JrrhEmxKz9h6eywKBWhuPDHzOHTYiabC\\\\\\\/C\\\\\\\/mrZ6Mgy\\\\\\\/HYoe+WifTOE6rKN04DdzP8yOiQHqbO6cj2AzlhX4lcVco+2z+k7D03BPptQo9LVJiFYuAWmh3zG9FelQ126L7XT7hKTeogWan9vGLGsX9r86GJ4QtBVST53DuzmfKL4YE2BwnDdZbHy1f\\\\\\\/\\\\\\\/kkDGJkGudThXRPVJ9U9NJHr+HjKcrKWMGJZ32VgLTYmqXgTyMgVunDit8bSnpGtcvMFpUBBD7RLFXKD2ad9VKhufieNA8IoIfO+MB9u7qHjDLs5fYuY8bcTIe3XrJh4P9VIXR+\\\\\\\/7qzjOiQmeXNq0Uw0CgTddhx0XVSiWvXFA46m\\\\\\\/thHz310ILRA26zPiFF1XWJXg8NRr9w4stjINzQMuoDfJRKYEiHJ6oY7cLZTfENduicPNcjFAtMp2fqBwi+fn+Ks7WSGUhXMQa0Abd9ds3HJYG5gBmz9xHTW\\\\\\\/yHDjb6Eohq2Dbj9kWJ7l\\\\\\\/BySHOvHXtEDLLiLu8THnpJOJ4V3OX7cwmPNV3HOpx70pDw0c9i6zxqAHo4lJRNu1lrRDrHb6XFnRYTzotHrXqHGKibyGwv5416MBSfwwpO60TzSP0rrPjvL+VugJ3uaHZymVpIDnPBUTqmIHiEQV2prEEWKac0mv1S1+5cdBttYHuGwLf6+EE8vRZUSa28lGJ\\\\\\\/nqPsHVGRW6YgieOqdjrgfDzN2x1UmHrjFvIlGjovaQ0RuDbqD6mgadYQFQmja\\\\\\\/lki3vwji+V66NvYEvrwyl1F\\\\\\\/UoO6xZcxuR91yph26haFyntSS0zpoz7lrPdHBczqi6RAbIRDHxNf+eLEoys12oO2P3yAdQBF8NtWUa3ecDK4Jdghd94CVidMJ2DBL8f38Voz5ykCORFjos7+xgMYkjnCsCqnWWlDr4EqdwsQMLG1hksChNnRbIjl5GqJKYzGU\\\\\\\/ssCf+1ySuJkFt5vnsuEYEr99YoPUzOzyx0hKzjnIwi7HLcljY3tpBcS87feJ\\\\\\\/9GC46TG2Qw3F\\\\\\\/uhEqw9CWtTH\\\\\\\/PmajSa\\\\\\\/1qvjX9pt01fw2iRAqZUzTRgF+y76WlO0iAO7Xz4GocaAVEEhSNakWJoIExaCinwxHRCrxr03tVEDNMiFsU0G2DIGRNFImJOJDELNEYzAtpDlau4+EbPLf34xSfugUoVPiXKozZABosI1iTgVMXrVHBcMYpFjWurghoDDS3GqWPlTtpoLcmEaayPnQH9NstiAudFuyP1N\\\\\\\/RUdGf3Lj+IJ33zY8x19rnXJrSg9an3K16sQSx4L3ajzHmK8cJQv64XF8G4tlaye9+axDBksKLE3cHkMY2pxYn+3HVWanRmT54bW8bQV1AmL7+RLxK\\\\\\\/otJ4EZNzs7SYLpi1sYPzz+qqr8DXhh6l2acq4KDaHjk+rOG3TT14U8Fu+sUtrug1Eydhr6iHMPfxCuma5GW93Jx\\\\\\\/F8wro9OU3ZUNeSQlczyzWUr44MIsa30TR8GFUCAQ8i8tj10EY+pmoIICT9qfFoYUWBygePeDGNJK9DB3EjlQybB23aHmmBqwIlPD6qGixvPawBkHQjGnX2m\\\\\\\/XzmdbO+Q2ZHvk5Ll5Fu7+4yORP7I00WT8gxL7XmivytqKpzbq\\\\\\\/Q1+J2GNqiv0VnWLMgO6gTcLsfSCvzQKMz29DJHnF9BBfQrFSwPFwklebq6KD9pRo9Ns+tEocABeHSLlm4P5S25b0J5ErmgV3XGP0RuFtP6WxU2sYMHUHzHTOghxK+1zPzH1DFxCA7dUww1\\\\\\\/PJ5K3q4B\\\\\\\/DVNs6iW+iHRSavgqdYE6BLIDBEN8gxr3ri\\\\\\\/VyhbdM+UJJvGKHzzUdyQZj1HW+5tBLkvQ8rJuP7nmV9JRBogLzjCJixK26amWaMbXcaI+WWnSAfcjzotpgWVRGvQJvvdStakIQVvlm2ZeV9X7M0wDDiEYt1354bSzmMlelbyvH+rR79upUd11N9b4PjTLeWCdFKpD\\\\\\\/OB62hpQnGpW57CzlelfEJAU8g+6apdgqsptVhlTHh6PYlxN+KPi4trNG6BGUubUoH3d9zluJTmXgFhi32ga\\\\\\\/UidPWhJRYHep69Lof6SxFCsoIkKQwuFFWZKzT2lqnvtIDkoixrcvza9nZZ1kWzQci1ayATMwgiBjy\\\\\\\/TX7umGPy8GccFjp1wZScC7wuERRvTNjXTj5B98\\\\\\\/a8vWft3O16l3vL6j8AnzXwzvIaatVwwsmmL1ujrFi0cR7o9E2YwH\\\\\\\/cLxrtcPvxjrytmzabc2hswl\\\\\\\/AkQI6bTzE0ee2gNWYsXKqzwePm4+3fgrj5ycacEDjdA8v1KUVFxQn9wtCJHpNIc8l9+4eiKgFxdyBxSQbHvZpocARbJRvSm\\\\\\\/OMEJjkt70HkbReEWCHheCRf4yx44h+KCb1YYeAwzIk6qVuJZmJV6oKEECQ3wDa2ouL3bzvX4xDVJO3lwkUQD43PGG990iaMemAgrynUjeGMvWdY\\\\\\\/Nws8e06e2WGRfjpuJeyK1pHPqG+3XsmGuNTsVo90cQnx4HzzZEiHVEQRuNXTEw+N6ZIEXFy6amlzqU6oVTlUoNPbcUMcpu8egLQhgvPt3wB9uL1xnAZg1eDKe5AMRUcZ4W8o587\\\\\\\/dbGcectKQNojZGMDapB4EMRKL56DuCJSfXBSW5JEipb2hGiSTwd15zSmwYRz3DYQBjoCE7hTujdFBN1LlJolK04IrysxAtsR+HpVXi+jaGghOfZ2zE5ua2UEpaUj0jRHOlWWcQNkXGKyHTaNcIaznPWpCSxtAW7anCkq90OsqH\\\\\\\/ofrp\\\\\\\/C73HDwMz6KXufh+01kDCy1NW3We4hH5DDTA7Zf\\\\\\\/n70RNHQvqLFh2sAoqay0wISnZatfG\\\\\\\/rHrbSMpHBrZ13sZI8mbJhPCU0JmG8HwDPuPbdfS0dJ8jFxj1rotEbMcAbwHEbx5+EUBky+YT4GF9Uny0mLhP9goi5qUcoFH8OSv2tHfEc36yO9zF5w7xp7uAqqJDTRUJ55We94dy\\\\\\\/wlRn78k3iW+e52VPGbyF5bbT81J\\\\\\\/TTKLmwvt5VNwajrBoo\\\\\\\/boeqZjU7PhLGlXorCUqNhkfVDyFyv2Nv+wMargntUXACrCx5zRI+Oe5eVoR0VVwGdhRK94KZiktdeA1OP3K8ICp2dYOBoU8nMoxc2pC9FQAvD5ADLPocAnxj7DWV98o9eOhYK2agm9Ul7dIvPT1WPEKqrcYWh4j8V+r7AnJXmvXIYEAO6DAAVYFPdJVzEEKIJiMQTRfkmkXcmzQ5tGp3JyI9et4vuMWe2+6tgK4vkfZzlUjcp4IGp04xeur\\\\\\\/ucoL\\\\\\\/nyuPFTPQWYh6CtxM4hqg6e4WVJaE1KZYTujhtRllMKb4SP68BjyFwYqnCiH9bNG0\\\\\\\/Buz4C\\\\\\\/4H6PiNpKCSD4LH2CX5W\\\\\\\/F5tuZ3cUQwaL5yjymtKU8zZoBIC04fwnHQKidz4UisCduAPbwoaoQeNtDBh\\\\\\\/1iXJez\\\\\\\/VG53F39PDj1a5lCewrXgSlzg7vaLLY36PA6aDVSIfj+UGvKfFuxZ\\\\\\\/PmZLR9g1mw7ric12LrOQEci5FllSiXpdUxAIq43Xe0k8y8u6PGpXWyyV\\\\\\\/vBAF1zitcYRH9Wy6Sawygrs\\\\\\\/tFNEjF2WEMb4c8dOVQXj5KczP8u62MUqGb7y5JgDCwFtH80Jo00fmurGwhjRW9jSyPtZ7X5oCHSp72iCLyX0L84tK497DRH0qsATj99WOSCUtOy5OPz6in1NqhHn0RKUB1vCK9QPf52BQgqqqyjqZbQvkNutQhL21vsP7ubtv6wSAAYjvkdqpxbBx5cqlrz9c9MxFsvjlWYGRKmqr7dP7ZdEWZk8iflubQT7bKVVeHY9eSgAYN2HCnz4Kf472qBT29eD\\\\\\\/A5wrlsMjWIui9iH19unIiLRdgAQM9vUvIR6U+muWjEBEnz4SuiunA2Z94fvGvCpS20WYBaF4q6SfkIFUu+PdekkJWecS+62Kf9yxp8u6UOZbouMks+y+mR6SZN+A\\\\\\\/0VqpBu5HyDCQs7jXfMHmWfNdi1yiM1x4RQsKjzwECXQOygCPJxzCcd2E7OpJxDX8tMlh3lwL9rsqjjLH4rtsKhj27SpS9TbkeTBt0jnKojnpa7o1FI7C\\\\\\\/9FNbmjW6cjphYD+38c9I7f0y+i\\\\\\\/3SotFlT6VcHUDndY0v7Ld6zSKh7W7Va5WE0JK4KST9lQtZamWzMrTdmHFAqb0xhSp5iG+y5g5NwoW+gb7IgVIkfy8Dys4RJKOZuXmWzWKyhJheCTT485aW8k07FRhWXKlRRHY308pF2dGfWVezk7L1vRpe+e3iiT1i2oi4+ss0T6B6F3CYM8CHQ2H7hebcKYgv1d9x7qDv\\\\\\\/rocPR9886szrT6vhaUHcLaqUdqKre5jqQV+cxuqSPaBFkeV5op25Yfm2F0gcvym0Uq9id4Do4LGgngf0gCqHvMnNtiM60R2jileZResFxhXFoXa\\\\\\\/2BiTW3ga\\\\\\\/+TOMbWCGXUifzyHQ87IkHiQlxX+9oZLl3YZyXJecucEEl20JGqkH+fGrxvcsnMvfp57Y9AaXgNbjDeYHNwgL\\\\\\\/H+V5wlck1jIPYkzl2FTIJ1+5tGCUKYldnFnVDSv0il4+uugJEHjbu\\\\\\\/370ziPbPoLwaRTeU4KXs3TReEl1wLjufXBnEMDuZNo5EqeGYI5eJT0cJtDfkEk+WQLp9gKnZKzcZK\\\\\\\/Ca8keay0n0LzrZ2vFuX5N3oWgb\\\\\\\/hft7N1VG2ZYr6wSIemXSuEXp9oFjNvSBBMTYtwdYGhuNUvM\\\\\\\/PKin9tO7AJ9i7bPDNICnQ4T8wl6RnHY\\\\\\\/pSolpLTILOC9V8W4bSf+wSuFWqZD9KxnTneb2q3HI\\\\\\\/Y0YawVJL6HYMQglMIEzH7d0kfYvSVD+ts1O56xkLs6JM9IUf+e7C94vq3GAXtKb049PDM0z20M5PFQsV2jSFqB\\\\\\\/IO+5KspABkm9WjLU8jf9Cn0ymT+UKkAecOeTNLZjWR6wxom7yhL63Ww0wNN7l5poTCDAnFOoYbVMtx8d+oTVFqu5kVSeiowpDC6br9KBy\\\\\\\/Ui8MujS8P3ZVERFLOqn3zvNdSZUzBp5dKTyVZ3e78Noa3Tz1QdtzqsiCgHmRAB2DDP1lQIMR6A5enUTw3bBOahjLxAW7x4B+ebQhcDypCDkpqny7tlbqzex0Fzl2tHU5dlly4i6OfZ30iN+QzdXgvcXFomssgT90Fp6Pa\\\\\\\/nLr6yA0iScgEdMyCMet6jrrNsY9ElRKen8ox6Mq\\\\\\\/nN18hsUnPyg\\\\\\\/ZswcWmVpMexPxB2SFCTzTiNv\\\\\\\/ck4M7fRucPy7Nf9ruPKud5EYJlKRUkwCMja9P++ruQZHL1IXBelj16qgnXCMid0rYFlpRas41oBn\\\\\\\/aSdmDWh8IlMmcLi\\\\\\\/qA4V6Si0ZyZ3HmsE38ZOjLT3+qDsJ0hwqyXewNYaQ2hiW+HUFDN0ZmwH\\\\\\\/4YELALt4Lzh6U1fkKa37R9wG2FOeOdgZB3Cw0SaCWf37UnLObtVMTOPL83wRK75qHCdXFsMiVIdgYzb7KfD5hD8VKGd\\\\\\\/E\\\\\\\/rDep1lcZMUPfrCkk8gJfI1AW2cHb4yLqGrexFosMQmvyPdX5AtjEgKlD\\\\\\\/sT5DXgB4LSJCfQUaTPQfDGDd5XIUDaarpgz4kcm4OXTJ+nSO0NFKZIr2g3djwapeTh1P3VX57KfzNSjc4y4+hUiutRDLj6JldvJv7XQ6DfCzhbvjy35yV2+oZRTVrlQ8mAkOGVsNai2X\\\\\\\/V77dUWm6yqTW0ljv05CQ9uFRLs2jMcwmXTLgxj4FRRB8\\\\\\\/cVW\\\\\\\/pHccWvcaN+wxRYRgha7JUAEZktWKSIVQW1fsT\\\\\\\/zRMewtl2g2qd56n5GquAUt0OaBuVZvPA9MoXF7YqV\\\\\\\/u0tOqPN6etlCT0+EoHKwljogHBuyTljOfIJbndZHS9rYoH5or0e14ZTNwJp6p+LVFnTWQPWP7uM2BRbMfH5+IQg5C\\\\\\\/tU5B3TFcBYlxa4KJst1L9pjnZN3khNoBQKGkyleW63+vHft8O+l+k1pTkTYerulKu4ruRCTXfm7EsrhL0z27l0VISG7EvR\\\\\\\/GtxzBceD7Xi4guhoqYPZj8nsm24PVxeN1AhXXeNYTxSg2gqbDnnJdNDpq\\\\\\\/RY4O6CHvVAke2K0HFTehIc7cqagpvFjNJVmZcdcklioqyk28c28tHnCp2ZoNfNbeL7ACWnojj0PuzDoQM3rm+Xnn5dk7B9NW7qZZXfahOC8f0Sm6bRIWbOY77G9FSJs056UV4ouwRixDO0L7t49spfY3BE5SIKyXIzm3+s08KDzM5y9Q\\\\\\\/\\\\\\\/PFbD4febGry65ro9fl3oGS\\\\\\\/ZbIQqhzGVh8A2vzCgqdIt1sDNl769lyYay0CM8SavlOaFQwmDNMiIiO53ubeNigk3vWo4AUCIEJhmdwHNwVXAWOZYCtt5NXUF3srpMB15vKUHk8MAmbeRq9u5pOiZSOqJYlGqhh2y0ZwvCozb3BGVuVl26\\\\\\\/RsebC\\\\\\\/GQ9jDJ6N2HGFmALmhkzDeOmJMCoLhlDnQvgUPVhtOmBAp\\\\\\\/jlpdChnxTkwYlHxFz4xscxtNqNLn11ggYt4sqPqBJaXHQyHzv7mGrMXr4aViLi3zUi0FLYSpR3gsnjF944oQorY7A08NDkEGEiA0OYhxMXHunk0ILksDEvKi0835BGsQKwe1\\\\\\\/f1UrGZeGNpxU9mz3kQrdNI\\\\\\\/hYR5Oy7eCXZB\\\\\\\/rFslP9HTTZn5XPB23uQw8UIzjVC+DZCPbcvN6wAqDokbVPpPBORjGTGfe4WXdQJlWpMswz7mQ2ijmuGHtOwjHBZ+gjtR3ZS0Yl69pdxAFWKxBUhy2f8zKwcDwUnQ7n8VmmZHXTYmyerVdPoRIZj6x15QPiZwH1u\\\\\\\/004uORC5q9uAxkUXI934fx9xkopt35ORdr7V8nXQBWY6tUMuCAw7honTnJLJPxHJXvyun7FCvFCpaCtn2W6hmUh7wAs4Jxb7YYvbskSF0SWVP40YNHt+KNANb996hr9LcN4nqrmfCA9KKBgZNVjS1nO1UCfzTQ58xLmwN4tzgfW4gNVCuNcvgu6WeqLcGbnaj5bPWp9N5ReCMQWvoxZLnCLKJDiaym8hhZQ8rvxAfJDoj5RNqwCXcFmW\\\\\\\/l95gMIOzeUKp2mCfGGcBDq5eeWRs1Ijna3QY\\\\\\\/MQI3Ncs5X8MVnqTomwqdmZjSXtLUbBIiL7XdqvJaYe8zVMQAY02SUUDXGdUJEgd\\\\\\\/lMNK\\\\\\\/5o001reKkCm3cFGvhi\\\\\\\/C+PXLe94I8BW2\\\\\\\/ey809R5JQmVzpsd8V+lHmU4QlNwp8KZa8jI\\\\\\\/XMBUDuw9QSfmdB\\\\\\\/lF+0lVEW4s4FZB7fdeVYqg9bWiU1lJ0PFfaJ5QL2Zcz34iC5S1ntFCCnjPYvUI2Ae1nDKTSQoQTRkAuPkQyv4kZypCpgyO4mgb1mDYtyiK1r5GNeoMoZOUfTf2uMuv3yOmS5QsPvgh4AIfVBFINLmErN0csLM8YOc5SfI1\\\\\\\/iSGMBadKa9JWjkIX1Py4\\\\\\\/gwPrDIigp1mTIpf45CFr33NWc3\\\\\\\/\\\\\\\/hL1wk+n6dMYZPuxWCtSyiBm6j\\\\\\\/OuJN6PDiu\\\\\\\/G7\\\\\\\/HQihBHsK3qbkRYJ57XT\\\\\\\/sJxGYxr7rfkbR+izjW395S2Ihe5\\\\\\\/d71Zx4rvlUOXwGH\\\\\\\/E2gjgzmVnElHedHPmXiVjtuBbWZFyoZFlKmCgxHEmJBe9iKEUyY8WJiEEsRU9igutBIDGgz5vXkSI256EwR3xGNaOtwZIMmWJtqYKMMsNNpFcYhwRZzvUhG3GunhLN0PM8VY3sOxw4bHsS8gXSBxA4NJ1ndRYFVmLGswgEUx7VvkycrUEDqkDMskgS4x1w3wy5z\\\\\\\/J1dwxAmDnT\\\\\\\/yoLClleKYTuASbhPk3LUdHXcJwqo1+0evhJ7W+EACwnh7yMc2o95pUgvI+odnu7+zlgvUPayYGe3jMcIuS7GHUYiLmPZtdM\\\\\\\/3nITfs6NOqUHytvGm+60CqgLZeZtOVK5bfyLHdaO40cTA4wEFj8Eo82tDsDjegWQ+w9+6CFSfReHam5kM\\\\\\\/a7MAQ64bj+A+S9rueIKuSOFfE3r0Fjv0MDp0L7SxwiRv+TgMwFDFMoshf7NuQLBEKIY+RP5gosYPGJYKlp5yAqKp1oCf4YdtYS43nhbWoqgD6\\\\\\\/yRU42jE1R45Jj\\\\\\\/kvf9mf0ncST+Y7Ph25eDxqWppp7eXG3hZ35pM0XJCtWFXirO1RmN16HCgBVu2k2HBpvXaDXYXyUVdoT0OuRj0L1Dj\\\\\\\/fqs4SVM8p06ueRIx3IaLUIr50FS+uupnmH2qqavULwhiQkq07XU6GtNhxdg\\\\\\\/Aou3+3Ot3ZIkJWo5lstC6P\\\\\\\/6z8I9iULXqzG7ubaRIkmHN0tHnjDpgu7K3heo4UI+sXWtZcz2amQFex+UBc1GfIF1aW3cBkIHVTAh5igk8ykQo6vViO4Sna3YM5z0nGAWYkNs9jEIcUz1jEZnp\\\\\\\/9Wuy5POrD7YXqdLOerGtKeouDp4Vq4doTchFK1XtoEajdLEGEFHrGzxASSyWQ9Q++l\\\\\\\/ixtaCIbTi4IDufQ\\\\\\\/DvwZ1EOvFiQIrtC4udvz\\\\\\\/JoUp6F\\\\\\\/M5aYeGE9ACNOM4OFs1bORMCBc16R0g5aVoCxeFo+EfD\\\\\\\/Zts6ThivTARyN4VEksky5G4aKFXNBPSOLq7FMuenSjYV+jy1TR65eN\\\\\\\/pa+str0uGq7o62Q7MAld10zBT\\\\\\\/K+FrDAUO7T213mWtaJpFeZ+l5d9fLejw4DghBgTZko6olLMJKrRbAdrPG7bQKafQUJv147J3MDlf51WtOOBjyoRyJXOLEf1kPR8CoCgc1R6ptL4KGwL4i3gtitvoB1zTVkUyBQCLkI\\\\\\\/8bL3mzK+IkRLrMLGjhK3XMQOVSyv0KyfNQeJFrCMkXYb1YF0HBCcVhif6QfydSPXJszjMWWbYY2AxEKrTm3s93zqAkUYVhfbobaoYsa\\\\\\\/dJs36Bx0LrC01aUbmhPewysscDzWpfSs8OS6TSayEiEROjwWhD8oNMYDwfU0B9uwRqZra38yrCXrG1UoccCmBuTCciK8zwFC1xiZHEknLxpEB90l36yPQxZGxF+elQlq3an4+YsYijxZevq89k8irWFNHPHFFy01nxlYkiLtGKSGdCqv4x2c9GPdzGtznwClK63piAmN5x1dw+bLTwLSDaadyaRGaHHEWN6ogq+sY8iIfosuEkT9u1DRWz38t1LjVulL6QyMpXW17HtWMTos9LatyW0s4eMQL8Ly5KCSQZTKuvZ4z0j0W5baJquK0z9KG\\\\\\\/lCxHThejdcJJjuIjVFhmo2IhWJNWFZRDxS1\\\\\\\/W0wl4OB7aWNa49k2JbAJDZJLzkpjKuUvzs1w5QSIvnJcTQ6Xhzapy9yfwBuVZVYbSguuUAMqZJP01H+eUXuVEq9g5pC66\\\\\\\/Vf07nwCQNiPhG9rVEsuEtDFPZzeR7dlYNv5VQPS3pjNVvs7D\\\\\\\/KhkTRW+DK5zPSUY6vnUnchee1tgyhK5leHNowsB\\\\\\\/w40HlOpVHGprLePGqReQV7E8g\\\\\\\/uvwDq9EuMXuG\\\\\\\/5COhvzz2be5qvcqTBRdG4lUzZCmG2bJcAKudE5vnkpHhy0QCX5pP\\\\\\\/9XIWHbHeMqZK85YKVzkO6rhj\\\\\\\/RbFkFVPOVktPm+Atm1Rwi5kVGiUmMmjS82OW22zchetRiRYBxetSl1uNnBajV66Lt0fCwU+SnTsaN6sRONrXu2H4Ya1tEroe067vS3Syeg\\\\\\\/md4YOYp3onnbEx4bHokPwEsjjap2RyTNQL3kR+pKh5DYIqDvqjeKfseazt+uvxAc9WHaaA6j09lu11HebRHVdg0H4OYnr8za9X8RFD09LEoWZd\\\\\\\/9MxMYG9fho2S1a7aCWcfMhRgzTbG6wGx9fsHe1wvNr2c2Ou04fihm9nTwSPSfmGh2ORfoaddThDpr46C0ngWvjPPaTKprQzO0Njv\\\\\\\/9IQ7x97tn64EOhqWDbe1uP+DH7ouEXZk5jeMdPYcUoGsdCd4vQ+\\\\\\\/jZ7m+AMaffCv0jfxavDgW93DtpMxFwMvuQ40b5iVTYQIfmIzkPbx5wheGrupJHVqvUmCkWDcamNtz01GybJPsyTVqENZwYLrKUarX0YJcqDn80Fkw6E4iWB0+E1+cJ4oCN1LB4uNMFExIJT0z4tR46DeTuQwJy6DBJRyVA1X2S14RbOiXTKqPkYgOcjihTZ2UAn8g4P+cz2Hfmcgxc9rCPyeG1gIB45vQGvQK59NgGyeu6cq+pnQ5Hp+xlqM125nv\\\\\\\/IjNatWZ\\\\\\\/yXrvHMurNA7LBxcUfUJqsFf30PblFKx19oS\\\\\\\/gKszzJUQHKlloygxQsrCIRjKSAt5hsmo\\\\\\\/BTCGUKkmFZdOBN9cDrlYauKQmtmciQAhbNxU2ECEbKuTgK\\\\\\\/7620JujXjatDiPzythnduyaYwxlyv2qjOib6TmyVX2apSSSnENSVBGQhOyKSyyvNprcmy+JEp5dpvL1TS3kozd+JmfzJN5lOemZtnUUYtat4cd8ILh4Gaf3PX8N8MWPKELf3zTXyy2lDRsnvGYCsmrZJHSlWuBAVYZb+vYYRkkzonzhKYZaPhEcjW33In30oted\\\\\\\/qkTEW5R4\\\\\\\/6pQP5HOVL8xPRvVH\\\\\\\/xnNRgf1Pdy1sSZvV036Kcp0KxJeMiB4r0am3eIqNiEx2G5GNOWQ\\\\\\\/5YxKzQrBzpNTbURsHDy+te65t2OMMqWEr1tUkDHbVVs6ojB4Bd\\\\\\\/6L\\\\\\\/cereErYDvW5iubTV150RUiiBvDvrkA4HgKYK5kjRl7TAoR+bgDe\\\\\\\/Yc9Vd9Ec4KNnXRt8FWUjCqdSUPZ6\\\\\\\/HsH117cqVU14A3NaiNozU8Br9MPrjWi4VEd4Ag5OWgvAgXXQFk8VXVwI1bHX4BKf1R9q\\\\\\\/cmHeAk9VMH33C0GWgEzcPVmIbztnsDTZXvNS3yujZ3RV9jXjbQq8A3Ts7NBK2l\\\\\\\/aafLZBqWe3bC\\\\\\\/MNc0ESjRip9ns3DJmAMu6JZ61JOXQa2EoGUomAgITb\\\\\\\/O28afglWJr0sSmKptuapLbA3eyJc9zXhKgkTFhQn78WsrDtUDlwKtUNbn3ffcWEo2vjEnsT8l\\\\\\\/POUR8ZQB1sT\\\\\\\/aDNCeXccbsP81TYlo4zhGc\\\\\\\/Gzs68BRQI+Zoy2hDHIDDFl3cfdcdkF7d6odR4z+WtfKE0JI35aT01PHpRNdfW2QBGj7toWKfyAuMVJ+gh4KhjrUjDOz3Ar4jsRa0RtJJfE8xp110AE83YpAzNjx+X8TdNoZXBmVDo0p7Dw13fhP44IQ6\\\\\\\/wGVZ38ashiMzp6x+UV+PIghPr7Wa9Qw1MCFI9jpHR7M0yBEg2svXIXuarf+c1cfGVShyj2Wzou\\\\\\\/IqVhtECc4gVBsr5qo5DTOohQ7dewmnuQ1laSJaFZuOFjT7B7wPN\\\\\\\/4Xx0vKpwZMrJ1NBR5BXyQ9m9CF2j1yq34qIu14iDaFiX\\\\\\\/r17GrP\\\\\\\/XQ9Lv1460rmQ\\\\\\\/dytExUxmAIvb8pn\\\\\\\/RYFh8Kc1fRb\\\\\\\/nVXiUFOGdIRfqMPwRJ6sgmovhnvDcChmLA+NxCDm9qDEOjtXK\\\\\\\/5DNSdYT5Ii5xPdtyGr177ofeyeJY7fVbDkufJmVq+eyPHxZUueDSWVBQK4gVcxoFWnG2fZRQ3KXS6Ncrc3WsVeis5+cSnZn5nhhpns6VAuChOw8nJCzzDV3hBgiMPsMFCPOZKFZFanX0pSvpY4pGYPljYSHqXZCqXOD9nCtsKe4FRglidF0fEaLavW7KW2r7hjtLMVZjfXT2SbHUWEccAedc5qL1LTzD0CnqwG2ymIERIFyCDutToqJCZRuJG05Y0w1kKzNKh0aQlUmsAEUcx1HvwxAneqaJMPMMp4zcACCu0zGDBKvpf4rHPc+oujkM+8fms9gk2ojmiRjfesPlCJVv7CxjEHg8diaBfP8rBE4r2TbaXEy6sQer4V+iIW4K9XnKjAmDRoCFY+WJNmiV3CkUX0lUScHufTdXl3PYqCqI6UHm0Iday0FbnLLXcFYgiV3+p3k0+ztItNI0+QrKjxasaL8Kf+360VIbyc9+yr8ugv1AiQpZSIZ4ThGZboSg6h9Db9LiD9TKhqzbikIwJvg2hd3z4tv6Izs3kpZ0b9PqIE5+HNXimc3lrNW\\\\\\\/W3Db+MAyFbHdHschAjCURbaV4LqgJSoqPbLQEhDf42Cax9eV8REaZapeueD03I2p6kn6uWb06OR17lYantwk8u3ckUPYsRL\\\\\\\/c2NCJndoZ0LQw546\\\\\\\/DApFdaBzOmlfLQau8EzewLaFAGfpYk1IDVKqBUby6nOolWbZmH5HIhgPRdUo8jYfvyrtOKll+ikXSZKhsOugYbpAx77pkZ46EjrFMMS2e5bSdyHAkL6NaWW\\\\\\\/U7OfFwNyCEcPOxff6lhPz66odtLZZ2uuVN6jV1nPTPT6vW+BYdMnBhMSqP9xgyfAksyHsIbXN9yc616w2Vd4d5PNve43zd8VvAEUlo0VIXiACfABQvS2aCj9C0oIkO3c+BvXyHwmKFErbDdeF1NX1XVhYvZF4bwpxAAwvovvCBb64HLPlQNOnT7DL21Mt9oFRlLdL8tKHmBi9Wa8DG6M44L\\\\\\\/fZfExftdl8JUjJIUyHjaCU08fvJCAlxjlgoWQuKufgPPAhKkOZdkiXDthRdWPkQmEwifuZP+TBTfPM46Nti3V97dPD6NMjIIaA2sEN8sZNn94NeL4smnelu+7WPW8ds22nLsr0zHG1qSvpzCznws4XQ2QHIzU32Npeulv5qhZGJwI0FkZbSvo0jZClWYxcm4lgUXjIjQU7XbPxcsJwh8jeCWAWqwwPg\\\\\\\/0RtmfOlQRPhbGtGPdvN8Oi0N6R499hVPBMOhpT7VZU0hfZN9ZSRPMIeK8HM9VVVgq0JDzrRSQuJIdxAiKIKwFoEu5TZ9CFdnnh705DigvW0vUnQ0w2zChObRCRwyDMuaZpSJGQIYHvYohC71YCEtcv74qXPYKk36jbBzWeAlAyrobpc\\\\\\\/w3yt\\\\\\\/EPrsIPF9qsEZlYb+mFik8Q6XByDoNXZZHIG6OqbpC5EMxWmmRU5AcLFn5\\\\\\\/FvE1auM6RJC3JrcxFQfnJm+bNvdUn1o2HKS+f0x+0HXfGuS\\\\\\\/87jMsYZcd8ao0lixaMrWEnJPLYlzgtbDAe1+o5CyIbyxh1z4JpX76G1rwvyHY+BMCjnsvbMCbpR1rtyIYyrFkmW7CqrF0TYAszXh65X0JtP1k0lk9kTrfzsmNHxuB1ryoi\\\\\\\/QFPuPOdtEb5SEiQTtH11GdcbSut+1dujxAMCNNRdwDDdu4BBLcElliMpZ81L2Wbkru5+iZu14doBOjJI41Nv9n0HD+krGZnMB4UDFAowgJIVXJkoOiIllYL7smgoK8CFw5sv7vcb2tiIPSVo+ZjRriKAkpdVWg1rlC\\\\\\\/f+SqmFTOoew\\\\\\\/eHzngtGuTtOe4oiZ1mWYOUX3zds4k8nEn8XoootUfCsOgeF6W\\\\\\\/4pNKSUH2y1thL5XnmoetYK7SCQ2oaTQR4C5ygI1QujeIcx26Ttim0ZgY0pB3UPxkCxnKy68gb7bmkapuEPiRpXIiM6BwWq+lj4g+Mx21X2cenCtMW1xSh6w8D9hZ7N2bT9OgXgJPchgng1sU1lL\\\\\\\/dp31E6PQW2yo0QsVPjNf5gvgTD++Wec3jHLvMEuYFbsG2IqrYwIPPB1\\\\\\\/mEYZhU+P9dPlsievqLlAf6F9ZCQ1ZZJcxzAWNqWhcLuJQTGP81f3At\\\\\\\/rudEgffWFc+3gjdgh4c3rMosimjOXGwdlnSvDWHyTcVmU1RxPSxkr2DetDQePC4etq27Rrjr5i3OnB2XxlddxmGs\\\\\\\/+Xll3V+9Fp9svorRWA4r\\\\\\\/KOXsUsWwXvGTzosQxL2swu8AIwzj9uYEXHmI\\\\\\\/BN1k1js0FmQ95IFXK6ltONMVX7N3o\\\\\\\/swImuQJsFJ76MqrYE9CwfiIp5IsliF9ojgovj6anL9VF9VaPvGmiXfn4DIsVD\\\\\\\/NMQERwCNKQjsTxPqmGQd+YJ+6reqmTx6sYpgEHhno9xuqmiOJq1WJgHHHS0hMc5g294xBwSdMxpQQctQFCQSgfJkQn\\\\\\\/T\\\\\\\/uG4BqnTHoLqsrnaVxv7poUa3QowqqJNfjax\\\\\\\/O5pm\\\\\\\/cAAL9ekHDgJfZlkVhLx0ObpydFbNqYMoFDOdq\\\\\\\/WEVU0YoWklyikHW3wuMQySf5xTR+rWgY0dNtvoQvBSU2VrKYRfN\\\\\\\/dOm6s8y6LR8SbId7p+BQTrsSRkqQsiP05ETvLUCO6v04iVr8Xcmgmz0PQmc5AkDVr3dQXFQs2wq6t3zoXwiORSARnTOsf6h19LLcGliKAdqhc5+6KJLlcJ+JwRI3pVn\\\\\\\/6e9h+gypa5Z8CuXb6t0GtXKWFNr+PeloPJFrbshFEJTvOkuVfzqWMWA8G8ZqgVPl1+zcmrMExLAG5VS0gVi41C8nZ03U1DmHObUwpC1L\\\\\\\/dajfM91rURLQTdG35IruLlOYjefevVb7CwVl2+vw6YJRxJExMghgl5mpihqd3uIcOOoPXVPtaxO6DIebbYSjHsqGIVVhjzyRflCpCGYFHHX+bmIGmu7hQf1Jht8vFjxi1xaEjtiwakzDXOHwKOxk5yHKj4BFx+gStUwZEA85l54ZvZSYppxxNKD\\\\\\\/smrRA8mJvaQ\\\\\\\/VStt\\\\\\\/BfRaGOh2CVr52BnXz+W0SiMdZq2CEbCae8HOVuL2vV1uNpnQr32NRuWSVQKsk4mI48\\\\\\\/GgbG5VLJ3qWaslwjC2DyQ7vr\\\\\\\/B\\\\\\\/fqaCAuOHMVuAA24xP+2hYl7rX3y+dZI\\\\\\\/rOQ5d2lbAfaQ4P9\\\\\\\/0+2OGwXWcjMAaZHqRBSiDqe\\\\\\\/yYMXSWpKu1+fH8rpfiVawKrfjpAt5DDrULPricB6GhbjGlQk1kswNeJeYFAXC\\\\\\\/fkvkOWzTGSSt1HPfdfYO+IUmnE4VnB0vfyT96SX4d5XcnEALh1UnB\\\\\\\/851JboYHOvIocoUvVrYIGtmcK+NI9r8CC8odMSlkW1IkkCIil5R1CKDi\\\\\\\/86zrv\\\\\\\/Emlkx7bx8jKfldeDeGYZ949GiuZc1F9zHqFzkj0soDldkQ48qDQWJUgXzSW3JN7DYbRY1+vkzDXpUshN5A\\\\\\\/Usqbp994Gn7gGdrc1EkeivA2GzQibBY5jEzVn28aa2dCi6s9FyI7gW1SCZzvfyZ6kDg3nFA8VfKJuXKey0EuTlplNJjdugFdlyNlsbBBf2h2QLgGA3cQm1GIO+OS2+OqtNm6V8xU+XM0Zv6K7VkrAeZ8YgUwncyVUxYj4q2QaIsqPsqv5XvzGZF5hW9WchJ\\\\\\\/Go3cSx2ELKnu8RyRr\\\\\\\/B\\\\\\\/8eYeuP8axE3zEFxuWjC9gEah7Co3hZ71BsRFFEb3VZkRidH23ogl1Bop0s\\\\\\\/tktop8pnVhzXbT+8RvYWsEs1fImLKXx2oC5eYIH7M\\\\\\\/V8OO8I4J3zE14VVhxDX0KRnyOePRRM1hCLK8NUdiBqPTW4nD\\\\\\\/\\\\\\\/3c4f\\\\\\\/UlIIKRDFKGp7QedRYBzNY0XW1W32cI07eiVTitP6eD9BAvtRkC6lo092HMpKZBmVwm86Mr7zACqhmBdDzFoRY05jrEDT1wQRGj4Pm61raRmEwDP58uqBlMhtUnvjaa8TzICSUnEcgMq94MeBe\\\\\\\/Hw3EZFbSJMY9g5Xn8SVKmr4DKIYo\\\\\\\/KveIEDCQGbbzxWTwfTk+ELAH0Fo2MTL7tTFs4UmLsPglnEwplubot2HU5nrwX4mlk\\\\\\\/cBd7Ci6z+oV+4EAAc80VIWXfyA2HiiX97vvQ1jBwVvqygHp3jnagMaJbRLizsv2pmisw76NZgaj6l7DG8\\\\\\\/AZ+7FCQoKyZ8puAyLJizpNgVjOyQf9M6ltvV9WbDru6WTLENLZNcNkpP32gIlmFHtQ1ty1+5W2Ni5KUYfR7AwpFuwNFm7pdSRv6RyX7\\\\\\\/ha4sEw6c1\\\\\\\/N51j3+JWfYeZaRzAi4fHXYiiZODiIDBwRQe4f0xjO0NmAqG0WeEKtCZuj3TDFHGI0AJAdo3B4bOF4MeBuQgyzaQTGuFfD+8H6rAW0lbcT7E2FRE8lpZNtmX2z5EKDO370C0EkyuviNT7spSHTII9F45DSg96d1ybBDaQOfsp57b4L+DcqvZKhyTU9vwJQNbQun6h3yVhwCd5flLDfzs4yahMnKK5FaNnnRwYMA3y8SjDhj\\\\\\\/vdbcx5GoIr+8wiDyfCXEcpx99hha5JWgEu0CzpMBUFA\\\\\\\/uCM7xJpVm8FQ3FbVUbRUsPdJ5GlPy7KQbN\\\\\\\/vebFedM5Me2v12LivW3LvHi6Op16itO6NRz4mq7MTR3szeGDoRzfS7kKRKiS+jza7QgmraDCXxu4WnaYXo4LW8dnSFEdUIYuHrmFe5QrL0T0lAa\\\\\\\/OmmyzbyTN7rlrPEg3xOwY0NKvccLW0paDgMG3R9mW31VBVKdbMU5BByXWSdo2Cbpsf52XWbZ6gRj2o2F8nf2xx07puLpOAoeUlfPy+IkQ7BNvFDkXNiuwEp8oaxfjsjzqRjWxnx2yBtaOBbFUK7G8gDTWzftxNycVKhrdu314W5OZq3TDrrJpdc55rm81YnaBC7fphP2fMMnDNkSFwCV6F5q4+FKlqYjYs+pfMv3dZLZjd6zaPljms6GLUK84QLSsjUOhrWCTa5vNGGXu0u1BLQMjm3jIGQUKu8EI0OQqlQemRt1uPIXO+DVycTSHIcCDSyE7O75EiGrJh3B7xAoxSwlvZP4BHs\\\\\\\/ngf5lUIMcv+uyHDKVuPjeyhuxXhynuR9cOSJ\\\\\\\/Oqp3+NRLy8nX022C1dlFI1SwNyxQDM6Bhruu+uOjM0E7iMJhcdmLlenBqb4sqZEwFy4U+yUVXi1ZB0mQy\\\\\\\/B17daaybfqPyCRDiOQ\\\\\\\/APoxNsz8lJgX70cKHE4yU3tA1+\\\\\\\/hmxGzPJPBUJWxX9O9HDrvHpJSLfFjxVxs9TckY8was0BsBf6LdU7\\\\\\\/jxuOnQLtOZqq\\\\\\\/d5G4rVk\\\\\\\/rf7ApaWwlLtkzfF2uCDEvrWbzUKlRq7I3GrQzqKdbatvj8oeAufvu\\\\\\\/0tMi\\\\\\\/fCLCcBX02Y7yzPCJ8SilAimeZ1+KDkiq3YigqjjxJPEjxEgnqt\\\\\\\/xTbfKEXeYFuJMuTziqHxWHmH7zmUwYhgk7PAkNZaZ5\\\\\\\/wlbfQk+d9JoVOmfqyw5GbZncnDJkjed4Y3cB8kLLUvpS2cUFZb7Kdu4ZhheCJX9FwAfSv\\\\\\\/aGsztIwd5VnMWF+zeZqZxpLqykaPqRw6l9ig19FJ5T\\\\\\\/eDsL7NFaktOXlSWadwUfKAU3sTIIgdETL7vEuI\\\\\\\/8II85ppLPhrudcB27QSH6Inrfz2DRIHkl2RV0sXkHck08J10\\\\\\\/z5\\\\\\\/VB4e\\\\\\\/LvrFe7RkVn1HKvbJHaQNl6Xhtl1bUqevXYPSaCkCGchFkkkxM1DmUiGqGxWZH82THdLHpmaArRb9w2bxGAk\\\\\\\/OZoiGnoTgRfUFXLZQxxAY8oR2\\\\\\\/lUvfdIJsS9Bq\\\\\\\/FZh61aHFOMAVGy\\\\\\\/KRMTI5YYGyR+E8kaSbOCkjur2jXjbRTCgtnxa+0Y78RH8OrPrlcVXkVHsYvwd1xpfCWFeV8i0hpgt93BnqMuBf26mWLDsMN20BXqsmZmJpd7YKVqckng\\\\\\\/u0yGTcvRXOZ5WuvMCDgiDvKSfnkpvbQyJ32C6NNzhADsQfkLMRBBfgLxXWfhoHwmNN0YJKnxSbGNM5Wl6vYpUyIO2\\\\\\\/R+KIx993BuDn85ZlcA9XGJdG9yWK1r03qo\\\\\\\/FEvqof3OyQ2gS\\\\\\\/XitZhFtVi3fg2M2Sd7VZfCKGHP84TEYxWEZdZbp1NoqLd+bNjeUC4aPIHq7T+WkQDv6PRCMgxst8sptoyJwoubFXeSr6DAkaA\\\\\\\/ACCxZnrsSIQAeX3oidhAndlXXQ5SeaD5E5LbSANLHuK3t4SbNvSJLCnkN1KK1Uo1kDZ24IKqWVjmN+cKGsmwv9YpRJpBuxMTSIdUwieU+XRXLwjHfRix7mfyQmHo5DGGDi6IgpMM0s\\\\\\\/cmxaKJ\\\\\\\/jlsDuz+zN88Inlf9GECgHSHz\\\\\\\/7I1SXA3DU1XXVmH660nISlgsQ\\\\\\\/Z1LxjQ6pBjlbaO\\\\\\\/DHl9FUkSojTZ3hRdlr1KvgdZPWGPyOSJf9FlyMEnvHNio1oDWvvCcA+6UA2XnOaafM7XVrkoVGDlvAWLyYIzzhfxp2P+rKCpb8FxKHwcET2pKTrAFq1FGv127\\\\\\\/DCtKjGurCl0HPJTY7mlWRQJDrL1qEbKt3gJTdqX8IIW5mqZn+CAtB0zsFTu25MZ6hHROm198GR6HfMZgHb3vVH96ixyGhhFrgDKnpIobRu8uD2r9\\\\\\\/8VmLwG3RkUVxn\\\\\\\/LC+PWy\\\\\\\/xheLi2YNRo2bPd1V2qhzUsSIZ9bOkeqacidBkY1LmH0nH1WFzP2ZXXiYNE74hFvbjz5C\\\\\\\/9\\\\\\\/NsRY+KLF7AZjaPm35wboLYsFM4X+WOrTDW+0USEi9PE6SopSV+U4Y0zI5DEnONQ5CWntSMphdFpZmwujK\\\\\\\/kQ5\\\\\\\/tfJB5J22uxXCuVOUHX8s5\\\\\\\/0zJoiCGY9+XE+B4\\\\\\\/iRY+ePzJI+5HHwg6s26hcZHLoPRYXCYeMH0Ifh4U+qCJK+gjAQl9PZqLKIahvozgpyzlL2FstMm853rr1ZdDnABOCSIJWE+smFWP1CA0A5EqRGxghjxm2mmHTe8RVV+Cp1GDcFzRlBLX3aOau5OF7vCNo4nVmw54q+kSAQa7txj64gU9fhMZ9n4owB0+d9\\\\\\\/3KjmQim190U6cQFS0R47U+ElV8ro0\\\\\\\/wDewGp7Z+OidvGPbyr81KDLcHY8u832vSu6e3f2QOK9GwhyQlCYydD3Gc8xRmvaU4iNLZklQCq3DEXh6edqOvqefY1096opCtOX8dliNcbKp7qy7CX26rlIDb1X+LqzEe2cYjuroqcka2\\\\\\\/2t\\\\\\\/27pF1\\\\\\\/llEIrZZiGRvcCg74fhUr0iag5S6lPALd0Ad+TLYm8l74E1qdCyYJDsw4RKwbcYBs7gq\\\\\\\/LBiLa8iNZw5qHp\\\\\\\/JfOn\\\\\\\/dBYCv4f+MYW31k0YFVjn5jw8\\\\\\\/hjz4noEVZRCtqCry14kjkKoJkcMMdIpBjHgkJ4DpImZ0blvLo99L7NrUOFz4czEf4QibzpzF9vMHrpPG1UyeHPzs07uwU7soVL+eSm8MErquNAECXJtK78Yr9\\\\\\\/agp6O6ls7XbCgP0GjGoGJOyJSg9EpS0kRVSDK\\\\\\\/DGkFr5Ez1icO+UZyHKBp5m0hQ\\\\\\\/hJHAxgyR9PzjJ1a1diHxb0h5DRiRBcl2B95ITLKPR2ko2PJe8N9nO9HrGgflMeQQhrzrG1aD1CtpF1jHe994NxbDpTVXpzwuutyZikw2EccSDIrgBNnLiOLjHAWzVWY0ptwd8I4ey92XFR2rQwQpve1v5C91j6BC6auuv8Hb0in8TENYsLmqO15KDeTs0Zs9APEmfZKngpLo4H\\\\\\\/FmBMCx+n2xQThbG5m94Mrbg\\\\\\\/tHTgVarEWAKQkSfvhK8s+C554thCVj\\\\\\\/pCq3xLjW2q4BfnLaKWRaus6kVYJDLwjn22HDqTiECME6ObF6EVHet8TZVo3NiPu9hwwLjfdGG4xZc6N48CiteChBI8h7xGUhpRHenTQAnT8Lg5Uv1TxrBDux0KFfOSCjG5SfoJMdxthjBr0OLNjvGtEtuJW+e+x879T8yN75tz288czEp45h9TdG10zOVkxsaMF59W4048UoxKSPyaO2sC5vnYrBFEdL0cu3AFpquhKyp9\\\\\\\/MSvcTx1hO\\\\\\\/6pK66mtVuIXBZAqC8gr1huX1YdjyRVWiGAwGnEUwCHOC6C\\\\\\\/wHf5f2I2en6m+xrx9ZbzEsUJrC1ZgGvS6JOMiRUv\\\\\\\/OC6KwKbWwVV8mttUTJrrDfDBJpKeWmyU9ntsckv8w2rIYBfqH1D8X4HV68+97GSpBPZW3XCuVAzijJ4+4QLEqA8OR4r7gbXhM1wB5b2kzUrRPaqQhiwWIlCaADcdXxWgXo\\\\\\\/o3YVkmSgG7FjN048ZcI5FAynaFfVfXpN+xmdl3v+y0nVrwkab6x0y50PeIpRH6PvIrw35\\\\\\\/EJcyFZyAVfEubjDtsax\\\\\\\/\\\\\\\/GlTUWdge1AQAynKC3YI7srwkSJMj4g\\\\\\\/+a6Vomy4wCuEdX9+GwZ8dTvN95hAdKPeo0G3Uqy1HtuqzIfWGUqUvVE4iaE\\\\\\\/7O5fbYwTbHWj4YaZq2ApTjwIYj1mpF7L65inLLSW+ltgirkfrac3OF80APqVYDJl2s1f8LFADiaRsivXNMyMDZ6koCVzjhmfW+z\\\\\\\/ej\\\\\\\/SMM9zz0GFkupTv6Puuj3d7rpyACIL0UkD\\\\\\\/5t95jJKUx\\\\\\\/YMAueWjWzVJSPbGI0POsf5lbnMHJqnQti7\\\\\\\/KZl6v6gQzPDYIy+luY64py1SFTj1K6l0WQZpUGRfSoX\\\u0022,\\\u0022iv\\\u0022:\\\u0022l\\\\\\\/Fr6aCujis1CnADYtoZPw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224f5995d6d358a9c8a4afd525007c3186740abdb3a92f78ef45cd163ea6b71f39\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Nf\\\\\\\/QsCGaOE9+SZkkA4wbEhvUVnmMpESXw0iaBTuBDXw3GQlT2SuYyTdoreNvohwaPdPaTB57iXwAWbiI4snef1XsIuQqyxqXZKfJ3Wx++fbRlutpft0pAhtWOydDmcMRYWcI5h0VyU7iYEqqwN+sErkmuK2MwRc00hyXd2lgMOQI2MDPytxXReAybbt5nB7me9YPQLD2+Ad7AnfneXiZNKd1yYxEtMJw4KWra15q7cMHuHS7duS9YIDj7KM1JvT0eInDJtUfdL8SymSvdOAmBmngHpDrXL\\\\\\\/HG+RxCpREYtvh31q3xGjNh4k\\\\\\\/BGmlMIOdr6oOaNMxD7C76AkcXnPscguvHB74oIKcLUZVUf83PZdGULUv4knsDgC3mROvEbm\\\\\\\/fos0PnKpBeKNcmYTOuSktKyY1\\\\\\\/36JUl+aGf4bFDaNci9E82W9H7rPAN+har+K+ZysFdK8OdT7cYDPrywRGSLPUxFtbrzj8BG0LHygWfvNaSsAIW+nl1ooi0O3Fss99QbmM7qush2VGKX6wE86biZ1q0O7DGX4ExQJkCJb5EWEOSLRXWb7WSCJWFW9K7h0moiPSS5LsQ1mJBnrkIM7GvMt0taxDX736lM8XbAZN0DrZnMMMQDh\\\\\\\/MpyVqB7QsLkpEZzg7GmHXX7ytUtjbhOyAiWK+NI44QSn5bFglPqL9ff1m5DKa34VzEjxYGN8VzmhY0JkWTSRAOamEsJYvs+aw8czg5p6gBAh\\\\\\\/5NDCN2Ehrtkm0aarcXaj0qAU1qArvfaDRkFp826fZIAsbmv6LMFHBigwYIxTBsWYlRhRL1rQQ\\\\\\\/vBLuGh7Dnkt1\\\\\\\/pKnchskFjZTuWNuVUqSRUMXHK3Ll1b8WxjPmseoywdflefXpX13NsInKBX4MGYDIEwJA4GiLJDYGNfpNS9tKIR13OsEA\\\\\\\/Y5xUj0K0fq8oOkq7iI2o6bzmHtWtyQsRdMMbNdIpfr6ThjMf\\\\\\\/7udhK6Jj+iJhdeN6znBGxc6nR3Qtwu0rC0wPdGADFxZBXN2trCRqXWXrt4OWyX6br+5erh\\\\\\\/Cdo8i5zCFOP4HM45ss6CaDRlNBOJp0wZvkiorwMFfoMrmsMPyHFb5v1US4umzFDVnV4Pka3ZXzn2SCw+ru1pd1K2Oc4LzZXCxHCGbEzoVfIk\\\\\\\/+nf5T+Jb\\\\\\\/0AQmWXsYh8\\\\\\\/LtA9Px9e8UlawZwCOoPfxnLWvqvO49eITstTA55QR3gtJnI0jBYqJyz4JjEz2PaBUm4TBNuHZe98CrWZFu9ucJt6qqutJQN2VpNbBH5WUMRexnf0k0wcRGMDMrO5Gg102lA+JUZut0SolyefhYLtN5P50EFOHQGeJQuEYB9KeZCbn2o+AhwFKgC1JHPGGJVxCQPIYLVNSHm2Ri654DRxhXdG3C7EXS5qtWThSEiG1j6aAUbUGatk6lJuhPV3i69MfkcTdgPomzv2hinn3zvsrI2xj3soGFFpUV19emn2cKn5A\\\\\\\/K8COVhxpUQRS\\\\\\\/\\\\\\\/tro+Sbm8ttG4ESEUCilbgqRD1YfkfJ6NzjC\\\\\\\/7Ywtbwqea5ha6h53smsMJRpI0MsSS\\\\\\\/8ussklciKqTgCR4Hw6Av3KWFcmuQ3g\\\\\\\/VJwCnGN0FmDMMibJv8IJ7+5D3C+n6AodT+HE9sbnIumBkVB+mcWo\\\\\\\/ylsavHv+lyBEJ64IZR163TDKLMeQO+alpwsyJK2WMifbeSze88pdzAIlU84p81w2cjeZPOVvJKVlVKJ0WtQhrF67QTpbgdLoHy+KHFvwrLUuBZLhbVeP9gvq++HxMQrV77ioTW4hcWDL1iE5C\\\\\\\/g5SzySXlsT1DuuLC5XF5BXXtj3MesZ5kTCox5g7pfikx7jEFFk37yigci7Q0ha\\\\\\\/5VKMnobkEEvMk3Mz++jfcCIBQsk88ynwc+l7AzZ\\\\\\\/6NCp5Hkk4HxmpQqjOCkQLHMsGJSwMU4XDeXiTk9UT0su4wtGsCZsDvS2rst4rD90MEoSF39VE8j9a9kTwmX1ybrjzJRyH+muhRLB1UjFd8HwtFZCM6d7Qijwg4FOzO1AisOtSL5Z+nh0Yh8u3l0o=\\\u0022,\\\u0022iv\\\u0022:\\\u0022gfJlNDoxmmLup+HGluSn0w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220d15bb3cc50cd149f26851e726967979c61029ecde13b2538ee326397d1cf4c3\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-484241 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1385" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-FrT60i68U5PoGvpat1bO8jOaLA91SDiygDv3x0AbFkQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1385" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3HYqTh2UfQmh2nUc-krpjpXcpNN4u2tE8xIG-wwHpBA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/4/1377" title="Genotype Imputation To Improve the Cost-Efficiency of Genomic Selection in Farmed Atlantic Salmon" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/5/1393" title="Genotype Calling from Population-Genomic Sequencing Data" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484241" id="node484241" data-pisa="ggg;7/5/1385" data-pisa-master="ggg;g3.116.033654" data-apath="/ggg/7/5/1385.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accurate Classification of Protein Subcellular Localization from High-Throughput Microscopy Images Using Deep Learning</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Tanel Pärnamaa</span> and <span class="highwire-citation-author" data-delta="1">Leopold Parts</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1385-1392;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.033654 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484241" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Tanel Pärnamaa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Computer Science, University of Tartu, 50409, Estonia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tanel%2BP%C3%A4rnamaa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A4rnamaa%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATanel%2BP%25C3%25A4rnamaa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Leopold Parts </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Computer Science, University of Tartu, 50409, Estonia</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire CB10 1SA, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leopold%2BParts%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Parts%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALeopold%2BParts%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:leopold.parts@sanger.ac.uk" class="" data-icon-position="" data-hide-link-title="0">leopold.parts@sanger.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/5/1385" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:484241" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:484241/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1385.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:484241" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:484241/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1385.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:484241" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:484241/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1385.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:484241" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:484241/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/5/1385.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:484241" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:484241/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">High-throughput microscopy of many single cells generates high-dimensional data that are far from straightforward to analyze. One important problem is automatically detecting the cellular compartment where a fluorescently-tagged protein resides, a task relatively simple for an experienced human, but difficult to automate on a computer. Here, we train an 11-layer neural network on data from mapping thousands of yeast proteins, achieving per cell localization classification accuracy of 91%, and per protein accuracy of 99% on held-out images. We confirm that low-level network features correspond to basic image characteristics, while deeper layers separate localization classes. Using this network as a feature calculator, we train standard classifiers that assign proteins to previously unseen compartments after observing only a small number of training examples. Our results are the most accurate subcellular localization classifications to date, and demonstrate the usefulness of deep learning for high-throughput microscopy.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/deep-learning" class="hw-term hw-article-keyword hw-article-keyword-deep-learning" rel="nofollow">deep learning</a></li><li class="kwd"><a href="/keyword/high-content-screening-0" class="hw-term hw-article-keyword hw-article-keyword-high-content-screening" rel="nofollow">high-content screening</a></li><li class="kwd"><a href="/keyword/machine-learning" class="hw-term hw-article-keyword hw-article-keyword-machine-learning" rel="nofollow">machine learning</a></li><li class="kwd"><a href="/keyword/microscopy" class="hw-term hw-article-keyword hw-article-keyword-microscopy" rel="nofollow">microscopy</a></li><li class="kwd"><a href="/keyword/yeast" class="hw-term hw-article-keyword hw-article-keyword-yeast" rel="nofollow">yeast</a></li></ul><p id="p-3">Microscopy images are a rich, and perhaps underutilized, source of high-throughput biological data. Endogenous proteins tagged with a fluorescent marker can report quantitative states of living cells, and help annotate gene function by recording spatial and temporal variation in localization or abundance. While biochemical assays of molecule concentrations require large lysed populations for readout, imaging can be performed on single live cells. The acquisition can be automated, producing thousands of micrographs an hour in an arrayed format. These engineering advances have paved the way for systematic screening of tagged protein collections (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Huh <em>et al.</em> 2003</a>), looking for mutant effects on protein abundance (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albert <em>et al.</em> 2014</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Parts <em>et al.</em> 2014</a>) and localization (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chong <em>et al.</em> 2015</a>), changes in cell (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Ohya <em>et al.</em> 2005</a>) and organelle (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Vizeacoumar <em>et al.</em> 2010</a>) morphology, and assigning gene function (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Farkash-Amar <em>et al.</em> 2014</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hériché 2014</a>).</p><p id="p-4">Output of a high-throughput microscopy screen has to be automatically processed (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Shamir <em>et al.</em> 2010</a>). A typical workflow consists of image normalization, cell segmentation, feature extraction, and statistical analysis; freely available tools exist that make sensible choices for each of these steps (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Collins 2007</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lamprecht <em>et al.</em> 2007</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Pau <em>et al.</em> 2010</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kamentsky <em>et al.</em> 2011</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wagih <em>et al.</em> 2013</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Wagih and Parts 2014</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bray <em>et al.</em> 2015</a>). Nevertheless, while the preprocessing stages of normalization and segmentation can be performed in a relatively standardized manner to obtain protein abundances, problem-specific feature extraction and statistical analysis are crucial for subcellular localization mapping. Image analysis pipelines need to carefully calculate more abstract features from raw pixel values, and select most informative ones to obtain numbers that matter in the context of the experiment at hand (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Glory and Murphy 2007</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Handfield <em>et al.</em> 2015</a>). Defining the correct features can be time-consuming and error-prone, and default quantities produced by existing software are not necessarily relevant outside the domain for which they were crafted (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Boland <em>et al.</em> 1998</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Conrad <em>et al.</em> 2004</a>).</p><p id="p-5">Deep neural networks (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">LeCun <em>et al.</em> 2015</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Schmidhuber 2015</a>) have recently become popular for image analysis tasks, as they overcome the feature selection problem. Methods based on deep learning have proved to be most accurate in challenges ranging from object detection (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">He <em>et al.</em> 2015</a>) to semantic segmentation (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Girshick <em>et al.</em> 2014</a>) and image captioning (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Vinyals <em>et al.</em> 2015</a>), as well as applications to biological domains (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Tan <em>et al.</em> 2015</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Angermueller <em>et al.</em> 2016</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Rampasek and Goldenberg 2016</a>), from regulatory genomics (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alipanahi <em>et al.</em> 2015</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kelley <em>et al.</em> 2016</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Zhou and Troyanskaya 2015</a>) to electron microscopy (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cireşan <em>et al.</em> 2012</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">2013</a>). For object identification from photos, these models already outperform humans (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">He <em>et al.</em> 2015</a>). Briefly, deep networks process images through consecutive layers of compute units (neurons), which quantify increasingly complex patterns in the data, and are trained to predict observed labels. One of their main appeals is that given a large enough training set, they are able to automatically learn the features most useful for the given classification problem, without a need to design them <em>a priori</em>.</p><p id="p-6">Here, we apply the deep learning paradigm to high-throughput microscopy data. We present DeepYeast, a neural network trained to classify fluorescent protein subcellular localization in yeast cells. Our network outperforms random forests trained on standard image features for determining the localization patterns, both at single cell and cell population levels, and achieves accuracies higher than previously reported. We interpret the internal outputs of the network, and find that neuron layers close to data correspond to low-level image characteristics, while deeper neurons inform of the classification state. The network can be used as a feature extractor, so that random forests trained on its output separate previously unobserved classes.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Data</h3><p id="p-7">We constructed a large-scale labeled data set based on high-throughput, proteome-scale microscopy images from <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Chong <em>et al.</em> (2015)</a>. Each image has two channels: a red fluorescent protein (mCherry) with cytosolic localization, thus marking the cell contour, and green fluorescent protein (GFP) tagging an endogenous gene in the 3′-end, which characterizes the abundance and localization of the protein. For ∼70% of the yeast proteome, the protein subcellular localization has been manually assigned (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Huh <em>et al.</em> 2003</a>). However, our data were acquired in a somewhat different genetic background and experimental setting, and labeling the images by eye can be error-prone. To obtain high confidence training examples, we therefore used images where (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Huh <em>et al.</em> 2003</a>; <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Chong <em>et al.</em> 2015</a>) annotations agree. Our final data set comprised 7132 microscopy images from 12 classes (cell periphery, cytoplasm, endosome, endoplasmic reticulum, Golgi, mitochondrion, nuclear periphery, nucleolus, nucleus, peroxisome, spindle pole, and vacuole) that were split into training, validation, and test sets. Furthermore, segmentations from <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Chong <em>et al.</em> (2015)</a> were used to crop whole images into 64 × 64 pixel patches centered on the cell midpoint, resulting in 65,000 examples for training, 12,500 for validation, and 12,500 for testing.</p></div><div id="sec-3" class="subsection"><h3>Convolutional neural network</h3><p id="p-8">We trained a deep convolutional neural network that has 11 layers (eight convolutional and three fully connected) with learnable weights (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1C</a>). We used 3 × 3 patterns with step size (stride) 1 for convolutional layers, 2 × 2 aggregation regions with step size 2 for pooling layers, and rectified linear unit nonlinearities for the activation function. The number of units in the convolutional layers was 64, 64, 128, 128, 256, 256, 256, and 256, and in the fully connected layers was 512, 512, and 12. We initialized the weights using the Glorot-normal initialization technique (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Glorot and Bengio 2010</a>), and used batch normalization (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Ioffe and Szegedy 2015</a>) after each convolutional or fully connected layer, but before activation functions. For each image, per-pixel training set mean was subtracted before use. Cross-entropy loss was minimized using stochastic gradient descent with momentum of 0.9, initial learning rate of 0.1, and a mini-batch size of 100. Learning rate was divided by two after every 16,250 iterations (25 epochs). To reduce overfitting, we used weight decay of 0.0005, and dropout with rate of 0.5 for the first two fully connected layers. The models were trained for 195,000 iterations (300 epochs over full training data), and based on validation loss, the model at iteration 130,000 was chosen for all experiments. The training took 3 days on an NVIDIA Tesla K20m graphical processing unit.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1385/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A deep neural network for protein subcellular classification. (A) Outline of the data generation and classification workflow. (B) Example pictures (two images) from each of the 12 classes (labeled above). Red fluorescence corresponds to a cytosolic marker to denote the cell, and green to the protein of interest. (C) Architecture of the &#x201C;DeepYeast&#x201D; convolutional neural network. Eight convolutional layers (yellow) are succeeded by three fully connected ones (green), producing the prediction (blue). All convolutional layers have 3 &#xD7; 3 filters with stride 1 (filter size and number of neurons in layer label), and all pooling operations (purple) are over 2 &#xD7; 2 nonoverlapping areas. ER, endoplasmic reticulum." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1909447608" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A deep neural network for protein subcellular classification. (A) Outline of the data generation and classification workflow. (B) Example pictures (two images) from each of the 12 classes (labeled above). Red fluorescence corresponds to a cytosolic marker to denote the cell, and green to the protein of interest. (C) Architecture of the &#x201C;DeepYeast&#x201D; convolutional neural network. Eight convolutional layers (yellow) are succeeded by three fully connected ones (green), producing the prediction (blue). All convolutional layers have 3 &#xD7; 3 filters with stride 1 (filter size and number of neurons in layer label), and all pooling operations (purple) are over 2 &#xD7; 2 nonoverlapping areas. ER, endoplasmic reticulum.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1385/F1.medium.gif" width="440" height="217"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/5/1385/F1.medium.gif" width="440" height="217"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1385/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1385/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484282" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-9" class="first-child">A deep neural network for protein subcellular classification. (A) Outline of the data generation and classification workflow. (B) Example pictures (two images) from each of the 12 classes (labeled above). Red fluorescence corresponds to a cytosolic marker to denote the cell, and green to the protein of interest. (C) Architecture of the “DeepYeast” convolutional neural network. Eight convolutional layers (yellow) are succeeded by three fully connected ones (green), producing the prediction (blue). All convolutional layers have 3 × 3 filters with stride 1 (filter size and number of neurons in layer label), and all pooling operations (purple) are over 2 × 2 nonoverlapping areas. ER, endoplasmic reticulum.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Random forest</h3><p id="p-10">For comparison, we trained a random forest classifier implemented in the R randomForest package (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Liaw and Wiener 2002</a>) on features from <a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Chong <em>et al.</em> (2015)</a> that were extracted using a CellProfiler (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bray <em>et al.</em> 2015</a>) pipeline. In total, there are 435 different features consisting of intensity, geometric, and texture measurements on different scales, such as Haralick texture features (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Haralick 1979</a>), Gabor (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Jain <em>et al.</em> 1997</a>), and Zernike (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">von Zernike 1934</a>) filters. We performed a grid search to select the number of trees to grow (50, 100, 250, 500, or 1000), the number of features to randomly sample at each split (10, 25, 50, 75, 100, 125, 150, 175, 200, 250, or 300), and the minimum size of terminal nodes (1, 2, 5, 10, or 50). Based on validation set performance, we chose 500, 100, and 1 for these hyperparameters, respectively. The final performance was evaluated on the same test data set as the neural network.</p></div><div id="sec-5" class="subsection"><h3>Bootstrap confidence intervals</h3><p id="p-11">To obtain C.I.s on the precision and recall estimates, we resampled the test data with replacement 20,000 times, such that the number of the different class labels remained the same, and calculated the precision and recall for each class in every bootstrap sample. 2.5 and 97.5% percentiles of the resulting distribution were used as the 95% C.I.</p></div><div id="sec-6" class="subsection"><h3>Protein-level classification</h3><p id="p-12">For both random forest and DeepYeast, we modeled the protein localization in one cell as a multinomial distribution with uninformative Dirichlet prior for the protein, and calculated the Dirichlet posterior for the protein from observations of individual cells. We used the maximum <em>a posteriori</em> estimate for protein localization. Intuitively, this approach corresponds to softly counting the number of cells assigned with each compartment and picking the compartment with the maximum count.</p></div><div id="sec-7" class="subsection"><h3>Determining good quality cells</h3><p id="p-13">To remove a prominent source of misclassifications, we trained a random forest to discriminate between cells and noncells (<em>e.g.</em>, inappropriately segmented regions, empty areas, and imaging artifacts) based on the CellProfiler features. For each of the 12 categories, we randomly sampled without replacement 100 examples from the validation set images that were correctly classified by both DeepYeast and random forest, and labeled them as good quality cells. In addition, we inspected the validation set, and manually picked 118 noncells, resulting in a total of 1200 cell and 118 noncell images. We performed 10-fold cross-validation to choose the number of features to randomly sample at each split (2, 110, 218, 326, or 435), and whether to downsample good quality images at every bootstrap sample. Based on cross-validation performance, the final model used 100 trees, 110 features at each split, no downsampling, and achieved an accuracy of 96.7%.</p></div><div id="sec-8" class="subsection"><h3>Transfer learning</h3><p id="p-14">To assess the generality of DeepYeast features learned in the classification task, we constructed a new data set from classes not present in the training data. The four new categories (actin, bud neck, lipid particle, and microtubule) each contained 1000 cell images for training, 500 for validation, and 1000 for testing. We fed the data into DeepYeast, and extracted the outputs of the first fully connected layer as features (every layer for the subsequent comparisons in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033654/-/DC1/FigureS6.tif">Figure S6</a>). We subsampled random data sets of different sizes (1, 3, 5, 10, 25, 50, 100, 250, and 500) from training data, fit a random forest classifier as implemented in the scikit-learn package (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Pedregosa <em>et al.</em> 2011</a>) to the corresponding DeepYeast and CellProfiler features, picked the best performing model on validation data, and evaluated the final performance on testing data for every data set size.</p></div><div id="sec-9" class="subsection"><h3>t-SNE visualizations</h3><p id="p-15">We picked 1000 cells at random across all classes, processed them with the DeepYeast network, and applied t-SNE (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Van der Maaten and Hinton 2008</a>) with default parameters to the neuron outputs at the different layers.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-16">The data used in this study were described in <a id="xref-ref-9-6" class="xref-bibr" href="#ref-9">Chong <em>et al.</em> (2015)</a>, and stored in the Cyclops database presented in <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Koh <em>et al.</em> (2015)</a>. The single cell images that were extracted and used for training are available at <a href="http://www.cs.ut.ee/~leopoldp/2016_DeepYeast">http://www.cs.ut.ee/∼leopoldp/2016_DeepYeast</a>. </p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Deep neural network to classify protein localization in yeast cell images</h3><p id="p-17">To perform accurate classification of protein localization in single cells and populations, we created the DeepYeast convolutional neural network learned from yeast high-throughput microscopy data generated by <a id="xref-ref-9-7" class="xref-bibr" href="#ref-9">Chong <em>et al.</em> (2015)</a> (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033654/-/DC1/FileS1.zip">File S1</a>). We used a data set comprising 90,000 cell images of 1783 proteins localized to exactly 1 of 12 cellular compartments (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>), as measured in two studies (<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Huh <em>et al.</em> 2003</a>; <a id="xref-ref-9-8" class="xref-bibr" href="#ref-9">Chong <em>et al.</em> 2015</a>). Each image records the cytoplasmic signal in the red channel, and a tagged protein of interest in the green channel. The network consists of 11 layers (eight convolutional layers with rectified linear units, followed by three fully connected layers, <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>), and a softmax output to assign one of the 12 class labels. DeepYeast’s parameters (over 10,000,000 in total) were learned in the Caffe framework (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Jia <em>et al.</em> 2014</a>), using stochastic gradient descent with momentum (<em>Materials and Methods</em>).</p></div><div id="sec-13" class="subsection"><h3>Accurate classification of protein localization in single cells and populations</h3><p id="p-18">We first compared the performance of DeepYeast trained on raw pixel values to random forests (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Breiman 2001</a>) trained on 435 features extracted using CellProfiler (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Bray <em>et al.</em> 2015</a>) by <a id="xref-ref-9-9" class="xref-bibr" href="#ref-9">Chong <em>et al.</em> (2015)</a>. We fitted the models on 72% of the single cell images using a range of parameter settings, picked the one with highest accuracy on another 14% of the images, and quantified its performance on the remaining 14% (<em>Materials and Methods</em>). No protein had a cell image in more than one of the training, testing, and validation folds.</p><p id="p-19">The deep neural network achieved classification accuracy of 87% [10,839/12,500 cells, Cohen’s κ (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cohen 1960</a>) = 0.85], compared to 75% (9375/12,500, Cohen’s κ = 0.72) for random forests (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033654/-/DC1/TableS1.xlsx">Table S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033654/-/DC1/FileS2.zip">File S2</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033654/-/DC1/FileS3.zip">File S3</a>). DeepYeast outperformed random forests for each class in recall (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>) and precision for all compartments except the nucleolus (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). The random forest performance is concordant with previous results for single cell classification on the same data set [70% accuracy (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kraus <em>et al.</em> 2015</a>)], which were obtained using an extended set of classes.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1385/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Cellular compartment classification accuracy. (A) DeepYeast outperforms random forests in classification precision. Recall (y-axis) for the 12 subcellular compartments (x-axis) for DeepYeast (red) and random forest (blue) classifiers. The dashed lines denote medians across compartments. The error bars denote the 95% C.I. from 20,000 bootstrap samples (Table S2). (B) Same as (A), but for precision on the y-axis. (C) Example classification mistakes stemming from technical issues (left) due to low signal (bottom left) or no cell (top left), population heterogeneity (middle) resulting in false positives (top middle) and false negatives (bottom middle), as well as frequent model errors (right) of classifying nucleus as nucleolus (top right), or nucleolus as spindle pole (bottom right). (D) Confusion matrix of DeepYeast classification. Error rates from the true (y-axis) to falsely predicted (x-axis) compartments. ER, endoplasmic reticulum." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1909447608" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Cellular compartment classification accuracy. (A) DeepYeast outperforms random forests in classification precision. Recall (&lt;em&gt;y&lt;/em&gt;-axis) for the 12 subcellular compartments (&lt;em&gt;x&lt;/em&gt;-axis) for DeepYeast (red) and random forest (blue) classifiers. The dashed lines denote medians across compartments. The error bars denote the 95% C.I. from 20,000 bootstrap samples (Table S2). (B) Same as (A), but for precision on the &lt;em&gt;y&lt;/em&gt;-axis. (C) Example classification mistakes stemming from technical issues (left) due to low signal (bottom left) or no cell (top left), population heterogeneity (middle) resulting in false positives (top middle) and false negatives (bottom middle), as well as frequent model errors (right) of classifying nucleus as nucleolus (top right), or nucleolus as spindle pole (bottom right). (D) Confusion matrix of DeepYeast classification. Error rates from the true (&lt;em&gt;y&lt;/em&gt;-axis) to falsely predicted (&lt;em&gt;x&lt;/em&gt;-axis) compartments. ER, endoplasmic reticulum.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1385/F2.medium.gif" width="440" height="333"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/5/1385/F2.medium.gif" width="440" height="333"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1385/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1385/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484307" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">Cellular compartment classification accuracy. (A) DeepYeast outperforms random forests in classification precision. Recall (<em>y</em>-axis) for the 12 subcellular compartments (<em>x</em>-axis) for DeepYeast (red) and random forest (blue) classifiers. The dashed lines denote medians across compartments. The error bars denote the 95% C.I. from 20,000 bootstrap samples (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033654/-/DC1/TableS2.xlsx">Table S2</a>). (B) Same as (A), but for precision on the <em>y</em>-axis. (C) Example classification mistakes stemming from technical issues (left) due to low signal (bottom left) or no cell (top left), population heterogeneity (middle) resulting in false positives (top middle) and false negatives (bottom middle), as well as frequent model errors (right) of classifying nucleus as nucleolus (top right), or nucleolus as spindle pole (bottom right). (D) Confusion matrix of DeepYeast classification. Error rates from the true (<em>y</em>-axis) to falsely predicted (<em>x</em>-axis) compartments. ER, endoplasmic reticulum.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">Mistakes occurred for each cellular compartment. Some of the errors were due to technical problems with the image, caused by low signal intensity, artifacts, or lack of proper cell (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>, left). While our results were generally robust to such noise in the input data, we further trained a classifier to distinguish good quality cell images from the CellProfiler features (<em>Materials and Methods</em>) and filtered out data deemed to have technical issues, as has been done in previous applications (<a id="xref-ref-9-10" class="xref-bibr" href="#ref-9">Chong <em>et al.</em> 2015</a>). After removing 1440 data points classified as noncell (12%), DeepYeast accuracy increased to 91% (10,080/11,060), and random forests to 79% (8756/11,060). Some remaining errors could be ascribed to labeling mistakes from contamination or population heterogeneity (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>, middle), causing the training data label to be discordant with the observed protein distribution in the cell. In the rest of the cases, DeepYeast classified the protein to the wrong compartment (<em>e.g.</em>, <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a>, right).</p><p id="p-22">The most difficult localizations to classify were endosome (recall 65%, 447 correct out of 689), spindle pole (76%, 595/781), peroxisome (80%, 131/164), Golgi (85%, 324/382), and nucleus (85%, 1386/1627). Endosomes, spindle poles, peroxisomes, and Golgi are mainly represented by varying numbers of puncta, which are not visible in all cells and may obscure each other, making them difficult to distinguish. Indeed, the most frequent misclassifications (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2D</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033654/-/DC1/FigureS1.tif">Figure S1</a>) were peroxisome to spindle pole (11%; 18 of 164 peroxisome cell images), and endosome to vacuole (8%, 56/689). Another recurring error was designating nucleolar proteins as nuclear (4%, 45/1263), both of which are large round patches. Random forests had additional common mistakes, but the most frequent misclassifications were shared with DeepYeast, reflecting the general difficulty in distinguishing punctate and patch-like patterns in a single cell (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033654/-/DC1/FileS4.zip">File S4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033654/-/DC1/FileS5.zip">File S5</a>).</p><p id="p-23">So far, we looked at individual cells, and classified the localization pattern of the fluorescent signal. Next, we asked how well we can infer the cellular compartment of a protein from all the cell images acquired for it. We assigned the localization class of each protein as the most probable class according to the posterior probability calculated from aggregating single cell data (<em>Materials and Methods</em>). Using this combined estimate, we achieved 99% classification accuracy (279/282) on the held-out test proteins, for which no single cell images were used for training. Two of the errors were nuclear proteins misclassified as nucleolar, with seven and nine cell images observed, respectively. The remaining error occurred for a protein with a single imaged cell. Thus, further requiring at least 10 cells to be recorded for each protein, the accuracy increased to 100% (222/222). Random forests were 95% accurate (269/282) on complete test data, and 96% accurate (214/222) when at least 10 cells were measured (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033654/-/DC1/FigureS2.tif">Figure S2</a>). As a baseline, <a id="xref-ref-9-11" class="xref-bibr" href="#ref-9">Chong <em>et al.</em> (2015)</a> reported per-protein accuracies of 50–90% (<a id="xref-ref-9-12" class="xref-bibr" href="#ref-9">Chong <em>et al.</em> 2015</a>) depending on the class on an overlapping data set, using the (<a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">Huh <em>et al.</em> 2003</a>) annotation as a gold standard, and support vector machine ensemble classification. While, to our knowledge, human accuracy on these or similar images has not been assessed directly, experts assign compartments to human proteins with over 80% accuracy (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Murphy <em>et al.</em> 2003</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Kraus <em>et al.</em> 2015</a>). All these performances are below what we report here, but as the data sets are not identical, direct comparisons should be interpreted with caution.</p></div><div id="sec-14" class="subsection"><h3>Neural network outputs are interpretable</h3><p id="p-24">Neural network models are often viewed as black boxes that are difficult to interpret. To gain intuition about DeepYeast features that aid prediction, we explored the characteristics of learned weights and neuron outputs. We first selected images and image patches that maximize or minimize activations of individual neurons, thus matching their weight pattern well (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). The first layers of neurons are closest to data, with small receptive fields made up by a limited number of pixels, and thus capture local, small-scale image characteristics. As an illustration, four neurons selected in the first layer were maximally activated by image patches containing edges (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>, left column), second layer neurons by patches with corners and lines, and third and fourth layers with more complex shapes (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>, middle columns). Neurons in deeper layers represent combinations of low-level features. The maximally activating patches for selected neurons started resembling class characteristics, such as punctate patterns, membrane structures, and large patches (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3A</a>, rightmost column). While these commonalities are suggestive, and may indicate the patterns that are learned by the network, they can also be due to uninteresting technical reasons, and should thus be interpreted with caution.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1385/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Visualization of the network features at different layers. Interpreting the first, second, fourth, eighth, and eleventh layers of DeepYeast (box diagram, top, see also Figure 1C). (A) Image patches that maximize some neuron output. For each of the layers, four neurons (y-axis) and image parts (x-axis) corresponding to a block of pixels that feed into them for maximum activation are shown. (B) 2D visualizations using the t-SNE algorithm (Van der Maaten and Hinton 2008). 1000 random images were fed through the network, hidden layer outputs were extracted, and the t-SNE algorithm was used to project the high-dimensional representations into two dimensions. The points are colored based on the true class categories. (C) Three closest images (x-axis) to two chosen points [1 and 2 in (B), y-axis] in the two-dimensional t-SNE projection space. (D) Distribution of mutual information (y-axis) between the multinomial class probability and discretized neuron outputs for each layer (left to right), as well as CellProfiler features (rightmost box, red)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1909447608" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Visualization of the network features at different layers. Interpreting the first, second, fourth, eighth, and eleventh layers of DeepYeast (box diagram, top, see also Figure 1C). (A) Image patches that maximize some neuron output. For each of the layers, four neurons (&lt;em&gt;y&lt;/em&gt;-axis) and image parts (&lt;em&gt;x&lt;/em&gt;-axis) corresponding to a block of pixels that feed into them for maximum activation are shown. (B) 2D visualizations using the t-SNE algorithm (Van der Maaten and Hinton 2008). 1000 random images were fed through the network, hidden layer outputs were extracted, and the t-SNE algorithm was used to project the high-dimensional representations into two dimensions. The points are colored based on the true class categories. (C) Three closest images (&lt;em&gt;x&lt;/em&gt;-axis) to two chosen points [1 and 2 in (B), &lt;em&gt;y&lt;/em&gt;-axis] in the two-dimensional t-SNE projection space. (D) Distribution of mutual information (&lt;em&gt;y&lt;/em&gt;-axis) between the multinomial class probability and discretized neuron outputs for each layer (left to right), as well as CellProfiler features (rightmost box, red).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1385/F3.medium.gif" width="440" height="346"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/5/1385/F3.medium.gif" width="440" height="346"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1385/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1385/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484327" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-25" class="first-child">Visualization of the network features at different layers. Interpreting the first, second, fourth, eighth, and eleventh layers of DeepYeast (box diagram, top, see also <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1C</a>). (A) Image patches that maximize some neuron output. For each of the layers, four neurons (<em>y</em>-axis) and image parts (<em>x</em>-axis) corresponding to a block of pixels that feed into them for maximum activation are shown. (B) 2D visualizations using the t-SNE algorithm (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Van der Maaten and Hinton 2008</a>). 1000 random images were fed through the network, hidden layer outputs were extracted, and the t-SNE algorithm was used to project the high-dimensional representations into two dimensions. The points are colored based on the true class categories. (C) Three closest images (<em>x</em>-axis) to two chosen points [1 and 2 in (B), <em>y</em>-axis] in the two-dimensional t-SNE projection space. (D) Distribution of mutual information (<em>y</em>-axis) between the multinomial class probability and discretized neuron outputs for each layer (left to right), as well as CellProfiler features (rightmost box, red).</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">Next, we applied t-SNE (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Van der Maaten and Hinton 2008</a>), a tool to visualize high-dimensional data in two dimensions, on different layers of DeepYeast outputs from 1000 randomly sampled images, and added compartment information in colors (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033654/-/DC1/FigureS3.tif">Figure S3</a>). The classes overlap substantially for lower layer outputs, while deeper layers that make use of fully connected network structure increasingly separate the localizations, such that nearby points correspond to the same class (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3C</a>). We also asked which neuron outputs are correlated to the CellProfiler features and class membership. To do so, we calculated the strongest Pearson correlation coefficient to a CellProfiler feature (as extracted by <a id="xref-ref-9-13" class="xref-bibr" href="#ref-9">Chong <em>et al.</em> (2015)</a>), as well as the largest mutual information with a class label for each unit output. The deep activations informed class labels (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3D</a>), while shallow ones were more highly correlated to CellProfiler features and Gabor filters (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033654/-/DC1/FigureS4.tif">Figure S4</a>).</p></div><div id="sec-15" class="subsection"><h3>DeepYeast can be used as a feature extractor</h3><p id="p-27">Classification of new localization classes requires creating new training sets, and if the pattern is rare, obtaining the necessary images is difficult and time-consuming. Further, while applying an existing network to new data can be simple, retraining it requires substantial effort. This motivates repurposing of trained networks as extractors of informative features, which can then be used as inputs to traditional models (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Donahue <em>et al.</em> 2013</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Razavian <em>et al.</em> 2014</a>).</p><p id="p-28">We tested whether a network trained on a large amount of data can be used to distill image information that is useful for distinguishing previously unobserved compartments as well. We processed images corresponding to four new challenging classes (actin, bud neck, lipid particle, and microtubule; <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033654/-/DC1/FileS6.zip">File S6</a>.) with DeepYeast, and calculated outputs from the first fully connected layer as features. The class labels were not independent of the features even without additional training (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>), indicating that the network extracted informative signals from the data. Next, we trained a random forest classifier on the calculated features, using an increasing number of training images. The classifiers using neural network features outperformed ones using CellProfiler features for small training set sizes (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033654/-/DC1/FigureS5.tif">Figure S5</a>), and accuracy increased further with additional data. However, the overall accuracy on these classes remained lower than others due to their punctate pattern. We repeated the experiment on the outputs of all DeepYeast layers, and found that they do not distinguish new classes equally well. In particular, the classifiers trained on the deepest convolutional layers outperformed models trained on CellProfiler features and other layers for larger training set sizes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033654/-/DC1/FigureS6.tif">Figure S6</a>). This supports previous reports that intermediate layer outputs can be more useful in a new domain, as they capture general enough features on one hand, but are not overspecialized to the trained task on the other (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Yosinski <em>et al.</em> 2014</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1385/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Transfer learning works. (A) Four example images of each of the additional analyzed classes. (B) Applying t-SNE to the network outputs of the additional data (see also Figure 3B) and coloring the points according to the classes demonstrates separation of new compartments based on features trained for classifying other localizations. (C) Classification accuracy on held-out data (y-axis) for different number of training images (x-axis) for DeepYeast outputs (red) or CellProfiler features (blue) used as inputs to a random forest. The error bars denote a 95% C.I. from 20,000 bootstrap samples." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1909447608" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Transfer learning works. (A) Four example images of each of the additional analyzed classes. (B) Applying t-SNE to the network outputs of the additional data (see also Figure 3B) and coloring the points according to the classes demonstrates separation of new compartments based on features trained for classifying other localizations. (C) Classification accuracy on held-out data (&lt;em&gt;y&lt;/em&gt;-axis) for different number of training images (&lt;em&gt;x&lt;/em&gt;-axis) for DeepYeast outputs (red) or CellProfiler features (blue) used as inputs to a random forest. The error bars denote a 95% C.I. from 20,000 bootstrap samples.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1385/F4.medium.gif" width="440" height="148"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/5/1385/F4.medium.gif" width="440" height="148"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1385/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1385/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484300" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-29" class="first-child">Transfer learning works. (A) Four example images of each of the additional analyzed classes. (B) Applying t-SNE to the network outputs of the additional data (see also <a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3B</a>) and coloring the points according to the classes demonstrates separation of new compartments based on features trained for classifying other localizations. (C) Classification accuracy on held-out data (<em>y</em>-axis) for different number of training images (<em>x</em>-axis) for DeepYeast outputs (red) or CellProfiler features (blue) used as inputs to a random forest. The error bars denote a 95% C.I. from 20,000 bootstrap samples.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">DeepYeast was trained on proteins that predominantly localize to a single compartment. Finally, we confirmed that proteins spread between multiple classes can be accurately inferred as such. From the set of proteins assayed by <a id="xref-ref-9-14" class="xref-bibr" href="#ref-9">Chong <em>et al.</em> (2015)</a> not used in our analyses so far, we selected the ones manually annotated to belong to both nucleus and cytoplasm, and calculated their posterior class probabilities. As expected, cytoplasmic and nuclear classes had high posterior probability, and were the two most probable classes in 21/24 cases, and in the top three for the remaining cases. The per-gene posterior probability of compartment assignment can further be interpreted as the frequency of cells for which the protein resides in the compartment; the model is not forced to make a sharp decision and assign each gene to a single location.</p></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-31">We have demonstrated that DeepYeast, an 11-layer convolutional neural network, can achieve classification accuracy of 91% for individual cells over 12 subcellular localizations, and 100% for proteins when entire cell populations of at least moderate size are considered. Far from being a black box, the internal outputs that DeepYeast produces can be visualized and interpreted in terms of image characteristics. The pretrained network functions as a feature extractor to successfully distinguish previously unseen classes, and infer mixtures of compartments in a population.</p><p id="p-32">The classification errors mostly occurred between compartments that are also difficult to distinguish by eye. The various numbers of puncta in peroxisomes, spindle poles, and endosomes can look like each other, or not be present at all. Nucleus and nucleolus are patches of similar size; when the characteristic crescent shape of the nucleolus is not showing, it is also difficult to distinguish from the nuclear marker. Overall, the single cell accuracy of 91% is approaching the protein compartment assignment performance of previous reports (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Boland and Murphy 2001</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Murphy <em>et al.</em> 2003</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Conrad <em>et al.</em> 2004</a>; <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Kraus <em>et al.</em> 2015</a>), and the remaining errors are often borderline cases, for which classification is difficult even for trained humans (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033654/-/DC1/FileS4.zip">File S4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033654/-/DC1/FileS5.zip">File S5</a>). Nevertheless, when at least 10 individual cells were measured, the correct cell classifications dominated the errors, and all test proteins were assigned to the right compartment in held-out data.</p><p id="p-33">The success of deep neural networks in image analysis relies on architectures that encapsulate a hierarchy of increasingly abstract features relevant for classification, and plentiful training data to learn the model parameters. While first applications used a smaller number of layers (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Boland <em>et al.</em> 1998</a>) and mostly operated on precalculated features (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Boland <em>et al.</em> 1998</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Boland and Murphy 2001</a>; <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Conrad <em>et al.</em> 2004</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chen <em>et al.</em> 2007</a>), pixel level analyses gave good results (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Danckaert <em>et al.</em> 2002</a>), especially using the latest training methods (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Cireşan <em>et al.</em> 2013</a>; <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Kraus <em>et al.</em> 2015</a>). Subcellular localization is defined by spatial variation on different length scales, from single small dots to extended thin membranes. Quantification of this covariance structure is thus important for accurate modeling, but deriving the right features for it requires mathematical sophistication and computational crafting (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Handfield <em>et al.</em> 2015</a>). The convolutional layers in the neural network are agnostic to the location of the signal in the image, and take inputs from progressively larger patterns, thus capturing spatial correlations of increasingly wide range in a data-driven manner.</p><p id="p-34">The choice of model architecture was guided by previous results and practical considerations. Even with modern GPUs, the end-to-end training of a deep neural network is computationally intensive; therefore, we did not attempt to evaluate the influence of model architecture on the results. Three fully connected layers and the use of 3 × 3 filters has previously been shown to offer a rich enough parametrization to capture interesting feature combinations, outperforming alternatives in standardized tasks (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Simonyan and Zisserman 2014</a>). The eight convolutional layers with three 2 × 2 pooling operations gives features that span the entire cell image, at which scale we expected the organelle characteristics to be reflected. Finally, the architecture we chose was the largest that could fit in the memory of the graphical processing unit we used.</p><p id="p-35">DeepYeast can be reused for other image analysis experiments with the same marker proteins and magnification, or trained further for specific applications. We demonstrated that a pretrained model can be applied for both classifying previously unseen compartments and inferring mixtures of localization patterns. The usual classification implementations do not always provide models that are easy to reuse. We envision a repository of networks trained on various bioimage compendia that can be downloaded and employed as out-of-the-box feature calculators, or fine-tuned with additional data to obtain niche-specific results, provided access to the necessary infrastructure is available. Similar resources already exist in other domains, and are being pioneered in bioimage analysis as well (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kraus <em>et al</em>. 2017</a>).</p><p id="p-36">While our DeepYeast network outperformed the random forest alternative, and achieved accuracies better than reported before, the direct comparisons must be interpreted with care. We used a clean training set of proteins localized to a single compartment as was done in previous work (<a id="xref-ref-9-15" class="xref-bibr" href="#ref-9">Chong <em>et al.</em> 2015</a>; <a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">Kraus <em>et al.</em> 2015</a>), but as the training data do not match completely, the performance differences may partly be due to the data set composition. Both <a id="xref-ref-9-16" class="xref-bibr" href="#ref-9">Chong <em>et al.</em> (2015)</a> and <a id="xref-ref-32-6" class="xref-bibr" href="#ref-32">Kraus <em>et al.</em> (2015)</a> relied on information from segmented cells for best classification performance; we considered only patches known to contain a cell without pixel-level segmentation information. This circumvents the need for very accurate segmentation pipelines, and indeed, centers of cells can also be derived from additional markers, <em>e.g.</em>, histone tags in the nucleus (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Parts <em>et al.</em> 2014</a>) that are cleanly separated, and therefore much easier to segment than entire cells.</p><p id="p-37">Deep neural networks have proved their value in extracting information from large-scale image data (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Krizhevsky <em>et al.</em> 2012</a>; <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">He <em>et al.</em> 2015</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">LeCun <em>et al.</em> 2015</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Ronneberger <em>et al.</em> 2015</a>). It would be unreasonable to believe that the same will not be true for high-throughput microscopy. Adaptation of the technology will depend on the ease with which it is deployed and shared between researchers; to this end, we have made our trained network freely available. The utility of these approaches will increase with accumulation of publicly shared data, and we expect deep neural networks to prove themselves a powerful class of models for biological image and data analysis.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-38">We thank Tambet Matiisen for technical assistance; Helena Friesen for consultations on yeast images; Yolanda Chong, Judice Koh, and Oren Kraus for help with accessing the data; and Martin Hemberg, Jared Simpson, Felicity Allen, Oliver Stegle, and Christof Angermüller for comments on the text. We gratefully acknowledge the support of NVIDIA Corporation for donating the graphics processing unit used in this research. T.P. was supported by the European Regional Development Fund through the BioMedIT project, and the Estonian Research Council (IUT34-4). L.P. was supported by the Wellcome Trust, and the Estonian Research Council (IUT34-4). </p><p id="p-39">Author contributions: T.P. trained the models and analyzed data. T.P. and L.P. conceived and designed the analysis approaches, and wrote the manuscript. The authors declare no competing financial interests.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-40">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033654/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033654/-/DC1</a> and the corresponding file descriptions in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033654/-/DC1/FileS7.docx">File S7</a>.</p></li><li class="fn" id="fn-2"><p id="p-41"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-07-18"><span class="received-label">Received </span>July 18, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-11-22"><span class="accepted-label">Accepted </span>November 22, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Parnamaa and Parts</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.1" data-doi="10.1038/nature12904"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Treusch</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shockley</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetics of single-cell protein abundance variation in large yeast populations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">506</span>(<span class="cit-issue">7489</span>): <span class="cit-fpage">494</span>–<span class="cit-lpage">497</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D506%26rft.spage%253D494%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12904%26rft_id%253Dinfo%253Apmid%252F24402228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12904&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24402228&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000332165100039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.2" data-doi="10.1038/nbt.3300"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alipanahi</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delong</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weirauch</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frey</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Predicting the sequence specificities of DNA- and RNA-binding proteins by deep learning.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">831</span>–<span class="cit-lpage">838</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D831%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3300%26rft_id%253Dinfo%253Apmid%252F26213851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3300&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26213851&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.3" data-doi="10.15252/msb.20156651"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Angermueller</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parnamaa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stegle</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Deep learning for computational biology.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">878</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DAngermueller%26rft.auinit1%253DC.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D878%26rft.epage%253D878%26rft.atitle%253DDeep%2Blearning%2Bfor%2Bcomputational%2Bbiology%26rft_id%253Dinfo%253Adoi%252F10.15252%252Fmsb.20156651%26rft_id%253Dinfo%253Apmid%252F27474269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjg6IjEyLzcvODc4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzEzODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.4" data-doi="10.1093/bioinformatics/17.12.1213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boland</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A neural network classifier capable of recognizing the patterns of all major subcellular structures in fluorescence microscope images of HeLa cells.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">17</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">1213</span>–<span class="cit-lpage">1223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBoland%26rft.auinit1%253DM.%2BV.%26rft.volume%253D17%26rft.issue%253D12%26rft.spage%253D1213%26rft.epage%253D1223%26rft.atitle%253DA%2Bneural%2Bnetwork%2Bclassifier%2Bcapable%2Bof%2Brecognizing%2Bthe%2Bpatterns%2B%2Bof%2Ball%2Bmajor%2Bsubcellular%2Bstructures%2Bin%2Bfluorescence%2Bmicroscope%2B%2Bimages%2Bof%2BHeLa%2Bcells%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F17.12.1213%26rft_id%253Dinfo%253Apmid%252F11751230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMTcvMTIvMTIxMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.5" data-doi="10.1002/(SICI)1097-0320(19981101)33:3&lt;366::AID-CYTO12&gt;3.0.CO;2-R"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boland</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Markey</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Automated recognition of patterns characteristic of subcellular structures in fluorescence microscopy images.</span> <abbr class="cit-jnl-abbrev">Cytometry</abbr> <span class="cit-vol">33</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">366</span>–<span class="cit-lpage">375</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytometry%26rft.stitle%253DCytometry%26rft.aulast%253DBoland%26rft.auinit1%253DM.%2BV.%26rft.volume%253D33%26rft.issue%253D3%26rft.spage%253D366%26rft.epage%253D375%26rft.atitle%253DAutomated%2Brecognition%2Bof%2Bpatterns%2Bcharacteristic%2Bof%2Bsubcellular%2Bstructures%2Bin%2Bfluorescence%2Bmicroscopy%2Bimages.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0320%252819981101%252933%253A3%253C366%253A%253AAID-CYTO12%253E3.0.CO%253B2-R%26rft_id%253Dinfo%253Apmid%252F9822349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0320(19981101)33:3&lt;366::AID-CYTO12&gt;3.0.CO;2-R&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9822349&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000076794900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bray</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vokes</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carpenter</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Using CellProfiler for automatic identification and measurement of biological objects in images.</span> <abbr class="cit-jnl-abbrev">Curr. Protoc. Mol. Biol.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">14.17.1</span>–<span class="cit-lpage">14.17.13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BProtoc.%2BMol.%2BBiol.%26rft.volume%253D109%26rft.spage%253D14171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.7" data-doi="10.1023/A:1010933404324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breiman</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Random forests.</span> <abbr class="cit-jnl-abbrev">Mach. Learn.</abbr> <span class="cit-vol">45</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">5</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMach.%2BLearn.%26rft.volume%253D45%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1010933404324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1010933404324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000170489900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.8" data-doi="10.1093/bioinformatics/btm206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Automated image analysis of protein localization in budding yeast.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>(<span class="cit-issue">13</span>): <span class="cit-fpage">i66</span>–<span class="cit-lpage">i71</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DChen%26rft.auinit1%253DS.-C.%26rft.volume%253D23%26rft.issue%253D13%26rft.spage%253Di66%26rft.epage%253Di71%26rft.atitle%253DAutomated%2Bimage%2Banalysis%2Bof%2Bprotein%2Blocalization%2Bin%2Bbudding%2Byeast%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm206%26rft_id%253Dinfo%253Apmid%252F17646347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyMy8xMy9pNjYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.9" data-doi="10.1016/j.cell.2015.04.051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chong</span>  <span class="cit-name-given-names">Y. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koh</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friesen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duffy</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Yeast proteome dynamics from single cell imaging and automated analysis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">161</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">1413</span>–<span class="cit-lpage">1424</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D161%26rft.spage%253D1413%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.04.051%26rft_id%253Dinfo%253Apmid%252F26046442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.04.051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26046442&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-book" id="cit-7.5.1385.10"><div class="cit-metadata unstructured">Cireşan, D., A. Giusti, L. M. Gambardella, and J. Schmidhuber, 2012 Deep neural networks segment neuronal membranes in electron microscopy images. Proceedings of Advances in Neural Information Processing Systems 2012, Lake Tahoe, NV, pp. 2843–2851.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-book" id="cit-7.5.1385.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cireşan</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giusti</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gambardella</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidhuber</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mitosis detection in breast cancer histology images with deep neural networks</span>, pp. <span class="cit-fpage">411</span>–<span class="cit-lpage">418</span> in <span class="cit-source"><em>Medical Image Computing and Computer-Assisted Intervention–MICCAI 2013</em></span>, edited by K. Mori, I. Sakuma, Y. Sato, C. Barillot, and N. Navab. <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">Berlin</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.12" data-doi="10.1177/001316446002000104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cohen</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1960</span> <span class="cit-article-title">Kappa: coefficient of concordance.</span> <abbr class="cit-jnl-abbrev">Educ. Psychol. Meas.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEducational%2Band%2BPsychological%2BMeasurement%26rft.stitle%253DEducational%2Band%2BPsychological%2BMeasurement%26rft.aulast%253DCohen%26rft.auinit1%253DJ.%26rft.volume%253D20%26rft.issue%253D1%26rft.spage%253D37%26rft.epage%253D46%26rft.atitle%253DA%2BCoefficient%2Bof%2BAgreement%2Bfor%2BNominal%2BScales%26rft_id%253Dinfo%253Adoi%252F10.1177%252F001316446002000104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1177/001316446002000104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1960CCC3600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.13" data-doi="10.2144/000112517"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">ImageJ for microscopy.</span> <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">43</span>(<span class="cit-supplement">Suppl. 1</span>) <span class="cit-fpage">25</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioTechniques%26rft.stitle%253DBiotechniques%26rft.aulast%253DLederman%26rft.auinit1%253DL.%26rft.volume%253D43%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D%2B29%26rft.atitle%253DHigh-content%2Bscreening.%26rft_id%253Dinfo%253Adoi%252F10.2144%252F000112517%26rft_id%253Dinfo%253Apmid%252F17695642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2144/000112517&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17695642&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.14" data-doi="10.1101/gr.2383804"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conrad</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erfle</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warnat</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daigle</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorch</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Automatic identification of subcellular phenotypes on human cell arrays.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">1130</span>–<span class="cit-lpage">1136</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DConrad%26rft.auinit1%253DC.%26rft.volume%253D14%26rft.issue%253D6%26rft.spage%253D1130%26rft.epage%253D1136%26rft.atitle%253DAutomatic%2BIdentification%2Bof%2BSubcellular%2BPhenotypes%2Bon%2BHuman%2BCell%2BArrays%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.2383804%26rft_id%253Dinfo%253Apmid%252F15173118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE0LzYvMTEzMCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danckaert</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez-Couto</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bollondi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Automated recognition of intracellular organelles in confocal microscope images.</span> <abbr class="cit-jnl-abbrev">Traffic</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">66</span>–<span class="cit-lpage">73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTraffic%2B%2528Copenhagen%252C%2BDenmark%2529%26rft.stitle%253DTraffic%26rft.aulast%253DDanckaert%26rft.auinit1%253DA.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D66%26rft.epage%253D73%26rft.atitle%253DAutomated%2Brecognition%2Bof%2Bintracellular%2Borganelles%2Bin%2Bconfocal%2Bmicroscope%2Bimages.%26rft_id%253Dinfo%253Apmid%252F11872144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11872144&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-web" id="cit-7.5.1385.16"><div class="cit-metadata unstructured">Donahue, J., Y. Jia, O. Vinyals, J. Hoffman, N. Zhang <em>et al.</em>, 2013 Decaf: a deep convolutional activation feature for generic visual recognition. Available at: <a href="https://arxiv.org/abs/1310.1531">https://arxiv.org/abs/1310.1531</a>. Accessed: January 16, 2017.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Farkash-Amar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimmer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eden</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cohen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geva-Zatorsky</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Noise genetics: inferring protein function by correlating phenotype with protein levels and localization in individual human cells.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">e1004176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.5.1385.18"><div class="cit-metadata unstructured">Girshick, R., J. Donahue, T. Darrell, and J. Malik, 2014 Rich feature hierarchies for accurate object detection and semantic segmentation. Proceedings of the 2014 IEEE Conference on Computer Vision and Pattern Recognition, Columbus, OH, pp. 580–587.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.5.1385.19"><div class="cit-metadata unstructured">Glorot, X., and Y. Bengio, 2010 Understanding the difficulty of training deep feed forward neural networks. Proceedings of the 13th International Conference on Artificial Intelligence and Statistics, Chia Laguna Resort, Sardinia, Italy, pp. 249–256.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.20" data-doi="10.1016/j.devcel.2006.12.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glory</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Automated subcellular location determination and high-throughput microscopy.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">7</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DGlory%26rft.auinit1%253DE.%26rft.volume%253D12%26rft.issue%253D1%26rft.spage%253D7%26rft.epage%253D16%26rft.atitle%253DAutomated%2Bsubcellular%2Blocation%2Bdetermination%2Band%2Bhigh-throughput%2Bmicroscopy.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2006.12.007%26rft_id%253Dinfo%253Apmid%252F17199037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2006.12.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17199037&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245086900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.21" data-doi="10.1093/bioinformatics/btu759"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Handfield</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strome</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chong</span>  <span class="cit-name-given-names">Y. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moses</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Local statistics allow quantification of cell-to-cell variability from high-throughput microscope images.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">940</span>–<span class="cit-lpage">947</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu759%26rft_id%253Dinfo%253Apmid%252F25398614%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIzMS82Lzk0MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haralick</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Statistical and structural approaches to texture.</span> <abbr class="cit-jnl-abbrev">Proc. IEEE</abbr> <span class="cit-vol">67</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">786</span>–<span class="cit-lpage">804</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BIEEE%26rft.volume%253D67%26rft.spage%253D786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-web" id="cit-7.5.1385.23"><div class="cit-metadata unstructured">He, K., X. Zhang, S. Ren, and J. Sun, 2015 Deep residual learning for image recognition. Available at: <a href="https://arxiv.org/abs/1512.03385">https://arxiv.org/abs/1512.03385</a>. Accessed: January 16, 2017.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-book" id="cit-7.5.1385.24"><div class="cit-metadata unstructured">Hériché J.-K. 2014 Systematic cell phenotyping, pp. 86–110 in <em>Phenomics</em>, edited by J. M. Hancock. CRC Press, Boca Raton, FL.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.25" data-doi="10.1038/nature02026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huh</span>  <span class="cit-name-given-names">W. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falvo</span>  <span class="cit-name-given-names">J. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerke</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howson</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Global analysis of protein localization in budding yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">425</span>(<span class="cit-issue">6959</span>): <span class="cit-fpage">686</span>–<span class="cit-lpage">691</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHuh%26rft.auinit1%253DW.%2BK.%26rft.volume%253D425%26rft.issue%253D6959%26rft.spage%253D686%26rft.epage%253D691%26rft.atitle%253DGlobal%2Banalysis%2Bof%2Bprotein%2Blocalization%2Bin%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02026%26rft_id%253Dinfo%253Apmid%252F14562095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14562095&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185924500033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-web" id="cit-7.5.1385.26"><div class="cit-metadata unstructured">Ioffe, S., and C. Szegedy, 2015 Batch normalization: accelerating deep network training by reducing internal covariate shift. Available at: <a href="https://arxiv.org/abs/1502.03167">https://arxiv.org/abs/1502.03167</a>. Accessed January 16, 2017.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.27" data-doi="10.1016/S0031-3203(96)00068-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ratha</span>  <span class="cit-name-given-names">N. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lakshmanan</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Object detection using Gabor filters.</span> <abbr class="cit-jnl-abbrev">Pattern Recognit.</abbr> <span class="cit-vol">30</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">295</span>–<span class="cit-lpage">309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPattern%2BRecognit.%26rft.volume%253D30%26rft.spage%253D295%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0031-3203%252896%252900068-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0031-3203(96)00068-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.5.1385.28"><div class="cit-metadata unstructured">Jia, Y., E. Shelhamer, J. Donahue, S. Karayev, J. Long <em>et al.</em>, 2014 Caffe: convolutional architecture for fast feature embedding. <em>Proceedings of the 22nd ACM International Conference on Multimedia</em>, Orlando, FL, pp. 675–678.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.29" data-doi="10.1093/bioinformatics/btr095"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kamentsky</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bray</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Logan</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Improved structure, function and compatibility for CellProfiler: modular high-throughput image analysis software.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">1179</span>–<span class="cit-lpage">1180</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr095%26rft_id%253Dinfo%253Apmid%252F21349861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyNy84LzExNzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.30" data-doi="10.1101/gr.200535.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snoek</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rinn</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Basset: learning the regulatory code of the accessible genome with deep convolutional neural networks.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">990</span>–<span class="cit-lpage">999</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.200535.115%26rft_id%253Dinfo%253Apmid%252F27197224%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI2LzcvOTkwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzEzODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koh</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chong</span>  <span class="cit-name-given-names">Y. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friesen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moses</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">CYCLoPs: a comprehensive database constructed from automated analysis of protein abundance and subcellular localization patterns in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">1223</span>–<span class="cit-lpage">1232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D5%26rft.spage%253D1223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-web" id="cit-7.5.1385.32"><div class="cit-metadata unstructured">Kraus, O. Z., L. J. Ba, and B. Frey, 2015 Classifying and segmenting microscopy images using convolutional multiple instance learning. Available at: <a href="https://arxiv.org/abs/1511.05286">https://arxiv.org/abs/1511.05286</a>. Accessed: January 16, 2017.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kraus</span>  <span class="cit-name-given-names">O.Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grys</span>  <span class="cit-name-given-names">B.T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ba</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chong</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frey</span>  <span class="cit-name-given-names">B.J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andrews</span>  <span class="cit-name-given-names">B.J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Automated analysis of high-content microscopy data with deep learning.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> DOI: 10.15252/msb.20177551</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-book" id="cit-7.5.1385.34"><div class="cit-metadata unstructured">Krizhevsky, A., I. Sutskever, and G. E. Hinton, 2012 Imagenet classification with deep convolutional neural networks. Proceedings of Advances in Neural Information Processing Systems 2012, Lake Tahoe, NV, pp. 1097–1105.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.35" data-doi="10.2144/000112257"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lamprecht</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabatini</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carpenter</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">CellProfiler: free, versatile software for automated biological image analysis.</span> <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">42</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">71</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioTechniques%26rft.stitle%253DBiotechniques%26rft.aulast%253DLamprecht%26rft.auinit1%253DM.%2BR.%26rft.volume%253D42%26rft.issue%253D1%26rft.spage%253D71%26rft.epage%253D75%26rft.atitle%253DCellProfiler%253A%2Bfree%252C%2Bversatile%2Bsoftware%2Bfor%2Bautomated%2Bbiological%2Bimage%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.2144%252F000112257%26rft_id%253Dinfo%253Apmid%252F17269487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2144/000112257&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17269487&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248948900021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.36" data-doi="10.1038/nature14539"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">LeCun</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bengio</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinton</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Deep learning.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">521</span>: <span class="cit-fpage">436</span>–<span class="cit-lpage">444</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D521%26rft.spage%253D436%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14539%26rft_id%253Dinfo%253Apmid%252F26017442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14539&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26017442&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liaw</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiener</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Classification and regression by randomForest.</span> <abbr class="cit-jnl-abbrev">R News</abbr> <span class="cit-vol">2</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">18</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR%2BNews%26rft.volume%253D2%26rft.spage%253D18%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.38" data-doi="10.1023/B:VLSI.0000003028.71666.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Velliste</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Porreca</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Robust numerical features for description and classification of subcellular location patterns in fluorescence microscope images.</span> <abbr class="cit-jnl-abbrev">J. VLSI Signal Process. Syst. Signal Image Video Technol.</abbr> <span class="cit-vol">35</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">311</span>–<span class="cit-lpage">321</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVLSI%2BSignal%2BProcess.%2BSyst.%2BSignal%2BImage%2BVideo%2BTechnol.%26rft.volume%253D35%26rft.spage%253D311%26rft_id%253Dinfo%253Adoi%252F10.1023%252FB%253AVLSI.0000003028.71666.44%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/B:VLSI.0000003028.71666.44&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.39" data-doi="10.1073/pnas.0509436102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohya</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sese</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yukawa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sano</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakatani</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">High-dimensional and large-scale phenotyping of yeast mutants.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>(<span class="cit-issue">52</span>): <span class="cit-fpage">19015</span>–<span class="cit-lpage">19020</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0509436102%26rft_id%253Dinfo%253Apmid%252F16365294%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzUyLzE5MDE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzEzODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.40" data-doi="10.1101/gr.170506.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tekkedil</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caudy</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Heritability and genetic basis of protein level variation in an outbred population.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">1363</span>–<span class="cit-lpage">1370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.170506.113%26rft_id%253Dinfo%253Apmid%252F24823668%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI0LzgvMTM2MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.41" data-doi="10.1093/bioinformatics/btq046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pau</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuchs</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sklyar</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boutros</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">EBImage—an R package for image processing with applications to cellular phenotypes.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">979</span>–<span class="cit-lpage">981</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq046%26rft_id%253Dinfo%253Apmid%252F20338898%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi83Lzk3OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pedregosa</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varoquaux</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gramfort</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michel</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thirion</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Scikit-learn: machine learning in Python.</span> <abbr class="cit-jnl-abbrev">J. Mach. Learn. Res.</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">Oct</span>): <span class="cit-fpage">2825</span>–<span class="cit-lpage">2830</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMach.%2BLearn.%2BRes.%26rft.volume%253D12%26rft.spage%253D2825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rampasek</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldenberg</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">TensorFlow: biology’s gateway to deep learning?</span> <abbr class="cit-jnl-abbrev">Cell Syst.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">12</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BSyst.%26rft.volume%253D2%26rft.spage%253D12%26rft_id%253Dinfo%253Apmid%252F27136685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=27136685&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.5.1385.44"><div class="cit-metadata unstructured">Razavian, A., H. Azizpour, J. Sullivan, and S. Carlsson, 2014 CNN features off-the-shelf: an astounding baseline for recognition. Proceedings of the 2014 IEEE Conference on Computer Vision and Pattern Recognition, Columbus, OH, pp. 806–813.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-book" id="cit-7.5.1385.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ronneberger</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brox</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">U-Net: convolutional networks for biomedical image segmentation</span>, pp. <span class="cit-fpage">234</span>–<span class="cit-lpage">241</span> in <span class="cit-source"><em>Medical Image Computing and Computer-Assisted Intervention–MICCAI 2015</em></span>, edited by N. Navab, J. Hornegger, W. M. Wells, and A. F. Frangi. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Heidelberg</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.46" data-doi="10.1016/j.neunet.2014.09.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmidhuber</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Deep learning in neural networks: an overview.</span> <abbr class="cit-jnl-abbrev">Neural Netw.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeural%2BNetw.%26rft.volume%253D61%26rft.spage%253D85%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neunet.2014.09.003%26rft_id%253Dinfo%253Apmid%252F25462637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neunet.2014.09.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25462637&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.47" data-doi="10.1371/journal.pcbi.1000974"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shamir</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delaney</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orlov</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckley</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldberg</span>  <span class="cit-name-given-names">I. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Pattern recognition software and techniques for biological image analysis.</span> <abbr class="cit-jnl-abbrev">PLoS Comput. Biol.</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">e1000974</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DShamir%26rft.auinit1%253DL.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253De1000974%26rft.epage%253De1000974%26rft.atitle%253DPattern%2Brecognition%2Bsoftware%2Band%2Btechniques%2Bfor%2Bbiological%2Bimage%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1000974%26rft_id%253Dinfo%253Apmid%252F21124870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1000974&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21124870&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-web" id="cit-7.5.1385.48"><div class="cit-metadata unstructured">Simonyan, K., and A. Zisserman, 2014 Very deep convolutional networks for large-scale image recognition. Available at: <a href="https://arxiv.org/abs/1409.1556">https://arxiv.org/abs/1409.1556</a>. Accessed: January 16, 2017.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ung</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greene</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Unsupervised feature construction and knowledge extraction from genome-wide assays of breast cancer with denoising autoencoders.</span> <abbr class="cit-jnl-abbrev">Pac. Symp. Biocomput.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">132</span>–<span class="cit-lpage">143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPac.%2BSymp.%2BBiocomput.%26rft.volume%253D20%26rft.spage%253D132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van der Maaten</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinton</span>  <span class="cit-name-given-names">G</span></span></li></ol><cite>. <span class="cit-pub-date">2008</span> <span class="cit-article-title">Visualizing data using t-SNE.</span> <abbr class="cit-jnl-abbrev">J. Mac. Learn. Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">2579</span>–<span class="cit-lpage">2605</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMac.%2BLearn.%2BRes.%26rft.volume%253D9%26rft.spage%253D2579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.5.1385.51"><div class="cit-metadata unstructured">Vinyals, O., A. Toshev, S. Bengio, and D. Erhan, 2015 Show and tell: a neural image caption generator. Proceedings of the 2015 IEEE Conference on Computer Vision and Pattern Recognition, Boston, MA, pp. 3156–3164.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.52" data-doi="10.1083/jcb.200909013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vizeacoumar</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Dyk</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vizeacoumar</span>  <span class="cit-name-given-names">F. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Integrating high-throughput genetic interaction mapping and high-content screening to explore yeast spindle morphogenesis.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">188</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">69</span>–<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.200909013%26rft_id%253Dinfo%253Apmid%252F20065090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjg6IjE4OC8xLzY5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzEzODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.53" data-doi="10.1016/S0031-8914(34)80259-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">von Zernike</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1934</span> <span class="cit-article-title">Beugungstheorie des schneidenver-fahrens und seiner verbesserten form, der phasenkontrastmethode.</span> <abbr class="cit-jnl-abbrev">Physica</abbr> <span class="cit-vol">1</span>(<span class="cit-issue">7–12</span>): <span class="cit-fpage">689</span>–<span class="cit-lpage">704</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysica%26rft.volume%253D1%26rft.spage%253D689%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0031-8914%252834%252980259-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0031-8914(34)80259-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000201401800080&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wagih</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">gitter: a robust and accurate method for quantification of colony sizes from plate images.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">547</span>–<span class="cit-lpage">552</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D4%26rft.spage%253D547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-web" id="cit-7.5.1385.55" data-doi="10.1093/nar/gkt400"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wagih</span>  <span class="cit-name-given-names">O</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usaj</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">VanderSluis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuzmin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">SGAtools: one-stop analysis and visualization of array-based genetic interaction screens.</span> <span class="cit-source">Nucleic Acids Res.</span> <span class="cit-vol">41</span>(<span class="cit-issue">Web Server issue</span>): <span class="cit-fpage">W591</span>–<span class="cit-lpage">W596</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt400%26rft_id%253Dinfo%253Apmid%252F23677617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MS9XMS9XNTkxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzEzODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-book" id="cit-7.5.1385.56"><div class="cit-metadata unstructured">Yosinski, J., J. Clune, Y. Bengio, and H. Lipson, 2014 How transferable are features in deep neural networks? Proceedings of Advances Neural Information Processing Systems 2014, Montréal, Canada, pp. 3320–3328.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1385.57" data-doi="10.1038/nmeth.3547"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troyanskaya</span>  <span class="cit-name-given-names">O. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Predicting effects of noncoding variants with deep learning-based sequence model.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">931</span>–<span class="cit-lpage">934</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D12%26rft.spage%253D931%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3547%26rft_id%253Dinfo%253Apmid%252F26301843%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3547&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26301843&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/5/1385.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/4/1377" title="Genotype Imputation To Improve the Cost-Efficiency of Genomic Selection in Farmed Atlantic Salmon" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/5/1393" title="Genotype Calling from Population-Genomic Sequencing Data" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/5"><span>Volume 7 Issue 5, May 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/5.cover-source.jpg?itok=VWLvGstM" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1385.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3Q9bjhJ4uo4uNrUx8n_0D8fOKCWbZFrM91rc3tC8vjs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/5/1385" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/5/1385" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/5/1385" class="active" data-icon-position="" data-hide-link-title="0">Accurate Classification of Protein Subcellular Localization from High-Throughput Microscopy Images Using Deep Learning</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/484241" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-rcZ9e-RcGVAAqyTFrdCmET_cb9WGhkOtrenOhvTO0vY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/5/1385.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/5/1385" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-My5uAocok0yIoAfP-jb9j4vrx3bg4en_cus6YLkXK2M" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/5/1385" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/5/1385.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484241" id="node484241--2" data-pisa="ggg;7/5/1385" data-pisa-master="ggg;g3.116.033654" data-apath="/ggg/7/5/1385.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accurate Classification of Protein Subcellular Localization from High-Throughput Microscopy Images Using Deep Learning</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Tanel Pärnamaa</span> and <span class="highwire-citation-author" data-delta="1">Leopold Parts</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1385-1392;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.033654 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484241--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Tanel Pärnamaa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Computer Science, University of Tartu, 50409, Estonia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tanel%2BP%C3%A4rnamaa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A4rnamaa%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATanel%2BP%25C3%25A4rnamaa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Leopold Parts </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Computer Science, University of Tartu, 50409, Estonia</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire CB10 1SA, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leopold%2BParts%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Parts%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALeopold%2BParts%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:leopold.parts@sanger.ac.uk" class="" data-icon-position="" data-hide-link-title="0">leopold.parts@sanger.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/5/1385" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/5/1385&amp;title=Accurate%20Classification%20of%20Protein%20Subcellular%20Localization%20from%20High-Throughput%20Microscopy%20Images%20Using%20Deep%20Learning" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/5/1385&amp;title=Accurate%20Classification%20of%20Protein%20Subcellular%20Localization%20from%20High-Throughput%20Microscopy%20Images%20Using%20Deep%20Learning" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/5/1385&amp;title=Accurate%20Classification%20of%20Protein%20Subcellular%20Localization%20from%20High-Throughput%20Microscopy%20Images%20Using%20Deep%20Learning" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/5/1385&amp;text=Accurate%20Classification%20of%20Protein%20Subcellular%20Localization%20from%20High-Throughput%20Microscopy%20Images%20Using%20Deep%20Learning" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/5/1385&amp;title=Accurate%20Classification%20of%20Protein%20Subcellular%20Localization%20from%20High-Throughput%20Microscopy%20Images%20Using%20Deep%20Learning" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/5/1385&amp;t=Accurate%20Classification%20of%20Protein%20Subcellular%20Localization%20from%20High-Throughput%20Microscopy%20Images%20Using%20Deep%20Learning" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/5/1385&amp;title=Accurate%20Classification%20of%20Protein%20Subcellular%20Localization%20from%20High-Throughput%20Microscopy%20Images%20Using%20Deep%20Learning" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/5/1385&amp;title=Accurate%20Classification%20of%20Protein%20Subcellular%20Localization%20from%20High-Throughput%20Microscopy%20Images%20Using%20Deep%20Learning" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/484241/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484241" id="node484241--3" data-pisa="ggg;7/5/1385" data-pisa-master="ggg;g3.116.033654" data-apath="/ggg/7/5/1385.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accurate Classification of Protein Subcellular Localization from High-Throughput Microscopy Images Using Deep Learning</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Tanel Pärnamaa</span> and <span class="highwire-citation-author" data-delta="1">Leopold Parts</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1385-1392;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.033654 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484241--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Tanel Pärnamaa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Computer Science, University of Tartu, 50409, Estonia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tanel%2BP%C3%A4rnamaa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A4rnamaa%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATanel%2BP%25C3%25A4rnamaa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Leopold Parts </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Computer Science, University of Tartu, 50409, Estonia</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire CB10 1SA, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leopold%2BParts%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Parts%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALeopold%2BParts%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:leopold.parts@sanger.ac.uk" class="" data-icon-position="" data-hide-link-title="0">leopold.parts@sanger.ac.uk</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1385&amp;rft.epage=1392&amp;rft.atitle=Accurate%20Classification%20of%20Protein%20Subcellular%20Localization%20from%20High-Throughput%20Microscopy%20Images%20Using%20Deep%20Learning&amp;rft.volume=7&amp;rft.issue=5&amp;rft.date=2017-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=P%C3%A4rnamaa%2C+Tanel&amp;rft.au=Parts%2C+Leopold"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/484241/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/484241/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/484241/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/484241/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/484241/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/484241/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/484241/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/484241/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/484241/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/484241/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/484241/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/484241/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/5/1385" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/5/1385.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/5/1385.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/5/1385.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/5/1385.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
