<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/5/1405" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/5/1405.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/5/1405.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/5/1405.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/5/1405.atom" />
<meta name="HW.pisa" content="ggg;7/5/1405" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A New Reference Genome Assembly for the Microcrustacean Daphnia pulex" />
<meta name="DC.Identifier" content="10.1534/g3.116.038638" />
<meta name="DC.Date" content="2017-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Ye et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Comparing genomes of closely related genotypes from populations with distinct demographic histories can help reveal the impact of effective population size on genome evolution. For this purpose, we present a high quality genome assembly of Daphnia pulex (PA42), and compare this with the first sequenced genome of this species (TCO), which was derived from an isolate from a population with &gt;90% reduction in nucleotide diversity. PA42 has numerous similarities to TCO at the gene level, with an average amino acid sequence identity of 98.8 and &gt;60% of orthologous proteins identical. Nonetheless, there is a highly elevated number of genes in the TCO genome annotation, with ∼7000 excess genes appearing to be false positives. This view is supported by the high GC content, lack of introns, and short length of these suspicious gene annotations. Consistent with the view that reduced effective population size can facilitate the accumulation of slightly deleterious genomic features, we observe more proliferation of transposable elements (TEs) and a higher frequency of gained introns in the TCO genome." />
<meta name="DC.Contributor" content="Zhiqiang Ye" />
<meta name="DC.Contributor" content="Sen Xu" />
<meta name="DC.Contributor" content="Ken Spitze" />
<meta name="DC.Contributor" content="Jana Asselman" />
<meta name="DC.Contributor" content="Xiaoqian Jiang" />
<meta name="DC.Contributor" content="Matthew S. Ackerman" />
<meta name="DC.Contributor" content="Jacqueline Lopez" />
<meta name="DC.Contributor" content="Brent Harker" />
<meta name="DC.Contributor" content="R. Taylor Raborn" />
<meta name="DC.Contributor" content="W. Kelley Thomas" />
<meta name="DC.Contributor" content="Jordan Ramsdell" />
<meta name="DC.Contributor" content="Michael E. Pfrender" />
<meta name="DC.Contributor" content="Michael Lynch" />
<meta name="article:published_time" content="2017-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A New Reference Genome Assembly for the Microcrustacean Daphnia pulex" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Comparing genomes of closely related genotypes from populations with distinct demographic histories can help reveal the impact of effective population size on genome evolution. For this purpose, we present a high quality genome assembly of &lt;i&gt;Daphnia pulex&lt;/i&gt; (PA42), and compare this with the first sequenced genome of this species (TCO), which was derived from an isolate from a population with &amp;gt;90% reduction in nucleotide diversity. PA42 has numerous similarities to TCO at the gene level, with an average amino acid sequence identity of 98.8 and &amp;gt;60% of orthologous proteins identical. Nonetheless, there is a highly elevated number of genes in the TCO genome annotation, with ∼7000 excess genes appearing to be false positives. This view is supported by the high GC content, lack of introns, and short length of these suspicious gene annotations. Consistent with the view that reduced effective population size can facilitate the accumulation of slightly deleterious genomic features, we observe more proliferation of transposable elements (TEs) and a higher frequency of gained introns in the TCO genome.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/05/01" />
<meta name="citation_mjid" content="ggg;7/5/1405" />
<meta name="citation_id" content="7/5/1405" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/5/1405" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/5/1405.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/5/1405.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/5/1405.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.038638" />
<meta name="citation_pmid" content="28235826" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1405" />
<meta name="citation_lastpage" content="1416" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Zhiqiang Ye" />
<meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author_email" content="zhiqiang.yes@gmail.com" />
<meta name="citation_author" content="Sen Xu" />
<meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author_institution" content="Department of Biology, University of Texas at Arlington, Texas 76019" />
<meta name="citation_author" content="Ken Spitze" />
<meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author" content="Jana Asselman" />
<meta name="citation_author_institution" content="Laboratory for Environmental Toxicology, Ghent University, B-9000, Belgium" />
<meta name="citation_author" content="Xiaoqian Jiang" />
<meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author" content="Matthew S. Ackerman" />
<meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author" content="Jacqueline Lopez" />
<meta name="citation_author_institution" content="Notre Dame Genomics and Bioinformatics Core Facility, University of Notre Dame, Indiana 46556" />
<meta name="citation_author" content="Brent Harker" />
<meta name="citation_author_institution" content="Notre Dame Genomics and Bioinformatics Core Facility, University of Notre Dame, Indiana 46556" />
<meta name="citation_author" content="R. Taylor Raborn" />
<meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author_institution" content="School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author" content="W. Kelley Thomas" />
<meta name="citation_author_institution" content="Hubbard Center for Genome Studies, University of New Hampshire, Durham, New Hampshire 03824" />
<meta name="citation_author" content="Jordan Ramsdell" />
<meta name="citation_author_institution" content="Hubbard Center for Genome Studies, University of New Hampshire, Durham, New Hampshire 03824" />
<meta name="citation_author" content="Michael E. Pfrender" />
<meta name="citation_author_institution" content="Department of Biological Sciences, University of Notre Dame, Indiana 46556" />
<meta name="citation_author_institution" content="Environmental Change Initiative, University of Notre Dame, Indiana 46556" />
<meta name="citation_author" content="Michael Lynch" />
<meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. Asselman;citation_author=M. E. Pfrender;citation_author=J. A. Lopez;citation_author=D. I. De Coninck;citation_author=C. R. Janssen;citation_title=Conserved transcriptional responses to cyanobacterial stressors are mediated by alternate regulation of paralogous genes in Daphnia.;citation_pages=1844-1855;citation_volume=24;citation_year=2015;citation_doi=10.1111/mec.13148" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=C. M. Bergman;citation_author=B. D. Pfeiffer;citation_author=D. E. Rincón-Limas;citation_author=R. A. Hoskins;citation_author=A. Gnirke;citation_title=Assessing the impact of comparative genomic sequence data on the functional annotation of the Drosophila genome.;citation_pages=RESEARCH0086-RESEARCH0086;citation_volume=3;citation_year=2002;citation_issue=12;citation_pmid=12537575" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=M. Boetzer;citation_author=W. Pirovano;citation_title=SSPACE-LongRead: scaffolding bacterial draft genomes using long read sequence information.;citation_pages=211;citation_volume=15;citation_year=2014;citation_pmid=24950923;citation_doi=10.1186/1471-2105-15-211" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data.;citation_volume=30;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_title=Environ. Sci. Technol.;citation_author=S. Chen;citation_author=G. J. McKinney;citation_author=K. M. Nichols;citation_author=J. K. Colbourne;citation_author=M. S. Sepulveda;citation_title=Novel cadmium responsive microRNAs in Daphnia pulex.;citation_pages=14605-14613;citation_volume=49;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Biological Journal of the Linnean Society;citation_journal_abbrev=Biological Journal of the Linnean Society;citation_author=J. K. Colbourne;citation_author=T. J. Crease;citation_author=L. Weider;citation_author=P. D. Hebert;citation_author=F. Duferesne;citation_title=Phylogenetics and evolution of a circumarctic species complex (Cladocera; Daphnia pulex);citation_pages=347-365;citation_volume=65;citation_year=1998;citation_issue=3;citation_georef=1999027354" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. K. Colbourne;citation_author=M. E. Pfrender;citation_author=D. Gilbert;citation_author=W. K. Thomas;citation_author=A. Tucker;citation_title=The Ecoresponsive Genome of Daphnia pulex;citation_pages=555-561;citation_volume=331;citation_year=2011;citation_issue=6017;citation_pmid=21292972;citation_doi=10.1126/science.1197761" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=T. J. Crease;citation_author=S. K. Lee;citation_author=S. L. Yu;citation_author=K. Spitze;citation_author=N. Lehman;citation_title=Allozyme and mtDNA variation in populations of the Daphnia pulex complex from both sides of the Rocky Mountains.;citation_pages=242-251;citation_volume=79;citation_year=1997;citation_doi=10.1038/sj.hdy.6882220" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. Deceliere;citation_author=S. Charles;citation_author=C. Biémont;citation_title=The dynamics of transposable elements in structured populations.;citation_volume=169;citation_year=2005;citation_pmid=15466430;citation_doi=10.1534/genetics.104.032243" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=J. F. Denton;citation_author=J. Lugo-Martinez;citation_author=A. E. Tucker;citation_author=D. R. Schrider;citation_author=W. C. Warren;citation_title=Extensive error in the number of genes inferred from draft genome assemblies.;citation_pages=e1003998;citation_volume=10;citation_year=2014;citation_pmid=25474019;citation_doi=10.1371/journal.pcbi.1003998" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=E. S. Dolgin;citation_author=B. Charlesworth;citation_author=A. D. Cutter;citation_title=Population frequencies of transposable elements in selfing and outcrossing Caenorhabditis nematodes.;citation_pages=317-329;citation_volume=90;citation_year=2008;citation_issue=4;citation_pmid=18840306;citation_doi=10.1017/S0016672308009440" />
<meta name="citation_reference" content="citation_journal_title=Quat. Sci. Rev.;citation_author=A. S. Dyke;citation_author=J. T. Andrews;citation_author=P. U. Clark;citation_author=J. H. England;citation_author=G. H. Miller;citation_title=QUAT SCI REV;citation_pages=9-31;citation_volume=21;citation_year=2002;citation_doi=10.1016/S0277-3791(01)00095-6" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Ellinghaus;citation_author=S. Kurtz;citation_author=U. Willhoeft;citation_title=LTRharvest, an efficient and flexible software for de novo detection of LTR retrotransposons.;citation_pages=18-18;citation_volume=9;citation_year=2008;citation_pmid=18194517;citation_doi=10.1186/1471-2105-9-18" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. C. English;citation_author=S. Richards;citation_author=Y. Han;citation_author=M. Wang;citation_author=V. Vee;citation_title=Mind the Gap: Upgrading Genomes with Pacific Biosciences RS Long-Read Sequencing Technology.;citation_pages=e47768-e47768;citation_volume=7;citation_year=2012;citation_issue=11;citation_pmid=23185243;citation_doi=10.1371/journal.pone.0047768" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. D. Finn;citation_author=A. Bateman;citation_author=J. Clements;citation_author=P. Coggill;citation_author=R. Y. Eberhardt;citation_title=Pfam: the protein families database.;citation_volume=42;citation_year=2014;citation_pmid=24288371;citation_doi=10.1093/nar/gkt1223" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=L. Fu;citation_author=B. Niu;citation_author=Z. Zhu;citation_author=S. Wu;citation_author=W. Li;citation_title=CD-HIT: accelerated for clustering the next-generation sequencing data.;citation_volume=28;citation_year=2012;citation_pmid=23060610;citation_doi=10.1093/bioinformatics/bts565" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Gnerre;citation_author=I. MacCallum;citation_author=D. Przybylski;citation_author=F. J. Ribeiro;citation_author=J. N. Burton;citation_title=High-quality draft assemblies of mammalian genomes from massively parallel sequence data.;citation_volume=108;citation_year=2011;citation_pmid=21187386;citation_doi=10.1073/pnas.1017351108" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Götz;citation_author=J. M. García-Gómez;citation_author=J. Terol;citation_author=T. D. Williams;citation_author=S. H. Nagaraj;citation_title=High-throughput functional annotation and data mining with the Blast2GO suite.;citation_volume=36;citation_year=2008;citation_pmid=18445632;citation_doi=10.1093/nar/gkn176" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=J. Gough;citation_author=K. Karplus;citation_author=R. Hughey;citation_author=C. Chothia;citation_title=Assignment of homology to genome sequences using a library of hidden Markov models that represent all proteins of known structure.;citation_pages=903-919;citation_volume=313;citation_year=2001;citation_issue=4;citation_pmid=11697912;citation_doi=10.1006/jmbi.2001.5080" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=M. G. Grabherr;citation_author=B. J. Haas;citation_author=M. Yassour;citation_author=J. Z. Levin;citation_author=D. A. Thompson;citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome.;citation_pages=644-652;citation_volume=29;citation_year=2011;citation_issue=7;citation_pmid=21572440;citation_doi=10.1038/nbt.1883" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. V. Han;citation_author=G. W. Thomas;citation_author=J. Lugo-Martinez;citation_author=M. W. Hahn;citation_title=Estimating gene gain and loss rates in the presence of error in genome assembly and annotation using CAFE 3.;citation_volume=30;citation_year=2013;citation_pmid=23709260;citation_doi=10.1093/molbev/mst100" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Holt;citation_author=M. Yandell;citation_title=MAKER2: an annotation pipeline and genome-database management tool for second-generation genome projects.;citation_pages=491-491;citation_volume=12;citation_year=2011;citation_pmid=22192575;citation_doi=10.1186/1471-2105-12-491" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_title=Insertion polymorphisms of mobile elements in sexual and asexual populations of Daphnia pulex.;citation_author=X. Jiang;citation_author=H. Tang;citation_author=Z. Ye;citation_author=M. Lynch;citation_volume=9;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Jones;citation_author=D. Binns;citation_author=H. Y. Chang;citation_author=M. Fraser;citation_author=W. Li;citation_title=InterProScan 5: genome-scale protein function classification.;citation_volume=30;citation_year=2014;citation_pmid=24451626;citation_doi=10.1093/bioinformatics/btu031" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=Y. Kato;citation_author=K. Kobayashi;citation_author=H. Watanabe;citation_author=T. Iguchi;citation_title=Environmental sex determination in the branchiopod crustacean Daphnia magna: deep conservation of a Doublesex gene in the sex-determining pathway.;citation_pages=e1001345-e1001345;citation_volume=7;citation_year=2011;citation_issue=3;citation_pmid=21455482;citation_doi=10.1371/journal.pgen.1001345" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=K. Katoh;citation_author=K.-i. Kuma;citation_author=H. Toh;citation_author=T. Miyata;citation_title=MAFFT version 5: improvement in accuracy of multiple sequence alignment;citation_pages=511-518;citation_volume=33;citation_year=2005;citation_issue=2;citation_pmid=15661851;citation_doi=10.1093/nar/gki198" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=N. Keith;citation_author=A. E. Tucker;citation_author=C. E. Jackson;citation_author=W. Sung;citation_author=J. I. Lledó;citation_title=High mutational rates of large-scale duplication and deletion in Daphnia pulex.;citation_volume=26;citation_year=2016;citation_pmid=26518480;citation_doi=10.1101/gr.191338.115" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=I. Korf;citation_title=Gene finding in novel genomes.;citation_pages=59-59;citation_volume=5;citation_year=2004;citation_pmid=15144565;citation_doi=10.1186/1471-2105-5-59" />
<meta name="citation_reference" content="citation_journal_abbrev=Ecol Lett;citation_author=L. C. Latta;citation_author=L. J. Weider;citation_author=J. K. Colbourne;citation_author=M. E. Pfrender;citation_title=The evolution of salinity tolerance in Daphnia: a functional genomics approach.;citation_pages=794-802;citation_volume=15;citation_year=2012;citation_issue=8;citation_pmid=22583985;citation_doi=10.1111/j.1461-0248.2012.01799.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. M. Leggett;citation_author=B. J. Clavijo;citation_author=L. Clissold;citation_author=M. D. Clark;citation_author=M. Caccamo;citation_title=NextClip: an analysis and read preparation tool for Nextera Long Mate Pair libraries.;citation_volume=30;citation_year=2014;citation_pmid=24297520;citation_doi=10.1093/bioinformatics/btt702" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Le Rouzic;citation_author=T. S. Boutin;citation_author=P. Capy;citation_title=Long-term evolution of transposable elements.;citation_volume=104;citation_year=2007;citation_pmid=18040048;citation_doi=10.1073/pnas.0705238104" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows–Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=L. Li;citation_author=C. J. Stoeckert;citation_author=D. S. Roos;citation_title=OrthoMCL: Identification of Ortholog Groups for Eukaryotic Genomes;citation_pages=2178-2189;citation_volume=13;citation_year=2003;citation_issue=9;citation_pmid=12952885;citation_doi=10.1101/gr.1224503" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=W. Li;citation_author=A. E. Tucker;citation_author=W. Sung;citation_author=W. K. Thomas;citation_author=M. Lynch;citation_title=Extensive, Recent Intron Gains in Daphnia Populations;citation_pages=1260-1262;citation_volume=326;citation_year=2009;citation_issue=5957;citation_pmid=19965475;citation_doi=10.1126/science.1179302" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=W. Li;citation_author=R. Kuzoff;citation_author=C. K. Wong;citation_author=A. Tucker;citation_author=M. Lynch;citation_title=Characterization of newly gained introns in Daphnia populations.;citation_volume=6;citation_year=2014;citation_pmid=25123113;citation_doi=10.1093/gbe/evu174" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=S. Lockton;citation_author=B. S. Gaut;citation_title=The evolution of transposable elements in natural populations of self-fertilizing Arabidopsis thaliana and its outcrossing relative Arabidopsis lyrata.;citation_pages=10-10;citation_volume=10;citation_year=2010;citation_pmid=20067644;citation_doi=10.1186/1471-2148-10-10" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Löytynoja;citation_author=N. Goldman;citation_title=An algorithm for progressive multiple alignment of sequences with insertions.;citation_volume=102;citation_year=2005;citation_pmid=16000407;citation_doi=10.1073/pnas.0409137102" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. Lynch;citation_title=Intron evolution as a population-genetic process;citation_pages=6118-6123;citation_volume=99;citation_year=2002;citation_issue=9;citation_pmid=11983904;citation_doi=10.1073/pnas.092595699" />
<meta name="citation_reference" content="citation_title=The Origins of Genome Architecture;citation_author=M. Lynch;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=M. Lynch;citation_author=M. Pfrender;citation_author=K. Spitze;citation_author=N. Lehman;citation_author=J. Hicks;citation_title=The quantitative and molecular genetic architecture of a subdivided species.;citation_pages=100-110;citation_volume=53;citation_year=1999;citation_doi=10.2307/2640923" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Lynch;citation_author=M. Ackerman;citation_author=K. Spitze;citation_author=Z. Ye;citation_author=T. Maruki;citation_title=Population genomics of Daphnia pulex.;citation_pages=315-332;citation_volume=206;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=A. G. Murzin;citation_author=S. E. Brenner;citation_author=T. Hubbard;citation_author=C. Chothia;citation_title=SCOP: a structural classification of proteins database for the investigation of sequences and structures.;citation_pages=536-540;citation_volume=247;citation_year=1995;citation_issue=4;citation_pmid=7723011;citation_doi=10.1006/jmbi.1995.0159" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. Nei;citation_author=W. H. Li;citation_title=Mathematical model for studying genetic variation in terms of restriction endonucleases.;citation_pages=5269-5273;citation_volume=76;citation_year=1979;citation_issue=10;citation_pmid=291943;citation_doi=10.1073/pnas.76.10.5269" />
<meta name="citation_reference" content="citation_journal_title=Res. Comput. Mol. Biol.;citation_author=S. Nurk;citation_author=A. Bankevich;citation_author=D. Antipov;citation_author=A. Gurevich;citation_author=A. Korobeynikov;citation_title=Assembling genomes and mini-metagenomes from highly chimeric reads.;citation_pages=158-170;citation_volume=7821;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Briefings in Bioinformatics;citation_journal_abbrev=Brief Bioinform;citation_author=S. A. Olson;citation_title=Emboss opens up sequence analysis;citation_pages=87-91;citation_volume=3;citation_year=2002;citation_issue=1;citation_pmid=12002227;citation_doi=10.1093/bib/3.1.87" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. R. Omilian;citation_author=M. Lynch;citation_title=Patterns of intraspecific DNA variation in the Daphnia nuclear genome.;citation_volume=182;citation_year=2009;citation_pmid=19304589;citation_doi=10.1534/genetics.108.099549" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. R. Omilian;citation_author=D. G. Scofield;citation_author=M. Lynch;citation_title=Intron presence-absence polymorphisms in Daphnia.;citation_volume=25;citation_year=2008;citation_pmid=18667441;citation_doi=10.1093/molbev/msn164" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. Parra;citation_author=K. Bradnam;citation_author=I. Korf;citation_title=CEGMA: a pipeline to accurately annotate core genes in eukaryotic genomes.;citation_volume=23;citation_year=2007;citation_pmid=17332020;citation_doi=10.1093/bioinformatics/btm071" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. N. Price;citation_author=P. S. Dehal;citation_author=A. P. Arkin;citation_title=FastTree: computing large minimum evolution trees with profiles instead of a distance matrix.;citation_volume=26;citation_year=2009;citation_pmid=19377059;citation_doi=10.1093/molbev/msp077" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=W. Qi;citation_author=G. Nong;citation_author=J. F. Preston;citation_author=F. Ben-Ami;citation_author=D. Ebert;citation_title=Comparative metagenomics of Daphnia symbionts.;citation_pages=172-172;citation_volume=10;citation_year=2009;citation_pmid=19383155;citation_doi=10.1186/1471-2164-10-172" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. T. Raborn;citation_author=K. Spitze;citation_author=V. P. Brendel;citation_author=M. Lynch;citation_title=Promoter architecture and sex-specific gene expression in Daphnia pulex.;citation_pages=593-612;citation_volume=2;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Rho;citation_author=H. Tang;citation_title=MGEScan-non-LTR: computational identification and classification of autonomous non-LTR retrotransposons in eukaryotic genomes.;citation_volume=37;citation_year=2009;citation_pmid=19762481;citation_doi=10.1093/nar/gkp752" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. Rho;citation_author=J.-H. Choi;citation_author=S. Kim;citation_author=M. Lynch;citation_author=H. Tang;citation_title=De novo identification of LTR retrotransposons in eukaryotic genomes.;citation_pages=90-90;citation_volume=8;citation_year=2007;citation_pmid=17407597;citation_doi=10.1186/1471-2164-8-90" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=V. M. Sarich;citation_author=A. C. Wilson;citation_title=Immunological Time Scale for Hominid Evolution;citation_pages=1200-1203;citation_volume=158;citation_year=1967;citation_issue=3805;citation_pmid=4964406;citation_doi=10.1126/science.158.3805.1200" />
<meta name="citation_reference" content="citation_journal_title=BMC Biotechnol.;citation_author=C. A. Schumpert;citation_author=J. L. Dudycha;citation_author=R. C. Patel;citation_title=Development of an efficient RNA interference method by feeding for the microcrustacean Daphnia.;citation_pages=91;citation_volume=15;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=A. Shafer;citation_author=C. I. Cullingham;citation_author=S. D. Cote;citation_author=D. W. Coltman;citation_title=Of glaciers and refugia: a decade of study sheds new light on the phylogeography of northwestern North America.;citation_pages=4589-4621;citation_volume=19;citation_year=2010;citation_issue=21;citation_pmid=20849561;citation_doi=10.1111/j.1365-294X.2010.04828.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. A. Simão;citation_author=R. M. Waterhouse;citation_author=P. Ioannidis;citation_author=E. V. Kriventseva;citation_author=E. M. Zdobnov;citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.;citation_volume=31;citation_year=2015;citation_pmid=26059717;citation_doi=10.1093/bioinformatics/btv351" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Tamura;citation_author=G. Stecher;citation_author=D. Peterson;citation_author=A. Filipski;citation_author=S. Kumar;citation_title=MEGA6: molecular evolutionary genetics analysis version 6.0.;citation_volume=30;citation_year=2013;citation_pmid=24132122;citation_doi=10.1093/molbev/mst197" />
<meta name="citation_reference" content="citation_journal_title=Genome Dyn.;citation_author=M. Tollis;citation_author=S. Boissinot;citation_title=The evolutionary dynamics of transposable elements in eukaryote genomes.;citation_pages=68-91;citation_volume=7;citation_year=2012;citation_pmid=22759814;citation_doi=10.1159/000337126" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. E. Tucker;citation_author=M. S. Ackerman;citation_author=B. D. Eads;citation_author=S. Xu;citation_author=M. Lynch;citation_title=Population-genomic insights into the evolutionary origin and fate of obligately asexual Daphnia pulex.;citation_volume=110;citation_year=2013;citation_pmid=23959868;citation_doi=10.1073/pnas.1313388110" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. M. Waterhouse;citation_author=F. Tegenfeldt;citation_author=J. Li;citation_author=E. M. Zdobnov;citation_author=E. V. Kriventseva;citation_title=OrthoDB: a hierarchical catalog of animal, fungal and bacterial orthologs.;citation_volume=41;citation_year=2013;citation_pmid=23180791;citation_doi=10.1093/nar/gks1116" />
<meta name="citation_reference" content="citation_journal_title=Sci. China Life Sci.;citation_author=L. Wei;citation_author=X. Cao;citation_title=The effect of transposable elements on phenotypic variation: insights from plants to humans.;citation_pages=24-37;citation_volume=59;citation_year=2016;citation_pmid=26753674;citation_doi=10.1007/s11427-015-4993-2" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=C. I. Wu;citation_author=W. H. Li;citation_title=Evidence for higher rates of nucleotide substitution in rodents than in man.;citation_pages=1741-1745;citation_volume=82;citation_year=1985;citation_issue=6;citation_pmid=3856856;citation_doi=10.1073/pnas.82.6.1741" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Xu;citation_author=M. S. Ackerman;citation_author=H. Long;citation_author=L. Bright;citation_author=K. Spitze;citation_title=A male-specific genetic map of the microcrustacean Daphnia pulex based on single sperm whole-genome sequencing.;citation_volume=201;citation_year=2015;citation_pmid=26116153;citation_doi=10.1534/genetics.115.179028" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Xu;citation_author=K. Spitze;citation_author=M. S. Ackerman;citation_author=Z. Ye;citation_author=L. Bright;citation_title=Hybridization and the origin of contagious asexuality in Daphnia pulex.;citation_volume=32;citation_year=2015;citation_pmid=26351296;citation_doi=10.1093/molbev/msv190" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Z. Yang;citation_title=PAML 4: phylogenetic analysis by maximum likelihood.;citation_volume=24;citation_year=2007;citation_pmid=17483113;citation_doi=10.1093/molbev/msm088" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=Z. Yang;citation_author=R. Nielsen;citation_title=Estimating Synonymous and Nonsynonymous Substitution Rates Under Realistic Evolutionary Models;citation_pages=32-43;citation_volume=17;citation_year=2000;citation_issue=1;citation_pmid=10666704;citation_doi=10.1093/oxfordjournals.molbev.a026236" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A New Reference Genome Assembly for the Microcrustacean Daphnia pulex" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
<meta name="twitter:description" content="Comparing genomes of closely related genotypes from populations with distinct demographic histories can help reveal the impact of effective population size on genome evolution. For this purpose, we present a high quality genome assembly of Daphnia pulex (PA42), and compare this with the first sequenced genome of this species (TCO), which was derived from an isolate from a population with &gt;90% reduction in nucleotide diversity. PA42 has numerous similarities to TCO at the gene level, with an average amino acid sequence identity of 98.8 and &gt;60% of orthologous proteins identical. Nonetheless, there is a highly elevated number of genes in the TCO genome annotation, with ∼7000 excess genes appearing to be false positives. This view is supported by the high GC content, lack of introns, and short length of these suspicious gene annotations. Consistent with the view that reduced effective population size can facilitate the accumulation of slightly deleterious genomic features, we observe more proliferation of transposable elements (TEs) and a higher frequency of gained introns in the TCO genome." />
<meta name="og-title" property="og:title" content="A New Reference Genome Assembly for the Microcrustacean Daphnia pulex" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/5/1405" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Comparing genomes of closely related genotypes from populations with distinct demographic histories can help reveal the impact of effective population size on genome evolution. For this purpose, we present a high quality genome assembly of Daphnia pulex (PA42), and compare this with the first sequenced genome of this species (TCO), which was derived from an isolate from a population with &gt;90% reduction in nucleotide diversity. PA42 has numerous similarities to TCO at the gene level, with an average amino acid sequence identity of 98.8 and &gt;60% of orthologous proteins identical. Nonetheless, there is a highly elevated number of genes in the TCO genome annotation, with ∼7000 excess genes appearing to be false positives. This view is supported by the high GC content, lack of introns, and short length of these suspicious gene annotations. Consistent with the view that reduced effective population size can facilitate the accumulation of slightly deleterious genomic features, we observe more proliferation of transposable elements (TEs) and a higher frequency of gained introns in the TCO genome." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
    <title>A New Reference Genome Assembly for the Microcrustacean Daphnia pulex | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"hcwzdXtwDwVdD8lDlzo0NMZlBEOjAPHuKZFKkxrY-NY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"484253","apath":"\/ggg\/7\/5\/1405.atom","pisa":"ggg;7\/5\/1405","ac":{"\/ggg\/7\/5\/1405.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/5\/1405.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/5\/1405"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/5\/1405"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/5\/1405.abstract":true,"\/content\/7\/5\/1405":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222YAThY15p8hPuwiyd+2NkFszIBX4ZSeKL1RdvDlo9ThlkNUuQJ9nPpASxHX09NxYnHWRKTDUXiDR8pbSFfjMq2fyfrEqnYYcJI5Llit6EMSnHBl7dxUzSCRVqIZKpVmIgqUwOMA64r8nn3TWKeFRtKfEw19\\\\\\\/5XgdDtTxjhFu3ONk31qkHp+9ZZluucziEqf9bFhxRW4N3+x38sRnwHxT\\\\\\\/QbRd8HhIBl4RwJxu0PmyzvYTejaQmAMUs9I2GsQwjVUXvgYm2SX79SpMQbuzCJD2CxBhyLOJFzpWJSnkb24XyhGaJBEmBtazBF1sX9xsLiXmZARwS6nAIS2Rpm2ub51Oc\\\\\\\/GVJOddD7wtCA\\\\\\\/YpovJ+CuC\\\\\\\/QCy4l8JV6XblLIX3Smx8VtZOWIXnwPhjzF3UuQezNb1XX1CKJpNbQxzpQK8AAY1+oW2Wz2\\\\\\\/LHU\\\\\\\/X7YqvC3Ht3mRgiuyYC+cIyEc3F0HavVGZ\\\\\\\/9zej29oNHfIK0mhMMa+FcAOxmedFCsOzmatyIiRODR6uQkee4VQVp7y34V3mFfyvIzwUqhxrUabcvMgG1WnE0gUvTUdQf42+KGOT6Ub2Bn5OeXpC8uvKq7\\\\\\\/jPjIKk2yqfeJnHCWjgzfSrnWvquBBMivIqqg7SfdB8iEWoNEqnf6edq72CEherWDck2IZmCZ01xsm6Fo0eDVKQ0Ntthw62pfoYDSC9uwaXdDV+z9zrT5Gyytzjtk9K3ZgUdxQqj+kocnNXzBk5nq1XGjSaj5HFFMeXqLSqcY7RLySjHgQA5RdZ6MsbKxbmzFDULay+PaYm8TPporQ\\\\\\\/naQ4n+H4OVIPXZMudVdooTUvxhCkTZy8DhZ1nC11e\\\\\\\/xMB1VjNk5CjVb7Tm9KvcDYw6pP79zPTqUBSmWDHAmKDxd9fWdrhcLa2T0mg5mzdkSfAsotNVZ\\\\\\\/YG6JbDy9jXOd24VSe2q1VlaNqVvUpD3MTo8ZSV1\\\\\\\/Pl5k0XHg+G3LP7MCcprM7buwYguu0BqZ+Obrn5oRXzxQvS\\\\\\\/bdbXNNg+jqnTDLVIZ9BJXECtk652+kR3dop8tzjm3lLERqHyur2JNcWNFu33aXPRyddy4eGjedGUV3bbsXLpkCYuQI5GaVbYkwAqKYDP2kbXhQdg2lfgVLLvvFciGafE5ScRFAptwHma21ChfvGa5JkN4zmXBE3sHTBxw5ZwF45P60ETQ7ihd3wrU3\\\\\\\/gKLGBF5j8UNxIR4Ml1GnDLLyEh8HeEp3U45oT6StezRXgXCJtsxofTV4M8diaEwi\\\\\\\/PjrEOXkm625au444ONhp+\\\u0022,\\\u0022iv\\\u0022:\\\u0022pu8bVOe7A6EapfeieRHb9g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224e5499f9b3c9c8f71b91a256284cf264c9f7f972b87f86769cd7cb8a23cd52c5\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022g3FeeBW4+Z4yEAzFPBvsFuKlX1oHxY+UkASY\\\\\\\/VVhC+FvRbRmsnbhIAYMRQ6cIsMsjbK6WsQ3qx1HPPFwzLGsuSgNfePISzxFis2mJQQ6dqhLoE0C7k2VThOqoMrKqpNazVuafYjIEHUY6Tl1EOh\\\\\\\/GAX4BAUlekFiM7oql\\\\\\\/vm7RKtIw7VoOMByXbqw\\\\\\\/ehf1hyPCJ2rbKoutdQ76mN7naJGAyl+z741TPrCOrvENH81kP1yxYGHJUUA1NF5IYLChl2hUbTK15I9IbVPAVKtaCJ4ea\\\\\\\/LPVuC+nkgYu5MukkXmARiwo7QGAR4CK\\\\\\\/KF+qVjVRnuYk5oBCwJ8ykXY9X0GFk8cu0T+B75E9mxMNRPkrbhgtt1sjWdXuJPGDyCKTTYlZvYUiMXgzrXZYtTNSd0y7D2A8g1mMh6FgbcVRw9xo7qcjFxAOVMjA3UpZm7g2nH4kp9ud93K1y\\\\\\\/0jn+ccKtshE3ZtXPEYLgJFXjOxSqMFXZqRBiW1FFvpxsja0n+NaB0tNfhz\\\\\\\/kWKYVwuZJkLuELXYloFxnFraHBLBtPDVuXsokX3Z0dRtFyv1+GQzuwy+PrkzSEfVwai9sQCkOHckg+9uu+ok4SUDOz61flwqmtcuy8KfKNhqD5JBGUjQZgka6wyDDPtkE5OEK5YeLcjQSW25YmjBhxt74pQ1hrS0ZfOXucz8M0pfnX0FdNw6hftI0Nq+PYFEq5flg6edgr4TJ2qwRMjqHPKb\\\\\\\/zICSOsxR5Vzf3dP3o5to1nZe7tqoP7r8FrVNJYRNY5jwMGnwzVF2kAZnedaLcXpeGs6vQvqDl38y+bGKbbJfc9Hb6hIGxOrRDEBWf88jANb0L9d\\\\\\\/\\\\\\\/nUG4HRbZBQk5ATYIKaR\\\\\\\/ZSbPMiVDYZJNrVZSlZ4tHsPfT2LFDutT14LNRKSvmmk6dnc0T9qB7Ffuy77IboJRal+T9MTpZcX+4Mj6zbJ\\\\\\\/jjWXY4qFmFQlz3I6wQcqNmVmw\\\\\\\/JQl2UYdoA1thA\\\\\\\/D7vNT7GMPyiaHUl4xHm2ZEYVD9TRl31kkrbkHs4lgVX6nvuu4Vj0MtFzS9dd3LdmFe7z\\\\\\\/VzU3Tjdl4ZswthGSH5K6mXEOosC+4O9dKL3XxwtyMO\\\\\\\/do8otjXsJcxaebbDP2Sx3T8JXS+THHocSxPd14N9aFw7jABbmhqiYMiFjhANCaGg01dhCF5K5LHfoAm+iiD8K3WcKSvB285R20pokorpYCzsCbS3t6w3nlRM8ERmm4aSPH6JpIUVi9AVjZVocXZ5dyaDfKuomkmhifkXEB5yx5MMdHbvSlYpRWsO8ABj6LKIuqvJzRI4yMRcKJFsp7Dw8oT\\\\\\\/bbWdxlcgrruQ0WMhtNiPYGpFqVEYQknmXR+4ZcqNEIUsM+APIdQRQvidbSYwq4t9a4I1TFf4QciidL7b4rXeO1w6ZAviYkAblxNAdl1C+P4nfh4nOdOxBzhsA7J1xjqTjgRyR\\\\\\\/xUS0KqGz6mcUYAzo+VkCgidd0ZIqsgXJwxCX6T99K5l6JVYkhfpOlVhO7Zfzu87Pv0G5XSbFw+2eKRK6iJnF\\\\\\\/vBn70JQmMhO+ese1DD3uyR\\\\\\\/Un68Lt5bplo4UT9CcTSeFw3Tec6zOnP0n1puDKdCHynx7SWHzE3oVsav9bpSSFH4JJIuZHP+5sflguf7Uhkqilm1WEW4MIkhZs\\\\\\\/8ToIUAdIB0fEBhbxqY1\\\\\\\/2GL+6\\\\\\\/tmASUc54e2f5C+fiHsz\\\\\\\/y4Q2dq0gyqd\\\\\\\/hjjwtxi00ngblkoNgOLLbir9NZ2K3Zpi2BRfSd6EY0KwekckZNrwfGePfd1Od3yV9IG4Epzdt0zWYb58SSXCR1Y1QNY3a8ahD1gHuG1g2QK\\\\\\\/SY9ANursz3+tU9UT4iJJcQmqe6FySdjL3X+dxpE3YPVV8efh9AKcWmIAfEm8z8VuCrzcw9OHOlIWD\\\\\\\/8qlFcss6UfN4y1sJTaNKl2MaStqswEZSc8iyNUJQkJyO+ddHI\\\\\\\/XpbzlSmlvRAzizYYQme33lWvw2Q4iy0EemUUAFU2t+2JWntWS8jXcxPy43+21QPT7hV09G2nWkyoipYJ6TQmFBFXdpmCAbrFUK1DchFVPM5PD+bCRHTF55l2bYYGJT59s7aXP7CUzc5oNXZkBwiNl\\\\\\\/m4JgUa+r\\\\\\\/oFRIf5opzwCbP2oy1cyvlg3g3Z6OhZAKWlCFCj3zJMnbGy8fgMaWvFZ0TwuN9hurP\\\\\\\/DADYuCg3r1aqygRr1B7p8r4VD5nNsJvh4NnsbubQEgaPa\\\\\\\/gVLQdL++T8AwxvDnwH090Hl\\\\\\\/r1j8AKzDIBqNwAqRkLnCbrq4om2StU7WXET6X\\\\\\\/uFSJNyaFBxPRUWrsBzzKak03Uf43RBbb2WB0Xd\\\\\\\/yoG6bGNJCOPLGROrCLKyK4wY\\\\\\\/nJPsUMmEtMe43ene\\\\\\\/ygqSXvuWks3KgK+QCuWsrSOST23EihRFAYtAas7WBHzRaQW6qTUPLEEDXXhvml8W3PoTEcLENsIZko3lhGu7JNCgbTIPkTISoilQXCPz6Rnuz836Y+O3nvXb6pkCEkwUDUvfpc3w6Bi8shEbk3DLDfhw5Lk+mIETg\\\\\\\/ErTTU3TYmIXC4alX7pSeCSFAdWtxRWzzy56W1t8nswZUzYgP0X1xdGeDDfKSGgJATLJcFvwHtbn7u8ZyuldlGAiEiFRoTt\\\\\\\/62Jzf1o1YatqgrkA49H7pDu\\\\\\\/Fp8n7n5QCFi7AxpTpDQHl7EQ1d8OZAcoRhvzxypq6fzOxhA5QrW\\\\\\\/Qs6W1Ws+lKpadiZQFsxI0xwFtoY3LcKVTATUzzVJa8E9j51IMGTrBk7JwiVP5rd2GR644G85mp67HzyEzC6tT9h0Q6EHTeQifVNU8N2ZT7tUksOJ5iEn3J5S+Yhh9K2y4n8WYS0msD0poC1Y8FufAgEtXcQTSFpRpC7nUmEMjAdRKJN1qrvnu597L1HIRt0ID\\\\\\\/wlqv7m3Wfxt4kVQ9PEFdSuk9mKpdCs4VsZdY7Qrj6M0QrYF6rTpx2ATW+b7EfohdSe1uy0LKreFggzhRQNg2KKOxF9+BTG5C7FXvhkrdTY5yS3G\\\\\\\/tZoPaWA07OxguAGvRGEAI8Bh9ioh\\\\\\\/d7nc4iU+4vI3PZYpIhaQUOtuIteH+Vu5U8TdVSNUD467QoMAdx+VrAr5xL+P9DAgJt1lXTdw8CgfDXjTussGFdHhcDDii5iu6yu7wN2z6bNKKyvy9JAbX\\\\\\\/TRsELTcvy0+pHnkk8VciL+sEEBsBguaT6CZUpsvQC6Z+vK1Q8DtvTueZJHaoJPuljpxA3ZU0jSQ7Es\\\\\\\/\\\\\\\/DAjiZu8YWkL7ZpGZNwQcuhFk5LwmjQgEWTfS1dN\\\\\\\/HFbe7PbzpmBJfjPR66hOcyNdTwWN2YCDIiH9Mx2ZV4M4a5IeGCr+LPGCpNsamO3IyL\\\\\\\/tEFqCz1DS82lXpZ78GT1bFAa2WThpgpNzlCajL1eEVlnXcInxlxU7bp+laMx+mGyYGBnOa2EwGODeuc\\\\\\\/PcOLU4YBA5exvDfBj2\\\\\\\/0jFlwKEWSPP4Q46+Zsgnw4gSpCKwpDOHoNsMkzt5BXh8SJ0CJl3u9rKXF7iCd\\\\\\\/dIsOzoV2KaMDXFnVtRTJ2ADGD55tGVWX0uZIjfX9KHp9ks31CZZd0unAgS19Bl\\\\\\\/B1k0LL4CwP1GTDIY7m3IX8GQ\\\\\\\/XSpG0XOuFDBh2B6BGLMmOlfjvS+mcv8QodIhKqISzCM3h\\\\\\\/eoNJYp\\\\\\\/AR\\\\\\\/lbHaSnQfCwTLchNM4\\\\\\\/KfPrcoW7xVWr2GBpP5qwDKKN6GThM4Rcu9QrPDzsbMIaDK5HlpWy8hQE\\\\\\\/7YXQWxdMMvhL\\\\\\\/Sak1c6ett8koL\\\\\\\/FQjO9L4D3eHCynR7yxYBRZRlAYyKs3\\\\\\\/odEP5HtRe2JqsiZaIODlj4lLRlfZ4R1xefDUS8k\\\\\\\/5ItDNfi8bTLQm5yrCq4tGDPl9qvHMyO4Qi+AW5qqSuL6Ybl0+KzM7slH6iLQc1yV9wAY34Y0SKYvI3Hz4MhquMUZK6K6i9r8ImFryfwURnPm9SwwmV4EfGxECLAyZy9koKPKqS2O9NeMAiXCxQBJ0P7T1y0tWdlf01F18YsDv1\\\\\\\/NwwjzcPgM0ULbs8PVq3gzDlpH0yTFK26pSHJHDvIwf4btFuoM6s9UfiM47Kj1+3yrWuF1q6XGJ4po+LJhGh9\\\\\\\/v318tEsQ0CRK6F48UeDPlhZPIWUZmPuHT9EvJsz\\\\\\\/cQc2ux\\\\\\\/nUyRAhrEa8UlhrutKF7xgGo\\\\\\\/Pbtv6unqVHahd9OKeka9ff2fRf7rn2Omx0ddNoUA5yZiu8+MwDrt+So00IMyKMcuwppaxnYKCOlBSjllb2JR69xMDNC9T2T7Ke4LMZfuWZOku1OHeSXOnaIA7d8GH4aXvICZZFcAnEBxs1wra3EAe\\\\\\\/1PhSvT3HgI+8cpq62+HXW6FCm3XfQi1CpyaAy4nkutv1qoXcKhmaOHSNBHpyu+fEHBt8zVrnZinyQIllQC6VivzfJRqSU\\\\\\\/J7RcHc87hvj69FZyPhT4PDAw7AdvRljC4yBASodGgsMd5jxCtDF1\\\\\\\/zm6eYzp91UY3bZNoHRUI+VKIJAw8dDS5oinea1XuqBv2mvHPzkSgIIyWPoTV2o1wAim4jid9EibjiJZqduVUryMTYvOCoB3y66taSDqST8usR79Lm+1qD0duRnr8NtFCmXwFerJpD8HzzcFWjVHQflgw77x9xHv47hF1WMEUCUL9PbtnybBOcIBCvx6vlj60ScSOXurQhQ4itLJ++MuNZ7wnk2MbgO0CrxTKKgHzpWbYTM4RmO+XwTo0hf1hu64pd3HAK4lAptP\\\\\\\/jS+pdFhmyWF8v0fpicsIQDEbECIVwpykOjskP8kAF+0Ihq7IMoWa4ZTwQ18h+mZR+LmDVWqT1BaiMtdQGcDFKK8Oe7OCtdL3qE7KVM7j7bSwTpozOfagqLQLaCLf+6rhF92owDoLr+HmvOHv2MDbS+Opp2W7+JGJzqPhR1ErcXvQrZew4icfefdEDE0VrBzn0Cpj1S4BysnS4m1aKZM+qwtZCZmlX3DVek3a27jd3Yt+EYiNQlP9K76wfgUs+lFuvIi2MdwYtwQFlbRrVPHJHvtVU9\\\\\\\/MBqWhf70jv4cCSnAOwAd04cmYHm6gePQxIYikawJeM4j8qv56Le+8uCLWyVPRunMdAN0bEDNTdxI2FgWISMq1RJj8BPJnoCAtKKnRFNCiFMXPqcClSDicm\\\\\\\/6LXu4KmWkl6APvi\\\\\\\/jUyLAdzQprasAUsRg2eeKZLBE2m2ykGzm995kA9sOiUVL1JLwhEn3787a5Mo6XYBmF\\\\\\\/yucB8r4EHqxm\\\\\\\/KnFGhOhr7EdBR5zVJXZWR5PLwSloqwt9rYbtPmLpR8lTMO+NfLZv4g2noopnh7puG2A+0LhRzDXq7WD8spORpdGgcTmQiNVuUFY0u9bBS1ciOLxucLohNjSdOf7m5NEFOyj0eHY5v6DkLxfbX8WOKGTkeby1FVs45pafrsvhpZwqYs2iqIJnFvkjqtgm3fCBTcq3aJHHTm6X63yWl55HfLozU6MePtoICZgd1xGaAhtrM5CRkbL8wfPcANVp2tRtp8UQaqVyy9OWJrcw3wpyuuVPgYMazHaa03hND919PfucKm3Cyb\\\\\\\/MlOModcEznvMBv4OvO9N7PDjs+CAFs3ZdmH\\\\\\\/dSX93sUR4fTf0+8Q54qoNjhuXIANQ2iEvnA87xq\\\\\\\/JD+SRoj+F3TQA3Ofvkd3fv4cmHryToZhWOKfJAetqdBHKMtn7kcP4VAU48wHNwZuJkLK06vsBWv1P6HE9AbBf4SNpxqNhqUbM9ZeaptV7sje+c884DCVYZqghRJY7eT2T98NduIaltYXjVtnDGECOn8oGdD9HOI7XIpaaHxWXAFcNmK55OYP03hujvaAnilfaweoWtMRJAcRY1U4z2EK5F+QM6DamYKCXbOkzZDwrFcCL6FjM7ubzY23S85ohc2sila0TommBCk2kdaKLPNM9Vu+sImOk2lO\\\\\\\/g4QeU7nQQWJbJtn\\\\\\\/tTnlj\\\\\\\/ND1MGsM33J7N1WjX5MtpRbgyqE6idrEg8J72WniauM\\\\\\\/b1uXEcDV1HxklGJMhdys7RMJV9spGBplMbHPWZuyQw\\\\\\\/SGgsP6UfYhtJdydQ3HQNYW6gywAlJvnGRgilQa1BoXz35CZLx1viQ03R3UusXooBUmtVl2+D1hoIM2rACughcCuEZaM\\\\\\\/mLSzCLRsCyvtyLWj4nE+\\\\\\\/m0GZa6tIBoudu\\\\\\\/1SQawC+9lwDoXhqKCBWo6uOeXbBJFUPOuy0k+5u4+YnRxVzUEbeeWXfsIRSiFnCD74rpcKvyUWAQm9jQx\\\\\\\/StsO\\\\\\\/52xqqy3v8YD3ocUVvY++HnLmQ8FWoH44rUZEI3xoRNo2oAiWftvUqybKuB3Y36KV9u88zGvCZ+n0xxq5JALL6p+EDryrIzpclELEq+7VKH0y\\\\\\\/gjb00lJmr12GBsOFpm5nvOvki+MNHPKyRDgWCIoScSyQTLFZ0sCdXKyZ+uaHxcbf0RdzY7m6T7vGVBcFJ\\\\\\\/wQIGz+C4+4vZpBucL3fjp1i9KoBViKqL++GxpVJpbzMT5JHTsxr+B+wPmAi\\\\\\\/E\\\\\\\/2az4vseP0kfxLIke6cH+xXKDMTRMcmIlziknaWvUOUZHkT9EuQvgJXAzxxeuo2VqUnOGAow\\\\\\\/H5UkpkL6eUsEm1ncij8MhHisCNjDDZaOUjAwbuk7t\\\\\\\/2pMxxTWZdMOflSYW2w76GGMonapdnSHuZV4zVeY5LbnjeQrrIif8I\\\\\\\/Iz0VXWVy08MJ3Rl9Y72EkDi7WBI9u2V8P40D+HenqZvLvlG2JXoW++7Mf0eqUmtBqUIGikR9ARaL0ZVxp95Q5KoTcaQxLeRq7tmESCxt9LiOb3kL977m12e98QGHVjn3cd4e\\\\\\\/xIzcegP\\\\\\\/EIWifH4KPUBNHASQ3Go9JMB0xVKMMeG1Nbs+R3DTXAnolr8Z7S4RBxmmqAunIy7FCMyJyVnhEoblCzAR020DA7mHu3fX2IptfWAZVfbCrEAc\\\\\\\/B13OKoPnDKaCs1tsK4FusdxIQFyqK49m4MVdSRLYB0I26n9zAzV4QEa2R4FfL5ZwnerWn7NjYS7h80yu3OQ+JxO9SaC14MHUflbqlLagm2T+Fh6Zy9gSOCc11WwhkAxw0EYAPyPiF1B32fOEIDK+NewcnHqcOtcPH5UjCYl9Uf7KGPprBs+OJArENgIikST3ED+ja7qbNeaU6Mb2cFmXWHz8zMBhNdTgu93alsMYzCKaXrB++aUIUgaYbbpCIsUOU8V0E\\\\\\\/rYd3XJjpReborTfnx9CDSg7PfJhD+jq\\\\\\\/PKBCLP7ol6vMOcW6PADnhZiCPbSnCjYs\\\\\\\/652YVCzxZJEYW9jtVUQLzKEnHzhSACCWM5zxzgBOkF\\\\\\\/SSDuWMp2N8qtqCXqOguGBr7nTSZr0PD+b8PA+39rFyXDYgpTGXyLJzabLGoC7AXPCrAVZpaZGQ4uHVh0ZCAemT2kjTTb4MlXLAkaOTSYg0pVZ7fLLbk2Hoa5t3hLbzjwsOzqVKwkQsjCYP7H9zGBkObjTVwaQ4wI7S2OIELIE+iE6nLhXlFcDX0WrTxGqzvP7wfOkyflxhTKWyXLK05ISCp+OUKeWFYO6BVT6jQvzY+wv0VMOSrYOVdPFUhN1P4hYq9yvCrXFGO41lMib4SNYZVQt2a8fusdI0YeTGedayR1D0m05Wwg3jCN\\\\\\\/v+Idtk5XW5u9S8PoyLdrTbSx+ZRPzzmOXVgdJohJ0Y8WOBz9x\\\\\\\/yJVExBzAg2DuVnPuYFzf9AqIvLoiGcFbqqcSKU2qw90t\\\\\\\/+6g5z83petHhv839T7OuR+4zimRkpaAMEgWNn8WWe\\\\\\\/Q+Pgcc8YmSjl6GySgtftZLTePDIvcfXROqXr8kkWec4Z7OtoCNmIO7UV0Ve3oS2RNQ3+4Rjyiqdr9ZYxfLL4ULvPMzgQ2mbWmpj7SndwyVhTk47eQDPExHn6wF7lcNZPTRLAxD2Lpl3QtlcmPfohbxjUHT6MyYpB0Dv09wJ8YivwKCDTCzzMtFyis5kCZKiEGgptjAMWyC9auYcGu6GvDLDjP5mJMQk9Io1WY5cTEvKt2J7QJxaOJgQK2E2we2eaqSG1NIB\\\\\\\/9bumAasweK\\\\\\\/TZAFSzlnFhvx57kWvE7Uzaro0i9VojriJvkiSaRr2ksOulxulVUwi7GYiIAggM60qkfVPbdQ2q\\\\\\\/FGDr6SHDSWS6cXc79hqyzWvePQi9\\\\\\\/QLvZy66k5hGf0g6KbGz58pH+Xxpw7xqRFdTgzP+l1JXrL\\\\\\\/wF2U5uXdVQb6JurjgflBi2Bp3e9ph+8mhphEGh5zCgaH0MtfXIItwLVAPC7NVoSH+xZG6VZfhJX5Lz5qwvFkJiud5LP\\\\\\\/zqfR1EdP2osfhpTTIy\\\\\\\/VWOmNEc5kqY9M09tUzEjzxcSNv8dfOZU1w7gKkmh02XruDOOHWYljIApqOaqRen3D626jgaqJwQtXiWVXOf7IgLyxhptp8T80mJQPQ7tzAKMziGlmttGBwe3OmaGbMEBiwKDjC2npabE9Mx+KC+CqD3N0orEghSQeBGhVJdEGd4AnYQi5xlvD3W41BYay51MB5Cus0v2ayYi1cd6Xo4y5kixu\\\\\\\/rMWRDWkd4fqI2DTUrcOtLBbBTM+VVnSWje\\\\\\\/obQklly35Aat7Mhy2oKlIT0BFuqAHvzEraxSzwN8NvXLzHzA735nkO4DDAqTJuCGqdACo9VVSLWC+dvYmDReDHZLrV8Zxrx7Z8\\\\\\\/E6mdeLwysGn2qZZbPCa0UTlIoflD\\\\\\\/xPKgFMLkC5X0zhjigjSNS82KHlhlKBIBqAsrmK2Oku5NoNBe7uMXGQ4A01hIiNWPOnup4VgCTVtDyGK3\\\\\\\/1nv2o6gXbyE7ZK702uwRWDiwhtiCko4koCjIiyo7enPKd3hTTQIhRRxoOQZTwImWdsoe4KumlPA6h932Nz7WkHJXLya7SjYoBmOfIeKVQYsDQPqz5W2I95n4+vl1S3zuQuB1k41j8PVUUw8IKme43v1JKIF679ecZ\\\\\\\/Fsiq9pkZQ6iOc1F6Oxh30R4ixJpo9h4xTIKF8HIISIXlKmRvCvs9buMkvkBZHwrs6yUs76+zNi8SQ9+GgpwXsghmbpq0\\\\\\\/zObCnBvpH93zwraAp5dtYS7NEbwDx5+B9ihBYYZCHmqY0YIQouDK3cRa6g4FE+Q0jHe2DaY6ImjUUbOJemWL7HhsiQX72eEq6TAHx1qh8t3nv+1dR++6Hbmc7lHXDn28R1o+ftBcNtO3vsJsUtncx\\\\\\\/1KJBPkhIqse44cTiaEF0oc34cw8gdi\\\\\\\/9zXMkP6llV+9gap7p+uFnHfNguP+U+iehPfVMjI2fc3FZgs2QlIGj6urwXuTMPCCTUCVjUy1+3sQeUsJtHoVgdDNN5+D5v5dGIsjxUPWUgiHg\\\\\\\/4\\\\\\\/QDFcbX56q81odxA41OKojJyX8ycvo8NPOC4q3jqOQUB1OTyJr7dIPUJUoTYk8yn5ATgWtjgp7rC0Fa6kyx1pegi2IN2qngkJc3+3uneJvICrKHmTqrVhYp3aagghukJ+79\\\\\\\/itJqtJAaKYIOZ74123P6iQv7wlPpjaZHUGHZmoH6FyvO76klhXHSrvwTSCn8rPH11vtALZWZeJCAZkU2LnITkiMANu2axAyKgepYwJDrfqf9klveqK1IStCb4Vo9xRzNUnXbUMaKYCmreSwahCaSdwL\\\\\\\/waZsJf35xa\\\\\\\/78mTlUv959mV4BOUg\\\\\\\/uGf3V34O8TlmAipRMfBaVXbv974n6eeTkylpmLhlQtrv3ukFTUQLFq9DteiSIvWxfgen2nHIt6z+3bqrpY5iojOQByWfgNMbNZvmE7+7Dzg9Kt3h4HCMm6E8tr4pcF6KXOzG\\\\\\\/aWdTfAJAOdT7Sv+TySc+ox3pbM7qClTeP2XVM8vr+FWWJWMB9imxnpaPPqB\\\\\\\/WK9yw5tv6rkXSl57QMewqwPfN7i9PeGa23Dn1df\\\\\\\/OI+MHPsBGfteAgHHAIv7ouUMdZ19Qnacfz8OfNmtomM74RPC46pPQmR1fe69FQsQ\\\\\\\/D8ZbVTfbG7vVlzYXNPD8f\\\\\\\/HsjNtVGM\\\\\\\/EmBtR6BguAZ71L3NxL0jVkgrylEyYF7ksh+9s5pUQsVbClu72Tp1JonFri1Nrgp0xhKNhLf1ql1bGNlZUjipe7fiiOURjC7Ff2Hanez9yeB52L1zyHCXrQ+TrLjRovKp5VU5W859VNBsVxC+G9alqdPT+IasTVxFBkiBy4wLddlfCt1MYXijXqvOfc+8xjuIiVCDR5UvOO0WYIiN+pv25aErJDXcnlNKuBec4MzQWzSevhdE1V3q4to5om\\\\\\\/Bhym\\\\\\\/ms1pdvHy+\\\\\\\/xlNxMElwXk4Qv1Cygar6+Ap4sqC7J1gPnZUR7Y\\\\\\\/mOecVdQtb4Ovn\\\\\\\/T1IjBh6b6\\\\\\\/6hnUXIMIt1nkGkYK+daaCR5v9MOmv\\\\\\\/N9wQJFrClowu4REdZN56u9hiNBC3DaTZYBDOl+jU1IC4c3L8KEXDVR3\\\\\\\/eMRjrjqCszFqo6NU9CId6kYl0YFh7fqdHb9yya0Es\\\\\\\/cQ8nfvAx3ffT9jxRyPNqR2Y8um3KpVcmsSIwez2+TDpSRrAo8NDCqyk230+r06Ubm6qFGX3c8YD709KO3C8PdhTv3kfr3MrgIt+82ZlwsX6SVh271uw10FcNfFKiXnXvpseIBY4LrIYp0BUWkVqMaJENvQuD2lpsX6vyDJuurR3p9LOrwUffnHtQqzMQjYg0s+tYfODpR7BSLC6kerg2aikJbk+c3lSbm+Avn6U4mn20Fb0JmrAzqDMTKXlPhJCmtrwdtL0MBx4xDBgK5IBe62icEZvaxdofaMjBMAuhd\\\\\\\/s69NOnKlQtMrtipynCw1BqyND56U5\\\\\\\/rwdV5usSXabXOGunRJnZrVSR90rwlCKMwwzSq4+ISa9DM4NkQOh1y8SDtvSh7o9opaZURA438J248hZjdQdCAf6TNzoyahNzMKkVj+OyKQQtkBBZb1AQCUtQoSMYLk+klKq7p5dikSRpwlHBTkP6nkES\\\\\\\/KvyuA3AWGq\\\\\\\/ymcUY2S6JyHUXOlxenFGfTjD8Gtj5ZDBZMmcQ8U1Ru2g6lSQpkxwZbikPaPKDnDn4OB8zE1zIq8Owj0iygljJTOWkSYq1mVxZQaCQIRjaQzMFOBHU2NiYQdOOlBehU9\\\\\\\/JJv1ehu7S8Ux\\\\\\\/xG+elv1yoat4bfkJJ2V50\\\\\\\/nLUq1uP0VgCvHvN3+D2wgNcOQ0mMXeG7lrlUe1q33UiUBIlhQG6yqFf2FLnDPrw8xzCn9paKB0Mb8KaL3v49A8f6yUSJ+ADXEW6bs3H4aX4AWPbGoJiKk43\\\\\\\/GYKF6rHIunFCp6cqER4qy8EHvrQTYuV2pY2gk2EWEgk2\\\\\\\/kpWIB4uJQwmipTVW36QeTmRT9xwRs330k+ITcBkmtxXGwthHqkPKTWQ+giyoz3wCOTxZx1SQXbkZsSWW19k65wBJ4Mec+DEZJLsbx9khpglxQGgy8VCHbNlmGrfESaS5hrTG\\\\\\\/jQ1sg7lAT+jH89Y2yi0iK8st1d3oQUNkj3DlG6vKjY09HSHI05I1IdKa8N\\\\\\\/vglENnwnpMFSWlNXeiO\\\\\\\/5F7GSh\\\\\\\/0Q5JLaL\\\\\\\/Bj+Zfbo\\\\\\\/mtUY\\\\\\\/WeJVu6ifiAjmMuVHRm6a9f3dN74R1kI+f0UxU12RcNLIximTPnfq3Blf1LV58umi\\\\\\\/k8M3s11C56KdU3G3PSA5bDdtpdfriz1Rc8ZK0gH2dV5Y4qezoPSYcswTEDEaYjURwElYZjFUX\\\\\\\/lfuQe5OXCtv2VzL3Gs9KmFJFG08T\\\\\\\/8HOlO9x3TePr271SZ6k6YtdGxLp+xjsBWvW7MnE+5JzkPV7lhIhEvnBTIhfRDDbXH+XdmaPoPcG5RUXP\\\\\\\/lIC1yvjSXY9SYmKGPM4x2eIa8pm8ZedV3Jp9lb3GJHGWx6o+jHLsu9LALyO0Lnw2PyFJNFgtVHrml0izHlDZsMp9g2ToOYmP6jUHOVet3U\\\\\\\/8EMmYrlQ7LboNmZH7eRZDiQQg+atNVe1GCVNHsEZAPAPBu3gj+X9BtzAk7nrh7fCV4hO9nb2gafg4YG88M++QU\\\\\\\/29reG3bUlFInCr4fgFeHN1FAjdxE0lCZ2TACsEyVxZ2U6EP8LNFjS+BsCnKYK7BnWsWWQS\\\\\\\/xIIUql8ixDBDcG64ixAmrc0KcQD1nhQlp1l\\\\\\\/KgZlbF0FLo0CcUt9EBDzahgtmmJrvbsqzkGLWHRKzJNS9fPuA5Ll+kewcvITG2sCeF\\\\\\\/RtOSl0hXrnEKroW4OrCqsVWDulyogFNhxfHPpg\\\\\\\/+KHvERpCIuPslnTs5G31rBeELChqP5d+M1lgor7o0mFE\\\\\\\/XQU12QjRSFqJn6NyEPIDxd\\\\\\\/NuqzHOylmIXbu\\\\\\\/Eow4+mx+wEVHh9OKDfNTXh4KHUQq84PnkJ13xvYudhxZYN0BdXbuTR4NOKKoj7D42rC4rib6GW6rDbORSSq+wEj38rbOhpzwWy81k8O8u9Zifke52jGFqtWwwuw1w6o4Omd7r1JC8e6aL+P82aYuh+l\\\\\\\/\\\\\\\/98c9pfGvHNwVm7TP0Y+ho\\\\\\\/N9vHOvTflovmIBWGuRJMBHkyGJxgg4vNqb4z2rgGtJ3QkKhxYQyGbqTg4+gPz55rYbVVeCclMWzzW5AhL1UhcRlGQMVoyD+jQrV\\\\\\\/PNwnKAgXOUDd8nkAemW0Eru0I4L9zAhG+wGxRZkmH5PBJrlpjIwhTn0zGE78quZhstQR2Cmja8TVZNbAZWne0rsjKceopWdSRuhRKO5hcB2VJEHTBbAFYnLphseSfUP6tCennlkQjQ9ipM83zFBRcuJHOMEUQ6vh4wqjXu7WoACq4JkBtYT1yBoOTfUjm0KzBT0F99ONJuRrdTypb0O3LWMGoop1bLygg7nshu4KlOJtgyktVFzaRjbpBklSeZ7rhu6uTPPCA3E7TIqkBqUYdFAUoURLUELT+kgxcXyR5Gjdj8K7XmKkqEiLk3HtV4F80D0R3QgRkBuCLk3+f38g1a5G2H1FHJGaazlr9K7v5BYBZXeB0Nf7UpjrWpprDRHtwXPg1gqmz0uGlWor6KWgnAuEOG4Y\\\\\\\/cmYfv82t+mQX\\\\\\\/IbgD6l8z6CW9jQlgy3qm8ykUzesZjVWgyDuSiMMrWSkdNlnHc3shtCqOoRV2CYMsUw3F3O6N1qrzE0zrnEffISjMrd2UhmOUfcVoDBt6TJVk\\\\\\\/tHtDft5zCOHCZ4kI7Hw68lB5pH9Wpj4+woPR3qfNj2RCjnsaVafAy2YhTox7xqhHRbyviiazRY2NeVnRa\\\\\\\/U8sIukjd6BGPJSMUvBotlI0LBmsmVraWUBaZwMWW\\\\\\\/dnyHMhdWTp5db0nyZMcvTtu+dv+srcbk4vE2fwHVvfJtj\\\\\\\/m2FpEgGQBo3DL9qbk3UBslVu1aL5vqpgtg9qsdXskCE5GvK846mBrvOgHXPd+k84VYeKHCOAvrDPpTQTaGb5r+vlcH9oZGgsQaVLlIDCifk9niXhJLUZ8qYqHNWj8n0N3rCVOJ3R8Ka68YGcVsfdkhoaOzG\\\\\\\/edOUlTqrITtBItAb7MLMbZtM0OePJ6dX+IP9eCi8EhWV1tP6YkYeEm+Q1uMBqiqKLGeRF0O1V9kWlDFDCD+SCCpByK+tvIhlBPAOH6knyWhaHYophtcin5sVyEkqCVGBC2TcqLuUhhNkAECxrUwUFLsSj0vXjns4aW91CC4aPCGWW\\\\\\\/8HMewiKIrg9nkiF\\\\\\\/K\\\\\\\/iq6kVdZMNmiCKWFRLRatE6WlqVgSBaRWgYYqtWoiLnZWM0rsSzXoavz7WDNRB5aCen0LNhXYf8N3hhZaRmDDQ0OlQn9rlzPO9LtS7jf4RKGzYCLt88sflulHpawchme3B9SJJUkuo8vQryELWiBwtz6shJTwDUgkd2x2XRRZ8UuJF5CYtOfdCA0SKKXMXWUM\\\\\\\/XJ+AMJ\\\\\\\/OG\\\\\\\/5NbXofhv0UtTamgC8K1t84jcZph2W3Lt4q2Blh2c4hoXLdSCVX\\\\\\\/1IROEeDRwmrGByF\\\\\\\/737XDga3APDJhBCcU\\\\\\\/+1X5Vfv7XqJB+WgJNukH+0jlodADtSpaaREoyBjq13bIz+W4mdCjY94gLmazb62SlM1rH5XnCLObwOB0oTcW6baDAc2rUqtI0\\\\\\\/7HwnSSGSukB5Ht9KD9Ixnf08WArMsylpl1\\\\\\\/vuiR+OIoHTYHuonEjXEOOfqcmooawvOmNDPmVn47DjI87tIDP7tkMICHkPYfl7ZGHcoGSqk5f5+tg35YRGem3LKwsF6i7RlEASME2kUPXvvC\\\\\\\/T7kOMUL9srnbdzVoVKBlYzKn\\\\\\\/I2dnOaAoQRBNONpmXWCGMkG8jU0mCFq5W0b7JlZOBpcWOH8FJjkV1iPxY2NGyxon9tHLGirpZNIcsjvgmUk1sJd+Z8fCpGQmyGkh0SB1x+9ixVoinNVaZnCTqb624ARNcCp7KZGo9wR0+boJFm7j10v6r9m9E2I8sZZiZzoaliRwu5+T2TV5li1NQgq6VCKajq9vuSSox3WRYsm\\\\\\\/HWD2oh4X79UJp54\\\\\\\/KeOjJL3H\\\\\\\/29LNRJB4PljmdJuWhGlbHm2fKUOoUqGyNWND1rlLvlhNqhHzR3Ue0iqlPrLacsMKQt2rRRqJv5bNujVpw\\\\\\\/gNL8\\\\\\\/1epEWuRSsW6BTmKsxCQnlQdfEkhQw2zWuJLwAxGEtI2OJ5KzrN7n36XLuGd9v8G41di8TbehxW1uIZOLq+F0M0ne2OU08yQf24dtaA6tWR\\\\\\\/v1MclTSlGXMEn\\\\\\\/idXykLoF\\\\\\\/VquVWhsqDCYwJBrX34lBAkYsauyQJQnZIMrSwJGbcTOb1k8Fyto2RVPsvQ+2SW6AvBCMB07pxk\\\\\\\/1UACbJGclR01hkVU171B\\\\\\\/GruBjfJqOwE7\\\\\\\/HJoxFlg0cHYBhKwkKnHulJW2mLYWBeKKle\\\\\\\/VDaRcySvp6vTdq3RdWBG40rONM6c55dp7ox1JmEyWFmJIzeynpmcKBjsehBi3u6BdNEDnBEiYzkRhZZBotPSeU\\\\\\\/EdmYTmFHr6WDLQ\\\\\\\/XSPXuu5zWk86XDrkdmtzUsztQlqCQdjYsaV1RzAe2dd9z92sjYe\\\\\\\/7PGMn0oXYQzFtJfhMIcNNxOCw+tgA9Zqi5bszSd2ddMm5I8ADyt6jObGV4LIq1fJX7lBl30dwSktkKbj6xDaMSm++yBGeyCDeTNk6IWSp9+yy8uOzur4CkSf\\\\\\\/YZKbx7pdmmm3wZugWK2Jeb9ATe4x6oAtYnFr5pUAdpCNtym0IHTd34jBUXcVsbCWs4ReAPHG9PGsTf+PZrecBu01zKpXhvthRoy9wtOLQFjAFYUabdv01IMgAJZo3HzYP9QU\\\\\\\/UX08qCspSYoXStimLtJkzZQn\\\\\\\/03nIy1RkjxOEoFgUVRwMq+iRw2o0b3PsMlJk\\\\\\\/GcesdCNCq5QdyInRVbpmjDlK4dAGnnmzxCOzc6p787Zooh2qqpcFv79nvoC9BF3mecamIwUksLHVrxv2HEBg8PpDSa89Rqya3KhF77j3OnvceDqDpMPDgv9AoAJlNzEwmLLZS98uhSkJ2Cbz0tLE+5O+TeNXpBIghLcLafqV0Mh8Vma00fmfcKUOo68yMh8y018dLS78dBpxJkgs1mYPJvEo72YSfSpbaNZcPP78jpfIeWrCYvNgIaQ2Ta8aX+0XRpgm+SruE8dwrotP1f3nzkW2EIL1RKP1JSyeon\\\\\\\/VbdxbcnwtapYx+sNvLTJOOPiqc\\\\\\\/0y5trgpGM7xIUGqZHe6LJIMMxe6DLawVwrB0mXBRmTLhL+tV2H8uf5qIV2DJ978P9frR64etMyOt+j03WplHTH3OU\\\\\\\/R43J53rfxCi4Z8BboLpyTpE2rkMRMgCyuGI0nYQiiDzR9yrLWypJKIYqEspslxfhe0v8x5wfn62RkQkhjo6taM88gyk2OQ6LWkPMPc+0BdiDfR+zh4HdScQpstY0PhRV7oZ5p\\\\\\\/jBx8rYBgnjoHa5MbLMYKB666nmDeV6AuBIBpGz7Aw6HrcDzxpkTHjrLfhM3p9zmmQ3746OmFIP6ab\\\\\\\/WqmST54IoHDAsBVDiUsBVjN6ajyLTwQZRFI6757iGTm0YBcWsYmwiG8AGrfsXLz2WK3Y9n74wtZlx69RUT+ClQEACaL6MOVNG02w6JnzOggGNE6w1MrCcyzpSAADJd1q5hrl0Sv7MRwUf+OVNQ4qxHqyyxNEcCI+6MSQlY+KNUFNXCVRUdvNosAz2k+vTkspRLpYhd+DEUtjcY7eUU0BBVMpC4osTEVGYMZ5kK0VJ\\\\\\\/j+FLJVjSP6Mxu6wFwl7DB6CnkeH+w5bJcGe28edAjm7txSljvkLfwHbAaQoTP+iPI5m2xlJH7cXk3KCzriotnRssPe6JnoYg656zl1E\\\\\\\/BWmbExDf1WuSH3GRpcKruwMOEPV5vdtBXJ8AovtsipDyb9n\\\\\\\/g7MZ4vUWbuxkoF4S+S5jIAcNII76xLyA9knI4QYh+I+xLruV5b4kXLrjGpye\\\\\\\/YhiZGm4Gn1b0hZE3MXF7nm4yqkTbuRomHrk8v73SAoYLelkzQn9LggCHH+Ga8PWpf13DDAkYN9PGxOpvnTn4XY6tLsvtO4Bem8JAm7gcJ1vWgxOdLsiIqEr7iDFprldG1QgYTySg\\\\\\\/xf\\\\\\\/06GkjgJCwa8yw+noJlgsYZyac6o1+bTQXg46jhwTFFXVEk9oNwuFcRa8Kc7tMaw5BvlQC\\\\\\\/bvsTI4SJDENH26and21qP7qTJljhSHyPebe6a+vwYhSF2HbOz4QltZQOzYkdUyP3NsRAABoCXT8OAdy\\\\\\\/wY0s11agi4VQUShM6NpHnEdl7Zs+cwLJb1hNZj5oDwoE7H\\\\\\\/XCbMFIq7FN0rnBKsLTOTwCqcAguDBDPMypaBRAwDqTof5G1In+VzItNNSMCxe12GXLfd\\\\\\\/5jEVN5qTjCMxkDuYsamZlTcMpZU5FrOfVsrsK\\\\\\\/zLaJrLlEQG+1OLocRZxSGBTFsl1\\\\\\\/6ePl1mlMUruEaTnqKKyAObLH3yjS5MmMd4oIMwDXTdgOPtjrpfTpNKWB8TOzjGRZZ0g7AWKtmn1cOzaa5y5k+ba82i15\\\\\\\/cQWWCktoWJX9DWS+5iKusn6vxewGRrfMlEhRCD22s8vaef6zj4psKONUQOUCERAmFN7j+vULkhh0zUJw1HPl9Ifo6prPURd4cOVyYnrkcgykIbPeZEV81FtMr1tdwsCRofdLidhPmiDZILONFZR9O8Y8NfdAsNCafozHOJz7uZe7YgUUVjmoEemObY+CGsm\\\\\\\/N9l3k4jPU4gNNyTzBVBxVyZCAhBnWsES6bO2D48k63frFXuba7SwfPtjk9rs3UsdX+cgPJV4EcB0k94bNx94eVwet4a4WjToC7VW\\\\\\\/NVqVPcw\\\\\\\/ZlYP4YExNZSBfklGuD5ggpED\\\\\\\/bDpkkNdEBYMiDkgzORu\\\\\\\/2YTbio48qumZD+AOeVqz6ZVCbjyM7TyPL3XwzUmpwWCTzeYBAE7dstdjTX71Xx9VueKMfVdDBFi8uKP3mWcKZmIiHnoONnxWNnhSLDYI0xDALnblGQZSmOKD8QC0TTjI4KVhJEF4eCzQauSO6UP6S52n+J5EIFVOhK13YR56wkd3pjnlimkElXlWRWc3gYlY\\\\\\\/wrANpdSdGrd0IMa2dKnOMCjQI6KEVqZpJB815FA32IE6mHtrRSEWOhqPlsSnSQqGvDni5u659kTz9XNJcXwy+JMu0LyWc223Kb+0KeIIC\\\\\\\/yMGmeJCcD1B8zCFsb530iwCS7RoJnRNou+XO4DV5e+SfDpIPjKG7P+3+QUoTL6l6Yp9O+WyHzyCcxLbaTWAEIalx9\\\\\\\/ITrrdfRrcEzJQkxA1RpKmvbgo8+5etr+qh+hH59YMSl5C6HBzQD71vbFSVUp\\\\\\\/OWDajbCnAB+EGviEVkogRSEBgV6hK7XC225c46T57XT2c00v1mbNfVwXGPptyafDKRH3v4Sigis3sPcgYSfk8LuTW\\\\\\\/SMn7fgFK6zMfJzAzp695\\\\\\\/sAd2j6oB0w5008Z295ydoayK4nAh1v649TcspU5L6KVIGbEfaGagnspUxdmSqVfK6l7HCFNngJCM\\\\\\\/60xxOVLg6hAhYaKzrZdlNYp5cEXT3HHEKI1DOGvAZnnMh6TYcTp0viU2yh7JlXfXnj1YzULVG2kFbyMT9y7uVQfmV4F3oY2YoTIxXBlr7UeXO9v170OvlRJ4MpoRFyTGWwaBfNWW+5GqcBBAgX8zJ+vOumfys\\\\\\\/GwgNSWGSYVTUH8jGq5Jt2FLUchDtSqEfUWSbTGojaAXmD0bilISdnVE74H49NZvIE9yezvtUlMULozlOm8Ra4nFS9xBu1\\\\\\\/ISE5hr8Lqv3QJ3\\\\\\\/tbfKBCdXsDf8oHcA5DexBd6gW1m4EtlZDtipEMm5QmMG03KSPEBuVMKQsWbEoQLOmQEZZ1xaDXPTX\\\\\\\/5VHy1iC1shD5fLsruxcosrdrN+PBkLgP0Z25mLpRC32RR8eftMHdHG4SLTOmUo8el+2MHU52139PyjU0492n8V33p33OQI4zs5ger\\\\\\\/rnuIq97hACNtNKdPVMJrsBH3Ew19n9J5pOvS83Jh27kmFPKny9Kic7SZh56RJ4icB5bW7rofpuT5Yfwx344SYiistpIbnocjKUvzmIUpl\\\\\\\/bmDpRiD6fYFXq6hyEsOQ1z5Y3BzuLdTu9qri\\\\\\\/6LVK+Fwk7CyjehobPoRqZlsoqn1VXi7rHLt\\\\\\\/bdMKnQHfmXBRLgtz8ML+62iRtpV5BMsTHy4TwV+cKiWKaR6dBjKrZZeJMUEA\\\\\\\/n7L8YO\\\\\\\/TP4fkMtVUZVgoF2LWi8xcIQrNa5QUawvzF61bm1VbSAUqIl4sVcMUNRZ2SQjFvYtkri+TTqxoU7PHCdLGLe4Pw+aeowR6\\\\\\\/vorUDMTFiZLaPBSUrvX8p\\\\\\\/hWRQZcgA2LiqTGLts2SlptJ2DxSyyI5+8Ud2bi52ZAqEfYY4EtHoIJn+o49ghAaXCy5E6FtZUsUYnr\\\\\\\/lzu+LdphvSkGh\\\\\\\/OJGI01xS31C1hY2OV8yxQ1U\\\\\\\/2qiaiF2ybIc5BkBFSRvPAgWd45JGtQIFq5SDnESi7ev7+1UZeULrVgNHg+m2kKMCemNfMh6nWUPGldI3bjL\\\\\\\/4IzXJhpYTXdVD1XGkz6AjtMryKJ9w0SCJQPu4BosNEuPVA2ADxd5OIyjDR33EItm0MpX6N1dszX7\\\\\\\/JQUuz\\\\\\\/2jaIVlBLB0ZsZw0vF4awj2euAI9VW5KGFvucps\\\\\\\/8h5fAsCy0MX9uamLNOtjbikQ2WktnuwnYMl6K9Ej4BksPZ8lBm3rd5XUmrN6icAq8+IQkebpLMNBp8GuosLeshGDwlzroVjNlTK9Z94TnoomCZu2ub45S2avxWgZH33kqq2TsjQmk36P9vbXATWJeX26zCZ84NgqwW5tD6K\\\\\\\/atLoyCaTIRJ+eKzRWDK+ksCi3W397zR+Q00GYhkZd4s\\\\\\\/+7n+uJazwkMTH5NqBY5A+U45m\\\\\\\/HHDjlRoDX0\\\\\\\/0+acJfq9\\\\\\\/udYRqLbzQiB69Qgu07K1Vp\\\\\\\/NNCnByvisZjnxmXyjEtxmYl5vCbRQU\\\\\\\/Q2nYUp8c+IiLTVnMCWIlsMm055aKEuz6B17md7cYvQgxnYzzCSdIMiWva234rX4KiNUwx4vaLLpgT9I58KzSdySSF7j\\\\\\\/j3Bp98\\\\\\\/gibeYdLt9X0WIzl0jgbmtqQY4yrQPTUwUZR+bUyxmjLvNVTiRnwNURLFCFVwC+gTVR92wZb6RfuG2ylvDWFtOzgp4NUnCoAmrkREN+BaIakuzBmC6ZnjujWvbvIjqrfuBaj1YuuaW7T7UK\\\\\\\/mOdv6aQykhQQQyIBIBc8IbOtvoH8PoPphocgH+B8Be5XFaRIq2SX+XSQ9TbAP0KaTUmtgzTVARh0sbqSxscxrBQczLkf3pENSmi9G0nW1fZla8DiOH4DyNibxeCDwz43afCOpAEwUMj091BXWoXOMScQ63Ys+jV4KjXbnV1I0+KYucU2xMYiXAWOBLJSjaaj9c+nykDQatvLQ1NO7Pqjl06bBJQW5xNMZKFEHEYC+b1TzTmiTavYya4bLWqYstlw0FuTNfslIRPOw2CNqBLWWZ4ML9PNfs8IfrLUDZJTSJ2IWACQvAD1Fsp+4h1R7R7UM2iUf6OPP6OWBW9LnCDeob5Co98t6oyHXagiS5c8cLXU0F0brYXFQH+\\\\\\\/S2Lrsc+JXpc3PZ2d\\\\\\\/gJG9Q4erYF7o7lFmIgJA7TN3sK0cbM3w4w6qNUzKhUooZ3IkzxpIbNO4QVEti6OIonJ1nh37uxl5uBrqurgT+FFykMKWfKToT8UWXmWQi8mYWo87Q4Qsp+Ksyr1HrJCbkxjwQFRvG5xMn0Ze5PhGgIkWcycz5dIU+wwtCyeYWyu9kmIM5DpF5P\\\\\\\/2tcIiF\\\\\\\/Lp5wSBgYxZ9UOIhv9nK1gEFhwMnkIEkpv2YlXw8mbGXbwV1\\\\\\\/opiZNd9uHKVBEDG9qLpI9fZ+OdcT1\\\\\\\/3Rz6ZCknL2Q+3ssMZccaviEJ2Cio2WpArbrAs8M\\\\\\\/\\\\\\\/lslCzdJF803bNh4daGnn57vIl2vJAkI7XcV3acTQ5o\\\\\\\/5YXvC84D\\\\\\\/lWKbvCaYYF6B+3VcLG4axf1xzJ5BJ2qbIYevZWPSJkYn+rqKtgpDnm3dgI4QaEHMQt0gv9F4SrX7xhG7i+6MtT8qnJzgVR6T4XNPI694im00i89+Bz58HNSKomkGCMi496HK+i0GgFWDGKKjKm5LPJggvulA2RwQV5otfZdzV3ZStZw5GXHhe+y30fwwN59u9PJW1w+cZlOKDMOxFKPM6AAw\\\\\\\/v8EbmlXFFILWdUqYH6GmZzHL2YbR51Tdn6wx2HJUSnEMpirE1lpQzJqpZT6ul9BBSDI6njsFSLEHuZkmjeW0GeKskHPVvyWqzeCI6rF7aKo8WUrxD4BCSEXpq9WIHYQW\\\\\\\/aSEytwOcjvzuqcLnnn0GbDezZZC66oU0Y+rhuhtFyWfLQVGXbBHFks1bLFqzULAaRaBvqWvcpoGQupfnluHk\\\\\\\/lEPzDlSSpQobBXGHire4xo8HwDcORPZjGkjp8nBxLW5fq4PFaE1UAA6sAkkNxBisiWRWkD4f4qvOai6TWL5b5QBQAa9U06f9aDcAxKmRhJL9OWR1yzxTOfEgkkmM+I51wAjnSPJhSmkw0I6kjw14WDmY6geY4O5V0K45juAYxGmypYwr6rTEtuBRuVmL0DHWUVpVol3a46obwYUjF+ykktc7uM11UlfPZIHmwuY7PC\\\\\\\/SNNNi4sufrI0t6PqHjcWbbsAAorzUuCAzHYDu5hjPnB3NMbxC42e1vO73zoGWmu+tzvR0fWflTJ9cgefKLp69zcXnr\\\\\\\/9kt5Ye45+clbb\\\\\\\/0vAnws+0miN0kn8f2N76wtGb1QPd32OII\\\\\\\/O0xanFYOkEWTdfC\\\\\\\/kOYju+W3W1eMfMdGD3y82VLZIO13CuL+eTEKUnj\\\\\\\/md50engzExsjE9AqdAl7GX\\\\\\\/72m2gwpppHL+dyPqWQTfSXyxphZpj4sJecYrdXqM\\\\\\\/MjhBEuPDWXtfH0A3mFjvrzr8MY0hzh9vkbtltKTp5xx0+Zhy9JTX+vmmwEK3Rvgsniniy+GgHjHYeQ\\\\\\\/1gdPeUwSaw572AeHkW0hZZFRF0AiJXusexVRJYxkjK2emTlX5\\\\\\\/xTCMMwCAUgSaIQtJvuko7ctfybAlBL\\\\\\\/9I8RP1d+kkzyrRru1JhlZ+VkJKpzI9B7egcdFuTnCUT7L8CGm0Sv4WxKuNJgaobUj+CNXm77kKdmmErbkbBR7PtYoqJesNvsWgl+hg9SbBO+9hJD3cKJ9bTecfIRrEtupYjoGPNQYJC0B8mXMiZq3wYkXJ7n3iOjCBUAF0HbpPjUNlm4Qq4DD5S9paTO7jxTkQTIbn+NQTgPQU24Txh3AqrRNjgDCoqZrIzui3QVGxjPO5iB\\\\\\\/x7HkXr\\\\\\\/pLrs3BqPIeUdWuVArYXhlZjQrJscN+l102tlPbjD8CnsXiWaUCP35PABJVRfShA7I7tmBA8hiM86VMN8s9OPKUNTK\\\\\\\/BBPr4DYjer6a\\\\\\\/\\\\\\\/OT\\\\\\\/LExeExX5+C8KBU+helQbL\\\\\\\/Ir8i5ulcE5ROsc8a5bfHuQl06TVN4uJvejnf\\\\\\\/I70aqYCKdm2ytKQkwO1y+a8IfZa8J72zLobYYmoNbgfe3U7w42V0m4kourUD0c+rln0+1vsIaeemD+Nn1ZVOt4hIrkpSrTushTp9LedmRA09BhPUB+fP0yo\\\\\\\/3YykEyjvWG20dvtTfdlqdZIulfKtsHY9dU9nIIinSL55+hTvrUOTAbUVqMOVFutErHNOQxv\\\\\\\/pyWVJIzCUQY4pklRxFyrbI1qLL99DcV3prdtZHH60+\\\\\\\/U0nKqlO9pvnVVdUaxZPXXDoG6DInKkNsWzhmR5kP1HOX7TOJMnHBXp+84C4zQyA6U26pzGxI\\\\\\\/V+zQ0PbLuR1\\\\\\\/ov3T4GRaUvyRZ1sX7ox\\\\\\\/cwzyeUl8GwC5JZk8PQgDSKF8f4iEIQdlYSh0JTfesVbCNGuJn\\\\\\\/oXU7Uqs6181IjXCLpfHfCsxJht33gAvCpzz+tkifd2gwKgC4FX0WjKy2bzLV\\\\\\\/wWbsm7ZcNHaQXfV2oFOKhpYl1+tWoZX7f1AmWqPJRe9+h6e4jIooh4dZQH3coNxVF5Yt\\\\\\\/WZjOxEYa8i\\\\\\\/bOtNKVNgxVQ6fjbNm6TPhy5cz4qpsGR0ECrJNPND4glLI5MNo3euDgiVHP1noTyCyAhLM7Zss0oOaLYRKMqIIo8hW8Ouvv\\\\\\\/FHFDWEOdUN157lDxtQ4Pib2x9Ou7NniLRJNI\\\\\\\/HDBEniQonHY0gDyNvIT4Yihaqjgh9FOXGWvbH\\\\\\\/rkJlp7wpHSuQsmX7jSknE4jNobWUl2faYA8rIenSUhugg4zi3+VXz0PgnOIVgOWMU6iZQlR\\\\\\\/UPiWqAR6b1Cf+Qgu\\\\\\\/18Ee9uP4Au3NzwKzbXzoyigWLhZbRHbAKWF+cyiT\\\\\\\/twsSglNRRCj3yNIww5Gas1K9ehx9dyH78\\\\\\\/b5RYHgY7hK+m6FjWI57p4ayAvgjU6LOK9vunUQMxTuJQ8XB\\\\\\\/kOn9f4lR5vll4RH75j\\\\\\\/pse7wAhZZK97mwprCZR+WVuJ3lDGJNNUPHCjapztPAl7rfCLa7bT0uALjL5pzdFb8eKvlTiwz3+fExA6ALJUqIcr1t2hs8xCHi16py\\\\\\\/NgPimNw6d8i13JqJCwmH\\\\\\\/qLiHH1pvKkZAqKzAGyeYfQSKIdgHylaWrxmfiS1HhHI7B\\\\\\\/xuoelkVnW0ebSUdur5FNcFgEThK7JzlOdZx6nDkzLS5E9ObnL5BzB74d3iGTNr9+SRJZVcvKp5suupzZj3b5d9S7wKAP\\\\\\\/+j08lSIzApFQog+IgBO5eMEQvRgcCsL8XO9UYS3UQcCdZ8ggeRAdD2YmfvkmDpfS8bbvwQlxVZunsniGr+h7V1NjSmI8zFjO6IQlYyyOl\\\\\\\/gT0MB0RkrpYgLcRdo59PgyRWx6xAXy\\\\\\\/YJPcsdKJAw7YzL0XK6QeE\\\\\\\/ofsY\\\\\\\/NxRkE2aiYoWMpKqHPKj39mrsVW7tQ5E4I95DgXHx+pYJpNSWiE77ZOHW7CQeDwPhbGg7AuUIb8rBKM3AVs6gtvHoQFWEz92npmPQSN7mc8jSsBDHYsTe5XnjWxm8FIIpibnvmjaJaJk8iq+wwfpanMsjDyiT08M9kcM+pnQ0f+EGbFWAJD51oc1xVsbpJUaI7gFc24k+osOYUfOozhDd+1Pdom6toAldfXmxQzzEVXMq6gb3dvZe3Zndz9RtYmSoFsliEqgwwZcmJ+OWWAX9iJYG3vyJfAXmUvWTVYb7L+PPkB5JqiIj5dLTFH9oZOyCLpn49WmhdBUygJIMUd\\\\\\\/dP3jX0CB\\\\\\\/0ctiwW2UPy1qq0xIbwGP25L7ztA\\\\\\\/SwfmmhAZmUhNdzvDj55ClyA9bMN8jDeEmhoj8uTex607zOuxjDzXcpZgDWw2PxpRaPlgTpd9vssvtU3YaZQnBze3lPLEJfawombgQ4Z+YG1LfPiBS1qcB3mB9gGQQSKyD70VIVAJRxEuHwDj4lWSbO2gCHR7RuXjaWLMdl6CPWWfL2Q2NqBYUq08RDXkY4XGsSm7sWNXuNm2Dql3W2CYKCv0DTQs88AvIDHThjMjCkmxd5bs4IIvy4hGyXbisRmkiFbt76RAbcVnABJ3NzmF\\\\\\\/il0QOIzkgG36h\\\\\\\/49zUd3exRO4CrfcpWqnRBsKLWpvFyhPZS+DiHnaZ0NtNAoWyMHbpjNuXu+6SnUkP9dKeIB4TExx\\\\\\\/KMg3zOb8S3z8ocI4DMEh8dyiIkADE7vHK1rtikr61aUO5R8yDBAL2c03iXJylN4EohAbbdqrYAliPmIjo3IWy93VOoOMWSPpYrr\\\\\\\/yqgKSeDh4yjO6MFas0j2GKlHKs3\\\\\\\/i5qGTd4wdPAtxEQ3H1T3oPwcR9gE+6tNxKAxx2eXXgU4Yw26Hr7ih66IxwhiPQcXxHub8RK2JTRlNuHKvH33mQV40\\\\\\\/99XpGR4KE03K0CAqbP7b8GBOzEJrEa4u3GSrOWkcYPeujaDWqxqHQpQWFPP+EgooBy\\\\\\\/w63D9XNPOZXyFiay0YY7boAs+x2OzJpxCCaLcPNsHxnWTfYwrNKDXKe7ZfA0B5avgwN2B797LmSAKtJnJeCxeJiv1pvbnQqTU\\\\\\\/fN2Oh3flu0YEXMaxmoIDlX6sl62NDF3r8iJDiAR3Eqyq5XbNXG+\\\\\\\/1yLKtT8eN7wqSZ\\\\\\\/2tCRKDSYKciqy0GwTr1oGrlkcdZjsHJxCBXQ4\\\\\\\/OknZyHtXhbAv+Z8nCwHt4Imt9nU+iAoJM1tCz26MtYO+kW8o5RDlcCu6Ay6jzkkgPrwVM974pojCuSZpZJIjkvwgH74xINXDfOrmnphyKMT2gCbc\\\\\\\/dI5TqOkkZbxPxH8Ha8nS0EBH0nMn9RG6yO3t0XE0VoGN52bHjAY6MO9Ihw38ijO\\\\\\\/p0dFjYbbbCAatxzIHfvfhg5hmSlQ1hM\\\\\\\/hEYiZNA19S8wOHxfBzdZSbk3o7lZWW72UMl2913JFtIUHcy7zFH4Cfzn0UrwNuM0J\\\\\\\/7fhj\\\\\\\/qP8N4C7Ul2wAou19oTfh5TuxqmMLE+gTTu\\\\\\\/bfoEBHNAsHoNbb0yQp6M11lgTVMM+BJPrrOlV\\\\\\\/nnGpwzclptYhA2gAsupoCKo4S2s4uuPynJdyFsSxDNpZ1Rc1lmcPDH+OesJJqWE0at1qLaIprmhKK0X9UFCAcqkvfW87\\\\\\\/OrqeFCR6VpE7FuM43mBjFlBOO66cx63W+IQ0\\\\\\\/rXxn9L61X4ACLUMCroQGZ0dnd3kO2YATlgHMFb2mIoO\\\\\\\/CSsLuZzURqrU8lMn5vdV7orGeewDlxo4dQaba83QnHSR63KTsARNVwEUfk0I3HBtcgd73T+tIwX4iYbT23NSQbs0cQqqOxntNZ+vbOdAMPPQ2EGene\\\\\\\/IUhKrnGS69DkxKxtQlMu0PQItoRcDsOdpvYPlMG4TBJIyQWC81n2qZaTWPSurBm4nZBmfB25qdlYfNcdTyv0ofaxR+SE6ko2fvXTHLp3HPhDTrnqIJDh7IypefU1CLeYsRTpnk3pVYjLqvXCXjCBZoDMi9OI8rL\\\\\\\/WCmJ+AACs5+zsYnuWuCGc1zPiiHf8p7mgZvAykm3u3ZpiwUqCm6d+5H8oXqb\\\\\\\/GFRb8uAmlutoCH36Mp+gJL8NtfmB8L9i+T1SUXUWuTQJQ5+OHmJFR+YXA8rVeKWZ1jC3WI5dL21BxYxqkCsBB0tlkoBUXNcAvML7uf8hMDEY6WrJWaRvGQoW0tyS1jQvGa4lZRvgaFNDWKAomB0oZ0OpxnCyv+1J9lYL3F3eZHh+pApu0Tlvd7SbJB9kfXqEytIRUohwrNPFz8M\\\\\\\/jNSXsAQAlI15hV3W+iTItlfZh\\\\\\\/wBTEEXhlsyR9LCBSvQ8OmZCUd97cWppOSHGGU48AQ3hvYIce5nr5PgnWgzG\\\\\\\/Zbiaaz5DjpolC9I\\\\\\\/LMA3cwoK6+mFl+hSh\\\\\\\/4B6yStmJQtgoBiX3w01\\\\\\\/0zNl2MhEs3oO+ZZVhRJ6DtSXliwYfEUuz2M67IDombUKnrbVCktCfubwprMCOoFTIuzrtXesgQEZ9z4vod066lL1uqH6bY4b5v2aVZLQtZ1PjI9nLrnI+fH2znLYyX9OR\\\\\\\/feF2\\\\\\\/CSKDMsqqLmSLKbYfJwycBXISyKa7omI2jIkrA555CbP8xi4Q7yhqsjZLo9xz3pKU3Q+rAmXnzVStBksJCfZ3t3Kf5q9nt+3l1jeXlo4wn+0GYXVG18bJoQQ76\\\\\\\/y28D6wQoyR0Sgko7\\\\\\\/8cpkzZlm3st30bhZGGCWpPVqq9Qrx578CPMJX2shc92s8uFZo\\\\\\\/4i\\\\\\\/jHq5h\\\\\\\/x+UHjh3RCy4Ro460ZpOZBeuVjn6g1xAprtM7lo4MdQcUdMNX9vvN7mJEst5b7Dz4PrjnSwkognj+7J1soZCMOHuEI0SHqU\\\\\\\/GbuYH6KJWyE6wIkynPqpCo\\\\\\\/WS5WXXjH0gDmVvLlLNU8ZOYSdhmovaAvHWYl8+YzyPZ4O7IW0Wv7zkhFSSeGigius3PSwcLRpDF\\\\\\\/Qd8TrFQzKbY1VbIa62E6qJwdx5JRWczejdSM2Qs0qH3q1hlHgGnVIphgsZPRvC0o\\\\\\\/5SaFS\\\\\\\/z2hRNv8mGmq5lt0ykWzdCqTGtTsP09lr91HS6t8YUJLnS8n6m8FzSqWHa7mq2bbrdzLimp7VvMkqwIFhzuooTI468JdOlSXazDPgPbDYCI16ugqN7qLJsq9rTw+9Q9ScDwV5PDZF\\\\\\\/KD52yR4+IxifDQQgrDL2DfVclqc8PW2vLx5zPmHCTlL1OYHdfty4nlW6y2\\\\\\\/UA0uqING2G5L+8RKx\\\\\\\/c0mn+q3kloFdDqrYibD4nhLZS5l+q4FpHGZP9z41Jldu7bN8piZZ0MrfWfw9QjwKAY8VeDXeLbl7hj+WcD6gM0+l4G\\\\\\\/W5RM4z7CDeBqigm7NdurO5HCF9eXElXqQIheAxe6+2LIUZftXddPRxsxlItGd3UNrAArnaa5KbxHw9ojvcjxwC8V+k2BHyLx\\\\\\\/q6yR1+V3GvAwoItc3R4YqiNs+t+4WjexMWeRvM3g23AQYwogSNxWT07zbl7J2M6Nro58SOJurZir5V9EXfO7QAzVbzL0eFE4Rn2gkbSPHms296iMKnq\\\\\\\/HkZnLAvsKVHtRdJh9bgprXU81UzDomq6YDJVn6lNp\\\\\\\/\\\\\\\/wmRWJye7e9WHw6BTUakjhLczrgh1gJNkhvLvzXpMjsTEdO\\\\\\\/xu8iJfa4STO7iJuA\\\\\\\/kgaWSNvmd+sUxulnMWL5wpf7cfyBAj0uhMOXFkiV\\\\\\\/GkYzMw9VSotGwV3cknrrEuPqP9kU+pyqf\\\\\\\/DeO+9Jof6jT\\\\\\\/Z\\\\\\\/mjw0dwK+k6TsN9UXuwjPeMb+qCBMJKHiGC4KBwOHYFJUAn\\\\\\\/vGiFbPdCrRYTLijl5jKAdVxsaQe9pMhyTHHcQyEg3m+k43p+Gg\\\\\\\/aaHOBB8OlHpe5YOX9sA5++tgUpS1DwqxVe+Hw8Bu5jNAd\\\\\\\/qd9CoP4kNO3OQjzaur+I6vSq7BcNham9UaEaj4Fbcc42qOB0gaGJDjGfBtiuJqWMnvO+z+SLQDoiUuJkq3S82TN0\\\\\\\/kI330yI+\\\\\\\/CRoN6Spu+4T3zFIh9ZlFGFSBqHIgiTQnC0oHB8+qboN\\\\\\\/rsGIUJtunIU6sAx225fXlhYMtq8bES60Ss32pbH+lkwB7rDFkQv27qUMXgeTF2hcCoA5wkUp3PQOs8rC\\\\\\\/7MobidixrPsldzTWWFx6\\\\\\\/TmnwW4dXPoEnnK9t2pBZFI\\\\\\\/DLdOvV00LpRewZyQxCkTejskPkl02fpxxRd0XaNO\\\\\\\/HiNpYFTNwFz4wBazA32IYuLmURC5x2sK6Gae\\\\\\\/OODFFIgsqCQgm0ZIlk3rI7Vsmr53E1ZJOeakbUo9xjI+ooh7pFXBjx5OwAf64w1tcNOw9MKf7Q5L6i6+\\\\\\\/HPQLvJ4PtiCw2PcMe4FBhpO3HQa66CrOT+3nVLtugt9ryzL87ABYrL4H9COuOVD7Og8WCvhZDVfgQ9ErG3fWtod0RENcVHBLsKDkxi0jh9fUOu24\\\\\\\/v2IuHAlc5veQB9hrj6GNk84pV4TKO9WJgzcBJnoXBXoj96+uLWmaOv5dSwPGJ8vKCPokwPznIm1oH+XuCmGahkm4AQ9Lk8XKs7W3ijs3oujkh20aW6r5JWL7j3j38lVaQQPS5pMOyy0I0gmZV9uaf\\\\\\\/1kwNQ\\\\\\\/uWEM753Fq5EVd6sC0GVOcu1buG8pcLxzhq6k27XPXPj73o+yk4LvTHx4N2Kt7zkqNzi\\\\\\\/8zl8nbzCF6gJ5hppAbsPg8NrW7J41dXPX89C2IWfdHAWrVL93mkS+r\\\\\\\/RbJCe1ojKOjoPdPdsdxSUkkCnrlFGAYgvSeehL4DER5li+urUeX+2vbM8aj\\\\\\\/MrxrEAyhHB4WAyI\\\\\\\/y2O7v51fSwF08VrSJBZ05b1+\\\\\\\/XRkSijf99e+eyCh\\\\\\\/RUtk0hr67eYflT847jUWPQToxooU07yxIdnTE6wVc8OwwxroPieoAfbr3GZ3qpJyaHnWJQ\\\u0022,\\\u0022iv\\\u0022:\\\u00229GXD\\\\\\\/HwbLorHEoHrQhWA\\\\\\\/A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ed5fff59457b96961a928e13472f48c83fb8a5ccf0480809d3d07450b418aa17\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022OR\\\\\\\/2Juc0wdzD5gqdQNphR9sx9ofNNZfeozv8zyTv7nNvLToaLwBff9Nt8eEtOSmuwYvrSKafjL\\\\\\\/jwLHPU0ZrBAOTgLASpAeMVJCOr+GV1NYu81FoEAa3n51rQM+gFodg0JC2H5ub0qWa+fizLrOXeu1gGSwvP4Ms81WwSGXeenmBtVaoBHw5MIhAmjVHp2I5lWvLssqpWfa4UojsKBZm1vBU4TsIZ3nGzazIKdOt6nUKlBB5OO1V5qOkijZuhZNgbPIGfzUO\\\\\\\/aVUYkN29A9fo2+HP0iqkFb4Zu2FTpni+A40MgglfG4hnUXWL5Fu41ZML2Qw32YqfQQFTRqPEkiLznJjMh+3\\\\\\\/po8TO73hnfkx5kpSGCmljBBEdfmlbjmLCz7JSW78ph6\\\\\\\/ff5anSUAranCbYxN4sn6o7cLbdV0lhwJObOwRXqktg5IkENHPzCY\\\\\\\/OlgCiGKghHHpGy0OKWiXXTQYWdhNqch\\\\\\\/gDzMQdT0Y0h7Ha8\\\\\\\/5SOw7sEPFg6+2rFuKiTuJdJEfNwN+48iKoSmzDUOQGPdueMce7d3a0FewXtXJVeoeXl81Z0whmjEEYFC3akE9L1a2A++ttOc12SJqkBVAsguEvd4GD2\\\\\\\/iw5nCHuklkkBHu0KghOM+UgIzwzyhPXSwLFHm7OO1da0i3hpVlGZILYeCOGPn6NxnewnRUGw1dc3\\\\\\\/oBbfucoVcAAQxVHBER4JvqGpFWXfFbNKf1Ifz4ZrCeoiTFq701ckf5AgziZQTwQ+1mHD2Fd\\\\\\\/ACoAYwLO48Kbx8PdRRGUa13QT3MBFrFp5VG4QM3Dk6iYCb+Ma+KaVyZXLxoqVpxQ\\\\\\\/Rc+8FML+f2lGULNUmnsj6Wk7ieHYOaFIaVgjE1K53Ge5CGF6wZMc72ITUixav9HvOEiv0Mjn6b4vifVZwoRGOD8wDeBXdI373NPsn4t0KzsaZPPmN8PyoMvYH4CHpvuWH2aAbR7rN5pUc73+b24lE7y8uC83bvvYPlbL+cHu0HBaswbwqqMHKHIebwsQb+QZiKlKNkG5gspafCNyxmsQaIXT0uA5VSoH6cwgvX39ZJqbzxsFQPqisNzD2rEw0BRdNMcuTq6U1cfiH3c2kIkrssLgF+5EJh4PS2HX4ytcgQUu9eIDeFOBflQbt\\\\\\\/JMUhuYJJa2JPMVq30+Cf0gbFpY0T4utglV2jr\\\\\\\/K8p8jE4JdT1UJ5yiD\\\\\\\/4CvmglhNveMUaIDc\\\\\\\/aW7qur2eiKqAlshBm47nbL7CmH18Ckwz8Z\\\\\\\/I6oo8\\\\\\\/1pnwFinC224eq76MNXY4kCTI\\\\\\\/gKwTg7na2kDAqc\\\\\\\/F4Q\\\\\\\/H8J28JDN65CT4qx0YR3SOyW0940QOTl6H4mAG1c6uw43Z8D\\\\\\\/QKofShHslhx\\\\\\\/SgvXcE5k+dHip4umZopcm0pgr23p+abT6+J8VBNtQVUFA18zyOvfbMj6FXPRXOmgebgXIHfqET5XZlj0SQxSypEZGriY+iy1ckVsUrOPXLAl2tJP1B2LYld\\\\\\\/x6k26UTMr1ccA+bgwGRT3aOQkyLNxQfsirUNWfQX4vZWxy455P9Bbk7gkNrgZrVViyYcglh89VBslvb7DC+dCFzWkUwEcdCrjquDAu4YygEjklJhOlfJX2CbLnAIlUsIuqkC7NpVO9XwPD2dBujdpROeJzok8yomHRpAZMV6C26opmwdbOkzAm9aqudy8drOD0td6I697qLVV6j550nfNMlqQoxqfIi+MOw6rq345wuApf1tMJWxAIDumhcEVSCyd1jsn7zZqA1Rc0FxVczOPU6TUybJ\\\\\\\/8m2iSdnZtY3u9c3WwPzU\\\\\\\/IH\\\\\\\/1i4ZaknXCVHzsTM3nRRr4mxB58xF3BoTkj4GBRxXFj2MuRZRRuXlgYfhk0oc1X9huDvVsb+1B4rdwOwFkTgZLTDORD\\\\\\\/ib7W0ZlAVr+j5wSUviUyegPuXBoW+iIVNlhTK1Epua0OtSpiSkXlcYf2NQat3mUL3W\\\\\\\/N\\\\\\\/SF6jxbyvANRqNR5EIa2Yq3XfvPTJGQuFPKKeY6p5DbOppmVtld2FKJRN\\\\\\\/IVvgBENmRQSBtnskRWDD3ZH6M=\\\u0022,\\\u0022iv\\\u0022:\\\u0022kLtuLtNJFf31i5PnWJNS1w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a780fd7d343d714d9a143700e3393126c9eddf05879af77f3147f5a7b57aa3ee\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-484253 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1405" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-L69hV7LvfIZSLbvqpdRpV9aIIG6wHpEyp3sYszmHAlg" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1405" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-kwobjTjhkuLChVwCT0nMJWOXvD7OTC-PYOpaoQzWr2E" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/5/1393" title="Genotype Calling from Population-Genomic Sequencing Data" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/5/1417" title="Dynamic Notch Signaling Specifies Each Cell Fate in Drosophila Spermathecal Lineage" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484253" id="node484253" data-pisa="ggg;7/5/1405" data-pisa-master="ggg;g3.116.038638" data-apath="/ggg/7/5/1405.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A New Reference Genome Assembly for the Microcrustacean <em>Daphnia pulex</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhiqiang Ye</span>, <span class="highwire-citation-author" data-delta="1">Sen Xu</span>, <span class="highwire-citation-author" data-delta="2">Ken Spitze</span>, <span class="highwire-citation-author" data-delta="3">Jana Asselman</span>, <span class="highwire-citation-author" data-delta="4">Xiaoqian Jiang</span>, <span class="highwire-citation-author" data-delta="5">Matthew S. Ackerman</span>, <span class="highwire-citation-author" data-delta="6">Jacqueline Lopez</span>, <span class="highwire-citation-author" data-delta="7">Brent Harker</span>, <span class="highwire-citation-author" data-delta="8">R. Taylor Raborn</span>, <span class="highwire-citation-author" data-delta="9">W. Kelley Thomas</span>, <span class="highwire-citation-author" data-delta="10">Jordan Ramsdell</span>, <span class="highwire-citation-author" data-delta="11">Michael E. Pfrender</span> and <span class="highwire-citation-author" data-delta="12">Michael Lynch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1405-1416;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.038638 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484253" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhiqiang Ye </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhiqiang%2BYe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ye%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhiqiang%2BYe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zhiqiang.yes@gmail.com" class="" data-icon-position="" data-hide-link-title="0">zhiqiang.yes@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sen Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, University of Texas at Arlington, Texas 76019</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sen%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASen%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ken Spitze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ken%2BSpitze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spitze%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKen%2BSpitze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jana Asselman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory for Environmental Toxicology, Ghent University, B-9000, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jana%2BAsselman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Asselman%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJana%2BAsselman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Xiaoqian Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaoqian%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaoqian%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Matthew S. Ackerman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BS.%2BAckerman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ackerman%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BS.%2BAckerman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jacqueline Lopez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Notre Dame Genomics and Bioinformatics Core Facility, University of Notre Dame, Indiana 46556</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacqueline%2BLopez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lopez%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJacqueline%2BLopez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Brent Harker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Notre Dame Genomics and Bioinformatics Core Facility, University of Notre Dame, Indiana 46556</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brent%2BHarker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Harker%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrent%2BHarker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">R. Taylor Raborn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div><div class='author-affiliation'><span class='nlm-sup'>**</span>School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BTaylor%2BRaborn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raborn%20RT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AR.%2BTaylor%2BRaborn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">W. Kelley Thomas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Hubbard Center for Genome Studies, University of New Hampshire, Durham, New Hampshire 03824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=W.%2BKelley%2BThomas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thomas%20WK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AW.%2BKelley%2BThomas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Jordan Ramsdell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Hubbard Center for Genome Studies, University of New Hampshire, Durham, New Hampshire 03824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jordan%2BRamsdell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramsdell%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJordan%2BRamsdell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Michael E. Pfrender </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Biological Sciences, University of Notre Dame, Indiana 46556</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>Environmental Change Initiative, University of Notre Dame, Indiana 46556</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BE.%2BPfrender%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pfrender%20ME&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BE.%2BPfrender%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Michael Lynch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BLynch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lynch%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BLynch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/5/1405" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:484253" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:484253/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1405.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:484253" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:484253/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1405.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:484253" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:484253/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1405.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:484253" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:484253/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/5/1405.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:484253" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:484253/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Comparing genomes of closely related genotypes from populations with distinct demographic histories can help reveal the impact of effective population size on genome evolution. For this purpose, we present a high quality genome assembly of <em>Daphnia pulex</em> (PA42), and compare this with the first sequenced genome of this species (TCO), which was derived from an isolate from a population with &gt;90% reduction in nucleotide diversity. PA42 has numerous similarities to TCO at the gene level, with an average amino acid sequence identity of 98.8 and &gt;60% of orthologous proteins identical. Nonetheless, there is a highly elevated number of genes in the TCO genome annotation, with ∼7000 excess genes appearing to be false positives. This view is supported by the high GC content, lack of introns, and short length of these suspicious gene annotations. Consistent with the view that reduced effective population size can facilitate the accumulation of slightly deleterious genomic features, we observe more proliferation of transposable elements (TEs) and a higher frequency of gained introns in the TCO genome.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genome-annotation" class="hw-term hw-article-keyword hw-article-keyword-genome-annotation" rel="nofollow">genome annotation</a></li><li class="kwd"><a href="/keyword/effective-population-size" class="hw-term hw-article-keyword hw-article-keyword-effective-population-size" rel="nofollow">effective population size</a></li><li class="kwd"><a href="/keyword/gene-number" class="hw-term hw-article-keyword hw-article-keyword-gene-number" rel="nofollow">gene number</a></li><li class="kwd"><a href="/keyword/intron" class="hw-term hw-article-keyword hw-article-keyword-intron" rel="nofollow">intron</a></li><li class="kwd"><a href="/keyword/mobile-elements" class="hw-term hw-article-keyword hw-article-keyword-mobile-elements" rel="nofollow">mobile elements</a></li></ul><p id="p-4">The ultimate goal of comparative genomics is to form a synthesis integrating the fundamental evolutionary forces explaining the variation of genomic architecture across a wide range of phylogenetic lineages. The reduced effective population size (<em>N</em><sub>e</sub>) of eukaryotic species relative to prokaryotic species is hypothesized to be centrally involved in the emergence and distribution of numerous genomic features unique to eukaryotes, and the associated principles should naturally extend to variation among lineages of closely related species (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lynch 2007</a>). However, until recently, it has been difficult to study the impact of reduced <em>N</em><sub>e</sub> on the evolution of genomic architecture because of the lack of genomic data from closely related species or populations with disparate population sizes. Moreover, a lack of parallel understanding of other fundamental microevolutionary parameters such as mutation and recombination rates complicates efforts to single out the role of <em>N</em><sub>e</sub> (and the associated power of random genetic drift) on genome evolution.</p><p id="p-5">Here, we focus on the comparative genomics of two cyclically parthenogenetic <em>Daphnia pulex</em> clones (PA42 and TCO), which come from populations differing substantially in historical <em>N</em><sub>e</sub>. The <em>D. pulex</em> species complex consists of a vast array of populations inhabiting hundreds of thousands of ponds and lakes, throughout the northern temperate zone, most of which (including PA42 and TCO) reproduce by repeated generations of clonal reproduction via unfertilized eggs, punctuated by a phase of sexual reproduction. PA42 was sampled from a woodland vernal pond within Portland Arch Nature Preserve, IN, whereas TCO was derived from a permanent pond in the Siuslaw National Forest, near the Pacific coast in OR. A complex geological history in western OR contributes to the presence of divergent lineages of multiple zooplankton species (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dyke <em>et al.</em> 2002</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Shafer <em>et al.</em> 2010</a>), including <em>D. pulex</em> (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Crease <em>et al.</em> 1997</a>). The lineage containing TCO is thought to have diverged from midwest US <em>D. pulex</em> as a consequence of geographic isolation during Pleistocene cycles of glaciation, and OR <em>D. pulex</em> is sometimes assigned a distinct species designation, <em>D. arenata</em>. However, the level of nucleotide sequence divergence between TCO and <em>D. pulex</em> in the Midwestern US is only twice that of the average within-population divergence of alleles in the latter (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Lynch <em>et al.</em> 2016</a>).</p><p id="p-6">Consistent with the glacial relict hypothesis, the level of nucleotide diversity within the population from which TCO is derived is &lt;10% of that found in typical midwest populations, including the source population of PA42 (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Lynch <em>et al.</em> 1999</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Omilian and Lynch 2009</a>). Given the similar estimates of the base-substitution mutation rate (<em>u</em>) derived from mutation-accumulation experiments applied to these two lineages (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Keith <em>et al.</em> 2016</a>), and given that silent-site diversity estimates the composite quantity 4<em>N</em><sub>e</sub><em>u</em>, we estimate that the TCO source population has a <em>N</em><sub>e</sub> ∼ 10% of that of typical populations in the midwest US.</p><p id="p-7">The substantially reduced heterozygosity in the TCO isolate served the practical purpose of avoiding the technical difficulties that can arise when assembling a highly heterozygous genome, and hence led to the first <em>Daphnia</em> genome sequencing project (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Colbourne <em>et al.</em> 2011</a>). The resultant TCO assembly and annotation implied a surprisingly large number of genes (∼31,000), greatly exceeding the numbers in most other arthropod genomes and including a large number of recent gene duplicates. Remarkably, an unusually high rate of intron gain is exhibited by the TCO genome, and more generally by OR <em>D. pulex</em>, relative to all other metazoans and arthropods (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Colbourne <em>et al.</em> 2011</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Li <em>et al.</em> 2009</a>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">2014</a>). Concerns have been raised about a potential overestimation of the gene number in the TCO genome, resulting from assembly issues such as the fragmentation of genes into different contigs, one argument being that ∼20% of the gene annotations in the original TCO assembly lack translation initiation and/or termination codons (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Denton <em>et al.</em> 2014</a>). However, no draft genome assembly is without problems, and it is plausible that many of the aberrant features of the TCO genome may reflect real evolutionary responses to a long-term reduction in <em>N</em><sub>e</sub> (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Lynch 2007</a>). Here, we present a <em>de novo</em> assembly of the genome of the PA42 isolate of <em>D. pulex</em>, constructed using a combination of paired-end, mate-pair libraries, and synthetic long reads (SLRs), with annotation guidance provided by RNA-sequencing (RNA-seq) data from animals grown under diverse environmental conditions. Although not perfect, this assembly is of considerably higher quality than the previous one reported for TCO, is likely much more representative of the genomic architecture of the study species, and yields insights into the genome-wide consequences of reduced effective population size.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Daphnia culture</h3><p id="p-8">The <em>D. pulex</em> isolate PA42 was obtained from a woodland vernal pond, Portland Arch (latitude: 40.2013, longitude: −87.3294), IN in May 2013. This isolate is cyclically parthenogenetic, <em>i.e.</em>, reproduces asexually in favorable environmental conditions and sexual reproduction is triggered by deteriorating conditions, such as high population density and food shortage. This isolate was maintained under favorable laboratory conditions so that they could reproduce parthenogenetically, and was fed <em>ad libitum</em> with a suspension of the algae <em>Scenedesmus obliquus</em>.</p></div><div id="sec-3" class="subsection"><h3>De novo genomic assembly</h3><p id="p-9">We assembled the genome of the <em>D. pulex</em> PA42 isolate <em>de novo</em>. DNA of the PA42 isolate was extracted from mass culture of clonally reproducing individuals. To minimize bacterial contamination, for any DNA collected here for sequencing libraries we starved and treated the culture for 2 d with 50 mg/L ampicillin and 50 mg/L streptomycin libraries in COMBO media. We made paired-end sequencing libraries with insert sizes of 350, 450, and 600 bp using the NEBNext Ultra DNA Library Prep Kit for Illumina (New England Biolabs, Ipswich, MA). Also, we made mate-pair libraries with insert sizes of 3, 5, 11, 19, and 27 kbp using Nextera Mate Pair Sample Prep Kit (Illumina, Inc., San Diego, CA) (Supplemental Material, Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1/FileS4.docx">File S4</a>). The libraries were sequenced on a HiSequation 2500 sequencing platform (Illumina) with 250 bp paired-end reads. Low quality regions and sequencing adapter contamination were trimmed from the raw reads using Trimmomatic (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bolger <em>et al.</em> 2014</a>) with parameters: ILLUMINACLIP:True-seq:2:30:10 HEADCROP:3 SLIDINGWINDOW:4:15 MINLEN:30. The adapter sequences on mate-pair sequences were removed using the software Nextclip with default parameters (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Leggett <em>et al.</em> 2014</a>). We used AllPathsLG version 43460 (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gnerre <em>et al.</em> 2011</a>) for generating genomic assembly. The parameter for running PrepareAllPathsInputs.pl are “PLOIDY = 2 GENOME_SIZE = 200,000,000 FRAG_COVERAGE = 45 JUMP_COVERAGE = 45 LONG_JUMP_COVERAGE = 1 PICARD_TOOLS_DIR = picard-tools-1.52.” Parameters for RunAllPathsLG are “HAPLOIDIFY = True REFERENCE_NAME = PA42 TARGETS = standard EVALUATION = BASIC.” Then we generated SLRs to connect the scaffolds further and remove the contaminants. We prepared a 10 kbp library using a TruSeq Synthetic Long-Read DNA Library Prep Kit (P/N FC-126-1001, Illumina, Inc.) according to the TruSeq Synthetic Long-Read DNA Library Prep Guide (P/N 15047264 Rev. B, Illumina, Inc.) with BluePippin Size-selection (TruSeq Synthetic Long-Read DNA Library Prep Guide Supporting Material). High quality gDNA (500 ng) was sheared by g-Tube (P/N 520079, Covaris, Woburn, MA). End repair, dA-tailing, and adaptor ligation was performed on sheared gDNA. The ligated products went through a BluePippin Size-selection protocol for 8 k–10 kbp range and were quantified with qPCR. Based on qPCR results, the library was diluted and distributed into a 384-well plate, whereby each well contained ∼6–8 fg of the size-selected library, for long range PCR. Next, the library went through Nextera tagmentation with 384 different barcoding PCR primers. Final products were pooled, concentrated, and were dual-sided size-selected with Sample Purification beads. Bioanalyzer was used for final validation of average size and absence of primer contamination. A single pool library was sequenced on HiSequation 2500 Rapid Run mode with 150 bp paired-end reads, producing 483 million reads. TruSeq adapter sequences were trimmed using Trimmomatic (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bolger <em>et al.</em> 2014</a>) in paired-end mode. Error correction and assembly of filtered reads was completed using the multi-cell SPADES 3.5.0 pipeline (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Nurk <em>et al.</em> 2013</a>) with default settings. Contigs larger than 8000 bp were used for further analysis. First, SLRs were used for rescaffolding the draft assembly PA42 1.0 using SSPACE-long reads software (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Boetzer and Pirovano 2014</a>). Then, we used PBjelly (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">English <em>et al.</em> 2012</a>) to fill the gaps; the result was PA42 2.0.</p><p id="p-10">Because the presence of bacteria in <em>Daphnia</em> culture media and/or possible endosymbiont species (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Qi <em>et al.</em> 2009</a>) can introduce foreign DNA sequences into the final assembly, we took several measures to remove bacterial DNA contamination. First, we blasted all the scaffolds against NCBI’s nonredundant nucleotide database (NR) to remove scaffolds with bacteria origin. Scaffolds with <em>e</em> value &lt; 1e−5 and &gt;95% identity to bacterial sequences were removed. Second, we used an independently derived paired-end sequencing library of PA42 (PE data) and a collection of 104 single sperm whole-genome sequence data to verify true <em>Daphnia</em> scaffolds. These two datasets were mapped to PA42 scaffolds using bwa-mem (version 0.7.2) with default parameters (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Li and Durbin 2009</a>), then we calculated the breadth of coverage for each scaffold (nucleotide bases in a scaffold that were covered by sequencing reads). In total, scaffolds summing up to 150 million bases of the PA42 genome have &gt;80% breadth of coverage by both PE and sperm reads (Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1/FileS4.docx">File S4</a>), strongly suggesting their noncontaminant status. We manually checked the scaffolds with high PE read coverage (&gt;80%) but lower sperm read coverage (&lt;80%) by blasting them against the NR; the results show that most of their top hits were <em>Daphnia</em> sequences in TCO except for scaffolds with &lt;30% breadth of sperm read coverage. Given that whole-genome amplification was performed before sequencing single sperm, which is likely to introduce bias in recovering different genomic regions, we decided to keep scaffolds with &gt;80% breadth of coverage from PE reads and &gt;30% from sperm read coverage in the final assembly.</p></div><div id="sec-4" class="subsection"><h3>Gene prediction and functional annotation</h3><p id="p-11">We annotated the PA42 draft genome using three different strategies, <em>i.e.</em>, RNA-seq-guided gene discovery, protein homology, and <em>ab initio</em> gene prediction. To perform annotation of the PA42 draft assembly using empirical gene expression data, we generated a set of RNA-seq libraries for the PA42 isolate (Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1/FileS4.docx">File S4</a>) under three different life stages (mature males, adult parthenogenetic female, and adult presexual reproduction female) and seven different environmental conditions (<em>i.e.</em>, treatments by temperature, Atrazine, UV light, pH, NaCl, and Nickel). The transcriptome for each life stage/condition was assembled <em>de novo</em> using the software Trinity (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Grabherr <em>et al.</em> 2011</a>), with the pooled, total transcriptome consisting of 177,598 transcripts. These transcripts were then used as seed for evidence-based gene annotation using the software Maker version 2.0 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Holt and Yandell 2011</a>). A total of 18,257 genes were found based on RNA-seq-guided annotation. Proteins from the Reference Species (Ecdysozoans <em>Caenorhabditis elegans</em>, <em>Strigamia maritime</em>, <em>Anopheles gambiae</em>, and <em>Drosophila melanogaster</em>, and <em>Homo sapiens</em>) together with 458 proteins from CEGMA (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Parra <em>et al.</em> 2007</a>) were used as input for MAKER (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Holt and Yandell 2011</a>) to identify genes with homologous evidence; in total, 19,830 genes were identified. Furthermore, we performed <em>ab initio</em> gene prediction on the PA42 draft assembly using the software SNAP (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Korf 2004</a>) trained by 800 genes with complete gene structure (<em>i.e.</em>, possessing 3′-UTR, 5′-UTR, and more than three exons) that are supported by RNA-seq. The <em>ab initio</em> annotation generated a set of 18,990 genes. As these three annotation strategies generated overlapping gene predictions, we used a conservative strategy to include genes with at least two pieces of evidence. Our gene annotation resulted in a final set of 18,440 genes (Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1/FileS4.docx">File S4</a>). We used the Benchmarking set of Universal Single-Copy Orthologs (BUSCO) (version 1.2; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Simão <em>et al.</em> 2015</a>) to estimate the completeness of our genome. The metazoans database for BUSCO was downloaded from <a href="http://busco.ezlab.org/">http://busco.ezlab.org/</a>, then all the proteins predicted in our analysis were used to blast the metazoan database in BUSCO with -trans parameter, and the rest of the parameters were set as default. All the predicted proteins were blasted against NCBI’s NR using blastp program with an <em>e</em>-value set to 1.0e−3. A total of 16,806 genes (91%) were assigned with a corresponding gene name after this NR blast. We searched motifs and domains against public databases, including Pfam, PRINTS, PROSITE, ProDom, SMART, PANTHER, Gene3D, HMMpham, and SuperFamily through InterProScan 5.14 (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Jones <em>et al.</em> 2014</a>). Gene Ontology (GO) IDs and terms for each gene were assigned based on InterPro results. We also mapped the reference genes to KEGG and Enzyme codes. Of the genes, 4570 had an enzyme code and could be assigned to 117 KEGG pathways. The annotation process was done using blast2go to select GO terms generated by blasting to InterPro, the parameters we used were <em>E</em>-value = 1.0E−6, annotation cutoff = 55, and GO-weight = 5. Then, ANNEX (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Götz <em>et al.</em> 2008</a>) was used to correct the annotation. Annex can add and correct biological processes and cellular component GOs based on molecular function GOs.</p></div><div id="sec-5" class="subsection"><h3>Identification of TEs</h3><p id="p-12">Three classes of full-length TEs, including LTR retrotransposons, non-LTR retrotransposons, and DNA transposons, were <em>de novo</em> identified in the PA42 reference genome, respectively. Two computer softwares, LTRharvest (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Ellinghaus <em>et al.</em> 2008</a>) and MGEScan (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Rho <em>et al.</em> 2007</a>), were used to identify all potential intact LTR retrotransposons. We further filter the candidate LTR TEs based on similarity (&gt;80%) and the lengths (&gt;80 bp), as well as the presence of putative <em>Pol</em> ORFs. The EMBOSS package (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Olson 2002</a>) was used to search against Pfam HMM files (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Finn <em>et al.</em> 2014</a>) to identify key protein domains of <em>Pol</em> ORFs (<em>E</em>-value &lt; 1e−10). Reverse transcriptase (RT) sequences of each element were extracted to conduct phylogenetic analysis using Mafft (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Katoh <em>et al.</em> 2005</a>) and FastTree (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Price <em>et al.</em> 2009</a>) to get the subclass information. Non-LTR retrotransposons were identified using MGEScan-nonLTR (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Rho and Tang 2009</a>). Similar to the procedure of classifying LTR elements into different subclades, RT sequences were extracted and used to classify different non-LTR elements. DNA transposons were identified by using program MGEScan-DT (W. M. Ismail and H. Tang, unpublished data), which searches for genes encoding transposases in the genome by HMMer and extends on both sides of the hit region to locate the TIRs. This program automatically classifies the DNA elements into different clades.</p><p id="p-13">All subclasses of TEs were combined and removed redundancy with full sequence similarity &gt;98% by CD-HIT (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Fu <em>et al.</em> 2012</a>), constituting a nonredundant full-length TE library. This library was used in a homology search to detect all the fragmented TEs in the PA42 reference genome using RepeatMasker (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Smit <em>et al.</em> 2004</a>). Only the elements with the best score in the overlapping ones with a length of at least 500 bp were kept in our analysis.</p></div><div id="sec-6" class="subsection"><h3>Relative rate test</h3><p id="p-14">To evaluate the differences of molecular evolution between the PA42 and TCO genome comparisons, we performed the relative-rate test (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Sarich and Wilson 1967</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Wu and Li 1985</a>) to compare the substitution rates for silent and replacement sites, <em>K</em><sub>s</sub> and <em>K</em><sub>a</sub>, respectively, using sequences of <em>D. obtusa</em>, a closely related species, as an outgroup (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Tucker <em>et al.</em> 2013</a>).</p><p id="p-15">To calculate <em>K</em><sub>s</sub> and <em>K</em><sub>a</sub> in PA42/<em>D. obtusa</em> and TCO/<em>D. obtusa</em>, we mapped <em>D. obtusa</em> reads to the PA42 and TCO genome assemblies using bwa-mem (version 0.7.2) (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Li and Durbin 2009</a>), respectively. Reads with multiple mapping were excluded and bases in the reference genome with &lt;4 <em>D. obtusa</em> reads covered were not used in further analyses. Then, the consensus sequence for <em>D. obtusa</em> was generated using SAMtools (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Li <em>et al.</em> 2009</a>). Next, we extracted the coding sequence of each gene in TCO and PA42 based on the TCO annotation file (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Colbourne <em>et al.</em> 2011</a>) and the PA42 annotation generated in this study. <em>K</em><sub>a</sub> and <em>K</em><sub>s</sub> for each gene were then calculated using yn00 from the PAML4.8a package (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Yang 2007</a>). The yn00 program implements several counting methods for estimating <em>K</em><sub>a</sub> and <em>K</em><sub>s</sub> between two sequences; here, we picked the results using realistic evolutionary models (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Yang and Nielsen 2000</a>).</p></div><div id="sec-7" class="subsection"><h3>Gene family analysis</h3><p id="p-16">TCO, PA42, and Reference Species’ protein sequences were used in this analysis. Genes from TCO that may generated due to false annotations were removed prior to the analysis. For those genes with multiple transcripts, we retained the longest one, translated proteins were put into OrthoMCL (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Li <em>et al.</em> 2003</a>) to identify orthologs among Reference Species. Gene clusters were generated by OrthoMCL standard protocol. Genes in the orthologous clusters were assigned to Structural Classification of Proteins (SCOP) families (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Murzin <em>et al.</em> 1995</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gough <em>et al.</em> 2001</a>). Each SCOP family contains gene domains descended from a common ancestor. To identify the expansion and contraction of gene families we used CAFÉ 3.0 (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Han <em>et al.</em> 2013</a>) which relies on phylogenetic history to provide a statistical foundation for evolutionary inferences. To generate the species tree required by CAFÉ 3.0, single-copy gene families, which have one gene in each species, were extracted for phylogenic tree construction. Protein sequences of each gene were aligned using PRANK (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Löytynoja and Goldman 2005</a>) and then concatenated using custom Perl scripts (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1/FileS3.zip">File S3</a>). Maximum Likelihood trees were constructed using MEGA6 (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Tamura <em>et al.</em> 2013</a>). The RelTime-ML program implemented in the MEGA6 package was used to estimate divergence time among species and calibration time was obtained from the TimeTree database (<a href="http://www.timetree.org/">http://www.timetree.org/</a>).</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-17">TCO sequences and gene models used in this analysis were downloaded from JGI (<a href="http://genome.jgi.doe.gov/Dappu1/Dappu1.download.ftp.html">http://genome.jgi.doe.gov/Dappu1/Dappu1.download.ftp.html</a>). The gene model we used is FrozenGeneCatalog20110204; proteins and coding sequences used are FrozenGeneCatalog20110204.proteins.fasta and FrozenGeneCatalog20110204.CDS.fasta.gz, respectively. <em>D. magna</em> data were downloaded from wfleabase (<a href="http://server7.wfleabase.org/genome/Daphnia_magna/openaccess/genome/altassembly/">http://server7.wfleabase.org/genome/Daphnia_magna/openaccess/genome/altassembly/</a>), the draft assembly used is nwbdmag24g7d_asm.fasta.gz, and the <em>D. magna</em> proteins used in this analysis are arp7s10b14nodmag.aa.gz. The Reference Species’ coding and protein sequences were downloaded from Ensembl (Ensembl 80: May 2015). The raw reads for this study are available at the Short Read Archive with the study accession no. PRJNA307976. The <em>D. pulex</em> genomic assembly PA42 v3.0 is available at the European Molecular Biology Laboratory (EMBL) Nucleotide Sequencing Database or NCBI under the accession no. PRJEB14656. Sperm reads used in this study can be found with accession no. SRP058678. The sequence of <em>D. pulex</em> clones and <em>D. obtusa</em> from <a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Tucker <em>et al.</em> (2013)</a> are deposited under the accession nos. SAMN02252729–SAMN02252752 in the NCBI Sequence Read Archive. There was an error in NCBI server that the <em>D. obtusa</em> sequences were replaced by something else. Therefore, we redeposited the <em>D. obtusa</em> to European Nucleotide Archive under the accession no. PRJEB17737. </p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Genome assembly of PA42</h3><p id="p-18">We achieved a <em>de novo</em> assembly of the genome of a natural <em>D. pulex</em> isolate (PA42) using AllPathsLG (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Gnerre <em>et al.</em> 2011</a>) and a combination of paired-end and mate-pair Illumina sequencing libraries (650 × total coverage, Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1/FileS4.docx">File S4</a>). An additional 52,443 SLRs (&gt;8000 bp) were used to support the further connection of scaffolds and filling gaps. The draft genome presented here, PA42 3.0, consists of 156 Mb of DNA sequences located on 1822 scaffolds with a N50 scaffold size of 494 kb (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1/FileS4.docx">File S4</a>). The N50 scaffold number is 96 and the largest scaffold is 1,661,524 bp (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). A genetic map based on whole-genome sequencing of 104 single sperm (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Xu <em>et al.</em> 2015a</a>) anchors 366 scaffolds into 12 chromosomes, constituting 86.6% of the draft assembly and leaving 13.4% of the physical map unassigned with respect to chromosomes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1/FileS1.txt">File S1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484328/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484328/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of the genomic metrics for the PA42 and TCO genomes</span><div class="sb-div caption-clear"></div></div></div><p id="p-20">To help achieve accurate annotation, we performed RNA-seq on PA42 individuals exposed to seven environmental conditions (low pH, UV exposure, salinity, etc.) and three male and female life stages (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1/FileS4.docx">File S4</a>). We predicted 18,440 genes using an annotation approach guided by RNA sequences in combination with protein homology and <em>ab initio</em> prediction with the software Maker 2.0 (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Holt and Yandell 2011</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1/FileS3.zip">File S3</a> and Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1/FileS4.docx">File S4</a>). In line with the expectation for a high quality set of annotated genes, we find that the completeness of our genome assembly is likely to be very high based on comparison with a list of nearly universal arthropod genes (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Waterhouse <em>et al.</em> 2013</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Simão <em>et al.</em> 2015</a>); of the subset of genes that are found in &gt;90% of described arthropod genomes, we find 96% of these within the PA42 annotation list.</p><p id="p-21">In the final annotation of protein-coding genes in PA42, 1535 (∼8%) lack a canonical translation initiation codon and/or a translation termination codon, with 4% lacking an ATG start codon, 5% lacking a stop, and 1% lacking both (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). In contrast, there are 3395 (11.2%) annotated genes in the TCO genome lacking one or the other, and 1102 (3.6%) lacking both. Not all genes have canonical start codons, and the lack of an annotated stop codon can simply be a consequence of a lack of RNA-seq support in the downstream region of a gene. Thus, it is difficult to establish whether these are erroneously or simply incompletely annotated genes. For the 977 genes lacking stop codons in the PA42 annotation, 735 have RNA-seq covering &gt;50% of the coding region, and 408 have identifiable homologs in a set of Reference Species (Ecdysozoans <em>C. elegans</em>, <em>S. maritime</em>, <em>A. gambiae</em>, and <em>D. melanogaster</em>, and <em>H. sapiens</em>) used for comparative analyses (described below) and TCO.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484380/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484380/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Comparison of TCO and PA42 gene annotations</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Comparison of the PA42 and TCO annotations</h3><p id="p-26">To determine the sources of the substantially different numbers of annotated genes between PA42 and TCO (18,440 <em>vs.</em> 30,097), we performed comparative analyses of the two gene sets. A search of the annotated protein sequences of each gene in PA42 against the NR in NCBI reveals homologous sequences for 16,500 genes, 94% of which have best hits against members of the deposited TCO annotation (Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1/FileS4.docx">File S4</a>). Based on reciprocal-blast analysis, the number of one-to-one orthologous genes between TCO and PA42 is estimated to be 11,780, which we judge to be <em>bona fide D. pulex</em> genes. The total PA42 assembly displays larger average exon and intron sizes than those of TCO, with an average protein-coding gene (including introns) span of ∼3000 bp in PA42 <em>vs.</em> ∼2300 bp in TCO (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). If this comparison is restricted to the set of 11,780 TCO-PA42 orthologs, the average lengths of coding exons and introns are ∼10% greater in PA42 than in TCO. However, this is primarily due to the presence of more introns and exons in the longest bin in PA42, as the size distributions are otherwise extremely similar, having the same modes (Figure S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1/FileS4.docx">File S4</a>). The protein sequences of the 1:1 orthologs on average have 98.8% identity, with nearly 60% of them being 100% identical (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). The average synonymous (<em>K</em><sub>s</sub>) and nonsynonymous (<em>K</em><sub>a</sub>) differences per nucleotide site are 0.0512 and 0.0136, respectively (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1, B and C</a>). The median <em>K</em><sub>a</sub>/<em>K</em><sub>s</sub> ratio is 0.254 (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1D</a>), similar to that found in comparisons of the fruit fly <em>D. melanogaster</em> to other <em>Drosophila</em> species (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bergman <em>et al.</em> 2002</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1405/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distributions of the features of protein-coding sequences for the 11,694 one-to-one orthologs in TCO and PA42 (&gt;300 bp alignment length). (A) Amino acid sequence identity. (B) Pairwise divergence at silent sites, Ks. (C) Pairwise divergence at replacement sites, Ka. (D) The ratio Ka/Ks." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1066862039" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distributions of the features of protein-coding sequences for the 11,694 one-to-one orthologs in TCO and PA42 (&gt;300 bp alignment length). (A) Amino acid sequence identity. (B) Pairwise divergence at silent sites, &lt;em&gt;K&lt;/em&gt;&lt;sub&gt;s&lt;/sub&gt;. (C) Pairwise divergence at replacement sites, &lt;em&gt;K&lt;/em&gt;&lt;sub&gt;a&lt;/sub&gt;. (D) The ratio &lt;em&gt;K&lt;/em&gt;&lt;sub&gt;a&lt;/sub&gt;/&lt;em&gt;K&lt;/em&gt;&lt;sub&gt;s&lt;/sub&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1405/F1.medium.gif" width="440" height="353"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/5/1405/F1.medium.gif" width="440" height="353"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1405/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1405/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484396" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-27" class="first-child">Distributions of the features of protein-coding sequences for the 11,694 one-to-one orthologs in TCO and PA42 (&gt;300 bp alignment length). (A) Amino acid sequence identity. (B) Pairwise divergence at silent sites, <em>K</em><sub>s</sub>. (C) Pairwise divergence at replacement sites, <em>K</em><sub>a</sub>. (D) The ratio <em>K</em><sub>a</sub>/<em>K</em><sub>s</sub>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">Given the base-substitution mutation rate in <em>D. pulex</em>, 5.7 × 10<sup>−9</sup>/site/generation (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Keith <em>et al.</em> 2016</a>), under the assumption of neutrality, which appears to be fulfilled for silent sites in this species (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Lynch <em>et al.</em> 2016</a>), the observed average level of <em>K</em><sub>s</sub> implies that the average pair of TCO-PA42 orthologs diverged ∼4.8 million generations ago (or assuming five generations per year, ∼1 million years ago). To put this in perspective, analysis of a population near the PA locality yields an estimate of π<sub>s</sub> = 0.0183 for the average within-population silent-site divergence between random alleles (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Lynch <em>et al.</em> 2016</a>). Because the population from which TCO was obtained is nearly completely inbred (with π<sub>s</sub> ≈ 0.001; <a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Colbourne <em>et al.</em> 2011</a>), a corrected measure of silent-site divergence between the OR and IN <em>D. pulex</em> is 0.0512 − (0.0183/2) = 0.0421 (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Nei and Li 1979</a>), reducing the previous time estimates by 18%.</p><p id="p-29">To investigate the potential phylogenetic source of the large difference in gene numbers between these two clones, we queried the protein sequences of TCO and PA42 against the gene lists for four other Ecdysozoans (<em>C. elegans</em>, <em>S. maritime</em>, <em>A. gambiae</em>, and <em>D. melanogaster</em>) and <em>H. sapiens</em>. Using an e-value cutoff of 0.01, we detected 17,062 genes in TCO and 13,260 in PA42 with homologs in these Reference Species (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). For PA42 and TCO genes with no identified Reference homologs, reciprocal blasts between these two genomes identified an additional 3818 genes in PA42 and 5715 in TCO to have homologs across the two assemblies; these are potentially <em>D. pulex</em>-, <em>Daphnia</em>-, cladoceran-, or crustacean-specific genes. The remaining annotated genes in each genome without any identified metazoan homologs number 7320 in TCO and 1362 in PA42.</p><p id="p-30">It has been argued that the TCO gene number estimate is inflated due to errors in genome assembly and/or annotation (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Denton <em>et al.</em> 2014</a>). To evaluate whether genome assembly and/or annotation error in TCO is responsible for the bulk of the inflated gene number, we performed several analyses. First, orthologous gene clusters were generated among PA42, TCO, and the above-noted Reference Species using OrthoMCL (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Li <em>et al.</em> 2003</a>). This program uses a Markov cluster algorithm to group orthologs and in-paralogs (genes duplicated after speciation) across multiple taxa. We found 13,375 orthologous clusters to contain genes from PA42 and/or TCO. In 68% of these cases, the gene number is identical for both clones, whereas 22% of the clusters have more genes from TCO and 10% have more genes from PA42. If divergent alleles were artificially assembled and annotated as paralogs in the TCO genome (but not in PA42), this might partially explain the elevated TCO gene numbers in orthologous clusters (with a 2× inflation expected for single genes that are erroneously annotated as two paralogs in TCO). For the orthologous clusters with more genes in TCO, 17% had a gene number &gt;2× that in PA42, and in 5% the number of TCO genes was ≤2× that in PA42. In principle, the latter category could include some erroneous annotation of alleles as genes (although this is unlikely owing to the extreme homozygosity of TCO), but clusters with &gt;2× inflation in gene number in TCO cannot be accommodated by such an explanation (unless the baseline gene number in a cluster in TCO also exceeds that in PA42).</p><p id="p-31">Another potential reason for the gene number discrepancy between TCO and PA42 is the artifactual splitting of a single gene into multiple pieces (and each piece being enumerated as a separate gene in the TCO annotation). Such errors are expected to result in two or more genes in one annotation mapping to different regions of a single gene in the other annotation. To quantify the impact of such merge/split events, we reciprocally blasted the protein sequences of the two assemblies and detected more split events in TCO compared to PA42 (179 <em>vs.</em> 58, <em>P</em> &lt; 0.0001, Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1/FileS4.docx">File S4</a>), with most of them being two genes in one assembly merged into one gene in the other. A total of 369 genes in TCO were merged to 179 genes in PA42, and 120 genes in PA42 were merged into 58 genes in TCO. Without additional information from an alternative genome or mRNA sequencing data for these genes we are unable to resolve which annotation is correct; nonetheless, this overall analysis implies that gene fragmentation is a relatively minor issue in both the TCO and PA42 assemblies.</p><p id="p-32">Looking more closely at the 17% of orthologous clusters with &gt;2× genes in TCO relative to PA42 (8346 genes in total), we found that most of these lack detectable orthologs in other species (Table S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1/FileS4.docx">File S4</a>). A total of 7320 such genes have no obvious orthologs in either PA42 or the Reference Species employed in this analysis, and of these TCO-specific genes, 631 are singletons, while the remaining 6689 form orthologous clusters of two or more genes. For this latter subset, the average pairwise <em>K</em><sub>S</sub> for paralogs is 0.050 ± 0.001, which is similar to the average pairwise <em>K</em><sub>S</sub> for the 1:1 TCO-PA42 orthologs (0.056 ± 0.001), implying recent duplication.</p><p id="p-33">Possible explanations for this substantial number of TCO-specific genes are errors in annotation, artifacts of some sort of microbial contamination in the TCO DNA source, products of rampant horizontal gene transfer into the TCO genome (possibly of multiple origins), or a combination of these effects. However, the “orphan” TCO genes are highly scattered at the contig and scaffold levels in the TCO assembly, in 59% of cases having nearest neighbors with orthologs in PA42 (Table S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1/FileS4.docx">File S4</a>), so it is unlikely that they are derived from misassembly associated with a contaminating bacterial genome sequence. Fortuitously, whole-genome sequence exists for several mutation-accumulation lines derived from TCO several years after the original genome sequencing project (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Keith <em>et al.</em> 2016</a>), and the raw reads from such clones map to 92% of the TCO orphans, providing some validation that much of the DNA associated with the TCO orphan genes is not a consequence of transient contamination. Genome-wide sequence is available for 11 other <em>D. pulex</em> clones derived from broadly separated geographic locations (including one from OR; <a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Tucker <em>et al.</em> 2013</a>), and we find that, on average, 11% of the reads from these clones map to TCO orphan gene DNA (Table S7 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1/FileS4.docx">File S4</a>), covering the same orphans as in the PA42 analysis. Nonetheless, of the 7320 orphan annotated TCO genes: 592 are not covered by any read from TCO-derived mutation-accumulation lines and could involve some sort of contaminant; 575 exhibit homologous DNA sequence in PA42 but without shared flanking genes; and 2402 are not detected at the DNA level in PA42 (Table S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1/FileS4.docx">File S4</a>).</p><p id="p-34">Of the 4918 annotated TCO orphan genes with homologous DNA in PA42, 65% exhibit &gt;95% nucleotide sequence identity and 85% exhibit &gt;90% identity. Yet, despite this clear orthology, none of these TCO orphans are annotated in PA42; this is because 75% are without any mRNA support and 48% contain premature stop codons. To determine whether the subset of TCO orphans with orthologous DNA in other <em>D. pulex</em> are under selection (as would be expected for genes with functional significance), we estimated the average <em>K</em><sub>a</sub>/<em>K</em><sub>s</sub> against PA42 and the 11 other <em>D. pulex</em> genomes. The distribution of average <em>K</em><sub>a</sub>/<em>K</em><sub>s</sub> for these genes is quite broad, with a mean of 0.57, which is significantly greater than that for 1:1 TCO-PA42 orthologs (Figure S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1/FileS4.docx">File S4</a>).</p><p id="p-35">Several additional lines of evidence highlight the unusual nature of these TCO orphan genes. For example, by contrasting the TCO-specific orthologous clusters with the 1:1 TCO-PA42 orthologs, the former are found to have a much higher likelihood of harboring zero to one introns (30% being intron-free), in contrast to the situation for the shared orthologs (only 2% of which are intron-free) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). In one of the largest TCO-specific orthology clusters, 147 genes are located on 106 different scaffolds, with only 19 of them having mRNA support, and 82 having zero and 37 having just one annotated intron, far below the genome-wide average of 5.9 introns/gene in TCO (obtained when these aberrant 7320 genes are excluded). The orphan genes in the TCO-specific clusters are also abnormally short (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>), have significantly lower GC content (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>) and higher coverage (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2D</a>) than the 1:1 TCO-PA42 orthologs. Finally, mapping of EST data from TCO (<a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">Colbourne <em>et al.</em> 2011</a>) shows that only 736 of the 7320 TCO-specific genes have &gt;200 bp covered by such reads; and further results on transcription initiation sites and promoter regions resulting from CAGE data in PA42 (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Raborn <em>et al.</em> 2016</a>) predict that no more than 143 of these genes are expressed (Table S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1/FileS4.docx">File S4</a>). These analyses indicate that the excess gene count in the TCO genome is largely a consequence of false positives resulting from overly aggressive gene annotation, while also indicating that the TCO genome harbors a substantial amount of nonfunctional DNA not present in PA42.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1405/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of the features of 1:1 TCO&#x2013;PA42 orthologs and TCO-specific genes. TCO-specific genes have no obvious orthologs in PA42 or the Reference Species (other metazoans). (A) Distributions of intron numbers. (B) Length distributions for all coding sequences (excluding introns). (C) Comparisons of average GC contents. (D) Comparisons of average sequence coverages. Error bars indicate SEs. Asterisk denotes significance at the P &lt; 0.05 level." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1066862039" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of the features of 1:1 TCO&#x2013;PA42 orthologs and TCO-specific genes. TCO-specific genes have no obvious orthologs in PA42 or the Reference Species (other metazoans). (A) Distributions of intron numbers. (B) Length distributions for all coding sequences (excluding introns). (C) Comparisons of average GC contents. (D) Comparisons of average sequence coverages. Error bars indicate SEs. Asterisk denotes significance at the &lt;em&gt;P&lt;/em&gt; &lt; 0.05 level.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1405/F2.medium.gif" width="440" height="337"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/5/1405/F2.medium.gif" width="440" height="337"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1405/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1405/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484321" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-36" class="first-child">Comparison of the features of 1:1 TCO–PA42 orthologs and TCO-specific genes. TCO-specific genes have no obvious orthologs in PA42 or the Reference Species (other metazoans). (A) Distributions of intron numbers. (B) Length distributions for all coding sequences (excluding introns). (C) Comparisons of average GC contents. (D) Comparisons of average sequence coverages. Error bars indicate SEs. Asterisk denotes significance at the <em>P</em> &lt; 0.05 level.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">Finally, confining our attention to the sets of nonorphan PA42 and TCO genes (<em>i.e.</em>, those with orthologs in other metazoan genomes or shared with each other), we evaluated the age distributions of paralogous genes using the magnitude of synonymous-site divergence (corrected for multiple mutations per site) as a measure of age. In TCO, we observed an excess number of duplicate genes having a distance &gt;0.0575 (the average silent-site divergence between orthologs across these two clones) compared to PA42 (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> and Figure S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1/FileS4.docx">File S4</a>). If these genes truly exist in TCO, we would expect to see many of them in PA42 due to the fact that they have larger divergence than the orthologs across these two clones. Yet the substantial excess of such duplicate genes annotated in TCO with divergence levels 0.10–0.35 appears to be too high even for sampling error in the latter. This surplus of intermediate-aged duplicate genes in TCO may reflect the fact that they have been lost in PA42, or they may be an artifact of assembly/annotation errors in TCO. On the other hand, the youngest age class of gene duplicates exhibits an excess abundance in PA42.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1405/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The frequency distribution of KS for paralogs within the PA42 genome vs. those within TCO. Only genes in orthologous clusters containing both TCO and PA42 genes were used. Sliding-window analyses were used to remove the low quality regions in the alignments, with a cutoff of 0.4 identity for each 15-bp window. The KS value for each paralog is the average value when comparing the paralog with others in an orthologous cluster of the genome. The vertical dashed line at 0.057 denotes the average silent-site divergence for pairs of TCO-PA42 orthologs, so that paralogous pairs to the left of this benchmark are younger than the average ortholog divergence between these two clones. KS, pairwise divergence at silent sites." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1066862039" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The frequency distribution of &lt;em&gt;K&lt;/em&gt;&lt;sub&gt;S&lt;/sub&gt; for paralogs within the PA42 genome &lt;em&gt;vs.&lt;/em&gt; those within TCO. Only genes in orthologous clusters containing both TCO and PA42 genes were used. Sliding-window analyses were used to remove the low quality regions in the alignments, with a cutoff of 0.4 identity for each 15-bp window. The &lt;em&gt;K&lt;/em&gt;&lt;sub&gt;S&lt;/sub&gt; value for each paralog is the average value when comparing the paralog with others in an orthologous cluster of the genome. The vertical dashed line at 0.057 denotes the average silent-site divergence for pairs of TCO-PA42 orthologs, so that paralogous pairs to the left of this benchmark are younger than the average ortholog divergence between these two clones. &lt;em&gt;K&lt;/em&gt;&lt;sub&gt;S&lt;/sub&gt;, pairwise divergence at silent sites.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1405/F3.medium.gif" width="440" height="401"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/5/1405/F3.medium.gif" width="440" height="401"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1405/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1405/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484395" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-38" class="first-child">The frequency distribution of <em>K</em><sub>S</sub> for paralogs within the PA42 genome <em>vs.</em> those within TCO. Only genes in orthologous clusters containing both TCO and PA42 genes were used. Sliding-window analyses were used to remove the low quality regions in the alignments, with a cutoff of 0.4 identity for each 15-bp window. The <em>K</em><sub>S</sub> value for each paralog is the average value when comparing the paralog with others in an orthologous cluster of the genome. The vertical dashed line at 0.057 denotes the average silent-site divergence for pairs of TCO-PA42 orthologs, so that paralogous pairs to the left of this benchmark are younger than the average ortholog divergence between these two clones. <em>K</em><sub>S</sub>, pairwise divergence at silent sites.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">To determine the impact of annotation errors on the TCO duplicate genes, we extracted all paralogous pairs within TCO and within PA42 with a silent-site distance &gt;0.0575. There are many cases involving such duplicates where PA42 and TCO have the same numbers of copies in the paralogous cluster, and these most likely represent true paralogs between the two clones. After excluding these cases, there are 950 cases of genes in PA42 <em>vs.</em> 1724 in TCO with different numbers of copies in the paralogous clusters of annotated genes. In total, only 476 of the 1724 genes in TCO are supported by EST and/or promoter data, whereas in PA42, 944 out of 950 are corroborated by RNA-seq (Table S8 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1/FileS4.docx">File S4</a>). This suggests that the annotated paralogous genes in PA42 are most likely correct, while a substantial number of those annotated in TCO may derive from false annotation. For the remaining 1248 (1724−476) paralogous genes within TCO with no transcriptomic support, a substantial proportion appear to be problematic: 320 of them are defined as absent from PA42 due to the lack of homologous DNA sequence, whereas 306 contain premature stop codons and/or have <em>K</em><sub>a</sub>/<em>K</em><sub>s</sub> ≥ 1.0 in PA42, and 72 do not fit a gene model in PA42.</p></div><div id="sec-12" class="subsection"><h3>Reduced effective population size and genome architecture</h3><p id="p-40">As noted above, the TCO source population is thought to have a substantially reduced effective population size relative to populations in the geographic region from which PA42 is derived. If prolonged enough, by increasing the power of genetic drift relative to natural selection, such a shift in the population-genetic environment is expected to enhance the chances of accumulation of mildly deleterious mutations and to facilitate the colonization and/or proliferation of exogenous elements such as TEs, introns, and other genomic insertions that impose slightly deleterious effects (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Lynch 2007</a>).</p><p id="p-41">To evaluate the degree to which patterns of molecular evolution differ between the PA42 and TCO genomes, we used the relative-rate test (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Sarich and Wilson 1967</a>; <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Wu and Li 1985</a>) to compare the substitution rates for silent and replacement sites, <em>K</em><sub>s</sub> and <em>K</em><sub>a</sub>, respectively, using sequence from <em>D. obtusa</em> as an outgroup (<a id="xref-ref-61-4" class="xref-bibr" href="#ref-61">Tucker <em>et al.</em> 2013</a>). The difference between the TCO/<em>D</em>. <em>obtusa</em> distance and the PA42/<em>D</em>. <em>obtusa</em> distance provides a measure of the degree of cumulative divergence on the TCO <em>vs.</em> the PA42 branch on this three-species tree. Averaging over all compared genes, the <em>K</em><sub>s</sub> difference is 0.0061 ± 0.0003 (mean ± SE), and the <em>K</em><sub>a</sub> difference is 0.0057 ± 0.0001. Both <em>K</em><sub>s</sub> and <em>K</em><sub>a</sub> are elevated in TCO, while the distribution for <em>K</em><sub>a</sub> is much more strongly asymmetric toward positive values (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Thus, there is either accumulation of deleterious variation or increased substitution rates in TCO.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1405/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distributions of the synonymous (Ks) and nonsynonymous (Ka) differences per site per gene in TCO and PA42, using D. obtusa as outgroup. Ks and Ka were first calculated in TCO vs. D. obtusa and PA42 vs. D. obtusa, with the plotted difference providing a measure of the increase in the subtending branch length for TCO above that for the PA42 lineage." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1066862039" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distributions of the synonymous (&lt;em&gt;K&lt;/em&gt;&lt;sub&gt;s&lt;/sub&gt;) and nonsynonymous (&lt;em&gt;K&lt;/em&gt;&lt;sub&gt;a&lt;/sub&gt;) differences per site per gene in TCO and PA42, using &lt;em&gt;D. obtusa&lt;/em&gt; as outgroup. &lt;em&gt;K&lt;/em&gt;&lt;sub&gt;s&lt;/sub&gt; and &lt;em&gt;K&lt;/em&gt;&lt;sub&gt;a&lt;/sub&gt; were first calculated in TCO &lt;em&gt;vs.&lt;/em&gt; &lt;em&gt;D. obtusa&lt;/em&gt; and PA42 &lt;em&gt;vs.&lt;/em&gt; &lt;em&gt;D. obtusa&lt;/em&gt;, with the plotted difference providing a measure of the increase in the subtending branch length for TCO above that for the PA42 lineage.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1405/F4.medium.gif" width="440" height="176"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/5/1405/F4.medium.gif" width="440" height="176"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1405/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1405/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484336" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-42" class="first-child">Distributions of the synonymous (<em>K</em><sub>s</sub>) and nonsynonymous (<em>K</em><sub>a</sub>) differences per site per gene in TCO and PA42, using <em>D. obtusa</em> as outgroup. <em>K</em><sub>s</sub> and <em>K</em><sub>a</sub> were first calculated in TCO <em>vs.</em> <em>D. obtusa</em> and PA42 <em>vs.</em> <em>D. obtusa</em>, with the plotted difference providing a measure of the increase in the subtending branch length for TCO above that for the PA42 lineage.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">There are several models showing that effective population size influences the insertion and deletion rates of TEs (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Deceliere <em>et al.</em> 2005</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Le Rouzic <em>et al.</em> 2007</a>). In general, larger effective population sizes lead to fewer TEs due to more effective purifying selection. Consistently, more TE copy numbers and higher TE allele frequencies are observed in self-fertilizing <em>Arabidopsis thaliana</em> and <em>Caenorhabditis</em> nematodes than their outcrossing relatives due to smaller population sizes (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dolgin <em>et al.</em> 2008</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lockton and Gaut 2010</a>). To understand whether the same pattern applies to <em>Daphnia</em>, we identified the TE content in PA42 using <em>de novo</em> searching software of a variety of types (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Rho <em>et al.</em> 2007</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Ellinghaus <em>et al.</em> 2008</a>; <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Rho and Tang 2009</a>), combined with a search for known homologs using RepeatMasker (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Smit <em>et al.</em> 2004</a>). In total, 530 full-length TEs and 7961 fragmental TEs were identified, constituting ∼7.4% of the assembled PA42 genome (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>), and partitioning into 6.6% retrotransposon- and 0.8% DNA transposon-associated DNA. In line with the idea that smaller <em>N</em><sub>e</sub> facilitates the spread of TEs, PA42 contains significantly less TE-associated DNA content than the 9.4% observed in TCO (<em>P</em> &lt; 0.05). To eliminate the impact of genome assemblies on TE contents, we mapped raw reads derived from TCO and PA42 clones to the TE library containing all the TEs from TCO and PA42 (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Jiang <em>et al.</em> 2017</a>). On average, ∼20% of the reads in clones derived from PA42 mapped to the TEs in the library, whereas ∼33.2% reads from clones derived from TCO mapped to TEs in the library (Table S9 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1/FileS4.docx">File S4</a>), indicating more TEs in the TCO genome. This difference is primarily caused by an inflated abundance of LTR retrotransposons in TCO (total genome content of 8.0% <em>vs.</em> 5.8% in PA42), which appears to be a product of relatively recent transpositions in TCO. This can be seen by comparing the age distributions of intact LTR elements (determined from the sequence divergence of the flanking LTRs, which are identical at birth), which show an elevation in the youngest class in the TCO genome (Figure S7 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1/FileS4.docx">File S4</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484342/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484342/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Summary of transposable elements in PA42 3.0</span><div class="sb-div caption-clear"></div></div></div><p id="p-45"><em>D. pulex</em> is known to exhibit high levels of intron proliferation, with the discovery of this deriving from observations on the TCO genome (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Omilian <em>et al.</em> 2008</a>; <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Li <em>et al.</em> 2009</a>, <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">2014</a>). To determine whether such activity is particularly pronounced in TCO, we conducted a parsimony-based analysis of intron gain and loss following <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Li <em>et al.</em> (2009</a>, <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">2014)</a>, using the flanking exon sequences of predicted introns in both PA42 and TCO to reciprocally blast each other and to query an outgroup species, <em>D. magna</em>, to identify intron-free alleles in these assemblies. RNA-seq/EST data were further used to validate the exclusion of introns during transcription.</p><p id="p-46">In total, 50,735 independent intron positions were used in this analysis, and 38,702 were verified using RNA-seq/EST data, among which 128 intron gain/loss events were detected. There were 29 cases in which both PA42 and TCO harbor an intron and <em>D. magna</em> does not, and for these cases we were unable to infer whether this represented an intron gain in <em>D. pulex</em> or loss in <em>D. magna</em>. For the remaining 99 events, we inferred that TCO has experienced 78% more intron gain events than PA42 (57 <em>vs.</em> 32; <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>), as expected if the reduced <em>N</em><sub>e</sub> of TCO has promoted the accumulation of newly gained introns by random drift (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Lynch 2002</a>). Although much less prevalent, intron loss is also overrepresented in TCO (10 <em>vs.</em> 0). The lengths of newly gained and lost introns are very similar in both genomes (65–75 bp, <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>), and very similar to the median intron lengths of 63 and 77 bp in TCO and PA42, respectively.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484330/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484330/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Summary of intron gain and loss events in TCO and PA42</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Gene family analysis</h3><p id="p-48">In the aforementioned analyses using OrthoMCL, 19,269 orthologous gene clusters among PA42, TCO, and the Reference Species were identified and assigned to SCOP families (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Murzin <em>et al.</em> 1995</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Gough <em>et al.</em> 2001</a>). In principle, each SCOP family contains protein domains descendent from a common ancestor, and the orthologous clusters containing <em>Daphnia</em> genes are present in 2506 such gene families.</p><p id="p-49">The gene family modeling pipeline CAFE 3.0 (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Han <em>et al.</em> 2013</a>) was used to infer expansion and/or contraction in specific gene families in PA42, TCO, and the Reference Species. Across the phylogeny, most species have higher numbers of gene family contractions than expansions, with TCO and the lineage leading to the common ancestor of TCO and PA42 being the exceptions (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). This high degree of gene family expansion is consistent with the elevated number of gene gains and retention previously inferred in the TCO genome (<a id="xref-ref-7-7" class="xref-bibr" href="#ref-7">Colbourne <em>et al.</em> 2011</a>). However, the PA42 genome shows a substantially different pattern, which is much more similar to that in other arthropod lineages. TCO has more gene family expansions and also more genes in a larger portion of families. In 42% of gene families, PA42 and TCO have the same number of genes, but in 40% of the families, the annotated gene number in TCO exceeds that in PA42 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1/FileS2.xlsx">File S2</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1405/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dynamic evolution of gene families. The gene family expansions and contractions were predicted by CAF&#xC9; 3.0. The species tree required by CAF&#xC9; 3.0 was constructed by 1:1:1 single-copy gene families using the Maximum Likelihood method in MEGA6 (Tamura et al. 2013). The RelTime-ML program implemented in the MEGA6 package was used to estimate divergence time among species; calibration time was obtained from the TimeTree database." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1066862039" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Dynamic evolution of gene families. The gene family expansions and contractions were predicted by CAF&#xC9; 3.0. The species tree required by CAF&#xC9; 3.0 was constructed by 1:1:1 single-copy gene families using the Maximum Likelihood method in MEGA6 (Tamura &lt;em&gt;et al.&lt;/em&gt; 2013). The RelTime-ML program implemented in the MEGA6 package was used to estimate divergence time among species; calibration time was obtained from the TimeTree database.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1405/F5.medium.gif" width="440" height="236"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/5/1405/F5.medium.gif" width="440" height="236"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1405/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1405/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484371" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-50" class="first-child">Dynamic evolution of gene families. The gene family expansions and contractions were predicted by CAFÉ 3.0. The species tree required by CAFÉ 3.0 was constructed by 1:1:1 single-copy gene families using the Maximum Likelihood method in MEGA6 (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Tamura <em>et al.</em> 2013</a>). The RelTime-ML program implemented in the MEGA6 package was used to estimate divergence time among species; calibration time was obtained from the TimeTree database.</p><div class="sb-div caption-clear"></div></div></div><p id="p-51">Due to the fact that TCO harbors a large number of orphan genes and has an excess of paralogs that may bias the pattern of gene family expansion/contraction, we only focused on the gene family expansion/contraction pattern in PA42. We found 29 gene families to be significantly expanded (<em>P</em> &lt; 0.01) in the PA42 lineage (Table S10 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1/FileS4.docx">File S4</a>). To further understand the functional enrichment of genes in expanded PA42 gene families, we assigned all of the genes in the expanded PA42 families into functional GO categories. We found that such expansions are mainly enriched in categories such as chitin metabolic process, response to oxidative stress, and protein glycosylation (Table S11 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1/FileS4.docx">File S4</a>). We hope that these results will stimulate future research on the role that these gene families play in the functional biology of <em>D. pulex</em>.</p></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><p id="p-52">The development of the first <em>Daphnia</em> genome assembly (TCO) greatly advanced <em>D. pulex</em> as an important model system for evolutionary, ecological, and environmental studies. The full suite of genomic tools have allowed researchers to investigate the genetic basis of phenotypic traits (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Schumpert <em>et al.</em> 2015</a>), the origin of obligate parthenogenesis (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Xu <em>et al.</em> 2015b</a>), environmental sex determination (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kato <em>et al.</em> 2011</a>), and genetic response to a wide range of environmental stressors including salinity (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Latta <em>et al.</em> 2012</a>), toxic cyanobacteria (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Asselman <em>et al.</em> 2015</a>), and heavy metals (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chen <em>et al.</em> 2015</a>). However, previous phylogenetic studies have illuminated the presence of multiple divergent lineages in the <em>D. pulex</em> complex (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Colbourne <em>et al.</em> 1998</a>), and this complicated evolutionary history raises questions about the suitability of the existing TCO genome as a reference for studies involving <em>D. pulex</em> isolates sampled outside of OR. Thus, a major motivation of the current study has been to develop a genomic assembly that will facilitate future studies employing animals derived from the geographically widespread lineage <em>D. pulex</em>.</p><p id="p-53">To achieve this goal, we established a high quality draft assembly of PA42, a <em>D. pulex</em> isolate sampled from a typical woodland pond in the midwest of the USA. We designed our genomic sequencing experiments using short insert-sized libraries and long jumping libraries, which are suitable for <em>de novo</em> assembly using AllPathsLG, a leading <em>de novo</em> genome assembler using short reads. To reduce the shortcomings of a short read-based assembly, the initial assembly was further improved using 8–10 kb reads generated via Illumina SLRs. Because bacterial DNA contamination can be problematic in the genomic assembly of small organisms, we used independently derived whole-genome sequences and single-sperm sequences of PA42 to remove scaffolds with suspicious contaminant DNA. Finally, because the transcriptome of <em>Daphnia</em> might vary substantially between the sexes and under different environmental conditions, to aid in the PA42 gene annotation, we performed RNA-seq for different male and female life stages and in a diverse array of environmental treatments.</p><p id="p-54">Our efforts to acquire an improved <em>D. pulex</em> genome sequence resource have proven successful, with the PA42 assembly having a scaffold N50 of 494 kb and 96% of essential eukaryotic single-copy genes appearing in the final product. Although the reduced number of annotated genes in PA42 relative to the earlier derived TCO genome is troublesome, our comprehensive analyses comparing these two draft genomes reveals a number of the sources of erroneous annotation in TCO genome. In TCO, 7320 genes do not have orthologs in PA42: 592 are not covered by any read derived from subsequent TCO mutation-accumulation lines (<em>i.e.</em>, appear to be contamination from exogenous DNA), and 2402 are not detected at the DNA level in PA42. Of the remaining TCO-specific genes that do have homologous DNA in PA42, although 85% exhibit &gt;90% sequence identity across the two genomes, none are annotated in PA42; half of them contain premature stop codons, and their average <em>K</em><sub>a</sub>/<em>K</em><sub>s</sub> is significantly greater than that for 1:1 TCO-PA42 orthologs. In addition to 7320 TCO-specific genes, there are 698 more annotated paralogous genes in TCO relative to PA42, with a silent-site divergence larger than the average divergence of 1:1 TCO-PA42 orthologs and a substantial proportion of these are caused by annotation errors. Therefore, we suggest that due caution should be taken in using the TCO genome annotation in future studies.</p><p id="p-55">We argue that the elevated gene number in TCO mainly comes from aggressive annotation strategies. To support our point of view, we checked the gene number in TCO when applying different annotation strategies. (1) The TCO gene annotation is based on various pieces of evidence including homology, <em>D. magna</em>, paralogy, and expressed genes (in total 26,649 genes), and there are 4040 additional genes in TCO without any comparative or empirical support. In contrast, in PA42, we did not use <em>D. magna</em> and paralogy as evidence to annotate genes. The quality of the <em>D. magna</em> genome is not verified; when contamination existed in <em>D. magna</em> genome and was annotated as genes, the same annotation error could be introduced to TCO. The evidence of paralogy should be used with caution, especially in a genome like TCO with a large amount of tandem gene clusters, where alleles could easily be separated and annotated as paralogs. Using the same evidence as we used in PA42, the total gene number in TCO will become 21,361. (2) In TCO, the authors annotated genes with one line of evidence or without any comparative or empirical support; however, in PA42 we were much more conserved and included genes with at least two lines of evidence. When applying our strategy to TCO, the gene number in TCO will be 19,662. If we apply the PA42 annotation strategies in both (1) and (2) to TCO, the total gene number in TCO should be &lt;19,662.</p><p id="p-56">After excluding erroneously annotated genes, there remain &gt;21,000 genes in the TCO genome (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1/FileS3.zip">File S3</a>), which is still higher than in the PA42 genome. Although further efforts are needed to achieving an accurately annotated TCO genome, we can still learn the potential impact of reduced effective population size on aspects of genomic architecture other than gene number. TE proliferation and intron gain and loss are thought to be strongly affected by effective population size (<a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Lynch 2007</a>). Most TEs are likely to be deleterious across the genome (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Tollis and Boissinot 2012</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Wei and Cao 2016</a>), but only those with selective disadvantages smaller than the power of random genetic drift are likely to increase in frequency. Consistent with the hypothesis that the TCO genome is contained within a lineage that has experienced a long-term population bottleneck, the proportion of the genome comprising TEs in TCO is higher than that in PA42 (9.40% <em>vs.</em> 7.36%). In addition, both the intron gain and loss ratios in TCO are much higher than in PA42, possibly due to the reduced effective population size of TCO promoting the fixation of newly gained introns by random drift (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Lynch 2002</a>).</p><p id="p-57">In conclusion, although the TCO genome is certainly of interest from an evolutionary genomics perspective, we argue that it is not a particularly good representative of the <em>Daphnia</em> genus, owing to both overly aggressive gene annotation and to evolutionary changes in genomic architectural features (<em>e.g.</em>, mobile elements and intron gain/loss) putatively resulting from reduced effective population size. Aside from the annotation issues, the TCO draft genome may still be reasonably suitable for studying the evolution of the <em>D. pulex</em> sublineage endemic to OR, whereas the new PA42 genome appears to be a more reliable assembly for examining genomic evolutionary features of <em>D. pulex</em> in the remainder of North America.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-58">We thank Kim Young for maintaining <em>Daphnia</em> cultures and Illumina, Inc. for assistance with the Long Synthetic Read methodology. This work is supported by National Institutes of Health grant R01GM101672 to M.L. J.A. was the recipient of a Belgian American Education Foundation fellowship for research. The authors confirm that there are no known conflicts of interest associated with this publication.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-9"><p id="p-59"><em>Note added in proof:</em> See Maruki and Lynch 2017 (pp. <a href="http://www.g3journal.org/content/7/5/1393.full">1393</a>) in this issue and Ackerman <em>et al</em>. 2017 (pp. <a href="http://www.genetics.org/content/206/1/105.full">105</a>) and Lynch <em>et al</em>. 2017 (pp. <a href="http://www.genetics.org/content/206/1/315.full">315</a>) in the GENETICS May issue for related work.</p></li><li class="fn" id="fn-10"><p id="p-60">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038638/-/DC1</a>.</p></li><li class="fn" id="fn-11"><p id="p-61"><em>Communicating editor: J. M. Comeron</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-12-21"><span class="received-label">Received </span>December 21, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-02-17"><span class="accepted-label">Accepted </span>February 17, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Ye <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.1" data-doi="10.1111/mec.13148"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Asselman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfrender</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopez</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Coninck</span>  <span class="cit-name-given-names">D. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janssen</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Conserved transcriptional responses to cyanobacterial stressors are mediated by alternate regulation of paralogous genes in <em>Daphnia</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1844</span>–<span class="cit-lpage">1855</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D24%26rft.spage%253D1844%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13148%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13148&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bergman</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfeiffer</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rincón-Limas</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoskins</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gnirke</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Assessing the impact of comparative genomic sequence data on the functional annotation of the <em>Drosophila</em> genome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">RESEARCH0086</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DBergman%26rft.auinit1%253DC.%2BM.%26rft.volume%253D3%26rft.issue%253D12%26rft.spage%253DRESEARCH0086%26rft.epage%253DRESEARCH0086%26rft.atitle%253DAssessing%2Bthe%2Bimpact%2Bof%2Bcomparative%2Bgenomic%2Bsequence%2Bdata%2Bon%2Bthe%2Bfunctional%2Bannotation%2Bof%2Bthe%2BDrosophila%2Bgenome.%26rft_id%253Dinfo%253Apmid%252F12537575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12537575&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.3" data-doi="10.1186/1471-2105-15-211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boetzer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pirovano</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">SSPACE-LongRead: scaffolding bacterial draft genomes using long read sequence information.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D15%26rft.spage%253D211%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-15-211%26rft_id%253Dinfo%253Apmid%252F24950923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-15-211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24950923&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.4" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usadel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMzAvMTUvMjExNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKinney</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nichols</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colbourne</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sepulveda</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Novel cadmium responsive microRNAs in <em>Daphnia pulex</em>.</span> <abbr class="cit-jnl-abbrev">Environ. Sci. Technol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">14605</span>–<span class="cit-lpage">14613</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnviron.%2BSci.%2BTechnol.%26rft.volume%253D49%26rft.spage%253D14605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Colbourne</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crease</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weider</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hebert</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duferesne</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Phylogenetics and evolution of a circumarctic species complex (Cladocera: <em>Daphnia pulex</em>).</span> <abbr class="cit-jnl-abbrev">Biol. J. Linn. Soc. Lond.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiological%2BJournal%2Bof%2Bthe%2BLinnean%2BSociety%26rft.stitle%253DBiological%2BJournal%2Bof%2Bthe%2BLinnean%2BSociety%26rft.volume%253D65%26rft.issue%253D3%26rft.spage%253D347%26rft.epage%253D365%26rft.atitle%253DPhylogenetics%2Band%2Bevolution%2Bof%2Ba%2Bcircumarctic%2Bspecies%2Bcomplex%2B%2528Cladocera%253B%2BDaphnia%2Bpulex%2529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=1999027354&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=000077338700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.7" data-doi="10.1126/science.1197761"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Colbourne</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfrender</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">W. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The ecoresponsive genome of <em>Daphnia pulex</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">331</span>: <span class="cit-fpage">555</span>–<span class="cit-lpage">561</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DColbourne%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D331%26rft.issue%253D6017%26rft.spage%253D555%26rft.epage%253D561%26rft.atitle%253DThe%2BEcoresponsive%2BGenome%2Bof%2BDaphnia%2Bpulex%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1197761%26rft_id%253Dinfo%253Apmid%252F21292972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzEvNjAxNy81NTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.8" data-doi="10.1038/sj.hdy.6882220"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crease</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spitze</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehman</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Allozyme and mtDNA variation in populations of the <em>Daphnia pulex</em> complex from both sides of the Rocky Mountains.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">242</span>–<span class="cit-lpage">251</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D79%26rft.spage%253D242%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6882220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6882220&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997XX61100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.9" data-doi="10.1534/genetics.104.032243"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deceliere</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charles</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biémont</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The dynamics of transposable elements in structured populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">467</span>–<span class="cit-lpage">474</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.032243%26rft_id%253Dinfo%253Apmid%252F15466430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY5LzEvNDY3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.10" data-doi="10.1371/journal.pcbi.1003998"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Denton</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lugo-Martinez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warren</span>  <span class="cit-name-given-names">W. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Extensive error in the number of genes inferred from draft genome assemblies.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1003998</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D10%26rft.spage%253De1003998%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1003998%26rft_id%253Dinfo%253Apmid%252F25474019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1003998&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25474019&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.11" data-doi="10.1017/S0016672308009440"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dolgin</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cutter</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Population frequencies of transposable elements in selfing and outcrossing <em>Caenorhabditis</em> nematodes.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">317</span>–<span class="cit-lpage">329</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DDolgin%26rft.auinit1%253DE.%2BS.%26rft.volume%253D90%26rft.issue%253D4%26rft.spage%253D317%26rft.epage%253D329%26rft.atitle%253DPopulation%2Bfrequencies%2Bof%2Btransposable%2Belements%2Bin%2Bselfing%2Band%2Boutcrossing%2BCaenorhabditis%2Bnematodes.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672308009440%26rft_id%253Dinfo%253Apmid%252F18840306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672308009440&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18840306&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260655800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.12" data-doi="10.1016/S0277-3791(01)00095-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dyke</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andrews</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">P. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">England</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The Laurentide and Innuitian ice sheets during the last glacial maximum.</span> <abbr class="cit-jnl-abbrev">Quat. Sci. Rev.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuat.%2BSci.%2BRev.%26rft.volume%253D21%26rft.spage%253D9%26rft.atitle%253DQUAT%2BSCI%2BREV%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0277-3791%252801%252900095-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0277-3791(01)00095-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.13" data-doi="10.1186/1471-2105-9-18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellinghaus</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurtz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willhoeft</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">LTRharvest, an efficient and flexible software for <em>de novo</em> detection of LTR retrotransposons.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DEllinghaus%26rft.auinit1%253DD.%26rft.volume%253D9%26rft.spage%253D18%26rft.epage%253D18%26rft.atitle%253DLTRharvest%252C%2Ban%2Befficient%2Band%2Bflexible%2Bsoftware%2Bfor%2Bde%2Bnovo%2Bdetection%2Bof%2BLTR%2Bretrotransposons.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-9-18%26rft_id%253Dinfo%253Apmid%252F18194517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-9-18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18194517&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.14" data-doi="10.1371/journal.pone.0047768"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">English</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vee</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mind the gap: upgrading genomes with Pacific Biosciences RS long-read sequencing technology.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e47768</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DEnglish%26rft.auinit1%253DA.%2BC.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De47768%26rft.epage%253De47768%26rft.atitle%253DMind%2Bthe%2BGap%253A%2BUpgrading%2BGenomes%2Bwith%2BPacific%2BBiosciences%2BRS%2BLong-Read%2BSequencing%2BTechnology.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0047768%26rft_id%253Dinfo%253Apmid%252F23185243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0047768&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23185243&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.15" data-doi="10.1093/nar/gkt1223"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finn</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bateman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clements</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coggill</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eberhardt</span>  <span class="cit-name-given-names">R. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Pfam: the protein families database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">D222</span>–<span class="cit-lpage">D230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1223%26rft_id%253Dinfo%253Apmid%252F24288371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0Mi9EMS9EMjIyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.16" data-doi="10.1093/bioinformatics/bts565"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">CD-HIT: accelerated for clustering the next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">3150</span>–<span class="cit-lpage">3152</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts565%26rft_id%253Dinfo%253Apmid%252F23060610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMjMvMzE1MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.17" data-doi="10.1073/pnas.1017351108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gnerre</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacCallum</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przybylski</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ribeiro</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">High-quality draft assemblies of mammalian genomes from massively parallel sequence data.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">1513</span>–<span class="cit-lpage">1518</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1017351108%26rft_id%253Dinfo%253Apmid%252F21187386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA4LzQvMTUxMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.18" data-doi="10.1093/nar/gkn176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Götz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">García-Gómez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terol</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagaraj</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">High-throughput functional annotation and data mining with the Blast2GO suite.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">3420</span>–<span class="cit-lpage">3435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn176%26rft_id%253Dinfo%253Apmid%252F18445632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzNi8xMC8zNDIwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.19" data-doi="10.1006/jmbi.2001.5080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gough</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karplus</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughey</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chothia</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Assignment of homology to genome sequences using a library of hidden Markov models that represent all proteins of known structure.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">313</span>: <span class="cit-fpage">903</span>–<span class="cit-lpage">919</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DGough%26rft.auinit1%253DJ.%26rft.volume%253D313%26rft.issue%253D4%26rft.spage%253D903%26rft.epage%253D919%26rft.atitle%253DAssignment%2Bof%2Bhomology%2Bto%2Bgenome%2Bsequences%2Busing%2Ba%2Blibrary%2Bof%2Bhidden%2BMarkov%2Bmodels%2Bthat%2Brepresent%2Ball%2Bproteins%2Bof%2Bknown%2Bstructure.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.2001.5080%26rft_id%253Dinfo%253Apmid%252F11697912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.2001.5080&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11697912&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172069600017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.20" data-doi="10.1038/nbt.1883"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">644</span>–<span class="cit-lpage">652</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DGrabherr%26rft.auinit1%253DM.%2BG.%26rft.volume%253D29%26rft.issue%253D7%26rft.spage%253D644%26rft.epage%253D652%26rft.atitle%253DFull-length%2Btranscriptome%2Bassembly%2Bfrom%2BRNA-Seq%2Bdata%2Bwithout%2Ba%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1883%26rft_id%253Dinfo%253Apmid%252F21572440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1883&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21572440&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.21" data-doi="10.1093/molbev/mst100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lugo-Martinez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Estimating gene gain and loss rates in the presence of error in genome assembly and annotation using CAFE 3.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1987</span>–<span class="cit-lpage">1997</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst100%26rft_id%253Dinfo%253Apmid%252F23709260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjMwLzgvMTk4NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.22" data-doi="10.1186/1471-2105-12-491"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yandell</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">MAKER2: an annotation pipeline and genome-database management tool for second-generation genome projects.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHolt%26rft.auinit1%253DC.%26rft.volume%253D12%26rft.spage%253D491%26rft.epage%253D491%26rft.atitle%253DMAKER2%253A%2Ban%2Bannotation%2Bpipeline%2Band%2Bgenome-database%2Bmanagement%2Btool%2Bfor%2Bsecond-generation%2Bgenome%2Bprojects.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-491%26rft_id%253Dinfo%253Apmid%252F22192575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-491&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22192575&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Insertion polymorphisms of mobile elements in sexual and asexual populations of <em>Daphnia pulex</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">9</span>.: 362–374</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.24" data-doi="10.1093/bioinformatics/btu031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Binns</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">InterProScan 5: genome-scale protein function classification.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1236</span>–<span class="cit-lpage">1240</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu031%26rft_id%253Dinfo%253Apmid%252F24451626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIzMC85LzEyMzYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.25" data-doi="10.1371/journal.pgen.1001345"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kato</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iguchi</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Environmental sex determination in the branchiopod crustacean <em>Daphnia magna</em>: deep conservation of a <em>Doublesex</em> gene in the sex-determining pathway.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1001345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKato%26rft.auinit1%253DY.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253De1001345%26rft.epage%253De1001345%26rft.atitle%253DEnvironmental%2Bsex%2Bdetermination%2Bin%2Bthe%2Bbranchiopod%2Bcrustacean%2BDaphnia%2Bmagna%253A%2Bdeep%2Bconservation%2Bof%2Ba%2BDoublesex%2Bgene%2Bin%2Bthe%2Bsex-determining%2Bpathway.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001345%26rft_id%253Dinfo%253Apmid%252F21455482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001345&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21455482&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.26" data-doi="10.1093/nar/gki198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuma</span>  <span class="cit-name-given-names">K.-i.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toh</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miyata</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">MAFFT version 5: improvement in accuracy of multiple sequence alignment.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">511</span>–<span class="cit-lpage">518</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DKatoh%26rft.auinit1%253DK.%26rft.volume%253D33%26rft.issue%253D2%26rft.spage%253D511%26rft.epage%253D518%26rft.atitle%253DMAFFT%2Bversion%2B5%253A%2Bimprovement%2Bin%2Baccuracy%2Bof%2Bmultiple%2Bsequence%2Balignment%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki198%26rft_id%253Dinfo%253Apmid%252F15661851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjMzLzIvNTExIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.27" data-doi="10.1101/gr.191338.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keith</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sung</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lledó</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">High mutational rates of large-scale duplication and deletion in <em>Daphnia pulex</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">60</span>–<span class="cit-lpage">69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.191338.115%26rft_id%253Dinfo%253Apmid%252F26518480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjI2LzEvNjAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.28" data-doi="10.1186/1471-2105-5-59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korf</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Gene finding in novel genomes.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DKorf%26rft.auinit1%253DI.%26rft.volume%253D5%26rft.spage%253D59%26rft.epage%253D59%26rft.atitle%253DGene%2Bfinding%2Bin%2Bnovel%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-5-59%26rft_id%253Dinfo%253Apmid%252F15144565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-5-59&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15144565&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.29" data-doi="10.1111/j.1461-0248.2012.01799.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Latta</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weider</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colbourne</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfrender</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The evolution of salinity tolerance in <em>Daphnia</em>: a functional genomics approach.</span> <abbr class="cit-jnl-abbrev">Ecol. Lett.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">794</span>–<span class="cit-lpage">802</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DLatta%26rft.auinit1%253DL.%2BC.%26rft.volume%253D15%26rft.issue%253D8%26rft.spage%253D794%26rft.epage%253D802%26rft.atitle%253DThe%2Bevolution%2Bof%2Bsalinity%2Btolerance%2Bin%2BDaphnia%253A%2Ba%2Bfunctional%2Bgenomics%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2012.01799.x%26rft_id%253Dinfo%253Apmid%252F22583985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2012.01799.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22583985&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.30" data-doi="10.1093/bioinformatics/btt702"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leggett</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clavijo</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clissold</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caccamo</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">NextClip: an analysis and read preparation tool for Nextera Long Mate Pair libraries.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">566</span>–<span class="cit-lpage">568</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt702%26rft_id%253Dinfo%253Apmid%252F24297520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIzMC80LzU2NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.31" data-doi="10.1073/pnas.0705238104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Le Rouzic</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boutin</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Capy</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Long-term evolution of transposable elements.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">19375</span>–<span class="cit-lpage">19380</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0705238104%26rft_id%253Dinfo%253Apmid%252F18040048%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzQ5LzE5Mzc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.32" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows–Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.33" data-doi="10.1101/gr.1224503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoeckert</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roos</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">OrthoMCL: identification of ortholog groups for eukaryotic genomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2178</span>–<span class="cit-lpage">2189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DLi%26rft.auinit1%253DL.%26rft.volume%253D13%26rft.issue%253D9%26rft.spage%253D2178%26rft.epage%253D2189%26rft.atitle%253DOrthoMCL%253A%2BIdentification%2Bof%2BOrtholog%2BGroups%2Bfor%2BEukaryotic%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1224503%26rft_id%253Dinfo%253Apmid%252F12952885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjEzLzkvMjE3OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.34" data-doi="10.1126/science.1179302"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sung</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">W. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Extensive, recent intron gains in <em>Daphnia</em> populations.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">326</span>: <span class="cit-fpage">1260</span>–<span class="cit-lpage">1262</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLi%26rft.auinit1%253DW.%26rft.volume%253D326%26rft.issue%253D5957%26rft.spage%253D1260%26rft.epage%253D1262%26rft.atitle%253DExtensive%252C%2BRecent%2BIntron%2BGains%2Bin%2BDaphnia%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1179302%26rft_id%253Dinfo%253Apmid%252F19965475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjYvNTk1Ny8xMjYwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.35" data-doi="10.1093/gbe/evu174"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuzoff</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Characterization of newly gained introns in <em>Daphnia</em> populations.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2218</span>–<span class="cit-lpage">2234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevu174%26rft_id%253Dinfo%253Apmid%252F25123113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjg6IjYvOS8yMjE4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.36" data-doi="10.1186/1471-2148-10-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lockton</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The evolution of transposable elements in natural populations of self-fertilizing <em>Arabidopsis thaliana</em> and its outcrossing relative <em>Arabidopsis lyrata</em>.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DLockton%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.spage%253D10%26rft.epage%253D10%26rft.atitle%253DThe%2Bevolution%2Bof%2Btransposable%2Belements%2Bin%2Bnatural%2Bpopulations%2Bof%2Bself-fertilizing%2BArabidopsis%2Bthaliana%2Band%2Bits%2Boutcrossing%2Brelative%2BArabidopsis%2Blyrata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-10-10%26rft_id%253Dinfo%253Apmid%252F20067644%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-10-10&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20067644&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.37" data-doi="10.1073/pnas.0409137102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Löytynoja</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldman</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">An algorithm for progressive multiple alignment of sequences with insertions.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">10557</span>–<span class="cit-lpage">10562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0409137102%26rft_id%253Dinfo%253Apmid%252F16000407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzMwLzEwNTU3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.38" data-doi="10.1073/pnas.092595699"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Intron evolution as a population-genetic process.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">6118</span>–<span class="cit-lpage">6123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D99%26rft.issue%253D9%26rft.spage%253D6118%26rft.epage%253D6123%26rft.atitle%253DIntron%2Bevolution%2Bas%2Ba%2Bpopulation-genetic%2Bprocess%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.092595699%26rft_id%253Dinfo%253Apmid%252F11983904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5OS85LzYxMTgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-7.5.1405.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-source">The Origins of Genome Architecture</span>. <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.40" data-doi="10.2307/2640923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfrender</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spitze</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehman</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hicks</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The quantitative and molecular genetic architecture of a subdivided species.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">100</span>–<span class="cit-lpage">110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D53%26rft.spage%253D100%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2640923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2640923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000078927800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ackerman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spitze</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maruki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Population genomics of <em>Daphnia pulex</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">332</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D206%26rft.spage%253D315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.42" data-doi="10.1006/jmbi.1995.0159"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murzin</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brenner</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubbard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chothia</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">SCOP: a structural classification of proteins database for the investigation of sequences and structures.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">247</span>: <span class="cit-fpage">536</span>–<span class="cit-lpage">540</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DMurzin%26rft.auinit1%253DA.%2BG.%26rft.volume%253D247%26rft.issue%253D4%26rft.spage%253D536%26rft.epage%253D540%26rft.atitle%253DSCOP%253A%2Ba%2Bstructural%2Bclassification%2Bof%2Bproteins%2Bdatabase%2Bfor%2Bthe%2Binvestigation%2Bof%2Bsequences%2Band%2Bstructures.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1995.0159%26rft_id%253Dinfo%253Apmid%252F7723011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1995.0159&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7723011&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QQ85600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.43" data-doi="10.1073/pnas.76.10.5269"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Mathematical model for studying genetic variation in terms of restriction endonucleases.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">5269</span>–<span class="cit-lpage">5273</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DNei%26rft.auinit1%253DM.%26rft.volume%253D76%26rft.issue%253D10%26rft.spage%253D5269%26rft.epage%253D5273%26rft.atitle%253DMathematical%2Bmodel%2Bfor%2Bstudying%2Bgenetic%2Bvariation%2Bin%2Bterms%2Bof%2Brestriction%2Bendonucleases.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.76.10.5269%26rft_id%253Dinfo%253Apmid%252F291943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiNzYvMTAvNTI2OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nurk</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bankevich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antipov</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gurevich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korobeynikov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Assembling genomes and mini-metagenomes from highly chimeric reads.</span> <abbr class="cit-jnl-abbrev">Res. Comput. Mol. Biol.</abbr> <span class="cit-vol">7821</span>: <span class="cit-fpage">158</span>–<span class="cit-lpage">170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRes.%2BComput.%2BMol.%2BBiol.%26rft.volume%253D7821%26rft.spage%253D158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.45" data-doi="10.1093/bib/3.1.87"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Emboss opens up sequence analysis.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">91</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBriefings%2Bin%2BBioinformatics%26rft.stitle%253DBrief%2BBioinform%26rft.aulast%253DOlson%26rft.auinit1%253DS.%2BA.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D87%26rft.epage%253D91%26rft.atitle%253DEmboss%2Bopens%2Bup%2Bsequence%2Banalysis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252F3.1.87%26rft_id%253Dinfo%253Apmid%252F12002227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJiaWIiO3M6NToicmVzaWQiO3M6NjoiMy8xLzg3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.46" data-doi="10.1534/genetics.108.099549"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Omilian</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Patterns of intraspecific DNA variation in the <em>Daphnia</em> nuclear genome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">325</span>–<span class="cit-lpage">336</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.099549%26rft_id%253Dinfo%253Apmid%252F19304589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzI1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.47" data-doi="10.1093/molbev/msn164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Omilian</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scofield</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Intron presence-absence polymorphisms in <em>Daphnia</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2129</span>–<span class="cit-lpage">2139</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn164%26rft_id%253Dinfo%253Apmid%252F18667441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xMC8yMTI5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.48" data-doi="10.1093/bioinformatics/btm071"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parra</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradnam</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korf</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">CEGMA: a pipeline to accurately annotate core genes in eukaryotic genomes.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1061</span>–<span class="cit-lpage">1067</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm071%26rft_id%253Dinfo%253Apmid%252F17332020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyMy85LzEwNjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.49" data-doi="10.1093/molbev/msp077"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dehal</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arkin</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">FastTree: computing large minimum evolution trees with profiles instead of a distance matrix.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1641</span>–<span class="cit-lpage">1650</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp077%26rft_id%253Dinfo%253Apmid%252F19377059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI2LzcvMTY0MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.50" data-doi="10.1186/1471-2164-10-172"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nong</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Preston</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ben-Ami</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebert</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Comparative metagenomics of <em>Daphnia</em> symbionts.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DQi%26rft.auinit1%253DW.%26rft.volume%253D10%26rft.spage%253D172%26rft.epage%253D172%26rft.atitle%253DComparative%2Bmetagenomics%2Bof%2BDaphnia%2Bsymbionts.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-172%26rft_id%253Dinfo%253Apmid%252F19383155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-172&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19383155&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raborn</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spitze</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brendel</span>  <span class="cit-name-given-names">V. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Promoter architecture and sex-specific gene expression in <em>Daphnia</em> pulex.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">593</span>–<span class="cit-lpage">612</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D2%26rft.spage%253D593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.52" data-doi="10.1093/nar/gkp752"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rho</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">MGEScan-non-LTR: computational identification and classification of autonomous non-LTR retrotransposons in eukaryotic genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">e143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp752%26rft_id%253Dinfo%253Apmid%252F19762481%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzNy8yMS9lMTQzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.53" data-doi="10.1186/1471-2164-8-90"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rho</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">J.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title"><em>De novo</em> identification of LTR retrotransposons in eukaryotic genomes.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DRho%26rft.auinit1%253DM.%26rft.volume%253D8%26rft.spage%253D90%26rft.epage%253D90%26rft.atitle%253DDe%2Bnovo%2Bidentification%2Bof%2BLTR%2Bretrotransposons%2Bin%2Beukaryotic%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-8-90%26rft_id%253Dinfo%253Apmid%252F17407597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-8-90&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17407597&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.54" data-doi="10.1126/science.158.3805.1200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sarich</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">A. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1967</span> <span class="cit-article-title">Immunological time scale for hominid evolution.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">1200</span>–<span class="cit-lpage">1203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSarich%26rft.auinit1%253DV.%2BM.%26rft.volume%253D158%26rft.issue%253D3805%26rft.spage%253D1200%26rft.epage%253D1203%26rft.atitle%253DImmunological%2BTime%2BScale%2Bfor%2BHominid%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.158.3805.1200%26rft_id%253Dinfo%253Apmid%252F4964406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIxNTgvMzgwNS8xMjAwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schumpert</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dudycha</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Development of an efficient RNA interference method by feeding for the microcrustacean <em>Daphnia</em>.</span> <abbr class="cit-jnl-abbrev">BMC Biotechnol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">91</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBiotechnol.%26rft.volume%253D15%26rft.spage%253D91%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.56" data-doi="10.1111/j.1365-294X.2010.04828.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shafer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullingham</span>  <span class="cit-name-given-names">C. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cote</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coltman</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Of glaciers and refugia: a decade of study sheds new light on the phylogeography of northwestern North America.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">4589</span>–<span class="cit-lpage">4621</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DShafer%26rft.auinit1%253DA.%2BB.%26rft.volume%253D19%26rft.issue%253D21%26rft.spage%253D4589%26rft.epage%253D4621%26rft.atitle%253DOf%2Bglaciers%2Band%2Brefugia%253A%2Ba%2Bdecade%2Bof%2Bstudy%2Bsheds%2Bnew%2Blight%2Bon%2Bthe%2Bphylogeography%2Bof%2Bnorthwestern%2BNorth%2BAmerica.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2010.04828.x%26rft_id%253Dinfo%253Apmid%252F20849561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2010.04828.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20849561&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283163700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.57" data-doi="10.1093/bioinformatics/btv351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simão</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ioannidis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kriventseva</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zdobnov</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3210</span>–<span class="cit-lpage">3212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv351%26rft_id%253Dinfo%253Apmid%252F26059717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMzEvMTkvMzIxMCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-web" id="cit-7.5.1405.58"><div class="cit-metadata unstructured">Smit, A., R. Hubley, and P. Green, 2004 RepeatMasker Open-3.0. Available at: <a href="http://www.repeatmasker.org">http://www.repeatmasker.org</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.59" data-doi="10.1093/molbev/mst197"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stecher</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Filipski</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">MEGA6: molecular evolutionary genetics analysis version 6.0.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2725</span>–<span class="cit-lpage">2729</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst197%26rft_id%253Dinfo%253Apmid%252F24132122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIzMC8xMi8yNzI1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.60" data-doi="10.1159/000337126"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tollis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boissinot</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The evolutionary dynamics of transposable elements in eukaryote genomes.</span> <abbr class="cit-jnl-abbrev">Genome Dyn.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">68</span>–<span class="cit-lpage">91</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BDyn.%26rft.volume%253D7%26rft.spage%253D68%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000337126%26rft_id%253Dinfo%253Apmid%252F22759814%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000337126&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22759814&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.61" data-doi="10.1073/pnas.1313388110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ackerman</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eads</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Population-genomic insights into the evolutionary origin and fate of obligately asexual <em>Daphnia pulex</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">15740</span>–<span class="cit-lpage">15745</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1313388110%26rft_id%253Dinfo%253Apmid%252F23959868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzM5LzE1NzQwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.62" data-doi="10.1093/nar/gks1116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tegenfeldt</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zdobnov</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kriventseva</span>  <span class="cit-name-given-names">E. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">OrthoDB: a hierarchical catalog of animal, fungal and bacterial orthologs.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">D358</span>–<span class="cit-lpage">D365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1116%26rft_id%253Dinfo%253Apmid%252F23180791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MS9EMS9EMzU4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.63" data-doi="10.1007/s11427-015-4993-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The effect of transposable elements on phenotypic variation: insights from plants to humans.</span> <abbr class="cit-jnl-abbrev">Sci. China Life Sci.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BChina%2BLife%2BSci.%26rft.volume%253D59%26rft.spage%253D24%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11427-015-4993-2%26rft_id%253Dinfo%253Apmid%252F26753674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11427-015-4993-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26753674&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.64" data-doi="10.1073/pnas.82.6.1741"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Evidence for higher rates of nucleotide substitution in rodents than in man.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">1741</span>–<span class="cit-lpage">1745</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DWu%26rft.auinit1%253DC.%2BI.%26rft.volume%253D82%26rft.issue%253D6%26rft.spage%253D1741%26rft.epage%253D1745%26rft.atitle%253DEvidence%2Bfor%2Bhigher%2Brates%2Bof%2Bnucleotide%2Bsubstitution%2Bin%2Brodents%2Bthan%2Bin%2Bman.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.82.6.1741%26rft_id%253Dinfo%253Apmid%252F3856856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI4Mi82LzE3NDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.65" data-doi="10.1534/genetics.115.179028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ackerman</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bright</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spitze</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">a</span> <span class="cit-article-title">A male-specific genetic map of the microcrustacean <em>Daphnia pulex</em> based on single sperm whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.179028%26rft_id%253Dinfo%253Apmid%252F26116153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMjAxLzEvMzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.66" data-doi="10.1093/molbev/msv190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spitze</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ackerman</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bright</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">b</span> <span class="cit-article-title">Hybridization and the origin of contagious asexuality in <em>Daphnia</em> pulex.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">3215</span>–<span class="cit-lpage">3225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsv190%26rft_id%253Dinfo%253Apmid%252F26351296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIzMi8xMi8zMjE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.67" data-doi="10.1093/molbev/msm088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PAML 4: phylogenetic analysis by maximum likelihood.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1586</span>–<span class="cit-lpage">1591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm088%26rft_id%253Dinfo%253Apmid%252F17483113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI0LzgvMTU4NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1405.68" data-doi="10.1093/oxfordjournals.molbev.a026236"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Estimating synonymous and nonsynonymous substitution rates under realistic evolutionary models.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">32</span>–<span class="cit-lpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DYang%26rft.auinit1%253DZ.%26rft.volume%253D17%26rft.issue%253D1%26rft.spage%253D32%26rft.epage%253D43%26rft.atitle%253DEstimating%2BSynonymous%2Band%2BNonsynonymous%2BSubstitution%2BRates%2BUnder%2BRealistic%2BEvolutionary%2BModels%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a026236%26rft_id%253Dinfo%253Apmid%252F10666704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjc6IjE3LzEvMzIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/5/1405.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/5/1393" title="Genotype Calling from Population-Genomic Sequencing Data" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/5/1417" title="Dynamic Notch Signaling Specifies Each Cell Fate in Drosophila Spermathecal Lineage" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/5"><span>Volume 7 Issue 5, May 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/5.cover-source.jpg?itok=VWLvGstM" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1405.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Qljzfou04UQyWKJ6e6IlRsCxdpCTDGiN6Lsl5-92BLo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/5/1405" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/5/1405" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/5/1405" class="active" data-icon-position="" data-hide-link-title="0">A New Reference Genome Assembly for the Microcrustacean Daphnia pulex</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/484253" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-MgNWQLgeWIhyX_-Yu4zf3deoXLYQE-G4_c-sr0wAA90" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/5/1405.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/5/1405" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-QCKHHTJUkSNnf1DNuiNJYUhKUBTQ2FuZfa9icwNsGeA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/5/1405" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/5/1405.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484253" id="node484253--2" data-pisa="ggg;7/5/1405" data-pisa-master="ggg;g3.116.038638" data-apath="/ggg/7/5/1405.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A New Reference Genome Assembly for the Microcrustacean <em>Daphnia pulex</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhiqiang Ye</span>, <span class="highwire-citation-author" data-delta="1">Sen Xu</span>, <span class="highwire-citation-author" data-delta="2">Ken Spitze</span>, <span class="highwire-citation-author" data-delta="3">Jana Asselman</span>, <span class="highwire-citation-author" data-delta="4">Xiaoqian Jiang</span>, <span class="highwire-citation-author" data-delta="5">Matthew S. Ackerman</span>, <span class="highwire-citation-author" data-delta="6">Jacqueline Lopez</span>, <span class="highwire-citation-author" data-delta="7">Brent Harker</span>, <span class="highwire-citation-author" data-delta="8">R. Taylor Raborn</span>, <span class="highwire-citation-author" data-delta="9">W. Kelley Thomas</span>, <span class="highwire-citation-author" data-delta="10">Jordan Ramsdell</span>, <span class="highwire-citation-author" data-delta="11">Michael E. Pfrender</span> and <span class="highwire-citation-author" data-delta="12">Michael Lynch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1405-1416;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.038638 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484253--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhiqiang Ye </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhiqiang%2BYe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ye%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhiqiang%2BYe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zhiqiang.yes@gmail.com" class="" data-icon-position="" data-hide-link-title="0">zhiqiang.yes@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sen Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, University of Texas at Arlington, Texas 76019</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sen%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASen%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ken Spitze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ken%2BSpitze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spitze%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKen%2BSpitze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jana Asselman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory for Environmental Toxicology, Ghent University, B-9000, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jana%2BAsselman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Asselman%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJana%2BAsselman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Xiaoqian Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaoqian%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaoqian%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Matthew S. Ackerman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BS.%2BAckerman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ackerman%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BS.%2BAckerman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jacqueline Lopez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Notre Dame Genomics and Bioinformatics Core Facility, University of Notre Dame, Indiana 46556</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacqueline%2BLopez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lopez%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJacqueline%2BLopez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Brent Harker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Notre Dame Genomics and Bioinformatics Core Facility, University of Notre Dame, Indiana 46556</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brent%2BHarker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Harker%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrent%2BHarker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">R. Taylor Raborn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div><div class='author-affiliation'><span class='nlm-sup'>**</span>School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BTaylor%2BRaborn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raborn%20RT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AR.%2BTaylor%2BRaborn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">W. Kelley Thomas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Hubbard Center for Genome Studies, University of New Hampshire, Durham, New Hampshire 03824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=W.%2BKelley%2BThomas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thomas%20WK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AW.%2BKelley%2BThomas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Jordan Ramsdell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Hubbard Center for Genome Studies, University of New Hampshire, Durham, New Hampshire 03824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jordan%2BRamsdell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramsdell%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJordan%2BRamsdell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Michael E. Pfrender </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Biological Sciences, University of Notre Dame, Indiana 46556</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>Environmental Change Initiative, University of Notre Dame, Indiana 46556</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BE.%2BPfrender%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pfrender%20ME&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BE.%2BPfrender%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Michael Lynch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BLynch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lynch%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BLynch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/5/1405" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/5/1405&amp;title=A%20New%20Reference%20Genome%20Assembly%20for%20the%20Microcrustacean%20Daphnia%20pulex" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/5/1405&amp;title=A%20New%20Reference%20Genome%20Assembly%20for%20the%20Microcrustacean%20Daphnia%20pulex" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/5/1405&amp;title=A%20New%20Reference%20Genome%20Assembly%20for%20the%20Microcrustacean%20Daphnia%20pulex" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/5/1405&amp;text=A%20New%20Reference%20Genome%20Assembly%20for%20the%20Microcrustacean%20Daphnia%20pulex" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/5/1405&amp;title=A%20New%20Reference%20Genome%20Assembly%20for%20the%20Microcrustacean%20Daphnia%20pulex" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/5/1405&amp;t=A%20New%20Reference%20Genome%20Assembly%20for%20the%20Microcrustacean%20Daphnia%20pulex" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/5/1405&amp;title=A%20New%20Reference%20Genome%20Assembly%20for%20the%20Microcrustacean%20Daphnia%20pulex" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/5/1405&amp;title=A%20New%20Reference%20Genome%20Assembly%20for%20the%20Microcrustacean%20Daphnia%20pulex" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/484253/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484253" id="node484253--3" data-pisa="ggg;7/5/1405" data-pisa-master="ggg;g3.116.038638" data-apath="/ggg/7/5/1405.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A New Reference Genome Assembly for the Microcrustacean <em>Daphnia pulex</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhiqiang Ye</span>, <span class="highwire-citation-author" data-delta="1">Sen Xu</span>, <span class="highwire-citation-author" data-delta="2">Ken Spitze</span>, <span class="highwire-citation-author" data-delta="3">Jana Asselman</span>, <span class="highwire-citation-author" data-delta="4">Xiaoqian Jiang</span>, <span class="highwire-citation-author" data-delta="5">Matthew S. Ackerman</span>, <span class="highwire-citation-author" data-delta="6">Jacqueline Lopez</span>, <span class="highwire-citation-author" data-delta="7">Brent Harker</span>, <span class="highwire-citation-author" data-delta="8">R. Taylor Raborn</span>, <span class="highwire-citation-author" data-delta="9">W. Kelley Thomas</span>, <span class="highwire-citation-author" data-delta="10">Jordan Ramsdell</span>, <span class="highwire-citation-author" data-delta="11">Michael E. Pfrender</span> and <span class="highwire-citation-author" data-delta="12">Michael Lynch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1405-1416;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.038638 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484253--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhiqiang Ye </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhiqiang%2BYe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ye%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhiqiang%2BYe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zhiqiang.yes@gmail.com" class="" data-icon-position="" data-hide-link-title="0">zhiqiang.yes@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sen Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, University of Texas at Arlington, Texas 76019</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sen%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASen%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ken Spitze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ken%2BSpitze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spitze%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKen%2BSpitze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jana Asselman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory for Environmental Toxicology, Ghent University, B-9000, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jana%2BAsselman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Asselman%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJana%2BAsselman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Xiaoqian Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaoqian%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaoqian%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Matthew S. Ackerman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BS.%2BAckerman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ackerman%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BS.%2BAckerman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jacqueline Lopez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Notre Dame Genomics and Bioinformatics Core Facility, University of Notre Dame, Indiana 46556</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacqueline%2BLopez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lopez%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJacqueline%2BLopez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Brent Harker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Notre Dame Genomics and Bioinformatics Core Facility, University of Notre Dame, Indiana 46556</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brent%2BHarker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Harker%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrent%2BHarker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">R. Taylor Raborn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div><div class='author-affiliation'><span class='nlm-sup'>**</span>School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BTaylor%2BRaborn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raborn%20RT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AR.%2BTaylor%2BRaborn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">W. Kelley Thomas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Hubbard Center for Genome Studies, University of New Hampshire, Durham, New Hampshire 03824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=W.%2BKelley%2BThomas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thomas%20WK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AW.%2BKelley%2BThomas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Jordan Ramsdell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Hubbard Center for Genome Studies, University of New Hampshire, Durham, New Hampshire 03824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jordan%2BRamsdell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramsdell%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJordan%2BRamsdell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Michael E. Pfrender </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Biological Sciences, University of Notre Dame, Indiana 46556</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>Environmental Change Initiative, University of Notre Dame, Indiana 46556</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BE.%2BPfrender%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pfrender%20ME&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BE.%2BPfrender%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Michael Lynch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BLynch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lynch%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BLynch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1405&amp;rft.epage=1416&amp;rft.atitle=A%20New%20Reference%20Genome%20Assembly%20for%20the%20Microcrustacean%20Daphnia%20pulex&amp;rft.volume=7&amp;rft.issue=5&amp;rft.date=2017-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Ye%2C+Zhiqiang&amp;rft.au=Xu%2C+Sen&amp;rft.au=Spitze%2C+Ken&amp;rft.au=Asselman%2C+Jana&amp;rft.au=Jiang%2C+Xiaoqian&amp;rft.au=Ackerman%2C+Matthew+S.&amp;rft.au=Lopez%2C+Jacqueline&amp;rft.au=Harker%2C+Brent&amp;rft.au=Raborn%2C+R.+Taylor&amp;rft.au=Thomas%2C+W.+Kelley&amp;rft.au=Ramsdell%2C+Jordan&amp;rft.au=Pfrender%2C+Michael+E.&amp;rft.au=Lynch%2C+Michael"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/484253/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/484253/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/484253/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/484253/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/484253/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/484253/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/484253/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/484253/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/484253/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/484253/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/484253/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/484253/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/5/1405" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/5/1405.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/5/1405.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/5/1405.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/5/1405.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
