<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/5/1429" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/5/1429.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/5/1429.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/5/1429.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/5/1429.atom" />
<meta name="HW.pisa" content="ggg;7/5/1429" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Reliable CRISPR/Cas9 Genome Engineering in Caenorhabditis elegans Using a Single Efficient sgRNA and an Easily Recognizable Phenotype" />
<meta name="DC.Identifier" content="10.1534/g3.117.040824" />
<meta name="DC.Date" content="2017-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 El Mouridi et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="CRISPR/Cas9 genome engineering strategies allow the directed modification of the Caenorhabditis elegans genome to introduce point mutations, generate knock-out mutants, and insert coding sequences for epitope or fluorescent tags. Three practical aspects, however, complicate such experiments. First, the efficiency and specificity of single-guide RNAs (sgRNA) cannot be reliably predicted. Second, the detection of animals carrying genome edits can be challenging in the absence of clearly visible or selectable phenotypes. Third, the sgRNA target site must be inactivated after editing to avoid further double-strand break events. We describe here a strategy that addresses these complications by transplanting the protospacer of a highly efficient sgRNA into a gene of interest to render it amenable to genome engineering. This sgRNA targeting the [dpy-10][1] gene generates genome edits at comparatively high frequency. We demonstrate that the transplanted protospacer is cleaved at the same time as the [dpy-10][1] gene. Our strategy generates scarless genome edits because it no longer requires the introduction of mutations in endogenous sgRNA target sites. Modified progeny can be easily identified in the F1 generation, which drastically reduces the number of animals to be tested by PCR or phenotypic analysis. Using this strategy, we reliably generated precise deletion mutants, transcriptional reporters, and translational fusions with epitope tags and fluorescent reporter genes. In particular, we report here the first use of the new red fluorescent protein mScarlet in a multicellular organism. wrmScarlet, a C. elegans -optimized version, dramatically surpassed TagRFP-T by showing an eightfold increase in fluorescence in a direct comparison.

 [1]: http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene" />
<meta name="DC.Contributor" content="Sonia El Mouridi" />
<meta name="DC.Contributor" content="Claire Lecroisey" />
<meta name="DC.Contributor" content="Philippe Tardy" />
<meta name="DC.Contributor" content="Marine Mercier" />
<meta name="DC.Contributor" content="Alice Leclercq-Blondel" />
<meta name="DC.Contributor" content="Nora Zariohi" />
<meta name="DC.Contributor" content="Thomas Boulin" />
<meta name="article:published_time" content="2017-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Reliable CRISPR/Cas9 Genome Engineering in Caenorhabditis elegans Using a Single Efficient sgRNA and an Easily Recognizable Phenotype" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;CRISPR/Cas9 genome engineering strategies allow the directed modification of the &lt;i&gt;Caenorhabditis elegans&lt;/i&gt; genome to introduce point mutations, generate knock-out mutants, and insert coding sequences for epitope or fluorescent tags. Three practical aspects, however, complicate such experiments. First, the efficiency and specificity of single-guide RNAs (sgRNA) cannot be reliably predicted. Second, the detection of animals carrying genome edits can be challenging in the absence of clearly visible or selectable phenotypes. Third, the sgRNA target site must be inactivated after editing to avoid further double-strand break events. We describe here a strategy that addresses these complications by transplanting the protospacer of a highly efficient sgRNA into a gene of interest to render it amenable to genome engineering. This sgRNA targeting the &lt;i&gt;dpy-10&lt;/i&gt; gene generates genome edits at comparatively high frequency. We demonstrate that the transplanted protospacer is cleaved at the same time as the &lt;i&gt;dpy-10&lt;/i&gt; gene. Our strategy generates scarless genome edits because it no longer requires the introduction of mutations in endogenous sgRNA target sites. Modified progeny can be easily identified in the F1 generation, which drastically reduces the number of animals to be tested by PCR or phenotypic analysis. Using this strategy, we reliably generated precise deletion mutants, transcriptional reporters, and translational fusions with epitope tags and fluorescent reporter genes. In particular, we report here the first use of the new red fluorescent protein mScarlet in a multicellular organism. wrmScarlet, a &lt;i&gt;C. elegans&lt;/i&gt;-optimized version, dramatically surpassed TagRFP-T by showing an eightfold increase in fluorescence in a direct comparison.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/05/01" />
<meta name="citation_mjid" content="ggg;7/5/1429" />
<meta name="citation_id" content="7/5/1429" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/5/1429" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/5/1429.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/5/1429.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/5/1429.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.040824" />
<meta name="citation_pmid" content="28280211" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1429" />
<meta name="citation_lastpage" content="1437" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Sonia El Mouridi" />
<meta name="citation_author_institution" content="Univ Lyon, Université Claude Bernard Lyon 1, CNRS UMR-5310, INSERM U-1217, Institut NeuroMyoGène, F-69622 Villeurbanne, France" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3674-3151" />
<meta name="citation_author" content="Claire Lecroisey" />
<meta name="citation_author_institution" content="Univ Lyon, Université Claude Bernard Lyon 1, CNRS UMR-5310, INSERM U-1217, Institut NeuroMyoGène, F-69622 Villeurbanne, France" />
<meta name="citation_author" content="Philippe Tardy" />
<meta name="citation_author_institution" content="Univ Lyon, Université Claude Bernard Lyon 1, CNRS UMR-5310, INSERM U-1217, Institut NeuroMyoGène, F-69622 Villeurbanne, France" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4313-2527" />
<meta name="citation_author" content="Marine Mercier" />
<meta name="citation_author_institution" content="Univ Lyon, Université Claude Bernard Lyon 1, CNRS UMR-5310, INSERM U-1217, Institut NeuroMyoGène, F-69622 Villeurbanne, France" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1508-8845" />
<meta name="citation_author" content="Alice Leclercq-Blondel" />
<meta name="citation_author_institution" content="Univ Lyon, Université Claude Bernard Lyon 1, CNRS UMR-5310, INSERM U-1217, Institut NeuroMyoGène, F-69622 Villeurbanne, France" />
<meta name="citation_author" content="Nora Zariohi" />
<meta name="citation_author_institution" content="Univ Lyon, Université Claude Bernard Lyon 1, CNRS UMR-5310, INSERM U-1217, Institut NeuroMyoGène, F-69622 Villeurbanne, France" />
<meta name="citation_author" content="Thomas Boulin" />
<meta name="citation_author_institution" content="Univ Lyon, Université Claude Bernard Lyon 1, CNRS UMR-5310, INSERM U-1217, Institut NeuroMyoGène, F-69622 Villeurbanne, France" />
<meta name="citation_author_email" content="thomas.boulin@univ-lyon1.fr" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1734-1915" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. A. Arribere;citation_author=R. T. Bell;citation_author=B. X. H. Fu;citation_author=K. L. Artiles;citation_author=P. S. Hartman;citation_title=Efficient marker-free recovery of custom genetic modifications with CRISPR/Cas9 in Caenorhabditis elegans.;citation_volume=198;citation_year=2014;citation_pmid=25161212;citation_doi=10.1534/genetics.114.169730" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=D. S. Bindels;citation_author=L. Haarbosch;citation_author=L. van Weeren;citation_author=M. Postma;citation_author=K. E. Wiese;citation_title=mScarlet: a bright monomeric red fluorescent protein for cellular imaging.;citation_pages=53-56;citation_volume=14;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=WIREs Dev Biol;citation_author=T. Boulin;citation_author=O. Hobert;citation_title=From genes to function: the C. elegans genetic toolbox.;citation_pages=114-137;citation_volume=1;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=Y. Chai;citation_author=W. Li;citation_author=G. Feng;citation_author=Y. Yang;citation_author=X. Wang;citation_title=Live imaging of cellular dynamics during Caenorhabditis elegans postembryonic development.;citation_pages=2090-2102;citation_volume=7;citation_year=2012;citation_issue=12;citation_pmid=23138350;citation_doi=10.1038/nprot.2012.128" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=C. Chen;citation_author=L. A. Fenk;citation_author=M. De Bono;citation_title=Efficient genome editing in Caenorhabditis elegans by CRISPR-targeted homologous recombination.;citation_volume=41;citation_year=2013;citation_pmid=24013562;citation_doi=10.1093/nar/gkt805" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. W. Cho;citation_author=J. Lee;citation_author=D. Carroll;citation_author=J.-S. Kim;citation_author=J. Lee;citation_title=Heritable gene knockout in Caenorhabditis elegans by direct injection of Cas9-sgRNA ribonucleoproteins.;citation_volume=195;citation_year=2013;citation_pmid=23979576;citation_doi=10.1534/genetics.113.155853" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. J. Dickinson;citation_author=B. Goldstein;citation_title=CRISPR-Based Methods for Caenorhabditis elegans Genome Engineering;citation_pages=885-901;citation_volume=202;citation_year=2016;citation_issue=3;citation_pmid=26953268;citation_doi=10.1534/genetics.115.182162" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=D. J. Dickinson;citation_author=J. D. Ward;citation_author=D. J. Reiner;citation_author=B. Goldstein;citation_title=Engineering the Caenorhabditis elegans genome using Cas9-triggered homologous recombination.;citation_pages=1028-1034;citation_volume=10;citation_year=2013;citation_pmid=23995389;citation_doi=10.1038/nmeth.2641" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. J. Dickinson;citation_author=A. M. Pani;citation_author=J. K. Heppert;citation_author=C. D. Higgins;citation_author=B. Goldstein;citation_title=Streamlined genome engineering with a self-excising drug selection cassette.;citation_volume=200;citation_year=2015;citation_pmid=26044593;citation_doi=10.1534/genetics.115.178335" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=J. G. Doench;citation_author=E. Hartenian;citation_author=D. B. Graham;citation_author=Z. Tothova;citation_author=M. Hegde;citation_title=Rational design of highly active sgRNAs for CRISPR-Cas9-mediated gene inactivation.;citation_pages=1262-1267;citation_volume=32;citation_year=2014;citation_pmid=25184501;citation_doi=10.1038/nbt.3026" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=J. G. Doench;citation_author=N. Fusi;citation_author=M. Sullender;citation_author=M. Hegde;citation_author=E. W. Vaimberg;citation_title=Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR-Cas9.;citation_pages=184-191;citation_volume=34;citation_year=2016;citation_pmid=26780180;citation_doi=10.1038/nbt.3437" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. A. Doudna;citation_author=E. Charpentier;citation_title=The new frontier of genome engineering with CRISPR-Cas9;citation_pages=1258096-1258096;citation_volume=346;citation_year=2014;citation_issue=6213;citation_pmid=25430774;citation_doi=10.1126/science.1258096" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. Farboud;citation_author=B. J. Meyer;citation_title=Dramatic enhancement of genome editing by CRISPR/Cas9 through improved guide RNA design.;citation_volume=199;citation_year=2015;citation_pmid=25695951;citation_doi=10.1534/genetics.115.175166" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=A. E. Friedland;citation_author=Y. B. Tzur;citation_author=K. M. Esvelt;citation_author=M. P. Colaiácovo;citation_author=G. M. Church;citation_title=Heritable genome editing in C. elegans via a CRISPR-Cas9 system.;citation_pages=741-743;citation_volume=10;citation_year=2013;citation_pmid=23817069;citation_doi=10.1038/nmeth.2532" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Frøkjær-Jensen;citation_title=Exciting Prospects for Precise Engineering of Caenorhabditis elegans Genomes with CRISPR/Cas9;citation_pages=635-642;citation_volume=195;citation_year=2013;citation_issue=3;citation_pmid=24190921;citation_doi=10.1534/genetics.113.156521" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=C. Frøkjær-Jensen;citation_author=M. W. Davis;citation_author=J. Taylor;citation_author=T. W. Harris;citation_author=D. G. Moerman;citation_title=Targeted gene deletions in C. elegans using transposon excision.;citation_pages=451-453;citation_volume=7;citation_year=2010;citation_issue=6;citation_pmid=20418868;citation_doi=10.1038/nmeth.1454" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=D. T. Ge;citation_author=C. Tipping;citation_author=M. H. Brodsky;citation_author=P. D. Zamore;citation_title=Rapid screening for CRISPR-directed editing of the Drosophila genome using white coconversion.;citation_pages=3197-3206;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Gendrel;citation_author=G. Rapti;citation_author=J. E. Richmond;citation_author=J.-L. Bessereau;citation_title=A secreted complement-control-related protein ensures acetylcholine receptor clustering.;citation_pages=992-996;citation_volume=461;citation_year=2009;citation_issue=7266;citation_pmid=19794415;citation_doi=10.1038/nature08430" />
<meta name="citation_reference" content="citation_journal_title=Methods in enzymology;citation_journal_abbrev=Methods Enzymol;citation_author=D. G. Gibson;citation_title=Enzymatic assembly of overlapping DNA fragments.;citation_pages=349-361;citation_volume=498;citation_year=2011;citation_pmid=21601685;citation_doi=10.1016/B978-0-12-385120-8.00015-2" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Commun;citation_author=J. Goedhart;citation_author=D. Stetten von;citation_author=M. Noirclerc-Savoye;citation_author=M. Lelimousin;citation_author=L. Joosen;citation_title=Structure-guided evolution of cyan fluorescent proteins towards a quantum yield of 93%.;citation_pages=751-751;citation_volume=3;citation_year=2012;citation_pmid=22434194;citation_doi=10.1038/ncomms1738" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=J. K. Heppert;citation_author=D. J. Dickinson;citation_author=A. M. Pani;citation_author=C. D. Higgins;citation_author=A. Steward;citation_title=Comparative assessment of fluorescent proteins for in vivo imaging in an animal model system.;citation_volume=27;citation_year=2016;citation_pmid=27385332;citation_doi=10.1091/mbc.E16-01-0063" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=P. D. Hsu;citation_author=D. A. Scott;citation_author=J. A. Weinstein;citation_author=F. A. Ran;citation_author=S. Konermann;citation_title=DNA targeting specificity of RNA-guided Cas9 nucleases.;citation_pages=827-832;citation_volume=31;citation_year=2013;citation_pmid=23873081;citation_doi=10.1038/nbt.2647" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=I. Katic;citation_author=H. Großhans;citation_title=Targeted heritable mutation and gene conversion by Cas9-CRISPR in Caenorhabditis elegans.;citation_volume=195;citation_year=2013;citation_pmid=23979578;citation_doi=10.1534/genetics.113.155754" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=I. Katic;citation_author=L. Xu;citation_author=R. Ciosk;citation_title=CRISPR/Cas9 genome editing in Caenorhabditis elegans: evaluation of templates for homology-mediated repair and knock-ins by homology-independent DNA repair.;citation_pages=1649-1656;citation_volume=5;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=W. G. Kelly;citation_author=A. Fire;citation_title=Chromatin silencing and the maintenance of a functional germline in Caenorhabditis elegans;citation_pages=2451-2456;citation_volume=125;citation_year=1998;citation_issue=13;citation_pmid=9609828" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Kim;citation_author=T. Ishidate;citation_author=K. S. Ghanta;citation_author=M. Seth;citation_author=D. Conte;citation_title=A co-CRISPR strategy for efficient genome editing in Caenorhabditis elegans.;citation_volume=197;citation_year=2014;citation_pmid=24879462;citation_doi=10.1534/genetics.114.166389" />
<meta name="citation_reference" content="citation_journal_title=Journal of Neuroscience;citation_journal_abbrev=J. Neurosci.;citation_author=M. T. Kunkel;citation_author=D. B. Johnstone;citation_author=J. H. Thomas;citation_author=L. Salkoff;citation_title=Mutants of a Temperature-Sensitive Two-P Domain Potassium Channel;citation_pages=7517-7524;citation_volume=20;citation_year=2000;citation_issue=20;citation_pmid=11027209" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology of the Cell;citation_journal_abbrev=Mol. Biol. Cell;citation_author=A. D. Levy;citation_author=J. Yang;citation_author=J. M. Kramer;citation_title=Molecular and genetic analyses of the Caenorhabditis elegans dpy-2 and dpy-10 collagen genes: a variety of molecular alterations affect organismal morphology.;citation_pages=803-817;citation_volume=4;citation_year=1993;citation_issue=8;citation_pmid=8241567;citation_doi=10.1091/mbc.4.8.803" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. D. Norris;citation_author=H.-M. Kim;citation_author=M. P. Colaiácovo;citation_author=J. A. Calarco;citation_title=Efficient genome editing in Caenorhabditis elegans with a toolkit of dual-marker selection cassettes.;citation_volume=201;citation_year=2015;citation_pmid=26232410;citation_doi=10.1534/genetics.115.180679" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Paix;citation_author=Y. Wang;citation_author=H. E. Smith;citation_author=C.-Y. S. Lee;citation_author=D. Calidas;citation_title=Scalable and versatile genome editing using linear DNAs with microhomology to Cas9 sites in Caenorhabditis elegans.;citation_volume=198;citation_year=2014;citation_pmid=25249454;citation_doi=10.1534/genetics.114.170423" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Paix;citation_author=A. Folkmann;citation_author=D. Rasoloson;citation_author=G. Seydoux;citation_title=High efficiency, homology-directed genome editing in Caenorhabditis elegans using CRISPR-Cas9 ribonucleoprotein complexes.;citation_volume=201;citation_year=2015;citation_pmid=26187122;citation_doi=10.1534/genetics.115.179382" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=S. Redemann;citation_author=S. Schloissnig;citation_author=S. Ernst;citation_author=A. Pozniakowsky;citation_author=S. Ayloo;citation_title=Codon adaptation-based control of protein expression in C. elegans.;citation_pages=250-252;citation_volume=8;citation_year=2011;citation_issue=3;citation_pmid=21278743;citation_doi=10.1038/nmeth.1565" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=V. Robert;citation_author=J.-L. Bessereau;citation_title=Targeted engineering of the Caenorhabditis elegans genome following Mos1-triggered chromosomal breaks;citation_pages=170-183;citation_volume=26;citation_year=2007;citation_issue=1;citation_pmid=17159906;citation_doi=10.1038/sj.emboj.7601463" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. L. Schwartz;citation_author=E. M. Jorgensen;citation_title=SapTrap, a toolkit for high-throughput CRISPR/Cas9 gene modification in Caenorhabditis elegans.;citation_volume=202;citation_year=2016;citation_pmid=26837755;citation_doi=10.1534/genetics.115.184275" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=N. C. Shaner;citation_author=M. Z. Lin;citation_author=M. R. McKeown;citation_author=P. A. Steinbach;citation_author=K. L. Hazelwood;citation_title=Improving the photostability of bright monomeric orange and red fluorescent proteins.;citation_pages=545-551;citation_volume=5;citation_year=2008;citation_issue=6;citation_pmid=18454154;citation_doi=10.1038/nmeth.1209" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=N. C. Shaner;citation_author=G. G. Lambert;citation_author=A. Chammas;citation_author=Y. Ni;citation_author=P. J. Cranfill;citation_title=A bright monomeric green fluorescent protein derived from Branchiostoma lanceolatum.;citation_pages=407-409;citation_volume=10;citation_year=2013;citation_pmid=23524392;citation_doi=10.1038/nmeth.2413" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Waaijers;citation_author=V. Portegijs;citation_author=J. Kerver;citation_author=B. B. L. G. Lemmens;citation_author=M. Tijsterman;citation_title=CRISPR/Cas9-targeted mutagenesis in Caenorhabditis elegans.;citation_volume=195;citation_year=2013;citation_pmid=23979586;citation_doi=10.1534/genetics.113.156299" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. D. Ward;citation_title=Rapid and precise engineering of the Caenorhabditis elegans genome with lethal mutation co-conversion and inactivation of NHEJ repair.;citation_volume=199;citation_year=2015;citation_pmid=25491644;citation_doi=10.1534/genetics.114.172361" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=B. Zetsche;citation_author=J. S. Gootenberg;citation_author=O. O. Abudayyeh;citation_author=I. M. Slaymaker;citation_author=K. S. Makarova;citation_title=Cpf1 is a single RNA-guided endonuclease of a class 2 CRISPR-Cas system.;citation_pages=759-771;citation_volume=163;citation_year=2015;citation_pmid=26422227;citation_doi=10.1016/j.cell.2015.09.038" />
<meta name="citation_reference" content="citation_journal_title=Nature Publishing Group;citation_author=P. Zhao;citation_author=Z. Zhang;citation_author=H. Ke;citation_author=Y. Yue;citation_author=D. Xue;citation_title=Oligonucleotide-based targeted gene editing in C. elegans via the CRISPR/Cas9 system.;citation_pages=247-250;citation_volume=24;citation_year=2014" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Reliable CRISPR/Cas9 Genome Engineering in Caenorhabditis elegans Using a Single Efficient sgRNA and an Easily Recognizable Phenotype" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
<meta name="twitter:description" content="CRISPR/Cas9 genome engineering strategies allow the directed modification of the Caenorhabditis elegans genome to introduce point mutations, generate knock-out mutants, and insert coding sequences for epitope or fluorescent tags. Three practical aspects, however, complicate such experiments. First, the efficiency and specificity of single-guide RNAs (sgRNA) cannot be reliably predicted. Second, the detection of animals carrying genome edits can be challenging in the absence of clearly visible or selectable phenotypes. Third, the sgRNA target site must be inactivated after editing to avoid further double-strand break events. We describe here a strategy that addresses these complications by transplanting the protospacer of a highly efficient sgRNA into a gene of interest to render it amenable to genome engineering. This sgRNA targeting the [dpy-10][1] gene generates genome edits at comparatively high frequency. We demonstrate that the transplanted protospacer is cleaved at the same time as the [dpy-10][1] gene. Our strategy generates scarless genome edits because it no longer requires the introduction of mutations in endogenous sgRNA target sites. Modified progeny can be easily identified in the F1 generation, which drastically reduces the number of animals to be tested by PCR or phenotypic analysis. Using this strategy, we reliably generated precise deletion mutants, transcriptional reporters, and translational fusions with epitope tags and fluorescent reporter genes. In particular, we report here the first use of the new red fluorescent protein mScarlet in a multicellular organism. wrmScarlet, a C. elegans -optimized version, dramatically surpassed TagRFP-T by showing an eightfold increase in fluorescence in a direct comparison.

 [1]: http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene" />
<meta name="og-title" property="og:title" content="Reliable CRISPR/Cas9 Genome Engineering in Caenorhabditis elegans Using a Single Efficient sgRNA and an Easily Recognizable Phenotype" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/5/1429" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="CRISPR/Cas9 genome engineering strategies allow the directed modification of the Caenorhabditis elegans genome to introduce point mutations, generate knock-out mutants, and insert coding sequences for epitope or fluorescent tags. Three practical aspects, however, complicate such experiments. First, the efficiency and specificity of single-guide RNAs (sgRNA) cannot be reliably predicted. Second, the detection of animals carrying genome edits can be challenging in the absence of clearly visible or selectable phenotypes. Third, the sgRNA target site must be inactivated after editing to avoid further double-strand break events. We describe here a strategy that addresses these complications by transplanting the protospacer of a highly efficient sgRNA into a gene of interest to render it amenable to genome engineering. This sgRNA targeting the [dpy-10][1] gene generates genome edits at comparatively high frequency. We demonstrate that the transplanted protospacer is cleaved at the same time as the [dpy-10][1] gene. Our strategy generates scarless genome edits because it no longer requires the introduction of mutations in endogenous sgRNA target sites. Modified progeny can be easily identified in the F1 generation, which drastically reduces the number of animals to be tested by PCR or phenotypic analysis. Using this strategy, we reliably generated precise deletion mutants, transcriptional reporters, and translational fusions with epitope tags and fluorescent reporter genes. In particular, we report here the first use of the new red fluorescent protein mScarlet in a multicellular organism. wrmScarlet, a C. elegans -optimized version, dramatically surpassed TagRFP-T by showing an eightfold increase in fluorescence in a direct comparison.

 [1]: http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
    <title>Reliable CRISPR/Cas9 Genome Engineering in Caenorhabditis elegans Using a Single Efficient sgRNA and an Easily Recognizable Phenotype | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"UPo8gaVtciXf766MHFAFEovpEJJqqxWHtO6mIF7x3Ww","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"484242","apath":"\/ggg\/7\/5\/1429.atom","pisa":"ggg;7\/5\/1429","ac":{"\/ggg\/7\/5\/1429.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/5\/1429.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/5\/1429"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/5\/1429"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/5\/1429":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022os+y7QP9\\\\\\\/zF\\\\\\\/clQpgi+MNuiiLli94EqRfPXSFYYHqeJSwXuCA5R0abRKMj\\\\\\\/87NAaM6Zzsqj51+tVMysVXUEJlKEFm3QwQu\\\\\\\/OqG9+OHLkAiI6yLXRHcY7HZ6uR6Wf9dNFQfVPKzozGQGd\\\\\\\/d\\\\\\\/RSe0233hdPhLUz0tDh1fDUHUpOPTWRTAn2n0zQoLr84O\\\\\\\/JGmyGu+RZon2ngRti3ISB5PZcG5miWIP7dm2\\\\\\\/uHLysEx12EDSIWghuX\\\\\\\/O6NWYwWCkry2seqK9hSyNy\\\\\\\/ZsfOX6ez6JQHJdM\\\\\\\/VJRGzGEN\\\\\\\/OvvqiMWlu18BLbEVR0vMndCw32ilotyLy7LkaIPSuKFvP5d\\\\\\\/PA8DD6+Yc2BSEptlhX5UsJmKEf4RCeo89jNSQ+JSnsW7lVBGxM9ohXRsY0Dk\\\\\\\/jL0o1dK6BAazcouH87lW\\\\\\\/K7q38E3IXeAQm\\\\\\\/GQl8PX72443H5DGwB5uLMM18oGnMc7GVFTs4carrN9EWE45\\\\\\\/OVPJh\\\\\\\/SC7aIvm3rlA+cPqyxs7eUt6Cok7bDDAz2kuxQQC7ELwcr6oxBPvxKjTEM0u6MEy\\\\\\\/dXOJ2lL\\\\\\\/+y8IQVkCQ1jbV4geuX5esWHPKTbxnHchDoBca9jc7M+xO0JyMCSwrOLCrqoZzCQ2GbCrctdyKFeDywj0xg\\\\\\\/Y279lw5C0x2DdK6y+jiW8jOrClv68+zq5q+aUw8qVmqCxZZ5l+IumYqb4Cx6yzyZK6RjcDnn7VZpYR7uo4RZkkIjQohXSnpYu8m36mYrabSStxMnDqhS\\\\\\\/9rfta0mSlozxBdIi1TPGPWwfyeNBzFo8CPzKzHXK4xTD6IR8phG5qxpwJDlDmyDjVYT+7VS5XyTcjr3Ph3QUKE159GFkMTN6ngOJNz5bfB2toHg1jyQ4xCvx2\\\\\\\/pM8WJU9C1BBCAf0ajWt\\\\\\\/4kK0QuKly7AC1D1O5RbodlHAMKUQNbIi0OY+AE9AYrM1MX3Yuy8fHJl9vCj2dvAL9Esshf9Q9vSk4ZNtCuMlTEyHL00FkdvDFG4SWOCpEAjpUp7kU691AYBeiM0Z4+2PIiP4fz0o4wzuUzgoWRtj\\\\\\\/FDA4CvSCjsD5DuNzfZoaMoFsQQaPdU0ivKANPLbX9nPyCmq9EwrPNgTcAXTJ6ViVP\\\\\\\/hdSIdvc1ExKKw1uwXq1ogcXAxOy4iZG5DaXWxCryl8YXUbdJaF4w2aWTUkvt069DeT54HgQ3tD+5+NfldPSHgoPV5U8JdGiqP8o+Qmqsmqod2PadiBp71yUAgrBYPdkdPuqSxXztza9L6PsqgK0Tmf2ZNqfQENzh0\\\u0022,\\\u0022iv\\\u0022:\\\u0022WcQpmSsEIN8bUGp7KgkTLg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e8edd3d2c787783c40fed34e7cba8102541962db7a824e07b9646d2c0867ea65\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022eDjq1TGNZX0C8YPk4tRNv5MRY2K2s9mO\\\\\\\/H+9Cnu\\\\\\\/MoO72qrOwXxtEyOpMNKIOPhhXhkdgXYdD8c+Iw1jS+YyPvD25gDLpLwHo5Tb8E2FH+DokguYQD54eJxY0f64IMEnawJ3qG0PB97O4n+FSwAwCiiyroHHQmVFuR4DAeOTqUSLJ+DxAI44XanNkrDxTxT033JbNwr7cESWqSxbkg\\\\\\\/5JkxA6GxNvvZE8Wji22Ey0j8iBqSqmMGdPtXx5zD6aYOyfQv72p9+WFdXgks2taUz9+52\\\\\\\/lV6KldxSJxjBvfbWQVf3Aw4yhw\\\\\\\/SOTlDYmr3JgFse3hWtcbK9LKUpZgb4wZK7p1o0fSxyWZNk8B0tqCl4GEef05oTSoS6Ta5p9reraoF5dq9ElPJ1+HoMjgB0D9cfebrN2f81HHQxaY5v\\\\\\\/TJJc3mlI+wg1VvTbVuQKl5tICyCfnsS4jHYTwwcdzzhw7+vGAK7xCqj4UeiHmZVZCtasiUuEEjShI02xEflFR87GsKxeY7pT1E72C8vlEd8YZhEmkf55\\\\\\\/C1NJL+ePu+nDS3E+eH1Qu5zDe\\\\\\\/NK54Ht+ohLvmO+3bSgGAtf9uoif+tQvKfgBoA2ArIb9Ucgv2WuXlBqGloKNioqsr3GIekU4fjg92L+hsz5ylvdRWV5S+VCxt7hvryK3ajr\\\\\\\/e91GVPlUUnLGvJBrnSgOyWtWgID6YEf\\\\\\\/Uyj05etcEgg3ytNWqwczXU2LFy3D2c6fiV7eM3xcBc4enFIN6g4jdbiQ52PPdIr5gwCE\\\\\\\/QHuHrzak+t7N9\\\\\\\/oAz6RvJlyozCHaG2hpn08SU0PY5qSGtd34l7FeG12ycVx9J8reejpNZyBFvsJvL18aS8fpxBS0716fjBJtz8x7tOn1cgOHmvO3WZKnZzqLRyYj52auuzYxJ84KUrdeT4S+KSwSTZIu1GQJewsezYfUZU63IOLZuhfsRSlsR9CHJqcu1CoLw3j7v2CavHoA4m+89v44vL8raBSuVe0Q8YRYQPLCjv9a87o\\\\\\\/z\\\\\\\/Brj824n1Hxr2kKAvwyC7up1W4qgHxWCma2OsjCED8wjrW8njueX1\\\\\\\/GJ6RFdZGihgaYSjLL1YuzEYgY8WNqK8XmuGfQ2wGLPDBuw+yFzbOCqW3jzS\\\\\\\/yhmSuLh0BT1ci1iCKOE7KhDa1vAFnQ6GlGnxfd2TJD9QzaSDDrtDVYbEJQTAlTLDbtNncMmxptYbP8dwumlKive79IUYCf7au09C50V4R1P49T4Bina9FkysQ3ibHN+3lt0fX3sGYrtNpAtBeyb0jtwJAu8zAMxxxh+f9xMUfnreFkNgWxJ245K+1gsdS\\\\\\\/pA3bUv2UIxJ\\\\\\\/GvSVUjX4pZP+0HSTx+UCde2eh92QUMPGTDuQHJX7nuSmXrQFFhH69epBGvhAsDUntVLeHyp0zA+jAJRjAuqvPes41ee5FAtbl2r5mfgP20NQGAmxXRk5RhurLrrCpc\\\\\\\/uOHA2JbvLLRQ8q4\\\\\\\/YFEfzMLbl1J7MEeBPoLTaadQrxeQKoni+Onj2X\\\\\\\/pc93pjJorK1ump3NZ8\\\\\\\/o6qvnwoctoim9tDXvG+Q6WnJZ+hZ8PM0H4ERpZWxoMBwhfN3A61B7pozUgv1Z3RHSl1BfzZ56MhXmGMVolYW3tPg9YT5NMQRiBAkBfimRv7+J3hkBVHEI3uq5eSaYWkHsaV8mMTUD8NaWaqleU6XxGrW11YDh3xvHygUUcxJrFo6FG35kj6Mo7ndTgri8Ijla9MsOPeMBhK4AF6Yhv1EIDtc6cOA+c6t2eYQi0SQR8R1AWkrgqiawVMxfPZaNxe+3TkXnGHnFom9WKB8FCfQL1VMDdATrHHCa9I2xEtvqqWY6NFiTIkmu0cNGuXqdTGfRFrDwXVwQIIkueVYq1rKfz0GjJTERl83kJ198ZQCcTYEtouHZvGKG48Fv3PuFPzhSkla3BATNoSc8pzZi0H5BKYdhf7N6NjoEouNrfLM30VmkuosQLQst0J8mN1iHGUwj0hE9wSaQ9JoTfFszHpejO0IS\\\\\\\/OoZ\\\\\\\/yPl212tKGnCpAaGJbuMHCqR7r0iNOc07bxd\\\\\\\/JYxaSLWh64LvGUrQOXfCSmZXjN4+76HzkEnGofUe+TgKdXRjc+IS9teSH8I5LV2GLpcdlTKMDpJsKy7Nj7oQ3cZWg8Lf9zBDKhp1gNsXTlIa3kcL8Tlhk+tOEjbnAS49mKu4eR6coOLJsRZU81oL5OcFEjdLqVlC9kZwQFBRPZVNIzthZ8oJDQWc0Ndtizt9FlyyT+E4w34oRIKm2yEvcb4CPBDzNyNzkhziP3dQBHJ6byKl+FP0iWJcMoswx4zsb192uQhdRvNiEQKEFrMtb3xUC4ba6H96\\\\\\\/3KH8XX08OYon3\\\\\\\/xpSNd2v7THxYxDtXxu+GohJrO5W1QPbsMC3BeBTky7sLkqmFoAoVjJs5Wh9nBShf8OvkMfeJ5\\\\\\\/hJXv8uZWwb+o7mxxiYQOZhs4uvpwtfu6Oz8CVukZagQcvNI0ks9iiG8e03Wb8ndm1B1+V82iekXfYHMuYKdGXufYU8pmiL+f8HZF5YDXf\\\\\\\/N4HTe\\\\\\\/qrKzEjxi6dfHS9IZU5VobNZ99o4YAp+R3Z8v8YehP83cGPXCkioWnT1YZgFhLvx4KOzDQ4lIFYDJkiR3zFVoIC2NL3iBIpR0jCuDgoKUiA7jn3mzfE9B7+6JC3UsUxHjk1HbMgLLKnGOfR39wHiDf7\\\\\\\/g61UI\\\\\\\/qgOKz9lfsvXhdphI76nz\\\\\\\/BcMyjLkeVdSVDhTMvvOd5QoxZ3MypeXqfist9QznDyG1MkVy27DbsbOE5riSzzodn0cXj\\\\\\\/NA7O7juE6vuJh2VBd3IIKO\\\\\\\/PaVidcFH4W9qIg7ZtE3pLCN+iUc\\\\\\\/T99XkGXXrNFbPqL6mCP3p3ueoSGcRTt1k0XGtO32EkrKJkQ7K\\\\\\\/Q\\\\\\\/GyJI7f3rf38VzY7wJ5jUv3\\\\\\\/\\\\\\\/P8L7Bc65OsITfFJU3yG3pyijwXs9jWzK9uCbQESzpqmvaMGMEjgwmHbV78uOOV7SDJb1sKVswMPTtS6s4j8CfRE4aphz920AynA4eoBkZsshjyP+RBIbnjpjOzA1Cj1G2GM7t26g2DLpTaxs0mDNCI63mjbQ6O0MHwUzZ1ER8LiO5iIGX+PrryOPKpJ7Yzdb8XZnaYa4RtIhQBZ+etfJxlZfwNqQbR\\\\\\\/sW4FOO\\\\\\\/JTsBkt2GMutAj+FjdBn9Mva5Yw5pMORlUj\\\\\\\/IIUqwsPkPe\\\\\\\/opngfbSLULN9lzjHyOpKG346GqHfkzUL\\\\\\\/MJsXQCTsVAYnzo6EbirbBVXp8leQ1EOyJEENGz1R8cCF+RlszCfssQQJhjDBx7svWm4YAra\\\\\\\/ryI9W7l8KmBMOS\\\\\\\/wm3j+oHjT6SGhALlDCx2p4eGx67eqjNVLYqw7G3XQMUrOTSpGwloZi\\\\\\\/BeWcK8+8eNwqb1EfsXowae9FM0I44RmjBI\\\\\\\/1HDiiv4EkD+LMUjF7EKvgZNHwT5hnJ5HDuQJBFpqeBTVurqNR2DLzam+Qm3imnwa7NlUshyn24b5q2uXdOrdZu4XwgH2yMJW4Zx7xQGj0hgVCjLV2skFuQUGrNHDlgzvEMvCCjBBauys5F2yavV7mZ3EQ65HCSxecyWl88YBduKP7hDwY3GD7sjXpxG7yeQsu1bg53CMkuao4P97Eq8L0O9UvSsdsYWqQTF1G8IzTEMNglXuyPPNUhlOGzBqQUztf3AEc3w9W98xogXuUTt6HqaDpJSqwAxFyzpzIFxqXDs0euYn5TdBjBjyWtVkuNHore+hrm5SWWZKDCApb3M2BOi5nGLwO5r3yqAqc6e1joEqQIqcKwWOQ27ZQRsNWquKh83HIhJMkK50gnJLc6CCGyfs9E4r\\\\\\\/4XSoHv2mdJc37WtKXnNkvMLuH0v9bXR8qVfu8Q4LLKXrTGN5BgtbEKed\\\\\\\/zawKMH+AQooWGseAtTUgoG3vGd6mq21VXzUGni794D8K7V7pd\\\\\\\/PFwYV2fBeBPk0lTjfyeSRhVIi5eXELZa4xHs07KPQo6mDlR+JBQuLxl8wXkpXlqy5YHlepQehJe7szJ44\\\\\\\/xVSFBvNx0yrploMnrkkydZGSkmRsOvwQ347tAFpGdHechu42WrC6fRbhMXf2fxWPi06YmOrJw+ZSK5ZgobHlkfTw2mLZ\\\\\\\/aWKqDFFvn3vQFRVZSBezzUYBEzdh2siknfXvULSgnYp8Fwpv7mFm3z21O4MNzBQ12kYLNcNEWFr93plEXLnWgNvnf\\\\\\\/ujIgnjvCbpHmo5UPg0Lb4Mst8ot\\\\\\\/l3cJr90Jk38+PLQAKqgTH8MogVNUXxM9vB2lM5WOmUQSIaBf+u4uX8hF6h4+U4sBHEIpx9llOGu2iGF9KDbNxnwMhPCw2ijurpiw8k6qViXfRCLB2gAbFarDZv4vnhrVy0TTLdKGYAQh\\\\\\\/mL1bzHzJw+Xr5xr2kSiMF3tCKja03cGzA1Xdo5Y78gXBB3rOj\\\\\\\/dZZrxOPz\\\\\\\/5zrCw+xNG5ONiUly4VPaRIx9fx30qV6S63CCiaxCtVICKL0sRS2ezEQav3I7vYywc1lHItQOYdGB2Rl8qQc9szhWESSNozqtxC88Nb1CFfj11+AqxcyYxhEFdYxbqEOVqSoegkajzUqhNMfRz6Mtq1Ud\\\\\\\/r5DSrLNQR+EJ71v4O9Z1dUGgAXcgzqX3ACrnjBViMCvyyZxRdbadolYkTlcIPk3iLunsgYiCxXUuwSwBxb4Ogxavti7N7Z5DNueFFgMZjYEPTEmKoKmpYjZMdvKjbdhnBo5m9h+0J08rPeaXudoNRx71Gilo0DyYfM\\\\\\\/sDHFvfQ9Q+Og57EcC+yhdC5mvDIE+QpgdBdGQHimfRpni8yZQ94KgXgrlKWTvnODyFqM8Xqq4jGX1dK7M4Mq5mXCR7bFJTVRjJjf32+OEksK3SODaiGzt0j8cq2FXwnLcTK+7BVtueFKFOuKBAZz+TUr96WIQfuRQgubsBgX8rlY25y\\\\\\\/LPDsy\\\\\\\/\\\\\\\/qa2+cbHIxb2FqYWvLGzNzPjgm\\\\\\\/\\\\\\\/NiPkV3z\\\\\\\/052lMnRFlQP75v3h\\\\\\\/H8ygf+p45MmBrT8a4Ahf\\\\\\\/gyx+aD16Rz8q0BsGMOja0SsA4JvSkYDxkua88GTR58qV3l3xtx8XL4nlPxYWRa5EFsT9qJ4q90cLv9REItW1kAyA3ai0CJoBVNLdS1NrgPbPSl43f9e6IvawxwEPvGnwSK9ABtCU6SojwJyOnnM+6S3V\\\\\\\/4Jhho\\\\\\\/o13eQHhDRGiO2vi51uJtzAabts9bCEBW9tbi4tqCezcBO9UYhiDd0jYH2wti7rQLBK2nPNTj5iuzXYjt75E7eb4uKbHHFKiXHHawUf4Kr1R5tVOp3EeKkTfhyVTONYJrvjSXan+DE1SWG\\\\\\\/nF87GEYN+FuMzftAJDKUlXcIrwgbYnkdM\\\\\\\/LhcMbkdvdXhh1QjN8XT+4OgwRv\\\\\\\/VVlvIcKu2eUSIDMJ9r87b0icH2PCmju4gQ7SF9nvvXoXC15ey9xIfltn0Eu5HpmDBiy585XqjLr2VdErRZyw\\\\\\\/mhLWC0WDOzAL0umyxe4nsDE85pzTD9WXUHIa4atvSfMOKARtT1hNGyeXWDBqlwO8CkepNgN\\\\\\\/CyU+hNQFuS2jsbXPxkaQy2wYVgxV5Uoi+WU59Gme2zdkXtEJ56U91fz9MH9GHvDoIQRKyWWlHqxXjZYQlNB3ommOkebGMRI\\\\\\\/RaL4LlMhdV04K2pnGNld22fUplcDs1CN9+PUklufd4wofKphrjK1Pyxrw+75RssOKfxQ\\\\\\\/a9dpM\\\\\\\/M3A2v6Ze+W3vkQcmTN3U6ZYEUNSn3NXjdn+iUPBaA7G1OgMNXmHz\\\\\\\/9++beV008\\\\\\\/UNkh0maKV+pf9cGu4ejq6u5BApDxJR7uyJiCUGjdoJ2Qcv8apq5l2GVxJi\\\\\\\/BnjhXHht3CX09EKo1nHJUwHrNn6VH\\\\\\\/C\\\\\\\/Cn2BWgnW4c1ZDQxBp3qPFD5D8+QaOowZ+bX3vCVREe2sxNlnJaRzTB0qx3sMHdzDYyk1LndDTMH\\\\\\\/gYIMrmBTGDw8la7x2uFgj4RxvB1BVToy1721tE\\\\\\\/gtsBVhv1q3WggtjsQZgzMgtL44Xs5gv4F9Iiu+3OHh0GoA\\\\\\\/a4BMMNbOFS6RHmdTl6Hsga5BCmipPpcLm9qpap9SrGzniQFHS7WqcfWhfqPRYWCEVTocDEX46mMWtfHiuhEN0pCxAybmYIHFtoA+gZS0ZwkG2Dj8zw38hnouERGX+h2ftS9+BzE0dT2GWwe3yCN7lh7VGxiAaISMZTbN+0eO6gdoHs1ZhYYjpDkK5ZL4QEKDGJlkLt\\\\\\\/Vz8QVDwgujzQ+dZ5DAYAM2Yje6dxPkkDJqCY5zFqlVQvf63jdwt3EQzHTAjISa569pOMl6dkNCdDFbG5lvHmteem+6PiYzrzPM9InQ4NQgJj4pUahYg8Jg4WbWYihhPy7uAa8JJtY6Hz6IM0UrHmamNwRL+wUMVejZpOdcqeuwfB\\\\\\\/iXwAeLtq5WdBgvWPVX6FVyOqO5ao5XUQb4CtJo9Glfx67yGMlYJCMsB4DxC7TLPXfuY3Zk+w7m6hizTpG9T5JTufHjrgkzO\\\\\\\/a3qX469F6F5zdLxbJ735SwuIMO\\\\\\\/j3npbalP6JB7MYILUrPrE\\\\\\\/IbCzq4iJ+PirM9c9gQS2qowBkBkOT\\\\\\\/rXi3ME5PudvTKJ63uoZb8S56ZW1Sk6pPuTeXrIY2M6bN8kvjzp4uedtfx50UJ1zd0eQItMSzJ6fPclz2d2lyC7qsotvxKQ8XqLJta9JjRWcr430s6l3Guptxfw5KTUIo1hNtf5rrP\\\\\\\/CeB0YQWMMnKk2q1t32p1fzYZ5eWuDUdJCZpqoXWaM+rgQ7NJJXbs\\\\\\\/7zozsjPnRe9yZvYbBb5L\\\\\\\/1CH+B8FmAuKfUj6evywsKOLxMXFyq5V9twg7hXIGV2BJb8NDaVeOL9GDwOx\\\\\\\/eOx52sVL7wlLch8odaRA9WbEba5TWU3V4rQBvjT09J4ixD4dmCM7FpQVRHvV6fc\\\\\\\/Ks6\\\\\\\/tl18jEunSRRXcAQOSaC2zuJpfZ+ESzxZr7AVnaWXmgZBtLh\\\\\\\/wygifdeZPbWv0seWGZjejZSXn9pigkoffz\\\\\\\/vdTXhRHCcQnMeHbTOV0MWLKbEOfaeackqkmh9m1tr1eAeLgleGDQvKH8+5RnSXlqpdm6BGjkdXpMORndI7+sP2TyFPhkAxQg1fM5A+8A646lEd1s+yB2t9s5Ht+ujV1RvBSuqSfMoWFGw7oGm4lz1n67SyWByiIIDow4Y2fg0GQBC0aUQ35b3rRPyptezgyDQ1Y4X0INT7mrTSD3MSpgpJHcqhfRg81YQizloNfpZKUrOtQQOJJB8b3GhHoAx4Zxy9ccGVTZLaHCg6P6+Th8vRSMaX+MQLBq+A+6x7ym+UW9AwzLS68xSjXtsMpnBGJgjL4x+jRBGaKbpFY59jk5fe2TyEKM8psBIa2OYs7g0tiWZ9B11r2W4Egj9EhwJdi61CfYQoYEbo1K0uEytplzUu5fRcbQzeV\\\\\\\/bLL8ldmfU35HySUQoyJnvJQqDqZKQPdz0r6M0Ae05aWSt42sAyqu3ITRuflFSXDd97pdbDklbAAEkBwo2hACZPG\\\\\\\/6+DUWLFHxQqPlAQo2\\\\\\\/qgWihJckT8fBw4HXU30vVcbiKCJ\\\\\\\/XlEWQ36+aSW44xxd5ww+DRw02taeQR9lv2dW279BWUcT7njmpd+3DyDMr+pNXMTlq\\\\\\\/jiMDsByDKaxzKh\\\\\\\/Dytj1SR9YzzneJwXnV4gRyeRE12sx3vQKPiuyrjTRrQT4ehj6xNMzuK2JyI6bfnbIgQvJfjh\\\\\\\/pKFyALOBlyUYAAqwkf37kwyXzuiAbB3HSZxqX5yB6wNBnIUYdGVfLgk2ZUCJig4Y8Ddv2REtbDIxUc3h0wqF4yw0PxJgMdb2RqHfzl827LyPMgul9swHV+\\\\\\\/F7+R0edtoSM57XUzhMjdEioBoF55IG18Lw\\\\\\\/vHPIR1NJCPmAXedKh3nT2WfPqds4KhdwD6tZLWn7Y1SuCFMbdJG3AmCVh71plMeCs60K2P2a13XFk8igkZRkRsFYjQOh7M3mnXd0XxvW3TC6qY4bNtHm1E2rXldHPdzvJgpcE6spT50C74sjaS02J7z21jB8a7d+xDdGJ\\\\\\\/pqr6lnGm74cJeLx2oKr0iYq1wq0mHxtAbWXnDrfTqHG+wYnLKscgns396LeW98Q61qwc8HyNHC4qotxytPhaGoiYdt3j16UdXQOj13lNFpiiJVG9Kex+XUTdr7BMnjlHNNguznaQW\\\\\\\/KaQ73bBnGzordbus9feR\\\\\\\/Pp3NYFlgOlB6QKaAPc23DbB6o5jhrGJrzVtFatjeLgM0LNxKb\\\\\\\/lXW4PUabtZIwSct6orp1\\\\\\\/VEIGheS21mbBQfAL6Wveop5Sri2RYSZZlcnUr8Xb1Lg2E2XYeJDwR02vtfp4IGST1vIvxIu7W3xZ3IjddXQs3Fh5jpwy8OPI1L1b6Jw92Cdhz2SDTgVZInsIXwljfLPgQoy74I2pzA9OGZuqciEKiiRGpESg1IipSNrJeVTgBOrqP61cPGSEdeaKpXd4125NWloKtqpo1pEKqwDJ6b3jxh26KOhzZTi86sHHeO\\\\\\\/AYwunkqffQ2oKWB6VJ+aEL5dEYVyQu9ml3x5IONFeT+uSJjQ1rjOTCVjFsIWD66M666iyuTLlZrj\\\\\\\/4GrY68CjDhh\\\\\\\/HiWWQ7BUtiobDBWvAC0VBTDzlRxh1L7GgWCSZYFsw77HU2xms6ff74U1Reob0xqBx3nv88QqVZ2xe3IJlgDdWL5ZqGlsNpUh50+kZXm8urCDPWPRRLnVntPKqZNyScck4iZzeUWz\\\\\\\/tGkrDkX5gyqo74Fju+xuxo\\\\\\\/TF1GGnu0cHXjP+XhlDFh4IFYUzMbSOsoffO\\\\\\\/WfqkhnBFoRAPdiQAjOd\\\\\\\/h5Jmd7RgDtA05KRaAl7famEiuB3FkJbG7Co7bhTFV638OV0ioce68NOPwFjalXHQpm3h+JOALh3z44drUwzJPF47l1GQuY\\\\\\\/wc+6XDC60bxnUwABlA\\\\\\\/9sapfIT17tI7S00vmHzAgGsGl63ia4r9rhaxiHJPG9AXBv+CmXNDVWai6NZMa\\\\\\\/vH\\\\\\\/zqcUmkvoGpf5KRZtzoeENH\\\\\\\/sj+OYPWAU+R5WKf9FPZpN0BbfjEww8mWlC9wydvoZcnvHMTX7Z1i8OhNTEoxJaivhCPUgoYNFGbuETgGph1uMwxblL3PiJhuXArsYsWjBj9ZwMsPn0oH8EqNcytQz5BgMVFZYwNhluwxSePDippzJup01sMmibvZNby4yGKktvd5MDMplHLHqt3t+mC\\\\\\\/jSYGE2YsvNBcIJXkE2tQZs3NP6f2lIYThkNkXYbv3EtEoVliAi2YjF+XGj2shm5R5Hb2AtqlTpmCs0sfXehIxAVXabEGW0ChsQU1LsZtoodx1D6JT\\\\\\\/TqlOGqAVr1TE5z2WPhjhH\\\\\\\/1CooKikYXD\\\\\\\/IOkDY3EI2lMHruZ7td6g7tnoDbSqFHAXglym0pssQSuRdBXHv\\\\\\\/YiFf9YZbokJJ5recMnSKGFIDYPcvBDvhpMQEsUf4\\\\\\\/5MOCJw6Jdch6R0fnDAgDw7UIBMNN9Lc9lsz2dnrHAPKSSLvAE+dljXyFtExILao6ipRPsBlcpaza2i19Mr5DQ4wA1diVU1V6vZIwTp9GvvysQbxUp9KjIFj5eSwwQywEjDGDHWzKNp7QVgiXZPnHXBtUS6OzCb\\\\\\\/dF9j5aSrj1GV6RBsOYzRu9wFVTjQVIuD6TOvgiiB82qXv4Q+thvwDdgzkeUpwoJStuHbSboFHCYx9ifIUn5IM503o39iZ2IIJKWTtfDDa\\\\\\\/pYkdO1TQISDGSG2nWuhP6UBCw3IO2Xro\\\\\\\/mLrsdIB76qzz1LDNPIP8fM3p7VlvYPOVYzKahoY7MOMCM+bnlRYouQCxX6iekOtPxulaWreyg91NGdc44U1DOHdYIESK6FTASZNmgLMTVKWfG4GSKMDXFNTjRLmlrmPYldQpgMgE5c58BUF7C4OmhWyMQKJBystUaLBuEWX\\\\\\\/By5I9HhQ3FaUmHnFW82zi4xGyJch\\\\\\\/PMmMRjgIB5HKdP6tDIU\\\\\\\/EGDATYAooilrl+FZXd+0qjjfhIF+MsbRKhYrCb68BxwPGCkxs+CRWDdD\\\\\\\/m3\\\\\\\/7AE3fhULp9KOetqujw1953cMDUe1r9Qr+CAQFBRm7cQKbA2zjw6t2onH5hN\\\\\\\/0emPLld8R4BqpD3wRgp18YBWeMXHCba8\\\\\\\/Q\\\\\\\/5RqbCplyABcMuBxIUWKEVBDE3jbHs19msdlxAB6iueBA1XMqXKhQ6XUYs8a8\\\\\\\/eM7p1yR2ZnsJiv2J8qHvPJcP7Cvm0joQQuMxbZ97kde4dSYQ4XUiMZJkt9ESSah04UCfk5nsI6je\\\\\\\/4d5u+AvT2FBNxQG9sfQAir87HXeFMIPept0URsRNRxPDdHE5o4BlirlScNrfG+YYBnUidIGYt9qhIr5+r3Esfy5ndmYVwgFvRbaubwOGMz\\\\\\\/4VeYgw1D5yLEr0jiTDs77VeIyUY5\\\\\\\/S9rRWBPuzRGFwJL9798xENlm+JI247\\\\\\\/01o18TC\\\\\\\/hocZuctJn\\\\\\\/qGeqWLtvhwjyBcrMkBWA2Wt167syot0Vzqoz0QBcgLjte1s\\\\\\\/WULFD7+O2rUVKGExdZ+nhgp8Co2lSYXrlu\\\\\\\/kuIpl0c+ggqit6gIzh2sfQJKfzh6\\\\\\\/Pcfg\\\\\\\/6ifd\\\\\\\/LVBBuhX\\\\\\\/y35xHbKhbWPqGw6o0L887hFv8Uw62b3XrkvqjmzQQYOsGKMSIWN5EkRcVxsGMYKt6jX9FD5CFcXNroSfvkwrGg9ML9dNnFvRJ3WiizPSZMnkN5B5qd+EMD4uKGWXqje28UWcoc4M8rUHvG0QhIiXvWJBfgHGzSuzoLdk7AzJhtTjVX1GlrEYOOBBXDkL1c+8u3kCtSOIOGpCn8eWur5tJREsfOpeyaBsrWm3UTxW0KSzPXLKKVyZCKih\\\\\\\/mwS9tggIlGsM3xVmh6LWnQsmMuwIa1rMj7kfN07Iz2Tu+EtRkLQfkd+GfS7yVpRkpL9pWr\\\\\\\/z7PkfKHWtSMeCNm+ZvED1990KIsUhwgif6jA9bcTgerl4ec6F1g9aArVL7y3g9y1eojvuK+P3DP9bdqcdhMTCYDuboDx\\\\\\\/sjLaHchoWQtER9ahfAbEKYX8lFhEjNhv1autzmNb8VG95xQYReMYAbG3IgYlPZ1LviSFQvd0J\\\\\\\/XjgkYYDvg\\\\\\\/aVzhz+nykrIAqTA9jZ1CBEnCbkhtW7M++UqhVwJzCWwrB2nwdpCqHHlc44B4douKaGc77y322fuiqkQokznCG8bA7BKXU4IUrS3NOpIoYi+Y7cuaxaSNx\\\\\\\/3fjNKKYrD84C3CRzSH1YLHFa8cMHdeax1jOouJq\\\\\\\/ON3d2m8ixVQZY9QpKTFN4fjw0TerAaAqGcaKTCtFJpSZWo6gb\\\\\\\/IuXlAbJ2xOMqej0JgCNcdv4MtpQ3tSyweJzLGX+0XSUbvw70I\\\\\\\/I8\\\\\\\/wAepAyq8liyCXpWZ9oX4EYJ2xVpiYIXPjazS1PadJBpw7Fc\\\\\\\/LLvDy1bhICHoAdosiXJywPdFOTq+dFyoo26rRsVW+A1dPd2zKP14aJFdL\\\\\\\/JP28uMcdP3Vj7b5AzKlmpW0Ni242qoqw7s\\\\\\\/DpxdfLrW3VwWyHx3u63wX1HWTOCUKfxC2KpnPBFlD1la44oeP4sFGpz1iueMSZlCLhe6B0PZjJ08k4M4SUdp1AOfrUR9c6KuP1tXOUoiH8jMZTNwJZ2J4Y5dPXJzfeGzAyYfH7tQ4N3VsvqLWZCC\\\\\\\/0zl+aUh4SuZ2gcAIFnLOkqJlsha2+NfEyDx\\\\\\\/ar59OTMigM+oEVEl70dF3Ayy+GBxzx2eYmd4q4vXhKJlpHTpi9EroVWv4p4gN+tU4+737lEhbrNpsEoTuU7t4OKeAtVhDPjB5q9ib75bwm00M\\\\\\\/1ADBmFVrOJHAisFF+I3wlyOOK0RSlyW2yiMZFYkf4ZhadCMKKxEF3FckMjPGp0kikQ5tQ9+ZXdCb+Z56sEFL02lXYBNQODCONQDERwYELz9yWWs5Umgb\\\\\\\/V1TGQyWn\\\\\\\/aTJkMSqGFRc31qhhhC26v84kYtGdnbvn7o7Go1pDaqHfEK1dAMOn\\\\\\\/vPlO74uIg00sOlccOsQPZzw+uOvECO22oU9121wS31+aCytwoLk9iU27W+9IG6fdjvLkBMc4lyvgMPgztmCkeRnGAnMhq\\\\\\\/0wSbxFgwtAcIvPJNPBn7frL6bM0wZHKSSqAYDNRj1jNG03KFqM3kQosVH9jVy83r4KURE4\\\\\\\/EywYFZD74Fv6COU9ScmIwUdBV7uJgpa4OcXw44yAuPTS2RjvJJNZBXtU2s9pf12ugdlkTDGnLW8DT2jUeuCaqlHFq+PeC1nyQs6YLEsQkylmcEJIneopBgJ6tWd5\\\\\\\/FI6P8w+c32snx00QcvrfJ3TDhpTxvv7agOTLIraHG+rPwXRHsm0AI2WZeZHFRj3va3X7HEsJGTvShGnQ1Ru5USIdeBo3v55fvIfonggvDu\\\\\\\/205BK3AHmsOUsc0VgKAhR0YSDghj0c+ZLYZtgp0lbIhgJ7TyQoejkTEfLU9z+1XGrbl5SiP6DoHL\\\\\\\/s2pDws9Hx9ViuNo0F1G39NMYFt6g+ejuzz1FOtknXwCGiSi3cBHuF7DeJ4zVpMHcol5e178ZgmOTRMtILPMG+YjdwAMvbt3Svo4qSNuQW6v5501Qn2BoL7gBN6PHgwpOfgNQ45\\\\\\\/gSjecdhbkraxPcvZUhAPmEAp1ZjcgxSuciqxZ5mxwYPwYOm63qMVxHiso4Se7idoTDcdLxz9PUSXx\\\\\\\/pnXZBSE9DZaPbV5tJoYhWvlR\\\\\\\/jdzSZRRL3DfNH91h9DpvD4q4SEA1Lj9gilK+UTxC2ZYiZQlENskOL9mG2YlbZdPcFluVxu49IeoYEec2C6MT7sbiNYMKAre\\\\\\\/RCMkB7cSGTHmwHKmPnrJVBJ9UWV\\\\\\\/MCXkZpMH4YtRHy5eggzY0IgBGvj0CjmBsPPJ3dXTUUPox9j8hq30GrP4EAgxDjKcSlSAgkGOvbn2FjYhcKqq5dnckceH1OypOCalyMb4ExXYckM4XTGFmegitaS1SH0+8VPbxnTJc+PDj6\\\\\\\/JxggHQh\\\\\\\/dJr7iRo\\\\\\\/eeOhMMomtOT9T9bI2uheoYjgGhU9EJTLZW9a+VdZvKPNvCnnVdlgYU1Hc1U1c\\\\\\\/CAOk1PQelTxtGRgPrCtd8rw05GsMnoZigxBJvePMf0okHOft92gqeZ3uvhpDitdZMYmD+SSWWWvcLyAUXmWIzZtj8\\\\\\\/q\\\\\\\/qJQzF+5mLbJsBzbodCFiGEOkVWHP7aOOomBEjBjPF76cORhui8FpOXMoqasuY8slVdTqKHjzcSTDw9xooNdQEqI\\\\\\\/kHNqM958\\\\\\\/6b8b5mLvKx0btfRftRByg2z2ybfNvHsC5DPPBkVNzOtHZJXWyKDPMJS69xO78pr6Z4CMcwdMiN3DkdXhJPaKXhtgk3THc+mQk7bfQtir3SIm1HyLrnUJeEh7H1q23lNotv+x6VD5A4bbVHSTdSXr9aS09ha9E20DZrw8len3WRWFnlAQpgQBeHoXPq7k1Xhj8M8GMQHipCLS34D59FYy6h6zun4DGZ2A5dZtZcijjvDFZeMi2XsSnBegRdPwt6q1xVt9MjAhn4NyfQA9LiCuAXxqFQsAD9sVCOSw4vfcfKJLDQH5YA87y+206ylWzkj8tUXDJykCS4eW8hYNSiehUvAlZJUTRYllXpyxZQxrhh\\\\\\\/RGjvXKT3wgIcC4CXdbwPzUboS4OY3VzG5m30wmhqkztWTC7wzkQKO0mhWbw7yZYIxVS3upeMllNV7c6gVoyRiDIAV1pCQy\\\\\\\/zw3\\\\\\\/YYKeRzzPPUECscftn1MLn98+xlcf0ZJY2LxMolE0MAhrXjzDFOKXlZm40VpemXj6QaCm\\\\\\\/LotrC\\\\\\\/uCify93dUmGZbfMCuoBNhNmY\\\\\\\/pkMe7Di7Nsk2\\\\\\\/T8S7xLgtx2vLxzpftdLreozBR725y\\\\\\\/8DXRFK5VaCL1blAQ\\\\\\\/AaIOy6IJIMjnFVGoxuJMmEFO8sBeuYjfJGGHXOAg5K2oo7QrbQWwBTZDdZj14a67lirmdgg4PLUk+Wc9IUnqvXt7HzYY\\\\\\\/uG6zQMQU1LdHpyWS695eFjgqlwgmebEYm+h0CY8gJwl\\\\\\\/S6NeEEq\\\\\\\/Q+d+hzN+FhSnmr9STvWsnIRaUURvY\\\\\\\/IHg4rufKzGtK43heskwhvkdMfhFHu3Y+c\\\\\\\/EuAYJNjzl9TL\\\\\\\/qgdFYLp5xMTWPoWhh3BPragewJln+xwSNAfkWW6wib8eXhzZ7XUOw9f0H0FAX0hJDH8kTA34AONmuvWJktS3HxsD4azfevbwQan9tD0FjDiPXXsJkn\\\\\\\/nLUn5DmGp7zC1dGnzaB5p6e2AF4g0V23NIiKSmq5LkoU2OrwkydBZ\\\\\\\/M95DhMY9EPSAZnp5upxH+DUm6xSdaVEyPEXbI0ngTBZ85FMqBeem1IQsshDibRxuQvt0Jsj3WB9tugS+f1LzBJwcIY32NfOeLtUHBc2sQdGEzYoxHa2EFYF3ZAF65hJvrlV2rTDMQBEFhx0OQrUiGW5oSOmh0N8qLUQQljKj8qJd52iVZaw9UlZ3qk1OsOo+8EifhBj1HJ55uhUt6uuJ7iaKQZVMiDnCdsoyFLlAgyxSmfFRm6N\\\\\\\/GolnPt9tTZcrIDA6RBqzSxgT5v\\\\\\\/9DjJU75HYbWYHkcBNkhzwdnLcUDfUY6Fb2epD22ZMEESzDLwu97rY+LiocLIcsJFGOWIrqXBlIdO\\\\\\\/MjfnV7FZFmnnjFy8UZ2qd\\\\\\\/7IKdKGFO12d\\\\\\\/UUqSzehcR7PcRxXIvW\\\\\\\/YwGT8QTCTknpn1m0Rz8purwR5RzHmKSzWxnEEDuI5JkzLxdvT7KetTkFceuDAuRytK0FEfyGdASw4wOrvGHnYEJIURsbsyvRQdDlX7czlOL9RMVVsO0L0kWk\\\\\\\/UfHly\\\\\\\/N8soeRjjd7s9Q1AwvyeOvEL8aaCfYy9H53\\\\\\\/iDe1sNd9268grO6aUuQq6UHxRlQMHkccc3QXziBUZL882IwCmC4X85UAbg+snDcQO16mtL9nl\\\\\\\/VovHU+y3gxdsywdTrblj1P+w1SeHxvWiicDjUneD6C9bB7an9Bqe6SIPs4fChM27ZWhOZklNVwif3epBXyTq3CEzYY\\\\\\\/lEBevPeHd2tVZmsCG1r1ck44uOZLgAjn46r6w3aPcWaj16h1AU9XgVarIDfpsiFjTITEu10nF5tKWuM5z9LjXfVTROf2KlE6IjbP5Qk+lFmEvhsk0gX79RqKSO2XZZsaRBvj6U5k3cvZr1r04PV93NMzkgVATya4GWusybJNoDEq8sMBbm1pOjQpcFCohbuLHa89G5sJiXD1CzxBbz9tT+N2nj+LiIeRYl9Uy1kurQppzpbJk9zFE7YOvqDkZD1pNpDM\\\\\\\/94VMi\\\\\\\/MzlaJnpm8bI\\\\\\\/13lVCtkvqo3jSuHRBt4tofC2MuPXbmeSR5wFoVCJX6hy\\\\\\\/fxIXPe5v44Nyl3XNDzU\\\\\\\/4OBTzL2PdSGOplk+6NaPjZK\\\\\\\/Fv4\\\\\\\/nHyPwEre5Hgo6Zrr8g7J1wLUA7yvD7IgoyMuPqtJ0nNpfYB7xtlbr8gIzLAlPjAuVQHV+YDLB0\\\\\\\/FUgo8Fvo5gNK0jJ3Tu9ktLvFkK7j61onVGHgQIH7E+YZ4bJ2AMumd\\\\\\\/o2hNz\\\\\\\/TgeWnZQfm1fpfj9bWeiWb\\\\\\\/3ivMUuXLx0y9N0b\\\\\\\/tf6\\\\\\\/4DGKtijAXdF4L24j3pHsFNE8XyWqZIh2Or6YBpJX501fxJM4knGA+1zUj7+zsENbeN5590G5sC\\\\\\\/WKGF0EMW\\\\\\\/j7FsMhsIqosj3Tj5LsfOrWwpBaBt\\\\\\\/pWHPAguR+bMVQu6HNtF0\\\\\\\/8KBd8XKr0sVerahROpJKShF78fFsS6BFk168KwhSS+YOw6yrM6yJTYspAf5\\\\\\\/eEkCC6+PjU4VBdn2eyEMG9aPkcfhXpGLl85GnN3sZ58gAM9LBq8ds2Lcyk7vJCrX3b3iRiiS5S4VYOdBSsKXUU\\\\\\\/SoovEadxhFOsQRsKZsFE\\\\\\\/\\\\\\\/w\\\\\\\/Mh+igy8bE2Z7xShum98zHO3SSYZvVIfoky5eycjdwynCvZ3\\\\\\\/N6YFu1eA2QC\\\\\\\/JKcZ0RVN5rE9OsFDPk80dm0fG2zGJC6br7Bq9LDr7kNarzO6fAyuzeQO3CSOSjdxY6g\\\\\\\/DN7RiKFvQZ+atz\\\\\\\/RKumQZdNzqgPurJckE2td+MhM+XivrrbkumjuB5EQLObv8SOuKIBWqMxgaX82lcEmh5D6H4TI4dyC+hJeNXOZSz6QYE3qCOs8c+eFMLRi50Kk9uZ6V3+spGx2x9nVoJ+JZM42cG7LwVi72Wdy29BsQqFhZZYhwJOX0N5NdC1EcAOEKVfYwXf80hXr22p+cBVs48q\\\\\\\/rRR4SQIU40VwH9t4aQQ1FhJ5uq04464fBE+\\\\\\\/7LLTx2Yw6W62T03Z1yyrLA7SVKu1afHSlupJG1\\\\\\\/Cgc3mDPyzpn6Pk09W4zK1ImSGDEBza1KsxMLbl06LaM8O64hZgc\\\\\\\/Kxo8RwXO9x6AtbOkVIt0U4Mkmkkyj34jm+IJHY2J23kIlwIguJiqwwJtsz\\\\\\\/ftc+D1eQ7fnbCtxbHyhvdQNcUvkoBZSMZDqDUd9oXK6g3ko8ZqdfS\\\\\\\/WgE7hLWYFJYVH5LfEYmZyf1UIATOrYMOQjE7eSQpGYOQSRuCzMFW9ygIZbd5uRhibwnzG+9rwamxrupxj5G2T2HarWYamkA\\\\\\\/BDBNyhfHMb4HQc2txx0TA+xrlzLhhIordVk9YZ8RlcyageJ4xOkTJlG574R4IWRx4gK1LRH+cBLRt4SrFiNsKg5J2fh6WB8BgkirT2fpg0EhN7PysaQL5j7svz5fUMza\\\\\\\/22gCJHgBjfbUtVxC2nw7qn3JKcEoMaWACR3EvbXxo+yxWm4fBCZCb5bRtftYMfpjhnqOBx6L24eEerXYBMiSrVPXlzevO\\\\\\\/Y4XOgsPnUMMJizvPVi7b8gCtXFTIFAoYC2OMRAYbwC8imn4AKPfjZIqC7\\\\\\\/nwDVCVnfaLx3\\\\\\\/f2hRGtcpLTvpbWRm+WbtNTComXNiq66PEFsxgVej8yusmRbVKLKizpceaSl63MBDcIQzckaVmfhtS0Qj16y3Kt5elcETt17UigEUhs3D36BJckj5cWEYNY3FMxU278y\\\\\\\/7txUP7HTerIxLqM2qjMv3VvH\\\\\\\/NOgdMf\\\\\\\/UH5aPR0o8H2bfZ9FCQsqNXDsL5NlolDKPHzpc0SJbVytzryeTCwwerlUYYkS1RlEL9xEaq6CQbhThIIT5ViosfmFGSrKyF1XRkhvbAeD2Lzp3KfWy3ZVbFLLh8e5q8M3sPKvzENJsvIIzAYvhYiqkiWD25S0tFXTQQJ80TzImcnnV2XJnVhsi2HV6j1aoz74epbZUqw3Lw3b3NSq3vRc1liaoNlN1xR1VC1Dg5HSj5x+5iQKyc2z4QNheGNniZ0UZaupja2yrbpRoi4QjdIMpHKjLn\\\\\\\/mLW4RFSuy8l86z9EJ0lEmhSb74pKpEMyyeoWqY3Rk7+DZFiN1hKtzIrY\\\\\\\/wBQPje5Z5Mw9+ZUwjH83NfN6JdeXLu7XW72gatb8CEvd7fP0HNLyTSerpWi830QulfgDWRuC4HBUJPYgxaOqDqes8ohMUAXhGSSI+3hFDecQcj25e1MnVTq+fcqrXyWAQrPNoPb1XvCjDX2vYyDzoRtkQEon5AV5KImZyXApefEMNch\\\\\\\/rEHlZCEjVZdQW+nT3w1YtWjoeKlRrTBxIKI7F6kK2CCS\\\\\\\/hRNa39xY0WpWTdZdjaSKOCg8s7OeFxvd3pGmkaP+1eMxMx5LGDS1E5FQrbqnb4QnCu6EYQFYpKsDfr3+a7xL17vJ5H2QRnZfzCUFoQHroYkEJXUDeBxKM9FjoWcI8nMzDcNB+jHU2qXrx9rnR4D1PaN0PsyvdixKrXehVshjGICbed3aXRJhh0kChtQ91wOjKjELoxF3pAnNKEfl41F3TOYOCMeLQN613t53ZzisA6nhvdketRiEUd1efEgL7OjFT4JwXBqOkvcLa9S42HYTEL02saeFirPdTlLwrn87tdcWW42VUcbMPg7jjAfVADeyxWtmtPyDwGTMHYR5i83znJUkc37hQ8DXk8dAmveajB5AGWjn\\\\\\\/+pPWLteeFZpHxGbXa5Y4w7f2r+vw5sd2gDIPVcVu42qb4fhJKoi0+SWgFTt9kPKhlj6UILySE\\\\\\\/qCcwFYsihgp+heZoa4Mo\\\\\\\/S22RAEDMsulVMmvF4Ck3SLf\\\\\\\/V8CQIqH5crnXk2stSxg6MeOql0HnAmOyYXJHefYf0IW97vMLCxHYB3NByCQHoaOTcYYrVKvc7C61lP1Y3s2+dUs3dz68t\\\\\\\/IjssEWoMpTOrhxtyc6qJPq3l7YoN42afFxIly1LETwUhkbjrWycD37e96GA06V7ePB+2Sj\\\\\\\/z9j1RzR2\\\\\\\/sXefXxsLhSouWfk3gp752R5kbFAX\\\\\\\/1Bl+fsvRdYlmHgH+TNJ45nqakxuTQmR6lq2tlKrG24MlCqsaC8w1XL+5rV7DJfKfDJbjwQiuFBJsB2J+V7pg39CnbCzqJo8Etz1QMw9fFg2uWHOeLHYkuyisNm+laCzZ4LCKg3kphFnF+KVqd+mm9JJ5MA3R0HVsUEkQmkwNOlQMdAWUSu8pXncB\\\\\\\/OAokT0Oam+4+ZwPwElUkWGoAuUqSYaNJkEtYCenKuPxM78n\\\\\\\/hVzdrRptbwUtcqGTHxOPDfDEP84hNhaTEQAbZGiMoGQCG90v8AumFeRRP6vS3LfqJsw9h1668\\\\\\\/vSAHnIjyN8yQf4xmJyEpWHsfHDHnFw6jFSRJ9Ieu39AaQTe+k0gI5AWtotcFxVI6EORr+MofqaK0nK+hX+BaLnVl4yMTAftktEM0VU6exXkPZ6+mEIrvYhND2ptRpl+1rA1rjTxEWft4dsL1o1L5jaTJ5OeiU\\\\\\\/EAuSVRuKnqTcdQyzssBoqUenPSl079GS33bKiX5O7UKF7yaACRCSwyi1UcaThlf8njuc9\\\\\\\/mJoGjZ+hHYrxNaz3\\\\\\\/kuzj4yphlg79LDOjaD6XV0Wq5Glm3CmWroDyf4xxRAAbmsZkPQEBILLAvGDXJRrbf4elS6XTqlNbtxz48UDhe5WYIpXTE7I7XQem9HncI8QmsR5ZzaXQvu\\\\\\\/IPlyq+WGZdYe+zizxhySxEKGqgdetiT\\\\\\\/\\\\\\\/plab+Hqh3q+WccvvAwMks\\\\\\\/1A1qsqS+aKHVPXgaxQguaU9XOXfb8LHAPRmPTbj9FJ\\\\\\\/5\\\\\\\/2K\\\\\\\/TShGtfBwGJl6YddMb2lGpm9HrrKjqQ9GJffVuHEBGJxQGSQRpQwVLZ\\\\\\\/AP87Hm5G38mJCmUuPme3hGfg3c2Wj7cu8iD9rUU\\\\\\\/vlcW8F+760ZsUrean1+qSeE0W1774HhuoIxiw3zg0aDtbMX9VnkNbXXzA1zxyDpy+pKC5CF9pszzH8LP4dKyGpWQknlwZw+fjiNjbtUDVYmDWAb1+qxJpTJODqHCwPCEC2ZumImwdINtf0T0hmZ1BZmOcDp0r\\\\\\\/3jSRdTw\\\\\\\/xZuOD5BYYS3tieB6f9dPnLZeLFyHTKO1lSFxA3MrCQ84Ho\\\\\\\/xWgv5hAuWtOCgtQAZW6CwmmKLlZDA+B1IdBQiwfSlAqmwb4AnRrWT13LB1c405qvAoT6LkY84czvq5kT8NaFj4Le5fzpVIaGVt5AwU05PenwsnjMYDDagdt1oTQ\\\\\\\/Lfl+0Po+jI3Feq5mRO1T03g\\\\\\\/i\\\\\\\/l8wWOUTXOy50D1vmkDi4\\\\\\\/G9Omtk2BMFhPwFu0JqTVusiJvDDmlzpUk+KjNQK4ObsaKGSmjvwj\\\\\\\/tpNpyh0Q8fyDErzHvL4PqLf0B+oQL6NWqhHeJYKkdY9zvqXzUf+gAJQk3NWW6rpnkFHXF53drFUn0n8GDqyrzMhbEW49nmcFPxOrxuI97wJqoW5nUuhVFSeHjLm4jNkOGml0+cNvJ85akGOYQ5iFn1FsLAVm4DyD0d2jHvhFktmszwgZ6EfSC9yZedaJY7cxsDKq4qO+ZSpIaWOB3KK7nb35WFZ55N2bZtcPqjpUOknkMfO1p1kee1K0dsZjp\\\\\\\/vDSP3ps41t42zHS3kjY9spL18LoyVlR9uLMP\\\\\\\/lYEFgqz1+T8wWUO90kaRtAxJ\\\\\\\/fBowlKV50bW7zABgiWrvSPDD7oZ793mhi1OLwYrYtg6CivZ1\\\\\\\/MPBRqcOI0j7vJOgrNU8TLOxKD6ahfg0FouI7fnkqMvz688COmUwwxQaEI6RJGbVALPziDcKvVvu6D7QN2QC09vhIIduTsDHmbECz3UpEr+5qzAwtW3CS46tFStYe78yGQjOBCXcFBswLVWRqzurNVd8ALaevwxVXjOiqR2k0CDozYL7zamu70q9Q2tjiZDeXFDfgmka0U1po2XBWHcgaEkbgeO5cyXVr1DzX7uN+0OuPRDSqkEB0QDsCiWPfCjLloem3k4KPj3zZNv33kubEqhQ9JoLjas6zRV4Zr6Wl9N2+v3kojHnl5Q\\\\\\\/IjF38T7Vx\\\\\\\/qFUwXtreTGTv9kBJONh\\\\\\\/WuKGVEMuho3XxMOrqBfsygWgs3lSsRIugHbrVLl2bFLlYOE4Ef1+e6uKHi6heGNSnlrhQOCH7La4\\\\\\\/l15DDXsl6\\\\\\\/4gUR\\\\\\\/jzlbvUswnlc+53Qbs\\\\\\\/RXYaG7f2wOUOkzz9F8UO7Fiesh84LwJUfh26ASEprj30DnQOR3N1PIE\\\\\\\/LQBE5f1LC9bEmkeAFj5CdX0xcdRXrCfC+hujCrB\\\\\\\/Z\\\\\\\/xdT0Yl4UgF7JZpqR4FygqdmtxCuj1xYuSgMHSRFbqK2WcftfrdFQmffUrBnVoICQrJ7se30rj83oR6Je14WHC4SsOip+vFgGcr8xX0S\\\\\\\/imPCjAQUAO199OKy1VOd86uCOZPAAyr4SBh\\\\\\\/BoAGoOcWLRXTDrcYvXBaHAmJf+ws59JvNooOIRm3s8\\\\\\\/TdukEQkEwf9nL8Fm19Yk8j2rwUuR6t5lWRCtck4\\\\\\\/EcAhSnG47UeM8ucsXVHTUVSuLhsOUTrdklXQOl0Hr4txqgilDVfbmrX0HE0TY3G\\\\\\\/BF6RRzb\\\\\\\/8\\\\\\\/JOoHfhzNW1KuGx2one0ZUwR9sBfv4OcOEbG25wnZa23fCsB08VKH0CfiqRlAM\\\\\\\/PloR5BRGALb0dKRrBxilStPwQaNiU00w6qK79pjqJagEbbef6VAicnmhLyr\\\\\\\/\\\\\\\/w+pMUfsLVXk6z4Qzdldgch6lFK0YJhTd293ffLdW3RnpkU5o+jeprLSs10VEmbKTB\\\\\\\/Ak2ZJq5YuM220uykohYP4UDbvg4Ch7SK+65K5ojhFpKAOxvnFzw\\\\\\\/esF\\\\\\\/g0Nqhr52VCa+OuqQGN3Flr1IZMT7oKZGz0KZ0DkXAp7BHysGBVHh8aRB5JbYCh4Tk+PeS+5VJ\\\\\\\/Hd+F\\\\\\\/9vMAbXuuG3kSDfDc1ElnLKwQvPzkNld2Jnz78HO\\\\\\\/P3p5G\\\\\\\/SrhVhXNkllH3BEcW87\\\\\\\/iGxaMJPvCXN1IelZT\\\\\\\/AnOnUBHprDAP1U+yOlNV6pFXMUjH7W37YQ4DZMQCE4uGew2jhwxXDK9VArIXepkAttiidLdaqg+ZYdUhZ5M\\\\\\\/NjkQlW\\\\\\\/HJMyBmMv5AI74yV9bu5rNvo+4eWYJOBmaRTAv9jhY8baDvTNYC4FVONPbrQw712EX+GF7ae7ofe2F9hw0rHlky\\\\\\\/IV3q44BzJvkTAMxUPF2KapjvGR8ytCNlGMKm8kv\\\\\\\/dSS8g94N2MujCthNH62E954XyxQVdyhX0gjcXhwOpRf4+oUkvsoibiX\\\\\\\/b5gjvluL5vXqhXVeAxufFEg2qdeurXaHa\\\\\\\/oH\\\\\\\/J\\\\\\\/bdGziozgEoQVcVceidHW3saaRyBMUI1vP5CmsBLTJE4pJaGMIFLH+\\\\\\\/J8ajNg7Kx4b7jPHeENcOgsS\\\\\\\/xFXv\\\\\\\/MdztCYpZui6jfI+L6TyUCfP6Vl+2ZVeiZqfzw8OtbZz0lWMiRtZYGNZkdom3Ur23veX83x2QG6LMzg23n2eZWmFZ5C40bvXl5JJhX7ID+f7\\\\\\\/idYiHVPUovYDjoIUZy8XfUwY6r\\\\\\\/QpNhneSp\\\\\\\/udr9QTQz1e8iFaCs2RNCRQbeSbFLDQYg5SSddX193mkzRXxMjcmC8XgKFXoPMUS7s7jVeWIYeTEaKeO2IGL4ezo2wHx0EgvpfhzQHgPdx8ampVXeWbs8dVSUq309GNMOPQGwcvi3vn\\\\\\\/\\\\\\\/hEG8T1N9zsBbrEFN\\\\\\\/+lgGe4xIjlMY3E2Gq7lP2\\\\\\\/SK3yG19vDmkmvBGeW+Y2VW8ou6MAge6NIneha3QziaxJkwQgIKLmKIhvt5IT5fb2y9CCfJmWsfrRBt2yFBzfqa9TWob8\\\\\\\/x4RO3FCXVaR5ADuZJF3PR+60WtheE9Jpdd+KZQAOungte4jOTNhCgxAMdm+5kh1AeMQucIb\\\\\\\/zXz6fCD1kQakABdv+NO5HbtICjsi+phREt20yuFgijWy2nMW8T0Rqq3L1G7SN6zySOpQU25e4JLbFAqSIHAg+Cehamlt9Y2KwvrP4tFPsrGwaNSCH5OaO5tykBcGH5Hc9UEQxfeYBp9q6zXtRxJBy08vE9oxXaQZGyd2vkK1hhL876JFRHOm9XeFGjf\\\\\\\/2kXE2Gs3i5pgtUDBwlpnbMwpXaetzWkBv1OOuGVOpiD5Xe3u0DOJtsPMB9iOxqDL0aqunVCztTIeYtxKjtybIJeMrM4LT8o2yiSBMOCcSEXuJ6F\\\\\\\/dXn\\\\\\\/H7uCL+DdeOh7euIhmZ4uEhbJnlMGVJDWCdZvlWBUIXiX+KDQZ4jTtmoPsBBvMxiv3acEQ3umpkOU5m0a5XXxjEXI1SEMHTBiE4pCvZ1ZnCgyO1+4T6AINZTETND4EoRtMmSuKnZSDkBnf4HgN6PqjpOfEwEXjC3zLbGfnYMTZU\\\\\\\/FXwr4vaqIDLpc8nPAmb4XhGAWf1T7O5wO3iYLK8l8BABIojI+vxq3cbQhByiyHvs8wQYZ0RD5J+sQnXbcgUMJDpge7En+snKgxmlvfPpgS0HaIj9xw5jwtr4bkiYqdLDt9ugoPlWTJhlUyPLEt1y\\\\\\\/+MFd\\\\\\\/z28GrMYp1u9ELyovmXk5cPNUskdRGtPpwnRM7TJ3x5a+rMmGQVE95IH6e6S7fMczqUyNfcrZpQN0QsfQNt2juTboBNLPXk\\\\\\\/8qh3YuJwRpRJQInrxgpLKjrHk05Z+iKhX3TvBDsEcTQ7fMMzHeQFah4Fr5wbRF6OT4vAHCT8qnTAYCSk5TCUlhVkOBFembDvK6Iq0PVjSHj9hQF1PgFOeWiczxi3eQOHAd4X2sr\\\\\\\/t0YNRTKKZp7SVXQsV0FXXH\\\\\\\/eGTvQUAMY1XaxK1GWPQsrZe+j95p8wQjCD5LRJmK9uXgO6fj1oWB\\\\\\\/ICwgzDb\\\\\\\/6MrX\\\\\\\/ozfrXRm7JbygouThh05bzyVNFRiu0q9H6qkv4ZTgBu\\\\\\\/Udu\\\\\\\/05CObaZzBDIc71dORAfyJXe587Uy9U5CXyuyOTpitck6Z+mlnzvSr7xFECmIgfPPHjQmQEQ5mAlhsv6gblv9VWgF2hwAfafD0MMzkcV5viW\\\\\\\/oJaTt4bwkXTX927tW1f7RXQv6fvXIs4E2YqSrwRC5MakTyrVnLCqJtEQeg+GP\\\\\\\/ffwg3nnFzoI6iKh7\\\\\\\/W0NhXhlt9R5iybXvm44FUsmwIFz5tbI2igyVURxyuiPblue3O2n56ImwgTUkxvvdhJg3VeuUQh7WyL4BKfoDWTnpeHVFP8Mqj5WgMJh5lfJ2mOX6bffrwuqkCUebzrwRqG4npn4EcSQDryojfAwEBxWUvz1z8YwJYEQJxZSd7l7rjlMl1nBnzLnAQF+kusqjnNkrVBNYeJ2OfmHT2eAhGUfuHLQ0r0kdBKWFgH\\\\\\\/rfNZor7KL0gP8tC3ATu\\\\\\\/gtcdImXuHcfoyamfi7lPR5NlQxtjvJExsSD14sZhrS2YhBMoJJPbiCV3ILxN3z7s6oObItfRXCw6xY0b0AbH4DaIHS6LxxtaQNh61kFJQIsnZXaBh6H2jQ2SmTfP0sxb8Li6hXOybDJzIzTddXvhxobT\\\\\\\/EGnd2Ax0XyK8GInaxlUTH+wZ3zfX6vpH3gIYr8+VXfu1yH0RLgmZNSvouQieuGLscwZI1Is7cLWTx\\\\\\\/OH3ZCrXi4DPqZhmubFkeZhGAdcyLUIQuMPy6Npu6L8WgOeJUH\\\\\\\/aTfEWsmZJsWxnBn8rVG+qU9DGd9Bab3kvh8h\\\\\\\/vdrUjdzmi69gvQVhDiWGFhTe6orJthYxO\\\\\\\/lhhLJTsz2ZHMUfyWQ3t8aWY3kYUiQE3vvDVJNY2c01\\\\\\\/E2H0Dsb3JrgmpcHYvdzASx0J\\\\\\\/xe+apSmAU0W2ciLWZ5e6GJkv8Zo0+rzq\\\\\\\/Uk6P2GTtIujCrt7MMD6unOYx2XCcarsoqvkZw08h4zZgpUPtcQ4e+4aUzS8SkDMe\\\\\\\/YjGmqcQaU27cb7SyBOfyIlu1VSN+6eA6Rg6g0\\\\\\\/zlJwRn9CLv1pQYwFfAq6ChbGJ4xTchgabFoTB+Cmq4HqxF3mSCgUhCi8QYWe1VVPFQSTx3KoYitDUnD3fy8NjbmrmjMtk84xsDCv6nThzEDTzErGyg7YRFRNd0g0Q+unFdAd5cbdjlRetZZ+7y8ZMlksJnMMREOnSJm4pDhLcFD5JvOBYLVCbr\\\\\\\/2PMX0bBkuULwPV7GYPRfOzh+T4Ay4ifyKozjRrYUy2eOGwjnLpcG35yA11H+vmiHcVz+0rMm58kf64+AM\\\\\\\/5QjekewvuBHjuFHgtjJspBhAvvSS6rfsvbICmG+t7nAc9l67uWxsiToXo0L5sNvkWYzDoYrBZB48g3P++K0EoOXhdtswbyQ19ZZ5oSqHZpyQxLvpWiLNxX2IyO\\\\\\\/XUIpBsXjcjp0fyHZ0FU\\\\\\\/54zbuz5Z9MltVZuTE7nYMYo1lgaOIYcwR4k0MlOI924+iN5l7lzIaBRHaV+8B+eWJAGNwp\\\\\\\/X6ElbgNIWTuZUger21pG+uEsTyzt8tRLbzlgSfgYW9aFCwuWPlWPnntR1\\\\\\\/QS\\\\\\\/ObcssKyKBv45cAckm8NGOG5QiF+sFTBWvbWh7Ijwn\\\\\\\/9ctHMyQhyZALTamsjXtmbKUQHLC1uw045FoQrmeiHhCSr3pOSwd9c234hiExNViMY8UwsxmCvZ8X2PlngIM1zS6MDv2R4Arn9GqtG9RqEibtg6HqsP7LOlL9QHg4ehIPfr96ePWTuWECNCio7IY5\\\\\\\/dOMXZtHa4DPkO8xPAv4POYvLYwOYYkEuZmh9s0A9uTPkNbdnmkdynfdxLDCPdS6B6XquG5\\\\\\\/UU+1USqeCHbpx8kpD6h\\\\\\\/EWFtNOfYBAUssIpj8J9AdEZZLwcNmLpODWXCvYR8aS0Jau5zYBY0hnkdH5Colwn6o3+e4WmHpTUscRdSEymzPjzsivE86E2b5rDpf7oIXsa80XAD47VCU6HoSRdE4WafQjjM8IdNDOZUBEbnnO4lQFKvHzRxhRnGN+Q7uhMzRz5j9T1alJ5jpOhNksCUmJb9p3EQQq3VMuh+5XVS6Y2jSPKCDj2T7gE\\\\\\\/\\\\\\\/VByLJPBPlXTrg7XzpfQHwm5xoZ5HLcC0UhoHTvOYEkgihrZETlg7nn7ZAHdw+x3+IfDXSCZB46ql4KFdJYKxHjiBOR2MwsW5e8hlvgYWAFChgvI7Kla4lO0hoPKA0CWkwGleaJNoh4arjypMHZCpC85ypBvuuHpB78XJrpMGeatZSYe2Gja8OwRebcQlVOZBU4wdp9HtLBI0mf6lGb+Z3MpNKG6UoSsXTwlpZMy69Dcl68xNXVuduOlJlrk8zsujVCnSWT4eZPOygggvLnYhQt1yOew2fOs8IsNJTrOYMohZKBgkT\\\\\\\/d7hAB06sUiUNBQHz0cBjnlZG5Bjt8qz17StXMdJN9VmTtT5E\\\\\\\/2Eip\\\\\\\/4bD3RTE5TwgCN9MvaODhAr6fH8PXM\\\\\\\/\\\\\\\/KQBKzNjy0lBVb+xylYsZSYAkuXxUXwmEB\\\\\\\/EEX5yV1vroSkilj\\\\\\\/7vNoSiQRw0a5Xme5qYZAQLohjgC4ZdxirSet5RstYlGH4AiSaml9M88ULPohXZm7MK+rJ0z7PLvYtc\\\\\\\/z7IJR59mbHX9vPlWnFA0psa5KZygXgnjILDRJY8llK4HRASaZi9eRH8RiIVko0SOYItaWY8f09\\\\\\\/gH2HWEXsCOAg8LLAEgzzNPdTSDSmt\\\\\\\/Gbbyf7JUx2qn+Tr9MGQaqQhXpHxX9uymzFGW6rBQFRn6yklALTwaf9L8\\\\\\\/bq+LjDlhaNCBqynCBs\\\\\\\/Ow5Ypi6MQ9SF5xv1iR\\\\\\\/VEGKcC4CYc7GJL5P1o2lW0yK57a3FiOd1v1lGDfsaAnAQEMWylH90LeBABMI8eFEHG\\\\\\\/UcuzOhvG3gyBjJ0hCz4vhiexGhv8Ug9DST6CzXLpEMT3zDRoIzvlJ8WEvTb\\\\\\\/PPoTrizn6F0AZ2akpuxQYFtTnmqdsH72VUilWh4J0U8OnFzOLdACR8jNxArL8ecotFfyUNrwYf779S\\\\\\\/VvL\\\\\\\/yhIUsR+FZkkcqBbIB6+QoReyENax\\\\\\\/ge8sNaZmbCbvyGHYCyNjmbt3MmCja6DS8cc8Xhg9jeclfxEMx5kLCOcJdW5lCzJRbpLCdMZzoFpxQ1KVqi6pMunGTEbIkvIPtNxbtINIiMOXfkFID+AEqk6CSvZiI2txvpd+qEZbQtso36I\\\\\\\/j+xBs9L9AyqBffyRZ7S04YC7zWv7C+WdmCDEPo3+5m0cCw0DPxx31d2YtLoxvz5KAVEkDTT6\\\\\\\/QZ161j\\\\\\\/QqCmtQPOBgieQfEkJQy+MLscFvz5k92zeFiE6DLzfgqafjElVE1GtABxYS6uN\\\\\\\/hW+M1Kj5LJoZb7w4grdYBkhXcdjwwjlawrzub7xCCy2DZvYhAVqmaAf9PFDbTAoJCAiu8JcAEdkrybF8CzdHSrYhCNZ4nD23Z5yP2h8zHum0MQH5i2J0QX6rKedmQO3o\\\\\\\/yEWKYA5JcoVhC9g9KYX\\\\\\\/LxgWiAf5bZS+m5MIPjpFbjpo\\\\\\\/KRG6ujdWZyC9Y5\\\\\\\/+\\\\\\\/EwoJRXwt9DtW524S1EZo4v90jYlXhh\\\\\\\/IDhpb3spSUoT\\\\\\\/qB7c\\\\\\\/tK6UvJSFrfGyZ4fAbUfpzrUcrff37rJ3FkWmlqTA99awhcyZdALs3shn1M1P1qhU5xHMt9Fx1QmJGierTlvUin3hfCBQNdGqYxNpE+i7bQ\\\\\\\/Z39P36byjfO6tHhzByoxluy78IDaihd85GJjyLPOz0AHZmm9mvw8Jz\\\\\\\/VstdgpHOuWN4WLEcsQ6wz+daUPhxaplN\\\\\\\/79sk3W+uvAVJt+fZv6urTr6fS5fMm+phGrhNiOSFH+To+EhmzJkAW6BQQqGyU+zJjm1jMED3d+XJu3OCnin++zK9\\\\\\\/drJPIUBJXMGv+PAVyVxPWCf+9pPdqcxRyOXIWn8ignKVNfdpCLPC+myYcPfkANtbpqeFCkEt6KctnAHxrrHzDTZ0ynsUxtNXOCTwacnx+12zebwuuy5\\\\\\\/KcszS4ujc8QTlC4WS2fVdype0RTuUrqCiYFzvSnCO34KMTDKxQjfiPKZ17QpYEhqcCoekOnysfBmES2HFygO\\\\\\\/L+7GozdEfdM1Gv+ZIKiipGq6Ml\\\\\\\/ZYWqXCd9M0fW50zVXE\\\\\\\/HHVbRM2yhQds32RPReB2LXA0USpfyCSBbDGx5Ja4JSzzX\\\\\\\/VwHKX8EJQbVzBpN0PAhyKqeZMbvOLgy+CIKH4cKvIeMKGwbCNiE2U7I3nl1NPMD6ICaDo6HVBHx+PIf8Pg9+UrRYYt1HAnxGMILNYhTgL+uZWLOAp+lQsfxHtgnaJud4IK9B3UVvPxfFjud8B\\\\\\\/8Eob9ZySj2bOU\\\\\\\/Dnx395Xhc+fHkHjf2dx8UO+Ww89HFOxQBXWiz3eSJLWDoCPNegVa+tWWp1\\\\\\\/uNhfEvz5JHHtolUWsDfII4zOtsR2ZHm9+ACzhBpO413pPqG2xAjr5A3blP\\\\\\\/VSfNLxd+JVUiuNYJekh0B\\\\\\\/PhoZs4yzYjBn6an2pTH\\\\\\\/+C584K\\\\\\\/OSKDgVbw\\\\\\\/pDtCeTyWEymG4ZORLi6a11aVXl3Rcswi0FeUOdb21vRLNOcYAWNoRH4qAZJJskmUtMXkF6fQ7cdsw5esuqmw55LilA\\\\\\\/MzxCX2Fz9ojti+Yvc5dihqrlL00U9ZENN055V3A+gQlG8MXJUvKtb5U44cgHIfb+fPTY9uZckqI8FAHqiwEVM1DPw+o\\\\\\\/5Zlzt8oNKgO22wwqv\\\\\\\/geTts3TJg7i0C+C8xndZVUL4S6smRYe7rUEGW6DghKhQ0VI+I0YTYD5cOfH6TU8EhnOgj7rzoTBdA2NR4ByDIlxGQEj2a1GacqjW4qvaX91xcuToDsdXZTpJmPJkNaLSHqJ10COqm6tj+0JRVCL1L77ScgPcrIj9XYar13azUFMzF3SPzut26askfRBktP0Qn5NxiLDLczLW+4ra5bBcL3m11QwEYEL70sUFBRnMDTyexKaKn1D0Ira+98LByUnG6ggrq9c+ADBOM\\\\\\\/IJbeXBGqh27FfIeObpIsjM+A87SOOAOjyp5+76B0NdlzjbPn+ea\\\\\\\/AQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022maxbDp2xuvqqTpKk0AWcmQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c07148a325379cb40a273245acff37fb44071ba66dfb644ccbf6071f268a5fdf\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022YuHmJzbROBdrYVz8EzyyJ2Ouv5hjK8Msbv2l0emonxMjwTdeHYm9qvnXHFDqwH2qQgsbGIKhX\\\\\\\/r1Blux\\\\\\\/5f7qEknu7Qzo0tBD2qDrhexTYlXCgwxFQDuVbOFy4r0HQ3I5gpfEo32cxQCFkcjvG5CHZXQ2F7cIjuR3tYZQbNLEDqtDmoSfWLT9dSbnClhi\\\\\\\/hnTyAB\\\\\\\/BsLFqRaiW77zhPNUwmM8dksraq6cTj96s+Ll7Dbi3QAMrMlXilOROqNN4qHolCpiNRQbskVpE5t3wLTaoWtaDvuHasLhO8rgJnM6HgnmUXJp2Gwjc8+romd+UuIhO2qIQAusfl7qpY\\\\\\\/fRLQiqMAtJsooixhhw8LDimMsPHp5qkTAqUu3K210GA0dROxICSV+Gir\\\\\\\/kGBpa3h1Z+RWclnuPdelUZ1MfIU6BldDAWVr+82DELhSMPN0b3iu9e\\\\\\\/yO1hxjWdSXfbVwDeCtFDKz8SWtnb64yv+J\\\\\\\/SmV6daHwEHr\\\\\\\/ScnRWX+O52RYnerYJFmo4MZHw5UzkrS0B90nIhNXs9XnkticUhBtLx7uGAZ\\\\\\\/Otz8NWErDiobmcH5+vzum1IU70GMvm5ZDx0Y9dhHquLPA\\\\\\\/WZHj4+H5b\\\\\\\/cicT1IRg45lLSx3qgtEJ8l4zLGc4pH6Ex+er1FR60kzXYjKrc2o8G740XPsRl590p6iXmDmszTDRraPktnh5Xg6V47Lo4L55ZiB\\\\\\\/DmZvOe9dhdaePqmETxmQPqCOueqmaM33ULZnpK\\\\\\\/UcCh3dHpegSTTnqYhBW\\\\\\\/nKQ6rTDiOeKlMXyhLm7S50YD1\\\\\\\/kHOmSiHZrZoB5hTNF7HUFphKTH7Z7jxHgmvrM\\\\\\\/WOaunG\\\\\\\/kyh\\\\\\\/3w8o3uIG6wBDDwUuXP2UktzpYPr7+alvpgH5N7z7WkB70TtXEVuXOa\\\\\\\/OpSx+ylGZAAhdLpKWwUB2NttiW5vLhC8y0pk0waA1HpaALX7AxyU2UTmz89bMuLfgu\\\\\\\/nSu8cBMJM\\\\\\\/X6voZ3v4bJPxfrhlSiqrICU4e5HVPlFCfGk309z\\\\\\\/ModOgHmNTAecnldjFlPAwLIoOTImQcjLU\\\\\\\/vZ\\\\\\\/YFUPOQlFbRkUN1ZDDyVXi6HKjPX7bSqMmRt+dcRUAy8\\\\\\\/m+HSb\\\\\\\/ncgP59RsKxkRUkiogcZynSsg\\\\\\\/vu3VnkvYa4SrDdI8xxu1jCw\\\\\\\/iONYDmwyYU03jVs94m9KPyDVS0FsVfV3R1IJwNbFjRg5hEHfsWLvXekTb1hMJ+w5lQv9zn0LQ7ap8he7MoK4RLXy38h1gy3jY6S0GCcbNlHzJ9xzp9J+K1YH5st9W1nniVHUohTt92xlm9VJCU6GQU7NjnCoMAEhVrpNpKMFTbBgpmRijI5SPMilMIJKWEiij3\\\\\\\/nQoMvJXG680AWlgGYxjAtfq5EQBOh0CYW2O9OkQ6QM\\\\\\\/Qd03yOyRcQdyY8d3LxycPesNQBtbu2WBHPyEc1w\\\\\\\/SwTmgMUVI5DaTRIwQuwIIt+xRKr6L8Y4dzjC24W90Jh5ClNpR4Unf0yz9cr6afWMyH3PGTCQ8hqxiIvK3ypQh4jLvtose4MFQEENuWAvhl4QHcvSrXO4nfyJrjhJZAMtvctX+tu0NLi6IffQYyqkBVOSqzW4d810etFeCe0W\\\\\\\/ngwghv1JVEeIKcxuB6JOOnpfkL8e7Lwb6rLG7yaw1PKnjsKRZKfj6VMDhm0dzWFfP27AryjQcRJNx52tHLZSV1cxDQxaP9+3DRUqstxqCzhAAbcvjz+uIX20l5HxQlC+2P4h9t8KoNm2UsKIkBBEL8mtRLGCyALxYgzFm\\\\\\\/I2x7s5Fr\\\\\\\/vJcCvfG7+cVjeTdiUNKBfi5iZjeG69YIhyXJ5GOuvKnkGfQykWqtCsuoFOV7vlkjOUKZKZIEk1RH6TQ2B+JZbkOP+EK8sG6lpvE5n2KB0C8bgBBye3qaIIcFH8TyOKjn0qSNlHPGNZXSidVty+n2KVA0TOLgIbHy6H5jZL6GK4xeMW2RNMDi5Mqo+p7QXd6urB7s1T77bkiFiuPB5JXdg1GL\\\\\\\/26rjkUfbLFyRAw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022s2y71nAh5xZ+kLrPfyV3Jg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224e7088e94c18e3590618d876bb8e5bea4333a6aa6db224500cae460facaa9367\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-484242 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1429" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Ql8E6VTQdZzeczylVOPUqVUhTGsqNk8qQJgWFLcqE_k" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1429" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-oHStO91bV7vNFEoqAmHlIl8IM48fUPDFqgRaeuALcN4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/5/1417" title="Dynamic Notch Signaling Specifies Each Cell Fate in Drosophila Spermathecal Lineage" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/5/1439" title="The Interaction of Genetic Background and Mutational Effects in Regulation of Mouse Craniofacial Shape" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484242" id="node484242" data-pisa="ggg;7/5/1429" data-pisa-master="ggg;g3.117.040824" data-apath="/ggg/7/5/1429.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Reliable CRISPR/Cas9 Genome Engineering in <em>Caenorhabditis elegans</em> Using a Single Efficient sgRNA and an Easily Recognizable Phenotype</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3674-3151" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sonia El Mouridi</span>, <span class="highwire-citation-author" data-delta="1">Claire Lecroisey</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-4313-2527" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Philippe Tardy</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-1508-8845" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marine Mercier</span>, <span class="highwire-citation-author" data-delta="4">Alice Leclercq-Blondel</span>, <span class="highwire-citation-author" data-delta="5">Nora Zariohi</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-1734-1915" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Thomas Boulin</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1429-1437;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.040824 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484242" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sonia El Mouridi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Univ Lyon, Université Claude Bernard Lyon 1, CNRS UMR-5310, INSERM U-1217, Institut NeuroMyoGène, F-69622 Villeurbanne, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sonia%2BEl%2BMouridi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=El%20Mouridi%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASonia%2BEl%2BMouridi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3674-3151" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sonia El Mouridi</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Claire Lecroisey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Univ Lyon, Université Claude Bernard Lyon 1, CNRS UMR-5310, INSERM U-1217, Institut NeuroMyoGène, F-69622 Villeurbanne, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claire%2BLecroisey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lecroisey%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClaire%2BLecroisey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Philippe Tardy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Univ Lyon, Université Claude Bernard Lyon 1, CNRS UMR-5310, INSERM U-1217, Institut NeuroMyoGène, F-69622 Villeurbanne, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philippe%2BTardy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tardy%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APhilippe%2BTardy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4313-2527" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Philippe Tardy</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Marine Mercier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Univ Lyon, Université Claude Bernard Lyon 1, CNRS UMR-5310, INSERM U-1217, Institut NeuroMyoGène, F-69622 Villeurbanne, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marine%2BMercier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mercier%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarine%2BMercier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1508-8845" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marine Mercier</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Alice Leclercq-Blondel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Univ Lyon, Université Claude Bernard Lyon 1, CNRS UMR-5310, INSERM U-1217, Institut NeuroMyoGène, F-69622 Villeurbanne, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alice%2BLeclercq-Blondel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Leclercq-Blondel%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlice%2BLeclercq-Blondel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Nora Zariohi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Univ Lyon, Université Claude Bernard Lyon 1, CNRS UMR-5310, INSERM U-1217, Institut NeuroMyoGène, F-69622 Villeurbanne, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nora%2BZariohi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zariohi%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANora%2BZariohi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Thomas Boulin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Univ Lyon, Université Claude Bernard Lyon 1, CNRS UMR-5310, INSERM U-1217, Institut NeuroMyoGène, F-69622 Villeurbanne, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BBoulin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boulin%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThomas%2BBoulin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1734-1915" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Thomas Boulin</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:thomas.boulin@univ-lyon1.fr" class="" data-icon-position="" data-hide-link-title="0">thomas.boulin@univ-lyon1.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/5/1429" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:484242" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:484242/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1429.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:484242" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:484242/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1429.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:484242" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:484242/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1429.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:484242" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:484242/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/5/1429.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:484242" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:484242/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">CRISPR/Cas9 genome engineering strategies allow the directed modification of the <em>Caenorhabditis elegans</em> genome to introduce point mutations, generate knock-out mutants, and insert coding sequences for epitope or fluorescent tags. Three practical aspects, however, complicate such experiments. First, the efficiency and specificity of single-guide RNAs (sgRNA) cannot be reliably predicted. Second, the detection of animals carrying genome edits can be challenging in the absence of clearly visible or selectable phenotypes. Third, the sgRNA target site must be inactivated after editing to avoid further double-strand break events. We describe here a strategy that addresses these complications by transplanting the protospacer of a highly efficient sgRNA into a gene of interest to render it amenable to genome engineering. This sgRNA targeting the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene">dpy-10</a></em> gene generates genome edits at comparatively high frequency. We demonstrate that the transplanted protospacer is cleaved at the same time as the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene">dpy-10</a></em> gene. Our strategy generates scarless genome edits because it no longer requires the introduction of mutations in endogenous sgRNA target sites. Modified progeny can be easily identified in the F1 generation, which drastically reduces the number of animals to be tested by PCR or phenotypic analysis. Using this strategy, we reliably generated precise deletion mutants, transcriptional reporters, and translational fusions with epitope tags and fluorescent reporter genes. In particular, we report here the first use of the new red fluorescent protein mScarlet in a multicellular organism. wrmScarlet, a <em>C. elegans</em>-optimized version, dramatically surpassed TagRFP-T by showing an eightfold increase in fluorescence in a direct comparison.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/crisprcas9-genome-engineering" class="hw-term hw-article-keyword hw-article-keyword-crispr-cas9-genome-engineering" rel="nofollow">CRISPR/Cas9 genome engineering</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/caenorhabditis-elegans" class="hw-term hw-article-keyword hw-article-keyword-caenorhabditis-elegans" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Caenorhabditis elegans</em></a></li><li class="kwd"><a href="/keyword/mscarlet" class="hw-term hw-article-keyword hw-article-keyword-mscarlet" rel="nofollow">mScarlet</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/dpy-10" class="hw-term hw-article-keyword hw-article-keyword-dpy-10" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">dpy-10</em></a></li><li class="kwd"><a href="/keyword/coconversion" class="hw-term hw-article-keyword hw-article-keyword-coconversion" rel="nofollow">coconversion</a></li></ul><p id="p-3">The pace of technical developments allowing the direct manipulation of genome sequences has seen a marked acceleration in recent years with the emergence of RNA-targeted nucleases derived from bacterial immune systems (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Doudna and Charpentier 2014</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Zetsche <em>et al.</em> 2015</a>). In particular, the binary system relying on the <em>Streptococcus pyogenes</em> Cas9 endonuclease targeted by CRISPR (clustered, regularly interspaced, short, palindromic repeat) RNAs has been successfully used to generate point mutations, deletion, or DNA insertions in an ever-growing number of experimental systems. <em>S. pyogenes</em> CRISPR/Cas9 has been adapted early on in the model nematode <em>Caenorhabditis elegans</em> (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Friedland <em>et al.</em> 2013</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dickinson <em>et al.</em> 2013</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chen <em>et al.</em> 2013</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Frøkjær-Jensen 2013</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Dickinson and Goldstein 2016</a>). Previously, heritable genome engineering could only be achieved in <em>C. elegans</em> by remobilizing a <em>Drosophila</em> <em>Mos1</em> transposon, which could be inserted and excised in the germline (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Robert and Bessereau 2007</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Frøkjær-Jensen <em>et al.</em> 2010</a>).</p><p id="p-4">Despite great promise and early success, day-to-day CRISPR experiments are often not straightforward. Different factors might explain variability and inefficiency of CRISPR/Cas9 genome engineering in <em>C. elegans</em>. One specific reason could be the limited expression of heterologous genes in the germline due to dedicated cosuppression mechanisms (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kelly and Fire 1998</a>). One approach to circumvent this problem has been to inject preassembled ribonucleoprotein (RNP) complexes of SpCas9 and CRISPR RNAs (crRNA – tracrRNA duplexes) directly into the germline (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cho <em>et al.</em> 2013</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Paix <em>et al.</em> 2015</a>).</p><p id="p-5">Another general reason for CRISPR failure is that efficacy and specificity vary greatly between different single-guide RNAs (sgRNA). Systematic analyses in different systems have led to the prediction that protospacers terminating by a single guanosine (GNGG) or ideally a double guanosine motif (GGNGG) are generally more effective (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Doench <em>et al.</em> 2014</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Farboud and Meyer 2015</a>). To estimate the prevalence of such sites, we selected a set of 22 genes coding for two-pore domain potassium channel subunits and collected the sequences of all sgRNA target sites in and close to exons of these genes. On average, these 22 loci contained 138 ± 40 protospacers. We found that 20 ± 5% of these matched the GNGG motif, and only 5 ± 2% matched the GGNGG motif (Supplemental Material, Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040824/-/DC1/FileS1.pdf">File S1</a>). Since, the proximity of an sgRNA to the target site has a positive impact on the likelihood to generate gene edits (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Paix <em>et al.</em> 2014</a>), it is therefore likely that few or no high-efficiency sgRNAs will be situated close to a given target region.</p><p id="p-6">One approach to compensate for low CRISPR/Cas9 activity has been to use selection strategies to increase the number of tested progeny. Antibiotic and phenotypic selection protocols have been adapted in <em>C. elegans</em> (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Ward 2015</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dickinson <em>et al.</em> 2015</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Norris <em>et al.</em> 2015</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Dickinson and Goldstein 2016</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Schwartz and Jorgensen 2016</a>). They have the further advantage of reducing hands-on time and facilitate the detection of successful genome editing events. When phenotypic or antibiotic selection is not applicable, Co-CRISPR strategies can be used to increase the likelihood of identifying individuals with genome edits. These coconversion approaches consist of injecting the sgRNA targeting a locus of interest together with a second sgRNA that targets a “marker gene” (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kim <em>et al.</em> 2014</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Arribere <em>et al.</em> 2014</a>). Progeny that carry a modification in the “marker” locus are then more likely to carry edits in the locus of interest. However, since two distinct sgRNAs do not necessarily cut with the same efficiency or in the same germ cell, effectiveness of traditional Co-CRISPR coconversion is variable and mostly indicates a successful injection and expression of Cas9 and sgRNA.</p><p id="p-7">In addition, all available strategies require the protospacer sequence to be disrupted once the edit is generated to prevent further CRISPR/Cas9 cutting/activity. This almost always requires the introduction of point mutations in the protospacer adjacent motif (PAM) or in multiple bases of the protospacer. The consequences of such mutations in introns and up- or downstream regulatory regions are difficult, if not impossible, to predict. Similarly, silent mutations in exons can have unfavorable effects due to codon usage bias. Therefore, it would be ideal if genome edits, in particular insertions or point mutations, could be generated without modification of the surrounding original genomic sequence.</p><p id="p-8">Finally, since CRISPR/Cas9 guide RNAs are short 19- to 20-bp-long sequences, there are often multiple closely matching sites (<em>i.e.</em>, differing only by a few base pairs) in the genome that could be targeted, albeit at lower frequency. While algorithms have been developed to easily predict such potential off-target sites (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hsu <em>et al.</em> 2013</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Doench <em>et al.</em> 2016</a>), the prevalence of undesired CRISPR events has not been systematically analyzed in <em>C. elegans</em> and would require <em>ad hoc</em> experiments for each sgRNA.</p><p id="p-9">We describe here a two-step strategy for reliable and scarless modification of the <em>C. elegans</em> genome using a single-guide RNA that facilitates the detection of genome engineering events based on an easily recognizable phenotype. Indeed, we reasoned that it should be possible to circumvent many practical hurdles described above if we transplanted the protospacer for a highly efficient sgRNA into a genomic locus of interest to create an “entry strain” that would be more amenable to genome engineering. Specifically, we inserted a protospacer and PAM from the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene">dpy-10</a></em> gene (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Arribere <em>et al.</em> 2014</a>) – further referred to as the “<em>d10</em> site” or “<em>d10</em> sequence” – close to the targeted genomic region. In this “<em>d10</em>-entry strain,” we could then induce double-strand breaks at both the transplanted <em>d10</em> site and the endogenous <em><a href="http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene">dpy-10</a></em> locus using a single sgRNA. We demonstrated that the <em>d10</em> site and the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene">dpy-10</a></em> locus were efficiently cut within the same nucleus. Finally, we found that coconversion events (insertions of fluorescent reporter genes and epitope tags) occurred on average in 8% (<em>i.e.</em>, 1 in 12 animals) of F1 progeny that also carried mutations in the marker gene <em><a href="http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene">dpy-10</a></em>, revealing a high incidence of coconversion events. Since this coconversion step no longer relied on an endogenous protospacer from the targeted locus, we did not need to introduce mutations in PAM or protospacer sequences and could generate perfectly accurate and scarless genome edits. Although our strategy is especially suited to insert sequences into the genome, we could also obtain large, precisely targeted gene deletions.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strains generated in this study</h3><p id="p-10"><a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> Bristol was used as a wild-type starting strain for transgenic lines generated in this study. Worms were raised at 20° on nematode growth medium and fed <em>Escherichia coli <a href="http://www.wormbase.org/db/get?name=OP50;class=Strain">OP50</a></em>. Worms were grown at 25° after injection. Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040824/-/DC1/FileS1.pdf">File S1</a> provides a comprehensive list of the strains constructed for this study.</p></div><div id="sec-3" class="subsection"><h3>Molecular biology</h3><p id="p-11">Single-guide RNA expression vectors (see Supplementary Methods in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040824/-/DC1/FileS1.pdf">File S1</a>) and plasmid repair templates were constructed using standard molecular biology techniques and Gibson assembly (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gibson 2011</a>). They were systematically validated by Sanger sequencing before injection. Tables S3 and S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040824/-/DC1/FileS1.pdf">File S1</a>, respectively, list the oligonucleotides and vectors used in this study. The Cas9-expression vector pDD162 was obtained from Addgene (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Dickinson <em>et al.</em> 2013</a>). Vectors generated for this study are available upon request.</p></div><div id="sec-4" class="subsection"><h3>DNA preparation and microinjection</h3><p id="p-12">The pDD162, pMD8, and pPT53 plasmids were purified using the Qiagen EndoFree Plasmid Mega Kit (Qiagen). All other vectors were prepared using Invitrogen PureLink HQ Mini Plasmid Purification Kit (ThermoFisher Scientific). Single-strand DNA repair templates were synthetized and PAGE-purified by Integrated DNA Technologies (IDT). These single-strand DNA oligonucleotides were aliquoted and stored at −80° upon resuspension in IDTE (10 mM Tris, pH = 7.5, 0.1 mM EDTA, IDT). Except specified otherwise, plasmid vectors and ssDNA were diluted in water and injected at a final concentration of 50 ng/µl; co-injection markers were injected at 5 ng/µl. DNA mixes were injected into a single gonad of 1-d-old adult hermaphrodites raised at 20°. They were then cloned onto individual plates after overnight incubation at 25°.</p></div><div id="sec-5" class="subsection"><h3>PCR screening</h3><p id="p-13">PCR DNA amplification was performed on crude worm extracts. In brief, worms were collected in ice-cold 1× M9 buffer, and 5 μl of packed worms were lysed by freeze thaw lysis in 14 μl of Worm Lysis Buffer [50 mM KCl, 10 mM Tris-HCl (pH = 8.3), 2.5 mM MgCl<sub>2</sub>, 0.45% Nonidet P-40, 0.45% Tween 20, 0.01% (w/v) Gelatin], to which 1 μl of proteinase K was added (1 mg/ml final concentration). After freezing at −80°, lysates were incubated for 1 hr at 65°, and proteinase K was inactivated by further incubation at 95° for 20 min.</p><p id="p-14">High-fidelity DNA polymerases (Q5 High-Fidelity DNA Polymerase, New England Biolabs; Phusion High-Fidelity PCR Kit, Thermo Fisher Scientific) were used for PCR amplification to maximize the chances of recovery of desired modifications. Indeed, when we generated the <em>TagRFP-T</em>::<em>twk-18</em> knock-in strain, we initially screened 77 F1 clones using a low-fidelity DNA Polymerase (Taq’Ozyme, Ozyme) and found no edits. When we immediately rescreened the same worm lysates with a more processive, high-fidelity DNA polymerase (Phusion, ThermoFisher Scientific) we identified five positive clones. PCR primers used for this study are listed in Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040824/-/DC1/FileS1.pdf">File S1</a>.</p></div><div id="sec-6" class="subsection"><h3>Generation of sgRNA expression vector by single-strand DNA isothermal ligation</h3><p id="p-15">All sgRNA expression vectors were built using the novel pPT2 vector (see Supplementary Methods in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040824/-/DC1/FileS1.pdf">File S1</a>). In brief, pPT2 was linearized by PmeI/SexAI double digestion. The protospacer sequence was then inserted by isothermal ligation using a single-strand oligonucleotide containing the protospacer sequence flanked by 20-bp-long homology arms corresponding to the sequences upstream of PmeI and downstream of SexAI. If it was not already present in the sequence, a guanine residue was manually added 5′ to the protospacer sequence to optimize U6 promoter activity. A unique identifier was given to each sgRNA using the following nomenclature: CRpXYn, where “CR” stands for “CRISPR/Cas9 recognition site” and pXYn is the name of the corresponding sgRNA expression plasmid.</p></div><div id="sec-7" class="subsection"><h3>Codon-optimization of mScarlet</h3><p id="p-16">wrmScarlet was generated by gene synthesis (Gblock, IDT) based on the mScarlet sequence (Genbank <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KY021423&amp;atom=%2Fggg%2F7%2F5%2F1429.atom">KY021423</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bindels <em>et al.</em> 2017</a>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040824/-/DC1/FigureS3.pdf">Figure S3</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040824/-/DC1/FileS1.pdf">File S1</a>). Codon-optimization was performed using the “<em>C. elegans</em> codon adapter” service (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Redemann <em>et al.</em> 2011</a>) with the following parameters: “<em>0 introns</em>,” “<em>optimize for weak mRNA structure at ribosome binding site</em>,” and “<em>avoid splice sites in coding region</em>.” The Gblock fragment library was combined by isothermal ligation with left and right homology regions flanking the <em>d10</em> sequence in <em><a href="http://www.wormbase.org/db/get?name=WBGene00006672;class=Gene">twk-18</a></em>(<em>bln213</em>) to generate the repair template pSEM87. The wrmScarlet cDNA sequence is available upon request.</p></div><div id="sec-8" class="subsection"><h3>Microscopy and fluorescence quantification</h3><p id="p-17">Freely moving worms were observed on nematode growth media (NGM) plates using an AZ100 macroscope (Nikon) equipped with a Flash 4.0 CMOS camera (Hamamatsu Photonics).</p><p id="p-18">Confocal imaging was performed using an inverted confocal microscope (Olympus IX83) equipped with a confocal scanner unit spinning-disk scan head (Yogokawa) and an electron multiplying charge-coupled device camera (iXon Ultra 888). Worms were imaged on 2% fresh agar pads mounted in M9 solution containing 50 mM NaN<sub>3</sub>.</p><p id="p-19">Comparison of wrmScarlet and TagRFP-T fluorescence was performed as follows: (1) confocal stacks of the head region were acquired for TagRFP-T and wrmScarlet knock-in strains on the same day, using identical settings, and NaN<sub>3</sub> immobilization; (2) the same number of confocal slices was selected from each stack; (3) stacks were projected by summing fluorescence at each pixel position in the stack; (4) total fluorescence was measured in areas of identical size and position relative to the anterior tip of the worm and pharynx; and (5) total fluorescence was corrected by subtracting equipment noise, <em>i.e.</em>, fluorescence measured in an area of the same size outside of the sample.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-20">All <em>C. elegans</em> strains and plasmids described in this study are available upon request.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Generation of d10-entry strains as a starting point for robust and precise gene modification</h3><p id="p-21">Our strategy starts with the insertion of the <em>d10</em> sequence (<em>i.e</em>., <em><a href="http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene">dpy-10</a></em> protospacer + PAM) into the locus of interest (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). First, we targeted three positions in two genes coding for two-pore domain potassium channel subunits: (1) the ATG start site of <em><a href="http://www.wormbase.org/db/get?name=WBGene00006318;class=Gene">sup-9</a></em>, (2) the ATG of the <em>egl-23b</em> isoform, and (3) the common stop codon of all <em><a href="http://www.wormbase.org/db/get?name=WBGene00001190;class=Gene">egl-23</a></em> isoforms (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). Next, we predicted all possible sgRNA sequences within a 50-base window around these positions, and selected sgRNAs close to the ATG or stop codons. Using multiple sgRNAs increases the chances of finding at least one sgRNA that cuts efficiently enough to insert the <em>d10</em> site at the desired location. We then defined the portion of the gene to be replaced by the <em>d10</em> site, based on the positions of the most upstream and most downstream PAM sequences. Finally, we designed a single-strand oligonucleotide sequence (ssON) containing the <em>d10</em> sequence flanked by up- and downstream homology regions of ∼50 bases (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>). This ssON could serve as a repair template with all selected sgRNAs since it did not contain their protospacer or PAM sequences.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1429/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Generation of d10-entry strains. (A) Insertion of the d10 sequence into sup-9, egl-23b, egl-23 (C-terminus), and twk-18 using a single-strand oligonucleotide repair template compatible with multiple sgRNAs. Genes and their intron/exon structure are displayed in the 5&#x2032;&#x2013;3&#x2032; orientation. The ssON repair templates are represented by black arrows (containing the d10 sequence in green) above the coding strand and translation of the target gene. Correspondence of homology regions between the ssON repair template and genomic locus is indicated in gray. sgRNA binding sites are indicated by blue open arrows. (B) unc-58 coconversion is used to detect the insertion of d10 sequences into a gene of interest. unc-58(e665) mutants are easily identified in the F1 progeny of injected P0 animals based on their straight body posture, lack of mobility, and characteristic rotation around the antero-posterior body axis. RT, repair template. (C) BanI, BssSI, and BsrBI restriction sites are present in the d10 protospacer sequence and are used for RFLP analysis. The Cas9 double-strand break site is indicated by an arrowhead. (D) R12E2.15 contains the only predicted off-target site of the d10 sgRNA. Four base changes (in pink) distinguish both sites. A BsrBI site follows the Cas9 double-strand break site (indicated by an arrowhead), between the &#x2212;3 and &#x2212;4 bases relative to the protospacer adjacent motif (PAM)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2069033146" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Generation of &lt;em&gt;d10&lt;/em&gt;-entry strains. (A) Insertion of the &lt;em&gt;d10&lt;/em&gt; sequence into sup-9, egl-23b, egl-23 (C-terminus), and twk-18 using a single-strand oligonucleotide repair template compatible with multiple sgRNAs. Genes and their intron/exon structure are displayed in the 5&#x2032;&#x2013;3&#x2032; orientation. The ssON repair templates are represented by black arrows (containing the &lt;em&gt;d10&lt;/em&gt; sequence in green) above the coding strand and translation of the target gene. Correspondence of homology regions between the ssON repair template and genomic locus is indicated in gray. sgRNA binding sites are indicated by blue open arrows. (B) &lt;em&gt;unc-58&lt;/em&gt; coconversion is used to detect the insertion of &lt;em&gt;d10&lt;/em&gt; sequences into a gene of interest. &lt;em&gt;unc-58&lt;/em&gt;(&lt;em&gt;e665&lt;/em&gt;) mutants are easily identified in the F1 progeny of injected P0 animals based on their straight body posture, lack of mobility, and characteristic rotation around the antero-posterior body axis. RT, repair template. (C) &lt;em&gt;Ban&lt;/em&gt;I, BssSI, and BsrBI restriction sites are present in the &lt;em&gt;d10&lt;/em&gt; protospacer sequence and are used for RFLP analysis. The Cas9 double-strand break site is indicated by an arrowhead. (D) &lt;em&gt;R12E2.15&lt;/em&gt; contains the only predicted off-target site of the &lt;em&gt;d10&lt;/em&gt; sgRNA. Four base changes (in pink) distinguish both sites. A BsrBI site follows the Cas9 double-strand break site (indicated by an arrowhead), between the &#x2212;3 and &#x2212;4 bases relative to the protospacer adjacent motif (PAM).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1429/F1.medium.gif" width="361" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/5/1429/F1.medium.gif" width="361" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1429/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1429/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484383" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-22" class="first-child">Generation of <em>d10</em>-entry strains. (A) Insertion of the <em>d10</em> sequence into sup-9, egl-23b, egl-23 (C-terminus), and twk-18 using a single-strand oligonucleotide repair template compatible with multiple sgRNAs. Genes and their intron/exon structure are displayed in the 5′–3′ orientation. The ssON repair templates are represented by black arrows (containing the <em>d10</em> sequence in green) above the coding strand and translation of the target gene. Correspondence of homology regions between the ssON repair template and genomic locus is indicated in gray. sgRNA binding sites are indicated by blue open arrows. (B) <em>unc-58</em> coconversion is used to detect the insertion of <em>d10</em> sequences into a gene of interest. <em>unc-58</em>(<em>e665</em>) mutants are easily identified in the F1 progeny of injected P0 animals based on their straight body posture, lack of mobility, and characteristic rotation around the antero-posterior body axis. RT, repair template. (C) <em>Ban</em>I, BssSI, and BsrBI restriction sites are present in the <em>d10</em> protospacer sequence and are used for RFLP analysis. The Cas9 double-strand break site is indicated by an arrowhead. (D) <em>R12E2.15</em> contains the only predicted off-target site of the <em>d10</em> sgRNA. Four base changes (in pink) distinguish both sites. A BsrBI site follows the Cas9 double-strand break site (indicated by an arrowhead), between the −3 and −4 bases relative to the protospacer adjacent motif (PAM).</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">Next, we built the necessary sgRNA expression constructs using a novel vector and assembly strategy. This vector (pPT2) is composed of an RNA Polymerase III U6 promoter from <em><a href="http://www.wormbase.org/db/get?name=WBGene00014431;class=Gene">K09B11.12</a></em> (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Friedland <em>et al.</em> 2013</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Katic <em>et al.</em> 2015</a>) followed by two restriction sites (PmeI and SexAI), followed by the sgRNA portion corresponding to the CRISPR tracrRNA and 3′ UTR of <em><a href="http://www.wormbase.org/db/get?name=WBGene00014431;class=Gene">K09B11.12</a></em>. This vector was linearized by restriction digest with PmeI and SexAI, and the crRNA sequence was incorporated by isothermal ligation (Gibson assembly; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Gibson 2011</a>) using a single single-strand DNA oligonucleotide (see <em>Materials and Methods</em> and Supplementary Methods in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040824/-/DC1/FileS1.pdf">File S1</a>). These sgRNA expression vectors were systematically validated using Sanger sequencing.</p><p id="p-24">Since it is not possible to predict the efficiency of an sgRNA <em>a priori</em>, we reasoned that we could increase the likelihood of finding a <em>d10</em> insertion at the locus of interest by using a moderately efficient Co-CRISPR. We chose a previously described reagent combination that introduces a mutation in the two-pore domain potassium channel <em><a href="http://www.wormbase.org/db/get?name=WBGene00006792;class=Gene">unc-58</a></em> and replicates the L428F amino acid change found in the <em><a href="http://www.wormbase.org/db/get?name=WBGene00006792;class=Gene">unc-58</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143382;class=Variation">e665</a></em>) reference allele (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Arribere <em>et al.</em> 2014</a>). <em><a href="http://www.wormbase.org/db/get?name=WBGene00006792;class=Gene">unc-58</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143382;class=Variation">e665</a></em>) produces a dominant and easily recognizable phenotype. Worms have a straight body posture and are essentially unable to move on solid NGM medium throughout their postembryonic development (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>). However, they are viable and fertile. <em><a href="http://www.wormbase.org/db/get?name=WBGene00006792;class=Gene">unc-58</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143382;class=Variation">e665</a></em>)-like progeny can be detected 2–3 d postinjection and individual F1 worms can be cloned right away to ensure that independent events are selected.</p><p id="p-25">To generate <em>d10-entry</em> strains for <em><a href="http://www.wormbase.org/db/get?name=WBGene00006318;class=Gene">sup-9</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00001190;class=Gene">egl-23</a></em>, we injected wild-type <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> worms with a mix of plasmid DNA and ssON repair templates (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1B</a>). In each case, the mix was composed of (i) a Cas9 expression vector (pDD162), (ii) the sgRNA expression vector targeting <em><a href="http://www.wormbase.org/db/get?name=WBGene00006792;class=Gene">unc-58</a></em> (pPT53), (iii) one gene-specific sgRNA expression vector, (iv) the ssON to introduce the <em><a href="http://www.wormbase.org/db/get?name=WBVar00143382;class=Variation">e665</a></em> mutation AF-JA-76 (<a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Arribere <em>et al.</em> 2014</a>), and (v) the ssON required to introduce the <em>d10</em> site (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1A</a> and Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040824/-/DC1/FileS1.pdf">File S1</a>). After 3–4 d, we cloned Unc-58-marked F1 worms to single plates. We then detected the presence of the <em>d10</em> site in the F2 population by PCR amplification and restriction digest. The <em>d10</em> sequence contains sites for three restriction enzymes (<em>Ban</em>I, <em>BsrBI</em>, and <em>BssI</em>) that can be used for restriction fragment length polymorphism analysis (RFLP) (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1C</a>). In each case, we designed a PCR primer pair that produced a fragment of 500–600 bp, centered on the <em>d10</em> site. In this way, we were able to generate multiple independent <em>d10</em>-entry strains for each of the targeted loci (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040824/-/DC1/FileS1.pdf">File S1</a>). In each case, we selected homozygous clones for the <em>d10</em> insertion that lacked the <em><a href="http://www.wormbase.org/db/get?name=WBGene00006792;class=Gene">unc-58</a></em> gain-of-function mutation, and validated them by Sanger sequencing around the <em>d10</em> sites.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484407/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484407/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Insertion of <em>d10</em> protospacer at four genomic loci</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">Next, we targeted the two-pore domain potassium channel subunits <em><a href="http://www.wormbase.org/db/get?name=WBGene00006669;class=Gene">twk-14</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00006672;class=Gene">twk-18</a></em>. We obtained 13 Unc-58-marked progeny in the <em><a href="http://www.wormbase.org/db/get?name=WBGene00006669;class=Gene">twk-14</a></em> experiment, none of which contained the <em>d10</em> sequence, and did not pursue this experiment further. In the <em><a href="http://www.wormbase.org/db/get?name=WBGene00006672;class=Gene">twk-18</a></em> experiment (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1A</a>d), only one of 41 injected P0 worms gave a single Unc-58 worm (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Since this marked F1 worm did not incorporate the <em>d10</em> site in <em><a href="http://www.wormbase.org/db/get?name=WBGene00006672;class=Gene">twk-18</a></em>, we decided to screen its unmarked siblings. Doing so, we found seven independent insertion events out of 93 tested clones. Similarly, we found three additional <em>d10</em> insertion events in 14 unmarked siblings of the <em><a href="http://www.wormbase.org/db/get?name=WBGene00006318;class=Gene">sup-9</a></em> experiment, and six additional <em>d10</em> insertions in 108 unmarked siblings of the experiment targeting the ATG of the <em>egl-23b</em> splice variant (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-27">In conclusion, screening for Unc-58-marked F1 progeny allowed us to rapidly identify P0 individuals for which the injection was successful and CRISPR/Cas9 activity was present in the germline. Cloning Unc-58 worms at the F1 generation ensured that we selected independent edits and decreased the number of animals to clone and analyze by PCR. In three cases, we also found <em>d10</em> protospacer insertions in nonmarked siblings, although at lower frequencies than in Unc-58-marked F1 progeny. In total, we successfully targeted five different sites in the genome using this protocol (Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040824/-/DC1/FileS1.pdf">File S1</a>).</p></div><div id="sec-12" class="subsection"><h3>Efficient and specific cutting of transplanted d10 sites</h3><p id="p-28">Different laboratories have independently reported that the sgRNA targeting the <em>d10</em> site is among the most efficient ones currently known (I. Katic, M. Boxem, C. Gally, J.-L. Bessereau, personal communication). The reasons for this high efficacy are unclear. For example, the site matches the GNGG motif and not GGNGG (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Farboud and Meyer 2015</a>). A more favorable chromatin organization or the sequence of the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene">dpy-10</a></em> locus itself might explain high CRISPR activity in this gene. Since we transplanted only the protospacer and PAM sequences of the <em>d10</em> site, we decided to estimate the frequency of cuts in transplanted <em>d10</em> sites before attempting to engineer these loci by homologous recombination.</p><p id="p-29">DNA double-strand breaks can be repaired by homologous recombination using the sister chromatid to restore a wild-type sequence or by nonhomologous end joining (NHEJ), which results in small indels close to the cut site. We reasoned that we could therefore estimate the double-strand break frequency by looking for the destruction of the restriction sites present in and around the −3/−4 position relative to the NGG, <em>i.e.</em>, the Cas9 cut site (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1C</a>). Note that only catastrophic events that result in sufficiently modified <em>d10</em> sites that could no longer be targeted by the Cas9/<em>d10</em>-sgRNA duplex would be detected in this way. This experiment therefore underestimates the double-strand break frequency since precise repair events using the sister chromatid would not be detected.</p><p id="p-30">We selected four <em>d10</em>-entry strains on three different chromosomes (<em><a href="http://www.wormbase.org/db/get?name=WBGene00006445;class=Gene">tag-68</a> I</em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00001190;class=Gene">egl-23</a> IV</em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00006672;class=Gene">twk-18</a> X</em>, and <em><a href="http://www.wormbase.org/db/get?name=WBGene00006792;class=Gene">unc-58</a> X</em>). Each strain was injected with a DNA mixture containing (i) a Cas9 expression vector (pDD162), (ii) an sgRNA expression vector targeting <em><a href="http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene">dpy-10</a></em> (pMD8), and (iii) a ssON to introduce the <em><a href="http://www.wormbase.org/db/get?name=WBVar00054207;class=Variation">cn64</a></em> mutation (AF-ZF-827) in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene">dpy-10</a></em> (<a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">Arribere <em>et al.</em> 2014</a>). Next, we singled F1 progeny showing a Dpy-10 phenotype, <em>i.e.</em>, <a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000645;class=Phenotype">Rol</a> (<em><a href="http://www.wormbase.org/db/get?name=WBVar00054207;class=Variation">cn64</a></em>/+), <a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000583;class=Phenotype">Dpy</a> (−/−), or DpyRol (<em><a href="http://www.wormbase.org/db/get?name=WBVar00054207;class=Variation">cn64</a></em>/−) (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Levy <em>et al.</em> 1993</a>; <a id="xref-ref-1-6" class="xref-bibr" href="#ref-1">Arribere <em>et al.</em> 2014</a>). Finally, we tested all clones that segregated the Dpy-10 phenotype in their progeny and observed the loss of the <em>Ban</em>I site in 14–26% of them (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Since <em>Ban</em>I is located 5′ to the cut site (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1C</a>), we tested the remaining <em>Ban</em>I-positive clones (<em>i.e.</em>, lacking mutations in <em>Ban</em>I) with BsrBI and BssSI. This led us to identify additional events, likely affecting the bases closest to the −3/−4 cut site. In total, we found that between 33 and 52% of Dpy-10-marked F1 worms had lost at least one restriction site, which demonstrates that heterologous <em>d10</em> sites can be cut at high frequency and are present in Co-CRISPR-marked F1 progeny.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484352/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484352/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">High CRISPR/Cas9 activity at transplanted <em>d10</em> site</span><div class="sb-div caption-clear"></div></div></div><p id="p-31">Bioinformatic analysis predicts a single, low scoring, off-target site for the <em>d10</em> sgRNA, situated in the uncharacterized gene <em><a href="http://www.wormbase.org/db/get?name=WBGene00020040;class=Gene">R12E2.15</a></em> (<a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1D</a>). We investigated potential off-target cutting of the <em>d10</em> sgRNA by analyzing the <em><a href="http://www.wormbase.org/db/get?name=WBGene00020040;class=Gene">R12E2.15</a></em> locus in 32 independent F1 worms that segregated the Dpy-10 phenotypes. None of these 32 lines showed scars around the potential off-target cut site of the <em>d10</em> sgRNA.</p><p id="p-32">Given the high correlation between worms displaying Dpy-10 phenotypes and double-strand break events in the transplanted <em>d10</em>-site, and given the high selectivity of the <em>d10</em> sgRNA for the endogenous and transplanted sites, we chose to focus only on Dpy-10-marked Co-CRISPR individuals in our coconversion experiments.</p></div><div id="sec-13" class="subsection"><h3>Generation of multiple knock-in lines using a single d10-entry strain</h3><p id="p-33">As a proof of principle for our strategy, we targeted the <em><a href="http://www.wormbase.org/db/get?name=WBGene00006672;class=Gene">twk-18</a></em> locus. <a href="http://www.wormbase.org/db/get?name=WBGene00006672;class=Gene">TWK-18</a> is one of 47 two-pore domain potassium channels in the <em>C. elegans</em> genome. Its expression pattern and localization in body wall muscle cells has been reported previously (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kunkel <em>et al.</em> 2000</a>). We decided to generate two N-terminal fusions (1) with the red fluorescent protein TagRFP-T (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Shaner <em>et al.</em> 2008</a>) and (2) with the blue fluorescent protein TagBFP (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chai <em>et al.</em> 2012</a>). As a repair template, we constructed two vectors with left and right homology regions of 2073 and 1993 bp (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). We injected each repair template separately into the <em><a href="http://www.wormbase.org/db/get?name=WBGene00006672;class=Gene">twk-18</a> d10</em>-entry strain (JIP1143) with (i) a Cas9 expression vector (pDD162), (ii) the sgRNA expression vector targeting <em><a href="http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene">dpy-10</a></em> (pMD8), (iii) the ssON to introduce the <em><a href="http://www.wormbase.org/db/get?name=WBVar00054207;class=Variation">cn64</a></em> mutation in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene">dpy-10</a></em> (AF-ZF-827), and (iv) the fluorescent reporter pCFJ90 as a co-injection marker to identify transgenic animals based on mCherry fluorescence in the pharynx (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). We selected 77 (TagBFP) and 98 (TagRFP-T) Dpy-10-marked F1 progeny. Finally, we used PCR screening to identify five and six clones respectively, which had integrated the TagRFP-T and TagBFP sequences in the <em><a href="http://www.wormbase.org/db/get?name=WBGene00006672;class=Gene">twk-18</a></em> locus, corresponding to a recombination frequency of 6% of Dpy-10-marked F1 progeny (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1429/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Generation of multiple knock-in lines using a single d10-entry strain. (A) A single d10-entry strain is used to engineer N-terminal TagBFP, TagRFP-T, and wrmScarlet fusions in the twk-18 locus. Correspondence of homology regions between the plasmid repair template and twk-18 genomic locus is indicated in gray. RT, repair template. (B) Two to 3 days following injection of a d10-entry strain with a CRISPR/Cas9 mix, F1 progeny with Dpy-10 phenotypes (Rol or Dpy) can be easily recovered, and further screened in the F2 generation to identify the desired genome edits by PCR or phenotype." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2069033146" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Generation of multiple knock-in lines using a single &lt;em&gt;d10&lt;/em&gt;-entry strain. (A) A single &lt;em&gt;d10&lt;/em&gt;-entry strain is used to engineer N-terminal TagBFP, TagRFP-T, and wrmScarlet fusions in the &lt;em&gt;twk-18&lt;/em&gt; locus. Correspondence of homology regions between the plasmid repair template and &lt;em&gt;twk-18&lt;/em&gt; genomic locus is indicated in gray. RT, repair template. (B) Two to 3 days following injection of a &lt;em&gt;d10&lt;/em&gt;-entry strain with a CRISPR/Cas9 mix, F1 progeny with Dpy-10 phenotypes (Rol or Dpy) can be easily recovered, and further screened in the F2 generation to identify the desired genome edits by PCR or phenotype.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1429/F2.medium.gif" width="440" height="240"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/5/1429/F2.medium.gif" width="440" height="240"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1429/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1429/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484318" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-34" class="first-child">Generation of multiple knock-in lines using a single <em>d10</em>-entry strain. (A) A single <em>d10</em>-entry strain is used to engineer N-terminal TagBFP, TagRFP-T, and wrmScarlet fusions in the <em>twk-18</em> locus. Correspondence of homology regions between the plasmid repair template and <em>twk-18</em> genomic locus is indicated in gray. RT, repair template. (B) Two to 3 days following injection of a <em>d10</em>-entry strain with a CRISPR/Cas9 mix, F1 progeny with Dpy-10 phenotypes (Rol or Dpy) can be easily recovered, and further screened in the F2 generation to identify the desired genome edits by PCR or phenotype.</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484303/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484303/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Summary of genome editing experiments</span><div class="sb-div caption-clear"></div></div></div><p id="p-35">When we prepared these knock-in lines for observation by confocal fluorescence microscopy, we noted that TWK-18-TagBFP had a very reproducible subcellular distribution at the exterior surface of body wall muscle cells (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3B</a>). The highly repetitive grid-like pattern was very different from the one reported previously since it appeared to show a strong green fluorescent protein signal in the endoplasmic reticulum (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Kunkel <em>et al.</em> 2000</a>). This intracellular localization was not consistent with the electrophysiological effect of <a href="http://www.wormbase.org/db/get?name=WBGene00006672;class=Gene">TWK-18</a> gain-of-function mutants, in which <a href="http://www.wormbase.org/db/get?name=WBGene00006672;class=Gene">TWK-18</a> most likely exerts its hyperpolarizing role at the plasma membrane. We believe these differences probably resulted from a strong overexpression of <a href="http://www.wormbase.org/db/get?name=WBGene00006672;class=Gene">TWK-18</a> in this study compared to our knock-in strain, highlighting the importance of physiological expression levels when observing the distribution of cell surface-targeted channels and receptors (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gendrel <em>et al.</em> 2009</a>). When comparing the TagRFP-T and TagBFP knock-in strains, we noticed not only a marked difference in brightness but also in the apparent resolution (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). The overall pattern of TagRFP-T was similar to TagBFP but the longer emission wavelength of TagRFP-T (emission maximum, 584 nm) did not afford the same resolution as the much shorter emission wavelength of TagBFP (emission maximum, 457 nm). This is in part explained by the fact that resolution is proportional to the emission wavelength, making TagBFP an interesting alternative to increase imaging resolution without changing imaging hardware.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1429/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of TagBFP, TagRFP-T, and wrmScarlet using reliable editing of the twk-18 locus. (A) wrmScarlet::TWK-18 is visibly brighter than TagRFP-T::TWK-18. Side-by-side comparison of two young adult hermaphrodites. wrmScarlet-associated fluorescence is visible by eye in freely moving worms on NGM plates, while TagRFP-T is not detectable by eye in this context. (B) The two-pore domain potassium channel TWK-18 decorates the plasma membrane of body wall muscle cells. Representative images of head muscle cells labeled with N-terminal fusions of TWK-18 to TagBFP, TagRFP-T, and wrmScarlet. Head is left. Bar, 10 &#xB5;m. (C) Quantification of fluorescence intensity shows an eightfold increase in fluorescence between TagRFP-T and wrmScarlet. Mean &#xB1; SD. Student&#x2019;s t-test, *P &lt; 0.0001." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2069033146" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of TagBFP, TagRFP-T, and wrmScarlet using reliable editing of the &lt;em&gt;twk-18&lt;/em&gt; locus. (A) wrmScarlet::TWK-18 is visibly brighter than TagRFP-T::TWK-18. Side-by-side comparison of two young adult hermaphrodites. wrmScarlet-associated fluorescence is visible by eye in freely moving worms on NGM plates, while TagRFP-T is not detectable by eye in this context. (B) The two-pore domain potassium channel TWK-18 decorates the plasma membrane of body wall muscle cells. Representative images of head muscle cells labeled with N-terminal fusions of TWK-18 to TagBFP, TagRFP-T, and wrmScarlet. Head is left. Bar, 10 &#xB5;m. (C) Quantification of fluorescence intensity shows an eightfold increase in fluorescence between TagRFP-T and wrmScarlet. Mean &#xB1; SD. Student&#x2019;s &lt;em&gt;t&lt;/em&gt;-test, *&lt;em&gt;P&lt;/em&gt; &lt; 0.0001.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1429/F3.medium.gif" width="440" height="338"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/5/1429/F3.medium.gif" width="440" height="338"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1429/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1429/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484263" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-36" class="first-child">Comparison of TagBFP, TagRFP-T, and wrmScarlet using reliable editing of the <em>twk-18</em> locus. (A) wrmScarlet::TWK-18 is visibly brighter than TagRFP-T::TWK-18. Side-by-side comparison of two young adult hermaphrodites. wrmScarlet-associated fluorescence is visible by eye in freely moving worms on NGM plates, while TagRFP-T is not detectable by eye in this context. (B) The two-pore domain potassium channel TWK-18 decorates the plasma membrane of body wall muscle cells. Representative images of head muscle cells labeled with N-terminal fusions of TWK-18 to TagBFP, TagRFP-T, and wrmScarlet. Head is left. Bar, 10 µm. (C) Quantification of fluorescence intensity shows an eightfold increase in fluorescence between TagRFP-T and wrmScarlet. Mean ± SD. Student’s <em>t</em>-test, *<em>P</em> &lt; 0.0001.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">Next, we targeted three additional loci on different chromosomes (<em><a href="http://www.wormbase.org/db/get?name=WBGene00006318;class=Gene">sup-9</a> II</em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00006691;class=Gene">twk-40</a> III</em>, and <em><a href="http://www.wormbase.org/db/get?name=WBGene00001190;class=Gene">egl-23</a> IV</em>) and generated seven different edits with a variety of insert types (TagRFP-T, TagRFP-T::ZF1, SL2::TagRFP-T, and TagBFP) (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a> and Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040824/-/DC1/FileS1.pdf">File S1</a>). We found that we could reliably edit these different loci. Indeed, edit frequencies in Dpy-10-marked F1 worms ranged from 3 to 19% (average 8%). Taken together, these experiments demonstrate that it is possible to take advantage of the high CRISPR activity of the <em>d10</em> sgRNA to robustly engineer the genome of <em>C. elegans</em>. This strategy significantly reduces hands-on work by focusing only on the animals that most likely carry genome edits. It generates scarless edits since it does not require the introduction of mutations in endogenous protospacer sequences.</p></div><div id="sec-14" class="subsection"><h3>wrmScarlet, a brighter red fluorescent protein</h3><p id="p-38">The development of improved blue (TagBFP; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Chai <em>et al.</em> 2012</a>), cyan (mTurquoise2; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Goedhart <em>et al.</em> 2012</a>), green (mNeonGreen; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Shaner <em>et al.</em> 2013</a>), and red fluorescent proteins (TagRFP-T; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Shaner <em>et al.</em> 2008</a>) has greatly increased our capacity to detect proteins expressed at physiological levels. However, the properties of these new fluorophores are generally characterized in bacteria or cell culture systems, and are not always retained in <em>C. elegans</em> cells or in specific subcellular compartments (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Heppert <em>et al.</em> 2016</a>).</p><p id="p-39">In an effort to improve the detection of fusion proteins <em>in vivo</em>, we have investigated the behavior of the recently described red fluorescent protein mScarlet (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Bindels <em>et al.</em> 2017</a>). mScarlet has currently the highest reported brightness, quantum yield, and fluorescence lifetime of any red fluorescent protein. We synthetized a <em>C. elegans</em> codon-optimized cDNA (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Redemann <em>et al.</em> 2011</a>) of mScarlet, which we named wrmScarlet (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040824/-/DC1/FigureS3.pdf">Figure S3</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040824/-/DC1/FileS1.pdf">File S1</a>). We combined this cDNA with homology arms flanking the <em>d10</em> site in <em><a href="http://www.wormbase.org/db/get?name=WBGene00006672;class=Gene">twk-18</a></em> to generate a <em>wrmScarlet</em>::<em>twk-18</em> repair plasmid (pSEM87, <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040824/-/DC1/FigureS3.pdf">Figure S3</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040824/-/DC1/FileS1.pdf">File S1</a>). Following the same strategy as before, we injected 29 P0 worms (<em><a href="http://www.wormbase.org/db/get?name=WBGene00006672;class=Gene">twk-18</a> d10</em>-entry strain, JIP1440) with an injection mix containing (i) pDD162 (Cas9), (ii) the ssON to introduce the <em><a href="http://www.wormbase.org/db/get?name=WBVar00054207;class=Variation">cn64</a></em> mutation in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene">dpy-10</a></em> (AF-ZF-827), (iii) the sgRNA expression vector targeting <em><a href="http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene">dpy-10</a></em> (pMD8), and (iv) <em>wrmScarlet</em>::<em>twk-18</em> repair plasmid (pSEM87). Out of 29 injected P0 worms, 11 produced Dpy-10 F1 progeny. In total, we analyzed 123 Dpy-10-marked F1 worms that segregated Dpy-10 progeny and found six clones incorporating the wrmScarlet sequence (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>).</p><p id="p-40">While undetectable by eye, specific fluorescence can be observed on NGM plates in <em>TagRFP-T::twk-18</em> worms with a macroscope (Nikon AZ100) coupled to a CMOS camera (Flash 4, Hamamatsu Photonics). Using the same macroscope, acquisition parameters and filter sets, wrmScarlet-TWK-18 was significantly brighter than the TagRFP-T fusion, so much so that it became visible to the naked eye (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>). We next compared the subcellular distribution and brightness of these two translational fusions using spinning-disk confocal imaging. Both protein fusions had grossly identical distribution patterns (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>). However, the wrmScarlet fusion was ∼8 times brighter than the TagRFT-T fusion in this assay (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3C</a>). In fact, the distribution of the wrmScarlet::TWK-18 fusion protein appeared more uniform than TagRFP-T::TWK-18, possibly due to the increased fluorescent signal, which compensated for the reduced resolution when compared to TagBFP (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3B</a>). These properties make wrmScarlet a very convincing replacement for TagRFP-T and should greatly facilitate the detection of protein fusions expressed at low, physiological expression levels.</p></div><div id="sec-15" class="subsection"><h3>Generation of an epitope-tagged knock-in using a long single-strand oligonucleotide</h3><p id="p-41">For short edits, single-strand DNA oligonucleotides can be very efficient repair templates (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Zhao <em>et al.</em> 2014</a>; <a id="xref-ref-1-7" class="xref-bibr" href="#ref-1">Arribere <em>et al.</em> 2014</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Katic <em>et al.</em> 2015</a>). We tested if a large ssON could be used as a repair template to integrate two repeats of the Myc tag sequence into the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001190;class=Gene">egl-23</a></em> locus (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040824/-/DC1/FigureS2.pdf">Figure S2</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040824/-/DC1/FileS1.pdf">File S1</a>). We synthetized a 182-nucleotide-long ssDNA fragment containing part of the last exon of <em><a href="http://www.wormbase.org/db/get?name=WBGene00001190;class=Gene">egl-23</a></em> to restore the full-length C-terminal sequence, followed by 75 nt encoding two Myc tag sequences (2xMyc), and the original stop codon and 3′ UTR region of the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001190;class=Gene">egl-23</a></em> gene (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040824/-/DC1/FileS1.pdf">File S1</a>). In theory, each strand could serve as a template for recombination, but we selected the strand complementary to the sgRNA following the observations of <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Katic <em>et al.</em> (2015)</a>. We injected 30 P0 worms (JIP1150) with a DNA mixture containing (i) a Cas9 expression vector (pDD162), (ii) the expression vector for the <em>d10</em> sgRNA (pMD8), (iii) the ssON that introduces the <em><a href="http://www.wormbase.org/db/get?name=WBVar00054207;class=Variation">cn64</a></em> mutation in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene">dpy-10</a></em> (AF-ZF-827), (iv) ssON containing the 2xMyc tag sequence (oSEM158), and (v) pCFJ90 as a co-injection marker to identify transgenic animals based on mCherry fluorescence in the pharynx. We selected 67 Dpy-10-marked F1 progeny and among these, nine carried the 2xMyc tag. This 14% edit frequency was comparable, yet slightly higher than the average efficiency of longer inserts using double-strand DNA repair templates (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>).</p><p id="p-42">The high edit efficiency observed in this experiment shows that our strategy is very effective to tag proteins of interest for immunohistochemical or protein biochemistry experiments. Generating this epitope-tagged strain required &lt;2 wk, with no additional cloning steps and could be repeated easily to integrate a variety of epitope tags, opening the way for different downstream applications.</p></div><div id="sec-16" class="subsection"><h3>Generation of a large, targeted deletion using a d10-entry strain</h3><p id="p-43">One starting point for many CRISPR experiments is the desire to engineer loss-of-function mutations in a gene of interest. Previously, researchers relied on random mutagenesis with chemical mutagens or ionizing radiation followed by fastidious screening and extensive backcrossing to wild-type strains to eliminate background mutations (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Boulin and Hobert 2012</a>). CRISPR/Cas9 engineering offers the possibility to generate gene deletions with minimal background mutations. A simple approach relies on the repair of double-strand breaks by NHEJ pathways (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Friedland <em>et al.</em> 2013</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Chen <em>et al.</em> 2013</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Katic and Großhans 2013</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Waaijers <em>et al.</em> 2013</a>; <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Dickinson and Goldstein 2016</a>). One, two, or more sgRNAs are injected together and phenotypic or PCR screening strategies are used to retrieve deletion mutants by PCR amplification. However, the exact breakpoints of these deletions are not controllable in this scheme and there is always the potential for undesired edits due to off-target effects for each sgRNA.</p><p id="p-44"><em>d10</em>-entry strains can also serve as a starting point to generate precisely defined gene deletions. As a proof of principle, we targeted the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001190;class=Gene">egl-23</a></em> locus. <em><a href="http://www.wormbase.org/db/get?name=WBGene00001190;class=Gene">egl-23</a></em> is a large locus comprising 12 exons, and removal of the entire <em>egl-23a</em> splice isoform required an 8-kb deletion (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040824/-/DC1/FigureS2.pdf">Figure S2</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040824/-/DC1/FileS1.pdf">File S1</a>). Our goal was to replace the complete <em><a href="http://www.wormbase.org/db/get?name=WBGene00001190;class=Gene">egl-23</a></em>a locus by a transgene expressing the red fluorescent protein mCherry in the pharynx which could be used as a genetic balancer and knock-out mutant of <em><a href="http://www.wormbase.org/db/get?name=WBGene00001190;class=Gene">egl-23</a></em>. We therefore constructed a repair template composed of two homology regions of 2 kb, which flanked the transcriptional reporter unit (<em>Pmyo-2</em>::<em>mCherry</em>::<em>unc-54 3′UTR</em>). This construct was then injected into the appropriate <em><a href="http://www.wormbase.org/db/get?name=WBGene00001190;class=Gene">egl-23</a> d10</em>-entry strain (JIP1150) along with (i) a Cas9 expression vector (pDD162), (ii) the expression vector for the <em>d10</em> sgRNA (pMD8), and (iii) the ssON that introduces the <em><a href="http://www.wormbase.org/db/get?name=WBVar00054207;class=Variation">cn64</a></em> mutation in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene">dpy-10</a></em> (AF-ZF-827). Out of 40 Dpy-10 progeny, we identified one knock-out line (JIP1253). We validated that the genome edit was accurate by Sanger sequencing. We further verified that the possible off-target site of the <em>d10</em> sgRNA was unaffected (<a id="xref-fig-1-12" class="xref-fig" href="#F1">Figure 1D</a>). While this particular trial was less efficient than smaller insertions, it confirmed that <em>d10</em>-entry strains can be used to generate large deletion and gene replacements, in addition to being ideally suited for the insertion of kilobase-sized inserts or epitope tags.</p></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><p id="p-45">The major conceptual innovation of our strategy is to render genes highly susceptible to CRISPR/Cas9 engineering by transplanting the <em>d10</em> sequence. As we have described above, highly effective sgRNAs matching the GGNGG motif are underrepresented in the genome, and are therefore rarely found in close proximity to the region of interest. While editing frequency is highly variable between sgRNAs at different loci or even within the same locus, we found that editing using our <em>d10</em> strategy was robust at different loci, with edit frequencies averaging 8%, <em>i.e.</em>, 1 in 12 F1 progeny. In addition, editing was also robust at a single locus. This is particularly valuable and time-saving when multiple edits need to be generated in the same locus, as is usually the case when a gene is being characterized in depth. For example, we took advantage of this high editing frequency to rapidly generate multiple chromatic variants in the <em><a href="http://www.wormbase.org/db/get?name=WBGene00006672;class=Gene">twk-18</a></em> locus. This allowed us for the first time to precisely compare the resolution and fluorescence intensity of TagBFP, TagRFP-T, and the recently published mScarlet. Based on the highly stereotypical distribution pattern of <a href="http://www.wormbase.org/db/get?name=WBGene00006672;class=Gene">TWK-18</a> at the muscle surface, we could show that (1) TagBFP fusions provided the best apparent resolution, (2) a codon-optimized wrmScarlet was ∼8 times brighter than TagRFP-T, and (3) the increased signal of wrmScarlet partly compensated for the lesser resolution of red <em>vs.</em> blue fluorescent proteins.</p><p id="p-46">One unique feature of our strategy is that edits can be designed so that all original genomic sequences are perfectly preserved. Indeed, by using the transplanted <em>d10</em> sgRNA instead of sgRNAs from the targeted locus, no mutations need to be introduced to avoid continued CRISPR/Cas9 activity once the edit is performed. This facilitates and accelerates experimental design, because only one repair template is designed instead of specific repair constructs for each endogenous sgRNA that would be considered for an edit.</p><p id="p-47">Another benefit of our strategy is that we could retrieve multiple independent lines from the same injected animal by cloning animals in the F1 generation, which is not possible in strategies that rely on the screening of mixed populations of F2 progeny, <em>e.g.</em>, with antibiotic selection strategies. Therefore, since we could focus on relatively few F1 clones, multiple methods could be used to detect the desired genome edit such as direct observation, phenotypic screening, or PCR detection. Limiting the number of animals that need to be analyzed could also mitigate PCR detection issues (see <em>PCR screening</em>).</p><p id="p-48">From a practical perspective, our strategy provides multiple layers of quality control. Based on the easily recognizable Dpy-10 Co-CRISPR phenotype, we could directly monitor the success of injections and assess the general efficiency of the experiment over time and between experimenters. We could determine if an experiment would likely be successful within 3 d postinjection by monitoring the number of marked F1 progeny. Finally, all steps of our protocol are only limited by the generation time of <em>C. elegans</em>, making it particularly time-efficient.</p><p id="p-49">Obtaining the <em>d10</em>-entry strain is the major bottleneck of our strategy. This step, like every CRISPR/Cas9 experiment, relies (1) on the ability to find an endogenous sgRNA that cuts efficiently and (2) on the rate of homology directed repair at the cut site, which could be influenced by the local genomic context or specific sequence features of the homology arms. During this study, we were unable to recover <em>d10</em> insertions in some of our target loci despite testing multiple sgRNAs. For some genes, we eventually succeeded by using double-stranded DNA repair templates with long homology arms instead of single-strand oligonucleotides.</p><p id="p-50">Another practical concern appears when targeting loci that are closely linked to <em><a href="http://www.wormbase.org/db/get?name=WBGene00006792;class=Gene">unc-58</a></em> (to build <em>d10</em>-entry strains) <em>or <a href="http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene">dpy-10</a></em> (to engineer <em>d10</em> loci), which are situated at the center of chromosome II and X, respectively. Since we select F1 progeny based on mutation of <em><a href="http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene">dpy-10</a></em> or <em><a href="http://www.wormbase.org/db/get?name=WBGene00006792;class=Gene">unc-58</a></em>, it is likely that genome edits will be linked to these marker mutations. In that case, one should consider the wild-type siblings in the progeny of an injected P0 individual that produced a significant fraction of marked progeny.</p><p id="p-51">So far, we have tested this strategy only with the <em>d10</em> sequence, but in principle, any highly effective sgRNA that targets a gene producing a dominant Co-CRISPR phenotype could be used. Conceptually, our strategy could also be extended to other genetic model organisms. In particular, a Co-CRISPR strategy based on the <em>white</em> locus has been recently published, and could be a starting point to adapt this strategy to engineer the <em>Drosophil</em>a genome (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Ge <em>et al.</em> 2016</a>).</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-52">We thank J.-L. Bessereau, I. Katic, and members of the Bessereau team for helpful discussion, M. Jospin for comments on the manuscript, and M. D’Alessandro for constructing pMD8. pCFJ90 was a gift of C. Frøkjær-Jensen. We thank Wormbase, which is supported by National Institutes of Health grant U41 HG002223. This work was supported by a research grant from Fondation Fyssen (T.B.) and an European Research Council Starting Grant (Project <em>Kelegans</em>) (T.B.). This work is dedicated to the memory of Christian Boulin.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-53">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040824/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040824/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-54"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-02-03"><span class="received-label">Received </span>February 3, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-03-02"><span class="accepted-label">Accepted </span>March 2, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 El Mouridi <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.1" data-doi="10.1534/genetics.114.169730"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arribere</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">B. X. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Artiles</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartman</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Efficient marker-free recovery of custom genetic modifications with CRISPR/Cas9 in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">837</span>–<span class="cit-lpage">846</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.169730%26rft_id%253Dinfo%253Apmid%252F25161212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzMvODM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bindels</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haarbosch</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Weeren</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Postma</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiese</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">mScarlet: a bright monomeric red fluorescent protein for cellular imaging.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D14%26rft.spage%253D53%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boulin</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hobert</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">From genes to function: the <em>C. elegans</em> genetic toolbox.</span> <abbr class="cit-jnl-abbrev">WIREs Dev Biol</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">114</span>–<span class="cit-lpage">137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWIREs%2BDev%2BBiol%26rft.volume%253D1%26rft.spage%253D114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.4" data-doi="10.1038/nprot.2012.128"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chai</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Live imaging of cellular dynamics during Caenorhabditis elegans postembryonic development.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">2090</span>–<span class="cit-lpage">2102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DChai%26rft.auinit1%253DY.%26rft.volume%253D7%26rft.issue%253D12%26rft.spage%253D2090%26rft.epage%253D2102%26rft.atitle%253DLive%2Bimaging%2Bof%2Bcellular%2Bdynamics%2Bduring%2BCaenorhabditis%2Belegans%2Bpostembryonic%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2012.128%26rft_id%253Dinfo%253Apmid%252F23138350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2012.128&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23138350&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.5" data-doi="10.1093/nar/gkt805"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fenk</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Bono</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Efficient genome editing in <em>Caenorhabditis elegans</em> by CRISPR-targeted homologous recombination.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">e193</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt805%26rft_id%253Dinfo%253Apmid%252F24013562%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MS8yMC9lMTkzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.6" data-doi="10.1534/genetics.113.155853"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Heritable gene knockout in <em>Caenorhabditis elegans</em> by direct injection of Cas9-sgRNA ribonucleoproteins.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">1177</span>–<span class="cit-lpage">1180</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.155853%26rft_id%253Dinfo%253Apmid%252F23979576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NS8zLzExNzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.7" data-doi="10.1534/genetics.115.182162"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dickinson</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">CRISPR-based methods for <em>Caenorhabditis elegans</em> genome engineering.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">885</span>–<span class="cit-lpage">901</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDickinson%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D202%26rft.issue%253D3%26rft.spage%253D885%26rft.epage%253D901%26rft.atitle%253DCRISPR-Based%2BMethods%2Bfor%2BCaenorhabditis%2Belegans%2BGenome%2BEngineering%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.182162%26rft_id%253Dinfo%253Apmid%252F26953268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAyLzMvODg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.8" data-doi="10.1038/nmeth.2641"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dickinson</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reiner</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Engineering the <em>Caenorhabditis elegans</em> genome using Cas9-triggered homologous recombination.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1028</span>–<span class="cit-lpage">1034</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D1028%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2641%26rft_id%253Dinfo%253Apmid%252F23995389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2641&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23995389&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000325073800029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.9" data-doi="10.1534/genetics.115.178335"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dickinson</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pani</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heppert</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Streamlined genome engineering with a self-excising drug selection cassette.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">1035</span>–<span class="cit-lpage">1049</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.178335%26rft_id%253Dinfo%253Apmid%252F26044593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMC80LzEwMzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.10" data-doi="10.1038/nbt.3026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doench</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartenian</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tothova</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hegde</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Rational design of highly active sgRNAs for CRISPR-Cas9-mediated gene inactivation.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1262</span>–<span class="cit-lpage">1267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D32%26rft.spage%253D1262%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3026%26rft_id%253Dinfo%253Apmid%252F25184501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25184501&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.11" data-doi="10.1038/nbt.3437"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doench</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fusi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sullender</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hegde</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaimberg</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR-Cas9.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">184</span>–<span class="cit-lpage">191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D34%26rft.spage%253D184%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3437%26rft_id%253Dinfo%253Apmid%252F26780180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3437&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26780180&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.12" data-doi="10.1126/science.1258096"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doudna</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charpentier</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title"> The new frontier of genome engineering with CRISPR-Cas9.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">346</span>: <span class="cit-fpage">1258096</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDoudna%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D346%26rft.issue%253D6213%26rft.spage%253D1258096%26rft.epage%253D1258096%26rft.atitle%253DThe%2Bnew%2Bfrontier%2Bof%2Bgenome%2Bengineering%2Bwith%2BCRISPR-Cas9%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1258096%26rft_id%253Dinfo%253Apmid%252F25430774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNDYvNjIxMy8xMjU4MDk2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.13" data-doi="10.1534/genetics.115.175166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Farboud</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Dramatic enhancement of genome editing by CRISPR/Cas9 through improved guide RNA design.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">959</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.175166%26rft_id%253Dinfo%253Apmid%252F25695951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk5LzQvOTU5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.14" data-doi="10.1038/nmeth.2532"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friedland</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tzur</span>  <span class="cit-name-given-names">Y. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esvelt</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colaiácovo</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Heritable genome editing in <em>C. elegans</em> via a CRISPR-Cas9 system.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">741</span>–<span class="cit-lpage">743</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D741%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2532%26rft_id%253Dinfo%253Apmid%252F23817069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2532&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23817069&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000322453600021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.15" data-doi="10.1534/genetics.113.156521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frøkjær-Jensen</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Exciting prospects for precise engineering of <em>Caenorhabditis elegans</em> genomes with CRISPR/Cas9.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">635</span>–<span class="cit-lpage">642</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFrokjaer-Jensen%26rft.auinit1%253DC.%26rft.volume%253D195%26rft.issue%253D3%26rft.spage%253D635%26rft.epage%253D642%26rft.atitle%253DExciting%2BProspects%2Bfor%2BPrecise%2BEngineering%2Bof%2BCaenorhabditis%2Belegans%2BGenomes%2Bwith%2BCRISPR%252FCas9%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.156521%26rft_id%253Dinfo%253Apmid%252F24190921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk1LzMvNjM1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.16" data-doi="10.1038/nmeth.1454"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frøkjær-Jensen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">T. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moerman</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Targeted gene deletions in <em>C. elegans</em> using transposon excision.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">453</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DFrokjaer-Jensen%26rft.auinit1%253DC.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253D451%26rft.epage%253D453%26rft.atitle%253DTargeted%2Bgene%2Bdeletions%2Bin%2BC.%2Belegans%2Busing%2Btransposon%2Bexcision.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1454%26rft_id%253Dinfo%253Apmid%252F20418868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1454&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20418868&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278150900017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ge</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tipping</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brodsky</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zamore</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Rapid screening for CRISPR-directed editing of the <em>Drosophila</em> genome using <em>white</em> coconversion.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">3197</span>–<span class="cit-lpage">3206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D6%26rft.spage%253D3197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.18" data-doi="10.1038/nature08430"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gendrel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rapti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richmond</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bessereau</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A secreted complement-control-related protein ensures acetylcholine receptor clustering.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">461</span>: <span class="cit-fpage">992</span>–<span class="cit-lpage">996</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGendrel%26rft.auinit1%253DM.%26rft.volume%253D461%26rft.issue%253D7266%26rft.spage%253D992%26rft.epage%253D996%26rft.atitle%253DA%2Bsecreted%2Bcomplement-control-related%2Bprotein%2Bensures%2Bacetylcholine%2Breceptor%2Bclustering.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08430%26rft_id%253Dinfo%253Apmid%252F19794415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08430&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19794415&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270817700050&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.19" data-doi="10.1016/B978-0-12-385120-8.00015-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">D. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Enzymatic assembly of overlapping DNA fragments.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">498</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">361</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DGibson%26rft.auinit1%253DD.%2BG.%26rft.volume%253D498%26rft.spage%253D349%26rft.epage%253D361%26rft.atitle%253DEnzymatic%2Bassembly%2Bof%2Boverlapping%2BDNA%2Bfragments.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-385120-8.00015-2%26rft_id%253Dinfo%253Apmid%252F21601685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-385120-8.00015-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21601685&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292007500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.20" data-doi="10.1038/ncomms1738"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goedhart</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stetten von</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noirclerc-Savoye</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lelimousin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joosen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Structure-guided evolution of cyan fluorescent proteins towards a quantum yield of 93%.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">751</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BCommun%26rft.aulast%253DGoedhart%26rft.auinit1%253DJ.%26rft.volume%253D3%26rft.spage%253D751%26rft.epage%253D751%26rft.atitle%253DStructure-guided%2Bevolution%2Bof%2Bcyan%2Bfluorescent%2Bproteins%2Btowards%2Ba%2Bquantum%2Byield%2Bof%2B93%2525.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms1738%26rft_id%253Dinfo%253Apmid%252F22434194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms1738&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22434194&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.21" data-doi="10.1091/mbc.E16-01-0063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heppert</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dickinson</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pani</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steward</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Comparative assessment of fluorescent proteins for in vivo imaging in an animal model system.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">3385</span>–<span class="cit-lpage">3394</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E16-01-0063%26rft_id%253Dinfo%253Apmid%252F27385332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyNy8yMi8zMzg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.22" data-doi="10.1038/nbt.2647"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hsu</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinstein</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ran</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Konermann</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">DNA targeting specificity of RNA-guided Cas9 nucleases.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">827</span>–<span class="cit-lpage">832</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D31%26rft.spage%253D827%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2647%26rft_id%253Dinfo%253Apmid%252F23873081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2647&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23873081&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.23" data-doi="10.1534/genetics.113.155754"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katic</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Großhans</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Targeted heritable mutation and gene conversion by Cas9-CRISPR in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">1173</span>–<span class="cit-lpage">1176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.155754%26rft_id%253Dinfo%253Apmid%252F23979578%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NS8zLzExNzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katic</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ciosk</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">CRISPR/Cas9 genome editing in <em>Caenorhabditis elegans</em>: evaluation of templates for homology-mediated repair and knock-ins by homology-independent DNA repair.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1649</span>–<span class="cit-lpage">1656</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D5%26rft.spage%253D1649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fire</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Chromatin silencing and the maintenance of a functional germline in Caenorhabditis elegans.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">2451</span>–<span class="cit-lpage">2456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DKelly%26rft.auinit1%253DW.%26rft.volume%253D125%26rft.issue%253D13%26rft.spage%253D2451%26rft.epage%253D2456%26rft.atitle%253DChromatin%2Bsilencing%2Band%2Bthe%2Bmaintenance%2Bof%2Ba%2Bfunctional%2Bgermline%2Bin%2BCaenorhabditis%2Belegans%26rft_id%253Dinfo%253Apmid%252F9609828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTI1LzEzLzI0NTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.26" data-doi="10.1534/genetics.114.166389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishidate</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghanta</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seth</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conte</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A co-CRISPR strategy for efficient genome editing in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">1069</span>–<span class="cit-lpage">1080</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.166389%26rft_id%253Dinfo%253Apmid%252F24879462%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Ny80LzEwNjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kunkel</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnstone</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salkoff</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Mutants of a temperature-sensitive two-P domain potassium channel.</span> <abbr class="cit-jnl-abbrev">J. Neurosci.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">7517</span>–<span class="cit-lpage">7524</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeuroscience%26rft.stitle%253DJ.%2BNeurosci.%26rft.aulast%253DKunkel%26rft.auinit1%253DM.%2BT.%26rft.volume%253D20%26rft.issue%253D20%26rft.spage%253D7517%26rft.epage%253D7524%26rft.atitle%253DMutants%2Bof%2Ba%2BTemperature-Sensitive%2BTwo-P%2BDomain%2BPotassium%2BChannel%26rft_id%253Dinfo%253Apmid%252F11027209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoiam5ldXJvIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8yMC83NTE3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.28" data-doi="10.1091/mbc.4.8.803"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kramer</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Molecular and genetic analyses of the Caenorhabditis elegans dpy-2 and dpy-10 collagen genes: a variety of molecular alterations affect organismal morphology.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">803</span>–<span class="cit-lpage">817</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Bof%2Bthe%2BCell%26rft.stitle%253DMol.%2BBiol.%2BCell%26rft.aulast%253DLevy%26rft.auinit1%253DA.%2BD.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253D803%26rft.epage%253D817%26rft.atitle%253DMolecular%2Band%2Bgenetic%2Banalyses%2Bof%2Bthe%2BCaenorhabditis%2Belegans%2Bdpy-2%2Band%2Bdpy-10%2Bcollagen%2Bgenes%253A%2Ba%2Bvariety%2Bof%2Bmolecular%2Balterations%2Baffect%2Borganismal%2Bmorphology.%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.4.8.803%26rft_id%253Dinfo%253Apmid%252F8241567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjc6IjQvOC84MDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.29" data-doi="10.1534/genetics.115.180679"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Norris</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colaiácovo</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calarco</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Efficient genome editing in <em>Caenorhabditis elegans</em> with a toolkit of dual-marker selection cassettes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">458</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.180679%26rft_id%253Dinfo%253Apmid%252F26232410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAxLzIvNDQ5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.30" data-doi="10.1534/genetics.114.170423"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paix</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">C.-Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calidas</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Scalable and versatile genome editing using linear DNAs with microhomology to Cas9 sites in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">1347</span>–<span class="cit-lpage">1356</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.170423%26rft_id%253Dinfo%253Apmid%252F25249454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OC80LzEzNDciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.31" data-doi="10.1534/genetics.115.179382"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paix</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Folkmann</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rasoloson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seydoux</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">High efficiency, homology-directed genome editing in <em>Caenorhabditis elegans</em> using CRISPR-Cas9 ribonucleoprotein complexes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.179382%26rft_id%253Dinfo%253Apmid%252F26187122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMjAxLzEvNDciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.32" data-doi="10.1038/nmeth.1565"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Redemann</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schloissnig</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ernst</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pozniakowsky</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayloo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Codon adaptation-based control of protein expression in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DRedemann%26rft.auinit1%253DS.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D250%26rft.epage%253D252%26rft.atitle%253DCodon%2Badaptation-based%2Bcontrol%2Bof%2Bprotein%2Bexpression%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1565%26rft_id%253Dinfo%253Apmid%252F21278743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1565&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21278743&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287734800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.33" data-doi="10.1038/sj.emboj.7601463"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robert</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bessereau</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Targeted engineering of the <em>Caenorhabditis elegans</em> genome following <em>Mos1</em>-triggered chromosomal breaks.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">170</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DRobert%26rft.auinit1%253DV.%26rft.volume%253D26%26rft.issue%253D1%26rft.spage%253D170%26rft.epage%253D183%26rft.atitle%253DTargeted%2Bengineering%2Bof%2Bthe%2BCaenorhabditis%2Belegans%2Bgenome%2Bfollowing%2BMos1-triggered%2Bchromosomal%2Bbreaks%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7601463%26rft_id%253Dinfo%253Apmid%252F17159906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo4OiIyNi8xLzE3MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.34" data-doi="10.1534/genetics.115.184275"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jorgensen</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">SapTrap, a toolkit for high-throughput CRISPR/Cas9 gene modification in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">1277</span>–<span class="cit-lpage">1288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.184275%26rft_id%253Dinfo%253Apmid%252F26837755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMi80LzEyNzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.35" data-doi="10.1038/nmeth.1209"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shaner</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">M. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKeown</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinbach</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hazelwood</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Improving the photostability of bright monomeric orange and red fluorescent proteins.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">545</span>–<span class="cit-lpage">551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DShaner%26rft.auinit1%253DN.%2BC.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253D545%26rft.epage%253D551%26rft.atitle%253DImproving%2Bthe%2Bphotostability%2Bof%2Bbright%2Bmonomeric%2Borange%2Band%2Bred%2Bfluorescent%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1209%26rft_id%253Dinfo%253Apmid%252F18454154%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1209&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18454154&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256308200019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.36" data-doi="10.1038/nmeth.2413"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shaner</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lambert</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chammas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ni</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cranfill</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A bright monomeric green fluorescent protein derived from <em>Branchiostoma lanceolatum</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">407</span>–<span class="cit-lpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D407%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2413%26rft_id%253Dinfo%253Apmid%252F23524392%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2413&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23524392&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318220000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.37" data-doi="10.1534/genetics.113.156299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waaijers</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Portegijs</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerver</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemmens</span>  <span class="cit-name-given-names">B. B. L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tijsterman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">CRISPR/Cas9-targeted mutagenesis in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">1187</span>–<span class="cit-lpage">1191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.156299%26rft_id%253Dinfo%253Apmid%252F23979586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NS8zLzExODciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.38" data-doi="10.1534/genetics.114.172361"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Rapid and precise engineering of the <em>Caenorhabditis elegans</em> genome with lethal mutation co-conversion and inactivation of NHEJ repair.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">363</span>–<span class="cit-lpage">377</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.172361%26rft_id%253Dinfo%253Apmid%252F25491644%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk5LzIvMzYzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.39" data-doi="10.1016/j.cell.2015.09.038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zetsche</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gootenberg</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abudayyeh</span>  <span class="cit-name-given-names">O. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slaymaker</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Makarova</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Cpf1 is a single RNA-guided endonuclease of a class 2 CRISPR-Cas system.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">759</span>–<span class="cit-lpage">771</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D163%26rft.spage%253D759%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.09.038%26rft_id%253Dinfo%253Apmid%252F26422227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.09.038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26422227&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1429.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ke</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yue</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Oligonucleotide-based targeted gene editing in <em>C. elegans</em> via the CRISPR/Cas9 system.</span> <abbr class="cit-jnl-abbrev">Nature Publishing Group</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">247</span>–<span class="cit-lpage">250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BPublishing%2BGroup%26rft.volume%253D24%26rft.spage%253D247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/5/1429.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/5/1417" title="Dynamic Notch Signaling Specifies Each Cell Fate in Drosophila Spermathecal Lineage" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/5/1439" title="The Interaction of Genetic Background and Mutational Effects in Regulation of Mouse Craniofacial Shape" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/5"><span>Volume 7 Issue 5, May 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/5.cover-source.jpg?itok=VWLvGstM" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1429" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-XknORzPDlRkshenFMR12I81Cq8rkUemH_jJ-ZPPj9WA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/5/1429" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/5/1429" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/5/1429" class="active" data-icon-position="" data-hide-link-title="0">Reliable CRISPR/Cas9 Genome Engineering in Caenorhabditis elegans Using a Single Efficient sgRNA and an Easily Recognizable Phenotype</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/484242" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-jyfCmtm3sDKbcE21s_m3LSWeP06vQEoNkbUVyY0BL-Q" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/5/1429.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/5/1429" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-NvrtMl7y9AHpHQVcd3tCFK48zvy1Fiko8WWTU4L6vfA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/5/1429" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/5/1429.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484242" id="node484242--2" data-pisa="ggg;7/5/1429" data-pisa-master="ggg;g3.117.040824" data-apath="/ggg/7/5/1429.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Reliable CRISPR/Cas9 Genome Engineering in <em>Caenorhabditis elegans</em> Using a Single Efficient sgRNA and an Easily Recognizable Phenotype</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3674-3151" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sonia El Mouridi</span>, <span class="highwire-citation-author" data-delta="1">Claire Lecroisey</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-4313-2527" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Philippe Tardy</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-1508-8845" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marine Mercier</span>, <span class="highwire-citation-author" data-delta="4">Alice Leclercq-Blondel</span>, <span class="highwire-citation-author" data-delta="5">Nora Zariohi</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-1734-1915" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Thomas Boulin</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1429-1437;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.040824 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484242--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sonia El Mouridi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Univ Lyon, Université Claude Bernard Lyon 1, CNRS UMR-5310, INSERM U-1217, Institut NeuroMyoGène, F-69622 Villeurbanne, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sonia%2BEl%2BMouridi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=El%20Mouridi%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASonia%2BEl%2BMouridi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3674-3151" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sonia El Mouridi</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Claire Lecroisey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Univ Lyon, Université Claude Bernard Lyon 1, CNRS UMR-5310, INSERM U-1217, Institut NeuroMyoGène, F-69622 Villeurbanne, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claire%2BLecroisey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lecroisey%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClaire%2BLecroisey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Philippe Tardy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Univ Lyon, Université Claude Bernard Lyon 1, CNRS UMR-5310, INSERM U-1217, Institut NeuroMyoGène, F-69622 Villeurbanne, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philippe%2BTardy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tardy%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APhilippe%2BTardy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4313-2527" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Philippe Tardy</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Marine Mercier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Univ Lyon, Université Claude Bernard Lyon 1, CNRS UMR-5310, INSERM U-1217, Institut NeuroMyoGène, F-69622 Villeurbanne, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marine%2BMercier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mercier%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarine%2BMercier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1508-8845" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marine Mercier</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Alice Leclercq-Blondel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Univ Lyon, Université Claude Bernard Lyon 1, CNRS UMR-5310, INSERM U-1217, Institut NeuroMyoGène, F-69622 Villeurbanne, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alice%2BLeclercq-Blondel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Leclercq-Blondel%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlice%2BLeclercq-Blondel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Nora Zariohi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Univ Lyon, Université Claude Bernard Lyon 1, CNRS UMR-5310, INSERM U-1217, Institut NeuroMyoGène, F-69622 Villeurbanne, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nora%2BZariohi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zariohi%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANora%2BZariohi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Thomas Boulin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Univ Lyon, Université Claude Bernard Lyon 1, CNRS UMR-5310, INSERM U-1217, Institut NeuroMyoGène, F-69622 Villeurbanne, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BBoulin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boulin%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThomas%2BBoulin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1734-1915" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Thomas Boulin</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:thomas.boulin@univ-lyon1.fr" class="" data-icon-position="" data-hide-link-title="0">thomas.boulin@univ-lyon1.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/5/1429" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/5/1429&amp;title=Reliable%20CRISPR/Cas9%20Genome%20Engineering%20in%20Caenorhabditis%20elegans%20Using%20a%20Single%20Efficient%20sgRNA%20and%20an%20Easily%20Recognizable%20Phenotype" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/5/1429&amp;title=Reliable%20CRISPR/Cas9%20Genome%20Engineering%20in%20Caenorhabditis%20elegans%20Using%20a%20Single%20Efficient%20sgRNA%20and%20an%20Easily%20Recognizable%20Phenotype" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/5/1429&amp;title=Reliable%20CRISPR/Cas9%20Genome%20Engineering%20in%20Caenorhabditis%20elegans%20Using%20a%20Single%20Efficient%20sgRNA%20and%20an%20Easily%20Recognizable%20Phenotype" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/5/1429&amp;text=Reliable%20CRISPR/Cas9%20Genome%20Engineering%20in%20Caenorhabditis%20elegans%20Using%20a%20Single%20Efficient%20sgRNA%20and%20an%20Easily%20Recognizable%20Phenotype" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/5/1429&amp;title=Reliable%20CRISPR/Cas9%20Genome%20Engineering%20in%20Caenorhabditis%20elegans%20Using%20a%20Single%20Efficient%20sgRNA%20and%20an%20Easily%20Recognizable%20Phenotype" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/5/1429&amp;t=Reliable%20CRISPR/Cas9%20Genome%20Engineering%20in%20Caenorhabditis%20elegans%20Using%20a%20Single%20Efficient%20sgRNA%20and%20an%20Easily%20Recognizable%20Phenotype" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/5/1429&amp;title=Reliable%20CRISPR/Cas9%20Genome%20Engineering%20in%20Caenorhabditis%20elegans%20Using%20a%20Single%20Efficient%20sgRNA%20and%20an%20Easily%20Recognizable%20Phenotype" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/5/1429&amp;title=Reliable%20CRISPR/Cas9%20Genome%20Engineering%20in%20Caenorhabditis%20elegans%20Using%20a%20Single%20Efficient%20sgRNA%20and%20an%20Easily%20Recognizable%20Phenotype" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/484242/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484242" id="node484242--3" data-pisa="ggg;7/5/1429" data-pisa-master="ggg;g3.117.040824" data-apath="/ggg/7/5/1429.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Reliable CRISPR/Cas9 Genome Engineering in <em>Caenorhabditis elegans</em> Using a Single Efficient sgRNA and an Easily Recognizable Phenotype</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3674-3151" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sonia El Mouridi</span>, <span class="highwire-citation-author" data-delta="1">Claire Lecroisey</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-4313-2527" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Philippe Tardy</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-1508-8845" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marine Mercier</span>, <span class="highwire-citation-author" data-delta="4">Alice Leclercq-Blondel</span>, <span class="highwire-citation-author" data-delta="5">Nora Zariohi</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-1734-1915" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Thomas Boulin</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1429-1437;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.040824 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484242--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sonia El Mouridi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Univ Lyon, Université Claude Bernard Lyon 1, CNRS UMR-5310, INSERM U-1217, Institut NeuroMyoGène, F-69622 Villeurbanne, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sonia%2BEl%2BMouridi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=El%20Mouridi%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASonia%2BEl%2BMouridi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3674-3151" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sonia El Mouridi</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Claire Lecroisey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Univ Lyon, Université Claude Bernard Lyon 1, CNRS UMR-5310, INSERM U-1217, Institut NeuroMyoGène, F-69622 Villeurbanne, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claire%2BLecroisey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lecroisey%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClaire%2BLecroisey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Philippe Tardy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Univ Lyon, Université Claude Bernard Lyon 1, CNRS UMR-5310, INSERM U-1217, Institut NeuroMyoGène, F-69622 Villeurbanne, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philippe%2BTardy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tardy%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APhilippe%2BTardy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4313-2527" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Philippe Tardy</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Marine Mercier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Univ Lyon, Université Claude Bernard Lyon 1, CNRS UMR-5310, INSERM U-1217, Institut NeuroMyoGène, F-69622 Villeurbanne, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marine%2BMercier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mercier%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarine%2BMercier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1508-8845" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marine Mercier</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Alice Leclercq-Blondel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Univ Lyon, Université Claude Bernard Lyon 1, CNRS UMR-5310, INSERM U-1217, Institut NeuroMyoGène, F-69622 Villeurbanne, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alice%2BLeclercq-Blondel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Leclercq-Blondel%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlice%2BLeclercq-Blondel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Nora Zariohi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Univ Lyon, Université Claude Bernard Lyon 1, CNRS UMR-5310, INSERM U-1217, Institut NeuroMyoGène, F-69622 Villeurbanne, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nora%2BZariohi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zariohi%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANora%2BZariohi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Thomas Boulin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Univ Lyon, Université Claude Bernard Lyon 1, CNRS UMR-5310, INSERM U-1217, Institut NeuroMyoGène, F-69622 Villeurbanne, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BBoulin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boulin%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThomas%2BBoulin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1734-1915" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Thomas Boulin</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:thomas.boulin@univ-lyon1.fr" class="" data-icon-position="" data-hide-link-title="0">thomas.boulin@univ-lyon1.fr</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1429&amp;rft.epage=1437&amp;rft.atitle=Reliable%20CRISPR/Cas9%20Genome%20Engineering%20in%20Caenorhabditis%20elegans%20Using%20a%20Single%20Efficient%20sgRNA%20and%20an%20Easily%20Recognizable%20Phenotype&amp;rft.volume=7&amp;rft.issue=5&amp;rft.date=2017-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=El+Mouridi%2C+Sonia&amp;rft.au=Lecroisey%2C+Claire&amp;rft.au=Tardy%2C+Philippe&amp;rft.au=Mercier%2C+Marine&amp;rft.au=Leclercq-Blondel%2C+Alice&amp;rft.au=Zariohi%2C+Nora&amp;rft.au=Boulin%2C+Thomas"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/484242/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/484242/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/484242/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/484242/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/484242/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/484242/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/484242/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/484242/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/484242/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/484242/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/484242/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/484242/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/5/1429" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/5/1429.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/5/1429.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/5/1429.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/5/1429.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
