<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/5/1569" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/5/1569.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/5/1569.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/5/1569.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/5/1569/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/5/1569.atom" />
<meta name="HW.pisa" content="ggg;7/5/1569" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="asymptoticMK: A Web-Based Tool for the Asymptotic McDonald–Kreitman Test" />
<meta name="DC.Identifier" content="10.1534/g3.117.039693" />
<meta name="DC.Date" content="2017-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Haller and Messer. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The McDonald–Kreitman (MK) test is a widely used method for quantifying the role of positive selection in molecular evolution. One key shortcoming of this test lies in its sensitivity to the presence of slightly deleterious mutations, which can severely bias its estimates. An asymptotic version of the MK test was recently introduced that addresses this problem by evaluating polymorphism levels for different mutation frequencies separately, and then extrapolating a function fitted to that data. Here, we present asymptoticMK, a web-based implementation of this asymptotic MK test. Our web service provides a simple R-based interface into which the user can upload the required data (polymorphism and divergence data for the genomic test region and a neutrally evolving reference region). The web service then analyzes the data and provides plots of the test results. This service is free to use, open-source, and available at &lt;http://benhaller.com/messerlab/asymptoticMK.html&gt;. We provide results from simulations to illustrate the performance and robustness of the asymptoticMK test under a wide range of model parameters." />
<meta name="DC.Contributor" content="Benjamin C. Haller" />
<meta name="DC.Contributor" content="Philipp W. Messer" />
<meta name="article:published_time" content="2017-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="asymptoticMK: A Web-Based Tool for the Asymptotic McDonald–Kreitman Test" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The McDonald–Kreitman (MK) test is a widely used method for quantifying the role of positive selection in molecular evolution. One key shortcoming of this test lies in its sensitivity to the presence of slightly deleterious mutations, which can severely bias its estimates. An asymptotic version of the MK test was recently introduced that addresses this problem by evaluating polymorphism levels for different mutation frequencies separately, and then extrapolating a function fitted to that data. Here, we present asymptoticMK, a web-based implementation of this asymptotic MK test. Our web service provides a simple R-based interface into which the user can upload the required data (polymorphism and divergence data for the genomic test region and a neutrally evolving reference region). The web service then analyzes the data and provides plots of the test results. This service is free to use, open-source, and available at http://benhaller.com/messerlab/asymptoticMK.html. We provide results from simulations to illustrate the performance and robustness of the asymptoticMK test under a wide range of model parameters.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/05/01" />
<meta name="citation_mjid" content="ggg;7/5/1569" />
<meta name="citation_id" content="7/5/1569" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/5/1569" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/5/1569.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/5/1569.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/5/1569.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.039693" />
<meta name="citation_pmid" content="28341700" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="7" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1569" />
<meta name="citation_lastpage" content="1575" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Benjamin C. Haller" />
<meta name="citation_author_institution" content="Department of Biological Statistics and Computational Biology, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1874-8327" />
<meta name="citation_author" content="Philipp W. Messer" />
<meta name="citation_author_institution" content="Department of Biological Statistics and Computational Biology, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author_email" content="messer@cornell.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8453-9377" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. Andolfatto;citation_title=Adaptive evolution of non-coding DNA in Drosophila.;citation_pages=1149-1152;citation_volume=437;citation_year=2005;citation_issue=7062;citation_pmid=16237443;citation_doi=10.1038/nature04107" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. R. Boyko;citation_author=S. H. Williamson;citation_author=A. R. Indap;citation_author=J. D. Degenhardt;citation_author=R. D. Hernandez;citation_title=Assessing the evolutionary impact of amino acid mutations in the human genome.;citation_pages=e1000083-e1000083;citation_volume=4;citation_year=2008;citation_issue=5;citation_pmid=18516229;citation_doi=10.1371/journal.pgen.1000083" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. D. Bustamante;citation_author=A. Fledel-Alon;citation_author=S. Williamson;citation_author=R. Nielsen;citation_author=M. T. Hubisz;citation_title=Natural selection on protein-coding genes in the human genome.;citation_pages=1153-1157;citation_volume=437;citation_year=2005;citation_issue=7062;citation_pmid=16237444;citation_doi=10.1038/nature04240" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. Charlesworth;citation_author=A. Eyre-Walker;citation_title=The McDonald-Kreitman test and slightly deleterious mutations.;citation_volume=25;citation_year=2008;citation_pmid=18195052;citation_doi=10.1093/molbev/msn005" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. G. Clark;citation_author=M. B. Eisen;citation_author=D. R. Smith;citation_author=C. M. Bergman;citation_author=B. Oliver;citation_title=Evolution of genes and genomes on the Drosophila phylogeny.;citation_pages=203-218;citation_volume=450;citation_year=2007;citation_issue=7167;citation_pmid=17994087;citation_doi=10.1038/nature06341" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. Egea;citation_author=S. Casillas;citation_author=A. Barbadilla;citation_title=Standard and generalized McDonald-Kreitman test: a website to detect selection by comparing different classes of DNA sites.;citation_volume=36;citation_year=2008;citation_pmid=18515345;citation_doi=10.1093/nar/gkn337" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=E. Elyashiv;citation_author=K. Bullaughey;citation_author=S. Sattath;citation_author=Y. Rinott;citation_author=M. Przeworski;citation_title=Shifts in the intensity of purifying selection: an analysis of genome-wide polymorphism data from two closely related yeast species.;citation_volume=20;citation_year=2010;citation_pmid=20817943;citation_doi=10.1101/gr.108993.110" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=A. Eyre-Walker;citation_title=The genomic rate of adaptive evolution.;citation_pages=569-575;citation_volume=21;citation_year=2006;citation_issue=10;citation_pmid=16820244;citation_doi=10.1016/j.tree.2006.06.015" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Eyre-Walker;citation_author=P. D. Keightley;citation_title=Estimating the rate of adaptive molecular evolution in the presence of slightly deleterious mutations and population size change.;citation_volume=26;citation_year=2009;citation_pmid=19535738;citation_doi=10.1093/molbev/msp119" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Eyre-Walker;citation_author=M. Woolfit;citation_author=T. Phelps;citation_title=The distribution of fitness effects of new deleterious amino acid mutations in humans.;citation_volume=173;citation_year=2006;citation_pmid=16547091;citation_doi=10.1534/genetics.106.057570" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=J. C. Fay;citation_title=Weighing the evidence for adaptation at the molecular level.;citation_pages=343-349;citation_volume=27;citation_year=2011;citation_issue=9;citation_pmid=21775012;citation_doi=10.1016/j.tig.2011.06.003" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=T. I. Gossmann;citation_author=B.-H. Song;citation_author=A. J. Windsor;citation_author=T. Mitchell-Olds;citation_author=C. J. Dixon;citation_title=Genome wide analyses reveal little evidence for adaptive evolution in many plant species.;citation_volume=27;citation_year=2010;citation_pmid=20299543;citation_doi=10.1093/molbev/msq079" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=B. C. Haller;citation_author=P. W. Messer;citation_title=SLiM 2: flexible, interactive forward genetic simulations.;citation_volume=34;citation_year=2017;citation_pmid=27702775;citation_doi=10.1093/molbev/msw211" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. D. Keightley;citation_author=A. Eyre-Walker;citation_title=Joint Inference of the Distribution of Fitness Effects of Deleterious Mutations and Population Demography Based on Nucleotide Polymorphism Frequencies;citation_pages=2251-2261;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073430;citation_doi=10.1534/genetics.107.080663" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Librado;citation_author=J. Rozas;citation_title=DnaSP v5: a software for comprehensive analysis of DNA polymorphism data.;citation_volume=25;citation_year=2009;citation_pmid=19346325;citation_doi=10.1093/bioinformatics/btp187" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. F. Mackay;citation_author=S. Richards;citation_author=E. A. Stone;citation_author=A. Barbadilla;citation_author=J. F. Ayroles;citation_title=The Drosophila melanogaster Genetic Reference Panel.;citation_pages=173-178;citation_volume=482;citation_year=2012;citation_issue=7384;citation_pmid=22318601;citation_doi=10.1038/nature10811" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. H. McDonald;citation_author=M. Kreitman;citation_title=Adaptive protein evolution at the Adh locus in Drosophila.;citation_pages=652-654;citation_volume=351;citation_year=1991;citation_issue=6328;citation_pmid=1904993;citation_doi=10.1038/351652a0" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. W. Messer;citation_author=D. A. Petrov;citation_title=Frequent adaptation and the McDonald-Kreitman test.;citation_volume=110;citation_year=2013;citation_pmid=23650353;citation_doi=10.1073/pnas.1220835110" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=G. Sella;citation_author=D. A. Petrov;citation_author=M. Przeworski;citation_author=P. Andolfatto;citation_title=Pervasive natural selection in the Drosophila genome?;citation_pages=e1000495-e1000495;citation_volume=5;citation_year=2009;citation_issue=6;citation_pmid=19503600;citation_doi=10.1371/journal.pgen.1000495" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=N. Stoletzki;citation_author=A. Eyre-Walker;citation_title=Estimation of the neutrality index.;citation_volume=28;citation_year=2011;citation_pmid=20837603;citation_doi=10.1093/molbev/msq249" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. Vos;citation_author=T. A. H. te Beek;citation_author=M. A. van Driel;citation_author=M. A. Huynen;citation_author=A. Eyre-Walker;citation_title=ODoSE: a webserver for genome-wide calculation of adaptive divergence in prokaryotes.;citation_pages=e62447;citation_volume=8;citation_year=2013" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="asymptoticMK: A Web-Based Tool for the Asymptotic McDonald–Kreitman Test" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
<meta name="twitter:description" content="The McDonald–Kreitman (MK) test is a widely used method for quantifying the role of positive selection in molecular evolution. One key shortcoming of this test lies in its sensitivity to the presence of slightly deleterious mutations, which can severely bias its estimates. An asymptotic version of the MK test was recently introduced that addresses this problem by evaluating polymorphism levels for different mutation frequencies separately, and then extrapolating a function fitted to that data. Here, we present asymptoticMK, a web-based implementation of this asymptotic MK test. Our web service provides a simple R-based interface into which the user can upload the required data (polymorphism and divergence data for the genomic test region and a neutrally evolving reference region). The web service then analyzes the data and provides plots of the test results. This service is free to use, open-source, and available at &lt;http://benhaller.com/messerlab/asymptoticMK.html&gt;. We provide results from simulations to illustrate the performance and robustness of the asymptoticMK test under a wide range of model parameters." />
<meta name="og-title" property="og:title" content="asymptoticMK: A Web-Based Tool for the Asymptotic McDonald–Kreitman Test" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/5/1569" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The McDonald–Kreitman (MK) test is a widely used method for quantifying the role of positive selection in molecular evolution. One key shortcoming of this test lies in its sensitivity to the presence of slightly deleterious mutations, which can severely bias its estimates. An asymptotic version of the MK test was recently introduced that addresses this problem by evaluating polymorphism levels for different mutation frequencies separately, and then extrapolating a function fitted to that data. Here, we present asymptoticMK, a web-based implementation of this asymptotic MK test. Our web service provides a simple R-based interface into which the user can upload the required data (polymorphism and divergence data for the genomic test region and a neutrally evolving reference region). The web service then analyzes the data and provides plots of the test results. This service is free to use, open-source, and available at &lt;http://benhaller.com/messerlab/asymptoticMK.html&gt;. We provide results from simulations to illustrate the performance and robustness of the asymptoticMK test under a wide range of model parameters." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
    <title>asymptoticMK: A Web-Based Tool for the Asymptotic McDonald–Kreitman Test | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"KcCmm_UDUkvmuUuVbZVGzQM4IADjDXgxCgxwg47BpqA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"484251","apath":"\/ggg\/7\/5\/1569.atom","pisa":"ggg;7\/5\/1569","ac":{"\/ggg\/7\/5\/1569.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/5\/1569.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/5\/1569"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/5\/1569"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/5\/1569":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wByPuRnio6GATTfSfvhNDDToknArpZ5BppyGRXchoP0lwexfrtQyaZSmX\\\\\\\/aHcK0YkmrhEYOpISbVp\\\\\\\/BOlDz8ROg+JhBLwJLy9TL9VDnJkapNw8NA2mAqYFVF+06xUgFjM1G3lyntbqIzlAspiQcH+KMaGusdY8hoe1Z4tSah6pIV1YSNdCC8yJ+QX19fceSt84jlT3UPUOQfe1oxVkqstWuFWjhZ0oN4GJ4MY4+lioAYHr0h4RZwijCTrF1EmtNBGw0MrN52TNgaC7RyrpMvWd5PvkgjnvzP1RCkpH463YctNQPk7fexsBpfwFUZby\\\\\\\/Hkz3xfr60eydpTFoWhdGAQ5zqA4eqHspPTiRjTHmBwOLhGhQeUFOwaEggDGv\\\\\\\/k66XSfaR59Slh47dLAAbmJLo293hjCoJFtd2rSH\\\\\\\/9buvfVvWjQIYIRnI7RictOaqrIXqYBp6iia1de7yrK\\\\\\\/wccKXo8u1vpuKcfH0t75YxfppkoOvqz7oEWPxvJ7mUWqdGAgZ2PDET5j4Uqw8uPCfasC8LbJfVTmxGRE4mV\\\\\\\/Upk8GPTZgEoJVmeNzc\\\\\\\/GxD+qG7EQ1fiwAIY8NLYcg14xUnKbc\\\\\\\/sJ+IXOqFgiEhLbKF5NX0PsI8d8uBx5JjVB3XkAMwA4kUgeNoVr4fNv22\\\\\\\/48TRkbHzzeH1uQ9AtGOlS10cpSyxEKzzWSseRlz2CGC6T0IzSpZJ3LMex\\\\\\\/fLqyHSnOxPm8IUYsMBVkqTCCWn0Yyf9d6i\\\\\\\/kwajZBAJ+pE2IW2HjBCWJF4m8+xqAkD34uKdoONGdC3bkgb+3sXe9AOA3AQcxlw0dTbaXaQnH7sYE73dh5JaNQWk\\\\\\\/H1\\\\\\\/AXAgJawDKJWDq9GAoKxMRnRDfVzGiQCwZ7PZJFEqTZgo5Q3RJvrVL5RyOTvUXCrCKHQddVNVZgbRZ77cTrfkOdDHbUw1hgk7PXJOBEzF71HE746loZht8gG0BvMi\\\\\\\/hxHAT4PbgeNXuPRsiwYc0MxmVa8zaDupOBlO2aRfTMm9RBTvLyPYutNf2oMob2mpiWLUknLnUbrM+5cwApzJeDT\\\\\\\/mp+HsnZda2GouJiHneeL8Id3g\\\\\\\/cJWJncJHc4lQouTfKFJGmVu7iN9c80S6hx4kw7NhK5j+YsixdygYzgE+q8U0nakLHWJIzckZA8gmWxa+pGVP53rGNpwyn9PUH5TXvNEtxUZyky7y7u\\\\\\\/Qbglg6lYeQXeqEO7RVt5CEMd9G0EpnAIaVaL8KA\\\\\\\/g9b7ZqYvsBvSUxTAeIjQpr7xeN3mURfsgmHtpl7dQKMqpf6\\\u0022,\\\u0022iv\\\u0022:\\\u0022ulX6q+IV027UclO0CUKYyw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225b0dcb29ca3d1167257630ea1166fc858219788b78800b272c1332a481afe560\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022tlgeOvOXFmmkRIhupsz2nq1eefmIhcgC6OmvioNb\\\\\\\/B3aOn\\\\\\\/FVrQJB13f8VUTjZhRNtCYWd4\\\\\\\/1TD2GKATYb3XBDIwUaIzDBbgpXdtKyQ2QYcnlBlF+8XvUFw4xZ0ycD1Lq5IvM9LF9Kr8AKHF5Aa44oJMWPYFZXmRuyUpYijYSBTgLQanGHpVVmJkfdYzCbgR0T0cXZDdV990nszm3Mr1gLHTqsqH41CXhzzl0zAZ3Oeo\\\\\\\/3FPnJWZiu0XDdtzKVgBXsccGJzT8EWdTpDSpvZfx4iIXF8y1HKE9aD6RRL5bLJU0Vrlci\\\\\\\/Y+ZE2UIaSu79Jrl+Bh8RXn3KxQQIGNoLV9Y96Jvc6lPO3bjkh2R25QwkgD1BBXCmlqeBRzbeRUl5i1hCgTnvtoGtRraKSEubzd06sxZ4wjqvbP5KvHttA1iRS+0VTTf4SiOHaSvjwy1VTB4VrxpNvmzXQ4ez9dPpY2CI9+BpK1E3pYnyEO2D4U7RTwXmvZHxe7Y5gU6c50wVfZ7fv96sGR8WY\\\\\\\/n8tIjnmSWldx1fRtXeZJA4faQJhejs8fMgtS5G6YlUggbwXtDMMfbcdYaXRcsP2YmJjKA5EyLV+O+\\\\\\\/8vUpe3IU6nUVNuqFr9xCnpqV0LSvGXuzuBs1QT5cqEJTLsNLjmQbDrk9WpmMFPezNGZU4Nj53i5uKz5KIJOsOFXMlb84gbAK+lGVkZi\\\\\\\/19Qn3aD4knkjamE7LYr40L8OXcDQs1sS0hN3i\\\\\\\/5ITnlcMp0dtODsMZhCLFOx3KK3mTkzkRS\\\\\\\/wpOl++RiOiZHmMQxxBhohARL\\\\\\\/zfFHsmQoayMp8cL\\\\\\\/2jOh9DLPdH3uOR+bs0A1eMHt0+CmItAnRuR6CYxm3+QRNOA18GmFYWshzAxhanOR8m\\\\\\\/+lP1npox1Fm\\\\\\\/20PVQnH+wWuoXDOcaILyZG5d5RMNAzT3NzBCukosVbXIW+ojOqLXfZQA8OZxbWooyLn9GqXIoq6fjJ0MIEra4ebmDncw\\\\\\\/r+8Dyvv0lgO1aBFrRni2M\\\\\\\/Twk9YTFf4fV11AWQR4svAyxHENKfF04JdCIesRUnx5XWypE0YePq\\\\\\\/0ZFpoHxLuc5IU9+bPBeiS3zeN8fG9WiPd11HnDUl9ciZ73jNA+p3tOj+3rCszJG1mVyIOkgoOLy2+N6IeGzxQPHtqKAgOF5rpXSghz9bIGJVJfgjR1fxkKlNhw4HbrNNPDBOwmkQWMTsAv84YrJrupsuv41hqKjfu01enTfx2uvVaNMLD37ltyo1s90HzQjpPjnm1W4XkKfoxH1sSi47jl1xDjS4bQ6AjUrHe6J1qUqDSdaGxdV3xv139KMCa3tFnEs84kYZoZVNmKJTqLeH50aYqa9ibOkOt+oW+ALdnnVDQ9vABX6fBP\\\\\\\/4nz\\\\\\\/92FjCNMyo0LSnegjZlKmTnMmNBm\\\\\\\/\\\\\\\/tupH6WAqooH1Nh2r071mThQp0slqH2LFsMVTe169KuJnzMl8YgqDx3RHRh9HjvP9\\\\\\\/O\\\\\\\/Ad1Yv9M+Re2EZTe7h1o8fa8ES+TdJGkyfTXQ1XsSVlyrnY68RSapH7jC8AFvbFDhXD7zl9eA+c0SZe0SLP82r6TpzsNQPlV8CFEFG4RQu8e7va0BrXHV4k1ZWaUbrTEDeTaW14OCnv7lg\\\\\\\/5fgNvbmDl0FW6WqjerVLQr7YhBrys8xxW3h5iCuONfnBWzq0tLRqLBJUsQ+ace4f18D\\\\\\\/3J6VyP\\\\\\\/IkPiYa\\\\\\\/KPEDv9ru4DLVRw0ZGcYuRXHJOoU3QI8ElCBgA9CYbmd6R7evIpciCq8YrQLKm7bODagKE0LH6s8HLGKm0GGmWOqClVE60YLiJt+qJnhkalv9hdzHZE8vQe5SmD1ph80kLfJW4gU9J9G8QZBUa72VN7qunR0xmxRWXleeukvua\\\\\\\/kyEDD6JpPDF8G7XPfSkESHo3UVi18eIw7jzuwqtvgqJ5PocWIr2\\\\\\\/sa01PWQft7bw8lW0HAS45lU1\\\\\\\/OtBgO7ZR8ePY43pILIGrBMp8LxqlvdU2uxkrq4MQuf\\\\\\\/dLSBIgISmwrpZqFTa6eBMNAwEgniN247TSIe3+bgYSHDfE7c\\\\\\\/1AEp5aO7zLaCVqzlwgyZ1O4pX83kVR\\\\\\\/O5+d1tIV6j1Lc7Uv\\\\\\\/X6GvTVjWT4FMhb9F5zWeG2PoO0ny1Li9FZF40Yk8qG3PJkEpyowaDHjQkyTfB6Xeg5Z7SgicNruIXTKE5zEYStVscURFwSWdENV7v0COQGdGmK1QDZ1rM8zDfD6qWBwZP52ssD667z5Y6HOvNkTgXPqHFVRFLZ5skMfDeC0QKMIuPPqhs0i7bhD7rXvSBSWEnKKqHWPWArkZIGObumZRz3dk3WXmJfpPg18jRJpsTGDRXqo3tCWx6wnE\\\\\\\/5v9E5kpRtjuNdiLZUosTuRG+OI9Jj1UOOvly4HnmBmc3ypjXoxhvEvEcwKkrt3Y8QhvLwbB93DrwZPnkQQQ4\\\\\\\/X+qjyRSq1XN7ivaC4KDmCPfcQbxdfaChxOGY11vC+CqGcDKIpR8dGXs06fuAEXanTlk3DcFgDBUupzUSxobA+C2WnimE2X785v4c8gnj6Cpx73bn9EKKZUW8twF2LwPWDN5U8a4DkomQHZoPqEey1SqY92yTxhK3fll8vhxY\\\\\\\/UwP7SR26\\\\\\\/MKkedpLgHyiGnel+vLHn1S3Q\\\\\\\/At6EiQWPPoSAYu+0Esu2NmKqBhNFAWLRzIk2euDx4gRP9H5sNU0syfyPBNj1EfwCvN\\\\\\\/urbwdcBjOiWLyI2LP8QUXgkeVSmveB4XvvNWrCtjy5ngykqXSv+P2LdM8ZY+U4Y4dO+4zamFMfCslzrAo+DgE6vPaUd9cuRxZ\\\\\\\/fa8ur7v\\\\\\\/d9gmy28iCrZAy5TrnHZ9wZ+J9lSjKpC\\\\\\\/f2LiLfkrz4GGUCaNdBrxS2d6g5iy9GTpDvhkZtc1LNEzcInppxjCs74bWHeDNMcAiSPwNcVYKGCvmzHoJClMl5n2CUJsDwpuhmSLkz8LiSbNcsQG0kYhEkQlPyPsjx\\\\\\\/2AyFsNcht\\\\\\\/A2LbLTQQXsJ0v\\\\\\\/Otc1+TvKlTnvzxywnNnLK1QaO3x6+hEVyhaQSysYXOPs+6Ic2Ay8lnaNwBHpzEY2tsPISoCcRcJg8r5RPZXA+6HH+xWgjVnfDy+IDN7FTXa7g\\\\\\\/mekZX\\\\\\\/qnOmXI+sD+IeTDwy0mB13Ah6rUN0wgMpjSl7kO45x2WQ+dm0d6K94o5+PxWzljbOeLTxhAFYfEaRYu9z+Ge4bQnS7eSRFLZWDr39HMG+ibQSU5Hgu2gZM04PjME95tCTbG1+AydpkEb99fU2SIuoVieGANtCaBpRlDsvLt6tc3Dvz+R7NukqLMfbUw2fyWOIm470HomddWYhkuLpyw0qSJJyBUxCwOzmbdV6JeNerrNnTpQF76TlReVsgttsoebErvVegVvHrRCmMZV4CPuqIa7nA2N8kw37PQ0wkXPrlngCre+lfw0tJIMHMaoyLN5G3vBrnfaxm4P2al\\\\\\\/Ies5EowkHmB300j4nB0xOWbK280fLj3w3si+ErmjMIbw5s6nKf\\\\\\\/dLduTG7GjjRrZTAnO6t49ZCEEPb+35yD3QP19tt2QYi6EE170Qzers6nUUfiAU2aer2cNVp+py4kP8PEPayfuxrpziGCaPhYIDMI6zt1fGlFth\\\\\\\/bZC5Huz8Uc\\\\\\\/I8qFyNw\\\\\\\/HLAQyGsvQBmNmOpIUv8I6q3uB1TrqdPAX9diSZVPcg6N0UayByAb7tV\\\\\\\/kKCfGhWQkKif2sgblZMBgSHtQmnzWGziwfbo1C9KpzdWhdLH1II23mVIHGkQO431wmlsAzQU6vpuhV77MG2b1Pu76WV4\\\\\\\/kluWsQmPnUIFOfW0TZPTmBHdHfUcZBo7BmnvizjYvAP+6nMvnniwsd8YeXEVn2wyIJxs2fnIzYO+W\\\\\\\/bElEpE40r\\\\\\\/OBwVdaKdbDdgah4N8MYRutDSytcHQm0+Dqqx5kwoI7+tZQYnXUJrWGDPwYRPYTzl929IsoPd4wrK1RC36MZCcCptwT3R3Eje0krmnxPWOwu9QSEzBsL2lO+UvdlftQZMCbzRqmnB3H5ZGKXFORPH9e0c39xYREl2llAP9NhHruUEZN05MhHZN6gKAaU6UiD9C7JcrNFyHr7CVLMlmCYiytj0Uw43lV\\\\\\\/4L+NQem+ayAibbOzFJySrcdVUsn14sfWA3JsipMA47awV4ISvZVNtcTbE2L51q48Lk7awUhH6YpPI4Dz6OHmnwQoKoLn1IYfrr2uf7BGRibVbk0ThLN1pGG5c\\\\\\\/Ix4j2JuId7ablZ8NYqyg+h0zvNbAG\\\\\\\/qV7gDLd19+9Q2E69jcmch8Gk4kJJ8\\\\\\\/B9xgi9zz60M5WIFYCxONJ8EpvMnhW6AvRTSOfymbKbVrjEwhPGLZ4jL6ClqQwOLf91Bx1qT5gCa5q9Dua5Cx475EhEb3GDoKvzH9XhGsZMmLuB\\\\\\\/qWG9LnH0toswdH3a8n2PbPkAUEBbKG+sCezvuoPtmY24vrTPQjl+7PtUlGaPF+X1qWxNvXyhfImM+XYcn3dZarIHTkMlBs8Qkwsr3dfvkAtQKxwU4T4kpdmLNNTRMYatHGgE+jeNpXuqhcfIcJsHg8sjhqEWuERYVgXbW9lmaihlS+mRhA4ZhGDP3xz79wr8zKtBCyPcBaCUfvmtZMaElUsOIG7tke1mUrh9emv+XB737SLRKqckvzNA3jgIQExR7UWFf5m8XQWQqj\\\\\\\/0ULWeis5w6KDpyUIUD2oT7RGEgMrX4J1VXhahdSeXvBB\\\\\\\/axyGlnCIoB5zYc5sLsoCCStcTiYFbyaEyeZSi4Pi6OoAJCfkOVKL4WYZ0GKurwNwKGxQ\\\\\\\/D0UWpPPZCbhFFizZou\\\\\\\/n+Qc8QbKWIR+Cdl7GD\\\\\\\/5Xu+6VarVSjaW4PO6gWE7OuSMJKAL7dDx9Af47XWKO7BhofSnKCvCAJwo54rDjJ1c6QakEovn6q96d3bQ\\\\\\\/rjFKUqTSo0lxJqTNyTuSKIfkyTK7GQ62UY9Sg7GXtk48LeCHBxtXk7ATok0NsNv0mfvYzOhtMIqPYwkAJX6SpPocxFu+GNnDwJmTyuWtmbbAPiNIB+dB9n6yPLZXKG8mZ28k8J6TEU23jVMIeMXtjh4YG8n2po\\\\\\\/o6aRvT9V3YxfKTe7tF5IN021\\\\\\\/xoUt\\\\\\\/V155rRAtA+sd0fNtlfHJv64qRwjasRCUMyxTpQU22kOOcQjyDSQ5UtlP47MDEomo5w0xxooYJOBnqmsNOaKbF42d\\\\\\\/cQWHlrBK5Idh4o0\\\\\\\/UWzQLry8SY9vwXb8brhpop06wyrQBnS0jx9\\\\\\\/9yQU7c8Jtovl\\\\\\\/ydOEphVpO2Nk6b6uF0itbr9lNcjCqXoU546ZeEY0LQebOCAl8QOUx6PhcqKczUwWUgtxhfFsIBhTiCNdUal1+pPXObqmVCE5zvuDtDmxbC\\\\\\\/uxDoN3rzW\\\\\\\/wtzHx02PiIQLFxtr\\\\\\\/dQba8FHihraJPwN2hwOpwqzDNaFKF47ERdt636Tb28zsPJ+HrU8RjMsYDS2W4Qfb\\\\\\\/SkTUSxUsSXGKYhm0Scfkz83o9xG1ONhreQs9qEUAQ4g+6P6u7YRQttnqVh5YNHsdOApGtaj49G2K1u1jbDbutOA1fJjgNCpIyeg\\\\\\\/4JawhZLsBoi7nAC2wf8An7lmREb2MtQwQ8CTyKeUEkQ6qxc74Z6JtWq79ER770CQreWfveTHxJBPVhiAbC4L8ViTVNtXQqFQ9gy+uBcM0yOh4qGJllO5I\\\\\\\/C5EbmbSTUHtuTTxGPlwuo55g3A+hFPlPTqf3tr61s6RGrG2k1W7xhIb4DQjt5FYaUCctYJIK4ZPjvtLqGbPyv6O9gYp\\\\\\\/kYei9kHq6kxkcOzQOWHbp0TAuxnBzSY2B\\\\\\\/5GnYyKw7kY2qlxVPqpXnw8lXvaomCKorg9sSH+jqNRrJrqH6+fa71tlk8q3u3DMU60NQQbITZ7grwBMy\\\\\\\/ns2zexQ9+e3onYaxlgP6nkdjyDAxzMYCNcgqS9Z1v9WWSvZJou1JOAqpWd+3Vg56GF3f4AWMnwIzZOdE7bisO\\\\\\\/zOuNH8QBtvsCumCVrGeVSdtJAMaATbpO144iS00nSj0\\\\\\\/PK1GTp9Y234M145s5\\\\\\\/02c2BiY5bxz9keDPyxj7yfpGJMIxnzxc6Pvkdw7sBcvZeOkTSimK2urip3ft1dqP0tvi0CbVw+G9qo1Cfk\\\\\\\/NS9JlO\\\\\\\/4e1BwKDJOgk9vs4IQ7OfG8WtgsGkCWX5xhCYqKHhSw\\\\\\\/DZR1\\\\\\\/cldlOrnWPbHJYe+xQbBWi3ZbEjfBqkT\\\\\\\/S1vDuohU3LYlo2VM4jfl1j4wTKUJWXT9Cmmwsous7lZ5dn5Hmf+0uCZxx99Z2P9lVKi\\\\\\\/ezkVbbawXIhqKKAUEbk104bXlHa2U0md6US9j15PXLtKQQoJbWqhX3iuYLXI6WjMnlRdMRZrEnUlBBnqG8LNgVfBDAfkifu6QrFyXwum9a5IQSPn+Ga9ltcMT+WY58UBNpKvC4U9fcqk6HPFIFhOLy3EVjI9KdTOKnh\\\\\\\/sPcYFHZlrKgz\\\\\\\/JhDAABK9WegWP81IjQ2WRF\\\\\\\/lCD2mJ4K\\\\\\\/7iyh42wLCWo9RzI\\\\\\\/\\\\\\\/MRda+w2Vv0HJRBdGZA73wEhOs3T1CxVmpmOm9Q1MS78cIF7iDRz3MKZpwDtZrr4+cy8tQ59CNSoO2qIzsuCAp5usQOI9WeWmaYhqBz3WjwDmIlxs3c1O\\\\\\\/vIQYtQIj3PDYNRURD7m83fiEnwkHVWZ6QUe8rpkhggu8TzuUBeA9vNaZyd5uHQfgir0cUEugoC2cXaKz7sz9M60nepPqd9z9unp6DMKktdbIpWvFTiXmDNCesC6nma05HEhy1z8h0DXDQSTix\\\\\\\/PQLOGxSBKwGYQpV2XsPW\\\\\\\/I+2dhyKrBa0pvKWEI2\\\\\\\/mN2GQT6TYvkBXz4VV2Dte7TQKQKps+hnourxECUSCeWdVX+ORk2HSUUqIPtarOUm32NYiNqioAS\\\\\\\/WmyVX\\\\\\\/KjVu9snMIMCHmtlaonxthLcBEa86ekfeFSAN0r1Azf7RVGz8jhuYERYG9Z3bXLkVNc8WqZEKE+weLEIQKc6Fyo9jEATnh4S4idon4Eq3o6MTzQZCkupevMZC8iXOilzuvcLhRpfVpWMfAf\\\\\\\/lPlAqmlIS\\\\\\\/tHwkClEAXzIf8QNU4qXQ6qOSOqpO5d3fUp\\\\\\\/FnzxE192nLzDht9mkSCKTa\\\\\\\/uxYRHkVUIh2VYEG3BCtxXIXWRxt5DvG02w1+u4LsEk6nd+kjVqGSR8o3nwRhjcelnUbjrPZVqFyW5hxf3Y4dForXkXDmt0ZcEZ2PpMWFoZa\\\\\\\/F\\\\\\\/aIdrZ3JjIOOxEgSEwLWlUjWJtl5bLqLHesiApyfWu8HBbMVcwqFi8ANs7FQFq63Zt45ZlFVHCw3YyASvtOwtCKVSQDJmoDmN9rSPBj2djrm3NdN8hSvYJLR+OhJTeqxtHFnXJlnCG89u+poKMoVZwGi4tchnB50eyy4cIiiJ4k89iKV0iwIg9lQ0rZHxsD7GyvltvytbPfcp+Ebf7vEC5wcs6INGFC\\\\\\\/U+jfDgpcaUU\\\\\\\/fSae2Xl9dg4OIKd8KLNovurcYXwDiGrzXNXhbacwdCazXNoW1M3UYNHGVw56ngXXRgXi9N5wY06aNiCRQxOU0QXvyK2Tkr703Rjk2AqC412k\\\\\\\/7jafMGvuC2HkElwabB89CaF1HcgZh4VBlFNuAVHenbmBeN9aEDOzahjkhqWkvTJorp889ctMq+EP25W+7EA8Hu1v5xeQ\\\\\\\/rT7p9IzY1UHL1hg4qt84jxHKYWjoNcy0Cwitg5kkqe9cU5BAgpi\\\\\\\/RTiFsUzNa2kGiyJ2F7Pa8Y2H\\\\\\\/viFz5zW5iBjAknxTVowzq7O9zgUBU3dDWrO2Cx2JN29hy7WTXUl4ZAjJyL1j1oUxo+gVTi7KeAEWqjW34QVYPFzVGQrMNrC7zG5u1soVaWCI+hVhds+q5\\\\\\\/4bOGaswSIZ+Wqt5Im0X05RMcDUtCjDCUigVQpfHvXf42BgrDYtSnnHgsUV6oTCNYK8jsPJOP6fyCjjFFaJC8grm2S6Aqjz+Cs2\\\\\\\/aCbbEz9bnMPeVkW7W30ugMe4jrMs9EF611NEzdG2ZseFc7i6mIxxeYaOEz5TKFaAjF4qyLL4LefAH6lJEXnB7Qr6HNgx5puHip7PQDp+JuBBLC2K5Q6gIw6X3ji6A0CG+d3LYk97LEnzO9E6jDGhyuts2T583KPPcmsyphBg7IGryKU9aTOBQtXCbo+4+gHBJe7L1lFmv0svpRlX1B1NEk7rwh0UjK2f90MuqA5wWkCXL3B5Uva34bYG5KqhiOpZgVJf5IE271NkRhvlWV1dvVKD9Mn1WMquiYDqij1ydPqmn3rgc\\\\\\\/vSvLJ8ZTHz6lgUa3+3T43QIPybj405t4QW837tbbQ9ddf4VOx16eZLrm55OYPfLwkjFagSgO4vSb6bJs82AP5fGNLjKDzR5i4MKRX8lKhlzs1eR2PawWtrMATWFfE7UaYnm6maz4csZ+84xxj6VoLx0OeMuENP1Rnw05OIPOa3EgRnw+IVbsLFsR72QeqnEYdEKX1BFS1iZ5LXo0R19JOv5ux0CAa9Uyxm055ZAWY6w2khc0R3CJgIR8aL89bt6YXxtZfrjyo1zPe69oflnj2FmVLCL3YpdgqkCthu74N7JDX2o+36Z4fsdpsOQZEPAxOex6j4v9unZb9AyB8ZQvpsajuFUS4HuJHq+jW9mDovmrmh74KuKozr7kwtK3PcEvZ2eTo6S34gEg24g4CQm57Yh1Ofqt5k00uFa3d\\\\\\\/qY8fwPzjW7lAG7XTolaWeWMW11uCEE9PfIVsxHV3ivX+EIuWSHwtsO7IGzx3COKUjSvyZ0hcpX93ZuW6XehSbfTUqA6IxqU9uy2zAUYqzz+DWXWBsCIlXkhW1tRlIVA89EJcZcWVzSpy5M+7rbQRneEcwGP1Bbbm58qd8Bajj6B3Jum29mTM898v9S4A0c5JSQtfBBQTxDhcUexh+eSIv0i1VBJjfA6CE9m0dPnl1KGBHeWGb4v9ccoyhEZsmK+BwB6fNows7RnHdyhnO7GCrLCQclIjmuMcJuxp9YzBuKBOCuUgi2Jk6VXjMdvgpk\\\\\\\/62+\\\\\\\/c+RhtcdRob7jSg9w1\\\\\\\/quw\\\\\\\/RP5Nzn1Qr9rhAO+9ed8Znbv1t6KKe3HkpMAlksSIr+Xr71Wd29MYh7\\\\\\\/lnGhOajQUAX110ld5JrK0N1xhL2hw5bD+6iLAz9cQlTzlJtmTaNPEpHJMGTkOAWsVbRKbPmVajKe9hsmaI6XKinkQ0UKd26TKihNM5wcixJUa\\\\\\\/r+1O2UDecSLpdMoZwERqxMg2FQ3aCp2z6pKSCKaU3kL6\\\\\\\/1ZfHopyg3VUE98pQ6cwz3VNJpalzOucppAQh9Trg8\\\\\\\/TcDkc+0wlNXVsi7CIrXG1CuJ6s9nvWqYwv41XOJ5RabxdcsQ6o4038PwYBDxeBvGBcqNW9YQEWqteKOZG\\\\\\\/KvTa9XB5asBVktkM4e1sRN7APt+4\\\\\\\/k4fKeVYaDCGaAkGc+rC39KX66TEupLWwybk1Cj47RClpy3pdcVvJhKYPvsvhnXUyeKy5BVCBPNqEKg18YuygD8O+41HKiRqDDaCdetvGuDuP2lCVJa7FVttLi0ZiZGf+j4MfGK3baO7HcHYENwT8tfOQZ23G593hykLJa9qKSR5BFaRJxcH512CBMIjZ8QLqzeeBmB9T1LKaCk6LF+IBauPFvgheliNZeYAG\\\\\\\/MVHWacnO2YZDSw2HQ9BpWpZV2plnnKbjiayzb3CYZDIz5yqXvZ9xUy7eYjRE+1XnPOmlwLX62dg4THw1+g6\\\\\\\/viPfKfncu0gizT8WrxU\\\\\\\/4xu0y7b2z1SF5bKxO6\\\\\\\/P2rjYLpWgn27WHPBPhFITWc3S\\\\\\\/EYPzEruAYSP8eJh675QxSFXQ9Hw6OxDa2nkAb2vYGZNkeSYsC3z6CfelLw4Bz+XUizt4gsW6JiBq5p9tJymviULNdJzB1GVN8OWQ9jJZOC7QIly59xOAQxpj+MCVRvLYluh+qGkSneZOHlZXPNCqm04ubcEgK1iRaeJ0m0lRADL2S2L5iwRjCw\\\\\\\/o7cfrJXE\\\\\\\/HJdjQ9u4+NEHE9Qr1Vz1\\\\\\\/Rwc2eJPxG\\\\\\\/Ee5x16VyMMsBrYbFR8xNraMn+dZoXXPbFZHYjvCAWDvU2zuz+eDzhHvc4HYTLtv\\\\\\\/sHMykWKqbZBguKed2Y1uoMGeOB2HoQNnHIusJ8chZ0EkUaKY2FzJVcUuEshFpG1gUMFhOwZcw+h7jhxICrRwZ4IDSYdg0vp8c5zzbNMsi4tjYMdKfcqXyMA+jyTWIqpNxvDuLl+f+gtohzxSHGTUUzheYQwCYoY1y+WxFEc\\\\\\\/SWA+hQN7nzRZeLkZ6lsU8WM9avzowxNSTgFaCqOhuBAnCn+DZL\\\\\\\/vdtzf4lvU+3O+TRllogYI1pc2SZkJ3osirhX602aUwx7Vvx5sx4PMbF\\\\\\\/bSR7S\\\\\\\/QltlM8cqzVDpT7XDkiJFOcx\\\\\\\/x2SEW\\\\\\\/x27nLxsjjbFCglZd2egWEFfln5qoMuivIEl9eNGnZSk8VhrS3HeJDoYnFfsURzasPmq2WT8\\\\\\\/zfTf1Mvb7RnZdQlXV8wMEBwn4w0c4o4\\\\\\\/ywLQv8hIJzikbVriumlH9eQ5oben72UoHiuCb\\\\\\\/TYbnIgQMHOcZX\\\\\\\/eiF51UjjdyLflr3pTg6LLsTpUZNgr9OB3yuDjY8h3+7AHBeSHI+ywzymQi21VEIswrOQKL0yjNyi9FN+35XZcs20MSUW1nsn19dP3EHmCGi3a04ZNb\\\\\\\/9H0CnRkVBlY0Kl8AyhwiJxxjm0MUFVfy3mpmpF2VxVdcXyEUCilZ7V1a9jqqgOTeAx2BHjoqMLaIS1\\\\\\\/cBmdkMs\\\\\\\/+U\\\\\\\/3Y4BLNb14aWeqKghVmgKDW9pvXERm6L3jvadeg9BXbaAx\\\\\\\/Au9F92WsyN97o8TzcSRS3higaqgcT\\\\\\\/\\\\\\\/qqMMcH4IWozQJI3yeEQ1ffeHafbu6M7yP3jhZEPqcNXfsm+EFDs9Hb95XpsifVqISRHdmR5YBQzpU75U5ygNCUg306GvN6HmwAlCZCAdk+K+m2kzlS81Kcz7LxIW11tCmPQVRieCt0wYvIwS8GpLKDCSoH9gCcIaTZa\\\\\\\/EeIRXroGxYGhDSbzB4oFgHYA2TxK3vC14ZqU9Uamk\\\\\\\/W5bcTebipssRlQ71Lk8iSa8hskJldfLz2SMm15gnaR2DAGEGSCHR+wJEQKIbGBKJG6wFW2kD9T8aXXcqed61BdVdMvMHXh4unX158cMOoBXJff+74ggIS1OYgaRXXJyEcP2kxPBkHMb2IUz99korY2Kqs193HSojx+VR6iBLtovGYL3Dc6I\\\\\\\/DKzaIq9BrBAW10SVb9C50JhYw6XNAG3THqlkexwjHXWb6qeYFIRsS1zMwFYHPiJJQ8t2YoNPFryqIKXpnawSIHCVpNZX\\\\\\\/NIcwL0BCLhGzybAtvQaNIRKvBIU5URrl+SwtPmKfVj2zEBtW4LB07GCzsgBFaQJZx4JTxh10CsxEUpWG2VwUxp8Emntqs5dZeOQdiNLyr7pRwj0bEQ\\\\\\\/A0AYj1ZTw+ledBuKmVD\\\\\\\/+J2dbNnCRvOwQz\\\\\\\/gqaxzTGQjKYKC8yFr+7VYjJF4rT+yPYSfLfiA60ISVLBhz8exEvXjNGFHJNwISzUJ6kOomYzc+FwaE9\\\\\\\/PGdSMquJnuM3+jPh20OObYF4LSFt1zXr\\\\\\\/JLy+BrTtW87LxVs24oPMq1+sW6r5Ohpp7VeoGYd6e17CFdxtGrUp54W9KzF\\\\\\\/X1r+onVGzv6DQqRdBkKkyiE1iIXmg6HKZYicf4dcKwbU4ZzS1Hf4JLPPJyLrGGTIzoMnKqX7u+cRWtjJf9uMjgTR6o37N+NUqLsIFXlUrv\\\\\\\/bBmfDJ2X04A43a32m+SZLvcRJl9AtQbLDneeJy9OLP3WuNqnIQEhpyPEEiaQPRzhollcTxqtZMpNd6vxZOU2bSw2Ycds7B0p\\\\\\\/6AjnKPK6vwqh+NtQqCSxAnzxZlcdA5ZJ5XGXAIwk2Wj7REcPLUD5KkpLcDgz+37xGJqn\\\\\\\/iI9b8mqJqhr7G6NEnKYV38cv46Us6fvgvxyoRzyV9IcJMjciKQyAmMEC73RTBD6bcDTr2eRx4XPR34Aq7MERGN2\\\\\\\/kmy6J6R58q14tCMJOCuJMfjusM99YkP7XDUzhpcJJSoAll9dJsfD9OQBRi\\\\\\\/pS48eK7hWNEuXsYDeiS34FG5BkQNrWoKkq2WZ+M5LgPdm37e\\\\\\\/i6RVezX+9pktaFqNNP8Csp5SBf2hq0P8NJUdT6KF8q8ZX9sAI9NzJZXr1gQee8ab9yZEH8oitqDhFxr7C2RnbidNTG1Ek1Y78zmwgBz6tvxBPqfGxf761Lc\\\\\\\/z0UcbfroJXslSNdPkllVk4YNX9HiIXsjwEvQjtOzfVmQI9WAIQmYHOdKdQSOfaPCugaN98Sjkz1ZBoTkSV+B\\\\\\\/T1Wum0C1WWy9kTkLQtVA7Q2Wixik6TcmjTx\\\\\\\/m5FTksLHvzDbDRMI3\\\\\\\/qauPJ7amHZ8msSEyE5VsVZ1B6HQyIbdN8Fz2IwcV9R2p1Z02jbTbHpmiauqo0IDevm\\\\\\\/6fF6kFFR8jp\\\\\\\/NdwG7bYgyZKdiqz89ZiPBmdeHjVuiyIULS4qsI1I177MidJAraNmqCcqJX3pxzPquUieEsGTSVok40doELFh09UWS\\\\\\\/rmlO66FiqqI2gW+9TpiAsFXCVn+7qRhu3p9sv7fomAcXjEzxwfvNrw7WhS2xTARrp2LMRD2MiPTAs5D5n3Rqu+xmb1QsqQRLRGXNHwc4VqWmjfBzctNZMG+nwFBvUWhRfp6qIBLA\\\\\\\/JCye90xCRaQkgQoA0CsUm08ltOR6Kujcd1q9fooXk6KS+kEuB6bWkY6M3DNI\\\\\\\/FLeE16dvbVMSTtWq8dIXmePFFnOj9OZ0hPEf6Oxs3NEWfJiXGfAiAHbVULd6GmXumiTA6fFlQ6x2lt2+DbSMM6+CXFWjKUEj6b6gSv79IRPG4yr4e317AKITB1y5usS3ZmGj3CPeouzPU+ozf27uFZlvNv9+xLZHu9dQxsgKdZ\\\\\\\/8MRAr9dlEQDTY1E+wqxe6E2dUU8yuRy641ADVo4FrQDzsOMVCJ88f9FNEAjwGA8Ciu7zvzBYkL8Mo1USp3LkMiBhZbDi+qyiX7Ti1+HgeIQJPge4FDvYHalSGullTDHjX4COS9HHR5OjZTUuOYIjnEbQm20413m2OHTfNwRseB4doPgAg2XB8HljOh82BUDcIHscOvTfVGkaXH0acDMXoTAbRLU4a5jtwz9AeHSZDUUDxIvzuWvZTepRjyb349seL+H69wxZjcbZj1LshhzS6VCR9cIXtFtCiL6COfawOhOMFv6kbO67G3If0TcbyOLbq\\\\\\\/P93saZQ9fwrFC9RPaZyqPr+bppDjBbI3NV3rgmWf9KAu5hHDBHAQz5cfaSduj+dNuT23ZqgvWQ9pXkx2alrumhuledOp8VLSLlAK1ME0FxrCWTrD45HOvfST5euXoPdGVAEd59SwJWBbds5ZFhNxisoYBpSQIPOn184Xkp7yzOw8OfWvWpzKS+VvHWwlb+o1Uv3dEzZmm93fiZ\\\\\\\/I2ypP63Wh\\\\\\\/eloEvOLCp1DSF2arBHFUwNqAiGl3w+cqc7tZg4GuHmWHbDeusE9RD27pbl1+TGavl\\\\\\\/RGq2zb6BbgPZ\\\\\\\/Qf4w1x6xtwunJsh5pYv2t6NYWnUfsly\\\\\\\/Pn5fc0+SytfqRaV+HQmLx5oGaSgxAbfZ7ZqKH30kqL5JMO0tfbNwOhhw+Ze+OuCwf1cIEVWYO7DwNDae9gqyRPVURhvn8gDnda\\\\\\\/9mHetSfvdkSEt7Ru0\\\\\\\/JQF5NlC4BERFYyHIVExypx4dK\\\\\\\/lEan74InI6K8eSBAwbDVc\\\\\\\/FKLxe\\\\\\\/Oflj7Tw040APu\\\\\\\/LZ3JjXIAf6MK1CfojVYp79Q3jWyp62Xba1h4zZ38Gb7TO4UmSsYrG2j1AvbxSRMnkygUR2FhiRsu24kSM7GKuLAoDUR7bIJhRp4NbN3bBX\\\\\\\/DjGzZIEOHSkP8ZM3WT+DFgDDlBReBD5norA9ILlg3XVBHcPHWPV9TX2eSuHGenEChu3PDbzniLK7R\\\\\\\/BuEUjL4dnj7AjFE4\\\\\\\/jIjxUxylCJXgEhCWHrFACFoG0\\\\\\\/xHJJV+0hYmJux5bLAodBJJUzrs0ort2zgCIUJQjl1bc\\\\\\\/6UjN3aFPpJn6EiNC3Pk2OYdGVi+Sp1R\\\\\\\/NocL7YzOG+mXAxBm0Ihb0zOjzFM2f\\\\\\\/XE6NIWvQhJaCSn516YasuF17VrjEwhZ8nO6tAGZ\\\\\\\/k0uqv5L\\\\\\\/yVs8Kt2yvx6A4KERB0Bg\\\\\\\/dZyY5BowiR4HO61V+a+oV4nxMdNg7oJXBLYi\\\\\\\/986OxxdoXk+ZEEtLl+W5YWXPsDT0OJcnZf25Ebmq7GE8Fs07YtXNndqTYOG7vEDzboXC1bDqt10hQWcmSDdQjbYG5goqHAXM8j9NQSYJ1rcuA6zAvSNbM9unRLPENto\\\\\\\/PNLDyk7Q7iLZ78qQdlSeh7F7Ye1F7BYaUjGDyYRwf9FJG89ZKEr1nN1SRmVJSllP2fXNtJ+Fa+iypH\\\\\\\/wpvLIbAIfUUjfZGcp6SQ3myapeuQ8Tdgb3\\\\\\\/8n9KZFzWjoE7MW0Xl9shhvHq+KGyI22veINOmppT77DZcft7y1qphDkjb\\\\\\\/\\\\\\\/C++X2H+A31l1c4XP3jpDmsd2HcXrpIG0PtxuLEpmu1NpAUmDBshEBOjzVOrgykJsa4lvm4nolpRSqzL38HE79PwcNbBQkAqh9fZVKutzA2ev2n+L8IJyTMdtOFazKZpmfBsVLx7v3rqqD7qXFajufmJ5MXc2VnptfV5aQbDx0fVcATSBn7qyDjDEgnUHft1tNZVCyFACtJ7q8ODcU3fh2jltj\\\\\\\/KzlpxvSODSDInAcogJegOlYf7alji9Cy57t9gd04ZgvR+LL7frztkfwlTreVpT+fNuYvr8Gw9ipRJEU2GYRFuHKbZ46fbI0bzFViBPRvnclLFg4BFYx50anf0adNfBI+eHmm+A2HvNjE\\\\\\\/FIVf67ooN+Oo1H91hcm1+MhYjHDKi6TG5iPVvqa\\\\\\\/EfOpEFj+bdg4xaDBdyYUQ6WH6I+zvk7\\\\\\\/FjdRSQZhPsWDVkf62V7kiwSD1YRXIjsG5vfxlJV0v9Hk09Pe1L\\\\\\\/YnPVhgDORTpngNehoCgDOTxSt9EJQOs40lnZJusrfKpPjCM1+QLPcbTXuPWn9RqURkOOZ83p2QJyC7iifNXoQ78mW1MXqU\\\\\\\/cUlpTEpRjZIW\\\\\\\/dtqwafPKMlRZIJeTJrTL9idlTuQbEBNjYlF1GSX0Gv6quyS5nRAO2C1zafbZkn9ZC9SHA4BEi6cq1Nnk+vYALEh2vv0OXFsCqOsONLpam07xwgGdaoiSYhJ8VLwFMqNaFyYLj6wqdiKOgM3yO8aHBrzQcC\\\\\\\/g7PTGswIzU6WjCEWoXBYLyiiTyz8EJGNlg4cabYhAhRn5p4rjoCfAONIi\\\\\\\/BxnzLf3I8F0cHZ\\\\\\\/c4GWK1bfMzeBx0RxNBLJgQh9BouRunacsupbAPobIEbSsm8lRxUbtWB\\\\\\\/16f17BBv+5DszawVXNP6ZPTeg5Tqc8FY7shdKTml\\\\\\\/gcMM5zqfoQzE+gAx2cS4W41jK7blwEAk\\\\\\\/vJumEl75xTg7dP8FBhx1e342s5ul14fvJISr4RSKHMXM3Jn9AGSchruYIQx85YQT1D\\\\\\\/2q8h9PjzEE7HJtk7hqyHsorBSh\\\\\\\/FijbWEFyj2lVALPtIr90BGfPyB2qFpKrjRDUGRF68aaPEUi5Hzzgz5e50kyFuwvMAE8FYr9mkPponbwuxWjC5dNTjbOUndiD65hsJDnmtCtHLtJy3SV86zFEuNLl8\\\\\\\/NC+qT1KJtBck0J2UEokT+yi+iRr4zKbkCJApYWhFNHYpjcRSkzsEH9nJ3KVPqAG\\\\\\\/78ulqsD4BKAO9blE+hcMkwYLUh41UNg8tMWoQxv8FZlKKtwLY0fqeNgUf\\\\\\\/Y6toHwrj6au2Wk+GYtqLOutTBtLwf3NnmSQx2an4q\\\\\\\/KITrR57NKi3hl2eIbyMVTDBhcbXlcXeeW5IPe8SnKByjzhCdMZzGFlVzQNZ2AKRQpYh59eVf76ZhZ6NCoqNk9R40JIzS6Yc3l4KZkZPd91USIzOc4Ieaavcl06T4\\\\\\\/D5QSMVaqtLgCkhHp0KdbXWzsAq1cqEnDJtYuY6LmhMZvdGDCkTOofQGMaf9pw0CnPu5zkkbyH4LpWcHdoKcKDxIinRXgMHUZ4wssxS\\\\\\\/XyrIdEglc1Ww56KBA1qsQn6zR8uXh9BeFclb4JPcd5nTF3l\\\\\\\/uUfkJDPKaNH5UTFqUF2ppUJ+qVxwCNovKjF9By3Q1YNeHXxUplZOAftk7yZcm6H7rq\\\\\\\/6vCX+VPYRatVJ40uHgXkye1NukYtwhtjuFBnCirxMLzUJYw4SqsrtPcHkeMVMALODMHihAf\\\\\\\/0jjWCPfn3uVEek7oG4z5puklYYZzFDZZhOzG408wxrsEqs17\\\\\\\/LB7hzL5prJEVqYVvqQvpX6x11TFR+lL0Vuuf9JzeH0NmdcogLesYNusfquVy3p+GbSCknuSphD71gfRg1unnRtMoFiWBzvZkF2BATyyNR4gnEV+b3MvxfS8YxP8IG4tHL5jDduiVLnwN17fTZCTRxvIrFlwr+bBmQgf1eMGGIFT+6YDoO8MEb6UzU+ke9kqVpmLnS3wNzNfBPiELmvDbyaQWL8v5UJdo9EKkjh6Eq9X6NaUOANPxIKnmNs\\\\\\\/6Wx1eNmUAsDKy7q2VgIjjBoGY0ZDGkEpqbp8anzf5GTjyeJYO4UKDvordpHgSC8p97aPk0th1CtsLa2DCWLehlrO70A6FCYQO\\\\\\\/u6lIhyrnLaNUuS99ODXe6E3ePWPMcLbOqvzaerfK8sIt2nIaOCPminNY0FJwpmIjBpfGp0Be01Ie6nZ\\\\\\\/aagB3XK\\\\\\\/efNv1Z\\\\\\\/XdJXeXt2HP\\\\\\\/gjt1a5LjrnckNbQMLRFJnKz02ohNaGwmfW59Ny0YGCIkdC9QAfqYpMTmYIGugvWWJYArkVfUhYSFC48oeqDrfnYojk4XuKa3VxoF8nmtGJMOWAxp+wzu+Luh1jaDtQST9GWYPSw27TToW9eb0S9IG3uBmv\\\\\\\/VExqw+3\\\\\\\/UODxcuL8Vh+jvYBnq+lx2\\\\\\\/u8tf0Q5WOp\\\\\\\/VuwW4PWepPzafnfj5jIn2M1b+7gpQ+OrqRTmCa4BzjITS7gL8k1j7icvWogFmp7SusDjseAz0HjpXdHR6Asvhw+hEVSl7o+cu3UTnrJXdibliDIWCt2RU8hNxW+ivmICc6cUgkz6yddEhtrVO5GUTNP1Kb9UPa+YXEp6ZELWbZgnppmYRN1zrcHPGpHPFNTnphNtj7wApJsJe3nodo\\\\\\\/IXVWdlRVXcYcEbrY7QhS1EDBSWhKjuMUAjZJKDlRZ+Cja8dcqsdZz2PfLKaYHAasJFI63iIAXWPNbqFGkACF4EMr2yZWKamzVwsBPXeDKEeNA70B9MwdmNIHZ2sey8KJkIS\\\\\\\/yFbNi444MmVXqwLWt8tANzg+co8W+qsVJjTV2MMkW+ogB5sZ54ZbUWuB2aOzDvZecGT0sMT4RuUeHigKImkMSQ\\\\\\\/iVWx6RL5ZzIurCN5Or5PGO46tjjqWU68oWfiHjhgq+2tX\\\\\\\/umP1sNmB9nFr\\\\\\\/HMTIRSOrFvaulrj2o5WyNxhSBdygdZWiGQlaAXcb5\\\\\\\/S4OAzkH4V64rmYBglEYxeNWRSB\\\\\\\/c18jrGSiBDleTKdNTbbz+oKRrxgM1P9KxVR031fYu7JDS1IiVvf8\\\\\\\/u5XZ3f5iM7qGN8mF9oqW8wahsT74XQhouMI6QYGT0ICxvsRF7i+XJuqwq8xXtyHl79MSk487xE04EC0\\\\\\\/1pnP56oj\\\\\\\/L7WRaKLdnxKWFv2NCClbssDRRL462ld0Rl9Z8Q4IlPcCsaKkmCgsENFWa\\\\\\\/GLHYsLFNaoKG2m0DJzYGzxgjqJqxUyPiH2HAUKaC9PernnLr8Cc\\\\\\\/hQ\\\\\\\/NEikc6MlzZGXPwBuk16OvHSNPCCqd9tXv+uuX9YJdbD+WQiBisHzSf+oNVcBG8RoMd5pS6BwERqDkCSW5SptxuOU\\\\\\\/u38qxvGUUZSk4X6zgDVoxUaocslqQG28T9xw8KVDIkh+g5D5kQATwoYsUVuUowelGSZgg6hIcMhiUbBM2B3zidJkU+PSs3p5JbBhpqZZI3fdxM9jxWyWkfkqL7qGbN215LUuq1gHKn24PUL6qB0Vxm6z2Cby0ZSwJBQaG1\\\\\\\/5iKBTSk5dbi0i9Aqss6Voin0OTbRLDgJ7jmccEfrgQS2XuC7OoKOt3P+l4OBGsIyGLTjwRsULQPOIVVVtTOfWo3BwHyNEscUY4duKM1S5tgjcI5+9zH9d9R2RB8AXeLKRVYolesYndHjdFdhkSLh6NCQotqbDFogM8QDVtZZYiwKx824mUhWUqepGryS0duIxcX\\\\\\\/UTKvJc2LA39heHLweJqew8q6r+Pi58uS52vmXEp+2+25cF3S9LtmWkNYqhuLkXCY6c2Ld4iL48JnfcBI5QuAiyIpC4phEmaFfI3Hg9phFYcomH\\\\\\\/TAvU+I4fFvqpzhO5jGBxOh4gD0ZbMkFDFzcP+eNPbMHEkw3qxeKEdB0ULXrF0RYCo0cN5p4dW5fljWb1+BWpK0zkpzywD43UG3B1CDLNcwdBj8QL3go9\\\\\\\/xXETIkPzVGgqfRSxiXHCL4zCTKzgmCpJ1KXZe8uJhsGIBdiYjguNn3yWrB5w197KaIxfbBO7PO8pVtPVZt8KpQi3rN5f96rCrMmMAGyug8suKHjHM2ARCNLyzQNShwVUzaLNaJJWI2rMbKvvkAwhd0EKm7NjHJBdS3YbXXSq\\\\\\\/7hPbYj6RLtyiHwBIdkazBvemkw4h4BHtjAwGBd\\\\\\\/WeUpaqZ91cr2YVd8PXI6G9e9TWHTM6k3AKXMJFhfxyIL48GdxQ+YC6hcR7WyIfRqjVUebHLpCnXB0eRNf\\\\\\\/B\\\\\\\/C3JRLhm7jSeFgSIs5TR84\\\\\\\/8jBe09BGT6444VqsgPdcpfQT91b\\\\\\\/lsp0qiYBPbh2wlIepmIXp\\\\\\\/j0mjMH8+ctr2q81fXONYmLzggez1IBSL+Jwjk4AlyPPrb3OSxUzOXg54sfthdshzhoB7h8uuU+v7MviqYbUpp63QwJeJPmuqT+lttJGpPRPxrA\\\\\\\/YEHMCzEe7oKXcUJdpyGnASvYSqvq7aO7SZ1B2zTXXvHbOiEHVmQWsEzdnZvFOlvdLijo48TclcEeAkq870zxwaPt2TKy1d+xToB1tf6zJWs1JpU9e8a83jjyGzeUfBU7rPXEIKakSt+q3xiSkOsQSTwNPSjR5Er+zqInb2L5o5aP4HYhV\\\\\\\/KjbCaWhM+G+3GGXfYWdOLowNseOgxhBG33c1vDQsEkn4wGzldM2eUlCIqAOtcfty1lN8edGeXmIYTWDABRcmIW5lq2yEyPjU5Wx5ERJaNjD0OSMhdfB5SfM3RxJGACHuYtJoPcM88N2JrS2VmXJxYikaGvi4yFbFDhLFlBCyfjSJvmAslt152E0f8pximWhzokVRuiCoeoCBfv71qPH8uCTUh2BJYF\\\\\\\/iwrNWbwRw+CQ2qHMGBxPtW\\\\\\\/VNjCRVe1FdwkU3txC+hLSyPgWW1DMzVb1lJin4FrlR31p+Hw7aMsOVcdR6YhgdcuHpzSoPapvu2UI1YxrBcbVxTisWYFZF1\\\\\\\/BBoityyoAjR1xeuPLlVjYYASo0gpovCwzZVGZ0urske9U71DgkZhFMjxb4jjLrq50Fd9fZL6EnRIJoghxOib1558nrB4oQBG\\\\\\\/fLM8bIuChhuLqJTbZqKTCimchdMVonPQS+CJ5pwDlgvPmQi7cEXDj9A545asv9sV+3ZcGwHWfZ40UrYDv8OvwfRuvsQD5Ae56F+sMKa8UswjlmGVOLpH6JPSR1j\\\\\\\/GSWGlgMQA4vwSt\\\\\\\/nXCzffOeitK1lbG4aaybs2PvFGLgK4Z7rMZq+xsZUvz+yS6QHNVIfF9Fvj\\\\\\\/jEnk05FWH6wWby0V\\\\\\\/IRae4vGZ1Fjc36v3LFHXn80UgF5WL9eI\\\\\\\/dHjyhWEFXuX1ukRM2zuRAuH2vZr32JquwBqyHq9nmpMet6gKjjAPU75Z6HjkZ\\\\\\\/rV7CY1qTy0Sa5Q083BM7tDGvs4te+UL6DRV3Gxd1KjmmbrL0dS6NwC2tAD3qYt78p5cf3aSmmnM9bhFrmWDBKyyZbxCro3a8e2VYQ0q6dhNT0osH8NAPGoY5vb5ZglhxXiyfc\\\\\\\/MWEUINC09WD4GNaZEwgtwubqdElTiPVmDKZECDBksusQlUeReCf0TtzoaIao9pnbyj5Y0GO94TynIZ59kU1kuLdVGVTroLezzZEbXfa04XXgxbnMwylBeF9csaCeWWDt7q49Egd+gzNX03uAyrrBeHBaW++IQb9ozII6Rtc6ajJOFD4NB8EsQ8WvFotVCt2m9ul3LqpSfQGIc8SEbXKVZ8DYxrkPqIjb8GOTjsozIsBI+rw\\\\\\\/dC4eDcvDGf3\\\\\\\/pFsX2zD9TV6JS\\\\\\\/T+yKxdy7PGjsaBMXy5GRdSPCxPSNCYhgfyTj3Td2vK7UZ3nRLjYsyP212F30oz7UMupw1gziZgY2WzuPd2Dfl9wdzUZeWpERe5xk4tYeOOJUtP1cNINW+x9Adk60Z6FQde0yr0YeCbKD\\\\\\\/LRU17vGNRCr\\\\\\\/KsqccUkmJE00uX1SAcYHd\\\\\\\/kqUhziGYbN6V46JgeWuqA8ZUysl9Oom+\\\\\\\/ouIVCFKR5jAF7Ybu\\\\\\\/W4ljD5kQdhn6FutjqUA2FNVp18UsnC4RyIq3j8Z8cHhJP4hxLoKkN44Vq9VaX6V8aTQNvUlU9x7YhkZ7hEZu+GtQ2CUvpcf7NDzJJYXTggrUvL0bdUfp8qNMq1DOgmOmxhFQQ95VGsGc665cheGUNt65FXcjYoe4uHqezathyTJNAJt4q3tG98pGlYRc\\\\\\\/Tt+il2XCFxV3l4oPFtSP53vIOFNegOwiR9gRQ1zyN7+KYEVwH7mMGj5gGEieKRFlPDwftbhez7f9XSysbnD3gz8KlriNTNXJKwac\\\\\\\/pNfPCbrdAYJZvp0KEobyCqUhtnSW+dmAeKIDsBGi44BGs8g8TcqT9HAiVsZKXHmFQrFLYdurKmbK4SMbwvQdWBJcKhb1eBT6JaOh2I4pvX9dsy0BGXMlDxpdXpYrnt8J0ggzcplOdzJG\\\\\\\/t0msVpjE43UBUqcBMxwMnmHHUjZDU0Um6AcTe1gb0v9TL+ykLW8vJmB3bm5U\\\\\\\/B15dePveCU91rDGxxvwKwcMmtb2BkYlD3y7BMnVHX92ki5fk9qZagnsmqIj4iFRrD8y9aoQT4qbbLTE9VglDLpxMbqZRbyfqhB60R52KqvfeQQ7CbmaysuW7\\\\\\\/3IbiwzHsSS3vRCNmFLNJ5GKkkp\\\\\\\/YWJJNuUC6B73LAUU4Ajloks0ZADoBmKhS2ish0vAhdgnFy5YtqrkEkS2uZbEHj0pnuo71qgGJSXzPkb9hTTCFQljDQDEctffyj4rHIAzI5Wp1nxw4c6ht0h1cZC3Xrb5QySTNesBsAczllER3yH1t8Lc+t5GucXkSV8hol+hBX6Ul\\\\\\\/+dm1aj7Q4NgThnwj5cfWQ6eyOFl1\\\\\\\/vVWxHPJJlPm9ohkbc8d6cs34F8laCA4aTAiKcyldTXQhpIHwc9oav1ObwX52BifJwuhRiJlhpUIxXqbNTjzWGqBeeGoLT072sWIdhBL0NNxd\\\\\\\/vmQjwTVYNvKYBDVmFQeN\\\\\\\/150tLKZX7ZxlGWman0JwSkVgXJXQFMOxebg7J7x5vOn5vIanGcibV+6SDVWpC5gsiU+iSQibj8YN4LPYyuURdfFZiPnpYDBHPCFO0H2Pd8SdQ5ML6gJmY4YK8ZvXeFxB7NbJxLAi2yMYo9HV\\\\\\\/KGz1aGvoQVcgHrQ9TN6Yov9gb\\\\\\\/tn11i52EqcbTp83pEO0NNlXAdubI+UGQhecUy6WwvkyNYZzjt4G25TChEFFd4YPBIqP6q5LC3aJ6OlWWMQCqru4CfVpHAG2Qk\\\\\\\/QFzI29JjmKrbgmfy4MEHsQPNsDsCwAnyFTYZ97Ow7RG5CooybtqSOERtEfHRxae6fCAPYMq01eakZsTPyXZCgqc6Uj5rqvlIyA8BiLikJ9rlpm0fQAVgOdx\\\\\\\/MvBouqw7DvrSaxV2duEembvElqxZl7\\\\\\\/z\\\\\\\/pgHVHyhQb0ObTQpHl54jcEjj9ANrrDwnl25pVPbqFjnfg1KjoFMHZpGk3to2OlLJmaad09VnKLQtfuECfsXR+\\\\\\\/NONPvmI1a\\\\\\\/SP\\\\\\\/1oN\\\\\\\/XgKU7fn27HcEmxq\\\\\\\/2qH7pgeGhK51q4xwJaoX\\\\\\\/suBZ7XdlNb1rY762kO0mVZQM7v+XlvSrQIXPqKyc\\\\\\\/fO21aXAPdOwdzZ9J99mF6EkXrbGerUtDg8ekoQt\\\\\\\/ZvaQYYBadGB0yaWhKIj9VniNCbj93opRruWM0UGpP2DSv5LWhb4DW+V6td\\\\\\\/Wh19BTWKygwN74PT05jbK1ell\\\\\\\/lzzioIqZWsgsUnI4WEqWgh99m5n9UcxIhBLcAkLdcfByIch5Wi9VATS0\\\\\\\/5POKJaScTeHyrz4QdF2PvDGgYyBKNYM\\\\\\\/W+IRPo5uP7dRNxYHluMFVXKYOx743j70tV8FEDi4YTb0ilp0aIyTTTfgzgKEzl1hhyaVkwTaCEosgvrIpx5RJdAgl1FBuLF0cJUppjxwqa6HVxir33IDtRtKJjTHymAfp+QvJRqv9Ebkko+\\\\\\\/gF9AjQs6q53DMhslpOQcBXArErsMEX3IyIC0Lqyl5YUF6KlNG3a\\\\\\\/gmIiveyNrtZcQ92lMrU7YYObzc6d0qvdX0jkAjlyPg03AEKb1ae6FOTn25oqKsa8fZb6Dv2+eMbf1amEWA\\\\\\\/EOunLZZYbKKvgluhFprErWmcmbo44DEv1dS55hvHJwCZxbuWBe0g7mzn48nhHVS9MmGmOqojW4Wv2lxXykKR9J\\\\\\\/G0QbyUwtNgJd7+C0VtpV2aankw82Gd\\\\\\\/Stdkw8BZvwtlProOU5JNSov9TnrTTBS4qRsS74BZW22v1fXmVrRPtQqfunA9BJYH9AkGj\\\\\\\/B6TPNWABV7+Jj4MKkTZ2mGQupcweyNlW5OFaatu1i3l90azzjdT7f0Ua9vqSK6\\\\\\\/n21bcuTLe6+uFmQPLQEyZyzesS9\\\\\\\/Hv9HoV7uKM79ZsBEvpM0IajYcGIAITn7j4Coe+70IvLtByhp7thawjPNSzQ1R7D1j4hiO7k557LzdaYYwDaCEBxP6UdnJMoeRx4ETOyWMYHZBQYs0doBTqRwbQAdjXKg\\\\\\\/MOjNqkRsOEvYjDXgX\\\\\\\/Y7h0WtLKJErr8ff4LPewrGigpZ01IzjvqCs\\\\\\\/bJw3xbBIRkTEz0EMxEUF+lxG6k+Dd7juhf1y3O8wBa7VqJ0\\\\\\\/zFqfxpSl2ai3dsBfIMasb4EsaNHrC60R30znVeM\\\\\\\/RkgapjS409+FFGYt87ynmjOSEXe8eLyo4mv9U8ZNnzMf8293BGKN8qLEXiT54bEGvhUmEoQuCH1UatWTifUcDUpm5w65NkygpIX3T2d2hvWngwtRxsJNFBeA+0fi5T6JOVrviohxD0NSw89oZBRJFBXBXnhqf3ys3HGIn2AxTyOoOmMLGdgAOKTUTo+Y\\\\\\\/a44I9Bsl1x4v2vUN6tf3J54S1o5qdGPA1X1tq5Mnzozq5pNeQ\\\\\\\/suh4EU4sj\\\\\\\/2lscove89OP6TYngjnxvaQHI2mBlz2a8xtOvcrwTvCmJz16Sl75SR7JDxzTZU\\\\\\\/BCXqwzEfb7zAOi44EFPz96WNrmQmMXN7YGxuhmkbq6LD+PTPbhOjcCDUVulVqjG7zdEU2TDDlp5QDrXlK64gs8Rxq+htdozwHDwFLgLbjUdCVk2oIJVTsqsl4z5+H4xGa47mxxkZc0MaF85TmEmgsLV0Ek+nAll2lHXbpRCWJQsf9zer0S+RqejdsY0X8BqPTSClDH6u85mjd2FMGUoOYiKTCz3KgUgMGCxm433XcMXZY+lZbTGOcxJm9WNxWmgPmzGISyhZWzGyiAg\\\\\\\/dptFQhy1pj0XwHYEWReGi8u+LQqtYysbmXXryjD3teV8CVB4tsrxPcbPwm5DT1noJzxlBhP6hM\\\\\\\/s+6jma+gReTmB46onnW\\\\\\\/q4qqIvQGKPZtI5JhpP7Fg0jt3Rnf6j\\\\\\\/vPrzlZqitATUX\\\\\\\/8l\\\\\\\/1dBFl\\\\\\\/bwhFz0TTdgrT0IucOudlkK0KBSurmxKH2+i2auXiLpeIF0oZoXWakxfeTW2WCf6hUKrGqES2UWTRqCYlRq0WDjAbtVWtITQFuJJ8fn0JetkVLhf0ZZhpjCDRb0TU0Z4qf9zmXCsVT7bIFjMx9B6zzlRlrE2gLFEUR\\\\\\\/5fbd73cvI4toQwKyAJaFhMc\\\\\\\/c2dIxMDRnqThoY2x\\\\\\\/CMrL+0Cw++YAh\\\\\\\/1rL0kG6NKtyaH46ARxfocjaOCjSYkLlesrCVgbGONl7WoatN66xcB91\\\\\\\/0jbT+lGFBaG4bFOvjNm7rO1irc6kiJcvEEUX\\\\\\\/Nd+1tijU1+Zs\\\\\\\/QRsxvCSDydbi64z24xqtiSRRHbamb7KFO0wdLnFGuddSKEgYl56UIkRzotc8f9+RQZhfFUb8KuZDytj5ISHGDmXkH8v3WYhPXBoH4WapRUF4w4NP+aqaayg3k2LR+xXASWVoshani2C+KX3rxMWLFNyHFBoNF3GkQqW+LdVmXPESXL6\\\\\\\/hbw9u6BxFKHL05eiD3BWMORMXRPkXe+Pi7OIcUDeb+unEorL70CnM+xy+wlc2KFRCdgzB1dsHFw3lwljYZK\\\\\\\/rtCyiJxTpj8jqGBrWlA4tZJJudgBPwxG+dtVALka34jJIOMFGzYg7cfHI7aV8r6gyMtZfXTQdQdQ3H3p9Km2rk3K355BB+WYcY+vbS2\\\\\\\/0SuT0W5ZzITMWw4juZ\\\\\\\/9MwyLtKzXad0eoMw\\\\\\\/yiB9LiayEEPB0+i5m1oDGAsRcV40SRTnU3RN1LzXNqnIEWIogjEASfxgbbwf46r0LunFi+inIKIlns3iiV3gN0Ydz8qqgWsPvRKycg2QHk1NXu\\\\\\\/XSni24kNTyPQrpZMxWOl2hGIGDXn+NBjNKVLdhvNWheuvuuC+m8eA8zOgcD+WQIMDh2YKOz6w1Hh4PbLvQcL\\\\\\\/vv6YsFhd3nXWHyQ8pKIBg8PPYp90oA7QdQ2rGY8qqJ3\\\\\\\/7uZMF5k5zsWDDH7DSsI6BgD4WCJ7A9V5X2oXuhzIeHMHQhJMLKQ+4mWNLnKEx\\\\\\\/z91bwoMKxvaIxL6TtSqaSPy591WKOSsZtGZE6xm93oznpVosmmtIwRLr+Zv0DNoWgLi4lfsau2a+NcMLRGvNRQualgixRhkr1IrQbuz9rG1Jzt81q7\\\\\\\/W9OvplUBl6F+o4sW1oJL\\\\\\\/qne7ilz0H8z1pi6m98jZJc4QqP0\\\\\\\/duRSR9A76Jqv5AKa9n4R8W3n1HfrctdMSdovcEzCOuxpXiBLCcfbs9Itf9SgFLQvgIxuVmL4b38k4Y4SSw2intErcw2qyeb2\\\\\\\/7zMwMZec+jiATviJrxJXs\\\\\\\/1q3+f1ajMtdx+XZH2\\\\\\\/MI\\\\\\\/A\\\\\\\/opGYfmEsjkmstSPQb3rMYso9iPZ9NjWzlQY7pBZQ0neDYW1h4hy8cqbMXL7cBsGdFckBgo9vz147Jgd1RByyLRcM\\\\\\\/0nTQdcL52Hy8aTQee1cO7OEjk554rGu5yeA2NjwhQSUEpY\\\\\\\/aV3Iuiwj4a4CJX7MDJXl0XjlRTWx+cUSQgA6O58H+mOwDzOqbDPIJoqofjO7+b1YJbbkbg\\\\\\\/bOh0b+rEe+sCzBt92lpNVe8ASoTB5LDuHFGZFT1QSXDxXSFrgpqbwziXWk82ojQRF+BIifvTttX5jZUgTzuJ6vI5jrRw\\\\\\\/jsNed1r0at3u05d3PwNzrONRlBCTq0VS1i5RuXmL6orQSS8rFfflUpDGgHPWXnXKY4fESXgXNNHE0\\\\\\\/G8WWgRs3Pe\\\\\\\/nZlTwKJuofqiSdJI7yvYTgA+2LyN2qLLaZcwYWEH7vKsT6t593XeGIghE0vbThyaSGa274ckWPVPUNzRuSmUdVNgKOLJ8VeSy0Ef6pIoW+IHtkhnX1I3WoFkDXxtFMMnMtm4CFZoo\\\\\\\/zJDv9SHsbM0UwNXOwB0GFG0gjsdeEG7F54AVCEBv8hflJgqeSCj1q+lUCHxd9fNvOV2FKZ5cINGn0gtZrUtO6bGSRoNTYuCv1Y8oTD\\\\\\\/U6rp0wuk5+luErga9S0tEku4wR6oSIVPNZo7bjwovVKhjuEl4AJpveCWb0Lax8\\\\\\\/J\\\\\\\/NyBmGgoe6wavN4U\\\\\\\/Ux5AuFheFca\\\\\\\/sVHhyhqfkYW2zsgeRf7ltcyyN0EQhWGqU9beIEkjqMAMtUTUfnApkDGcsPtKUdoXbGZ5YCNorh8EHJKbvUhCkDvgReapBY63TqmuSRS0qnIA08uK8ZXNYHNzYJF8MuOdVD6UtnKpnnVt0BejLCXzxOvsJmegeN6XGYDYwZ4voNm5FooFQHBNmcpHJVDl2dfgBEKYmLN86MaJ6DiH6eEPlqwR0wW6fb\\\\\\\/XTSuMtgpncg+ETtnKtZL8Y9sKJo8I5rHrbn0sCuzFvRktU974iZyvci5MZ3cuWrrEjjlS5r6m74Ih9ZGNiGay9RU0+88nrMmmEFDnD3oumSVZTIeCiZEDZUutr\\\\\\\/AKmPdldVdeqaFwSort\\\\\\\/mgGW6g9ZaqAvoUS9egEysK3ulIPsf3R4ZCdtQGT1wtiOuCmi\\\\\\\/fryA0EDGreRP8LpxOySAX34aU3kKKHkil1MBnmZfrf9TXklA3b7TwcDc2oBtSjKFeMladOP3QHmghl7u28LEnjB0pdGsdHh3P0dgtC29y53d92tndO\\\\\\\/v6Sd4EBhC341G\\\\\\\/OeGSildwUa71c61FpQ6PU+IABAYeIUxHSnmW0lQOKGdJiS+qqIm2A3uV7kwXBwT7lIRGxRmoWV5p1b\\\\\\\/irk\\\\\\\/1XZtQsDpI9KLUGbf\\\\\\\/VvtxVcwAKiVR\\\\\\\/z+wXlyXfOrC3ZfQVIVyCJvYrXN9\\\\\\\/RY6LxPMxEQXf71rabwJYwl48bibY+QVuYubbZTZdKjoAhPPBg7hv\\\\\\\/IsmC8w\\\\\\\/cwu6UMDIdi35QFa41P57h07FbdGGrDABlE6DuyIacgAzu1DYADHfQFn4NaUW4isILTsg4FaHuCAx5AV6EBHPBgoOBW1Z4+u9qg0xfcA4R8tN6cHMdhlbJNVERrI6FqNE8IwVuW\\\\\\\/NAKZg2afpSY\\\\\\\/zdTCP+oRJsJDErQ5ee+xUAOgYDPBV7PnNpxxW8iUk1mLr1sKZs9FP50bPpkUmanddNQ\\\\\\\/fZAQCPKmCfdyR\\\\\\\/Jw251YDXvDNhVr6xW7MFmRFZ8UpHGTkHtoPnR4nKrTUfmXkriC8GbWXid6HCDZFV4Pjm7krB\\\\\\\/BAj3FNmtJ+EcTzP2tTJw7vKxMGxZ7XNOemYLaH7ofPMjUhJk7BkL2cKpdhv0iNdaIxdra7GDd9XWwwp5o9TLavH4B0X+hGteH+OdtZVG3HVLk9L9j6cdOLMxrkWnhrXK\\\\\\\/m0u2WTPIuORdGIzvXit05wxY+gdOusO6cwLal124sOdfTAWEECR6euIUB3kHiHJx\\\\\\\/Zp0ZB9\\\\\\\/XJXjjuYiQ3tWC5prqoIokk5\\\\\\\/+38AT7QXXdra7e4dq5FFMyaPfLAb52DHo6kIVyX11X1rxsKqVur\\\\\\\/f9AiEYz6t1K+grsryDanN\\\\\\\/G1SPknVGu0TPVOaAbrt1ji\\\\\\\/asNop2YQCQabp1qImV5wjB9A24dmGeiaFYbPsPbZxLLo4sor+qAMEzWc0UTvgmAA6vEBjlWhiNU76DVsbJ6G3qqYquvGEqAATNsXgWeemRdoCf39oByfd6N8A12u4hvGzluVx3LGespo6Lo\\\\\\\/7EQAhtqGbQt0V9\\\\\\\/8l\\\\\\\/j1QAmFf7q8b2Ns+TCd6gJIjL6ITHKheYBIk\\\\\\\/K7YPZ3SitIdT4pBKIDATmtF\\\\\\\/LdA0x7qb+KznspBq1oPmthV5hysfpI8h\\\\\\\/\\\\\\\/ra9flcBYur6qB+\\\\\\\/GW2udJx\\\\\\\/YwCxuzEiHqCo7OZ\\\\\\\/ZyXCMRGYW3nSTBvHAzo7XWu02\\\\\\\/H\\\\\\\/ZlAYC9Vgihtk\\\\\\\/thbm1nIlTeEnBZb9YgJCV4EJQgfekCybdOD+XS00RugBHjbiOjb0fowXJnultEEe9QU2yFDT184tmT8TGLDeRbamFZ9Zb8X4b0VzVKjxK081nI6aRnTokU5D1uNnoox6ZaWOOpRBzl3OVhNf0JXSTJ4tf2HdyGMVf1QNFHoTFeJgzTmV5lMlm+D6Eug8XCPsUT3YJ0pr4Cb4f7dOIdvcmXFTVmIvMJC5P2fS3gCLjYUI79sG4HiRMmlkxksHyK0aBmeoIyX9RI2X4ijer3gaiV+btB22mWTZkmxakJxrVG0ZrEcItpjkj41dEk5PFh7hWaAiZV1RRCEv2NGrGuKcGE8tky8D0nAA9BWguBVWWeyM6IoiO2f5N1oHWZvePd+HvOvxUeb9mOqEpfPB0D2jHuYyG+4cx1qbI3XozjPvWXSWtEExzh17h+zVOWSZOl\\\u0022,\\\u0022iv\\\u0022:\\\u0022iKc+8kdoyRd0LJ30FLjIyA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229e34eaa1c3389973475ed7689cfcf048e5f60bd049fb98e6e54ddbb2fc666878\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00226n+ERYE\\\\\\\/GRrSYvvmy7jxdtX+lxOavEGghSZhrLsXzIuWRnnZKD7FYIJMZwxPsoyOa588T7vSjYk9JJ+glgpfo3vNd3pzX2VmJpt6JNEWirakkfqGqf0li4tm91wENQQzcpYzcNed41EpqG8dV5cD9Q1ZQ+2NJcLfpyRDRrIjur716r\\\\\\\/hdyRKRcp3TY43\\\\\\\/O\\\\\\\/5oQJ1M5giVOm\\\\\\\/mMpF1+bM2dMTFIysjS38\\\\\\\/rmasNc7uv0icr9jUVazlGW4W7IINZpqVHPEUiInkNKtsAtU\\\\\\\/Nf\\\\\\\/\\\\\\\/BC7TsyX3JKpHx1Sov1Ftm8wr4WL0CMi20mErDL8QNonxRDVG5CyHU4knSj3Pw2MzcMRz+b\\\\\\\/xiqcoOkDyYUssPdMdliFPEwMFXLO4O+tl97uXX\\\\\\\/e81bCOIKFKQivUGpLYoB8HkxM3OA42rz4p11sDlY2dRKkvX+vljO6TrMtuB7Go5mmEpXYBrf5MSNiJgbd5BNKjz+81bkTJyjKsNg3a+4\\\\\\\/wLxhFAIYerCqbYvzqIWtE4ghVfAVmETiSjH3BZ8yfgJ2rHmLQCwN72izPL2XxDDJvUYYSVKT3K0dLu6rqxb6sgt22ggQuWkU3XMPaBZmHuOpBI1ZKrtUfQx\\\\\\\/fadtlj47CdFm6jeFHUqyn1+twsZlxtapCvs7lI9rGqkC1E+6+iFvEmBWwZ3HYXdMuo8VJmExx8wrK5HjDIt1iQ8gCwBX+2Yl22A06l6wBbDa\\\\\\\/6nppNaNEpKInp9SxdWHNyjb4wa1UpeU2TcHSzLU1BDND1jXAi2fjbove\\\\\\\/VlMKuDFoRfVtfWYulbQYMEL8qfEva\\\\\\\/blsXExTJMxj5b0C2VHEWyHHAwmikgPlAXMm+CPMh2TApjH6PmqkCBZnUCyz6vbDAAJIDLMxSgKAkHwVWwR4HJSUgG1SQAARF2D0hpmydaOKhruCL6i8MLd74gn2MUZ2bExuUvju4WKMwAtCwBu6LFSgMQlnbVfWC9VrJbu9S1MimaAO7Q1bSNNpGNRH8+VNn6MYfqYJXW3IMcCnwPsfbeEKXvat6N3N9U1XBJW0y5jsSUYlJV52us6IBOHZMPP\\\\\\\/e4WNa1wLkNtoL3ZfDZPNE1ZdWjrxGe3jZVEyVQ0Vz8HfuXM3hjVW5sVcvX+AoKxSnPJgbjNx255ulpQpzrKTFlioKF7rGHn3VW2V6JEV1O6GAlw7Jd0QzS17UrZU+Ocrqlj24aFCKBJ3RxCYFWSlFoo0eva2UFNfThkaGfEYGPCoRqkclslAL4\\\\\\\/c+OQsTVMzwhY3RsXWlv1VRVOmmohQJRQfS+V7yPDvGNIsPZGsFyOoJkyCoiWXsOQ7nbGLyZ3b91wpEp\\\\\\\/Dzb0EugmOmHKY05vVSHRSJke85GhBkx3Vd+Zrahg5kLCb9jya0hxJP8uHePKKNloNcDUMRnLCfiKzNcp2YLsD2wXqqtdzEqjWJmBKl7tzEhrjNl38r4Kwn3zRHX60iwn1GgCDYgQcLoYKY5Ss72kHTvJ+zhfphiGfY16W0s3dcJQpwWFQNxYq4EC9ddwsSXskNB6Zhbgm8Bs6b1zQx3CmvH8DJkSTHjbch1ajvbDEbSEQWd4je8o9wwUM2WM7hIQW51K+\\\\\\\/\\\\\\\/Nsmml1sx9T8rk5srKdSaTsbClyJpd6c2Z3cFUR7dOMis1FV5mciXb0YqbdlIbXNG0NQlbA5jwhC38BbJ\\\\\\\/XS\\\\\\\/g0kFGYKzDb+6pFNOiUIjw3II3CQ0hl64J2CrRpj4k5H55KQGT4Psn3JR6vnhUY\\\\\\\/i8vJP1ZrLFqmjcYMiMSx2J8Umy+pLgLM59IeQCurl1nRWNKW+qzROHlrHKZr1ytIzdpT1gxRtEADwBsq+QIbdrefM4SoLfCYPfBAIqVF6VUE5j2xVJXgxGNy2sya1AyLtJ8XnmcijV7pHeq4IYqkn2cnB1XpXnr8CqdReTHv+oY43SfgwErp8PdmkroG\\\\\\\/R++osIFiyGheuSD2yXRblNXQRhOpwqV0DI4\\\\\\\/Z+agAWdZ2el\\\\\\\/KsxVTdDLKD5lsLPIa1uGQelfXZtP0KdQEsEKXPmJb\\\\\\\/kRZtCAbZdDXA=\\\u0022,\\\u0022iv\\\u0022:\\\u0022hj\\\\\\\/rQXmNarwR8r2is\\\\\\\/rB8g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225ff2958b5d418c37fe051ef099d850dc72c02d67b4cce49fd71b391e9ef93290\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-484251 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1569" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-h2AbqE2CJDF_iXW_fKwd8zcp2C4AKlljAxeIFbIXBKo" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1569" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-RNtYrjm0osVKNKb8Tt3nTLhK4Ky3qQzOYYTE9g5euXg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/5/1563" title="From Pine Cones to Read Clouds: Rescaffolding the Megagenome of Sugar Pine (Pinus lambertiana)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/5/1577" title="Distinct Patterns of Gene Gain and Loss: Diverse Evolutionary Modes of NBS-Encoding Genes in Three Solanaceae Crop Species" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484251" id="node484251" data-pisa="ggg;7/5/1569" data-pisa-master="ggg;g3.117.039693" data-apath="/ggg/7/5/1569.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">asymptoticMK: A Web-Based Tool for the Asymptotic McDonald–Kreitman Test</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-1874-8327" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Benjamin C. Haller</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8453-9377" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Philipp W. Messer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1569-1575;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.039693 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484251" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Benjamin C. Haller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Statistics and Computational Biology, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benjamin%2BC.%2BHaller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haller%20BC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABenjamin%2BC.%2BHaller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1874-8327" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Benjamin C. Haller</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Philipp W. Messer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Statistics and Computational Biology, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philipp%2BW.%2BMesser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Messer%20PW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APhilipp%2BW.%2BMesser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-8453-9377" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Philipp W. Messer</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:messer@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">messer@cornell.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/5/1569" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:484251" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:484251/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1569.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:484251" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:484251/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1569.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:484251" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:484251/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/5/1569.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:484251" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:484251/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The McDonald–Kreitman (MK) test is a widely used method for quantifying the role of positive selection in molecular evolution. One key shortcoming of this test lies in its sensitivity to the presence of slightly deleterious mutations, which can severely bias its estimates. An asymptotic version of the MK test was recently introduced that addresses this problem by evaluating polymorphism levels for different mutation frequencies separately, and then extrapolating a function fitted to that data. Here, we present asymptoticMK, a web-based implementation of this asymptotic MK test. Our web service provides a simple R-based interface into which the user can upload the required data (polymorphism and divergence data for the genomic test region and a neutrally evolving reference region). The web service then analyzes the data and provides plots of the test results. This service is free to use, open-source, and available at <a href="http://benhaller.com/messerlab/asymptoticMK.html">http://benhaller.com/messerlab/asymptoticMK.html</a>. We provide results from simulations to illustrate the performance and robustness of the asymptoticMK test under a wide range of model parameters.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/molecular-evolution" class="hw-term hw-article-keyword hw-article-keyword-molecular-evolution" rel="nofollow">molecular evolution</a></li><li class="kwd"><a href="/keyword/positive-selection" class="hw-term hw-article-keyword hw-article-keyword-positive-selection" rel="nofollow">positive selection</a></li><li class="kwd"><a href="/keyword/web-service" class="hw-term hw-article-keyword hw-article-keyword-web-service" rel="nofollow">web service</a></li></ul><p id="p-3">The extent to which molecular evolution is driven by positive selection, rather than neutral evolutionary processes such as random genetic drift, is one of the central questions of modern evolutionary biology. This question can be studied quantitatively by estimating the parameter <em>α</em>, which specifies the fraction of nucleotide substitutions in a given genomic region that were driven to fixation by positive selection (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Eyre-Walker 2006</a>). Values of <em>α</em> close to one indicate that most substitutions in the region were indeed the result of positive selection, whereas values close to zero indicate neutral evolution.</p><p id="p-4">One of the most widely used approaches for inferring <em>α</em> from polymorphism and divergence data is the McDonald-Kreitman (MK) test (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">McDonald and Kreitman 1991</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Eyre-Walker 2006</a>), which compares levels of divergence between a genomic test region and a neutrally evolving reference region with the levels of polymorphism in the two regions. Early applications of the MK test typically focused on nonsynonymous sites in protein-coding regions as the test region, while synonymous sites were used as the neutral reference. However, the approach can also be applied to arbitrary genomic compartments or classes of mutations (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andolfatto 2005</a>).</p><p id="p-5">The original MK test makes several critical assumptions about the nature of the evolutionary process. First, it assumes that the positively selected mutations that ultimately contribute to divergence in the test region go to fixation quickly, such that they do not contribute noticeably to polymorphism levels. Second, it assumes that deleterious mutations in the test region are sufficiently deleterious to be lost quickly, such that they contribute to neither polymorphism nor divergence. Finally, neutral mutations in the test region are assumed to be subject to drift similar to the mutations in the neutral reference region and can therefore contribute to both polymorphism and divergence. Under these assumptions, it holds that<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1569/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1569/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <em>d</em> and <em>d</em><sub>0</sub> are substitution rates per site in the test region and neutral reference region, respectively, while <em>p</em> and <em>p</em><sub>0</sub> specify the respective levels of polymorphism per site in the two regions (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Eyre-Walker 2006</a>). Note that if polymorphism and divergence levels are estimated over the same region, the total number of sites cancels out in the ratios <em>p</em>/<em>d</em> and <em>d</em><sub>0</sub>/<em>p</em><sub>0</sub>, and one may then simply use the actual counts of observed substitutions (<em>D</em> and <em>D</em><sub>0</sub>) and polymorphic sites (<em>P</em> and <em>P</em><sub>0</sub>) instead of rates per site (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Eyre-Walker 2006</a>).</p><p id="p-6">With the growing availability of genome-level polymorphism and divergence data sets, the MK test has become a popular method for inferring positive selection in various organisms (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fay 2011</a>). Several software tools and web services with implementations of the test have also been developed (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Egea <em>et al.</em> 2008</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Librado and Rozas 2009</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Eyre-Walker and Keightley 2009</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Stoletzki and Eyre-Walker 2011</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Vos <em>et al.</em> 2013</a>). The estimates of <em>α</em> obtained in these studies range from as high as ∼0.5 for nonsynonymous substitutions in <em>Drosophila</em> (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Sella <em>et al.</em> 2009</a>), to close to zero in organisms such as yeast (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Elyashiv <em>et al.</em> 2010</a>) or many plants (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gossmann <em>et al.</em> 2010</a>). Indeed, estimates of <em>α</em> obtained from Equation (1) are often negative, indicating that at least some of the assumptions of the test were likely not met (since negative values of <em>α</em> have no biological meaning - estimates of <em>α</em> may be negative, but the true value cannot be).</p><p id="p-7">One major problem with the original MK test lies in its assumption that deleterious mutations do not contribute to polymorphism in the test region. This stands in contrast to the frequent observation of weakly deleterious mutations in many organisms, and the fact that such mutations can substantially affect the site frequency spectrum (SFS) of polymorphisms in functional genomic regions (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bustamante <em>et al.</em> 2005</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Eyre-Walker <em>et al.</em> 2006</a>). In the presence of weakly deleterious mutations, the observed level of polymorphism in the test region (<em>p</em>) in Equation (1) will overestimate the rate at which neutral polymorphisms are expected to go to fixation in this region, which will bias estimates of <em>α</em> downward (providing one possible explanation for the frequent observation of negative <em>α</em> estimates).</p><p id="p-8">As one strategy to address this problem, it has been proposed to only consider polymorphisms for which the derived allele is above a certain threshold frequency when estimating <em>p</em> and <em>p</em><sub>0</sub> (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Charlesworth and Eyre-Walker 2008</a>). This is because the fraction of weakly deleterious mutations among all polymorphisms should be lower for higher derived- allele frequencies. Ideally, one would wish to set this cutoff high, to minimize the bias due to weakly deleterious mutations; however, the higher this cutoff, the fewer polymorphisms will actually remain in the data set, thus increasing statistical noise. To circumvent this problematic tradeoff, more sophisticated extensions of the original MK test first attempt to infer the actual distribution of fitness effects among new mutations in the test region from the SFS, and then correct fixation probabilities accordingly (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Boyko <em>et al.</em> 2008</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Eyre-Walker and Keightley 2009</a>). Yet these approaches can still suffer from unknown effects of demography or linked selection that are also expected to affect the shape of the SFS. The most sophisticated extensions of the test therefore additionally incorporate basic demographic models to improve their estimates (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Keightley and Eyre-Walker 2007</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Boyko <em>et al.</em> 2008</a>; <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Eyre-Walker and Keightley 2009</a>), which requires additional (and often uncertain) assumptions about the demographic history of the population of interest.</p><p id="p-9">In contrast to such model-based approaches, a considerably simpler, heuristic approach was recently proposed by <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Messer and Petrov (2013)</a>. This approach generalizes the frequency-cutoff approach described above, without the need to discard polymorphism data. Instead of setting a specific frequency cutoff, it separately estimates <em>α</em> for each of a set of discrete mutational frequency classes:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1569/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1569/embed/mml-math-2.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>Here, <em>p</em>(<em>x</em>) and <em>p</em><sub>0</sub>(<em>x</em>) specify the levels of polymorphism per site in the test and reference regions, respectively, considering only those polymorphisms for which the derived allele is present at frequency <em>x</em> in the population (estimated from a population sample, for example). In the presence of deleterious mutations, <em>α</em>(<em>x</em>) will underestimate the true value of <em>α</em> for small <em>x</em>, yet should converge to the correct value as <em>x</em> approaches one (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Messer and Petrov 2013</a>). The asymptotic estimate of <em>α</em> is then obtained by fitting a function <em>α</em><sub>fit</sub>(<em>x</em>) to the empirical <em>α</em>(<em>x</em>) values and extrapolating this function to <em>x</em> = 1:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1569/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1569/embed/mml-math-3.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>One key advantage of this approach is that, because <em>α</em>(<em>x</em>) does not depend on the individual functions <em>p</em>(<em>x</em>) and <em>p</em><sub>0</sub>(<em>x</em>) but only on their ratio, any biases due to demography or linked selection that affect the SFS in the test and reference regions in the same way will effectively cancel out (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Messer and Petrov 2013</a>). Another advantage over model-based approaches is that the asymptotic MK approach is much more computationally efficient, as it requires only fitting a simple curve to the data.</p><p id="p-10">In this paper, we present asymptoticMK, a web-based tool for executing the asymptotic MK test quickly and easily in any web browser. After the necessary values are entered, asymptoticMK generates analyses and plots that are directly usable in publications. It is based internally on R, but no knowledge of R is needed to use it, nor does the user of asymptoticMK need to have R installed on their computer. For those who do wish to run the test themselves in R, the necessary code is freely available online. The asymptoticMK service can also be run in an automated fashion at the command line, for bulk analysis in script-based workflows. Finally, we present results from forward genetic simulations to illustrate the performance and robustness of the asymptotic MK test in various scenarios.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Implementation</h3><p id="p-11">The asymptoticMK web service is implemented in R (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">R Development Core Team 2016</a>). It uses the package FastRWeb (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Urbanek 2008</a>) to parse HTTP requests and generate responses, and uses the package Rserve (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Urbanek 2003</a>) as the lower-level interface that communicates with the web server through the standard CGI mechanism. A version of asymptoticMK that runs in R on the user’s local machine is also provided. Source code and additional resources related to asymptoticMK are posted on GitHub at <a href="https://github.com/MesserLab/asymptoticMK">https://github.com/MesserLab/asymptoticMK</a>.</p></div><div id="sec-3" class="subsection"><h3>Usage</h3><p id="p-12">The web service is free to use, without license restrictions of any kind, and is available at <a href="http://benhaller.com/messerlab/asymptoticMK.html">http://benhaller.com/messerlab/asymptoticMK.html</a>. That URL displays an entry page (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) with an input form in which the user may enter the necessary data for the test: <em>d</em> (the substitution rate in the test region), <em>d</em><sub>0</sub> (the substitution rate in the neutral reference region), and an uploaded file containing tab-delimited rows of data with values for <em>x</em> (the derived allele frequency), <em>p</em>(<em>x</em>) (the polymorphism level in the test region at that frequency), and <em>p</em><sub>0</sub>(<em>x</em>) (the polymorphism level in the neutral reference region at that frequency). A sample polymorphism file is provided on the website. In practice, it is often advisable to combine polymorphism levels into a smaller number of frequency bins, where <em>x</em> then specifies the central frequency of the bin. This is particularly relevant when the data includes frequencies at which no polymorphisms are actually present in the neutral region, in which case <em>α</em>(<em>x</em>) would be undefined for those particular frequencies according to Equation (2). The frequency bins supplied to asymptoticMK do not need to be equally spaced, but to obtain the best possible <em>α</em> estimate it is preferable to have bins providing good coverage across the full frequency spectrum. The input form also allows entry of minimum and maximum values defining a cutoff interval for <em>x</em>, such that the test is run using only the polymorphisms whose frequencies fall within that cutoff interval; this is usually desirable as a means of excluding the lowest- and highest-frequency polymorphisms, where SNP quality issues and polarization errors are generally most pronounced. This frequency cutoff is set to [0.1, 0.9] by default, but should be adjusted as needed.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1569/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A screenshot of the web page for asymptoticMK. After entering values for d and d0, choosing an input file with binned values for x, p, and p0, and choosing the x interval to fit, the user can click the Submit button and asymptoticMK will provide its results in a new browser window or tab." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-700763318" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A screenshot of the web page for asymptoticMK. After entering values for &lt;em&gt;d&lt;/em&gt; and &lt;em&gt;d&lt;/em&gt;&lt;sub&gt;0&lt;/sub&gt;, choosing an input file with binned values for &lt;em&gt;x&lt;/em&gt;, &lt;em&gt;p&lt;/em&gt;, and &lt;em&gt;p&lt;/em&gt;&lt;sub&gt;0&lt;/sub&gt;, and choosing the &lt;em&gt;x&lt;/em&gt; interval to fit, the user can click the Submit button and asymptoticMK will provide its results in a new browser window or tab.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1569/F1.medium.gif" width="440" height="301"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/5/1569/F1.medium.gif" width="440" height="301"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1569/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1569/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484379" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-13" class="first-child">A screenshot of the web page for asymptoticMK. After entering values for <em>d</em> and <em>d</em><sub>0</sub>, choosing an input file with binned values for <em>x</em>, <em>p</em>, and <em>p</em><sub>0</sub>, and choosing the <em>x</em> interval to fit, the user can click the Submit button and asymptoticMK will provide its results in a new browser window or tab.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">Upon submission of the web form, asymptoticMK conducts its analysis and then opens a results page in a new browser tab, presenting a summary of the input data and the results from the analysis. The first plot on this results page shows binned polymorphism counts, <em>p</em><sub>0</sub>(<em>x</em>) and <em>p</em>(<em>x</em>), for the submitted data; the second plot shows that same data normalized (<em>i.e.</em>, the normalized SFS in the test and reference regions). A third plot shows the calculated empirical <em>α</em>(<em>x</em>) as a function of <em>x</em>, estimated from the input data according to Equation (2). The fourth plot shows the same <em>α</em>(<em>x</em>) data, with the best-fitting model and the asymptotic estimate of <em>α</em> from Equation (3) superimposed upon the data.</p><p id="p-15">Below these plots, the results of the analysis are presented in two tables. The first table provides the coefficients <em>a</em>, <em>b</em>, and (for exponential fits) <em>c</em> of the model yielding the best fit to the data. The second table provides the estimated <em>α</em><sub>asymptotic</sub> according to Equation (3), and the upper and lower limits of the 95% C.I. around that estimate, as well as the estimated <em>α</em> from the original nonasymptotic MK test (<em>α</em><sub>original</sub>) for comparison (also estimated from all polymorphisms falling within the frequency cutoff interval specified on the input page).</p><p id="p-16">For purposes of automation, the asymptoticMK web service can also be run at the command line using the Linux/Unix curl command. For example, the command</p><blockquote id="disp-quote-1" class="disp-quote"><p id="p-17"><kbd>curl -F”d = 593” -F”d0 = 930” -F”xlow = 0.1” -F”xhigh = 0.9” -F”datafile=@polymorphisms.txt” -F”reply = table” -o “MK_table.txt” <a href="http://benhaller.com/cgi-bin/R/asymptoticMK_run.html">http://benhaller.com/cgi-bin/R/asymptoticMK_run.html</a></kbd></p></blockquote><p id="p-18">would run asymptoticMK with the given values of <em>d</em> and <em>d</em><sub>0</sub>, the given <em>x</em> cutoff interval, and polymorphism data uploaded from the local file polymorphisms.txt, and would output a simple table of results to the file MK_table.txt. Further documentation on the use of this feature is provided on the asymptoticMK web page.</p><p id="p-19">Finally, it is also possible to run asymptoticMK in R on the user’s local machine. The R code for doing so can be found on asymptoticMK’s GitHub repository. In addition to allowing the user to modify asymptoticMK’s analysis as desired, this option also allows PDF plots to be created, rather than the PNG plots provided by the web-based service.</p></div><div id="sec-4" class="subsection"><h3>Fitting and analysis procedure</h3><p id="p-20">The asymptotic MK test first involves calculating values of <em>α</em>(<em>x</em>) by applying Equation (2) to each frequency bin provided, as described by <a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Messer and Petrov (2013)</a>. The next step involves fitting a function <em>α</em><sub>fit</sub>(<em>x</em>) to these empirical <em>α</em>(<em>x</em>) values. For greater robustness, asymptoticMK fits two functions to the data. The first function is exponential, of the form <em>α</em><sub>fit</sub>(<em>x</em>) = <em>a</em> + <em>b</em> exp(−<em>cx</em>), and is fitted using the nls2() function, from the R package nls2 (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Grothendieck 2013</a>). This fit is done in two steps. First, a brute-force scan for the closest fit is conducted across the likely portion of the three-dimensional parameter space defined by <em>a</em>, <em>b</em>, and <em>c</em>, by exhaustive search. This supplies reasonably good starting values for the second step, which refines those starting values using standard nonlinear least-squares regression. While this two-step procedure generally works well, it can fail to converge if the data are not exponential in form.</p><p id="p-21">To address this possibility of nonconvergence of the exponential fit, asymptoticMK also fits a linear function of the form <em>α</em><sub>fit</sub>(<em>x</em>) = <em>a</em> + <em>bx</em>, with the lm() function that is part of the stats package included in R. This fit always converges, and thus provides a backstop that allows the test to complete even when given irregular or extremely noisy data; however, it is always recommended that the results of the analysis be inspected visually to confirm that they are in fact meaningful.</p><p id="p-22">Once these two models have been fitted, asymptoticMK chooses which model will be used for the remainder of the analysis. If the exponential fit failed to converge, then the linear model is chosen; if both fits succeeded, then the better model is chosen using the Akaike Information Criterion (AIC). Occasionally, in pathological cases, the exponential fit will have the better AIC but will have extremely large coefficient SEs; in this case, the linear fit is chosen since predictions from the exponential model would be effectively worthless.</p><p id="p-23">The chosen model is then used to provide an estimate of the value of <em>α</em><sub>asymptotic</sub> according to Equation (3), by evaluating the fitted function <em>α</em><sub>fit</sub>(<em>x</em>) at <em>x</em> = 1; this is the primary result of the test, and provides the test’s estimate of the true value of <em>α</em> within the test region. A 95% C.I. around this estimate is also calculated. For the exponential model, this is done using Monte Carlo simulation based upon the fitted model, using the predictNLS() function published online by <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Spiess (2013)</a>; for the linear model, it is done using the standard R function predict().</p></div><div id="sec-5" class="subsection"><h3>Test data sets</h3><p id="p-24">To provide a test of asymptoticMK using empirical data, we used the same <em>Drosophila melanogaster</em> data set that <a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Messer and Petrov (2013)</a> used in their Figure 3C. This data set consists of SNPs obtained from the genome sequences of 162 inbred fly lines generated by the <em>Drosophila</em> genetic reference panel (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Mackay <em>et al.</em> 2012</a>). Divergence data were obtained from genome alignments between <em>D. melanogaster</em> and <em>D. simulans</em>, extracted from the 12 <em>Drosophila</em> genomes data (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Clark <em>et al.</em> 2007</a>). The test data in the asymptoticMK analysis (<em>d</em> and <em>p</em>) are genome-wide nonsynonymous mutations, while synonymous sites were used as the neutral reference (<em>d</em><sub>0</sub> and <em>p</em><sub>0</sub>). The polymorphism data are available online at asymptoticMK’s GitHub repository, with associated values <em>d</em> = 59570 and <em>d</em><sub>0</sub> = 159058. The default frequency cutoff interval of [0.1, 0.9] was used in the analysis of this data set with asymptoticMK.</p><p id="p-25">We also tested asymptoticMK on simulated data, using the forward genetic simulation framework SLiM 2 (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Haller and Messer 2017</a>). A population of 1000 diploid individuals was simulated to evolve in a total of 13 different scenarios, with 20 replicates for each scenario. Simulation runs depended upon six free parameters (<em>T</em>, <em>L</em>, <em>μ</em>, <em>r</em><sub>b</sub>, <em>s</em><sub>d</sub>, and <em>s</em><sub>b</sub>) as described hereafter. After an initial burn-in period of 10,000 generations to equilibrate the model, runs executed for <em>T</em> additional generations. The simulated chromosome was <em>L</em> base pairs long. Nucleotide mutations occurred uniformly at a rate of <em>μ</em> per base per generation, and recombination occurred uniformly at a rate of 10<sup>−7</sup> per base per generation. Each new mutation was either of neutral type “m1” (relative proportion of 0.5 of all new mutations), of functional nonbeneficial type “m2” (relative proportion of 0.5 of all new mutations), or of functional beneficial type “m3” (relative proportion of <em>r</em><sub>b</sub> of all new mutations); these relative proportions were automatically rescaled by SLiM to be absolute proportions. The neutral m1 mutations always had a selection coefficient of <em>s</em> = 0.0; the selection coefficients of m2 mutations were drawn from a gamma distribution with a mean of <em>s</em><sub>d</sub> and a shape parameter of 0.2; and m3 mutations always had a selection coefficient of <em>s</em><sub>b</sub>. Fitness effects were assumed to be codominant. Every 500 generations after the burn-in period, all polymorphisms were recorded in the population by dividing them according to their frequency into 50 equal-width frequency bins, and then adding them to an ongoing binned tabulation. The SLiM configuration script used for these simulations is provided on asymptoticMK’s GitHub repository.</p><p id="p-26">The “baseline” parameterization of this model utilized parameter values of <em>L</em> = 10<sup>7</sup>, <em>μ</em> = 10<sup>−9</sup>, <em>r</em><sub>b</sub> = 0.0005, <em>s</em><sub>b</sub> = 0.1, <em>s</em><sub>d</sub> = −0.02, and <em>T</em> = 2 × 10<sup>5</sup>. The other 12 parameterizations involved either a “high” or a “low” value of one of the six parameters, replacing the “central” value used in the baseline scenario: <em>L</em> = 10<sup>8</sup> or 10<sup>6</sup>, <em>μ</em> = 10<sup>−8</sup> or 10<sup>−10</sup>, <em>r</em><sub>b</sub> = 0.001 or 0.0001, <em>s</em><sub>b</sub> = 0.2 or 0.02, <em>s</em><sub>d</sub> = −0.2 or −0.002, and <em>T</em> = 2 × 10<sup>6</sup> or 2 × 10<sup>4</sup>. At the end of each model run, we obtained binned values for <em>p</em>(<em>x</em>) and <em>p</em><sub>0</sub>(<em>x</em>), where <em>p</em><sub>0</sub> was estimated from all polymorphisms involving mutations of type m1, while <em>p</em> was estimated from the combined mutations of types m2 and m3. Values for <em>d</em> and <em>d</em><sub>0</sub> were obtained from the set of mutations fixed during the simulation; as with <em>p</em><sub>0</sub> and <em>p</em>, <em>d</em><sub>0</sub> was estimated from all mutations of type m1, while <em>d</em> was estimated from the combined mutations of types m2 and m3. These values, output by the model, were used in asymptoticMK with the default <em>x</em> cutoff interval of [0.1, 0.9] to calculate an <em>α</em> estimate. The <em>α</em> estimate from the original MK test was also calculated using the data within the same interval. Finally, the true value of <em>α</em> was estimated from the simulation run as the fraction <em>d</em><sub>3</sub> / (<em>d</em><sub>2</sub> + <em>d</em><sub>3</sub>), where <em>d</em><sub>2</sub> is the number of m2 mutations fixed and <em>d</em><sub>3</sub> is the number of m3 mutations fixed. This value provides a metric for the accuracy of the <em>α</em> estimates– a benefit of using simulated data, where the true <em>α</em> can be calculated.</p><p id="p-27">From this raw data provided by each set of 20 replicates for a given parameterization, summary statistics for that parameterization were computed. In particular, we calculated (i) the mean and SD of the true <em>α</em> values, (ii) the mean and SD of the asymptoticMK <em>α</em> estimates, (iii) the mean of the absolute differences between the true <em>α</em> and the asymptoticMK estimate (<em>i.e.</em>, the mean estimation error for asymptoticMK), (iv) the mean and SD of the estimates of <em>α</em> using the original nonasymptotic MK test, (v) the mean of the absolute differences between the true <em>α</em> and the original MK test estimate (<em>i.e.</em>, the mean estimation error for the original MK test), and (vi) the fraction of the 20 replicates for which asymptoticMK chose an exponential (as opposed to linear) fit (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484284/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484284/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Results from asymptoticMK for simulation runs conducted with SLiM 2</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-29">The asymptoticMK web service can be used online at <a href="http://benhaller.com/messerlab/asymptoticMK.html">http://benhaller.com/messerlab/asymptoticMK.html</a>. The R source code for asymptoticMK, the SLiM model for the simulations conducted, the <em>Drosophila</em> data set analyzed, and other related files are available on GitHub at <a href="https://github.com/MesserLab/asymptoticMK">https://github.com/MesserLab/asymptoticMK</a>. </p></div></div><div class="section results-discussion" id="sec-7"><h2 class="">Results and Discussion</h2><p id="p-30">Results from our test of asymptoticMK with the empirical <em>D. melanogaster</em> data set are shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2, A and B</a>. The fitted exponential function is: <em>α</em><sub>fit</sub>(<em>x</em>) = 0.585 − 0.622 exp(−3.80<em>x</em>). The asymptotic MK estimate provided by this model is <em>α</em><sub>asymptotic</sub> = 0.571. These results match those obtained by <a id="xref-ref-19-6" class="xref-bibr" href="#ref-19">Messer and Petrov (2013)</a> using the same data set (their Figure 3C), as expected. The estimate provided by the original MK test is <em>α</em><sub>original</sub> = 0.407, by comparison (shown in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1569/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results from asymptoticMK for three test data sets. (A) Normalized site frequency spectrum (SFS) for the Drosophila data set used in Messer and Petrov (2013). Points show normalized binned polymorphism frequencies for the neutral region (black) and the test region (red). (B) Result of asymptoticMK&#x2019;s analysis of that data set. The two vertical blue lines show the limits of the frequency cutoff interval used for fitting. Points indicate binned values of &#x3B1;(x), estimated according to Equation 2; points are gray if they are outside the cutoff interval (and thus not used in fitting). The solid red curve shows the fitted &#x3B1;fit(x) (here, exponential). The dashed red line shows the estimate of &#x3B1;asymptotic, obtained from the fitted function according to Equation 3. The gray band indicates the 95% C.I. around this &#x3B1;asymptotic estimate. The dotted gray line shows the estimate of &#x3B1;original, obtained from the original (nonasymptotic) McDonald&#x2013;Kreitman (MK) test, for comparison (also calculated using only the data within the cutoff interval). (C) and (D) show corresponding results from one SLiM simulation run, and (E) and (F) show results from another SLiM simulation run; in each case, the first panel shows the result of an automated fit using asymptoticMK, whereas the second shows the improvement after hand tailoring of the fit (see Results and Discussion). Note that in all four cases, the linear fit was deemed more appropriate by asymptoticMK. The solid green horizontal lines, finally, show the true value of &#x3B1; in the simulation runs for comparison." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-700763318" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Results from asymptoticMK for three test data sets. (A) Normalized site frequency spectrum (SFS) for the &lt;em&gt;Drosophila&lt;/em&gt; data set used in Messer and Petrov (2013). Points show normalized binned polymorphism frequencies for the neutral region (black) and the test region (red). (B) Result of asymptoticMK&#x2019;s analysis of that data set. The two vertical blue lines show the limits of the frequency cutoff interval used for fitting. Points indicate binned values of &lt;em&gt;&#x3B1;&lt;/em&gt;(&lt;em&gt;x&lt;/em&gt;), estimated according to Equation 2; points are gray if they are outside the cutoff interval (and thus not used in fitting). The solid red curve shows the fitted &lt;em&gt;&#x3B1;&lt;/em&gt;&lt;sub&gt;fit&lt;/sub&gt;(&lt;em&gt;x&lt;/em&gt;) (here, exponential). The dashed red line shows the estimate of &lt;em&gt;&#x3B1;&lt;/em&gt;&lt;sub&gt;asymptotic&lt;/sub&gt;, obtained from the fitted function according to Equation 3. The gray band indicates the 95% C.I. around this &lt;em&gt;&#x3B1;&lt;/em&gt;&lt;sub&gt;asymptotic&lt;/sub&gt; estimate. The dotted gray line shows the estimate of &lt;em&gt;&#x3B1;&lt;/em&gt;&lt;sub&gt;original&lt;/sub&gt;, obtained from the original (nonasymptotic) McDonald&#x2013;Kreitman (MK) test, for comparison (also calculated using only the data within the cutoff interval). (C) and (D) show corresponding results from one SLiM simulation run, and (E) and (F) show results from another SLiM simulation run; in each case, the first panel shows the result of an automated fit using asymptoticMK, whereas the second shows the improvement after hand tailoring of the fit (see &lt;em&gt;Results and Discussion&lt;/em&gt;). Note that in all four cases, the linear fit was deemed more appropriate by asymptoticMK. The solid green horizontal lines, finally, show the true value of &lt;em&gt;&#x3B1;&lt;/em&gt; in the simulation runs for comparison.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1569/F2.medium.gif" width="284" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/5/1569/F2.medium.gif" width="284" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1569/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1569/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484365" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-31" class="first-child">Results from asymptoticMK for three test data sets. (A) Normalized site frequency spectrum (SFS) for the <em>Drosophila</em> data set used in <a id="xref-ref-19-7" class="xref-bibr" href="#ref-19">Messer and Petrov (2013)</a>. Points show normalized binned polymorphism frequencies for the neutral region (black) and the test region (red). (B) Result of asymptoticMK’s analysis of that data set. The two vertical blue lines show the limits of the frequency cutoff interval used for fitting. Points indicate binned values of <em>α</em>(<em>x</em>), estimated according to Equation 2; points are gray if they are outside the cutoff interval (and thus not used in fitting). The solid red curve shows the fitted <em>α</em><sub>fit</sub>(<em>x</em>) (here, exponential). The dashed red line shows the estimate of <em>α</em><sub>asymptotic</sub>, obtained from the fitted function according to Equation 3. The gray band indicates the 95% C.I. around this <em>α</em><sub>asymptotic</sub> estimate. The dotted gray line shows the estimate of <em>α</em><sub>original</sub>, obtained from the original (nonasymptotic) McDonald–Kreitman (MK) test, for comparison (also calculated using only the data within the cutoff interval). (C) and (D) show corresponding results from one SLiM simulation run, and (E) and (F) show results from another SLiM simulation run; in each case, the first panel shows the result of an automated fit using asymptoticMK, whereas the second shows the improvement after hand tailoring of the fit (see <em>Results and Discussion</em>). Note that in all four cases, the linear fit was deemed more appropriate by asymptoticMK. The solid green horizontal lines, finally, show the true value of <em>α</em> in the simulation runs for comparison.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">The results from the analysis of the SLiM simulations are shown in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>. In 12 of the 13 parameterizations, the mean estimation error of asymptoticMK was markedly lower than that of the original MK test; in the other parameterization (<em>T</em> = 2 × 10<sup>4</sup>) the tests performed similarly (mean estimation errors of 0.126 and 0.120). For three of the 13 parameterizations (<em>L</em> = 10<sup>6</sup>, <em>μ</em> = 10<sup>−10</sup>, and <em>T</em> = 2 × 10<sup>4</sup>), however, the mean estimation error of asymptoticMK was &gt; 0.1, indicating that <em>α</em> estimates were relatively inaccurate for those simulations. These three parameterizations involved a shorter chromosome, a lower mutation rate, or a shorter duration, and thus all provided ∼10 times less polymorphism data upon which to base estimations than our baseline scenario. Accordingly, parameterizations that provided more polymorphism data (<em>L</em> = 10<sup>8</sup>, <em>μ</em> = 10<sup>−8</sup>, and <em>T</em> = 2 × 10<sup>6</sup>) provided more accurate <em>α</em> estimates (mean estimation errors below 0.03). This pattern was weak or absent for the original MK test; even for the high-data parameterizations the original MK test always showed a mean estimation error &gt; 0.1, and its mean estimation error for the high-data <em>L</em> = 10<sup>8</sup> case was actually substantially higher than for the low-data <em>L</em> = 10<sup>6</sup> case (0.151 <em>vs.</em> 0.120). This is consistent with the fact that the original MK test systematically underestimates <em>α</em> in the presence of deleterious mutations (as discussed in the opening paragraphs). The asymptotic MK test may still have a tendency toward underestimation as well, but errors are much smaller.</p><p id="p-33">Another noteworthy observation is that in the high-data parameterizations the exponential fit was chosen by asymptoticMK in 100% of cases, whereas in the low-data parameterizations the linear fit was chosen a majority of the time (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). It is not the case that the linear model always produces a poor <em>α</em> estimate; we observed many runs where the linear fit performed well. However, it may indicate that a poor cutoff interval was chosen, that the binning of the polymorphism data ought to be done differently, or that the data are simply too noisy. We suggest that the result of the asymptotic MK test should always be inspected visually to verify that the fit is reasonable and that appropriate cutoff intervals and bin sizes were used.</p><p id="p-34">To illustrate how such manual inspection can help improve estimates, we examine two of the simulation runs from <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a> in more detail. <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a> shows the automated fit for one of the simulations in the low-data <em>L</em> = 10<sup>6</sup> scenario. A linear fit function produced an asymptotic <em>α</em> estimate of 0.0313, which is quite distant from the true value of 0.3462. The binned polymorphism data within the cutoff interval of [0.1, 0.9] is rather flat, but the data appears reasonable across the whole frequency spectrum in this case, and the upward trend of the data is much more visible outside of the cutoff interval used. Changing the cutoff interval to [0.0, 1.0] produces the fit shown in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2D</a> with an asymptotic <em>α</em> estimate of 0.2829, much closer to the true value. <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2E</a> shows the automated fit for another <em>L</em> = 10<sup>6</sup> scenario run. This fit also used a linear fit function, producing an asymptotic <em>α</em> estimate of 0.0103 compared to the true value of 0.2462. Here, the data are very noisy, which could be an indication that more bins have been used than can be robustly supported by the data. Rebinning the polymorphism data into half as many bins provides a less noisy data set that results in a much better fit (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2F</a>), with an <em>α</em> estimate of 0.1813– again, a substantial improvement. These examples illustrate that automated fits can be particularly problematic in low-data situations such as the <em>L</em> = 10<sup>6</sup> scenario, but that hand inspection and tailoring of the fitting process can sometimes improve the result noticeably.</p><div id="sec-8" class="subsection"><h3>Conclusions</h3><p id="p-35">In this paper, we presented asymptoticMK, a new web-based tool for executing the asymptotic MK test. To demonstrate its functionality, we analyzed both empirical and simulation-generated data sets. Our results illustrate the greater power of the asymptotic MK test to estimate the true value of <em>α</em>, compared to the original nonasymptotic test. However, our results also underline the need for a large data set to obtain reasonably accurate results from the asymptotic test; estimates of <em>α</em> from a single gene, or from a system with a very short divergence time, are unlikely to be meaningful. In addition, visual inspection of the quality of the fit used to estimate <em>α</em> is necessary for accuracy. With attention to these caveats, the asymptoticMK service presented here allows the user to obtain <em>α</em> estimates quickly and easily through any web browser, or using R on any machine.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-36">The authors thank G. Grothendieck, J. Horner, and S. Urbanek for their contributions to the R packages used here. We also thank S. Urbanek for his unstinting help with the installation and use of FastRWeb, A.-N. Spiess for his predictNLS() function, and most of all the R Core Team for R itself. asymptoticMK is only possible because of the free software developed by these and other contributors. We also thank Dmitri Petrov for his important role in the original development of the asymptotic MK test, and the editor and two anonymous reviewers for their helpful comments. This work was supported by funds from the College of Agriculture and Life Sciences at Cornell University to P.W.M. </p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-37"><em>Communicating editor: J. C. Fay</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-01-19"><span class="received-label">Received </span>January 19, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-03-21"><span class="accepted-label">Accepted </span>March 21, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Haller and Messer</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1569.1" data-doi="10.1038/nature04107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Adaptive evolution of non-coding DNA in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">437</span>: <span class="cit-fpage">1149</span>–<span class="cit-lpage">1152</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAndolfatto%26rft.auinit1%253DP.%26rft.volume%253D437%26rft.issue%253D7062%26rft.spage%253D1149%26rft.epage%253D1152%26rft.atitle%253DAdaptive%2Bevolution%2Bof%2Bnon-coding%2BDNA%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04107%26rft_id%253Dinfo%253Apmid%252F16237443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04107&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16237443&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232660500043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1569.2" data-doi="10.1371/journal.pgen.1000083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boyko</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Indap</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Degenhardt</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Assessing the evolutionary impact of amino acid mutations in the human genome.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000083</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBoyko%26rft.auinit1%253DA.%2BR.%26rft.volume%253D4%26rft.issue%253D5%26rft.spage%253De1000083%26rft.epage%253De1000083%26rft.atitle%253DAssessing%2Bthe%2Bevolutionary%2Bimpact%2Bof%2Bamino%2Bacid%2Bmutations%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000083%26rft_id%253Dinfo%253Apmid%252F18516229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18516229&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1569.3" data-doi="10.1038/nature04240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fledel-Alon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubisz</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Natural selection on protein-coding genes in the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">437</span>: <span class="cit-fpage">1153</span>–<span class="cit-lpage">1157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBustamante%26rft.auinit1%253DC.%2BD.%26rft.volume%253D437%26rft.issue%253D7062%26rft.spage%253D1153%26rft.epage%253D1157%26rft.atitle%253DNatural%2Bselection%2Bon%2Bprotein-coding%2Bgenes%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04240%26rft_id%253Dinfo%253Apmid%252F16237444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04240&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16237444&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232660500044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1569.4" data-doi="10.1093/molbev/msn005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The McDonald-Kreitman test and slightly deleterious mutations.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1007</span>–<span class="cit-lpage">1015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn005%26rft_id%253Dinfo%253Apmid%252F18195052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI1LzYvMTAwNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1569.5" data-doi="10.1038/nature06341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergman</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Evolution of genes and genomes on the <em>Drosophila</em> phylogeny.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">450</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DClark%26rft.auinit1%253DA.%2BG.%26rft.volume%253D450%26rft.issue%253D7167%26rft.spage%253D203%26rft.epage%253D218%26rft.atitle%253DEvolution%2Bof%2Bgenes%2Band%2Bgenomes%2Bon%2Bthe%2BDrosophila%2Bphylogeny.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06341%26rft_id%253Dinfo%253Apmid%252F17994087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06341&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17994087&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250746200042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1569.6" data-doi="10.1093/nar/gkn337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Egea</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casillas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbadilla</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Standard and generalized McDonald-Kreitman test: a website to detect selection by comparing different classes of DNA sites.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">W157</span>–<span class="cit-lpage">W162</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn337%26rft_id%253Dinfo%253Apmid%252F18515345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzNi9zdXBwbF8yL1cxNTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTU2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1569.7" data-doi="10.1101/gr.108993.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elyashiv</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bullaughey</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sattath</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rinott</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przeworski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Shifts in the intensity of purifying selection: an analysis of genome-wide polymorphism data from two closely related yeast species.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1558</span>–<span class="cit-lpage">1573</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.108993.110%26rft_id%253Dinfo%253Apmid%252F20817943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8xMS8xNTU4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE1NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1569.8" data-doi="10.1016/j.tree.2006.06.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The genomic rate of adaptive evolution.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">569</span>–<span class="cit-lpage">575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DEyre-Walker%26rft.auinit1%253DA.%26rft.volume%253D21%26rft.issue%253D10%26rft.spage%253D569%26rft.epage%253D575%26rft.atitle%253DThe%2Bgenomic%2Brate%2Bof%2Badaptive%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2006.06.015%26rft_id%253Dinfo%253Apmid%252F16820244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2006.06.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16820244&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241256900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1569.9" data-doi="10.1093/molbev/msp119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Estimating the rate of adaptive molecular evolution in the presence of slightly deleterious mutations and population size change.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2097</span>–<span class="cit-lpage">2108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp119%26rft_id%253Dinfo%253Apmid%252F19535738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI2LzkvMjA5NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1569.10" data-doi="10.1534/genetics.106.057570"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolfit</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phelps</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The distribution of fitness effects of new deleterious amino acid mutations in humans.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">891</span>–<span class="cit-lpage">900</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.057570%26rft_id%253Dinfo%253Apmid%252F16547091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTczLzIvODkxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE1NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1569.11" data-doi="10.1016/j.tig.2011.06.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Weighing the evidence for adaptation at the molecular level.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DFay%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D27%26rft.issue%253D9%26rft.spage%253D343%26rft.epage%253D349%26rft.atitle%253DWeighing%2Bthe%2Bevidence%2Bfor%2Badaptation%2Bat%2Bthe%2Bmolecular%2Blevel.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2011.06.003%26rft_id%253Dinfo%253Apmid%252F21775012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2011.06.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21775012&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294743200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1569.12" data-doi="10.1093/molbev/msq079"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gossmann</span>  <span class="cit-name-given-names">T. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">B.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Windsor</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell-Olds</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dixon</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome wide analyses reveal little evidence for adaptive evolution in many plant species.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1822</span>–<span class="cit-lpage">1832</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq079%26rft_id%253Dinfo%253Apmid%252F20299543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI3LzgvMTgyMiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-web" id="cit-7.5.1569.13"><div class="cit-metadata unstructured">Grothendieck, G., 2013 nls2: non-linear regression with brute force. Available at: <a href="https://CRAN.R-project.org/package=nls2">https://CRAN.R-project.org/package=nls2</a>. Accessed: December 14, 2016.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1569.14" data-doi="10.1093/molbev/msw211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haller</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messer</span>  <span class="cit-name-given-names">P. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">SLiM 2: flexible, interactive forward genetic simulations.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">230</span>–<span class="cit-lpage">240</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw211%26rft_id%253Dinfo%253Apmid%252F27702775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjM0LzEvMjMwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE1NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1569.15" data-doi="10.1534/genetics.107.080663"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Joint inference of the distribution of fitness effects of deleterious mutations and population demography based on nucleotide polymorphism frequencies.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2251</span>–<span class="cit-lpage">2261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKeightley%26rft.auinit1%253DP.%2BD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2251%26rft.epage%253D2261%26rft.atitle%253DJoint%2BInference%2Bof%2Bthe%2BDistribution%2Bof%2BFitness%2BEffects%2Bof%2BDeleterious%2BMutations%2Band%2BPopulation%2BDemography%2BBased%2Bon%2BNucleotide%2BPolymorphism%2BFrequencies%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080663%26rft_id%253Dinfo%253Apmid%252F18073430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIyNTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTU2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1569.16" data-doi="10.1093/bioinformatics/btp187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Librado</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rozas</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">DnaSP v5: a software for comprehensive analysis of DNA polymorphism data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1451</span>–<span class="cit-lpage">1452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp187%26rft_id%253Dinfo%253Apmid%252F19346325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTEvMTQ1MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1569.17" data-doi="10.1038/nature10811"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbadilla</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The <em>Drosophila melanogaster</em> genetic reference panel.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">482</span>: <span class="cit-fpage">173</span>–<span class="cit-lpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMackay%26rft.auinit1%253DT.%2BF.%26rft.volume%253D482%26rft.issue%253D7384%26rft.spage%253D173%26rft.epage%253D178%26rft.atitle%253DThe%2BDrosophila%2Bmelanogaster%2BGenetic%2BReference%2BPanel.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10811%26rft_id%253Dinfo%253Apmid%252F22318601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10811&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22318601&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299994100029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1569.18" data-doi="10.1038/351652a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreitman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Adaptive protein evolution at the <em>Adh</em> locus in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">351</span>: <span class="cit-fpage">652</span>–<span class="cit-lpage">654</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMcDonald%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D351%26rft.issue%253D6328%26rft.spage%253D652%26rft.epage%253D654%26rft.atitle%253DAdaptive%2Bprotein%2Bevolution%2Bat%2Bthe%2BAdh%2Blocus%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F351652a0%26rft_id%253Dinfo%253Apmid%252F1904993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/351652a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1904993&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FT11200066&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1569.19" data-doi="10.1073/pnas.1220835110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Messer</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Frequent adaptation and the McDonald-Kreitman test.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">8615</span>–<span class="cit-lpage">8620</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1220835110%26rft_id%253Dinfo%253Apmid%252F23650353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzIxLzg2MTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTU2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-7.5.1569.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Development Core Team</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-source"><em>R: A Language and Environment for Statistical Computing</em></span>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1569.21" data-doi="10.1371/journal.pgen.1000495"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sella</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przeworski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Pervasive natural selection in the <em>Drosophila</em> genome?</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSella%26rft.auinit1%253DG.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253De1000495%26rft.epage%253De1000495%26rft.atitle%253DPervasive%2Bnatural%2Bselection%2Bin%2Bthe%2BDrosophila%2Bgenome%253F%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000495%26rft_id%253Dinfo%253Apmid%252F19503600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000495&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19503600&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-web" id="cit-7.5.1569.22"><div class="cit-metadata unstructured">Spiess, A.-N., 2013 predictNLS (Part 1, Monte Carlo simulation): confidence intervals for “nls” models. R-bloggers. Available at: <a href="https://www.r-bloggers.com/predictnls-part-1-monte-carlo-simulation-confidence-intervals-for-nls-models/">https://www.r-bloggers.com/predictnls-part-1-monte-carlo-simulation-confidence-intervals-for-nls-models/</a>. Accessed: December 14, 2016.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1569.23" data-doi="10.1093/molbev/msq249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stoletzki</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Estimation of the neutrality index.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">63</span>–<span class="cit-lpage">70</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq249%26rft_id%253Dinfo%253Apmid%252F20837603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjc6IjI4LzEvNjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTU2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-web" id="cit-7.5.1569.24"><div class="cit-metadata unstructured">Urbanek, S., 2003 Rserve - A fast way to provide R functionality to applications. in <em>Proceedings of the 3rd International Workshop on Distributed Statistical Computing</em>. Available at: <a href="https://rforge.net/Rserve/">https://rforge.net/Rserve/</a>. Accessed: December 14, 2016.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-web" id="cit-7.5.1569.25"><div class="cit-metadata unstructured">Urbanek, S., 2008 FastRWeb: fast interactive web framework for data mining using R, in IASC 2008 World Congress. Available at: <a href="https://rforge.net/FastRWeb/">https://rforge.net/FastRWeb/</a>. Accessed: December 14, 2016.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1569.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vos</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">te Beek</span>  <span class="cit-name-given-names">T. A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Driel</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huynen</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">ODoSE: a webserver for genome-wide calculation of adaptive divergence in prokaryotes.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e62447</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De62447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/5/1569.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/5/1563" title="From Pine Cones to Read Clouds: Rescaffolding the Megagenome of Sugar Pine (Pinus lambertiana)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/5/1577" title="Distinct Patterns of Gene Gain and Loss: Diverse Evolutionary Modes of NBS-Encoding Genes in Three Solanaceae Crop Species" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/5"><span>Volume 7 Issue 5, May 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/5.cover-source.jpg?itok=VWLvGstM" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1569" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-TBK9VPj8U9Z6INCmYZ9nVE31s7eeltW4X8YWBP1ItMs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/5/1569" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/5/1569" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/5/1569" class="active" data-icon-position="" data-hide-link-title="0">asymptoticMK: A Web-Based Tool for the Asymptotic McDonald–Kreitman Test</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/484251" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Yk3kQCkIlPiASTZHmddVp4Xo2Jx8hviJ0Smtxq7VwOU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/5/1569.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/5/1569" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Q6G0l9YUjhwEBLay_cpfaRXkUZmroCsJMOu2v4TeP_Y" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/5/1569" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/5/1569.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484251" id="node484251--2" data-pisa="ggg;7/5/1569" data-pisa-master="ggg;g3.117.039693" data-apath="/ggg/7/5/1569.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">asymptoticMK: A Web-Based Tool for the Asymptotic McDonald–Kreitman Test</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-1874-8327" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Benjamin C. Haller</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8453-9377" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Philipp W. Messer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1569-1575;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.039693 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484251--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Benjamin C. Haller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Statistics and Computational Biology, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benjamin%2BC.%2BHaller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haller%20BC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABenjamin%2BC.%2BHaller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1874-8327" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Benjamin C. Haller</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Philipp W. Messer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Statistics and Computational Biology, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philipp%2BW.%2BMesser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Messer%20PW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APhilipp%2BW.%2BMesser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-8453-9377" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Philipp W. Messer</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:messer@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">messer@cornell.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/5/1569" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/5/1569&amp;title=asymptoticMK%3A%20A%20Web-Based%20Tool%20for%20the%20Asymptotic%20McDonald%E2%80%93Kreitman%20Test" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/5/1569&amp;title=asymptoticMK%3A%20A%20Web-Based%20Tool%20for%20the%20Asymptotic%20McDonald%E2%80%93Kreitman%20Test" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/5/1569&amp;title=asymptoticMK%3A%20A%20Web-Based%20Tool%20for%20the%20Asymptotic%20McDonald%E2%80%93Kreitman%20Test" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/5/1569&amp;text=asymptoticMK%3A%20A%20Web-Based%20Tool%20for%20the%20Asymptotic%20McDonald%E2%80%93Kreitman%20Test" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/5/1569&amp;title=asymptoticMK%3A%20A%20Web-Based%20Tool%20for%20the%20Asymptotic%20McDonald%E2%80%93Kreitman%20Test" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/5/1569&amp;t=asymptoticMK%3A%20A%20Web-Based%20Tool%20for%20the%20Asymptotic%20McDonald%E2%80%93Kreitman%20Test" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/5/1569&amp;title=asymptoticMK%3A%20A%20Web-Based%20Tool%20for%20the%20Asymptotic%20McDonald%E2%80%93Kreitman%20Test" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/5/1569&amp;title=asymptoticMK%3A%20A%20Web-Based%20Tool%20for%20the%20Asymptotic%20McDonald%E2%80%93Kreitman%20Test" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/484251/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484251" id="node484251--3" data-pisa="ggg;7/5/1569" data-pisa-master="ggg;g3.117.039693" data-apath="/ggg/7/5/1569.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">asymptoticMK: A Web-Based Tool for the Asymptotic McDonald–Kreitman Test</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-1874-8327" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Benjamin C. Haller</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8453-9377" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Philipp W. Messer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1569-1575;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.039693 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484251--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Benjamin C. Haller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Statistics and Computational Biology, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benjamin%2BC.%2BHaller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haller%20BC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABenjamin%2BC.%2BHaller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1874-8327" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Benjamin C. Haller</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Philipp W. Messer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Statistics and Computational Biology, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philipp%2BW.%2BMesser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Messer%20PW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APhilipp%2BW.%2BMesser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-8453-9377" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Philipp W. Messer</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:messer@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">messer@cornell.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1569&amp;rft.epage=1575&amp;rft.atitle=asymptoticMK%3A%20A%20Web-Based%20Tool%20for%20the%20Asymptotic%20McDonald%E2%80%93Kreitman%20Test&amp;rft.volume=7&amp;rft.issue=5&amp;rft.date=2017-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Haller%2C+Benjamin+C.&amp;rft.au=Messer%2C+Philipp+W."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/484251/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/484251/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/484251/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/484251/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/484251/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/484251/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/484251/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/484251/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/484251/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/484251/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/484251/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/484251/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/5/1569" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/5/1569.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/5/1569.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/7/5/1569.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
