<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/5/1499" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/5/1499.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/5/1499.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/5/1499.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/5/1499/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/5/1499.atom" />
<meta name="HW.pisa" content="ggg;7/5/1499" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Evaluating Methods of Updating Training Data in Long-Term Genomewide Selection" />
<meta name="DC.Identifier" content="10.1534/g3.117.040550" />
<meta name="DC.Date" content="2017-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Neyhart et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genomewide selection is hailed for its ability to facilitate greater genetic gains per unit time. Over breeding cycles, the requisite linkage disequilibrium (LD) between quantitative trait loci and markers is expected to change as a result of recombination, selection, and drift, leading to a decay in prediction accuracy. Previous research has identified the need to update the training population using data that may capture new LD generated over breeding cycles; however, optimal methods of updating have not been explored. In a barley ( Hordeum vulgare L.) breeding simulation experiment, we examined prediction accuracy and response to selection when updating the training population each cycle with the best predicted lines, the worst predicted lines, both the best and worst predicted lines, random lines, criterion-selected lines, or no lines. In the short term, we found that updating with the best predicted lines or the best and worst predicted lines resulted in high prediction accuracy and genetic gain, but in the long term, all methods (besides not updating) performed similarly. We also examined the impact of including all data in the training population or only the most recent data. Though patterns among update methods were similar, using a smaller but more recent training population provided a slight advantage in prediction accuracy and genetic gain. In an actual breeding program, a breeder might desire to gather phenotypic data on lines predicted to be the best, perhaps to evaluate possible cultivars. Therefore, our results suggest that an optimal method of updating the training population is also very practical." />
<meta name="DC.Contributor" content="Jeffrey L. Neyhart" />
<meta name="DC.Contributor" content="Tyler Tiede" />
<meta name="DC.Contributor" content="Aaron J. Lorenz" />
<meta name="DC.Contributor" content="Kevin P. Smith" />
<meta name="article:published_time" content="2017-05-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Evaluating Methods of Updating Training Data in Long-Term Genomewide Selection" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genomewide selection is hailed for its ability to facilitate greater genetic gains per unit time. Over breeding cycles, the requisite linkage disequilibrium (LD) between quantitative trait loci and markers is expected to change as a result of recombination, selection, and drift, leading to a decay in prediction accuracy. Previous research has identified the need to update the training population using data that may capture new LD generated over breeding cycles; however, optimal methods of updating have not been explored. In a barley (&lt;i&gt;Hordeum vulgare&lt;/i&gt; L.) breeding simulation experiment, we examined prediction accuracy and response to selection when updating the training population each cycle with the best predicted lines, the worst predicted lines, both the best and worst predicted lines, random lines, criterion-selected lines, or no lines. In the short term, we found that updating with the best predicted lines or the best and worst predicted lines resulted in high prediction accuracy and genetic gain, but in the long term, all methods (besides not updating) performed similarly. We also examined the impact of including all data in the training population or only the most recent data. Though patterns among update methods were similar, using a smaller but more recent training population provided a slight advantage in prediction accuracy and genetic gain. In an actual breeding program, a breeder might desire to gather phenotypic data on lines predicted to be the best, perhaps to evaluate possible cultivars. Therefore, our results suggest that an optimal method of updating the training population is also very practical.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/05/01" />
<meta name="citation_mjid" content="ggg;7/5/1499" />
<meta name="citation_id" content="7/5/1499" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/5/1499" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/5/1499.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/5/1499.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/5/1499.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.040550" />
<meta name="citation_pmid" content="28315831" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="1499" />
<meta name="citation_lastpage" content="1510" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jeffrey L. Neyhart" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108" />
<meta name="citation_author" content="Tyler Tiede" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108" />
<meta name="citation_author" content="Aaron J. Lorenz" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108" />
<meta name="citation_author" content="Kevin P. Smith" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108" />
<meta name="citation_author_email" content="smith376@umn.edu" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=D. Akdemir;citation_author=J. I. Sanchez;citation_author=J.-L. Jannink;citation_title=Optimization of genomic selection training populations with a genetic algorithm.;citation_pages=38;citation_volume=47;citation_year=2015;citation_pmid=25943105;citation_doi=10.1186/s12711&amp;ndash;015&amp;ndash;0116&amp;ndash;6" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome J.;citation_author=F. G. Asoro;citation_author=M. A. Newell;citation_author=W. D. Beavis;citation_author=M. P. Scott;citation_author=J.-L. Jannink;citation_title=Accuracy and training population design for genomic selection on quantitative traits in elite North American oats.;citation_pages=132-144;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.02.0007" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=F. G. Asoro;citation_author=M. A. Newell;citation_author=W. D. Beavis;citation_author=M. P. Scott;citation_author=N. A. Tinker;citation_title=Genomic, marker-assisted, and pedigree-BLUP selection methods for beta-glucan concentration in elite oat.;citation_pages=1894-1906;citation_volume=53;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=H. J. Auinger;citation_author=M. Schönleben;citation_author=C. Lehermeier;citation_author=M. Schmidt;citation_author=V. Korzun;citation_title=Model training across multiple breeding cycles significantly improves genomic prediction accuracy in rye (Secale cereale L.).;citation_pages=2043-2053;citation_volume=129;citation_year=2016" />
<meta name="citation_reference" content="citation_title=Breeding for Quantitative Traits in Plants.;citation_author=R. Bernardo;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=R. Bernardo;citation_title=Genomewide selection of parental inbreds: classes of loci and virtual biparental populations.;citation_pages=2586-2595;citation_volume=55;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_author=J. Yu;citation_title=Prospects for Genomewide Selection for Quantitative Traits in Maize;citation_pages=1082-1090;citation_volume=47;citation_year=2007;citation_issue=3;citation_doi=10.2135/cropsci2006.11.0690" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=Y. Beyene;citation_author=K. Semagn;citation_author=S. Mugo;citation_author=A. Tarekegne;citation_author=R. Babu;citation_title=Genetic gains in grain yield through genomic selection in eight bi-parental maize populations under drought stress.;citation_pages=154-163;citation_volume=55;citation_year=2015;citation_doi=10.2135/cropsci2014.07.0460" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=V. C. Blake;citation_author=C. Birkett;citation_author=D. E. Matthews;citation_author=D. L. Hane;citation_author=P. Bradbury;citation_title=The Triticeae Toolbox: combining phenotype and genotype data to advance small-grains breeding.;citation_pages=1-10;citation_volume=9;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. S. Buckler;citation_author=J. B. Holland;citation_author=P. J. Bradbury;citation_author=C. B. Acharya;citation_author=P. J. Brown;citation_title=The Genetic Architecture of Maize Flowering Time;citation_pages=714-718;citation_volume=325;citation_year=2009;citation_issue=80;citation_pmid=19661422;citation_doi=10.1126/science.1174276" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=D. Bustos-Korts;citation_author=M. Malosetti;citation_author=S. Chapman;citation_author=B. Biddulph;citation_author=F. Van Eeuwijk;citation_title=Improvement of predictive ability by uniform coverage of the target genetic space.;citation_pages=3733-3747;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=T. J. Close;citation_author=P. R. Bhat;citation_author=S. Lonardi;citation_author=Y. Wu;citation_author=N. Rostoks;citation_title=Development and implementation of high-throughput SNP genotyping in barley.;citation_pages=582-582;citation_volume=10;citation_year=2009;citation_pmid=19961604;citation_doi=10.1186/1471-2164-10-582" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_title=Accuracy of genomewide selection for different traits with constant population size, heritability, and number of markers.;citation_author=E. Combs;citation_author=R. Bernardo;citation_volume=6;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. Cros;citation_author=M. Denis;citation_author=L. Sánchez;citation_author=B. Cochard;citation_author=A. Flori;citation_title=Genomic selection prediction accuracy in a perennial crop: case study of oil palm (Elaeis guineensis Jacq.).;citation_pages=397-410;citation_volume=128;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=H. D. Daetwyler;citation_author=B. Villanueva;citation_author=P. Bijma;citation_author=J. A. Woolliams;citation_title=Inbreeding in genome-wide selection.;citation_pages=369-376;citation_volume=124;citation_year=2007;citation_issue=6;citation_pmid=18076474" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=M. Denis;citation_author=J. M. Bouvet;citation_title=Efficiency of genomic selection with models including dominance effect in the context of Eucalyptus breeding.;citation_pages=37-51;citation_volume=9;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. P. W. de Roos;citation_author=B. J. Hayes;citation_author=R. J. Spelman;citation_author=M. E. Goddard;citation_title=Linkage disequilibrium and persistence of phase in Holstein-Friesian, Jersey and Angus cattle.;citation_volume=179;citation_year=2008;citation_pmid=18622038;citation_doi=10.1534/genetics.107.084301" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome J.;citation_author=J. B. Endelman;citation_title=Ridge regression and other kernels for genomic selection with R package rrBLUP.;citation_pages=250-255;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. L. Heffner;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic Selection for Crop Improvement;citation_pages=1-12;citation_volume=49;citation_year=2009;citation_issue=1;citation_doi=10.2135/cropsci2008.08.0512" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=E. L. Heffner;citation_author=J. Jannink;citation_author=M. E. Sorrells;citation_title=Genomic selection accuracy using multifamily prediction models in a wheat breeding program.;citation_pages=65-75;citation_volume=4;citation_year=2011" />
<meta name="citation_reference" content="citation_title=Applications of Linear Models in Animal Breeding;citation_author=C. R. Henderson;citation_year=1984" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=W. G. Hill;citation_author=A. Robertson;citation_title=Linkage disequilibrium in finite populations.;citation_pages=226-231;citation_volume=38;citation_year=1968;citation_pmid=24442307;citation_doi=10.1007/BF01245622" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. Isidro;citation_author=J.-L. Jannink;citation_author=D. Akdemir;citation_author=J. Poland;citation_author=N. Heslot;citation_title=Training set optimization under population structure in genomic selection.;citation_pages=145-158;citation_volume=128;citation_year=2015;citation_pmid=25367380;citation_doi=10.1007/s00122-014-2418-4" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=J.-L. Jannink;citation_title=Dynamics of long-term genomic selection.;citation_pages=35-35;citation_volume=42;citation_year=2010;citation_pmid=20712894;citation_doi=10.1186/1297-9686-42-35" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=E. Karaman;citation_author=H. Cheng;citation_author=M. Z. Firat;citation_author=D. J. Garrick;citation_author=R. L. Fernando;citation_title=An upper bound for accuracy of prediction using GBLUP.;citation_pages=1-18;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0162755" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. Kumar;citation_author=D. Chagné;citation_author=M. C. A M. Bink;citation_author=R. K. Volz;citation_author=C. Whitworth;citation_title=Genomic selection for fruit quality traits in apple (MalusÃdomestica Borkh.).;citation_pages=e36674-e36674;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22574211;citation_doi=10.1371/journal.pone.0036674" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=D. Laloe;citation_title=Precision and information in linear models of genetic evaluation.;citation_pages=557-576;citation_volume=25;citation_year=1993;citation_doi=10.1051/gse:19930604" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. Lande;citation_author=R. Thompson;citation_title=Efficiency of marker-assisted selection in the improvement of quantitative traits.;citation_pages=743-756;citation_volume=124;citation_year=1990;citation_issue=3;citation_pmid=1968875" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=L. Lian;citation_author=A. Jacobson;citation_author=S. Zhong;citation_author=R. Bernardo;citation_title=Prediction of genetic variance in biparental maize populations: genomewide marker effects vs. mean genetic variance in prior populations.;citation_pages=1181-1188;citation_volume=55;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=A. J. Lorenz;citation_author=K. P. Smith;citation_title=Adding genetically distant individuals to training populations reduces genomic prediction accuracy in barley.;citation_pages=2657-2667;citation_volume=55;citation_year=2015;citation_doi=10.2135/cropsci2014.12.0827" />
<meta name="citation_reference" content="citation_title=Advances in Agronomy;citation_title=Genomic selection in plant breeding: knowledge and prospects;citation_author=A. J. Lorenz;citation_author=S. Chao;citation_author=F. G. Asoro;citation_author=E. L. Heffner;citation_author=T. Hayashi;citation_pages=77-123;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=A. J. Lorenz;citation_author=K. P. Smith;citation_author=J.-L. Jannink;citation_title=Potential and optimization of genomic selection for Fusarium head blight resistance in six-row barley.;citation_pages=1609-1621;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.09.0503" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. E. Lorenzana;citation_author=R. Bernardo;citation_title=Accuracy of genotypic value predictions for marker-based selection in biparental plant populations.;citation_pages=151-161;citation_volume=120;citation_year=2009;citation_issue=1;citation_pmid=19841887;citation_doi=10.1007/s00122-009-1166-3" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=M. Mohammadi;citation_author=T. Tiede;citation_author=K. P. Smith;citation_title=PopVar: a genome-wide procedure for predicting genetic variance and correlated response in biparental breeding populations.;citation_pages=2068-2077;citation_volume=55;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome J.;citation_author=M. Muñoz-Amatriaín;citation_author=M. J. Moscou;citation_author=P. R. Bhat;citation_author=J. T. Svensson;citation_author=J. Bartoš;citation_title=An improved consensus linkage map of barley based on flow-sorted chromosomes and single nucleotide polymorphism markers.;citation_pages=238;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0023" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing.;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=M. D. V. Resende;citation_author=M. F. R. Resende;citation_author=C. P. Sansaloni;citation_author=C. D. Petroli;citation_author=A. A. Missiaggia;citation_title=Genomic selection for growth and wood quality in Eucalyptus: capturing the missing heritability and accelerating breeding for complex traits in forest trees.;citation_pages=116-128;citation_volume=194;citation_year=2012;citation_issue=1;citation_pmid=22309312;citation_doi=10.1111/j.1469-8137.2011.04038.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Rincent;citation_author=D. Laloe;citation_author=S. Nicolas;citation_author=T. Altmann;citation_author=D. Brunel;citation_title=Maximizing the reliability of genomic selection by optimizing the calibration set of reference individuals: comparison of methods in two diverse groups of maize inbreds (Zea mays L.).;citation_volume=192;citation_year=2012;citation_pmid=22865733;citation_doi=10.1534/genetics.112.141473" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_title=Efficient use of historical data for genomic selection: a case study of stem rust resistance in wheat.;citation_author=J. Rutkoski;citation_author=R. P. Singh;citation_author=J. Huerta-Espino;citation_author=S. Bhavani;citation_author=J. Poland;citation_volume=8;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_title=Assessing genomic selection prediction accuracy in a dynamic barley breeding population.;citation_author=A. H. Sallam;citation_author=J. B. Endelman;citation_author=J.-L. Jannink;citation_author=K. P. Smith;citation_volume=8;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=A. Toosi;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=Genomic selection in admixed and crossbred populations.;citation_volume=88;citation_year=2010;citation_pmid=19749023;citation_doi=10.2527/jas.2009-1975" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Nat. Plants;citation_author=X. Yu;citation_author=X. Li;citation_author=T. Guo;citation_author=C. Zhu;citation_author=Y. Wu;citation_title=Genomic prediction contributing to a promising global strategy to turbocharge gene banks.;citation_pages=16150;citation_volume=2;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_title=Establishment and optimization of genomic selection to accelerate the domestication and improvement of intermediate wheatgrass.;citation_author=X. Zhang;citation_author=A. Sallam;citation_author=L. Gao;citation_author=T. Kantarski;citation_author=J. Poland;citation_volume=9;citation_year=2016" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Evaluating Methods of Updating Training Data in Long-Term Genomewide Selection" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
<meta name="twitter:description" content="Genomewide selection is hailed for its ability to facilitate greater genetic gains per unit time. Over breeding cycles, the requisite linkage disequilibrium (LD) between quantitative trait loci and markers is expected to change as a result of recombination, selection, and drift, leading to a decay in prediction accuracy. Previous research has identified the need to update the training population using data that may capture new LD generated over breeding cycles; however, optimal methods of updating have not been explored. In a barley ( Hordeum vulgare L.) breeding simulation experiment, we examined prediction accuracy and response to selection when updating the training population each cycle with the best predicted lines, the worst predicted lines, both the best and worst predicted lines, random lines, criterion-selected lines, or no lines. In the short term, we found that updating with the best predicted lines or the best and worst predicted lines resulted in high prediction accuracy and genetic gain, but in the long term, all methods (besides not updating) performed similarly. We also examined the impact of including all data in the training population or only the most recent data. Though patterns among update methods were similar, using a smaller but more recent training population provided a slight advantage in prediction accuracy and genetic gain. In an actual breeding program, a breeder might desire to gather phenotypic data on lines predicted to be the best, perhaps to evaluate possible cultivars. Therefore, our results suggest that an optimal method of updating the training population is also very practical." />
<meta name="og-title" property="og:title" content="Evaluating Methods of Updating Training Data in Long-Term Genomewide Selection" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/5/1499" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genomewide selection is hailed for its ability to facilitate greater genetic gains per unit time. Over breeding cycles, the requisite linkage disequilibrium (LD) between quantitative trait loci and markers is expected to change as a result of recombination, selection, and drift, leading to a decay in prediction accuracy. Previous research has identified the need to update the training population using data that may capture new LD generated over breeding cycles; however, optimal methods of updating have not been explored. In a barley ( Hordeum vulgare L.) breeding simulation experiment, we examined prediction accuracy and response to selection when updating the training population each cycle with the best predicted lines, the worst predicted lines, both the best and worst predicted lines, random lines, criterion-selected lines, or no lines. In the short term, we found that updating with the best predicted lines or the best and worst predicted lines resulted in high prediction accuracy and genetic gain, but in the long term, all methods (besides not updating) performed similarly. We also examined the impact of including all data in the training population or only the most recent data. Though patterns among update methods were similar, using a smaller but more recent training population provided a slight advantage in prediction accuracy and genetic gain. In an actual breeding program, a breeder might desire to gather phenotypic data on lines predicted to be the best, perhaps to evaluate possible cultivars. Therefore, our results suggest that an optimal method of updating the training population is also very practical." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
    <title>Evaluating Methods of Updating Training Data in Long-Term Genomewide Selection | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Ia_NqmdVshtkQBigPusZQIN4vov-suWBlIsTf5ch8n8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"484256","apath":"\/ggg\/7\/5\/1499.atom","pisa":"ggg;7\/5\/1499","ac":{"\/ggg\/7\/5\/1499.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/5\/1499.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/5\/1499"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/5\/1499"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/5\/1499":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022UByvfyRz6nWYvp5rhPULbGpc+tm2SMtqm41aGplBVS1KqoDA7Vw5A1V7UnadJvb+fCkjzwhl7x4m2b7ATJY8tpdoOF9LKpxO0aS6ljHjwrStE8HT3aBWrXanA\\\\\\\/oZ8TXgR1MBwqyTSYCasoeHfX9jaEuR229aQGrArHUgH0eWWBI5yzlMsdm0Mm94KQxRogpvYWmzJjunjRTT26mND7c6S2\\\\\\\/x7XxpHLadS1FsXAjOchUVIku3bC0EK+CGpUL417yU7nOsMuXFmTov5ektZ0317NvHcVIakqdlAetJh2PDX79qNbA8xzz5Ay7RzYzDpqacFuJo3K+GbqEYxVj\\\\\\\/uDSrhLW+DbgORX1mpEwmt2zuOlz7P9vRm8cnZZrYSk8ACIeuKlLChkavMG3i+WE9MRNoAzKftN\\\\\\\/+CyrM4Wk\\\\\\\/hT4ldLki8gPQki4uGpiJRZYVf2fJR\\\\\\\/6o2HEUoWNbuUYN9yabuFpCgBWTKivVF3boA+c4B937jJJtiqhWeAyA9cdKbrGVlgpFzKFpcDC31g0md8FfMH\\\\\\\/f227DIduJFVlD6yPAW+51fKy6DHzQUKus1ihWcnGuersrFZA2D7QrtlcXsU4KTUAFOjMo6d3EgMh5IUpYHWBWaZ3HSwMEjkgzv3Zb4xN\\\\\\\/b8mrrqGGQL4IdUToXmxLCp6JZdzNZZLQj2\\\\\\\/mjee7iCSt75IgkwBEE4JOFMhipxAojgUNLCGAzhWFmHBMqYF\\\\\\\/d0l51Uact8\\\\\\\/UdETN\\\\\\\/x6fiAAgPxlAKnVAQ8Ewx3yJeoWxum8tGg\\\\\\\/FB57qq5UuveLKnaMvL2IbqZoE8GZLlzLCFO+FC6sIkW\\\\\\\/2nf3FXAu4Y7VsljhnkY5lz8yBoKAhkOLxxW7bnbwCfxZ\\\\\\\/qSDANdU2WA5n6VuNvsTgqpAjvRDY2KM9SXqhBs0iWxH+pHzrEGkwbsfvJi2IWT1SdQH4R\\\\\\\/xfxpC8LYeSiX8tl3xCl0zrS+iEfhiKynybPUvHp93AZRsbubcaFPF3O7ZlOIyzBM923yibRHTSLasqdoGdSn+YLU96TCjBlZ8dNIUnqNVbt0KrKLxUUEitV4oxYkW2Lrhka881XtPSKR9mZqZMkHf1RmQ\\\\\\\/oDbZ5MmaDAosD3Gal9j6C196cuBWAKTOQenFCG3yopUx0c9RGDMQLrGE1idqaipl0S8t1bU3i7xAahavTNFYoOr+ySa\\\\\\\/BIXqHj6mYRJELQCkm3vWT31FOKvQVvIa7ByHPMQDkB\\\\\\\/jQH40w7yRSyNpL7jF1NLEJooxry7sNh+UGR8+lFqZgP+FlE+Mzs8+d8v2\\\u0022,\\\u0022iv\\\u0022:\\\u0022qVQMVHVsn0ui13av5P+ong==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224d43a34bb1ee717772e9a81fe4a8571f24eda7dc13bf4f5d2886f4d21ca098d9\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022GfzFZHpqw0uI\\\\\\\/90wl6B+eXcxGZdWF+m9CIyLEH9oS4J\\\\\\\/1o91Swkpr7RaNxuX5++odEq82mCA9vImzb+AoNG79QAhTO+D4mlWzGNYAm3vqRZRp51mWTPrgN4bQ\\\\\\\/r0Vk\\\\\\\/BtvYuiDZfKPZjL3j9jwee15ICM+zevpEogow\\\\\\\/MZuF8owZdVnPWr0Qhh0xUHFVdf9Nr3VMYGrV68SLCSu2QmpvPpmgLtFKYYAxFJO9ok6k0Wm8\\\\\\\/dHidDeLxlLpZIqDKLvI8nmET3yxjEOiIr1jdvLA6I6Ca7SQU\\\\\\\/rzG5ju1SBaXS\\\\\\\/01UgLdQUQWiSCg4qFW0zB72V7rstCpN72k6wvQ5p1Js+GGVNklptsxgB2OS19CYbhLYToQ+dgW9hS2QbbNGMDUo9zh+9\\\\\\\/2KRk4Fhu4UsaK+5eLVd1G0QPR+o8k\\\\\\\/\\\\\\\/2x+msmaj53lH3SAnmq++fAQxhWI2ZLNINgYXwTywQKJ\\\\\\\/A4RLdCZSlHFNIoy5p80pvm8nKWTThhiWfTGY1R4H+qca7o7TMo6GwRkvluWOAOo6HV680PLoOK1Sm4NBkQfAuO7BhKKEo7F3u754m80Kz6suNiMj3Jb35z+OyTws7dzbtNc7GOlGxfiBjX6WrQuMmCEh7s8a3Jcm0SslJt6kBF3SBP4NGAdtXSLn7pO9XqAjZII3P34DOcAdFIvfpu\\\\\\\/K6\\\\\\\/+hr+5p+YGToI86bfhWdDbNCGzuemw86ySRp7S\\\\\\\/FVTxV3t8rxf800QatYTDrKgUBmx2JC1eGaK2TuN5RwSbGrQM8ze0zHwiZIWMEi+W3DAiRAzSUIn\\\\\\\/4j\\\\\\\/5IlIFDgpLkhe1Bj9zoUsHugTtaBEfaZI+njDcIQ9LjBYKPb3X1d7Pcs3Fx74W0udP2bMlZaUUxm3\\\\\\\/RrWu5Q+JvPyhTrdssFzasSdkwhjq0sGnKZ3fN1nRVuk5rtZYEALEL2otnuvfJpAsQ9dh10+wy0q7hin\\\\\\\/h2qzDYahvNjCJYSTw7+ES5MkGAsh9A8JsF9SqCIHVGt2VSoC2GCZiQn+hoU+5S2UnFyBziLdaIfL7XWoyGBtq+u2eFM9IixZTy7JoL9aXKumbYca+s3qbulWfg+RY3O4SqIAZP5ig670S\\\\\\\/\\\\\\\/kR4El\\\\\\\/iLmPoBMchzVCpRoTAbgSmspH7Nbo1qUUga4PT4c4600qdFDp0b6loCynvBtdMSm6SHeHm+hjt3gs8LTjfPRz4vK80F\\\\\\\/3uy34DY6hAuXpGlbnwvmLrewDyPo8snmk2EXiV4fp0GuwTCpItnzukvWRIsVb8dlO3Kd8Cok0z+Jys65OsvkE4bYDHsWzbYuAGu7Qr6iOpxkAVaJ9CfczkIaxbScCINOsvdABejYoEkDIVza8wI3kd1crTk0uC+aJznZI04sOOmoa8BjgwlzY7AwSmgakvAG2PnMYa\\\\\\\/0FfQMa9ZcIkrVsVF5cxDf+Isz0UEQEJ+q2SZjckn5Q6b4aB9pVMUGIe2A3U7XE\\\\\\\/CxkO2VV28mUVyZan+b8+bX\\\\\\\/Dfws1+xo\\\\\\\/6KGcvgQy7m7xA5B0s5k4Apt8KMesPuY8aIWYhUXM7hGTDfCl+PcIJTRrvaerjj1V7ef3X2JlXZZz4wHraqagafzvYqcP1hkC8b1T4O+f4Q0b6YbioyDlF+NsXIz2khU6cT\\\\\\\/wuaRF+VfV+ssFXTCfCrJuYJbgvrSjhlgnmtSJEWqYFC96D4YQ6D1H+31trDh6kFJelr8xpHLiCszX1yF5sj5UlVEvVttt2XJy08m9DCRhd\\\\\\\/eSH94pyrsJrVtf8H4RpOPwOE8+fBaxeMT2T6He0K6Ll1BoGAf5kGunr8XPZk9vm0PkbOS36iwDOO81zTy6b5KhmZkZlqRFVc5cmtzCpwIpJ6XvsGVLKTIey7Fl0ZaPJ186WlmpsHtY3YxGSJ5ib+Wsj\\\\\\\/qdrwb3yCdCfqQdlqsJO8Mi9D1NjSBCDlzRadED3cBFHoGqx73aOohk+A6dYSPer76T1cNSjQapq3UbL+bj50LsQIR7IbeXhkqvkHQyMvqzup21qC\\\\\\\/cqDIhqHjcF9ZGUWD1jKeRzTRZzUv3fkO0l2yOANiYv32KUHllvPC2EHn82oakCCjkAzb4oSM+n2oBICMb3rPrYhfBlwZkNahCn17pdEtJlp\\\\\\\/xik0ivDUykoudJ4iAR1NO\\\\\\\/r2fFD6RS2JnDVTvm5ccavl8MZ7ReZjP81PbgxcrG5NrZ\\\\\\\/V3CPBfz\\\\\\\/uW\\\\\\\/B2bI+Uw6tTt0lkRm9B5lx+Klfw9kHCwo4yxdM4TL8+s1ymmzYM9ltjih3ru5WWg2hIVRk0uyIAoKPQF4N11TWmyW5rWS0l5Bp10aVeuaS8rt8OC6VnoJRXMlkdh2LZ6ZGSr7sSzewuusSogP9UUqB1jb7LadnpMsXDjlZcy4bQjkLuNi8WFkdgFbz2LcdOAs3eOGOGiMuwQ2u64kMe0UoWNuh77CREYAtw58INmnUCZ1\\\\\\\/oB6SR5Iz8QqLpKsrotFjTslZb01CNhoPTFlndQhOy+nkwRhMmYTSUZF7nTkHKIZHz\\\\\\\/0eRvttRULfUmei1DZuQRlAwpZrHrp7MmFGmF2a3OxD8uW19\\\\\\\/inyUX0zUtWaniwwsqCeN2HAsnjuv5ChMZB54U9jxoC7ZG9gASFYQVeRqg5cu+Q1qx4b\\\\\\\/SpKr+13T5POEW6MKDHp5hacoIt5loozAwzQiHZ\\\\\\\/ubmt1ifbv8jrXmEcNz7cMW6rykKS6UMTioQ3ZIypm4LHJIooWHTxYLMmrYZruPbrZoHlZDFILmU9W3CJ4TVhNuzjftVKNOEZroSp7cKRw+h3C1U5\\\\\\\/yEMA3WnXaZp3Y26VMsh0+KJwDG4mEtGqedVbP1v391qZw8opOgKc9t\\\\\\\/Js\\\\\\\/wJO4x3xYPwFt5cRouf0spsbtLKY53NCJjJdXc9IJN5Vcet6VW4vseOasyxVI4V9w4W0WQqEPLgQFpW3v8riXAyaI1qYgsUanMgQYSAmr+kEw1RXdg11+t0i\\\\\\\/YDkUtJMlew9U3OMYrMuTcUCCilYqGY5RtpVSl06VYbF+qYFRs7xtXg7cmj+jasxrFjQA6vmY2Y927tR3aiv9BZstn7tKoK\\\\\\\/lSn4S2o88PO0Z3DfN3VO57cPEdmw48LnEnVAOjtrtFNUIv2XSuDAEDWu2N++4HrjKSMpLqLcMiXlin6OdzwySAsTRgxWbjj2fpZMgVdJuZeomug+ZHMgbzVbndK26\\\\\\\/jIysDO4394ZrJAyDchIj0tasOm+O3dxO4VNmQwstQEuEmpgf6IQWFcqlDVsygQHCArVrCLSkfyGZRK3fl6\\\\\\\/oAbALEtcBYHSflaLaJ1+wDAlz5UNmot3SNnvyMyJhpxxuu+\\\\\\\/9H7agiCWE+MNldEjLpB+onngoHLGdKCl0tpc+3ZMMKJN+A6lK+2OAKoTiO\\\\\\\/l5WTIScw5aU7QbuKh+a5M8oh3hV5VB15dTWUWjswp0df4HoXTHpIvvjLuFs6r6mRmV8kV6ctapkmd4qnPkA1ulajJhhPhZbqEqLlnc78MOx2DGRt3RsQvlywL7g6HGc3T6yBL28A7FtkGaQv9P9TZL2QEaTwkfljcchdGwojk7az+zXEJgr3MllkANYlg4gbgSws0F8sOa+FJ03snnDRs1txeeNHO\\\\\\\/n\\\\\\\/MRHa53HA2CCYiElriucgp165vxfGzVU9I4mGhdtnB4PbXieybfjvJw3Uc0sAPbOSUSRA9ccRMyHn2qNs1mOQN2AjkwFUcXMZ+muLraFcX3JT2VgvP7FpJrn0bCjC\\\\\\\/c6NLxiGPB8M2CNOrrKjwbV4cXtSlYlqaRjZt8dHvrSY43kFHPdNhB99Y2roFONqdomHJ0aPksTOBNFXCFnoJTWoknMBY+tJxwDf1jKy9MgQbdGO0P3v3iN4Z7QlHOTzTXHyQmTyB853DXsjwaxOXC9p+UdNJwTrG1qNmeachPr\\\\\\\/2wn18dqk+p4iXCIM7AK4CwhjBKnxgrRSE8YXnpmaSA5Q4cuWvdzS\\\\\\\/2ob68s9osiZR1aio41b6jwE1BxpW22g5rR2KeJhxcgioOCgQon6bM+lrBIi9fGPqTZO+e+XMihahLAIz+ckLw435PpaThSLRNq+9VbROe2EE+idjTbqJTKHx8oQYNjvN\\\\\\\/qRz36Hj1Rxa0ByTnppEwG0+8K0+KXw\\\\\\\/PFGWpEv9zsBArs\\\\\\\/nC6MlgEsbc+HwNHy4lHtYH8aeuVu525s6e1ktcMKanZogHe12HqDQlNjpDuCJdsiPOx3id2Zus2luDC02o9rYnYhRRB9SPM9OAvbxR67hGk20h1WTT8FMW\\\\\\\/e6zuFCRTTHx4uSVoIuc8fGI0f3NpMKRr\\\\\\\/z4TFMJ60bnWxEcfcwc\\\\\\\/RsjMAJtguGrcyM2x\\\\\\\/ALZCFlrzqPGGGXe6UZfcYdCp0VZwG8bjcItEekvEew+jFlgdxdVRa2OMHdyXY0hrgO3W7ASztYCQ0PCmGZj+u2TWqHBpafsj1fWTD35+OOQkwXnizbPkKK5YQyrMgp7i9k051Z+U5y59VmR\\\\\\\/\\\\\\\/skQwNxu6U1hn\\\\\\\/bKKY9+kjCQ1OHVOWpYEbkFxp5nhfrpCUQgz8valwTZ3g\\\\\\\/cxTQ4KBzmUVYGijhGMrbfxRqC1+4Sj6egr\\\\\\\/GXXPRMG3BhJEEkl\\\\\\\/ON3\\\\\\\/VFVoPXSidOPVA9ps\\\\\\\/sRpNhHMIqLEJz8sLG+qtarFHB5mALux9IsIOlTVCcDcjuWt\\\\\\\/3D\\\\\\\/SUvwedyzNEStAQVrr2IH9Xots8A1hwrJK2A14syOEWZZ5AaUH2SkbUkEp+IydnuS0EGqKvWRa73cJW5CQTPsK9pwP7rFRGTBIHBaOC+DzmXn2yaKVNaBplm3cDRQTbugWpYWbVOdhbaNiG1X+MwOM2MN0g0ODuVJ\\\\\\\/nFK2P4O2QEtlvTZnBecHN9jfcGOdTYCjmE05aj9lCS+ZvJURMZp9mgxyNmkSEfgcyQF9\\\\\\\/T\\\\\\\/kGOzcuOgB5kkEWZpGW+7Ui7iNJcp\\\\\\\/hnA8krKBxYoQTw+nAdvfa1XZmCMhgRso6aTnPktqcKzIB8YQ5YFC02cb+fVpiFlEXzMJN3mtPje1fNmTwh3u8pB0pCMi12DCH2tCk2g0T51j0H4E8\\\\\\\/75ynYE3zhiGUP6+CeeTozGvZvMLVzOJb9NsJglBY7peR+NTyj7zXy5Ud6fvH2ssdC4b+fvcnx\\\\\\\/2gaW2gOMTdeb7O3Neyvr22NaHeNEFwiBSGZ4E\\\\\\\/ki8rDj9lXLfPIXZaExROmB0fjqKXQ\\\\\\\/SQlTRSDZ3z1dh+WlCGw6cdY9UtEQbUvu2aB29SVBx8iM3oFdb5aP+APTsUsjE4y8FQMK5LYw77e9xxwI+EIdOQIKUXhi661v9euvYEzRRTaLlt5yTRZCkiMpgZz1VEn\\\\\\\/O93u0r9CsSnQIW48hAjO0USQdPAuIuLdY\\\\\\\/wCmR39V4nm5U3Ua8U+TmQHRUnpTSydLBNgLXVrkFNhBXIyJJSmWiUb+RPm\\\\\\\/CyDm2oeuR7XDx\\\\\\\/5n5A+Ul2zIiPFoOW19ewF29imXBlPS1In3Ffg1hAgLN\\\\\\\/ODaE1ffWeSWTafiKdgDSney4Na6Oz4OCZ5Rez6muWp7jIZ0TdrLjmaNAcH6N4yv+KofXe9d2LvLQqsHy27yxFKBWT8urr7rNuUwUxlX2jk\\\\\\\/aBdUGJJd2w+4m6uiiwLnPDPyTBczUQG7PALGo5iN6hMl0qE7G\\\\\\\/xJJ+TLixyqmI8Zyoyz\\\\\\\/aYXdF9PCF+HVD5PYb06bvfJQhSgv4ycn1KbHEj4MNHamd12UM4VrPB9PCNwJqhkI\\\\\\\/fyw0AyVAK4LUYgiqR5tZS+n46+PwSmhjgJhLF4\\\\\\\/kLPaub3Q58t+JrO8ZXwkxX15deZ3DDzKhFHp6bvLLJ1tWxeMiQKhzLN4fBEIpFhemfCTZc0r0CvBIS04ZVQOBnRwJ8FEFyZAUIyDhjlxhzrdIVpm0ockyNW5Vw5Q1F\\\\\\\/Ha3dwhLZy2cwu2JbDkVa2vkyCWP04ItK4pro1voE2fkbjtPI2+z5TfFK3WVx24gFdReJTfKxxoFUo8Qh3DzjtBJmRDwYEHnScIi3\\\\\\\/yaoLBJS3Mke0tIU8LSrsxTzJBbjLmix5hdpEOzy9vDXaIohtksWNwF11dZcZH8iyWp4vchZEfZMDWL4RxJlxdO2jpGYed1NXuZ79OK5o8s3CkNnzcsynONpnh16wxUmtWFgNTnxAjws6jVbETCZsMTi\\\\\\\/BAl02iDaZHsKa5HQj9Y1S9XMaHpKQIcPd5RcsQhSWuxTFqaOKybIGob4HbZTrKi8yhpRXNEGxHASeTbOXAuEn43S1rNMdC7o1+hSJiLBdcpi3TAflhaR5LUZBoXyh9FJfmAb0Mo8TQaPTGWMnI5mVgsEHp9fHebSdAxlMmvArGJ1h+VHsJXpFzJthObsQKSVk0KITCe+KPgyEcAWc+l3TFCPSOIxY+XnLZADh1Z8qNPWmLQ9jqRRBUuZIHpPoVG1dWA10Ss4jh\\\\\\\/lDi6A9erJXUj\\\\\\\/qU7VQiDIWzIjCN1bw14HQCnmg\\\\\\\/agk18aAdAiAM53\\\\\\\/vryllZ3Trgg1uPOdbcGhixTsx5Tlh4KmvsDJEmQ4xuhKsf2lwO1cV26whNmhfWGUPYOEWkhi5UlVgZZDn2o05IpLd7SmL2YUnDlEmCJt8RrDVQwbIKSjYZ3xOVeMoljZX\\\\\\\/EJtVQbUCzEDtNZYtW2aSagNVtjs3aZR\\\\\\\/lGbQ+pplhtCfKn6RRjjM7vqUVYbUwnyEXs577u2s7J53Hw\\\\\\\/KMmIvXJp1\\\\\\\/4oUAzWgUSJXkP\\\\\\\/zrP2Z1mJ2Tav6Qyy\\\\\\\/77L7\\\\\\\/yd2OQ3l1pVnFOhA97Ko91rtD+iVje4z3XClTuGWgbcHWi+hGFhbLTNVCKFCjceYdxuysk2AQlp3KXxU71jwJfKnGAj70uIG3C49qBU8tqhfSomfHcB0nd7hE36OF8a6nNDccuhxmZ25fdfRGV5bmYNfaFgvu41kUBUUZzv2HGxwv3p+CPcQ32OGX6FSuY6O5KZiVVQ2ST6VQDW0uIGmFKaK4Qmfvh0KWEOy6L\\\\\\\/W9h0jAHfNN1FEa4OEu1q+jW7qkFGAmnAB4lumgypqiXQy2xR239TN7NsyWPlNU+Jein7NjzHIAqq5n3swqZdkYui87eEvKRGNRDeiDLJyOM+HZqlqFnBbox8a1i3dSly17yLRYmghcgWU4+nLpcKbI\\\\\\\/v0TPD42uDuSTV04wKbLGTdXYrjqg1g\\\\\\\/2wbGfeInoEwXvV2wCo1qVtKI\\\\\\\/VqFfWk18WHviuEd5m8d6a443rarSrzmKPtJtR8AQWnWSFg+p0l4frZ++EhuY3DHnnrnvk1UZxvVZ9mxYzkkS80kUqRrCNVv7rGLkFl0S8iV5brfWnZbuxsTs\\\\\\\/yev2DzPsttV2xV5d8fam8qbH6kWNljdvJJQC\\\\\\\/w+xQCc13k2msQRAbALGV80aEH7ULOznFYt805bRNFUCB2nor84xeQ6uJfhTNsRmUeNOmvluHb3veDBlbHYz4kY4Mkd2FyfTV0QBBdtLAo2QXMOD5eJKdz1sknTe1JNdNiEXqTRt8CEeIOa3aqY9qjwf6G+SDllV2Jk+hi7hYTcFi9BROiONl0eZnVOpoT5eev48DqcQI+RPqaTjzqkdwBX1R8yrs0ypxYfHGnttoFlO+8hc2GYcDc04M4nsKFjMFBmoSJh7ekqmpdZ8jXzyz829otO74jYPvLxY1OnEZsrl3Y9KHCd4kx9RzbsFykUsJawaKw12ZtxDuDGrxCxU5W5k3SjZbc9XT\\\\\\\/g4EjcxuXawSztlDhWXsttS2MqJl9uuFzZL1rdYMnx8sTDNmYExbq3kwjQkauKZj19vQ\\\\\\\/IG7CXvI5r3pdB\\\\\\\/yxReekzwSDZ7agt7djE3serfVthqjd0hZFkMiZcs7v52ioEJFyCegZXWhIBZgABC7VLm7Hx705LUBkXWJMFLVlJ9RRzz6C0BOrjGn3evr\\\\\\\/keSmTzYdaxwAmC5MPgvobF4oa5yciHPa7BNVcXEeWkOz\\\\\\\/CNIMH815UmPPW0SC1jc9tiU0x2LmHOJPMANVLyvf3+wMx3dWc4HOyN2jk2MpmdYXuJQZg3XT4Gs1R8Tb9rSD6B7TEEXL1sNS34vwsONmc5n2eAT4T\\\\\\\/+ozbH3cukPz52dS3\\\\\\\/u91vdc\\\\\\\/3\\\\\\\/5kGdPrFsXda7opOr\\\\\\\/TS07HMT5XPODfzA+hHEKpRD1gFE7yFPjjN15iv95EkM2mK0D1shd\\\\\\\/Fb5X9AMJq1jO2kv5QaEYF0XBiM2f1Rv\\\\\\\/m6YCl0FcEqYIYCYmYoU+szLZDuOv2+gv1\\\\\\\/oLuZ5fKaZ4OOs0rn55D7M6zklIRd\\\\\\\/fWol2M\\\\\\\/Vnmz2\\\\\\\/c0aKIndKEDYC0fDnTUxKjCoevTU5hMtCJsEgTNDlDCNCKBFyWDsOrRgEr1JQTmuDkrnbv6VmJG5lBErsbkOLyLrK3BcnD3yOFS\\\\\\\/89UvM+b\\\\\\\/pZcJUXVv8nfrTWp+vgl9iG6rTxuUSCzLoYgxw\\\\\\\/zrmirOksecKpDjb17Vh1AKZbWpaD29GV9SXocE8x9NrWL6sl\\\\\\\/t1XWL8ckYbGDoHuGpGqXyK+FvIZss77CFjkW0\\\\\\\/G06FqNd0HloUf\\\\\\\/Xof7evhIzmDKntwkBdMjfRw5Xa5jNpmBp7LW\\\\\\\/P9LfrinWMvvNS+l12Bp1LwBARdYwpDo8kAvTUV1xTDfDxh\\\\\\\/7Huz1DxUxUTtLirljgi\\\\\\\/hTMXkXYeASByGHNKlJEyMYQ59CEY5HbhvJWxkKZ4eBSeXwYPC7Zgzu5qM1V3V3S2C1YVKsou5qL2xt2lsDpoHOvWd8+qCS+\\\\\\\/oEB\\\\\\\/XSzmXjUpLuf9JpgHPfvErWPcdBHN620xXxTO6blYWGeGPCJCXhh1luvNK7ldwfSpwxAA5b7e7O4vuA6bRHWvpwkaqTim9SearmFdQ24KFclGru8V7BDyWryDqiSqWwQYSXGw4wX\\\\\\\/TnnTCxvBBvkE2a6QjuibrLAsFQ3VrlyH7HUid3\\\\\\\/wExcLHk4XTu7A4YJcCoxDC+knosUr7kPLdJrdR0CVD56h9fMA81pvePXvStNwyGqBR244FKjAbOYsWjXmSrfuO1gi5jXYWNcMAoXByTnc6jMvihGlIj7ML95IbvtMpMe11FkNAMwLEYDBT66wTvLT8J48TQtksCkRkiHXS0h1\\\\\\\/01ftJADlDwYRBWwwFqVzmvxpA5pd+gAPtcF52KXoi4XHvVqa3Y9N2sJoBij+9tQE9FZ6iEy4nQjkR0hzb2icW\\\\\\\/UxExpZ7dR7435xKpcGZ0Gswk5jxl3Cj9oXS0aq3YXhIP7geSQabWCTJfEiwPyZr4BsDBBUolJh\\\\\\\/VeSlTTefdNP61pwwAgeiIXU6WjoAH+K3XSZ3b8jfFJoIGDYo94vhknuRbnaPYUFm\\\\\\\/IQ0GfFN2qHU2AsFkkcaskBgqOLHAolMr11JEOnL9aeCjwB0P1mODMp3t4h3DAzVIdvchauoJJLE5zh5loSBR83NXxah+MOQYJ4Ed1ZKCFCyofTwR+oYhSAd7y86tk7ffFLRxGqnypHTae11Zv4ngCkxYo1DslMGVPXB5rP2ftFY6HhXAtz3O5BisAz+fn5JWDH8R9aLu16LafbeCbJ0VpwybP6xjtDJAXSXceh19Ix02HzB8pH0uA6wg7mfZ7KnsMZMbCTKPKJdpcQ+CHvNS\\\\\\\/KUvncI\\\\\\\/RWNeCpk863aN9w+QeBsRH603mzjyZj2mOyuLDBA4Dxrltlw+XE+7+MHPx6zC+U1asADhx6HbrbZ\\\\\\\/t4b4f\\\\\\\/5dVOVl1D4xrz\\\\\\\/oVQWdCj6rjt\\\\\\\/dMQb4Z8nMwuMLz3E4OWtwnHuTqTX5Zk5laQmcAyklFoWdEmYqaja853H6nDNmgvnk5qbg8Zp6zYQToDX2aH7Vjsrmvpj39urIshfCd+6sHQ4Q4NtvG27tbC+tLTimVjtIx\\\\\\\/h1w8BRMtCso0rw\\\\\\\/IxjljChPEsY0EtMauduv6LXNpYvg8zLC93XfgTeocF03R6B+HTUsnG7+GaWKBXY5+JsJYKGl3im3rUjZN9VGa0BLGi1mhE7bXEaUio2GATQEJqO3Qj2SeyfrndOtvv66kBzFN5EYHdrr7rDgLtHbpslm3TuviOgqm77jJ4rAJZCJbhTP4zBR7EXhDsAnctnroZS5Zo1Ft9Q1tTODn9JmBw1JUXb30AT9GgfG8DrSQYi11dVw+v42E+fY+qWNGBReq\\\\\\\/KttdkOTyiPvYOIMP5\\\\\\\/709C+3yETZSB12QufujnuuyMtq8wWCYUl+Ft7If+rAjhVDYmylMSBXqnEs0j5AVnM7MnNLjG\\\\\\\/enuorKwjSCEy6ZgltOzM1T5w63UvGHmCc0fv2gZsUdY+gQ+Znr0SKJ7BOvS0sTwbJTZmSd\\\\\\\/LFJzLvKhuW1B61YOrUaWax4FrWtyjZzVyMESTUB2fFzkqC\\\\\\\/epcL8IbzpUwR9r8JX+HOula1aoCq5wLZ+dyMgCu9WRb7XWj6+dG00ZLul\\\\\\\/4UNpQ+X8pVkzKJTejeG67QC+XecvtlZaq5qiCFoHIbL+XQaf6gC+ZV\\\\\\\/wnUiA2wXYirgC1jtkXtH817YNDF1VRd4Fmu98b8lMuAElHpcMoiaU\\\\\\\/tv47N+jw0RxZ9057r8jHDmYz2YUPoaoToMDSwdZy8U7Baq0JEz9p2CXXA0WDYmKTZu+mltnmMeKubkME6L29hxpqM7M+jZotcuo1yE1vmyQIlKD26jZD2o3fgzaEcGMTVZ\\\\\\\/EICx0x8SwLd9Ohd3xKF2Vu7UnlAJ0Oiy1SZpeOfLII0Vjb3zuuLqQf\\\\\\\/wXzizEYVJ7aGqkN3\\\\\\\/WSQVqMwyB1eJJ3B9P83fUDr8X4wA9xyq209e\\\\\\\/yiYpH09TJTslbpDvRLsFMnVHwS9itxSmXDRUUhkUgC5I6RWcEP1UZ0RycbP+C5p8heY6daIMpWDOK8CUVWmlla91PyTIRyQ7M2wcyWY3\\\\\\\/BTpFimuP7BCvE7D401lB0hjOyfsH9zNZemjnZWNQhpAP3G8zxlaPW5HV3jDXFRMqh+Bjv3uaE\\\\\\\/sbaztgq6a20NXGWXbZqPcN1jk4RtAxDGeBu4o\\\\\\\/xRsjiMPuUA+dKOw7ATSemr2Q85l2CidKEiSoL9tYvgbK3dLKITbYodqJqMdx\\\\\\\/yQlDb2iaQlhZ8kHFjUF\\\\\\\/twKrAmM\\\\\\\/lHNNwS1pTc7aPpq8himIbSkicQvn0\\\\\\\/MLMMdVY+eHlWDL99q\\\\\\\/2mfjucGPS\\\\\\\/byXKNY2S+0i\\\\\\\/7P3jrPRsV2n\\\\\\\/+VtMuVgWTLlsAMNhOhvH92Btbq2brzBcR\\\\\\\/Dnq64T97cCZPRLmGhzzn++2mAJK+hkXuhYYKDJ2fLeIzn247W\\\\\\\/PonCnq4PWLaxbrTQHA7Gqc4UioMN+bbda2N6mspNBrB\\\\\\\/Mhi9bTKGvJCTiHBIHqdlPmyt+p+WcIE4Q3Olz5Zf+v2dpGh75o1g7rrJU114fLOAYVG81127c8+ngsFeHs7Rs28N9Wm3equnHEwN9idgAUHUF9D+VC27PIZtZm4tLu9lp0kS5OlFobCdC6qp23wXKxj9Gy5vmgoVEEanhe5WXjctFE01QcqQkoPc8AhmwAsGbyHbCxKwAmbVoF3hElNbx01elHjLSoKYG7mb\\\\\\\/c54AmHjDwgXAXTpmobO3gbXtxij9+Bb0VVfXy5BICwpBpbLxdelaBBSG1Fw1l5bK2FqGYkrUnstkzNt04zXdUOhYwGPKafhX32NRvU9WGApfmMbQ7Nwptrx\\\\\\\/FjTnWFT+9+ceGzIrMtwgi+ZfWg0cH40BvXAF7J6yGoIPxJqZ71mGc59SBU5iTeMGDMeH2nrIWCmChoJFdq2xEEZyG5VwAdpfFwutmXZiPDChxMkLsDS\\\\\\\/jqq3BBdFpJKNelChySBilwuc26X3hLu5Z8UFMqRRqjmUqJN622o2dQTR2tOFuPJxCQy2VJciFzOkCY2epL3O3+U53o1Qo7yaOnSxGtcorWWL0mv+KQV0yDE\\\\\\\/8\\\\\\\/9CgWUYSHhn5ExXXpPo7nVLNCzqmldfvNzM00vhkLDmgBHRSObeOUax3hK2qsCjG1m\\\\\\\/7qrlHBS9hhsQeS94HbPMVLz+BGb0abW0x73qf9nX\\\\\\\/X18uZEqYMtOFo1iqcU4qjxF6pRoCTYPlIO\\\\\\\/XlghMF9KaO6ywr02OgpoUtKAq\\\\\\\/7C7mLEu\\\\\\\/bTBXDB47VyLVLM\\\\\\\/5ftoroFtG\\\\\\\/8UBoolIyXra04vmI6Rk8Xm0A9FzxHffQFhZKhBGlsMDbTb3e+u243focmBYx8qpwzMI76NHu4ttDCLqFjs1LqzMd44VaJgI1zSikBvm0hDB9WvueDPUozBcAkB3Kte8Zavs7bjMGcVd1VmmXJ88cpZVVwRcymFQMbsuZ9nof4qbyLcvkYpg9DNnDH\\\\\\\/rhrFF\\\\\\\/K+3GYTT+Mluh+9kAxRqKNXchqK9eMdBmUUL9wFuRtARPfPGa4eb\\\\\\\/SN6zPlGt1FvvnjleoHnju4LJV1L+LEIfnp45VYqYpLbdkQHgCS5ifyk8sI7hiGS8qtWm4HMUzXcCnjT3DjqJEdJ7RHP67WgdJ3OPVQtQAvQ51LP\\\\\\\/qQNAjg\\\\\\\/RKG1VKLpN5jzFV+f8WA9OglhoVrYBJQvZCJkHZzp7qK6yBDFxaZ0a8DFYhtu9KbFTdCf5MVPKeEe0ZSc8dqJzNGDXQHMCSSnaxIftKqNJM+NLI4sLe\\\\\\\/vqAkDZe8ZlLkq2P1HR1LJ6KZYU94tQb7oBYHANWN4bey\\\\\\\/jqj0vYq52E2tPoBxPkVN\\\\\\\/Fp23nJPnxfhsmidMKGLf4eeZAroPeCuejzur5Dg7kQwwNFs1w2WN9y0YS7O+NP2SqS2DhfxwcdbuUfdVK1z4q0DK03+fhCP1AvuA9QoXekEl\\\\\\\/mwIEj3ikDg\\\\\\\/NmftvgquFwCjgWEF2tKqy1L+CCKErpa1fdb6kN9QE8hubqEKYCTjk+\\\\\\\/Yuxtet4hThL\\\\\\\/VJw4AuRWOP22qn8J5Ttm7cZF6BwSRrT9r6wFRw6qt487dfMxff\\\\\\\/ZJsx+dlvWknd3VK9glVNBWyGGTTW2A+cvr+uYu57cbhad9nH2AaTTdD2+kntxIEM7nyXVwW+DgDuijwZrHmgotqC+t4cTGfUjd5Ojq3NVnXpXSZ6JkDKqY+eqdULIoCdCZlQtuSvjHlMsmxwqxYG5+AApZDERxOa6AGDeloA38RN0kyj3+fPI3fNuapKfvJr\\\\\\\/ebLmQ5b7IyA\\\\\\\/ANAOEm6uK8b6M39sSGkvX9qPOZEp6Cvy5awkrAL+1Jf24L593uF+ax2ugrOmbKPB4N7D9c0jrBCgeUmiSHo4d+\\\\\\\/bfmJQIeN5A3JLkZsmugx\\\\\\\/cKVPyOm2fJHu9Yck9PJQHC1xw\\\\\\\/qqb45Thkln06rIFg0IbmMoqQPKu2AP3ZsMFv5O+plZGUfCDjt7M7xtTCpwE91AMdDLj5RqWppmuJJbe9ShpJJi8poRkMwbC8yQQCcK8L+M7w2evFAvp7qHnsnzudx0HMK24Uj0Kei\\\\\\\/mdXXQtK\\\\\\\/UyhxUyGGSVeGY3hDnrYd4Re6vcWkTNXEclG\\\\\\\/0fqcfR38xfFFX6rM0pmxz6fkQzm6t0t6Ow4L8qEXz9Y1P7viVa6rOhtTB7Px5iKTy\\\\\\\/z72zKorzaWJBpUfpdBB0KeKwVaI8VZjxxDs3mfywoub5nNtxAEA9n0mHbq3sRR9Sb9ctoSgGaDVnnytIo1XNtGcRdd8cK5FxD7gcXDKdOfvwaPs67AixYOqtfMQ4ki9qa7F5Sbr30ssykdAPHyBoUwbdKC1+QHUTGZqLd1kIJ1+EGPsvXV2jaZxqgapfa9Ekmp8YY7Q5VRGS6MVWbYdJ1a+NiFuxQ6sDj9UNzK905C0SmqLKSD\\\\\\\/iF3F0OT0hYmEZtXm08X5R2Qsnf88nathoLq3zFZwm8ob2c4BNNzO33Hc4haBV4nKj1f6AGM9uCamuWCh\\\\\\\/tok55O6LxJPYIzc2vrzoIkQG2+GvflZmZXeyG+zqurd0alUukUXhCMi3ELFQ5o5gLMInTYj\\\\\\\/1WTKt\\\\\\\/W7moQ8tRuDUh9U0isdAuhG+4brRyR5Ou01fSfwHz52jMP7+k6BtsIGbaRewcJsKMvA6lfwgSvm0QmJ6obpMX+6KfNctxQUD6eAxd2VfgUNLMI1hAjENUFPtbdRggoAHasMY3klXU\\\\\\\/8WyPdrSPKZEB\\\\\\\/JfXACWyt3YGVlI7pm2U8s9iPJH4wSXGcyMAz7DHcQcsi\\\\\\\/tmx7IvI9815qp9OzrgjmJnB2tiPCWPES9xXdCN5OMpHb0i6R5EDyEDtYw4iqzcl5XH7ZLUvl2kp5aWcDjv5MIoCKNAasxNpGj5htJscrXwysRZTWU3uaQ1R3xOkRzfLkXPt0BumsFBXZso8ebDKezRDiGDyKALkxYAiVMXkMzY\\\\\\\/gDa0tTblqfPstfUtbto6OL3Q5vFyfrEj4M7vmnw\\\\\\\/caA0JeP\\\\\\\/LZXuvGMwXy\\\\\\\/3qxf4zMy4Klr3\\\\\\\/3VvaGiPcff5i8PBkqI+7f50K17xkoPenZlYUZnmBWQi79fh3w0I9evaY7PVkHBY6AYvWKi2RVLC2fnOyGk0QEV\\\\\\\/bAtaatsLZLFMmjy3BWsPQq3B5CgnIMg6OszmhRAdOnH\\\\\\\/NcQ41GINN1xeEJKptVYLQtzmFlpLdOyI7bIPXYEbfO6vMDpNmQ8Chx2xApd2lrUw2zuNnG01QorQsDO4aFtjmqmvhXdbU7SdAbBkNvUUNsGaUJp6HD\\\\\\\/oh8yKC4h+LJsCxpvO+f059TXny6wkKbR642M9FzIjl\\\\\\\/cc\\\\\\\/buNnW\\\\\\\/cDgd0U2SsNVi9MX0lzWnK7elzuGuO8AGL4Rsr\\\\\\\/Pl7WLLJOrcZPcglBX9\\\\\\\/2Zn0kMroOG3wuj3SkJiWp1gex2vvx3Dw3wq4kj4vHzAbD6JzrodGv96kPXEcGQv3r5M3rCTMDA6eacJp\\\\\\\/W9cHtNssa0SULqiCHxZRlraaTKqqYNIxsTs3uQROSaQyOd13bDk9RS4PwM0lGP\\\\\\\/68ZzGghlCb4cCsZSrONPYi\\\\\\\/WwxrK7Bf6s45ZTkQfkhiLZ3zOX89Niaa\\\\\\\/2P8EUZ4OsE7Qu01NUGQ5zdJXhn80RWEJuIGwX2phVTgI41HbOtoOfOX5cGtPfKYa\\\\\\\/y8JNTOcegFdF+g9qFU8EfhuzfkW51YlbLmucmP+rSYqrfw64A2E7YaK\\\\\\\/I30KwDWpHnnYzpmNwGPQ7a7NBpMrZcqfar+\\\\\\\/6xgGhTmUnOhh\\\\\\\/TB+VtFdQQ7YOffqKXjbBQMwmdokZdvGmfItVDVA1\\\\\\\/UKJuz\\\\\\\/UHdoRLP5w+y7Q0IkxIhWGTNTAXMooy4XINLLVuSTN2QqJqeKqklZbR2oPVUv82NNXV8k0YrTMh3RztC28WgsP2zTsY4Ta7g8zINWGPsjvUP\\\\\\\/5nHNzFU\\\\\\\/DYtWR9\\\\\\\/JQ+t0NUCzWSAUzBg92kvSUYn9ZqoWnHarJb6ulrmZTWVduz5MFmrMIe1dr\\\\\\\/FPY1UcTcY3WOv71Vb4ODIykY3AeURTT9D7eGiDghn\\\\\\\/H02WHoA22QpEdit6MUrGP5K\\\\\\\/ZPEt5+SzPKwSdSFd0EXAKaJRlbz+WD5ST99Cuplcfcd0cK08PbaOs+4qUzEgqWEvQc\\\\\\\/fkF7gKXNgYCpI6IQgCuKG0KV71dVYMln59eobP+HzUXfWbS70+Ha39LiaA1hw32KN8vjyWXusLVt14Y8O4D8SqiiN\\\\\\\/ebnMp4vPhbCi8N6AahS8mnEa3aTZdoTrjZwkQ3\\\\\\\/+esb4zpEAo+ffy6gtL79dI88NnbLK6Y+3\\\\\\\/XTtEH7efZLxNPcOzqeDIlZyV\\\\\\\/ZOMOJSodsRPq3G6bfQzoqL1dtrih3PnDUW0LGr2TMZr9nFXJ8yYT1NtbYq76VjfAEhFsy6GBf\\\\\\\/fqOyG7qegcT6oiAKfl66BTf8q+wk2bUGwtGgEEaI88GNgHVQaVEpMMoYsmhfePAj\\\\\\\/NvtYjduz1lIR2HItZeZ5V7h+SycACVN3bAGiSxtbWWGxpoeEazndJt45o8upfMa072xs7JFeqsCrZr5UL1+cMzbxnJbjSw2isocIddoQXDymTb4j4wBxhSZIwJpkO6IXwCsuEAa0tatY0yLrbr7cp5Ho5hd+MWRBFc1lTPBvrTZyTVrAb\\\\\\\/4lsg2y8nSb4BKJMvPbYQTwu84zNm1h3iZG7gn9ukf5TiYZ36n+RDTFRa818H+\\\\\\\/5bJa13+Ri3SRaEmQylakVTcap2fjDx3HwiV70ey8S\\\\\\\/OtXRKG\\\\\\\/jHGtQ+Tl7+1bVtCjdJ1lgR23nPSwwYs+KZSyE9Z69oSVyCJE0hTcVMsPSYoG0bPtjG\\\\\\\/XIxm9xkbfx\\\\\\\/WdmrjCZiz\\\\\\\/XuIRoeLM6jkQkxK8OVIC2pSh7QSoXFEG2jzQO5JcCivh7ZJeIvM9P9+eVNx15J4ZcbFgORZD9kMKQWx1PQh9GdTfXOOBeCd94TE+HmLJyhcf1Xzjsbqn2Hni7YLiL5wljHreLVBD61gLJ2e3IuZUAjQSVk6SHmaZy4McEWjKgfczlvnFbD9VFVjRYgQj3j\\\\\\\/H0ijccfo7qayZ9pWvnBjQircNvH4TZpVzm7WQDYiu15TgKCAGWtuNf5XpVQtCehDn7JS\\\\\\\/S\\\\\\\/6kWUf0NsIzigt0PRoOwYTUcg6tAxJEnBw4xK6xeMdOjwoZmNYS2oifsvbux4CHtItUxLWVnnHQshlDkOJjp0u+p1xh6KBcgmL92Uje+Y4bpQouxT4ENby98TxTIVfo4PTuY5qYwJZz2SxHg87RdwJDLZqQ5rWIUgnvMdmyDuRiaQdo\\\\\\\/swyVTS+CjVH8uUnfyJ0Qvl\\\\\\\/uEHR1Tomt06eRDyKegLdgs5eSRWYwkMOk0aqTIrg1D+iy1SSq1ZBuBOkNADphQsDyO1EavyMw3+Nl4Tn4f6qeUuSBUEuLsSpDWKan3eDoEstLZmCrErxhOrgxifk1ypCdMqwafoPi1CBYYpJo2PzjkfQ3Pms\\\\\\\/+\\\\\\\/VULlcUOSPzQCnmtHPOISS4ddDV0ta2vuucXfK3wAjhKaU5Vt1RLCdHtjIAqNabVX71f6p6SMz8hjokmecoBp1uv1SpOAh+2LkEbQnyL+Rmy4eyuWv43ELBh2V1mjCfS1ihm3ymDWZMgoIYtRaOJUEGVEVJeKM3AE8llgXRdjywFxiiTbULTXWP0J8jNwUBtVmf1qTbeDCh85SZ2P4Tl6JdzuM\\\\\\\/QU5AzrBU7KZnVXo6igwaMAkGfDoeQlPU3hwVJdFBMzL8pitGRja5MUoiJKftNiFTzZVIbud31dm36HaMMADQKSjnHdqVaiGve42sm9HEj3WS4mcbjfKtkJZFc\\\\\\\/rb5+5QPkFLMAjSSRUzAqGlGDkswRNVFYiuehe1JZYHQnaFB3+Aay6ZinGKJZsQ3eqsi0dyT\\\\\\\/RyB5wW7MazJj\\\\\\\/xH3yvO9P3X7Bo+F7H6pCZEL0dQ2LgN4FKSuK9rmyiTgrLaAaNowZBw1RGBmPvP7mU206ERcNf5pjBQRCNyp+rNCCuPPhzd7rVbkKChrd0JwnfI6NTABiOOUvwKBFWEFwE8DhJLFYLg\\\\\\\/Kkn+JIpvIe7oTBvANkU15uVsnNc4yodrzyO25Ij1SNXvy9jhTrXWMWH+WKxbAMQA4K2xAtzKgRZ89LGlDiBz2nUTfqUCvT\\\\\\\/0QxNRJbl1frx4OKE79YPJyVFo9xPio79NXXd35XdNQ7mjBhTlwAub6cuaREqdW+isiFr4Ow06SyGGed8pZ+j3hV1aamXOjcktJ1t100NK1fv1kZ13uIlqBPWwtZp+aMBVnqXYFYGv4aoEYHNoramtpzkqtb0yU90W2SdWcjflh09gTIkSKXiS+RwPI3sJ9CFMZZGe5sbDzikPVqLnq09jEFOmFXhp\\\\\\\/yE7yiQ0XxC2ixlQw7EVtiBD7PsKJ3TRi+F8IGgfN8cAbviiyayylO4b6rMav9lBbxPli+\\\\\\\/KPrNz0MRnJdiOe3WODgiF1lDVEXwVVkBbIHOyNCs+56vi0oSUkHDFG9Y9ejP60XH0wl9jVgiU1BQB6jwTBc67ff1a6bGf1ytSJMdaQfUthd\\\\\\\/D8VBR3QJgHo67AqwQstDQqJSdsjCCMXRkw8KXQtPtU\\\\\\\/k9iNNgPnof3zxe8g71LlPqSlAfCnmvlM2HngyXni++c+WGEFQXhuUr07n2MszrUsqr+9hC2DvowmXmAFX3yiQioFSckbuzt0Xxs72rLS2CK\\\\\\\/0Vw144XLTxArfxLBGPGFd1LGy\\\\\\\/u4tH0EWWvPPee5xubaL4VeBLgU0jJFwVc0Vw\\\\\\\/ZwnKZEgGiLgYV29AAl2KUd42o5y1LHbwp\\\\\\\/gqQMboswYp7GvxVOtHSObqFkabfVbQvrE5NRVQiJR\\\\\\\/YrFQIfc9HzzKvIKQ1xCADvZ\\\\\\\/vqf3F488rZ3SekZD1zDe3\\\\\\\/k9x+5GSj9z7Y9b1b7TWfug8+po39bIQHVfFPMaDeEy6efkMCXUoG8odxg+Ms0BNHU8rvRXST0SALCdVIZl\\\\\\\/YuB8ch062kTCg0fT+kCEYF2y67PEHRZLCueJu+gDpKFxedKfSeQ6T5beGe3xXcKSEM+t+1GYHhT52U89U\\\\\\\/fws0NgieOzK0bt9Idfo7RAbuqG21v0j7mRaAnMkzmhLG+2Lw44MPSRDDvlTLZWLg\\\\\\\/u0mAu+uYu8jbDZzEBnIEn6O4wHrA0tucPVfQc2YbzPTPEAR\\\\\\\/YZd2Vzr1duCgtGrct6WY9c6oF22RavHxiyuJ65fgJ2u3DVto37JYyaZgjvRha\\\\\\\/s3WrpyYMTH\\\\\\\/NqIx0NCFXVM1BqDHl+nULjQy4ETiYJDp77+\\\\\\\/T7CoybrD5A+oCXMAj3VEDD5IrjnEPJEueUI1hN0X6fvzk0NvPN1bNVSw2D6wLK\\\\\\\/x33qyB5VzHEzcOkyxx86mqRqQ7Tl0EoxzSt8CWys6tO3PYcQ7rjYENAmSC+j4FQZeIiXUlJj4lvhtDPi0h\\\\\\\/ddpMvJkc95doZlfd+0sSu\\\\\\\/Hs8kOfDk\\\\\\\/dFmb1h6yB1b600EG+m3aTYdrPVuJqsJ+5DkyJ\\\\\\\/iTInM\\\\\\\/\\\\\\\/jZgQqx2Yaot2hLNnyFNSMyJ3U6vD5fhfp1Ip+lGLGKD8dXx6pazfmbrIJlloPipiDmtN9S95Wdfxu\\\\\\\/coI7sPq0Gmlwfle+FdRBIyYL+2\\\\\\\/8sPNdjETipuxZxfvc\\\\\\\/p+BpvvjDGFrMcDJozAAMpdXnbn62Du1Rs13azOl32NGzuQw7yf\\\\\\\/uQ4Lc3NolOwqpXVrvRKLAezU9lqRXVdSMv9\\\\\\\/uU\\\\\\\/wFepQ5ehSJUpTWmOZJqsLagBGPKmOPrF+KgNOBl966LpLhUBGYVTqI+YI+efjQXBDlFpakNE2Qeto603zVrMsZRUCBSZGRSD0RNw4pnp+1gt2PmXni1afxglgD6em+6JuDpE2uEJ2tGgnbw\\\\\\\/YVhajF9i1T8+9G+nE1oUm9dfiMR3\\\\\\\/hPCv8a\\\\\\\/NRSFQ+2c+V7T5E8viZM9K065qvmfA0Hjh+acSq7wecWxT\\\\\\\/jIHSbF4vrsL0Estk5bNCHn6aru6Y\\\\\\\/oijfTjOiO\\\\\\\/8vZ6HqNWtFmqJiVvVW113Z5BmbtyGsLH4U9j07qT\\\\\\\/WGEK1k1PLATar18W2mISWxkeUFx+auw9k3kPYGHTGUelHbstnTu+Vkz6\\\\\\\/VfHMg0u0uU\\\\\\\/By4t5P29IQrbNdYiJWT1aprSY1nLEY6YMuEtAK5kDFjSqQJSfwgz2X+uRjgExUL0Z\\\\\\\/LYZkvq\\\\\\\/FkypID38RVS+bbX89RWVJ8rv+KgH+vhmr8kBvXh+rmnSdW2ieqMZM31clz3xzmwl8iU3Xn4HMXNxyUdmTbVqCCLWqDfFB459BhBOwiXUUirysIps7VIT+l3Shlxh+XxFoFIcW+P8I3Izo56WtuoKrMhYuaVo4dMkaTrFCO2x8xBR1MeCOHMiRMlByu7vOrNL6AFfAYPCj9FDVoq6eySPD3k9\\\\\\\/k7BLKV7xrYO8HJJQPd\\\\\\\/VMDQ4YVgl+10KJzgIUM9w+lP0SlzXzmEUFihRBjkpOxNXMOF1HljWq+2fmI0h9JA+eyuDX\\\\\\\/pXwRW88C8rA+yVwjhOVTIJrzoGAkZs4sGKN+MjhStKihAGxFEfUAkQ4yLoRHvql4ZYfljfMhbiQfaz8vanFLZZ3QoKezZ0UczlQ3tsu8lNBrPUbfmuUUit7QzsOl+3AqoP73ELsmsOkS1GESL8lR\\\\\\\/wROSEXuxl+AQdOpCCUgSP+Zen0kyHZo\\\\\\\/ccuvXkbwBbZnDsOxNKFRAZrA4\\\\\\\/3qUNTRBA0ppYFu6IgknQEo\\\\\\\/80lEb4l8Q2w5iz3ebSuU\\\\\\\/r\\\\\\\/slzVn0ZwEUntyi0fdOYYrpbmUguXnRUOWr8ssTZnDXhbanQuKPUP0\\\\\\\/VYnB6nDN5f8qnuAjp3AL6Afw4FBLc4vizfwlABCG8FSo1Z7GA+ac9olHSd7\\\\\\\/BosQDCq8H8zlmA6DNeKRktMh3g7pN6iliC4U6vQ6QwwQ7rgeIZFlLgD92n2sjckyfyVo9BIF3ToZcYIdiJdeAbkNen+Nk5Fe6ajC2iH5aQ3myB7qrWj00Hn\\\\\\\/xWjktHrs6Z1N3HaQlcIcduPjSp58ZUzNc+2MTf3Tgo+\\\\\\\/0sRyrqawCI\\\\\\\/lWRRCxvhE+5dGFZxVgN+H2y3GUowk62A2UmilhtiSLOOMUlUSwNF09R6R7K0O6MCTSCT5995yuIiiQmODzWqZQ0HYv+YQX\\\\\\\/AnttLsgllzzPablfX7M7J6Y29\\\\\\\/MNxR+EMMjHNmOP2yBSfMbGK\\\\\\\/V08G8dSEWpRBwZ\\\\\\\/XTV29IrYd3Yl2OcAEY2eL8CZfToZ4thK6f1srCXyNRigBy7D53x1aGxITfDX7MRdRctQaEa\\\\\\\/uP0b8dUDddzezc8TkhW0zURoLeJwz3PGQ3ROJX75Q99V\\\\\\\/7ijsoggRcFwdFeDqOCkqVupC6ejqC1n+bhq9AZcgtT8Ief\\\\\\\/oli3DTlZDVVp1XdvFwXdH3FdJ9+doPGW2bQUrI3k7CkPVxHqgJRmzNYdsVd1e7Ha3aQXSXitgeCY2XTiW84\\\\\\\/I6SnhJprKUR70CaPzZAlwmfgqP6JzO6IaFAWLx2disTc\\\\\\\/Ubd\\\\\\\/VYKNhuSQKcztwVSNSEtu2p36GRAIqOJQ0IZWimYR6VHbXzWMxZOXAQNQVG9RnYrs9T4tpc7tCZyKMBsHdv7uGuBYEVcYUCc1AOfT2agjM2q11WtMd2w9wYWzYgSYrhJ+Q\\\\\\\/7BjklM+s6w2FTSuB6uAejz\\\\\\\/c7TOwxlNzAsTjJHfT\\\\\\\/ewBLAhXDGfLT9il1bR4w+2UUxkGlZkCgJmJ6zA3BvVA3UD7v0jEDs96RVT3uXm\\\\\\\/z9yr4u3TE7QHkVlAEqWtE8nRCgDxI8QkPXs\\\\\\\/hKaXqirJM7pZa7pN1q8Su9w+iBGR\\\\\\\/z3leYppGgPnek2TAV8UsGTCV7mDGjuPUm6UfU88dwKgntRJSl1y9azCt+1OleNB08DxKh\\\\\\\/eXQEDM7pQToFNDiMRQ8byb+zRq+RjapkViLPF6OZVZ551tTnmo5lrWXASVZ6xNen7fNSofvXrtqV9MQO6lXaV6nUMhJ5g8Fy4CahfwW2\\\\\\\/26nTECEtcq8ceM6x7dgCZ2ZdVHTNHdQDFMH4souif7H7HbpVRvpUhJONbQlyfQew7taMJwt8jIVSJtWYazVZgg83kvI4Q2QN\\\\\\\/va2SFBtyE1AIm8JMoFeUuldzmjDPBIrKS2M3rKOoS2YKls1o3opXGdmPM8y5yourmheb\\\\\\\/xnJ5HHoef58zh845LT9+xLhGMIoAgV3wRpQdA5k3DjFaeX7O24eb+96jdPGBlKCAqxTHG5oogYCzZd2RmiZfFGQlvpyNi98QzfIk\\\\\\\/AeHw1nf71IxS1UcfwI5K3HBQ\\\\\\\/Q96\\\\\\\/B9E9RYiZms8+KAorhRxO8Fdma4r+BN9PVZjPs2zB64oMQadbK7STqWgvhK73s+BB8QPBQCBBWipM+LXWOui0XvGkAiTbjYHOW\\\\\\\/4+7aS2kYeemab6d4FBnZ6um8JD3zgMNTdtptwZaUf4S\\\\\\\/LOX2IaAu2YQhcB1PdMAyI9RvFGjKjqncwRueaLyLU9lQuaWYUhWEB8QoFH+zJakq94QzRgYBIoMkEFjKwQbJopcDOuDV3\\\\\\\/NNx2JHI+N6NK71noHIAQP2fXlra5nMXwf618OpZ6p7bX2Ol3Vymg5XLTIDh4EhwUEr69fpaQXkUQvtYEEttthpi0dD\\\\\\\/aEtUztS\\\\\\\/d7VDI6diXhmTOXD0qiICYy+ct\\\\\\\/kUCB2LwvQfkFRsePh6CGmdXga0oXVKgeO\\\\\\\/vlOfn1\\\\\\\/sgMNxu11y7+f1xIJcvk8AsvXdgNZCV8svJfMbxKh1ZRHlbRscW9epTNoci3pX8LhEkjQvDwCBw8j6mXV3p3np0zaLqHSopMNH36EiDTVDJ8Z6BpAG2xf1eg+B9Bxi3U\\\\\\\/fnWUjrE0hxdBbGdRYYmFmCof7N8bgzzLZGLuplG5mByXP1RQEI4HY3jlbAWlnJr6XL5x4pFHWWD\\\\\\\/75e0iYx6CWGCMb\\\\\\\/y16k\\\\\\\/FNuMsD+Ft2CFjvFuh12a0wZ6vCyfoasgG9F55epFDpJAsQLS76rRTn7\\\\\\\/NUSlzGJ57lhrAJ\\\\\\\/clSzGRlJRsAGtXNd0o28a1tSrDcIcCNGqTtA4k639LrtLfTHepglUNa5czNizWmFngY4m6Q4b1nCZT+VuzKJgpnwgCbQQ0VbEsgnOsmKHroWoJYo32K0XdpAG+l\\\\\\\/jaGPDdIDqOR0CvniujcMfTgl8NUc8yUEUFlor4wLiIVx3GDOyIcYsBSK0fIZhRuGK1yviu5UMS6+WHWd93iObjJAjk5Ve08F4Wo5lrVx6Cv0dYiyjhpAo1t2GA18NqHUhRyo+PV9UNG+fkZg5yttiTDMTHaZxpd0dq7HejNXUgLSg0rHtEaGwSujXG+COe\\\\\\\/NIRaopzhE2++ZWtuGkl+PSbCiG++WNU6rYAIJoDq4jbfODVXpjbo8bG\\\\\\\/7\\\\\\\/66PoAqa6OG+d1QgcMjp59IhE8WodzqAawpzsEk72O+bRj3ZB0RixQjGJzw4AeNQWRiEY2ZdzKpVBoxCROZSlDdESH9RaQ3F9r7TnLm7z990ibkIyClGUvSDNEs4wwLrwKZC44KneAldeRur1Pinr39wYmZOFLukya\\\\\\\/UKKJjaJUSpozJDzv\\\\\\\/ukjcFquCIYaYs5hOyFozXm7F2VW\\\\\\\/rBUgW5xPb8IdCguF3nkMwXxzu3dk\\\\\\\/3SuGInkBya2SiaqIMwbj73x8G39H49UReUNv2vAFSPGW+\\\\\\\/R8Y1a+9wrLg3\\\\\\\/2yE4dWKXEOGL6S1gSXmDAMVMNC2kkSZA0xaZm8ITYezRVb7LhIGIPEB5eyqrRXGFeyroLPgYolrGApMvNLEJPWElXfCcXTXAEY4rZGPetguQs6\\\\\\\/0s5UwFhobrHG0vt2eYm\\\\\\\/y69h6BgTQyw4epb023\\\\\\\/JHc5WDLoCvQZFUv4CQMTZQ77qp89Pu0SQoJw4k82Xg5BFiB+xdj909R7JiAQK9J9tImVJ1rM8fMf\\\\\\\/iWwZr70hG\\\\\\\/tpBA0GzPpWS2SXolRULR7oFecmPAGNtAhOhP\\\\\\\/t6CmuZmATSusS4haNrVKz571N0ZGawY0V\\\\\\\/MK+VtgLjbQYYv13wblicOwZ0WSiq2dHf\\\\\\\/1yTl4Yaf8I5JeIh9ZZkJpA0G4J\\\\\\\/IknQrAssy3WkRM+sbCnS8rUTHtX2HPoJsL6qdNdeHT8f7RY6OjDgm1WjRX87lSRtj16d2cTzlipLtF3dNR9opEKQfCfs8bimFtdniyLz890OMV18n47wAxW8X42w4vxzFGumkdLs+dOrkVr+KBge9QXyirSFNAlqEaTVs+7EddUmHvFLJSIC6sq3oy87NBBw1g\\\\\\\/qaDmEBala7veDTCnRrbPYuIGiCgz+skowR0JfuQV25q3S+GggtuwL53jQs6YoKZ+D8Br5jmsjwLmGzKWq2gsITBTxknwWjHS\\\\\\\/mwz4U4uK88NVoousTlcyaiKMo9o+r1DqejQ42\\\\\\\/DruJ+vsDAL39z2DxuitIUuC0K10hf3X0MZi4dd+js2rMpLQlgyQt0hqml6Ny+37Whs1Fe9RrkkGFVme6KRQH7uKSvzy0PZDth8fq8IUymA7AldePuzH622Aa8+dM7oh+s8ENmSNCzcHdDSjgg8QD6TYi2jUtzJRy9D8ktDKqHGa1r9IkNDLPv9DkQolGOEAejVmfR\\\\\\\/aOQ23thJxUtNw3G2Y2BqwsHbbtONLSiSTgKy8dB5HfcPbAc4v5MXIKq9mXzsEGHmBahK2BqLWvkmbDPDQKVr5A9eYz1T0koEb0HoRHltUEw+uhrfMr100U6VJ9okp79OQmS3AHvJ7eLPGqcRErLhuaMVnypgUo17sL5Ed5Rxlfom7978+u0zUmDYeocEUUNN0PXmGqUseXAs7P8nofrjfV5LcfU\\\\\\\/x5+KX6ZlRg1+Thv4h0wncTojBMwE2X\\\\\\\/AOFN2MLKfo6UoPbpaZqagUEQ1b5SKGDmw\\\\\\\/5\\\\\\\/nEEU56XGyH3d6z\\\\\\\/IEcP9xPEFhBgD52SDez0nqEmsLb9n5jAEKUkzOuM4upUzl4Z81r8dpT+EXF9cucScrWWeKjl38zs3wxKJRnooV8CVFRo83OT28gxuEdeoyidpti8AwNdY5AowWg6sqGe25rKHUbhWFWH\\\\\\\/Z36lmkqEJvFkyLu62TIgmJGiEvrBTKqmOyre8s4xJLN+8ldwm\\\\\\\/W0hvp8xTZX2KqcJvVmn89G4xCHppCDRythkP8M6v\\\\\\\/HuwIBthqrMta8Q8WWvyrBAO5uxbQJkb0QlVrINFYu0ogw6efH9IUvXbZdYQt7\\\\\\\/Efu5GytM0hROAFQP+iBUwHKqQxhZNslwEnsPzuKaRZATAGpJu1Nxo1SyMwmjfVJSQFjAMJPFiLWIZ3fKSbR\\\\\\\/zX8rb3nOvbFkwt1hiEKmESOTNpMd3EpdqTrB0DGcxYNAERWiBMpc\\\\\\\/1flU25NOD+EZb22O4XBahxCnaueWlaxliUDiDA6WAj7DqnTykFFd7ni\\\\\\\/CmQfpPvVspcvQix4dKESx9jVRxaw4d93QkgWU3WD45ntKGMmwejcPyPDe+++Ofb\\\\\\\/zI0\\\\\\\/wNZ+kcWrpZAnXdULFYt1ij\\\\\\\/ZvBK2Ur0fzFM+tX6W1\\\\\\\/6NmguLeCDTPUrhMH4i1iZ2w1RGP22Kbg3oq42HKiAFmrMXAQHvY9sbvvxOxRY\\\\\\\/XNpc+0lcNgEEvqT8laiqb4\\\\\\\/UdF7hUjI9\\\\\\\/QWN65ZxMaYkv6c26Xt6Nm+IFKwZjn3kK9vL62a+aMycwMstLIaPEbyofWJNOGZk3IWFz4eonYc\\\\\\\/CbZqC9OxQdgxB+sooyiAiyS013a2yAfJ0BWecZZRHGCYQYTBK7SrtOmmIIKD2rFoi9dN4yCEWO6CS+DQtN2w+n8QCccJXTJ6UbsuO8CEc4C5OXbENm+x5kuX0XvBm7KV+AHeODINuh1EZl\\\\\\\/NT3o3G1yLcs+HTP6UZ\\\\\\\/iDBE3vUtL6imnDbqhl7J2LVHIWiB9K7IF6H6fF9FMYoj4CWxnEiJ420tgNfKVaDdhmW5cioE0x4ydv8CiA6vv8B0nq9Gf\\\\\\\/3LtwcIThNfaetHoJg6Abg7tCcUL4wTi4FEoWJ2HFKPCVFAy\\\\\\\/b593H8ya7aenPVuZfAQF+zJKVin0BYOlppIkTO00nhsF5YVqwQe15W1y7T8QMtYTwMNKn67a2clbi389mC6wRK2mKmTLywPJDYWBj4cywj+kEvBswGAdusw7estRxofyHKvabAbUhpvHwQQRFDXBEIVE+MFVKINKUaPK18yxegseZnzlYMeoQInS0n5gysBOUyYQCC0K44dxIDI7WDiiXapNw8m+Cld+GOt\\\\\\\/IAgcEnl1KMot92Tu4B+IQQ2dON11V4OSu28v1AXcaUT4z\\\\\\\/W0\\\\\\\/xx5WBBLA0ZAeL1FbtrLB9YxwKXOAApZqZXm6jJtRhRIS2xrANZmcVSZaeXFzehUO51ehBlOwss2wqXHKa1Q++2Unf+++QiXn+Ot0e1r7G4vXLE5m5bAQ9F0LfL4E5RidlmTJcpzoIRHmVlL+8uNXfe4Gv5X+6a1q51b8\\\\\\\/IYx1nRbG3ioAHsWzWOiUGc9es4SB0650zHIrARc3rHesdDo\\\\\\\/TX0138DdPzwSr9Ak2ZSarFRgDNG3FIF9kGrmqi8sEj+CZfpNP\\\\\\\/YeR+6E6T8zvo8nsHqNxj24\\\\\\\/SRhbHBj79oPx\\\\\\\/rRHam4sO+q5Up5SPCZtLzmxWfepAhfqvAO2htFH83+lDZQeJ7F01w5k6b4o0XfSmxnMWeOBNyktLc5sZUtm06MKGXKkjXrYlxlB6JZRv7OiJpqWnxnYOcQLih9q0mFrg8ATye\\\\\\\/73FfdQPztqaoKISflFVRwc\\\\\\\/rdwQ6bZHnjWXKNkuB5PrNP95t79\\\\\\\/39COdsX+ZCN9R+gFQ047dJD\\\\\\\/uv94adEHwdhca\\\\\\\/wp7a7K7yu8f7T5ktywKK04j3WYG\\\\\\\/nlp4GYKiz\\\\\\\/dCAszkdfUQiybTxGpBoAooA\\\\\\\/kEHPwYg757b9K5TzXuqVl5myJcc2iO6PNRMPlYpzQqPIJc4tJ6cYfaIbwkyjc8u6gVlgnZPixCb1j\\\\\\\/kyNakY1QjEnT515VjJUcQICNtxJrsBwDl9NqTdur47Bcky\\\\\\\/vElN+PzYxuaLGhXx5QG0OJ1yOKJuTB2JmRzusc+qHNSgmxzNTJ2jBvh7UPcHc\\\\\\\/TbFCT2JnC7qk\\\\\\\/GsnVAfYaUkTY5POb5M9hXgjpCDpjYiQyTwqWo4inHpfnoiBgFKVRS7ZJ6\\\\\\\/Wq3k0VJO5UJVFet\\\\\\\/xnl2gFTObt+8Gb0ULi+GFT+QtnqQAwdUYaRhUyMY6BR8BLv3OjEKqp08v0nj8vPPMhBUw0nw0jpQe5FyBwg4JS2aQct6pczDOS09zEX21WdIE7jlNywZkhkYEAFxUoNoZqneWsHMl0Jha4Zpfh0DZohVOw6S0mv2a7m4GvJVE\\\\\\\/axBuyx06yGuiWswyIi\\\\\\\/P3fLb2oDccP3bpZh1iDPARITHn63SYXOOS8cKpzGiwiNnpKgG8MpWSBG1hxUAt+\\\\\\\/9E8+2rSH9B0VpDyAvfOt+RGNDrxAkTTGB0fcN1wNGgxIgN2GUVv1\\\\\\\/FDphybaWekHUzu32dEtWSbfVTsW3uZxP3iacitrdPBYvlWOKn2uyiGMen8AJj5ZuZi18B7ixUB7NOw3nNU00+WQuyc0PHEV9fwviiks+Cxi8J4xacrM27sGt5GUwuIcV01QnqAPP+ZUfJfZaWWIcTXuTJ3WY3Y76RigYcqw8xlRFWvPZ6n0aoBZDZQHuQ1d3NlKJ2ZW0pyYFvN\\\\\\\/rHIhre719gLa64xRIfHmt24CVS5a6aLGlndb2s3PGrsZ5hMKiMVfA8yOcifk7lidUoZnQGlEgq0pfdcPuFfV\\\\\\\/G8MuTxGt2X7zrNteEgubPxvJ9QKhh6dNp+4uhh29i4RsAuDI7iKT5RuLMsDcBeWYDrnSGX9vtgZ8CJaJOo3nqzz5DTuQDtjBsN60TBIBkqacEiunuuK\\\\\\\/q\\\\\\\/ZvvOt4s87mzI8nIz+SNDfqsoQPbHLPzQLGPLcEGVvIWhO6TfhpXtCgQjNv++dOIy\\\\\\\/FdYuAR\\\\\\\/b7z0wHxkjt5xKc3tIKVqdED2WcxQGZdp7uMYgyvk1xkd4lIcMEILBlgBheVmt0Mu75ebyen5KPh+pjHsZc5sMXHLt12ylGYqCTGrp0c0TntW41Fwtu0AXgbOGuos7QUcKffFzbLd9Rb32yaPpHAeqaJxtV8R7ziHRM1skBYqm5ucn4ygOQAw2jLRYetZY4YFzEHqdxRfeH7lVQHsLkJop9NrZMAGrTzGl3MJ\\\\\\\/HFGv4DlAisL7lpIvqzxuQUsMYi6pFsPCgwA8DTyUDci8OK1XX5uP1NFzPe66ln2L6jqFnaZ3ZOfpg0lRuneU0VS0\\\\\\\/BfoW+Y4AA6R4LTnIlXR4IVs4zzUjfUDkeR\\\\\\\/rmbiu1hsuh5qSx\\\\\\\/98LCfTq7OBRQgT1Hw8Pux8MdRGRqeaHdEkoNzC8ULhoTxA04Uto0Lg5u8Lg9+iNPsDESuLhLGquig45l0htYD6ghCX9Ri0O81Xx12ewZj+Bcb2aK2bNCEW+uSqzOq5LqaM88Gz27JFR6H0mPGR8PquYlVrZNhA9re9ia4KpHtN3LRrSulK9vcKq8MWRuauDOOWkfOyXr1gcP1Wbgk2f7bq6FNhx39I3GGgLGIsFDc72jXG0O3wbXKY+23UgFLN6sAga4GwOMBfjXT807Ztx9ussBsdEPw7yiAnojlHCl9kbltptBJ\\\\\\\/z4\\\\\\\/9D\\\\\\\/3VrDi4TXnqvG1k2IoN9hh6T5TJOt4VPk0ImlnSy3rJgUD0L5gmn5PI0KJI1+QALyb\\\\\\\/g2h+MmfxcFPz\\\\\\\/Gwt7WXD3iERFm94DTqQJqKuQDLagCLplTgjjyNbjbU6P4\\\\\\\/sOdg+bny\\\\\\\/fuKiKLDdBdgZGLSRPZJLc6RmSQNivOqfctED9ENYkpwcmgc3TdEvCoh5PPUywEORYuf\\\\\\\/rViPdfUGh1QTIbSF8F+oxU7Xn9+wIzNoh34sqfAzy7aIxtBP4ZCSd\\\\\\\/COit40VaFU5\\\\\\\/mQPP8MxpUXXj3vGYbdymBic0mpMPpCUfktmyy\\\\\\\/WXsr5E4HrGPxhhFqwqnckKXFAiB+3Cnqa0uR4q54kY6WFv53jdb+hSsEf5LWdFXOiOOd6\\\\\\\/IEDru3BDpMDlzUOxyfKCKw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022aFNvGlMPkwHMWCcQX508YA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002291d9279ecb895f5ca82e99f3e488b9f477704d87aa31d21f1c90f21eb80ec5dc\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022MTtcpVtylPfMMD3UGnlR6MYLoop2O8w4oY9z34mYmFQL99k5iFiIRsxXOgOuZFfwb5h\\\\\\\/hiqTSaUIO5jV44bpWzsMs6AyrS1a8Hqy2SNc28zlDcUKa8EuymTt33AAPDX6d4T0\\\\\\\/y2A8+Mldt4mu2+1Ed6kmZp9MHoN3PxQkrsM10ZurvMUJhe2\\\\\\\/5QJb3dS\\\\\\\/\\\\\\\/4mGqpaWu4HbKq2XWSj4U6wFg\\\\\\\/jSxBZ\\\\\\\/JKgy6JdfuyRQVmHZINm0ugu9GE4NEFD4RQd8Z4HQP\\\\\\\/wr\\\\\\\/dccURvJg1KQjEHDLejRI6pZWYRkfihttAgrKNi73hJo2zTK\\\\\\\/4VVcrV7Qzukq1ylaUAoZ4l4s0vWrxGqUc53idneWcMjfwbMMXx7Sr0z76kWFjQ9ockIi0NVU\\\\\\\/gUuEkGLQLirHCEoe2IED5H9Xsy\\\\\\\/ZTewPc9gLxuQ2IxtYlFiPCzoDZBc\\\\\\\/zuh1VIoS86HlOxYlGQO8az\\\\\\\/mY\\\\\\\/3Dwy70t6pInKiFiwmiRrWMvBKA3K+YXrpUmgcvVTDVdh5JrvPxMovwK7m4f\\\\\\\/Fdz4n+yC6cxfRD4xHJTlZMN9aAir\\\\\\\/L\\\\\\\/bXZyH85RLEe957yZ0qOET3F0k6bupsMQzwdP3sqjlHuD1J7+z4yx72aIem+PZcoiFGt4xyt00R3SDnbHoGO\\\\\\\/U4KgwpvvFIW6yw7JkAp9xSJSnoJoFymiBts0jTEoNvlFQ9ZIPLqyXOwnMeQ2OfllKQZ\\\\\\\/PJnvtavRJVYp9eWG+gnilH9hTYKRjc1lDu6fPd+HkJUXIuSJvQrucosz+acaCff8we9lLJ0gqi4eZ78v7bQIgBNlRxSQHPbeYgA1zVIvPtKxTlwDHZvshem+RtT75BlLplswHOAuhZ9uxZRUipNGvP+qDo\\\\\\\/WeDIWIJEYtoj4uchvyCzauUFpPtvOvf21PqdCHdghXgn8a3rRJwQIW97Jdd8OQSGDDq+YAFL\\\\\\\/LPFeAmdRV1YoOE6BZqBNqotU8cWrULMm+Z25\\\\\\\/SC+mvE9AffxhzmzlyMyXC5vS1mLTdzMJ05HovZFMf6NVpofwarRvk9+7yupWbfuI7QKWsfTKMpzmXE82FjwlB4s4miUsuHBlPzQUqfARaTXPlffGSiOL2aGnUPxrL7Qc+\\\\\\\/uWr\\\\\\\/oPV30n6jfYr2HdAZTAHK5jqh0BwN68JgfvP4v4ytPhOyG6Xpwt44Sj8favyK2+ZdyP4D8yBRAVaLhirNhaEonklbCHWJvvSblgZ88AEoMZfX9KErP6UkJeb43kCniLkXodtaCrTu7YfpD0ZotWDU\\\\\\\/hho9lacxRNpuaXTlBc0U5R7sMAz\\\\\\\/o5j5cYwiRQeXN7UHynFjYkizppwqDbQu3fNqYDhztkgi\\\\\\\/TQ\\\\\\\/vXZAY89KFCC+hjarP7Vclq\\\\\\\/znHFCop13yVuICd3B8\\\\\\\/a5kuIfOraGVNivy0rPgRLaUcDhUqlT617g4sjA3yX1vcKlSEjqENTEIX5+Qc0A\\\\\\\/POWebYfHWK9CZ5VqP2wb53N6\\\\\\\/B2uP\\\\\\\/M5cUUgdIYeAc1yMqnITt79tmZN0i6LUX01eDpE+i\\\\\\\/SE45TQp0bdOatAmpbYJHZXo\\\\\\\/Z1vBw3MdAZ5JSjpG52evAhcvCAssveK9Pm5FrB\\\\\\\/t84Gl49RvV25cZ3ZP7TaZXlLY1PRd0ggTYGfvfV6SOoI3vhNv2jHuP8cR+AO9fv4zW8ekWtrMIYkchZdF8ImSXZ3NUvQJf2nJW5BmucWeQqviCbAyHG\\\\\\\/u21t61hJyv7o5S\\\\\\\/89biq1GHLZAQEUNRBiVyZcYe1QRsx\\\\\\\/uL6K0pVTqA9ntRCdOewW73Frp5BhE+aPTl3zsHrXhJKnXY27kYz3+E79JLD7RgD8VkvswUoeKyUXDwt2SAAA\\\\\\\/0dx8FB7oup6sHUfPWa+IbUhWIq+SEkcTKTIhMN16XWyKtotFDJrR5AJk9fjknnL6AWvTm33TXwFo4FWHpSpkn6L1VF9xuV6qbWkION4uihp7WIryOIgoS9H9YQsWC2yZwDltRFrYPhfgApod9+eosa\\\\\\\/Ndmv5CNBRc1ppe2VAH2wEk6tuTt\\\\\\\/MpqxVw==\\\u0022,\\\u0022iv\\\u0022:\\\u00223eMHILhWaXzCoEJcJg6Geg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a9b3e7d02653bee8f46482806c688cc45dfe6aa91f727b6002acff3628a89aea\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-484256 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1499" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ZZ34jMBtVEYvtTftbM-n90bx-uiGG5R2KWGXRqdbrSM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1499" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-cj94Utc7rxaiXbomXBFjvKfBEGoDThArunuBwaNGmfA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/5/1487" title="Paternal Induction of Hybrid Dysgenesis in Drosophila melanogaster Is Weakly Correlated with Both P-Element and hobo Element Dosage" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/5/1511" title="Modulating Crossover Frequency and Interference for Obligate Crossovers in Saccharomyces cerevisiae Meiosis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484256" id="node484256" data-pisa="ggg;7/5/1499" data-pisa-master="ggg;g3.117.040550" data-apath="/ggg/7/5/1499.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Evaluating Methods of Updating Training Data in Long-Term Genomewide Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jeffrey L. Neyhart</span>, <span class="highwire-citation-author" data-delta="1">Tyler Tiede</span>, <span class="highwire-citation-author" data-delta="2">Aaron J. Lorenz</span> and <span class="highwire-citation-author" data-delta="3">Kevin P. Smith</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1499-1510;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.040550 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484256" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jeffrey L. Neyhart </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BL.%2BNeyhart%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neyhart%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeffrey%2BL.%2BNeyhart%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tyler Tiede </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tyler%2BTiede%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tiede%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATyler%2BTiede%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Aaron J. Lorenz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BJ.%2BLorenz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lorenz%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAaron%2BJ.%2BLorenz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kevin P. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BP.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20KP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKevin%2BP.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:smith376@umn.edu" class="" data-icon-position="" data-hide-link-title="0">smith376@umn.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/5/1499" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:484256" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:484256/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1499.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:484256" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:484256/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1499.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:484256" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:484256/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1499.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:484256" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:484256/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/5/1499.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:484256" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:484256/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genomewide selection is hailed for its ability to facilitate greater genetic gains per unit time. Over breeding cycles, the requisite linkage disequilibrium (LD) between quantitative trait loci and markers is expected to change as a result of recombination, selection, and drift, leading to a decay in prediction accuracy. Previous research has identified the need to update the training population using data that may capture new LD generated over breeding cycles; however, optimal methods of updating have not been explored. In a barley (<em>Hordeum vulgare</em> L.) breeding simulation experiment, we examined prediction accuracy and response to selection when updating the training population each cycle with the best predicted lines, the worst predicted lines, both the best and worst predicted lines, random lines, criterion-selected lines, or no lines. In the short term, we found that updating with the best predicted lines or the best and worst predicted lines resulted in high prediction accuracy and genetic gain, but in the long term, all methods (besides not updating) performed similarly. We also examined the impact of including all data in the training population or only the most recent data. Though patterns among update methods were similar, using a smaller but more recent training population provided a slight advantage in prediction accuracy and genetic gain. In an actual breeding program, a breeder might desire to gather phenotypic data on lines predicted to be the best, perhaps to evaluate possible cultivars. Therefore, our results suggest that an optimal method of updating the training population is also very practical.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/simulation" class="hw-term hw-article-keyword hw-article-keyword-simulation" rel="nofollow">simulation</a></li><li class="kwd"><a href="/keyword/training-population" class="hw-term hw-article-keyword hw-article-keyword-training-population" rel="nofollow">training population</a></li><li class="kwd"><a href="/keyword/optimization" class="hw-term hw-article-keyword hw-article-keyword-optimization" rel="nofollow">optimization</a></li><li class="kwd"><a href="/keyword/barley" class="hw-term hw-article-keyword hw-article-keyword-barley" rel="nofollow">barley</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic Selection</a></li></ul><p id="p-3">The improvement of populations in plant breeding through recurrent selection may benefit tremendously from genomewide selection. Of particular worth are the high accuracies and shortened breeding cycles of genomewide selection, which allow for greater genetic gains per unit time (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bernardo and Yu 2007</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Heffner <em>et al.</em> 2009</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lorenz <em>et al.</em> 2011</a>). While genomewide selection has already been employed in established breeding programs for major cultivated species (<em>e.g.</em>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Asoro <em>et al.</em> 2013</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Beyene <em>et al.</em> 2015</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Sallam <em>et al.</em> 2015</a>), this tool also has broad appeal across other species. For instance, breeding programs for tree or perennial crops with long generation times could find utility in making selections before the plants are mature enough to phenotype. Additionally, orphan, undomesticated, or unimproved crops may benefit from rapid breeding progress. Indeed, researchers have already investigated the use of genomewide selection in species such as apple (<em>Malus</em> x <em>domestica</em>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kumar <em>et al.</em> 2012</a>), <em>Eucalyptus</em> (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Resende <em>et al.</em> 2012</a>), oil palm (<em>Elaeis guineensis</em> Jacq.; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Cros <em>et al.</em> 2015</a>), and intermediate wheatgrass [<em>Thinopyrum intermedium</em> (Host) Barkworth &amp; D.R. Dewey; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Zhang <em>et al.</em> 2016</a>]. The population improvement necessary in newly established breeding programs, regardless of species, may be expedited through genomewide selection.</p><p id="p-4">Of course, the aforementioned advantages of genomewide selection depend on maintaining sufficient genetic gain. This requires accurate predictions of the genotypic value of selection candidates based on markers located throughout the genome (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Meuwissen <em>et al.</em> 2001</a>). Accurate predictions depend on reliable phenotypic measurements and sufficient marker data on a training population. Genomewide marker coverage that captures genomic relationships between individuals and ensures linkage disequilibrium (LD) between markers and quantitative trait loci (QTL) will lead to higher prediction accuracy, especially when predictions are applied to selection candidates more distantly related to the training population (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Habier <em>et al.</em> 2007</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Lorenz <em>et al.</em> 2011</a>). The predicted genotypic values under these conditions will more closely reflect the true genotypic values, and selection can then act to increase the frequency of favorable QTL alleles in a population and shift the mean of a population in a desirable direction.</p><p id="p-5">Characteristics of long-term recurrent selection create impediments to maintaining effective genomewide selection. Over generations, recombination between markers and QTL will cause LD to decay, while selection and drift will potentially act to generate new LD or tighten the LD between closely linked loci (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hill and Robertson 1968</a>; <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Lorenz <em>et al.</em> 2011</a>). Shifts in the pattern of QTL-marker LD, if not captured, will result in decreased prediction accuracy. This suggests that training populations must be updated during recurrent selection to maintain prediction accuracy, a notion that is indeed supported by studies using simulations and empirical data. Studies exploring simulations of recurrent selection in a clonally propagated crop (<em>Eucalyptus</em>) and an inbreeding small grain (barley: <em>Hordeum vulgare</em> L.) both revealed that the accuracy of genomewide selection was improved by updating the training population with data from previous breeding cycles (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Jannink 2010</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Denis and Bouvet 2013</a>). Similarly, using empirical data from an advanced-cycle rye (<em>Secale cereal</em> L.) breeding program, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Auinger <em>et al.</em> (2016)</a> found that aggregating training population data over multiple cycles enhanced prediction accuracy. These investigations all demonstrate the benefit of including previous cycle data into a training population; however, they did not test different methods of selecting that data.</p><p id="p-6">Though updating the training population may be required, there are practical considerations in how a breeder selects individuals to fulfill this need. Consider a breeding program employing genomewide recurrent selection in barley. Each year, the breeder must allocate phenotyping resources between testing potential cultivars and population improvement. Though genomewide selection offers to reduce the overall phenotyping costs of the latter (<em>e.g.</em>, through early-generation selection), promising breeding lines will undoubtedly be included in field trials. Under genomewide selection, it seems a breeder must also contend with the composition of their training population, placing emphasis on methods to build or maintain this population that both maximize prediction accuracy and minimize costs.</p><p id="p-7">Given the resource limitations of practical breeding and the importance of the training population, it is fitting that much research has been devoted to the composition and design of such populations. Using data from a North American barley breeding program, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lorenz <em>et al.</em> (2012)</a> reported reduced prediction accuracy when the training population and selection candidates belonged to separate subpopulations. Multiple studies have found that a training population that is more closely related to the selection candidates leads to more accurate predictions (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Asoro <em>et al.</em> 2011</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lorenz and Smith 2015</a>). Other researchers have suggested more explicit criteria to determine the optimal training population for a set of selection candidates. <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Rincent <em>et al.</em> (2012)</a> described training population design based on minimizing the mean prediction error variance (PEV) or maximizing the expected reliability of predictions [<em>i.e.</em>, generalized coefficient of determination (CD)]. When applied to empirical datasets, several investigations supported using the expected reliability criterion to optimally construct training populations (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Rincent <em>et al.</em> 2012</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akdemir <em>et al.</em> 2015</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Isidro <em>et al.</em> 2015</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Rutkoski <em>et al.</em> 2015</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Bustos-Korts <em>et al.</em> 2016</a>). These studies generally explored the construction of training populations from a single set of calibration individuals, therefore the usefulness of this criterion over multiple breeding cycles to maintain prediction accuracy is unknown.</p><p id="p-8">The objective of this study was to investigate various methods of updating a training population and their impact on genomewide recurrent selection. Using simulations, we envisioned a breeding program implementing genomewide recurrent selection for an inbreeding, small grain species (<em>i.e.</em>, barley). Six different training population update methods were compared, along with two scenarios of training population composition. We tracked important variables in breeding, including prediction accuracy, response to selection, and genetic variance. Additionally, we attempted to explain some of our observations using other parameters, including persistence of LD phase and genomic relationship.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-9">A barley breeding program employing genomewide selection can realistically complete a breeding cycle in a single year (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Following this breeding timeline, our experiment simulates a breeding population undergoing 15 cycles of recurrent genomewide selection.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1499/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Realistically, a cycle of genomewide recurrent selection in barley may only be 1 yr in length. Crosses are made in the autumn (year n) and progeny undergo single-seed descent through the following winter and summer. (1) At the F3 generation during the next autumn (year n + 1), lines are genotyped and predicted genotypic value (PGVs) are determined using training data from the previous cycle. These predictions determine the lines to use as parents in the next cycle of crosses (blue arrow). (2) Predictions are also used to select lines to phenotype in the following summer (year n + 2). (3) This phenotypic information is then incorporated into the training data for the next cycle of predictions and crosses during the subsequent autumn." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1033035519" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Realistically, a cycle of genomewide recurrent selection in barley may only be 1 yr in length. Crosses are made in the autumn (year &lt;em&gt;n&lt;/em&gt;) and progeny undergo single-seed descent through the following winter and summer. (1) At the F&lt;sub&gt;3&lt;/sub&gt; generation during the next autumn (year &lt;em&gt;n&lt;/em&gt; + 1), lines are genotyped and predicted genotypic value (PGVs) are determined using training data from the previous cycle. These predictions determine the lines to use as parents in the next cycle of crosses (blue arrow). (2) Predictions are also used to select lines to phenotype in the following summer (year &lt;em&gt;n&lt;/em&gt; + 2). (3) This phenotypic information is then incorporated into the training data for the next cycle of predictions and crosses during the subsequent autumn.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1499/F1.medium.gif" width="440" height="370"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/5/1499/F1.medium.gif" width="440" height="370"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1499/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1499/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484287" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">Realistically, a cycle of genomewide recurrent selection in barley may only be 1 yr in length. Crosses are made in the autumn (year <em>n</em>) and progeny undergo single-seed descent through the following winter and summer. (1) At the F<sub>3</sub> generation during the next autumn (year <em>n</em> + 1), lines are genotyped and predicted genotypic value (PGVs) are determined using training data from the previous cycle. These predictions determine the lines to use as parents in the next cycle of crosses (blue arrow). (2) Predictions are also used to select lines to phenotype in the following summer (year <em>n</em> + 2). (3) This phenotypic information is then incorporated into the training data for the next cycle of predictions and crosses during the subsequent autumn.</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">To incorporate the observed LD structure in barley breeding populations into our simulations, we used empirical marker data from two North American barley breeding programs: the University of Minnesota (UMN) and North Dakota State University (NDSU). Marker genotypes from 768 six-row spring inbred lines at 3072 biallelic SNP loci were obtained from the Triticeae Toolbox (T3) database (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Close <em>et al.</em> 2009</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Blake <em>et al.</em> 2016</a>). The genetic map position of markers was based on the consensus linkage map created by <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Muñoz-Amatriaín <em>et al.</em> (2011)</a>. Markers with &gt;10% missing data and lines with &gt;10% missing data were excluded. Markers were also filtered for redundancy, defined as those located at identical genetic map positions and with identical allele calls. A 0.01 cM interval was forced between markers with nonidentical allele calls and shared map positions (<em>i.e.</em>, due to low genetic map resolution). We set all heterozygous genotype calls to missing and imputed missing genotypes using the mode genotype across all samples. This left a set of 764 breeding lines and 1590 homozygous markers spanning 1137 cM.</p><div id="sec-2" class="subsection"><h3>Genetic model to simulate QTL</h3><p id="p-12">Each iteration of the simulation was initiated by randomly selecting <em>L =</em> 100 SNP loci to become causal QTL, regardless of genetic position or minor allele frequency. Genotypic values for QTL were drawn from a geometric series, as suggested by <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lande and Thompson (1990)</a>. At the <em>k</em>th QTL, the value of the favorable homozygote was <em>a<sup>k</sup></em>, the value of the heterozygote was 0, and the value of the unfavorable homozygote was –<em>a<sup>k</sup></em>, where <em>a</em> = (1 − <em>L</em>)/(1 + <em>L</em>). The value of the first allele of a QTL was randomly assigned to be favorable or unfavorable. Dominance and epistasis were assumed absent and higher values of the trait were considered favorable. The genotypic value of a given individual was calculated as the sum of the effects of QTL alleles carried by that individual.</p><p id="p-13">Phenotypic values were simulated by adding nongenetic effects to the genotypic values according to the model <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-1.gif"/></noscript></span></span> where <em>y<sub>ij</sub></em> was the phenotypic value of the <em>i</em>th individual in the <em>j</em>th environment, <em>g<sub>i</sub></em> was the genotypic value of the <em>i</em>th individual, <em>e<sub>j</sub></em> was the effect of the <em>j</em>th environment, and <em>ε<sub>ij</sub></em> was the residual effect of the <em>i</em>th individual in the <em>j</em>th environment. Environmental effects were assumed to be samples of a normally distributed random variable with mean 0 and SD <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-2.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-3.gif"/></noscript></span></span> was eight times the variance among genotypic values (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-4.gif"/></noscript></span></span>) (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bernardo 2014</a>). Residual effects were assumed to be samples of a normally distributed random variable with mean 0 and SD <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-5.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-6.gif"/></noscript></span></span> was scaled to achieve a target entry-mean heritability of <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-7.gif"/></noscript></span></span> in the base population. Phenotyping was assumed to take place in three environments with one replication, therefore within-environment variance and genotype-by-environment variance were confounded into <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-8.gif"/></noscript></span></span> The variance of environmental effects and the variance of residual effects remained unchanged over cycles of selection, allowing the heritability to vary. The mean phenotypic value of each individual over the three environments was used in genomewide prediction.</p></div><div id="sec-3" class="subsection"><h3>Base population and cycle 1 of genomewide selection</h3><p id="p-14">The base population (<em>i.e.</em>, cycle 0 training population) consisted of genotypic and simulated phenotypic data on the 764 breeding lines. Based on these simulated phenotypes, the top 50 UMN lines and the top 50 NDSU lines were intermated between breeding programs to generate the cycle 1 population. Specifically, 50 crosses were simulated, using each parent once, and 20 F<sub>3</sub>-derived lines were generated per cross. Gametes were generated following Mendelian laws of segregation, with recombination events simulated according to the genetic map positions of all loci (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Muñoz-Amatriaín <em>et al.</em> 2011</a>) and assuming no crossover interference or mutation. Population development resulted in a pool of 1000 F<sub>3</sub> selection candidates.</p><p id="p-15">The marker data for the training population and selection candidates consisted of genotypes at all loci except the 100 QTL. This essentially simulated genotyping with complete accuracy. Monomorphic markers and those with a minor allele frequency &lt;0.03 were removed prior to genomewide prediction. Marker effects were predicted using ridge-regression best linear unbiased prediction (RR-BLUP) according to the model<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-9.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <strong>y</strong> was an <em>N</em> × 1 vector of the phenotypic means of <em>N</em> training population lines, 1 was an <em>N</em> × 1 vector of ones, <em>μ</em> was the grand mean, <strong>Z</strong><sub>TP</sub> was an <em>N</em> × <em>m</em> incidence matrix of training population genotypes for <em>m</em> markers, <strong>u</strong> was an <em>m</em> × 1 vector of marker effects, and <strong>e</strong> was an <em>N</em> × 1 vector of residuals. Elements of <strong>Z</strong><sub>TP</sub> were 1 if homozygous for the first allele, −1 if homozygous for the second allele, and 0 if heterozygous. Genotypic values of the F<sub>3</sub> selection candidates were predicted using the equation <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-10.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-11.gif"/></noscript></span></span> was a 1000 × 1 vector of predicted genotypic values, <strong>Z</strong><sub>SC</sub> was a 1000 × <em>m</em> incidence matrix of selection candidate genotypes, and <strong>û</strong> was an <em>m</em> × 1 vector of predicted marker effects. Elements of <strong>Z</strong><sub>SC</sub> were the same as those in <strong>Z</strong><sub>TP</sub>.</p></div><div id="sec-4" class="subsection"><h3>Cycles 2–15 of genomewide selection</h3><p id="p-16">Subsequent cycles of the simulation consisted of three steps: (1) crossing and population development, (2) prediction and selection, and (3) training population updating. These are outlined in the diagram presented in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. Parents selected in the previous cycle were randomly intermated to form a pool of selection candidates. Again, 50 crosses were simulated and 1000 F<sub>3</sub>-derived selection candidates were generated. Prior to predictions, we removed monomorphic markers and those with a minor allele frequency &lt;0.03 in both the pool of selection candidates and in the training population. Since markers could become monomorphic due to selection or drift, the number of markers used for prediction decreased over breeding cycles. We predicted marker effects by Equation 1, using phenotypic and genotypic data on the training population. These marker effects were then used to predict genotypic values of the 1000 selection candidates, and those with the top 100 predicted genotypic values were designated as parents for the next cycle. A subset of all selection candidates were then designated as new additions to the training population according to one of the updating methods described below. We simulated phenotypes for these additions and merged the phenotypic and genotypic data to the pool of training population data.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1499/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A single breeding cycle in our simulations may be broken down into two main streams. Blue indicates steps involving the training population and red indicates steps involving crossing and population development. Green indicates the intermediate step of selection. (1) Fifty crosses are made using 100 randomly intermated parents from the previous cycle. Population development follows and 1000 selection candidates are genotyped at the F3 stage. Concurrently, marker effects are estimated using genotypic and phenotypic data from the training population (TP). (2) The predicted genotypic values (PGVs) of the selection candidates are used in decision-making. (3) The 100 selection candidates with the highest predicted genotypic values are selected as parents for the next cycle. Additionally, 150 selection candidates are selected based on the six different update methods. These candidates are phenotyped, and phenotypic and genotypic data are added to the pool of training data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1033035519" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A single breeding cycle in our simulations may be broken down into two main streams. Blue indicates steps involving the training population and red indicates steps involving crossing and population development. Green indicates the intermediate step of selection. (1) Fifty crosses are made using 100 randomly intermated parents from the previous cycle. Population development follows and 1000 selection candidates are genotyped at the F&lt;sub&gt;3&lt;/sub&gt; stage. Concurrently, marker effects are estimated using genotypic and phenotypic data from the training population (TP). (2) The predicted genotypic values (PGVs) of the selection candidates are used in decision-making. (3) The 100 selection candidates with the highest predicted genotypic values are selected as parents for the next cycle. Additionally, 150 selection candidates are selected based on the six different update methods. These candidates are phenotyped, and phenotypic and genotypic data are added to the pool of training data.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1499/F2.medium.gif" width="440" height="358"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/5/1499/F2.medium.gif" width="440" height="358"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1499/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1499/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484268" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-17" class="first-child">A single breeding cycle in our simulations may be broken down into two main streams. Blue indicates steps involving the training population and red indicates steps involving crossing and population development. Green indicates the intermediate step of selection. (1) Fifty crosses are made using 100 randomly intermated parents from the previous cycle. Population development follows and 1000 selection candidates are genotyped at the F<sub>3</sub> stage. Concurrently, marker effects are estimated using genotypic and phenotypic data from the training population (TP). (2) The predicted genotypic values (PGVs) of the selection candidates are used in decision-making. (3) The 100 selection candidates with the highest predicted genotypic values are selected as parents for the next cycle. Additionally, 150 selection candidates are selected based on the six different update methods. These candidates are phenotyped, and phenotypic and genotypic data are added to the pool of training data.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Methods of updating the training population</h3><p id="p-18">Seven different methods of updating the training population were explored in the simulations. For each method, 150 selection candidates from each cycle were selected and added to the training population. These methods are termed Top, Bottom, Random, PEVmean, CDmean, Tails, and No Change and are described below. For Top, Bottom, and Tails, selection candidates were ranked based on predicted genotypic value. The 150 selection candidates with the highest (Top) or lowest (Bottom) values were added to the training population. For the Tails method, the 75 selection candidates with the highest values and the 75 selection candidates with the lowest values were added to the training population. For Random, a random sample of selection candidates were added to the training population, and for No Change, the training population was not updated over breeding cycles.</p><p id="p-19">Two methods involved optimization algorithms previously described by other researchers, specifically PEVmean and CDmean (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Rincent <em>et al.</em> 2012</a>). Using only the genotypic data on all individuals, these algorithms aim to create a training population by optimally sampling individuals to be phenotyped in order to predict the value of individuals that would be unphenotyped. Our intention is similar, except that the individuals we sampled to be phenotyped are one cycle removed from the individuals that would be unphenotyped. For PEVmean, selection candidates were chosen to minimize the mean PEV of the genotypic values. As described in <a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Rincent <em>et al.</em> (2012)</a>, the general PEV can be computed using a matrix of contrasts, <strong>C</strong>, between the unphenotyped individuals and the mean of the whole population (phenotyped and unphenotyped individuals). In solving <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Henderson’s (1984)</a> equations, the PEV of any contrast can be computed as<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-12.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <strong>Z</strong> is an incidence matrix, <strong>M</strong> is an orthogonal projector (<a id="xref-ref-40-5" class="xref-bibr" href="#ref-40">Rincent <em>et al.</em> 2012</a>), and <strong>A</strong> is the genomic relationship matrix (described below). For the variance of the residuals (<span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-13.gif"/></noscript></span></span>), we used the restricted maximum likelihood estimate of <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-14.gif"/></noscript></span></span> from the RR-BLUP linear model in Equation 1. The additive genetic variance (<span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-15.gif"/></noscript></span></span>) was calculated by multiplying the number of markers, <em>N<sub>m</sub></em>, by the restricted maximum likelihood estimate of the variance of marker effects (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bernardo 2014</a>). The PEVmean was then calculated as <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-16.gif"/></noscript></span></span></p><p id="p-20">Similarly, for CDmean, candidates were chosen to maximize the reliability of the predictions, measured as the mean generalized CD. This can also be expressed as the expected reliability of the contrasts (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Laloe 1993</a>; <a id="xref-ref-40-6" class="xref-bibr" href="#ref-40">Rincent <em>et al.</em> 2012</a>), computed as<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-17.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>The values of <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-18.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-19.gif"/></noscript></span></span> were the same as described for Equation 2. The CDmean was then calculated as <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-20.gif"/></noscript></span></span></p><p id="p-21">We implemented an exchange algorithm similar to that described by <a id="xref-ref-40-7" class="xref-bibr" href="#ref-40">Rincent <em>et al.</em> (2012)</a>, with one modification in the designation of individuals to predict and individuals to sample for phenotyping. The situation outlined by <a id="xref-ref-40-8" class="xref-bibr" href="#ref-40">Rincent <em>et al.</em> (2012)</a> assumes that the genotypic data for the individuals to sample and for the individuals to predict is available concurrently. In our simulation, this is not the case, since phenotyping of the selections in one cycle (cycle <em>n</em>) will occur before genotypic data on selection candidates of the next cycle (cycle <em>n +</em> 1) becomes available (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). We therefore chose the 100 parents of the cycle <em>n +</em> 1 selection candidates to be a proxy for the unphenotyped individuals, while the entire 1000 selection candidates (including the parents) constituted the population of individuals to be sampled by the algorithm. To maintain a reasonable computation time, the exchange algorithms were iterated 500 times. Preliminary data showed that a reasonable optimum for either criterion was reached after 500 iterations (data not shown). The PEVmean or CDmean algorithms were used to select individuals from the selection candidates to be included in the training population for the next cycle.</p><p id="p-22">We also considered two scenarios of using the updated training population data. The first scenario represented a situation where a breeder may want to use all available information, and in this case, the training population grew by 150 lines in each cycle. This was termed the Cumulative scenario, and over cycles the size of the training population ranged from 764 to 2864 individuals. In the next scenario, we attempted to control for the effect of training population size by using a “sliding window” of 764 lines along breeding cycles. Specifically, in each cycle the 150 new training population additions from the latest breeding cycle took the place of the 150 training population additions from the earliest breeding cycle. Since the 764 base population lines all constituted cycle 0, these lines were discarded randomly until no base population lines remained in the training population. Afterward, lines from earlier cycles were discarded as lines from later cycles were added. This was termed the Window scenario.</p></div><div id="sec-6" class="subsection"><h3>Variables tracked over breeding cycles</h3><p id="p-23">To better interpret the observations in the simulations, we tracked a number of additional variables, including persistence of LD phase, mean realized additive genomic relationship, prediction accuracy, genetic variance, mean genotypic value, inbreeding coefficient, and the frequency of QTL and marker alleles.</p><p id="p-24">The genetic variance in each cycle was calculated as the variance among the genotypic values of the selection candidates. Prediction accuracy was measured by computing the correlation between the predicted genotypic values of the selection candidates and their true genotypic values.</p><p id="p-25">We measured the LD between QTL and markers as follows: for each and every polymorphic QTL in a given population (<em>i.e.</em>, the training population or the selection candidates), we computed the correlation between that QTL and each and every polymorphic marker in the genome. We calculated persistence of LD phase by first measuring QTL-marker LD in the training population and in the selection candidates. QTL or markers that were not polymorphic in either of these populations were excluded. We then computed the correlation between the measures of QTL-marker LD in the training population and in the selection candidates. This metric, also known as the correlation of <em>r</em>, evaluates whether patterns of QTL-marker LD are similar between two populations. High correlations of <em>r</em> indicate that QTL-marker LD phases are consistent, and presumably the predicted marker effects in one population would accurately represent the marker effects in the second population (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">de Roos <em>et al.</em> 2008</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Toosi <em>et al.</em> 2010</a>).</p><p id="p-26">Additive relationships between lines in the simulation were measured with respect to the base population. Before initiating the simulations, a matrix <strong>P</strong> was calculated as <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-21.gif"/></noscript></span></span> where <em>p<sub>i</sub></em> is the frequency of the second allele at locus <em>i</em> in the base population. Additionally, a normalization constant <strong>c</strong> was calculated as <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-22.gif"/></noscript></span></span> Both calculations are described in <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">VanRaden (2008)</a>. To compute additive relationships at any one cycle in the simulation, the genotype matrices (including QTL) of the training population and selection candidates were combined into a matrix <strong>M</strong>. The matrix <strong>P</strong> was subtracted from <strong>M</strong> to obtain matrix <strong>W</strong>. We then calculated the relationship matrix as <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5/1499/embed/mml-math-23.gif"/></noscript></span></span> This ensured that the relationship matrix was scaled to reflect the allele frequencies in the base population (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">VanRaden 2008</a>). We calculated the mean additive relationship as the mean value of the training population-selection candidate combinations. Inbreeding coefficients for each individual were also calculated from this matrix as the diagonal elements minus one.</p><p id="p-27">All simulations were performed in R (version 3.3.1, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">R Core Team 2016</a>) using the packages <em>hypred</em> (version 0.5, Technow 2014) and <em>rrBLUP</em> (version 4.4, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Endelman 2011</a>). Each simulation experiment was repeated 250 times. The methods of updating the training population (<em>i.e.</em>, Top, Bottom, Random, CDmean, PEVmean, Tails, and No Change) each constituted an independent experiment. With the two updating scenarios (<em>i.e.</em>, Window and Cumulative), there were 14 different simulations.</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-28">Simulation scripts, starting marker genotypes, and summarized data are provided in the R package <em>GSSimTPUpdate</em>, available from the GitHub repository <a href="https://github.com/UMN-BarleyOatSilphium/GSSimTPUpdate">https://github.com/UMN-BarleyOatSilphium/GSSimTPUpdate</a>. Included is a vignette on how to obtain the marker data from the T3 database.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Long-term prediction accuracy</h3><p id="p-29">Prediction accuracy (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> and Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040550/-/DC1/TableS1.docx">Table S1</a>) consistently decreased over cycles of selection for all methods of updating the training population and in both updating scenarios. Within and between scenarios, we observed differences among the update methods in the decay rate of prediction accuracy. A prominent observation was the precipitous decline in accuracy when not updating the training population (<em>i.e.</em>, No Change). Early in breeding cycles, prediction accuracy for this method was similar to the remaining methods, but by cycle five it had decayed beyond the remaining methods. As expected, identical trends were observed for No Change in both updating scenarios.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1499/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracy over breeding cycles of the simulation. Accuracy was measured as the correlation between the predicted and true genotypic values of the selection candidates. Line colors and point shapes delineate the different methods of updating the training population. Plots are separated into the (A) Cumulative and (B) Window updating scenarios. Average values are shown with 95% confidence intervals. To help reduce plot clutter, points for each update method are given a small, consistent jitter along the x-axis. Because the plotting jitter may accentuate small differences between updating methods, these data are also provided in Table S1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1033035519" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Prediction accuracy over breeding cycles of the simulation. Accuracy was measured as the correlation between the predicted and true genotypic values of the selection candidates. Line colors and point shapes delineate the different methods of updating the training population. Plots are separated into the (A) Cumulative and (B) Window updating scenarios. Average values are shown with 95% confidence intervals. To help reduce plot clutter, points for each update method are given a small, consistent jitter along the &lt;em&gt;x&lt;/em&gt;-axis. Because the plotting jitter may accentuate small differences between updating methods, these data are also provided in Table S1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1499/F3.medium.gif" width="440" height="246"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/5/1499/F3.medium.gif" width="440" height="246"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1499/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1499/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484353" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-30" class="first-child">Prediction accuracy over breeding cycles of the simulation. Accuracy was measured as the correlation between the predicted and true genotypic values of the selection candidates. Line colors and point shapes delineate the different methods of updating the training population. Plots are separated into the (A) Cumulative and (B) Window updating scenarios. Average values are shown with 95% confidence intervals. To help reduce plot clutter, points for each update method are given a small, consistent jitter along the <em>x</em>-axis. Because the plotting jitter may accentuate small differences between updating methods, these data are also provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040550/-/DC1/TableS1.docx">Table S1</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">Among methods of actively updating the training population (<em>i.e.</em>, excluding No Change), differences in prediction accuracy were observed in early cycles, but became increasingly similar in later cycles. The Top and Tails methods resulted in a nonsignificant but noticeable accuracy advantage early on that persisted for several cycles (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040550/-/DC1/TableS1.docx">Table S1</a>). On the other hand, the Bottom method displayed a noticeable disadvantage that persisted for a similar length of time. The Random, PEVmean, and CDmean methods were highly comparable and yielded accuracies intermediate of the Top and Bottom methods. By cycle 10, the differences between active methods of updating were negligible. These patterns were observed in both the Cumulative and Window scenarios.</p><p id="p-32">One noticeable difference between the trends in the Cumulative and Window scenarios was in the rate of prediction accuracy decay. Among the active methods of updating, the rate of prediction accuracy decay was slightly greater in the Cumulative scenario (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>) compared to the Window scenario (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>). By the 15th breeding cycle, the difference in these decay rates amounted to a difference in prediction accuracy of roughly 0.02–0.04.</p></div><div id="sec-10" class="subsection"><h3>Genetic variance and response to selection</h3><p id="p-33">Genetic variance among the selection candidates (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4, A and B</a>) similarly decreased across cycles for all training population update methods. For this variable, however, the rank among methods remained more consistent. That is, compared to the remaining update methods, the genetic variance in the Top and Tails methods was consistently less and the genetic variance in the Bottom method was consistently greater. The Tails method resulted in slightly higher genetic variance compared to the Top method; however, this difference was never significant. Genetic variance across the CDmean, PEVmean, and Random methods was very similar within and between scenarios. Not updating the training population resulted in genetic variance similar to CDmean, PEVmean, and Random in early breeding cycles. After seven cycles, however, the loss of genetic variance was abated compared to remaining methods. By the end of the breeding timeline, the genetic variance for No Change was noticeably and significantly (95% confidence interval) higher than the remaining methods.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1499/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic variance (A and B) and genotypic values (C and D) among the selection candidates over breeding cycles of the simulation. Line colors and point shapes delineate the different methods of updating the training population. Plots are separated into the (A and C) Cumulative and (B and D) Window updating scenarios. Average values are shown with 95% confidence intervals. To help reduce plot clutter, points for each update method are given a small, consistent jitter along the x-axis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1033035519" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genetic variance (A and B) and genotypic values (C and D) among the selection candidates over breeding cycles of the simulation. Line colors and point shapes delineate the different methods of updating the training population. Plots are separated into the (A and C) Cumulative and (B and D) Window updating scenarios. Average values are shown with 95% confidence intervals. To help reduce plot clutter, points for each update method are given a small, consistent jitter along the &lt;em&gt;x&lt;/em&gt;-axis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1499/F4.medium.gif" width="440" height="405"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/5/1499/F4.medium.gif" width="440" height="405"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1499/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1499/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484374" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-34" class="first-child">Genetic variance (A and B) and genotypic values (C and D) among the selection candidates over breeding cycles of the simulation. Line colors and point shapes delineate the different methods of updating the training population. Plots are separated into the (A and C) Cumulative and (B and D) Window updating scenarios. Average values are shown with 95% confidence intervals. To help reduce plot clutter, points for each update method are given a small, consistent jitter along the <em>x</em>-axis.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">Overall, the mean genotypic value of the selection candidates (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4, C and D</a>) displayed a similar, but opposite pattern compared to the genetic variance. Updating the training population by the Top or Tails methods yielded an advantage in genotypic value, a trend that became more apparent in later breeding cycles. Conversely, the genotypic values under the Bottom method ranked lowest among the active updating methods. This disadvantage was often slight and nonsignificant, especially in the Cumulative scenario (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>). As in the observations of genetic variance, the CDmean, PEVmean, and Random methods responded similarly. Most noticeable was the rapid plateau in genotypic value under the No Change method, particularly around the eighth breeding cycle. By the end of the breeding timeline, the No Change method appeared to have reached a limit, and although the trajectory of the remaining methods suggested further increases, their trends implied a limit as well (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4, C and D</a>). Curiously, the Top method was generally superior to the Tails method in the Cumulative scenario; however, the opposite was true in the Window scenario. In both scenarios, the Tails method exhibited a trend suggesting that this method would eventually yield selection candidates with an average genotypic value superior to that of the Top method. The trends among the remaining training population update methods were similar in both updating scenarios.</p></div><div id="sec-11" class="subsection"><h3>Drivers of prediction accuracy</h3><p id="p-36">Average relationship between training population individuals and selection candidate individuals, as measured by marker information, varied among the update methods (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5, A and B</a>). As expected, the average relationship did not change in either updating scenario when the training population remained unaltered. Across both scenarios, the relationship generally remained highest under the Top method, lowest under the Bottom method, and intermediate under the CDmean, PEVmean, Random, and Tails methods. In the Cumulative scenario (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>), actively updating the training population resulted in a linear increase in average relationship for all methods. Additionally, the different update methods, particularly Top and Bottom, displayed slight divergence, especially in later breeding cycles. The Window scenario (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>) presented a more sigmoidal trend, eventually resulting in slight convergence in average relationship among active update methods. Interestingly, after cycle 12, the average relationship between the training population and the selection candidates in the Tails method remained greater than that in the Top method.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1499/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Other variables tracked over the course of the simulations. (A and B) The average genomic relationship was calculated between the training population and the selection candidates using marker genotypes. Relationships were scaled to reflect the allele frequencies in the base population. (C and D) The level of inbreeding was measured on the selection candidates and was derived from the relationship matrix described above. (E and F) The number of QTL fixed for an allele was measured in the selection candidates. (G and H) Persistence of LD phase was measured as the correlation of r between the training population and the selection candidates. Line colors and point shapes delineate the different methods of updating the training population. Plots are separated into the (A, C, E, and G) Cumulative and (B, D, F, and H) Window updating scenarios. Average values are shown with 95% confidence intervals. To help reduce plot clutter, points for each update method are given a small, consistent jitter along the x-axis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1033035519" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Other variables tracked over the course of the simulations. (A and B) The average genomic relationship was calculated between the training population and the selection candidates using marker genotypes. Relationships were scaled to reflect the allele frequencies in the base population. (C and D) The level of inbreeding was measured on the selection candidates and was derived from the relationship matrix described above. (E and F) The number of QTL fixed for an allele was measured in the selection candidates. (G and H) Persistence of LD phase was measured as the correlation of &lt;em&gt;r&lt;/em&gt; between the training population and the selection candidates. Line colors and point shapes delineate the different methods of updating the training population. Plots are separated into the (A, C, E, and G) Cumulative and (B, D, F, and H) Window updating scenarios. Average values are shown with 95% confidence intervals. To help reduce plot clutter, points for each update method are given a small, consistent jitter along the &lt;em&gt;x&lt;/em&gt;-axis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1499/F5.medium.gif" width="318" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/5/1499/F5.medium.gif" width="318" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1499/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1499/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484271" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-37" class="first-child">Other variables tracked over the course of the simulations. (A and B) The average genomic relationship was calculated between the training population and the selection candidates using marker genotypes. Relationships were scaled to reflect the allele frequencies in the base population. (C and D) The level of inbreeding was measured on the selection candidates and was derived from the relationship matrix described above. (E and F) The number of QTL fixed for an allele was measured in the selection candidates. (G and H) Persistence of LD phase was measured as the correlation of <em>r</em> between the training population and the selection candidates. Line colors and point shapes delineate the different methods of updating the training population. Plots are separated into the (A, C, E, and G) Cumulative and (B, D, F, and H) Window updating scenarios. Average values are shown with 95% confidence intervals. To help reduce plot clutter, points for each update method are given a small, consistent jitter along the <em>x</em>-axis.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">Generally, we observed a curvilinear increasing trend in the level of inbreeding (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5, C and D</a>). The No Change method performed similarly in the different updating scenarios, but differed markedly from the active updating methods. This method resulted in a more rapid increase in inbreeding, beginning after the fourth breeding cycle. By the end of the breeding timeline, the trend had not yet plateaued and suggested that inbreeding would continue to increase. Considering the active updating methods, there were slight differences in inbreeding trends between the two updating scenarios. In the Cumulative scenario (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5C</a>), these methods performed similarly, showing no significant differences. Inbreeding was slightly greater for these methods in this scenario than in the Window scenario (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5D</a>). In this case, differences between the updating methods were more apparent. The Top method displayed noticeably lower levels of inbreeding, particularly after the eighth breeding cycle. Remaining methods performed similarly between each other.</p><p id="p-39">We noticed consistent trends among methods of updating the training population in the rate of fixation of QTL (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5, E and F</a>). In both updating scenarios, the Top method maintains a higher number of fixed QTL across breeding cycles, followed by the CDmean, PEVmean, Tails, and Random methods, which performed similarly, followed by the Bottom and No Change methods, which also performed similarly. Additionally, we observed that ∼10% of the QTL became fixed in cycle 1 of the breeding timeline, while by cycle 15 ∼70% of the QTL were fixed. There were two slight, noteworthy differences in these trends between the updating scenarios. First, active updating methods generally displayed a higher proportion of fixed QTL in the Window scenario (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5E</a>) than in the Cumulative scenario (<a id="xref-fig-5-9" class="xref-fig" href="#F5">Figure 5F</a>). Second, the degree of separation between the Top method and the CDmean, PEVmean, and Random methods appeared greater in the Cumulative scenario.</p><p id="p-40">There were marked differences in the persistence of LD phase between the methods of updating the training population within and between the updating scenarios (<a id="xref-fig-5-10" class="xref-fig" href="#F5">Figure 5, G and H</a>). Under the Cumulative scenario (<a id="xref-fig-5-11" class="xref-fig" href="#F5">Figure 5G</a>), persistence of phase for all update methods declined quickly in initial cycles, but reached equilibrium around the 10th cycle. The Top and Tails methods maintained the highest degree of persistence across breeding cycles, but the Tails method trended closer to the other active update methods by cycle 12. Furthermore, the initial decay was much lower under the Top and Tails methods, and the equilibrium point was higher than other methods. Persistence of phase under the Bottom method was initially much less than the other active update methods, and although it soon became similar to these methods, it still remained less. The remaining active update methods were quite similar in this scenario.</p><p id="p-41">In comparison, actively updating the training population under the Window scenario (<a id="xref-fig-5-12" class="xref-fig" href="#F5">Figure 5D</a>) yielded increasing persistence of phase over the course of the breeding timeline. Each of these methods saw a small drop in persistence of phase initially, but after the fifth cycle values began to increase. Interestingly, none of these methods appeared to reach an equilibrium point. The disparity between update methods, especially between Top and Bottom, was highly apparent under this scenario. Conversely, CDmean, PEVmean, and Random resulted in very similar levels of persistence of phase. Finally, the persistence of phase under the Tails method was initially intermediate between the Top method and the CDmean, PEVmean, and Random methods; however, it eventually became more similar to the latter.</p><p id="p-42">Expectedly, the No Change method resulted in identical trends in both updating scenarios. In the same way as prediction accuracy, we observed a precipitous, exponential decay in persistence of phase. The trend appeared to reach an equilibrium point at around the same breeding cycle as the active updating methods in the Cumulative scenario; however, this equilibrium point was much lower than the others.</p></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><div id="sec-13" class="subsection"><h3>Updating the training population can be simple and effective</h3><p id="p-43">We observed similar patterns in prediction accuracy (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>), mean genotypic value (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4, C and D</a>), and genetic variance (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4, A and B</a>) among active methods of updating the training population (<em>i.e.</em>, excluding No Change). The high similarity between these methods suggests that simply including more recent data in the training population provides a marked advantage in improving the breeding population in the long term. This is encouraging in a practical sense, as any phenotypic information generated on breeding lines, regardless of how they may have been selected, would probably be helpful in preventing severe long-term loss in prediction accuracy.</p><p id="p-44">Although we only tested six active methods of updating the training population, we might expect that any method should outperform doing nothing. Over breeding cycles, including recent genotypic and phenotypic information in the training population helps to capture new LD generated by selection and drift (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Hill and Robertson 1968</a>). Older training population lines will of course not provide any information on this new LD; however, we may presume most or all selection candidates will share a proportion of this new LD as long as the parents of these lines are not unrelated. Therefore, even the selection candidates most distantly related to those chosen as parents will provide informative training data for the next cycle. In the long term, we might expect a decrease in the relative importance of how selection candidates are chosen to add to the training population. Over continued cycles of selection in a closed population, parents will become increasingly related (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Daetwyler <em>et al.</em> 2007</a>), thus the pool of selection candidates will share a greater proportion of the new, informative LD.</p><p id="p-45">Though it appears updating the training population is favorable regardless of method, it is worth pointing out differences in the methods we tested. The Top method achieved high prediction accuracy and high mean genotypic value across breeding cycles. These results are not entirely surprising, since the candidates selected to update the training population were mostly those selected as parents for the next cycle (100 out of 150). These additions to the training population will be highly related to the selection candidates in the next cycle, and will therefore provide the training population with the most useful information shared through genomic relationships and QTL-marker LD (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Lorenz and Smith 2015</a>). Indeed, this is readily apparent in measures of relatedness between the training population and the selection candidates (<a id="xref-fig-5-13" class="xref-fig" href="#F5">Figure 5, A and B</a>) and in measures of persistence of LD phase (<a id="xref-fig-5-14" class="xref-fig" href="#F5">Figure 5, C and D</a>).</p><p id="p-46">With this in mind, it is not surprising that the Bottom method delivers the lowest prediction accuracy (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3, A and B</a>) and lowest mean genotypic value (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4, C and D</a>), as zero lines added to the training population overlap with the selected parents. This lack of overlap would suggest that QTL-marker LD information in the training additions and that observed in the selection candidates will be in high disagreement. Indeed, we observe that this method produces training populations with the lowest average relationship to the selection candidates (<a id="xref-fig-5-15" class="xref-fig" href="#F5">Figure 5, A and B</a>) and the lowest persistence of LD phase (<a id="xref-fig-5-16" class="xref-fig" href="#F5">Figure 5, G and H</a>).</p><p id="p-47">The Tails method, as a combination of the Top and Bottom method, offers some curious results. Though the prediction accuracy achieved from this method is, for the most part, not significantly different than that of the Top method, it is often higher, leading to low genetic variance (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4, A and B</a>) and high average genotypic value (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4, C and D</a>). This is in spite of the observation that under the Tails method, the average relationship between the training population and selection candidates (<a id="xref-fig-5-17" class="xref-fig" href="#F5">Figure 5, A and B</a>) and persistence of LD phase (<a id="xref-fig-5-18" class="xref-fig" href="#F5">Figure 5, G and H</a>) are roughly equal to or lower than the Top method. A possible explanation for this observation could be that this method produces training populations that satisfy different conditions for accurate genomewide predictions. First, 75 out of the 150 training population additions overlap with the 100 selected parents. Just as in the Top method, these additions will be highly related to the selection candidates of the next cycle and contribute useful QTL-marker LD information. The other 75 additions will presumably be more unrelated to these selection candidates, leading to the intermediate average relationship (<a id="xref-fig-5-19" class="xref-fig" href="#F5">Figure 5, A and B</a>) and often lower persistence of LD phase (<a id="xref-fig-5-20" class="xref-fig" href="#F5">Figure 5, G and H</a>); however, these training population additions may provide information for more reliable predictions. In a study where the training population was a subset of a larger population, <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Yu <em>et al.</em> (2016)</a> found that individuals in the validation population (<em>i.e.</em>, selection candidates) with the highest and lowest predicted genotypic values had the greatest upper bound for the reliability of those predictions (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Karaman <em>et al.</em> 2016</a>). It may be the case in our simulations that the training population additions in the Tails method had more reliably predicted genotypic values. This reliability may have led to better identification of individuals that, when added to the training population, could provide information that more clearly differentiated the effects of QTL alleles, leading to more accurate predictions of marker effects. Thus, the Tails method may have taken advantage of both high relatedness and greater genotypic diversity in the training population.</p><p id="p-48">The criterion-based updating methods (CDmean and PEVmean) performed very similarly to the Random method in prediction accuracy (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3, A and B</a>). This observation is generally in agreement with previous research (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Akdemir <em>et al.</em> 2015</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Isidro <em>et al.</em> 2015</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Bustos-Korts <em>et al.</em> 2016</a>) and may be related to the size of the training population used in our simulations. In several examples in these studies, the prediction accuracy of a randomly selected training population was similar to that of a training population selected by the CDmean or PEVmean criteria, particularly at larger sizes of the training population. While these investigations examined training populations ranging from 25 to 300 individuals, our simulations looked at much larger training populations, ranging from 764 to 2864 individuals. It may be, then, that as the size of the training population becomes sufficiently large, the performance of the CDmean and PEVmean criteria becomes more similar to a random sampling. This, of course, does not suggest that these criteria have no use in selecting training populations. If these criteria are in fact superior in smaller training populations, they may be advantageous when performing genomewide selection on a trait that is expensive or low-throughput to phenotype.</p><p id="p-49">It is worth addressing the continued loss in prediction accuracy in all updating methods and in both updating scenarios. This occurs even as two known components of prediction accuracy, persistence of LD phase and genomic relationship (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">de Roos <em>et al.</em> 2008</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Toosi <em>et al.</em> 2010</a>; <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Lorenz <em>et al.</em> 2011</a>; <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Lorenz and Smith 2015</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Sallam <em>et al.</em> 2015</a>), stabilize or increase. The primary reason for these observations is undoubtedly the reduction in heritability as genetic variance declines over cycles (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4, A and B</a>). Since residual variance remains constant, the phenotypic data measured on lines becomes increasingly uncorrelated with the true genotypic value (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Bernardo and Yu 2007</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bernardo 2010</a>). Thus, the data included in the training population will not capture the effects of QTL alleles, decreasing the accuracy of predicted marker effects. A second potential contributor is the fixation of marker loci over cycles. Since monomorphic markers are removed prior to model training, fewer markers will be used in later cycles. Indeed, by cycle 7, on average 55% of the original markers are used, and by cycle 15 this drops to 30% (data not shown). Though previous studies have stated the benefit of greater marker density (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Combs and Bernardo 2013</a>), many others have noted diminishing returns (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lorenzana and Bernardo 2009</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Heffner <em>et al.</em> 2011</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Lorenz <em>et al.</em> 2012</a>). Reasonably high marker densities were maintained in our simulations, so this is likely not a strong driver of the decay in prediction accuracy.</p><p id="p-50">The performance of the Top method suggests a simple procedure to optimize genomewide selection in an applied breeding program. Our results indicate that a breeder may prevent severe loss of prediction accuracy in recurrent selection by updating the training population to include information on lines that would be selected anyway. Ultimately, this method should be more cost effective than the others. A breeder would likely desire to evaluate selected parents in field trials, perhaps for variety development or to gather phenotypic data to accompany predicted genotypic values. The Top method provides an advantage here, as the number of additional lines to phenotype for updating the training population is minimal. The breeder can use this information for dual purposes, using phenotypic data to build a more accurate training dataset while making informed decisions on potential variety selections.</p><p id="p-51">Although the Tails method led to slightly greater prediction accuracy than the Top method, there are at least three reasons why it may not be the most practical method. First, the difference in prediction accuracy between these methods was generally not significant (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040550/-/DC1/TableS1.docx">Table S1</a>). Second, the overlap between training population additions and candidates that would be prioritized for phenotyping by the breeder (<em>i.e.</em>, parents and superior lines) is lower, and therefore, third, because of this lack of overlap, the breeder would expend costly resources on phenotyping lines that may not provide any utility outside of model training for genomewide selection.</p><p id="p-52">Encouragingly, empirical data in a barley breeding program supports the Top method in enhancing prediction accuracy. Over a few cycles of recurrent genomewide selection for yield and deoxynivalenol content (a mycotoxin produced by the fungal pathogen <em>Fusarium graminearum</em> Schwabe.), T. Tiede (unpublished data) found that updating the training population improved prediction accuracy. Specifically, including data only on lines selected for favorable predicted genotypic values in previous cycles enhanced the prediction accuracy in subsequent cycles. This method was superior to a random selection of lines and was often superior to a selection based on criteria optimization.</p></div><div id="sec-14" class="subsection"><h3>Not updating the training population is unfavorable</h3><p id="p-53">It is quite apparent from our simulations that in the long term, not updating the training population is highly unfavorable. Prediction accuracy decreases rapidly in this case (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3, A and B</a>), and as a consequence, response to selection also collapses, leading to the observed plateau in genotypic value (<a id="xref-fig-4-11" class="xref-fig" href="#F4">Figure 4, C and D</a>). Here selection is acting on nongenetic noise, preventing the mean genetic value in the population from changing.</p><p id="p-54">The genetic composition of the breeding populations underscores the negative consequences of leaving the training population unaltered. Although genetic variance appears to be preserved in the long term (<a id="xref-fig-4-12" class="xref-fig" href="#F4">Figure 4, A and B</a>), considering the decrease in accuracy and the plateau in genotypic value, this may be due to a larger number of QTL that remain segregating. We do indeed observe this (<a id="xref-fig-5-21" class="xref-fig" href="#F5">Figure 5, E and F</a>), but given the similarity in the number of fixed QTL under the No Change method and that under the remaining methods, we may also surmise that a greater proportion of QTL are becoming fixed for unfavorable alleles. We also observe alarming levels of inbreeding among the selection candidates when not updating the training population (<a id="xref-fig-5-22" class="xref-fig" href="#F5">Figure 5, C and D</a>). This result is not surprising, since previous theory and simulations into genomewide selection show that more accurate predictions better capture the Mendelian sampling term (<em>i.e.</em>, within-family variance), preventing high rates of inbreeding (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Daetwyler <em>et al.</em> 2007</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Jannink 2010</a>). Although higher inbreeding does not reduce genetic variance, it invariably will reduce the number of usable, polymorphic markers. Collectively, this suggests that continued genomewide selection without updating the training population will impose a lower selection limit on population improvement.</p><p id="p-55">The results of our simulations indicate that severe consequences of not updating the training population were delayed until later cycles. Although prediction accuracy declines very rapidly (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3</a>), mean genotypic value and genetic variance track closely with the other updating methods (<a id="xref-fig-4-13" class="xref-fig" href="#F4">Figure 4</a>). It is not until the fifth cycle or later that the impact of an unaltered training population is readily apparent. This can be encouraging in practical breeding scenarios. For instance, in a new breeding program, the stock of germplasm with phenotypic data may be low, and it may be several cycles before enough individual are tested to add to the training population. One may also consider a crop where the time between making a cross and gathering phenotypic data on the progeny is long. Several cycles of selection could be performed before data are available to update the training population. Our results suggest that the same training population could be used for a small number of cycles without serious detriment.</p></div><div id="sec-15" class="subsection"><h3>A smaller and more recent training population may provide long-term advantages</h3><p id="p-56">We observed nonsignificant but noticeable differences in prediction accuracy, mean genotypic value, and genetic variance between the Cumulative and Window updating scenarios. In the short term, prediction accuracy was slightly greater under the Cumulative scenario for most of the active updating methods, particularly the Top method (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3A</a>); however, in the long term, prediction accuracy was higher when the training population consisted of only more recent data (<em>i.e.</em>, the Window scenario). Although the trends in genotypic value suggest that the Cumulative scenario is slightly advantageous in the short term, the trend under the Window scenario suggested that additional gains may be greater (<a id="xref-fig-4-14" class="xref-fig" href="#F4">Figure 4D</a>). Indeed, given the slightly higher prediction accuracy observed at the end of the breeding timeline for this scenario, we would expect response to selection to be greater in the long term (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bernardo 2010</a>).</p><p id="p-57">In addition to the explanations provided earlier in the <em>Discussion</em>, other factors may be responsible for these observations. Most notable are the differences between updating scenarios in genomic relationship (<a id="xref-fig-5-23" class="xref-fig" href="#F5">Figure 5, A and B</a>) and persistence of LD phase (<a id="xref-fig-5-24" class="xref-fig" href="#F5">Figure 5, G and H</a>). Retaining older training data results in lower average relationship between the training population and the selection candidates (<a id="xref-fig-5-25" class="xref-fig" href="#F5">Figure 5A</a>). This is not unexpected, since selection candidates in earlier cycles will be increasingly unrelated to those in later cycles. Maintaining a training population with more recent data results in higher average relationship and a higher rate of increasing relationship (<a id="xref-fig-5-26" class="xref-fig" href="#F5">Figure 5B</a>). This result corroborates previous research demonstrating higher prediction accuracy when retaining individuals in the training population that are more closely related to the selection candidates (<a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Lorenz and Smith 2015</a>).</p><p id="p-58">Perhaps most drastic are the differences in persistence of LD phase between updating scenarios. A training population with older data (<em>i.e.</em>, Cumulative) results in decayed persistence of LD phase (<a id="xref-fig-5-27" class="xref-fig" href="#F5">Figure 5G</a>). Over cycles, recombination breaks down LD and training population additions capture new LD. Older training data does not reflect this new LD, decreasing the persistence of phase. The observed stabilization in <a id="xref-fig-5-28" class="xref-fig" href="#F5">Figure 5C</a> could be due to new training data capturing as much LD as what is broken down by recombination. Evidence for this may be seen under the Window scenario (<a id="xref-fig-5-29" class="xref-fig" href="#F5">Figure 5H</a>), where persistence of LD phase increases when actively updating the training population. A training population of only recent data captures the new LD generated by recombination in the previous cycle, but without the uninformative LD present in older training data. In addition, it may be possible that recent training additions capture more of the informative new LD than what is lost through recombination, leading to the observed increase in persistence of phase.</p></div><div id="sec-16" class="subsection"><h3>Simulation considerations</h3><p id="p-59">It is important to address the limitations of our simulations, including assumptions that could be violated in a real-life breeding program. First, random mating may be unrealistic, and we might expect a breeder to impose a more sophisticated procedure for parent selection. For instance, mating pairs may be prioritized for complementation of favorable values of multiple traits. Additionally, an individual may be used as a parent over multiple breeding cycles, especially if observed phenotypic values agreed with the predicted genotypic values. More sophisticated methods of parental selection, such as those based on virtual biparental populations (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Bernardo 2014</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lian <em>et al.</em> 2015</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Mohammadi <em>et al.</em> 2015</a>), may be used. These nonrandom mating schemes could affect genetic variance or contribute to different patterns of LD, both of which would impact the accuracy of genomewide prediction; however, incorporating such nuances into our simulation would likely rest on additional assumptions and would be intractable to model. Random mating provides a simple approach, and given the recurrent selection scheme, it is a reasonable assumption. Our simulation also made the assumption that the breeding population was closed. This is obviously inaccurate in a practical program, as the exchange and incorporation of new germplasm is common. Realistically, we might expect prediction accuracy to decrease when adding germplasm from different breeding programs or subpopulations to the pool of selection candidates (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Lorenz <em>et al.</em> 2012</a>). In recurrent selection, however, the objective is to improve a population rapidly, so a closed population may be desirable (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Bernardo 2010</a>).</p><p id="p-60">Other assumptions may not reflect biological reality. First, our simulation forced QTL to be biallelic, but, as noted by <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Jannink (2010)</a> and suggested in <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Buckler <em>et al.</em> (2009)</a>, many QTL may have multiallelic genetic architecture. Second, we assumed the processes of mutation and crossover interference were absent, which is, of course, unrealistic.</p></div><div id="sec-17" class="subsection"><h3>Conclusions</h3><p id="p-61">In our simulation experiment of recurrent genomewide selection, we confirmed the need to update the training population over breeding cycles. Clearly, the LD between QTL and markers in the base population is decaying, likely as a result of recombination. When new data are not added to the training population, the change in LD is not captured, and prediction accuracy collapses. Among the tested methods of updating the training population, adding the lines predicted to have the greatest genotypic value (<em>i.e.</em>, the Top method) is the most attractive. The desirability of this method stems not only from the resulting prediction accuracy and response to selection, but also from its simplicity and practicality. A breeder will undoubtedly desire to confirm the predictions of genotypic value with empirical phenotypic data, especially for the most promising lines or those selected to become parents. Updating the training population becomes simple, then, as this new data can be combined with previous training data. This method also facilitates updating the training model every cycle, likely the best option to capture the changes in LD as a result of recombination, selection, and drift. Nevertheless, our experiment leaves room for additional research, including fine-tuning the updating scenarios to choose the most informative training population from a pool of data. Additionally, optimizing other streams in the breeding program deserves research, including methods of selecting markers and parents. Long-term genomewide selection may benefit greatly from such investigations.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-62">We thank Celeste Falcon and Ahmad Sallam for their contributions to the discussions from which this research originated. We also thank the Minnesota Supercomputing Institute for the high performance computing resources used to conduct the experiments. This research was partially supported by funding from the Minnesota Department of Agriculture, Rahr Malting Company, the United States Wheat and Barley Scab Initiative, and the National Research Initiative or Agriculture and Food Research Initiative Competitive Grants Program (grant no. 2012-67013-19460) from the United States Department of Agriculture National Institute of Food and Agriculture.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-63">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi">www.g3journal.org/lookup/suppl/doi</a>: 10.1534/g3.117.040550/-/DC1.</p></li><li class="fn" id="fn-2"><p id="p-64"><em>Communicating editor: P. J. Brown</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-02-14"><span class="received-label">Received </span>February 14, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-03-10"><span class="accepted-label">Accepted </span>March 10, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Neyhart <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.1" data-doi="10.1186/s12711&amp;ndash;015&amp;ndash;0116&amp;ndash;6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Optimization of genomic selection training populations with a genetic algorithm.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D38%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711%2526ndash%253B015%2526ndash%253B0116%2526ndash%253B6%26rft_id%253Dinfo%253Apmid%252F25943105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711&amp;ndash;015&amp;ndash;0116&amp;ndash;6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25943105&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1499.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.2" data-doi="10.3835/plantgenome2011.02.0007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Asoro</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newell</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accuracy and training population design for genomic selection on quantitative traits in elite North American oats.</span> <abbr class="cit-jnl-abbrev">Plant Genome J.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">132</span>–<span class="cit-lpage">144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%2BJ.%26rft.volume%253D4%26rft.spage%253D132%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.02.0007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.02.0007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Asoro</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newell</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tinker</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic, marker-assisted, and pedigree-BLUP selection methods for beta-glucan concentration in elite oat.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">1894</span>–<span class="cit-lpage">1906</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D53%26rft.spage%253D1894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Auinger</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schönleben</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehermeier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korzun</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Model training across multiple breeding cycles significantly improves genomic prediction accuracy in rye (<em>Secale cereale</em> L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">2043</span>–<span class="cit-lpage">2053</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D2043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-7.5.1499.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-source"><em>Breeding for Quantitative Traits in Plants</em>.</span> <span class="cit-publ-name">Stemma Press</span>, <span class="cit-publ-loc">Woodbury, MN</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomewide selection of parental inbreds: classes of loci and virtual biparental populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">2586</span>–<span class="cit-lpage">2595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D2586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.7" data-doi="10.2135/cropsci2006.11.0690"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Prospects for genomewide selection for quantitative traits in maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1082</span>–<span class="cit-lpage">1090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D1082%26rft.epage%253D1090%26rft.atitle%253DProspects%2Bfor%2BGenomewide%2BSelection%2Bfor%2BQuantitative%2BTraits%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.11.0690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.11.0690&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000247424200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.8" data-doi="10.2135/cropsci2014.07.0460"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Semagn</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mugo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tarekegne</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic gains in grain yield through genomic selection in eight bi-parental maize populations under drought stress.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">154</span>–<span class="cit-lpage">163</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D154%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.07.0460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.07.0460&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blake</span>  <span class="cit-name-given-names">V. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birkett</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matthews</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hane</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The Triticeae Toolbox: combining phenotype and genotype data to advance small-grains breeding.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D9%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.10" data-doi="10.1126/science.1174276"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The genetic architecture of maize flowering time.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">325</span>(<span class="cit-issue">80</span>): <span class="cit-fpage">714</span>–<span class="cit-lpage">718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBuckler%26rft.auinit1%253DE.%2BS.%26rft.volume%253D325%26rft.issue%253D5941%26rft.spage%253D714%26rft.epage%253D718%26rft.atitle%253DThe%2BGenetic%2BArchitecture%2Bof%2BMaize%2BFlowering%2BTime%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1174276%26rft_id%253Dinfo%253Apmid%252F19661422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjUvNTk0MS83MTQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQ5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bustos-Korts</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malosetti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biddulph</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Eeuwijk</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Improvement of predictive ability by uniform coverage of the target genetic space.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">3733</span>–<span class="cit-lpage">3747</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D6%26rft.spage%253D3733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.12" data-doi="10.1186/1471-2164-10-582"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhat</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lonardi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rostoks</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Development and implementation of high-throughput SNP genotyping in barley.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">582</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DClose%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D10%26rft.spage%253D582%26rft.epage%253D582%26rft.atitle%253DDevelopment%2Band%2Bimplementation%2Bof%2Bhigh-throughput%2BSNP%2Bgenotyping%2Bin%2Bbarley.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-582%26rft_id%253Dinfo%253Apmid%252F19961604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-582&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19961604&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1499.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Combs</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Accuracy of genomewide selection for different traits with constant population size, heritability, and number of markers.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">6</span>. Available at: <a href="https://dl.sciencesocieties.org/publications/tpg/abstracts/6/1/plantgenome2012.11.0030">https://dl.sciencesocieties.org/publications/tpg/abstracts/6/1/plantgenome2012.11.0030</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cros</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sánchez</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cochard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flori</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic selection prediction accuracy in a perennial crop: case study of oil palm (<em>Elaeis guineensis</em> Jacq.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">397</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bijma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Inbreeding in genome-wide selection.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">369</span>–<span class="cit-lpage">376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DDaetwyler%26rft.auinit1%253DH.%2BD.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D369%26rft.epage%253D376%26rft.atitle%253DInbreeding%2Bin%2Bgenome-wide%2Bselection.%26rft_id%253Dinfo%253Apmid%252F18076474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18076474&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1499.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251432800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Denis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouvet</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Efficiency of genomic selection with models including dominance effect in the context of <em>Eucalyptus</em> breeding.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">37</span>–<span class="cit-lpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D9%26rft.spage%253D37%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.17" data-doi="10.1534/genetics.107.084301"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Roos</span>  <span class="cit-name-given-names">A. P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spelman</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Linkage disequilibrium and persistence of phase in Holstein-Friesian, Jersey and Angus cattle.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1503</span>–<span class="cit-lpage">1512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.084301%26rft_id%253Dinfo%253Apmid%252F18622038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS8zLzE1MDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQ5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.18" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge regression and other kernels for genomic selection with R package rrBLUP.</span> <abbr class="cit-jnl-abbrev">Plant Genome J.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%2BJ.%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.19" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQ5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.20" data-doi="10.2135/cropsci2008.08.0512"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection for crop improvement.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHeffner%26rft.auinit1%253DE.%2BL.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D12%26rft.atitle%253DGenomic%2BSelection%2Bfor%2BCrop%2BImprovement%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.08.0512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.08.0512&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000263344300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection accuracy using multifamily prediction models in a wheat breeding program.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D65%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-7.5.1499.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-source">Applications of Linear Models in Animal Breeding</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">L. R.</span></span>. <span class="cit-publ-name">University of Guelph</span>, <span class="cit-publ-loc">Guelph</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.23" data-doi="10.1007/BF01245622"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1968</span> <span class="cit-article-title">Linkage disequilibrium in finite populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">226</span>–<span class="cit-lpage">231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D38%26rft.spage%253D226%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01245622%26rft_id%253Dinfo%253Apmid%252F24442307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01245622&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24442307&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1499.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.24" data-doi="10.1007/s00122-014-2418-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Isidro</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Training set optimization under population structure in genomic selection.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">145</span>–<span class="cit-lpage">158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D145%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2418-4%26rft_id%253Dinfo%253Apmid%252F25367380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2418-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25367380&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1499.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.25" data-doi="10.1186/1297-9686-42-35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Dynamics of long-term genomic selection.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DJannink%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D42%26rft.spage%253D35%26rft.epage%253D35%26rft.atitle%253DDynamics%2Bof%2Blong-term%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-42-35%26rft_id%253Dinfo%253Apmid%252F20712894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-42-35&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20712894&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1499.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.26" data-doi="10.1371/journal.pone.0162755"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karaman</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Firat</span>  <span class="cit-name-given-names">M. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">An upper bound for accuracy of prediction using GBLUP.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0162755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0162755&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.27" data-doi="10.1371/journal.pone.0036674"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chagné</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bink</span>  <span class="cit-name-given-names">M. C. A M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Volz</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitworth</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection for fruit quality traits in apple (<em>Malus</em>×<em>domestica</em> Borkh.).</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e36674</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DKumar%26rft.auinit1%253DS.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De36674%26rft.epage%253De36674%26rft.atitle%253DGenomic%2Bselection%2Bfor%2Bfruit%2Bquality%2Btraits%2Bin%2Bapple%2B%2528Malus%25C3%2597domestica%2BBorkh.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0036674%26rft_id%253Dinfo%253Apmid%252F22574211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0036674&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22574211&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1499.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.28" data-doi="10.1051/gse:19930604"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laloe</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Precision and information in linear models of genetic evaluation.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">557</span>–<span class="cit-lpage">576</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D25%26rft.spage%253D557%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A19930604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:19930604&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lande</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Efficiency of marker-assisted selection in the improvement of quantitative traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">743</span>–<span class="cit-lpage">756</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLande%26rft.auinit1%253DR.%26rft.volume%253D124%26rft.issue%253D3%26rft.spage%253D743%26rft.epage%253D756%26rft.atitle%253DEfficiency%2Bof%2Bmarker-assisted%2Bselection%2Bin%2Bthe%2Bimprovement%2Bof%2Bquantitative%2Btraits.%26rft_id%253Dinfo%253Apmid%252F1968875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI0LzMvNzQzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lian</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Prediction of genetic variance in biparental maize populations: genomewide marker effects <em>vs.</em> mean genetic variance in prior populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1181</span>–<span class="cit-lpage">1188</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D1181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.31" data-doi="10.2135/cropsci2014.12.0827"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Adding genetically distant individuals to training populations reduces genomic prediction accuracy in barley.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">2657</span>–<span class="cit-lpage">2667</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D2657%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.12.0827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.12.0827&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-book" id="cit-7.5.1499.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asoro</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection in plant breeding: knowledge and prospects</span>, pp. <span class="cit-fpage">77</span>–<span class="cit-lpage">123</span> in <span class="cit-source">Advances in Agronomy</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Sparks</span>  <span class="cit-name-given-names">D. L.</span></span>. <span class="cit-publ-name">Elsevier Inc.</span>, <span class="cit-publ-loc">Amsterdam</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.33" data-doi="10.2135/cropsci2011.09.0503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Potential and optimization of genomic selection for Fusarium head blight resistance in six-row barley.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">1609</span>–<span class="cit-lpage">1621</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D1609%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.09.0503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.09.0503&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000305804900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.34" data-doi="10.1007/s00122-009-1166-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenzana</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Accuracy of genotypic value predictions for marker-based selection in biparental plant populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLorenzana%26rft.auinit1%253DR.%2BE.%26rft.volume%253D120%26rft.issue%253D1%26rft.spage%253D151%26rft.epage%253D161%26rft.atitle%253DAccuracy%2Bof%2Bgenotypic%2Bvalue%2Bpredictions%2Bfor%2Bmarker-based%2Bselection%2Bin%2Bbiparental%2Bplant%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1166-3%26rft_id%253Dinfo%253Apmid%252F19841887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1166-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19841887&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1499.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271939900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQ5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mohammadi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiede</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">PopVar: a genome-wide procedure for predicting genetic variance and correlated response in biparental breeding populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">2068</span>–<span class="cit-lpage">2077</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D2068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.37" data-doi="10.3835/plantgenome2011.08.0023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muñoz-Amatriaín</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moscou</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhat</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svensson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartoš</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">An improved consensus linkage map of barley based on flow-sorted chromosomes and single nucleotide polymorphism markers.</span> <abbr class="cit-jnl-abbrev">Plant Genome J.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">238</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%2BJ.%26rft.volume%253D4%26rft.spage%253D238%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000312661700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-book" id="cit-7.5.1499.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-source"><em>R: A Language and Environment for Statistical Computing</em>.</span> <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.39" data-doi="10.1111/j.1469-8137.2011.04038.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F. R.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sansaloni</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petroli</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Missiaggia</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection for growth and wood quality in <em>Eucalyptus</em>: capturing the missing heritability and accelerating breeding for complex traits in forest trees.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">116</span>–<span class="cit-lpage">128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DResende%26rft.auinit1%253DM.%2BD.%26rft.volume%253D194%26rft.issue%253D1%26rft.spage%253D116%26rft.epage%253D128%26rft.atitle%253DGenomic%2Bselection%2Bfor%2Bgrowth%2Band%2Bwood%2Bquality%2Bin%2BEucalyptus%253A%2Bcapturing%2Bthe%2Bmissing%2Bheritability%2Band%2Baccelerating%2Bbreeding%2Bfor%2Bcomplex%2Btraits%2Bin%2Bforest%2Btrees.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2011.04038.x%26rft_id%253Dinfo%253Apmid%252F22309312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2011.04038.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22309312&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1499.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300719400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.40" data-doi="10.1534/genetics.112.141473"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rincent</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laloe</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altmann</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunel</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Maximizing the reliability of genomic selection by optimizing the calibration set of reference individuals: comparison of methods in two diverse groups of maize inbreds (<em>Zea mays</em> L.).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">715</span>–<span class="cit-lpage">728</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.141473%26rft_id%253Dinfo%253Apmid%252F22865733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzIvNzE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huerta-Espino</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhavani</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Efficient use of historical data for genomic selection: a case study of stem rust resistance in wheat.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">8</span>. Available at: <a href="https://dl.sciencesocieties.org/publications/tpg/abstracts/8/1/plantgenome2014.09.0046">https://dl.sciencesocieties.org/publications/tpg/abstracts/8/1/plantgenome2014.09.0046</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sallam</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Assessing genomic selection prediction accuracy in a dynamic barley breeding population.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">8</span>. Available at: <a href="https://dl.sciencesocieties.org/publications/tpg/abstracts/8/1/plantgenome2014.05.0020">https://dl.sciencesocieties.org/publications/tpg/abstracts/8/1/plantgenome2014.05.0020</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.43" data-doi="10.2527/jas.2009-1975"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Toosi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic selection in admixed and crossbred populations.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">32</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2009-1975%26rft_id%253Dinfo%253Apmid%252F19749023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2009-1975&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19749023&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1499.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.44" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1499.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction contributing to a promising global strategy to turbocharge gene banks.</span> <abbr class="cit-jnl-abbrev">Nat. Plants</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">16150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BPlants%26rft.volume%253D2%26rft.spage%253D16150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1499.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sallam</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kantarski</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Establishment and optimization of genomic selection to accelerate the domestication and improvement of intermediate wheatgrass.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span>. Available at: <a href="https://dl.sciencesocieties.org/publications/tpg/abstracts/9/1/plantgenome2015.07.0059">https://dl.sciencesocieties.org/publications/tpg/abstracts/9/1/plantgenome2015.07.0059</a>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/5/1499.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/5/1487" title="Paternal Induction of Hybrid Dysgenesis in Drosophila melanogaster Is Weakly Correlated with Both P-Element and hobo Element Dosage" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/5/1511" title="Modulating Crossover Frequency and Interference for Obligate Crossovers in Saccharomyces cerevisiae Meiosis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/5"><span>Volume 7 Issue 5, May 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/5.cover-source.jpg?itok=VWLvGstM" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1499" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-NGSzlFa9yMnjdfYDkn4loB-fprgWaZSlOt2lVhDOrCM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/5/1499" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/5/1499" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/5/1499" class="active" data-icon-position="" data-hide-link-title="0">Evaluating Methods of Updating Training Data in Long-Term Genomewide Selection</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/484256" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-yEX_UWdt8U72mBs38oC-EJdJ-gOffPYGoiZd6MLHgl4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/5/1499.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/5/1499" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-weuNcAB7wib_JXqEyMrDxUq69uu_tVLTbsScoGYdH-g" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/5/1499" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/5/1499.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484256" id="node484256--2" data-pisa="ggg;7/5/1499" data-pisa-master="ggg;g3.117.040550" data-apath="/ggg/7/5/1499.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Evaluating Methods of Updating Training Data in Long-Term Genomewide Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jeffrey L. Neyhart</span>, <span class="highwire-citation-author" data-delta="1">Tyler Tiede</span>, <span class="highwire-citation-author" data-delta="2">Aaron J. Lorenz</span> and <span class="highwire-citation-author" data-delta="3">Kevin P. Smith</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1499-1510;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.040550 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484256--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jeffrey L. Neyhart </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BL.%2BNeyhart%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neyhart%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeffrey%2BL.%2BNeyhart%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tyler Tiede </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tyler%2BTiede%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tiede%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATyler%2BTiede%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Aaron J. Lorenz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BJ.%2BLorenz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lorenz%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAaron%2BJ.%2BLorenz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kevin P. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BP.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20KP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKevin%2BP.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:smith376@umn.edu" class="" data-icon-position="" data-hide-link-title="0">smith376@umn.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/5/1499" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/5/1499&amp;title=Evaluating%20Methods%20of%20Updating%20Training%20Data%20in%20Long-Term%20Genomewide%20Selection" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/5/1499&amp;title=Evaluating%20Methods%20of%20Updating%20Training%20Data%20in%20Long-Term%20Genomewide%20Selection" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/5/1499&amp;title=Evaluating%20Methods%20of%20Updating%20Training%20Data%20in%20Long-Term%20Genomewide%20Selection" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/5/1499&amp;text=Evaluating%20Methods%20of%20Updating%20Training%20Data%20in%20Long-Term%20Genomewide%20Selection" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/5/1499&amp;title=Evaluating%20Methods%20of%20Updating%20Training%20Data%20in%20Long-Term%20Genomewide%20Selection" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/5/1499&amp;t=Evaluating%20Methods%20of%20Updating%20Training%20Data%20in%20Long-Term%20Genomewide%20Selection" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/5/1499&amp;title=Evaluating%20Methods%20of%20Updating%20Training%20Data%20in%20Long-Term%20Genomewide%20Selection" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/5/1499&amp;title=Evaluating%20Methods%20of%20Updating%20Training%20Data%20in%20Long-Term%20Genomewide%20Selection" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/484256/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484256" id="node484256--3" data-pisa="ggg;7/5/1499" data-pisa-master="ggg;g3.117.040550" data-apath="/ggg/7/5/1499.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Evaluating Methods of Updating Training Data in Long-Term Genomewide Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jeffrey L. Neyhart</span>, <span class="highwire-citation-author" data-delta="1">Tyler Tiede</span>, <span class="highwire-citation-author" data-delta="2">Aaron J. Lorenz</span> and <span class="highwire-citation-author" data-delta="3">Kevin P. Smith</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1499-1510;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.040550 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484256--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jeffrey L. Neyhart </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BL.%2BNeyhart%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neyhart%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeffrey%2BL.%2BNeyhart%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tyler Tiede </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tyler%2BTiede%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tiede%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATyler%2BTiede%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Aaron J. Lorenz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BJ.%2BLorenz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lorenz%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAaron%2BJ.%2BLorenz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kevin P. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BP.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20KP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKevin%2BP.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:smith376@umn.edu" class="" data-icon-position="" data-hide-link-title="0">smith376@umn.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1499&amp;rft.epage=1510&amp;rft.atitle=Evaluating%20Methods%20of%20Updating%20Training%20Data%20in%20Long-Term%20Genomewide%20Selection&amp;rft.volume=7&amp;rft.issue=5&amp;rft.date=2017-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Neyhart%2C+Jeffrey+L.&amp;rft.au=Tiede%2C+Tyler&amp;rft.au=Lorenz%2C+Aaron+J.&amp;rft.au=Smith%2C+Kevin+P."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/484256/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/484256/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/484256/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/484256/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/484256/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/484256/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/484256/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/484256/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/484256/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/484256/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/484256/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/484256/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/5/1499" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/5/1499.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/5/1499.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/5/1499.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/5/1499.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
