<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/5/1511" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/5/1511.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/5/1511.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/5/1511.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/5/1511.atom" />
<meta name="HW.pisa" content="ggg;7/5/1511" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Modulating Crossover Frequency and Interference for Obligate Crossovers in Saccharomyces cerevisiae Meiosis" />
<meta name="DC.Identifier" content="10.1534/g3.117.040071" />
<meta name="DC.Date" content="2017-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Chakraborty et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Meiotic crossover frequencies show wide variation among organisms. But most organisms maintain at least one crossover per homolog pair (obligate crossover). In Saccharomyces cerevisiae , previous studies have shown crossover frequencies are reduced in the mismatch repair related mutant [mlh3][1] Δ and enhanced in a meiotic checkpoint mutant [pch2][2] Δ by up to twofold at specific chromosomal loci, but both mutants maintain high spore viability. We analyzed meiotic recombination events genome-wide in [mlh3][1] Δ, [pch2][2] Δ, and [mlh3][1] Δ [pch2][2] Δ mutants to test the effect of variation in crossover frequency on obligate crossovers. [mlh3][1] Δ showed ∼30% genome-wide reduction in crossovers (64 crossovers per meiosis) and loss of the obligate crossover, but nonexchange chromosomes were efficiently segregated. [pch2][2] Δ showed ∼50% genome-wide increase in crossover frequency (137 crossovers per meiosis), elevated noncrossovers as well as loss of chromosome size dependent double-strand break formation. Meiotic defects associated with [pch2][2]∆ did not cause significant increase in nonexchange chromosome frequency. Crossovers were restored to wild-type frequency in the double mutant [mlh3][1] Δ [pch2][2] Δ (100 crossovers per meiosis), but obligate crossovers were compromised. Genetic interference was reduced in [mlh3][1] Δ, [pch2][2] Δ, and [mlh3][1] Δ [pch2][2] Δ. Triple mutant analysis of [mlh3][1] Δ [pch2][2] Δ with other resolvase mutants showed that most of the crossovers in [mlh3][1] Δ [pch2][2] Δ are made through the [Mus81][3]-[Mms4][4] pathway. These results are consistent with a requirement for increased crossover frequencies in the absence of genetic interference for obligate crossovers. In conclusion, these data suggest crossover frequencies and the strength of genetic interference in an organism are mutually optimized to ensure obligate crossovers.

 [1]: http://www.yeastgenome.org/locus/S000006085/overview
 [2]: http://www.yeastgenome.org/locus/S000000390/overview
 [3]: http://www.yeastgenome.org/locus/S000002794/overview
 [4]: http://www.yeastgenome.org/locus/S000000302/overview" />
<meta name="DC.Contributor" content="Parijat Chakraborty" />
<meta name="DC.Contributor" content="Ajith V. Pankajam" />
<meta name="DC.Contributor" content="Gen Lin" />
<meta name="DC.Contributor" content="Abhishek Dutta" />
<meta name="DC.Contributor" content="G. Nandanan Krishnaprasad" />
<meta name="DC.Contributor" content="Manu M. Tekkedil" />
<meta name="DC.Contributor" content="Akira Shinohara" />
<meta name="DC.Contributor" content="Lars M. Steinmetz" />
<meta name="DC.Contributor" content="K. Thazath Nishant" />
<meta name="article:published_time" content="2017-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Modulating Crossover Frequency and Interference for Obligate Crossovers in Saccharomyces cerevisiae Meiosis" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Meiotic crossover frequencies show wide variation among organisms. But most organisms maintain at least one crossover per homolog pair (obligate crossover). In &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt;, previous studies have shown crossover frequencies are reduced in the mismatch repair related mutant &lt;i&gt;mlh3&lt;/i&gt;Δ and enhanced in a meiotic checkpoint mutant &lt;i&gt;pch2&lt;/i&gt;Δ by up to twofold at specific chromosomal loci, but both mutants maintain high spore viability. We analyzed meiotic recombination events genome-wide in &lt;i&gt;mlh3&lt;/i&gt;Δ, &lt;i&gt;pch2&lt;/i&gt;Δ, and &lt;i&gt;mlh3&lt;/i&gt;Δ &lt;i&gt;pch2&lt;/i&gt;Δ mutants to test the effect of variation in crossover frequency on obligate crossovers. &lt;i&gt;mlh3&lt;/i&gt;Δ showed ∼30% genome-wide reduction in crossovers (64 crossovers per meiosis) and loss of the obligate crossover, but nonexchange chromosomes were efficiently segregated. &lt;i&gt;pch2&lt;/i&gt;Δ showed ∼50% genome-wide increase in crossover frequency (137 crossovers per meiosis), elevated noncrossovers as well as loss of chromosome size dependent double-strand break formation. Meiotic defects associated with &lt;i&gt;pch2∆&lt;/i&gt; did not cause significant increase in nonexchange chromosome frequency. Crossovers were restored to wild-type frequency in the double mutant &lt;i&gt;mlh3&lt;/i&gt;Δ &lt;i&gt;pch2&lt;/i&gt;Δ (100 crossovers per meiosis), but obligate crossovers were compromised. Genetic interference was reduced in &lt;i&gt;mlh3&lt;/i&gt;Δ, &lt;i&gt;pch2&lt;/i&gt;Δ, and &lt;i&gt;mlh3&lt;/i&gt;Δ &lt;i&gt;pch2&lt;/i&gt;Δ. Triple mutant analysis of &lt;i&gt;mlh3&lt;/i&gt;Δ &lt;i&gt;pch2&lt;/i&gt;Δ with other resolvase mutants showed that most of the crossovers in &lt;i&gt;mlh3&lt;/i&gt;Δ &lt;i&gt;pch2&lt;/i&gt;Δ are made through the Mus81-Mms4 pathway. These results are consistent with a requirement for increased crossover frequencies in the absence of genetic interference for obligate crossovers. In conclusion, these data suggest crossover frequencies and the strength of genetic interference in an organism are mutually optimized to ensure obligate crossovers.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/05/01" />
<meta name="citation_mjid" content="ggg;7/5/1511" />
<meta name="citation_id" content="7/5/1511" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/5/1511" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/5/1511.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/5/1511.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/5/1511.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.040071" />
<meta name="citation_pmid" content="28315832" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1511" />
<meta name="citation_lastpage" content="1524" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Parijat Chakraborty" />
<meta name="citation_author_institution" content="School of Biology, Indian Institute of Science Education and Research, Thiruvananthapuram, Trivandrum 695016, India" />
<meta name="citation_author" content="Ajith V. Pankajam" />
<meta name="citation_author_institution" content="School of Biology, Indian Institute of Science Education and Research, Thiruvananthapuram, Trivandrum 695016, India" />
<meta name="citation_author" content="Gen Lin" />
<meta name="citation_author_institution" content="Genome Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany" />
<meta name="citation_author" content="Abhishek Dutta" />
<meta name="citation_author_institution" content="School of Biology, Indian Institute of Science Education and Research, Thiruvananthapuram, Trivandrum 695016, India" />
<meta name="citation_author" content="G. Nandanan Krishnaprasad" />
<meta name="citation_author_institution" content="School of Biology, Indian Institute of Science Education and Research, Thiruvananthapuram, Trivandrum 695016, India" />
<meta name="citation_author" content="Manu M. Tekkedil" />
<meta name="citation_author_institution" content="Genome Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany" />
<meta name="citation_author" content="Akira Shinohara" />
<meta name="citation_author_institution" content="Institute for Protein Research, Osaka University, 565-0871, Japan" />
<meta name="citation_author" content="Lars M. Steinmetz" />
<meta name="citation_author_institution" content="Genome Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany" />
<meta name="citation_author_institution" content="Department of Genetics, Stanford University, California 94305" />
<meta name="citation_author_institution" content="Stanford Genome Technology Center, Palo Alto, California 94304" />
<meta name="citation_author" content="K. Thazath Nishant" />
<meta name="citation_author_institution" content="School of Biology, Indian Institute of Science Education and Research, Thiruvananthapuram, Trivandrum 695016, India" />
<meta name="citation_author_institution" content="Center for Computation, Modelling and Simulation, Indian Institute of Science Education and Research, Thiruvananthapuram, Trivandrum 695016, India" />
<meta name="citation_author_email" content="nishantkt@iisertvm.ac.in" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=T. Allers;citation_author=M. Lichten;citation_title=Differential timing and control of noncrossover and crossover recombination during meiosis.;citation_pages=47-57;citation_volume=106;citation_year=2001;citation_issue=1;citation_pmid=11461701;citation_doi=10.1016/S0092-8674(01)00416-0" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=C. M. Anderson;citation_author=S. Y. Chen;citation_author=M. T. Dimon;citation_author=A. Oke;citation_author=J. L. DeRisi;citation_title=ReCombine: a suite of programs for detection and analysis of meiotic recombination in whole-genome datasets.;citation_pages=e25509-e25509;citation_volume=6;citation_year=2011;citation_issue=10;citation_pmid=22046241;citation_doi=10.1371/journal.pone.0025509" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=C. M. Anderson;citation_author=A. Oke;citation_author=P. Yam;citation_author=T. Zhuge;citation_author=J. C. Fung;citation_title=Reduced crossover interference and increased ZMM-independent recombination in the absence of Tel1/ATM.;citation_pages=e1005478;citation_volume=11;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=J. L. Argueso;citation_author=A. W. Kijas;citation_author=S. Sarin;citation_author=J. Heck;citation_author=M. Waase;citation_title=Systematic Mutagenesis of the Saccharomyces cerevisiae MLH1 Gene Reveals Distinct Roles for Mlh1p in Meiotic Crossing Over and in Vegetative and Meiotic Mismatch Repair;citation_pages=873-886;citation_volume=23;citation_year=2003;citation_issue=3;citation_pmid=12529393;citation_doi=10.1128/MCB.23.3.873-886.2003" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. L. Argueso;citation_author=J. Wanat;citation_author=Z. Gemici;citation_author=E. Alani;citation_title=Competing Crossover Pathways Act During Meiosis in Saccharomyces cerevisiae;citation_pages=1805-1816;citation_volume=168;citation_year=2004;citation_issue=4;citation_pmid=15611158;citation_doi=10.1534/genetics.104.032912" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. M. Baker;citation_author=A. W. Plug;citation_author=T. A. Prolla;citation_author=C. E. Bronner;citation_author=A. C. Harris;citation_title=Involvement of mouse Mlh1 in DNA mismatch repair and meiotic crossing over.;citation_pages=336-342;citation_volume=13;citation_year=1996;citation_issue=3;citation_pmid=8673133;citation_doi=10.1038/ng0796-336" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=M. Barchi;citation_author=I. Roig;citation_author=M. Di Giacomo;citation_author=D. G. de Rooij;citation_author=S. Keeney;citation_title=ATM promotes the obligate XY crossover and both crossover control and chromosome axis integrity on autosomes.;citation_pages=e1000076-e1000076;citation_volume=4;citation_year=2008;citation_issue=5;citation_pmid=18497861;citation_doi=10.1371/journal.pgen.1000076" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=T. Benaglia;citation_author=D. Chauveau;citation_author=D. Hunter;citation_author=D. Young;citation_title=mixtools: an R package for analyzing finite mixture models.;citation_pages=1-29;citation_volume=32;citation_year=2009;citation_doi=10.18637/jss.v032.i06" />
<meta name="citation_reference" content="citation_journal_abbrev=Curr Genomics;citation_author=L. E. Berchowitz;citation_author=G. P. Copenhaver;citation_title=Genetic interference: don't stand so close to me.;citation_pages=91-102;citation_volume=11;citation_year=2010;citation_issue=2;citation_pmid=20885817;citation_doi=10.2174/138920210790886835" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. K. Bishop;citation_title=RecA homologs Dmc1 and Rad51 interact to form multiple nuclear complexes prior to meiotic chromosome synapsis.;citation_pages=1081-1092;citation_volume=79;citation_year=1994;citation_issue=6;citation_pmid=7528104;citation_doi=10.1016/0092-8674(94)90038-8" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data.;citation_volume=30;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=G. V. Borner;citation_author=N. Kleckner;citation_author=N. Hunter;citation_title=Crossover/noncrossover differentiation, synaptonemal complex formation, and regulatory surveillance at the leptotene/zygotene transition of meiosis.;citation_pages=29-45;citation_volume=117;citation_year=2004;citation_issue=1;citation_pmid=15066280;citation_doi=10.1016/S0092-8674(04)00292-2" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=G. V. Borner;citation_author=A. Barot;citation_author=N. Kleckner;citation_title=Yeast Pch2 promotes domainal axis organization, timely recombination progression, and arrest of defective recombinosomes during meiosis.;citation_volume=105;citation_year=2008;citation_pmid=18305165;citation_doi=10.1073/pnas.0711864105" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. Chen;citation_author=A. Jomaa;citation_author=J. Ortega;citation_author=E. E. Alani;citation_title=Pch2 is a hexameric ring ATPase that remodels the chromosome axis protein Hop1.;citation_volume=111;citation_year=2014;citation_pmid=24367111;citation_doi=10.1073/pnas.1310755111" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=S. Y. Chen;citation_author=T. Tsubouchi;citation_author=B. Rockmill;citation_author=J. S. Sandler;citation_author=D. R. Richards;citation_title=Global analysis of the meiotic crossover landscape.;citation_pages=401-415;citation_volume=15;citation_year=2008;citation_issue=3;citation_pmid=18691940;citation_doi=10.1016/j.devcel.2008.07.006" />
<meta name="citation_reference" content="citation_journal_title=Nature Cell Biology;citation_journal_abbrev=Nature Cell Biology;citation_author=F. Cole;citation_author=L. Kauppi;citation_author=J. Lange;citation_author=I. Roig;citation_author=R. Wang;citation_title=Homeostatic control of recombination is implemented progressively in mouse meiosis.;citation_pages=424-430;citation_volume=14;citation_year=2012;citation_issue=4;citation_pmid=22388890;citation_doi=10.1038/ncb2451" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. P. Copenhaver;citation_author=E. A. Housworth;citation_author=F. W. Stahl;citation_title=Crossover Interference in Arabidopsis;citation_pages=1631-1639;citation_volume=160;citation_year=2002;citation_issue=4;citation_pmid=11973316" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=V. E. Cotton;citation_author=E. R. Hoffmann;citation_author=R. H. Borts;citation_title=Distinct regulation of Mlh1p heterodimers in meiosis and mitosis in Saccharomyces cerevisiae.;citation_volume=185;citation_year=2010;citation_pmid=20382827;citation_doi=10.1534/genetics.110.116806" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. Davis;citation_author=G. R. Smith;citation_title=Nonrandom Homolog Segregation at Meiosis I in Schizosaccharomyces pombe Mutants Lacking Recombination;citation_pages=857-874;citation_volume=163;citation_year=2003;citation_issue=3;citation_pmid=12663528" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. de los Santos;citation_author=N. Hunter;citation_author=C. Lee;citation_author=B. Larkin;citation_author=J. Loidl;citation_title=The Mus81/Mms4 Endonuclease Acts Independently of Double-Holliday Junction Resolution to Promote a Distinct Subset of Crossovers During Meiosis in Budding Yeast;citation_pages=81-94;citation_volume=164;citation_year=2003;citation_issue=1;citation_pmid=12750322" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=A. De Muyt;citation_author=L. Jessop;citation_author=E. Kolar;citation_author=A. Sourirajan;citation_author=J. Chen;citation_title=BLM helicase ortholog Sgs1 is a central regulator of meiotic recombination intermediate metabolism.;citation_pages=43-53;citation_volume=46;citation_year=2012;citation_issue=1;citation_pmid=22500736;citation_doi=10.1016/j.molcel.2012.02.020" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. A. DePristo;citation_author=E. Banks;citation_author=R. Poplin;citation_author=K. V. Garimella;citation_author=J. R. Maguire;citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data.;citation_pages=491-498;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21478889;citation_doi=10.1038/ng.806" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. Farmer;citation_author=E. J. Hong;citation_author=W. K. Leung;citation_author=B. Argunhan;citation_author=Y. Terentyev;citation_title=Budding yeast Pch2, a widely conserved meiotic protein, is involved in the initiation of meiotic recombination.;citation_pages=e39724-e39724;citation_volume=7;citation_year=2012;citation_issue=6;citation_pmid=22745819;citation_doi=10.1371/journal.pone.0039724" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. Fledel-Alon;citation_author=D. J. Wilson;citation_author=K. Broman;citation_author=X. Wen;citation_author=C. Ober;citation_title=Broad-scale recombination patterns underlying proper disjunction in humans.;citation_pages=e1000658-e1000658;citation_volume=5;citation_year=2009;citation_issue=9;citation_pmid=19763175;citation_doi=10.1371/journal.pgen.1000658" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. J. Getz;citation_author=S. A. Banse;citation_author=L. S. Young;citation_author=A. V. Banse;citation_author=J. Swanson;citation_title=Reduced Mismatch Repair of Heteroduplexes Reveals &quot;Non&quot;-interfering Crossing Over in Wild-Type Saccharomyces cerevisiae;citation_pages=1251-1269;citation_volume=178;citation_year=2008;citation_issue=3;citation_pmid=18385111;citation_doi=10.1534/genetics.106.067603" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=R. D. Gietz;citation_author=R. H. Schiestl;citation_author=A. R. Willems;citation_author=R. A. Woods;citation_title=Studies on the transformation of intact yeast cells by the LiAc/SS-DNA/PEG procedure.;citation_pages=355-360;citation_volume=11;citation_year=1995;citation_issue=4;citation_pmid=7785336;citation_doi=10.1002/yea.320110408" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=A. L. Goldstein;citation_author=J. H. McCusker;citation_title=Three new dominant drug resistance cassettes for gene disruption in Saccharomyces cerevisiae.;citation_pages=1541-1553;citation_volume=15;citation_year=1999;citation_issue=14;citation_pmid=10514571;citation_doi=10.1002/(SICI)1097-0061(199910)15:14&lt;1541::AID-YEA476&gt;3.0.CO;2-K" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=V. Guacci;citation_author=D. B. Kaback;citation_title=Distributive disjunction of authentic chromosomes in Saccharomyces cerevisiae.;citation_pages=475-488;citation_volume=127;citation_year=1991;citation_issue=3;citation_pmid=2016050" />
<meta name="citation_reference" content="citation_journal_title=Developmental genetics;citation_journal_abbrev=Dev Genet;citation_author=R. S. Hawley;citation_author=H. Irick;citation_author=A. E. Zitron;citation_author=D. A. Haddox;citation_author=A. Lohe;citation_title=There are two mechanisms of achiasmate segregation in Drosophila females, one of which requires heterochromatic homology.;citation_pages=440-467;citation_volume=13;citation_year=1992;citation_issue=6;citation_pmid=1304424;citation_doi=10.1002/dvg.1020130608" />
<meta name="citation_reference" content="citation_journal_title=G3 Bethesda;citation_author=M. Higashide;citation_author=M. Shinohara;citation_title=Budding yeast SLX4 contributes to the appropriate distribution of crossovers and meiotic double-strand break formation on bivalents during meiosis.;citation_pages=2033-2042;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=K. J. Hillers;citation_title=Crossover interference.;citation_pages=R1036-R1037;citation_volume=14;citation_year=2004;citation_issue=24;citation_pmid=15620632;citation_doi=10.1016/j.cub.2004.11.038" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=K. J. Hillers;citation_author=A. M. Villeneuve;citation_title=Chromosome-wide control of meiotic crossing over in C. elegans.;citation_pages=1641-1647;citation_volume=13;citation_year=2003;citation_issue=18;citation_pmid=13678597;citation_doi=10.1016/j.cub.2003.08.026" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=H. C. Ho;citation_author=S. M. Burgess;citation_title=Pch2 acts through Xrs2 and Tel1/ATM to modulate interhomolog bias and checkpoint function during meiosis.;citation_pages=e1002351-e1002351;citation_volume=7;citation_year=2011;citation_issue=11;citation_pmid=22072981;citation_doi=10.1371/journal.pgen.1002351" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=A. Hochwagen;citation_author=W. H. Tham;citation_author=G. A. Brar;citation_author=A. Amon;citation_title=The FK506 binding protein Fpr3 counteracts protein phosphatase 1 to maintain meiotic recombination checkpoint activity.;citation_pages=861-873;citation_volume=122;citation_year=2005;citation_issue=6;citation_pmid=16179256;citation_doi=10.1016/j.cell.2005.07.010" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=E. A. Housworth;citation_author=F. W. Stahl;citation_title=Crossover interference in humans.;citation_pages=188-197;citation_volume=73;citation_year=2003;citation_issue=1;citation_pmid=12772089;citation_doi=10.1086/376610" />
<meta name="citation_reference" content="citation_journal_title=Cell cycle (Georgetown, Tex.);citation_journal_abbrev=Cell Cycle;citation_author=N. Hunter;citation_title=Double duty for Exo1 during meiotic recombination.;citation_pages=2607-2609;citation_volume=10;citation_year=2011;citation_issue=16;citation_pmid=21785266;citation_doi=10.4161/cc.10.16.16452" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=N. Hunter;citation_author=N. Kleckner;citation_title=The single-end invasion: an asymmetric intermediate at the double-strand break to double-holliday junction transition of meiotic recombination.;citation_pages=59-70;citation_volume=106;citation_year=2001;citation_issue=1;citation_pmid=11461702;citation_doi=10.1016/S0092-8674(01)00430-5" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=R. W. Hyppa;citation_author=G. R. Smith;citation_title=Crossover invariance determined by partner choice for meiotic DNA break repair.;citation_pages=243-255;citation_volume=142;citation_year=2010;citation_issue=2;citation_pmid=20655467;citation_doi=10.1016/j.cell.2010.05.041" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=N. Joshi;citation_author=A. Barot;citation_author=C. Jamison;citation_author=G. V. Borner;citation_title=Pch2 links chromosome axis remodeling at future crossover sites and crossover distribution during yeast meiosis.;citation_pages=e1000557-e1000557;citation_volume=5;citation_year=2009;citation_issue=7;citation_pmid=19629172;citation_doi=10.1371/journal.pgen.1000557" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=N. Joshi;citation_author=M. S. Brown;citation_author=D. K. Bishop;citation_author=G. V. Borner;citation_title=Gradual implementation of the meiotic recombination program via checkpoint pathways controlled by global DSB levels.;citation_pages=797-811;citation_volume=57;citation_year=2015;citation_pmid=25661491;citation_doi=10.1016/j.molcel.2014.12.027" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. B. Kaback;citation_author=D. Barber;citation_author=J. Mahon;citation_author=J. Lamb;citation_author=J. You;citation_title=Chromosome Size-Dependent Control of Meiotic Reciprocal Recombination in Saccharomyces cerevisiae: The Role of Crossover Interference;citation_pages=1475-1486;citation_volume=152;citation_year=1999;citation_issue=4;citation_pmid=10430577" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=H. Kaur;citation_author=A. De Muyt;citation_author=M. Lichten;citation_title=Top3-Rmi1 DNA single-strand decatenase is integral to the formation and resolution of meiotic recombination intermediates.;citation_pages=583-594;citation_volume=57;citation_year=2015;citation_pmid=25699707;citation_doi=10.1016/j.molcel.2015.01.020" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=B. Kemp;citation_author=R. M. Boumil;citation_author=M. N. Stewart;citation_author=D. S. Dawson;citation_title=A role for centromere pairing in meiotic chromosome segregation.;citation_volume=18;citation_year=2004;citation_pmid=15289462;citation_doi=10.1101/gad.1227304" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=N. Kleckner;citation_author=D. Zickler;citation_author=G. H. Jones;citation_author=J. Dekker;citation_author=R. Padmore;citation_title=A mechanical basis for chromosome function.;citation_volume=101;citation_year=2004;citation_pmid=15299144;citation_doi=10.1073/pnas.0402724101" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=N. K. Kolas;citation_author=A. Svetlanov;citation_author=M. L. Lenzi;citation_author=F. P. Macaluso;citation_author=S. M. Lipkin;citation_title=Localization of MMR proteins on meiotic chromosomes in mice indicates distinct functions during prophase I.;citation_volume=171;citation_year=2005;citation_pmid=16260499;citation_doi=10.1083/jcb.200506170" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. N. Krishnaprasad;citation_author=M. T. Anand;citation_author=G. Lin;citation_author=M. M. Tekkedil;citation_author=L. M. Steinmetz;citation_title=Variation in crossover frequencies perturb crossover assurance without affecting meiotic chromosome segregation in Saccharomyces cerevisiae.;citation_volume=199;citation_year=2015;citation_pmid=25467183;citation_doi=10.1534/genetics.114.172320" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=D. E. Libuda;citation_author=S. Uzawa;citation_author=B. J. Meyer;citation_author=A. M. Villeneuve;citation_title=Meiotic chromosome structures constrain and respond to designation of crossover sites.;citation_pages=703-706;citation_volume=502;citation_year=2013;citation_pmid=24107990;citation_doi=10.1038/nature12577" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. M. Lipkin;citation_author=P. B. Moens;citation_author=V. Wang;citation_author=M. Lenzi;citation_author=D. Shanmugarajah;citation_title=Meiotic arrest and aneuploidy in MLH3-deficient mice.;citation_pages=385-390;citation_volume=31;citation_year=2002;citation_issue=4;citation_pmid=12091911;citation_doi=10.1038/ng861" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. Mancera;citation_author=R. Bourgon;citation_author=A. Brozzi;citation_author=W. Huber;citation_author=L. M. Steinmetz;citation_title=High-resolution mapping of meiotic crossovers and non-crossovers in yeast.;citation_pages=479-485;citation_volume=454;citation_year=2008;citation_issue=7203;citation_pmid=18615017;citation_doi=10.1038/nature07135" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=E. Martini;citation_author=R. L. Diaz;citation_author=N. Hunter;citation_author=S. Keeney;citation_title=Crossover homeostasis in yeast meiosis.;citation_pages=285-295;citation_volume=126;citation_year=2006;citation_issue=2;citation_pmid=16873061;citation_doi=10.1016/j.cell.2006.05.044" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. Matos;citation_author=M. G. Blanco;citation_author=S. Maslen;citation_author=J. M. Skehel;citation_author=S. C. West;citation_title=Regulatory control of the resolution of DNA recombination intermediates during meiosis and mitosis.;citation_pages=158-172;citation_volume=147;citation_year=2011;citation_issue=1;citation_pmid=21962513;citation_doi=10.1016/j.cell.2011.08.032" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. S. McPeek;citation_author=T. P. Speed;citation_title=Modeling interference in genetic recombination.;citation_pages=1031-1044;citation_volume=139;citation_year=1995;citation_issue=2;citation_pmid=7713406" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=H. J. Muller;citation_title=The mechanism of crossing-over.;citation_pages=284-305;citation_volume=50;citation_year=1916;citation_doi=10.1086/279541" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=H. Murakami;citation_author=V. Borde;citation_author=A. Nicolas;citation_author=S. Keeney;citation_title=Gel electrophoresis assays for analyzing DNA double-strand breaks in Saccharomyces cerevisiae at various spatial resolutions.;citation_pages=117-142;citation_volume=557;citation_year=2009;citation_pmid=19799180;citation_doi=10.1007/978-1-59745-527-5_9" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=T. Nakagawa;citation_author=H. Ogawa;citation_title=The Saccharomyces cerevisiae MER3 gene, encoding a novel helicase-like protein, is required for crossover control in meiosis;citation_pages=5714-5723;citation_volume=18;citation_year=1999;citation_issue=20;citation_pmid=10523314;citation_doi=10.1093/emboj/18.20.5714" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. J. Neale;citation_author=S. Keeney;citation_title=Clarifying the mechanics of DNA strand exchange in meiotic recombination.;citation_pages=153-158;citation_volume=442;citation_year=2006;citation_issue=7099;citation_pmid=16838012;citation_doi=10.1038/nature04885" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. Newnham;citation_author=P. Jordan;citation_author=B. Rockmill;citation_author=G. S. Roeder;citation_author=E. Hoffmann;citation_title=The synaptonemal complex protein, Zip1, promotes the segregation of nonexchange chromosomes at meiosis I.;citation_volume=107;citation_year=2010;citation_pmid=20080752;citation_doi=10.1073/pnas.0913435107" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. T. Nishant;citation_author=A. J. Plys;citation_author=E. Alani;citation_title=A mutation in the putative MLH3 endonuclease domain confers a defect in both mismatch repair and meiosis in Saccharomyces cerevisiae.;citation_volume=179;citation_year=2008;citation_pmid=18505871;citation_doi=10.1534/genetics.108.086645" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. E. Novak;citation_author=P. B. Ross-Macdonald;citation_author=G. S. Roeder;citation_title=The Budding Yeast Msh4 Protein Functions in Chromosome Synapsis and the Regulation of Crossover Distribution;citation_pages=1013-1025;citation_volume=158;citation_year=2001;citation_issue=3;citation_pmid=11454751" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=S. D. Oh;citation_author=J. P. Lao;citation_author=A. F. Taylor;citation_author=G. R. Smith;citation_author=N. Hunter;citation_title=RecQ helicase, Sgs1, and XPF family endonuclease, Mus81-Mms4, resolve aberrant joint molecules during meiotic recombination.;citation_pages=324-336;citation_volume=31;citation_year=2008;citation_issue=3;citation_pmid=18691965;citation_doi=10.1016/j.molcel.2008.07.006" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=A. Oke;citation_author=C. M. Anderson;citation_author=P. Yam;citation_author=J. C. Fung;citation_title=Controlling meiotic recombinational repair - specifying the roles of ZMMs, Sgs1 and Mus81/Mms4 in crossover formation.;citation_pages=e1004690;citation_volume=10;citation_year=2014;citation_pmid=25329811;citation_doi=10.1371/journal.pgen.1004690" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. Pan;citation_author=M. Sasaki;citation_author=R. Kniewel;citation_author=H. Murakami;citation_author=H. G. Blitzblau;citation_title=A hierarchical combination of factors shapes the genome-wide topography of yeast meiotic recombination initiation.;citation_pages=719-731;citation_volume=144;citation_year=2011;citation_issue=5;citation_pmid=21376234;citation_doi=10.1016/j.cell.2011.02.009" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. Petronczki;citation_author=M. F. Siomos;citation_author=K. Nasmyth;citation_title=Un menage a quatre: the molecular biology of chromosome segregation in meiosis.;citation_pages=423-440;citation_volume=112;citation_year=2003;citation_issue=4;citation_pmid=12600308;citation_doi=10.1016/S0092-8674(03)00083-7" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=G. S. Roeder;citation_title=Meiotic chromosomes: it takes two to tango;citation_pages=2600-2621;citation_volume=11;citation_year=1997;citation_issue=20;citation_pmid=9334324;citation_doi=10.1101/gad.11.20.2600" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=M. V. Rogacheva;citation_author=C. M. Manhart;citation_author=C. Chen;citation_author=A. Guarne;citation_author=J. Surtees;citation_title=Mlh1-Mlh3, a meiotic crossover and DNA mismatch repair factor, is a Msh2-Msh3-stimulated endonuclease.;citation_volume=289;citation_year=2014;citation_pmid=24403070;citation_doi=10.1074/jbc.M113.534644" />
<meta name="citation_reference" content="citation_author=M. D. Rose;citation_author=F. M. Winston;citation_author=P. Heiter;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=P. A. San-Segundo;citation_author=G. S. Roeder;citation_title=Pch2 links chromatin silencing to meiotic checkpoint control.;citation_pages=313-324;citation_volume=97;citation_year=1999;citation_issue=3;citation_pmid=10319812;citation_doi=10.1016/S0092-8674(00)80741-2" />
<meta name="citation_reference" content="citation_journal_title=Experimental cell research;citation_journal_abbrev=Exp Cell Res;citation_author=M. E. Serrentino;citation_author=V. Borde;citation_title=The spatial regulation of meiotic recombination hotspots: are all DSB hotspots crossover hotspots?;citation_pages=1347-1352;citation_volume=318;citation_year=2012;citation_issue=12;citation_pmid=22487095;citation_doi=10.1016/j.yexcr.2012.03.025" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. Shinohara;citation_author=S. L. Gasior;citation_author=D. K. Bishop;citation_author=A. Shinohara;citation_title=Tid1/Rdh54 promotes colocalization of Rad51 and Dmc1 during meiotic recombination;citation_pages=10814-10819;citation_volume=97;citation_year=2000;citation_issue=20;citation_pmid=11005857;citation_doi=10.1073/pnas.97.20.10814" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. Shinohara;citation_author=S. D. Oh;citation_author=N. Hunter;citation_author=A. Shinohara;citation_title=Crossover assurance and crossover interference are distinctly regulated by the ZMM proteins during yeast meiosis.;citation_pages=299-309;citation_volume=40;citation_year=2008;citation_issue=3;citation_pmid=18297071;citation_doi=10.1038/ng.83" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. A. Shonn;citation_author=R. McCarroll;citation_author=A. W. Murray;citation_title=Requirement of the Spindle Checkpoint for Proper Chromosome Segregation in Budding Yeast Meiosis;citation_pages=300-303;citation_volume=289;citation_year=2000;citation_issue=5477;citation_pmid=10894778;citation_doi=10.1126/science.289.5477.300" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=T. Snowden;citation_author=S. Acharya;citation_author=C. Butz;citation_author=M. Berardini;citation_author=R. Fishel;citation_title=hMSH4-hMSH5 recognizes Holliday Junctions and forms a meiosis-specific sliding clamp that embraces homologous chromosomes.;citation_pages=437-451;citation_volume=15;citation_year=2004;citation_issue=3;citation_pmid=15304223;citation_doi=10.1016/j.molcel.2004.06.040" />
<meta name="citation_reference" content="citation_journal_title=G3 Bethesda;citation_author=M. Sonntag Brown;citation_author=E. Lim;citation_author=C. Chen;citation_author=K. T. Nishant;citation_author=E. Alani;citation_title=Genetic analysis of mlh3 mutations reveals interactions between crossover promoting factors during meiosis in baker’s yeast.;citation_pages=9-22;citation_volume=3;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. W. Stahl;citation_author=H. M. Foss;citation_author=L. S. Young;citation_author=R. H. Borts;citation_author=M. F. Abdullah;citation_title=Does Crossover Interference Count in Saccharomyces cerevisiae?;citation_pages=35-48;citation_volume=168;citation_year=2004;citation_issue=1;citation_pmid=15454525;citation_doi=10.1534/genetics.104.027789" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=V. V. Subramanian;citation_author=A. J. MacQueen;citation_author=G. Vader;citation_author=M. Shinohara;citation_author=A. Sanchez;citation_title=Chromosome synapsis alleviates Mek1-dependent suppression of meiotic DNA repair.;citation_pages=e1002369;citation_volume=14;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Elife;citation_author=X. Sun;citation_author=L. Huang;citation_author=T. E. Markowitz;citation_author=H. G. Blitzblau;citation_author=D. Chen;citation_title=Transcription dynamically patterns the meiotic chromosome-axis interface.;citation_volume=4;citation_year=2015;citation_doi=10.7554/eLife.07424" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. Sym;citation_author=G. S. Roeder;citation_title=Crossover interference is abolished in the absence of a synaptonemal complex protein.;citation_pages=283-292;citation_volume=79;citation_year=1994;citation_issue=2;citation_pmid=7954796;citation_doi=10.1016/0092-8674(94)90197-X" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Terasawa;citation_author=H. Ogawa;citation_author=Y. Tsukamoto;citation_author=M. Shinohara;citation_author=K. Shirahige;citation_title=Meiotic recombination-related DNA synthesis and its implications for cross-over and non-cross-over recombinant formation.;citation_volume=104;citation_year=2007;citation_pmid=17384152;citation_doi=10.1073/pnas.0611490104" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Vader;citation_author=H. G. Blitzblau;citation_author=M. A. Tame;citation_author=J. E. Falk;citation_author=L. Curtin;citation_title=Protection of repetitive DNA borders from self-induced meiotic instability.;citation_pages=115-119;citation_volume=477;citation_year=2011;citation_issue=7362;citation_pmid=21822291;citation_doi=10.1038/nature10331" />
<meta name="citation_reference" content="citation_journal_title=Cell Cycle;citation_author=S. Wang;citation_author=D. Zickler;citation_author=N. Kleckner;citation_author=L. Zhang;citation_title=Meiotic crossover patterns: obligatory crossover, interference and homeostasis in a single process.;citation_pages=305-314;citation_volume=14;citation_year=2015;citation_pmid=25590558;citation_doi=10.4161/15384101.2014.991185" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=L. Wojtasz;citation_author=K. Daniel;citation_author=I. Roig;citation_author=E. Bolcun-Filas;citation_author=H. Xu;citation_title=Mouse HORMAD1 and HORMAD2, two conserved meiotic chromosomal proteins, are depleted from synapsed chromosome axes with the help of TRIP13 AAA-ATPase.;citation_pages=e1000702-e1000702;citation_volume=5;citation_year=2009;citation_issue=10;citation_pmid=19851446;citation_doi=10.1371/journal.pgen.1000702" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=H. Y. Wu;citation_author=S. M. Burgess;citation_title=Two distinct surveillance mechanisms monitor meiotic chromosome metabolism in budding yeast.;citation_pages=2473-2479;citation_volume=16;citation_year=2006;citation_issue=24;citation_pmid=17174924;citation_doi=10.1016/j.cub.2006.10.069" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=K. Zakharyevich;citation_author=S. Tang;citation_author=Y. Ma;citation_author=N. Hunter;citation_title=Delineation of joint molecule resolution pathways in meiosis identifies a crossover-specific resolvase.;citation_pages=334-347;citation_volume=149;citation_year=2012;citation_issue=2;citation_pmid=22500800;citation_doi=10.1016/j.cell.2012.03.023" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. Zanders;citation_author=E. Alani;citation_title=The pch2Delta mutation in baker's yeast alters meiotic crossover levels and confers a defect in crossover interference.;citation_pages=e1000571-e1000571;citation_volume=5;citation_year=2009;citation_issue=7;citation_pmid=19629178;citation_doi=10.1371/journal.pgen.1000571" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Zanders;citation_author=M. Sonntag Brown;citation_author=C. Chen;citation_author=E. Alani;citation_title=Pch2 modulates chromatid partner choice during meiotic double-strand break repair in Saccharomyces cerevisiae.;citation_volume=188;citation_year=2011;citation_pmid=21515575;citation_doi=10.1534/genetics.111.129031" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=L. Zhang;citation_author=S. Wang;citation_author=S. Yin;citation_author=S. Hong;citation_author=K. P. Kim;citation_title=Topoisomerase II mediates meiotic crossover interference.;citation_pages=551-556;citation_volume=511;citation_year=2014;citation_pmid=25043020;citation_doi=10.1038/nature13442" />
<meta name="citation_reference" content="citation_journal_title=Semin. Cell Dev. Biol.;citation_author=D. Zickler;citation_author=N. Kleckner;citation_title=A few of our favorite things: pairing, the bouquet, crossover interference and evolution of meiosis.;citation_pages=135-148;citation_volume=54;citation_year=2016" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Modulating Crossover Frequency and Interference for Obligate Crossovers in Saccharomyces cerevisiae Meiosis" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
<meta name="twitter:description" content="Meiotic crossover frequencies show wide variation among organisms. But most organisms maintain at least one crossover per homolog pair (obligate crossover). In Saccharomyces cerevisiae , previous studies have shown crossover frequencies are reduced in the mismatch repair related mutant [mlh3][1] Δ and enhanced in a meiotic checkpoint mutant [pch2][2] Δ by up to twofold at specific chromosomal loci, but both mutants maintain high spore viability. We analyzed meiotic recombination events genome-wide in [mlh3][1] Δ, [pch2][2] Δ, and [mlh3][1] Δ [pch2][2] Δ mutants to test the effect of variation in crossover frequency on obligate crossovers. [mlh3][1] Δ showed ∼30% genome-wide reduction in crossovers (64 crossovers per meiosis) and loss of the obligate crossover, but nonexchange chromosomes were efficiently segregated. [pch2][2] Δ showed ∼50% genome-wide increase in crossover frequency (137 crossovers per meiosis), elevated noncrossovers as well as loss of chromosome size dependent double-strand break formation. Meiotic defects associated with [pch2][2]∆ did not cause significant increase in nonexchange chromosome frequency. Crossovers were restored to wild-type frequency in the double mutant [mlh3][1] Δ [pch2][2] Δ (100 crossovers per meiosis), but obligate crossovers were compromised. Genetic interference was reduced in [mlh3][1] Δ, [pch2][2] Δ, and [mlh3][1] Δ [pch2][2] Δ. Triple mutant analysis of [mlh3][1] Δ [pch2][2] Δ with other resolvase mutants showed that most of the crossovers in [mlh3][1] Δ [pch2][2] Δ are made through the [Mus81][3]-[Mms4][4] pathway. These results are consistent with a requirement for increased crossover frequencies in the absence of genetic interference for obligate crossovers. In conclusion, these data suggest crossover frequencies and the strength of genetic interference in an organism are mutually optimized to ensure obligate crossovers.

 [1]: http://www.yeastgenome.org/locus/S000006085/overview
 [2]: http://www.yeastgenome.org/locus/S000000390/overview
 [3]: http://www.yeastgenome.org/locus/S000002794/overview
 [4]: http://www.yeastgenome.org/locus/S000000302/overview" />
<meta name="og-title" property="og:title" content="Modulating Crossover Frequency and Interference for Obligate Crossovers in Saccharomyces cerevisiae Meiosis" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/5/1511" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Meiotic crossover frequencies show wide variation among organisms. But most organisms maintain at least one crossover per homolog pair (obligate crossover). In Saccharomyces cerevisiae , previous studies have shown crossover frequencies are reduced in the mismatch repair related mutant [mlh3][1] Δ and enhanced in a meiotic checkpoint mutant [pch2][2] Δ by up to twofold at specific chromosomal loci, but both mutants maintain high spore viability. We analyzed meiotic recombination events genome-wide in [mlh3][1] Δ, [pch2][2] Δ, and [mlh3][1] Δ [pch2][2] Δ mutants to test the effect of variation in crossover frequency on obligate crossovers. [mlh3][1] Δ showed ∼30% genome-wide reduction in crossovers (64 crossovers per meiosis) and loss of the obligate crossover, but nonexchange chromosomes were efficiently segregated. [pch2][2] Δ showed ∼50% genome-wide increase in crossover frequency (137 crossovers per meiosis), elevated noncrossovers as well as loss of chromosome size dependent double-strand break formation. Meiotic defects associated with [pch2][2]∆ did not cause significant increase in nonexchange chromosome frequency. Crossovers were restored to wild-type frequency in the double mutant [mlh3][1] Δ [pch2][2] Δ (100 crossovers per meiosis), but obligate crossovers were compromised. Genetic interference was reduced in [mlh3][1] Δ, [pch2][2] Δ, and [mlh3][1] Δ [pch2][2] Δ. Triple mutant analysis of [mlh3][1] Δ [pch2][2] Δ with other resolvase mutants showed that most of the crossovers in [mlh3][1] Δ [pch2][2] Δ are made through the [Mus81][3]-[Mms4][4] pathway. These results are consistent with a requirement for increased crossover frequencies in the absence of genetic interference for obligate crossovers. In conclusion, these data suggest crossover frequencies and the strength of genetic interference in an organism are mutually optimized to ensure obligate crossovers.

 [1]: http://www.yeastgenome.org/locus/S000006085/overview
 [2]: http://www.yeastgenome.org/locus/S000000390/overview
 [3]: http://www.yeastgenome.org/locus/S000002794/overview
 [4]: http://www.yeastgenome.org/locus/S000000302/overview" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
    <title>Modulating Crossover Frequency and Interference for Obligate Crossovers in Saccharomyces cerevisiae Meiosis | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"_lrr9W7dZrUip6dPsDfyi8_ud9OYOIka6AKypG35QyE","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"484259","apath":"\/ggg\/7\/5\/1511.atom","pisa":"ggg;7\/5\/1511","ac":{"\/ggg\/7\/5\/1511.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/5\/1511.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/5\/1511"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/5\/1511"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/5\/1511":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00223qJ2tmoyoW1Isws5fic4GmHUIjQ2uNEdRC6TNr3Zzviyzih5ZAPyxn+6AMb97HcvHifwRGF4PJDq1nzioE5vr3DAlwHESCixX7JW\\\\\\\/Gsxr3X1MrrOoDDgPIS9FLpwz7fVWn7gwS\\\\\\\/aqpV+Zs3CfatB9MGAMq7DyGRTJfz8zMlZlet5dODq4AEyEcPP7zORZjQfvZ6CZVoivvJ4NtBahxGwCZShW9wLFt2Jepd8i\\\\\\\/+oP1Omt00l8RoSrzkml0sEAflaL988mGmXdcwJhyhif2jo1VVcEyGeWjabmfiW7N6SqBluen14N6UVSJD1iq5wfyEabvGJfHrcsh\\\\\\\/tsb26utVE8xQgS44vK5fkV0AU1fPoSjTW0wfBZz9N11QZUZ1IObrP7p71uuCc53tbpBcFAqjRy6CEHovXdGYi1yEr+e9a4qICKTh\\\\\\\/ZUhYBsf8FD\\\\\\\/kZ5nwTWpIhoK\\\\\\\/oAZ64Pj2eSj5re+nyB4MFOhI8YPYQl+liANsW7dK3v2reGodIasPVmFODmj+Y9CzR9tvthYj+qfg2hhVJp4eAgyLGJzDxX+Q+6gCzJdDKN1Vdy7qk9SHt4lly2ezJHpRhhZZ7dTpTkTwufCyut+PrKhdKRp\\\\\\\/zF\\\\\\\/Wze1qReX0h0ofbOTg\\\\\\\/OrpxggO7jKS3CofthVVaK11jRw3b+lQ11s4DFbHCRG7EjraHxcwS7qsEO3SVVlVVpVbPUZMVbA8N\\\\\\\/kxFVVKfE6LmYQGTNDC25y6kF79ulIZlNXs2zTquBCPKBjpklFkcUsLXcsdkPZEq8iCuJgjCxCe9jRCBdqRX9U6sulPljUR81Yj+oR54327TwzXswJd2aQ+mgXu4lvy5ueRlPiM0\\\\\\\/F8qHUXR11yqW\\\\\\\/qSGnvQT9Vtkopsa1z+Y33ZqFMcZlXH3vRHKSB+Co1Sb2CFsSfxMA0hWCt3f7VdW+B6P4atqUOdCK7hzUCrLuvv6DZOKZ3PmDUSOcXmqlgH18I5Dzf5WBEXU176UlWUJW3sJsvchR+vcUY0Lm7CUYIxfCzBN4KQ3\\\\\\\/CjdohAoOTyXvPKVAXAR0IOdHvv\\\\\\\/t4bqbuCzovLH9hDvVlJiPMmJ3UF2mSN0v0SVXQEtu\\\\\\\/rcss5RtPs1dF5WGXHG\\\\\\\/Ez0gAkjq+veW\\\\\\\/iZ7Ci7n0wb0rBLwg2G2FFNXh656M3TQiCegaSxNyqtguDqzYx5B4RMmohRsj\\\\\\\/FOEtWhrjPtELROJi0Uo85xemzAVcRuLk359oJLecEs0OTvwowOkoGyqxYDwsZ606vVy1CZJyycM\\\\\\\/mJa4MKqLTjCffkZITm7I47\\\\\\\/\\\u0022,\\\u0022iv\\\u0022:\\\u0022P9fWoe1GlI5FyFFMAmyspg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228786aeac21926bfd49fb1919a6d20fc98541064dd51fdb90a297ed444a1a673a\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+a3TjigAqepI72hBDmyh6AAGpZGKqtnAZ2UOzsa+zdYv3\\\\\\\/N15WD16gGQ845SxRHKNkYIaQh3CPaQmFDoadtB\\\\\\\/zqM\\\\\\\/91A1JVUJ78kVlWNcq\\\\\\\/IjltL41hLju+dBpb0O+mTfBVsPFgMriyfUIU9H8AACw\\\\\\\/ajsVHFZ49BrpiJzkkSmntOKQ2nv4eKDHu3Hx6Lk6XsDqkei0BGFAm1qXEZqKoI5UWYuiyHOWyrwpa2mo0UHKynKxFJqWpxgpmQH3PkU6imnWK1rrtWHGStZlY7Gx+Dft44FH\\\\\\\/C1DtiWqASw7XXYCgngEZnQMG0x\\\\\\\/ylN9DJKR9iltKdggALwcxDIO7LO2wA7d8xzI62y03Z9PEY8BD6yEe1cbUj6MED+ADmcRlsVekml3ljPSxJrWNH97q5qPmvpZizphymOpvqFOKAC46D26w4ckii6im6kfT4zAJ+ojiQjJBWNJvDay17yYhwQo2iD3SJ+AvjKjXTiOrNmS\\\\\\\/14F0liUmFNHy4yB+4VpVOYIRGS96XIHMkXeJbFufPacMvt6sCy81MJL5NENLxLlqdxn2U+Tq14P48+Pm+pbZclTij\\\\\\\/3GJTJjep2ddr9TYrcQvg6JsCmWnBJpOoclB+N8qgKFmxnPLPWOJP5uUbmcrqfbGwkPqg7VqM6FVdMSwqfmydDfgokY184D\\\\\\\/8PqsbjB2glt2TUQ0XqDeEQVwUKdvEYOLs7nzoBpLSUS2MEUK9dTMXDBT0Im9twgi37Pg6tf72vp4KF8rGkSzmBOiHnntcNh+EZetBcPdii4\\\\\\\/ngZ8HaU2rb6iGcN8E8yIUf\\\\\\\/gtiHhcKE0PN\\\\\\\/NsSbwhN8XvZtzvZskUtKPB4Wgob3HcyL7lKowrN3oBSuzHjvE8Ia7bRvIe9\\\\\\\/ouxdJwZGKuS8ylDBe1Q5wGvmLBsUgnS8AxWB+wAOYeStxBSf08ZwnJLMZWGy6jxxyY54X1GMf4lww22xk4fSXetBuO8vAXo6yHqVgARi3FbQT\\\\\\\/BNOix2lywPhUjPoL5dynCNDhqMHXP3kOwfLwZwTPv2hFvS6K9g\\\\\\\/RbI9yLLwXRv7OFRPtf\\\\\\\/x8YgbyBxQgzli\\\\\\\/zqL7lgORHVC2pU9UTK3E71\\\\\\\/nM41WiUZi5W6V4gSLadoYSDcvghyKy4FvK2w7HnH3qGjTQMtxIl\\\\\\\/EP8YXInqr4iPPhboyTiCSLVtTP0CI\\\\\\\/hI2gkTsSfbQTPDGstd505YLHe8KTkXSSYAVW1o+xn3S7nPeIoHz38aLt47EE6TjdNUfPAJjDRsdRF9Bbat3hm8uoT6lBSm2ebYMlDIg\\\\\\\/67lQkME6p+zM97yxbvXLTsguQu2mJwN2OZPk+k\\\\\\\/lPEeMMyYIHWlejTjf8O\\\\\\\/JINAHcOMo26z\\\\\\\/JgI\\\\\\\/+Dblyc6FC7TXJg1LHRAybumdD0ePMKzM6uuSbnqZ\\\\\\\/P3ZHweghkx9aNEI85NMnkP8caR1BuXVkrbnHj3jC9eSHVLfSMk36KVLOnt41+wUsoj8DF01Us8plnUJjqkgleiDdn2AcSxV6iyFsrOhMBwIJ\\\\\\\/7vHUfhXFQV4H5ZNqTRO0Iqnjm1JlK05Rsw8tdSbrA\\\\\\\/T1KkJfwiyNL\\\\\\\/OsDsEhfZet+eG68SXm3fcbJO\\\\\\\/yZ2qOulZNNxQJkV+MPR3anZQRI1qTpHoFHcGqCxw4g1iaoyE+ZvZwPFLIc6Cj\\\\\\\/ambTLvEuh61bOopwGxzMaq0xgw1R1aiB8U\\\\\\\/s+eTTKvUSvjEYc76jYx\\\\\\\/xHgifz2BTQI+35otDJy5ZRUMDIE18UR7upzNWyQ+zjTpTWsZv8gr7HHd8qKmFKkAt2\\\\\\\/ad+BW+BJoYva14bf7X72NhITgp7nrVPha40esrad0DwdRCB5BmZnhlEh04\\\\\\\/HPPUIje1KC+tENKpdzFCv9GVoPicJxAJUvLBWRNmtIP3CGZqV2DFRSqx26lc6DQ+Ikt1ADDeYkExk5QLI7H92kRuWHLzdTUXoE3J7nyYjnPj2jeCmabxLMCRQoldg2J+ob\\\\\\\/bRH9uolm1V0NGAp8c4DEk5njtRw6GWz16W0Fuoyc3BPk\\\\\\\/i2Ii+tKv4E2A4DeOg\\\\\\\/RosQnpsAjlt3nYMdM1oTsFpOTAXzP8V1ulAvd5ehyd2z8zHcMkPyTJODXjbCVe0PIqtCP+3NhXlJtqj2kTWpqOMLlXf\\\\\\\/Xl05VKxL5khscuTIzfsGXeacz\\\\\\\/DLpzONjbtEwthxzlT4Mv5tX+HrNXNNHqsMpEsVBtrv70EPY7zKhi4KKyn+A\\\\\\\/l6n+3KA1cL0RgZfIkfSGGrH+ipO2D5Jv0iRNoqJ+Tu8\\\\\\\/IkvJoD5RtRu2HMXq37G6IS3\\\\\\\/lVqh1XiGm8b3sBa8dze6SxXJYaOy5EWLai3DuMjofXt2kR5DA+2nLx5O2BgM6\\\\\\\/LVwO4+fzuTGVDPe7EkmjIIt46Qrk4hTGl8gxGtQCjx0PD+WoDlru8yYZpqCdpyqn4mUjnSegBwNNu1gOE0vRMEdNSDxZxtRgpx\\\\\\\/f8Su16RWMW3VezZrwZ1VXiNCo0cYW0xiuk\\\\\\\/x0Df8tfxko6ZBL9xIeHtP7mBGv4zlAicsaX8fS0PVm9jZRAIUkmGAgA\\\\\\\/Naa45JpcAmP3+b0cWD\\\\\\\/6ggmI\\\\\\\/Dn6VQBtyXsTdhgOysVOMakgqFzsNuQgT1M0aCj1SJcL\\\\\\\/5A0t9JGr0nxyC\\\\\\\/duNPK3quXjbHTVLeyvBcD31OZLYnu4ULE6SqMYcbackNN8d1I+qgwiw1L5QzQT\\\\\\\/PYTTTfyiICat18apBmYNAwl2Ub0q+UEx3JeW93+NteIB65O24aKVkZv2vJp2IGxVQuo76LPuKjJTsziCNRGTuKDDT7+ERA0CfOZXJywweL3qLabiUPCLgumFrpcjYRIH1z30ntvXeidzZZR\\\\\\\/YZHPMRSyQZyCDz5FGunUbFjC40Y3BHEpWSXnZPnqUMJf84eIum4R5VxvyflIZDqGQQecgA\\\\\\\/5PFWX1fMfFGuWWJfrMruUCLm2RIgTBzj6H6+qOIgI\\\\\\\/NAs1AQ1rf0\\\\\\\/BEQ7hrYcQEN7p1ijw4qFY\\\\\\\/+DahlWnK1I4vsv9p\\\\\\\/eMcCjrf7KkkkV1GGgT9Prz2U93Zj4TVVF3IH1Bhemdh9e\\\\\\\/R8y9eQE1qYSXiuL25u2iU85ZBj04V4tGHE\\\\\\\/H+6Yh2PbrbCYedXfpKoZXQdghTNBEqNHdY\\\\\\\/elxyWV7q5FyKpNDd7+YvEay0D51KypJxtWs1Dq\\\\\\\/ReRABK1bjc12k1AOhBbOwZbR3I\\\\\\\/s5knNRaAh2PJ3h7xNRLBsYizq9nm4h+Mj4lWoN24guqkZ2VXLobFREmvYqveCOe\\\\\\\/6OvoMU1v40SkI9+qC7LjBB2XdwwBq\\\\\\\/C+UDatvmZGa2XWcKC8utJ1mH7SWMZaj14p8OOp3RxbofgFSK+70+\\\\\\\/pSkWb+2YQiiGXxmQp2WYBSkWZNiPS+i2YClSL3a84zpxZAzh8fiV8AmSw0dsmTSLGbsBMghWFOl5d2cXThcIWKdpDD\\\\\\\/gWVfdvFP0CT5NDiAMLAA9FPEoeJjUh\\\\\\\/XyGo8SLptlzo\\\\\\\/dAaJS5w\\\\\\\/HpdpaHOp5yuw3dzluOJdOD35WUfKEPJM9QhDaugiB6QfyvrBpyQmoviis6xCtrVgRIjYnOWlJ2oN1Kpf\\\\\\\/UslSG0Gxf1jtyOefL9j04dOdA34Gqu8pGcdXYYh9sInQVlmNwNugxMYd7jesuiKRMRGKKNRibZUJEjc13hRvR9zp4UGQ+SCn1hkVPxc9YCGhN5n22zY0u2Hl1s\\\\\\\/HuRHWgTp8+NLYI31GjPlIHIDYeraRPWFtiD8\\\\\\\/9y\\\\\\\/WnhsH6JS8mjAFx7yr4chNt7w+V+leqRJjBOQYs3m4AXE0qVS+Kl2W6fsrZdCAvoKs7w\\\\\\\/7iNTXSW+zIg28kNPzP5CWCIMLkzCX7YyxIfpJYeROfdgpcHk+FfI0d5hrMjBQ8G+A0bT7jY2r4cd3r5T7JMypPoOKPH05rqvSicY0L+1QqpRrTzuWfbA+CemhnR3sp16Wc4SQlOAid3BgMAB9RW4iqEVXYtjGNiTwR4z2af2ghpY\\\\\\\/72toK9h2nah7WoTe0Cu9cULsqrF7mEyEWWsEa\\\\\\\/GQc3loCexXi4ehRNX+UaJ1W9dYj\\\\\\\/LAQDn4R3ogMi+uxcCkIAjmMqK6Y2MFygXjZxAkJtojqXgm6C4aZ2ob2m54+VN53wOCcd5PCgboGDHw6ea9cv6r+l9f3ViReyLMNKd4eUU8srmJc0G521K6NHtWEpUsrtAnKebtc9yziKvqNMo44ZsNn8LuXHWw0BYy9KP\\\\\\\/u6+Mr0JCw9x16B30iH26XLSH6\\\\\\\/gz1Db46hstCDAvE\\\\\\\/Jnis8pKiUqt3ofYlV08JX7fgMRn6e8XmzofzRSdXtk9YS0smWsFIozuVRJ1rOYEM+2m558qbYcD\\\\\\\/ypACMhrDgix\\\\\\\/i5MYtdBJdKQm7srUm7fiDO0kxRo745rBeeJGbCBU6HQ+bkXkCf\\\\\\\/nGGLIxLiCWf5YS9dCnZjwuJS+MkgbA321pWYaoXZSW7pIt1v50TadAUc0yrD7j2y0qyDjMUan+RzaUe0Qb2Qs+jpKsL6cTJnzNyZzV3yuhUTB4NujfzgZm33n4Ldyn5LTbowGfo2aYVEKekU6oth9lOU7BMwxaeogqLdSoh+hiNlQx6ZMkzDrudzrPg9vvUfCi0IFF5QXxw+8pDRmN\\\\\\\/Kc38\\\\\\\/iKJCb7pPAJesCNoPHf4lIYtMiobmxLoIDNlYqDeC7ceMcgRFHaYbVQgQeowzGOiaoEJKRZGWIz1J3FphXUKUr05sPC++ia9y6FTWRPhagMmXS8n6VUmSipRg3M0pHeQSXVQDLGPd1WGSMnHZwh9IJ4dHI65Zh9RrGiaUAQi2AxA++1WtuzN6c6vsOoeg6CUr4hqtGMsJucVt\\\\\\\/gCAbPE7B8xwr+2p6YXh6eKcIbdMF3pDFYYOcP37INc8OFVAVqvwI+Y1YeU\\\\\\\/YH\\\\\\\/KHVvp5KQAZjyWamb2ua5ktWtDYysDuDxBFiqljZFxdv+VZj6sg84wyV5M8Ngco\\\\\\\/TetvFmH4jrv6jZt1i+q0SkkbQZ8wlAE6lXTAwcRLaRIghHYvsrYxoWiNyXbyp7TNKdR9NS0g+TVMwVQFCIJpV+XdSHeScRpdu0Jn02vcAo4hhlqMcBCQXLsbqMXnWrUHbBggZo6Csoaf43YMaPtk8DQaQ2FFFh8VINXB3FPZTIhqcnCsle1+TiKGcUes21hoSkZ6EMOKaJgFooIQmSG1xPQn17ZUuFox26wBzRWbNBl1GIbKJ3X33gStj2tGCQoCG0C1Fh\\\\\\\/vgkByTt73KXPZYe4Xom51cP2wxaaE0r7QGsXKfiUnkfX7hrx+N\\\\\\\/+s0S4BhIUk9vvLPnGfQZvoC1GrpeOI0r9aXNJTT4+DOqL5qoFRlTgJb1xy+oevz1uJLMWNu9KnkkRLiZ5Onb3BCAshEy9E3GyUQYBPrc1bpXELD2yn8My19zfOOJ+CiFCZDbSGv\\\\\\\/El02AHIoKjcTFPpRC9cpLRhUm1NDIPI56RpQtTcSO3r7buWV0\\\\\\\/Yq5Wu1zek\\\\\\\/j5lVDyBgCWeBS6kJVFWaTXWGhS5ZmUq\\\\\\\/lkTKNLdyYnHoQwZhbcgBtJMu9sb+y1SiPdZOBRnpZG3+rSpt8F7VCVMRSiDI3z0AHGimuVTpcFmosBGq\\\\\\\/7RjAjNPdrBqbytyeR7HrBPkCYHGn7ZLaz\\\\\\\/MWVfL5WsrYMB3lMYNzI8qMsh6pknAL2QRKO094b6Ck2q0Ji1sDhkavwPC7yES92E4m5vSSNYUCHdeCnpZe3We6EJAikYdjC+B6vMzhAyQ3RCSBbrVJ674UH\\\\\\\/U+NHgtnZxzPw+dF1zwhKKUR28QuS5Qhza+Ay3NcguJQvSYsnX+9ywDG16MrClDTh4R2HIP4SwsO67ie6Xq59xU4uY6kQ\\\\\\\/zBcXVPmrM57\\\\\\\/KP4gsNW5VDTUtdeuzvdUmOAWRTYOJ7uwh22JqfpUW9DT6Xic75L8O8DpJi1GP9Dg5iJI1Bh6nR645xbKYXKQ3ScaCDhG7cSvvbGu7vaN2eyH9CKsk8taJd3YnOcryr3OGsm5PjOGy5UC9rUkNAh5gfKMRLVZ06BabJR7w6s0V0WqAue4oFMsgDBphv15dj9fRmesQTi2CMCSqBz9ape4dg1ysVJ\\\\\\\/\\\\\\\/Fp\\\\\\\/IBK\\\\\\\/5Iv\\\\\\\/cBdFM9tSG5080MyWH+UIUPXCJKmSG\\\\\\\/ONtVrEhS9YAzKZDBvnXKXL0IxptWHB3i\\\\\\\/52L03SD23OSGDdnbJ7oMu9B2EeNTNb5NKQv4XbYbUn5NYE6l8E4hjiJSmxVAOwnxM0\\\\\\\/N2LV+w02P6zw\\\\\\\/C844rBBz9FIuN1KseefOK6bQhaez5OVdcDlFHi0XUERiK2HzJBqc2HCWQK\\\\\\\/bIVpByVosVC6qHZvnDvnSTG\\\\\\\/p0cuDL5j\\\\\\\/yogakyOGcoIPOtSCcgrakHAmuhxrAO2S5pDbU+915upWmEZPKhyErvhxHpfxmJ4mx2rMLfdYwwLPJQgEe3bKMtx83Y3tcytkCZwVx7mi+3KleQyFwgAq6E8K8arMi0zhpMQxmVJhRDhLF4WK2Dl2+B0P67oiaO3mttn5+8yX8JbnouNOmxC7naQUTwDt4VePUix84+r2lmuGerHkm24HemhuJx36GU+ef3adpsmav9p6lHP5WllK6DUCf2xu8EkncfEQf7lTOnxxRKqNGIPwhPdMKdvbKdRZ2ED3TJpOUtYf9UOmDbaUlbLhlncxUoqkB3j9gDlvZwGQGTHIBZIyNy1oO+sQD1hbyjGJehnr5LBh\\\\\\\/AOjInf5BswIdO6uNpQBOxC5xXTf09p88ThB4wPqIYINUnGdtu6HUYbFomjlkyEBsn92nPEUNr4DidzjmJYX0BWKuKMOi2Va6HTM09XuPptsj+7VwIw9cHq60OyvKy4iJ6PaZLGdVF1FY2R8QIZtGB+UrObqlNSwRTeazpUaQI5WWlEXGJXSl8dqryfguQ886rAgdfkjJM2HeF9brMy7lwtYRYhAIpSdqUvnvvBYzVqfDJWwQbfhkoJR8aHK8VGBBcXD8Z6vYfIsSdOAfAVkDxFIiIu\\\\\\\/JZBCBNgr5kMsK4idetuEeIOFr\\\\\\\/hrwv624SAgzJ0YNY2HLKOepqce3kTGMT0h8YkMk6TcReTtpIDLwtS7xox53iXyXgDG1\\\\\\\/DSwCWNqCczh\\\\\\\/sNiKAApf\\\\\\\/vW1K7q7DfdYYxt7OlufvI6XuRfspCtbToQwrxny0yGvFO8RIiQCK7m\\\\\\\/\\\\\\\/E\\\\\\\/UTpgp3N5PHDaAXf92zuHKMSZ3GkdMMCxi6s+87FQnQB4HpcyzO0I9dpm5+eWDAyR89lX0bKwq3xRbnZRP7p7+QEPHd2GUUGKBTMkU2WWtamBv9Eo+MTTR+Yq5LARSVVQIzZtU7350VAr\\\\\\\/jS\\\\\\\/W\\\\\\\/GvZI5NDZl3fLLOED2sPTP9nOxf2kF9yy2m9Xz5keaJn7TtubxkSe07HQUswfitDt3PPjR3luCH+dKsMQ3khSLBhp+isa4OH+f5MiDKSUhTcWihhW9R1hTnVOKCQ\\\\\\\/wdpZ89EH9\\\\\\\/1hfnvwMoVqEce5ujseiEo0NPwq9tr3xJ7NBKYQ4JQqIPVHPJQMme6RhywmBlBJ9neJz6tQw8Hl1iyOEBAfiQlFhiWOnB80A1UMtCKnOqL6KrF8kejt5GpF246w6ZM2uYSCIhAtmt3kh0Zox0hG9daIaDbNpUY6ne0Nv+ZblRCbq\\\\\\\/7Ar4FFHNlViyjcCQ8NNv8eWQc9YSVb9P6jxsLp0ak44bwpdTwO2SNBExuG20v3MezPCfJ5dzeT2KX3eCtFrra7mmx9ofZ3CBAI0qTLGKfyaDhBTnxx788+Y0Vw1ijux7ZKBkYvt8urXD1iHKk2pRLTZ8vd1S8zpOAWhCgfMnPn8prchOjwb+dU95fksUgahBW+PyMJ3hI6+sXDK2xYlU\\\\\\\/o2315MLdaID3Pxi2QLO8tG5IRJy+RI\\\\\\\/8gkK6hSK8OCLcewyq+oFzxMWJw\\\\\\\/XtiF80VEr6+AqfSyRyTOMZ2XAhuRi7g3GVcBwMb5M7ZnlE4BS\\\\\\\/BuSSAJxzJdX0oszQu9ecXkE+vTX5VuH3w0TIdmRT5zdc\\\\\\\/A+Z6BmX4wI9bM7XcHQyeYiuFMefjrSp07iN9AkpxzNiM811j2wBeLz4JgvIbBebN5ksqGkT1Bcy2XxNc1ebWOMbODYdm4Yz6jNCJgieyMBKLxbp+AAEJRFm+6gqduWS4NfDbKm1z+kBBMYu9UtnTcTKWuipSaBTqqUqUb1ex70D8Cb6M\\\\\\\/ytTWQd205pqm9+Jf9Ao4V3xjNcX4g6QCmnnU+wQel9iY\\\\\\\/J92ccp3gYReWkt\\\\\\\/LmNllUKUrli2n3sDkP0B0CM+naml6EH1ZCn0D4jMCwzBP7cS68my\\\\\\\/40cn4Lu5zBK0sc85qLDnyFWy9VO8+by5JDutJlyXrrjBM2\\\\\\\/8yTdsnx7lrgzEZRsI\\\\\\\/sfyGAM2NNOvDrXvwD8\\\\\\\/Ol+68MYS8yg38Cldm1qRTyWNPBGiIWAYa50U3ZrDrIbkzQLR3CCfAgc+bFm6nKlS6+nYHX0PWI61gkKuG7c+a024sy4ztoC6sSAAbIlChT7XZG9WqvQOfEELrMwYmmc2XQyogDMbDJXnGkRzcx6dqCj0L+xMNPGWuNJtqZ9e8hodKebTrO7MgrDepca\\\\\\\/0kVMzsBzil3mpGJoZbferTTqXPorZ1FmSFz4Bq69np\\\\\\\/j0mHntDzXICUIDMZ13\\\\\\\/LT9rOGBlaUVT6uAs5z1gmYlgYqZeFA9O9mTzmRHmyzLiXL4QaRWOGjMB2FiF0o\\\\\\\/39Id83cNM6+tGU8yL4XM8q7Vaqc4VnaQsh8nyBIqgiLjBE7gciOmyY3C7s1xWg\\\\\\\/iaORCIAo6oiaQoPW7JACBRusezOpLHdymyGRYnzdWiUSC7VA3Typkoqo6GxI21lj\\\\\\\/8My1Hf2GqZlW2pThPpTWBJTVfPeiOIM17CQHY1KXu0+oNjQQflWvm2ZQkCGDDAuDLmT+Y1m2NnDVhv3CgjNMnQH6P9UmA2omnZyiYZ+sj+3m6WfXk9L68soKy3\\\\\\\/qS8OOr4ApF+ea\\\\\\\/SdS+o21uHgM3c9hGsuJljd8T19hBY54x3FCoE6WkQZ20hkVsksaoL3xBDeJRLqgEEenMXW0VHjeh7iHUFczZ85guL7scaYH5QO9tkyfEbYdIMi82Em1koU8wpMe+6BrX5gkdliLCAGx8aygyRwqYPBmbTKVIr4DC5Ru1Jf+sUqAejkPRNongskbqMZYTT6acEtbnJAbW5pzpm+W8Dho5KpqpoFhRJ\\\\\\\/CjsMoexsSSylGxkz3a4p\\\\\\\/F7ltM8emF1Bx95YnLybBnIHYrOjKbkFcrA8zfR5t\\\\\\\/AAevR4mxEaIz5hFIPQZnMo2E0llPnWj0G7Rj56b77U2DZX6FoOSjvK2XlL+YYtIP1TUIkGnbOcMHAUI\\\\\\\/MsfmBJ5Zmh5t6Yd6CYKBdmd\\\\\\\/nvSU5Hyw7BwaFwKHIelHDJHaod8GJbGBS6WdpsugmgA5K2\\\\\\\/N3zSlFee8mHr4FQ5O6MLiPu7G2G9KI3JMAD+kAbJnsTz2n3iMtXeC6YLrp8auEMIfmyOykTho5z\\\\\\\/kX7YBIx5Cuzl8optxs\\\\\\\/NltsHIxYrA\\\\\\\/E51FTRYBOT\\\\\\\/eNO+\\\\\\\/yNiSQje9OT4SZb6iFSdkOA2Q0tES1EZLcwWvhMR98Tk5e3RkZEwDHi7T6s90uSvgUPRouN4laG4f\\\\\\\/QtgSSqGJT8SHxge6zmZ02CIjD6RZqXEGuCbzX8ME35tLrQm8R5uqGC7m62Hs17Mn2\\\\\\\/3NdbLiLRdlN++\\\\\\\/qJn7hiZuO76MR\\\\\\\/ijxlnn7n3GpXD4B7svOpbe4hF7\\\\\\\/SdM80G6g4jCmx9lP2J2yK8+6gkDz8aIumLUwdzpNFii2a\\\\\\\/L30OeBHME3dPY995w4KPZk6nP3Gq6Y9aZemYIqSgumYw9l4+nQTa6kOd34NKYro91zPBptZCUUOAtkkOkjkMbtxt\\\\\\\/XVe1+P81Pe4tVcEIWtD2k76Y6IrpAPXF6Lts33q4ICOMn5EGtQbbR0stH6+pCX6ElI\\\\\\\/humVkCKbKTvaMDtjEkBufe7pGE\\\\\\\/+WArupaxFuL8GP3aPvnIqzwvjgZ+IQ8XcrWJ3VuuGGWg8JOwkAMTp3+mIQ63Kr5hhCb4DH3I3Z39TqZoO6YqELCwilIG5lxe00TEUyIkx5QvHre7BT0Rl4gtAPAlLUAMWPUfnQbtnZfZHPJzt1nuhg\\\\\\\/2aiOcBrwxgXm7Elb5jZEQB2+r90V5+B+4Bq4lxiZAbL0xayhmMA6NSz+Q6hpWnxbtFSJxv173H+WPCNMoAyNDOAqJDcHaC22iD8b3wrOLfDSIZPp6Radih6OjDYIKH3D4i8IDA0BIYZraHwc4Y\\\\\\\/gKi0eLnsOhftCLdCaUOsZ2sy3aA9wPqk5DhHCuXPSVQkofeojwvqIz2Lr6hZ29CLFLQYC0trakxIwbzxT0WH\\\\\\\/g\\\\\\\/UxM4amfQ94uYinkOD+ZAF3dHnXSA765KeMeDs0Il3Ynj9jrbcoVtzB3sXmuiysImPkhNATQQs1EtEY9n6hX2lxYaqhfjARKvaorixKxO3apuaS6jnqAnFH5RzTMgdcFyyex+3y3aBKclOfpYOGfRNG\\\\\\\/g4Mn2jPfPJrZTpG86WoR4BVRQawF1jPF3wUd24uHSGu\\\\\\\/6hBDvn\\\\\\\/pOkWcYWeiqEOfe\\\\\\\/1gJLpR+ItGxwbjVtgD7sOsIbF3eKuyu+uFQPqyAo8V6i6dnmViX4rBO4uj2Z1WgmsIC+AEq6reCNpxeIvG7X9BSDwW9gFUhBUqBuBn3rotHMFmQJP9zmrYU8OVUcrwGOqeQE8OwsxkcfzqihoTUdTK+HnB4pzv3gosUMayyj3Y8DqT8i\\\\\\\/chVtI9nyr2iUXTnWh0zchi6TfqaTXaYpbacIUBBoPfD9dCAv8qEVfdbKJ6SqzvctXZFRp33StdhYFSMH3Z0uT9zbJGAw\\\\\\\/C8BlisOVy04Z6Ms2vP8NvIQNIu\\\\\\\/J9Q1h8pfi\\\\\\\/o+1P3tHvWuBEbWrKWYBEfryFd+JXQKIs2\\\\\\\/20oU7zhRKbOZVmVfUClFsD6lKCnVrB93PzZYOudVEXNMtFnKAHujgsKwr3rQBub1rQASWSO9PXVv40eqaDxkHVmNL3muffEsn+EEAYLC4cqzfCtQcxy7QHXK1CoEU0t+Vhlut8G8eFhuO5Uatny+kmK\\\\\\\/Qot7ac7fm4prxPUqXGjLkYDTyzihJdPSfily1ZF\\\\\\\/qM7rF\\\\\\\/thCdX7vCcADA0+8DmbLB1nl1kO7YtzH96F0LeKxtz4Ea1CrH+8hcb98GI4fXc5c3jM75GlAYCjBRMq3Gma1qPYORmGkDUasCo1dPA4qUz+7pwu3O1yVUKK1tVXsPBOrObiHglg4oBWHJsTIMZ5Evf\\\\\\\/jgJWgmntc+NbpbFvr1Fh39e8MzbUOnjX+KTim3ThY5V8pJOiCAc6NFxlHcP0OkrJvIrxehchf9CX5H43JshYo40lvnSUdaGz6qRfU6ZRwbPO0lY4mKpnriYXH\\\\\\\/zSx29sHfKBuGnVQ7CGYGJ5K59UMvmri0UtZUwiVPqgNCtu5Snj3nvhy+wosGrVhi0RIKWWeFMk2\\\\\\\/sJxwVI\\\\\\\/sWa4CfJfvCkxx2434f6PrLrg\\\\\\\/XOcr5ia1qfnMnUPBr0KK6KWDb8LYY8HnjSmaBRGABAwBZOfuMhLmPA+w14ZLo\\\\\\\/wJJ00oEC\\\\\\\/62ymBzP1\\\\\\\/5FwRDr+IDLaE1\\\\\\\/Kfcr+rRehUnMtTx5eD0iVZwWzilJ34gS+CHLTN4BDHXSkhr0+JJQwFsJ7moZfp4gCyMB69N\\\\\\\/6cEDiaeCYKlRuQxj7E4WvZ8gAHfsAKGPWRlqIFOu2XpvCTpIMq7SpABzrLiqwzqQhIHlgXcP3EABXzny6flytniAnTQFMMQfxCdt71KjXF1NCiLiMoFQEKzqjO9uGxehH+gh4VyKtCtgZEqrCYboldWW89ccGApHGd4FsV\\\\\\\/eXz3my\\\\\\\/MUyIQPtxz7Ex5thCL\\\\\\\/Em9gZGYGZuYRr4gmPlZBhxCRvEr5G7yPDyjukRFC\\\\\\\/IDeusNpBItL+HxnbzMhXSsfgx29\\\\\\\/4QibNVG66Ui8h\\\\\\\/T6rCvMYUkanNxN9Z+fUriCGzixfLHl9E21xVkya+f9TBTSAgAj44YW9ttHxVW6RmV312Ur6V4PxQrch5dH\\\\\\\/JRRbEOxepOPRzrsL+FWi51cExq+j\\\\\\\/us1hevHtD7lmqqfAPYmYy\\\\\\\/OX0\\\\\\\/Dq4GlIAVe4GWMcU+SNTAf9my7awf3SHXaV6PGpfpv4DxTpUtzszgiWlFP8tSvw\\\\\\\/+HT1DUx5BXmYit5GUMCx1Z3w2rB4num9IPYlIFirsCJmu0cgfbbtvVLdcl+dD2fY1WDsW+D1q+Ag1YpQK08BuiPbrP+vZPb6wiZEdNnifP6rUX2HWtwSgr5P\\\\\\\/4NDyNd9\\\\\\\/r7HkFE4g6uSIg79RbvtpRIScvY\\\\\\\/HT8q0eVRqxbdeb4C0S4SnNO4LhXLDr5n9+xekuYUx+ts8yRCd4PEHJ6MWFl8S0tlG3TgSXyrBBC8XPgiO1Bt\\\\\\\/8SiDqEWHzHO5B1HO16ZRjlS7Z++TUVjfCLArJIcEyGB65Tqf3IsKilbRIh1xiGa5k3dYj1yO+oORxItQk5sHFYqK9DyxRbqgjXm3dkpmyo08u3aSBQIMXGSjc6PKrn8lf4bR2aJz5eTxUGJRY59CBmN3Dv0YrltXOM7gebhcb2fpRfexNpVQQRBXaqOCSLxdRLch7MLv3r9YRjiMexXWftKEvVyxVlkG\\\\\\\/IhsH7W3zC0\\\\\\\/WLAuHmAMo43V6h3wctScWgjpC8M9W\\\\\\\/GZHCdVOy0Rk57LQmWC7jAwcRzlGLRq8L3oEuDdqCOiSE1u79gUb9BkQBAKcFieAFL3C\\\\\\\/OV0j+IC\\\\\\\/pD01XG9TIUhgriE77NOMZ9x1oN8xT3mBkGUQqdIMIBYldEstKzdEIWq2odJ4JV1GnUO411sVv64DRjzyYwMF31uy3rnTUsNIoLTKJiNA9EBnwlJQ6OyY2Aln\\\\\\\/OLqYTlxfiWZkD9Ji2E9UBwNg7WKMZfiOl\\\\\\\/WMQaxMbSyY7Wa5vRmufu97hfVxK4gfEuzAPVUoLAGBGiiVWJU83mXS8ygX0i7iK2uCrXsMMq7N6dZnEeLmQCzRxE\\\\\\\/0w25i1KxXYcmgHCLoGOIK2ZefThvYU2NmQAmBdWBHVqxn4cn4QJxMahgncbr2qXICh5+lFocujeNVcrsgV6ZRrroDeKXvAup1NQFihknSV20aJqJZ6v4T2FKeVrjAb5\\\\\\\/v0\\\\\\\/rlX\\\\\\\/h9YcufU95ISrcfbYCUn4kzg1aTSf1QWJHcWZpCWWwp6G3aWGlBLDMDDDXbVLTHO9\\\\\\\/zQl7dKm5y6lRA870HVEpOCvGhCANbGeOIUBnq8qc\\\\\\\/x\\\\\\\/a+57MAOgHXXw81VY\\\\\\\/zYCFMhSosoK8so1IOVZzLtTE3GNOMgS2MnQ8xXNvf+w\\\\\\\/gH6bm2\\\\\\\/0v+sBTQCpkl8zbARXxArx2EQfZgAKwIW0g1WO9NwJ92Ar08UEh5+x9PjFwrbVE2bLxIEIlpBiIArgCiFMIxd1YXTf0fNcPgWIvThVhg+jW0onH\\\\\\\/I\\\\\\\/haN1jbyKvC7XZUXGJoYPq\\\\\\\/pAHtv6rWlNe4DJhIepqNbjP5XF2eDYWJQyu88\\\\\\\/XCTfKSX\\\\\\\/guXYGR7EWpmjetT63khxgDGdMjHjbUUtGw\\\\\\\/8ZdbutgygZFaGoJItVlufKlv9oKFEfVoxgEKxNVioZ3jWu5YPjQQUxvkiFPi+RiD10bXqwaeolg+y2YknrIBTsk0d9WqA4spIthFdyA3ijp+eLF\\\\\\\/ZzQq7sXxhPmxhx10mx51hmHxfFH7AE7sHc1kFK6P7xt\\\\\\\/E8FzQYZtFk49VUOYpmLLw0nt48gL1S+FDF4DDjbIOa5XNjfU2uRoU9BFKB6KlQre65OxyESPQvFYhmexAr4dw4aKJwRKW\\\\\\\/qsePkqJE534cfaOopqXk0TIPcEc0+2hSTXk0NnEEImu2+2HeJpH5+py61SOuS3zlyszxwx+2Oal1QSS3ElYCC+nZafIsR5DK099CY4ElpyGF6cz3A7rL3\\\\\\\/TtdxkQ7o477tV6Jj5LyegUfnFIWWmCeH7Y9lr0fDSmtHhhnSzvu8gdXzWaGpSe5Jn3Wgn\\\\\\\/uX5zBOX+j9c3R13ILCacxhj+ahl3xI7NI19Kx8R0id9wcG8a6js4dDSseipzXvNJhKkCUE\\\\\\\/RSTtF4hgazY03g0McMEj\\\\\\\/l+lWY2yWCzllMeYLuCVGTwf\\\\\\\/E6ugaahwL+8ddAFQKz734viYaqOeIRymJik1vr7hqxAf65JFkobUqFUFghf214rtMtEkWGc4RUvMWgEeT45RSZ5r04+2mdG7JHJ4ABCL3MNMFQkKjJX5a6tPj8c2tGKnF6ixspb79c7B54+Dhy59A\\\\\\\/id5x\\\\\\\/6pSXeNHMBQaYq\\\\\\\/yLk6lH2Q6yw0h37eLWb+q4S4qjjayAQJyJ1KBXi+2k0agWbGKDx87bEbFqW\\\\\\\/gDJIALr+MfpVT6\\\\\\\/1+JpZjAxFXNNUtUyKCXS6KEye4X3sS48vMGMpIKmUcLz1M0VY+VsGrLJGtnMsw7BGj5EyCApnFiwJSjeU4fkFvVx75laItTxc9n7kVJVc7E5BGRjf7f9fnqGrgK3oA9iXZL\\\\\\\/pfN3qTqLXxeUoHsj7ZCm+WasCOdnFVrNXEAdZWMd0hz\\\\\\\/qlinxr+LrHiBLELYu1c2gURz1Ru9tOU7OXQzCicEcITuAqep7sUuAmBhlCnBOZIDl1a3d7JyrRuyjFmHPCxFXBlkLPGy\\\\\\\/vDoVjNLk2en+puVPZ6CEcJtU0m7DYRnHwxN3Pk\\\\\\\/FYaeJiRZ1WEKEY0kChRHoL1JDI+GklM56FTzE0EA2eojmkks2tqEfN\\\\\\\/n5lzI4tEYDySl3IPAxDI34U+9Gsu1OZsFONMp0usns6a82Lp1zpJm1Gz0JOYy1uEMjw2L4vh1q5vTQ5y1HVy9EaUhxBehZKJd85NKWytZsf5QwkOsqV6bxt0EtSQch1gc7+ng\\\\\\\/P0ivu242PPcmklg+SZtlzHI5o5aM6ft6alK6QQKYXwiOY+Q3D7pVcL58s2ErP6R4YSICxnD+01dtHg9ln03+ZmWJjsibdzKnIna8lNT0FVK4\\\\\\\/Yc4XFo6ZeZ5ZNnJ6SlsqqBV46pozLw90HHkJEfqn\\\\\\\/oXfqulVdlFFcfwUf2mxJBLXzNV8hBzezyUUmk1T8heyYfHMByfynnI1lW87aKZeGedWGxUaxxvFHKjgUZr00zn30oMq4jk11coUDmgPTImOIx\\\\\\\/Le8q9aD1znG8JLt+6a+NPHaWD3\\\\\\\/W8kBoibh3rJygJjJIVyVVTXchFr36lwNKls6ghJOe7M7dCo9o\\\\\\\/gjpjv6fdUpmIXrXezWcD5v6QvWLL53cT2qlaFNjs1SHyJphU172pGCBXL2CCVvXHY702wI1EXZrK2qnHu8pZMCAR8W0Iy0xowIGRXi1SZ80Hf\\\\\\\/YLjFmIOKjfTHQVOyTOvT4xqUUuxOcZqK5ZbBBGXjcG4I7QCZQzKnDxUVnpOTLHDvL5t1C7sdl8GpGRCtfTqWyFsI7GCAEl\\\\\\\/Dsj2Q6XQ+jqmu0YC\\\\\\\/t6\\\\\\\/XYrLcWwPMdGZwnqXPLgZjzwC2k5LnXFDd2c5skwZV57G2r9qBPcdpPq1DoQK5nPYGsFt\\\\\\\/vLeW8SK9Gcy83bHer9ZudfPd23UozRdmksv+OVyZetjpUxnF9X1NaYSqVD3jxqg0aQquM4HksukAOBhOMgQ030YG0S7FY7QyGC+A31I32FE2xKTP4H1ZCGeOFnO3arrKocvcmnPlryidF0QM6Q+ddIuNARvpoXPAjf\\\\\\\/5RdYX+YmmGiWD18n3qlbaQeaUAlzEdEFQ7CUyYuJDeCEnzqJbzZbcs2mmKLboelJKJUSqpuUUohA5s6n5dTavGzoGxInca\\\\\\\/Wr4ApEOE4ZGXbxMj\\\\\\\/DVIGL33B\\\\\\\/Dsdcnko9UHMFfGtEpKuClEGlDAkf9Sr38oFTFQXZ+P4pxWo8tbh2KJ\\\\\\\/yUUFNsWqOP6rfalOReXE8EQ0e\\\\\\\/8Eep9jQJx05VKPJs18E8AWmHEm8Lg2o4LI5OR7P81wo2D7XyurKLIB4ovWu2\\\\\\\/xN9pvDRbSoeqfMciMn1ilpKlCRUr74AMO3hnz8rTB3ELHWN4bWAjvr4l4pRJiELenEJaCaMkdRy\\\\\\\/aSoDgYVu+t14pni+nRg0tTG8nilzJoMQ28+FhaGnszPM1Gso8xdyQKnHHQEFkoNEXoFHDdLk6ClLpebJ6BytXQ72CXKCBRGooVxTLRlqU8IamURpCxGEr0hbXik+endw+wiWpAInEOOYT4DuxLPsF53l27EglhhemfaPqyLGWnxuEswvyH4shgCVWFw74YZ4pW3rD7S8bRX5u7xzMEqNvTsKgk\\\\\\\/Uv0AFxbnBVcryv3d3IQsnBbS+CsE6EEUuZKGr\\\\\\\/4oEUy9qZQCEVSGF\\\\\\\/lSeFXPXW2LW6BxD4s5B4TKexP98wT3X0TBDvSSHABXkxDIqdzwUot5epdUMKTWvEOSFNkVr9tlzjGhDzjYkH1MGvMmFJ1Xsh\\\\\\\/VCuzNOMeVfTdmL4rh053X8zvJXsx1o6ozord\\\\\\\/DnnQM3MCEVERcB8Burc0qBfK9hyUSkTiz0E96Ezh80MbSVfU7DB\\\\\\\/MLKI\\\\\\\/Vxm8BB1rYi1GHBZKDerlja+ILcyRl\\\\\\\/+B2cv7Fwh3lg9HWMtLdAt4W6F+PtFKKrl+54I0N1doYp0TTUPa9Ctz8RPqC8N1sk+NAnENYfDSVCMFulrMpK67UsEAJy0LJE4lElnQW+514vt61guSGnjCZ1lk74iKmrdo9nEEZ1FtswPFsWchmY9GGFxCuHd+trzz88WuZLPKkoYHWU+8YXM277w6EZwGlOk6fSWtbXmBn1oxNqdDaliO2h5iB3xWMCP4Kk7v3Gk5HFLCL7YoCZ0yCMhYqy0UuZS\\\\\\\/zfNg+We9v6rN90jTlEepzRYibQgtjMtF9K55I98M4DyBH0q4CXsb05Qa0ZkmpbOPTKc\\\\\\\/38I66JTnIWQvxURf4W+kIYLYxD97T7aRBrQq2BLKIbMhO58C\\\\\\\/4BujbDxAvWArPzsBjxTVTC3JAUsT8w8up9WkpDa0hogUQQsYiWydirdeps6j9puWqurEta\\\\\\\/24XdxTVzEz9QsT\\\\\\\/y4intSQz7Bzq4ufoX3LvyVX0xYya+iPCyKqgBKpDqAzxkfeUw06oU3M9pclq41Nvnn7cN6f+7rdIq17cYXgk4gOSo368Q2qhua9rJNX4t4k5jPmWXI4mCvNQzYuZID3A49mz7uf23Wtzs8yZwTiZ2vDkD8aIuyvOYnJ\\\\\\\/kYHD6EzstojBeRYIH5V0T92YPIuAt3PM4A5aSS0Cz9yeGmKRjIL+qr6QuG1bTxIGrNz+hzo9LEzEdNCdtZlcfPyop5uewRY3jFJuoyM17uUsUhCf61UekM2sn9nJ7hWnw2fS6FXL5PP6WEAGza0E\\\\\\\/CRKg6Y1CvQECk7Q7rEbYedNqIl\\\\\\\/S\\\\\\\/he37txSJT1Ikj1eHbGUZXeiOFIHcVhupaeyq5MC8WGad0iW2vfElBnWrMwpAtpkHYmNz\\\\\\\/snxDTj0RU2YKMsGbSuBiV7\\\\\\\/24C2YxyKJKz9Gl1kEBXdB4hKdTqkpcd1ahaxnMQeo6\\\\\\\/Ya9S0+g5JZAHU2UH9E7ByK0evu9oofQaikLz1UBIF0fN9OcXkrqcr0RIg+NC4RieUmY96erG4Hg7brSJEe1yE0P5JhReoiSvjIKP+ZRQeIbZ0BKSIz\\\\\\\/HQqw6EhqgKmLIRVeTLLt\\\\\\\/7B589p+sphuR0\\\\\\\/RLxpZo1icG0bxUgFxLVzffsFLqeY3yY7PTOOx3ONX13Ji4CgyH5vCQ9jEHAwWTCmP8SAgzyWnq++FPZhahCSr2RaYHY9loiR5kgyQDEBqLh4XGq637MuSEt3+PkflS3Om+I\\\\\\\/qUveTfcmaPjkkn3FqJ0j3cMsU+U34i9ZYoXVo2Cw9bnpb9lAAmAfm1oQzjjFGw22CTiMCxGPQmVFY3ZCmj\\\\\\\/iYgNWp7qf8F\\\\\\\/2r8gqtyDB503HtS3qkwvTuXMjXX7ZpDPsZw62y2DYw9Aa4eaG9SSbJvF+xKS073uPwPSkyHJPjFyN4QL86SStPFO1IUHoq8q0QgDlmhoWhM2FoHqETkpqNu7ys8VbPUMwI7OX9t7QDcdzN3Rmvw4TI67\\\\\\\/oFGSJ5YNxSwXsHB9nACiHAh5NtFGMAFJPkKy2xpDGmlVT2McfxIweY7DidsZMD\\\\\\\/SU9mQO5ymCcSCxniQ5EysYw9oHu4iQESLcfqFtDG+EsMuJzvHYMLnv5CPfK71dy1L3E8klR+kKknR\\\\\\\/pddBXR+u+2zw1Jdi9iTOMJCCPnD0AMm+XJg44XIoAoaLfZiFfjGYHWCiBG4SLy9MrVEUvZv0QdqFlnCXYVr2cBX+0diekHd\\\\\\\/KtLvsUYb5PCkK1gqGP\\\\\\\/GgMYRmJFxv\\\\\\\/2dP7rvaBSE4CcyOfctV8VEEtYNqje6exJMAGBQp95uDxAjtcKAS4sbuBvdelj30jdh78Nd1Zaamjdci1ALfGn1KSY0pehhG0DC6Omioy3\\\\\\\/WTWCupM+tEgrX++uJQvKlDpqWmsAPn5cNX2HvTMfzWm1P25HC91Lok7TDEcVNnyDEjn1T+7c5RdyzFhs2CeTD032AqXpwxKNynzQfE4nOqnY\\\\\\\/cPd6TMXoe9Nx5E0U0v\\\\\\\/nqb0Rj0HYfvZOZN+Z45kIB1+Jkw0+AiWLsa9YrsHbv\\\\\\\/yJg4E\\\\\\\/t2Cq+tIbqMzSeq5kjEAx5Q5EBMi5P57DC8JdE5dFpMll05XPS3+dtreecPDuk3Oy9\\\\\\\/F0OQDdAOwZ6OHaSoBU9ez8O6f239WP\\\\\\\/Oi++pgL4Gd1AjW3Gzwsq4F26Y5ptVSsRsVl24QYWs7lsxeGCU8aKzUOsxlVSSsYjH9Ojz2ko3OLDPagI6PhPq5p8+UDKszb79okEHv+AY8lWF8MhoNjVYPYM2kM9VE2fF4VioVhFgyTyv6sJ47GIDk8YMKepAZTT0kpb562dop0YnAAlpD1VqK6z4dXESlc1m\\\\\\\/lCscykdjGPLbMJGpOIquRZLVNODU0PT4fy8uoHoio+fs5aDR9JRAeIU5UUk+wuLmwwRJQoS8bOTqbDIiTPhSdfJrJT4RHSQaIvCMe\\\\\\\/hsQ7Hxvp4QUHPcPq2PbZbij7yjD0qFVgIBMAQctzNEHfGCrSDsjq14AZwG4bGkq2JFcvEjRCZGXk\\\\\\\/C3wcLiBcDmKjJ19tjQSw68RFU0n6tf1GaG8Ar\\\\\\\/zsRgO4s9sDkyfGSmVj8PTi6ZQqpOjdL8ztQ2ckmhRiy5tt3n7AFZLczDM7TLsWqIbuyD7\\\\\\\/EhtXPEcozks5OdQbSEaX85VHtxtg1t2\\\\\\\/4nx50Iqiud5mwWt9Ebm44dPveGooSOjnL0QG6nXqIBETaIwfDzY2wcktb6OrUR3S8QO81xbeTBWh+TdOvA2\\\\\\\/nQROnN9wbko6WuBlf0uurO4awolB6MPysTcfqtuju5FpIgvFuWKJCDe\\\\\\\/kjGzDsrx9pGYaAgiVWEqScQsQ0bT5Oo30x+GhZ7APa\\\\\\\/+xMJI42JOhcE70yvO8fCBG+qVSO65HV6lRtDcCTVFdl0Q9qC0AG0O13rDIvVQTraJI68mtTblPKvCQR2ZL5SmmiP6fTETCCmIA3vYWcY2EfGFUSBMFmuYczW5TTobuHlcu6GnQK22xS1kTOUb+TJkP8dAp1eYyVB4o4wXXsjp0mmFRc11tMudLScXyMO1Ucde41TWWUa4+Hyzl1hIbRLM4kZCVZHmHwKdh8U0AkEXQBM97T\\\\\\\/NbAgX28yGruGkiQJiUw+FkRfTn+HAqIqpoKI0Xf2EV+SlzINitiufVjoRTuCxLb6b8O3g3YRWncECBLwv09OOizSpxJTsbkKzwaOllmWueTRfBpwbbkCTdyGxJJ+fir74oQ7F7wA5v\\\\\\\/iTUdhkdQaEVzN\\\\\\\/DNDNgoeYXCl1q1aXXsVFR3HsSBL2C4G+Shl3yk\\\\\\\/33AuYN6RakYRXda9\\\\\\\/6i6m40\\\\\\\/x12+37dekDxw5t86\\\\\\\/jKGQMAzzUKE+Ep0r5xiiMIZ4Slha5YxJzeFVZ6iZ3OSuVcboFEOzfV9pyPpHdxB3tRO2hPdKEmMxBpbfU8HuImyrds9sKPJAqAA0YKKm1CrJED7pvQ2\\\\\\\/Hbw9jERNOWr257bHl8aTb1QNk8p2OsxeVwHkFcO8TSzV\\\\\\\/jl+Q2MgP4URydOuXn+KhlSCClzmm6smtgiAxNgK2Pn83LVHX0RywyrxEtczEafIzKL56P0q95RdCoNSDXQUVaITu\\\\\\\/veHI6W1aHaOPeoa97dgsyS3c+t3W6yUPwHK+o4D5Se2NNx2dhmS8Jxkym5\\\\\\\/etJhJ6hRhxfQlrm59lsgvVWwh4FEOyBV\\\\\\\/m\\\\\\\/YbsAVP4vxTdjRCTWeyf0Qu1jSxHU6LQSB6cHmkj3cI7P+yClsJ2L4zHEQ1BpzTDwHRx4de4+c3KS9F8kv6yb4J\\\\\\\/D3WdXiOl2u8In1iCrBGPZjCxXD9RsS\\\\\\\/XSeflWlwj5PHgoT+8dHwpw\\\\\\\/Vjogzufhq+106XkSWZTWiRJCOQPxVJ\\\\\\\/mLJIKeWgB2ppMUcKra44SCWYkX6MpO2P8OLMZqtZ7VAxKmoJBLKVz5zdxdrpBmar+BjeP6TTnT+qXiwsezdT\\\\\\\/VcWZX1y6F8hHtlAA166MYlVbFs\\\\\\\/QOcqIgeNsMRHfMEji\\\\\\\/sQqvVY9uS6iC\\\\\\\/L3mYvcE+BY4YcqeoGoEqvencKUQq82lTNKEzC2aAFxGAran7kbIEf0CV2D5NViL7lmViXAFJbsT8hYGYboWw+ekRc+CBnzSVBvrHSXpnztxNr\\\\\\\/01pPjNtODjC82JQkLNBY93ko8YjOhfDusdHvp51wBiESDFIhoFdlAT8PSCrJviDS2sxnHhg01CmBp\\\\\\\/IGG4R4EOeDLJeyRxexaG8v5KkOondIrEuYqvXDLvcubjycAh3\\\\\\\/VySQT78RxawXdCSV8WFvG47MBA+W1SpYAbnLcqY85XZFjkumzglTIzbExsxm5k8D1WyHgERgH1JQtgjIZAuRMcgEnrpH7vdfT6fx28q96B+HvMyUF78fmp7gq3or1rVzyBZP5XAMbcrBSq024pRJ7dVewdHc2lF9n\\\\\\\/3367dJfHUjtJxkWLkGQ+20yUKpmrFhBglFl00BmG1Gf30lwWe8Uq5ATmZOS2F4ockDzNCMZ4205TJWxrvKLqnAozUQWHJWov3RaukAjxhhoJCtE11Stci+Aqx+g0VaYzj4UVULi+\\\\\\\/b6FqoN0X4wtu\\\\\\\/tu\\\\\\\/GXlh+qvxuJuLDDnAom9WSTWLqGD5rjOwdr4FUD3qxLcT7cU3GCyMir952fKoeSluYqnZURNB0cXiuqta7dDTPYp1PgohOmg0mFZMl2b4jickngK2yf29tEoLv5R\\\\\\\/B+YLAhC6uIaERAHi\\\\\\\/8Tz\\\\\\\/oYjCj1ZiRplf44UhcxzdrY7pFo\\\\\\\/HhGtfM5xbTxOdbrrrjZv85mga5eoxxi7mxwy2NtCgSpzkh\\\\\\\/cSipFZKBGjkBHacOygaAIJ3Km0vk6XTFLIq3GC\\\\\\\/O\\\\\\\/1wDhhQJCl6n8+T5mCAYhSecAGty10gMOyQp2SPDey3fId3ToaTToxQ3e7FkCQ94c5E5h41L7G+2U2yO6h\\\\\\\/k\\\\\\\/9\\\\\\\/INX7yYLrkoOY04jOBZChSCTAbMp26NM3irexCymzT84244IWXfYvNnC3SE1cM4ukdp0qzuZiPJmvA6anFPzCT+0tutT4iEJ4JAUtOExrLpo4zxaMwdrvmdnwY26Ab65jf6zPgXTz0cWAmsf2POcyqU0sIkRxylrF05Z\\\\\\\/+nZXmiKzLMIoLcjH0St\\\\\\\/DBouzyR4v2hFWe0La8ztYMGi5hEQGtDbdcRG0h\\\\\\\/gzEVjV01zsEnEvxhvnjdnWuI\\\\\\\/+PKVTwJYvCF8N91UXLqw3WdDS\\\\\\\/ascZxFdag4IWN8VRCAFAZLGJcLHYPQia2wE5SrwmW42QpU8qiyh97VXOqDEjBe0mHQsZEUuVdGncUQ\\\\\\\/GpMIHNKYUj+XKsEm9LW6v4SaCXy7ZlJmFe2aCBff2Jbn1MbTA8gHVRNArgRD2R3sih\\\\\\\/pLwZi6rUyMY2bVfOBWRkwhni\\\\\\\/btnsSZUKGBidueDHobIApkGXpCIu3DzLTC6wzOtnMZ7arxpAO5qH\\\\\\\/CAppBC6B9Uhya8XPZavKeI\\\\\\\/q3sa9xS7m5vrOGk\\\\\\\/qRxQuMHQ2thN9qzsCRZMsx5pMbsKz4wdbRNg9XpKReX2AZQx9n8f7I9OV+lcgY6snW3wZtOnfmBXIfTJCx06F09IY4huZml4YcgFb7gKKOX67fqccm9UsZ4rOIe2IUJlUb8v4nvGMMNp0AOVYSZBH2mQR7a1glTQHev24mL24iPiYeES2TkhGQjrMnHGDNyW0GCJI0HQSjNIQOrsVZlj7qCHYJqUpcX05VOV\\\\\\\/2noPC\\\\\\\/3KtNI+Z9PNCxMf9xtTcgBgrmrk1JjrQcw\\\\\\\/2oNbcGuQWHcZZgh29PW\\\\\\\/CyWFUBFjSCk8PvIuTWYblDwMRSIJtp5iGigOEfBuSOM5s3rzAtcfl3qXJgfArII6WifncBWI\\\\\\\/t61yRI1PQFSBV2MdeDCeDk6ASrHL\\\\\\\/IyP40xTVXqMEmcNN7pOwYxEgWzgScS282N+XF0Rm6zK9yBOZ68xWzffiz5wdbimU30jLEwwGUrZVMFM36MnL09tbNdyAc1hpZWihVCVKdY1GCXhWj10oVYb0YffwiQn32ogKSN+H2QY0KEWjl6fR1rP7LU8VEueMpGFVm1JJpMoOHBKhtlOjvrKSNGvhS7SpCRIsS2mK5HYUjkMNeRvzFVHJ57hqZxxkjnvsIWmUhm9YZKN8BUClpY4O\\\\\\\/k7GiZjxL9aiwKdn15qlzKXcmtwbpMk35Hxp7vpfibTveR4P+uDLX7S0vsQqZrnLZna3pjTShxbjJdyRSiHUD+9vihqE2kwzWxEF2f1VXcotvYBKE0Pxw22DeoLRHS0Lfzn9XsgmkhKXP1Cs3dSeOtnBo8\\\\\\\/RilrEz7pUORV3zbZAQFbIgmF6hLYFnnM4bHYSU1WaNI4qQwT6dAZEP1I0SkmpM0c9Dp1Kcrd5U9uFSlDMGOPMQJ5pmZuprCZDafKN\\\\\\\/6\\\\\\\/wm0xDcrDY2vonKONqHrlfNfwuzBRUmKrdfZCOdhFAAXM8LyXWwlw\\\\\\\/2ZQbsKbI0MJlGWIfRCOb6m0IWyMVZqHZpLeUnS0qk4YQ8B9gDUmBjUbSP8mSKcZfjXPsn8m98n+oIPxOtY8i4Q263jCp2EgKq5jrXkTNAEAmTzX2SpfZhjbs\\\\\\\/6xpQdEy1Qo1jiM8b8dn6Ye3ngxn2aPKHT1ffMTNwLaTSjqxwGZX9rpFtz\\\\\\\/lQMv\\\\\\\/E6+9\\\\\\\/P+xfJ4k4rBp8E1LGJQoHrgRpedm9S+ZawerlptAQTl3nm\\\\\\\/vYFN2wvtVhqV3jBb4eKrvOvWSwTnfIriJnbqPf8u7POcWMyd24bUN1HWWS2Wnhas0pruBitX1\\\\\\\/FnHynBFNuJqGJ95eH2EEGkmTMXguLVVo8Hofg06\\\\\\\/HQUJ3YepfLoXohIUcCdogp5Y2qcqPIsMuwFvcJNgWG3guRJb628RzKojeWpmyRtF5wnSX1oHiVGZ6U4+80KAaZxLHLsTlNjmOKImHFoJMP65+lKPuPEietkRAn4KtUK1lxR3RQfNAqmMDy3ySar3eK6AEINFDIhatrPOVeH8HH5OEi74jORQ7x2+b5l9b+KOmyxy4l+tNAlITvWy9ED\\\\\\\/zCOF+N9bhIY7+BezatUMiL2CzS\\\\\\\/IFPU\\\\\\\/oOd6cgm\\\\\\\/uETSR3m79nMCOLHYp+I9HW7CWjL2tF1JY\\\\\\\/4kLOVTSrCpBVW7ndgSotJopGf\\\\\\\/\\\\\\\/QtBJU5M+mJWAf2eGaQxAF1ESjNY7mYz3b0kqftx+3S5l+Z28x5KDTxCRXnMJqpx8ypf656U67ZofKbTdlKPhusBu5ZdxTCIxlWvi36w\\\\\\\/\\\\\\\/Wy3xP61+bPUk2yxgBvpWuqJOoiht2s3DD21OCNRD5C+ql87VJqXVwyaz\\\\\\\/ckbFh+9gwPKkWp9gsJrYgvsR8OrdzBSYJMi3kHNNoA12B3eoFzVP40+BPuOEzjv0J3TU5UQUQcqZ1VR8p8kb0V06b5ZZfwvE0IpGsLpXNOPjaG22YnGpgXWROnzf7H7sCwZGP3nPur5PmMr5\\\\\\\/2oHZ08GCFEDstbh86uokC+QfeqHRWSibG1YMwD7zatXHYVwvCUKNaWXdsd7fhP8e3C2nqD4MdMfwLHbPYtrGj4Bnyl6jEiRwcglXkZHJId3PdN59cT22Rmn9l5bffJSC1vN3A\\\\\\\/YDsB\\\\\\\/FvqCVWIXI9FBTj0oX\\\\\\\/w1PKmUIaEuFDdDqhKYmTPDGj\\\\\\\/1xnRstpfLDVDThhd+d3z1zXsXmR79C5EVcAOR4DOlLl1+gmq61VgMz12DiOe7CVCCKqhDttw4y0Xh2yo9zDso7JIESyR2ciXjBO7qOCpvWy\\\\\\\/\\\\\\\/NgDpyjTJFNtvVE0x+\\\\\\\/Laek3xvLk2XmoEd3eVkf6yrMZmITMSdICmKyLsbL8IjpHrKF68ntLNe4k5RexTuWtXjDejfwX9ZsXj27vHvBoD9icXirtQCaoc0uvnLGZyc200l3al0byzFQ6w9BE6L65i1uVJ2R8aiHqz5DN9qa5gBeqJlAuDZ2DKRkGMtTbPN2qjwKGLcS8Y+A7FcqO23F8fk\\\\\\\/nQsDNxws5OGHdKzCoX0OEtQDzjIjghUHTh8fT2dPP\\\\\\\/Wa\\\\\\\/NCfBcpkfBEjdayLmbhzvgxFWYCErRad8UW3Fkg1cFd9rc4jcMck+dpB5IungbAD4GGD83oC3LFmZnBW\\\\\\\/XoJWC6P\\\\\\\/sS6DlzEUkNIhdDfiMefRTzLFfFGE7HzO4BLA8L0xQL1UccyBlUmEEVBrQNwEeOPd\\\\\\\/GKwS2TSk\\\\\\\/P5n3h70kF3iB59OYay\\\\\\\/hHeb8PKIeXxUc3cPa7gYAncQQPaUAwRh3Lt0AyKXmCg74f3ljZhwEUg+sO4ZmcH6qrOU6Y2G3XNlYE2IDP8ScoIcB+\\\\\\\/keW9kyv\\\\\\\/6Y+IDpro5m\\\\\\\/oc12PxPOgCAVyj+lJYV9An7teDgcVcVcQw5cQaIJYZbxiXEBF\\\\\\\/3mICoEyCS4\\\\\\\/7ZgS+qNyZlBLxBuG3Lxffw6512DgHX7GXB9zfqB6rAIxsmYE4f94RSgmcz4+Jf\\\\\\\/G3Hh8vTprlzNDa3zUNnToBFhK3O+DGuAPHAAI50qBt5SmK5e8ypk0Go8w8P5gnpSOvT9ksjnzRhiXD4Z2QapXRqjkkP6zzFsj3MGkk52pwrH8qRhac8bmUMpP28O99AwTfE2E+A5ZvBdS6K6CJqIWzq9EblpNjTg+bAJiEpb3Hl+5fdDPhpo+lnoH49er4XktRgXjonbQkprbR\\\\\\\/3SohG+fpyIQUO+VRdxacKfIDdObtDvC6b8NLPgDElP26HVIaytlW3RyUEGeLaaUz34S6x1jtvr0BTrePm95Rh3xWSKPlsheiLOpwh+21vvGziw8iFqwkld0NytiloyNNfBCuFh8W2rK4iGSl8aAnBRMJKfLdCPmIMH5GJ4xy45xwIJLmQyzx\\\\\\\/a1wSdWlrFgwMUrUvLzkJ\\\\\\\/RQ2j4KbxYhOHbQj0PldX5KsDTP5ggXAO2yH3viNAzMCK6HuugcnI5\\\\\\\/WxwIrjNrEmh9xxVX6S2ZlCKVKQjl45XGHbQD55YFjcPa9gLzyvkV5fFp6YA1ExyyoeTuFUQA7tOlErLoZ5dLZjB\\\\\\\/DWUU4nH4TGu5kGP\\\\\\\/MkCK+C4yxXiAaQcEA+RyLkqmwkMU9xFzzI7MfD7s+kTxXiQboHgRMXM9BPohmtdiWWE4ZUsGQjYqs7KC9ovIGo0e5yTL2Q0f6Awr5rg53bDJcQHnP+hfL9TfX+40gyHLX\\\\\\\/vRD2IO+xCcqo4aTlVvQSZDjCZP+qaUXgaeNjgbWda6e6mVaJNwd\\\\\\\/yPGIgNnuvdU7Y5pq5NFMv+XPIQxeUZEN95xBgIbJ1ifQ3uD6CvmgUBmTw1FFwsQXCtpGZnjed+xQO8m3QYmm1KoeAPVJujfDgTyJOiiVTu+3nJ9TLSnTZ9R69Hqb9tEHiS95L2jEaLuHRA\\\\\\\/riGg84U5Oz1oZruF8PUaMb68N12JXs\\\\\\\/lKwThg8g9ALJvQ7eZKhTwJybGuWpbynojMRLwXJ35DcaoiR+1OBthABNIrkmTHxbA0Nq62Fa5uaT8bOJbrvPGRv+Y0\\\\\\\/jGJm4NH7fiMUOcUFLQAs0aL\\\\\\\/u+K2Lqbt0J35Dy2xhcZdQhfe7xH+Voy4OK4QsbarRfRz0sPF+3ojHz6b\\\\\\\/TqGzIif+QTQJI1cx68AdEvQfkaYeSk6YqTaxGxigVsmblpjD1j3i2QuB\\\\\\\/yQ6rCGC4x4JKfbdYKxBqzkBGSJWiavEi3s8SkA19E0kdxeur+\\\\\\\/hH0eNcbTLNKlcDAXLUSI3ddmiRXquFv9wrH++Naw+drH6m16O1JB2dSIWD1vqVESv6p+UfVh8FeMXpEkCQvypmkj7fS2T1Jp\\\\\\\/Y8eCrKFSG4Sf10BMz6BEwNKWfdqcLDsQIF48lGawdaGSluCjJECYcaqtfIerrA7jqGQJHZDg2kZfCJgITViAwl0VLsamFeLC87F\\\\\\\/M4jK9R961y8mhvXq4m2KKX1Ge\\\\\\\/JcTD9jnOQpv0iQaqRm8VVRTTLaDfOE7civIldnVJEw7l4skNjgYYtLboRVwyrSnqWRsuxxk0Ma1bPP22Plw\\\\\\\/iZ2BTyCHimVL9EErpe6cV9kTwd+kySvA9oSW4FSYJj5J1mm9Vayd05WEtxuggjdcptKr\\\\\\\/U+Nbd+jWbOTNhRqIM5a8LuE8Uk087uDu1gEZo4raKYreaczAy337e8t3cQEJWhaJVWbVFi6lKwsOls\\\\\\\/aK2nZIu\\\\\\\/eehwCL16UyUHb9Ujtk+OvK2wpTpPVZzjN4MB6AMLZw7HjDwTHbZ5xbB1glDFbXI4Md\\\\\\\/0OQPHy9PoUo9yjA3APCI457xcowUeZ4sPalehXBydJdqbZKE6CDbFp9E06sMziRNorLAJrE+cyDpScYUdUmyec9zt+Lm\\\\\\\/xrS8vnIRmKvKR5z0eGRfLmLO9AfN7wNXabTzabWZT5ehOobeJQ+KHqp8zaS1XCpv73mXvcEJ2hyav9X+ephbYBuuxv2Ir03Z5a+rr70xDfGFYMOTV2psPP9yfy9R+LvR\\\\\\\/Tz6fyTbZG8BufHW58NU1K0X1hEM0FG9FS\\\\\\\/8SLWTiTy9PZ9RdskqzoKVoBYpiN3OSmQaT3E8WM+8CkWhFFr1fQJX9Yr9fpsW3pYnYdvZcXCwclvI\\\\\\\/N3KsHd+b+k67Ddb7bD4t7McEA7giQ+vsCZDX27iibq7l1RlAXitPW1L8Wln7P8ZE8jYP6EpE1IUBzKwfcqP3pTfPAYIts4HQF3fMEDfTeTwXaAs+XkA48c39H\\\\\\\/gND7XGTJZ4N4nVXIq4MxzpeEZx9JNXiYsXW9B8WeQTzzF9fznYx4ueLVcvXH6BxdE1HvEV7+lzLBu\\\\\\\/+W\\\\\\\/dyn7mgxqmPhLroQXsnfEJ3lLqbRCQ3gFgkhGstczjYTn7pRazmEpG53tWVVl5euKt865ZvwhJjxtIiEUF4el7YoGO65nIb+O8yGGHZqxlblqzzWSGzsU2CX8vXRIUA8IDc1S1dtxNHKUexBiCZzsvmyvdI\\\\\\\/BKUbEq3jWK4tu8YTI5bohNE7TAC+AhfAgV0Lkq5nR36ZY2PrZ8m9FldwncwwiPaSEj7WIzL7ZoXxDdkO5LkaHXw\\\\\\\/uFKAk\\\\\\\/k44jozM31FJBBM8PEk7aYjGnfBOBnpPAL+O0LKkz9HRXolgK\\\\\\\/SoXhAV8EOVUhMg0R8UesYQPhKCjxjzIN6xjVgwjHIKTb0XBevsZQj5GsHm6j\\\\\\\/kIWYJW2NT9\\\\\\\/ei7Ad3+3GS9cAJ71W4c0RM5NzYw7CUDq4zlAFzDgai\\\\\\\/GP2U0BhIuMbGQoq3vuWJXkeBOkpVhB0T\\\\\\\/OXmIFX5PPUBTSjzm9d\\\\\\\/U1oB2OfVCozDyQJP\\\\\\\/7dV54iXUIyv9aTPB5WfJ8KC0Tw0aWTSzcn1XwteMDZg4yMF3jYmPBr0CDjzAGujST2cNi8T7CrjlsDpd7c0\\\\\\\/2FFbzTZKr5Hu8qyT3tuMt7NbH1jQz6cAm0P2\\\\\\\/EdPFdJDo1DlfYOuqYn3JHLwPOfYyMh9pA2IoHQMpc4pJpWhGJBkNsJYWBo+xN8CcJfG2aUVHDfOwxHbGUV1bxypmHLYLFiQziWGRj37KMBHM+C1c8kxYsr9rqlsM9YhZWrtq2BU2Q28uUfSEbv4Cg6wvuTnUan4Wu8yKOmr2zDFxdgUA2qJYboRs37450m+0J\\\\\\\/QeauEEhgXkx7gjr2v4MWPbgk6IvpgvxGVuNIhnLJ22SSrzurUF\\\\\\\/9Gr8bcpPU50NExeRnSuV8YOylxM8E322Olx085qxLX\\\\\\\/dFxw6uHzFn5te70Lps2x+pSDCdbrB8lqA3mLSSdIZq7tB+xJ\\\\\\\/UO4aZ0MXW00SNpqCuiUiRbO66U9BsOrHfJOzqIGiLEmWQRJN23BnZrdJ7NmptGq6T9o0tvP9PRe9FzaK5OE4h\\\\\\\/XvymLbPkAoIeU21Tu4Rge\\\\\\\/OzNFxebfHVUC3oCYqwPmHrzJLNlNKw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022CCntyAJ0uF+Y6vkkJw+BNg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b7e2bc22efa5df06c0d7e260c58f08388161c71c0bc2b35e0bbfa0a94f3c4ff2\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022KD6EOPjdgRsy7+wOYTpIN481Cn7VrDPm\\\\\\\/a2to4C2TMs9MFlZQp6FGSqxBzjm6JUxx3HHJd4v\\\\\\\/tLEGcEW3Zr7jQIHJ5iYobPMF13BskKGNylNeAfjfOoX0\\\\\\\/9NzrgX3UB+Yfrovz42MPBpjj9UcCwt5uzkYG0FlBu+lttKYrKhwAtjhpewhAcD9FrOtQrWbdkuSzpGuC8VKM51BTGHMjQvMsBSTag76nEhM89+7Sbx+hU7J4WEwrr2d1y7+igva9g9diusSqoIkQz+JxCzFdjqBbdGF9c6Mu7UtpCmOFkOc\\\\\\\/Cd+B1BmzC2GaJFZJvU8CqUT5u4Sggz+O9+vdeQrLc\\\\\\\/5ulZ4HS+ufFT5ch3dOb7tJa8YtmVjKoQkYRlFITHYplIdNC9PDqCQY31vwOWSQoDQDJacKdU5O2tjKI85P6I8cN6jFZWZOHyNSOZ63h0nQAFgjyzQGvxx9wS3peUYbBcgj9\\\\\\\/cD5X8l3Cz5sgr1piA5uMjuWK2N3kHOH8NnSq4GTKF269dCDNlGnBB4JODrEx+NzqzcMiXAwNvWxA0vEJQzJpxfH8Q2wS6mSXmXAaTmZI5Sq4HVjWssoCYW2ipw7G0uRH2uHPXlWLUCSCRoP2OQUQjHn9kJHcomLjnHOEzibvKVbWyBBSYoCQWwK2qPYOI8H1cwBKifZRR71YHhRnttpos05gm9PPBuKZuc78iObwgqOXrnKFiz6w9Q+AhkemwnqIQFL9gBd83jSHkuwA2xN5p7BD7IUYBZk5deX1gtQlIbLNZaGtRfuSxlaz+8aYEdGS4wPbI9C0iq\\\\\\\/vxtEtyzakszCbMA5k4YeFi594knhC6h9Wh5tnJDuvj05Ba4zMa7fK8CELZ+tQ0Kr6x81Wly7uU42Jxy\\\\\\\/p8kG+iNWAD0cSPAiS1RlIw9CI04UFTCdQcwxeystvvR3bqvn1hCU0uCDWEJle7yN8n0zzlDqgAIMmCALUeHAIH6w\\\\\\\/5D2qiWTrs3WeCHNt9UqulvGsAUImhWAPxF2CnIfVtb2M2lo5i1eT6stAH4rma38+PPJi+B\\\\\\\/4HgwcCQokQL0kAg4o2t2EIGFiVP6KU4fqDexgQHMNpPPCj82gOYNOpXSlXs4a2zpECyC6LkZAjgLqyaB5Qodx+HUm8BbEyxp+a9h4KkUSTBXzGwIhQ48A4xTs3m9QgSG91WtqArNPwBpIU7fwTKy6032VOfr8krHS7ISTF3YyUOLxbyl7feA2oJSysWxHP1bZ0FH9SjNe08AU1DLgRsI9Y+b\\\\\\\/lWxzGszhKTMoy7HmFM2wMbi3EoMcTqN9meVuxyScgvgN7tGKPQnwuNHQENdoBfifhrvxla7Eh\\\\\\\/Glwsm9irGj6XqxP0+CPUK5u8rwWyrTtQUlHa1cfl661ekkAwHXs5LoFCzV0KQ5k\\\\\\\/wIzEYgyAaGiCWudflyIHSioWCO1l9yvZyDpww8wHiBIKrVpmI0FJgT\\\\\\\/LD\\\\\\\/moqG2TyUrWdUf7rpDQa15gC3YB0ZGYvRTeTVsPVDcfucyRdBLwmDdutO0tPcrs9GG1IvS5J6ZS3Uf3EkTYyAnZnezwjDYTmL3r6hiVzgQtlDx1PMRs3VtWB0fTUAqhpdE7\\\\\\\/aHadawkwAk6SOTlc5+ufSp8a8qxpOuw0jwi2e5ADVZQiA2L+NHTUi0mMl1UVeW4iKi\\\\\\\/VDFH1MILDsJ8BdW8eNaPJRrWA0ggAVXU1DDJm04o9epQ\\\\\\\/OMdBWoc+ZVdw\\\\\\\/ni6EJnnW3qlK9XwPGFHDtOMgtmZfRXIE9bpCdkE3elklZYJeRHuhbPaffN+ZLLB4KAqiVRFQ+TiDkErOEW7KJkLZPQS52+UvTJF\\\\\\\/ShJORDrs4PUDjh3u3\\\\\\\/rNveLLWdmdx\\\\\\\/BoaG7b3SgLnI6wHYVUUBd7DMe\\\\\\\/wAReOgT7BPWKVlAsYhErnhUtkf6ESG5KfttvDuEGIJWrI006tzgt+KS4yXInM4z+f\\\\\\\/v7eoMgKjBmPxxdMTaIfuUFEFI6J8UN09fKqVz8lT\\\\\\\/mGxV5HgRt6xHsFfX0c4AJxwZfPIBe6A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ghEzo7DJAtb5ZydT4NwYGA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a03fa76a888114802a5c6b6abf9c38fc2c2b0b33694bbfbf042a2cff25016606\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-484259 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1511" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-iDCsxHXCXFvkhnWH3sTiSvcps81iOaa0LoljZcx1g48" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1511" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-AIN6rlKSlHYYQ5S6SBiLaHFNdn23YY4mJdCD_c24xnc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/5/1499" title="Evaluating Methods of Updating Training Data in Long-Term Genomewide Selection" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/5/1525" title="Genomic Comparison of Indigenous African and Northern European Chickens Reveals Putative Mechanisms of Stress Tolerance Related to Environmental Selection Pressure" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484259" id="node484259" data-pisa="ggg;7/5/1511" data-pisa-master="ggg;g3.117.040071" data-apath="/ggg/7/5/1511.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Modulating Crossover Frequency and Interference for Obligate Crossovers in <em>Saccharomyces cerevisiae</em> Meiosis</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Parijat Chakraborty</span>, <span class="highwire-citation-author" data-delta="1">Ajith V. Pankajam</span>, <span class="highwire-citation-author" data-delta="2">Gen Lin</span>, <span class="highwire-citation-author" data-delta="3">Abhishek Dutta</span>, <span class="highwire-citation-author" data-delta="4">G. Nandanan Krishnaprasad</span>, <span class="highwire-citation-author" data-delta="5">Manu M. Tekkedil</span>, <span class="highwire-citation-author" data-delta="6">Akira Shinohara</span>, <span class="highwire-citation-author" data-delta="7">Lars M. Steinmetz</span> and <span class="highwire-citation-author" data-delta="8">K. Thazath Nishant</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1511-1524;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.040071 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484259" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Parijat Chakraborty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research, Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Parijat%2BChakraborty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chakraborty%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AParijat%2BChakraborty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ajith V. Pankajam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research, Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ajith%2BV.%2BPankajam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pankajam%20AV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAjith%2BV.%2BPankajam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Gen Lin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genome Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gen%2BLin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lin%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGen%2BLin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Abhishek Dutta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research, Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abhishek%2BDutta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dutta%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbhishek%2BDutta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">G. Nandanan Krishnaprasad </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research, Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=G.%2BNandanan%2BKrishnaprasad%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Krishnaprasad%20GN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AG.%2BNandanan%2BKrishnaprasad%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Manu M. Tekkedil </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genome Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manu%2BM.%2BTekkedil%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tekkedil%20MM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AManu%2BM.%2BTekkedil%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Akira Shinohara </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Protein Research, Osaka University, 565-0871, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Akira%2BShinohara%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shinohara%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAkira%2BShinohara%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Lars M. Steinmetz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genome Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, Stanford University, California 94305</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Stanford Genome Technology Center, Palo Alto, California 94304</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lars%2BM.%2BSteinmetz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steinmetz%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALars%2BM.%2BSteinmetz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">K. Thazath Nishant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research, Thiruvananthapuram, Trivandrum 695016, India</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Center for Computation, Modelling and Simulation, Indian Institute of Science Education and Research, Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=K.%2BThazath%2BNishant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nishant%20KT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AK.%2BThazath%2BNishant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nishantkt@iisertvm.ac.in" class="" data-icon-position="" data-hide-link-title="0">nishantkt@iisertvm.ac.in</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/5/1511" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:484259" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:484259/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1511.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:484259" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:484259/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1511.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:484259" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:484259/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1511.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:484259" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:484259/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/5/1511.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:484259" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:484259/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Meiotic crossover frequencies show wide variation among organisms. But most organisms maintain at least one crossover per homolog pair (obligate crossover). In <em>Saccharomyces cerevisiae</em>, previous studies have shown crossover frequencies are reduced in the mismatch repair related mutant <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ and enhanced in a meiotic checkpoint mutant <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ by up to twofold at specific chromosomal loci, but both mutants maintain high spore viability. We analyzed meiotic recombination events genome-wide in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutants to test the effect of variation in crossover frequency on obligate crossovers. <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ showed ∼30% genome-wide reduction in crossovers (64 crossovers per meiosis) and loss of the obligate crossover, but nonexchange chromosomes were efficiently segregated. <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ showed ∼50% genome-wide increase in crossover frequency (137 crossovers per meiosis), elevated noncrossovers as well as loss of chromosome size dependent double-strand break formation. Meiotic defects associated with <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a>∆</em> did not cause significant increase in nonexchange chromosome frequency. Crossovers were restored to wild-type frequency in the double mutant <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ (100 crossovers per meiosis), but obligate crossovers were compromised. Genetic interference was reduced in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ. Triple mutant analysis of <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ with other resolvase mutants showed that most of the crossovers in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ are made through the <a href="http://www.yeastgenome.org/locus/S000002794/overview">Mus81</a>-<a href="http://www.yeastgenome.org/locus/S000000302/overview">Mms4</a> pathway. These results are consistent with a requirement for increased crossover frequencies in the absence of genetic interference for obligate crossovers. In conclusion, these data suggest crossover frequencies and the strength of genetic interference in an organism are mutually optimized to ensure obligate crossovers.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/crossover-frequency" class="hw-term hw-article-keyword hw-article-keyword-crossover-frequency" rel="nofollow">crossover frequency</a></li><li class="kwd"><a href="/keyword/crossover-assurance" class="hw-term hw-article-keyword hw-article-keyword-crossover-assurance" rel="nofollow">crossover assurance</a></li><li class="kwd"><a href="/keyword/meiotic-chromosome-segregation" class="hw-term hw-article-keyword hw-article-keyword-meiotic-chromosome-segregation" rel="nofollow">meiotic chromosome segregation</a></li><li class="kwd"><a href="/keyword/genetic-interference" class="hw-term hw-article-keyword hw-article-keyword-genetic-interference" rel="nofollow">genetic interference</a></li><li class="kwd"><a href="/keyword/genome-wide-recombination-map" class="hw-term hw-article-keyword hw-article-keyword-genome-wide-recombination-map" rel="nofollow">genome wide recombination map</a></li></ul><p id="p-3">Meiosis is a reductional division that produces haploid gametes or spores from diploid progenitor cells. Ploidy reduction is achieved by one round of DNA replication, followed by two consecutive nuclear divisions (Meiosis I and II), producing four daughter cells (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Roeder 1997</a>). Crossovers promote the formation of chiasma which serves as a physical linkage between two homologs and opposes the spindle generated forces that pull apart the homolog pairs. This opposing set of forces provides the tension necessary to promote proper disjunction of homolog pairs at Meiosis I (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Petronczki <em>et al.</em> 2003</a>). Failure to maintain at least one crossover per homolog pair increases the probability of nondisjunction, resulting in aneuploid gametes (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Serrentino and Borde 2012</a>). Although crossovers are important for chromosome segregation, nonexchange chromosomes have been observed to segregate accurately forming viable gametes (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hawley <em>et al.</em> 1992</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Davis and Smith 2003</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Kemp <em>et al.</em> 2004</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Newnham <em>et al.</em> 2010</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Krishnaprasad <em>et al.</em> 2015</a>).</p><p id="p-4">Meiotic crossovers (and noncrossovers) are initiated in <em>S. cerevisiae</em> by 150–170 programmed double-strand breaks (DSBs) formed by an evolutionarily conserved topoisomerase-like protein <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> and other accessory proteins (reviewed in <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Neale and Keeney 2006</a>). The frequency and distribution of crossovers on chromosomes is a tightly regulated process. Four known aspects of this regulation are crossover interference, obligate crossovers, crossover homeostasis, and crossover invariance. Crossover interference regulates the spatial patterning of crossovers, where the presence of a crossover prevents the formation of a second crossover in its vicinity. As a consequence, an even distribution of crossovers is ensured, and closely spaced crossovers are less frequently observed than expected from a random distribution (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Berchowitz and Copenhaver 2010</a>). An obligate crossover per homolog pair facilitates disjunction. The obligate crossover is not a specific type of crossover, but refers to the observation that process(es) that generate most of the crossovers result in at least one crossover per homolog pair (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Barchi <em>et al.</em> 2008</a>). The obligate crossovers are ensured in different organisms with different crossover frequencies. For example, organisms that have strong genetic interference like <em>Caenorhabditis elegans</em>, <em>Drosophila</em>, <em>Arabidopsis</em>, mouse, and humans ensure obligate crossovers, although the number of crossovers per bivalent pair is low (∼1–2) (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Copenhaver <em>et al.</em> 2002</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Hillers and Villeneuve 2003</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Berchowitz and Copenhaver 2010</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Libuda <em>et al.</em> 2013</a>). But organisms with weaker interference, like <em>S. cerevisiae</em>, make ∼90 crossovers, or ∼6 times the number of bivalent pairs (16), to ensure an obligate event. Crossover homeostasis ensures that the number of crossovers is maintained (at the expense of noncrossovers) when the DSB precursors become limiting (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Martini <em>et al.</em> 2006</a>). This crossover buffering mechanism is also seen when the DSBs are increased (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Cole <em>et al.</em> 2012</a>). Crossover invariance is another mechanism that maintains the number of crossovers in organisms that lack interference. This phenomenon refers to the observation that, in <em>Schizosaccharomyces pombe</em>, the ratio of intersister to interhomolog repair is modulated in response to variations in DSB intensity to ensure uniform crossover distribution (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Hyppa and Smith 2010</a>). Recent studies suggest crossover interference, obligate crossovers, and crossover homeostasis are the outcomes of a single crossover patterning process regulated by the “mechanical stress relief” of chromosomes (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Wang <em>et al.</em> 2015</a>; <a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Zickler and Kleckner 2016</a>). As per this model, crossover interference and homeostasis are linked processes. But the designation of an obligate crossover is not dependent on either interference or homeostasis (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Zhang <em>et al.</em> 2014</a>; <a id="xref-ref-82-2" class="xref-bibr" href="#ref-82">Wang <em>et al.</em> 2015</a>).</p><p id="p-5">The majority of the crossovers (Class I) in <em>S. cerevisiae</em> show interference, and are formed through a pathway involving the ZMM (<a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a>, <a href="http://www.yeastgenome.org/locus/S000003218/overview">Zip2</a>, <a href="http://www.yeastgenome.org/locus/S000004386/overview">Zip3</a>, <a href="http://www.yeastgenome.org/locus/S000003218/overview">Zip4</a>, <a href="http://www.yeastgenome.org/locus/S000003220/overview">Mer3</a>, <a href="http://www.yeastgenome.org/locus/S000001196/overview">Spo16</a>, and <a href="http://www.yeastgenome.org/locus/S000001891/overview">Msh4</a>-<a href="http://www.yeastgenome.org/locus/S000002313/overview">Msh5</a>) and STR (<a href="http://www.yeastgenome.org/locus/S000004802/overview">Sgs1</a>, <a href="http://www.yeastgenome.org/locus/S000005945/overview">Rmi1</a>, and <a href="http://www.yeastgenome.org/locus/S000004224/overview">Top3</a>) procrossover factors and mismatch repair related proteins <a href="http://www.yeastgenome.org/locus/S000005559/overview">Exo1</a> and MutLγ (<a href="http://www.yeastgenome.org/locus/S000004777/overview">Mlh1</a>-<a href="http://www.yeastgenome.org/locus/S000006085/overview">Mlh3</a>) (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Allers and Lichten 2001</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Hunter and Kleckner 2001</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Borner <em>et al.</em> 2004</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Nishant <em>et al.</em> 2008</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Shinohara <em>et al.</em> 2008</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">De Muyt <em>et al.</em> 2012</a>; <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Zakharyevich <em>et al.</em> 2012</a>). Cytological studies in the mouse have shown that the MutLγ complex acts downstream of <a href="http://www.yeastgenome.org/locus/S000001891/overview">Msh4</a>-<a href="http://www.yeastgenome.org/locus/S000002313/overview">Msh5</a> (MutSγ) (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Baker <em>et al.</em> 1996</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Lipkin <em>et al.</em> 2002</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Kolas <em>et al.</em> 2005</a>). Further physical, genetic, and biochemical analysis in <em>S. cerevisiae</em> suggest MutLγ has an endonuclease activity that can generate crossovers from dHJs in a pathway comprising MutSγ, <a href="http://www.yeastgenome.org/locus/S000005559/overview">Exo1</a> and <a href="http://www.yeastgenome.org/locus/S000004802/overview">Sgs1</a> (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Nishant <em>et al.</em> 2008</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Hunter 2011</a>; <a id="xref-ref-85-2" class="xref-bibr" href="#ref-85">Zakharyevich <em>et al.</em> 2012</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Rogacheva <em>et al.</em> 2014</a>). Noninterfering crossovers are formed by a separate pathway involving the structure specific endonuclease <a href="http://www.yeastgenome.org/locus/S000002794/overview">Mus81</a>-<a href="http://www.yeastgenome.org/locus/S000000302/overview">Mms4</a> (Class II crossovers). Mainly aberrant and multichromatid joint molecules are resolved by this pathway (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">de los Santos <em>et al.</em> 2003</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Oh <em>et al.</em> 2008</a>). The structure specific endonucleases <a href="http://www.yeastgenome.org/locus/S000000843/overview">Yen1</a> and <a href="http://www.yeastgenome.org/locus/S000000432/overview">Slx1</a>-<a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> have a minor role in meiotic crossover formation, which is observed in <em><a href="http://www.yeastgenome.org/locus/S000000302/overview">mms4</a></em>Δ<em>/<a href="http://www.yeastgenome.org/locus/S000000302/overview">mms4</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000004802/overview">sgs1</a></em>Δ background (<a id="xref-ref-85-3" class="xref-bibr" href="#ref-85">Zakharyevich <em>et al.</em> 2012</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Higashide and Shinohara 2016</a>). Genetic interference between crossovers is compromised in most ZMM mutants (<em><a href="http://www.yeastgenome.org/locus/S000002693/overview">zip1</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000003220/overview">mer3</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000001891/overview">msh4</a></em>Δ, and <em><a href="http://www.yeastgenome.org/locus/S000002313/overview">msh5</a></em>Δ) (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Sym and Roeder 1994</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Nakagawa and Ogawa 1999</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Novak <em>et al.</em> 2001</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Argueso <em>et al.</em> 2004</a>; <a id="xref-ref-85-4" class="xref-bibr" href="#ref-85">Zakharyevich <em>et al.</em> 2012</a>), but the distribution of <a href="http://www.yeastgenome.org/locus/S000004386/overview">Zip3</a> foci (measured in physical distances), which mark DSB sites designated to be repaired as crossovers, show interference in <em><a href="http://www.yeastgenome.org/locus/S000002693/overview">zip1</a></em>Δ and <em><a href="http://www.yeastgenome.org/locus/S000001891/overview">msh4</a></em>Δ mutants (<a id="xref-ref-88-2" class="xref-bibr" href="#ref-88">Zhang <em>et al.</em> 2014</a>).</p><p id="p-6">It is not clear why organisms have vastly different crossover frequencies and how this is related to ensuring an obligate crossover. In this study, we address the relationship of obligate crossovers with crossover frequency using <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ and <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutants. In <em>S. cerevisiae</em>, both these mutants have very different crossover frequencies but show high spore viability. At specific chromosomal loci, <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ shows up to a 50% reduction in crossover frequency but with good spore viability ranging from 72% and up to 92% depending on the strain background (<a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Nishant <em>et al.</em> 2008</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Cotton <em>et al.</em> 2010</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Sonntag Brown <em>et al.</em> 2013</a>). <a href="http://www.yeastgenome.org/locus/S000000390/overview">Pch2</a> is an evolutionarily conserved hexameric ring AAA+ ATPase protein that has roles in meiotic checkpoint signaling triggered by unrepaired DSBs, remodeling the chromosome axis at the future crossover sites and interhomolog bias (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">San-Segundo and Roeder 1999</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Borner <em>et al.</em> 2008</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Ho and Burgess 2011</a>; <a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Zanders <em>et al.</em> 2011</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Chen <em>et al.</em> 2014</a>). <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutants are observed to make 50% more crossovers than wild type at specific loci, and also have good viability (95%) (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Zanders and Alani 2009</a>; but see <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Joshi <em>et al.</em> 2009</a>). The <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ and <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutants are therefore useful to test the effect of a wide range of crossover frequencies on obligate crossovers. It is important to note that pleiotropic effects of <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ could affect the interpretation of changes in crossover frequencies on the obligate crossover.</p><p id="p-7">We performed genome-wide mapping of meiotic recombination events in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ using the S288c/YJM789 hybrid. Crossovers in all three mutants, <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ (64 crossovers per meiosis on average), <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ (137 crossovers per meiosis on average) and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ (100 crossovers per meiosis on average) showed reduced genetic interference. Noncrossovers were not affected in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, but increased in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ. Crossover and noncrossover data suggest loss of chromosome size dependent DSB formation in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutants. <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ showed uniform crossover density on all chromosomes consistent with loss of genetic interference. Nonexchange chromosomes were abundant in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a>∆</em> (47% of meioses), but efficiently segregated. <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ (7%) had a reduced percentage of meioses with nonexchange chromosomes, showing that high crossover frequencies can promote crossover assurance in the absence of genetic interference. Even though the <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutant made as many crossovers as wild type, 20% of meioses showed nonexchange chromosomes, consistent with noninterfering crossovers being inefficient in promoting crossover assurance.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Media and strains</h3><p id="p-8">All <em>S. cerevisiae</em> strains used in this study (S288c, YJM789, and SK1) were grown on either YPD (yeast extract, peptone, and dextrose) or synthetic complete medium at 30° (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Rose <em>et al.</em> 1990</a>). The drugs, geneticin (Invitrogen), nourseothricin (Werner BioAgents), and hygromycin (HiMedia) were added to YPD as described in <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Goldstein and McCusker (1999)</a>. Sporulation media for the genetic analysis was prepared as described in <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Argueso <em>et al.</em> (2004)</a>. For whole genome recombination analysis, S288c and YJM789 strains were used. For tetrad analysis and cytology, the SK1 strains, EAY1108/EAY1112, NHY1168/NHY1162, and their derivatives, were used (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Argueso <em>et al.</em> 2004</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Martini <em>et al.</em> 2006</a>). All strains were transformed using standard techniques (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gietz <em>et al.</em> 1995</a>). The <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ::<em>kanMX4</em> strains were created using plasmid pEAI168. <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ::<em>natMX4</em>, <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>Δ::<em>kanMX4</em> and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ::<em>hphMX4</em> were made using deletion constructs amplified by PCR. The meiotic depleted allele of <em><a href="http://www.yeastgenome.org/locus/S000000302/overview">MMS4</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000000302/overview">mms4</a>-md</em>) in EAY1108/EAY1112 was created by transformation with the <em>kanMX-pCLB2-3HA-<a href="http://www.yeastgenome.org/locus/S000000302/overview">MMS4</a></em> PCR product generated from the haploid parents of MJL3172 (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">De Muyt <em>et al.</em> 2012</a>). Strain information is given in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/TableS1.docx">Table S1</a>.</p></div><div id="sec-3" class="subsection"><h3>Tetrad analysis</h3><p id="p-9">Diploids were sporulated following the zero growth mating protocol (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Argueso <em>et al.</em> 2003</a>). Haploid strains were mixed together on synthetic complete media. After incubation for 4 hr at 30°, the resulting diploids were patched on sporulation media. After 2 d of incubation at 30°, tetrads were dissected on synthetic complete medium using a Zeiss dissection microscope. The dissected tetrads were grown for 2 d, after which they were replica plated to various selective media. The replica plates were scored after 1 d of incubation. Marker segregation was analyzed using the recombination analysis software RANA (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Argueso <em>et al.</em> 2004</a>).</p></div><div id="sec-4" class="subsection"><h3>Meiotic time course</h3><p id="p-10">Wild type and <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ diploids were streaked on YPD media, and a single colony was inoculated in 4 ml YPD and grown overnight at 30°. To obtain synchronized meiosis, cultures were grown in presporulation medium (SPS) (0.5% yeast extract, 1% peptone, 0.67% yeast nitrogen base, 1% potassium acetate, and 0.05 M potassium biphthalate) before transfer to the sporulation medium (2% potassium acetate, amino acids, and 0.001% polypropylene glycol) (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Murakami <em>et al.</em> 2009</a>).</p></div><div id="sec-5" class="subsection"><h3>Cytology</h3><p id="p-11">Meiotic cultures (5 ml) of wild type and <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ from 2 to 8 hr postinduction into meiosis were collected at hourly intervals. Cultures were treated with Zymolyase 100T to obtain spheroplasts. Spheroplasts were washed and resuspended in MES/sorbitol solution (1 M sorbitol, 1 mM EDTA, 0.5 mM MgCl<sub>2</sub>, 0.1 M MES pH 6.5). Chromosome spreads were prepared on ethanol washed glass slides, using detergent (1% Lipsol) and 4% paraformaldehyde/3.4% sucrose as fixative. Slides were dried overnight and immunostained as described previously (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bishop 1994</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Shinohara <em>et al.</em> 2000</a>).</p></div><div id="sec-6" class="subsection"><h3>Analysis of Rad51 foci</h3><p id="p-12">Slides were stained with rabbit anti-Rad51 antibody (1:500) (a gift from Dr. Akira Shinohara), incubated overnight at 4°, followed by 2 hr incubation with secondary antibody anti-rabbit TRITC (1:1500) (Jackson ImmunoResearch). Immunofluorescence images of <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> foci were captured using 100× oil immersion objective on a Leica SP5 confocal microscope. Images were captured at 0.25 μm interval Z sections. Maximum intensity images were produced by projecting the optical sections (Z stacks). For each time point 100 images were analyzed in three independent experiments. Image analysis and 3D deconvolution was done using Leica Application Suite (LAS) Advanced Fluorescence (AF) Lite 2.8.0 software.</p></div><div id="sec-7" class="subsection"><h3>DNA extraction and whole genome sequencing of meiotic spores</h3><p id="p-13">Spore colonies from the four spore viable tetrads of <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a>∆</em>, <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a>∆</em>, and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a>∆ <a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a>∆</em> were grown overnight at 30° in 4 ml YPD broth. DNA was isolated from the culture using PrepEase DNA isolation kit (Affymetrix). Whole genome sequencing on Illumina platform was performed at Fasteris, Switzerland, as well as at the European Molecular Biology Laboratory (EMBL) Genomics Core Facilities (GeneCore), Heidelberg, Germany.</p></div><div id="sec-8" class="subsection"><h3>Annotation of recombination events</h3><p id="p-14">The raw reads were demultiplexed using the NGS QC Toolkit (version 2.3.3, <a href="http://www.nipgr.res.in/ngsqctoolkit.html">http://www.nipgr.res.in/ngsqctoolkit.html</a>), and processed for quality control (QC) using the trimmomatic tool (Version 0.36, <a href="http://www.usadellab.org/cms/index.php?page=trimmomatic">http://www.usadellab.org/cms/index.php?page=trimmomatic</a>) (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Bolger <em>et al.</em> 2014</a>). These QC filtered high quality reads were then mapped to the <em>S. cerevisiae</em> genome (version R64-1-1, 2011) using bowtie 2 (version 2.2.6) (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Langmead and Salzberg 2012</a>). Only uniquely mapped reads were considered for variant calling using Picard tools. Local realignment around indels was performed to reduce misalignment. Genotyping was done using GATK (UnifiedGenotyper, version 3.4-46) (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">McKenna <em>et al.</em> 2010</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">DePristo <em>et al.</em> 2011</a>). From the alignment, an average of 59,215 SNPs was genotyped. Recombination events, such as crossovers, noncrossovers (type 0 gene conversions exhibiting 1:3 or 3:1 segregation of SNP markers), crossover independent gene conversions (type 0, 2, 3, and 4 gene conversions), and crossover associated gene conversions were annotated using the CrossOver program (v6.3) of the ReCombine program suite (v2.1) (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Anderson <em>et al.</em> 2011</a>). Parameters used for the CrossOver program were as described previously in <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Krishnaprasad <em>et al.</em> (2015)</a>. Custom R scripts were used to convert vcf files to segregation files (input file for the CrossOver program), to generate plots from the CrossOver program output and to perform various statistical tests. Detection of copy number variation was performed as described in <a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Krishnaprasad <em>et al.</em> (2015)</a>.</p></div><div id="sec-9" class="subsection"><h3>Interference analysis</h3><p id="p-15">Interference analysis using the one pathway gamma model was performed as described previously (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">McPeek and Speed 1995</a>; <a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Krishnaprasad <em>et al.</em> 2015</a>). For the two pathway analysis of interference, crossover distances between each crossover per chromosome was calculated for each tetrad, and converted to centimorgans. A gamma mixture model was fitted using an EM algorithm similar to that in the R package mixtools (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Benaglia <em>et al.</em> 2009</a>). The number of mixture components was fixed at two and the shape parameter for one of the components (Class II crossovers) was fixed at one. For the Coefficient of Coincidence (CoC) analysis, the genome was partitioned into 25 kb bins. The frequency of crossover events in each bin was calculated along with the expected and observed frequencies of double crossovers in all pairs of bins. The CoC (observed crossover frequency/expected crossover frequency) was analyzed for all bin pairs (with nonzero expected frequency) separated by a given interval (in kilobase) and averaged. Chi-square test was used to measure the statistical significance of observed and expected double crossover frequencies between wild type and the mutants.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-16">All strains listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/TableS1.docx">Table S1</a> are available upon request. Sequence data are available from the National Centre for Biotechnology Information Sequence Read Archive: Accession number SRP082254. The raw recombination data files and the custom R scripts are available online at the Dryad data repository (<a href="http://datadryad.org">http://datadryad.org</a>; <a href="http://dx.doi.org/10.5061/dryad.5vm4g">http://dx.doi.org/10.5061/dryad.5vm4g</a>).</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Genome-wide recombination maps for mlh3<strong>Δ</strong>, pch2<strong>Δ</strong>, and mlh3<strong>Δ</strong> pch2<strong>Δ</strong> by whole genome sequencing</h3><p id="p-17"><em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutations were introduced in the S288c and YJM789 strains. Spore viability of <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ (85%) was similar to the wild-type S288c/YJM789 hybrid (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FigureS1.pptx">Figure S1</a> and <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Spore viability of <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ (75%), in the S288c/YJM789 hybrid, was reduced compared to wild-type hybrid (<em>P</em> = 0.0001, Fischer’s exact test) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FigureS1.pptx">Figure S1</a> and <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutant also showed significantly less viability compared to wild type (59%, <em>P</em> = 7.9 × 10<sup>−21</sup>, Fischer’s exact test). We performed whole genome sequencing of spores from 54 four-viable spore tetrads of the S288c × YJM789 hybrid bearing <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/TableS2.xlsx">Table S2</a>) (sequence data available from National Centre for Biotechnology Information Sequence Read Archive under accession number: SRP082254). High-resolution genome-wide recombination data were generated in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutants by analyzing segregation of SNPs in the 54 tetrads (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FileS1.zip">File S1</a>). These include 19 tetrads in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, 15 tetrads in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ and 20 tetrads in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ. Genome-wide recombination data for wild type were generated from 66 tetrads by combining previously published studies (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Mancera <em>et al.</em> 2008</a>; <a id="xref-ref-46-5" class="xref-bibr" href="#ref-46">Krishnaprasad <em>et al.</em> 2015</a>). Crossover and noncrossover counts for each of the 120 tetrads are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/TableS3.xlsx">Table S3</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484373/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484373/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Crossover (CO), noncrossover (NCO) counts and spore viability of <em>mlh3</em><strong>Δ</strong>, <em>pch2</em><strong>Δ</strong>, <em>mlh3</em><strong>Δ</strong> <em>pch2</em><strong>Δ</strong> mutants in the S288c/YJM789 hybrid</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Crossover defects in mlh3<strong>Δ</strong> are distinct from msh4<strong>Δ</strong></h3><p id="p-20">The <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ mutant showed a significant genome-wide reduction in crossovers compared to wild type (64 crossovers on average in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em>vs.</em> 93 in wild type, <em>P</em> = 1.7 × 10<sup>−14</sup>, <em>t</em>-test) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). These results are consistent with crossover data obtained from analysis of specific loci in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ (<a id="xref-ref-60-4" class="xref-bibr" href="#ref-60">Nishant <em>et al.</em> 2008</a>; <a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Sonntag Brown <em>et al.</em> 2013</a>). Genome-wide crossover defects are therefore stronger in <em><a href="http://www.yeastgenome.org/locus/S000001891/overview">msh4</a></em>Δ (average of 49 crossovers per meiosis) compared to <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ (<em>P</em> = 2.3 × 10<sup>−3</sup>, <em>t</em>-test), although the <a href="http://www.yeastgenome.org/locus/S000001891/overview">Msh4</a>-<a href="http://www.yeastgenome.org/locus/S000002313/overview">Msh5</a>, <a href="http://www.yeastgenome.org/locus/S000004777/overview">Mlh1</a>-<a href="http://www.yeastgenome.org/locus/S000006085/overview">Mlh3</a> proteins are thought to act in the same pathway for making crossovers (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Chen <em>et al.</em> 2008</a>; <a id="xref-ref-46-8" class="xref-bibr" href="#ref-46">Krishnaprasad <em>et al.</em> 2015</a>). Previous work by <a id="xref-ref-75-3" class="xref-bibr" href="#ref-75">Sonntag Brown <em>et al.</em> (2013)</a> also showed stronger crossover defects in <em><a href="http://www.yeastgenome.org/locus/S000002313/overview">msh5</a></em>Δ (37 cM) compared to <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ (54.5 cM) for chromosome XV in the SK1 EAY1108/EAY1112 background. Average noncrossover counts in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ (50 per meiosis) were similar to wild type (46 per meiosis) (<em>P</em> = 0.29, <em>t</em>-test). Since the crossover defects in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ were not associated with a concomitant increase in noncrossovers, perhaps in the absence of the <a href="http://www.yeastgenome.org/locus/S000006085/overview">Mlh3</a> protein DSBs that were not repaired as crossovers may be repaired using the sister chromatid.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1511/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Crossover, noncrossover and total events per meiosis for wild type, mlh3&#x394;, pch2&#x394;, and mlh3&#x394; pch2&#x394;. The box plots show minimum, first quantile, median, third quantile, and maximum count." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1906276771" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Crossover, noncrossover and total events per meiosis for wild type, &lt;em&gt;mlh3&lt;/em&gt;&#x394;, &lt;em&gt;pch2&lt;/em&gt;&#x394;, and &lt;em&gt;mlh3&lt;/em&gt;&#x394; &lt;em&gt;pch2&lt;/em&gt;&#x394;. The box plots show minimum, first quantile, median, third quantile, and maximum count.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1511/F1.medium.gif" width="440" height="268"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/5/1511/F1.medium.gif" width="440" height="268"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1511/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1511/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484348" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-21" class="first-child">Crossover, noncrossover and total events per meiosis for wild type, <em>mlh3</em>Δ, <em>pch2</em>Δ, and <em>mlh3</em>Δ <em>pch2</em>Δ. The box plots show minimum, first quantile, median, third quantile, and maximum count.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">The effects of chromosome size on crossover distribution were analyzed using average crossover counts per chromosome (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/TableS4.docx">Table S4</a>). The <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ mutant showed a significant decrease in crossovers on medium and large chromosomes (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FigureS2.pptx">Figure S2A</a>). Crossovers on small chromosomes also showed a decrease in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, but these were not significant except for chromosome IX. Crossover defects in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ are therefore different from <em><a href="http://www.yeastgenome.org/locus/S000001891/overview">msh4</a></em>Δ and <em><a href="http://www.yeastgenome.org/locus/S000001891/overview">msh4</a>-R676W</em>, which had significant crossover reduction on all (small, medium, and large) chromosomes (<a id="xref-ref-46-9" class="xref-bibr" href="#ref-46">Krishnaprasad <em>et al.</em> 2015</a>). Noncrossover counts in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ did not show significant differences on any of the chromosomes compared to wild type (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FigureS2.pptx">Figure S2D</a>). The noncrossover data in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a>∆</em> are different from <em><a href="http://www.yeastgenome.org/locus/S000001891/overview">msh4</a></em>Δ, where average noncrossover counts were increased on several chromosomes (<a id="xref-ref-46-10" class="xref-bibr" href="#ref-46">Krishnaprasad <em>et al.</em> 2015</a>). When the noncrossover counts are regressed against the crossover counts, we observed no correlation in wild type and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ (wild type, <em>r</em> = 0.23, <em>P</em> = 0.06; <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, <em>r</em> = 0.01, <em>P</em> = 0.95; <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1511/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Crossover and noncrossover distribution on chromosomes for wild type, mlh3&#x394;, pch2&#x394;, and mlh3&#x394; pch2&#x394;. (A, B) Bar plot showing average crossover and noncrossover counts per chromosome for wild type, mlh3&#x394;, pch2&#x394;, and mlh3&#x394; pch2&#x394;. The asterisk symbol (*) shows chromosomes with significant difference (P&#xA0;" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1906276771" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Crossover and noncrossover distribution on chromosomes for wild type, &lt;em&gt;mlh3&lt;/em&gt;&#x394;, &lt;em&gt;pch2&lt;/em&gt;&#x394;, and &lt;em&gt;mlh3&lt;/em&gt;&#x394; &lt;em&gt;pch2&lt;/em&gt;&#x394;. (A, B) Bar plot showing average crossover and noncrossover counts per chromosome for wild type, &lt;em&gt;mlh3&lt;/em&gt;&#x394;, &lt;em&gt;pch2&lt;/em&gt;&#x394;, and &lt;em&gt;mlh3&lt;/em&gt;&#x394; &lt;em&gt;pch2&lt;/em&gt;&#x394;. The asterisk symbol (*) shows chromosomes with significant difference (&lt;em&gt;P&lt;/em&gt;&#xA0;&lt;&#xA0;0.05, &lt;em&gt;t&lt;/em&gt;-test) in crossover or noncrossover counts compared to wild type. Chromosomes are arranged according to size from left to right. Error bars are &#x201C;mean &#xB1; SE.&#x201D; (C) Crossover &lt;em&gt;vs.&lt;/em&gt; noncrossover scatter plot with correlation coefficient (&lt;em&gt;r&lt;/em&gt;) and &lt;em&gt;P&lt;/em&gt; values for wild type, &lt;em&gt;mlh3&lt;/em&gt;&#x394;, &lt;em&gt;pch2&lt;/em&gt;&#x394;, and &lt;em&gt;mlh3&lt;/em&gt;&#x394; &lt;em&gt;pch2&lt;/em&gt;&#x394;. (D) Crossover density (centimorgans per kilobase) plotted as a function of chromosome length (megabase) for wild type, &lt;em&gt;mlh3&lt;/em&gt;&#x394;, &lt;em&gt;pch2&lt;/em&gt;&#x394;, and &lt;em&gt;mlh3&lt;/em&gt;&#x394; &lt;em&gt;pch2&lt;/em&gt;&#x394; along with the correlation coefficient (&lt;em&gt;r&lt;/em&gt;) and &lt;em&gt;P&lt;/em&gt; values.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1511/F2.medium.gif" width="440" height="318"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/5/1511/F2.medium.gif" width="440" height="318"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1511/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1511/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484313" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-23" class="first-child">Crossover and noncrossover distribution on chromosomes for wild type, <em>mlh3</em>Δ, <em>pch2</em>Δ, and <em>mlh3</em>Δ <em>pch2</em>Δ. (A, B) Bar plot showing average crossover and noncrossover counts per chromosome for wild type, <em>mlh3</em>Δ, <em>pch2</em>Δ, and <em>mlh3</em>Δ <em>pch2</em>Δ. The asterisk symbol (*) shows chromosomes with significant difference (<em>P</em> &lt; 0.05, <em>t</em>-test) in crossover or noncrossover counts compared to wild type. Chromosomes are arranged according to size from left to right. Error bars are “mean ± SE.” (C) Crossover <em>vs.</em> noncrossover scatter plot with correlation coefficient (<em>r</em>) and <em>P</em> values for wild type, <em>mlh3</em>Δ, <em>pch2</em>Δ, and <em>mlh3</em>Δ <em>pch2</em>Δ. (D) Crossover density (centimorgans per kilobase) plotted as a function of chromosome length (megabase) for wild type, <em>mlh3</em>Δ, <em>pch2</em>Δ, and <em>mlh3</em>Δ <em>pch2</em>Δ along with the correlation coefficient (<em>r</em>) and <em>P</em> values.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">Wild-type cells show variation in crossover density (centimorgan per kilobase) with chromosome size (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2D</a>, and see <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Pan <em>et al.</em> 2011</a>). Previous studies have shown that high crossover density on small chromosomes is mostly a consequence of higher DSB density on small chromosomes (<a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Pan <em>et al.</em> 2011</a>). We plotted crossover density <em>vs.</em> chromosome length for wild type, and observed a pattern similar to that observed for DSB density (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FigureS3.pptx">Figure S3</a>, and see <a id="xref-ref-64-3" class="xref-bibr" href="#ref-64">Pan <em>et al.</em> 2011</a>). First, the crossover density decreases with chromosome size, indicating that larger chromosomes have fewer crossovers than expected for their physical length (assuming crossover density is equal on all chromosomes). Second, if we take into account this inverse relationship between chromosome length and crossover density, we observed that the smaller chromosomes deviate a lot from the regression line, indicating that they have a much higher crossover density than expected (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FigureS3.pptx">Figure S3</a>). <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ showed a significant negative correlation coefficient (<em>r</em> = −0.76, <em>P</em> = 6.3 × 10<sup>−4</sup>), implying an inverse relation between crossover density and chromosome size (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2D</a>). The correlation coefficient for <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ was comparable to wild type (<em>r</em> = −0.84, <em>P</em> = 5.7 × 10<sup>−5</sup>). Crossover density was higher on smaller chromosomes in wild type and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, even when the analysis excluded the three smallest chromosomes (I, III, and VI), which have the highest DSB densities (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FigureS4.pptx">Figure S4</a>A, and see <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Sun <em>et al.</em> 2015</a>). Noncrossover density, as well as crossover plus noncrossover density, also showed a significant negative correlation with chromosome size in wild type and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FigureS4.pptx">Figure S4</a>, B–D). The relationship between crossover or noncrossover density and chromosome size was therefore similar between wild type and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ. These results suggest the regulation of DSB distribution based on chromosome size is intact in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ like in wild type. Further, since most crossovers in wild type and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ are Class I and Class II, respectively, these results suggest the chromosome size effect seen for crossover density variation is dependent on DSB density, and not on the recombination pathway as shown previously (<a id="xref-ref-64-4" class="xref-bibr" href="#ref-64">Pan <em>et al.</em> 2011</a>). It is important to note that using crossover and noncrossover density as a proxy for DSBs does not consider the effect of any changes in homolog bias in these mutants. Crossover and noncrossover distributions at centromere and telomere regions were comparable to wild type, but noncrossovers were elevated within 10 kb of centromere region in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FileS2.docx">File S2</a>). Chromosome-wide crossover distribution for <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ is shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FigureS5.pptx">Figure S5</a>.</p></div><div id="sec-14" class="subsection"><h3>Genome-wide increase in crossovers and noncrossovers in pch2<strong>Δ</strong></h3><p id="p-25">Average crossovers in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ (137) were significantly more than wild type (<em>P</em> = 1.15 × 10<sup>−5</sup>, <em>t</em>-test) (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). The increase in crossovers was statistically significant on all the medium and large chromosomes, and on two of the smaller chromosomes (III and VI) (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FigureS2.pptx">Figure S2B</a>). This result is consistent with increased crossovers observed at specific loci on medium and large chromosomes in SK1 background (<a id="xref-ref-86-2" class="xref-bibr" href="#ref-86">Zanders and Alani 2009</a>), but different from <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Joshi <em>et al.</em> (2009)</a> who observed no increase in crossovers across nine intervals on three chromosomes. The increase in crossovers on chromosome III is different from the results observed by <a id="xref-ref-86-3" class="xref-bibr" href="#ref-86">Zanders and Alani (2009)</a> and <a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">San-Segundo and Roeder (1999)</a>. Such differences may be because of the limited number of chromosomal loci analyzed in the previous studies (<a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">San-Segundo and Roeder 1999</a>; <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Joshi <em>et al.</em> 2009</a>; <a id="xref-ref-86-4" class="xref-bibr" href="#ref-86">Zanders and Alani 2009</a>). Noncrossovers in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ (average of 86 per meiosis) were also significantly higher than wild type (46, <em>P</em> = 2.9 × 10<sup>−5</sup>, <em>t</em>-test) (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). Similar to the trend seen for crossovers, increased noncrossovers in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a>∆</em> were significant for all medium and large chromosomes (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FigureS2.pptx">Figure S2E</a>). These results suggest that specific regulation of DSB formation on medium and large chromosomes is affected in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a>∆</em>. Among the small chromosomes, Chromosome III was exceptional in showing an increase in both crossovers and noncrossovers. This might be due to an enhanced distribution of DSBs on chromosome III in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutants. An increase in DSB formation at the <em><a href="http://www.yeastgenome.org/locus/S000000535/overview">HIS4</a></em>::<em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> locus has been observed previously in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Farmer <em>et al.</em> 2012</a>). Interestingly <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Borner <em>et al.</em> (2008)</a> reported increased noncrossovers and decreased crossovers for the <em><a href="http://www.yeastgenome.org/locus/S000000535/overview">HIS4</a></em>::<em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> locus on chromosome III for <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ. When the noncrossover counts are regressed against the crossover counts, we observed a well correlated linear relationship (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2C</a>). <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ cells with more crossovers also had more noncrossovers (<em>r</em> = 0.82, <em>P</em> = 1.9 × 10<sup>−4</sup>). Although <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ showed a negative correlation coefficient (<em>r</em> = −0.30) between crossover density and chromosome size, it was not significant (<em>P</em> = 0.26), consistent with the enhanced crossovers on large and medium chromosomes (<a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2D</a>). Unlike wild type, <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ did not show a significant negative correlation between chromosome size and crossover density, noncrossover density, as well as crossover plus noncrossover density (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FigureS4.pptx">Figure S4</a>).</p><p id="p-26">Crossover and noncrossover distributions within 10 kb of the centromere were comparable between wild type and <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FileS2.docx">File S2</a>). But crossover and noncrossover distributions in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ were suppressed for an extended distance from the telomere (up to 80 kb) compared to the wild type (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FileS2.docx">File S2</a>). These differences may reflect changes in DSB distribution near telomeres. Chromosome-wide analysis of crossover distribution showed distinct crossover peaks compared to wild type, including elevated crossovers around the rDNA locus (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FigureS6.pptx">Figure S6</a>). This is consistent with the observation of increased DSBs and recombination at the outermost rDNA repeats in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ and the role of <a href="http://www.yeastgenome.org/locus/S000000390/overview">Pch2</a> in maintaining the integrity of the rDNA cluster (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Vader <em>et al.</em> 2011</a>).</p><p id="p-27">Genome-wide recombination analysis showed a correlated increase in both crossovers and noncrossovers in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2C</a>). The simplest explanation is that <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ makes more DSBs. <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> focus analysis can provide information on whether <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutants show increased DSBs. We measured <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> focus counts in wild type and <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutants in three independent meiotic time courses to determine changes in cellular DSB frequency. <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutants showed a 2 hr delay in meiotic divisions, as observed previously (<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Wu and Burgess 2006</a>; <a id="xref-ref-86-5" class="xref-bibr" href="#ref-86">Zanders and Alani 2009</a>) (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). We analyzed <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> foci in 1 hr increments from 2 to 7 hr time points in wild type, and from 2 to 8 hr time points in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ. For wild type and <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, maximum <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> foci were detected at 3 hr, and representative images are shown in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>. The maximum (average) <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> focus counts for <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ (37) were reduced compared to wild type (49). However, unlike wild type, where the <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> foci peaked at 3 hr and could not be detected after 7 hr, <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ showed a gradual decline of <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> foci over an extended time from 3 hr until 8 hr (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>). In wild type, the percentage of cells that show <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> foci was negligible (3.7%) at 7 hr, while, in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, almost 50% of the cells showed <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> foci (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3D</a>). The persistent presence of DSBs may be due to ongoing formation and repair of new DSBs (turnover), which may explain the increased crossovers and noncrossovers observed in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ. In addition, such a pattern may be caused by delayed repair of DSBs, which may cause more DSBs, and, in turn, result in more crossovers and noncrossovers (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Hochwagen <em>et al.</em> 2005</a>; <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Borner <em>et al.</em> 2008</a>). The cytological observations are consistent with a prophase delay in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ for a variable amount of time, during which additional DSBs are made asynchronously, so that there is a much greater range in the total number of crossover and noncrossover events (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-fig-2-12" class="xref-fig" href="#F2">Figure 2C</a>, and <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). Persistent DSBs in the <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a>∆</em> mutant have been observed previously at the <em><a href="http://www.yeastgenome.org/locus/S000000535/overview">HIS4</a></em>::<em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> locus by Southern blot analysis, and also from analysis of <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> foci in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000001166/overview">ndt80</a></em>Δ mutants (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Hochwagen <em>et al.</em> 2005</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Subramanian <em>et al.</em> 2016</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1511/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Rad51 foci analysis in wild type and pch2&#x394; mutants. (A) Meiotic time course assay for wild type and pch2&#x394;. Data are averaged across three independent meiotic time courses. (B) Representative image of Rad51 foci in wild type and pch2&#x394; at the 3&#xA0;hr time point. (C) Average Rad51 foci counts with SE from three independent experiments in wild type and pch2&#x394;. Rad51 foci from 100 images were counted for each time point and each experiment. (D) Percentage of cells in a field that are positive for Rad51 foci at each time point. Cells with &gt;5 Rad51 foci are counted as positive for Rad51 foci. Data are averaged across three independent experiments and plotted with SE measurements." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1906276771" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Rad51 foci analysis in wild type and &lt;em&gt;pch2&lt;/em&gt;&#x394; mutants. (A) Meiotic time course assay for wild type and &lt;em&gt;pch2&lt;/em&gt;&#x394;. Data are averaged across three independent meiotic time courses. (B) Representative image of Rad51 foci in wild type and &lt;em&gt;pch2&lt;/em&gt;&#x394; at the 3&#xA0;hr time point. (C) Average Rad51 foci counts with SE from three independent experiments in wild type and &lt;em&gt;pch2&lt;/em&gt;&#x394;. Rad51 foci from 100 images were counted for each time point and each experiment. (D) Percentage of cells in a field that are positive for Rad51 foci at each time point. Cells with &gt;5 Rad51 foci are counted as positive for Rad51 foci. Data are averaged across three independent experiments and plotted with SE measurements.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1511/F3.medium.gif" width="440" height="286"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/5/1511/F3.medium.gif" width="440" height="286"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1511/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1511/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484292" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-28" class="first-child">Rad51 foci analysis in wild type and <em>pch2</em>Δ mutants. (A) Meiotic time course assay for wild type and <em>pch2</em>Δ. Data are averaged across three independent meiotic time courses. (B) Representative image of Rad51 foci in wild type and <em>pch2</em>Δ at the 3 hr time point. (C) Average Rad51 foci counts with SE from three independent experiments in wild type and <em>pch2</em>Δ. Rad51 foci from 100 images were counted for each time point and each experiment. (D) Percentage of cells in a field that are positive for Rad51 foci at each time point. Cells with &gt;5 Rad51 foci are counted as positive for Rad51 foci. Data are averaged across three independent experiments and plotted with SE measurements.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>The mlh3<strong>Δ</strong> pch2<strong>Δ</strong> mutant has wild-type crossover frequency and uniform crossover density on all chromosomes</h3><p id="p-29">Since <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ shows increased crossovers in wild-type background as well as in crossover mutants like <em><a href="http://www.yeastgenome.org/locus/S000002313/overview">msh5</a></em>Δ and <em><a href="http://www.yeastgenome.org/locus/S000000302/overview">mms4</a></em>Δ (<a id="xref-ref-86-6" class="xref-bibr" href="#ref-86">Zanders and Alani 2009</a>), we were curious to see if a <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutation in the <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ background would restore crossovers to wild-type frequencies. <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ made, on average, 100 crossovers per meiosis, comparable to the wild-type S288c × YJM789 hybrid (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>) (<em>P</em> = 0.22, <em>t</em>-test). None of the chromosomes (except chromosomes I and XV) in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ showed significant differences in crossovers compared to wild type (<a id="xref-fig-2-13" class="xref-fig" href="#F2">Figure 2A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FigureS2.pptx">Figure S2C</a>). Noncrossovers in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ (average of 94 per meiosis) were significantly more than wild type (<em>P</em> = 1.84 × 10<sup>−7</sup>, <em>t</em>-test) and comparable to <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ (<em>P</em> = 0.40, <em>t</em>-test). These results suggest most of the increase in noncrossovers in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutants comes from the activity of the structure-specific endonucleases, which are less biased toward resolution of the dHJs into crossovers. The increase in noncrossovers was observed across small (VI, III, and IX), and all medium and large chromosomes (except V) (<a id="xref-fig-2-14" class="xref-fig" href="#F2">Figure 2B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FigureS2.pptx">Figure S2F</a>). Similar to the pattern observed in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, crossover, and noncrossover counts in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ were positively correlated (<em>r</em> = 0.83, <em>P</em> = 5 × 10<sup>−6</sup>) (<a id="xref-fig-2-15" class="xref-fig" href="#F2">Figure 2C</a>).</p><p id="p-30">The crossover density in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ was uniform across all chromosome sizes and the correlation coefficient (<em>r</em> = 0.04, <em>P</em> = 0.88) was close to zero (<a id="xref-fig-2-16" class="xref-fig" href="#F2">Figure 2D</a>). A similar lack of correlation was seen for crossover, noncrossover, and crossover plus noncrossover density with chromosome size (including/excluding the smallest chromosomes I, III, and VI) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FigureS4.pptx">Figure S4</a>). Like <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, crossover and noncrossover distributions were suppressed for an extended distance from the telomere (up to 80 kb) in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FileS2.docx">File S2</a>). Chromosome-wide distribution showed distinct crossover peaks in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ compared to wild type, especially around the rDNA locus (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FigureS7.pptx">Figure S7</a>).</p></div><div id="sec-16" class="subsection"><h3>Enhanced gene conversion frequencies and long gene conversion tracts in pch2<strong>Δ</strong> mutants</h3><p id="p-31">Most markers (&gt;97.5%) showed 2:2 segregation in wild type and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ mutant (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/TableS5.docx">Table S5</a>). <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutants showed higher gene conversion frequencies than wild type or <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ mutant. The percentage of SNP markers segregating 2:2 was 94.7% in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a>∆</em>, compared to &gt;97.5% in wild type and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ. Like the <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ single mutant, the <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutant also showed higher gene conversion frequencies than wild type (94% of markers segregating 2:2). Increases in gene conversion frequencies in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ at specific loci have been previously observed by <a id="xref-ref-86-7" class="xref-bibr" href="#ref-86">Zanders and Alani (2009)</a> and <a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Joshi <em>et al.</em> (2009)</a>. The median gene conversion tract lengths associated with crossovers (2.4, 2.7, and 3.9 kb) were significantly longer than the noncrossover gene conversion tract lengths (1.7, 1.8, and 2.2 kb) for the <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutants (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FigureS8.pptx">Figure S8</a>, A–C and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/TableS5.docx">Table S5</a>). These results are similar to observations in wild type and other crossover mutants (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Terasawa <em>et al.</em> 2007</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Chen <em>et al.</em> 2008</a>; <a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Mancera <em>et al.</em> 2008</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Oke <em>et al.</em> 2014</a>; <a id="xref-ref-46-11" class="xref-bibr" href="#ref-46">Krishnaprasad <em>et al.</em> 2015</a>). Both crossover- and noncrossover-associated gene conversion tract lengths were significantly longer in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ compared to wild type (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FigureS8.pptx">Figure S8</a>, A–C and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/TableS5.docx">Table S5</a>). In addition, long gene conversion tracts associated with crossovers (&gt;4 kb) and noncrossovers (&gt;2 kb) were over-represented in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ compared to wild type and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FigureS8.pptx">Figure S8</a>, B and C, see <em>Discussion</em>).</p></div><div id="sec-17" class="subsection"><h3>Most crossovers in mlh3<strong>Δ</strong> pch2<strong>Δ</strong> are made by the Mus81-Mms4 pathway</h3><p id="p-32">Although <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ has ∼30% reduction in crossovers, <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ has wild-type crossover frequency. In <em>S. cerevisiae</em>, the major pathway for resolving joint molecules into crossovers involves <a href="http://www.yeastgenome.org/locus/S000004777/overview">Mlh1</a>-<a href="http://www.yeastgenome.org/locus/S000006085/overview">Mlh3</a>/<a href="http://www.yeastgenome.org/locus/S000005559/overview">Exo1</a> proteins (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">De Muyt <em>et al.</em> 2012</a>; <a id="xref-ref-85-5" class="xref-bibr" href="#ref-85">Zakharyevich <em>et al.</em> 2012</a>). Joint molecules are also resolved into crossovers by minor pathways involving the structure selective nucleases <a href="http://www.yeastgenome.org/locus/S000002794/overview">Mus81</a>-<a href="http://www.yeastgenome.org/locus/S000000302/overview">Mms4</a>, <a href="http://www.yeastgenome.org/locus/S000000843/overview">Yen1</a>, and <a href="http://www.yeastgenome.org/locus/S000000432/overview">Slx1</a>-<a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> (<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">De Muyt <em>et al.</em> 2012</a>; <a id="xref-ref-85-6" class="xref-bibr" href="#ref-85">Zakharyevich <em>et al.</em> 2012</a>). <a href="http://www.yeastgenome.org/locus/S000000843/overview">Yen1</a> is thought to process joint molecules that escape <a href="http://www.yeastgenome.org/locus/S000002794/overview">Mus81</a>-<a href="http://www.yeastgenome.org/locus/S000000302/overview">Mms4</a> (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Matos <em>et al.</em> 2011</a>). We were curious to know how wild-type frequency of crossovers is possible in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ in the absence of the major resolvase (<a href="http://www.yeastgenome.org/locus/S000006085/overview">Mlh3</a>). We tested the role of the two joint molecule resolvases, <a href="http://www.yeastgenome.org/locus/S000002794/overview">Mus81</a>-<a href="http://www.yeastgenome.org/locus/S000000302/overview">Mms4</a> and <a href="http://www.yeastgenome.org/locus/S000000432/overview">Slx1</a>-<a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a>, in making crossovers in the <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ background. For testing the role of <a href="http://www.yeastgenome.org/locus/S000002794/overview">Mus81</a>-<a href="http://www.yeastgenome.org/locus/S000000302/overview">Mms4</a>, a meiotic depleted allele of <em><a href="http://www.yeastgenome.org/locus/S000000302/overview">MMS4</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000000302/overview">mms4</a>-md</em>) was used (<a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">De Muyt <em>et al.</em> 2012</a>). Spore viability and genetic map distances of <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000302/overview">mms4</a>-md</em>, and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>Δ, were compared with <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ in the EAY1108/EAY1112 background (<a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Argueso <em>et al.</em> 2004</a>). <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ has 71% spore viability in the EAY1108/EAY1112 background and has a spore viability pattern (4, 2, and 0 viable spore tetrad class &gt;3 and 1 viable spore tetrad class) suggesting Meiosis I nondisjunction (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Introduction of <em><a href="http://www.yeastgenome.org/locus/S000000302/overview">mms4</a>-md</em> and <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>Δ mutations reduced the spore viability to 34% and 41%, respectively (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Genetic map distances in four intervals in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ (113.6 cM), based on four-viable spore tetrads was similar to wild type (105.4 cM), in agreement with the whole genome sequencing data from the S288c/YJM789 hybrid (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). However, map distances when all viable spores were considered showed a reduction in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ (79.9 cM) relative to wild type (96.6 cM) (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). These results suggest restoration of crossovers in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ to wild-type frequency occurs through an increase in both single crossovers measured as tetratypes (TT), as well as double crossovers measured as nonparental ditypes (NPDs). The increase in double crossovers (NPDs) contributes to map distances from tetrad data but not from spore data (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/TableS6.docx">Table S6</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/TableS7.docx">Table S7</a>). More NPD tetrads are also observed in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutants relative to wild type (<a id="xref-ref-39-5" class="xref-bibr" href="#ref-39">Joshi <em>et al.</em> 2009</a>; <a id="xref-ref-86-8" class="xref-bibr" href="#ref-86">Zanders and Alani 2009</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1511/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Spore viability analysis of wild type, mlh3&#x394; pch2&#x394;, mlh3&#x394; pch2&#x394; slx4&#x394;, and mlh3&#x394; pch2&#x394; mms4-md in the EAY1108/EAY1112 genetic background. The viable spores per tetrad (x axis) and the percentage of each tetrad class (y axis) are shown. N, number of tetrads dissected; S.V, percentage spore viability." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1906276771" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Spore viability analysis of wild type, &lt;em&gt;mlh3&lt;/em&gt;&#x394; &lt;em&gt;pch2&lt;/em&gt;&#x394;, &lt;em&gt;mlh3&lt;/em&gt;&#x394; &lt;em&gt;pch2&lt;/em&gt;&#x394; &lt;em&gt;slx4&lt;/em&gt;&#x394;, and &lt;em&gt;mlh3&lt;/em&gt;&#x394; &lt;em&gt;pch2&lt;/em&gt;&#x394; &lt;em&gt;mms4-md&lt;/em&gt; in the EAY1108/EAY1112 genetic background. The viable spores per tetrad (&lt;em&gt;x&lt;/em&gt; axis) and the percentage of each tetrad class (&lt;em&gt;y&lt;/em&gt; axis) are shown. &lt;em&gt;N&lt;/em&gt;, number of tetrads dissected; S.V, percentage spore viability.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1511/F4.medium.gif" width="440" height="336"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/5/1511/F4.medium.gif" width="440" height="336"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1511/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1511/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484333" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-33" class="first-child">Spore viability analysis of wild type, <em>mlh3</em>Δ <em>pch2</em>Δ, <em>mlh3</em>Δ <em>pch2</em>Δ <em>slx4</em>Δ, and <em>mlh3</em>Δ <em>pch2</em>Δ <em>mms4-md</em> in the EAY1108/EAY1112 genetic background. The viable spores per tetrad (<em>x</em> axis) and the percentage of each tetrad class (<em>y</em> axis) are shown. <em>N</em>, number of tetrads dissected; S.V, percentage spore viability.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1511/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Crossovers in mlh3&#x394; pch2&#x394; are made predominantly by the Mus81-Mms4 pathway. Genetic map distances for complete tetrads (A) and total spores (B) are shown for a set of four intervals on chromosome XV in the EAY1108/EAY1112 genetic background. Raw data are shown in Table S6 and Table S7 for complete tetrads and total spores, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1906276771" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Crossovers in &lt;em&gt;mlh3&lt;/em&gt;&#x394; &lt;em&gt;pch2&lt;/em&gt;&#x394; are made predominantly by the Mus81-Mms4 pathway. Genetic map distances for complete tetrads (A) and total spores (B) are shown for a set of four intervals on chromosome XV in the EAY1108/EAY1112 genetic background. Raw data are shown in Table S6 and Table S7 for complete tetrads and total spores, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1511/F5.medium.gif" width="357" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/5/1511/F5.medium.gif" width="357" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1511/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1511/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484375" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-34" class="first-child">Crossovers in <em>mlh3</em>Δ <em>pch2</em>Δ are made predominantly by the Mus81-Mms4 pathway. Genetic map distances for complete tetrads (A) and total spores (B) are shown for a set of four intervals on chromosome XV in the EAY1108/EAY1112 genetic background. Raw data are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/TableS6.docx">Table S6</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/TableS7.docx">Table S7</a> for complete tetrads and total spores, respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35"><em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>Δ (91.2 cM) showed reduced genetic map distances compared to <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ (113.6 cM) in measures based on complete tetrads, but the genetic map distances from spore data were similar: 77.8 cM in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>Δ compared to 79.9 cM in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5, A and B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/TableS6.docx">Table S6</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/TableS7.docx">Table S7</a>). These results suggest <a href="http://www.yeastgenome.org/locus/S000000432/overview">Slx1</a>-<a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> does not significantly contribute to increased crossovers in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ except for contributing to the NPD class. This is consistent with the meiotic role of the <a href="http://www.yeastgenome.org/locus/S000000432/overview">Slx1</a>-<a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> complex in the non-ZMM pathway characterized by multi-chromatid joint molecules (<a id="xref-ref-85-7" class="xref-bibr" href="#ref-85">Zakharyevich <em>et al.</em> 2012</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Kaur <em>et al.</em> 2015</a>). <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000302/overview">mms4</a>-md</em> mutants showed strong reduction in genetic map distances (23.7 cM) compared to <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ (79.9 cM) (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/TableS7.docx">Table S7</a>). Map distances from tetrad data could not be accurately estimated in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000302/overview">mms4</a>-md</em> due to the low frequency of four-spore viable tetrads and the strong reduction in the map distances. These results suggest that the majority of the crossovers in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ are made through the <a href="http://www.yeastgenome.org/locus/S000002794/overview">Mus81</a>-<a href="http://www.yeastgenome.org/locus/S000000302/overview">Mms4</a> pathway.</p></div><div id="sec-18" class="subsection"><h3>Genetic interference is reduced across a range of crossover frequencies in the mlh3<strong>Δ</strong>, pch2<strong>Δ</strong>, and mlh3<strong>Δ</strong> pch2<strong>Δ</strong> mutants</h3><p id="p-36">Interference limits crossover number and ensures that multiple crossovers along the chromosome are widely spaced (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Muller 1916</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hillers 2004</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Kleckner <em>et al.</em> 2004</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Stahl <em>et al.</em> 2004</a>). Intercrossover distances modeled using a gamma distribution can be used to determine the strength of genetic interference: <em>γ</em> = 1 corresponds to no interference, while <em>γ</em> &gt; 1 indicates positive interference (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Anderson <em>et al.</em> 2011</a>). Intercrossover distances in physical units were converted into genetic distances (centimorgans) in wild type, <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, to account for differences in crossover frequencies (<em>Materials and Methods</em>) (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). To account for differences in the number of tetrads analyzed, we also converted the counts for intercrossover distances as a fraction of the total for wild type and each mutant (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>). For the wild-type strain, the <em>γ</em> value was 1.83. Interference was reduced in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ (<em>γ</em> = 1.33), <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ (<em>γ</em> = 1.29) and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ (<em>γ</em> = 1.13) compared to wild type. Interference analysis using the two pathway model that incorporates contribution from both interfering and noninterfering pathways also showed reduced interference in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FileS3.docx">File S3</a>, and see <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Copenhaver <em>et al.</em> 2002</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Housworth and Stahl 2003</a>). We also analyzed interference from the genome wide crossover data using the CoC method (<em>Materials and Methods</em>, and see <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Anderson <em>et al.</em> 2015</a>). Statistical significance of the difference between observed and expected frequency of double crossovers was estimated using the chi-square test. Interference (1 – CoC) was significantly reduced in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ (<em>P</em> = 0.00033), <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ (<em>P</em> = 4.36 × 10<sup>−6</sup>), and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ (<em>P</em> = 1.85 × 10<sup>−9</sup>) relative to the wild type (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6C</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1511/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of crossover interference in wild type, mlh3&#x394;, pch2&#x394;, and mlh3&#x394; pch2&#x394;. (A) Histogram showing actual count of intercrossover distances. (B) Line plot showing intercrossover distances as a fraction of the total for wild type, mlh3&#x394;, pch2&#x394;, and mlh3&#x394; pch2&#x394;. (C) Interference (1&#x2013;CoC) in wild type (0.56), mlh3&#x394; (0.31), pch2&#x394; (0.30), and mlh3&#x394; pch2&#x394; (0.17) plotted for 0&#x2013;25&#xA0;kb adjacent intervals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1906276771" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Analysis of crossover interference in wild type, &lt;em&gt;mlh3&lt;/em&gt;&#x394;, &lt;em&gt;pch2&lt;/em&gt;&#x394;, and &lt;em&gt;mlh3&lt;/em&gt;&#x394; &lt;em&gt;pch2&lt;/em&gt;&#x394;. (A) Histogram showing actual count of intercrossover distances. (B) Line plot showing intercrossover distances as a fraction of the total for wild type, &lt;em&gt;mlh3&lt;/em&gt;&#x394;, &lt;em&gt;pch2&lt;/em&gt;&#x394;, and &lt;em&gt;mlh3&lt;/em&gt;&#x394; &lt;em&gt;pch2&lt;/em&gt;&#x394;. (C) Interference (1&#x2013;CoC) in wild type (0.56), &lt;em&gt;mlh3&lt;/em&gt;&#x394; (0.31), &lt;em&gt;pch2&lt;/em&gt;&#x394; (0.30), and &lt;em&gt;mlh3&lt;/em&gt;&#x394; &lt;em&gt;pch2&lt;/em&gt;&#x394; (0.17) plotted for 0&#x2013;25&#xA0;kb adjacent intervals.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1511/F6.medium.gif" width="440" height="159"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/5/1511/F6.medium.gif" width="440" height="159"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1511/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1511/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484280" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-37" class="first-child">Analysis of crossover interference in wild type, <em>mlh3</em>Δ, <em>pch2</em>Δ, and <em>mlh3</em>Δ <em>pch2</em>Δ. (A) Histogram showing actual count of intercrossover distances. (B) Line plot showing intercrossover distances as a fraction of the total for wild type, <em>mlh3</em>Δ, <em>pch2</em>Δ, and <em>mlh3</em>Δ <em>pch2</em>Δ. (C) Interference (1–CoC) in wild type (0.56), <em>mlh3</em>Δ (0.31), <em>pch2</em>Δ (0.30), and <em>mlh3</em>Δ <em>pch2</em>Δ (0.17) plotted for 0–25 kb adjacent intervals.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">The reduced interference in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ is consistent with the two pathway model for crossover formation (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Copenhaver <em>et al.</em> 2002</a>; <a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Stahl <em>et al.</em> 2004</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Getz <em>et al.</em> 2008</a>). <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ has been shown previously to lack genetic interference (<a id="xref-ref-39-6" class="xref-bibr" href="#ref-39">Joshi <em>et al.</em> 2009</a>; <a id="xref-ref-86-9" class="xref-bibr" href="#ref-86">Zanders and Alani 2009</a>). However, analysis of <a href="http://www.yeastgenome.org/locus/S000004386/overview">Zip3</a> foci that mark DSB sites that will be repaired as crossovers, suggests that interference is observed in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ at the crossover designation stage (<a id="xref-ref-88-3" class="xref-bibr" href="#ref-88">Zhang <em>et al.</em> 2014</a>). The loss of genetic interference in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ was thought to be due to the misregulation of the crossover/noncrossover decision step, and not because of the excess crossovers (<a id="xref-ref-86-10" class="xref-bibr" href="#ref-86">Zanders and Alani 2009</a>). We observe an increase in both crossovers and noncrossovers in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutants and <a href="http://www.yeastgenome.org/locus/S000004386/overview">Zip3</a> localization is known to be maintained in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>, <a id="xref-ref-88-4" class="xref-bibr" href="#ref-88">Zhang <em>et al.</em> 2014</a>). Therefore, the reduced interference in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ is most likely due to the excess crossovers made by the MutLγ-dependent interfering, and the <a href="http://www.yeastgenome.org/locus/S000002794/overview">Mus81</a>-<a href="http://www.yeastgenome.org/locus/S000000302/overview">Mms4</a>-dependent noninterfering crossover pathways.</p><p id="p-39">Since <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ also makes more noncrossovers, it is unlikely that the loss of interference is due to impairment of the crossover/noncrossover decision. The loss of interference is most likely because nearly all crossovers in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ are made through the <a href="http://www.yeastgenome.org/locus/S000002794/overview">Mus81</a>-<a href="http://www.yeastgenome.org/locus/S000000302/overview">Mms4</a> pathway (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5B</a>). Loss of interference in the <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutant is also supported from the uniform crossover density observed across all chromosome sizes (<a id="xref-fig-2-17" class="xref-fig" href="#F2">Figure 2D</a>). The loss of genetic interference in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ is particularly interesting as it makes as many crossovers on average as wild type. In summary, genetic interference is reduced across all three categories of crossover frequencies (reduced, equivalent, and greater than wild-type crossovers for <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, and <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, respectively). No chromatid interference was observed among wild type, <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, and <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ <em>mutants</em> (<em>P</em> &gt; 0.05, chi-square test), but negative chromatid interference characterized by a slight excess of two strand crossovers may be present in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutants (0.01 &lt; <em>P</em> &lt; 0.05, chi-square test) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/TableS8.docx">Table S8</a>).</p></div><div id="sec-19" class="subsection"><h3>Noninterfering crossovers are less efficient in promoting crossover assurance</h3><p id="p-40">We tested the effect of a wide range of crossover frequencies on obligate crossovers using data from genome-wide analysis of meiotic recombination in the <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutants. We determined the frequency of meiosis with nonexchange events in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutants. The percentage of meiosis with 1 or &gt;1 nonexchange chromosome was 3% for wild type, 7% for <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, 47% for <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, and 20% for <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/TableS3.xlsx">Table S3</a>). <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, which showed a decrease in both genetic interference and crossover frequencies, had the highest number of meioses with nonexchange events. In 5% of the meioses, the <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ mutant also had &gt;1 nonexchange chromosome. However, <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ had an overall lower amount of nonexchange events among four-spore viable tetrads (47%) compared to <em><a href="http://www.yeastgenome.org/locus/S000001891/overview">msh4</a></em>Δ (72%) but similar to <em><a href="http://www.yeastgenome.org/locus/S000001891/overview">msh4</a>-R676W</em> (42%, <a id="xref-ref-46-12" class="xref-bibr" href="#ref-46">Krishnaprasad <em>et al.</em> 2015</a>). These results suggest that, even though <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ makes 64 crossovers, the obligate crossover is not maintained. These findings reinforce our earlier observations that simultaneous reductions in crossover frequencies and genetic interference compromise the obligate crossover (<a id="xref-ref-46-13" class="xref-bibr" href="#ref-46">Krishnaprasad <em>et al.</em> 2015</a>). In <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, the number of meioses with nonexchange events (7%) was comparable to wild type (3%) (<em>P</em> = 0.37, Binomial test). This is likely due to the excess crossovers made in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ that compensate for the loss of genetic interference. Previously, we had estimated that, if the crossovers are distributed randomly, a wild-type yeast cell would require up to ∼200 crossovers to ensure (with probability &gt;0.98) the absence of nonexchange chromosomes (<a id="xref-ref-46-14" class="xref-bibr" href="#ref-46">Krishnaprasad <em>et al.</em> 2015</a>). The observations with the <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutant support the theoretical predictions that genetic interference reduces the number of crossovers required for crossover assurance. The results also suggest that <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ does not significantly increase nonexchange chromosome frequency through other mechanisms. The <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutant was most interesting, since, although it made as many crossovers as wild type, the percentage of meiosis with nonexchange events (20%) was significantly more than wild type (3%) (<em>P</em> = 2.7 × 10<sup>−3</sup>, Binomial test). These data experimentally demonstrate that if the wild-type frequency of crossovers were randomly distributed in <em>S. cerevisiae</em>, there would be more nonexchange chromosomes.</p><p id="p-41">To further test how crossover frequency affects obligate crossovers, we estimated the percentage of meiosis with no nonexchange chromosomes expected for <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ by modeling the crossovers as a Poisson distribution (the Poisson distribution assumes the crossover events are independent) (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). The expected percentage of meiosis with no nonexchange chromosomes, 46% (<em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ), 80% (<em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ), 93% (<em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ) matched well with the experimental observations: 53% (<em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ), 80% (<em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ) and 93% (<em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ). Crossover distributions in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, therefore, approximate a Poisson process. For wild type, the percentage of meiosis with nonexchange chromosomes (3%) is significantly lower than expected (24%) due to the presence of interference. These observations are consistent with the requirement for higher crossover frequencies in the absence of genetic interference to ensure obligate crossovers (<a id="xref-ref-46-15" class="xref-bibr" href="#ref-46">Krishnaprasad <em>et al.</em> 2015</a>; <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Berchowitz and Copenhaver 2010</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Kaback <em>et al.</em> 1999</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1511/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Probability of observing no E0 (chromosomes with zero crossovers per meiosis) for wild type, mlh3&#x394;, pch2&#x394;, and mlh3&#x394; pch2&#x394;. The probability of observing no E0 events is plotted against the crossover frequency modeled as a Poisson distribution. The dotted lines indicate the probabilities for finding no E0s in the absence of genetic interference for the experimentally observed average crossover counts per cell (mlh3&#x394;: 64; wild type: 93; mlh3&#x394; pch2&#x394;: 100; pch2&#x394;: 137)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1906276771" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Probability of observing no E0 (chromosomes with zero crossovers per meiosis) for wild type, &lt;em&gt;mlh3&lt;/em&gt;&#x394;, &lt;em&gt;pch2&lt;/em&gt;&#x394;, and &lt;em&gt;mlh3&lt;/em&gt;&#x394; &lt;em&gt;pch2&lt;/em&gt;&#x394;. The probability of observing no E0 events is plotted against the crossover frequency modeled as a Poisson distribution. The dotted lines indicate the probabilities for finding no E0s in the absence of genetic interference for the experimentally observed average crossover counts per cell (&lt;em&gt;mlh3&lt;/em&gt;&#x394;: 64; wild type: 93; &lt;em&gt;mlh3&lt;/em&gt;&#x394; &lt;em&gt;pch2&lt;/em&gt;&#x394;: 100; &lt;em&gt;pch2&lt;/em&gt;&#x394;: 137).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1511/F7.medium.gif" width="440" height="410"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/5/1511/F7.medium.gif" width="440" height="410"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1511/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1511/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484317" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-42" class="first-child">Probability of observing no E0 (chromosomes with zero crossovers per meiosis) for wild type, <em>mlh3</em>Δ, <em>pch2</em>Δ, and <em>mlh3</em>Δ <em>pch2</em>Δ. The probability of observing no E0 events is plotted against the crossover frequency modeled as a Poisson distribution. The dotted lines indicate the probabilities for finding no E0s in the absence of genetic interference for the experimentally observed average crossover counts per cell (<em>mlh3</em>Δ: 64; wild type: 93; <em>mlh3</em>Δ <em>pch2</em>Δ: 100; <em>pch2</em>Δ: 137).</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">In mutants with reduced genetic interference, adjacent crossovers may be closely spaced. As a result, they could be annotated as double noncrossovers inflating the number of nonexchange chromosomes. We inspected each nonexchange chromosome in the wild type, <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutants for ambiguity in the annotation for double crossovers and double noncrossovers, and did not find any significant difference in the estimate of nonexchange chromosomes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/TableS9.xlsx">Table S9</a>). All nonexchange chromosomes, except one, had gene conversion events (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/TableS9.xlsx">Table S9</a>). This observation suggests that lack of an obligate crossover was not a consequence of the absence of recombination interactions between the homologs, but rather a failure to convert even one of them into a crossover.</p></div><div id="sec-20" class="subsection"><h3>Small chromosomes are most sensitive to loss of the obligate crossover</h3><p id="p-44">We analyzed the influence of chromosome size on the occurrence of nonexchange chromosomes in the wild type, <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutants (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>). For <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, nonexchange chromosomes included small (I, VI, and IX), medium (XI), and large chromosomes (XVI). This observation is similar to the distribution of nonexchange chromosomes in <em><a href="http://www.yeastgenome.org/locus/S000001891/overview">msh4</a></em>Δ observed previously, except that the number of nonexchange chromosomes is much less in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ (<a id="xref-ref-46-16" class="xref-bibr" href="#ref-46">Krishnaprasad <em>et al.</em> 2015</a>). For <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, the only nonexchange chromosome was a small chromosome (IX). For <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, only small chromosomes (I and VI) were nonexchange. These observations suggest that small chromosomes are particularly susceptible to the loss of the obligate crossover when there are crossover and/or interference defects. A stronger reduction in crossover frequencies, as observed in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ mutants, is required for the loss of obligate crossovers on medium and large chromosomes. These observations further consolidate our previous results showing that small chromosomes are particularly sensitive to variations in crossover frequency (<a id="xref-ref-46-17" class="xref-bibr" href="#ref-46">Krishnaprasad <em>et al.</em> 2015</a>). A high frequency of nonexchange has been observed for smaller chromosomes in other crossover defective mutants, and also in human meiosis (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Chen <em>et al.</em> 2008</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Fledel-Alon <em>et al.</em> 2009</a>). Among the small chromosomes, we observed a lack of E0s for the chromosome III. A similar phenomenon was seen in <a id="xref-ref-46-18" class="xref-bibr" href="#ref-46">Krishnaprasad <em>et al.</em> (2015)</a>, where chromosome III had the fewest E0s among the small chromosomes. Analysis of copy number variation using read depth information did not detect aneuploidy in any of the <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ sequenced tetrads (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FigureS9.pdf">Figure S9</a>).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1511/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The percentage of meiosis that are nonexchange for each chromosome for wild type, mlh3&#x394;, pch2&#x394;, and mlh3&#x394; pch2&#x394;. Chromosomes are shown in increasing order of size." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1906276771" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The percentage of meiosis that are nonexchange for each chromosome for wild type, &lt;em&gt;mlh3&lt;/em&gt;&#x394;, &lt;em&gt;pch2&lt;/em&gt;&#x394;, and &lt;em&gt;mlh3&lt;/em&gt;&#x394; &lt;em&gt;pch2&lt;/em&gt;&#x394;. Chromosomes are shown in increasing order of size.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1511/F8.medium.gif" width="440" height="280"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/7/5/1511/F8.medium.gif" width="440" height="280"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1511/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1511/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484346" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-45" class="first-child">The percentage of meiosis that are nonexchange for each chromosome for wild type, <em>mlh3</em>Δ, <em>pch2</em>Δ, and <em>mlh3</em>Δ <em>pch2</em>Δ. Chromosomes are shown in increasing order of size.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-21"><h2 class="">Discussion</h2><p id="p-46">We previously analyzed genome-wide crossovers in a <em><a href="http://www.yeastgenome.org/locus/S000001891/overview">msh4</a></em> hypomorph to demonstrate that a random distribution of reduced number of crossover events can increase nonexchange chromosome frequencies (<a id="xref-ref-46-19" class="xref-bibr" href="#ref-46">Krishnaprasad <em>et al.</em> 2015</a>). Our current work generalizes this observation to a larger context by analyzing nonexchange events when there is a random distribution of reduced, wild-type, and greater than wild-type frequency of crossovers using the <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutants.</p><div id="sec-22" class="subsection"><h3>Obligate crossover is sensitive to crossover frequencies and genetic interference</h3><p id="p-47">Genetic interference is affected in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ due to increased contributions from the noninterfering pathway. Interference is also compromised in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ because of defects in converting crossover designated sites into actual crossovers. The <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ results suggest that mutants with reduced genetic interference can maintain an obligate crossover if crossover frequencies are significantly higher than in wild type (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>). If the crossover frequency is reduced compared to wild type, there is a loss of the obligate crossover, as in the case of <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ. In <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, we observed crossover frequencies equivalent to wild type (100 crossovers per meiosis), but an increased proportion of nonexchange chromosomes. Three lines of evidence support enhanced nonexchange chromosome frequency in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ. First, we observe nonexchange chromosomes in 20% of meiosis in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ in the S288c/YJM789 hybrid (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/TableS3.xlsx">Table S3</a>). Second, theoretical predictions by modeling the <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ crossover distribution as a Poisson process also support higher nonexchange chromosome frequencies consistent with the experimental results (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7</a>). Third, <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ showed spore viability pattern typical of Meiosis I nondisjunction in the SK1 EAY1108/1112 background (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). Since most crossovers in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ are made by the <a href="http://www.yeastgenome.org/locus/S000002794/overview">Mus81</a>-<a href="http://www.yeastgenome.org/locus/S000000302/overview">Mms4</a> pathway (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5B</a>), these observations are consistent with noninterfering crossovers being less efficient in supporting an obligate crossover. The higher E0 frequency in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a>∆ <a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a>∆</em> also supports the prediction that <em>S. cerevisiae</em> will require up to 200 crossovers in the absence of genetic interference to have &gt;0.98% probability of an obligate crossover on every homolog pair (<a id="xref-ref-46-20" class="xref-bibr" href="#ref-46">Krishnaprasad <em>et al.</em> 2015</a>).</p><p id="p-48">Triple mutant analysis with another resolvase (<a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a>) showed that genetic map distances measured from spore data were not significantly different between <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ (79.9 cM) and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>Δ (77.8 cM). However, map distances from tetrad data showed significant differences between <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ (113.6 cM) and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>Δ (91.2 cM). These results suggest <a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> only makes a minor contribution to crossovers in the <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutant. <a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> may be involved in resolving a fraction of closely spaced double crossovers that contribute to the NPD class of tetrads.</p></div><div id="sec-23" class="subsection"><h3>Segregation of nonexchange chromosomes in mlh3<strong>Δ</strong></h3><p id="p-49">Since <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ and wild type have identical spore viability in the S288c/YJM789 hybrid and similar proportion of the four-viable spore tetrad class, we tested the association of spore viability with the nonexchange chromosome frequency. <a href="http://www.yeastgenome.org/locus/S000006085/overview">Mlh3</a> acts downstream of <a href="http://www.yeastgenome.org/locus/S000001891/overview">Msh4</a>/5 at the final steps of Holliday Junction resolution (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Baker <em>et al.</em> 1996</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Lipkin <em>et al.</em> 2002</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Snowden <em>et al.</em> 2004</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Kolas <em>et al.</em> 2005</a>). <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ mutants, therefore, provide an advantage in relating variations in crossover frequency with the loss of obligate crossovers without the confounding effects that may arise in other mutants that have an early role in the crossover/noncrossover decision step or that affect DSB formation. It is also important to recognize that the data for the nonexchange chromosome frequency comes from the analysis of the four viable spore tetrads only. <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ has high spore viability (85%), strong crossover defects (64 crossovers per meiosis), and abundant single nonexchange chromosomes (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8</a> and <a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>). These data are similar to the pattern seen with <em><a href="http://www.yeastgenome.org/locus/S000001891/overview">msh4</a>-R676W</em> earlier (<a id="xref-ref-46-21" class="xref-bibr" href="#ref-46">Krishnaprasad <em>et al.</em> 2015</a>). The <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ mutant reinforces the idea that a reduction in crossover frequency results in loss of the obligate crossover, even though the average crossover number is still four times the number of homologs. The high spore viability of <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, suggests these single nonexchange chromosomes are efficiently segregated as observed with <em><a href="http://www.yeastgenome.org/locus/S000001891/overview">msh4</a>-R676W</em> (<a id="xref-ref-46-22" class="xref-bibr" href="#ref-46">Krishnaprasad <em>et al.</em> 2015</a>), supporting the hypothesis that crossovers facilitate but are not essential for chromosome segregation. Other mechanisms, such as distributive segregation, spindle checkpoint, and centromere pairing may also contribute to the segregation of achiasmate chromosomes in <em>S. cerevisiae</em> (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Guacci and Kaback 1991</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Shonn <em>et al.</em> 2000</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Kemp <em>et al.</em> 2004</a>). These observations also suggest spore viability is a poor measure of the status of obligate crossovers. The <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ mutant showed 64 crossovers genome-wide compared to 49 reported in <em><a href="http://www.yeastgenome.org/locus/S000001891/overview">msh4</a></em>Δ (<a id="xref-ref-46-23" class="xref-bibr" href="#ref-46">Krishnaprasad <em>et al.</em> 2015</a>). These results provide support to previous observation on <a href="http://www.yeastgenome.org/locus/S000001891/overview">Msh4</a>/5 dependent but <a href="http://www.yeastgenome.org/locus/S000004777/overview">Mlh1</a>/3 independent crossover pathways (<a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">Argueso <em>et al.</em> 2004</a>). It is also expected if, in <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ, some of the crossover precursor joint molecules are resolved by structure selective nucleases as both crossovers and noncrossovers.</p></div><div id="sec-24" class="subsection"><h3>Recombination maps in pch2<strong>Δ</strong> capture DSB variation</h3><p id="p-50">We observed a simultaneous increase in crossovers and noncrossovers in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, which suggested that DSBs are increased in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ. Previous studies have shown a complex role for <a href="http://www.yeastgenome.org/locus/S000000390/overview">Pch2</a> in DSB formation, with increased DSB formation at certain loci, such as the rDNA locus (<a id="xref-ref-81-2" class="xref-bibr" href="#ref-81">Vader <em>et al.</em> 2011</a>), and a general reduction in DSBs at other loci (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Farmer <em>et al.</em> 2012</a>). <a href="http://www.yeastgenome.org/locus/S000000390/overview">Pch2</a> also has a role in the processing of early DSBs (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Joshi <em>et al.</em> 2015</a>). These studies suggest overall DSB levels may remain constant in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, and the effects need to be examined on a per locus/chromosome basis (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Farmer <em>et al.</em> 2012</a>). High-resolution crossover and noncrossover data in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ provide an indirect readout of variation in DSB formation. For example, <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutants do not show negative correlation of chromosome size with crossover plus noncrossover density. We observe more crossovers around the rDNA locus in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutants. <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, as well as <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, also show suppression of both crossovers and noncrossovers near telomeres, which may suggest an altered DSB distribution near telomeric regions. Also, we observed increased crossovers as well as noncrossovers on Chr III, consistent with increased DSB formation on chr III (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Farmer <em>et al.</em> 2012</a>). Such differences in DSB distribution, possibly by regulating <a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a> function (<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Borner <em>et al.</em> 2008</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Wojtasz <em>et al.</em> 2009</a>), and the use of the <a href="http://www.yeastgenome.org/locus/S000002794/overview">Mus81</a>-<a href="http://www.yeastgenome.org/locus/S000000302/overview">Mms4</a> pathway, may be responsible for the differences in crossover and noncrossover patterns in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ compared to wild type (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FigureS4.pptx">Figure S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FigureS6.pptx">Figure S6</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FigureS7.pptx">Figure S7</a>).</p><p id="p-51">Although <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> foci analyses do not give information on regional DSB variation, it can provide information on total cellular DSB levels and its turnover. We did not observe peak <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> focus counts in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ to be greater than wild type. This is consistent with the physical analysis of DSBs, which suggests that, at least at specific loci, there is no increase in DSB formation in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutants (<a id="xref-ref-87-2" class="xref-bibr" href="#ref-87">Zanders <em>et al.</em> 2011</a>; <a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">Farmer <em>et al.</em> 2012</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Joshi <em>et al.</em> 2015</a>). Instead, <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ showed the persistence of peak DSB foci over an extended period of the meiotic time course from 3 to 8 hr. It is possible there is DSB turnover in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ, so that DSBs are made and repaired and the process is repeated, leading to an overall increase in crossover and noncrossovers. Such a possibility is supported by previous observations of the accumulation of <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> foci in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutants (<a id="xref-ref-77-2" class="xref-bibr" href="#ref-77">Subramanian <em>et al.</em> 2016</a>). The only other alternative is that there is more repair of DSBs by the interhomolog pathway in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutants, which may account for increased crossovers and noncrossovers. But all available evidence suggests that in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ there is more intersister recombination than in wild type (<a id="xref-ref-87-3" class="xref-bibr" href="#ref-87">Zanders <em>et al.</em> 2011</a>; <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Joshi <em>et al.</em> 2015</a>). Defects in DSB processing (<em>e.g.</em>, longer resection) may also result in longer ssDNA filaments stabilized by <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> that may account for the long gene conversion tracts observed in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutants (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1/FigureS8.pptx">Figure S8</a>, B and C). The role of <a href="http://www.yeastgenome.org/locus/S000000390/overview">Pch2</a> in DSB regulation requires further investigation.</p><p id="p-52">In conclusion, the <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ data provides experimental support from genome-wide analysis that wild-type crossover frequencies distributed randomly cannot maintain an obligate crossover on all homolog pairs (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Kaback <em>et al.</em> 1999</a>; <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Berchowitz and Copenhaver 2010</a>; <a id="xref-ref-46-24" class="xref-bibr" href="#ref-46">Krishnaprasad <em>et al.</em> 2015</a>). The distribution of crossovers and noncrossovers suggest that chromosome-size dependent DSB formation is affected in <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a>∆</em>. <em><a href="http://www.yeastgenome.org/locus/S000000390/overview">pch2</a></em>Δ mutants also show that obligate crossovers can be ensured through a random distribution of excess crossovers. These results are consistent with MutL gamma dependent interfering crossovers being more efficient in promoting homolog disjunction compared to the <a href="http://www.yeastgenome.org/locus/S000002794/overview">Mus81</a>-<a href="http://www.yeastgenome.org/locus/S000000302/overview">Mms4</a> dependent noninterfering crossovers.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-53">We thank Eric Alani and Michael Lichten for helpful comments on the manuscript. We are also thankful to Michael Lichten and Scott Keeney for the gift of strains used in this study. This study was technically supported by the EMBL GeneCore. Part of this work was performed under the International Cooperative Research Program of Institute for Protein Research, Osaka University, ICRa-16-03. N.K.T. is supported by a Wellcome Trust-DBT India Alliance Intermediate fellowship (IA/I/11/2500268) and Indian Institute of Science Education and Research-Thiruvananthapuram (IISER-TVM) intramural funds. N.K.T. was also supported by a Department of Science &amp; Technology-Japan Society for the Promotion of Science (DST-JSPS) Exploratory Exchange visit under the India-Japan Cooperative Science Program. The study was financially supported by a European Research Council Advanced Investigator Grant (AdG-294542) to L.M.S., and A.S. was supported by JSPS (KAKENHI Grant Number 22125002 and 15H05973). P.C. and A.V.P. are supported by a University Grants Commission fellowship. A.D. is supported by a fellowship from IISER-TVM. K.G.N. is supported by a fellowship from the Council for Scientific and Industrial Research, New Delhi.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-54">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.040071/-/DC1</a>.</p></li><li class="fn" id="fn-4"><p id="p-55"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-11-16"><span class="received-label">Received </span>November 16, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-03-12"><span class="accepted-label">Accepted </span>March 12, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Chakraborty <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.1" data-doi="10.1016/S0092-8674(01)00416-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allers</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Differential timing and control of noncrossover and crossover recombination during meiosis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">57</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DAllers%26rft.auinit1%253DT.%26rft.volume%253D106%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D57%26rft.atitle%253DDifferential%2Btiming%2Band%2Bcontrol%2Bof%2Bnoncrossover%2Band%2Bcrossover%2Brecombination%2Bduring%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252801%252900416-0%26rft_id%253Dinfo%253Apmid%252F11461701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(01)00416-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11461701&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169870300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.2" data-doi="10.1371/journal.pone.0025509"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dimon</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oke</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeRisi</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">ReCombine: a suite of programs for detection and analysis of meiotic recombination in whole-genome datasets.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e25509</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DAnderson%26rft.auinit1%253DC.%2BM.%26rft.volume%253D6%26rft.issue%253D10%26rft.spage%253De25509%26rft.epage%253De25509%26rft.atitle%253DReCombine%253A%2Ba%2Bsuite%2Bof%2Bprograms%2Bfor%2Bdetection%2Band%2Banalysis%2Bof%2Bmeiotic%2Brecombination%2Bin%2Bwhole-genome%2Bdatasets.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0025509%26rft_id%253Dinfo%253Apmid%252F22046241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0025509&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22046241&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oke</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yam</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhuge</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fung</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Reduced crossover interference and increased ZMM-independent recombination in the absence of Tel1/ATM.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005478</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.4" data-doi="10.1128/MCB.23.3.873-886.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Argueso</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kijas</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heck</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waase</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Systematic mutagenesis of the <em>Saccharomyces cerevisiae</em> MLH1 gene reveals distinct roles for Mlh1p in meiotic crossing over and in vegetative and meiotic mismatch repair.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">873</span>–<span class="cit-lpage">886</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DArgueso%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D23%26rft.issue%253D3%26rft.spage%253D873%26rft.epage%253D886%26rft.atitle%253DSystematic%2BMutagenesis%2Bof%2Bthe%2BSaccharomyces%2Bcerevisiae%2BMLH1%2BGene%2BReveals%2BDistinct%2BRoles%2Bfor%2BMlh1p%2Bin%2BMeiotic%2BCrossing%2BOver%2Band%2Bin%2BVegetative%2Band%2BMeiotic%2BMismatch%2BRepair%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.23.3.873-886.2003%26rft_id%253Dinfo%253Apmid%252F12529393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjIzLzMvODczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE1MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.5" data-doi="10.1534/genetics.104.032912"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Argueso</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wanat</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gemici</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alani</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Competing crossover pathways act during meiosis in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">1805</span>–<span class="cit-lpage">1816</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DArgueso%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D168%26rft.issue%253D4%26rft.spage%253D1805%26rft.epage%253D1816%26rft.atitle%253DCompeting%2BCrossover%2BPathways%2BAct%2BDuring%2BMeiosis%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.032912%26rft_id%253Dinfo%253Apmid%252F15611158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OC80LzE4MDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTUxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.6" data-doi="10.1038/ng0796-336"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plug</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prolla</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bronner</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Involvement of mouse Mlh1 in DNA mismatch repair and meiotic crossing over.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">336</span>–<span class="cit-lpage">342</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DBaker%26rft.auinit1%253DS.%2BM.%26rft.volume%253D13%26rft.issue%253D3%26rft.spage%253D336%26rft.epage%253D342%26rft.atitle%253DInvolvement%2Bof%2Bmouse%2BMlh1%2Bin%2BDNA%2Bmismatch%2Brepair%2Band%2Bmeiotic%2Bcrossing%2Bover.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng0796-336%26rft_id%253Dinfo%253Apmid%252F8673133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng0796-336&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8673133&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UU28300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.7" data-doi="10.1371/journal.pgen.1000076"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barchi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roig</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Di Giacomo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Rooij</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keeney</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">ATM promotes the obligate XY crossover and both crossover control and chromosome axis integrity on autosomes.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000076</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBarchi%26rft.auinit1%253DM.%26rft.volume%253D4%26rft.issue%253D5%26rft.spage%253De1000076%26rft.epage%253De1000076%26rft.atitle%253DATM%2Bpromotes%2Bthe%2Bobligate%2BXY%2Bcrossover%2Band%2Bboth%2Bcrossover%2Bcontrol%2Band%2Bchromosome%2Baxis%2Bintegrity%2Bon%2Bautosomes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000076%26rft_id%253Dinfo%253Apmid%252F18497861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000076&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18497861&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.8" data-doi="10.18637/jss.v032.i06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benaglia</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chauveau</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">mixtools: an R package for analyzing finite mixture models.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D32%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v032.i06%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18637/jss.v032.i06&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.9" data-doi="10.2174/138920210790886835"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berchowitz</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Copenhaver</span>  <span class="cit-name-given-names">G. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic interference: don’t stand so close to me.</span> <abbr class="cit-jnl-abbrev">Curr. Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCurr%2BGenomics%26rft.aulast%253DBerchowitz%26rft.auinit1%253DL.%2BE.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D91%26rft.epage%253D102%26rft.atitle%253DGenetic%2Binterference%253A%2Bdon%2527t%2Bstand%2Bso%2Bclose%2Bto%2Bme.%26rft_id%253Dinfo%253Adoi%252F10.2174%252F138920210790886835%26rft_id%253Dinfo%253Apmid%252F20885817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2174/138920210790886835&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20885817&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275150100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.10" data-doi="10.1016/0092-8674(94)90038-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">D. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">RecA homologs Dmc1 and Rad51 interact to form multiple nuclear complexes prior to meiotic chromosome synapsis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">1081</span>–<span class="cit-lpage">1092</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBishop%26rft.auinit1%253DD.%2BK.%26rft.volume%253D79%26rft.issue%253D6%26rft.spage%253D1081%26rft.epage%253D1092%26rft.atitle%253DRecA%2Bhomologs%2BDmc1%2Band%2BRad51%2Binteract%2Bto%2Bform%2Bmultiple%2Bnuclear%2Bcomplexes%2Bprior%2Bto%2Bmeiotic%2Bchromosome%2Bsynapsis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252894%252990038-8%26rft_id%253Dinfo%253Apmid%252F7528104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(94)90038-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7528104&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PY08600017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.11" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usadel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMzAvMTUvMjExNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNTExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.12" data-doi="10.1016/S0092-8674(04)00292-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borner</span>  <span class="cit-name-given-names">G. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Crossover/noncrossover differentiation, synaptonemal complex formation, and regulatory surveillance at the leptotene/zygotene transition of meiosis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DB%25C3%25B6rner%26rft.auinit1%253DG.%2BV.%26rft.volume%253D117%26rft.issue%253D1%26rft.spage%253D29%26rft.epage%253D45%26rft.atitle%253DCrossover%252Fnoncrossover%2Bdifferentiation%252C%2Bsynaptonemal%2Bcomplex%2Bformation%252C%2Band%2Bregulatory%2Bsurveillance%2Bat%2Bthe%2Bleptotene%252Fzygotene%2Btransition%2Bof%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252804%252900292-2%26rft_id%253Dinfo%253Apmid%252F15066280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(04)00292-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15066280&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221500300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.13" data-doi="10.1073/pnas.0711864105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borner</span>  <span class="cit-name-given-names">G. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barot</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Yeast Pch2 promotes domainal axis organization, timely recombination progression, and arrest of defective recombinosomes during meiosis.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">3327</span>–<span class="cit-lpage">3332</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0711864105%26rft_id%253Dinfo%253Apmid%252F18305165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA1LzkvMzMyNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNTExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.14" data-doi="10.1073/pnas.1310755111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jomaa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ortega</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alani</span>  <span class="cit-name-given-names">E. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Pch2 is a hexameric ring ATPase that remodels the chromosome axis protein Hop1.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">E44</span>–<span class="cit-lpage">E53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1310755111%26rft_id%253Dinfo%253Apmid%252F24367111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMTEvMS9FNDQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTUxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.15" data-doi="10.1016/j.devcel.2008.07.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsubouchi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockmill</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandler</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Global analysis of the meiotic crossover landscape.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">401</span>–<span class="cit-lpage">415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DChen%26rft.auinit1%253DS.%2BY.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D401%26rft.epage%253D415%26rft.atitle%253DGlobal%2Banalysis%2Bof%2Bthe%2Bmeiotic%2Bcrossover%2Blandscape.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2008.07.006%26rft_id%253Dinfo%253Apmid%252F18691940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2008.07.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18691940&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259358100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.16" data-doi="10.1038/ncb2451"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cole</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kauppi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roig</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Homeostatic control of recombination is implemented progressively in mouse meiosis.</span> <abbr class="cit-jnl-abbrev">Nat. Cell Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">424</span>–<span class="cit-lpage">430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCell%2BBiology%26rft.stitle%253DNature%2BCell%2BBiology%26rft.aulast%253DCole%26rft.auinit1%253DF.%26rft.volume%253D14%26rft.issue%253D4%26rft.spage%253D424%26rft.epage%253D430%26rft.atitle%253DHomeostatic%2Bcontrol%2Bof%2Brecombination%2Bis%2Bimplemented%2Bprogressively%2Bin%2Bmouse%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb2451%26rft_id%253Dinfo%253Apmid%252F22388890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncb2451&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388890&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302503300016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Copenhaver</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Housworth</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stahl</span>  <span class="cit-name-given-names">F. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Crossover interference in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">1631</span>–<span class="cit-lpage">1639</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCopenhaver%26rft.auinit1%253DG.%2BP.%26rft.volume%253D160%26rft.issue%253D4%26rft.spage%253D1631%26rft.epage%253D1639%26rft.atitle%253DCrossover%2BInterference%2Bin%2BArabidopsis%26rft_id%253Dinfo%253Apmid%252F11973316%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2MC80LzE2MzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTUxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.18" data-doi="10.1534/genetics.110.116806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cotton</span>  <span class="cit-name-given-names">V. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borts</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Distinct regulation of Mlh1p heterodimers in meiosis and mitosis in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">459</span>–<span class="cit-lpage">467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116806%26rft_id%253Dinfo%253Apmid%252F20382827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg1LzIvNDU5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE1MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Nonrandom homolog segregation at meiosis I in <em>Schizosaccharomyces pombe</em> mutants lacking recombination.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">857</span>–<span class="cit-lpage">874</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDavis%26rft.auinit1%253DL.%26rft.volume%253D163%26rft.issue%253D3%26rft.spage%253D857%26rft.epage%253D874%26rft.atitle%253DNonrandom%2BHomolog%2BSegregation%2Bat%2BMeiosis%2BI%2Bin%2BSchizosaccharomyces%2Bpombe%2BMutants%2BLacking%2BRecombination%26rft_id%253Dinfo%253Apmid%252F12663528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYzLzMvODU3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE1MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Santos</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larkin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loidl</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The Mus81/Mms4 endonuclease acts independently of double-Holliday junction resolution to promote a distinct subset of crossovers during meiosis in budding yeast.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253Dde%2Blos%2BSantos%26rft.auinit1%253DT.%26rft.volume%253D164%26rft.issue%253D1%26rft.spage%253D81%26rft.epage%253D94%26rft.atitle%253DThe%2BMus81%252FMms4%2BEndonuclease%2BActs%2BIndependently%2Bof%2BDouble-Holliday%2BJunction%2BResolution%2Bto%2BPromote%2Ba%2BDistinct%2BSubset%2Bof%2BCrossovers%2BDuring%2BMeiosis%2Bin%2BBudding%2BYeast%26rft_id%253Dinfo%253Apmid%252F12750322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTY0LzEvODEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTUxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.21" data-doi="10.1016/j.molcel.2012.02.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Muyt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jessop</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolar</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sourirajan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">BLM helicase ortholog Sgs1 is a central regulator of meiotic recombination intermediate metabolism.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DDe%2BMuyt%26rft.auinit1%253DA.%26rft.volume%253D46%26rft.issue%253D1%26rft.spage%253D43%26rft.epage%253D53%26rft.atitle%253DBLM%2Bhelicase%2Bortholog%2BSgs1%2Bis%2Ba%2Bcentral%2Bregulator%2Bof%2Bmeiotic%2Brecombination%2Bintermediate%2Bmetabolism.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2012.02.020%26rft_id%253Dinfo%253Apmid%252F22500736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2012.02.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22500736&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302838000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.22" data-doi="10.1038/ng.806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garimella</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478889&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.23" data-doi="10.1371/journal.pone.0039724"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Farmer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leung</span>  <span class="cit-name-given-names">W. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Argunhan</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terentyev</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Budding yeast Pch2, a widely conserved meiotic protein, is involved in the initiation of meiotic recombination.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e39724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DFarmer%26rft.auinit1%253DS.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253De39724%26rft.epage%253De39724%26rft.atitle%253DBudding%2Byeast%2BPch2%252C%2Ba%2Bwidely%2Bconserved%2Bmeiotic%2Bprotein%252C%2Bis%2Binvolved%2Bin%2Bthe%2Binitiation%2Bof%2Bmeiotic%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0039724%26rft_id%253Dinfo%253Apmid%252F22745819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0039724&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22745819&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.24" data-doi="10.1371/journal.pgen.1000658"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fledel-Alon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ober</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Broad-scale recombination patterns underlying proper disjunction in humans.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000658</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DFledel-Alon%26rft.auinit1%253DA.%26rft.volume%253D5%26rft.issue%253D9%26rft.spage%253De1000658%26rft.epage%253De1000658%26rft.atitle%253DBroad-scale%2Brecombination%2Bpatterns%2Bunderlying%2Bproper%2Bdisjunction%2Bin%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000658%26rft_id%253Dinfo%253Apmid%252F19763175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000658&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19763175&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.25" data-doi="10.1534/genetics.106.067603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Getz</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banse</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banse</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swanson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Reduced mismatch repair of heteroduplexes reveals “non”-interfering crossing over in wild-type <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">1251</span>–<span class="cit-lpage">1269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGetz%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D178%26rft.issue%253D3%26rft.spage%253D1251%26rft.epage%253D1269%26rft.atitle%253DReduced%2BMismatch%2BRepair%2Bof%2BHeteroduplexes%2BReveals%2B%2522Non%2522-interfering%2BCrossing%2BOver%2Bin%2BWild-Type%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.067603%26rft_id%253Dinfo%253Apmid%252F18385111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC8zLzEyNTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTUxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.26" data-doi="10.1002/yea.320110408"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gietz</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schiestl</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willems</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woods</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Studies on the transformation of intact yeast cells by the LiAc/ss‐DNA/PEG procedure.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DGietz%26rft.auinit1%253DR.%2BD.%26rft.volume%253D11%26rft.issue%253D4%26rft.spage%253D355%26rft.epage%253D360%26rft.atitle%253DStudies%2Bon%2Bthe%2Btransformation%2Bof%2Bintact%2Byeast%2Bcells%2Bby%2Bthe%2BLiAc%252FSS-DNA%252FPEG%2Bprocedure.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.320110408%26rft_id%253Dinfo%253Apmid%252F7785336%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.320110408&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7785336&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QQ03300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.27" data-doi="10.1002/(SICI)1097-0061(199910)15:14&lt;1541::AID-YEA476&gt;3.0.CO;2-K"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Three new dominant drug resistance cassettes for gene disruption in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1541</span>–<span class="cit-lpage">1553</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DGoldstein%26rft.auinit1%253DA.%2BL.%26rft.volume%253D15%26rft.issue%253D14%26rft.spage%253D1541%26rft.epage%253D1553%26rft.atitle%253DThree%2Bnew%2Bdominant%2Bdrug%2Bresistance%2Bcassettes%2Bfor%2Bgene%2Bdisruption%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0061%2528199910%252915%253A14%253C1541%253A%253AAID-YEA476%253E3.0.CO%253B2-K%26rft_id%253Dinfo%253Apmid%252F10514571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0061(199910)15:14&lt;1541::AID-YEA476&gt;3.0.CO;2-K&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10514571&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083104100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guacci</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaback</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Distributive disjunction of authentic chromosomes in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">475</span>–<span class="cit-lpage">488</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGuacci%26rft.auinit1%253DV.%26rft.volume%253D127%26rft.issue%253D3%26rft.spage%253D475%26rft.epage%253D488%26rft.atitle%253DDistributive%2Bdisjunction%2Bof%2Bauthentic%2Bchromosomes%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Apmid%252F2016050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI3LzMvNDc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE1MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.29" data-doi="10.1002/dvg.1020130608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hawley</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Irick</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zitron</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haddox</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohe</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">There are two mechanisms of achiasmate segregation in <em>Drosophila</em> females, one of which requires heterochromatic homology.</span> <abbr class="cit-jnl-abbrev">Dev. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">440</span>–<span class="cit-lpage">467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bgenetics%26rft.stitle%253DDev%2BGenet%26rft.aulast%253DHawley%26rft.auinit1%253DR.%2BS.%26rft.volume%253D13%26rft.issue%253D6%26rft.spage%253D440%26rft.epage%253D467%26rft.atitle%253DThere%2Bare%2Btwo%2Bmechanisms%2Bof%2Bachiasmate%2Bsegregation%2Bin%2BDrosophila%2Bfemales%252C%2Bone%2Bof%2Bwhich%2Brequires%2Bheterochromatic%2Bhomology.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fdvg.1020130608%26rft_id%253Dinfo%253Apmid%252F1304424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/dvg.1020130608&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1304424&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992KY12700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Higashide</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Budding yeast SLX4 contributes to the appropriate distribution of crossovers and meiotic double-strand break formation on bivalents during meiosis.</span> <abbr class="cit-jnl-abbrev">G3 Bethesda</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2033</span>–<span class="cit-lpage">2042</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2BBethesda%26rft.volume%253D6%26rft.spage%253D2033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.31" data-doi="10.1016/j.cub.2004.11.038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hillers</span>  <span class="cit-name-given-names">K. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Crossover interference.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R1036</span>–<span class="cit-lpage">R1037</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DHillers%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D14%26rft.issue%253D24%26rft.spage%253DR1036%26rft.epage%253DR1037%26rft.atitle%253DCrossover%2Binterference.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2004.11.038%26rft_id%253Dinfo%253Apmid%252F15620632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2004.11.038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15620632&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226069400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.32" data-doi="10.1016/j.cub.2003.08.026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hillers</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villeneuve</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Chromosome-wide control of meiotic crossing over in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1641</span>–<span class="cit-lpage">1647</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DHillers%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D13%26rft.issue%253D18%26rft.spage%253D1641%26rft.epage%253D1647%26rft.atitle%253DChromosome-wide%2Bcontrol%2Bof%2Bmeiotic%2Bcrossing%2Bover%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2003.08.026%26rft_id%253Dinfo%253Apmid%252F13678597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2003.08.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=13678597&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185404800026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.33" data-doi="10.1371/journal.pgen.1002351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgess</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Pch2 acts through Xrs2 and Tel1/ATM to modulate interhomolog bias and checkpoint function during meiosis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002351</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHo%26rft.auinit1%253DH.%2BC.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De1002351%26rft.epage%253De1002351%26rft.atitle%253DPch2%2Bacts%2Bthrough%2BXrs2%2Band%2BTel1%252FATM%2Bto%2Bmodulate%2Binterhomolog%2Bbias%2Band%2Bcheckpoint%2Bfunction%2Bduring%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002351%26rft_id%253Dinfo%253Apmid%252F22072981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22072981&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.34" data-doi="10.1016/j.cell.2005.07.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hochwagen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tham</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brar</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amon</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The FK506 binding protein Fpr3 counteracts protein phosphatase 1 to maintain meiotic recombination checkpoint activity.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">861</span>–<span class="cit-lpage">873</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHochwagen%26rft.auinit1%253DA.%26rft.volume%253D122%26rft.issue%253D6%26rft.spage%253D861%26rft.epage%253D873%26rft.atitle%253DThe%2BFK506%2Bbinding%2Bprotein%2BFpr3%2Bcounteracts%2Bprotein%2Bphosphatase%2B1%2Bto%2Bmaintain%2Bmeiotic%2Brecombination%2Bcheckpoint%2Bactivity.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2005.07.010%26rft_id%253Dinfo%253Apmid%252F16179256%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2005.07.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16179256&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232130200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.35" data-doi="10.1086/376610"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Housworth</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stahl</span>  <span class="cit-name-given-names">F. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Crossover interference in humans.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">188</span>–<span class="cit-lpage">197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DHousworth%26rft.auinit1%253DE.%2BA.%26rft.volume%253D73%26rft.issue%253D1%26rft.spage%253D188%26rft.epage%253D197%26rft.atitle%253DCrossover%2Binterference%2Bin%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F376610%26rft_id%253Dinfo%253Apmid%252F12772089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/376610&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12772089&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183904900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.36" data-doi="10.4161/cc.10.16.16452"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Double duty for Exo1 during meiotic recombination.</span> <abbr class="cit-jnl-abbrev">Cell Cycle</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">2607</span>–<span class="cit-lpage">2609</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bcycle%2B%2528Georgetown%252C%2BTex.%2529%26rft.stitle%253DCell%2BCycle%26rft.aulast%253DHunter%26rft.auinit1%253DN.%26rft.volume%253D10%26rft.issue%253D16%26rft.spage%253D2607%26rft.epage%253D2609%26rft.atitle%253DDouble%2Bduty%2Bfor%2BExo1%2Bduring%2Bmeiotic%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fcc.10.16.16452%26rft_id%253Dinfo%253Apmid%252F21785266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/cc.10.16.16452&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21785266&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.37" data-doi="10.1016/S0092-8674(01)00430-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The single-end invasion: an asymmetric intermediate at the double-strand break to double-Holliday junction transition of meiotic recombination.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">70</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHunter%26rft.auinit1%253DN.%26rft.volume%253D106%26rft.issue%253D1%26rft.spage%253D59%26rft.epage%253D70%26rft.atitle%253DThe%2Bsingle-end%2Binvasion%253A%2Ban%2Basymmetric%2Bintermediate%2Bat%2Bthe%2Bdouble-strand%2Bbreak%2Bto%2Bdouble-holliday%2Bjunction%2Btransition%2Bof%2Bmeiotic%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252801%252900430-5%26rft_id%253Dinfo%253Apmid%252F11461702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(01)00430-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11461702&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169870300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.38" data-doi="10.1016/j.cell.2010.05.041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hyppa</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Crossover invariance determined by partner choice for meiotic DNA break repair.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">142</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHyppa%26rft.auinit1%253DR.%2BW.%26rft.volume%253D142%26rft.issue%253D2%26rft.spage%253D243%26rft.epage%253D255%26rft.atitle%253DCrossover%2Binvariance%2Bdetermined%2Bby%2Bpartner%2Bchoice%2Bfor%2Bmeiotic%2BDNA%2Bbreak%2Brepair.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2010.05.041%26rft_id%253Dinfo%253Apmid%252F20655467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2010.05.041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20655467&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280204300016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.39" data-doi="10.1371/journal.pgen.1000557"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Joshi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barot</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jamison</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borner</span>  <span class="cit-name-given-names">G. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Pch2 links chromosome axis remodeling at future crossover sites and crossover distribution during yeast meiosis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DJoshi%26rft.auinit1%253DN.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253De1000557%26rft.epage%253De1000557%26rft.atitle%253DPch2%2Blinks%2Bchromosome%2Baxis%2Bremodeling%2Bat%2Bfuture%2Bcrossover%2Bsites%2Band%2Bcrossover%2Bdistribution%2Bduring%2Byeast%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000557%26rft_id%253Dinfo%253Apmid%252F19629172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000557&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19629172&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.40" data-doi="10.1016/j.molcel.2014.12.027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Joshi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borner</span>  <span class="cit-name-given-names">G. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Gradual implementation of the meiotic recombination program via checkpoint pathways controlled by global DSB levels.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">797</span>–<span class="cit-lpage">811</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D57%26rft.spage%253D797%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2014.12.027%26rft_id%253Dinfo%253Apmid%252F25661491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2014.12.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25661491&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaback</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barber</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamb</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">You</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Chromosome size-dependent control of meiotic reciprocal recombination in <em>Saccharomyces cerevisiae</em>: the role of crossover interference.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">152</span>: <span class="cit-fpage">1475</span>–<span class="cit-lpage">1486</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKaback%26rft.auinit1%253DD.%2BB.%26rft.volume%253D152%26rft.issue%253D4%26rft.spage%253D1475%26rft.epage%253D1486%26rft.atitle%253DChromosome%2BSize-Dependent%2BControl%2Bof%2BMeiotic%2BReciprocal%2BRecombination%2Bin%2BSaccharomyces%2Bcerevisiae%253A%2BThe%2BRole%2Bof%2BCrossover%2BInterference%26rft_id%253Dinfo%253Apmid%252F10430577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Mi80LzE0NzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTUxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.42" data-doi="10.1016/j.molcel.2015.01.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaur</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Muyt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Top3-Rmi1 DNA single-strand decatenase is integral to the formation and resolution of meiotic recombination intermediates.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">583</span>–<span class="cit-lpage">594</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D57%26rft.spage%253D583%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2015.01.020%26rft_id%253Dinfo%253Apmid%252F25699707%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2015.01.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25699707&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.43" data-doi="10.1101/gad.1227304"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kemp</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boumil</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A role for centromere pairing in meiotic chromosome segregation.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1946</span>–<span class="cit-lpage">1951</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1227304%26rft_id%253Dinfo%253Apmid%252F15289462%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE4LzE2LzE5NDYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTUxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.44" data-doi="10.1073/pnas.0402724101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zickler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Padmore</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A mechanical basis for chromosome function.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">12592</span>–<span class="cit-lpage">12597</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0402724101%26rft_id%253Dinfo%253Apmid%252F15299144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzM0LzEyNTkyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE1MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.45" data-doi="10.1083/jcb.200506170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kolas</span>  <span class="cit-name-given-names">N. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svetlanov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenzi</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macaluso</span>  <span class="cit-name-given-names">F. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipkin</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Localization of MMR proteins on meiotic chromosomes in mice indicates distinct functions during prophase I.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">447</span>–<span class="cit-lpage">458</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.200506170%26rft_id%253Dinfo%253Apmid%252F16260499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjE3MS8zLzQ0NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNTExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.46" data-doi="10.1534/genetics.114.172320"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krishnaprasad</span>  <span class="cit-name-given-names">G. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anand</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tekkedil</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Variation in crossover frequencies perturb crossover assurance without affecting meiotic chromosome segregation in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">412</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.172320%26rft_id%253Dinfo%253Apmid%252F25467183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk5LzIvMzk5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE1MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.47" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.48" data-doi="10.1038/nature12577"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Libuda</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uzawa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villeneuve</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Meiotic chromosome structures constrain and respond to designation of crossover sites.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">502</span>: <span class="cit-fpage">703</span>–<span class="cit-lpage">706</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D502%26rft.spage%253D703%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12577%26rft_id%253Dinfo%253Apmid%252F24107990%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12577&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24107990&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000326293200052&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.49" data-doi="10.1038/ng861"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lipkin</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moens</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenzi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shanmugarajah</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Meiotic arrest and aneuploidy in MLH3-deficient mice.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">385</span>–<span class="cit-lpage">390</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DLipkin%26rft.auinit1%253DS.%2BM.%26rft.volume%253D31%26rft.issue%253D4%26rft.spage%253D385%26rft.epage%253D390%26rft.atitle%253DMeiotic%2Barrest%2Band%2Baneuploidy%2Bin%2BMLH3-deficient%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng861%26rft_id%253Dinfo%253Apmid%252F12091911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng861&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12091911&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177147100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.50" data-doi="10.1038/nature07135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mancera</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bourgon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brozzi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">High-resolution mapping of meiotic crossovers and non-crossovers in yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">454</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMancera%26rft.auinit1%253DE.%26rft.volume%253D454%26rft.issue%253D7203%26rft.spage%253D479%26rft.epage%253D485%26rft.atitle%253DHigh-resolution%2Bmapping%2Bof%2Bmeiotic%2Bcrossovers%2Band%2Bnon-crossovers%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07135%26rft_id%253Dinfo%253Apmid%252F18615017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18615017&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257860300043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.51" data-doi="10.1016/j.cell.2006.05.044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martini</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diaz</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keeney</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Crossover homeostasis in yeast meiosis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMartini%26rft.auinit1%253DE.%26rft.volume%253D126%26rft.issue%253D2%26rft.spage%253D285%26rft.epage%253D295%26rft.atitle%253DCrossover%2Bhomeostasis%2Bin%2Byeast%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2006.05.044%26rft_id%253Dinfo%253Apmid%252F16873061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2006.05.044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16873061&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239552600015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.52" data-doi="10.1016/j.cell.2011.08.032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matos</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanco</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maslen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skehel</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">West</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Regulatory control of the resolution of DNA recombination intermediates during meiosis and mitosis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">147</span>: <span class="cit-fpage">158</span>–<span class="cit-lpage">172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMatos%26rft.auinit1%253DJ.%26rft.volume%253D147%26rft.issue%253D1%26rft.spage%253D158%26rft.epage%253D172%26rft.atitle%253DRegulatory%2Bcontrol%2Bof%2Bthe%2Bresolution%2Bof%2BDNA%2Brecombination%2Bintermediates%2Bduring%2Bmeiosis%2Band%2Bmitosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.08.032%26rft_id%253Dinfo%253Apmid%252F21962513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.08.032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21962513&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295396700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.53" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNTExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McPeek</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Speed</span>  <span class="cit-name-given-names">T. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Modeling interference in genetic recombination.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">139</span>: <span class="cit-fpage">1031</span>–<span class="cit-lpage">1044</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMcPeek%26rft.auinit1%253DM.%2BS.%26rft.volume%253D139%26rft.issue%253D2%26rft.spage%253D1031%26rft.epage%253D1044%26rft.atitle%253DModeling%2Binterference%2Bin%2Bgenetic%2Brecombination.%26rft_id%253Dinfo%253Apmid%252F7713406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzOS8yLzEwMzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTUxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.55" data-doi="10.1086/279541"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1916</span> <span class="cit-article-title">The mechanism of crossing-over.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">284</span>–<span class="cit-lpage">305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D50%26rft.spage%253D284%26rft_id%253Dinfo%253Adoi%252F10.1086%252F279541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/279541&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.56" data-doi="10.1007/978-1-59745-527-5_9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murakami</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borde</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keeney</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Gel electrophoresis assays for analyzing DNA double-strand breaks in <em>Saccharomyces cerevisiae</em> at various spatial resolutions.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">557</span>: <span class="cit-fpage">117</span>–<span class="cit-lpage">142</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DMurakami%26rft.auinit1%253DH.%26rft.volume%253D557%26rft.spage%253D117%26rft.epage%253D142%26rft.atitle%253DGel%2Belectrophoresis%2Bassays%2Bfor%2Banalyzing%2BDNA%2Bdouble-strand%2Bbreaks%2Bin%2BSaccharomyces%2Bcerevisiae%2Bat%2Bvarious%2Bspatial%2Bresolutions.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-59745-527-5_9%26rft_id%253Dinfo%253Apmid%252F19799180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-59745-527-5_9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19799180&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.57" data-doi="10.1093/emboj/18.20.5714"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakagawa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogawa</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The <em>Saccharomyces cerevisiae</em> MER3 gene, encoding a novel helicase-like protein, is required for crossover control in meiosis.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">5714</span>–<span class="cit-lpage">5723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DNakagawa%26rft.auinit1%253DT.%26rft.volume%253D18%26rft.issue%253D20%26rft.spage%253D5714%26rft.epage%253D5723%26rft.atitle%253DThe%2BSaccharomyces%2Bcerevisiae%2BMER3%2Bgene%252C%2Bencoding%2Ba%2Bnovel%2Bhelicase-like%2Bprotein%252C%2Bis%2Brequired%2Bfor%2Bcrossover%2Bcontrol%2Bin%2Bmeiosis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F18.20.5714%26rft_id%253Dinfo%253Apmid%252F10523314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMTgvMjAvNTcxNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNTExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.58" data-doi="10.1038/nature04885"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keeney</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Clarifying the mechanics of DNA strand exchange in meiotic recombination.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">442</span>: <span class="cit-fpage">153</span>–<span class="cit-lpage">158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DNeale%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D442%26rft.issue%253D7099%26rft.spage%253D153%26rft.epage%253D158%26rft.atitle%253DClarifying%2Bthe%2Bmechanics%2Bof%2BDNA%2Bstrand%2Bexchange%2Bin%2Bmeiotic%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04885%26rft_id%253Dinfo%253Apmid%252F16838012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04885&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16838012&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238979700037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.59" data-doi="10.1073/pnas.0913435107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Newnham</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jordan</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockmill</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The synaptonemal complex protein, Zip1, promotes the segregation of nonexchange chromosomes at meiosis I.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">781</span>–<span class="cit-lpage">785</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0913435107%26rft_id%253Dinfo%253Apmid%252F20080752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDcvMi83ODEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTUxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.60" data-doi="10.1534/genetics.108.086645"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nishant</span>  <span class="cit-name-given-names">K. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plys</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alani</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A mutation in the putative MLH3 endonuclease domain confers a defect in both mismatch repair and meiosis in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">747</span>–<span class="cit-lpage">755</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.086645%26rft_id%253Dinfo%253Apmid%252F18505871%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc5LzIvNzQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE1MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Novak</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross-Macdonald</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The budding yeast Msh4 protein functions in chromosome synapsis and the regulation of crossover distribution.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">1013</span>–<span class="cit-lpage">1025</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNovak%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D158%26rft.issue%253D3%26rft.spage%253D1013%26rft.epage%253D1025%26rft.atitle%253DThe%2BBudding%2BYeast%2BMsh4%2BProtein%2BFunctions%2Bin%2BChromosome%2BSynapsis%2Band%2Bthe%2BRegulation%2Bof%2BCrossover%2BDistribution%26rft_id%253Dinfo%253Apmid%252F11454751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1OC8zLzEwMTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTUxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.62" data-doi="10.1016/j.molcel.2008.07.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lao</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">RecQ helicase, Sgs1, and XPF family endonuclease, Mus81-Mms4, resolve aberrant joint molecules during meiotic recombination.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">324</span>–<span class="cit-lpage">336</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DOh%26rft.auinit1%253DS.%2BD.%26rft.volume%253D31%26rft.issue%253D3%26rft.spage%253D324%26rft.epage%253D336%26rft.atitle%253DRecQ%2Bhelicase%252C%2BSgs1%252C%2Band%2BXPF%2Bfamily%2Bendonuclease%252C%2BMus81-Mms4%252C%2Bresolve%2Baberrant%2Bjoint%2Bmolecules%2Bduring%2Bmeiotic%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2008.07.006%26rft_id%253Dinfo%253Apmid%252F18691965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2008.07.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18691965&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258556500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.63" data-doi="10.1371/journal.pgen.1004690"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oke</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yam</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fung</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Controlling meiotic recombinational repair - specifying the roles of ZMMs, Sgs1 and Mus81/Mms4 in crossover formation.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004690</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004690%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004690%26rft_id%253Dinfo%253Apmid%252F25329811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004690&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25329811&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.64" data-doi="10.1016/j.cell.2011.02.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kniewel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murakami</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blitzblau</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A hierarchical combination of factors shapes the genome-wide topography of yeast meiotic recombination initiation.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">719</span>–<span class="cit-lpage">731</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DPan%26rft.auinit1%253DJ.%26rft.volume%253D144%26rft.issue%253D5%26rft.spage%253D719%26rft.epage%253D731%26rft.atitle%253DA%2Bhierarchical%2Bcombination%2Bof%2Bfactors%2Bshapes%2Bthe%2Bgenome-wide%2Btopography%2Bof%2Byeast%2Bmeiotic%2Brecombination%2Binitiation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.02.009%26rft_id%253Dinfo%253Apmid%252F21376234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.02.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21376234&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288007100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.65" data-doi="10.1016/S0092-8674(03)00083-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petronczki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siomos</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nasmyth</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Un menage a quatre: the molecular biology of chromosome segregation in meiosis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DPetronczki%26rft.auinit1%253DM.%26rft.volume%253D112%26rft.issue%253D4%26rft.spage%253D423%26rft.epage%253D440%26rft.atitle%253DUn%2Bmenage%2Ba%2Bquatre%253A%2Bthe%2Bmolecular%2Bbiology%2Bof%2Bchromosome%2Bsegregation%2Bin%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252803%252900083-7%26rft_id%253Dinfo%253Apmid%252F12600308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(03)00083-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12600308&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181252600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.66" data-doi="10.1101/gad.11.20.2600"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Meiotic chromosomes: it takes two to tango.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">2600</span>–<span class="cit-lpage">2621</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DRoeder%26rft.auinit1%253DG.%2BS.%26rft.volume%253D11%26rft.issue%253D20%26rft.spage%253D2600%26rft.epage%253D2621%26rft.atitle%253DMeiotic%2Bchromosomes%253A%2Bit%2Btakes%2Btwo%2Bto%25A0tango%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.11.20.2600%26rft_id%253Dinfo%253Apmid%252F9334324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjExLzIwLzI2MDAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTUxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.67" data-doi="10.1074/jbc.M113.534644"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rogacheva</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manhart</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guarne</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Surtees</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mlh1-Mlh3, a meiotic crossover and DNA mismatch repair factor, is a Msh2-Msh3-stimulated endonuclease.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">289</span>: <span class="cit-fpage">5664</span>–<span class="cit-lpage">5673</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M113.534644%26rft_id%253Dinfo%253Apmid%252F24403070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyODkvOS81NjY0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE1MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-book" id="cit-7.5.1511.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heiter</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <em>Methods in Yeast Genetics: A Laboratory Course Manual</em>. <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>, <span class="cit-publ-loc">Cold Spring Harbor, New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.69" data-doi="10.1016/S0092-8674(00)80741-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">San-Segundo</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Pch2 links chromatin silencing to meiotic checkpoint control.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSan-Segundo%26rft.auinit1%253DP.%2BA.%26rft.volume%253D97%26rft.issue%253D3%26rft.spage%253D313%26rft.epage%253D324%26rft.atitle%253DPch2%2Blinks%2Bchromatin%2Bsilencing%2Bto%2Bmeiotic%2Bcheckpoint%2Bcontrol.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252980741-2%26rft_id%253Dinfo%253Apmid%252F10319812%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)80741-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10319812&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000080100500005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.70" data-doi="10.1016/j.yexcr.2012.03.025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Serrentino</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borde</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The spatial regulation of meiotic recombination hotspots: are all DSB hotspots crossover hotspots?</span> <abbr class="cit-jnl-abbrev">Exp. Cell Res.</abbr> <span class="cit-vol">318</span>: <span class="cit-fpage">1347</span>–<span class="cit-lpage">1352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExperimental%2Bcell%2Bresearch%26rft.stitle%253DExp%2BCell%2BRes%26rft.aulast%253DSerrentino%26rft.auinit1%253DM.%2BE.%26rft.volume%253D318%26rft.issue%253D12%26rft.spage%253D1347%26rft.epage%253D1352%26rft.atitle%253DThe%2Bspatial%2Bregulation%2Bof%2Bmeiotic%2Brecombination%2Bhotspots%253A%2Bare%2Ball%2BDSB%2Bhotspots%2Bcrossover%2Bhotspots%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.yexcr.2012.03.025%26rft_id%253Dinfo%253Apmid%252F22487095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.yexcr.2012.03.025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22487095&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.71" data-doi="10.1073/pnas.97.20.10814"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gasior</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Tid1/Rdh54 promotes colocalization of rad51 and dmc1 during meiotic recombination.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">10814</span>–<span class="cit-lpage">10819</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DShinohara%26rft.auinit1%253DM.%26rft.volume%253D97%26rft.issue%253D20%26rft.spage%253D10814%26rft.epage%253D10819%26rft.atitle%253DTid1%252FRdh54%2Bpromotes%2Bcolocalization%2Bof%2BRad51%2Band%2BDmc1%2Bduring%2Bmeiotic%2Brecombination%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.97.20.10814%26rft_id%253Dinfo%253Apmid%252F11005857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTcvMjAvMTA4MTQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTUxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.72" data-doi="10.1038/ng.83"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Crossover assurance and crossover interference are distinctly regulated by the ZMM proteins during yeast meiosis.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DShinohara%26rft.auinit1%253DM.%26rft.volume%253D40%26rft.issue%253D3%26rft.spage%253D299%26rft.epage%253D309%26rft.atitle%253DCrossover%2Bassurance%2Band%2Bcrossover%2Binterference%2Bare%2Bdistinctly%2Bregulated%2Bby%2Bthe%2BZMM%2Bproteins%2Bduring%2Byeast%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.83%26rft_id%253Dinfo%253Apmid%252F18297071%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.83&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18297071&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253548400020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.73" data-doi="10.1126/science.289.5477.300"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shonn</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarroll</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Requirement of the spindle checkpoint for proper chromosome segregation in budding yeast meiosis.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">289</span>: <span class="cit-fpage">300</span>–<span class="cit-lpage">303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DShonn%26rft.auinit1%253DM.%2BA.%26rft.volume%253D289%26rft.issue%253D5477%26rft.spage%253D300%26rft.epage%253D303%26rft.atitle%253DRequirement%2Bof%2Bthe%2BSpindle%2BCheckpoint%2Bfor%2BProper%2BChromosome%2BSegregation%2Bin%2BBudding%2BYeast%2BMeiosis%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.289.5477.300%26rft_id%253Dinfo%253Apmid%252F10894778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODkvNTQ3Ny8zMDAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTUxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.74" data-doi="10.1016/j.molcel.2004.06.040"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Snowden</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berardini</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fishel</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">hMSH4-hMSH5 recognizes Holliday junctions and forms a meiosis-specific sliding clamp that embraces homologous chromosomes.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DSnowden%26rft.auinit1%253DT.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D437%26rft.epage%253D451%26rft.atitle%253DhMSH4-hMSH5%2Brecognizes%2BHolliday%2BJunctions%2Band%2Bforms%2Ba%2Bmeiosis-specific%2Bsliding%2Bclamp%2Bthat%2Bembraces%2Bhomologous%2Bchromosomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2004.06.040%26rft_id%253Dinfo%253Apmid%252F15304223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2004.06.040&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15304223&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223312300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.75"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sonntag Brown</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishant</span>  <span class="cit-name-given-names">K. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alani</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic analysis of <em>mlh3</em> mutations reveals interactions between crossover promoting factors during meiosis in baker’s yeast.</span> <abbr class="cit-jnl-abbrev">G3 Bethesda</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2BBethesda%26rft.volume%253D3%26rft.spage%253D9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.76" data-doi="10.1534/genetics.104.027789"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stahl</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foss</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borts</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abdullah</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Does crossover interference count in <em>Saccharomyces cerevisiae?</em></span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">35</span>–<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DStahl%26rft.auinit1%253DF.%2BW.%26rft.volume%253D168%26rft.issue%253D1%26rft.spage%253D35%26rft.epage%253D48%26rft.atitle%253DDoes%2BCrossover%2BInterference%2BCount%2Bin%2BSaccharomyces%2Bcerevisiae%253F%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.027789%26rft_id%253Dinfo%253Apmid%252F15454525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTY4LzEvMzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTUxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Subramanian</span>  <span class="cit-name-given-names">V. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacQueen</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vader</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Chromosome synapsis alleviates Mek1-dependent suppression of meiotic DNA repair.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">e1002369</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D14%26rft.spage%253De1002369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.78" data-doi="10.7554/eLife.07424"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Markowitz</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blitzblau</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Transcription dynamically patterns the meiotic chromosome-axis interface.</span> <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e07424</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.07424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZWxpZmUiO3M6NToicmVzaWQiO3M6MTA6IjQvMC9lMDc0MjQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTUxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.79" data-doi="10.1016/0092-8674(94)90197-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sym</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Crossover interference is abolished in the absence of a synaptonemal complex protein.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">283</span>–<span class="cit-lpage">292</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSym%26rft.auinit1%253DM.%26rft.volume%253D79%26rft.issue%253D2%26rft.spage%253D283%26rft.epage%253D292%26rft.atitle%253DCrossover%2Binterference%2Bis%2Babolished%2Bin%2Bthe%2Babsence%2Bof%2Ba%2Bsynaptonemal%2Bcomplex%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252894%252990197-X%26rft_id%253Dinfo%253Apmid%252F7954796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(94)90197-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7954796&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PN63000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.80" data-doi="10.1073/pnas.0611490104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Terasawa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogawa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsukamoto</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shirahige</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Meiotic recombination-related DNA synthesis and its implications for cross-over and non-cross-over recombinant formation.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">5965</span>–<span class="cit-lpage">5970</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0611490104%26rft_id%253Dinfo%253Apmid%252F17384152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA0LzE0LzU5NjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTUxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.81" data-doi="10.1038/nature10331"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vader</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blitzblau</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tame</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falk</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curtin</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Protection of repetitive DNA borders from self-induced meiotic instability.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">477</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DVader%26rft.auinit1%253DG.%26rft.volume%253D477%26rft.issue%253D7362%26rft.spage%253D115%26rft.epage%253D119%26rft.atitle%253DProtection%2Bof%2Brepetitive%2BDNA%2Bborders%2Bfrom%2Bself-induced%2Bmeiotic%2Binstability.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10331%26rft_id%253Dinfo%253Apmid%252F21822291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10331&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21822291&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294404300043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.82" data-doi="10.4161/15384101.2014.991185"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zickler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Meiotic crossover patterns: obligatory crossover, interference and homeostasis in a single process.</span> <abbr class="cit-jnl-abbrev">Cell Cycle</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">305</span>–<span class="cit-lpage">314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BCycle%26rft.volume%253D14%26rft.spage%253D305%26rft_id%253Dinfo%253Adoi%252F10.4161%252F15384101.2014.991185%26rft_id%253Dinfo%253Apmid%252F25590558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/15384101.2014.991185&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25590558&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.83" data-doi="10.1371/journal.pgen.1000702"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wojtasz</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daniel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roig</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolcun-Filas</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mouse HORMAD1 and HORMAD2, two conserved meiotic chromosomal proteins, are depleted from synapsed chromosome axes with the help of TRIP13 AAA-ATPase.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000702</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWojtasz%26rft.auinit1%253DL.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253De1000702%26rft.epage%253De1000702%26rft.atitle%253DMouse%2BHORMAD1%2Band%2BHORMAD2%252C%2Btwo%2Bconserved%2Bmeiotic%2Bchromosomal%2Bproteins%252C%2Bare%2Bdepleted%2Bfrom%2Bsynapsed%2Bchromosome%2Baxes%2Bwith%2Bthe%2Bhelp%2Bof%2BTRIP13%2BAAA-ATPase.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000702%26rft_id%253Dinfo%253Apmid%252F19851446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000702&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19851446&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.84" data-doi="10.1016/j.cub.2006.10.069"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgess</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Two distinct surveillance mechanisms monitor meiotic chromosome metabolism in budding yeast.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">2473</span>–<span class="cit-lpage">2479</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DWu%26rft.auinit1%253DH.%2BY.%26rft.volume%253D16%26rft.issue%253D24%26rft.spage%253D2473%26rft.epage%253D2479%26rft.atitle%253DTwo%2Bdistinct%2Bsurveillance%2Bmechanisms%2Bmonitor%2Bmeiotic%2Bchromosome%2Bmetabolism%2Bin%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2006.10.069%26rft_id%253Dinfo%253Apmid%252F17174924%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2006.10.069&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17174924&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.85" data-doi="10.1016/j.cell.2012.03.023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zakharyevich</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Delineation of joint molecule resolution pathways in meiosis identifies a crossover-specific resolvase.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">334</span>–<span class="cit-lpage">347</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DZakharyevich%26rft.auinit1%253DK.%26rft.volume%253D149%26rft.issue%253D2%26rft.spage%253D334%26rft.epage%253D347%26rft.atitle%253DDelineation%2Bof%2Bjoint%2Bmolecule%2Bresolution%2Bpathways%2Bin%2Bmeiosis%2Bidentifies%2Ba%2Bcrossover-specific%2Bresolvase.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.03.023%26rft_id%253Dinfo%253Apmid%252F22500800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.03.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22500800&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302805600013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.86" data-doi="10.1371/journal.pgen.1000571"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zanders</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alani</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The pch2Delta mutation in baker’s yeast alters meiotic crossover levels and confers a defect in crossover interference.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000571</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DZanders%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253De1000571%26rft.epage%253De1000571%26rft.atitle%253DThe%2Bpch2Delta%2Bmutation%2Bin%2Bbaker%2527s%2Byeast%2Balters%2Bmeiotic%2Bcrossover%2Blevels%2Band%2Bconfers%2Ba%2Bdefect%2Bin%2Bcrossover%2Binterference.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000571%26rft_id%253Dinfo%253Apmid%252F19629178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000571&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19629178&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.87" data-doi="10.1534/genetics.111.129031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zanders</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonntag Brown</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alani</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Pch2 modulates chromatid partner choice during meiotic double-strand break repair in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">511</span>–<span class="cit-lpage">521</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.129031%26rft_id%253Dinfo%253Apmid%252F21515575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzMvNTExIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE1MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.88" data-doi="10.1038/nature13442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Topoisomerase II mediates meiotic crossover interference.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">511</span>: <span class="cit-fpage">551</span>–<span class="cit-lpage">556</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D511%26rft.spage%253D551%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13442%26rft_id%253Dinfo%253Apmid%252F25043020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13442&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25043020&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000339566300026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1511.89"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zickler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A few of our favorite things: pairing, the bouquet, crossover interference and evolution of meiosis.</span> <abbr class="cit-jnl-abbrev">Semin. Cell Dev. Biol.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">135</span>–<span class="cit-lpage">148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSemin.%2BCell%2BDev.%2BBiol.%26rft.volume%253D54%26rft.spage%253D135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/5/1511.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/5/1499" title="Evaluating Methods of Updating Training Data in Long-Term Genomewide Selection" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/5/1525" title="Genomic Comparison of Indigenous African and Northern European Chickens Reveals Putative Mechanisms of Stress Tolerance Related to Environmental Selection Pressure" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/5"><span>Volume 7 Issue 5, May 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/5.cover-source.jpg?itok=VWLvGstM" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1511" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-mjdN3yGS3a3ARpzvbPDrnzahS_suhhHS5XJohU2ZdBs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/5/1511" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/5/1511" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/5/1511" class="active" data-icon-position="" data-hide-link-title="0">Modulating Crossover Frequency and Interference for Obligate Crossovers in Saccharomyces cerevisiae Meiosis</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/484259" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Vo2Unsw1ef_H7s_STn3NgSMGi6NiIuMwGaWZbLdNCXM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/5/1511.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/5/1511" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-x2voFFje9tsj-oky0YRo16q0u5bma9eRDXu5LqAgr9Q" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/5/1511" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/5/1511.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484259" id="node484259--2" data-pisa="ggg;7/5/1511" data-pisa-master="ggg;g3.117.040071" data-apath="/ggg/7/5/1511.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Modulating Crossover Frequency and Interference for Obligate Crossovers in <em>Saccharomyces cerevisiae</em> Meiosis</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Parijat Chakraborty</span>, <span class="highwire-citation-author" data-delta="1">Ajith V. Pankajam</span>, <span class="highwire-citation-author" data-delta="2">Gen Lin</span>, <span class="highwire-citation-author" data-delta="3">Abhishek Dutta</span>, <span class="highwire-citation-author" data-delta="4">G. Nandanan Krishnaprasad</span>, <span class="highwire-citation-author" data-delta="5">Manu M. Tekkedil</span>, <span class="highwire-citation-author" data-delta="6">Akira Shinohara</span>, <span class="highwire-citation-author" data-delta="7">Lars M. Steinmetz</span> and <span class="highwire-citation-author" data-delta="8">K. Thazath Nishant</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1511-1524;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.040071 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484259--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Parijat Chakraborty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research, Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Parijat%2BChakraborty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chakraborty%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AParijat%2BChakraborty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ajith V. Pankajam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research, Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ajith%2BV.%2BPankajam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pankajam%20AV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAjith%2BV.%2BPankajam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Gen Lin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genome Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gen%2BLin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lin%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGen%2BLin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Abhishek Dutta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research, Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abhishek%2BDutta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dutta%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbhishek%2BDutta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">G. Nandanan Krishnaprasad </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research, Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=G.%2BNandanan%2BKrishnaprasad%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Krishnaprasad%20GN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AG.%2BNandanan%2BKrishnaprasad%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Manu M. Tekkedil </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genome Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manu%2BM.%2BTekkedil%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tekkedil%20MM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AManu%2BM.%2BTekkedil%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Akira Shinohara </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Protein Research, Osaka University, 565-0871, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Akira%2BShinohara%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shinohara%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAkira%2BShinohara%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Lars M. Steinmetz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genome Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, Stanford University, California 94305</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Stanford Genome Technology Center, Palo Alto, California 94304</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lars%2BM.%2BSteinmetz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steinmetz%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALars%2BM.%2BSteinmetz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">K. Thazath Nishant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research, Thiruvananthapuram, Trivandrum 695016, India</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Center for Computation, Modelling and Simulation, Indian Institute of Science Education and Research, Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=K.%2BThazath%2BNishant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nishant%20KT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AK.%2BThazath%2BNishant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nishantkt@iisertvm.ac.in" class="" data-icon-position="" data-hide-link-title="0">nishantkt@iisertvm.ac.in</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/5/1511" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/5/1511&amp;title=Modulating%20Crossover%20Frequency%20and%20Interference%20for%20Obligate%20Crossovers%20in%20Saccharomyces%20cerevisiae%20Meiosis" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/5/1511&amp;title=Modulating%20Crossover%20Frequency%20and%20Interference%20for%20Obligate%20Crossovers%20in%20Saccharomyces%20cerevisiae%20Meiosis" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/5/1511&amp;title=Modulating%20Crossover%20Frequency%20and%20Interference%20for%20Obligate%20Crossovers%20in%20Saccharomyces%20cerevisiae%20Meiosis" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/5/1511&amp;text=Modulating%20Crossover%20Frequency%20and%20Interference%20for%20Obligate%20Crossovers%20in%20Saccharomyces%20cerevisiae%20Meiosis" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/5/1511&amp;title=Modulating%20Crossover%20Frequency%20and%20Interference%20for%20Obligate%20Crossovers%20in%20Saccharomyces%20cerevisiae%20Meiosis" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/5/1511&amp;t=Modulating%20Crossover%20Frequency%20and%20Interference%20for%20Obligate%20Crossovers%20in%20Saccharomyces%20cerevisiae%20Meiosis" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/5/1511&amp;title=Modulating%20Crossover%20Frequency%20and%20Interference%20for%20Obligate%20Crossovers%20in%20Saccharomyces%20cerevisiae%20Meiosis" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/5/1511&amp;title=Modulating%20Crossover%20Frequency%20and%20Interference%20for%20Obligate%20Crossovers%20in%20Saccharomyces%20cerevisiae%20Meiosis" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/484259/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484259" id="node484259--3" data-pisa="ggg;7/5/1511" data-pisa-master="ggg;g3.117.040071" data-apath="/ggg/7/5/1511.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Modulating Crossover Frequency and Interference for Obligate Crossovers in <em>Saccharomyces cerevisiae</em> Meiosis</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Parijat Chakraborty</span>, <span class="highwire-citation-author" data-delta="1">Ajith V. Pankajam</span>, <span class="highwire-citation-author" data-delta="2">Gen Lin</span>, <span class="highwire-citation-author" data-delta="3">Abhishek Dutta</span>, <span class="highwire-citation-author" data-delta="4">G. Nandanan Krishnaprasad</span>, <span class="highwire-citation-author" data-delta="5">Manu M. Tekkedil</span>, <span class="highwire-citation-author" data-delta="6">Akira Shinohara</span>, <span class="highwire-citation-author" data-delta="7">Lars M. Steinmetz</span> and <span class="highwire-citation-author" data-delta="8">K. Thazath Nishant</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1511-1524;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.040071 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484259--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Parijat Chakraborty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research, Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Parijat%2BChakraborty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chakraborty%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AParijat%2BChakraborty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ajith V. Pankajam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research, Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ajith%2BV.%2BPankajam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pankajam%20AV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAjith%2BV.%2BPankajam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Gen Lin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genome Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gen%2BLin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lin%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGen%2BLin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Abhishek Dutta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research, Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abhishek%2BDutta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dutta%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbhishek%2BDutta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">G. Nandanan Krishnaprasad </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research, Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=G.%2BNandanan%2BKrishnaprasad%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Krishnaprasad%20GN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AG.%2BNandanan%2BKrishnaprasad%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Manu M. Tekkedil </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genome Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manu%2BM.%2BTekkedil%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tekkedil%20MM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AManu%2BM.%2BTekkedil%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Akira Shinohara </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Protein Research, Osaka University, 565-0871, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Akira%2BShinohara%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shinohara%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAkira%2BShinohara%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Lars M. Steinmetz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genome Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, Stanford University, California 94305</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Stanford Genome Technology Center, Palo Alto, California 94304</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lars%2BM.%2BSteinmetz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steinmetz%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALars%2BM.%2BSteinmetz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">K. Thazath Nishant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biology, Indian Institute of Science Education and Research, Thiruvananthapuram, Trivandrum 695016, India</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Center for Computation, Modelling and Simulation, Indian Institute of Science Education and Research, Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=K.%2BThazath%2BNishant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nishant%20KT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AK.%2BThazath%2BNishant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nishantkt@iisertvm.ac.in" class="" data-icon-position="" data-hide-link-title="0">nishantkt@iisertvm.ac.in</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1511&amp;rft.epage=1524&amp;rft.atitle=Modulating%20Crossover%20Frequency%20and%20Interference%20for%20Obligate%20Crossovers%20in%20Saccharomyces%20cerevisiae%20Meiosis&amp;rft.volume=7&amp;rft.issue=5&amp;rft.date=2017-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Chakraborty%2C+Parijat&amp;rft.au=Pankajam%2C+Ajith+V.&amp;rft.au=Lin%2C+Gen&amp;rft.au=Dutta%2C+Abhishek&amp;rft.au=Krishnaprasad%2C+G.+Nandanan&amp;rft.au=Tekkedil%2C+Manu+M.&amp;rft.au=Shinohara%2C+Akira&amp;rft.au=Steinmetz%2C+Lars+M.&amp;rft.au=Nishant%2C+K.+Thazath"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/484259/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/484259/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/484259/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/484259/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/484259/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/484259/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/484259/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/484259/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/484259/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/484259/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/484259/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/484259/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/5/1511" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/5/1511.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/5/1511.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/5/1511.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/5/1511.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
