<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/3/895" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/3/895.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/3/895.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/3/895.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/3/895/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/3/895.atom" />
<meta name="HW.pisa" content="ggg;7/3/895" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Predicted Residual Error Sum of Squares of Mixed Models: An Application for Genomic Prediction" />
<meta name="DC.Identifier" content="10.1534/g3.116.038059" />
<meta name="DC.Date" content="2017-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Xu. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genomic prediction is a statistical method to predict phenotypes of polygenic traits using high-throughput genomic data. Most diseases and behaviors in humans and animals are polygenic traits. The majority of agronomic traits in crops are also polygenic. Accurate prediction of these traits can help medical professionals diagnose acute diseases and breeders to increase food products, and therefore significantly contribute to human health and global food security. The best linear unbiased prediction (BLUP) is an important tool to analyze high-throughput genomic data for prediction. However, to judge the efficacy of the BLUP model with a particular set of predictors for a given trait, one has to provide an unbiased mechanism to evaluate the predictability. Cross-validation (CV) is an essential tool to achieve this goal, where a sample is partitioned into K parts of roughly equal size, one part is predicted using parameters estimated from the remaining K – 1 parts, and eventually every part is predicted using a sample excluding that part. Such a CV is called the K-fold CV. Unfortunately, CV presents a substantial increase in computational burden. We developed an alternative method, the HAT method, to replace CV. The new method corrects the estimated residual errors from the whole sample analysis using the leverage values of a hat matrix of the random effects to achieve the predicted residual errors. Properties of the HAT method were investigated using seven agronomic and 1000 metabolomic traits of an inbred rice population. Results showed that the HAT method is a very good approximation of the CV method. The method was also applied to 10 traits in 1495 hybrid rice with 1.6 million SNPs, and to human height of 6161 subjects with roughly 0.5 million SNPs of the Framingham heart study data. Predictabilities of the HAT and CV methods were all similar. The HAT method allows us to easily evaluate the predictabilities of genomic prediction for large numbers of traits in very large populations." />
<meta name="DC.Contributor" content="Shizhong Xu" />
<meta name="article:published_time" content="2017-03-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Predicted Residual Error Sum of Squares of Mixed Models: An Application for Genomic Prediction" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genomic prediction is a statistical method to predict phenotypes of polygenic traits using high-throughput genomic data. Most diseases and behaviors in humans and animals are polygenic traits. The majority of agronomic traits in crops are also polygenic. Accurate prediction of these traits can help medical professionals diagnose acute diseases and breeders to increase food products, and therefore significantly contribute to human health and global food security. The best linear unbiased prediction (BLUP) is an important tool to analyze high-throughput genomic data for prediction. However, to judge the efficacy of the BLUP model with a particular set of predictors for a given trait, one has to provide an unbiased mechanism to evaluate the predictability. Cross-validation (CV) is an essential tool to achieve this goal, where a sample is partitioned into &lt;i&gt;K&lt;/i&gt; parts of roughly equal size, one part is predicted using parameters estimated from the remaining &lt;i&gt;K&lt;/i&gt; – 1 parts, and eventually every part is predicted using a sample excluding that part. Such a CV is called the K-fold CV. Unfortunately, CV presents a substantial increase in computational burden. We developed an alternative method, the HAT method, to replace CV. The new method corrects the estimated residual errors from the whole sample analysis using the leverage values of a hat matrix of the random effects to achieve the predicted residual errors. Properties of the HAT method were investigated using seven agronomic and 1000 metabolomic traits of an inbred rice population. Results showed that the HAT method is a very good approximation of the CV method. The method was also applied to 10 traits in 1495 hybrid rice with 1.6 million SNPs, and to human height of 6161 subjects with roughly 0.5 million SNPs of the Framingham heart study data. Predictabilities of the HAT and CV methods were all similar. The HAT method allows us to easily evaluate the predictabilities of genomic prediction for large numbers of traits in very large populations.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/03/01" />
<meta name="citation_mjid" content="ggg;7/3/895" />
<meta name="citation_id" content="7/3/895" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/3/895" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/3/895.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/3/895.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/3/895.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.038059" />
<meta name="citation_pmid" content="28108552" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="15" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="895" />
<meta name="citation_lastpage" content="909" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Shizhong Xu" />
<meta name="citation_author_institution" content="Department of Botany and Plant Sciences, University of California, Riverside, California 92521" />
<meta name="citation_author_email" content="shizhong.xu@ucr.edu" />
<meta name="citation_reference" content="citation_journal_title=Technometrics;citation_author=D. M. Allen;citation_title=Mean square error of prediction as a criterion for selecting variables.;citation_pages=469-475;citation_volume=13;citation_year=1971;citation_doi=10.2307/1267161" />
<meta name="citation_reference" content="citation_journal_title=Technometrics;citation_author=D. M. Allen;citation_title=The relationship between variable selection and data augmentation and a method for prediction;citation_pages=125-127;citation_volume=16;citation_year=1974;citation_doi=10.2307/1267500" />
<meta name="citation_reference" content="citation_journal_title=Technometrics;citation_author=D. Cook;citation_title=Detection of influential observation in linear regression.;citation_pages=15-18;citation_volume=19;citation_year=1977;citation_doi=10.2307/1268249" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=D. Cook;citation_title=Influential observation in linear regression.;citation_pages=169-174;citation_volume=74;citation_year=1979;citation_doi=10.2307/2286747" />
<meta name="citation_reference" content="citation_journal_title=Numer. Math.;citation_author=P. Craven;citation_author=G. Wahba;citation_title=Smoothing noisy data with spline functions: estimating the correct degree of smoothing by the method of generalized cross-validation.;citation_pages=377-403;citation_volume=31;citation_year=1979;citation_doi=10.1007/BF01404567" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Public Health Nations Health;citation_author=T. R. Dawber;citation_author=G. F. Meadors;citation_author=F. E. Moore;citation_title=Epidemiological approaches to heart disease - the Framingham study.;citation_pages=279-286;citation_volume=41;citation_year=1951;citation_pmid=14819398;citation_doi=10.2105/AJPH.41.3.279" />
<meta name="citation_reference" content="citation_journal_title=Annals of the New York Academy of Sciences;citation_journal_abbrev=Ann. N. Y. Acad. Sci.;citation_author=T. R. Dawber;citation_author=W. B. Kannel;citation_author=L. P. Lyell;citation_title=An approach to longitudinal studies in a community: the Framingham Study.;citation_pages=539-556;citation_volume=107;citation_year=1963;citation_pmid=14025561" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=H. Naya;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=A. Legarra;citation_title=Predicting quantitative traits with regression models for dense molecular markers and pedigree.;citation_volume=182;citation_year=2009;citation_pmid=19293140;citation_doi=10.1534/genetics.109.101501" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. L. Calus;citation_title=Whole-genome regression and prediction methods applied to plant and animal breeding.;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=G. de los Campos;citation_author=A. I. Vazquez;citation_author=R. Fernando;citation_author=Y. C. Klimentidis;citation_author=D. Sorensen;citation_title=Prediction of complex human traits using the genomic best linear unbiased predictor.;citation_pages=e1003608;citation_volume=9;citation_year=2013;citation_pmid=23874214;citation_doi=10.1371/journal.pgen.1003608" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=G. de los Campos;citation_author=D. Sorensen;citation_author=D. Gianola;citation_title=Genomic heritability: what is it?;citation_pages=e1005048;citation_volume=11;citation_year=2015;citation_pmid=25942577;citation_doi=10.1371/journal.pgen.1005048" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_title=Priors in whole-genome regression: the Bayesian alphabet returns.;citation_volume=194;citation_year=2013;citation_pmid=23636739;citation_doi=10.1534/genetics.113.151753" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=D. Gianola;citation_author=C.-C. Schon;citation_title=Cross-validation without doing cross-validation in genome-enabled prediction.;citation_pages=3107-3128;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_author=R. L. Fernando;citation_author=A. Stella;citation_title=Genomic-assisted prediction of genetic value with semiparametric procedures.;citation_volume=173;citation_year=2006;citation_pmid=16648593;citation_doi=10.1534/genetics.105.049510" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=M. E. Goddard;citation_author=B. J. Hayes;citation_title=Genomic selection.;citation_pages=323-330;citation_volume=124;citation_year=2007;citation_issue=6;citation_pmid=18076469;citation_doi=10.1111/j.1439-0388.2007.00702.x" />
<meta name="citation_reference" content="citation_journal_title=Technometrics;citation_author=G. H. Golab;citation_author=M. Heath;citation_author=G. Wahba;citation_title=Generalized cross-validation as a method for choosing a good ridge parameter.;citation_pages=215-223;citation_volume=21;citation_year=1979;citation_doi=10.2307/1268518" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. Gong;citation_author=W. Chen;citation_author=Y. Gao;citation_author=X. Liu;citation_author=H. Zhang;citation_title=Genetic analysis of the metabolome exemplified using a rice population.;citation_volume=110;citation_year=2013;citation_pmid=24259710;citation_doi=10.1073/pnas.1319681110" />
<meta name="citation_reference" content="citation_title=The Elements of Statistical Learning: Data Mining, Inference, and Prediction;citation_author=T. Hastie;citation_author=R. Tibshirani;citation_author=J. Friedman;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. L. Heffner;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic Selection for Crop Improvement;citation_pages=1-12;citation_volume=49;citation_year=2009;citation_issue=1;citation_doi=10.2135/cropsci2008.08.0512" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=C. R. Henderson;citation_title=Best linear unbiased estimation and prediction under a selection model.;citation_pages=423-447;citation_volume=31;citation_year=1975;citation_issue=2;citation_pmid=1174616;citation_doi=10.2307/2529430" />
<meta name="citation_reference" content="citation_journal_title=Technometrics;citation_author=A. E. Hoerl;citation_author=R. W. Kennard;citation_title=Ridge regression: applications to nonorthogonal problems.;citation_pages=69-82;citation_volume=12;citation_year=1970;citation_doi=10.2307/1267352" />
<meta name="citation_reference" content="citation_journal_title=Technometrics;citation_author=A. E. Hoerl;citation_author=R. W. Kennard;citation_title=Ridge regression: biased estimation for nonorthogonal problems.;citation_pages=55-67;citation_volume=12;citation_year=1970;citation_doi=10.2307/1267351" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=X. Huang;citation_author=S. Yang;citation_author=J. Gong;citation_author=Y. Zhao;citation_author=Q. Feng;citation_title=Genomic analysis of hybrid rice varieties reveals numerous superior alleles that contribute to heterosis.;citation_pages=6258;citation_volume=6;citation_year=2015;citation_pmid=25651972;citation_doi=10.1038/ncomms7258" />
<meta name="citation_reference" content="citation_journal_title=Signal Process.;citation_author=M. Jansen;citation_author=M. Malfait;citation_author=A. Bultheel;citation_title=Generalized cross validation for wavelet thresholding.;citation_pages=33-44;citation_volume=56;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=K.-C. Li;citation_title=Asymptotic optimality for Cp, CL, cross-validation and generalized cross-validation: discrete index set.;citation_pages=958-975;citation_volume=15;citation_year=1987" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=B. Mathew;citation_author=J. Léon;citation_author=M. J. Sillanpää;citation_title=Integrated nested Laplace approximation inference and cross-validation to tune variance components in estimation of breeding value.;citation_pages=99;citation_volume=35;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Biometrika;citation_journal_abbrev=Biometrika;citation_author=H. D. Patterson;citation_author=R. Thompson;citation_title=Recovery of inter-block information when block sizes are unequal;citation_pages=545-554;citation_volume=58;citation_year=1971;citation_issue=3;citation_doi=10.1093/biomet/58.3.545" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=R. R. Picard;citation_author=D. Cook;citation_title=Cross-validation of regression models;citation_pages=575-583;citation_volume=79;citation_year=1984;citation_pmid=Can't run EX;citation_doi=10.2307/2288403" />
<meta name="citation_reference" content="citation_title=Proceedings of the Twenty-Ninth Annual SAS Users Group International Conference;citation_title=Mixed model influence diagnostics;citation_author=O. Schabenberger;citation_pages=1-17;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Twin research :  the official journal of the International Society for Twin Studies;citation_journal_abbrev=Twin Res;citation_author=K. Silventoinen;citation_author=S. Sammalisto;citation_author=M. Perola;citation_author=D. I. Boomsma;citation_author=B. K. Cornes;citation_title=Heritability of adult body height: a comparative study of twin cohorts in eight countries.;citation_pages=399-408;citation_volume=6;citation_year=2003;citation_issue=5;citation_pmid=14624724;citation_doi=10.1375/136905203770326402" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B;citation_author=R. Tibshirani;citation_title=Regression shrinkage and selection via the Lasso.;citation_pages=267-288;citation_volume=58;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. I. Vazquez;citation_author=G. de los Campos;citation_author=Y. C. Klimentidis;citation_author=G. J. M. Rosa;citation_author=D. Gianola;citation_title=A comprehensive genetic approach for improving prediction of skin cancer risk in humans.;citation_volume=192;citation_year=2012;citation_pmid=23051645;citation_doi=10.1534/genetics.112.141705" />
<meta name="citation_reference" content="citation_journal_title=Numer. Math.;citation_author=G. Wahba;citation_title=Smoothing noisy data with spline functions;citation_pages=383-393;citation_volume=24;citation_year=1975;citation_doi=10.1007/BF01437407" />
<meta name="citation_reference" content="citation_title=International Conference on Approximation Theory in Honour of George Lorenz;citation_title=Spline bases, regularization, and generalized crossvalidation for solving approximation problems with large quantities of noisy data;citation_author=G. Wahba;citation_pages=905-912;citation_year=1980" />
<meta name="citation_reference" content="citation_title=Spline Models for Observational Data;citation_author=G. Wahba;citation_year=1990" />
<meta name="citation_reference" content="citation_title=Support Vector Machines, Reproducing Kernel Hilbert Spaces and the Randomized GACV;citation_author=G. Wahba;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Commun. Stat.;citation_author=G. Wahba;citation_author=S. Wold;citation_title=A completely automatic french curve: fitting spline functions by cross validation.;citation_pages=1-17;citation_volume=4;citation_year=1975" />
<meta name="citation_reference" content="citation_journal_title=Commun. Stat.;citation_author=G. Wahba;citation_author=S. Wold;citation_title=Periodic splines for spectral density estimation: the use of cross-validation for determining the degree of smoothing.;citation_pages=125-141;citation_volume=4;citation_year=1975" />
<meta name="citation_reference" content="citation_journal_title=Ann. Numer. Math.;citation_author=G. Wahba;citation_author=Z. Luo;citation_title=Smoothing spline ANOVA fits for very large, nearly regular data sets, with application to historical global climate data.;citation_pages=579-598;citation_volume=4;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=G. Wahba;citation_author=Y. Wang;citation_author=C. Gu;citation_author=R. Klein;citation_author=B. Klein;citation_title=Smoothing spline ANOVA for exponential families, with application to the Wisconsin epidemiological study of diabetic retinopathy.;citation_pages=1865-1895;citation_volume=23;citation_year=1995;citation_doi=10.1214/aos/1034713638" />
<meta name="citation_reference" content="citation_title=Advances in Large Margin Classifiers;citation_title=GACV for support vector machines;citation_author=G. Wahba;citation_author=Y. Lin;citation_author=H. Zhang;citation_pages=297-311;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. Series B Stat. Methodol.;citation_author=Y. Wang;citation_title=Mixed effects smoothing spline analysis of variance.;citation_pages=159-174;citation_volume=60;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=Y. Wang;citation_title=Smoothing spline models with correlated random errors.;citation_pages=341-348;citation_volume=93;citation_year=1998;citation_doi=10.2307/2669630" />
<meta name="citation_reference" content="citation_title=Memorandum Report;citation_author=M. A. Woodbury;citation_title=Inverting modified matrices;citation_pages=4;citation_volume=Volume 42;citation_year=1950" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Xu;citation_title=Mapping quantitative trait loci by controlling polygenic background effects.;citation_volume=195;citation_year=2013;citation_pmid=24077303;citation_doi=10.1534/genetics.113.157032" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Xu;citation_author=D. Zhu;citation_author=Q. Zhang;citation_title=Predicting hybrid performance in rice using genomic best linear unbiased prediction.;citation_volume=111;citation_year=2014;citation_pmid=25114224;citation_doi=10.1073/pnas.1413750111" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=H. Yu;citation_author=W. Xie;citation_author=J. Wang;citation_author=Y. Xing;citation_author=C. Xu;citation_title=Gains in QTL detection using an ultra-high density SNP map based on population sequencing relative to traditional RFLP/SSR markers.;citation_pages=e17595-e17595;citation_volume=6;citation_year=2011;citation_issue=3;citation_pmid=21390234;citation_doi=10.1371/journal.pone.0017595" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Predicted Residual Error Sum of Squares of Mixed Models: An Application for Genomic Prediction" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3.cover-source.jpg" />
<meta name="twitter:description" content="Genomic prediction is a statistical method to predict phenotypes of polygenic traits using high-throughput genomic data. Most diseases and behaviors in humans and animals are polygenic traits. The majority of agronomic traits in crops are also polygenic. Accurate prediction of these traits can help medical professionals diagnose acute diseases and breeders to increase food products, and therefore significantly contribute to human health and global food security. The best linear unbiased prediction (BLUP) is an important tool to analyze high-throughput genomic data for prediction. However, to judge the efficacy of the BLUP model with a particular set of predictors for a given trait, one has to provide an unbiased mechanism to evaluate the predictability. Cross-validation (CV) is an essential tool to achieve this goal, where a sample is partitioned into K parts of roughly equal size, one part is predicted using parameters estimated from the remaining K – 1 parts, and eventually every part is predicted using a sample excluding that part. Such a CV is called the K-fold CV. Unfortunately, CV presents a substantial increase in computational burden. We developed an alternative method, the HAT method, to replace CV. The new method corrects the estimated residual errors from the whole sample analysis using the leverage values of a hat matrix of the random effects to achieve the predicted residual errors. Properties of the HAT method were investigated using seven agronomic and 1000 metabolomic traits of an inbred rice population. Results showed that the HAT method is a very good approximation of the CV method. The method was also applied to 10 traits in 1495 hybrid rice with 1.6 million SNPs, and to human height of 6161 subjects with roughly 0.5 million SNPs of the Framingham heart study data. Predictabilities of the HAT and CV methods were all similar. The HAT method allows us to easily evaluate the predictabilities of genomic prediction for large numbers of traits in very large populations." />
<meta name="og-title" property="og:title" content="Predicted Residual Error Sum of Squares of Mixed Models: An Application for Genomic Prediction" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/3/895" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genomic prediction is a statistical method to predict phenotypes of polygenic traits using high-throughput genomic data. Most diseases and behaviors in humans and animals are polygenic traits. The majority of agronomic traits in crops are also polygenic. Accurate prediction of these traits can help medical professionals diagnose acute diseases and breeders to increase food products, and therefore significantly contribute to human health and global food security. The best linear unbiased prediction (BLUP) is an important tool to analyze high-throughput genomic data for prediction. However, to judge the efficacy of the BLUP model with a particular set of predictors for a given trait, one has to provide an unbiased mechanism to evaluate the predictability. Cross-validation (CV) is an essential tool to achieve this goal, where a sample is partitioned into K parts of roughly equal size, one part is predicted using parameters estimated from the remaining K – 1 parts, and eventually every part is predicted using a sample excluding that part. Such a CV is called the K-fold CV. Unfortunately, CV presents a substantial increase in computational burden. We developed an alternative method, the HAT method, to replace CV. The new method corrects the estimated residual errors from the whole sample analysis using the leverage values of a hat matrix of the random effects to achieve the predicted residual errors. Properties of the HAT method were investigated using seven agronomic and 1000 metabolomic traits of an inbred rice population. Results showed that the HAT method is a very good approximation of the CV method. The method was also applied to 10 traits in 1495 hybrid rice with 1.6 million SNPs, and to human height of 6161 subjects with roughly 0.5 million SNPs of the Framingham heart study data. Predictabilities of the HAT and CV methods were all similar. The HAT method allows us to easily evaluate the predictabilities of genomic prediction for large numbers of traits in very large populations." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3.cover-source.jpg" />
    <title>Predicted Residual Error Sum of Squares of Mixed Models: An Application for Genomic Prediction | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"So3o3ugrZDG7ZDP3L5ZTpa1V1LOTeTPTb87VxAprIwA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"483532","apath":"\/ggg\/7\/3\/895.atom","pisa":"ggg;7\/3\/895","ac":{"\/ggg\/7\/3\/895.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/3\/895.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/3\/895"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/3\/895"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/3\/895":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022FjzwT40RxXN+9Hu0jZ3sdgFWUNAqJBuR2dSo4NjvqB9g1EkjU5dL3WdIsVlw73OAT0jpLfdH2Hc8lJ4oKVxtZt9Phg7AXUsnYOabHxXmpNTPD9hK9HHR4oxEOxPSsz5f9Z3sCw1axz59gEquDjO1CLsBpDipKVAa7fCmAVftb1XW1RpEfoRdBkpq8Y8PhuXLPkNJWdwVfW3D60dJD5573KYUfMNhTeqME22f1bS4QUCqSdmryB2z0Sf5LPJBdQNF5sYjAZLtdKCE6RJqC9onNqwXhSzdMTOzOTU6Ha+JdimdTQ2Rc2KaH3\\\\\\\/4441ucoIV6B6RN3EZYqAe7mOs\\\\\\\/dXn3tujLCJntTP9LlAOWVII468lbpOt29z1Ww1t4eR8etp7VtviO+anwZsFH+UMT9rh\\\\\\\/dhi1XkfsoZJSZdd\\\\\\\/wbAi8R+YYcTp1XYZSdPNM+C\\\\\\\/m8vEK8dsN7OtWbFeOxvt40et6B5LCuR905W9CgBINo5MnD9WnITrUtYGZ+jUUjJEqyCufUhLW3mMJS2pejmmjWqmCH88qLJLRpM9A8S+f65pRl49AsZb1OCeI8mBOkVderzg4SmSojhlWxc3iUJ0jFaeJq2R7QibSV3qQc+zwn6KaNTUdCX9T9j9RJI5ssfuwszc3pszmT14QXVlpml2NFL4hSJTpqZNxSYJN2AP7\\\\\\\/yyJiwv300j6WqaEI9uYQ9xl7RPG6eVTrmQIiy9ocMmPpGVAp9V1DwD4z2RszlwiCRUyW6fooYvycHSQfMt8JNpdml8VASRkN8EpzbKB5j0Th9VAWSKBHCaZAbrOR+ZQiHH4grZJx0JWq5JkcXEwL5uwVlOALZ2D\\\\\\\/z3B+6EG03RdSQ9cHLnAWQJDZ8p8+\\\\\\\/TCZ7b8kmS16Pe6y4CIvOK76Ulhvkc4solSjbUwcem503bUIGS39Kc4ZIKw1YxRWw\\\\\\\/cKNN8qHG4KbUEBEDqrp4ffOn27LN5mKtVn\\\\\\\/1thg7YyzE74zjjcEXKdHkQdLy2E3PZRu1wc405d5JA9bhhMBsn8vdNbbH5YZQ7lRGiXTBVuHBg6odtCgnFr6LQ9V5E3w6Tgs4UkAYC\\\\\\\/N2Ptt9QsqrsjpGtQrTiyf69TvZVwPxBRjtx\\\\\\\/yIzzolfX5Tq2pG6\\\\\\\/t85fbHL4\\\\\\\/G1KnL\\\\\\\/cReVEhLCT8dTS56WZMzplupE1gbvNSsTBhRdDk5HjMvlRMhr+uuOqWmFvcVq6boJS64eSIKu\\\\\\\/f0vtRk3dbmJULyuiLW\\\\\\\/9EgYZlXZV3VhGpr4jMnvsbZAuG4+H3GTPYUfJLfhzxdNxymJld\\\u0022,\\\u0022iv\\\u0022:\\\u00227tWufw76WoWYiCrDnr3Y7g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002212516a378ba3336353a76c9de7057273d8620620c907b5cc2f6cf4c07376e149\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022g\\\\\\\/5D8Lz4W6xwNNsndpO9CQmqBMvHnMZNoU0f4J2u8EbOIpQFWvlMA1mSXNiLG3MeP2HZmQfu1o9I7hyBtNP+Q1e1aR8E2XmE3KQ\\\\\\\/AAum3fC87zsMU3I4jpvd0S+VjqPRGtMlTCIrb\\\\\\\/sj6TGKS\\\\\\\/QsRdUFvbsTXnI+ITKyme0JsY6c0zs3rvphp6sVCtg+IZ3iiCesFDPfJaNiAGSDvTPvWnfl6op+PW6yxzwDYIOgrZp3iQ1oEsUvJ2xQjPCbA6ODJDr5cWRt2j2rBVrNktzOrqawbTNzvC2dB0X16SgpWLXeYz76Q9SKR\\\\\\\/6ru7MO0KcS9Yr7JEXFN8gUf94\\\\\\\/jsDVvl0wyxn8\\\\\\\/B3DNTxhvJK4K\\\\\\\/0d9inVoKa4fsXDDMO9Rxkvw4Ll2YTaROIWBhbmPDKvS894Uog6tC7bhRlnfMh8ocv\\\\\\\/QYJDiXJyue8yz5\\\\\\\/Tfc6wI48FtVvz49Jc13X8MAXE1gYLmBxMfAbOeUB3FaiE2yFJiMZpZzlgWUc+++mba+ywpygKSvqZ5A39N8XiMEj3cInAesf5RVujl0WuRXCqlgt\\\\\\\/Fj5YXp9UgU433leyth8gAF6q4vJGSGmHeozeaHXIfYFPMKcEULNMZpKZFVEvYPSbfmEJNQdUoDqEqYnqACeuXwAuLgy\\\\\\\/RKDsqKXc838Iuv8+V5haFEpBxZudmIMDgj1p3KtQ2wjYSwzUNIrBuo4gtRNIMQRy3nmqhwteJ4wLd\\\\\\\/a4t5JgnITuFV14pN0jaPqMYsKZu4g2MlNJv+Pf3+XTaqKg0m+Q0jwFW9iMleO++4cQKPajY1SW5YGJQcBF4XICFu58Vv4+nO1\\\\\\\/4V8SG2i\\\\\\\/Mh6e\\\\\\\/xo+RcdvDb3w4Q9bj\\\\\\\/XUGJDTmeJiehj+msWEI561hBGLMZ99UdD\\\\\\\/dzz\\\\\\\/MdLeT46VL1GitWANXAHF8uesZfkHn9svGeBBL\\\\\\\/JAkAEQ5IcrYpZq7rhLYjWPtTFRJLZSAv+zKSfjvPNYgxMP2IT1kwaL27tXNDwusQD24ZWanswpprxLqag1tEmmYvyyoXV2wHiuhtktKxlO6nORGjBSu6wU\\\\\\\/r5ETdIlVnEI\\\\\\\/Bj\\\\\\\/fNs10q7RmfpLEqSp5zC+G2sx315nNsTaelLtNUwlcbsDPDxskGm2Aigg4ogNUtAkawpcyv0nKE8l4ovcQuTAdnbv77A+WjC\\\\\\\/t82pMb5dfODB\\\\\\\/\\\\\\\/fLJtcqRJl9lexiQn97WdMNDJ5Iss\\\\\\\/DuSdlmDn+zAV3Y4YtRCJuLPK\\\\\\\/pwrT71uaegW3oMd0jZuYoLR976x\\\\\\\/hBzuW9z7TFWfQPRYsHk7B4jMAMyk77FQjTT11kMWEaURUt\\\\\\\/ze1kk1txPpuYPKO7LyxkLW5Nj2YiwehKowJ\\\\\\\/EhemrLWMHJdrVcoHIb0Ll4bYfqK0T34Kc\\\\\\\/UpXLNwExpqs3hA\\\\\\\/1sYBcrFtOG3wa\\\\\\\/mnrUf\\\\\\\/C5MSnb2Si+v9Yo4QZSwG\\\\\\\/lyzByVYR\\\\\\\/NAbhu\\\\\\\/Dz8XlxVKU\\\\\\\/0m33XQzfi\\\\\\\/QKnOWCZa4\\\\\\\/HENhgpzkkslNDmeR+FRWTCF4P+IBkI6oeLtTUMzggNJmDaz\\\\\\\/zF74HkkVf0801ksH2Zx8C3EYh3Sk7jEqSv5FkU3KPwU9WKP2rr8JPaQbWIpLXGlNG4QIYuzOeIoEcj8HFpeh\\\\\\\/rjLIwgH21KMoL0L1CEPAF\\\\\\\/NVGOKvOiYbOKj2kFW7qBNINoevue3WWqv0LXeukp+4ckVMWEntnKeglP1S9PG77iNP43ZRBvYb3vgUsW4Fip1\\\\\\\/F3Eo5pDnqhrS9XmxeOarmieEAemjBbn1gFHC+pJjtzkWpj33xExiOk0F1zQdRcHzm4z0kjMAgciKe150sZNUCIYbPyTqgkX\\\\\\\/0qnToD4D2lmgCUK+Ex2QtsVNt\\\\\\\/7PVis0U7WrO2QusZw0T0z0vtCqnlrQXjZhws9f9amKPnklfWKOc96DCoDd1tFHu4j8EbSebzyagtqxQzAp8+rKek1OVxnkyfrJUGthTuI8Fu9FoWJPLwYnz6JbGfxCIUke+CH\\\\\\\/cIqyg6PMrIVTFg35183lh63JbIfEJeNjEEvZ36jLyJ9dPRdIjO8JaLApMFdgOcmPWq\\\\\\\/LAWYfSJlD2\\\\\\\/KjEKXuJADhO0fNGlBgO1rDvDY1cmQQFNMU9ntz7YHn0AbfTHdK2l9w9\\\\\\\/mp8x5ABT6N+tVDG9XAzPsPM0wNtcgbhMl6pibDwVINu3pqV0xTohB\\\\\\\/VuRQ4kJrDKwzoPtiaVtvLctpYlnnnyPd2HA8Ryquf3890idvQ902V9OcpuO0cRLOGuiOwDHzkW\\\\\\\/6pyLlSVLh+wgRivRA\\\\\\\/8A0gJCaszbHzJk6AfuUvhImLu4N0\\\\\\\/K4qfCIc0w5f1KD4BGTCwjdT\\\\\\\/0PKCDv\\\\\\\/fxNAaTD9OQOUibWHpCdtHRSj9CXjVUdfpogYamm7oEgoGkMXOYjMyilZkFM9DQ7\\\\\\\/6Bs3lDMVB3Aefw0nmRblIZZGweOdu\\\\\\\/J5r3it67CKHnl48hkNdt6Wk8cO3olqLe\\\\\\\/5CY\\\\\\\/wmLiXCvRM1mBQfp2BI8TlJnaaLbHB3ZUD6PkILfOj6JNSQiw51+R9pULpgCiIDk+Q7hK4UG2BmIvDZchdxNpo0NxUMkq2QU+m2biOFcLNIdOO326rJk6nvDzo9BXO74aY8XOSZ85FdNk1rL4mJhsfxglwJCQEysYIHDC5L69QASRYUu64ZO0LksWQTY+hUsPDsHKUvcEnNMpRBpYLl0+eEJX6f0OUbK58qmUPGWsvBB9KJCBZ0TgSju0GPNasUkBEZL5I6M5NI6nQrvS+WREnXk06tIZDFNjeDrk7It1NECeJg1w1eFAEuO8cb0ewtB3Bv6A68l5urtB0bzRCl6Dpm45EgTtth44TmCRvNw36wvvPWgK1S4k0N7Wml0D71QTzm7+wMz+bQGYJZRa+qFEL5Fa\\\\\\\/7UvoaW5+sI8kgm7g+Z4PoIPUvn7ckvG16FWzEt8Pcwlyrw+QRmMZtmGvRKXZdWjsfbudAQni+qW7\\\\\\\/2\\\\\\\/6jBdB48RWsFjpdB7YmLG1UEmT98ZzsvkTSpj2DGNLsUT9jWGmr6lUp0brjCi8iNsXVgmB3ztExdesQBNP\\\\\\\/kCbpBB3A5g7tjPg8lZAN5BK+ySgT1IwIUI5Y+pyKD7tWtQ6GJoYK3FDotWVrQ7UTr\\\\\\\/IVDeKBfeVRJKRK5RzW57DHqwFfgEWNNKdefcCo+ddppHVkezW2DS8xeBbMxzhLsUn0DVfmoJtrvUwiMLts9kcCdboBFjxSKkWDzC7tWB5UmaAUI+QdQs3w7WdZ3oyCcAJAb4sLGzn1MrL9xk8v3mZ6KqxFILczMeZA1n+PZ4Na6sWGOAtTz19JXBjoQsPfdi9VZdXMFGtAeNrmSJxIody3hYo6sBYLWtqIx8Q6KTVDC3vgG68LSD5\\\\\\\/Pwcu8sAzORW\\\\\\\/RhoSYnr8g1Uqx8c6mirD2lgifPTDcg6s3+BP+k1K9oSlqtzrJonKw2TSZS0pK8SSWl\\\\\\\/wtUjO7nAO+bf3qz+jYa+W8kxXLlgLjTm3Y2LOb2HwltAg5185BSya1slNeymwfADKIhc0yMkeESYkiNgzaLrcq8MvqvGUGgX8HQ1F89Avi1mJSgJLWW64tJ+Lt4fKyxO1cRsZ1EVuLoy1OKMNJFLem7kpN8j60bXwzsCPgWMbPKadLTOH+cKWZr6kpsBI\\\\\\\/Zt6zdZjT7CSMzo7KieggzhRyssRJyUgeGuqjMCDmd6qbNKBuGkCLoNW89TIbqhz1yVuaJOGcoG00UXaLwnzZ6PBLWdGVX1E4Su91LNYUmsHmDP0Q\\\\\\\/djvzRySMCZ+Vi0tOFKpI6JujBrn8Wxyw3JR0f7ll5EVxVAWYW3J2xmoaeEZ\\\\\\\/F8EDl\\\\\\\/EiIAHG2iVejPB3om4IgKkzUUvuwy+2ZJmLNw3f3z0CypZUZpGtXxu0vgz7\\\\\\\/hfms0ywBE6CUO6Or24Iay7wf8vIyezMpES0byIdjgMKMMJNF35wN4S1AwBnYABwQJcor5\\\\\\\/nwhKVFK\\\\\\\/Au74KluoOFBnXifQVevK+CrZwS0Q3Zsz\\\\\\\/FSbTdfJt1YHv4TuxgsRzs6mg4Qkdhf7XRr3WPX1I02FDN5ob0zWJ9Ny1JA5lZtccyzxnbSwgkY247C95V4IHXCGAUB7NjcQlKTrAraCBRKOZrslvVGBb3dgtvbNtgnRf4mUYAbLXS6CuFiugnY+agPfRT016vRL5FY83TTvmRva2k\\\\\\\/B61M4hzVytjrC2d5xnqInoMpF0kY47YPpr1hahKqIldJRwOBkxhSUpLSij1TDgMPCNyLTQF9xnuMXMcSfCjDuexJ1sMPPSHay6VVjNQeaqUM7GZ+FMt0COSf6tqQ4UM8nh4MUj3b3mJHlQewk2vR9IJx4yexi+siBp+\\\\\\\/vR+HK+tbGm5cnwx55k5dk4omXrtd0LlWJdzpTIrg3OHGCalS0OsXRdUmRUoHsIOWaZKA6gnNO\\\\\\\/DL4NHOiuptiP4hAFoYDBnm+n7u\\\\\\\/1y7L4MUP2lVTF9bzPXHnb14ss6g+KcwIDaS6rH2CwlnsCi3ch0HNEVna0\\\\\\\/emdRWqrgU6OmMkHb\\\\\\\/tpnqlemW4y3sxFbYbgDXq4PthhEqXP1KyQsFz658fhD6I5JkEMYR+yiytJ+U0MPZIN2TgqfiQs4hYuDMvfw4FlmmdmI661sO+ej8tXST3t6ttp5L1LYrFwPYWg7jwYs3xWDon+7EIDFHvETYb0iBzcqJ0hXYKne3HqZ0IbMCwMN7gFNYHtR8k9ok3sKSxQcfZ9Pn\\\\\\\/h1NsoIOktInptCEAMHtSVHqpvHhK4sdTnV+hHBPqDqPbJ4MWAh3YsV6xZO4nT4+nf6Mz8i9GUMiBboUodojwBG55HFYheniqiDpzBjFN4Mz35pH8BlEms\\\\\\\/GqNjd4uP8fYR+PPuzbzuBajeJi98utBsLQ9fPLCjdWWozmPgsx6lnkPs\\\\\\\/VcFEguqGRH3oE3LU7O+aDhW98SYn9eY+JyzclMEm5vdOA2LO0wGv+9NyZ8la0CEcUDEH6MP2imzrFaVU6fW6zAlnfI9AHM2KvmycywKcq+vos2p1ymPzHo7Fx7G3Lh\\\\\\\/AF4Slz5\\\\\\\/JXn0qoI+KCpkvsYGo\\\\\\\/9J29Cu0wNWFEk\\\\\\\/NlDecn8hAen9yK8ddwnXOzohP7Ozq5b7uj+9Vw2Ybh0NgNbMIoKZPgCadDx5aPBNioXXENLvhvVgU8lL+ZpHSkKUhJHZFdNtVN29yiYqArpu8zTsX5piulm+XhZaIUEDg6S9GxdtTRZRV9uAIN+CChtcalR71fcxHbKroz2ZHrEk5mfSJBAneMJIXN61uHLwIZkRCRKKrQ9Rf\\\\\\\/IEyUtRzBSu3W6\\\\\\\/1TcAY1rLx9w\\\\\\\/hH+0ZZ\\\\\\\/OcRMuPc+UdtxNzgZ8w7sTaio50N95Gi2G+qTNDnHEZJjQDEKGiOheP29nKyrOguAZg6JgWlNBQmyEaMaKXCtN23Toqden7Lju9MiepO40Y2mW4gyVlPqXtnvZhnnchJkOGvqJIDaDhVeZH0pnMCazMTu5YHUflR+Vbx6Tq3xSOq4UMTr40ttsGimKEemlNxxm6WWpIGSKj2W33aV6KGYg0d3JRVMfkHalSJZJ\\\\\\\/X8Boi\\\\\\\/yRznStOuQpQUa2I2LskiPevLlMQRRdnUBEdV\\\\\\\/CXPHZC9CXob+xHXlrcO4SxTIeL8krxZ+AsZCESWfjiiWk\\\\\\\/7LTe\\\\\\\/mI5HjON5wV9lFBnfJN+G73KnAIQjnxnY5UeARnA7NYNMrK+OyvGxm+TygPPTM6rGcRiOjhTksMMn8ew4dusS7Szzo+66j2Thh1Et0QQJ+Ws6kVidZQCKkUXNA1QnzAa3W7SZ\\\\\\\/t055ceouGJb5xdyWtoUP67qh\\\\\\\/jOmRI9Xqwa7EL2ep0zXeoj0MNT91EbzYN6n2kNk6ANBmjj0Z4F+gFsnS3YaF955PYNox7UivOMKRjP+IO7ByouR\\\\\\\/eM0Gnh6WKkeyfv7lzwsZR8ZoBvE9r2bd8rAbfiUqnJGNyG9\\\\\\\/jC2E0Y6CPkUjOpQAg6TFVk9msZy\\\\\\\/rjwvwq8Rc0Xfx5SqCD2+V\\\\\\\/I8wkJxonbQbiuv3StG3CJ1XXUZJ26v5Qr1OaM9fZdl7eScQGNf1j6MUku6Gl3uCqUwDnQMqaQ5IL3CmVImz\\\\\\\/NobunAwBJcbeh+R4kI\\\\\\\/FKyhHDQk\\\\\\\/a\\\\\\\/luvY1QJULWVqJcj4pD\\\\\\\/CVH6F0w0Y\\\\\\\/gLGoPp4NHvv5JLwGOVSXeY8gVYiC6FF+QXMCupysggVlt7FhlTH9Z3LT9mB1JrTtm77JlfuCgbv+31hoFy0EFkO4uePdYowzWY04iD\\\\\\\/fx\\\\\\\/LQ9smYyMRV3SHRYljYP\\\\\\\/5inbmZ20NGX0IsxONlGfngzKcBcMDixcTGn+cb2RDe7TZE6KQkipEmJtlF77mjkcYdkIDmW5gKFKT16jfPnFY0Doyan++Wsfz6IuEHEe0lzZfUP2gCPOKNOooVezNd1gqEIl8klk3dyEcXLhmPXOF+oIyIBYHdcLtKFrf6BJAMX6kw9WekebsQpIrl1bL2vkkA9woMnw8MRgfyNHGn0y3CFcDxJtpNMVrJzJj9H3U+E4TJbvkMjIOlwNEg+epnJ1yZEi8kRZrciSMbovTNveESc8HFAinlvqislZ93AsHg9ntAtZd1kdF\\\\\\\/xw0iCqBaTTEf3vsFQ5Jt77rIIsz7sytidE9vy1gf5d5oiP6Cu4kgjIKFgJM8Eu3P5+BNubwUixn8cEFfsPuRPHpiRQKxpwKyJKoqFjiIESHIV8BnUte3GlyITa5iNNBig6bsookS0AiqbqvEEicBIgNSAaCtcU4sLUl3okV9vfTeDrUepEBME+gkEY5NK\\\\\\\/KaHHzHxqKmC+2zteoSXpwrunt4rITaoTWP\\\\\\\/w6edkV4lOWIA1sZu8021pVO6Pxm\\\\\\\/lM0Q1z\\\\\\\/kvrOaqbcVvVZn0dZrw\\\\\\\/mBFBZLFrLffE6YqQE8pfi5pc3FnoWCE\\\\\\\/hLTufNykEOyu7Nf2A6XHxX\\\\\\\/HJZiJLUFJZqsn1j9Alre+9Mv4Jq2Cx3LPdpqGq1JU4aFC+UlvMheS4M2OAQe2CVvyfGGy5B2f3vcMtZaPBlKcaLxfX64Mp0nPJdU2Dgrz0yEFIebnE\\\\\\\/h9TJ2y7ukwCzy3\\\\\\\/1IUESYIiPgC3+Qe1F8uH13UGPjSwlxX+MqVYsYT\\\\\\\/eM5FO03EsyakXgYKbOREnybBvWvwl+d1nI5c1d46gPz590LFM2Gz970jUiiqRyA564soZpYImrxhK711yktd+KGdYEBKqCN+e2bt8Dt2DHZH2nYR\\\\\\\/wkb1ROJaFldePJogziN9oozFCuvBe820IStuk5w7xGnDtr5nqyKhpaj7VWb18FII3kcKO5stQ4cHo2auAhqnOsurq7EtVUIJQmjHM\\\\\\\/zjkwjFjON+WANCJPP1kY5Uf\\\\\\\/ZjbPMC3t0B+rhxNqE5st1\\\\\\\/cymgS69fh7eisBaFQ34CCc7tQavDikbQI6RZ6\\\\\\\/0zCNpFLIGvoYd+7wz1B1+X\\\\\\\/1IbrUxJ\\\\\\\/8Ixpw6KXuAB1agEdBOKgX3DuGpWqt7hAYcLXHdvXLwrrS7Qq45pUFhZ8gCJ5D7fG8mMzqdWgqzZI0+v\\\\\\\/qP3guZY6AKWVhN5Jc\\\\\\\/2liqwD\\\\\\\/i9BYTlHn18XXUN8Kl5fr8vakjpJEioe1RlUfohiKdHfFVtJbUJjYCb\\\\\\\/b9WVTWtGAT\\\\\\\/95lDt79xWJ\\\\\\\/J9bDWcKoP3lLspRdi+QNrpOqRSeN1qTt8gRao+F3+s3g+OPa\\\\\\\/KeBkh4PUeMPYPBOZ\\\\\\\/VrLe45QdnjE5M\\\\\\\/dO\\\\\\\/2yPVNxyQ+Qv+O2tP\\\\\\\/Q5ei3MAuN2czJcDS4JcsDxSxIyp\\\\\\\/MV2ho0ZgapFubEnYmMJLfpCBBbi\\\\\\\/qaXW\\\\\\\/N41KHbpZxmwVXlPcWcvMMm52weD72BYoemHxwTakML\\\\\\\/P20rdwKh8FuPRNRSxu3SMZih0FCdFqcVh4OI3FClKkjKtwrEV4XzmInWE4BeDol0eE6p\\\\\\\/CB6hLsWH4STPtrG4Fd5wr9e8gQQBJEessVQnlwnllsVV\\\\\\\/a\\\\\\\/JCiqOGEFMuVrysnhkOasr2O13UeRikJxY+a8zD3D9nVvdoyUJd9dDkmSJnlxohlkEwmCmcafCtDMPNEW6a2E9ZQyNkVQBXzfcSdj2R6BdAj2ZFf0gtoNgVHzpuXhbshebBimmDZWaZdxmOs92xaicxTaNbaD1jHa45Y\\\\\\\/RfgbGgE7UFguRqxR7OIJBgC3uOU4X\\\\\\\/lrkEhCCFaeHAmjyNzWsQpTOa\\\\\\\/EDyswgMwVOkXACgo7R3B9PtpP2j7HKf9IMuejbL4vm1lL3ze6vafb0oBa2Z78H4iD97KHVa9yHg61SnFybfdR1MWUJa3ES2jrQjM\\\\\\\/f9rI38NcahZMyIK76C69f\\\\\\\/BRNikVj4beYcul27r3tzjbdTR32UELqNkRcBEWtRmIRXi8K+QNqUtmQDlsNIz+WVGVyYW+7ImDkCedfYYqsmnDUwKOmz6a22ra4pICqQyYpMmNVxCqRBkaKdrBPfzwmnOxVS20UfqdwmHvTR820Un9gKaIJtivQJFrIVuxxgn3jGFFcpIqSg5idElsOWrvMRR5IlAbLTD6GML3SW0ooEJPTnjYGNK4nWKxRjugqQy8EwRcdo294Q6guW\\\\\\\/OMcs1JKcvWo9SbgJbQHHSqCpfIATzrQV\\\\\\\/e9aOf02zqPx6w0l3+DuIGap7pW\\\\\\\/Vd7usrmhIGsrZkWezHm6WCdG1CHUydf\\\\\\\/8KLgG+lMohpSlVHxEGLFk+PCJslP7h7CLZiH2RkZXZXvZuG19HNaSFmjNJYDdEMUXJCNnWu6Lk5DMnkNFYY49AFyWJMPbZIxhlQswbfeXO7AqldkYJ17ukPkkMCfFAQuv7G5zh7m+g8bZ4dSnd5kYs+WJbLbdxWf02pWoe958A48Gl6dGBMaNKpAQ5ibPsoVYn+\\\\\\\/\\\\\\\/G6qEXNrRRb0wYuDrIQ8gY82neAOQSvQ62XDsHuyyekYqJYImvk\\\\\\\/n7oEusPV0ZKom8tToUZv1ZRukE+kZM1KkiJFFOdWIrBDtLi4+p59t75TuNbg5t3CS1hZSC44QVpckEaPK90lX5vitl186iQhyIiACG7Okp3IjFmCYPvxULqKLOuGNCQXNGpzu4MwuIHAQ9yhuuWiDwvhNa7ZT9NdGMIq1gZQGmHQ1XcW4DCbgFbKbe40nLuNkYpYqfIBeZdHXE+61MGaXE\\\\\\\/RZVuxQtRPtM5yFVz1hrT\\\\\\\/h4FuBYp5DjaiMTe0NrG6UH460DALwNWRC7K\\\\\\\/W+Ru3sFzDFnYf1TaKhs1tpmC7RWeNQyvBJBrBZid4gtf3kgOpPyNwpxYalXluR+KCyv8YR7bEXqdymuJ9iBWZxQOwAw9t+MByd\\\\\\\/pqgokO1cOYM\\\\\\\/5ED8w2XBBD\\\\\\\/8qHBAOnJvKdTY\\\\\\\/MQwEt8B82sUX6x2ihk5spVb8xBAQyEPH6MtQz\\\\\\\/hYxQ75NOH0bmr+neI68gnqNBu6foAKFzV666iQ+FQTtUM8rfn4btYRpUhyxz9BDxVdCIxRCkyfi1g4Gvc9Ec8ZcRqPtvNL4dnYw3E\\\\\\\/UC9qL24JxUvOKsY7WKROAGHuvHtJHFJlIx64UxqSVEND8JeeBB70+LgPkVX5JTkpx\\\\\\\/5TyuC\\\\\\\/yQXausRDoiMBQdj2FUa2y7y6BRshWirLjGyP\\\\\\\/OOstlZ3Xil9CTnMEL6pP2XFF\\\\\\\/mnDX2\\\\\\\/TuIdAZq9QUlKsA18m7uVBiWL63HCu6JZL9t9hqm0DdpLRwH0T6in\\\\\\\/aufuzYAWJr4uQ9unPfpKC6tWbfRuPKUDBcijfJtNnr7Ac4eYtfqSiQa0CdUqVQai8D8i6DqPgXunI0k7Z6RVxTy49+Ya2rBY3DRZX7fK7VGnYw01d0o6Fzd56Juxdr6eugV5l2oNh9b618G\\\\\\\/4AjZr661W9pGtEJtjhftZfWYRM53yh9G0ooXDyZepMB99yHSzwKPQScu7LwxUuKXYVMT45ZpBuNQTLlVBM2O059cLH11ercfG0lFon44l+I4VGdhMsvpmfoheE\\\\\\\/8XtlPmQ0cDkPrXDXE4GYyKmmofPLu+zZrl8b4NkjHqQV8IV+kgZpQDfMgUu4v3TUk67WYjk6qFXJDImZ07nSo3mxqhtOeRIvImK8aH6tNQVBz5o3TDA8Xg7v1xj3j9XtEvicyq5ZPnRNsMwIGVmdX1QTCY1Zm155EXi7nM7Nn3dLO4OergoimLsLgDLAV\\\\\\\/HmSdXLyL3K3WzkCWG6XaujKKlV15TBeZC3yaFUiJbM8ACz6FADUxIPXxVxqmu+A+O2Z5h6gK89ta0tusyzwz0yNCKZbw0bIfL5EUz7x+Nh6EAhnsug1nGYiyU18YUqUNlT3Fcz1kUUogA+DLt\\\\\\\/I9d2YmtQ1gGaqkDqYULSQ72aj9fumeq9URvquisisIGnO1YF4dw1ilzO6bgBqaKyQ9snBp6Mu3KQQ1PjfDlLGaUnXmb3d+EetLcuGcL3ubbUKEwx6HQ5V179C1x5pgNtcn0NYn5ZbaBSKRMM1UY0718wiG+aLmvhfgv7g5sD8BocGwPB8gUqQtKWhAc1ts40n+DTu5Uqdr6YuI0r6XImwuNu216Nc\\\\\\\/dB1qCsGo1eBNOP80EKh\\\\\\\/pEVYJWFhKsGe2LQXzvqL9IlDo074uIXB9ve083adnfClcDCvd\\\\\\\/i8uqC0orgEHQHG4xlc6gmqebl\\\\\\\/GU5OEsqTbRRUTIgOk4zUKqe7sb7I3P5YLGW+mtNRi9nCIg5zF66MiGfb0X0Jpx\\\\\\\/wAnptsmX\\\\\\\/KA\\\\\\\/2RWQPKQr+ARQLD+qJGc0tI1SWt5cVTP\\\\\\\/SQi0k20MXi21Sku5XqgKz9Y47AT+gqnmoE8PZmYCfvOUM1ZHtugngt3cjWsm+dKtIOk8w\\\\\\\/QkgLdLsjB0vmgo4GrD9aPJqDwuSssSRsOzDvgXL4NLfPXX3bvxou9fo9w8q2Um5435tss1lTd0WfZxc+2cjYUzAzsq3I60xWsfqaYWo\\\\\\\/0mwHEMhC9OKa5gpRXimZ0wiQ7N0P3mn7rubrBDO1BQnO1PCWhBUpcBSOHXwv7uCsI1mPDIgFZN97oGNgb15O2nXvx2fLP8yWKYCq+vSQbafxyai5Rk3UUsbIO4D5434iUCxkFcE9zoB9mBROEPOm2aezxc79hXHdQ2Q25f3mr4mpEXTMTKSF6IYFHVFH8VeT9K0VEZ8cJgJLfpIFvwEhCi3WZtIlaDZefxn3eGw2nROib8no8Y2+iOe+5AJMyti0k8OT+I8LCbNTAo7q4xWuwss\\\\\\\/WfiA0qDj3DuMDtD1l28Lrzks4\\\\\\\/Jhc2S9AnZZQpFZSrVFeA8yw7cjTcVgrCkPSZrUBigen\\\\\\\/skE3q\\\\\\\/hBzWzYuGiQf1cCwAujUoJAuhkmyZLdayx1h2QeVPMuZ7S+Ii\\\\\\\/YVZfzrQmKEOlFSyu5b4sxUPNAofxohQjv2GeVtbcp59svl5hGyuPgTto7b4OvBHnOs9PonWBQ1UPie8KeReWbJd4muQYBEdKbanJdY2jNS3L2xHkXm94KGlBBJgnJS+SoeHJPCqWo\\\\\\\/wiPzvHicPR1gVA7iNUxgRUEfCjSNDt\\\\\\\/ixtbO\\\\\\\/XKaSsK8D+ij2IxkhAbtm4+KkFvIZ+C2\\\\\\\/p4bAl4KCTPKiMLQw5m7apfOoQfU0kX2Hkq6mr9K13AdZiUxFZen+mWj0unofavdBfCYtZ82PzvDDJaCUeeTWzei+rKDWBUQ2CBsEsRM1zidVVxOq2K5R40mN0L\\\\\\\/Gg0DptEHAYb2pfa8VhzQ3BPsJhrzOEqRCVOP2Q48aOkA7obLIOTx8z+aq+DA4etpdj02ZO8LYIB2mI7T5yku5ckoArhocF9ye6llpRQu9gdwxpeKDlElHMyQk+k2kPiZXvhMneSxuVHcTTfNZHm4pwgkkZkPueZK4GTVjBgnQILRFbNm2NYDVxHXY6kCP8gA1j8sBDP71dLc4u1ypwZcstzXyeiT1RFzyP3\\\\\\\/qmKV9Yo3bYvt3vBr2iCWWVh9IWyzLnKll6J+JJc9cFDik6Knh0dDE1ilG90wEVMpRyzJwcf4NkHbuRDTWZO4RiLZ+2XlOukzfLuGNh\\\\\\\/vG7pdBr9aFdR19cTFHGQgDvD4V8UsG0FuxO3wDB3nr2Vi2FwkcWqxaoxXhTAgZOSZkvgnMoXx5s8WDX47\\\\\\\/xhn4aKET1Mlvwf7fh9wi+oa556tfjMawUnjAM5lm8cnISL3iG3E1Pann7IOb\\\\\\\/6LrnNLj3cnNt0nglW4JXzv8PBL\\\\\\\/cdPHgp7gozaty\\\\\\\/mIIHFUzVIRNtpDg2EXxywqA7DljqpClDycrTYhXzYut0VIVgBO83F3G\\\\\\\/DaTBxKMNlxy3csosol7puwE5cpEsNBQ+YLccWl\\\\\\\/ZzId1Jot3scBSXArgeZqJ6rzg19T67qYpA0EhIhSXt3M0b7wRfvTJjLAEZkcoCPplYQ8Mv\\\\\\\/hSF0+51fot4JPEFbsZA0kp79ycVpwTPGn9qs6b43hRuxH+VkVhZjeVFHAuahmIEfjQ8LeX5Mf931vSaADOTVk5n4FUxmje8jn8tXWS98Tvp0bnmSXwcJrPeR2dGAeRhibCvxhj6c8zX3LXD6EzzRvv+exr2lPj27YDv0av0fZ4oqEPZ4pwArMfOoszAlo+VAbDvgxa07vBc90C3P9WgJDgwDL+j6kaHzv8AWJ21qps3hHH\\\\\\\/kZ3MVRcWH6R1tE1D7ate\\\\\\\/myoZO7R4GLHNz5QuFII9hs3rmUaBIT2nqEzXLQtOTXXkgmaewg91WUsQMvz2MISAAMIKL6lnMeTph50BTvFquIKd7Fr3oNlLDMoKMWx86WRYyALNDGd84f5pL65zB8gIEe1xcU4IA3M4yJ1fO6HBuHrrbCGmlK9SCy8rUTkQSAht6PMu+5m5hSmNfV0o2HevS6y\\\\\\\/MSE2r8QzffBzMAvWm+RZZ+n+pa+epO2zTvA1eWKOfMg76rTITZBz2H\\\\\\\/622XmsGTTzCILTtBUfcaCQXS7xnAh2MoBT06yQlMUhf5W53+sJgiJKN8mNKIDBvVpCNijeWXe9nP+1AYZ\\\\\\\/w29Wr6FdZql13FrVS47TV3+5opWjKCwhvvusjec9aHfuuvGZlV1S52E1Mrlr6IPzrKJntEJyBvAhjkfnFjX+dJPDaxJdtcsNejbs2XAaCp67jOl80ILMuvoKaRvbE+U32f4RS37EY25024ywGRe0gBtNGGMI8n+7Ggb9asOhvAkgsAOfV75\\\\\\\/2b57+MyHjCraIRaCJW3cCM4WZnjoS4+wXayZzY00N6SshEFOUYM7uDb0KJS\\\\\\\/pRZCW99ZMqza1p1F4X7a96Wlzn+YZxTFHKfAOpURbcu2jAMgDphQvfR0bVsv6NfZnaeZuyIGgl1UxKt3BHyTNIhAZkt\\\\\\\/5Kqv9RcCVLKjR2by0B85bUby4xT+6nwXM2V15zW12dNXXnmGA2jisvDuyBcKZATSXS0DugAP+cfwzrZHZPHWTIwUMKtYUbLAZuXq82HclkfDkAC6DMRnsD3wsIKqSI8WEFZq8hs8BDpESDEXoFGkVhe6\\\\\\\/eI6xCCtEyrI8A03FIsazLr\\\\\\\/\\\\\\\/KvY76N5A3wBwB+rgTCqjKBtp\\\\\\\/sYcoyjzsIMo96B8AzEsxtr+LP6\\\\\\\/j7cQ4NsJC7905W199LgM2aHxHaGJGGQ\\\\\\\/cganHdAitaF2nycFpAbHvzVe1esQhfD+n6k2XJqAiYHfvZwgyjiObQ22NQxaTGpuuGb2J\\\\\\\/kqhTZ9rqhj1FhDUyDnD3jLw\\\\\\\/8wfGU0b8TGmObDsDgBgNjGYE5TYupMmm6qBkaxYQoUDjao9eOM0QJJs6tyFEf2ybg\\\\\\\/QtpFLzxEqgCEaVyNr5BNWl78CyZf5Fgqa93DVSDR5dnv\\\\\\\/Jcusy3MYW4zMAjEjvfUxywTcHM\\\\\\\/d6q+90FTuf04kURzQsUiGBuKTUsiYCJgiKomEHLBDVwdqUJW4PG+wqQgmipmu13OuA3GUNjUzc6drWJ88NnrrqB7I\\\\\\\/nwSIopTznWhNCcWGfWBQiA0WwzbxWGa7DGgZaO3h1Oq1M9frJSHgw2UWzN\\\\\\\/uICZBJPSz5pEZgB2IYZiS\\\\\\\/dNxjyt6IJzsYzhhU\\\\\\\/3OX2Q9iCGIOEbzktbKnktoMMJbsza9bSr95g61extQY5MR2RguE+fjAYSNiFfkYV51ilVHcAPEMu3IQqgsLavRUGzNXy9dWJuY5poq\\\\\\\/0QoSXqVQNUrNvcTbMmfdGJbPMA4p8fCSZgkLSJt0pBEqTkuJokx3ZGiAjj1hMN5Ei0Wbf9tSxUERY8bf\\\\\\\/YwSAczjzy\\\\\\\/VV9QVXzGyJH8zpOmRsgvTwpPU4BclD\\\\\\\/SFa2y7qiuEKQbZWl10IrImchXPplJSN4JjUTV4fARnugg08euck8Qiwur4mvcnjx6uAOvljcz3yfaJnhFnadUYeEgmcaKT7VG\\\\\\\/\\\\\\\/DBhnbgYij8v4cEjWug7xx+ULbduzeI9mkcoM4vemG56b1sq0oEoIlJgrlKu0zQ8DRaTDvSWaBpaXek3t4USMKGdfKOXpZhq8seUFg5YEUhsKbcVgK6ZWPwPk0N4g4rGDXjX4RNWzRB+IhQPY0zX\\\\\\\/wHaNW3jfiesDc\\\\\\\/bO9kwQ2jv3QN8oJ7EgnLMhAVutALPLisJvnwR1KTUl0i+u+craB83ZtKmf5qNTqQkiEzSkGNegs1LfO2ur9xJYxPXCaLc0mycqs9jx0AzlHP10\\\\\\\/GTpP9BXkKeSFEAFf5z1tI2aHyqytbSUw0P\\\\\\\/mcMI5ICsZKNPRbEOkOfCsjtJt400dv+BZO+uviU6rR\\\\\\\/RPDu7W6kKG\\\\\\\/zp2HqMabE2Wu6wvcqlawfT7C+tKbpDiLYePZllMsZGmLaZaOEOoYuqXhGXKTLtO1X\\\\\\\/wAr6yiojeEcP2BZ2gSmq3h4UmZBDpmQyA8kF5xHfYTYxGcnvQ7+3UwgSqGmuZ5iZqprELR7XLe50mi1Qq+Smcbb0XcVsrK\\\\\\\/KpejcaKR+Jib9w75qjdy2ElsMnU1E\\\\\\\/4CArLmd+zZW6VtbY0m7xlPtQleUrHz1ZPLcSvEtR0kkIpXOt1EYAJgAMMKHhlJFVrs\\\\\\\/7jQudJCZ\\\\\\\/mT80pZzVhLqv7yOmwaSs5m9aX8RpAX1gA2Mh0l+Or8rQdjcw+KLgjYrTtGWTSADCJ5q9s3yuDY8gsVcqlKb5ZFAfvcoHJafHTXfO0ejwALKdyrsW+9JEAMiSlzRolKwfM2Ouh4PxLMnhI0Qk8NvkMKeQ0XCnMhky3HkhG69S9XoVLtkgauzn1aOynuKYJ7DXh+D76f50VrqLlRr1J0Bi4a49fT9TMccsTH+5TLt384sTYzR7v0xPVWZWaWLM1Nt5dAtKg+q88qr6NmEjLsiVDWrcmscx5X8vgsP5lWpT8ufs\\\\\\\/3lS9w4fzmCCfS+JSMGbj4pvYJIQE7YgSAn2mVRrwl3SkA1rBGNIMUmypWwoxjnFFHzae\\\\\\\/1wwGYgvtc6M6FewmxKc2jpmr1tR2CV7gaxNbQT2Ni9l4cJfGMF7YO0cJEOfysiihdIjOzCTEdrU+tjjntNeL91UW3j3t6yt31hzE\\\\\\\/y0UCSTnAEkJ2j8qiRu2tIuNCAtChq05jILGu+vj+DLpBepORLB\\\\\\\/G0+pplXHTarSYnMKEYLXtBo\\\\\\\/Dvh+xogWbBmEQr+O9z5u+\\\\\\\/Wf9opl6WNdO5V2WdomWKZQ\\\\\\\/C6nBzlU5xI0Tv\\\\\\\/vqW3iJSTv1nu1bNqneI9FuVvTEN7C1hER6SBxyGXSDOycfcS1qUAPId2RbV72N2JrOVmAcQHnN73ZlvULZusQ2JcGKaq0pxsdZuL4hC3WzFF1yJhNbkFqDrMqDQZt9n45ThYqsXvzn8cMK+L\\\\\\\/SRo+KODNlZG36nt3r+ZgqgJtU4lri3gQHNM5o+J5wxs\\\\\\\/3Lqv+enooc\\\\\\\/pecHJ4KylLkv4MzdClPErpbPacWwDtQqa8QaWYukgjm8Qw6f6aQ1\\\\\\\/FexCotnq9nEl9wsOyfvdTl54SQ9HoIwRF44zeCiDDs8yQWmRrGrx9ipYvXP8KL7goITM5TPwiYKCgUSUVfRlwH6df6f\\\\\\\/e3CNt47SQUsMsXTN6GBDYpfy9BbL89d4cfw3tKcxYlJMufosLfh21cOUup3lKU2c2DDPnj4rVdpmeOClD6C30IO+A3t5+9efbVUPbygY3y1v0RqgtYJJhZhstUKOB2mP8pQSBHGooSQbkpxzs5AQZvckddZgicjW3dkynb0gj0Gr19HSATPZTA7Rv0i6TOdfEgHwnmm481N+I9xooXNKzP93hODBZkMPVz145fKDQ6IEqnnDPKAG1ylYwmh\\\\\\\/EFvOl7uPT+NqjbRLhQsk1ZBs1BwGaH1A1g5D8I8tgm6iWM5LPlu0SUS203XmXfjn1+vrBFk2vETsnI8ypWvhJdfpAbNjDKFV4UsRzp4yCmOT790mo9Js\\\\\\\/UpJzuk1WJ2jvg\\\\\\\/K35G+Hy218gRDRMZidoEwTmgT2fWyFF63E0kIpz+s0z6Tyqvxri7qKcD17au2CjiNaY1iNk6j1ZqA1ip7n+b5z6fQslgAZ2eDR3eljJXRmfgY+NMXlUivSPYBIMISgPxjrP1EKnfKrlJThgx7GCZpbaeOcx15Qkp8pNnpk8nJKTl7smGO+96thh63TWE2sQG9s2mhGlgRtINJVrUtuAcBckqNFY7gBDaBZ7LQq\\\\\\\/\\\\\\\/2vBdK\\\\\\\/OTY4g5mFaq1+mVh7zy\\\\\\\/cjU5WgXEJHE9XcYQQigs4FEXZOXzR1Z+LJSI12kObALLIRuKq+y73d+ejseo71fTEdn5uOazZ9wxriTffwk8a0pl5tqRnf9HCihT60YmIL2G839noDQZXZDhBTFWpZntztXYJ8luJxQPopJtZDNicdHpZ2ICi\\\\\\\/0yQUOxGl+jORQ4z28zL\\\\\\\/OYLrvUt3wYbn35asMVVTGtc9sRFtsr9AN+qvRRDmwaukfU61f1a\\\\\\\/0ObJMsAJXzERt5qZAUeFptJBHOmiYhaY99M8vXrgDZI6faxHEfDDjjdI9ncU+mHpH4AHy22ZSMiZgNc16T4JoyUSTqJB9uGD6MCzU7fTkGR9tE19JkJbIq24c4DRQUdHYFX3D3lbw4YLXvj2c5AExSZy+JYSy73eIhHqzOZTBMkanB+HCxYIfzaBA096Ev4jHNJ1iXsXGdyqWDvbzXVrDn2Eph1IscZ97+c9vdceWgwwkTUb2XtjCFl6SQ8pfDrp9sns2veyoiZaEgYz3scPzpxCFO8jTcszZclWOpPXeiGZUG2wbONXIr7ftIs+cb1T\\\\\\\/ad7qex5wnKh\\\\\\\/r\\\\\\\/M7fSrTNPfnwvz2ETTrFMHaCCxxpmaGdwLeNTuo5qLWdwiAwXTii0JYOZTQsWUtJwgYt\\\\\\\/DsaYY+YFzJh0S4jayWkRVEdWMil5efo2Jys\\\\\\\/teHAQmT5n1o5B\\\\\\\/66guA+a9ZuwuCzaNVD0XMfORK32OMGPXFCrukEFNrPAAiv0v55U+6WuNqZeQXxrClPW0LJ7yTP5f6IrAYL1a\\\\\\\/JRgGHtOKC4vDhKLg8nBBT6sMqY5xAIBkNb8NdAUtWnSvUORVrNPQqJJVDXzMPyfJC2pVeDAdMur7t6w2RWd\\\\\\\/lzVN12UwVLM122s8xJ6VAV47d1X\\\\\\\/3rhVQQfgk46IxheyWVds\\\\\\\/MweyM2X0Ggrclm0FuYO0NEBetug6eyz5YRd5lEwu45pjc0NS9B6OB32WOrcEhpd2xF7WwfyhfQtqnF6X31kAHrfdQB4Y+CMeuaZEh1gXv+zO4U3zDeYYPo6ooQs9m8mf6rzXLvO+PsHkTQcGq3QfY4Hh5S\\\\\\\/s98i0fNOokzgp\\\\\\\/03xY38ftkAzSV\\\\\\\/E7EFou+GuJnRjGO1e4TNeCJNzfBHgNhdhrFgBhT\\\\\\\/aq1sTCf3p\\\\\\\/GWNY4DASndCnrn2rrG0++egG1vpDXK4xiDm5Qjdgpo32ioW+4rpHRekaJlP+jDUU7jDPKvEM3DM5TxXec0yaSfv1LpplEv2iuZBbRIlJ1vjng6ACI2P3EyDsceZQiGZxAGR0KRN\\\\\\\/ozlabsQ3U13QCKJEV010k6FZ2ii0TyFcjlnbocqiqGab4POMc0JRcxSZq2wWVNfGyobLaB4BatIkhJsA4B6vEeXCNefvMerXLRkR8MqFSF7QbbFk0aQ8S0xbTsB3d1SZ+KccrEMXsjMvXYjvlB9otgYkwrfyic2fifzafBXw0GjcrBkPha0yFvxqTt7ut+N5Y\\\\\\\/O7DwRIDj1oZi5AszBit+EY17lzOTfAlQ21Lc6Snbz8ckqYhi1VzCEKfNnzNDB2FmEPkws6PfXLa\\\\\\\/2oU9nU0Z741bNVlCywX9fF6mabOBZJvg1m\\\\\\\/qldU6eXydgq51BayvOBRQDR5U1VT\\\\\\\/lbP82GvIU+rsZaRN08hB0+Vr+ySwY+g6WIwjLrjPSSt4nFuJ6YIETowUzMllADWTLHKSXLQ5R0gTS8+G\\\\\\\/6fRe7MrITdWKwVumaWbQ\\\\\\\/trWUHOC7B\\\\\\\/hzfQecrH\\\\\\\/QgI1ou1bCrU7jalkH0ruXbQcn6AKZXl891T9I2+2NFq\\\\\\\/PAljKjpJ73a0MzQTY4bY135a+etni8jyS5N+LsHV1Rp2TJqVVNqDD2JBd+cnNeJor9KUlngTH8Mm7u4A+dGnKOR29y\\\\\\\/895uYyROtsYP6KvqkcAmYygbb3v9NX1SzdAjofvxhXEkpvgmB15wDCwVjBI5md6\\\\\\\/IyZITnICapEBRX38nWogJgKFgnxCfDzmaAUeZauqKIXGvM6NSvNnqfahUsF0+oasefU1N2o22t0\\\\\\\/+nxwpr3E0nz5e9SFpZVzuQV7TqRcB1tj3H1XKBaHrPumK\\\\\\\/L6zZXXJYTeXUKc+ZVv9bROtMkycR\\\\\\\/38\\\\\\\/hlm1GvnpMGZCNt\\\\\\\/\\\\\\\/8rgfH1aBVC6vqgw7nmQDElDzsTWpmWF5xhKNivw+Gi+vaf5Or0E+5lmb7TDwQptpJeuNC7Qc2qOmGz789KN+qlj014OjfI6Dc0kyxo9fHCcxXTQGMS2u5m7u3GUj3wzd6CElmJdncQo9WAfVUCDwb+BAgHfhSQC5Z8VmT188Kil2XWgF6mANxsuM3zzwie08jO1RHeL9IMjFdpwcKePK3wg207qq6X4AChwKVW0Q6NJjeqJYdSJpY2GkMjPIVXPRHG+rbjDqGbOAJqbCy7UWia2pxi\\\\\\\/bMdmtPhhB+oCIaDAtN5MNsI\\\\\\\/8\\\\\\\/mLNpi8yrR9gNTuFA3JA9enrQYaLKLI18gZqGwAiFsWmYsyhN28axWQb8p291rKh0E0ANxvLy94WSPYIRug3u8RCN+fTxCAW276aCJYJLO1vXGKt7lsjGmRdAxXr1C86O5DdOO8kfermrW1PErBpj9JQhIXbNVeFfXmqWAqH9X9c7OUKq1sJ0msRTbvR1Kz1E3qQn6t4rymh17jhSlIXmnxyQZ9OoMHLkUOdMGxsXf\\\\\\\/87TXw0mGxYbme92l7YUwIdcOjSWOvI1OicdADVEZADroNpQJ+XVCxCKIQEH5uFEz8n+UvhsEA8gfDOjrk6uXI3JIzZOOgTaJTphLA7hrys1UFirfYLNYQ0F0M7Kg7MaUAHwUlGe53+\\\\\\\/oBc6rWSlrq+gHqf601pGrtDt5uL7hfNwOO+peHL9Ou0ZsEobiupn6Wvr9rdp2Te2C+ZX+2rTKtpEx8SRgnnvZ6I+Jg+FkwdQ5RE33R5fcoy2Z868NKmaunJIUoDI3mCTENW+BNJec+K5oiTtpDee3W8lb7E5HsFI00cAtLqoT541K3GpowrckD7lZ3oBQhyfsqV10M\\\\\\\/KpHVe\\\\\\\/OpGXIRK4CGBSvXMuX9ldZIPDHghyzg6BlJnst4CrDA4EyJvcB48q9Zf8B3Yzpqlc\\\\\\\/MpmUZGEVjBTmwVIIafnCKJ3gkkcYVeoMFRHj2B5ab38aVnX8J8hkxOWMbm9s4AflHfJhYaZtbUK8OE6HIQeeTrva42fpl0fus5yNGfZ+eYI373i097pVgJb0WK+vtWmE1SFecInuEALFIb0NB8\\\\\\\/2sjGOy4K+Wm2ux76ukcSUKnKVPef0N8SARUf3nkkTsLWf0tFYDqluhLkZyI5p4pP100JI0h2Xk3pLvvV7COXql9RzQuRM42VtmMiJ2ePcPoJhzJdmlg141TjrW9aPOB48sSyknnRZ8Dsrbo8ApOgoyoCr56K3PRX8q89r5reLk1DEchIzoqr7l5q\\\\\\\/d9AR++QFL2dUK9kLPScIkdqs\\\\\\\/LnFlxUyu2BTrWfvZq\\\\\\\/D2Oan9rrZt4eSZImoiD8Btj8iTXVYqMeNqZlCJK84rNfOnpMELPF\\\\\\\/eqCeTOnXcr0HV1Okf3ynG9bHU3EeR17hnoMqzC\\\\\\\/lDjihpTt5AekS6sz\\\\\\\/mQw5Z5ew2hDOE6jIZ39+GekVT+Oun3S1M2w2be+dEKYcl+2aQZOvTn3mhjHR5ELARf2mnLUDwtpmBhXzbFnOlbnmqjS0vnCE1hIeRSuEmYIhcPHs0yphPYcrr\\\\\\\/avQ1co7UtMRSqqpcihIliyZ0+ifUNhhNv7A3PPQg3fq\\\\\\\/lQQx3Is8h0SyxcDygpfXW4d6VzOizb00M4nCDSIBf+YwW9fNs4fqLp7Kg8ZUmc+4GXIljrPoMmhhSRNbJPRZOxQxx0H8DWypPoZ7Njpp+C8TaJUIVLClvacJxll6EGHgwZENDJksh5WS4m2WxH5RbT2QboWVNUq+WOfxFzAXoDE9KKG7102Z6KtNJaAm9hDTzVhexU6jaxN3srd39prYTRz28KyXeUsumxH14fjLkB6K1pDYZ7w8BP+aLYHSXVGGa+YINts6b1Aem\\\\\\\/oYvH9+7nkZ9bt9EarpvnCe02lhbH+mpRC6jEXM3WMT0qUrbvP6SWPnybMTmFSxuINdpWEHBSVbLyFyStL2D\\\\\\\/gNFoX3WW8TpSvQnkWIWRfITHh1aIK0oYj0cAxDOQFreZ6uw1jJPfYfoUKTkccNw\\\\\\\/VP1+oGKMy9v96N9RlWfw4NeTluWK10FqtokMBZ9Wda1KxbN\\\\\\\/BwmeFa3Kr4ddb5\\\\\\\/cyHT3tmkmiHyNKHxAeVgEzGmrPvZa5SNt3WENrKlYLYbR9tlDkac7OcQ7Ryq3bwbLmw787k3S9oxbh+ueM07sn1w8JIwKb+II5x9+TJAfeIYUHk1kvHYz4+jNWphTwIk99bkZVex1QBctN7ECvjk+pb3CxBuAi05WaNYMMQt1V6hb0zYEtNDkhr19EEgx3QpIWHvdDdVu2Nj5iD0y1Y+Fx9TcKEXVoO3Rg6gc0tyK0ufHv9m2qNYjucn+PFPHxatGqcqXtZCTbIsA59x1pRkofHH05fX+c\\\\\\\/yl+1pU8XSmjuM6blXpLCA06Atml6hVT2p1SYx5mipo2fYDrZAcRYFfoadpnC5SDYy3qPg8SZcJBI\\\\\\\/P0+RVPRVgvAyDot9JvNt5e9alpeKlNVmF77Iut4yEf0s5iLhD2hZg99C5IMfIIbGvJl7mGYeuo6CUgkyVfVdXs3R6hiLVVdcBxOiOsCZqKksRplp6u0kkxhTzGfvWHJhS3wmt50zlxaTZxNXDyZEXIaBpGtfI5nwI2x6TovFNVTWCKIXgfAMKp\\\\\\\/WvE2msltvuerRtF7HyPv4H09Ursx1KSdWGH2njW5aldR7Rl7pN0FnqeFufAbXR881A8U8CHZZ6QgFPw46TFHhGLFsbdEYRLgHwnoy66Da4WsR1nYEwbaJH0gUNw2aKIZGNFFuGieImVFCe0l3EAeomiOnWUSH6qDwJFbfIk1qifgSVfjKPb5jszvd0+7xMjgYgHxQwFYDhlpH4KVVGNri\\\\\\\/kfT455JEhOlX6Cd536VmVbrBtKM6ezQCGxGqul6qUG8G\\\\\\\/NvQYRTQsFQDZrN4se+jAeZDzA0FRpQhQZNZ4xh\\\\\\\/dF55XWm0GoArZKP++EKEmNVbIHhSrCoQmBFLloRnyeQPCu4gIDaIF3GWMqXsqfKNYLyFUe4Hm4rsMLJaIAaqkCnjGLr83u96pfvEbeNzd+ZIKKkf7EG7um7aqoCmFT8\\\\\\\/6E1P3Ir5x6bfnWveAkHHS75+Ur7AkFKwxhzuIjDSQa4QD8rI\\\\\\\/HoqtQFHiPukz84VHbGDkX3V07mcpuWeUSES6fB5102hvtElBRp5XJ6fYMdOTQt2WSDVANMKwV\\\\\\\/PljShE2F\\\\\\\/skBDsw\\\\\\\/DBbXl\\\\\\\/OHQ+OQs\\\\\\\/T5VyomeAgtmtulvJOxMyGoJanAWoPv2\\\\\\\/iTH1VdOB2m1Hn0Q6vG0HsF6DTFwyxBL0F0WApaDZKkxLN8Ih\\\\\\\/vN2nsCRPLypGPgyDtXVGqnjNCB+dRQV61igl8tDrb5jbGdeZCamF1eaxr37DYpOJ9ijgwsVYxt\\\\\\\/TnuUmu4pkomExXcAMBmumPFNuJXO7h2oqaVeeTP0Z1TZqbz4\\\\\\\/l2B5mByAeJTqK1Di+55hhbsFk5GSMr8LJSiyY23soU5Cg+p5BRt\\\\\\\/zXncpDvgenuaUp5iUbYx2nm+EhX6WT7YX5\\\\\\\/MOXyWKAOHgAMmx5n1yC6unsd9h+a\\\\\\\/KGSeoLU60z6eiYxKQ+bkw04ifnO6hNgC1OdCsSWfAgMITH8O0y\\\\\\\/QjNU7G6BnNAqnvWkY\\\\\\\/XH9gcNPMO5vwgKzyq9uPaF7F0UweQES3YGDhwCTMY4q78O+ipylls5B9dd3Fq138M0XEjClZO+FREPR86W09rxEYnvTTE7ZYTOL6mtV1DMDXqPWBg\\\\\\\/OqjGy2peMk01b5GL5ozD+aqZyLAdLf+xo29nplP1gLJC3tCqIuJ+r5m\\\\\\\/CKEH6FH3+DyMflP+hg47iUDwFY8BAqW9UtFqra1xFpOpJDPLBNZqOvefDu5Zrt66DVOOflXj7x8pLfMYwfIimIHZ5G+63He2vrddopHiq6SZROs11NZMbXbNNeSI5MpekHZYOdlNn9n2tkmY079PTD9BtaJWmKKPxF5cX+3Q+E0XdA7b1ue2WfsTsqAw02oYRSXOc2wN59dG742TmRXW\\\\\\\/xviuBBbnVxx+YmCFgNA4tEQrh31gPRinOLai+LDhH4j9jcCKE9DHN6zxNHi19PgRztLTAYdi1Plc7TB+\\\\\\\/ARsAZIScVUKCZONIaQ+GzFbgqdf++AmhkLqBD9mqwSpaEC4OF+yVI\\\\\\\/2SmKdVSMDzZwDageJtmcUwp6pM2NwMORYnPTHEJoVDEY+bpo5gwFGYfFM7Q5PvLNDBHm9cnzd2Epuln93yoFIMgsMGB6YRIEzTp5RJxLcGRDggr7RAUPDOIaxFSZVt7LCaNkOJx2a1MOC5o5+M4rROzQnZLYWY8yNWnC6vbipST888RlNDGxvVi\\\\\\\/HMrkU08A6uxg+qvlVQTHMh5IO+6U05B2isBhP3JmGCWjn4UED\\\\\\\/i130ViIk1mrfTSU9eLe5JZ8IVElEJw+1GxeI9jtgUr7k4OfFafaDBoEqZRWSSBWU4C7H4du+pQs5FMMdhgA15riR1mWbrRQMrpsTZCYPggzJNrw1SoD4SgusyL34MuyirFGdHe\\\\\\\/3YLGmwEHqeNXXXZ+hhdUHgdIcoaubCOA7nVAWGy6jTvC4pXDfTzHjMzVGz9oV+XaSIp\\\\\\\/rxOMASpS+ksdboINc6FT3SMHLNDXEg\\\\\\\/L9RRWe\\\\\\\/ao6AVd6wTOGasxcsxwUZzv1JKzTUF+Kmry+jdgcR5zRwAbzr7LYk4VCVAARbiGVxcbwydafxqGk\\\\\\\/lQrAe6VsXanGvEnAw7NPsTfVQJYZncy\\\\\\\/+CsvlHNp22wbGDfYjc4AzBZ4LAnImxy9X+NBRPAZDGng6bqXZjsxeRupzZ28CuWDjOkJIYiSy1nXjo6z9yVcglXXLJs2WAdAvdM+CJAky0z\\\\\\\/P5L41YeE9GntClLIOHIl6xVEMoty98Zm8kL\\\\\\\/whWE9H7+b0lZE+3U+LnnSkMLhPHVCCdRvdvaxPD9t3gi+9hxrMj9j5pE1JpfMup\\\\\\\/BPtDo0UHtkGgl7ZdWuCX2XI+dk5T2N2BNDZPYBLZ7aziQRSEUR1e6pUI59uCA5BKWeKzqeZNa8YlU7TGHHD50p4x9WdermrPVXcTQy5me7j+C55c3PUCV+LfnbbziCzMDLf4k\\\\\\\/EwWJTapd8R7W2nKKld5dJccdYU6nDPOzfGNVhzOKJKPZXNHpCk\\\\\\\/Ua8qJUhAlvwDToScqgcDGpoV6AXW3\\\\\\\/Pq9+JRtQnX00nrzR07NVGCiHmNnVYQitO+1pkhkAzNHzhM5HQaAKyKn3IYp6Kmrfzthbq2Y\\\\\\\/jE2FTfzeg8aPzNSv0hpUFFJq6x1nghdD13bsjcxsj4v+9gB\\\\\\\/YUYqA7jfQVnf4wGVqqOYOGt29AFErxbxcfMuwjAxNji+P9NxchUy64BZoy7hlXyhpjaAjeCHG6pp8vBlVtXzs00CVkR2gsPS2uym0Wkbc0JgZxLSrdGF0cT+fcriS\\\\\\\/ckd8iXDmXpM4yg1c9WXO5JBJUlga+hWlTpmfmrXptGiW9mxR7B7GjZhpTEoQWL2CoHz3JukZRFdMd4CwSUuCpW+v9oA\\\\\\\/J1jMzyXj5ONvfU4WYD459SlOC1URY+eFWE044CBqC\\\\\\\/611JKXippbmHSJWj0TDQfJTAneHqecXlXCjnIOokva1xS\\\\\\\/KaMdy4OiX9DhtHVWkTMknKOvu15O9fDefs3bflFNl5BMO0I1jY1Tgl6CLTr5G3Q2ImqnDp3wuCIBoCpS5x00OnulDWs42xHmjcpTy78WWI9wvOk74neY2dPCvH55jz6Fz+B8J7YnrCu1Ah\\\\\\\/Y0p6Rub\\\\\\\/OnOLD2RiBnKQCSGOgxBiJCqSvP2cGUDkCNArB\\\\\\\/7JU\\\\\\\/kd\\\\\\\/xtYTZLKn9QeT\\\\\\\/YmDQCvFF5vhMDhcirfOGckQvqFYDyMT+VPvLioismOYSYWZEDGyn4MCUKcNd3I\\\\\\\/1wvJ8qAmWtv\\\\\\\/K2mZLfaxzXD7A9zJh8NZgudAsNSKaX3TtiWfYj51FTbttvdMpIaJp5nRhzm7uKoqsqQpP\\\\\\\/e9k6I5HaT9bx6UAwtRhnpan51xXrPO1CiU0PdXa7Q5JMSQf6ue4+yjc7CBUHvFrPi72ADOF1g+HWinpA0JYC8SMZttHR9ZAw90drZacPw31SHjRD\\\\\\\/6EKTEd\\\\\\\/uBeVXGVjnk4zHlC6YKVaYviVE0SVgYh6wItBkEg6LUjg3h2CXS7u\\\\\\\/Iabgg1WzUd8QfDKUhV+zxd+fSPqaq3xuqMSVGuW6HQl1qzKYjXIaVdOztqxqiW4hCt5407ZAZjZhEWdsjQuHBiKqs7U3vGkeIKCqn6jS+noVpi7mog0tFe2sOyeGltLqCjVeQsvpa\\\\\\\/ECwa+rZ6lTn+BN5euoFh5WFMVclruNb2HkjNVpfl+gzGjrQ7WjdbSbaQvluggaFD1UD0EeHPUP9YVhyUrMP8pkfZzP6L7F9XQ4rp+AWXx4eylfPwnmqXt\\\\\\\/IReNwQLL9b4ftlClGFKp0gO84fP6RLXUYOwXdqna3Ovn8TGIg9b5TxxwGCjXQ5WA9ro+2cP2HSSChtC9J\\\\\\\/+HdqHplZAGoxsLoatgbvkiQPOFtc\\\\\\\/RrU0lhol+LUqoen2d1dyuF0TlvuEuTo9YwOsKCqvQz7H6\\\\\\\/ByjWTgvJIHd1aALwoafCvI6yAxGeeeiwPLpnOuMHFNxdJwKbAUduMauRdWnRLyrQRHkcTUhzCWOrvGbYFAxlrCga988dXcpl973o1I\\\\\\\/cfo4IcvuolX\\\\\\\/9mp3pDhr\\\\\\\/OgmoD2ZLN\\\\\\\/LrD5yNXcKEWdYjbY3VUuY0vWlnCSwufdmQIBNoIwXCfzKyh6n0fRdDz5pF97V90SC3Ogu9OhjREpiiCqkxR0DVLykwkfmo04vYxUJYQUVO8wq1xFB2bEkxpm5PqZkvmiIImCS1JOVgBD72Gi2fPLxwVqhxmA5X1F6sFHaagJfnwYny7O3nIj+2lXV0BGVLUQVqSMnGfFZ8nunLjQ+1jeLimc5FyzA6he340175SwOSUM41jActQqqjS4GqLRN3NcRIZql9B6i7PEZuKbbQvZnrIdezFNpFV69K6TUL6UPrPdv\\\\\\\/QWCZchiv3BM48UK4mxX0NjRqB73JdLPJcoF0aLxeX\\\\\\\/erraIrWSeCmB9wAZ5GdMvq0PZKk1cbGehraE1mtqbJTriVXnjlqEVY9t4+3rr7Fr5TVT3ycqZO5mmsTCIMvWN4l6P\\\\\\\/Ncc4tXcQ37CKYuIeoFhzOVx0q0ZsoZZyHNzo2UcLMuAxkrUBuI7+sMMjvAEfmAM0KxDvOn4DArEZl0P6edlu7uJW1naJx7JeC0ZYP8fHvMlR25QcJtqfeGYni53FnFmsnVeNsWMHbx0ibURina9T0YXVG0whd49Y4JCDoKmIop4mnkZYEguKvM5QXXbnZX5vqKNJMKW8tdvNrKyIPx7qPNkF9kT2iwDc5OdKzZgidNfK7jXinrz96Xjt9sxCPFRaCLWKDRfNYO0t\\\\\\\/nq2ApHVq8Teg\\\\\\\/1QxLb+uPlbbMRvuewP0FFFZPzQ0Dzfj6J1dD3OWL2NcDGmErYB73cwevpxb+cYtj5w9aYblGIhR6qYpqfD\\\\\\\/mVS2tiTot8+HQC6uRA9zsTi1TogDgutG150g4JI60gxw0oY2aEEbIsQdnN2ck0VneP7CFQWFAE2+d5rtePf9Z6+PNOj8CXsLkRlEyEnjve1uEXXL19D1\\\\\\\/3fVb32wQlR9+2idqsbDsnbQC6y\\\\\\\/tL71my+ozKYu9\\\\\\\/aVsKh8G+\\\\\\\/ll0tlfj+gFX9R\\\\\\\/Xnwf2q6yxtCD+0nOD1kMziDqwqRKYrRpfihZc5O7HKcTDyyHLTysxNkWhama8AV09UdrrZb3FniXCYKRdM8h3XzGFeHMxWbVQI+LwlLP5Y8DGywHk1ANQCGRW44ixjndVCK66TbbY\\\\\\\/gFVu23zfW3Ah9gy3QvhtDWyQqU7vYr\\\\\\\/Vc9DJD\\\\\\\/tNS9en6LwZg2Hsa\\\\\\\/8ZnccudZaU9qnA\\\\\\\/jNjpF\\\\\\\/uDY38tvhsHZZVN3d\\\\\\\/INHgi0QBGSQ3GuOVSEZUz7H301EeyA2s0Rb0\\\\\\\/lq23gdBvsVowpD3qVq+zPRtGd6IkpeRP1V0IEhrRSY5OIsEgDlroSAsbHiSv7w6WkMNZlVGsVIwjg4tuqN+\\\\\\\/r5qQjGMeYMqM9UgOQ7On\\\\\\\/oACXJFYsOuyeIyVhDWkLfJfVWgazskeFilIdsaRuFBQJsbhq0z+DC8trrRUcIRVXNT55leH6lWFmbCnBqo\\\\\\\/0599wHxMwbKEcU61pjZWVAwCukGf\\\\\\\/PvtUq45SrJmtpim9BUpwR4GhzLJerknLoZOSYISer9LNmTn3REA0dse7wq07iiHhpbe799utyYMwElrFErCgxijDGmJYAChheCKLA1romyfgfytn\\\\\\\/L1KLbfxEDAB90fiNOcupFxiJx72nfg637lrUcW1BFXAaFE8cGnXGoh8owdc6u80fApJwmIMon+D+xORdgwBMftSq26BkizVNhPbjrTbhV0RtA+kBNJL1bAF9J0kArR8uwjOoQRAMYqNWH2eo+qXBMi2i+sazmk93e7owoga688lh3dA3PhoBJ+uAb1KVFL+Faytd\\\\\\\/\\\\\\\/GFg9WZ\\\\\\\/gc2nqXwt6T\\\\\\\/psKIaqbEfy0gGh4XRlLSj4aMBgxKjh02PfoDZN47ZbyaoZWRyYTQJhaDKkt9InjJhyC3UNXzY7wluFb\\\\\\\/DrRcbMABI5oj3PClxs0NOD5X0LQp+Gpgikrs8r504RDbszK3qpet4aycBW0eqIRm3MfdoudW54UEhaWRF9bfo0IuEuNViVhKQRg\\\\\\\/BwVpkA9rHen+U9p0BuVBffDFufSkJbMrEQ8ergwmzP9Qmutjrlffz5hBkKeq9JnCBpB9zuqiN+JjPLbvUemJLoN5H3s+5A50RX0n+HH83gB8m88V1D4TybpoDkiCwO50AYgZKY2QTS\\\\\\\/98qMZ8l6WWe9WuwVWYiWqHOmvumra9m0Jws33QGE4eu1+yFIQ3RzrQJgybZ38+yRfop9rQSU7f00cRbVAFZeZOIkon1v+yQ+3Pey2uZEv0i9+S8aDp6S9cOVxU5ZO7VRp0UgEOWxX9YmkCzYLCtAOsDFapnjkvZExO0yzMLpJ78uPvG31Fw3C0p7Q9zvrIkZjaqhQqXSaCoYGLuaAvzQXAGlerevmg0wvdTj6X9fXlhyzOS9bO947TEoP1FeboJcXnXa329Z1\\\\\\\/qhydZaW+EnZFYis5n0c2KmA7xRtT0lJQUSM1PsVKUU7MTFyaKrW1\\\\\\\/iZtlzHavIr1con63A4FN0Ly4d5wlXSDn9N87MAk0wGgbo30ATglQvxeigGmT7VILM1gaWGO1GuAlyaL03vfk7G0AY6DgQTAeem0qZNZAHt3bGz8mM9lbRPaAe5seZqQq1RyeH09ub2xIvekVV9yq2ndsmuBo4aU3mGcNeYO6C3I9x4rQ0kMaF1cu+8LhzBAIoZF7ideN2viZpt8AlJWfoB\\\\\\\/3RmTdsuL7xqJx8WCd755WhIN51pMyPSaM5DqF2ZQw7jqBPWoohP30wSsmQd12VNFkU\\\\\\\/HpRjTUw3C18MXzuGX5k6NNvuc7LxoaquRcGpx2oaGV8QrGMu4iUK9TwT4KRIDcRC0LQMAtsAS9ROSGoupJJqW69wZDrgBV38PuWDE2Im+0zT\\\\\\\/9oBerbb2dBIFIi+cUJQ2lTFFscEE\\\\\\\/Q2UDikzgD710lBUhiNvFhceyQjmXaVzho8SRg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022vzZwT\\\\\\\/y0mHjpNtWPAJuBow==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228a3f12b6a477311af67317eb3dfef44eff447acebc17d289c8385deae37b3af6\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+BcHkAkT\\\\\\\/yV3DY9y70baKUkrwQ1YI4cQCFVug4EvWLWu10tl6AfkGWnVgyaiD46kN\\\\\\\/hj\\\\\\\/XzRqLmEzo0WJbjjCqW0hmL9K175tkTQbh2lHf\\\\\\\/vI5naQsZDfiGJ\\\\\\\/ly0eKrDGb5\\\\\\\/rUqYOjFdhCe0e4dvbV8Mt\\\\\\\/5aA9nHZ88x5O+IRXGVZpbcYB6vAgwUf3utsk2t6oyiTXvWr2Hi3SmrypzSHUgtPHWIUwV4tQVgVPNXtOtEdqoVPOsQ8QHhHsh1Ev4ZkIayccb7F\\\\\\\/cKorLiuQOxNgTRkVufEXb8VLuOXyNQENMWZGdo5wyIkUbZMLmHJWmmBQGbC4n41YI2kSOZXqz9v+tIgSp+Bpl\\\\\\\/59u0MDdU1LJ381DNcqd9eLD9ubfDGt6u+5n6Y2IZfvVYVP1a1xcWyKNNkCHrMe54QITMlzyDS93s9JS0vtPI4DSLABSFvwJyNKuGF21dqhKl\\\\\\\/vOOA2\\\\\\\/EYZ2G7wlmv8NVd6NfdNOblyfqgTZPmHBqJijjYcm5qiKEtsX5S7aQ\\\\\\\/JF8pl2xLN7McFsMqA\\\\\\\/8RA5LkSLlKR3bANk99ZUS71FBcmaT0mYncVAIMmmDNEPF0A7G19gS2lWx+LijiVDmxZ\\\\\\\/9dYhK1F2AqRbEXKhXuRM0ixxDydR1VxMqN\\\\\\\/dc1AG2nqKXlLO3QRohGJZIxiYJ2esnEUjTUYfz4oM5Ti+g\\\\\\\/t5xFP5r81UBg+t8CsV8vOqrGFGfR9akhFG+w06Wx7BpKGysKpYw4bJ4mjyjaI+YJQF27aPngM2mQZXSf7Zzsw4XSyv+UcZCMFfu5pX99J5sO6mWNRC5+VVGSTCHVZXjDoXCO9sxwGBErrg95eoZMIld3ph5J0jAADCTaV8+IwSvcyjVC5WfYNwRU5wkVftoUPbUx+JoDypFsSYw4rPoHtxYOQNQ\\\\\\\/kTnror9NRJQIdb8QKQ7kV7EnE92nhzO03SAvTVE0p973kMBdRTGsfl4Auj1FGFCe55aGOcbq+pBfrZAU5XpqCUG7mzPQ5K6LH7zbxa1vX6s9ruYR4gyxsWQ+MtX616CyL3yLEWCux4bXJ8AO26qIoMCHWv13EEouXOOVgJviDI5FN+JqmwQGZ+XrPFXxzqrKHhwrKDl1a+sf7BLPTvZIQSAplaf\\\\\\\/fIjKv5thqZ\\\\\\\/9SncyOwlHQocS8Pxwbpt+XmapO6xh40aOnpaN14gENX\\\\\\\/F77icetD88TGOdasZImyytNRi\\\\\\\/pl05hzrUwvIS+ZIFWKuWpJuXvGrgyb6HgmOeNtuFCcJPmTfDa6mC0wOLUdW+5oXZbqOhKmqQOW\\\\\\\/djXnTqSeMVJFQDQGQUu2cABOTmZBMCuuz7V1AexehxscAWzUDoojHXHtornB6Q0o7ce0lUMDivZNeK+vGWdX4FC+I950KZcMhc7cNR3B8Chs\\\\\\\/RoGWfWIS19OpmVmvVj6xxU2qJ7kMuTsWJo1QeSZPcBA7tAF1rDtYaQwgMENT+Vf7q2ujXLkM9vJP5Yuut2mTtWcE5jMalAgY52IMdryWoMNXYeBbVdNVphhhVmDAQZgqFML2vnuhMl7RGlj0Jgm+h9DsM9Sh8toMtRjF1+zjRGho\\\\\\\/geuxQt0kPNV6ZycCnT8k\\\\\\\/Yw0sBqsTBu9Da489fHBLDhmk3M+PAsKyugIgTqw4RsL9V+nlPzt4bT\\\\\\\/5WqiicgJldp8kkKNhRwTUNyObmNto44T0WC6k3dNZF3z7bgUa+FdTId0NWQ2fh1YoJvZUF9YGkAklV+oBNR9T\\\\\\\/266hMqpTA4M9KBdDwYZ9biHbjwNZmtTcSMps3+sq805p8x6ukR1NRGEWi8URY2SlYAL0qb2f51YvVW4j6OWK+wjRSQyyGaMecYhz4Z1W+Ia977fqJdv35elWUUopBGLtvqCZVe4epNFBQZ5qQR6Rtc0MC\\\\\\\/sTcgTY6VQ7WuFBgwCbOdLTMm85CSzi3Zq1uPezQhu21dwQeMg57Uc6IqbqmGrhZPCOPKhdBxOhEoW7nC7JyDkbcWQj04Rg\\\\\\\/CxVz5ub\\\\\\\/NWvEfi4mJ9zsD1aPAYlw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022INEbhV2A8CbV\\\\\\\/m1mz3wloA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c7deea122aaf2e07a90fae046818d0597cfd7eab448805ec1d6de42a5c4d3e26\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-483532 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/3/895" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CsDWOF1tfIlIvJsksjHmZacW7UUgJzdbF_rjptWhROY" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/3/895" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Uoy6ZdcrFP3qZ9JCMlDrLdZIfrhrqgNph5ToU8HKfw4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/3/881" title="Adapting Genotyping-by-Sequencing for Rice F2 Populations" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/3/911" title="Functional Analysis of Kinases and Transcription Factors in Saccharomyces cerevisiae Using an Integrated Overexpression Library" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483532" id="node483532" data-pisa="ggg;7/3/895" data-pisa-master="ggg;g3.116.038059" data-apath="/ggg/7/3/895.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Predicted Residual Error Sum of Squares of Mixed Models: An Application for Genomic Prediction</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Shizhong Xu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">895-909;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.038059 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483532" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Shizhong Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Botany and Plant Sciences, University of California, Riverside, California 92521</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shizhong%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShizhong%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:shizhong.xu@ucr.edu" class="" data-icon-position="" data-hide-link-title="0">shizhong.xu@ucr.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/3/895" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:483532" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:483532/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/3/895.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:483532" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:483532/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/3/895.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:483532" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:483532/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/3/895.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:483532" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:483532/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/3/895.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:483532" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:483532/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genomic prediction is a statistical method to predict phenotypes of polygenic traits using high-throughput genomic data. Most diseases and behaviors in humans and animals are polygenic traits. The majority of agronomic traits in crops are also polygenic. Accurate prediction of these traits can help medical professionals diagnose acute diseases and breeders to increase food products, and therefore significantly contribute to human health and global food security. The best linear unbiased prediction (BLUP) is an important tool to analyze high-throughput genomic data for prediction. However, to judge the efficacy of the BLUP model with a particular set of predictors for a given trait, one has to provide an unbiased mechanism to evaluate the predictability. Cross-validation (CV) is an essential tool to achieve this goal, where a sample is partitioned into <em>K</em> parts of roughly equal size, one part is predicted using parameters estimated from the remaining <em>K</em> – 1 parts, and eventually every part is predicted using a sample excluding that part. Such a CV is called the K-fold CV. Unfortunately, CV presents a substantial increase in computational burden. We developed an alternative method, the HAT method, to replace CV. The new method corrects the estimated residual errors from the whole sample analysis using the leverage values of a hat matrix of the random effects to achieve the predicted residual errors. Properties of the HAT method were investigated using seven agronomic and 1000 metabolomic traits of an inbred rice population. Results showed that the HAT method is a very good approximation of the CV method. The method was also applied to 10 traits in 1495 hybrid rice with 1.6 million SNPs, and to human height of 6161 subjects with roughly 0.5 million SNPs of the Framingham heart study data. Predictabilities of the HAT and CV methods were all similar. The HAT method allows us to easily evaluate the predictabilities of genomic prediction for large numbers of traits in very large populations.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/best-linear-unbiased-prediction" class="hw-term hw-article-keyword hw-article-keyword-best-linear-unbiased-prediction" rel="nofollow">best linear unbiased prediction</a></li><li class="kwd"><a href="/keyword/cross-validation" class="hw-term hw-article-keyword hw-article-keyword-cross-validation" rel="nofollow">cross-validation</a></li><li class="kwd"><a href="/keyword/generalized-cross-validation-0" class="hw-term hw-article-keyword hw-article-keyword-generalized-cross-validation" rel="nofollow">generalized cross-validation</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/hybrid-breeding" class="hw-term hw-article-keyword hw-article-keyword-hybrid-breeding" rel="nofollow">hybrid breeding</a></li><li class="kwd"><a href="/keyword/mixed-model" class="hw-term hw-article-keyword hw-article-keyword-mixed-model" rel="nofollow">mixed model</a></li><li class="kwd"><a href="/keyword/gen-pred" class="hw-term hw-article-keyword hw-article-keyword-gen-pred" rel="nofollow">Gen Pred</a></li><li class="kwd"><a href="/keyword/shared-data-resource" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resource" rel="nofollow">Shared data resource</a></li></ul><p id="p-3">Many diseases, anatomic structures, physiological characteristics, and behaviors in humans are polygenic traits. Most agronomic traits in agriculture, <em>e.g.</em>, yield, are also polygenic. These complex traits require whole-genome study to understand the genetic mechanisms and to genetically improve the quality and quantity of agricultural products (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">de los Campos <em>et al.</em> 2009</a>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">2013a</a>,<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">b</a>). Genomic prediction (selection) is a statistical method of whole-genome study (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Meuwissen <em>et al.</em> 2001</a>). It can lead to earlier detection of acute polygenic cancers (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Vazquez <em>et al.</em> 2012</a>). Genomic prediction is also an effective tool to select superior cultivars in plant breeding (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Heffner <em>et al.</em> 2009</a>). Genomic hybrid prediction will provide an opportunity to evaluate all potential hybrids and allow breeders to select superior hybrids that will have little chance to be discovered based on traditional hybrid breeding schemes (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Xu <em>et al.</em> 2014</a>). Genomic selection has been very successful in the dairy cattle industry (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Goddard and Hayes 2007</a>) and will soon become a routine procedure for breeding of a vast number of agricultural species.</p><p id="p-4">Among the commonly used methods for genomic prediction, BLUP (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Henderson 1975</a>) is one of a few suitable methods for handling high-throughput genomic data with millions of genetic variants (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">VanRaden 2008</a>). Reproducing kernel Hilbert spaces (RKHS) regression (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gianola <em>et al.</em> 2006</a>) is another method with such an ability, but RKHS has not been as well recognized as the BLUP method. Although variable selection approaches such as Bayes B (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Meuwissen <em>et al.</em> 2001</a>) and LASSO (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Tibshirani 1996</a>) are optimal for traits with a few detectable loci of large effects plus many undetectable modifying loci under low and intermediate marker density, BLUP is the most robust method and one of the most commonly used genomic selection methods (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">de los Campos <em>et al.</em> 2013b</a>). More importantly, the computational speed does not depend on marker density because it takes a marker-inferred kinship matrix (covariance structure) as the input data, albeit computing kinship matrix taking additional time. To evaluate the predictability of the BLUP model, one has to resort to some other tools, such as validation or CV, where individuals predicted do not contribute to estimated parameters that are used to predict these individuals. If individuals predicted are not excluded from the training sample, serious bias will occur in prediction.</p><p id="p-5">The predictability of a model is often represented by the squared correlation coefficient between the observed and predicted phenotypic values (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Xu <em>et al.</em> 2014</a>). This squared correlation is approximately equal to <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-1.gif"/></noscript></span></span>, where PRESS is the predicted residual error sum of squares and SS is the total sum of squares of the phenotypic values. <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Allen (1971</a>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">1974)</a> proposed to use PRESS as a criterion to evaluate a regression model, in contrast to using the estimated residual error sum of squares (ERESS) as the criterion. To calculate PRESS, <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Allen (1971</a>, <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">1974)</a> used an approach that is now called the leave-one-out cross-validation (LOOCV) or ordinary CV (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Craven and Wahba 1979</a>), in which an individual is predicted using parameters estimated from the sample that excludes this individual. When the sample size (<em>n</em>) is large, LOOCV presents a high computational cost because one will virtually have to analyze the data <em>n</em> times. The K-fold CV (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Picard and Cook 1984</a>) is an extension of LOOCV in which the sample is partitioned into <em>K</em> parts of roughly equal size. Individuals in a part are predicted simultaneously using all individuals in the remaining <em>K</em> − 1 parts. Eventually, all parts are predicted once and used to estimate parameters <em>K</em> − 1 times. When <em>K</em> is small, there are many different ways of partitioning the sample, leading to variation in the calculated predictability. This variation can be very large for small sample sizes. Therefore, people often repeat the K-fold CV a few times and use their average values to reduce the error due to random partitioning. If possible, LOOCV (also called the n-fold CV, a special case of K-fold CV when <em>K</em> = <em>n</em> and <em>n</em> is the sample size) is recommended because it eliminates all problems associated with this random partitioning variation. However, such a CV is not realistic for large samples under the mixed model methodology. Although a simple split CV (50% training and 50% test) should suffice with very large samples, still 50% of the sample is wasted. The LOOCV method may slightly overpredict the model compared with the K-fold CV when K is substantially smaller than <em>n</em> (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hastie <em>et al.</em> 2008</a>).</p><p id="p-6"><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Cook (1977</a>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">1979)</a> developed an explicit method to calculate PRESS by correcting the deflated residual error of an observation using the leverage value of the observation without repeated analyses of the partitioned samples. This method applies to least square regression under the fixed model framework, where the predicted <em>y</em> is a linear function of the observed <em>y</em> as shown below,<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-2.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-3.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>is called the hat matrix. The predicted residual error for observation <em>j</em> is <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-4.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-5.gif"/></noscript></span></span> is the so-called estimated residual error and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-6.gif"/></noscript></span></span> is the leverage value of the <em>j</em>th observation (the <em>j</em>th diagonal element of the hat matrix). It is the contribution of the prediction for an individual from itself and may be called the conflict of interest factor. The predicted residual error is the estimated residual error after correction for the deflation. The sum of squares of the predicted residual errors over all individuals is the PRESS, which is a well-known statistic in multiple regression analyses. To find an explicit expression of PRESS for a mixed model, we need to identify a random effect version of the hat matrix and use the leverage value of the <em>j</em>th observation to correct for the deflated residual error.</p><p id="p-7">The HAT method is a fast algorithm for the ordinary CV for a linear model (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Allen 1971</a>, <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">1974</a>) because the regression analysis is only done once on the whole sample and then the estimated residual errors are modified afterward. Extension of the HAT method to mixed model has been made by <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Golab <em>et al.</em> (1979)</a> in finding the optimal ridge factor in ridge regression (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hoerl and Kennard 1970a</a>,<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">b</a>). It is well known that ridge regression can be formulated as a mixed model problem with the variance ratio replaced by a given ridge factor. <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Golab <em>et al.</em> (1979)</a> proposed a generalized cross-validation (GCV) method to find the optimal ridge factor so that the generalized residual error variance is minimized. These authors showed that the GCV-calculated residual error sum of squares is a rotation-invariant version of Allen’s PRESS. The residual error variance obtained from the GCV method is equivalent to calculating the residual error variance by dividing the ERESS by an “effective” degree of freedom. Properties of the GCV method have been extensively studied by <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Li (1987)</a>. <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Jansen <em>et al.</em> (1997)</a> applied GCV to wavelet thresholding. When performing genomic prediction, we prefer to see the actual predicted residual errors (errors in prediction of future individuals) obtained from the ordinary CV because the residual errors obtained via GCV may not be intuitive to most of us. The important gain from the GCV method of ridge regression analysis to genomic selection is the HAT matrix of the random model when the genomic variance is given.</p><p id="p-8">There is rich literature on smoothing spline analysis that also helped us to develop the fast HAT method for evaluation of mixed model predictability (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Wahba 1975</a>, <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">1980</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">1990</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">1998</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Wahba and Wold 1975a</a>,<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">b</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Craven and Wahba 1979</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Wahba <em>et al.</em> 1995</a>, <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">2000</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Wahba and Luo 1997</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wang 1998a</a>,<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">b</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Hastie <em>et al.</em> 2008</a>). In smoothing spline curve fitting, a response variable is fitted to a predictor with an arbitrary functional relationship. The common approach is to fit the curve using B-spline or another type of nonparametric approach. Several spline bases (more than necessary) are constructed from the original predictor. These bases are considered as new predictors, which are then used to fit the response variable with linear relationship. The regression coefficients are then estimated using a penalized shrinkage method such as the ridge regression. The ridge parameter in smoothing splines is then called the smoothing parameter <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-7.gif"/></noscript></span></span> which is often found so that the GCV residual error variance is minimized (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Craven and Wahba 1979</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Wahba 1980</a>). Given the smoothing parameter, the predicted responses of all individuals are linear functions of all observed responses. <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Hastie <em>et al.</em> (2008)</a> collectively called these linear functions the smoother matrix and denoted it by <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-8.gif"/></noscript></span></span> This smoother matrix is the random effect version of the HAT matrix,<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-9.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where <em>Q</em> is a known diagonal matrix. A HAT matrix under the random model was also given by <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">de los Campos <em>et al.</em> (2013b)</a> in the form of <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-10.gif"/></noscript></span></span> although it was not derived for calculating PRESS. The HAT matrix of the fixed model introduced in <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">Equation 2</a> is then denoted by <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-11.gif"/></noscript></span></span> The difference between the two HAT matrices is clear in form. <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Hastie <em>et al.</em> (2008)</a> stated that both <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-12.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-13.gif"/></noscript></span></span> are symmetric and positive semidefinite, <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-14.gif"/></noscript></span></span> is idempotent (<span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-15.gif"/></noscript></span></span>) but <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-16.gif"/></noscript></span></span> is not, and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-17.gif"/></noscript></span></span> has a rank of <em>m</em> (number of predictors) while <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-18.gif"/></noscript></span></span> has a rank of <em>n</em> (number of observations). So, the HAT matrix for a random model has been defined by the smoothing splines community. We may implement this HAT matrix in our BLUP prediction to evaluate the predictability of our models and avoid the lengthy CV analysis. The smoothing parameter (our variance ratio) should be given a reasonable value and the REML estimate from the whole sample is a natural choice. However, replacing the prechosen <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-19.gif"/></noscript></span></span> by a data-driven estimate makes the HAT matrix a complicated function of the data. The question is, what is the difference between the HAT method (when <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-20.gif"/></noscript></span></span> is estimated from the whole sample) and the actual CV (when <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-21.gif"/></noscript></span></span> is estimated anew within each fold)? This becomes the main objective of this study.</p><p id="p-9">When revising this manuscript, a similar study was published in the same journal (<em>G3: Genes| Genomes | Genetics</em>) by <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gianola and Schon (2016)</a>. They also recognized the approximation nature of the new method and stated that using the whole-sample-estimated <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-22.gif"/></noscript></span></span> in place of the prechosen <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-23.gif"/></noscript></span></span> will not affect the result too much, especially when the LOOCV is compared, because the training sample only differs from the whole sample by one observation. However, this is only a speculation (most likely true) and they did not explicitly investigate the difference. Since the new method represents a significant technical improvement in genomic selection, the community must be aware of the difference before widely adopting the new method to evaluate a genomic selection program. In this study, we explicitly answer this question by analyzing several agronomic traits and 1000 metabolomic traits from two rice populations. Further comparison was also made in genomic prediction of human height from the Framingham heart study data (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Dawber <em>et al.</em> 1951</a>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">1963</a>).</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Fixed model</h3><p id="p-10">The HAT method for calculating PRESS under the fixed model is given by <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Cook (1977</a>, <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">1979)</a> for the LOOCV scenario but not for the leave <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-24.gif"/></noscript></span></span> out CV (the K-fold CV). We extended Cook’s method to leave <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-25.gif"/></noscript></span></span> out for weighted least squares regression analysis. The predicted <em>y</em> is a linear function of the observed <em>y</em> as shown below,<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-26.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-27.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>is the hat matrix. This <em>H</em> matrix is still idempotent. In a K-fold CV analysis, let <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-28.gif"/></noscript></span></span> be the number of observations in the <em>k</em>th fold for <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-29.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-30.gif"/></noscript></span></span> Define <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-31.gif"/></noscript></span></span> as an <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-32.gif"/></noscript></span></span> matrix of independent variables for individuals in the <em>k</em>th fold, where p is the number of independent variables. The “leverage” value for the <em>k</em>th fold is defined as an <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-33.gif"/></noscript></span></span> matrix,<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-34.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-35.gif"/></noscript></span></span> is the <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-36.gif"/></noscript></span></span> subset of matrix <em>W</em> corresponding to the <em>k</em>th fold. This matrix must appear in the end, not in the beginning, of the above equation. Let<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-37.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>be the estimated residual errors where <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-38.gif"/></noscript></span></span> is estimated from the whole sample. The predicted residual errors for the <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-39.gif"/></noscript></span></span> individuals in the <em>k</em>th fold is<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-40.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>Therefore, the PRESS is defined as<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-41.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span>which is the weighted sum of squares of the predicted residual errors. Derivation of <a id="xref-disp-formula-9-1" class="xref-disp-formula" href="#disp-formula-9">Equation 9</a> is given in Appendix A.</p></div><div id="sec-3" class="subsection"><h3>Mixed model</h3><p id="p-11">The linear mixed model for genomic prediction is written as<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-42.gif"/></noscript></span><span class="disp-formula-label">(10)</span></span>where <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-43.gif"/></noscript></span></span> represents the fixed effects, <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-44.gif"/></noscript></span></span> is a vector of random (polygenic) effects with an assumed <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-45.gif"/></noscript></span></span> distribution, and <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-46.gif"/></noscript></span></span> is a vector of residual errors. The expectation and variance of <em>y</em> are <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-47.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-48.gif"/></noscript></span></span> respectively, where <em>A</em> is a marker-inferred kinship matrix (explained in detail below), <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-49.gif"/></noscript></span></span> is the polygenic variance, and <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-50.gif"/></noscript></span></span> is the residual error variance. The parameters are <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-51.gif"/></noscript></span></span> and the variances are estimated using the restricted maximum likelihood method (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Patterson and Thompson 1971</a>) by maximizing the following likelihood function,<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-52.gif"/></noscript></span><span class="disp-formula-label">(11)</span></span>The estimated genomic heritability (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">de los Campos <em>et al.</em> 2015</a>) from the markers is <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-53.gif"/></noscript></span></span> The best linear unbiased estimates (BLUE) of the fixed effects are <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-54.gif"/></noscript></span></span> and the BLUP of the polygenic effects are <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-55.gif"/></noscript></span></span> The fitted phenotypic values are <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-56.gif"/></noscript></span></span> which is a conditional prediction (not a marginal prediction). Corresponding to the predicted polygenic effect <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-57.gif"/></noscript></span></span> we now define <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-58.gif"/></noscript></span></span> as the “observed” polygenic effect (it is indeed observed because <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-59.gif"/></noscript></span></span> is used). The model goodness of fit (FIT) for the random effects is defined as the squared correlation between <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-60.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-61.gif"/></noscript></span></span></p></div><div id="sec-4" class="subsection"><h3>Marker-inferred kinship matrix</h3><p id="p-12">The marker-inferred kinship matrix <em>A</em> is calculated from all markers of the genome using the following equation,<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-62.gif"/></noscript></span><span class="disp-formula-label">(12)</span></span>where <em>m</em> is the total number of markers, <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-63.gif"/></noscript></span></span> is a normalization factor to make the diagonal elements of matrix <em>A</em> as close to unity as possible, and <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-64.gif"/></noscript></span></span> is an <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-65.gif"/></noscript></span></span> vector of genotype indicator variables for all individuals at marker <em>k</em>. For individual <em>j</em>, the numerical code for a genotype is<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-66.gif"/></noscript></span><span class="disp-formula-label">(13)</span></span>where <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-67.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-68.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-69.gif"/></noscript></span></span> are the three genotypes of the marker. People often standardize the <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-70.gif"/></noscript></span></span> vectors before using them to calculate the kinship matrix (see <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">VanRaden 2008</a>).</p></div><div id="sec-5" class="subsection"><h3>Cross validation</h3><p id="p-13">For a K-fold CV, we randomly partitioned the sample into <em>K</em> parts of roughly equal size. We then used <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-71.gif"/></noscript></span></span> parts to predict the remaining part. Let <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-72.gif"/></noscript></span></span> be the vector of phenotypic values that are partitioned into <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-73.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-74.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-75.gif"/></noscript></span></span> is a vector of phenotypic values of all observations in the <em>k</em>th part (test sample) and <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-76.gif"/></noscript></span></span> is a vector of phenotypic values for all individuals excluding observations in the <em>k</em>th part (training sample). Corresponding to this partitioning of the sample, we have<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-77.gif"/></noscript></span><span class="disp-formula-label">(14)</span></span>and<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-78.gif"/></noscript></span><span class="disp-formula-label">(15)</span></span>The predicted phenotypic values in the test sample are<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-79.gif"/></noscript></span><span class="disp-formula-label">(16)</span></span>Let <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-80.gif"/></noscript></span></span> be the “observed” polygenic effect (phenotypes of the test sample adjusted by the fixed effects or centered phenotypes) and<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-81.gif"/></noscript></span><span class="disp-formula-label">(17)</span></span>be the predicted polygenic effect for the test sample. After all parts of the sample are predicted, we calculate the PRESS using<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-82.gif"/></noscript></span><span class="disp-formula-label">(18)</span></span>The predictability is defined as<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-83.gif"/></noscript></span><span class="disp-formula-label">(19)</span></span>where<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-84.gif"/></noscript></span><span class="disp-formula-label">(20)</span></span>is the total sum of squares of <em>y</em> adjusted by the fixed effects.</p></div><div id="sec-6" class="subsection"><h3>The HAT method</h3><p id="p-14">With the HAT method, we first defined the adjusted or centered phenotypic vector by the fixed effects <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-85.gif"/></noscript></span></span> as the “observed” <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-86.gif"/></noscript></span></span> and define <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-87.gif"/></noscript></span></span> as the predicted <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-88.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-89.gif"/></noscript></span></span> is estimated from the whole sample. We then used the whole sample to predict the polygenic effects<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-90.gif"/></noscript></span><span class="disp-formula-label">(21)</span></span>Comparing the second form of the above equation (<span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-91.gif"/></noscript></span></span>) with the fixed model HAT function, <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-92.gif"/></noscript></span></span> we realized that <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-93.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-94.gif"/></noscript></span></span> is the HAT matrix of the random effects. Substituting <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-95.gif"/></noscript></span></span> by <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-96.gif"/></noscript></span></span> and after a few steps of algebraic derivation leads to<span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-97.gif"/></noscript></span><span class="disp-formula-label">(22)</span></span>where <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-98.gif"/></noscript></span></span> is the variance ratio. With eigen-decomposition for the <em>A</em> matrix, we have <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-99.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-100.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-101.gif"/></noscript></span></span> Therefore,<span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-102.gif"/></noscript></span><span class="disp-formula-label">(23)</span></span>This expression (the second form) is exactly the one defined by <a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">Hastie <em>et al.</em> (2008)</a> for the smoothing spline analysis given in <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">Equation 3</a>, where their <em>X</em> is replaced by the eigenvector <em>U</em>, their <em>Q</em> is replaced by the inverse of eigenvalue matrix <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-103.gif"/></noscript></span></span> (diagonal), and their smoothing parameter is our variance ratio. The HAT matrix is easy to compute because <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-104.gif"/></noscript></span></span> is diagonal. When some eigenvalues are zero, <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-105.gif"/></noscript></span></span> does not exist (very often), we reformulate it by <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-106.gif"/></noscript></span></span> Therefore,<span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-107.gif"/></noscript></span><span class="disp-formula-label">(24)</span></span>where <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-108.gif"/></noscript></span></span> is the <em>j</em>th eigenvalue of matrix <em>A</em>. Although the HAT method does not need to refit the model for each part predicted, it still needs to partition the sample into <em>K</em> parts if comparison with the traditional CV is of interest. Let <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-109.gif"/></noscript></span></span> be the estimated residual errors for all individuals in the <em>k</em>th part and <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-110.gif"/></noscript></span></span> be the diagonal block of matrix <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-111.gif"/></noscript></span></span> corresponding to all individuals in the <em>k</em>th part. The predicted residual errors for the <em>k</em>th part are <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-112.gif"/></noscript></span></span> Proof of this predicted residual error is provided in Appendix B. The PRESS under this random model becomes<span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-113.gif"/></noscript></span><span class="disp-formula-label">(25)</span></span>The predictability is measured by<span class="disp-formula" id="disp-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-114.gif"/></noscript></span><span class="disp-formula-label">(26)</span></span>where <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-115.gif"/></noscript></span></span> is the total sum of squares for the centered <em>y</em> (adjusted by the fixed effects). The n-fold HAT approach is a special case where the <em>k</em>th part to be predicted contains only one individual, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-116.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-117.gif"/></noscript></span></span> Therefore, the leave-one-out version of the PRESS is<span class="disp-formula" id="disp-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-118.gif"/></noscript></span><span class="disp-formula-label">(27)</span></span>This predictability is roughly equal to the squared correlation between the fixed-effect-adjusted phenotypes and the predicted polygenic effects. The ERESS is <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-119.gif"/></noscript></span></span> and the usual R-square reported in regression analysis is <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-120.gif"/></noscript></span></span> which is a measurement of model FIT, not predictability.</p></div><div id="sec-7" class="subsection"><h3>Generalized cross validation</h3><p id="p-15">GCV (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Golab <em>et al.</em> 1979</a>) is an alternative method to correct the deflated residual error variance. The GCV-calculated residual error sum of squares is called generalized residual error sum of squares (GRESS), which is defined by<span class="disp-formula" id="disp-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-121.gif"/></noscript></span><span class="disp-formula-label">(28)</span></span>where <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-122.gif"/></noscript></span></span> is the predicted polygenic effect from the whole sample. It is equivalent to dividing each estimated residual error by the average <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-123.gif"/></noscript></span></span> across all observations. Therefore, an intuitive expression of the above equation is<span class="disp-formula" id="disp-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-124.gif"/></noscript></span><span class="disp-formula-label">(29)</span></span>where <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-125.gif"/></noscript></span></span> is the average leverage value across all observations and <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-126.gif"/></noscript></span></span> The predictability is defined as<span class="disp-formula" id="disp-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-127.gif"/></noscript></span><span class="disp-formula-label">(30)</span></span><a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Golab <em>et al.</em> (1979)</a> stated that GRESS is a rotation-invariant PRESS. It is not intuitive to interpret GRESS and therefore we prefer to report PRESS and thus <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-128.gif"/></noscript></span></span></p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-16">All data analyzed in this study have been previously published. Sources of these data are provided by the references cited in the text. The Framingham Heart Study Data were downloaded from NCBI dbGaP with an IRB number HS -11-159. The rice data along with the R codes are provided in Supplemental <a href="http://www.g3journal.org/content/suppl/2017/01/19/g3.116.038059.DC1/FileS1.zip">Files S1</a>, <a href="http://www.g3journal.org/content/suppl/2017/01/19/g3.116.038059.DC1/FileS2.zip">S2</a>, <a href="http://www.g3journal.org/content/suppl/2017/01/19/g3.116.038059.DC1/FileS3.zip">S3</a>, <a href="http://www.g3journal.org/content/suppl/2017/01/19/g3.116.038059.DC1/FileS4.zip">S4</a>, <a href="http://www.g3journal.org/content/suppl/2017/01/19/g3.116.038059.DC1/FileS5.csv">S5</a> and <a href="http://www.g3journal.org/content/suppl/2017/01/19/g3.116.038059.DC1/FileS6.csv">S6</a>. Description of the supplemental files can be found in <a href="http://www.g3journal.org/content/suppl/2017/01/19/g3.116.038059.DC1/FileS7.docx">File S7</a>.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Properties of the HAT method</h3><p id="p-17">Properties of the HAT method will be demonstrated using an experimental rice population consisting of 210 recombinant inbred lines (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Yu <em>et al.</em> 2011</a>). These lines were derived from the cross of two rice varieties. A total of 270,820 SNPs were used to infer breakpoints of the genome for each line, resulting in a total of 1619 bins. A bin is a haplotype block within which there are no breakpoints across the entire population. In the original analysis of <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Yu <em>et al.</em> (2011)</a>, each bin was treated as a genetic marker. In this study, we used all the 1619 bins to infer a 210 × 210 kinship matrix. The matrix represents the genetic relationships of the lines and is used to model the covariance structure of the polygene. The population size is reasonably small and enabled us to compare the HAT method with CV in great detail.</p><p id="p-18">Seven agronomic and 1000 metabolomic traits were included in the analysis. The agronomic traits are yield per plant (YD), tiller number per plant (TP), grain number per panicle (GN), 1000-grain weight (KGW), grain length (GL), grain width (GW), and heading day (HD). The first four traits (YD, TP, GN, and KGW) were field evaluated four times (two locations in 2 yr), and GL and GW were replicated twice (two different years), and HD was replicated three times (three different years). The phenotypic value of each trait for each line is the average of the replicates. The 1000 metabolites were measured from seeds (317) and leaves (683) with two biological replications (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gong <em>et al.</em> 2013</a>). The phenotypic values of the metabolites are the average expression levels of the two replicates after log2 transformation.</p><p id="p-19">Predictability of the HAT method was compared with that of the CV method starting at twofold and ending at n-fold incremented by one, as shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> for the seven agronomic traits. The two methods produced very similar values of <em>R</em>-squares, with a slight upward bias for the HAT method due to the use of <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-129.gif"/></noscript></span></span> estimated from the whole sample. The biases are quite small for high predictability traits, <em>e.g.</em>, KGW and GL. They appear to be large for low predictability traits such as YD and HD. However, this is partly due to the small scale of the <em>y</em>-axis (a visual effect). For example, the predictabilities of HAT and CV for trait KGW are 0.7564 and 0.7534, respectively, and the corresponding predictabilities for trait HD are 0.0774 and 0.0653. <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> also shows that when the numbers of folds are small, the predictabilities vary wildly and the variation progressively reaches zero at n-fold. The variation is caused by the ways that the folds are partitioned within the sample. Therefore, when a low number of folds are used in CV, it is necessary to repeat the CV a few times to reduce this variation. Although multiple CV will cause extra computational time, the HAT method can easily evaluate this variation.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/895/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of predictabilities of the HAT and CV methods for seven agronomy traits in inbred rice. The first seven panels are the predictabilities of the HAT (blue) and CV (red) for the seven traits. The last panel shows the average predictabilities and the 95% confidence band of the HAT method for KGW from 100 random partitionings of the sample." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-401273018" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of predictabilities of the HAT and CV methods for seven agronomy traits in inbred rice. The first seven panels are the predictabilities of the HAT (blue) and CV (red) for the seven traits. The last panel shows the average predictabilities and the 95% confidence band of the HAT method for KGW from 100 random partitionings of the sample.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/895/F1.medium.gif" width="343" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/3/895/F1.medium.gif" width="343" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/895/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/895/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483598" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-20" class="first-child">Comparison of predictabilities of the HAT and CV methods for seven agronomy traits in inbred rice. The first seven panels are the predictabilities of the HAT (blue) and CV (red) for the seven traits. The last panel shows the average predictabilities and the 95% confidence band of the HAT method for KGW from 100 random partitionings of the sample.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">Since computing the HAT method is sufficiently fast, we were able to perform random partitioning of the sample 100 times within a few minutes for all folds running from 2 to <em>n</em>. The last panel of <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> shows the mean and 95% confidence band for the replicated HAT predictability for trait KGW. The average predictability reaches a plateau at ∼10-fold, but the 95% band is still very wide. This result did not support the claim that the LOOCV seriously biased the predictability compared with K-fold CV (<a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">Hastie <em>et al.</em> 2008</a>).</p><p id="p-22"><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a> shows the plot of predictability from n-fold CV against that from HAT for the seven agronomic traits of rice. The differences between the two methods are visually indistinguishable. We then compared the two methods for the 1000 metabolomic traits with n-fold CV. The CV method took a few days to complete the n-fold CV but the HAT method, again, took no more than a few minutes. The corresponding plots for the 1000 metabolomic traits are shown in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>. Except for three outliers, all points fall on the diagonal line. The three outliers show that the HAT prediction is overoptimistic.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/895/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predictability of the cross-validation (CV) method plotted against that of the HAT method under n-fold CV for different traits. (A) Seven traits of the inbred rice. (B) 1000 metabolomic traits of the inbred rice. (C) 10 traits in hybrid rice. (D) Plot of the R2 of the CV method, the HAT method, and the goodness of fit (FIT) against the estimated heritability (H2) obtained from replicated experiments for 10 agronomy traits of the hybrid rice." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-401273018" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Predictability of the cross-validation (CV) method plotted against that of the HAT method under n-fold CV for different traits. (A) Seven traits of the inbred rice. (B) 1000 metabolomic traits of the inbred rice. (C) 10 traits in hybrid rice. (D) Plot of the &lt;em&gt;R&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; of the CV method, the HAT method, and the goodness of fit (FIT) against the estimated heritability (H&lt;sup&gt;2&lt;/sup&gt;) obtained from replicated experiments for 10 agronomy traits of the hybrid rice.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/895/F2.medium.gif" width="440" height="426"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/3/895/F2.medium.gif" width="440" height="426"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/895/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/895/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483606" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-23" class="first-child">Predictability of the cross-validation (CV) method plotted against that of the HAT method under n-fold CV for different traits. (A) Seven traits of the inbred rice. (B) 1000 metabolomic traits of the inbred rice. (C) 10 traits in hybrid rice. (D) Plot of the <em>R</em><sup>2</sup> of the CV method, the HAT method, and the goodness of fit (FIT) against the estimated heritability (H<sup>2</sup>) obtained from replicated experiments for 10 agronomy traits of the hybrid rice.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Genomic hybrid prediction in rice</h3><p id="p-24">We used a hybrid population of rice (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Huang <em>et al.</em> 2015</a>) to demonstrate the application of the HAT method to genomic hybrid breeding. The population consists of 1495 hybrid rice with 10 agronomic traits measured in two locations in China (Hangzhou and Sanya). The 10 traits are YD, panicle number (PN), GN, seed setting rate (SSR), KGW, HD, plant height (PH), panicle length (PL), GL, and GW. The phenotypic value of each hybrid is the average of the two locations. We used 1.6 million SNPs to infer the kinship matrix and then performed predictions using both the HAT and CV methods. Although the n-fold sample partitioning can be easily accomplished with the HAT method, it would be too costly to do it with the CV method. Therefore, we compared the two methods under the 10-fold CV. We replicated the experiment 20 times per 10-fold CV to reduce the variation caused by random partitioning of the sample. The average of the 20 replicates presents the predictability for each method.</p><p id="p-25">The two replications of hybrid rice experiments allowed us to estimate trait heritability of the hybrid population using the traditional ANOVA method (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). We partitioned the phenotypic variance into variance due to hybrids (genotypes) and variance due to residual error with systematic difference between the two locations excluded from the phenotype.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/483691/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/483691/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Analysis of variance table to estimate heritability of agronomic traits from replicated experiments of hybrid rice</span><div class="sb-div caption-clear"></div></div></div><p id="p-28">First, we compared the predictability of the CV method with the HAT method. <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a> shows the plot of the CV-generated predictability against the HAT-generated predictability. All the 10 points (one point per trait) fall on the diagonal line, indicating very good agreement between the two methods. We then compared the trait heritability (H2) from the two replicated environments with the predictability drawn from 10-fold CV, the predictability obtained from the HAT method (HAT), and the FIT. The plots are illustrated in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2D</a>. The <em>R</em><sup>2</sup> of HAT and CV are the same (the red circles overlap with the blue circles). Both HAT and CV fall around the diagonal line with some upward biases compared to H2. The FIT are severely biased upwards and are not good representatives of H2 at all.</p><p id="p-29"><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows a side-by-side comparison of H2 (trait heritability), <em>R</em><sup>2</sup> of HAT, CV, and FIT for all 10 traits, where FIT is equivalent to genomic heritability (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">de los Campos <em>et al.</em> 2015</a>). Different traits have very different H2, ranging from 0.08 (YD) to 0.92 (GL). The difference between HAT and CV is virtually zero across all traits and both are higher than H2 for the majority of the traits. For the three highly heritable traits (KGW, GL, and GW), the H2 is higher than or equal to HAT and CV. Interestingly, HAT and CV are substantially higher than H2 for HD.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/895/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of R2 of the estimated heritability from replicated experiments (H2), the cross-validation (CV) method, the HAT method, and the model goodness of fit (FIT) for 10 traits of the hybrid rice." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-401273018" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of &lt;em&gt;R&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; of the estimated heritability from replicated experiments (H2), the cross-validation (CV) method, the HAT method, and the model goodness of fit (FIT) for 10 traits of the hybrid rice.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/895/F3.medium.gif" width="440" height="188"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/3/895/F3.medium.gif" width="440" height="188"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/895/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/895/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483699" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-30" class="first-child">Comparison of <em>R</em><sup>2</sup> of the estimated heritability from replicated experiments (H2), the cross-validation (CV) method, the HAT method, and the model goodness of fit (FIT) for 10 traits of the hybrid rice.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Prediction of human height</h3><p id="p-31">We analyzed human height of 6161 subjects from the Framingham heart study (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Dawber <em>et al.</em> 1951</a>, <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">1963</a>) with ∼0.5 million SNPs using the mixed model methodology incorporating the marker-inferred kinship matrix. The model included effects of generation (two levels) and gender (male and female) as fixed effects. The estimated polygenic and residual variances are <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-134.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-135.gif"/></noscript></span></span> respectively, yielding a <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-136.gif"/></noscript></span></span> and an estimated genomic heritability of <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-137.gif"/></noscript></span></span> This genomic heritability is close to the reported gender average heritability of human height (0.75–0.88) (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Silventoinen <em>et al.</em> 2003</a>). The 10-fold CV and the HAT method gave predictabilities of <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-138.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-139.gif"/></noscript></span></span> respectively. Note that the predictabilities are the averages of 20 replicated random partitions and thus there are small SEs associated with the average values. The predictability obtained from the leave-one-out HAT method is 0.3278, slightly higher than the 10-fold partitioning approach.</p></div><div id="sec-13" class="subsection"><h3>GCV and optimization of <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-140.gif"/></noscript></span></span></h3><p id="p-32">Before we perform the following analysis, it is worthwhile to refresh our mind that the HAT method will slightly overestimate the predictability because of the approximation nature. We first used the human height trait as an example to demonstrate the difference between the HAT method and the GCV method. The REML estimate of the variance ratio is <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-141.gif"/></noscript></span></span> and the corresponding predictability from the n-fold HAT method is <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-142.gif"/></noscript></span></span> This REML estimate generates a GCV predictability of <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-143.gif"/></noscript></span></span> different from that of the HAT method. We now treated <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-144.gif"/></noscript></span></span> as a tuning parameter to maximize the predictability, as done by <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Mathew <em>et al.</em> (2015)</a> in GCV for estimating breeding values. Using a grid search around the REML-estimated value (<span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-145.gif"/></noscript></span></span>), we found that the maximum achievable predictability for the HAT method is <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-146.gif"/></noscript></span></span> when <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-147.gif"/></noscript></span></span> leading to a gain of <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-148.gif"/></noscript></span></span> which represents a <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-149.gif"/></noscript></span></span> gain in predictability. Although this gain is negligible, it demonstrates that the REML-estimated parameter does not give the maximum predictability. The good news is that <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-150.gif"/></noscript></span></span> is almost optimal, at least in this example. The corresponding maximum achievable predictability in GCV is <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-151.gif"/></noscript></span></span> when <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-152.gif"/></noscript></span></span> leading to a gain of <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-153.gif"/></noscript></span></span> <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> shows the predictability profiles around <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-154.gif"/></noscript></span></span> By tuning the parameter, the gain in predictability of the HAT method (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>) is visible but the gain of the GCV method (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>) is not recognizable.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/895/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Tuning parameter () that maximizes the genomic predictability () of human height. (A) predictability profile of the HAT method, where the red point represents the predictability () when the tuning parameter takes the REML estimate () and the blue point represents the maximum achievable predictability () when the tuning parameter is  (B) predictability profile of the GCV method, where the red point represents the predictability () when the tuning parameter takes the REML estimate () and the blue point represents the maximum predictability () when the tuning parameter is GCV, generalized cross-validation; REML, restricted maximum likelihood." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-401273018" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Tuning parameter (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-155&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) that maximizes the genomic predictability (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-156&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) of human height. (A) predictability profile of the HAT method, where the red point represents the predictability (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-157&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) when the tuning parameter takes the REML estimate (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-158&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) and the blue point represents the maximum achievable predictability (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-159&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) when the tuning parameter is &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-160&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (B) predictability profile of the GCV method, where the red point represents the predictability (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-161&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) when the tuning parameter takes the REML estimate (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-162&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) and the blue point represents the maximum predictability (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-163&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) when the tuning parameter is &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-164&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;GCV, generalized cross-validation; REML, restricted maximum likelihood.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/895/F4.medium.gif" width="288" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/3/895/F4.medium.gif" width="288" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/895/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/895/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483626" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-33" class="first-child">Tuning parameter (<span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F4/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F4/embed/mml-math-155.gif"/></noscript></span></span>) that maximizes the genomic predictability (<span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F4/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F4/embed/mml-math-156.gif"/></noscript></span></span>) of human height. (A) predictability profile of the HAT method, where the red point represents the predictability (<span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F4/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F4/embed/mml-math-157.gif"/></noscript></span></span>) when the tuning parameter takes the REML estimate (<span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F4/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F4/embed/mml-math-158.gif"/></noscript></span></span>) and the blue point represents the maximum achievable predictability (<span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F4/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F4/embed/mml-math-159.gif"/></noscript></span></span>) when the tuning parameter is <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F4/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F4/embed/mml-math-160.gif"/></noscript></span></span> (B) predictability profile of the GCV method, where the red point represents the predictability (<span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F4/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F4/embed/mml-math-161.gif"/></noscript></span></span>) when the tuning parameter takes the REML estimate (<span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F4/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F4/embed/mml-math-162.gif"/></noscript></span></span>) and the blue point represents the maximum predictability (<span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F4/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F4/embed/mml-math-163.gif"/></noscript></span></span>) when the tuning parameter is <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F4/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F4/embed/mml-math-164.gif"/></noscript></span></span>GCV, generalized cross-validation; REML, restricted maximum likelihood.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">To further compare the predictabilities of the HAT and GCV methods with their maximum achievable predictabilities, we used the “Brent” method of the “opim()” function in R to search for the optimal tuning parameter (<span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-165.gif"/></noscript></span></span>) for all 1000 metabolomic traits in the inbred rice population (210 lines). These optimal values of <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-166.gif"/></noscript></span></span> may be called the maximum predictability estimates (MPE). <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> illustrates the comparisons of predictabilities across all 1000 traits, where more than a dozen traits show visible gains in predictability by tuning the parameter around the REML-estimated value for the HAT method (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>). Similar comparison is shown in <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a> for the GCV method where tuning the parameter achieves more than 20 visible gains in predictability. <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5C</a> compares the predictabilities of GCV and HAT when the tuning parameter is fixed at the REML-estimated value. The two methods provided very similar predictabilities for all 1000 traits except a half dozen traits with visible differences. All three comparisons shown in <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a> have fitted <em>R</em>-squares at ∼0.9995 and the regression coefficients are not significantly different from one (<em>P</em> &gt; 0.05) except C, where the regression coefficient is significantly &gt; 1 (<em>P</em> &lt; 0.05).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/895/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of predictability of REML-estimated  with the maximum achievable predictability by tuning  for 1000 metabolomic traits of rice. (A) Maximum achievable predictability of the HAT method by tuning  plotted against the predictability when  takes the REML estimate. (B) Maximum achievable predictability of the GCV method by tuning  plotted against the predictability when  takes the REML estimate. (C) Predictability of the GCV method plotted against the predictability of the HAT method when  takes the REML estimate. The red points indicate traits with visible differences in predictability between the method shown on the x-axis and the method shown on the y-axis. The linear regression equation is given at the bottom of each panel and the fitted  of the regression is given at the top of each panel. GCV, generalized cross-validation; REML, restricted maximum likelihood." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-401273018" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Comparison of predictability of REML-estimated &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-167&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; with the maximum achievable predictability by tuning &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-168&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for 1000 metabolomic traits of rice. (A) Maximum achievable predictability of the HAT method by tuning &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-169&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; plotted against the predictability when &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-170&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; takes the REML estimate. (B) Maximum achievable predictability of the GCV method by tuning &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-171&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; plotted against the predictability when &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-172&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; takes the REML estimate. (C) Predictability of the GCV method plotted against the predictability of the HAT method when &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-173&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; takes the REML estimate. The red points indicate traits with visible differences in predictability between the method shown on the &lt;em&gt;x&lt;/em&gt;-axis and the method shown on the &lt;em&gt;y&lt;/em&gt;-axis. The linear regression equation is given at the bottom of each panel and the fitted &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-174&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; of the regression is given at the top of each panel. GCV, generalized cross-validation; REML, restricted maximum likelihood.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/895/F5.medium.gif" width="440" height="409"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/3/895/F5.medium.gif" width="440" height="409"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/895/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/895/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483634" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-35" class="first-child">Comparison of predictability of REML-estimated <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F5/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F5/embed/mml-math-167.gif"/></noscript></span></span> with the maximum achievable predictability by tuning <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F5/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F5/embed/mml-math-168.gif"/></noscript></span></span> for 1000 metabolomic traits of rice. (A) Maximum achievable predictability of the HAT method by tuning <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F5/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F5/embed/mml-math-169.gif"/></noscript></span></span> plotted against the predictability when <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F5/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F5/embed/mml-math-170.gif"/></noscript></span></span> takes the REML estimate. (B) Maximum achievable predictability of the GCV method by tuning <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F5/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F5/embed/mml-math-171.gif"/></noscript></span></span> plotted against the predictability when <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F5/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F5/embed/mml-math-172.gif"/></noscript></span></span> takes the REML estimate. (C) Predictability of the GCV method plotted against the predictability of the HAT method when <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F5/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F5/embed/mml-math-173.gif"/></noscript></span></span> takes the REML estimate. The red points indicate traits with visible differences in predictability between the method shown on the <em>x</em>-axis and the method shown on the <em>y</em>-axis. The linear regression equation is given at the bottom of each panel and the fitted <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F5/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/F5/embed/mml-math-174.gif"/></noscript></span></span> of the regression is given at the top of each panel. GCV, generalized cross-validation; REML, restricted maximum likelihood.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><p id="p-36">Very recently, <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Gianola and Schon (2016)</a> published methods that are very similar to our HAT method to evaluate the predictability of a genomic selection model. They also recognized the approximation nature of the method when the smoothing parameter <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-175.gif"/></noscript></span></span> is replaced by the estimated value from the whole sample. Their justification of the use of this whole sample-estimated parameter, particularly in LOOCV, is that the estimated <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-176.gif"/></noscript></span></span> from the whole sample will not be much different from the ones obtained from the training samples that differ from the whole sample by just one observation. They actually investigated the variation of <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-177.gif"/></noscript></span></span> across all training samples and found that the variance is indeed small. <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Gianola and Schon (2016)</a> investigated the properties of the new methods in many different situations using an inbred population of wheat (<em>n</em> = 599) to see how the predictability changes when the training and test sample size ratio changes. These exhaustive investigations would take months or years to complete if the ordinary CV were carried out. In addition to BLUP, these authors also extended the method to RKHS (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Gianola <em>et al.</em> 2006</a>) and the Bayesian alphabetic series (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gianola 2013</a>) by modifying the importance sampling schemes.</p><p id="p-37">One important issue that was not addressed in <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Gianola and Schon (2016)</a> is how much difference in predictability calculated between the fast method and the classical CV method can be expected. This question is fundamental because the new method represents a significant technical improvement in genomic selection and will be adopted widely soon after the GS community recognizes it. In our study, we particularly focused on this question and investigated the difference using seven traits from an inbred population of rice, 1000 metabolomic traits from the same inbred population, 10 traits from a hybrid population of rice, and one trait (human height) from a large human population. We found that the HAT method always provides a slightly biased predictability over that of the CV method. However, the bias is never sufficiently severe to distort the conclusion on the predictability of a model. For example, in the human height prediction, the 10-fold CV produced a predictability of <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-178.gif"/></noscript></span></span> and the corresponding number from the 10-fold HAT method was <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-179.gif"/></noscript></span></span> However, the model FIT is 0.8789. The HAT method gave a number much closer to the CV predictability than the model FIT.</p><p id="p-38">In addition to comparing the differences between the HAT method and the ordinary CV, we also compared the new HAT method with the GCV method (<a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">Golab <em>et al.</em> 1979</a>) and found that the two produced very similar results. <a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Craven and Wahba (1979)</a> compared GCV with CV and concluded that the smoothing parameter that maximizes the CV was amazingly close to the parameter that maximizes GCV. The GCV method has been available for almost four decades, but the genomic selection community, except <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Mathew <em>et al.</em> (2015)</a>, has never paid attention to it. Our study showed that both GCV and HAT can be applied to genomic selection. However, the HAT method directly addresses prediction of future individuals and therefore it is more intuitive to interpret the result.</p><p id="p-39"><a id="xref-ref-18-7" class="xref-bibr" href="#ref-18">Hastie <em>et al.</em> (2008)</a> claimed that LOOCV provides a biased prediction compared with CV with lower number of folds. We observed that when the number of folds is 10 or above, the predictability stabilizes (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>, last panel). We did not observe a progressive increase of the predictability as the number of folds increases. Therefore, from our study, we recommend to perform LOOCV with the HAT method to avoid variation caused by random partitioning of the samples when the number of folds is small. When 10-fold or fivefold CV is carried out, the analysis will only be conducted 10 or 5 times, which may not be significant; therefore, the HAT method may lose its appeal. This statement may not be true considering the fact that the 10-fold CV must be run many times to reduce the variation caused by random partitioning of the samples. A multiple CV analysis for large samples is a significant burden to investigators. Therefore, the HAT method is a good alternative to CV to evaluate a genomic selection program.</p><p id="p-40">We originally hoped to see a significant improvement in predictability by tuning the smoothing parameter around the REML-estimated parameter. It is disappointing that there were very few significant improvements from predictions of 1000 traits. The largest improvement occurred for the 422th metabolite with an improvement of <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-180.gif"/></noscript></span></span> (see the red point most deviating away from the diagonal line in <a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5A</a>). The good news is that, in most cases, the REML estimate is close to the MPE and, therefore, the parameter does not need to be tuned. On the other hand, since the computation is simple and fast, why not go ahead to tune the parameter and, if lucky, we may get an improved predictability, like the 422th metabolite in the inbred rice population.</p><p id="p-41">In mixed model prediction, the random effects are often the targets for prediction. This is the case in genomic prediction because the genetic values are treated as random effects. However, if the investigators are interested in prediction using the fixed effects only under the mixed model, the estimated marginal residual error needs to be adjusted by the leverage values from the fixed model hat matrix <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-181.gif"/></noscript></span></span> (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Schabenberger 2004</a>). Let <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-182.gif"/></noscript></span></span> be the estimated marginal residual errors for individuals in the <em>k</em>th fold, the predicted marginal residual errors are approximated by <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-183.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-184.gif"/></noscript></span></span> is the diagonal block of <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-185.gif"/></noscript></span></span> corresponding to observations in the <em>k</em>th fold. The MIXED procedure in SAS calls this method the noniterative influence diagnostics while the iterative influence diagnostics is through actual CV (refit model and reestimate covariance parameters). The noniterative and iterative influence diagnostics can be interpreted as the HAT method and the CV method, respectively. PROC MIXED does not provide influence diagnostics for prediction of random effects. If there is an interest in both the fixed and random effects for prediction, the HAT matrix should include both the fixed model part and the random model part of the HAT matrix, <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-186.gif"/></noscript></span></span> The estimated conditional residual errors are <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-187.gif"/></noscript></span></span> and the predicted conditional residual errors are obtained by <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-188.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-189.gif"/></noscript></span></span> is the diagonal block of <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-190.gif"/></noscript></span></span> corresponding to observations in the <em>k</em>th fold.</p><p id="p-42">When the mixed model includes multiple covariance structures, say <em>S</em> covariance structures, a similar <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-191.gif"/></noscript></span></span> matrix is used except that the <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-192.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-193.gif"/></noscript></span></span> matrices in <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-194.gif"/></noscript></span></span> are replaced by <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-195.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-196.gif"/></noscript></span></span> respectively, where <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-197.gif"/></noscript></span></span> is the <em>s</em>th covariance structure and <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-198.gif"/></noscript></span></span> is the corresponding variance. An example of the multiple variance component model is the model with nonadditive variances that include dominance and epistasis (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Xu 2013</a>). <a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Gianola and Schon (2016)</a> also extended the new method to handle multiple kernels.</p><p id="p-43">The HAT method applies to fixed models (exact result) and linear mixed models (approximate result). Is it possible to extend the HAT method to LASSO and PLS (partial lest squares)? An approximate extension may be possible by fixing the shrinkage parameter, like the extension to BLUP, but there is no exact extension. To carry out that approximate extension, we need to find the HAT function of the predicted <em>y</em> on the observed <em>y</em>, <em>e.g.</em>, <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-199.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-200.gif"/></noscript></span></span> In general, the HAT matrix is <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-201.gif"/></noscript></span></span> (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Schabenberger 2004</a>), a Jacobian matrix holding each derivative of a predicted quantity with respect to an observed response.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-52">The author thanks two anonymous reviewers and the associate editor for their constructive comments and suggestions on the first version of the manuscript. The author is also grateful to Yanru Cui (postdoc) and Yang Xu (student) for their help in calculating the marker-inferred kinship matrices for the hybrid rice and human populations. The project was supported by a National Science Foundation Collaborative Research grant (DBI-1458515) to S.X.</p></div><div class="section app" id="app-1"><h2 class="">Appendix A</h2><div class="section" id="sec-15"><h3>Derivation of HAT prediction under the fixed model methodology</h3></div><div class="section" id="sec-16"><h3>Predicted residual error sum of squares (PRESS)</h3><p id="p-44">In a K-fold CV analysis, let <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-202.gif"/></noscript></span></span> be the number of observations in the <em>k</em>th fold for <span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-203.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-204.gif"/></noscript></span></span> Define <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-205.gif"/></noscript></span></span> as an <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-206.gif"/></noscript></span></span> matrix of independent variables for the individuals in the <em>k</em>th fold. The “leverage” values for the <em>k</em>th fold is defined as an <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-207.gif"/></noscript></span></span> matrix,<span class="disp-formula" id="disp-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-208.gif"/></noscript></span><span class="disp-formula-label">(A1)</span></span>where <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-209.gif"/></noscript></span></span> is the <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-210.gif"/></noscript></span></span> subset of matrix <em>W</em> corresponding to the <em>k</em>th fold. This matrix must appear in the end, not in the beginning, of the above equation. Let<span class="disp-formula" id="disp-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-211.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-211.gif"/></noscript></span><span class="disp-formula-label">(A2)</span></span>be the estimated residual errors where <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-212.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-212.gif"/></noscript></span></span> is estimated from the whole sample. The predicted residual errors for the <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-213.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-213.gif"/></noscript></span></span> individuals in the <em>k</em>th fold is<span class="disp-formula" id="disp-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-214.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-214.gif"/></noscript></span><span class="disp-formula-label">(A3)</span></span>Therefore, the PRESS is defined as<span class="disp-formula" id="disp-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-215.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-215.gif"/></noscript></span><span class="disp-formula-label">(A4)</span></span>which is the weighted sum of squares of the predicted residual errors.</p></div><div class="section" id="sec-17"><h3>Derivation of PRESS</h3><p id="p-45">The linear model for <em>p</em> independent variables (including the intercept) and <em>n</em> observations is<span class="disp-formula" id="disp-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-216.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-216.gif"/></noscript></span><span class="disp-formula-label">(A5)</span></span>The weighted least squares estimates of all regression coefficients are obtained using<span class="disp-formula" id="disp-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-217.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-217.gif"/></noscript></span><span class="disp-formula-label">(A6)</span></span>The estimated residual error variance is<span class="disp-formula" id="disp-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-218.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-218.gif"/></noscript></span><span class="disp-formula-label">(A7)</span></span>The variance-covariance matrix of the estimated regression coefficients are calculated using<span class="disp-formula" id="disp-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-219.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-219.gif"/></noscript></span><span class="disp-formula-label">(A8)</span></span>which is a <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-220.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-220.gif"/></noscript></span></span> matrix with diagonal elements being the variances and off-diagonal elements being the covariance.</p><p id="p-46">The fitted values for all individuals in the population are<span class="disp-formula" id="disp-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-221.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-221.gif"/></noscript></span><span class="disp-formula-label">(A9)</span></span>Let us define a hat matrix by<span class="disp-formula" id="disp-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-222.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-222.gif"/></noscript></span><span class="disp-formula-label">(A10)</span></span>Therefore, the fitted values are a hat function of the observed values,<span class="disp-formula" id="disp-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-223.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-223.gif"/></noscript></span><span class="disp-formula-label">(A11)</span></span>Let us partition the sample into <em>K</em> parts (folds) and denote the number of individuals in the <em>k</em>th fold by <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-224.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-224.gif"/></noscript></span></span> Define <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-225.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-225.gif"/></noscript></span></span> as an <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-226.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-226.gif"/></noscript></span></span> vector, which is a subset of <em>y</em> that contains all observations in the <em>k</em>th fold. Define <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-227.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-227.gif"/></noscript></span></span> as the <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-228.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-228.gif"/></noscript></span></span> rows of matrix <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-229.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-229.gif"/></noscript></span></span> corresponding to the observations in the <em>k</em>th fold. The predicted residual errors are<span class="disp-formula" id="disp-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-230.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-230.gif"/></noscript></span><span class="disp-formula-label">(A12)</span></span>where<span class="disp-formula" id="disp-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-231.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-231.gif"/></noscript></span><span class="disp-formula-label">(A13)</span></span>are the estimated regression coefficients from the data with the <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-232.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-232.gif"/></noscript></span></span> observations in the <em>k</em>th fold being excluded. Let us make the following matrix decomposition,<span class="disp-formula" id="disp-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-233.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-233.gif"/></noscript></span><span class="disp-formula-label">(A14)</span></span>Therefore,<span class="disp-formula" id="disp-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-234.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-234.gif"/></noscript></span><span class="disp-formula-label">(A15)</span></span>Similarly, we can rewrite<span class="disp-formula" id="disp-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-235.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-235.gif"/></noscript></span><span class="disp-formula-label">(A16)</span></span>Using Woodbury matrix identity (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Woodbury 1950</a>), we have<span class="disp-formula" id="disp-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-236.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-236.gif"/></noscript></span><span class="disp-formula-label">(A17)</span></span>where<span class="disp-formula" id="disp-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-237.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-237.gif"/></noscript></span><span class="disp-formula-label">(A18)</span></span>is an <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-238.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-238.gif"/></noscript></span></span> matrix of leverage values for the <em>k</em>th fold. This matrix is the <span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-239.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-239.gif"/></noscript></span></span> diagonal block of the hat matrix <em>H</em>. Further derivation leads to<span class="disp-formula" id="disp-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-240.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-240.gif"/></noscript></span><span class="disp-formula-label">(A19)</span></span>and<span class="disp-formula" id="disp-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-241.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-241.gif"/></noscript></span><span class="disp-formula-label">(A20)</span></span>Therefore, the predicted residual errors are<span class="disp-formula" id="disp-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-242.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-242.gif"/></noscript></span><span class="disp-formula-label">(A21)</span></span>Note that<span class="disp-formula" id="disp-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-243.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-243.gif"/></noscript></span><span class="disp-formula-label">(A22)</span></span>Therefore,<span class="disp-formula" id="disp-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-244.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-244.gif"/></noscript></span><span class="disp-formula-label">(A23)</span></span>The coefficient of <span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-245.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-245.gif"/></noscript></span></span> in <a id="xref-disp-formula-51-1" class="xref-disp-formula" href="#disp-formula-51">Equation A21</a> is<span class="disp-formula" id="disp-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-246.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-246.gif"/></noscript></span><span class="disp-formula-label">(A24)</span></span>which is identical to the coefficient of <span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-247.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-247.gif"/></noscript></span></span> in <a id="xref-disp-formula-51-2" class="xref-disp-formula" href="#disp-formula-51">Equation A21</a>. Therefore,<span class="disp-formula" id="disp-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-248.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-248.gif"/></noscript></span><span class="disp-formula-label">(A25)</span></span>We can see that the predicted residual errors are a linear function of the estimated residual errors. The next step is to simplify the linear function,<span class="disp-formula" id="disp-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-249.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-249.gif"/></noscript></span><span class="disp-formula-label">(A26)</span></span>Therefore, the predicted residual errors have been expressed as a simple linear function of the estimated residual errors,<span class="disp-formula" id="disp-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-250.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-250.gif"/></noscript></span><span class="disp-formula-label">(A27)</span></span>The predicted residual sum of squares (PRESS) is<span class="disp-formula" id="disp-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-251.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-251.gif"/></noscript></span><span class="disp-formula-label">(A28)</span></span>Let us define<span class="disp-formula" id="disp-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-252.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-252.gif"/></noscript></span><span class="disp-formula-label">(A29)</span></span>The PRESS is written as<span class="disp-formula" id="disp-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-253.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-253.gif"/></noscript></span><span class="disp-formula-label">(A30)</span></span>The PRESS is often translated into <em>R</em>-square to represent the predictability of a model,<span class="disp-formula" id="disp-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-254.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-254.gif"/></noscript></span><span class="disp-formula-label">(A31)</span></span>where SST is the total sum of squares of the response variable.</p></div></div><div class="section app" id="app-2"><h2 class="">Appendix B</h2><div class="section" id="sec-18"><h3>Proof of the HAT Method for PRESS in Mixed Models</h3></div><div class="section" id="sec-19"><h3>Estimated random effects</h3><p id="p-47">Let us define<span class="disp-formula" id="disp-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-255.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-255.gif"/></noscript></span><span class="disp-formula-label">(B1)</span></span>as the phenotypic values of the trait adjusted by the fixed effects, assuming that <span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-256.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-256.gif"/></noscript></span></span> is known. The estimated random effects are more appropriately called the fitted random effects. Let us define the estimated vector of random effects by<span class="disp-formula" id="disp-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-257.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-257.gif"/></noscript></span><span class="disp-formula-label">(B2)</span></span>where <span class="inline-formula" id="inline-formula-195"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-258.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-258.gif"/></noscript></span></span> is the HAT matrix, <span class="inline-formula" id="inline-formula-196"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-259.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-259.gif"/></noscript></span></span> is the variance ratio and <span class="inline-formula" id="inline-formula-197"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-260.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-260.gif"/></noscript></span></span> is the kinship matrix. In the main text, we used <em>A</em> in place of <em>K</em>. Here we used <em>K</em> again to be consistent with the genomic selection literature. Let us define <span class="inline-formula" id="inline-formula-198"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-261.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-261.gif"/></noscript></span></span> as the estimated residual error for the <em>j</em>th observation or <em>j</em>th block of observations. The predicted residual error for the <em>j</em>th block of individuals is<span class="disp-formula" id="disp-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-262.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-262.gif"/></noscript></span><span class="disp-formula-label">(B3)</span></span>The purpose of this appendix is to prove <a id="xref-disp-formula-64-1" class="xref-disp-formula" href="#disp-formula-64">Equation B3</a> that the predicted residual error can be obtained from the estimated residual error via the leverage value (diagonal element or diagonal block) of the HAT matrix.</p><p id="p-48">Let us partition the <em>K</em> matrix into<span class="disp-formula" id="disp-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-263.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-263.gif"/></noscript></span><span class="disp-formula-label">(B4)</span></span>where <span class="inline-formula" id="inline-formula-199"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-264.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-264.gif"/></noscript></span></span> is the <em>j</em>th diagonal element of the <em>K</em> matrix, <span class="inline-formula" id="inline-formula-200"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-265.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-265.gif"/></noscript></span></span> is the <em>j</em>th row of matrix <em>K</em> that excludes the <em>j</em>th column, and <span class="inline-formula" id="inline-formula-201"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-266.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-266.gif"/></noscript></span></span> is the <em>K</em> matrix excluding the <em>j</em>th row and the <em>j</em>th column. Corresponding to this partitioning, matrix <span class="inline-formula" id="inline-formula-202"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-267.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-267.gif"/></noscript></span></span> can also be partitioned into<span class="disp-formula" id="disp-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-268.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-268.gif"/></noscript></span><span class="disp-formula-label">(B5)</span></span>The inverse of the above partitioned matrix is denoted by<span class="disp-formula" id="disp-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-269.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-269.gif"/></noscript></span><span class="disp-formula-label">(B6)</span></span>where<span class="disp-formula" id="disp-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-270.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-270.gif"/></noscript></span><span class="disp-formula-label">(B7)</span></span>The estimated (fitted) value of the <em>j</em>th individual is<span class="disp-formula" id="disp-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-271.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-271.gif"/></noscript></span><span class="disp-formula-label">(B8)</span></span>which is eventually expressed as</p><span class="disp-formula" id="disp-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-272.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-272.gif"/></noscript></span><span class="disp-formula-label">(B9)</span></span></div><div class="section" id="sec-20"><h3>Predicted random effects</h3><p id="p-49">The predicted value for the <em>j</em>th individual is obtained by excluding the contribution from the same individual, as expressed below,<span class="disp-formula" id="disp-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-273.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-273.gif"/></noscript></span><span class="disp-formula-label">(B10)</span></span>Let us examine the four terms in the fitted value given in <a id="xref-disp-formula-70-1" class="xref-disp-formula" href="#disp-formula-70">Equation B9</a>,<span class="disp-formula" id="disp-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-274.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-274.gif"/></noscript></span><span class="disp-formula-label">(B11)</span></span>Substituting these four terms into <a id="xref-disp-formula-70-2" class="xref-disp-formula" href="#disp-formula-70">Equation B9</a>, we get<span class="disp-formula" id="disp-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-275.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-275.gif"/></noscript></span><span class="disp-formula-label">(B12)</span></span>Note that the fitted random effect for the <em>j</em>th individual has been expressed as a linear function of the predicted random effect.</p></div><div class="section" id="sec-21"><h3>Estimated and predicted errors</h3><p id="p-50">Let us define <span class="inline-formula" id="inline-formula-203"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-276.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-276.gif"/></noscript></span></span> as the estimated error and <span class="inline-formula" id="inline-formula-204"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-277.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-277.gif"/></noscript></span></span> as the predicted error. We then define<span class="disp-formula" id="disp-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-278.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-278.gif"/></noscript></span><span class="disp-formula-label">(B13)</span></span>After a few steps of manipulation, we have<span class="disp-formula" id="disp-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-279.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-279.gif"/></noscript></span><span class="disp-formula-label">(B14)</span></span>Therefore, the estimated and predicted errors have the following relationship,<span class="disp-formula" id="disp-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-280.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-280.gif"/></noscript></span><span class="disp-formula-label">(B15)</span></span>We want to prove that the <em>j</em>th diagonal element of the HAT matrix (the leverage value for observation <em>j</em>) is<span class="disp-formula" id="disp-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-281.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-281.gif"/></noscript></span><span class="disp-formula-label">(B16)</span></span>which leads to<span class="disp-formula" id="disp-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-282.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-282.gif"/></noscript></span><span class="disp-formula-label">(B17)</span></span>As a result,<span class="disp-formula" id="disp-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-283.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-283.gif"/></noscript></span><span class="disp-formula-label">(B18)</span></span>We now go back to the HAT matrix to see what <span class="inline-formula" id="inline-formula-205"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-284.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-284.gif"/></noscript></span></span> is. Using partitioned matrix, we have<span class="disp-formula" id="disp-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-285.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-285.gif"/></noscript></span><span class="disp-formula-label">(B19)</span></span>Therefore,<span class="disp-formula" id="disp-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-286.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-286.gif"/></noscript></span><span class="disp-formula-label">(B20)</span></span>From <a id="xref-disp-formula-72-1" class="xref-disp-formula" href="#disp-formula-72">Equation B11</a>, we know<span class="disp-formula" id="disp-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-287.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-287.gif"/></noscript></span><span class="disp-formula-label">(B21)</span></span>Substituting <a id="xref-disp-formula-82-1" class="xref-disp-formula" href="#disp-formula-82">Equation B21</a> into <a id="xref-disp-formula-81-1" class="xref-disp-formula" href="#disp-formula-81">Equation B20</a> yields<span class="disp-formula" id="disp-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-288.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-288.gif"/></noscript></span><span class="disp-formula-label">(B22)</span></span>which is exactly the same as <a id="xref-disp-formula-77-1" class="xref-disp-formula" href="#disp-formula-77">Equation B16</a>; thus, we conclude the derivation of <a id="xref-disp-formula-79-1" class="xref-disp-formula" href="#disp-formula-79">Equation B18</a>. Unlike the HAT matrix in fixed models, the random model HAT matrix is not idempotent, although it remains symmetric.</p><p id="p-51">The PRESS is now defined as<span class="disp-formula" id="disp-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-289.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-289.gif"/></noscript></span><span class="disp-formula-label">(B23)</span></span>If the residual errors are heterogeneous with <span class="inline-formula" id="inline-formula-206"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-290.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-290.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-207"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-291.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-291.gif"/></noscript></span></span> is a known diagonal matrix, all the above derivations apply except that we have to replace <span class="inline-formula" id="inline-formula-208"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-292.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-292.gif"/></noscript></span></span> by <span class="inline-formula" id="inline-formula-209"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-293.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-293.gif"/></noscript></span></span> in all occurrences. In addition, the PRESS should be modified as a weighted PRESS,<span class="disp-formula" id="disp-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-294.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-294.gif"/></noscript></span><span class="disp-formula-label">(B24)</span></span>where <span class="inline-formula" id="inline-formula-210"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-295.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/895/embed/mml-math-295.gif"/></noscript></span></span> is the weight for the <em>j</em>th observation or the <em>j</em>th block of observations.</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-53">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038059/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038059/-/DC1</a>.</p></li><li class="fn" id="fn-4"><p id="p-54"><em>Communicating editor: G. A. de los Campos</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-11-30"><span class="received-label">Received </span>November 30, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-01-09"><span class="accepted-label">Accepted </span>January 9, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Xu</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.1" data-doi="10.2307/1267161"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">Mean square error of prediction as a criterion for selecting variables.</span> <abbr class="cit-jnl-abbrev">Technometrics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">469</span>–<span class="cit-lpage">475</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTechnometrics%26rft.volume%253D13%26rft.spage%253D469%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1267161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1267161&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1971K151400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.2" data-doi="10.2307/1267500"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The relationship between variable selection and data augmentation and a method for prediction</span>. <abbr class="cit-jnl-abbrev">Technometrics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">125</span>–<span class="cit-lpage">127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTechnometrics%26rft.volume%253D16%26rft.spage%253D125%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1267500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1267500&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1974S183000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.3" data-doi="10.2307/1268249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Detection of influential observation in linear regression.</span> <abbr class="cit-jnl-abbrev">Technometrics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTechnometrics%26rft.volume%253D19%26rft.spage%253D15%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1268249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1268249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1977CV78500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.4" data-doi="10.2307/2286747"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Influential observation in linear regression.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">169</span>–<span class="cit-lpage">174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D74%26rft.spage%253D169%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2286747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2286747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1979GQ49900035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.5" data-doi="10.1007/BF01404567"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Craven</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wahba</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Smoothing noisy data with spline functions: estimating the correct degree of smoothing by the method of generalized cross-validation.</span> <abbr class="cit-jnl-abbrev">Numer. Math.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">377</span>–<span class="cit-lpage">403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNumer.%2BMath.%26rft.volume%253D31%26rft.spage%253D377%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01404567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01404567&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1979GE19400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.6" data-doi="10.2105/AJPH.41.3.279"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dawber</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meadors</span>  <span class="cit-name-given-names">G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">F. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1951</span> <span class="cit-article-title">Epidemiological approaches to heart disease - the Framingham study.</span> <abbr class="cit-jnl-abbrev">Am. J. Public Health Nations Health</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">279</span>–<span class="cit-lpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BPublic%2BHealth%2BNations%2BHealth%26rft.volume%253D41%26rft.spage%253D279%26rft_id%253Dinfo%253Adoi%252F10.2105%252FAJPH.41.3.279%26rft_id%253Dinfo%253Apmid%252F14819398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2105/AJPH.41.3.279&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14819398&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F895.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000206444200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dawber</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kannel</span>  <span class="cit-name-given-names">W. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyell</span>  <span class="cit-name-given-names">L. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1963</span> <span class="cit-article-title">An approach to longitudinal studies in a community - Framingham study.</span> <abbr class="cit-jnl-abbrev">Ann. N. Y. Acad. Sci.</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bthe%2BNew%2BYork%2BAcademy%2Bof%2BSciences%26rft.stitle%253DAnn.%2BN.%2BY.%2BAcad.%2BSci.%26rft.aulast%253DDawber%26rft.auinit1%253DT.%2BR.%26rft.volume%253D107%26rft.spage%253D539%26rft.epage%253D556%26rft.atitle%253DAn%2Bapproach%2Bto%2Blongitudinal%2Bstudies%2Bin%2Ba%2Bcommunity%253A%2Bthe%2BFramingham%2BStudy.%26rft_id%253Dinfo%253Apmid%252F14025561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=14025561&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F895.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A19634812A00008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.8" data-doi="10.1534/genetics.109.101501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting quantitative traits with regression models for dense molecular markers and pedigree.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101501%26rft_id%253Dinfo%253Apmid%252F19293140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzc1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzg5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.9" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">a</span> <span class="cit-article-title">Whole-genome regression and prediction methods applied to plant and animal breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzg5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.10" data-doi="10.1371/journal.pgen.1003608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klimentidis</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">b</span> <span class="cit-article-title">Prediction of complex human traits using the genomic best linear unbiased predictor.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003608%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003608%26rft_id%253Dinfo%253Apmid%252F23874214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003608&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23874214&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F895.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.11" data-doi="10.1371/journal.pgen.1005048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic heritability: what is it?</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005048</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005048%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005048%26rft_id%253Dinfo%253Apmid%252F25942577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25942577&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F895.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.12" data-doi="10.1534/genetics.113.151753"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Priors in whole-genome regression: the Bayesian alphabet returns.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">596</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.151753%26rft_id%253Dinfo%253Apmid%252F23636739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTczIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzg5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schon</span>  <span class="cit-name-given-names">C.-C.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Cross-validation without doing cross-validation in genome-enabled prediction.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">6</span>(<span class="cit-supplement">10</span>): <span class="cit-fpage">3107</span>–<span class="cit-lpage">3128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D6%26rft.spage%253D3107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.14" data-doi="10.1534/genetics.105.049510"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stella</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genomic-assisted prediction of genetic value with semiparametric procedures.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">1761</span>–<span class="cit-lpage">1776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.049510%26rft_id%253Dinfo%253Apmid%252F16648593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3My8zLzE3NjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvODk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.15" data-doi="10.1111/j.1439-0388.2007.00702.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genomic selection.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%2BE.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D323%26rft.epage%253D330%26rft.atitle%253DGenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2007.00702.x%26rft_id%253Dinfo%253Apmid%252F18076469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2007.00702.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18076469&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F895.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251432800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.16" data-doi="10.2307/1268518"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Golab</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heath</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wahba</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Generalized cross-validation as a method for choosing a good ridge parameter.</span> <abbr class="cit-jnl-abbrev">Technometrics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">215</span>–<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTechnometrics%26rft.volume%253D21%26rft.spage%253D215%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1268518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1268518&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1979GW16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.17" data-doi="10.1073/pnas.1319681110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic analysis of the metabolome exemplified using a rice population.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">20320</span>–<span class="cit-lpage">20325</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1319681110%26rft_id%253Dinfo%253Apmid%252F24259710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzUwLzIwMzIwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzg5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-book" id="cit-7.3.895.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-source"><em>The Elements of Statistical Learning: Data Mining</em>, <em>Inference</em>, <em>and Prediction</em></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.19" data-doi="10.2135/cropsci2008.08.0512"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection for crop improvement.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHeffner%26rft.auinit1%253DE.%2BL.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D12%26rft.atitle%253DGenomic%2BSelection%2Bfor%2BCrop%2BImprovement%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.08.0512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.08.0512&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000263344300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.20" data-doi="10.2307/2529430"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Best linear unbiased estimation and prediction under a selection model.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">447</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DHenderson%26rft.auinit1%253DC.%2BR.%26rft.volume%253D31%26rft.issue%253D2%26rft.spage%253D423%26rft.epage%253D447%26rft.atitle%253DBest%2Blinear%2Bunbiased%2Bestimation%2Band%2Bprediction%2Bunder%2Ba%2Bselection%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2529430%26rft_id%253Dinfo%253Apmid%252F1174616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2529430&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1174616&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F895.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975AF09900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.21" data-doi="10.2307/1267352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoerl</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kennard</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span><span class="cit-comment">a</span> <span class="cit-article-title">Ridge regression: applications to nonorthogonal problems.</span> <abbr class="cit-jnl-abbrev">Technometrics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTechnometrics%26rft.volume%253D12%26rft.spage%253D69%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1267352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1267352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1970F898700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.22" data-doi="10.2307/1267351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoerl</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kennard</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span><span class="cit-comment">b</span> <span class="cit-article-title">Ridge regression: biased estimation for nonorthogonal problems.</span> <abbr class="cit-jnl-abbrev">Technometrics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">55</span>–<span class="cit-lpage">67</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTechnometrics%26rft.volume%253D12%26rft.spage%253D55%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1267351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1267351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1970F898700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.23" data-doi="10.1038/ncomms7258"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic analysis of hybrid rice varieties reveals numerous superior alleles that contribute to heterosis.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">6258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D6%26rft.spage%253D6258%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms7258%26rft_id%253Dinfo%253Apmid%252F25651972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms7258&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25651972&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F895.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malfait</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bultheel</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Generalized cross validation for wavelet thresholding.</span> <abbr class="cit-jnl-abbrev">Signal Process.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSignal%2BProcess.%26rft.volume%253D56%26rft.spage%253D33%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">K.-C.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Asymptotic optimality for C<sub>p</sub>, C<sub>L</sub>, cross-validation and generalized cross-validation: discrete index set.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">958</span>–<span class="cit-lpage">975</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D15%26rft.spage%253D958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mathew</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Léon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sillanpää</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Integrated nested Laplace approximation inference and cross-validation to tune variance components in estimation of breeding value.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">99</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D35%26rft.spage%253D99%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvODk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.28" data-doi="10.1093/biomet/58.3.545"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">Recovery of inter-block information when block sizes are unequal.</span> <abbr class="cit-jnl-abbrev">Biometrika</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">545</span>–<span class="cit-lpage">554</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft.stitle%253DBiometrika%26rft.aulast%253DPATTERSON%26rft.auinit1%253DH.%2BD.%26rft.volume%253D58%26rft.issue%253D3%26rft.spage%253D545%26rft.epage%253D554%26rft.atitle%253DRecovery%2Bof%2Binter-block%2Binformation%2Bwhen%2Bblock%2Bsizes%2Bare%2Bunequal%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252F58.3.545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYmlvbWV0IjtzOjU6InJlc2lkIjtzOjg6IjU4LzMvNTQ1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzg5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.29" data-doi="10.2307/2288403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Picard</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Cross-validation of regression models</span>. <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">575</span>–<span class="cit-lpage">583</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D79%26rft.spage%253D575%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2288403%26rft_id%253Dinfo%253Apmid%252FCan%2527t%2Brun%2BEX%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2288403&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=Can't run EX&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F895.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984TK97500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.3.895.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schabenberger</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Mixed model influence diagnostics</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">17</span> in <span class="cit-source"><em>Proceedings of the Twenty-Ninth Annual SAS Users Group International Conference</em></span>. <span class="cit-publ-name">SAS Institute Inc.</span>, <span class="cit-publ-loc">Montréal, Canada</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.31" data-doi="10.1375/136905203770326402"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Silventoinen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sammalisto</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perola</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boomsma</span>  <span class="cit-name-given-names">D. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cornes</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Heritability of adult body height: a comparative study of twin cohorts in eight countries.</span> <abbr class="cit-jnl-abbrev">Twin Res.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">408</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTwin%2Bresearch%2B%253A%2B%2Bthe%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BSociety%2Bfor%2BTwin%2BStudies%26rft.stitle%253DTwin%2BRes%26rft.aulast%253DSilventoinen%26rft.auinit1%253DK.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253D399%26rft.epage%253D408%26rft.atitle%253DHeritability%2Bof%2Badult%2Bbody%2Bheight%253A%2Ba%2Bcomparative%2Bstudy%2Bof%2Btwin%2Bcohorts%2Bin%2Beight%2Bcountries.%26rft_id%253Dinfo%253Adoi%252F10.1375%252F136905203770326402%26rft_id%253Dinfo%253Apmid%252F14624724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1375/136905203770326402&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14624724&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F895.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187573800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Regression shrinkage and selection via the Lasso.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB%26rft.volume%253D58%26rft.spage%253D267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.33" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F895.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.34" data-doi="10.1534/genetics.112.141705"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klimentidis</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A comprehensive genetic approach for improving prediction of skin cancer risk in humans.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1493</span>–<span class="cit-lpage">1502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.141705%26rft_id%253Dinfo%253Apmid%252F23051645%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzE0OTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvODk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.35" data-doi="10.1007/BF01437407"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wahba</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Smoothing noisy data with spline functions</span>. <abbr class="cit-jnl-abbrev">Numer. Math.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">383</span>–<span class="cit-lpage">393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNumer.%2BMath.%26rft.volume%253D24%26rft.spage%253D383%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01437407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01437407&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.3.895.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wahba</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Spline bases, regularization, and generalized crossvalidation for solving approximation problems with large quantities of noisy data</span>, pp. <span class="cit-fpage">905</span>–<span class="cit-lpage">912</span> in <span class="cit-source"><em>International Conference on Approximation Theory in Honour of George Lorenz</em></span>. <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">Austin, TX</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-book" id="cit-7.3.895.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wahba</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-source"><em>Spline Models for Observational Data</em></span>. <span class="cit-publ-name">SIAM</span>, <span class="cit-publ-loc">Philadelphia</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-book" id="cit-7.3.895.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wahba</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-source"><em>Support Vector Machines</em>, <em>Reproducing Kernel Hilbert Spaces and the Randomized GACV</em></span>. <span class="cit-publ-name">University of Wisconsin</span>, <span class="cit-publ-loc">Madison, WI</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wahba</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wold</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span><span class="cit-comment">a</span> <span class="cit-article-title">A completely automatic french curve: fitting spline functions by cross validation.</span> <abbr class="cit-jnl-abbrev">Commun. Stat.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCommun.%2BStat.%26rft.volume%253D4%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wahba</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wold</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span><span class="cit-comment">b</span> <span class="cit-article-title">Periodic splines for spectral density estimation: the use of cross-validation for determining the degree of smoothing.</span> <abbr class="cit-jnl-abbrev">Commun. Stat.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">125</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCommun.%2BStat.%26rft.volume%253D4%26rft.spage%253D125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wahba</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Smoothing spline ANOVA fits for very large, nearly regular data sets, with application to historical global climate data.</span> <abbr class="cit-jnl-abbrev">Ann. Numer. Math.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">579</span>–<span class="cit-lpage">598</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BNumer.%2BMath.%26rft.volume%253D4%26rft.spage%253D579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.42" data-doi="10.1214/aos/1034713638"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wahba</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Smoothing spline ANOVA for exponential families, with application to the Wisconsin epidemiological study of diabetic retinopathy.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1865</span>–<span class="cit-lpage">1895</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D23%26rft.spage%253D1865%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faos%252F1034713638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aos/1034713638&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995UP38900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-book" id="cit-7.3.895.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wahba</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">GACV for support vector machines</span>, pp. <span class="cit-fpage">297</span>–<span class="cit-lpage">311</span>. <span class="cit-source"><em>Advances in Large Margin Classifiers</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Smola</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Bartlett</span>  <span class="cit-name-given-names">P.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Schölkopf</span>  <span class="cit-name-given-names">B.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Schuurmans</span>  <span class="cit-name-given-names">D.</span></span>. <span class="cit-publ-name">MIT Press</span>, <span class="cit-publ-loc">Cambridge, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span><span class="cit-comment">a</span> <span class="cit-article-title">Mixed effects smoothing spline analysis of variance.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. Series B Stat. Methodol.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">159</span>–<span class="cit-lpage">174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BSeries%2BB%2BStat.%2BMethodol.%26rft.volume%253D60%26rft.spage%253D159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.45" data-doi="10.2307/2669630"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span><span class="cit-comment">b</span> <span class="cit-article-title">Smoothing spline models with correlated random errors.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">341</span>–<span class="cit-lpage">348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D93%26rft.spage%253D341%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2669630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2669630&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000072355300034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-other" id="cit-7.3.895.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woodbury</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1950</span> <span class="cit-article-title">Inverting modified matrices</span>, pp. <span class="cit-fpage">4</span>. <span class="cit-source">Memorandum Report</span>, <span class="cit-vol">Volume 42</span>. <span class="cit-publ-name">Statistical Research Group, Princeton University</span>, <span class="cit-publ-loc">Princeton, NJ</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMemorandum%2BReport%26rft.volume%253D42%26rft.spage%253D4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.47" data-doi="10.1534/genetics.113.157032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mapping quantitative trait loci by controlling polygenic background effects.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">1209</span>–<span class="cit-lpage">1222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.157032%26rft_id%253Dinfo%253Apmid%252F24077303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NS80LzEyMDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvODk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.895.48" data-doi="10.1073/pnas.1413750111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Predicting hybrid performance in rice using genomic best linear unbiased prediction.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">12456</span>–<span class="cit-lpage">12461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1413750111%26rft_id%253Dinfo%253Apmid%252F25114224%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzM0LzEyNDU2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzg5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-web" id="cit-7.3.895.49" data-doi="10.1371/journal.pone.0017595"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Gains in QTL detection using an ultra-high density SNP map based on population sequencing relative to traditional RFLP/SSR markers.</span> <span class="cit-source">PLoS One</span> <span class="cit-vol">6</span>: <span class="cit-fpage">e17595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DYu%26rft.auinit1%253DH.%26rft.volume%253D6%26rft.issue%253D3%26rft.spage%253De17595%26rft.epage%253De17595%26rft.atitle%253DGains%2Bin%2BQTL%2Bdetection%2Busing%2Ban%2Bultra-high%2Bdensity%2BSNP%2Bmap%2Bbased%2Bon%2Bpopulation%2Bsequencing%2Brelative%2Bto%2Btraditional%2BRFLP%252FSSR%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0017595%26rft_id%253Dinfo%253Apmid%252F21390234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0017595&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21390234&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F895.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/3/895.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/3/881" title="Adapting Genotyping-by-Sequencing for Rice F2 Populations" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/3/911" title="Functional Analysis of Kinases and Transcription Factors in Saccharomyces cerevisiae Using an Integrated Overexpression Library" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/3"><span>Volume 7 Issue 3, March 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/3.cover-source.jpg?itok=h8yj2GBo" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/3/895" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Nq8QHQs5VCL61SgjgTW943aOjWU4guiPrW2CVYd_T_4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/3/895" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/3/895" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/3/895" class="active" data-icon-position="" data-hide-link-title="0">Predicted Residual Error Sum of Squares of Mixed Models: An Application for Genomic Prediction</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/483532" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-d7SfcXrZForYH6_d-S3I8tgYZz49CLQsMbWRqU8HXns" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/3/895.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/3/895" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-yZnuKnBI7pxkFP5BvzalFvPbOqCCpXAw3E2zngwf2eM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/3/895" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/3/895.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483532" id="node483532--2" data-pisa="ggg;7/3/895" data-pisa-master="ggg;g3.116.038059" data-apath="/ggg/7/3/895.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Predicted Residual Error Sum of Squares of Mixed Models: An Application for Genomic Prediction</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Shizhong Xu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">895-909;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.038059 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483532--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Shizhong Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Botany and Plant Sciences, University of California, Riverside, California 92521</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shizhong%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShizhong%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:shizhong.xu@ucr.edu" class="" data-icon-position="" data-hide-link-title="0">shizhong.xu@ucr.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/3/895" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/3/895&amp;title=Predicted%20Residual%20Error%20Sum%20of%20Squares%20of%20Mixed%20Models%3A%20An%20Application%20for%20Genomic%20Prediction" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/3/895&amp;title=Predicted%20Residual%20Error%20Sum%20of%20Squares%20of%20Mixed%20Models%3A%20An%20Application%20for%20Genomic%20Prediction" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/3/895&amp;title=Predicted%20Residual%20Error%20Sum%20of%20Squares%20of%20Mixed%20Models%3A%20An%20Application%20for%20Genomic%20Prediction" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/3/895&amp;text=Predicted%20Residual%20Error%20Sum%20of%20Squares%20of%20Mixed%20Models%3A%20An%20Application%20for%20Genomic%20Prediction" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/3/895&amp;title=Predicted%20Residual%20Error%20Sum%20of%20Squares%20of%20Mixed%20Models%3A%20An%20Application%20for%20Genomic%20Prediction" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/3/895&amp;t=Predicted%20Residual%20Error%20Sum%20of%20Squares%20of%20Mixed%20Models%3A%20An%20Application%20for%20Genomic%20Prediction" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/3/895&amp;title=Predicted%20Residual%20Error%20Sum%20of%20Squares%20of%20Mixed%20Models%3A%20An%20Application%20for%20Genomic%20Prediction" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/3/895&amp;title=Predicted%20Residual%20Error%20Sum%20of%20Squares%20of%20Mixed%20Models%3A%20An%20Application%20for%20Genomic%20Prediction" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/483532/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483532" id="node483532--3" data-pisa="ggg;7/3/895" data-pisa-master="ggg;g3.116.038059" data-apath="/ggg/7/3/895.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Predicted Residual Error Sum of Squares of Mixed Models: An Application for Genomic Prediction</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Shizhong Xu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">895-909;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.038059 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483532--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Shizhong Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Botany and Plant Sciences, University of California, Riverside, California 92521</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shizhong%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShizhong%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:shizhong.xu@ucr.edu" class="" data-icon-position="" data-hide-link-title="0">shizhong.xu@ucr.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=895&amp;rft.epage=909&amp;rft.atitle=Predicted%20Residual%20Error%20Sum%20of%20Squares%20of%20Mixed%20Models%3A%20An%20Application%20for%20Genomic%20Prediction&amp;rft.volume=7&amp;rft.issue=3&amp;rft.date=2017-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Xu%2C+Shizhong"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/483532/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/483532/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/483532/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/483532/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/483532/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/483532/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/483532/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/483532/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/483532/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/483532/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/483532/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/483532/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/3/895" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix A</a></li><li class="odd"><a href="#app-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix B</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/3/895.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/3/895.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/3/895.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/3/895.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
