<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/3/813" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/3/813.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/3/813.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/3/813.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/3/813/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/3/813.atom" />
<meta name="HW.pisa" content="ggg;7/3/813" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Novel Resampling Improves Statistical Power for Multiple-Trait QTL Mapping" />
<meta name="DC.Identifier" content="10.1534/g3.116.037531" />
<meta name="DC.Date" content="2017-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Cheng et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Multiple-trait analysis typically employs models that associate a quantitative trait locus (QTL) with all of the traits. As a result, statistical power for QTL detection may not be optimal if the QTL contributes to the phenotypic variation in only a small proportion of the traits. Excluding QTL effects that contribute little to the test statistic can improve statistical power. In this article, we show that an optimal power can be achieved when the number of QTL effects is best estimated, and that a stringent criterion for QTL effect selection may improve power when the number of QTL effects is small but can reduce power otherwise. We investigate strategies for excluding trivial QTL effects, and propose a method that improves statistical power when the number of QTL effects is relatively small, and fairly maintains the power when the number of QTL effects is large. The proposed method first uses resampling techniques to determine the number of nontrivial QTL effects, and then selects QTL effects by the backward elimination procedure for significance test. We also propose a method for testing QTL-trait associations that are desired for biological interpretation in applications. We validate our methods using simulations and Arabidopsis thaliana transcript data." />
<meta name="DC.Contributor" content="Riyan Cheng" />
<meta name="DC.Contributor" content="R. W. Doerge" />
<meta name="DC.Contributor" content="Justin Borevitz" />
<meta name="article:published_time" content="2017-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Novel Resampling Improves Statistical Power for Multiple-Trait QTL Mapping" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Multiple-trait analysis typically employs models that associate a quantitative trait locus (QTL) with all of the traits. As a result, statistical power for QTL detection may not be optimal if the QTL contributes to the phenotypic variation in only a small proportion of the traits. Excluding QTL effects that contribute little to the test statistic can improve statistical power. In this article, we show that an optimal power can be achieved when the number of QTL effects is best estimated, and that a stringent criterion for QTL effect selection may improve power when the number of QTL effects is small but can reduce power otherwise. We investigate strategies for excluding trivial QTL effects, and propose a method that improves statistical power when the number of QTL effects is relatively small, and fairly maintains the power when the number of QTL effects is large. The proposed method first uses resampling techniques to determine the number of nontrivial QTL effects, and then selects QTL effects by the backward elimination procedure for significance test. We also propose a method for testing QTL-trait associations that are desired for biological interpretation in applications. We validate our methods using simulations and &lt;i&gt;Arabidopsis thaliana&lt;/i&gt; transcript data.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/03/01" />
<meta name="citation_mjid" content="ggg;7/3/813" />
<meta name="citation_id" content="7/3/813" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/3/813" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/3/813.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/3/813.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/3/813.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.037531" />
<meta name="citation_pmid" content="28064191" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="813" />
<meta name="citation_lastpage" content="822" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Riyan Cheng" />
<meta name="citation_author_institution" content="Research School of Biology, The Australian National University, Acton, Australian Capital Territory 2601, Australia, ARC Center of Excellence in Plant Energy Biology, The Australian National University, Acton, ACT 2601, Australia" />
<meta name="citation_author_email" content="riyan.cheng@anu.edu.au" />
<meta name="citation_author" content="R. W. Doerge" />
<meta name="citation_author_institution" content="Department of Statistics, Department of Biological Sciences, Carnegie Mellon University, Pittsburgh, PA 15213" />
<meta name="citation_author" content="Justin Borevitz" />
<meta name="citation_author_institution" content="Research School of Biology, The Australian National University, Acton, Australian Capital Territory 2601, Australia, ARC Center of Excellence in Plant Energy Biology, The Australian National University, Acton, ACT 2601, Australia" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. Arends;citation_author=P. Prins;citation_author=R. C. Jansen;citation_author=K. W. Broman;citation_title=R/qtl: high-throughput multiple QTL mapping.;citation_volume=26;citation_year=2010;citation_pmid=20966004;citation_doi=10.1093/bioinformatics/btq565" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Banerjee;citation_author=B. S. Yandell;citation_author=N. Yi;citation_title=Bayesian quantitative trait loci mapping for multiple traits.;citation_volume=179;citation_year=2008;citation_pmid=18689903;citation_doi=10.1534/genetics.108.088427" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=S. Bolormaa;citation_author=J. E. Pryce;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Multivariate analysis of a genome-wide association study in dairy cattle.;citation_pages=3818-3833;citation_volume=93;citation_year=2010;citation_issue=8;citation_pmid=20655452;citation_doi=10.3168/jds.2009-2980" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B;citation_author=K. W. Broman;citation_author=T. P. Speed;citation_title=A model selection approach for identification of quantitative trait loci in experimental crosses.;citation_pages=641-656;citation_volume=64;citation_year=2002;citation_doi=10.1111/1467-9868.00354" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Plant Biol.;citation_author=T. B. Brown;citation_author=R. Cheng;citation_author=X. R. R. Sirault;citation_author=T. Rungrat;citation_author=K. D. Murray;citation_title=Traitcapture: genomic and environment modelling of plant phenomic data.;citation_pages=73-79;citation_volume=18;citation_year=2014;citation_pmid=24646691;citation_doi=10.1016/j.pbi.2014.02.002" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Cheng;citation_author=J. Borevitz;citation_author=R. Doerge;citation_title=Selecting informative traits for multivariate quantitative trait locus mapping helps to gain optimal power.;citation_volume=195;citation_year=2013;citation_pmid=23979571;citation_doi=10.1534/genetics.113.155937" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. A. Churchill;citation_author=R. W. Doerge;citation_title=Empirical threshold values for quantitative trait mapping.;citation_pages=963-971;citation_volume=138;citation_year=1994;citation_issue=3;citation_pmid=7851788" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=G. A. Churchill;citation_author=D. C. Airey;citation_author=H. Allayee;citation_author=J. M. Angel;citation_author=A. D. Attie;citation_title=The Collaborative Cross, a community resource for the genetic analysis of complex traits.;citation_pages=1133-1137;citation_volume=36;citation_year=2004;citation_issue=11;citation_pmid=15514660;citation_doi=10.1038/ng1104-1133" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Darvasi;citation_author=M. Soller;citation_title=Advanced intercross lines, an experimental population for fine genetic mapping.;citation_pages=1199-1207;citation_volume=141;citation_year=1995;citation_issue=3;citation_pmid=8582624" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=B. Efron;citation_title=Bootstrap methods: another look at the jackknife.;citation_pages=1-26;citation_volume=7;citation_year=1979;citation_doi=10.1214/aos/1176344552" />
<meta name="citation_reference" content="citation_title=The Design of Experiments;citation_author=R. A. Fisher;citation_year=1935" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. M. Henshall;citation_author=M. E. Goddard;citation_title=Multiple-Trait Mapping of Quantitative Trait Loci After Selective Genotyping Using Logistic Regression;citation_pages=885-894;citation_volume=151;citation_year=1999;citation_issue=2;citation_pmid=9927477" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. V. Hernandez;citation_author=J. Crossa;citation_author=P. K. Singh;citation_author=N. S. Bains;citation_author=K. Singh;citation_title=Multi-trait and multi-environment QTL analyses for resistance to wheat diseases.;citation_pages=e38008-e38008;citation_volume=7;citation_year=2012;citation_issue=6;citation_pmid=22679489;citation_doi=10.1371/journal.pone.0038008" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Jiang;citation_author=Z.-B. Zeng;citation_title=Multiple trait analysis of genetic mapping for quantitative trait loci.;citation_pages=1111-1127;citation_volume=140;citation_year=1995;citation_issue=3;citation_pmid=7672582" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C.-H. Kao;citation_author=Z.-B. Zeng;citation_author=R. D. Teasdale;citation_title=Multiple Interval Mapping for Quantitative Trait Loci;citation_pages=1203-1216;citation_volume=152;citation_year=1999;citation_issue=3;citation_pmid=10388834" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. J. Kliebenstein;citation_author=M. A. West;citation_author=H. van Leeuwen;citation_author=K. Kim;citation_author=R. Doerge;citation_title=Genomic survey of gene expression diversity in Arabidopsis thaliana.;citation_volume=172;citation_year=2006;citation_pmid=16204207;citation_doi=10.1534/genetics.105.049353" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. A. Knott;citation_author=C. S. Haley;citation_title=Multitrait Least Squares for Quantitative Trait Loci Detection;citation_pages=899-911;citation_volume=156;citation_year=2000;citation_issue=2;citation_pmid=11014835" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. B. Korol;citation_author=Y. I. Ronin;citation_author=V. M. Kirzhner;citation_title=Interval mapping of quantitative trait loci employing correlated trait complexes.;citation_pages=1137-1147;citation_volume=140;citation_year=1995;citation_issue=3;citation_pmid=7672584" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. B. Korol;citation_author=Y. I. Ronin;citation_author=A. M. Itskovich;citation_author=J. H. Peng;citation_author=E. Nevo;citation_title=Enhanced Efficiency of Quantitative Trait Loci Mapping Analysis Based on Multivariate Complexes of Quantitative Traits;citation_pages=1789-1803;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290731" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. Korte;citation_author=B. J. Vilhjálmsson;citation_author=V. Segura;citation_author=A. Platt;citation_author=Q. Long;citation_title=A mixed-model approach for genome-wide association studies of correlated traits in structured populations.;citation_pages=1066-1071;citation_volume=44;citation_year=2012;citation_issue=9;citation_pmid=22902788;citation_doi=10.1038/ng.2376" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=H. Lan;citation_author=M. Chen;citation_author=J. B. Flowers;citation_author=B. S. Yandell;citation_author=D. S. Stapleton;citation_title=Combined expression trait correlations and expression quantitative trait locus mapping.;citation_pages=e6-e6;citation_volume=2;citation_year=2006;citation_issue=1;citation_pmid=16424919;citation_doi=10.1371/journal.pgen.0020006" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Lange;citation_author=J. C. Whittaker;citation_title=Mapping Quantitative Trait Loci Using Generalized Estimating Equations;citation_pages=1325-1337;citation_volume=159;citation_year=2001;citation_issue=3;citation_pmid=11729173" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=J. Liu;citation_author=Y. Pei;citation_author=C. J. Papasian;citation_author=H.-W. Deng;citation_title=Bivariate association analyses for the mixture of continuous and binary traits with the use of extended generalized estimating equations.;citation_pages=217-227;citation_volume=33;citation_year=2009;citation_issue=3;citation_pmid=18924135;citation_doi=10.1002/gepi.20372" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=O. Loudet;citation_author=S. Chaillou;citation_author=C. Camilleri;citation_author=D. Bouchez;citation_author=F. Daniel-Vedele;citation_title=Bay-0 x Shahdara recombinant inbred line population: a powerful tool for the genetic dissection of complex traits in Arabidopsis.;citation_pages=1173-1184;citation_volume=104;citation_year=2002;citation_issue=6-7;citation_pmid=12582628;citation_doi=10.1007/s00122-001-0825-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. S. Lund;citation_author=P. Sørensen;citation_author=B. Guldbrandtsen;citation_author=D. Sorensen;citation_title=Multitrait Fine Mapping of Quantitative Trait Loci Using Combined Linkage Disequilibria and Linkage Analysis;citation_pages=405-410;citation_volume=163;citation_year=2003;citation_issue=1;citation_pmid=12586725" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=M. Malosetti;citation_author=J. M. Ribaut;citation_author=M. Vargas;citation_author=J. Crossa;citation_author=F. A. Van Eeuwijk;citation_title=A multi-trait multi-environment QTL mixed model with an application to drought and nitrogen stress trials in maize (Zea mays L.).;citation_pages=241-257;citation_volume=161;citation_year=2008;citation_doi=10.1007/s10681-007-9594-0" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=Y. I. Ronin;citation_author=V. M. Kirzhner;citation_author=A. B. Korol;citation_title=Linkage between loci of quantitative traits and marker loci: multi-trait analysis with single marker.;citation_pages=776-786;citation_volume=90;citation_year=1995;citation_pmid=24172919" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=J. Shao;citation_author=C. J. Wu;citation_title=A general theory for jackknife variance estimation.;citation_pages=1176-1197;citation_volume=17;citation_year=1989;citation_doi=10.1214/aos/1176347263" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=L. D. C. E. Silva;citation_author=S. Wang;citation_author=Z.-B. Zeng;citation_title=Multiple trait multiple interval mapping of quantitative trait loci from inbred line crosses.;citation_pages=67-67;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=22852865" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=W. Valdar;citation_author=L. C. Solberg;citation_author=D. Gauguier;citation_author=S. Burnett;citation_author=P. Klenerman;citation_title=Genome-wide genetic association of complex traits in heterogeneous stock mice.;citation_pages=879-887;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16832355;citation_doi=10.1038/ng1840" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. Valdar;citation_author=C. C. Holmes;citation_author=R. Mott;citation_author=J. Flint;citation_title=Mapping in structured populations by resample model averaging.;citation_volume=182;citation_year=2009;citation_pmid=19474203;citation_doi=10.1534/genetics.109.100727" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=C. J. Verzilli;citation_author=N. Stallard;citation_author=J. C. Whittaker;citation_title=Bayesian modelling of multivariate quantitative traits using seemingly unrelated regressions.;citation_pages=313-325;citation_volume=28;citation_year=2005;citation_issue=4;citation_pmid=15789447;citation_doi=10.1002/gepi.20072" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. Wang;citation_author=N. D. Weaver;citation_author=M. Kesarwani;citation_author=X. Dong;citation_title=Induction of Protein Secretory Pathway Is Required for Systemic Acquired Resistance;citation_pages=1036-1040;citation_volume=308;citation_year=2005;citation_issue=5724;citation_pmid=15890886;citation_doi=10.1126/science.1108791" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Wang;citation_author=J. A. Dawson;citation_author=M. P. Keller;citation_author=B. S. Yandell;citation_author=N. A. Thornberry;citation_title=A model selection approach for expression quantitative trait loci (eQTL) mapping.;citation_volume=187;citation_year=2011;citation_pmid=21115971;citation_doi=10.1534/genetics.110.122796" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. A. L. West;citation_author=H. van Leeuwen;citation_author=A. Kozik;citation_author=D. J. Kliebenstein;citation_author=R. W. Doerge;citation_title=High-density haplotyping with microarray-based expression and single feature polymorphism markers in Arabidopsis.;citation_volume=16;citation_year=2006;citation_pmid=16702412;citation_doi=10.1101/gr.5011206" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. A. L. West;citation_author=K. Kim;citation_author=D. J. Kliebenstein;citation_author=H. van Leeuwen;citation_author=R. W. Michelmore;citation_title=Global eQTL mapping reveals the complex genetic architecture of transcript-level variation in Arabidopsis.;citation_volume=175;citation_year=2007;citation_pmid=17179097" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. C. Whittaker;citation_author=R. Thompson;citation_author=M. C. Denham;citation_title=Marker-assisted selection using ridge regression.;citation_pages=249-252;citation_volume=75;citation_year=2000;citation_issue=2;citation_pmid=10816982;citation_doi=10.1017/S0016672399004462" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. J. Wisser;citation_author=J. M. Kolkman;citation_author=M. E. Patzoldt;citation_author=J. B. Holland;citation_author=J. Yu;citation_title=Multivariate analysis of maize disease resistances suggests a pleiotropic genetic basis and implicates a GST gene.;citation_volume=108;citation_year=2011;citation_pmid=21490302;citation_doi=10.1073/pnas.1011739108" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=W.-R. Wu;citation_author=W.-M. Li;citation_author=D.-Z. Tang;citation_author=H.-R. Lu;citation_author=A. J. Worland;citation_title=Time-Related Mapping of Quantitative Trait Loci Underlying Tiller Number in Rice;citation_pages=297-303;citation_volume=151;citation_year=1999;citation_issue=1;citation_pmid=9872968" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Xu;citation_author=Z. Li;citation_author=S. Xu;citation_title=Joint mapping of quantitative trait loci for multiple binary characters.;citation_volume=169;citation_year=2005;citation_pmid=15489509;citation_doi=10.1534/genetics.103.019406" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Xu;citation_title=Estimating Polygenic Effects Using Markers of the Entire Genome;citation_pages=789-801;citation_volume=163;citation_year=2003;citation_issue=2;citation_pmid=12618414" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=N. Yi;citation_author=V. George;citation_author=D. B. Allison;citation_title=Stochastic Search Variable Selection for Identifying Multiple Quantitative Trait Loci;citation_pages=1129-1138;citation_volume=164;citation_year=2003;citation_issue=3;citation_pmid=12871920" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Z.-B. Zeng;citation_author=J. Liu;citation_author=L. F. Stam;citation_author=C.-H. Kao;citation_author=J. M. Mercer;citation_title=Genetic Architecture of a Morphological Shape Difference Between Two Drosophila Species;citation_pages=299-310;citation_volume=154;citation_year=2000;citation_issue=1;citation_pmid=10628989" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=H. Zhong;citation_author=J. Beaulaurier;citation_author=P. Y. Lum;citation_author=C. Molony;citation_author=X. Yang;citation_title=Liver and adipose expression associated SNPs are enriched for association to type 2 diabetes.;citation_pages=e1000932-e1000932;citation_volume=6;citation_year=2010;citation_issue=5;citation_pmid=20463879;citation_doi=10.1371/journal.pgen.1000932" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Novel Resampling Improves Statistical Power for Multiple-Trait QTL Mapping" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3.cover-source.jpg" />
<meta name="twitter:description" content="Multiple-trait analysis typically employs models that associate a quantitative trait locus (QTL) with all of the traits. As a result, statistical power for QTL detection may not be optimal if the QTL contributes to the phenotypic variation in only a small proportion of the traits. Excluding QTL effects that contribute little to the test statistic can improve statistical power. In this article, we show that an optimal power can be achieved when the number of QTL effects is best estimated, and that a stringent criterion for QTL effect selection may improve power when the number of QTL effects is small but can reduce power otherwise. We investigate strategies for excluding trivial QTL effects, and propose a method that improves statistical power when the number of QTL effects is relatively small, and fairly maintains the power when the number of QTL effects is large. The proposed method first uses resampling techniques to determine the number of nontrivial QTL effects, and then selects QTL effects by the backward elimination procedure for significance test. We also propose a method for testing QTL-trait associations that are desired for biological interpretation in applications. We validate our methods using simulations and Arabidopsis thaliana transcript data." />
<meta name="og-title" property="og:title" content="Novel Resampling Improves Statistical Power for Multiple-Trait QTL Mapping" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/3/813" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Multiple-trait analysis typically employs models that associate a quantitative trait locus (QTL) with all of the traits. As a result, statistical power for QTL detection may not be optimal if the QTL contributes to the phenotypic variation in only a small proportion of the traits. Excluding QTL effects that contribute little to the test statistic can improve statistical power. In this article, we show that an optimal power can be achieved when the number of QTL effects is best estimated, and that a stringent criterion for QTL effect selection may improve power when the number of QTL effects is small but can reduce power otherwise. We investigate strategies for excluding trivial QTL effects, and propose a method that improves statistical power when the number of QTL effects is relatively small, and fairly maintains the power when the number of QTL effects is large. The proposed method first uses resampling techniques to determine the number of nontrivial QTL effects, and then selects QTL effects by the backward elimination procedure for significance test. We also propose a method for testing QTL-trait associations that are desired for biological interpretation in applications. We validate our methods using simulations and Arabidopsis thaliana transcript data." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3.cover-source.jpg" />
    <title>Novel Resampling Improves Statistical Power for Multiple-Trait QTL Mapping | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"GtWJW95DcvzXoLxGy8rnJYTpzjqSpQ4S2SyjHbASdLI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"483521","apath":"\/ggg\/7\/3\/813.atom","pisa":"ggg;7\/3\/813","ac":{"\/ggg\/7\/3\/813.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/3\/813.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/3\/813"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/3\/813"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/3\/813.figures-only":true,"\/content\/7\/3\/813":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022hMq2zNDwR\\\\\\\/r6d0NpMpOBX\\\\\\\/5CBm+4nqujzi52mF+t5UdvQaNGn\\\\\\\/x+MCce9XEmwjWprQID1O32P+hR3WhUcNb5\\\\\\\/T6kzytuhulreorcck3UJjnNArAJAwDov0zROOn1PpZQ2IfDmCVRS+0RJgk1xjg8GuKN+edBpFiN9fh5Ay\\\\\\\/Wr94xnkzUza3QZEzjZwDZYT6sFZUO+ctMC80DDPd71KgUeCxFMcSiB4nYXZGjU+v79LZni4pHOtTR5PfiK81vMPBgvf8YQ994FnYELA2h9P8jJeaycF59tiMlx+JpC9fB0eF4J0WRgA1P08n09zHCi0pW4yjUpkQeSasX0efyvK9thHXU0eqoR\\\\\\\/+trJkHIeCF5v3YZjbaRDbYVYKrlvafqiXW8pM3u6ZrhaHFLurnYPkH3zbzxi2VRJcXO1N0oFXRCKVBrSXTK\\\\\\\/QsSTZw6xYNSRunfM5b28ksvpkLKLP5zkZwpCQIIbUArSB6ZSrhamjZ1Tc7yljtU1K4WPLlJBWK8+10PlbxNbXhl4pu6LweHDCXVymRMQUWGDFHoszVe7GSclakMBu04YduHHp5dm0dv7CpUvuZftu\\\\\\\/LYOMZzM\\\\\\\/1us9oMcBGmATvRNcmKIibByD8SingXOoTwKKqF069i345tYhvkN1ppRGyt7Wmf14CjFl8Sq1mFm4EtJe++lbrKWZdOsj6LeWB6I5F06fzBRfX2pAVWZncbwNp9YG8aufkdFN\\\\\\\/ZS\\\\\\\/JvXdFNaT0TBoAny51sHStFNU5mQbiip+qIGOwvqJAEyYfWF3JXG\\\\\\\/8QSnZ5kjGc9ziH0qV3zYPiUusFe9ODaqGPPCluyrG2CH1JCz3AconjMPjP9SF+eXzVzRgJKIQ0LWc\\\\\\\/ggeyn36QW5iy2bIob\\\\\\\/gdPxt\\\\\\\/aG\\\\\\\/fyAnVityqVqWpUebFx1TKfVIw\\\\\\\/FH1lF42ppISHQRyyCXMDEFnBSAZG9Q4Cs6wjx48f5ZmgprI5P+TpyPysVqG3IjUUHCSN3wAjRNa3y8\\\\\\\/5Nub2po6APcNhBY84I\\\\\\\/M2badswtCmQg6QogjYjKF\\\\\\\/UcAYmvgWsq+t46tc6BAQ\\\\\\\/GVxYNaySWRIcoM0wMuKdFa7hIhWz87OkaEABy0u+djn\\\\\\\/vh4KO2VDmM5uSNi+7mykUvT5WSp17VP1FEWRRmM1rI5o2REHKAq5lbV+Pu4\\\\\\\/bI0Vy6bR4TfyZ8sjtw6Kfc\\\\\\\/n8st2jJ5edbhZaLi4Z2GPtHKUVU7EgUvoPmd9ff99qVuCh5AXi84AoBifGI2RfuQ+R0HjM\\\\\\\/78lYB+meBANdBKlUkUtYc56TbfPm3k\\\u0022,\\\u0022iv\\\u0022:\\\u0022pjTJPvvBLyjBtGQfWulpxg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002259af5cf8d8ab60098ce08cee38b689b40c052a0c592bbbc64d7a4f737a9a3ed1\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022pUHGgpl4bIeFf5bJ\\\\\\\/437zyrhXzrhOnuUXiLYEa\\\\\\\/eDngO3aJW2bsO+zT56lvztPKy91OiafEZjdARKsQMBMIOZ+SBwiLlLBA3W6KKawhnBv0Bo2wJJMONV0kfbwSbikqcsqJS\\\\\\\/a3iK9MgPdflBkmph7u5oUd1J7fblAVJesLLadOYATAN\\\\\\\/+tkGYHxu3bjNfELelZnbMuF6Bu\\\\\\\/gQO35gVtL09FnKj35GtcyxDLgkR10Q8CZXNlbulsY0+LG5gjez\\\\\\\/FSpdcMhm1kJM65UPPS+MBN+OyHy7UmnfBbpVsO00FUk2LQKXAbJ6oDAfvFUGQ8ONArVS1DgnoQGkW9FfaaIcvpyq0E5Bock6+cVm6NQ3ie2H8VqGok+DaDOrvTd7GrpmyyNh921hXzA2Vnr0TUpX6Kupwhrenjy9FKB3QdkpgZ9RtqsQT285ep6a54XSASYHK4FpUM+GO8pBEmmhZymZSHcFyTjfc8rhAaSmaudEocTUvkUnMfxfAW4xBnwZ5NGcPiBHXPkoA2rdVJdpVmL6oElzAVg7TGUpodqF8A3+GxOP9AibaYnPwPPqMo7Kr0H5EyAa8gtpCVXkoS2\\\\\\\/kXLD1WfhV1SHQHaUOednpSXsDAV30Z9T2RD+LJOCWOcp4kofnDTiHqhYrUPwCKj\\\\\\\/8GJ6T\\\\\\\/GN8\\\\\\\/3qiunrOzDIcKJleXI34HlKnuYMLMBR5kfJjJoNqj7erqb4K49RL14aIz8aaBQ6IxJFB3ppbI4Ma4kdG9G9CWm4KX5eXzta0L10E7r3m1\\\\\\\/9syAdUnw5vkeW8vjytXN7w701rJ9CSbpmxQbGgHt1bGDLU7aGoJdUfq5WvQ3sTu0ktg3a5ziyoi5rtiyo1GQQ0qK0w0sJqV0G5NNbu2VQaXjB4EEXCqEtBL87Fj09ZMjLErmGUxBce1HkkbNsA915ha9Z\\\\\\\/e34cXpRyF5n78ofKJ7jMHeKp1PraUFmB8zwnIejcMGhsJ9nZudp1xxFwe+XQ+Ns6Pi2q3yuO0\\\\\\\/XpNTuieds04TdjrI\\\\\\\/t5khgRxtLKzs0OzkRxsJhJ2sAj15YEvK4gCxNxpF\\\\\\\/+jDTzo4T3y+koG0alkncbb7JoaH+Va88D1C7Ugf0jyG+vRQfSGjiNsq5E9EjZJNKMy4qyOeG9791Uf\\\\\\\/GTD+TvHpUUTCR41bgYqi3QJkQCyFHxmQfO4ahjmuu2VCANQhMhJFkx0NZX2Iq9djh8Ww1HZFkxHPwejc7A++QZJqaTtI7uWbkAtL48XTXygi2jNDPgg5BrhLDCRdLHYlWJROgnhI8Ote09OGSPGEuCqG6uOZqR0lo7oHnRpOLVbs9ftMFKXr9jUqVFSrKsHb1qhWzdKJ4c3hjhM2GbRE67VPwyXlN4TicXbN1RjUCLz09kPb1ZdrUppG5FSFBB6hGlkfxqxzcd4Q9kg8M4mXxkqZVyRYOIY8SuNS9mvB6P9om4TvLTq1Iikqer4pqe3Dny40KGhzQZ4reulKBEA3rQLV+xl\\\\\\\/VyoQqjXHjr6llYbKFbdy97tyRi0TEq5clzK4kA7A8knBJL2B8D1VqvbWAxtTsit2PF74XuQouyh9cAkhhC83xGGHnx36wsIOhjxeDVzpoCS7bZ97Peyass4EQIOS5Yi+QtI5KJa9SXMl17aK3+\\\\\\\/6e0V4vg8tcCq89tqJdTkp0O0WAr\\\\\\\/SPLP+tRIwZzOXD4GbAiEYZg2YpY3PhaVDNxV6B7DA6U6JDSuCR4RenFKL02UhgFxQWUWixrJw5c1ZkkEgamDDFOlNZlWXpFzoss9G67KIf3RC36P8rkHY3GN3JUfUcKHlwfmdTTLc0HoeCVK9Z1eS+48pxqLdNs6J8oz5ydKAUAx8ewvLFMyM5PVP57JCvJUiSKgfDDyMh0l3Js4hCDcbCvRTdpJDg5S9aad+07EAZ7pkIE1bvyYZjguMfncuxmAm8xFgCxW7FyxsnQ0wcwUdaZNDU7+gf36qj4+eBBp8+bCI36YJ+AaiPiNexKZZNbFq8UsIxcvw2eFEbcuTkBcYovpDuxYvNHq1ssts\\\\\\\/j2PW9mZ7X3dVe4\\\\\\\/gDcb07cX8inr9AMYcH51FpZYeQtsuwoXnKWU5q03ZshvNcW\\\\\\\/kfP7+y3Q8p5XJ1u5s\\\\\\\/ywW\\\\\\\/inZkSm3DHYNE34L5yQZmm82MZh29Tn+FTmwdELLafpuMr9WCTtBzgWWkfV\\\\\\\/MCuSTDAyCisbGKdQvYBWtlKol3C\\\\\\\/0EA6DSMTKzhTYwelUcIwztKur3hgvoAxgQP75JF\\\\\\\/adAqBxE1XHOx+LOeXItvYe7Dwax2vMiLD7uCUPlsc3NVef+zksnSz9E0CMN6i7YeMbNQWn1zZi+OG+6HefG9HSRoQebn8IgqMUi3CrVzt6ODJouVzDsjtPHVcavuKBOM5YKT77UcwkXouaRhiCDk1a97QYKlUGZ3SD9Xktf1Cf8nPbtx54C6\\\\\\\/WAcHis633s73xaBk7\\\\\\\/AfZ4daCbxyE5U9l06fRdGRO8gYR3yw\\\\\\\/oTewWRTAy60nfrAZ4BFvoOuUKGUqfF1sQSRIPBi\\\\\\\/MT+xvmjhvSKY1NAZAXoIpD3EdSD1q8X+BoLjwg8Kf1fWFQu0H2UqI2zsQFBa7N0eaYVGmhSwyew2qFyENblBVehT057CPhtq7pJ2guE0YD2pphdIt839E7\\\\\\\/HLxFZ6mG92VBTXq65dqaTfjPEu\\\\\\\/Q6ipnXbqx5MkmE7GmSiyqNt7Ty1TUhc98mJC7omgIsFhZ8g+EcS64xTz42YjKH5oFI+\\\\\\\/fcXHjxpxK+R\\\\\\\/i1bupWWy0Aa1nK+9FnWWJNMUSROB2A+pNFancl1Y6OHE\\\\\\\/52jF0NOW0x\\\\\\\/hLIpMCPvGRJ5zBjchiAtewuOy14Hp7aqdWrEa4uwnq8MmmF3PWu9uPfhX3r9+kDJDzzLClLpphqex9vM3HKf\\\\\\\/lDRi+Q7lBqfPx8KH2pw3Mczkhk+mBgMbG1vSBdVC2l6TOv5l6N39bMvnW8MvIgMi+dEsQ\\\\\\\/7fLpgSZn1wqinoMCg8V5AJ1r25yGkL4rIlKyOM1OnNk4Cttv5ClQ9MXLqNRKy9lOLvHwJfPXzychnFVgyHDN3jTS03nmJX\\\\\\\/sR\\\\\\\/64fVkNlCGPU2ew7NF7VjU4tCBffIyhkupTbRrfuITw8SBNIe2bL4auMp3HoOAP6E47F6rj0MQZcZ9xWpOmZDOoMKc64ED0+LYyqm4p0+mHTUCsE87iPkvIp9gUYBfO6sQl3LrobqjEW2iYsqedGRbpQ22scxHL8GiNv72H91v6TuP0gtxddWkJ7cvmBXw3+Go9e4UpIqRDQtLd5OislyD6CSvTzrxF5h0YpqGoeBEox+P8ixGazJV4oskF3zaQqVkIjwrtJ8u5UezOu5euPwrR7EZBh6ZoHK9qp4LH2e6KtYihK3O6+4wPWF7kayJ+7LiTmc7BlWMIN5owXR2lYNjQzyZ+utW70uBRwC7a8k1JbY1x0Y8MqRZ5ZLRUdPzYXSibxMjd+fpj8K2gFSAo0f5LsBYsyDlbOyDxIiYPDYuuqFAtyneARHSnK9kEKxKbqLgeeIEwoCQ8mAbqv4+Eo+t83q8t4gMDmiW4dzThK9MWwLP8CWAxixxPPrFQksUDPvxZ8AWJp3+6xUYcmV\\\\\\\/xVWI6+1lhneyVJ5LzI\\\\\\\/2mbc3tWjzm83gJJpQo63t1W7D6qwaOEHTJP+JTM1RrjdJLTzTXSnW3TIIp3fcnklxRwsqw4+nORANo9B+OB0M8hm+5alV+NpLodexKD5ILCk9oyLd3Wn2+fKOFJ1tSJPnS+ZA\\\\\\\/cB++8N3stNhGlPeNQ5TpjNy3s96D4JI0zsJyzGIigtdAJjAGQaALCTeRH9Luk4YTTWxCRrFtDAxEbpmlp92LJ8CR\\\\\\\/MXfjNeZEBqzLPbUqbvIy46y4IAM+ZlayH6eHEQTxsa0uQl0Ue6cL2McY9FV8yfaxOCl5c\\\\\\\/8\\\\\\\/aGooiAUuJYgjTwGJYmxrsQ0hNgwfLfCCtZ+E\\\\\\\/kih7OdPPXuYXtaSZuJOtHqWuiN9BEzbITfUfrNc8SulvitqkvSI1aHfAHmZhi9tnGY7wAExT9PNWjdstylhNyDZrmCZCeY5kdkqy2Te49GU022y\\\\\\\/oT8lh\\\\\\\/QlXPHGkgwuAQauYY8+Bra8yHIdvuIk6zW4Nb2l2TZteUI10N9tqiyrStGgz7F27d8\\\\\\\/0+L4TO0NNQa+mwk479fRz6vLR4fa9BHyi7p44oyt73KP3c7+13\\\\\\\/LsJpYpLZeyY5Jd04QT3MI41yiHjHNNdVXl\\\\\\\/RYaE0Msqe+O4XpFOF179VSwt6Hm6P3M0DgvOf6eH33ImNCYoBcIKfH9dyJRwe+dF0i1iFeZMTWhcLbKwXdcrRsWqxkr3tE27hw8I0f5z3Qr6VXjMDkHvOQ44fEo+fQMZ+o+PWe20oHlFIFUJutYLprVbnU9m5bj8msdILcsn3wootgthRiAp\\\\\\\/PooBXxhsJ\\\\\\\/szDKZLHzTUpz0A2aoZQ1w3hruVzVoilGWdmOoxSBXME0+dwCINMt\\\\\\\/Ky9bL9yriQyrnR156M1gM1W7n+WVIYCzxbfA7wii2ZNaf7c3007sOxNKayVi2NPIA9\\\\\\\/NQ\\\\\\\/Z6lHvAndWB9Vdme\\\\\\\/NR+Ea0gFZyPu\\\\\\\/J6W02LLgxv0J6pfhhiSeiDrWrSNCRHvLYH6A\\\\\\\/EBzTfzXesZad8bhYVe+AftC7waAa9fzLsMrH8pks6MzeoirvZeVVp2GeSXSvJ\\\\\\\/zF39+2kw+coUfpbwLgLOU2QyHLdMvaMDQ9dtH0OBafIErylmYOmQcIXxm3WmgCTKSQgxMciPm2tr4HIayMRqsbpjZ25VUrjjU7\\\\\\\/IzJk5FzJ0czjlnxFNchWS8ZpF2vK0H2zNSH5\\\\\\\/WBWZCtK9hCUwyxaOGK3ECcXnkpdDKYzAi+qD6JJ2UoNDdn357\\\\\\\/Pfqj0paT9DGwxgtC4hzPmwo5Rp8FjYgB70j\\\\\\\/7tNQgaCSmMLMHj2bGOYM2oNi+M0OQ9CKm8bHAbo1MwZclo8tqnieMXOer0+QY\\\\\\\/uzFjgQzhnApCmBsRfRkWC5EBUSC0FvZp9eMfHkon7WmNJjlwhdTOth\\\\\\\/dp2N8vqj7fSN8YJw8TKVfJcR9njVmUDkcaDUrjSsIOYp2MGhYIK753xT20adBg6NjLYac\\\\\\\/v33FqgXOZP31sAMyWEi1QJ+m\\\\\\\/zsjqkRChDMZnD+MtrOMf\\\\\\\/KcdRezuy1kvaylSA7MnkvxCqcq0CoKrhg4Ol9lk7Ub1jxhsRTSq+WkeoOnUZxGvXfGwzFQY1yIKeTMa2L2ss21KQ9YFeOK2GfOnGIApcv4rAhqUmw3BlTJifNtg3ILwR\\\\\\\/bUra2k8y3HT7NLf4mKrsEg0nsPhNPZE7m48tiTTw0iOkaBI+5toahmUB7AhGNN5pSZlB3ypMCfOyz+kdYeM5MzdcTRy+tkMGYRU2ByGClUTUk69Dx7CO\\\\\\\/nokkgiW1ZKUqFeozOSY3\\\\\\\/7tFGKexFbYPkdAmLfvvHOWrPs2bK7HRNThX4C1WIS4iReuABztCNmyqKyJaivbq5bGoovK6XbV\\\\\\\/TSlMBtG3MQzU\\\\\\\/hy4SAg+Z3gAZd+WRlwzCUA7bi6QL6TcgXvcp43QW8FhGRhOjH9pPga322QPx0+GMnOWGYnau7\\\\\\\/hjXzavthVZr7GY0dbKEEc6\\\\\\\/gGmfY1nT3ld4bmGWxzwsxTBUw4uYrX9vJoNYmlr8VImKO5KtCzvcFT7bQeDoxCP4y+3PQh4imdKPkSFeiYVf9rL6HHzloZYwhZKoQfqTphP4VTLri5JL67R7M4C4POfbaPQIY9PHRXBaSL0HeGcplKX3nWV29OGEFbGo3Vu05LWo\\\\\\\/FGSugVmEvMOSshkOojq05Fjow4AXy0FuZ4083B8+plPDFP6wuO7AAxxNmAVwPdCgMQkubIomTMdOJFPgw81RWSb\\\\\\\/f23scgwXwzRzO1s+G+tC7u8N7M6eh56wQqBrB4w8tFOg1x+H2J43yTaM2ybm+Cjh3qDa5zcDAn5aLwQANm1q\\\\\\\/PJGoQMfdnZvTuMIoWvXI+8zqMR+PipP7bhdkP3g10dMX1wOhIY9Q+jX7OzQinVxIPfqXENC70xHDYABoaUeTX+qQdwaQdunWkS876E8mx+MyuzE7\\\\\\\/yUOafXnNQ+rjRyYQiVBYYvZIUDk3tT\\\\\\\/lmCdi5285toPqOhWbxWvOrRww3FKMMaawicw93RG3R6FGaCX7S0RPX5O0oB1arIJmrbeEQFkQiJH4V2Pw15CxvqQWEbmqLkFSHY\\\\\\\/rOLfLnwv87Abl9rlH1edcyulaVYeYt2nQfWe2z1mhV\\\\\\\/u\\\\\\\/7k\\\\\\\/LOYUPROFdX\\\\\\\/yY5kzS7NZ1XtpSm+8eMwD0Pi216Gd9xPffwDkStjc2nMJkWwVgWwT9+x01Q3UtRRGCD2zL8iMKkGzTWY0suZb4Z5hOzy7liFq5rIJaur1btx9Ngxaw9lhj\\\\\\\/auQzrZGdrbuBNs\\\\\\\/7K1epRu1DWwMyFqeezoNsiOhHRFHa6qtR7LFzwqvzNcTZU1yKb\\\\\\\/9nvUKTapt0GunODc46n5uIooQoZFX570SxJL1Lk6DFovmdbnmBtAf+n8IE34dGXCcH5lf+59jCWisJrEqPmPO5c6P7RaTe6jjrbKg6tTaSd3xQ2iVYV+esOIZvNq\\\\\\\/zN9\\\\\\\/30l2mQIhqFcmPi7NVUKYV2bSY00Q6f\\\\\\\/JgWXubhEZegtyT4iv0RlvegN7eXG7txzTF8omI9E0thpDpFOW0gI1bUJaBtDcDrN+oZ8T5O4b8nvMcIH3uEiHSVLLGUwFr79bTMZSqlROXUuFq9B6DQsx45mthdAQ5hcLv4mKiQhxVTJXUEqjFZk3ZW86eRfYH5bQmsfA\\\\\\\/y6SWUBztFM0wOX3FYe6FzLy+suAQCHcmVAjLkTPGnqmGTWBR5iJp1tKC12MD\\\\\\\/AeHsEb9yVVluXJQw1fFdDNCU\\\\\\\/DGDzMT1XqnL2EmB53EEZUpqgBoAR8GC5WsbPcUnGOc7ShdvdrtC6gEQsxAGjwCofBcqHd25JjY0qkvAEVsrpcW9ebFUcx9SAlDLX1iiVTH4bEOQ7BXB6gHIWKgnUcOFOBSIqrLlgITEuLdBdQMlUwX37rZW+AdfIuapqBNx0ja0MgnymMi3eTp1lCZmZg5\\\\\\\/u9vnQsTMGhK6EBq8tPKhRzfgLeE7rhZbtJRwjLJeSqOxcVFJ5rYeiSKuNyNda8c+jZFSmcGwvZLgl1KiAQLuOUmxJ4oaWLq0wPRC79qgEebJYDlqxTOvUYYydrma\\\\\\\/hZy5HKDyCTvjx01iX5cvtcf1ce7VNoITA8UBNJNmvzfNOkCvN0KlVsd+X0klK58N4lDmNhw1vi\\\\\\\/Csu4gXydm0EI0g6t3sTnxx1X7qLcDrHyncNbKZJEP91cXfMeoKAq2rhNMlKqxuw1KGL8yp9jLe6MXcW8+3rgwZs+T5jl\\\\\\\/VWYQ+D4bK+hYJKdOBiubvbZClksqf3zEvq3g+SyQqU7ysqSIoRwvCJBUe7dZ+fyHqQA49jGLI6GK+5FcqPvrXczbtC565q1MP\\\\\\\/j1xEwrj4h0+T5wRmbUxm6KDeC3LXdcG+d8GzftdZqJmZlmVA8yMNK6ampqEZp1D4+OcZe9sw8PH5xPHE8ihRk9o2IOlPqipOg43MAgF7m7EXDIcNkQOzLIVvsU9y\\\\\\\/i8w6H1Cwz+9KLqJ2O1aNtyMrDRYc7qdteMtHMWoaJfKCcCjWEfJ7a6PA5bDIz+47mLY0hyk8nLI2yoWLox\\\\\\\/\\\\\\\/gZc7VOioRLsGDNEHCJCbeCMcXcvp7mhZTLCpZ\\\\\\\/cwNrYQ9lDcAKd+gH0WSqNWBALS+X\\\\\\\/gLtOJRO5hmf\\\\\\\/OPxV9t9CzAGPTZ+mJQAl0geqYqiKCIarO3tqWR9uZ5X0n27CdBqKVQDuLWIysbxJE3D+gVb0ewo8PzaJDqvRnPmFv2hWMHnY5p8uwDVnSWQuzpnVZ3bLk3UXZj\\\\\\\/LMFJMm+66hya0+LQnYnfJXN7bJOj3GfCB8fyJZn33BeDH3Is8sTjhVjenyQQanwGhiqP4GPI2hNG8NoxAjdwPI55FtfSurdWyeJvQfkObBhytEHi6Hj0I\\\\\\\/6AVvmeyYRmhLN6+taBTT3ovVxol1JcSp7PyVpoobg3Ha2Il8ODU9BX8mNh+fq5ZdDYinyj+FkEXtBJ6wKK5op73MDOA4qCjwf9Gt6ieCyjBOh\\\\\\\/+8fVHv3KVRS3E8F+zF1RgoaLS5HfKI1dSJBFY3Hb0qMrJisJv5WTBOxe5Mh\\\\\\\/ptSp8KUJtLTVkYR7chuvRxPrKuFNGTYM7gY9O+INV784P+pn6i5nKtJS8LFoQnUhtEOoCxtD101U\\\\\\\/u6sX5UZLWTBkzfJ78bpm9QRaxvYU8mt0Rx8k9DSa+bPRfAYy3tVL1WgbgVSCDfEEsbEXduV1dcfbO+l+AXVxzUpi8ZXGObGgHG6hcZ\\\\\\\/CKklPzlWw4u6+Nc3IpDFN70+h4MyFhFSO9poED43gR6yuwDHkgTvo44R+EmLdluuwLNzsn7EEGTz9ZAi2CdM4EUKzm0Y\\\\\\\/b6lGI+9H8QrdjjDfjOlVPI6+1EmWcU0XFYWzHm8tPl+NwYjzXeoQ2\\\\\\\/PE61tsbLgagmU\\\\\\\/X3MTygHpJYkA27HexJN7J3AxuWdsc6uROwjHgWNdfSja7WE8rWCsxLt6gKRmX0hhL\\\\\\\/s9kkgPgKC\\\\\\\/kX8oMpFCbBOuyYyaCXlDPs2UrAk4dAp4y5cAxuLuX3cEdpnJEJY5GW0kPFpzRxZdzw4rfPczfFuw3pTbqmAg7Jg+MvDU+LkiXlTI\\\\\\\/zOlw4yQTLdfh7zqf7l+0oPGPL0QDZCV0ImuxroD+ldksYMecQtgOsOhFszVmsnxppMl682YcO31oU3GajPsKbTJ7ZZV7soOFF6d5t4LnFAhJMOgHZ66JAET0+8N32r4LxUSaz2ziU6rYjzs1wywcwUNwzkLO0KOsiQKfnqSC24j2cSDI\\\\\\\/MZRCA\\\\\\\/EymytSvV5Tz38ZSIlf0dtwb\\\\\\\/HWuXvM1aMJClPnEqs7UaDBLv676PLb0XpB8DH4pt3wj9ol8mJlD2PsW\\\\\\\/1n8O0ef+Cv0dvxcqaCHNNg9NrcIHwETW3aWTDSxi+Wy+\\\\\\\/OKIpI+pbhF1jHtVr88qnAFk5jyCR96f5JOssydr5UcOvZJAvT1EoE4XXgxtl8Sj31YWyL9TEDvt2Yzcw5Qc+M7DVmNXd9BXDR1bQ83BFkUmIMB+qATQUaKHUvBXkean+yWI9FkeRjvX+XKpQ+O\\\\\\\/QNV8K5J8jGpYviRo7oRHVF9k6NFyAlA7SaHvCK8PXWHSQGL9WlBfE\\\\\\\/sp1Td0dVWCqGGlk9ZWqk\\\\\\\/Vr1uEY3Qzjl153Vdw88Ho8GJ\\\\\\\/Uml\\\\\\\/GACPvFPRbEUTxH17CSI36wp+LYI1+RCjipCNSqe8hS8KV186f4kJmYxS8grIFVuxtNT9Tuom26OoNy6H\\\\\\\/waD097fcTSrwcbIg0SnuKOxB3rJ+QdpE15uHVj0KLB53qJGnuEgEhk6QJYPJhehfVwZTU7+3sG1ZeTVmzrDw6PdsQNinPgsK0nzKejSdajzeIEZF76dqgm4r3o1ZKC8resDYiFXY9BTUVt0uJpB7zcQEc9JvFApjZWAtXCem1FS\\\\\\\/95j8nABXRY2UTlPylCUhXFt4aJwA7UhUuM6HomjpwOwsbSLy\\\\\\\/RnjUE0GvzSmIPE7foxactcxvVK0F9eF9NBKybqVlBUvBp91SZlkylwJecicKO7UKx74DNvIFyoMcXb5wA\\\\\\\/H3NYoTPQQ+4hUTYdQvfjWY671NmIjr3qLzxeO3aDjHGQ1gm3rArhmfIQIumzMCZ\\\\\\\/B7g64h\\\\\\\/AKu5R0T7y6pGb9mkIoRRvnFVPzXSPQ\\\\\\\/XvXstWQsNcdv2yfA2PASLFJf9bMgYj9xLCi400SQ6r8jn7awQSHC2Eqdm4stNNf6RBGSzk5\\\\\\\/2HyFMGlDnK+uXy3XezB5v6IFViDSZegaAgEDO1fldVFNrPXYdoVaAwQdLPWAP8ghi48W3zhLVfWFqR\\\\\\\/w\\\\\\\/n2K7jd7oOGCLfGMYyMDBxNvB\\\\\\\/XCQGCgzp3zi2dZGKyYpwqEhjhEsAbvqe\\\\\\\/4ebOugkCiAsZHpQNieMcuYy0vXzx8wLbnPrXMm2EnHnLa2RNNN5X+8o3rPbN9EQSiJopREPb7Gvp0N06DFApclMTS\\\\\\\/gHTE23WStO57mwFNDl8AfSW2GE0Nn\\\\\\\/ijmdX\\\\\\\/WPsCObGCC6kJiPmWx2IIOfJA093zrjbxbLQG+M2cxkX8iraVl1vLh\\\\\\\/TfAhGchWGIHpJTPtNn919pB3+lqrf2jkhOhuZuKwumhwVkAiBVJZe0H4G+ErXI8IrorjeiDVqT\\\\\\\/E6BIy2\\\\\\\/ugglLjTAaU7aV7\\\\\\\/Uo3Th88RDF2H11UCtjfF2uLwK68PAqOOxw+dsBziWgA+VkDBxOjq4npYbGDNyUGm3jpPIRQ2HN3KbTVWMhDwyCyqHnTiHTns+dXYiT\\\\\\\/KVElTcOoxQJg8YE5kjxAFz\\\\\\\/hnhMyC2hJuEn0h+XYjmPnG+9D+qqgqvQa+zCB9DymBtbdmXnGcqBGLyFYEi30Qr4oP4eY92QQ14XxvghrxUhDAcrq9enDMxvfjNr3iFpKiDI+bKmSMfSm7yf5WiHyjrHjr\\\\\\\/atDhEc3J3Qz9kARQlsGRJYQlr8uCGM1HcS5ZWVo+jBzjYwq7R2VlUmOCNN3BzEC8n4Cz7H7ZejHx3LRpPbyDfNYoY\\\\\\\/llcgA6Q\\\\\\\/081CfLthuH++upZJ6KHXlFrLnU0RfHQIkZr+PTM58SCZ5ea1\\\\\\\/6GbZXDfAQG\\\\\\\/dZc50L23H9xzOVB8+aZ8OBqcpHF62RDP8W4Vbkmfq+hW0pOz1Sn91bJXcsThp3nK\\\\\\\/AFqu8oQe1AII2RLewSEW+rQgoKrkCwiyThSzf0+T+HCjKtdDU\\\\\\\/K5SLke0DEKTR6R5p\\\\\\\/GNYHVFSEdooz8CcA9he4wQC6Jllr1h2O2Yt7DEHLTsLPc+tcmLdffgZ2yGNLLm8yqcVioGxhnk7uRijcUnyZjaj9YLwBThpvQXarWmTKvGOGBBcJOku7Ru6UWWD1H2Y8ubtEGUFDYXe4WUNCPKxasmKO591M0HtKBg8ubNt\\\\\\\/k\\\\\\\/\\\\\\\/MEYu9UsO\\\\\\\/ULoXSiACSxAVkfwNgOf0YSIunrCQeSo29El0k\\\\\\\/It5oNRf1ZaT29h0zPxe1DYyzHn04\\\\\\\/oZJn50mCqO5mocVhJAP2z9St8Al+x7E4M+RFxLUjdIICT8MCsNta56TF1YTMeWypFfbdPKOtYzxZOkDalNvWGksSbjY6TRO99IVGAjHM3K\\\\\\\/QhSoy5Nb4mjnx7BjEVhejKvR7mMM7ax1BF1FuVOv9AOCh7Oj+FxG\\\\\\\/bQhhKt+xeuMSBpewJTrUz9MySo+8WjhUkZOONUc88ZlLknql3A6KUfhumY0r1AZ5TjIqyobDGibECGYKQT5VaL8xfQoxkOtgRamBtS6qzeRv\\\\\\\/SAGgLik6UWJNOBLyj76sI3At80zSJ3aJ0vB7cB34cBfA46\\\\\\\/kQ7UW8ZzAdmotogbeI1llsXHUdxCqGqgKyh+nE9QFg5tEuohLwxaizYv4wSnJ4f0ovfT1xGIlFvkIJlSiPDAbC5JryceDRAgfhtI0D4Q+sSO0K\\\\\\\/F1JkdxLtajvEHJ0SFVY8bQ4nYIY78LlJWZyux\\\\\\\/os7TOPSOEYgHVMc2MAi8yWapy\\\\\\\/Iz+O1OcZZmmsd71Grz59iwA2NhlBXYy5MEZVdwpOYrFkIVvDfjoMG2V6jKGYvE0cFMM\\\\\\\/1yi1OP65xChx59yf6\\\\\\\/u+YxyDqdNFoxnKpPaKkLMm7VWnT3FSXMzRB0p1GAGdGnm3Aoti+vI+qKxEqSP3VAtKVSvLmARpYyb5ZXv8oChsWBBIUOKyDsrfOzvtzzRV934QfQbpaORnIjLogN0RbUT+8Uq1zjAJ3ExAxOWgefN8pNtyr1H11ERAvNhufO8fYQV\\\\\\\/hFc8hqEPFbfc0Jy0aRZ0YF+2P9UiBeR15wfC5z5BKd0Y6hKT4V4BNrLlLh60lEHL+AIFMUd3S4z3jWr3\\\\\\\/kAan4VhtJnDc3SxAtljJe4bCG8I12k0UZs4CqfRpK4\\\\\\\/ISmgEJqcJIqpT90WhZSA5ENXxKp7vucX+YNSEU+KPyOk9pwmrwRT\\\\\\\/NpGTGF6fh\\\\\\\/TWV0gjLWgOLMvD5DIzh5ncGYJiUVU67kH0PTut5D73oWQ\\\\\\\/SkrIXrnA8MPVxN4dZ8lXFT+05u2OACxPu0QFT748bocmxneNoda8V6lKno4mulv+HTuRIxrHXIztBWqlbUznxkO7htZF5Htskfv6lYod8Axjw6Vpb+3pBGoH8alLYsf9tmg4kscAzIGdgqtSnMmvSVgJ3z9JqsSIZmxAl\\\\\\\/suOiEb73gySAMh8GWOKVPFJs1j79l8qkOBC7Jibsgo3p0YNoftb5\\\\\\\/jpd2GdfH1MAB2RdRnIJxbhwmSujGfFD7kUQs01t0LvX9KhICwO6lbTwXJc36ty+E72oLMxHikUuDdzSfNYuYaae+ITcCrkQjdAglfPwD1RL+4UQfsuUGaCHCfA8XB7em5MGM174k6BZ7\\\\\\\/GXWVdwyBsC1foL5BcR0hJEjHEg1U9jKwWECtG3dwAfmPy7hSUSwrxFlkTHkEnd27vec2EMzetKAXs02tPfE0yy0LLUebapR3wGQJo4qLe3Ul8Grb8M9zereQhyTt1x\\\\\\\/9k4S9jXNuMq9t2og6LN8oTo+MYfCcCq2EgLYcjrfF4+PdZA+9eRUrJBLnTdyqN7Q8CifTeFSYQOdclA4dReFDxlmA4mp\\\\\\\/xjueeWPDtDA1DY03M4l3eRdP1ov3iym3L8b4oSuhnRfVmFjUYnBFB233bkDenMMP8nj2yPeey4QZkPiqNBTuCCrju\\\\\\\/4k1D+jc+E1xVmrZvqxEGja+eDRKdjtHJ+cpxSuiF046Q\\\\\\\/rtxjzfT2hbPTjWlji8gy7l4+9sNgzPD\\\\\\\/cdOHu3McoZdA\\\\\\\/hI3xHuETHeGb+5LtKTZ+cJo68f0bKAtYhnvYf0q1VrKEKGpFCwNvYj9r\\\\\\\/fVS\\\\\\\/2P4laA22ilhL\\\\\\\/+fYulZ15+HGZNEhjI954aw4QcRgI50dXkudXD\\\\\\\/iy4EjXd3UMgzg7ynfZN40LW5GhLBn64nvahhcNlut67GoWQRb5uMwqZ+h3LADkNhkKyGpaunU2RR+8Bdh+pq9P6m4tVBPoGD0kuQ5vT0Pxjg3z5OGibHM2nlfT01qiQDsNCO3GJf4nQauysXRYwd\\\\\\\/BVOh+N9e+1eMQ2yBukx0cUQsLw50D1J2bGUrwsbgIEpP2yQzgX2R88Op1UhGWhnQoXB462K6zYLBS9DDGw9Imd0j1WkCR4tW2aoxrN03UoiBAksuXkCE\\\\\\\/0wvzIPRSHsu9QVlyoxWJYMbhUgvuvdsb7WVJQWpuZMUcA2agAdmwrW4CWts4zNekuue8qsws\\\\\\\/SXX7wjEKSz0dpwI1Uhb+IVzNCNkvumGeLs\\\\\\\/AJ9TQ9p9GsZnv3tnteZRR2a+WrGTxW5qDsETt0vCgPOy2AA+TQlN0KKY7O0VH7S5R1rNH2rerm2RgXFIQflf8wU0Aiy6Zh5Oc2JTHVrlbAi40+hPjFVcwpsCi3lcxdPX+TBJ1nwGYqS3zJOrWW3mZ+jZexileepDRVViK5eWDoMGkdsyngGEV3J8zOrmxymcwvblFULV+lnSVso388wI+LWnnBXcer+jXXmSW1j5Ez+W4pOcBgJ4+4zeJlHb91pIvI5m9iWuMLSDFIPTUMXBGfnBbDBWxUWsNKb+jrWKq9lx4CkaOOq+lAAzL\\\\\\\/vE738XiiqFON4iQzEaxhUPROsw+SeaCPFEZ8CSHYYH8GJZF4sn+CE3p+dNQf5C6UhIoaNGpILESCESK1o4jtfHc98M47ydRE9pNmMbeVA2q99UOxBcW\\\\\\\/p\\\\\\\/Gk2hFxJpvYuaTI755sgAQlaKFTfCmCUyNp7ds9PYoqw0ZIdvt8GGi7ldJUicLVFh61AEuTcwvQQBG2KNDMqUzEj9ayXykCEd5ZUeqyifcb9OrZ754pA7BsGPlKkoGz9rOa++2Ft5DG6GoXlpCBc8xexCZxIuIHPcj0BspuNr\\\\\\\/rwNycBxHj\\\\\\\/efu4hx4ctMrXPPyl1LTuTD9F5sV8ESrvUWqnTgjeFp1BgF2THt78GLxsPnUFK06Yi3RFU8BhwyTXCFTlYthsMkSrpfSZBHGU4GisXeGil4qVSJ+7L2n93RVXaHZbuaF4v8aS+5xwSkBfUGGtU8EtYeM8\\\\\\\/nSqMhB\\\\\\\/yfH49618QcI5yKy1Kx41l2UhDNRwjDd+rAXYYyk8t0Pisd0NFIShjNTCCbFY7yu2Zydv7XQeGsieejk0K7PdmapvHAiu7qFE3yvuiUp6RYwa6VvZfDo8bPBRtoysp1Pw6vCHgmGvE6Iva8HbjHvb2QQyL0tEfwfo2plPJ0TLCnVXt6b4P6Ird1sAosWlHc7yTEQ1BCAKFJhNfKFAxY5R5jLQSuJgS17soXyDY4jesOSSDsxwU6kLgx3n7vZJvcxnFsL7IpU+WxBEtH3lWoNPSNsdw15uPAXGt5IYsjSIiopS9Zob6as86FRj+qHCKiFUo6uhQeiaCzgmws7PcEuPUfwyDPnjJpdIHQrstU9d0AMb0bQMcHenkZo50aSz87TY5HxWwHPXuNicM5dw3cFfN0rcE+wYFUtmWq7r2UNRne\\\\\\\/\\\\\\\/Z9hDcSfW4DEAyO6nXQiUt1+ixMx7cvIDrZhZcuvEP9fkParp8sjCR0K83g5iQtJysRr2M7XU5CGEyljd+a0h+Eb6CagvFX5lGeU53qVFAs0ubh6aPj2UTG4bbIQw+VWxQz0W\\\\\\\/hgjLZ4uuB6X09V\\\\\\\/fQiuYbvKUeWkPbqq2orPi6Cs2gN3khSp3Xk2XTaL6Sk2HT2Gz+rFoLcmPAC7kLcpvxoGG3+cntSfc8oyz9XuDta3iIltKNnI1pNjZzqGmAmm9rZA20lXUIEYpU7VwsL2wRWVSzzf0dajpEUCxr22LLr6PMeGRoagAPpr+ZGeALhm+mLMhgnJrPvFSET0F+8zeyoVQLLDnGT2H1aW\\\\\\\/o6JVak59wFMvh0kMqsCk+dCIuVeEdsf7a3U5Z8tGjtDuPUH+tu1QrQIfHqpNMB7HXeAF7Fs2cs\\\\\\\/LjRZ+kW+vvzhQD1vvnYtdxN0a5Of2T5JxwpgEolBRhqylfCl4gASqVruGuwNwtwxWrjcMiGMM4Z\\\\\\\/xRF7s20Vsd0plIf1t+JXQPVmQ3+w8jOoSMbA1a4MDAui7piczpfdULCgW3De5d3Gwahxpi7J1WhW3q9kroVktJvfNTb7tqgc9uhpC9wHgTaCfRIev0VM101n4CW6TIjH7y0wyaHbIpdBd\\\\\\\/RXWGm6qjyU4zb\\\\\\\/OZPBrZJEpTpfgiTwi1jqN8j3vwNz+8Q76WdXLLxFPam124p6YLVYyjizRVB65lxBQe\\\\\\\/WPlNlZThi3eOlw\\\\\\\/vECSbqwpmei\\\\\\\/QgVQcZUSAxlDalFmOPEkP8fwXMRQejBCx1Pr3EZmOsZRLzTGb+D7Aj\\\\\\\/ripaA79ogQQoj4p7Ko7VPaTajBD4ZzlI3rXYmlzn1YdPxwdzMcqhPRU7hTcKMFmBABh9Kz6DU1OhJeHkplKhI67nEalusqwxnko+IsJmuCvadc\\\\\\\/\\\\\\\/eot2ewk+NIyvLX6GA63mT9Gd4ALox78Y58fXwjxLu9YWQ6PSNrSDjgfyKOFfDPT8rV3khlts0rf5sDYaB95LjPAC0JZyQOambyQn\\\\\\\/LOJogVk9l8ZopqDlwS\\\\\\\/DOCNuDclQHfJMwWaBCq\\\\\\\/y3zdSa+1MFCVr+ZwzSHZkLRbQ+LIWKRYo1YVEL08bxUbb0lR5bNNVh7NaHImWkY3G\\\\\\\/fq5IXacaVXfXg3hPDZMKIJWp985RtdXzxV1SeolP81fcw2xK3Nk7PkMwnGtaxZxLaN1CXVHV7AEP8h7FFMb3P4FVQ\\\\\\\/mqNdNTFznLcH5aScK6WYDS9yfuxYWe71uf4A8Nv7VjFp8S97huIIVtc8tx57MwdSWrSBs+Mzcijbw1RhxafZ7fV8h4ayT\\\\\\\/4G8ta1O7g9pPE6nk+7vBsqSofuQGNSBM3RO9YWuPGHTG93CTnP2wEDb8rzdr6jJhO+4zyZV9rJFbPGMQfRm+\\\\\\\/WHe3pRb4Ng9phX7nmJreTiVii4naaJ3szlHZ35dOBbiau9DMjuqgeeXrR2PkR1IA1YqqsqYWr3oiUhiY5DNHNgRcV60NOjH5EXtbfcwhG4Oj2lmuLIgnvwLc1QieUHvoqNaqiUYyMmSopxeGCRN5VjpcIMVLC7OA7TuJCnGaz8IIfXGP6yFx7Y7IbcxcJb2YRo+n8+1sn9m\\\\\\\/F84kVTfbQC10hUp72u9KBrjpvsoazt5O0THCSHjEQNubH02UFXS5oElT0Vw+3xqnHhOjLJI+w7cJpeHCiAXtSKvctKfkQChVmDv\\\\\\\/516Wov1eLiE\\\\\\\/TW2MX86N9I1CTL1ivFQukjHcYEYIooClzJExGWRzQQpgR82Vt+LCZjwcD8JVCBM7GKDOVxcQy16Ss\\\\\\\/tm32CwNPLMdOixY21KfLS1ICbN0Gvu\\\\\\\/vRbBDGRk4WfGD109dt2\\\\\\\/qlwCsqeK8JjYVSTgkwvaXbvEd9M5uLzhauxYkH87PLVee3Ahc1mbT\\\\\\\/0nsA3+ua3Q9v4TrZY3TaSjoB4B7HfVD7+XSXuhis1s9Xpf7E1wpF6zRY9\\\\\\\/iP5NDvy56n5Xw8HzzXLgQ0hve9u+jW0vrP7aFhDxQYrg8\\\\\\\/xsarCeck2TJb8lfrfPRkypReQCSYP2jAVfwHJ8Rwbauzo0tZLqhZNnAZ8k0OxnLQFN2mLGhCvOJiRI8J9ALwyhptYvvy9hVH2PQ6GGBEkRLavVxu4r+mG\\\\\\\/8IvNMXaodgbWRpM7W+rmtIMkXWtMSQygO6w9GzxHGGJRpBADcvU6Z+ousmHXJhbWH8b90BN+V3lns9jfRROHrut6uxHUjd60SgNR97UCTy3XUPxN3RCaYw4lRWCC+HjO+09tNM0yD+gMFLj7rribfEaTnU5oHhMgzpCCYzT5C7iG\\\\\\\/EfLdPLXCCgHi65io8HXrmWecgq0zBFhI8aKAqSjPumo2A3fyQ7ZnXIgM1MSwD1BQyJ8pbzYkVXtJ5Tx\\\\\\\/O+DhUucB+P4\\\\\\\/XSoasFiJG2TKlmffpMJWADGuzCBbHiPM5VnOFYotS5OzmjDHOdjvij2YP3QsABV5lDaDDUccukBJCqZiOKCzGJAuZ+Sjf\\\\\\\/J\\\\\\\/Wztl8qWiL+UQ\\\\\\\/Qw0yIRtfppwjqku1J+avu6cTPh9lTwQE0vYaGt3eYXIsKdiaKjz74AK8vjM4zeo4\\\\\\\/xKxhym0Xlk3Q3+MCbNgdk3brg3X41vdpTMnA2JLJwDCZ+D88KlTrYd6agU8Adso+Ddsoz5Ms\\\\\\\/+ReJER8tAAhcGbYYPkshUOX8zfzDG\\\\\\\/dj\\\\\\\/pPBFzbbJ65iL82skn\\\\\\\/nI3I7Fbk1Sas4OuBXf2wmY5GWTr+QrDXpk4NlH1DLgDh9F4bZXYV+9FYaQ8Pc2OrLVRB3nbGUxYkv3sPJXTx9MNhYLUi2tly+Snw4DVWSlEpkfEVg99YvM1PP5hVa1SVqbm6gtH6DZ6PTg9yDiv6rvn2OZQqzbOCcos+zY1LS+dDhcGeCD3RVsun7wudEOFYPOgrKEvv\\\\\\\/X00C12BKT8OvydoaHyvkWxinxn9rCexf41Inb1PC\\\\\\\/x8YpUZgKqXZX8XfFiRt\\\\\\\/YTPgHKf5a9tnnK0CoZzTZvO6+glq+a63NKDwsOAbRnSJMiDU784kZnlvspNdfPiNHL6H18dCxmWnqN7MTWb1f6E28vqksYBosMnGRbuwo9AETCO2JRXowCeZrEoU+4\\\\\\\/JZ4S4C\\\\\\\/fPGn1\\\\\\\/V0R4Fc4gLz\\\\\\\/8ME6Au6f5tFvq6QqJhLpEWPSxP4RpXqc8CWjPS4q0CSY+U816t+d\\\\\\\/CN9m531FGwI6lR+kowyECNNQp6t+unAhK7nHBzCJEBwga4JNHbjUFua7BdFd5CspMJO8UGZB1XBqW9FJoYFPUZj8JdI68RP2n\\\\\\\/C\\\\\\\/SkiGCO+TEKdwRRsRE3qOMIsHdBAxgo4hU8d4XZI2yx65YKYXBzXYFs46ZbK380kVsG+F81\\\\\\\/Yqk3T3wcL4IT4ac3DCwBxmm7+m\\\\\\\/SwaIgYoymKsYLaM0WyHEe6lbMp1Sm9Qfwxfac2nCLClccuX0HSXcA8riUoiIMQ7aPW75lzCU\\\\\\\/5EUxLA\\\\\\\/4nJ\\\\\\\/CsWI8xun8nytDmvJyyABOYu7bcvqGwuw7Z\\\\\\\/56k52LLb5b+Rq1wOvNpOq3Qc0vAOhiAhMjK4wautfszmVh8K3lF2guYC0SDmU2O8RtQ4r6KrukJzqV4YG31HiAcL4UP6hCIt9ricdJjWD90eJrqfIU9hs8\\\\\\\/XLOchQ8e2raAHRd2noweUd+txdDRLL3RccsEo\\\\\\\/Tkm0IBdM2RkLgxdrvZjk0K9oErF2ShVzEY3GqFdzuCXJAS4rzIPtDyRZSYVuXlEtuToDDVgEJDCY5ARo9mPhOp33\\\\\\\/NmwHDb0\\\\\\\/Ky9PpLMgKxDyo9rAbmg1HZulpc4VrGTDbaVMDF5U9IjVixiLum9+glm1HN62YRO753r2epusn7LL1O7g5knWzy4DVUPzvda+KCvyGos7oIwQAvTo0N0K+YE0vrbKfXr5KSsZT56AHsPq47tsMq2gh1kEPc+L9tWsBOkQtuekUHwdUT6O7J3os1b7Q4MH1MkDBWzJFA2dMIavufNQNq9tt+7vqzKBqwV9+juX9eV+E4CT89+OOQFzX6D5SgLCUERp9sEVnbk2xMGiOH00LF3oEvMvQiueYads9oK3ItigcsI8xQxx96M9IuuFNc2xBBssh4J1KC9MGZNnproscLHoDYy30WO4UeaZvYJZqibOwz3uGmOOQyVbF02UilKjQroCdGN+ZvTm7\\\\\\\/\\\\\\\/8Z741HcKcYA1KODk+1Bk15m+7BWRNdW56ExPeo5gxpeGs\\\\\\\/s+oi+YVFqMzeJoWSJ0dReq1fQs1r4LLtnuS6p3hsUPtr1dD96w7Ydxv4vk1VeCPrl8EZgT24yr5a97k8IbGhEjVSQKOzcnxXpZ7TvvsO74QBhLEH3G75dA4CD+gIQM8EUSOB\\\\\\\/6DTbnEWAqWy25AEiAWz5\\\\\\\/hLidCmwS4DbqkLMJ6R\\\\\\\/QGYswDH3nyKw1Q0LIU4DbdGgk7j\\\\\\\/TATP5WI1rzjRvZwk1bLZEv5ZWPQFbh8NUnF9zbh+JyKLBmBEuxZjndpbdWszmC87V3YYWoOhg8sjjqMkf6SLfNd+keJIJqMPF2a29pV5QUY9AUkwJUbGbjtpEZKlSwjjUlm9GPdDQvYE0oyYQmvjABXhvs9IZAy6W\\\\\\\/JmLkJhQ3EQGB87wZNPxXqkH3cczV2kK18GFGLukidtiWCvlGsl1Lr\\\\\\\/HmMfEFCZ3XCzGR3n8YEKram2DafYODIWunpBDuBWN\\\\\\\/U0Sf25EfTE6A1AGkwNo4FDWM0k56kh3fXJmnJKAC\\\\\\\/waYTcN49mCo+qjCwjY43tY8D\\\\\\\/nXLc9V76li13vfoOuyO2tlaVMKvTyt3mqwjcPUTK+6876MHYcyIXUpnm+R9ewaPdUQi2z6G5nF7Jsoda3qw6KDJJQ4xH6LyGy+xwdhcIp8MZhJ9WU616f5RY8Y8iqq0oh39T4gRAtVMwksQGDbH6DmEJ1i1xXLjHGgBcr3jm4lLGfpayxFrdhijCUAvhf5UvTbsd1LERPjZsN0F1FgAS2IpLbJIe5ZCoZkNLhDIGv1vIXJDKMKE63V90mfx0wr6VxsGN\\\\\\\/cVdUfqQRGzJgw8MBpV3JUVlJrliyLt7sUaqBTe7V67\\\\\\\/9\\\\\\\/a+1sGopT9gqHijCmiPfyovpEAAybG+mr\\\\\\\/AvN+znV3xCb81WRLBYtR6NroIyiJNP0l7e3zFgVL31fmWyI073fv\\\\\\\/8m69c1smXu3lOFQ0VsxdNZMk+iScPukHI7EaMjR8rowjZW7\\\\\\\/z+XeU4zT7aRvHVCqFg+Rpd8XfNCZYAuuVVLvAiPzdoZefUeRdIuZR3nanaG7umUeRJP1BByt0OkQUJjIKjS\\\\\\\/NaQuRkzLAWWRwPlffRlzRLN6WBoojPtJrpRy1IGleqi1WcCa6Gk2pK6HXKAKOoVZ31VTj+YLEQwdiePztERTXcWrvpjuInKBowizpS8PsBHPM7R2ehZCwZ0rUd7LCoitcg5HPIXnB95P3tJGKUN\\\\\\\/qc2n8VmaB92vuBke4YC0UTrnIudcoh6+TNTPafgDUzEhwih8tg+txvkl1ZfX+CVGQDvUDlLnDWgYAD1635eHV1ydHf+zfKNtMdgMd2AHb+CMTmpZGamFmfs0fGp5qpNCKUN\\\\\\\/270NYiycjpRJZmGuI9A+2JKh\\\\\\\/jLw23WfBnTsxw4gKK9dlKiiVymfEJUzuDZWjgGQ2BK0ShjTHaHn3CUaIwJxJR5zNwYBD+yx468onv7tAYnIg6\\\\\\\/XkWOv7BA7hFXVRFiPI3\\\\\\\/9mBncY1u6kJXbbnO9Ztsx1I2OWHpEsnsaEtZ9GzNZc0HLTmPIaKSLUZzursJTXF4CNZSSX5b+HYm4N9R8c+JOafyQqesguTzACNF9VqW1QzdsBIG8RJ1f7RYbuMtY+XsMqtSuLykKr8CReDN48dXIHzf9QnV5ffGUDftjp8et1BSKrkzLz2dKeUaL5hzpbibn9rx6y69aqRE+jdILOJh+Y3MpKNwBDgGZYYXWy2\\\\\\\/vqXbDY68s6uNktUhF1R+o0UjYy3XQYMJfT6YU8GRyD5RntGZ0MdrLpczTcyz+KsXUBrx5c2LyJtpmvxTJcMetxZqMJuILcE4V8N6irYBng6bVSI4qpwt2q5nNzWLlLQiK\\\\\\\/YH39\\\\\\\/IcsHeN4UONlvT+dMJA7Ds\\\\\\\/7cUwvzaV13ZAcO1ngwxoDHFJp86nlREFu5ugeyrjCBTwsp4cl86YXA4Uzj+nMlcrqSSVDx4WlKnNidxTWwVyZR66EeJeUGtMRVu2+oUigfhf3GrW0qs0VN8TW78LIJwpcMGqx6C56ep5QAFNpVWMRLseoe\\\\\\\/9424Mokws5Hpx9jpXNloWMcmvwBBI53PpK\\\\\\\/bnMeehS2hOilYZBUTzH3hrUFfvfK5b8ITb0ZMhJhWD4COiqgoA0b6oTojGOkem2lITTE3xyFdaDIyWnrS\\\\\\\/NHeNteuHpilY8wwBrQvupNzlXhkE2yawy1RsTeIa0b5CoUqk580p7TxfSUK2vHKXpICrpH3CFRC8eFVCDL0Skdj4XrmOViQ52GeRnkMpQWu4iYEeRH6oDTW8ynfMN1ljksRLo06YwbdsNt3UHGfVuasdb6f4SI8B8w+0GcaWDtZS3cYQnYnoILliNDq2aZveWVZdi0cAphF1tvZSm9oENcmmNbyXPHFTiTazUfKYWKTylZRRalEq4StNa1O7Pz5DYzRZa2lOxbLKz38\\\\\\\/8JO4ZYK++TdrJZHGRcOVkaCiHvWVutZ46my0n+j8hrW29b3NW1a8fevaJbnKioKM4UTjxxW0P4ewYkg+cBhPcJEyNn1Iq0VOVRHAv3Xl0l0A8wd4GcVihmlUAMLI9E399yYa79BMjbrNwl7C1mawVQI50xgaK\\\\\\\/9SLQexqZGOomUhypHS0k8KXevWOhkRLNsUnrqVlnp8YmzftqyRCJUNucMqeHCBsriYfbKUyIXVaku8EKDP0pVF4xfMYx0B6ylnn0aPQrWrfYdc5Tmd+bZk\\\\\\\/9RolOYZkvFPopM5H0BTQMinEvC9vOsmcvzqJGTnTSGqgHPJrlrqKFzUq10p7D9qhrfHYfWK46cUuO2DyimhSWT4hMyZKJeW92nnCzDHqhgzjPpuvAFduxdwVu0zrzfj9vuCjz55wWBsUz9Md4ISDNxPQiLq\\\\\\\/yYhhOR+WlQi9CRfJp+RmxCnsrCVxg2jY82SyxEnLAxyMJUoLBDVAgD\\\\\\\/I9X029Zo04gro4kawGZ9TQnnQPKRx4eVBOZHVWlTOI54x7aAOauesWk3tYV1mR2bewVZowGnMv1NtTQ7RYZtdjSa9q3HXISHqRU\\\\\\\/jVSn15k3zN32RsVtN3x9ggAIoQs5KrW+wDCZGYL9sv60AoLchuJ1Jl2buz9uvVcaaGaNv3jBO52\\\\\\\/wAlGQzhPDSvGreofWCklVV\\\\\\\/CbXtCR43vvWH6TPdyNYl16osAy\\\\\\\/N63xMJ3oGMxHyPipP\\\\\\\/G8Cj+pkQO8s\\\\\\\/q2Isyy2N\\\\\\\/23DEPsaQR3hRX1B1odh4j4S1wC2xGzMnkSLAgS+iW\\\\\\\/3hNP7mVH6Y6ynnbxixL1Vkl9L5+clYsHe6QksP3QQ9iCBXlkSYn1d9rGH9Ucy9+Q1GYH33Zn+\\\\\\\/nuVq9s4vLu7G3vqMGicCA2KwazJ2fzV+cS\\\\\\\/Im\\\\\\\/GI5ipstg+k30zmIFjxd2Vaaw0A7dpFkH9p89Moe0G0591sXqnQTPSJZ+gb3Yt21oO8uml2E+ZM8nZV9+Vp5H1gIMVxtWGYzu61G7DWgp++HCJCiSNVfQ17SiKLl2Amrkal7HD+h6+LnDMqEeLY+G4VS3iGSwSMflY9cFAflUenjFJen1t4adwIaztohXtSsI22Kz9Cyl2H8HrvzC4JJzTXyoOJ7X\\\\\\\/npGzjFO64OJCdAt+ACT2E7RVvq6LwfSIjP3MiFLOOzFYIzkG09Wkutu\\\\\\\/oMYuEFjRlqeJ9Q0KxyrTS3EGUumBZHCIkdIVnHtPzDOy9j9VhAR\\\\\\\/me1Z7g+PoWr3x5OtpPYbZOkU0dcECOn4QrMla6hmYMtU5LMR6Iskb85w60GmzhpLPcWMh0cs8WEsoIgYYeiXBaVo0Pw7T9k+zMiwiBsY0afUH00Vbq+io7ZBlu1JGLtOXVATZQ4tY8tU\\\\\\\/fYWN+XhPCEowRV44bL6wjYaVckRQpe6SzJgcpXgenVwmKTaA04m6ScveYn3a1DTpyhsyHa1Ry7WvF03GF97fC8JZTEbk+QI0Iaac3FfxDw3I8e1\\\\\\\/z9NHe1FPPMj6Y4HY0yNWGiZpl5XjREkmikFQJcXX9oG4Waa6l61ymEkffpW6iVZjmFi7jIbzsRn6RYn9YhTBkf1GoeJ2L8opo2xLpvIYB\\\\\\\/zwf0if5w4SRG5KZFdjRJXxPiz8rSvbtVCj9H2IRa0JZjDWgCWYPR\\\\\\\/+58hsEdmoSQ9XgSkpCkHLlzOwjXLKMs26cSayCP6S\\\\\\\/BFa5\\\\\\\/+flcu\\\\\\\/5tJueO32akhPwWZ2w\\\\\\\/K6cA+ZQ8j4lUsYXQlmIZckfOhfAyZzyBDEwkI2VfrHwmjK815RQXGX840QWv0M9fkr24mYLsTHHehVs+XGGqbixhdrvNp63Xt41FFQ9jSzpDNShJGO8tI8\\\\\\\/zPekXxHEMCBFM5a08zkAMjhLgB10B7VjqfK4gwOevAVbau2GpS5mdtyxKm0PV2CfFuwMPuXRZcVvdmPSPhOQi0+mPGnVfmP3ENnkQpD2I\\\\\\\/viXssKfJlrE51AKlf4LvbjLkkrzvU3lEHoLN3yqb5Lwf4XnJsb39Pvsag1+RDUTgHd1vTU5PGVQxxG4NuZQ6TvmijM5kZpfJqk9iBb7DaU3SgrtzjK58hpWlmgibXC3sHxsIDGSMRTF\\\\\\\/UklVSMY\\\\\\\/9ZWVIuXSRyhe6ajmRhLP7TbOqt7xA319xn5ADjMzREuuu\\\\\\\/qEYMT4rFxmZZG2F0M6Rlpp6+ebeZfCfahsgYKS2TP9yrenZTuGMmM8LDb2PGDFX84zaGsNrcTl58FZCmI0az1084xl9jjo7386uuY46w\\\\\\\/bGsxOacVIbLYGEXIiTbUFqLBbjmY05jlmtpfnT6PSWCbH5ldjv7UGUP\\\\\\\/HLUEo+OaFbQLaHqxEw4VAZe3H1KpbRuPkUpUTHaGIVlUcOmIMA+aB1oWVVKregvxhRzoWk5k61W5ubWN7QZwdzNNfWrZdXyjaDFZL7Sbj3fqAUK7ZeDLLWTCTTsGst9km1vZoxt6Hs1utTl07JTSIw2BK9iOlOpo27c2bWE4MII42l0rHakRXn8k+37wcAM87+0uzhuPM18pDWl4e+NSy70Xm5yAx98rdUJtGK2gCf5jmdfpbHAOcQE3RvfzO9cAewT4gc9y0IY3oKEGsKmNGlH7XTgeL4U+c4brsgmqZg0esX3X0XGfiq8vnb5hp7OjCamQnZt16rOqn7pooPaA4v5LcITTNeVHwP\\\\\\\/S0VuUNav6qfz\\\\\\\/F56ZIBq+Fg6163E8zCMlYZk4ViRyQtUVexZU6J0cl0813R1SNiuK2IVsJhgpR0ah3W\\\\\\\/79y0aw5QNwVhaM8RhwRgMNdY6IDF7S3QptQDg9E1p\\\\\\\/VzKJX8PgC+lPKLB4V\\\\\\\/bDekepQxewMVZWrcHSDCwtDaLCj0IKLBW6LSPWCIfAvl+GxZJE2x\\\\\\\/mctYk\\\\\\\/51zuIBUEg2K1NSMK1aXeNpyb2nAdPZxoaDD0jQHE3aPZynLYll5lN+76IaRaSNH7qgvgFfI8zXFnFymyIcPvYR3gBY2B69tMFjasuiH9hOaWrd4CgC9v6HgbHFDhWUy5\\\\\\\/vXgvyIjbWSHuxbOiWXqYy6OVtmscW78l9f7yqrsMouwS8ZgDtISlqmdLn5qwNLcWSDWbswkYjLS2VV+19BdoNv6sBPTjCZscOFi94kfLzPSkt67BA5acf60sdVRLCVMWLkvNkfLAY8dMI5zexJAb\\\\\\\/0kVPjw1te434i+6uVGISS\\\\\\\/acHt2\\\\\\\/O8Epy17Bp9fKLmrDlUCU\\\\\\\/EfWtf75n6BuXY5n8o1eAq95ZBeo+llugb2k520MRg+QYY0rVR4KsNCmh\\\\\\\/ahxiQRoJLA18Mp6DTkeHVHXwAoOiqZJkAS1k3UZ4q9pq6y0ou1B5GBd2G6pz88kX9HilKbAVA68hbUW8sbE6fHtYyawJTgsJE1e6f0Vj0ls81PACz+mtI5z0guSvqUCXE\\\\\\\/3y81v+oDG698IZ6FAcfBZQuooteqMfkqfB\\\\\\\/i1rxY52a94mxyLOBdvCBd+BrIZz8M5QAyAHT\\\\\\\/2zMiByqtFZGw47yPB6nLokzlm7U7QDgRwxj+iWSBOHtIGK5nsJKISm3TX9UuQrSxUzJyRAOns0tyARvCpudEQLRGIBtQDy864PB0uljqMZabpSmdNQ7THQSEatjgc9ahvb6ziK\\\\\\\/h5wEd\\\\\\\/0hB8EOFvLfZYkJWt4akUbjh+cORhhsICGDYh\\\\\\\/fEGekm+msyGHWdhKZB7Ns+kNKQ7Zq\\\\\\\/p3G+3Mbi2ga1Xk1per3T62hpb86METAewlR+4\\\\\\\/88YUfI5zzGkDGEoCsT\\\\\\\/KE+clY8eaceEzQBV+vzhyoOeAAE4qLXDMq\\\\\\\/4e86M113QIKHCYYRm02DOYof4GFtRgmL+\\\\\\\/dzEzT0MAgoaEZPUOfou0gYRE3DraxEV5rNwxVmbDMOB8PFdNgt46nh1BtajBepgrmoeCBmgd3M1Da4Izkl3ybNzTG33XRNsf\\\\\\\/hGqRN16sd7M\\\\\\\/7IjggbHcAH2ZoidW9jC5aQ79kS8sr3lZJakQ9c2NIUFD\\\\\\\/ujs5ekUuKlBamjOKPWKg1AopVJt\\\\\\\/PHgst12H4j0KqGod8oMIimD9kcP0w0dkSL6FmAU\\\\\\\/ixmwxYEAHmAQGGKFCYgVuBGb8k8rV78ddeVcIw6Po4mx4wwRLBHUIr163wRW19rGsA8zFX1OrvYNYoi967+clQ1ypf2AJKT5wFadBL9yh8ypRjx+1PTnj411f9IzeaYo9a4Du2gRJFKf+hsBBABGkOvpQ4ET5rmBJmZYWk6f\\\\\\\/kCS8FcaqaFfOaMw9yDwU4PtsTySMoIK74xRJwCm\\\\\\\/zMx8uytFX9uC60NrNvdWYwszYc5maFM4gjMyCKKD+EpUVCbua6gwSWxqc20f4gFKxTMlNr\\\\\\\/teLh6PkBoaC2TLDxo53DT\\\\\\\/j9LinKs3\\\\\\\/pgQ+BWhtzPXJHl\\\\\\\/PzR5R7sgLlhP\\\\\\\/GbHaLgNaD3i1TLNCuFLUXsiyuYg5cgmaZXiJjpFn5Ys5Jd7Xw2aGax\\\\\\\/OveUl7fyxQZOjbXVmBLsOG\\\\\\\/vDeamHhS5PGKpnWlBDwwm5K9aWcWSNTFVZG1VkeYdokcqAoOET3\\\\\\\/x6c+kA1dS0Fq0Cd1eKvHHH4uvizG1XR9TQRmb\\\\\\\/\\\\\\\/6MQ22IKm6cFWNs8TGsrHDL9xMr9Wemql1HDAZwgvShJMBEX+q9WPdlxtW0dbzdCzASk8\\\\\\\/dxHttRguhN\\\\\\\/kL0JN099xPbgSyB6nN6bGGqfDE8aVCk+ViCNGsA2UbMFbxnQXZe\\\\\\\/2WiWa8AExrROsxb70FaKY\\\\\\\/IDkTrdjmbRaxQ7JQv2FeaIshG3wxkmQgAkhRXbor8bAAh+tXMl79p7kttkd9fWULuIGRZywqj1nCbN\\\\\\\/FDPwzlQdT5N5D\\\\\\\/oKTcODC7TQxezQq4Opa5XrVeXF7v6wZHsafBD0vKXdC3UkJc9LtjeNWBlYTc9oj6jZvp3LCcNHY\\\\\\\/yeOkNOjZUdEcCnwtQE84+VG7FUZSWizIMnKxi1zJGOaS3kWThncklB8TYFDAxOf0HuJb7FxkErsq5y5JEDJtcrnNSBMT+jkDLYhZxo5ORFiN+s96cmLk6Z3W8QUGiRBLKLKdMn07YkNQFp3x+VKHSSu2owOnkKKQN\\\\\\\/IilWnp8yHovtgxKypk3iHlq2Ygz3Ra1T6kAKLed3w1odbsd0PyNYe4xFAxT0o16\\\\\\\/A+bSDb+j6tihaKlteVj3RswRwMQmvxJ2DksU1qzqR4NSRJM7LqbmGUC3QDb+gt+xyjKi7XfpZWaQ7pnlRnf82GYX\\\\\\\/Xn7DTmxwG3YveioJ9tLtWMTzpLC74YqbahaOTZfIgsyUiKstj8VY\\\\\\\/wmGM8Aactg119CUyVhNPFSdQ30xKEHJONru0gXuGIkupL4uKL\\\\\\\/Kgm4TgG4fAV\\\\\\\/eZC2YqsMgOGU3uh1vIcEUvjp1v\\\\\\\/gYTMfYOkPy4+crZYEwQyFh6maqOSQFYYrKlpdexnInL8tIwXJLSI2X7ORwxBU9tYp24Gn8QRFJs9Bb6u9211dQXebhrtt4B1e\\\\\\\/7hBTl+8BUlRIvH2IOxTOsKwuexyanJB1Bh9nByvlV9pEiVw8dIocYlsj3OP1LpdTizIKOW2gybpvV+nNWnjbmL2qCruACipnQyXHBuNP4QtFKE7U6vCCuy20Q\\\\\\\/PV5saEd5fDZaM1n6zO10mp0txjbLVasCdlV\\\\\\\/VKdoRrYGIJ4J0Yqi1aKyGcoX3nbseh+IDOMAFwxybTplf+ecjn32ZxQ1r+oquKmZwwdAcQ+fyDfEcojXUGQBGPpWbyRwFIQNzaBJcp6iLzOB40oWkkxAFSekWnyDk1y1gVyGgbGY8BUbRbenq3b7SJqHGbBrpkgtzxTkkmX4aGJXs+VqosGjB5ncFvE\\\\\\\/BR9HjxTM4Xo4E3yYYEO68DoT\\\\\\\/Zmd4o4PPm3YpfLwWS4jf1StWa4oQpWr8sfkVjlZipN7dChDkwL3D\\\\\\\/YAeCg7lZSINdPxl8dl33YfCS4YU5WnYSfSmMkmQo8n\\\\\\\/RbPH5SZSn+fD0rg8lo7Fiqo0+f2WQyZ6odGJPWde80Z4Q+L7vGdqA+qPYw4WbVaHkcdaah3CibOMB\\\\\\\/97\\\\\\\/1gdIOk3lNIzoOgXOlAOetlmVw16IIpNPeKwsVTGDhORECDPu6x5+6\\\\\\\/N3vPINrZzXu\\\\\\\/TEOHx9QA2biJJ\\\\\\\/jx2CYVrJP\\\\\\\/QatW9Sc9iR6\\\\\\\/8er9V5hiHZFOQYkyL5EnEXcG60V3iAooAphBkkXOhTAJyInHPouu+DZVKwMtiL2WId8aCullKctPaDO4nxUigBA6cnvDXWFfmfsGIz7F1nrZLJmDb5jH3KAjhjlz5qLoEjCaGhAe2WptljsTNmdkV+lZwVnb7UBNWGiO49eq5KB3UyNiZmJ08I5Tc0LqYbjFby2gMGrTeqaVaEPZYUj47Urd3fSVyrAZK+KaMtbwKAYfNLP6LjoGlabu0QBQ+ULtA4vhd\\\\\\\/nUWOd+rJWrcnYYwi8sCFvKiakbU47IITYJNxzUjmqWc0LwQJaCKPnI\\\\\\\/yWOXtoy+pBTXY3eBv8Vv++Q3m1LPlqTzXjdMFcrw6ETDjNyucVLoJPUmFEAlSGt68N2+hro1WoYlgZwbbDD5d2HrnJcuwFUH9pMb0XiQlDpC5KlBPzT2ZyoV6RlRPOkklPnC5QPHpugIDbhKJGBiKMWOTS7Wn3hos+wyP9yAemQmY62RjiWlINeL7Cxyphg+I373UrZaGsK\\\\\\\/ZTz0fm8Z1+dYqq3NMPWQmT3Vex4kAQ3ouoYgCbVVZqrQ3lDVTZyICo7GeRpraPtpaY3BsOJ4zskwhYoltGaLMOIYlfx2EfvIxWv2gsEaXbx6TdhSE6Ya8GcZR3LwacI\\\\\\\/le2M9MPn5uWeh3Jl4KLA0iWiE0EcdVM9\\\\\\\/fEM6HoououPJD+VihqDiW2P7iBwpqAO1GjbDcITSoWsYM5JLYjBb35ij3nFdh0T11qqAO\\\\\\\/6uxvy92hfig2SwQtbi9dWpV6NBCowdYyGg3VL\\\\\\\/qg17VxyNXme0vhWh2jbiOMYfd\\\\\\\/L2jp\\\\\\\/vhynN\\\\\\\/KJlKX7i5bJnOUO4mOG1JV66oPB5emREbc8Ow3nj00DDY\\\\\\\/p5zNVDSaCDzvOpcEiN0SWhJbSIYCKJ\\\\\\\/2+osjtZ\\\\\\\/6jArHzxNKJztWudu89nfDlexoTlzdyD7ZBiUx6OsgN44aCYzdt7kzfie\\\\\\\/++WJ+yWZZ+NCU4bJ3\\\u0022,\\\u0022iv\\\u0022:\\\u00226\\\\\\\/YJ2Vg+i4OfGV6ZOFqYaA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022085eec5b29a30dd397e0127fdc7925ecf7e1967bd29b5e19ce41cf43a8ef2cd7\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Pg1t4T5gqa0NhDFLiSvBwV\\\\\\\/g+8hPF+xbiubjlB\\\\\\\/P1nSyrBEWW\\\\\\\/OfaSIVGldcVr0FIeYDQtag42zXhYHNfxx8X3oKSUV8Xj2pNNbtUuFferouM2OKD4Qgu+7EA3mBBqewdhXP8o0fcalxjvEKzXmlHZ0JPg0dMfCJ2QnYMH4gtvtQsodCu9s8Civ+dtabjgnL0KsULa0bb2VtamkEk\\\\\\\/RWjIxVTzvR5KB4w52rNOQRboP1zmRdwwJhHAw6R25EPMfkf\\\\\\\/SEwr2poB9uFgcVNuF3DWQxP1pBCRvphekFs8KUvv31J6xuOOL6Et2UTAMyEe+guwqBQcJZDb9Tl3jui85RGHz0fQUpdWIQkBzKrufvcOvbp+YmiCkD9eSMx7\\\\\\\/BgnSbbe32lqnXDlVYtD9Ctdxpw9x8hYM8iMYOMrfeWk3VSxwtIuTrkAAVArKa4KWQlqKYg7uRa818w5avqbXUmhc\\\\\\\/MvP4FcHNGoDl0V1h9OVIQEZYXGbANcSD1Lxabiw+qZ06RL+JurQwQUQiW1HrAuC3EoZeZiIrxpTRatv9OFamcNVPxy5UkbIwz3HR2e9YcuzdrgW4qteHYBGbcvDCupZkDpKMIIrPaczU+Jf9fQXlVzyEZdikTbDNe\\\\\\\/PvU9oWNNT+Fom4PcT0S\\\\\\\/0vz9P5epsYSGzX7WELdJRdhZ7r+R+AZKPiAyjM9fjWB1cUOxgg3c9nTAWFbyIuYeGf5KSBJ5LGUyh8k9pmvveEfS5\\\\\\\/uvmZzESwqd4N4LhpAihnCGxzQbs9jzq0q+2xOfw7QIXuW5N9ozJc4yOkigTSSwhToWq3nHzblvyOT3O3zPPuPfE9whDXOXTZiqkr8zfhm+H6TFPyk4nXyDcS5Nu\\\\\\\/pe\\\\\\\/bTDYuSzHTZ4GV9bio3O6nWjqtqZ4lmC8PTtCVQcGKCiJtJCaWfnEereZTQ72sppR3VkvP6h4OcKwvk9Fw0AB2Sj4fpcv18PXp2AYuKzzT9jPdgTcSMix\\\\\\\/CJVKIl1zzeYKWFcoXHp6fZKkauJLtNUAFFxOijfcMhDQos4lNSusx1va2a7etW0lHQA1LYlkphuLTjdjzfsGD5lWWfErxEcpntVlnwDkXO8kHamQy4cZzS7CE3kJ4GiU6CYej5HnYBs+hV7jbghP6QRosDTYnGAPw9SRRjmLqst7HP0lFdETR4dZMaVh\\\\\\\/TnaSgTxzD0oiIfhmOiL1jMn3JIDRe1yWq+Qnk0r3RgeUNvLDPqOXUF\\\\\\\/yWUheCCulycftgARAmC1Wk8vd5jPMCOsF\\\\\\\/a6F+G8rBWkaM6slHtnkeltBaNXCXRg+6FcrFna50AVX1jwLtCTohTytaX7zmdO3SVMnoW6wk575lYGZREsYU\\\\\\\/cfC26LscLrrHYnCr15JMWQqeX6vU5gS7g1VN7CW+USleKBRwb3vLaQeHcvDchUV6kGi8mldROFcMetfvWPRHDIfQW66FXClo6rYoC4qH0WrVQXC6ePs0DFD1lTb+p+uAsFFjTJjFeE\\\\\\\/SV+gyNSeFjm6ZgbDl19ltk\\\\\\\/uGnop6OUNYFidvL1wMEvtzl4CW8NwvIRRs9tX6TUtid9cORPcMRrjExXjSXjKJ\\\\\\\/Rs8AV1OUA4G2vgjZLkBS\\\\\\\/MVFac8Lg51fJjP4s5KBRFWMPTjJ9OD91YlcYZ1S7gtH\\\\\\\/G7N6PDlA6robaZVErkJxgBcmJpO3JV3V3h5zPOWYtPDhOLJlIKLhQ7TqIom+0JDWaCz6m\\\\\\\/UpyQ+46C53CsNLN1HnG7ZVBSJbOcjcGpUXMjEwlwWnTWvpX9QwgsfJ0ZKI0BGzl4z0Gb2SbedGjIza6POtTlXKD8oBcPo9pDmBvQ1ywROfgm82446KK9hxziUn1TVt15IZMzitcQol4FCEy\\\\\\\/l+QaPIOlUibnIk9PtrIopuZHwOEy59oXuQuGxno73G+LwA49IfpVz7I9k3XBhl2WhFGn8Tw4+NPsSW9ad6wd5pCXse5tchNLPfrJHJRjTa+RHlJb1LqzIhACbM9gI1VUjbehjt9xoxXSZqmrW0TprK0p\\\\\\\/She310WJauKcpaGqjL83sLVoXEmYplWMt29FS9dkAOXV\\\u0022,\\\u0022iv\\\u0022:\\\u00225Scb3jGOnjhrJmhVV0M80w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227a6e18bcb87a6bf9c6e82a0bd65cb556cf7f258362b81e9b02126d7e98147d76\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-483521 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/3/813" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-J_Ke80jkj0hd8JKQaZuQHjb0Evu7t-rFSzvbGb50rn4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/3/813" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-qhKx3U2pztCpxtGbyP53nxx-EedsOfjk1Dkjy4ZMu_Y" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/3/801" title="Persistency of Prediction Accuracy and Genetic Gain in Synthetic Populations Under Recurrent Genomic Selection" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/3/823" title="Cytoplasmic–Nuclear Incompatibility Between Wild Isolates of Caenorhabditis nouraguensis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483521" id="node483521" data-pisa="ggg;7/3/813" data-pisa-master="ggg;g3.116.037531" data-apath="/ggg/7/3/813.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Novel Resampling Improves Statistical Power for Multiple-Trait QTL Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Riyan Cheng</span>, <span class="highwire-citation-author" data-delta="1">R. W. Doerge</span> and <span class="highwire-citation-author" data-delta="2">Justin Borevitz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">813-822;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.037531 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483521" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Riyan Cheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Research School of Biology, The Australian National University, Acton, Australian Capital Territory 2601, Australia, ARC Center of Excellence in Plant Energy Biology, The Australian National University, Acton, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Riyan%2BCheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cheng%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARiyan%2BCheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:riyan.cheng@anu.edu.au" class="" data-icon-position="" data-hide-link-title="0">riyan.cheng@anu.edu.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">R. W. Doerge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Statistics, Department of Biological Sciences, Carnegie Mellon University, Pittsburgh, PA 15213</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BW.%2BDoerge%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Doerge%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AR.%2BW.%2BDoerge%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Justin Borevitz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Research School of Biology, The Australian National University, Acton, Australian Capital Territory 2601, Australia, ARC Center of Excellence in Plant Energy Biology, The Australian National University, Acton, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BBorevitz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Borevitz%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustin%2BBorevitz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/3/813" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:483521" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:483521/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/3/813.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:483521" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:483521/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/3/813.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:483521" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:483521/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/3/813.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:483521" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:483521/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/3/813.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:483521" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:483521/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Multiple-trait analysis typically employs models that associate a quantitative trait locus (QTL) with all of the traits. As a result, statistical power for QTL detection may not be optimal if the QTL contributes to the phenotypic variation in only a small proportion of the traits. Excluding QTL effects that contribute little to the test statistic can improve statistical power. In this article, we show that an optimal power can be achieved when the number of QTL effects is best estimated, and that a stringent criterion for QTL effect selection may improve power when the number of QTL effects is small but can reduce power otherwise. We investigate strategies for excluding trivial QTL effects, and propose a method that improves statistical power when the number of QTL effects is relatively small, and fairly maintains the power when the number of QTL effects is large. The proposed method first uses resampling techniques to determine the number of nontrivial QTL effects, and then selects QTL effects by the backward elimination procedure for significance test. We also propose a method for testing QTL-trait associations that are desired for biological interpretation in applications. We validate our methods using simulations and <em>Arabidopsis thaliana</em> transcript data.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/multiple-trait-mapping" class="hw-term hw-article-keyword hw-article-keyword-multiple-trait-mapping" rel="nofollow">multiple-trait mapping</a></li><li class="kwd"><a href="/keyword/quantitative-trait-locus-qtl" class="hw-term hw-article-keyword hw-article-keyword-quantitative-trait-locus-qtl" rel="nofollow">quantitative trait locus (QTL)</a></li><li class="kwd"><a href="/keyword/resampling" class="hw-term hw-article-keyword hw-article-keyword-resampling" rel="nofollow">resampling</a></li><li class="kwd"><a href="/keyword/statistical-power" class="hw-term hw-article-keyword hw-article-keyword-statistical-power" rel="nofollow">statistical power</a></li></ul><p id="p-3">Quantitative trait locus (QTL) mapping has been a powerful tool for dissecting genetic variants underlying quantitative traits in numerous biological studies and breeding programs. There are two primary concerns in QTL mapping. One is the power for QTL identification under a controlled false positive rate, and the other is accuracy of QTL localization. Advanced intercross lines (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Darvasi and Soller 1995</a>), collaborative cross (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Churchill <em>et al.</em> 2004</a>), and heterogeneous stock (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Valdar <em>et al.</em> 2006</a>) are examples that research efforts have been made to create experimental mapping resources to enhance mapping resolution. In the meantime, various statistical methodologies have been developed to improve statistical power as well as parameter estimation. Joint analysis of multiple complex traits was proposed for QTL mapping in this context as quantitative genetic studies commonly collect data on several to dozens of phenotypes. While multiple traits are usually analyzed separately (referred to as single-trait analysis), there has been plenty of interest in joint analysis of multiple traits over the last two decades. Multivariate analysis of multiple complex traits (referred to as multiple-trait analysis or multitrait analysis) is known for its potential for a higher statistical power and more accurate QTL localization (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Korol <em>et al.</em> 1995</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Jiang and Zeng 1995</a>), and has a wide range of successful applications in various genetic studies such as crops (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Wu <em>et al.</em> 1999</a>), dairy cattle (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bolormaa <em>et al.</em> 2010</a>), model organisms (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Zeng <em>et al.</em> 2000</a>), diseases (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Zhong <em>et al.</em> 2010</a>), and other (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Lan <em>et al.</em> 2006</a>).</p><p id="p-4">The most common methodology for multiple-trait analysis includes multivariate regression (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Ronin <em>et al.</em> 1995</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Knott and Haley 2000</a>), and multiple-trait interval mapping (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Jiang and Zeng 1995</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Korol <em>et al.</em> 1995</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">2001</a>). Multiple-trait interval mapping usually relies on multivariate regression models, also when genotypes of a putative QTL are assumed to be known; however, at a putative QTL other than marker loci, the genotypes are only certain with probabilities, and, therefore, the actual model underlying interval mapping is typically a mixture. Generalized linear models, as well as methods that deal with non-normality, also have their applications in multiple-trait analysis (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Henshall and Goddard 1999</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lange and Whittaker 2001</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Xu <em>et al.</em> 2005</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Liu <em>et al.</em> 2009</a>). In recent years, mixed linear models popular in single-trait analysis have been extended for multiple-trait analysis (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lund <em>et al.</em> 2003</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Malosetti <em>et al.</em> 2008</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hernandez <em>et al.</em> 2012</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Korte <em>et al.</em> 2012</a>). While these multiple-trait approaches generally specify parameters for the effects of a putative QTL on all traits, models that associate different QTL with different traits have been introduced to multiple-trait mapping in both Bayesian and Frequentist frameworks (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Verzilli <em>et al.</em> 2005</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Banerjee <em>et al.</em> 2008</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Wisser <em>et al.</em> 2011</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Silva <em>et al.</em> 2012</a>).</p><p id="p-5">As indicated above, the common practice of multiple-trait analysis uses multivariate regression models, or models of this type, where parameters are specified for QTL effects on all traits. This corresponds to assuming that a putative QTL is associated with all of the traits, which is usually not true in reality. There are two potential consequences (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brown <em>et al.</em> 2014</a>). First, a model with overly excessive QTL effect parameters can reduce the power for QTL detection because the increase in a test statistic due to additional parameters may not compensate for the increase in the degrees of freedom. Second, it is desirable to know which traits are associated with a QTL, and/or which are not; however, multiple-trait analysis that is based on multivariate regression models, or models of this type, does not facilitate such biological interpretation. On the other hand, methods that allow different traits to have different QTL, which facilitates biological interpretation, often focuses on methodological development with little effort to exploit the power for QTL identification (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Verzilli <em>et al.</em> 2005</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Banerjee <em>et al.</em> 2008</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Wisser <em>et al.</em> 2011</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Silva <em>et al.</em> 2012</a>). Motivated to resolve these limitations, we developed methods for multiple-trait analysis with two aims: to (1) improve statistical power for QTL detection; and (2) derive QTL-trait associations with improved power. We validated our methods using simulations and <em>Arabidopsis thaliana</em> transcript data.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-6">Before we proceed to present our methodology, we assume a mapping population of recombinant inbreed lines (RILs) for QTL analysis, and the likelihood ratio test (LRT) statistic for hypothesis testing or model comparison (<em>i.e.</em>, we will use LRT statistics in simulations, and real data analysis unless stated otherwise). These assumptions are simply for ease of description of our methodology; however, the methods to develop will not be limited to RILs but will apply to a general mapping population.</p><p id="p-7">We assume a multiple-trait single-QTL model (without covariates)<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-1.gif"/></noscript></span> <span class="disp-formula-label">(M1)</span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-2.gif"/></noscript></span></span> is the <em>n</em>-th (<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-3.gif"/></noscript></span></span>) observation of the <em>k</em>-th trait, <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-4.gif"/></noscript></span></span> is the intercept, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-5.gif"/></noscript></span></span> is <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-6.gif"/></noscript></span></span>, or <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-7.gif"/></noscript></span></span> if the genotype of the <em>l</em>-th scanning locus (referred to as putative QTL) is <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-8.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-9.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-10.gif"/></noscript></span></span> is the coefficient representing QTL effect, and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-11.gif"/></noscript></span></span> is a <em>p</em>-variate random error term. We may assume that <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-12.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-13.gif"/></noscript></span></span> are independent and follow a normal distribution <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-14.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-15.gif"/></noscript></span></span> being a <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-16.gif"/></noscript></span></span> positive definite matrix. In model (M1), we may omit <em>l</em> if there is no confusion.</p><p id="p-8">Usually, hypotheses for testing the putative QTL in model (M1) are<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-17.gif"/></noscript></span> <span class="disp-formula-label">(H1)</span></span>If we assume the <em>k</em>-th trait, <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-18.gif"/></noscript></span></span> is not associated with the putative QTL, then model (M1) becomes<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-19.gif"/></noscript></span> <span class="disp-formula-label">(M2)</span></span>and the hypotheses for testing the putative QTL are<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-20.gif"/></noscript></span><span class="disp-formula-label">(H2)</span></span>if we use model (M1), or<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-21.gif"/></noscript></span> <span class="disp-formula-label">(H2a)</span></span>if we look at model (M2).</p><p id="p-9">We next present a few methods that can be used to detect QTL: maximum bootstrap power (MBP), test QTL effects individually (Indv), remove trivial QTL effects sequentially (Seq), and model selection by Bayesian information criterion (BIC) (BIC<em><sub>δ</sub></em>). While we propose MBP for multiple-trait analysis, we consider the others for the sake of comparison, as they are natural and intuitive choices when we are interested in QTL-trait associations in applications. Especially, we compare them with the most common approach, <em>i.e.</em>, test hypotheses (H1) for QTL, which we denote by “All”. These methods, except Indv, do not necessarily adequately provide information about QTL-trait associations, but instead can yield excessive false positive QTL-trait associations even though the type I error rate of QTL is appropriately controlled at a nominal significance level. Therefore, we also propose a method that tests QTL-trait associations after a scanning locus is identified as QTL. At the end of this section, we describe a real data set and simulation settings.</p><div id="sec-2" class="subsection"><h3>MBP</h3><p id="p-10">It is known that multiple-trait analysis is likely, but not always, more powerful than single-trait analysis, depending on QTL effects and the residual correlation structure of the traits (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Jiang and Zeng 1995</a>; <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Korol <em>et al.</em> 1995</a>). In <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cheng <em>et al.</em> (2013)</a>, we argued that addition of a trait to multiple-trait analysis can lead to a reduced statistical power for QTL identification if, given other traits, it contributes little to the test statistic. Therefore, we proposed to select a subset of traits for multiple-trait analysis as follows: (1) determine the maximum number, <em>K</em> (up to the total number of traits), of traits to be selected; (2) take 1000 (say) nonparametric bootstrap samples, and estimate the (pseudo) statistical power and its SE for <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-22.gif"/></noscript></span></span> “best” traits, denoted by <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-23.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-24.gif"/></noscript></span></span> respectively; (3) choose the largest <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-25.gif"/></noscript></span></span> such that <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-26.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-27.gif"/></noscript></span></span> The key point is to select a number of traits such that any additional trait contributes little to the statistical power according to certain criterion. This method improves statistical power as we showed in <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Cheng <em>et al.</em> (2013)</a>. On the other hand, we showed that addition of a trait increases statistical power for QTL detection if this trait is reasonably correlated with traits of interest, and is not associated with the QTL. As a useful application, we can include additional traits that satisfy the above conditions when we do not have sufficient power to detect QTL underlying traits of our interest. In such a case, the QTL is not associated with all traits in the model.</p><p id="p-11">Now, we consider a situation where we are interested in joint analysis of multiple traits without excluding any of them or including additional traits. In practice, a QTL may be only associated with some, but not all, of the traits, <em>i.e.</em>, some <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-28.gif"/></noscript></span></span> values in model (M1) are zero. As shown in the <em>Results</em> section, power for QTL detection is not optimal if the model contains parameters representing QTL effects on all of the traits, but not all of the traits are influenced by the QTL. Then, we may improve power by excluding “trivial” QTL effects that are either actually zero or statistically nonsignificant. How to exclude trivial QTL effects is our focus in this paper. We adopt the idea for variable selection of traits in <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Cheng <em>et al.</em> (2013)</a> as introduced above, and propose a procedure as follows.</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-12">Obtain significance threshold <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-29.gif"/></noscript></span></span> for the best <em>k</em> (<span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-30.gif"/></noscript></span></span>) effects at genome-wide significance level, <em>α</em>, where <em>p</em> is the total number of traits;</p></li><li id="list-item-2"><p id="p-13">Take <em>B</em> (say 250) nonparametric bootstrap samples, and, for the <em>b</em>-th bootstrap sample, obtain the test statistic, <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-31.gif"/></noscript></span></span> for the best <em>k</em> (<span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-32.gif"/></noscript></span></span>) effects at a putative QTL;</p></li><li id="list-item-3"><p id="p-14">For each <em>k</em>, calculate the proportion, <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-33.gif"/></noscript></span></span>, of the bootstrap samples where <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-34.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-35.gif"/></noscript></span></span> <em>i.e.</em>, <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-36.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-37.gif"/></noscript></span></span> if <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-38.gif"/></noscript></span></span>, or 0 otherwise;</p></li><li id="list-item-4"><p id="p-15">Find <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-39.gif"/></noscript></span></span> such that <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-40.gif"/></noscript></span></span> if <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-41.gif"/></noscript></span></span> is unique, or take the integer part of the average of the smallest and the largest <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-42.gif"/></noscript></span></span>value otherwise;</p></li><li id="list-item-5"><p id="p-16">Claim a QTL if <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-43.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-44.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-45.gif"/></noscript></span></span>) is the test statistic for the best <em>k</em> effects at the putative QTL in a genome scan.</p></li></ol><p id="p-17">Note that <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-46.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-47.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-48.gif"/></noscript></span></span> are specific to a scanning locus, <em>e.g.</em>, <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-49.gif"/></noscript></span></span> will be used instead of <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-50.gif"/></noscript></span></span> if the best <em>k</em> effects at the <em>l</em>-th of <em>L</em> scanning loci are tested for QTL. Obviously, <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-51.gif"/></noscript></span></span> for any <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-52.gif"/></noscript></span></span> under that null hypothesis of no QTL as <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-53.gif"/></noscript></span></span> is the genome-wide threshold at significance level <em>α</em> for the best <em>k</em> QTL effects, and, therefore, if <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-54.gif"/></noscript></span></span> is chosen purely by chance, we also have <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-55.gif"/></noscript></span></span> The best effects of a putative QTL are the remaining effects after trivial ones are excluded, say, by forward selection, backward elimination, or stepwise selection. Backward elimination is often preferable as it usually has a better performance than forward selection in our current situation, and is less computationally intensive than stepwise backward selection. In (3), we can use a more stringent criterion than <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-56.gif"/></noscript></span></span> because <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-57.gif"/></noscript></span></span> can easily reach 1 when QTL effects are moderately large, so that it is difficult to choose a desirable <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-58.gif"/></noscript></span></span> in (4). For example, we can replace <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-59.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-60.gif"/></noscript></span></span> such that <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-61.gif"/></noscript></span></span> under the null hypothesis of no QTL. The threshold <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-62.gif"/></noscript></span></span>values correct multiplicity in both <em>l</em> and <em>k</em> by adjusting the significance level from <em>α</em> to <em>β</em> (generally <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-63.gif"/></noscript></span></span>), which can be accomplished numerically. We can further assume <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-64.gif"/></noscript></span></span> if we do not give preference to any <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-65.gif"/></noscript></span></span> If <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-66.gif"/></noscript></span></span> is not unique in (4), which is often the case when <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-67.gif"/></noscript></span></span>values reach 1, the smallest <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-68.gif"/></noscript></span></span> can result in underestimation of the number of nontrivial effects, whereas the largest can lead to overestimation. Therefore, we take a trade-off to guard against a serious bias.</p><p id="p-18">Steps 2–4 in the above procedure determine nontrivial effects, and step 5 assumes trivial effects are zero and tests for QTL using model (M2). We note that bootstrap samples generally produce a greater <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-69.gif"/></noscript></span></span> than independent samples, that is, bootstrap <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-70.gif"/></noscript></span></span> overestimates the type I error rate if there is no QTL, or the power if a QTL exists (see Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037531/-/DC1/FileS1.pdf">File S1</a>, for more information). If we still regard <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-71.gif"/></noscript></span></span> an estimate of power at a putative QTL, then the above procedure seeks <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-72.gif"/></noscript></span></span> that maximizes (pseudo) statistical power at the putative QTL, which we refer to as MBP.</p></div><div id="sec-3" class="subsection"><h3>Indv</h3><p id="p-19">The MBP method excludes trivial QTL effects using bootstrap samples. The number of nontrivial QTL effects, <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-73.gif"/></noscript></span></span> is determined by an ensemble of bootstrap samples rather than a single sample. Alternatively, we may be tempted to exclude trivial QTL effects individually if the corresponding test statistics are smaller than a cutoff, <em>τ</em>. Specifically, we consider the test statistics, <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-74.gif"/></noscript></span></span>, in model (M1) with hypotheses<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-75.gif"/></noscript></span> <span class="disp-formula-label">(H3)</span></span><span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-76.gif"/></noscript></span></span> and regard <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-77.gif"/></noscript></span></span> as trivial if <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-78.gif"/></noscript></span></span> This subsequently determines <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-79.gif"/></noscript></span></span>, and testing for QTL existence is then based on the best <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-80.gif"/></noscript></span></span> effects. Though simple yet intuitive, this unfortunately does not work well. A large cutoff <em>τ</em> can result in a great loss of power, especially if the number of QTL effects is relatively large, whereas a small <em>τ</em> may lead to inflated type I error rates, or little gain in power.</p><p id="p-20">If we aim to identify associations between a putative QTL and traits of interest, we can directly test whether QTL effects on the traits are zero, which in turn indicates QTL existence. It is intuitively reasonable to test multiple QTL effects one at a time in a multiple-trait framework. There are two options. One is to test an effect given all other effects in the model, and the other is to test an effect with all other effects being excluded from the model. The former is somewhat similar to looking at the type III sums of squares or the <em>z</em>-test statistics constructed from estimates and their SE in multiple regression, which are usually reported in statistic software applications. The latter is similar to single-trait analysis, but is now under a multiple-trait framework. As seen in the <em>Results</em> section, this will lead to a damage in power due to underfitting if the number of QTL effects is &gt;1. Therefore, this is not a good option, and will not be taken for further consideration. The first option, denoted Indv, will be referred to when we test QTL effects individually. The hypotheses are those in (H3). To control the genome-wide type I error rate at significance level <em>α</em>, the threshold <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-81.gif"/></noscript></span></span> should be such that <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-82.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-83.gif"/></noscript></span></span> is assumed to be the LRT statistic for testing the effect, <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-84.gif"/></noscript></span></span> in model (M1), of a putative QTL at the <em>l</em>-th scanning locus on the <em>k</em>-th trait, given the effects of the putative QTL on all other traits, that is, <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-85.gif"/></noscript></span></span> is adjusted for multiplicity in both traits and scanning loci.</p></div><div id="sec-4" class="subsection"><h3>Seq</h3><p id="p-21">Again, assume we are interested in QTL-trait associations, which, in turn, indicates QTL existence. In statistics, it is not rare to test multiple effects one after another until a significant effect is disclosed. However, selection bias can be induced if the process sequentially searches for, and removes, the least significant effect among all the remaining effects. We may choose to ignore, or take into account, selection bias when we determine significance thresholds. In case selection bias is ignored, a simple way is to use a single threshold with Bonferroni correction, which is basically the BIC<em><sub>δ</sub></em> method we present next, and is stringent for the least significant QTL effects. We will not consider this option, but instead can refer to the BIC<em><sub>δ</sub></em> method for its performance in terms of power. To take selection bias into account, we can estimate thresholds through the permutation test, where the permuted data are analyzed in the same way as the original data (<em>i.e.</em>, the data that we permute). The significance threshold can be adjusted for multiplicity in the same way for MBP and Indv. We denote this method by Seq.</p></div><div id="sec-5" class="subsection"><h3>BIC<em><sub>δ</sub></em></h3><p id="p-22">Model selection is a useful tool in multiple-QTL mapping, in either a Frequentist (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Kao <em>et al.</em> 1999</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Broman and Speed 2002</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Arends <em>et al.</em> 2010</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Wang <em>et al.</em> 2011</a>) or a Bayesian (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Yi <em>et al.</em> 2003</a>) framework. While modern model selection techniques such as shrinkage methodology have appealing applications in genetic analysis (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Whittaker <em>et al.</em> 2000</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Xu 2003</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Wang <em>et al.</em> 2005</a>), the traditional backward elimination and stepwise selection approaches are still favored for their simplicity and interpretability. <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Broman and Speed (2002)</a> recommends backward elimination along with BIC<em><sub>δ</sub></em> for multiple-QTL mapping. BIC<em><sub>δ</sub></em> is different from the well-known BIC only in the penalty, and <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Broman and Speed (2002)</a> suggest a penalty that is associated with the genome-wide threshold rather than the sample size. In our current situation of multiple-trait analysis, we consider BIC<em><sub>δ</sub></em>, and obtain a penalty as follows if we are interested in QTL-trait associations.</p><ol class="list-ord " id="list-2"><li id="list-item-6"><p id="p-23">Permute the genotype data;</p></li><li id="list-item-7"><p id="p-24">Select the best effect for the permuted data by one-step forward selection starting from the model without QTL effects;</p></li><li id="list-item-8"><p id="p-25">Repeat 1) and 2) for 1000 (say) times, and take the <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-86.gif"/></noscript></span></span> quantile, <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-87.gif"/></noscript></span></span> of the test statistics adjusted for multiplicity in scanning loci for the best effect.</p></li></ol><p id="p-26">The hypotheses in (2) are<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-88.gif"/></noscript></span><span class="disp-formula-label">(H4)</span></span><span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-89.gif"/></noscript></span></span> The threshold <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-90.gif"/></noscript></span></span> in (3) is such that <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-91.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-92.gif"/></noscript></span></span> is the statistic that tests hypotheses (H4) in the permuted data. This penalty is expected to control the type I error rate at the genome-wide significance level, <em>α</em>, if forward selection is performed for QTL identification, and, as seen in the <em>Results</em> section, works reasonably well if backward elimination is implemented.</p></div><div id="sec-6" class="subsection"><h3>Test individual QTL effects after a QTL is identified</h3><p>The MBP is expected to reasonably control the type I error rate for QTL identification at a nominal significance level; however, it does not necessarily control the type I error rate for a QTL effect at the nominal significance level. As a matter of fact, <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-93.gif"/></noscript></span></span> tends to overestimate the number of QTL effects. Overfitting is not necessarily a bad thing. First, MBP aims to remove some trivial effects to improve power, rather than estimate the number of nonzero QTL effects. Second, as shown in the <em>Results</em> section, overestimation of the number of QTL effects does not damage power as much as underestimation, though it may not lead to an optimal power. To test individual QTL effects at a controlled type I error rate, <em>α</em>, we can proceed with the following testing procedure (denoted idv).</p><ol class="list-ord " id="list-3"><li id="list-item-9"><p id="p-28">Calculate the test statistic, say <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-94.gif"/></noscript></span></span> for a QTL effect given all other effects;</p></li><li id="list-item-10"><p id="p-29">Claim the QTL effect is not zero if <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-95.gif"/></noscript></span></span>, where <em>υ</em> is the degrees of freedom and <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-96.gif"/></noscript></span></span> is the number of nontrivial QTL effects.</p></li></ol><p>Here, we assume LRT for ease of description, and its asymptotic <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-97.gif"/></noscript></span></span>-distribution. We also assume the trivial effects, which are determined by MBP, are not statistically significantly different from zero, and thus excluded from testing in (2). Denote the trivial set by <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-98.gif"/></noscript></span></span> The hypotheses are<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-99.gif"/></noscript></span><span class="disp-formula-label">(H5)</span></span><span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-100.gif"/></noscript></span></span> The multiplicity in <em>k</em> is corrected by the Bonferroni procedure in (2). As a matter of fact, inclusion of trivial effects does not appreciably promote power, but potentially contributes false positives. To sum up, we can combine MBP and idv to test QTL-trait associations as follows.</p><ol class="list-ord " id="list-4"><li id="list-item-11"><p id="p-30">Test whether a locus is a QTL by the MBP procedure;</p></li><li id="list-item-12"><p id="p-31">If the locus is a QTL, go with the above procedure, idv, to test whether a QTL effect is zero.</p></li></ol><p>Denote this two step-procedure by MBP+idv. Since MBP+idv is based on MBP, identified individual QTL effects will not result in a greater genome-wide type I error rate for QTL detection. Similarly, we can combine any of the other methods with idv to test individual QTL effects, as demonstrated in the <em>Results</em> section.</p></div><div id="sec-7" class="subsection"><h3>Resampling</h3><p id="p-32">Unlike in simulations, where we can generate independent data sets, we typically have only one set of data in an application. While we can make inference just from one set of real data, modern statistical methodology, such as bagging, often takes advantage of an ensemble, <em>e.g.</em>, (nonparametric) bootstrap samples, to improve accuracy and reduce uncertainty. In this regard, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Valdar <em>et al.</em> (2009)</a> proposes model averaging for QTL mapping. Resampling techniques have a wide range of applications, <em>e.g.</em>, assessment of sample statistics, such as variance, hypothesis testing, and model validation. The most common resampling methods include permutation (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fisher 1935</a>), bootstrapping (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Efron 1979</a>), and subsampling. Subsampling draws a subset of the data without replacement. Jackknife is such a resampling method, and, like bootstrap, is usually used to assess variance and bias of an estimate. While the conventional jackknife takes delete-1 samples, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Shao and Wu (1989)</a> propose delete-<em>d</em> jackknife for sample statistics, such as median, for which the delete-1 jackknife estimate is not consistent.</p><p id="p-33">In MBP, we use resampling, bootstrapping, to determine the number of nontrivial QTL effects. We now consider resampling for comparing methods, in terms of both power and making inference in real data applications. Suppose we have <em>B</em> subsamples, and, for the <em>b</em>-th subsample, <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-101.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-102.gif"/></noscript></span></span> is the test statistic at the <em>l</em>-th scanning locus, and <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-103.gif"/></noscript></span></span> is the threshold at genome-wide significance level <em>α</em>. Define <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-104.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-105.gif"/></noscript></span></span> if <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-106.gif"/></noscript></span></span> or 0 otherwise. The statistic <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-107.gif"/></noscript></span></span> can be used to test whether there is QTL. If the subsamples are replaced by independent replicate samples, <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-108.gif"/></noscript></span></span> estimates the type I error rate or statistical power, and asymptotically follows a normal distribution if <em>B</em> is reasonably large. However, subsamples are not independent, and <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-109.gif"/></noscript></span></span> is not asymptotically normal (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037531/-/DC1/FileS1.pdf">File S1</a>). The distribution of <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-110.gif"/></noscript></span></span> can be estimated by using simulations. The subsamples may be jackknife samples, whereas bootstrap does not seem to be a good choice in our current situation (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037531/-/DC1/FileS1.pdf">File S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037531/-/DC1/TableS1.pdf">Table S1</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037531/-/DC1/FigureS5.jpg">Figure S5</a>). For convenience, we call <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-111.gif"/></noscript></span></span> pseudo type I error rate or pseudo power, or simply relative frequency. We note that <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-112.gif"/></noscript></span></span> is positively related to <em>p</em>-values; however, we cannot estimate <em>p</em>-values sufficiently accurately by a limited number of simulations to disclose the advantage of a method over another in terms of power. The above <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-113.gif"/></noscript></span></span> can also be defined at scanning locus <em>l</em> as <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-114.gif"/></noscript></span></span> Depending on the context, <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-115.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-116.gif"/></noscript></span></span> may be relevant to, say, the best <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-117.gif"/></noscript></span></span> QTL effects.</p></div><div id="sec-8" class="subsection"><h3>Gene transcript data and simulation settings</h3><p id="p-34">Simulation studies are often used to check an idea or validate a method in scientific research, and may be essential if there is a lack of theory. Proper settings are crucially important in simulation studies. While it is easy to set up simulations if there are only two traits, it will be hard to come up with a suitable variance–covariance structure of traits when the number of the traits is relatively large. Therefore, we considered the expression trait (e-trait) data that we analyzed in <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Cheng <em>et al.</em> (2013)</a>. Briefly, 211 RILs were derived from two parental inbred <em>A. thaliana</em> accessions, Bayreuth-0 (Bay-0) and Shahdara (Sha), by selfing (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Loudet <em>et al.</em> 2002</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">West <em>et al.</em> 2007</a>), and their gene transcripts (“E-TABM-126” in the ArrayExpress database) were generated by Affymetrix microarray technology (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Kliebenstein <em>et al.</em> 2006</a>). We chose the transcripts of 16 genes from a pathway as phenotypic data. The genotypic data consisted of 95 markers (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">West <em>et al.</em> 2006</a>) across five chromosomes, with the maximum genetic distance between two adjacent markers being 10.944 cM, the minimum 2.224 cM, and the median 4.771 cM. This data are suited for demonstration of multiple-trait methodology. The sample correlations between the e-traits range from 0.02 to 0.96, with a median 0.66. In addition, the known locations of the genes allow us to roughly verify the results of real data analysis.</p><p id="p-35">The data were analyzed in <a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Cheng <em>et al.</em> (2013)</a> using both single-trait and multiple-trait approaches, and several QTL were identified. Single-trait analysis indicated that QTL at markers 3, 10, 37, and 78 were possibly associated with one trait, whereas QTL at markers 27 and 89 seemed to influence multiple traits. In this study, we chose one marker from each of the five chromosomes, and used their genotypic data directly (coded as 0.5 or <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-118.gif"/></noscript></span></span> if the genotype was <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-119.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-120.gif"/></noscript></span></span>, respectively). These five selected markers were 3, 27, 46, 65, and 89. (Markers 46 and 65 were located near the middle of chromosomes 3 and 4, respectively.) In simulations, phenotypic data were generated as follows. We fitted a multivariate multiple regression model with the responses being the 16 e-traits, and predictors being the five chosen marker variables. Denote the estimated intercepts <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-121.gif"/></noscript></span></span> the estimated effects <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-122.gif"/></noscript></span></span>, corresponding to marker <em>l</em> (<span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-123.gif"/></noscript></span></span>), and the estimated residual variance-covariance <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-124.gif"/></noscript></span></span> Let <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-125.gif"/></noscript></span></span> if <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-126.gif"/></noscript></span></span>, or 0 otherwise. The resulting <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-127.gif"/></noscript></span></span> had 1, 11, 0, 3, or 7 nonzero elements for <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-128.gif"/></noscript></span></span> 27, 46, 65, or 89. We redefined <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-129.gif"/></noscript></span></span> (<em>i.e.</em>, there was no truncation). Then, <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-130.gif"/></noscript></span></span> had 16 nonzero elements. We then generated the <em>i</em>-th phenotypic observation <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-131.gif"/></noscript></span></span> when we looked at type I error rates or <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-132.gif"/></noscript></span></span> when we compared methods for statistical power, where <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-133.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-134.gif"/></noscript></span></span> was the coded genotype of the <em>i</em>-th individual at marker <em>l</em>. In the power study, the variation in a simulated trait explained by a QTL ranged from 0.20 to 5.9% (excluding zero effects; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037531/-/DC1/FigureS1.jpg">Figure S1</a>).</p><p id="p-36">Note that we truncated <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-135.gif"/></noscript></span></span> values in order to have different numbers of nonzero effects (NZE), and scaled them so that the estimated power would not be too large (<em>e.g.</em>, nearly 100%) or too small (<em>e.g.</em>, nearly 0%), to disguise the differences among the methods that we compared. There is no room (or need) for improvement on power that is nearly 100%, whereas, if power is too low, even an increase of 15% (say) in power may not be disclosed, or may prove to be significant, by a limited number of simulations. Moreover, it is not common in QTL analysis of quantitative traits that QTL effects are so large that they can be identified in nearly all similar experiments. As it was not easy to simulate reasonable QTL effects and correlation structure, we took advantage of the variance–covariance structure in the data, and used the e-traits as a starting point for generating phenotypic data in our simulation studies.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-37">As stated above, the gene transcript data can be obtained from the ArrayExpress database (query ID “E-TABM-126”) and the genetic marker data was published in <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">West <em>et al.</em> 2006</a>. File S1 in <a id="xref-ref-6-6" class="xref-bibr" href="#ref-6">Cheng <em>et al.</em> (2013)</a> provides information about the 16 genes, and a genetic map of the 95 markers. Both genetic marker data and phenotypic data are also accessible from the webpage of the Borevitz laboratory at the Australian National University <a href="https://borevitzlab.anu.edu.au/resources/association-studies/">https://borevitzlab.anu.edu.au/resources/association-studies/</a>.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>The number of QTL effects and statistical power</h3><p id="p-38">A QTL may or may not influence all traits of interest. However, the most common practice of multiple-trait mapping does not make this distinction. For instance, the multivariate simple regression model, which is often used in multiple-trait analysis, has parameters representing QTL effects on all of the response components. The power for QTL detection may not be optimal if only a small portion of the traits are influenced by the QTL. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> demonstrates this point (refer to simulation settings for power study) by showing the power for various numbers of the “best” QTL effects, which are selected by the backward elimination procedure, and for only NZE to be fitted in the model to identify a putative QTL. NZE is the most powerful, but, in real applications, it is unfortunately impossible to know exactly if a scanning locus is a QTL, or which traits are associated with a QTL. In all the other cases where model selection is involved, an optimal power is achieved when the number of the best QTL effects is near the number of NZE. The power decreases at marker 3, where the QTL has an effect only on one trait, as the number of the best QTL effects increases. At the other markers, the gain in power is negligible or null (markers 27 and 89), or somewhat negative (marker 65) after the number of selected best effects reaches the number of true NZEs. We note that underfitting QTL effects generally damages the power for QTL detection, while overfitting may result in a lower power. As in other statistical applications, power is a primary concern in QTL mapping studies. It is therefore worth exploring methodology for removing trivial QTL effects to potentially gain power for QTL identification.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/813/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimated statistical power for identifying five simulated QTL at genome-wide significance level  using various numbers of the best QTL effects, or exactly all the NZE (shaded area). The numbers of nonzero QTL effects at markers 3, 27, 46, 65, and 89 are 1, 11, 16, 3, and 7, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2073985994" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimated statistical power for identifying five simulated QTL at genome-wide significance level &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-136&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; using various numbers of the best QTL effects, or exactly all the NZE (shaded area). The numbers of nonzero QTL effects at markers 3, 27, 46, 65, and 89 are 1, 11, 16, 3, and 7, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/813/F1.medium.gif" width="440" height="321"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/3/813/F1.medium.gif" width="440" height="321"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/813/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/813/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483604" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-39" class="first-child">Estimated statistical power for identifying five simulated QTL at genome-wide significance level <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/F1/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/F1/embed/mml-math-136.gif"/></noscript></span></span> using various numbers of the best QTL effects, or exactly all the NZE (shaded area). The numbers of nonzero QTL effects at markers 3, 27, 46, 65, and 89 are 1, 11, 16, 3, and 7, respectively.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Simulation results</h3><p id="p-40">We conducted 250 simulations to estimate type I error rates or statistical power. In each simulation, significance thresholds were estimated via the permutation test (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Churchill and Doerge 1994</a>), with 1200 permutations of the genotypic data, and 250 bootstrap samples were taken to determine the number of the best (<em>i.e.</em>, nontrivial) QTL effects. The genome-wide significance level was 0.05 when we compared different methods in terms of type I error rates and statistical power. We expected the estimated type I error rates to be comparable, and fall by ∼0.05 for all methods, so as to not only avoid report excessive false positives but also assure a fair comparison of different methods. When we looked at the power of a method, we primarily compared with the case where all the effects of a putative QTL were included in the model and tested together (denoted by All), unless we clearly state otherwise. We used the LRT statistics as stated in the <em>Materials and Methods</em> section.</p><p id="p-41"><a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> displays the estimated genome-wide type I error rates. As expected, all of them were reasonably around the nominal significance level 0.05. <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows the estimated statistical power at each scanning locus at genome-wide significance level 0.05. Estimated power was basically the same for both MBP and All at markers that were relatively far away from any of markers 3, 27, 46, 65, and 89 where QTL were simulated, and that were near marker 46 where QTL had effects on all of the 16 traits. Otherwise, MBP tended to be more powerful than All. The estimated power for MBP was larger by 14.4% at marker 3, where only one QTL effect was simulated, by up to 5.6% near marker 65, where three QTL effects were simulated, and by 2.8% near marker 89, where seven QTL effects were simulated.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/483655/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/483655/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Genome-wide type I error rate and its SE estimated from 250 simulations for five methods All, MBP, Indv, Seq, and BIC<em><sub>δ</sub></em></span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/813/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relative frequency over 250 simulations that a marker was identified as a QTL by five methods: All, MBP, Indv, Seq, and BIC&#x3B4; at genome-wide significance level 0.05. Five chromosomes are indicated by light-gray colored shading." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2073985994" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relative frequency over 250 simulations that a marker was identified as a QTL by five methods: All, MBP, Indv, Seq, and BIC&lt;em&gt;&lt;sub&gt;&#x3B4;&lt;/sub&gt;&lt;/em&gt; at genome-wide significance level 0.05. Five chromosomes are indicated by light-gray colored shading.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/813/F2.medium.gif" width="440" height="189"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/3/813/F2.medium.gif" width="440" height="189"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/813/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/813/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483602" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-42" class="first-child">Relative frequency over 250 simulations that a marker was identified as a QTL by five methods: All, MBP, Indv, Seq, and BIC<em><sub>δ</sub></em> at genome-wide significance level 0.05. Five chromosomes are indicated by light-gray colored shading.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">We now compare Indv, Seq, and BIC<em><sub>δ</sub></em> with All for QTL detection, though the comparison is more relevant to QTL-trait associations. Indv was less powerful than All, especially at markers 3, 27, 65, and 89. While we might blame overfitting at marker 3, or underfitting at markers 27 and 89, it had little power to identify QTL at marker 65, whose contribution to heritability was small (see <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Interestingly, its power at marker 46, where none of the QTL effects were zero, was still lower than that of a joint test of all effects. This indicates that all NZEs are best tested together. Seq was more powerful than All at marker 3 but less powerful in other regions. This may be partially related to the problem of underfitting or overfitting. Actually, there are many ways to determine thresholds for Seq such that each can control the genome-wide type I error rate, but lead to a different result that reflects a trade-off between underfitting and overfitting in different situations. BIC<em><sub>δ</sub></em> was the most powerful at marker 3, where there was only one QTL effect, but was the least powerful when the number of QTL effects was intermediate or large (and lots of QTL effects were relatively small), as BIC<em><sub>δ</sub></em> can cause serious underfitting.</p><p id="p-44">Indv, Seq, and BIC<em><sub>δ</sub></em> tested scanning loci for QTL by imposing a penalty, or cutoff, on the contributions of individual QTL effects to the test statistic. However, Seq and BIC<em><sub>δ</sub></em> could not appropriately tell if a QTL was associated with a trait (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037531/-/DC1/FigureS2.jpg">Figure S2</a>). Actually, Seq yielded excessive false positives of QTL-trait associations. To derive QTL-trait associations, we need to proceed with further hypothesis testing, <em>e.g.</em>, use the previously proposed method, idv after All, MBP, Seq, or BIC<em><sub>δ</sub></em>. Indv does not need further testing to provide adequate information about QTL-trait associations because the control of the genome-wide type I error rate for QTL identification is based on identification of individuals QTL-trait associations, and the multiplicity in QTL effects is already accounted for. <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> displays the relative frequency that marker 3, 27, 46, 65, or 89 was identified as a QTL underlying individual traits by using MBP+idv, Indv, Seq+idv, or BIC<em><sub>δ</sub></em>+idv. There was no longer any concern about excessive false positives. The advantage of MBP+idv over Indv, Seq+idv, or BIC<em><sub>δ</sub></em>+idv was striking in terms of power.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/813/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relative frequency over 250 simulations that a scanning locus was identified as a QTL for each trait by using (A) MBP+idv, (B) Indv, (C) Seq+idv, and (D) BIC&#x3B4;+idv. The horizontal dotted line represents " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2073985994" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Relative frequency over 250 simulations that a scanning locus was identified as a QTL for each trait by using (A) MBP+idv, (B) Indv, (C) Seq+idv, and (D) BIC&lt;em&gt;&lt;sub&gt;&#x3B4;&lt;/sub&gt;&lt;/em&gt;+idv. The horizontal dotted line represents &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-137&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/813/F3.medium.gif" width="440" height="392"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/3/813/F3.medium.gif" width="440" height="392"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/813/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/813/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483664" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-45" class="first-child">Relative frequency over 250 simulations that a scanning locus was identified as a QTL for each trait by using (A) MBP+idv, (B) Indv, (C) Seq+idv, and (D) BIC<em><sub>δ</sub></em>+idv. The horizontal dotted line represents <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/F3/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/F3/embed/mml-math-137.gif"/></noscript></span></span></p><div class="sb-div caption-clear"></div></div></div><p id="p-46"><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> displays the difference in the relative frequencies that a scanning locus was identified as a QTL for individual traits by using MBP+idv and All+idv. The gain by using MBP over All was apparent at markers 3, 27, and 89. There were 16 QTL effects at marker 46, so it was not of much help to remove trivial effects. QTL at marker 65 explained little variation in the traits, and, consequently, the difference was negligible.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/813/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Difference in the relative frequencies over 250 simulations that a scanning locus was identified as a QTL for traits T1, T2,  T16, by using MBP+idv and All+idv." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2073985994" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Difference in the relative frequencies over 250 simulations that a scanning locus was identified as a QTL for traits T1, T2, &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-138&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; T16, by using MBP+idv and All+idv.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/813/F4.medium.gif" width="440" height="349"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/3/813/F4.medium.gif" width="440" height="349"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/813/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/813/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483645" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-47" class="first-child">Difference in the relative frequencies over 250 simulations that a scanning locus was identified as a QTL for traits T1, T2, <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/F4/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/F4/embed/mml-math-138.gif"/></noscript></span></span> T16, by using MBP+idv and All+idv.</p><div class="sb-div caption-clear"></div></div></div><p id="p-48">In the above simulation study, we preselected five markers and estimated their effects <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-139.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-140.gif"/></noscript></span></span> and then adjusted <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-141.gif"/></noscript></span></span> values to generate QTL. We performed additional simulations to further validate our methods. The settings were the same as above except (1) the scale was 1/4; (2) the scale was 1/2; or (3) the scale was still 1/3 but the sign of <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/813/embed/mml-math-142.gif"/></noscript></span></span> was randomly assigned. Results supported the above conclusions, <em>e.g.</em>, there were no excessive false positives of QTL-trait associations even when simulated QTL were identified over nearly 100% of the simulations (<a href="http://www.g3journal.org/content/suppl/2017/01/06/g3.116.037531.DC1/FigureS6.jpg">Figure S6</a>, B2). We further simulated two traits with various configurations for QTL effects and correlation structures. Results again supported our proposed method (see details in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037531/-/DC1/FileS1.pdf">File S1</a>, <a href="http://www.g3journal.org/content/suppl/2017/01/06/g3.116.037531.DC1/TableS2.pdf">Tables S2</a>, <a href="http://www.g3journal.org/content/suppl/2017/01/06/g3.116.037531.DC1/TableS3.pdf">S3</a>, <a href="http://www.g3journal.org/content/suppl/2017/01/06/g3.116.037531.DC1/TableS4.pdf">S4</a> and <a href="http://www.g3journal.org/content/suppl/2017/01/06/g3.116.037531.DC1/TableS5.pdf">S5</a> and <a href="http://www.g3journal.org/content/suppl/2017/01/06/g3.116.037531.DC1/FigureS6.jpg">Figures S6</a>, <a href="http://www.g3journal.org/content/suppl/2017/01/06/g3.116.037531.DC1/FigureS7.jpg">S7</a> and <a href="http://www.g3journal.org/content/suppl/2017/01/06/g3.116.037531.DC1/FigureS8.jpg">S8</a>).</p></div><div id="sec-13" class="subsection"><h3>Real data example</h3><p id="p-49">We applied the above methods to the e-trait data. Genome-wide significance thresholds were estimated via the permutation test with 1200 permutations of the genotypic data. Each of the 211 jackknife samples of the e-trait data, for which we took 250 bootstrap samples to derive the number of the best QTL effects, was analyzed by using All, BMP, Indv, Seq, and BIC<em><sub>δ</sub></em> and the relative frequency—proportion of the jackknife samples that a scanning locus was identified as a QTL—was obtained for each of the methods (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037531/-/DC1/FigureS3.jpg">Figure S3</a>). The relative frequency reached 1 in a significant portion of the genome due to large QTL effects. This disguised the advantage of one method over another. However, the benefit of using BMP over All was apparent when we looked at individual QTL-trait associations: the relative frequency that a scanning locus was identified by MBP+idv as a QTL underling a trait was greater (by up to 55.9%) in many regions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037531/-/DC1/FigureS4.jpg">Figure S4</a>). The relative frequency obtained from jackknife samples is not the statistical power, but can indicate whether a method is more powerful than another.</p><p id="p-50"><a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> displays the relative frequency, zeroed if &lt;0.25 (which should control the type I error rate of QTL under 0.05; see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037531/-/DC1/FileS1.pdf">File S1</a> for more information), over 211 jackknife samples of the e-trait data that a scanning locus was identified by MBP+idv as a QTL for individual traits at genome-wide significance level 0.05. This provides useful information for the study of <em>cis</em>-acting and <em>trans</em>-acting elements, and potentially identify new genes in addition to the existing network of the 16 genes, about which we will not go into detail as it is not the focus of this article.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/813/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relative frequency over 211 jackknife samples of the e-trait data that a scanning locus was identified by MBP+idv as a QTL for individual traits at genome-wide significance level 0.05. Relative frequency was zeroed if it was " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2073985994" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Relative frequency over 211 jackknife samples of the e-trait data that a scanning locus was identified by MBP+idv as a QTL for individual traits at genome-wide significance level 0.05. Relative frequency was zeroed if it was &lt;0.25. The 16 genes are given as the labels on the vertical axis, and their approximate locations are indicated on the horizontal axis.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/813/F5.medium.gif" width="440" height="288"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/3/813/F5.medium.gif" width="440" height="288"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/813/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/813/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483684" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-51" class="first-child">Relative frequency over 211 jackknife samples of the e-trait data that a scanning locus was identified by MBP+idv as a QTL for individual traits at genome-wide significance level 0.05. Relative frequency was zeroed if it was &lt;0.25. The 16 genes are given as the labels on the vertical axis, and their approximate locations are indicated on the horizontal axis.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><p id="p-52">Multiple-trait analysis has long been advocated for QTL mapping, but its application is somewhat limited due to several possible reasons. First, multiple-trait analysis may not be as computationally easy as single-trait analysis, <em>e.g.</em>, when mixed-effect models are fitted. Second, a main motivation for multiple-trait analysis is a potential gain in power for QTL detection, <em>i.e.</em>, multiple-trait analysis potentially has a higher statistical power than single-trait analysis by taking advantage of the residual correlation structure among the traits. Unfortunately, multiple-trait analysis is not always more powerful but depends on QTL effects, and the correlation structure among the traits (<a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Korol <em>et al.</em> 1995</a>; <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Jiang and Zeng 1995</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Wu <em>et al.</em> 1999</a>). Inclusion of a trait in multiple-trait analysis may not be justified in terms of power for QTL detection, as the presence of other traits can greatly reduce its unique information about the QTL. To overcome this limitation, <a id="xref-ref-6-7" class="xref-bibr" href="#ref-6">Cheng <em>et al.</em> (2013)</a> proposed a variable selection approach, <em>i.e.</em>, to choose a subset of traits for multiple-trait analysis in such a way that any trait in the subset is not redundant given the others, and demonstrated that the proposed method achieves a higher power. Third, multiple-trait analysis in applications usually relies on multivariate regression models, or models of this type, and does not readily provide information on QTL-trait associations to facilitate biological interpretation.</p><p id="p-53">Statistical power is a primary concern with respect to hypothesis testing in statistical applications. This work exploits the potential gain in statistical power from multiple-trait analysis. Complementary to <a id="xref-ref-6-8" class="xref-bibr" href="#ref-6">Cheng <em>et al.</em> (2013)</a>, it deals with how to select QTL effects given a number of traits of interest all being included in multiple-trait analysis. The MBP method we proposed first determines the number of nontrivial effects of a putative QTL, and then fits the best QTL effects in the model such that additional effects will contribute little to, or even have an adverse impact on, QTL detection. The gain of MBP in power tends to increase as the proportion of trivial effects of a putative QTL increases. The proposed method is expected to be generally applicable. On one hand, it is not typical that a QTL will influence all traits of interest. On the other hand, we may intentionally add traits that are closely correlated with a trait of interest, but is not associated with a QTL underlying the trait of interest, to increase power for detecting the QTL (<a id="xref-ref-6-9" class="xref-bibr" href="#ref-6">Cheng <em>et al.</em> 2013</a>). In such cases, a statistical model that associates a QTL with all traits is not parsimonious, and, consequently, has a suboptimal power to identify the QTL. As another aim of this work, we addressed how to derive QTL-trait associations, as it is desirable to know which traits are influenced by a QTL, and which are not. We proposed a two-step procedure, MBP+idv, for this purpose. First, we test by MBP if a scanning locus is a QTL. Second, we test which traits are associated with a QTL. The second step uses information about the number of nontrivial QTL effects from the first step, such that it can adjust to get less stringent significance thresholds for multiple testing. As a result, MBP+idv is able to further improve statistical power for testing QTL-trait associations, in addition to the gain from MBP. We validated our methods by theoretical reasoning, simulations, and real data.</p><p id="p-54">MBP searches a model space that contains the full model for All (<em>i.e.</em>, all QTL effect parameters are included in the model), and judges a QTL effect based on its contribution to the test statistic for All. Therefore, all that matters is how much a QTL effect contributes given other QTL effects. The performance of MBP relies on how well trivial QTL effects are excluded. A hard cut-off leads to either overfitting or underfitting, and thus does not work well for a general situation where QTL effects and the number of QTL effects can be either relatively large or small. In contrast, an ensemble of bootstrap samples provides adaptive selection criteria. Nonetheless, MBP overestimates QTL effects. This guards against underfitting, which can greatly hurt power but may not ultimately improve power due to overfitting. How to best exclude trivial QTL effects is an open question. As a shortcoming, MBP is computationally intensive, which can be a problem if the number of scanning loci is large. It is desirable to develop a method that is less computationally intensive.</p><p id="p-55">We also explored a few other intuitive methods: Indv, Seq, and BIC<em><sub>δ</sub></em>. These can be a natural choice when we are interested in QTL-trait associations in applications. There are many ways to choose thresholds and penalties for Seq and BIC<em><sub>δ</sub></em>. For instance, penalties can be adapted to the number of the best QTL effects in the model by using corresponding significance thresholds rather than a fixed penalty in BIC<em><sub>δ</sub></em>. This will improve power if the number of nontrivial QTL effects is relatively large, but, as a trade-off, reduce power otherwise. We note that inference about QTL-trait associations based on Seq and BIC<em><sub>δ</sub></em> can be unsatisfactory (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037531/-/DC1/FigureS2.jpg">Figure S2, C and D</a>). As mentioned previously, underfitting of QTL effects can be more problematic than overfitting, especially when model selection is involved. BIC<em><sub>δ</sub></em> with a fixed penalty generally leads to underfitting since small QTL effects can be excluded easily by the stringent penalty. This, however, is not always in line with our expectation. Our simulation study provides an example that a scanning locus contributes little or no variation to a trait can be more frequently identified as a QTL for the trait (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037531/-/DC1/FigureS2.jpg">Figure S2D</a>). An explanation for such insensible results would be that the test statistic depends not only on QTL effects and correlation structure, but also on what NZEs should be excluded from the model, which leads to a certain amount of bias.</p><p id="p-56">Lastly, we described our method using quantitative traits, RILs, and linear models, but the principles and methodology should apply to qualitative traits, other mapping populations, and other models, such as linear mixed-effect models, which are now popular in genome-wide association studies.</p><p id="p-57">The analysis was performed by using <em>R</em> package “qtlmt,” which is intended for multiple-trait multiple-QTL analysis (currently suitable for backcross and RIL populations) and is available on <em>R</em> CRAN (<a href="https://cran.r-project.org/web/packages/qtlmt/index.html">https://cran.r-project.org/web/packages/qtlmt/index.html</a>). <em>R</em> code for this study is available from the webpage of the Borevitz laboratory at the Australian National University.</p></div><div class="section" id="sec-15"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-58">Two anonymous reviewers provided useful comments. Computationally intensive simulation studies used the computing facilities at the Australian National University, and the National Computational Infrastructure in Australia. This research was supported by the Australian Plant Phenomics Facility under the National Collaborative Research Infrastructure Strategy of the Australian Government.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-59">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037531/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037531/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-60"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-11-16"><span class="received-label">Received </span>November 16, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-12-29"><span class="accepted-label">Accepted </span>December 29, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Cheng <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.1" data-doi="10.1093/bioinformatics/btq565"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arends</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prins</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">R/qtl: high-throughput multiple QTL mapping.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2990</span>–<span class="cit-lpage">2992</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq565%26rft_id%253Dinfo%253Apmid%252F20966004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMjMvMjk5MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMy84MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.2" data-doi="10.1534/genetics.108.088427"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Banerjee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yandell</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yi</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Bayesian quantitative trait loci mapping for multiple traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">2275</span>–<span class="cit-lpage">2289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.088427%26rft_id%253Dinfo%253Apmid%252F18689903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS80LzIyNzUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvODEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.3" data-doi="10.3168/jds.2009-2980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolormaa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pryce</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Multivariate analysis of a genome-wide association study in dairy cattle.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">3818</span>–<span class="cit-lpage">3833</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DBolormaa%26rft.auinit1%253DS.%26rft.volume%253D93%26rft.issue%253D8%26rft.spage%253D3818%26rft.epage%253D3833%26rft.atitle%253DMultivariate%2Banalysis%2Bof%2Ba%2Bgenome-wide%2Bassociation%2Bstudy%2Bin%2Bdairy%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2009-2980%26rft_id%253Dinfo%253Apmid%252F20655452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2009-2980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20655452&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F813.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280203400048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.4" data-doi="10.1111/1467-9868.00354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Speed</span>  <span class="cit-name-given-names">T. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A model selection approach for identification of quantitative trait loci in experimental crosses.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">641</span>–<span class="cit-lpage">656</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB%26rft.volume%253D64%26rft.spage%253D641%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1467-9868.00354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1467-9868.00354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.5" data-doi="10.1016/j.pbi.2014.02.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sirault</span>  <span class="cit-name-given-names">X. R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rungrat</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Traitcapture: genomic and environment modelling of plant phenomic data.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">73</span>–<span class="cit-lpage">79</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BPlant%2BBiol.%26rft.volume%253D18%26rft.spage%253D73%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2014.02.002%26rft_id%253Dinfo%253Apmid%252F24646691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2014.02.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24646691&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F813.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.6" data-doi="10.1534/genetics.113.155937"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borevitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Selecting informative traits for multivariate quantitative trait locus mapping helps to gain optimal power.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">683</span>–<span class="cit-lpage">691</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.155937%26rft_id%253Dinfo%253Apmid%252F23979571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk1LzMvNjgzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzgxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Empirical threshold values for quantitative trait mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">963</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D138%26rft.issue%253D3%26rft.spage%253D963%26rft.epage%253D971%26rft.atitle%253DEmpirical%2Bthreshold%2Bvalues%2Bfor%2Bquantitative%2Btrait%2Bmapping.%26rft_id%253Dinfo%253Apmid%252F7851788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM4LzMvOTYzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzgxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.8" data-doi="10.1038/ng1104-1133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Airey</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allayee</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Angel</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Attie</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The collaborative cross, a community resource for the genetic analysis of complex traits.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1133</span>–<span class="cit-lpage">1137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D36%26rft.issue%253D11%26rft.spage%253D1133%26rft.epage%253D1137%26rft.atitle%253DThe%2BCollaborative%2BCross%252C%2Ba%2Bcommunity%2Bresource%2Bfor%2Bthe%2Bgenetic%2Banalysis%2Bof%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1104-1133%26rft_id%253Dinfo%253Apmid%252F15514660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1104-1133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15514660&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F813.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224832800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darvasi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soller</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Advanced intercross lines, an experimental population for fine genetic mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">1199</span>–<span class="cit-lpage">1207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDarvasi%26rft.auinit1%253DA.%26rft.volume%253D141%26rft.issue%253D3%26rft.spage%253D1199%26rft.epage%253D1207%26rft.atitle%253DAdvanced%2Bintercross%2Blines%252C%2Ban%2Bexperimental%2Bpopulation%2Bfor%2Bfine%2Bgenetic%2Bmapping.%26rft_id%253Dinfo%253Apmid%252F8582624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0MS8zLzExOTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvODEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.10" data-doi="10.1214/aos/1176344552"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Efron</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Bootstrap methods: another look at the jackknife.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faos%252F1176344552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aos/1176344552&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1979GL00300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-book" id="cit-7.3.813.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1935</span> <span class="cit-source"><em>The Design of Experiments</em></span>, <span class="cit-edition">Ed. 3</span>. <span class="cit-publ-name">Oliver and Boyd</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henshall</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Multiple-trait mapping of quantitative trait loci after selective genotyping using logistic regression.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">885</span>–<span class="cit-lpage">894</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHenshall%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D151%26rft.issue%253D2%26rft.spage%253D885%26rft.epage%253D894%26rft.atitle%253DMultiple-Trait%2BMapping%2Bof%2BQuantitative%2BTrait%2BLoci%2BAfter%2BSelective%2BGenotyping%2BUsing%2BLogistic%2BRegression%26rft_id%253Dinfo%253Apmid%252F9927477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTUxLzIvODg1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzgxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.13" data-doi="10.1371/journal.pone.0038008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bains</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Multi-trait and multi-environment QTL analyses for resistance to wheat diseases.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e38008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSukhwinder-Singh%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253De38008%26rft.epage%253De38008%26rft.atitle%253DMulti-trait%2Band%2Bmulti-environment%2BQTL%2Banalyses%2Bfor%2Bresistance%2Bto%2Bwheat%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0038008%26rft_id%253Dinfo%253Apmid%252F22679489%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0038008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22679489&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F813.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z.-B.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Multiple trait analysis of genetic mapping for quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">1111</span>–<span class="cit-lpage">1127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DJiang%26rft.auinit1%253DC.%26rft.volume%253D140%26rft.issue%253D3%26rft.spage%253D1111%26rft.epage%253D1127%26rft.atitle%253DMultiple%2Btrait%2Banalysis%2Bof%2Bgenetic%2Bmapping%2Bfor%2Bquantitative%2Btrait%2Bloci.%26rft_id%253Dinfo%253Apmid%252F7672582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0MC8zLzExMTEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvODEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teasdale</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Multiple interval mapping for quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">152</span>: <span class="cit-fpage">1203</span>–<span class="cit-lpage">1216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKao%26rft.auinit1%253DC.-H.%26rft.volume%253D152%26rft.issue%253D3%26rft.spage%253D1203%26rft.epage%253D1216%26rft.atitle%253DMultiple%2BInterval%2BMapping%2Bfor%2BQuantitative%2BTrait%2BLoci%26rft_id%253Dinfo%253Apmid%252F10388834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Mi8zLzEyMDMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvODEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.16" data-doi="10.1534/genetics.105.049353"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kliebenstein</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">West</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Leeuwen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genomic survey of gene expression diversity in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">1179</span>–<span class="cit-lpage">1189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.049353%26rft_id%253Dinfo%253Apmid%252F16204207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi8yLzExNzkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvODEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Multitrait least squares for quantitative trait loci detection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">899</span>–<span class="cit-lpage">911</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKnott%26rft.auinit1%253DS.%2BA.%26rft.volume%253D156%26rft.issue%253D2%26rft.spage%253D899%26rft.epage%253D911%26rft.atitle%253DMultitrait%2BLeast%2BSquares%2Bfor%2BQuantitative%2BTrait%2BLoci%2BDetection%26rft_id%253Dinfo%253Apmid%252F11014835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU2LzIvODk5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzgxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korol</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronin</span>  <span class="cit-name-given-names">Y. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirzhner</span>  <span class="cit-name-given-names">V. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Interval mapping of quantitative trait loci employing correlated trait complexes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">1137</span>–<span class="cit-lpage">1147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKorol%26rft.auinit1%253DA.%2BB.%26rft.volume%253D140%26rft.issue%253D3%26rft.spage%253D1137%26rft.epage%253D1147%26rft.atitle%253DInterval%2Bmapping%2Bof%2Bquantitative%2Btrait%2Bloci%2Bemploying%2Bcorrelated%2Btrait%2Bcomplexes.%26rft_id%253Dinfo%253Apmid%252F7672584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0MC8zLzExMzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvODEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korol</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronin</span>  <span class="cit-name-given-names">Y. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Itskovich</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nevo</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Enhanced efficiency of quantitative trait loci mapping analysis based on multivariate complexes of quantitative traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1789</span>–<span class="cit-lpage">1803</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKorol%26rft.auinit1%253DA.%2BB.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1789%26rft.epage%253D1803%26rft.atitle%253DEnhanced%2BEfficiency%2Bof%2BQuantitative%2BTrait%2BLoci%2BMapping%2BAnalysis%2BBased%2Bon%2BMultivariate%2BComplexes%2Bof%2BQuantitative%2BTraits%26rft_id%253Dinfo%253Apmid%252F11290731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE3ODkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvODEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.20" data-doi="10.1038/ng.2376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korte</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilhjálmsson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Segura</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A mixed-model approach for genome-wide association studies of correlated traits in structured populations.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1066</span>–<span class="cit-lpage">1071</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKorte%26rft.auinit1%253DA.%26rft.volume%253D44%26rft.issue%253D9%26rft.spage%253D1066%26rft.epage%253D1071%26rft.atitle%253DA%2Bmixed-model%2Bapproach%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies%2Bof%2Bcorrelated%2Btraits%2Bin%2Bstructured%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2376%26rft_id%253Dinfo%253Apmid%252F22902788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22902788&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F813.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.21" data-doi="10.1371/journal.pgen.0020006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flowers</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yandell</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stapleton</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Combined expression trait correlations and expression quantitative trait locus mapping.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLan%26rft.auinit1%253DH.%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253De6%26rft.epage%253De6%26rft.atitle%253DCombined%2Bexpression%2Btrait%2Bcorrelations%2Band%2Bexpression%2Bquantitative%2Btrait%2Blocus%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020006%26rft_id%253Dinfo%253Apmid%252F16424919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16424919&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F813.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whittaker</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Mapping quantitative trait loci using generalized estimating equations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">1325</span>–<span class="cit-lpage">1337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLange%26rft.auinit1%253DC.%26rft.volume%253D159%26rft.issue%253D3%26rft.spage%253D1325%26rft.epage%253D1337%26rft.atitle%253DMapping%2BQuantitative%2BTrait%2BLoci%2BUsing%2BGeneralized%2BEstimating%2BEquations%26rft_id%253Dinfo%253Apmid%252F11729173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1OS8zLzEzMjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvODEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.23" data-doi="10.1002/gepi.20372"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pei</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papasian</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">H.-W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Bivariate association analyses for the mixture of continuous and binary traits with the use of extended generalized estimating equations.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DLiu%26rft.auinit1%253DJ.%26rft.volume%253D33%26rft.issue%253D3%26rft.spage%253D217%26rft.epage%253D227%26rft.atitle%253DBivariate%2Bassociation%2Banalyses%2Bfor%2Bthe%2Bmixture%2Bof%2Bcontinuous%2Band%2Bbinary%2Btraits%2Bwith%2Bthe%2Buse%2Bof%2Bextended%2Bgeneralized%2Bestimating%2Bequations.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20372%26rft_id%253Dinfo%253Apmid%252F18924135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20372&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18924135&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F813.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264689500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.24" data-doi="10.1007/s00122-001-0825-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loudet</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaillou</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camilleri</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouchez</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daniel-Vedele</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Bay-0 x Shahdara recombinant inbred line population: a powerful tool for the genetic dissection of complex traits in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">1173</span>–<span class="cit-lpage">1184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLoudet%26rft.auinit1%253DO.%26rft.volume%253D104%26rft.issue%253D6-7%26rft.spage%253D1173%26rft.epage%253D1184%26rft.atitle%253DBay-0%2Bx%2BShahdara%2Brecombinant%2Binbred%2Bline%2Bpopulation%253A%2Ba%2Bpowerful%2Btool%2Bfor%2Bthe%2Bgenetic%2Bdissection%2Bof%2Bcomplex%2Btraits%2Bin%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-001-0825-9%26rft_id%253Dinfo%253Apmid%252F12582628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-001-0825-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12582628&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F813.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176083200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sørensen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guldbrandtsen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Multitrait fine mapping of quantitative trait loci using combined linkage disequilibria and linkage analysis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">405</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLund%26rft.auinit1%253DM.%2BS.%26rft.volume%253D163%26rft.issue%253D1%26rft.spage%253D405%26rft.epage%253D410%26rft.atitle%253DMultitrait%2BFine%2BMapping%2Bof%2BQuantitative%2BTrait%2BLoci%2BUsing%2BCombined%2BLinkage%2BDisequilibria%2Band%2BLinkage%2BAnalysis%26rft_id%253Dinfo%253Apmid%252F12586725%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYzLzEvNDA1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzgxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.26" data-doi="10.1007/s10681-007-9594-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malosetti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ribaut</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vargas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A multi-trait multi-environment QTL mixed model with an application to drought and nitrogen stress trials in maize (<em>Zea mays</em> L.).</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D161%26rft.spage%253D241%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-007-9594-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-007-9594-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ronin</span>  <span class="cit-name-given-names">Y. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirzhner</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korol</span>  <span class="cit-name-given-names">A. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Linkage between loci of quantitative traits and marker loci: multi-trait analysis with single marker.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">776</span>–<span class="cit-lpage">786</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D90%26rft.spage%253D776%26rft_id%253Dinfo%253Apmid%252F24172919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24172919&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F813.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RB06600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.28" data-doi="10.1214/aos/1176347263"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">A general theory for jackknife variance estimation.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1176</span>–<span class="cit-lpage">1197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D17%26rft.spage%253D1176%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faos%252F1176347263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aos/1176347263&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">L. D. C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z.-B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Multiple trait multiple interval mapping of quantitative trait loci from inbred line crosses.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">67</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DE%2BSilva%26rft.auinit1%253DL.%2BD.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D67%26rft.epage%253D67%26rft.atitle%253DMultiple%2Btrait%2Bmultiple%2Binterval%2Bmapping%2Bof%2Bquantitative%2Btrait%2Bloci%2Bfrom%2Binbred%2Bline%2Bcrosses.%26rft_id%253Dinfo%253Apmid%252F22852865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22852865&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F813.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.30" data-doi="10.1038/ng1840"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solberg</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gauguier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burnett</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klenerman</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genome-wide genetic association of complex traits in heterogeneous stock mice.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">879</span>–<span class="cit-lpage">887</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DValdar%26rft.auinit1%253DW.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D879%26rft.epage%253D887%26rft.atitle%253DGenome-wide%2Bgenetic%2Bassociation%2Bof%2Bcomplex%2Btraits%2Bin%2Bheterogeneous%2Bstock%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1840%26rft_id%253Dinfo%253Apmid%252F16832355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1840&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16832355&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F813.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.31" data-doi="10.1534/genetics.109.100727"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mapping in structured populations by resample model averaging.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">1263</span>–<span class="cit-lpage">1277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.100727%26rft_id%253Dinfo%253Apmid%252F19474203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Mi80LzEyNjMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvODEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.32" data-doi="10.1002/gepi.20072"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verzilli</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stallard</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whittaker</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Bayesian modelling of multivariate quantitative traits using seemingly unrelated regressions.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">325</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DVerzilli%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D28%26rft.issue%253D4%26rft.spage%253D313%26rft.epage%253D325%26rft.atitle%253DBayesian%2Bmodelling%2Bof%2Bmultivariate%2Bquantitative%2Btraits%2Busing%2Bseemingly%2Bunrelated%2Bregressions.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20072%26rft_id%253Dinfo%253Apmid%252F15789447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20072&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15789447&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F813.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.33" data-doi="10.1126/science.1108791"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weaver</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kesarwani</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Induction of protein secretory pathway is required for systemic acquired resistance.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">308</span>: <span class="cit-fpage">1036</span>–<span class="cit-lpage">1040</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWang%26rft.auinit1%253DD.%26rft.volume%253D308%26rft.issue%253D5724%26rft.spage%253D1036%26rft.epage%253D1040%26rft.atitle%253DInduction%2Bof%2BProtein%2BSecretory%2BPathway%2BIs%2BRequired%2Bfor%2BSystemic%2BAcquired%2BResistance%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1108791%26rft_id%253Dinfo%253Apmid%252F15890886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDgvNTcyNC8xMDM2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzgxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.34" data-doi="10.1534/genetics.110.122796"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yandell</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornberry</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A model selection approach for expression quantitative trait loci (eQTL) mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">611</span>–<span class="cit-lpage">621</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.122796%26rft_id%253Dinfo%253Apmid%252F21115971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg3LzIvNjExIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzgxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.35" data-doi="10.1101/gr.5011206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">West</span>  <span class="cit-name-given-names">M. A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Leeuwen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kliebenstein</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">High-density haplotyping with microarray-based expression and single feature polymorphism markers in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">787</span>–<span class="cit-lpage">795</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5011206%26rft_id%253Dinfo%253Apmid%252F16702412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE2LzYvNzg3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzgxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">West</span>  <span class="cit-name-given-names">M. A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kliebenstein</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Leeuwen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Global eQTL mapping reveals the complex genetic architecture of transcript-level variation in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">1441</span>–<span class="cit-lpage">1450</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Apmid%252F17179097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NS8zLzE0NDEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvODEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.37" data-doi="10.1017/S0016672399004462"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whittaker</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denham</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Marker-assisted selection using ridge regression.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DWhittaker%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D75%26rft.issue%253D2%26rft.spage%253D249%26rft.epage%253D252%26rft.atitle%253DMarker-assisted%2Bselection%2Busing%2Bridge%2Bregression.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672399004462%26rft_id%253Dinfo%253Apmid%252F10816982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672399004462&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10816982&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F813.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086569100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.38" data-doi="10.1073/pnas.1011739108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wisser</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolkman</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patzoldt</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Multivariate analysis of maize disease resistances suggests a pleiotropic genetic basis and implicates a GST gene.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">7339</span>–<span class="cit-lpage">7344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1011739108%26rft_id%253Dinfo%253Apmid%252F21490302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzE4LzczMzkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvODEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">W.-R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">D.-Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">H.-R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Worland</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Time-related mapping of quantitative trait loci underlying tiller number in rice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">297</span>–<span class="cit-lpage">303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWu%26rft.auinit1%253DW.-R.%26rft.volume%253D151%26rft.issue%253D1%26rft.spage%253D297%26rft.epage%253D303%26rft.atitle%253DTime-Related%2BMapping%2Bof%2BQuantitative%2BTrait%2BLoci%2BUnderlying%2BTiller%2BNumber%2Bin%2BRice%26rft_id%253Dinfo%253Apmid%252F9872968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTUxLzEvMjk3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzgxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.40" data-doi="10.1534/genetics.103.019406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Joint mapping of quantitative trait loci for multiple binary characters.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">1045</span>–<span class="cit-lpage">1059</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.103.019406%26rft_id%253Dinfo%253Apmid%252F15489509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS8yLzEwNDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvODEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Estimating polygenic effects using markers of the entire genome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">801</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DXu%26rft.auinit1%253DS.%26rft.volume%253D163%26rft.issue%253D2%26rft.spage%253D789%26rft.epage%253D801%26rft.atitle%253DEstimating%2BPolygenic%2BEffects%2BUsing%2BMarkers%2Bof%2Bthe%2BEntire%2BGenome%26rft_id%253Dinfo%253Apmid%252F12618414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYzLzIvNzg5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzgxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">George</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allison</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Stochastic search variable selection for identifying multiple quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">1129</span>–<span class="cit-lpage">1138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DYi%26rft.auinit1%253DN.%26rft.volume%253D164%26rft.issue%253D3%26rft.spage%253D1129%26rft.epage%253D1138%26rft.atitle%253DStochastic%2BSearch%2BVariable%2BSelection%2Bfor%2BIdentifying%2BMultiple%2BQuantitative%2BTrait%2BLoci%26rft_id%253Dinfo%253Apmid%252F12871920%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NC8zLzExMjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvODEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stam</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mercer</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genetic architecture of a morphological shape difference between two <em>Drosophila</em> species.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">310</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DZeng%26rft.auinit1%253DZ.-B.%26rft.volume%253D154%26rft.issue%253D1%26rft.spage%253D299%26rft.epage%253D310%26rft.atitle%253DGenetic%2BArchitecture%2Bof%2Ba%2BMorphological%2BShape%2BDifference%2BBetween%2BTwo%2BDrosophila%2BSpecies%26rft_id%253Dinfo%253Apmid%252F10628989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU0LzEvMjk5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzgxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.813.44" data-doi="10.1371/journal.pgen.1000932"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beaulaurier</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lum</span>  <span class="cit-name-given-names">P. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Molony</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Liver and adipose expression associated SNPs are enriched for association to type 2 diabetes.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000932</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DZhong%26rft.auinit1%253DH.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De1000932%26rft.epage%253De1000932%26rft.atitle%253DLiver%2Band%2Badipose%2Bexpression%2Bassociated%2BSNPs%2Bare%2Benriched%2Bfor%2Bassociation%2Bto%2Btype%2B2%2Bdiabetes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000932%26rft_id%253Dinfo%253Apmid%252F20463879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000932&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20463879&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F813.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/3/813.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/3/801" title="Persistency of Prediction Accuracy and Genetic Gain in Synthetic Populations Under Recurrent Genomic Selection" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/3/823" title="Cytoplasmic–Nuclear Incompatibility Between Wild Isolates of Caenorhabditis nouraguensis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/3"><span>Volume 7 Issue 3, March 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/3.cover-source.jpg?itok=h8yj2GBo" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/3/813.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8KKq5Gjfw567YVIdbg15DJB1zrAOBpFJBy7tNV5_fGs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/3/813" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/3/813" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/3/813" class="active" data-icon-position="" data-hide-link-title="0">Novel Resampling Improves Statistical Power for Multiple-Trait QTL Mapping</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/483521" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-BMOcFPjFMiBL73e5X-h2eYBJfB47THKgWFajFItpehA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/3/813.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/3/813" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-AAdJ7Col4OCQLTUHbZpWUGwHe_cMvDHAsGuDn0TwfDs" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/3/813" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/3/813.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483521" id="node483521--2" data-pisa="ggg;7/3/813" data-pisa-master="ggg;g3.116.037531" data-apath="/ggg/7/3/813.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Novel Resampling Improves Statistical Power for Multiple-Trait QTL Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Riyan Cheng</span>, <span class="highwire-citation-author" data-delta="1">R. W. Doerge</span> and <span class="highwire-citation-author" data-delta="2">Justin Borevitz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">813-822;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.037531 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483521--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Riyan Cheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Research School of Biology, The Australian National University, Acton, Australian Capital Territory 2601, Australia, ARC Center of Excellence in Plant Energy Biology, The Australian National University, Acton, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Riyan%2BCheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cheng%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARiyan%2BCheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:riyan.cheng@anu.edu.au" class="" data-icon-position="" data-hide-link-title="0">riyan.cheng@anu.edu.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">R. W. Doerge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Statistics, Department of Biological Sciences, Carnegie Mellon University, Pittsburgh, PA 15213</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BW.%2BDoerge%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Doerge%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AR.%2BW.%2BDoerge%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Justin Borevitz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Research School of Biology, The Australian National University, Acton, Australian Capital Territory 2601, Australia, ARC Center of Excellence in Plant Energy Biology, The Australian National University, Acton, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BBorevitz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Borevitz%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustin%2BBorevitz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/3/813" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/3/813&amp;title=Novel%20Resampling%20Improves%20Statistical%20Power%20for%20Multiple-Trait%20QTL%20Mapping" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/3/813&amp;title=Novel%20Resampling%20Improves%20Statistical%20Power%20for%20Multiple-Trait%20QTL%20Mapping" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/3/813&amp;title=Novel%20Resampling%20Improves%20Statistical%20Power%20for%20Multiple-Trait%20QTL%20Mapping" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/3/813&amp;text=Novel%20Resampling%20Improves%20Statistical%20Power%20for%20Multiple-Trait%20QTL%20Mapping" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/3/813&amp;title=Novel%20Resampling%20Improves%20Statistical%20Power%20for%20Multiple-Trait%20QTL%20Mapping" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/3/813&amp;t=Novel%20Resampling%20Improves%20Statistical%20Power%20for%20Multiple-Trait%20QTL%20Mapping" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/3/813&amp;title=Novel%20Resampling%20Improves%20Statistical%20Power%20for%20Multiple-Trait%20QTL%20Mapping" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/3/813&amp;title=Novel%20Resampling%20Improves%20Statistical%20Power%20for%20Multiple-Trait%20QTL%20Mapping" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/483521/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483521" id="node483521--3" data-pisa="ggg;7/3/813" data-pisa-master="ggg;g3.116.037531" data-apath="/ggg/7/3/813.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Novel Resampling Improves Statistical Power for Multiple-Trait QTL Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Riyan Cheng</span>, <span class="highwire-citation-author" data-delta="1">R. W. Doerge</span> and <span class="highwire-citation-author" data-delta="2">Justin Borevitz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">813-822;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.037531 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483521--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Riyan Cheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Research School of Biology, The Australian National University, Acton, Australian Capital Territory 2601, Australia, ARC Center of Excellence in Plant Energy Biology, The Australian National University, Acton, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Riyan%2BCheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cheng%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARiyan%2BCheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:riyan.cheng@anu.edu.au" class="" data-icon-position="" data-hide-link-title="0">riyan.cheng@anu.edu.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">R. W. Doerge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Statistics, Department of Biological Sciences, Carnegie Mellon University, Pittsburgh, PA 15213</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BW.%2BDoerge%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Doerge%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AR.%2BW.%2BDoerge%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Justin Borevitz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Research School of Biology, The Australian National University, Acton, Australian Capital Territory 2601, Australia, ARC Center of Excellence in Plant Energy Biology, The Australian National University, Acton, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BBorevitz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Borevitz%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustin%2BBorevitz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=813&amp;rft.epage=822&amp;rft.atitle=Novel%20Resampling%20Improves%20Statistical%20Power%20for%20Multiple-Trait%20QTL%20Mapping&amp;rft.volume=7&amp;rft.issue=3&amp;rft.date=2017-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Cheng%2C+Riyan&amp;rft.au=Doerge%2C+R.+W.&amp;rft.au=Borevitz%2C+Justin"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/483521/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/483521/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/483521/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/483521/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/483521/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/483521/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/483521/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/483521/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/483521/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/483521/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/483521/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/483521/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/3/813" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/3/813.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/3/813.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/3/813.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/3/813.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
