<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/3/967" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/3/967.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/3/967.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/3/967.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/3/967/embed/mml-math-16.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/3/967.atom" />
<meta name="HW.pisa" content="ggg;7/3/967" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The Impact of Selection at the Amino Acid Level on the Usage of Synonymous Codons" />
<meta name="DC.Identifier" content="10.1534/g3.116.038125" />
<meta name="DC.Date" content="2017-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Błażej et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="There are two main forces that affect usage of synonymous codons: directional mutational pressure and selection. The effectiveness of protein translation is usually considered as the main selectional factor. However, biased codon usage can also be a byproduct of a general selection at the amino acid level interacting with nucleotide replacements. To evaluate the validity and strength of such an effect, we superimposed &gt;3.5 billion unrestricted mutational processes on the selection of nonsynonymous substitutions based on the differences in physicochemical properties of the coded amino acids. Using a modified evolutionary optimization algorithm, we determined the conditions in which the effect on the relative codon usage is maximized. We found that the effect is enhanced by mutational processes generating more adenine and thymine than guanine and cytosine, as well as more purines than pyrimidines. Interestingly, this effect is observed only under an unrestricted model of nucleotide substitution, and disappears when the mutational process is time-reversible. Comparison of the simulation results with data for real protein coding sequences indicates that the impact of selection at the amino acid level on synonymous codon usage cannot be neglected. Furthermore, it can considerably interfere, especially in AT-rich genomes, with other selections on codon usage, e.g. , translational efficiency. It may also lead to difficulties in the recognition of other effects influencing codon bias, and an overestimation of protein coding sequences whose codon usage is subjected to adaptational selection." />
<meta name="DC.Contributor" content="Paweł Błażej" />
<meta name="DC.Contributor" content="Dorota Mackiewicz" />
<meta name="DC.Contributor" content="Małgorzata Wnętrzak" />
<meta name="DC.Contributor" content="Paweł Mackiewicz" />
<meta name="article:published_time" content="2017-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="The Impact of Selection at the Amino Acid Level on the Usage of Synonymous Codons" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;There are two main forces that affect usage of synonymous codons: directional mutational pressure and selection. The effectiveness of protein translation is usually considered as the main selectional factor. However, biased codon usage can also be a byproduct of a general selection at the amino acid level interacting with nucleotide replacements. To evaluate the validity and strength of such an effect, we superimposed &amp;gt;3.5&amp;nbsp;billion unrestricted mutational processes on the selection of nonsynonymous substitutions based on the differences in physicochemical properties of the coded amino acids. Using a modified evolutionary optimization algorithm, we determined the conditions in which the effect on the relative codon usage is maximized. We found that the effect is enhanced by mutational processes generating more adenine and thymine than guanine and cytosine, as well as more purines than pyrimidines. Interestingly, this effect is observed only under an unrestricted model of nucleotide substitution, and disappears when the mutational process is time-reversible. Comparison of the simulation results with data for real protein coding sequences indicates that the impact of selection at the amino acid level on synonymous codon usage cannot be neglected. Furthermore, it can considerably interfere, especially in AT-rich genomes, with other selections on codon usage, &lt;i&gt;e.g.&lt;/i&gt;, translational efficiency. It may also lead to difficulties in the recognition of other effects influencing codon bias, and an overestimation of protein coding sequences whose codon usage is subjected to adaptational selection.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/03/01" />
<meta name="citation_mjid" content="ggg;7/3/967" />
<meta name="citation_id" content="7/3/967" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/3/967" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/3/967.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/3/967.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/3/967.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.038125" />
<meta name="citation_pmid" content="28122952" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="15" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="967" />
<meta name="citation_lastpage" content="981" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Paweł Błażej" />
<meta name="citation_author_institution" content="Department of Genomics, Faculty of Biotechnology, University of Wrocław, 50-383, Poland" />
<meta name="citation_author" content="Dorota Mackiewicz" />
<meta name="citation_author_institution" content="Department of Genomics, Faculty of Biotechnology, University of Wrocław, 50-383, Poland" />
<meta name="citation_author" content="Małgorzata Wnętrzak" />
<meta name="citation_author_institution" content="Department of Genomics, Faculty of Biotechnology, University of Wrocław, 50-383, Poland" />
<meta name="citation_author" content="Paweł Mackiewicz" />
<meta name="citation_author_institution" content="Department of Genomics, Faculty of Biotechnology, University of Wrocław, 50-383, Poland" />
<meta name="citation_author_email" content="pamac@smorfland.uni.wroc.pl" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Akashi;citation_title=Synonymous codon usage in Drosophila melanogaster: natural selection and translational accuracy.;citation_pages=927-935;citation_volume=136;citation_year=1994;citation_issue=3;citation_pmid=8005445" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Akashi;citation_title=Translational Selection and Yeast Proteome Evolution;citation_pages=1291-1303;citation_volume=164;citation_year=2003;citation_issue=4;citation_pmid=12930740" />
<meta name="citation_reference" content="citation_journal_title=Journal of biomolecular structure &amp; dynamics;citation_journal_abbrev=J Biomol Struct Dyn;citation_author=T. Banerjee;citation_author=S. Basak;citation_author=S. K. Gupta;citation_author=T. C. Ghosh;citation_title=Evolutionary forces in shaping the codon and amino acid usages in Blochmannia floridanus.;citation_pages=13-23;citation_volume=22;citation_year=2004;citation_issue=1;citation_pmid=15214801" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=R. A. Bartoszewski;citation_author=M. Jablonsky;citation_author=S. Bartoszewska;citation_author=L. Stevenson;citation_author=Q. Dai;citation_title=A synonymous single nucleotide polymorphism in DeltaF508 CFTR alters the secondary structure of the mRNA and the expression of the mutant protein.;citation_volume=285;citation_year=2010;citation_pmid=20628052;citation_doi=10.1074/jbc.M110.154575" />
<meta name="citation_reference" content="citation_journal_title=Biol. Lett.;citation_author=G. A. Bazykin;citation_title=Changing preferences: deformation of single position amino acid fitness landscapes and evolution of proteins.;citation_volume=11;citation_year=2015;citation_pmid=26445980;citation_doi=10.1098/rsbl.2015.0315" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=J. L. Bennetzen;citation_author=B. D. Hall;citation_title=Codon selection in yeast;citation_pages=3026-3031;citation_volume=257;citation_year=1982;citation_issue=6;citation_pmid=7037777" />
<meta name="citation_reference" content="citation_journal_title=Mol. Syst. Biol.;citation_author=K. Bentele;citation_author=P. Saffert;citation_author=R. Rauscher;citation_author=Z. Ignatova;citation_author=N. Bluthgen;citation_title=Efficient translation initiation dictates codon usage at gene start.;citation_volume=9;citation_year=2013;citation_pmid=23774758;citation_doi=10.1038/msb.2013.32" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Bulmer;citation_title=The selection-mutation-drift theory of synonymous codon usage.;citation_pages=897-907;citation_volume=129;citation_year=1991;citation_issue=3;citation_pmid=1752426" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=G. Cannarrozzi;citation_author=N. N. Schraudolph;citation_author=M. Faty;citation_author=P. von Rohr;citation_author=M. T. Friberg;citation_title=A role for codon order in translation dynamics.;citation_pages=355-367;citation_volume=141;citation_year=2010;citation_issue=2;citation_pmid=20403329;citation_doi=10.1016/j.cell.2010.02.036" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=A. Carbone;citation_author=R. Madden;citation_title=Insights on the evolution of metabolic networks of unicellular translationally biased organisms from transcriptomic data and sequence analysis.;citation_pages=456-469;citation_volume=61;citation_year=2005;citation_issue=4;citation_pmid=16187158" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Biophys.;citation_author=J. L. Chaney;citation_author=P. L. Clark;citation_title=Roles for synonymous codon usage in protein biogenesis.;citation_pages=143-166;citation_volume=44;citation_year=2015;citation_pmid=25747594;citation_doi=10.1146/annurev-biophys-060414-034333" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. L. Chen;citation_author=W. Lee;citation_author=A. K. Hottes;citation_author=L. Shapiro;citation_author=H. H. McAdams;citation_title=Codon usage between genomes is constrained by genome-wide mutational processes.;citation_volume=101;citation_year=2004;citation_pmid=14990797;citation_doi=10.1073/pnas.0307827100" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=B. Clarke;citation_title=Darwinian Evolution of Proteins;citation_pages=1009-1011;citation_volume=168;citation_year=1970;citation_issue=3934;citation_pmid=4909620;citation_doi=10.1126/science.168.3934.1009" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=T. F. Clarke;citation_author=P. L. Clark;citation_title=Increased incidence of rare codon clusters at 5' and 3' gene termini: implications for function.;citation_pages=118-118;citation_volume=11;citation_year=2010;citation_pmid=20167116;citation_doi=10.1186/1471-2164-11-118" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. R. Coleman;citation_author=D. Papamichail;citation_author=S. Skiena;citation_author=B. Futcher;citation_author=E. Wimmer;citation_title=Virus Attenuation by Genome-Scale Changes in Codon Pair Bias;citation_pages=1784-1787;citation_volume=320;citation_year=2008;citation_issue=5884;citation_pmid=18583614;citation_doi=10.1126/science.1155761" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. M. Comeron;citation_title=Selective and Mutational Patterns Associated With Gene Expression in Humans: Influences on Synonymous Composition and Intron Presence;citation_pages=1293-1304;citation_volume=167;citation_year=2004;citation_issue=3;citation_pmid=15280243;citation_doi=10.1534/genetics.104.026351" />
<meta name="citation_reference" content="citation_journal_title=Haematologica;citation_author=V. Daidone;citation_author=L. Gallinaro;citation_author=M. G. Cattini;citation_author=E. Pontara;citation_author=A. Bertomoro;citation_title=An apparently silent nucleotide substitution (c.7056C &gt; T) in the von Willebrand factor gene is responsible for type 1 von Willebrand disease.;citation_volume=96;citation_year=2011;citation_pmid=21393328;citation_doi=10.3324/haematol.2010.036848" />
<meta name="citation_reference" content="citation_title=Handbook of Evolutionary Computation;citation_title=A history of evolutionary computation;citation_author=K. De Jong;citation_author=D. B. Fogel;citation_author=H.-P. Schwefel;citation_pages=A2.3:1-12;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. dos Reis;citation_author=R. Savva;citation_author=L. Wernisch;citation_title=Solving the riddle of codon usage preferences: a test for translational selection;citation_pages=5036-5044;citation_volume=32;citation_year=2004;citation_issue=17;citation_pmid=15448185;citation_doi=10.1093/nar/gkh834" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. A. Drummond;citation_author=C. O. Wilke;citation_title=Mistranslation-induced protein misfolding as a dominant constraint on coding-sequence evolution.;citation_pages=341-352;citation_volume=134;citation_year=2008;citation_issue=2;citation_pmid=18662548;citation_doi=10.1016/j.cell.2008.05.042" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=L. Duret;citation_author=D. Mouchiroud;citation_title=Expression pattern and, surprisingly, gene length shape codon usage in Caenorhabditis, Drosophila, and Arabidopsis;citation_pages=4482-4487;citation_volume=96;citation_year=1999;citation_issue=8;citation_pmid=10200288;citation_doi=10.1073/pnas.96.8.4482" />
<meta name="citation_reference" content="citation_journal_title=Current issues in molecular biology;citation_journal_abbrev=Curr Issues Mol Biol;citation_author=M. D. Ermolaeva;citation_title=Synonymous codon usage in bacteria.;citation_pages=91-97;citation_volume=3;citation_year=2001;citation_issue=4;citation_pmid=11719972" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Eyre-Walker;citation_author=M. Bulmer;citation_title=Synonymous substitution rates in enterobacteria.;citation_pages=1407-1412;citation_volume=140;citation_year=1995;citation_issue=4;citation_pmid=7498779" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=A. Fedorov;citation_author=S. Saxonov;citation_author=W. Gilbert;citation_title=Regularities of context-dependent codon bias in eukaryotic genes;citation_pages=1192-1197;citation_volume=30;citation_year=2002;citation_issue=5;citation_pmid=11861911;citation_doi=10.1093/nar/30.5.1192" />
<meta name="citation_reference" content="citation_title=Inferring Phylogenies.;citation_author=J. Felsenstein;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=A. C. Frank;citation_author=J. R. Lobry;citation_title=Asymmetric substitution patterns: a review of possible underlying mutational or selective mechanisms.;citation_pages=65-77;citation_volume=238;citation_year=1999;citation_issue=1;citation_pmid=10570985;citation_doi=10.1016/S0378-1119(99)00297-8" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=S. Garcia-Vallve;citation_author=E. Guzman;citation_author=M. A. Montero;citation_author=A. Romeu;citation_title=HGT-DB: a database of putative horizontally transferred genes in prokaryotic complete genomes;citation_pages=187-189;citation_volume=31;citation_year=2003;citation_issue=1;citation_pmid=12519978;citation_doi=10.1093/nar/gkg004" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Ghaemmaghami;citation_author=W. K. Huh;citation_author=K. Bower;citation_author=R. W. Howson;citation_author=A. Belle;citation_title=Global analysis of protein expression in yeast.;citation_pages=737-741;citation_volume=425;citation_year=2003;citation_issue=6959;citation_pmid=14562106;citation_doi=10.1038/nature02046" />
<meta name="citation_reference" content="citation_journal_title=Biochemical and biophysical research communications;citation_journal_abbrev=Biochem Biophys Res Commun;citation_author=R. M. Goetz;citation_author=A. Fuglsang;citation_title=Correlation of codon bias measures with mRNA levels: analysis of transcriptome data from Escherichia coli.;citation_pages=4-7;citation_volume=327;citation_year=2005;citation_issue=1;citation_pmid=15629421;citation_doi=10.1016/j.bbrc.2004.11.134" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=D. B. Goodman;citation_author=G. M. Church;citation_author=S. Kosuri;citation_title=Causes and effects of N-terminal codon bias in bacterial genes.;citation_volume=342;citation_year=2013;citation_pmid=24072823;citation_doi=10.1126/science.1241934" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. Grantham;citation_title=Amino Acid Difference Formula to Help Explain Protein Evolution;citation_pages=862-864;citation_volume=185;citation_year=1974;citation_issue=4154;citation_pmid=4843792;citation_doi=10.1126/science.185.4154.862" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. Grantham;citation_author=C. Gautier;citation_author=M. Gouy;citation_author=R. Mercier;citation_author=A. Pave;citation_title=Codon catalog usage and the genome hypothesis.;citation_pages=R49-R62;citation_volume=8;citation_year=1980;citation_pmid=6986610" />
<meta name="citation_reference" content="citation_journal_title=Trends in biotechnology;citation_journal_abbrev=Trends Biotechnol;citation_author=C. Gustafsson;citation_author=S. Govindarajan;citation_author=J. Minshull;citation_title=Codon bias and heterologous protein expression.;citation_pages=346-353;citation_volume=22;citation_year=2004;citation_issue=7;citation_pmid=15245907;citation_doi=10.1016/j.tibtech.2004.04.006" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=A. L. Halpern;citation_author=W. J. Bruno;citation_title=Evolutionary distances for protein-coding sequences: modeling site- specific residue frequencies;citation_pages=910-917;citation_volume=15;citation_year=1998;citation_issue=7;citation_pmid=9656490;citation_doi=10.1093/oxfordjournals.molbev.a025995" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_journal_abbrev=Microbiology;citation_author=J. T. Herbeck;citation_author=D. P. Wall;citation_author=J. J. Wernegreen;citation_title=Gene expression level influences amino acid usage, but not codon usage, in the tsetse fly endosymbiont Wigglesworthia;citation_pages=2585-2596;citation_volume=149;citation_year=2003;citation_issue=9;citation_pmid=12949182;citation_doi=10.1099/mic.0.26381-0" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=R. Hershberg;citation_author=D. A. Petrov;citation_title=Selection on codon bias.;citation_pages=287-299;citation_volume=42;citation_year=2008;citation_pmid=18983258;citation_doi=10.1146/annurev.genet.42.110807.091442" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=T. Ikemura;citation_title=Correlation between the abundance of Escherichia coli transfer RNAs and the occurrence of the respective codons in its protein genes.;citation_pages=1-21;citation_volume=146;citation_year=1981;citation_issue=1;citation_pmid=6167728;citation_doi=10.1016/0022-2836(81)90363-6" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=T. Ikemura;citation_title=Codon usage and tRNA content in unicellular and multicellular organisms;citation_pages=13-34;citation_volume=2;citation_year=1985;citation_issue=1;citation_pmid=3916708" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=N. T. Ingolia;citation_title=Ribosome profiling: new views of translation, from single codons to genome scale.;citation_pages=205-213;citation_volume=15;citation_year=2014;citation_pmid=24468696;citation_doi=10.1038/nrg3645" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=N. T. Ingolia;citation_author=S. Ghaemmaghami;citation_author=J. R. S. Newman;citation_author=J. S. Weissman;citation_title=Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling.;citation_volume=324;citation_year=2009;citation_pmid=19213877;citation_doi=10.1126/science.1168978" />
<meta name="citation_reference" content="citation_journal_title=Skand. Aktuar. J.;citation_author=J. L. Jensen;citation_title=Markoff chains as an aid in the study of Markoff processes.;citation_pages=87-91;citation_volume=1953;citation_year=1953" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=S. Kanaya;citation_author=Y. Yamada;citation_author=Y. Kudo;citation_author=T. Ikemura;citation_title=Studies of codon usage and tRNA genes of 18 unicellular organisms and quantification of Bacillus subtilis tRNAs: gene expression level and species-specific diversity of codon usage based on multivariate analysis.;citation_pages=143-155;citation_volume=238;citation_year=1999;citation_issue=1;citation_pmid=10570992;citation_doi=10.1016/S0378-1119(99)00225-5" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Kertesz;citation_author=Y. Wan;citation_author=E. Mazor;citation_author=J. L. Rinn;citation_author=R. C. Nutter;citation_title=Genome-wide measurement of RNA secondary structure in yeast.;citation_pages=103-107;citation_volume=467;citation_year=2010;citation_issue=7311;citation_pmid=20811459;citation_doi=10.1038/nature09322" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=C. J. Kimchi-Sarfaty;citation_author=M. Oh;citation_author=I. W. Kim;citation_author=Z. E. Sauna;citation_author=A. M. Calcagno;citation_title=A ‘silent’ polymorphism in the MDR1 gene changes substrate specificity.;citation_pages=525-528;citation_volume=315;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=R. D. Knight;citation_author=S. J. Freeland;citation_author=L. F. Landweber;citation_title=A simple model based on mutation and selection explains trends in codon and amino-acid usage and GC composition within and across genomes.;citation_pages=RESEARCH0010-RESEARCH0010;citation_volume=2;citation_year=2001;citation_issue=4;citation_pmid=11305938" />
<meta name="citation_reference" content="citation_journal_title=Journal of applied genetics;citation_journal_abbrev=J Appl Genet;citation_author=M. Kowalczuk;citation_author=P. Mackiewicz;citation_author=D. Mackiewicz;citation_author=A. Nowicka;citation_author=M. Dudkiewicz;citation_title=DNA asymmetry and the replicational mutational pressure.;citation_pages=553-577;citation_volume=42;citation_year=2001;citation_issue=4;citation_pmid=14564030" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=G. Kudla;citation_author=A. W. Murray;citation_author=D. Tollervey;citation_author=J. B. Plotkin;citation_title=Coding-Sequence Determinants of Gene Expression in Escherichia coli;citation_pages=255-258;citation_volume=324;citation_year=2009;citation_issue=5924;citation_pmid=19359587;citation_doi=10.1126/science.1170160" />
<meta name="citation_reference" content="citation_journal_title=FASEB J.;citation_author=A. Lazrak;citation_author=L. Fu;citation_author=V. Bali;citation_author=R. Bartoszewski;citation_author=A. Rab;citation_title=The silent codon change I507-ATC-&gt;ATT contributes to the severity of the DeltaF508 CFTR channel dysfunction.;citation_volume=27;citation_year=2013;citation_pmid=23907436;citation_doi=10.1096/fj.13-227330" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=J. Li;citation_author=J. Zhou;citation_author=Y. Wu;citation_author=S. H. Yang;citation_author=D. C. Tian;citation_title=GC-content of synonymous codons profoundly influences amino acid usage;citation_pages=2027-2036;citation_volume=5;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=J. R. Lobry;citation_title=Asymmetric substitution patterns in the two DNA strands of bacteria;citation_pages=660-665;citation_volume=13;citation_year=1996;citation_issue=5;citation_pmid=8676740;citation_doi=10.1093/oxfordjournals.molbev.a025626" />
<meta name="citation_reference" content="citation_journal_title=J. Appl. Genet.;citation_author=P. Mackiewicz;citation_author=A. Gierlik;citation_author=M. Kowalczuk;citation_author=M. R. Dudek;citation_author=S. Cebrat;citation_title=Asymmetry of nucleotide composition of prokaryotic chromosomes.;citation_pages=1-14;citation_volume=40;citation_year=1999" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=P. Mackiewicz;citation_author=A. Gierlik;citation_author=M. Kowalczuk;citation_author=M. R. Dudek;citation_author=S. Cebrat;citation_title=How Does Replication-Associated Mutational Pressure Influence Amino Acid Composition of Proteins?;citation_pages=409-416;citation_volume=9;citation_year=1999;citation_issue=5;citation_pmid=10330120" />
<meta name="citation_reference" content="citation_journal_title=Physica A;citation_author=P. Mackiewicz;citation_author=A. Gierlik;citation_author=M. Kowalczuk;citation_author=D. Szczepanik;citation_author=M. R. Dudek;citation_title=Mechanisms generating long-range correlation in nucleotide composition of the Borrelia burgdorferi genome.;citation_pages=103-115;citation_volume=273;citation_year=1999" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. O. McInerney;citation_title=Replicational and transcriptional selection on codon usage in Borrelia burgdorferi;citation_pages=10698-10703;citation_volume=95;citation_year=1998;citation_issue=18;citation_pmid=9724767;citation_doi=10.1073/pnas.95.18.10698" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=B. R. Morton;citation_title=Selection on the codon bias of chloroplast and cyanelle genes in different plant and algal lineages.;citation_pages=449-459;citation_volume=46;citation_year=1998;citation_issue=4;citation_pmid=9541540;citation_doi=10.1007/PL00006325" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=B. R. Morton;citation_title=Selection at the Amino Acid Level Can Influence Synonymous Codon Usage: Implications for the Study of Codon Adaptation in Plastid Genes;citation_pages=347-358;citation_volume=159;citation_year=2001;citation_issue=1;citation_pmid=11560910" />
<meta name="citation_reference" content="citation_journal_title=Molecular phylogenetics and evolution;citation_journal_abbrev=Mol Phylogenet Evol;citation_author=B. R. Morton;citation_author=U. Sorhannus;citation_author=M. Fox;citation_title=Codon adaptation and synonymous substitution rate in diatom plastid genes.;citation_pages=1-9;citation_volume=24;citation_year=2002;citation_issue=1;citation_pmid=12128023" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=R. A. Morton;citation_author=B. R. Morton;citation_title=Separating the effects of mutation and selection in producing DNA skew in bacterial chromosomes.;citation_pages=369-369;citation_volume=8;citation_year=2007;citation_pmid=17935620;citation_doi=10.1186/1471-2164-8-369" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. Mrazek;citation_author=S. Karlin;citation_title=Strand compositional asymmetry in bacterial and large viral genomes;citation_pages=3720-3725;citation_volume=95;citation_year=1998;citation_issue=7;citation_pmid=9520433;citation_doi=10.1073/pnas.95.7.3720" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=A. Muto;citation_author=S. Osawa;citation_title=The guanine and cytosine content of genomic DNA and bacterial evolution.;citation_pages=166-169;citation_volume=84;citation_year=1987;citation_issue=1;citation_pmid=3467347;citation_doi=10.1073/pnas.84.1.166" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=M. Oresic;citation_author=D. Shalloway;citation_title=Specific correlations between relative synonymous codon usage and protein secondary structure.;citation_pages=31-48;citation_volume=281;citation_year=1998;citation_issue=1;citation_pmid=9680473;citation_doi=10.1006/jmbi.1998.1921" />
<meta name="citation_reference" content="citation_journal_title=Nat. Struct. Mol. Biol.;citation_author=S. Pechmann;citation_author=J. Frydman;citation_title=Evolutionary conservation of codon optimality reveals hidden signatures of cotranslational folding.;citation_pages=237-243;citation_volume=20;citation_year=2013;citation_pmid=23262490;citation_doi=10.1038/nsmb.2466" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. B. Plotkin;citation_author=J. Dushoff;citation_title=Codon bias and frequency-dependent selection on the hemagglutinin epitopes of influenza A virus.;citation_volume=100;citation_year=2003;citation_pmid=12748378;citation_doi=10.1073/pnas.1132114100" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. B. Plotkin;citation_author=G. Kudla;citation_title=Synonymous but not the same: the causes and consequences of codon bias.;citation_pages=32-42;citation_volume=12;citation_year=2011;citation_issue=1;citation_pmid=21102527;citation_doi=10.1038/nrg2899" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. B. Plotkin;citation_author=J. Dushoff;citation_author=H. B. Fraser;citation_title=Detecting selection using a single genome sequence of M. tuberculosis and P. falciparum.;citation_pages=942-945;citation_volume=428;citation_year=2004;citation_issue=6986;citation_pmid=15118727;citation_doi=10.1038/nature02458" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. B. Plotkin;citation_author=J. Dushoff;citation_author=M. M. Desai;citation_author=H. B. Fraser;citation_title=Estimating selection pressures from limited comparative data.;citation_volume=23;citation_year=2006;citation_pmid=16754640;citation_doi=10.1093/molbev/msl021" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=V. Presnyak;citation_author=N. Alhusaini;citation_author=Y. H. Chen;citation_author=S. Martin;citation_author=N. Morris;citation_title=Codon optimality is a major determinant of mRNA stability.;citation_pages=1111-1124;citation_volume=160;citation_year=2015;citation_pmid=25768907;citation_doi=10.1016/j.cell.2015.02.029" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=T. E. F. Quax;citation_author=N. J. Claassens;citation_author=D. Soll;citation_author=J. van der Oost;citation_title=Codon bias as a means to fine-tune gene expression.;citation_pages=149-161;citation_volume=59;citation_year=2015;citation_pmid=26186290;citation_doi=10.1016/j.molcel.2015.05.035" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=C. Rispe;citation_author=F. Delmotte;citation_author=R. C. H. J. van Ham;citation_author=A. Moya;citation_title=Mutational and selective pressures on codon and amino acid usage in Buchnera, endosymbiotic bacteria of aphids.;citation_volume=14;citation_year=2004;citation_pmid=14672975;citation_doi=10.1101/gr.1358104" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=E. P. C. Rocha;citation_title=Codon usage bias from tRNA’s point of view: redundancy, specialization, and efficient decoding for translation optimization.;citation_volume=14;citation_year=2004;citation_pmid=15479947;citation_doi=10.1101/gr.2896904" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=E. P. Rocha;citation_author=A. Danchin;citation_author=A. Viari;citation_title=Universal replication biases in bacteria.;citation_pages=11-16;citation_volume=32;citation_year=1999;citation_issue=1;citation_pmid=10216855;citation_doi=10.1046/j.1365-2958.1999.01334.x" />
<meta name="citation_reference" content="citation_title=Technical Report.;citation_title=Armadillo: an open source C++ linear algebra library for fast prototyping and computationally intensive experiments;citation_author=C. Sanderson;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=Z. E. Sauna;citation_author=C. Kimchi-Sarfaty;citation_title=Understanding the contribution of synonymous mutations to human disease.;citation_pages=683-691;citation_volume=12;citation_year=2011;citation_issue=10;citation_pmid=21878961;citation_doi=10.1038/nrg3051" />
<meta name="citation_reference" content="citation_title=Strategy adaptation by competing subpopulations;citation_author=D. Schlierkamp-voosen;citation_author=H. Muhlenbein;citation_pages=199-208;citation_year=1994" />
<meta name="citation_reference" content="citation_title=Codon Evolution - Mechanisms and Models;citation_title=Background;citation_author=A. Schneider;citation_author=G. M. Cannarozzi;citation_pages=3-11;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=P. Shah;citation_author=Y. Ding;citation_author=M. Niemczyk;citation_author=G. Kudla;citation_author=J. B. Plotkin;citation_title=Rate-limiting steps in yeast protein translation.;citation_pages=1589-1601;citation_volume=153;citation_year=2013;citation_pmid=23791185;citation_doi=10.1016/j.cell.2013.05.049" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Z. Q. Shao;citation_author=Y. M. Zhang;citation_author=X. Y. Feng;citation_author=B. Wang;citation_author=J. Q. Chen;citation_title=Synonymous codon ordering: a subtle but prevalent strategy of bacteria to improve translational efficiency.;citation_pages=e33547-e33547;citation_volume=7;citation_year=2012;citation_issue=3;citation_pmid=22432034;citation_doi=10.1371/journal.pone.0033547" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=P. M. Sharp;citation_author=W. H. Li;citation_title=An evolutionary perspective on synonymous codon usage in unicellular organisms.;citation_pages=28-38;citation_volume=24;citation_year=1986;citation_issue=1-2;citation_pmid=3104616;citation_doi=10.1007/BF02099948" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=P. M. Sharp;citation_author=W. H. Li;citation_title=The rate of synonymous substitution in enterobacterial genes is inversely related to codon usage bias;citation_pages=222-230;citation_volume=4;citation_year=1987;citation_issue=3;citation_pmid=3328816" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. M. Sharp;citation_author=D. C. Shields;citation_author=K. H. Wolfe;citation_author=W. H. Li;citation_title=Chromosomal location and evolutionary rate variation in enterobacterial genes;citation_pages=808-810;citation_volume=246;citation_year=1989;citation_issue=4931;citation_pmid=2683084;citation_doi=10.1126/science.2683084" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. M. Sharp;citation_author=E. Bailes;citation_author=R. J. Grocock;citation_author=J. F. Peden;citation_author=R. E. Sockett;citation_title=Variation in the strength of selected codon usage bias among bacteria;citation_pages=1141-1153;citation_volume=33;citation_year=2005;citation_issue=4;citation_pmid=15728743;citation_doi=10.1093/nar/gki242" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=D. C. Shields;citation_author=P. M. Sharp;citation_author=D. G. Higgins;citation_author=F. Wright;citation_title=&quot;Silent&quot; sites in Drosophila genes are not neutral: evidence of selection among synonymous codons;citation_pages=704-716;citation_volume=5;citation_year=1988;citation_issue=6;citation_pmid=3146682" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=U. Sorhannus;citation_author=M. Fox;citation_title=Synonymous and nonsynonymous substitution rates in diatoms: a comparison between chloroplast and nuclear genes.;citation_pages=209-212;citation_volume=48;citation_year=1999;citation_issue=2;citation_pmid=9929389;citation_doi=10.1007/PL00006459" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=N. Stoletzki;citation_author=A. Eyre-Walker;citation_title=Synonymous codon usage in Escherichia coli: selection for translational accuracy.;citation_volume=24;citation_year=2007;citation_pmid=17101719;citation_doi=10.1093/molbev/msl166" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=F. Supek;citation_author=N. Skunca;citation_author=J. Repar;citation_author=K. Vlahovicek;citation_author=T. Smuc;citation_title=Translational selection is ubiquitous in prokaryotes.;citation_pages=e1001004-e1001004;citation_volume=6;citation_year=2010;citation_issue=6;citation_pmid=20585573;citation_doi=10.1371/journal.pgen.1001004" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=H. Suzuki;citation_author=B. R. Morton;citation_title=Codon adaptation of plastid genes.;citation_pages=e0154306;citation_volume=11;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Protein Science;citation_journal_abbrev=Protein Sci.;citation_author=T. A. Thanaraj;citation_author=P. Argos;citation_title=Ribosome-mediated translational pause and protein domain organization;citation_pages=1594-1612;citation_volume=5;citation_year=1996;citation_issue=8;citation_pmid=8844849;citation_doi=10.1002/pro.5560050814" />
<meta name="citation_reference" content="citation_title=A First Course in Stochastic Processes.;citation_author=H. Tijms;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=E. R. Tillier;citation_author=R. A. Collins;citation_title=The contributions of replication orientation, gene direction, and signal sequences to base-composition asymmetries in bacterial genomes.;citation_pages=249-257;citation_volume=50;citation_year=2000;citation_issue=3;citation_pmid=10754068" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=T. Tuller;citation_author=A. Carmi;citation_author=K. Vestsigian;citation_author=S. Navon;citation_author=Y. Dorfan;citation_title=An evolutionarily conserved mechanism for controlling the efficiency of protein translation.;citation_pages=344-354;citation_volume=141;citation_year=2010;citation_issue=2;citation_pmid=20403328;citation_doi=10.1016/j.cell.2010.03.031" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=E. W. J. Wallace;citation_author=E. M. Airoldi;citation_author=D. A. Drummond;citation_title=Estimating selection on synonymous codon usage from noisy experimental data.;citation_volume=30;citation_year=2013;citation_pmid=23493257;citation_doi=10.1093/molbev/mst051" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=X. Xia;citation_title=How Optimized Is the Translational Machinery in Escherichia coli, Salmonella typhimurium and Saccharomyces cerevisiae?;citation_pages=37-44;citation_volume=149;citation_year=1998;citation_issue=1;citation_pmid=9584084" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=X. H. Xia;citation_title=Maximizing Transcription Efficiency Causes Codon Usage Bias;citation_pages=1309-1320;citation_volume=144;citation_year=1996;citation_issue=3;citation_pmid=8913770" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=Z. Yang;citation_title=Estimating the pattern of nucleotide substitution.;citation_pages=105-111;citation_volume=39;citation_year=1994;citation_issue=1;citation_pmid=8064867;citation_doi=10.1007/BF00178256" />
<meta name="citation_reference" content="citation_title=Computational Molecular Evolution.;citation_author=Z. Yang;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Z. Yang;citation_author=R. Nielsen;citation_title=Mutation-selection models of codon substitution and their use to estimate selective strengths on codon usage.;citation_volume=25;citation_year=2008;citation_pmid=18178545;citation_doi=10.1093/molbev/msm284" />
<meta name="citation_reference" content="citation_journal_title=Trends in microbiology;citation_journal_abbrev=Trends Microbiol;citation_author=Y. M. Zalucki;citation_author=I. R. Beacham;citation_author=M. P. Jennings;citation_title=Biased codon usage in signal peptides: a role in protein export.;citation_pages=146-150;citation_volume=17;citation_year=2009;citation_issue=4;citation_pmid=19307122;citation_doi=10.1016/j.tim.2009.01.005" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_author=Y. M. Zhang;citation_author=Z. Q. Shao;citation_author=L. T. Yang;citation_author=X. Q. Sun;citation_author=Y. F. Mao;citation_title=Non-random arrangement of synonymous codons in archaea coding sequences.;citation_pages=362-367;citation_volume=101;citation_year=2013;citation_pmid=23603537;citation_doi=10.1016/j.ygeno.2013.04.008" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=T. Zhou;citation_author=M. Weems;citation_author=C. O. Wilke;citation_title=Translationally optimal codons associate with structurally sensitive sites in proteins.;citation_volume=26;citation_year=2009;citation_pmid=19349643;citation_doi=10.1093/molbev/msp070" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="The Impact of Selection at the Amino Acid Level on the Usage of Synonymous Codons" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3.cover-source.jpg" />
<meta name="twitter:description" content="There are two main forces that affect usage of synonymous codons: directional mutational pressure and selection. The effectiveness of protein translation is usually considered as the main selectional factor. However, biased codon usage can also be a byproduct of a general selection at the amino acid level interacting with nucleotide replacements. To evaluate the validity and strength of such an effect, we superimposed &gt;3.5 billion unrestricted mutational processes on the selection of nonsynonymous substitutions based on the differences in physicochemical properties of the coded amino acids. Using a modified evolutionary optimization algorithm, we determined the conditions in which the effect on the relative codon usage is maximized. We found that the effect is enhanced by mutational processes generating more adenine and thymine than guanine and cytosine, as well as more purines than pyrimidines. Interestingly, this effect is observed only under an unrestricted model of nucleotide substitution, and disappears when the mutational process is time-reversible. Comparison of the simulation results with data for real protein coding sequences indicates that the impact of selection at the amino acid level on synonymous codon usage cannot be neglected. Furthermore, it can considerably interfere, especially in AT-rich genomes, with other selections on codon usage, e.g. , translational efficiency. It may also lead to difficulties in the recognition of other effects influencing codon bias, and an overestimation of protein coding sequences whose codon usage is subjected to adaptational selection." />
<meta name="og-title" property="og:title" content="The Impact of Selection at the Amino Acid Level on the Usage of Synonymous Codons" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/3/967" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="There are two main forces that affect usage of synonymous codons: directional mutational pressure and selection. The effectiveness of protein translation is usually considered as the main selectional factor. However, biased codon usage can also be a byproduct of a general selection at the amino acid level interacting with nucleotide replacements. To evaluate the validity and strength of such an effect, we superimposed &gt;3.5 billion unrestricted mutational processes on the selection of nonsynonymous substitutions based on the differences in physicochemical properties of the coded amino acids. Using a modified evolutionary optimization algorithm, we determined the conditions in which the effect on the relative codon usage is maximized. We found that the effect is enhanced by mutational processes generating more adenine and thymine than guanine and cytosine, as well as more purines than pyrimidines. Interestingly, this effect is observed only under an unrestricted model of nucleotide substitution, and disappears when the mutational process is time-reversible. Comparison of the simulation results with data for real protein coding sequences indicates that the impact of selection at the amino acid level on synonymous codon usage cannot be neglected. Furthermore, it can considerably interfere, especially in AT-rich genomes, with other selections on codon usage, e.g. , translational efficiency. It may also lead to difficulties in the recognition of other effects influencing codon bias, and an overestimation of protein coding sequences whose codon usage is subjected to adaptational selection." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3.cover-source.jpg" />
    <title>The Impact of Selection at the Amino Acid Level on the Usage of Synonymous Codons | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"2roaSFW7okjBTLn8kNXA_AuRW3NKjnx-LPC7hJblWJ0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"483527","apath":"\/ggg\/7\/3\/967.atom","pisa":"ggg;7\/3\/967","ac":{"\/ggg\/7\/3\/967.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/3\/967.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/3\/967"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/3\/967"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/3\/967":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00227+KeqfQpcE8B9j9Ux+NBX64wA+XMaH\\\\\\\/kpOVxy4TwC16J+sNjvPIjGH++5H3pbtPz0GyKH70IY6Zryk6M3edvMUrWWyzzl8VFcv+86J\\\\\\\/zif4Hx+Lge0cI8zQHQyGmqCUp4Hj0uWwCmD\\\\\\\/43yrLJbo4Fo\\\\\\\/E56v5mKLgBmV3toI0jeMR1++m+NjvKYmLm\\\\\\\/AGFL7PRhR1iv0fTUqsfTXAqVFeyTTb+U1zrHA2wfoQkkZVMPEEbNRqIbFGdySjvhWrKHNR1HTqa\\\\\\\/gqig5rqG3\\\\\\\/0b9JkQyToo7aWTpoG8y+Jas2lLPHkM2uMCc53c3rCUwmCEp5u5\\\\\\\/A3fpajenb6Y2a8OxIpx0sRx3rQ\\\\\\\/UG5ZoWPOotQTYUQ8tvjVgm\\\\\\\/oHDCY7MhzNzo0V\\\\\\\/LWB8xy7Whjo8\\\\\\\/fy8w5vsAAT\\\\\\\/y9VR7r0700sygyUSywR56mt9Lf7gCpH8j8dm+dHczL11joLFlJ\\\\\\\/iMmHKT11B5uBp2Mm\\\\\\\/JVcw6hcXQLas1922qtX2oDVzVni9\\\\\\\/BPzzGZhUK3tBQMEcTAhkIgnnlE2JaUeG+Bl6hLRKQL1YRyRozarEnDdRYViw6y7xSoW1HAVonzb\\\\\\\/IKAjQyoO1TcTWCgUp7O8fBqLRIOp7weiDtVcUEXs0AB1YcA3gs8jLkoWgUlKVsBJ7KKCJ1yoPzqzVtthHgZajP\\\\\\\/1K41Plj7aJO8FUX9iPSgqQuveOONaeIE7hah8f6gDBSgGdO3ZBojQOIn3P5NCpQkS3oUIe7Uu0hFCE0MqhUINfXEd7dm43Vw6DVKRM2YdL83ofYXRUf9nPFNTMesbjbfvaOTSvxwxfMEOYWsuEKi0kTs6xEBG0tYaXnqXVy9wgB9FmtWOp\\\\\\\/wdgEDv3Ilq\\\\\\\/vAY\\\\\\\/QQGB0Gr1795Tc+jVfjOfD7QH8WUFZp4uQGn43sfH99uYFYGqxZGyGpXqmzbrPCc5EX1BRDzqghzK23lAXzLN+1U14aSfV5x4dIQvdf\\\\\\\/fGmk\\\\\\\/g6WK4qjCwXEGnl1YswzIPtTdyeNg3L9vAdMHJDGGRx+o\\\\\\\/\\\\\\\/d2rIK\\\\\\\/FMjmLIkuLzbSvJABh5o9pIPqZ5OSgPBAugTAASeUH3hoqkA\\\\\\\/P16WcHZHyC+bbLg9owSwP0yE7qCb6Yknlu+M0Fa6NB6+PGtwK6eP7v3+2mbW+LHXsh\\\\\\\/HAORXTcUeC6n3vaLuEmLGHdYFzPD0BiBetVkx8F5sS2dHzodzVKVo4G3qAgdRiiLR1\\\\\\\/yGId8sdmYrYSEedT13eOXntVnZ7pjtZ2+KKcYeKGIbbYni2eOsEaGVKgkgylhhUoxkLFAVOX\\\u0022,\\\u0022iv\\\u0022:\\\u0022Rb7y24zSKoZ+aVuV49EoHQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002272849d466a6bd584d41ff74b3314dae0bac2e425a67ee316a3d53791272ed250\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00221g1uNGdnXFRRAgzNSRc5ES3PgTwZGTytLrlkmqBWI8Vf+hDbhta9frXnGEAAeGSXwPNjRCKwswOtyJZkIQmiespGHo0PDRAtxxAhSHgRzP0TNgx0dXB1Yb5n+Be11P+4K\\\\\\\/Z1A4494ev0j5xpT4Syo0QosVJFCT01nSCn+ELlr+DBSuaqiPclPS8TniwD67P6zp\\\\\\\/WzcYBQaVLGfr9HfBunNZ04iXR64YWr3mqng9029PxPWRF\\\\\\\/m8e4SbPamBYmtkPFImFGbIHSfoa1JZvx4qslzJ8nQXLgZwmbyGzlurb4RmJRnDpqOCb2AEmz79O8UDCQOZw9GBL7Vmq5zCt5ceIVQA3xSAa36ODpjyJSdPx8AtagejdSb+n\\\\\\\/uSUDJxni2Xp063PZGPBu8ZRF+O6AKqBHhMyP5PAkl8y2T2V7RYF\\\\\\\/hdBr99XmE1NpHoCEM5QBu7MahdR4RVGd1YLwpKmRr1zAUIwGOFl26nz8Y\\\\\\\/qt+fKAgy0lxAHKtXKf0hFKrIq3i23k85wE9rDgwLl773rPFyYHeLf54XvOYFUjYqRG\\\\\\\/jFytpmV5urk1ur8K3QAbzOcjdqcs63rb4+oC2PPGKWSP\\\\\\\/ym6roF86\\\\\\\/IwFkliySLKvAInpmkrlyB5Aj8levGoVsPB\\\\\\\/qLwLNignf+RoMreZcwdrKGZsRsJXB9xVN5kAvtW\\\\\\\/BoqaaVlUiUMGODXeVT6NIsFr2+MVKWM1iBNqQw5pykEZ4AD0z1WE1YcLmdGa+xO\\\\\\\/dsILLHUB6xnO21kQGAPciu4njxxTTpNdoB9c\\\\\\\/uhn0dM6q8XZfUKS9lxlQrG8\\\\\\\/+r3Glnq54mDSQmQVu+wolrTpJrXPWq7Ws85LT0MtoIhMJaafoa0MT5q6YS23W+oTDdUAkkPb\\\\\\\/7C6UXo0+TPKBT739xeAM3VFNJpvXFFeRn0eEbJr1PVDMsz1nxEHpgenBfqkLH8N+sPLQdI+fgwcItlNQgVR7hpulz1J5YsEwXrNKtN0zrn61745M5EaG\\\\\\\/tMLctOcyRwGkk\\\\\\\/09jmoYGAibvEf1bDYcEFcDLz6fuxgxR77\\\\\\\/YP38ZdyzU49vSRvs5acuXv6bGxEG+8AOcEWKd45+4aozSK7VBkYEaEmRkdJEiFF7bBczfSsrn1vjyKQFuMXOz2MOrpwER5i2gFvjwUO0Mkgij\\\\\\\/g6m7WAcxZlI\\\\\\\/oQtNYb7AL2uVHH4dbhsVC6d3eTSytkQS8vhr5ZjwrG28i\\\\\\\/S0iBPQrkQRPAzrD8ADGv6oAAEdtZYEku8J2FF0DnMaZfR\\\\\\\/aEtPUqWJ2TppEj\\\\\\\/xA3z2DjZyUwkldaUSWwUwedSTtsDQH2BOGaxBToP68jVN3P6EyySpnHYnsMsvwR\\\\\\\/Wv70ijJkEgJpQ3v8PABRfUksqTCvG\\\\\\\/R7J5ubkoEVQsFgjTg2KVlaVGrccxkEvDJXThtlbyumCEir8YDUuuWM0uyw+FR1+o8DJFrsrqpLrHSllr9QKOLH7rpkWcF7em1rJCuuBe0lCjKtCl\\\\\\\/rVJDWMw5RZdc6L41gtv+hnlWdS6yUYZCIt0asYkHWr4pPSDwhV2EFcW1gLknReAVPodUierxYAW3H4HR4mjSlcjcF9tbADJH07HCH4fsfcmMFhK1nn7bchodOuP\\\\\\\/UK4t7s0js9KsGOBEWyKpxvM0Txs765bVDdal4ekLjZlLo2HouuIKU\\\\\\\/qvYIQxfDOyCiaZvuJp5dOJIyV+64aEazxSq1o0IMawdlF9rNb\\\\\\\/g8qWf2vT7E8z1d7gmZF5OCZIZ2Y1sYKF745YZ1d+zo2uwtrjA\\\\\\\/\\\\\\\/n+UU\\\\\\\/oq2tfF\\\\\\\/4mRWI+lhXB+m\\\\\\\/GTh01+UibK9\\\\\\\/zRLQJlwMo1BdgkQrRwdPuV8g0VRbNiJC56mGJn1tX86rtuLK8hs5xmZCbE3Q3YMVWCirXTBbTtXG7NAxMseeduvn+LmfLo61yIWmhD2+tGt+9je0kMX3\\\\\\\/yhH5pyb1M2vYnjIHBST4VZ56EIMQLElAn6hq8RmYizBuGg7AsBRsaI4Gs7jY+gkU1cN0835yP0X4N3cttjYMqsO8mRq8pq\\\\\\\/SW+skhAQ95MtynTcYGp0TM2CA0+TxBkhDRUDrosFBhK2qrfzVnTpzn0zsMWdBVr+n2YUYu\\\\\\\/FGIGd5+tm\\\\\\\/iAexY6ytKZlmK1iEI1EnXsv3MPHiVl3hYPeJlf8090NYcW\\\\\\\/ui2eqdM2Lti1MauJmZUY8Pnrw4oqtCrFHH5gzQQUF\\\\\\\/VEe8+Bd8l++p3eBU3h2J68th9MDHbpJQYV+sOEAMLeITT8ua6Rc+Rf\\\\\\\/pTjLZLDSxwxZWG3w1YgLi9nIeKqydjP8enXMTnOqX\\\\\\\/VzpDh2vjc9u0i3hs0JzcEcmAtClqWKHFG596P8sF\\\\\\\/kvdxSjwyXkJGXX894Rjk3i2t0fZ6qKtfTUb9lzjh8RGVapzkMMgnPQgnttlCrQk7TMQFDcYFReViT0gnHfIAdDZXO7G+SU1MGbX5kYdOTy35YILZJPdaT5dzcRadwLsQYprhfDaYFAv\\\\\\\/NYT\\\\\\\/EftVegXdo1FEomQ3Oi5D8DC4gVILMwywKlb37q3IgJ3I4\\\\\\\/UcK4shSAGuZl7YLoxu\\\\\\\/W7w19pQfgkPx66XzDqqpN6Sl+iaraahlMU\\\\\\\/R3Y88fu+3\\\\\\\/ytLP3W4FMqDSLiRIFo+zgZZEbd22l7p15bsC5uJ+KUx6ISsk9Yh+R2ItPblT68C5TCTyRrzYFARGvBdD1aLhMaKpHJ2AHOULNVUDALGf07wGXhixW0luJHxufrX7wqVgGE5XXAJheEIM\\\\\\\/Lv0zw07vk79psVeYZQRvluDskGsCZjueMhOw0Y2rsZXfHUialqlOIAJOh3XBhCO8JcfUlbAoByffbiVZOl5ulInQWY9mAxpXrxB5RkO\\\\\\\/PgVXn3zkNjwXzplzyaKqemRPXGS+wZQyi+Vyz8HWbI\\\\\\\/W2lrBAXWmllEjJGZ6gutAuJ1BGGvlhDbhyCSVv9xPTg5ov4TBaius4MH\\\\\\\/3mBQNxEur1NmNSbLlhkiu4oI8BOoy15MqgOdjs7K1qtriX2BOeURH5uW\\\\\\\/ZZXYQC2+Fqzl9qGRkRJuDK1LcajQgRZ9LJuS1f31fJUmPmHLSBZ4lpjcrBHzFjTIO2Y\\\\\\\/Q\\\\\\\/O9O\\\\\\\/ScQkMISpnFH2HmMwZEkW4Zm6vhVAS2nx52N3IlFtJAkv\\\\\\\/wBLWQ41D5UyyHlqtcgNYKyEP\\\\\\\/FLym+dFu+ps\\\\\\\/H48JmhbS\\\\\\\/zz0uxxV91C16ASa1z+ekjGNISm6kbPGYXk9xQ0Yz\\\\\\\/o9qi2egYozI0lVkFOEUxWos2BEpmnUKCjBcJP0+mAeES40KHGuMBsJIh1II0p2Kzmkw4qv8fkZ7cSJcDQcCJn7RurN+Uf40JJcGr4Q6vOmm\\\\\\\/eUul\\\\\\\/45taYj+Zb36kGtQdYdMWr7zuiS\\\\\\\/jfFYzDqA7dB2XyxL8WM4oTbVYfoceBugOe6Yud3alepQXZiTI3FNP8D4np08My+9jAu2Ed8EnNn199zcMIx+AoVNeDNbo6RIEkAd9m76r38o4qZp48f\\\\\\\/h5+uxP9+jGXsEdIzXm\\\\\\\/YQy7hoGWKgV0y2EBc3ESwUuhfh\\\\\\\/ujJ7FL9Rt+sUY7h\\\\\\\/XrNhyX3c43tL9dydbEkKSIazqrK6f\\\\\\\/WVbarrlOLoVkYoZe3I9jiHdpEMxf6eFuEhTgI2R39dw6POvTQfYbaa4OW1JHty9tKJ8ji7MjOMwlzD0goGbkSK\\\\\\\/MSl3+ioXsXrMYBjJ\\\\\\\/iNgT9qGXP0MXOOA+27kmKfdsU0dMuuHIouo1pWHoqB3jrAfANz33cdZ82ZaWlVf2iQP1fVkhivos2v8V0ne9fUzSrm5wsv8Coms1qFBf4BJ7I5I3piW2p4ne0GYy\\\\\\\/h9Tijvq+sS9XpJY76Zn3+QXibcnIGh4WdcOvzI1L5aH5abkHzI22DFuLFZOeMXQOJf4cMZsCdfLxttSC6\\\\\\\/09Pd5ARcESZExA8KHx9D4m0z2EdiXbvETbQANB7yv4bgwIxXEttuNxBZxO4HlZmTxdWIwpfVXGtDgp+18flkACXnnpTecCBBErMZmSZ1yIHmCQTyIfKvMxIn3HwNUTXI8ZcmLQ5PnUkYaufnK0ECSMa\\\\\\\/1Mgw0DrzJnelWBfZEyVD\\\\\\\/ykSmHkQGOAERmNPW8JKht5Z9+14KuAL6uCZgcLJFLUxxgHdxs4XVBeBD26lRMnc4diBd+dEpqakG54dF18sMt32iM9grAgDV9QzRso3dGv3Wz7wDF2IEZtsTJQCeDaIej63gd7xFuJALrCH+EmYJdo4CjbrYX\\\\\\\/gOxqN2joqEaetmIAROanP5fhwEWYzPn9jABuTl1uNNWl5ADy7ydjs\\\\\\\/NHKJeQsrXwwoUSqfKkptd3R\\\\\\\/Q5QTzDvDmyBkLWEujXq9j20yazH3omvl7cmNRzsww6XhhaDyo6IOoH1a55s3Nq1i9n9rpmUcNk1qXtZWEk00muOKZThKIKS8yoyz+ppyAn2orb0P7MHJu5A0RnX6YCSkj5ziFzRu4mBrVy7CovjD\\\\\\\/QdATaxUTOoJkCAl7Q9nf060dAnspMsIRd6mMtL\\\\\\\/th9gSoW86++Qr8nkXu7pfrffQ0pkrUppMJ0RIvqOZ6AOiabyKjggoN3GS33FVFcebLxZz0cPcIHVNjkYBN2GtE5ESNWMfvsesBC6xQMd9W1u6U8tR0Njg259OAq6ucD75zOczjQTXgN8Qv\\\\\\\/1WNQqFfHv7ztY6qgERSA9pBt5qLCBhR3XGt5F4e7tDatoTNLZ9VSwtMv1J31YRUvs6JN9O3qOyVSM9hJXD2p5IPaErffILpMe9MEL60Wt4rSuaBpD+b4siYQshQtrkhMpt9drsG1mIRvmAlxF0skrV5x5M1b5ysoikHGxGAbnEa1ijyCup6v0jKY2Qh2xPVIlzpSF1X9kJLTOyQ0WG\\\\\\\/seWXEWIDbGMa1cii5V8iVFgeHPNgQPnUMFVwyE1Xmlim7l0HDttEuFIOU8\\\\\\\/eIGtJws5bvXaI0C0WOP3VxXlH4dOpVgHxg5T66Fzg16KmwVSF+KRAc4SwaKPcJJua\\\\\\\/AI5HxyxpHC+lTAjdCvQc9bF9\\\\\\\/JK8zMTsDiP\\\\\\\/qmettlvVJDII9Sdm6LiormBtiA6pnqfKa6LaYnjIOjNhoXHrC1Pk3DjGu1r0C+RYXjn7Hm\\\\\\\/MPNQ26T0NgNcQpR5asCEqRZGPoqTDcoErKQn4hug2DRJZtIo2H7Kph5ZNysjInrTQrRt438\\\\\\\/4J\\\\\\\/tbyi+nriJ7WSmyx3RIFDCn3xrG0Xfz75dvRKhA2L8RtAc9qznZh\\\\\\\/WdQbTF3oZIahOBTZZg6J+KktuygTgYUXvTSP4lcjzVEPDDSJWP2nNaNtMiQ7382\\\\\\\/UKey20LF2QabZBMynxwm2Z51JHSW41VpExT+WEz8eTr66BfUhow5Bm0ob3iSS4R4SJrnsSo8Le07PrWfnS3POpVt0qjFLkJyM8WDTyT55TdUkrcEPViIZQHX0q3EdEpwWcdQUr647v6pmm9tstC9XKJwsiPM5Gme442Ovc3CAHgqX3khDHjH+zObIRI+0FAkb+z4IVilGuvKhTrJECT9BaoMegPmTFP3M\\\\\\\/u2jZFgYeOXsbhmhLXw2UN0sv+9jX+b\\\\\\\/9RLB5IGW126ySY9ewN+MZXKdUe5+PLVGgbrhZj\\\\\\\/OviW\\\\\\\/A9iCYe9m9QK372arGA8UgYE7l8xzqreNB8\\\\\\\/fpHe3xSOnSj5h1sg4SOv+GpV90BvXw5OCOF7HB2yvUNerMdnxDFVnCrMa0\\\\\\\/D70A\\\\\\\/KXbssltmRZVmF90agAM8jxQxR7uRlu9S\\\\\\\/w79MaiYiuRfk13dPyWSpQRErd6\\\\\\\/mjB80KLoeD1XEY+LCueBPDVqYiG67dFwqmCMNBmu6vcLzcn6uTxv6+kmo0fPrTRiBjwpx7hYaxEuXFZVX3dmZWLqYySP5ilwdQGzd6WBdzhdgmPR+YoxSh\\\\\\\/vmp8C\\\\\\\/fMlAK57PGpAV1LMoqXuBSxE1qB2xi3Vro441Ao\\\\\\\/xVwXRzp7VP77nAWrJ9AOvChCL0Rxdk\\\\\\\/hp3rBxc1eGI846EBAyfroAq6qdj2e1NGW+g6bk0BRrJLw2sn8zv83TPka8GJlATbhCr1Xjv3u8ry09HEVNV6Iabj\\\\\\\/g6MkEMUIjc9LIK3TjQn+ZpP1KCmFE7+HbYVHy11QAjS1eQMp9qgMSLl0qT6NJGNtLSPA7ajbOnTGLhfOT9yX5xSmY1RgF4B5ALfhFtECPrDDBoWR1MTWIQ9aMtKyiNSjRrXXZOljFgo4\\\\\\\/plQgR4z\\\\\\\/OxHtzBk4muWhPk8\\\\\\\/h7\\\\\\\/rjiKI4xCQUPUEUf5biXVkVJbP7TyokoQvqiBGD04EFNDU\\\\\\\/o4YuJEFHqXHbbMBxcbkEFU6bh+8HIR8WHSRSWLMDjO+K27MRCD+h0q5+sNGdzBm3pAONkImOFBjthEZ30\\\\\\\/ARa3nAxZGiMhhDxd0VI1TCARl\\\\\\\/lDWkZGTxnubeu52yg+2zqQ9Pibbjw\\\\\\\/8MFr6HScbuoE3ks85KBJ0J3\\\\\\\/Ve2Tqn6K9sREyOUr2aRJKlWO6kDLhySQNAKUO5SKkcJDE3aPTzw6OG075xDF1Ll3kx7sZG5tgY0kV8Q+97\\\\\\\/fhWmItayMrwM6zov+PXfdDFJtZnSPoNAA+3nXNg4aN2GCaPUMlIMXcqrKNqBhQQfsBAUF3JwqnBX+7ccR34wAu7esELrAJg81zue\\\\\\\/Hqg3StatlVkVdyGZd\\\\\\\/dfeoSf8tg2akLO4Bt1bRCxWCCH5yO3PoavXleqIBdQLiIyalfrp7Gafl2SS8daWN15FIkohGenf\\\\\\\/zeyzoMVMq5bRP6zENCagUyx+I7QFcxsp7CZriSCTuTznycE6XQJakYL7g5+px0No3xVi7O3RZ7t8E7ZGH37\\\\\\\/uMMMrIOubzqa19ZO8d9JsuO1NLHZDR\\\\\\\/9Rz4DTw82yYkWQrj+kbtrLQGyye9sTdZp8t9GHePUF1zOAR7k8bcbqF9RPYjoU36LxiEj2W5koMxSS+TC3IRbqXJSoJvwNPaoxRJvHmxCPd9lIaU+Q6SVxoEMYbzDWcUSfMWXe6gIzAtAU6wEtumGtcpH\\\\\\\/6AcuNHAqXKG6d7RVobN\\\\\\\/QbASAkvak\\\\\\\/tZla164xhZ4M00GCfww9YeZHospCHCBrrauwnYRcRoQ0+5Gkh1LGbdCE3J7YwjX0vMP89GNYk4PiZ0Xd0xGsmfI51x\\\\\\\/RbHX3XhBufdXXba6bveXi\\\\\\\/DZn6gOyRdpSF5UWIc0ypLrf5hIGzBiD+6KyZu2J\\\\\\\/\\\\\\\/KGDre0hwSaGebY2jGmyw4RzKNcONwCcvJl+JFGeMbLmZ2U4SymDEuzNidOxv7daoNq6ZZ\\\\\\\/iACYPL3Zr6RXFIkPatn6JTtsQ4LLhx85ToW1gmOzvj1U9dF5DWh5t6l6VfIdmwRzf3J5WZVXkKZXk0VxYzQPwcL8Nitiaq2gobLGzQjaoE2eh9HoNx8kLRCFBolAvq76h\\\\\\\/MnMO2U5RIk5SSWawaapUtl2T1YP6bMzCg7UUtD9PqW5PlUfUcoYcLsNqxQx9c5ZkyELvRbXzBvzvHGdETzGDhcnt9Iw8M8ckXwgX8HgEn5sTB9epBt4ZOy\\\\\\\/bYpJdBarU8eZMgC0NeWccQXCBD9KyHU2S8s50WvFe\\\\\\\/Z2MkYiQH7GUE6DDq9ZxYNsA0TtxICNfgbUC13hA3RCuiz9noDTxpuVOZcmA6NEROAhulpSapdd6mt7bNGVrX5LJ0Wbkl8CM4gjtqk1GnqKaSeSjqlWFwyqkNyFE+Rl38SqBHYpTFdFDXnsB7QLauvljPm5q7gyfaw6eQ\\\\\\\/qcU7tB0LqhtSrhXyvopjueyXV6rBijCLr+KlgHvl+Rjo7aA4G1JZCWYlDGwWQYueq1W+pQ5obYiLpTS4jE4kZXsGYLwcGMNuXxoKXrS\\\\\\\/ncsUOA\\\\\\\/2EL8XUjmRBDx10Katy0VwJVNemVySlPqvrYMZHTebqvmBMGwrCs7GqTEVBvkZQV28qB\\\\\\\/hSHw1qoGWpRK015VwFKgbNa5RMkD3\\\\\\\/+Hrjq3ptNCmBhsJVmGfk6UIIeOmM7IQz5lsrNpZLC5cjzMZDkeavnCj1HGrcQtFuTFNU+S\\\\\\\/S9LANSSBcKhbD682o5G6VXhI8dXBpH8SahhoD31prcdEZ0a853iNuZCFt46K+jPevhnTBA6FbC8nAjYJDu1kuLQCD3wRgPLN9eup7wx3KL1x320PIX6VmCIVYKeqaASR3H+hAPN3CWnjmiTVgcD+3aSrMRr5XGB1KecBLcindBbZUi\\\\\\\/0KaCIJL450vfyhfLzkTwNyw1\\\\\\\/Q7Xkpo3Ij\\\\\\\/vbZkkSi3Wede\\\\\\\/zHtVoZOlbIZzAMsn0Ez\\\\\\\/PxXf7DDUXgAER4Zn2acpZLuez08TvVK6Y+kj2aoXZYQgfkUc4aci8Hux5XlCLN6MPePoCwcpLlP81m6sYd6azNn7M4oA6xw92vQvMfqT9vRcuaOQBkgXNVp6z+Q8oadPKRG5CrfDFTskQSnNjmol\\\\\\\/1WJp1c9WrOHei9J0BGhJM\\\\\\\/F5UwTlaEo8wrNlTvC3gif5m+IGTpCUinScjCACpTTzrVkUTrTZcSaqatH1kIGiue+\\\\\\\/kmnpwVDyo1OdVxvleFHQ9l8p9FzuQK6kxzzlAs\\\\\\\/nqBo1AZyANszoNfxwe0QXEyCvGJH5Gt4glM+DoHnRW3ypLr1Pakxf1j6EoPEZpyKLAEqXJhgoLfXDQjqawmXaT34yFmfr1+QTM\\\\\\\/Q14eHvaEMiG4LRfVX8gXECZtgQka+uoOhcSbULoESXZP1mB3aDC7N867AUWbpW6QIJ863EiOuCm3\\\\\\\/r0LJGNx4eyUAq3DtSyD1r3zYFPOzKkty07haViP\\\\\\\/HPC8Nfn5Oz0iHAtdNaHlAwsuoJkDh3seMzclxWG7uNjB6rFrWqUgXm5gLNG5j8ExPgv5djwDVtzaMlhWcRuMwtWMbf\\\\\\\/+kj9u2mui5BqZNfi8Nu1P6qvByZKCr+oBOsHqFkh9R9ToFUhKWMAGvE0L3lFTBj\\\\\\\/kfkqyKKmKYxuONwYlbpikkWhJx9qFXZDgpGO45Yg2Dn4UqAC9ux5HaqMS\\\\\\\/XkA7qYwG04KCSYh\\\\\\\/dWsrEX7rMqOFS72NZNcf92iW5IYnuZ\\\\\\\/tMXOfDV40FoCeK4Z9eiqU0Sp37xIaqvD2M6vDdgEfOxUHUfCSppxu+r1959kWFkbrSyD2xJT+kW2smAkcJCaDPYI4w4k1hhrff3y4FuZCxqgoI94KUdRu1IGrpKwnXauF5I7Mt\\\\\\\/6UJ3AamYh30hNR55Q7Ek+tyBvxxfNT1qnk7k3qrNy9kgHXnPqNzGK+Bn0YlBOZjM7VOk3TDlQZ2H20SGqffbqkMYqM9Uxt8P3srLfddU3HSVDLwvvbgkVNDnDHXpVurj0MWU1RMOdEFTKe2GoKkzumfV1BffALOftLp601zr8OVh4vdhsovB+Ev8XqZiNjjHw+0LbXi2VIEPnz6fZYNWRc+tmbBzSFAUX8BEyV4YkDHBR37VZg4pWXihWpfOJ3pz9qu1tSH8IxmOXDizjKwx2LHqldxSaIT7DmITbAUtw1k81+76e0KKXHkUfzaVNcswkO65mFDRvwWSkEvv4i1By42gvP+8dIVdNUX7EofPod0Gs9BiLcWH+Gnqrhh9IoiWjGcDFQ6whH0S7YGFErQDe4UPWXbffq6jZOq32SEPY\\\\\\\/Voj2xyFocIkDqIGDZQryoWShfOQ7olS6tWZ\\\\\\\/qcizKmCcMdKbqn8L88hy4s7d0r5qWW3v6ZhUZDqniNTMxSotPdojlW8+MGLVSjQEvJ55VZl0Q9C17CIyEpGyLj+k2qqMLcpTgqpBH1OIw+G6DjTwm5dODaZdnTUOZNYJZk+mpyowQqK7c\\\\\\\/lmC0opAkEFAOH9aNqeHsyftnWbGxbMneGDIo7DHE3F4sH15vOIRFfiCeY2Tp5mJQeoksJeGaBFj9xmceZfXU7V\\\\\\\/OiyZud3jWVVlPokrazGYyKDg3JqGkyErm3B43SSbS8Pp6+fQOk4dUi9S49AbN8gXsAjc+Uo7pVRv\\\\\\\/1W\\\\\\\/m8JgynFp+iaM+BLUbMXL6On3ipzbjUzHDS1WxSAwCwdhHffmVfIQmv9DkdCw9VWL1GlCJY4\\\\\\\/jUkqem4nX+tAFhfZ0KUQKKPkNZEz\\\\\\\/24dS340fuBbrMiS0OZJpnnqnbcYXjreuzrGdVGr2qHnPSihrZeUWBOLuDocAgCPvzQxcm6ncZQ76LdlHrv85ZDAsXu2H1r+qWT\\\\\\\/ioMN6j8kxVfLjzU1kHsZnpcPhNTUxVSd6bG50jYc9+sF6rUnJXlAVxh35jUKk\\\\\\\/M5DR1zctqBBXgdfeRyb9+q6Pr6Tb5BRF5QKyHQ48+QlMKpSwqTWHCZtuByBQIgZMs0btVGxM6uWFkNxKWNglIOkBV9aMp\\\\\\\/+tqh7e0dMNSnjzCgJ6z3zKDHfHptMRBgmwGlBfttJG62CsuNMQ41Hx51pceWGLWD8l82PjpuMYnUCZozNa3UNjiuv7ukUkUpdYbdfGsRVkNn18JCdJEalzNQabKbfMiLGPch6ex7XuwniYmwHXRAIOTwdV1lrah1sp9jwZNnNM3eUcGmDvhKQKElR1IB18ZwLOOg5IiHATwqRnPkepXsELFn+dhj5u0AgV0ys7+v0DuanUH6uPUm+VLtqpL0NB5hDMeUB\\\\\\\/4jLuDUux9Sgv2UnMbkZ97MxvqZxlljv21NfmLKiAKIH7qjW+u3oNLzA3uBgHiKPLl+alvm6uVtPW9PmMUTNQgFCx3u2tiVqr042ziOmoY8PzXZwQwmWZ3DGRq99MvNcyVwQBrQIt0xbOQcQ6uM1jrtwzrgS+vfM\\\\\\\/awUxcwaDVe0L0AWPYR1hdqyMXQp+7\\\\\\\/GBXNhm+BLQJf9cIzsT3Cg4zZaNDbS\\\\\\\/6FiiTfmm+UsVkHCoKJoGPnWaPi\\\\\\\/vbQzdm9QbhV9T2GpiyA2w8LrDomaKKFWU4QaSdbDAM+pRXnKxBX+sYa6xTZCSsGq2hb2DmKemDurv9tCAkf6GtWGcrwyM\\\\\\\/4Xs8HspnisrbUbsWgKM8rBTUYV66QudcjDguMbr95CZoM1aTdT0FIC6KASZMciYy+e4LZw\\\\\\\/qgZD0XjLy3r8oBcKtS92WsPIY3dIbq\\\\\\\/3wQXIT5A924t\\\\\\\/KXa\\\\\\\/\\\\\\\/w1c\\\\\\\/jrgINiv640LRWW9skRlReN5WtXNIKDIem+Z5vKuHuUtpuIaOt2GcOqBBbGgdn55I8xem\\\\\\\/DXMRqCytYa9vt8hvj0hEHSrjBkDP8u3vzv7ur2yp0jpUuSjRFRp+A5K4GvLdc5MgXPZwq6aAqpmTd9weYOXpDWvvOuQSS9X9Y5dffj99vbgRCey1tQgjzEMTgVubnKePKHNgShTaJQJos024ks1PkoGcydl\\\\\\\/K4O+QAs\\\\\\\/i1vocRyfFsW2NHKCKmbGiRyXsE7DeZ\\\\\\\/s4d4dsSXTrSThmzLyQRFb7FaJGBsijqcMyBBPgX+vGuAUrEiDctmkt29VqkZVuQCEWPysWyuyCqJ8Iw5BBOi1bV4tluM+r0YWsfagU9LDQQlJXPxZ+bsQV2hn+unhAkHC\\\\\\\/+\\\\\\\/vI9W9Kh0DFAjVh7IYD16wjDXLi0eNZbqKkksTX19kDS3Yw7gg5J9vWgz+VcMKL2qN2tjxLnB1TI4SeDdvVXyQ2fV07dbKUAuBQOHkGd5vfakdshQDUz14KkYLbfyQkBqHYE9SM+\\\\\\\/lXHYFQuWH3OX+3InXOp48zWzVVmdn4xigZkwMDR\\\\\\\/HPFwdY9OHhShjUOHy2mROSGa7LWteX4hSzr\\\\\\\/mBVAzVRtj6meOH0yJVKoMtBMC0jLcCMIw\\\\\\\/XpmoAJ5ZZTvCoFWnmTvDe6+RWjvrkKDKSJBz4g0cLRck\\\\\\\/2QtEvu8cV2es1FEJ+AZdXayPPtquV2Pwf2W68Nz6mDpUlcU4lWeWLZuna28wpK1sVjr6cWhh6RI\\\\\\\/psc8PReHTjX\\\\\\\/ErJ9KN\\\\\\\/\\\\\\\/ZbKAYemq+3Y\\\\\\\/ryhlMNENFR\\\\\\\/T0ki9O2osurtWieTvYu7SI77RiXO\\\\\\\/+keakLuApm80UPN4OLkdZg\\\\\\\/6KYNTvYJE9fVDPCw89x7kt+XFZR6kpH8BnzF6q1FwgzolJKd2CVS93f0YUb2xRo5dELlsWAbERUl10JojqXrS1uTtmqIdMiUqcALiGdVWOEDtqWByfMOOLhAQIyl1P3ggHq3\\\\\\\/WkRcK2JOZ28loJCySvABqeONhho5GQQmD4cZbeeXLTpmPlxEzFqKdiTmsYL9whVFnvrSB9SirPb5cACjVlCkao1k6NlDAZVdcm4lGe0D960+A1YpZEctjCIjHbgM24clc+XGWAz8skipEkDjiuDzlRSnEm9TeD7ymfuSmhc1h3q3DPLL1tYyX27j8MHTKBgwELl4gC3cWaC\\\\\\\/8AXe0lrmjWJFmS37BL2ZzrSkCzTtHjnatrXrcsBTPdPK7X\\\\\\\/lM82yKe2cVX\\\\\\\/lvtUks7d\\\\\\\/YKNm4UGARLnmcRNAq\\\\\\\/VPbZxy8M0wJtyx0RvhEc0wCV2AA0HDvrWzu\\\\\\\/\\\\\\\/jRs7xt7Otyoxa\\\\\\\/OT0FMnNuqQGyt8RU7Sp0NJZVQy9dlBSRs5l2Jod9BSlFKpqniupvD\\\\\\\/QySbJ9sobm6XmB4JaiVucAD65ifBADHJY6i0rPKqKrwFxQEqH3MAF+900zvotRjjH7Q\\\\\\\/1xhD9T0CR675RXol7bsh11YcYnOyv3S99DM3YaeMlyC7zCgekOYErkXKSiMO8PETSnuWECD\\\\\\\/e4rQbqqYj9fmAZXum1APl9zUX421PtPHMzo2I4yMuUo6vZqNJvHWGRXk+E+Ta4Th16qBLz8lsFY7hC\\\\\\\/+rY9yzSXd9F2JemH88ranDmd2E1UWSkK4QhUiOgnwBneE\\\\\\\/cEAnqlCJJgp3XMj7ez8\\\\\\\/O6j+CexSTqx6zW1YbFXC8sLb8RHdNuzD8j8YceolL62zqYw\\\\\\\/sFg19v0zcR6Cp4HcpHK1NzJpeNon9ZZvq\\\\\\\/GZUkrhx52VmD+u\\\\\\\/o6dU7zHALIN+cXOcEk4yMxDh1BQEGU6OfoIWWmmzVnr1xfrBSu+bTytOWfTUwOBNp8TLPY34XVgAknSDd4hU\\\\\\\/0ovX+lo11D3YGvJwG0g72BTCs7AjcSBRFUtujT1o3m1jEPRuvd9ah+TjIB6BlbGtvF4E0K2mnU6EKcDG3Nu+YWybSyUxGc8JM8G53xyc4F9oY5pfvrJh7zVKtEdzPhtLG1heCQXf7iRw3xwiEXNUgJVGBi47ABjHkE2j55h9FVQMSFCIT88rHhmyQepXTMUPQ9UIKzaeWqsUFbQLmj\\\\\\\/CrApJL5FtTc68Rh07gU2ka3IJcVEsSGRk28l3oIiI+eeA5sL36jeaQLMfXZdQTXZ+he1vmPZTItfMN5vWmk6V4OjKaSKgk4bYtVWiae3hejhgPNkSP9YNfFhESGZNYfouAfkpQ8OtSnKpQtjV73W2Me8Guml6LSY9zm0gcTbuMUPHIVF02k3SiFdSBj3HAJZfbxCIVvG1OHbNu2KFLKPaKHev3SkqruVTDJe8Sh743i9NJ9Jj5VNuwfStTEi2tOfBB4hyXNSWQOT2h9rRs9tDEBpz8AvButbYB5g43EqpgTgqCP8wOevCKYccz8Y6um8DJ35lnO7bAI9XN3poOAXrir8wNbVapdco3UKXPygF6fLSkXpUFqEnHlO9rwDiWbgfkz1so2VD8UvpRRwx2Aql351gm12L27W\\\\\\\/+xFGLfELj1kjC2A3tp6VQ4n\\\\\\\/pfrDn6P+ak7EdiDfxHUeV137abmsiTd4qs0oyHbfEWBfdl1lRzvpVz4PqDCnertONJ\\\\\\\/IKUUxPH58FG+DSdIirvnhB8Ca7yuRlM73d\\\\\\\/Rng1JoAsthnKkheQetr0lRTUxE5C55+BVVH3tzZ5JH6JjuS30N4OsdC0gzgPW23fmTgzmWgDeH0IP\\\\\\\/DKzXJPjzxpCOm7WudfPMx7cfRYAMiEbNZE285hso9mDl7LCsVUrw225yuNmbhBJETiM4lfDskSz6N7bNB2CN2dGC9y07e6g2AeErl1+JxRQSeVG01BvhSLyPuzzqjwDSjSHA39qEa6bd4wCK5UmdBlkrp0t4871KvwezcaK5BEaHlTw5MK7jRV8wnGALa3AaRcSTsmKK6n3GBwZghlVHy7ufjhanbJrNIYBxziMcYoSVwCLtCcB0K5xmFPwKbxXhRfWC8gnuypa5LINREEjWeDz\\\\\\\/9ialGCMbvPBZcP5jSjBjXZhy+YPN7\\\\\\\/4H\\\\\\\/ksWxnFKoevRvZxcZ2G+FSfKEb71DgHK\\\\\\\/5CT8LQQxG\\\\\\\/NWeg\\\\\\\/jHRPOo3KG614+Z3xIV908Nk8SP4udy0928AMdR2GSQ7JQsQXG+nhEpXMD2XrjSeC2xg9cfDgftQBIvLohgJZLyytFu459AhNFxrksor9KFqAktDLYDD6Vpah4nnnm7vSCiBxiguUZstffYdrkFEQcAezFGWO5PmUO75J3f+hmJr0ODYj1ARp3UaSmPHnxTTRkudqQdNVoN4Qnt6Q\\\\\\\/OEAyUfa5fjP1H6ZGbBXinE4qqKzYxAyu5XP0V7YBdr5pp9u9aBmsNwm3YlgH5vomie5m6tJstctT\\\\\\\/XxDIoSfbvEPah2iFpEq3we5dBHTQott9NeOkb06oCUwhNNhAnWoFFR94luNnp0LtNQGCp4xzc\\\\\\\/ljujIhgO83uvcMw7D1hoG0ENSjS6l0cjSYMZULQzVjAEDj\\\\\\\/nKUMV6YR9ZtzZ+kEM0\\\\\\\/7ak1y\\\\\\\/ycjYzaZ+T9HaMREQqMeeaZMNJ3Q9yVzMserQNiVz3B1CidLGnzmXz+65H0RuUy10vU6xDMMMko+ckMC1xuI17HBDWefT1fEOvA4VCmkoy9PxHmi3uiDQgndTTH\\\\\\\/O+POoTjQqcTHBtRjU+MHF9r4\\\\\\\/4VRHKMxBui61o4kjAVgO213q\\\\\\\/s+SRUANxg55hZ6w8JtX2HIUf2jLKgJvCuRR5Q+72cySMJH5gre490Xl7mv2EHzKZiBl4DZ3lW9KmP0EjzYhTigAqltN3cKr0SmqvETR5zEbR09YUETbVyoVWFHY6TlqUk7rNGFAhfaKMPyzO5clWvgzWyd7e2z40V3ZH21lyCyprluw25UaqUcvN9\\\\\\\/t5Ygq8\\\\\\\/tNpnlUh4OryEhDUI43SBwzI9tLUAz4YlqE8HvwiWb6apTBAMSLAhruzjk6bVaIvikFWF0A8jAiarnMjiSQ98mh0\\\\\\\/aiKdqFBsP\\\\\\\/2+LsiSRjgg6XBn5l3tg4VSvBf4t2L0ipVTm5jcdwIoM7HC5BQogXluexgC4dhf+bKiRsrnV4Aqg0vqJ3WIzDTYJiYHK9JmfXpJVe3Dh5lnpXQDOjjvJv2Jbw\\\\\\\/5C0RaqWGQq1gT6QkTfX6lVNFW5y2yC9y7xFZ\\\\\\\/GbYXPE5n+awYI8B4W5t3KagdkHWuYBMJDCSNKSJeA9DXdXeQmgm8wOols70quDgc1wxm5sQ2UspZQDFNsvLW10UQRmaUfslkuQP7dXmTKs0g6rC3UX1PEqpzGM4tnWaCSyMIo\\\\\\\/wuFzg+wV98wX\\\\\\\/+0+FIn6JrrJTZA\\\\\\\/VCnIcTBdWpBnsP\\\\\\\/vEfnvPgC9YuNm\\\\\\\/2I0lVvKJI6Aux+dLfQfJuDwAaxBPjMTdGtOYB4bG\\\\\\\/t3FetOYQaQPfiyw7OV1rT+EwH\\\\\\\/\\\\\\\/D5s\\\\\\\/0aswAlo+\\\\\\\/3gM+a7Gs4DytIEJ8qRYLvl8+MP2E6FAwERt0QES\\\\\\\/uwrGWFF3g3Em7jBYpKzoZLXgZF7AVwzNk0j6trtI540lCimnvYsMIQ59\\\\\\\/PMBM\\\\\\\/vHKBq7JK08QdNZTlpgnVX\\\\\\\/rkP0rrl2JLbiwdl2UAw5IntaHrztoIAv5uguhGx7AjgScJf3+hdsZYOcanzBFFVEfFq1xuHQQEQnYwHNvJg13L\\\\\\\/CjvCI6GwKmKS+NZpUDgI7JDC24gxjZDllyjh01ai6LSyw637vVtuVxeCy3DL0Ys21w0Ec1ir3Jh5hZ8e9qY9wyvPnnO1A69GvR2EfWw2lf9o\\\\\\\/sAm5N6+E2ghXkvH0u+ibR1S5taVr9Vxuw62EgjHlzWEk5EwCwqdRyngJ5Mh1AB8ieHT6W1auYFTSRAe\\\\\\\/0qiKPK415TtCkZvIADawghHDWrPyVJH8rO9vDJ9SIIUJDOWq6GmcZQxLLXPP+OEMeHXDBNQNO3SO7pv2gHgc2qiC0TVbD2ME3UkmAXpqf\\\\\\\/wWdlIZV+\\\\\\\/KNAGhuge0uUrHbkfyf5FcLSQWEiPvYWtaF4qh8PokDTt+SMRiyhOLey2Gqu2S7ydKKlzaOqGGWQrpZA+pkXOFgRPQd5o6CUdnqY8oF4vLI74A4pj9Muquyo4n0\\\\\\\/samGGXPvzz1s5Isft3RuwjfWCv631LPnRuoRZ0UChBobH7JFOxzZd+vhMDFpwzkNfI1Af1XY35dAYhEp5LT1ogenF7KMAirEZH9I5PPuZsNn+upa3tyD0\\\\\\\/+R5OlEAsyzRR85C4vAgjNhco+Zsxz+EkhdxZ5OUrWuJ6eN6AqUCwAmBYKOhNoCKFwLNeDzOQfr3kvnOxyx1YYKk46LrlQXcPSw+FIvmRnI42SPl9S\\\\\\\/DSF8u49CbcwBElpVZs3j0zYV+FLe+aBjJiQ9QISQrMDlnzE5I8xd\\\\\\\/AIQM3QnkTVW1hD\\\\\\\/RXDYGdEVF3QaAwqzLEzvqzIhvDxSU6iGpwmaR7l\\\\\\\/0vXcy4zC7LeAQioek2OgGDvppfK5E1fRBjC7h+QvykgciOq7\\\\\\\/AjN840FrR2i\\\\\\\/RbLEe0jpLFGofYc9IkZdBIUB68bGICJUOlgfZ4sCMc\\\\\\\/8ZzrgSLsPdgfX+K8IBSwbThN4MV+YlS8TfHp0I1RiHpLdDehy6PywSJDWlv7JBciuXwfXiCFuwFD8EHEz++OhFL8Oe4+4puu6QW7qT9GajfeyIVSOrciPMtE7JecEOpdd+LcHBS3pRR0qQ+rYFNmy+OSZo8oBWiawP7Ln0rPN8lsZLwW7KZe0V8oyZMLLxx4oFQmampMyWOjJN6ASk99yl5\\\\\\\/0AzDPAX2giNthyH6vGFqB+7BN6vQc7WCsJh6OtpM++mcRGvKSZrAaqMBHQajkG8JDDZYKZhvzAV1EZhkCneo+BknImJcyzNSUcY1MXLUk4wrZMOd4Qc2lNpCcYkHkj+YvsxOREtPeEktIPybJ2P9sB8yOdv7rN3UKR2YRkMMfGDey37B9JPI+OZvmgql2zDloUABvHm\\\\\\\/5BxxPgPj31xgUCYhV8+HKdk18j4HRGO3Pmu7atKlAAjttcmaX0qogKJjgxmYy\\\\\\\/MGQpRBomhlhAnI5ytjyZ8DnVTjhcyxmByEiax3LwZCbCFLxOsqpv+qBaYcDWK8AsDiiQKdl+DS1X0DcpoUc3j8x0oPrJmoeYn6WE\\\\\\\/ArYgFN8WB1lUwDWirRqbj67tMCTRMNYN\\\\\\\/WyW02BgOQZLc8pnizKXTv5bLe07Ok29y7Zca167MyFAepPLpvHntNrgXV09N9tLxhytV2jRGCK3i\\\\\\\/f6v+G0M7\\\\\\\/GxUa9Yvy\\\\\\\/OdbaBoyTyXC7NSt1Y8OJ4CEr3VxRKWtPpjatmnI1NwBDLaq+MEiYU0FMfJJ+FgLU\\\\\\\/0WgN+M8E9J+lDpdiBuKPFHIy4hNIVjd5hSd8JqZIMXzch9p258Sn4blOo3t1uhRwm3BYow3HBv0O6ScV+5jt6hof9QCs97Mo+lhgC2on2+Y0UbH3MgJc69mi2tHUnlpe99lrkMDJchNcvQf6VYNvpex+YoKn0MUk7iF9h2Hkf57hHw7O9jxkNNdO3d8nfAiTcDJoxBG5nexbmEUNLOgrNI0Ky7v5+1YVoO42mXtoSSAE6qrfr1Lr7XVdxXni7\\\\\\\/M8JZdNkLCPlyXs2bLqJ\\\\\\\/XPrQpfJXnSm3jZQ3wVRs8Az5xbO2ElHc32NxJh\\\\\\\/p1BRFzbZaASUxHnxg5ugrSiHIn83Q1zUQPvPKQu89+ccq1eRbbbHLphh6IFc3jo\\\\\\\/q3\\\\\\\/POsT6KVx9UqJbFFK3j6D1mehTg9PtllsNIXcJ1S\\\\\\\/0MjgmYvsjXeqqRn8m5onT\\\\\\\/0nXpBWS19JgbDZu9PPRU4i1Yf90qZf5WlNQ5irZ1\\\\\\\/uIKkY6armPzmKRo++Qx1dcbM3duF1lkYBHLRB+3zEQWapnvpZ9CDEenU75b9DFjz+mc4FKM49X5lxiFog8BRR6K+BvSbLoxomukwm0MpCcI71I5N\\\\\\\/g4zktdXfl2+0og9Ta6MeijWyfA0kzBCaF+rheHoJl9xuJ27v+a9bm07f\\\\\\\/1aTzcT17SY8fCidP9B92479HsJlyVdnEunGexwBqUh9UxcSw\\\\\\\/pY1rtSa\\\\\\\/p4tN34TQaGEoItPpLbkJD+DpPKbOweoksKevD8UkPqmGL1fdaDGp2Wxl2XQ43UmPrfQPEO5tA1cCMKlq416Ump9lAxFvOsYMfdoRjgPkGqo4Gm09p3LhMQvRjI\\\\\\\/TQt+CM3UhOUlUyHHkzC\\\\\\\/mFuJjZSL\\\\\\\/JIrrTEokKtsGCSFmtxkDnLhNODSN19+LGeH1SAbUm9Wl5l6fwaQspfOzDki09k\\\\\\\/0bWbDZS3aWJkZXfAMO6umh3g87ZYcbjpt46\\\\\\\/DK2v3hjmyvyhjYdAAX9WP6\\\\\\\/Alzy3\\\\\\\/ywv3ffzsmUKHCGzTzZ4V+O8iHy0UzlMKJ5\\\\\\\/2XqK7g3zvcN3Kj6UofBHxZaRDSmt6pRai9JM4QMk0\\\\\\\/fyRW6RbeYvlmkzIAY\\\\\\\/SaNAl+vd37qHoh7Up+0gTOUUth9kLXmK96IIvpCWGsy0TClOKxe\\\\\\\/Gdg0EHZhA1igbnlIT0ebIy2RU0aznQAaxm\\\\\\\/fy\\\\\\\/ZfOx\\\\\\\/mh8Zg3+5nOZUj860zs+bCOSJzU0YdsvC5NrN4KLwGWzsoRyO61tKHtpPSOq\\\\\\\/OdlOzLWTl+dl4Eruokpkpia99U7QDGk7JzPok0SgdktZSRmu8pSU6KQlgZ\\\\\\\/N1tP8wbKUlyrgBehkVssYbJPqv1QkpP7JWRbaXZBomayJdJ+ghOO\\\\\\\/U6CN8EnyFmcyEfKTDRN7PkclYKIyK1mmV8eWl2BEIyjHvp5E+8WLt3y\\\\\\\/ZXqLtXnQpRb7NqvMhzMrOZ+6QEAUmGk+puCIKYxxMhyS2QTwPBTLIWxn2M7gigENY\\\\\\\/7y6Tft9CBiW3bKLUNJ0tZB462kYqMHM2Qioz\\\\\\\/JXkOVo4H3VdRrTbgBrcfHUQxXIPm6Dv2P5Bj0i\\\\\\\/aq0Di2Qqye\\\\\\\/lK8lZqLRZO0M+OEacNaG7v2p\\\\\\\/vlYQosDNTdzskUDu+iZwKY+E\\\\\\\/63sCs7wPCzhUON32WFZkw1kyk4gJAJVG8jH7z8ZU8sO57fqJdmvbg8StQB3GeYoDXRj8icX+JS5c0k4Govjtw\\\\\\\/PsFm9Q2GhRJ5FVLcij4hjuaEQmDVKJ9qNUAhFmFprNl71JvGTFWSnZe8G84aijAv2ou91E5Pyq7nFFvYQVpcCXv+DQEngDDaLwbp7x7mfPYX\\\\\\\/HIDi0N95vlwnjcL7oPuSaAurFaGd+RnZDWuXR8ZNqbHtAFJXCdgzTZtlio8p+32cOZfEsIzOp35qmgcVQBQpMdx6jL4Sy0KSIT3LWPs\\\\\\\/gq8vZpi1mnenH6uEsy3HSARfI+KmWxurtFYzXIjrabLsst7+\\\\\\\/pROQx4FiSQww6G3p7CZSPG5XWKzSHwoybIY9yj3cvmow89gSiQrfw1W1x\\\\\\\/Tlv0sjrCJiqdL59h7T1DIlLLcjmTD8PgXXAQ5wn9r6b2sLzrD\\\\\\\/waei4ZhDsp3Pr9LxpFcmQk5jThWhD9Nw7GsLPOLPotBs37xJRUyNw\\\\\\\/lsfhAtUhPz4RHJVMi38utxUAhZ6Q2uoUbWcKY9JrZaGDPMVpvpqY57bvnAMeUOjpf2FTRmPL6AgIK98jsyHp1Pe0NICLtTqBaRR4I+1kE4dUMY6UYZQ8F8QZLjriSPJx3uJdJ4XQeq8KCAOPT2l8lnnWBNDo\\\\\\\/moVstDbvPDhYdOfGURtszTlFe2kdzFswrhp9tMKA+U+crC01zWbmXVFhL3caTo1AB+CBg4GVGgeDZHBfNjxBFa1laeAjobgoDZvdPI\\\\\\\/DK8KfVAzIGHw6Iy8qx5fkCE94FLtgPLA39UMkmZ3qNjWlYB7kgLUsZe3H4Jh0J3BknoWS5eA+U0V\\\\\\\/6cuy6WI0TWIhhuNJPfrAItdNMYMa3sOCScFfqoXF1ialUMwexY1+qmZ12CKC3B2ysdsZOsvOtJziFg1IcBy5kHSr\\\\\\\/1YQSE68fvQ1FuPpabeucOh2sGozIce83h50UtUZ9oyGdsF7uWCAdOvYbfYcQrUGU2iB8J4hhiH4VRSigMqFW3dNQl8l1lenKT5gfoHHLcghBQjWsVnYKn7evFSn+Y4MoqcudcqgfyRAJvRUxQbgORdQvSBihDW\\\\\\\/l2CYGRtmpBc797lAQ6Wy1G\\\\\\\/TvacaiQ02HQUotpS8Y+cg1uUrE5pigJJqKqihj9B4o8bB4jwFiUCMFtJYlAymYdUxxmqbhSic4Zj\\\\\\\/+GmiyKaEaf2CxGe1wkVHY28i5gamWqn8aE6Dm1Ot11mLtkU6bWyyBEGP\\\\\\\/QlCtZXjy+Nxw2qOHkXC+PFoYXfKimQsL6ARVkVvKLbDfaatx2kXcZUXJWKxwwasxWKKpA529EHSR3z07b7xz9YXtDfvh7eL4pQkh+H72nE+nlk7t+IAbcVhMbNlsow0gS8lwvRXgi0FvSrX8wLFCRc6HARXoS38Hn6Bp63Tz3wHVDCVGuDaBIEHb5Ub\\\\\\\/6LtXCRBAojOjxlV69O4yuGyOelJHbb3cc25K\\\\\\\/XikAX5QNr1mNHgiswlT20vCRZs2Mp\\\\\\\/9PlFJHPhmYlIUrRFQNwgAwdWEH7Pkmvgron8AjaAZVsezTTQKoMRnOp8v6y9wmGtS1COtA3f7la3ZV2RjeKhToIOV2UncGRuFYkbPEYswKfW+OgITLYdEuaG4ma3dxcf2zPe1qZPH+oLC3Fxgy8\\\\\\\/Ju6N0amMkjIEYywkdZxe0jV0a58sjM0Hidxfb83WPvVb2lXfnbbpsoCOW0CYUTXVupkB5gVV5UVcCDL\\\\\\\/GDce6ZsfQcjuI\\\\\\\/+WHF+KPWy4Aei1+sd8Nq4+P5tzpipj6PofSLt\\\\\\\/VD2ZsAMwRkYQRtkD+DXXWwQSa4uDGDnlSbqPrS4Uvj5y0hTHBgXx5tzK\\\\\\\/0uJ\\\\\\\/SI\\\\\\\/S5suXtGXQL9nOEGMuY+SOz7UZMsgflawhv7XPUnU6AhCJm1NfDa1BHzcK4zQLR+Es+DCnhZh+jrmMe+D2AVdBRt3ylUw6uE9qXokCGu0OIItBfDz0OJ7X851knEDAQZr\\\\\\\/dzddzYuoUXOgpiyOpAsUpgsubMEVJr8HoJuJS08aWe1SOdQBt2wo4aPZmfK\\\\\\\/J\\\\\\\/gh2TdzcXFu8M\\\\\\\/ibugdThRRinwZPX\\\\\\\/A+v1n2BkYCTSFulNjMGABvGd2zdmNzAodqzQ2JdFyWEPN5AVSrJKYglJz0F2nNx23XrYEWNqm9z4+jht6QPxwoIdtYHfMkjWwJWi55XUVDQJsR1XAR8b9QYwAkyo3bCJyci7\\\\\\\/iClH4KsP9MhZVEiXMCLWKjNBt95NNjikvnc9HUtOZknPNTAV8YQIK6uoE1A+txQWEzbrIqw5fbU\\\\\\\/wdxauXUFxNH+fePpAUUJZNttbrPL8VTIbQTo76\\\\\\\/mDW+Bmm9y5sitC2Xlp3HZXFG0wB9QFv3R3IpB+mGw3f\\\\\\\/7B1uMNNf\\\\\\\/8CTOwFI+QqoFR+7jMVfyGdKt8YnnxogKc8bHKCfmMiJbT9aG\\\\\\\/fUnJUZQjfMEG1db9nQTG+Yt0X+6frJWi3rv8oZXU8nv8I27LCYFde55NKmnA0pvtXbzR3+CmZ4roByiKpwQ2ArSUSHn1jkBRLTBwkPeExxyTI+L1akiViS3QZFT6WVhbckuyCqbxc7zUrLq3Jm1npI4DNaMxRrZfyE0I3Ib+5EZi6d9k67URpa4UFHrzx1RN4NtjxcoumRcCvI2nc\\\\\\\/zwcpVV4o\\\\\\\/P\\\\\\\/E9DkBqWd4FmXJiUl55fv7bQ+In7oC4U7q8wKyZbkzmOVfukzO2GdTTDWlFvrJ+dk3H14PCeB7jR0G35Ycuz5jEk1rxA+qmWWTlvm1yvbJapKXXkWChRnqksT3BQZnSLQny6kr2K41mRg1BQjVp3e4+njLuucm5I4DU6xzims5CpXhm+Ck3FdgWnfATPzcbWqu3eO8C9ZXYoEjRhwFKQH6A7rxU4VI3kPUFhjFeEF\\\\\\\/8vigQZR\\\\\\\/QRhrBm53ssaqhOdX3ke8ko6X1a69uH7tyX96ojJJ80Y+m0jajxcj6zZk6jkOhIIjg444KK6i1u0PgmngwttbppIxN8UaGhxCuO5ZBVl9cgvOBupHV3EFdu3rcUNiH2lTJ5ulgeb16S3A\\\\\\\/VnNMkncL3jL080xfpXEaWIvpf4Hg59rXYFFbKra9wVIcCGUEZmmHjSUgeI20TBomivGzBJOxm40y6fmoI5s8kLUsX4z\\\\\\\/UzShZ1KO0rVFr3wJ7zcKIGDnuCIOCTuzgVY95b\\\\\\\/gJ0RVvGKKgHio5nnFWTPxuuRKgYFiKj4bXK4+1ccK1\\\\\\\/zySm0CP7B\\\\\\\/t5eaBp1bmrsLttkIjyLkXv7MzhvzQ\\\\\\\/Vo1s5nxUmoKw1UwRBktuJdpBsV1pHQ2fGTloN9+dmjTAyakGU560fz6FwhkNSMHVsIBcK2pmQPcC8\\\\\\\/iOUS26VtLxQkZw0aW913kdK3tLd9U2nsHxP+2\\\\\\\/JtQW1Rb9a38FWjYUAkxNcA1KJVRVVAENivLrl+yHUiFoT3LIKVNXdNN86X1KSsabJdHnyKmgtkLA1hbrsQrPoXJ1E591Ioe4J6IFh3xG8iD3JyMf1MNJK1fTc5jrLZDjeoR1xhtPVXFam+ozu4BoqClpBhFcl+VLCBB7trqtMwTuk50Berd4HNJLdJSsfV9YKcs7NyocjDRoIk1QmNR+AddNGjnVPXERaCZA\\\\\\\/j86HzS4FJ5XtywNPArajn1abMObU9PLsS0wuYV+kQWwMuUe2LjJDnhEcxgmLA+LyHV0wGvQyafQkaFtEqscDOPJ7ElMqYDxcdMs4IMcBNN2WjCGoPnXIllhlzD+hUcQP37dNOKr8obsvi9CI5EL2TZeNN2kKORX03u0QZSSR+pqJlv5DG5ivE34BE0yjJ0QZSAFyr+HKpxNXApp7rQjl86Ecw\\\\\\\/LHCXNk2W\\\\\\\/I2dJUKsNJLYFdOngaYSFW4b6x2SBfrUdo+xtbkDfcTprbx6qCdoot9cAUrz9dsopvS2tawsOqYgsmOUMEjocdMSEgMgixlUhfyZEUiOvopG+zu9mrOruExk8sAONCSwdoSFpveIQL7xx2f7DzfIORHZG4aiMBwcUDDsf4W7bJipA6rXQAwxZLsGwP7cl+7fBuWOPlLz9npn+NBP0UJgeYz0Gv0L4DiEGSFrwLIl3ZJHz9MiWjKXJIrZKPnLMOQzhVMjvjUFh1\\\\\\\/pOnRUVFVW0fDP1DTPbLy04dtQYDpPoSyHIKKeg6eEGrYykuEcWQ+NJcvEbWyh9\\\\\\\/hPbTqnWmxBdgghBWIHqyOM62+hld+woaWTqU4msvyJDzKe4hPsnhAD10mQQ1vPRAHOQv94TtFTogM1MODMaH8DNUY\\\\\\\/ojExze7VUs7i21VWNgAZl7fz1sjIWydsDpDh3fedGU0NNqI4wrXnY\\\\\\\/2\\\\\\\/BsMW1oZLyC8ZiyuAaXafyZDnhjYr3JNT9LqaRSVl0pGnl1pjDtJalYnwE\\\\\\\/\\\\\\\/NPs+4nC5sZoYMnSskmr44Rh68ewGzWMTBDpStIzw9p8xUkSmPT7XnPFM+u3x8PKgNbCtIW1PC+YKQFW7kg8s0mJTeYtlg48VNhlN6WGR2uSnXYDL9oyF2fC+1AALoZX5DhB6Hk3iGVOKpWMbJvxjYPjPiZt4KKAOspVmgjKbSM4z8Tn8Ek0apWV+KrAlMRbKI6hHlIfW14p1H74MEaJ4hDxDET2ZhMSA2vLOzAE\\\\\\\/wepQHjZERP11aXKk90rocdq+JkXEPkALLQvxyhcaYAezQyxwj0vXJRZxw0LVYBDg5eY75HqeaIWKt1+fXPxufeCGHF8Rhlx3mjfjbU+P93kWFDmkJfQU6hjWCLQdtPM\\\\\\\/x1uNvLihnZeOzPMLnGABsBRDnYqBsyzzbzBvad9l1lPnFrgkJRghNTESum1g9rNs\\\\\\\/biXCG3+VR2Txsqs4+W+nw8igiE0h1CWPmqA4a0DjFN7uL0fcHKnh8naLsXIFD\\\\\\\/R5rECsnCe6dAO7\\\\\\\/UMd2dDxKQCewKZzDtHkk5Sds6o97JynaAhx1iLDlh6pCBu77KifI0teD5JyseCAYXVH2Dp1tDcaEsqCPRHrsK6kh37KyU42RaHeDbyx0TqDvl+fmXEguCM5WGqrnBHT4mU9GyAp6E84LxDE6RhQRNOlM9icPh68ywzYyhQm2oD9MTknX0onmsdS7na3mZr2uv3ne+FhqH2SvfPeHtebw0E7RNZ8glFJlrsGaMwk6imdw8\\\\\\\/Lnu72sCE7Qo97hziRqUGjECUcIT5ePj8Uq8157l9KXBCq6sMwdZ3sh\\\\\\\/vpty0rriotp3uDm6mhDYi\\\\\\\/Pvs9oqT4mDgg3Y94oW0snq9qAmm8FT2NhxoodNiMDalGWo2yGaYi7djBXv9euMJTc52FbBGHTwXnzfqSDIQky8Ltm7dhJb3Q14YcKllKrIYS2A5XS78EdaPBtd7Sqah0fgbrxwbPujRX+KuD7U78ycFxLu8WkMpy2Ao9PLEFQT0XdOeyx5yD73J0U8hHejv5DwEFMWgCVYdvTAKXkJ5GvZUx3QlE\\\\\\\/TmL+D4QeFnL9EZvLjw0ETm1o7p\\\\\\\/lyND57J0VWuwjvXXZu0Nw9YY83zvquibpCSoxxbqNg8PsbIXdLtc1StK0dKYmR8x4oY68zxH8PuQJvlpVWK0nH+vrdBufFbEPjIEpeGLsCxPEiXa3KZ3T4oiGUYTSDSYk8uPy3j27a\\\\\\\/8PMMJEcBFKkDzUSQm2GiaYaoygD3KvYi3Uqs+G1Kp1qvuHcYYwnbJXwVBA+kfrlzdymnDA7BTZg\\\\\\\/nLMtLgzgrHVtv3VqZZ\\\\\\\/AG044r4apd0g2Ixh7in9Ed13mZNKtXLZb5sJr5KOfPeZyBYb4C2Hydk9yM5s4D9WjzG\\\\\\\/2MrQ4ZxU87DTQzRdHwSg\\\\\\\/bFZcQRxfvlgMiwTmCqj0VzRcZy7hoCBlLa7ANrCrZ0EujEUQ\\\\\\\/gHdnmov08XO64o\\\\\\\/ZEvVAtp1xg+f+o\\\\\\\/5jmdrWZuf+W13Q7GVQyVw+26RQDXhJNIrzPLveQ19nxMVlNpJzuJQrSDMqhn\\\\\\\/NkeoOsuJeoFF0uoJWWyhGeMoBJ2dg1YTHW8TUzd+gXa9V49LYUjv6xgWyCG7AjV5FYiQS6AVSir2Dj+5H607wCISmPecqvbWgWyI4YBWzUxqqjr\\\\\\\/umZuOUCE0LZ+yMktgKdqbdgt6hgqqJojX+YBzOKTdWcFAMnUa2N0JHYsuo\\\\\\\/AU43UuDY9AM8xvMpyJkFoCqSWZ03TgIfbHEpG3thQbS7bwpJUT7iVP\\\\\\\/vZ+MqgUbiOlKhz\\\\\\\/GXDKh1x3esMxStMe94VCF3CjBvfCug3flKgPKUW6YwTHYMUVpAiKqa7oZzYkfFNuLIiuAF7hL\\\\\\\/kOBmAjM89z\\\\\\\/xFju28Z2q6zs6DYLMQK0zqCjFct91gxMHn7xSihgZI9jgpQ9Htet2rQnb02G+HE1UGjguxIVTuV0Ex5qvF17oFlk3dqtVk+lg6oZbKhD7U5Xpem6fH0qQX813+vu41zjFjiIR3chd0KGInKlOqi8j65nJw120HVpVMhkDoUxC\\\\\\\/Vtn\\\\\\\/GRQcFk\\\\\\\/iChRmXX6DKbwOPuK8xTSq1YmWq1k80V5w81sv7BJ9D2+m687e9so0nK\\\\\\\/tS+wxTxmKltBJW4w8Pdz5mgAAdK\\\\\\\/mUoOEt0Iq6CigKnouRs1umu8kXi1S33wW\\\\\\\/4UtuAZ3IwTuGUtQkPFfVoFdl9+1zFZMObwFgE4gHPavsXcoS1oNuLYQ9UbFA64KnRanegAPNIHNLIwf\\\\\\\/iY2krXvcaijbhemof4hk1qRDxAQqvoPg\\\\\\\/PmG+OJm3pUe\\\\\\\/8dYIjHqr9ob4CdO13vOhqIb+ZtkLFNwj5tW5328d1ju1XOLaPyYVqO9kwiFf5rn9JrFUAbi1U2P\\\\\\\/M1IVzv0ke\\\\\\\/XkI0sWI55aR8d5bHJ7vTNrSXM8O1AwqUJmYPgy47Hk9FboI1WMtobTW4Vu8AjA0gQ0iyS6Yiyt71R0ItXV2K+3oStrMXp8NEQ4rjHQ1QPF6515T9ZaWhEN\\\\\\\/6aFkXz\\\\\\\/YlbNDD2Lnh14l5IB5d0UjJGK13zzzgmO40Xhmvvu3ULFU3rLheuQk4wo7S2UZssqQxdQKYnvy\\\\\\\/WCsa\\\\\\\/EJ+tyaPYIu+z7zIOQbIqXDNUPWJPShEGGgpy0y+4GhenksAahmI8V64Fptx2rXPR2zX9Kzxg\\\\\\\/KDwibwjLCDo1TiPPWWpaLxoK7IyIsfAfT85OTUQ60CvBkFwnyfZEwDCVMCycwX9BO1qeD0bwNohpKLk1tnXmZ4e+FHLjRwyM37LyNxs7C5fySyzA14uvT384smf3rm\\\\\\\/ueLTVAzn0yNAbeQ2C\\\\\\\/VgpKQFeQWqtvuLA6mJ\\\\\\\/KLaoGrNOn5hnhcB1ddV0c4GhnTpqv9nts7ZUAwV5a1v7LrXI10zbDWry8xXCI6j1mV4Y8C\\\\\\\/1vgfG0b3hm62fMBP+YOpq6GfvfRkR+UHUunlEi9z541ZZQ2WWDHV7si1kK9gE5zK1jsGTfhHc2h0dwKGDAz1nQeYLsaymfED77DX8Jbg1iE4VdvXQ+NldmPbayRiIBMnF+GNUjD\\\\\\\/14FNOwKJ+6ggBHXyGR+DyFkDOUx6+ewzOQz4CGT0lMUuU2ZHsxLNS2SQqp8lrgwsXszWabLfcpGLp4TvaBiDWCGpJcXXkvc1l6d1rbi9fMoH2Xf3soa1WKNN0fTLribUFeUbkkD9xKVB4CUjtoTvPBFasbPePFMwKYTYtiQ2hbN0lPVzubAeUbjwuwH1Bd4W4VAsMGD1QoGrHcN9Qony8VLQhWPkaC34+LlFNTzZML1yID6bUz0OYrCUw01sMfryfjDof5ystnH18fKyv4VE3TZr72thhvhKI8Rc4bYRmnhFgTyo7IZrDJsaVYiRWqS6yXF0qIU6RICaFCxIyiLG0h1L+erAhOCKwMJBkMVWFx6phvWhBMzMB9kbrwn5CjAOfPYr+Ea7P7eYv3iI74Hy32kTPVtOP05x8oOu78\\\\\\\/z2i1+H2BLAYwYJFnhrTsSzYQf7aF+djF9Lt9alUdpUl\\\\\\\/aX8IUz3LqAYvku8uWKmkFMT1BDUQgr+\\\\\\\/vHepQNRlGHV7RWeWHGUmF9YveP6feGPi7BA3+EZwSo8A1hwwm0G\\\\\\\/QPO8YLRhxN51EuJyKgT6UTFRSAHYjiPR93xVx48I\\\\\\\/cJ56nK0VJO5dWx2agVQ4LkQwC3oRhr5qPFFaYYYkTlf3speSZUJMLT+c8feDP96lQEWWhdT0IeRTb7pNsqEBhdl43OBWF0J+S33a7qEtzOTHa0a6DAcZ1NxGRLsuX4yU8qE29aIR8JEMJ+yA4V5nzi7CRa3wmzrzLY7t2gFyms850q7CQZyh0JFW13fFG5wn8si61mHBDE7UuijodBdE2DpeV3zBNOFouvwNVbyh0WQZKESquAJvsS1orjhd5iI1c+lHIoREplv8RgwF\\\\\\\/wGlvz5L\\\\\\\/EQ0+puckvIaue6gpp7SLEoJJ1+AUl16K8tAZ8iFtrh0ijUydD7dc+\\\\\\\/RGyqU0rzK3yFqM3DFTCpdzC0PFq0r3Hga0uhl8wq\\\\\\\/qqBv8bIVtjjMB3mJiSQplf3GauVEPXcMqV9GUQsY58DQgNSvpNeJnMTbz5+Zqd9DYKyB8Sx8UcMILZ\\\\\\\/mNkXBzCPRVQNkfAviEbIJ4CnR8kwQgLshuzo0Zq9SQtFgmS8r0IF99gt2\\\\\\\/s7BjgPiLy7Cnx9bYS0v0VAYBYpdoGQVxrQf7\\\\\\\/R64\\\\\\\/GXwDBHlGJDKkikwIemZMhaZ8f95IXVeUV6aL7caxv6x45hn6MX4dPlJFJtby+aNuVR2ey6a5zW1FVWAS15mFc\\\\\\\/A5fzuTToaPtHy+h0RBuSy14Q4LXP9DVQKW6fIeqacswl\\\\\\\/GF1XVjmvEvfe\\\\\\\/LEmgm2e4bI7qfK76SOpKl1K6+t+EUqQL5q3phjIrlrxfeDDKYhw\\\\\\\/LVgAQp9YkEXlshDjx1af8uQg\\\\\\\/0oDh9fIA71lFBCvh\\\\\\\/lqrsx1hdePrA7EPjKKYXh52AuPGs26OhCqnleErpB7dla+I01x2Zi\\\\\\\/Lx+OSwBNTO+mDiRvwutbTQj51vFWLKO2MIOmc7+9QwlS6afLJaqkIFRvQZdZOOQTHPkFBH8GGwr6JLKto4SRbYSCFzsWjzrGDxKPzQqwT40PSi0m0xU9hGMnS3dyIyrjqCwse5nB2ECmq4hBEsyYmcWPm31Kht1f+SYsJyeJQYkEMpycETeBNVoI8tCL9y9QHDpqwWhxfwGYtRKnib+6bRssyHB04uO+8bXhyKPCJqQjU4391WxxkB07O7Br1WYS+A9z2o2hpo0t5DrPktZZqLBt1zWJVCLPcIxFvtxU+bKLgaZoJiPYl8XqX5RbGQjJ3v9kP5z2NMVnAPD5mQakEWPAdKrluARXtbzencA05uKB09fmC+\\\\\\\/aT2lbfWSzQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022tJ3MNcIPmhteV1j2764u4g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220901f497a62643490a9ed1b227d579464abd55725c647497815844952145def4\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022F9M2+Kv8WllLyby0J9uzJ4KCp8MGU7FYqDixveaIB6blX5+QPLgld2knOpEjAmybU4xWwaG72uGxfgJISJYwc2ObwT2yfQ6+8HKyM98UCua0yvTG9zobWqEFfL553xy8CGwco0\\\\\\\/hO8FKjOsWE+k8Msn58SqbKX+9PHIEODk3z3l+JluzSc5XADmiDKefcWhKP1uFpAcr6Sg3UsLHm82oOPjmUIqxBIy0H8Rw4i0+LeKJn\\\\\\\/fpuw+6t5G+\\\\\\\/RjiI2OUlBoItt+946m1gB8p2LrNQF2MK7XklL4d3f9t6fWVzc8g0bnf7cwjn3InfQ6Vs4au6KxrtuDBm4Pp3gESzkyvZnF41FOsoKR7DuBZEpN0ORyoXP61zsSkeYq5lMcgWXyappkoR70HNt6kNRBRedeVDpeOXZkvmmpYxuIzLT1N23ZhTIr\\\\\\\/aBSufwptcIRt4ffqUuFCPtW+G4sRaKqpcncB\\\\\\\/QwbAWK\\\\\\\/pgUSiT2OC5ockmZ9Ko79fTh4389xUU3+joiHcRdIc4ecjIq5nIxM2XYJnoym6qvXhmrIBZoPlVod+DqQ1odV2G5p944uwy8v08FYWuF1ZtG1G7IZORHnBcerFSkEnFsPmw9ErGJa4MeYvS4rGsKSB6T7cVpq3zE6Uk07VWcuccbiTLaFcMCY0cbjmTjHEGp9YKFigVazIZFiM7zv+2y3uBmy+CdIP7Zh65kLiyckGXXYIMwXKUOmQIlxb46VJGrE3tq+lC8CFtC8qH\\\\\\\/ysHe6SvFCTr9IKqNqquRtrSd0CAIYFNXCZWCODPpgBs6bJIsLZK1pIGTJOHIdjV1anPmff88z2HV9hGOX5aOwNWlY5d5\\\\\\\/2gdDJSroBELmXKDiWcVoxGSTV6KaVY2oycvme+HlW14HEeBmKOiQFRwpcjh3pMISWU3UBedm3rSoUM8Ynd0z2OPwQZmgvhm6ddJMPolLpJlHoe0J47Nd4sYbxUNKM5vZSyety1sCBYyIOxuxgM+GkLlQ1bcgHGRy02DDfbSLEEpysHFSXIYG0FJM3xZ8COJQiCevOFXbLFNqAiTvS6QmMwXoZaJNYJGrhvYl9yESUfCiSB9gzfcEsEnmds+LTKsDHt68GHaYKinR8ejByZFRzepVPmezhMaYINykwBXJb8bJGHwfwOzp6ft6AmSaHOTVLSCdQ+VsF\\\\\\\/TrayArMwY0+ZMJGLQ6vjnbh3LpXEXYPGEfFzf\\\\\\\/26BbFxAVzHHX4VyhoEhOtYE8U35V\\\\\\\/FC8KVK4pr7zF6\\\\\\\/m5DivHYvKtPX2byMxpkhs9Rzhc6jfDhZK07yD\\\\\\\/ujZ0C0ijthhMt0+VQsaQB8A6cgIQ6QuihOaTonil0yZvDNn9OspP8grW8\\\\\\\/4aqsoftYVZ6arKHVe8ZTAWHx8Ly\\\\\\\/wo8gslgtiVPfLtEezrEEMhEk\\\\\\\/LNfcWc9PTovkJNH13nI3JPcrkvfrq386Lv5fIz0NqaSxQrW2Bnm50v0taBOSW08FLK2ae5QCdOfPw86\\\\\\\/0cmp3QjLndcBtrPB9QbWQhavqDxetTXWd+pCUnU8d0oe49dF64Td7RS9J21VwTslK28i22dgokVfxBC4+\\\\\\\/SeDAsjmwGL6Z6Er7Q7ZWYhZsK9BHCndwDkn1Jv3f6K6BFpjE0N6xlFYSS2Sgig+j\\\\\\\/NRJmSVMO2mhSLpgbsxkNXk5EhQeqrlPbOZYOipE3RFaNC7IEu06IlcDqGYkYvUgMXq5Wb+j7rGhEcDV6CUdUNfRlqh7YxLhiqmpX5Iqh9yIwI1egpb2zr7BriTp2Qqp67\\\\\\\/p92aP3FtXMHobYwnrYtz5EY4ywtN4mrWddWWE6uSPZc9AYY5GBR+HREBxrschz2EzAgP8qQYI7ZAWr\\\\\\\/Z\\\\\\\/Beq5eUSq9Z5Xw3w0FBUhvgz1dhI4sg2jrmlkbuqp3w\\\\\\\/CJt5ypVAbQI7Qgek02e+nL+3oY1BMYGAPtae7iddWQu7lpZc1Zm0drbzov3OjDygklhYGfp0AIPfeqfit2ZCX4cSMRoJ2ks2jGE1fBKRC7a8Em1lRV\\\\\\\/Y3\\\\\\\/5Zx35\\\\\\\/g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022O7o47aWYCW1h98fSipjadw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002218fbae7969e28e6838bcb0041090c4ce50c50cd1f0e09357eac2d3851f3f13c8\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-483527 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/3/967" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-NPA0X3YA85VgaEYrntQPbaELDY9gglmkIeCVc3To-vc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/3/967" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-g1MGzXxA62pDagl0L4e1clmgR6EcJdhOETNnJqePYBg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/3/953" title="Genome-Wide Analysis of lncRNA and mRNA Expression During Differentiation of Abdominal Preadipocytes in the Chicken" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/3/983" title="Identification of Unannotated Small Genes in Salmonella" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483527" id="node483527" data-pisa="ggg;7/3/967" data-pisa-master="ggg;g3.116.038125" data-apath="/ggg/7/3/967.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Impact of Selection at the Amino Acid Level on the Usage of Synonymous Codons</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Paweł Błażej</span>, <span class="highwire-citation-author" data-delta="1">Dorota Mackiewicz</span>, <span class="highwire-citation-author" data-delta="2">Małgorzata Wnętrzak</span> and <span class="highwire-citation-author" data-delta="3">Paweł Mackiewicz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">967-981;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.038125 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483527" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Paweł Błażej </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genomics, Faculty of Biotechnology, University of Wrocław, 50-383, Poland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pawe%C5%82%2BB%C5%82a%C5%BCej%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=B%C5%82a%C5%BCej%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APawe%25C5%2582%2BB%25C5%2582a%25C5%25BCej%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dorota Mackiewicz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genomics, Faculty of Biotechnology, University of Wrocław, 50-383, Poland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dorota%2BMackiewicz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mackiewicz%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADorota%2BMackiewicz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Małgorzata Wnętrzak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genomics, Faculty of Biotechnology, University of Wrocław, 50-383, Poland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ma%C5%82gorzata%2BWn%C4%99trzak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wn%C4%99trzak%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMa%25C5%2582gorzata%2BWn%25C4%2599trzak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paweł Mackiewicz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genomics, Faculty of Biotechnology, University of Wrocław, 50-383, Poland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pawe%C5%82%2BMackiewicz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mackiewicz%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APawe%25C5%2582%2BMackiewicz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:pamac@smorfland.uni.wroc.pl" class="" data-icon-position="" data-hide-link-title="0">pamac@smorfland.uni.wroc.pl</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/3/967" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:483527" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:483527/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/3/967.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:483527" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:483527/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/3/967.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:483527" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:483527/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/3/967.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:483527" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:483527/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/3/967.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:483527" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:483527/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">There are two main forces that affect usage of synonymous codons: directional mutational pressure and selection. The effectiveness of protein translation is usually considered as the main selectional factor. However, biased codon usage can also be a byproduct of a general selection at the amino acid level interacting with nucleotide replacements. To evaluate the validity and strength of such an effect, we superimposed &gt;3.5 billion unrestricted mutational processes on the selection of nonsynonymous substitutions based on the differences in physicochemical properties of the coded amino acids. Using a modified evolutionary optimization algorithm, we determined the conditions in which the effect on the relative codon usage is maximized. We found that the effect is enhanced by mutational processes generating more adenine and thymine than guanine and cytosine, as well as more purines than pyrimidines. Interestingly, this effect is observed only under an unrestricted model of nucleotide substitution, and disappears when the mutational process is time-reversible. Comparison of the simulation results with data for real protein coding sequences indicates that the impact of selection at the amino acid level on synonymous codon usage cannot be neglected. Furthermore, it can considerably interfere, especially in AT-rich genomes, with other selections on codon usage, <em>e.g.</em>, translational efficiency. It may also lead to difficulties in the recognition of other effects influencing codon bias, and an overestimation of protein coding sequences whose codon usage is subjected to adaptational selection.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/amino-acid" class="hw-term hw-article-keyword hw-article-keyword-amino-acid" rel="nofollow">amino acid</a></li><li class="kwd"><a href="/keyword/codon-usage" class="hw-term hw-article-keyword hw-article-keyword-codon-usage" rel="nofollow">codon usage</a></li><li class="kwd"><a href="/keyword/mutation" class="hw-term hw-article-keyword hw-article-keyword-mutation" rel="nofollow">mutation</a></li><li class="kwd"><a href="/keyword/selection" class="hw-term hw-article-keyword hw-article-keyword-selection" rel="nofollow">selection</a></li><li class="kwd"><a href="/keyword/synonymous-codons" class="hw-term hw-article-keyword hw-article-keyword-synonymous-codons" rel="nofollow">synonymous codons</a></li></ul><p id="p-3">Redundancy of the genetic code implies that there are more codons than amino acids. Consequently, many amino acids are encoded by more than one codon, which are known as synonymous codons. As a result, some substitutions between these codons are silent and do not change the coded amino acid. For example, in the case of the codons known as fourfold degenerated (4FD), the third codon positions can be freely changed to any nucleotide, without consequences for the coded amino acid, and subsequently for protein composition and function. However, synonymous codons are not used uniformly in real protein coding sequences (<em>e.g.</em>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Comeron 2004</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Grantham <em>et al.</em> 1980</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Ikemura 1985</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Plotkin and Kudla 2011</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Sharp and Li 1986</a>). Such preference of one synonymous codon over others is commonly known as codon usage bias (<a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Sharp and Li 1986</a>). Usage can differ for various genomes and genes within one genome, and even within a single gene.</p><p id="p-4">As far as the evolution of codon bias is concerned, two explanations, which are not mutually exclusive, have been proposed: directional mutations and specific selection (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bulmer 1991</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Hershberg and Petrov 2008</a>). From the mutational point of view, GC content is the strongest single determinant of codon usage in genomes (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chen <em>et al.</em> 2004</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Ermolaeva 2001</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Knight <em>et al.</em> 2001</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Li <em>et al.</em> 2015</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Muto and Osawa 1987</a>). Thus, in genomes with a high average GC content, the most frequent synonymous codons typically end with guanine or cytosine, whereas, in genomes with a low average GC content, they usually have adenine or thymine in their silent positions. The GC content also fluctuates periodically along vertebrate chromosomes, creating an isochore structure and influencing the local codon usage of genes (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Chen <em>et al.</em> 2004</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Fedorov <em>et al.</em> 2002</a>). In prokaryotic genomes, the chromosome-wide codon bias is related to various mutational pressures acting on differently replicating DNA strands, <em>i.e.</em>, the leading and lagging strands (<em>e.g.</em>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Frank and Lobry 1999</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Lobry 1996</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Mackiewicz <em>et al.</em> 1999b</a>,<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">c</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Morton and Morton 2007</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Mrazek and Karlin 1998</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Rocha <em>et al.</em> 1999</a>; <a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Tillier and Collins 2000</a>). As a result, GT-rich codons are usually over-represented in the leading strand genes, whereas AC-rich codons are found in the lagging strand. These codon biases, which are characteristics of genomes, enable the identification of potential genes that have been transferred horizontally (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Garcia-Vallve <em>et al.</em> 2003</a>).</p><p id="p-5">The bias resulting from mutational effects can be modified by many selectional factors. The first reported influence on the selection on codon bias was based on the observation that highly expressed sequences tend to use generally more frequent codons (<em>e.g.</em>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Akashi 2003</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bennetzen and Hall 1982</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Clarke 1970</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Duret and Mouchiroud 1999</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Ghaemmaghami <em>et al.</em> 2003</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Goetz and Fuglsang 2005</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Ikemura 1981</a>, <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">1985</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Kanaya <em>et al.</em> 1999</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Morton 1998</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Rocha 2004</a>). This was interpreted as an adaptation to the effectiveness of the translation process and accuracy of protein synthesis, and is known as codon adaptation or translational selection on codon usage. In addition, a substantial coincidence between gene copy number and the frequency of codons with a concentration of tRNA isoacceptors in their complementary anticodons was detected. The abundant tRNA isoacceptors, through their more fluent recognition of frequently used codons, enable the processivity of translational elongation (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Kanaya <em>et al.</em> 1999</a>; <a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Xia 1998</a>). Accordingly, there is a significant positive correlation between gene expression level and codon bias, and, likewise, a negative correlation between gene expression level and the rate of synonymous substitutions between compared sequences (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Eyre-Walker and Bulmer 1995</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Sharp and Li 1987</a>). The selection of synonymous codon usage can also result from selection for translational accuracy to reduce the costs of both missense and nonsense errors (<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Stoletzki and Eyre-Walker 2007</a>). The effectiveness of translation is also enhanced by clustering some synonymous codons in highly expressed genes, which is called codon co-occurrence bias (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cannarrozzi <em>et al.</em> 2010</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Shao <em>et al.</em> 2012</a>; <a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">Zhang <em>et al.</em> 2013</a>).</p><p id="p-6">However, the analysis of many genomes has revealed that there is a fraction of genes that show no evidence for translational selection linked to codon usage (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Carbone and Madden 2005</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">dos Reis <em>et al.</em> 2004</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Sharp <em>et al.</em> 2005</a>). This observation is not supported by recent, multi-genome, studies indicating that the translational selection for codon usage seems universal, at least in prokaryotes (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Hershberg and Petrov 2008</a>; <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Supek <em>et al.</em> 2010</a>) and plastids (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Suzuki and Morton 2016</a>). On the other hand, recently developed techniques measuring endogenous expression have shown that it is the initiation rather than the elongation process that limits the rate of protein production for most endogenous genes (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Ingolia 2014</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Ingolia <em>et al.</em> 2009</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Kertesz <em>et al.</em> 2010</a>; <a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Tuller <em>et al.</em> 2010</a>).</p><p id="p-7">Although translational selection is thought to be the dominant explanation of systematic variation in codon usage among genes (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chaney and Clark 2015</a>; <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Plotkin and Kudla 2011</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Quax <em>et al.</em> 2015</a>), several other factors related to codon bias have been put forward. One such factor is the formation of the functional native structure of proteins, which is realized by the preference of common codons in regions critical for protein folding and structure (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Oresic and Shalloway 1998</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Pechmann and Frydman 2013</a>; <a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Thanaraj and Argos 1996</a>; <a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">Zhou <em>et al.</em> 2009</a>). Furthermore, bias in synonymous codon usage within the coding sequence is also thought to be an additional layer of information influencing the stability of mRNA structure (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bartoszewski <em>et al.</em> 2010</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Lazrak <em>et al.</em> 2013</a>), mRNA half-life (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Presnyak <em>et al.</em> 2015</a>), and the effectiveness of transcription (<a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Xia 1996</a>).</p><p id="p-8">It was initially postulated that enrichment of the 5′ end of coding sequences in rare codons is intended to create a ramp at the 5′ end that prevents ribosome traffic jams further down the length of the mRNA, and increases translational efficiency (<a id="xref-ref-90-2" class="xref-bibr" href="#ref-90">Tuller <em>et al.</em> 2010</a>). Other authors proposed that rare codons cause a translational pause, which helps targeting and export of secreted proteins (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Clarke and Clark 2010</a>; <a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">Zalucki <em>et al.</em> 2009</a>). The ramp concept was revised in further studies showing that the reduced formation of stable mRNA structure is rather responsible for the higher translation rate (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bentele <em>et al.</em> 2013</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Goodman <em>et al.</em> 2013</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Kudla <em>et al.</em> 2009</a>), whereas a computational model predicted that this ramp is caused by rapid initiation of short genes rather than rare codons at the 5′ end of transcripts (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Shah <em>et al.</em> 2013</a>).</p><p id="p-9">The presence of many selective constraints on codon usage has consequences for the slower synonymous substitution rate of genes subjected to these selections, as demonstrated by the inverse correlation between the rate and the degree of codon adaptation (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Morton <em>et al.</em> 2002</a>; <a id="xref-ref-78-3" class="xref-bibr" href="#ref-78">Sharp and Li 1986</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Sharp <em>et al.</em> 1989</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Shields <em>et al.</em> 1988</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Sorhannus and Fox 1999</a>). An understanding of the rules in codon usage is also important in order to better optimize heterologous gene expression (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Gustafsson <em>et al.</em> 2004</a>), produce vaccines with attenuated viruses (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Coleman <em>et al.</em> 2008</a>), or find association of diseases with synonymous single nucleotide polymorphism (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Daidone <em>et al.</em> 2011</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Kimchi-Sarfaty <em>et al.</em> 2011</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Sauna and Kimchi-Sarfaty 2011</a>). Therefore, it is still important to better recognize mechanisms that induce codon usage biases in nature, to understand how the codon landscape evolves with time, and to search for other factors affecting codon bias.</p><p id="p-10">In his seminal work, <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Morton (2001)</a> postulated another important factor influencing synonymous codon usage. Interestingly, it is not related to direct adaptational selection of codon usage, but results only from a general selection of protein coding sequences at the amino acid level. His analyses showed that the composition of the silent sites of codons deviates from the composition of noncoding (neutral) sites even in the absence of selective differences between synonymous codons. This results from various probabilities of fixation of codon replacements. Morton nicely demonstrated that, after considering this type of selection, there are far fewer genes with codon adaptation bias than previously thought. This implies that selection acting on codon usage associated with translational efficiency may be overestimated. However, the study considered only four selected mutational processes, generating equal frequencies of complementary nucleotides.</p><p id="p-11">To further explore this subject, we created a mutation–selection model that includes the most general and unrestricted model of nucleotide substitutions, and examines a large number of possible mutation processes, generating almost 90,000 stationary distributions of codons. We applied an adapted version of the evolutionary optimization algorithm to find conditions in which mutation processes, together with selection at the amino acid level, maximizes the degree of codon bias (defined as deviation from uniform codon usage). The results demonstrate that the effect under study cannot be neglected.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Overview</h3><p id="p-12">One of the best ways to assess the influence of selection, at the amino acid level, on synonymous codon usage is to compare the codon frequency resulting from selection with the expected frequency without this constraint. To achieve that, we constructed a mutation–selection model similar to that of <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Morton (2001)</a>. This model is based on the theory of homogeneous and continuous-time Markov processes. In contrast to <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Morton (2001)</a>, we examined the most general nucleotide substitution model, which was superimposed on the codon selection process associated with the physicochemical properties of coded amino acids. Moreover, we tested almost 90,000 stationary distributions of codons, which corresponds to the mutational process. The effect of selection based on differences between relative codon usage was measured before and after the applied selection. Since a given nucleotide stationary distribution can be realized by many Markov processes, we applied an evolutionary based optimization algorithm in order to find conditions in which the differences between relative codon usage are maximized. As a result, we were able to determine the effect produced by the model with amino acid selection on synonymous sites. In the following sections, we describe in detail the stages of this approach, which is presented in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038125/-/DC1/FigureS1.pdf">Figure S1</a>. Finally, the theoretical calculations were compared with results provided by bacterial genome analyses.</p></div><div id="sec-3" class="subsection"><h3>Mutation process</h3><p id="p-13">To model the process of pure mutational pressure expressed by single nucleotide substitutions, we applied a homogeneous, stationary, and continuous-time Markov process. The process is described by a substitution rate matrix, <em>Q</em>, and stationary distribution of nucleotides, <em>π</em>. This approach is most commonly used in the description of DNA sequence evolution (<a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Yang 2006</a>). Here, we used the most general unrestricted model of nucleotide substitution, called UNREST (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) (<a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Yang 1994</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/483562/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/483562/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Substitution rate matrix <em>Q</em> for the unrestricted model of nucleotide substitutions (UNREST). The diagonals of <em>Q</em> are determined by the requirement that each row sum to zero</span><div class="sb-div caption-clear"></div></div></div><p id="p-15">The assumption about the stationarity of this process implies immediately the need to determine its exact stationary distribution, which should correspond in this case to the stationary frequency of nucleotides generated by the mutational process. In his work on this topic, <a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">Morton (2001)</a> considered only four selected mutational processes, with fixed nucleotide stationary distributions characterized by high A+T content. To formulate more general conclusions, and to assess the influence of selection, at the amino acid level, on various mutational processes, we analyzed 88,560 different nucleotide stationary distributions for the potential processes, which cover various mutational pressures. The frequencies of particular nucleotides range from 0.05 to 0.85, with 0.01 increments. Therefore, they form the set:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-16.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where every <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-17.gif"/></noscript></span></span> is chosen according to the following assumptions:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-16"><span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-18.gif"/></noscript></span></span></p></li><li id="list-item-2"><p id="p-17">for every <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-19.gif"/></noscript></span></span>, the Euclidean distance to the nearest stationary distribution, <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-20.gif"/></noscript></span></span>, = 0.02, and the difference between <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-21.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-22.gif"/></noscript></span></span> in one coordinate = 0.01 or 0.</p></li></ol><p id="p-18">As a result, we obtained a dense subset of all possible nucleotide stationary distributions.</p><p>To find the rates of the matrix <em>Q</em> for particular distributions, we rested on the assumption that, for the homogeneous, continuous-time, and stationary Markov process, the following set of equations holds:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-23.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>These equations can be reformulated easily into a system of three equations:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-24.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-25.gif"/></noscript></span></span>and</p><ul class="list-simple " id="list-2"><li id="list-item-3"><p id="p-20"><span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-26.gif"/></noscript></span></span> is composed of 12 substitution rates of matrix <em>Q</em>:</p></li><li id="list-item-4"><p id="p-21"><span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-27.gif"/></noscript></span></span> under the constraint:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-28.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span></p></li></ul><p>which is necessary to create a homogeneous, continuous-time Markov processes with fixed stationary distribution.</p><p id="p-22">The set of Equation 3 has infinitely many nontrivial solutions. Moreover, each solution can be described by a linear combination of independent vectors, <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-29.gif"/></noscript></span></span>, with coefficients <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-30.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-31.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>The <strong>β</strong> allows creation of the matrix <em>Q</em>, from which we derived a nucleotide transition probability matrix, <em>P</em>, by adopting the uniformization method (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Jensen 1953</a>; <a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Tijms 2003</a>). Generally, the uniformization procedure is used to transform the original continuous-time Markov process with nonidentical leaving rates into an equivalent of stochastic process, in which the transition epoch is generated by a suitable Poisson process with a fixed rate. Following this method, for a given <em>Q</em> with stationary distribution <em>π</em>, we could define a transition probability matrix <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-32.gif"/></noscript></span></span> assuming that:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-33.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-34.gif"/></noscript></span></span> Clearly, <em>P</em> is the transition probability matrix describing the Markov chain with stationary distribution <em>π</em>, which is the same for the continuous case. Moreover, the sum of all its off-diagonal elements is equal to one. This representation turned out to be very useful in our mutation–selection model because it allowed the construction of quite a large set of possible nucleotide transition probability matrices, <em>P</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038125/-/DC1/FigureS1.pdf">Figure S1</a>), under relatively weak mathematical assumptions.</p><p id="p-23">Obviously, we are interested in a codon substitution process, and, for this reason, we calculated a codon <em>k</em> to codon <em>l</em> transition probability matrix <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-35.gif"/></noscript></span></span> using the nucleotide transition probability matrix <em>P</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038125/-/DC1/FigureS1.pdf">Figure S1</a>). In the matrix <em>P*</em>, we took into account all independent substitutions between codons resulting from a single nucleotide change. The Markov chain defined by <em>P*</em> is also stationary, with codon stationary distribution <em>π<sup>cod</sup></em>, which is in accordance with the following system of equations:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-36.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>Moreover, under the assumptions presented above, the stationary relative frequencies of 4FD are determined solely by the stationary distribution <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-37.gif"/></noscript></span></span></p></div><div id="sec-4" class="subsection"><h3>Process of selection</h3><p id="p-24">Similarly to <a id="xref-ref-56-5" class="xref-bibr" href="#ref-56">Morton (2001)</a>, we were interested in a model of sequence evolution that combines mutation and selection at the amino acid level. At the selection stage, we introduced the acceptance matrix, <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-38.gif"/></noscript></span></span> which contains probabilities that a change of amino acid <em>m</em> to amino acid <em>n</em> will be “accepted.” All diagonals of matrix <em>D</em> are equal to one, which means no selective costs of substitutions between synonymous codons. In this work, we employed the acceptance matrix presented in <a id="xref-ref-56-6" class="xref-bibr" href="#ref-56">Morton (2001)</a>, which is based on <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Grantham’s (1974)</a> chemical similarity matrix (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038125/-/DC1/FigureS1.pdf">Figure S1</a>). Additionally, we took into account two cases involving substitutions to and from stop codons. In the first case, we assumed that such mutations are lethal (SL), and we set the probability of acceptance to zero. In the second case, the probability of acceptance of such substitutions was equal to the minimal probability in the matrix <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-39.gif"/></noscript></span></span> (SM).</p><p id="p-25">As a consequence, we defined a general model, including the mutation and selection processes, in the same way as in <a id="xref-ref-56-7" class="xref-bibr" href="#ref-56">Morton (2001)</a>. This model is expounded by a codon to codon transition probability matrix <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-40.gif"/></noscript></span></span> Furthermore, every codon to codon substitution <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-41.gif"/></noscript></span></span> is defined by the following equation:<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-42.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>where <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-43.gif"/></noscript></span></span>is the transition probability between codons <em>k</em> and <em>l</em>, whereas <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-44.gif"/></noscript></span></span> is the probability of accepting a change from amino acid <em>m</em> to amino acid <em>n</em> coded by codons <em>k</em> and <em>l</em>, respectively. Obviously, all diagonals in matrix <em>C</em> are set to make the rows sum to one. In addition, the Markov chain described by <em>C</em> has its own stationary distribution <em>π<sup>sel</sup></em>.</p></div><div id="sec-5" class="subsection"><h3>Measure of selection strength</h3><p id="p-26">The strength of selection at the amino acid level, which affects the composition in neutral sites of codons, was assessed for each stationary distribution <em>π</em> by the normalized difference between the relative frequency of 4FD codons after selection, and their expected frequency resulting only from a mutation process:<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-45.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span>where <em>s</em> means a group of 4FD codons coding for one amino acid; <em>s<sub>i</sub></em> is a codon from this group, in which a nucleotide <em>i</em> occurs at the third position; <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-46.gif"/></noscript></span></span>is the stationary frequency of this codon group after selection (<em>sel</em>);<span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-47.gif"/></noscript></span></span> is the stationary frequency of codon <em>s<sub>i</sub></em> after selection; <em>π<sub>i</sub></em> is the relative stationary frequency of codon <em>s<sub>i</sub></em> obtained only from the mutation process. We analyzed all five groups of 4FD codons, and calculated the summarized effect of the selection at the amino acid level on these groups for each stationary distribution <em>π</em>:<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-48.gif"/></noscript></span><span class="disp-formula-label">(10)</span></span>where <em>S</em> is the set of all groups of 4FD codons <em>s</em>. Clearly, large values of <em>F<sub>π</sub></em> suggest a strong impact of selection on the usage of 4FD codons, whereas values equal to zero indicate a lack of such an effect on the relative frequencies of 4FD codons.</p></div><div id="sec-6" class="subsection"><h3>Simulation procedure</h3><p id="p-27">A nucleotide stationary distribution <em>π</em> can be realized by many Markov processes described by various substitution matrices, <em>P<sub>π</sub></em>, which can imply differences in stationary frequencies of codons after the selection <em>π<sup>sel</sup></em>, and, consequently, different <em>F<sub>π</sub></em> values. To deal with this problem, we decided to find the probability <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-49.gif"/></noscript></span></span> that maximizes the <em>F<sub>π</sub></em> measure. The maximum value of <em>F<sub>π</sub></em> was denoted by <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-50.gif"/></noscript></span></span> Consequently, we were able to assess the range of selection strength at the amino acid level on synonymous codon usage for a given nucleotide stationary distribution <em>π</em>.</p><p id="p-28">The task of finding <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-51.gif"/></noscript></span></span> is, in fact, an example of a single objective optimization problem, where <em>F<sub>π</sub></em> is a fitness function. Therefore, we decided to use the Evolutionary Strategies (ES) approach (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">De Jong <em>et al.</em> 1997</a>), which is a commonly used technique in optimization problems when the solution is hard to find analytically. For each nucleotide distribution <em>π</em>, we ran simulations with a population of 100 random candidate solutions according to ES principles. At the beginning of each simulation run, our candidate solutions were, in fact, substitution rate matrices <em>Q</em> selected at random according to the procedure described by Equation 3 and condition (4). In every simulation step, we applied mutation and selection operators. For a given rate matrix (individual), the process of mutation was realized by a random modification of its vector of coefficients <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-52.gif"/></noscript></span></span> according to the normal distribution <em>N</em>(0,<em>σ</em>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038125/-/DC1/FigureS1.pdf">Figure S1</a>). The <em>σ</em> parameter was tuned during preliminary simulation tests to obtain a quick convergence to the satisfactory solution. The crossover operator used in this problem was a modified version of the Linear Crossover LBGA (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Schlierkamp-voosen and Muhlenbein 1994</a>). This produced an offspring that was a random linear combination of its parents in terms of Equations 3 and 5. Understandably, at the end of these procedures, we checked the quality of newly produced offspring, to find out whether they possess a proper representation, and fulfill condition (4). In the next step, we made transformations of substitution nucleotide matrices <em>Q</em> to <em>P</em>, and next to substitution codon matrices <em>P*</em> and <em>C</em>. This was done according to the procedure described in the previous sections. Therefore, we were able to calculate the codon stationary distribution after selection <em>π<sup>sel</sup></em>, and values of the fitness function <em>F<sub>π</sub></em>. Finally, we used tournament selection as the selection operator. Depending on the assumed fitness function, the algorithm selected individuals (rate matrices) that maximized the measure <em>F<sub>π</sub></em>. The main program was developed by the authors (P.B., D.M. M.W., and P.M) in C++ language. The stationary vectors <em>π<sup>sel</sup></em> were calculated using the Armadillo library (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Sanderson 2010</a>).</p></div><div id="sec-7" class="subsection"><h3>Analysis of deviation in codon usage in protein coding sequences</h3><p id="p-29">The values determined for the <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-53.gif"/></noscript></span></span> measure were compared with an analogous parameter calculated for 4FD codons, using protein coding sequences from 4879 fully sequenced bacterial genomes, whose sequences and annotations were downloaded from the NCBI database (<a href="ftp://ftp.ncbi.nlm.nih.gov/genomes">ftp://ftp.ncbi.nlm.nih.gov/genomes</a>). We examined separately the genes located on the leading and the lagging DNA strands. The boundaries between the DNA strands were determined according to DNA walk methods, using DNA asymmetry parameters, <em>i.e.</em>, the differences in complementary nucleotides: [G–C] and [A–T] (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Kowalczuk <em>et al.</em> 2001</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Mackiewicz <em>et al.</em> 1999a</a>). For these data, we calculated the summarized deviation from the expectation in the codon usage for all 4FD groups <em>S</em>. Clearly, this corresponds to Equations 9 and 10, <em>i.e.</em>:<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-54.gif"/></noscript></span><span class="disp-formula-label">(11)</span></span>where:<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-55.gif"/></noscript></span><span class="disp-formula-label">(12)</span></span>is the normalized difference between the relative frequencies of 4FD codons in one group, and their expected frequencies. Therefore,<span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-56.gif"/></noscript></span></span>is the observed frequency of a 4FD codon <em>s<sub>i</sub></em>, with a nucleotide <em>i</em> at the third position,<span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-57.gif"/></noscript></span></span> is the frequency of all codons in the group, and <em>e<sub>i</sub></em> is the expected frequency established as the average of relative frequencies of all 4FD codons with a nucleotide <em>i</em> at the third codon position.</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-30"><a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038125/-/DC1/FigureS1.pdf">Figure S1</a> illustrates the procedure leading to the assessment of the selection strength, at the amino acid level, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038125/-/DC1/FigureS2.pdf">Figure S2</a> compares nucleotide substitution probabilities for 5% of top matrices that maximized the values of <em>F<sub>π</sub></em>.</p></div></div><div class="section results-discussion" id="sec-9"><h2 class="">Results and Discussion</h2><div id="sec-10" class="subsection"><h3>The summarized effect of selection on all codon groups</h3><p id="p-31">In total, we performed 88,560 simulations to find the maximum <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-58.gif"/></noscript></span></span> values for the normalized difference between the relative frequency of 4FD codons after selection on amino acids, and their expected frequency triggered only by a mutation process. This parameter expresses the most extreme impact of selection at the amino acid level on the usage of 4FD codons. This impact can be found for a given mutation process with its specific nucleotide stationary distribution <em>π</em>. Selection with lethal stop codons’ substitutions (SL) and the variant with the minimal acceptance probability of such substitutions (SM) were studied separately.</p><p id="p-32">The applied optimization algorithm enabled an effective maximization of the fitness function <em>F<sub>π</sub></em> for all the nucleotide stationary distributions <em>π</em> under study. Our results indicate that it is possible to find transition probability matrices <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-59.gif"/></noscript></span></span> for each nucleotide stationary distribution <em>π</em>, which maximize the impact of such a selection measured by <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-60.gif"/></noscript></span></span> Generally, depending on the applied stationary distribution <em>π</em>, <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-61.gif"/></noscript></span></span>varied from 0.25 to over 9.22 under SL variant, and from 0.26 to over 9.1 under SM variant.</p><p id="p-33">The nucleotide stationary distributions for which the extreme values of <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-62.gif"/></noscript></span></span> were found are presented in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. The findings indicate that the largest impact of selection at the amino acid level on deviations in 4FD codons usage is for mutation processes that generate thymine with high frequency at the expense of guanine and cytosine. The next most frequent nucleotide is adenine. On the other hand, the smallest <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-63.gif"/></noscript></span></span> is for nucleotide distributions with high content of cytosine, and next guanine. To systematically analyze the relationship of <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-64.gif"/></noscript></span></span> to the nucleotide stationary distributions, we carried out additional studies. Since the results for models SL and SM were very similar, we focused on the latter.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/483596/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/483596/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Nucleotide stationary distributions of mutation process for which extreme <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/T2/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/T2/embed/mml-math-65.gif"/></noscript></span></span>values were found</span><div class="sb-div caption-clear"></div></div></div><p id="p-35">The results for extreme values are supported by the radar chart, in which two sets of 100 stationary distributions responsible for the highest and the lowest values of <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-67.gif"/></noscript></span></span> are presented (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). It can be seen that the <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-68.gif"/></noscript></span></span>value is clearly related to the frequency of nucleotides in the stationary distribution. The excess of thymine, and then adenine, leads to the highest values of <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-69.gif"/></noscript></span></span> whereas the lowest values of <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-70.gif"/></noscript></span></span> are observed for the domination of cytosine, and then guanine.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/967/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of two sets of 100 stationary distributions for which  (the normalized difference between the relative frequency of 4FD codons after selection on amino acids, and their expected frequency resulting only from a mutation process) takes the highest (red) and the lowest values (green). The  is the highest for the distributions with the high frequency of thymine and adenine, respectively, whereas the lowest for the distributions rich in cytosine and guanine, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2365689" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of two sets of 100 stationary distributions for which &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-71&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (the normalized difference between the relative frequency of 4FD codons after selection on amino acids, and their expected frequency resulting only from a mutation process) takes the highest (red) and the lowest values (green). The &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-72&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; is the highest for the distributions with the high frequency of thymine and adenine, respectively, whereas the lowest for the distributions rich in cytosine and guanine, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/967/F1.medium.gif" width="437" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/3/967/F1.medium.gif" width="437" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/967/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/967/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483632" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-36" class="first-child">Comparison of two sets of 100 stationary distributions for which <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F1/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F1/embed/mml-math-71.gif"/></noscript></span></span> (the normalized difference between the relative frequency of 4FD codons after selection on amino acids, and their expected frequency resulting only from a mutation process) takes the highest (red) and the lowest values (green). The <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F1/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F1/embed/mml-math-72.gif"/></noscript></span></span> is the highest for the distributions with the high frequency of thymine and adenine, respectively, whereas the lowest for the distributions rich in cytosine and guanine, respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">In order to study how the possible maximum deviation in the usage of 4FD codons <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-73.gif"/></noscript></span></span> depends on the whole range of nucleotide stationary distributions in the combination of two nucleotides, we made the Wafer map, in which the gradient coloring corresponds to the <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-74.gif"/></noscript></span></span> value (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Dark green denotes the lowest values, and dark brown the highest values, of <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-75.gif"/></noscript></span></span> The relationships are clearly nonlinear. The highest values are observed for distributions with high frequency of thymine, and a substantially smaller contribution of other nucleotides, especially for <em>π<sub>T</sub></em> &gt; 0.6 and <em>π<sub>A</sub></em> in the range from 0 to 0.25, as well as for <em>π<sub>T</sub></em> &gt; 0.7 and <em>π<sub>G</sub></em> &lt; 0.2 or <em>π<sub>C</sub></em> &lt; 0.2. The increase in <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-76.gif"/></noscript></span></span> also correlates with the high frequency of adenine <em>π<sub>A</sub></em> &gt; 0.7, but only together with the decline of guanine and cytosine frequencies to values &lt;0.2. However, there is a growth of <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-77.gif"/></noscript></span></span>also for <em>π<sub>A</sub></em> from 0.2 to 0.4 with the excess of guanine in the range 0.5–0.7.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/967/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between the  value and combination of two nucleotides presented as colored Wafer maps. The colors correspond to the value of  which depends on the frequency of the compared nucleotides. Dark green corresponds the lowest values, and dark brown the highest values of  Its highest values are for the high content of thymine and adenine, with simultaneous decrease in the guanine and cytosine frequency. The lowest values are for the low frequency of A and T, as well as for moderate content of G and C." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2365689" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Relationship between the &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-78&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; value and combination of two nucleotides presented as colored Wafer maps. The colors correspond to the value of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-79&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; which depends on the frequency of the compared nucleotides. Dark green corresponds the lowest values, and dark brown the highest values of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-80&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; Its highest values are for the high content of thymine and adenine, with simultaneous decrease in the guanine and cytosine frequency. The lowest values are for the low frequency of A and T, as well as for moderate content of G and C.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/967/F2.medium.gif" width="393" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/3/967/F2.medium.gif" width="393" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/967/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/967/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483543" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-38" class="first-child">Relationship between the <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F2/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F2/embed/mml-math-78.gif"/></noscript></span></span> value and combination of two nucleotides presented as colored Wafer maps. The colors correspond to the value of <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F2/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F2/embed/mml-math-79.gif"/></noscript></span></span> which depends on the frequency of the compared nucleotides. Dark green corresponds the lowest values, and dark brown the highest values of <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F2/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F2/embed/mml-math-80.gif"/></noscript></span></span> Its highest values are for the high content of thymine and adenine, with simultaneous decrease in the guanine and cytosine frequency. The lowest values are for the low frequency of A and T, as well as for moderate content of G and C.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">The lowest <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-81.gif"/></noscript></span></span> values are obtained for the substitution matrices generating a low frequency of adenine (<em>π<sub>A</sub></em> &lt; 0.1), with <em>π<sub>T</sub></em> &lt; 0.5, <em>π<sub>G</sub></em> from 0.3 to 0.6 and <em>π<sub>C</sub></em> from 0.2 to 0.4 (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-82.gif"/></noscript></span></span> has low values also for the frequency of cytosine in the range from 0.45 to 0.65, when the content of thymine is very small (<em>π<sub>T</sub></em> &lt; 0.1), and for guanine from 0.4 to 0.6, when thymine shows a moderate content of 0.3–0.5. The values of <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-83.gif"/></noscript></span></span>are also low for <em>π<sub>G</sub></em> from 0.2 to 0.5, and <em>π<sub>C</sub></em> from 0.4 to 0.7.</p><p id="p-40">We also analyzed the impact of stationary frequencies of particular nucleotides on the values of<span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-84.gif"/></noscript></span></span> Therefore, we created the sets <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-85.gif"/></noscript></span></span> which are defined in the following way:<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-86.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-87.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-88.gif"/></noscript></span></span> For example, <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-89.gif"/></noscript></span></span> is the set of all stationary distributions, <em>π</em>, when the frequency of adenine is 0.05, <em>i.e.</em><span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-90.gif"/></noscript></span></span> whereas frequencies of other nucleotides sum up to 0.95, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-91.gif"/></noscript></span></span> The sets <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-92.gif"/></noscript></span></span> were described in the same way.</p><p id="p-41">Following this approach, we decided to calculate <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-93.gif"/></noscript></span></span> <em>i.e.</em>, the median value of <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-94.gif"/></noscript></span></span> for every nucleotide <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-95.gif"/></noscript></span></span>, and the stationary distribution <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-96.gif"/></noscript></span></span> separately. The main reason for using the median can be explained by the fact that it is an estimator of location parameter that is most resistant to outliers. Therefore, it is a useful and stable measure with which to detect general tendencies in large data sets. In addition, <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-97.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-98.gif"/></noscript></span></span> is a function of <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-99.gif"/></noscript></span></span> for every <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-100.gif"/></noscript></span></span> In other words, the median was calculated from <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-101.gif"/></noscript></span></span> values that were derived from substitution models generating nucleotide stationary distributions with the fixed frequency of one nucleotide and random frequencies of others.</p><p id="p-42">In <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, we illustrate the dependence of the median value of <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-102.gif"/></noscript></span></span> on the stationary frequencies of four nucleotides. Interestingly, the relationships are not linear, and <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-103.gif"/></noscript></span></span> shows a similar course for complementary nucleotides, especially for guanine and cytosine. The median value of <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-104.gif"/></noscript></span></span> starts from a relatively high value for small frequencies of G and C, and decreases gradually with their growth, reaching a minimum for their frequencies ∼0.36. After that, the median rises steadily, reaching its maximum for the highest frequencies of G and C. However, for the adenine frequency, <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-105.gif"/></noscript></span></span> grows steadily to 0.6, and then increases rapidly for the highest frequencies. In the case of thymine, the median remains quite constant until it reaches 0.4, and then also quickly increases. The median values of <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-106.gif"/></noscript></span></span> for the fixed frequencies of G and C are generally lower than for A and T, with the exception of the frequency of G and C &lt;0.2.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/967/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dependence of median value of  i.e.,  on stationary frequencies of four nucleotides &#x3C0;. The median was calculated from  values that were derived from substitution models generating nucleotide stationary distributions, with the given fixed frequency of one nucleotide &#x3C0;i and random frequencies of others. The dots represent exact values of  whereas lines are the best approximation based on generalized additive models with integrated smoothness estimation. The  depends nonlinearly on the stationary distribution of particular nucleotides. Its strongest increase is for the growth of A and T." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2365689" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Dependence of median value of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-107&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; &lt;em&gt;i.e.&lt;/em&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-108&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; on stationary frequencies of four nucleotides &lt;em&gt;&#x3C0;&lt;/em&gt;. The median was calculated from &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-109&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; values that were derived from substitution models generating nucleotide stationary distributions, with the given fixed frequency of one nucleotide &lt;em&gt;&#x3C0;&lt;sub&gt;i&lt;/sub&gt;&lt;/em&gt; and random frequencies of others. The dots represent exact values of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-110&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; whereas lines are the best approximation based on generalized additive models with integrated smoothness estimation. The &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-111&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; depends nonlinearly on the stationary distribution of particular nucleotides. Its strongest increase is for the growth of A and T.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/967/F3.medium.gif" width="354" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/3/967/F3.medium.gif" width="354" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/967/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/967/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483677" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-43" class="first-child">Dependence of median value of <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F3/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F3/embed/mml-math-107.gif"/></noscript></span></span> <em>i.e.</em>, <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F3/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F3/embed/mml-math-108.gif"/></noscript></span></span> on stationary frequencies of four nucleotides <em>π</em>. The median was calculated from <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F3/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F3/embed/mml-math-109.gif"/></noscript></span></span> values that were derived from substitution models generating nucleotide stationary distributions, with the given fixed frequency of one nucleotide <em>π<sub>i</sub></em> and random frequencies of others. The dots represent exact values of <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F3/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F3/embed/mml-math-110.gif"/></noscript></span></span> whereas lines are the best approximation based on generalized additive models with integrated smoothness estimation. The <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F3/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F3/embed/mml-math-111.gif"/></noscript></span></span> depends nonlinearly on the stationary distribution of particular nucleotides. Its strongest increase is for the growth of A and T.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">Since the median value of <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-112.gif"/></noscript></span></span> depends on the complementary nucleotides in a similar way, we examined the dependence of <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-113.gif"/></noscript></span></span> on the aggregated frequencies of the nucleotides, A+T and G+C, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-114.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-115.gif"/></noscript></span></span> They are both defined in the analogous way. For example, in the case of <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-116.gif"/></noscript></span></span> we have:<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-117.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-118.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-119.gif"/></noscript></span></span> In this case, we observed a sigmoidal increase of <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-120.gif"/></noscript></span></span> with A+T content (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>), and the opposite trend for G+C (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/967/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dependence of median value of  i.e.,  on stationary content of: adenine + thymine (A), guanine + cytosine (B), adenine and thymine (C), and guanine and cytosine (D) with equal frequencies, as well as purines (E) and pyrimidines (F). There is a clear nonlinear relationship with the minimum for equal proportions of purines and pyrimidines." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2365689" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Dependence of median value of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-121&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; &lt;em&gt;i.e.&lt;/em&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-122&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; on stationary content of: adenine + thymine (A), guanine + cytosine (B), adenine and thymine (C), and guanine and cytosine (D) with equal frequencies, as well as purines (E) and pyrimidines (F). There is a clear nonlinear relationship with the minimum for equal proportions of purines and pyrimidines.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/967/F4.medium.gif" width="151" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/3/967/F4.medium.gif" width="151" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/967/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/967/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483558" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-45" class="first-child">Dependence of median value of <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F4/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F4/embed/mml-math-121.gif"/></noscript></span></span> <em>i.e.</em>, <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F4/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F4/embed/mml-math-122.gif"/></noscript></span></span> on stationary content of: adenine + thymine (A), guanine + cytosine (B), adenine and thymine (C), and guanine and cytosine (D) with equal frequencies, as well as purines (E) and pyrimidines (F). There is a clear nonlinear relationship with the minimum for equal proportions of purines and pyrimidines.</p><div class="sb-div caption-clear"></div></div></div><p id="p-46">The similar dependence of <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-123.gif"/></noscript></span></span> on the frequencies of complementary nucleotides justifies considering also simpler models and stationary distributions, <em>e.g.</em>, assuming equal frequencies of the complementary nucleotides: <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-124.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-125.gif"/></noscript></span></span> This assumption was tested by <a id="xref-ref-56-8" class="xref-bibr" href="#ref-56">Morton (2001)</a> on the example of four mutation-selection models. Here, we included a generalization of this model, analyzing a wider range of possible nucleotide frequencies <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-126.gif"/></noscript></span></span> The <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-127.gif"/></noscript></span></span> shows an exponential growth from 0.541 (for <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-128.gif"/></noscript></span></span>) to 5.010 (for <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-129.gif"/></noscript></span></span>), with an increase in the A and T frequency (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4, C and D</a>).</p><p id="p-47">The results show that a strong relationship exists between <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-130.gif"/></noscript></span></span>, and the frequencies of complementary nucleotides, regardless of the type of model assumed. Therefore, we can infer that the impact of selection, at the amino acid level, on the usage of 4FD codons is connected with the structure of the stationary distribution generated by its mutation accumulation process. Generally, selection is responsible for the high deviation in the synonymous codon usage when the nucleotide substitution process generates a high frequency of A and T nucleotides, while the processes with a high frequency of G+C in their stationary distributions reduces the impact of selection.</p><p id="p-48">Surprisingly, we observed a completely different dependence of <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-131.gif"/></noscript></span></span> on the total frequency of purines (A+G) and pyrimidines (C+T) in the assumed stationary distributions, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-132.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-133.gif"/></noscript></span></span> This dependence turned out to be nonlinear and nonmonotonic, in contrast to the complementary nucleotides. In the case of purines, <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-134.gif"/></noscript></span></span> contains the local maximum at about <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-135.gif"/></noscript></span></span>, and then it drops below 1.5 at about <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-136.gif"/></noscript></span></span> reaching the global minimum (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4, E and F</a>). Next, it significantly increases to the global maximum at <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-137.gif"/></noscript></span></span> with a value of ∼3.9. The dependence of <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-138.gif"/></noscript></span></span> on pyrimidines shows a symmetrical course (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4, E and F</a>), with the global maximum at <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-139.gif"/></noscript></span></span> the global minimum at <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-140.gif"/></noscript></span></span>, and the local maximum at <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-141.gif"/></noscript></span></span></p></div><div id="sec-11" class="subsection"><h3>The effect of selection on particular codon groups</h3><p id="p-49">The results presented above referred to the summarized effect of the selection at the amino acid level on all five groups of 4FD codons. However, it is interesting to analyze how selection influences deviation in the expected relative usage of particular groups of the codons <em>s</em> for particular nucleotide stationary distributions <em>π</em>, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-142.gif"/></noscript></span></span>The extreme values of <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-143.gif"/></noscript></span></span>found for the particular codon groups are shown in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. The results demonstrate that the biggest deviation concerns codons coding glycine, whereas the smallest is for valine codons. The other three groups of codons have comparable values. This effect could be explained by differences in the acceptance probabilities of substitutions of amino acids coded by these codon blocks. However, differences between groups of codons disappear in the case of the lowest <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-144.gif"/></noscript></span></span> values, where we observed very similar values of <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-145.gif"/></noscript></span></span></p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/483564/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/483564/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">The highest and the lowest values of <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/T3/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/T3/embed/mml-math-146.gif"/></noscript></span></span> which were found for five 4FD codon blocks denoted by coded amino acids</span><div class="sb-div caption-clear"></div></div></div><p id="p-51">We additionally tested the deviation in the expected relative usage of particular codon groups <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-147.gif"/></noscript></span></span> as a function of stationary nucleotide distribution for the SM model (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). As in our analysis of the summarized effect on these groups, we likewise calculated the median of <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-148.gif"/></noscript></span></span> using the values that were obtained from the substitution matrices generating nucleotide stationary distributions with a fixed frequency of one nucleotide and random frequencies of others. Similarly to the global effect, the same tendency in the case of adenine and thymine was noted. The median of <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-149.gif"/></noscript></span></span>increases with a comparable intensity for all codon groups as a function of <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-150.gif"/></noscript></span></span>but, in the case of <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-151.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-152.gif"/></noscript></span></span> for glycine codons grow substantially faster than other codon blocks (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). The trends are different for the codon blocks depending on guanine and cytosine frequencies. In the case of guanine frequency, <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-153.gif"/></noscript></span></span> decreases substantially for the Gly codon group with <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-154.gif"/></noscript></span></span> growth, in contrast to the other codon blocks, whose <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-155.gif"/></noscript></span></span> values begin to increase at <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-156.gif"/></noscript></span></span> Two groups of codons can be distinguished when the relationship between <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-157.gif"/></noscript></span></span> and cytosine frequency is taken into account. One group, including the codons for Pro, Ala and Thr, shows a decreasing trend in their <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-158.gif"/></noscript></span></span> whereas the median values of <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-159.gif"/></noscript></span></span> of the other group, containing the Gly and Val codons, increase substantially with <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-160.gif"/></noscript></span></span></p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/967/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dependence of the median value of  i.e.,  for 4FD codon groups (assigned by their coded amino acids) on the stationary frequencies of four nucleotides &#x3C0;: adenine (A), thymine (B), guanine (C), and cytosine (D). The dots represent exact values of  whereas lines are the best approximation based on generalized additive models with integrated smoothness estimation. The median value depends differently on the codon groups and nucleotides." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2365689" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Dependence of the median value of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-161&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; &lt;em&gt;i.e.&lt;/em&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-162&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for 4FD codon groups (assigned by their coded amino acids) on the stationary frequencies of four nucleotides &lt;em&gt;&#x3C0;&lt;/em&gt;: adenine (A), thymine (B), guanine (C), and cytosine (D). The dots represent exact values of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-163&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; whereas lines are the best approximation based on generalized additive models with integrated smoothness estimation. The median value depends differently on the codon groups and nucleotides.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/967/F5.medium.gif" width="437" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/3/967/F5.medium.gif" width="437" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/967/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/967/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483660" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-52" class="first-child">Dependence of the median value of <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F5/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F5/embed/mml-math-161.gif"/></noscript></span></span> <em>i.e.</em>, <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F5/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F5/embed/mml-math-162.gif"/></noscript></span></span> for 4FD codon groups (assigned by their coded amino acids) on the stationary frequencies of four nucleotides <em>π</em>: adenine (A), thymine (B), guanine (C), and cytosine (D). The dots represent exact values of <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F5/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F5/embed/mml-math-163.gif"/></noscript></span></span> whereas lines are the best approximation based on generalized additive models with integrated smoothness estimation. The median value depends differently on the codon groups and nucleotides.</p><div class="sb-div caption-clear"></div></div></div><p id="p-53">The median of <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-164.gif"/></noscript></span></span> for all codon blocks shows a concordant increasing trend with A+T content (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4A</a>), and decreasing for G+C (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4B</a>) for all codons’ groups. The smallest deviation was observed in the codons for valine. As expected, <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-165.gif"/></noscript></span></span> also grows for all codon groups with A and T frequencies, with the assumption that <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-166.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-167.gif"/></noscript></span></span> (data not shown).</p><p id="p-54">The analysis of <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-168.gif"/></noscript></span></span> for particular codon groups well explains the nonlinear relationship between the summarized effect of selection at the amino acid level on all 4FD codons, and the purines and pyrimidines content (cf. <a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4, E and F</a> and <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). The median of <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-169.gif"/></noscript></span></span> for the Thr, Pro and Ala codons increases with A+G frequency, whereas <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-170.gif"/></noscript></span></span> for the Gly codons decreases. This measure for Val codons also declines with purine content, but reaches its minimum at <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-171.gif"/></noscript></span></span> and then goes up. The relationships between <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-172.gif"/></noscript></span></span> and C+T content are mirrored. The superposition of these various trends for particular codon groups leads to the nonlinear course of the relationship for the global measure <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-173.gif"/></noscript></span></span> for all synonymous codons (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4, E and F</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/967/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dependence of the median value of  i.e.,  for 4FD codon groups (assigned by their coded amino acids) on the stationary frequencies of purines (A) and pyrimidines (B). The dots represent exact values of  whereas lines are the best approximation based on generalized additive models with integrated smoothness estimation. The groups of codons response differently to the frequencies." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2365689" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Dependence of the median value of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-174&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; &lt;em&gt;i.e.&lt;/em&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-175&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for 4FD codon groups (assigned by their coded amino acids) on the stationary frequencies of purines (A) and pyrimidines (B). The dots represent exact values of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-176&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; whereas lines are the best approximation based on generalized additive models with integrated smoothness estimation. The groups of codons response differently to the frequencies.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/967/F6.medium.gif" width="440" height="419"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/3/967/F6.medium.gif" width="440" height="419"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/967/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/967/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483587" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-55" class="first-child">Dependence of the median value of <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F6/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F6/embed/mml-math-174.gif"/></noscript></span></span> <em>i.e.</em>, <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F6/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F6/embed/mml-math-175.gif"/></noscript></span></span> for 4FD codon groups (assigned by their coded amino acids) on the stationary frequencies of purines (A) and pyrimidines (B). The dots represent exact values of <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F6/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F6/embed/mml-math-176.gif"/></noscript></span></span> whereas lines are the best approximation based on generalized additive models with integrated smoothness estimation. The groups of codons response differently to the frequencies.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Characteristics of mutational probability matrices maximizing selection effect</h3><p id="p-56">The present study revealed that the effect of selection, at the amino acid level, on synonymous codon usage depends strongly on the nucleotide stationary distributions, which are the result of mutational processes described by substitution probability matrices. Therefore, it is interesting to check what types of nucleotide substitutions are responsible for enhancing this effect. <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> presents the best matrix, which, together with selection on nonsynonymous substitutions, maximizes <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-177.gif"/></noscript></span></span> The matrix is characterized by the most frequent substitution C→G, and next G→A and C→A, whereas the rarest substitution in the matrix is A→G (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/483683/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/483683/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Transition probability matrix <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/T4/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/T4/embed/mml-math-178.gif"/></noscript></span></span> that maximizes the effect of selection on the relative usage of 4FD codons</span><div class="sb-div caption-clear"></div></div></div><p id="p-58">To check if these properties are universal, we compared 5% (<em>i.e.</em>, 4428) of top matrices that generated the highest values of <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-181.gif"/></noscript></span></span> (shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038125/-/DC1/FigureS2.pdf">Figure S2</a>). The maximizing matrices are characterized by a higher probability of staying the same for adenine and thymine than for guanine and cytosine. Their most frequent substitutions are C→A and C→G. This may result from the fact that these transitions belong to the most regular of all the 120 possible nonsynonymous and single-nucleotide mutations of 4FD codons. Each of them occurs in 16 cases, and they constitute, in total, 27% of the possible substitutions. On the other hand, the lowest probabilities show substitutions A→G and A→T (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038125/-/DC1/FigureS2.pdf">Figure S2</a>). They are the least frequent mutations of all possible nonsynonymous mutations involving 4FD codons. Each of them applies in only four cases.</p><p id="p-59">Generally, the maximizing matrices have a tendency to generate more adenine and thymine at the expense of guanine and cytosine. These findings correspond well to the relationships observed between <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-182.gif"/></noscript></span></span> and nucleotide stationary distributions, indicating a greater deviation in synonymous codon usage for nucleotide distributions rich in A and T (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4</a>).</p><p id="p-60">We also noted that the maximizing matrices are characterized by a preponderance of transversions over transitions, which enhances the impact of selection on relative synonymous codon usage. The median and quartile range of transitions to transversions ratio is 0.207 [0.133–0.329]. It is &lt;0.5 when there is no bias toward either transitions or transversions because there are twice as many possible transversions as transitions. This may be attributed to lower acceptance probabilities for amino acid substitutions in <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Grantham’s (1974)</a> matrix employed in the research, which result from transversions rather than from transitions of the corresponding codons. Actually, the mean acceptance probability for transversions and transitions is 0.409 and 0.538, respectively. The difference is statistically significant in the Mann–Whitney test, with <em>P</em> = 0.00001. A higher rate of transversions can, understandably, increase the rare substitutions of codons, and lead to a marked bias in the relative usage of 4FD codons.</p><p id="p-61">The maximizing matrices are also characterized by a significant deviation in the pairs of symmetric nucleotide substitutions, <em>e.g.</em>, A→C and C→A expressed by:<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-183.gif"/></noscript></span><span class="disp-formula-label">(13)</span></span>Median and quartile range for these matrices were 0.623 [0.491–0.747]. Deviation in pairs of such nucleotide substitutions can enhance the impact of selection on the relative 4FD codon usage through an unbalanced influx and outflow of these codons. For example, C→A substitution, which is more frequent than A→C substitution, can lower the content of alanine codon GCC at the expense of GAC coding for asparagine. In the case of comparable probabilities of these substitutions, reversions could recover the numbers of disappearing codons.</p></div><div id="sec-13" class="subsection"><h3>Comparison of estimated deviations in codon usage with that observed in protein coding genes</h3><p id="p-62">It would be desirable to assess the strength of the measured effect of selection, at the amino acid level, on the relative usage of 4FD codons in the context of empirical data. Therefore, we compared the difference observed between the relative frequencies of 4FD codons after selection, and their expected frequencies resulting only from the applied mutational process, with an analogous measure for such codons calculated in protein coding sequences present in almost 4900 bacterial genomes. In an ideal situation, the expected occurrence of the observed relative frequencies of 4FD codons in protein coding sequences should be an aftermath of pure mutational pressures only. These are, however, not known. Therefore, we approximated the expected frequencies of 4FD codons by the average of the relative frequencies of 4FD codons in genes. Since bacterial genomes are characterized by a strong chromosome-wide compositional bias determined by two mutational pressures associated with differently replicated, leading and lagging, DNA strands, we examined the codon usage of genes separately from these DNA strands.</p><p id="p-63">The distribution of the summarized deviation from the expectation distribution in codon usage for all 4FD groups in protein-coding sequences was compared with the distributions of the calculated maximum deviation in codon usage resulting from selection at the amino acid level (the measure <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-184.gif"/></noscript></span></span>). It was also compared with the starting values before the optimization procedure (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). The maximized values are clearly shifted from the initial distribution, and overlap the distribution from genes. The average figure for the starting values is 0.3, for the maximized values ∼2, whereas for the genes it is 5.2. The figures for the maximized nucleotide substitution matrices constitute, on average, 37% of the deviation in codon usage found for protein coding sequences.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/967/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of the deviation from the expectation in the codon usage for all 4FD groups calculated for protein coding sequences, starting (randomly selected) nucleotide substitution matrices, and matrices that maximized this measure. The maximized values are of the same order of magnitude as the deviation based on empirical data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2365689" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of the deviation from the expectation in the codon usage for all 4FD groups calculated for protein coding sequences, starting (randomly selected) nucleotide substitution matrices, and matrices that maximized this measure. The maximized values are of the same order of magnitude as the deviation based on empirical data.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/967/F7.medium.gif" width="440" height="361"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/3/967/F7.medium.gif" width="440" height="361"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/967/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/967/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483542" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-64" class="first-child">Distribution of the deviation from the expectation in the codon usage for all 4FD groups calculated for protein coding sequences, starting (randomly selected) nucleotide substitution matrices, and matrices that maximized this measure. The maximized values are of the same order of magnitude as the deviation based on empirical data.</p><div class="sb-div caption-clear"></div></div></div><p id="p-65">The data seem to indicate that the estimated effect of the selection by the measure <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-185.gif"/></noscript></span></span> is not negligible. It is likely that the deviation calculated for the real genes is higher because of additional factors influencing codon bias, and linked, for example, with the effectiveness of translation, which appears universal in prokaryotic genomes, and concerns a substantial fraction of their genes (<a id="xref-ref-85-2" class="xref-bibr" href="#ref-85">Supek <em>et al.</em> 2010</a>). It is conceivable that the applied selection model based on the general <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Grantham’s (1974)</a> amino acid matrix deviates from synonymous codon usage less than in real selections, which can be different for various genes and their products. Nevertheless, comparison with the observed data demonstrates that the effect of selection at the amino acid level could help explain a substantial proportion of the observed codon usage bias, and, as such, cannot be disregarded.</p></div><div id="sec-14" class="subsection"><h3>Modeling of codon substitution process</h3><p id="p-66">In this study, we used a mutational–selection model the same as that proposed by <a id="xref-ref-56-9" class="xref-bibr" href="#ref-56">Morton (2001)</a>. This approach has many advantages that are relevant to our study. The model consists of separate mutation and selection components, which are easy to control. The mutation process can be defined simply by mutational matrices based on fixed nucleotide stationary distributions without any influence of selection. Apart from stationarity, the mutational matrix does not require the assumption on time-reversibility, which makes this model more general. The selection part is also expressed easily by an amino acid matrix, which does not require complicated transformations and implementations.</p><p id="p-67">We decided to apply this model because others commonly used in the modeling of codon substitutions (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Halpern and Bruno 1998</a>; <a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Yang and Nielsen 2008</a>) are not flexible enough to investigate the studied phenomenon. Particularly, they describe the mutation substitutions as a time-reversible stationary Markov process. They also introduce a selection mechanism in such a way that the final mutation-selection process is also time-reversible and stationary. Thanks to that, the models are computationally effective tools. However, there are no biological reasons to expect that the substitution process proceeds in a reversible manner (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Felsenstein 2004</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Schneider and Cannarozzi 2012</a>; <a id="xref-ref-95-2" class="xref-bibr" href="#ref-95">Yang 2006</a>). This assumption is used only because of theoretical and practical computational benefits, as well as mathematical convenience.</p><p id="p-68">It is worth pointing out that, when we implement, in these models, a selection based on amino acid frequencies determined by functional requirements in proteins, then these models will produce the relative stationary frequencies of synonymous codons that will be the same as the stationary frequencies resulting from the strict mutational process (<a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Wallace <em>et al.</em> 2013</a>; <a id="xref-ref-96-2" class="xref-bibr" href="#ref-96">Yang and Nielsen 2008</a>). In consequence, under these assumptions, it is impossible to investigate the impact on synonymous codon usage of selection at the amino acid level. Therefore, the model applied in this work appears to be more elastic and general because of less restrictive assumptions. In addition, it does not exclude “<em>a priori</em>” any possible additional factors that could influence the usage of synonymous codons.</p><p id="p-69">The time-reversibility assumption is crucial in our consideration because if we assume that the nucleotide substitution process defined by the matrix <em>P*</em> is time-reversible, and the acceptance matrix <em>D</em> is symmetric, then the impact of selection at the amino acid level on the usage of synonymous codons disappears. To prove this, it is enough to show that, under the above assumptions, the combined mutation–selection process defined by the probability matrix <em>C</em> has the same stationary distribution as the mutational process alone, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-186.gif"/></noscript></span></span> Thus, from the time-reversibility, we get:<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-187.gif"/></noscript></span><span class="disp-formula-label">(14)</span></span>and assuming the symmetry of the acceptance matrix <em>D</em>, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-188.gif"/></noscript></span></span> we obtain the following equalities:<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-189.gif"/></noscript></span><span class="disp-formula-label">(15)</span></span>Clearly, Equation 15 is a detailed balance equation of the process generated by the mutation–selection matrix <em>C</em>. As a result, this process is also time-reversible, and, consequently, <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-190.gif"/></noscript></span></span> Thus, the studied influence of selection at the amino acid level on synonymous codons usage cannot be demonstrated under the assumption of time-reversibility of the mutational process and the symmetry of the acceptance matrix <em>D</em>.</p><p id="p-70">Nevertheless, this property (15) can be used in validation of the searching algorithm applied in our study. Since this algorithm considers the general class of nucleotide mutational matrices, including also the time-reversible models as a subset, it should be possible to find, by this algorithm, such nucleotide transition probability matrices that would generate the exact equality between stationary codon distributions before and after selection, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-191.gif"/></noscript></span></span> Such results would imply that the algorithm works efficiently. As expected, we received this equality for nucleotide substitution matrices that minimized the objective function <em>F<sub>π</sub></em>. The average values of <em>F<sub>π</sub></em> were, in these simulations, almost equal to zero for all considered nucleotide stationary distributions.</p><p id="p-71">It should also be added that, if the deviation in synonymous codon usage is obtained under the time-reversible nucleotide substitution matrices, then the acceptance probabilities matrix must be asymmetric. However, commonly used matrices describing physicochemical or biochemical differences/similarities between amino acids are symmetric. In our approach, we applied <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Grantham’s (1974)</a> matrix of acceptance probabilities corresponding to chemical similarities between amino acids. Since the matrix is symmetric, it favors no direction of amino acid replacement, in contrast to a mutational matrix. Therefore, both mutation and selection are necessary to generate bias in the usage of 4FD codons.</p><p id="p-72">In our model, it is also possible to apply other acceptance probability matrices based on various physicochemical or biochemical amino acid properties, <em>e.g.</em>, hydropathy or polarity. Since such properties, and resulting matrices, are usually quite strongly correlated, their use would not change the general conclusion about the influence of selection at the amino acid level on synonymous codon usage. The other matrices can slightly increase or decrease the codon bias observed for <a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Grantham’s (1974)</a> matrix, depending on the stationary distributions of mutational matrices, but comprehensive and detailed studied are necessary to assess these relationships, and the intensity of this effect. However, commonly used PAM (Point Accepted Mutation) matrices are not appropriate because they are not free of a mutational influence, which is important in our considerations. It should also be noted that <a id="xref-ref-31-6" class="xref-bibr" href="#ref-31">Grantham’s (1974)</a> matrix represents a mean field approximation of a model with fluctuating selection, because this matrix describes only general similarities in chemical properties of amino acids, but not specific selection for a particular protein. Various types of proteins can be characterized by different selection requirements because of their specific structure and function. The <a id="xref-ref-31-7" class="xref-bibr" href="#ref-31">Grantham’s (1974)</a> matrix is a general representation of constant selection, but may not be a good approximation to the true evolutionary dynamics under time-varying selection. Such variable selections can produce their own distinctive pattern in codon usage bias in different types of sequences or regions (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Plotkin and Dushoff 2003</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Plotkin <em>et al.</em> 2006</a>).</p></div><div id="sec-15" class="subsection"><h3>Properties of <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-192.gif"/></noscript></span></span> measure</h3><p id="p-73">To assess the strength of selection at the amino acid level on 4FD codon usage, we applied the <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-193.gif"/></noscript></span></span> measure, being the normalized difference between the relative frequency of 4FD codons before and after selection (Equation 9 and Equation 10). This measure was inspired by chi-squared statistics, and has useful features, like other standard measures, which can also be applied in the calculation of differences between probability distributions. Above all, <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-194.gif"/></noscript></span></span> is always non-negative and equals zero if, and only if, the relative stationary frequency of a codon subjected to a mutational process equals its frequency after selection, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-195.gif"/></noscript></span></span> for all <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-196.gif"/></noscript></span></span> This property is called the identity condition, and should be fulfilled by any measure that is used to calculate the difference between two probability distributions. Thanks to that, the detection of any differences between stationary frequencies of <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-197.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-198.gif"/></noscript></span></span> is independent of the measure. In contrast to standard measures like Kullback-Leibler divergence, or total variation distance, <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/embed/mml-math-199.gif"/></noscript></span></span> includes information about the absolute value of the change of codon frequencies in relation to expectation under the mutational process, which is useful to easily interpret the results obtained.</p><p id="p-74">Nevertheless, our measure gives values compatible with Kullback-Leibler divergence and total variation distance. The Spearman correlation coefficient between values of our measure with those of Kullback-Leibler divergence and the total variation distance calculated for all 88,560 considered cases of nucleotide stationary distributions is very high and statistically significant (<em>p</em>-values &lt;2.2E−16, 0.921 and 0.915, respectively). Therefore, the application of these measures would change neither conclusions nor important results. With such great correlation, the trends presented in figures between the measure and nucleotide composition would be also the same.</p></div><div id="sec-16" class="subsection"><h3>Concluding remarks</h3><p id="p-75">The study undertaken here estimated the influence of selection, at the amino acid level, on the relative usage of 4FD codons. This impact was determined by different selection constraints on the nonsynonymous replacement of these codons with others, which proceeded in a complex manner and depended on the probability of fixation of such substitutions, as well as on the probability of particular nucleotide substitutions. We tested a wide range of conditions in which such influence can be valid, by the inclusion of nearly 90,000 stationary nucleotide distributions and associated unrestricted mutational processes. Selection was based on differences in the physicochemical properties of amino acids.</p><p id="p-76">We noticed that mutational processes generating more adenine and thymine than guanine and cytosine enhance the influence of selection. The same is true for the processes yielding more purines than pyrimidines. It is noteworthy that the relationship between the effect under study and the content of these nucleotides is nonlinear. On the other hand, the impact of selection at the amino acid level diminishes when the nucleotide processes generate 50% content of purines and pyrimidines as well as more guanine and cytosine than adenine and thymine. The nucleotide substitution matrices maximizing the consequence of amino acid selection are also characterized by a greater probability of transversions outnumbering transitions, and a greater deviation in pairs of reversible nucleotide substitutions.</p><p id="p-77">The influence of selection at the amino acid level was different for particular groups of 4FD codons. Generally, glycine codons show the strongest response to the selection impact under study, whereas codons for valine the weakest. However, the deviation in the codon usage generated by the process, with and without selection, depends nonlinearly on nucleotide stationary distribution. This effect could be explained by the discrepancies in the acceptance probabilities of substitutions of amino acids coded by these codon blocks.</p><p id="p-78">The results indicate that selection acting on nonsynonymous substitutions, <em>i.e.</em>, leading to amino acid replacements, can affect the usage of 4FD codons. This effect, however, is complex, and depends on the properties of mutational pressure, which superimposes on the selection. Interestingly, we discovered that, for each nucleotide distribution, it is possible to find such mutational probability matrices that will minimize and maximize the effect. This seems to suggest that the influence of selection, at the amino acid level, on synonymous codon usage, can vary in different organisms. Since the mutational pressure in genomes is not known, and selection at the amino acid level is also complicated, it is difficult to assess the exact contribution of this process in real protein coding sequences. Selection can both enhance and suppress the other effects on codon usage, <em>e.g.</em>, selection related to translation efficiency. Nevertheless, the effect cannot be neglected because it correlates with the comparison between the calculated deviation in the codon usage subjected to this selection and an analogous measure estimated for protein coding sequences.</p><p id="p-79">Our results show that substitution matrices generating high A+T content affect 4FD codon usage to the greatest extent. Assuming that the global genome content corresponds to a global mutational pressure (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Muto and Osawa 1987</a>), we can conclude that the effect of the selection under study would be most pronounced in AT-rich genomes. Consequently, the selection on nonsynonymous substitutions can interfere in such genomes with other selections on codon usage, <em>e.g.</em>, related to translational efficiency. In agreement with that, we found that the difference between highly expressed genes coding for ribosomal proteins and other genes, as far as the relative usage of particular 4FD codons is concerned, becomes smaller with A+T genomic content (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>). There seems to be some evidence that it could be more difficult to maintain the appropriate codon bias in highly expressed genes in AT-rich genomes. Likewise, in genomes with &gt;70% A+T, no influence of translational selection was reported, <em>i.e.</em>, <em>Borrelia burgdorferi</em> (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">McInerney 1998</a>), <em>Buchnera</em> (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Rispe <em>et al.</em> 2004</a>), <em>Wigglesworthia</em> (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Herbeck <em>et al.</em> 2003</a>) and <em>Blochmannia floridanus</em> (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Banerjee <em>et al.</em> 2004</a>). This may result from a greater difficulty in predicting genes with translational efficiency in AT-biased genomes using standard methods (<em>e.g.</em>, CAI, codon adaptation index), because other methods based on random forest classifier revealed that, in these genomes, codon bias was associated with translational efficiency (<a id="xref-ref-85-3" class="xref-bibr" href="#ref-85">Supek <em>et al.</em> 2010</a>). Notwithstanding, our results show that AT-rich genomes either have to cope with the greater influence of selection, at the amino acid level, on synonymous codon usage or adapt to it. It is possible that this type of selection can trigger codon bias in some genes, which can be misleading with regard to selection on translational effectiveness. Accordingly, <a id="xref-ref-56-10" class="xref-bibr" href="#ref-56">Morton (2001)</a> carried out an appropriate test, decreasing the number of genes believed to have codon usage associated with translational selection.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/967/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dependence on the genomic A+T content of the difference in the relative usage of 4FD codons between genes coding for ribosomal and nonribosomal proteins. The difference was calculated based on 4802 genomes, with at least 30 genes annotated for ribosomal proteins, separately for the leading and lagging strand. In total, 5124 pairs of genes, with at least 15 ribosomal genes on one strand, were considered. The bars represent an average value for the given class of A+T content, whereas whiskers represent SD. The difference was calculated according to:  where is the observed frequency of a 4FD codon si with a nucleotide i at the third codon position, and  is the frequency of all codons in the 4FD codon group S. Indices rib and nonrib mean genes for ribosomal and nonribosomal proteins, respectively. The calculated difference decreases with AT%, and is the largest for the moderate AT content." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2365689" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Dependence on the genomic A+T content of the difference in the relative usage of 4FD codons between genes coding for ribosomal and nonribosomal proteins. The difference was calculated based on 4802 genomes, with at least 30 genes annotated for ribosomal proteins, separately for the leading and lagging strand. In total, 5124 pairs of genes, with at least 15 ribosomal genes on one strand, were considered. The bars represent an average value for the given class of A+T content, whereas whiskers represent SD. The difference was calculated according to: &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-200&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; where &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-201&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;is the observed frequency of a 4FD codon &lt;em&gt;s&lt;sub&gt;i&lt;/sub&gt;&lt;/em&gt; with a nucleotide &lt;em&gt;i&lt;/em&gt; at the third codon position, and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-202&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; is the frequency of all codons in the 4FD codon group &lt;em&gt;S&lt;/em&gt;. Indices &lt;em&gt;rib&lt;/em&gt; and &lt;em&gt;nonrib&lt;/em&gt; mean genes for ribosomal and nonribosomal proteins, respectively. The calculated difference decreases with AT%, and is the largest for the moderate AT content.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/967/F8.medium.gif" width="440" height="317"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/7/3/967/F8.medium.gif" width="440" height="317"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/967/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/967/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483569" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-80" class="first-child">Dependence on the genomic A+T content of the difference in the relative usage of 4FD codons between genes coding for ribosomal and nonribosomal proteins. The difference was calculated based on 4802 genomes, with at least 30 genes annotated for ribosomal proteins, separately for the leading and lagging strand. In total, 5124 pairs of genes, with at least 15 ribosomal genes on one strand, were considered. The bars represent an average value for the given class of A+T content, whereas whiskers represent SD. The difference was calculated according to: <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F8/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F8/embed/mml-math-200.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F8/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F8/embed/mml-math-201.gif"/></noscript></span></span>is the observed frequency of a 4FD codon <em>s<sub>i</sub></em> with a nucleotide <em>i</em> at the third codon position, and <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F8/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/967/F8/embed/mml-math-202.gif"/></noscript></span></span> is the frequency of all codons in the 4FD codon group <em>S</em>. Indices <em>rib</em> and <em>nonrib</em> mean genes for ribosomal and nonribosomal proteins, respectively. The calculated difference decreases with AT%, and is the largest for the moderate AT content.</p><div class="sb-div caption-clear"></div></div></div><p id="p-81">The aim of our study was to verify the effect of amino acid selection on 4FD codon usage on global and general scales for a large number of possible mutational pressures, and fixed selection for amino acid replacements. Nevertheless, our results can be helpful to explain some effects related to codon bias also at the local scale, <em>i.e.</em>, codon usage variation across sites within a gene. Such variation was noted by <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akashi (1994)</a> in orthologous genes from fruit flies. He found that the frequency of preferred codons is significantly higher at conserved amino acid positions than in nonconserved ones. This finding was further confirmed in bacteria (<a id="xref-ref-84-2" class="xref-bibr" href="#ref-84">Stoletzki and Eyre-Walker 2007</a>), as well as yeast, worm, mouse, and human (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Drummond and Wilke 2008</a>). This codon bias was interpreted as a result of selection for minimization of the chance for translation errors and protein misfolding during this process. On the other hand, <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Plotkin and Dushoff (2003)</a> and <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Plotkin <em>et al.</em> (2004)</a> found that some variable sequences coding for antigens and surface proteins, or regions interacting with antibodies in pathogens <em>Mycobacterium tuberculosis</em>, <em>Plasmodium falciparum</em>, and influenza A virus, are rich in “volatile” codons that can mutate with larger probability to codons encoding other amino acids. Such elevated volatility of these genes may be associated with a positive selection, and greater pressure for amino-acid substitutions, which is favored in order to avoid interactions with the host immune system. Although we showed that a synonymous codon bias can be generated by a general selection at the amino acid level, it cannot be excluded that more specific selections influencing particular sites in protein sequences with various intensity or pattern (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bazykin 2015</a>) may also contribute, with other effects, to the observed codon biases at specific sequence sites.</p></div></div><div class="section" id="sec-17"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-82">We are very grateful to the Reviewers for their valuable comments and insightful remarks that significantly improved the paper. Publication editing was funded by the Krajowy Naukowy Ośrodek Wiodący (KNOW) Consortium, Faculty of Biotechnology, University of Wroclaw.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-83">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038125/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038125/-/DC1</a>.</p></li><li class="fn" id="fn-6"><p id="p-84"><em>Communicating editor: K. Thornton</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-12-03"><span class="received-label">Received </span>December 3, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-01-17"><span class="accepted-label">Accepted </span>January 17, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Błażej <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akashi</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Synonymous codon usage in <em>Drosophila-melanogaster</em>—natural-selection and translational accuracy.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">927</span>–<span class="cit-lpage">935</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAkashi%26rft.auinit1%253DH.%26rft.volume%253D136%26rft.issue%253D3%26rft.spage%253D927%26rft.epage%253D935%26rft.atitle%253DSynonymous%2Bcodon%2Busage%2Bin%2BDrosophila%2Bmelanogaster%253A%2Bnatural%2Bselection%2Band%2Btranslational%2Baccuracy.%26rft_id%253Dinfo%253Apmid%252F8005445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM2LzMvOTI3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzk2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akashi</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Translational selection and yeast proteome evolution.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">1291</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAkashi%26rft.auinit1%253DH.%26rft.volume%253D164%26rft.issue%253D4%26rft.spage%253D1291%26rft.epage%253D1303%26rft.atitle%253DTranslational%2BSelection%2Band%2BYeast%2BProteome%2BEvolution%26rft_id%253Dinfo%253Apmid%252F12930740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NC80LzEyOTEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvOTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Banerjee</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basak</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghosh</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Evolutionary forces in shaping the codon and amino acid usages in <em>Blochmannia floridanus</em>.</span> <abbr class="cit-jnl-abbrev">J. Biomol. Struct. Dyn.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">13</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbiomolecular%2Bstructure%2B%2526%2Bdynamics%26rft.stitle%253DJ%2BBiomol%2BStruct%2BDyn%26rft.aulast%253DBanerjee%26rft.auinit1%253DT.%26rft.volume%253D22%26rft.issue%253D1%26rft.spage%253D13%26rft.epage%253D23%26rft.atitle%253DEvolutionary%2Bforces%2Bin%2Bshaping%2Bthe%2Bcodon%2Band%2Bamino%2Bacid%2Busages%2Bin%2BBlochmannia%2Bfloridanus.%26rft_id%253Dinfo%253Apmid%252F15214801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15214801&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222865400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.4" data-doi="10.1074/jbc.M110.154575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bartoszewski</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jablonsky</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartoszewska</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevenson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dai</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A synonymous single nucleotide polymorphism in DeltaF508 CFTR alters the secondary structure of the mRNA and the expression of the mutant protein.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">285</span>: <span class="cit-fpage">28741</span>–<span class="cit-lpage">28748</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M110.154575%26rft_id%253Dinfo%253Apmid%252F20628052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODUvMzcvMjg3NDEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvOTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.5" data-doi="10.1098/rsbl.2015.0315"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bazykin</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Changing preferences: deformation of single position amino acid fitness landscapes and evolution of proteins.</span> <abbr class="cit-jnl-abbrev">Biol. Lett.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">20150315</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BLett.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsbl.2015.0315%26rft_id%253Dinfo%253Apmid%252F26445980%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InJveWJpb2xldHQiO3M6NToicmVzaWQiO3M6MTQ6IjExLzEwLzIwMTUwMzE1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzk2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bennetzen</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">B. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Codon selection in yeast.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">257</span>: <span class="cit-fpage">3026</span>–<span class="cit-lpage">3031</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DBennetzen%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D257%26rft.issue%253D6%26rft.spage%253D3026%26rft.epage%253D3031%26rft.atitle%253DCodon%2Bselection%2Bin%2Byeast%26rft_id%253Dinfo%253Apmid%252F7037777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNTcvNi8zMDI2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzk2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.7" data-doi="10.1038/msb.2013.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bentele</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saffert</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rauscher</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ignatova</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bluthgen</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Efficient translation initiation dictates codon usage at gene start.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">675</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2013.32%26rft_id%253Dinfo%253Apmid%252F23774758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjc6IjkvMS82NzUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvOTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bulmer</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">The selection-mutation-drift theory of synonymous codon usage.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">897</span>–<span class="cit-lpage">907</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBulmer%26rft.auinit1%253DM.%26rft.volume%253D129%26rft.issue%253D3%26rft.spage%253D897%26rft.epage%253D907%26rft.atitle%253DThe%2Bselection-mutation-drift%2Btheory%2Bof%2Bsynonymous%2Bcodon%2Busage.%26rft_id%253Dinfo%253Apmid%252F1752426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI5LzMvODk3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzk2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.9" data-doi="10.1016/j.cell.2010.02.036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cannarrozzi</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schraudolph</span>  <span class="cit-name-given-names">N. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faty</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Rohr</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friberg</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A role for codon order in translation dynamics.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DCannarrozzi%26rft.auinit1%253DG.%26rft.volume%253D141%26rft.issue%253D2%26rft.spage%253D355%26rft.epage%253D367%26rft.atitle%253DA%2Brole%2Bfor%2Bcodon%2Border%2Bin%2Btranslation%2Bdynamics.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2010.02.036%26rft_id%253Dinfo%253Apmid%252F20403329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2010.02.036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20403329&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276738400024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carbone</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madden</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Insights on the evolution of metabolic networks of unicellular translationally biased organisms from transcriptomic data and sequence analysis.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">456</span>–<span class="cit-lpage">469</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DCarbone%26rft.auinit1%253DA.%26rft.volume%253D61%26rft.issue%253D4%26rft.spage%253D456%26rft.epage%253D469%26rft.atitle%253DInsights%2Bon%2Bthe%2Bevolution%2Bof%2Bmetabolic%2Bnetworks%2Bof%2Bunicellular%2Btranslationally%2Bbiased%2Borganisms%2Bfrom%2Btranscriptomic%2Bdata%2Band%2Bsequence%2Banalysis.%26rft_id%253Dinfo%253Apmid%252F16187158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16187158&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.11" data-doi="10.1146/annurev-biophys-060414-034333"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chaney</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Roles for synonymous codon usage in protein biogenesis.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biophys.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">143</span>–<span class="cit-lpage">166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BBiophys.%26rft.volume%253D44%26rft.spage%253D143%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-biophys-060414-034333%26rft_id%253Dinfo%253Apmid%252F25747594%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-biophys-060414-034333&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25747594&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.12" data-doi="10.1073/pnas.0307827100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hottes</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McAdams</span>  <span class="cit-name-given-names">H. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Codon usage between genomes is constrained by genome-wide mutational processes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">3480</span>–<span class="cit-lpage">3485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0307827100%26rft_id%253Dinfo%253Apmid%252F14990797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzEwLzM0ODAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvOTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.13" data-doi="10.1126/science.168.3934.1009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clarke</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Darwinian evolution of proteins.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">1009</span>–<span class="cit-lpage">1011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DClarke%26rft.auinit1%253DB.%26rft.volume%253D168%26rft.issue%253D3934%26rft.spage%253D1009%26rft.epage%253D1011%26rft.atitle%253DDarwinian%2BEvolution%2Bof%2BProteins%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.168.3934.1009%26rft_id%253Dinfo%253Apmid%252F4909620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJzY2kiO3M6NToicmVzaWQiO3M6MTM6IjE2OC8zOTM0LzEwMDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvOTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.14" data-doi="10.1186/1471-2164-11-118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clarke</span>  <span class="cit-name-given-names">T. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Increased incidence of rare codon clusters at 5’ and 3’ gene termini: implications for function.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DClarke%26rft.auinit1%253DT.%2BF.%26rft.volume%253D11%26rft.spage%253D118%26rft.epage%253D118%26rft.atitle%253DIncreased%2Bincidence%2Bof%2Brare%2Bcodon%2Bclusters%2Bat%2B5%2527%2Band%2B3%2527%2Bgene%2Btermini%253A%2Bimplications%2Bfor%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-118%26rft_id%253Dinfo%253Apmid%252F20167116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-118&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20167116&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.15" data-doi="10.1126/science.1155761"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coleman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papamichail</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skiena</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Futcher</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Virus attenuation by genome-scale changes in codon pair bias.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">320</span>: <span class="cit-fpage">1784</span>–<span class="cit-lpage">1787</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DColeman%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D320%26rft.issue%253D5884%26rft.spage%253D1784%26rft.epage%253D1787%26rft.atitle%253DVirus%2BAttenuation%2Bby%2BGenome-Scale%2BChanges%2Bin%2BCodon%2BPair%2BBias%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1155761%26rft_id%253Dinfo%253Apmid%252F18583614%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjAvNTg4NC8xNzg0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzk2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.16" data-doi="10.1534/genetics.104.026351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comeron</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Selective and mutational patterns associated with gene expression in humans: influences on synonymous composition and intron presence.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">167</span>: <span class="cit-fpage">1293</span>–<span class="cit-lpage">1304</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DComeron%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D167%26rft.issue%253D3%26rft.spage%253D1293%26rft.epage%253D1304%26rft.atitle%253DSelective%2Band%2BMutational%2BPatterns%2BAssociated%2BWith%2BGene%2BExpression%2Bin%2BHumans%253A%2BInfluences%2Bon%2BSynonymous%2BComposition%2Band%2BIntron%2BPresence%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.026351%26rft_id%253Dinfo%253Apmid%252F15280243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Ny8zLzEyOTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvOTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.17" data-doi="10.3324/haematol.2010.036848"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daidone</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallinaro</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cattini</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pontara</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertomoro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">An apparently silent nucleotide substitution (c.7056C &gt; T) in the von Willebrand factor gene is responsible for type 1 von Willebrand disease.</span> <abbr class="cit-jnl-abbrev">Haematologica</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">881</span>–<span class="cit-lpage">887</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHaematologica%26rft_id%253Dinfo%253Adoi%252F10.3324%252Fhaematol.2010.036848%26rft_id%253Dinfo%253Apmid%252F21393328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiaGFlbWF0b2wiO3M6NToicmVzaWQiO3M6ODoiOTYvNi84ODEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvOTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-book" id="cit-7.3.967.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Jong</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fogel</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwefel</span>  <span class="cit-name-given-names">H.-P.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">A history of evolutionary computation</span>, pp. <span class="cit-fpage">A2.3:1</span>–<span class="cit-lpage">12</span> in <span class="cit-source">Handbook of Evolutionary Computation</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Back</span>  <span class="cit-name-given-names">T.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Fogel</span>  <span class="cit-name-given-names">D.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Michalewicz</span>  <span class="cit-name-given-names">Z.</span></span>. <span class="cit-publ-name">IOP Publishing Ltd. and Oxford University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.19" data-doi="10.1093/nar/gkh834"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">dos Reis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Savva</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wernisch</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Solving the riddle of codon usage preferences: a test for translational selection.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">5036</span>–<span class="cit-lpage">5044</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DReis%26rft.auinit1%253DM.%2Bd.%26rft.volume%253D32%26rft.issue%253D17%26rft.spage%253D5036%26rft.epage%253D5044%26rft.atitle%253DSolving%2Bthe%2Briddle%2Bof%2Bcodon%2Busage%2Bpreferences%253A%2Ba%2Btest%2Bfor%2Btranslational%2Bselection%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh834%26rft_id%253Dinfo%253Apmid%252F15448185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMi8xNy81MDM2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzk2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.20" data-doi="10.1016/j.cell.2008.05.042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilke</span>  <span class="cit-name-given-names">C. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mistranslation-induced protein misfolding as a dominant constraint on coding-sequence evolution.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">341</span>–<span class="cit-lpage">352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DDrummond%26rft.auinit1%253DD.%2BA.%26rft.volume%253D134%26rft.issue%253D2%26rft.spage%253D341%26rft.epage%253D352%26rft.atitle%253DMistranslation-induced%2Bprotein%2Bmisfolding%2Bas%2Ba%2Bdominant%2Bconstraint%2Bon%2Bcoding-sequence%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.05.042%26rft_id%253Dinfo%253Apmid%252F18662548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.05.042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18662548&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257891700022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.21" data-doi="10.1073/pnas.96.8.4482"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duret</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mouchiroud</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Expression pattern and, surprisingly, gene length shape codon usage in <em>Caenorhabditis</em>, <em>Drosophila</em>, and <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">4482</span>–<span class="cit-lpage">4487</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DDuret%26rft.auinit1%253DL.%26rft.volume%253D96%26rft.issue%253D8%26rft.spage%253D4482%26rft.epage%253D4487%26rft.atitle%253DExpression%2Bpattern%2Band%252C%2Bsurprisingly%252C%2Bgene%2Blength%2Bshape%2Bcodon%2Busage%2Bin%2BCaenorhabditis%252C%2BDrosophila%252C%2Band%2BArabidopsis%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.96.8.4482%26rft_id%253Dinfo%253Apmid%252F10200288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5Ni84LzQ0ODIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvOTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ermolaeva</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Synonymous codon usage in bacteria.</span> <abbr class="cit-jnl-abbrev">Curr. Issues Mol. Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">97</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bissues%2Bin%2Bmolecular%2Bbiology%26rft.stitle%253DCurr%2BIssues%2BMol%2BBiol%26rft.aulast%253DErmolaeva%26rft.auinit1%253DM.%2BD.%26rft.volume%253D3%26rft.issue%253D4%26rft.spage%253D91%26rft.epage%253D97%26rft.atitle%253DSynonymous%2Bcodon%2Busage%2Bin%2Bbacteria.%26rft_id%253Dinfo%253Apmid%252F11719972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11719972&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bulmer</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Synonymous substitution rates in enterobacteria.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">1407</span>–<span class="cit-lpage">1412</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DEyre-Walker%26rft.auinit1%253DA.%26rft.volume%253D140%26rft.issue%253D4%26rft.spage%253D1407%26rft.epage%253D1412%26rft.atitle%253DSynonymous%2Bsubstitution%2Brates%2Bin%2Benterobacteria.%26rft_id%253Dinfo%253Apmid%252F7498779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0MC80LzE0MDciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvOTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.24" data-doi="10.1093/nar/30.5.1192"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fedorov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saxonov</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Regularities of context-dependent codon bias in eukaryotic genes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1192</span>–<span class="cit-lpage">1197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DFedorov%26rft.auinit1%253DA.%26rft.volume%253D30%26rft.issue%253D5%26rft.spage%253D1192%26rft.epage%253D1197%26rft.atitle%253DRegularities%2Bof%2Bcontext-dependent%2Bcodon%2Bbias%2Bin%2Beukaryotic%2Bgenes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F30.5.1192%26rft_id%253Dinfo%253Apmid%252F11861911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjMwLzUvMTE5MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMy85NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-7.3.967.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-source"><em>Inferring Phylogenies</em>.</span> <span class="cit-publ-name">Sinauer Associates, Inc.</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.26" data-doi="10.1016/S0378-1119(99)00297-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frank</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lobry</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Asymmetric substitution patterns: a review of possible underlying mutational or selective mechanisms.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">238</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">77</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DFrank%26rft.auinit1%253DA.%2BC.%26rft.volume%253D238%26rft.issue%253D1%26rft.spage%253D65%26rft.epage%253D77%26rft.atitle%253DAsymmetric%2Bsubstitution%2Bpatterns%253A%2Ba%2Breview%2Bof%2Bpossible%2Bunderlying%2Bmutational%2Bor%2Bselective%2Bmechanisms.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1119%252899%252900297-8%26rft_id%253Dinfo%253Apmid%252F10570985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1119(99)00297-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10570985&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083005400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.27" data-doi="10.1093/nar/gkg004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Vallve</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guzman</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montero</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romeu</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">HGT-DB: a database of putative horizontally transferred genes in prokaryotic complete genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">187</span>–<span class="cit-lpage">189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DGarcia-Vallve%26rft.auinit1%253DS.%26rft.volume%253D31%26rft.issue%253D1%26rft.spage%253D187%26rft.epage%253D189%26rft.atitle%253DHGT-DB%253A%2Ba%2Bdatabase%2Bof%2Bputative%2Bhorizontally%2Btransferred%2Bgenes%2Bin%2Bprokaryotic%2Bcomplete%2Bgenomes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg004%26rft_id%253Dinfo%253Apmid%252F12519978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjMxLzEvMTg3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzk2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.28" data-doi="10.1038/nature02046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ghaemmaghami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huh</span>  <span class="cit-name-given-names">W. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bower</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howson</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belle</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Global analysis of protein expression in yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">425</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">741</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGhaemmaghami%26rft.auinit1%253DS.%26rft.volume%253D425%26rft.issue%253D6959%26rft.spage%253D737%26rft.epage%253D741%26rft.atitle%253DGlobal%2Banalysis%2Bof%2Bprotein%2Bexpression%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02046%26rft_id%253Dinfo%253Apmid%252F14562106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14562106&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185924500046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.29" data-doi="10.1016/j.bbrc.2004.11.134"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goetz</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuglsang</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Correlation of codon bias measures with mRNA levels: analysis of transcriptome data from <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Biochem. Biophys. Res. Commun.</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">4</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2Band%2Bbiophysical%2Bresearch%2Bcommunications%26rft.stitle%253DBiochem%2BBiophys%2BRes%2BCommun%26rft.aulast%253DGoetz%26rft.auinit1%253DR.%2BM.%26rft.volume%253D327%26rft.issue%253D1%26rft.spage%253D4%26rft.epage%253D7%26rft.atitle%253DCorrelation%2Bof%2Bcodon%2Bbias%2Bmeasures%2Bwith%2BmRNA%2Blevels%253A%2Banalysis%2Bof%2Btranscriptome%2Bdata%2Bfrom%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbrc.2004.11.134%26rft_id%253Dinfo%253Apmid%252F15629421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbrc.2004.11.134&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15629421&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226345400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.30" data-doi="10.1126/science.1241934"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goodman</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kosuri</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Causes and effects of N-terminal codon bias in bacterial genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">342</span>: <span class="cit-fpage">475</span>–<span class="cit-lpage">479</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1241934%26rft_id%253Dinfo%253Apmid%252F24072823%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDIvNjE1Ny80NzUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvOTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.31" data-doi="10.1126/science.185.4154.862"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grantham</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">Amino acid difference formula to help explain protein evolution.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">862</span>–<span class="cit-lpage">864</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGrantham%26rft.auinit1%253DR.%26rft.volume%253D185%26rft.issue%253D4154%26rft.spage%253D862%26rft.epage%253D864%26rft.atitle%253DAmino%2BAcid%2BDifference%2BFormula%2Bto%2BHelp%2BExplain%2BProtein%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.185.4154.862%26rft_id%253Dinfo%253Apmid%252F4843792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIxODUvNDE1NC84NjIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvOTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grantham</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gouy</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mercier</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pave</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Codon catalog usage and the genome hypothesis.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">R49</span>–<span class="cit-lpage">R62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D8%26rft.spage%253DR49%26rft_id%253Dinfo%253Apmid%252F6986610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=6986610&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.33" data-doi="10.1016/j.tibtech.2004.04.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gustafsson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Govindarajan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minshull</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Codon bias and heterologous protein expression.</span> <abbr class="cit-jnl-abbrev">Trends Biotechnol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">346</span>–<span class="cit-lpage">353</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bbiotechnology%26rft.stitle%253DTrends%2BBiotechnol%26rft.aulast%253DGustafsson%26rft.auinit1%253DC.%26rft.volume%253D22%26rft.issue%253D7%26rft.spage%253D346%26rft.epage%253D353%26rft.atitle%253DCodon%2Bbias%2Band%2Bheterologous%2Bprotein%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibtech.2004.04.006%26rft_id%253Dinfo%253Apmid%252F15245907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibtech.2004.04.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15245907&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222712000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.34" data-doi="10.1093/oxfordjournals.molbev.a025995"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Halpern</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruno</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Evolutionary distances for protein-coding sequences: modeling site-specific residue frequencies.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">910</span>–<span class="cit-lpage">917</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DHalpern%26rft.auinit1%253DA.%26rft.volume%253D15%26rft.issue%253D7%26rft.spage%253D910%26rft.epage%253D917%26rft.atitle%253DEvolutionary%2Bdistances%2Bfor%2Bprotein-coding%2Bsequences%253A%2Bmodeling%2Bsite-%2Bspecific%2Bresidue%2Bfrequencies%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a025995%26rft_id%253Dinfo%253Apmid%252F9656490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjE1LzcvOTEwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzk2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.35" data-doi="10.1099/mic.0.26381-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Herbeck</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wernegreen</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Gene expression level influences amino acid usage, but not codon usage, in the tsetse fly endosymbiont Wigglesworthia.</span> <abbr class="cit-jnl-abbrev">Microbiology</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">2585</span>–<span class="cit-lpage">2596</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft.stitle%253DMicrobiology%26rft.aulast%253DHerbeck%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D149%26rft.issue%253D9%26rft.spage%253D2585%26rft.epage%253D2596%26rft.atitle%253DGene%2Bexpression%2Blevel%2Binfluences%2Bamino%2Bacid%2Busage%252C%2Bbut%2Bnot%2Bcodon%2Busage%252C%2Bin%2Bthe%2Btsetse%2Bfly%2Bendosymbiont%2BWigglesworthia%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fmic.0.26381-0%26rft_id%253Dinfo%253Apmid%252F12949182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/mic.0.26381-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12949182&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185342900027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.36" data-doi="10.1146/annurev.genet.42.110807.091442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hershberg</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Selection on codon bias.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DHershberg%26rft.auinit1%253DR.%26rft.volume%253D42%26rft.spage%253D287%26rft.epage%253D299%26rft.atitle%253DSelection%2Bon%2Bcodon%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.42.110807.091442%26rft_id%253Dinfo%253Apmid%252F18983258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.42.110807.091442&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18983258&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261767000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.37" data-doi="10.1016/0022-2836(81)90363-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ikemura</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Correlation between the abundance of <em>Escherichia coli</em> transfer RNAs and the occurrence of the respective codons in its protein genes.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DIkemura%26rft.auinit1%253DT.%26rft.volume%253D146%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D21%26rft.atitle%253DCorrelation%2Bbetween%2Bthe%2Babundance%2Bof%2BEscherichia%2Bcoli%2Btransfer%2BRNAs%2Band%2Bthe%2Boccurrence%2Bof%2Bthe%2Brespective%2Bcodons%2Bin%2Bits%2Bprotein%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252881%252990363-6%26rft_id%253Dinfo%253Apmid%252F6167728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(81)90363-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6167728&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1981LC53900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ikemura</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Codon usage and tRNA content in unicellular and multicellular organisms.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">13</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DIkemura%26rft.auinit1%253DT%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253D13%26rft.epage%253D34%26rft.atitle%253DCodon%2Busage%2Band%2BtRNA%2Bcontent%2Bin%2Bunicellular%2Band%2Bmulticellular%2Borganisms%26rft_id%253Dinfo%253Apmid%252F3916708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjY6IjIvMS8xMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMy85NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.39" data-doi="10.1038/nrg3645"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ingolia</span>  <span class="cit-name-given-names">N. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Ribosome profiling: new views of translation, from single codons to genome scale.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">205</span>–<span class="cit-lpage">213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D205%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3645%26rft_id%253Dinfo%253Apmid%252F24468696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3645&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24468696&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.40" data-doi="10.1126/science.1168978"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ingolia</span>  <span class="cit-name-given-names">N. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghaemmaghami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newman</span>  <span class="cit-name-given-names">J. R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">218</span>–<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1168978%26rft_id%253Dinfo%253Apmid%252F19213877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjQvNTkyNC8yMTgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvOTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1953</span> <span class="cit-article-title">Markoff chains as an aid in the study of Markoff processes.</span> <abbr class="cit-jnl-abbrev">Skand. Aktuar. J.</abbr> <span class="cit-vol">1953</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">91</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSkand.%2BAktuar.%2BJ.%26rft.volume%253D1953%26rft.spage%253D87%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.42" data-doi="10.1016/S0378-1119(99)00225-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kanaya</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamada</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kudo</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ikemura</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Studies of codon usage and tRNA genes of 18 unicellular organisms and quantification of <em>Bacillus subtilis</em> tRNAs: gene expression level and species-specific diversity of codon usage based on multivariate analysis.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">238</span>: <span class="cit-fpage">143</span>–<span class="cit-lpage">155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DKanaya%26rft.auinit1%253DS.%26rft.volume%253D238%26rft.issue%253D1%26rft.spage%253D143%26rft.epage%253D155%26rft.atitle%253DStudies%2Bof%2Bcodon%2Busage%2Band%2BtRNA%2Bgenes%2Bof%2B18%2Bunicellular%2Borganisms%2Band%2Bquantification%2Bof%2BBacillus%2Bsubtilis%2BtRNAs%253A%2Bgene%2Bexpression%2Blevel%2Band%2Bspecies-specific%2Bdiversity%2Bof%2Bcodon%2Busage%2Bbased%2Bon%2Bmultivariate%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1119%252899%252900225-5%26rft_id%253Dinfo%253Apmid%252F10570992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1119(99)00225-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10570992&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083005400015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.43" data-doi="10.1038/nature09322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kertesz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mazor</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rinn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nutter</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide measurement of RNA secondary structure in yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">107</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKertesz%26rft.auinit1%253DM.%26rft.volume%253D467%26rft.issue%253D7311%26rft.spage%253D103%26rft.epage%253D107%26rft.atitle%253DGenome-wide%2Bmeasurement%2Bof%2BRNA%2Bsecondary%2Bstructure%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09322%26rft_id%253Dinfo%253Apmid%252F20811459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20811459&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281461200044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimchi-Sarfaty</span>  <span class="cit-name-given-names">C. J</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">M</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">I. W</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sauna</span>  <span class="cit-name-given-names">Z. E</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calcagno</span>  <span class="cit-name-given-names">A. M</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A ‘silent’ polymorphism in the MDR1 gene changes substrate specificity.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">315</span>: <span class="cit-fpage">525</span>–<span class="cit-lpage">528</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.volume%253D315%26rft.spage%253D525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knight</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freeland</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landweber</span>  <span class="cit-name-given-names">L. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A simple model based on mutation and selection explains trends in codon and amino-acid usage and GC composition within and across genomes.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">RESEARCH0010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DKnight%26rft.auinit1%253DR.%2BD.%26rft.volume%253D2%26rft.issue%253D4%26rft.spage%253DRESEARCH0010%26rft.epage%253DRESEARCH0010%26rft.atitle%253DA%2Bsimple%2Bmodel%2Bbased%2Bon%2Bmutation%2Band%2Bselection%2Bexplains%2Btrends%2Bin%2Bcodon%2Band%2Bamino-acid%2Busage%2Band%2BGC%2Bcomposition%2Bwithin%2Band%2Bacross%2Bgenomes.%26rft_id%253Dinfo%253Apmid%252F11305938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11305938&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kowalczuk</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackiewicz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackiewicz</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nowicka</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dudkiewicz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">DNA asymmetry and the replicational mutational pressure.</span> <abbr class="cit-jnl-abbrev">J. Appl. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">553</span>–<span class="cit-lpage">577</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bapplied%2Bgenetics%26rft.stitle%253DJ%2BAppl%2BGenet%26rft.aulast%253DKowalczuk%26rft.auinit1%253DM.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D553%26rft.epage%253D577%26rft.atitle%253DDNA%2Basymmetry%2Band%2Bthe%2Breplicational%2Bmutational%2Bpressure.%26rft_id%253Dinfo%253Apmid%252F14564030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=14564030&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.47" data-doi="10.1126/science.1170160"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kudla</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tollervey</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Coding-sequence determinants of gene expression in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKudla%26rft.auinit1%253DG.%26rft.volume%253D324%26rft.issue%253D5924%26rft.spage%253D255%26rft.epage%253D258%26rft.atitle%253DCoding-Sequence%2BDeterminants%2Bof%2BGene%2BExpression%2Bin%2BEscherichia%2Bcoli%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1170160%26rft_id%253Dinfo%253Apmid%252F19359587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjQvNTkyNC8yNTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvOTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.48" data-doi="10.1096/fj.13-227330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lazrak</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bali</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartoszewski</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rab</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The silent codon change I507-ATC-&gt;ATT contributes to the severity of the DeltaF508 CFTR channel dysfunction.</span> <abbr class="cit-jnl-abbrev">FASEB J.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">4630</span>–<span class="cit-lpage">4645</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFASEB%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1096%252Ffj.13-227330%26rft_id%253Dinfo%253Apmid%252F23907436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmFzZWJqIjtzOjU6InJlc2lkIjtzOjEwOiIyNy8xMS80NjMwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzk2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">GC-content of synonymous codons profoundly influences amino acid usage</span>. <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">2027</span>–<span class="cit-lpage">2036</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D5%26rft.spage%253D2027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.50" data-doi="10.1093/oxfordjournals.molbev.a025626"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lobry</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Asymmetric substitution patterns in the two DNA strands of bacteria.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">660</span>–<span class="cit-lpage">665</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DLobry%26rft.auinit1%253DJ.%26rft.volume%253D13%26rft.issue%253D5%26rft.spage%253D660%26rft.epage%253D665%26rft.atitle%253DAsymmetric%2Bsubstitution%2Bpatterns%2Bin%2Bthe%2Btwo%2BDNA%2Bstrands%2Bof%2Bbacteria%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a025626%26rft_id%253Dinfo%253Apmid%252F8676740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjEzLzUvNjYwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzk2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackiewicz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gierlik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kowalczuk</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dudek</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cebrat</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span><span class="cit-comment">a</span> <span class="cit-article-title">Asymmetry of nucleotide composition of prokaryotic chromosomes.</span> <abbr class="cit-jnl-abbrev">J. Appl. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAppl.%2BGenet.%26rft.volume%253D40%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackiewicz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gierlik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kowalczuk</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dudek</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cebrat</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span><span class="cit-comment">b</span> <span class="cit-article-title">How does replication-associated mutational pressure influence amino acid composition of proteins?</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">409</span>–<span class="cit-lpage">416</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DMackiewicz%26rft.auinit1%253D1.%26rft.volume%253D9%26rft.issue%253D5%26rft.spage%253D409%26rft.epage%253D416%26rft.atitle%253DHow%2BDoes%2BReplication-Associated%2BMutational%2BPressure%2BInfluence%2BAmino%2BAcid%2BComposition%2Bof%2BProteins%253F%26rft_id%253Dinfo%253Apmid%252F10330120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjkvNS80MDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvOTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackiewicz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gierlik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kowalczuk</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szczepanik</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dudek</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span><span class="cit-comment">c</span> <span class="cit-article-title">Mechanisms generating long-range correlation in nucleotide composition of the <em>Borrelia burgdorferi</em> genome.</span> <abbr class="cit-jnl-abbrev">Physica A</abbr> <span class="cit-vol">273</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysica%2BA%26rft.volume%253D273%26rft.spage%253D103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.54" data-doi="10.1073/pnas.95.18.10698"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McInerney</span>  <span class="cit-name-given-names">J. O.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Replicational and transcriptional selection on codon usage in <em>Borrelia burgdorferi</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">10698</span>–<span class="cit-lpage">10703</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DMcInerney%26rft.auinit1%253DJ.%2BO.%26rft.volume%253D95%26rft.issue%253D18%26rft.spage%253D10698%26rft.epage%253D10703%26rft.atitle%253DReplicational%2Band%2Btranscriptional%2Bselection%2Bon%2Bcodon%2Busage%2Bin%2BBorrelia%2Bburgdorferi%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.95.18.10698%26rft_id%253Dinfo%253Apmid%252F9724767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTUvMTgvMTA2OTgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvOTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.55" data-doi="10.1007/PL00006325"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morton</span>  <span class="cit-name-given-names">B. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Selection on the codon bias of chloroplast and cyanelle genes in different plant and algal lineages.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">459</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DMorton%26rft.auinit1%253DB.%2BR.%26rft.volume%253D46%26rft.issue%253D4%26rft.spage%253D449%26rft.epage%253D459%26rft.atitle%253DSelection%2Bon%2Bthe%2Bcodon%2Bbias%2Bof%2Bchloroplast%2Band%2Bcyanelle%2Bgenes%2Bin%2Bdifferent%2Bplant%2Band%2Balgal%2Blineages.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FPL00006325%26rft_id%253Dinfo%253Apmid%252F9541540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/PL00006325&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9541540&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000072696800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morton</span>  <span class="cit-name-given-names">B. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Selection at the amino acid level can influence synonymous codon usage: implications for the study of codon adaptation in plastid genes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">358</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMorton%26rft.auinit1%253DB.%2BR.%26rft.volume%253D159%26rft.issue%253D1%26rft.spage%253D347%26rft.epage%253D358%26rft.atitle%253DSelection%2Bat%2Bthe%2BAmino%2BAcid%2BLevel%2BCan%2BInfluence%2BSynonymous%2BCodon%2BUsage%253A%2BImplications%2Bfor%2Bthe%2BStudy%2Bof%2BCodon%2BAdaptation%2Bin%2BPlastid%2BGenes%26rft_id%253Dinfo%253Apmid%252F11560910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU5LzEvMzQ3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzk2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morton</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorhannus</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fox</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Codon adaptation and synonymous substitution rate in diatom plastid genes.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bphylogenetics%2Band%2Bevolution%26rft.stitle%253DMol%2BPhylogenet%2BEvol%26rft.aulast%253DMorton%26rft.auinit1%253DB.%2BR.%26rft.volume%253D24%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D9%26rft.atitle%253DCodon%2Badaptation%2Band%2Bsynonymous%2Bsubstitution%2Brate%2Bin%2Bdiatom%2Bplastid%2Bgenes.%26rft_id%253Dinfo%253Apmid%252F12128023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12128023&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.58" data-doi="10.1186/1471-2164-8-369"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morton</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morton</span>  <span class="cit-name-given-names">B. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Separating the effects of mutation and selection in producing DNA skew in bacterial chromosomes.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">369</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMorton%26rft.auinit1%253DR.%2BA.%26rft.volume%253D8%26rft.spage%253D369%26rft.epage%253D369%26rft.atitle%253DSeparating%2Bthe%2Beffects%2Bof%2Bmutation%2Band%2Bselection%2Bin%2Bproducing%2BDNA%2Bskew%2Bin%2Bbacterial%2Bchromosomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-8-369%26rft_id%253Dinfo%253Apmid%252F17935620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-8-369&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17935620&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.59" data-doi="10.1073/pnas.95.7.3720"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mrazek</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karlin</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Strand compositional asymmetry in bacterial and large viral genomes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">3720</span>–<span class="cit-lpage">3725</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DMrazek%26rft.auinit1%253DJ.%26rft.volume%253D95%26rft.issue%253D7%26rft.spage%253D3720%26rft.epage%253D3725%26rft.atitle%253DStrand%2Bcompositional%2Basymmetry%2Bin%2Bbacterial%2Band%2Blarge%2Bviral%2Bgenomes%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.95.7.3720%26rft_id%253Dinfo%253Apmid%252F9520433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5NS83LzM3MjAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvOTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.60" data-doi="10.1073/pnas.84.1.166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muto</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osawa</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">The guanine and cytosine content of genomic DNA and bacterial evolution.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DMuto%26rft.auinit1%253DA.%26rft.volume%253D84%26rft.issue%253D1%26rft.spage%253D166%26rft.epage%253D169%26rft.atitle%253DThe%2Bguanine%2Band%2Bcytosine%2Bcontent%2Bof%2Bgenomic%2BDNA%2Band%2Bbacterial%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.84.1.166%26rft_id%253Dinfo%253Apmid%252F3467347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI4NC8xLzE2NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMy85NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.61" data-doi="10.1006/jmbi.1998.1921"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oresic</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shalloway</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Specific correlations between relative synonymous codon usage and protein secondary structure.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">281</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DOresic%26rft.auinit1%253DM.%26rft.volume%253D281%26rft.issue%253D1%26rft.spage%253D31%26rft.epage%253D48%26rft.atitle%253DSpecific%2Bcorrelations%2Bbetween%2Brelative%2Bsynonymous%2Bcodon%2Busage%2Band%2Bprotein%2Bsecondary%2Bstructure.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1998.1921%26rft_id%253Dinfo%253Apmid%252F9680473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1998.1921&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9680473&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000075255200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.62" data-doi="10.1038/nsmb.2466"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pechmann</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frydman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Evolutionary conservation of codon optimality reveals hidden signatures of cotranslational folding.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">243</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BStruct.%2BMol.%2BBiol.%26rft.volume%253D20%26rft.spage%253D237%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.2466%26rft_id%253Dinfo%253Apmid%252F23262490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.2466&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23262490&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.63" data-doi="10.1073/pnas.1132114100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dushoff</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Codon bias and frequency-dependent selection on the hemagglutinin epitopes of influenza A virus.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">7152</span>–<span class="cit-lpage">7157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1132114100%26rft_id%253Dinfo%253Apmid%252F12748378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAwLzEyLzcxNTIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvOTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.64" data-doi="10.1038/nrg2899"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kudla</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Synonymous but not the same: the causes and consequences of codon bias.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">32</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPlotkin%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D12%26rft.issue%253D1%26rft.spage%253D32%26rft.epage%253D42%26rft.atitle%253DSynonymous%2Bbut%2Bnot%2Bthe%2Bsame%253A%2Bthe%2Bcauses%2Band%2Bconsequences%2Bof%2Bcodon%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2899%26rft_id%253Dinfo%253Apmid%252F21102527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2899&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21102527&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285410500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.65" data-doi="10.1038/nature02458"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dushoff</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">H. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Detecting selection using a single genome sequence of <em>M. tuberculosis</em> and <em>P. falciparum</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">428</span>: <span class="cit-fpage">942</span>–<span class="cit-lpage">945</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPlotkin%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D428%26rft.issue%253D6986%26rft.spage%253D942%26rft.epage%253D945%26rft.atitle%253DDetecting%2Bselection%2Busing%2Ba%2Bsingle%2Bgenome%2Bsequence%2Bof%2BM.%2Btuberculosis%2Band%2BP.%2Bfalciparum.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02458%26rft_id%253Dinfo%253Apmid%252F15118727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02458&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15118727&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221083000041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.66" data-doi="10.1093/molbev/msl021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dushoff</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desai</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">H. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Estimating selection pressures from limited comparative data.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1457</span>–<span class="cit-lpage">1459</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl021%26rft_id%253Dinfo%253Apmid%252F16754640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjIzLzgvMTQ1NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMy85NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.67" data-doi="10.1016/j.cell.2015.02.029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Presnyak</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alhusaini</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Codon optimality is a major determinant of mRNA stability.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">1111</span>–<span class="cit-lpage">1124</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D160%26rft.spage%253D1111%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.02.029%26rft_id%253Dinfo%253Apmid%252F25768907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.02.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25768907&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.68" data-doi="10.1016/j.molcel.2015.05.035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quax</span>  <span class="cit-name-given-names">T. E. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Claassens</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soll</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Oost</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Codon bias as a means to fine-tune gene expression.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">149</span>–<span class="cit-lpage">161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D59%26rft.spage%253D149%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2015.05.035%26rft_id%253Dinfo%253Apmid%252F26186290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2015.05.035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26186290&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.69" data-doi="10.1101/gr.1358104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rispe</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delmotte</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Ham</span>  <span class="cit-name-given-names">R. C. H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moya</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Mutational and selective pressures on codon and amino acid usage in <em>Buchnera</em>, endosymbiotic bacteria of aphids.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">44</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1358104%26rft_id%253Dinfo%253Apmid%252F14672975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjE0LzEvNDQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvOTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.70" data-doi="10.1101/gr.2896904"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rocha</span>  <span class="cit-name-given-names">E. P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Codon usage bias from tRNA’s point of view: redundancy, specialization, and efficient decoding for translation optimization.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2279</span>–<span class="cit-lpage">2286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.2896904%26rft_id%253Dinfo%253Apmid%252F15479947%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNC8xMS8yMjc5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzk2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.71" data-doi="10.1046/j.1365-2958.1999.01334.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rocha</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danchin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viari</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Universal replication biases in bacteria.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DRocha%26rft.auinit1%253DE.%2BP.%26rft.volume%253D32%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D16%26rft.atitle%253DUniversal%2Breplication%2Bbiases%2Bin%2Bbacteria.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.1999.01334.x%26rft_id%253Dinfo%253Apmid%252F10216855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.1999.01334.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10216855&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000079767100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-book" id="cit-7.3.967.72"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanderson</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Armadillo: an open source C++ linear algebra library for fast prototyping and computationally intensive experiments</span>, <span class="cit-source"><em>Technical Report</em>.</span> <span class="cit-publ-name">NICTA</span>, <span class="cit-publ-loc">Sydney</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.73" data-doi="10.1038/nrg3051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sauna</span>  <span class="cit-name-given-names">Z. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimchi-Sarfaty</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Understanding the contribution of synonymous mutations to human disease.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">683</span>–<span class="cit-lpage">691</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DSauna%26rft.auinit1%253DZ.%2BE.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253D683%26rft.epage%253D691%26rft.atitle%253DUnderstanding%2Bthe%2Bcontribution%2Bof%2Bsynonymous%2Bmutations%2Bto%2Bhuman%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3051%26rft_id%253Dinfo%253Apmid%252F21878961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21878961&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.74"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schlierkamp-voosen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muhlenbein</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Strategy adaptation by competing subpopulations</span>. Proceedings of Parallel Problem Solving from Nature III, Jerusalem, Israel, pp. <span class="cit-fpage">199</span>–<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-book" id="cit-7.3.967.75"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cannarozzi</span>  <span class="cit-name-given-names">G. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Background</span>, pp. <span class="cit-fpage">3</span>–<span class="cit-lpage">11</span> in <span class="cit-source">Codon Evolution - Mechanisms and Models</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Cannarozzi</span>  <span class="cit-name-given-names">G. M.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">A.</span></span>. <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.76" data-doi="10.1016/j.cell.2013.05.049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niemczyk</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kudla</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Rate-limiting steps in yeast protein translation.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">1589</span>–<span class="cit-lpage">1601</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D153%26rft.spage%253D1589%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.05.049%26rft_id%253Dinfo%253Apmid%252F23791185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.05.049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23791185&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000320722000025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.77" data-doi="10.1371/journal.pone.0033547"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shao</span>  <span class="cit-name-given-names">Z. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">X. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J. Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Synonymous codon ordering: a subtle but prevalent strategy of bacteria to improve translational efficiency.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e33547</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DShao%26rft.auinit1%253DZ.%2BQ.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253De33547%26rft.epage%253De33547%26rft.atitle%253DSynonymous%2Bcodon%2Bordering%253A%2Ba%2Bsubtle%2Bbut%2Bprevalent%2Bstrategy%2Bof%2Bbacteria%2Bto%2Bimprove%2Btranslational%2Befficiency.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0033547%26rft_id%253Dinfo%253Apmid%252F22432034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0033547&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22432034&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.78" data-doi="10.1007/BF02099948"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">An evolutionary perspective on synonymous codon usage in unicellular organisms.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">28</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D24%26rft.issue%253D1-2%26rft.spage%253D28%26rft.epage%253D38%26rft.atitle%253DAn%2Bevolutionary%2Bperspective%2Bon%2Bsynonymous%2Bcodon%2Busage%2Bin%2Bunicellular%2Borganisms.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02099948%26rft_id%253Dinfo%253Apmid%252F3104616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02099948&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3104616&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986F784400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.79"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">The rate of synonymous substitution in enterobacterial genes is inversely related to codon usage bias.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">222</span>–<span class="cit-lpage">230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DSharp%26rft.auinit1%253DP.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253D222%26rft.epage%253D230%26rft.atitle%253DThe%2Brate%2Bof%2Bsynonymous%2Bsubstitution%2Bin%2Benterobacterial%2Bgenes%2Bis%2Binversely%2Brelated%2Bto%2Bcodon%2Busage%2Bbias%26rft_id%253Dinfo%253Apmid%252F3328816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjc6IjQvMy8yMjIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvOTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.80" data-doi="10.1126/science.2683084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shields</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Chromosomal location and evolutionary rate variation in Enterobacterial genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">246</span>: <span class="cit-fpage">808</span>–<span class="cit-lpage">810</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSharp%26rft.auinit1%253DP.%26rft.volume%253D246%26rft.issue%253D4931%26rft.spage%253D808%26rft.epage%253D810%26rft.atitle%253DChromosomal%2Blocation%2Band%2Bevolutionary%2Brate%2Bvariation%2Bin%2Benterobacterial%2Bgenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.2683084%26rft_id%253Dinfo%253Apmid%252F2683084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNDYvNDkzMS84MDgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvOTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.81" data-doi="10.1093/nar/gki242"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailes</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grocock</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peden</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sockett</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Variation in the strength of selected codon usage bias among bacteria.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1141</span>–<span class="cit-lpage">1153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D33%26rft.issue%253D4%26rft.spage%253D1141%26rft.epage%253D1153%26rft.atitle%253DVariation%2Bin%2Bthe%2Bstrength%2Bof%2Bselected%2Bcodon%2Busage%2Bbias%2Bamong%2Bbacteria%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki242%26rft_id%253Dinfo%253Apmid%252F15728743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjMzLzQvMTE0MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMy85NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.82"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shields</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Silent sites in <em>Drosophila</em> genes are not neutral—evidence of selection among synonymous codons.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">704</span>–<span class="cit-lpage">716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DShields%26rft.auinit1%253DD.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253D704%26rft.epage%253D716%26rft.atitle%253D%2522Silent%2522%2Bsites%2Bin%2BDrosophila%2Bgenes%2Bare%2Bnot%2Bneutral%253A%2Bevidence%2Bof%2Bselection%2Bamong%2Bsynonymous%2Bcodons%26rft_id%253Dinfo%253Apmid%252F3146682%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjc6IjUvNi83MDQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvOTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.83" data-doi="10.1007/PL00006459"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sorhannus</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fox</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Synonymous and nonsynonymous substitution rates in diatoms: a comparison between chloroplast and nuclear genes.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">209</span>–<span class="cit-lpage">212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DSorhannus%26rft.auinit1%253DU.%26rft.volume%253D48%26rft.issue%253D2%26rft.spage%253D209%26rft.epage%253D212%26rft.atitle%253DSynonymous%2Band%2Bnonsynonymous%2Bsubstitution%2Brates%2Bin%2Bdiatoms%253A%2Ba%2Bcomparison%2Bbetween%2Bchloroplast%2Band%2Bnuclear%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FPL00006459%26rft_id%253Dinfo%253Apmid%252F9929389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/PL00006459&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9929389&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000078325600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.84" data-doi="10.1093/molbev/msl166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stoletzki</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Synonymous codon usage in <em>Escherichia coli</em>: selection for translational accuracy.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">374</span>–<span class="cit-lpage">381</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl166%26rft_id%253Dinfo%253Apmid%252F17101719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI0LzIvMzc0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzk2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.85" data-doi="10.1371/journal.pgen.1001004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Supek</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skunca</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Repar</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vlahovicek</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smuc</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Translational selection is ubiquitous in prokaryotes.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSupek%26rft.auinit1%253DF.%26rft.volume%253D6%26rft.issue%253D6%26rft.spage%253De1001004%26rft.epage%253De1001004%26rft.atitle%253DTranslational%2Bselection%2Bis%2Bubiquitous%2Bin%2Bprokaryotes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001004%26rft_id%253Dinfo%253Apmid%252F20585573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20585573&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.86"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morton</span>  <span class="cit-name-given-names">B. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Codon adaptation of plastid genes.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0154306</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0154306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.87" data-doi="10.1002/pro.5560050814"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thanaraj</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Argos</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Ribosome-mediated translational pause and protein domain organization.</span> <abbr class="cit-jnl-abbrev">Protein Sci.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1594</span>–<span class="cit-lpage">1612</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtein%2BScience%26rft.stitle%253DProtein%2BSci.%26rft.aulast%253DTHANARAJ%26rft.auinit1%253DT.%2BA.%26rft.volume%253D5%26rft.issue%253D8%26rft.spage%253D1594%26rft.epage%253D1612%26rft.atitle%253DRibosome-mediated%2Btranslational%2Bpause%2Band%2Bprotein%2Bdomain%2Borganization%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpro.5560050814%26rft_id%253Dinfo%253Apmid%252F8844849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/pro.5560050814&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8844849&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VA13800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-book" id="cit-7.3.967.88"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tijms</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-source">A <em>F</em>irst <em>C</em>ourse in <em>S</em>tochastic <em>P</em>rocesses.</span> <span class="cit-publ-name">John Wiley &amp; Sons Ltd.</span>, <span class="cit-publ-loc">Hoboken, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.89"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tillier</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The contributions of replication orientation, gene direction, and signal sequences to base-composition asymmetries in bacterial genomes.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DTillier%26rft.auinit1%253DE.%2BR.%26rft.volume%253D50%26rft.issue%253D3%26rft.spage%253D249%26rft.epage%253D257%26rft.atitle%253DThe%2Bcontributions%2Bof%2Breplication%2Borientation%252C%2Bgene%2Bdirection%252C%2Band%2Bsignal%2Bsequences%2Bto%2Bbase-composition%2Basymmetries%2Bin%2Bbacterial%2Bgenomes.%26rft_id%253Dinfo%253Apmid%252F10754068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10754068&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086183400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.90" data-doi="10.1016/j.cell.2010.03.031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tuller</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carmi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vestsigian</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorfan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">An evolutionarily conserved mechanism for controlling the efficiency of protein translation.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">344</span>–<span class="cit-lpage">354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DTuller%26rft.auinit1%253DT.%26rft.volume%253D141%26rft.issue%253D2%26rft.spage%253D344%26rft.epage%253D354%26rft.atitle%253DAn%2Bevolutionarily%2Bconserved%2Bmechanism%2Bfor%2Bcontrolling%2Bthe%2Befficiency%2Bof%2Bprotein%2Btranslation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2010.03.031%26rft_id%253Dinfo%253Apmid%252F20403328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2010.03.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20403328&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276738400023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.91" data-doi="10.1093/molbev/mst051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wallace</span>  <span class="cit-name-given-names">E. W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Airoldi</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Estimating selection on synonymous codon usage from noisy experimental data.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1438</span>–<span class="cit-lpage">1453</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst051%26rft_id%253Dinfo%253Apmid%252F23493257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjMwLzYvMTQzOCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMy85NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.92"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">How optimized is the translational machinery in <em>Escherichia coli</em>, <em>Salmonella typhimurium</em> and <em>Saccharomyces cerevisiae</em>?</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">37</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DXia%26rft.auinit1%253DX.%26rft.volume%253D149%26rft.issue%253D1%26rft.spage%253D37%26rft.epage%253D44%26rft.atitle%253DHow%2BOptimized%2BIs%2Bthe%2BTranslational%2BMachinery%2Bin%2BEscherichia%2Bcoli%252C%2BSalmonella%2Btyphimurium%2Band%2BSaccharomyces%2Bcerevisiae%253F%26rft_id%253Dinfo%253Apmid%252F9584084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTQ5LzEvMzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvOTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.93"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">X. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Maximizing transcription efficiency causes codon usage bias.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">1309</span>–<span class="cit-lpage">1320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DXia%26rft.auinit1%253DX.%26rft.volume%253D144%26rft.issue%253D3%26rft.spage%253D1309%26rft.epage%253D1320%26rft.atitle%253DMaximizing%2BTranscription%2BEfficiency%2BCauses%2BCodon%2BUsage%2BBias%26rft_id%253Dinfo%253Apmid%252F8913770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0NC8zLzEzMDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvOTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.94" data-doi="10.1007/BF00178256"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Estimating the pattern of nucleotide substitution.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DYang%26rft.auinit1%253DZ.%26rft.volume%253D39%26rft.issue%253D1%26rft.spage%253D105%26rft.epage%253D111%26rft.atitle%253DEstimating%2Bthe%2Bpattern%2Bof%2Bnucleotide%2Bsubstitution.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00178256%26rft_id%253Dinfo%253Apmid%252F8064867%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00178256&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8064867&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994NQ00800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-book" id="cit-7.3.967.95"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-source"><em>Computational Molecular Evolution</em>.</span> <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.96" data-doi="10.1093/molbev/msm284"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mutation-selection models of codon substitution and their use to estimate selective strengths on codon usage.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">568</span>–<span class="cit-lpage">579</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm284%26rft_id%253Dinfo%253Apmid%252F18178545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI1LzMvNTY4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzk2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference  in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.97" data-doi="10.1016/j.tim.2009.01.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zalucki</span>  <span class="cit-name-given-names">Y. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beacham</span>  <span class="cit-name-given-names">I. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jennings</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Biased codon usage in signal peptides: a role in protein export.</span> <abbr class="cit-jnl-abbrev">Trends Microbiol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">146</span>–<span class="cit-lpage">150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bmicrobiology%26rft.stitle%253DTrends%2BMicrobiol%26rft.aulast%253DZalucki%26rft.auinit1%253DY.%2BM.%26rft.volume%253D17%26rft.issue%253D4%26rft.spage%253D146%26rft.epage%253D150%26rft.atitle%253DBiased%2Bcodon%2Busage%2Bin%2Bsignal%2Bpeptides%253A%2Ba%2Brole%2Bin%2Bprotein%2Bexport.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2009.01.005%26rft_id%253Dinfo%253Apmid%252F19307122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2009.01.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19307122&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265477600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference  in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.98" data-doi="10.1016/j.ygeno.2013.04.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shao</span>  <span class="cit-name-given-names">Z. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">L. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">X. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mao</span>  <span class="cit-name-given-names">Y. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Non-random arrangement of synonymous codons in archaea coding sequences.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">362</span>–<span class="cit-lpage">367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.volume%253D101%26rft.spage%253D362%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2013.04.008%26rft_id%253Dinfo%253Apmid%252F23603537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2013.04.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23603537&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F967.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319844800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference  in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.967.99" data-doi="10.1093/molbev/msp070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weems</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilke</span>  <span class="cit-name-given-names">C. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Translationally optimal codons associate with structurally sensitive sites in proteins.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1571</span>–<span class="cit-lpage">1580</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp070%26rft_id%253Dinfo%253Apmid%252F19349643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI2LzcvMTU3MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMy85NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/3/967.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/3/953" title="Genome-Wide Analysis of lncRNA and mRNA Expression During Differentiation of Abdominal Preadipocytes in the Chicken" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/3/983" title="Identification of Unannotated Small Genes in Salmonella" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/3"><span>Volume 7 Issue 3, March 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/3.cover-source.jpg?itok=h8yj2GBo" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/3/967" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-t4nDvS1W4SH9eJWclpbVCdK_nXk6KneIyW4TbF0Ql0E" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/3/967" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/3/967" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/3/967" class="active" data-icon-position="" data-hide-link-title="0">The Impact of Selection at the Amino Acid Level on the Usage of Synonymous Codons</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/483527" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-2rUDOnNyB80G6SwGXNY79bnwPyzQxBRW--I9Q1aASvo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/3/967.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/3/967" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ypgb9cZ_RdoL_2kbjMuNg-n52ahVCYsHvNGz3nUYhqg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/3/967" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/3/967.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483527" id="node483527--2" data-pisa="ggg;7/3/967" data-pisa-master="ggg;g3.116.038125" data-apath="/ggg/7/3/967.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Impact of Selection at the Amino Acid Level on the Usage of Synonymous Codons</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Paweł Błażej</span>, <span class="highwire-citation-author" data-delta="1">Dorota Mackiewicz</span>, <span class="highwire-citation-author" data-delta="2">Małgorzata Wnętrzak</span> and <span class="highwire-citation-author" data-delta="3">Paweł Mackiewicz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">967-981;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.038125 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483527--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Paweł Błażej </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genomics, Faculty of Biotechnology, University of Wrocław, 50-383, Poland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pawe%C5%82%2BB%C5%82a%C5%BCej%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=B%C5%82a%C5%BCej%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APawe%25C5%2582%2BB%25C5%2582a%25C5%25BCej%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dorota Mackiewicz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genomics, Faculty of Biotechnology, University of Wrocław, 50-383, Poland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dorota%2BMackiewicz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mackiewicz%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADorota%2BMackiewicz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Małgorzata Wnętrzak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genomics, Faculty of Biotechnology, University of Wrocław, 50-383, Poland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ma%C5%82gorzata%2BWn%C4%99trzak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wn%C4%99trzak%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMa%25C5%2582gorzata%2BWn%25C4%2599trzak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paweł Mackiewicz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genomics, Faculty of Biotechnology, University of Wrocław, 50-383, Poland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pawe%C5%82%2BMackiewicz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mackiewicz%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APawe%25C5%2582%2BMackiewicz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:pamac@smorfland.uni.wroc.pl" class="" data-icon-position="" data-hide-link-title="0">pamac@smorfland.uni.wroc.pl</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/3/967" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/3/967&amp;title=The%20Impact%20of%20Selection%20at%20the%20Amino%20Acid%20Level%20on%20the%20Usage%20of%20Synonymous%20Codons" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/3/967&amp;title=The%20Impact%20of%20Selection%20at%20the%20Amino%20Acid%20Level%20on%20the%20Usage%20of%20Synonymous%20Codons" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/3/967&amp;title=The%20Impact%20of%20Selection%20at%20the%20Amino%20Acid%20Level%20on%20the%20Usage%20of%20Synonymous%20Codons" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/3/967&amp;text=The%20Impact%20of%20Selection%20at%20the%20Amino%20Acid%20Level%20on%20the%20Usage%20of%20Synonymous%20Codons" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/3/967&amp;title=The%20Impact%20of%20Selection%20at%20the%20Amino%20Acid%20Level%20on%20the%20Usage%20of%20Synonymous%20Codons" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/3/967&amp;t=The%20Impact%20of%20Selection%20at%20the%20Amino%20Acid%20Level%20on%20the%20Usage%20of%20Synonymous%20Codons" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/3/967&amp;title=The%20Impact%20of%20Selection%20at%20the%20Amino%20Acid%20Level%20on%20the%20Usage%20of%20Synonymous%20Codons" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/3/967&amp;title=The%20Impact%20of%20Selection%20at%20the%20Amino%20Acid%20Level%20on%20the%20Usage%20of%20Synonymous%20Codons" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/483527/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483527" id="node483527--3" data-pisa="ggg;7/3/967" data-pisa-master="ggg;g3.116.038125" data-apath="/ggg/7/3/967.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Impact of Selection at the Amino Acid Level on the Usage of Synonymous Codons</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Paweł Błażej</span>, <span class="highwire-citation-author" data-delta="1">Dorota Mackiewicz</span>, <span class="highwire-citation-author" data-delta="2">Małgorzata Wnętrzak</span> and <span class="highwire-citation-author" data-delta="3">Paweł Mackiewicz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">967-981;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.038125 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483527--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Paweł Błażej </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genomics, Faculty of Biotechnology, University of Wrocław, 50-383, Poland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pawe%C5%82%2BB%C5%82a%C5%BCej%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=B%C5%82a%C5%BCej%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APawe%25C5%2582%2BB%25C5%2582a%25C5%25BCej%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dorota Mackiewicz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genomics, Faculty of Biotechnology, University of Wrocław, 50-383, Poland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dorota%2BMackiewicz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mackiewicz%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADorota%2BMackiewicz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Małgorzata Wnętrzak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genomics, Faculty of Biotechnology, University of Wrocław, 50-383, Poland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ma%C5%82gorzata%2BWn%C4%99trzak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wn%C4%99trzak%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMa%25C5%2582gorzata%2BWn%25C4%2599trzak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paweł Mackiewicz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genomics, Faculty of Biotechnology, University of Wrocław, 50-383, Poland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pawe%C5%82%2BMackiewicz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mackiewicz%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APawe%25C5%2582%2BMackiewicz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:pamac@smorfland.uni.wroc.pl" class="" data-icon-position="" data-hide-link-title="0">pamac@smorfland.uni.wroc.pl</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=967&amp;rft.epage=981&amp;rft.atitle=The%20Impact%20of%20Selection%20at%20the%20Amino%20Acid%20Level%20on%20the%20Usage%20of%20Synonymous%20Codons&amp;rft.volume=7&amp;rft.issue=3&amp;rft.date=2017-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=B%C5%82a%C5%BCej%2C+Pawe%C5%82&amp;rft.au=Mackiewicz%2C+Dorota&amp;rft.au=Wn%C4%99trzak%2C+Ma%C5%82gorzata&amp;rft.au=Mackiewicz%2C+Pawe%C5%82"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/483527/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/483527/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/483527/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/483527/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/483527/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/483527/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/483527/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/483527/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/483527/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/483527/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/483527/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/483527/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/3/967" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/3/967.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/3/967.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/3/967.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/3/967.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
