<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/3/843" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/3/843.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/3/843.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/3/843.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/3/843.atom" />
<meta name="HW.pisa" content="ggg;7/3/843" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Linkage Mapping and Comparative Genomics of Red Drum (Sciaenops ocellatus) Using Next-Generation Sequencing" />
<meta name="DC.Identifier" content="10.1534/g3.116.036350" />
<meta name="DC.Date" content="2017-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Hollenbeck et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Developments in next-generation sequencing allow genotyping of thousands of genetic markers across hundreds of individuals in a cost-effective manner. Because of this, it is now possible to rapidly produce dense genetic linkage maps for nonmodel species. Here, we report a dense genetic linkage map for red drum, a marine fish species of considerable economic importance in the southeastern United States and elsewhere. We used a prior microsatellite-based linkage map as a framework and incorporated 1794 haplotyped contigs derived from high-throughput, reduced representation DNA sequencing to produce a linkage map containing 1794 haplotyped restriction-site associated DNA (RAD) contigs, 437 anonymous microsatellites, and 44 expressed sequence-tag-linked microsatellites (EST-SSRs). A total of 274 candidate genes, identified from transcripts from a preliminary hydrocarbon exposure study, were localized to specific chromosomes, using a shared synteny approach. The linkage map will be a useful resource for red drum commercial and restoration aquaculture, and for better understanding and managing populations of red drum in the wild." />
<meta name="DC.Contributor" content="Christopher M. Hollenbeck" />
<meta name="DC.Contributor" content="David S. Portnoy" />
<meta name="DC.Contributor" content="Dana Wetzel" />
<meta name="DC.Contributor" content="Tracy A. Sherwood" />
<meta name="DC.Contributor" content="Paul B. Samollow" />
<meta name="DC.Contributor" content="John R. Gold" />
<meta name="article:published_time" content="2017-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Linkage Mapping and Comparative Genomics of Red Drum (Sciaenops ocellatus) Using Next-Generation Sequencing" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Developments in next-generation sequencing allow genotyping of thousands of genetic markers across hundreds of individuals in a cost-effective manner. Because of this, it is now possible to rapidly produce dense genetic linkage maps for nonmodel species. Here, we report a dense genetic linkage map for red drum, a marine fish species of considerable economic importance in the southeastern United States and elsewhere. We used a prior microsatellite-based linkage map as a framework and incorporated 1794 haplotyped contigs derived from high-throughput, reduced representation DNA sequencing to produce a linkage map containing 1794 haplotyped restriction-site associated DNA (RAD) contigs, 437 anonymous microsatellites, and 44 expressed sequence-tag-linked microsatellites (EST-SSRs). A total of 274 candidate genes, identified from transcripts from a preliminary hydrocarbon exposure study, were localized to specific chromosomes, using a shared synteny approach. The linkage map will be a useful resource for red drum commercial and restoration aquaculture, and for better understanding and managing populations of red drum in the wild.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/03/01" />
<meta name="citation_mjid" content="ggg;7/3/843" />
<meta name="citation_id" content="7/3/843" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/3/843" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/3/843.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/3/843.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/3/843.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.036350" />
<meta name="citation_pmid" content="28122951" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="843" />
<meta name="citation_lastpage" content="850" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Christopher M. Hollenbeck" />
<meta name="citation_author_institution" content="Marine Genomics Laboratory, Department of Life Sciences, Texas A&amp;M University, Corpus Christi, Texas 78412" />
<meta name="citation_author_email" content="ch263@st-andrews.ac.uk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0227-7225" />
<meta name="citation_author" content="David S. Portnoy" />
<meta name="citation_author_institution" content="Marine Genomics Laboratory, Department of Life Sciences, Texas A&amp;M University, Corpus Christi, Texas 78412" />
<meta name="citation_author" content="Dana Wetzel" />
<meta name="citation_author_institution" content="Mote Marine Laboratory and Aquarium, Sarasota, Florida 34236" />
<meta name="citation_author" content="Tracy A. Sherwood" />
<meta name="citation_author_institution" content="Mote Marine Laboratory and Aquarium, Sarasota, Florida 34236" />
<meta name="citation_author" content="Paul B. Samollow" />
<meta name="citation_author_institution" content="Department of Veterinary Integrative Biosciences, Texas A&amp;M University, College Station, Texas 77845" />
<meta name="citation_author" content="John R. Gold" />
<meta name="citation_author_institution" content="Marine Genomics Laboratory, Department of Life Sciences, Texas A&amp;M University, Corpus Christi, Texas 78412" />
<meta name="citation_reference" content="citation_journal_title=Aquatic Toxicology;citation_journal_abbrev=Aquatic Toxicology;citation_author=M. D. C. Alvarez;citation_author=L. A. Fuiman;citation_title=Environmental levels of atrazine and its degradation products impair survival skills and growth of red drum larvae.;citation_pages=229-241;citation_volume=74;citation_year=2005;citation_issue=3;citation_pmid=16009435;citation_doi=10.1016/j.aquatox.2005.05.014" />
<meta name="citation_reference" content="citation_journal_title=Environmental Toxicology and Chemistry;citation_journal_abbrev=Environmental Toxicology and Chemistry;citation_author=M. D. C. Alvarez;citation_author=L. A. Fuiman;citation_title=Ecological performance of red drum (Sciaenops ocellatus) larvae exposed to environmental levels of the insecticide malathion.;citation_pages=1426-1432;citation_volume=25;citation_year=2006;citation_issue=5;citation_pmid=16704078" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Amores;citation_author=J. Catchen;citation_author=A. Ferrara;citation_author=Q. Fontenot;citation_author=J. H. Postlethwait;citation_title=Genome Evolution and Meiotic Maps by Massively Parallel DNA Sequencing: Spotted Gar, an Outgroup for the Teleost Genome Duplication;citation_pages=799-808;citation_volume=188;citation_year=2011;citation_issue=4;citation_pmid=21828280;citation_doi=10.1534/genetics.111.127324" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Amores;citation_author=J. Catchen;citation_author=I. Nanda;citation_author=W. Warren;citation_author=R. Walter;citation_title=A RAD-tag genetic map for the platyfish (Xiphophorus maculatus) reveals mechanisms of karyotype evolution among teleost fish.;citation_volume=197;citation_year=2014;citation_pmid=24700104;citation_doi=10.1534/genetics.114.164293" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. L. Anderson;citation_author=A. Rodríguez Marí;citation_author=I. Braasch;citation_author=A. Amores;citation_author=P. Hohenlohe;citation_title=Multiple sex-associated regions and a putative sex chromosome in zebrafish revealed by RAD mapping and population genomics.;citation_pages=e40701-e40701;citation_volume=7;citation_year=2012;citation_issue=7;citation_pmid=22792396;citation_doi=10.1371/journal.pone.0040701" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=A. D. Ball;citation_author=J. Stapley;citation_author=D. A. Dawson;citation_author=T. R. Birkhead;citation_author=T. Burke;citation_title=A comparison of SNPs and microsatellites as linkage mapping markers: lessons from the zebra finch (Taeniopygia guttata).;citation_pages=218-218;citation_volume=11;citation_year=2010;citation_pmid=20359323;citation_doi=10.1186/1471-2164-11-218" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. W. Baxter;citation_author=J. W. Davey;citation_author=J. S. Johnston;citation_author=A. M. Shelton;citation_author=D. G. Heckel;citation_title=Linkage mapping and comparative genomics using next-generation RAD sequencing of a non-model organism.;citation_pages=e19315-e19315;citation_volume=6;citation_year=2011;citation_issue=4;citation_pmid=21541297;citation_doi=10.1371/journal.pone.0019315" />
<meta name="citation_reference" content="citation_journal_title=Evol. Appl.;citation_author=I. R. Bradbury;citation_author=S. Hubert;citation_author=B. Higgins;citation_author=S. Bowman;citation_author=T. Borza;citation_title=Genomic islands of divergence and their consequences for the resolution of spatial structure in an exploited marine fish.;citation_pages=450-461;citation_volume=6;citation_year=2013;citation_pmid=23745137;citation_doi=10.1111/eva.12026" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=K. R. Bradnam;citation_author=J. N. Fass;citation_author=A. Alexandrov;citation_author=P. Baranay;citation_author=M. Bechner;citation_title=Assemblathon 2: evaluating de novo methods of genome assembly in three vertebrate species.;citation_pages=10;citation_volume=2;citation_year=2013;citation_pmid=23870653;citation_doi=10.1186/2047-217X-2-10" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Camacho;citation_author=G. Coulouris;citation_author=V. Avagyan;citation_author=N. Ma;citation_author=J. Papadopoulos;citation_title=BLAST+: architecture and applications.;citation_pages=421-421;citation_volume=10;citation_year=2009;citation_pmid=20003500;citation_doi=10.1186/1471-2105-10-421" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=J. M. Catchen;citation_author=A. Amores;citation_author=P. Hohenlohe;citation_author=W. Cresko;citation_author=J. H. Postlethwait;citation_title=Stacks: building and genotyping loci de novo from short-read sequences.;citation_pages=171-182;citation_volume=1;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=Y. Chutimanitsakun;citation_author=R. W. Nipper;citation_author=A. Cuesta-Marcos;citation_author=L. Cistué;citation_author=A. Corey;citation_title=Construction and application for QTL analysis of a Restriction Site Associated DNA (RAD) linkage map in barley.;citation_pages=4-4;citation_volume=12;citation_year=2011;citation_pmid=21205322;citation_doi=10.1186/1471-2164-12-4" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Biol. Lett.;citation_author=C. Diaz-Gil;citation_author=I. A. Catalan;citation_author=M. Palmer;citation_author=C. K. Faulk;citation_author=L. A. Fuiman;citation_title=Ocean acidification increases fatty acids levels of larval fish.;citation_pages=201503;citation_volume=11;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=J. L. Fierst;citation_title=Using linkage maps to correct and scaffold de novo genome assemblies: methods, challenges, and computational tools.;citation_pages=1-8;citation_volume=6;citation_year=2015;citation_pmid=25674101;citation_doi=10.3389/fgene.2015.00001" />
<meta name="citation_reference" content="citation_journal_title=Contrib. Mar. Sci.;citation_author=J. R. Gold;citation_author=K. M. Kedzie;citation_author=D. Bohymeyer;citation_author=J. D. Jenkin;citation_author=W. J. Karel;citation_title=Studies on the basic structure of the red drum (Sciaenops ocellatus) genome.;citation_pages=57-62;citation_volume=30;citation_year=1988" />
<meta name="citation_reference" content="citation_journal_title=Transactions of the American Fisheries Society;citation_author=J. R. Gold;citation_author=M. Liang;citation_author=E. Saillant;citation_author=P. S. Silva;citation_author=R. R. Vega;citation_title=Genetic Effective Size in Populations of Hatchery-Raised Red Drum Released for Stock Enhancement.;citation_pages=1327-1334;citation_volume=137;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. Gonen;citation_author=N. R. Lowe;citation_author=T. Cezard;citation_author=K. Gharbi;citation_author=S. C. Bishop;citation_title=Linkage maps of the Atlantic salmon (Salmo salar) genome derived from RAD sequencing.;citation_pages=166;citation_volume=15;citation_year=2014;citation_pmid=24571138;citation_doi=10.1186/1471-2164-15-166" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=F. Henning;citation_author=H. J. Lee;citation_author=P. Franchini;citation_author=A. Meyer;citation_title=Genetic mapping of horizontal stripes in Lake Victoria cichlid fishes: benefits and pitfalls of using RAD markers for dense linkage mapping.;citation_pages=5224-5240;citation_volume=23;citation_year=2014;citation_doi=10.1111/mec.12860" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=P. A. Hohenlohe;citation_author=S. Bassham;citation_author=M. Currey;citation_author=W. A. Cresko;citation_title=Extensive linkage disequilibrium and parallel adaptive divergence across threespine stickleback genomes;citation_pages=395-408;citation_volume=367;citation_year=2012;citation_issue=1587;citation_pmid=22201169;citation_doi=10.1098/rstb.2011.0245" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=C. M. Hollenbeck;citation_author=D. S. Portnoy;citation_author=J. R. Gold;citation_title=A genetic linkage map of red drum (Sciaenops ocellatus) and comparison of chromosomal syntenies with four other fish species.;citation_pages=265-274;citation_volume=435;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=C. M. Hollenbeck;citation_author=D. S. Portnoy;citation_author=J.R. Gold;citation_title=A method for detecting recent changes in contemporary effective population size from linkage disequilibrium at linked and unlinked loci.;citation_pages=207-216;citation_volume=117;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=R. Houston;citation_author=J. Davey;citation_author=S. Bishop;citation_author=N. Lowe;citation_author=J. Mota-Velasco;citation_title=Characterisation of QTL-linked and genome-wide restriction site-associated DNA (RAD) markers in farmed Atlantic salmon.;citation_pages=244-244;citation_volume=13;citation_year=2012;citation_pmid=22702806;citation_doi=10.1186/1471-2164-13-244" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. A. Khan;citation_author=Y. Han;citation_author=Y. F. Zhao;citation_author=M. Troggio;citation_author=S. S. Korban;citation_title=A Multi-Population Consensus Genetic Map Reveals Inconsistent Marker Order among Maps Likely Attributed to Structural Variations in the Apple Genome.;citation_pages=e47864-e47864;citation_volume=7;citation_year=2012;citation_issue=11;citation_pmid=23144832;citation_doi=10.1371/journal.pone.0047864" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. Krzywinski;citation_author=J. Schein;citation_author=İ. Birol;citation_title=Circos: an information aesthetic for comparative genomics.;citation_volume=19;citation_year=2009;citation_pmid=19541911;citation_doi=10.1101/gr.092759.109" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=Z. Liu;citation_author=J. Cordes;citation_title=DNA marker technologies and their applications in aquaculture genetics.;citation_pages=1-37;citation_volume=238;citation_year=2004;citation_doi=10.1016/j.aquaculture.2004.05.027" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=T. Manousaki;citation_author=A. Tsakogiannis;citation_author=J. B. Taggart;citation_author=C. Palaiokostas;citation_author=D. Tsaparis;citation_title=Exploring a non-model teleost genome through RAD sequencing - linkage mapping in common pandora, Pagellus erythrinus and comparative genomic analysis.;citation_pages=509-519;citation_volume=6;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=G. McKinney;citation_author=L. Seeb;citation_author=W. Larson;citation_author=D. Gomez-Uchida;citation_author=M. Limborg;citation_title=An integrated linkage map reveals candidate genes underlying adaptive variation in Chinook salmon (Oncorhynchus tshawytscha).;citation_pages=769-783;citation_volume=16;citation_year=2015" />
<meta name="citation_reference" content="citation_title=Fishes of the World;citation_author=J. S. Nelson;citation_author=T. C. Grande;citation_author=M. V. H. Wilson;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=C. Palaiokostas;citation_author=M. Bekaert;citation_author=A. Davie;citation_author=M. E. Cowan;citation_author=M. Oral;citation_title=THE COMPLEX TRANSCRIPTIONAL LANDSCAPE OF THE ANUCLEATE HUMAN PLATELET.;citation_pages=1-12;citation_volume=14;citation_year=2013;citation_issue=1;citation_pmid=23323973;citation_doi=10.1186/1471-2164-14-1" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. Palaiokostas;citation_author=M. Bekaert;citation_author=M. G. Q. Khan;citation_author=J. B. Taggart;citation_author=K. Gharbi;citation_title=Mapping and validation of the major sex-determining region in Nile tilapia (Oreochromis niloticus L.) using RAD sequencing.;citation_pages=e68389;citation_volume=8;citation_year=2013;citation_pmid=23874606;citation_doi=10.1371/journal.pone.0068389" />
<meta name="citation_reference" content="citation_title=Distribution and Abundance of Fishes and Invertebrates in Gulf of Mexico Estuaries, Volume II: Species Life History Summaries.;citation_author=M. Pattillo;citation_author=T. Czapla;citation_author=D. Nelson;citation_author=M. Monaco;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=B. K. Peterson;citation_author=J. N. Weber;citation_author=E. H. Kay;citation_author=H. S. Fisher;citation_author=H. E. Hoekstra;citation_title=Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species.;citation_pages=e37135-e37135;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22675423;citation_doi=10.1371/journal.pone.0037135" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=D. S. Portnoy;citation_author=J. B. Puritz;citation_author=C. M. Hollenbeck;citation_author=J. Gelsleichter;citation_author=D. Chapman;citation_title=Selection and sex-biased dispersal in a coastal shark: the influence of philopatry on adaptive variation.;citation_pages=5877-5885;citation_volume=24;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=PeerJ;citation_author=J. B. Puritz;citation_author=C. M. Hollenbeck;citation_author=J. R. Gold;citation_title=dDocent: a RADseq, variant-calling pipeline designed for population genomics of non-model organisms.;citation_pages=e431;citation_volume=2;citation_year=2014;citation_pmid=24949246;citation_doi=10.7717/peerj.431" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Z. I. Talukder;citation_author=L. Gong;citation_author=B. S. Hulke;citation_author=V. Pegadaraju;citation_author=Q. Song;citation_title=A high-density SNP map of sunflower derived from RAD-sequencing facilitating fine-mapping of the rust resistance gene R12.;citation_pages=e98628;citation_volume=9;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=M. Tine;citation_author=H. Kuhl;citation_author=P.-A. Gagnaire;citation_author=B. Louro;citation_author=E. Desmarais;citation_title=European sea bass genome and its variation provide insights into adaptation to euryhalinity and speciation.;citation_pages=5770;citation_volume=5;citation_year=2014;citation_pmid=25534655;citation_doi=10.1038/ncomms6770" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. W. van Ooijen;citation_title=Multipoint maximum likelihood mapping in a full-sib family of an outbreeding species.;citation_pages=343-349;citation_volume=93;citation_year=2011;citation_issue=5;citation_pmid=21878144;citation_doi=10.1017/S0016672311000279" />
<meta name="citation_reference" content="citation_title=JoinMap 4.1, Software for the Calculation of Genetic Linkage Maps in Experimental Populations of Diploid Species;citation_author=J. W. van Ooijen;citation_year=2012" />
<meta name="citation_reference" content="citation_title=ggplot2: Elegant Graphics for Data Analysis;citation_author=H. Wickham;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=S. C. Willis;citation_author=C. M. Hollenbeck;citation_author=J. B. Puritz;citation_author=J. R. Gold;citation_author=D. S. Portnoy;citation_title=Haplotyping RAD loci: an efficient method to filter paralogs and account for physical linkage.;citation_year=2016;citation_doi=10.1111/1755-0998.12647" />
<meta name="citation_reference" content="citation_journal_title=IEEE/ACM Trans. Comput. Biol. Bioinformatics;citation_author=Y. Wu;citation_author=T. J. Close;citation_author=S. Lonardi;citation_title=Accurate construction of consensus genetic maps via integer linear programming.;citation_pages=381-394;citation_volume=8;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Xu;citation_author=M. S. Ackerman;citation_author=H. Long;citation_author=L. Bright;citation_author=K. Spitze;citation_title=A male-specific genetic map of the microcrustacean Daphnia pulex based on single sperm whole-genome sequencing.;citation_volume=201;citation_year=2015;citation_issue=1;citation_pmid=26116153;citation_doi=10.1534/genetics.115.179028" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Linkage Mapping and Comparative Genomics of Red Drum (Sciaenops ocellatus) Using Next-Generation Sequencing" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3.cover-source.jpg" />
<meta name="twitter:description" content="Developments in next-generation sequencing allow genotyping of thousands of genetic markers across hundreds of individuals in a cost-effective manner. Because of this, it is now possible to rapidly produce dense genetic linkage maps for nonmodel species. Here, we report a dense genetic linkage map for red drum, a marine fish species of considerable economic importance in the southeastern United States and elsewhere. We used a prior microsatellite-based linkage map as a framework and incorporated 1794 haplotyped contigs derived from high-throughput, reduced representation DNA sequencing to produce a linkage map containing 1794 haplotyped restriction-site associated DNA (RAD) contigs, 437 anonymous microsatellites, and 44 expressed sequence-tag-linked microsatellites (EST-SSRs). A total of 274 candidate genes, identified from transcripts from a preliminary hydrocarbon exposure study, were localized to specific chromosomes, using a shared synteny approach. The linkage map will be a useful resource for red drum commercial and restoration aquaculture, and for better understanding and managing populations of red drum in the wild." />
<meta name="og-title" property="og:title" content="Linkage Mapping and Comparative Genomics of Red Drum (Sciaenops ocellatus) Using Next-Generation Sequencing" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/3/843" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Developments in next-generation sequencing allow genotyping of thousands of genetic markers across hundreds of individuals in a cost-effective manner. Because of this, it is now possible to rapidly produce dense genetic linkage maps for nonmodel species. Here, we report a dense genetic linkage map for red drum, a marine fish species of considerable economic importance in the southeastern United States and elsewhere. We used a prior microsatellite-based linkage map as a framework and incorporated 1794 haplotyped contigs derived from high-throughput, reduced representation DNA sequencing to produce a linkage map containing 1794 haplotyped restriction-site associated DNA (RAD) contigs, 437 anonymous microsatellites, and 44 expressed sequence-tag-linked microsatellites (EST-SSRs). A total of 274 candidate genes, identified from transcripts from a preliminary hydrocarbon exposure study, were localized to specific chromosomes, using a shared synteny approach. The linkage map will be a useful resource for red drum commercial and restoration aquaculture, and for better understanding and managing populations of red drum in the wild." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3.cover-source.jpg" />
    <title>Linkage Mapping and Comparative Genomics of Red Drum (Sciaenops ocellatus) Using Next-Generation Sequencing | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"I6czneWFdfQSC-NuBq2ZU1vmKZe0sTOnznxyWO-tMZI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"483531","apath":"\/ggg\/7\/3\/843.atom","pisa":"ggg;7\/3\/843","ac":{"\/ggg\/7\/3\/843.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/3\/843.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/3\/843"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/3\/843"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/3\/843.abstract":true,"\/content\/7\/3\/843":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022pP0E1fQUPfflBukGbzVqyP83AM+TegxUzCh2gysxrmZ8b1ExUf6J2CMn5x7xahRRFTOViS96fy5CSudu2pbhlCQYzBqob4YApyCmCLfaKzimy830N9TPkwVDhg+Vyhi6FtnfPaZNcfLZOm9uhblFuWTwGXnx5X+NXooIM7zGtd7gxtv2OQGkgOlCYOVAnHb\\\\\\\/nDn069kwKiXUONLg0aGY37eWaBpEZtNOwPj+kR0GMXMX8ogrrrZkeQuVDf1ubllbs9r4BhSlYC85jFtJu8iA6yh+wHpmR652qP32E8iHMYEwd4RJJg5JFliOnK+D3m2aTEgLNGE3cPSnhelFHT7vIgAbSUmO4+wBORk9jR4DGvydyB121P4pfrGIM53ezKE8Km\\\\\\\/4Ot\\\\\\\/xC2w4qLRiyny8lDhSt9WGl+KAH+G\\\\\\\/hfrvf7996ZvwvJ3o1kkKcV3F405keHsnsi8j70jshn9QXsf84k5BKDnLfVeeKDOgw99W+2ajR2eOx0Pl3CFZhcHtBpjNc6ptquDfwKFgq9\\\\\\\/\\\\\\\/DulRc\\\\\\\/8pygaQJfRJdm9uQKn4qq5Vm7bpO0Z471E+WMiiidgIzOhpDRo5HH95lIDHRzwZ6QI9S8XwDBxgIF2\\\\\\\/jrCEofT6KrXYjTBHBx91TOmejtf+A02Py+n2xB8n3+myHBdWVgObzNN7HpgqSwFE8wxhOk2T6Ij+JjRu6iIWb40KAinEjsOW5d4UpGGJHehHzRhv5\\\\\\\/UJYfHPJNIrOuH9fK66Jwc2kKzqHlw0h+0Q2prGrKsJsdmux2hypsbLY9feVUuT\\\\\\\/XmCHDgGUhg6xhXazjZCs0xcQWnlrMzKvUerBIIYiuC2orbtVxJDJGOTTPcSBMBpRg7rLTN+CzBf2\\\\\\\/o9BRS8ERkiYETaxFyXyrqjSMzHwTdfXLyPYf0vT\\\\\\\/WRSWy0ai8a4hEayzH2UC56R+JipOfs3VRr9fjuia5acJQ5SavjUpvaVhk1noCyGDgZCV59rdtPfAXsZ+hVmCh0VVy\\\\\\\/QXKSxW5LBnJknlsQy0I\\\\\\\/wF7POoG594wYjuREToHMNK7MD3P9Dl65H3XXo0dWiAUYp6+9kC5+SsW3RBx8sjpKVaahoa1SfXvnRnL5ZzAswlKOIcOT0RSQ5659REaxvCq0zu8R3rwuF8pgvViGGw9SLLV0RFBAoMQR3cLl9N93TubK0FjAYhZMaY\\\\\\\/h8KRscJRFXCYMSiwPdMRn05sesLoUBR2ixNNgHy90XPwTjQ5bZyJPil1Z9DpN+3ztTZyRsygC1\\\\\\\/mHITrfZL9vJQrwIkoKSZeCEhk6\\\u0022,\\\u0022iv\\\u0022:\\\u0022eDtcbXsV3kB8oIKtO4hfbA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226fba22bd40c37e6529e4d398453eb0b6aced735b1e3c443786f2d35a12f43801\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022nWkTB+ciGB38OhnVvh6fFWsdwmXSMymeq4JaqTH2EH\\\\\\\/fCIiaZoTCLJoq1KNQrZ1Z8xGkMBcX+E9NG\\\\\\\/JdjYzynryopS19GjMga66iYD9IsYyz4oPFmUS\\\\\\\/gJcGDgvtKLm2GLO9hbnTjuuA7buYfQ2AH0xpK7jWNEuQ9Z5X8u0MeUk7MUbyqiE+Iqw46cSaJaPqnlfuRArhC+iILE\\\\\\\/tCxZHUclMO2B+bwm9SOlU8XKLeA4va\\\\\\\/iL4hNK\\\\\\\/iqBHWKSON01fRlJopnjq+0nqw7d2GW8LCEDgi286uwXb+Ua8oKWX7H8CaY58NXMM4MXwYRWmnRXLVLDLmZEedVOQbuf0l747mmPzQ655n7m+8EZa41OviIoXhfiw305Y5pTn8cpuabOt5WQHvAwoD3ll3eS8GttRxRtgquELhFc1vBFlbyKgsnIokVQxJXgzlTn8Evbdjy+V1\\\\\\\/8E0YZqHadHd17qlQNLVVpKxccee2aX1zPb2JcoqgY41h+QykhEXvKAxjVY9qpqc+KbtAw88TOJhuK3VuR+v+niZUT7QWbWol\\\\\\\/7UZjW+tbYFtsJThTinI+kN0P4WAG4dA1q1x2dH6b5F7oNWVJFqqWWuex4BHVdkFH+1tktbL9gCCCF3s2KrblaVoGXYfYWQ8Lemk\\\\\\\/10iQQ0N6riGM\\\\\\\/L+YGmt9tuuGZgMCwXFm0+y0NWAAnOdK9746Px0ed6G6zg\\\\\\\/f5l66bzZgoATZmPDVht6Bicqfge4w5+k0d3nu67tDotl99jDxHOqn7G2+AXRafijWq9xaE9+0ycFnNzNPQxuYBGLe3EHdaZI1w9uy5K+dhUAO56vGsrvFen5s9ZQwLEYFQ6sOP1gCOxTs1jysu8BbLLJcn0O2ca3uJIPMXkiBuU+m651jEEYEPmFk7yktpwzsK45UJqLsGQNh8XfjkYzbWYkSHdznA2prJIbBxnCJ+geXVOmZ4qwJR2mfGt5t\\\\\\\/kKVdr+R+BUCMGpS7q8lqdfU4twJGtSSJbeN6RJJAY4qBGD6SZh8Fr7c+R9KdAXm4\\\\\\\/MM5JWVSEilvCO3h043ct6SGp3LPpRGxymi\\\\\\\/hgkM+aLhUWUBG0SUP4DUWEk1Y8GdH7z6L99SqWmThhQD5lLPXt5B8Q2Ib0UCPZGxn7UwhE2tfWks4Fjd53lqLB7jrV2HD7fVliGIlHanMTrTAyKk\\\\\\\/mLo5hSrxhQ1rM4a7VIPm7J8R73cXOCJ+K36p5wAlaZzG8R2\\\\\\\/Fskbbb54UDu8LSLcuA3T5KuwiFH+GAo+icZ90W4U1FMRQQwl7Rn0EEv7NWvnaGP8iwMMvn+Rzboclo0ung2YGraMMJll0m3JC5ALNQibbhETjZhhMZtgRG3gsbMn\\\\\\\/rnIpScrgt8ZRTyZKC4ZXwNtDgs+qIPvp1XRCBDCBkWtBP7ghLOm3Z\\\\\\\/r0EW2PfVZr8wgWgkNbbSyhtvqTriBZNsnuTMo86eZ62tCN34fcb+6zmF9s4+wSgmQARZd43Qz2UTjlLwdpLQybiFpfA37pPm7ZGhe86sk+YcU+hmzAJgV4NuFiR2o5gHvjJTuhRwQMr2\\\\\\\/NAEs\\\\\\\/v0sUeHCLIqsPulfafMuVMHMSXK2AQaPf2\\\\\\\/\\\\\\\/IiXdl1O9j0B1iK5BzE42d\\\\\\\/uhU\\\\\\\/DZnQ0mDOG7M2wUdClQgJ99Vn1mBaPa+TuRm6TDYTunW8XRSyDebIzN+osSCPHn6E\\\\\\\/YBk6+hGcUR\\\\\\\/FRY0ERbwKxc1lGH1j2AfG6xAi7xNy1TTfgOr\\\\\\\/pit\\\\\\\/dv95vpaKFb9Yfagy9vylmRN04lw2\\\\\\\/WMann\\\\\\\/zHJzuIQrNaPoEesmOqW5BNTIYGmfsbUE5jNEGMvhiEyVTKXSFj\\\\\\\/gq3NK5o4yzBCeznDg79ViHKAoEOQDw54VI1TwiZ7FYH99UNBlaKManXytWKDiu+wXnNw6VGXZbhCsFArcYR6c2TdTIbFVtMNgO3sjXPFTg1AK0v3uTtRYa2nM7uzqAEIZv5ER5kQYWy1RcgT4L\\\\\\\/DAvPeXX06gzkEcWgiZhjr5C\\\\\\\/h6P0rbrXWtjnIC2b5VPnFPX6Iror5VFB5PlhqJVFKuLgnjU1Jki7+tnuZUQNgYabechgq+8ecKpsHeaSK0YvtNY+yWxTscQjuAeiuhPmJD25vr\\\\\\\/mNSu+ggtG7BJVt5E3maYD6wgmqX1BNtuIUT7Fi2CWmm9mllSwGD7+zAYXSXUg3ItqmApt5VFmkQC6CzAz11HEhQPVcNY9x+c60czxwD9Ww22x6seoQRllqIGrsGuhWRKNgn9c6WgF4K2IsmmXCnpQFLsUEm9IToRiQsGMZU7kAO7+k6fTeozv7UpZhq6VsX2Y3KsPe6h0PUuOelAx8sKa65l5kfcGPM9XXNjqDBJdxQO5C3RMPyDy7qQEXstTZCeWHOMjiOBTybf3pV+QB0YoFgxE90RgyWfLehDGTAPoxaEDYEDa3ehUV0axLYyV6vKt4txZotn6sS8h7WW3AJOCAUgMInaWdhsA7RK5zThOU+EzZN9jSgA3hFZMD5c0Gfc2UijDlaA6XCH9C56gA4xOvp76ltWaBUxdxUoYDWVu5CcRINVSKknnwqU6lWl5L6Jnz6VQ7pEajLQ7NrkTOfx18RSkYDxnNh9C5vSUsHpf76OoR\\\\\\\/xc9kGrBpE\\\\\\\/PYn3Wv+D5RU3jvInmppNGx4vIz+QqeI0l6zgFPZUQjkfzJLW61Km\\\\\\\/v5Curb46pOTBJzDjRTmMRcueBbzRbm\\\\\\\/G1Y4UHt2fjvrOkWiIISsMMv0EofOeYAdJmlGyyqehqtZ5vZfrcDCEgqike0Ikb8B9HF6FngLCC4qdW8ezN6Z2Xtc61kp9r\\\\\\\/uGpSnuRQeEU\\\\\\\/G5hAmb6Pt93EjuLYPmZyjPT+7VIG1ja+KtJSNcfNtWDklZLjq0fl+dcy8VmGMyyfhZjYyXtNNAeV+BgseBY7QMMvWGKJHQ5MPDRL5xjIWFfjKP\\\\\\\/XN4ESiyNcbrsJoDQM6w6O071MC1rHVGs0KOG20iw+4p5Bc\\\\\\\/rjjvTK24LMITRJI3ZXIeu9DhA6m8ledt9tydi1u2qGBy\\\\\\\/Xbc+63FN\\\\\\\/95nM5HHKLWSLsYXYMrWJTrxSyjnaJTtjef6fCfoV4q0vc9izK2Id+PF9X0jwwccxS8a+Uxl4fSIf7BDYyteR8E9MO0V+ylcCDdtRwpHCVZW7rt7y4qIaqmLJtqrJmiZjkNOeiCqwPrjcMk+t4TVKNZRlZekUd5IXH7SwDJM25uO9OraXSVXhItPhJ\\\\\\\/J\\\\\\\/tHVbj5bkNi9EhLDY\\\\\\\/D6I6tsxsbuWpVreVE7FFlbfQzOaLT3K7L0RVU5mXUixH5z2Xc4YifcfQ8bobWRd8wfwD3tmrJm0yxSNNqDwFXa4lLTROa8XejTrn55AnE3A4VOf+YK3fT4Bg0OltopxoEdKk+aJPf7wqVWofGfZmVulTIzabxlvSTyxFCp+6lWyL0dq8GEmX0w\\\\\\\/b9HiU5I5U68nHtZk5\\\\\\\/duSSuARJSBprlxCwcabXJG5xhVPzTI7MaJB0UFDpIP9wJ+Q9ARXUhVHC0G\\\\\\\/pvMyLZ\\\\\\\/rMmnUf0No8IYUGZPjz7HUCmlxr35pVpNJPMOEWy+5BI8XWKH5Dh\\\\\\\/sySoe4TOEhMLH1OtiEAGPNT3v8XEB5DrEaZtKcDCsWfQXSqmFfuLkSCQvAaqD0o\\\\\\\/W+KelZcIRAFqQPXQBERSm\\\\\\\/HNm\\\\\\\/WxQbu1berarb3DrFAX28Dq1vLgrn2ShIZxVkQ4xn4cn6Hqeaz5y+W33vwqSfmYIB15HbOlMiV4VI71Xsf0ELNMKOp+OG3da3Fdxas1hyYRjYQNpmLY8UGdP9k9mb91f\\\\\\\/93FMkmrBDtKhV7EbqLa8ysZvMSnW89BksXWsZGtO0diYfYYaTYgtw2FvFhjJHOXwjVs26mti1lUj3bwUXkF8H8HQvcCrtTHK8eQ5bj7NFZuEjRQ6zqd+snKB8xYdI+V\\\\\\\/LLWno5OF6oOVDn5ylM6vREGp57dKdegau++sVXKvG6nxLQ+ApK3OWPSuVvJRhfq+h69iXeR0BdlrDxMHrc3GRjVewYYetMjN2ctQb+am7TUkuVElM1k3hxyAiV23HySEwzP+hMy1ryjXFDcLSbXX02DhXwC84zaLvvfRJ3Y1g2f03b6BKOE\\\\\\\/e6a1X\\\\\\\/fbJv0mP79xFQKXeNwYT+RQ91EvM7Fz\\\\\\\/DofkqfS+hvLFUu51uqpzWHrJeaj1LL5Qgp6KKMPFmTtVvKjtkSQTO04HeqTqBrxw3ULRZLc9MaYNAx3sKfQPWWqD2v7ERp+oB1YjoUQtcN+eUatUXkwT6oBkS6p2H49SH7qipcQsPqcRDR59DJ1QKwkY7pMPiS3G0lS4\\\\\\\/JRVNzOr5DtOqtgsMTPHbMvm3QxxuYeLlF+u0QZWqf75e\\\\\\\/hWubIYyNetaO9FiX0dSq51y\\\\\\\/7hcnwsBJrG1zYTMOZdX3fRGQ8Zg6IA76A4jfloRDFNdRPSPDt4wmR+4Z37gYwzCR5ydSMsEJ0wD2cldx9nXeYRc7ZRkJmnLVhfhT50d5KHcxqt716hD3TcBSnoVEytYxRnYON1ztGr129N2vhughpSKdNM76+qXuAFKCL4n1Lip9ZbPduDpY\\\\\\\/dfJcRq1ghpJ\\\\\\\/uJ5ytyKvwU1Rt6XZ3M6OzzyTXuBNHYWWHfIrbG86NFKPYJOLCAsvde50dwWX6BnF5LTuBt4hjReMC\\\\\\\/RsnmPU8eDS77X9IBGnONED5yxRgZpHwK4dWy+aBhbNo88ChOwHr+kTGHNFFpI\\\\\\\/P5nfKeXc0d5zxm8J15tJPZl2G0KS+gqujaDjbrthDNryci4Q3\\\\\\\/fJQsz9DRl2kaXY1auAxtN5VIuG21CIZvT3B9QRHtSaLqVeE\\\\\\\/JL6o5PfxKQkVGugt30y4R4Vr5HGPcAEcc6+bTFGEuO9lsEq0Mvwh4sV\\\\\\\/jdB3gvt7kME0UqPl3tRfMUDTqYclYsYnsTkCtqDJVJBC\\\\\\\/jKreXIgMrGcWjlIqWbzJiraKJXUFmGtxk2saZJ184urjFobnRwQuxfwrsvUt+5aAnMiOICysNVT9EA15ytoNCDSnau+UBSxxEmJCnlgppMqUpJMTr9VLW0HZasO\\\\\\\/pnxXv6yUnAOvnQLOhCUJH4RHSQH0fKAFQmoS8WK52JF8iJDa0i3TmXzzjMDDSjdFz02JGGZJSbcx\\\\\\\/6mml28CJz7AeEcoUzJ2qAOWIaKJLUE4fakUlTk0Dqy\\\\\\\/RK14\\\\\\\/xtV8nt\\\\\\\/rFYyjrttBZtIQJY6q+aWO\\\\\\\/ith2sR5vklg6xxBQuIFH0zFRQLFWd0CrScd\\\\\\\/oEQstbS95HS7NrCU2xrjCPfQZTmsjyu6AHlKfn5Z\\\\\\\/ils2IOW3s3t090Fl6ysbN3ze7JbWjsLzUp8Rjnq7hvxqX2nOrJ\\\\\\\/GpSVaTlqIiCnJuAwbbwBg9ZgcuUOZcSNvdYEmcnFpOI7zAoxCHD1CADOIuz2TsgS2igFBsbFP5PM5vPjwDKM53bKm7pjUjAd+IOhUetYUTUzQvgPFQWwQvOFXPzHBwR8Oy8hBKaBQaocQwmlgKb1yvt++qUqu4cuarYycMdnRRlYSH\\\\\\\/4XfxO+jnjajtOvgikH7oW+A3USexMPzgKilJyyl6Mj34rLjdrCKBHSN4cExk7jUHqaisFuvHlp4ap6UpXH\\\\\\\/mH3P1tF\\\\\\\/UA2eZIqR+0vz6wyWwb7FFmvyU1oTQQVmHOob+9KbjrWnqi6JTwIXujDAT2fKxRFERGl21GeREPLSll\\\\\\\/RH+0irHtZwFS0iXxCfUYn\\\\\\\/Kq43hQW0iP62TqR4XuvRlUjHxcB4tm2oPX6g1CxhaOSOWav6dcXojFykasRLPUYTCJYcw7h0VS3M5Ey3UxRrERVsNxTbFQWtXGs0XEjH7k31wvLtEYL3AZNO6+ay9qCnhQQaUyqPZ4zeegiI3cvI+0rFO8Fp4CdFUpPw\\\\\\\/+VjfQVKsKM\\\\\\\/UgzfSbV3KpjEkSJ+7+KPPVJ6sDxJB5RET9iO591s5AG5Ts4Pu+GOebEQJr65sYcUc9HxupE9wr46tRI2NdRHEZQ4AMMYAnciMrC5eJCr8WgZCG5hjjUHqFzMkZEZWorHTRJ1DWtOOAjEiqoAlEsovQa50VGjQ1f4SAYyQW3F6FJftMko4zpDiUUtecIaUx8ji0Rnx0rOAaBd3mRTHSTr\\\\\\\/VHdk1W3R73acEVOC9xjSsq+r59NZgrlNNUed9b7sYbqBzRF1EubGx3Ue4TPtVZ2j\\\\\\\/VFIvPFLgedCDW5wWTHXXQtdQlrA4IKas7wRECC8ZYgzhYwmYHvEaHJUV8NZ+7eBg7csnDEn\\\\\\\/nOiQQKscPSuHnPnL8tYPcNtf+52tvRFATjWo+Q3orCZD1iOfy2ztlSmQ0K9iTQwOKK3xXUM3oGB7dClgoC82FvzFJsNxnjwmKREKdM2dkHg+q4NGMSueXfDUPxxHToc69\\\\\\\/tq11VuonXaSZGFN4ZBljONorZwKll7ptgxbHSwpeHOmtPQwhQC4vb\\\\\\\/\\\\\\\/qu5erWArr1PNV8\\\\\\\/ytYWAfURULOGQEzXIvaQ8AmKzpmv\\\\\\\/o488ZCqiuXMQp5FcgQLnwqDUxbQ7d+tuw4vqRqm4JOxllxlBBuwYelacnlDiiy0O3Rzfu1BUnfCI8bm\\\\\\\/K8OTN6JCtenMcLSdelqO\\\\\\\/3F6fLfFLOIcoshO25f0tokLA0vaqga\\\\\\\/Ia1YwOFbj2T+euOGkZPUL3rTICStK3gY0lK6A\\\\\\\/ELfmQ3PQiTame\\\\\\\/ZvLsRuPHYanEsLIdSppFrQaFAEVITyn5qhHBEDPL0YVphSHhdAVRMKPjMHA4k+zK\\\\\\\/XABUH7Fmn9MnnbQUcYGwW\\\\\\\/Ib6l1c+5e3nQPv71tJt2qbqoRgpMcflJXdsaQPdLGbA5aL\\\\\\\/75woMYENa77mb4qH\\\\\\\/DXf73ssfPO2\\\\\\\/wa94y4eEId9pDiMrwKFH5JaxTdEgWJXA9r6eovxFLEwwyKnmWuc7v4ZMIl+\\\\\\\/y98Cp4gItoPnVMeiW9Ju\\\\\\\/3Pr4\\\\\\\/wBlWVPepFPLnx\\\\\\\/VoZihJAac0yPTg5bkD3oMGhtoNBjziMa0pJBcUHsxDbRpVZ2VMGUAO4a6+Ti11aii4ddY2QZUO\\\\\\\/63H\\\\\\\/SbbrbSAk+rr41hdAR22VAxLbh98TNAK\\\\\\\/2M0FxtDnmNS7r4BKJ6TWKFlqJknvcH5F5Dk+2cuBGBDLo\\\\\\\/HKHEZkKXbEOEQpQDkEhpPc2WxX\\\\\\\/Okbu0MCvLtTCtktTqO7vLz0o1ibAZOPk3NJOosLtvSThW+AVgNemmWs+qzz0OdkFawRgyOUg1UsVfodDHKz3Nf+IJxeyPQ6QFtO1nGBQZdr1cZBKTdCN9fqlOIuJkMCuUeyZQtgE\\\\\\\/K+AE+VlgwoIXg+EF3iQBYtqE5iKLFQPFkjMcYY4lALyYKMmliR7+ajb30vrdNntBkF2MzUz3xCU92tnnSg53d2Yv+9W3UullqpKT57JVuwqP1CaPc\\\\\\\/Ys1D1EO2XNrnqT+03qRZ2PoYIcBzfT9J\\\\\\\/8nqVU7ZUH9wGqwyaGv1nRMQdXb7151KQHjmZLFDnkD68KXvzDWDxirZS+32j9oHFF1nZ\\\\\\\/4eYxXpl\\\\\\\/dCrz83rAP6GpM49xJCTQZZZrZJnawRcfUOfJao5Fy16IqaefGVwmKqHcr7J\\\\\\\/\\\\\\\/AgH9dKd+Y7EE0Tboge9GZa9mYh3vYuyCK3UFdWeMPvoGWBSFty\\\\\\\/it0d\\\\\\\/w65uGrE0aQg3Rqav28FVPl4syqn+N85aLCaEMnnqA0cAGchRzJEN29sdbB0SL9w8Gol067+trAIGTzLu+V9DqEpQjoNWPNGuC55\\\\\\\/UNYXF9kdrHGAsTRgOOOFq1TcfLNYBCRCngQu2YmO4IrA2SA8pFCnO6GNr+CfmweCjSFysJ+enGSKApa6jb0K6+ThmNHZudPiPWYh4hmd5OUe85shXcYmvS1o\\\\\\\/0wafoFHHqM\\\\\\\/18qfZAuZZizWzP5S9peVbYPJLrJe7i0X8OK1Y\\\\\\\/NaP2ba5Aic1NiyivST7mEwt1sNZqDM8aSZzWYwnazk84DyWGSXbI\\\\\\\/TP3Ro7oHR+5512gSOBAQID9xMlhSnCwxL3BmmZNMdsu7Hm29sfXo1X1OrtsoEzcT\\\\\\\/W6frwdZS0\\\\\\\/XMC+ME8W75CY3rHBwVBgDfnMzV8A2Xq0Tgk18Ooy4wpfepAcLJiCEyXts+ojgNXO6ZxCiigjqUPC4y56tsDtqjbeAa4JhKUIbWje6gXSp9FrZpUJ4sUYzoFRsLYgx8MHtEi+OogIAYHyQoREjk7Ni1GTeyRBDVcAaY02D5Wfr3RpmoSW\\\\\\\/TZZFfZnNPJOxytKOiPyZ1c+r6aeyY7YFJbhkSm6B8w4WlfvrRJE+6lEMzze9OY1g2bmpPLLk1UZPYhZrDppV86djlZgplYIis6gJecbnibHjCEcXqB+J0N9SFR2MJFmezU0YtlavJvSh2eC6710pV1zfKC9hF47akPwomnBw21LSh7zE23B7gPJbZJdfiK3XpPcICjhb4q4SjFqW76yhEz1qTJBjhgeP4pv+eu+RCx5VTC+hlbufdXtsH1iuznkPutieh0hQro1rTUEcuW4p5ptSMqYCRMLPSVD\\\\\\\/Kgqdgb9Ia3jpgSzENnEu5ZpKdLHYyCCAEOUKCKtzxpYs05PgLyNBF50MhJGJ3n1JCT77wy9aMEF72IXFX8tCH16v\\\\\\\/XkBkEOQ8+wJBodB+ti9SyACJyqCkC64+UhYsQjKaGcwx6JiGH9tbtd2y2qkE9trNEhp3Sj6mxVL0OfQxK69MJ6QPR8W6qeVnqSp3fgjs9xhqpBr1T\\\\\\\/mYCFxKEVSONurAtBBS8OFDStj0+URLwE39q2dzjx2CB1FOuala\\\\\\\/3XYAERs\\\\\\\/D98IZ6n\\\\\\\/YOo\\\\\\\/zcCfse8uVDVBsRKZVxzvPz9PDu8XDmD5KaAujp+vh2QGX5Y\\\\\\\/REoXpmhatdAxFJk4RYVp+orrbP+pXyleJHo0bB368rIXzs066Y4SGQTaGOovMG6TL+OBOyK6jrv+tlQbuk2b9ZkOZCge6fv\\\\\\\/dhhmqWTosr7UA5ohnM2DZNkKc6IRdkvLO5vS7HT4ESunoRwt18bhLhap8UDxovYxBfgFY8qhgwJslhynGhhK7uUr2gPvofcz8u6maavs465yBi8MJXVZc4OwgSVoLQWea7En\\\\\\\/GrGfmb6XSZ4ctVsz7q7k7ddUeB3HcP8fAAdFSWdNA+8tZ7p6q1\\\\\\\/JIRyT8XozInG1B1XPPgTEw76qljDQkAgWNnOeNixgM0Mvx22kSFKOFNvvDQXDTZydnS5RttVit4Dtug0JiSqT21BfdYLVSHKl\\\\\\\/h3slAHk51QaF91lO58WRMUx7z6f30Iu2zLnSwHlN5\\\\\\\/V90yLpaplxjYzhk3bUAY8iDtMcuxi0fTZ10ugnJty1BiBgeBqWWZoL7DwDVJVLd6T4wHtrYgwUx88aZZNkKXMCiy3EeH+f7YTcDUkqtboYeCfRs8yBsL7AfE17S1jC9jM6nU4u\\\\\\\/n8lQoFlCukO78H3pDwhEWZwYc4UZ0MQoMle+sJ1hVUSaHb4gY36b4JKuSLHrgRs1PCOvB2LlaNxaKaTuTVahI8amkXR+FRmgz7akmDP4phLCNFeb7YF8HWMS9yznTyJoreKoXum4WE1Et7UVE7BakIhVoJHp\\\\\\\/aw4MhfpMs3rJU+wPmbHx2WE6Rhnho7vYjhWrPZ7TQU84rdH2SUkSw2LXLe2EKUvBquaOrmvfBz9Nj8gcMkJGM097Lf9aShqlmz\\\\\\\/aHMp8wXUqu\\\\\\\/nTXTGnBSkJgBNL+Bm7kCXY1B6rMcNsTAJls8G5yj\\\\\\\/F+nxU+BlTY5f6HVtyL5oxCZWTrbli+\\\\\\\/LZXC27onueGbzcucE5atQRORvfD3Ok7xi1tP3N3eSokjLXw1Fmc3hS8gPeIzOPFfwy7mWvlgsGqflbn\\\\\\\/R\\\\\\\/1SLpBB8V5dpCl+aerojv2P7jfP1zayaHt7Hp98cKeR1QuU49tXshArL7K9QHFArWKSfZRUfikgHY+1FDoA6GoNobdkBGsEzzs11Hl16m\\\\\\\/U8AUQLI\\\\\\\/rEf4qqDvOE8jArCuyDePNHCke7aP2V\\\\\\\/ERqZgr9MQiaICA\\\\\\\/M2TqWmAAsEup5Mk4ROo88LRy4Ev7QaI5\\\\\\\/Ibire3FSmcpQNCFuJjiuezbvE0jcZPvZWft11S0XRcNIXMpH7n7aGJqXJRltEZalcRVw117MEFJmxeDbUmcgDxprymUbRKNETJb7Efxx8\\\\\\\/zded02fFdyNXzXXEZ4JdB4AqeRNRzjavPcXulFtRgGU6gL3\\\\\\\/QehvoGVSVocJx0VkUin2ecYRQn7\\\\\\\/zyWVXRncFvhBpz3khy0ciFrIbhyxwEJnMmkcigw\\\\\\\/py0d1ISXAHfxgcliGUELbYYMmsqsGrnaNGS59TIaSgtUiXtpjfhlS6t\\\\\\\/aBMupz4183hW9Z7CMCGClua1tOXcrTm0FRaGsskJ9LFjUs7cxiB7YhDF6ZBLX2uj\\\\\\\/pinZIBWVazM2LEn8fZgMMUgoclQi57UxmQGFYi6tAZADxh\\\\\\\/hJ24CWH7KGz9vec6A94knjxlZC5H4Ei2TxCpjim5kF85XYhLI9XEYRfrGNCySDBBcCuN5OYQsB04Mdi0Vr6FDmdx9f\\\\\\\/7u6po12z+bG+HR\\\\\\\/hE3\\\\\\\/iK5jQQ\\\\\\\/h1TSrFLOGhIcReXkcIjxujUxxXUqW3blsnRPbMYIO4e4GuvCWo5qpFa8GUVODgOb742VzI\\\\\\\/OHfncqRxkMcMau93FZ4hMA+YRGWF0\\\\\\\/NMqXWsu\\\\\\\/Dxh8Y41c9Z0q6G\\\\\\\/fomhufYztr3CPiim8lV+2JLRlcYVTeFCg3gHyQVtD5oUK38jwvCOQnCBOLC3WZ44QmUpHGomMXMEqZPHW2vSh4xI3mul9njzvyu68DnralfkVauIxO6n61nXbQ32qILHD5NHrbWAMefhOFIno0q5lKQONyoQ4YaFdGedI2pjov4lDarv34QNEKmbJW\\\\\\\/gk8Au3\\\\\\\/ZyqMxnESGjAbFHuZy7qi0yjfXzDbSW9vVpfYFmhN3pFFl47GFU3KdudCy17ZE+NUmmm\\\\\\\/QV1JCNWgYqzMDmUTjYcUHlWIB9Pq018OnSt7W4qNOdVMqWSpD5swLgBPaVyL0Yc5XPcAGVdPSi9x9aLobDEHxoJnLrfd2im9sIIgA7z34aJpj+XiM\\\\\\\/H0qfriVXNoIowFbhjNjAGtdV\\\\\\\/WJuUa63zlQqNfmWWs4gy1EzmwCK6XM71512GGsHgTQW2+7ABQeRY4DXLyw6lM44vHYA1rMasXMx2NQ3KZdtOTJl8DvzFkHXLPfmtGmRsKqgwzKDw0aKP04NF0tcwIjbHTxqe8SqAlxIo8nG87SJPP4fLw5hwOVaRm5wGXQwooFY044twpqzmar7FPaiEOZ6roLbI9jCt0kVez\\\\\\\/j0YG4zMyLBcMfGoyR\\\\\\\/vvGXA5HCl4aEmIxtBJgSGJNyWAjEfJg4HjElzk7lBE4+TXxbdTmsbr9G9rcQeHOFD1b4Lo+mXC2nr2b1eCXXkrY3kzVF1\\\\\\\/dIxSoFgUoEQlWW49uxRkVpG4s5V5kyBTFTw9mgfalCctCnKnEg4dUGYSWTdP\\\\\\\/1l+eiuSiAGNBbTNIhd\\\\\\\/4TCHTjAQxfycJn2qil4pfqaOpj08VEO7e+WxjeI\\\\\\\/3XNr5dTEI0YJ9G+\\\\\\\/UHm0b9UTumcp4KPg7rSsy7KDIm+FOPMlTN36tHc4mJKD+kIqmaERQptdvqRSb+AbEH1zU9tr9rdMC7V3Alv2isfUPCgdKiUN\\\\\\\/ONvWPcnTHvQT\\\\\\\/JnQF7KM8IDosz+cuwwi3nimWE55kCEEI7JG29gUhI5no6s9Fq3DuEjzidQCD9byNYYSEzusiT956cX5nyiyDOf61O5kQzgnuzIncTbQ4Bty1JKKgCymA9+85t16uB4QOyEOrz2jglItD4lne9kulssscx+Voxo6YwAgZsD\\\\\\\/rQ4MSXoy3LsLmcLrKd9NZPaNpKMzpIiAqgGdBw7TcHzwkjxWd4LUJn9mFMsUUT8f6mq20CfpH2b5j9eDVsERa6zYl2Zk9PreAQUfkBVX2LOo05B58I9EFTi2BKobkH0N\\\\\\\/xiDse8n2qY3wa7mkNihChaYixdm\\\\\\\/yloEuWU6AP8OuIFXg8bq3rPP\\\\\\\/hEEfgLO6iAWsEmBh+rAJuK1a3v4Bov3+Wscz4P5lC5S1STqwhT43FLIZJ8sT\\\\\\\/vAZyamxZA1RYp0GAN3pjPIM+dPh3mB8QUAPKOPb4ek95LSbkhm6AzB32665tbsU+MIpMP6\\\\\\\/OQ8NtUsbxncdHas+mR46u3wCu\\\\\\\/FfReNxSEtMjYCjoD5iAcP8UVaCFAPrMshqB7jwzb8WiQpbXi9iVUqiz3XsxFlWm9oIDd\\\\\\\/VMZLjNHTBZDDZ8mtPX8jWfwKcI9ImycUtTprV59+LxP1slqn\\\\\\\/wfEj+NKfEpdDGwNpJ8wShzJRchvWcORGuH3EoEzVParLxTfxzhejyFVTKOpqC4cFv6IsAti9WYjV7aVoi9YP8OEEKpTehtOATesbEOB8eksuazxSJXcqvtxRknpogePa47b6Zt2PkeAhIvLXfT9PNNKRG4SlL0iiUBAaIzGZmOc31I2eU6qqZemRM8FuXk4739psLAWGfBD2bt64YypNFFpBUiU4jvKYfI0UbLPYCUm2GRMspcthZfBL2cNdTBIsdgB1\\\\\\\/Upb39pF+MBE5BISiMSg6NJP1O3bkOsr7MhRWWYJ\\\\\\\/1dEvjwwT2Qur0RtFdzoiZ3WwvtbvpxzI1TWt4gVpD1ZaIZn6GrMskoYLRvv4llgpglFcqfKO8spRjU+7pKgn0OmxWo57KXDMtCONbeRV7f12Xy5IGHMtYD+PUrEVteCZSq2I1SMaj5YXdWjaq3h8daIB0R35QjJAMlu497W0v0W4amXovP5qx7FnHy6ZUl1DULqplAfLN5NAVJNRMqzQqV36L+vtU0MmCixXBjVc+CxBIX\\\\\\\/AxJd\\\\\\\/INaL2AAKhkYxAYitjWtm+rAotIAP7WpjW15GuUvOdxUeRVy10ffRNt\\\\\\\/AhRPNZxn8tvq0sTyOCn45CwZZnIwn3z1A9Jh8pvBqhoDEOYmFUJo57hJJ0jjRRJyK1dJn8evG7GCjUw0nuxBlkFTpqjkVKrULX6dxdVRTZchKmPXTItqgbsOtfI4l6yb88HkXSKSZIzq3+Q2ho\\\\\\\/QiRmqE+e4c7xowR0pLCxVm4SHjVBivCtBUU5+I6X0zt2jAWvmHsyW7JEz73yKQN9G\\\\\\\/YYleAoogmdZoNqn4abtySAOhFLxzu4lE13SjfRY2frj8VB8hTqMhLH\\\\\\\/o08ANCMLyL4UQNi92EwULHUMbx1xMFBgRNXME+8WPcGZ9PECU337IqPQ9VVCdd0Mo8Er0osJTVVUzpbO7cpka+XL+1J\\\\\\\/TwCz7DOrvQ61KBuF+aTBY7Dw5bMmJdE+zg4+5CHwVcIFyxrCwbvDiHcYRKm4TU5xsYnrHNVJ8Mn2o\\\\\\\/2hhPegFHp2fwL0zZHN25aSY\\\\\\\/H7V2CD3Fd0Q8BrObmJUN7zHIHSrgWQ\\\\\\\/inOf\\\\\\\/EQ19nvo0B4BQLdJ97eSNDmHoTIdLOSkQcb5LjPi+91jKQ0qLmg4lGp34GOdYZfYtCHF2x\\\\\\\/sHegYgtLWWlzW\\\\\\\/Qn1yLTrdkW4HFiuyYEccjLqQGw1uXfyEZ7NNOGJeL9Ub0CYf+KJaMSvDkXY9Ginmn52Q+gjHzkcf\\\\\\\/\\\\\\\/6qseAuMfKGNKYp65y3DDKRNG6gNFODcRV+dhqRcC0V0gWuRmk7eEZSukd3mJ688uIW\\\\\\\/TxpeMpFNa7BlezosjILYeaLrU2a0M8vd\\\\\\\/eOOPr4plF13C6KBPXry8CKofzQ+G8V2dovxKrvpY\\\\\\\/ZhhQv79uB0B1pJmPnAazGEfhEJLPjaKEYYGXgDUznOSZxxgOZt6GZN3z3G3s4elo\\\\\\\/kYUo88NaU+K\\\\\\\/O1eWtm2f\\\\\\\/+PzzGQHo9usVD4F9cth93cOraGNuL3+3muiFg33tjbmr0fOaZqTtl56K0xrbX0oddT22Ymg1ImaGWjpe2JCIUaXQ6daZTIEFHpJFVfFWBBeM4VKRpB5BTQT\\\\\\\/pw+rP9oEi7vIoEoYvy+RdIcgd6A3An3x3daMDPerytmyXzoophPx5MuUog4AXhoi2WocJuRSKuCeAvV3bxljrgmqNTXMTEywcCTOVofNHHTqiiq38DOfh4Tq22Yp29ghiZWicB4lUK75VfRMh7zGZbDy4+WAiTsSk7a+uAhQBMl\\\\\\\/Zc5E6ys+zsvsvxscR3xVP4NP+cNa99ULuD\\\\\\\/zw8mU3M7m\\\\\\\/+dj1JbLlHEs8rIf9xlG2wMLEw4YaI4K4iOpJhgBtm+IbVw2Fd1soBifeSPNYySmwpSjG+m2zsuO0kjuwS2pwhiT5BKy7O6oc28nWWjiTrmgjoNoueUGuOj1vmSHeAELxKnCLIzVPku0eicwE7c\\\\\\\/RXxstlJU9tX8PjCS8\\\\\\\/5Uo9iSNO06A9zjAwwgnygYX9eBWV2IbuExUmdx+yC1rlqqWmAFBKWpSXlUQZNuKKfZaL0s9NGg0g1hs8qsDRan5JYKZX\\\\\\\/WDVZIgCer\\\\\\\/MFgWG132f+Jt1X47jLNi3opzFWOS2NCRcBdwLwGoLykX7W9DWKhOx4elVhQzwYrR8hzhjDlBEJKEgqV8sC4ib3a+fS5vEPQTh8HFWuRmRqGyzKMkUandBQswMb6HZXW37M5is89d3WV1X20DRca4kGl557UXwNUeqJ7z\\\\\\\/aDti0JcwW4yR\\\\\\\/26QtGyO55jysmpRd4aik3X9eFzR1nSP6ATEa8qrK7hI79HoRiCfNGgVZgh9jRgRuqjfsDsUPHWQfHqp6WhXm76YO\\\\\\\/VJuAWfeqRJRTzDaXKeWxt7892PLMsR7YmiENLliKf3ik1NNPccPIDZF+AkJIapoc5izYKL6tNq+MgUf5y3+MMJAYOX4+HkG5f8GbfJzxQrbjmc2nSHtV3gG4ldKgpEVj2RT4QfQN+RHI5GMx8O\\\\\\\/MZYIwMMSsl6n\\\\\\\/pIn+P63J6VKju7aYnWJizJVnCYr5raLoCYCp2JY02F6zio\\\\\\\/uA0u88DgjsVmzhZKKbRC1leUiHVKhKM+t2xBxhZ8h19jZoNfp0mdbo4TB7+PpdqbP7Im\\\\\\\/oKmi+NhTWtxaPBDN2J5X9yFifAijm9I3YKiM0wUIBujMHGypML8AHHxUNqdq6eQ4NQk\\\\\\\/n4Q57BCVFIR0GK\\\\\\\/KHjdIkbb9d7C+FJZDPQ3S9DsYxBTKvSVVzTLAB2B39dYnWQVoYTsqHdwXjgJ9hVOpFDOO3OeCTzJS9TZFfbscVRThQnmKQlPrrDQbVDWqVG5HCdRUACCha0vagXE9KKr8RKW7VZ6gDw\\\\\\\/bVKAcC5vl04CMaA6ysS5KvXvBXXfLiNPkUS0CYUUZkrhROeNwKgsgQ0BFVMEoNPwa2yPqA6GKowtB9fJrf9nLYa5DRDG\\\\\\\/j7s2w5N4k+axBO4XKvm5lznL9Dq1TooA\\\\\\\/V46kduOT8M7MTHEJvyiuwzLLu+CKfIrqm2eooLJg52NL6gzJlo+A+QQEZrmGRjwMHShUF0OYTmNgOuF+5c1TMBTIOSvywcwpLu5tGVqAfG4xlo8HMssCiHMFXuOeVQOrIe5+J5gD8Eu7eDIOE+QzKmTt8RZ0fPQQ4NR9J6INCiaLDj47uIhiFgSKcidk2MslirlUhAI315qozoJf9kVhRfhRqp9KwHtRf\\\\\\\/BWQA+K+HZXMzB\\\\\\\/u8FOtDrtg0OQ39KbY+cIqY+18jLPfcXkz90O0VsD5hybk2N\\\\\\\/8y9v2iqLZjftQSR07RxuntcplnWqvGHt8\\\\\\\/LEyHSTyrFZE37VQs9RPGMC+gKsxIY065QuXp0HIg2en9t1vDN6cQxYPG9N9CjH08c1Wg62kzhMTj2Xc4XODvm6EWIS23eI2QOqTRn6We9aPEx2FGb2EcYvSEEdM6NTgxTlm2QX+9GladjpvKSM5jtbYAAZJNNFXMNT1lrvt2Qo07N3A+h1e+NwuEoWMkvINY8sYkMv2Nduag3szhL7BSBLUmnOC2NOCFBAsGJ73j9SdPaSUsUyOWG8TVJP3MPpffZV0XCbxJUETkinhq2qJSEkXHPbla28WccKkDjHwKyHn+T3e95CSLIrxHmPUFiFmTId6lLx7cNJvQjfFtLwoSRMFjTYj34kc5bkARGbb+Y3In\\\\\\\/NUi3L3jl75+YKAe3Idk6TAEFnzwwgbzadx3GA\\\\\\\/Fs1f4q\\\\\\\/\\\\\\\/VEKlFA17F5Y5XW5Q9a+UvBsjWNlu8Mw8V991vEYw2bdWU9rY2A7NdzO0MRod4Zava6i4G3EfeNZuuCnXt8Y\\\\\\\/EbDNFRB0QJOuPNolYOhFSApdrX\\\\\\\/19yG1+D1Ds\\\\\\\/TGMfzgKmJJYoqW9+yEygUcpSaf32M5Sq85rbJ9Kg78LhbqP3pI3LTCN+b1wTN3eyJUkgj0bO2wu1zgdCEolZxVl\\\\\\\/bllj5gmSAdWM6uY3NODNfK80PSK9meUVzaJvAmRydhl8l+RPx0KrpxCySjXyX6l3ffrMgfrtX2Zr3\\\\\\\/5TvtVa7TLvRMiV38hVeBNNjP8pg3UQyqkcnSeHIM\\\\\\\/x9G1UcAhjogqbaxjNl1h14hLY4i2AW+LMm3Pi30hJgiF1CSgn5KRsm5QhvIBJmkR2L0\\\\\\\/xelsc6dicPvs5hRAa8SabPmj+wlVi6y45orhf3o257\\\\\\\/V4tADeJr5W\\\\\\\/djp5Lfsv4d13SaOd2Fk3sLVBtM1wwqUHeCq7MJkbUtXaaty\\\\\\\/nzzH+dEmfZMvu663TTqgofb6gXryBsQMOIAf5J2ApvF75IIA7ekZUBKhWIV2xdv4wnVqrpqr5uDKQmzw17MCdTNJ7aAJIsY2sqh2IOvkaDsnIHLx8S3Mh7XKjXEaDVIiYw\\\\\\\/uf453+DAeMcxzDduNPVv7bF2xLZxuLLNJWXMEnExJtcGcHRDcuXbI0ksp8aeguESfQf1GXKwQTPiC+a7yhG4VAS\\\\\\\/72EV5UkUVZjGGbibEjjn47VNaAouCAWFmF1u2ocpGVr1NQPJjcU4lUCRQhPWz78bQeLn3GVWbRBiS0s5FSbezSx6IZtpDVexCwpvFfaAK3G16Dd+TGPcJk0heC7vTuoXplPwl4w4gwoeWSyZqGKy87bmXMq3\\\\\\\/C63OOnnzN6L5fmsZVcs6Ab1ogVkO8QzV8RNZeVyXsloZs9NurZ\\\\\\\/LiTdIZtCT0gITAXCVIzxxmcbpxDhVwuw4QMWC2R9eln6mtpoA1e1GNLEK111EgiCmNeY6c29X\\\\\\\/cYDDeejoPAgK\\\\\\\/T9JjhS6ClTAuJtOYuMFTYlZD3a8uN8DeoTbRZZyp4vEP4BRKRyzPVHI7HYFRnfkLLiGASEmSc5btRDqbEysM76AI2UbNFuGMNMSqbbIdBcPkJ7uuq+Hn4J2rCMJhLZpiiwjPP3hTX7Y9AhUJ6el5jcjXMLb3JQAmw8xOwNLwI5m+PdHgaXkw6M0xD6ay5XCrHuhYQizDNv28i8VuOzzkUkU7ttaaGTW2eT0Z+Bf1wlbfNsc8lq6O1uu+C5j35XL\\\\\\\/EzVyuWfJZvj+mdS9ZxxJsVpY0hq8wc+VZdymwTmRHFEQ07RdsaOROHOeAlHbS9MD5oQlepp1htLErzhyeiFNCuaVha0G\\\\\\\/znfvBy76gOsawM7hxkHMptZ2AwgKUYof\\\\\\\/FgZEdtc3mOe1JtIXk\\\\\\\/gpoqh9+\\\\\\\/PeSJqGMPeCeWRvBzNSv+Rr78DhVsnVPmWDEikr5Dg+ZxtWrx7XCbINiRXxIc7Jzt3VygCURhX7J+k+lxezS6\\\\\\\/vBfMwbkciFeSJmcbj2N0Vhm0ybuby9uIeuy8514EYf7SzSiI3K6mfra6VvQQ5GRcn7MFIsCJ9GI\\\\\\\/7xqFwJifoqS+RMzAG36J9NaqtXEMKFhmctSpQjE3qg0pXy5WvkssEjo8Kcs5EB3b3S+7Ydu4VnfP22EUjKFlAX0ZkUWJoXOb2ZhLtNpewi3CIsY1yYEVXtkD60RJqsASZC5hzYss5TFIEFcJtkWbVpua0RinXjt5JGycsy3hPCoIyupyLKNkBJTrurBlVCMZq5VdMumdPDUZv3nvMVGuGIaP\\\\\\\/90H98BJoJ0v99KKKzmK0vXXj2a3OH7IrjD\\\\\\\/V0ruA6\\\\\\\/km38IynfaIIE3qkwo+0GpXZGXTAm+kuCp4lsIFRRWd9HicwCcuIBLvicTSsYRkuFnO\\\\\\\/i3SMhPnY1NVqIUZOwm6eEawQy4\\\\\\\/jX7q66DW701SI+cdQbmEXGISxIfVfvazpt3GtBa4uUkNxHoY3yoEDTzTR+tTCIXB6eVk2KiBr6O+3OPFtVUkFpsIiYyH4VmhqhHsG\\\\\\\/fv5lrTTYDqkpau7ezsNF3cYMDq9lzFSheIOn6CtWRbc2FYpRj3JRLc8ehFxjk1Hg4IirvrGQV1ETCqJHtnRxs2OS0zMYcpbl\\\\\\\/HHhDMfYjDM+osB7pYkiS7n45k3Nig1CeNSFamS\\\\\\\/OifhbfATj6vK3FHxBx1kvhmGIBZfDqnCIBpvP1i6EVjb6Jtp6J8q0iRY8XCRMOrQWMHJL8hc9qZbsNQSAay7gAitDodBpVY7e875M6HxoPcTDpavyiPtWiwi\\\\\\\/W6t6VeTrVaOfvVEpxX3u+eEwWieuYjSwa4YhX2ex64E1wt6Kku+BnWlUJRTmEAmophhiAkUNKJXnL7+1VBGfk+HOADlB9sVPhayWEwYf4k3VfvY9z\\\\\\\/ZPHveTpoWzsf8STwWK86tSmd46zwNJAT7FYZyidx2+ASgtM+98BItc37uRWU1M5Z0MEOrXkzUbMcG2PEXAEcMHRt+OO9jeFYbpUQH4MJtnE0OSzJpBcz5OMpv+P\\\\\\\/Y+hH\\\\\\\/ciD7Q3GckPjjBYYZlHRwUCRJWFXMEYklrqpEpQixqkUYskK5LJUxLENBF6WdG38Q+ZJuP2ccPZv3nYLPaZmL2k92zWYTGmQIYKvFQMZdmSYa3DRm1hcvYyUIV9KtxM0UIpn1bUs8S3MG9+FPYjFvNUxJIVVM3bOhrCmivZiQu4T3ohAv2iU+P8u\\\\\\\/XcTGxoZLEcDAqyaDr+PxYKi0tnf\\\\\\\/zXCCZ3flfHNeW9N9nmDzQNCIwQRsFJOytO09wAZPdFy3ku7hU0efOpYW+2a26KWfdR7qwF1lyWzeDKvsbKZOtI3EQjl\\\\\\\/sjqg\\\\\\\/9DoY6ZJdTglsyeDOGL0NNfcGhkNbHVtSMkU\\\\\\\/Hycaths3CJQ1hyZkRYiyTh4jiWWxDDu9pwB7IRWa3SD1PAG46gMrs4jrG4ihZAcGEEqxnFo5ucPXW4QI+WetXCqVD05Z6R2sB8PjkqWmzFyqoY1Q2IzUkVeB2K5vGKfcAqUrE5V5O33ikBqSUtgDkOTnz+On\\\\\\\/82Eep6v4YWB9e5jGQFMQdseKS7UG\\\\\\\/bX6+rxfBZ\\\\\\\/2AKdvfv6JQu5EdUxJgZlRmAwV5kl9uv0qqBjJhxYpZfKhTjL8oIfGLNTi0BGogxTBvN4SQ6sCVU4afvsQzMvguYieaCyJ6DP1tYJVCeJY7cccmkbgxRdzNnakpPG9bK+tobHMEsC1FYej\\\\\\\/LLE6tjrtv3WeQjyzljfCrvkOBeZf5Vn3UtNnM1XWH9JNH2\\\\\\\/Y9xXELX\\\\\\\/OjmAHCeimRskaPDkSIH6xR9ZHUvMNY2Bu5swGP6WWcxuiocbwzYsUdg1HE3Hgj2NTnlmohVA90n3P+qJ4BnIHfml9b37jLnhHzw1txDdCcBxQ492x3n+NCahZSMY4Np+ibCNVHfe66RRnvb6ToSCofLNcgzcUWO2fStEsisBUI0GdKRgUyjgMB2zKwKM+OdJiz4PnL05Rcs9U02xYAKw8GlCzDE2ClxE7dz8BNK6+fYBBYzVFG2EbzPUtNozw6\\\\\\\/FOpW3l9uSR38ydLQZaUCxXNb4iD3XX6KaopR2pqtvRRABb5Qie\\\\\\\/unvNRTLtuGBS\\\\\\\/ZNL8w0UeiJzd2TJJjs7jn+RbHeGVm8v8+hQ\\\\\\\/iJtZunA0hJ4qxgvL1psduksRz0fbz2J89O6MYwExq4J6ARUjsCddunR1iwQgdeZgXJtiSaGSV5QKljh2jynYv2kMRoocXrhDT2+Ug5ZPa3\\\\\\\/tIPCDojO8QpTkicIj+KYi+soobszdBTztdEyEJsBrolEVs1EI0GHeQZE+iMiaF4RKksYryHpLmw+1Phf7rv0\\\\\\\/yfKzigEDCOeQOeJT\\\\\\\/0Iu2FIYB9qTT9SZLVz5\\\\\\\/CiCUEuXg+iumVA3Gu7tlF1xNkg7qjtKKfkFOmH33aVOHEyLvAv9VpZAiDUm2d7\\\\\\\/A5n1htI0MFO\\\\\\\/zxq48S7SeXAjRMKyFyXzobBq\\\\\\\/R5cn7IZcuc4hIUSy29XRR3uYI6bUfykcESaIushlScQJQBkB9sbj66R3\\\\\\\/ylwwuVetW1gbAY1n87vJCkf7lOrQFum5LIX3sZWokjnZFzSLzGy7\\\\\\\/sshu+ujBPbJKelgVruVb+aazrz1TsZpbPbi216pdTHq+60ZsCG40Lxe2vElt\\\\\\\/2dnlXGRwT+J7qPIOJYq5pvbxgOLN8frwagcoR6W\\\\\\\/LcHIiZdtMLEo3APNH7SumhV0UQgdAcDPSGjv47seZkKeG6D5c27jjLxAzQpzTnENLKALNcqHzpJgPY3aTkHKytoLd7nEnTblDNraK+aY9ZYFIgy5ofjtQwV7oRg3cq6p3p6q7xrw2X0+xbSlVPpOVmHWFgUoKQEMafHM7D2orsZ0Uvkj2rx4FxBKKg65iKGP2+QtT2+77arIRfeT4EGybA+hGqZiBZ13g8+QQAxglTnCw9qbQorJkNGAGsDdgfafls+4sfewwBnKrc4adDm3Y1LKVvqkiU\\\\\\\/FATmN2tsUdfZHbGxJaA4s2DuKbrAf6clRgS3dfGwO+nzoAMTSuen3wPUl4Ie50In3BcwlblB5O+4QByr9T3xK9QV9Lfynvw2FfZj8uADGvJzCppkGASldTSh8rrCoxoW9cSTBr4WKY8lsTqGKusKqHgzVcS+NQTHwh4ZF9mfkjRbKnFE\\\\\\\/\\\\\\\/O3bLYMaruY2UvnsvYLLl82HHhWsHv5cJIBZnh43UBUkvyliaMgVKSO2Xn15orWe1OD1J1UufQT0yMgTFBGjtrN3gUEUvz7VJWzrbc7DxVy15H69WQHQ7DgHtUPYLMJr+eoFYO90WZMrvgFyOYUw+CKT+9Yez1ozrkO5hCEQlJKIKqwel9B2T8ShtBsKaKIPUUGJh24fFonCbOohBO+vnbEGZX+br1Z1GyQq+k95FyNzPX9cHpJEIpUVVtgRAxRBHs5XLv6bW+wcJcdgraqFrqfkBp6qd6ilTTWvIky6WOBOKkwVQNInOyRtagc5BMYtoRIPZBSDOYB40rkPgED6UUD5RImd7UgbI4GzISYKAJWvHtuh2L6se0niiZebxCPTyWRniu7VBoEIUgQfzyxuWtb3cbawzfd7ETmDD+lg5OCR1rHNXqIhX8QkQdRzErwTxvVN7BV3gFItIGSG8kwwfbV2QNKRvz4\\\\\\\/ERDZzXvICN+4RNOBgicfxcwKAEdnPsVVcLxBs9dOw2kayT97ND\\\\\\\/MxDJSjhEUj1Nux58vjmFXSVtmS8kCmdDZgckbOHTKv5fEWUhW7r51Im2Vyfy+TS+MwxJJJTigRneL9rTpNskwsM2yE5Szafk7\\\\\\\/Ujs6IYCW\\\\\\\/CgGgp6D86n9a\\\\\\\/MMI6nXcqLktLBcwkpkVSL0ldlEtg7KSdtuVYOtMHiq+rfguZ8YhW74x5bGXPTw3i2aSlR7oXZYe5FuiRqRH4vXn8a8GlHCVpaMQ\\\\\\\/C8u\\\\\\\/B9GVaHVEmK+tC1G\\\\\\\/LMvLjbEJqAQBO70bR\\\\\\\/aFgueGBSqLOpLezATmxOYE8DBYwezCsYU6ekBVQxr1zITZqOJLL7ChyQ+mZiIN8coJQr5WXtnX1hc\\\\\\\/114fEaocFN3ynOHWjpbaiiOr+UyCCWWzdwWuEkjIHlk9dzPE\\\\\\\/+QU19Ap5\\\\\\\/X5kAMIbiQZF00PwVAZEcEmXLeydk+wYwAyFL0lfRlQpAbdgVmlvQIRmdlyW+7uGSHKLZSlAb0n8L4PFFo3CzHCQ3OP1mGdiBmCHE9tL+hZESCk73XBXGoRqoqlKvayzdNXFttrfo5GvKOPg5Rw9+rX2snBtv0DLnEdsGajN8nANQdn5Vn8psuXXbXT82R8ih8UPy9B\\\\\\\/AATR6bxcRisxLAuw3gXJCoaQAl6OiFq08gDeBDsszvYTtSYPJkDmfz3ALUcTj+ccAn4xZJxRHxMi4lidkvUqjSYZoJ5cwAsgFuamOzrHORKYlTm6wgdXFiGZlJLwFZs44eo0G\\\\\\\/bAUXmmMqFgg8KxQiOobBjqHCuibw\\\\\\\/D9uWiXbhI1OvYhnCj2yF0gnXAgSza92o1QSyJCOKK8MepYPLVR+PrkyUK7qOIrUnHzA8SDPtEDI1AGKBMd6oxmnEBB2Bcz8swjznhm8+9Oe8an7lwY6LnQHR73kexKgbte41vkjefN6LkFS851nQ99D5sS56H\\\\\\\/WV1vxnEI8JlinBwMJ3Fj51pg8rvI+O8Pfsj6VS7N8h4kqsAjAw+6s7tp7Izqg5TpC5WzntJuqQTitSxPsKsqLBxu7y\\\\\\\/+w36zraZ2K+ViUXlXzQRPFmI2jrc5tpHQpD4xdF7Vy8MkYXvTMr0ygDPCfyi5i1niVMxCoqhIVxrJO2L\\\\\\\/RQkAl2\\\\\\\/NCtffk78jJQdShyurc3JvaULLNWhAvUujIKR4U9kRGul7IGPf6TtP6v8BwOY5gA+67rtwknXGC6QkrS5DZWBGDUevDGH2k75uq5+GtAGdFwOoQ+krFaKjA5k5lS0SUDauVbkX4UVvNotgLiGFDoUbXX1m\\\\\\\/82HMtuvZFaCGoHQSFzcJqKyc5uKz1ljBBkw8LmbNuh96xclS76s6t4GunuXn9y4vG\\\\\\\/IcFuVOlirbfKAR+Z9LB8JBFFJj03AxjlVcnDhFUt48XPac9XOEVC4+KIX0LTEZme2MPH9lI5WDTexErEzyOy2\\\\\\\/BoNDLvmQsDT1braXf8vrEYVxHAGaFW41WAZf1z4KJPVfBV5TayaxJdfj\\\\\\\/nNIZVDJavWcby\\\\\\\/PcISvvHzithrrpujTnSaep9e6+lGJngc6nYi6Y3q28V4xy7lg9pvBwJqRY8i09rB3TYN6EOhZCCCpamyQl01KAMtjMp30Jc4pYpnU+021Fs0p+mnDnYCwiXwodqwHgzE8O05JSDnYO6kAZd7GuYbaf2xexEX4CL7mJljsojpTS4rl3we4zj2OquxthtTFesIU1FoCHmKSz6OkfsNifwuheBmHuqQHtbYLjBbPwmCBv0Wq5rZf95tX0MIXIjPo7IqjF4UtEOle3\\\\\\\/N+xHSrwCj4O1aqKleGtv2q\\\\\\\/YMU0dxrIxrC0vm4Y1tVhVDdA+uRio6PM6U8hS187Q2Cdncf8Ia6T1PvH7JQ2obZMMOkUXQYnK3qKvx3gkBEHreut27o8i\\\\\\\/SzM9R2jUbzf0+EHm9spR4nsNd1nV+\\\\\\\/+7GD08rSa5XboijpQXeByqUfIDicqLAo1z6zWpHhPF1y37kCLy7m9dM4E5YnKS59+IaJS9aXN6ltvPVc\\\\\\\/GB5lZURlpfWJnJ249\\\\\\\/viu8uE54ZfaIZjnFGG5iEYVOx0yi9+FYE7YMpjBZpMxi2M\\\\\\\/P8OcRGf41WETCaQRcxuZz0HFUphyE1719jNDrdTe\\\\\\\/rAjmkZn+B7i9UePr30GGZF\\\\\\\/KalDfxjyb4PKcarDX5okYlqfGkDyp6GXQt3LCDyzrUeaFjD9d4tfehSvMfJsMPzpDSGKPMA+Gw\\\\\\\/D2ELHNY3A\\\\\\\/v4wM9EPADrBCR\\\\\\\/QTEfzkf0xARgn86KrQLud9emFoIwclmqBnArUmlTueKGVORdl2O\\\\\\\/Uroenj5nX0Fk+NNXjU61ZHUrVyCsNWkFGmnJWkyhvft\\\\\\\/O7oxkVRBAcen6V4XPnAlVD0GeyXAZXLz6jUN6EHmclGjSp0\\\\\\\/X+BrTJ4t0jlpFhSpJMvpi9CuCSZFRDXzuEyL6VS4bCCSfe26onA8U2hPx7FbizusfP0kjqOQywMevigvrYNKgc7WBR14NS6qlvuGDeU6b\\\\\\\/d5U7v3QNfsm2gNRB4lQvZx+NWqd25eT4qto0GW0Zs9ywZDLhPLvLWD8NBffiU3x2Tz\\\\\\\/luxueCWzg0HnNvaZDr\\\\\\\/1IoHWHFMmIRl397GOPyMAXt4KUEIzQzKdDGAnBE7DsiS3NdbKdbNqrkpjKqOdaofAU+maYxCc56OX7KQua3Vu1k\\\\\\\/4bCC0PIu5SabfL5xTbUITNzBV+kCq4OkqT9VOyaGIpoHvUeCliIs5h\\\\\\\/1RtqPmhKEgSvS3ionFO0ff6n02hqt8P1xrrcgCFtJmlcPZWvDbceP4fsjILCBVHxTsJH79XVX71eJgSSF0r4m6hC5h+nN1NQ5HDsdnatq5S7kGxrrX33jgHgBePaDCgA47h9adqJW\\\\\\\/VB9Dfw1m+\\\\\\\/CiNjwAaYmGqxFFj8569GNr9OByeIUN5uap0dI3PW\\\\\\\/5coQOiXI9p6mbk98q9rrn26RI9StueBkH2nwi58HTthVmh4xxhGA5aSZeO9UVt+9ppU6ZDD331nOD1TNh4g2itnb8dLbXelok0w4tsiKw2+\\\\\\\/ybIc50KxZrfnm5hS427R9cwTwyIYucVV7dS5rOoQBLNoWh+GI6dzgH+BpZYu0GaNAInLo3ZDA0wl06bPflIosKBqmP3rG7T2EFYUAWoN5mTuj0eGtXwq6e37MM\\\\\\\/iSx5jcyPk\\\\\\\/fg4XZMICcQQA59I9vP8Vh3\\\\\\\/oa\\\\\\\/wVsgZOxlYF9piHt4q4ekfwWpq4sEFvWvIZcPJfuWmun5c+wFHkHtt2Tafs2tRsfs+Gn42il3qgqeUbte7bwYJBrPo5NLW3KMOPHAy+CBD8NDaQTBYS7LvzWWbC0ZhP0pF8sNKW\\\\\\\/uLiNLWKmcYSkgHIFs3WpAkbgGt3G6Cl0ceMZ1\\\\\\\/bDXOKhhowDGkdsw5xHsQk7gq0HWwttMzak\\\\\\\/HZnuro4z\\\\\\\/bjzF4R8o\\\\\\\/riPvKNUf+y2V70LZfjMCXGsSC5NUtszrvoT3\\\\\\\/uo0sZWhHA\\\\\\\/T7K2iSJ\\\\\\\/9\\\\\\\/W4DutJYHeeeorC3yHqSuEeqdCaN\\\\\\\/OkMuCFSqKKicJ1CbvdaYPvCMBKZ37fZyaRyLzRe4BegKfsl1b3cVgOFkvQKYpjPsa+3OVcHo6\\\\\\\/8g\\\\\\\/nB74HaeC2v3eHdrTB32dzVBsCr6M4uct7M+OD1xgC7EOFIThTdxp7IeSL5HiNimn5cx0MCwP1AwTUj4DK23IR\\\\\\\/AEBuXoPFfPyq7W+EJrH6+RQBXtyndIUVrEwNHkg1q3YtWOHw8y5UG6DOGnpdFEwXdx1b4vaiOkWN7o3vjP4J6dO\\\\\\\/i+XjQnemDt1ekwzHdxRSDEGW5cd2mT4YI70AhEei9oqyL2ljbYqFmmykjKSP7F61qbxEVpUL8\\\\\\\/opw7Co0bJc23ativlD+1wVyMvpK9mvqxca\\\\\\\/wl\\\\\\\/XLNOZOZUT4GMP6CtCjTjHetLxod5+w7MPLN8gQEzMUqIdxCW+t7wlxQtEMVR+qETYgYi0s0af1Fj7cPNCfLnsWn1UEw3Gcu0pgupXeJGQil29YD5Qu9gvOLvciJPvu5aQuKp5tX9YG5d5XwS74TOPagUgQ7SGKRV8KxcqUyx027Jpjfynne47DnHvl4w0Gp07CD43VNuxahZWHGyh+4C4pXDCSsszmGgybmT3CIXITluZBVm\\\\\\\/5Ap5POyFNjokfSIh4VckC\\\\\\\/xfrg8V8mHehjJatpCQBX2TihIWvxbCUK0eZfbk5VRPrt1mM3S5vw9aZJc5dQ\\\\\\\/lk70fb5fvpXHuGZlIoBvJyqAemkICrGDuaJake9XrQlw1816\\\\\\\/8dItdwXqhi00L9Lkk5apTjGGp5AtbtBnM08bkn3pUrxTpOgFkaA\\\\\\\/Ud29wowkMGhO8ZTYmjTkzSKTbkURLwMIUp4864mv+hxIjf5DIktLRai9Eu8Z+kiDMvZuI6yPPUpsFih0AvaJSvdonBdSHXEA0AiIj2vbuNH5rRphj5zLhoEpWvzDFaBOE+PcD8jsAz5d3nuKRLqRLpmFwB2mKCB96RaWxeyz5FyH3mWeirpR2j\\\\\\\/g6Bt6iPhQKkLMgAycnB4qex5GYqegXkYFrsWESPT1CwsydGnfg3+Jbckj0U5DVLJpPYQ2hCLsSaIJjd2Yb0SUtzN8JXcTpUVWz5Kj0xfd628mHruNhHMNSl9bha95jhDwIaCyRI5H15b\\\\\\\/oE7+3jE1208qJiRoVkcN1C2inl9t9r3KCUJmpLIqqsCj7h2OAnPEa+cIsu\\\\\\\/rTwpg1NJc+JekhAGoW6UtL4mi\\\\\\\/kRleC6q72rqyNXwO0hR25HbvRnDy6ORowr6Sb1PW2eUZBje8+HrcBsZMiJRpsvk8NHWeY3PFDLzZAiwMlOL8dpH173VOpdBELWHoCOS3xWciNDKW7RjzLtRw5dWwlUgJAYkCD4Bz+3RgSX7BR+8NQK6IE6h5\\\\\\\/+AQUENB\\\\\\\/i652wHH6TMLf0ddviMPfsACCLhrrDODsatyJKP587f6gAd\\\\\\\/lhF4ciDBMYZFcu7IzRL5trBNeCM\\\\\\\/DCY74qiZN8lZEX45APLwrlxdMC61ej8LiFIS4NZpkyv2cGzmKMQbN7HCotKEF8kqZkneAZBGj1RyWuDJjsK71fRCNuvB7WnQ0ovHYbhe\\\\\\\/Syk5tgb0iInnUx6txROIrudmrvVyVlyqtcQzQDaf1W+aa0spev9b+mCEn+HyY2OoveFgzp2eOHdk9aSXKIWFXyGOEeY77vgD8TrcSfSyY\\\\\\\/BjiMqja8YJQrFNmHCGtjOthwvNX2+FDs45x15F5q9XztdcM1\\\\\\\/w94ADwhYDgJLs8Cq7rW8RCDLWVPvis2QHc6wy0xVaszzgSVazVbYE\\\\\\\/zD4UiM5yS0N6ADYVtTdHPhaZPL4kZ7ZSv5NCW7haJtCL+o1ZjKStg4Q+W9AIbjJrV2jG0a8hm8KyO+YsnFi0Wj4lDmWt5doU1TypaxxDcNyVTvhcRDB5Js1RxmIv9X6lErqILPaZ1uK0LYuOl2Gn9xM9ALTfqPtDw8n\\\\\\\/nEjWsnf+hG8N6cTwTxWPZ4oSKdjAadXd9E5mhUgs97KHSMvVZhEtKVrU8rUN59rWm\\\\\\\/9hhJPU2OKWZYYfcaTFqibDYTKqsFqSYIcUjkZ\\\\\\\/Bsr2gtuQ3ZoJ6XbbL27eXEWBLp13KgUryNfcsHJTQssO\\\\\\\/9bIQYJYFlpg8oxFUPZqi9epWHdRPlI\\\\\\\/34VWMDNZb2TocS3eTR62DGaQ+DyoMRVTXzO\\\\\\\/NpBu00z0pRLsQ1wt2CWAAQUnz99OQbjHT5UqCkdoWOCSCZ5Nkd3GwZ2pqV69F4+0uwDKDp\\\\\\\/smuCHMDQUPBxVORBSoNTtoJtnfMkSjqD6Mn4LJBQDsCcXYiJo+WQSP3LQEM\\\\\\\/6abG3s77i4QVqXGqytxku4azd35XIYaiHb1LLaZJ\\\\\\\/9HJPcFbik+i4TemMkwcDBqmZLymEpQAAFHvSTh3SqL56l5nWPvuBi0VPoqUnaYoCXT\\\\\\\/mGv5r2xXfJf9J526N\\\\\\\/Cws\\\\\\\/yDzksDKpQAblGw76p0WqBxuYyYEkwX+e0MNvuaYtZ0Iu2ptRUQwAQdL6h3Idl60inc\\\\\\\/8HPVyuTYUjRj9ozSVY6yuCxOIAWSsrRMRHeiRmPnobea4X+0l+0rM3Rlv6RXOvus+hM49x0t4Xe0YaCKotucNFIH2J8itznzfPORMxq8IhyLx79qA5mwkA2gxnKfOE0byppzH5js+9M1XIiSf9E\\\\\\\/hSae4bmumDPyczpkzQmWiG0g0b7Z6FmULLkiVb6v8eBmyCeKGnwn8O77WTlWOg9V3QfLZD24mSLfvUTOROdMkbU66j1vz3ES\\\\\\\/+E8mJXspCkLPCPpp84bzFc\\\\\\\/VHoNWQRMDuZ2OHv5BjYZZRvr67hWl+g\\\\\\\/JYUrrwOrBiIqC4tDg9s+un5W2Tc\\\\\\\/5bh0Vk7w23ydOfVgQpflMiRic8zN+75Yt8I49SL0F2OXTDwJnK4JLyKH9E72ktMJMgCXGhM8XZYn1bOM2NURN3HSzxZISS+0s9fiZ9hmfDVZo+g\\\\\\\/1Nj0hLBcr14Mcr3DHXvZpFfxqNJ1fAu1D0OPxBCNR86FTHZVbVFJAGTj4FXJ6da1BTT07M4dRMX\\\\\\\/AjcvXTcIhhAcn0XVrNMY4fmAtzP5S4iN1kxwjHh4MKTwKc\\\\\\\/CasUNkHgUWpek\\\\\\\/9tcrOlH7O93R+5PZxNU0ie7pc1cRAbJfapqFgL5N\\\\\\\/RrM4fpVNMZvAthUwnI6vpbIA6nDv\\\\\\\/FX\\\\\\\/1bttqXjePCSSpBMre7VRmSFLHhj6hnYP3zeEmC+OZYgePVWaPA3\\\\\\\/3\\\\\\\/vrKiUVa+p4eUKLjS5F2Tw9\\\\\\\/SMs40Vzg2OUdq4AGetFqBEWKmpfB2fyFD\\\\\\\/iiBl4a53omQHViz7lcA9hAXFnIuwIfGKH8CmXYxzn89+b0xNGPUdfvzRbC6Ls6FXZSwMCxd\\\\\\\/ZLj0l290fUv99kJnmKUQjZ\\\\\\\/FGlSViXOPSSFR2tgvD2bvAEDVI1H4JgZEmYv11xuzupUSGdvi3twIYCKlimJEgebNEAsUUS4LSkPkDaS1c6j17JHL8NgiOj08cT1Uy30hsz58Zq4a4VyhoGMNEWi8LtlW41x8J87vvM8Rp6iOqZ4Nbw7KByqQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022HvXXzRXVZ6MqdHUPTHOWwA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022290a62ded2e840606c4877b68d2ce9f0ee2dbc90bef4cd889cc792d42a0842f9\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mJ2LQNOeH72rydXOsvc2rvBSx0zFEXDuNCsCRw\\\\\\\/hteNlp9xFg0rFCQpdP4feJpa86vI2YTXplMqdzw5SQd2FMULV+9PTxVERCsIrd9ux8oCtbDW9IeeZ5TNaKoPu1LrxUNVHwCL2yChjDdK02E5sZl6Oced4rnDvcMyT2ItPy0SBW35q+yZRo39vaZZFV\\\\\\\/p0Ihy5b\\\\\\\/36Pbj3J5A3O8DfIzQCh04oK5jfh5ih9mGs1vFx2su8ZnVBu+gfme0wY8zAkiyn+jm5nkkxIYNHIvlUA+tzbrn7l5+RDpdtcZBc7s88mc1mS59vMeq2kQYEuAi65eV7osobT1VaY4N4jxUTAbA7PzCP5+ac3Zxb8txYQX8Bsv0Ak24JENbW80v+kV7FRmpvc4pMiMSAUMDP+hdX6mcOKdbg2oiOdym3q9sri6uYrdMvbZIUgmVQCbAjQY8uLKCIV\\\\\\\/I9RNhiGfd56Ey7KOXl6trUMvjj2QhuDHJcrs6E0\\\\\\\/ySwGkz76ODa432vWjNT8eDLSPpr3az2nNLSOb2zrg4o3QyWkCPXW9yqEDAKLZ9eQAn0taBW0UAVOe98UshBciVw\\\\\\\/gGg4+4X6BmfIfVFSz5bAtMPeKj09zf3KucTjpowGuff97g5zJam835ZzQLCWN3Ci4M2ZvBEOCDWbSx8efYTHYrEmAD9Sqkfx80hk4LkPKXJWWlyuQugKmThXNk0Ih1n4VVda6YtQWaqOJqUSrTdxLd0Bi4KxQKfJ1OQ+aOcxhOKbeb4izqHFepc\\\\\\\/0AhNVrWhmi3q2EIN\\\\\\\/7o58nspIqcc2pO6zXwrC3AG4I+5yjDe5ZpNFaiQ34GXfJ2mGZTUrkF2mjXsY2nW141yX2CBVjYRsvP7\\\\\\\/Sz5aq2XLzjdY1A2awIk7PXGts1UiMvS32+jwbCfgGBWm31ZqWbXfPN06UxDARIqQkjZ9jb8bszBmFS4rO6cBfnEAzQG7AGI40sWkg8fMr3PDgILslhX1DlORFw3Uwwa72jM5s56kSdKpAWyibf88CbiJDhMBw817tc+em3RuCQ7FmLJkcYQltQyLfcm9rosqIFbudQSMjlmft7Am8\\\\\\\/kOzeaGo\\\\\\\/lzK3mI4hCbqJ8mnky0OXKoY4kL4zDgghab6Zh3EZNfg7yPoo4UUfvtU9XP6KujYYc7hw3AH3mw9JDiGx24WPDna+huMPxQgWOIfAHfEacM8zwbEOsFvUUcHEdChOQFIwdZOso+FlcebbuGTItzC6XDLcd9hnGqRhx47gJUIqk90bB2sMFHG4btOg1bKPjEsFygKvpnjENCqNUoo1XfvKFIf3BlTVeWlpunDNTyh+2eoVkr96DxNxg+VozNXjLE6NLi9utZ67\\\\\\\/XwLem8awjtnSMiFWBNlcAE6d5vqtqrXX3nO7iDbTTl7wnbQrlyha4ZnE2quCSk9u\\\\\\\/jce0JQnVwRr6DMmuMmPXxcWGkPuhSdtM3NfVHbWmXoyf3af\\\\\\\/jNfDEQRBy5nUcV4Rpqx6L1nhNPdpv2sgWVttZGVY20vJoI2K2NCJwa2WIVXpOOAwur8UZSSAiMxEhW7rxi5yFIG40EoigRXnCXyMREK2Lir3vhpvOTQgJY\\\\\\\/ZOqxKcCZd4HoDYDbM8tP2XdBVpN2QwW2wI\\\\\\\/ThGz567zBJXbB2dhin8K9ZqTQhDP0oVv8ePwfsmzHvrdhE+WA13+EW\\\\\\\/S\\\\\\\/2HkX3ym2JTQCiI+mkVlUn0M5YvWKukEuvY9yU5ogpESa6AkcNqNLHrkcGU59D\\\\\\\/CjXxBz\\\\\\\/Ql4ZBdfokL+7oYr5oRru0VENSh+gETFnAwXKV0QhPdZQxbmXqKiXJluZWoQD2Z9S6LesTgMSxwbDGWjs4HinLGbXzBvshHeeJX8tl1NUOc9+Nh99frl84rRH\\\\\\\/LELVehn3ZmpvAszXoNPYbzZFWBZm2APh\\\\\\\/Vf1Wyd+LGyC3Zr66DnG43LetWLFDuh6AbFcXP0v77uWe346swryS0YBW+qP0xlAQKj2K+KE\\\\\\\/N+VQkyMzCuEu+Z9rArbxmTKYR+HHrEjWMbM7LYzpQ==\\\u0022,\\\u0022iv\\\u0022:\\\u00226rqnIgrLVOreIzJSoxEaFg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a007628393dbe8f41ce0f56a8d402f9713b6e8e116487e13468adb6f4d8ef1ff\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-483531 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/3/843" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1xDXLx1U9KP7y7dIk56MnZdHJJB7Pekf8yshvCk0YTQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/3/843" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-oxCbX6NlfWAIbPy2NXoODcDXdppIE3fnZgTloWFeaIg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/3/835" title="Genetics and Adaptation of Soybean Cyst Nematode to Broad Spectrum Soybean Resistance" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/3/851" title="SHH Protein Variance in the Limb Bud Is Constrained by Feedback Regulation and Correlates with Altered Digit Patterning" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483531" id="node483531" data-pisa="ggg;7/3/843" data-pisa-master="ggg;g3.116.036350" data-apath="/ggg/7/3/843.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Linkage Mapping and Comparative Genomics of Red Drum (<em>Sciaenops ocellatus</em>) Using Next-Generation Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0227-7225" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christopher M. Hollenbeck</span>, <span class="highwire-citation-author" data-delta="1">David S. Portnoy</span>, <span class="highwire-citation-author" data-delta="2">Dana Wetzel</span>, <span class="highwire-citation-author" data-delta="3">Tracy A. Sherwood</span>, <span class="highwire-citation-author" data-delta="4">Paul B. Samollow</span> and <span class="highwire-citation-author" data-delta="5">John R. Gold</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">843-850;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.036350 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483531" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Christopher M. Hollenbeck </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Marine Genomics Laboratory, Department of Life Sciences, Texas A&M University, Corpus Christi, Texas 78412</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BM.%2BHollenbeck%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hollenbeck%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristopher%2BM.%2BHollenbeck%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0227-7225" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christopher M. Hollenbeck</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ch263@st-andrews.ac.uk" class="" data-icon-position="" data-hide-link-title="0">ch263@st-andrews.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">David S. Portnoy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Marine Genomics Laboratory, Department of Life Sciences, Texas A&M University, Corpus Christi, Texas 78412</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BS.%2BPortnoy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Portnoy%20DS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BS.%2BPortnoy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Dana Wetzel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Mote Marine Laboratory and Aquarium, Sarasota, Florida 34236</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dana%2BWetzel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wetzel%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADana%2BWetzel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Tracy A. Sherwood </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Mote Marine Laboratory and Aquarium, Sarasota, Florida 34236</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tracy%2BA.%2BSherwood%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sherwood%20TA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATracy%2BA.%2BSherwood%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Paul B. Samollow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Veterinary Integrative Biosciences, Texas A&M University, College Station, Texas 77845</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BB.%2BSamollow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Samollow%20PB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaul%2BB.%2BSamollow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">John R. Gold </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Marine Genomics Laboratory, Department of Life Sciences, Texas A&M University, Corpus Christi, Texas 78412</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BR.%2BGold%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gold%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BR.%2BGold%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/3/843" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:483531" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:483531/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/3/843.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:483531" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:483531/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/3/843.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:483531" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:483531/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/3/843.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:483531" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:483531/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/3/843.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:483531" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:483531/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Developments in next-generation sequencing allow genotyping of thousands of genetic markers across hundreds of individuals in a cost-effective manner. Because of this, it is now possible to rapidly produce dense genetic linkage maps for nonmodel species. Here, we report a dense genetic linkage map for red drum, a marine fish species of considerable economic importance in the southeastern United States and elsewhere. We used a prior microsatellite-based linkage map as a framework and incorporated 1794 haplotyped contigs derived from high-throughput, reduced representation DNA sequencing to produce a linkage map containing 1794 haplotyped restriction-site associated DNA (RAD) contigs, 437 anonymous microsatellites, and 44 expressed sequence-tag-linked microsatellites (EST-SSRs). A total of 274 candidate genes, identified from transcripts from a preliminary hydrocarbon exposure study, were localized to specific chromosomes, using a shared synteny approach. The linkage map will be a useful resource for red drum commercial and restoration aquaculture, and for better understanding and managing populations of red drum in the wild.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/rad-seq" class="hw-term hw-article-keyword hw-article-keyword-rad-seq" rel="nofollow">RAD-seq</a></li><li class="kwd"><a href="/keyword/snps" class="hw-term hw-article-keyword hw-article-keyword-snps" rel="nofollow">SNPs</a></li><li class="kwd"><a href="/keyword/genetic-map" class="hw-term hw-article-keyword hw-article-keyword-genetic-map" rel="nofollow">genetic map</a></li><li class="kwd"><a href="/keyword/haplotypes" class="hw-term hw-article-keyword hw-article-keyword-haplotypes" rel="nofollow">haplotypes</a></li><li class="kwd"><a href="/keyword/synteny" class="hw-term hw-article-keyword hw-article-keyword-synteny" rel="nofollow">synteny</a></li></ul><p id="p-3">Next generation sequencing (NGS) has provided a set of powerful tools for characterizing the genome of nearly any species. However, genome assembly remains a challenge for organisms with large, repetitive genomes, such as those of many plants and vertebrates (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bradnam <em>et al.</em> 2013</a>). For many applications, a suitable alternative to genome sequencing and assembly is construction of a genetic linkage map. While linkage maps traditionally have been possible only for species with pedigree information, or those that could be crossed experimentally, a combination of reduced-representation NGS (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Baird <em>et al.</em> 2008</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Elshire <em>et al.</em> 2011</a>) and advances in single-cell genomics (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Xu <em>et al.</em> 2015</a>) have made it possible to obtain a high-density linkage map for almost any species. High-density linkage maps generated recently have been used to understand the genetic basis of adaptive phenotypic traits (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Baxter <em>et al.</em> 2011</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Henning <em>et al.</em> 2014</a>), study the genetic basis of sex determination (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Anderson <em>et al.</em> 2012</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Palaiokostas <em>et al.</em> 2013a</a>,<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">b</a>), map quantitative trait loci in economically important species (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Chutimanitsakun <em>et al.</em> 2011</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Houston <em>et al.</em> 2012</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Talukder <em>et al.</em> 2014</a>), and for population and comparative genomics (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Amores <em>et al.</em> 2011</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hohenlohe <em>et al.</em> 2012</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bradbury <em>et al.</em> 2013</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Manousaki <em>et al.</em> 2015</a>).</p><p id="p-4">The red drum (<em>Sciaenops ocellatus</em>) is an estuarine-dependent, marine fish distributed throughout the northern Gulf of Mexico (GoM) and along the east coast of the United States (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Pattillo <em>et al.</em> 1997</a>). The species supports a large recreational fishery in U.S. waters and is reared in captivity for both restoration aquaculture in the U.S. and commercial aquaculture in China, the U.S., and elsewhere (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gold <em>et al.</em> 2008</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">FAO 2015</a>). The species also has been used recently as a model to investigate anthropogenic impacts on marine systems, including studies assessing impacts of agricultural chemicals (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alvarez and Fuiman 2005</a>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2006</a>) and ocean acidification (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Diaz-Gil <em>et al.</em> 2015</a>). Following the Deepwater Horizon oil spill in the GoM in 2010, red drum also has been selected as a model to study physiological and genetic effects of acute oil exposure in a marine fish.</p><p id="p-5">Here, we present a dense genetic linkage map for red drum, combining 481 previously mapped anonymous and gene-linked microsatellite loci (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hollenbeck <em>et al.</em> 2015</a>) with 1794 haplotyped contigs derived from double-digest restriction-site associated DNA (ddRAD) sequencing. Using the high degree of shared synteny observed between red drum and six other fish species whose genomes are published, we demonstrate how synteny data can be used to infer likely positions of candidate genes, in this case identified from transcripts obtained during a preliminary hydrocarbon exposure study, which could not be mapped using traditional linkage approaches.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-6">Prior to analysis of genetic data, a reduced-representation, reference genome for red drum was constructed. Twenty individuals from across the range of the species, including parents from two mapping crosses, were used to produce a ddRAD library following <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Peterson <em>et al.</em> (2012)</a> as modified by <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Portnoy <em>et al.</em> (2015)</a>. The library was sequenced on an Illumina MiSeq DNA sequencer producing 300 bp, paired-end reads. Raw sequencing reads were demultiplexed using the program process_radtags from the <em>Stacks</em> package (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Catchen <em>et al.</em> 2011</a>), and a <em>de novo</em> reference genome was assembled using the <em>dDocent</em> pipeline (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Puritz <em>et al.</em> 2014</a>). Because sampled RAD contigs had a mean size of 300 bp, the entire sequence of each RAD contig was recovered during reference assembly. A preliminary annotation of the reference genome with the BLAST algorithm revealed the presence of multi-copy nuclear ribosomal RNA (rRNA) genes. To avoid downstream issues with read mapping caused by multi-copy loci, a custom script was used to remove rRNA contigs, and all contigs with a total length of &lt;150 bp. All further analysis of RAD sequences utilized this reference genome for read mapping and SNP calling.</p><p id="p-7">Tissue samples from two outbred, full-sibling mapping crosses (Family A: <em>n</em> = 117; Family B: <em>n</em> = 116) used to generate the microsatellite-based linkage map (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Hollenbeck <em>et al.</em> 2015</a>), were extracted using Mag-Bind Tissue DNA kits (Omega Bio-Tek). RAD libraries were constructed following procedures outlined in <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Portnoy <em>et al.</em> (2015)</a>, and sequenced on two lanes of an Illumina HiSeq 2000 DNA sequencer. Raw sequencing reads were demultiplexed, using the program process_radtags, to produce a file containing raw reads for each individual. Read mapping and SNP calling were conducted for each family separately, using the <em>dDocent</em> pipeline and the reduced-representation reference genome. Raw SNP genotypes were filtered stringently using the VCFtools package (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Danecek <em>et al.</em> 2011</a>). First, individual genotypes called from &lt;10 reads were removed, followed by all loci with a mean Phred quality score of &lt;20. An iterative filtering process was then used to maximize the number of individuals and loci in the final dataset. Loci with &gt;50% missing data were excluded, followed by individuals with a mean depth of less than five reads and &gt;95% missing data. Next, loci and individuals with &gt;25% missing data were excluded, and loci with a minor allele frequency &lt;0.05 were removed. The bash script <em>dDocent_filters</em> (<a href="https://github.com/jpuritz/dDocent/tree/master/scripts">https://github.com/jpuritz/dDocent/tree/master/scripts</a>) was used to remove loci based on numerous criteria, including mean read depth, ratio of quality to depth, strand representation, allelic balance in heterozygous individuals, and proper read pairing. Complex polymorphisms were then decomposed to individual SNP or indel loci, using the <em>vcfallelicprimitives</em> command in the <em>vcflib</em> package (<a href="https://github.com/vcflib/vcflib">https://github.com/vcflib/vcflib</a>). Loci were then collapsed into haplotypes within each RAD contig, using the program <em>rad_haplotyper</em> (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Willis <em>et al.</em> 2016</a>; <a href="http://www.github.com/chollenbeck/rad_haplotyper">http://www.github.com/chollenbeck/rad_haplotyper</a>). Briefly, the program uses read alignments to record combinations of SNPs (haplotypes) present across paired-end reads. It also applies filters to flag loci that have an excess of haplotypes (potentially indicative of paralogous loci) or a deficit of haplotypes (potentially indicative of genotyping error), given the SNP genotypes. The program removed loci that were haplotyped successfully in &lt;75% of individuals, and kept indel loci when the indel was the only polymorphism on the contig; other indels were excluded from the analysis to avoid complications in haplotyping. The resulting file for each family contained one diploid genotype per individual for all remaining RAD contigs. After filtering, the dataset for Family A consisted of 72 progeny with genotypes at 786 RAD contigs, and the dataset for Family B consisted of 81 progeny with genotypes at 1340 RAD contigs.</p><p id="p-8">RAD-based genotype data were combined with microsatellite genotype data obtained previously (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Hollenbeck <em>et al.</em> 2015</a>), and the resulting data file imported into J<span class="sc">oin</span>M<span class="sc">ap</span> (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">van Ooijen 2012</a><span class="sc">).</span> RAD loci were then added to previously defined linkage groups <span class="sc">(</span><a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Hollenbeck <em>et al.</em> 2015</a>). To reduce the chance of incorrectly adding loci to existing linkage groups, an initially conservative LOD score of 9.0 was applied, followed by two more rounds of grouping at an LOD of 6.0 and 3.0. After assigning loci to linkage groups, groups of loci that exhibited an observed recombination rate of zero were identified, and only a single representative locus was retained for initial ordering. Tests for segregation distortion were carried out using chi-square goodness-of-fit tests, implemented in J<span class="sc">oin</span>M<span class="sc">ap</span>. Family-specific maps were generated by applying the multipoint maximum likelihood algorithm for outbred crosses (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">van Ooijen 2011</a>), as implemented in J<span class="sc">oin</span>M<span class="sc">ap</span>. Marker order for shared loci was compared between families, and incongruities corrected by identifying and removing problematic loci, which most often displayed either segregation distortion or contained probable genotyping errors. Loci excluded initially because of lack of recombination with another locus were then added to family-specific maps by placing them at the same map position as the representative mapped locus. Family-specific maps were combined into a consensus map with the program M<span class="sc">erge</span>M<span class="sc">ap</span> (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Wu <em>et al.</em> 2011</a>), using equal weights for both families. To investigate whether mapped loci were located in protein coding genes, loci in the consensus map were screened against the NCBI nonredundant nucleotide (nt) database, filtered to include only vertebrate sequences, using the NCBI BLAST+ suite (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Camacho <em>et al.</em> 2009</a>). Hits were considered a successful match if the <em>e</em>-value for the hit was &lt;1 × 10<sup>−10</sup>.</p><p id="p-9">A custom pipeline (<a href="https://github.com/chollenbeck/synteny_mapper">https://github.com/chollenbeck/synteny_mapper</a>), written in the Perl programming language, was used to compare the linkage map for red drum to the genomes of six fish species for which chromosome-level scaffolding was available. Genome assemblies were downloaded for stickleback (<em>Gasterosteus aculatus</em>; gasAcu1), Nile tilapia (<em>Oreochromis niloticus</em>; onil1.1), green spotted puffer (<em>Tetraodon nigroviridis</em>; tnig_v8), fugu (<em>Takifugu rubripes</em>; FUGU5), European seabass (<em>Dicentrarchus labrax</em>; dicLab v1.0c), and barramundi (<em>Lates calcarifer</em>; v3). The pipeline first screened all mapped red drum loci for which sequences were available (2252 of 2275) for matches (<em>e</em>-value &lt;1 × 10<sup>−10</sup>) to the genome of each comparison species, using the discontiguous megablast algorithm in the BLAST+ suite. Red drum loci with significant matches to multiple chromosomes within a comparison species were discarded to prevent ambiguities related to multi-copy loci. For each locus that matched a single chromosome, the start position of the sequence on the chromosome was recorded. Next, regions of shared synteny, defined as regions consisting of at least two loci that shared a common marker order (collinearity) between the red drum linkage map and the genome of a comparison species, and which are uninterrupted by other mapped loci, were identified. The algorithm used to identify regions of shared synteny takes into account the possibility of small errors in genome assembly and map order, and allows small differences in marker order. In this case, departures from collinearity between loci separated by &lt;2.5% of the total linkage group or chromosome length were tolerated.</p><p id="p-10">By using regions of shared synteny, it is possible to infer positions of unmapped loci, provided that a sequence for the locus is available, and it matches unambiguously to the genome of a comparison species (<a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Hollenbeck <em>et al.</em> 2015</a>). To demonstrate the utility of the red drum linkage map as a tool for inferring the genomic position of loci of interest, this approach was applied to an unpublished dataset of candidate genes identified from red drum transcripts obtained in a preliminary hydrocarbon exposure study (dataset doi: 10.7266/N73T9F7J). Using the final module of the <em>synteny_mapper</em> pipeline, cDNA sequences of 724 gene transcripts were screened against the genome of each comparison species, using BLAST+, as above. Transcript sequences with single hits to the genome of at least one comparison species were identified, and for each locus it was determined whether the locus was located in a region of shared synteny between the red drum linkage map and the genome of the comparison species. When transcript sequences were located in a region of shared synteny and the region contained more than two mapped loci, the script identified the smallest possible interval within the region into which the locus could be positioned and recorded the nearest left and right flanking loci. The distribution of shared syntenic blocks and inferred positions of synteny-mapped loci, relative to the red drum linkage map, were visualized using <em>ggplot2</em> (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wickham 2009</a>) and <em>circos</em> (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Krzywinski <em>et al.</em> 2009</a>).</p><div id="sec-2" class="subsection"><h3>Data availability</h3><p id="p-11">Unpublished oil exposure data are publicly available through the Gulf of Mexico Research Initiative Information &amp; Data Cooperative (GRIIDC) at <a href="https://data.gulfresearchinitiative.org">https://data.gulfresearchinitiative.org</a> (doi: 10.7266/N73T9F7J). Raw, demultiplexed sequence reads may be found in NCBI’s Short Read Archive (SRA) under accession number PRJNA357008. Additional data files, including final SNP dataset (VCF format) and SNP haplotype dataset (JOINMAP format) for both families can be found in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036350/-/DC1/FileS1.zip">File S1</a>. The final reduced representation reference genome can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036350/-/DC1/FileS2.zip">File S2</a>. Scripts to reproduce tables and figures, as well as additional data files, can be found at <a href="http://www.github.com/chollenbeck/red_drum_map">http://www.github.com/chollenbeck/red_drum_map</a>. </p></div></div><div class="section results" id="sec-3"><h2 class="">Results</h2><p id="p-12">Following assembly, the reduced-representation reference genome contained 38,887 RAD contigs, with a mean size of 264.9 bp. Additional filtering for contigs below minimum threshold size (&lt;150 bp), and those containing rRNA sequences resulted in a final reference assembly of 33,865 RAD contigs, with a mean size of 284.6 bp. The total length of the reference sequence was 9,638,003 bp. Assuming a total red drum genome size of ∼810 Mb (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gold <em>et al.</em> 1988</a>), the reference covered ∼1.19% of the genome.</p><p id="p-13">After filtering, the dataset for Family A (72 progeny) consisted of 786 RAD contigs (containing 1383 SNPs), and the dataset for Family B (81 progeny) consisted of 1340 RAD contigs (containing 2620 SNPs). The difference in number of usable RAD contigs between families was largely the result of differences in overall DNA quality of individual samples from the two families. A detailed summary of filtering results is presented in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036350/-/DC1/TableS1.xlsx">Table S1</a>. After combining RAD contig data with microsatellite and EST-SSR genotypes assayed previously from the same individuals, the total mapping dataset consisted of 1218 and 1779 loci for Families A and B, respectively.</p><p id="p-14">The consensus linkage map (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) contained 2275 loci, including 1794 RAD contigs (consisting of 3462 SNP loci), 437 anonymous microsatellite loci, and 44 EST-SSRs; the combined length of mapped fragments totaled 692.3 kb. The mean number of loci per linkage group was 94.79 and the mean marker interval was 0.94 cM. The average length of a linkage group was 87.72 cM; the total map length was 2105.30 cM. Because of the tendency of M<span class="sc">erge</span>M<span class="sc">ap</span> to inflate the total size of linkage groups when combining maps (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Khan <em>et al.</em> 2012</a>), the average of individual-specific maps may represent a more accurate estimate of total map size. Averaged across individual-specific maps, the mean length of linkage groups and total map length were 71.81 and 1704.40 cM, respectively. Of 2275 total loci, 278 (12.2%) were located directly in coding regions and could be assigned a putative function based on a BLAST search. Summary statistics for the consensus-, family-, and individual-specific maps are presented in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>; detailed information regarding map positions and annotations for the consensus linkage map are available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036350/-/DC1/TableS2.xlsx">Table S2</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/843/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Red drum consensus linkage map." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-498519465" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Red drum consensus linkage map.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/843/F1.medium.gif" width="440" height="289"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/3/843/F1.medium.gif" width="440" height="289"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/843/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/843/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483702" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-15" class="first-child">Red drum consensus linkage map.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/483639/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/483639/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary statistics for red drum linkage maps</span><div class="sb-div caption-clear"></div></div></div><p id="p-17">The total number of loci with significant BLAST hits to single chromosomes ranged from 342 (15.1% of all loci; green spotted puffer) to 1249 (55.3% of all loci; European seabass). The total number of blocks of shared synteny also varied among comparison species, ranging from 87 (green spotted puffer) to 284 (European seabass). The mean number of loci per block of shared synteny was less variable, ranging from 3.047 (stickleback) to 3.196 (fugu). Similarly, while the total size of blocks of shared synteny was highly variable among comparison species, ranging from 92.9 Mb (green spotted puffer) to 259 Mb (Nile tilapia), the proportion of the genome of the comparison species covered by blocks was less variable, ranging from 0.330 (green spotted puffer) to 0.438 (European seabass). Summary statistics for blocks of shared synteny for each of the six comparison species are presented in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>; a summary of shared syntenic regions in the form of a circular ideogram displaying blocks for each species relative to red drum linkage groups is presented in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/483646/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/483646/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary statistics for shared synteny analysis</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/843/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Circular ideogram of shared synteny analysis. Gray rectangles on the outside perimeter represent red drum linkage groups. Colored segments on the inside of the circle represent blocks of shared synteny between the red drum genetic map and genomes of six comparison species: red: stickleback (Gasterosteus aculatus); blue: Nile tilapia (Oreochromis niloticus); teal: green spotted puffer (Tetraodon nigroviridis), green: fugu (Takifugu rubripes); dark gray: European seabass (Dicentrarchus labrax); purple: barramundi (Lates calcarifer). Dark red regions on the outside of the circle represent regions where differentially expressed genes in oil exposure experiments were localized via a synteny-based mapping approach. Putative gene locations that overlap are vertically stacked for clarity." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-498519465" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Circular ideogram of shared synteny analysis. Gray rectangles on the outside perimeter represent red drum linkage groups. Colored segments on the inside of the circle represent blocks of shared synteny between the red drum genetic map and genomes of six comparison species: red: stickleback (&lt;em&gt;Gasterosteus aculatus&lt;/em&gt;); blue: Nile tilapia (&lt;em&gt;Oreochromis niloticus&lt;/em&gt;); teal: green spotted puffer (&lt;em&gt;Tetraodon nigroviridis&lt;/em&gt;), green: fugu (&lt;em&gt;Takifugu rubripes&lt;/em&gt;); dark gray: European seabass (&lt;em&gt;Dicentrarchus labrax&lt;/em&gt;); purple: barramundi (&lt;em&gt;Lates calcarifer&lt;/em&gt;). Dark red regions on the outside of the circle represent regions where differentially expressed genes in oil exposure experiments were localized via a synteny-based mapping approach. Putative gene locations that overlap are vertically stacked for clarity.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/843/F2.medium.gif" width="440" height="430"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/3/843/F2.medium.gif" width="440" height="430"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/843/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/843/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483671" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-19" class="first-child">Circular ideogram of shared synteny analysis. Gray rectangles on the outside perimeter represent red drum linkage groups. Colored segments on the inside of the circle represent blocks of shared synteny between the red drum genetic map and genomes of six comparison species: red: stickleback (<em>Gasterosteus aculatus</em>); blue: Nile tilapia (<em>Oreochromis niloticus</em>); teal: green spotted puffer (<em>Tetraodon nigroviridis</em>), green: fugu (<em>Takifugu rubripes</em>); dark gray: European seabass (<em>Dicentrarchus labrax</em>); purple: barramundi (<em>Lates calcarifer</em>). Dark red regions on the outside of the circle represent regions where differentially expressed genes in oil exposure experiments were localized via a synteny-based mapping approach. Putative gene locations that overlap are vertically stacked for clarity.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">Of 724 candidate gene transcripts identified in oil exposure experiments, 227 (31.4%) were assigned a putative position, using the synteny-mapping strategy (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036350/-/DC1/TableS3.xlsx">Table S3</a>). Of 277 synteny-mapped loci, 62 (27.3%) were annotated by a BLAST search. Inferred positions of all synteny-mapped genes are presented in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>. The distribution of shared syntenic blocks and synteny-mapped genes on red drum linkage group 22, where eight candidate gene transcripts were localized to a region of ∼10 cM, are shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/843/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Shared syntenic blocks and synteny-mapped genes on red drum linkage group 22. Colored rectangles represent blocks of shared synteny between the red drum genetic map (horizontal axis) and genomes of six comparison species. Dark-red line segments represent regions where differentially expressed genes in oil exposure experiments were localized via a synteny-based mapping approach. Asterisks denote locations of genes that were synteny-mapped to zero-recombination intervals on the linkage map." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-498519465" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Shared syntenic blocks and synteny-mapped genes on red drum linkage group 22. Colored rectangles represent blocks of shared synteny between the red drum genetic map (horizontal axis) and genomes of six comparison species. Dark-red line segments represent regions where differentially expressed genes in oil exposure experiments were localized via a synteny-based mapping approach. Asterisks denote locations of genes that were synteny-mapped to zero-recombination intervals on the linkage map.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/843/F3.medium.gif" width="440" height="279"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/3/843/F3.medium.gif" width="440" height="279"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/843/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/843/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483719" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-21" class="first-child">Shared syntenic blocks and synteny-mapped genes on red drum linkage group 22. Colored rectangles represent blocks of shared synteny between the red drum genetic map (horizontal axis) and genomes of six comparison species. Dark-red line segments represent regions where differentially expressed genes in oil exposure experiments were localized via a synteny-based mapping approach. Asterisks denote locations of genes that were synteny-mapped to zero-recombination intervals on the linkage map.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section discussion" id="sec-4"><h2 class="">Discussion</h2><p id="p-22">In total, 1794 haplotyped RAD contigs, comprising 3462 SNP loci, were added to the red drum linkage map. Addition of these loci reduced the mean marker interval to &lt;1 cM. The total length of the consensus map (2105.30 cM) is larger than reported previously for the microsatellite-based map (1815.3 cM, <a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">Hollenbeck <em>et al.</em> 2015</a>); this difference is likely a result of combining maps from different families, using MERGEMAP, which is reported to inflate total map length (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Khan <em>et al.</em> 2012</a>). Relative to the average total length of individual-based maps reported here, the total length of the consensus map was inflated by ∼24%, consistent with the results of <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">McKinney <em>et al.</em> (2015)</a>, who found that MERGEMAP increased the size of the Chinook salmon consensus map by ∼30% per additional family. However, there exists a tradeoff between accuracy of map lengths and inclusion of loci on the map (<a id="xref-ref-24-7" class="xref-bibr" href="#ref-24">Hollenbeck <em>et al.</em> 2015</a>). Because the purpose of our study was to determine the relative positions of loci, rather than the exact frequency of recombination between them, we chose to add more loci to the consensus map by combining information between mapping families.</p><p id="p-23">The RAD-seq linkage mapping process employed here has a number of benefits. First, using long reads from the Illumina MiSeq platform enabled recovery of the entire sequence of each discrete ddRAD contig, thus providing more sequence coverage for comparative genomics and other downstream applications. Second, using haplotypes instead of individual SNPs eliminated redundancy and saved computational time by condensing tightly linked SNPs into single, multi-SNP loci, potentially possessing multiple alleles, thus discriminating more discrete alleles than would otherwise be identified, and, thereby, increasing the number of alleles that segregate in an informative manner (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Ball <em>et al.</em> 2010</a>). Last, haplotyping allows possible multi-copy loci to be filtered from the dataset because multi-copy loci often display more than two haplotypes within an individual and can be identified on this basis.</p><p id="p-24">Of all comparison species, European seabass, followed by barramundi, had the highest degree of similarity with red drum in terms of total number of homologous loci (established via BLAST search), number and proportion of the genome contained in shared-synteny blocks. The lowest degree of similarity in all three measures was found between red drum and green spotted puffer. Correspondingly, shared-synteny block size was smallest in seabass and barramundi, and largest in fugu and green spotted pufferfish. Finally, in contrast to the considerable variation in the total number of BLAST hits and number of shared syntenic blocks among comparison species, the mean number of loci per block, and the proportion of the comparison species genome in blocks of shared synteny were much more similar among comparisons. There are a number of factors that likely produce this pattern, including the phylogenetic relationship between species, structural changes during genome evolution, and the quality of genome assemblies used in the analysis. European seabass, for example, is more closely related phylogenetically to red drum than green spotted pufferfish (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Nelson <em>et al.</em> 2016</a>), which likely explains the larger number of BLAST hits in European seabass (1249) as compared to the green spotted pufferfish (342). Having more homologous loci affords the opportunity to detect smaller blocks of shared synteny, as well as the fine-scale rearrangements and translocations that tend to break up larger regions of shared synteny into smaller blocks (<a id="xref-ref-24-8" class="xref-bibr" href="#ref-24">Hollenbeck <em>et al.</em> 2015</a>).</p><p id="p-25">Results of the synteny analyses were consistent with a high degree of stability in karyotype evolution among teleost fishes (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Amores <em>et al.</em> 2014</a>). An important implication of this high level of shared synteny is that the information from well characterized genomes can be used to localize candidate genes of interest in nonmodel species. In our study for example, putative locations of previously unmapped candidate gene transcripts were achieved with a success rate of 31.4%, suggesting that the general approach will be useful in localizing candidate genes from environmental impact studies, or other gene expression studies in species for which a complete reference genome is unavailable.</p><p id="p-26">The linkage map described herein will be a valuable resource for studies of red drum and other nonmodel species. Red drum, for example, are cultured both for restoration enhancement and commercial production purposes (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Gold <em>et al.</em> 2008</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">FAO 2015</a>). A linkage map can be used to great advantage for commercial aquaculture by enabling the mapping of quantitative traits and facilitating marker-assisted selection for genetic improvement (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Liu and Cordes 2004</a>), including the identification of chromosomal regions impacting disease resistance (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Houston <em>et al.</em> 2012</a>) and sex determination (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Palaiokostas <em>et al.</em> 2013a</a>,<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">b</a>). In addition, a combination of linkage and linkage disequilibrium (LD) data also has been shown to be effective in detecting changes in contemporary effective population size of wild red drum over time (<a id="xref-ref-24-9" class="xref-bibr" href="#ref-24">Hollenbeck <em>et al.</em> 2015</a>). High-density linkage maps also can be useful tools for chromosome-level scaffolding in genome assembly (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Fierst 2015</a>), and linkage map/genome integrations are increasingly common (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gonen <em>et al.</em> 2014</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Tine <em>et al.</em> 2014</a>). Finally, linkage maps, when combined with population genetics data, can provide a genomic context for loci used in population-level studies to identify genomic islands of adaptation (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Bradbury <em>et al.</em> 2013</a>) and associations among loci implicated in adaptation (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Hohenlohe <em>et al.</em> 2012</a>).</p></div><div class="section" id="sec-5"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-27">We thank S. O’Leary, J. Puritz, and S. Willis for helpful discussions regarding analytical methods. Work was supported in part by institutional grants (NA10OAR4170099, NA14AR4170102) to the Texas Sea Grant College Program from the National Sea Grant Office, National Oceanic and Atmospheric Administration, United States Department of Commerce, by a grant (#447715) from the Texas Parks and Wildlife Department, and by a grant from The Gulf of Mexico Research Initiative/C-IMAGE II. Additional funding for C.M.H. was also provided by the Harte Research Institute for Gulf of Mexico Studies. This paper is number 108 in the series ”Genetic Studies in Marine Fishes” and contribution number 14 of the Marine Genomics Laboratory.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-28">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036350/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036350/-/DC1</a>.</p></li><li class="fn" id="fn-4"><p id="p-29"><em>Communicating editor: R. Houston</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-10-17"><span class="received-label">Received </span>October 17, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-01-03"><span class="accepted-label">Accepted </span>January 3, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Hollenbeck <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.1" data-doi="10.1016/j.aquatox.2005.05.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alvarez</span>  <span class="cit-name-given-names">M. D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuiman</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Environmental levels of atrazine and its degradation products impair survival skills and growth of red drum larvae.</span> <abbr class="cit-jnl-abbrev">Aquat. Toxicol.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">229</span>–<span class="cit-lpage">241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquatic%2BToxicology%26rft.stitle%253DAquatic%2BToxicology%26rft.aulast%253Ddel%2BCarmen%2BAlvarez%26rft.auinit1%253DM.%26rft.volume%253D74%26rft.issue%253D3%26rft.spage%253D229%26rft.epage%253D241%26rft.atitle%253DEnvironmental%2Blevels%2Bof%2Batrazine%2Band%2Bits%2Bdegradation%2Bproducts%2Bimpair%2Bsurvival%2Bskills%2Band%2Bgrowth%2Bof%2Bred%2Bdrum%2Blarvae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquatox.2005.05.014%26rft_id%253Dinfo%253Apmid%252F16009435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquatox.2005.05.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16009435&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F843.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alvarez</span>  <span class="cit-name-given-names">M. D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuiman</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Ecological performance of red drum (<em>Sciaenops ocellatus</em>) larvae exposed to environmental levels of the insecticide malathion.</span> <abbr class="cit-jnl-abbrev">Environ. Toxicol. Chem.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1426</span>–<span class="cit-lpage">1432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnvironmental%2BToxicology%2Band%2BChemistry%26rft.stitle%253DEnvironmental%2BToxicology%2Band%2BChemistry%26rft.aulast%253DDel%2BCarmen%2BAlvarez%26rft.auinit1%253DM.%26rft.volume%253D25%26rft.issue%253D5%26rft.spage%253D1426%26rft.epage%253D1432%26rft.atitle%253DEcological%2Bperformance%2Bof%2Bred%2Bdrum%2B%2528Sciaenops%2Bocellatus%2529%2Blarvae%2Bexposed%2Bto%2Benvironmental%2Blevels%2Bof%2Bthe%2Binsecticide%2Bmalathion.%26rft_id%253Dinfo%253Apmid%252F16704078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16704078&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F843.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.3" data-doi="10.1534/genetics.111.127324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrara</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fontenot</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Postlethwait</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome evolution and meiotic maps by massively parallel DNA sequencing: spotted gar, an outgroup for the teleost genome duplication.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">799</span>–<span class="cit-lpage">808</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAmores%26rft.auinit1%253DA.%26rft.volume%253D188%26rft.issue%253D4%26rft.spage%253D799%26rft.epage%253D808%26rft.atitle%253DGenome%2BEvolution%2Band%2BMeiotic%2BMaps%2Bby%2BMassively%2BParallel%2BDNA%2BSequencing%253A%2BSpotted%2BGar%252C%2Ban%2BOutgroup%2Bfor%2Bthe%2BTeleost%2BGenome%2BDuplication%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.127324%26rft_id%253Dinfo%253Apmid%252F21828280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzQvNzk5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzg0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.4" data-doi="10.1534/genetics.114.164293"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nanda</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warren</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walter</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A RAD-tag genetic map for the platyfish (<em>Xiphophorus maculatus</em>) reveals mechanisms of karyotype evolution among teleost fish.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">625</span>–<span class="cit-lpage">641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164293%26rft_id%253Dinfo%253Apmid%252F24700104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk3LzIvNjI1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzg0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.5" data-doi="10.1371/journal.pone.0040701"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodríguez Marí</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Braasch</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Multiple sex-associated regions and a putative sex chromosome in zebrafish revealed by RAD mapping and population genomics.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e40701</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DAnderson%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De40701%26rft.epage%253De40701%26rft.atitle%253DMultiple%2Bsex-associated%2Bregions%2Band%2Ba%2Bputative%2Bsex%2Bchromosome%2Bin%2Bzebrafish%2Brevealed%2Bby%2BRAD%2Bmapping%2Band%2Bpopulation%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0040701%26rft_id%253Dinfo%253Apmid%252F22792396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0040701&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22792396&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F843.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.6" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwood</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currey</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiver</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F843.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.7" data-doi="10.1186/1471-2164-11-218"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ball</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stapley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birkhead</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A comparison of SNPs and microsatellites as linkage mapping markers: lessons from the zebra finch (<em>Taeniopygia guttata</em>).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DBall%26rft.auinit1%253DA.%2BD.%26rft.volume%253D11%26rft.spage%253D218%26rft.epage%253D218%26rft.atitle%253DA%2Bcomparison%2Bof%2BSNPs%2Band%2Bmicrosatellites%2Bas%2Blinkage%2Bmapping%2Bmarkers%253A%2Blessons%2Bfrom%2Bthe%2Bzebra%2Bfinch%2B%2528Taeniopygia%2Bguttata%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-218%26rft_id%253Dinfo%253Apmid%252F20359323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-218&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20359323&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F843.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.8" data-doi="10.1371/journal.pone.0019315"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baxter</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shelton</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heckel</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Linkage mapping and comparative genomics using next-generation RAD sequencing of a non-model organism.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19315</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaxter%26rft.auinit1%253DS.%2BW.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253De19315%26rft.epage%253De19315%26rft.atitle%253DLinkage%2Bmapping%2Band%2Bcomparative%2Bgenomics%2Busing%2Bnext-generation%2BRAD%2Bsequencing%2Bof%2Ba%2Bnon-model%2Borganism.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019315%26rft_id%253Dinfo%253Apmid%252F21541297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019315&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21541297&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F843.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.9" data-doi="10.1111/eva.12026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">I. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubert</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borza</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic islands of divergence and their consequences for the resolution of spatial structure in an exploited marine fish.</span> <abbr class="cit-jnl-abbrev">Evol. Appl.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">450</span>–<span class="cit-lpage">461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BAppl.%26rft.volume%253D6%26rft.spage%253D450%26rft_id%253Dinfo%253Adoi%252F10.1111%252Feva.12026%26rft_id%253Dinfo%253Apmid%252F23745137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/eva.12026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23745137&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F843.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.10" data-doi="10.1186/2047-217X-2-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradnam</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fass</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alexandrov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baranay</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bechner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Assemblathon 2: evaluating de novo methods of genome assembly in three vertebrate species.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D2%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.1186%252F2047-217X-2-10%26rft_id%253Dinfo%253Apmid%252F23870653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/2047-217X-2-10&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23870653&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F843.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.11" data-doi="10.1186/1471-2105-10-421"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Camacho</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulouris</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Avagyan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papadopoulos</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">BLAST+: architecture and applications.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DCamacho%26rft.auinit1%253DC.%26rft.volume%253D10%26rft.spage%253D421%26rft.epage%253D421%26rft.atitle%253DBLAST%252B%253A%2Barchitecture%2Band%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-421%26rft_id%253Dinfo%253Apmid%252F20003500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-421&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20003500&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F843.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Postlethwait</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Stacks: building and genotyping loci de novo from short-read sequences.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D1%26rft.spage%253D171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.13" data-doi="10.1186/1471-2164-12-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chutimanitsakun</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nipper</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuesta-Marcos</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cistué</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corey</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Construction and application for QTL analysis of a restriction site associated DNA (RAD) linkage map in barley.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DChutimanitsakun%26rft.auinit1%253DY.%26rft.volume%253D12%26rft.spage%253D4%26rft.epage%253D4%26rft.atitle%253DConstruction%2Band%2Bapplication%2Bfor%2BQTL%2Banalysis%2Bof%2Ba%2BRestriction%2BSite%2BAssociated%2BDNA%2B%2528RAD%2529%2Blinkage%2Bmap%2Bin%2Bbarley.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-4%26rft_id%253Dinfo%253Apmid%252F21205322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21205322&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F843.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.14" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMTUvMjE1NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMy84NDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Diaz-Gil</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catalan</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faulk</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuiman</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Ocean acidification increases fatty acids levels of larval fish.</span> <abbr class="cit-jnl-abbrev">Biol. Lett.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">201503</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BLett.%26rft.volume%253D11%26rft.spage%253D201503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.16" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F843.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-web" id="cit-7.3.843.17"><div class="cit-metadata unstructured">FAO, 2015 Global aquaculture production: 1950–2014. Available at: <a href="http://www.fao.org/fishery/statistics/global-aquaculture-production/query/en">http://www.fao.org/fishery/statistics/global-aquaculture-production/query/en</a>. Accessed Date: December 9, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.18" data-doi="10.3389/fgene.2015.00001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fierst</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Using linkage maps to correct and scaffold de novo genome assemblies: methods, challenges, and computational tools.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D6%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2015.00001%26rft_id%253Dinfo%253Apmid%252F25674101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2015.00001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25674101&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F843.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gold</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kedzie</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bohymeyer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkin</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karel</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Studies on the basic structure of the red drum (<em>Sciaenops ocellatus</em>) genome.</span> <abbr class="cit-jnl-abbrev">Contrib. Mar. Sci.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DContrib.%2BMar.%2BSci.%26rft.volume%253D30%26rft.spage%253D57%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gold</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saillant</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vega</span>  <span class="cit-name-given-names">R. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic Effective Size in Populations of Hatchery-Raised Red Drum Released for Stock Enhancement.</span> <abbr class="cit-jnl-abbrev">Transactions of the American Fisheries Society</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">1327</span>–<span class="cit-lpage">1334</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTransactions%2Bof%2Bthe%2BAmerican%2BFisheries%2BSociety%26rft.volume%253D137%26rft.spage%253D1327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.21" data-doi="10.1186/1471-2164-15-166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cezard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Linkage maps of the Atlantic salmon (<em>Salmo salar</em>) genome derived from RAD sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D166%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-166%26rft_id%253Dinfo%253Apmid%252F24571138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-166&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24571138&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F843.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.22" data-doi="10.1111/mec.12860"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henning</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franchini</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic mapping of horizontal stripes in Lake Victoria cichlid fishes: benefits and pitfalls of using RAD markers for dense linkage mapping.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">5224</span>–<span class="cit-lpage">5240</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D5224%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12860%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12860&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.23" data-doi="10.1098/rstb.2011.0245"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassham</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Extensive linkage disequilibrium and parallel adaptive divergence across threespine stickleback genomes.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">367</span>: <span class="cit-fpage">395</span>–<span class="cit-lpage">408</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DHohenlohe%26rft.auinit1%253DP.%2BA.%26rft.volume%253D367%26rft.issue%253D1587%26rft.spage%253D395%26rft.epage%253D408%26rft.atitle%253DExtensive%2Blinkage%2Bdisequilibrium%2Band%2Bparallel%2Badaptive%2Bdivergence%2Bacross%2Bthreespine%2Bstickleback%2Bgenomes%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2011.0245%26rft_id%253Dinfo%253Apmid%252F22201169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEyOiIzNjcvMTU4Ny8zOTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvODQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hollenbeck</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Portnoy</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gold</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A genetic linkage map of red drum (<em>Sciaenops ocellatus</em>) and comparison of chromosomal syntenies with four other fish species.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">435</span>: <span class="cit-fpage">265</span>–<span class="cit-lpage">274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D435%26rft.spage%253D265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.3.843.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hollenbeck</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Portnoy</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gold</span>  <span class="cit-name-given-names">J.R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A method for detecting recent changes in contemporary effective population size from linkage disequilibrium at linked and unlinked loci.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D117%26rft.spage%253D207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.26" data-doi="10.1186/1471-2164-13-244"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Houston</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mota-Velasco</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Characterization of QTL-linked and genome-wide restriction site-associated DNA (RAD) markers in farmed Atlantic salmon.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DHouston%26rft.auinit1%253DR.%2BD.%26rft.volume%253D13%26rft.spage%253D244%26rft.epage%253D244%26rft.atitle%253DCharacterisation%2Bof%2BQTL-linked%2Band%2Bgenome-wide%2Brestriction%2Bsite-associated%2BDNA%2B%2528RAD%2529%2Bmarkers%2Bin%2Bfarmed%2BAtlantic%2Bsalmon.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-244%26rft_id%253Dinfo%253Apmid%252F22702806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-244&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22702806&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F843.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-book" id="cit-7.3.843.27" data-doi="10.1371/journal.pone.0047864"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khan</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troggio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korban</span>  <span class="cit-name-given-names">S. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A multi-population consensus genetic map reveals inconsistent marker order among maps likely attributed to structural variations in the apple genome.</span> <span class="cit-source">PLoS One</span> <span class="cit-vol">7</span>: <span class="cit-fpage">e47864</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DKhan%26rft.auinit1%253DM.%2BA.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De47864%26rft.epage%253De47864%26rft.atitle%253DA%2BMulti-Population%2BConsensus%2BGenetic%2BMap%2BReveals%2BInconsistent%2BMarker%2BOrder%2Bamong%2BMaps%2BLikely%2BAttributed%2Bto%2BStructural%2BVariations%2Bin%2Bthe%2BApple%2BGenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0047864%26rft_id%253Dinfo%253Apmid%252F23144832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0047864&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23144832&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F843.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.28" data-doi="10.1101/gr.092759.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krzywinski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schein</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birol</span>  <span class="cit-name-given-names">İ.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Circos: an information aesthetic for comparative genomics.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1639</span>–<span class="cit-lpage">1645</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.092759.109%26rft_id%253Dinfo%253Apmid%252F19541911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTYzOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMy84NDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.29" data-doi="10.1016/j.aquaculture.2004.05.027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cordes</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">DNA marker technologies and their applications in aquaculture genetics.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">238</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D238%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2004.05.027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2004.05.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000224004300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manousaki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsakogiannis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taggart</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palaiokostas</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsaparis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Exploring a non-model teleost genome through RAD sequencing - linkage mapping in common pandora, <em>Pagellus erythrinus</em> and comparative genomic analysis.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">509</span>–<span class="cit-lpage">519</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D6%26rft.spage%253D509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKinney</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomez-Uchida</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Limborg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">An integrated linkage map reveals candidate genes underlying adaptive variation in Chinook salmon (<em>Oncorhynchus tshawytscha</em>).</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">769</span>–<span class="cit-lpage">783</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D16%26rft.spage%253D769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-book" id="cit-7.3.843.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grande</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">M. V. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-source">Fishes of the World</span>. <span class="cit-publ-name">John Wiley &amp; Sons</span>, <span class="cit-publ-loc">Hoboken, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.33" data-doi="10.1186/1471-2164-14-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palaiokostas</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bekaert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davie</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cowan</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oral</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">a</span> <span class="cit-article-title">Mapping the sex determination locus in the Atlantic halibut (<em>Hippoglossus hippoglossus</em>) using RAD sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.issue%253D1%26rft.spage%253D1%26rft.atitle%253DTHE%2BCOMPLEX%2BTRANSCRIPTIONAL%2BLANDSCAPE%2BOF%2BTHE%2BANUCLEATE%2BHUMAN%2BPLATELET.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-1%26rft_id%253Dinfo%253Apmid%252F23323973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23323973&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F843.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.34" data-doi="10.1371/journal.pone.0068389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palaiokostas</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bekaert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khan</span>  <span class="cit-name-given-names">M. G. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taggart</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">b</span> <span class="cit-article-title">Mapping and validation of the major sex-determining region in Nile tilapia (<em>Oreochromis niloticus L</em>.) using RAD sequencing.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e68389</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De68389%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0068389%26rft_id%253Dinfo%253Apmid%252F23874606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0068389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23874606&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F843.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-7.3.843.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pattillo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Czapla</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monaco</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-source"><em>Distribution and Abundance of Fishes and Invertebrates in Gulf of Mexico Estuaries</em>, <em>Volume II: Species Life History Summaries</em>.</span> <span class="cit-publ-name">United States Department of Commerce</span>, <span class="cit-publ-loc">Rockville, MD</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.36" data-doi="10.1371/journal.pone.0037135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kay</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e37135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPeterson%26rft.auinit1%253DB.%2BK.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De37135%26rft.epage%253De37135%26rft.atitle%253DDouble%2Bdigest%2BRADseq%253A%2Ban%2Binexpensive%2Bmethod%2Bfor%2Bde%2Bnovo%2BSNP%2Bdiscovery%2Band%2Bgenotyping%2Bin%2Bmodel%2Band%2Bnon-model%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0037135%26rft_id%253Dinfo%253Apmid%252F22675423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0037135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22675423&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F843.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Portnoy</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puritz</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollenbeck</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gelsleichter</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Selection and sex-biased dispersal in a coastal shark: the influence of philopatry on adaptive variation.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">5877</span>–<span class="cit-lpage">5885</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D24%26rft.spage%253D5877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.38" data-doi="10.7717/peerj.431"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Puritz</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollenbeck</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gold</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">dDocent: a RADseq, variant-calling pipeline designed for population genomics of non-model organisms.</span> <abbr class="cit-jnl-abbrev">PeerJ</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e431</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPeerJ%26rft.volume%253D2%26rft.spage%253De431%26rft_id%253Dinfo%253Adoi%252F10.7717%252Fpeerj.431%26rft_id%253Dinfo%253Apmid%252F24949246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7717/peerj.431&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24949246&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F843.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Talukder</span>  <span class="cit-name-given-names">Z. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hulke</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pegadaraju</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A high-density SNP map of sunflower derived from RAD-sequencing facilitating fine-mapping of the rust resistance gene R12.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e98628</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De98628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.40" data-doi="10.1038/ncomms6770"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tine</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhl</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gagnaire</span>  <span class="cit-name-given-names">P.-A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louro</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desmarais</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">European sea bass genome and its variation provide insights into adaptation to euryhalinity and speciation.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">5770</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D5770%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms6770%26rft_id%253Dinfo%253Apmid%252F25534655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms6770&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25534655&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F843.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.41" data-doi="10.1017/S0016672311000279"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Ooijen</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Multipoint maximum likelihood mapping in a full-sib family of an outbreeding species.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DVAN%2BOoijen%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D93%26rft.issue%253D5%26rft.spage%253D343%26rft.epage%253D349%26rft.atitle%253DMultipoint%2Bmaximum%2Blikelihood%2Bmapping%2Bin%2Ba%2Bfull-sib%2Bfamily%2Bof%2Ban%2Boutbreeding%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672311000279%26rft_id%253Dinfo%253Apmid%252F21878144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672311000279&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21878144&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F843.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-book" id="cit-7.3.843.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Ooijen</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-source"><em>JoinMap 4.1</em>, <em>Software for the Calculation of Genetic Linkage Maps in Experimental Populations of Diploid Species</em></span>. <span class="cit-publ-name">Kyazma BV</span>, <span class="cit-publ-loc">Wageningen, Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-book" id="cit-7.3.843.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wickham</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">ggplot2: Elegant Graphics for Data Analysis</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.44" data-doi="10.1111/1755-0998.12647"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollenbeck</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puritz</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gold</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Portnoy</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Haplotyping RAD loci: an efficient method to filter paralogs and account for physical linkage.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/1755-0998.12647</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12647&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lonardi</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accurate construction of consensus genetic maps via integer linear programming.</span> <abbr class="cit-jnl-abbrev">IEEE/ACM Trans. Comput. Biol. Bioinformatics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">394</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%252FACM%2BTrans.%2BComput.%2BBiol.%2BBioinformatics%26rft.volume%253D8%26rft.spage%253D381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.843.46" data-doi="10.1534/genetics.115.179028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ackerman</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bright</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spitze</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A male-specific genetic map of the microcrustacean <em>Daphnia pulex</em> based on single sperm whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">31</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.179028%26rft_id%253Dinfo%253Apmid%252F26116153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMjAxLzEvMzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvODQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/3/843.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/3/835" title="Genetics and Adaptation of Soybean Cyst Nematode to Broad Spectrum Soybean Resistance" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/3/851" title="SHH Protein Variance in the Limb Bud Is Constrained by Feedback Regulation and Correlates with Altered Digit Patterning" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/3"><span>Volume 7 Issue 3, March 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/3.cover-source.jpg?itok=h8yj2GBo" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/3/843.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-6QKNA34v60m10qtXYF1d0PRDWgdWPDSrGlI9OJ23Bus" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/3/843" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/3/843" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/3/843" class="active" data-icon-position="" data-hide-link-title="0">Linkage Mapping and Comparative Genomics of Red Drum (Sciaenops ocellatus) Using Next-Generation Sequencing</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/483531" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-r-q5BBY2zqWWeHNMUC1Agz5OGfPY-bzbUZfX_4oDp-M" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/3/843.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/3/843" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-bVKlmiv1wnGPfFGCH6UWQpQFfNwcSS39oUPar5Yu38I" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/3/843" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/3/843.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483531" id="node483531--2" data-pisa="ggg;7/3/843" data-pisa-master="ggg;g3.116.036350" data-apath="/ggg/7/3/843.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Linkage Mapping and Comparative Genomics of Red Drum (<em>Sciaenops ocellatus</em>) Using Next-Generation Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0227-7225" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christopher M. Hollenbeck</span>, <span class="highwire-citation-author" data-delta="1">David S. Portnoy</span>, <span class="highwire-citation-author" data-delta="2">Dana Wetzel</span>, <span class="highwire-citation-author" data-delta="3">Tracy A. Sherwood</span>, <span class="highwire-citation-author" data-delta="4">Paul B. Samollow</span> and <span class="highwire-citation-author" data-delta="5">John R. Gold</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">843-850;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.036350 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483531--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Christopher M. Hollenbeck </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Marine Genomics Laboratory, Department of Life Sciences, Texas A&M University, Corpus Christi, Texas 78412</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BM.%2BHollenbeck%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hollenbeck%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristopher%2BM.%2BHollenbeck%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0227-7225" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christopher M. Hollenbeck</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ch263@st-andrews.ac.uk" class="" data-icon-position="" data-hide-link-title="0">ch263@st-andrews.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">David S. Portnoy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Marine Genomics Laboratory, Department of Life Sciences, Texas A&M University, Corpus Christi, Texas 78412</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BS.%2BPortnoy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Portnoy%20DS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BS.%2BPortnoy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Dana Wetzel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Mote Marine Laboratory and Aquarium, Sarasota, Florida 34236</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dana%2BWetzel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wetzel%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADana%2BWetzel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Tracy A. Sherwood </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Mote Marine Laboratory and Aquarium, Sarasota, Florida 34236</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tracy%2BA.%2BSherwood%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sherwood%20TA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATracy%2BA.%2BSherwood%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Paul B. Samollow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Veterinary Integrative Biosciences, Texas A&M University, College Station, Texas 77845</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BB.%2BSamollow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Samollow%20PB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaul%2BB.%2BSamollow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">John R. Gold </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Marine Genomics Laboratory, Department of Life Sciences, Texas A&M University, Corpus Christi, Texas 78412</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BR.%2BGold%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gold%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BR.%2BGold%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/3/843" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/3/843&amp;title=Linkage%20Mapping%20and%20Comparative%20Genomics%20of%20Red%20Drum%20%28Sciaenops%20ocellatus%29%20Using%20Next-Generation%20Sequencing" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/3/843&amp;title=Linkage%20Mapping%20and%20Comparative%20Genomics%20of%20Red%20Drum%20%28Sciaenops%20ocellatus%29%20Using%20Next-Generation%20Sequencing" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/3/843&amp;title=Linkage%20Mapping%20and%20Comparative%20Genomics%20of%20Red%20Drum%20%28Sciaenops%20ocellatus%29%20Using%20Next-Generation%20Sequencing" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/3/843&amp;text=Linkage%20Mapping%20and%20Comparative%20Genomics%20of%20Red%20Drum%20%28Sciaenops%20ocellatus%29%20Using%20Next-Generation%20Sequencing" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/3/843&amp;title=Linkage%20Mapping%20and%20Comparative%20Genomics%20of%20Red%20Drum%20%28Sciaenops%20ocellatus%29%20Using%20Next-Generation%20Sequencing" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/3/843&amp;t=Linkage%20Mapping%20and%20Comparative%20Genomics%20of%20Red%20Drum%20%28Sciaenops%20ocellatus%29%20Using%20Next-Generation%20Sequencing" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/3/843&amp;title=Linkage%20Mapping%20and%20Comparative%20Genomics%20of%20Red%20Drum%20%28Sciaenops%20ocellatus%29%20Using%20Next-Generation%20Sequencing" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/3/843&amp;title=Linkage%20Mapping%20and%20Comparative%20Genomics%20of%20Red%20Drum%20%28Sciaenops%20ocellatus%29%20Using%20Next-Generation%20Sequencing" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/483531/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483531" id="node483531--3" data-pisa="ggg;7/3/843" data-pisa-master="ggg;g3.116.036350" data-apath="/ggg/7/3/843.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Linkage Mapping and Comparative Genomics of Red Drum (<em>Sciaenops ocellatus</em>) Using Next-Generation Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0227-7225" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christopher M. Hollenbeck</span>, <span class="highwire-citation-author" data-delta="1">David S. Portnoy</span>, <span class="highwire-citation-author" data-delta="2">Dana Wetzel</span>, <span class="highwire-citation-author" data-delta="3">Tracy A. Sherwood</span>, <span class="highwire-citation-author" data-delta="4">Paul B. Samollow</span> and <span class="highwire-citation-author" data-delta="5">John R. Gold</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">843-850;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.036350 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483531--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Christopher M. Hollenbeck </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Marine Genomics Laboratory, Department of Life Sciences, Texas A&M University, Corpus Christi, Texas 78412</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BM.%2BHollenbeck%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hollenbeck%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristopher%2BM.%2BHollenbeck%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0227-7225" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christopher M. Hollenbeck</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ch263@st-andrews.ac.uk" class="" data-icon-position="" data-hide-link-title="0">ch263@st-andrews.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">David S. Portnoy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Marine Genomics Laboratory, Department of Life Sciences, Texas A&M University, Corpus Christi, Texas 78412</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BS.%2BPortnoy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Portnoy%20DS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BS.%2BPortnoy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Dana Wetzel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Mote Marine Laboratory and Aquarium, Sarasota, Florida 34236</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dana%2BWetzel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wetzel%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADana%2BWetzel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Tracy A. Sherwood </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Mote Marine Laboratory and Aquarium, Sarasota, Florida 34236</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tracy%2BA.%2BSherwood%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sherwood%20TA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATracy%2BA.%2BSherwood%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Paul B. Samollow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Veterinary Integrative Biosciences, Texas A&M University, College Station, Texas 77845</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BB.%2BSamollow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Samollow%20PB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaul%2BB.%2BSamollow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">John R. Gold </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Marine Genomics Laboratory, Department of Life Sciences, Texas A&M University, Corpus Christi, Texas 78412</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BR.%2BGold%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gold%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BR.%2BGold%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=843&amp;rft.epage=850&amp;rft.atitle=Linkage%20Mapping%20and%20Comparative%20Genomics%20of%20Red%20Drum%20%28Sciaenops%20ocellatus%29%20Using%20Next-Generation%20Sequencing&amp;rft.volume=7&amp;rft.issue=3&amp;rft.date=2017-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Hollenbeck%2C+Christopher+M.&amp;rft.au=Portnoy%2C+David+S.&amp;rft.au=Wetzel%2C+Dana&amp;rft.au=Sherwood%2C+Tracy+A.&amp;rft.au=Samollow%2C+Paul+B.&amp;rft.au=Gold%2C+John+R."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/483531/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/483531/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/483531/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/483531/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/483531/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/483531/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/483531/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/483531/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/483531/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/483531/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/483531/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/483531/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/3/843" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/3/843.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/3/843.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/3/843.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/3/843.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
