<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/3/935.full.txt" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/3/935.full.pdf" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/3/935/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/3/935.atom" />
<meta name="HW.pisa" content="ggg;7/3/935" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Single-Step BLUP with Varying Genotyping Effort in Open-Pollinated Picea glauca" />
<meta name="DC.Identifier" content="10.1534/g3.116.037895" />
<meta name="DC.Date" content="2017-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Ratcliffe et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Maximization of genetic gain in forest tree breeding programs is contingent on the accuracy of the predicted breeding values and precision of the estimated genetic parameters. We investigated the effect of the combined use of contemporary pedigree information and genomic relatedness estimates on the accuracy of predicted breeding values and precision of estimated genetic parameters, as well as rankings of selection candidates, using single-step genomic evaluation (HBLUP). In this study, two traits with diverse heritabilities [tree height (HT) and wood density (WD)] were assessed at various levels of family genotyping efforts (0, 25, 50, 75, and 100%) from a population of white spruce ( Picea glauca ) consisting of 1694 trees from 214 open-pollinated families, representing 43 provenances in Québec, Canada. The results revealed that HBLUP bivariate analysis is effective in reducing the known bias in heritability estimates of open-pollinated populations, as it exposes hidden relatedness, potential pedigree errors, and inbreeding. The addition of genomic information in the analysis considerably improved the accuracy in breeding value estimates by accounting for both Mendelian sampling and historical coancestry that were not captured by the contemporary pedigree alone. Increasing family genotyping efforts were associated with continuous improvement in model fit, precision of genetic parameters, and breeding value accuracy. Yet, improvements were observed even at minimal genotyping effort, indicating that even modest genotyping effort is effective in improving genetic evaluation. The combined utilization of both pedigree and genomic information may be a cost-effective approach to increase the accuracy of breeding values in forest tree breeding programs where shallow pedigrees and large testing populations are the norm." />
<meta name="DC.Contributor" content="Blaise Ratcliffe" />
<meta name="DC.Contributor" content="Omnia Gamal El-Dien" />
<meta name="DC.Contributor" content="Eduardo P. Cappa" />
<meta name="DC.Contributor" content="Ilga Porth" />
<meta name="DC.Contributor" content="Jaroslav Klápště" />
<meta name="DC.Contributor" content="Charles Chen" />
<meta name="DC.Contributor" content="Yousry A. El-Kassaby" />
<meta name="article:published_time" content="2017-03-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Single-Step BLUP with Varying Genotyping Effort in Open-Pollinated Picea glauca" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Maximization of genetic gain in forest tree breeding programs is contingent on the accuracy of the predicted breeding values and precision of the estimated genetic parameters. We investigated the effect of the combined use of contemporary pedigree information and genomic relatedness estimates on the accuracy of predicted breeding values and precision of estimated genetic parameters, as well as rankings of selection candidates, using single-step genomic evaluation (HBLUP). In this study, two traits with diverse heritabilities [tree height (HT) and wood density (WD)] were assessed at various levels of family genotyping efforts (0, 25, 50, 75, and 100%) from a population of white spruce (&lt;i&gt;Picea glauca&lt;/i&gt;) consisting of 1694 trees from 214 open-pollinated families, representing 43 provenances in Québec, Canada. The results revealed that HBLUP bivariate analysis is effective in reducing the known bias in heritability estimates of open-pollinated populations, as it exposes hidden relatedness, potential pedigree errors, and inbreeding. The addition of genomic information in the analysis considerably improved the accuracy in breeding value estimates by accounting for both Mendelian sampling and historical coancestry that were not captured by the contemporary pedigree alone. Increasing family genotyping efforts were associated with continuous improvement in model fit, precision of genetic parameters, and breeding value accuracy. Yet, improvements were observed even at minimal genotyping effort, indicating that even modest genotyping effort is effective in improving genetic evaluation. The combined utilization of both pedigree and genomic information may be a cost-effective approach to increase the accuracy of breeding values in forest tree breeding programs where shallow pedigrees and large testing populations are the norm.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/03/01" />
<meta name="citation_mjid" content="ggg;7/3/935" />
<meta name="citation_id" content="7/3/935" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/3/935" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/3/935.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/3/935.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/3/935.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.037895" />
<meta name="citation_pmid" content="28122953" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="935" />
<meta name="citation_lastpage" content="942" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Blaise Ratcliffe" />
<meta name="citation_author_institution" content="Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, British Columbia V6T 1Z4, Canada" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4469-2929" />
<meta name="citation_author" content="Omnia Gamal El-Dien" />
<meta name="citation_author_institution" content="Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, British Columbia V6T 1Z4, Canada" />
<meta name="citation_author_institution" content="Pharmacognosy Department, Faculty of Pharmacy, Alexandria University, Alexandria, 21521, Egypt" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9112-8522" />
<meta name="citation_author" content="Eduardo P. Cappa" />
<meta name="citation_author_institution" content="Instituto Nacional de Tecnología Agropecuaria (INTA), Instituto de Recursos Biológicos, Las Cabañas y De Los Reseros s/n, 1686 Hurlingham, Buenos Aires, Argentina" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6234-2263" />
<meta name="citation_author" content="Ilga Porth" />
<meta name="citation_author_institution" content="Départment des Sciences du Bois et de la Forêt, Faculté de Foresterie, de Géographie et Géomatique, Université Laval Québec, G1V 0A6, Canada" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9344-6348" />
<meta name="citation_author" content="Jaroslav Klápště" />
<meta name="citation_author_institution" content="Department of Genetics and Physiology of Forest Trees, Faculty of Forestry and Wood Sciences, Czech University of Life Sciences Prague, 165 21 Praha 6, Czech Republic" />
<meta name="citation_author_institution" content="Scion (New Zealand Forest Research Institute Ltd.), Whakarewarewa, Rotorua 3046, New Zealand" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5504-3735" />
<meta name="citation_author" content="Charles Chen" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, Oklahoma State University, Stillwater, Oklahoma 74078-3035" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2203-0433" />
<meta name="citation_author" content="Yousry A. El-Kassaby" />
<meta name="citation_author_institution" content="Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, British Columbia V6T 1Z4, Canada" />
<meta name="citation_author_email" content="y.el-kassaby@ubc.ca" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4887-8977" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=I. Aguilar;citation_author=I. Misztal;citation_author=D. L. Johnson;citation_author=A. Legarra;citation_author=S. Tsuruta;citation_title=Hot topic: a unified approach to utilize phenotypic, full pedigree, and genomic information for genetic evaluation of Holstein final score.;citation_pages=743-752;citation_volume=93;citation_year=2010;citation_issue=2;citation_pmid=20105546;citation_doi=10.3168/jds.2009-2730" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=G. R. Askew;citation_author=Y. A. El-Kassaby;citation_title=Estimation of relationship coefficients among progeny derived from wind-pollinated orchard seeds.;citation_pages=267-272;citation_volume=88;citation_year=1994;citation_pmid=24185937" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. Beaulieu;citation_author=T. Doerksen;citation_author=S. Clément;citation_author=J. MacKay;citation_author=J. Bousquet;citation_title=Accuracy of genomic selection models in a large population of open-pollinated families in white spruce.;citation_pages=343-352;citation_volume=113;citation_year=2014;citation_pmid=24781808;citation_doi=10.1038/hdy.2014.36" />
<meta name="citation_reference" content="citation_journal_title=N. Z. J. For. Sci.;citation_author=R. D. Burdon;citation_author=C. J. A. Shelbourne;citation_title=Breeding populations for recurrent selection: conflicts and possible solutions.;citation_pages=174-193;citation_volume=1;citation_year=1971" />
<meta name="citation_reference" content="citation_title=ASReml-R Reference Manual.;citation_author=D. G. Butler;citation_author=B. R. Cullis;citation_author=A. R. Gilmour;citation_author=B. J. Gogel;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. P. L. Calus;citation_author=T. H. E. Meuwissen;citation_author=A. P. W. De Roos;citation_author=R. F. Veerkamp;citation_title=Accuracy of Genomic Selection Using Different Methods to Define Haplotypes;citation_pages=553-561;citation_volume=178;citation_year=2008;citation_issue=1;citation_pmid=18202394;citation_doi=10.1534/genetics.107.080838" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=E. P. Cappa;citation_author=J. Klápště;citation_author=M. N. Garcia;citation_author=P. V. Villalba;citation_author=S. N. M. Poltri;citation_title=SSRs, SNPs and DArTs comparison on estimation of relatedness and genetic parameters’ precision from a small half-sib sample population of Eucalyptus grandis.;citation_pages=1-19;citation_volume=36;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=O. F. Christensen;citation_author=M. S. Lund;citation_title=Genomic prediction when some animals are not genotyped.;citation_pages=2-2;citation_volume=42;citation_year=2010;citation_pmid=20105297;citation_doi=10.1186/1297-9686-42-2" />
<meta name="citation_reference" content="citation_journal_abbrev=Animal;citation_author=O. F. Christensen;citation_author=P. Madsen;citation_author=B. Nielsen;citation_author=T. Ostersen;citation_author=G. Su;citation_title=Single-step methods for genomic evaluation in pigs.;citation_pages=1565-1571;citation_volume=6;citation_year=2012;citation_issue=10;citation_pmid=22717310;citation_doi=10.1017/S1751731112000742" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=T. K. Doerksen;citation_author=C. M. Herbinger;citation_title=Impact of reconstructed pedigrees on progeny-test breeding values in red spruce.;citation_pages=591-600;citation_volume=6;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Interbull Bull.;citation_author=V. Ducrocq;citation_author=C. Patry;citation_title=Combining genomic and classical information in national BLUP evaluation to reduce bias due to genomic pre-selection.;citation_pages=33-36;citation_volume=41;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Can. J. For. Res.;citation_author=G. W. Dutkowski;citation_author=J. C. e Silva;citation_author=A. R. Gilmour;citation_author=G. A. Lopez;citation_title=Spatial analysis methods for forest genetic trials.;citation_pages=2201-2214;citation_volume=32;citation_year=2002;citation_doi=10.1139/x02-111" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=Y. A. El-Kassaby;citation_author=M. Lstibůrek;citation_title=Breeding without breeding.;citation_pages=111-120;citation_volume=91;citation_year=2009;citation_issue=2;citation_pmid=19393127;citation_doi=10.1017/S001667230900007X" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Y. A. El-Kassaby;citation_author=E. P. Cappa;citation_author=C. Liewlaksaneeyanawin;citation_author=J. Klápště;citation_author=M. Lstibŭrek;citation_title=Breeding without breeding: is a complete pedigree necessary for efficient breeding?;citation_pages=e25737-e25737;citation_volume=6;citation_year=2011;citation_issue=10;citation_pmid=21991342;citation_doi=10.1371/journal.pone.0025737" />
<meta name="citation_reference" content="citation_journal_title=New For.;citation_author=Y. A. El-Kassaby;citation_author=J. Klápště;citation_author=R. D. Guy;citation_title=Breeding without breeding: selection using the genomic best linear unbiased predictor method (GBLUP).;citation_pages=631-637;citation_volume=43;citation_year=2012;citation_doi=10.1007/s11056-012-9338-4" />
<meta name="citation_reference" content="citation_title=Introduction to Quantitative Genetics.;citation_author=D. S. Falconer;citation_author=T. F. Mackay;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=S. Forni;citation_author=I. Aguilar;citation_author=I. Misztal;citation_title=Different genomic relationship matrices for single-step analysis using phenotypic, pedigree and genomic information.;citation_pages=1-1;citation_volume=43;citation_year=2011;citation_pmid=21208445;citation_doi=10.1186/1297-9686-43-1" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=F. D. Frentiu;citation_author=S. M. Clegg;citation_author=J. Chittock;citation_author=T. Burke;citation_author=M. W. Blows;citation_title=Pedigree-free animal models: the relatedness matrix reloaded;citation_pages=639-647;citation_volume=275;citation_year=2008;citation_issue=1635;citation_pmid=18211872;citation_doi=10.1098/rspb.2007.1032" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=O. El-Dien;citation_author=B. Ratcliffe;citation_author=J. Klápště;citation_author=I. Porth;citation_author=C. Chen;citation_title=Implementation of the realized genomic relationship matrix to open-pollinated white spruce family testing for disentangling additive from nonadditive genetic effects.;citation_pages=743-753;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=D. Grattapaglia;citation_author=M. D. Resende;citation_title=Genomic selection in forest tree breeding.;citation_pages=241-255;citation_volume=7;citation_year=2011;citation_doi=10.1007/s11295-010-0328-4" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=D. J. Garrick;citation_title=Genomic BLUP decoded: a look into the black box of genomic prediction.;citation_volume=194;citation_year=2013;citation_pmid=23640517;citation_doi=10.1534/genetics.113.152207" />
<meta name="citation_reference" content="citation_title=Applications of Linear Models in Animal Breeding.;citation_author=C. R. Henderson;citation_year=1984" />
<meta name="citation_reference" content="citation_journal_title=For. Sci.;citation_author=D. A. Huber;citation_author=T. L. White;citation_author=G. R. Hodge;citation_title=The efficiency of half-sib, half-diallel and circular mating designs in the estimation of genetic parameters in forestry: a simulation.;citation_pages=757-776;citation_volume=38;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Silvae Genet.;citation_author=K. J. S. Jayawickrama;citation_author=M. J. Carson;citation_title=A breeding strategy for the New Zealand radiata pine breeding cooperative.;citation_pages=82-89;citation_volume=49;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=S. T. Kalinowski;citation_author=M. L. Taper;citation_author=T. C. Marshall;citation_title=Revising how the computer program CERVUS accommodates genotyping error increases success in paternity assignment.;citation_pages=1099-1106;citation_volume=16;citation_year=2007;citation_issue=5;citation_pmid=17305863;citation_doi=10.1111/j.1365-294X.2007.03089.x" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=J. Klápště;citation_author=M. Lstibŭrek;citation_author=Y. A. El-Kassaby;citation_title=Estimates of genetic parameters and breeding values from western larch open-pollinated families using marker-based relationship.;citation_pages=241-249;citation_volume=10;citation_year=2014;citation_doi=10.1007/s11295-013-0673-1" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=J. Koreckỳ;citation_author=J. Klápště;citation_author=M. Lstibŭrek;citation_author=J. Kobliha;citation_author=C. D. Nelson;citation_title=Comparison of genetic parameters from marker-based relationship, sibship, and combined models in Scots pine multi-site open-pollinated tests.;citation_pages=1227-1235;citation_volume=9;citation_year=2013;citation_doi=10.1007/s11295-013-0630-z" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=A. Legarra;citation_author=I. Aguilar;citation_author=I. Misztal;citation_title=A relationship matrix including full pedigree and genomic information;citation_pages=4656-4663;citation_volume=92;citation_year=2009;citation_issue=9;citation_pmid=19700729;citation_doi=10.3168/jds.2009-2061" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=H. Liu;citation_author=A. C. Sørensen;citation_author=T. H. E. Meuwissen;citation_author=P. Berg;citation_title=Allele frequency changes due to hitch-hiking in genomic selection programs.;citation_pages=8;citation_volume=46;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=T. H. E. Meuwissen;citation_author=T. Luan;citation_author=J. A. Woolliams;citation_title=The unified approach to the use of genomic and pedigree information in genomic evaluations revisited.;citation_pages=429-439;citation_volume=128;citation_year=2011;citation_issue=6;citation_pmid=22059576;citation_doi=10.1111/j.1439-0388.2011.00966.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=I. Misztal;citation_author=A. Legarra;citation_author=I. Aguilar;citation_title=Computing procedures for genetic evaluation including phenotypic, full pedigree, and genomic information;citation_pages=4648-4655;citation_volume=92;citation_year=2009;citation_issue=9;citation_pmid=19700728;citation_doi=10.3168/jds.2009-2064" />
<meta name="citation_reference" content="citation_title=Tree Breeding: Principles and Strategies.;citation_author=G. Namkoong;citation_author=H. C. Kang;citation_author=J. S. Brouard;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=J. Ødegård;citation_author=T. H. E. Meuwissen;citation_title=Estimation of heritability from limited family data using genome-wide identity-by-descent sharing.;citation_pages=16-16;citation_volume=44;citation_year=2012;citation_pmid=22568689;citation_doi=10.1186/1297-9686-44-16" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. A. Oliehoek;citation_author=J. J. Windig;citation_author=J. A. Van Arendonk;citation_author=P. Bijma;citation_title=Estimating relatedness between individuals in general populations with a focus on their use in conservation programs.;citation_volume=173;citation_year=2006;citation_pmid=16510792;citation_doi=10.1534/genetics.105.049940" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=N. Pavy;citation_author=F. Gagnon;citation_author=P. Rigault;citation_author=S. Blais;citation_author=A. Deschênes;citation_title=Development of high-density SNP genotyping arrays for white spruce (Picea glauca) and transferability to subtropical and nordic congeners.;citation_pages=324-336;citation_volume=13;citation_year=2013;citation_pmid=23351128;citation_doi=10.1111/1755-0998.12062" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=I. Porth;citation_author=J. Klápště;citation_author=O. Skyba;citation_author=B. S. Lai;citation_author=A. Geraldes;citation_title=Populus trichocarpa cell wall chemistry and ultrastructure trait variation, genetic control and genetic correlations.;citation_pages=777-790;citation_volume=197;citation_year=2013;citation_pmid=23278123;citation_doi=10.1111/nph.12014" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. E. Powell;citation_author=P. M. Visscher;citation_author=M. E. Goddard;citation_title=Reconciling the analysis of IBD and IBS in complex trait studies.;citation_pages=800-805;citation_volume=11;citation_year=2010;citation_issue=11;citation_pmid=20877324;citation_doi=10.1038/nrg2865" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=P. Rigault;citation_author=B. Boyle;citation_author=P. Lepage;citation_author=J. E. Cooke;citation_author=J. Bousquet;citation_title=A white spruce gene catalog for conifer genome analyses.;citation_volume=157;citation_year=2011;citation_pmid=21730200;citation_doi=10.1104/pp.111.179663" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=K. Ritland;citation_title=Estimators for pairwise relatedness and individual inbreeding coefficients.;citation_pages=175-185;citation_volume=67;citation_year=1996;citation_doi=10.1017/S0016672300033620" />
<meta name="citation_reference" content="citation_journal_title=Silvae Genet.;citation_author=A. E. Squillace;citation_title=Average genetic correlations among offspring from open-pollinated forest trees.;citation_pages=149-156;citation_volume=23;citation_year=1974" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. M. Visscher;citation_author=S. E. Medland;citation_author=M. A. Ferreira;citation_author=K. I. Morley;citation_author=G. Zhu;citation_title=Assumption-free estimation of heritability from genome-wide identity-by-descent sharing between full siblings.;citation_pages=e41-e41;citation_volume=2;citation_year=2006;citation_issue=3;citation_pmid=16565746;citation_doi=10.1371/journal.pgen.0020041" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=Z. G. Vitezica;citation_author=I. Aguilar;citation_author=I. Misztal;citation_author=A. Legarra;citation_title=Bias in genomic predictions for populations under selection.;citation_pages=357-366;citation_volume=93;citation_year=2011;citation_issue=5;citation_pmid=21767459;citation_doi=10.1017/S001667231100022X" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Wang;citation_title=Sibship Reconstruction From Genetic Data With Typing Errors;citation_pages=1963-1979;citation_volume=166;citation_year=2004;citation_issue=4;citation_pmid=15126412;citation_doi=10.1534/genetics.166.4.1963" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Single-Step BLUP with Varying Genotyping Effort in Open-Pollinated Picea glauca" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3.cover-source.jpg" />
<meta name="twitter:description" content="Maximization of genetic gain in forest tree breeding programs is contingent on the accuracy of the predicted breeding values and precision of the estimated genetic parameters. We investigated the effect of the combined use of contemporary pedigree information and genomic relatedness estimates on the accuracy of predicted breeding values and precision of estimated genetic parameters, as well as rankings of selection candidates, using single-step genomic evaluation (HBLUP). In this study, two traits with diverse heritabilities [tree height (HT) and wood density (WD)] were assessed at various levels of family genotyping efforts (0, 25, 50, 75, and 100%) from a population of white spruce ( Picea glauca ) consisting of 1694 trees from 214 open-pollinated families, representing 43 provenances in Québec, Canada. The results revealed that HBLUP bivariate analysis is effective in reducing the known bias in heritability estimates of open-pollinated populations, as it exposes hidden relatedness, potential pedigree errors, and inbreeding. The addition of genomic information in the analysis considerably improved the accuracy in breeding value estimates by accounting for both Mendelian sampling and historical coancestry that were not captured by the contemporary pedigree alone. Increasing family genotyping efforts were associated with continuous improvement in model fit, precision of genetic parameters, and breeding value accuracy. Yet, improvements were observed even at minimal genotyping effort, indicating that even modest genotyping effort is effective in improving genetic evaluation. The combined utilization of both pedigree and genomic information may be a cost-effective approach to increase the accuracy of breeding values in forest tree breeding programs where shallow pedigrees and large testing populations are the norm." />
<meta name="og-title" property="og:title" content="Single-Step BLUP with Varying Genotyping Effort in Open-Pollinated Picea glauca" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/3/935" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Maximization of genetic gain in forest tree breeding programs is contingent on the accuracy of the predicted breeding values and precision of the estimated genetic parameters. We investigated the effect of the combined use of contemporary pedigree information and genomic relatedness estimates on the accuracy of predicted breeding values and precision of estimated genetic parameters, as well as rankings of selection candidates, using single-step genomic evaluation (HBLUP). In this study, two traits with diverse heritabilities [tree height (HT) and wood density (WD)] were assessed at various levels of family genotyping efforts (0, 25, 50, 75, and 100%) from a population of white spruce ( Picea glauca ) consisting of 1694 trees from 214 open-pollinated families, representing 43 provenances in Québec, Canada. The results revealed that HBLUP bivariate analysis is effective in reducing the known bias in heritability estimates of open-pollinated populations, as it exposes hidden relatedness, potential pedigree errors, and inbreeding. The addition of genomic information in the analysis considerably improved the accuracy in breeding value estimates by accounting for both Mendelian sampling and historical coancestry that were not captured by the contemporary pedigree alone. Increasing family genotyping efforts were associated with continuous improvement in model fit, precision of genetic parameters, and breeding value accuracy. Yet, improvements were observed even at minimal genotyping effort, indicating that even modest genotyping effort is effective in improving genetic evaluation. The combined utilization of both pedigree and genomic information may be a cost-effective approach to increase the accuracy of breeding values in forest tree breeding programs where shallow pedigrees and large testing populations are the norm." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3.cover-source.jpg" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.g3journal.org/content/7/3/935" />
    <title>Single-Step BLUP with Varying Genotyping Effort in Open-Pollinated Picea glauca | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"pTgqbpJVh_fI-JDYwIrTsiZv7hB1XMGkCPY1Iy0tGLw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"483525","apath":"\/ggg\/7\/3\/935.atom","pisa":"ggg;7\/3\/935","ac":{"\/ggg\/7\/3\/935.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/3\/935.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/3\/935"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/3\/935"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/3\/935":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PTzimDUe69hLj26jYJ40PktlFFoNk5mBcJK2PzqkFjaxojHkuvSb9txfClFeixcwqb2i6PmthfA4MmR5KQ\\\\\\\/wYU0BhS+fX1PBPR4YSM710pterOVmWNuAO3V8vqHcahv7ih4yhgEhKjsF3R5ALQ28MTn8HB4gUIc+ORHGXVUq3zH\\\\\\\/6u2PB+7Izo4lUixQC6Cu2L2jvkW4i2AiAYMJm1gkbtTeyPO7UVWDIsR9pBkNkQ2KopN3Qc1rUURyNL2hvPrCn1wLZDpoe2O+AtrkH8\\\\\\\/KMZxIfGRaiH7uviHpYkOeGdHLGIXcYrAa3KU\\\\\\\/LoNqVYEsqdQTdY+4Tf1S9+3bma4CgQWih34pFIBi4NX60Oa9\\\\\\\/aW+YekapfEiIXDlEFyCS\\\\\\\/pOp1h7OI5UPDTGw9Iq0VMG8UW+i5PCIPGhtUbA6XiQZfMzyio6v4IYS6TFj58Ygq1p2N\\\\\\\/dHfB1rQ1mQnsKbmZmCh244vUXyo0JfVkpqIJi0T7Eod0m6JNVO48+okJSv8xVUxw9hj1xWM14eZ2rlu68lVwoRdKAJBrSu1uaxMKq7LcGu7Vm6ITHIhdekmMJsWJ+ZA43nMZZ1gqS4Zu4A\\\\\\\/jPrr+ZQr6xYP9JAxjMXDEqDWWudXJLrPPtXdnmUgLaQNFLUxXvQhi\\\\\\\/9tXpL9h7R72HoWmhlQ4DtO2jCfdxkdBc\\\\\\\/D+THq8SUMSuTp+7NgskZ01Q0AYw+Ky0V45SzXkMoDYcFQaDCkE6\\\\\\\/jlskdSfLNET5aDVeq2DHVt5YlEWyq88o7ELEtYKDYRaIgfuDBEZ+ukepFBdHvE3lsYDJ54ks2nXJFW5bwIMqA2Pd9LiFWz5fYX3Le1c6uA5dEGGNQnk1HLNtDuqbj1p1UXWsTcRlc\\\\\\\/WEXlelIVrI9sbTiX0f5bj2a6NjmFHFJ7FMgnjo22XCzY9k8Et6Md5M917drY5ZORCwbxIc5PM6zqWfXGU4h7y3KVui8m0t4GY\\\\\\\/TejMtigXNEqS+yQrmwDLyqGRxDfFfO6e2Qc\\\\\\\/ykGzpd1lDz0qj7B3RR63TCnOIW+xK3dvQoQILog6xOH4Fu9yBYi4Mu8yKmuHOgVq9adS5WmMCgkLjOi0WT9OjQYhVvRrkVNsAeEeQmLpPcaz5ZxWt3uhbVMtVMSTQnF9YC32jKwvw0FQAf04Z\\\\\\\/9rp2JD1DNRux0E0MdckAOrrIL6eKjTgYjY6C3fg24shU0ICEsx+bqKQe9n3zQ5JuKACqXoSsvAOToPsE9f1HGNNemWQh7XRp9vPF8V1VrMH3URbxJfgtzzd0bhMTcT6pD\\\u0022,\\\u0022iv\\\u0022:\\\u0022XglZK6gPhMqiTBBRYG7Gqw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002286afb0b83f1cf93f4c7ad641db12d64443674ab235821d7daf71f1715ae591d1\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Xenojb+inZNZTVF76MvWUBQuaRCHtcaikLhtu01BNYeMdfSfa+jdyREqabt0dks8XFNtn7OxXKMCbphok76djS9KVDfFWmQe89IkqgxRqYSn2XguZii9veESfN+0G8Az+kzlPPrtQ3jfq2c9z7UcghpWZOsfyx8ui3Ga30\\\\\\\/zBWw1BRBgUr4XDC+ZM+wBS5kVfovJQURvsmeNWtyVtjF1Jb\\\\\\\/9SdNyjstN7FJK7tHegAprtCruwyb6mp3KuQUisJMveWLo3B4JDTfxJXhsrc1HPmJnhYOqgmgqiaHqnmkwji1OiUJdqf6WNQwHBbpy86R5c+hqQSedx\\\\\\\/wyYBx3ulkiRUpf3wEEqCIkvoF+SY8hLZCrQwgOvvj7VPStmjeESgvvTTQv9Utw2JE\\\\\\\/1SiGwttqFXQwHaTDwJzzp\\\\\\\/C0IVDI5aN8CIY0icz0tC8D9Zhn6XPmI6wp01edAZ1BXmz4sbvfgYCAjoR+Tte+FugvwffOg10Ol5xnPCeL232oCBj9LaN5NXw43HPE1aEqN+TkD9S00VNQiB6WZfKElbIsB+A8uR\\\\\\\/4j+d16EJZjWgLzeF6hTN5bPMooqgfU2W9xQM7TimesZn3mtJpxv87HGjf3zoVHk2Ue4qK7dyOlXAubQGvdw71Xl1cPZZldfBRwFgpQJ+JjZF7HvoqF71SwaOGL\\\\\\\/ylJ6zrmti+B\\\\\\\/s8AVtOCYwbxz9U3hhSydFWYDTol3xJ4\\\\\\\/8MjYUHuBsrIp\\\\\\\/pcEWFv4VCt7w\\\\\\\/EXzncWPDagnYu9ruL1KEOeQXsDOQ0uHFU\\\\\\\/newWntEZDFPwgwz8LbBqVaj0ra\\\\\\\/TOl9qaUIIcI9l1WgWNQAtstDLopUeV\\\\\\\/8YmFIXPzKY8Acsgxhce1NcPCa9oU4Uyl0HJ8fd9XRV8HxFOnEePHiG6fcpwWv+UtrHOghUXeZLtzB1FSpW0OqGHpc9l0A0bWN+19dxh6X44dOWN2B1htFpaJaxO1KkrFXnHOhjHYyczYJfK9HnqNYK0Z+GFVCbJPcCcCujFOB6MTv8M63ipTbuvMH2wMMC9FX9TATWM\\\\\\\/zzNjlUsCiUX2xKDrVTnV+vzz8zqmPw\\\\\\\/RnQLRcUUayOS\\\\\\\/4n3qnWNCFJn0TZ3P+3\\\\\\\/qOglzPm8erCy+KH2SV5rZIrVWvpeAjI0m1QAwjpIDOluIXzUhNl9U8KolgkIeq3ZC2R95shtEBiwDS012ef1OI6syBYPOTUfRJM2Dnp\\\\\\\/BslB6oVRGZfWKcnjBD9crT8cX4uNrjecqUgM4bAirT6bzf0pW\\\\\\\/ba225GD1H6VFf4PkOutQowQhnzjr90yDTU\\\\\\\/\\\\\\\/MTtMiRWo85Cysz7XDDBzfedHUrWk9K8hGWSOBCs38GTydV4l7mfWYBZDWymZzI9WAcScnh+BdBreQLmaw9PhyfUuMGGFxWBXcSnEYJj0kynNCQWQm2w6S+bj\\\\\\\/xtS8nnhgyHDmTNIzORqkCpOaEsVjc3m2n1K7SYVBVoh4CcMi+enjiR0eHsk2DGd17VAF+ml8UpHR7Cy4un08d2r0tQi6K9uzuGTw7pCxFAHGMESCkq7q9XySN7Z6wEdw11jQlqh4f7vx7nDFGRPn3nqiVbIh1BdG+7x9Rtu6pS1PemxkbUQYKZ0vMp56yaU6CvHKRGNvUkOuATAd4ogf4QX0uQVYYf2+rVyKPr1Qsfd124lXS+sWefvlDbu3wBXLjwQtuWFQNlcFMIJ5vCzDtvvTbtz9hkggJhixIFviIlefVFl1I35yKrtmNhOy4xhLTM2lsRRhFYht3GYBS\\\\\\\/OGbBpNSSy\\\\\\\/W6BEVDYUstt0ck7Xf8bpk6DE61jeD3Taag3EySKjPaHTwvMdjk1Vql7GBFkR81qPg2FFo3sSowvBfZGuRwQ4slhUYGtSQbq4xkWzlcP6LRat1Mlt181bDMLVqUIwpj0s8v6gVZ+nmk\\\\\\\/Gu6uZ6x9iK0DMUtf0N5\\\\\\\/mOFbvWfa2v4hi89a5\\\\\\\/4NDDYR0r2UoFU00p4+HOyv310XpVDreapIVoi+RjTWRxfU7RSz\\\\\\\/8Ew2uTIm0kPTC6W83hI1L+i4xE0XSH3xccjLgJ+QhrYH3f\\\\\\\/y1Kd5tea8GVB283VlW+QYdrQOrFezeHhfH08cjWhZOjSbbCYPz8MH7vC6DJw0AnJJhSXkf0L0JfMMtkz5HOqsuz8Q1jpWE9K32Jes1BIXGXyOKTyPlTdBlnHJFaPMdSg+cjSjZx1pj4+jsp1C3bI0mBJ0anPRbABafkEe5lwv4wCsyznyHz821SqG17PHJ92sefDUO6EEUjZd5orWykT9h2VT9wy0cRfiOxKk7ESrq6n3bpgII43IepTx9uhLXvENN+3AEy7BTqGMwRn1wOG\\\\\\\/SE3IbWODOa4JQ5qRvbqHlY1c933RUwFRR1BNMR+\\\\\\\/jqWNkTaslFMXCsp3tJLikfX0CAmTPOBmaEQ+va5RowUPs4pMAr5xa+O\\\\\\\/2LoDhHAC\\\\\\\/GBUR7w3becoEngrQ05cYseoSHowcghaic9Q+5LmKohetFX4s+NxiuqKi8q6rC4D0h5RD8dWBgB0tTb7XOujN44VQVz97Agsy7dgAlXcf1iybgz32aH1AmQ607lhOH5LhaxJUonQ+7ySA2tq6Sn\\\\\\\/MdLPvl8r\\\\\\\/wEQeLN0wv5f19FsJNPJ7ZCiiyi\\\\\\\/oziGcS2NwMuPlUId\\\\\\\/gCB9fJKioOlMWTiof3u4pehKKR0MC\\\\\\\/5ZaUOHqT5hVgMoGIQcdj5efKalinADbUk9wVZ9IJ8LBr1wGZeYdOCD\\\\\\\/G7Q6zmSVkrY21hNkG4+wFDIm5GBFRyHe8ZkgT0Y\\\\\\\/wp4\\\\\\\/gn7xIF4zwWusux1Qqhl1LPNO1q9oggWI4FIB9yo8E5G5f9\\\\\\\/f0Xet9CITtxTXSCMNgVUhtCE0+eRcf2VTWEtD3gIvvbLXTSXIz4ZwLJkQoCTkZx84JctEBak0DsI\\\\\\\/lh83eJKhWmrMIRNe4kWNK6biP0OqJONhfBsvlWSTdfHHU7Psi4+Q0Y0c5sSwXskhXRxisQeVtH1P57ODBY0QRaaetuHU357fzfFrAlv9Af\\\\\\\/3xqC2AcwipN5G9EXPd04h1wQcyfXNOzrWvM7yz1sBmCAZU0x+IqHYurWKQbxzpaXVUpRkERKGpwh5Ufsq68GJCKxbhYgPUhRi+b4DpdE2mJbwpxEthjTObAU8FmSvLBqY7DD1GmzQvpJ87fN5FdJKWW3RaDi3twawKg+ni9tDRzOLXSyg2AklVgegcp4eRRu27rwSLuXNSX9eG8ozgGTv0WzCVXWKZfHuODdcd2UdVVqA7+hnPakf84VRYVnjck6bAjnSyu6IM3ZbuOrfubBlOlGtS+IvPg\\\\\\\/V0dCEulIM\\\\\\\/+A62+kB25fX9v\\\\\\\/3WIcSfvH1QCU30txGFF4zdmt79aR5B1zQSKK0GNGKrozFxO04XEj3JE6ueAyuc+v1TN2XpzLlZCqbydc0\\\\\\\/DmK0I6v74YZBYbOLpCrVDAmzBhmuuvzQ4c6yDH2kPepYtjK8vYNZbX0ICeY9NCXmEkiS2A0uEuPAhdfl\\\\\\\/FQSXlzie9UYRdhy2NrK9EvOOWzlDzvugQaS0KmiOGLwarimaWyGrsnq4czWQGLIIL1XbzwuGRQws\\\\\\\/3SRTC\\\\\\\/3kCE3Dgim\\\\\\\/eTF\\\\\\\/wHGDPekC9C7MsDVdXfk0INQma6clnS2I45I+oAYB8FFifW7HCtnS5SpUKg4xy1o06CbzXs8B3Zk9ahQy2\\\\\\\/CWrPeR3OGXyoArPb6hXpkCmjf0MVVTr5f9mtCVqwJTh0cggpRAx4miVNlk5WeGLoY5JXiL3dYQUwZWt7PLZXa6W3Dy\\\\\\\/IB2qXhOhkDTcc\\\\\\\/Xrg+nTrVOioyU1QexGyJmtZNfBTknSwJEHLnF4I4NHuYmarYjeNz\\\\\\\/PtuqhXcnsrBiGxBzNmjvJ7oMXcO1LicTxpGymW3kEMl5IR7po6K+r8rqF1QCZoHnE0YtonHSUVbQ\\\\\\\/DlRg3rdI0uqpN7GRU32\\\\\\\/aki1chHYKlyJ\\\\\\\/nuEP8zF3yKcRXRxqBTqTcbl85K\\\\\\\/BD1ofGjgJMOxYqVqsnuJ4HKLJaBlb0tCRlEM4bvb\\\\\\\/ONlb7vRMszmJw\\\\\\\/qW5nQg2X49w+Zn44ifOArsTbvRvhFwhhIHVbBGSgKYvhEEqpKxkX2V31gSm\\\\\\\/FOEjNA\\\\\\\/gVrOImWPiY0xmUZRsJm05kfxriZcmCq4Hm4weLWR27k3jWxrLswQ3pZuwKcDUF9SFnQZlLDqGu+2+UjLM5p6mp\\\\\\\/cZ8UPfiTcFfG\\\\\\\/jEo7hVlZ5wXUNLisWOKrKNjn5aPSPJ2YH1Eq5PmqRUp13r1107d4LuJf0GBiUVduNt8ZMh0yJpnhEjVJOh5tSnzqnhyBXia4dlf+uCkBMh5DjkkAX7kKUAFqlAcYobgBJ1S8FWEgWfKD4C98o4DBzMrkN+UAaGSOMfyvZBHLKDpzrU0f42rtbOuCgusbgWmxkZvm0eQ1qJcVdZ+rPPeQWTi7TWzKVOClQjMILab+r17Q6pEVeZa9HYuOkIrEJQM6gL4rSmWjg\\\\\\\/kLCjrV0tcELWPEPPeXEY7z4c1+xPPQnzwF07x4DwZbOfVUNy6kfUjkhIo9Sku0OACRPg7IYj63m\\\\\\\/89W7U+KURlTuHX6jupGtftfDcGIKo9F\\\\\\\/OURoPAhRleSGIqtwdCokgJsXHWFCC4eUrKQ\\\\\\\/eTlysRmQKEItyFONQYWAlfai9MVwyIsZ5\\\\\\\/XsMgg1cbxo\\\\\\\/iXX+4muUqohpkYCjy\\\\\\\/80YfY6dKUvcMv5F7uoFeKvSqVH52TNkjgXvH57eZdLSZN992tdieKj549ByDCbyyp3iwS3kWAA22ETYNI\\\\\\\/1pkl7qw88DaXXoOG0YYhiZUqlYwko7yFg3f38PZOkCm0TxUbeQQNNv9dX7FMzgVzxKNE20g9rt14k+5L0sZEk6uyAzgWqQH7gF5OBoqiQVa7N\\\\\\\/\\\\\\\/NHtpg6a93lNUcp3jOdQcmJczsmS+wUzRw7reY3avBvQUawZGzl4iSqkaMorczGBPO\\\\\\\/v7H28Zh3xk2fb0p36EJp0MR5LdWWHGdm4Y9MMqpqj\\\\\\\/ikwqH55v3rDy77qk4LCJA6c27UuIQgxQj4nFpUrQVhifccFLVUmvXGOkBcvT0i97YW6j6gzUpAKS6SREt\\\\\\\/WgYB3LIcQ3ed2qxphwytd+Yx1HuFT\\\\\\\/4dG2AiIGwHxyrW3bnigf7WM1y\\\\\\\/NzhYsc1m12QuhsRbaWTjFalTOeoW1FTQnXeu2gZVC42C3czIGwJ4G9pj2IpuMl6U1HPltrptgOIZzsf0kOa8ygmLTrCsAmznFiA1C6LYugXMuEpe8FMmAeeVP\\\\\\\/O3dpAtZQqH2Xfx5S8ttaV8mgUjoHvaO5u61q5fT6fbHimRtjnu7NyU90Of9SctYETkiaYA\\\\\\\/R78dpvaLQtEycLX1DkVLKmh0Jn7od4DSZk83ITlC1AoQQnGn9IQs7bpPum1IJPH0Nw\\\\\\\/tUIORozxQhBuZzvllIyrWNvgOI6QipbyEbA3+DZH\\\\\\\/5ujJlpmZQs0v6pKonaeYLXixy0F\\\\\\\/2Yiy4fq9YdBGNieVffXJtmUI1oDlytq6pZVeADseUT1HzmGb6VrN7AxMXuH8jV5ZNUcg7x0uLN2swkrlLZNcQaZdkiH6t80jweKaEyPvFFpbeBRfx8mZ0d90d\\\\\\\/BFVE\\\\\\\/nWziOFO6LOow6QsZ9VRePehiqDtIiZDlEBnnbaarZbyfuyOjnTudmOpguddWnX0\\\\\\\/d\\\\\\\/RDdgoD57ciL2wcjXLX81cDVGcrGcOMn8SmatTfV7OqANmaq38GsLvcwV6h2t+VslEX1T2061X4A946Ln8bYO7cfX8LPXX1nv1GKGHEBsTjJfAlZ\\\\\\\/D4VAOQ32jSDc3mTYruIJ2HphUwFG5i7Dad4DIjpE4P2fn6b6UsTCkt5lZft9t81SF9w9GkO9ZIiQtlZB8nwjQeQkGGNrdGTBx8JlhZTXVL3UuNQT9nMcgNdnFxnDEREYINmbAzY\\\\\\\/OPwAH190tw+FiBNxN51n5cVo36s5FiY0Z6ApyCF4zGekHKpnT01Rv2OmVDziPVYPpaOteFcJVYPSWEWQnApkasPhxa4PKfAwRIj\\\\\\\/3Qy9Ez6wylCo1fyLpwT1PwUSWNmAYL4bNkgmTcK4uea0JQNiYA06yORL24pc6nOLceb+7ke\\\\\\\/BAJnZxXg0bHMlKziIlXhGdigd1fGY56F4Wu9DLcOJcOxZSpN6+DAzKpkqGSl7K9aOko3Y+OwLNhTLpKfLojcKlQKUnmyMmHbQLiTpNKeNbOKb\\\\\\\/XHFM8yj+tElvRLlfFDjsyi502Af+fvXlIFHBlFBAXYt3wLwvUvpZVlM+7Pp4GS\\\\\\\/QPlsnO1yqbq8Fb2zGD5B+fbpACDdIU+dIlLAhWtTIUPwapdQzb1Almk5XouqRdDz5GCP1Bqw9xqnMMPN96VHvY1gpQ+OCmTzFLLuT1XJc2geQNrty39dFn6Z82JM+ibyo01kp9GNbx0IZW4SWEbj\\\\\\\/uM5pwTC\\\\\\\/fk70KNC8cr31TU6lF3zSI7zHv6CYm8N7FkmMY4GMQLl\\\\\\\/WUNkO6nDwIfMAIXNH3MxDVEj9ESYjBdzRkg7iMuv1A2gTfc2RpY1Vfd+EXxf4ZENKhGCQiseRbLPmls6fqvE\\\\\\\/m2ZWs2AnD4OoguDshHGPuLbl+I0bptnyZzDqVkYIWohyB7DzWnmagjmo0Zt4JSn9YHRWrhZQpLmFGlqubv\\\\\\\/4UafBgY4enFTJt9k2Pql2x+aAJ5ZWrBnnPtbDq6zEWLSIStfP9JSo7cFk67YnrvpdP4RCzl2b2BQZZhxPgGsSdk1TPxxTH1ZEZH85RwGkeAfMeqEfs9jjjoXi2iDGcBE2XOsS3P7b2lFeys9Y16ZN5BqG4hCvhK5DRwTHahZliHiX799+NYqom6HLElUz4bSLAdwqMWS69UNIalVHj9M6CAS3vIwlWp7k6dDq6ZHI0rOMCctNJ2Qc2OFvQOtLi3Nsor3huNEH6x03X7HjIwMUo6JRLV5dvvMYx9MFYmpixo39bJNStoaFFz+wK8Qba28\\\\\\\/T\\\\\\\/1nzPAWSazoki1suqUqj+otVu67iCWTdFDXrfV8G3OHzCY4EM4CrJ3GtJ7STd\\\\\\\/PgfQKb4g8lvEUbFWUqTY0akBjGVxAZXxP6wksaqeOWuklKoBXkxrJ1gAZPrpi6Q7jyrIwWbTGE6YqxqMHdypggKy0Ce5CwYxj0yrrzr0JW4v1bxm9Hvo+1slSD6q8SR1kdd8oHXREflL4ud4imju6tg7E30Y+eLAvdXufs36X\\\\\\\/nBIKVHPJyRu+CDhiOczfbWkONkfjm5dk+ENX4KVyAP83tfyEl4hGAAZ+TdFSAiJ0o7U8l1kxlUhCEWCE3nmiNgklzPuD2fXUczRuAEnJ0ky8jSjSZ7rSXCh1cFd+OyTQCs2PVnDKsq6aFIAg2S6ub9KFwNDwTHKPFRF1TUJ5tNKFYHOhtXRmAw4uCF1DwfABSRWYDXkncqwom1N5howr2xohFmfC5v6gLHVfySRLxkh22iT\\\\\\\/6SfLH7bPSL9tEP93Y9ax+piLoJN9N2doQD6o\\\\\\\/B74ll20h3OjUEe346m9+Kxg6o3FKby4UNe8sqDHB4Rg6z9z1AWrNit3PhBZunz6rmf+vyoaqmsifOhvKxrHyNU3NpglDdoaVaFLbb8i9lpwqE6f\\\\\\\/fd8jWHq1IhwL4o+xzmpyVIm0lFALwxHZwGZHDCwM7ABhSyvaapFssLulBZ7y77hTAd\\\\\\\/IQYe7NRRFasFdN2r05a+P\\\\\\\/+U9GJzAIicGYD9p30fr51UUGLe3nkG2IbLhIhQZj9HZ8557Xd6NCJj7qpbmEM9N6IXBxInhbyQur56CgWgwRcWwyVrK8zQw8oSP3QvaFDcMnc3umL0wfo2IpUqII1V9oSdKgnSi2deigRP2c5JKVry2viOQhggmjraWcUnKkxlgXTsZfBIMXCkMVTaBJ8hcn1wBJiBpViVBUE2k1gRNVRKNRv\\\\\\\/ht0ideaTWa8uCaMCDlypbTX26GezbJSoJg8hwqblENe81sCcDkMtaAzcXgVQbTC3dr5YV1SUlAlt3rYcc1yxfuqaJhyqea9nbnYG\\\\\\\/FgG0Vib\\\\\\\/BjotRhyDpQG8HtaiHgGbD1AuN5S3S39Cd6mmbdbY3tuPPLePB78PNbeSKAXzS7UqnU9Cs6cSmRH4sHI90JofB4FD2mxoa02H0YTbCiQOVaROAsWXs8HAlTY2wt6oHfjl7q6ApjNg26mqzWMRyOFM1hGeezW11jt8kfKdMESWrnyMagIec5VvxnRfAJvY4a\\\\\\\/rZkQKxTsjFC\\\\\\\/KjB+MB93wZsZnaqHMd\\\\\\\/9EEL3\\\\\\\/IAVTLcWgwCmCa1UwChUlaZ8aWf4mX6OeRqlh8V+0JGZ+LV2D8uvaCRIEjcUsbfZGvbWDGI\\\\\\\/XfZ2GTpKWEM8hd5qeHOrR2x6ZB+MojF0BvbnzbkSWdIv3\\\\\\\/Fg37XOI\\\\\\\/8W7GaUaWHRgyfXFbmW6BcLIw45KmY5d2SpmDqgEwvh1vjkEqCblNnFRSinTCV11LfGFsxUexaDGAhwY2j8Y1cF77PCsB7zHC\\\\\\\/D05GVXfd7YFmZwmIdZEOYf\\\\\\\/jJ2CA1FMvgVbjkuT4aJc1x2K34PdWJ5hRDwxbJ3kyE1Yacl9nbJYmatgNVNQAKT+vCZ44\\\\\\\/M38hb2gwich+A4k3\\\\\\\/d+EfDgGAdNG0btYAP\\\\\\\/sDxAEuXTaGR90GPvP6SEeRaCKNl5Dop0XvDEJGCSL6WJPXO85jDAMFyitkJZbdSB5uw54vsRHPSu7eVwvxyrNRpL56Ip76Mvd5Z0TGtkQYm6UNFd0g0WOIz1+rUHQX4aNSOwOnrnoOwoR7ayKUCuI1OEamYWlsLmXUyp0y\\\\\\\/18BttQb0pS8C7iKaw3FYPMPN58AfOvbcrEZhq4Xsay2qUYMKBO\\\\\\\/\\\\\\\/BgEUiUGtG50hWM9f2YAKTpJlWIQDmvizue8TGUi+K\\\\\\\/vf9+XnLrKABS6uATgFpTY4MNoHRTEtQbLb8BECwMvaHXbMz+iq8O1XQX9HGw77niRQlGuUpapChEsSUvjrtMnAfx4EJ71vftZHHFevm9TXT\\\\\\\/Il4\\\\\\\/xPemUB2D5silpq1uKnUD57aeH0CynI0p1kOpZtkajeJKyfSkjB7oVEJloUtNRtgvBi+aFS8iXfYXaZ+gTaHVaqQtkV386Ux9hJSZiVXtXbm34dr1saz9ed9kcvbbZM8sdAFZpnnp9UALedeaUdXeQ8UvwwlsTsQ1v6bxkqJEswPJxnhF2rqc\\\\\\\/hszxO6fmTMW+vgU\\\\\\\/C1NiidLZFEfKJ4d4oJqh7CqVFyfeIg5a+78NsepL9XnlfUIv4yFMIdSK2jwMRF70FK2QJKsqhRCh35fHXxmIW+YZteGB5adruuB1dj7tkc5EFb+EVR+UBhwWt7ji7buERU1acuNQN0KGwPTZci07ggamiUEyktOCU0N8pJ1Rs5gMruO\\\\\\\/qfw+Y+yrxNDKVxEZc4uvZb\\\\\\\/DG2w0asok7F7xahn1HWiKfFwFCQ8ULmoAVVKNetMPDNwfAIzUQeHMeqRAQtkglMbuxiE6z8zzHiYssbVIvuSep4AS\\\\\\\/NLqzO2qKcqZC0D7GElV2TQ+VAt8paFwh0pSIqjYdZyWdtpAuZi5SYrT3CJSCJ+sr6l6SoiwLq\\\\\\\/kNLPq6KcBjahZxi9uPnFMwv8h+gnCFB0is9k7+CYHrO9yp8JzamBx3R3iqJAqjJBxCOK23xxH8RHGV\\\\\\\/FLE8QeKe+knsK\\\\\\\/KAhEVA8oy9lRyoCERrdkXpg2CR67xOoMG39qH87kEdKkGTeMzR67q4PIftbFCKuaKlHqNPpNkuvWKdChsXcyKhg\\\\\\\/ReA0tFoRWxN7vKbDeuX6nnUOsS1OoQn6N2W1Jz9BEQzP2H+bbvQfHkV0F4J0XPcmVYKFYHMwLlNTfOZ7tMB0vPJcSoTqN9JqSxhzLgZqP5iIXodp4\\\\\\\/0DxPufbbyw2ETtSbwWo1CYrAhnbj2Iyi00Ei+cgSrxttj9H4qLugcwrRfJxOQAxywyFLOoJK3itOvjv9qDQBuAED0dl8yzG9muadDrOeTaHmhLWRhgj0wz52IiP96m4mdiQnsilbzMCIGk9w7LHSaXW9lSD5y3cXdOlWrXdJDNbG16ZM\\\\\\\/NtvPEVomXKGPyZeBfK6J7qLy+iEhVR4UsCKh1v3p\\\\\\\/RGSwW0G7bW1cn0WxJXgmxyJQz031ISfgO2Pcuxqc\\\\\\\/ClqOnz0q3xpKfjbknn0KSQ0+KbHwZYdvnPIBAYDQEktJvwWfl5anhnxaAYw8FaH0m22akpYNAQYgXxhp2mCOgZmmcf9rmALLNnPXlZ+S0IXeVQ3Vg5NtVDkjDksX8DnE2FHz5EnJIjiwVzt5oFsKvbLCWLWs+BddcyfkBu0JCX+fKK804KzAEN5SYNyn3CMhDkuVPjnrmgxPaKMFyI1prLDCf+637D\\\\\\\/FFC7YtLu\\\\\\\/CkkNp7cfQ3gUsyAVs5CbIWJC2jKArhhiR\\\\\\\/XGw7Y3fp4oTy96U3tfWu74mQFXzcb3KGPkE+gMGvuYYeLyBAEYZ8U\\\\\\\/5E\\\\\\\/0YLvHa0f99u3uIGss4qJJYK\\\\\\\/Y1NTd+0DKx0kBsUSWnWsdj8ZqAXw2hxt99LVb4FSUtyEdHxSTv0lRzBmet4ATbL0CdJLe5S5bwo\\\\\\\/v3IwW7EBiJveR7dmFwYS4u6ynE+gc9B2A3KiBJHhmaUJBPEfUJtp5Bn8Q415460KW9M6IObc5JA1inthzeMs0NiydAWAhyYCVKvIouiRPcPAFQ1HTMa0yjH47QpmiiUAUtV0iUmGvyktK55yIHGDEERzfT4EVLM5RnL\\\\\\\/VaCPWVBcw0j+ApQlhayrLDFOB\\\\\\\/ENFWxYHbqFQICKKyvUaImOzMAdRmxVVXO3EHeE+wMmzTQTH9zulFLzWJUKbU5SpTeA\\\\\\\/teB67F4V9+vFO14YSXSzUJ4NzYXs8iaF8Q5QJsgd1mB9pxGfqSweDTZQVIa76fCth2lrpgYwQx4Mf58SafdGPj4PnYIbZLuVEfDztI7y8wxxxGiwm3SufLV37xMNSYFVCq0IC7p0X9oTXROSiXU7atXhJ\\\\\\\/j72RrrRfKupApMuHss9pIPIjZCdDDpBc5Qh2wmroUkGOYzpSRpXlIs4nmLY74mgPaZ5IQYKWT7Z4vAwaM8lWt6HlQFHz6p5MmEVGR5HCk3w3shNfo8i+yLmoei6cC+NaIlVPyxJpS26VjrAxdJLt\\\\\\\/BqmPPEevLBh4R5qYxQjU3qyRUMhwIT3rKKoFXVfz4ICwYJ7XFlRVFrSUd9rq\\\\\\\/rz563gC0BJmQvROhSBpm5C4gzziQN2\\\\\\\/EuWfXtf6ZmJEp\\\\\\\/2t3rywiWXnizOOSwFeH83nkXNk08HPGloo+eqrCNMRrxA91jCPsIqkxIbGqg5BLw4Ypn19ZvlFwHj\\\\\\\/i4h1+\\\\\\\/xvAIbuV8ezU1J1Irvxzwsv8F1\\\\\\\/EUiGWiwhI6uIj\\\\\\\/WHSYFX0QfP1n1qtAsCGRaILE5aPySjAcHqU8efEtQr4JiIVBLRlYxuz2IaAN3cTQQAOn8JJtWzzYNEDyT0yi\\\\\\\/uqIYtosXiozDV8n13qHNAoJpB5RVEOgnp4otDu1DlfctT\\\\\\\/dmIIO1ky3ctYqCqEaC9x81JJgAlKOhBMfSS1m\\\\\\\/LTj1JNOof6t7RgLZO+gYTl6SjP63EhkbwSi7e\\\\\\\/5jbXMhNxY4SRJLU1mznzeDllRejkBbzUDnT92vTe8so6\\\\\\\/nZIR8fcgqNniQJln2A0LOC6r\\\\\\\/oYBXSeAHaumfrFJ2KqOMabqaM3qZzq8JoRl3wa2\\\\\\\/6Jv2qKROsHp0FW3useoYwVAVChFU5P9YMr5E1B8VkRo48Xj5cgnqFPhv4wUITJykjtPkoqF\\\\\\\/t3CGF1XkNrOlIh6SNOFqaAXCu9WUiYbU3VJ5Jlb8ingkb4SvFJcEu3as56g\\\\\\\/RYB8Bi7hlq9cah7HdrPCeWLfnn58d8K\\\\\\\/GC1GVVQpHZ9k0EP8CoYl86nX\\\\\\\/Qo0EK33xvFlVdTvzJHs2EM7IehiqOtDjs5Qs+T6nxWEuCkhs0JatbBXpQ4V934\\\\\\\/AGz9hl7zCoNPuaTizlTdBVh1fMzVHzEvdFSDurX9NwkSnptJJQ5St10XwyrTOCxV4yJKiKPraJerDsFUXR\\\\\\\/awvoaAzfdcQ63RrEae7bo\\\\\\\/m\\\\\\\/QEaoHzA1UoH7kQxy9bL5HPQtjRyej2hWMGWgu0MXqy\\\\\\\/2UwuJIQWKIhtClRn9rHd2dK3oMnTPmSIXBWpTxbq1WU3YsPJKf7hQnMwkOa2xBlaEopRkv4hcBOcsUU51qxgz1RXd01dMhLk+NByDB92ysNb1JZfUyfftWPHl1ntMfNDaJYMB4VL\\\\\\\/JtwQRasGHnZgI3zky+4ar4r1RW+miCwWuaI0aoX0IN8RIcxfv0Gep5qsQUs4FurGhc3nD\\\\\\\/UANyEIwGhXohAOH2mVsuKHYTLVgujhwjW1aCYMKaxix2ftZetjsdgRuGIBTO7\\\\\\\/zvBBuqic3xudyzshwIIBeUljdqpdVpnbkFTBneXdum9kChF9pOlRdIjOhI\\\\\\\/JQw20qtmTokHH8eYedWCOMkknA0DMrg12OqOdOIZ8YD+9q8gIwKsCQ93jqDniGpHzEDzEbuwsugEqwofbfA8BMXF1y5tLBUzDalGhyfqPRrJV2Z8eHdT6rJFIEMsar+xGMRNFda8Blb\\\\\\\/AXOAIMVih76mISZcHYU1nxFTEJN6DJU5QU7OxQIYIrWTrE595uq+eIWh0RMIUFs4brEYKT61G3mrHhtNf2kto7PNJixyPpydjtwW+r5u2fDalr2hERbCEM7JZQy3d2II4duKshD+sLXjuG3G2frbHvst56D4C1U0s6Y4cl5gXJ4KnwTbuMxDgJ6t\\\\\\\/pXVtkZfzabZEySnbCnH6wD9CG\\\\\\\/Yqy3yEJTE4gQP2VWq2IN9QJEdva612tvenuirHodjvcqABtaEKfrJPYkFeH6kcGTg76EgXqth+XYXAVDkV9Tif+9BX6YUaN2q+QZ+U5tqR8FD2THqoiIUQMTJCtYyDPeNANCS2u\\\\\\\/rfEuSj3gfbRqIySy+8FIz7o6jRdncomAezCSg3owU3IAvv6ige3T6MB0djvU1LR1lNr44qITMYWdO18vEdeR+bPThucO0q8KV5WmV5\\\\\\\/dlc6QI7eNB8VHX6HXyxjKSjPDnzK3Qu6bwlzNBIgvDbKkkKI0c\\\\\\\/U0KzTLtTB5y5f0FsGx5KRw1IGKUCuXZ28cZR7hONUHTSYESSwgSUa9fcm7JZWvw7ZbPzC1XJN2YMRMvUlRJFmBjiuM2l3i0pzYJfNQY5qOCA6oXBZrvcBAtK4idnLiEGfPePuH+XFMnRKlUX5w8W0RB1nE7kzKmqctEc3+8bQHTDlRaSUdXxE7\\\\\\\/4AXbOGV2cFMTR8ODQvZvM4CWMGYuwTpanWBER\\\\\\\/HdVEVr4mObH5rauetwvianShPhsvlGuLNpkUX3hU\\\\\\\/d4UZkh+dFsSUdxhWefAp1xcmwsSFljBvXTVwjPX86ztewY7EQqthVogZnnb18VFuxEfkzYVzT14VX9PgE\\\\\\\/BllEZjZyqE9Jmcf4SP6SJdwfCHK8n5PZh8blA34SS2+ZdbVwNfoFnhhldAUOTwE4+bPQlIJjOeRgWuaIeT8DuLWrBsMu3ejV9SKuP6DsioN5aZRlDrc\\\\\\\/JtRsEvPpwzptbBu8j4H6wJSNtvBPPTlsAUUgExl2tbsdO19R1XeOxQX8EYhUZEt2IuD9RxDbB5MpOjNEFkCzk0bsWJFWZkYhP\\\\\\\/+ckVOvqNN4vkHGM+qY\\\\\\\/vHLONi2qrD0w3V6xTzTFpUeGAVyNIksmtcLCUvMA3S79+7hTnWzyL\\\\\\\/bcNujDSkP7XaD9rFAWpBNlC+GDTSlnlSE2xP4C3DAKJcJUxM2Whu+zXohoX2volWUDk7dii2ZucphAGEiLlXZtNknvyErzN6ZGgvRaU+ksPNGxqx318NR1Dn87FtF3oOhuxUkyK4dgI4eJDgzg6o8dwVcM1mbiXloqDg\\\\\\\/WD8OyoldyI\\\\\\\/InhTXUlKqydIaLp8zhBTMwwi2qfKSwx3YrsyOx+NrVWsJcKt2T9fUuODkjsNmdM+RcyjChH\\\\\\\/rL6wXV5fosO820DdDQNG\\\\\\\/DgBqFbtVtngqnhr8q\\\\\\\/iUQaXveLDD5+sNOCQcOorHEMYxMKbh84sm9jBdQc6KU7YgWOwjHYq4saP6sq\\\\\\\/z9qREag+oZj0WCzDu3mbIHIVdFiQGFnu46P53ixVJfuK39TOnTn58+hLDE1CYMCMrEGl1W82OthO7hNkgN5UH6Y2oSJWNkdAOTR84l5WSqU6xnEbXVOOn8LenzUxNozgcmpng40n1lQ1WdDzwLV6nE7KQr4E86lvg2aGNkC8zkAt65XX3+05NBZCUJuPRsxTg9gR2IyH+xzBYvdtFgrALxwn+hyIUKkmyqBdrAY6h68MjLVHYVad1GUJEdBJAreIBlfn1PWEQGK7nEtqzd7qv59Kw49VdiOWOOYDTQ87Y7HXod3h3KeZkeaUQQSg4fquKPgoIr1TzoaM2gP8SKV7k3wdlgD4qDrPdLd7umsJkyA9qbex6nl\\\\\\\/UG2PScYqGreRZfnzrxUSiITc+6bbmBR8yUeyVpw21Pk88kDehdXTobSppKG8CRqLU0LM+eVjoPlx3wxxHxA\\\\\\\/RtvDGBNGKwOx\\\\\\\/dZlUph1qXodkaZsE\\\\\\\/bAwwCor28YVM2rXkjhV09WQp14tXbnyoxVm6VtV5BbxjWpWZctgEOzFN0OyxK4Gteqh9GIU4WBb1RDjD3AmHU0SK0dkGalgtYsxhj1yGxX4iOfvH0HGIU8Egnfhc+r6nU5bCCP6pe7CAlOt6LlQWVHDSN3uhudNMBc+f2g69hfgErwmtP9NmKoVIvVFaMyXiwBEii\\\\\\\/7IUd0yqbEU369HjWp6kN8A\\\\\\\/mWSfvnzQOSK1nPLlkBSYCGUqyLF8+9q22HUO\\\\\\\/59RmmWDJwyRi37dILeL9AkaM5ADKzg6oA\\\\\\\/uubnbcVNJRNZmQR8knrhiBsCjpMJkUmki6wwm1m3x920s\\\\\\\/X5hts9typ7s\\\\\\\/Ej2R8ADzGKRR2KKe3Ii3gxTXZSgYfxJlCo9YoCKGo9h2J\\\\\\\/06xwGaD+GWxtdynObFRnQZr2un3EiDEB0Q\\\\\\\/k93YJx4Z\\\\\\\/JrKqJsPGoiCUqCcrR+K4eLGuxgj+tiFIIUao1OnLjPcs6lykw4aCDGohbzv2j9IgL7u0aN4IrUOSnd9M\\\\\\\/b\\\\\\\/nbsuuDrYqRFI1fNCWO46\\\\\\\/wyL0avgxeNOty\\\\\\\/ogB+WYQCkpdT99F5kJXjsNHiiLr49uo4nrrofpiRQ0uTKOzwJ2qoKyAy0RXxOUQR7qM+vInOhgf8kfY\\\\\\\/g46bcpLEEH65gm9D8zNxoMQDwxIk6hjdzwUiFLo3VqLamuu1rK2f4P2h7KT1xkndxlwvvImDOLIe8ut2A89mBHFHFZY8EQ+KOSsLsLZaWu5z5OluMN8yQKHj33ouNILsuCvs6n43zHzv6OGvRxjuKB4s95yQN1y6\\\\\\\/9B9a7pDY6bEGrChvKPxb2Jx74niqaDtpmB4Ow1nc9AhmfoczQc+bbHPCUC702qBAtwIMxm\\\\\\\/0VErGIUbWrNhvsKD90EFTnkbPn\\\\\\\/a5jSpINYYrk\\\\\\\/U3O7xi9QAfyvb2l0ia3g3kwQRY\\\\\\\/jxMtAIWQE3IWb\\\\\\\/iYVz\\\\\\\/WeVQli\\\\\\\/1\\\\\\\/keKTEe+yhhxx053WzykRuTYNv1ejcX6ZOtq\\\\\\\/+fVjNZMReTTTTAGF+3y8n1tIPyxqFMa1G8erYN\\\\\\\/0YEQwLqpDdwMYbMZcsJaXmqrH4iKxsltm5CehDJDgqoaRl8XxPqm5vMCBIfWJocwa+CYeq7\\\\\\\/GR9VnFiln2gQp2DMKKYwobJ0bwPDfy97InxDnMPpeZpeBZNCB+abGLi12uPbfBMMjXf1TDZPyykhFNaHXey717t8rPXW7RSk76Qf4opSTDrtKxJALjfBoUByFPCTGm\\\\\\\/7fhfgBcrL4ezPs\\\\\\\/XfCZLXny7phW7W3Vbg8j7GJ6G7rkraYXOYVOamDPq84xPUwdDY6Sq4QbOF4NCtEHqKLNlsNXcx8kE4NhJjohy\\\\\\\/iQYh5185AC1s67mXraAX9i5IbHKcsHJqx8qZETXdonvnvzW4deiQahwoXM0MgW4W9PVBHPYZc1+OtD4JWMfq9sivB3r5avi3nIP113unrblbHvQ5XguU+ieKKGNPMsL4DQcxEFxRS5YT1uL9L6Rw8UlunbjLX4MlyDngf\\\\\\\/CtH+OH+u2z5vlVYI\\\\\\\/K1kE0NrXjUVH3I\\\\\\\/7HurrF8vHzcBHxpNbW8RrUoHqyXdNCdgelxg3f\\\\\\\/HkH1L0+FbTlIOz+64\\\\\\\/o1RVNOZmYEX9Bzbj\\\\\\\/pofnkgBwNfIeH9H8lzriViZAFErYRPgpTSOkz7Y\\\\\\\/itq7zt\\\\\\\/I0+zAbZUi7AdLdlq\\\\\\\/a6pf2WLvcF9zW+kFVFxUsJr1u2dh1SWMxfXQYu5qsKZYm2w5FmZFTQXYkcHtucNF1vZvkL4b3BCkqaE1x7WIxlKP\\\\\\\/PczQXDN8uoiHaJNgSZXEgeyBYiHV9mUfYcO7rwbNz4QEwuJyj4ORLxQ1ST2r8Q2Ih8diq0HxjPwyTyNDiFUOPVFhJPxtGKlTDqwrwhZOGbpkiATu25Dj\\\\\\\/vqbXiK9nrPpGTK96Xv5NrjYG\\\\\\\/sd6clLg3vdJOue\\\\\\\/RBKj1OUIjogBkkRqTeeGbdeSvREayOcFXCypScayif6551zFnOODwWXr1fRSLSNQi\\\\\\\/VjBwuvGgIJVuB+oGfvJyTKPCapOq6dVhtqmjV9z9uvSNyADKPowjNMvfOxlzTowr7SD0FVgt69VZu4B4lqxckjZBPDUU1oKuLsKxemmi0l9q37E2VuvfnmPZwF9DYGDhUOOFuGWwFgp43u5a6SrtcurSoBfiWnaLzgIKVORLuCAbrV\\\\\\\/R5wFiwlWHpl6I3yBPfOnRQBt5H3f3ydebqQCSo2QsGTYCjz+rb9Fl0q17X2r+t6JB\\\\\\\/TM1uxPx\\\\\\\/Y8kCbweOlk2HrStgaSI+WrEy8lb9o0sGVAUm4I5kwH1pgIbz+5SXZiFhZQZQEmWz69F75p\\\\\\\/50VQS+AE7hvfXr2AzEbKdG55e95wNz5qOF4QOcVVtnenfii095zsirB7VOR\\\\\\\/6nysmF3YUtiDYd5UKuTiq0T\\\\\\\/Wog9W0mcptEj8dY51Dk25o4fRg+ZJMpm2ERIJ30sJtjinkgc+6LdPqX4HJoc9X6aPH9JC3n5alWG+0hGNt7yIQwJTFinfAIgW1BZYQJyuxgurJl5qQlo5clXVF6oIOoZ1iWg74uVGG5mN+hhJlOyctAYzYx2GCpLb84wvyePmFv1iVCqeoV5HL0Mi9sIaPiKQvZQc4eYU7bnwEG4XKAiWsMkDvkoXqI2wWbiOjnIb7EI4AbRjQqrKupX2Q\\\\\\\/aMWWi7Og3yhWoxEQKZr1LS8xzs1wdELJT4Z5D9z6XcO1UUE36KXziwIurO4kIRbUnNkMjWLYtG3tSLpxmbyyRJlhhA4l5VmbD+eyL4yq1nHuAMYy9bOTA5wDKYUK80XHsJkpjEqooo4lrKux9xelvDBx7El3JymKfTgBGaugd38Uw160HZyX6fVC9wdZFuCPlc1xtFiX2t3v2jPuuuytqYV3BwfbyZVybd6SZSTJlOqRP0GZ5W6oP7oa7sSHeSrIeYgLHYGzW22+mJSLlfr\\\\\\\/ip2CMxOAB7QxQ7RVlBcy+aU0RHDFctaLqKioiX2ZnXDMnVkOU9YYjaOCBxYzV42aBY7Eh6po41gfy7nbqMh6VOgoucL2OZi4Ns\\\\\\\/qNBCuUSNp6PqSKk08ukcE5q4vM6zx6SWYcOzPc\\\\\\\/ztXaoAkhbEFiIT6h8hsDRCml7Dq6t43xO53VR022Yd1cIvafBm9+BN1PSn9+axL6U47kHrjsuF\\\\\\\/tD8dux7Ev50DJtiyvOdDM0sDMCKU9CES\\\\\\\/I+jYPcHps7K7AhdB0YKpzeB4sv1Zt9gcKbNlTm1RDvRroDJ9\\\\\\\/a9\\\\\\\/SC4RGWAffolCoENHjP4MoeTTMDFrrZQyb7mGVgVxGGaw+DRXWH8HdUP9ATU644YFIBNgfuz+0XDoNyC0J5beVuX33AgqFbTWgvqorj5CEvSQv1cYwDiptAlnBy6w0MhEKe5SGcbEvfcdfBZ8Xgup2hR+mhGJpiSquJlQWRq4y+EYOtb3Bf7mZAtWBFk2u9FwBg\\\\\\\/SF3EUQSf9g2877OAHVIvvRcjUPIChtBgXBuuTHYvlUuIg6M\\\\\\\/uxKg42KE78SGctsuDc4x4DLktnFuwbdd2FHWcfveUo196MzT0u+CdH28KeAOP2cW95os6ysmN8Eg0xMrRbXHDm1OsV2pgTI\\\\\\\/K9pwYCWdVSVuWj5KOjEUc8fpZ74MYPl3qiMmAaYtPG+C0anhkQ+I\\\\\\\/RmQ1kvwM7C8JqBaxyTxjVvIA57dLlPbIhbV6nn50p9Jx1CHLWERXVY86EZM92\\\\\\\/HOkUX+PjrscX+G3LOdEQZH2dDDLfdMdAmtXC7SsyOhQfYpwWeXAs2RvAN+aM7AnmXJlJk35D2AW7nOFdMmN+c6gX0j+QnYYdLj+CxDfv9lX0yUJ5YQrzMGMRxfqaVPE6UgeGMTrIp6WPKNYkgNAUZTGKufcE2jWjJd4drppQrSG2YvtaDBXWMtFSQx1Xq4ifwoF\\\\\\\/xUK0BbLggWs5+NGFrwJEYNc9\\\\\\\/YxHONqHr9uDAlzYpQJmKoMLW49e1dGifbe46WZuWG11X3u3Hbk9+RJ9X3RJFvrZxq\\\\\\\/mMuIH1Sc0m1kDzPHDBZA087xdnqQRKc3RD4W1coK+blFCdqihCZAWHQv7REVv7JZbB1sQbEdYwMMXmW++XOfgklJ+Hey34JhfLO5JMMVhVBJsowIto+7bOH1+nJEsLEouiKVdmqf04U0JNRMMSRmvZw2kPnq7KbGln0MAWQbX8luRwhhxKTwNJ91NScIMnSGDf9fzLxv3CTGd9bINTFThxQdQwf1gSgJfdZQc9GD9QBeJost78MN6YUFSFs4Jdwfuho0f7J9DD9R+Ub0dk6QIA7binDs5Apr2lXmWbxGJUFgYCO976xsSWWt0qlX1LAD8cIsOv54NI3cXIJsQvj+2Hea+HuF8C9G\\\\\\\/b2lQbuobd6pBc9cAn4OHE\\\\\\\/YK7zwLxvgAI\\\\\\\/lOGLpGqUSFCcBJIMznQ1AKt47CEZUFwBzOIxWKwa6jtu\\\\\\\/BjNqYExIcwHK9bcIHk2kL3Rqc6MrEd85yrnbHyymkkPlC6zr3bznXsug24p6JlMjw2t7zLsKlH\\\\\\\/2HGtsV3PIWud91fYrR0reDHv6zpFchA1q1bT7o5v6+0QkJyXn1V2E70cLo6ighXzxq6hFgUD4XY9V0ZDSiaZ2Col+U0h7t+aF3Nb5XA6s36NJXiGje8NywlEz3qPXjwoUsKL3CLaNY1dND3eSkYLE5GFnmimr3McyaFVNIVuOpy+EbBetR9bYKOFoWv4WPO1w25c1sERTVu5P+r7GOvJJIDyW0r2DtlGxrLGHx\\\\\\\/yU\\\\\\\/M+MT1u9Hx3UgCFeqoJEPkr4S5bVwaP8BpBcCw1Fb3dS8sm1Uwm4c6COqP7ycbxJm8FankN9j2xsrM2qhXy8DKimSvho3dH6bGsBV1xp6kcEpHyRZ9+E0w3kxu\\\\\\\/VJOVMSG3Udl2bpHsO247bEHrrEfL\\\\\\\/U6Z8bN6TWJBER2feiQ1gxE3GD7zBkneK46yJ8eitDkAoUD0NDbrU57jdpbGo7bULBgA58cQZKmD6gbPhicsG6j1sj3IGXGRo0u6FrCtcwzlOJbnPKV9ZOOmGzpCQ\\\\\\\/SS8goKG2mmeHE96ofUBOuMvCJuqloRKk11ehBvDi+meIL8Qf5WHf6X0ILWmLpt+afD2YusMHydSbXhaevxgttUfNBb2IOwWr+mtDW0iRrLdAj2I+sobcKSGaCPOpQot9JJpGwtoiZLgZFjWOfPQwmG4mdFZbtsmk0g96O\\\\\\\/mEpdB\\\\\\\/w+OWLI74QET2VfWiUg8yDtruBG5pBI\\\\\\\/Q\\\\\\\/0Nz2D8hJF\\\\\\\/6Rxmd8H+sXe1\\\\\\\/7GPWmAb8846m0xAEBZzLRAhZzSJvZjJhBocfyOH5PcbRXr0zJ2qT2Ol4ARf6hzxO+2YIKB7\\\\\\\/fxcp9xGtKVQWZepjfrdQWu5YyDjhFYReEfY+poWl2JivimqVVFusb9mgklOvR16RRBAX6cWR++uXFYMFN6bd5DWkCcq27Y+sv9raYFdlDDSUnav\\\\\\\/zkwJseehK6DJmF7DvIeF4gXzTGIRoHv1qLVrfV0DvRifWFhKjD2eErXLUbVL3pDO+bXBABLAEhKZBBKN2EYSw87NLyuq7qEjpnBE1\\\\\\\/YU3MSQc3vMLNa\\\\\\\/ioFhtPy952bRmuKa+V3LKtNy32WlPivSamg1Q9KIaMgWeO1Qgaf\\\\\\\/nu76PoO4qilRE++l4GHQItKEvc\\\\\\\/RhiID6loOCKIce2mKR6TewpCopGD5SHGpCzBHAfHvcZfIO2DFX1z\\\\\\\/5VxYe2uS3wSazs6wUuPi32\\\\\\\/WtzkoNiXHBvAwnq8xGPo4yanOvWyKDHKOKqWjV1iohHyrVLlvwV047NmNTf9vWBZxRD8e3eMaQefA39yb+81D19JtEMmV8HWJ25GhMbCne0UupI9M9hcOWUvUlBJDWPAhYCZMP3gCT9TTPaB1F6Z4llGrM4EFUnt13PS7Gx2VYXe8cBj19+FaAfU4eCEzJIi3k4zPTAJUTVl5eYEi6NFhTqnhdcXJcVLFWWCMVfw4hFCE7K3r3q4TLPaeUMdRYO35vTCU7zW9d6jexwv3XvAKx\\\\\\\/Jui0Inn0KPChDNd56WZ0Tq4LPyAOv2XRjt\\\\\\\/phOaO0wxYgA71XTPBKb7dmfXMj+N9s\\\\\\\/suGAQlXWhtcyWMlUEnurDOpf+GbPP\\\\\\\/s5H0rAp7wvCFbH26Bd+hDZXhOInl0wnBVBI8ho0DM50McvhiwJXlMPZcOdqHDRDi7qfiCXwK0cbkJjr8CiFlY\\\\\\\/oQEOX17wnCjInghDQEc9rexjSVJxR58RxO6iMyKq07+VrS9G0Jt7hvze0H9GxDOnj3Cs1kWMuwbil9PRw3Kdre9CxWF7FPNrmBRfdM4hmdZK+\\\\\\\/4qpVbwF3c0bPURlwjJbjxCH+WObqNrJHbapBb2kOLADC3pDqmqbBVj2uf6gu5DsnixBz91I8bqlSVgjDUldWCuDWHNrwaa8e32Q\\\\\\\/EIx\\\\\\\/QRE2kyTfT\\\\\\\/RAyafksPNsaMjvczLsUCJVst7O06Fm8TIrg3kq+R00W8w1hOh7n6ncoCko1gIcR6Tb0RgRDphcvobg3DNvY\\\\\\\/q9X+5HOwXvMr8emN8izu9u2Pvi8Ac53iUCIo1jUzfr\\\\\\\/+ZsovVJipKwvbnlujBtmArkViIUQ45XgYr5pC5\\\\\\\/nMFt13l2TATq5P2P1SQ6q4mlEqkLL5hemqRjD55i8bKl5Ok9eBXjo07tqkRbrx5M9y4CzRUJi6eD2eyBx83BRZWOa6WAJslvIVosLFxu3aUt4G0qDdsdbu5wDh8YicR4QXxhEoTTAw3tC3WeIDPsWpV9JypyleuX96J4Ov5Yiljywg9uIlLP1ZUshz3KqwSmeK4iyS5Dz7wn+BmmieplwlxC7GzZOVij1jb06zJoy\\\\\\\/6wGZgTt0bFKor06e5tg\\\\\\\/Yik9mEZCGVyH+PZoxcWzmImmyNxLMuM4X51hhzb6Y6na0LTIyiK3NPAkWnWwx\\\\\\\/MzAN5wnZTpDYdyo1Ar5oIhT61ZT+gfOgS+J\\\\\\\/F4yoeHUIoAhowKyD+UWH7zHeGvGs7FWJmAdA\\\\\\\/eCo80KAzGMfmlX4znHgsSqOSBB7J+WOaKEAMgrU7FaCYchYlo3vl0\\\\\\\/0TMatFc6d3tOCnVaqkmsNlHNGGH0oDb8fVde5OQeVM8BROULajB3z\\\\\\\/eU+SutYCe2+OOFgK0jqTXvMocSmN+0RTOJEoySeubL54KugVwi1miWIzD2JPIgKPHH9EUVt8GrP1kOsKxgY6XvmlophBpNpxDNj0PW8h8ckAg1VcHhM3qLf94Du+9dgpZDEeB9oJ4KRT5BjBlZvTkVo9v1VeOu41+BREBX3DNfdsFkTZKoiyY6tClFLpdcFwgYODV2+r13B\\\\\\\/xA6gW2SjLROk7GHwQ5TVyuGL8CsqJKU1lihPeGSdC8UTxwKdI1MC5esWptfVLFA6N+KganNj7+zQtdGihTcOGezmwZ2m1IxAM2KtWx+zd4VlEDsV9Uc3w6ZvXuENB3u0C4IdaxD7W0L9ReWdrFnY7CV\\\\\\\/rAz8a50NiBG+upARhpTLUIobLIPyu\\\\\\\/u5tkRgNR7lUFM1XBmRzJ\\\\\\\/YmW0uEA13SE9mQoM7wUlcYhZFzQFN18brfjEfohYhwczsZU0xTt7J8dhVxEWZ9bl6eWGRB76RT0S71AAU5Zh4kQ919o6W5\\\\\\\/GogIa\\\\\\\/53efncM7a9T2kDrj5ShfyNrBlYtEeyyXLe1oGW65y0ybcIPxoElaD9mE\\\\\\\/RRrAxzX2bpE708Trz5dipr17YqSlpCNpMF+eKnJk+TiG4X3EEhmSrMtcL7WAV10edseFJuwaHBrwh6AoGI1GHQrjtR3fkByy1TfqzBAu09ktPtLUW6NB2K+C8rJBnPFQnSRJ+WytKhTn\\\\\\\/Kl7kG7Rsj7pehacx0MYa10\\\\\\\/Wi4A7YWPX1yZjLQwt1MZKdC8XHnWlCaPI3uFxvq\\\\\\\/FCgvYJnr1WE6Y8EiN8GFT2xRX+dfoIPnDV65\\\\\\\/UIvq5Lt\\\\\\\/n7wDNMrRT44xg3h8AYhIfYpiPzUiVGeLa+YlFxt4H67fuPYCDC4zUbTqWYeV8N5nfKLulIT0IsgkOvetsY+hM7m3CGcI6iH2IQQfi1sxvNPHf9+VgXg5ZZ1qwp2JEICGkGZWYBJTwbCdvQqe6w7ekYMVayyY7zWeM9MEuRc20589XKUgumPXKdtvm9f4GlAWT7SIczRlhRXIpYPR4iCaArwhUjkk2s\\\\\\\/cVnFFI9UsIWxo81IJf8KVjsdgEd74ZfjdjvlFMR78QP5yaQACPYS4AnWZe7ymD2MOpYJYRbvAVXTV4mU6FLANLGdKCq2zYzS5aq4+gy\\\\\\\/3uSEQOndgAgDJDNukn3x6NcAttp13SMGrzwDzqzABfUd+Dp3CYpbFn2geWQ+JuflG1q\\\\\\\/jsZ4NtbZYjMIep1b2+Nv+nGaHRDnQ5M3vzuBuhHuhJ4QFLSaPwqFQp221pNQfqvZUFjoZdIBA4osQelnBzi+oLN+7xfZjQUYS5va7yqerOsXUKT1Sg5+avzB4F3LvE8siH1HIKEPdvo0vlAvCR3Xp+aD0BzxptHhs4hMzoWxQTfy\\\\\\\/HmFlUIZ5C2w+Ff2bSk\\\\\\\/SMp13+rCFC7EFXlPGpe++vSEU7PRUs2ReDsLsPFhZ2zNUZ5gzJinBHhzFMlY7I+h6Js2UHGZKEukY+EUu6LS95N7ldDqtsEkC\\\\\\\/ahY7vkJrLBp8mLd+rQ4VE8tgVpAeWhC\\\\\\\/hcX\\\\\\\/HUPegxcz6JNYtQnYeiFNhs0lwgCIzYs+8wNGek9WmRF8znxQYkmYpw4JvqmGKrV2lB9rWT\\\\\\\/XI8pQz7XTb9r0K15hLiHzHubOGfyTMDbzLPIdjQh7FqO\\\\\\\/ylvY7RB6a2f84r1\\\\\\\/dZvGSctrbBkIX4K\\\\\\\/sWSRK9fsNu+ZRpSZq\\\\\\\/RQfMuUrGxYz+dwA5Ge8ULHWw5AC86ca8rRaCeQQi6FfHV1uJxGve2wZGYWdNnTHzthmlavn6QNT9En9a5vuZOsh0nqzY4MO8TvqTefdSxXCfSSSlhEcEKQ\\\\\\\/H5WkyxC\\\\\\\/wPY6roiBAaRTchJjUGPFKHvRxNdwWEN9SrV7kCqYoxUgw1J6mdxeNAnM\\\\\\\/\\\\\\\/AVKP2XChX391TPUtcFWyhrMih+F2Fh6vte6VZyYrZImN50nYiidTlgaggBaxVkm9w0CHv19tEXjR\\\\\\\/LADuGkFK6yoVJfvLR5UE+JmzAClgC\\\\\\\/YkGgy5zIWdxLgkwzZZysAzBXrCW3BlfW51mbMeOwxoeh\\\\\\\/15s1aCoO+GLNoFnq2vfzFzJiRYzKNDSOiuYcb2qW5SeEmttc9L38t2W\\\\\\\/fE9lhapJC1DXguKVnmGmIbdrBcrc9cx1SHcJUxMpaaNEjzwZVhDjsgOpor4xlxPl+lMuWxVTyRGnL2ZMM\\\\\\\/7eHv00FkNP2e2ZmkvlTb4nUENFryL6Ad7Y30D6X70RR4EKG5t9kt7EkhRiTcXwqevGGKRSV+Ku+IgbkbPhfO4fAow9ggxg6BkQycmVT9ZShgmAdlOMgNGRi9u\\\\\\\/CwLaCPohYe9X7mupeKbBcpkhhMsi7fp5RIy0PmMYMT+XUC\\\\\\\/\\\\\\\/KQY9jPvJr3sCGx8ErN1dSd44qRYh1eMxWeVv87p\\\\\\\/9Zi7pZg8pO8h7Z7Le0kf50CJ+YopuFb3lHfX7PhaFj48L5QW5mLCQs\\\\\\\/HblHzN5X1xQrwmUiHwRDNGy0qRNvPcOsrFh0z\\\\\\\/zbQSkOeNDkQABr160QRcANrlQe7iW10bpEPbRaRYsxiOXLxC+ue8VdBR9BxpRxPmDIlCyhKQUhuRUOn\\\\\\\/Dk8mTeleMxrRLnAr8Nq8noQyKZrBLery1N7UQ2YvTAAfyvP7\\\\\\\/vZxwbcZRfloXd+\\\\\\\/I3GDuqanrYYzyi2q4iPToow3xNsUYUZtQUy3NCCQ3CWTDIG8JlSxPy4JEyhlBS5duowwiPHtcTioG37wycfpE6jbkvPkfPKzx6IPWzOJWVCGW\\\\\\\/mY08zKHydpMkowUfrrQfRsEhiiW9ydv3+2xIvRm47jq7OwSQl+RH1lCjbMLIakMSN0yfIlVxg0jvodtxhlryxmjXEOMS3aa2uerxarHmsMJuSwKY5bfJiQuEX7lpr\\\\\\\/DTn1ORKcoJoI1v+y8JWG8uzJgu96NI1bxbn30fB96xBEMKb9Lo1QxNtyo9u\\\\\\\/hhB9hyH372Wt+h\\\\\\\/aLdk0B3t0XgrlAk6yU6akh0OO2jgVaV+xGvgabeTHT4eUC8lSDRPn9tOVfelv8Qh9+rcOfkbWhc7OlsF1mKhz6ivm4MYE6sCyHxvYkj6LGIanLLMJlKrCy\\\\\\\/W0Y5b2X1GLv0uNq4+2HPvUfV5NP+rAuvMtaT14pIQY+jM7TvL5CO2jXM+oOyaMqQyrEovrn1z0tDkfSMPpR+8Me5uygVhownjuW6HVuHr1C8okPUyLi7Cr2hLV\\\\\\\/fzsOyfUxKMSVEQXZXeO8xXPv2JWVG9PwvgXtMk0hN5s9DZS77\\\\\\\/ZKkj4XlFxfpeZFI2nymUa5gE\\\\\\\/AZkBnYCyx79MZ0fJK1naJXRAAhdcFZs1cORmvCLWaMPmGcvPqJknC2U1hpWk0Z4RBb5KG7VP92eKiFH8nF+P+hhj8nQsVnxqOyw3oA0Td8X34872xOenyBEw+jaYWMh+YJvVkT5nSWudMDnXH6w3gZp71meeBcvMdK4bcZtaiJpiquTYgKmlg5zcVcTfoQQKt3XQyLCUD\\\\\\\/PIqlUngvetqLH2AzgcHekmb7u+trxa3a73ERUMfst\\\\\\\/GFd8P5zCWmSiJOmTIDy3rhc1cmYPRXtAsOq84w1nlGyIMo9iME6auwZ45+vcXxwP8ZFhKdyQwygrSkTwVh\\\\\\\/oi5q2iaNNHBLaXLF1OHMwHzeLyv2slvHO1X2ZGwRWVOn\\\\\\\/5tb1bWsY\\\\\\\/BRpSgBFn2jtw+XEgyYPFObKn2y+C6IkKgsbgpFuZHTvese3STCRFmWmkaF0mL0KLyAq5eAeQEl94z7wIXXaNWCflmh6m9ubvX+PCgT8eVU6FodxjNVb5AyJFiHOJoZd8otc4iHYPGZHdrsVTHPAaaSSiTAh1zdVDbI3nBlqQqk9pPVnHPJAgbMW9NLG8YmlkinSZvr2ae3hXOg\\\\\\\/nvLjrG77htgTLw3pWM0lUAnBQa9X\\\\\\\/d0AsKmrEtnnF1A0A9+1\\\\\\\/AN84e4VC86TzMECb7t0MD2qLDh\\\\\\\/NHpERH\\\\\\\/xqwCYjqnphP\\\\\\\/ZfTfulzw6DnzdKQpBjXhoqMtLCyZABxld2r\\\\\\\/ClTqAmUZztF8voCHWvTWYlldSpzcC+L5V+eJ5ZU1FzP++XCIGhmI7aY\\\\\\\/H5VSQXnRLV8MB+yNyQKfSzgdRfD5Z0tWYZjNYNq67djjTp2uDKlOw4co6mzrVbAoGhAJctgtnSJLggnHZ03uF9WhFNEyxCMlHfjSe1Gur0UadYZ+PvfsUfc4mfvn42PVfSfq6c9c7UDafIYzEvEwgjFHhSRNuM3FNDzoZZNxeOuplbaHVMVqE37gH1N8++2z4+FwL3GeGxoo9yJTuLjt3fQ0PwPjU+kGa8JBGUS3RQe+SbNv4fUwPQwL2mBneuvgKKudL+PEgvYjvk4sclLGjcJ0btoPBagP57kS65gnnN5VvOFIPEcHVsfcWuuysCFUpMbBmqypHvfZWBqGVsyKsjZPuuHCWqVHxaYwouAsdzY2qpl09TNDBwiatf0g+R+X1SG1k38+3HclPZFVak4ueb5ayBrZnpLM1sJxJJZKW+rkQSaFBeAn72UvFEXcs0AfiTB9GZMoBZTgEArGR\\\\\\\/SjlXar5Za\\\\\\\/yxPlip\\\\\\\/+XXsvAkiALqEldF\\\\\\\/57VqmmuTtiT5EFl43\\\\\\\/FVGbbCbAe4Cz5OEPABOtgyu6IpCO\\\\\\\/T2OFgND+4vuB8snLHtcOyPZqgFTl\\\\\\\/wfNcP4Ua9CeiKyGLyEnKFEKr73Zd4dogAdpKRN5SXouUPiQ8+2oAepbGk09D2GjxxjNzjR51X6T7bARkZi5yH4L0EHy4kU8jcI3TMsmcRtKuABYXLKVszOUyLe0su7qOKGd\\\\\\\/Y+9+SIVcGCIcLqQgSpEebapkI5x9+n1m4bL7tc9c5yhkMSWKYcYVUU6NUzxcZZQhxv+CPudZ7ZtG15HAMAkWDpP6c+qor1YjJnqEaAujLZLczRC\\\\\\\/vTMqiQqLKjkhvHuwzX2Y2wXlCu69dOCfvbS\\\\\\\/XXRNJT+uDKlab7VqdRh7nnZuy2W9dyV+uEMHAu2ZqwyHyv5YxgCnNdiOyp1qIaso7+lMU14n5eEbmvxAUNAWOxMrtns6wD9JfxTAFtMADCfqzvMBtWaOB0YxqmQSA\\\\\\\/7pUlmtZvfUXH9HWy4CJ470mMbvNFL\\\\\\\/3cYt\\\\\\\/gk+rkKlnboI6qth0+Exf8jYDjFB6yPxmQlNDLNrQ9xQQcIqmgZa9uoXZbaHEstUHdsIN9vE6XGLRBV3e2ya1ua43zv\\\\\\\/Z2l25RDrsG3ih33FO\\\\\\\/RzNYtALC+6Lk7uA49\\\\\\\/QRZ\\\\\\\/xK0Zp5vaXcp518oJyy9po76x2IPgGxUQUY3pw0ahHfVNyrAHXptRwaV5qsuCoWXWGkV98Dd8vg+OFzSm9BN+UKde9ggd8fpS5ElxppekVl+GRFDP0CeWSu9divsI1zU4H4NxBITPe26FLjgKJheYlpun1Ex3JENUXt\\\\\\\/a1dwm+5nr6so\\\\\\\/ZaDW0aoTDVAEaBFnuK2j1DiA4A62G1+5d+UCKFgJGaYGT6OGPshRl7\\\\\\\/86lTdy2z\\\\\\\/pCmTs8Cs+GDf+dbJxhB9\\\\\\\/hyesa\\\\\\\/UJSjbzIpNjTx9oyFPAqV7IIpmVWmNbEXyXzr0t6XUwq0xyz35eKAg\\\\\\\/nHEAaPC0Qf0kgvYs4f9j25us9BN6xbV0\\\\\\\/x8HEMa27CBgjHOkkXdcdv9zN6i9ELKUUVcMHrlFCs\\\\\\\/EI6olm063PzQs0IzNrMg3eyFkHAG87GLOd5SHlHI9Lk\\\\\\\/hjG+ci7qCsDMDucGnvpCfTE9+mnvL0\\\\\\\/aTlBHr9\\\\\\\/TsUGgqZf+AmAC4drv0FF2b5mCNp8EeHBtNqG5MwlC24yDzIO4qdGRJQ6e2q63Pn0tK8FCrE+rVy05HyKrh0iR0KZBfCpGyKaT\\\\\\\/Xbdk32gSo4Dh5wjjopFLP71HDGheC0FhxsBUx+ClJv2kX7N2mpr0VHcCs9Hcb9HahTAdJaJPGcQt2O9adxFBAcwVuOncjdIZKQn0VbZeP0ypgtIDRoGxNTT9AxyrQNuwG+S8ZxgeA5eN5mLjeYNvybLiWb3V31yXsndwjazjZjm46JBNxOO3URbs84tK+8NXSUSU1WNTLtPpnS2lhFm4yby8C+VEzSBxvy5m8Ip+7qYEbj9Ai9tdc\\\\\\\/4Oq54\\\\\\\/p19j6y3+\\\\\\\/QYAwtfy+73ajB58BuUFhp0lrZkwo5jDIjJz9j7Ip1t9WrPOUCqMsCieprl4OHQ+pXggauwk4Hh8jRIJ8eAHzujcnF3Wx4sH28kr3EdrJFg9AS8M3jZjjHzDrjHy+OB8Q+SkPKVhkSxj16XDrEfK4aRbno+gVcg0qBozJ9bVA1qUZENZk+\\\\\\\/5F+myaG+hjU6QZHmRqXkwoDEuuPnX\\\\\\\/3C0gPJHGVg07XOkJJxuyH9mGiyvKQgCXbdAIriuTahJFhiPFMO3AOD3CGj5EbpK9R6GJ2NR4y3N3mSq0mjGxq8Z9SVJJkp3\\\\\\\/ylArRdtjz41U+Fgne9eurWkE0oSSPFXhiExzgWgXQspBzdAr7w95cm6FDZmxUPC78S+MnO\\\\\\\/DqEDfatufWEsxKRX9Vmf13Q+O0nQX19jeJZqGVjY6NdLQgShImA45HOzLun5TLrA0zvaE7P3NeILZMhLmRobOa7XgBfUJ\\\\\\\/RVUuEXn\\\\\\\/rUGwELgGXvIwzbiQLanqJoVnhwFGJfT3g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022J6dutgS5Sz5Pjt034JfETA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022aa9d2292f3550c4dfb3f76c7f6b15340a69ce3b7867afc1edb5332b2c45974eb\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00225nMcd\\\\\\\/KEGfm05JW9jFDetLokRMOnxDDI2Ie6vChFV6kGaWmeYqetKOY1JHw1ZOqZhubF7I2Yz1eJ6oNKpqwc53oA52S7KL\\\\\\\/8Px\\\\\\\/tLByXkRZdnxYfiFlUF4uI8HiIt\\\\\\\/Hqn+p54Zq0RyL5NOvbsUZSNi3vxW8f1+ouNzSL2iYUnjaJOj04WchKxUygtK1pBWggWCHK5UEcbJd6zHklwUKzZuoxZQthaREpL9xtuC0ZXPPtF2SrUVSqHzsZdCeuTOdSLquJqgfhS\\\\\\\/TDvwsKIr2WrjsQgyRMDnYx7gNMytDn\\\\\\\/NxA1w1jJVfqBnzauHQsBILof3bxvJCzaVvGq1lU2xhnQn1Ma5Jq2cnZRXW23hti8R2FN\\\\\\\/G3jVLxk+uENcRM4EroFSERE9uDZLGXDMpCqrY2lcDi3RjKT7cTrGl9UAN4TLQBfHajzIEBgvO4TSXQOXEGptao9UL6hdGl87\\\\\\\/ibdAr3SpR6uPfNzfYc4bTJRJkpQfrdXHYFrPDiQi7Rqv7bqK7UU6Bggt15RXJyXFiyOUkWcEklJeDDgn+YA+5wdTKSO5OPDNarW7MySWikBK+ZdS3RK6QVvZX\\\\\\\/bmfNW6b2wu+1miX4gUsvX58h7f0wyl7jxjskLZmf\\\\\\\/YBCtZx59HDpQfwz2PHPiOGAj7X95jhceKmAG5HEGrv6MOnlY7ZEtPh2aeqahINBx851LyK2iEcdw5GB4qrptab0sEeA\\\\\\\/IFo3IjvXn9hBivzh9yAmFdyvY\\\\\\\/PBzAs51RozxnvZnbWeoY5tS19rO91QEHupbOrwIX1Ar3LK\\\\\\\/mby3VvBGUPsU4Bj0D\\\\\\\/PAzSVM6w6Q277FopWpmc9jzAI5fwHxEAqih\\\\\\\/2iaIux2PqGHfHPZYxngpZxatdFZHCP1mq2Y5kv\\\\\\\/WiMjqcOj+\\\\\\\/rZh3OLlRv8R7dj6CCrkgpRFFPhQN2JquhuzcNKeYYi1jfLC8nRMOwq5hHhNLNdd1eewn714NuizdMSCxZ5vbw+NnA7idRcVvt5tiSvbsU846LYYukdNVKo31r8z2Ll6S5OW4Of3jvdKtaEElUarwqdg7Cy0WsN4JaDzGsUvKQhxW5CBzyTDB4UUMDmqZLfjCmTM0DnF1\\\\\\\/lQ4Kk+q\\\\\\\/Tst4fRiHE5OKQnfn7t4msc4i0Xa1MFMkIr9HbeJ5GKVieVeSFbK9ZrNeXClE33s0YrirdjwVKvWE\\\\\\\/GrNBZxOew+NhCddOJl6qyPIIF15ys+TBN3HHoMvIGqR2WXWaHROziYb\\\\\\\/HND0hLp0SfpiYsmOoGUrJX2wr18HSFTypN3Ga3AiPTpo9VfHQjB7AulLNN2uEBr+uQahPuMTWtgnDXnwgonlYORIju2h6D+5dpldUNJi0urFg6aSrD8VrnZuSE2aXM3DK+Gy1VgN+8guXRetnPkemMXIMvoCpvbdIZ\\\\\\\/0nmsBgfiTOrBmayCgsnwp6Uu2nKT7fSndhNK6Y04P3jW4notJHzyfO99Piw6mzs3rgqhq+DvzseHRdZrOTVxxWEGbK+ck09ZhUsYtWTflniojSaWdKC2R2dZoWjhpoy0ZHL4tk1FpojSZqnmIX+PFoXsnBeRWlaM09qK3nqgh0AO7J86diLZ13St\\\\\\\/wu+r43J2rhXtv1L4Z\\\\\\\/tEqt5JnSBWJNmG5m2wQ2qwroFHsMhoTJUR+FWuFs+07v9ZwkCGOq7hBYm9c2q6e4OayeLpxf+ZXbwRSPWGwQ5b+tlD07EX2PdDbSZsiTeOj01WrRkWOEoBl3C\\\\\\\/2rmIPFuzgEansk79XpZFNnq2R7XRgNfY\\\\\\\/1aBN+U2+GU\\\\\\\/Y66cGr+VC7H\\\\\\\/oyym6iMwrFI1AIqkMDTXIEClWP++aBdVValwLfpn+POLbuoq5Ro5u0MSwuAHrAoPcEGqs8SdtXY23fz7wJY7x8xH\\\\\\\/oKKMcVxLdI1iZkLnsYPnikJzFUq4iiXvA3kdvIrqxEnPuATFUiaj1HY7S59z74IAFjEAK99\\\\\\\/aZchFBtyVUMsfarQNtWFT5M2YiuRtfWGG9AdtmM2C1TuFZTQVuVdp+NKErTyotctA==\\\u0022,\\\u0022iv\\\u0022:\\\u00222RU4+wuFdI+WNy4jXOLWCA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225c75b2f26091d9022626ac1d29960020d29ce690276de012795a0ef1590a8eab\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-483525 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/3/935" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YgUmgrXkpwo20wzIhowjaVUG0VIYBzbZafxV-96-Y3k" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/3/935" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ty0owCcIMRr9IFEeZacc2TCOp9ZuzAftckOj1aQ0daU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/3/923" title="CRISPR-Cas-Induced Mutants Identify a Requirement for dSTIM in Larval Dopaminergic Cells of Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/3/943" title="Reproduction and In-Depth Evaluation of Genome-Wide Association Studies and Genome-Wide Meta-analyses Using Summary Statistics" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483525" id="node483525" data-pisa="ggg;7/3/935" data-pisa-master="ggg;g3.116.037895" data-apath="/ggg/7/3/935.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Single-Step BLUP with Varying Genotyping Effort in Open-Pollinated <em>Picea glauca</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4469-2929" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Blaise Ratcliffe</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-9112-8522" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Omnia Gamal El-Dien</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6234-2263" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Eduardo P. Cappa</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-9344-6348" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ilga Porth</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-5504-3735" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jaroslav Klápště</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-2203-0433" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Charles Chen</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-4887-8977" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yousry A. El-Kassaby</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">935-942;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.037895 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483525" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Blaise Ratcliffe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, British Columbia V6T 1Z4, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Blaise%2BRatcliffe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ratcliffe%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABlaise%2BRatcliffe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4469-2929" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Blaise Ratcliffe</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Omnia Gamal El-Dien </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, British Columbia V6T 1Z4, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Pharmacognosy Department, Faculty of Pharmacy, Alexandria University, Alexandria, 21521, Egypt</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Omnia%2BGamal%2BEl-Dien%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=El-Dien%20OG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOmnia%2BGamal%2BEl-Dien%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9112-8522" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Omnia Gamal El-Dien</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Eduardo P. Cappa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Instituto Nacional de Tecnología Agropecuaria (INTA), Instituto de Recursos Biológicos, Las Cabañas y De Los Reseros s/n, 1686 Hurlingham, Buenos Aires, Argentina</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eduardo%2BP.%2BCappa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cappa%20EP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEduardo%2BP.%2BCappa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6234-2263" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Eduardo P. Cappa</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ilga Porth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Départment des Sciences du Bois et de la Forêt, Faculté de Foresterie, de Géographie et Géomatique, Université Laval Québec, G1V 0A6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ilga%2BPorth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Porth%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIlga%2BPorth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9344-6348" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ilga Porth</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jaroslav Klápště </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Genetics and Physiology of Forest Trees, Faculty of Forestry and Wood Sciences, Czech University of Life Sciences Prague, 165 21 Praha 6, Czech Republic</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Scion (New Zealand Forest Research Institute Ltd.), Whakarewarewa, Rotorua 3046, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaroslav%2BKl%C3%A1p%C5%A1t%C4%9B%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kl%C3%A1p%C5%A1t%C4%9B%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJaroslav%2BKl%25C3%25A1p%25C5%25A1t%25C4%259B%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5504-3735" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jaroslav Klápště</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Charles Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Biochemistry and Molecular Biology, Oklahoma State University, Stillwater, Oklahoma 74078-3035</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharles%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2203-0433" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Charles Chen</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Yousry A. El-Kassaby </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, British Columbia V6T 1Z4, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yousry%2BA.%2BEl-Kassaby%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=El-Kassaby%20YA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYousry%2BA.%2BEl-Kassaby%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4887-8977" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yousry A. El-Kassaby</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:y.el-kassaby@ubc.ca" class="" data-icon-position="" data-hide-link-title="0">y.el-kassaby@ubc.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/3/935" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:483525" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:483525/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/3/935.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:483525" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:483525/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/3/935.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:483525" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:483525/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/3/935.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:483525" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:483525/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/3/935.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:483525" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:483525/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Maximization of genetic gain in forest tree breeding programs is contingent on the accuracy of the predicted breeding values and precision of the estimated genetic parameters. We investigated the effect of the combined use of contemporary pedigree information and genomic relatedness estimates on the accuracy of predicted breeding values and precision of estimated genetic parameters, as well as rankings of selection candidates, using single-step genomic evaluation (HBLUP). In this study, two traits with diverse heritabilities [tree height (HT) and wood density (WD)] were assessed at various levels of family genotyping efforts (0, 25, 50, 75, and 100%) from a population of white spruce (<em>Picea glauca</em>) consisting of 1694 trees from 214 open-pollinated families, representing 43 provenances in Québec, Canada. The results revealed that HBLUP bivariate analysis is effective in reducing the known bias in heritability estimates of open-pollinated populations, as it exposes hidden relatedness, potential pedigree errors, and inbreeding. The addition of genomic information in the analysis considerably improved the accuracy in breeding value estimates by accounting for both Mendelian sampling and historical coancestry that were not captured by the contemporary pedigree alone. Increasing family genotyping efforts were associated with continuous improvement in model fit, precision of genetic parameters, and breeding value accuracy. Yet, improvements were observed even at minimal genotyping effort, indicating that even modest genotyping effort is effective in improving genetic evaluation. The combined utilization of both pedigree and genomic information may be a cost-effective approach to increase the accuracy of breeding values in forest tree breeding programs where shallow pedigrees and large testing populations are the norm.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/bivariate-mixed-model" class="hw-term hw-article-keyword hw-article-keyword-bivariate-mixed-model" rel="nofollow">bivariate mixed model</a></li><li class="kwd"><a href="/keyword/hblup" class="hw-term hw-article-keyword hw-article-keyword-hblup" rel="nofollow">HBLUP</a></li><li class="kwd"><a href="/keyword/single-step-blup" class="hw-term hw-article-keyword hw-article-keyword-single-step-blup" rel="nofollow">single-step BLUP</a></li><li class="kwd"><a href="/keyword/tree-improvement" class="hw-term hw-article-keyword hw-article-keyword-tree-improvement" rel="nofollow">tree improvement</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li></ul><p id="p-3">Maximizing genetic gain in tree breeding programs depends on the reliability of the estimated genetic parameters. Traits’ heritabilities and their genetic correlations are among the most critical genetic parameters determining potential genetic progress, therefore their precision is of great importance. Progeny testing is the vehicle by which these genetic parameters are estimated, while individual tree data, family composition, pedigree depth and connectedness, and field performance determine their reliability (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Huber <em>et al.</em> 1992</a>). Open-pollinated testing is often used to efficiently screen and assess large numbers of candidate trees (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Burdon and Shelbourne 1971</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Jayawickrama <em>et al.</em> 2000</a>). However, it is recognized that treating open-pollinated families as “half-sibs” is inaccurate due to the effect of hidden relatedness, which cannot be easily unraveled in traditional pedigree-based analyses, thus affecting the resulting genetic parameters and candidate rankings (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Squillace 1974</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Askew and El-Kassaby 1994</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Namkoong <em>et al.</em> 2012</a>).</p><p id="p-4">The use of molecular markers can uncover hidden relatedness and potential pedigree errors in open-pollinated populations via pedigree reconstruction and paternity assignment (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wang 2004</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kalinowski <em>et al.</em> 2007</a>). Pedigree reconstruction has been effectively used to determine the genealogical relationship among groups of related individuals leading to estimating genetic parameters with increased precision (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">El-Kassaby and Lstibůrek 2009</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Doerksen and Herbinger 2010</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">El-Kassaby <em>et al.</em> 2011</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Koreckỳ <em>et al.</em> 2013</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Klápště <em>et al.</em> 2014</a>). <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Ritland (1996)</a> proposed an innovative approach where marker-based relationship estimates could be used for estimating traits’ heritabilities and demonstrated the method’s utility in wild populations. Capitalizing on the availability of dense genomic marker sets, <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">VanRaden (2008)</a> derived estimates of marker-based relationships between pairs of individuals as a genomic relationship matrix (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-1.gif"/></noscript></span></span>), which can be used as a substitute to the traditional pedigree-based average numerator relationship matrix (<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-2.gif"/></noscript></span></span>) in Henderson’s animal model (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Henderson 1984</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Frentiu <em>et al.</em> 2008</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Habier <em>et al.</em> 2013</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Klápště <em>et al.</em> 2014</a>)</p><p id="p-5">The advantages of using marker-based relationship estimates are: (1) bypassing the classical mating designs needed for generating structured pedigree (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">El-Kassaby <em>et al.</em> 2011</a>; <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Klápště <em>et al.</em> 2014</a>), (2) applicability to natural and experimental populations (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">El-Kassaby <em>et al.</em> 2012</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Porth <em>et al.</em> 2013</a>; <a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Klápště <em>et al.</em> 2014</a>), and (3) increased precision of genetic parameter estimates by accounting for genetic similarity due to common ancestry (<em>i.e.</em>, historical pedigree) (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Powell <em>et al.</em> 2010</a>) and Mendelian segregation (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Visscher <em>et al.</em> 2006</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Ødegård and Meuwissen 2012</a>).</p><p id="p-6">Combining the <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-3.gif"/></noscript></span></span> with the marker-based relationship matrix <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-4.gif"/></noscript></span></span> in a single genetic covariance matrix <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-5.gif"/></noscript></span></span> (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Legarra <em>et al.</em> 2009</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Misztal <em>et al.</em> 2009</a>), has proven to be an effective way to improve relationship coefficients for better estimation of genetic parameters and tracking of genetic diversity in animal breeding systems. This method was dubbed “HBLUP,” since the best linear unbiased predictors (BLUPs) of breeding values are derived using the combined, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-6.gif"/></noscript></span></span> genetic covariance matrix. The HBLUP method may be particularly useful in forest tree progeny test evaluation as the cost and logistics of genotyping the entirety of trials is prohibitive; thus, combining the marker-based relationship from a subset of trees from some families from few sites with the pedigree-based relationship estimator would be an efficient option for improving the derived genetic parameters’ precision for the entire test.</p><p id="p-7">The aim of this study is to compare the traditional ABLUP approach to the HBLUP approach by comparison of the precision of genetic parameters, accuracy of breeding values, and rankings of selection candidates using variable proportions (0, 25, 50, 75, and 100%) of genotyped individuals within the open-pollinated families.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>White spruce open-pollinated progeny test, phenotype data, and genotyping</h3><p id="p-8">Data for the present study was obtained from <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Beaulieu <em>et al.</em> (2014)</a> and downloaded from the Dryad Digital Repository: doi 10.5061/dryad.6rd6f. Briefly, this study concerns a single white spruce [<em>Picea glauca</em> (Moench) Voss] provenance-progeny test site that was established in 1979 in Québec, Canada. The test site contains 214 open-pollinated families from 43 Québec provenances planted in randomly assigned five tree row plots at 2.4 m between row and 1.2 m within row spacing to attain a randomized block design with six blocks. A subset of 1694 trees were sampled from the site, the mean number of trees per family was ∼7.9, and the mean number of trees per family per block was ∼1.3. Complete details of the test site are available in <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Beaulieu <em>et al.</em> (2014)</a>. Phenotypic data for this study included 22 yr tree HT (cm) and WD (kg/m<sup>3</sup>) obtained via X-ray densitometry.</p><p id="p-9">Single-nucleotide polymorphisms (SNPs) of the 1694 trees were genotyped using Illumina Infinium HD iSelect bead chip PgAS1 (Illumina, San Diego, CA) (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Beaulieu <em>et al.</em> 2014</a>). The SNPs were located within 2814 genes and separated by a minimum distance of 200 bp. Genes were selected from the white spruce gene catalog “GCAT” (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Rigault <em>et al.</em> 2011</a>) and were those of diverse functions (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Pavy <em>et al.</em> 2013</a>). A total of 6716 SNPs were available for use in the analyses.</p></div><div id="sec-3" class="subsection"><h3>Data analyses</h3><p id="p-10">Traditional pedigree-based genetic analyses were performed under the assumptions of unrelated half-sibling family structure and a minimal population structure effect, as shown by <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Beaulieu <em>et al.</em> (2014)</a>. Variance components, heritabilities, genetic correlations, and individual tree breeding values were estimated with the bivariate animal model (herein referred to as “ABLUP”) (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Henderson 1984</a>) as follows:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-7.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-8.gif"/></noscript></span></span> is the response matrix of individually scaled and centered phenotypic observations for the two traits; <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-9.gif"/></noscript></span></span> is the incidence matrix for the fixed effect <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-10.gif"/></noscript></span></span> (trait means); <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-11.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-12.gif"/></noscript></span></span> are the corresponding incidence matrices related to random additive genetic effects [breeding values, <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-13.gif"/></noscript></span></span>] and block effects [<span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-14.gif"/></noscript></span></span>]; and the random residual error effects are distributed as <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-15.gif"/></noscript></span></span> The covariance matrix for the random additive genetic effects was modeled using the heterogeneous covariance structure as <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-16.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-17.gif"/></noscript></span></span> is the average numerator relationship matrix, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-18.gif"/></noscript></span></span> is the additive covariance between traits 1 and 2, and <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-19.gif"/></noscript></span></span> is the Kronecker product operator. The covariance matrix for the random block effects was modeled using diagonal structure as <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-20.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-21.gif"/></noscript></span></span> is an identity matrix. The random residual error effect was modeled using an unstructured covariance matrix structure as <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-22.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-23.gif"/></noscript></span></span> is the residual covariance between the two traits. Random effects were assumed to be independent.</p><p id="p-11">The genomic relationship matrix (GRM) (<span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-24.gif"/></noscript></span></span>) was constructed after <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">VanRaden (2008)</a>:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-25.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>let <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-26.gif"/></noscript></span></span> be the number of individuals and <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-27.gif"/></noscript></span></span> be the number of markers, then <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-28.gif"/></noscript></span></span> is an <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-29.gif"/></noscript></span></span> matrix of centered genotype scores with missing values replaced with zeros (<em>i.e.</em>, mean imputation), and <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-30.gif"/></noscript></span></span> is the reference allele frequency of the <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-31.gif"/></noscript></span></span> marker.</p><p id="p-12">In the single-step model (herein referred to as “HBLUP”), the <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-32.gif"/></noscript></span></span> matrix was substituted by the combined additive relationship matrix <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-33.gif"/></noscript></span></span> (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Legarra <em>et al.</em> 2009</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Christensen and Lund 2010</a>), resulting from accommodation of the genomic marker-based relationship matrix <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-34.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-35.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-36.gif"/></noscript></span></span> is the block of <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-37.gif"/></noscript></span></span> for the nongenotyped individuals of the population, <strong><em>A</em><sub>22</sub></strong> is the block of genotyped individuals, and <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-38.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-39.gif"/></noscript></span></span> are the blocks containing the expected additive genetic relationships between genotyped and nongenotyped individuals. The inverse of <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-40.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-41.gif"/></noscript></span></span>) can be obtained via (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aguilar <em>et al.</em> 2010</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Christensen and Lund 2010</a>):<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-42.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-43.gif"/></noscript></span></span> is the inverse of the pedigree-based relationship matrix <strong><em>A</em></strong>, and <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-44.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-45.gif"/></noscript></span></span> are the pedigree and the genomic relationship matrices for the genotyped individuals, respectively.</p><p id="p-13">Prior to calculation of <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-46.gif"/></noscript></span></span> the GRMs were scaled as <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-47.gif"/></noscript></span></span> such that the average of diagonal and off-diagonal elements of <strong><em>G</em></strong> were equivalent to those of <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-48.gif"/></noscript></span></span> by solving the following system of equations for the parameters <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-49.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-50.gif"/></noscript></span></span> (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Christensen <em>et al.</em> 2012</a>).<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-51.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>To avoid potential problems with inversion of <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-52.gif"/></noscript></span></span> it was weighted as (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Aguilar <em>et al.</em> 2010</a>):<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-53.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>Five levels of genotyping effort were tested in the HBLUP method: 0, 25, 50, 75, and 100%. In the case of 0% of genotyping effort, variance components, genetic correlations, and breeding values were obtained strictly using expected additive genetic relationships defined in <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-54.gif"/></noscript></span></span> (<em>i.e.</em>, ABLUP). At 100% genotyping effort, the <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-55.gif"/></noscript></span></span> matrix used in Equation 4 containing all possible genotyped individuals was combined with the <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-56.gif"/></noscript></span></span> matrix. In this last case, <strong><em>A</em></strong><sub>11</sub> represents only the unrelated and nongenotyped maternal parents (<em>i.e.</em>, a diagonal square matrix of order 214). To avoid possible sampling bias, a replicated resampling scheme was used to obtain parameter estimates at 25–75% genotyping effort. In detail, two, four, or six individuals per family, of a possible maximum of eight, were randomly removed from the <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-57.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-58.gif"/></noscript></span></span> matrices prior to rescaling and combining with <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-59.gif"/></noscript></span></span> in Equation 4. This process was repeated 30 times at each level of genotyping effort and mean values for the results are reported.</p><p id="p-14">The restricted maximum likelihood (REML) was used to estimate variance and covariance for the random effects in the bivariate mixed model (1), and were obtained with the ASReml-R v3.0 program (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Butler <em>et al.</em> 2009</a>).</p><p id="p-15">The single-trait narrow-sense heritability for of each trait from the bivariate ABLUP and HBLUP models were estimated as:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-60.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>where <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-61.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-62.gif"/></noscript></span></span> are the estimates of additive genetic and residual variances, respectively (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Falconer and Mackay 1996</a>).</p><p id="p-16">The additive genetic correlation between the two traits was estimated as Pearson’s product moment:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-63.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>where, <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-64.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-65.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-66.gif"/></noscript></span></span> are the genetic covariance and additive genetic variances for traits 1 and 2 respectively, estimated from the bivariate model (1) (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Falconer and Mackay 1996</a>).</p><p id="p-17">The theoretical accuracy of breeding values (<span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-67.gif"/></noscript></span></span>) was estimated after (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dutkowski <em>et al.</em> 2002</a>):<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-68.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span>where <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/embed/mml-math-69.gif"/></noscript></span></span> is the standard error, and <em>F<sub>i</sub></em> is the inbreeding coefficient of the <em>i</em>th individual. SEs of the breeding values were obtained from the bivariate model with variance components fixed to estimates from the same model with 100% genotyping effort for the respective traits, under the assumption that these were most accurate.</p><p id="p-18">Additionally, the Akaike information criterion (AIC) was computed to compare the fit of the bivariate ABLUP and HBLUP models. A smaller AIC value indicates better fit. Furthermore, Spearman rank correlations and the proportion of common candidates in the top 5% of mothers and progenies were also calculated to compare whether the predicted breeding values differed among the ABLUP and HBLUP models.</p></div><div id="sec-4" class="subsection"><h3>Data availability</h3><p id="p-19">The data used are available from the Dryad Digital Repository: doi:10.5061/dryad.6rd6f (<a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Beaulieu <em>et al.</em> 2014</a>). </p></div></div><div class="section results" id="sec-5"><h2 class="">Results</h2><div id="sec-6" class="subsection"><h3>Relationship matrices</h3><p id="p-20">Descriptive statistics for the expected (<strong><em>A</em></strong>) and combined (<strong><em>H</em></strong>) relationship estimators presented the ranges of the <strong><em>H</em></strong> relationship groups to be larger than that of <strong><em>A</em></strong> (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). As expected, the Pearson correlation between <strong><em>A</em></strong> and the <strong><em>H</em></strong> relationship matrices decreased with an increase in genotyping effort. The ranges of the values within the various relationship groups also tended to expand with increasing genotyping effort. Particularly noteworthy was the presence of non-zero relationship coefficients between maternal parents given by the <strong><em>H</em></strong> relationship estimators, indicating that maternal parents are not completely unrelated as assumed in the pedigree. Likewise, large differences from the expected pairwise half-sib and unrelated values in <strong><em>A</em></strong> were observed in the <strong><em>H</em></strong> relationship estimators. These deviations imply the possible presence of pedigree errors on one side, and full-sibs within the half-sib families on the other side.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/483622/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/483622/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Descriptive statistics of the pedigree-based (<em>A</em>) and combined pedigree marker-based (<em>H</em>) relationship estimators at 0, 25, 50, 75, and 100% genotyping effort</span><div class="sb-div caption-clear"></div></div></div><p id="p-23">Family and provenance structure was visually evident in the heat maps of the relationship estimators (see Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037895/-/DC1/FigureS1.tiff">Figure S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037895/-/DC1/FigureS2.tiff">Figure S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037895/-/DC1/FigureS3.tiff">Figure S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037895/-/DC1/FigureS4.tiff">Figure S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037895/-/DC1/FigureS5.tiff">Figure S5</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037895/-/DC1/FileS1.docx">File S1</a>). Samples were sorted according to family and provenance, then the pairwise relationship coefficients of the matrices were plotted. Historical relationships (<em>i.e.</em>, provenance) not captured by the pedigree estimator <strong><em>A</em></strong> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037895/-/DC1/FigureS1.tiff">Figure S1</a>) were visually apparent in <strong><em>H</em></strong> with as little as 25% genotyping effort (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037895/-/DC1/FigureS2.tiff">Figure S2</a>). Further, potential pedigree errors not accounted for in the contemporary pedigree were uncovered, these errors may be due to mislabeling or technical errors in the lab. Population structure and potential pedigree errors were further evidenced by the large deviations observed between the ranges maternal parent–parent, half-sib, and unrelated relationship coefficients between <strong><em>A</em></strong> and the <strong><em>H</em></strong> relationship estimators (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>).</p></div><div id="sec-7" class="subsection"><h3>Variance components</h3><p id="p-24">The additive genetic variance estimates obtained from the bivariate HBLUP models were substantially lower than the bivariate ABLUP model estimates for both HT and WD (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Further, the additive genetic component estimates of the HBLUP models continually decreased as a result of increased genotyping effort. It should be noted that the observed differences in additive genetic variance estimates between the ABLUP and HBLUP models reflects on the known bias associated with considering wind-pollinated offspring as half-sib progeny, which often results in the overestimation of additive genetic variance and subsequently incorrect parental and progeny rankings. This bias is attributable to hidden relationships in assumed open-pollinated half-sib pedigrees.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/483686/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/483686/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Pedigree-based (<em>A</em>) and combined pedigree marker-based (<em>H</em>) estimates of variance components for white spruce tree height and wood density at 0, 25, 50, 75, and 100% genotyping effort</span><div class="sb-div caption-clear"></div></div></div><p id="p-27">Narrow-sense heritability estimates of the bivariate HBLUP models mirrored those of the additive genetic variance estimates (HT: from 0.178 to 0.295; WD: from 0.310 to 0.594). These estimates at 100% genotyping effort for HT and WD were intermediate to the polygenic and marker-based estimates described by <a id="xref-ref-3-6" class="xref-bibr" href="#ref-3">Beaulieu <em>et al.</em> (2014)</a>. The trend observed in the estimates of additive genetic variance was coupled with an increase in the residual variance component for the HBLUP models compared to the ABLUP model, with the lowest estimates observed for the ABLUP model, indicating that some residual variance was shifted to the additive variance, hence the overestimation of heritability (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Comparison of the block variance estimates provided no pattern of association with any factor, and the estimates were relatively stable across all models for both WD and HT. Across all the models, the additive genetic correlations between HT and WD were minor, negative, and not significant (based on SEs). The variation in model parameters’ estimates under 25, 50, and 75% genotyping effort, due to random sampling, was low, indicating that the selection of genotyped individuals within family has minimal impact (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037895/-/DC1/TableS1.txt">Table S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037895/-/DC1/TableS1.txt">Table S2</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037895/-/DC1/TableS1.txt">Table S3</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037895/-/DC1/FileS1.docx">File S1</a>).</p><p id="p-28">Large incremental improvements in the model fit, determined by AIC, were observed for the HBLUP models over the ABLUP model with increasing genotyping effort. Model fit was optimal in the case with complete genotyping effort (100%), using the <strong><em>H</em></strong> relationship estimator (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>).</p></div><div id="sec-8" class="subsection"><h3>Breeding value accuracy</h3><p id="p-29">The mean breeding value accuracies for HT and WD are presented for three classes of individuals: maternal parents and genotyped and nongenotyped progeny (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Comparison of <strong><em>H</em></strong> and <strong><em>A</em></strong> relationship estimators on mean breeding value accuracies for the maternal parents showed minor increases for HT (differences were limited to only the second decimal place; from 0.520 to 0.539) and no changes for WD (from 0.635 to 0.635) with increasing family genotyping efforts. The mean breeding value accuracy of nongenotyped progeny using the <strong><em>H</em></strong> estimator across all levels of genotyping effort was stable and equivalent to the mean breeding value accuracy of the progeny using the <strong><em>A</em></strong> estimator, for both HT and WD. As expected, the largest improvements over the <strong><em>A</em></strong> relationship estimator were noted for genotyped progeny using the <strong><em>H</em></strong> estimators. Increasing genotyping effort produced large increases in mean breeding value accuracy for both HT and WD over mean breeding value accuracies of the progeny using <strong><em>A</em></strong> (HT: 0.474; WD: 0.605), with the effect greater for HT (HT: from 0.498 to 0.536; WD: from 0.626 to 0.661).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/483571/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/483571/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Mean breeding value accuracy estimates (<span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/T3/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/935/T3/embed/mml-math-86.gif"/></noscript></span></span>) for white spruce tree height and wood density using pedigree-based (<em>A</em>) and combined pedigree marker-based (<em>H</em>) relationship estimators at 0, 25, 50, 75, and 100% genotyping effort for three classes of individuals: maternal parent, genotyped progeny, and nongenotyped progeny</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Candidate rankings</h3><p id="p-31">The Spearman rank correlations for maternal and progeny breeding values, and the proportion of common candidates in the top 5% of maternal and progeny rankings, between the <strong><em>A</em></strong> and <strong><em>H</em></strong> relationship estimator at 25, 50, 75, and 100% genotyping effort, for both HT and WD, are presented in <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>. The pairwise Spearman rank correlations were always less than perfect indicating at least some disagreement in the rankings of candidate maternal parents and progeny. Overall, maternal and progeny Spearman rank correlations between the HBLUP models and the ABLUP model were greater for WD than HT, further suggesting that genomic information was less impactful on candidate rankings for the higher heritability trait. Furthermore, the Spearman rank correlations between the ABLUP model and the HBLUP models decreased with increasing genotyping effort in all cases. The top 5% analysis again showed overall better agreement for WD over HT for both maternal and progeny candidates. Furthermore, agreement with the top 5% ABLUP candidates decreased with increasing genotyping effort in the HBLUP models. In summary, the Spearman rank correlations and the proportion of common candidates in the top 5% suggests that the inclusion of genomic information affects candidate rankings and has the potential to alter both forward and backward selections.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/483638/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/483638/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Proportion of common candidates in the top 5% of maternal and progeny rankings (above diagonals), and estimated Spearman breeding value rank correlations (diagonals and below diagonals), for pedigree-based (<em>A</em>) and combined pedigree marker-based (<em>H</em>) relationship estimators at 0, 25, 50, 75, and 100% genotyping effort for white spruce tree height and wood density</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-10"><h2 class="">Discussion</h2><p id="p-34">The investigation of genetic architecture in forest trees requires the establishment of large scale testing populations with predefined mating design. The results’ quality (<em>i.e.</em>, accuracy and precision of genetic parameters) are often a reflection of the extent of efforts invested into the design of the field experiment. Commonly, quantitative genetics analyses are based on the animal model implemented in mixed model theory (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Henderson 1984</a>). Such analyses are based on the average numerator relationship matrix tracking probability of identity by descent (<strong><em>A</em></strong> matrix). The mixed model theory operates with the assumption that variance–covariance matrices of random terms used in the animal model are error free (<a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Henderson 1984</a>). The average numerator relationship matrix cannot fulfill this assumption as it is based on the pairwise expected relationship values and does not account for the Mendelian sampling term. Moreover, the contemporary shallow pedigrees of many forest tree breeding programs are unable to account for historical coancestry. The use of genotypic data enables greater insight into genetic covariances and the precise mapping of the Mendelian sampling term (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Visscher <em>et al.</em> 2006</a>). The use of marker-based relationship matrix analyses can be beneficial even in small populations with limited pedigree information (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">El-Kassaby <em>et al.</em> 2012</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Ødegård and Meuwissen 2012</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Porth <em>et al.</em> 2013</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cappa <em>et al.</em> 2016</a>)</p><p id="p-35">However, the cost and logistics of obtaining genotypic information for the large testing populations in tree improvement programs still represents a barrier to widespread implementation of this methodology (<a id="xref-ref-3-7" class="xref-bibr" href="#ref-3">Beaulieu <em>et al.</em> 2014</a>). Additionally, deployment of genetic analysis using the entire testing population (<em>i.e.</em>, genotyped and nongenotyped individuals) is important to eliminate bias connected to the total reliance on using genotyped individuals only or error-prone multi-step methods. While the genomic information benefits from lower inbreeding build-up during the selection phases as compared to pedigree-based BLUP (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Liu <em>et al.</em> 2014</a>) due to the added knowledge about the Mendelian sampling term and historical coancestry, it is recommended that including nongenotyped individuals in the same data analysis is beneficial as it effectively reduces the bias associated with selection of individuals for genotyping (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Ducrocq and Patry 2010</a>). This is increasingly important, especially in forest tree breeding programs, which are based on shallow pedigree selection from large base populations. The HBLUP method combining both pedigree- and marker-based relationship matrices generally provides as good as, or better, results over pedigree-based BLUP (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Christensen <em>et al.</em> 2012</a>) or multi-step approaches (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Vitezica <em>et al.</em> 2011</a>).</p><p id="p-36">By permitting both pedigree and marker information in a single genetic evaluation analysis, the combined HBLUP approach offers resolution to these problems. Information gained through genomic relationship estimates for a subset of genotyped individuals is reflected via pedigree relationships to the nongenotyped individuals in the relationship matrix <strong><em>H</em></strong> (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Legarra <em>et al.</em> 2009</a>). Thus, more accurate estimates of relatedness are obtained in a cost-efficient manner. Our results support this notion through large improvement in the model fit as represented by the AIC statistics from the exclusive pedigree-based model to models utilizing the combination of marker and pedigree information at various genotyping efforts (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). Furthermore, the presence of hidden relatedness, particularly in open-pollinated trials, can be a serious problem in their genetic evaluation resulting in upwardly biased estimates of heritability (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Squillace 1974</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Askew and El-Kassaby 1994</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Namkoong <em>et al.</em> 2012</a>) (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). This bias is caused by overestimation of additive genetic variance due to the unrealistic assumption of pure half-sibling relatedness within the open-pollinated families, as well as complete unrelatedness among the parental donors. Reality and empirical evidence suggests the existence of hidden relationships within these families (<em>i.e.</em>, full-sibs, half-selfs, and full-selfs) (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Squillace 1974</a>; <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Askew and El-Kassaby 1994</a>; <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Namkoong <em>et al.</em> 2012</a>; Gamal El-Dien <em>et al.</em> 2016). Our results confirm that hidden relatedness within the OP families and historical coancestry exists in this <em>P. glauca</em> population, and was better accounted for through the HBLUP method, with increasing genotyping effort helping to approach more realistic estimates of genetic parameters and better model fit as compared to the analysis performed on the complete pedigree alone (<a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>). It should be stated that all hidden relatedness can only be accounted for under the GBLUP analysis, where individuals are genotyped (including parents); however, the HBLUP is composed of a mixture of genotyped and nongenotyped individuals, thus a slight overestimation of the additive genetic variance is expected. The degree of additive variance overestimation should lie between the GBLUP and pedigree analyses (<a id="xref-ref-3-8" class="xref-bibr" href="#ref-3">Beaulieu <em>et al.</em> 2014</a>).</p><p id="p-37">The accuracy of the estimated breeding values is of practical importance to tree breeders. Correct ranking of the selection candidates based on accurate estimates of relatedness is an important factor. While traditional pedigree-based analyses have been proven to deliver increased genetic gain, a reduction in the size of the testing population can be achieved with improvement in accuracy through the use of genetic markers in the evaluation. The comparison of breeding value accuracies in this study was based on the assumption that the variance component estimates produced by the bivariate HBLUP model with full genotyping effort were most accurate. We fixed the variance components of each model to these estimates prior to calculating theoretical breeding value accuracy for each scenario. Our results show incremental improvement in the accuracy of progeny breeding values for the genotyped subset for both HT and WD with increasing genotyping effort (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). However, this improvement was not translated to the nongenotyped progeny subset in either trait. The simple, disconnected pedigree structure of this population is a probable explanation for this observation, as the genomic information is only transferred from genotyped to nongenotyped individuals via pedigree relationships. We expect that in systems with more complex and interconnected pedigree structure, such as diallel mating designs, multi-generational complex pedigrees will benefit more from the HBLUP method as genomic information can be inferred from multiple family sources to the nongenotyped individuals.</p><p id="p-38">Furthermore, family or maternal parent breeding value accuracy remained constant across all levels of genotyping effort for the trait with greater heritability, WD, whereas slight increases were observed for the lower heritability trait, HT, at 25, 50, 75, and 100% genotyping effort (from 0.525 to 0.539). The difference in contrasting heritabilities of the traits and these findings agree with previous discussions that genomic selection is expected to be more efficient in low heritability traits (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Calus <em>et al.</em> 2008</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Grattapaglia and Resende 2011</a>). Yet, the inclusion of even a small proportion (<em>i.e.</em>, 25%) of genomic information per family produced differences in ranking the top 5% of both maternal and progeny candidates for both traits (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>).</p><p id="p-39">The accuracy of estimated breeding values in HBLUP is not a function of only average diagonal and off-diagonal elements in the <strong><em>G</em></strong> matrix, but also the difference between them as it reflects the level of relatedness in the testing population (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Forni <em>et al.</em> 2011</a>). Generally, the genomic relationship matrix uses actual allele frequencies instead of those of the base population, which are usually unknown (in this case, the base population is treated as the studied pedigree population), and sets the genomic-base population as the genotyped population (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Oliehoek <em>et al.</em> 2006</a>). However, such analysis is affected by incompatibility between pedigree- and marker-based relationship matrices due to the inferences made about the base population using information from the studied population. <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Powell <em>et al.</em> (2010)</a> highlighted the importance of scaling these matrices by correcting the genomic relationship matrix by a factor reflecting the mean difference between the <strong><em>A</em></strong> and <strong><em>G</em></strong> matrices. Several approaches were proposed to rescale the <strong><em>G</em></strong> matrix according to the <strong><em>A</em></strong> matrix (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Meuwissen <em>et al.</em> 2011</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Vitezica <em>et al.</em> 2011</a>; <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Christensen <em>et al.</em> 2012</a>); however, an opposite approach was also proposed by <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Christensen <em>et al.</em> (2012)</a>, whereby the <strong><em>A</em></strong> matrix is rescaled according to its <strong><em>G</em></strong> counterpart. To avoid incompatibility between the <strong><em>G</em></strong> and <strong><em>A</em></strong> matrices, a normalized <strong><em>G</em></strong> matrix containing the average of diagonal elements equal to one needs to be constructed. Such a matrix is efficient when the population is inbreeding free, and if inbreeding is present then inbreeding coefficients have to be included in the denominator (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Forni <em>et al.</em> 2011</a>). We chose to scale <strong><em>G</em></strong> according to <a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Christensen <em>et al.</em> (2012</a>); however, all analyses were tested without scaling <strong><em>G</em></strong> and yielded comparable results (results not shown). This result is likely due to the shallow pedigree structure and minor differences between the <strong><em>A</em></strong><sub>22</sub> and <strong><em>G</em></strong> matrices. Additionally, <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Vitezica <em>et al.</em> (2011)</a> observed discrepancies in the accurate genomic breeding values with incorrectly scaled <strong><em>G</em></strong>, but only under strong selection (<em>i.e.</em>, nonrandom) of genotyped individuals and when genotyping was across 10 generations. However, in the present study, the 25–75% of genotyped trees were randomly removed (<em>i.e.</em>, unselected) and all these trees came from only one generation.</p><p id="p-40">As a preliminary investigation into the combined use of genomic and pedigree information in the applied genetic analysis of white spruce, the HBLUP method has proven to be a beneficial tool to forest tree breeders. The inclusion of nongenotyped and genotyped trees in a single analysis produced improvements in breeding value accuracy and model fit, particularly for the trait with low heritability (HT). Further, discordance in candidate rankings between the HBLUP and the traditional method that utilizes <strong><em>A</em></strong> were observed. Improvement in the results were continuous with increasing genotyping effort; however, improvements were also seen at the minimum level of 25% genotyping effort.</p></div><div class="section" id="sec-11"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-41">We thank <a id="xref-ref-3-9" class="xref-bibr" href="#ref-3">Beaulieu <em>et al.</em> (2014)</a> for making the data available on the Dryad Digital Repository: doi:10.5061/dryad.6rd6f. This study was funded by the Johnson’s Family Forest Biotechnology Endowment, FPInnovations’ ForValueNet, a Natural Sciences and Engineering Research Council of Canada (NSERC) Discovery grant to Y.A.E.-K., and a National Science Foundation Major Research Instrumentation Program grant (NSF-MRI 1626257) to C.C. The authors declare no conflicts of interest.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-8"><p id="p-42">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037895/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037895/-/DC1</a>.</p></li><li class="fn" id="fn-9"><p id="p-43"><em>Communicating editor: D. Zamir</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-11-28"><span class="received-label">Received </span>November 28, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-01-12"><span class="accepted-label">Accepted </span>January 12, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Ratcliffe <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.1" data-doi="10.3168/jds.2009-2730"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsuruta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Hot topic: a unified approach to utilize phenotypic, full pedigree, and genomic information for genetic evaluation of Holstein final score.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">743</span>–<span class="cit-lpage">752</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DAguilar%26rft.auinit1%253DI.%26rft.volume%253D93%26rft.issue%253D2%26rft.spage%253D743%26rft.epage%253D752%26rft.atitle%253DHot%2Btopic%253A%2Ba%2Bunified%2Bapproach%2Bto%2Butilize%2Bphenotypic%252C%2Bfull%2Bpedigree%252C%2Band%2Bgenomic%2Binformation%2Bfor%2Bgenetic%2Bevaluation%2Bof%2BHolstein%2Bfinal%2Bscore.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2009-2730%26rft_id%253Dinfo%253Apmid%252F20105546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2009-2730&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20105546&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F935.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274102500032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Askew</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">El-Kassaby</span>  <span class="cit-name-given-names">Y. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Estimation of relationship coefficients among progeny derived from wind-pollinated orchard seeds.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D88%26rft.spage%253D267%26rft_id%253Dinfo%253Apmid%252F24185937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24185937&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F935.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.3" data-doi="10.1038/hdy.2014.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beaulieu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerksen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clément</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacKay</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bousquet</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Accuracy of genomic selection models in a large population of open-pollinated families in white spruce.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D113%26rft.spage%253D343%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2014.36%26rft_id%253Dinfo%253Apmid%252F24781808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2014.36&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24781808&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F935.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000342564000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burdon</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shelbourne</span>  <span class="cit-name-given-names">C. J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">Breeding populations for recurrent selection: conflicts and possible solutions.</span> <abbr class="cit-jnl-abbrev">N. Z. J. For. Sci.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">174</span>–<span class="cit-lpage">193</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BZ.%2BJ.%2BFor.%2BSci.%26rft.volume%253D1%26rft.spage%253D174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-7.3.935.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogel</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source"><em>ASReml-R Reference Manual</em>.</span> <span class="cit-publ-name">The State of Queensland, Department of Primary Industries and Fisheries</span>, <span class="cit-publ-loc">Brisbane, Australia</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.6" data-doi="10.1534/genetics.107.080838"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Roos</span>  <span class="cit-name-given-names">A. P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Accuracy of genomic selection using different methods to define haplotypes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">553</span>–<span class="cit-lpage">561</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCalus%26rft.auinit1%253DM.%2BP.%2BL.%26rft.volume%253D178%26rft.issue%253D1%26rft.spage%253D553%26rft.epage%253D561%26rft.atitle%253DAccuracy%2Bof%2BGenomic%2BSelection%2BUsing%2BDifferent%2BMethods%2Bto%2BDefine%2BHaplotypes%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080838%26rft_id%253Dinfo%253Apmid%252F18202394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc4LzEvNTUzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzkzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cappa</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klápště</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villalba</span>  <span class="cit-name-given-names">P. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poltri</span>  <span class="cit-name-given-names">S. N. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">SSRs, SNPs and DArTs comparison on estimation of relatedness and genetic parameters’ precision from a small half-sib sample population of Eucalyptus grandis.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D36%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.8" data-doi="10.1186/1297-9686-42-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic prediction when some animals are not genotyped.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DChristensen%26rft.auinit1%253DO.%2BF.%26rft.volume%253D42%26rft.spage%253D2%26rft.epage%253D2%26rft.atitle%253DGenomic%2Bprediction%2Bwhen%2Bsome%2Banimals%2Bare%2Bnot%2Bgenotyped.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-42-2%26rft_id%253Dinfo%253Apmid%252F20105297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-42-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20105297&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F935.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.9" data-doi="10.1017/S1751731112000742"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madsen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ostersen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Single-step methods for genomic evaluation in pigs.</span> <abbr class="cit-jnl-abbrev">Animal</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1565</span>–<span class="cit-lpage">1571</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAnimal%26rft.aulast%253DChristensen%26rft.auinit1%253DO.%2BF.%26rft.volume%253D6%26rft.issue%253D10%26rft.spage%253D1565%26rft.epage%253D1571%26rft.atitle%253DSingle-step%2Bmethods%2Bfor%2Bgenomic%2Bevaluation%2Bin%2Bpigs.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS1751731112000742%26rft_id%253Dinfo%253Apmid%252F22717310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S1751731112000742&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22717310&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F935.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308833700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doerksen</span>  <span class="cit-name-given-names">T. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herbinger</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Impact of reconstructed pedigrees on progeny-test breeding values in red spruce.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">591</span>–<span class="cit-lpage">600</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D6%26rft.spage%253D591%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ducrocq</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patry</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Combining genomic and classical information in national BLUP evaluation to reduce bias due to genomic pre-selection.</span> <abbr class="cit-jnl-abbrev">Interbull Bull.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInterbull%2BBull.%26rft.volume%253D41%26rft.spage%253D33%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.12" data-doi="10.1139/x02-111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dutkowski</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">e Silva</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopez</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Spatial analysis methods for forest genetic trials.</span> <abbr class="cit-jnl-abbrev">Can. J. For. Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">2201</span>–<span class="cit-lpage">2214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BFor.%2BRes.%26rft.volume%253D32%26rft.spage%253D2201%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fx02-111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/x02-111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.13" data-doi="10.1017/S001667230900007X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">El-Kassaby</span>  <span class="cit-name-given-names">Y. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lstibůrek</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Breeding without breeding.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DEl-Kassaby%26rft.auinit1%253DY.%2BA.%26rft.volume%253D91%26rft.issue%253D2%26rft.spage%253D111%26rft.epage%253D120%26rft.atitle%253DBreeding%2Bwithout%2Bbreeding.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS001667230900007X%26rft_id%253Dinfo%253Apmid%252F19393127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S001667230900007X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19393127&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F935.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266659200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.14" data-doi="10.1371/journal.pone.0025737"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">El-Kassaby</span>  <span class="cit-name-given-names">Y. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cappa</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liewlaksaneeyanawin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klápště</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lstibŭrek</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Breeding without breeding: is a complete pedigree necessary for efficient breeding?</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e25737</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DEl-Kassaby%26rft.auinit1%253DY.%2BA.%26rft.volume%253D6%26rft.issue%253D10%26rft.spage%253De25737%26rft.epage%253De25737%26rft.atitle%253DBreeding%2Bwithout%2Bbreeding%253A%2Bis%2Ba%2Bcomplete%2Bpedigree%2Bnecessary%2Bfor%2Befficient%2Bbreeding%253F%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0025737%26rft_id%253Dinfo%253Apmid%252F21991342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0025737&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21991342&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F935.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.15" data-doi="10.1007/s11056-012-9338-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">El-Kassaby</span>  <span class="cit-name-given-names">Y. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klápště</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guy</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Breeding without breeding: selection using the genomic best linear unbiased predictor method (GBLUP).</span> <abbr class="cit-jnl-abbrev">New For.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">631</span>–<span class="cit-lpage">637</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BFor.%26rft.volume%253D43%26rft.spage%253D631%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11056-012-9338-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11056-012-9338-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-7.3.935.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source"><em>Introduction to Quantitative Genetics</em>.</span> <span class="cit-publ-name">Longmans Green</span>, <span class="cit-publ-loc">Harlow, Essex</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.17" data-doi="10.1186/1297-9686-43-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forni</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Different genomic relationship matrices for single-step analysis using phenotypic, pedigree and genomic information.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DForni%26rft.auinit1%253DS.%26rft.volume%253D43%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DDifferent%2Bgenomic%2Brelationship%2Bmatrices%2Bfor%2Bsingle-step%2Banalysis%2Busing%2Bphenotypic%252C%2Bpedigree%2Band%2Bgenomic%2Binformation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-1%26rft_id%253Dinfo%253Apmid%252F21208445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21208445&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F935.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.18" data-doi="10.1098/rspb.2007.1032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frentiu</span>  <span class="cit-name-given-names">F. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clegg</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chittock</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blows</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Pedigree-free animal models: the relatedness matrix reloaded.</span> <abbr class="cit-jnl-abbrev">Proc. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">275</span>: <span class="cit-fpage">639</span>–<span class="cit-lpage">647</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DFrentiu%26rft.auinit1%253DF.%2BD%26rft.volume%253D275%26rft.issue%253D1635%26rft.spage%253D639%26rft.epage%253D647%26rft.atitle%253DPedigree-free%2Banimal%2Bmodels%253A%2Bthe%2Brelatedness%2Bmatrix%2Breloaded%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2007.1032%26rft_id%253Dinfo%253Apmid%252F18211872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMjoiMjc1LzE2MzUvNjM5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzkzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.3.935.19"><div class="cit-metadata"><cite>Gamal<span class="cit-auth"><span class="cit-name-surname">El-Dien</span>  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ratcliffe</span>  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klápště</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Porth</span>  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Implementation of the realized genomic relationship matrix to open-pollinated white spruce family testing for disentangling additive from nonadditive genetic effects.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">743</span>–<span class="cit-lpage">753</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D6%26rft.spage%253D743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.20" data-doi="10.1007/s11295-010-0328-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grattapaglia</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection in forest tree breeding.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D7%26rft.spage%253D241%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-010-0328-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-010-0328-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000288656800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.21" data-doi="10.1534/genetics.113.152207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic BLUP decoded: a look into the black box of genomic prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">597</span>–<span class="cit-lpage">607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.152207%26rft_id%253Dinfo%253Apmid%252F23640517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTk3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzkzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-7.3.935.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-source"><em>Applications of Linear Models in Animal Breeding</em>.</span> <span class="cit-publ-name">University of Guelph</span>, <span class="cit-publ-loc">Guelph, Canada</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hodge</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">The efficiency of half-sib, half-diallel and circular mating designs in the estimation of genetic parameters in forestry: a simulation.</span> <abbr class="cit-jnl-abbrev">For. Sci.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">757</span>–<span class="cit-lpage">776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFor.%2BSci.%26rft.volume%253D38%26rft.spage%253D757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jayawickrama</span>  <span class="cit-name-given-names">K. J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carson</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A breeding strategy for the New Zealand radiata pine breeding cooperative.</span> <abbr class="cit-jnl-abbrev">Silvae Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">82</span>–<span class="cit-lpage">89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSilvae%2BGenet.%26rft.volume%253D49%26rft.spage%253D82%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.25" data-doi="10.1111/j.1365-294X.2007.03089.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kalinowski</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taper</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Revising how the computer program CERVUS accommodates genotyping error increases success in paternity assignment.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1099</span>–<span class="cit-lpage">1106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DKalinowski%26rft.auinit1%253DS.%2BT.%26rft.volume%253D16%26rft.issue%253D5%26rft.spage%253D1099%26rft.epage%253D1106%26rft.atitle%253DRevising%2Bhow%2Bthe%2Bcomputer%2Bprogram%2BCERVUS%2Baccommodates%2Bgenotyping%2Berror%2Bincreases%2Bsuccess%2Bin%2Bpaternity%2Bassignment.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2007.03089.x%26rft_id%253Dinfo%253Apmid%252F17305863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2007.03089.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17305863&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F935.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244245300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.26" data-doi="10.1007/s11295-013-0673-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klápště</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lstibŭrek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">El-Kassaby</span>  <span class="cit-name-given-names">Y. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Estimates of genetic parameters and breeding values from western larch open-pollinated families using marker-based relationship.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D10%26rft.spage%253D241%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-013-0673-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-013-0673-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.27" data-doi="10.1007/s11295-013-0630-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koreckỳ</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klápště</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lstibŭrek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kobliha</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comparison of genetic parameters from marker-based relationship, sibship, and combined models in Scots pine multi-site open-pollinated tests.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1227</span>–<span class="cit-lpage">1235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D9%26rft.spage%253D1227%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-013-0630-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-013-0630-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.28" data-doi="10.3168/jds.2009-2061"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A relationship matrix including full pedigree and genomic information.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">4656</span>–<span class="cit-lpage">4663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DLegarra%26rft.auinit1%253DA.%26rft.volume%253D92%26rft.issue%253D9%26rft.spage%253D4656%26rft.epage%253D4663%26rft.atitle%253DA%2Brelationship%2Bmatrix%2Bincluding%2Bfull%2Bpedigree%2Band%2Bgenomic%2Binformation%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2009-2061%26rft_id%253Dinfo%253Apmid%252F19700729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2009-2061&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19700729&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F935.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269156600059&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sørensen</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berg</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Allele frequency changes due to hitch-hiking in genomic selection programs.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D46%26rft.spage%253D8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.30" data-doi="10.1111/j.1439-0388.2011.00966.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luan</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The unified approach to the use of genomic and pedigree information in genomic evaluations revisited.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">429</span>–<span class="cit-lpage">439</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%26rft.volume%253D128%26rft.issue%253D6%26rft.spage%253D429%26rft.epage%253D439%26rft.atitle%253DThe%2Bunified%2Bapproach%2Bto%2Bthe%2Buse%2Bof%2Bgenomic%2Band%2Bpedigree%2Binformation%2Bin%2Bgenomic%2Bevaluations%2Brevisited.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2011.00966.x%26rft_id%253Dinfo%253Apmid%252F22059576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2011.00966.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22059576&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F935.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.31" data-doi="10.3168/jds.2009-2064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Computing procedures for genetic evaluation including phenotypic, full pedigree, and genomic information.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">4648</span>–<span class="cit-lpage">4655</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DMisztal%26rft.auinit1%253DI.%26rft.volume%253D92%26rft.issue%253D9%26rft.spage%253D4648%26rft.epage%253D4655%26rft.atitle%253DComputing%2Bprocedures%2Bfor%2Bgenetic%2Bevaluation%2Bincluding%2Bphenotypic%252C%2Bfull%2Bpedigree%252C%2Band%2Bgenomic%2Binformation%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2009-2064%26rft_id%253Dinfo%253Apmid%252F19700728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2009-2064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19700728&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F935.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269156600058&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-book" id="cit-7.3.935.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Namkoong</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brouard</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-source"><em>Tree Breeding: Principles and Strategies</em>.</span> <span class="cit-publ-name">Springer Science &amp; Business Media</span>, <span class="cit-publ-loc">Heidelberg, Germany</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.33" data-doi="10.1186/1297-9686-44-16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ødegård</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Estimation of heritability from limited family data using genome-wide identity-by-descent sharing.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DOdegard%26rft.auinit1%253DJ.%26rft.volume%253D44%26rft.spage%253D16%26rft.epage%253D16%26rft.atitle%253DEstimation%2Bof%2Bheritability%2Bfrom%2Blimited%2Bfamily%2Bdata%2Busing%2Bgenome-wide%2Bidentity-by-descent%2Bsharing.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-44-16%26rft_id%253Dinfo%253Apmid%252F22568689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-44-16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22568689&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F935.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.34" data-doi="10.1534/genetics.105.049940"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oliehoek</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Windig</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Arendonk</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bijma</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Estimating relatedness between individuals in general populations with a focus on their use in conservation programs.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">496</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.049940%26rft_id%253Dinfo%253Apmid%252F16510792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTczLzEvNDgzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzkzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.35" data-doi="10.1111/1755-0998.12062"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pavy</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gagnon</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rigault</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blais</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deschênes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Development of high-density SNP genotyping arrays for white spruce (Picea glauca) and transferability to subtropical and nordic congeners.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">324</span>–<span class="cit-lpage">336</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D13%26rft.spage%253D324%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12062%26rft_id%253Dinfo%253Apmid%252F23351128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12062&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23351128&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F935.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.36" data-doi="10.1111/nph.12014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Porth</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klápště</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skyba</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geraldes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Populus trichocarpa cell wall chemistry and ultrastructure trait variation, genetic control and genetic correlations.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">777</span>–<span class="cit-lpage">790</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D197%26rft.spage%253D777%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.12014%26rft_id%253Dinfo%253Apmid%252F23278123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.12014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23278123&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F935.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000313266000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.37" data-doi="10.1038/nrg2865"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Reconciling the analysis of IBD and IBS in complex trait studies.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">800</span>–<span class="cit-lpage">805</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPowell%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D11%26rft.issue%253D11%26rft.spage%253D800%26rft.epage%253D805%26rft.atitle%253DReconciling%2Bthe%2Banalysis%2Bof%2BIBD%2Band%2BIBS%2Bin%2Bcomplex%2Btrait%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2865%26rft_id%253Dinfo%253Apmid%252F20877324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2865&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20877324&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F935.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.38" data-doi="10.1104/pp.111.179663"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rigault</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyle</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lepage</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooke</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bousquet</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A white spruce gene catalog for conifer genome analyses.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">14</span>–<span class="cit-lpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.111.179663%26rft_id%253Dinfo%253Apmid%252F21730200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo4OiIxNTcvMS8xNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMy85MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.39" data-doi="10.1017/S0016672300033620"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ritland</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Estimators for pairwise relatedness and individual inbreeding coefficients.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">185</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D67%26rft.spage%253D175%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300033620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300033620&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996UK49900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Squillace</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">Average genetic correlations among offspring from open-pollinated forest trees.</span> <abbr class="cit-jnl-abbrev">Silvae Genet.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">149</span>–<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSilvae%2BGenet.%26rft.volume%253D23%26rft.spage%253D149%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.41" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F935.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.42" data-doi="10.1371/journal.pgen.0020041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Medland</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morley</span>  <span class="cit-name-given-names">K. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Assumption-free estimation of heritability from genome-wide identity-by-descent sharing between full siblings.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e41</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DVisscher%26rft.auinit1%253DP.%2BM.%26rft.volume%253D2%26rft.issue%253D3%26rft.spage%253De41%26rft.epage%253De41%26rft.atitle%253DAssumption-free%2Bestimation%2Bof%2Bheritability%2Bfrom%2Bgenome-wide%2Bidentity-by-descent%2Bsharing%2Bbetween%2Bfull%2Bsiblings.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020041%26rft_id%253Dinfo%253Apmid%252F16565746%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16565746&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F935.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.43" data-doi="10.1017/S001667231100022X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vitezica</span>  <span class="cit-name-given-names">Z. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Bias in genomic predictions for populations under selection.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">366</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DVitezica%26rft.auinit1%253DZ.%2BG.%26rft.volume%253D93%26rft.issue%253D5%26rft.spage%253D357%26rft.epage%253D366%26rft.atitle%253DBias%2Bin%2Bgenomic%2Bpredictions%2Bfor%2Bpopulations%2Bunder%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS001667231100022X%26rft_id%253Dinfo%253Apmid%252F21767459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S001667231100022X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21767459&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F935.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.935.44" data-doi="10.1534/genetics.166.4.1963"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Sibship reconstruction from genetic data with typing errors.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">1963</span>–<span class="cit-lpage">1979</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWang%26rft.auinit1%253DJ.%26rft.volume%253D166%26rft.issue%253D4%26rft.spage%253D1963%26rft.epage%253D1979%26rft.atitle%253DSibship%2BReconstruction%2BFrom%2BGenetic%2BData%2BWith%2BTyping%2BErrors%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.166.4.1963%26rft_id%253Dinfo%253Apmid%252F15126412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Ni80LzE5NjMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvOTM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/3/935.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/3/923" title="CRISPR-Cas-Induced Mutants Identify a Requirement for dSTIM in Larval Dopaminergic Cells of Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/3/943" title="Reproduction and In-Depth Evaluation of Genome-Wide Association Studies and Genome-Wide Meta-analyses Using Summary Statistics" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/3"><span>Volume 7 Issue 3, March 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/3.cover-source.jpg?itok=h8yj2GBo" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/3/935" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1E_Qht_rJiTBUIRqjum_C9ULoifPjjsxEEvdgZcYPwY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/3/935" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/3/935" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/3/935" class="active" data-icon-position="" data-hide-link-title="0">Single-Step BLUP with Varying Genotyping Effort in Open-Pollinated Picea glauca</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/483525" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-F5Eloieq5N0U1GZjyKY5da672oYOiQkSnAhpCIlqS5s" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/3/935.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/3/935" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-0QzGiDxNFtzIAuvM-GXzx4sre-1-9h5s0ov--TtYWLk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/3/935" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/3/935.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483525" id="node483525--2" data-pisa="ggg;7/3/935" data-pisa-master="ggg;g3.116.037895" data-apath="/ggg/7/3/935.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Single-Step BLUP with Varying Genotyping Effort in Open-Pollinated <em>Picea glauca</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4469-2929" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Blaise Ratcliffe</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-9112-8522" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Omnia Gamal El-Dien</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6234-2263" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Eduardo P. Cappa</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-9344-6348" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ilga Porth</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-5504-3735" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jaroslav Klápště</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-2203-0433" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Charles Chen</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-4887-8977" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yousry A. El-Kassaby</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">935-942;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.037895 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483525--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Blaise Ratcliffe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, British Columbia V6T 1Z4, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Blaise%2BRatcliffe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ratcliffe%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABlaise%2BRatcliffe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4469-2929" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Blaise Ratcliffe</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Omnia Gamal El-Dien </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, British Columbia V6T 1Z4, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Pharmacognosy Department, Faculty of Pharmacy, Alexandria University, Alexandria, 21521, Egypt</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Omnia%2BGamal%2BEl-Dien%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=El-Dien%20OG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOmnia%2BGamal%2BEl-Dien%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9112-8522" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Omnia Gamal El-Dien</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Eduardo P. Cappa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Instituto Nacional de Tecnología Agropecuaria (INTA), Instituto de Recursos Biológicos, Las Cabañas y De Los Reseros s/n, 1686 Hurlingham, Buenos Aires, Argentina</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eduardo%2BP.%2BCappa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cappa%20EP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEduardo%2BP.%2BCappa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6234-2263" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Eduardo P. Cappa</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ilga Porth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Départment des Sciences du Bois et de la Forêt, Faculté de Foresterie, de Géographie et Géomatique, Université Laval Québec, G1V 0A6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ilga%2BPorth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Porth%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIlga%2BPorth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9344-6348" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ilga Porth</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jaroslav Klápště </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Genetics and Physiology of Forest Trees, Faculty of Forestry and Wood Sciences, Czech University of Life Sciences Prague, 165 21 Praha 6, Czech Republic</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Scion (New Zealand Forest Research Institute Ltd.), Whakarewarewa, Rotorua 3046, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaroslav%2BKl%C3%A1p%C5%A1t%C4%9B%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kl%C3%A1p%C5%A1t%C4%9B%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJaroslav%2BKl%25C3%25A1p%25C5%25A1t%25C4%259B%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5504-3735" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jaroslav Klápště</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Charles Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Biochemistry and Molecular Biology, Oklahoma State University, Stillwater, Oklahoma 74078-3035</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharles%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2203-0433" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Charles Chen</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Yousry A. El-Kassaby </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, British Columbia V6T 1Z4, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yousry%2BA.%2BEl-Kassaby%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=El-Kassaby%20YA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYousry%2BA.%2BEl-Kassaby%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4887-8977" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yousry A. El-Kassaby</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:y.el-kassaby@ubc.ca" class="" data-icon-position="" data-hide-link-title="0">y.el-kassaby@ubc.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/3/935" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/3/935&amp;title=Single-Step%20BLUP%20with%20Varying%20Genotyping%20Effort%20in%20Open-Pollinated%20Picea%20glauca" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/3/935&amp;title=Single-Step%20BLUP%20with%20Varying%20Genotyping%20Effort%20in%20Open-Pollinated%20Picea%20glauca" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/3/935&amp;title=Single-Step%20BLUP%20with%20Varying%20Genotyping%20Effort%20in%20Open-Pollinated%20Picea%20glauca" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/3/935&amp;text=Single-Step%20BLUP%20with%20Varying%20Genotyping%20Effort%20in%20Open-Pollinated%20Picea%20glauca" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/3/935&amp;title=Single-Step%20BLUP%20with%20Varying%20Genotyping%20Effort%20in%20Open-Pollinated%20Picea%20glauca" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/3/935&amp;t=Single-Step%20BLUP%20with%20Varying%20Genotyping%20Effort%20in%20Open-Pollinated%20Picea%20glauca" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/3/935&amp;title=Single-Step%20BLUP%20with%20Varying%20Genotyping%20Effort%20in%20Open-Pollinated%20Picea%20glauca" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/3/935&amp;title=Single-Step%20BLUP%20with%20Varying%20Genotyping%20Effort%20in%20Open-Pollinated%20Picea%20glauca" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/483525/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483525" id="node483525--3" data-pisa="ggg;7/3/935" data-pisa-master="ggg;g3.116.037895" data-apath="/ggg/7/3/935.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Single-Step BLUP with Varying Genotyping Effort in Open-Pollinated <em>Picea glauca</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4469-2929" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Blaise Ratcliffe</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-9112-8522" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Omnia Gamal El-Dien</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6234-2263" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Eduardo P. Cappa</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-9344-6348" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ilga Porth</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-5504-3735" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jaroslav Klápště</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-2203-0433" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Charles Chen</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-4887-8977" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yousry A. El-Kassaby</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">935-942;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.037895 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483525--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Blaise Ratcliffe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, British Columbia V6T 1Z4, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Blaise%2BRatcliffe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ratcliffe%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABlaise%2BRatcliffe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4469-2929" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Blaise Ratcliffe</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Omnia Gamal El-Dien </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, British Columbia V6T 1Z4, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Pharmacognosy Department, Faculty of Pharmacy, Alexandria University, Alexandria, 21521, Egypt</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Omnia%2BGamal%2BEl-Dien%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=El-Dien%20OG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOmnia%2BGamal%2BEl-Dien%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9112-8522" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Omnia Gamal El-Dien</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Eduardo P. Cappa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Instituto Nacional de Tecnología Agropecuaria (INTA), Instituto de Recursos Biológicos, Las Cabañas y De Los Reseros s/n, 1686 Hurlingham, Buenos Aires, Argentina</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eduardo%2BP.%2BCappa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cappa%20EP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEduardo%2BP.%2BCappa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6234-2263" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Eduardo P. Cappa</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ilga Porth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Départment des Sciences du Bois et de la Forêt, Faculté de Foresterie, de Géographie et Géomatique, Université Laval Québec, G1V 0A6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ilga%2BPorth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Porth%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIlga%2BPorth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9344-6348" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ilga Porth</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jaroslav Klápště </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Genetics and Physiology of Forest Trees, Faculty of Forestry and Wood Sciences, Czech University of Life Sciences Prague, 165 21 Praha 6, Czech Republic</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Scion (New Zealand Forest Research Institute Ltd.), Whakarewarewa, Rotorua 3046, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaroslav%2BKl%C3%A1p%C5%A1t%C4%9B%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kl%C3%A1p%C5%A1t%C4%9B%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJaroslav%2BKl%25C3%25A1p%25C5%25A1t%25C4%259B%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5504-3735" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jaroslav Klápště</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Charles Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Biochemistry and Molecular Biology, Oklahoma State University, Stillwater, Oklahoma 74078-3035</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharles%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2203-0433" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Charles Chen</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Yousry A. El-Kassaby </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, British Columbia V6T 1Z4, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yousry%2BA.%2BEl-Kassaby%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=El-Kassaby%20YA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYousry%2BA.%2BEl-Kassaby%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4887-8977" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yousry A. El-Kassaby</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:y.el-kassaby@ubc.ca" class="" data-icon-position="" data-hide-link-title="0">y.el-kassaby@ubc.ca</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=935&amp;rft.epage=942&amp;rft.atitle=Single-Step%20BLUP%20with%20Varying%20Genotyping%20Effort%20in%20Open-Pollinated%20Picea%20glauca&amp;rft.volume=7&amp;rft.issue=3&amp;rft.date=2017-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Ratcliffe%2C+Blaise&amp;rft.au=El-Dien%2C+Omnia+Gamal&amp;rft.au=Cappa%2C+Eduardo+P.&amp;rft.au=Porth%2C+Ilga&amp;rft.au=Kl%C3%A1p%C5%A1t%C4%9B%2C+Jaroslav&amp;rft.au=Chen%2C+Charles&amp;rft.au=El-Kassaby%2C+Yousry+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/483525/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/483525/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/483525/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/483525/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/483525/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/483525/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/483525/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/483525/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/483525/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/483525/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/483525/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/483525/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/3/935" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/3/935.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/3/935.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/3/935.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/3/935.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__CK8qNg7zHnCPfipcSU0qjCyIiw2pe-WKGx2V0TgizGw__wprDSLLH9CuacxJkxX2DuWt0EaauZA-f_LGjovfY3aw__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
