<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/3/871.full.txt" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/3/871.full.pdf" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/3/871.atom" />
<meta name="HW.pisa" content="ggg;7/3/871" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="De Novo Transcriptome Characterization of a Sterilizing Trematode Parasite (Microphallus sp.) from Two Species of New Zealand Snails" />
<meta name="DC.Identifier" content="10.1534/g3.116.037275" />
<meta name="DC.Date" content="2017-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Bankers and Neiman. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Snail-borne trematodes represent a large, diverse, and evolutionarily, ecologically, and medically important group of parasites, often imposing strong selection on their hosts and causing host morbidity and mortality. Even so, there are very few genomic and transcriptomic resources available for this important animal group. We help to fill this gap by providing transcriptome resources from trematode metacercariae infecting two congeneric snail species, Potamopyrgus antipodarum and P. estuarinus . This genus of New Zealand snails has gained prominence in large part through the development of P. antipodarum and its sterilizing trematode parasite Microphallus livelyi into a textbook model for host–parasite coevolutionary interactions in nature. By contrast, the interactions between Microphallus trematodes and P. estuarinus , an estuary-inhabiting species closely related to the freshwater P. antipodarum , are relatively unstudied. Here, we provide the first annotated transcriptome assemblies from Microphallus isolated from P. antipodarum and P. estuarinus . We also use these transcriptomes to produce genomic resources that will be broadly useful to those interested in host–parasite coevolution, local adaption, and molecular evolution and phylogenetics of this and other snail–trematode systems. Analyses of the two Microphallus transcriptomes revealed that the two trematode types are more genetically differentiated from one another than are the M. livelyi infecting different populations of P. antipodarum , suggesting that the Microphallus infecting P. estuarinus represent a distinct lineage. We also provide a promising set of candidate genes likely involved in parasitic infection and response to salinity stress." />
<meta name="DC.Contributor" content="Laura Bankers" />
<meta name="DC.Contributor" content="Maurine Neiman" />
<meta name="article:published_time" content="2017-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="De Novo Transcriptome Characterization of a Sterilizing Trematode Parasite (Microphallus sp.) from Two Species of New Zealand Snails" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Snail-borne trematodes represent a large, diverse, and evolutionarily, ecologically, and medically important group of parasites, often imposing strong selection on their hosts and causing host morbidity and mortality. Even so, there are very few genomic and transcriptomic resources available for this important animal group. We help to fill this gap by providing transcriptome resources from trematode metacercariae infecting two congeneric snail species, &lt;i&gt;Potamopyrgus antipodarum&lt;/i&gt; and &lt;i&gt;P. estuarinus&lt;/i&gt;. This genus of New Zealand snails has gained prominence in large part through the development of &lt;i&gt;P. antipodarum&lt;/i&gt; and its sterilizing trematode parasite &lt;i&gt;Microphallus livelyi&lt;/i&gt; into a textbook model for host–parasite coevolutionary interactions in nature. By contrast, the interactions between &lt;i&gt;Microphallus&lt;/i&gt; trematodes and &lt;i&gt;P. estuarinus&lt;/i&gt;, an estuary-inhabiting species closely related to the freshwater &lt;i&gt;P. antipodarum&lt;/i&gt;, are relatively unstudied. Here, we provide the first annotated transcriptome assemblies from &lt;i&gt;Microphallus&lt;/i&gt; isolated from &lt;i&gt;P. antipodarum&lt;/i&gt; and &lt;i&gt;P. estuarinus&lt;/i&gt;. We also use these transcriptomes to produce genomic resources that will be broadly useful to those interested in host–parasite coevolution, local adaption, and molecular evolution and phylogenetics of this and other snail–trematode systems. Analyses of the two &lt;i&gt;Microphallus&lt;/i&gt; transcriptomes revealed that the two trematode types are more genetically differentiated from one another than are the &lt;i&gt;M. livelyi&lt;/i&gt; infecting different populations of &lt;i&gt;P. antipodarum&lt;/i&gt;, suggesting that the &lt;i&gt;Microphallus&lt;/i&gt; infecting &lt;i&gt;P. estuarinus&lt;/i&gt; represent a distinct lineage. We also provide a promising set of candidate genes likely involved in parasitic infection and response to salinity stress.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/03/01" />
<meta name="citation_mjid" content="ggg;7/3/871" />
<meta name="citation_id" content="7/3/871" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/3/871" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/3/871.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/3/871.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/3/871.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.037275" />
<meta name="citation_pmid" content="28122948" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="871" />
<meta name="citation_lastpage" content="880" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Laura Bankers" />
<meta name="citation_author_institution" content="Department of Biology, University of Iowa, Iowa City, Iowa 52242" />
<meta name="citation_author_email" content="laura-rice@uiowa.edu" />
<meta name="citation_author" content="Maurine Neiman" />
<meta name="citation_author_institution" content="Department of Biology, University of Iowa, Iowa City, Iowa 52242" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1543-8115" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=L. Bankers;citation_author=P. Fields;citation_author=K. E. McElroy;citation_author=J. Boore;citation_author=J. M. Logsdon;citation_title=Genomic evidence for population-specific responses to coevolving parasite in a New Zealand freshwater snail.;citation_year=2016;citation_doi=10.1101/045674" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Camacho;citation_author=G. Coulouris;citation_author=V. Avagyan;citation_author=N. Ma;citation_author=J. Papadopoulos;citation_title=BLAST+: architecture and applications.;citation_pages=421-421;citation_volume=10;citation_year=2009;citation_pmid=20003500;citation_doi=10.1186/1471-2105-10-421" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=D. Carmona;citation_author=F. P. Connor;citation_author=M. T. Johnson;citation_title=Fifty years of co-evolution and beyond: integrating co-evolution from molecules to species.;citation_pages=5315-5329;citation_volume=2;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=K. Chen;citation_title=Transcriptome and molecular pathway analysis of the hepatopancreas in the Pacific white shrimp Litopenaeus vannamei under chronic low-salinity stress.;citation_pages=e0131503;citation_volume=10;citation_year=2015;citation_pmid=26147449;citation_doi=10.1371/journal.pone.0131503" />
<meta name="citation_reference" content="citation_journal_title=BioTechniques;citation_journal_abbrev=Biotechniques;citation_author=P. Chomczynski;citation_title=A reagent for the single-step simultaneous isolation of RNA, DNA and proteins from cell and tissue samples.;citation_pages=532-532-4, 537;citation_volume=15;citation_year=1993;citation_issue=3;citation_pmid=7692896" />
<meta name="citation_reference" content="citation_journal_title=Analytical biochemistry;citation_journal_abbrev=Anal Biochem;citation_author=P. Chomczynski;citation_author=N. Sacchi;citation_title=Single-step method of RNA isolation by acid guanidinium thiocyanate-phenol-chloroform extraction.;citation_pages=156-159;citation_volume=162;citation_year=1987;citation_issue=1;citation_pmid=2440339;citation_doi=10.1016/0003-2697(87)90021-2" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Conesa;citation_author=S. Götz;citation_author=J. M. García-Gómez;citation_author=J. Terol;citation_author=M. Talón;citation_title=Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research.;citation_volume=21;citation_year=2005;citation_pmid=16081474;citation_doi=10.1093/bioinformatics/bti610" />
<meta name="citation_reference" content="citation_title=Interrelationships of the Platyhelminthes;citation_author=T. H. Cribb;citation_author=T. A. Bray;citation_author=D. T. Littlewood;citation_author=S. P. Pichelin;citation_author=E. A. Herniou;citation_title=The digenea;citation_pages=168-173;citation_volume=1;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. DePristo;citation_author=E. Banks;citation_author=R. Poplin;citation_author=K. V. Garimella;citation_author=J. R. Maguire;citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data.;citation_pages=491-498;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21478889;citation_doi=10.1038/ng.806" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=M. F. Dybdahl;citation_author=C. M. Lively;citation_title=The geography of coevolution: comparative population structures for a snail and its trematode parasite.;citation_pages=2264-2275;citation_volume=50;citation_year=1996;citation_doi=10.2307/2410696" />
<meta name="citation_reference" content="citation_journal_title=Journal of evolutionary biology;citation_journal_abbrev=J Evol Biol;citation_author=M. F. Dybdahl;citation_author=A. C. Krist;citation_title=Genotypic vs. condition effects on parasite-driven rare advantage.;citation_pages=967-973;citation_volume=17;citation_year=2004;citation_issue=5;citation_pmid=15312069" />
<meta name="citation_reference" content="citation_journal_title=WiadomoÅ?ci parazytologiczne;citation_journal_abbrev=Wiad Parazytol;citation_author=J. Dziekońska-Rynko;citation_author=J. Rokicki;citation_title=Activity of selected hydrolases in excretion-secretion products and extracts of adult Contracaecum rudolphii.;citation_pages=227-231;citation_volume=51;citation_year=2005;citation_issue=3;citation_pmid=16913527" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=A. R. Ellison;citation_author=G. V. DiRenzo;citation_author=C. A. McDonald;citation_author=K. R. Lips;citation_author=K. R. Zamudio;citation_title=First in vivo Batrachochytrium dendrobatidis transcriptomes reveal mechanisms of host exploitation, host-specific gene expression, and expressed genotype shifts;citation_volume=7;citation_year=2017;citation_pmid=27856699;citation_doi=10.1534/g3.116.035873" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=M. C. Fischer;citation_author=C. Rellstab;citation_author=A. Tedder;citation_author=S. Zoller;citation_author=F. Gugerli;citation_title=Population genomic footprints of selection and associations with climate in natural populations of Arabidopsis halleri from the Alps.;citation_pages=5594-5607;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12521" />
<meta name="citation_reference" content="citation_journal_abbrev=Parasitology;citation_author=K. V. Galaktionov;citation_author=I. Blasco-Costa;citation_author=P. D. Olson;citation_title=Life cycles, molecular phylogeny and historical biogeography of the 'pygmaeus' microphallids (Digenea: Microphallidae): widespread parasites of marine and coastal birds in the Holarctic.;citation_pages=1346-1360;citation_volume=139;citation_year=2012;citation_issue=10;citation_pmid=22717011" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=M. G. Grabherr;citation_author=B. J. Haas;citation_author=M. Yassour;citation_author=J. Z. Levin;citation_author=D. A. Thompson;citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome.;citation_pages=644-652;citation_volume=29;citation_year=2011;citation_issue=7;citation_pmid=21572440;citation_doi=10.1038/nbt.1883" />
<meta name="citation_reference" content="citation_journal_title=IEEE/ACM Trans. Comput. Biol. Bioinform.;citation_author=G. Gremme;citation_author=S. Steinbiss;citation_author=S. Kurtz;citation_title=GenomeTools: a comprehensive software library for efficient processing of structured genome annotations.;citation_pages=645-656;citation_volume=10;citation_year=2013;citation_doi=10.1109/TCBB.2013.68" />
<meta name="citation_reference" content="citation_title=Fastx-toolkit. FASTQ/A short-read pre-processing tools.;citation_author=A. Gordon;citation_author=G. Hannon;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=Y. Guo;citation_author=K. E. Wiegert-Rininger;citation_author=V. A. Vallejo;citation_author=C. S. Barry;citation_author=R. M. Warner;citation_title=Transcriptome-enabled marker discovery and mapping of plastochron-related genes in Petunia spp.;citation_pages=726;citation_volume=16;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Nat. Protoc.;citation_author=B. J. Haas;citation_author=A. Papanicolaou;citation_author=M. Yassour;citation_author=M. Grabherr;citation_author=P. D. Blood;citation_title=De novo transcript sequence reconstruction from RNA-seq, using the Trinity platform for reference generation and analysis.;citation_pages=1494-1512;citation_volume=8;citation_year=2013;citation_pmid=23845962;citation_doi=10.1038/nprot.2013.084" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=Y. Hara;citation_author=K. Tatsumi;citation_author=M. Yoshida;citation_author=E. Kajikawa;citation_author=H. Kiyonari;citation_title=Optimizing and benchmarking de novo transcriptome sequencing: from library preparation to assembly evaluation.;citation_pages=977;citation_volume=16;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Zootaxa;citation_author=R. F. Hechinger;citation_title=Faunal survey and identification key for the trematodes (Platyhelminthes: Digenea) infecting Potamopyrgus antipodarum (Gastropoda: Hydrobiidae) as first intermediate host.;citation_pages=1-27;citation_volume=3418;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=K. E. Holsinger;citation_author=B. S. Weir;citation_title=Genetics in geographically structured populations: defining, estimating and interpreting F(ST).;citation_pages=639-650;citation_volume=10;citation_year=2009;citation_issue=9;citation_pmid=19687804;citation_doi=10.1038/nrg2611" />
<meta name="citation_reference" content="citation_journal_title=Molecular and biochemical parasitology;citation_journal_abbrev=Mol Biochem Parasitol;citation_author=A. M. C. Horemans;citation_author=A. G. M. Tielens;citation_author=S. G. van den Bergh;citation_title=The reversible effect of glucose on the energy metabolism of Schistosoma mansoni cercariae and schistosomula.;citation_pages=73-79;citation_volume=51;citation_year=1992;citation_issue=1;citation_pmid=1565140;citation_doi=10.1016/0166-6851(92)90202-U" />
<meta name="citation_reference" content="citation_journal_title=J. Appl. Res. Child.;citation_author=P. J. Hotez;citation_title=Pediatric tropical diseases and the world’s children living in extreme poverty.;citation_pages=10;citation_volume=4;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Y. Huang;citation_author=B. Niu;citation_author=Y. Gao;citation_author=L. Fu;citation_author=W. Li;citation_title=CD-HIT Suite: a web server for clustering and comparing biological sequences.;citation_volume=26;citation_year=2010;citation_pmid=20053844;citation_doi=10.1093/bioinformatics/btq003" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=A. D. Jurberg;citation_author=P. J. Brindley;citation_title=Gene function in schistosomes: recent advances toward a cure.;citation_pages=144;citation_volume=6;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=K. C. King;citation_author=L. F. Delph;citation_author=J. Jokela;citation_author=C. M. Lively;citation_title=The geographic mosaic of sex and the Red Queen.;citation_pages=1438-1441;citation_volume=19;citation_year=2009;citation_issue=17;citation_pmid=19631541;citation_doi=10.1016/j.cub.2009.06.062" />
<meta name="citation_reference" content="citation_journal_title=Biol. Lett.;citation_author=K. C. King;citation_author=J. Jokela;citation_author=C. M. Lively;citation_title=Trematode parasites infect or die in snail hosts.;citation_volume=7;citation_year=2011;citation_pmid=20961880;citation_doi=10.1098/rsbl.2010.0857" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. Kofler;citation_author=P. Orozco-terWengel;citation_author=N. De Maio;citation_author=R. V. Pandey;citation_author=V. Nolte;citation_title=PoPoolation: a toolbox for population genetic analysis of next generation sequencing data from pooled individuals.;citation_pages=e15925-e15925;citation_volume=6;citation_year=2011;citation_issue=1;citation_pmid=21253599;citation_doi=10.1371/journal.pone.0015925" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. Kofler;citation_author=R. V. Pandey;citation_author=C. Schlötterer;citation_title=Popoolation2: identifying differentiation between populations using sequencing of pooled DNA samples (Pool-Seq).;citation_volume=27;citation_year=2011;citation_pmid=22025480;citation_doi=10.1093/bioinformatics/btr589" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=M. Konczal;citation_author=P. Koteja;citation_author=M. Stuglik;citation_author=J. Radwan;citation_author=W. Babik;citation_title=Accuracy of allele frequency estimation using pooled RNA-Seq.;citation_pages=381-392;citation_volume=14;citation_year=2014;citation_pmid=24119300;citation_doi=10.1111/1755-0998.12186" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=G. Krautz-Peterson;citation_author=S. Camargo;citation_author=K. Huggel;citation_author=F. Verrey;citation_author=C. B. Shoemaker;citation_title=Amino acid transport in schistosomes: characterization of the permease heavy chain SPRM1hc.;citation_volume=282;citation_year=2007;citation_pmid=17545149;citation_doi=10.1074/jbc.M703512200" />
<meta name="citation_reference" content="citation_journal_title=Anim. Behav.;citation_author=E. P. Levri;citation_author=C. M. Lively;citation_title=The effect of size, reproductive condition, and parasitism on foraging behavior in a freshwater snail, Potamopyrgus antipodarum.;citation_pages=891-901;citation_volume=51;citation_year=1996;citation_doi=10.1006/anbe.1996.0093" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=W. Li;citation_author=L. Jaroszewski;citation_author=A. Godzik;citation_title=Tolerating some redundancy significantly speeds up clustering  of large protein databases;citation_pages=77-82;citation_volume=18;citation_year=2002;citation_issue=1;citation_pmid=11836214;citation_doi=10.1093/bioinformatics/18.1.77" />
<meta name="citation_reference" content="citation_journal_title=Biostatistics;citation_author=J. Li;citation_author=D. M. Witten;citation_author=I. M. Johnstone;citation_author=R. Tibshirani;citation_title=Normalization, testing, and false discovery rate estimation for RNA-sequencing data.;citation_volume=13;citation_year=2012;citation_pmid=22003245;citation_doi=10.1093/biostatistics/kxr031" />
<meta name="citation_reference" content="citation_journal_title=Anim. Genet.;citation_author=C. Li;citation_author=B. H. Beck;citation_author=S. A. Fuller;citation_author=E. Peatman;citation_title=Transcriptome annotation and marker discovery in white bass (Morone chrysops) and striped bass (Morone saxatilis).;citation_pages=885-887;citation_volume=45;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=C. M. Lively;citation_author=J. Jokela;citation_title=Clinal Variation for Local Adaptation in a Host-Parasite Interaction;citation_pages=891-897;citation_volume=263;citation_year=1996;citation_issue=1372;citation_doi=10.1098/rspb.1996.0132" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=C. M. Lively;citation_author=M. F. Dybdahl;citation_author=J. Jokela;citation_author=E. E. Osnas;citation_author=L. F. Delph;citation_title=Host sex and local adaptation by parasites in a snail-trematode interaction.;citation_pages=S6-S18;citation_volume=164;citation_year=2004;citation_pmid=15540142;citation_doi=10.1086/424605" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=T. F. Mitterboeck;citation_author=A. Y. Chen;citation_author=O. A. Zaheer;citation_author=Y. E. T. Ma;citation_author=S. J. Adamowicz;citation_title=Do saline taxa evolve faster? Comparing relative rates of molecular evolution between freshwater and marine eukaryotes.;citation_pages=1960-1978;citation_volume=70;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Syst. Parasitol.;citation_author=K. O’Dwyer;citation_author=I. Blasco-Costa;citation_author=R. Poulin;citation_author=A. Faltýnková;citation_title=Four marine digenean parasites of Austrolittorina spp. (Gastropoda: Littorinidae) in New Zealand: morphological and molecular data.;citation_pages=133-152;citation_volume=89;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=N.Z. J. Zool.;citation_author=E. E. Osnas;citation_author=C. M. Lively;citation_title=Using definitive host feces to infect experimental intermediate host populations: waterfowl hosts for New Zealand trematodes.;citation_pages=83-90;citation_volume=38;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Negl Trop Dis;citation_author=A. V. Protasio;citation_author=I. J. Tsai;citation_author=A. Babbage;citation_author=S. Nichol;citation_author=M. Hunt;citation_title=A systematically improved high quality genome and transcriptome of the human blood fluke Schistosoma mansoni.;citation_pages=e1455-e1455;citation_volume=6;citation_year=2012;citation_issue=1;citation_pmid=22253936;citation_doi=10.1371/journal.pntd.0001455" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=Z. Qiu;citation_author=F. Liu;citation_author=H. Lu;citation_author=Y. Huang;citation_title=Characterization and analysis of a de novo transcriptome from the pygmy grasshopper Tetrix japonica.;citation_year=2016;citation_doi=10.1111/1755&amp;ndash;0998.12553" />
<meta name="citation_reference" content="citation_journal_title=Acta Trop.;citation_author=D. Rollinson;citation_author=S. Knopp;citation_author=S. Levitz;citation_author=J. R. Stothard;citation_author=L. A. Tchuem Tchuenté;citation_title=Time to set the agenda for schistosomiasis elimination.;citation_pages=423-440;citation_volume=128;citation_year=2013;citation_pmid=22580511;citation_doi=10.1016/j.actatropica.2012.04.013" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. Routtu;citation_author=D. Ebert;citation_title=Genetic architecture of resistance in Daphnia hosts against two species of host-specific parasite.;citation_pages=241-248;citation_volume=114;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=F. Sievers;citation_author=A. Wilm;citation_author=D. Dineen;citation_author=T. J. Gibson;citation_author=K. Karplus;citation_title=Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega;citation_pages=539-539;citation_volume=7;citation_year=2011;citation_issue=1;citation_pmid=21988835;citation_doi=10.1038/msb.2011.75" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. A. Simão;citation_author=R. M. Waterhouse;citation_author=P. Ioannidis;citation_author=E. V. Kriventseva;citation_author=E. M. Zdobnov;citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.;citation_volume=31;citation_year=2015;citation_pmid=26059717;citation_doi=10.1093/bioinformatics/btv351" />
<meta name="citation_reference" content="citation_journal_title=BMC GigaScience;citation_author=E. E. Tassone;citation_author=S. M. Geib;citation_author=B. Hall;citation_author=J. A. Fabrick;citation_author=C. S. Brent;citation_title=De novo construction of an expanded transcriptome assembly for the western tarnished plant bug, Lygus hesperus.;citation_pages=6;citation_volume=5;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=S. Thavamanikumar;citation_author=S. Southerton;citation_author=B. Thumma;citation_title=RNA-Seq using two populations reveals genes and alleles controlling wood traits and growth in Eucalyptus nitens.;citation_pages=e101104;citation_volume=9;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Mar. Genomics;citation_author=K. Theissinger;citation_author=C. Falckenhayn;citation_author=D. Blande;citation_author=A. Toljamo;citation_author=J. Gutekunst;citation_title=De novo assembly and annotation of the freshwater crayfish Astacus astacus transcriptome.;citation_pages=7-10;citation_volume=28;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=C. Trapnell;citation_author=D. G. Hendrickson;citation_author=M. Sauvageau;citation_author=L. Goff;citation_author=J. L. Rinn;citation_title=Differential analysis of gene regulation at transcript resolution with RNA-seq.;citation_pages=46-53;citation_volume=31;citation_year=2013;citation_pmid=23222703;citation_doi=10.1038/nbt.2450" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=I. J. Tsai;citation_author=M. Zarowiecki;citation_author=N. Holroyd;citation_author=A. Garciarrubio;citation_author=A. Sanchez-Flores;citation_title=The genomes of four tapeworm species reveal adaptation to parasitism.;citation_pages=57-63;citation_volume=496;citation_year=2013;citation_pmid=23485966;citation_doi=10.1038/nature12031" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Untergasser;citation_author=I. Cutcutache;citation_author=T. Koressaar;citation_author=J. Ye;citation_author=B. C. Faircloth;citation_title=Primer3 – new capabilities and interfaces.;citation_volume=40;citation_year=2012;citation_pmid=22730293;citation_doi=10.1093/nar/gks596" />
<meta name="citation_reference" content="citation_journal_title=Curr. Protoc. Bioinformatics;citation_author=G. A. Van der Auwera;citation_author=M. O. Carneiro;citation_author=C. Hartl;citation_author=R. Poplin;citation_author=G. Del Angel;citation_title=From FastQ data to high-confidence variant calls: the Genome Analysis Toolkit best practices pipeline.;citation_pages=11101-11.10.33;citation_volume=43;citation_year=2013;citation_pmid=25431634" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=D. Vergara;citation_author=C. M. Lively;citation_author=K. C. King;citation_author=J. Jokela;citation_title=The geographic mosaic of sex and infection in lake populations of a New Zealand snail at multiple spatial scales.;citation_pages=484-493;citation_volume=182;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=D. P. Wall;citation_author=H. B. Fraser;citation_author=A. E. Hirsh;citation_title=Detecting putative orthologs;citation_pages=1710-1711;citation_volume=19;citation_year=2003;citation_issue=13;citation_pmid=15593400;citation_doi=10.1093/bioinformatics/btg213" />
<meta name="citation_reference" content="citation_journal_title=Worm;citation_author=K. Warnhoff;citation_author=K. Kornfeld;citation_title=New links between protein N-terminal acetylation, dauer diapause, and the insulin/IGF-1 signaling pathway in Caenorhabditis elegans.;citation_pages=e1023498;citation_volume=4;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=I. A. Warren;citation_author=K. L. Ciborowski;citation_author=E. Casadei;citation_author=D. G. Hazlerigg;citation_author=S. Martin;citation_title=Extensive local gene duplication and functional divergence among paralogs in Atlantic salmon.;citation_volume=6;citation_year=2014;citation_pmid=24951567;citation_doi=10.1093/gbe/evu131" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=G. A. Watterson;citation_title=On the number of segregating sites in genetical models without recombination.;citation_pages=256-276;citation_volume=7;citation_year=1975;citation_issue=2;citation_pmid=1145509;citation_doi=10.1016/0040-5809(75)90020-9" />
<meta name="citation_reference" content="citation_journal_title=Mauri Ora;citation_author=M. L. Winterbourn;citation_title=Larval trematode parasitising the New Zealand species of Potamopyrgus (Gastropoda: Hydrobiidae).;citation_pages=17-30;citation_volume=2;citation_year=1973" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="De Novo Transcriptome Characterization of a Sterilizing Trematode Parasite (Microphallus sp.) from Two Species of New Zealand Snails" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3.cover-source.jpg" />
<meta name="twitter:description" content="Snail-borne trematodes represent a large, diverse, and evolutionarily, ecologically, and medically important group of parasites, often imposing strong selection on their hosts and causing host morbidity and mortality. Even so, there are very few genomic and transcriptomic resources available for this important animal group. We help to fill this gap by providing transcriptome resources from trematode metacercariae infecting two congeneric snail species, Potamopyrgus antipodarum and P. estuarinus . This genus of New Zealand snails has gained prominence in large part through the development of P. antipodarum and its sterilizing trematode parasite Microphallus livelyi into a textbook model for host–parasite coevolutionary interactions in nature. By contrast, the interactions between Microphallus trematodes and P. estuarinus , an estuary-inhabiting species closely related to the freshwater P. antipodarum , are relatively unstudied. Here, we provide the first annotated transcriptome assemblies from Microphallus isolated from P. antipodarum and P. estuarinus . We also use these transcriptomes to produce genomic resources that will be broadly useful to those interested in host–parasite coevolution, local adaption, and molecular evolution and phylogenetics of this and other snail–trematode systems. Analyses of the two Microphallus transcriptomes revealed that the two trematode types are more genetically differentiated from one another than are the M. livelyi infecting different populations of P. antipodarum , suggesting that the Microphallus infecting P. estuarinus represent a distinct lineage. We also provide a promising set of candidate genes likely involved in parasitic infection and response to salinity stress." />
<meta name="og-title" property="og:title" content="De Novo Transcriptome Characterization of a Sterilizing Trematode Parasite (Microphallus sp.) from Two Species of New Zealand Snails" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/3/871" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Snail-borne trematodes represent a large, diverse, and evolutionarily, ecologically, and medically important group of parasites, often imposing strong selection on their hosts and causing host morbidity and mortality. Even so, there are very few genomic and transcriptomic resources available for this important animal group. We help to fill this gap by providing transcriptome resources from trematode metacercariae infecting two congeneric snail species, Potamopyrgus antipodarum and P. estuarinus . This genus of New Zealand snails has gained prominence in large part through the development of P. antipodarum and its sterilizing trematode parasite Microphallus livelyi into a textbook model for host–parasite coevolutionary interactions in nature. By contrast, the interactions between Microphallus trematodes and P. estuarinus , an estuary-inhabiting species closely related to the freshwater P. antipodarum , are relatively unstudied. Here, we provide the first annotated transcriptome assemblies from Microphallus isolated from P. antipodarum and P. estuarinus . We also use these transcriptomes to produce genomic resources that will be broadly useful to those interested in host–parasite coevolution, local adaption, and molecular evolution and phylogenetics of this and other snail–trematode systems. Analyses of the two Microphallus transcriptomes revealed that the two trematode types are more genetically differentiated from one another than are the M. livelyi infecting different populations of P. antipodarum , suggesting that the Microphallus infecting P. estuarinus represent a distinct lineage. We also provide a promising set of candidate genes likely involved in parasitic infection and response to salinity stress." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3.cover-source.jpg" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.g3journal.org/content/7/3/871" />
    <title>De Novo Transcriptome Characterization of a Sterilizing Trematode Parasite (Microphallus sp.) from Two Species of New Zealand Snails | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"CCBMw3aBN347vbGLKpx9YEN7Ybql8r2J_-F3nySJTh0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"483519","apath":"\/ggg\/7\/3\/871.atom","pisa":"ggg;7\/3\/871","ac":{"\/ggg\/7\/3\/871.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/3\/871.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/3\/871"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/3\/871"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/3\/871.full":true,"\/content\/7\/3\/871":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022aLHGl5ugbrmCpOf6YVT9ROOh9jWbNa9O27KwZmSl3jOTmhtd\\\\\\\/v3kXrYstCEyIjWlw3m6RAQv7VRyZVIJ6HnrqXR50DdewQUht9uPqN3PEga\\\\\\\/C1IeLTpJs0gqKcEefSYCQjSvg7q7SOQRcAMKF\\\\\\\/wOZ+XjdGlHupoROTPlQeN+RKZNXDNlWJyzhRhwkbPXwFyMmf6ofBnTi6gsXinGWK3gn0mQPqpIoAyGkjpBJHvB+fS3UmdXb1m7x8kNwMNmiJpJMW0fgABKBlwazShi6tVLZfh9tuc0UM20gfqjwgB4v6A8GtHIGgOPdaJwX7g98palSsPeSh+87QuneppemheLVyHvOWS82EN4sJ2XhmnfGIrql\\\\\\\/hFRdjhlOZe9aXxHFNrIL\\\\\\\/Y8o7zYFQqHXgpdk\\\\\\\/DFE8p8B6Nx2jsypWkRahTdasgrjZwKP19c66t+Ad4AqOXRZf4+3X1ujvGDfKU+OuBENSCNZZt6fljgBB6obYaiADUF\\\\\\\/sEYpEdWtmRHSZtswc2YTSCtj2HDFKWopPtXPoQKSIjELskEmgjMojabg0yrAJFku\\\\\\\/DCHtofnstv8SoSiulWA48S3cxLbGc9CxN0dVFZIzmQbDuTp\\\\\\\/xEAnIvEzOMihLcrnGqutvVmXkFqSqpDSi3Cnx1wG\\\\\\\/HFkwJ6jWimZSHbjj0LWHA3IIq1mOvidLQXhzGmFpU8oPgOL85SudiMZ0GFcNl95TYQOQALs23lob5Bu3ny1tdNcX1ILCDPkVJNrsGOkbeXr5aYUsR9owGlWZFzKjUVpYaBW9UZFUWmVXm4Ws2FGTcvxjllcN03+5sJWCWYhhrTco1FoBoUdWOV4p9xHSiM4lKGn6cXcTgvtSzB\\\\\\\/LOjNtUZA53N3d4FG8gEWVUPxJ\\\\\\\/aACaQJHMZAWvmKaycrVZRbo63os49RP2tYG1x5E9f4tVzaozOqVCYU21g9OOoCfnIfvSSumjM3VkAdfVC\\\\\\\/17GYTFYa7qrJiNLJ+e5HxG2GDnYDsk5Oo0a2HKwEMhalqCKHjP1d1SeB8iwQr1t5aXL2LI0VXVtAJTny3XF3hnmqNsbx2Qtl6MiQU1gIHgLKzY9DUEimkdTJVxaqQbw6IJ7fUVJbLorFvIgA7qKKvFupP4MZNOPCIL8bDPYBCGWunLZdJQECJrkOhvgFjJtP3bv1tKg9kQcfcOCmJXD1u9WxN6t6tDNFFURU+NR\\\\\\\/BSvLj6mI88dmLkASy+rQXYme7bLQCKCRhMDt747jXGY9QjYIJvz4TNDu+tmgZIYh1FpmKQcYwQ6uFnT8eSjFj\\\u0022,\\\u0022iv\\\u0022:\\\u0022NcBCIox2C9KwlIPDtep8sw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022939caf2cbac2c993467cbbac7cb7d6dc95709234ba1f6927ebe45db2ba7b4c19\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022zj2uhDqnW+yvb3vg\\\\\\\/HRs2cJGN\\\\\\\/OCuds1jnkqN2OLjQoHZZ56ltjGaa\\\\\\\/HiIGhI\\\\\\\/N493mbmpJ2VTucp3D7+\\\\\\\/l7+nePDg1Xbjp+kjg36P0zZ8Vpe9Fok6C1zkqQgINzzxw59tLmvpmO6hrKy+NtxGu5yoX5r3SjzOPtgTWLTymq6Y27wynxApDj8ndw3ciTb4Xc3bBCiBunr5H20fVY8M2rzbThbwbaOluH48Oosndcc6VA1nzaCJjCwgqvzbr20EQtpVXthx5I2TzeFSEVbnt8lhS3hrhGOvdQSp+2qfXZCeVPiQ9924uohBYgPTU4MFtRdsgsUqrSlKOF4Ogl+tFCh9G\\\\\\\/r1gWTuUrMLjDKKhDuhHffWSWZRV3vojs5xrySWMT0Vwi68MJyVJ7bYZDz6Og43gSIrYx5MU3YUeSk4BZDk1K0axa234N6QdhL2qxpeY8TJNwCaJg0t6ASvog+wqHfAOE8\\\\\\\/R4PjZKMkt2iMhntokgu3WxdU4dpIZOUF1SgSXKkW4elr1u3vkq4AIL1wjyjUsfKn3F0pmwypzMWH+\\\\\\\/dNidcrlorqqjFNrqxMAl0Z2q4xHn8B2kFf6GCjX8j1JlMIgVyKkTtBaS5YyygU28oCIwG4RJoesmmlhjONgABijyFtS5XdRAV6zEoj5wXUBCWuGwsiU3H3\\\\\\\/c2GtPRDSNBcy1s6fiBgZIQw4zMb7JftDaZq0PyjvLFejuhzd3bWZsP88a\\\\\\\/63zt5dEQMm4EtH25dKQuxe1XPzBO03k1NLRMlzySawXvgepLK41cw6LGbOvaOu0punTpw6SsFcCyw76v\\\\\\\/JVf\\\\\\\/RwPPJB0lzaNupRe9QE6S8q\\\\\\\/aL5MwzfWUSKOZw9X9gPNlmkJVSJxYO0voacvzU1139l3HFzuB61pjsqpa8ma+gqWa3oBvEm3TCVQTNPXnnnpxMlxeWiNhbOBmIsfDnB4ARyzRY5bKT5aDPK0NRrZObCjLCrno6gI9EO0pbn5sE5KDOzvLj9eMRF0foszQVx1C49Vw9t7nTWblA8Pi02rMxAZ06Pqops9t9CJDYj1Rk5lLJqi7ssx9JbbAbgiWiBGMB2Gmg3XmdD5pq6rQ4OAJb8epIvd6ltQaIIYK2gVm5Py7e\\\\\\\/rSD8\\\\\\\/zTWGTZtw145qP+Aho3gW2NSx487uS838uds77cZ4TqoAxt6VuMCaxTHZHvOV\\\\\\\/TEhXC+tXJEMdrFQqdder+W56NjwBFVUQm7QrZ\\\\\\\/Ww9saSXdIGTrKDLl1OCzwKsvP\\\\\\\/s\\\\\\\/hwOK9YDBzTQF5Z9WQxo5meeTccK+DpGyMykFn92Wc5XsHGIPQjesLeavx6dc\\\\\\\/7BwcNgg7KpZWG+n8iRqv8krxjpjfCdIcecCQ5R5KAXq\\\\\\\/dOTt0g+QEAlaq\\\\\\\/celsquM1OcLCjgQN3pfYNsLWSXsYfgEqlBHcQ54tlgOoIYHXIqr0wLZVY7wbzq2sawarrSO5\\\\\\\/4TlA9xRl2NQHB\\\\\\\/teJVYPbIFcAL6d5suTVtky\\\\\\\/41M0ZhSd2wpwhXGA5L+t5R+2uZhz8vnDIeKJBrbkhiGGYJqIEbC0tuKvLAZSYTlW1JAVQJVJwb1J+DYVrCOBTbypcRJ8YJqxj3JXCmOH2kbB0BfjTRbqayMNdyBKKFkveDeaLWpSBw7k8VUbZzV9Fmr65rv0cDlkxEhIeSLvdEFp\\\\\\\/1sLvfNaHaXUMgyrIjRGdwRyT1jAdYhPwGWSBXqQpRPknu\\\\\\\/oFji4rL5DYs6YlnJAw3Bxi5Bsow8pMgOZgVqhVY3XhUBK2djdwstWLX8PFe1jXH9\\\\\\\/6gElrY\\\\\\\/u9Yh2gpasiZmN0SWr8NcY22DTlCeJQG6ui4kZepzTParNqgV2fDALAjQxgzXYs34fHMthO+y4pE+hE\\\\\\\/xV11nHOsoGkJS6dFKrXERVMeYwy6VM8gO3X1\\\\\\\/BBHtuNCqrMYaSmSf4J72LJOG\\\\\\\/UVVMsNLOeIWP+xwM18M5FMxs6UkOU4xP\\\\\\\/oNKyfcsOYfkYwQeDi9NVcSOB39EWJ3T7XytHNPn56OQ\\\\\\\/BCdTTd5eJXoWy6FTEL5BSPg6P1dtOgt22FPorRUS+Z4\\\\\\\/FL0tkpOiESAWkKhTGsAmbS60U8at0+M\\\\\\\/ktz6xClxGsAyTMZvcXjxiFK76HuVMXCTeCGsOWB08f+tdf8U5cJGn5xOJEcbJKGtTQbF5+h6wwblHf+2XG8TjjkRKBqw5\\\\\\\/45ckdCXYspUavV1q1nvWqneIDUT8DOe3t1693cA+ho9EMBPRcsqDOiKZ2f2kU\\\\\\\/tZPPSYudJDaC3RKQUK4A7dH3HKPKCBHY9DAxLdLsW5xUPECmE6LZjP4jA59Vo0IGZVke6TBFUozNDeIF0gk8tXjZKJr3fmvV5iOEMTGYYM086O+CH3TnTLHpeZ\\\\\\\/Est6ecu2ZR1eUvDAsjczCCuZvbjTLb28yGVPCyvb67H1WO14vRs1yqwBrUrpfnz8BCVzFd8R9aNmTMDGRPMnElfzSgE0rdaaDtkWIgn3OpzUGAPLDVGP4\\\\\\\/9ZoA4qj2NAFygRhg8M8g6bb+F\\\\\\\/+aAUW5+cIW9FKP4CR1oktYBADksztXPXkfgLeGtTzfS6HLpvSXJyTnFP2gDfo3BEC2ZSXMoYyt49DpVfbSME3Gt5eni7nm\\\\\\\/bKFedzL7UEcnW1QFdtrNsn6zKVBmKuQ24QHzKghMU2t15zJHd2qWYH0iSrd2HMf+gzgqerJEp3tgShT82tUP4vbTv1nc2h5fmempd3YaQZmgsMi6NErWmAyggFtkyzgsVLRKrOPOekqtWTyUn0eziTfiLZijbD3+2AsMoK\\\\\\\/9+nH7NcV9GvZLIp0EWcSqvWczG8B4qG1Y+ZdpBo8VRyKJK4YSNZGYvDHzIOhGrvNi+JEHaWVKpHA6q578VSKqZzy9PlV63aa3y1ios1m1+1yrhJL1xrFK\\\\\\\/NNjwmCFFgnDdFhr3CGu\\\\\\\/MtQclXGhg35slZxdTuLdYLBiyty1xB5OxNZzTzrwlQ6Lc9ID2YO6xI3GE5iFAi17XwYZINUbhif8ezgjzeayLZTNPCGDTYpwi3tG8XtqmsJ8LVOw1ZrLBxV8\\\\\\\/JyELL+yoS\\\\\\\/BqKjZeotJ5kWvzQhdCbP\\\\\\\/SMpHGpPBh+SREi12wuh6hEn9sF2JmGFX86FR9DODHGdlVGGEEQy9JsYQ1OCbtrSFGQ3gVxdEB8h9fxHSWBs8wK9OaBjjawocAFySNHuFuweZeIsiK1k1ZP4LHDch2svZRdIbGQRZOeGLVYlxsHfv7CEOcFzWz74ngqdQkuX6PDnKDIrSqUDK6dmFIrJ49WYCFmayJE9UY8DdVMVJn0Ec8X7CI0EV6\\\\\\\/CADugmIo3BIK4w1PUTmfzMlShgrEq\\\\\\\/ShKtDry5\\\\\\\/kqD+uuYWOYI4mwUUlsNgQp\\\\\\\/iX4zYZD8gvCB3o+DHmlzCwKKIHRcBtzz+piUDTHO4NEynU9J5T04IzCXp+2EZuV58wgzzlt44x+uE5B298quqCY2If2yYcxcA0+352\\\\\\\/ozrmx+qz+jUdXwmnRytl\\\\\\\/\\\\\\\/oR88dlbWSH1lIqLWKk8oyGxuNjRHLpflGlNBMu7N4J6gJTQvs\\\\\\\/7Gja+7saqrJFzfofKhEUNQ+7X4WoTbM56xnOXh0CY5LoBKO5WpTBztgLnqwPfYTB3iLIvi2f\\\\\\\/9w0RE\\\\\\\/MOmvUQKY0nRAp2WULdMnTxpEID3H67Durzm1Y16QIc81eQO9lltsO8VvcWeD5LAvpRSNjmeZfYhPvP9oqC6cz8SqjrQdlGy0yOTOd703MWgOEVNX19lbsFyvPtwFZxL+7Hnnbcc0wE8OXQq96Lzuo65u5t\\\\\\\/WciMoSU2VsdmkA4dNWkLUey40Lhc4E45z\\\\\\\/zB4ZgGbnyC3+bFjC7x7MNc8e19\\\\\\\/BxC3GgacEZbZTx6rGDuLmnOX2Q0thOw7NZ1G6Z780yvkuT7dArMWklQe1iT5UXIoB6jkA3R91LyofeSE99LcrCb5K7BGMlhuy1CbrMSrJ8E+HSGsRdmHHOFtgcLu2BocVj8ZXWY1Rd5jP\\\\\\\/saUobLNUh0XrJHTKUF0BNlehMwAF4TcS4c0s1\\\\\\\/0JNroNiK4t8oNBACDYkcaJwRS5S+6VALjZ67aT34L5Jomo4Ja\\\\\\\/I\\\\\\\/uNOsIiCN6uE9\\\\\\\/LJVAffdVLUtC8zEtjDB6HXK4ybEtIxkV8ZwJF4TEVCaiQFEYK6muibm+HmUWQ7t\\\\\\\/4mO91kh2N0S8\\\\\\\/Yo+65GSpP3bh4\\\\\\\/AI+saXDvEHMfdPHDOdlD76QwoTKtlvimGgoBY6jqKg5cLwWGmyo00GvpMAEjgxLIorZ5sAAnz0OSj4ESKEKa84RJmYgDkiY2Tx7rQ2kGqhqZq852V\\\\\\\/jC6YfYKMeN+wml2vlqjKHPKKv3jnI3HY3RODG\\\\\\\/5Usb\\\\\\\/v12fEbJvBJ3Z\\\\\\\/163Gf1ym1evYoJ4PVqBPOGnBRWI06hPDXUK3axAl\\\\\\\/3+iwcZXVoBcXQWfkyoFuAox7zosxVJDaCjym05X6QeUT1e\\\\\\\/0ZMUtacRjLAqpqIoVKTSc9Q41bP6DccjMU16QoMGn7IqvVVoMO0LQH8Tdzr6DDxrR+weeEqojVxlaEsfooiYCLVpgMWH\\\\\\\/XYZDN5IfTSk1jH1lNLYtk1c+7agyUiAkl6GWoR6ggKIcqXkOHp3Cy+kuw1NHVm9pOOy0akuMd9mdAsPBdliybuYTF8r85iP4eOihNt71HpY1zNxEY+ORM3OawRvnXKmCIwlymq4lG74jKr34jNJe2R+z4ztG4K8FPUOXgH3CNRDUE3tM89ySHdKRHl9YyBBGalipI3wDXUwe8BUJm34pN5r\\\\\\\/SMjK5TpOOQgB7PbLG\\\\\\\/MEP2qKdOBSXsK5YWXyw4+V9FCOE7m9IeLgMEyWkeOKWeFrOgFjuEPlqDmEduoVaRa50B775axGgc9nce\\\\\\\/qtzfhIMC1G4nc8HU1lhFx9VkHubk8UMO0QThSE5G3VV7g3yUuamQ9BtYk7nrPuLUH77IgQ6mhnmWcoU7ey4FZlhEmTs+QmPZ6zAJHpiLLbaO+mqJa7GNmpbquQMV12CxHQduDoRInMHEVICobp95GXGeuJ3LwsiJr0mB1tzuqikyNzfDHWevsaHxk1NXlffiVw95tb91kOUrGZg4piZsUegoYh+9pjOf5n8Vz1epMX9vCDLq1ThSrGGCpKYH86VNnWodflbxXXp3qm+s4g6tRgaK6S3giZIwL5y\\\\\\\/gZxez0tJ3MMm\\\\\\\/N5vpft2eNeyXr1UajtYxNXhagSx1HxSZY17Qk2sOhCg07dVf9IJK0LTyulnEpx9WRaHpS+hPteFVaWacP\\\\\\\/6xi2+eUwcvGVPKlmeLBM5lG\\\\\\\/3wd40e01Q1jm4lS0KUAdKasBjx2gEoVd0v+r3LCnMLxQsDtLgdLGw8wKZEDrHYFNep6SPEf6y9YbRZ7o5Jabu2p7REBe0UsekpaHoi+wUnkIxHzEluhoZAGi6GOgBkoOqAzxk92\\\\\\\/G1hA\\\\\\\/yEzrmH0fclFcDqRRHIfPbgBbt37HnJ3bwrb9wq89vR6q39afwU1KROogZOwFm6ui8OJwGY8qTL0WKzljCwIem4wl193eQ8tseFfV6gGnNoDnEKetRFetN74kaBtws\\\\\\\/r\\\\\\\/pjWdHj0oIvA7r0918nk+9\\\\\\\/uy4L37L3ZAQgGe4sUyOc7MMinjcSHmEq0f\\\\\\\/xX9acQy\\\\\\\/\\\\\\\/0EO6NXqp4cM+9sZxYDnmeI79mmTdKhecWFcJPsCmc+HRtHmYmxqOVClGhsXxZuVgB16CoceITiohhLGT8gVCWu7qmelUCzib+zCoODNPoeRlrmyNR\\\\\\\/eZgRGeRupVAnk54kSdzsbsr606uci1+7MrtotMtL7Hx6JuUiAJp33hj2juUZwpg5tgSyNzRGPWBWVk881fgqLqSApyydyYIjJlnBceJi0TPsLZmKGYoj5mA5qaCyJmQ9ly\\\\\\\/wbUUBOtKCh1AwmKkz0MUAY6miTskQ\\\\\\\/JGspdoTBrnE86Gv4hxH3SCJzTJfa3C9\\\\\\\/+Uvukkrn4R0ALfK92A+A8RJtiAPnr+q\\\\\\\/Haua4PsYclUnScUM7xBNpLBG98uQNcO\\\\\\\/5W6NL11RYZKHsYFN+isSbEzRRYQalEwI04fXwkGRjeM7Pz5ml8idNXvN6b3ijxjD6\\\\\\\/IBCeYsaWODhbtlITcIR3\\\\\\\/7QPLCkxvJ0OWskzv8V184t9rgcHqhqRONYUM8IfIjrKaecv09MkTCS0kqlc0RBxM2O98nbC6O8rn+B5rmb5PteAb3+Ceh0drLQwhMe9glE4YHD2wo9YX4PmiR0pPbF36VjI1bbmomOrq3fM8CMmf+YKU4cgayNm6J3CzHJlfWfvk6IiKDieb4kuJruMgAEijEQPQ8+53zApI5uMQaHByJxN4AbSNoNXc+vYg+jPPIw5hQIr8ygrEFYoTwTYB7bRTGgZiUVYqBzQA\\\\\\\/f38QyYpA\\\\\\\/fpMe75B51tS23WIJJSNOuwZo2oRC2cF7v9356mFca1PEZGJRCM0w3JuUidq+caJgqmFIel2\\\\\\\/pSBfCDxL63Rc0wDR4QwOPKxd8R0IjmYQ07spVHy02SNfFTcUaHvBGu9mAX3RrMY7pBgVO2HVg0xAPhNLrLFEUBO46IanB9tMVO7lXaSn7RtsySyM0hgyvfdJhmWtwiwTUpVF8HahWOdp1hwwtkllMzaQm3jmdJWg+By1XNC\\\\\\\/KlZu719NXip8TpaY4BFsrkEWahN1mLCG3FlPd8Icb98+Gt6A3+UqYLK9+thdUaFZY14DneGERWFnLCsXBoR3SwTtfxqM5lSk9UURruMF\\\\\\\/wPYrRxrF4RvJZKthXlS61aUvdkgdgQnZODEdotR7YCTyJaMRoAe31yM8\\\\\\\/TVRvLGWHwQzAK\\\\\\\/Tt4Gq5Q1CRjcO9cH6pggiqdtTfRcoMR8LH+F07B2tzJak2\\\\\\\/4O8bum1vz1ZM+6\\\\\\\/UH4c6uzVjM1mCMndaNkbu1l1EfoT4ZlDA5J\\\\\\\/+IOgdr7fZihyBNxUslNONS1iPyIZeNth4GFQGb6i9bBtKVq1ebZL4QN0u6nW7rgCXYryFnwk7VQE+BJ+SWUhrZ7YI\\\\\\\/hlyk+uB2zRdLhgSMU\\\\\\\/tLpQOaHEQfIH610G8fsQq9Wn8FJtV8jiBzBARDITahvd+vKgIHzY+qhA1BYD1VxBb\\\\\\\/493KlG6NAvqF\\\\\\\/CapK9vEuzC6Q1pTsnlaeXSdUltp6\\\\\\\/7OphpLoz82QH73u5m2GxF7Gc8MZEToL\\\\\\\/dp3KNEuwAX4CLdqfdh+Lftz5JkXlvMIP+ov64hY8q1+Y3Kf+ZR3oAPcedVyPPe2bsGPYyDNUmyV+2stK6du5g5iFxXmuhNXfkV1LOx8\\\\\\\/UF\\\\\\\/kYrVyqZH8qiEWDoicuqngIsikEC7OSqtbT6KA1IV11h+66bbkABY9ZL3XhEGPdT965WnCfLq0WUOuI72ahpW+Ld\\\\\\\/a\\\\\\\/iD19cTmckRGF3f+YLZ00TQP9y\\\\\\\/jxIQEVdFF3C9W\\\\\\\/VVFWwPb9DYuzzgt3jkHgiVc9Z8f8mIK3EewSfBbhsvZgEjhdbZL918fLem5mi\\\\\\\/LICprETZl0v06qvsVBivVO7ywtYmNXr7lIeG30VFcRQRg5eiPcXOIMoxPE+N9WECZa9dcUKd\\\\\\\/FNz3QbSS779BHaf3GapRgW5hYldunApkW1g\\\\\\\/03f8X8l0t8QuQXG7P81bw4znG+3okp6C8krgDbhhkA\\\\\\\/ucX9vOR46TzlcVYmKbYB8gM2UU0+WsK3cgCn+oSVh+mq6tUJ20tNcgGg7OixYUMoWGetxxOtO3BPh98wVFIJ5i9qpaGHWNrNkHcu\\\\\\\/m0ryb1T3TW0fpR82y8jJp2zLiVrKHr+ogDdxL2EaWHWBY41S5NmYIbdPPn4VQ4SSgqiy15zX9pnP96eIVzg5jOmWdIP\\\\\\\/rNbYrOPA5uZhCGV0DgxmDfAUs7RnJt6XlWvUUVIHMbJmO2ap88wxYeJ9rRrjKqA3K5N0zE6s6IuUMCTC4fd0xf\\\\\\\/DXp1a24eWEIVQ3u1\\\\\\\/ZGySp8osAYmlRLZuCNraefLgV+OaOqH\\\\\\\/nd25T2zE1ta3jyswEYKTxw0CZ\\\\\\\/DdbFCAaLMGHXynIB2pwDk6v1rqfYEMe+xC3PbDmCSMTU5WQJzBo2VeCnGEo51opNr3Ry6gz6qqXuk+lg5vYkCdRIvKZiVSXKZUmI2XtJ1djolt4\\\\\\\/EmCKfaPK8+Nmhyi7hA1smgJF\\\\\\\/3Sx3BO8w191MWkdmNUrP\\\\\\\/L4aGVYI9dnP6pLfac8bkJc\\\\\\\/SqExxqBV9zo6mHjXrC7QE0xgvPQHY7w3tTIfxHB1OkIuO83ASQn86zDwREEFdDpQt9aC7Qqm63sbCkUIS19F5i3EQTh6OF44XBzeI588DzXOUsmQO9SoiJNr631LTCd2J+NuWiyOYVdiIfQt9uRoMkrgbbQkozkiCGS63c0lvvVNzbXY59Iw6L27SdIAj1osHQsWwkdFAaJUT1ohWXJjfPil0n0n8qDXvgVtYN3CdUUe6BukU1gAePbJNLJI8befY0FDiHiJdpsFxXbbVgiQ9RFBplwhgc0JWyUv+i8Pu\\\\\\\/dpTC\\\\\\\/yvqxBlBUDaJYFSytVCjs\\\\\\\/CsRr\\\\\\\/3KZ\\\\\\\/oXn03UmSUaPS+NPsRsU1vPpwdfi7wIZ0F4tDI\\\\\\\/ZUNPQNPrwmRJeE7V0hcn0tfuF8kcg04B74j2IDdMu8nFzh4X1DRF6ELKJ\\\\\\\/D46SvuC8J8bwfZiQACRxN9Is00Rq8jECay5Fe6vw8g+o5XLP5HcMLIM4UNufX036bGou9g7EOkUsBA5IdZHP983WRjdncUPpyWNa2UlWR+J7RXMuXTsaBHlibdkHuySWIEd717RmO4b7coskP+sVdc0eDqgO0ExfB\\\\\\\/bJkrWaLZ0aSZx6CH9xxsEp4Q7osCWLj54vBsqd0A0e5gNXYRxaRw3GwtYSGQqjhKvGnMoTyqBdVNIeYsHfpZNEKq7IclXLBmAf4J\\\\\\\/P3FXxQA74McIsNH5pu9rjfs+ymZGzI0SdoaUymSyLyKwbrc0XRACJVsHobHtgRVnqmcDlz9W5ECG65VrSNnzuiTUkcwJauHig6Gyicpr50WV8CZ2pIQG+bbZOLilPPk1juF1X8c6atIOJqp2Rz8V+u7C6Cl4dtlW4bYr3pcdNhL\\\\\\\/wEjSMlijkpBi+kg0B7K09KuIURoqM3t0FPZbIeubEJFxrrPsUWywI7k6h1p8\\\\\\\/ECEs2TDEp5R5y+VxVPirAFFT4k1jNOWT+tmkujwZCZ790oBy43J9LrqHO3OiLS5CARBAC82Gresar7980OxlbyNfbtZdvNOB4dSyzGSq2w5BpXma4lOi3m4S6EVx\\\\\\\/mtagau1NN4LkLgJP31bP6UQqVDVBVrNKTcB1yr3Oy0WOi1fEyM+PMJ+\\\\\\\/pDUorLIP21cGRVVx2pxyC89DHRJCKX3y1Iu+O6g6mHLyFL\\\\\\\/rAq6HmkfAdKjOBwJEhOsPSEd0CPzrF1JicmxcF3HzFLRUROEfIO9rG19mTlVMb1gHAUmdZqIPAgPBI2KRczr+IUEWDzImYzhHbNoWKY3N4VuKiNjyH9ZO7QrxaBl+CfCODbHmiOOgADrOsg+uRnqFJOQAUqM++OuAQSuJKM9jWXCISjEqy2+T77sVmAiYLJjU7YQFf4L0Cgd0qeUOL9o9ikmkPTK\\\\\\\/IsMrldkpjgEf8+TG+NyAMkv5h3MUgDlZQ7ccg5OGTyvbO1TZE8LEgX\\\\\\\/i0UOjLh22nYdCdkQSz86E3gIkZuRYRxDCcUEai9BW\\\\\\\/JAgoWa\\\\\\\/LU6gkqi5+PknXJr8BY03ZjvnvnMltWsIHAV9fSj6fnV5OMmQKMRaz0mIAiOtpFdOo6\\\\\\\/hI6nMRBHiyWeqfxm7to7sNI1cjhlXDRDZAjOITGpxUvMAsGvmhRSw9VWMXLN8j7uQNRrkj7fJm0tGjdSV1s9Hk7KS90zQ+05XqHk2\\\\\\\/xCv3rOTIwwNHGJAj5Ux\\\\\\\/Cn1zxO5uOFI+LUgoLXsNckVk016NSBA6sfuc3+\\\\\\\/sv1Trp5TuQOOHX2WBZTT02jZ78shri7EqsAJQoSux0laQZ9hmmY4hL+tsoiQ8X0xwCWe2nHY7d10UiubcTCWjjqpYkEEKrp8dIV4IkIgNpCgLZxBgj5bUGamb7YOqEq8meoKnE1RLXmpm7ShMEzllP+TPqNWguRR1pszece+bunLol3xt5MZwyZ3mK7NUqQN\\\\\\\/hgCsSvHW1oM3TfhC0Wymk93hl2dyQ46kO304sopk1+uKCETDqMSpa2k+H72ZW1UnDTv1pbBqsAOAggU5fMR62KV95\\\\\\\/lhAKZ\\\\\\\/TWzkHYkWYQBNgt0Psphh5VmqpPDtJOQSwsJDAoJ0FDldKNFjvqmoTrgwWdRdVbtYLu\\\\\\\/DTsv5rZy5QYgwGvLke+YNSaz3+yzd+5bqDnb+ZXwAa6kuav8AH2gNgc0kdEB6mwvZzzRvutkPbHR23MR5pf6YxI2wVsFD7yqAteBx5pJ\\\\\\\/C5k1ZhItOYsVpSsORUproos5mGRZc6ZIjNyZFp\\\\\\\/msoxjBmfHkxjkjBUDGt\\\\\\\/8lia86T\\\\\\\/yUDxnPoBd3Jb7obK\\\\\\\/SHXYtfvZzv6QZmTnVCU2uIc5yhPOsKjGgIZQHHbMjM2Qeh3Uu+8eDMDQJETbV9ehWq0yxy7XeEvtNyvfA8dWOO0Ki3rcORcV6Ga7Sx30xyQp5OWzFWD9pWD1zcu1FceHxmqg9LlRIkwN9FwvzdjqFWFjRzamUObMcOA35ieXwuhHfWl9GLwMMW188C6RpGpw199MZ0bta+UBqrLqgLuqVkxd11oW3M6fhnLKgFvVNVGtefucdX\\\\\\\/C60IZq6cTnY98mfq4vgy\\\\\\\/Kh5x63QbupFoR1bNX+fNsXDhnqYkrPphrb0hHqco8n0qBKnkbfy7Win21+dXtiHw4jcnKwepCFqKdN73\\\\\\\/ioekfslMLGdkYSTNHGue1qOm+uqCZIh453ckAPF6blxPl0dvFklD\\\\\\\/aotSE8+F74aIyj6vuvjj6LaJfWb4LXDX\\\\\\\/C1CSw6uE3zkvUrPGROGQJBIPUpDGbC3VcBY0mA+0pxZy4r23llaty1lr9Ke+f+S\\\\\\\/RqAoTSS08OxkhyNy5arm80vkEDbSvtgPBFWeNsCC4cqY1N\\\\\\\/IccaV8CPcJUTGddkb5ZSEwODzCxsC7rzMgp0CevgR2XcoFcJPBLLod3p57iz1xtTcWyeSGlH34oHBdtUUrenBwiWKzv7nZaQC\\\\\\\/RMreWYJ+XHRgvCYyy1hnmj+c\\\\\\\/A5VzVT4wVgmEC9FuUsoUJbai7j9s18i4XM5OR2J\\\\\\\/0+xabIJqeLY50ZI7Fe10h872UxM\\\\\\\/OKf3AgIKgQ4pj4dQAHgRGsUGOmESRvBDQQ8Nf5DwZ9S1FW0LIaViF+NHBCtql3z2MtN8ol57UeTSHZ2HqNoO\\\\\\\/2RuVJG62Tv50xtQjT9CmWVtDGAAdIANpDs3T2RFvAK8MecHw6Bn+PaQW44A7GUCIf+J\\\\\\\/tDC75i9ApZ0dp1WVp+yy7HhAJF6Gj6hNFLhODkrIP8uQLFiOAFJh9M904kz0jHEUhLGBVvUIYua6Sfrfck94TABfxI7dtJJRQ9Z0Wi7Xp2\\\\\\\/IAmPsFuggY+8tK7HvZzruDyk7FhJaSwfmNbGpTCKH2pQcxkrTx0s\\\\\\\/whmzI5QV625SYc7sOu01AitSLnfyV82Gf4ID7wvN+aBRxswBy23qUDWhPid4QjJDTIhi6GoGO0rzjADw93Q4hUWdYujz\\\\\\\/oHyPPfC+tIVYSEzO6U9VQpSMmxZ\\\\\\\/OIlIZ1VCAXnlQd27thxWQ09wTGDCk6yYfkHTf\\\\\\\/B2ahUV+oGMSmU+DxG\\\\\\\/\\\\\\\/3JSDYb5n4fLwdCYik5pk\\\\\\\/ueSVzrPHgBIEqOX1ZuQswiSxpf8NT+4J8FD5Sg\\\\\\\/KzlNNWgQyPvp5Cq1juJUQgj6yzNPUGHrN\\\\\\\/FKHNFPIpZy9hjKCfvd61jj1+nRdNgmfHG6w+AWBKKT8z490wH9xbQPKW9FihuU+8UaXk6CWDXYn6KcqcUZ2pxYAjBx8IG+msdXCbkV9gmCce9EJgtBKxEJM\\\\\\\/XD3ktNPlMkVo+LyNo6hGSyto07jKrW2UJ7apHrn9eAWOS4vi5Wp5XvnJkpFi0jSOZeZf+mLQKyCOsvYZ3WQ8jO1Y4pRjXirGy+d8\\\\\\\/8nLbwrZiDQ\\\\\\\/1jr0JF8Da0nLfPYIGKQ+hIVxe1nUmLOqgCuUpqLFBiQnaRioY3do9xh+2oVSby\\\\\\\/ZWW74Kkj6B2H350tC00ZZ9RPlQjNHgErpT5mCkUAHu5dIzDehH06Pk3ovgNA64DT1Y\\\\\\\/u9lWZKFhfHWUPvcVoW0pdhdGnvB5Y5v4ZkiAJnVQLmEbfjtJPpxXGotxo89tlti+zx3pgbVb1w5TTXiqJKHuvA7l25icv2VegPkbeZSFR7\\\\\\\/kYOi6xHExqOUXZYHJcS3ZRF7cSg4Rylb7rqfNFKqaakPu56ueVRek8jrya2X7v5zgla4ME6hUAzMMvONDjP38rW1YBczZOwLVWPpN6XHa3AGiGEBnvcSqYsfXLCgk5ZwGu+MKuMp3fYRlXQ4NAFwzKIYuITC33W6Dcz6\\\\\\\/CRwhLzCJajY+CGwPpF1MugEYE\\\\\\\/e1fcbYS1w0IHVVwgyF0IZsU7SzhiAzVmEISqR8PqijcNivxy3qxjDBdrwBURQG8Jg1o9eL5SzqxucdOshQ6Jmk2kFAKTCVRRpx3WeyWsLxST8+HPN3nTaSSrWF1gnUecsqVIZ63OlLcmeJzIwR+0UN9lYE61jzUhHEC0QWA4LpIeyQ9YPou3f3UE2jfhZO56pMOJ6HnpwQRv9btlal4QGoudsjMyqL1RFdD34e2u0WJ4lsZY8NYalTjY\\\\\\\/LOFMQlxlxRcoRYVMQLGQo2G+OSdkpah18HLSpF2+FwadwcTybJ5Lqib7+06GznjesLpyukzZojrp2Lo6xn5uBgAnljiZZ3+gtXBsrKf5c+Sh\\\\\\\/9497N1BOV02hmaDXOv1z6Dltph00KF7Z4xGXTP44GVtxxYsFgMKlCapgVAV0tjOqTTYPM3KMHq1GpzMCd1HIVMJBGU0gERSGeKzxTIWvZPFq07hRu\\\\\\\/Xr8lqGjXiqgQHOXwt5P7kX3CMCzM58gqEEeRUhjaMqhTkEB0W9wOgT3jMQTExTcvDKEAoNJVWa0uUcw\\\\\\\/EOaYSfSDqaa30ATIEL30dhMvNfTb5ePsUNg5W4ufvmVH8Y8rhoDtr20M9KOUHYthmtewWwV89cvQyMNQuLSkWabq63IKdn4FYlAnM+oVPWaAIn\\\\\\\/ZWEaUWG+MAq3wp4Eh4DnIavM5RjgvDmQ6iFWoNzfiLA2jjDty\\\\\\\/lMyO\\\\\\\/yGdsTZwEiApb87lp2KjaBzOIJA0V9Rwkkr2t\\\\\\\/jKDyeZbEdGAj\\\\\\\/D8IBR3Zg1fPh6dlag2F8pToOxKuo\\\\\\\/k3ITZ6M+HaHtQSPX7dNgAFjI883qR8CNeOG1CMV68G9ezNpkrpiIadTvqC9E9YTvcOlkKCcOSlb15GRfUa\\\\\\\/jKLc\\\\\\\/SosGbSq6KYTCPADeOcRNao7AIcDrGda4CufAjAYANK5YsqKyZmuzCrikovfJScEuATuz5WIvhjkwDqWmMnh3XA+\\\\\\\/gZGykK4SD+w8OEQgVeMUf9\\\\\\\/4V9hKNMkgyuXjx5c343olOq8JdwJMYdx+3lfHdEvOJcuXtD2g5ltQDDxLME2JYXJScrtvs1TSqI7ktemT1g0nZ+0UIGz0SkIMmmveAShS1mrsO1Iw095Lt9GNDO8a7HmOadQnJM5fe8p2teyH2A13tQzNOEWEKnQCTAGSZ0njLEbLwjnl+UdS59lZSdMe9Y8Xyl+G3ZCO8NrNY4UDSBqxuTzpvYfHleX7WzGXRR6P7KgW6Zh2X\\\\\\\/rDHeae1lBK9nGWsRxnoEuVhvb9wUradxJeGO8q3mfnhHihUPlcn0wjRz9I6ciKnF9Iv7XBMDjEMmf2lcgkZjrupNxxhVBstEx8OCuQ40MO8GCcN6ZFAV0RchLgP6TEDpa1v9hvDNS+E6U+4p0a06hijpxz+Vaw3fZaWd7OKBQ9MTXQseBfEXLXMmED+wFC33OXU9x3qKXXcNc5qPfRdTSsesxNzY26j9j03IsfAQYFti1iElAo517YENtCwMXsczxIt+\\\\\\\/Dn4tOmVH1JANHzJzxL41hLLe1fmEYeEAfObaE6eSY3O5BKcW2xxvxJrg\\\\\\\/oaORG0+4HY2lT2uSEEdg4cuXdQJYmfIvbhv2r4AsHPsiIWCSkZc0Ab8gtcSRJCDHtPA9ZaS4YGgI9twAEcZKtHJPnCqN7ID6LXN+xoUCmwft9rJUs9yCGi+H3bF6PXGbjb8TyxsFMCZuNHvNlqolY7Myln1IiD74oku22PmfRD9JVcshkcI0kAWCUJIsDBGnYa1KpE4Pn0NWn5XB1iklpAna2rrVs3wcGRfVQGCipCpSods96\\\\\\\/OwNsl7CP4qV8T2ZcrGyCCYHlg8Bl5mMeZTf9ysB89Ml2eO6\\\\\\\/mWU4X5vbjU\\\\\\\/EiClBT8dtp2XYLuLQprzXVEsnnfHf5VPX0NpqZi6HcbBmXKQq66Km4UYGUxGQ3efOPkcs0+2RlQFNBJNPXxwiJlTfunYHZLjExuK7CJTmHR+iMCAS+Ycc7EUMQhTypyiKIyeKKlt4UNaNu6fCfXtLfyPYpcqcKbjGqu95z6dn1yY5tm3C3sgwTlxUI3yjjG6qfOofJTN0J90lzwZE29M0Zm+TCfwDSPSiMesCN2KeHbRKUampQeAzhVWbkbQLcebxNxm\\\\\\\/XUh4KuUrFWzMyuMBcHj5+jSvv6TRZcuppAjgE6LSJLYwzVcj8UeJrmvYvzbuYcGBju1nuW0D\\\\\\\/Uopgt5LDOePLPXWjEdbn1Vl0LxfAK3O8DKL4iAR6t1W6eHgOET\\\\\\\/F8MFUJFXsry1HKsDlmuv1tsGpDF6wdTCEVsEzQjsp\\\\\\\/GOftHFEGYYXnT\\\\\\\/YbW2mFuc3aWIcRrY3pYMBgpoXA2nkWpI4ff38YyVCFiAaS73KaiCwOAfl+8R1gm6VtM1IGXa5CGMN3bn6NQFQPUR\\\\\\\/3K44xtYSNbYOteJ+219BJap5Q5KqE49OisnbOmDcZoYnpNBhC1vFegl7AeNGblFUv8twMZzPJwRBTD9WQFF1BUptFp2MNFuBf826mtrEVKPP8iwPBWcuU7yI4jL74MYks53YlcXnldJn3IBIIsEDGVRaPlWoTe66dnZaN1zq+AQ0pf5kXgrgswN+Hlk9zeKAR+Zwgaoxfy1M+jOu9Q9RhIeyd0HEStQgqOtpZDnLC8zV6xLwuBW\\\\\\\/Xp7ReOHGhvdqeBI63jzigt0wSY5hj6njN+YH8OHBS6bOi3eyHZZuJong23BrrBbBmAsqE\\\\\\\/7GnzT12TMq8d5pFLy8kYNLDfG3oWqO1jj9IPhmSz7hnLiVhgTy+AE9sNrN8yzu6A1o3fqXGMnCOJMIv32qb1KgHw8d0mWVgR79C5731alvDvd72EMD0nUg2ewCtRMBA2CvDgqGwLUhvHHjPrhm71yJnWjqnJYQo7T05sQEt\\\\\\\/izJ8uyJAPUpGsS6lbWuVR3RpcK\\\\\\\/BEEkYkdoMlT+xgePLC3M7yBamBqf2Q+Zr9Hz3vnUGjvON\\\\\\\/xz6TtWlX6BfA7xie1dXcWkeW+lKEh\\\\\\\/LjF500tYEaxCbgp9XMTtdcuLMGy+hhdtx1XW3iW9G2dV7zK6xnEIq\\\\\\\/AgbuDFHDxesMPYRH\\\\\\\/SCEhVwoZgUJDwoYk4UjSUuYXzSmI3rddJ6LzxJ8rZmpAaQJMZaTOfTegVX4AWLLh14lMP3vN4hch54JwGX4BaDdl6qmDbVXwcb76raQszTi4nIzZSKeiiJ2cjC65YPWZkw0z2zB5RtXVGrnk2P8gHJGSCtFEKuUrPMQLzAE0J2MduO2An9U9wv\\\\\\\/Ysyj5IB+8zD7XnXGxHfu79y\\\\\\\/I1v4XIYKXt3odhR59kX1CE0SKqbi2qGIcM43XvvPnn88vVPMrXcz4uT7yebuDcfPyKIwKEec\\\\\\\/FoBvJxC3oSBUDkwnwpHwzKoXptpq83qlUStxxSboyNGyyaYnIwhuiLA0j+GVKkGbUcuvPYfD2wM\\\\\\\/AGymF+Wsb7mWfiolmFPilg4KParvXBt6PdtbORnFgKwCGNyNY+f1O4RKtEgbS9X85fnCrs1hloZXnKJ4HBBuqhEHTd8yuJ8q1JgKh2+aLwk8t19ujKnWxkfWp01eE3kgYAEdDOQkU0N81dy3MCZfjJXiCLrXWmWMR2YQ6jT8g\\\\\\\/wk1OReu4QRZFBd7b9qzC1Rg0hBBbLPPMxaHqiy5rczmJjDUQOjYKLE6S6vQIooWq60+osVOns05RP+mjW4hiBb56pndpN1TwSxlnWF555GOy3Rn3VA0iGNzPUWhLA\\\\\\\/+ydzKVWTj2l9A5jO0pa3OM\\\\\\\/zGWdQil4t6Io3hRiH59GJF+B+cYrVlo3onOxW5LS8hcBfHuLdmwtR6azSn\\\\\\\/rSMRlc94RfNVsCySMQ4zulapqXU58MiB4dgAp8zMLKDx0PlzwOyPp3g\\\\\\\/\\\\\\\/OGxXY9S\\\\\\\/pwySdv96IujoUvG0zxscD0kvV9WzMboB72QAJLSaneXJuWfJhbm\\\\\\\/0TAYyJv\\\\\\\/ey8yCBjozXJJTlgFeeQT\\\\\\\/jUeo2vujBsFIleTok8LQ1Q+OzdtW0RJ++5+es+AG\\\\\\\/WmEMYfeftTfw5qTtxK9bPWg59JsOVSNV7mkj5jnI3KRuFEYhbvGCDB1GYpLIPcMCMzuIgG4cXSbC0Y8LX0DdERhdpMrR\\\\\\\/4GEutl7pK39Tm3ghAaaxTiRR0fxlMR2wxHBpMXP\\\\\\\/YDW9O10Ik\\\\\\\/BJwhyzlLCQIY5jw2T8TJm4DISowKXeZbi0zp3m0k0Wc4q9mJdAKfNNPR2Gi\\\\\\\/k2YviXcSjY9UgtNK37BKnh128BBw56j+nxUjxSoX\\\\\\\/Pv1Oq6Klzdjqw01qg5oT19GFmhnJMF7fu70fGvYb8BA8EmwD0YVzs9sdISzPXVC5PH8ggawkRgQ6HRA6pyG3RMCceWZ1BtysPXcXlflk+VZzhZE59vL\\\\\\\/89uoqvNr\\\\\\\/RUB9XGU8oYKdqAple6dB4fBMh\\\\\\\/fnlP615BbTWHe9qcM0UFp00ot4Yusq5X2mdyceyjTOyzkAdfihtRDfsf2+BJQtPofvNnaVukHNiuRTce24HKi\\\\\\\/L\\\\\\\/UU3ajWm259MsLqgxxhg4GchOR6g7Z7UugTmfiF8nzk2xu2CVNmtOfSWZxBx3e3J3OHsBo\\\\\\\/YsjWk7PhgMdyEXXf+fMIc4DE1ZqOxIWm\\\\\\\/6k4albCdW18fjCRmg7KGLXPPz4GYlIR06A40hq+HrQv+WDdiReHVG6NmGmv13EK4EZ8dWtD7QsTfG3eZ3p7tZgsObIvRx9mQE\\\\\\\/jYPAolD+oZsm0sq9YTtI3RYbpl3qKAfhiBU3noLoHU2QhJ1h68NCblJDS2aq2pbPC0\\\\\\\/nHWMbyLPLBzGNwu61qIzqNigUepwPQCr2zbqcubOjZtaLFt9kWKMaFiWLat7xoZPtr7LNZCUSiwtyWqITFnd3zichKZrB8IulMuHOKOMujYSBvnSLSnGi2MlxNpJP+SWqD44kzfSIHepV351QenII8FKqxA\\\\\\\/LT3Q03O86hqtYf5YF7LtpezG7ZQ1yeFGf8U4xL4zqrmGpWWpV2owLR9jLW7ShUA7h3zJTvymNrWyONKvS2iOgYLSzqwXn1jbcg4BwLiVXnxd5LEs0sygQbONnrCBFguqmPQfkMe42k43YM5q3y+uX\\\\\\\/QPKZhXW6cv+hJIvj2vzAkiv3AsduX2ciYg6YsTWFbmRZqzsM5TPWvXz1d+wR4y+TH5UBYHjEVCpN\\\\\\\/iPeUb9HfNn0V1AInnhTgVna7k5zDDzNDRJih1bRgs0o+O4hjVgZJjwo8CqXyXVnwT1\\\\\\\/faeC4qdMXBsAxCUQMdpbmhEtDnH63e6lKngHtCr+MbYbQWy71LvBAogDCjgQPyjgr3jn1nzoyG+2xitqzKr3vKqFz8SyywkEG7TDGLp+P7J\\\\\\\/u1DY96ho\\\\\\\/iLh3OeFSfrS\\\\\\\/cFoF1yVop6dp6dxJeoBEeA8P79fdeYUtckU4YJpZU0EIMwousYLDQX3716zDFDRuVz93C++EuNDpEDzRf0i3q1Aep4HbLv4al9MiqnIsZh52xYG2pLsu3lCMbh0GLejOCQZAcDHtYAIzhjr06xwjsiQGXPNk0mj\\\\\\\/rKxaoUwLE+R24ZLjsH2A8wavlCqZEYarA5GF58TvgfOXf3SvINQaUk9mWDxTstWtSJ3TiVpG9Bwax+3p87JAgrNxy8GGN0ncnwGxHwI3sEygKq62r6\\\\\\\/y+w6OGIopfzgN9l9xQptUhKn8D349rX0E\\\\\\\/HOjM+cJMfhqkOd5ghpD9KlO9siX\\\\\\\/ZCPPRtMboXLVRMar3BntMqsgB+nSwo7BnDBawB9AdiRwG6LzQAUqpTIxikKZ4A1Qpnuq\\\\\\\/0VpjKjNYWwi7+6dQu9GkR3rKj5+hxzivk3aW5ZO4kivzx9bRqhQDHReNFOdw5V394FT4rqvJx1iWkP5FWI8mQUnG5adbmiBirJtWuLMH1pn+7g6I7TvzLuJLN9nc6A+Oei+HP3MHVJ4sZHGw5\\\\\\\/J9WZ6FVabEQ+A\\\\\\\/ilaAtyJF3vkvZJIKJ7AoFLsOkxJ8ihJDCbtOTq5hNirayE0BXM7ou4srTU7q+lE37an7K7MOXT+jD\\\\\\\/FpvodCuGEZeFaFjGLbHPvxtrT3hzOkGqLvMarqFNQrOYJ0hbXEMQb7dyVPdknkhCelYb6WBPAk94VGYwtu1GbigFHnSuUHjBgxG0ZYxBX7v3KsW3myVeMmVecgse9CMsjdSZtBPyaGYHDfc9FFikuPvqBL4n0ptty4rhdH5miNqEiRuaVzZIHPs1b+hIL76beRfMPVyn4OjAjva4j3xPKJtPs6sfeGpoQLNYPKOM8m\\\\\\\/+MC9mPwMD0kYwcuM0w3hpHsUVeuam4zuBZHXAOwyrNZqJl8HSRd2C9SUfftKQocdXyx7dzSSH\\\\\\\/gaY+YZeWrtPR2buqBBtGd2R7w2MlTadu408INBW15vvd\\\\\\\/7OFySVVbX7Bv+QHB8BIwPmGor2BX\\\\\\\/Ax4Sj6gXm8Qzw\\\\\\\/YJW3ftdn4g4W68nhXzkiYyxOb6kFIMbBXgbsbPNL4mq6xwSH5PZfPBU4XFx3ImDfRBT8hq34FcLK8M3\\\\\\\/i29Ve6JTa9VzgyPIgeL9ea7CdlDvWMlLMDv\\\\\\\/43QW651KNuN+TguexjXAQU5R7iVhrFxwIt8juHDaElwPVgXuSheqce8eD5itiPPuL23h3T3LlQuZtm9EpJsREu0hAfHrWmDSq8wJUa630bm+\\\\\\\/DdPTbX9sFAhqCsqzYGqQT3lkoEG6NMw1wXw\\\\\\\/eDKgUPCNg4pGooxJveYUYoXwFRMSOngix8KJ5XGhq18La7V2J2LkpDJ+hl\\\\\\\/PtPnRIo63gRYhpozeKvED6r4uaPbM0lu1Nq7y7GqX5qGYGB4+i7eKZsvRm9jLMg+Ie14T2CuxdTmtWdnGRKXYyQj8EKtK2\\\\\\\/q3pjym1mjE9bsuX4p248jNcXnvL+Y0kuFidgC7qvGQX15l2Be9AhjpntzX3NXnrogeFjWGKAR6o59cyKQ6y0YjVJoqli+hqI2guhYbJ6HqD2f+CnG7x5Pqkzg2gohlKI9l2\\\\\\\/2kaydM7mqXLeTEmzigsEEy6fXJaD7jW\\\\\\\/xVjqVI+pV\\\\\\\/gKCOkko1DeqPOxPpS+k\\\\\\\/Z+UAzuF6aHa8F+xiPSWhWG1ckUbqY5JKoi\\\\\\\/IorknKZeCVWRoa+FrrYFmthw\\\\\\\/5stgvzBoovbmuMA6kGmGrUYjJa6iq\\\\\\\/qTu8hojkPyrir7iZwJoW6M6hUbEApnv3FDmh\\\\\\\/zYCiwwTGyx4VyqujRzfh7aXPXp9BCMHUlZ7QOA3hsMpGC2aJdWegOGTK3UYDx3OiPOKb56Ei6nzSnp2gPLkrqAD4DoMkQJPOxoBdoBsQHyBDI\\\\\\\/8ejY9HRUUhiHESm8tlZ5V0BIzdbQj+CyA937BYAObditMij9jftq81iNfCuUtN35EDcbmwMEqQvhTIcLbbbdFLSdyYIFGqUKOYD1ue55ukPhqlEyPgBAyuTttaEea6q1cvI3sVwIWFgXlj1c4L6VKaV6M9em6frJk3PDnMjDcUjrBfRikZH1ATjfmj4Rwym8jlr7jGx055tqNbwxweHDUDSt4Ve0ilfj3xoJUC1HchQPWl5AH3SPSmgHPg+ZxdRplTMX3GoLm+p\\\\\\\/e8xRJ+GqShlp1F\\\\\\\/rGuWanz1A3\\\\\\\/lAsXyWMBuMFj7xBJ2dzQ9gvU7v8gTEjawEEW9LWBwMNmHfyY4fSuFSP+99AO15hJyvQuCEkERYfKUAJYtW+utvCbUdsmSoa7rBcjOUbsiI1TMFIAZxs5j9vDBHeQUzUwjdPq9Vx6DOIBFhbMOGrhHVLdLcW2yGhGMm2LpA\\\\\\\/BirfOP4KcX6vUZqEMk0q9FPYTQL\\\\\\\/IQt+OxqEtelzGvM5cbQuuQQrJQKBAwFf6ITkFzmv559rx7PbYJZwvhDnateh9vO5VtmT3Mv9PGA8P0aoxLxtIvsZ5N06dH1agvo+4qIAZWZPTmmBVYBQ7iL7j9PTINI6OPwJsm0tgEaFPUiauvdxHsS3EVs5pRUHvbD9pw2UpOQ3PsRH52mpY1ELtRSzCVMB0AaA7ZpFFG9ewTGvhOfp9xppLP2v8A2x6sXerx+HwMm\\\\\\\/xMATFJbIBn7VboxfF1lpG3P0BLMwQc3CqTkhLMGDgvuZNlMJ8\\\\\\\/ynymb\\\\\\\/io3Idb3RTEsmkQu2r1DLtcpm\\\\\\\/QFDoWoun8IRfW5b9erYfpuzUOYCIohdgrnrcqQRX4dS8Jtxwvq8ZWdIou1\\\\\\\/xpr7ygYGGO7Hb5UrnORWYXAwNUxubR7qaNlU+7UcFkt96MdT+LrMlz7suzz0KlqSWUmIJUFagKCsWXlCbfAIA2wquUVE0FkacE9NDVu7X2yj79Xifi2ZxDb9K7B2o\\\\\\\/NZWGOmHd\\\\\\\/O6ddXh\\\\\\\/bQcOoc7ON09QrwazbJZOTPUWWxBW9TQ0KA+vQet8LeIBlg4bLhhgxCKk0uNcd5DDgewVOoHpXYFeMFLIuRbf3JXXIOg2Ym5mJqm8ElHSyq1LThJvlHQRuk0U3zhnjUrbU6EpsUwCyeCy\\\\\\\/Yo7b9uJxEorF0OFHBRdGIouV6wbxrNzkTbE3uqQrU6Ik26GQdqVaRAmMohv7QsB\\\\\\\/\\\\\\\/qJmw\\\\\\\/CEhLn1QHtM9MsC+1Sf8TwV+TAYwALpFB9dgIHLMZrfzm7Izpi64o7Kx\\\\\\\/LBTPd4IjwYoaOYqzCw+oUhbHQvD4Yh3sCyvxDhyQUZPQQVM0Fl6wYBxVLsac4rOQFCdxdbAKWGcB+ZAgS1GthDS61DjUpzISFN7AtBZ\\\\\\\/U+gL23vIpNOQ9FNNoEHj9V\\\\\\\/3narl5bGlLcGuP8DbJ0U1\\\\\\\/WZ916W5Hc2Yq8nEljfQ2Khr2\\\\\\\/XI6ZYmoKMyY3izoAXRsjDJwMjhmt+qDvXHlLPgcm+0bkYR4872qqH6ZETHGUPYnqOXZL\\\\\\\/w8Urk44rNg+r0sxCJiWT\\\\\\\/jxVoiyzlaxi4uJeXoypVFFcch+Gg9hQa102Mqyufug+r7kup5xkvjM9QvA3IYPs4D6zg6CTlPxUHslrl\\\\\\\/KRUc\\\\\\\/jz9tnwC2AS9jwQTGOxLR4utgXqrsECuUnexudYy9uokL\\\\\\\/MLD7nzZvSGpgGAHeZT2ro0Ea1T3Cyeu8FpMIlZZz\\\\\\\/\\\\\\\/UD8Xg58CJoUhmEtvN5Um\\\\\\\/fX5vmNs0mdRK0TWOrhSxGL\\\\\\\/UJ2FaFs43TfChYwD+F7BePk2+hZw7u9eIBZy1zO8sonhgdVYrN\\\\\\\/9sn2WnAzWjvf4Ls8syYthmIqU2NlCqgr+hu8mutxyzwHxqH6MARUXco7cP6R9A86fLLtEMwNunxWRd23rx0RJ7NpImuqztXur6548uVg179NewUGZTPpOK1SHl5Qk381QZMw6SFLs5QSlFE66H8dErjFSefTAKwHoG4qAr5zngcnpUOtMiVAbzdUOT7xI\\\\\\\/C135OXeboTqKLRtmx3EBXf+yIvLwqEPguadSRme1hAgONqlNdTb1FcIQkCJpnbL1uAxuxUlFGUuuLjZGC7K7CoWA4I5SeK8MmQnMsHnsGp\\\\\\\/4g3MUzPg8NZn68Q60QWu5nNcNcelpNd3YkSvnX2VfAaH6aC8h\\\\\\\/ug6dn63bDQFa8hp28rQfOUjLSv1SKXMk7J10\\\\\\\/pW1nHUH4F280FCDfBv2rxJunZIvNketI3hdiI5B6XYzJNcUBF1SNV+yl44SvffuUA1gachcuGGLI+pZbBa58ntJVm2hOq5UL1M8z3f61pcoKqklka4TDixMcjbexP\\\\\\\/rw+K4c7lF2TCT890zoiedYG8sCwKXVpFZELjUqana5wlqItgtwPeGYzpObeFRbRG57X8aQdKpHuxh5EZtRPEV\\\\\\\/ws90zc5znN4dwgPkJJmmqTw5QoqDTQ+Z0wWhfd2Wap7Aw7ZF3PENKrfV7onFWKRyp0suSStILJb7TWRGJgnN8t8Z2ncAWxOOzmQJDWaE\\\\\\\/5JiFmbRcpOo40J8YmZR6C3aynca4yDeBB\\\\\\\/hNiLJjBWex6tyclQr4SYo3zFG3CYdas3+Z9xr4J5ltNO3qKNFKmwGVb06pWk0rIoM9CsiG9Xbg2sQELSnTuhZqqIowWC9OnP4e9rXQKRElaSDAO+nozA\\\\\\\/GvbCrRJmnsQnfXNTu64nlakYX48\\\\\\\/97Q6kR\\\\\\\/F3yP81VVjshghbxZZWJ\\\\\\\/J03AuXn9ZEtuFq6ctEvMiK5fTuHcLDiHzOW3\\\\\\\/TeaubJGc0A9IQACGOC5RF8SeYRcU3mGWXPQZXbbslgfBZyKY7R8SWqfJBmy6S0Rn+IycFAC1M2tcpaZWhPB6Hz3heMcdOnummoD9F8BpbHq3ZOFUPlHGsEkj78JtYYJB05szIQYmcWqrDZ6NZe9NCn\\\\\\\/4nmp8D5ggOQYxNCLTICVCdh6uKDFxiHRDotkE8GUwF0iNvzKODSNGU7XPzs8fNG791z4r+t0PNeoFWKbaAIk9JUlCRreR5TlxBXtBqLD4rSJycKvNtJYCvPP24NdnTdQy5rEYUJMfH2jddnvxRH0uox+jtCW+4oAs4nPrk9n3I4euf9P+THSKfM7jPO5S5\\\\\\\/I1U+Hhoj6FmYY7wwi9y4fUmUAHRUbruG7aNZIBe0pVlE7vg1XURKJa7eWGRYtMtAbQ\\\\\\\/zQesYOrwvuuUe2jitIQiFZE1UzM6UEtF1ckZbUek4n75mJ+6pXaioap7KKQJihUqRFN8AXztyRBn8lToIhkaSaLNFdBeYOEzF3+\\\\\\\/geZoMHOq1qIZVxJLNrjdq1xTrSzEBfscdiRfMc2S2id9uPUhBgvMF\\\\\\\/E9BEsxeFNSvw0Dz\\\\\\\/v0BriY8j3f2prCIvSuD3gBtQ\\\\\\\/fr\\\\\\\/EsqEriiAZgDhISHU7AsyhVr9uznvGPQEo75dY8Q9ot7CKUas3ktbRxED0ia673bJyhCCY86vR3kH2tPHSHcin47zxCgfU+2R9ZAFVwQYycziIakanydiOsnC3TQTSPfQSPYKJua2+tVw3IqloeoZ3JFIpjyMzTOOUqsPGXQtyRqhqlE838+QnQrUlg+Qs77tsXp7hxc7NjINtJkfQ5xj63+vd0og\\\\\\\/M1mc8rs6gCYZdkWZOQ5ntst2CNNu8D\\\\\\\/G\\\\\\\/M0hkNE6KPQWGjz75MsK5Tqg33XyhkAhscs0E1YT2ZufN1LjCrXN+zcEjDd4J1+QIJp1gRyNzl\\\\\\\/CWwr77VpSHyP1j9xo6pFjN+PquOEj7meUiovXvGcduz9PGmxAeh+LkCAF+o\\\\\\\/y04SKfIb0hfy+q0k7NyaqdcTW+LS8atJEoYpTZLR1kwUoKVPcpizACloTBJ8RidEnrcysVJCBnVTCKw2lr9cvxn3mcgqJZbwdepAONVGWjAB\\\\\\\/0fanG7gpAF5GsmToT6Y7ky1EnrIYAIZJD4P2fN9VyaP47K4aZxK0u\\\\\\\/4Dm97PNdDzEPrau\\\\\\\/D0qSBSaC\\\\\\\/z04kWXw\\\\\\\/IFU7FMU3Ec098S22VVC9Bv9cOCXdhy6qZCsnJz\\\\\\\/QopXlra+t5tAnr2Aq1hmvaHPHHqxeINzrJAsRKXTMH7wh8XtiOi17rJo7jlF4TrURwbGP7aX8pN5H050j5wzan4w4jo+Vyj3cp6BGU7ywaEN6C0ptlltmqUOCsYPfizWRXWCYKg0wb28hAMgLghBhW\\\\\\\/eFpVvj1Dc18C8E3MnH0gaxJJ8pSrnxST8CDwkdbLbLvi5wKBc3LMU\\\\\\\/Ay68F1KqXSmXgovjF1mrRfYvOCihndTrrXnDWWvS+PNwwl7xhUTmPcVKypBbBbXFd+BP5Ui7Kr3PjJXvSBqO++arsAMMI7MdKrxZgSTqx\\\\\\\/iwwlxuFgXEOD0PVQVYqoaiBzZMmFPUQ5+KK3FFDzoATQANtm3xAqkqr8eHW+mT8ncgE6M2z6nLmdouwiR7F8uYeEdFyKQBQbF6XmpwhzPJnMudIIyKnttLuZkP9GY98PBZfIyikx4UYBwSMmXZZ9wavUn+WqVPIvu7GCBtI1NrrCyKjBmMENXrwSSRv9UtHi97kiSneTOlxWaOwnQ+jie15bd69C6IWW5XpsT0U0n5JV0oOKcCnkw5TeV\\\\\\\/B2bIM38WXjy1l2q5kmjLI5vyhXcM1vAm\\\\\\\/nTowIaR6mD8hJSGnnbveIain+hIhkfSZTtTG1EqdNsxiSoVJbKdW+a7rTKxsXD26Z3MDyfuPRpiol2X0UnOlhPBAVdYQI3ovL5IHQF7BCvd5I3Y5N9+kNx789A15+Nkaswm3YgJ+q0WQV8p4PKThivV15+9bHn2i8vbCzRbD\\\\\\\/IYccTieCd5IbjaKTQEbnpnszYb1A7WjI2t13kHu+qpO22O74h+CDZeB0as0rVtn7cid5Dse5vt6U5bz30Qm0zZhcB+POHUGxcVgkKr7kS+hGN31hIlFPVwg5+lY9XTbpJn3pL9dLfa4BlX2JqQDRd5sqkNwYbdnLHQ6x2Qfzt6fGxDkwhJ0NBXl6x4dOyrrLTHyTA436MH+69q1IYbkNbW\\\\\\\/moXP3OBvKThyjgXOKvYXYMksMYwWNkDBjvkWOiJypWv5pU+yqUvmEGFufT8l9e9mxYuQWJxlBOv1UKCHAmjsLEfRAyHZ65LuydX\\\\\\\/nedqAA98wFb652BRhgnWLbW2GjKWq86Kag\\\\\\\/iBKehDnA9j\\\\\\\/w64cVLUP8zHwCt7eIntS6OHRMu7NOP5tlrkeH\\\\\\\/k7Mwg6qAJTl0QQ3x3\\\\\\\/+32R7BiJt6ASk3\\\\\\\/6JsrJdWWCkTUuFfrTUamJZA21FyDZa0A9AuyM2oIpQXqzrg0LdF+SZPyZGN7omvqjzKkqduWAnodtt4upMmwpBoqj49RIb0h8n7Ph49zATEC8BdcERVlu45zlLIH264kugVQ5Cq8DDLHB0uwTpM\\\\\\\/8pWayrqp0hjDv6EG9ngcRb5lUc2ftkE3s58pZ5of8D9QV+txqWi46o8WNGgvSPjs6I0rgiFj3HaLNmKzoe\\\\\\\/3+Gcl3K+MsvpOFgeFF13ObWbvjYnNLG53S0wufmTmmXwNwqRxjVmYJcIGoqT5Foi7H4h+r1SZ940FnSetHBLwoDC+MfubgK9ZiLwlRP3mtLFl3d0XCNyGvZ4Y3h19xavxHJGVLS06FCmWDLdKNmZcANoJx7+c\\\\\\\/4L\\\\\\\/m8hp+dL+ux2hn7xkFNy19yQS8WgRbHEHBXnt275wkbg5Fn+mmBRi6kgN+0h9e8LkFeF3ubYMYH8aUxoKg1dlVyinV9\\\\\\\/W8mnftung\\\\\\\/elgIAtGMMMO\\\\\\\/VFT3JWCs5a45Z0MpPSpJJCKIMy4gfbXIH57WjnRjTHsQsGz3XN73rNrkPwbdueUmdFiVNd+2B30+qfGqltMSpL7Z3aCmGxtp3RfDJ9laGUJQoNDwQZynPTXjcbCp9nxh4siwx0MuDg8HOQC3llbxWrXmXVa5unw+OZ\\\\\\\/EpWY6lMnVklZ6ogxxwFH\\\\\\\/+WfWkh4dl+RnL2vY3YXP+IySxjsE5vUtYROts5TIy4ySJDshGqTxyUkZtxqo99kr4vdXyJ6OVw199398OexmcxyL23IazUuHIYFeZ5s7\\\\\\\/ae6SKOP+DveP5KxISvI8QdnRAnS8l6ZU0mLhxu\\\\\\\/iLJ+AGL6TwwCXiRVSebY8l2IMih+oSRkh2ds6u7Ap+\\\\\\\/0\\\\\\\/lecHt8S61SAyv0zigBPX54qfj4z8fU\\\\\\\/COc1qByIQiu8ddPzuGWwLHYocCR0mC6f8KOtSJXtXrwRQwO5qfzEJO7VBaVgPZO\\\\\\\/vYJXEmIVVuNU6obTHY9qwsqCNbkborD4UOEOmcijtdGld0g5VbYfZYy3X0PtrvvENPTA9crf7mnKdFw8kVVjX+A0mDGoAYyKxUfkE0OxPSpGRoxpf0uFwCnJ74LpDHrKNwC1JrMA9+mJI+cW7ocV7h9C+Ymn2J7VBtI3kR5JHflq\\\\\\\/naunx2OxkjhTrcOJHUgDA\\\\\\\/pbnb60fzIyKk75rpekXVmV+wvf9WnwyyT96+\\\\\\\/lgMNU+eapyS\\\\\\\/GNYOz4YsDpCaSqUjFbYfVRZ2sCtohD6+kw5y\\\\\\\/Zcv0UZaVMfDkyALZGpfrqXJDpyj7blTPgz0EC8DyS3m5MZnac3ziYrgS\\\\\\\/KfFTUCcM3kcN6B2ajECXDZr8vXrEh0iA2ZJltIvSWfRoJcIK2zojpU0EvLQyhJcCyB86kGL14nXMWORN0AZnEpenzohofjwzoX8k0Yk6CUFf0VJLKpPZFv5zAwBnwEq5fgtipXekHIC7GGmh\\\\\\\/ZgNzNukWswN35WozTB\\\\\\\/AQoiR9NrX+OkU1DOaODzRYkc11DaHb+oSJDKxChR8+IPnpNIIJWkZp6\\\\\\\/DU2zuQkO+aGdBFtkIpUq9igQs5t5A7trq+jCWHN6IdUBV9Syeun60cBjTNtc02So\\\\\\\/F\\\\\\\/Io77\\\\\\\/aLSXYMdh+t\\\\\\\/pXC6KL7D6hDEIJj4kW4bgOVFV7dik0frs+6xHO6Mt5HdLTj3BrhOwK4syBlamucVjSVMjr8g5U1nTQkoBQ9EESEWe28vqf5hXySZBHBym2gEdfxAG0UFRqBxwjXYgJ5xmQO3cRSMihbTeYMGlfDZ9a91CLWlZ\\\\\\\/iBdX9wiCTIxTTDfo9FGri\\\\\\\/nLSuz9kbJF25\\\\\\\/duSVpOJgF6WX+TzdD4dDCSqMp0y+E8x8awYtTQZma+wQnDszAEFzIUbow1lrS16ZRCq4y3HzyZou2BtLVk+1thACOH0pjn7WmJXlF0eO6s+CwtZzFCzWIURvps+K6f1nXbUbxvhOOS8OkyyMLHkVgLq16IgiJkTtYa81ynYDWwuf6FfkdrHmiGyKINcDTmF9gBP3cEVuSSqmhImQ7b+HF5QHsuVPFtDSWW2BHS6I+4ju+8+I5d72OloByM\\\\\\\/aO\\\\\\\/ofWJsFXEn++vv5f79nevJp3hJnHpit4qh0ddFTwyuquiym6UvdhMc2FRCko2RUscns0ccPDXfupF+wYFS668l4nPacROodDqAM3kG55Phyn7vE5cLVyjyzQlGDphuOkIf+pHsJkEChc9aLFlhYOadgUYB5VgptFviXHnArwOjCs90GgRWu+NK+NEd1GYN0DpWQ3gj7YfAA28bHAtVoFsxHZme36z3F6e1pLhAANUSwPjlKX+iOLR9MLG1AahvoyD\\\\\\\/XdHvNLWGEDKGW0h2hskwAL+9xtWPjhJ89LjKIeNhsVW4l408T\\\\\\\/j4I\\\\\\\/cNRPmoiGU11M2goo2UlOxeJZqoec3OpmhLaWdYZpf4s3k4utytcYF4HAN7YvkB3D+hDTOBB0w7m8UaJ0EUBtPKy9Aw83pnu2aMFRtPcRpQHoCLsntiEbvDFoYvbZLQUOiuD1D\\\\\\\/mHlxJfOSxy3o4mjizdxfxyPmOYbYYvwbH4VwTGNc\\\\\\\/JD6S+vtL57BspYV6zjob9s5\\\\\\\/7AiAY2qYta5KGSCTkVQGQ3oU+zejnoQoLLUsTrtB4eSbJO8F2cx4SIHb+orpQVefeyGTMi03rGjYnDU3SWPXRJTRbuOisLBnUtxboL14XXcsymDYcWOcZnombtUouzHz760TvY1b8EzkwVhyFaN22lSYQyKYHCLwA+QaCAaVJwz3a248A5dGm7RC4dkedv5kcB39+\\\\\\\/AcCBlVNgt5rKD9uswl2sbPKqiEHUGHg0b7vMpZV+5IezGf4JCj3FjOpcGoMSd7nofv2wWEqzbuWs6CWox0vYRuLe5GIfuRc6BLHZgliaxoIchAhgAL8SR4+6m4D91Z7ww7ay\\\\\\\/DuI7h19PnuQWIKwFgI7ox4zjjndQcpzyRyMUqaXG3SQAmMdVO0CN8FGsf5rcG023Ch8qvdmGH4B5VnQRIvJua4MYcyPSIQtGKcF\\\\\\\/Z+a6fRiU3agOseVNFK\\\\\\\/+qww09aEGIZ74I1v4PhLr1\\\\\\\/hsn3ew1BJRI1\\\\\\\/AOsf1TfrBRUB1cASVb5GNHSCeywxvBC6phzkYks0hYBI5AefpX6r9Fx8ldfOqPnvetwyDij4FXji9aqSB1hNqWsYb8MxSGoLndxdSAMg4ZjulsTUawYOE3qzaYUPHEcXHBPe+8SMheQmdzLMTcJv89NG\\\\\\\/sPeA3JXI2Yyxuqs5SSvE1YZumAF5\\\\\\\/bHrxk4Dj2+pc4b0K7U2oPE08XAo9PSIdcdnUByBPOGV\\\\\\\/LhkVaZD88U8\\\u0022,\\\u0022iv\\\u0022:\\\u0022YDZ\\\\\\\/0zKj0IvT64aUyC6Pmw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002225c1a9b86ef3830fc2d2ac00895a0a1ff04d1f4baaf30c42c485222ea8a47060\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022DpHPgXU5nh9DIcl0nQ7YqEdYRBYhO\\\\\\\/OIJs0cE7Wr6TQAJFIltOXcM7YT\\\\\\\/mp6kpmuFZJapEHMMszWJ\\\\\\\/kuAyTdx3xkeTiqR9WFNNBQXZHq7qMDpV6WB1N2NO29MUNeAXyiL+QirSli+gNDN8YaY7lDBCRENi++W5QWp+qyevsWpauzuwarvf23LZgTQkhaWj6pcrwECBtn+48Gq63WyJtJU2xXK2xGu\\\\\\\/9BtBlH5s0wSGwO+3rG7qc\\\\\\\/Jy4Bo654mU9sRvtZ6ITVUjAdES1GgmGmXs3DO4DhOsGbQ\\\\\\\/UDPW6KeBd9BXN4FrhELl04AqzzMCayY9cppOQbTbQl\\\\\\\/qD+WW\\\\\\\/HFRRgbOH99oJ9enOgWGDsuyO\\\\\\\/+jPoT9noy\\\\\\\/BTg3R294pyRy+\\\\\\\/vzpkJJlLz9UG9h9bpwO8Ld3ePSPb01WSWM1Pu6EJSw6LRqAjY9JS3\\\\\\\/J7tr\\\\\\\/5eH7pHM78PIwIesrfPIBdc1+2XcSJCIuDBg46XsMei0wa8UPZswnXLjNJfbCJyrQYI1W4bt2nRLtqC713tSQ3yUWBeKux+7zcdBY1ErVMQxtY2qyyNXuEq2Fi4bRMloO7T6oj4Hz6TO9qf53w8ebtNmE0EJj8RTrxYUlwNzaxj636Q9BG1K+yuYEJknhidsPHQmVuh8MF7DiaIJEv+XEdOfsbl1pSqLsgEHnVgY3mxFruof3uR9KhmspmdEbndwb+RwvZBiEyprponz2txk8J7S6y60zL99LO3n\\\\\\\/U2P80AQdUEtIKvTxsjJ2h\\\\\\\/o5OaVsGDUqAs9N4wQw2HHXo7nXQ4x1bDpp\\\\\\\/sLkxu36mRZ+9JBjCRC4GsoBEFWEx3mK6K2Bxkmb0eJ0UkF+I3kYU4LvySEe+TKLdgO8pUyNreF4DrwiBzW4X4P8R+49czcAklTVoR7YlhTMMb4V3ZHLTrSqi9dpY6bqH9Fs6Qr\\\\\\\/4Fbct8ZAzxzcNJcdR2P+7Y8s+VUZyLYng7dpytQIuO9lg81utFQ1a9dZAYIPbOUJvz4O+h7kNbDLtbhT13QfE4SoqUoecnRlhIKOmtVwIMqiEodExHkX811oHwNW94+oARJuOEA4gJxsCl0wRUCQyjFzdnW1WjCVPdcYKDCshsxW7MFgAB6gV17CpbMqgUfkoaodaCrVvCqstKI9UCgCuk5VohHElv0CjpF+RgHQAL5Si6GWhQ0d\\\\\\\/b4eARa1xlTjT5OK6Nr7CFOS9SsGvmuNjl66reuoAtvtzoViAKwH5SdFGkuaB1Gef5nJqO5mETm\\\\\\\/5gUN3VZBlP8pF0ddC3AO1+wxBH2m280gj3+X2r\\\\\\\/FvlUqNP\\\\\\\/YsXl97JTaUJEhSxB3CoeyCxSHnA\\\\\\\/FwdKcjzXZHYn2HZo3k1zkwLDbYeIuD+Zi0XMQdoXftJTkWdQhPXbpyazf\\\\\\\/sptKp9H\\\\\\\/fJdL\\\\\\\/hDjLOv9iVC1MMCwWq8R9Lqr9\\\\\\\/lNAuF97gJZQNOrrRN9ttnIC1kN6eqKdgLkQ+vfSQ6BXi85olC45qfg\\\\\\\/v1KoSv7MVLXnzRLtLt3zb0iRNQVPUyrfKnfZ5+GB9U47GjWcYdlD20puH3ZAMptlZkrUCYhFr9hju9MARQJrGcIUfAgQ0yDU+VB2vRKCvppQyitLD8S78I31bzlkFwbGdMOrUrbVKlbRBX1wk6v1fPi4CK+41vn9U9NsqgjlZZdDsuJpCYqagWEXd2ZFaiIfbvoOpV8hBlHaTDrbewUamaysWIv854qXdskxK7QkHskL1afPNfq1tS81DNi\\\\\\\/zJkM4HKiDHnr+7ozLb0NCctiaXcFtGuT5SUrEdAO16FSXSsaYly\\\\\\\/emduKHDt80140gQwr+L2RhIRdUF88nqmBVseGVTx9KHoEDdHbxn69KaJTiBsjC+QYZMO4GmbsraWkFbK34MaSWDtHOxHjuXozaJs5GURZ1pAN51Avm+z4qIinyU7CgAXBArBW91e1gOsD+WGO2PHMS9fvfU6u6M+UkyulVWq45h8b3Jv9t31z5RrMhjg6c1CXS\\\\\\\/6b3O9cox4eVrVDqqfVHLKee01nA3oyyQqgedzRHXqvvS1m4FGFE=\\\u0022,\\\u0022iv\\\u0022:\\\u0022SlTWmH9kEfr7AxBwVQiYHA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f107bfb10b2c21d2acafeb22148905e73801e4fa714be35ce57ff2470d076369\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-483519 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/3/871" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-599Z7WAQP3MSDXi4dPg-APn34mdEzRw0Svbo_7u7qn4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/3/871" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jJmjEC4JAhxm5fz5Jg9DMC8Kj-CzuI2SNAFfQaL6rRo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/3/865" title="Genetic Dissection of a QTL Affecting Bone Geometry" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/3/881" title="Adapting Genotyping-by-Sequencing for Rice F2 Populations" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483519" id="node483519" data-pisa="ggg;7/3/871" data-pisa-master="ggg;g3.116.037275" data-apath="/ggg/7/3/871.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>De Novo</em> Transcriptome Characterization of a Sterilizing Trematode Parasite (<em>Microphallus</em> sp.) from Two Species of New Zealand Snails</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Laura Bankers</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-1543-8115" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Maurine Neiman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">871-880;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.037275 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483519" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Laura Bankers </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Iowa, Iowa City, Iowa 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laura%2BBankers%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bankers%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALaura%2BBankers%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:laura-rice@uiowa.edu" class="" data-icon-position="" data-hide-link-title="0">laura-rice@uiowa.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Maurine Neiman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Iowa, Iowa City, Iowa 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maurine%2BNeiman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neiman%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaurine%2BNeiman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1543-8115" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Maurine Neiman</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/3/871" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:483519" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:483519/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/3/871.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:483519" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:483519/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/3/871.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:483519" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:483519/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/3/871.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:483519" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:483519/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/3/871.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:483519" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:483519/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Snail-borne trematodes represent a large, diverse, and evolutionarily, ecologically, and medically important group of parasites, often imposing strong selection on their hosts and causing host morbidity and mortality. Even so, there are very few genomic and transcriptomic resources available for this important animal group. We help to fill this gap by providing transcriptome resources from trematode metacercariae infecting two congeneric snail species, <em>Potamopyrgus antipodarum</em> and <em>P. estuarinus</em>. This genus of New Zealand snails has gained prominence in large part through the development of <em>P. antipodarum</em> and its sterilizing trematode parasite <em>Microphallus livelyi</em> into a textbook model for host–parasite coevolutionary interactions in nature. By contrast, the interactions between <em>Microphallus</em> trematodes and <em>P. estuarinus</em>, an estuary-inhabiting species closely related to the freshwater <em>P. antipodarum</em>, are relatively unstudied. Here, we provide the first annotated transcriptome assemblies from <em>Microphallus</em> isolated from <em>P. antipodarum</em> and <em>P. estuarinus</em>. We also use these transcriptomes to produce genomic resources that will be broadly useful to those interested in host–parasite coevolution, local adaption, and molecular evolution and phylogenetics of this and other snail–trematode systems. Analyses of the two <em>Microphallus</em> transcriptomes revealed that the two trematode types are more genetically differentiated from one another than are the <em>M. livelyi</em> infecting different populations of <em>P. antipodarum</em>, suggesting that the <em>Microphallus</em> infecting <em>P. estuarinus</em> represent a distinct lineage. We also provide a promising set of candidate genes likely involved in parasitic infection and response to salinity stress.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/transcriptome" class="hw-term hw-article-keyword hw-article-keyword-transcriptome" rel="nofollow">transcriptome</a></li><li class="kwd"><a href="/keyword/host-parasite-interactions" class="hw-term hw-article-keyword hw-article-keyword-host-parasite-interactions" rel="nofollow">host-parasite interactions</a></li><li class="kwd"><a href="/keyword/microsatellites" class="hw-term hw-article-keyword hw-article-keyword-microsatellites" rel="nofollow">microsatellites</a></li><li class="kwd"><a href="/keyword/coevolution" class="hw-term hw-article-keyword hw-article-keyword-coevolution" rel="nofollow">coevolution</a></li><li class="kwd"><a href="/keyword/trematode" class="hw-term hw-article-keyword hw-article-keyword-trematode" rel="nofollow">trematode</a></li></ul><p id="p-3">Snail-borne trematodes are important sources of morbidity and mortality in animals, including humans (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hotez 2013</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Jurberg and Brindley 2015</a>). For example, infections by trematodes in the genus <em>Schistosoma</em> affect &gt;200 million people in sub-Saharan Africa (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Rollinson <em>et al.</em> 2013</a>). Trematodes and their mollusk intermediate hosts are also important models for the study of host–parasite coevolutionary interactions (<em>e.g.</em>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lively and Jokela 1996</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">King <em>et al.</em> 2009</a>). While digenetic trematodes (parasitic flatworms) are one of the largest groups of metazoan parasites, with ∼18,000 species (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cribb <em>et al.</em> 2001</a>), the genomic and transcriptomic resources needed to understand trematode evolution, host–parasite coevolution, and the genomic basis of trematode infection are only available for 13 of these species (&gt;50% members of the <em>Schistosoma</em> genus; NCBI, January 2017). Further development of such resources for this medically and scientifically important group of parasites is a necessary component of characterizing the molecular and genomic underpinnings of parasitic infection and host–parasite coevolutionary interactions.</p><p id="p-4">New Zealand snails in the genus <em>Potamopyrgus</em> are often the intermediate host of sterilizing trematode parasites in the genus <em>Microphallus</em> (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Winterbourn 1973</a>). Here, we focus on <em>Microphallus</em> ‘<em>livelyi</em>’ (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hechinger 2012</a>; hereafter “<em>M. livelyi</em>”), a species of digenetic trematode that uses <em>Potamopyrgus antipodarum</em> as an intermediate host and dabbling ducks (Anatinae) as the final host (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dybdahl and Lively 1996</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Lively and Jokela 1996</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">King <em>et al.</em> 2009</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Osnas and Lively 2011</a>). Dabbling ducks are infected by <em>M. livelyi</em> while foraging, typically consuming infected <em>P. antipodarum</em> from submerged rocks and aquatic vegetation. The trematodes lay sexually produced eggs within the ducks, which are excreted within duck feces (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Osnas and Lively 2011</a>). The next round of snail infection begins after detritus-feeding <em>P. antipodarum</em> consume <em>M. livelyi</em> eggs while foraging along lake bottoms and in aquatic vegetation (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Lively and Jokela 1996</a>). Following snail consumption, the <em>M. livelyi</em> eggs clonally develop into thousands of encysted larvae (metacercariae) that rapidly sterilize the snail (<a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Winterbourn 1973</a>; <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">King <em>et al.</em> 2009</a>). <em>M. livelyi</em> individuals that cannot infect the first snail by which they are ingested will die before completing their lifecycle, and successful infection means that the <em>M. livelyi</em> metacercariae sterilize the snail, leading to strong reciprocal antagonistic selection pressures between these two species (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">King <em>et al.</em> 2011</a>). These metacercariae are also the <em>M. lively</em> stage that is transmissible to the duck final host (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Levri and Lively 1996</a>) and thus represent an especially important point of no return in the life history of <em>Microphallus</em>: if the snail-inhabiting metacercariae are not ingested by the duck final host, the parasite will be unable to complete its life cycle (<a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Lively and Jokela 1996</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">King <em>et al.</em> 2011</a>). The importance of this metacercariae life stage is emphasized by new RNA sequencing (RNA-Seq) evidence that the presence of <em>M. livelyi</em> metacercariae influences patterns of gene expression in <em>P. antipodarum</em> (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bankers <em>et al.</em> 2016</a>). By this logic and informed by these data, we chose to focus on the metacercariae stage for our analyses.</p><p id="p-5">How interactions between <em>M. livelyi</em> and <em>P. antipodarum</em> play out in <em>Microphallus</em> are not well characterized, particularly at the genetic and molecular levels. The initial insights we do have into the mechanisms underlying <em>Microphallus</em> infectivity come from local adaptation and reciprocal infection experiments, providing preliminary lines of evidence in support of the likelihood that coevolutionary interactions between <em>P. antipodarum</em> and <em>M. livelyi</em> fit a “matching alleles” infection genetics model (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dybdahl and Krist 2004</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Lively <em>et al.</em> 2004</a>; <a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">King <em>et al.</em> 2009</a>). With a matching alleles system, parasite infection success is thought to be determined by whether the parasite genotype at the loci involved in mediating infection match the corresponding loci in the host, enabling successful parasites to evade detection by host defenses. While matching alleles systems for infection have been demonstrated in several host–parasite interactions, the specific genetic or molecular mechanisms underlying such interactions remain unclear in <em>P. antipodarum</em> and in every other system (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Carmona <em>et al.</em> 2015</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Routtu and Ebert 2015</a>).</p><p id="p-6">The transcriptomic resources described here, the first genomic resources for the <em>Microphallus</em> genus of which we are aware, will provide a valuable starting point to characterize the molecular underpinnings of <em>M. livelyi</em> infection and the coevolutionary interactions between <em>M. livelyi</em> and <em>P. antipodarum</em>. We present annotated transcriptomes assemblies from <em>M. livelyi</em> metacercariae isolated from <em>P. antipodarum</em>, as well as from <em>Microphallus</em> of unknown species isolated from <em>P. estuarinus</em>, a closely related species native to New Zealand estuaries. While the coevolutionary interactions between <em>M. livelyi</em> and <em>P. antipodarum</em> are well characterized, much less is known about the <em>Microphallus</em> that infect <em>P. estuarinus</em> (<a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">Winterbourn 1973</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Hechinger 2012</a>). Expanding the genomic resources available for these two closely related types of <em>Microphallus</em> will allow for the characterization of genes that contribute broadly to infection processes, as well as genes that play species-specific roles in host–parasite interactions.</p><p id="p-7">We identified and annotated 7584 pairs of one-to-one orthologous transcripts between the two transcriptomes, analyzed levels of genetic variation within and genetic differentiation between the two types of <em>Microphallus</em> at these genes, and performed gene ontology (GO)-based functional comparisons between these two samples. We also provide a broad set of genomic resources that set the stage for future studies aimed at characterizing the molecular underpinnings of the coevolutionary interactions between <em>Microphallus</em> and <em>Potamopyrgus</em>.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Sample collection and RNA-Seq</h3><p id="p-8">We isolated <em>M. livelyi</em> metacercariae from four individual adult female <em>P. antipodarum</em> (PA-<em>Microphallus</em>) collected in January of 2013 in shallow regions (&lt;1 m lake depth) of each of the two New Zealand lakes Alexandrina and Kaniere, for a total of eight PA-<em>Microphallus</em> samples. We chose these lakes because their <em>P. antipodarum</em> populations contain relatively high frequencies of <em>M. livelyi</em> infection (10–20%; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Vergara <em>et al.</em> 2013</a>). We also isolated <em>Microphallus</em> metacercariae from a single adult female <em>P. estuarinus</em> (PE-<em>Microphallus</em>) collected from the Ashley River estuary (near Christchurch, New Zealand) in January of 2015. Metacercariae were collected from stage five infections (<a href="http://www.indiana.edu/~curtweb/trematodes/DATA_KEY.HTM">http://www.indiana.edu/∼curtweb/trematodes/DATA_KEY.HTM</a>), in which the entire snail body cavity is filled with clonally produced metacercariae cysts. We chose the metacercariae life stage both because it represents a critical time point in the life cycle of <em>Microphallus</em> and because this life stage is easily identifiable and allows for larger volumes of tissue to be collected than earlier stages.</p><p id="p-9">We dissected individual <em>Potamopyrgus</em> snails to determine infection status and saved parasite tissue from snails containing stage five <em>Microphallus</em> infections. We separated parasite from snail tissue with a forceps and used a micropipette to isolate metacercariae cysts. We immediately placed the metacercariae in RNAlater Solution (Life Technologies Corporation) at 4° for 24 hr, and subsequently stored the RNAlater-submerged metacercariae at −80° (according to the manufacturer’s protocol) until RNA extraction. The eight PA-<em>Microphallus</em> metacercariae samples were pooled in a single tube. We had to use this pooling strategy for the PA-<em>Microphallus</em> because at this time (early 2013), pooling of cysts from multiple snails was necessary to obtain a sufficient amount of RNA to perform RNA-Seq. Technological advances in library preparation and RNA-Seq between 2013 and 2015 meant that we were able to obtain a sufficient amount of RNA from the metacercariae isolated from a single <em>P. estuarinus</em> individual for RNA-Seq.</p><p id="p-10">We extracted RNA from metacercariae following the TRIzol protocol (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chomczynski and Sacchi 1987</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chomczynski 1993</a>). We assessed RNA quantity and quality using a Bio-Rad Experion Automated Electrophoresis Station and Experion RNA analysis kit, requiring an RQI ≥ 8 and minimum of 2 µg of total RNA (following the manufacturer’s protocol). We prepared cDNA libraries following the Illumina Truseq LS protocol (Illumina, San Diego, CA) for the PA-<em>Microphallus</em> and PE-<em>Microphallus</em> samples. We barcoded the PA-<em>Microphallus</em> library and pooled the barcoded library with 11 other barcoded samples that were part of a different project in one Illumina HiSeq 2000 lane in 2013. For the PE-<em>Microphallus</em> library, we barcoded the library and pooled the barcoded library with five other samples that were part of a different project in one Illumina HiSeq 2000 lane in 2015. We performed 2 × 100 bp paired-end RNA-Seq for both <em>Microphallus</em> libraries. We obtained 31,635,450 paired-end reads (mean read length = 101 bp) for PA-<em>Microphallus</em> and 4,16,15,863 paired-end reads for PE-<em>Microphallus</em> (mean read length = 126 bp).</p></div><div id="sec-3" class="subsection"><h3>Transcriptome assembly and annotation</h3><p id="p-11">We used FASTQC (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andrews 2010</a>) to assess RNA-Seq quality and the FASTX Toolkit (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gordon and Hannon 2010</a>) to trim adapter sequences and remove poor quality reads from raw RNA-Seq data, requiring a mean Phred score ≥ 20. After quality filtering, 31,096,527 paired-end reads remained for PA-<em>Microphallus</em> and 40,413,790 paired-end reads remained for PE-<em>Microphallus</em>. We used these filtered reads to generate <em>de novo</em> transcriptome assemblies for each of the two <em>Microphallus</em> samples.</p><p id="p-12">We performed the Trinity <em>in silico</em> normalization, requiring a minimum Kmer coverage of 2× and maximum Kmer coverage of 100× (following <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Haas <em>et al.</em> 2013</a>), and otherwise default parameters (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Grabherr <em>et al.</em> 2011</a>). We then generated preliminary <em>de novo</em> transcriptome assemblies for both types of <em>Microphallus</em> using Trinity software with default parameters (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Haas <em>et al.</em> 2013</a>). Preliminary assemblies contained 65,665 contigs and 2,26,896 contigs for PA-<em>Microphallus</em> and PE-<em>Microphallus</em>, respectively. We used the Trinity plugin TransDecoder with default parameters to annotate likely protein-coding regions, identify long open reading frames of putative genes, and filter miscalled isoforms (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Haas <em>et al.</em> 2013</a>). Next, we used hierarchical clustering based on sequence identity to further reduce redundancy in the transcriptome assemblies, as implemented by CD-HIT-EST (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Huang <em>et al.</em> 2010</a>). To reduce the likelihood of collapsing truly nonredundant transcripts, we used a sequence identity of 0.95 and word size of 8 nt (<em>e.g.</em>, <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Li <em>et al.</em> 2002</a>). These steps resulted in assemblies containing 15,435 contigs for PA-<em>Microphallus</em> and 29,564 contigs for PE-<em>Microphallus</em>. We mapped reads back to their respective filtered transcriptome assemblies using TopHat2 and then used Cufflinks (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Trapnell <em>et al.</em> 2013</a>) to estimate mean FPKM for each transcript in each transcriptome. Mean FPKM (±SD) for the transcriptome assemblies were 71.6 (±274.8) for PA-<em>Microphallus</em> and 46.8 (±284.3) for PE-<em>Microphallus</em>. The large SD in FPKM for both types of <em>Microphallus</em> is due to the large ranges in FPKM (∼0 to 10,866 for PA-<em>Microphallus</em>; ∼0 to 19,279 for PE-<em>Microphallus</em>). The nearly twofold higher number of transcripts for PE-<em>Microphallus</em> relative to PA-<em>Microphallus</em> was likely a combination of advances in sequencing chemistry and technology between 2013 and 2015, higher sequencing depth for PE-<em>Microphallus</em>, and/or differences in heterozygosity of the sequenced individuals and/or samples. We annotated the transcriptomes using blastx with an E-value cut-off of 1e−6, followed by Blast2GO (using default parameters) to assign GO terms to blastx-annotated transcripts (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Conesa <em>et al.</em> 2005</a>). For PA-<em>Microphallus</em>, we obtained both blastx and GO annotations for 9272 transcripts and only blastx annotations for an additional 3814 transcripts. For PE-<em>Microphallus</em> we obtained both blastx and GO annotations for 13,778 transcripts and only blastx annotations for an additional 8822 transcripts. This analysis revealed 143 transcripts involved in photosynthesis in the PE-<em>Microphallus</em> assembly (none in PA-<em>Microphallus</em>), representing potential contaminants in the snail gut from which the PE-<em>Microphallus</em> was isolated. These 143 transcripts were filtered out of subsequent analyses. Finally, we used BUSCO (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Simão <em>et al.</em> 2015</a>) to assess the completeness of our transcriptome assemblies. This program uses tblastn to identify conserved single-copy orthologs among groups of organisms. We used the BUSCO OrthoDB Metazoan database to search our assemblies for the presence, absence, duplication, and/or fragmentation of 843 conserved single-copy Metazoan orthologs.</p></div><div id="sec-4" class="subsection"><h3>Identification of orthologous transcripts</h3><p id="p-13">We used “reciprocalblast_allsteps.py” (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Warren <em>et al.</em> 2014</a>) to identify putative orthologous transcripts between the two <em>Microphallus</em> transcriptomes based on one-to-one reciprocal best blastn hits. We performed reciprocal blastn searches between the two transcriptomes with an E-value cutoff of 1e−20 and identified pairs of transcripts that were reciprocally one another’s top blastn hit and that occurred once and only once in each transcriptome query. We identified 11,160 of such transcript pairs, which we will hereafter refer to as “one-to-one orthologs.” Using a custom python script (github.com/jsharbrough/grabContigs), we extracted the 11,160 orthologous sequences from each <em>Microphallus</em> transcriptome to generate putative ortholog transcriptomes for each sample. To improve the accuracy of our ortholog predictions, we then used blastx against the NCBI nonredundant dataset with an E-value cutoff of 1e−6 (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Camacho <em>et al.</em> 2009</a>) to annotate the two transcriptomes and filter out putative orthologs that received different blastx annotations. This step resulted in 7935 remaining putative one-to-one orthologs. To further ensure that we accurately predicted orthologs, we also performed reciprocal shortest molecular distance comparisons between the 7935 pairs of transcripts using “reciprocal_smallest_distance.py” (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Wall <em>et al.</em> 2003</a>). Here, we inferred orthologs using a combination of global sequence alignments between the two transcriptomes and maximum likelihood-based evolutionary pairwise distances, and identified orthologous pairs of transcripts as those sets of transcripts with the shortest pairwise molecular distance between them relative to their pairwise distances from any of the other 7934 transcripts in the dataset (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Wall <em>et al.</em> 2003</a>). This filtering step left 7584 remaining pairs of one-to-one orthologous transcripts that met all three of our criteria: (1) reciprocal best blastn hits, (2) the same blastx annotation, and (3) shared the shortest pairwise evolutionary distance relative to all other transcripts. There were 7851 and 21,980 transcripts for PA-<em>Microphallus</em> and PE-<em>Microphallus</em>, respectively, that did not meet these three criteria (hereafter, “nonone-to-one orthologs”). We extracted the 7584 orthologous transcripts from each reference transcriptome to generate ortholog transcriptomes for each of the two <em>Microphallus</em> datasets. We estimated mean FPKM for each transcript in the ortholog transcriptomes as described above. Mean FPKM (±SD) for the transcriptome assemblies were 114.3 (±403.2) for PA-<em>Microphallus</em> and 111.7 (±569.2) for PE-<em>Microphallus</em>.</p></div><div id="sec-5" class="subsection"><h3>Comparisons of genetic variation and genetic differentiation</h3><p id="p-14">We used Genome Tools to split both of the ortholog transcriptomes into individual fasta files for each transcript (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gremme <em>et al.</em> 2013</a>). We then merged pairs of orthologous sequences into multifasta files and used Clustal Omega to align and compute Jukes–Cantor-corrected pairwise molecular distances for each pair of orthologous transcripts (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Sievers <em>et al.</em> 2011</a>). Finally, we used the pairwise molecular distance values between each pair of orthologous transcripts obtained from Clustal Omega to estimate mean and median pairwise molecular distance between the two transcriptomes. We used SPSS Statistics v.23 to generate a boxplot of pairwise molecular distance that enabled us to visualize the distribution of these values (see Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/FigureS1.jpg">Figure S1</a>).</p><p id="p-15">We used the PoPoolation pipeline to assess allelic diversity (Watterson’s θ) within and relative levels of genetic differentiation (<em>F<sub>ST</sub></em>) between the two <em>Microphallus</em> samples (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kofler <em>et al.</em> 2011a</a>,<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">b</a>). Watterson’s θ is an estimator of nucleotide diversity based on the number of segregating sites in a sample (here, SNPs) (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Watterson 1975</a>). <em>F<sub>ST</sub></em> is a descriptive statistic that estimates relative levels of genetic differentiation among populations based on variance in allele frequencies (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Holsinger and Weir 2009</a>). Analyses of levels of nucleotide variation and genetic differentiation from pooled RNA-Seq data with programs like PoPoolation and PoPoolation2 (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Kofler <em>et al.</em> 2011a</a>,<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">b</a>), which were developed to perform variant calling and calculate <em>F<sub>ST</sub></em> from pooled sequencing data, can effectively identity SNPs and candidate genes and perform <em>F<sub>ST</sub></em>-based measures of genetic differentiation (<em>e.g.</em>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Fischer <em>et al.</em> 2013</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Konczal <em>et al.</em> 2014</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Li <em>et al.</em> 2014</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Thavamanikumar <em>et al.</em> 2014</a>; Ellison <em>et al.</em> 2016). In order to lower the amount of memory needed for analysis and to reduce biases linked to variation in gene expression among loci, we required a minimum coverage of 10× and a maximum coverage of 200× for both the θ and <em>F<sub>ST</sub></em> analyses described below. We also required that each SNP was sequenced at least four times and filtered out SNPs with minor allele frequencies &lt;5% (<em>e.g.</em>, <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Fischer <em>et al.</em> 2013</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Konczal <em>et al.</em> 2014</a>).</p><p id="p-16">First, we used TopHat2 (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Trapnell <em>et al.</em> 2013</a>) to map <em>Microphallus</em> reads to their respective ortholog transcriptomes. For the Watterson’s θ estimates, this process included mapping the PA-<em>Microphallus</em> reads to the PA-<em>Microphallus</em> ortholog transcriptome and mapping the PE-<em>Microphallus</em> reads to the PE-<em>Microphallus</em> ortholog transcriptome. To estimate <em>F<sub>ST</sub></em> between the two types of <em>Microphallus</em>, PoPoolation2 requires the mapping of the two samples (PA-<em>Microphallus</em> and PE-<em>Microphallus</em>) to a single reference. In order to account for the ortholog transcriptome to which reads were mapped, we performed this analysis twice, first by mapping both sets of <em>Microphallus</em> reads to the PA-<em>Microphallus</em> ortholog transcriptome, and second by mapping both sets of <em>Microphallus</em> reads to the PE-<em>Microphallus</em> ortholog transcriptome. After mapping, we used Picard Tools v.1.119 (<a href="http://broadinstitute.github.io/picard/">http://broadinstitute.github.io/picard/</a>) to process and filter the bam files. This step included the addition of read groups and removal of duplicate reads from each bam file. We then used GATK (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">McKenna <em>et al.</em> 2010</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">DePristo <em>et al.</em> 2011</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Van der Auwera <em>et al.</em> 2013</a>) to realign reads around indels and to reassign mapping quality. Finally, we used SAMtools (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Li <em>et al.</em> 2012</a>) to sort each bam file by coordinates relative to the ortholog transcriptome to which they were mapped and to generate mpileups.</p><p id="p-17">We generated separate mpileup files for each bam file and used PoPoolation (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Kofler <em>et al.</em> 2011a</a>) for further analyses. PoPoolation is a suite of programs designed to handle pooled sequencing data and is able to account for the number of individuals sequenced in each pool. First, we used PoPoolation’s Variation-at-position.pl script (which is not available in newer program versions) to calculate Watterson’s θ for each orthologous transcript for the two <em>Microphallus</em> types. We compared the medians and distributions of θ for each transcriptome using Bonferroni-corrected Mann–Whitney <em>U</em> tests and Kolmogorov–Smirnov tests, respectively, as implemented in SPSS Statistics v.23. We also used SPSS Statistics v.23 to generate a boxplot for Watterson’s θ and <em>F<sub>ST</sub></em> per SNP in order to visualize the distribution of values for each statistic (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/FigureS1.jpg">Figure S1</a>).</p><p id="p-18">We then used the most up-to-date version of PoPoolation (PoPoolation2) (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Kofler <em>et al.</em> 2011b</a>) to generate synchronized mpileups (mpileup2sync.pl) for each bam file associated with the ortholog transcriptomes (<em>i.e.</em>, PA-<em>Microphallus</em> and PE-<em>Microphallus</em> mapped to the PA-<em>Microphallus</em> ortholog transcriptome, and PA-<em>Microphallus</em> and PE-<em>Microphallus</em> mapped to the PE-<em>Microphallus</em> ortholog transcriptome). Next, we used the PoPoolation2 fst-sliding.pl to calculate <em>F<sub>ST</sub></em> per SNP to assess levels of relative genetic differentiation between the two <em>Microphallus</em> types. We used these <em>F<sub>ST</sub></em> values to determine mean and median <em>F<sub>ST</sub></em> per SNP between PA-<em>Microphallus</em> and PE-<em>Microphallus</em> relative to both of the ortholog transcriptomes. We applied outlier analyses within SPSS Statistics v.23 to identify transcripts containing <em>F<sub>ST</sub></em> outlier SNPs (<em>i.e.</em>, SNPs with <em>F<sub>ST</sub></em> values outside of the 95% C.I. around the mean), which can be interpreted as representing significant genetic differentiation for these genes between these two types of <em>Microphallus</em>. Finally, we used blastx and Blast2GO to annotate <em>F<sub>ST</sub></em> outlier-containing transcripts and determine putative functions.</p></div><div id="sec-6" class="subsection"><h3>GO and KEGG-based functional comparisons of transcriptomes and orthologs</h3><p id="p-19">We used blastx (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Camacho <em>et al.</em> 2009</a>) and Blast2GO (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Conesa <em>et al.</em> 2005</a>) to annotate the orthologous sequences for both <em>Microphallus</em> types. We also annotated the transcripts from each transcriptome that did not fall into orthologous gene sets (nonone-to-one orthologs). We then performed GO functional enrichment analyses, using Fisher’s Exact tests as implemented by Blast2GO (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Conesa <em>et al.</em> 2005</a>), to identify overrepresented <em>vs.</em> underrepresented functional groups in the set of orthologous transcripts relative to the rest of the transcriptome, and for the set of nonone-to-one orthologous transcripts relative to the rest of the transcriptome. All functional enrichment analyses were performed on both <em>Microphallus</em> samples. This analysis approach allowed us to compare overrepresented <em>vs.</em> underrepresented functional groups for orthologous and nonorthologous transcripts between the two types of <em>Microphallus</em> in order to identify which functional groups were expressed in both <em>Microphallus</em> samples <em>vs.</em> those functional groups that were uniquely enriched in each <em>Microphallus</em> type. We also predicted KEGG (Kyoto Encyclopedia of Genes and Genomes database) pathways for each transcriptome as implemented by Blast2GO. This analysis allowed us to (1) identify potential among-gene interactions, (2) identify sets of interacting genes of relevance to the <em>Microphallus</em> life cycle stage that we sequenced, and (3) assess whether different pathways are being expressed between the two samples. Finally, we used Blast+ to generate local BLAST databases for both annotated transcriptomes.</p></div><div id="sec-7" class="subsection"><h3>Microsatellite loci and primer prediction</h3><p id="p-20">We used PrimerPro v.1.0 (<a href="https://webdocs.cs.ualberta.ca/~yifeng/primerpro/">https://webdocs.cs.ualberta.ca/∼yifeng/primerpro/</a>) to identify potential microsatellite loci in the two <em>Microphallus</em> transcriptomes. We limited these analyses to the ortholog transcriptomes to ensure the transcripts in which we identified microsatellites were distinct loci and to allow future direct comparisons among different types of <em>Microphallus</em>. The PrimerPro v.1.0 pipeline first predicts microsatellite loci by identifying simple sequence repeats (SSRs) using MISA (MIcroSAtellite Identification Tool; <a href="http://pgrc.ipk-gatersleben.de/misa/misa.html">http://pgrc.ipk-gatersleben.de/misa/misa.html</a>) and then uses Primer3 (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Untergasser <em>et al.</em> 2012</a>) and BLAST (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Camacho <em>et al.</em> 2009</a>) to generate primers isolating these loci. After these steps, PrimerPro v.1.0 outputs the putative microsatellites, the primer sequences, their transcriptomic locations, and the primer properties. Following <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Qiu <em>et al.</em> (2016)</a>, we allowed for a minimum of 10 repeats for mononucleotide repeats, six for dinucleotide repeats, five for trinucleotide repeats, four for tetranucleotide repeats, three for pentanucleotide repeats, and three for hexanucleotide repeats, and identified compound microsatellites as instances where there were more than one microsatellite separated by ≤150 nt.</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-21">The raw RNA-Seq reads are deposited at NCBI Sequence Read Archive under the accessions SRR5170514 and SRR5170515 for PA-<em>Microphallus</em> and PE-<em>Microphallus</em>, respectively. The Transcriptome Shotgun Assembly projects have been deposited at DDBJ/EMBL/GenBank under the accessions GFFK00000000 and GFFL00000000, for PA-<em>Microphallus</em> and PE-<em>Microphallus</em>, respectively. The versions described in this paper are the first versions, GFFK01000000 and GFFL01000000. BLAST databases for each transcriptome are available at <a href="http://bioweb.biology.uiowa.edu/neiman/Potamomics.php">http://bioweb.biology.uiowa.edu/neiman/Potamomics.php</a>. SNP and molecular distance data, microsatellite loci with predicted primer details, and transcriptome annotations are deposited at Dryad (doi: 10.5061/dryad.3ft30). Transcriptome annotation statistics and annotation details (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/TableS1.docx">Table S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/TableS4.docx">Table S4</a>), a list of orthologous transcript IDs (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/TableS2.xlsx">Table S2</a>), boxplots to visualize distributions of θ, <em>F<sub>ST</sub></em>, and pairwise distances (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/FigureS1.jpg">Figure S1</a>), GO functional enrichment and KEGG analyses of orthologous transcripts (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/TableS6.docx">Table S6</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/TableS7.docx">Table S7</a>), annotations of <em>F<sub>ST</sub></em> outlier-containing transcripts (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/TableS3.docx">Table S3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/TableS5.docx">Table S5</a>), and information about predicted microsatellite loci (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/FigureS2.jpg">Figure S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/TableS8.xlsx">Table S8</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/TableS9.xlsx">Table S9</a>) are provided in the online supplemental information. The custom python program is available at <a href="https://github.com/jsharbrough/grabContigs">github.com/jsharbrough/grabContigs</a>. </p></div></div><div class="section results-discussion" id="sec-9"><h2 class="">Results and Discussion</h2><div id="sec-10" class="subsection"><h3>Transcriptome assembly and annotation</h3><p id="p-22">We generated <em>de novo</em> transcriptome assemblies for stage five <em>Microphallus</em> metacercariae isolated from field-collected PA-<em>Microphallus</em> and PE-<em>Microphallus</em>. Our final transcriptome assemblies contained 15,435 and 29,564 transcripts for PA-<em>Microphallus</em> and PE-<em>Microphallus</em>, respectively. The assembly for PE-<em>Microphallus</em> contained approximately twice as many transcripts and was 33% longer than the assembly for PA-<em>Microphallus</em>. The PA-<em>Microphallus</em> assembly had, on average, 22% longer transcripts and 23% larger N50 than the PE-<em>Microphallus</em> assembly (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The marked differences in these assembly statistics are, at least in part, likely linked to differences in sequencing depth and technological advances between the two rounds of sequencing but could also reflect, for example, biological differences between <em>Microphallus</em> found in freshwater (inhabiting <em>P. antipodarum</em>) <em>vs.</em> estuarine (inhabiting <em>P. estuarinus</em>) habitats and/or differences in genetic variation (<em>e.g.</em>, heterozygosity) of the sequenced samples.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/483669/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/483669/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Assembly statistics for whole transcriptome assemblies and ortholog transcriptome assemblies for PA-<em>Microphallus</em> and PE-<em>Microphallus</em> (present study) as well as for two previously published Platyhelminthes transcriptomes: the trematode <em>Schistosoma mansoni</em> and cestode <em>Echinococcus granulosus</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-24">We obtained both blastx and GO annotations for 60.1 and 46.6% of transcripts and only blastx annotations (including those with GO mapping, but not GO annotation) for an additional 24.6 and 29.8% of transcripts for PA-<em>Microphallus</em> and PE-<em>Microphallus</em>, respectively. There was a mean of ∼3.4 GO annotations per transcripts per transcriptome. The remaining 15.2% of PA-<em>Microphallus</em> transcripts and 23.6% of PE-<em>Microphallus</em> transcripts were unable to be annotated, which is typical for <em>de novo</em> assemblies of nonmodel taxa (<em>e.g.</em>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Guo <em>et al.</em> 2015</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Theissinger <em>et al.</em> 2016</a>). The species that were most frequently the top BLAST hit for a given transcript were most often other trematodes (10,053 transcripts, 77.4% for PA-<em>Microphallus</em>; 12,342 transcripts, 54.9% for PE-<em>Microphallus</em>), or nontrematode members of Platyhelminthes (an additional 332 transcripts, 2.6% for PA-<em>Microphallus</em>; 382 transcripts, 1.7% for PE-<em>Microphallus</em>) (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/TableS1.docx">Table S1</a>).</p><p id="p-25">We used BUSCO to estimate the completeness of our transcriptome assemblies based on the presence, absence, duplication, and/or fragmentation of conserved Metazoan genes. Both transcriptomes contain complete transcripts for ∼68% of the 843 conserved single-copy Metazoan genes in the BUSCO database. Our transcriptome assemblies contain gene fragments for an additional 7.8% (PA-<em>Microphallus</em>) and 14.5% (PE-<em>Microphallus</em>) of the conserved single-copy Metazoan genes, for a total of 76.5% (PA-<em>Microphallus</em>) and 82.4% (PE-<em>Microphallus</em>) of the 843 BUSCO genes represented in the transcriptomes (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Considered together with our assembly statistics (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), these results indicate that our transcriptomes are of good quality, are reasonably complete, and are qualitatively similar to other recently published <em>de novo</em> transcriptome assemblies (<em>e.g.</em>, <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Guo <em>et al.</em> 2015</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hara <em>et al.</em> 2015</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Tassone <em>et al.</em> 2016</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Theissinger <em>et al.</em> 2016</a>), despite the differences in transcript number and N50 between the two transcriptome assemblies. Because we analyzed transcriptomes generated from only one stage of infection from a parasite with a complex life cycle, at least some of the missing BUSCO genes likely represent genes that are not expressed in metacercariae rather than due to sequencing or assembly limitations. Regardless, because stage five metacercariae are ready for transmission to the final host (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Levri and Lively 1996</a>), these data provide an excellent first step toward identifying the genes involved in a critical stage of parasite development.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/483688/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/483688/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">BUSCO assessment for transcriptome completeness</span><div class="sb-div caption-clear"></div></div></div><p id="p-27">We generated local BLAST databases for both annotated <em>Microphallus</em> transcriptomes that allow for simple queries of nucleotide and protein sequences (<a href="http://bioweb.biology.uiowa.edu/neiman/Potamomics.php">http://bioweb.biology.uiowa.edu/neiman/Potamomics.php</a>). This resource will be broadly useful for future studies geared toward identifying candidate genes from <em>Microphallus</em> as well as other related ecologically and/or medically important parasites by providing an easy way to search for genes in a well-characterized host–parasite system. These databases will also be an asset for phylogenetic or molecular evolution-focused studies that would benefit from the addition of data from Trematoda.</p></div><div id="sec-11" class="subsection"><h3>Ortholog identification and levels of genetic variation within and genetic differentiation between Microphallus samples</h3><p id="p-28">We identified 7584 pairs of one-to-one orthologous transcripts between the two transcriptomes (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/TableS2.xlsx">Table S2</a>) and 7851 and 21,980 nonone-to-one orthologous transcripts for PA-<em>Microphallus</em> and PE-<em>Microphallus</em>, respectively. The two one-to-one ortholog transcriptomes contain higher mean transcript lengths (18.6% longer for PA-<em>Microphallus</em>; 38.7% longer for PE-<em>Microphallus</em>) and larger N50 values (11.7% greater N50 for PA-<em>Microphallus</em>; 33.9% greater N50 for PE-<em>Microphallus</em>) than their corresponding reference transcriptomes (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). The longer transcripts and higher N50 values in the one-to-one ortholog sets relative to the reference transcriptomes indicate that the one-to-one orthologs represent a set of approximately full-length high quality genes that occur once and only once in both transcriptome assemblies. More than half (PA-<em>Microphallus</em>: 51.7%; PE-<em>Microphallus</em>: 51.2%) of the BUSCO genes are retained in this reduced gene set, further indicating that our filtering steps likely removed poor quality transcripts from our ortholog transcriptomes (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). The annotation statistics for the ortholog transcriptomes are broadly similar to the annotation statistics for the reference transcriptomes (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/TableS1.docx">Table S1</a>), suggesting the ortholog transcriptomes are likely representative of the reference transcriptomes. We also obtained transcriptome assemblies for two previously published Platyhelminthes transcriptomes that were readily available, well-characterized, and among the top-hit species in our transcriptome annotations (<a href="http://www.sanger.ac.uk/resources/downloads/helminths/">http://www.sanger.ac.uk/resources/downloads/helminths/</a>), the trematode <em>Schistosoma mansoni</em> (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Protasio <em>et al.</em> 2012</a>) and the cestode <em>Echinococcus granulosus</em> (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Tsai <em>et al.</em> 2013</a>). We used these transcriptomes to further assess the quality of our transcriptomes relative to previously published assemblies (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/TableS1.docx">Table S1</a>); this comparison reinforces our conclusion that our transcriptomes are of high quality (<em>e.g.</em>, higher N50, mean transcript length).</p><p id="p-29">Of the 7584 transcripts in the ortholog transcriptomes, a distinct majority received both blastx and GO annotation for PA-<em>Microphallus</em> (5746 transcripts, 75.8%) and PE-<em>Microphallus</em> (5793 transcripts, 76.4%). An additional 15.9% (1211 transcripts) and 15.8% (1203 transcripts) of these transcripts received only blastx annotations for PA-<em>Microphallus</em> and PE-<em>Microphallus</em>, respectively. The species that were top BLAST hits most often and the species that were hit most frequently in blastx searches include other trematodes (five species), as well as nontrematode Platyhelminthes (two species) and other metazoans with well-annotated genomic and/or transcriptomic resources (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/TableS1.docx">Table S1</a>). Species within Platyhelminthes were the top blastx hit for 90.6% (6337 transcripts) and 90.3% (6356 transcripts) of the orthologous transcripts that received blastx hits for PA-<em>Microphallus</em> and PE-<em>Microphallus</em>, respectively. The remaining 663 annotated orthologs for PA-<em>Microphallus</em> received top BLAST hits from mollusks (393 transcripts, 5.6%), other metazoans (253 transcripts, 3.6%), or nonmetazoan taxa (17 transcripts, 0.2%), and the remaining 682 annotated PE-<em>Microphallus</em> transcripts received top BLAST hits from mollusks (418 transcripts, 5.9%), other metazoans (250 transcripts, 3.6%), or nonmetazoan taxa (14 transcripts, 0.2%). The relatively small proportion of orthologous transcripts with a mollusk as their top hit could reflect low levels of contamination during the process of isolating metacercariae from snail tissue, but is likely also linked in part to availability of well-annotated genomic and/or transcriptomic resources for the specific species that were hit (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/TableS1.docx">Table S1</a>).</p><p id="p-30">We used PoPoolation to calculate allelic diversity using SNPs (Watterson’s θ) for each orthologous transcript by mapping PA-<em>Microphallus</em> and PE-<em>Microphallus</em> reads to their own ortholog transcriptomes. Mean (±SD) θ per transcript for PA-<em>Microphallus</em> mapped to the PA-<em>Microphallus</em> orthologs was 0.0044 (±0.0039) and median θ was 0.0036. Mean (±SD) θ per transcript for PE-<em>Microphallus</em> mapped to the PE-<em>Microphallus</em> orthologs was 0.0051 (±0.0044) and median θ was 0.0046 (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/FigureS1.jpg">Figure S1A</a>). These estimates of θ are broadly consistent with observations from a previous allozyme-based estimate of θ (0.001–0.021) in several populations of <em>P. antipodarum</em>-infecting <em>Microphallus</em> (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Dybdahl and Lively 1996</a>). Comparisons of medians (Mann–Whitney <em>U</em> test) and distributions of θ (Kolmogorov–Smirnov test) for each transcriptome to one another revealed that θ values are significantly different between the two types of <em>Microphallus</em> (<em>P</em> &lt; 0.001 for both analyses), with θ of PE-<em>Microphallus</em> being on average ∼0.9 times higher than PA-<em>Microphallus</em>. This result thus provides an initial line of evidence for higher genetic variation in PE-<em>Microphallus</em> relative to PA-<em>Microphallus</em>, though one obvious caveat to definitive interpretation of these results is the differences in sampling between the two types of <em>Microphallus</em> in our study. In particular, because the PA-<em>Microphallus</em> sample was produced from pooled RNA from multiple individuals, we cannot tie sequence differences back to individual worms. On the other hand, because we only sequenced a single PE-<em>Microphallus</em> individual, we can definitively ascribe within-sample variation to individual variation, but are not able to evaluate across-individual polymorphism.</p><p id="p-31">Next, we used PoPoolation2 to estimate <em>F<sub>ST</sub></em> at each SNP between PA-<em>Microphallus</em> and PE-<em>Microphallus</em>. This analysis allowed us to estimate relative levels of genetic differentiation among populations based on variance in allele frequencies between the two <em>Microphallus</em> samples (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Holsinger and Weir 2009</a>). In order to account for the ortholog transcriptome to which reads were mapped, we calculated <em>F<sub>ST</sub></em> per SNP between PA-<em>Microphallus</em> and PE-<em>Microphallus</em> mapped to the PA-<em>Microphallus</em> ortholog transcriptome and calculated <em>F<sub>ST</sub></em> per SNP between PA-<em>Microphallus</em> and PE-<em>Microphallus</em> mapped to the PE-<em>Microphallus</em> ortholog transcriptome. We found a mean (±SD) <em>F<sub>ST</sub></em> per SNP between the two types of <em>Microphallus</em> mapped to the PA-<em>Microphallus</em> ortholog transcriptome of 0.129 (±0.129); median <em>F<sub>ST</sub></em> per SNP was 0.085. Mean (±SD) <em>F<sub>ST</sub></em> per SNP between the two types of <em>Microphallus</em> mapped to the PE-<em>Microphallus</em> ortholog transcriptome was 0.148 (±0.138), and median <em>F<sub>ST</sub></em> per SNP was 0.102 (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/FigureS1.jpg">Figure S1B</a>). We supplemented these population genetic-based estimates of genetic differentiation with a phylogeny-based divergence estimation approach, using Clustal Omega to generate multiple sequence alignments and to calculate Jukes–Cantor-corrected pairwise molecular distance for each pair of orthologous transcripts. The mean pairwise molecular distance (±SD) between the two transcriptomes was 0.034 (±0.062) and the median was 0.019 (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/FigureS1.jpg">Figure S1C</a>).</p><p id="p-32">These comparisons provide the first report of which we are aware suggesting that the <em>Microphallus</em> that infect <em>P. antipodarum</em> and <em>P. estuarinus</em> represent distinct genetic lineages. In particular, our <em>F<sub>ST</sub></em>-based estimates of divergence between these two <em>Microphallus</em> types are an order of magnitude higher than the <em>F<sub>ST</sub></em> values reported in an early allozyme-based study comparing <em>F<sub>ST</sub></em> across populations of <em>Microphallus</em> infecting <em>P. antipodarum</em> (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Dybdahl and Lively 1996</a>). This result thus provides an initial line of evidence for marked genetic differentiation between these two <em>Microphallus</em> types relative to the across-population differentiation observed for <em>Microphallus</em> infecting <em>P. antipodarum</em>. Our pairwise molecular distance analyses further suggest that these two types of <em>Microphallus</em> are genetically distinct and are consistent with estimates of genetic variation from prior studies from across the genus. <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">O’Dwyer <em>et al.</em> (2014)</a> estimated pairwise molecular distance (uncorrected p-distance) at <em>cox1</em> to be ∼0.002 for lineages of <em>Microphallus</em> that were isolated from the gastropod mollusk <em>Austrolittorina cincta</em>, collected in Otago Harbor, New Zealand. By contrast, pairwise distances between these sequences and previously published <em>cox1</em> sequences of <em>Microphallus</em> infecting the mud snail <em>Zeacumantus subcarinatus</em> were orders of magnitude higher (0.21–0.25) than within the <em>A. cinta</em>-infecting <em>Microphallus</em>. Another relevant recent example comes from <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Galaktionov <em>et al.</em> (2012)</a>, who measured absolute pairwise differences and p-distances of ITS1 and ITS2 between six <em>Microphallus</em> species that infect gastropods from the genus <em>Littorina</em> from across the Holarctic, and found that pairwise distances within the <em>Microphallus</em> genus infecting congeneric gastropods (<em>Littorina</em>) ranged from 0 to 0.16 at these genes. Our results for mean pairwise molecular distance between PA-<em>Microphallus</em> and PE-<em>Microphallus</em> (0.034) are thus broadly consistent with other reports of variation within and among other <em>Microphallus</em> taxa. These new data also permit some useful comparisons: these pairwise distances are an order of magnitude higher than pairwise distances between <em>Microphallus</em> lineages collected from a single molluscan species (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">O’Dwyer <em>et al.</em> 2014</a>), an order of magnitude lower than pairwise distances between <em>Microphallus</em> lineages collected from different genera of mollusks (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">O’Dwyer <em>et al.</em> 2014</a>), and are within the range of previously reported pairwise distances for <em>Microphallus</em> lineages infecting congeneric gastropods (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Galaktionov <em>et al.</em> 2012</a>). Taken together, the outcomes of these analyses are consistent with a scenario where PA-<em>Microphallus</em> and PE-<em>Microphallus</em> represent distinct genetic lineages relative to <em>Microphallus</em> collected from different populations of <em>P. antipodarum</em>.</p></div><div id="sec-12" class="subsection"><h3>Orthologous transcripts containing F<sub>ST</sub> outlier SNPs</h3><p id="p-33">Using the <em>F<sub>ST</sub></em> analyses described above, we identified 36 <em>F<sub>ST</sub></em> outlier-containing transcripts between PA-<em>Microphallus</em> and PE-<em>Microphallus</em> when mapping to the PA-<em>Microphallus</em> ortholog transcriptome. Of these 36 transcripts, 31 received blastx and GO annotation, one received only blastx annotation, and four were unable to be annotated (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/TableS3.docx">Table S3</a>). A total of 31 of the annotated transcripts had top blastx hit species within Trematoda (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/TableS4.docx">Table S4</a>). Based on Blast2GO annotations, these <em>F<sub>ST</sub></em> outlier-containing transcripts appear to be involved in cellular processes, metabolism, biological regulation, and localization (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/TableS5.docx">Table S5</a>). When mapping to the PE-<em>Microphallus</em> ortholog transcriptome, we identified 30 <em>F<sub>ST</sub></em> outlier-containing transcripts. Of these transcripts, 22 received both blastx and GO annotations, three received only blastx annotation, and three were unable to be annotated (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/TableS3.docx">Table S3</a>). All 25 of the annotated transcripts received top blastx hit species within Platyhelminthes, 24 of which were within Trematoda (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/TableS4.docx">Table S4</a>). The annotated <em>F<sub>ST</sub></em> outlier-containing transcripts identified when mapping to the PE-<em>Microphallus</em> orthologs appear to be involved in similar biological processes as those identified by mapping to PA<em>-Microphallus</em> (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/TableS5.docx">Table S5</a>).</p><p id="p-34">Only two of the transcripts were identified as <em>F<sub>ST</sub></em> outliers in both comparisons. Both of these transcripts could represent genes that are evolving relatively rapidly compared to their respective ortholog transcriptome and/or may be potential targets of selection in one or both <em>Microphallus</em> types. These candidate genes are <em>N</em>-α-acetyltransferase, which is involved in the <em>N</em>-terminal acetyltransferase C (NatC) complex and acetylation of amino acids, and permease 1 heavy chain, which is involved in maltose metabolic processes (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/TableS3.docx">Table S3</a>). Genes involved in the NatC pathway have been implicated in modulating stress tolerance in <em>Caenorhabditis elegans</em> (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Warnhoff and Kornfeld 2015</a>). In schistosomes (Trematoda), permease 1 heavy chain is characterized as playing an important role in nutrient uptake from hosts and is expressed during all major schistosome life stages (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Krautz-Peterson <em>et al.</em> 2007</a>). This information from a related trematode group with a broadly similar life cycle indicates that permease 1 heavy chain might also play a role in nutrient acquisition in <em>Microphallus</em>. More directed study of patterns of molecular evolution and the roles that these two genes play in response to physiological stresses associated with freshwater <em>vs.</em> estuarine environments (<em>N</em>-α-acetyltransferase), as well as nutrient uptake by parasites from hosts (permease 1 heavy chain), in <em>Microphallus</em> will provide more definitive insights into their functions.</p></div><div id="sec-13" class="subsection"><h3>GO functional enrichment and KEGG pathway identification</h3><p id="p-35">We used Blast2GO to perform functional enrichment analyses comparing our ortholog transcriptomes against their associated reference transcriptomes, with the goal of identifying overrepresented <em>vs.</em> underrepresented functional groups that are shared between the two types of <em>Microphallus</em>. We also used functional enrichment analyses of our nonone-to-one orthologs against their associated reference transcriptomes to assess the types of genes that are overrepresented <em>vs.</em> underrepresented among the transcripts that did not represent one-to-one orthologs between PA-<em>Microphallus</em> and PE-<em>Microphallus</em>. These analyses revealed seven GO terms that were significantly underrepresented among orthologous transcripts in PA-<em>Microphallus</em> relative to the PA-<em>Microphallus</em> reference transcriptome. These same seven functional groups are significantly overrepresented among the nonone-to-one orthologs for PA-<em>Microphallus</em> relative to the PA-<em>Microphallus</em> reference transcriptome (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/TableS6.docx">Table S6</a>). For the PE-<em>Microphallus</em> orthologs relative to the PE-<em>Microphallus</em> reference, we identified 20 significantly underrepresented and six significantly overrepresented functional groups. There were five underrepresented and 14 overrepresented functional groups among the nonone-to-one PE-<em>Microphallus</em> orthologs relative to the PE-<em>Microphallus</em> transcriptome. All but four of the functional groups among the PE-<em>Microphallus</em> transcripts were only significantly overrepresented or underrepresented among either the one-to-one or nonone-to-one orthologs but were never significantly overrepresented or underrepresented in both transcript sets (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/TableS6.docx">Table S6</a>).</p><p id="p-36">There were only two GO categories that were that were overrepresented or underrepresented among orthologs from both types of <em>Microphallus</em>. First, genes with functions related to hydrolase activity are underrepresented among one-to-one orthologs and overrepresented among nonone-to-one orthologs for both PA-<em>Microphallus</em> and PE-<em>Microphallus</em>. The presence of genes involved in hydrolase activity among genes that were significantly functionally enriched is interesting in light of evidence that hydrolases are involved in damaging epithelial lining of the digestive tracts of hosts by the parasitic nematode <em>Contracaeum rudolphii</em> (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dziekońska-Rynko and Rokicki 2005</a>). Second, genes involved in carbohydrate metabolic processes are underrepresented among one-to-one orthologs for both types of <em>Microphallus</em> relative to the reference transcriptomes. Evidence that schistosome parasites dynamically shift carbohydrate metabolic processes throughout the course of their life cycle (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Horemans <em>et al.</em> 1992</a>) hint that these genes could also be of relevance to <em>Microphallus</em>. Altogether, both of these functional categories of genes provide a promising set of candidates for assessment of the changes in gene expression and physiological responses associated with the infection process in <em>Microphallus</em>. The marked differences in functional groups that are significantly overrepresented <em>vs.</em> underrepresented in the ortholog transcriptomes relative to the reference transcriptomes between the two <em>Microphallus</em> types suggest that the types of genes present in the two reference transcriptomes are often different. Whether this result is more a consequence of biological differences between PA-<em>Microphallus</em> and PE-<em>Microphallus</em> or differences in sequencing depth cannot be determined at this time but warrants future study.</p><p id="p-37">Finally, we performed KEGG analyses as implemented in Blast2GO to identify the types of enzyme pathways that were being expressed in each transcriptome. This analysis identified 123 KEGG pathways in the PA-<em>Microphallus</em> transcriptome and 128 KEGG pathways in the PE-<em>Microphallus</em> transcriptome (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/TableS7.docx">Table S7</a>). All but 11 of the KEGG pathways that we identified were expressed in both transcriptomes, three were expressed in PA-<em>Microphallus</em> only, and eight were expressed in PE-<em>Microphallus</em> only. Most of the 120 pathways present in both <em>Microphallus</em> transcriptomes are involved in various metabolic processes (49 pathways, 40.2%) or biosynthetic processes (41 pathways, 33.6%). Of the 11 pathways only identified in a single transcriptome, the three pathways identified in only PA-<em>Microphallus</em> include the riboflavin metabolic pathway, the penicillin and cephalosporin biosynthesis pathway, and the biosynthesis of siderophore group nonribosomal peptides. The eight pathways that were identified in only PE-<em>Microphallus</em> include a variety of biosynthesis and metabolic processes (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/TableS7.docx">Table S7</a>). One such pathway, glycosphingolipid biosynthesis, has been implicated in response to salinity stress in <em>Litopenaeus vannamei</em>, the Pacific white shrimp (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chen <em>et al.</em> 2015</a>). This result is suggestive of a scenario where these genes contribute to the ability of PE-<em>Microphallus</em> to tolerate salinity fluctuations associated with estuarine habitats. This important habitat difference between PA-<em>Microphallus</em> and PE-<em>Microphallus</em> is likely to differentially influence the selective pressures experienced by these two parasites and thus has the potential to influence patterns and rates of molecular evolution (<em>e.g.</em>, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Mitterboeck <em>et al.</em> 2016</a>), making genes involved in salinity tolerance and/or stress particularly interesting candidates for future study.</p></div><div id="sec-14" class="subsection"><h3>Predictions of microsatellite loci and primers</h3><p id="p-38">We used MISA software to identify SSRs within the ortholog transcriptomes and used Primer3 and BLAST to predict primers that could be used for future microsatellite-based analyses of <em>Microphallus</em>. We identified 109 SSRs in 100 transcripts for PA-<em>Microphallus</em> and 117 SSRs in 107 transcripts for PE-<em>Microphallus</em>. We identified all six types of microsatellites for which we searched, with trinucleotide sequences with at least five repeats representing the most common microsatellite type (62.3% of SSRs identified in PA-<em>Microphallus</em> and 60.7% of SSRs identified in PE-<em>Microphallus</em>) (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>; see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/FigureS2.jpg">Figure S2</a>). Eight and four SSRs, respectively, were defined as compound for PA-<em>Microphallus</em> and PE-<em>Microphallus</em> (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). We also used perl scripts available with MISA (<a href="http://pgrc.ipk-gatersleben.de/misa/">http://pgrc.ipk-gatersleben.de/misa/</a>) to compile specific details about the primer sets (<em>e.g.</em>, location, size of product, and melting temperature). We generated predicted primers for 84 of the PA-<em>Microphallus</em> SSRs from 100 sequences and for 83 of the PE-<em>Microphallus</em> SSRs from 113 sequences (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/TableS8.xlsx">Table S8</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1/TableS9.xlsx">Table S9</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/483560/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/483560/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Summary of potential microsatellite loci for the PA-<em>Microphallus</em> and PE-<em>Microphallus</em> one-to-one ortholog transcriptome assemblies</span><div class="sb-div caption-clear"></div></div></div><p id="p-39">Future studies will be able to use these markers to characterize, for example, population genetic structure within and genetic differentiation between these two types of <em>Microphallus</em> isolated from two ecologically distinct snail taxa. Comparing patterns of molecular evolution among these orthologous genes from more <em>Microphallus</em> individuals isolated from <em>P. antipodarum</em> and <em>P. estuarinus</em> will allow for research directed at characterizing the strength and efficacy of selection and the importance of population structure in the context of a prominent model for host–parasite interactions and coevolution. Researchers can also use these resources to perform cost-effective analyses incorporating many <em>Microphallus</em> individuals from multiple populations to address key questions regarding the evolutionary connections between these two types of <em>Microphallus</em> (<em>e.g.</em>, whether there is gene flow between the <em>Microphallus</em> infecting <em>P. antipodarum</em> and <em>P. estuarinus</em> and whether there is similar evidence for local adaptation and coevolution between <em>P. estuarinus</em> and <em>Microphallus</em> as exists for <em>P. antipodarum</em> and <em>M. livelyi</em>).</p></div><div id="sec-15" class="subsection"><h3>Caveats and conclusions</h3><p id="p-40">Multiple recent studies have shown that pooled RNA-Seq data can be used to effectively identify SNPs and candidate genes and to perform <em>F<sub>ST</sub></em>-based measures of genetic differentiation and other population genetic measures (<em>e.g.</em>, <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Fischer <em>et al.</em> 2013</a>; <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Konczal <em>et al.</em> 2014</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Li <em>et al.</em> 2014</a>; <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Thavamanikumar <em>et al.</em> 2014</a>; Ellison <em>et al.</em> 2016). We used analysis approaches [PoPoolation (used to estimate θ), and PoPoolation2 (used to estimate <em>F<sub>ST</sub></em>)] that are designed to handle pooled sequencing data and explicitly incorporate number of individuals in the sequencing pools as a parameter in estimations of nucleotide variation and genetic differentiation (<a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Kofler <em>et al.</em> 2011a</a>,<a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">b</a>). For these reasons, we believe that we applied the most appropriate means available to analyze our data.</p><p id="p-41">Even so, it is important to acknowledge the potential caveats and limitations associated with pooled RNA-Seq approaches (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Konczal <em>et al.</em> 2014</a>). First, in order to reduce the likelihood of including artefactual SNPs, we required that each SNP to be sequenced at least four times and filtered out SNPs with minor allele frequencies &lt;5%, at the risk of missing alleles at low frequencies. Second, differences in gene expression among individuals and/or loci can complicate comparisons among samples. We tried to minimize the effects of these potential expression differences by requiring a minimum of 10× coverage and a maximum of 200× coverage for both the θ and <em>F<sub>ST</sub></em> estimates. Another concern is that allele frequency estimates may be less accurate in regions of low expression (<a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Konczal <em>et al.</em> 2014</a>). As such, requiring a minimum of 10× coverage should reduce this effect, but does generate potential to miss some lowly expressed SNPs. The existence of allele-specific expression may be problematic in leading to underestimates of the number of heterozygous sites and overestimates of <em>F<sub>ST</sub></em> in pooled sequencing data. These concerns are at least partially obviated by results of simulations presented in <a id="xref-ref-33-6" class="xref-bibr" href="#ref-33">Konczal <em>et al.</em> (2014)</a>, who found that allele-specific expression only influences allele frequency estimation in a small fraction (∼1%) of genes (in a data set).</p><p id="p-42">In conclusion, our study provides a set of widely useful transcriptomic and genomic resources for an evolutionarily and ecologically important group of parasites. <em>P. antipodarum</em> and <em>M. livelyi</em> are a textbook system for the study of host–parasite coevolutionary interactions, and the data generated herein represent the first large-scale genome-based resources for <em>Microphallus</em>. The annotated transcriptome assemblies, orthologous gene sets, BLAST databases, SNPs, and microsatellite loci and primers will facilitate research for scientists interested in host–parasite coevolution, local adaption, and/or patterns of molecular evolution and phylogenetics of these and other trematode parasites. Our <em>F<sub>ST</sub></em>, functional enrichment, and KEGG pathway analyses provide a promising set of candidate genes and pathways for future study and further characterization for potential roles in physiological and stress responses relevant to organisms inhabiting freshwater <em>vs.</em> estuarine environments, as well as genes involved in parasitic infection of host taxa.</p></div></div><div class="section" id="sec-16"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-43">We thank Gery Hehman for RNA sequencing assistance; Curt Lively, Daniela Vergara, Katelyn Larkin, and Mike Winterbourn for field collections; and Joel Sharbrough for the custom python program (github.com/jsharbrough/grabContigs). We are grateful to several anonymous reviewers for constructive feedback on earlier versions of this manuscript. This project was funded by Sigma-Xi (grant number 18911100 BR01 to L.B.), the Iowa Science Foundation (grant number 14-03 to M.N. and L.B.), the National Geographic Society (grant number 9595-14 to K. Larkin), and the National Science Foundation (National Science Foundation-Division of Molecular and Cellular Biosciences grant number 1122176 to M.N.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-44">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037275/-/DC1</a>.</p></li><li class="fn" id="fn-4"><p id="p-45"><em>Communicating editor: M. Walhout</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-09-11"><span class="received-label">Received </span>September 11, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-01-09"><span class="accepted-label">Accepted </span>January 9, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Bankers and Neiman</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-web" id="cit-7.3.871.1"><div class="cit-metadata unstructured">Andrews, S., 2010 FastQC: a quality control tool for high throughput sequence data. Available at: <a href="http://www.bioinformatics.babraham.ac.uk?/projects/fastqc/">http://www.bioinformatics.babraham.ac.uk?/projects/fastqc/</a>. Accessed: November 4, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.2" data-doi="10.1101/045674"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bankers</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fields</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McElroy</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boore</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Logsdon</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic evidence for population-specific responses to coevolving parasite in a New Zealand freshwater snail.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1101/045674</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F045674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiIwNDU2NzR2MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMy84NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.3" data-doi="10.1186/1471-2105-10-421"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Camacho</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulouris</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Avagyan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papadopoulos</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">BLAST+: architecture and applications.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DCamacho%26rft.auinit1%253DC.%26rft.volume%253D10%26rft.spage%253D421%26rft.epage%253D421%26rft.atitle%253DBLAST%252B%253A%2Barchitecture%2Band%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-421%26rft_id%253Dinfo%253Apmid%252F20003500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-421&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20003500&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carmona</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connor</span>  <span class="cit-name-given-names">F. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fifty years of co-evolution and beyond: integrating co-evolution from molecules to species.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">5315</span>–<span class="cit-lpage">5329</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D2%26rft.spage%253D5315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.5" data-doi="10.1371/journal.pone.0131503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Transcriptome and molecular pathway analysis of the hepatopancreas in the Pacific white shrimp <em>Litopenaeus vannamei</em> under chronic low-salinity stress.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0131503</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0131503%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0131503%26rft_id%253Dinfo%253Apmid%252F26147449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0131503&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26147449&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chomczynski</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">A reagent for the single-step simultaneous isolation of RNA, DNA, and proteins from cell and tissue samples.</span> <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">532</span>–<span class="cit-lpage">537</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioTechniques%26rft.stitle%253DBiotechniques%26rft.aulast%253DChomczynski%26rft.auinit1%253DP.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D532%26rft.epage%253D532-4%252C%2B537%26rft.atitle%253DA%2Breagent%2Bfor%2Bthe%2Bsingle-step%2Bsimultaneous%2Bisolation%2Bof%2BRNA%252C%2BDNA%2Band%2Bproteins%2Bfrom%2Bcell%2Band%2Btissue%2Bsamples.%26rft_id%253Dinfo%253Apmid%252F7692896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=7692896&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LW41700037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.7" data-doi="10.1016/0003-2697(87)90021-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chomczynski</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sacchi</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Single-step method of RNA isolation by acid guanidinium thiocyanate-phenol-chloroform extraction.</span> <abbr class="cit-jnl-abbrev">Anal. Biochem.</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">156</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnalytical%2Bbiochemistry%26rft.stitle%253DAnal%2BBiochem%26rft.aulast%253DChomczynski%26rft.auinit1%253DP.%26rft.volume%253D162%26rft.issue%253D1%26rft.spage%253D156%26rft.epage%253D159%26rft.atitle%253DSingle-step%2Bmethod%2Bof%2BRNA%2Bisolation%2Bby%2Bacid%2Bguanidinium%2Bthiocyanate-phenol-chloroform%2Bextraction.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0003-2697%252887%252990021-2%26rft_id%253Dinfo%253Apmid%252F2440339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0003-2697(87)90021-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2440339&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987G774500020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.8" data-doi="10.1093/bioinformatics/bti610"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conesa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Götz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">García-Gómez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terol</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talón</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">3674</span>–<span class="cit-lpage">3676</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti610%26rft_id%253Dinfo%253Apmid%252F16081474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjEvMTgvMzY3NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMy84NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-7.3.871.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cribb</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bray</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Littlewood</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pichelin</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herniou</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The digenea</span>, Chapter 16 pp. <span class="cit-fpage">168</span>–<span class="cit-lpage">173</span> in <span class="cit-source">Interrelationships of the Platyhelminthes</span>, Vol. <span class="cit-vol">1</span>, Ser. 60. <span class="cit-publ-name">Taylor and Francis</span>, <span class="cit-publ-loc">New York, NY</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInterrelationships%2Bof%2Bthe%2BPlatyhelminthes%26rft.volume%253D1%26rft.spage%253D168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.10" data-doi="10.1038/ng.806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garimella</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478889&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.11" data-doi="10.2307/2410696"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dybdahl</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lively</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The geography of coevolution: comparative population structures for a snail and its trematode parasite.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">2264</span>–<span class="cit-lpage">2275</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D50%26rft.spage%253D2264%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2410696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2410696&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996WB94600013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dybdahl</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krist</span>  <span class="cit-name-given-names">A. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genotypic <em>vs.</em> condition effects on parasite-driven rare advantage.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">967</span>–<span class="cit-lpage">973</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.stitle%253DJ%2BEvol%2BBiol%26rft.aulast%253DDybdahl%26rft.auinit1%253DM.%2BF.%26rft.volume%253D17%26rft.issue%253D5%26rft.spage%253D967%26rft.epage%253D973%26rft.atitle%253DGenotypic%2Bvs.%2Bcondition%2Beffects%2Bon%2Bparasite-driven%2Brare%2Badvantage.%26rft_id%253Dinfo%253Apmid%252F15312069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15312069&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dziekońska-Rynko</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokicki</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Activity of selected hydrolases in excretion-secretion products and extracts of adult <em>Contracaecum rudolphii</em>.</span> <abbr class="cit-jnl-abbrev">Wiad. Parazytol.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">227</span>–<span class="cit-lpage">231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWiadomo%25C5%253Fci%2Bparazytologiczne%26rft.stitle%253DWiad%2BParazytol%26rft.aulast%253DDziekonska-Rynko%26rft.auinit1%253DJ.%26rft.volume%253D51%26rft.issue%253D3%26rft.spage%253D227%26rft.epage%253D231%26rft.atitle%253DActivity%2Bof%2Bselected%2Bhydrolases%2Bin%2Bexcretion-secretion%2Bproducts%2Band%2Bextracts%2Bof%2Badult%2BContracaecum%2Brudolphii.%26rft_id%253Dinfo%253Apmid%252F16913527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16913527&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.3.871.14" data-doi="10.1534/g3.116.035873"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellison</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DiRenzo</span>  <span class="cit-name-given-names">G. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lips</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zamudio</span>  <span class="cit-name-given-names">K. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">First <em>in vivo Batrachochytrium dendrobatidis</em> transcriptomes reveal mechanisms of host exploitation, host-specific gene expression, and expressed genotype shifts</span>. <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">269</span>−<span class="cit-lpage">278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.035873%26rft_id%253Dinfo%253Apmid%252F27856699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjcvMS8yNjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvODcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.15" data-doi="10.1111/mec.12521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rellstab</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tedder</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zoller</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gugerli</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Population genomic footprints of selection and associations with climate in natural populations of <em>Arabidopsis halleri</em> from the Alps.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">5594</span>–<span class="cit-lpage">5607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D5594%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000329786700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galaktionov</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blasco-Costa</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Life cycle, molecular phylogeny and historical biogeography of the ‘<em>pygmaeus</em>’ microphallids (Digenea: Microphallidae): widespread parasites of marine and coastal birds in the Holarctic.</span> <abbr class="cit-jnl-abbrev">Parasitology</abbr> <span class="cit-vol">139</span>: <span class="cit-fpage">1346</span>–<span class="cit-lpage">1360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DParasitology%26rft.aulast%253DGalaktionov%26rft.auinit1%253DK.%2BV.%26rft.volume%253D139%26rft.issue%253D10%26rft.spage%253D1346%26rft.epage%253D1360%26rft.atitle%253DLife%2Bcycles%252C%2Bmolecular%2Bphylogeny%2Band%2Bhistorical%2Bbiogeography%2Bof%2Bthe%2B%2527pygmaeus%2527%2Bmicrophallids%2B%2528Digenea%253A%2BMicrophallidae%2529%253A%2Bwidespread%2Bparasites%2Bof%2Bmarine%2Band%2Bcoastal%2Bbirds%2Bin%2Bthe%2BHolarctic.%26rft_id%253Dinfo%253Apmid%252F22717011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22717011&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.17" data-doi="10.1038/nbt.1883"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">644</span>–<span class="cit-lpage">652</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DGrabherr%26rft.auinit1%253DM.%2BG.%26rft.volume%253D29%26rft.issue%253D7%26rft.spage%253D644%26rft.epage%253D652%26rft.atitle%253DFull-length%2Btranscriptome%2Bassembly%2Bfrom%2BRNA-Seq%2Bdata%2Bwithout%2Ba%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1883%26rft_id%253Dinfo%253Apmid%252F21572440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1883&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21572440&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.18" data-doi="10.1109/TCBB.2013.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gremme</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinbiss</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurtz</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">GenomeTools: a comprehensive software library for efficient processing of structured genome annotations.</span> <abbr class="cit-jnl-abbrev">IEEE/ACM Trans. Comput. Biol. Bioinform.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">645</span>–<span class="cit-lpage">656</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%252FACM%2BTrans.%2BComput.%2BBiol.%2BBioinform.%26rft.volume%253D10%26rft.spage%253D645%26rft_id%253Dinfo%253Adoi%252F10.1109%252FTCBB.2013.68%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/TCBB.2013.68&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-web" id="cit-7.3.871.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hannon</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fastx-toolkit. FASTQ/A short-read pre-processing tools.</span> <a href="http://hannonlab.cshl.edu/fastx_toolkit">http://hannonlab.cshl.edu/fastx_toolkit</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiegert-Rininger</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vallejo</span>  <span class="cit-name-given-names">V. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barry</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warner</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Transcriptome-enabled marker discovery and mapping of plastochron-related genes in <em>Petunia</em> spp.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">726</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D726%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.21" data-doi="10.1038/nprot.2013.084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papanicolaou</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blood</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title"><em>De novo</em> transcript sequence reconstruction from RNA-seq, using the Trinity platform for reference generation and analysis.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1494</span>–<span class="cit-lpage">1512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BProtoc.%26rft.volume%253D8%26rft.spage%253D1494%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2013.084%26rft_id%253Dinfo%253Apmid%252F23845962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2013.084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23845962&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hara</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tatsumi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshida</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kajikawa</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kiyonari</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Optimizing and benchmarking <em>de novo</em> transcriptome sequencing: from library preparation to assembly evaluation.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">977</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hechinger</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Faunal survey and identification key for the trematodes (Platyhelminthes: Digenea) infecting <em>Potamopyrgus antipodarum</em> (Gastropoda: Hydrobiidae) as first intermediate host.</span> <abbr class="cit-jnl-abbrev">Zootaxa</abbr> <span class="cit-vol">3418</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZootaxa%26rft.volume%253D3418%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.24" data-doi="10.1038/nrg2611"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holsinger</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetics in geographically structures populations: defining, estimating and interpreting <em>F<sub>ST</sub></em>.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">639</span>–<span class="cit-lpage">650</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DHolsinger%26rft.auinit1%253DK.%2BE.%26rft.volume%253D10%26rft.issue%253D9%26rft.spage%253D639%26rft.epage%253D650%26rft.atitle%253DGenetics%2Bin%2Bgeographically%2Bstructured%2Bpopulations%253A%2Bdefining%252C%2Bestimating%2Band%2Binterpreting%2BF%2528ST%2529.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2611%26rft_id%253Dinfo%253Apmid%252F19687804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2611&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19687804&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269063200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.25" data-doi="10.1016/0166-6851(92)90202-U"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Horemans</span>  <span class="cit-name-given-names">A. M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tielens</span>  <span class="cit-name-given-names">A. G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van den Bergh</span>  <span class="cit-name-given-names">S. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">The reversible effect of glucose on the energy metabolism of <em>Schistosoma mansoni</em> cercariae and schistosomula.</span> <abbr class="cit-jnl-abbrev">Mol. Biochem. Parasitol.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">73</span>–<span class="cit-lpage">79</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2Bbiochemical%2Bparasitology%26rft.stitle%253DMol%2BBiochem%2BParasitol%26rft.aulast%253DHoremans%26rft.auinit1%253DA.%2BM.%26rft.volume%253D51%26rft.issue%253D1%26rft.spage%253D73%26rft.epage%253D79%26rft.atitle%253DThe%2Breversible%2Beffect%2Bof%2Bglucose%2Bon%2Bthe%2Benergy%2Bmetabolism%2Bof%2BSchistosoma%2Bmansoni%2Bcercariae%2Band%2Bschistosomula.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0166-6851%252892%252990202-U%26rft_id%253Dinfo%253Apmid%252F1565140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0166-6851(92)90202-U&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1565140&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hotez</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Pediatric tropical diseases and the world’s children living in extreme poverty.</span> <abbr class="cit-jnl-abbrev">J. Appl. Res. Child.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAppl.%2BRes.%2BChild.%26rft.volume%253D4%26rft.spage%253D10%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.27" data-doi="10.1093/bioinformatics/btq003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">CD-HIT Suite: a web server for clustering and comparing biological sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">680</span>–<span class="cit-lpage">682</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq003%26rft_id%253Dinfo%253Apmid%252F20053844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi81LzY4MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMy84NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jurberg</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brindley</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Gene function in schistosomes: recent advances toward a cure.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D6%26rft.spage%253D144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.29" data-doi="10.1016/j.cub.2009.06.062"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delph</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jokela</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lively</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The geographic mosaic of sex and the Red Queen.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1438</span>–<span class="cit-lpage">1441</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DKing%26rft.auinit1%253DK.%2BC.%26rft.volume%253D19%26rft.issue%253D17%26rft.spage%253D1438%26rft.epage%253D1441%26rft.atitle%253DThe%2Bgeographic%2Bmosaic%2Bof%2Bsex%2Band%2Bthe%2BRed%2BQueen.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2009.06.062%26rft_id%253Dinfo%253Apmid%252F19631541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2009.06.062&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19631541&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269920500031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.30" data-doi="10.1098/rsbl.2010.0857"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jokela</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lively</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Trematode parasites infect or die in snail hosts.</span> <abbr class="cit-jnl-abbrev">Biol. Lett.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">265</span>–<span class="cit-lpage">268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BLett.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsbl.2010.0857%26rft_id%253Dinfo%253Apmid%252F20961880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InJveWJpb2xldHQiO3M6NToicmVzaWQiO3M6NzoiNy8yLzI2NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMy84NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.31" data-doi="10.1371/journal.pone.0015925"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kofler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orozco-terWengel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Maio</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolte</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">a</span> <span class="cit-article-title">Popoolation: a toolbox for population genetic analysis of next generation sequencing data from pooled individuals.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e15925</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DKofler%26rft.auinit1%253DR.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253De15925%26rft.epage%253De15925%26rft.atitle%253DPoPoolation%253A%2Ba%2Btoolbox%2Bfor%2Bpopulation%2Bgenetic%2Banalysis%2Bof%2Bnext%2Bgeneration%2Bsequencing%2Bdata%2Bfrom%2Bpooled%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0015925%26rft_id%253Dinfo%253Apmid%252F21253599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0015925&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21253599&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.32" data-doi="10.1093/bioinformatics/btr589"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kofler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">b</span> <span class="cit-article-title">Popoolation2: identifying differentiation between populations using sequencing of pooled DNA samples (Pool-Seq).</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">3435</span>–<span class="cit-lpage">3436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr589%26rft_id%253Dinfo%253Apmid%252F22025480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMjQvMzQzNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMy84NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.33" data-doi="10.1111/1755-0998.12186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Konczal</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koteja</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stuglik</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Radwan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babik</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Accuracy of allele frequency estimation using pooled RNA-Seq.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">392</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D14%26rft.spage%253D381%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12186%26rft_id%253Dinfo%253Apmid%252F24119300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24119300&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.34" data-doi="10.1074/jbc.M703512200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krautz-Peterson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camargo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huggel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verrey</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Amino acid transport in schistosomes: characterization of the permease heavy chain SPRM1hc.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">21767</span>–<span class="cit-lpage">21775</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M703512200%26rft_id%253Dinfo%253Apmid%252F17545149%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODIvMzAvMjE3NjciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvODcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.35" data-doi="10.1006/anbe.1996.0093"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levri</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lively</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The effect of size, reproductive condition, and parasitism on foraging behavior in a freshwater snail, <em>Potamopyrgus antipodarum</em>.</span> <abbr class="cit-jnl-abbrev">Anim. Behav.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">891</span>–<span class="cit-lpage">901</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BBehav.%26rft.volume%253D51%26rft.spage%253D891%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fanbe.1996.0093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/anbe.1996.0093&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996UJ53400019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.36" data-doi="10.1093/bioinformatics/18.1.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaroszewski</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Godzik</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Tolerating some redundancy significantly speeds up clustering of large protein databases.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DLi%26rft.auinit1%253DW.%26rft.volume%253D18%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D82%26rft.atitle%253DTolerating%2Bsome%2Bredundancy%2Bsignificantly%2Bspeeds%2Bup%2Bclustering%2B%2Bof%2Blarge%2Bprotein%2Bdatabases%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F18.1.77%26rft_id%253Dinfo%253Apmid%252F11836214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo3OiIxOC8xLzc3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzg3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.37" data-doi="10.1093/biostatistics/kxr031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witten</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnstone</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Normalization, testing, and false discovery rate estimation for RNA-sequencing data.</span> <abbr class="cit-jnl-abbrev">Biostatistics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">523</span>–<span class="cit-lpage">538</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiostatistics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiostatistics%252Fkxr031%26rft_id%253Dinfo%253Apmid%252F22003245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYmlvc3RzIjtzOjU6InJlc2lkIjtzOjg6IjEzLzMvNTIzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzg3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beck</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuller</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peatman</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Transcriptome annotation and marker discovery in white bass (<em>Morone chrysops</em>) and striped bass (<em>Morone saxatilis</em>).</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">885</span>–<span class="cit-lpage">887</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BGenet.%26rft.volume%253D45%26rft.spage%253D885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.39" data-doi="10.1098/rspb.1996.0132"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lively</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jokela</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Clinal variation for local adaptation in a host-parasite interaction.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">263</span>: <span class="cit-fpage">891</span>–<span class="cit-lpage">897</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DLively%26rft.auinit1%253DC.%2BM.%26rft.volume%253D263%26rft.issue%253D1372%26rft.spage%253D891%26rft.epage%253D897%26rft.atitle%253DClinal%2BVariation%2Bfor%2BLocal%2BAdaptation%2Bin%2Ba%2BHost-Parasite%2BInteraction%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.1996.0132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMjoiMjYzLzEzNzIvODkxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzg3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.40" data-doi="10.1086/424605"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lively</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dybdahl</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jokela</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osnas</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delph</span>  <span class="cit-name-given-names">L. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Host sex and local adaptation by parasites in a snail-trematode interaction.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">S6</span>–<span class="cit-lpage">S18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D164%26rft.spage%253DS6%26rft_id%253Dinfo%253Adoi%252F10.1086%252F424605%26rft_id%253Dinfo%253Apmid%252F15540142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/424605&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15540142&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225295500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.41" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMy84NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mitterboeck</span>  <span class="cit-name-given-names">T. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">A. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaheer</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">Y. E. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adamowicz</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Do saline taxa evolve faster? Comparing relative rates of molecular evolution between freshwater and marine eukaryotes.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">1960</span>–<span class="cit-lpage">1978</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D70%26rft.spage%253D1960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Dwyer</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blasco-Costa</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poulin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faltýnková</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Four marine digenean parasites of <em>Austrolittorina</em> spp. (Gastropoda: Littorinidae) in New Zealand: morphological and molecular data.</span> <abbr class="cit-jnl-abbrev">Syst. Parasitol.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">152</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BParasitol.%26rft.volume%253D89%26rft.spage%253D133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Osnas</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lively</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Using definitive host feces to infect experimental intermediate host populations: waterfowl hosts for New Zealand trematodes.</span> <abbr class="cit-jnl-abbrev">N.Z. J. Zool.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.Z.%2BJ.%2BZool.%26rft.volume%253D38%26rft.spage%253D83%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-2" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.45" data-doi="10.1371/journal.pntd.0001455"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Protasio</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babbage</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nichol</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A systematically improved high quality genome and transcriptome of the human blood fluke <em>Schistosoma mansoni</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Negl. Trop. Dis.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BNegl%2BTrop%2BDis%26rft.aulast%253DProtasio%26rft.auinit1%253DA.%2BV.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253De1455%26rft.epage%253De1455%26rft.atitle%253DA%2Bsystematically%2Bimproved%2Bhigh%2Bquality%2Bgenome%2Band%2Btranscriptome%2Bof%2Bthe%2Bhuman%2Bblood%2Bfluke%2BSchistosoma%2Bmansoni.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pntd.0001455%26rft_id%253Dinfo%253Apmid%252F22253936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pntd.0001455&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22253936&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.46" data-doi="10.1111/1755&amp;ndash;0998.12553"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Characterization and analysis of a <em>de novo</em> transcriptome from the pygmy grasshopper <em>Tetrix japonica</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/1755–0998.12553</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755%2526ndash%253B0998.12553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755&amp;ndash;0998.12553&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.47" data-doi="10.1016/j.actatropica.2012.04.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rollinson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knopp</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levitz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stothard</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tchuem Tchuenté</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Time to set the agenda for schistosomiasis elimination.</span> <abbr class="cit-jnl-abbrev">Acta Trop.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BTrop.%26rft.volume%253D128%26rft.spage%253D423%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.actatropica.2012.04.013%26rft_id%253Dinfo%253Apmid%252F22580511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.actatropica.2012.04.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22580511&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000326771900027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Routtu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebert</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic architecture of resistance in <em>Daphnia</em> hosts against two species of host-specific parasite.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D114%26rft.spage%253D241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.49" data-doi="10.1038/msb.2011.75"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sievers</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilm</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dineen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karplus</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Fast, scalable generation of high-qualiy protein multiple sequence alignment using Clustal Omega.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DSievers%26rft.auinit1%253DF.%26rft.volume%253D7%26rft.issue%253D1%26rft.spage%253D539%26rft.epage%253D539%26rft.atitle%253DFast%252C%2Bscalable%2Bgeneration%2Bof%2Bhigh-quality%2Bprotein%2Bmultiple%2Bsequence%2Balignments%2Busing%2BClustal%2BOmega%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2011.75%26rft_id%253Dinfo%253Apmid%252F21988835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjc6IjcvMS81MzkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvODcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.50" data-doi="10.1093/bioinformatics/btv351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simão</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ioannidis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kriventseva</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zdobnov</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3210</span>–<span class="cit-lpage">3212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv351%26rft_id%253Dinfo%253Apmid%252F26059717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMzEvMTkvMzIxMCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMy84NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tassone</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geib</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fabrick</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brent</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title"><em>De novo</em> construction of an expanded transcriptome assembly for the western tarnished plant bug, <em>Lygus hesperus</em>.</span> <abbr class="cit-jnl-abbrev">BMC GigaScience</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGigaScience%26rft.volume%253D5%26rft.spage%253D6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thavamanikumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Southerton</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thumma</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">RNA-Seq using two populations reveals genes and alleles controlling wood traits and growth in <em>Eucalyptus nitens</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e101104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De101104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Theissinger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falckenhayn</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blande</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toljamo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gutekunst</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title"><em>De novo</em> assembly and annotation of the freshwater crayfish <em>Astacus astacus</em> transcriptome.</span> <abbr class="cit-jnl-abbrev">Mar. Genomics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">7</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMar.%2BGenomics%26rft.volume%253D28%26rft.spage%253D7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.54" data-doi="10.1038/nbt.2450"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hendrickson</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sauvageau</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goff</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rinn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Differential analysis of gene regulation at transcript resolution with RNA-seq.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">46</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D31%26rft.spage%253D46%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2450%26rft_id%253Dinfo%253Apmid%252F23222703%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2450&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23222703&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-2" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.55" data-doi="10.1038/nature12031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zarowiecki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holroyd</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garciarrubio</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez-Flores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The genomes of four tapeworm species reveal adaptation to parasitism.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">496</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D496%26rft.spage%253D57%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12031%26rft_id%253Dinfo%253Apmid%252F23485966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23485966&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316993400024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.56" data-doi="10.1093/nar/gks596"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Untergasser</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cutcutache</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koressaar</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faircloth</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Primer3 – new capabilities and interfaces.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">e115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks596%26rft_id%253Dinfo%253Apmid%252F22730293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MC8xNS9lMTE1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzg3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van der Auwera</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carneiro</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Del Angel</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">From FastQ data to high-confidence variant calls: the Genome Analysis Toolkit best practices pipeline.</span> <abbr class="cit-jnl-abbrev">Curr. Protoc. Bioinformatics</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">11.10.1</span>–<span class="cit-lpage">11.10.33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BProtoc.%2BBioinformatics%26rft.volume%253D43%26rft.spage%253D11101%26rft_id%253Dinfo%253Apmid%252F25431634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25431634&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vergara</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lively</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jokela</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The geographic mosaic of sex and infection in lake populations of a New Zealand snail at multiple spatial scales.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">484</span>–<span class="cit-lpage">493</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D182%26rft.spage%253D484%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.59" data-doi="10.1093/bioinformatics/btg213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">H. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirsh</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Detecting putative orthologs.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1710</span>–<span class="cit-lpage">1711</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DWall%26rft.auinit1%253DD.%2BP.%26rft.volume%253D19%26rft.issue%253D13%26rft.spage%253D1710%26rft.epage%253D1711%26rft.atitle%253DDetecting%2Bputative%2Borthologs%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg213%26rft_id%253Dinfo%253Apmid%252F15593400%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMTkvMTMvMTcxMCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMy84NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Warnhoff</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kornfeld</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">New links between protein N-terminal acetylation, dauer diapause, and the insulin/IGF-1 signaling pathway in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Worm</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1023498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWorm%26rft.volume%253D4%26rft.spage%253De1023498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.61" data-doi="10.1093/gbe/evu131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Warren</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ciborowski</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casadei</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hazlerigg</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Extensive local gene duplication and functional divergence among paralogs in Atlantic salmon.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1790</span>–<span class="cit-lpage">1805</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevu131%26rft_id%253Dinfo%253Apmid%252F24951567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjg6IjYvNy8xNzkwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzg3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.62" data-doi="10.1016/0040-5809(75)90020-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watterson</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">On the number of segregating sites in genetical models without recombination.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">256</span>–<span class="cit-lpage">276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DWatterson%26rft.auinit1%253DG.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253D256%26rft.epage%253D276%26rft.atitle%253DOn%2Bthe%2Bnumber%2Bof%2Bsegregating%2Bsites%2Bin%2Bgenetical%2Bmodels%2Bwithout%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252875%252990020-9%26rft_id%253Dinfo%253Apmid%252F1145509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(75)90020-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1145509&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F871.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975W295600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.871.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winterbourn</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Larval trematode parasitising the New Zealand species of <em>Potamopyrgus</em> (Gastropoda: Hydrobiidae).</span> <abbr class="cit-jnl-abbrev">Mauri Ora</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">17</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMauri%2BOra%26rft.volume%253D2%26rft.spage%253D17%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/3/871.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/3/865" title="Genetic Dissection of a QTL Affecting Bone Geometry" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/3/881" title="Adapting Genotyping-by-Sequencing for Rice F2 Populations" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/3"><span>Volume 7 Issue 3, March 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/3.cover-source.jpg?itok=h8yj2GBo" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/3/871.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-FLGEGWyZF1i23dfIG2ay_4eVRACKB9gXtHzFrWknzoM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/3/871" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/3/871" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/3/871" class="active" data-icon-position="" data-hide-link-title="0">De Novo Transcriptome Characterization of a Sterilizing Trematode Parasite (Microphallus sp.) from Two Species of New Zealand Snails</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/483519" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ugGkyxzJUzuvEj-Rqw-4jendV2pIa3hd1fiT1kTtKLI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/3/871.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/3/871" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-vHuWySxTXqk5OSYT62rcTXZ0FjoJMVQd2QkWLmivmHc" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/3/871" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/3/871.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483519" id="node483519--2" data-pisa="ggg;7/3/871" data-pisa-master="ggg;g3.116.037275" data-apath="/ggg/7/3/871.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>De Novo</em> Transcriptome Characterization of a Sterilizing Trematode Parasite (<em>Microphallus</em> sp.) from Two Species of New Zealand Snails</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Laura Bankers</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-1543-8115" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Maurine Neiman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">871-880;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.037275 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483519--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Laura Bankers </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Iowa, Iowa City, Iowa 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laura%2BBankers%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bankers%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALaura%2BBankers%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:laura-rice@uiowa.edu" class="" data-icon-position="" data-hide-link-title="0">laura-rice@uiowa.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Maurine Neiman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Iowa, Iowa City, Iowa 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maurine%2BNeiman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neiman%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaurine%2BNeiman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1543-8115" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Maurine Neiman</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/3/871" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/3/871&amp;title=De%20Novo%20Transcriptome%20Characterization%20of%20a%20Sterilizing%20Trematode%20Parasite%20%28Microphallus%20sp.%29%20from%20Two%20Species%20of%20New%20Zealand%20Snails" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/3/871&amp;title=De%20Novo%20Transcriptome%20Characterization%20of%20a%20Sterilizing%20Trematode%20Parasite%20%28Microphallus%20sp.%29%20from%20Two%20Species%20of%20New%20Zealand%20Snails" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/3/871&amp;title=De%20Novo%20Transcriptome%20Characterization%20of%20a%20Sterilizing%20Trematode%20Parasite%20%28Microphallus%20sp.%29%20from%20Two%20Species%20of%20New%20Zealand%20Snails" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/3/871&amp;text=De%20Novo%20Transcriptome%20Characterization%20of%20a%20Sterilizing%20Trematode%20Parasite%20%28Microphallus%20sp.%29%20from%20Two%20Species%20of%20New%20Zealand%20Snails" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/3/871&amp;title=De%20Novo%20Transcriptome%20Characterization%20of%20a%20Sterilizing%20Trematode%20Parasite%20%28Microphallus%20sp.%29%20from%20Two%20Species%20of%20New%20Zealand%20Snails" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/3/871&amp;t=De%20Novo%20Transcriptome%20Characterization%20of%20a%20Sterilizing%20Trematode%20Parasite%20%28Microphallus%20sp.%29%20from%20Two%20Species%20of%20New%20Zealand%20Snails" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/3/871&amp;title=De%20Novo%20Transcriptome%20Characterization%20of%20a%20Sterilizing%20Trematode%20Parasite%20%28Microphallus%20sp.%29%20from%20Two%20Species%20of%20New%20Zealand%20Snails" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/3/871&amp;title=De%20Novo%20Transcriptome%20Characterization%20of%20a%20Sterilizing%20Trematode%20Parasite%20%28Microphallus%20sp.%29%20from%20Two%20Species%20of%20New%20Zealand%20Snails" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/483519/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483519" id="node483519--3" data-pisa="ggg;7/3/871" data-pisa-master="ggg;g3.116.037275" data-apath="/ggg/7/3/871.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>De Novo</em> Transcriptome Characterization of a Sterilizing Trematode Parasite (<em>Microphallus</em> sp.) from Two Species of New Zealand Snails</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Laura Bankers</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-1543-8115" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Maurine Neiman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">871-880;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.037275 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483519--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Laura Bankers </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Iowa, Iowa City, Iowa 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laura%2BBankers%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bankers%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALaura%2BBankers%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:laura-rice@uiowa.edu" class="" data-icon-position="" data-hide-link-title="0">laura-rice@uiowa.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Maurine Neiman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Iowa, Iowa City, Iowa 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maurine%2BNeiman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neiman%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaurine%2BNeiman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1543-8115" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Maurine Neiman</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=871&amp;rft.epage=880&amp;rft.atitle=De%20Novo%20Transcriptome%20Characterization%20of%20a%20Sterilizing%20Trematode%20Parasite%20%28Microphallus%20sp.%29%20from%20Two%20Species%20of%20New%20Zealand%20Snails&amp;rft.volume=7&amp;rft.issue=3&amp;rft.date=2017-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Bankers%2C+Laura&amp;rft.au=Neiman%2C+Maurine"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/483519/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/483519/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/483519/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/483519/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/483519/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/483519/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/483519/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/483519/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/483519/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/483519/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/483519/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/483519/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/3/871" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/3/871.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/3/871.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/3/871.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/3/871.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__DNYVmlR6RlVWGCviGjgljNup52yhnZXIncP3-_bmyQA__xJKgaB9J7btTrapa3xItcGD9KGApkh4nzPcIVGC2naU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
