<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/3/943" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/3/943.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/3/943.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/3/943.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/3/943/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/3/943.atom" />
<meta name="HW.pisa" content="ggg;7/3/943" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Reproduction and In-Depth Evaluation of Genome-Wide Association Studies and Genome-Wide Meta-analyses Using Summary Statistics" />
<meta name="DC.Identifier" content="10.1534/g3.116.038877" />
<meta name="DC.Date" content="2017-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Niu et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="In line with open-source genetics, we report a novel linear regression technique for genome-wide association studies (GWAS), called Open GWAS algoriTHm (OATH). When individual-level data are not available, OATH can not only completely reproduce reported results from an experimental model, but also recover underreported results from other alternative models with a different combination of nuisance parameters using naïve summary statistics (NSS). OATH can also reliably evaluate all reported results in-depth ( e.g. , p -value variance analysis), as demonstrated for 42 Arabidopsis phenotypes under three magnesium (Mg) conditions. In addition, OATH can be used for consortium-driven genome-wide association meta-analyses (GWAMA), and can greatly improve the flexibility of GWAMA. A prototype of OATH is available in the Genetic Analysis Repository (&lt;https://github.com/gc5k/GEAR&gt;)." />
<meta name="DC.Contributor" content="Yao-Fang Niu" />
<meta name="DC.Contributor" content="Chengyin Ye" />
<meta name="DC.Contributor" content="Ji He" />
<meta name="DC.Contributor" content="Fang Han" />
<meta name="DC.Contributor" content="Long-Biao Guo" />
<meta name="DC.Contributor" content="Hou-Feng Zheng" />
<meta name="DC.Contributor" content="Guo-Bo Chen" />
<meta name="article:published_time" content="2017-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Reproduction and In-Depth Evaluation of Genome-Wide Association Studies and Genome-Wide Meta-analyses Using Summary Statistics" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;In line with open-source genetics, we report a novel linear regression technique for genome-wide association studies (GWAS), called Open GWAS algoriTHm (OATH). When individual-level data are not available, OATH can not only completely reproduce reported results from an experimental model, but also recover underreported results from other alternative models with a different combination of nuisance parameters using naïve summary statistics (NSS). OATH can also reliably evaluate all reported results in-depth (&lt;i&gt;e.g.&lt;/i&gt;, &lt;i&gt;p&lt;/i&gt;-value variance analysis), as demonstrated for 42 &lt;i&gt;Arabidopsis&lt;/i&gt; phenotypes under three magnesium (Mg) conditions. In addition, OATH can be used for consortium-driven genome-wide association meta-analyses (GWAMA), and can greatly improve the flexibility of GWAMA. A prototype of OATH is available in the Genetic Analysis Repository (https://github.com/gc5k/GEAR).&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/03/01" />
<meta name="citation_mjid" content="ggg;7/3/943" />
<meta name="citation_id" content="7/3/943" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/3/943" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/3/943.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/3/943.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/3/943.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.038877" />
<meta name="citation_pmid" content="28122950" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="943" />
<meta name="citation_lastpage" content="952" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Yao-Fang Niu" />
<meta name="citation_author_institution" content="State Key Laboratory of Rice Biology, China National Rice Research Institute, Hangzhou, 310006 Zhejiang, China" />
<meta name="citation_author" content="Chengyin Ye" />
<meta name="citation_author_institution" content="Department of Health Management, College of Medicine, Hangzhou Normal University, Hangzhou, 310021 Zhejiang, China" />
<meta name="citation_author" content="Ji He" />
<meta name="citation_author_institution" content="Department of Neurology, Peking University Third Hospital, Beijing 100191, China" />
<meta name="citation_author" content="Fang Han" />
<meta name="citation_author_institution" content="Department of Pulmonary and Critical Care Medicine, Peking University People’s Hospital, Beijing 100044, China" />
<meta name="citation_author" content="Long-Biao Guo" />
<meta name="citation_author_institution" content="State Key Laboratory of Rice Biology, China National Rice Research Institute, Hangzhou, 310006 Zhejiang, China" />
<meta name="citation_author" content="Hou-Feng Zheng" />
<meta name="citation_author_institution" content="Institute of Aging Research, College of Medicine, Hangzhou Normal University, Hangzhou, 310021 Zhejiang, China" />
<meta name="citation_author_institution" content="The Affiliated Hospital of Hangzhou Normal University, Hangzhou, 310015 Zhejiang, China" />
<meta name="citation_author_email" content="hfzheng@hznu.edu.cn" />
<meta name="citation_author_email" content="chenguobo@gmail.com" />
<meta name="citation_author" content="Guo-Bo Chen" />
<meta name="citation_author_institution" content="Evergreen Landscape and Architecture Studio, Hangzhou, 310026 Zhejiang, China" />
<meta name="citation_author_email" content="hfzheng@hznu.edu.cn" />
<meta name="citation_author_email" content="chenguobo@gmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5475-8237" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=N. R. Apuya;citation_author=R. Yadegari;citation_author=R. L. Fischer;citation_author=J. H. Harada;citation_author=R. B. Goldberg;citation_title=RASPBERRY3 gene encodes a novel protein important for embryo development.;citation_volume=129;citation_year=2002;citation_pmid=12068112;citation_doi=10.1104/pp.004010" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=H. Aschard;citation_author=B. J. Vilhjalmsson;citation_author=A. D. Joshi;citation_author=A. L. Price;citation_author=P. Kraft;citation_title=Adjusting for heritable covariates can bias effect estimates in genome-wide association studies.;citation_pages=329-339;citation_volume=96;citation_year=2015;citation_pmid=25640676;citation_doi=10.1016/j.ajhg.2014.12.021" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=G.-B. Chen;citation_title=Estimating heritability of complex traits from genome-wide association studies using IBS-based Haseman–Elston regression.;citation_pages=107;citation_volume=5;citation_year=2014;citation_pmid=24817879" />
<meta name="citation_reference" content="citation_journal_title=Eur. J. Hum. Genet.;citation_author=G.-B. Chen;citation_title=On the reconciliation of missing heritability for GWAS data.;citation_pages=1810-1816;citation_volume=24;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Eur. J. Hum. Genet.;citation_author=G.-B. Chen;citation_author=S. H. Lee;citation_author=M. R. Robinson;citation_author=M. Trzaskowski;citation_author=Z. Zhu;citation_title=Across-cohort QC analyses of GWAS summary statistics from complex traits.;citation_pages=137-146;citation_volume=25;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=F. R. Day;citation_author=P. R. Loh;citation_author=R. A. Scott;citation_author=K. K. Ong;citation_author=J. R. B. Perry;citation_title=A robust example of collider bias in a genetic association study.;citation_pages=392-393;citation_volume=98;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=G. de los Campos;citation_author=D. Sorensen;citation_author=D. Gianola;citation_title=Genomic heritability: what is it?;citation_pages=e1005048;citation_volume=11;citation_year=2015;citation_pmid=25942577;citation_doi=10.1371/journal.pgen.1005048" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=B. Devlin;citation_author=K. Roeder;citation_title=Genomic control for association studies.;citation_pages=997-1004;citation_volume=55;citation_year=1999;citation_issue=4;citation_pmid=11315092;citation_doi=10.1111/j.0006-341X.1999.00997.x" />
<meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond.;citation_author=R. A. Fisher;citation_title=On the mathematical foundations of theoretical statistics.;citation_pages=309-368;citation_volume=222;citation_year=1921" />
<meta name="citation_reference" content="citation_journal_title=J. Minist. Agric. Gt. Britain;citation_author=R. A. Fisher;citation_title=The arrangement of field experiments.;citation_pages=503-513;citation_volume=33;citation_year=1926" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=F. Han;citation_author=J. Faraco;citation_author=X. S. Dong;citation_author=H. M. Ollila;citation_author=L. Lin;citation_title=Genome wide analysis of narcolepsy in China implicates novel immune loci and reveals changes in association prior to vs. after the 2009 H1N1 influenza pandemic.;citation_pages=e1003880;citation_volume=9;citation_year=2013;citation_pmid=24204295;citation_doi=10.1371/journal.pgen.1003880" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J.-W. Han;citation_author=H.-F. Zheng;citation_author=Y. Cui;citation_author=L.-D. Sun;citation_author=D.-Q. Ye;citation_title=Genome-wide association study in a Chinese Han population identifies nine new susceptibility loci for systemic lupus erythematosus.;citation_pages=1234-1237;citation_volume=41;citation_year=2009;citation_issue=11;citation_pmid=19838193;citation_doi=10.1038/ng.472" />
<meta name="citation_reference" content="citation_journal_title=Funct. Plant Biol.;citation_author=Y. Hariadi;citation_author=S. Shabala;citation_title=Screening broad beans (Vicia faba) for magnesium deficiency. II. Photosynthetic performance and leaf bioelectrical responses.;citation_pages=539-539;citation_volume=31;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. W. Horton;citation_author=A. M. Hancock;citation_author=Y. S. Huang;citation_author=C. Toomajian;citation_author=S. Atwell;citation_title=Genome-wide patterns of genetic variation in worldwide Arabidopsis thaliana accessions from the RegMap panel.;citation_pages=212-216;citation_volume=44;citation_year=2012;citation_issue=2;citation_pmid=22231484;citation_doi=10.1038/ng.1042" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. Korte;citation_author=B. J. Vilhjálmsson;citation_author=V. Segura;citation_author=A. Platt;citation_author=Q. Long;citation_title=A mixed-model approach for genome-wide association studies of correlated traits in structured populations.;citation_pages=1066-1071;citation_volume=44;citation_year=2012;citation_issue=9;citation_pmid=22902788;citation_doi=10.1038/ng.2376" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. Mayer;citation_author=C. Schüller;citation_author=R. Wambutt;citation_author=G. Murphy;citation_author=G. Volckaert;citation_title=Sequence and analysis of chromosome 4 of the plant Arabidopsis thaliana.;citation_pages=769-777;citation_volume=402;citation_year=1999;citation_issue=6763;citation_pmid=10617198;citation_doi=10.1038/47134" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Tabata;citation_author=T. Kaneko;citation_author=Y. Nakamura;citation_author=H. Kotani;citation_author=T. Kato;citation_title=Sequence and analysis of chromosome 5 of the plant Arabidopsis thaliana.;citation_pages=823-826;citation_volume=408;citation_year=2000;citation_issue=6814;citation_pmid=11130714;citation_doi=10.1038/35048507" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Reproduction and In-Depth Evaluation of Genome-Wide Association Studies and Genome-Wide Meta-analyses Using Summary Statistics" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3.cover-source.jpg" />
<meta name="twitter:description" content="In line with open-source genetics, we report a novel linear regression technique for genome-wide association studies (GWAS), called Open GWAS algoriTHm (OATH). When individual-level data are not available, OATH can not only completely reproduce reported results from an experimental model, but also recover underreported results from other alternative models with a different combination of nuisance parameters using naïve summary statistics (NSS). OATH can also reliably evaluate all reported results in-depth ( e.g. , p -value variance analysis), as demonstrated for 42 Arabidopsis phenotypes under three magnesium (Mg) conditions. In addition, OATH can be used for consortium-driven genome-wide association meta-analyses (GWAMA), and can greatly improve the flexibility of GWAMA. A prototype of OATH is available in the Genetic Analysis Repository (&lt;https://github.com/gc5k/GEAR&gt;)." />
<meta name="og-title" property="og:title" content="Reproduction and In-Depth Evaluation of Genome-Wide Association Studies and Genome-Wide Meta-analyses Using Summary Statistics" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/3/943" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="In line with open-source genetics, we report a novel linear regression technique for genome-wide association studies (GWAS), called Open GWAS algoriTHm (OATH). When individual-level data are not available, OATH can not only completely reproduce reported results from an experimental model, but also recover underreported results from other alternative models with a different combination of nuisance parameters using naïve summary statistics (NSS). OATH can also reliably evaluate all reported results in-depth ( e.g. , p -value variance analysis), as demonstrated for 42 Arabidopsis phenotypes under three magnesium (Mg) conditions. In addition, OATH can be used for consortium-driven genome-wide association meta-analyses (GWAMA), and can greatly improve the flexibility of GWAMA. A prototype of OATH is available in the Genetic Analysis Repository (&lt;https://github.com/gc5k/GEAR&gt;)." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3.cover-source.jpg" />
    <title>Reproduction and In-Depth Evaluation of Genome-Wide Association Studies and Genome-Wide Meta-analyses Using Summary Statistics | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"nXKqBS2OgqOWN15wdKmSd5SjmwcnQHn8Q0UKV6gr4Ds","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"483528","apath":"\/ggg\/7\/3\/943.atom","pisa":"ggg;7\/3\/943","ac":{"\/ggg\/7\/3\/943.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/3\/943.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/3\/943"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/3\/943"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/3\/943":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022hdpeY2WdhzZXHtpkUgqZTZVzhQwvTUXZQkhQt+6\\\\\\\/sTXOXzuW+kLC1Wn0TmcF6IC2aNxlm9Z1dC1s\\\\\\\/kDkkSI9N\\\\\\\/Tv3LCnfC5Cs\\\\\\\/EVRGf+k8a\\\\\\\/TfLH7+6IA865HhX5ompFypdaaoMEeAGjcK8+mglGsiFUBb9H91Uc+FYRiTOvF0ZCgDzzfRYG05Uo1Z2k2MoYKPu+hxiFZd1IENyWEreq3x\\\\\\\/79uh1RGms7OkFWUVDhIpsADsETEEnaDEShXLnqp0ZVdq5+UaI7v9CTadICXPlxGC7iorMIL0k4OwxD6L7z1owFXPhBlqQAoQoMaARyZbLWlOcSD8cDi1NQT\\\\\\\/7IIsk0xpmbXaFjNHSm7KfEP88LCbPgn4mn4boPZNGaNacpZ4kPKd1pZPVcua+oBV1AqrBeuGfiNydmMNcv4Np2s3u4HQL4ZV8rRxYSYZjw9aEBdTvxTrInERhFpuN3+ZcB\\\\\\\/uUPmU0mjZ6eRMpXWIep8bUSymV1SEcLTwZt7MOXmT+Psu\\\\\\\/ia6rezn9pfHuUKNt7GxQgAYcdQRNI8sQxQhzNl7pnx\\\\\\\/kzxyR\\\\\\\/Z41H7s2iOR\\\\\\\/lX0sum34cJ2I\\\\\\\/tlAuvz2diIgEm0P8CoJq\\\\\\\/znpryaUlngwRAWlXVgiSsoWecOAER46M3pOlp0EPVVCQvuYh\\\\\\\/hmgxH0qdv+yWVlw39oDabrbEIRlbhqc5tdNuVz9kij6wqq81TBqRMxz1Qe4ZrakYpZaJSeU5um6T8\\\\\\\/L1ji53LGM7SUTzgzfB2SDvRE37TUVERlvx7azxxNrLGS0sca3tbYMT52ZHbAYuqgaKV8otPVD8RGYn0zvLDNnfn9KOlJkfSzYKDJ0ROKHlrkz0juwp1oVov8pUVoddt2uxqZSlJbikxjcMdDzf4rc+K31AXNcODeTBT28bX1n3k89s\\\\\\\/e7koX5hZ1at5NTb+nPFvZT454qhdGHWg3qUnVS0zYQmgZw+NnrW8BMaV21FypHJ8v2cx3j0356ieNgOU6dq5Q6+i\\\\\\\/ID8pr27VXWw6RPEXOVOtgQ4x4JcK6j3hNmFs6a6+oeYCSMK5+3Lp251Ti0nzxj0BwjP4\\\\\\\/RvsAWcs62DIcnUlhpv5yOoH8saIdarmONZQPuXdCcu3zfBxnPjwa\\\\\\\/ccI2FO0yB0xiobM6ajYmwNCoNldOhchfEzRJrf7Ttl7wXZKMuXiYqA1mfTj2vmfYjH+Ykkfc9EAyhlwKV6c0bItdTjfKGHYiTbSzsc8yCO3OAP9NzYKkvgbVVxCZkmpXw\\\\\\\/6WcZTvxAKZqI7yzytNSeGBEqmuk+Y90\\\u0022,\\\u0022iv\\\u0022:\\\u0022EZZ5cdgqlCRQCSaQaxWnew==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226f07b034fc603b792c81e19c0ccca2882d7bb376a4bf3c77e3940a5bd650f79b\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022BlIvQjBGUD1ITf5ZDWdo3NIVLYUJiXq34mxbtyD9wTqH0PvQcKgCm0z4ako8loHzDkVdiN\\\\\\\/pn3oeAgvoBrwTwwV36FpcBQFmOZiKaM6uwoyW19FnsnIkFEYfqSAsoLQgmU5L5rXrJeDug6y8yURfsdrmn1QXmdYM0UT7TTV4UvzyAdyGo7rL9+0ReWwzncX1LKea1G6mM51\\\\\\\/R7J6U\\\\\\\/cNisG3c9qtAW0ooJzox5YjpaDOvUdvuHkAf5dRGwkcK6w0nR1h\\\\\\\/AK+wWCxM021GfYp3hccAwdVoziC6xnd8a8e8xtzAhu4esvwZYINndedirO8Bldg9eFJBFgz7JXbxwT6eYe8n4kz9caaVz9BXwFeCK0vJLtRhHySiFvkaHK\\\\\\\/8kaEotg8D5vVwfzRiDafZClNcPOwFeXgwMW4qYH0llcPNs54T1KEkgqRuGk0VzrXVg+VGfsVPU7gHikWbe+2g\\\\\\\/xktl8d1SPXxVTtdSJnwTDLnRK0GGdDjWByxdwafXdhPOVBmGwme2SkqCL0srUH6+QYLVZQ87LHMDu3wuPwgkmyJ9I0\\\\\\\/tuiRBBS8PP71osUzu\\\\\\\/g5BhkSuNeEh9CulwZZpJOXoLMSDDjWFOMiA5jGC5aq+iVfTSBcj+nh0iXpVQh0kWEh2o3IAjw7vOomPzYY4WdT3pYlbTKiL8rYwEScRDaYeChZl+mfvYKljKtuLeF71W3Brr5EKVxlZAqTzrHB8tIzKOKesYNiMygkP+F2P8dRqzjOVr1S9kjoCDEAMMFpVJP5pSvTBjl1S1eM1tgiY1aPB1wnXnhhsmfY71SQtKfd8GfmRXJfotYVUmyOWNvfyuXSVsQqHlaEyARJ\\\\\\\/FH2j+z9sMDcLgzsMFiTzhyaR3uiE\\\\\\\/pFq6CV2wxG4t4Cydm3E1Mql8g13wuXEektrHSgutlRpI4eIeSpkemiao2Nd9Uu2mIig8VMhCL\\\\\\\/ZMO7qh1BF8qSn7ow2TvCIklfNglsuaEyLtCDRiY8B4KqbkEMnKp4xYJ7IxpUStVZJECl4gAKWbHz3q5A5NzGeE6zCsH9u69I4gZ75mRTFCDhG+c2efw\\\\\\\/0wkccuSf2YuVtW75kHFBvgRQPOns1izagRbFNbm9dZ6tBL2nlgA3kbkVJFsBITBfws9k27vlb6L6+bu0HSuYvNv0+MZbDOzLWeUYd+ActIyMV59efg8kUYI6rvooZESVWGIeYJC\\\\\\\/qc9kNAT9J2sORS+deIwo0\\\\\\\/C5RoSUI4pXN4Olw4Wxsyodr8zhg\\\\\\\/l+tQXUXitvdEgWzzTjOei7FhPMEq+zjdV8wR+duBt45TiYAT36uW0GjzZfs6ql4HL8uLL5d9rnxAgJXzJoPYWBMV5IycigD4OjQmHkdq2pQXTU8c9Oi+KdLG8s4gi6WNk26H4xSTBiI+RlsW+f3q2j5EPuqIFepHh2rVpg6hDB6K3QHtq54x29b\\\\\\\/JVG0Um\\\\\\\/7jXOl8qo5spO8lyl0CFyGgdlRnPdrG59f1kAM33V+NrdFjefjk36GD03typ33Wo\\\\\\\/oNL6Hq3Ig4rgvR+IUavuS0jxrwf7PIBMCYQpwPRagfzkMclHnV4junQHickPJ584k\\\\\\\/gMaXK0PYEvv1GH8ZHceBeXjMcQflAsc1hjqONhu4HkX0PayYxmM3Bt\\\\\\\/xGURHJIrdFd3FAyRhESooogauDZjdytVeBX6Z+CwKYZpdxf6PscikDyiZ7hiGruYOA7OgR9IryVNwaTPvJnjun5WIumr4QlHfQWQlzfoNFLbzOLemV9MO7LUM5Wri6hMROyHtW1WBkVaYI2BrcRTh1y2CzHIVS+aCQpdJJxVBlZbtKN0iSuhCdgKF6QPlPB\\\\\\\/0juCIGtRkZpPRJpzWKMm50TaDOgWHD\\\\\\\/\\\\\\\/CVku04\\\\\\\/\\\\\\\/etbFkHf+fgPWeo+bL5BJ09BFAb9mc\\\\\\\/e2yzf5JeZCzamzaFf\\\\\\\/PTn\\\\\\\/1CSWSK6XZC9NYczrl2GhDf+e0xP0v12pLkOEWJPx8IbhE04cIo44IkMl\\\\\\\/WB34Vm5NcV1I73+F\\\\\\\/JTdg9uqyppUPN2VvkwBqbY7aYETxPLSQTgUmdLpyRGoT0dPAl2B578MpQYoaqD5YkOyYZRC9uHhhlE7pIcy6gGLRiMLA9pcW8wNZf\\\\\\\/sNsi5SGQUFiSOPOsWpme1Va8NIaDOF2NcP2kckoZOZ7v4hmY2xSjIpjcfMXwzB0TlnrSXa7\\\\\\\/gKNW3uhsy9K65OcgiYdcDPyiGQUFr5i457RWwMnR4PYnm63kMx5X0nk1ie6POqLollpuOiKvSUAlXJkqkup0Ujxnv3y9piB5kpSQeIP0+2JZG6uYY81y1ryk+MjzLqNDFZcuF5pjcgopQTEI6xasn7aAuUt5WErppgp4s2A0ftf4CYSryuUt37ZlKgAGhoUuCxxKxZd64L8usQ\\\\\\\/FKu+HbS4YGKxidw97sBiAfO+Ix5Kf+pyKIJQemqkujh8+Wz2kPmXQ1LdadTfcQy+l5qkOzX6fuyei7y+PSpb\\\\\\\/jdpazGGSQ6RvvKOn0Dzw0BBobMNnEhGeBXtQ7hf0VtVfzTuM6qjSWGSfvYUxzN28bEm23nCMasdM7a6V7+aTukQmlDmedDSEfS5kUQn9Dby08qEpI61b\\\\\\\/RKBaMc66GcIWqWCgbYiIJVnuoh6q5ehCRbfKj5pxKQ68RDydFy3IGA1lYzH0Jnq9Zz9FxvKkV5Nj3ozZ1xDsD9jQjt1wPkyyf0tMtgKhiCkn8oCy0kfVPko0u+OM\\\\\\\/ceNtx7DbhSGJS5J8ft8b2YSXESQIfJobOJewMbDtZV6V9W6MADmYkV70Dz4FzH+Nut5zG2B58GGvpy0junNaEgpPT\\\\\\\/9vwa6neCP7CaCHVeic3yb1zZo0gcKUx0k\\\\\\\/euS5J3ZGr+4Z8X3YcI8\\\\\\\/otwXgycQw46zjQlNw3+G3MHXzJaBKaf4zm829SGKE5urY2d\\\\\\\/\\\\\\\/hakjJnSl3CWGGSPUT\\\\\\\/CS1SDngI3JL33Pc7uXLSMHKNCfy0WM8jXvevWVE3HHe0boyHhNyoFFdc7tWAHwR6yQj69ubGp+sEvOvdt6impmDuKge29CWAEnSpjhQhQUAXl\\\\\\\/cIVQcaqCgIVylAi+GjPjtsNw63A2HqdjQEOW1nexKOEXp7jsSd7jzBuz72EO5OP9GZ9TAjFGdwUupX5gWVk8Ea3AnRrDatIkXiMAPK1ciCoKJVqqNVOMRjH4To8FZHVmxU90H+KVyoTovf+IPjI4FOWA31tfWwbrsnF5cGSu7IuCn7xp9tdZQbo\\\\\\\/vojFqK7KizdhSBChxdLmVQb7BK7smu+80N7POZIRVedDBBEfrSQ7uY5cYC6xqlJZjddFAKOj3exmvrgjUfe9m6u9A1+n66eegIHyXyUhlMu5rRzqgOApoPcaGaqCeafjhebuz12yS3HOdX2oGzyfyXq8NbjX3xRPk6iu1AVAlN5k5p9a5QaKqfA7R2P+0\\\\\\\/H7n8RhCsBQ4xOBhd+DwOXIvVQnZ+tjdtZ8vA09QqrXizrzpRln3x+tmAcORZQpZsdtUBDyYpC2LTeti1Urh\\\\\\\/cACuBHDJyd\\\\\\\/1nETOj\\\\\\\/XbZHpIV6RAtUpVdLOgZNiEhSgiU6e60qkM\\\\\\\/n9NldR0D0Bw8TxlMOvhdSztY3TdXYUI8sfUGe1w4p1BH6tsWnDSP0XCp4KV5h\\\\\\\/6nNaqsyuNeHZY42jaWTaGYD3Ut6Il3po6JztNQQt6IJTamFJ8cGqzao43gUaw6dI5k4qNMCXCEidDD04ccQwPO566vl7f2\\\\\\\/rmbyPcNs1w7\\\\\\\/o\\\\\\\/YwKqUP+UnJ1dWnlGq10GyX+79fR2lnlRZdd5wYeQsZXXJ3CMrsNafWIzBCfYJRp\\\\\\\/ZOHG2Br0w+1GCuYD3dDSgXjkeVVZDoNLLO7ig+BB6VJSHxEwDsm8NNwSGL1RdvChJmDPubqZ5CkgU7VHA2EtNIlA\\\\\\\/U9FqZrhn\\\\\\\/3Idasl+6D3sYT45OFQWeMvRElSpiUtT8MQqZLT30keda6mp727AcwV7JsjKN2Huhyf1T6pA4RiPKUtwrcuYuURudMEzrGC8205ZT7aSUwTjISvtNPU1JZCLyzAjdv+svq8DZrigCyhmAs00h7xL5Z6qzi37JTNxwK14XAqlvWX6DRv1F3zC7uOHumIo5pESEcMSvKosgfnNAtMH4c5QMNTBldx\\\\\\\/xQJbZVGKuzL3dGx8bZeBMr97AZd1s3OvKCz4LSJFgKDY6bE1NiBzz5sv9nZVfIPMb2kOlt\\\\\\\/jMoE9urOKy97szJSRDh5834rPC8pSBCN1Ipwxi66DhgUEKmsJs4T5kldbRoVwnNCgGdUMl52LMu9TUYneoCgmTP3Fh9m9TQwiBsyP+LopQoXXEn1zo9Ab5vxNfKUh4TRFP+RFY01LuRPpbi4PBk2dWArOxPXlfH\\\\\\\/4rzotHoOEgn4WCuWvThlp9w7ef99yMxAlFWL9dgnxjSUGXcThHzfVRE2D37eqIqQJoAI7\\\\\\\/24S0kzO8Iq0Pj9ELmKT0lhTwsPHh58c\\\\\\\/3mRYlsRSIs0oX0GNSvlTxecgDjl6q23eCtnZPYZL8DDWLcfLPK1VeKvVYconmOKwH5WHyIOEg1g8ojigh1qi5PAH5v\\\\\\\/Kxmxxg7nQ2ECpIqXy3pfkmTWWDyFecHs7BgDB8nxolFJcSCRNQvmybFWfbJ6pd5BTDwmvD+DbKQ+M2ueHvo35SivgJt8TAjn\\\\\\\/QUxs1vlyYDDJzoSbYP3Gfs6PNG9IWP67lDDoyFzthSL0ccDeO39G5KuSsEavxa\\\\\\\/L23kEQz6\\\\\\\/LY13a+R1Ze9A3WUBBno\\\\\\\/CWneoMtSKdGsBPZFsl0uihPnU91JeDTgwGg0mFSO7dnda2Fv\\\\\\\/ENuNMh758DDzq67rX\\\\\\\/vw8VojYkPWEiGQMGls8QnihOhDdys1keZAboXbwFg4T0+1yhxXTReZzqKveZatDdY8lvNwx44SEsNeKrCU2JuITQcx5N0KdnwIwHcS6GZ1m15oQfyusNrZiIt8J\\\\\\\/TWaLyAmuJd0kS26SBa5+isfGJ1cxfP+yYlZlUEAUF7KsV5KUJfLJbYf\\\\\\\/fhjOBBa9Za\\\\\\\/V6GdAWPR7WR8O\\\\\\\/mUx3JdMhi6wrKoob+Luvd+r3TjEV1EODJDswYfytYs02vhXXsOj1nD4NkLYaw+hmh01yBOjWsUl6UldqCfE0a89sATuO3\\\\\\\/Ckz3yoLB45rTAROPRXMasMXbnB8fvRUJVHnKf9+l6jKy8JLGKzlxsJlY7+5QI+tWLRzklT4aiEuxV1FY8eI00e+4vqBhfjwaBRb55YNsVglpqhq3JX\\\\\\\/H41Iw74KEqWBBrPpytb9HTjxgGTSlrY\\\\\\\/+kP58aLjZL0Pt+L8zCN1Me5BfMWyez0tBwpMrvd4dLv8vlwQkwLD8ZP\\\\\\\/cIW6sztbDEnYEEN2p81Qkhyh6sbNhb\\\\\\\/PaO4EkOX6rCNYkPIl4eLA06xMHGnCxkQc6csm39Vk4DK1P0murY1DkrT34ngkBLc1ZE+sBK2ZfwFfUQaVgn2DoClgyGV+4M4RO5+Lw+pAct3jjKLMd6BTLz\\\\\\\/iplCGyIONHvHY49ZMtL8DgX8OLOHVHkH2LDdn8MFeS4KqXL1OVLEYzvqzpr3vciZc\\\\\\\/3bQmmgPl6bgtcB2e6\\\\\\\/DSlYC5pvc\\\\\\\/mQXTWlN9Ds8W6veK7y3GP4CrdvrmPIbg0LN75K1paBCdv6sMy5MfEgWIphdwbHjN4Xm8D3JzSCDBUXXKTd7fuJIPs7aJstFrjA+vSRGjEOJf8bLsXfyrDThXgwLUccNlZKPxJS5juv2oBU4RCoN3St22J4DIHQXIsG0rkHo9A3jXOFozlCB8gl\\\\\\\/BObGurZCwRcMf2Pks97ITSjbRBTFf1+S6HKODtHQpHSPje\\\\\\\/vBfI0c3JHMlJ\\\\\\\/ybXASRKFdkkqYsl9LS65NNUQc1vf\\\\\\\/BohP\\\\\\\/HCTRx8OZY80r0wyDB3EVUQH\\\\\\\/MqjuL4r7uTgXnvZVgMbxNBGbcGuRA3wUAlJg2FSBOTuruRKo8RmLHi5azVEO0Y+EjUkB5niZIV\\\\\\\/wi86+hk\\\\\\\/cuYbCOIlaftcavfB3m1wZSVJp+q0aVAkYLHqE2xAdCZ7toO9mtr8YLf+rdfonwMgndQvkxPWFeHc4wd0cvf\\\\\\\/\\\\\\\/qChbjyhPlvVakfiwquVrQId\\\\\\\/SJ0CcK099aimz1UG1xdPaUURBoK64Zg8tnsZ4Qy75Jp5n3nTPtikVnNE4VxsjVQpqV7GeGia0Qu8NduXm+v9gxWcepCXMvkxWdHnVNddzWq1O+dCnczYXCgysYQ1Uc4BZDO1N8WdFujv+5mCxX\\\\\\\/O6NSFN4M3UuNB6qNU8ijlEQyQShUUnoYCs1mIOdLyudB3hXP6EUPsXPKqYusfVQHc+GjzGn+kwej9J34jfF65wnXyZRS\\\\\\\/iGXbBZnVu\\\\\\\/OHgyFAPRRjM63KSAfdMFN\\\\\\\/hcKIue6zZSiBVeDxPgmE3sMI55gXyxGMFgv4\\\\\\\/c6XkWGZcB2Mh5I+h4WtDTj7WNxbDcL2CTXFTEmguMM\\\\\\\/YfqUNrduoaBPu+w2Hn1RVCj\\\\\\\/K4c8fencmAHPEDWsKK4TFNWGb5DmdnG3Mxx4eTPZjd6OLe1jMxP1CR\\\\\\\/\\\\\\\/cYRgPIK+zPicPpxPUBcL6vIBYFs3En3VoAhXL4eGR4ixybD0gHiDnx55c6HB257eIew8dHOkKoFAUloemEOBzNC30nQLE2eZrt2z0MobM\\\\\\\/lc8HOkHyaDWViorXj7fqSmTZqlDVjqZLCA85P9JCjS3kC0mlkv4i6jCRwDyxsAUp1qbgYXTqpTpdfpWA0bXx57VpNR2gtGUHGGWf7lyJSYEiesEgAR5Vm46C4fGhhbAst+A9+v+wcBS0IbK06htBN15wtCg3kMyxwARFCgExJqGe4Z5zY5+vPqglv+kzpK9s9Mo2Dpp3eXSoUPniwp6O6peDI5gPtlLYXsPKrLcdTwoiSCJ8Ey1gIMKBBRwpgqmNSHBEs4FudlFEGwoYuQWJnAuBVWlx0iwKzPqqDXLRCCWkoXTkoc3FvxtiNxkqtXUNuKr4Ms3IS2uAk3gPN8KzgpA5K9rtzCPM5O3zf+0TJwDQmcWXpbGRbuILXPjtVr3cSZGPIH0jKLKH2xmD4k2Wm9H8U4aunrnpFsHi4KMqckpB\\\\\\\/V8ObDspij1MPdzDBsQGX5Mu3O1pOi8vkQo3gq6bnSweKImS6o95y1SI\\\\\\\/+TIFPg0rUnAwbKPO7n9dS0N0TAIqHj7+QYGV7dW5iPLkUwLpd5DSGd8YzNpXAu6p2vf7UnFD0B\\\\\\\/7HKuqIaiHy+8NuWD0pmBThTODxpvLfcFdg5WibFAAxRHPCPRAy+bf3yC+5s6N1gfdNss\\\\\\\/b4nl7\\\\\\\/6F54FcqdiuEqrL8bWWhIQyHoALW5N\\\\\\\/M6vV5HwvrOse0NIZGQUg\\\\\\\/IluyuT0xqfNcALOHE\\\\\\\/oCEsnLGRVxhdPqGWM2NndJ2GG\\\\\\\/HIhyCO0b5nV68AkualWokFKjr2+4Js5fvmvlwS3rkYnpVGS+vKFScaFnLjddq\\\\\\\/OBtevnBJ\\\\\\\/lLBurg\\\\\\\/LSguKWKdfGoOZuOfXdh\\\\\\\/aKpvA6I2FtDHzB4pVh6\\\\\\\/pnAUZQ9AR3CqalU3F\\\\\\\/J6LldZ5Of0u6j3ka2ZcakEtTlHymjGv9XxUzwUJ+4aK776hdj0K3i2HRj3\\\\\\\/R9EkO9qEI5vvkV+XlhJ15KmThdbB\\\\\\\/+HmeffaPE65PhkYRN75vtki+1oellqiZJ9tpfR42cDvZx027Uc4KoMHZ+kHQIe4HFffvu19El9TgFHbEsMK8L5L21qY2KLN3XLhV6BPxiT23bSCZY5VvusSx9DNv8WpyoOq9RQY3XwuXREF5Jlx5QxAqmTJAveJrKaXFI9jCyGDfak92wVGJee\\\\\\\/NllKi0\\\\\\\/tzqjuspdRfgJbotD9nhXe8zkM6aDujkdFA0AoNZPIM2AcyNU8UgKVujVzd0lHb9pa4y0b6xiyz7S2vciTOOs2GOivVHJkWvtYc\\\\\\\/GUMdt9aS7o59fx2w3d1kYLnlNpCbgZKyeIV17714W1j6Ty4GjJ5iCZBFiZLbO5+z4HeBzUUZQAW1OjOORFzel45JZUZuniVZUGfaggaRBsmENOjzKNFrHM9f\\\\\\\/cWo8G41Jk\\\\\\\/l2VSFglFdsrIcdsws+xDOu\\\\\\\/I8IZ46PYt7tssAEvAiKvJ6Fkylet9KTcCxkni6CRno8Bv4D+aY+dUIuxUytYQnyaj4ZwPkgpkuoaqxn3dGHAs6k1sZyk5ctWQXVG\\\\\\\/Y0nZVqilwX3\\\\\\\/RI2TDh+bkV8PiB59o0vRFn0d1bpazknRitc5UjLswOzza+U2yfPr6oj6CFKdImHoQbegV5Ec5qBwE9fwViFh0pUAaER0xsrlqhPHvfMj3eFELwFA4S4DeKQs1f1i6IwE3SYtQCzAHiKFS51Z1dH+EyjfC0mwRNEKRX9mx+AmqVqJW0Cq1SgM0y6FusViDzlQshsBGVL7VY0tZ1R8kXmlxIL4L8NN2Pe0R+0D3HAVHd2M+lp6kdo\\\\\\\/6aLJ1wrWAG5ITv1HaucrJ+Wr8x54\\\\\\\/ho3g6IIYp2W19tI3lbOSprhyUCsmvC4zMmxVjlcoMA6zGtqeKl63zdkbtGHSx279N9CPhogYfySR8Up0tgNID4STL\\\\\\\/pqHfvPA7GoWnVYBnLmrNk4Wjd9kQeWeCkzN\\\\\\\/ZL8fY5Jc7tpLnFPRSFQDgUXvXpjbnXzWtIpm1In+8CCqUPc9n\\\\\\\/rYHqpUVDtT6K\\\\\\\/HAaARue482lDLaik0gzlkqiSOM0KrM4WkQ1mS2iX6LaF3S46nTJgQfz\\\\\\\/GM1npxDA\\\\\\\/F4iJiR100zRM2oOb3FVodcCwWQK1Qm3InP0HkM0xlB2Oj9btKm3OrwioqjpP1b5VYx4BPvdfo42WRTnTa56uOe6hK\\\\\\\/cGCrJxzYWt6HxEuTjQsxuvTciwkOYeldM7lD7Mli1WC6VyaE022iMoemU1Me1H552+DZ8yokRa7Bt6f7XrxX180FtULdjR+bTFJ6Dzy\\\\\\\/zFxylxRHQoy\\\\\\\/YKa0a5k59AcxpTK6nHyrz\\\\\\\/xgCjnR3l3PIzvpRGbt7yAaqvEZ5nRVpF6HlHYm\\\\\\\/JnkzQHAIkOL7K8+LsqKOaJ9TF+8TXL6hUVwpfCCwAXvLRVbz8Vm+YJCfc8uBDS0c05ob1OOFOqbdWMFl0HaUWB1YNReoMBb9kfZS98J8iXT6umEuCj9W1+x0a3m5afLPk47QZxDs7vvtYzIOPxgyCg6ycnA4FjC0NaQ6IwwPZkvxMB1szASOw0Bs8w6P2Urx5fcgrqyh6sIgB14bUKeJNNNyTDCToat4HS7AJ6I+0rIXoqOjWkGCtY2IQzyqqJUrg8xKIVHFOCSSKmmaFbOS0ugK0zcxrwgNjXboPKRh8Il2eViahHpZ+icaojqhOK4TbgWzVdL04FivBYHI3jE2g8ClZEY+scIReLnacLWlSE4gBhOU0d40z0ZgGG8Dw01i0ReofxBX26Zz6InB4kMiZu6jy38LOPR8KOArABusqClvSxt8zAcrdxNdTjfTDGyz2R4mGz4ZuLPgVAyoB+KC4cqn2i+50EQOuDtEvUYn5CHte3wmOFVnGCtvVYkUV07IfpkK+bwOge9yQxL3QHozLDgGTJv2CsPiCu+W3hs1iabXBsgomZNlHZiyJ7l4tpLwFEMkfoyfwJ28cWwtFg3OY1a7XrTD5tJqZfKlDhkqUYUMkoxMLqQeFDldhkEnuybsvc\\\\\\\/X3bcB8xe3N1dtFqLZ5Efb0D1HicKs24rVkxJQl2b1KvUr\\\\\\\/V2Mf41ykuipnWY5v2NYjypusoawqwEsUqyb1H3Gmfj89UX0tCxjjgxjo2cqM1+DiW3dc7k8IIhlYInL0pD+7dJTpys+4a3pOwOqTNemWH\\\\\\\/c1XuJ0mpkwTj8c7rRYMmgMydU7EJdH62dPEDAB0jKnQAr3jHbZSZBasS3HtZyOrvSUyl3dUWm5+VKAjtYKerpBxAGXKtT3ucfxehRMVUtKhQQnHS7nOTxb3fxqazNf3WUcJadE9APZTuGbRojGCZJprFfWAYFVxv8tI2JPPdz5Z6z7aRsecQ2RZ7Lfcr+ZsJaICy7DiJbjKhWbksZP\\\\\\\/SYDvb+CmzJBhI821gZPVpySFWoUAmLfkonFF3XMfuk8FW31ONtXK7z7PxK71gJwzAJ\\\\\\\/\\\\\\\/RSHQSShZ7zPpsWyXDZkC1kr5xzDe6VcmNnjlwRmgotIFttu4t4FBfeNKesP0RwgQj+6Lhc5jK9Ff8EQvhXIRPbnIS8WHmuwoTAJj93wLeP5E7YUpmzC4YRfeW9Ur1ikX4sgwr0kF3OYhtec9dGlXuemYSM63h9EgF7jAs5BBMuvWDJjoHrOnXDAqbKNcEStx+Ug7znFiN8jyDjWqJaxVz43F+Gt4aSlx60ZmMSlNgcDJ3vs83MZfuxD3\\\\\\\/F\\\\\\\/hwCfe\\\\\\\/4Ub1LlII8\\\\\\\/5nBVIEb8yWaaY3MMnkks+o+rAfEujPr4EDGYT0rqOu2SHU6phHnGVCOZmeAfvdhdpUz6ynJiyzS5A+uu8uT1bGmAxwNEdwHSlHb7cJ0T8Aey3rveH1hwHF0\\\\\\\/ANvMQu7mnlX0E3G4RSxfkZG5N9xvj+byATmxLWL8Jhyl72AUBmkiCj23io\\\\\\\/8bzcg8e7sii9Z89lkq5QTU8zc2dj9+3w5Z2ZqGoBdErHJEHXETdY5UZVGENiALeLjaz51iKhsHwkn6cUTNVzx7RG\\\\\\\/K7Rg6mXS8iXLcvlMm4gPNjJHRH4BSfbRX+P7pIbQmRquvP\\\\\\\/TBZnGVmaXH\\\\\\\/GqYgyguTDJbZOVEgWZ50gk6hSsoXDFf+ZWgAEF52ezhR1M6DNskj38VvRbq3zkz+wm5Rn0bxghziK8FrGACFwbxQNHDttUnK71Q1ckdGY5Bv7y9xZG7pym+OwpMiBhxf2xT5EQAWvbAL5TCjU\\\\\\\/SMYR3jlBtvkNAxphIHNlcWJlycQo2usjGZkrd3xJy96GfYc2WxozkvwNwgo8v+EGO0ni+s\\\\\\\/uFuGW6a+eGJIQ17UYZFPsLvu4DDHEwTAM6PGPUgQQLyQeoQJaahEw1xNzwveu2Y3V7424CFX9m9Kfle7bh8JfTrJJeMuZZBs7ciiS\\\\\\\/3kBzJ+vW1V9SmvGSTMtreA+uechwVMUHrEQQPNFK94h\\\\\\\/wNx\\\\\\\/v9zmcBvMoloQ3mqS8Qb8GeCcHAHOsdhqD3XIGuKCvmieMfBjKf1Z9UWf7MhyxBCA8eSQpL8KTxv6c6FSZtT\\\\\\\/flig60wQXHQYdeVkCXXzwdAqgfC5ZqSpLe3P9dKWj\\\\\\\/b811xuK5\\\\\\\/9hTVVAB3xanGLodTOjkgJwZZ6GIseuuZEzq2M3m34qxOW2cWnfG2idK2ijejURb\\\\\\\/bAguBv\\\\\\\/twGZL6ULa7cb7BJh9vZu6M8\\\\\\\/kYrBGHZd389Bv+hGT5PM3MIHfLD7wSD+tJaQ3666WAqrhO1713C6G\\\\\\\/FdsBL4lipGjPIEKz0\\\\\\\/oQtcrRHTxRcRogIbOgQGYOpO4myknOp8W0rVILXC4UKgnZIEovpgVULuplmQSPlF0nx97Mgj7HdkrmvGdDODuJEWTYPNb9yN1DNXwI+Kk4BPzp94WW0zWN1bxPaTwAmouuxnI\\\\\\\/GltK9RqRjfoJQYQBE95uBURbJh2p8ZWa\\\\\\\/HBcu6p7FOjAjz\\\\\\\/XmK4HRpg7\\\\\\\/hjF3+zO9pXoI6bw5oKwRthGUAmjirKg9NmfCD7UF+l6qPV0PxoTQGnbC1zBaxvctkNWf+067PiI0eW5lPBAhOymQny36LMaoxtYDNOrpPKWXUq1qvJ9beJZfBqhrsZ+gVGrNTSs78zCWUP8\\\\\\\/N07vEB6laNlW5Oax7ghM03qXkPzCDCcEdz47\\\\\\\/ALDM2YV+SKgYUeN+KZbpwqGqKPYKEO3JTYVH9zbq0ke\\\\\\\/ngcWukiOjwbvJQfmEXrc8jz3jUxd\\\\\\\/OXdST5lC9oNeKPkyPK\\\\\\\/FhBgLD9PkkADewylNPWNhebcMRYiHwvfSqYuZBwGNuQ75hRAAIjO+mT\\\\\\\/Zrh97hlKIm\\\\\\\/pWmTypWOb4j95lH5PQKIIXL0yDxsfo8OG52OlvVnwhcZUa56AAnvSYJvcsxEL6+\\\\\\\/yRZBbVF+SheENkHgOXby6fgV7lMiuZjf8gqUvv0pJNZ145CrCa9kBqrXqvY0l5yIdsJZNKbXokfrqKFrq7Yu7LIphVi94Z8kzD7N+IH8UeTaN5aDlSHta1KkHsGEDrQkW\\\\\\\/mTrY674vmwQ6XdFu\\\\\\\/eMXR6rMdaYDvbLPdJtJFz9Ds7K\\\\\\\/Nmh69chDKuoBXQjHVKM4e3QZ4s2hXS5EGs8gC83r8fAT6pJIme+t3VquyzVpFIVfzTXAoZwk\\\\\\\/Cy4WQFs5J5o\\\\\\\/bCQZ\\\\\\\/5wY3sle\\\\\\\/bTZvBZSG3vRGVHijgpv8S584QuSbC554JgrbnpkW\\\\\\\/dN6WROTbu6CRz4dVldASo9fIIuLP6ayWINXX310XkNdpdo0NkRsjb4+I2gYXDHztRKOT1NrsdOcBr7Ucxk3FtSNDRR4UNpdDv4HIVMXGYvou\\\\\\\/N1H7v0IhINBxn45EzIg6ojK10O37idJKzNKDqOeqM7RpI8Gwdfm6t1\\\\\\\/uw0nhSEXcf6z42bDiApvLrYlHeUH2inokkoUTyIF+\\\\\\\/Xwuol4zpDjyV3MPwY0WDio3gyQ8HAFn7fDcSb5t8\\\\\\\/A+5Ye3h9xie7cuSxlB5kiq9GCnbYW4oijNfkNTgSp6Y5roY6S51xd1mqbVlX6f\\\\\\\/QIWr590JPSsU92t7330bABUvOx\\\\\\\/ciPhEUxDNd\\\\\\\/W+PQrBqQ5JFQkjYcUr7tcqwpZ0pDpK5pMk09vppYjjG\\\\\\\/fZ9d\\\\\\\/eCeB+o+A80FeYrCFv8KjKjIp7G\\\\\\\/PPklhHUxng6QvBK3m6xCAadFsdIVEydS5CxkL42x\\\\\\\/hR+8IbuJNlMATJLHiGSP+oJzabUWL3c6NpXC0dYCgtcEz6zU0lxPZ9sNps+zKYUlCH99IaijjM2ygWnBb531LlmhVO5zRtRtU93RfuQtWeWDDyuJuSoM+PDUbFzbZ2l\\\\\\\/Ebq5OV9RdMMQKCv7b3YZ0NXqro\\\\\\\/aTv1KrvQH6X8EH8v4dDfijMvxl+GlOF9QhuhtAqFeJYNvvF2ws3eEnXIcDaFdVgX8ukwS\\\\\\\/PNov3rj9ff61qMOdFUE3iJFVES4FBXXzuS\\\\\\\/PIbVTHW4vA4+EmlpBUrMXJgQsPvuew3w4QXs7qqEsBwfKxR1LZQ1FGsU3nbxU0GZEKLRwkEaqLXRrkWdwpMoGjJstuw1423xkxfhnCqDTNsPsUwRYv3FQx\\\\\\\/eHRn3mueB+Y8KJp6t7NyHFIWpsfKlNgEadNlYd1QJxNp61ypqWn5CfmAw8EfUxvSMFSn4G1cC6dojYtepGlJ2otbRiAXFhi7m6PVdI7xwz7pdYSy0w4gbdxrDziTXcoMGW9nlSc1+JrDRnohppVToLDwfZbS\\\\\\\/fs+vVEe\\\\\\\/6eRhKp8Vb6m9sOk1RebPGkAq2ER+Q9WXM2nqkGyZ5Pcas+IvIXwCdz6jDvePI84\\\\\\\/j6hQJ6dystAqtj63MA4Z1QiyCoLximDOQAHf6MFTMP7me7o0\\\\\\\/RA1ZBhgi5CBryL+lIV1q24whsbiNO2M95vKDKOWj8T9K8+ct3G+svd9eg2RS1pvoKlYjfKUCOHtDOSST\\\\\\\/phKZ6oNrrkt3BAoht7gSgA4w8YMVcZnUzfYivn1duN7WzZ37qPecadZZo8+1pFS+QZxIL9gKJvlBf5jEs7f5qxfqUwciTzn6Cv+DVD+vt\\\\\\\/6aaeQKM\\\\\\\/l0BJ8UT+m4V8gSuEZ6xmg9f2o+4Ynu\\\\\\\/laoJk4dS7gAl9ifYLbIAteDlMt6TtwKIJuJ0jZ\\\\\\\/DXlmzVNGPKZTUIPz\\\\\\\/Tw6cLBPdx3WMX0YADPeAz75ucsMtpBcQxVtu7U9CttEwBwmhjbaN1Y\\\\\\\/hQGKTLrSBmZUwb5IN1qBSK8d5F0OKqttn5VFAbixQc4pYxthvAmlR38MN2jeu+Qa8ch549OTNIGXY6GfT+rsmZAMPoV6MDZfGV8uf9htNf8WvP\\\\\\\/+3qdHxONNOPfMbJGE05OVDLWchvoM\\\\\\\/hFjBfCfNvomHSqcm7yBVphXa5frC23VuRARfDP08QFhM9vlggkJVLAVV04HvHQGahj69Gj8m+0of4VG5ybnXcRRQlOFklUfXe3EAtwhbULEoY90Wf\\\\\\\/DAwMbgE5BCgHz2GZYLWsC969au6zu2KXffEwLWFnOwK8F9JzyF1yhEqoN798ceCx1Alf9lQcCr0UG\\\\\\\/hHYPOXJqo35AJSbrEvfylhZ9wF\\\\\\\/zdK+8XzLoMOoJgT85w0EXh16YDhACn203Wt97GWM9FQEwdqIgWtJbebHTJgns92mANdD\\\\\\\/KeH8mYL8K7RxBCyFQ5A4tHKs4mYbnU6QqgjiN5EVqXDdGJTM8w2outJ+fgnAG4vcq0H\\\\\\\/URtWTUTojObEXg7t7Q5NzXokNV70Chu1xsgcIj2qYWRhwtmaMikz5fkFUFNlCYoNU2jel5PB2YXagQ1m7KzxLvcJhTe5dbULO7xEyXPVT5Sj9fttlSvOvmxl\\\\\\\/G9Dk3nDhD0mtyFu0Tm4wiPTja+gMEiJbNUf5MtpU24v7qNA3SdQQe87KLrxQi4fnlRxAiOymtHVVZ9ufJBv4uEDqpNULcfx02a1O6SGafDaxKA9TgEP19FkohZIPd3RCmbZE6dEwWzqIQXYnpMz+JWb5w2EAGhYt\\\\\\\/t0jZL47kF2mmN5M095beY1YomHAVP5YMO0fc5J9Mtj0Mj2JN1zTEOA5DdoWu5S87LB05eBGRTDIVYB3vSU84coTI3AOLgrTRKNEsmE002wwEpiSyJsna7JdZHhIZVsYD4ZrF6kf3Ofe2SPvpf3LXQiga7F\\\\\\\/V2W6TYCcm2ZNRhi8hsaZqCb2cLYSnaUzjnNL2E64Gm6\\\\\\\/g80VraC1uLt\\\\\\\/nO0nTFDfda7gyAh8jVOpg5pCiEJvpueiPVE2dVtiFV8hYypr+V4ApppkpcRC2+Gt70SDius4MdAkuhInbVGQEvrDjpYKchpyNUs1+SygtBqsSQwPNkD6S2HyiEUc5KeC19fAB0FY4x3q\\\\\\\/qzpFZsqlROEJdK+ZOjoR7EdrnB7aBtJp8gDlK+c+2FhCZ2SIFOmf9upADYkS\\\\\\\/Av5DUCnlg9GP8dXxhzD3PPnCIWGzUo5r506EnzgtDCLKSsktuCj8ZWZ2Q94JGtvG53Zb7Pi+n9o6c8xaz1m9mjh1l3oZqigmv0LLuOK\\\\\\\/lCYPgSgnIPSbTPhKWLGjIgCe45fqTaFewE\\\\\\\/i0QEB8+5msT0wGsvNPijPXGKpCJsLbqfg+eAeGBnBgwHcVAGRPBmkqHDivJYrmPTrp4YjxMHQjZdE2zDZNGFPzSb\\\\\\\/C0Kqn3xjPmQLPpHo1ONY\\\\\\\/YvY+2QP44IVARNGJbqC9fhKvz2y9YbaOmYiZHMhZQloerO35OQcFHGT7gLngiNTjulHbEYanX2nR8A9sQQLVJ4W\\\\\\\/3n\\\\\\\/xf77ib2ng3SL5H9lme8jDwksL77oHiUHh0SwtW98VpFdJaY27s2E+zS8hQyF0\\\\\\\/Ts1rP9af5kqbzKs1asSPGqKp+iq7BjEshIVe\\\\\\\/ekXfFNer3ANrIy2+LMzrELBcXgr3xCIr\\\\\\\/33hhvFRCptDi9To8no0ZkI976R7ncf+VJud22QOwoHBPCRo1v1MBgrEng\\\\\\\/LKVA+XGOhYhkyWrx9AxVvWnFi910ugXR6o4p8AUFXfjGjOPa0kinNfCz5D\\\\\\\/RhZhf6fRKywZ8OSd7yapL6DQhbcOsImUiF79LMR3GtI93Q9OE54ThkAJVWthQ4krQuz8z8FuC0649WNrBaxq6yDOFcX\\\\\\\/cYc48DFmqG5CDrdW3Co0f5FNAyll0+9j+ZD4F4A5EXPhaY9SjTM46\\\\\\\/eG2OhMMla7V\\\\\\\/nwxv4SCDM4gQz92p1kIMHmBX5j2r76HsEfuiqlZPrmDwlseewU+2lw90LN+obVMEiIUuVKpzyAi3CU1pxJD6WShWl7R\\\\\\\/SjEqFNtcx3EktThRFTchMoVB8cyXfef8tiJYpAgyTTZaq3HyV+1+sUlSImGaMumhnbgPfiMA1yKKwjYfrYnPdF84Ko88fR8nZ14xnR\\\\\\\/CA3Wnk95dHiTBojlTxJ1C4ZpNb3XoHHYlNUscO1Wzfsu7QOSWNv4dsnz8uKz5vQiwO3aWqeKNrxkTAHMJ3AXFsdZdZ02bBGCyT1F2hHkvUExQzwtz3r6yQVQ9d7EKCWIeQ0n4ANEAimLENk0IrPCQPhFonc2opl+oIta71TpmDGM6mEkRqNIh\\\\\\\/8ywhWahpY2YseO7995FXNKyZx0\\\\\\\/xnBkW\\\\\\\/Pm6srujcS9x+Ri3sh69kac3sxI4IMh6GYRENrJHXoecE1aQYqo28KRA+pqj\\\\\\\/Jyqf6jIP88T\\\\\\\/Rrm3nqT4oiJ5s0QvypLjO2PLDuAfrlveWXzY9W3clpj86i6mxY6cTYXIj06zne6\\\\\\\/fnawLkknIAQXXjMbRShBDzyAa6GN1+FEa0nceXZQOBkrMUe54MvXgG3udi2XG02typN8TrRp21c7q0MCIrU6jkNJQDYVxzZq+6lOrtDFqg0SPliPRNOefIw998S+ER5m5HLZjnoptpG\\\\\\\/2IihSBbrpDa\\\\\\\/8J3S18zIIUu1R1aAT6rd+mZBfVS6lV9Homh8xpmLUxr58WsgELYAtHaZOJB0bXyRnTSKxBgmHYbGRbJLNdF+IeIq96epObZqfFEmUkkgkbk8H2ENBWHaXTpnMwLQIPnkdShUkgGWYgIBHceiiHkn4yisUB\\\\\\\/Xk39KRhStIKy1Q3+2LVeuFZcSHg337XLctaFdTOqzGASds8HMh5+aG\\\\\\\/2oAUP1K6tdMUQ0bGcTV9KQLUll7RiSASgIl16qYTq6Q1\\\\\\\/cWY28idx0Ttk8He6nbn447fHxJ+Rkk9VDZ\\\\\\\/FJj9IPOgwT9RDvDIE8ZsJ1xC1J3v92eotkGPnpJoWiGI7eITYchUL6AL\\\\\\\/DCfC7CNsKsOHGjwEnOCYRQ7r6PWH8u3V\\\\\\\/n+z2+qunt\\\\\\\/MR+ARukSrllV+4CpWq2kkPQ6aodXYBZQsufc82pqbQ3sSHROTYO3y5oXgsZM1Rpzr7+oMsUZxK5VYcni7TLjd\\\\\\\/ADcztTtt01Mu0AXKMRccwU\\\\\\\/4g\\\\\\\/+J4gcmN3wYmqTmDYyMQp1A+gpvcvNGPvaFEFdDxBdmhniJBtZRh0wo6DlmzVCRoVZTO+epKFH6A55ld5NR9M51LFo3cb+ebCzwgY\\\\\\\/eYI9mzXWuiPhT5mHG0QS8dGMG\\\\\\\/kLoEaA74uBAt7cDPTLDEF1fHFcYByV845pcMgV24hebsE9OTaGkf8L\\\\\\\/Fj2d2Gv26j8aFqzwOosAo3KXGMrhMgXs7bWlOc1XD5eYbO7LduPZ31DqQP0A3DX4jlWAY+koOIpns4j5uuSVbDMAACslgoyJw8EZx7Lr\\\\\\\/pA4R\\\\\\\/Lgk0A94MVUKEXPvSXz7a49B2ka37ZsbAlkyxf+QatNWig5cuoBZW5jqGYUllhjNouGQAng2yg680BbbM6ahJR3vdL4PbgjCLdG7Il4bnASn2LfPxnmtboyExlFZZGKcXzSTeapSgd5JAF9W40Ke4ZfNg9mnFu\\\\\\\/P6B\\\\\\\/T2lgW4euc9\\\\\\\/0\\\\\\\/ORj7L6uxNXj5sKIT40FHeJk\\\\\\\/PRuXUkJIGD8TOxmtGZDS1dn8ecgObW9+ihcxlt6ynb12boe1TEyhcsTlumkiob9mo+baj9lrlvUKy9N17OFgKvZMpby0txQpu\\\\\\\/sQPGJyXhZZ27vfUOa6p0ZWpcQzWXpz5WuQXxQBH5EEY2uPp2diT+\\\\\\\/WXXYTWpIFI4UE6zhYrEZq6QX7ACoNuCyrYkM5Kn27\\\\\\\/N\\\\\\\/YqDGuQcUhmeabBafgM1WQ0EbRKBGCmrNzZXIaQyE0k2A8wwBf+w59Ji4lwMIeFl2JFKJxyWQwhelNTqbD31hl0HXzJzxfyVr03EBP1eTGieeiZAZQqMgLJR5ieqAqGJib+6qQo\\\\\\\/DLNEvMoLafLMH\\\\\\\/n6xkRLaZ8USrt8XTkd7CaL0T33OWotWeVT4\\\\\\\/FycJxNnOSPHOFe4NzwPqlBap2GfQs4vT9ACXw+qLlBmN+uJBerISAz23NSRzyb7AubtJdTtWz5IR22dHIgCUGNSJi4ZL7Fo9JQ++RMxx73ucTv87bud4X9HF4ORBEcm8YDCOWCTgKIXiUvyRSt3l1twz20PXI7L7UiQcu4iHeAiutt9ZbzOxTe+RW4ytEy7hWxz6bfrE63owOLz6S2+pX1+hdRVABIJE0h4mhHWtQsJAG9YR8ER4OGCxERtc47FE34LizNVUIZaV84YEj8PvUG3WhPzEOtDNzTtI7uBIiis9+ruAjFwoLI98cC6DOGxzwMx18n3ZVXSdDoZTyRNH2o0whpT14\\\\\\\/2YM1ukV53JDq2C4JbhwDKaSOqVRc6+uOYMVuD\\\\\\\/HManTLftDUuxGbAPcZ\\\\\\\/X2d44uL\\\\\\\/CzK903MvYE7ae3P2sxt2V9NMzkQfJ2zTsfz4FMOtuCwrLFQpooQ5VATXQtYXVTCc+HiDOtpLscCKiXxEKH\\\\\\\/bIrVY5SUg97YRP+1bap7C+vLd25ocmINdULcJI3HD6QiJzx2ZJQtSXs3xTF762X\\\\\\\/X6XBTd\\\\\\\/5rQXIMGi\\\\\\\/x9rziPMXyjB+Vjmqu9jUDKtRAGkh5ULK6C6RwnGCBvYIiHc6KFndaFbOj04x1hAA7e59C8OjJRaHtAT6mBwCBVAjWY2174n7wt3MMKYwqSduvvNeWBy\\\\\\\/UES+W89GDOTM0DusiSWoZOHnqz5U8mQ4oKQB44++3663yJNg4f+X+jbCz3n4vdrw4ICy\\\\\\\/HAS7dEptgYUuQeVgI1Yo3VwmTpLK9R+u2hQcmdwZR7CDDXJuxUkc8ThLKx4zYq4d+fdINKhSkslvtE\\\\\\\/cGrKSbXe+AKY4b+vwbyqK2Qpe1kZjJZBjNNdiETgf8Aka4yTo9ldnUTnBDm0VdY6p7+lz+lDMnORoqNjxDNJxoxKW+vVHBrAElrZK4xB2L2VNSpK6\\\\\\\/FpeIBN7fI5Y0RkeCl0Oqm8X+w53DMVqD5zpT8w42oaCeHXIe\\\\\\\/3Bf1oD9S3iYWVdU5d1CNca2fEItMU2AU56DxxzUPUWYa9bd\\\\\\\/vHlM2HXM4XXNknMaPvRTPNwr1FdSZJc4L0+PNj0\\\\\\\/IhjKilh\\\\\\\/ICNexvG6RMbhUfvmzlOUCv5cXp4uJ7VLXZB1tzzgPWMHIsVN7t0Cl6FHpNWMksRan29MVmnRvhTjZFfK5YgN5005wpKsjojCi6+Tb\\\\\\\/RA+qDgrHD9y6LjosoZ9wqUbWiu5lfTRJc5tQJ9ADd63ypsrbQtQ+HI6GdiGeW2o5xolfayrzOsJqL2eN7AwMg9TonGwgAfMtjrH55oGAnV1Yet40u8sKFk4pOFKlB6j0TPHkxEHBPt8GbyAZ5BD6EZCzo8SO\\\\\\\/266ztK+7b5\\\\\\\/hQ5XuvBbPpjZNViCso7GWT3THKwND1Q\\\\\\\/cQ+GY+AFm1gs62XaRnLCnJuICRSct4YDsAEJLLqb96j2t7YV\\\\\\\/+ptZDI2wfeB3GhBLDGMkBYbfVeVLqhJqUqUJ34SXok1rK8BLqOCWj2IElObKd72qFpYoLxr0PlHRoiCH6LfKeGCQik2wmFEV2WRajIeQuMxR4DjMp5Aw\\\\\\\/yOOa7CMriN5W\\\\\\\/eYbeJ7yaPXZq4aHY0+h2qIZyGN26\\\\\\\/093z7J85UC44LVl3j9j19HvqGq6KDxKLjDkVZ\\\\\\\/sP35GJkbBlE6k0ZXdTX72x+cNXSOY09yvXV9upzlMvkdHEU2zv37LtOAE6G5XHWBqTjrBs1g8R9IqLkVFi6t45iFTj72s20XKvZGatboL1RTSv1dljuCLXdY1kB9KIM0oJfF2Qm2o6kzLfbRWF2Tc4KCYSjUhl8oxmPZCY3oK2hclTHrWxykzN8fX5scoUibp1rJbZ8yHfnahA0ZsHkbrjhHsjhGE6e0Uh\\\\\\\/e7AMMkVtyWDbRNbmOezAOhWbb7HzI8OLGUrsUGtp6uYyRTNqlOVVGEmvRK4GHySNzf1890byumogfjjJXsDM\\\\\\\/Q2sn6PoOHccVLj5wlV5+0hvlNxCM9Xfw6ORbJtCLhvZyfnk2c0wXjr3L0kEQzTaBcIZkbIIksUY4pAZ3LlnN+8u90NxIwhhlF7aOAZOqZjAMnMjY6u6ZOMLtISK43HH8OhsmSuzqEue4xQe4tUqXzBaU08Cb39xwyvOZ+520NKSazg5ZGDX27osJVV7BsYtiR04howtz6oS2hV0B2cBMTzeGFA4BAwe0aswh2WL9IFrVQL6elYwPoMv\\\\\\\/HzWRGUF\\\\\\\/CXlkEjVt53eW3ofMya61NhsJad2iWYezO9TSyCZdRX2XV429PdMk+X+Zbt9Y4HO4aRC20kmp0xniolWDq+ky+Rp2d4XtKk6RPdPudke\\\\\\\/D6zmzVsP3oueshnzmKYNp+7E6h1BMth8hrh8peOJxMWv6ynpRG2SNaOw9YmMXu8QL\\\\\\\/9CwRAJkGTgzG5O4o0ZUF2x4g09Uvgsp1SQhaPwUFFXZ+OppWmtbPUamkVXqfSwDos0zaOj7yoAIEzG8d6DRuhte3y5OxIJn+DOhwHATrILwTB75olytcSSseYYnDNihIVnxgKL6kPOCPwOunOQ6MgRMIF6mAgNQKrC5JhxiEUMJdqPYcBC4yHYEctdO1x+XOS5SY0mi+UhYaKem9T9LynL4xeuuEEZP4QJKzYVSbKb+xWxWt4dmDBUPJVJjn30E4j27XdIQQFkFXkc0ErqEPrZEXFMJZUJTEhPvo48gABINwMDCBSFkQYFcmrzcGE0gy1bSvihmKZF67aS2f8Gau2n7xGV4xfzGEHLjwdgpuc5uhBoewqv9DzBfXDviaHaMDWjLvNvT48b6\\\\\\\/s+\\\\\\\/jvYECeNqBs1EcUenYVVuMcz11rN\\\\\\\/7n5zzOE+SqsbkTyxKz6O9UeYNcUztaNe4MofGYo6vjZ8nM\\\\\\\/YYD9rbne7xGxNnZgguK8xpoNkOg+yM1dZVGaGHVCZmGUs12wWqhYflSB8DBgH8A26IPgWp0kKkP0PXsROwud\\\\\\\/R7PQajEMpYacHnsBOPcy7YQc8OQg0UVKBdhYEk\\\\\\\/WqXevUVeScQ4MXHUJy4vLcbAnBFqVWpnEoX13ZapVxnSUGl5y5Lj698YuWD2zOOdG0UvdcLFjVkkIqsUjLblAejVIQ2iU7kklWL2CLuUAb5aWcmDHEmXUfSAqZbCx1s4kFoXW9XocfVdXnm0sTz5tZxgnu\\\\\\\/+TzgmA3zTEoQ4TaVRgqHDii+3ltulSJyitX8RUDCcLX7twqbppmUbNCn\\\\\\\/QS9yvSYBN\\\\\\\/3r1f6fsNcVvJD44HrwvjX030hIbUGFkrh83N3WBdiiYhBJieJgwPENaPt8C7\\\\\\\/euSCKxhxv4yXnKzD4X8+PxA5UEVTQOWKje+M+anZxVGFRUIC24rukPNYJO+Us8PS8U+V50OkRw7TmHcf7VkuoPisDsFbuO4vycoPprfuLtA0jt0FuJVZi8nuhHGn\\\\\\\/ju7Ob8I2AjFSqSeO6JLFaZ2E7VIVx60xM41c8OJy+yIqQpFo5pokJWXl4nICdAhYhBwoyd8QYJwAGND2+0bRvhgON\\\\\\\/DYz6U153ofGIiBvpjFBxQD7q\\\\\\\/Hrllox8g6CEfl3LklpomQ53w0OuEZbxJKsgt905t19zdfeSEUhFmm9JV02+8iiakwTx1Bks29IituZjsGVz3Ky1G5WWQs4Xf\\\\\\\/9CHPBDgI+g4NDO2kw6DZXk8JKxtRNoPFL6O86rZyHYaNX1qBENScUabfDY4YywjYCSjoBsv6FfAiszCB\\\\\\\/CFggL5WS7JMCJSDX0CT9Afss1udtSy0gyFEKXdp\\\\\\\/GP4gW+PXbRZsLcR5ugMKc\\\\\\\/LUkeFVrlaOY3yOj322Wg9a66UpUUTT6+B29bllAakyHxKqOQT17jHbZZimZUW8KdG2GrNzPQrsZFSiFewAF6uHV0VJpz+EvJX\\\\\\\/niCypch9OnBCNZ3CInJQVjz2+A+XkoLrhBSEbRrlJbgwbRMh\\\\\\\/OOy372h5ojGBz7st\\\\\\\/kTBKzBxxEX7T\\\\\\\/ugkddY5foBEiHNNmEsN1b0mvNBimanOncqKcxw+ukT1JIC5R\\\\\\\/Bb9HOxg0g4L6TfgLShUJCQ2Y9jt+8tekZrmG8TtWIumKefpws7UMiLFRouXXxhv8KqDP0qYwETY1n8zsTA0mowWBGms7QBijirjaXjzf4sol6IN1gvv2w4gbl4AgUzTJ9zpXVI0R6XTEjwSSkX+SGMcty5GM6WaegwrPxnAp0uQN5Pbhx7KKghDGGNwmuIUjMnnCAgHcdqRTSqpWLyHXdB0jDseuLiXC\\\\\\\/KBDjerH99megksvYvieJAhq\\\\\\\/haYFP1G4vIa3kKgIcoHqDiR6kyuIKJcKkRgu+9TPXnx0oVIcA\\\\\\\/+ZwTWIqNXx0pGZ\\\\\\\/iQ4eMa0rp+Q9tSuFKiZ6IrpHDlkXs7Vwnu3yoHcDRMQq6etYQMCOgM8ilUa\\\\\\\/x8EQFU2b1r4BiQFkt5C4qQNfYJ\\\\\\\/05yTtjhqB6kiZRub0vBxCDhJN3cOAFQJmt8IslTmM0J2xJ88A7xSPvAbDGbCcylGb32wGyzY3WqYjzgyiOFmynlWGFMXCHaab5jBNVmPuyGmuRdscS1Mk2mlwcoskQFhTDwywMuAYNuIF+PDbqG1CA\\\\\\\/YLEwb47I1RKCn6vTj\\\\\\\/XTqs4PK9PnbovitcSe+SQxuR7mvg1yhg4UGqZUJCWZP0FZ7mkEf+qn6Pr9\\\\\\\/UGXkt7s5e3XzZJtybLOY8GgCyjtRhOyIyrWxL+QNGoHlnIP3lrd82V2HfBC4i27wCV9XIcHGZoZEfJTQlIMQvcjln02MJUmwD+pX7cHT+IAtA2b\\\\\\\/MHCvTZ\\\\\\\/2xndl2KhmwlatpajBJ6bhegSp+lX7ntcYeB6IFXMV8l2jEQc0zhPfQTsF74xvYc9p4QCsEA+RjXHwa8J60JCfZE46WLIqGBiwFfOt6tdrc5uc01QIxJwvzN+Z19V1SWrYHCTiaEHiLqUP68rhshYylywaK0iqgFRcxb1ejQfpbG5tSFb75t5igSaEbRIpu6i5D6ADjWvKXn+yT2C+2\\\\\\\/OYGZ1wtp7zvrBIrZnM54TFAcRlHL6DWq9ZH6drkvPxgq5XT1ROUlapcaiTuqX4pZT86P8gHLV8fpRfg9FejpJWBWxkmQGirJemQv3GWzosHr1tSTqHHaJF9UbUyptUe4r3aOZJyPaX5xVXZdZqzW1+msWimQb4Mv\\\\\\\/pJXuQ4EXgZwTgLh+PxEHgWsuXfoIYXGEVuSBi7x+vwjwToWwOG0bg3wq9pvVzzw\\\\\\\/hed5frpbOEBsZwMvejYcJ6wQCvS5++Cgx7p2JhTKCv6bqKgRNJgxIr+XhgcdWo53Pvm46kvA611GRuMdMObNU001aIhCoqITdG7ebJ3Lp2eJLj2p0+EyMYTvWxev0AIar\\\\\\\/\\\\\\\/GL34300nmflnL24j3NEBmTRAaQeYxn8OK1aAXt57Orah8P3vbYJYxW93\\\\\\\/4mH9dI9il2je9wTdtBWIcppJxC4y39Gmm2Aldp2h3YnGkJgeXd\\\\\\\/BxnTXPJRCE1\\\\\\\/yz8DZJg41VQtlfCSdjhZzCvJH0EhlrH4KiZahnBCu3x4Uo69QqblWdkhpglLbdrzzVbwDDcot+ENpvwMHPrH5SCzPKbGgpcorHaaJ7mTXm2eZSxHg\\\\\\\/K22tfPLIEGTVQqDJm7wABP4mI1g6MhXfK0F2Rsdx6XrizimkwSEsef0qmxfj+FawyEN9EQW+1qXkN3IWLCH082muECcfntlwJkJY8bWwSBtKlmjnDmHpaYZzi1q4cyJ9ykS\\\\\\\/D4F4HjRLsBkw6b6GT9UbH5xIad07Sqzh140WyIqP43\\\\\\\/gRKFTshQiHify3hEp8bNoDhwB5uzKGR+\\\\\\\/l0uWq\\\\\\\/YIzE5gDKhuYdiaC4LvYq5x+qlWsE0oHCjDKJsFSq3kvDm8MjMkSTEXbd0sljz+NkwSMX6+INsR+GoAOydZEdrsy7wZEPIFdI\\\\\\\/sHgDzakOhzey6FIsFT7hHS6OmnczmZAtdACaopxpOYJlYoc3UhYJKjF0kDbEcHdWfAb32pJuYhKdKdFu8y8opxE\\\\\\\/9gQJUj1lDi7H1\\\\\\\/GHmxYijiF4cXZHiVRDYsFjOaN9x3RpoPujN2LytPKaTOiaS7jJki0o3Ik9LOob\\\\\\\/UyndkTyUhwtf3OMtr2kBM2VfLLfHjpIbK61yTuIXZV1fMtgqHtg2dUjdbBX3wO3chh3dSNueDKNZHXf5Ii8DDU9nBefVTc2\\\\\\\/q24rYwn2S6KSLDW9+YVVbGE4GP1M1zlpg1mAm\\\\\\\/Do977+kY3S+HR7AxGkZBXK8\\\\\\\/N3ngygvYMJqnHdbVkYlL64rGDZt5UjjOtimbmLv0bSR8ArVf5heZolLPhsvPfDqhDDTncaX3MVdv9Z3yRufsay2XGITaZiayAz3oKAvmhenBaysJ9UYsqsg4ql58hZH7SEIQLOC4lCQJSemE2M8opYw7nvVkdqvynUd7sW6XFYPOug27Vc3M5geyxq6T8wNhq+0NXWAUVKsdWMutrF82UPmfu7qk8+yXGzPmvsxTPydnxaOVrhVqJtbMdOz0eIaL+cV8Qf5s9coxQrEt\\\\\\\/C+PghUkiw4ozV4xdY97yt6\\\\\\\/oDYQTJu95++C7afc6JeOe3YCDJGwstjf3HUJfiZWTlfcFtHeeQ2YsX2fsqx8upmcEbtG0BdpXTNKNAztKQepfdY+\\\\\\\/Y0axghZm4relkMt1C\\\\\\\/GqtuACUijov0aVUInJ+8jADCn6LZfMfWrhIFpQfbbD+kt36xLrPDx4ytsbDn4pCeUczKH3n0WbhcawpA0z+suZxu6VXlkVoQgTdJV+QbgDD6selZ2VfUWaT\\\\\\\/QqXbPNm\\\\\\\/f3BShsUd078pgpkLWY31NOcG+8SYXwtY9fVUHoYK4dbR5xQU159RK3wIhyq868eAO31\\\\\\\/wwqG8a9A9adH8EZxzQSRqtEAh0j1YSCmgftxRnx4pNDy\\\\\\\/M1ttYQxyGOnubusCNaT\\\\\\\/MM0676kU7De8cTSMbz84fgl6CG61qaJKJHp2Nq9sIe84uQydzSmPWuJgVwPBZL9IwZVdbUyIhmt+IWs0ee\\\\\\\/kwoq7weeo\\\\\\\/q07HzDTqibmZREyP+Qwzwg4E9TAEOBAjoJew4oGFZIeIxouuVRQHbdupXFkOmvm5RkHLxhz9QfvphlbWw2IRgJp5QGNl4OHmu7dyPDyd7AD8WOD86Iqz2ezeda4lLdll3ktL4kT0vSwipr2HIwM4uBJ9avhod6COJdWAkjQzAAMzG5hdXjB55KQBTwz89CFiAwVAFOEBggJORf+Szlm57RiFVtYufgve6GAVF7jrXJq7YBJDvIqQbv2tE+5NbNveDh3UheZlOn+catFZQtovhNajFMfXeSINEa8WH\\\\\\\/YgFvzuyiUYcyvThVn2o9eYtdvzmr\\\\\\\/XTWU\\\\\\\/jbmSTostuHBKS8PbT1HFoV9adquW\\\\\\\/lidizgiRPGU726E8aiJZCktiXYKp988XeaB537RxH1SPUyJD58y\\\\\\\/Yu\\\\\\\/jFHjQFk7OCJA6TCTRX+AdGOaXlvn+su20NnZBIOXvFBMgMo0hVUlfzbms7o\\\\\\\/lOkBg9vpCiMQGrQI\\\\\\\/kGqTTpm664Ju6KfpvF+TIhcEQP\\\\\\\/ctOBZm9qfeAY7v+ZX9ZqY3BCFW8M+Xekc7e01OjkRwg6wQW\\\\\\\/ilF2t17zp4fG7HSXIrXocGiFdr\\\\\\\/z+JYYPJqal50y5WHBZZOJYDZUQHL8hRPcDcJLBoXwd5dM\\\\\\\/icwZ1T+Ggnlw4ezD4UlwEjvigjAn8m68uSG6RqVLBqJZ2a4WVkT+JUSlx9rHAopHrK4c59P\\\\\\\/vpbiCecEoXW8OR3z6qd6b6FZGxxyg7wSLka+vF6cY0oh3k2Vf7XWrsQq2\\\\\\\/g3\\\\\\\/wY+Ro+w5MbQmfKxhoRWHbS\\\\\\\/C6PTLpP7OIHJHa2O7h+GQQ\\\\\\\/FdFRbbhpItMW6S+BZGeYpPF4Cde0ksBqCcZ30PMZvIXF3+1VP0sOqzqX7iodoqnZ0IGQG\\\\\\\/qcN\\\\\\\/Hg7jCrb08uW33ao9+A2wWF4wVOEiAQ7fH0KFPoGCfN\\\\\\\/0jTpS\\\\\\\/aYj\\\\\\\/deIuo8iQf3vZ\\\\\\\/0qBMqEpseGWmgh6BkMgFJTJgusaR\\\\\\\/LYjVYk6TpRVJG68kiX3kvnaiCY2Qjdy9swyTg3CvghbtKVALlfFLabK7rfK+7UFRoQvXru6aGG6mWZ\\\\\\\/j6Z37ZhIDuJMpnjryMw+ziUflhixcuk3SZsHvDdBeCJhUR6KQct82vFAQto1jCOI7e3nAoSEnXxCIhjqzGj7L8NYoLQYrYdY\\\\\\\/1kXB+j7Xzn6CO6c5REjca70id9pIB35nqmXbAM2Pm9xitS5jjrsHqYFbRNTYFR1MU+gYMWJ26jA0MV7TKqTC\\\\\\\/OcMazR+67sloxYVbJpsq7XokqKFpuJIyJrnR3YgjJDEMl+gpSSW\\\\\\\/O8J9dXdmejxFqH0P3QaQjunNH\\\\\\\/i5\\\\\\\/mxHFTHCLg+t1loZcAJ7oR6DepSl+26l57w6KcV7pSJhVsIfRDcvlXfB2GpRqwtiMzBlyfVwZSWvvqYtujuxEBpvo4pY1sUcCam9adMXDrWkXaVE4nzfZ59zxYuB\\\\\\\/YkmBDTQq5OOc2L7Q4QSyF7IxwIt5Dg9LPqqJ3r9Yalrcq6FGhFxp5XaoQbHmYgAoWhv91OkPeLdaCpHfVR7BI+864QemG0so\\\\\\\/dJgAKug\\\\\\\/XTOxHiMJuJ2qoyxiMgpmLjakl8z36naEgFIobPsZln+CxOqbDz4cZku4fcG7+g3hT3lPcnSDB5s\\\\\\\/7g1\\\\\\\/k4\\\\\\\/goXgjjXC0XY4x3AmzT4wkLh3HOi9bNZoKS4cRWXQVY6p0kgim9L4Lwme2XiqAbp7gRQu0RGc3LNiv0iEnwpeuee4wAPHpVceGuEH1zz7wUJsN42etYJktkT8O\\\\\\\/9Z3SZqF0Qp4j6uixok9QetDaLaOiPPK8z77waWWjBde8HEwVv5NcYNWDuoxvm6Qq+uHEfNCxWEfAQvrm4cN+zJx6ImC8CPL+FUNtLN8opz14\\\\\\\/O3WxFhrXLD8WM9zm03vhGe9z8FLxnT7xeCSd8nOB0XhkXzYCeK3HqlznPY4NmAnCDUk+8Jhuyjtmdd9C84DLTEWnxYVmliDlaRlcFKqsTZn8AgBrbdvDTNxXtVkqLtRMJkqc4bKUAcCX3IuRfSYynHz\\\\\\\/DX+aokevbZAgi3O1l\\\\\\\/eNvXd\\\\\\\/Tk25MyklcOv6lVy+Nt\\\\\\\/XOR4txiN0wLYqNE39aFemPS9q4OdyM8g6x0DlYlti3kIoHfBteBfo2mQkSdV+tI0zAJozys6sNwTEynayXfxkrV1hEmZA8oA+W6jH\\\\\\\/r9u+x4TlD1kPXUXcqNqmbE7YmjzWr\\\\\\\/2g84OWExhSjwPCgXxl++OphB4pB13R3WVEkVYtuvc4YqD4cYyhvgDh1xLGDg4y\\\\\\\/brlJwdwjbirO8BrFsEzMy\\\\\\\/fwdn1qBYVWVRIOcXteipQXmwSOqrrgjbuZV3juzlfoU+ocYgGN0rL19Qtfb9ig3WarEVKUR\\\\\\\/7Qt2L7Z+06gFtODZC9Cc+mykBGYsPnqhOc\\\\\\\/tpuPixEWbo0D7Fq7l5PafKoESjYLuylg7FmV0IMeaF6eSBGzGPuLmUdto4aV2FEm\\\\\\\/zxUWWcKG0KX6tlknakDZppbf\\\\\\\/oADxu4khYKwoamPMA9\\\\\\\/EpK0RlsnfhfFXCsluf7RN6yraCszPLpCNEx4oks3U9WpSnEvGO5GST7bDq+ni5h6C4AE8Bygg8e9e6TIunnSriSbwX9SGDpBSSa8qsPu\\\\\\\/p9IX0hPY\\\\\\\/jp8pc9lnvCK19NSdDQdetYMxBeGbGMETbVgz302iqEkwM7psI0gdiLrEg8pKDtpR3oCpM+Vh1fCBWSEkiq459Qs66oMRE3pKkk1kc8ls5ONUeKve7pkq1EqrTxz1wUbYafixDmp3rdLkUvCAXipFHI6P6kgJ\\\\\\\/30f3Xxl6L9Fen7dHJC90d\\\\\\\/FbOMaz1GIJjLTiNGuCSr6vnRMe4kyqy2YYUts1wW3y1kVUGDmPz0S4llm+jvj6T\\\\\\\/PHe0+qlDdln9Ch0k5RjrYgbhN5rSxtuqu9touxdDPQcUYP71f2RBz9SUJI1LPpNOAv368w8xcp1\\\\\\\/JQpGqLTDcFeP0UOAt4MpiN\\\\\\\/j2oldODNGVJxVxkdRsvwXUQY7fXLFWDqkSWwjbUXyaIE604UI6q\\\\\\\/h\\\\\\\/loi2ErF9dSLx1oabYaQhJeZ1SkyIBfcsS+uPF3883iBBAyQABP2PhoVKEG3FqwUXwaxy2bUN5SaYz54R1rDq1jik4jbe0nVmq2vjQCAmKOCWcWF2LqY6j1Cuwsygt1Q9ZZ8a\\\\\\\/aC8LZemwW+dtdqYSvSWCXU9haEVJwTW5HubftzEPUaLIkl5STC4IxRTaGYTdPlRobxImNdwcbMiE9YcbeWExN9XXaGz18YCWKcueFPU6wNHNTKpqOXDHyq42gn2+H9uN9SgXtQ\\\\\\\/GSn4H4EjmDycBpx+NeM9MEwK4LWFUBClx2wGWC5OFhPBmXUNsx9ObcYmWseWNjCaF\\\\\\\/1YKQfSO2IIG7q0HeifqnphkUyrw67yGwrp6EfWXByhJzRzGX3hwXmtJ8j00+0qkKcxeoP4\\\\\\\/brsyverDzhWYzU\\\\\\\/OTAvb+uYqE\\\u0022,\\\u0022iv\\\u0022:\\\u0022xGGzKnkzjx1J0TzSzcFU7Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e2af78d5a6afe4e3b17250a46955116c99e6e10263ac1274a6e4272a1f66606a\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022K9NFojf7dntXsn7WV0K1Y9HYYC1G7EFDrDXlQoSUDJoFfGlrztG427ythc0RnDX6Mu+cTXiKPrhNbjKH4mnA0N25pyQlTYyZYZbUtHQMujwta49l7ZJV\\\\\\\/uDrxaJG6iQqKros2cSFXS28+7LT736kSS2iSBESBNKIiDXhbbVi+3r0V\\\\\\\/PXPiGV1EUhQIZXdm+FRDrwRAmfBA8XnD\\\\\\\/QO5BU\\\\\\\/47+WSyPib4atjXtgbn\\\\\\\/YEL+bAczzqzu\\\\\\\/u5kTxeAxvVJE1UFVu6OFTWMljjC5J1NZZpiZRcGjvB\\\\\\\/F5TWHH4sRNIHWu8ETOv9axcim84unyGAmo3ZYOSYgoS+7RZlY+iFRU0B2jjbZUKeFAXmG\\\\\\\/fvi67PQvKoy3rZMoj8P2CSYqcapQtPSnObgzYJ4pxL3nI3DapwdjeQ0xDsx+tnE0Qa55chT+JTpku3QXxgHt2zwC3RUCERxvfM4gj0XT0Pz9RiGOTHplHpNMpgDeblpZeO+KIt56a21qRpnnKWZIiV7QoEZy6nlhto9\\\\\\\/3JZ7QUk2dA5ldIvKWo6rtyPByzHq6IzF9KpxlQEGJL4Ck6MFv7CdPuxgRS0QcTFSBbXEtjYovFPNfLHX3AFyHfh4x\\\\\\\/1vRyNcypewxD8EHFK\\\\\\\/n6nRNnbYiA43CzfNCyT5vC1EmnV\\\\\\\/FnmGvGeGx3k91oxx0Rgf153Tu1V+SJNBC4e+7NKN15OKNvdDravfyzviRMzF5TDqeHvjoFgN7cz++sBqKQay+Cehztlihk5XX60qCuaib+AHOT4dHrCZiPYDX708RLR\\\\\\\/+X9ydEK+ylADfNmVHJBCBNMPkZEjMN2PFzGtQOpwWhBqTNI8sGIdG6YtprLTiSD1pbSjqgRTzAKrwBdZ3JqV78P\\\\\\\/exD4nDdAzQqk7p4KvvmEvSWsjJcqmIS03jnKMh1Vl14+rVNCsRNG91z7kWCjPXBEW9lN\\\\\\\/fCrpaHDuU5lRPeiMQZ0Yhw8LvBNrW9qllOeF7sDwaJl3IGwyIFNKEsy42NbbjjuTGvDGUE584pFSEx\\\\\\\/QQajOB3if6+UkJNzolRXkza1xdZKZ07Z4KBSpQ97jO8hQLDU2ETul8V\\\\\\\/7h1\\\\\\\/54ak20B9LqOffSJJvCY7kpeYOBtcUfEpwVBDy5extBsnvdiNsa0GEBVFZvjAOxGmX+sLJEUiAiCDM+g5Ja7M8RIaW0fLca+mLM\\\\\\\/G3DXPgxGPm6ugq5L0Hv7OHUBpGas4+Lu6IyCw7NWkG62uB6c51X5JkSYknZF2nLY4\\\\\\\/iisio2L8hG+5vc5ZjjtaJ7nq6bfZ0+tdPllx0BNgy2fIhfBcPJTleZilTdagAFsOmo9tkbUMfVst\\\\\\\/l7NYyzwQvCiIsOB7ZoNgBooneqwKSYYRw\\\\\\\/8oJ+v\\\\\\\/bjQxhDsTbwbXAb4fXP6CNQoEJA3vt2OSelWpB8Hg2p0uJ4v3F6N14EBFNP3lA+e49lgig8PYzG2+eMMyc2HJwn\\\\\\\/EvCVsKyC5Xn5J6QWJ44Z1WOOb3VFQtIJ4QKPk2YyWus00nRh7dUNfCUThIJaBGlarmoIXFAsn7kwzRh6wiiIfSX4zXNKgtALY1Cg\\\\\\\/J8U+BPemhR8SBaUTTZXvrBI1gi11FDVMGpQEQ2H98L6KRsnCDzUOfTK7pZGmEdgoseNHpesWyOrsQCx0D7U9sEvdxWE549sIJoAMqHoTBB+g4dy+Fq59vx\\\\\\\/Tpa3heKC2UuhEwvQrgm2C0dq8qTQvthlkIyfN185RSoH5UsVGz\\\\\\\/tSrIc9UIsdaokAp5rbYSxTG96fQSx4hLHQnDDrOXDmXjZrGK8K1eg\\\\\\\/JGo0w1RSnosi+13aAerUZTz3PXtkdmzro4qH4KqaMPJiu2XZu0Zh0HUQSDZm8pPilCChNHCEkPp2rDdD+1M068WFi1Sn5pkMQnyha4jgcMJTaAhfBvOBkR+3hARje6Y5VvpCqbiylOUvvz6ynQCwXGi1k\\\\\\\/9wBO02zjcK1eULv1Jujh\\\\\\\/3AWfBjkOC7NNJyJmIyVhU\\\\\\\/86QIpTO0M0nKPQgrAPjykCEMmsxg1j9S9JTLueB3RjlKqqvohOFqKs=\\\u0022,\\\u0022iv\\\u0022:\\\u0022vDOeh4vRHPp2GgJg11QeLA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c9159a0298d32129dfb0d5b48c8f9ec2ce8194967e2d5948f9613bc2aef3f86b\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-483528 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/3/943" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-P1z-w35Kc7Ofr8620Qg1oDA2Vu7TE42NuRxUE2n7pqg" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/3/943" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-5lZmqKUe-gA2igNciXiY-zL_PoCOyVe9g9x8ALJJiak" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/3/935" title="Single-Step BLUP with Varying Genotyping Effort in Open-Pollinated Picea glauca" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/3/953" title="Genome-Wide Analysis of lncRNA and mRNA Expression During Differentiation of Abdominal Preadipocytes in the Chicken" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483528" id="node483528" data-pisa="ggg;7/3/943" data-pisa-master="ggg;g3.116.038877" data-apath="/ggg/7/3/943.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Reproduction and In-Depth Evaluation of Genome-Wide Association Studies and Genome-Wide Meta-analyses Using Summary Statistics</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yao-Fang Niu</span>, <span class="highwire-citation-author" data-delta="1">Chengyin Ye</span>, <span class="highwire-citation-author" data-delta="2">Ji He</span>, <span class="highwire-citation-author" data-delta="3">Fang Han</span>, <span class="highwire-citation-author" data-delta="4">Long-Biao Guo</span>, <span class="highwire-citation-author" data-delta="5">Hou-Feng Zheng</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-5475-8237" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Guo-Bo Chen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">943-952;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.038877 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483528" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yao-Fang Niu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory of Rice Biology, China National Rice Research Institute, Hangzhou, 310006 Zhejiang, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yao-Fang%2BNiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Niu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYao-Fang%2BNiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Chengyin Ye </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Health Management, College of Medicine, Hangzhou Normal University, Hangzhou, 310021 Zhejiang, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chengyin%2BYe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ye%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChengyin%2BYe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ji He </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Neurology, Peking University Third Hospital, Beijing 100191, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ji%2BHe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=He%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJi%2BHe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Fang Han </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Pulmonary and Critical Care Medicine, Peking University People’s Hospital, Beijing 100044, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fang%2BHan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Han%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFang%2BHan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Long-Biao Guo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory of Rice Biology, China National Rice Research Institute, Hangzhou, 310006 Zhejiang, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Long-Biao%2BGuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guo%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALong-Biao%2BGuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Hou-Feng Zheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Institute of Aging Research, College of Medicine, Hangzhou Normal University, Hangzhou, 310021 Zhejiang, China</div><div class='author-affiliation'><span class='nlm-sup'>††</span>The Affiliated Hospital of Hangzhou Normal University, Hangzhou, 310015 Zhejiang, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hou-Feng%2BZheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zheng%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHou-Feng%2BZheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hfzheng@hznu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">hfzheng@hznu.edu.cn</a>
<a href="mailto:chenguobo@gmail.com" class="" data-icon-position="" data-hide-link-title="0">chenguobo@gmail.com</a></span></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Guo-Bo Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Evergreen Landscape and Architecture Studio, Hangzhou, 310026 Zhejiang, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guo-Bo%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGuo-Bo%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5475-8237" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Guo-Bo Chen</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hfzheng@hznu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">hfzheng@hznu.edu.cn</a>
<a href="mailto:chenguobo@gmail.com" class="" data-icon-position="" data-hide-link-title="0">chenguobo@gmail.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/3/943" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:483528" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:483528/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/3/943.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:483528" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:483528/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/3/943.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:483528" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:483528/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/3/943.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:483528" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:483528/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/3/943.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:483528" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:483528/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">In line with open-source genetics, we report a novel linear regression technique for genome-wide association studies (GWAS), called Open GWAS algoriTHm (OATH). When individual-level data are not available, OATH can not only completely reproduce reported results from an experimental model, but also recover underreported results from other alternative models with a different combination of nuisance parameters using naïve summary statistics (NSS). OATH can also reliably evaluate all reported results in-depth (<em>e.g.</em>, <em>p</em>-value variance analysis), as demonstrated for 42 <em>Arabidopsis</em> phenotypes under three magnesium (Mg) conditions. In addition, OATH can be used for consortium-driven genome-wide association meta-analyses (GWAMA), and can greatly improve the flexibility of GWAMA. A prototype of OATH is available in the Genetic Analysis Repository (<a href="https://github.com/gc5k/GEAR">https://github.com/gc5k/GEAR</a>).</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/gwas" class="hw-term hw-article-keyword hw-article-keyword-gwas" rel="nofollow">GWAS</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/arabidopsis" class="hw-term hw-article-keyword hw-article-keyword-arabidopsis" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Arabidopsis</em></a></li><li class="kwd"><a href="/keyword/magnesium" class="hw-term hw-article-keyword hw-article-keyword-magnesium" rel="nofollow">magnesium</a></li><li class="kwd"><a href="/keyword/transparency" class="hw-term hw-article-keyword hw-article-keyword-transparency" rel="nofollow">transparency</a></li><li class="kwd"><a href="/keyword/reproducibility" class="hw-term hw-article-keyword hw-article-keyword-reproducibility" rel="nofollow">reproducibility</a></li><li class="kwd"><a href="/keyword/na%C3%AFve-summary-statistics" class="hw-term hw-article-keyword hw-article-keyword-na&#xEF;ve-summary-statistics" rel="nofollow">naïve summary statistics</a></li><li class="kwd"><a href="/keyword/meta-analyses" class="hw-term hw-article-keyword hw-article-keyword-meta-analyses" rel="nofollow">meta-analyses</a></li><li class="kwd"><a href="/keyword/gwama" class="hw-term hw-article-keyword hw-article-keyword-gwama" rel="nofollow">GWAMA</a></li><li class="kwd"><a href="/keyword/gear" class="hw-term hw-article-keyword hw-article-keyword-gear" rel="nofollow">GEAR</a></li></ul><p id="p-4">Reproducibility and transparency are the cornerstones of scientific integrity. In addition to artifacts that may compromise a study, analysis itself is becoming more complicated and poses another obstacle to reproducing discoveries. For big data studies involving high-throughput computation, such as GWAS, the reported findings are subject to criticism, as the results may differ among models even when the experimental design is sound. Therefore, the choice of a model as well as its conclusion (<em>i.e.</em>, false positive or false negative) are often justified by an analyst’s prior knowledge (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aschard <em>et al.</em> 2015</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Day <em>et al.</em> 2016</a>). However, given practical constraints, such as data sharing policies and computational burden, it is not feasible to present all possible results found under alternative models. Although many consortia encourage open-source genetics and have released GWAS summary statistics, including the Genetic Investigation of Anthropometric Traits (GIANT) Consortium and the Psychiatric Genomic Consortia (PGC), it is still difficult to thoroughly evaluate a published study. Consequently, reproducibility and the success rate of subsequent studies are hampered. What kind of method and set of summary statistics are needed to fully reproduce results and to explore studies using unreported analyses?</p><p id="p-5">Statistical analyses can be reproduced in the absence of individual-level data; this is possible due to the theory of sufficient statistics (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Fisher 1921</a>). In this study, we propose a complementary method to reproduce each GWAS hit in the absence of shared original data. We report an algorithm called OATH that works directly on summary statistics. When individual-level data are not available, OATH can not only completely reproduce the reported results from an experimental model but also recover underreported results from other alternative models using only summary statistics. The utility of OATH will be demonstrated for 42 phenotypes: 14 traits of 295 <em>Arabidopsis</em> inbred lines grown under three Mg conditions.</p><p id="p-6">Furthermore, as OATH is based on linear regression, its application to other analyses is possible as long as linear regression was employed. For example, OATH can be embedded into consortium-driven GWAMA. Without loss of generality, the literature-driven meta-analyses can be considered a “retrospective” study, which is often an irreversible process under which a meta-analyses conductor can rarely customize the summary statistics. In contrast, a consortium-driven GWAMA can be a “prospective” study; quality control can be conducted more thoroughly (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chen <em>et al.</em> 2017</a>) and the summary statistics from each cohort can be customized under the request of the consortium. As demonstrated below with two Chinese GWAS cohorts, a consortium-driven GWAMA can more efficiently adjust covariates using OATH.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-7">We begin this section with a brief explanation of the OATH algorithm; a more detailed description can be found in the Supplemental Material. To demonstrate the use of OATH, an introduction of <em>Arabidopsis</em> GWAS data under Mg treatments and two Chinese GWAS cohorts will follow.</p><div id="sec-2" class="subsection"><h3>OATH</h3><p id="p-8">For a saturated GWAS analysis, its multiple regression model is written as (for the ease of discussion, all variables are centered, but the method can be applied to data not centered)<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>in which <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-2.gif"/></noscript></span></span> is the observed phenotype of <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-3.gif"/></noscript></span></span> individuals, <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-4.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-5.gif"/></noscript></span></span> is the residual. <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-6.gif"/></noscript></span></span> codes the counts of the reference alleles at the <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-7.gif"/></noscript></span></span> locus and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-8.gif"/></noscript></span></span> is the <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-9.gif"/></noscript></span></span> covariate. <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-10.gif"/></noscript></span></span> in which <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-11.gif"/></noscript></span></span> is the effect size of the marker and <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-12.gif"/></noscript></span></span> is the partial regression coefficient. The least-squares estimator is <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-13.gif"/></noscript></span></span> in which <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-14.gif"/></noscript></span></span> Both <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-15.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-16.gif"/></noscript></span></span> are individual-level data in the estimator.</p><p id="p-9">The least-squares estimator for <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-17.gif"/></noscript></span></span> can also be expressed in the following form (see Supplemental Material; hereafter referred to as OATH):<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-18.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>in which <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-19.gif"/></noscript></span></span> is the diagonal of <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-20.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-21.gif"/></noscript></span></span> in which <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-22.gif"/></noscript></span></span> is for <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-23.gif"/></noscript></span></span> The variance–covariance matrix of <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-24.gif"/></noscript></span></span> is<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-25.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>The information [known as sufficient statistics for data reduction (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Fisher 1921</a>)] required for <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">Equations 2</a> and <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">3</a> is contained in <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-26.gif"/></noscript></span></span> the variance–covariance matrix of all variables in <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a>; no individual-level data are needed. As illustrated in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, all elements for <a id="xref-disp-formula-2-2" class="xref-disp-formula" href="#disp-formula-2">Equations 2</a> and <a id="xref-disp-formula-3-2" class="xref-disp-formula" href="#disp-formula-3">3</a> can be extracted from <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-27.gif"/></noscript></span></span> Rather than summary statistics from complicated models, <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-28.gif"/></noscript></span></span> involves variance and covariance only; therefore, we call them NSS in the text below. Of note, as the second row/column of <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-29.gif"/></noscript></span></span> is locus-specific, only the locus-specific part of <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-30.gif"/></noscript></span></span> should be provided for each locus (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> and Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038877/-/DC1/FileS1.pdf">File S1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/943/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic illustration of open GWAS algorithm (OATH). The first row is the OATH equation. The second row shows (A) the sufficient statistics  a symmetric matrix, and (B)  can be split into generic G (red) and locus-specific  (yellow) parts. The third row represents how the elements in  can be extracted to build (C)  (D)  and (E)  respectively. GWAS, genome-wide association studies; OATH, Open GWAS algoriTHm." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1547837089" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Schematic illustration of open GWAS algorithm (OATH). The first row is the OATH equation. The second row shows (A) the sufficient statistics &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-31&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; a symmetric matrix, and (B) &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-32&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; can be split into generic &lt;em&gt;G&lt;/em&gt; (red) and locus-specific &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-33&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (yellow) parts. The third row represents how the elements in &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-34&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; can be extracted to build (C) &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-35&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (D) &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-36&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and (E) &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-37&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; respectively. GWAS, genome-wide association studies; OATH, Open GWAS algoriTHm.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/943/F1.medium.gif" width="440" height="304"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/3/943/F1.medium.gif" width="440" height="304"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/943/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/943/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483668" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">Schematic illustration of open GWAS algorithm (OATH). The first row is the OATH equation. The second row shows (A) the sufficient statistics <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F1/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F1/embed/mml-math-31.gif"/></noscript></span></span> a symmetric matrix, and (B) <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F1/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F1/embed/mml-math-32.gif"/></noscript></span></span> can be split into generic <em>G</em> (red) and locus-specific <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F1/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F1/embed/mml-math-33.gif"/></noscript></span></span> (yellow) parts. The third row represents how the elements in <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F1/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F1/embed/mml-math-34.gif"/></noscript></span></span> can be extracted to build (C) <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F1/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F1/embed/mml-math-35.gif"/></noscript></span></span> (D) <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F1/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F1/embed/mml-math-36.gif"/></noscript></span></span> and (E) <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F1/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F1/embed/mml-math-37.gif"/></noscript></span></span> respectively. GWAS, genome-wide association studies; OATH, Open GWAS algoriTHm.</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">In general, <a id="xref-disp-formula-2-3" class="xref-disp-formula" href="#disp-formula-2">Equation 2</a> can be written as <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-38.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-39.gif"/></noscript></span></span> indicating the set of covariates included. If any covariates are dropped from <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-40.gif"/></noscript></span></span> <a id="xref-disp-formula-2-4" class="xref-disp-formula" href="#disp-formula-2">Equations 2</a> and <a id="xref-disp-formula-3-3" class="xref-disp-formula" href="#disp-formula-3">3</a> can be tailored to generate a corresponding estimate for the target marker effect <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-41.gif"/></noscript></span></span> Thus, recovering underreported results for any combination of covariates is possible if the summary statistic <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-42.gif"/></noscript></span></span> is provided.</p><p id="p-12">One possible application for OATH is GWAMA. If each cohort sends <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-43.gif"/></noscript></span></span> to the central hub, the whole GWAMA gains more flexibility because the central hub will be able to customize the GWAS model to any combination of covariates. The technical details on how to integrate OATH into GWAMA can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038877/-/DC1/FileS1.pdf">File S1</a>.</p></div><div id="sec-3" class="subsection"><h3>Arabidopsis GWAS data</h3><p id="p-13">The seeds of all 295 lines were acquired from the <em>Arabidopsis</em> Biological Resources Center stock. Then, 234 accessions were sampled from 1307 worldwide accessions, which were genotyped using a 250 K single nucleotide polymorphism (SNP) chip (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Horton <em>et al.</em> 2012</a>), and 61 were extracted from the <em>Arabidopsis</em> 1001 Genomes Project (<a href="http://1001genomes.org">http://1001genomes.org</a>) (Figure S1A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038877/-/DC1/FileS2.pdf">File S2</a>). The geographical distribution of the 295 lines was consistent with the <em>Arabidopsis</em> lines collected in RegPanel (<a href="http://regmap.uchicago.edu">http://regmap.uchicago.edu</a>) (see Figure S1B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038877/-/DC1/FileS2.pdf">File S2</a>). After quality control [triallelic or tetra-allelic loci, minor allele frequency (MAF) &lt; 0.05, genotyping rate &lt; 0.998, and homozygosity rate &lt; 0.99 were removed], 156,744 biallelic loci remained for 42 GWAS (Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038877/-/DC1/FileS2.pdf">File S2</a>). Genetic relatedness was estimated using these 156,744 markers, resulting in a <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-44.gif"/></noscript></span></span> genetic relationship matrix (GRM). The eigenvectors were estimated in the GRM.</p><p id="p-14">The 295 inbred lines were grown under three Mg conditions: the low, normal, and high conditions contained 1, 1000, and 10,000 µM MgSO<sub>4</sub>, respectively, which was in accordance with the concentrations of Mg<sup>2+</sup> in soil solutions (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hariadi and Shabala 2004</a>). Fourteen traits were investigated under each treatment: seven were morphological traits and seven were nutrient concentration traits (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Under the three treatments, there were 42 total phenotypes for each line (Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038877/-/DC1/FileS2.pdf">File S2</a>). To reduce environmental influences, the median value of biological replicates was used as the phenotypic value. To reduce the maternal effects prior to phenotyping, inbred lines were grown for one generation under controlled greenhouse conditions at Zhejiang University (N30°18′25, E120°04′54), Hangzhou, Zhejiang Province, China, in 2015. For the ease of analysis, each phenotype was standardized (Figure S4 and Figure S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038877/-/DC1/FileS2.pdf">File S2</a>). See the supplementary notes in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038877/-/DC1/FileS1.pdf">File S1</a> for more details on these traits.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/483552/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/483552/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Fourteen <em>Arabidopsis</em> traits investigated under three magnesium (Mg) treatments</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Two Chinese GWAS cohorts</h3><p id="p-17">Two Chinese GWAS cohorts, NA (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Han <em>et al.</em> 2013</a>) and SLE (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Han <em>et al.</em> 2009</a>), were used to demonstrate the application of OATH to consortium-driven meta-analyses. The NA cohort was originally recruited for the study of narcolepsy, an autoimmune disorder affecting hypocretin (orexin) neurons; 3191 samples were genotyped. The SLE cohort was recruited for the study of systemic lupus erythematosus in the Chinese population; 2309 samples were genotyped. In order to mimic a consortium-driven GWAMA, these two GWAS cohorts provided the required NSS to the central hub. Using the meta-PCA technique (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Chen <em>et al.</em> 2017</a>), the general genotyping quality of these two cohorts was validated by the GWAMA central hub, based only on the reported allele frequencies; individual-level data were not required (Figure S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038877/-/DC1/FileS2.pdf">File S2</a>).</p></div><div id="sec-5" class="subsection"><h3>Data availability</h3><p id="p-18">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article.</p></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><div id="sec-7" class="subsection"><h3>An OATH simulation example</h3><p id="p-19">In order to demonstrate the OATH kernel, a single-locus analysis is shown. The MAF of the biallelic locus was 0.23, and the effect size was set to zero. Three covariates, each sampled from the standard normal distribution <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-45.gif"/></noscript></span></span> were simulated. The phenotype was sampled from <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-46.gif"/></noscript></span></span> The sample size was <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-47.gif"/></noscript></span></span> In this simulated sample, <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-48.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-49.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-50.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-51.gif"/></noscript></span></span> Including one, two, or three covariates, it generated seven possible models. The reproducibility of the partial regression coefficients estimated by OATH agreed well with those estimated from the individual-level data (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). An R script is available for this example at <a href="https://github.com/gc5k/OATH">https://github.com/gc5k/OATH</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/943/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Demonstration of OATH for a single locus. Three covariates were simulated, and OATH generated seven models by including one, two, or three covariates. The subtitle for each plot indicates the set of covariates included. The x-axis represents the regression coefficients estimated from OATH, whereas the y-axis shows the regression coefficients for the corresponding model but estimated from the individual-level data. The vertical and horizontal lines across each point indicate the SE of the partial regression coefficient estimated via OATH and individual-level data, respectively. The subtitle in each panel indicates the covariates included. OATH, Open GWAS algoriTHm." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1547837089" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Demonstration of OATH for a single locus. Three covariates were simulated, and OATH generated seven models by including one, two, or three covariates. The subtitle for each plot indicates the set of covariates included. The &lt;em&gt;x&lt;/em&gt;-axis represents the regression coefficients estimated from OATH, whereas the &lt;em&gt;y&lt;/em&gt;-axis shows the regression coefficients for the corresponding model but estimated from the individual-level data. The vertical and horizontal lines across each point indicate the SE of the partial regression coefficient estimated via OATH and individual-level data, respectively. The subtitle in each panel indicates the covariates included. OATH, Open GWAS algoriTHm.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/943/F2.medium.gif" width="440" height="250"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/3/943/F2.medium.gif" width="440" height="250"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/943/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/943/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483697" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">Demonstration of OATH for a single locus. Three covariates were simulated, and OATH generated seven models by including one, two, or three covariates. The subtitle for each plot indicates the set of covariates included. The <em>x</em>-axis represents the regression coefficients estimated from OATH, whereas the <em>y</em>-axis shows the regression coefficients for the corresponding model but estimated from the individual-level data. The vertical and horizontal lines across each point indicate the SE of the partial regression coefficient estimated via OATH and individual-level data, respectively. The subtitle in each panel indicates the covariates included. OATH, Open GWAS algoriTHm.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Two Arabidopsis GWAS models</h3><p id="p-21">For these 295 <em>Arabidopsis</em> lines, we conducted a GWAS for each of the 42 phenotypes in the saturated GWAS models, which included the top five eigenvectors (Figure S7 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038877/-/DC1/FileS2.pdf">File S2</a>). In contrast, we also conducted naïve/simple linear regressions (<em>i.e.</em>, no covariates) for these phenotypes, denoted as naïve GWAS (nGWAS) (Figure S8 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038877/-/DC1/FileS2.pdf">File S2</a>). Under the 42 sGWAS, <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-52.gif"/></noscript></span></span> a metric measuring population stratification (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Devlin and Roeder 1999</a>), had a mean of <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-53.gif"/></noscript></span></span> whereas that of the nGWAS was <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-54.gif"/></noscript></span></span> indicating adjustment of the covariates in differentiating GWAS outcomes. For each phenotype, the correlations of the estimated β (additive genetic effect) and <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-55.gif"/></noscript></span></span> between the sGWAS and the nGWAS were <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-56.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-57.gif"/></noscript></span></span> respectively (Figure S9 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038877/-/DC1/FileS2.pdf">File S2</a>). Using <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-58.gif"/></noscript></span></span> as the nominal genome-wide significance threshold, the sGWAS had 284 hits in total; 84, 84, and 116 under the low-, normal-, and high-Mg conditions, respectively. The nGWAS had 397 hits in total; 89, 188, and 120 under the low-, normal-, and high-Mg conditions, respectively. Between the sGWAS and the nGWAS, 206 hits were shared (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). As demonstrated in this example, an alternative model could lead to different results, which might cause controversy over reproducibility.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/943/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="sGWAS and GWAS hits for 14 traits under three Mg conditions. The x-axis shows the chromosomal coordinates for Arabidopsis; the y-axis represents 14 traits. The GWAS hits observed in the saturated model (sGWAS), which was adjusted by the top five eigenvectors, are presented on the top; the hits observed in the na&#xEF;ve model (nGWAS) are represented in the bottom panel. GWAS hits are represented by black circles; and those hits shared by both nGWAS and sGWAS are filled with color. A total of 51, 74, and 81 GWAS hits were shared under low-, normal-, and high-Mg conditions, respectively. A GWAS hit was defined as  in which the number of markers  was 156,744. GWAS, genome-wide association studies; Mg, magnesium; nGWAS, na&#xEF;ve GWAS; sGWAS, saturated GWAS." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1547837089" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;sGWAS and GWAS hits for 14 traits under three Mg conditions. The &lt;em&gt;x&lt;/em&gt;-axis shows the chromosomal coordinates for &lt;em&gt;Arabidopsis&lt;/em&gt;; the &lt;em&gt;y&lt;/em&gt;-axis represents 14 traits. The GWAS hits observed in the saturated model (sGWAS), which was adjusted by the top five eigenvectors, are presented on the top; the hits observed in the na&#xEF;ve model (nGWAS) are represented in the bottom panel. GWAS hits are represented by black circles; and those hits shared by both nGWAS and sGWAS are filled with color. A total of 51, 74, and 81 GWAS hits were shared under low-, normal-, and high-Mg conditions, respectively. A GWAS hit was defined as &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-59&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; in which the number of markers &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-60&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; was 156,744. GWAS, genome-wide association studies; Mg, magnesium; nGWAS, na&#xEF;ve GWAS; sGWAS, saturated GWAS.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/943/F3.medium.gif" width="440" height="347"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/3/943/F3.medium.gif" width="440" height="347"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/943/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/943/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483673" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-22" class="first-child">sGWAS and GWAS hits for 14 traits under three Mg conditions. The <em>x</em>-axis shows the chromosomal coordinates for <em>Arabidopsis</em>; the <em>y</em>-axis represents 14 traits. The GWAS hits observed in the saturated model (sGWAS), which was adjusted by the top five eigenvectors, are presented on the top; the hits observed in the naïve model (nGWAS) are represented in the bottom panel. GWAS hits are represented by black circles; and those hits shared by both nGWAS and sGWAS are filled with color. A total of 51, 74, and 81 GWAS hits were shared under low-, normal-, and high-Mg conditions, respectively. A GWAS hit was defined as <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F3/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F3/embed/mml-math-59.gif"/></noscript></span></span> in which the number of markers <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F3/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F3/embed/mml-math-60.gif"/></noscript></span></span> was 156,744. GWAS, genome-wide association studies; Mg, magnesium; nGWAS, naïve GWAS; sGWAS, saturated GWAS.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Reproducing sGWAS for Arabidopsis</h3><p id="p-23">In order to reproduce the sGWAS results for each SNP in the absence of shared original data, for each phenotype, the following NSS were used: the variance–covariance matrix of a phenotype, five eigenvectors, and information from 156,744 specific loci were also provided. Of note, the covariance matrix of the five eigenvectors was a diagonal matrix because the eigenvectors were mutually orthogonal.</p><p id="p-24">As expected, OATH synthesized the NSS as prepared above to reproduce the 42 sGWAS with high precision, as illustrated for days to root germination (RGT) (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>), as well as for the 41 phenotypes (Figure S10 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038877/-/DC1/FileS2.pdf">File S2</a>). For 14 traits under the normal-Mg condition, the consistency between the estimated β from OATH and those from the sGWAS was <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-61.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-62.gif"/></noscript></span></span> OATH found the same 284 hits that were found in the sGWAS. This indicated that, even without access to the individual-level data, OATH could retrospectively scrutinize the reported results. Furthermore, we also conducted individual-level data GWAS for these 295 <em>Arabidopsis</em> lines by including the top 10 eigenvectors, leading to <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-63.gif"/></noscript></span></span> possible outcomes for the association between a phenotype and a marker. OATH also almost perfectly reproduced the results (data not shown).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/943/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Reproducibility of sGWAS results using OATH with NSS for RTG under three Mg conditions. Each column represents &#x3B2; (top) and p-values (bottom) under low-, normal-, and high-Mg conditions. The y-axis represents the statistics from OATH synthesized from NSS, and the x-axis from the sGWAS; the red points are QTL detected in the sGWAS. Correlations are shown in the top left corner of each panel. GWAS, genome-wide association studies; Mg, magnesium; NSS, na&#xEF;ve summary statistics; OATH, Open GWAS algoriTHm; QTL, quantitative trait loci; RTG, days to root germination; sGWAS, saturated GWAS." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1547837089" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Reproducibility of sGWAS results using OATH with NSS for RTG under three Mg conditions. Each column represents &#x3B2; (top) and &lt;em&gt;p&lt;/em&gt;-values (bottom) under low-, normal-, and high-Mg conditions. The &lt;em&gt;y&lt;/em&gt;-axis represents the statistics from OATH synthesized from NSS, and the &lt;em&gt;x&lt;/em&gt;-axis from the sGWAS; the red points are QTL detected in the sGWAS. Correlations are shown in the top left corner of each panel. GWAS, genome-wide association studies; Mg, magnesium; NSS, na&#xEF;ve summary statistics; OATH, Open GWAS algoriTHm; QTL, quantitative trait loci; RTG, days to root germination; sGWAS, saturated GWAS.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/943/F4.medium.gif" width="440" height="254"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/3/943/F4.medium.gif" width="440" height="254"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/943/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/943/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483566" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-25" class="first-child">Reproducibility of sGWAS results using OATH with NSS for RTG under three Mg conditions. Each column represents β (top) and <em>p</em>-values (bottom) under low-, normal-, and high-Mg conditions. The <em>y</em>-axis represents the statistics from OATH synthesized from NSS, and the <em>x</em>-axis from the sGWAS; the red points are QTL detected in the sGWAS. Correlations are shown in the top left corner of each panel. GWAS, genome-wide association studies; Mg, magnesium; NSS, naïve summary statistics; OATH, Open GWAS algoriTHm; QTL, quantitative trait loci; RTG, days to root germination; sGWAS, saturated GWAS.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Recovering underreported results for Arabidopsis</h3><p id="p-26">These 295 <em>Arabidopsis</em> lines resulted in the generation of <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-64.gif"/></noscript></span></span> models, given all possible combinations of the five eigenvectors. With the inclusion or exclusion of certain eigenvectors, OATH was capable of synthesizing another 30 GWAS that had at least one of the five eigenvectors as covariates. For each of the 42 phenotypes, an OATH hit was claimed if a SNP had any of its 32 models in which the OATH <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-65.gif"/></noscript></span></span> OATH found 637 hits for 42 phenotypes; 163 hits were not found by either the sGWAS or the nGWAS. Of these 163 new hits, 25 had <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-66.gif"/></noscript></span></span> indicating a nominal overall significance under 42 phenotypes and 32 models. We validated these OATH hits by implementing their exact models using individual-level data from the 295 <em>Arabidopsis</em> lines; the consistency of the β and <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-67.gif"/></noscript></span></span> was <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-68.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-69.gif"/></noscript></span></span> respectively (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Therefore, OATH found all possible underreported results with high consistency. These 637 OATH hits were found on 575 unique SNPs, for which 430 were within genes and 145 were between genic regions.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/943/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Validation of OATH hits by their exact models. A total of 637 OATH hits, which had a  were found; 25 loci had a  Within each panel, the positive part of the y-axis represents the  observed in OATH; the negative part of the y-axis is the corresponding  evaluated by their exact models using individual-level data from 295 Arabidopsis lines. OATH, Open GWAS algoriTHm." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1547837089" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Validation of OATH hits by their exact models. A total of 637 OATH hits, which had a &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-70&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; were found; 25 loci had a &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-71&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; Within each panel, the positive part of the &lt;em&gt;y&lt;/em&gt;-axis represents the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-72&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; observed in OATH; the negative part of the &lt;em&gt;y&lt;/em&gt;-axis is the corresponding &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-73&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; evaluated by their exact models using individual-level data from 295 &lt;em&gt;Arabidopsis&lt;/em&gt; lines. OATH, Open GWAS algoriTHm.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/943/F5.medium.gif" width="440" height="379"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/3/943/F5.medium.gif" width="440" height="379"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/943/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/943/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483572" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-27" class="first-child">Validation of OATH hits by their exact models. A total of 637 OATH hits, which had a <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F5/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F5/embed/mml-math-70.gif"/></noscript></span></span> were found; 25 loci had a <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F5/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F5/embed/mml-math-71.gif"/></noscript></span></span> Within each panel, the positive part of the <em>y</em>-axis represents the <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F5/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F5/embed/mml-math-72.gif"/></noscript></span></span> observed in OATH; the negative part of the <em>y</em>-axis is the corresponding <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F5/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F5/embed/mml-math-73.gif"/></noscript></span></span> evaluated by their exact models using individual-level data from 295 <em>Arabidopsis</em> lines. OATH, Open GWAS algoriTHm.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>In-depth evaluation of the GWAS hits for Arabidopsis</h3><p id="p-28">In the experimental design theory established by R. Fisher, a single high/low value, such as productivity in a field experiment, is often confounded by a combination of other factors (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Fisher 1926</a>) of little interest when compared with the values under different factors. Therefore, we further investigated whether the combination of the eigenvectors influenced each OATH hit.</p><p id="p-29">For those 637 OATH hits, the smallest range of 32 <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-74.gif"/></noscript></span></span> from 7.01 to 7.14, was found for SNP 3_8965883 (chromosome 3, 8,965,883 bp, and MAF = 0.0508) associated with sulfur under the low-Mg condition. SNP 3_8965883 was located within <em>RASPBERRY</em> 3 (<em>RSY3</em>), a gene related to embryogenesis (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Apuya <em>et al.</em> 2002</a>) (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Across the 32 models, its βs and SEs remained relatively stable (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/483565/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/483565/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Three single nucleotide polymorphism (SNP) examples from <em>Arabidopsis</em> inbred lines</span><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/943/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Evaluation of the modeling for three OATH hits. The top four rows represent    and  estimated using 32 possible OATH models given five covariates for these three SNPs.    and  are in ascending order according to their  In the top row, the corresponding OATH models are denoted by colored squares, indicating the inclusion or exclusion of covariates; there are two bars, gray and pink, in each cluster, representing  with or without adjustment for  An asterisk indicates that this (SNP) is significant under the corresponding model without adjustment for  three asterisks indicate that this SNP is significant under the corresponding model, both with and without adjustment for  OATH, Open GWAS algoriTHm; SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1547837089" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Evaluation of the modeling for three OATH hits. The top four rows represent &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-88&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-89&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-90&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-91&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; estimated using 32 possible OATH models given five covariates for these three SNPs. &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-92&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-93&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-94&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-95&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; are in ascending order according to their &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-96&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; In the top row, the corresponding OATH models are denoted by colored squares, indicating the inclusion or exclusion of covariates; there are two bars, gray and pink, in each cluster, representing &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-97&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; with or without adjustment for &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-98&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; An asterisk indicates that this (SNP) is significant under the corresponding model without adjustment for &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-99&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; three asterisks indicate that this SNP is significant under the corresponding model, both with and without adjustment for &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-100&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; OATH, Open GWAS algoriTHm; SNP, single nucleotide polymorphism.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/943/F6.medium.gif" width="440" height="228"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/3/943/F6.medium.gif" width="440" height="228"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/943/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/943/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483589" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-32" class="first-child">Evaluation of the modeling for three OATH hits. The top four rows represent <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F6/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F6/embed/mml-math-88.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F6/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F6/embed/mml-math-89.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F6/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F6/embed/mml-math-90.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F6/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F6/embed/mml-math-91.gif"/></noscript></span></span> estimated using 32 possible OATH models given five covariates for these three SNPs. <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F6/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F6/embed/mml-math-92.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F6/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F6/embed/mml-math-93.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F6/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F6/embed/mml-math-94.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F6/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F6/embed/mml-math-95.gif"/></noscript></span></span> are in ascending order according to their <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F6/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F6/embed/mml-math-96.gif"/></noscript></span></span> In the top row, the corresponding OATH models are denoted by colored squares, indicating the inclusion or exclusion of covariates; there are two bars, gray and pink, in each cluster, representing <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F6/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F6/embed/mml-math-97.gif"/></noscript></span></span> with or without adjustment for <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F6/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F6/embed/mml-math-98.gif"/></noscript></span></span> An asterisk indicates that this (SNP) is significant under the corresponding model without adjustment for <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F6/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F6/embed/mml-math-99.gif"/></noscript></span></span> three asterisks indicate that this SNP is significant under the corresponding model, both with and without adjustment for <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F6/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F6/embed/mml-math-100.gif"/></noscript></span></span> OATH, Open GWAS algoriTHm; SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">In contrast, the largest range of <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-101.gif"/></noscript></span></span> from 3.19 to 12.69 (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>), was found for SNP 5_200100406 (chromosome 5, 200,100,406 bp, and MAF = 0.058) associated with <em>K</em> under the high-Mg condition. SNP 5_200100406 was located within <em>AT5G49350</em>, a gene encoding glycine-rich protein (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Tabata <em>et al.</em> 2000</a>) (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Of its 32 <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-102.gif"/></noscript></span></span> 16 were &gt; 6.5. We partitioned its 32 sorted <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-103.gif"/></noscript></span></span> into different groups if any two neighboring <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-104.gif"/></noscript></span></span> differed by a unit. Its 32 <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-105.gif"/></noscript></span></span> could be split into three groups (<em>F</em>-statistic = 434.06 and <em>p</em>-value &lt; 1e−16). The four OATH models in the highest <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-106.gif"/></noscript></span></span> group included the first, second, and fourth eigenvectors (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). Its βs were increased in the highest group but the corresponding SEs decreased, resulting in a much higher <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-107.gif"/></noscript></span></span></p><p id="p-34">In another example, SNP 4_6353940, associated with RGT under the high-Mg condition, had its 32 <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-108.gif"/></noscript></span></span> partitioned into two groups via inclusion or exclusion of the second eigenvector (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>). SNP 4_6353940 had a MAF of 0.0507 and was located within <em>AT4G10200</em>, a gene related to TTF-type zinc finger proteins with a HAT dimerization domain (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Mayer <em>et al.</em> 1999</a>) (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). Inclusion or exclusion of the second eigenvector also resulted in two groups for the β. Among 637 OATH hits, this SNP had the most significant difference for its <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-109.gif"/></noscript></span></span> group, and the <em>F</em>-statistic was 5168.142 (<em>p</em>-value &lt; 1e−16).</p><p id="p-35">An R script is available at <a href="https://github.com/gc5k/OATH">https://github.com/gc5k/OATH</a> for the demonstrated <em>Arabidopsis</em> analyses with OATH.</p></div><div id="sec-12" class="subsection"><h3>Application of OATH to GWAMA</h3><p id="p-36">Two Chinese GWAS datasets, the NA (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Han <em>et al.</em> 2013</a>) and SLE cohorts (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Han <em>et al.</em> 2009</a>), were used to confirm the utility of OATH for meta-analyses. From these two cohorts, 9124 common variants on chromosome 1 in both cohorts were analyzed in NA (3191 samples) and SLE (2309 samples), respectively. For both cohorts, the SNPs were aligned on the same reference alleles. SNP rs4144542 was set as the causal locus explaining 5% of the total phenotypic variation. Three eigenvectors were used as covariates. In order to mimic a real consortium-driven GWAMA, one author (HFZ) generated NSS for these two GWAS cohorts; another author (GBC), who was blind to the individual-level data, ran OATH and the meta-analyses. After receiving <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-110.gif"/></noscript></span></span> the central hub synthesized seven corresponding <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-111.gif"/></noscript></span></span> given <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/embed/mml-math-112.gif"/></noscript></span></span> consequently, meta-analyses could be implemented for each locus. As demonstrated in <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>, rs4144542 was successfully identified in all seven GWAMA analyses. Other loci had very similar estimated effects under these seven models.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/943/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide association meta-analyses (GWAMA) of the NA and SLE cohorts. The subtitle in each panel indicates a customized GWAMA. For example,  indicates that the first and third eigenvectors are covariates for each of the two cohorts. The dashed line indicates the chromosome-wise threshold, given " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1547837089" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genome-wide association meta-analyses (GWAMA) of the NA and SLE cohorts. The subtitle in each panel indicates a customized GWAMA. For example, &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-113&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; indicates that the first and third eigenvectors are covariates for each of the two cohorts. The dashed line indicates the chromosome-wise threshold, given &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-114&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/943/F7.medium.gif" width="440" height="279"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/3/943/F7.medium.gif" width="440" height="279"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/943/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/943/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483647" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-37" class="first-child">Genome-wide association meta-analyses (GWAMA) of the NA and SLE cohorts. The subtitle in each panel indicates a customized GWAMA. For example, <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F7/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F7/embed/mml-math-113.gif"/></noscript></span></span> indicates that the first and third eigenvectors are covariates for each of the two cohorts. The dashed line indicates the chromosome-wise threshold, given <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F7/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/943/F7/embed/mml-math-114.gif"/></noscript></span></span></p><div class="sb-div caption-clear"></div></div></div><p id="p-38">An R script is available for this GWAMA demonstration at <a href="https://github.com/gc5k/OATH">https://github.com/gc5k/OATH</a>.</p></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-39">The scientific community is seeking reproducibility, and efforts have been made to improve reproducibility as well as transparency. Reproducibility may vary among studies; however, false discovery due to controversial or improper modeling can be monitored and even avoided, as demonstrated for the 295 <em>Arabidopsis</em> lines. Since the establishment of experimental design theory for field experiments (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Fisher 1926</a>), it has been known that a single outcome may be confounded, such as nutrition level factors. A high or low outcome makes little sense when it departs from its context, such as the conditions that led to the observed extreme values. In particular, as justification for the inclusion of covariates is controversial, variation in studies due to modeling makes reproducibility challenging (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Aschard <em>et al.</em> 2015</a>). As GWAS results are often reported using a particular model, the interpretation of a GWAS hit should be reasonably scrutinized, as demonstrated in this study.</p><p id="p-40">We developed OATH and demonstrated its utility in GWAS of 295 <em>Arabidopsis</em> inbred lines. OATH successfully reproduced the GWAS results generated from a model with five covariates. In addition, underreported results, possibly generated by alternative models, were recovered. Given these comprehensive results, we could evaluate GWAS hits more thoroughly. As OATH is based on summary statistics, this implementation was compatible with GWAS data sharing policy, including those involving human subjects. For <em>Arabidopsis</em>, a typical admixed population, a linear mixed model technique provides an alternative solution (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Korte <em>et al.</em> 2012</a>); however, the complicated statistical properties of linear mixed models (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Chen 2014</a>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">2016</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">de los Campos <em>et al.</em> 2015</a>) may be beyond OATH’s linear regression model capabilities.</p><p id="p-41">Given the many possible ways to utilize OATH, GWAMA would most likely benefit from OATH integration. Using OATH, GWAMA would be more efficient at switching from one GWAS model to another whenever necessary, a procedure that often leads to logistical burden under a conventional GWAMA design. Many consortia that encourage open-source genetics have released GWAS summary statistics, such as GIANT and PGC. If those consortia would also release the naïve summary data required by OATH, efficiency and reproducibility can be dramatically boosted and the utility of the GWAS data maximized because the recovery of underreported GWAS discoveries becomes possible, as demonstrated in our study.</p><p id="p-42">In summary, in line with the open-source movement, we believe that reproducibility, transparency, and in-depth evaluation of GWAS are possible or can be improved using the proposed method. OATH as a solution is simple and easily embedded into other applications, and the information technology seems mature enough for implementation. To facilitate application of the proposed method, we deposited OATH in Genetic Analysis Repository (GEAR; <a href="https://github.com/gc5k/GEAR">https://github.com/gc5k/GEAR</a>). Three “one-click-for-all” R scripts for the demonstrated examples are available at <a href="https://github.com/gc5k/OATH">https://github.com/gc5k/OATH</a>.</p></div><div class="section" id="sec-14"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-43">We thank the associate editor and two reviewers for constructive comments that significantly improved the quality of the manuscript. This work was supported by the Natural Science Foundation of China (31601277 to Y.-F.N.; 81402762 to C.Y.; 81601105 to J.H.; and 81501145 to H.-F.Z.), ninth special grants for postdoctoral research (2016T90156 to Y.-F.N.), the China Postdoctoral Science Foundation (2014M551754 and 2015M581216 to Y.-F.N.), and the Zhejiang Provincial Natural Science Foundation for Distinguished Young Scholars of China (LR17H070001 to H.-F.Z.). We thank Hangzhou Guhe Information and Technology Co., Ltd., for bioinformatics assistance. The authors declare no competing financial interests.</p><p id="p-44">Author contributions: G.-B.C. and Y.-F.N. conceived and designed the study. G.-B.C. developed the theory, performed the <em>Arabidopsis</em> GWAS analysis, GWAMA, and developed GEAR::OATH. Y.-F.N. performed the material collection and <em>Arabidopsis</em> experimental operations, wrote the protocol for the material growth, and conducted phenotype analysis. F.H. and H.-F.Z. cleaned and provided the naïve summary statistics of the NA and SLE cohorts. C.Y. prepared the R scripts for demonstration. G.-B.C. and Y.-F.N. wrote the manuscript. J.H. and L.-B.G. contributed to the improving of the study and manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-6"><p id="p-45">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038877/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038877/-/DC1</a>.</p></li><li class="fn" id="fn-7"><p id="p-46"><em>Communicating editor: G. A. de los Campos</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-08-16"><span class="received-label">Received </span>August 16, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-01-16"><span class="accepted-label">Accepted </span>January 16, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Niu <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.943.1" data-doi="10.1104/pp.004010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Apuya</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yadegari</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harada</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldberg</span>  <span class="cit-name-given-names">R. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">RASPBERRY3 gene encodes a novel protein important for embryo development.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">691</span>–<span class="cit-lpage">705</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.004010%26rft_id%253Dinfo%253Apmid%252F12068112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxMjkvMi82OTEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzMvOTQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.943.2" data-doi="10.1016/j.ajhg.2014.12.021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aschard</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilhjalmsson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joshi</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kraft</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Adjusting for heritable covariates can bias effect estimates in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">329</span>–<span class="cit-lpage">339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D96%26rft.spage%253D329%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2014.12.021%26rft_id%253Dinfo%253Apmid%252F25640676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2014.12.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25640676&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F943.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.943.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">G.-B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Estimating heritability of complex traits from genome-wide association studies using IBS-based Haseman–Elston regression.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">107</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D5%26rft.spage%253D107%26rft_id%253Dinfo%253Apmid%252F24817879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24817879&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F943.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.943.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">G.-B.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">On the reconciliation of missing heritability for GWAS data.</span> <abbr class="cit-jnl-abbrev">Eur. J. Hum. Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1810</span>–<span class="cit-lpage">1816</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEur.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D24%26rft.spage%253D1810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.943.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">G.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trzaskowski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Across-cohort QC analyses of GWAS summary statistics from complex traits.</span> <abbr class="cit-jnl-abbrev">Eur. J. Hum. Genet.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">137</span>–<span class="cit-lpage">146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEur.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D25%26rft.spage%253D137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.943.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Day</span>  <span class="cit-name-given-names">F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loh</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ong</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perry</span>  <span class="cit-name-given-names">J. R. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A robust example of collider bias in a genetic association study.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">392</span>–<span class="cit-lpage">393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D98%26rft.spage%253D392%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.943.7" data-doi="10.1371/journal.pgen.1005048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic heritability: what is it?</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005048</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005048%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005048%26rft_id%253Dinfo%253Apmid%252F25942577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25942577&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F943.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.943.8" data-doi="10.1111/j.0006-341X.1999.00997.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Devlin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Genomic control for association studies.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">997</span>–<span class="cit-lpage">1004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DDevlin%26rft.auinit1%253DB.%26rft.volume%253D55%26rft.issue%253D4%26rft.spage%253D997%26rft.epage%253D1004%26rft.atitle%253DGenomic%2Bcontrol%2Bfor%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0006-341X.1999.00997.x%26rft_id%253Dinfo%253Apmid%252F11315092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0006-341X.1999.00997.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11315092&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F943.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084218000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.943.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1921</span> <span class="cit-article-title">On the mathematical foundations of theoretical statistics.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond.</abbr> <span class="cit-vol">222</span>: <span class="cit-fpage">309</span>–<span class="cit-lpage">368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BLond.%26rft.volume%253D222%26rft.spage%253D309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.943.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1926</span> <span class="cit-article-title">The arrangement of field experiments.</span> <abbr class="cit-jnl-abbrev">J. Minist. Agric. Gt. Britain</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">503</span>–<span class="cit-lpage">513</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMinist.%2BAgric.%2BGt.%2BBritain%26rft.volume%253D33%26rft.spage%253D503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.943.11" data-doi="10.1371/journal.pgen.1003880"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faraco</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">X. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ollila</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome wide analysis of narcolepsy in China implicates novel immune loci and reveals changes in association prior to <em>vs.</em> after the 2009 H1N1 influenza pandemic.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003880</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003880%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003880%26rft_id%253Dinfo%253Apmid%252F24204295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003880&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24204295&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F943.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.943.12" data-doi="10.1038/ng.472"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">J.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">H.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">L.-D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">D.-Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome-wide association study in a Chinese Han population identifies nine new susceptibility loci for systemic lupus erythematosus.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">1234</span>–<span class="cit-lpage">1237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHan%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D41%26rft.issue%253D11%26rft.spage%253D1234%26rft.epage%253D1237%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bin%2Ba%2BChinese%2BHan%2Bpopulation%2Bidentifies%2Bnine%2Bnew%2Bsusceptibility%2Bloci%2Bfor%2Bsystemic%2Blupus%2Berythematosus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.472%26rft_id%253Dinfo%253Apmid%252F19838193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.472&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19838193&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F943.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271247600017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.943.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hariadi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shabala</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Screening broad beans (Vicia faba) for magnesium deficiency. II. Photosynthetic performance and leaf bioelectrical responses.</span> <abbr class="cit-jnl-abbrev">Funct. Plant Biol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">539</span>–<span class="cit-lpage">539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunct.%2BPlant%2BBiol.%26rft.volume%253D31%26rft.spage%253D539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.943.14" data-doi="10.1038/ng.1042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hancock</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toomajian</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide patterns of genetic variation in worldwide Arabidopsis thaliana accessions from the RegMap panel.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">212</span>–<span class="cit-lpage">216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHorton%26rft.auinit1%253DM.%2BW.%26rft.volume%253D44%26rft.issue%253D2%26rft.spage%253D212%26rft.epage%253D216%26rft.atitle%253DGenome-wide%2Bpatterns%2Bof%2Bgenetic%2Bvariation%2Bin%2Bworldwide%2BArabidopsis%2Bthaliana%2Baccessions%2Bfrom%2Bthe%2BRegMap%2Bpanel.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.1042%26rft_id%253Dinfo%253Apmid%252F22231484%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.1042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22231484&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F943.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.943.15" data-doi="10.1038/ng.2376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korte</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilhjálmsson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Segura</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A mixed-model approach for genome-wide association studies of correlated traits in structured populations.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1066</span>–<span class="cit-lpage">1071</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKorte%26rft.auinit1%253DA.%26rft.volume%253D44%26rft.issue%253D9%26rft.spage%253D1066%26rft.epage%253D1071%26rft.atitle%253DA%2Bmixed-model%2Bapproach%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies%2Bof%2Bcorrelated%2Btraits%2Bin%2Bstructured%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2376%26rft_id%253Dinfo%253Apmid%252F22902788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22902788&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F943.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.943.16" data-doi="10.1038/47134"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mayer</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schüller</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wambutt</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Volckaert</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Sequence and analysis of chromosome 4 of the plant Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">402</span>: <span class="cit-fpage">769</span>–<span class="cit-lpage">777</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMayer%26rft.auinit1%253DK.%26rft.volume%253D402%26rft.issue%253D6763%26rft.spage%253D769%26rft.epage%253D777%26rft.atitle%253DSequence%2Band%2Banalysis%2Bof%2Bchromosome%2B4%2Bof%2Bthe%2Bplant%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F47134%26rft_id%253Dinfo%253Apmid%252F10617198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/47134&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10617198&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F943.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084330500054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.943.17" data-doi="10.1038/35048507"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tabata</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaneko</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakamura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kotani</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kato</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Sequence and analysis of chromosome 5 of the plant Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">408</span>: <span class="cit-fpage">823</span>–<span class="cit-lpage">826</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DTabata%26rft.auinit1%253DS.%26rft.volume%253D408%26rft.issue%253D6814%26rft.spage%253D823%26rft.epage%253D826%26rft.atitle%253DSequence%2Band%2Banalysis%2Bof%2Bchromosome%2B5%2Bof%2Bthe%2Bplant%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35048507%26rft_id%253Dinfo%253Apmid%252F11130714%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35048507&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11130714&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F943.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165831300040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/3/943.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/3/935" title="Single-Step BLUP with Varying Genotyping Effort in Open-Pollinated Picea glauca" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/3/953" title="Genome-Wide Analysis of lncRNA and mRNA Expression During Differentiation of Abdominal Preadipocytes in the Chicken" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/3"><span>Volume 7 Issue 3, March 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/3.cover-source.jpg?itok=h8yj2GBo" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/3/943" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-T4zlUCr8beCV1I5phpcCzuS_Wv1uHnlJAmm8ZNQNV2g" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/3/943" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/3/943" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/3/943" class="active" data-icon-position="" data-hide-link-title="0">Reproduction and In-Depth Evaluation of Genome-Wide Association Studies and Genome-Wide Meta-analyses Using Summary Statistics</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/483528" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-JafJJd77m1Klbg-NVNPpdA8amiWTpjLSUdw6RYCGB6k" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/3/943.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/3/943" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-z3rpkEvxDZWnoYZF3nFarXjkYqU6Jycv0RmRx2RnER4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/3/943" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/3/943.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483528" id="node483528--2" data-pisa="ggg;7/3/943" data-pisa-master="ggg;g3.116.038877" data-apath="/ggg/7/3/943.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Reproduction and In-Depth Evaluation of Genome-Wide Association Studies and Genome-Wide Meta-analyses Using Summary Statistics</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yao-Fang Niu</span>, <span class="highwire-citation-author" data-delta="1">Chengyin Ye</span>, <span class="highwire-citation-author" data-delta="2">Ji He</span>, <span class="highwire-citation-author" data-delta="3">Fang Han</span>, <span class="highwire-citation-author" data-delta="4">Long-Biao Guo</span>, <span class="highwire-citation-author" data-delta="5">Hou-Feng Zheng</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-5475-8237" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Guo-Bo Chen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">943-952;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.038877 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483528--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yao-Fang Niu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory of Rice Biology, China National Rice Research Institute, Hangzhou, 310006 Zhejiang, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yao-Fang%2BNiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Niu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYao-Fang%2BNiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Chengyin Ye </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Health Management, College of Medicine, Hangzhou Normal University, Hangzhou, 310021 Zhejiang, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chengyin%2BYe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ye%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChengyin%2BYe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ji He </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Neurology, Peking University Third Hospital, Beijing 100191, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ji%2BHe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=He%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJi%2BHe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Fang Han </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Pulmonary and Critical Care Medicine, Peking University People’s Hospital, Beijing 100044, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fang%2BHan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Han%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFang%2BHan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Long-Biao Guo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory of Rice Biology, China National Rice Research Institute, Hangzhou, 310006 Zhejiang, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Long-Biao%2BGuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guo%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALong-Biao%2BGuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Hou-Feng Zheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Institute of Aging Research, College of Medicine, Hangzhou Normal University, Hangzhou, 310021 Zhejiang, China</div><div class='author-affiliation'><span class='nlm-sup'>††</span>The Affiliated Hospital of Hangzhou Normal University, Hangzhou, 310015 Zhejiang, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hou-Feng%2BZheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zheng%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHou-Feng%2BZheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hfzheng@hznu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">hfzheng@hznu.edu.cn</a>
<a href="mailto:chenguobo@gmail.com" class="" data-icon-position="" data-hide-link-title="0">chenguobo@gmail.com</a></span></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Guo-Bo Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Evergreen Landscape and Architecture Studio, Hangzhou, 310026 Zhejiang, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guo-Bo%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGuo-Bo%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5475-8237" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Guo-Bo Chen</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hfzheng@hznu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">hfzheng@hznu.edu.cn</a>
<a href="mailto:chenguobo@gmail.com" class="" data-icon-position="" data-hide-link-title="0">chenguobo@gmail.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/3/943" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/3/943&amp;title=Reproduction%20and%20In-Depth%20Evaluation%20of%20Genome-Wide%20Association%20Studies%20and%20Genome-Wide%20Meta-analyses%20Using%20Summary%20Statistics" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/3/943&amp;title=Reproduction%20and%20In-Depth%20Evaluation%20of%20Genome-Wide%20Association%20Studies%20and%20Genome-Wide%20Meta-analyses%20Using%20Summary%20Statistics" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/3/943&amp;title=Reproduction%20and%20In-Depth%20Evaluation%20of%20Genome-Wide%20Association%20Studies%20and%20Genome-Wide%20Meta-analyses%20Using%20Summary%20Statistics" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/3/943&amp;text=Reproduction%20and%20In-Depth%20Evaluation%20of%20Genome-Wide%20Association%20Studies%20and%20Genome-Wide%20Meta-analyses%20Using%20Summary%20Statistics" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/3/943&amp;title=Reproduction%20and%20In-Depth%20Evaluation%20of%20Genome-Wide%20Association%20Studies%20and%20Genome-Wide%20Meta-analyses%20Using%20Summary%20Statistics" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/3/943&amp;t=Reproduction%20and%20In-Depth%20Evaluation%20of%20Genome-Wide%20Association%20Studies%20and%20Genome-Wide%20Meta-analyses%20Using%20Summary%20Statistics" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/3/943&amp;title=Reproduction%20and%20In-Depth%20Evaluation%20of%20Genome-Wide%20Association%20Studies%20and%20Genome-Wide%20Meta-analyses%20Using%20Summary%20Statistics" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/3/943&amp;title=Reproduction%20and%20In-Depth%20Evaluation%20of%20Genome-Wide%20Association%20Studies%20and%20Genome-Wide%20Meta-analyses%20Using%20Summary%20Statistics" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/483528/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483528" id="node483528--3" data-pisa="ggg;7/3/943" data-pisa-master="ggg;g3.116.038877" data-apath="/ggg/7/3/943.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Reproduction and In-Depth Evaluation of Genome-Wide Association Studies and Genome-Wide Meta-analyses Using Summary Statistics</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yao-Fang Niu</span>, <span class="highwire-citation-author" data-delta="1">Chengyin Ye</span>, <span class="highwire-citation-author" data-delta="2">Ji He</span>, <span class="highwire-citation-author" data-delta="3">Fang Han</span>, <span class="highwire-citation-author" data-delta="4">Long-Biao Guo</span>, <span class="highwire-citation-author" data-delta="5">Hou-Feng Zheng</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-5475-8237" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Guo-Bo Chen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">943-952;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.038877 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483528--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yao-Fang Niu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory of Rice Biology, China National Rice Research Institute, Hangzhou, 310006 Zhejiang, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yao-Fang%2BNiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Niu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYao-Fang%2BNiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Chengyin Ye </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Health Management, College of Medicine, Hangzhou Normal University, Hangzhou, 310021 Zhejiang, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chengyin%2BYe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ye%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChengyin%2BYe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ji He </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Neurology, Peking University Third Hospital, Beijing 100191, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ji%2BHe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=He%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJi%2BHe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Fang Han </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Pulmonary and Critical Care Medicine, Peking University People’s Hospital, Beijing 100044, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fang%2BHan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Han%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFang%2BHan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Long-Biao Guo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory of Rice Biology, China National Rice Research Institute, Hangzhou, 310006 Zhejiang, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Long-Biao%2BGuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guo%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALong-Biao%2BGuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Hou-Feng Zheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Institute of Aging Research, College of Medicine, Hangzhou Normal University, Hangzhou, 310021 Zhejiang, China</div><div class='author-affiliation'><span class='nlm-sup'>††</span>The Affiliated Hospital of Hangzhou Normal University, Hangzhou, 310015 Zhejiang, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hou-Feng%2BZheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zheng%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHou-Feng%2BZheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hfzheng@hznu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">hfzheng@hznu.edu.cn</a>
<a href="mailto:chenguobo@gmail.com" class="" data-icon-position="" data-hide-link-title="0">chenguobo@gmail.com</a></span></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Guo-Bo Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Evergreen Landscape and Architecture Studio, Hangzhou, 310026 Zhejiang, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guo-Bo%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGuo-Bo%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5475-8237" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Guo-Bo Chen</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hfzheng@hznu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">hfzheng@hznu.edu.cn</a>
<a href="mailto:chenguobo@gmail.com" class="" data-icon-position="" data-hide-link-title="0">chenguobo@gmail.com</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=943&amp;rft.epage=952&amp;rft.atitle=Reproduction%20and%20In-Depth%20Evaluation%20of%20Genome-Wide%20Association%20Studies%20and%20Genome-Wide%20Meta-analyses%20Using%20Summary%20Statistics&amp;rft.volume=7&amp;rft.issue=3&amp;rft.date=2017-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Niu%2C+Yao-Fang&amp;rft.au=Ye%2C+Chengyin&amp;rft.au=He%2C+Ji&amp;rft.au=Han%2C+Fang&amp;rft.au=Guo%2C+Long-Biao&amp;rft.au=Zheng%2C+Hou-Feng&amp;rft.au=Chen%2C+Guo-Bo"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/483528/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/483528/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/483528/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/483528/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/483528/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/483528/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/483528/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/483528/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/483528/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/483528/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/483528/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/483528/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/3/943" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/3/943.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/3/943.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/3/943.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/3/943.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
