<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/3/1027" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/3/1027.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/3/1027.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/3/1027.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/3/1027/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/3/1027.atom" />
<meta name="HW.pisa" content="ggg;7/3/1027" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Temporal Perspective on the Interplay of Demography and Selection on Deleterious Variation in Humans" />
<meta name="DC.Identifier" content="10.1534/g3.117.039651" />
<meta name="DC.Date" content="2017-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Koch and Novembre. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="When mutations have small effects on fitness, population size plays an important role in determining the amount and nature of deleterious genetic variation. The extent to which recent population size changes have impacted deleterious variation in humans has been a question of considerable interest and debate. An emerging consensus is that the Out-of-Africa bottleneck and subsequent growth events have been too short to cause meaningful differences in genetic load between populations; though changes in the number and average frequencies of deleterious variants have taken place. To provide more support for this view and to offer additional insight into the divergent evolution of deleterious variation across populations, we numerically solve time-inhomogeneous diffusion equations and study the temporal dynamics of the frequency spectra in models of population size change for modern humans. We observe how the response to demographic change differs by the strength of selection, and we then assess whether similar patterns are observed in exome sequence data from 33,370 and 5203 individuals of non-Finnish European and West African ancestry, respectively. Our theoretical results highlight how even simple summaries of the frequency spectrum can have complex responses to demographic change. These results support the finding that some apparent discrepancies between previous results have been driven by the behaviors of the precise summaries of deleterious variation. Further, our empirical results make clear the difficulty of inferring slight differences in frequency spectra using recent next-generation sequence data." />
<meta name="DC.Contributor" content="Evan Koch" />
<meta name="DC.Contributor" content="John Novembre" />
<meta name="article:published_time" content="2017-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Temporal Perspective on the Interplay of Demography and Selection on Deleterious Variation in Humans" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;When mutations have small effects on fitness, population size plays an important role in determining the amount and nature of deleterious genetic variation. The extent to which recent population size changes have impacted deleterious variation in humans has been a question of considerable interest and debate. An emerging consensus is that the Out-of-Africa bottleneck and subsequent growth events have been too short to cause meaningful differences in genetic load between populations; though changes in the number and average frequencies of deleterious variants have taken place. To provide more support for this view and to offer additional insight into the divergent evolution of deleterious variation across populations, we numerically solve time-inhomogeneous diffusion equations and study the temporal dynamics of the frequency spectra in models of population size change for modern humans. We observe how the response to demographic change differs by the strength of selection, and we then assess whether similar patterns are observed in exome sequence data from 33,370 and 5203 individuals of non-Finnish European and West African ancestry, respectively. Our theoretical results highlight how even simple summaries of the frequency spectrum can have complex responses to demographic change. These results support the finding that some apparent discrepancies between previous results have been driven by the behaviors of the precise summaries of deleterious variation. Further, our empirical results make clear the difficulty of inferring slight differences in frequency spectra using recent next-generation sequence data.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/03/01" />
<meta name="citation_mjid" content="ggg;7/3/1027" />
<meta name="citation_id" content="7/3/1027" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/3/1027" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/3/1027.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/3/1027.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/3/1027.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.039651" />
<meta name="citation_pmid" content="28159863" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1027" />
<meta name="citation_lastpage" content="1037" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Evan Koch" />
<meta name="citation_author_institution" content="Department of Ecology and Evolution, University of Chicago, Illinois 60637" />
<meta name="citation_author" content="John Novembre" />
<meta name="citation_author_institution" content="Department of Ecology and Evolution, University of Chicago, Illinois 60637" />
<meta name="citation_author_institution" content="Department of Human Genetics, University of Chicago, Illinois 60637" />
<meta name="citation_author_email" content="jnovembre@uchicago.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5345-0214" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. F. Agrawal;citation_author=M. C. Whitlock;citation_title=Inferences about the distribution of dominance drawn from yeast gene knockout data.;citation_volume=187;citation_year=2011;citation_pmid=21098719;citation_doi=10.1534/genetics.110.124560" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=D. J. Balick;citation_author=R. Do;citation_author=C. A. Cassa;citation_author=D. Reich;citation_author=S. R. Sunyaev;citation_title=Dominance of deleterious alleles controls the response to a population bottleneck.;citation_pages=1-23;citation_volume=11;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. R. Boyko;citation_author=S. H. Williamson;citation_author=A. R. Indap;citation_author=J. D. Degenhardt;citation_author=R. D. Hernandez;citation_title=Assessing the evolutionary impact of amino acid mutations in the human genome.;citation_pages=e1000083-e1000083;citation_volume=4;citation_year=2008;citation_issue=5;citation_pmid=18516229;citation_doi=10.1371/journal.pgen.1000083" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=Y. Brandvain;citation_author=S. I. Wright;citation_title=The limits of natural selection in a nonequilibrium world.;citation_pages=201-210;citation_volume=32;citation_year=2016;citation_pmid=26874998;citation_doi=10.1016/j.tig.2016.01.004" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Cao;citation_author=K. Schneeberger;citation_author=S. Ossowski;citation_author=T. Günther;citation_author=S. Bender;citation_title=Whole-genome sequencing of multiple Arabidopsis thaliana populations.;citation_pages=956-963;citation_volume=43;citation_year=2011;citation_issue=10;citation_pmid=21874002;citation_doi=10.1038/ng.911" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Castellano;citation_author=G. Parra;citation_author=F. A. Sanchez-Quinto;citation_author=F. Racimo;citation_author=M. Kuhlwilm;citation_title=Patterns of coding variation in the complete exomes of three Neandertals.;citation_volume=111;citation_year=2014;citation_pmid=24753607;citation_doi=10.1073/pnas.1405138111" />
<meta name="citation_reference" content="citation_title=Elements of Evolutionary Genetics;citation_author=B. Charlesworth;citation_author=D. Charlesworth;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. M. Comeron;citation_author=M. Kreitman;citation_title=Population, Evolutionary and Genomic Consequences of Interference Selection;citation_pages=389-410;citation_volume=161;citation_year=2002;citation_issue=1;citation_pmid=12019253" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=G. M. Cooper;citation_author=E. A. Stone;citation_author=G. Asimenos;citation_author=E. D. Green;citation_author=S. Batzoglou;citation_title=Distribution and intensity of constraint in mammalian genomic sequence.;citation_volume=15;citation_year=2005;citation_pmid=15965027;citation_doi=10.1101/gr.3577405" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=E. V. Davydov;citation_author=D. L. Goode;citation_author=M. Sirota;citation_author=G. M. Cooper;citation_author=A. Sidow;citation_title=Identifying a high fraction of the human genome to be under selective constraint using GERP++.;citation_pages=e1001025-e1001025;citation_volume=6;citation_year=2010;citation_issue=12;citation_pmid=21152010;citation_doi=10.1371/journal.pcbi.1001025" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. A. DePristo;citation_author=E. Banks;citation_author=R. Poplin;citation_author=K. V. Garimella;citation_author=J. R. Maguire;citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data.;citation_pages=491-498;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21478889;citation_doi=10.1038/ng.806" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=R. Do;citation_author=D. Balick;citation_author=H. Li;citation_author=I. Adzhubei;citation_author=S. Sunyaev;citation_title=No evidence that selection has been less effective at removing deleterious mutations in Europeans than in Africans.;citation_pages=126-131;citation_volume=47;citation_year=2015;citation_pmid=25581429;citation_doi=10.1038/ng.3186" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. M. Durbin;citation_author=D. L. Altshuler;citation_author=G. R. Abecasis;citation_author=D. R. Bentley;citation_author=A. Chakravarti;citation_title=A map of human genome variation from population-scale sequencing.;citation_pages=1061-1073;citation_volume=467;citation_year=2010;citation_issue=7319;citation_pmid=20981092;citation_doi=10.1038/nature09534" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=E. Elyashiv;citation_author=K. Bullaughey;citation_author=S. Sattath;citation_author=Y. Rinott;citation_author=M. Przeworski;citation_title=Shifts in the intensity of purifying selection: an analysis of genome-wide polymorphism data from two closely related yeast species.;citation_volume=20;citation_year=2010;citation_pmid=20817943;citation_doi=10.1101/gr.108993.110" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=S. N. Evans;citation_author=Y. Shvets;citation_author=M. Slatkin;citation_title=Non-equilibrium theory of the allele frequency spectrum.;citation_pages=109-119;citation_volume=71;citation_year=2007;citation_issue=1;citation_pmid=16887160;citation_doi=10.1016/j.tpb.2006.06.005" />
<meta name="citation_reference" content="citation_journal_title=Mathematical Population Genetics 1: I Theoretical Introduction.;citation_author=W. J. Ewens;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Eyre-Walker;citation_title=Genetic architecture of a complex trait and its implications for fitness and genome-wide association studies.;citation_volume=107;citation_year=2010;citation_pmid=20133822;citation_doi=10.1073/pnas.0906182107" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Eyre-Walker;citation_author=M. Woolfit;citation_author=T. Phelps;citation_title=The distribution of fitness effects of new deleterious amino acid mutations in humans.;citation_volume=173;citation_year=2006;citation_pmid=16547091;citation_doi=10.1534/genetics.106.057570" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=W. Fu;citation_author=R. Gittelman;citation_author=M. Bamshad;citation_author=J. Akey;citation_title=Characteristics of neutral and deleterious protein-coding variation among individuals and populations.;citation_pages=421-436;citation_volume=95;citation_year=2014;citation_pmid=25279984;citation_doi=10.1016/j.ajhg.2014.09.006" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. Gazave;citation_author=D. Chang;citation_author=A. G. Clark;citation_author=A. Keinan;citation_title=Population growth inflates the per-individual number of deleterious mutations and reduces their mean effect.;citation_volume=195;citation_year=2013;citation_pmid=23979573;citation_doi=10.1534/genetics.113.153973" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=B. H. Good;citation_author=A. M. Walczak;citation_author=R. A. Neher;citation_author=M. M. Desai;citation_title=Genetic diversity in the interference selection limit.;citation_pages=e1004222;citation_volume=10;citation_year=2014;citation_pmid=24675740;citation_doi=10.1371/journal.pgen.1004222" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Gravel;citation_title=When is selection effective?;citation_volume=203;citation_year=2016;citation_pmid=27010021;citation_doi=10.1534/genetics.115.184630" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=R. N. Gutenkunst;citation_author=R. D. Hernandez;citation_author=S. H. Williamson;citation_author=C. D. Bustamante;citation_title=Inferring the joint demographic history of multiple populations from multidimensional SNP frequency data.;citation_pages=e1000695-e1000695;citation_volume=5;citation_year=2009;citation_issue=10;citation_pmid=19851460;citation_doi=10.1371/journal.pgen.1000695" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_title=The genetic cost of Neanderthal introgression.;citation_author=K. Harris;citation_author=R. Nielsen;citation_pages=030387;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. L. Hartl;citation_author=E. N. Moriyama;citation_author=S. A. Sawyer;citation_title=Selection intensity for codon bias.;citation_pages=227-234;citation_volume=138;citation_year=1994;citation_issue=1;citation_pmid=8001789" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. M. Henn;citation_author=L. L. Cavalli-Sforza;citation_author=M. W. Feldman;citation_title=The great human expansion.;citation_volume=109;citation_year=2012;citation_pmid=23077256;citation_doi=10.1073/pnas.1212380109" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. M. Henn;citation_author=L. R. Botigué;citation_author=S. Peischl;citation_author=I. Dupanloup;citation_author=M. Lipatov;citation_title=Distance from sub-Saharan Africa predicts mutational load in diverse human genomes.;citation_volume=113;citation_year=2016;citation_pmid=26712023;citation_doi=10.1073/pnas.1510805112" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=R. D. Hernandez;citation_author=S. H. Williamson;citation_author=C. D. Bustamante;citation_title=Context dependence, ancestral misidentification, and spurious signatures of natural selection.;citation_volume=24;citation_year=2007;citation_pmid=17545186;citation_doi=10.1093/molbev/msm108" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. D. Hernandez;citation_author=J. L. Kelley;citation_author=E. Elyashiv;citation_author=S. C. Melton;citation_author=A. Auton;citation_title=Classic Selective Sweeps Were Rare in Recent Human Evolution;citation_pages=920-924;citation_volume=331;citation_year=2011;citation_issue=6019;citation_pmid=21330547;citation_doi=10.1126/science.1198878" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_title=The strength of selection against Neanderthal introgression.;citation_author=I. Juric;citation_author=S. Aeschbacher;citation_author=G. Coop;citation_pages=030148;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=B. Y. Kim;citation_author=K. E. Lohmueller;citation_title=Selection and reduced population size cannot explain higher amounts of Neandertal ancestry in East Asian than in European human populations.;citation_pages=454-461;citation_volume=96;citation_year=2015;citation_pmid=25683122;citation_doi=10.1016/j.ajhg.2014.12.029" />
<meta name="citation_reference" content="citation_journal_title=J. Appl. Probab.;citation_author=M. Kimura;citation_title=Diffusion models in population genetics.;citation_pages=177-232;citation_volume=1;citation_year=1964;citation_doi=10.2307/3211856" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=M. Kircher;citation_author=D. M. Witten;citation_author=P. Jain;citation_author=B. J. O’Roak;citation_author=G. M. Cooper;citation_title=A general framework for estimating the relative pathogenicity of human genetic variants.;citation_pages=310-315;citation_volume=46;citation_year=2014;citation_pmid=24487276;citation_doi=10.1038/ng.2892" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=M. Lek;citation_author=K. J. Karczewski;citation_author=K. E. Samocha;citation_author=E. Banks;citation_author=T. Fennell;citation_title=Analysis of protein-coding genetic variation in 60,706 humans.;citation_pages=285-291;citation_volume=536;citation_year=2016;citation_pmid=27535533;citation_doi=10.1038/nature19057" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=S. Li;citation_author=B. Li;citation_author=C. Cheng;citation_author=Z. Xiong;citation_author=Q. Liu;citation_title=Genomic signatures of near-extinction and rebirth of the crested ibis and other endangered bird species.;citation_pages=557;citation_volume=15;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Genet. Dev.;citation_author=K. E. Lohmueller;citation_title=The distribution of deleterious genetic variation in human populations.;citation_pages=139-146;citation_volume=29;citation_year=2014;citation_pmid=25461617;citation_doi=10.1016/j.gde.2014.09.005" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=K. E. Lohmueller;citation_title=The impact of population demography and selection on the genetic architecture of complex traits.;citation_pages=e1004379;citation_volume=10;citation_year=2014;citation_pmid=24875776;citation_doi=10.1371/journal.pgen.1004379" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. E. Lohmueller;citation_author=A. R. Indap;citation_author=S. Schmidt;citation_author=A. R. Boyko;citation_author=R. D. Hernandez;citation_title=Proportionally more deleterious genetic variation in European than in African populations.;citation_pages=994-997;citation_volume=451;citation_year=2008;citation_issue=7181;citation_pmid=18288194;citation_doi=10.1038/nature06611" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=K. E. Lohmueller;citation_author=A. Albrechtsen;citation_author=Y. Li;citation_author=S. Y. Kim;citation_author=T. Korneliussen;citation_title=Natural selection affects multiple aspects of genetic variation at putatively neutral sites across the human genome.;citation_pages=e1002326-e1002326;citation_volume=7;citation_year=2011;citation_issue=10;citation_pmid=22022285;citation_doi=10.1371/journal.pgen.1002326" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. D. Marsden;citation_author=D. Ortega-Del Vecchyo;citation_author=D. P. O’Brien;citation_author=J. F. Taylor;citation_author=O. Ramirez;citation_title=Bottlenecks and selective sweeps during domestication have increased deleterious genetic variation in dogs.;citation_volume=113;citation_year=2016;citation_pmid=26699508;citation_doi=10.1073/pnas.1512501113" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. McDonald;citation_author=M. Kreitman;citation_title=Adaptive protein evolution at the Adh locus in Drosophila.;citation_pages=652-654;citation_volume=351;citation_year=1991;citation_issue=6328;citation_pmid=1904993;citation_doi=10.1038/351652a0" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=G. McVicker;citation_author=D. Gordon;citation_author=C. Davis;citation_author=P. Green;citation_title=Widespread genomic signatures of natural selection in hominid evolution.;citation_pages=e1000471-e1000471;citation_volume=5;citation_year=2009;citation_issue=5;citation_pmid=19424416;citation_doi=10.1371/journal.pgen.1000471" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. R. Nelson;citation_author=D. Wegmann;citation_author=M. G. Ehm;citation_author=D. Kessner;citation_author=P. St Jean;citation_title=An abundance of rare functional variants in 202 drug target genes sequenced in 14,002 people.;citation_volume=337;citation_year=2012;citation_pmid=22604722;citation_doi=10.1126/science.1217876" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=S. Peischl;citation_author=I. Dupanloup;citation_author=M. Kirkpatrick;citation_author=L. Excoffier;citation_title=On the accumulation of deleterious mutations during range expansions.;citation_pages=5972-5982;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12524" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=F. Racimo;citation_author=J. Schraiber;citation_title=Approximation to the distribution of fitness effects across functional categories in human segregating polymorphisms.;citation_pages=e1004697;citation_volume=10;citation_year=2014;citation_pmid=25375159;citation_doi=10.1371/journal.pgen.1004697" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=S. Sankararaman;citation_author=S. Mallick;citation_author=M. Dannemann;citation_author=K. Prüfer;citation_author=J. Kelso;citation_title=The genomic landscape of Neanderthal ancestry in present-day humans.;citation_pages=354-357;citation_volume=507;citation_year=2014;citation_pmid=24476815;citation_doi=10.1038/nature12961" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Sawyer;citation_author=D. Hartl;citation_title=Population genetics of polymorphism and divergence.;citation_pages=1161-1176;citation_volume=132;citation_year=1992;citation_issue=4;citation_pmid=1459433" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Schubert;citation_author=H. Jónsson;citation_author=D. Chang;citation_author=C. Der Sarkissian;citation_author=L. Ermini;citation_title=Prehistoric genomes reveal the genetic foundation and cost of horse domestication.;citation_volume=111;citation_year=2014;citation_pmid=25512547;citation_doi=10.1073/pnas.1416991111" />
<meta name="citation_reference" content="citation_title=The impact of recent population history on the deleterious mutation load in humans and close evolutionary relatives.;citation_author=Y. B. Simons;citation_author=G. Sella;citation_pages=150-158;citation_volume=41;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=Y. B. Simons;citation_author=M. C. Turchin;citation_author=J. K. Pritchard;citation_author=G. Sella;citation_title=The deleterious mutation load is insensitive to recent population history.;citation_pages=220-224;citation_volume=46;citation_year=2014;citation_pmid=24509481;citation_doi=10.1038/ng.2896" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Subramanian;citation_title=The abundance of deleterious polymorphisms in humans.;citation_volume=190;citation_year=2012;citation_pmid=22267501;citation_doi=10.1534/genetics.111.137893" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=J. A. Tennessen;citation_author=A. W. Bigham;citation_author=T. D. O’Connor;citation_author=W. Fu;citation_author=E. E. Kenny;citation_title=Evolution and functional impact of rare coding variation from deep sequencing of human exomes.;citation_volume=337;citation_year=2012;citation_pmid=22604720;citation_doi=10.1126/science.1219240" />
<meta name="citation_reference" content="citation_journal_abbrev=Front Genet;citation_author=A. Torkamani;citation_author=P. Pham;citation_author=O. Libiger;citation_author=V. Bansal;citation_author=G. Zhang;citation_title=Clinical implications of human population differences in genome-wide rates of functional genotypes.;citation_pages=211-211;citation_volume=3;citation_year=2012;citation_pmid=23125845" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. D. Wall;citation_author=M. A. Yang;citation_author=F. Jay;citation_author=S. K. Kim;citation_author=E. Y. Durand;citation_title=Higher levels of Neanderthal ancestry in east Asians than in Europeans.;citation_volume=194;citation_year=2013;citation_pmid=23410836;citation_doi=10.1534/genetics.112.148213" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. H. Williamson;citation_author=R. Hernandez;citation_author=A. Fledel-Alon;citation_author=L. Zhu;citation_author=R. Nielsen;citation_title=Simultaneous inference of selection and population growth from patterns of variation in the human genome.;citation_volume=102;citation_year=2005;citation_pmid=15905331;citation_doi=10.1073/pnas.0502300102" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=Y. Xue;citation_author=J. Prado-Martinez;citation_author=P. H. Sudmant;citation_author=V. Narasimhan;citation_author=Q. Ayub;citation_title=Mountain gorilla genomes reveal the impact of long-term population decline and inbreeding;citation_pages=242-245;citation_volume=348;citation_year=2015;citation_issue=6231;citation_pmid=25859046;citation_doi=10.1126/science.aaa3952" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=N. Yu;citation_author=F. C. Chen;citation_author=S. Ota;citation_author=L. B. Jorde;citation_author=P. Pamilo;citation_title=Larger Genetic Differences Within Africans Than Between Africans and Eurasians;citation_pages=269-274;citation_volume=161;citation_year=2002;citation_issue=1;citation_pmid=12019240" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Temporal Perspective on the Interplay of Demography and Selection on Deleterious Variation in Humans" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3.cover-source.jpg" />
<meta name="twitter:description" content="When mutations have small effects on fitness, population size plays an important role in determining the amount and nature of deleterious genetic variation. The extent to which recent population size changes have impacted deleterious variation in humans has been a question of considerable interest and debate. An emerging consensus is that the Out-of-Africa bottleneck and subsequent growth events have been too short to cause meaningful differences in genetic load between populations; though changes in the number and average frequencies of deleterious variants have taken place. To provide more support for this view and to offer additional insight into the divergent evolution of deleterious variation across populations, we numerically solve time-inhomogeneous diffusion equations and study the temporal dynamics of the frequency spectra in models of population size change for modern humans. We observe how the response to demographic change differs by the strength of selection, and we then assess whether similar patterns are observed in exome sequence data from 33,370 and 5203 individuals of non-Finnish European and West African ancestry, respectively. Our theoretical results highlight how even simple summaries of the frequency spectrum can have complex responses to demographic change. These results support the finding that some apparent discrepancies between previous results have been driven by the behaviors of the precise summaries of deleterious variation. Further, our empirical results make clear the difficulty of inferring slight differences in frequency spectra using recent next-generation sequence data." />
<meta name="og-title" property="og:title" content="A Temporal Perspective on the Interplay of Demography and Selection on Deleterious Variation in Humans" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/3/1027" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="When mutations have small effects on fitness, population size plays an important role in determining the amount and nature of deleterious genetic variation. The extent to which recent population size changes have impacted deleterious variation in humans has been a question of considerable interest and debate. An emerging consensus is that the Out-of-Africa bottleneck and subsequent growth events have been too short to cause meaningful differences in genetic load between populations; though changes in the number and average frequencies of deleterious variants have taken place. To provide more support for this view and to offer additional insight into the divergent evolution of deleterious variation across populations, we numerically solve time-inhomogeneous diffusion equations and study the temporal dynamics of the frequency spectra in models of population size change for modern humans. We observe how the response to demographic change differs by the strength of selection, and we then assess whether similar patterns are observed in exome sequence data from 33,370 and 5203 individuals of non-Finnish European and West African ancestry, respectively. Our theoretical results highlight how even simple summaries of the frequency spectrum can have complex responses to demographic change. These results support the finding that some apparent discrepancies between previous results have been driven by the behaviors of the precise summaries of deleterious variation. Further, our empirical results make clear the difficulty of inferring slight differences in frequency spectra using recent next-generation sequence data." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3.cover-source.jpg" />
    <title>A Temporal Perspective on the Interplay of Demography and Selection on Deleterious Variation in Humans | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"A8M7AiNbdkE-jjEV6tu7VrFeSqOuKtk4XnsmCjcW_Gs","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"483518","apath":"\/ggg\/7\/3\/1027.atom","pisa":"ggg;7\/3\/1027","ac":{"\/ggg\/7\/3\/1027.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/3\/1027.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/3\/1027"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/3\/1027"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/3\/1027.long":true,"\/content\/7\/3\/1027":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022hXvf\\\\\\\/OUcahuXFbjMkXqBp6ysddCFoXBCE7oXkp79WBr2cdB3hqITT6BeNU4fK24FkvZ9D2zmXOGzF9mLoVj6IEqtmlPXNRuAcy0bbwGsegTtZ7sL8A0EwqgGk6csjUQ1l4QRgEtZlNL7Ty0WomA7h3IqfUKrfoEBG026Ay6YCRrQAKcKKgZfDIh5ae3AoLVcqPj8SEgCy2VS9GOKvCoYthWnZWJMpFf7Y5YwHOGaB1hRCcZHYxWL2NkI\\\\\\\/Eromowj0D7IkG6aKcxfIqfbC6+tKeLNWSp6lFGBirH3s8Es6Uxc6j3T0AIhcOA93bZnO0dX0nqni+QrrbWjnHFrFJeT\\\\\\\/YxM81qAguxq9uZsuxElnEeUsPMFOrtFPMUoyZDTei4vIxYHYXW3vwQzSQjXK28f\\\\\\\/qYmyUQFVNJPHrxcFp8pGyiJQlte+m3Vjah36SaGqBem1UQMlIPOvgld1bQi9NEDrnqO325RF+tR5EaottYNziJBqG9jP\\\\\\\/AfQ0vu4dm7AQl3X78Dsene5IC0T8A6REot7RYZc14zRiLYrpXJcgtudX9XQxjUNpKRTMWLZ7udRaPYt7UxrGrqAEkfqFAjOB7eWt2e7IFk57vRWtQ9VaoLEDlQFlN7pbw5Wxl2Mp96PjTwv5zSRmXiDMD6iocz0tJNe9OBbwtzzeAXjM4lyX2rCbG7xNptyLUA5Eq9vfrpbg\\\\\\\/UuOqWuZ7VFN\\\\\\\/jrv\\\\\\\/xVVmmsu9YSD1e\\\\\\\/UPiF\\\\\\\/sbfQ3SfJHCG1hG7VJNdZoHuvoKc0GW+M87rvIaZ4EHqErZmT5uQ\\\\\\\/EHDtC3Pxy+nWrVommwQ699exeAtD2tdhOilJk7vWcoJF+Ifjq8XwjEX4WT5pDwOBot9EZIAwOtGubMFu0OqgW66Xvyzn10YZJ3DpfWU\\\\\\\/pncM6llokCt0QEYzKGRw7f4JLrOpoGLhMT4RlMW7et+csLPG4dKOQClm5x0dRGrzfG4mTfgsQe5BiEdioL4CsYduQuPt51pR7JbsuC7yeXQgfl3nNAY1reikf\\\\\\\/ebWcay8swxaq3YW0aIHK2G5SSihbbvXLc5JlcjUpV+3SX5YOwYlnXTJoFis9Y4a+M8UoBpBjspAmIY\\\\\\\/0o50Ov2J\\\\\\\/9ROK4eLSi8us9mxv2W2IAWGFLxt0lVjLsOiqHiSyAMo8FKW1HHNpCmmYaSzHo5o3D\\\\\\\/YaYZ4VSfo3QiObTFuaDVWn2J18uHDeHzRZSJnkksczmFeOi6BcgkJU9VAC8U3YbLPbBilF5vU8ATpu8aY6GYiQ+PZWVpznTwv3uii4FdZDV2ST\\\u0022,\\\u0022iv\\\u0022:\\\u0022Y73mRCz0IFC0ZMQdnB8\\\\\\\/ng==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022cdbb959bcfe8f2a669d90b852d4c6b82beabba53c1c423ba5b3636a63942cf75\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022JID6\\\\\\\/Gx\\\\\\\/TKST9cWvtD80JuXp\\\\\\\/zac2\\\\\\\/hKrLHjuu4yrCPABKX6Xonu+T2wO4tTaCzaNB\\\\\\\/tutA8PGCzTYSp01+vqJrB8BcFKCK4doDFoIxuCgPpQV4ggg9C4vVblKkbIW9+WPBMaqv6F2gxciyuv9uAm13NAcT9UgkQBhl8986dMn5STX1QQXzHPOf6ihBNOMlm5O8UUOJlEkysqEWZLd2ReBugvvlWRkXLrn7GqCCUPtyoibm8orofYlZzgtMFXCVX9ozFielHnbKRj3IZ0R1T+qsqIICcEpXGGwbnqN4Gr4Mm\\\\\\\/IkySqceTxZok12RP1285XB\\\\\\\/uh26IqJ08liLwT9Fb2QNhU59zyaLclIh82tRudF4\\\\\\\/fHkeykhUrFTvCMiWlEPI4drMoRRS7I7tswSg8ZqiEDs1swiWGSOPFVZrAVRueDQXjnKlhuIBxKU7StMANvu0FistOOy8\\\\\\\/c4C0iNUPnjrEJwlOOBFNbmk9Zo2BG\\\\\\\/e+4viGh1SVPe\\\\\\\/Zshk9i5mLHWsv8yVS36RCEchExa5v0Dt9aye0z3\\\\\\\/sfWtws+phZTFqwCrEthFu\\\\\\\/nvhe9NluctlBPA\\\\\\\/uDo50POnZFkKsZeOg9fA2rVP4UBB9HLurVMIeNYqUAS6CPi44\\\\\\\/U1eH9zkMo5PNPB081fb551hw8K3KKTRSY4+rRRgKxJXkK7DZdD3lwuuEucYJkjV1AbG89RI\\\\\\\/b7DEzUD+l4fkIL6hmUQQs5zPK8c8xttMsuhhv9PE2pbS52X027566WlAFAK7QXGIjiXBNjKH0goFL8Ui9yEQdAmq4F3S3+qWBOkZZyYed8aX3Q51ZGjgn++jBmrrY9MziYrkoj3lUXVG1Kx+6hqrIOVKsaeuSdi5MshBI6M4Rf45Zk9XBP+EmjU02Akxwc+f3TvkcbKgHfNEM0m8BYsBqvWG6INl2zNQEtG8OKL1GDnCPKDkYbhToeBRTW9VRnmBgLUcddtxrhqOnL7V6KsQW6\\\\\\\/Tg7gb6X1pVhom6Bbn6MjKlIkbCE1GeBWx8YvpfvZngqbWvly8Um9\\\\\\\/b9BtSUFF3JB58wTlhZ8hVhLlN8QkpuR5FvyLGtqa7C\\\\\\\/IVL5sm74j05NQ6PN9uuPvLSKNLA03rZbwww6hjnm\\\\\\\/34bgqsAW78OHlQ0ANGa1vW4LBbTSh+Vj+sG5Zt8KKtkeUwOx5ZiXxnt\\\\\\\/q3DZJ465ltKMXQrmB3ZaNiqob7GqNvCsJ3H55rIjokjogrSzNnc\\\\\\\/roqXJ\\\\\\\/kjwncrONEU3k9M6wPxKw9floPd1WYzCEvokj48NiEZMsuyAxynxxUwh6wC+Q9Ef00uMKmY86cjDYczkrTv4NM9JM+WYp0n91S5b+sOLtlPJF6VIVaLnyOVKWXl3U\\\\\\\/+XysG0X3Zg05FQEt39NN6LcC+RvSeLvz452GTUCAfsTNtAITTa4qoINpWS5XzToWCM6IcTEgh+WphxKMj0i4dmZTPdwHq+Q5KtVqG33O\\\\\\\/wAP7woG4+2QKqHDy8F+LXd18rabfM8rGUad1aAwBJQzqivzDnEzzxEBYGjODIBVJ26emu7oG58HuYqExoZHTCjx7cNKNtOFag4IVkvJ7wNsnsyQ\\\\\\\/BPHjaaakygz238NjVzIQkF9LgGWA1ISrZq+qNQ6KKHQwFGAp0jHQv3F82VVyog+DhApj3F0rpF3an5w3pvbxAvf8GkDs8PNwUlL1OO30bDnKa4F2LF5sUyyWBpJYJpFtC8RblqW5kAVK\\\\\\\/oUaoyXZF+EBzG2rECJeB8RUofbhP+bJZsjlU9weBSEEh0ZFde2RpX4MDMUNfrHVru2TjWD\\\\\\\/dEKlZaESgbKYTt1odSOetheQJ1uPv\\\\\\\/LZHqIXrwZ4rblawZgmgK+Q09v3L8ZjAPo8vp7Lwk0G1uYxBnQkso4VR5\\\\\\\/xqAFpVxJbd8sFRXew8YXu3aiH3pwWWza8Sl9PlIDGGK+\\\\\\\/8Nhpe7qpicl+vojfCM+9Hmb6O16xa+PXRbxZLoXr+g\\\\\\\/0y9HUDlPOwd0lxW2ck+E6ioLNexEAER0tnlXp3utJVCBfuvDVSOP5drSclYDqtCFIdL6XRuXLQzHaZUxcg1gTg\\\\\\\/cy5RLJse1\\\\\\\/n5CRx+79eo1mhyrZikN+JXXn\\\\\\\/tVh8aAcmtWD5FZd\\\\\\\/0QuCE43w16pX6I0cpXnPrxxLKSDHjVW+n6lus2\\\\\\\/+0TcW07jDrsQsGlKnGpR3lCbFMFGXEl90oQwl27qwWcpn+6Ysq02EdVcphp5+MJlwr6UX8WREEh5pHGNPS6Hji60njaruK3CJJvUxDjy66fuTMEvjyujAxVBu381R2Y8gGN3+XvccvfLUdUTMaZr8vtZ6oCykIowld+CH09bxO2\\\\\\\/lheSjA9yatTmuvOSovn3IO\\\\\\\/B81RkVyQ9pb4A8p9o9Sm7ALwOiwXK6ZGIaO5622rWruGN5gdjX9QC7fA2WYbz3ziY+Dh1thEox+p7M8E3sC\\\\\\\/5kqXuVAQOva3pSuPZFYJlPzga9+HSt8UarbHUCqlNjbSihTpfib38+4Io1Xw1WckjID9snVpw3rIKiBz3yQ++bAeyCo+Tshd4vqQW8aR58Z+dNgSQTnv91gme+BdIyN2gOSvx9PSDLX6wxLbPijrqCg8Sb+PeWV\\\\\\\/jG8q\\\\\\\/GNjo5T2rFs9iV8PQkQWmad27Q+H+Oa8\\\\\\\/8olGyBVloyEC8XbGJPDWmHfjbIreJEaU4bRc8bjEk4VI8vSJJHIQJoUUE1D90VkDrQL75udOe3u\\\\\\\/MJ7bx5I9oEUFHUJESOK73YFc4aKBVgZ7rwTxtC1ky9C0wrPxfZ\\\\\\\/KnjuG3wLw+zwHLOCLqkNKislDX2QB2kFl+\\\\\\\/09lTlwI\\\\\\\/3JnTCfOdtEzMpvdsAqRu3hqLnT3qI13DMJREJcooaFVs557PbMIX\\\\\\\/EaXUteraXkCVeVijStIyLQYwvOelgfJYzGhLWs7YAAmZDchjTUW5nNHhP0e4X\\\\\\\/5ViaJkV78mkOqSb49Nb9S9heAZjeJG5KsQ1cJZWMbmX1AUZWU\\\\\\\/2betbirnrQY1kEVDIRz\\\\\\\/FjGqZvpkQm6d+GmrX6NKZ+TiCVWHFaPiUoCtV1EEA+T4s\\\\\\\/vrO\\\\\\\/u7AuNJLTElcEVjrBKEQ3HxW1MZ0e3Hd36ihc5QO90gFS139MdxVOR4GilnoMamixhY2KAjumSlHfA4qk258wiBcLeGCZ7+OjEgGqVbeV1LHlMj6m0dmFoNl87PowoUjqy90xd5qOE0Pnhj78pYJBid0uuhG8zi4DM6QdDM1n7GhXKPn0rlQ0Xdlsm6FoAZ57SKRGgflIqRuGr9ikJxrFGxFj9A7nmPcNk7pn4Q+O5B4nSYzAZPPWtC2Xc1MduyxLrBcqqkNY9g6UWALbpi6cj4GnShzqtKMF8FBCD9j6flv+kroWann7V2r4Op62GyCkq65qy4YzbzBuef2Z+lT1ph1wvD5kVW++6XDz5eF7nTU9j9gxeZrWrWxMtEII9ysXs0n0V4O7h0r4AJ6tiVQQfFlnZkiFrAiDs45pkL4\\\\\\\/B818QBKZWvgCb\\\\\\\/abufScsHNNEVa13MVQtr3Scy6BxNnD8fXsctAPjEX9q+y9n4MHtrq+qNIR6E+G+Wo2Pe+bYJkHeRuwARCzASWQ1576oRbFqW9Hst4CDLtDYijC6oTzK7lXT7Gyuqla3SJmahIQkiwLSU9WMsXKVT9n9NFMsMb1o8UIw7dCAP4tt1z9vguA\\\\\\\/RF1IxOVhou6q9G2sxmzFPGMjqIuea67qaXkWF\\\\\\\/jw7gcCfTZ05xuQeBtEHCh4YlyZFOCK+MvsO6CnopizzpjKXZRk6Y9TLDpTz2W\\\\\\\/xKx4O0\\\\\\\/pOGgnEK8dhtbaBFOohpo2ncfUGs27msVYuk0vNd9JY\\\\\\\/zb8MK9bv6awzDRpl1WrX6J2Gh9\\\\\\\/BsuHqn9JveUd0CAtzRqhKvOIXwS7Em1+nmddbVzvdAO\\\\\\\/ZeHkSjlm4BsLdqOGP8X1W5Sv0uf8ZP9dCo0BIYaOimcheqSAh6U78XqMaTRC6myjAX6GCXolrOYqmhGeTNX4FUH7NuFGH5EvTqyu0Cvtk85Jcb4GpTbLJ7DaLAPHzS02iecG\\\\\\\/+SM3zOEsqGWWKmJQkd9O6cOtin3jSctD4bQ2aymC5ajCAdrH\\\\\\\/BjI8s1+dJrXnPnMy23CJPDRUsxb2SOzGliMyr\\\\\\\/6GzwbL+6ygdavsApbLvymXa3aUOjpzr6Wood5f+5A5c6aMxxvcJkyoE31OdXh+Kc6lHhskbE47j5YVGAp3dPF1s6PYa49xYkWnp96tbCzQ\\\\\\\/S9g4pIPLZ5lD1rgJQ4xCtEhVHAP1nKoQVi5xDY5Yy9n6Wk5JXvm6lSkFYfMHvfPohKibvUzWLBdl7CtYyByPu9hqlsYzkC63wLnsatSrbO\\\\\\\/PsLxiFC3tz4mZZeshjpUddZh3jQ2nSF+oXhLqXFisKXP4nwoA5R6DjvxyI9WWBN6Fa7cXVmx3pHg+QMCyZhx6jpaOZxz5twCBfoN\\\\\\\/kzuwpi0Aq4W3tcJFW6P77uL01sYymjkClts7bVEdeD6kZzm4szWA2NzR0mcb5YZWLTcilaPUBxXBcKM7Y6ygXh5alaXJPSDv7NHV12s2+rKKgT\\\\\\\/1d2wkeAMrP+8EVTuQ4QyOxinZDt9qFf2tx+zIxiwyxhbzYGQeOFJdUNvdPoY0ZdZOqaqZYY8VtjRt8NZp72pYtO6T7lEOwVMqqMxkRaMqEZP\\\\\\\/24V5ylGal6nACcOVpk3jKwZjGD9ry\\\\\\\/XxZUFeBKuwxHf4o5YLYhZoQZ6kTohYX+ZwLNVqLqc8+Q3TLXLLci+NWZV0TnJ5RyC34RNsVyDNz6t3\\\\\\\/NmzI0igedyxg0FobjvTkHi0Gmcn\\\\\\\/rke3zs8VhAQ3yUX21ElebCCVmShbfooHvlopykjwN+HAZFKk0Cq3HPzA8iVGqS4P8mEeixpsen9S9LFGz2OayZ2yldG5g7U2fDyYh1x00c5wK8vrlMKVN5t2Pno7qfb4lLs1\\\\\\\/BIZxbPF75\\\\\\\/tkjye3G6txkRwGPxp0h1hVddidpoPGm+98YXUZzBwLZ4W+V9yS7k6QWiChIsP3RNXa7OyFGjEoXKubVIZnShX8jWgd+NRwBmAksWYywCLOCBhBNDvybMkS\\\\\\\/kvb3aAxUWw64K\\\\\\\/xZx84Zh\\\\\\\/RBppUIt1BuW2gH0\\\\\\\/oOi4qJjdQVoknAJZXlZRD27Bxn2xXbkS5Ts6tI\\\\\\\/s4FGGZvAIWcmGXcQG+7u5+Fwor6WTu5801R713JpGYsI4n1EQkUQW8Kw\\\\\\\/iqM2eARnvEEivVDLhcVaLiE+d9Z0r5pvX2iagyGyrzK1a4HHzKEgQ6QMD17XkeBgqrkJVdDVenh6a0zAZHnpX\\\\\\\/KgdnxDfexfn0TpzR9j2MkeyLXXcpqNTXiIRNBPyfWLDYypN2AT2FPU\\\\\\\/DgKRMQHQpdcz\\\\\\\/spxC\\\\\\\/wBIBVQ3vps6UjhtA7EEAziFFguni3808Y2W9X90Ca53Ne5dmPZmATKbHnx++LCNMh5A6Jfj1rplCNtP8XGtdgDLAsJW4\\\\\\\/exBB5oe0UU2T8Vb2ZwwpVG7\\\\\\\/YiLuN1wzvJvZMzZahm0I7WJRoXkI+YGPxkkCEGP1QywoxzZsi\\\\\\\/cszFKGSfZgQM\\\\\\\/atRA0IELxwH1+hf4os2O6W+Oaej0Q6XmGkz0OJXlo0SESrS+MItSFzai2g3CjaQO1xPHXmvfrbA18UGURGufiWlY9m\\\\\\\/p3Li2Zjnf+NQj\\\\\\\/3TFZiovV3HAUYgratJPFsSSFHxlbYs7BR\\\\\\\/DiVm6VNDHqrSkUdb\\\\\\\/bJ+4DZ2LWZ1oMyYS4GqpkQpcOBjY4mQlyN8ZeVMPDTO4LR7S3ZHu5AF9onmrB080Eb\\\\\\\/bTGyrKIHcnQjmJ2pbyjlwUvQ0GO\\\\\\\/ZdURgVSEQc6mwK8MScHBb3A4\\\\\\\/kxuyAHQhl\\\\\\\/l0RKmrIXz3ukDL0mimqbTBu3eBnJTgzayq\\\\\\\/fXqj5nwEwURQzYD2j4Io\\\\\\\/WS2l5KturAC0EmBidkjS75Bh5P9J7aafqMzrp2mYT7VlmLelI9Bu5GBKOilqN1m770yFzChn+otG7uzN+xXhwui0f9huI4TFI4RI0ahSL53T+4xgTSHtMDRlhdN1wjT9K9XtZSY7ic+ru40ZrmAZ5UHIL5R2BcXvNEPIM\\\\\\\/88Uhln34AkZcczzwKlGiV\\\\\\\/f80QTwxl83BbAVSaswWtnPi6Tq1\\\\\\\/9HM2xoNrdjeRoazTL70iIU9fS3Ii4IZe+cROlcAI5KdpeQvC6+1a4Q1OOd1jW4bZOPfsRM+Z49XIVALiS7dJd9NqXZ+lFEYlCyv7KvZoABXJs9j1lhEhDaSbhKmoPl6Ng+xKbB\\\\\\\/kSmPJei\\\\\\\/mkA2z4hEVPIn17EnSOjhaVnnJ7meTbjtdzzqr4p1G\\\\\\\/bWlZr1W07gD3J9LR8qjd3i1KCrGqrv5+ZA382re\\\\\\\/ZXGCi\\\\\\\/w9HYKw2FBB+KzwBDT17B0jlaEmY2RagSvSNKZ1mgcl0x1EpHyktoUroPT5VBEeC2TMhaOZvxYYi8xfrSrhIT12Nlgup8KgKtId5gx+NHdY\\\\\\\/psehmEI2KFS1nec8TMacrUm8p+Ms+7umlm9ePGbAfYIbcWFEmFLvBrJKXScATvtiktQbabUCxTXYUfORPlaWWKFOVeswFtxOa2GoZXv4JSq235eTyrQO0B5omjEvDOxD1\\\\\\\/6hrvxSeRa34CES4j5KVHZqekIxJBMj\\\\\\\/M3LoncRG6XFNEdeOzHd\\\\\\\/0zmbowhy0TuCCO\\\\\\\/4Bb4lfC9el6srpLR67Q4tdcWgyfNiMgZxGsCCifzwxXdM4m0lCMl41PwC3OLKtBiyUoxt9dkFtr7WIfEOlOQdV8tERHBD6b0Z09sjcdE47vW+wNPzYydLrHE7sYdKcUVOiSulQwtau9zDyz5YiF1m7u3UViv5GocEFnKTvIL4N2xy7wf89cbIh8EXwyBJAhyIMxRQjFrgNkvpPbZsE\\\\\\\/JPCM9xrqXynz+KYfpBokm1KMe3TEo2AH88zFbCp3QsTWEi3hTwqw6bmF\\\\\\\/2vpeKjk1vmgOdeCfi9lEV2qvyw8b9VAf6VWE40YrAeo0XXw1om9iBG5cuF04vcdnDzaq3fxax\\\\\\\/55wSHwX\\\\\\\/FTvdLy2lEkTa5hV7ikkOBvcIfU2AZ1fyJ5JvbA\\\\\\\/EFdtq1LafU+pXt0uxIhpKB9dw0ONCa6R5RrJkHnEpLOt7KXvvegJiqNUNH1ifKyhc1Bz3B2AuFwtiF2hUpIU+DykaideoTEAeTvrcCupOLYD7PjpkKeKYQzK37vll69MDmhfAsw+yaFjli8qKamqbZ27sJ58CPr4HaakQZNJmWXyJmOrrcHRGiMLYvC2h\\\\\\\/4CVnGtIwDzOkYY43OrnyOFl7BJr3UlkgYgyaIX8J2QCt0kSEXFXHCm9Y8uCgv0asWdr4fWfwRAN2XZx9m2A+PkJBw1y0kuWRk\\\\\\\/kx+sR0MvlUY4vP+BI4VLKUpgbesCS9NY9JtnQNmK5n8nTwhoDwjOnDrgv8HxwWyRGXmAhsh6U5QS5WgF8rGBDF5yUU477cgqeyf5oeJQTdQkIr2DJx33GPp2EkTHFl\\\\\\\/+c90PlM8k5uYiufNa+lULXjBiPXx+naWbTyZXedLAFO6ZG0qKxTKF3dbHczhaWEp+4De0MFZ15D5rNmSQ+nMNAnpnSQ06DWL2cC1U16ZhtnpGtSJu5RpgaFB6xzENvCnD1ADBcXYGP0gn3l7nPf5zn1v31o3iKXy4s0sNCI\\\\\\\/Odww7M6S0U4dZn5wONqB33YVh01Xy\\\\\\\/xi3SzClt+cC7CwDt9BC0bsN1GxvMN3OhKQ7E9qMi0hpVZnOcMilfUf1xsWVynL9xsAj8JglzdCuM\\\\\\\/H6eaoerNZOt2f\\\\\\\/Tg3kOoukYSYirRaoJ7N5+GEbBBwTkUL7Ryoz2EaRLYN+OELPXmZlq+sxlUcVdVztM7OFnGCS34ywCyR4tl0pf9y7DfECCpBWglT4rDlp9fk1bsWJOoVGeAfZwG7RqDzdaXQzRce\\\\\\\/nelFbvO9bOWY6Wq2LdJMcis7kBjj5Tm+b0bTUn10mq5kpLJk\\\\\\\/9M9T2IoWn39wu0PzBd0qID8VXuEUQc4FBJC7R9WBZXK62Tfo6FiBEFwmqfl6ybyjd+gAwlAAs25EeOief+zQDFAIAQzCuoGOC3\\\\\\\/\\\\\\\/4RhJoiKFtWFvp1uAlcd6XTjNUjaEAoji1aa5QfM97meL8V6c2s7M+kIhZ2THP+FvpTAcyjXPsjz3D2u7+xuRBRZ\\\\\\\/VG75UGTgCQtIYFToAmd4SceDnDQy8\\\\\\\/qKgRM7tnDbZ8S0OC+r02FNx1YPsNYLUxYaemuOF9n8Sjb6cGEz7FOERR\\\\\\\/YkOC\\\\\\\/NtPc7wwxW3rhq5oyGfEf6QL4ImmNBJGbIuY17rvsyLj8J2EvOcnI\\\\\\\/PO1053ZCh+4SxhAW+SSPxqsR+nJuJH5nk7LTbytgnDPtdw9ZoAktUVklBCNgFKMpfNZnmEwCOdb9VNorOeinNkR21kadPPQDbWcGKzOtJb8rouOiHys5nXAxP+Pk6WUckncLc4gtEPUonY44B1T8X\\\\\\\/B2lPgPiP6nMtDahTnk5n7xeAK8VPmRUREwwsk0qQwP3TDRSwjy0hryp7CtEMnhygWrjwWo8cIHPtwuixVX5xEU0PHPDwP6jo6S+6wY9GlL8nsE2ySuaBOYmBrQH2epTFjJj\\\\\\\/BkHNwDRSl8e2hEgDNzHgjSMbO5jN0rfh5dG+Q0ChXvdaAoUsuXY7\\\\\\\/Pim54mbHA3dWIkAzZOfgmMOOHLessRbxJNeGWq9\\\\\\\/2Vz93O3xMJAhAZHA7tAC9zyKYLxL20TFHsZy2sVszPnM2OkuCfA5zLY6PpyAwldvftgWSmgyhg\\\\\\\/xN+JiUYdQaQcdNeNUZ7AF1ChQEYyZSOt90HOZps80JdBh2qg1qVSbmOfeO3kXlCNSCsiG7AroNaLc9TeFJZsbOe5Vg0Qtmj7uhsokIMH95Vp7UN3w7YJUxq0r+iK4kR\\\\\\\/\\\\\\\/+VsWFnJhjqnnm4FkgT6G5T4F4otAsgvfMiYOxqCORlS+K63wrmEMJdNONnbsCv\\\\\\\/h6bDk1zxHpRoVFmEbCTgS+dJ16twAyAA6QTLQI9FDNF7PHxVNu6HNFPhfmfUX2zFnAzewsNEcz5F9NF23qf+GyeSu\\\\\\\/g3LFSnyGCGGkuN\\\\\\\/EQDOcPQ3A577J3jqRlsebqYYoqd9EbAr7vhua5CM3mKKuNDCT1nQhIfpRk2R6B3ocFt9MlMFOdw\\\\\\\/E2nHHnwprAAuIji3zxPBLlkipWNgwBVTNguk6ssvTKhgN5KIxVhkAoqsLv7uG\\\\\\\/ld7GY2lr2SgjpECSwj5zDtYlPgBAzNZvbTgHiPsO+OAf3y8chH2WKFoHfRJg9mO3tleZzfOSmFR+9VEg0wpzdfyPM1P6j4fEiilrMIcCcavAvLukqcfF5bUJxVFj\\\\\\\/9QvinicA6\\\\\\\/AxG8w9E2CLvcmApYr651Y+2WXKlCpsf0x\\\\\\\/76SWVs7xr02FcQe93J608jnzM1EdkPStDiisDj3ptI11ZR0oihqUoQs0NE4oqFuYrCWlt6d2urGN30X+mtbvp+zaCrII9\\\\\\\/\\\\\\\/SA9bPEWk9NZSgzwPkZ4rg72V8G1pw\\\\\\\/C06IhFRLVd1e+P1LRx+wPlnmJfqkWzcmgl6JKiO9tBjjwhxjZSfGYG2xRHbSr5FQKG1YSrmxkE2JqFRtFxt3UxgHu3DrQAqADwHOZyn+k4wC\\\\\\\/jsYo+UyPPQi1majcwoTwy4hOOVwQocnpSPqDU7QijJhyQ8e\\\\\\\/pr1mF7qYBtNidL+\\\\\\\/UMd87mP7bN\\\\\\\/PShVlwiRT1j3stKyC\\\\\\\/eXohjWXDmfy5O4fxH492z9kTk6XY42g0FSktpxom9892WhahBvXg5wS9Lx6RKn3XXVwGNIMmP0PPT15rYPpBtl5ZHx1Djpx5R09f2vpODk+5fZUQLQgTk782TpucJFCM6kLN3oekOe2SmTx8IbGvDfke7jWCFnbCoaS8USoAAsbrO7WOt8eE8\\\\\\\/XvNJW5NhAZ7pijJz8A8VlTzWcsYJ3HwW4Lat21zS+akwEIjKxio1vLWwyYSS4k5B1khJ\\\\\\\/\\\\\\\/jABSSDileoy+oj0pPvtDDwlIYN7FG+t\\\\\\\/g7M\\\\\\\/HRi38NCkc+JV34Lw+2a5lNCBzsmuzneYJbhaTZEKpbpSSSPk6rLFS3NQXQBrQuJHh\\\\\\\/t8lOwIc0qCsdyCJHJJqSQMkpafPAOnDtK0x3HRvp28KPeTf752BfGbMTqbY2A+e9l\\\\\\\/PGx7TXf2o\\\\\\\/tuOpCzpQ1Pf806iEQ02cMzD\\\\\\\/XA1hD3k\\\\\\\/gXMbc+cRrkgtjbD4qbAnVHRpUgUPfhGOguVZcx4ssnZcRjfy95lsCAHAvmd7NV6rAGPf9OrUZqsLPDHHqdiQVtOPrbe7VtnzWz+7X5wwf8S+let4TRBs52Tt6yn5uREe\\\\\\\/YpgxRmmTZ55SpSBbiQZj5LA8FTB43FnAZJeArKcHhKEcRRvTPad\\\\\\\/Dvm4mcTkPBxXCpUA2KLev1mXthslpaWXm6EphIe0t0N3w544EBusfp8QIGe3jk5GBseuJd7Rpe+x120KM+CRqO3dEmdN95pDwfsLVHkfaC0Z5f1fQqy+g7nkScvcxaKxxFbfx3JkIoS72z7ZXSrMYdOhFhSQ+g\\\\\\\/CRVumVVp7i2Sq8zVKEaL8dT8skqzezg2vaQOex7TmnZ9fADrYhxGuWanN82u0RLhKrANTaZflbTkYVj0H3vSVkg7Vlso+MdveROpms\\\\\\\/BSsx7s0rfm9SOcmZOoSLMZzeJYe\\\\\\\/ThHsGTzfqfTGF0bvkyBNLJ0PIIxYSkeOGXABxK3YUf0yM08I1qWZ8eNuejw1N8If50FVR3vDRjSnby+q75JvNrw6xlQ\\\\\\\/z4WrhBJxckNUC6S2ZFUaG86QVm2hRI6I7BVqwcx6PSxXnb4RxV+Z85Ygqe\\\\\\\/+MWQaFcqbr95TMsW4WPlrReixwRAACtCZSWx1L2ovOKbGiYHyq9rE4+SA3rCADbi8OmLtCf5sSCmXqcxXWqDV\\\\\\\/\\\\\\\/CRmeD95SPDg9MP7duD848zeDnfJJYH\\\\\\\/rBtnK4dlq8ogkg\\\\\\\/jIra1\\\\\\\/Y8SDdfECausHZGvLNqvSKq6t16jqJ6Fu7a1h1LJC\\\\\\\/NwO\\\\\\\/WySEseXKCJ\\\\\\\/cmOFjJGNIFmhEGBBjc0A8BQWpBs58bUUSEM1ANhUQw4KoXBKqLyIxCLaNiALVv+0WvG+RDAIubHImUmibcxXP2ZiDQuNbjn4yzNvIvmTr+gSFJ8FYORTMi1b+DnGyNfRPF5jqj0kcTEoAhwFQC05cwvYs3wl1BuyKriBQxpXTMeljleTzh\\\\\\\/zorqVdPF8go6Vv0A6OmHeXUCpgUHfzEjWGkyRCkP6twO2hFl2XoDSQ9GYLAVgcDUqJ378xyhElab+gLBu+602f4V40YldsVdZA6n0\\\\\\\/vprEsBf7fIYkMhN2kFm8rDmbKaN1ljgAbQ+VtoZLSxAC\\\\\\\/EX0go3\\\\\\\/qqY+TYXe8ZpXT7TK8WmQOgn0hnUj5EHh7phxrwYgS2qMfIwyhTvQyLqJe21KRUv+X4oETVoVTVNLQY9vSvikwvQvjgoKSAewYjUxVw4YZRw1KqMf3aZXNmUuqDsTbgq15Mzx9o3Ft+MMI1VcAz39WVPUZ1VPxIeImd0KyZZUmExWFhb4qGPmWHzf0DruNpjkycBg0mFI7rih6g2v0C5+DfcfoKK5pXVhv8o2nqh82PWT\\\\\\\/a2HqZDHuiGxLSRcm54n+H6jBTMffZb0v+nZuoeBV\\\\\\\/\\\\\\\/qK2fifjPiZAJq2XsrcUfgoQ3yFEEhEVc\\\\\\\/A8+8MlD0qhc5BWYJ\\\\\\\/SCvWDN0MT7xGsvXhX7rSopMwQA0\\\\\\\/RJjZWoPCcfPrlMsni65QYPOAgntegUGgNHHkLHBhRXRle4\\\\\\\/qUXMSl+MO6AiNFkrWwJD8SEHKT4CD11fFDJRikU\\\\\\\/OWSpBLk\\\\\\\/ZQpmTZp6Tx9hfMRkxlc3dWwQBoHSrGvXNkoCChMkj\\\\\\\/Vx7sKGmmNjmF2zrkc3zedncLPPZiRtxNdobTM6sLpWz2YOi1FxbLsk6KgofZSg76oWa6QTuzv6+y8iaoiThwcd0p4yxB9vm\\\\\\\/qZr+WXTMcXC91UQDlOwcAWIOjl4L4k6elS421mQjun+Z65M8oj\\\\\\\/hZS88za85WBgqXwQia0o8fMJJBuRM40+IgHEKUS+pWlUwUkparDdEBifExHBX36A5BREHh9P\\\\\\\/su7bvvbP4R4UjDwcP87E+DRA+6UWGNtAiACzhWkTcj0P0hTfPMVrAEHecJYA3e5oPFJLQRc6MuJ0YTguYs5FGN3ak2MU\\\\\\\/o3dgbdk5ZfJSUTpDtF0lde22LX3xBheA7xe6pR1Mk2r2LPf5vneUu+kH5u+7PjXfVO5ilrdApz1fMJAu1bONfliO1X7EN2ba\\\\\\\/\\\\\\\/CV6VmJLTAjb5Yzwd7\\\\\\\/WBt4ySVNXooNNp1nRUhkwLp+R5yGKvejvsuznd9AzWVdl1Yr4Y4Ih+EXIDPCie0m+Zf3pW28qol2ZA+Lx7GZQTAZn6PZRzaFiSzQQK5\\\\\\\/h9pGFQmRknbvnaqKN\\\\\\\/1c7Ky9EtX04iVQrdHQ9viRoF1RR0kQiouR2Y2TcVPZzqIB4U6dP9x4MpuZq7kT05nHhqVe6SH\\\\\\\/u7ACVwWJh0ePAvIVfq3bW99BtSzIteTecTCEkIFySRqORHtPFoKbgyESzD5ZtoATmFdGrTteXKrsfeMzGB7ZOpJh3ootTnDSpo1AOedLn7NgX2EK5V2\\\\\\\/EFTSM\\\\\\\/U5d1xY1JWRZAZMpSpQmGFQsA9qeejlihcOkNIbWnCfNt3S4fmsng9vVfIe+G7X+CYz5xuDTrmhbukd9N57HtPuw7VLyKBk4mEVMvUMFOTq37YuTtbV7juL9NqUuxPbw0ubGTwY2y+TuPSJ4iqUWKQB86aNVdeatxI2Al8V3WGzXl3dPDN9n8gnpQihV9Sz+vWNbkTi0C2tclHLSDjZ7k7MF6mKn8n+8kb4jPpLjIU096wUBQpx0YGG1wPMCB+FNX6lwIedeIbnxc5u7IJ01uhKnjCEc8K2MSnty2levyMf1FC4EybjmKBHC6bZ9WFawo5iWGJE9BdFhIqTsQJqzwVGs6vVCkclbcRQYWHihuoq2JZS+48ES17KqwzALkM2z9W+BftuoxyMcyIyLrRkX8G2A6BlFFpDi4T5am\\\\\\\/2MBxV0aiGio05uORRjMg84QamjmutN76wWv8HFDw36wvpSCGcsDuYkUID+cTdCfPaTLIu2qiflh5CBLHanyEp8VUtEaJju4oZkIPNTJm48OqMQIkYb70JcduncHYCg+gTnGMMEIdiK2zLotsxNIxroWReLofzzvlhNCdA9pAQUD+RPf0rPj+B+rfKqO0iOfUVwTJJ8W9ZiBXlle1An3NaWwr9b0i6IUNdjlsbULCoAhJsSgWlQhBMMrDMECJgTPGpSGynJ7ljN+eKQuNyYfYpLjEFfmDWZ6BHdj9m+3TxVm03fzj0cD6Iq9ESEGzyf1Au55iUVX0zDpzBQfwbB3iAFhT7c+D3YN3A4AzqXNPuVUIcGzR3TfboBp0WPcL5NyC0wTf8u79UX4D+uCak3mRrqxgKRQx+n6ZaIpr\\\\\\\/hZOKt2vbSm\\\\\\\/nQ5iIDqmdnt5hl34RkXzVJhPBK8g+deAjralDgef8mjKAkId9nEXvenCl41A99tXX3B+U8X8DFztsTAIRdwj5yiWqaHgsmeWHoWgJrbzikHAX4HrAB05rS0JC7sJV5ess5FM3CLTDu1XwAsyMoOAO5eF1D08C1BJ3se2GS7DBitL\\\\\\\/FAp8DPCBxMg3EtAEpOkOwoSEGgvO\\\\\\\/+pNf4l3UQnJxodgkqMfwgAu3IMlnyR9\\\\\\\/qpFtRwGiCYGzMhPqCDRokYK7CE3XLt3xXd3mmkAZgZRw78CpGspSqK6EA8Fnsytfq0sPHH0qwoZDkOuseMppinkDiYuue9V8TZTcxE\\\\\\\/5SUORQtA\\\\\\\/dlSRbWf4VzSpo7Aai89UB\\\\\\\/xChoNasTWQduNHrt\\\\\\\/57D6nvTphJvMduL3ENW6H3sXvdKPGYQELFmoK1xcBpCCDoGHoEXjIvKcS0wF+9Zpb80KO61m6VC6sE70LVcBjoAxxtRJmxbHUVsIfleuXjBDB55V\\\\\\\/CgDhsl5+R8eOk3MENxjmN9zvGhhu8XOK7UO\\\\\\\/1jRsXX3LlJm2SqsTudbZkOjZ7Lg1MQJ+g+rE8LYIhRbYQLChp+XqHYW+So8yDx1vWAQqsnlB6o7PCTxc4crjpwssO98WR9aeQmgtNEZi73WibJe8iuDfYd7skOdT8\\\\\\\/snrwA32uhVJlHi92qqYV2VQpcSGuU3fiQ7+WVaND4l3dCESSIXqJDNUyZI37Csftw10PRjKBip18Jw5VVEb1fSHlMuWGIkmak85SaKyscGmPCPRFOchI5aThWCflmDu3yNLkQNeaSfGahGyPsCmExtEce271ntXp5TfKhSIH\\\\\\\/SK6tGlvPb98KWst7XYOQIDfpVUMORt1cQvWHiNGAm6FNkMyjog8ow2pROekX2zlF\\\\\\\/xmy\\\\\\\/yKJIJLvQcXtHmzTEacvVLkkidGpItCRImAU4ifwWxi0I7M20tClg4I6eTo5e7XPCU8tOFtIp6OwgrotLj2Gkz\\\\\\\/k9uQOxWl5P3GTQlg7IoVV+3sFKksxk2C3DRpJCPoaMfzPnQWRaovJzJ7Ep8A2guaF1Wp5yw\\\\\\\/0aW2ukcMKuB54YnOXn\\\\\\\/5ke4nMorIszmYM+c9lURHPBcjYZNKLkfvQw5p7lNJsGSH1fNLe0efEYCWjZZCM7tJv2eODCe9qNAOPviwV3nj8BctsiKODmBlfBAJckOBpdPgJL2m6273f9gnZ72ekZxDgN4zYnipIoQC9mzfPmQino\\\\\\\/9e7KYwyFrabCVcHw4Yq4U09L9Bmc4cF8SmQqx\\\\\\\/YtA640s5wQct77iIWz5Ro7KrGwEy356WSA1tJcXtY73XeLPIx1lw7PQIXbDAycJBuQkhb\\\\\\\/qTaYjOCI2nUK644\\\\\\\/TQtRNY5DngGhi7g8Z1eIWlsfOwUbXsSEF7lwPN9Nc6m86sRZ7zHxjlbBGIP2EtzqxiJfIC0+ZzsXyOLDJKl4LeOCq6Xfjx+sVy3gQj53x9RYL5lXWS4JqQk77ECkwbPD7zlIt4wHsdF6PdB5XMJpDyqawxsCTOT03LzS1ZXu\\\\\\\/klzzMdFy9Hfe4zw\\\\\\\/FHtswGaiE6CUhZmttXsXlogPvhKd5A\\\\\\\/RSDtpjxYF6pIKPPK6jRkA9ISK7KlR7TjOUxqSmB\\\\\\\/ghqrlSuPQpKUFi5i8DdPeSQ3cfAxEkxzzM4OgvX19Lg7IGP06NRPtS9QR03J+aHPHaD7EEEeahoBM0gBP7JdmW+dIYIvd0JawdJyA2dH6EyFpqSZCE82bLjE4i7GtVDrteZA474hsUb6b86HDBGLofAYJAvBDaFbeh+3At7iMxdtQ1Ichc3Q3\\\\\\\/V9WbMjw0vr8L3OHEfQZdmT5V8H88g595fJ\\\\\\\/ZJ7bYCWN7otGd76KDbE42cqFLvaBFXL9XVGKgA4YinqY+Rf\\\\\\\/N0YyVo6z6INzQ3skNDH2+zTHG6umpPKgccxm6SBPY3OV5Yx1JYkbj35wXB8sN3BDfzmDNjOQ66tBnV5Ah40QyuJ6AwjFwMXm4t+ga042l2WePbhYpXkU1mJ7CKbI8UayCd423b58aivEO2Zme7n9sEv2ZPORsLWasorxJM7oOlS\\\\\\\/kTYH7Fxt44uS8XIMzZePgu8PhD7ifbESnlK8LRb2PP2msihgY5wSoir3SUsIqTtJgl0Fe84LLkcNBwu5SDfdDxeroc2Z6gbSEE4YesF7ibHmNlHmaXQFLkniz95hnRN+xc6zaAJovGMhPoCPBQisUxzhdV6Nqy7XbWSrE2P7bO1T1jMn2gWW3B5CYwlKncaO2EzxjJqPZwkgehsHpD1RGOeHgOQSBExov99IE60aVbbMP7xx7H3KVtanlRyizzCMcqvSzsqBfaZGhghkxaMC4U2F6Eeegu\\\\\\\/emzGRcaycg9EaMJk7Nw8MH8oQyvd8tgrILFqqjck1Bn+YZxcGYK2UsxLi5bs1Yl5AC4Ks4HXq8X81zw45c1TudEHBAVgsh0mtxCLZApoK68lbnttAD7W3w4u\\\\\\\/b8qfDT1wZl+6iJMs9q0mXXSQ8pNjO6JF0kfHRENrUDPtNxvPCm4DYNhwNsBVfxpHPmtEi3dMpoclqqlEkKEKfYAqfB+FSVa3212966xnpHaRDewyoCXnDwockkyT0JUMqxeGh0vYafMAe3wWExfYD5mzwC9qG0KvRObo0qljaHjyhhewzDtmTm5CC9SaBSbUL4Xaj8ectcnvjteweSJnrbG4NmNARoXumzowsltaA4A3V9yGVU4wb3dlzFDq9fQnMYXYh6YvydfTXIhXVe8BIzRue2YoAVPzj17dEjwoQ7TvNbgMPGVTLSH2KvBDYWlqZxLQoYBozmKWPzFW79TMMAwqR3F4MhNchcHTx75d0\\\\\\\/dHjQuLjpIIi0BcyxC8+GTi4y58jLoVXm1l+phtUwjqqrE5xa\\\\\\\/UNaIDjHRug51HxeMC0VrFEhi4Fm0rfZSftSn4Qu\\\\\\\/f3Wf9lK6YiEqgXayZdGAMIDctoR8h2JsgJJjLDcAcFvJBO\\\\\\\/cn5rMvGC6OnP5Nx4GYxUih5ar3nk76cIxt9vKqthwfUb93SV2VyfT\\\\\\\/MYcWYuv5ULv94\\\\\\\/M248UVvzCC9C9hW5EPlrn3lr37d7P7BiEWn0Zt7JcRDgyKVC2qvVwQZdAWk+V8o\\\\\\\/Xg7vLd4NxLDDJmvt25SIUUSJH\\\\\\\/AgedVf8d58WfCKXq7cu9Faoky2pdjuHzi5R5iwfIxP1DDWr25eHFsX\\\\\\\/N1WSDklAtAFVhRjwRKBNb3lMeJnHz+9vNx7XSzNzFhAdNS0azOCRwdI6RBtD\\\\\\\/3doM8O3ZRCUKo29vgLRz1PhRi8b6nyCvkxtiuygO9h40UifmTFJ7lA5Q9mIqbcXxr3jeg7DRJDZUSJevijWvpEYEoHRLQDS4b2CWwdkuNs1PoDm8i0eENa1zrVi2FKQogq7NwjcNTmnXBkll8XiSkydQHn2\\\\\\\/sYHsuVCFPvS1jZezQo\\\\\\\/G61qFXDtPnKb\\\\\\\/1nS+3uyWwEyOnfOkaOMwB56avTP63QbbVi0JewA7QZTotiYUL+IShaF6OAyLfg5cBg6yVeKiM9fk2Zv8TLBH0+b2btJEoO+I52AZShMguAHHjPvG07++NZQsiq3clN8RaxsudDumvH7i4aS3PKW4EivlkWfWGamio4jr4LVELF91LmJYGXJy4QWXnzfnWEo\\\\\\\/XHvC+PuTBgD6PvN\\\\\\\/pSPUVX36XPzMxLbAeHNjI8ooQ4yTJwXgkxd9r4jXCeLUASk3kAXIflkrYHsqm7aW4sLQOKzoygYnTTxy8N6EVU3jv1pqaWNbqF59xApHRN+bSv6EaAFL3L53Oy4KayJDPVA6dKCxHTUozowi0QajGGA7jhfVaFGRkSnq4LD5mZ7x+Qnf4o4vw0Hb09nD\\\\\\\/tI7Ptz9Kks3RmSyC1ht5UgjUDLe6iUqa1u++rManLMojSGWTVrHo0iE8Y3Tsynlj7uryySyEYtJIzlr18omcFJpywXIBbRyyfyWqnoZrVBkvvwXUyon0Am+esOprfzkgR1sGtCxiTzGqGkAGODnIK\\\\\\\/NS9c\\\\\\\/QDjhKSXWW4qR4IRoRM+\\\\\\\/c5yhjrQBUzcuExHp6\\\\\\\/It6jj4wWIWX3VnpFrCNJqLGiO6zk\\\\\\\/+eTsso1jwpWAbD\\\\\\\/GVPH6gZB2JyscKUW2dSHFyisIoZA3x0hLi74zD2nMcpjBA5LbK4w\\\\\\\/PRB\\\\\\\/1k8kYXf+t8Hv0YWBMzy7cwOGOXHNamr\\\\\\\/sZSo9aMafXTnRt0fq0AlT4GxqG38J7ruyqBfUGk7ogCq7f\\\\\\\/lK6qSoQ7ok7jf7hXXNIkeNygIV9Eeft2NHclE6zb3jMXPdhvYe2orOkcCgkmUHpRcH3bvjXWLklbsGy4Skyq\\\\\\\/W13Fk8bmkvYCWu7PyB8iz8QynQGNmCdTjsGXYY1SHiP1HqSAHS+9HI2c97o8nfXQjaPN\\\\\\\/+K4qXERFn9Xth2fvqTFGS05P8ZRfaEez\\\\\\\/p5X6lMUNvssQ1PPqxZ1fdq9p6FuItfWQZZofOpl7r75RVNFreAcs5C99k2IbVd59WxrHtEPN\\\\\\\/4ZcCwNlEGFlrKtyLhEUrRs2SNIApSh5P4US0dulgg17bxbQcgJKlPzXsKzsamIUx3Z6\\\\\\\/O8xIu5SrxMQJek2H0vANjt1iHr8p19qZmIJ93SPwIzu+j3wsHLABT61f\\\\\\\/8tgvA\\\\\\\/2NW4xmRc2OrgO11XBfRlZMW3zW1yUTHCDCl2Zxv5+7qKUuwqo2K8CQW7H+hfkpBh\\\\\\\/PbUXZ4WCtrOrc1CVgzfI\\\\\\\/EnsMaanLstfL2gtoeKh8DXgR9b66U4aPqphg6kyRA+X7+03o5UgxCQizqvB1r\\\\\\\/ZX2+GBaK9FNzH8c7H6D4wqox6blM7bdh5sDQIltsqvBjOIAMRAJS+Qb2MCDP+wY4RyFyU\\\\\\\/DLFnBHfxMAv8dAymi\\\\\\\/bDFwGW8Q8PDe8rEpMs+2d1IYNXskLozbv6qkCC6lQT3ke5fJXratnxd5I5JKvo6io5x6iclHdnlTMXvriqUjW\\\\\\\/+c5FdVBXsQRvYZ+TdMrMxk8aVv41tyZ59IEVVgUZdNy\\\\\\\/bglXYTTOLFOZ1mHhQ3+PJwBJuhJTqr7QKNG5nClBe2DSXJJQTnmnAgb71uhFAgAWn2OrwH1Khxbxg2EgLVcz\\\\\\\/R4DfAK9loY9WJ4qIaNeDEpPiol9hV2B+b5IINtL\\\\\\\/WhdtdzBLyH7PEAs7Jb2JsgICQGiL4vJJ8MJXiKnGTApQlc3WY131Y9iAN++35\\\\\\\/MkTgie1NmZXlLSn9uvvbS3z3FsCRwSmjA+U10X0CpPa++QNR3lRrSgFAwgMAL4Mf\\\\\\\/KyOao15KGULxy187785p6IrciDpupG3Yx+rZBfbys1+p3f3bn+eDUGDO\\\\\\\/a\\\\\\\/hqVovsTM\\\\\\\/4i9bd8iRZZx84GPUvsEqjyirE1QXXAfBJFFxCsR1NZW3NbeNoMxCBh0lCzgaVzpJH27NinUHvWsHIMLRG70bo9DWovzoo5cJFjMzdfWvwZQ699cYJy+8trcYpVyZ8bQsp853sXAJn73P856dfBw3Asr+RkvEJtxDgtzmV\\\\\\\/24lTgnyHhvlo1OZhnuDNqzZi5a79ZvFKeSGfcQL+hQQJpaOniUpJ2Xhaq1S1OvK1X\\\\\\\/tVDFwXecOtGhwg\\\\\\\/QFPpvg4U8t+rIndsVIBAh9MW4jnFJDnjOVgy+9zs4oQira6rmWH+CEWGSUSxoDEOuASBJdFQoYw8l8a7V3XmjP4NXcNRaVWM3U+M2bULH82aJqgCJOB9rgqnJmVdy8Jw4bEYn58QlZXBgLfOZl0GrSkj6W1JztjLUBRS26Vks8BRkRulgrle5jopwQH0QDJ05hrIVIgoGNtdiWBgfsdgqETtv3Epsvry0HLhFiYEVcmCncUq6r0FchdpgJphzCdg1sei9USjBD6\\\\\\\/GnbcG6u3Y49ecDDN+922dAty0Z9u7oX70fUEDnCJ2XadQKU0nBHdUESbGS3ACcJwaRIuTNJsVSo5sFXL7i7ZjgIvmt9jmUAZUDk5cRWbr7mhRZRb84UQtVQMMxo3VfraPw4keJK4YdtU4JG2fWWyCQJ6gcvAPf65C7VP09rYgz3EyGL+vI6g4cxUwfAdTmlyhFr+JuEkWNYG0bJqMNjG1iEObPcEtmK4\\\\\\\/OV50\\\\\\\/MAlctPy+iVttNlLJ2DnIn5DfSWXBG35bOhAiB55UJNFhipGWpe9YEW0RgQEKSjajskHs1qiZDXoZoLKLKlXOrwDjYrnVhHAJKeU2YIdqbRb8KIJ\\\\\\\/qo4uflECYxbEczDDvuubS4E+JD2xvbbiB8i7V87Od4j0cv6eDNnMZoFtsWYrIp\\\\\\\/8Tp7qgYrAnfEl3pyOBX11P6AD1ahBDHkCFh4oubs5jQeSt8rvx+BdANgA57GZUW1QR9Ju6rmoIQ6ENQ63N9Djkx8\\\\\\\/KZusyTLSedEnY8F3oFzFbVLFyiR71fxJuTsDG0lJBKga+5BZYoqjSChDlofv4cJve+CPgEjWPKK7U3jTv0Zn5tIGU5e4MkUeMZAIzodA36O407o66qzI8uE9PWrzGmbdB82TR83o3ufG7DiuugEbMTUSFzKEzfhFd97ldi\\\\\\\/ZV4uZFbN\\\\\\\/Xw4VKZ+h4NOdLxIDiUxtwVTxa045+idNphMSXsgHffWGXGlJxp6EJUNQxx22nBDT3lR5Bshj8Mk0Ual\\\\\\\/xNt7e40kB6yYRGq8sJfK6dmmAnIVcMe6ngfyz9L0ws560f4M\\\\\\\/bBcnDNc18tZyTtZDCn3JcmJsFgxEWjQG5sDN2pMNPBTq0gr9IMgzee1\\\\\\\/frTRuqUpugyrcBnrKFOcUi1b51PHO7uriSJ3hZlREs9IP32UwCHvObCR7U2Q754vIS3UaH8BYjC3BzprTMVLDUlUWAg\\\\\\\/JrzQCUDB+CZjyM7h\\\\\\\/WBB2CtnVMwvujp2Y+4xR+8M0ECcNuDNNch9mE7ogcswOFCbFrdKkQrC5MulRPSdMfhPIjr3w3D+sSGCFtvz1NZI92i7UTCQzDAiiWQNkqiKOQwZ3mgrVInDS+fKWv9axhDlXL\\\\\\\/rpuZdSeoIdTmBL8vp0UnsvwP1355Tm6WByB4flw5bSbMONKWSBjxlL7U7x5UyMPMYOYwC9XFtp3tL4u6uOlpRXg4r7yVEHh7AVcKfK+LcXHLPXpyHED4\\\\\\\/R5vA3iaiiC\\\\\\\/ToqyV6GvYTUrgFjzIgeGGaOJkbn\\\\\\\/205m6iz1I9dTaPsvVMY65XLxVfiqsS9LRXD803fzdUREjipqJ82BN9KG80c4DDJjGzGmX1L5AxpXOhQdphOZVcUuG03+nqlbR3xChJgtWf0xqlDeS5ZSNG5r061AleswOazF2\\\\\\\/9GVFERxTJkQ2EPvqdyuILjxQz\\\\\\\/trNn4PBVjri3wn9NHUKJzhlD1MpuumYrnNq+ew01OaRI0zpAk2J0p29Xju5LPHblqDwFR7jIS0M30wgL8cap8N0fD6ZRXY4NJU\\\\\\\/Q1PHszee3tO7PJwY5WY7GyItz4Ju97\\\\\\\/SXCMGci83h+qGQma18IusWzr7+eZ8AEp+1lC+MkN3AX\\\\\\\/yyLtanCcaB9qNYEUosyMqimMbpLpwsHvpP0MIwBGD5NOKbGArYk3US69nWFol9\\\\\\\/CAzPR8wEezNTVPTWNAJfhCYpUR4TGVA7JQLgnR5gh61Kyye5crsr6CQR1qDR+Jfuc7HiZJgmX34ZLbh\\\\\\\/WJr+5vXa1qwZbq1lo14k\\\\\\\/R6RFJF7pqfy+1ymsHpNPdCnrDW7IJER2AUmDiGaU1dcmTNbcUiZJBVS9MiV13OVlU43Ld2k8wt41Wb8C9p1OD497Xp6tSGRwPxyr81rreBURuiInExRr6UI5sl4\\\\\\\/0da5m5LgEPpUGbGjkNK3Hf20NJZ4+vNxPe\\\\\\\/B\\\\\\\/PRsefWFA432s\\\\\\\/r1+FSHKOMRnpX\\\\\\\/Qbdl++BP2ZUZ8mQKSn+trNmVJWiuQTD3PKiyzIivaeZCv3wZxDkoAE3mAEKeBH7tyC\\\\\\\/6h8mGU8xBxeQhQIbaeii7vSDUkidCtoYntWSGvSUF77lBXVYYRszjQgsXcMZsVBxVRFZkI7PpIoepCM5QaM0Rky2H4ShMqQIlYmAZK5ruA+JF+wvg\\\\\\\/g0ZcOCI3Crm\\\\\\\/jZWGZUSrjmzxNATDwERbANiKQeO8pvh3Ttp12nnz1KMzpRqal8acvwIR9XSw6XfcVqZqUr320V8UXD78pG4tdK+yiJXIMThU6IbjbV2si9EFyENREVv6Ev9HrUNQldqfm\\\\\\\/23rzejAwrIStB+V6I4DNAVy\\\\\\\/DNrmWzWouy+Jum4qqxmAMfpKkvvk7t\\\\\\\/VuiBu0RkngGAOY2jpes6XaJHpfFVOCxgGKIMPVQJtW5KThotIgzIeocYL6D9XEkJn0M64emYi0D0+Os3xE3un9QB\\\\\\\/juNisEmvVXKZDrmZRuYabD37aEl\\\\\\\/iHV0hrU6iPfGcV8kBQy8uuf45JHOL3bAf9azMQ0z328iOY0+k3OqjUyGjORGimjI9pGALnxNWO\\\\\\\/hJZ\\\\\\\/TEH7ahci8hUqP1d+JS7LPBLH0rSXvXWfy3GUsHMuZ2mYZGv3+1w3IXJYJEWwGkKCN3ZIP49PahWV0vz3XVEN6YYlwNp0o5aLoXzevfqwFGOp8bslvr045pBl5ptZqkmP0C6XO1BBQRheL8MmF5u1kBKvWqOybChR8qHb4GJ+XjcBgwFA67L89MH5A7DkByHRjUlYAbyFifTu2pRONlfh4w0ZeVhMc4m\\\\\\\/voql1hYkJOx5wKcZTrYz2PjU\\\\\\\/gzuPYBRx17WZeUDSA6rmRLDcGyzZVi23tQYjKe4rsOoZYnH+UkCUALevCJkUcLoVhdXgGzqJFkpBEEHyhH2CxuAbiUdmbQ1H7TSAsXr6owMq0bs9r\\\\\\\/Db2cLG1Lb+Qo\\\\\\\/cmCQyh8ukO68EblQOM53L63w5CVAwxGwL1L99p4BMvuroxdcw6LibH2Eq9YnoPmc1teYzEM+4W+puZ8iIgdwvU\\\\\\\/e\\\\\\\/0Ik3hXBmoC2CpOF6bjJs5yz1sSYmMybC3fYv5xUZEY7zmxKbEtfYa6Ys9HHmSKL86lKmw1Bqbff5et5o0weV3VYvJx746ZqjJnEDOURoConUyR1Zfjbd7Wq7BFv6K+jrJL0\\\\\\\/aid9YYsyevnH+gQaUK7wd3lJmw0DPLXowPfZ8YtLUyYOKXZPb1v5JmN8GvylOcQr6Yj3pex0IwQXT\\\\\\\/ApTeVMUGj\\\\\\\/75A0rFxZSbda6LJF\\\\\\\/KJz0qTFYYHqgQjr6yXLf5WDIQ2TjH5wV00vo2\\\\\\\/3eNykLwVbuWbpjygtJ1XfqPh7slmZU8XoPczi0fGgPATGoUYKa1LcQHXxb\\\\\\\/q47X+06J3t+TgK9qRwK+QTHOkRwnVJwZgRUFh7dxKGdOFoq5YszBleVq8dBbrjUOlK0Ggcbgp2Gj75cu45po\\\\\\\/em1v\\\\\\\/yr\\\\\\\/o1hlOyYhMtJ9qIJfUp\\\\\\\/SH6h6fu+Jb4eGHexQ7D6GRYVIi2zqHd\\\\\\\/c5\\\\\\\/0OzrcV\\\\\\\/m4RJYLt1JFHY7EoN0C47JNruKI3MYQbwt+yea0NO3dwVM1kjvSyazUxl26T+7l7ay7lK0VVt8O1OMyvSi2s2BDSZuwhRqcpRnaSe\\\\\\\/zdV4XUf\\\\\\\/3+QjBWcWS3fmC8OXx+8lWKvnQq4TM7Dx7EzSwf0WN55l3nBniQT9xSMB\\\\\\\/pgqXkwWfZohQuwCGprOOwuDJVeuFEhk3FZoJmg51lgOj0ulriV2wfjREdxS8m6IoM5PCQ+pI5PmFLT+OeUzYdU94WhHEUMxQsphuJ+yY20HINu2EigwXmY6FeYb0WZzzdWbj5VkkeHSYLue1ZW8zSvj4bFkXylFD547uAM6SMIauqM9G5gXpxVafRJmxpjnm00JI9ILc48CWSjsbgOe64EkH4ECNxLh8HwarYbGBPLMOvKvG1iYw9FECU2PpGhxsdHHWI\\\\\\\/qZjAnzx4fgGVY4BNBgyLhY94gd7reV2w+KA232mIqJdWodvHCinaeLOcxEvvaVGhib5UiWMdgi5PxrnR\\\\\\\/TNpeGVcigeogRz6SxF\\\\\\\/gr0w6ToyteeAUzfAV8WtLej5raOwIrtdPaDXK0rJCL12Puz8IMfLR6VE57WDxFdZRVjpUCYGHnrQ27QcHJf+9cg6DZu1Qtb21NmiFBWxKAOcMOeaR3j5TNvdgIutR0gt0iXOk5kvrdpY9kNnKgn2f1R9Y4d3XNJr+lBl0CMqv0ChVc\\\\\\\/kMcogvdnlqMH\\\\\\\/dTogShVVLyJpJ684QUp2CfCjTnTqRvR2tZUZHMmahRDtHlTZkM\\\\\\\/+JHPsjNB\\\\\\\/d+9NAbzlx1M706HuUp7wDyh7VYV+owfKtIgAuA+lDZpiITed8DTh75gbS9Ji1ip32NGZG\\\\\\\/lGBzJM\\\\\\\/6UHTs4tBVH50cSXWVoIcsrg6tBVaFmx0j9RX3y0u515urXjBeieFMGvWWf5dSCyvWz67n0TLuRN4VcSmVgxu0rbwRDq7Mrl435rrjgkuGfIqSXNc+yjklr2WNScbHN3qKAYKjF83SHVkdgFubOYbFyUrxJtlO6On76nPeCYQm1M9MC0aaEczyFFsmVx8JyzdcjFxJtSm9ygQqCidbbOZg9zPr5DD+fsHkwNPlhw5PL6yENCzYfufWkyuIoi\\\\\\\/5txo6ciT8I24iMD1AYqWGlFR6B+I\\\\\\\/cpijasK6qbKbWZ8mLZHovHe4VOEedpCU1aH9EzDYrqpNWNWd2bK1pIH6mljwlGMW9SBZWkIUgJxOyf1wKVVs3sCaZcyJNQM00UJadCzvgtAK6lMZL9IRRyAummzMQ4BGhUR1JpsOCTfDfpK1nhYHM9eNrXsX8B+eomBbILuUReEPO3EMHF\\\\\\\/IXJgtOHGbrt361ypKp\\\\\\\/wIuxsKxcZKdIQDnwrPcGAY5EE6gFfkvZiYaxymaLhSXYE6b4\\\\\\\/EUbVNuDj\\\\\\\/xH0Jo4gwXX4l6iYKFp+fEtH0f7YAgLM55kspwedOpJqj2fmLN755AA+4PEfG8vc33l\\\\\\\/2jQ+5ixcOrJ8kN2Ju3ShRyM6bxcxOw93OSX\\\\\\\/b9EUDbuQh4zQ8+b\\\\\\\/dNkZ9HOl594dONYqgsH3Ryy23ghez2pr9en3YR408QuXEFlmmxF6cfA7rEgMAFxikx6qfuWMVAEEaYwGyPFuz3k8FzUrpmPzJnU5aObNEbuFy3J7VDTEhrF0Hn2lK9AWBz7atWyJXj9cYrkV+jQ4niZut9cYYhR53oDWLyAdpeeaiAHpBWWz1TD6JNQzta4bNKMHgKOakdyq7m6CmaJOgdma7QGZbWqOEbYE1fdrQgfRGcEyg5bUzekAnQitgOBxl1W7RFwsKQMansUp23qGgahZFVCRd\\\\\\\/oceZNkaa4Cl2nKlwPwdA9QPG6dU+yvu\\\\\\\/MeZmOyIxE2nI1pNtFT56+zrltBZVhxVKfWZOwJyuByCLAQZ4YOIbIi2kMWsppuZDZNoaaEEKUohVBO14WByC62Sy+cbiwvnM7Yf2oLwzB2cgiAW3Kpcnh\\\\\\\/G20jFG9yXJ6vF8W8zdy9gmM6hu7de0OYf8C6sQ9gByjnxwYwPlp914aGdZsQBnWXrSa6mGcwvpIXQPiiUnBEFukR4YlfDKat3EUG6BK\\\\\\\/lUkJjfvDDrf9HnlywjkB22uHqnrhdCntgPzSrnGYOj2AfUPyOMm5zCPq8dFKPfLxe7mbce9hx4ItPQH7gz\\\\\\\/WKR\\\\\\\/4zaqTApI8V1USF76IFB6w\\\\\\\/zrNkEaY+veMtPRFEXTX6bucbxq7Ph4RMuAfs9kMizdKxzAbFjIibqMIwoQskUoxiGh25HUCI6+DJ8IsMTtw7gytGndePbMxCQ7pB1ugiFZLPK32Xc3jGNwebwxWFcsNmW9O1d2VlUbfVPl4nf1RF2sBWhjKpetwrInQAi\\\\\\\/676JqkTcDFs\\\\\\\/v886B9n2Uiv5x8\\\\\\\/1fVWMqUUwTGHzibnHAaS6oY618p7Q8pRCpymcXjyLDqfQA71Bw+y8rRsapl9TblY0SaW1h05\\\\\\\/ODGxkvaP2EiVS+O1zy2tUobvGqDiFhTrOLpT3ELlF1s2881ynbLnwlstrGK\\\\\\\/hEsbJvRtvw07hdV1COMubGcX8Bb9fIAnh1xpE5IribN5jXx8MdnpGpyWRDfFu0HY8iQI4lgN2aue40ofkGNnhf1dw78\\\\\\\/gbusoqWI75YYZJhD4NFwRmGQakjDaJpjC51lYivcpEBnXTGY1UWC3CrFU9HC+nnku5LO2T0r8szYWX62F11qaK5c7MceRD4D05tgc\\\\\\\/+HApWgVndbvTPumopA0BKHZFWmADqbAKnc9budYer36\\\\\\\/myV5z2qZ6k\\\\\\\/pi6UwXFtcU9Dua\\\\\\\/5nH1xQW3Utbd2KkRpa9+0we6wbYJJRkOnd+BiOOtC2twW+ukIVQhaajjLy8bBN6YOaHM0RwZA+Fek+KUbt66QOFUHsKCj5ndWHHGyIjneVOL29macpUBC1IRR20x2QStu5Vlmx7bqJxTqLeER7mn86zm2WRm7cj8LPx9Y22VGL2ndkJyrieaU5hHxRZu2cZ+MB15KczhbnY2jhlSyXPRdpD9N6xjyGDKz2AynKwuWbACzPt+yvQsNXIBzdleYfNPbewR22EOdBbtOqc9k1pGAIg9uHlyJbF6RLF+KHnlUr+IV6WkBTO8HkELjAdYW5lljqfAS31jxsyWv\\\\\\\/10vmKe4dzEF9jo2BlvWoO\\\\\\\/L1d2R216kEz4tgSy39w1tUtN6o3nxkX6RRk3ZhR9ZBvQSKbQ6OhXLjiNclCe1\\\\\\\/kIYH5FQ73Pwbg7hv59ZmROFEp73MZTKicqpq\\\\\\\/pVkVJFYyYNLVCtZLK4DODSbhi3zeKMamhcg53saa6ydJRA3Z0xyBcoRn0qEdByKsgtl2MuGFdUy9KGEDzltpcDWOl1MAxI7KTBUfUI96Rj16NmNzbJNqK1SV97W2HsrZfZa\\\\\\\/gHnxzy2iapEYOouvO8L4vvbM6Ayv15QCap69+i65D+X4CG70pN4XyAdTgO6ZrU5gdtsaO2+2G0hNksQ9U1LWwATQw5DmgLq6+Ga5\\\\\\\/0b\\\\\\\/FMHMJfWborKXiKzJXDgiX70DtlmwtIRgBnrdoQtUqs4EPU\\\\\\\/R3u1QRtfvJlU5Wipelbebv3U9BrNaGElApCu\\\\\\\/\\\\\\\/0I91ic4blyt80TpaA5KcWmQ3m5kBlTw8y8QXcRC1Mt+aVl49ZkqUYEaMQx0Tqt2SiVRUI+AEEKzNjq3QkNWAhqX+kk9wHBTtdlZVnbTDVkFc9LFeGQe3e\\\\\\\/2JzCTJ7mELoAgBFgs8yaYM\\\\\\\/c31p73yzWrmKQu\\\\\\\/lopBg76AakFAq+8b+q\\\\\\\/lyb+9KLKPNpLrBRQw7f5GgMCgjyPMPldiRInyRrIUcV\\\\\\\/zlnYzUY8SJrBWwp\\\\\\\/YNw3kwvDFu5Ms2JBIUpwMeIOqpvw6iCTD6OxXnrylP9uB0USr7tzVBK+p4LUeKQIszJDzNV5n2vz8WUzJJeJ8a9YNbUpdtIu8MyNzNWcyge1A51wCu2me0EbsF8iPMLinyyoPF9ojsDSc7Mst5+K7RwDGZ2pjcHodgng16Nv+Xb8dHQ1pJY5g0z195MUR8dNZCBEaPfKmOP740i+R5jOLYXM2vIZBnNNMapHWP8E8zjS5nDK5QAxqeZqQt3UE\\\\\\\/VrUgWnxNi5BcPXJJ726CxaDkLJmbsuaD4Rim5OxFFgLXP2LWuSJcuoojAkxqRa1g+S9QmRSiCMxKEVKwUhaehWIQCXOXAjHstrwSIviGHiUjrmoALBgb1larrl9iqL6NIf5Gt2J2t4\\\\\\\/5ku3oNPaPAwKT\\\\\\\/PZxa+QKcC\\\\\\\/5LGC\\\\\\\/d2wEr4atcax21yp0zLQsjiHpbnWEnvQBuLQWKBEdI4CpIQsSKPBhDk3p1jrvuikAnfIbSDdKX3VjxOcf6FAymouzdeHsrZnFWyLduQYSMDETVTWD+iTDlT+n4RMIu7jIvfkH4KyJeKDiySiY\\\\\\\/zEMz09XSNB\\\\\\\/Hq4HrzY7va43G+lMDo9WNV3PyQydh4qiqFQWGUfuzzZddmsWuvVXG0ZZ6rAzYvcJfxhoHPhdK4OKT0teW7anszAT5qPQPtAhtgYcGzPzmNAow2iFY9QISvCpa2gMEf3GwWHvIEkj6ymg4NpSFm3hhklbfWh8STFCnDodf56ZsT9Z8Lv3Q3+ZAzOB0Rg\\\\\\\/49xaVTetIB5LgqqtzEv30PwVLnCdicC\\\\\\\/p+3Lbg7XlOB\\\\\\\/cSQ7AlDq1LM4uVcjcyLJusTpK6cAlw7tiBgfHiu4DLOvJVizq95kZZ6PY+rNdebl2UvXdTilexrUxJqRQAV0b6QSzY7i51hVXJVaQCKkZUN1Q+VUGNTpzAefcHYx5BWM+RRu1WnsJm+ESYBbhyGevsqonSB8yeMLiaE2jHAX9V7Hf5hiSf3xWIFxPPpJ8ElGASw0F3iilG47u\\\\\\\/xMT7rtVDw3Gnb1k3PfCuyn\\\\\\\/yPFEVo\\\\\\\/LCSB3o948cHtXB15mMcbngteKEsYrLupUM+2uWrMbxpd26XUqX+8cognoIc7OARCTci6d7k1qUH+i6AAPE7fi+XpQbgXcrBtxwCT60cuR3Hc9iIv9B5IShRD1nVZyBeaM5aFWNBNTpJ5IoSnaO8Dp32sbCEYH763ROc74szHlkGXD0fUKTGnC8g6ieHxNRuc2+WvB3wf6BJG7bXkIbng6x4J+l65me\\\\\\\/qQaqynV5BRAWo1ULYJWWWBFimiBt3JptkkKDLb9ie4olNsUe4NkvGHkMJIiK+VYS4IGhxqOxr9dAQ1b\\\\\\\/Y0MPvdDtnEmrI\\\\\\\/7mbVmZYBHfTReqPxYNYiuAcdZJO6kulxOwFUfUOisvH1uHrGTe3JSAmbEfKwPiqhrfHYJuybqODdzfakD1GRXIsL0VsLOxzjhr8ck9f2o8riKhEnav5h2HuUTvgUxRmLlINku9H9a+HoJlwg9\\\\\\\/pSVWXbdhZqIVP2ANwZXa2JTA0liQTVNZLN\\\\\\\/\\\\\\\/1nyA15td9RLgoL2LFNl4jGt3wfRkr6UdAOectvbiOkQvPniLJiUKpB761QJ6fwsnVdIrwKlEiof92Or48rY4JEW\\\u0022,\\\u0022iv\\\u0022:\\\u0022qg0iJSqqjq4VWNln3H7FPA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227acc7a2ec270d654b900f7cca6377c193a7a798414d439280c44ef6c3f9947c8\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022VfY2lJMuttxXOL6MxtddeMpZ9saIrlLP7zsr\\\\\\\/QKeYAwp3XbwKcA\\\\\\\/BIXrSoKSoRwB4X\\\\\\\/stsdhfn+7pJhZVrSlfiPA\\\\\\\/NpB2WFl9g6PUI0Ptwscy6KM8PMu3jjX2gebrePtzhGaKZ3IhGAVNQRAZq3I5oibYGyj4fEuWJqoLlBGla0RDn\\\\\\\/jWX41LM+wTMMh5CQtvbNUkkVtc\\\\\\\/Ym5JbALesizucYixe+9KyrdkKi\\\\\\\/YFVrwnstAD0RRhtyXQjV83QHcFHUUsMjo+xlCEc5cb\\\\\\\/3\\\\\\\/kA+9GITY9amWpNRNQDjJhp4v39A1yXjRwV5jJ\\\\\\\/fE6woOmUpT01iWfiV\\\\\\\/3qOR8nVu5cvAI5BHxi1paGzP3sCJj6YnjqwV5qfDUHqeiD+44MdkdnIPvWqr4sqk0CowlTPx968IueM30GwDHSFhy16e\\\\\\\/o2HAoEJdIpxv39R8g\\\\\\\/vhlly8vcH80oZsRUTthWztUDNPtSspdYTqqzErWUEcSxuX6TbyFKe\\\\\\\/HYBADjrPjBvYFF8qM1iUdIkndvPVGsH6D7iGJg3dTii09U1BjauMD1E\\\\\\\/t0fbBu8bX1vMgdV3r2SUwAQDDF+iiqbp9Gi7jZ6icA25Ui5pb0QFdEY0o\\\\\\\/zADLwlee4FUtG\\\\\\\/twOmc6X7pOhdRuC+FQwztox3qDqZQCg4nhNQ\\\\\\\/m3lkpCvqjJg7bfP4Nrn9C6AyLTWcy9FhaQGT82+rmxmgco\\\\\\\/2ErY9y\\\\\\\/6\\\\\\\/EQczHd0CaCweDxfhAsoFqUmCSROkB\\\\\\\/pBCRAsx3EOjpk1ALRtfebn6breMbuWXdsw9GBwpZZ1X3zHgKY2iYdrQVuldKWVj0qVMuIOvKGnqf5zn+L93Ae3v4WWeemzYAkM5y38gemA1sDJWAQQ2abX\\\\\\\/ceKpSIPZ9J5eZbYQLk230nW6IJbj36TK\\\\\\\/NzseNZR8Vi0C9bumvJiETfMtlhYOzEUt7c0EgyVhATMydFxmwTiYOEIIPwPOvYQz4j2fZyUzF\\\\\\\/WiRYHUh5sp3C4ZDy\\\\\\\/qiQp9REW81q0VHpvztN+HXCHWjTNY2cYm5\\\\\\\/zrpbvyHinedrvqQXOWYKpx5wpLlmMecCCMeCOLOiOXxEk0rBQUUsyvAqvQi72ass17Xxeka1DeRJnUeIgxnN0M3eY+6I94GUEn3BkWTsqIa9hh+oVxmW8dcq\\\\\\\/E05U0GrigFTeuEC0q4u11IMfBx1xoPk6zRB4iUb99pniweV92HTdBzBZ6LQF\\\\\\\/1wZjH3KEbBqEyPicTRe4P4M1bSZU1cnAZZMHALVQqfBBIt1txkh1iPJP8PSprQCvmNDoSrFI0yBKlWULQayYADumR09dXx8AQZnzABGyMsHu6cDvuQMdnO56G2ALaPE\\\\\\\/SKkDjp96\\\\\\\/ZPJNUev4cgxOsuRR9fO+8sriXzhlMbtF1bog0H5Ng+70x9hnLb9ZDIhQeyNH43QzodQYl1R1l2\\\\\\\/0sjK29umGI81tR3LRi9TrEZJFdlentycGGvDVQFjTlwLvUzy9KkmjwjmXo\\\\\\\/vuAOzDAu3Ip5eNQOvXgZTnf7cctmX78oZvYtuvW4f8P9lg7oAoivjTSbQp8YTefjf\\\\\\\/3dh2I3VG669fYDOcXFZUdx8au4ulJ3e1J9RiVeNltP8eEoUY4A\\\\\\\/9V4VNTb71vsHJ9ey8VWdLVytHLHYIQAb9oTaRCuw5WNJViEOUeMG8hkqw6dUMI6oO4L4ul1yNzzlYGPZbb+XF16nlaE8vTmsXr0RG0PvybABuiD8zYIhKbY\\\\\\\/Zjy0oUBrbjgh\\\\\\\/GTkgSqHAO6\\\\\\\/4Rc3ggIEstfoiKGo4YwCj8c2poM4kMusLbLVlebe1oITf5o2khoiAKgBsvoSHmKl\\\\\\\/oGj1l30zElbgW8Kb8XOh8\\\\\\\/cY3mYXVhu2kbjp8+yXxdKIC1KQp0sYMlc3Z9klAx6MvVF9uiwsFdIzImXI8TW5bRWRDqJzmLP8Bbw2OUHaTUYPryWeGg3IQrZDR5QDXb9aNyHjMlfNpBpdRYf+6b8fhlaE\\\\\\\/8PPjnqPQrJCUHXCgGnd9KYPJY\\\\\\\/OMglB8yq6ec3VxUqwf9\\\\\\\/2Up209oOWmMu871sAgQZZe3cI=\\\u0022,\\\u0022iv\\\u0022:\\\u00224D27ThnoFSlxfr80HZkbRQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c93220ebc2d90683640963629c31a300ab2f551b6eea9ce4f59142af2eb80a13\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-483518 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/3/1027" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Fa1kQbWwelANg1n_b8SgoJIP6UaM84YsgqdcvAJqPoQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/3/1027" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0JyG4x66Oq1vmYwGYLvnNFkTMejO2xhC2XIaagJDgFQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/3/1019" title="Allelic Dropout During Polymerase Chain Reaction due to G-Quadruplex Structures and DNA Methylation Is Widespread at Imprinted Human Loci" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/3/1039" title="Saccharomyces cerevisiae FLO1 Gene Demonstrates Genetic Linkage to Increased Fermentation Rate at Low Temperatures" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483518" id="node483518" data-pisa="ggg;7/3/1027" data-pisa-master="ggg;g3.117.039651" data-apath="/ggg/7/3/1027.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Temporal Perspective on the Interplay of Demography and Selection on Deleterious Variation in Humans</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Evan Koch</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5345-0214" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>John Novembre</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1027-1037;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.039651 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483518" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Evan Koch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolution, University of Chicago, Illinois 60637</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Evan%2BKoch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Koch%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEvan%2BKoch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John Novembre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolution, University of Chicago, Illinois 60637</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Human Genetics, University of Chicago, Illinois 60637</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BNovembre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Novembre%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BNovembre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5345-0214" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John Novembre</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jnovembre@uchicago.edu" class="" data-icon-position="" data-hide-link-title="0">jnovembre@uchicago.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/3/1027" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:483518" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:483518/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/3/1027.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:483518" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:483518/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/3/1027.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:483518" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:483518/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/3/1027.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:483518" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:483518/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/3/1027.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:483518" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:483518/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">When mutations have small effects on fitness, population size plays an important role in determining the amount and nature of deleterious genetic variation. The extent to which recent population size changes have impacted deleterious variation in humans has been a question of considerable interest and debate. An emerging consensus is that the Out-of-Africa bottleneck and subsequent growth events have been too short to cause meaningful differences in genetic load between populations; though changes in the number and average frequencies of deleterious variants have taken place. To provide more support for this view and to offer additional insight into the divergent evolution of deleterious variation across populations, we numerically solve time-inhomogeneous diffusion equations and study the temporal dynamics of the frequency spectra in models of population size change for modern humans. We observe how the response to demographic change differs by the strength of selection, and we then assess whether similar patterns are observed in exome sequence data from 33,370 and 5203 individuals of non-Finnish European and West African ancestry, respectively. Our theoretical results highlight how even simple summaries of the frequency spectrum can have complex responses to demographic change. These results support the finding that some apparent discrepancies between previous results have been driven by the behaviors of the precise summaries of deleterious variation. Further, our empirical results make clear the difficulty of inferring slight differences in frequency spectra using recent next-generation sequence data.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/deleterious-variation" class="hw-term hw-article-keyword hw-article-keyword-deleterious-variation" rel="nofollow">deleterious variation</a></li><li class="kwd"><a href="/keyword/demography" class="hw-term hw-article-keyword hw-article-keyword-demography" rel="nofollow">demography</a></li><li class="kwd"><a href="/keyword/human-genetics" class="hw-term hw-article-keyword hw-article-keyword-human-genetics" rel="nofollow">human genetics</a></li><li class="kwd"><a href="/keyword/nearly-neutral-theory" class="hw-term hw-article-keyword hw-article-keyword-nearly-neutral-theory" rel="nofollow">nearly neutral theory</a></li><li class="kwd"><a href="/keyword/selection" class="hw-term hw-article-keyword hw-article-keyword-selection" rel="nofollow">selection</a></li></ul><p id="p-3">Inferences comparing the relative strengths of selection in different populations are particularly difficult in populations with nonequilibrium demographic histories (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Brandvain and Wright 2016</a>). Yet understanding differences in selection on deleterious variation between populations is vital to explaining observed patterns of genetic variation.</p><p id="p-4">One major context for research on deleterious variation has been in studies of human populations (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lohmueller 2014a</a>). Nonsynonymous variants in humans show many patterns that are similar to those seen for neutral variants (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Yu <em>et al.</em> 2002</a>). Studies have observed decreased nonsynonymous heterozygosity and increased derived allele homozygosity in European populations relative to African ones (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Lohmueller <em>et al.</em> 2008</a>). These effects increase as distance from Africa increases (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Henn <em>et al.</em> 2012</a>, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">2016</a>). However, considerable debate and interest has focused on, beginning from <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Lohmueller <em>et al.</em> (2008)</a>, the finding that European populations have proportionally more deleterious variation than neutral variation when compared to African populations (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Peischl <em>et al.</em> 2013</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Tennessen <em>et al.</em> 2012</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Torkamani <em>et al.</em> 2012</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Subramanian 2012</a>). <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Simons <em>et al.</em> (2014)</a> and <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Do <em>et al.</em> (2015)</a> show that this does not imply a biologically important difference in the deleterious allele burden or genetic load. These two studies fail to detect a difference between populations in the mean number of putatively deleterious variants contained in a single genome. <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Lohmueller (2014a)</a> explains that the apparent differences between the <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Simons <em>et al.</em> (2014)</a> and <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Do <em>et al.</em> (2015)</a> studies <em>vs.</em> previous ones is due to different ways of summarizing patterns of genetic variation because previous studies did not use statistics for detecting a difference in genetic load. However, subsequent work has generated conflicting observations of the derived allele burden. <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Fu <em>et al.</em> (2014)</a> and <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Henn <em>et al.</em> (2016)</a> observe greater numbers of deleterious variants in genomes from Out-of-Africa (OOA) populations, but the first did not perform any formal statistical test, and the test used by the second does not account for variation in the evolutionary process [as noted by <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Simons and Sella (2016)</a> and <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gravel (2016)</a>]. <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Simons and Sella (2016)</a> analyze different data and conclude that there is little or no difference in nonsynonymous allele burden or genetic load among contemporary human populations.</p><p id="p-5">Recently, <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Brandvain and Wright (2016)</a> summarize the results of many of these studies (and related ones in nonhumans) and emphasize that, while many insights have been gained by describing differences in the distribution of deleterious variation between populations, further work is necessary in order to interpret these differences in terms of natural selection. Studies of deleterious variation often focus on the genetic load or the evolution of fitness differences between populations and species. However, the distribution of deleterious variation has other important consequences including effects on the trait architecture (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Simons <em>et al.</em> 2014</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gazave <em>et al.</em> 2013</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Eyre-Walker 2010</a>) and the future evolution of the population.</p><p id="p-6">Here, we use numerical solutions of diffusion equations for the frequency spectrum and analyze the empirical site frequency spectrum of deleterious alleles in a large-scale human sequencing study. Analyzing numerical solutions can help illustrate the response of frequency spectrum summaries to changes in population size; our goal is to show how equilibrium logic can mislead because deleterious genetic variation has complex responses to different demographies.</p><p id="p-7">We do this by stratifying expected changes by the strength of purifying selection and by time. We first investigate two simple demographic events: (1) a size reduction and (2) a size reduction followed by growth. We then examine previously fitted demographic models for West African and European population history. Like many studies, we take these two models as examples of African and OOA population histories and hereafter refer to them as such. We elaborate further on differences between African and OOA populations by addressing the relative impacts of drift <em>vs.</em> selection and by analyzing the properties of a commonly used statistic: the proportion of sample variants that are deleterious. We then compare theoretical predictions to patterns of heterozygosity, derived allele burden, and homozygosity in the Exome Aggregation Consortium data set (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lek <em>et al.</em> 2016</a>) when stratified by a measure predictive of evolutionary constraint (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cooper <em>et al.</em> 2005</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Davydov <em>et al.</em> 2010</a>). Relative to previous work examining the response of deleterious variation to demographic events, we emphasize the temporal pattern and sensitivity to the degree of purifying selection.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Basic model assumptions and numerical solutions</h3><p id="p-8">To model the evolution of the site frequency spectrum through time, we use the diffusion approximation to a Wright-Fisher model with selection in an infinitely many sites model with no linkage (Poisson random field model) (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Ewens 2004</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Sawyer and Hartl 1992</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hartl <em>et al.</em> 1994</a>). Since the demographies of many human populations are far from equilibrium, we numerically solve for the time evolution of the derived allele frequency density using a forward Kolmogorov diffusion equation similar to that first widely applied in population genetics by <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kimura (1964)</a>. Specifically, we use a numerical solution described by <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Evans <em>et al.</em> (2007)</a> to obtain the function <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-1.gif"/></noscript></span></span> such that <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-2.gif"/></noscript></span></span> gives approximately the expected number of derived alleles in the small range <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-3.gif"/></noscript></span></span> where <em>x</em> is the frequency of a derived mutation. <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-4.gif"/></noscript></span></span> is the <em>frequency spectrum</em> of the population. This approach is similar to that used in frequency spectrum-based methods of estimating demographic histories but differs trivially in the boundary conditions (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Williamson <em>et al.</em> 2005</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gutenkunst <em>et al.</em> 2009</a>). We use an additive model of selection where derived allele heterozygotes and homozygotes have relative fitnesses <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-5.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-6.gif"/></noscript></span></span>, respectively. The parameter <em>s</em> is the strength of selection against the derived variant. The system we solve numerically is<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-7.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>Here, <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-8.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-9.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-10.gif"/></noscript></span></span> is the initial population size, <em>l</em> is the number of sites under consideration, and <em>μ</em> is the per base pair mutation rate. Time (<em>t</em>) is measured in <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-11.gif"/></noscript></span></span> generations. <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-12.gif"/></noscript></span></span> is the population size at time <em>t</em> relative to <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-13.gif"/></noscript></span></span> The numerical solutions use a nonuniform grid on <em>x</em> with finer spacing at low values to account for the highly peaked nature of the frequency spectrum that arises during strong selection and population growth (see Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039651/-/DC1/FileS1.pdf">File S1</a> for details). The grid on <em>t</em> uses a step size of <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-14.gif"/></noscript></span></span> coalescent units, and increasing the resolution does not affect results.</p><p id="p-9">To validate our implementation of this methodology we performed numerical solutions for a constant size population and compared them to equilibrium expressions to determine the deviation due to inherent error in the numerical scheme. Doing so shows that low-order moments of the frequency spectrum are stable enough that subsequent results are nearly unaffected (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039651/-/DC1/FigureS2.eps">Figure S2</a>). We also compare diffusion results to the Wright–Fisher Markov model it approximates and find very close agreement (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039651/-/DC1/FileS1.pdf">File S1</a> for details).</p><p id="p-10">For many results we consider the expected <em>site frequency spectrum</em> (SFS) in a sample of <em>n</em> haploid genomes. Assuming sampling with replacement from the population this is given by<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-15.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>(<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Hartl <em>et al.</em> 1994</a>), where <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-16.gif"/></noscript></span></span> is the number of alleles at count <em>k</em> in a sample of size <em>n</em>. It can be seen that <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-17.gif"/></noscript></span></span> depends on moments <em>n</em> through <em>k</em> of the frequency spectrum. For instance, <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-18.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-19.gif"/></noscript></span></span> is the <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-20.gif"/></noscript></span></span> moment of <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-21.gif"/></noscript></span></span> Equation 2 is computed by numerical integration over the grid on <em>x</em> for which <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-22.gif"/></noscript></span></span> (Equation 1) was solved.</p></div><div id="sec-3" class="subsection"><h3>Demographic scenarios</h3><p id="p-11">As examples of African and OOA population histories we use the demographies inferred by <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Tennessen <em>et al.</em> (2012)</a> from a large sample of 1088 African-American and 1351 European-American individuals. These demographies are characterized by an OOA bottleneck (13%, 2000 generations ago), a second European bottleneck (55%, 920 generations ago) with immediate recovery at a rate of 0.31%, and recent exponential growth in both populations (1.95% in European, 1.66% in African, 205 generations ago, <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/1027/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Representative population histories used for African and Out-of-Africa demographic models. The estimated effective population sizes as a function of time estimated by Tennessen et al. (2012). Demographic events are shown by dashed vertical lines. Lines of the same color denote the same event in subsequent plots. Event a is an approximate doubling of the population size before the OOA split. Event b is the OOA bottleneck (). Event c is a bottleneck () followed by exponential growth (). Event d corresponds to recent and rapid population growth experienced by both populations (African: 1.66% OOA: 1.95%). This is a simplification of the Tennessen et al. (2012) model because it ignores a low rate of migration inferred to have occurred post bottleneck between the African and OOA populations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-37178502" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Representative population histories used for African and Out-of-Africa demographic models. The estimated effective population sizes as a function of time estimated by Tennessen &lt;em&gt;et al.&lt;/em&gt; (2012). Demographic events are shown by dashed vertical lines. Lines of the same color denote the same event in subsequent plots. Event &lt;em&gt;a&lt;/em&gt; is an approximate doubling of the population size before the OOA split. Event &lt;em&gt;b&lt;/em&gt; is the OOA bottleneck (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-23&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). Event &lt;em&gt;c&lt;/em&gt; is a bottleneck (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-24&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) followed by exponential growth (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-25&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). Event &lt;em&gt;d&lt;/em&gt; corresponds to recent and rapid population growth experienced by both populations (African: 1.66% OOA: 1.95%). This is a simplification of the Tennessen &lt;em&gt;et al.&lt;/em&gt; (2012) model because it ignores a low rate of migration inferred to have occurred post bottleneck between the African and OOA populations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/1027/F1.medium.gif" width="440" height="184"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/3/1027/F1.medium.gif" width="440" height="184"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/1027/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/1027/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483581" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-12" class="first-child">Representative population histories used for African and Out-of-Africa demographic models. The estimated effective population sizes as a function of time estimated by <a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Tennessen <em>et al.</em> (2012)</a>. Demographic events are shown by dashed vertical lines. Lines of the same color denote the same event in subsequent plots. Event <em>a</em> is an approximate doubling of the population size before the OOA split. Event <em>b</em> is the OOA bottleneck (<span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F1/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F1/embed/mml-math-23.gif"/></noscript></span></span>). Event <em>c</em> is a bottleneck (<span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F1/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F1/embed/mml-math-24.gif"/></noscript></span></span>) followed by exponential growth (<span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F1/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F1/embed/mml-math-25.gif"/></noscript></span></span>). Event <em>d</em> corresponds to recent and rapid population growth experienced by both populations (African: 1.66% OOA: 1.95%). This is a simplification of the <a id="xref-ref-52-4" class="xref-bibr" href="#ref-52">Tennessen <em>et al.</em> (2012)</a> model because it ignores a low rate of migration inferred to have occurred post bottleneck between the African and OOA populations.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Analysis of exome sequence data</h3><p id="p-13">We analyze exome sequence data from 33,370 individuals taken from a non-Finnish European (NFE) ancestry cluster and 5203 individuals from a West African (AFR) ancestry cluster from the Exome Aggregation Consortium (ExAC) (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Lek <em>et al.</em> 2016</a>). An advantage of the large size of the ExAC sample is that it provides more precise estimates of heterozygosity and derived allele frequency at low-frequency sites such as those under strong purifying selection. Ancestry clusters were determined by the ExAC authors using principal components analysis. We use the NFE and AFR clusters to roughly correspond to the OOA and African population models respectively. Variants were called in the data by the ExAC authors, and we filtered variants based on information they provide. This involves filtering variants by their variant quality score log-odds (VQSLOD) to obtain a set with a tranche sensitivity level of <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-26.gif"/></noscript></span></span> in the ExAC training set and then removing sites with missing data in &gt;<span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-27.gif"/></noscript></span></span> of samples in both the African and NFE groups (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039651/-/DC1/FileS1.pdf">File S1</a> for more details).</p><p id="p-14">To obtain a set of high-confidence derived alleles we first subset the data by only considering sites where the human-chimpanzee ancestral state is inferred with high confidence in the six primate EPO (Enredo, Pecan, Ortheus) alignments (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Durbin <em>et al.</em> 2010</a>). In a sample as large as ExAC, some sites will have both experienced a substitution along the human lineage and be polymorphic in the sample. At such sites the identity of the last substitution (the ancestral state) is not obvious. In such cases, if the human–chimpanzee ancestral allele is present at a site, then we call all other alleles derived. Otherwise, we assign the highest frequency allele as ancestral and call all other alleles derived. Using a more sophisticated procedure, similar to that of <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hernandez <em>et al.</em> (2007)</a>, to correct the SFS for misidentification of derived alleles does not substantially affect results (results not shown), and so for efficiency we use the simpler procedure.</p><p id="p-15">As a measure of selection against derived alleles, we used rejected substitution scores obtained through Genomic Evolutionary Rate Profiling (GERP) (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Cooper <em>et al.</em> 2005</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Davydov <em>et al.</em> 2010</a>), which we hereafter refer to as GERP scores. High GERP scores indicate greater levels of phylogenetic constraint. When analyzing data by GERP score, we divide the observed range into 20 equally spaced bins along the GERP axis and following <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Henn <em>et al.</em> (2016)</a> put all sites with a score <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-28.gif"/></noscript></span></span> into a separate bin because these are a mix of highly constrained and poorly aligned sites. SDs were calculated by bootstrapping across sites within a GERP bin.</p></div><div id="sec-5" class="subsection"><h3>Data availability</h3><p id="p-16">Code used to run numerical solutions was implemented in <em>R</em> and is available at <a href="http://github.com/emkoch/sfs-num-analysis">http://github.com/emkoch/sfs-num-analysis</a>. ExAC data and GERP scores are both publicly available and were downloaded from <a href="http://exac.broadinstitute.org/downloads">http://exac.broadinstitute.org/downloads</a> and <a href="http://mendel.stanford.edu/SidowLab/downloads/gerp/">http://mendel.stanford.edu/SidowLab/downloads/gerp/</a>, respectively.</p></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><p id="p-17">We analyze the dynamics of deleterious variation by first exploring evolution within populations and then move on to differences between populations and compare results from the OOA and African models to data. To begin our analysis we use two basic example demographies, taking parameters from a model of OOA demographic history (<a id="xref-ref-52-5" class="xref-bibr" href="#ref-52">Tennessen <em>et al.</em> 2012</a>).</p><div id="sec-7" class="subsection"><h3>The response of heterozygosity to simple demographies</h3><div id="sec-8" class="subsection"><h4>Bottleneck:</h4><p id="p-18">At equilibrium under the infinite sites model a smaller population will always have lower expected heterozygosity regardless of the strength of selection (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Charlesworth and Charlesworth 2010</a>). When the population size drops from <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-29.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-30.gif"/></noscript></span></span> and equilibrium is reached, heterozygosity at neutral sites is reduced by the fraction <em>ρ</em>. At sites under purifying selection heterozygosity is also reduced, but the fractional reduction is less than <em>ρ</em>. Sites under very strong selection (<span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-31.gif"/></noscript></span></span>) experience almost no heterozygosity decrease because these are maintained by a mutation-selection balance that is insensitive to population size (<a id="xref-ref-50-4" class="xref-bibr" href="#ref-50">Simons <em>et al.</em> 2014</a>).</p><p id="p-19">After a bottleneck, the approach to the new equilibrium heterozygosity is not always monotonic through time, as shown here by the numerical solutions for a population starting at equilibrium and going through a prolonged bottleneck (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). If sites are neutral or have a <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-32.gif"/></noscript></span></span> value with magnitude <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-33.gif"/></noscript></span></span> following the bottleneck, then the approach to the new equilibrium value is monotonic. When the magnitude of <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-34.gif"/></noscript></span></span> is <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-35.gif"/></noscript></span></span> then the bottleneck causes heterozygosity to undershoot its new equilibrium value (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). Breaking down the expected heterozygosity into its contributions from alleles at different frequencies shows that the undershooting is due to a faster heterozygosity loss from fixation of low-frequency variants than heterozygosity increase from variants drifting to intermediate frequencies (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>). Heterozygosity later increases as many variants drift to higher frequencies and compensate for the initial loss.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/1027/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The response of heterozygosity at sites under purifying selection to a prolonged bottleneck. In each panel,  corresponds to the population size at the start of the trajectory (before the bottleneck) and  to the population size after the bottleneck. (A) The response of heterozygosity to a population bottleneck for selection coefficients that are neutral or nearly neutral throughout. (B) Heterozygosity trajectories for selection coefficients that are strongly deleterious before the bottleneck and nearly neutral afterward. (C) How the total heterozygosity is distributed across different frequency alleles by plotting the contribution to heterozygosity () at different time points in () for the orange trajectory from B. Integrating over the contributions to heterozygosity from zero to one gives the total expected heterozygosity. The frequency spectrum shifts from being strongly deleterious before the bottleneck to nearly neutral afterward. The initial loss of variation at low frequency (times  ) is not immediately compensated for by increased variation at higher frequency as would occur at equilibrium. The difference between the time it takes to lose rare variants and the time it takes to accumulate variants at higher frequencies explains the heterozygosity dip and recovery in B." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-37178502" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The response of heterozygosity at sites under purifying selection to a prolonged bottleneck. In each panel, &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-36&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; corresponds to the population size at the start of the trajectory (before the bottleneck) and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-37&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; to the population size after the bottleneck. (A) The response of heterozygosity to a population bottleneck for selection coefficients that are neutral or nearly neutral throughout. (B) Heterozygosity trajectories for selection coefficients that are strongly deleterious before the bottleneck and nearly neutral afterward. (C) How the total heterozygosity is distributed across different frequency alleles by plotting the contribution to heterozygosity (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-38&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) at different time points in (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-39&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) for the orange trajectory from B. Integrating over the contributions to heterozygosity from zero to one gives the total expected heterozygosity. The frequency spectrum shifts from being strongly deleterious before the bottleneck to nearly neutral afterward. The initial loss of variation at low frequency (times &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-40&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-41&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) is not immediately compensated for by increased variation at higher frequency as would occur at equilibrium. The difference between the time it takes to lose rare variants and the time it takes to accumulate variants at higher frequencies explains the heterozygosity dip and recovery in B.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/1027/F2.medium.gif" width="440" height="148"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/3/1027/F2.medium.gif" width="440" height="148"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/1027/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/1027/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483689" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">The response of heterozygosity at sites under purifying selection to a prolonged bottleneck. In each panel, <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F2/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F2/embed/mml-math-36.gif"/></noscript></span></span> corresponds to the population size at the start of the trajectory (before the bottleneck) and <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F2/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F2/embed/mml-math-37.gif"/></noscript></span></span> to the population size after the bottleneck. (A) The response of heterozygosity to a population bottleneck for selection coefficients that are neutral or nearly neutral throughout. (B) Heterozygosity trajectories for selection coefficients that are strongly deleterious before the bottleneck and nearly neutral afterward. (C) How the total heterozygosity is distributed across different frequency alleles by plotting the contribution to heterozygosity (<span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F2/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F2/embed/mml-math-38.gif"/></noscript></span></span>) at different time points in (<span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F2/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F2/embed/mml-math-39.gif"/></noscript></span></span>) for the orange trajectory from B. Integrating over the contributions to heterozygosity from zero to one gives the total expected heterozygosity. The frequency spectrum shifts from being strongly deleterious before the bottleneck to nearly neutral afterward. The initial loss of variation at low frequency (times <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F2/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F2/embed/mml-math-40.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F2/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F2/embed/mml-math-41.gif"/></noscript></span></span>) is not immediately compensated for by increased variation at higher frequency as would occur at equilibrium. The difference between the time it takes to lose rare variants and the time it takes to accumulate variants at higher frequencies explains the heterozygosity dip and recovery in B.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h4>Bottleneck + growth:</h4><p id="p-21"><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows how heterozygosity approaches mutation-selection balance in a population starting from equilibrium, going through a bottleneck, and then growing exponentially. The initial heterozygosity drop is similar to that in the lone bottleneck model, but the recovery is rapid as the population grows exponentially. Similar to how heterozygosity in the bottleneck model can undershoot its equilibrium value, in the bottleneck + growth model it overshoots the asymptotic value at mutation-selection balance. Both cases demonstrate how easily equilibrium intuition can fail. For a period following a bottleneck heterozygosity may be increasing, and conversely, heterozygosity can be decreasing during a period of population growth.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/1027/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The response of heterozygosity at sites under purifying selection to a bottleneck followed by exponential growth. (A and B) The response of heterozygosity to a bottleneck of about 50% followed by exponential growth.  corresponds to the population size at the start of the trajectory (before the bottleneck). Heterozygosity initially drops due to the bottleneck, regardless of selection coefficient, but begins to increase as the population size grows. When the population size becomes large relative to the selection coefficient, heterozygosity overshoots the equilibrium value that it would approach for mutation-selection balance. (C) How the total heterozygosity is distributed across different frequency alleles by plotting the contribution to heterozygosity () at different points in time () for the orange trajectory in B. This demonstrates how the contribution to heterozygosity shifts toward lower frequency alleles as the population size grows." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-37178502" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The response of heterozygosity at sites under purifying selection to a bottleneck followed by exponential growth. (A and B) The response of heterozygosity to a bottleneck of about 50% followed by exponential growth. &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-42&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; corresponds to the population size at the start of the trajectory (before the bottleneck). Heterozygosity initially drops due to the bottleneck, regardless of selection coefficient, but begins to increase as the population size grows. When the population size becomes large relative to the selection coefficient, heterozygosity overshoots the equilibrium value that it would approach for mutation-selection balance. (C) How the total heterozygosity is distributed across different frequency alleles by plotting the contribution to heterozygosity (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-43&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) at different points in time (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-44&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) for the orange trajectory in B. This demonstrates how the contribution to heterozygosity shifts toward lower frequency alleles as the population size grows.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/1027/F3.medium.gif" width="440" height="148"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/3/1027/F3.medium.gif" width="440" height="148"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/1027/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/1027/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483715" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-22" class="first-child">The response of heterozygosity at sites under purifying selection to a bottleneck followed by exponential growth. (A and B) The response of heterozygosity to a bottleneck of about 50% followed by exponential growth. <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F3/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F3/embed/mml-math-42.gif"/></noscript></span></span> corresponds to the population size at the start of the trajectory (before the bottleneck). Heterozygosity initially drops due to the bottleneck, regardless of selection coefficient, but begins to increase as the population size grows. When the population size becomes large relative to the selection coefficient, heterozygosity overshoots the equilibrium value that it would approach for mutation-selection balance. (C) How the total heterozygosity is distributed across different frequency alleles by plotting the contribution to heterozygosity (<span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F3/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F3/embed/mml-math-43.gif"/></noscript></span></span>) at different points in time (<span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F3/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F3/embed/mml-math-44.gif"/></noscript></span></span>) for the orange trajectory in B. This demonstrates how the contribution to heterozygosity shifts toward lower frequency alleles as the population size grows.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-10" class="subsection"><h3>The response of <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-45.gif"/></noscript></span></span> to complex demography</h3><p id="p-23">The complex behavior of heterozygosity in simple demographic scenarios suggests the difficulty of comparing deleterious genetic variation between populations. When we consider the OOA trajectory fitted by <a id="xref-ref-52-6" class="xref-bibr" href="#ref-52">Tennessen <em>et al.</em> (2012)</a> (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>), the response of heterozygosity to the bottleneck and bottleneck + growth events is similar to when these events are considered in isolation (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039651/-/DC1/FigureS1.eps">Figure S1</a>). The response becomes complex when we consider the evolution of a more elaborate summary: the proportion of sample variants that are predicted to be deleterious.</p><p id="p-24">The proportion of sample variants that are predicted to be deleterious has been used as a statistic to look for differences in the distribution of deleterious genetic variation between human populations [reviewed by <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Lohmueller (2014a)</a>]. We write this proportion as <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-46.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-47.gif"/></noscript></span></span> is the number of deleterious polymorphic sites, and <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-48.gif"/></noscript></span></span> is the total number of polymorphic sites (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039651/-/DC1/FileS1.pdf">File S1</a> for details of calculating this quantity). <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-49.gif"/></noscript></span></span> depends on sample size because it counts all variants equally regardless of frequency, and larger sample sizes will contain a greater proportion of rare variants. In empirical human studies, a higher <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-50.gif"/></noscript></span></span> is found in OOA populations compared to African ones when the sample size is relatively small [<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Peischl <em>et al.</em> (2013)</a>, <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-51.gif"/></noscript></span></span> African and <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-52.gif"/></noscript></span></span> non-African ancestry individuals]. However, a study with a much larger sample size found no difference in <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-53.gif"/></noscript></span></span> between African and European ancestry samples [<a id="xref-ref-52-7" class="xref-bibr" href="#ref-52">Tennessen <em>et al.</em> (2012</a>), <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-54.gif"/></noscript></span></span> African and <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-55.gif"/></noscript></span></span> non-African ancestry individuals)].</p><p id="p-25"><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> shows how demography, selection, and sample size interact to determine the evolution of <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-56.gif"/></noscript></span></span> The OOA bottleneck 2000 generations ago initially causes a drop in <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-57.gif"/></noscript></span></span> (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>) because deleterious alleles are more likely than neutral ones to be at low frequencies and therefore lost during the bottleneck. Interestingly, if <em>s</em> is large and the sample size is small enough (<em>e.g.</em>, <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-58.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-59.gif"/></noscript></span></span>), then OOA <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-60.gif"/></noscript></span></span> can increase above African levels within the duration of the bottleneck.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/1027/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Temporal trajectories of  We plot  trajectories over time in order to show how patterns in Figure S5 are created for (A) weak (), (B) intermediate (), and (C) strong () selection. Samples of size two (heterozygosity proportions) result in increased  following the OOA bottleneck, while larger samples show a decrease first and may or may not begin to rise before the bottleneck ends. The OOA model  outpaces the African one for most sample sizes during the population growth following event c." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-37178502" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Temporal trajectories of &lt;strong&gt;&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-61&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&lt;/strong&gt; We plot &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-62&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; trajectories over time in order to show how patterns in Figure S5 are created for (A) weak (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-63&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;), (B) intermediate (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-64&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;), and (C) strong (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-65&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) selection. Samples of size two (heterozygosity proportions) result in increased &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-66&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; following the OOA bottleneck, while larger samples show a decrease first and may or may not begin to rise before the bottleneck ends. The OOA model &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-67&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; outpaces the African one for most sample sizes during the population growth following event &lt;em&gt;c&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/1027/F4.medium.gif" width="440" height="140"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/3/1027/F4.medium.gif" width="440" height="140"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/1027/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/1027/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483568" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-26" class="first-child">Temporal trajectories of <strong><span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F4/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F4/embed/mml-math-61.gif"/></noscript></span></span></strong> We plot <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F4/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F4/embed/mml-math-62.gif"/></noscript></span></span> trajectories over time in order to show how patterns in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039651/-/DC1/FigureS5.eps">Figure S5</a> are created for (A) weak (<span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F4/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F4/embed/mml-math-63.gif"/></noscript></span></span>), (B) intermediate (<span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F4/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F4/embed/mml-math-64.gif"/></noscript></span></span>), and (C) strong (<span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F4/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F4/embed/mml-math-65.gif"/></noscript></span></span>) selection. Samples of size two (heterozygosity proportions) result in increased <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F4/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F4/embed/mml-math-66.gif"/></noscript></span></span> following the OOA bottleneck, while larger samples show a decrease first and may or may not begin to rise before the bottleneck ends. The OOA model <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F4/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F4/embed/mml-math-67.gif"/></noscript></span></span> outpaces the African one for most sample sizes during the population growth following event <em>c</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">For OOA populations, <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-68.gif"/></noscript></span></span> increases during the growth period following the second bottleneck, but whether this increase is sufficient to surpass <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-69.gif"/></noscript></span></span> for an African sample depends on <em>s</em> and the sample size. In both phases, <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-70.gif"/></noscript></span></span> often decreases when the population size has decreased, and increases when the population size has increased. These are both transient, nonequilibrium behaviors. We observe that the OOA <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-71.gif"/></noscript></span></span> becomes greater than the African <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-72.gif"/></noscript></span></span> during the growth period after the OOA bottleneck. This behavior was originally noted by <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Lohmueller <em>et al.</em> (2008)</a> and was advanced by <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Do <em>et al.</em> (2015)</a> as the main cause of the higher <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-73.gif"/></noscript></span></span> in OOA populations. This observation argues against the interpretation that a greater <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-74.gif"/></noscript></span></span> reflects deleterious variants drifting to higher frequencies during the OOA bottleneck.</p><p id="p-28">The magnitude of <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-75.gif"/></noscript></span></span> and the expected difference between an African and OOA sample vary dramatically with sample size (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-76.gif"/></noscript></span></span> increases with sample size as more low-frequency deleterious variants are discovered. Recent exponential growth in both population trajectories produces a large number of rare variants, and as the sample size becomes large these eventually overwhelm the signal from common ones because the majority of variants at the population level are rare. Since rare variants are only slightly affected by selection, <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-77.gif"/></noscript></span></span> in a very large sample will eventually resemble the ratio of the input mutation rate between synonymous and nonsynonymous changes. In concordance with this prediction, both a large number of rare variants and a smaller difference in <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-78.gif"/></noscript></span></span> between African and OOA samples have been observed in sequencing studies with large sample sizes (<a id="xref-ref-52-8" class="xref-bibr" href="#ref-52">Tennessen <em>et al.</em> 2012</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Nelson <em>et al.</em> 2012</a>). As noted above, we do not predict a greater <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-79.gif"/></noscript></span></span> in the OOA <em>vs.</em> African model for all combinations of sample size and <em>s</em>, and <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> shows combinations of <em>s</em> and sample size where we expect the opposite (where dots exceed crosses, <em>e.g.</em>, sample size 40, <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-80.gif"/></noscript></span></span>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/1027/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Expected current differences in  at different sample sizes. The expected  decreases as s gets larger, but how this happens is dependent on the number of sampled chromosomes (shown on the right). Notice the range of s and sample size for which the expected  is actually greater in the African population model than in the OOA one (dots are slightly higher than crosses). The ratio of deleterious to neutral mutations is assumed to be two to one." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-37178502" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Expected current differences in &lt;strong&gt;&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-81&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&lt;/strong&gt; at different sample sizes. The expected &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-82&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; decreases as &lt;em&gt;s&lt;/em&gt; gets larger, but how this happens is dependent on the number of sampled chromosomes (shown on the right). Notice the range of &lt;em&gt;s&lt;/em&gt; and sample size for which the expected &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-83&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; is actually greater in the African population model than in the OOA one (dots are slightly higher than crosses). The ratio of deleterious to neutral mutations is assumed to be two to one.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/1027/F5.medium.gif" width="440" height="397"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/3/1027/F5.medium.gif" width="440" height="397"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/1027/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/1027/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483619" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-29" class="first-child">Expected current differences in <strong><span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F5/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F5/embed/mml-math-81.gif"/></noscript></span></span></strong> at different sample sizes. The expected <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F5/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F5/embed/mml-math-82.gif"/></noscript></span></span> decreases as <em>s</em> gets larger, but how this happens is dependent on the number of sampled chromosomes (shown on the right). Notice the range of <em>s</em> and sample size for which the expected <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F5/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F5/embed/mml-math-83.gif"/></noscript></span></span> is actually greater in the African population model than in the OOA one (dots are slightly higher than crosses). The ratio of deleterious to neutral mutations is assumed to be two to one.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">It has been previously appreciated that <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-84.gif"/></noscript></span></span> differences do not correspond to changes in the mean deleterious allele frequency or differences in genetic load between populations (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lohmueller 2014b</a>; <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Do <em>et al.</em> 2015</a>). <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-85.gif"/></noscript></span></span> instead reflects the site frequency spectra of putatively deleterious alleles in a complex manner. Specifically, what <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-86.gif"/></noscript></span></span> differences reveal about the evolution of deleterious variation depends on the sample size and strength of selection [a sensitivity also recently noted by <a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Simons and Sella (2016)</a>]. In our example, a greater <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-87.gif"/></noscript></span></span> may primarily reflect deleterious variants drifting to higher frequency during a period of small population size, or it may reflect a proportionally faster recovery of deleterious variation during the growth phase.</p></div><div id="sec-11" class="subsection"><h3>The contribution of selection to population differences</h3><p id="p-31">The extent to which differences in site frequency spectra between populations can be attributed to selection is an open problem (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Brandvain and Wright 2016</a>). We investigate two examples of this problem as instances where we are interested in the distribution of deleterious variation but are not focused on the genetic load or mean fitness.</p><div id="sec-12" class="subsection"><h4>Simple summaries:</h4><p id="p-32"><a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Do <em>et al.</em> (2015)</a> argue that differences between synonymous and nonsynonymous frequency spectra in African and OOA populations can be largely explained without needing to invoke selection following their split. We investigate whether this is true for simple summaries of the SFS. To do so we revisit how heterozygosity, derived allele homozygosity, and the derived allele burden evolve following the OOA split (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/1027/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Stratification of expected differences by selection coefficient. We show, for a range of selection coefficients, the expected nonsynonymous difference per megabase pair between the OOA and African model in (A) heterozygous genotypes, (B) homozygous genotypes, and (C) derived alleles. We obtain a number in terms of nonsynonymous differences by setting the mutation rate to the approximate amount of human coding sequence times a mutation rate of  then multiply by two thirds to approximate the number of new mutations that induce nonsynonymous changes. The vertical axis gives the expected difference per megabase pair per diploid genome. For derived allele count and derived allele homozygosity this includes fixations since the start of the population histories shown in Figure S1. No selection + mutation refers to numerical solutions setting  following the OOA bottleneck in the European trajectory. No selection + no mutation refers to the same, but turning off new mutations as well. The difference in derived allele count is small, meaning the heterozygosity and homozygosity differences must be nearly the same, though with opposite signs, as can be seen." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-37178502" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Stratification of expected differences by selection coefficient. We show, for a range of selection coefficients, the expected nonsynonymous difference per megabase pair between the OOA and African model in (A) heterozygous genotypes, (B) homozygous genotypes, and (C) derived alleles. We obtain a number in terms of nonsynonymous differences by setting the mutation rate to the approximate amount of human coding sequence times a mutation rate of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-88&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; then multiply by two thirds to approximate the number of new mutations that induce nonsynonymous changes. The vertical axis gives the expected difference per megabase pair per diploid genome. For derived allele count and derived allele homozygosity this includes fixations since the start of the population histories shown in Figure S1. &lt;em&gt;No selection + mutation&lt;/em&gt; refers to numerical solutions setting &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-89&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; following the OOA bottleneck in the European trajectory. &lt;em&gt;No selection + no mutation&lt;/em&gt; refers to the same, but turning off new mutations as well. The difference in derived allele count is small, meaning the heterozygosity and homozygosity differences must be nearly the same, though with opposite signs, as can be seen.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/1027/F6.medium.gif" width="440" height="138"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/3/1027/F6.medium.gif" width="440" height="138"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/1027/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/1027/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483694" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-33" class="first-child">Stratification of expected differences by selection coefficient. We show, for a range of selection coefficients, the expected nonsynonymous difference per megabase pair between the OOA and African model in (A) heterozygous genotypes, (B) homozygous genotypes, and (C) derived alleles. We obtain a number in terms of nonsynonymous differences by setting the mutation rate to the approximate amount of human coding sequence times a mutation rate of <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F6/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F6/embed/mml-math-88.gif"/></noscript></span></span> then multiply by two thirds to approximate the number of new mutations that induce nonsynonymous changes. The vertical axis gives the expected difference per megabase pair per diploid genome. For derived allele count and derived allele homozygosity this includes fixations since the start of the population histories shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039651/-/DC1/FigureS1.eps">Figure S1</a>. <em>No selection + mutation</em> refers to numerical solutions setting <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F6/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F6/embed/mml-math-89.gif"/></noscript></span></span> following the OOA bottleneck in the European trajectory. <em>No selection + no mutation</em> refers to the same, but turning off new mutations as well. The difference in derived allele count is small, meaning the heterozygosity and homozygosity differences must be nearly the same, though with opposite signs, as can be seen.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">Separating the effects of mutation, selection, and drift is not straightforward. The diffusion process specified by Equation 1 describes the instantaneous change in the frequency spectrum due to selection and drift. However, over any appreciable length of time the effects of these evolutionary forces are not separable because each distorts the shape of the frequency spectrum and affects the operation of the other. Despite this, there are two ways one can investigate the importance of drift <em>vs.</em> selection. One is to calculate the selection and drift terms of Equation 1 each generation and compare these between populations. Another is to turn selection off in one or both populations and see if this affects outcomes. This is closer to the separation of drift and selection which <a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">Do <em>et al.</em> (2015)</a> refer to. If similar patterns of heterozygosity, derived allele homozygosity, and derived allele burden are seen in the absence of selection, then we might conclude that any observed differences are primarily a product of drift and mutation.</p><p id="p-35">We calculate numerical solutions turning off selection or both selection and mutation following the OOA split. This means that the initial frequency spectrum is set to the equilibrium distribution under selection and evolves under selection up until event <em>b</em> in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>. Without selection there is still a greater expected heterozygosity in the African model relative to the OOA model. When new mutations are included, this provides a good approximation to the differences in heterozygosity and homozygosity below about <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-90.gif"/></noscript></span></span> (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). Above this, the heterozygosity difference is about 5% greater than that in the model with selection, and the magnitude of this deviation increases rapidly with <em>s</em>. The derived allele burden difference in models with no selection is zero because selection is necessary for differences to accumulate. For the heterozygosity difference at nonsynonymous sites, results suggest we can ignore selection for alleles with <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-91.gif"/></noscript></span></span> where <em>N</em> is the size of the bottlenecked population. Overall, these results show it is difficult to conclusively say whether simple differences between the OOA and African models are due to selection because it depends on what level of selection one is interested in and what magnitude of difference one considers important.</p></div><div id="sec-13" class="subsection"><h4>P<sub>N</sub>/P<sub>T</sub>:</h4><p id="p-36">The question of whether differences between the OOA and African model are due to selection can also be asked of <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-92.gif"/></noscript></span></span> <a id="xref-ref-12-7" class="xref-bibr" href="#ref-12">Do <em>et al.</em> (2015)</a> claimed using simulations that the higher <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-93.gif"/></noscript></span></span> in European <em>vs.</em> in West African samples reflects neutral processes. To make this claim they use the first approach mentioned above: they calculate the changes due to selection and neutral forces separately each generation and compare them between populations. We investigate these rates in an equilibrium population to see if they agree with the intuition that selection is more effective in larger populations.</p><p id="p-37">In an equilibrium population, <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-94.gif"/></noscript></span></span> decreases both with increasing <em>s</em> and increasing population size, and this result does not depend on sample size (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). This is because a greater value of <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-95.gif"/></noscript></span></span> corresponds to a greater ability to remove deleterious alleles relative to their accumulation through mutation and drift. Lower <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-96.gif"/></noscript></span></span> at greater <em>S</em> can therefore be taken to reflect a greater efficacy of selection in removing deleterious alleles.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/1027/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The equilibrium behavior of  The equilibrium behavior of  is contrasted between two population sizes ( ) roughly corresponding to the effective population size pre- and post-OOA bottleneck. The sample size is denoted by k. (A) shows how  decreases with s and is greater at a lower equilibrium effective size. (B and C) analyze the difference in the per generation rate at which selection acts to decrease  B does so for samples from the population, while C shows the relationship for the population as a whole. The vertical axis gives the per generation rate of  change in the small population minus that in the larger one. Positive values indicated that the per generation rate at which selection acts to reduce  is greater in the smaller population." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-37178502" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;The equilibrium behavior of &lt;strong&gt;&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-97&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&lt;/strong&gt; The equilibrium behavior of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-98&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; is contrasted between two population sizes (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-99&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-100&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) roughly corresponding to the effective population size pre- and post-OOA bottleneck. The sample size is denoted by &lt;em&gt;k&lt;/em&gt;. (A) shows how &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-101&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; decreases with &lt;em&gt;s&lt;/em&gt; and is greater at a lower equilibrium effective size. (B and C) analyze the difference in the per generation rate at which selection acts to decrease &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-102&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; B does so for samples from the population, while C shows the relationship for the population as a whole. The vertical axis gives the per generation rate of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-103&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; change in the small population minus that in the larger one. Positive values indicated that the per generation rate at which selection acts to reduce &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-104&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; is greater in the smaller population.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/1027/F7.medium.gif" width="440" height="143"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/3/1027/F7.medium.gif" width="440" height="143"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/1027/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/1027/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483675" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-38" class="first-child">The equilibrium behavior of <strong><span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F7/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F7/embed/mml-math-97.gif"/></noscript></span></span></strong> The equilibrium behavior of <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F7/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F7/embed/mml-math-98.gif"/></noscript></span></span> is contrasted between two population sizes (<span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F7/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F7/embed/mml-math-99.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F7/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F7/embed/mml-math-100.gif"/></noscript></span></span>) roughly corresponding to the effective population size pre- and post-OOA bottleneck. The sample size is denoted by <em>k</em>. (A) shows how <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F7/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F7/embed/mml-math-101.gif"/></noscript></span></span> decreases with <em>s</em> and is greater at a lower equilibrium effective size. (B and C) analyze the difference in the per generation rate at which selection acts to decrease <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F7/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F7/embed/mml-math-102.gif"/></noscript></span></span> B does so for samples from the population, while C shows the relationship for the population as a whole. The vertical axis gives the per generation rate of <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F7/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F7/embed/mml-math-103.gif"/></noscript></span></span> change in the small population minus that in the larger one. Positive values indicated that the per generation rate at which selection acts to reduce <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F7/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F7/embed/mml-math-104.gif"/></noscript></span></span> is greater in the smaller population.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">At equilibrium, mutation, selection, and drift cancel out and cause the expected value of <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-105.gif"/></noscript></span></span> to remain constant. We analyze this process by calculating the instantaneous rate due to selection. The equilibrium rate of change in <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-106.gif"/></noscript></span></span> per <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-107.gif"/></noscript></span></span> generations due to selection is<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-108.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>or<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-109.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>Here, <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-110.gif"/></noscript></span></span> is population-scaled mutation rate to deleterious alleles, <em>k</em> is the sample size, and the subscript <em>γ</em> denotes that this is the portion of the rate of change that is due to selection (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039651/-/DC1/FileS1.pdf">File S1</a> for details). We calculate these rates for different sample sizes in a population resembling that before the OOA bottleneck (<span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-111.gif"/></noscript></span></span>) and in one resembling the bottlenecked size (<span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-112.gif"/></noscript></span></span>) (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). Surprisingly, even though <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-113.gif"/></noscript></span></span> is greater in smaller populations, the <em>per generation</em> rate at which selection decreases this value can actually be greater in smaller populations (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7B</a>). This is true for a large range of <em>s</em> at small samples sizes and persists at strongly constrained sites even when we consider the full population (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7C</a>).</p><p id="p-40">In their simulations, <a id="xref-ref-12-8" class="xref-bibr" href="#ref-12">Do <em>et al.</em> (2015)</a> observe that the rate at which selection decreases <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-114.gif"/></noscript></span></span> on a per generation basis is stronger in an OOA population trajectory than an African one. They use this to conclude that primarily nonselective forces have driven the dynamics of this statistic. Given our equilibrium results, we note that a greater per generation change in <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-115.gif"/></noscript></span></span> due to selection does not necessarily imply evidence for a greater efficacy of selection or the primacy of drift <em>vs.</em> selection. The sign of this rate difference between two populations depends on the strength of selection and the sample size.</p></div></div><div id="sec-14" class="subsection"><h3>Empirical relationship between strength of selection and the distribution of deleterious variation</h3><p id="p-41">The strength of selection, represented by <em>s</em>, greatly impacts how both heterozygosity and <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-116.gif"/></noscript></span></span> respond to the demographic events that lead to differences between populations (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>, and <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). We investigate how well GERP scores, a putative measure of the strength of selection based on phylogenetic conservation (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Cooper <em>et al.</em> 2005</a>; <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Davydov <em>et al.</em> 2010</a>), predict heterozygosity. In order to do this, we compare the expected differences, stratified by <em>s</em>, in heterozygosity, derived allele homozygosity, and derived allele burden between the African and OOA trajectories with empirical results from the ExAC data for the AFR and NFE ancestries. We switch to using more simple summaries than <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-117.gif"/></noscript></span></span> because these are not sensitive to sample size.</p><p id="p-42">We first examine how the expected differences between Africa and OOA in heterozygosity, derived allele homozygosity, and the derived allele burden evolve over time and depend on <em>s</em>. Doing so shows that the present increased homozygosity and decreased heterozygosity in OOA <em>vs.</em> Africa originates during the OOA bottleneck. This effect persists at present, but the recovery of the OOA population size beginning around 1000 generations ago decreases the magnitude of this difference and relatively more so for more strongly selected alleles (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8, B and C</a>). For derived alleles, we predict a slight excess for all <em>s</em> in OOA, but this difference decreases during the recovery when <em>s</em> is large (at least <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-118.gif"/></noscript></span></span>). This emphasizes the need to consider demography following the OOA bottleneck when studying selected variation in human populations (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Gravel 2016</a>). The variation between different <em>s</em> values is not as great here as for <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-119.gif"/></noscript></span></span> but it is clear that the differences evolve on a faster time scale for sites under greater selection.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/1027/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Stratification of expected differences by time. We show for a range of  where  is the population size preceding event b, how the expected difference in (A) heterozygous genotypes, (B) homozygous derived genotypes, and (C) derived alleles changes over time, relative to their levels in the ancestral population that existed before event b. Vertical lines indicate demographic events shown in Figure S1. Substantial changes in heterozygosity and homozygosity differences occur following event c, emphasizing the importance of the recovery from the OOA bottleneck." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-37178502" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Stratification of expected differences by time. We show for a range of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-120&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; where &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-121&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; is the population size preceding event &lt;em&gt;b&lt;/em&gt;, how the expected difference in (A) heterozygous genotypes, (B) homozygous derived genotypes, and (C) derived alleles changes over time, relative to their levels in the ancestral population that existed before event &lt;em&gt;b&lt;/em&gt;. Vertical lines indicate demographic events shown in Figure S1. Substantial changes in heterozygosity and homozygosity differences occur following event &lt;em&gt;c&lt;/em&gt;, emphasizing the importance of the recovery from the OOA bottleneck.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/1027/F8.medium.gif" width="440" height="145"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/7/3/1027/F8.medium.gif" width="440" height="145"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/1027/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/1027/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483690" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-43" class="first-child">Stratification of expected differences by time. We show for a range of <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F8/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F8/embed/mml-math-120.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F8/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F8/embed/mml-math-121.gif"/></noscript></span></span> is the population size preceding event <em>b</em>, how the expected difference in (A) heterozygous genotypes, (B) homozygous derived genotypes, and (C) derived alleles changes over time, relative to their levels in the ancestral population that existed before event <em>b</em>. Vertical lines indicate demographic events shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039651/-/DC1/FigureS1.eps">Figure S1</a>. Substantial changes in heterozygosity and homozygosity differences occur following event <em>c</em>, emphasizing the importance of the recovery from the OOA bottleneck.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">The present expected heterozygosity difference between the OOA and African models decreases with <em>s</em>, while the expected derived allele homozygosity difference mirrors this and increases with <em>s</em> (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>). The expected difference in derived allele burden is small and peaks at an intermediate value around <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-122.gif"/></noscript></span></span> This is consistent with results from <a id="xref-ref-50-5" class="xref-bibr" href="#ref-50">Simons <em>et al.</em> (2014)</a> showing only a very small expected increase in genetic load in an OOA model.</p><p id="p-45">In the ExAC, heterozygosity and homozygosity show similar trends (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a>) as the theoretical prediction (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6</a>) with heterozygosity higher in AFR and derived allele homozygosity higher in NFE. Differences between AFR and NFE also decrease with increasing GERP score, similar to how the expected differences decrease with increasing <em>s</em>. However, there is no clear relationship between GERP score and the mean difference in derived allele burden between AFR and NFE individuals. This is perhaps not surprising because the expected burden difference is so small for all <em>s</em>.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/1027/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Observed differences in ExAC by GERP score. The top row (A&#x2013;C) shows heterozygosity, homozygosity, and derived allele frequency (respectively) for the African and NFE population groups in ExAC plotted against binned GERP scores. The bottom row (D&#x2013;F) shows the differences between them (AFR-NFE). Dotted lines provide  confidence intervals obtained by bootstrapping across sites within each bin." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-37178502" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Observed differences in ExAC by GERP score. The top row (A&#x2013;C) shows heterozygosity, homozygosity, and derived allele frequency (respectively) for the African and NFE population groups in ExAC plotted against binned GERP scores. The bottom row (D&#x2013;F) shows the differences between them (AFR-NFE). Dotted lines provide &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-123&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; confidence intervals obtained by bootstrapping across sites within each bin.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/1027/F9.medium.gif" width="440" height="256"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/7/3/1027/F9.medium.gif" width="440" height="256"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/1027/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/1027/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483609" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-46" class="first-child">Observed differences in ExAC by GERP score. The top row (A–C) shows heterozygosity, homozygosity, and derived allele frequency (respectively) for the African and NFE population groups in ExAC plotted against binned GERP scores. The bottom row (D–F) shows the differences between them (AFR-NFE). Dotted lines provide <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F9/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/F9/embed/mml-math-123.gif"/></noscript></span></span> confidence intervals obtained by bootstrapping across sites within each bin.</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">Another approach to look for a relationship between derived alleles and GERP scores is to calculate a GERP score burden, which weighs the frequency of each derived allele by its GERP score (<span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-124.gif"/></noscript></span></span>) (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Marsden <em>et al.</em> 2016</a>). We calculate a cumulative GERP score burden and use bootstrapping across sites to assess confidence. While the NFE sample does appear to have an excess GERP burden for mildly deleterious alleles in the GERP score range of 2–4, this trend is not apparent in most bootstrap replicates, and we do not see a significant overall difference between the AFR and NFE samples (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039651/-/DC1/FigureS3.eps">Figure S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039651/-/DC1/FigureS4.eps">Figure S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039651/-/DC1/FigureS7.eps">Figure S7</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039651/-/DC1/FigureS9.jpg">Figure S9</a>).</p><p id="p-48">The lack of difference between AFR and NFE samples in derived allele frequency or GERP score burden could be attributable to a sensitivity to quality filters (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039651/-/DC1/FigureS6.eps">Figure S6</a>). This might then also explain the lack of a relationship between the derived allele frequency and GERP score. Additionally, there is only a weak trend in relative heterozygosity with GERP score (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039651/-/DC1/FigureS8.eps">Figure S8</a>). This suggests that GERP scores better reflect the probability a variant is strongly selected rather than the selection coefficients of weakly deleterious variants and that the majority of heterozygous sites within each GERP bin are neutral. This makes GERP scores less useful when we are interested in stratifying genetic variants by whether they are strongly or weakly selected.</p></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-49">In this article, we have demonstrated four ways in which equilibrium population genetic logic can mislead when applied to populations with nonequilibrium histories. (1) Heterozygosity can be increasing for some period following a population decline and can be decreasing while the population is growing. (2) In the OOA and African population models, differences in <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-125.gif"/></noscript></span></span> depend strongly on the strength or selection and sample size. In particular, which demographic event has the greatest effect on the <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-126.gif"/></noscript></span></span> difference is greatly influenced by these parameters. (3) The interpretation of differences in <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-127.gif"/></noscript></span></span> heterozygosity, and derived allele homozygosity depends on the strength of selection, but we find that GERP scores are imprecise predictions of selection coefficients at particular sites. (4) We find that it is difficult to decide whether differences in deleterious variation between nonequilibrium populations are due to drift <em>vs.</em> selection.</p><p id="p-50">A number of recent theoretical investigations have supplied useful intuition into the effects of bottlenecks and population growth on deleterious variation (<a id="xref-ref-50-6" class="xref-bibr" href="#ref-50">Simons <em>et al.</em> 2014</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Balick <em>et al.</em> 2015</a>; <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Peischl <em>et al.</em> 2013</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Gazave <em>et al.</em> 2013</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Lohmueller 2014b</a>; <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Gravel 2016</a>). These have been spurred by particular interest in the effects of human demographies (<a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Lohmueller 2014a</a>) and have used forward-in-time models as these allow selection to be easily incorporated.</p><p id="p-51">Similar in spirit to our point (1), <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Balick <em>et al.</em> (2015)</a> have developed analytical approximations to the change in the mean derived allele frequency following a short population bottleneck with the purpose of contrasting additive and recessive modes of selection. They find nonmonotonic behavior when selection is recessive, wherein deleterious variants are purged after recovery from the bottleneck before mutation builds them up again. The nonmonotonic behavior we see in heterozygosity (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>) is less severe than this because it is most pronounced at strongly constrained sites that have low expected heterozygosity to begin with. However, it is interesting because it results from a simpler, additive model and makes the counterintuitive prediction that heterozygosity should sometimes be decreasing in a growing population.</p><p id="p-52">Regarding point (2), <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-128.gif"/></noscript></span></span> differences in nonequilibrium populations do not have the same interpretation as in equilibrium ones. This was noted previously in simulation studies that showed the bottleneck effect of initially decreasing <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-129.gif"/></noscript></span></span> before causing it to increase due to deleterious variants drifting to higher frequencies, and the fact that recovery from a bottleneck can also increase <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-130.gif"/></noscript></span></span> (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Lohmueller <em>et al.</em> 2008</a>; <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Lohmueller 2014b</a>; <a id="xref-ref-12-9" class="xref-bibr" href="#ref-12">Do <em>et al.</em> 2015</a>; <a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">Simons and Sella 2016</a>). Our analyses have looked in greater detail at how demography and selection have interacted to produce these patterns and emphasize the strong dependence of <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-131.gif"/></noscript></span></span> evolution on the sample size and strength of selection. We emphasize that whether <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-132.gif"/></noscript></span></span> is increasing or decreasing at any point in a demographic history is highly dependent on both the sample size and strength of selection (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7</a>).</p><p id="p-53">While human studies often use <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-133.gif"/></noscript></span></span> (<a id="xref-ref-36-5" class="xref-bibr" href="#ref-36">Lohmueller 2014a</a>), investigations in other species use the ratio of nonsynonymous to synonymous heterozygosity instead. Many have found a negative relationship between this ratio and synonymous heterozygosity (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Elyashiv <em>et al.</em> 2010</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cao <em>et al.</em> 2011</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Marsden <em>et al.</em> 2016</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Li <em>et al.</em> 2014</a>). The heterozygosity ratio has the obvious advantage of not being dependent on the sample size, and a sample of size two in <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-134.gif"/></noscript></span></span> appears to chiefly respond to deleterious variants drifting to higher frequencies during the OOA bottleneck as opposed to subsequent growth events (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). In the recovery from a bottleneck, deleterious alleles reach their equilibrium heterozygosity before neutral ones. This may cause <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-135.gif"/></noscript></span></span> to be misleading because its value can increase transiently while the effectiveness of selection to remove deleterious alleles is actually increasing.</p><p id="p-54">Regarding point (3), we did not find a strong indication that GERP scores could reflect intermediate selection coefficients, but other studies have noticed interesting patterns between measures of selection and the distribution of deleterious variation. <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Racimo and Schraiber (2014)</a> fit selection coefficients to sites binned by a different measure of selection, combined annotation dependent depletion (CADD) scores (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kircher <em>et al.</em> 2014</a>). They found that the fitted selection coefficient did decrease monotonically with severity of CADD scores. In another study, <a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Henn <em>et al.</em> (2016)</a> observe a greater number of putatively deleterious derived alleles (classified using GERP scores) in East Asian and American genomes and explain this in terms of a serial founder model that can produce a higher genetic load with more founder events (<a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Peischl <em>et al.</em> 2013</a>). The difference in putatively deleterious alleles came from sites with intermediate GERP scores similar to the theoretical expectation in <a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6</a>. This is in contrast to findings of no significant increase in derived allele burden in any contemporary human population compared to an African one (<a id="xref-ref-12-10" class="xref-bibr" href="#ref-12">Do <em>et al.</em> 2015</a>; <a id="xref-ref-49-5" class="xref-bibr" href="#ref-49">Simons and Sella 2016</a>). Future research should seek to resolve these conflicting observations because they are based on samples from different populations and used different predictors of deleteriousness and different forms of statistical tests. More extensive sampling of American, Asian, and other populations geographically far from Africa will also add clarity.</p><p id="p-55">In point (4), we note that the per generation rate at which selection changes <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-136.gif"/></noscript></span></span> is not an indicator of the efficacy of selection, as it can be greater in a smaller than in a larger population for the same <em>s</em>. Because it has such complex behavior in an equilibrium model, we argue that the per generation rate at which selection changes <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-137.gif"/></noscript></span></span> should not be used to say whether changes in <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3/1027/embed/mml-math-138.gif"/></noscript></span></span> have been caused by neutral or selective forces (<a id="xref-ref-12-11" class="xref-bibr" href="#ref-12">Do <em>et al.</em> 2015</a>).</p><p id="p-56">The model we have analyzed here is simple in many regards. For instance, the <a id="xref-ref-52-9" class="xref-bibr" href="#ref-52">Tennessen <em>et al.</em> (2012)</a> demographic model includes some migration between the African and European populations after the OOA split, and we did not include this. Other studies analyzing the response of deleterious variation to this particular demographic scenario come to similar conclusions when including this migration (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Fu <em>et al.</em> 2014</a>; <a id="xref-ref-50-7" class="xref-bibr" href="#ref-50">Simons <em>et al.</em> 2014</a>; <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Gravel 2016</a>). A larger second issue is that we have only considered alleles acting additively within genotypes. It is well known that a large amount of strongly deleterious variation acts recessively, and that there is likely to be a negative relationship between the degree of dominance and how deleterious a mutation is (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Agrawal and Whitlock 2011</a>). However, the dominance effects of nearly neutral mutations are still mostly unknown because these are much less amenable to analysis. Studies that have inferred a distribution of fitness (DFE) effects of new mutations from the SFS have almost exclusively used additive models (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Eyre-Walker <em>et al.</em> 2006</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Boyko <em>et al.</em> 2008</a>). <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Balick <em>et al.</em> (2015)</a> find evidence of recessive selection in humans at some sets of genes known to act recessively in causing disease. Specifically, they found a higher derived allele burden in recessive disease genes in a European sample compared to an African-American one. That we do not observe a difference in the derived allele burden overall may thus be partially explained by this effect canceling out the effects of additive mutations. <a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Henn <em>et al.</em> (2016)</a> fit a model of serial founder effects under the assumption of complete recessivity and found that such a model is consistent with the observed heterozygosity cline away from Africa in different GERP score bins.</p><p id="p-57">An additional simplification in the model we used is that sites are modeled to be independent. Linked variants under selection will interfere with each other, reducing the effectiveness of selection and levels of polymorphism (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Comeron and Kreitman 2002</a>). Although there is substantial evidence for background selection influencing patterns of variation in humans (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">McVicker <em>et al.</em> 2009</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hernandez <em>et al.</em> 2011</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lohmueller <em>et al.</em> 2011</a>), if interference between deleterious alleles is rare it should not substantially affect our results. If there is substantial interference between deleterious alleles, it is not clear how such interference would affect the response of deleterious variation to demographic events like bottlenecks and growth periods, as even equilibrium models of interference selection can be quite complex (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Good <em>et al.</em> 2014</a>). In species with larger population sizes than humans, it is likely that linked selection and interference cannot be ignored so easily.</p><p id="p-58">Studying fitness differences and the predicted accumulation of deleterious mutations in smaller populations remains a challenge. The derived allele burden, at least for semidominant alleles, is likely to be a robust statistic for identifying populations accumulating deleterious variation, and it has been used to identify a number of cases where this seems to have occurred (<a id="xref-ref-27-6" class="xref-bibr" href="#ref-27">Henn <em>et al.</em> 2016</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Xue <em>et al.</em> 2015</a>; <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Marsden <em>et al.</em> 2016</a>; <a id="xref-ref-12-12" class="xref-bibr" href="#ref-12">Do <em>et al.</em> 2015</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Schubert <em>et al.</em> 2014</a>). The precise interpretation of these results is much more difficult because converting them to genetic load or fitness differences requires knowing something about the underlying fitness effects of mutations, and differences in dominance can yield opposite results (<a id="xref-ref-27-7" class="xref-bibr" href="#ref-27">Henn <em>et al.</em> 2016</a>; <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Balick <em>et al.</em> 2015</a>). Differences in the adaptive substitution rate might also confound inference from the derived allele burden because new adaptive alleles will also be counted as derived (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Brandvain and Wright 2016</a>). A more rigorous approach would be to jointly infer the DFE, demography, dominance, and mutational load. Future work should explore the possibility of doing so, and in the meantime any inference based on summary statistics is best supported by extensive simulations (<em>e.g.</em>, <a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Marsden <em>et al.</em> 2016</a>).</p><p id="p-59">One final factor not considered here was the effects of deleterious alleles introgressing into human populations. Sequencing of ancient DNA has strongly suggested that two archaic humans, Neanderthals and Denisovans, accumulated significantly more deleterious mutation than contemporary humans prior to their extinction (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Castellano <em>et al.</em> 2014</a>; <a id="xref-ref-12-13" class="xref-bibr" href="#ref-12">Do <em>et al.</em> 2015</a>). <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Harris and Nielsen (2015)</a> estimate that the average Neanderthal would have had at least 40% lower fitness than the average human at the time of admixture. This admixture would then have introduced a large number of deleterious alleles into human populations, resulting in a contemporary load of deleterious alleles that arose in Neanderthals (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Harris and Nielsen 2015</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Juric <em>et al.</em> 2015</a>). It will be interesting to see whether this contributes to the excess derived allele burden in East Asian and American populations (<a id="xref-ref-27-8" class="xref-bibr" href="#ref-27">Henn <em>et al.</em> 2016</a>), given that East Asian populations contain a greater fraction of Neanderthal DNA (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Wall <em>et al.</em> 2013</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Sankararaman <em>et al.</em> 2014</a>) likely due to a greater gene flow from Neanderthal populations (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kim and Lohmueller 2015</a>).</p></div><div class="section" id="sec-16"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-60">We acknowledge Kirk Lohmueller and members of the Novembre laboratory, particularly Ben Peter, Natalia Gonzales, and Mark Reppell, for helpful discussions and comments. Funding for this work was provided by the National Institutes of Health (GM108805 to J.N., training grant T32 GM007197 to E.K.) and a National Science Foundation Graduate Research Fellowship (E.K.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-61">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039651/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039651/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-62"><em>Communicating editor: J. D. Wall</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-11-01"><span class="received-label">Received </span>November 1, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-01-24"><span class="accepted-label">Accepted </span>January 24, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Koch and Novembre</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.1" data-doi="10.1534/genetics.110.124560"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Agrawal</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitlock</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Inferences about the distribution of dominance drawn from yeast gene knockout data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">553</span>–<span class="cit-lpage">566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.124560%26rft_id%253Dinfo%253Apmid%252F21098719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg3LzIvNTUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy8zLzEwMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Balick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Do</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cassa</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sunyaev</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Dominance of deleterious alleles controls the response to a population bottleneck.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.3" data-doi="10.1371/journal.pgen.1000083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boyko</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Indap</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Degenhardt</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Assessing the evolutionary impact of amino acid mutations in the human genome.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000083</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBoyko%26rft.auinit1%253DA.%2BR.%26rft.volume%253D4%26rft.issue%253D5%26rft.spage%253De1000083%26rft.epage%253De1000083%26rft.atitle%253DAssessing%2Bthe%2Bevolutionary%2Bimpact%2Bof%2Bamino%2Bacid%2Bmutations%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000083%26rft_id%253Dinfo%253Apmid%252F18516229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18516229&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.4" data-doi="10.1016/j.tig.2016.01.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brandvain</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The limits of natural selection in a nonequilibrium world.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">201</span>–<span class="cit-lpage">210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D32%26rft.spage%253D201%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2016.01.004%26rft_id%253Dinfo%253Apmid%252F26874998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2016.01.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26874998&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.5" data-doi="10.1038/ng.911"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ossowski</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Günther</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bender</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Whole-genome sequencing of multiple <em>Arabidopsis thaliana</em> populations.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">956</span>–<span class="cit-lpage">963</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DCao%26rft.auinit1%253DJ.%26rft.volume%253D43%26rft.issue%253D10%26rft.spage%253D956%26rft.epage%253D963%26rft.atitle%253DWhole-genome%2Bsequencing%2Bof%2Bmultiple%2BArabidopsis%2Bthaliana%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.911%26rft_id%253Dinfo%253Apmid%252F21874002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.911&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21874002&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.6" data-doi="10.1073/pnas.1405138111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Castellano</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parra</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez-Quinto</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Racimo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhlwilm</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Patterns of coding variation in the complete exomes of three Neandertals.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">6666</span>–<span class="cit-lpage">6671</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1405138111%26rft_id%253Dinfo%253Apmid%252F24753607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTExLzE4LzY2NjYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzMvMTAyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-book" id="cit-7.3.1027.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-source"><em>Elements of Evolutionary Genetics</em></span>. <span class="cit-publ-name">Roberts and Company Publishers</span>, <span class="cit-publ-loc">Greenwood Village, CO</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comeron</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreitman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Population, evolutionary and genomic consequences of interference selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DComeron%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D161%26rft.issue%253D1%26rft.spage%253D389%26rft.epage%253D410%26rft.atitle%253DPopulation%252C%2BEvolutionary%2Band%2BGenomic%2BConsequences%2Bof%2BInterference%2BSelection%26rft_id%253Dinfo%253Apmid%252F12019253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYxLzEvMzg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy8zLzEwMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.9" data-doi="10.1101/gr.3577405"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asimenos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batzoglou</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Distribution and intensity of constraint in mammalian genomic sequence.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">901</span>–<span class="cit-lpage">913</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3577405%26rft_id%253Dinfo%253Apmid%252F15965027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE1LzcvOTAxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy8zLzEwMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.10" data-doi="10.1371/journal.pcbi.1001025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davydov</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goode</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sirota</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sidow</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Identifying a high fraction of the human genome to be under selective constraint using GERP++.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001025</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DDavydov%26rft.auinit1%253DE.%2BV.%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253De1001025%26rft.epage%253De1001025%26rft.atitle%253DIdentifying%2Ba%2Bhigh%2Bfraction%2Bof%2Bthe%2Bhuman%2Bgenome%2Bto%2Bbe%2Bunder%2Bselective%2Bconstraint%2Busing%2BGERP%252B%252B.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1001025%26rft_id%253Dinfo%253Apmid%252F21152010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1001025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21152010&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.3.1027.11" data-doi="10.1038/ng.806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garimella</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478889&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.12" data-doi="10.1038/ng.3186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Do</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balick</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adzhubei</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sunyaev</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">No evidence that selection has been less effective at removing deleterious mutations in Europeans than in Africans.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">126</span>–<span class="cit-lpage">131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D126%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3186%26rft_id%253Dinfo%253Apmid%252F25581429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25581429&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.13" data-doi="10.1038/nature09534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altshuler</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bentley</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chakravarti</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A map of human genome variation from population-scale sequencing.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">1061</span>–<span class="cit-lpage">1073</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDurbin%26rft.auinit1%253DR.%2BM.%26rft.volume%253D467%26rft.issue%253D7319%26rft.spage%253D1061%26rft.epage%253D1073%26rft.atitle%253DA%2Bmap%2Bof%2Bhuman%2Bgenome%2Bvariation%2Bfrom%2Bpopulation-scale%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09534%26rft_id%253Dinfo%253Apmid%252F20981092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20981092&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283548600039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.14" data-doi="10.1101/gr.108993.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elyashiv</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bullaughey</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sattath</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rinott</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przeworski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Shifts in the intensity of purifying selection: an analysis of genome-wide polymorphism data from two closely related yeast species.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1558</span>–<span class="cit-lpage">1573</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.108993.110%26rft_id%253Dinfo%253Apmid%252F20817943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8xMS8xNTU4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy8zLzEwMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.15" data-doi="10.1016/j.tpb.2006.06.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shvets</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Non-equilibrium theory of the allele frequency spectrum.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">109</span>–<span class="cit-lpage">119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DEvans%26rft.auinit1%253DS.%2BN.%26rft.volume%253D71%26rft.issue%253D1%26rft.spage%253D109%26rft.epage%253D119%26rft.atitle%253DNon-equilibrium%2Btheory%2Bof%2Bthe%2Ballele%2Bfrequency%2Bspectrum.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tpb.2006.06.005%26rft_id%253Dinfo%253Apmid%252F16887160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tpb.2006.06.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16887160&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244021700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ewens</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <abbr class="cit-jnl-abbrev">Mathematical Population Genetics 1: I Theoretical Introduction.</abbr> <span class="cit-publ-name">Springer Science &amp; Business Media</span>, <span class="cit-publ-loc">New York, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.17" data-doi="10.1073/pnas.0906182107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic architecture of a complex trait and its implications for fitness and genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">1752</span>–<span class="cit-lpage">1756</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0906182107%26rft_id%253Dinfo%253Apmid%252F20133822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxNjoiMTA3L3N1cHBsXzEvMTc1MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvMy8xMDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.18" data-doi="10.1534/genetics.106.057570"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolfit</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phelps</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The distribution of fitness effects of new deleterious amino acid mutations in humans.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">891</span>–<span class="cit-lpage">900</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.057570%26rft_id%253Dinfo%253Apmid%252F16547091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTczLzIvODkxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy8zLzEwMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.19" data-doi="10.1016/j.ajhg.2014.09.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gittelman</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bamshad</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akey</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Characteristics of neutral and deleterious protein-coding variation among individuals and populations.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">421</span>–<span class="cit-lpage">436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D95%26rft.spage%253D421%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2014.09.006%26rft_id%253Dinfo%253Apmid%252F25279984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2014.09.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25279984&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.20" data-doi="10.1534/genetics.113.153973"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gazave</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keinan</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Population growth inflates the per-individual number of deleterious mutations and reduces their mean effect.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">969</span>–<span class="cit-lpage">978</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.153973%26rft_id%253Dinfo%253Apmid%252F23979573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk1LzMvOTY5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy8zLzEwMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.21" data-doi="10.1371/journal.pgen.1004222"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Good</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walczak</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neher</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desai</span>  <span class="cit-name-given-names">M. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic diversity in the interference selection limit.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004222%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004222%26rft_id%253Dinfo%253Apmid%252F24675740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004222&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24675740&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.22" data-doi="10.1534/genetics.115.184630"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gravel</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">When is selection effective?</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">203</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">462</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.184630%26rft_id%253Dinfo%253Apmid%252F27010021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAzLzEvNDUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy8zLzEwMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.23" data-doi="10.1371/journal.pgen.1000695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gutenkunst</span>  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Inferring the joint demographic history of multiple populations from multidimensional SNP frequency data.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000695</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGutenkunst%26rft.auinit1%253DR.%2BN.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253De1000695%26rft.epage%253De1000695%26rft.atitle%253DInferring%2Bthe%2Bjoint%2Bdemographic%2Bhistory%2Bof%2Bmultiple%2Bpopulations%2Bfrom%2Bmultidimensional%2BSNP%2Bfrequency%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000695%26rft_id%253Dinfo%253Apmid%252F19851460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19851460&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The genetic cost of Neanderthal introgression.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-fpage">030387</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moriyama</span>  <span class="cit-name-given-names">E. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sawyer</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Selection intensity for codon bias.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">227</span>–<span class="cit-lpage">234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHartl%26rft.auinit1%253DD.%2BL.%26rft.volume%253D138%26rft.issue%253D1%26rft.spage%253D227%26rft.epage%253D234%26rft.atitle%253DSelection%2Bintensity%2Bfor%2Bcodon%2Bbias.%26rft_id%253Dinfo%253Apmid%252F8001789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM4LzEvMjI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy8zLzEwMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.26" data-doi="10.1073/pnas.1212380109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henn</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cavalli-Sforza</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The great human expansion.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">17758</span>–<span class="cit-lpage">17764</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1212380109%26rft_id%253Dinfo%253Apmid%252F23077256%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzQ0LzE3NzU4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy8zLzEwMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.27" data-doi="10.1073/pnas.1510805112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henn</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botigué</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peischl</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dupanloup</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipatov</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Distance from sub-Saharan Africa predicts mutational load in diverse human genomes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">E440</span>–<span class="cit-lpage">E449</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1510805112%26rft_id%253Dinfo%253Apmid%252F26712023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTEzLzQvRTQ0MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvMy8xMDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.28" data-doi="10.1093/molbev/msm108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Context dependence, ancestral misidentification, and spurious signatures of natural selection.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1792</span>–<span class="cit-lpage">1800</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm108%26rft_id%253Dinfo%253Apmid%252F17545186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI0LzgvMTc5MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvMy8xMDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.29" data-doi="10.1126/science.1198878"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elyashiv</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melton</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Classic selective sweeps were rare in recent human evolution.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">331</span>: <span class="cit-fpage">920</span>–<span class="cit-lpage">924</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHernandez%26rft.auinit1%253DR.%2BD.%26rft.volume%253D331%26rft.issue%253D6019%26rft.spage%253D920%26rft.epage%253D924%26rft.atitle%253DClassic%2BSelective%2BSweeps%2BWere%2BRare%2Bin%2BRecent%2BHuman%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1198878%26rft_id%253Dinfo%253Apmid%252F21330547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzEvNjAxOS85MjAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzMvMTAyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Juric</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aeschbacher</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The strength of selection against Neanderthal introgression.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-fpage">030148</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.31" data-doi="10.1016/j.ajhg.2014.12.029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">B. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohmueller</span>  <span class="cit-name-given-names">K. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Selection and reduced population size cannot explain higher amounts of Neandertal ancestry in East Asian than in European human populations.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">454</span>–<span class="cit-lpage">461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D96%26rft.spage%253D454%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2014.12.029%26rft_id%253Dinfo%253Apmid%252F25683122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2014.12.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25683122&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.32" data-doi="10.2307/3211856"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">Diffusion models in population genetics.</span> <abbr class="cit-jnl-abbrev">J. Appl. Probab.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">177</span>–<span class="cit-lpage">232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAppl.%2BProbab.%26rft.volume%253D1%26rft.spage%253D177%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3211856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3211856&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.33" data-doi="10.1038/ng.2892"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kircher</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witten</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Roak</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A general framework for estimating the relative pathogenicity of human genetic variants.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">310</span>–<span class="cit-lpage">315</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D310%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2892%26rft_id%253Dinfo%253Apmid%252F24487276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2892&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24487276&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.34" data-doi="10.1038/nature19057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karczewski</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samocha</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Analysis of protein-coding genetic variation in 60,706 humans.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">536</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D536%26rft.spage%253D285%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature19057%26rft_id%253Dinfo%253Apmid%252F27535533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature19057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27535533&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiong</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic signatures of near-extinction and rebirth of the crested ibis and other endangered bird species.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253D557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.36" data-doi="10.1016/j.gde.2014.09.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lohmueller</span>  <span class="cit-name-given-names">K. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">a</span> <span class="cit-article-title">The distribution of deleterious genetic variation in human populations.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BGenet.%2BDev.%26rft.volume%253D29%26rft.spage%253D139%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2014.09.005%26rft_id%253Dinfo%253Apmid%252F25461617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2014.09.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25461617&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.37" data-doi="10.1371/journal.pgen.1004379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lohmueller</span>  <span class="cit-name-given-names">K. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">b</span> <span class="cit-article-title">The impact of population demography and selection on the genetic architecture of complex traits.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004379%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004379%26rft_id%253Dinfo%253Apmid%252F24875776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24875776&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.38" data-doi="10.1038/nature06611"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lohmueller</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Indap</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyko</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Proportionally more deleterious genetic variation in European than in African populations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">451</span>: <span class="cit-fpage">994</span>–<span class="cit-lpage">997</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLohmueller%26rft.auinit1%253DK.%2BE.%26rft.volume%253D451%26rft.issue%253D7181%26rft.spage%253D994%26rft.epage%253D997%26rft.atitle%253DProportionally%2Bmore%2Bdeleterious%2Bgenetic%2Bvariation%2Bin%2BEuropean%2Bthan%2Bin%2BAfrican%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06611%26rft_id%253Dinfo%253Apmid%252F18288194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06611&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18288194&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253313100049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.39" data-doi="10.1371/journal.pgen.1002326"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lohmueller</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korneliussen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Natural selection affects multiple aspects of genetic variation at putatively neutral sites across the human genome.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002326</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLohmueller%26rft.auinit1%253DK.%2BE.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De1002326%26rft.epage%253De1002326%26rft.atitle%253DNatural%2Bselection%2Baffects%2Bmultiple%2Baspects%2Bof%2Bgenetic%2Bvariation%2Bat%2Bputatively%2Bneutral%2Bsites%2Bacross%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002326%26rft_id%253Dinfo%253Apmid%252F22022285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002326&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22022285&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.40" data-doi="10.1073/pnas.1512501113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marsden</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ortega-Del Vecchyo</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Brien</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramirez</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Bottlenecks and selective sweeps during domestication have increased deleterious genetic variation in dogs.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">152</span>–<span class="cit-lpage">157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1512501113%26rft_id%253Dinfo%253Apmid%252F26699508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMTMvMS8xNTIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzMvMTAyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.3.1027.41" data-doi="10.1038/351652a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreitman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Adaptive protein evolution at the <em>Adh</em> locus in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">351</span>: <span class="cit-fpage">652</span>–<span class="cit-lpage">654</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMcDonald%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D351%26rft.issue%253D6328%26rft.spage%253D652%26rft.epage%253D654%26rft.atitle%253DAdaptive%2Bprotein%2Bevolution%2Bat%2Bthe%2BAdh%2Blocus%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F351652a0%26rft_id%253Dinfo%253Apmid%252F1904993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/351652a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1904993&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FT11200066&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.42" data-doi="10.1371/journal.pgen.1000471"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McVicker</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Widespread genomic signatures of natural selection in hominid evolution.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000471</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DMcVicker%26rft.auinit1%253DG.%26rft.volume%253D5%26rft.issue%253D5%26rft.spage%253De1000471%26rft.epage%253De1000471%26rft.atitle%253DWidespread%2Bgenomic%2Bsignatures%2Bof%2Bnatural%2Bselection%2Bin%2Bhominid%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000471%26rft_id%253Dinfo%253Apmid%252F19424416%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000471&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19424416&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.43" data-doi="10.1126/science.1217876"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegmann</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehm</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kessner</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">St Jean</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An abundance of rare functional variants in 202 drug target genes sequenced in 14,002 people.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">100</span>–<span class="cit-lpage">104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1217876%26rft_id%253Dinfo%253Apmid%252F22604722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzcvNjA5MC8xMDAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzMvMTAyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.44" data-doi="10.1111/mec.12524"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peischl</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dupanloup</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirkpatrick</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Excoffier</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">On the accumulation of deleterious mutations during range expansions.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">5972</span>–<span class="cit-lpage">5982</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D5972%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12524&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.45" data-doi="10.1371/journal.pgen.1004697"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Racimo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schraiber</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Approximation to the distribution of fitness effects across functional categories in human segregating polymorphisms.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004697</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004697%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004697%26rft_id%253Dinfo%253Apmid%252F25375159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004697&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25375159&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.46" data-doi="10.1038/nature12961"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sankararaman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mallick</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dannemann</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prüfer</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelso</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The genomic landscape of Neanderthal ancestry in present-day humans.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">507</span>: <span class="cit-fpage">354</span>–<span class="cit-lpage">357</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D507%26rft.spage%253D354%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12961%26rft_id%253Dinfo%253Apmid%252F24476815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12961&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24476815&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000333029000033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sawyer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Population genetics of polymorphism and divergence.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">1161</span>–<span class="cit-lpage">1176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSawyer%26rft.auinit1%253DS.%2BA.%26rft.volume%253D132%26rft.issue%253D4%26rft.spage%253D1161%26rft.epage%253D1176%26rft.atitle%253DPopulation%2Bgenetics%2Bof%2Bpolymorphism%2Band%2Bdivergence.%26rft_id%253Dinfo%253Apmid%252F1459433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzMi80LzExNjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzMvMTAyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.48" data-doi="10.1073/pnas.1416991111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schubert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jónsson</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Der Sarkissian</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ermini</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Prehistoric genomes reveal the genetic foundation and cost of horse domestication.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">E5661</span>–<span class="cit-lpage">E5669</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1416991111%26rft_id%253Dinfo%253Apmid%252F25512547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzUyL0U1NjYxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy8zLzEwMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simons</span>  <span class="cit-name-given-names">Y. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sella</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The impact of recent population history on the deleterious mutation load in humans and close evolutionary relatives.</span> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Simons%2C+Y.+B.%2C+and+G.+Sella%2C+2016%E2%80%83The+impact+of+recent+population+history+on+the+deleterious+mutation+load+in+humans+and+close+evolutionary+relatives.+bioRxiv">Curr. Opin. Genet. Dev.</a> <span class="cit-vol">41</span>: <span class="cit-fpage">150</span>–<span class="cit-lpage">158</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.50" data-doi="10.1038/ng.2896"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simons</span>  <span class="cit-name-given-names">Y. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turchin</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sella</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The deleterious mutation load is insensitive to recent population history.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">220</span>–<span class="cit-lpage">224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D220%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2896%26rft_id%253Dinfo%253Apmid%252F24509481%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2896&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24509481&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.51" data-doi="10.1534/genetics.111.137893"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Subramanian</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The abundance of deleterious polymorphisms in humans.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1579</span>–<span class="cit-lpage">1583</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.137893%26rft_id%253Dinfo%253Apmid%252F22267501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC80LzE1NzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzMvMTAyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.52" data-doi="10.1126/science.1219240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tennessen</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bigham</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Connor</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenny</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evolution and functional impact of rare coding variation from deep sequencing of human exomes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">64</span>–<span class="cit-lpage">69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1219240%26rft_id%253Dinfo%253Apmid%252F22604720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMzcvNjA5MC82NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvMy8xMDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Torkamani</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pham</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Libiger</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bansal</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Clinical implications of human population differences in genome-wide rates of functional genotypes.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFront%2BGenet%26rft.aulast%253DTorkamani%26rft.auinit1%253DA.%26rft.volume%253D3%26rft.spage%253D211%26rft.epage%253D211%26rft.atitle%253DClinical%2Bimplications%2Bof%2Bhuman%2Bpopulation%2Bdifferences%2Bin%2Bgenome-wide%2Brates%2Bof%2Bfunctional%2Bgenotypes.%26rft_id%253Dinfo%253Apmid%252F23125845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23125845&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.54" data-doi="10.1534/genetics.112.148213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jay</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durand</span>  <span class="cit-name-given-names">E. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Higher levels of Neanderthal ancestry in east Asians than in Europeans.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">199</span>–<span class="cit-lpage">209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.148213%26rft_id%253Dinfo%253Apmid%252F23410836%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzEvMTk5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy8zLzEwMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.55" data-doi="10.1073/pnas.0502300102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fledel-Alon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Simultaneous inference of selection and population growth from patterns of variation in the human genome.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">7882</span>–<span class="cit-lpage">7887</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0502300102%26rft_id%253Dinfo%253Apmid%252F15905331%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAyLzIyLzc4ODIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzMvMTAyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.56" data-doi="10.1126/science.aaa3952"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prado-Martinez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sudmant</span>  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Narasimhan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayub</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Mountain gorilla genomes reveal the impact of long-term population decline and inbreeding.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">348</span>: <span class="cit-fpage">242</span>–<span class="cit-lpage">245</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DXue%26rft.auinit1%253DY.%26rft.volume%253D348%26rft.issue%253D6231%26rft.spage%253D242%26rft.epage%253D245%26rft.atitle%253DMountain%2Bgorilla%2Bgenomes%2Breveal%2Bthe%2Bimpact%2Bof%2Blong-term%2Bpopulation%2Bdecline%2Band%2Binbreeding%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaa3952%26rft_id%253Dinfo%253Apmid%252F25859046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDgvNjIzMS8yNDIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzMvMTAyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.1027.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ota</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jorde</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pamilo</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Larger genetic differences within Africans than between Africans and Eurasians.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">269</span>–<span class="cit-lpage">274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DYu%26rft.auinit1%253DN.%26rft.volume%253D161%26rft.issue%253D1%26rft.spage%253D269%26rft.epage%253D274%26rft.atitle%253DLarger%2BGenetic%2BDifferences%2BWithin%2BAfricans%2BThan%2BBetween%2BAfricans%2Band%2BEurasians%26rft_id%253Dinfo%253Apmid%252F12019240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYxLzEvMjY5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy8zLzEwMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/3/1027.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/3/1019" title="Allelic Dropout During Polymerase Chain Reaction due to G-Quadruplex Structures and DNA Methylation Is Widespread at Imprinted Human Loci" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/3/1039" title="Saccharomyces cerevisiae FLO1 Gene Demonstrates Genetic Linkage to Increased Fermentation Rate at Low Temperatures" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/3"><span>Volume 7 Issue 3, March 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/3.cover-source.jpg?itok=h8yj2GBo" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/3/1027.long" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-DS3r_Uy1i0_9GibhwuPKU_zycsTCE00XfVm4D2mVxx4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/3/1027" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/3/1027" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/3/1027" class="active" data-icon-position="" data-hide-link-title="0">A Temporal Perspective on the Interplay of Demography and Selection on Deleterious Variation in Humans</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/483518" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-kg2cIDaRrKAiJoQyqSKuEEnvPDcxa4pCei_EI2Qo7fY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/3/1027.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/3/1027" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-iZ0k1IBZ9hDmfZtxRUhMUk1z-ys7wLL7ZGJj5wHMTMY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/3/1027" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/3/1027.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483518" id="node483518--2" data-pisa="ggg;7/3/1027" data-pisa-master="ggg;g3.117.039651" data-apath="/ggg/7/3/1027.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Temporal Perspective on the Interplay of Demography and Selection on Deleterious Variation in Humans</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Evan Koch</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5345-0214" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>John Novembre</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1027-1037;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.039651 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483518--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Evan Koch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolution, University of Chicago, Illinois 60637</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Evan%2BKoch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Koch%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEvan%2BKoch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John Novembre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolution, University of Chicago, Illinois 60637</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Human Genetics, University of Chicago, Illinois 60637</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BNovembre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Novembre%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BNovembre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5345-0214" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John Novembre</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jnovembre@uchicago.edu" class="" data-icon-position="" data-hide-link-title="0">jnovembre@uchicago.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/3/1027" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/3/1027&amp;title=A%20Temporal%20Perspective%20on%20the%20Interplay%20of%20Demography%20and%20Selection%20on%20Deleterious%20Variation%20in%20Humans" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/3/1027&amp;title=A%20Temporal%20Perspective%20on%20the%20Interplay%20of%20Demography%20and%20Selection%20on%20Deleterious%20Variation%20in%20Humans" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/3/1027&amp;title=A%20Temporal%20Perspective%20on%20the%20Interplay%20of%20Demography%20and%20Selection%20on%20Deleterious%20Variation%20in%20Humans" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/3/1027&amp;text=A%20Temporal%20Perspective%20on%20the%20Interplay%20of%20Demography%20and%20Selection%20on%20Deleterious%20Variation%20in%20Humans" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/3/1027&amp;title=A%20Temporal%20Perspective%20on%20the%20Interplay%20of%20Demography%20and%20Selection%20on%20Deleterious%20Variation%20in%20Humans" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/3/1027&amp;t=A%20Temporal%20Perspective%20on%20the%20Interplay%20of%20Demography%20and%20Selection%20on%20Deleterious%20Variation%20in%20Humans" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/3/1027&amp;title=A%20Temporal%20Perspective%20on%20the%20Interplay%20of%20Demography%20and%20Selection%20on%20Deleterious%20Variation%20in%20Humans" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/3/1027&amp;title=A%20Temporal%20Perspective%20on%20the%20Interplay%20of%20Demography%20and%20Selection%20on%20Deleterious%20Variation%20in%20Humans" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/483518/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483518" id="node483518--3" data-pisa="ggg;7/3/1027" data-pisa-master="ggg;g3.117.039651" data-apath="/ggg/7/3/1027.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Temporal Perspective on the Interplay of Demography and Selection on Deleterious Variation in Humans</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Evan Koch</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5345-0214" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>John Novembre</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1027-1037;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.039651 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483518--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Evan Koch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolution, University of Chicago, Illinois 60637</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Evan%2BKoch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Koch%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEvan%2BKoch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John Novembre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolution, University of Chicago, Illinois 60637</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Human Genetics, University of Chicago, Illinois 60637</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BNovembre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Novembre%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BNovembre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5345-0214" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John Novembre</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jnovembre@uchicago.edu" class="" data-icon-position="" data-hide-link-title="0">jnovembre@uchicago.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1027&amp;rft.epage=1037&amp;rft.atitle=A%20Temporal%20Perspective%20on%20the%20Interplay%20of%20Demography%20and%20Selection%20on%20Deleterious%20Variation%20in%20Humans&amp;rft.volume=7&amp;rft.issue=3&amp;rft.date=2017-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Koch%2C+Evan&amp;rft.au=Novembre%2C+John"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/483518/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/483518/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/483518/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/483518/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/483518/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/483518/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/483518/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/483518/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/483518/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/483518/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/483518/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/483518/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/3/1027" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/3/1027.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/3/1027.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/3/1027.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/3/1027.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
