<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/3/881" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/3/881.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/3/881.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/3/881.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/3/881.atom" />
<meta name="HW.pisa" content="ggg;7/3/881" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Adapting Genotyping-by-Sequencing for Rice F2 Populations" />
<meta name="DC.Identifier" content="10.1534/g3.116.038190" />
<meta name="DC.Date" content="2017-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Furuta et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Rapid and cost-effective genotyping of large mapping populations can be achieved by sequencing a reduced representation of the genome of every individual in a given population, and using that information to generate genetic markers. A customized genotyping-by-sequencing (GBS) pipeline was developed to genotype a rice F2 population from a cross of Oryza sativa ssp. japonica cv. Nipponbare and the African wild rice species O. longistaminata . While most GBS pipelines aim to analyze mainly homozygous populations, we attempted to genotype a highly heterozygous F2 population. We show how species- and population-specific improvements of established protocols can drastically increase sample throughput and genotype quality. Using as few as 50,000 reads for some individuals (134,000 reads on average), we were able to generate up to 8154 informative SNP markers in 1081 F2 individuals. Additionally, the effects of enzyme choice, read coverage, and data postprocessing are evaluated. Using GBS-derived markers, we were able to assemble a genetic map of 1536 cM. To demonstrate the usefulness of our GBS pipeline, we determined quantitative trait loci (QTL) for the number of tillers. We were able to map four QTL to chromosomes 1, 3, 4, and 8, and partially confirm their effects using introgression lines. We provide an example of how to successfully use GBS with heterozygous F2 populations. By using the comparatively low-cost MiSeq platform, we show that the GBS method is flexible and cost-effective, even for smaller laboratories." />
<meta name="DC.Contributor" content="Tomoyuki Furuta" />
<meta name="DC.Contributor" content="Motoyuki Ashikari" />
<meta name="DC.Contributor" content="Kshirod K. Jena" />
<meta name="DC.Contributor" content="Kazuyuki Doi" />
<meta name="DC.Contributor" content="Stefan Reuscher" />
<meta name="article:published_time" content="2017-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Adapting Genotyping-by-Sequencing for Rice F2 Populations" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Rapid and cost-effective genotyping of large mapping populations can be achieved by sequencing a reduced representation of the genome of every individual in a given population, and using that information to generate genetic markers. A customized genotyping-by-sequencing (GBS) pipeline was developed to genotype a rice F2 population from a cross of &lt;i&gt;Oryza sativa&lt;/i&gt; ssp. &lt;i&gt;japonica&lt;/i&gt; cv. Nipponbare and the African wild rice species &lt;i&gt;O. longistaminata&lt;/i&gt;. While most GBS pipelines aim to analyze mainly homozygous populations, we attempted to genotype a highly heterozygous F2 population. We show how species- and population-specific improvements of established protocols can drastically increase sample throughput and genotype quality. Using as few as 50,000 reads for some individuals (134,000 reads on average), we were able to generate up to 8154 informative SNP markers in 1081 F2 individuals. Additionally, the effects of enzyme choice, read coverage, and data postprocessing are evaluated. Using GBS-derived markers, we were able to assemble a genetic map of 1536&amp;nbsp;cM. To demonstrate the usefulness of our GBS pipeline, we determined quantitative trait loci (QTL) for the number of tillers. We were able to map four QTL to chromosomes 1, 3, 4, and 8, and partially confirm their effects using introgression lines. We provide an example of how to successfully use GBS with heterozygous F2 populations. By using the comparatively low-cost MiSeq platform, we show that the GBS method is flexible and cost-effective, even for smaller laboratories.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/03/01" />
<meta name="citation_mjid" content="ggg;7/3/881" />
<meta name="citation_id" content="7/3/881" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/3/881" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/3/881.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/3/881.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/3/881.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.038190" />
<meta name="citation_pmid" content="28082325" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="881" />
<meta name="citation_lastpage" content="893" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Tomoyuki Furuta" />
<meta name="citation_author_institution" content="Bioscience and Biotechnology Center, Nagoya University, 464-8601, Japan" />
<meta name="citation_author" content="Motoyuki Ashikari" />
<meta name="citation_author_institution" content="Bioscience and Biotechnology Center, Nagoya University, 464-8601, Japan" />
<meta name="citation_author" content="Kshirod K. Jena" />
<meta name="citation_author_institution" content="Plant Breeding Division, International Rice Research Institute, 1301 Manila, Philippines" />
<meta name="citation_author" content="Kazuyuki Doi" />
<meta name="citation_author_institution" content="Associated Field Science and Research Center, Nagoya University, 470-0151, Japan" />
<meta name="citation_author_email" content="kdoi@agr.nagoya-u.ac.jp" />
<meta name="citation_author_email" content="reuscher@agr.nagoya-u.ac.jp" />
<meta name="citation_author" content="Stefan Reuscher" />
<meta name="citation_author_institution" content="Bioscience and Biotechnology Center, Nagoya University, 464-8601, Japan" />
<meta name="citation_author_email" content="kdoi@agr.nagoya-u.ac.jp" />
<meta name="citation_author_email" content="reuscher@agr.nagoya-u.ac.jp" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=H. Begum;citation_author=J. E. Spindel;citation_author=A. Lalusin;citation_author=T. Borromeo;citation_author=G. Gregorio;citation_title=Genome-wide association mapping for yield and other agronomic traits in an elite breeding population of tropical rice (Oryza sativa).;citation_pages=e0119873;citation_volume=10;citation_year=2015;citation_pmid=25785447;citation_doi=10.1371/journal.pone.0119873" />
<meta name="citation_reference" content="citation_title=A Guide to QTL Mapping with R/qtl.;citation_author=K. W. Broman;citation_author=Ś. Sen;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=S. Sen;citation_author=G. A. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=A. M. Burrell;citation_author=A. E. Pepper;citation_author=G. Hodnett;citation_author=J. A. Goolsby;citation_author=W. A. Overholt;citation_title=Exploring origins, invasion history and genetic diversity of Imperata cylindrica (L.) P. Beauv. (Cogongrass) in the United States using genotyping by sequencing.;citation_pages=2177-2193;citation_volume=24;citation_year=2015;citation_doi=10.1111/mec.13167" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. Darvasi;citation_title=Experimental strategies for the genetic dissection of complex traits in animal models.;citation_pages=19-24;citation_volume=18;citation_year=1998;citation_issue=1;citation_pmid=9425894;citation_doi=10.1038/ng0198-19" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. W. Davey;citation_author=P. A. Hohenlohe;citation_author=P. D. Etter;citation_author=J. Q. Boone;citation_author=J. M. Catchen;citation_title=Genome-wide genetic marker discovery and genotyping using next-generation sequencing.;citation_pages=499-510;citation_volume=12;citation_year=2011;citation_issue=7;citation_pmid=21681211;citation_doi=10.1038/nrg3012" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. De Donato;citation_author=S. O. Peters;citation_author=S. E. Mitchell;citation_author=T. Hussain;citation_author=I. G. Imumorin;citation_title=Genotyping-by-sequencing (GBS): a novel, efficient and cost-effective genotyping method for cattle using next-generation sequencing.;citation_pages=e62137;citation_volume=8;citation_year=2013;citation_pmid=23690931;citation_doi=10.1371/journal.pone.0062137" />
<meta name="citation_reference" content="citation_journal_title=Phytochem. Bull.;citation_author=J. J. Doyle;citation_author=J. L. Doyle;citation_title=A rapid DNA isolation procedure for small quantities of fresh leaf tissue.;citation_pages=11-15;citation_volume=19;citation_year=1987" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. Duitama;citation_author=A. Silva;citation_author=Y. Sanabria;citation_author=D. F. Cruz;citation_author=C. Quintero;citation_title=Whole genome sequencing of elite rice cultivars as a comprehensive information resource for marker assisted selection.;citation_pages=e0124617;citation_volume=10;citation_year=2015;citation_pmid=25923345;citation_doi=10.1371/journal.pone.0124617" />
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol.;citation_author=I. Elmer;citation_author=S. Humira;citation_author=B. François;citation_title=Association mapping of QTLs for sclerotinia stem rot resistance in a collection of soybean plant introductions using a genotyping by sequencing (GBS) approach.;citation_pages=5;citation_volume=15;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. C. Glaubitz;citation_author=T. M. Casstevens;citation_author=F. Lu;citation_author=J. Harriman;citation_author=R. J. Elshire;citation_title=TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline.;citation_pages=e90346;citation_volume=9;citation_year=2014;citation_pmid=24587335;citation_doi=10.1371/journal.pone.0090346" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=J. L. Gualdrón Duarte;citation_author=R. O. Bates;citation_author=C. W. Ernst;citation_author=N. E. Raney;citation_author=R. J. Cantet;citation_title=Genotype imputation accuracy in a F2 pig population using high density and low density SNP panels.;citation_pages=38;citation_volume=14;citation_year=2013;citation_pmid=23651538" />
<meta name="citation_reference" content="citation_title=Statistical Genomics: Methods and Protocols;citation_title=Visualizing genomic data using Gviz and Bioconductor;citation_author=F. Hahne;citation_author=R. Ivanek;citation_pages=335-351;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=J. He;citation_author=X. Zhao;citation_author=A. Laroche;citation_author=Z.-X. Lu;citation_author=H. Liu;citation_title=Genotyping-by-sequencing (GBS), an ultimate marker-assisted selection (MAS) tool to accelerate plant breeding.;citation_pages=484;citation_volume=5;citation_year=2014;citation_pmid=25324846" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=C. Heffelfinger;citation_author=C. A. Fragoso;citation_author=M. A. Moreno;citation_author=J. D. Overton;citation_author=J. P. Mottinger;citation_title=Flexible and scalable genotyping-by-sequencing strategies for population studies.;citation_pages=979;citation_volume=15;citation_year=2014;citation_pmid=25406744;citation_doi=10.1186/1471-2164-15-979" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=N. Honsdorf;citation_author=T. March;citation_author=A. Hecht;citation_author=J. Eglinton;citation_author=K. Pillen;citation_title=Evaluation of juvenile drought stress tolerance and genotyping by sequencing with wild barley introgression lines.;citation_pages=1475-1495;citation_volume=34;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Y.-F. Huang;citation_author=J. A. Poland;citation_author=C. P. Wight;citation_author=E. W. Jackson;citation_author=N. A. Tinker;citation_title=Using genotyping-by-sequencing (GBS) for genomic discovery in cultivated oat.;citation_pages=e102448;citation_volume=9;citation_year=2014;citation_pmid=25047601;citation_doi=10.1371/journal.pone.0102448" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=K. E. Hyma;citation_author=P. Barba;citation_author=M. Wang;citation_author=J. P. Londo;citation_author=C. B. Acharya;citation_title=Heterozygous mapping strategy (HetMappS) for high resolution genotyping-by-sequencing markers: a case study in grapevine.;citation_pages=e0134880;citation_volume=10;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. L. Johnson;citation_author=H. Wittgenstein;citation_author=S. E. Mitchell;citation_author=K. E. Hyma;citation_author=S. V. Temnykh;citation_title=Genotyping-by-sequencing (GBS) detects genetic structure and confirms behavioral QTL in tame and aggressive foxes (Vulpes vulpes).;citation_pages=e0127013;citation_volume=10;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Rice (N. Y.);citation_author=Y. Kawahara;citation_author=M. de la Bastide;citation_author=J. P. Hamilton;citation_author=H. Kanamori;citation_author=W. R. McCombie;citation_title=Improvement of the Oryza sativa Nipponbare reference genome using next generation sequence and optical map data.;citation_pages=4;citation_volume=6;citation_year=2013;citation_pmid=24280374;citation_doi=10.1186/1939-8433-6-4" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=M. J. Kearsey;citation_author=A. G. Farquhar;citation_title=QTL analysis in plants; where are we now?;citation_pages=137-142;citation_volume=80;citation_year=1998;citation_pmid=9503632;citation_doi=10.1046/j.1365-2540.1998.00500.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=S. G. Krishnan;citation_author=D. L. Waters;citation_author=R. J. Henry;citation_title=Australian wild rice reveals pre-domestication origin of polymorphism deserts in rice genome.;citation_pages=e98843;citation_volume=9;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. Lin;citation_author=S. Cai;citation_author=S. Wang;citation_author=S. Liu;citation_author=G. Zhang;citation_title=Genotyping-by-sequencing (GBS) identified SNP tightly linked to QTL for pre-harvest sprouting resistance.;citation_pages=1385-1395;citation_volume=128;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=H. Liu;citation_author=M. Bayer;citation_author=A. Druka;citation_author=J. R. Russell;citation_author=C. A. Hackett;citation_title=An evaluation of genotyping by sequencing (GBS) to map the Breviaristatum-e (ari-e) locus in cultivated barley.;citation_pages=104;citation_volume=15;citation_year=2014;citation_pmid=24498911;citation_doi=10.1186/1471-2164-15-104" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=N. J. Loman;citation_author=R. V. Misra;citation_author=T. J. Dallman;citation_author=C. Constantinidou;citation_author=S. E. Gharbia;citation_title=Performance comparison of benchtop high-throughput sequencing platforms.;citation_pages=434-439;citation_volume=30;citation_year=2012;citation_issue=5;citation_pmid=22522955;citation_doi=10.1038/nbt.2198" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=F. Lu;citation_author=A. E. Lipka;citation_author=J. Glaubitz;citation_author=R. Elshire;citation_author=J. H. Cherney;citation_title=Switchgrass genomic diversity, ploidy, and evolution: novel insights from a network-based SNP discovery protocol.;citation_pages=e1003215;citation_volume=9;citation_year=2013;citation_pmid=23349638;citation_doi=10.1371/journal.pgen.1003215" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. A. Poland;citation_author=T. W. Rife;citation_title=Genotyping-by-sequencing for plant breeding and genetics.;citation_pages=92-102;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.05.0005" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. A. Poland;citation_author=P. J. Brown;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.;citation_pages=e32253-e32253;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22389690;citation_doi=10.1371/journal.pone.0032253" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_author=W. Pootakham;citation_author=N. Jomchai;citation_author=P. Ruang-Areerate;citation_author=J. R. Shearman;citation_author=C. Sonthirod;citation_title=Genome-wide SNP discovery and identification of QTL associated with agronomic traits in oil palm using genotyping-by-sequencing (GBS).;citation_pages=288-295;citation_volume=105;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Virus Res.;citation_author=I. Y. Rabbi;citation_author=M. T. Hamblin;citation_author=P. L. Kumar;citation_author=M. A. Gedil;citation_author=A. S. Ikpan;citation_title=High-resolution mapping of resistance to cassava mosaic geminiviruses in cassava using genotyping-by-sequencing and its implications for breeding.;citation_pages=87-96;citation_volume=186;citation_year=2014;citation_pmid=24389096;citation_doi=10.1016/j.virusres.2013.12.028" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=J. M. Ramos;citation_author=T. Furuta;citation_author=K. Uehara;citation_author=N. Chihiro;citation_author=R. B. Angeles-Shim;citation_title=Development of chromosome segment substitution lines (CSSLs) of Oryza longistaminata A. Chev. &amp; Röhr in the background of the elite japonica rice cultivar, Taichung 65 and their evaluation for yield traits.;citation_pages=151-163;citation_volume=210;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=P. Rice;citation_author=I. Longden;citation_author=A. Bleasby;citation_title=EMBOSS: the European Molecular Biology Open Software Suite.;citation_pages=276-277;citation_volume=16;citation_year=2000;citation_issue=6;citation_pmid=10827456;citation_doi=10.1016/S0168-9525(00)02024-2" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. C. Romay;citation_author=M. J. Millard;citation_author=J. C. Glaubitz;citation_author=J. A. Peiffer;citation_author=K. L. Swarts;citation_title=Comprehensive genotyping of the USA national maize inbred seed bank.;citation_pages=R55;citation_volume=14;citation_year=2013;citation_pmid=23759205;citation_doi=10.1186/gb-2013-14-6-r55" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=B. A. Rowan;citation_author=V. Patel;citation_author=D. Weigel;citation_author=K. Schneeberger;citation_title=Rapid and inexpensive whole-genome genotyping-by-sequencing for crossover localization and fine-scale genetic mapping.;citation_pages=385-398;citation_volume=5;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=S. Schröder;citation_author=S. Mamidi;citation_author=R. Lee;citation_author=M. R. McKain;citation_author=P. E. McClean;citation_title=Optimization of genotyping by sequencing (GBS) data in common bean (Phaseolus vulgaris L.).;citation_pages=1-9;citation_volume=36;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Ś. Sen;citation_author=G. A. Churchill;citation_title=A Statistical Framework for Quantitative Trait Mapping;citation_pages=371-387;citation_volume=159;citation_year=2001;citation_issue=1;citation_pmid=11560912" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=H. Sonah;citation_author=L. O’Donoughue;citation_author=E. Cober;citation_author=I. Rajcan;citation_author=F. Belzile;citation_title=Identification of loci governing eight agronomic traits using a GBS-GWAS approach and validation by QTL mapping in soya bean.;citation_pages=211-221;citation_volume=13;citation_year=2015;citation_pmid=25213593;citation_doi=10.1111/pbi.12249" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. Spindel;citation_author=M. Wright;citation_author=C. Chen;citation_author=J. Cobb;citation_author=J. Gage;citation_title=Bridging the genotyping gap: using genotyping by sequencing (GBS) to add high-density SNP markers and new value to traditional bi-parental mapping and breeding populations.;citation_pages=2699-2716;citation_volume=126;citation_year=2013;citation_pmid=23918062;citation_doi=10.1007/s00122-013-2166-x" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=H. Takagi;citation_author=A. Abe;citation_author=K. Yoshida;citation_author=S. Kosugi;citation_author=S. Natsume;citation_title=QTL-seq: rapid mapping of quantitative trait loci in rice by whole genome resequencing of DNA from two bulked populations.;citation_pages=174-183;citation_volume=74;citation_year=2013;citation_pmid=23289725;citation_doi=10.1111/tpj.12105" />
<meta name="citation_reference" content="citation_journal_title=Journal of evolutionary biology;citation_journal_abbrev=J Evol Biol;citation_author=L. Wang;citation_author=L. Hao;citation_author=X. Li;citation_author=S. Hu;citation_author=S. Ge;citation_title=SNP deserts of Asian cultivated rice: genomic regions under domestication.;citation_pages=751-761;citation_volume=22;citation_year=2009;citation_issue=4;citation_pmid=19243488" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=M. Wang;citation_author=Y. Yu;citation_author=G. Haberer;citation_author=P. R. Marri;citation_author=C. Fan;citation_title=The genome sequence of African rice (Oryza glaberrima) and evidence for independent domestication.;citation_pages=982-988;citation_volume=46;citation_year=2014;citation_pmid=25064006;citation_doi=10.1038/ng.3044" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant;citation_author=Y. Zhang;citation_author=S. Zhang;citation_author=H. Liu;citation_author=B. Fu;citation_author=L. Li;citation_title=Genome and comparative transcriptomics of African wild rice Oryza longistaminata provide insights into molecular mechanism of rhizomatousness and self-incompatibility.;citation_pages=1683-1686;citation_volume=8;citation_year=2015" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Adapting Genotyping-by-Sequencing for Rice F2 Populations" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3.cover-source.jpg" />
<meta name="twitter:description" content="Rapid and cost-effective genotyping of large mapping populations can be achieved by sequencing a reduced representation of the genome of every individual in a given population, and using that information to generate genetic markers. A customized genotyping-by-sequencing (GBS) pipeline was developed to genotype a rice F2 population from a cross of Oryza sativa ssp. japonica cv. Nipponbare and the African wild rice species O. longistaminata . While most GBS pipelines aim to analyze mainly homozygous populations, we attempted to genotype a highly heterozygous F2 population. We show how species- and population-specific improvements of established protocols can drastically increase sample throughput and genotype quality. Using as few as 50,000 reads for some individuals (134,000 reads on average), we were able to generate up to 8154 informative SNP markers in 1081 F2 individuals. Additionally, the effects of enzyme choice, read coverage, and data postprocessing are evaluated. Using GBS-derived markers, we were able to assemble a genetic map of 1536 cM. To demonstrate the usefulness of our GBS pipeline, we determined quantitative trait loci (QTL) for the number of tillers. We were able to map four QTL to chromosomes 1, 3, 4, and 8, and partially confirm their effects using introgression lines. We provide an example of how to successfully use GBS with heterozygous F2 populations. By using the comparatively low-cost MiSeq platform, we show that the GBS method is flexible and cost-effective, even for smaller laboratories." />
<meta name="og-title" property="og:title" content="Adapting Genotyping-by-Sequencing for Rice F2 Populations" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/3/881" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Rapid and cost-effective genotyping of large mapping populations can be achieved by sequencing a reduced representation of the genome of every individual in a given population, and using that information to generate genetic markers. A customized genotyping-by-sequencing (GBS) pipeline was developed to genotype a rice F2 population from a cross of Oryza sativa ssp. japonica cv. Nipponbare and the African wild rice species O. longistaminata . While most GBS pipelines aim to analyze mainly homozygous populations, we attempted to genotype a highly heterozygous F2 population. We show how species- and population-specific improvements of established protocols can drastically increase sample throughput and genotype quality. Using as few as 50,000 reads for some individuals (134,000 reads on average), we were able to generate up to 8154 informative SNP markers in 1081 F2 individuals. Additionally, the effects of enzyme choice, read coverage, and data postprocessing are evaluated. Using GBS-derived markers, we were able to assemble a genetic map of 1536 cM. To demonstrate the usefulness of our GBS pipeline, we determined quantitative trait loci (QTL) for the number of tillers. We were able to map four QTL to chromosomes 1, 3, 4, and 8, and partially confirm their effects using introgression lines. We provide an example of how to successfully use GBS with heterozygous F2 populations. By using the comparatively low-cost MiSeq platform, we show that the GBS method is flexible and cost-effective, even for smaller laboratories." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/3.cover-source.jpg" />
    <title>Adapting Genotyping-by-Sequencing for Rice F2 Populations | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"rOlh8cezcWyXQL9YBw7GXdPRLf7QNIGFdoppayWcfUA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"483523","apath":"\/ggg\/7\/3\/881.atom","pisa":"ggg;7\/3\/881","ac":{"\/ggg\/7\/3\/881.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/3\/881.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/3\/881"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/3\/881"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/3\/881":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022aQIet\\\\\\\/ca9T9cdlJ4U3ha8YBV8jVU3EZhm+STV897pRdZOw2lDzhOcq4j\\\\\\\/rt4G4D9wz9lMjEic9wKob33DsNWTaTA5TFyLygbQDj76gmOkoAOXfJTNvDbnKstxEdZSoaAbF\\\\\\\/L8Sv5SB4UNbb4J3N6\\\\\\\/RuExmN4m0Of2lyL1JObfp\\\\\\\/7RdLv6hNC7vaFgCsqUGEdLmv\\\\\\\/KGQWJxFLG2UdzPPVQYJ32ImumI1Ne0VleYrJnt2kJuKwYzVZUiNV8JZ0PJPkAADrP3hq\\\\\\\/h2rWBqaO04L0x5PK8cNIe8Z0SZWbROiiJcFfUIq6TljWmGLFhwwZfUJzaVjqygNdiwDJACOL98yAL4ZaSeNkOaur\\\\\\\/+1tND53N0UDye\\\\\\\/Xzq+z0s09dMmTsq1lpsRjW1wF6PcK24LwIq0y5n+89U5MEWY1qF4CPygNTEm43XHwRNT5LikSe1fqonEbTPJgTEuWE5Ftn+uTjfn1B5I0jw83Y+9jBWnpB1BuhYVSSh6M0Xy5N6Nx7GspEiqEY0HTgf48qJj8NDIn7mNP1KWqGOpU9nacNfQbBiQZHf6emSgppweo6+Dhv9MYKO764anz4sLN8uMc\\\\\\\/JUbvtlPJYPg\\\\\\\/FwdmBrFjvUEDDOcQkVwlg\\\\\\\/0AZ\\\\\\\/NDEB8moBT\\\\\\\/fRjx50VDhJJ0EKP+uMDh2gwlXiLotW+frtUlVCd+tsU6CIMKRSpnugKdTzZppnIjjCVuxIb68mMHgWWamwvEEZjwlxK7pxOiLVI8bHXXBvumObgh1mNq\\\\\\\/E5jt8j8xtf612fvRpEADzjDV9mGGBW763Ri8GWm59qvBXVfVrWWiviJq+wI2chNkqf6qjmLJDBzozcE5GcM17k3PVmys5pLYXGbNhdhNaQMMhiuKNlbxJNyprU7uGy5MwfW3QT4PpwysnzHOnr8PRvTRXXRwlh6Ip1fql4EuQkmnTDCu1Z258Jz3PsSBgcertZESO4Ed52DKTvStOyBdNtEjGWqsaVEiKpINSa3FFpQlI\\\\\\\/jpXBSWxFcices\\\\\\\/MekJV9l288yN\\\\\\\/PCNdjxGyuQC7FdAZa2m7wS0EHD9G2PYV+pDOL502S0Zc\\\\\\\/q35RqiBTQgZ1vR5siVXfmKi+XPpojsztVXm8JLDoLWLQgqXTDaDBn+G6klLpGFXPHV1pDLVPtHDHB6Cde\\\\\\\/YogePqArJ5gx3LRRQOd3vCt4ydYR0F4K7Ic+WhMd8FWOUFwm4GPKxbKUw2zWyDLE6NMTXt2RSbBWYPk2jWokYsfVcTu16KfTRA6oXORN5fM3OBXNc2GkyVIpGNTwuI9cCvMf6\\\u0022,\\\u0022iv\\\u0022:\\\u0022QoryVBCs746JRxd3wNtq3g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229be67f33c66182545d3c40899ef6b65bffff1a3cc5ef698a47c0fcad82fed5e8\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022DjM5xvWeTOL\\\\\\\/v4zDdvfO3OmXaDmZ2r0tp0lE8tQdu8pNedvNEh6avJl4ROb7iXecBnChtTzAA2WoPNugcZTEPeU8q3aKvpAG9xGE+fYHGJ3KV+4CB4vzvaVOlzCyiHuhfAxwEYEbcn4vzEvBXtfEB4y+1ReIokdVpV2KyyF9AzGseDPhJiqSP5dxLFXsIHWbxADT3F17Fv4rbmPqQZItGHpsU448NjrQHhq7UG\\\\\\\/f8dZtihHU3AB\\\\\\\/XYIBHrPdsG+WFYTbPyttt9bhkFcyhiiLXaAEz06g+RF8AE0fPiPV70eMC4nDicdtxY9Lwc4fSfZCV+jYdtLxBpZ1h8rVgdwGIDNRyTfJf8egvXl+kCRSueTg6jCdiMzuk+EhGdN56xE2zufoT8vaOj43NvNd8DTv4dC96gRuNm89U9iWAJ7Z5WOWJNoEeSMJ\\\\\\\/dHlCjekEwTl9ouSgpFhK73XC9uF3cj3lbYpS8Or83wc6uTihU29iZsVOCfeI7\\\\\\\/LqrCknM2OwEeFT3krzTv40z\\\\\\\/miZO5NWbuMUvNphV20dPTFQOdgSAsYx2Xl9W+kQWWe\\\\\\\/CMgx6sB9sJQ\\\\\\\/Rm0EgIwZSI\\\\\\\/\\\\\\\/ZNlcBWEhSff7l48Go7jNMOO2sXULGMVR7PDlqVSJ+Ng95eghg9tiZ2XXaKR\\\\\\\/oc7dUjAyWUiq\\\\\\\/tO2XQ9vLUbNkqexRqBRh2en\\\\\\\/iO3zU92HK7aKfSlFmrlladWw6f4uCNk2zdOLxUemFbfxQb5r4CEMVXvlAVSg33Dr7l5gCZ7DQSL2A9h5Q71QnzA95uCCPL+8WZjOzzIns2JrPckoHbB2cAX99j\\\\\\\/RoNOBE8rtcYdPthEDI5BpcXPFu6l04nI49OwcDtlrX0sXm6meO8E0CSZrxJeF53K1NPYC9kHS4zThq6gGeRmar2HlPYccUGWBhOatcHf3bH4WC9n7Kx1ts514zr0UhlVUzl6dbsZWmpoYlEyfyyELbEaP3Mf7Mz1MFiS\\\\\\\/JKH3xySnhO6YYOIfI+eEgphMqJtvTiUqgP4IlCGiKZWH0RF0SbZ2YfQao98cCoYpmKbFlW5cPEg6J5YYpc5nW+sQ6Gj4T3MLGin0N2Pkt3sax3fzMBrUderIoEHyPArlWdQIvtS7dXShTBpRivojONSGi++SXaLisypu14Sk9Jd0i5U3g6rGM0a6BYNp1fVEnKyN2vtFyhA7wisRN5lOUeLXtYaH\\\\\\\/czjSe7N8g9OeYShzbBnG7sq4FQakY++78POKLWKALpjr6nJ+0bNTfSxvKZtQiq3OzPYos7QGX2wp8XsrWbazPvN\\\\\\\/9N5p3sFf2aLIuvRyHbgAtRRzRRcAEnSdDv97GDkmvM+vZYAz6wKj1DGVUoYGmWhtnZUmVHXENcdE5tAZExgXZFNh0WMhROXKmO4+qQaTNgHfiynnKaf9Gk\\\\\\\/+mTG2nWbeUWlveROlh2AT5PMFMcDtJJOVwGuHyjMOwls8Pa69kxaJTw1OVibjWcCBDssOonBd2v6v5Cnv7rPFwgVfdFD\\\\\\\/g\\\\\\\/Sr+f9f928Fxd5awJsceV0s30fFqjaQlsubDXyDBOL1X6V\\\\\\\/WwJaY35LKWzuS2NscgQHe4zrU1WmYZkWMlRmgvz3x8Gsoo6oxhID+tZiR57FW7T2JhTvxVIEvs+j9NNNFUnpzUP88d+n0dvDOf0eVTAVnlcT\\\\\\\/bz4PTVso2qEMsydfzja3rvOUyP6oSuUYPjkv+\\\\\\\/sutE\\\\\\\/0VFmeN0HC64wPhCKRx9jgyhGtN4Zf0FiXnLMkGLd2srOHyOeVQdTLq+5vDUhMUPT96qiA7RaJ+J\\\\\\\/FaX3ZLgu3QNgwgV\\\\\\\/RpHMsLY+OKaD6tT0t5EBF\\\\\\\/GIvwVg1Xqq\\\\\\\/T3gD\\\\\\\/3R1e0A+T9BdUgMTNC0pDevNa6n\\\\\\\/IzYyAGjmRs5ZmsL1EilVNnYTfSSIkFwp5kv7wdIkOJnhYCllQa4EI6kXCkSt6P+HwlWEtdRT3X3rK3hfMS8oi5HCavx1zpvgW6yPC8yTKwgYsgYrAls+\\\\\\\/wuChRv2pM3uqjLg05nW7Aqo6gPOhON1Jhj+uZi19pqhROrxO3R3uht5IG0MMJpl6yJnz+9GA4F7pArvDuOR54ES+aRiKpNeiLtgkHExMjhssGNOtUTFXWXhrXRziCue7DV7nk9+Zki0xHT3OC0A4oKIVRrdqPPejWg8C4zzrRgZOS9LKuJx9iKFUJAWOUIwVWF4XBp3qvtm254T9dlU2gzXEyBWF9\\\\\\\/RfkiuYZT9f7rF6rn4osSyGmUmFJ9S82AvN+VS23wNhTjIxFK1iJ\\\\\\\/oUdFvlLGmkosfVPe1O7m\\\\\\\/wWQFWIeARi\\\\\\\/b+GIDwEXCK\\\\\\\/QT0wBmRUvBzFgqOQ\\\\\\\/6LqyJ0iq+l4zhMdFgoQX\\\\\\\/WpPqx1SjGtc\\\\\\\/CU13itDtZgsrCWSXTL8cLq4FuiQStGyas7zubf\\\\\\\/Ih9tnwpAXOzsAVxqrHPjNUYdZuvGnLuLjiS1Kl2+tVXA95009dwzjw3GQWJuuI2l0SofTlktP6t4qn1UHWEmBgyjJfNG4m1MiXBIwmtjkpV2CLcWJ2a0dx4aSbYFTL3OCU0r\\\\\\\/VLiVKGBXBaqkEg8QVwwbkzp3KmNvKZSr\\\\\\\/H940+T9CfMQudYY8mGM+AUaxhhNNHEwUlCp9X0k4pRz3ODEhRV6Wx5K3XVKmpFI4AWoD23\\\\\\\/cJiqHnHYudQ5VL8vtRPQAU7nTWdpBRd+UBJk1xWlmrLGKerIyLSBqPIDfgA17c\\\\\\\/xj2HqzULtuPvxidDAGrB+n+SYpiJCNvd\\\\\\\/wAJBuGxIWvpfESS1PSorLni8g2dLSgnpx2HLzCAm6RZiXFwUjUqUTvdLfUtWucl2\\\\\\\/wM5LXO0C8Q3dSjj3URWspJ\\\\\\\/NyI2pcfl5dvvv8ckTke7lFxBIKue1MQfaToqqUUPVG80nNn4okLhUQe9v8ZfhjrwI9EF9g2nJHF0gqRrmPLcwINHNMJHB12kH5pL862nzUZCX\\\\\\\/37E5UsWwKFwD+ky6Pxcraf3cebTg6mYyEzFhbFO+av\\\\\\\/7tCc+UUTDQsgYgWh4ocWzn8PgkbNyfwzdK5ih1sfPEI7xFgqezp\\\\\\\/eG9sXoWU+u3ktz0LHdCh04IxIcxVI4WSHmhGHdUgxwdeRTQywrbMCNcBMEcAUKh6\\\\\\\/IYORR4Jwbdh9gnQckxtm6xKoksdiFnsyrjsWv4GeYjkUWtuoTrdd\\\\\\\/u3RKB6WasxA0BTth9zz29RPgl9DYGHeNl8kWk1Yh7jSrtul\\\\\\\/1yJnqwC4p\\\\\\\/4SnOJNTbJ1hyby4Ar51RIrQNA50ZxVjZyCmTRlnXuAz2EHi1EZblxVJCjUZpQ+Y3RUT1Wiei4HsszNzEULJ2+tpMxe2oDzzXDlUUlgPPmNOnAP5PgFYJJOYg0a6A1Yz9A4dQqOib\\\\\\\/mM+FQUos0jz\\\\\\\/DWbMzzkpf1fJ6viOI3eoWc0a99EobKQbsvj1ZZ4AYd4FU1QoaCFAgo6cRpjq0fcV1jvTYQIt90hc4Pdx+tAxCa2MtliKaCeR1Yfr3rKvOoAINmIQ\\\\\\\/kMK2WPOGeZdX40fOv5mMUQobfgkC0P85CTf0tiCPLGl8McQXIztMkapRCIyE4GO1Os8UnfWLlD2zv0gBLu5bg4QjVdhqR0AaJRwq1BlS26xFbr3BEVy\\\\\\\/CnUnhgNLbVqM+eSa2LN\\\\\\\/v18IXu3WFPfRanBbmIPX2lw+LrKhyUzWbz4utC2MdW7J5rtHcsfHV6Ry\\\\\\\/IwkeMbpmnPpxcPuiuXYndYQtp67qVnEqpadtYUnBkTynO1Gbe4\\\\\\\/i1hJJrtdxqBwubEy5oNAgJU5LWteFdmyOC4F9+RzK6CXICBqIJte8TWt0Q9pVlDGBMWPJ6xDrTv2eV\\\\\\\/xRn+zENY1SvgtWLp0uNuinXuHwtnY7NA1Xkn9ter\\\\\\\/0XfNYF9QLs13+i+OWzCAS1ZEV08kIImeB8vmk5ekBvKkUzxdbABOHf0eSE9BeBf6pwIM0VCC4tVGlKkbWf6l3JjBj\\\\\\\/R1x4SMAdzqgiAIQsVwwM+\\\\\\\/Dd+N5R\\\\\\\/jM2brVH1RWnFfnD8riLfy3NFC69EjCF4zwZ3PWlYoBIeYkocv+czp3ie+pppmDMWHHKxrxoSNmnxEHDzHefosDtExPUqLQeQRkLreySENWihF\\\\\\\/csRzsp15u9+E7y4ABxU0gjijV6EtF94NRC0uTM0fWyTcP9XWrDpW+HhDIRUV3cPlAjzpS3TRZ5LG317Zilrz4J9VucVTjl9GQXxMO9UJtgoyzrKMsQ\\\\\\\/ExOTMzDMRAoHU5NTYd4ddNu+2kNS6kPs5Er072rdE6Z0T12C9BCOniJrFNv7zWTz+dtxHAyzHHIQg1NQtwS2RdVl5xhbl+zovvcclVSaFZQcG6AFfMHud82Men98e+\\\\\\\/scRHvFFWTV4hns+DiD97LDn3IdIiKkzB0HBEOw+aDm0hT59E6eYnHZMrS0OhiUyY86XfPSrIe1eFbn6YhcojHtxP3j\\\\\\\/q2w3QAqQ7oUpL6lDoJzgnXvndW3716C2XmRNBUV910pKJU6U6iyTLG3E0VZs4wJUWefDycCifnTHx0O\\\\\\\/cKMMwBIAaYXfcJvcbOmZ4ehVaVr4KfWHWsDnMT62aH8Mgxgp4KlLkR46jYyhR45pNm+azaZYpX65XQA3nD\\\\\\\/INWlrxJduOxmfafDoSHfO31BoChsxwlI+N0a22md9k9QUwsyEuEqNXGBymJwXNicE\\\\\\\/ByM3bTNSFzG8+hoQzd3U8OqJVBjoJXKxEjf+gLwLXNeyWe2pgB8CT7ZTfYv98mfFDntBJ7j5ODTvgFmiZz1E30YMa4BVH8J+Moza0yRsKgtNQ2nj8AL8LfB9yWWcdcGl9wXskaXRUwgeZFDohOPdrwa13JWPQDCry3ZkR0FIZbX5fdpngT0e5KcpUqabxHHVkdvWrsKRYOC9O4A+AZcj1wLmU9\\\\\\\/Dh9\\\\\\\/7FyM3rAzAu5E9YvosCAsNhAttve4EXSa1GkVS39TcRZKSZnqmkd9W20YZcyEwu9vJEeY8rji1STADmTZzJFxYQua10VL8bqEBwd56bdceW5AwxGitSdJgM+3BuDQ8W9MX0NqVVT601A2jGH0xTptCTEh4mNCwj9PT6pjVC9rX1X5p5XIRITA1k4PJaZHxwELGu\\\\\\\/Qd3XC3et8YNIbWzbcxgJofiHgPpXI3wTqM7wph+I1pszOGdK+ueySN1Bs\\\\\\\/gADw4JrIzaaWqwN7mA9Cu22h\\\\\\\/mlAO+VCYzJ3Jn5aOUYWnolUbjvMNh8iPCCoPdrAWqXo0R8raBCmg38bGCfzkJPDBcU8l\\\\\\\/i99xQ8IkFRf2etnekTczGT\\\\\\\/3ujKE9TsKE2avLC\\\\\\\/nYC2e7TbqinfBgKtx179RLIcPteQ8LAkCAu4kRbGkePrSHHk5sM+neJ0sOYvA2WNKVo22PSgZIt8TOcDVCvICxjiWRvNCzSznhoLUe3isNYbyxE84mnWvXU+4bQRSN0X9TelOUkkE\\\\\\\/MQ3x9uuojcadjZCtIJBL\\\\\\\/UkiPqm3Od2d2Q0JPQtxmtxw\\\\\\\/3SAk+j6Ep3iUEtiGkvN\\\\\\\/Sv8ux7D41E1RdDUo8iI5DgsafGrqjTY+xPkSYZzSZTrDPQj6vgNpOm\\\\\\\/14Jw9\\\\\\\/iYZlbMzWDghq13aReG40CLd4Kal1WibAqYLoJb+u\\\\\\\/yEVehz+Ss0f033IjMjmEWv7WQCEuIMzkvDecUmbOcmClXkVfh0+8F0X68Iej1YJZjIzX10MwcOmGtMKP+R6Ke+XNmmPZcgOeNSlWTETqT6lxxedhEl\\\\\\\/dhL8pUr++4JNaZMQKUl5LMjfAVVaaqA9qFm6ysme7YO9c2jo0yLV85FFt\\\\\\\/LXLuw+8Ox3joN1n1byLSvVWVwNSXOFrQLkOjLjl1wp9d0S31Jedn1IC3TwIQLYgzDm83vt27iHqP2UGlJnH5qjYLurR8PPp+cGSltnhzQKK\\\\\\\/9Wm3faCBl2ul35gP9mJj+HYeeyOhL5gn+IYHW6C3vNO9r328+mXmxlYjspOJDG7EBI9AdrJrGOZpqqbKM\\\\\\\/3CB60MVqNN7YASYkOCri4GHfYKacb\\\\\\\/40PeZFDb28LdCih0q\\\\\\\/RXlfnE5d8SYIU1XWBLhFrOo7CmvRhGaSUsX5zfwKMM\\\\\\\/OLPYj3G7ib2Z6nYX+UnatmYnt9DEDqljbYIFBlAVOmRQ7yssYTUn59GfFy+luIG2+fuavVFehji5yT2bLxK5mX6KU7+MPMMLFw02SyZ7X6qyqPzVGFglR\\\\\\\/IeigEBNalrMUDqdSAJ4PMXjZR4WfrCw3FAd25ob1cWX2PO13OFtCJTAv7rwlDed+IfOXPrc6FGgUehRhgps5trGFWLcVoMv\\\\\\\/\\\\\\\/4bQkTxFdDHljvOFN1WnUhNfoxoZklKtE+7c5pdFh6YxMw9\\\\\\\/BiDMlJUFbTcEcrdKuJELAB8BPfV3nTY\\\\\\\/kL0VZmeYZpS5IKYH9TtAiPL79FWLio5EJFa5dC\\\\\\\/G5vpfG7YWogyOBTmZ7X0Lk\\\\\\\/DY7xXbeIJkClPtkpI\\\\\\\/LXbg8I1wXW+0D7ROJmHK8YhQjK9QHF7buh0hQs6phyHVRrexp6XcorJGz5kLSmZu+h0eIYVWK95obVsVatT3ylDNQANfjpZuwpaXkqvv57VraVxuqkpOUIXij7UY1usTnFRs0zZL07NkaeJybUkdzmlSAmVIQl+LzJqyUpSQc7JsXVgWm6bMQViY7k8pq64N3rkc2LLpUvIIV93XLGtQW8utOS8PxM3cOpLADMu5pWj9UgKOygeNLQShF57uBruBJliX1KJdroHaoFVcuSfKnlN3qWjDv603jlubeOMFOMi4et9iHrUdiXaFaTAKUVA2DXbTu+b48FuuNypfL9Ja67Ce3T7KhpKTel+0ETwLSG6LqjQJUa0yuqg\\\\\\\/0ZwK7oCoqUEuuX3X4Z2qZ3bkYv+iS\\\\\\\/51dz58OwIPRbWUWC0rp5337\\\\\\\/yQ\\\\\\\/nSJqQMl561QT2gQTRzJ4svvMfSlU97oa2X9b3uIAE9K7SAdMkc+P+gmqKQ8JGlXJh22op8Dhph1CCgbJl6iGT536HyydfcJdzkR3c5vRTtXvFrCXEcGvS75WQiz7SezPMg48T9Ez+91jw07NEPb7AJ4mWMJFa5S1yz0e9pgGoM47AFS7ny66NyG3IQD38yZkfP3RGFKjCjOBNpX9WT09Bf00+C0fh\\\\\\\/13UZgPmVnOiMTWTWqQwbXP0ggG3L4uComT4PFmJEFBWRZRQ+w1qxBSNheAAmNW8VkXmqyTOTYTkbD5bbqtCma+Z0S0i6qtK8X\\\\\\\/ukZEqsFeRGNu7k1I68Bild8XHxH+hVFiBgHMwMRfX7EuzTe3EGX2Dl8mZh+a9vdtx8qzxuh27XzqQEGiTNQHo\\\\\\\/XkmMN7+fcejWrCShupgnkyo0atdWu5iZbO4S4g9ML0lEE0lH2w2Jkw+9LeVeRZdiOjOF5LL7FvjOhwgZh+fwtCdSdqhKtYYS\\\\\\\/B572ub9bLBoRy0IceZ9AizAKyBLmNVLQtehnX3bGn7Ctadmcdfn+eFuU3YLqLmG1+iXw9m3Fb7Wx7\\\\\\\/YK8wj4wjr+nkHrzpmVq1C\\\\\\\/CfCGJiQ+6m67mN8xx3KEjUe7b+5+z+XjYYTqBGkQUsS7vRmFZ0C4pQ4eI2PPjAMoGVWjOgS1IVzfzvig6pBpBlgvFktgsG3t24Y\\\\\\\/cnVyIo+A2Xgzpw4z8G4k9DZqet3gE0G3PaEYxhfbVTTqCAD9jJ40vL4Jgm1TnTcEXt3a\\\\\\\/oMMqoBfJDn6WvFX1otseAK9Nre9JJHpTJ1FV9qgP\\\\\\\/egXT5xnBtjE5d7N0kIRJY\\\\\\\/5J2uFX5xVaPw9mptxOLYezIibtjt7YzZRHpnPQxHbZxLs8SCEE1guumjxrTUcW2BfrPUG7OtqUd7ir7I5MNfUi8jYoShtaU9vKnTx9GSnQwo3grf0hwPA5\\\\\\\/1WjunxV26TuCudKyEa8EEVytFBjQamEoJTAjKAXGPKenlsfGKTYZ+b7qSwmInfxtDpKV9SOnRk1V7wiHPYN7MFkB9C8Msani8uc9NMbZRtJEnf3QB2D1Z\\\\\\\/Q5GxuonMsAkiduv6nbYp5Bm0eyyxjbavey2udgv3yfsBPxmVjBN8fE2lNJLnJG3AEfH\\\\\\\/iU\\\\\\\/WgWZJR3Ys4LeobNss3Mud8pqQ46vWXFFcLwE6QIo6V348GRPFsUYdWfUHPMND49UzwQvRikcnuBCcHfN49PwRMBC0QE7FT8KplobLIwH1gnx0q4XdSRnEDdsh3LNjfz+MUxdx7Sd9fuAg\\\\\\\/nvfq8Gc6am10F1fsGiy7v5F\\\\\\\/1jygzuJjIjia2HogMzKbCx\\\\\\\/klZL\\\\\\\/xTuzpV6GTcXodcO2HZwoIiBhQTtyRI7H7oQy5aE0qglUM9DjVYEFAmx30ewjh544covXIkaOk7zNwUDONJDqZIMa4SI2VHC9ddEHh6fW+gtYPE1lKbq9imm+naf2i6wQ3UcZBpZePPd\\\\\\\/WMUmTcrCh5k6yrJeUm03gcKvGxkpr4pYWFLfObI5tub\\\\\\\/uaLPklSv0CgaelO7pZRXiYttYObkJd9WG6nxRivmD0hNL7lZzXPQs5mUvqM9knk3Cad2fjVzAQv\\\\\\\/8ln5Dt3\\\\\\\/opTj\\\\\\\/aiRSRwatp5ZQn276oo2Nnx+za+UMpOtfji4TgGysb9GEPe7BykfXPDliPGhU\\\\\\\/\\\\\\\/dbScMZTOV3tb5l\\\\\\\/0S0vzuATmQvsOp0cV0C6jPiNpBXYUApahcYywfMUsN4Cwk+ZlPaOnQkaVIZKPUozEEXIhXnmH9RL5o\\\\\\\/6\\\\\\\/Xf5mlZpS+2\\\\\\\/FZGN88unplmpcF1hOv\\\\\\\/yUWdmiIM+5ezX48QAXld3\\\\\\\/W1mm1hhkOkN6jcoGDNP4PzveF0OIKb+DvR7z+6GkxYS2c3RtzJ1g0JdBXBCif0dbPHNpmggdUgAxyIIUH7e2H5WZ1FpXklXGxqgU+Up16p\\\\\\\/9KMvUKIboJ5z0PvbKiGzZ6iMHCeNzJx7Q7LA7rfMIFvZJmQEnAv8HyChpjnWOWrfg7QguFPx1vcZLgq6ebJnnySJuBAPH3dODd1KU\\\\\\\/WT6fwUS\\\\\\\/3HFf7gANZ4nxqIHaQHRdluQBMe\\\\\\\/xt4SFj\\\\\\\/7IhfSzAZSQIOXkj2oChAWQ3kDna+VuAME3GTyefMCvW7dJ\\\\\\\/Ps26h8YXUqOd\\\\\\\/azx4jVvPPQyF21Oub7ChR5u3Ar7Wg3085GFCTOl0U25hp+MOw8fJUX7LiDpREhLOSKm1upMdkQQ65sDKHiI4kcKwd\\\\\\\/7opIbrHt7eMCYLLG+waSLbKBlVFzIr+\\\\\\\/smmu0M2zttpcLbH6UjivDIS3EjEwuauuveGf0u9SLR90VRGK7PfDS7k9gz1Smlz\\\\\\\/w8dZ20D7\\\\\\\/n4Ky+HhgJSK4Eh1i8ter8ko6KP73j\\\\\\\/r07YTZW\\\\\\\/O7zxyvAE\\\\\\\/ovSsu7s45Wr+DtP9W1xS2+pbmd\\\\\\\/Dpv613DxJ\\\\\\\/IRwRvM4hiTTz0r0VnjEuckLqhpS2lonJkbFftJaEzx9yOSMv\\\\\\\/GDZJrPfHm+J1b8IHjTZxLTQOby96PTNNRCUGcUFany\\\\\\\/UG7YSvv7bsr6131tZyOBZZYlqJN6m0I3G3Uip1WvrCRBPkwKcw7EWHdW3eaJMbgq3g4uiI\\\\\\\/MEsHalOePIg5w+1FZXixNVo8bRdJlGvpw+f\\\\\\\/Cf+AyUpIvH0gzcvc8ANk+uOL6TrHayS\\\\\\\/XqNHeguZnqE6Dy8eThkXBuqpui6xeiQAHErh8O7fOrSJlwkZk1pP0l8LCgqIKCqCRj9B6uRpnF7snhRrSPlesrjp4bPj8z6GDjIV9I1FOWsJY4SS2kRxo+pAo+xWtuw2wPkmCRn+Goa3CrOGMf8S0yTHqQ251ZhsT2h7idmxkTe2PxJ1RV\\\\\\\/jzbCPrv3KYLokF2NTbp8KW\\\\\\\/py6A4EZYQJ1t2KqgMtbHVH\\\\\\\/o7M052ApEo36LC9cW\\\\\\\/tWjbSN9PAygDWBd2kl5wubvmvOPJFQentDxBFnlJMxA5a20ffMmSsHm4FyWVYN\\\\\\\/BVDf9VluMQzbbdPXqky00Vo+7kbn4c9WnobF4Gj4jyU24UEw779jU4t8SKgYem20neE4k+veriG5h7O58DoNbj\\\\\\\/al3QqavgzMG0shElEQQLzOdEu4jlo9YijIDx7vegZXDAuE6aWQ4Z+VCw23c\\\\\\\/3I0KaqDo2\\\\\\\/OthBG4pC3AK+E0yTMVusofrBQMXKzg0VrSwAh8yL8KZ01mrdBEuOqEJ29UTCC+f5lfP105M5lZHyOaMOmQOpppewBajq3y3GuxWBXC5C5xysdZP9QN1pvFkMLCX86c1\\\\\\\/j7C8rph5jJ1ESS0IW+eG1+YRVLhAvqIJb+kkyWuoVgM33S9lOtmTLFb58fY6Wd0AAxvNpPDrKj5ndeib2sgyyfNnaVkPeqW9pDVU6tfp9IJCCrXFMzHx4\\\\\\\/qoEZmVjS3wkhur8lzCv1QXNwG75PvaNgTCWHunlHKp6YVGnh7AJMqZQsb+pAoBZ75LLtMxPt8wc1zb1E1OwbaRG+OMl\\\\\\\/EiYOaLcc\\\\\\\/7m+LSh4QWm1WpO9g3PCR1bscV56W5BkUZDi8ecpz70N3J\\\\\\\/bJevIA8gU1LkV5YBY1Sv9JK9XvHGSHyen4H2jJIpDOMz6WQhuO0AgV19dOww8gxDFiQ+kieEsOan7F\\\\\\\/WNQ7pu8eKFxKvTsFz+Gdpo0E0eB04rqLsqW1jRP4woSqJPosA1JmeD9g+V4tpzgWCmD588M90THI0fHgK6n0\\\\\\\/CFP+rUTdd2fG+BG6fclbBUjQwY0ErA5LvzD+xvOWGVo7BBR5K8tFUnG54EN0y3icJjtgUY10B66g6IkpHUiTRisxvXun6s1CWd8Tb6599xaEEtxa5L8Ux7Y6tXFLdF1yWCIvR2pJD+irSkrTxuFFEUw\\\\\\\/ARC4isEaGSDq0Gn2F5Cr1M\\\\\\\/QUzp9QC1p9agluWq76bPvJa0WBPvLxgxH29dub4kvH8w0N8qsVVlCKIA3Do572jXjGbLOHc24ZAEvCu3t9S5UiVRRlgbqWVzsrJ7Bk5tWNPj9hyjmiRdmDVu0sBFMM9mtYcPYswT4aVWmT\\\\\\\/\\\\\\\/Szr0cm+cixWiDQRRFCBQIl4o66FXBphs1ZEL5q82xvAtWwlEO2bAn+kzEUV\\\\\\\/fjLfsyKDpYbJpArPzv3lpx0pspQhrTs1mvZijYzkNT7r9a+Cmg4xFGvIEYIxDakSk2sW51qnnphzzPcnF7GbIm\\\\\\\/5nO6Q4ICcMcDpvY+5u2OamZsOl4tYb0rSfDWVr1SawQ1ZuAsx5mT\\\\\\\/Sop6HlI9tbYug2oxsph4vkx1hQv+T7sEiVbySrlURwecTdakZxnyI0ADsBH1I9GVqM8\\\\\\\/3Lw3iGyfQEDPiNXgGPPLhMMlKo+fXe96beJ1k55RZVKv0euNsukyaXDXzpJB7wHlbByuU6MIyyHbdV0HurC6YEhN228t7FkgsyQB+rkw6b0nBN2zxM8FuWdwLSK1iW+0RaT0Sw18QwqhziQkMapYNiNrxyTW4SBw7BKppfx6BbO20\\\\\\\/n3xRVRUwqMJ8troxpP7EtaWg3wuQVxfbxuOYYD3t3LcrpzKFPs0EdpVJ4sV7+1WYLJfP8q04g+3G6bqMXqZlnnVr8hc9+kMhjKK30PPnRzsqWDnJGzFG83yRml2q1jStZ16attMK+3ZSUU6AAQjJniXR36ab7JuA8OCHoAvO3OJUI43F74QBav6tOj9ya0pagDzRJSuyeQOeTgaGOIzsY1P\\\\\\\/+5VsAvk4x8l\\\\\\\/GuMZowbWDnTmDmcQWjC3giNut1pzarjXTa8j7K+MXhmgZzYyznKQ0hz7OyFBMBK4Bztk5eH7CMwqETbkM9A7YUJzy1fivbNF7o7dPqOcDD16IfWwu0oMyMD27fPbBZ5ABb33JFu2eSyJ2D8TYzqxdzThbpvvinrvQpWiWatO6+wrB0OwSLw8ooZQclGOMY7fn756UX8pQ6dD9If91Y1o6zekE5E6BOPabvRgCyZdqEdhnvIZwB4f+Dn1BqfNyhHnGK\\\\\\\/QQY1NpZU9ynxu52JQV2XdHJgCWgnrSLa3crKhU8PfiLkv7NpZ+WrFJywCtfiZVVKJZonEtKZmrfCELJZJ5+cUSAeFdHaKbm2\\\\\\\/SQjTlRp0pU5jHjnwTjozOVk+mD16OoTy5CnYOR4NMaLHyedmmeUXcdz0I2KYuQyVdyCc6ZlUANo9K\\\\\\\/hdBfGtZw7hwvkguxAKCq2AHciEzEQV9ITa8SP72+u18Bu\\\\\\\/YVmxw7aX4lz+ljobfzuZGFHlI6fl\\\\\\\/qgaEC1sTXwVu57DpWbi\\\\\\\/toM1RjMuJ1llHvfPshExrGnxl+LxXll4DAn7iIhnZGg3FuviQumyW5Lt7Xa47yLKiklh5iXaZ2AqqEMgQ2aeB+Kl4cS6\\\\\\\/vCrhoj+D2NR4W+xdMcj2yzNI+QX2w5Ig2RIskYAvGVZi1dQOJpPDp9YwfXhqdpiKdNyt8Z7OsTdlWJnzCDJtf3tG4vd4r4cZKhBIPlXiPqeWQvpg\\\\\\\/a2fCVipSSd\\\\\\\/H+uWucV92tfR0GLRu3C2J\\\\\\\/Gw5MqwIDUckjFv2eqSFkO6xZzbXQHAKcRdc578CkQDKzoc6RrQPi1uxM7Nc2JoeUxtDIXL7dzw+jsIbZ\\\\\\\/sGwggdMONjW0OiLrAUPgBt58iJk66VgpQ2Y45ayw6+nNgj+3CLFOb0mxPMVGspJndkJzThvWq3SMtHnYESUARGmMrS1\\\\\\\/QDmFZuo4W2+iIhPXfWWzDQqJ0RtdwSC4VAwCbm3hJkV5EV1ZUKoP0FX4qq25VoKCxuFbG4NtosIHFtCybChf1Xyc\\\\\\\/lzXqL9mte1uZaCbf595x5rFcoJMM5Qd7vcf+PxIKWXoraixr0GM4xcfghLRHND244EohOtpZGRtoFplB1mHwG1VKapueNv9rz60iu3vD8D1V\\\\\\\/2USnLQd+c+hfW5xuCVV93BqrKxlMNCp6mJXPGyckHPLZkWLUgAfG9uJYaiXoe1sduM9yZL3zE891eItjF2APOpRy9DTT3nNd3Tb20bIsFhkevFt8jAIvnpgtsth10swacD53W0ieegeUmqn+Hfc2GDirIGkhF87I+ebcIh\\\\\\\/ijAQm1a1m\\\\\\\/f3wswRtWXUcagk5ije7CBMbWDcFq4xhUNY6cvSo0OZHWjXkKJaEGAxAKjhHCyzR0Pc53kzWwp3mBm7iH8sbmj1iyQHrqZBz1dQ4jyc+54x4\\\\\\\/hdUrI1z5RY0Nt4UCyWcvCZZoOATh\\\\\\\/Et3e1Q\\\\\\\/CedWNxERPBXrw92+s+wrrbcOclg++qc7Kp3zUgxnWiUICw2snwsXXKEXNO0\\\\\\\/RYk4t0HnqrWzOeWqGvsFHkIf\\\\\\\/BkGmxiUy+oSbJLreCeOJ1SPVI9B9b4ecp1lyYl9uFhpQdWJqeeVbC7DoHXAlukCchlqlTSAsoNx2Y49mrbocIaM0fjBmWp8VZSSoXnFh7yx5OnZzeKjd4zPrDwmYtN6MmKWktf91vrLOlwdrAA0BJkUCYaUCX\\\\\\\/m5AJ6Oxa9F2qtIvZqUKxm8Aq\\\\\\\/L9xSrBOokVaH70EPaAVyEPrcyRx6DF46RECFz8u4pEqVJvAGbB92YzpHBcaSdD5EhU6+H17hcvAMJyS+4BNOS2AAP2jMzqrxQfS\\\\\\\/adanFKfVnbM2YId5w3aTDuiNufY\\\\\\\/V3G9cXRcxwh0W7CHEROMSzH5H3Jur8LkPMDkPwMnsBp8hrSQwCLMOu7SFWjXBntYmaNpkgLe6RHFo6mj+iJYsPIY1y7LwhsiS2p7UvUq4R3sZ7Tjf5l8\\\\\\\/Sunx0oq245tfWDsbxA42E+XongaNxdFeHRwFfB4xZsPzaiVFM0ZeHyRY6CLiD\\\\\\\/fM41IS3CpW8J0oCigq2LsD9TfKdTcdpZKgGVSkUX4LA4tx7tALmNx5fGs8uODhvov25YR67e+ejugbdy1oPtk51YUywMZx3TMCAdd63J3b5LIkN8MrwQbgSvgyndv7Mfyt+msSOTVjJI0R9TStg6WukysI3y+Kdqrmg7BFucubg5aymRlG\\\\\\\/0qNmOjVCkdCX2PjGiOWkhlLeGWjvdakcIOGDzkSh+y9mTM\\\\\\\/7tzTuGvQEFLex6XhxGHMzNLWTtojHAJDBVba2cGzrqdrUl5WbXdw7BCPqu0U3S38IEJzQNFjKZUJJeyW0TsVyTSwDU0pwxH8UE+NltLZgeyH\\\\\\\/kfzr9LPiAy+iH\\\\\\\/+O72G+L+9PsCxrpMZAzfMpG0y7H7yEIf72UPlkiKwch4FB4vMNEdym2g7PwYy2bEugkG0bkMcsygDkd0Ap2jd2g1SQN3FnHD9\\\\\\\/hWPKvWIM\\\\\\\/y71KaU6D7mwDIkbNVypIbgeLWikXqet0lO8QvFrdH4Df\\\\\\\/mEj3m0ZSMTT8BCYAv+WDSLTJPYYP5fWtjKf7ysMVg6zL\\\\\\\/\\\\\\\/HslXJ0jAKcRVDV1mhpZYp5Twbae55Mc4j70HfiBEo0SJjLcKpIWfS1xGzj4lyyRkRlCrZUvzSQw5BfpJdTxH3WoxKHNv6lZzEkSkQBF50XPMGpuWnydAvfhtt4xu1hVHsnBzYYqul6Svp7cEGe\\\\\\\/0sILitAOnUoUY\\\\\\\/dNcQZHCvCiRpXsOOulLFoGP2DSDE4kioS138ELx2OJdkb9C\\\\\\\/GKt4pkDyMSCAEp4j5xzkejOZFK13JMk4b72lC6ksawOEVnhkGwYXRFfzXjBu0TiKeEgs2YkUXwgb5xJ+CEyByh+GqT3l9JGLS1q\\\\\\\/yTtqI+Hh2ca397vB2wcKZ4fJOo9yjnjIsfjHeGh\\\\\\\/H42elJON4Q5zUZgVComxVLJIlNErgbI9RoH5gJ0DqyMYLJVEz7lTtiRpfMg4\\\\\\\/GqujwiShk\\\\\\\/8ZKxNsGbzVsdTwgmEFcTYlbnZtUFAqCmdwD1kFQIxAsLr0\\\\\\\/q5q9xnQPPVbmarcKb\\\\\\\/GPBGtngaNtOQFb+VeGzGPdS3wEOcSadriFPm3uvGIKxKYqndbnet9C88KsFqC5B5bDzWRVowzeiQGoc35fy+7K\\\\\\\/uHRjWXXuP6aU+ODx6VYrhJoocVBj+DebGGQs+29TAzLrfJWSH5o4U4r+H\\\\\\\/V35FjZpJ\\\\\\\/lzzskLi4T9osfWuP79XAe9xK\\\\\\\/YNn3sEbqYQRuPTfO0hMAIWV7kWsiApujRYkfrnW9UDRt5CfERWPEu9qzNE3wyJ70IAwPP40NDoyNU+8bJVcPICLUwOPAZHodx8AOh4lRdB7V+ULo+nYIB1WGtcLjhyT4NdFM0qkSDfOkFHJxppdM6pTzuf22okoJEmzpPJnqa7wyQdk3QFDvve6MSAcVm3CBRzJFd\\\\\\\/s6zjYOdV213Vj6XKhVkV4mzACLua\\\\\\\/Tcx1iyf2u07i\\\\\\\/dgrJ2dVhka2GfPnAdfOBJtviFFMt7Brk\\\\\\\/dVYsuTpiVYDBV8olcLFmQHf834zr6IVB6VIsbvQmPAERrzW65iiZzQvGti5D+OmhdJni83KciF5uP6dcupEMJd9v8ftBEjkWPe9gKG0iJN5HUdyLLr0WwcrgOn9E5JpqkuCDRY8+1xWwTdkDCQCG9iQRJweGon3Sp6a7YgrTWk1LejUCr7GDrULV2sLOqrfF1dnnsD54JE1wZ8X2I\\\\\\\/CkI1jswwPL3ZFUeyLuLTv36\\\\\\\/AUfndssKz9fMQ3QmU\\\\\\\/hF5rOaGQeYiv4TzLsogcBz\\\\\\\/C+oaJ4fg0\\\\\\\/bcrAQ\\\\\\\/idqQRD4vhGd7urBnia7qIOzOX7cpwDuvPh8yTSrKl\\\\\\\/VMrCj1DtunKdeOMGsjDdHbmgn4E8FmT2Qg4SkyX3cDCH1jBfgp7xm\\\\\\\/6lihzrwb+8GzCvYBKgdXFDzAGJH4fQ\\\\\\\/kpDIwMOIv10WhJF47x4eG5XM+i7xKkV2mChBVQrBMtX4NCEyF2YtSeUfU8jbnyf1nwrKz+2NUMZrWTPwTuaV9M+UVoNnZ4OXz4e08Bx5xh1ejw6gHD73D7c5kObObCQeEkfr77u6EGzqLa4x2B5KpwRNJHv0YEesDrEgVKlqCkmLI8cHErV4hOVMePPMoh8p+2Vh3XjYvZxihk31KbhmFRzd3aUqXw7U1uGKkooSMgu+zzDozSMbel7Tw06B2EhmTNCWPmDaCOSN+1Gi29+ZwhAItSAGTzpzxhKD2tmL+cROWH26WkcEn4zlaDiQtPMx4arUz7GCnUsTvKMiZdB5frlnmJDmb8nXS5yp8VW478oW6LWhMJbhv1qoypvcieiP5GuCqKr898aE+JJTIhBYgHg5y99bM0INcKL+lv5XTeSCIOdkyc\\\\\\\/TEUSPxuTMBQgj7UzLVsiDUz09EC7TncpJ8bwHOEPxqUg0+DoIay4P7vGW60Tgr0ojS3p3cIJLiSPTrtF61AN32kZHFTp6gcjH\\\\\\\/2jDM7wKp0eZPGHjMDo98fnSYOAn5cCT2moOVsqKm+VM245vsq9gykMr6cMX0tm0puilampp+s6yLgRdJNH6BOMF9jw1t7C6HODUg0GsAB8N8TPUfpU3ca3\\\\\\\/+wX+uj5MCwGax1WHHqMe2euNoM1oz63I1TU2x4czN5JXIiTB8M97R\\\\\\\/gOdvDXLMVIQHHbYwNLUPaAyivHpaqq5J9rey9mNB25nUjS0vR5WZKdZAJRKvGXIvqNWTxNmNbLEcNuB9Zb0dCxtNh6CQS+XmVtf5aS0+j0MTcdvTgg0sugAKhwEt9V5SPhroRYfuPcJmxhYrkeQibaUVpOlBj6nC6WYR7cd0JuxnOzZeJnqH+aluIlc1pjV26xoORBAK++JyF5aGVhXOSN+HMYxE0GaSgygEbTL5Z2xK6wS7Ml53ejFwjuUPfxXmEr1K4P0aUnBuKbSH\\\\\\\/jmItdW7gkUxYv24Sf+7RR2cPXu5zSeVvPZ\\\\\\\/lzXdoKnrRxTWXTKTBd7+7VA1zSSprpG4wiO4EI+EFJfDufOwT306Kvylo8iAFtcp2R5EjfsFL9hNLMFunjsCrjim9BoiFbomu4Iz\\\\\\\/THLQJIi+Kiz8epU21rjoAlLC0R03mbQxWy3wCwX9PqPn\\\\\\\/H11nCunCl0V52LB6Qcj+JesiLDvj57vmbLpGkIetc5EnLzOiCvKC9zoEyGuf8VBKxFokLRZwgB4LA2zMhOiT7KoBaOhU9UmfXnwL5rEBi9ARSB9t8Anulh4u3Z0vLQf6FKgt0xOES2AmmCjYPslE1jXeL+j7vwBkg6NMKKSMscvThpUkOFHDT38O0KJ4Y6+W0143XCh7VkfgyhnNgr6L1Z39CNNiTZIpPDRcvsH3tTWf76neawXDVbD0pdJvvj5mNPZWn2h2kkWmrjncl31zKZFgJ8gJX1\\\\\\\/\\\\\\\/yEX36n6sOvHasvYc\\\\\\\/Q34r9V06R1CQvrMieU4r09Q+zQpVHEZ0yCjadF2yibZ02eK668CXJcbLzV4jJVBVTIdEuKK7gH2fQdOozOJKGssOGMTNCxxr+9DJVcfgpdcSpod1P3nml7593Pocn010vBxL+MyreXY\\\\\\\/\\\\\\\/Zo+43AMtgwTy5wJPm2tJFVzyZnDHmfL41Sez4tDFE7fPw1qXWQItz1aADiE5OKoRNe\\\\\\\/N3tURBrgjF1oFuyJnaueShIL+WE6Ikm1RaT6hLjfFhkrI3GUo3epoj0VQmjZgWTgcGANrIG4cHbICeISz0edtVTq4a2wWnrT2aXFXclr2M7H7ys1uUpIkVqlWRjbF9IG2vZvwCYO3UV2eh\\\\\\\/cUAOGQ2Mb8XLtdeYtHW6U54aovtfFRVBqcGU6UH8nHForwuUuvXre5Fg5GBjkv8\\\\\\\/ho2VzIJktbO01HftadtyMrNd8ux14ear7ufjvz5CihljzzcikJfqXN+2LzaeD3ImBNhLGFR9xaD743jfMQf5TR\\\\\\\/myP5jzEQtHfdHWtDuzK4BtLItJp74hDL0ufY+zmzkoqtl1S9VoBCN+OKq+kq2if\\\\\\\/oYPvRHv+\\\\\\\/s4cWj64nclCfnt19CfJxBJqKjNO7XeEoHXgvW+24m49OB04NV1KlHYenRMyH+oHyIJoEptFpGiNVhKCY2Fy5gAVLjxGK2u+NxPPpsyEZ2OifdR3to9FyGkeh0+DT5vR0DQfm5BK7qzVYvzFvMd+ZgIcVXfgKM1GUrsxyrDyvLz0mn237DdoPixivj1fchQDApzIOpQeuRHr0AwO3fYBmA1hMxv+eAYCS5cTICeeUiYlqi96PZ4qJ2AalMj6hq5OKk5MRvz5n7+HS96BJHN9NdDDSws3qkbIG2g+La0x4glJt1Z0YXz3GErMMUSGEs\\\\\\\/zhbV5hs1IpoJ0bJwL3G1uX878SWutsTWBBmqA\\\\\\\/sxRLkZB4cYe\\\\\\\/8ZM6tltgebTl\\\\\\\/3SaUNxoiojS2inCjf\\\\\\\/bn0CPH5eYqBBXtFNJ6UFJZDocqJCGgqiXnYWka2MVLceSrCYj9ZvgUSXl+k13S8dj2+hLqkmLpRpz8OQdM1lrfedyr9wh6dE\\\\\\\/s4mn5QoQpdTcwdCqkIRyAiN+3CUr0cgWjPwWfXKwUWR3kiEv0WnBi64onexituiOF\\\\\\\/M9z0cC0XkmZiFGe9n9FaC032OV58yPKAoUkvV5MFDosNNb\\\\\\\/Tw5of1q03SyYfz9DFPJr2lHZnmB0OkinlNBGbUr8PPdw2Z0n++iTnWyf7LvpIThYyv7hk7ADnxT6MugalxHkTPt1PSy7HiOeXSRkNhcaishXDwfszaKJiDz4AIkN2UOS6fqPEXSv7rEB7LlaTms1OPjKuc\\\\\\\/Wcou7puei4rLviHdS7sNfkxTUofH2L8jVj5r4JJkKOCRxjfzkEJa1Lff9UOyeAJv\\\\\\\/iSPl25uHsLQTKDu9AW1ckq7ONQ1iYmv1LPbB7wPFeIcTTRV8Ebe5HPKacGilnPtTNWftr3\\\\\\\/HeapGcJWJujtktcsJ3sozJNlvQvqJ2veFn2Y+uXkHV494DHXqylGZzL8Aq\\\\\\\/AimpZG4HZsio0ia5D+V89UoNQdbPVGfJLOaa3McdgLgBHN979DFiPvgTFalbPCv0+wz2TkNv+JMKY9+EDjCkhtoU3rdJdyT+n0dGJP3\\\\\\\/pBdNp1YraS92ePUiRuSmAwGw2EUEkNwNp4dkwHwdfK0pWruhV9rJsvnWi9uFXIzkaBz4KXAVdbxjkfxX0oHlCbwrKZCHpE21pEp2bDAOwd3AIXYkUbHDYG7B3clEyvHQTffyvZrXDWK0iQhcNLHfiRLRG7dGkFRiZePpXgcsBe2g\\\\\\\/ly\\\\\\\/7P0tnkG3hz6GV+Wxl2qteQcr\\\\\\\/1QShgKTb5HNUTgBpBycWE9kDdqX0tAdLbMRkF66ymoNlXe2cbSNjS61rosNRqHs\\\\\\\/zi1rHKeN6ptq3NwKE1n894hdyobdQIigo+jb3nOKiV\\\\\\\/hHpmIDxO\\\\\\\/EG4bCgKo1lxscRRFn1Pt6DqbOQjjp1spKfKDlssD7fjiyP5o2oya36BFX5gl4X1yEslVBPogp4tKKeJeopXbAEOJQCq0jNEjaedxduOutCEP7jLqiVkyxarO0FzE\\\\\\\/zv4CC1LuDO9fD3JuJqUqr4snmVrFeah++27uDQVgEATO9zvA6aSnF5ZUCJyEAb6AKGDWO1rwmvC00WqIyPl8NIKblYmkJbiTrnEvRZRLucYuDfORIRlLHChG65ad0p88PHkRPq2BcDnYhrxvrZ6jvF21xMIPfPp\\\\\\\/akQP32DakLbmQ41sXIuFqY64XnX8nJgn86ryHVsjVutDVQ9xmhgs+pKfukKHjBHh3zIk3\\\\\\\/bpkMT8wcijbaLM89lJX+BGXEzfU2O1gZF73Eg9hUEW3H\\\\\\\/Hv7yCXh64xwtQqU+d5ybpWEPyU41UEIZls\\\\\\\/1x1A+uwbgx\\\\\\\/Xuay\\\\\\\/qNMNp6An1j\\\\\\\/KISe405nILxKAsRgbhIAsf7mlkPVjffM86pPJGMiPm9uM49hbthfIQX3RO2HQ+hJXzqVlS\\\\\\\/qaEJVmGsKPALZiHNmj7E31s038RFbVPjOhD2wE0KO1S6yW3y+B4AfoYK3xOqMBCyfycqIJ6RErllWdH8ByGCtq7yQXuOlYBrNf\\\\\\\/3nPgMGjBc0L7cLx+zfz892u4tBrzmxuiZzmTWmhPn3l+lQiVhKD46ofxtkN8Sl9mwupDaNx1AhamoojguYDccexKPmoXZk\\\\\\\/Nz3nPf3hjHzTOFUhPT3jYQTZ8V1CVo\\\\\\\/EzbILxAvPJ1m35GXyPxlzG4+jtdgkF6eHkpmXrSxnAAja8Jmf0\\\\\\\/txacNfHPOyiHX1qEs7p5lZaY7DvU7PfBQuFxrdW1OEsCGbFmYBlD1XOkkFGn+HylkRBcU\\\\\\\/PGgSsm2L1sSH4eZ\\\\\\\/zzymR8b2nQUOxtXRVru573q4DwDOl1KFn7bxRs3fKYlVZYaUEP+ELElBeQ5YE9hOgFWasu\\\\\\\/iPzI4fDoAu59PS5i0xOhic50NauvbvZElujHV+7Fubx4mnPWsAUoG8iGu0\\\\\\\/58q+PrNZfzMLRbHNlto0F50eKZRmAuMIwfYahdErzxyR0pAYVLMnrbpDJhAEZVY68JeFI+Vu32YYiTcLlWrTHJWzze6h+vzlannA3TAEJJb8iOp9b3GS03ZoEWo0VvPbHcbpukbxwaJQR2Tl29Yxydq3khci2AJ0gpDGL9nXY7AxxGxZ+lucErf+YOchaKASPJR2u7fxlbcMUpRR1uSTQkP0mMKz\\\\\\\/iQqi2zR\\\\\\\/\\\\\\\/jxxk71TAm2eChZQRGXzS1+6sCikzjvV2zFTzdB1lEmd2z2UAxYO3MStVeg7bAI6FSP8y78E68+BNEu2dJtMqbyJvQUWUget0YRKRm3mZcPX\\\\\\\/BvKcfdwWsX8XlvLkoqwP3uqL9ILuY+RPSzNwWC981cv\\\\\\\/QVR423GT\\\\\\\/ib9FNq1pP3L8Dw7ymF3yEzbVqMrbEqQgPW3A1KqvypMToMn9sNVuRVaBQe\\\\\\\/\\\\\\\/t0S082t0EbXCiKH6519RC7BYrpMiKiJQjPWoJxHslAo2E1o\\\\\\\/Zj\\\\\\\/GbAlEyyMBlUnJuaKSnF4XyU6TWNnKIk\\\\\\\/o7BOZhD\\\\\\\/TsS3ed7FI54EjutFMJMGD2C6PEe96pIkJgrl+7NOx5tkO4J3FWLE1OAfTJYWNWHlB7Q44GYMuLf4+4EJzBuCcYhgX6oQ5dpCEZt43PL+fZOqoRFTC63T6zutTaJ5rvyBUubapks4XTykdGRRFptkH1pFf\\\\\\\/aeY+nBqatrurMZp470SqABDoWkr0ypqHLu92zfs4gyCiKcm4ULnocF\\\\\\\/imUrbVNRcendsPNyCwv6YEUrrv1hJe8JxIY9MFCagg1Ry0kaW0\\\\\\\/O0mpm5r3kIB075jU0HsKEG5IM1GDnnsVIKRQlGBbh1Rnv\\\\\\\/AewHukFIWJ\\\\\\\/LLUD1q99NI18PPVU2mUky05XaNVk9okcv47boVdDZmq47+eTg0uzKVbpZ54VKS6dVphjojE+PjTP4Dqt8\\\\\\\/IiB4RmOCVFeF72Qr2gES\\\\\\\/5XUdpN4WFdAa41J9T6ohAa0BqYYXcbATekT6D3VbAgDpG8jK5rgwzrlK8hg9KHiO3Ny4P\\\\\\\/VYSK3a6v9xcMge8w0sOvPRUyPinX1NhzdCiZkEACo1UQp3kxW9R31JPlbPQCEv1ptyeTIxbQHegdJC094iFlg\\\\\\\/3CNrTBnNYZjt1tYPQXRl3PgGeLOLPmonsAZGfRcNg4uiT2yOJCidRV6Gm8n5WZ3Hk2RvkaCupaCTarO2K\\\\\\\/3+iTEHsRBpKxb334sOEt3nFY43fIkLmhius60jHhpIlbvl5m9SLIw369xlrGe9ceZmU2Kz015TNbnTu98fnrGoLRPypdk0JbF1LAdTKsCxlE7Bxc7N7OJkk4XYrc6AJw9gsJyZfUPD9ngITT8tS7WjlKyX9IP+\\\\\\\/EdJmrwNCHYzkrLSQ8g6u2BQ5sbMvjTPascBfzk2IGePgFDS+qhwvTQSLlsCOP7cUDGd0gOThkzwB0+tJwSqloQdKc3gXYvfgi0kltI36E1hsbLQIDPpap4LxrDohakBsC8Ln\\\\\\\/bNgPtkMt5IV9RAvlVJ6U7i1BL3GatwuoT5QxT3YmIO\\\\\\\/FXUWUEShOIhuU2HRjwhXRx5KgKDPjx2g2Yf+buYm9pG2I2aSPHuxMaKIiAfRddwtw460UfR6Z\\\\\\\/JQtvX5glFKfjO628MO3JBEBdI9N6pyQOKDeDM3mQOcifMCqlnsBJUxRlRdZUco+K6RtydD56kgilyYuwaY1lDrddFEkX9c4tcyPH4gD6DKRwTXcUJ\\\\\\\/Rj\\\\\\\/AJLjlM\\\\\\\/\\\\\\\/LQOWY7jwIcYTt53Brfo\\\\\\\/zJZpEemDihftRcJchUCAe6FY+56rUkxYphEDkmoy5H2QFHdkNOR\\\\\\\/nxccVulnLd+tDbI\\\\\\\/F+SYered1Sy5CDzizhexK\\\\\\\/UMgg2z2PajVmvILht3OOFPUtrBrSxmqruy7GzBJDRSt3hP9CPn64SKHD5JCJ79giSnsRviIlt\\\\\\\/e6RyTClvNjfiN5PebyeoStsACHHHTViBmFzxSH1r+3EY1CHe0K+tvcZiOEydgXPer1Reg8HgwFj53dFo43jCauQefVtcNgh8HrvzmBlLCpfzezfrGCYT\\\\\\\/R94k6PMTu9O9sdwsIEBzi\\\\\\\/ZrhXgVV7pCEuaAFVvoLsVZ5o7NJ6m7PFiig9+xhvYZAtRlU\\\\\\\/ScA1fKJX0NQY3AycpC9Bd4LCUTXkBFkcFHoQvoPTgfetHnnEY7OIVc6OvP879HyKUTqK+fMJqqryiw9mExIWPnSdeb6ksYUjWgDo0K7SPo8IoYbI\\\\\\\/Lizx6Ib50abMwNkoavN8GSuoHwzD0l1WvLHee3RQikhxydzgz+geNRaWffGyHNRMOWZbVWkUkg3IPe4jTHwyPRJG1w1xow1bpifKHZVDOb6JsbUhAUp9yb5bvuZrQalRnGTMkk4npn4IZQx64+D85CwG1Sn7Y5AiNo7Km6QQrDn\\\\\\\/P\\\\\\\/DobUeUFAN8pWaT8zoz2gXThkcsIYQQSPmR+SEC6uTRQqeNlu+5uVgIMfSdcmUhAZbdvnhXyPO6dhZKfsWalTIrClgAgd6QjIiFGElyJ4mUgE6NP\\\\\\\/bj4m1qqqNcvY\\\\\\\/htLifwwFibdc2rpS8y\\\\\\\/vc4tOFEJq\\\\\\\/5f1Rby3+t03Vh+f6BiMyfDix257aJ6qXDyfWKPt5li61uP8jIXpR2e7bXwz2hzep0m3BeU6rbX0cHALg22xInFcwe0sYmbxlDqB9cu9Q5BpmghKiF2X0paFeFjMf7hzDTnONYMj1nKS7F3hbyhGiOAE3ryXRJyW+irdIw2YoV+yMIj4D4eQoJszPH5ndeYL6UDMEHcBERdbTOpMsNH30hjhil4gcd7qmpsxKCyVU7opieBYWWCt6TQb5O4T\\\\\\\/e4uGdXCWXK2FuwVmrK8ELzf2604LPq5vzAZxehhCnsX7wCrSQzTR1QZK3uE+WNhn4uTNUgFjtYymHXL8zB5CPhWcbrWyq2kCwKV2AZ79XwQiet9Fp\\\\\\\/+kkSPJ3JvLC6hjeEqdKU7FTj31U5f9ysBmI2xJ2wdn3enzzZPHdp0BdD8eRwsPAmkjxcWZcSPHgxYI25wL26gipoW5eBnv0D+A2DGVD905b1oV44x4QTomiJolXvRLbBqIKW\\\\\\\/BvrUW3GBNbqzArr7Ob62bEsb5USrlwqr4QrltuIOQTLF8wHWLx8TNkTn0zhRWrBHgL6lNVqHulWsb7xbqb\\\\\\\/sG\\\\\\\/Ulo+reXpn2rMEHhi1uWk\\\\\\\/0yp2Fjdg++GtToGf0KB8ewnxmcgPdn\\\\\\\/oEK1l8xQzIjQ5LxzaIn+Sd0NLI647oZ\\\\\\\/h5dvdGByha9kF7TY0ZPmNB8eu4dhVMY7IsReDb5fO9Nu6vZYllwx7rb3Vfb6zlaUu\\\\\\\/OpGDROn1QIKd729ucvtj5+wV45cCID8lZXMytXm1ekrfh7x0Pxs4DcdZewae2077DhSMxVlg2P9ENeNTPFKkTDodgDH24hupSJtkq+m2mjp3OnN1BQjSwfG2rkEfCMOtuKkCt5npzf5p47FO29gXr+Q07sLmVpFzEeobklbaXm2IjyZiNYtjPmKw+XUZXvb1rE4wCpLLp5fnYLXOgGS3xkvMBb+iyIX5acTHhTQ4hUP6vFf509DlgxGfyzbPbQbSyDdADAu0GmkB5nxH6XBY2q2ViTCjDQNalkL0OxRM\\\\\\\/xYqOLPKnkf85MzRiHPj1q\\\\\\\/jHZLI5P78X69l7R+KqIMRWHBXoRFgoiLm56Br6CMDGUDOsdm4g3PDcxm53rIRlkMX6D\\\\\\\/2fIkTPRWJw8UaNTAXP2rbT2KyhSFqFFa5iaFE4Nzts0+NCB\\\\\\\/uJR0GJCTc+P8Ofs9PzkKVlhXNa4S1Ii6URtPiGDTTMZZzTSC0W+N+At9tG1Ty69iLGRGxqVXDctGPluVpg82PmDTDQTrEvoIeplMwaWtH\\\\\\\/9q4qSfKQl+o92yjEdTSrrO5Eg+FbVEUhMWTUyIJ21rle3OOMivGcz+tDmOZ0CiBMYQzHPZZf\\\\\\\/HPw0Ro1mNVCiaWWl4igkLN6eexktWFIo+qvmtOkweOF5UqkSd9tDTUxxmpRIDzV7MxNGWEinl8Wq4OBCUNn0pHm4ujU6rJl8CRSBknzxKaJb04NRTxSKKOnPyRq52388uRc1Szl9AX84mJVZPd59xSCjbgc9Lc4+Jst1JelKgK4vdOogasKCzkGei5ScoPTeRY24hfeEzdc2bf1NW7DTymPQ\\\\\\\/eHK+OnF7NCWsW9CjYoVlKUT0wDGgRPusgces6f1JxE5xL3LrMUgZmHWa9CLT0B0DqKTU\\\\\\\/Qcaugb4bibB89UPiHO9vfZVvskDTs08rRhNJESWByyx52RkHi2RFQfxixS2ezpaawv9ORh0lZ3OdJisVH4d+c0GaOHLY5hnIlbFKl26JOMv5grDHTPKH7DQ3hKEi9DMJqHZzM5MgR4Hk9ONsbQAtsSOxM6WJfWZSm8kh+36VI\\\\\\\/FyEXtwSprypb4svlGrgyZ5TC2Plez9gRLGUemnE23VEU3en9qkyZZ8zk7RIXUTKHTvhjI\\\\\\\/zU3qtWjGcZqZriEHvGM9OBNX3zoTbiJT8F97rwtITKaqp0eOxgIt\\\\\\\/45W8EBrg7crQLntn5b\\\\\\\/iT0BrRKywAZB3a9U2X8H7ahxSoIFfououh98\\\\\\\/B5f8hpgsaVNE7DkD2vKUc2VLQE1u2zTkI1VFeSOHCOcRedLwSRymxgu\\\\\\\/QrXE+h+CS10y96U42k218hUPegFjGVpBi9LDn6h0VRjd4lb7vYC8mYKgbGs0Sadf3ycsVaG4nBhk+UCrm\\\\\\\/kYWXECwh3OBx5cjD34mH8\\\\\\\/yfVC+pbcR+CSLZdUfapG\\\\\\\/MvtiiUhU+GiJXMOD9J1sLQxisr1gadFeOHKmmgjzbt5zejnwg8HiUEYObHyZNY59rH19sAbwbaogLAx9\\\\\\\/FvhmiU0HQCEy4pVaNgiwBdwQ3ZEVsylNAe9TWRe7flCaBcZwV65hGzMkMsIL0HUO+wOXBrmXxdV\\\\\\\/3esZO\\\\\\\/WPC2q2dIpgNAFRDZS9dvkvFiDiuQ9FbOK4NflWBrXxANynagfMQLWVoxJ6e9DPPl92QzLq3BM0W1VI+xzTCC6mEEgwV21BTO7puBK60sNRXBxtL6bt4DvU0DqVAU2vkNMpaQ+T5T9IOthN9LzaQhmttbCvgthYS4SVI1KwEq0\\\\\\\/d8gcA\\\\\\\/Fu7hnPsvOeS\\\\\\\/\\\\\\\/6l3hq2oIXnQRR\\\\\\\/zGfauY9UoaR3YRK6au2\\\\\\\/XtRYtEXPaWlP4juFbFPUcHSAx+L5r36OkpxVvLzuK\\\\\\\/FkJWki3qJxLbTREmHz3kWCVC0z+huqRAiy20cnUZdBjfNci76KtZwBJIPhq1s20vuQH8UdPvTLrSAdyt2+1mIsbiYWMmtLn5TcgMTTBNS2SF63wfavQsYt2OFYkR+oTL08ra+HDufBPwPalsY0BXMAz6q\\\\\\\/GoM7N2aRchKF6B0lYGZcVkmXdJ0s4pnv9+LYN+RiKVVhe9Rf73iAlq8MRmcTqNOaiZPDRJhPMHZVx1wXtXae0UUN9ZGGtRpWLzNjJLPldPQczZ3t3JDP\\\\\\\/ZedrcxMZ85r\\\\\\\/CJ8I7cxBqhw1SEf3xfv\\\\\\\/KipJvweDWKkk0IZKw2zf7v0SzH8P3QXv0v2lagJJ7sLNPAwSaa+oqeuqTccct7X\\\\\\\/9iMYYBlfIR8GWkY9Vzz3O3zzQFHvciOUvsiYY2SlaMCRMYJeu4mwPOGvZz9dhmSjy2gBoDn7qFR+1fP8E0yY32+uXMKUbam7q7Lz5o8wL2ZHab7\\\\\\\/zIYHKhk\\\\\\\/M4RXgbHR1E9Nh3XCZt99f9morsmk477BC4YNqMqFHYjMJjEt84hKig98Kbh\\\\\\\/RfTAFQZjJwKO5KRgGm1d2N11BM8\\\\\\\/YN9soYX2b\\\\\\\/ZbiITLfea1\\\\\\\/s3vQv2Ot84FfAlkGX0M5JCVp\\\\\\\/pmLjfuWxBkQom\\\\\\\/4AB4kaASZbJs5wd6Qm50tVvmbnXl\\\\\\\/roXJg0jmnxWJYoh\\\\\\\/qFbx0qSOrW57bno05u0rkvOF0GHlSsSPQKil7NWO66BkcJRmeeeDhzJAlpJIKGA0kHFJT8a\\\\\\\/mW5B6GH\\\\\\\/fQn3cMcbKs3O0WlvvoKNJ98bp2rEpJbqBh7KRRF6agw4yt+3gmK0fy7XxfJFCZfGRqre\\\\\\\/tqF7NFl3iTkHfWaAxoTmrnAkq\\\\\\\/3FFAGBMiYI4LRPuncYe7ssX5ZObwcuLBY8u4DGA8mxufu43ZJfhI4HlHjihxxWRmksNrK8rrmtO3Pa\\\\\\\/aGkjrjKOTBlN14SyZFbuOmbq8U3Ggj3KM5NgLg1gismOLxe7lOXqSpWCK\\\\\\\/835eD448JkggoOK8qNIN4jVPiScWUoGioNrhEvt\\\\\\\/dpWseEHZUN5yxDdCPT1d0rZ4bS0kgHisO27V0WJihdtS9e+w1rdRLTXkwEpZl0GG9c67tXP3Q3AwhobRX4YLBMeM6k7gIk16S+mBhr5xLBqLNZOCUqihXOkjdSSNtUyp1iF\\\\\\\/q6zqS8cl6+ERYBCCXZsNn8MwLOOhIyB1lNhtIw7SuItfajFlQzN0pcpR6\\\\\\\/o8IppfSZaK7Svzypxks5IWsKy6qjwaZ6pluK\\\\\\\/\\\\\\\/HZnjKh0eoH8PfrFrQmudkpqmGAGDo5L+vQPO1N5FpyQ+nvG4TwTKuQm\\\\\\\/LoFInvhRVUMucbxUgvoiXoUjWrbUn7jUJXZ5TVeM8EFHF4kSySfdrDVq+fOgScimRiCDzxJXNWPAK6I66nGIhP+FYiTYktkyAp1VRAzoLcZOUNDHr1Wcn5z6MhfOq0GGrVlV5vrnnirOz6HkQZFM+RQNeVPwBW\\\\\\\/K1IrzywM1t1YPtMRsOolZmIrW68S3nRoRvrCTFu1YiAG9sbLuTga\\\\\\\/0\\\\\\\/D3Sfb+ynjKnWTMg6WkOiWMCJseQo8e586wsBmYlPzIOi4fbHLD+f6WGa5x0wWu8Q3vp4IVzZnNDHIG8cC2p0muxPrD\\\\\\\/hor80vJi2E75eKVw4kGzESy9f9i3HX1PrI\\\\\\\/ixqNAs\\\\\\\/pEGUIjhbCHzNQYMnqFBN+K7hYtVdYJsCYPD9jbU8Q8anhIQW8wjmsKtBpHksRYP5FM2V4WTlh7RgrRF7+uA2uA8\\\\\\\/VFxZhrqgV\\\\\\\/7bMx6HyA+6scxRECcum8SH1TxzutZiyyTrgm2ISLn07pxkFHtgWyR5ONPu1q2dxA9SDZBpIb20T2gtK+sfLPbS+3UX\\\\\\\/66vJB6b23l76FqXevDrL\\\\\\\/XMmOFdznohIvnj31jMXx9+PdTx2OJBZBbiDKG8pXP3S\\\\\\\/hNaxXN0Nbfb3pLxCsEkazuku9T9RrHuPAWU2Kjrugv+7z5FVSeV5HQe2IDuKp2w7OHND2J\\\\\\\/KxA\\\\\\\/f8VDjYIB2SWfWavdFUfAEvbBjuYwVLKoF3NALozLnvKEduAMgLq0e6M6rSvjXGac28cIJrAUMKn6a0+6fPb8QYBF6fJcB+TvLXgUf7UAug0hIXbH3O1v5YPrHpKCgoiSmI365SL+3zdnqdr7FffJ+\\\\\\\/+SncNdyqduLuMtmhnP3+iDOoCDSz11pfhKcU7H\\\\\\\/sdmCbfZfSwXveWF2wxxdB4hJ6tAaRliuQFVu\\\\\\\/qXVLIx4oVwZa1QqSQugQzSK1zF8kUzqQF4RH5XoxB+GBBS6lYTIcmfqJaUyzmRlLCDou1XgHfHhIEEtdl9N6xJLuj1DgRuOh2uOt3hgj8hE377jszslNncS+rshbf206erJPL6QkkOOqrgKH3aqhWYEVZB4rJkVx0SYflMN7PzfyYVLe+k4JkHB3QfZ0w4kQPi87Dc5wX6P\\\\\\\/AC1CLFbNsFReyJnidiN98tbQTcbtNz2mfI8lBCbWfkO\\\\\\\/zEgByvyWPkHGZjoAt7zXqIgplulNabLFX6UX3f3LFt3oo0XIfBD1XvRhFKGbm2v2Ek7UJNBnCqBAF7nUMtwkKcXgINZsw401QCAlDdXfqSqQuD5yeDipCrAxs6x8kyVmdC0GP\\\\\\\/QrViWmCaA\\\\\\\/+pt3Os06bXH\\\\\\\/ZRG2u8S+ClkAUC5y1ReVjhw28gVQ8WBFYOSlwO1X+eoh993TZW2vDtIDDFbPu36OIlBqq1oSPDQs4aG59xiL\\\\\\\/hMRiSH9zZybxyYl4gWDlutC+2JQKzgQSNOwAtQIBByBCFwtoosKClBXwQ8HWciA0jPE4hES0hs+CD+G4woG7I4rH0k5+wyUB\\\\\\\/4MB7U4H15X+KKmvKW4LhKG5A8qT0OR83e1bImn9MVDZ\\\\\\\/J9QaScyYTIOn392YMxxNqf+ufbx5rcZfUWNWJKHR3fWf4tCh+qxS4UlBWhaKyCkTO0eydD33iX4GrCi3rCcdzb+pwQu675IxMQYebhpQJ\\\\\\\/OTEIYJi58\\\u0022,\\\u0022iv\\\u0022:\\\u0022RJ2Rdj0iFJSB+peX61bdcw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c58226bd433e55b10700cd0b171ff9fd0c6d0b20f2f1ff60143357bcde877478\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Qpsz2bFf6SvAc3dxm+YGd0MyCaIQ0bSBrcG4RYliX7CVwHHMq5NYBXUPY+EGkhSBjEVHe\\\\\\\/iD2NCWm61hVUESIrlfIegxzQzUmlaWxTWUdn+5sOqBnAU6DJGXhL7p92Gim0zJLxyR\\\\\\\/ZDOP2Lyq2vWu3Oog4BjVFf4IrlO6sKu3raKg9ZG5S1y5gSCKdfBEFaSr957I78SGI9zHMP7PP\\\\\\\/k9oxXr4cI9m+kuJ8G+tXPCDwaK3zVSuwB83UuhpOnTp\\\\\\\/CcdOdYqbt80ZZJ781VQhB3\\\\\\\/1Qq+U7kjZ3XH5C+mpBcsqcuUFDsD05XRyvOMSUeJj3xlN0I65vOhcJdMGixVBjyD1INj4YgNoZZCGo3ook03JNqzqzxcqZHjoUa1He7FDn+NesxZqEFpL+i11cpRbbSWCqRXRglRUhwgo9tA0IVjUKgXjhtPlOA86zVr5ZSrZp6LnoKx2IPL8wy2qSIlJyReh4m2EyIjdOpDjWDqJBc5jU9o6FIJYcO1nBXjIlA+pAEzbksJr119PyaMABaDnw\\\\\\\/e6KTa3PnAUIjnt3D2xz03kwAUC6CRYGVJbtSKogfAcNy5Eh7h4qBN5XzTWkRN0GoCkBYiw0xPLZTznQ5vJsyZCR922UhqQSkZESyWy+ovY2wkNLYNMsHZ7cyc5\\\\\\\/2JpIVBATQf23W4YTEbAfkw1p4lTtRqnLuVceTuGjRpNd\\\\\\\/X1yJv1P3mLH+ZHad0LSWVdQ23BmDzwydNn1eQkeuE68fP\\\\\\\/1aUxQ1GAZtQRVffcVirLMNVnzfqR2gR9eTcfm76yblIusS7rJjPlUgEkv7f6QEgPj8SNAsTnlcHqgiB2x2hcjwmloRO6IrrVS\\\\\\\/IWrtLMzQNqjGEVAKRSDdV5KORfqPIjUPxgmJPvBm\\\\\\\/Jh6M2q9a9lWdnUpNGIT2mn5j3tjnBW21VPnxJsUKQFe4rJ6GylSzpxPWm2P8FKIj6fpUglxE8ToATaBx1Ay+QPj892pxIP9aWbq7iPdE2VdJwNmVKpEh2nFU1\\\\\\\/WI0A2C4aJw1+OG+f4hDYtQ\\\\\\\/S4I8rvszDCLgA2IHvuErcwLiisMGSgctx6D3HFu8fdzptdKODeweTuJllNoUwoz3q6XsALj6xjwGuwVFpgpMFW\\\\\\\/\\\\\\\/686eFbDqiD\\\\\\\/koAxb3+1p6ZQzByyjyJlBhANt7AXZd+or5scj2rkW8bEzbCyWT1ru8RyIWpaqNPsLde7ckNBIImzr4ayLm7JCEKG9\\\\\\\/3HksEDEKnBnhyG1\\\\\\\/rAQxcLXrOyxSlBEkbZvOt0XgpnmUTY3XXI8e78wqfsoznofT\\\\\\\/hITmHzJJcINdKFOQSyNT55c6Orflii09khjUfG6qzTZAknDHxyx9hUcU5P7fh84\\\\\\\/n2VxW\\\\\\\/rDSX9gx7WWN5jEplXQFthT4jmUezrUK+G0obMrNpu9u4senurTIGLOtelE3FujiH\\\\\\\/YDUAJJ94JaSvFSo\\\\\\\/jTEycL2cTy+FtP22TpWNBT6vlA9a8ZDNoHOE3p1A0CZ35DihP6qw90SKYsT8jw6ZoofGvU5CZeXVRVvKSlhFFtnoj7i\\\\\\\/VNeyRMaNL2azIORuC8zz06pKjXJb1eLMJEuo1AzsFIHwwuK9YWPH\\\\\\\/XIJQBF6GnPP\\\\\\\/awqK697YYcoK7Y8ecJTtPVVShAP8QOEz1XI1CCgVa7E9Ljv7+nlrB9ez\\\\\\\/b068h2rE+rFFBcrYcqwVIHmKuasyCFg9ja4gRKQTUo8k1D7A7e6kECHGQiwLIXMc1uN9OtznASL6t7MhSI5iROxog7VTY231CooB4Q3zMGYdPGVqeQ8W2154xrrIGwnzfHDrEYosZLIbM1Y+g2HXVvSYBBsOSytBEgLpV6A6eAfZLoZXBJD5fIzMOQXXDRHjSve74ydGT\\\\\\\/qhAOyEWoPWTO9Gfq+cnmThX6I5K5dYvILEyayH+BvBs8F0FjUSYaHdK4W6EGjyCl15nnQ8GFuYZwFt5vT1tuDXDrn5vGkaIdv+6sInmkEJH\\\\\\\/MX29f0JGdx5u0gmRVnTV8Q2c\\\\\\\/8t\\\\\\\/jeKTuVR6LHTWQCfirJIjEquMQf9EKlODWBaZgGpR9v\\\\\\\/iTYM6JvFEwONlUdMu\\\u0022,\\\u0022iv\\\u0022:\\\u0022rxYmX\\\\\\\/bLjASlt5O3qs7zhA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022fb1abf49f35177abe2444576f6574346e3d7a4546d8edb3d15430bff4c1eb79e\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-483523 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/3/881" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-wll76z-0EIikfWY1S4QcmrUiqgnNlYebb_acD_DDYpU" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/3/881" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-OekSve5Sgz-PlsMfhVfnSDSHbfftEsUtwH8N18x8-8E" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/3/871" title="De Novo Transcriptome Characterization of a Sterilizing Trematode Parasite (Microphallus sp.) from Two Species of New Zealand Snails" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/3/895" title="Predicted Residual Error Sum of Squares of Mixed Models: An Application for Genomic Prediction" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483523" id="node483523" data-pisa="ggg;7/3/881" data-pisa-master="ggg;g3.116.038190" data-apath="/ggg/7/3/881.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Adapting Genotyping-by-Sequencing for Rice F2 Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Tomoyuki Furuta</span>, <span class="highwire-citation-author" data-delta="1">Motoyuki Ashikari</span>, <span class="highwire-citation-author" data-delta="2">Kshirod K. Jena</span>, <span class="highwire-citation-author" data-delta="3">Kazuyuki Doi</span> and <span class="highwire-citation-author" data-delta="4">Stefan Reuscher</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">881-893;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.038190 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483523" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Tomoyuki Furuta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioscience and Biotechnology Center, Nagoya University, 464-8601, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tomoyuki%2BFuruta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Furuta%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATomoyuki%2BFuruta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Motoyuki Ashikari </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioscience and Biotechnology Center, Nagoya University, 464-8601, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Motoyuki%2BAshikari%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ashikari%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMotoyuki%2BAshikari%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kshirod K. Jena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant Breeding Division, International Rice Research Institute, 1301 Manila, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kshirod%2BK.%2BJena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jena%20KK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKshirod%2BK.%2BJena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kazuyuki Doi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Associated Field Science and Research Center, Nagoya University, 470-0151, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kazuyuki%2BDoi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Doi%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKazuyuki%2BDoi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kdoi@agr.nagoya-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">kdoi@agr.nagoya-u.ac.jp</a>
<a href="mailto:reuscher@agr.nagoya-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">reuscher@agr.nagoya-u.ac.jp</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Stefan Reuscher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioscience and Biotechnology Center, Nagoya University, 464-8601, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BReuscher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reuscher%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStefan%2BReuscher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kdoi@agr.nagoya-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">kdoi@agr.nagoya-u.ac.jp</a>
<a href="mailto:reuscher@agr.nagoya-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">reuscher@agr.nagoya-u.ac.jp</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/3/881" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:483523" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:483523/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/3/881.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:483523" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:483523/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/3/881.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:483523" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:483523/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/3/881.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:483523" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:483523/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/3/881.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:483523" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:483523/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Rapid and cost-effective genotyping of large mapping populations can be achieved by sequencing a reduced representation of the genome of every individual in a given population, and using that information to generate genetic markers. A customized genotyping-by-sequencing (GBS) pipeline was developed to genotype a rice F2 population from a cross of <em>Oryza sativa</em> ssp. <em>japonica</em> cv. Nipponbare and the African wild rice species <em>O. longistaminata</em>. While most GBS pipelines aim to analyze mainly homozygous populations, we attempted to genotype a highly heterozygous F2 population. We show how species- and population-specific improvements of established protocols can drastically increase sample throughput and genotype quality. Using as few as 50,000 reads for some individuals (134,000 reads on average), we were able to generate up to 8154 informative SNP markers in 1081 F2 individuals. Additionally, the effects of enzyme choice, read coverage, and data postprocessing are evaluated. Using GBS-derived markers, we were able to assemble a genetic map of 1536 cM. To demonstrate the usefulness of our GBS pipeline, we determined quantitative trait loci (QTL) for the number of tillers. We were able to map four QTL to chromosomes 1, 3, 4, and 8, and partially confirm their effects using introgression lines. We provide an example of how to successfully use GBS with heterozygous F2 populations. By using the comparatively low-cost MiSeq platform, we show that the GBS method is flexible and cost-effective, even for smaller laboratories.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genotyping-sequencing" class="hw-term hw-article-keyword hw-article-keyword-genotyping-by-sequencing" rel="nofollow">genotyping-by-sequencing</a></li><li class="kwd"><a href="/keyword/snp-marker" class="hw-term hw-article-keyword hw-article-keyword-snp-marker" rel="nofollow">SNP marker</a></li><li class="kwd"><a href="/keyword/rice-breeding" class="hw-term hw-article-keyword hw-article-keyword-rice-breeding" rel="nofollow">rice breeding</a></li><li class="kwd"><a href="/keyword/trait-mapping" class="hw-term hw-article-keyword hw-article-keyword-trait-mapping" rel="nofollow">trait mapping</a></li></ul><p id="p-4">Advances in sequencing technology have drastically improved our ability to determine and simultaneously genotype genetic markers (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Davey <em>et al.</em> 2011</a>). The enormous number of short (50–200 bp) reads produced by sequencing platforms has drastically reduced the costs and time associated with DNA sequencing. Those advances may be utilized in whole-genome resequencing approaches to generate a collection of reads from untargeted sites in the genome (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Takagi <em>et al.</em> 2013</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Duitama <em>et al.</em> 2015</a>). Other approaches aim at reducing the complexity of the genome by sequencing only a targeted fraction of the genome. Such genotyping-by-sequencing (GBS) approaches were successful in generating tens of thousands of markers, even in plant species with large and repetitive genomes, like maize, wheat, or barley (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Poland <em>et al.</em> 2012</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Romay <em>et al.</em> 2013</a>), or in more heterozygous animal species like cattle or pig (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">De Donato <em>et al.</em> 2013</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gualdrón Duarte <em>et al.</em> 2013</a>).</p><p id="p-5">It was shown that GBS can be used as a fast and cost-effective tool in population genetics, QTL (quantitative trait locus) discovery, high-resolution mapping, and genomic selection (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Spindel <em>et al.</em> 2013</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Rabbi <em>et al.</em> 2014</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Huang <em>et al.</em> 2014</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Burrell <em>et al.</em> 2015</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Begum <em>et al.</em> 2015</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Elmer <em>et al.</em> 2015</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lin <em>et al.</em> 2015</a>). Since GBS data typically generate relatively dense marker data, a popular analysis choice is a genome-wide association study (GWAS) (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">He <em>et al.</em> 2014</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Sonah <em>et al.</em> 2015</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Begum <em>et al.</em> 2015</a>). This kind of study employs a panel of cultivars or varieties. In addition, there are some examples of QTL analyses using biparental populations combined with GBS (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Spindel <em>et al.</em> 2013</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Honsdorf <em>et al.</em> 2014</a>). In those studies, recombinant inbred lines that had already undergone several rounds of selfing were used to detect QTL. There are also examples of the use of GBS to genotype less fixed populations, like F2s (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Rowan <em>et al.</em> 2015</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Pootakham <em>et al.</em> 2015</a>). In many cases, desirable traits are found only in wild relatives or are spread across diverse elite cultivars. The application of GBS to genotype F2s or breeding materials will greatly facilitate gene discovery and marker-assisted selection in breeding projects.</p><p id="p-6">While GBS certainly has huge benefits for scientists and the breeding community, there are some inherent drawbacks to which no universal solution has yet been found (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Poland and Rife 2012</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">He <em>et al.</em> 2014</a>). The data produced by GBS, and similar strategies, has many missing datapoints compared to datasets from classical, “manually” produced genetic marker data or chip-based systems. Furthermore, there is a considerable error-rate associated with GBS-derived genotypes. Both of these issues can be dealt with at the cost of intensive postprocessing, data correction, and imputation, which is time consuming and requires specific bioinformatics attention. Also, for each GBS project, the researcher has to balance the cost of the sequencing platform with the goal of generating high enough read coverage, and, in turn, marker resolution for the intended analysis. Most GBS strategies aim to sequence only a defined fraction of the whole genome to reduce the number of reads necessary for adequate per-marker read coverage. A common approach is the use of one or two restriction enzymes (RE) to produce fragments with defined endpoints, instead of random shearing of input DNA. A recent protocol (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Elshire <em>et al.</em> 2011</a>) uses a combination of a RE with a 6 bp recognition sequence to target specific sites in the genome, and a RE with a more common 4 bp recognition sequence to generate fragments of suitable length. It was also shown that the choice of RE can influence sequencing results (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Heffelfinger <em>et al.</em> 2014</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Schröder <em>et al.</em> 2016</a>). Another common strategy to reduce sequencing costs is the use of multiplexed libraries. By ligating a sample-specific, unique adapter sequence (also called a barcode) to the DNA fragments before pooling and library preparation, DNA from multiple individuals may be processed in a single library. Currently, between 96-fold and 384-fold multiplexed libraries seem to be most common, with between 500,000 and a few million reads dedicated to each individual sample.</p><p id="p-7">In most cases, GBS aims to detect and simultaneously genotype a large number of single nucleotide polymorphism (SNP) markers. In this study, we used GBS on a rice F2 population derived from a cross of an elite cultivar from East Asia (<em>Oryza sativa</em> ssp. <em>Japonica</em> cv. Nipponbare, NB) and a West African wild rice (<em>O. longistaminata</em>, OL). Several complex traits are found in OL but are absent in NB. For example, OL is capable of perennial growth, while NB is an annual plant. Furthermore, OL is capable of clonal propagation through the use of rhizomes. To identify the genetic basis of those traits, we wanted to perform linkage analysis in an F2 population. Since there are only few markers available for this cross in public datasets, and traditional marker development and genotyping can be laborious, we established a GBS pipeline.</p><p id="p-8">Performing GBS on an F2 population incurs some specific difficulties, since 50% of all SNP sites are expected to be in a heterozygous state. This demands higher read coverage to accurately call genotypes, since correctly calling a heterozygous allele requires the presence of reads from both allele states (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hyma <em>et al.</em> 2015</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Johnson <em>et al.</em> 2015</a>). Some existing GBS pipelines and imputation algorithms deal with that problem by omitting heterozygous calls. In our case that solution was not acceptable, since this would potentially eliminate 50% of all genotype information. Another problem associated with using a wild variant in a cross is that there is considerable heterozygosity in the wild parent’s genome. This can lead to the inability to correctly infer parental haplotypes. In addition, it might be possible that the wild parent (OL) has genome rearrangements, or gene copy number variations, as compared to the cultivated parent (NB). Those rearrangements might cause erroneous genotypes in specific regions and linkage of markers, which, in reality, are located on different chromosomes.</p><p id="p-9">By a combination of the comparatively low-cost Illumina MiSeq platform (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Loman <em>et al.</em> 2012</a>) and high multiplexing, we created a flexible, medium throughput (a few hundred to 1000 individuals) genotyping pipeline. This approach is more flexible than hybridization-based assays (<em>e.g.</em>, Illumina’s Infinium Chips), can be adapted easily to new populations, and increased in scale to larger sequencing platforms. The pipeline was designed to specifically address rice F2 populations, but it should be useful for any F2 population. We investigated the effects of two different REs and different levels of multiplexing on the number of detected SNP markers. Also, we provide an example of how relatively low-coverage data (<em>ca</em>. 150,000 reads per sample) can be sufficient to generate high density genetic maps. Our pipeline uses simple error correction and imputation methods that take advantage of the long, uniparental haplotype blocks found in F2 populations. To show that our GBS pipeline is producing useful genotypes, we mapped QTL for tiller number, and partially confirmed these QTL using introgression lines derived from the same parents as the F2 population.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant cultivation and population development</h3><p id="p-10">The population used in this study was produced and cultivated in the International Rice Research Institute (IRRI), Los Baños, Philippines. An African wild rice, <em>O. longistaminata</em> Acc. IRGC110404 (OL) as male was crossed with the cultivar <em>O. sativa japonica</em> cv. Nipponbare (NB) as female to produce F1 plants, and subsequently F2 populations by self-pollination. Since NB and OL are rather distant relatives within the <em>Oryza</em> genus, there is some degree of incompatibility between both parents. Specifically, the cross between NB and OL led to a failure of endosperm development, resulting in embryonic death. Therefore embryo-rescue had to be performed to avoid embryonic death of F1 seeds. In total 301 and 813 F2 plants were grown in a paddy field enclosed by mesh (the screen house) to prevent insect damage at IRRI in the spring (Feb–May) and fall (Sep–Dec) seasons of 2014, respectively. The total number of tillers (primary and branched shoots of grass plants) was determined after digging up those F2 plants from the paddy field. Leaf blades of the F2s, and three replicate individual plants of each, NB and OL, were sampled for DNA extraction.</p><p id="p-11">Previously, we developed a set of introgression lines (ILs) that harbor between one and three substituted genomic segments derived from OL in the NB genomic background (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Ramos <em>et al.</em> 2016</a>). The ILs consist of BC<sub>4</sub>F<sub>7</sub> and BC<sub>5</sub>F<sub>6</sub> plants derived from a cross between OL as female, and NB as male, and successive backcrosses by NB followed by self-fertilization. Four ILs were selected based on the QTL regions found in this study. The ILs and the recurrent parent NB were germinated in a greenhouse, and cultivated for 30 d. The seedlings were then transplanted to paddy fields at the research station of Nagoya University, Togo, Aichi Prefecture, Japan. Ten plants per line were planted in each row. The number of tillers was counted at the flowering stage in the ILs and NB, excluding damaged plants and plants next to the border of the plot to avoid position effects.</p></div><div id="sec-3" class="subsection"><h3>Library preparation and sequencing</h3><p id="p-12">Genomic DNA from plant material was extracted using the cetyltrimethylammonium bromide (CTAB) method (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Doyle and Doyle 1987</a>). DNA integrity was analyzed by electrophoresis using a 1% agarose gel. DNA concentration of each sample was measured using a QuantusTM Fluorometer with a QuantiFluorTM dsDNA system (Promega, Madison, WI), and adjusted to 10 ng/μl. Libraries were prepared using a combination of two restriction enzymes according to (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Poland <em>et al.</em> 2012</a>) with the following modifications: genomic DNA samples (100 ng each) were digested in 20 μl of CutSmart Buffer by eight units of <em>Pst</em>I or <em>Kpn</em>I, each with eight units of <em>Msp</em>I [all New England Biolabs (Ipswich, MA); for <em>Pst</em>I and <em>Kpn</em>I the High-Fidelity version was used]. The digestion was performed at 37° for 1 hr, followed by an inactivation step at 65° for 20 min. Ligation was conducted in CutSmart Buffer without any modifications to the original protocol. A set of 192 unique barcodes were selected from the list of 384 barcodes designed for <em>Pst</em>I listed in <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Poland <em>et al.</em> (2012)</a>. These barcodes were utilized for both adapters with <em>Pst</em>I overhang and <em>Kpn</em>I overhang. Then 32-multiplexed libraries for samples digested by <em>Pst</em>I and <em>Msp</em>I, or 96-multiplexed libraries for <em>Kpn</em>I and <em>Msp</em>I, were prepared by pooling samples and subsequent PCR-amplification. DNA qualities and fragment sizes in the prepared libraries were evaluated using a Microchip Electrophoresis System for DNA/RNA analysis (MCE-202 MultiNA, SHIMADZU, Kyoto, Japan). In total, 10 32-multiplexed libraries and nine 96-multiplexed libraries were prepared. The libraries were sequenced using a MiSeq instrument with the MiSeq reagents kit v3 for 150 cycles (Illumina Inc., San Diego, CA).</p></div><div id="sec-4" class="subsection"><h3>Detection of SNPs from raw sequencing data</h3><p id="p-13">To detect informative SNPs from raw sequencing data, the TASSEL 4 (Trait Analysis by Association, Evolution and Linkage 4) GBS pipeline (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Glaubitz <em>et al.</em> 2014</a>) was used (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038190/-/DC1/FileS1.zip">File S1</a>). This included creation of a collection of unique, 64 bp long sequences (tags) from the raw sequencing data, alignment of tags to the IRGSP V1.0 <em>O. sativa</em> Nipponbare reference genome (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kawahara <em>et al.</em> 2013</a>) using BWA (Burrows-Wheeler Aligner) (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Li and Durbin 2009</a>) with the –aln and –samse options, SNP calling, and filtering of SNPs based on minor allele frequency. To identify samples with poor read coverage, the TASSEL 4 log files for each library were inspected for individuals with very low read coverage (&lt;1000 reads in our case). These individuals were removed from the analyses, or resequenced in another library if enough plant material was available. We noted that there is a positive correlation between the number of reads and the integrity of the extracted DNA. Initially, SNPs were called without specifying a filter using the DiscoverySNPCallerPlugin from TASSEL 4. Then, all SNPs with a minor allele frequency of &lt;0.25 were removed, as those likely represented sequencing errors or rare alleles.</p><p id="p-14">In the next step, the SNPs were filtered based on parental alleles to leave only SNPs which have fixed, but alternate alleles at any given locus. To achieve this, we selected only those SNPs which were: (1) not variable within each set of triplicate parental samples, (2) not heterozygous in either parent, and (3) different between both parents. Filtering was performed using the hapmap-formatted files and awk. The resulting collection of SNPs was then thinned out using vcf-tools (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Danecek <em>et al.</em> 2011</a>) to a minimum distance of 64 bp between two SNP sites. This eliminated redundant SNPs originating from the same tag, which, in most cases, had identical parental genotypes within each tag. This collection of SNPs was then used to explore the effects of different levels of missing data and imputation.</p><p id="p-15">Preliminary analyses indicated that a large source of error would be undercalled heterozygous alleles (true heterozygous alleles wrongly called as homozygous alleles due to the absence of reads from one of the two states of a heterozygous allele). To counter this, we used vcf-tools to only allow genotypes that were supported by at least seven reads per site and sample. This limits the probability of undercalling a heterozygous site to a theoretical maximum of 1.6% (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Swarts <em>et al.</em> 2014</a>). Following this step, we directly compared the number of reads thought to originate from each parent at each heterozygous site. We calculated the relative allelic depth as the ratio of NB reads/OL reads, and found that &gt;90% of all heterozygous genotypes had a &lt;2-fold difference between the number of reads from each parent. This result indicated that there is no particular bias for reads from one of the parents (<em>e.g.</em>, different PCR efficiencies) that might have led to miscalled heterozygous genotypes. In the next step, a filter for different levels of missing data was implemented. Specifically, we generated (preimputation, pre-error-correction) datasets, in which up to 5, 50 or 75% of all genotypes for any given site were missing (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038190/-/DC1/FileS2.zip">File S2</a>).</p><p id="p-16">To rule out errors in data processing, we also directly analyzed alignments of reads to the Nipponbare reference genome independently from our GBS pipeline. For this, reads were demultiplexed using fastq-multx (<a href="https://github.com/brwnj/fastq-multx">https://github.com/brwnj/fastq-multx</a>), and aligned using bwa as described above. We then inspected read alignments from selected, error-prone markers using the R package Gviz (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hahne and Ivanek 2016</a>). When we analyzed the 10 F2 individuals that showed the highest number of single heterozygous genotypes flanked on both sides by Nipponbare alleles (AHA-type error, for more details see “<em>Results</em>”), we found that raw read alignments supported the called parental genotype in all analyzed cases.</p></div><div id="sec-5" class="subsection"><h3>Imputation and error correction</h3><p id="p-17">As shown here, and in <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Spindel <em>et al.</em> (2013)</a>, GBS data inherently contains errors, and has to be imputed to be useful for linkage analysis. For our work, we took advantage of the fact that missing data, and wrongly called alleles, are distributed randomly across sites and samples. Furthermore, the F2 population in this study is characterized by long-range, uniform parental haplotypes that are long compared to the putative errors. We thus developed a simple imputation and error correction algorithm that is based on regular expressions and executed in R (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">R Development Core Team 2008</a>).</p><p id="p-18">In the first step, the data are transformed from the nucleotide-based hapmap format to an ABH-based format, where A represents NB, B represents OL, and H represents heterozygous alleles. After conversion, we first imputed missing data. Stretches of missing genotypes were filled with the appropriate allele if both flanking, not missing alleles were of the same state (<em>e.g.</em>, the sequence A<strong>NNN</strong>A, with A being a parental allele and <strong>N</strong> being missing data, would be imputed to A<strong>AAA</strong>A). Imputation of missing genotypes was not restricted by the length of each imputed stretch. <em>Post hoc</em> analyses showed that the maximum length of a stretch of consecutive imputed genotypes was seven, but &gt;99% of all imputations were restricted to stretches of one or two genotypes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038190/-/DC1/TableS1.xlsx">Table S1</a>). The median imputed distance between two known genotypes varied between 571 kb (fall 2014 dataset, up to 75% missing data) and 1661 kb (fall 2014 dataset, up to 5% missing data). This imputation resulted in an almost complete elimination of missing alleles. Next, we tried to address the undercalling of heterozygous sites. Empirically, we set a minimum haplotype length of four sites. In any given F2 individual, if a series of homozygous or missing sites of length ≤4 was flanked on both sites by a heterozygous allele, this stretch was replaced with heterozygous sites. The other main error type seemed to be single erroneous alleles interspersed in longer homozygous haplotypes. We assumed those errors are cause by structural differences in the genome of OL compared to the NB reference genome. To counter this, we used a similar strategy as the one used to correct undercalled heterozygous alleles, but used a minimum haplotype length of one. This procedure reduced the number of missing genotypes as a percentage of all genotypes from 2.07 to 0.18%, while it increased the number of heterozygous alleles from 46.27 to 54.57% (data from the fall 2014 population, with up to 75% missing data per site, full dataset in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038190/-/DC1/TableS2.xlsx">Table S2</a>). In the final step, data from both analyzed populations was combined based on the assumed physical position of SNP markers. Since two different enzymes were used for the spring 2014 and the fall 2014 population, no SNP marker was found in both datasets, as different enzymes generate different sets of reads. Thus, we imputed missing data again, using the rules devised above to fill in sites. This imputation step was characterized by a median length of consecutive imputed genotypes of six, and a median length of imputed distance between genotypes of 0.86 Mbp when the missing <em>Pst</em>I-derived genotypes (more common) were imputed in the <em>Kpn</em>I-derived genotypes (less common) in the datasets with up to 5% of missing data (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038190/-/DC1/TableS1.xlsx">Table S1</a>).</p><p id="p-19">Our imputation method relies on identical alleles in two flanking markers to make predictions about the genotypes in between. This approach does not take into account rare double crossover events between those markers. We estimated the possibility for a double crossover event to occur within 1 Mb to be ∼0.15%. This is based on a genetic map with a total length of 1536 cM and a physical length of 400 Mb, which can be expressed as 3.84 cM/Mb. The probability for a double crossover event within 1 Mb was thus calculated as 0.0384<sup>2</sup> = 0.0014756. Given that median imputed distances in the two imputation steps were mostly below 1 Mb, we concluded that our imputation will miss a double crossover event in approximately &lt;1 out of 1500 imputation events.</p><p id="p-20">All TASSEL scripts, and the scripts used for post-TASSEL data processing, can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038190/-/DC1/FileS1.zip">File S1</a>. The imputation and error correction logic described here (in addition to functions for graphical analyses of genotypes) is also available in the “ABHgenotypeR” package for R, which is available at <a href="https://github.com/StefanReuscher/ABHgenotypeR,">https://github.com/StefanReuscher/ABHgenotypeR,</a> or via CRAN (Comprehensive R archive network).</p></div><div id="sec-6" class="subsection"><h3>Confirmation of genotypes by Sanger sequencing</h3><p id="p-21">Ten error-prone SNPs identified by the GBS pipeline were selected for Sanger sequencing (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038190/-/DC1/TableS3.xlsx">Table S3</a>). PCR primer sets to amplify a 150–300 bp region around the selected SNPs were designed based on the IRGSP V1.0 <em>Oryza sativa</em> Nipponbare reference genome sequence (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038190/-/DC1/TableS4.xlsx">Table S4</a>). Genomic DNA extracted from NB, OL and 10 F2 samples with a high error rate were subjected to PCR for the region of the selected SNP. The PCR products were purified by gel purification using the Wizard SV Gel and PCR Clean-Up System (Promega, Madison, WI). After confirming the presence of a single PCR product using 1.5% agarose gels, Sanger sequencing was performed with either of the PCR primers as a sequencing primer using BigDye Terminator v3.1 Cycle Sequencing Kit (Applied Biosystems) and a 3130xl Genetic Analyzer (Applied Biosystems, Foster City, CA). The sequence data, including the raw chromatograms, was analyzed using the sequence assembly software ATGC (Genetyx Corporation, Tokyo, Japan) to identify nucleotides at the SNP positions detected in GBS.</p></div><div id="sec-7" class="subsection"><h3>Data analysis</h3><p id="p-22">General data analysis was performed using the TASSEL graphical user interface, and R. QTL analyses and simulations were performed using the R package “qtl” (v1.37.11) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Broman <em>et al.</em> 2003</a>). For QTL simulations, phenotypic values and genotypes of simulated F2 populations were generated using the function “sim.cross” implemented in the “qtl” package and described in detail in <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Broman and Sen (2009)</a>. “sim.cross” requires a genetic map of markers, the number of individuals, and a model of QTL to generate a simulated population. For simulating the genetic map of markers, we used the “sim.map” function, which requires chromosome lengths and marker numbers. The lengths of the chromosomes were set to 140, 115, 130, 110, 100, 105, 110, 100, 75, 80, 100, and 105 cM for chromosomes 1–12, respectively, based on a genetic map of microsatellite markers developed in our previous QTL study for F2 populations derived from a cross between NB and OL. Simulations with 50, 100, 200, or 400 equally spaced markers were performed. For simulating phenotypic values that were affected by a number of simulated QTL, we assumed the existence of eight QTL (on 8 out of 12 chromosomes), each of which had an additive effect of 0.5. The residual phenotypic variation was assumed to be normally distributed with a variance of one. Under these assumptions, each of the simulated QTL had 4.17% contribution to the phenotypic variance. With the simulated genetic map and the QTL model, data sets of F2 populations for 200, 400, 600, 800, and 1000 individuals were generated using “sim.cross.” We performed simple interval mapping in the simulated F2 populations using the function “scanone,” with the multiple imputation method (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Sen and Churchill 2001</a>). In the multiple imputation method, genotypes between markers were imputed with 1 cM intervals based on genotypes of flanking markers, and multiple imputed genotype data were generated for each individual. Then, a linear regression model was fitted for each marker using the imputed genotype data, and the phenotype data with the assumption of normal distribution of phenotypic values. The threshold for significant LOD scores was calculated from 1000 permutation tests. According to past studies, confidence intervals of detected QTL were usually &gt;10 cM (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kearsey and Farquhar 1998</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Darvasi 1998</a>), so we used that size as a threshold. If a significant QTL (<em>P</em> ≤ 0.05) was detected around the simulated, true QTL position (±10 cM), we counted it as correctly detected. For each condition, 100 simulations were performed, and the probability to correctly detect all QTL was calculated.</p><p id="p-23">Genetic maps using real data were constructed using the “est.map” function with default parameters. QTL analyses for the number of tillers in 1081 F2 plants was performed using a linear regression model with the multiple imputation method by “scanone.” The threshold for significant LOD scores was calculated from 1000 permutation tests. The 95% confidence intervals of significant QTL were estimated using the function “bayesint,” which takes 10<sup>LOD score</sup> values for an obtained LOD profile and rescales it to have an area of one, followed by calculating the connected interval having 95% coverage of the area. The function “fitqtl” was used for calculating percentages of variance of the significant QTL by calculating the coefficient of determination for each single-QTL model obtained using “scanone.” Additive and dominant effects of the significant QTL were calculated from mean phenotypic values for each genotype at the QTL positions obtained by using the function “effectplot.”</p><p id="p-24">Genome-wide analysis of restriction sites was performed using the “restric” tool from the emboss software suite (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Rice <em>et al.</em> 2000</a>). Random sampling of reads from fastq files was performed using fastq-tools (<a href="http://homes.cs.washington.edu/~dcjones/fastq-tools/">http://homes.cs.washington.edu/∼dcjones/fastq-tools/</a>).</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-25"><a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038190/-/DC1/FileS1.zip">File S1</a> contains all code necessary to replicate the GBS-pipeline. The data imputation and error-correction logic is also available in the R package “ABHgenotypeR.” <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038190/-/DC1/FileS2.zip">File S2</a> contains all genotypes from this study, including marker order and position. All demultiplexed reads used for SNP calling were submitted to the DNA Data Bank of JAPAN (<a href="http://www.ddbj.nig.ac.jp/">http://www.ddbj.nig.ac.jp/</a>), and are available under bioproject PRJDB5346. </p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Application of GBS to a rice F2 population</h3><p id="p-26">A population of 268 F2 plants from a cross of NB and OL, including triplicate parental samples, from the spring 2014 season was sequenced first. From this population, libraries of 32 samples each were prepared, and processed with the GBS pipeline (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). This approach resulted in 618,844 average reads per individual, which yielded (with up to 5% missing data) 2144 SNPs (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Analyses using simulated data to determine QTL detection probabilities showed that this number of markers is more than sufficient to detect even weak QTL (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038190/-/DC1/FigureS1.pdf">Figure S1</a>). In fact, a few hundred markers gave sufficient detection power, while, at the same time, the number of F2 individuals appears to be the limiting factor. We therefore optimized our GBS pipeline to process more F2 individuals, at the expense of generating a lower number of SNP markers, by multiplexing more samples per library.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/881/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Flowchart of the GBS data processing. A schematic overview of the different steps of the GBS pipeline." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-927714956" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Flowchart of the GBS data processing. A schematic overview of the different steps of the GBS pipeline.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/881/F1.medium.gif" width="302" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/3/881/F1.medium.gif" width="302" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/881/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/881/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483650" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-27" class="first-child">Flowchart of the GBS data processing. A schematic overview of the different steps of the GBS pipeline.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/483662/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/483662/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Basic parameters of both GBS experiments described in this work</span><div class="sb-div caption-clear"></div></div></div><p id="p-29">For a larger population of 813 F2 plants, and triplicate parental samples from the fall 2014 season, the following changes were implemented: (1) instead of using <em>Pst</em>I as the rare-cutting enzyme, we used <em>Kpn</em>I. There are 107,953 <em>Pst</em>I cut sites reported in the NB reference genome, while there are only 45,065 <em>Kpn</em>I cut sites according to <em>in silico</em> digests. Thus, if all parameters were kept constant, in libraries prepared with <em>Kpn</em>I, the resulting reads will be distributed among fewer sites, but reach a higher per-site coverage. (2) Taking advantage of the higher per-site coverage using <em>Kpn</em>I, we increased the number of samples per library. Prior to library preparation, we examined the effects of decreased read coverage per F2 individual by randomly sampling a fraction of reads from each input fastq file. In these simulated multiplexing analyses, it became clear that the undercalling of heterozygous sites (50% in an F2 population) would become a large source of errors if multiplexing is increased (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038190/-/DC1/FigureS2.pdf">Figure S2</a>). Based on those results, 96-fold multiplexing was deemed feasible, and was implemented with the fall 2014 population. This resulted in an average of 134,447 reads per F2, which yielded (with up to 5% missing data) 301 SNPs (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-30">As expected, higher multiplexing, and a change to <em>Kpn</em>I, led to a lower number of detected SNP sites. When processed through our GBS pipeline, however, both datasets led to similar genotype patterns, the main difference being the number of sites that were reliably detected. As the final step of the GBS pipeline, both datasets were merged. To describe and evaluate the results of the GBS pipeline, we subsequently used data from the fall 2014 dataset. For results regarding the genetics of the NB × OL F2 population and linkage analysis, we used the combined datasets to maximize detection power and resolution.</p></div><div id="sec-11" class="subsection"><h3>Analysis of general SNP properties</h3><p id="p-31">The unfiltered GBS dataset contained a high proportion of missing data (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>) and only ∼4500 out of 37,938 sites were detected in all samples. Also, a substantial number of SNPs was observed, with very low minor allele frequencies (MAF) (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). We used a threshold of MAF &gt;0.25, and different proportions of missing data (&lt;5, &lt;50, &lt;75%), and analyzed the MAF and the proportion of heterozygous sites. When using a very stringent filter of &lt;5% missing data, both the MAF and the proportion of heterozygous sites reached a lower limit at ∼0.35 (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2, C and D</a>). At a higher proportion of missing data, some sites could be observed that had a MAF and proportion of heterozygosity as low as the set threshold of 0.25 (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2, E–H</a>). The bigger spread in allele frequencies and heterozygosity observed for datasets with a higher percentage of missing data might be explained by the inclusion of sites with low read coverage in those datasets. SNP sites that are supported by a small number of reads are more prone to errors. For example, reads representing either NB or OL alleles could have different amplification efficiencies during library preparation. For SNPs with high read coverage this might have no effects, but, for SNPs with low read coverage, this might skew our ability to detect a specific allele. This observation highlights the importance of both adequate read coverage and post SNP-calling error correction.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/881/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Basic SNP characteristics using different filter settings for missing data. Shown are histograms representing the number of SNP sites that exhibit a certain sample coverage (A), minor allele frequency (B, C, E, and G), or proportion of heterozygous sites (D, F, and H). Data are from 813 F2 plants from the fall 2014 population, and were generated using the TASSEL 4 site report function. For (A) and (B), unfiltered data directly after SNP calling was used. For (C)&#x2013;(H), SNP sites were filtered by the indicated proportion of missing data per sample, but no further data imputation or error correction was performed." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-927714956" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Basic SNP characteristics using different filter settings for missing data. Shown are histograms representing the number of SNP sites that exhibit a certain sample coverage (A), minor allele frequency (B, C, E, and G), or proportion of heterozygous sites (D, F, and H). Data are from 813 F2 plants from the fall 2014 population, and were generated using the TASSEL 4 site report function. For (A) and (B), unfiltered data directly after SNP calling was used. For (C)&#x2013;(H), SNP sites were filtered by the indicated proportion of missing data per sample, but no further data imputation or error correction was performed.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/881/F2.medium.gif" width="359" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/3/881/F2.medium.gif" width="359" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/881/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/881/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483586" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-32" class="first-child">Basic SNP characteristics using different filter settings for missing data. Shown are histograms representing the number of SNP sites that exhibit a certain sample coverage (A), minor allele frequency (B, C, E, and G), or proportion of heterozygous sites (D, F, and H). Data are from 813 F2 plants from the fall 2014 population, and were generated using the TASSEL 4 site report function. For (A) and (B), unfiltered data directly after SNP calling was used. For (C)–(H), SNP sites were filtered by the indicated proportion of missing data per sample, but no further data imputation or error correction was performed.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">To evaluate the fidelity of GBS genotypes, we independently genotyped 93 F2 plants using simple-sequence repeat (SSR) markers, and compared both sets of genotypes. It was found that the majority of parental genotypes (&gt;90%) was identical when the results of both genotyping systems were compared (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038190/-/DC1/FigureS3.pdf">Figure S3</a>). The 10% disagreeing markers are explained by single SSR markers, in which up to 1/3 of all genotypes disagree, probably by the SSR marker and the closest GBS marker being on different sides of a recombination event.</p><p id="p-34">Next, we evaluated the distribution of SNP sites along the chromosome (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). SNP sites were notably sparser in the centromeric regions, probably as a result of a high amount of repetitive sequence elements, which prevent reads to be mapped to a unique position. Also, the distribution of sites along the chromosome arms was not even. In general, the SNP density at any given chromosome position increased with the amount of missing data allowed. However, there were some chromosomal regions with low SNP density in which the number of SNPs was hardly affected by the amount of missing data. This was not caused by uneven distribution of <em>Kpn</em>I recognition sites (data not shown). For example, a SNP density below the average was observed on the long arms of chromosome 4 and chromosome 9. The occurrence of such SNP deserts was observed before (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Wang <em>et al.</em> 2009</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Krishnan <em>et al.</em> 2014</a>), but it is unclear if and how those regions are associated with domestication. Lower than average SNP density might also be caused by methylation sensitivity of the <em>Msp</em>I enzyme used during library preparation. <em>Msp</em>I cannot cut its CCGG recognition sequence if the external C is methylated. This might lead to an under-representation of SNPs from heavily methylated regions, such as the centromere, or other repeat-rich regions.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/881/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Marker densities along the chromosomes. Shown is the marker density along the 12 rice chromosomes. The number of markers was determined for bins of 1&#xA0;Mb. Different colored lines represent datasets with the indicated proportion of missing data. Data are from the fall 2014 population (n&#xA0;=&#xA0;813 individuals)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-927714956" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Marker densities along the chromosomes. Shown is the marker density along the 12 rice chromosomes. The number of markers was determined for bins of 1&#xA0;Mb. Different colored lines represent datasets with the indicated proportion of missing data. Data are from the fall 2014 population (&lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;813 individuals).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/881/F3.medium.gif" width="440" height="430"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/3/881/F3.medium.gif" width="440" height="430"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/881/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/881/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483653" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-35" class="first-child">Marker densities along the chromosomes. Shown is the marker density along the 12 rice chromosomes. The number of markers was determined for bins of 1 Mb. Different colored lines represent datasets with the indicated proportion of missing data. Data are from the fall 2014 population (<em>n</em> = 813 individuals).</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">In an ideal F2 population, one would expect that the parental alleles segregate according to a 1:2:1 ratio (parent A: heterozygous: parent B). However, a plot of allele states along the chromosomes revealed regions with distorted genotype ratios (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). As a general trend, the OL alleles seemed to be transmitted at slightly lower levels. As an extreme example, the long arm of chromosome 4 has a drastically reduced frequency of the OL allele, with OL genotype frequencies decreasing to &lt;10%, as opposed to the expected 25%. In most chromosomal regions where one parental allele was found under-represented, the frequency of heterozygous genotypes in turn was increased to &gt;50%. Very likely, those effects are due to chromosomal regions associated with reproductive incompatibility.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/881/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Parental allele frequencies along the chromosomes. Shown are the frequencies of parental alleles observed along the 12 rice chromosomes. Data are from the joined datasets from spring and fall 2014. Only markers present in 95% of all samples in the respective dataset are shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-927714956" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Parental allele frequencies along the chromosomes. Shown are the frequencies of parental alleles observed along the 12 rice chromosomes. Data are from the joined datasets from spring and fall 2014. Only markers present in 95% of all samples in the respective dataset are shown.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/881/F4.medium.gif" width="440" height="430"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/3/881/F4.medium.gif" width="440" height="430"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/881/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/881/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483678" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-37" class="first-child">Parental allele frequencies along the chromosomes. Shown are the frequencies of parental alleles observed along the 12 rice chromosomes. Data are from the joined datasets from spring and fall 2014. Only markers present in 95% of all samples in the respective dataset are shown.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Constructing a genetic map</h3><p id="p-38">To inspect GBS genotypes and haplotypes, we constructed graphical representations of genotypes (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, full dataset in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038190/-/DC1/FileS2.zip">File S2</a>). This made it obvious that GBS data without imputation and error correction contains wrongly called genotypes (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>). Since F2 populations have relatively long haplotypes, the observed very short (1–2 markers) uniform genotype stretches found as islands in longer stretches are most likely errors. After imputation of missing data (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>), we used a simple error correction algorithm based on haplotype length to efficiently correct those errors (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5C</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/881/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Graphical representations of GBS-derived genotypes at different stages of post-processing. Shown are graphical representations of genotypes after inferring parental alleles (A), after inferring parental alleles and imputation of missing data (B), and after inferring of parental alleles, imputation, and error correction (C). Genotypes of 50 representative F2 individuals are shown, with each F2 as a single horizontal track. The chromosome length is proportional to the number of markers, and only chromosomes 1 to 3 are shown. In total, 312 markers (fall 2014 population, up to 50% missing data) are displayed, with genotypes color-coded as blue (NB), orange (OL), green (heterozygous), and black (not determined)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-927714956" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Graphical representations of GBS-derived genotypes at different stages of post-processing. Shown are graphical representations of genotypes after inferring parental alleles (A), after inferring parental alleles and imputation of missing data (B), and after inferring of parental alleles, imputation, and error correction (C). Genotypes of 50 representative F2 individuals are shown, with each F2 as a single horizontal track. The chromosome length is proportional to the number of markers, and only chromosomes 1 to 3 are shown. In total, 312 markers (fall 2014 population, up to 50% missing data) are displayed, with genotypes color-coded as blue (NB), orange (OL), green (heterozygous), and black (not determined).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/881/F5.medium.gif" width="440" height="253"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/3/881/F5.medium.gif" width="440" height="253"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/881/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/881/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483698" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-39" class="first-child">Graphical representations of GBS-derived genotypes at different stages of post-processing. Shown are graphical representations of genotypes after inferring parental alleles (A), after inferring parental alleles and imputation of missing data (B), and after inferring of parental alleles, imputation, and error correction (C). Genotypes of 50 representative F2 individuals are shown, with each F2 as a single horizontal track. The chromosome length is proportional to the number of markers, and only chromosomes 1 to 3 are shown. In total, 312 markers (fall 2014 population, up to 50% missing data) are displayed, with genotypes color-coded as blue (NB), orange (OL), green (heterozygous), and black (not determined).</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">When we used the fall 2014 dataset to construct a genetic map, it became again clear that raw GBS data cannot be used directly (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). When uncorrected data with up to 75 or 50% (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6, A, B, D, and E</a>) of missing data per site was used to generate a genetic map, chromosomes appeared expanded with chromosomes of up to 3500 cM. The map distention we observed was conspicuously similar to the distention shown in <a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Spindel <em>et al.</em> (2013)</a>, and we applied a similar strategy to consolidate our genetic map. Both a rigorous restriction on missing data (up to 5% missing, <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6, G–I</a>) or imputation and error correction (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6, C and F</a>) seemed to alleviate the problem. Restricting missing data led to a strong reduction of available SNP sites (compare 837 for 50% missing to 301 for 5% missing), but also shortened the genetic map. Using filtering, imputation, and error correction, we gradually improved the genetic map, even when up to 75% of genotypes were initially missing for each individual site. The final genetic map (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6I</a>) had a total size of 1536 cM, which is in agreement with other data. We still observed some distention, for example on chromosome 5 and chromosome 12. Although haplotypes and alleles appear to be correct in those regions, we can observe strong linkage of markers in those regions with markers from different chromosomes (data not shown).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/881/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic maps from datasets with different proportions of missing data and post-processing. Shown are linkage maps of GBS marker datasets. Panels show datasets with SNP-calling thresholds allowing up to 75% (A&#x2013;C), 50% (D&#x2013;F), and 5% (G&#x2013;I) missing data, at different steps of the GBS pipeline. Uncorrected (A, D, and G) indicates data without further postprocessing. Imputed (B, E, and H) indicates data with missing data imputed, but no error correction performed. Corrected (C, F, and I) indicates data with both imputation and error-correction performed. Data are from 813 F2 plants from the fall 2014 dataset. Distances between markers are shown in cM." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-927714956" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genetic maps from datasets with different proportions of missing data and post-processing. Shown are linkage maps of GBS marker datasets. Panels show datasets with SNP-calling thresholds allowing up to 75% (A&#x2013;C), 50% (D&#x2013;F), and 5% (G&#x2013;I) missing data, at different steps of the GBS pipeline. Uncorrected (A, D, and G) indicates data without further postprocessing. Imputed (B, E, and H) indicates data with missing data imputed, but no error correction performed. Corrected (C, F, and I) indicates data with both imputation and error-correction performed. Data are from 813 F2 plants from the fall 2014 dataset. Distances between markers are shown in cM.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/881/F6.medium.gif" width="440" height="429"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/3/881/F6.medium.gif" width="440" height="429"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/881/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/881/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483681" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-41" class="first-child">Genetic maps from datasets with different proportions of missing data and post-processing. Shown are linkage maps of GBS marker datasets. Panels show datasets with SNP-calling thresholds allowing up to 75% (A–C), 50% (D–F), and 5% (G–I) missing data, at different steps of the GBS pipeline. Uncorrected (A, D, and G) indicates data without further postprocessing. Imputed (B, E, and H) indicates data with missing data imputed, but no error correction performed. Corrected (C, F, and I) indicates data with both imputation and error-correction performed. Data are from 813 F2 plants from the fall 2014 dataset. Distances between markers are shown in cM.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Investigating error-prone SNP markers</h3><p id="p-42">Although we were able to correct most assumed errors in the parental genotypes, and construct a reasonable genetic map, we further analyzed the types of errors that occurred. We first defined all haplotypes of length = 1 as errors; <em>e.g.</em>, a single A (representing NB) flanked by B genotypes (representing OL), as this is very unlikely to reflect the true allele state in an F2 population at the given marker density. Using the imputed, but not error-corrected, data we counted the occurrences of each possible error-type (HAH, HBH, AHA, BHB, ABA, and BAB) in the dataset from fall 2014 (5% max missing) for each of the 301 SNPs, and for each of the 813 F2 individuals, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038190/-/DC1/TableS3.xlsx">Table S3</a>).</p><p id="p-43">We first analyzed the distribution of the occurrence of each possible error-type per SNP (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>). This distribution made clear that most SNPs have very few (75%-tile ≤5) errors (as defined above). However, for HAH, HBH, AHA, and BHB-type errors, a small number of SNPs accounted for most of the errors. The two most frequent types of errors were “undercalled” OL alleles (HBH, 36.63% of all errors), and “overcalled” heterozygous alleles within a NB context (AHA, 35.33% of all errors). When we performed a scatterplot analysis using the number of HBH and AHA errors (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7B</a>), the SNPs formed two distinct clusters. SNPs in the low-error cluster were correctly called most of the time. SNPs in the high-error cluster tended to accumulate both HBH and AHA errors at the same time. This observation was confirmed when we visualized uncorrected genotypes from 10 F2s in which the most AHA-type errors were detected (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7C</a>). In those F2s, some markers consistently showed HBH- and AHA-type errors. This hints at a hidden, common cause for those two error-types.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/881/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analyzing types of putatively wrong allele calls. The distribution of all six possible error-types is shown as a boxplot (A). In (B), each point represents one SNP plotted according to the two most prevalent error-types (HBH and AHA). The scatterplot revealed two distinct clusters of error frequencies indicated by the two shaded regions. In (C), graphical genotypes are shown as described in Figure 5. Data from 10 F2 with the most AHA errors are shown. Uncorrected data from the fall 2014 dataset (maximum missing 5%) are shown in all panels." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-927714956" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Analyzing types of putatively wrong allele calls. The distribution of all six possible error-types is shown as a boxplot (A). In (B), each point represents one SNP plotted according to the two most prevalent error-types (HBH and AHA). The scatterplot revealed two distinct clusters of error frequencies indicated by the two shaded regions. In (C), graphical genotypes are shown as described in Figure 5. Data from 10 F2 with the most AHA errors are shown. Uncorrected data from the fall 2014 dataset (maximum missing 5%) are shown in all panels.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/881/F7.medium.gif" width="440" height="358"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/3/881/F7.medium.gif" width="440" height="358"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/881/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/881/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483679" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-44" class="first-child">Analyzing types of putatively wrong allele calls. The distribution of all six possible error-types is shown as a boxplot (A). In (B), each point represents one SNP plotted according to the two most prevalent error-types (HBH and AHA). The scatterplot revealed two distinct clusters of error frequencies indicated by the two shaded regions. In (C), graphical genotypes are shown as described in <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>. Data from 10 F2 with the most AHA errors are shown. Uncorrected data from the fall 2014 dataset (maximum missing 5%) are shown in all panels.</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">In the next step, we analyzed whether obvious genomic rearrangements might be the cause for error-prone SNPs. To this end, we isolated a 65 bp sequence from the NB reference genome centered on the location around the SNP. This sequence was then used as a query for a BLASTN search in the recently published OL genome (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Zhang <em>et al.</em> 2015</a>), retaining the best three hits. Out of 301 sequences covering the SNP positions, 206 had their best BLASTN hit on the same chromosome in both NB and OL, indicating that there is a syntenic relationship (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038190/-/DC1/TableS3.xlsx">Table S3</a>). For 27 sequences, no BLASTN hit was found, which is probably due to the lower coverage of the OL genome compared to the NB genome. For 68 sequences, however, the best BLASTN hit in the OL genome was on a different chromosome when NB and OL were compared. In addition, for 28 out of the 206 syntenic markers, BLASTN reported at least one additional hit. When we combined the data from BLASTN searches with the occurrences of overcalled AHA alleles per marker, we found that the top three most error-prone markers appeared to be nonsyntenic in NB and OL. However, from the top 10 most error-prone markers, only four appeared to be nonsyntenic, while six appeared to be syntenic. This indicated that nonsyntenic markers might explain some of the observed errors, but they cannot explain all observed errors.</p><p id="p-46">In addition to using SSR markers (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038190/-/DC1/FigureS3.pdf">Figure S3</a>), we also used Sanger sequencing to determine the nucleotides at SNP positions. We sequenced the genomic region around 10 particular error-prone SNP sites in 10 F2 plants and both parents. This enabled us to directly compare GBS- and Sanger sequencing derived genotypes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038190/-/DC1/FileS3.xlsx">File S3</a>). For the 10 analyzed F2 plants, 7 out of ten SNPs showed identical parental alleles between GBS and Sanger sequencing derived genotypes, despite many of those genotypes forming haplotypes of length = 1, and thus being flagged as errors as described above. This result rules out that mere technical errors (wrong base calls, misaligned reads) are a major source of seemingly erroneous alleles. Instead, it is possible that structural differences between the parental genomes are a major source of error in this population. To still allow the calculation of a genetic map for later QTL analysis, we assumed haplotypes of length = 1 to be errors and corrected them as described before.</p></div><div id="sec-14" class="subsection"><h3>QTL analysis</h3><p id="p-47">Being able to produce a correct genetic map using the combined dataset reassured us that our GBS data are sufficient for linkage analysis. For QTL analysis in 1081 F2 plants, we chose to use the number of tiller as the phenotype. We detected four significant QTL on chromosomes 1, 3, 4, and 8, which were named qOLTN1, qOLTN2, qOLTN3, and qOLTN4, respectively (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8A</a>). Among these four QTL, qOLTN1 on chromosome 1 showed the highest LOD score with 20.15 (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8B</a>), while the other QTL showed LOD scores &lt;6.9. To analyze these QTL in more detail, we calculated 95% confidence intervals, percentages of variance, and effects for each QTL (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The confidence interval of qOLTN1 spanned a 3.6 Mb region from 27.1–30.7 Mb on chromosome 1. This QTL explained 8.23% of the variance in the number of tillers of the F2 population, and showed a negative additive effect of −9.17 and a positive dominant effect of 5.22. These results suggested that an OL allele at qOLTN1 acts recessive to decrease the number of tillers as compared to NB. Unlike the case of qOLTN1, the other QTL gave only little contributions on the differences in the number of tillers, and relatively smaller effects (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Interestingly, qOLTN4 exhibited a positive superdominant effect, in which the additive effect was −2.44, while the dominant effect was 4.51. This result means that heterozygotes at qOLTN4 produce more tillers than either NB homozygotes or OL homozygotes.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/3/881/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Detection of QTL for tiller number using GBS markers. Shown are the results of a linkage analysis to detect QTL that have an effect on tiller number using data from joined spring and fall datasets with up to 75% missing data per marker. LOD scores are shown as black lines for all 12 chromosomes (A), or for chromosome 1 only (B). A LOD threshold for significance (P&#xA0;&#x2264;&#xA0;0.05) is shown as a dashed orange line. The blue area in (B) highlights the 95% confidence interval of qOLTN1 (QTL1 for tiller number O. longistaminata). Distances are shown in cM." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-927714956" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Detection of QTL for tiller number using GBS markers. Shown are the results of a linkage analysis to detect QTL that have an effect on tiller number using data from joined spring and fall datasets with up to 75% missing data per marker. LOD scores are shown as black lines for all 12 chromosomes (A), or for chromosome 1 only (B). A LOD threshold for significance (&lt;em&gt;P&lt;/em&gt;&#xA0;&#x2264;&#xA0;0.05) is shown as a dashed orange line. The blue area in (B) highlights the 95% confidence interval of qOLTN1 (QTL1 for tiller number &lt;em&gt;O. longistaminata&lt;/em&gt;). Distances are shown in cM.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/3/881/F8.medium.gif" width="440" height="308"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/7/3/881/F8.medium.gif" width="440" height="308"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/3/881/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/3/881/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/483654" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-48" class="first-child">Detection of QTL for tiller number using GBS markers. Shown are the results of a linkage analysis to detect QTL that have an effect on tiller number using data from joined spring and fall datasets with up to 75% missing data per marker. LOD scores are shown as black lines for all 12 chromosomes (A), or for chromosome 1 only (B). A LOD threshold for significance (<em>P</em> ≤ 0.05) is shown as a dashed orange line. The blue area in (B) highlights the 95% confidence interval of qOLTN1 (QTL1 for tiller number <em>O. longistaminata</em>). Distances are shown in cM.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/483670/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/483670/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Percentages of variance and effects of the significant QTL</span><div class="sb-div caption-clear"></div></div></div><p id="p-55">To evaluate the results of our QTL simulation (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038190/-/DC1/FigureS1.pdf">Figure S1</a>) against this real data, we performed linkage analyses for random subsets of varying numbers of F2 plants. As predicted in our simulations, we found that up to 1000 F2 plants are necessary to reliably detect all significant QTL (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038190/-/DC1/FigureS4.pdf">Figure S4</a>). When we used all 1081 F2 plants for linkage analysis, but varied the amount of missing data allowed in the prefiltered datasets, we found very similar LOD score profiles (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038190/-/DC1/FigureS5.pdf">Figure S5</a>). We thus used the dataset with up to 75% missing data per site before postprocessing to maximize marker resolution.</p><p id="p-56">To verify the QTL, we conducted a field experiment to measure the number of tillers in ILs having OL genomic segments at each of the QTL locations. Four ILs having OL chromosomal segments around QTL locations were selected from the pool of ILs, and named IL-qOLTN1, IL-qOLTN2, IL-qOLTN3, and IL-qOLTN4, for having OL chromosomal segments around qOLTN1, qOLTN2, qOLTN3, and qOLTN4, respectively. IL-qOLTN1 and IL-qOLTN2 showed a significant decrease in the number of tillers compared with NB (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). The reductions of tillers in these two ILs are in agreement with the negative additive effects of qOLTN1 and qOLTN2 (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Furthermore, IL-qOLTN3 and IL-qOLTN4 produced more and fewer tillers than NB, respectively, although the differences were not significant. However, the results observed in IL-qOLTN3 and IL-qOLTN4 also corresponded to the positive and negative additive effects of qOLTN3 and qOLTN4, respectively. In summary, we could successfully detect four QTL using our GBS data for the number of tillers, and partially verify the effects of those QTL in ILs.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/483584/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/483584/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Tiller number in the introgression lines</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-63">Our aim for this study was to utilize GBS for rapid genotyping of rice F2 populations. As expected, GBS proved to be a robust and efficient method to genotype large populations (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-42-5" class="xref-bibr" href="#ref-42">Spindel <em>et al.</em> 2013</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lu <em>et al.</em> 2013</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Liu <em>et al.</em> 2014</a>). For successful application of GBS, it is necessary to generate adequate read coverage across the genome, and also for each individual that is sequenced. In our approach to genotype a rice F2 population, we further took into account the number of individuals and markers that are necessary to detect QTL. Since one of the main motivations to perform GBS is to save time and money compared to classical markers, one would like to use as few sequencing runs on any platform as necessary to achieve the desired sequencing depth. Our choice to change the enzyme from <em>Pst</em>I to <em>Kpn</em>I led to predictable changes in the resulting SNP collection. Other reports have also shown that enzyme choice is an important parameter to optimize GBS for any given species, highlighting the importance of using <em>in silico</em> digests of the genome of the target organism beforehand (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Heffelfinger <em>et al.</em> 2014</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Schröder <em>et al.</em> 2016</a>). Marker density depends also partially on sequencing depth, which in turn depends on the number of individual per sequencing run. To be most efficient, it is thus advisable to take into account the desired marker density when laying out a genotyping project involving GBS. In our experience, performing a small-scale pilot experiment using the desired population and sequencing platform, combined with linkage analysis on simulated data, allowed us to use GBS more efficiently. The results of linkage analyses using both simulated (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038190/-/DC1/FigureS1.pdf">Figure S1</a>) and experimental (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038190/-/DC1/FigureS4.pdf">Figure S4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038190/-/DC1/FigureS5.pdf">Figure S5</a>) data suggested that our GBS approach resulted in a saturation of markers. The fact that our linkage analysis yields comparable results, even when up to 75% of missing data for each marker were acceptable in the raw data, shows that even simple imputation algorithms can reinforce the usefulness of GBS data tremendously. We speculate that, for certain applications, even fewer markers, and, in turn, fewer reads per individual would be sufficient, thus allowing even higher multiplexing and sample throughput. Of course, this might also depend on the genome size of the analyzed species, and the amount of repetitive elements in that genome.</p><div id="sec-16" class="subsection"><h3>Optimizing GBS strategies</h3><p id="p-64">Although we were successful in using GBS, we noticed several shortcomings where no best practice seems yet to be established. There seems to be very little consensus about how GBS protocols should be adapted to different species, and to different populations. For variant calling, filtering, and exploration of our dataset we used the TASSEL4 (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Glaubitz <em>et al.</em> 2014</a>), which was develop to work efficiently with large maize populations. It became apparent that additional specific bioinformatics analyses were necessary to get the most information from our dataset. This shows that a given GBS protocol needs to be optimized for a specific species or population. Another issue is the high error rate of raw GBS data. While it is possible to eliminate most errors using post SNP-calling error correction, some errors will inevitably remain. Our results indicated that simple technical errors (<em>e.g.</em>, wrong base calls or misaligned reads) are not a major source of errors. It would thus be worth investigating the source of errors, as this might lead to new insights into the population in question. In our case, where a wild species is crossed to a cultivated one, it can be assumed that there will be large-scale differences between the two parental genomes contributing to the F2 individuals (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Wang <em>et al.</em> 2014</a>). Those differences most likely include gene copy number variations, or even rearrangements of regions between chromosomes. We found indirect evidence for such large rearrangements when we analyzed the genome-wide linkage of markers. Several regions in which seemingly correct haplotypes were in strong linkage disequilibrium with both neighboring regions and regions on other chromosomes were found (data not shown). In addition, error-types do not occur at random. In our dataset, &gt;70% of all errors were caused by two error-types (HBH and AHA) (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7A</a>), indicating a common, systematic cause for both error-types. Future GBS pipelines could address those issues, either by taking into account improved reference genome information, through linkage disequilibrium filtering, or by rearranging marker order. However, at the moment, GBS software that explicitly addresses the problems associated with a wide cross does not exist.</p><p id="p-65">When we established our GBS pipeline, we noticed several irregularities in the genome-wide SNP statistics. For example, we noticed that several regions of the genome were sparsely covered with SNPs (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Also, we noted that, in several regions, parental allele frequency deviated from the expected 1:2:1 ratio (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). It is important to note that this population is affected by reproductive incompatibilities, and we had to routinely use embryo-rescue to propagate plant materials. It is very likely that the deviating allele frequency is a consequence of reproductive incompatibility, which has its genetic basis in these regions. To further analyze this, it would be necessary to genotype the offspring of multiple F1 crosses. We suggest that GBS might be a useful tool to study reproductive isolation and preferential transmission, since it can quickly define regions with allele distortion.</p></div><div id="sec-17" class="subsection"><h3>Conclusion</h3><p id="p-66">In summary, we show an application of GBS to perform linkage analysis in a rice F2 population. We also provide an example of how to plan and carry out adequate, cost effective, reduced-representation sequencing. With our dataset, we successfully detected QTL for tiller number on chromosomes 1, 3, 4, and 8, which we could partially confirm using ILs. For future GBS genotyping efforts, we suggest evaluating enzyme choice, multiplexing of libraries and post-processing to meet the requirements of the desired post-GBS analyses.</p></div></div><div class="section" id="sec-18"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-67">The authors thank Rosalyn B. Angeles-Shim, Ruby S. Lapis, and Angelito A. Aragon from the International Rice Research Institute for cultivating their plant material, Mikako Tasaki for assisting library preparation and MiSeq runs, and Yoko Niimi for performing genotyping using SSR markers. This work was supported by Core Research for Evolutional Science and Technology from the Japan Science and Technology Agency to M.A., the Council for Science, Technology and Innovation, Cross-ministerial Strategic Innovation Promotion Program, Technologies for creating next-generation agriculture, forestry and fisheries (funding agency Bio-oriented Technology Research Advancement Institution, NARO) to K.D., and the National Bio Resource Project (NBRP) “Rice” to K.D.</p><p id="p-68">Author contributions: T.F. created the NB × OL F2 populations, prepared sequencing libraries, performed MiSeq sequencing, performed linkage analysis, and assisted in drafting the manuscript. S.R. developed and optimized the GBS pipeline, performed bioinformatics analyses, and drafted the manuscript. K.D. and M.A. conceived the experimental design. K.K.J. oversaw the production of plant materials. M.A. oversaw the project and assisted in drafting the manuscript. All authors have read and approved the manuscript. The authors declare that they have no competing interests.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-15"><p id="p-69">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038190/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038190/-/DC1</a>.</p></li><li class="fn" id="fn-16"><p id="p-70"><em>Communicating editor: S. A. Jackson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-10-16"><span class="received-label">Received </span>October 16, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-01-09"><span class="accepted-label">Accepted </span>January 9, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Furuta <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.1" data-doi="10.1371/journal.pone.0119873"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Begum</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spindel</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lalusin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borromeo</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gregorio</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-wide association mapping for yield and other agronomic traits in an elite breeding population of tropical rice <em>(Oryza sativa</em>).</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0119873</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0119873%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0119873%26rft_id%253Dinfo%253Apmid%252F25785447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0119873&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25785447&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-book" id="cit-7.3.881.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">Ś.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source"><em>A Guide to QTL Mapping with R/qtl</em>.</span> <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.3" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxOS83Lzg4OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMy84ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.4" data-doi="10.1111/mec.13167"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burrell</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pepper</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hodnett</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goolsby</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Overholt</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Exploring origins, invasion history and genetic diversity of <em>Imperata cylindrica</em> (L.) P. Beauv. (Cogongrass) in the United States using genotyping by sequencing.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2177</span>–<span class="cit-lpage">2193</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D24%26rft.spage%253D2177%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13167%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13167&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.5" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMTUvMjE1NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMy84ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.6" data-doi="10.1038/ng0198-19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darvasi</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Experimental strategies for the genetic dissection of complex traits in animal models.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDarvasi%26rft.auinit1%253DA.%26rft.volume%253D18%26rft.issue%253D1%26rft.spage%253D19%26rft.epage%253D24%26rft.atitle%253DExperimental%2Bstrategies%2Bfor%2Bthe%2Bgenetic%2Bdissection%2Bof%2Bcomplex%2Btraits%2Bin%2Banimal%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng0198-19%26rft_id%253Dinfo%253Apmid%252F9425894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng0198-19&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9425894&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071259600015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.7" data-doi="10.1038/nrg3012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">J. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide genetic marker discovery and genotyping using next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDavey%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D510%26rft.atitle%253DGenome-wide%2Bgenetic%2Bmarker%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3012%26rft_id%253Dinfo%253Apmid%252F21681211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21681211&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.8" data-doi="10.1371/journal.pone.0062137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Donato</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peters</span>  <span class="cit-name-given-names">S. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hussain</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imumorin</span>  <span class="cit-name-given-names">I. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genotyping-by-sequencing (GBS): a novel, efficient and cost-effective genotyping method for cattle using next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e62137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De62137%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0062137%26rft_id%253Dinfo%253Apmid%252F23690931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0062137&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23690931&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doyle</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doyle</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">A rapid DNA isolation procedure for small quantities of fresh leaf tissue.</span> <abbr class="cit-jnl-abbrev">Phytochem. Bull.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytochem.%2BBull.%26rft.volume%253D19%26rft.spage%253D11%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.10" data-doi="10.1371/journal.pone.0124617"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duitama</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanabria</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cruz</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quintero</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Whole genome sequencing of elite rice cultivars as a comprehensive information resource for marker assisted selection.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0124617</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0124617%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0124617%26rft_id%253Dinfo%253Apmid%252F25923345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0124617&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25923345&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elmer</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Humira</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">François</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Association mapping of QTLs for sclerotinia stem rot resistance in a collection of soybean plant introductions using a genotyping by sequencing (GBS) approach.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiol.%26rft.volume%253D15%26rft.spage%253D5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.12" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.13" data-doi="10.1371/journal.pone.0090346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harriman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e90346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De90346%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0090346%26rft_id%253Dinfo%253Apmid%252F24587335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0090346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24587335&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gualdrón Duarte</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">R. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ernst</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raney</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantet</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genotype imputation accuracy in a F2 pig population using high density and low density SNP panels.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D14%26rft.spage%253D38%26rft_id%253Dinfo%253Apmid%252F23651538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23651538&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-book" id="cit-7.3.881.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hahne</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ivanek</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Visualizing genomic data using Gviz and Bioconductor</span>, pp. <span class="cit-fpage">335</span>–<span class="cit-lpage">351</span> in <span class="cit-source">Statistical Genomics: Methods and Protocols</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Mathé</span>  <span class="cit-name-given-names">E.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">S.</span></span>. <span class="cit-publ-name">Springer New York</span>, <span class="cit-publ-loc">New York, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laroche</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Z.-X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genotyping-by-sequencing (GBS), an ultimate marker-assisted selection (MAS) tool to accelerate plant breeding.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">484</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D5%26rft.spage%253D484%26rft_id%253Dinfo%253Apmid%252F25324846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25324846&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.17" data-doi="10.1186/1471-2164-15-979"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffelfinger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fragoso</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreno</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Overton</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mottinger</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Flexible and scalable genotyping-by-sequencing strategies for population studies.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">979</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D979%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-979%26rft_id%253Dinfo%253Apmid%252F25406744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-979&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25406744&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Honsdorf</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">March</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hecht</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eglinton</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pillen</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Evaluation of juvenile drought stress tolerance and genotyping by sequencing with wild barley introgression lines.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">1475</span>–<span class="cit-lpage">1495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D34%26rft.spage%253D1475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.19" data-doi="10.1371/journal.pone.0102448"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wight</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tinker</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Using genotyping-by-sequencing (GBS) for genomic discovery in cultivated oat.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e102448</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De102448%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0102448%26rft_id%253Dinfo%253Apmid%252F25047601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0102448&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25047601&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hyma</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barba</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Londo</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Heterozygous mapping strategy (HetMappS) for high resolution genotyping-by-sequencing markers: a case study in grapevine.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0134880</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0134880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wittgenstein</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyma</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Temnykh</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genotyping-by-sequencing (GBS) detects genetic structure and confirms behavioral QTL in tame and aggressive foxes (<em>Vulpes vulpes</em>).</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0127013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0127013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.22" data-doi="10.1186/1939-8433-6-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawahara</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de la Bastide</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanamori</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCombie</span>  <span class="cit-name-given-names">W. R</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Improvement of the <em>Oryza sativa</em> Nipponbare reference genome using next generation sequence and optical map data.</span> <abbr class="cit-jnl-abbrev">Rice (N. Y.)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRice%2B%2528N.%2BY.%2529%26rft.volume%253D6%26rft.spage%253D4%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1939-8433-6-4%26rft_id%253Dinfo%253Apmid%252F24280374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1939-8433-6-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24280374&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.23" data-doi="10.1046/j.1365-2540.1998.00500.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kearsey</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farquhar</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">QTL analysis in plants; where are we now?</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">137</span>–<span class="cit-lpage">142</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D80%26rft.spage%253D137%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2540.1998.00500.x%26rft_id%253Dinfo%253Apmid%252F9503632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2540.1998.00500.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9503632&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000072141900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krishnan</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waters</span>  <span class="cit-name-given-names">D. L</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henry</span>  <span class="cit-name-given-names">R. J</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Australian wild rice reveals pre-domestication origin of polymorphism deserts in rice genome.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e98843</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De98843%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.25" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMy84ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genotyping-by-sequencing (GBS) identified SNP tightly linked to QTL for pre-harvest sprouting resistance.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">1385</span>–<span class="cit-lpage">1395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D1385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.27" data-doi="10.1186/1471-2164-15-104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bayer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Druka</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">An evaluation of genotyping by sequencing (GBS) to map the <em>Breviaristatum-e</em> (<em>ari-e</em>) locus in cultivated barley.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D104%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-104%26rft_id%253Dinfo%253Apmid%252F24498911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24498911&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.28" data-doi="10.1038/nbt.2198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loman</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misra</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dallman</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Constantinidou</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbia</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Performance comparison of benchtop high-throughput sequencing platforms.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">434</span>–<span class="cit-lpage">439</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DLoman%26rft.auinit1%253DN.%2BJ.%26rft.volume%253D30%26rft.issue%253D5%26rft.spage%253D434%26rft.epage%253D439%26rft.atitle%253DPerformance%2Bcomparison%2Bof%2Bbenchtop%2Bhigh-throughput%2Bsequencing%2Bplatforms.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2198%26rft_id%253Dinfo%253Apmid%252F22522955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2198&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22522955&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.29" data-doi="10.1371/journal.pgen.1003215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cherney</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Switchgrass genomic diversity, ploidy, and evolution: novel insights from a network-based SNP discovery protocol.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003215</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003215%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003215%26rft_id%253Dinfo%253Apmid%252F23349638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23349638&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.30" data-doi="10.3835/plantgenome2012.05.0005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rife</span>  <span class="cit-name-given-names">T. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genotyping-by-sequencing for plant breeding and genetics.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">92</span>–<span class="cit-lpage">102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D5%26rft.spage%253D92%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.05.0005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.05.0005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000312663100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.31" data-doi="10.1371/journal.pone.0032253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e32253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPoland%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De32253%26rft.epage%253De32253%26rft.atitle%253DDevelopment%2Bof%2Bhigh-density%2Bgenetic%2Bmaps%2Bfor%2Bbarley%2Band%2Bwheat%2Busing%2Ba%2Bnovel%2Btwo-enzyme%2Bgenotyping-by-sequencing%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0032253%26rft_id%253Dinfo%253Apmid%252F22389690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0032253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22389690&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pootakham</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jomchai</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruang-Areerate</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shearman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonthirod</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-wide SNP discovery and identification of QTL associated with agronomic traits in oil palm using genotyping-by-sequencing (GBS).</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">288</span>–<span class="cit-lpage">295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.volume%253D105%26rft.spage%253D288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-web" id="cit-7.3.881.33"><div class="cit-metadata unstructured">R Development Core Team, 2008 <em>R: A Language and Environment for Statistical Computing</em>. R Foundation for Statistical Computing, Vienna, Austria. Available at: <a href="http://www.R-project.org">http://www.R-project.org</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.34" data-doi="10.1016/j.virusres.2013.12.028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rabbi</span>  <span class="cit-name-given-names">I. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamblin</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gedil</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ikpan</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">High-resolution mapping of resistance to cassava mosaic geminiviruses in cassava using genotyping-by-sequencing and its implications for breeding.</span> <abbr class="cit-jnl-abbrev">Virus Res.</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BRes.%26rft.volume%253D186%26rft.spage%253D87%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2013.12.028%26rft_id%253Dinfo%253Apmid%252F24389096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2013.12.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24389096&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramos</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furuta</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uehara</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chihiro</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Angeles-Shim</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Development of chromosome segment substitution lines (CSSLs) of <em>Oryza longistaminata</em> A. Chev. &amp; Röhr in the background of the elite <em>japonica</em> rice cultivar, Taichung 65 and their evaluation for yield traits.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">210</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">163</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D210%26rft.spage%253D151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.36" data-doi="10.1016/S0168-9525(00)02024-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Longden</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bleasby</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">EMBOSS: the European molecular biology open software suite.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">276</span>–<span class="cit-lpage">277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DRice%26rft.auinit1%253DP.%26rft.volume%253D16%26rft.issue%253D6%26rft.spage%253D276%26rft.epage%253D277%26rft.atitle%253DEMBOSS%253A%2Bthe%2BEuropean%2BMolecular%2BBiology%2BOpen%2BSoftware%2BSuite.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252800%252902024-2%26rft_id%253Dinfo%253Apmid%252F10827456%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(00)02024-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10827456&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087457200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.37" data-doi="10.1186/gb-2013-14-6-r55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Romay</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Millard</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swarts</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comprehensive genotyping of the USA national maize inbred seed bank.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR55%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-6-r55%26rft_id%253Dinfo%253Apmid%252F23759205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-6-r55&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23759205&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rowan</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Rapid and inexpensive whole-genome genotyping-by-sequencing for crossover localization and fine-scale genetic mapping.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">385</span>–<span class="cit-lpage">398</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D5%26rft.spage%253D385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schröder</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mamidi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKain</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McClean</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Optimization of genotyping by sequencing (GBS) data in common bean (<em>Phaseolus vulgaris</em> L.).</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D36%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">Ś.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A statistical framework for quantitative trait mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">387</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSen%26rft.auinit1%253DS.%26rft.volume%253D159%26rft.issue%253D1%26rft.spage%253D371%26rft.epage%253D387%26rft.atitle%253DA%2BStatistical%2BFramework%2Bfor%2BQuantitative%2BTrait%2BMapping%26rft_id%253Dinfo%253Apmid%252F11560912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU5LzEvMzcxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8zLzg4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.41" data-doi="10.1111/pbi.12249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sonah</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Donoughue</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cober</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rajcan</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belzile</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Identification of loci governing eight agronomic traits using a GBS-GWAS approach and validation by QTL mapping in soya bean.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">211</span>–<span class="cit-lpage">221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D13%26rft.spage%253D211%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12249%26rft_id%253Dinfo%253Apmid%252F25213593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25213593&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.42" data-doi="10.1007/s00122-013-2166-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spindel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cobb</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gage</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Bridging the genotyping gap: using genotyping by sequencing (GBS) to add high-density SNP markers and new value to traditional bi-parental mapping and breeding populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">2699</span>–<span class="cit-lpage">2716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D2699%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2166-x%26rft_id%253Dinfo%253Apmid%252F23918062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2166-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23918062&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-web" id="cit-7.3.881.43"><div class="cit-metadata unstructured">Swarts, K., H. Li, J. A. Romero Navarro, D. An, M. C. Romay <em>et al.</em>, 2014 Novel methods to optimize genotypic imputation for low-coverage, next-generation sequence data in crop plants. Plant Genome 7. Available at: <a href="https://dl.sciencesocieties.org/publications/tpg/abstracts/7/3/plantgenome2014.05.0023">https://dl.sciencesocieties.org/publications/tpg/abstracts/7/3/plantgenome2014.05.0023</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.44" data-doi="10.1111/tpj.12105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takagi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abe</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshida</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kosugi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Natsume</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">QTL-seq: rapid mapping of quantitative trait loci in rice by whole genome resequencing of DNA from two bulked populations.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">174</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D74%26rft.spage%253D174%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.12105%26rft_id%253Dinfo%253Apmid%252F23289725%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.12105&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23289725&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316618500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ge</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">SNP deserts of Asian cultivated rice: genomic regions under domestication.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">751</span>–<span class="cit-lpage">761</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.stitle%253DJ%2BEvol%2BBiol%26rft.aulast%253DWang%26rft.auinit1%253DL.%26rft.volume%253D22%26rft.issue%253D4%26rft.spage%253D751%26rft.epage%253D761%26rft.atitle%253DSNP%2Bdeserts%2Bof%2BAsian%2Bcultivated%2Brice%253A%2Bgenomic%2Bregions%2Bunder%2Bdomestication.%26rft_id%253Dinfo%253Apmid%252F19243488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19243488&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.46" data-doi="10.1038/ng.3044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haberer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marri</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The genome sequence of African rice (<em>Oryza glaberrima</em>) and evidence for independent domestication.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">982</span>–<span class="cit-lpage">988</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D982%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3044%26rft_id%253Dinfo%253Apmid%252F25064006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25064006&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F3%2F881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.3.881.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome and comparative transcriptomics of African wild rice <em>Oryza longistaminata</em> provide insights into molecular mechanism of rhizomatousness and self-incompatibility.</span> <abbr class="cit-jnl-abbrev">Mol. Plant</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1683</span>–<span class="cit-lpage">1686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%26rft.volume%253D8%26rft.spage%253D1683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/3/881.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/3/871" title="De Novo Transcriptome Characterization of a Sterilizing Trematode Parasite (Microphallus sp.) from Two Species of New Zealand Snails" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/3/895" title="Predicted Residual Error Sum of Squares of Mixed Models: An Application for Genomic Prediction" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/3"><span>Volume 7 Issue 3, March 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/3.cover-source.jpg?itok=h8yj2GBo" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/3/881" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-mY29_jN5wiMbA0ZRpfOF5z0mwigp4Ny4mqYf-dgwllA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/3/881" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/3/881" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/3/881" class="active" data-icon-position="" data-hide-link-title="0">Adapting Genotyping-by-Sequencing for Rice F2 Populations</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/483523" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-tl4_W6-2kAQtN9rmgM8e6AQjU-3g4GIkzbcHJgH58IY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/3/881.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/3/881" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-jXwNbSm6Watrdbx0kyo67cezbNGCTmc5GIL54sdtQzg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/3/881" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/3/881.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483523" id="node483523--2" data-pisa="ggg;7/3/881" data-pisa-master="ggg;g3.116.038190" data-apath="/ggg/7/3/881.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Adapting Genotyping-by-Sequencing for Rice F2 Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Tomoyuki Furuta</span>, <span class="highwire-citation-author" data-delta="1">Motoyuki Ashikari</span>, <span class="highwire-citation-author" data-delta="2">Kshirod K. Jena</span>, <span class="highwire-citation-author" data-delta="3">Kazuyuki Doi</span> and <span class="highwire-citation-author" data-delta="4">Stefan Reuscher</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">881-893;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.038190 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483523--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Tomoyuki Furuta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioscience and Biotechnology Center, Nagoya University, 464-8601, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tomoyuki%2BFuruta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Furuta%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATomoyuki%2BFuruta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Motoyuki Ashikari </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioscience and Biotechnology Center, Nagoya University, 464-8601, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Motoyuki%2BAshikari%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ashikari%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMotoyuki%2BAshikari%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kshirod K. Jena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant Breeding Division, International Rice Research Institute, 1301 Manila, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kshirod%2BK.%2BJena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jena%20KK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKshirod%2BK.%2BJena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kazuyuki Doi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Associated Field Science and Research Center, Nagoya University, 470-0151, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kazuyuki%2BDoi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Doi%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKazuyuki%2BDoi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kdoi@agr.nagoya-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">kdoi@agr.nagoya-u.ac.jp</a>
<a href="mailto:reuscher@agr.nagoya-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">reuscher@agr.nagoya-u.ac.jp</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Stefan Reuscher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioscience and Biotechnology Center, Nagoya University, 464-8601, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BReuscher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reuscher%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStefan%2BReuscher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kdoi@agr.nagoya-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">kdoi@agr.nagoya-u.ac.jp</a>
<a href="mailto:reuscher@agr.nagoya-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">reuscher@agr.nagoya-u.ac.jp</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/3/881" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/3/881&amp;title=Adapting%20Genotyping-by-Sequencing%20for%20Rice%20F2%20Populations" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/3/881&amp;title=Adapting%20Genotyping-by-Sequencing%20for%20Rice%20F2%20Populations" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/3/881&amp;title=Adapting%20Genotyping-by-Sequencing%20for%20Rice%20F2%20Populations" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/3/881&amp;text=Adapting%20Genotyping-by-Sequencing%20for%20Rice%20F2%20Populations" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/3/881&amp;title=Adapting%20Genotyping-by-Sequencing%20for%20Rice%20F2%20Populations" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/3/881&amp;t=Adapting%20Genotyping-by-Sequencing%20for%20Rice%20F2%20Populations" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/3/881&amp;title=Adapting%20Genotyping-by-Sequencing%20for%20Rice%20F2%20Populations" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/3/881&amp;title=Adapting%20Genotyping-by-Sequencing%20for%20Rice%20F2%20Populations" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/483523/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="483523" id="node483523--3" data-pisa="ggg;7/3/881" data-pisa-master="ggg;g3.116.038190" data-apath="/ggg/7/3/881.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Adapting Genotyping-by-Sequencing for Rice F2 Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Tomoyuki Furuta</span>, <span class="highwire-citation-author" data-delta="1">Motoyuki Ashikari</span>, <span class="highwire-citation-author" data-delta="2">Kshirod K. Jena</span>, <span class="highwire-citation-author" data-delta="3">Kazuyuki Doi</span> and <span class="highwire-citation-author" data-delta="4">Stefan Reuscher</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">881-893;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.038190 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node483523--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Tomoyuki Furuta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioscience and Biotechnology Center, Nagoya University, 464-8601, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tomoyuki%2BFuruta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Furuta%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATomoyuki%2BFuruta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Motoyuki Ashikari </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioscience and Biotechnology Center, Nagoya University, 464-8601, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Motoyuki%2BAshikari%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ashikari%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMotoyuki%2BAshikari%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kshirod K. Jena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant Breeding Division, International Rice Research Institute, 1301 Manila, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kshirod%2BK.%2BJena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jena%20KK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKshirod%2BK.%2BJena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kazuyuki Doi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Associated Field Science and Research Center, Nagoya University, 470-0151, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kazuyuki%2BDoi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Doi%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKazuyuki%2BDoi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kdoi@agr.nagoya-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">kdoi@agr.nagoya-u.ac.jp</a>
<a href="mailto:reuscher@agr.nagoya-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">reuscher@agr.nagoya-u.ac.jp</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Stefan Reuscher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioscience and Biotechnology Center, Nagoya University, 464-8601, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BReuscher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reuscher%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStefan%2BReuscher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kdoi@agr.nagoya-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">kdoi@agr.nagoya-u.ac.jp</a>
<a href="mailto:reuscher@agr.nagoya-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">reuscher@agr.nagoya-u.ac.jp</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=881&amp;rft.epage=893&amp;rft.atitle=Adapting%20Genotyping-by-Sequencing%20for%20Rice%20F2%20Populations&amp;rft.volume=7&amp;rft.issue=3&amp;rft.date=2017-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Furuta%2C+Tomoyuki&amp;rft.au=Ashikari%2C+Motoyuki&amp;rft.au=Jena%2C+Kshirod+K.&amp;rft.au=Doi%2C+Kazuyuki&amp;rft.au=Reuscher%2C+Stefan"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/483523/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/483523/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/483523/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/483523/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/483523/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/483523/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/483523/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/483523/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/483523/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/483523/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/483523/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/483523/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/3/881" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/3/881.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/3/881.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/3/881.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/3/881.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
