<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/6/1253" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/6/1253.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/6/1253.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/6/1253.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/6/1253.atom" />
<meta name="HW.pisa" content="ggg;5/6/1253" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Improved Detection and Characterization of Copy Number Variations Among Diverse Pig Breeds by Array CGH" />
<meta name="DC.Identifier" content="10.1534/g3.115.018473" />
<meta name="DC.Date" content="2015-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Wang et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="As a major component of genomic variation, copy number variations (CNVs) are considered as promising markers for some phenotypic and economically important traits in domestic animals. Using a custom-designed 1M array CGH (aCGH), we performed CNV discovery in 12 pig samples from one Asian wild boar population, six Chinese indigenous breeds, and two European commercial breeds. In total, we identified 758 CNV regions (CNVRs), covering 47.43 Mb of the pig genome sequence. Of the total porcine genes, 1295 genes were completely or partially overlapped with the identified CNVRs, which enriched in the terms related to sensory perception of the environment, neurodevelopmental processes, response to external stimuli, and immunity. Further probing the potential functions of these genes, we also found a suite of genes related important traits, which make them a promising resource for exploring the genetic basis of phenotype differences among diverse pig breeds. Compared with previous relevant studies, the current study highlights that different platforms can complement each other, and the combined implementation of different platforms is beneficial to achieve the most comprehensive CNV calls. CNVs detected in diverse populations herein are essentially complementary to the CNV map in the pig genome, which would be helpful for understanding the pig genome variants and investigating the associations between various phenotypes and CNVs." />
<meta name="DC.Contributor" content="Jiying Wang" />
<meta name="DC.Contributor" content="Jicai Jiang" />
<meta name="DC.Contributor" content="Haifei Wang" />
<meta name="DC.Contributor" content="Huimin Kang" />
<meta name="DC.Contributor" content="Qin Zhang" />
<meta name="DC.Contributor" content="Jian-Feng Liu" />
<meta name="article:published_time" content="2015-06-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Improved Detection and Characterization of Copy Number Variations Among Diverse Pig Breeds by Array CGH" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;As a major component of genomic variation, copy number variations (CNVs) are considered as promising markers for some phenotypic and economically important traits in domestic animals. Using a custom-designed 1M array CGH (aCGH), we performed CNV discovery in 12 pig samples from one Asian wild boar population, six Chinese indigenous breeds, and two European commercial breeds. In total, we identified 758 CNV regions (CNVRs), covering 47.43 Mb of the pig genome sequence. Of the total porcine genes, 1295 genes were completely or partially overlapped with the identified CNVRs, which enriched in the terms related to sensory perception of the environment, neurodevelopmental processes, response to external stimuli, and immunity. Further probing the potential functions of these genes, we also found a suite of genes related important traits, which make them a promising resource for exploring the genetic basis of phenotype differences among diverse pig breeds. Compared with previous relevant studies, the current study highlights that different platforms can complement each other, and the combined implementation of different platforms is beneficial to achieve the most comprehensive CNV calls. CNVs detected in diverse populations herein are essentially complementary to the CNV map in the pig genome, which would be helpful for understanding the pig genome variants and investigating the associations between various phenotypes and CNVs.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/06/01" />
<meta name="citation_mjid" content="ggg;5/6/1253" />
<meta name="citation_id" content="5/6/1253" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/6/1253" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/6/1253.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/6/1253.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/6/1253.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.018473" />
<meta name="citation_pmid" content="25908567" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1253" />
<meta name="citation_lastpage" content="1261" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jiying Wang" />
<meta name="citation_author_institution" content="Key Laboratory of Animal Genetics, Breeding, and Reproduction, Ministry of Agriculture, National Engineering Laboratory for Animal Breeding, College of Animal Science and Technology, China Agricultural University, Beijing 100193, China" />
<meta name="citation_author_institution" content="Shandong Provincial Key Laboratory of Animal Disease Control and Breeding, Institute of Animal Science and Veterinary Medicine, Shandong Academy of Agricultural Sciences, Jinan 250100, China" />
<meta name="citation_author" content="Jicai Jiang" />
<meta name="citation_author_institution" content="Key Laboratory of Animal Genetics, Breeding, and Reproduction, Ministry of Agriculture, National Engineering Laboratory for Animal Breeding, College of Animal Science and Technology, China Agricultural University, Beijing 100193, China" />
<meta name="citation_author" content="Haifei Wang" />
<meta name="citation_author_institution" content="Key Laboratory of Animal Genetics, Breeding, and Reproduction, Ministry of Agriculture, National Engineering Laboratory for Animal Breeding, College of Animal Science and Technology, China Agricultural University, Beijing 100193, China" />
<meta name="citation_author" content="Huimin Kang" />
<meta name="citation_author_institution" content="Key Laboratory of Animal Genetics, Breeding, and Reproduction, Ministry of Agriculture, National Engineering Laboratory for Animal Breeding, College of Animal Science and Technology, China Agricultural University, Beijing 100193, China" />
<meta name="citation_author" content="Qin Zhang" />
<meta name="citation_author_institution" content="Key Laboratory of Animal Genetics, Breeding, and Reproduction, Ministry of Agriculture, National Engineering Laboratory for Animal Breeding, College of Animal Science and Technology, China Agricultural University, Beijing 100193, China" />
<meta name="citation_author" content="Jian-Feng Liu" />
<meta name="citation_author_institution" content="Key Laboratory of Animal Genetics, Breeding, and Reproduction, Ministry of Agriculture, National Engineering Laboratory for Animal Breeding, College of Animal Science and Technology, China Agricultural University, Beijing 100193, China" />
<meta name="citation_author_email" content="liujf@cau.edu.cn" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. Alkan;citation_author=J. M. Kidd;citation_author=T. Marques-Bonet;citation_author=G. Aksay;citation_author=F. Antonacci;citation_title=Personalized copy number and segmental duplication maps using next-generation sequencing.;citation_pages=1061-1067;citation_volume=41;citation_year=2009;citation_issue=10;citation_pmid=19718026;citation_doi=10.1038/ng.437" />
<meta name="citation_reference" content="citation_journal_title=BioTechniques;citation_journal_abbrev=Biotechniques;citation_author=M. Ballester;citation_author=A. Castelló;citation_author=E. Ibáñez;citation_author=A. Sánchez;citation_author=J. M. Folch;citation_title=Real-time quantitative PCR-based system for determining transgene copy number in transgenic animals.;citation_pages=610-613;citation_volume=37;citation_year=2004;citation_issue=4;citation_pmid=15517974" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=N. P. Carter;citation_title=Methods and strategies for analyzing copy number variation using DNA microarrays.;citation_pages=S16-S21;citation_volume=39;citation_year=2007;citation_issue=7 Suppl;citation_pmid=17597776;citation_doi=10.1038/ng2028" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=C. Chen;citation_author=R. Qiao;citation_author=R. Wei;citation_author=Y. Guo;citation_author=H. Ai;citation_title=A comprehensive survey of copy number variation in 18 diverse pig populations and identification of candidate copy number variable genes associated with complex traits.;citation_pages=733;citation_volume=13;citation_year=2012;citation_pmid=23270433;citation_doi=10.1186/1471-2164-13-733" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=A. Clop;citation_author=O. Vidal;citation_author=M. Amills;citation_title=Copy number variation in the genomes of domestic animals.;citation_pages=503-517;citation_volume=43;citation_year=2012;citation_issue=5;citation_pmid=22497594;citation_doi=10.1111/j.1365-2052.2012.02317.x" />
<meta name="citation_reference" content="citation_journal_title=Cytogenetic and genome research;citation_journal_abbrev=Cytogenet Genome Res;citation_author=A. De Smith;citation_author=R. Walters;citation_author=P. Froguel;citation_author=A. I. Blakemore;citation_title=Human genes involved in copy number variation: mechanisms of origin, functional effects and implications for disease.;citation_pages=17-26;citation_volume=123;citation_year=2008;citation_issue=1-4;citation_pmid=19287135;citation_doi=10.1159/000184688" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=J. J. Emerson;citation_author=M. Cardoso-Moreira;citation_author=J. O. Borevitz;citation_author=M. Long;citation_title=Natural selection shapes genome-wide patterns of copy-number polymorphism in Drosophila melanogaster.;citation_volume=320;citation_year=2008;citation_pmid=18535209;citation_doi=10.1126/science.1158078" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. Fadista;citation_author=M. Nygaard;citation_author=L. E. Holm;citation_author=B. Thomsen;citation_author=C. Bendixen;citation_title=A snapshot of CNVs in the pig genome.;citation_pages=e3916-e3916;citation_volume=3;citation_year=2008;citation_issue=12;citation_pmid=19079605;citation_doi=10.1371/journal.pone.0003916" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. Fadista;citation_author=B. Thomsen;citation_author=L. E. Holm;citation_author=C. Bendixen;citation_title=Copy number variation in the bovine genome.;citation_pages=284-284;citation_volume=11;citation_year=2010;citation_pmid=20459598;citation_doi=10.1186/1471-2164-11-284" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=M. Fang;citation_author=X. Hu;citation_author=T. Jiang;citation_author=M. Braunschweig;citation_author=L. Hu;citation_title=The phylogeny of Chinese indigenous pig breeds inferred from microsatellite markers.;citation_pages=7-13;citation_volume=36;citation_year=2005;citation_issue=1;citation_pmid=15670125;citation_doi=10.1111/j.1365-2052.2004.01234.x" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=L. Feuk;citation_author=A. R. Carson;citation_author=S. W. Scherer;citation_title=Structural variation in the human genome.;citation_pages=85-97;citation_volume=7;citation_year=2006;citation_issue=2;citation_pmid=16418744;citation_doi=10.1038/nrg1767" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=E. Gazave;citation_author=F. Darré;citation_author=C. Morcillo-Suarez;citation_author=N. Petit-Marty;citation_author=A. Carreño;citation_title=Copy number variation analysis in the great apes reveals species-specific patterns of structural variation.;citation_volume=21;citation_year=2011;citation_pmid=21824994;citation_doi=10.1101/gr.117242.110" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=E. Giuffra;citation_author=A. Törnsten;citation_author=S. Marklund;citation_author=E. Bongcam-Rudloff;citation_author=P. Chardon;citation_title=A large duplication associated with dominant white color in pigs originated by homologous recombination between LINE elements flanking KIT.;citation_pages=569-577;citation_volume=13;citation_year=2002;citation_issue=10;citation_pmid=12420135;citation_doi=10.1007/s00335-002-2184-5" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. R. Haraksingh;citation_author=A. Abyzov;citation_author=M. Gerstein;citation_author=A. E. Urban;citation_title=Genome-wide mapping of copy number variation in humans: comparative analysis of high resolution array platforms.;citation_pages=e27859-e27859;citation_volume=6;citation_year=2011;citation_issue=11;citation_pmid=22140474;citation_doi=10.1371/journal.pone.0027859" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=Y. Hou;citation_author=G. E. Liu;citation_author=D. M. Bickhart;citation_author=M. F. Cardone;citation_author=K. Wang;citation_title=Genomic characteristics of cattle copy number variations.;citation_pages=127-127;citation_volume=12;citation_year=2011;citation_pmid=21345189;citation_doi=10.1186/1471-2164-12-127" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. J. Iafrate;citation_author=L. Feuk;citation_author=M. N. Rivera;citation_author=M. L. Listewnik;citation_author=P. K. Donahoe;citation_title=Detection of large-scale variation in the human genome.;citation_pages=949-951;citation_volume=36;citation_year=2004;citation_issue=9;citation_pmid=15286789;citation_doi=10.1038/ng1416" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. Jiang;citation_author=J. Wang;citation_author=H. Wang;citation_author=Y. Zhang;citation_author=H. Kang;citation_title=Global copy number analyses by next generation sequencing provide insight into pig genome.;citation_pages=593;citation_volume=15;citation_year=2014;citation_pmid=25023178;citation_doi=10.1186/1471-2164-15-593" />
<meta name="citation_reference" content="citation_journal_title=Anim. Gent.;citation_author=A. Johansson;citation_author=G. Pielberg;citation_author=L. Anderson;citation_author=I. Edfors-lilja;citation_title=Polymorphism at the porcine dominant white/KIT locus influence coat colour and peripheral blood cell measures.;citation_pages=288-296;citation_volume=36;citation_year=2005;citation_doi=10.1111/j.1365-2052.2005.01320.x" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=Y. Li;citation_author=S. Mei;citation_author=X. Zhang;citation_author=X. Peng;citation_author=G. Liu;citation_title=Identification of genome-wide copy number variations among diverse pig breeds by array CGH.;citation_pages=725;citation_volume=13;citation_year=2012;citation_pmid=23265576;citation_doi=10.1186/1471-2164-13-725" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=G. E. Liu;citation_author=Y. Hou;citation_author=B. Zhu;citation_author=M. F. Cardone;citation_author=L. Jiang;citation_title=Analysis of copy number variations among diverse cattle breeds.;citation_volume=20;citation_year=2010;citation_pmid=20212021;citation_doi=10.1101/gr.105403.110" />
<meta name="citation_reference" content="citation_journal_title=Methods (San Diego, Calif.);citation_journal_abbrev=Methods;citation_author=K. J. Livak;citation_author=T. D. Schmittgen;citation_title=Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method.;citation_pages=402-408;citation_volume=25;citation_year=2001;citation_issue=4;citation_pmid=11846609;citation_doi=10.1006/meth.2001.1262" />
<meta name="citation_reference" content="citation_journal_title=Trends in microbiology;citation_journal_abbrev=Trends Microbiol;citation_author=F. Meurens;citation_author=A. Summerfield;citation_author=H. Nauwynck;citation_author=L. Saif;citation_author=V. Gerdts;citation_title=The pig: a model for human infectious diseases.;citation_pages=50-57;citation_volume=20;citation_year=2012;citation_issue=1;citation_pmid=22153753;citation_doi=10.1016/j.tim.2011.11.002" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. E. Mills;citation_author=K. Walter;citation_author=C. Stewart;citation_author=R. E. Handsaker;citation_author=K. Chen;citation_title=Mapping copy number variation by population-scale genome sequencing.;citation_pages=59-65;citation_volume=470;citation_year=2011;citation_issue=7332;citation_pmid=21293372;citation_doi=10.1038/nature09708" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=Y. Paudel;citation_author=O. Madsen;citation_author=H. Megens;citation_author=L. A. Frantz;citation_author=M. Bosse;citation_title=Evolutionary dynamics of copy number variation in pig genomes in the context of adaptation and domestication.;citation_pages=449;citation_volume=14;citation_year=2013;citation_pmid=23829399;citation_doi=10.1186/1471-2164-14-449" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. Pielberg;citation_author=C. Olsson;citation_author=A. C. Syvänen;citation_author=L. Andersson;citation_title=Unexpectedly High Allelic Diversity at the KIT Locus Causing Dominant White Color in the Domestic Pig;citation_pages=305-311;citation_volume=160;citation_year=2002;citation_issue=1;citation_pmid=11805065" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. Pinkel;citation_author=R. Segraves;citation_author=D. Sudar;citation_author=S. Clark;citation_author=I. Poole;citation_title=High resolution analysis of DNA copy number variation using comparative genomic hybridization to microarrays.;citation_pages=207-211;citation_volume=20;citation_year=1998;citation_issue=2;citation_pmid=9771718;citation_doi=10.1038/2524" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=D. Pinto;citation_author=K. Darvishi;citation_author=X. Shi;citation_author=D. Rajan;citation_author=D. Rigler;citation_title=Comprehensive assessment of array-based platforms and calling algorithms for detection of copy number variants.;citation_pages=512-520;citation_volume=29;citation_year=2011;citation_issue=6;citation_pmid=21552272;citation_doi=10.1038/nbt.1852" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=Y. Ramayo-Caldas;citation_author=A. Castelló;citation_author=R. N. Pena;citation_author=E. Alves;citation_author=A. Mercadé;citation_title=Copy number variation in the porcine genome inferred from a 60 k SNP BeadChip.;citation_pages=593-593;citation_volume=11;citation_year=2010;citation_pmid=20969757;citation_doi=10.1186/1471-2164-11-593" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. Redon;citation_author=S. Ishikawa;citation_author=K. R. Fitch;citation_author=L. Feuk;citation_author=G. H. Perry;citation_title=Global variation in copy number in the human genome.;citation_pages=444-454;citation_volume=444;citation_year=2006;citation_issue=7118;citation_pmid=17122850;citation_doi=10.1038/nature05329" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=G. Rosengren Pielberg;citation_author=A. Golovko;citation_author=E. Sundström;citation_author=I. Curik;citation_author=J. Lennartsson;citation_title=A cis-acting regulatory mutation causes premature hair graying and susceptibility to melanoma in the horse.;citation_pages=1004-1009;citation_volume=40;citation_year=2008;citation_issue=8;citation_pmid=18641652;citation_doi=10.1038/ng.185" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. J. Rubin;citation_author=H. J. Megens;citation_author=A. M. Barrio;citation_author=K. Maqbool;citation_author=S. Sayyab;citation_title=Strong signatures of selection in the domestic pig genome.;citation_volume=109;citation_year=2012;citation_pmid=23151514;citation_doi=10.1073/pnas.1217149109" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=D. R. Schrider;citation_author=M. W. Hahn;citation_title=Gene copy-number polymorphism in nature.;citation_volume=277;citation_year=2010;citation_pmid=20591863;citation_doi=10.1098/rspb.2010.1180" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Sebat;citation_author=B. Lakshmi;citation_author=J. Troge;citation_author=J. Alexander;citation_author=J. Young;citation_title=Large-Scale Copy Number Polymorphism in the Human Genome;citation_pages=525-528;citation_volume=305;citation_year=2004;citation_issue=5683;citation_pmid=15273396;citation_doi=10.1126/science.1098918" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=E. Seroussi;citation_author=G. Glick;citation_author=A. Shirak;citation_author=E. Yakobson;citation_author=J. I. Weller;citation_title=Analysis of copy loss and gain variations in Holstein cattle autosomes using BeadChip SNPs.;citation_pages=673-673;citation_volume=11;citation_year=2010;citation_pmid=21114805;citation_doi=10.1186/1471-2164-11-673" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. H. Sudmant;citation_author=J. O. Kitzman;citation_author=F. Antonacci;citation_author=C. Alkan;citation_author=M. Malig;citation_title=Diversity of Human Copy Number Variation and Multicopy Genes;citation_pages=641-646;citation_volume=330;citation_year=2010;citation_issue=6004;citation_pmid=21030649;citation_doi=10.1126/science.1197005" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. Suzuki;citation_author=H. Shimodaira;citation_title=Pvclust: an R package for assessing the uncertainty in hierarchical clustering.;citation_volume=22;citation_year=2006;citation_pmid=16595560;citation_doi=10.1093/bioinformatics/btl117" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=M. Tian;citation_author=Y. Wang;citation_author=X. Gu;citation_author=C. Feng;citation_author=S. Fang;citation_title=Copy number variants in locally raised Chinese chicken genomes determined using array comparative genomic hybridization.;citation_pages=262;citation_volume=14;citation_year=2013;citation_pmid=23594354;citation_doi=10.1186/1471-2164-14-262" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=E. Tuzun;citation_author=A. J. Sharp;citation_author=J. A. Bailey;citation_author=R. Kaul;citation_author=V. A. Morrison;citation_title=Fine-scale structural variation of the human genome.;citation_pages=727-732;citation_volume=37;citation_year=2005;citation_issue=7;citation_pmid=15895083;citation_doi=10.1038/ng1562" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. J. Vilella;citation_author=J. Severin;citation_author=A. Ureta-Vidal;citation_author=L. Heng;citation_author=R. Durbin;citation_title=Ensembl Compara GeneTrees: Complete, duplication-aware phylogenetic trees in vertebrates.;citation_volume=19;citation_year=2009;citation_pmid=19029536;citation_doi=10.1101/gr.073585.107" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. Wang;citation_author=J. Jiang;citation_author=W. Fu;citation_author=L. Jiang;citation_author=X. Ding;citation_title=A genome-wide detection of copy number variations using SNP genotyping arrays in swine.;citation_pages=273-273;citation_volume=13;citation_year=2012;citation_pmid=22726314;citation_doi=10.1186/1471-2164-13-273" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=J. Wang;citation_author=H. Wang;citation_author=J. Jiang;citation_author=H. Kang;citation_author=X. Feng;citation_title=Identification of Genome-Wide Copy Number Variations among Diverse Pig Breeds Using SNP Genotyping Arrays.;citation_pages=e68683;citation_volume=8;citation_year=2013;citation_pmid=23935880;citation_doi=10.1371/journal.pone.0068683" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=J. Wang;citation_author=J. Jiang;citation_author=H. Wang;citation_author=H. Kang;citation_author=Q. Zhang;citation_title=Enhancing Genome-Wide Copy Number Variation Identification by High Density Array CGH Using Diverse Resources of Pig Breeds.;citation_pages=e87571;citation_volume=9;citation_year=2014;citation_pmid=24475311;citation_doi=10.1371/journal.pone.0087571" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=D. Wright;citation_author=H. Boije;citation_author=J. R. S. Meadows;citation_author=B. Bed’hom;citation_author=D. Gourichon;citation_title=Copy number variation in intron 1 of SOX5 causes the Pea-comb phenotype in chickens.;citation_pages=e1000512-e1000512;citation_volume=5;citation_year=2009;citation_issue=6;citation_pmid=19521496;citation_doi=10.1371/journal.pgen.1000512" />
<meta name="citation_reference" content="citation_journal_abbrev=Curr Genomics;citation_author=C. Zhang;citation_author=G. Plastow;citation_title=Genomic Diversity in Pig (Sus scrofa) and its Comparison with Human and other Livestock.;citation_pages=138-146;citation_volume=12;citation_year=2011;citation_issue=2;citation_pmid=21966252;citation_doi=10.2174/138920211795564386" />
<meta name="citation_reference" content="citation_title=Pig Breeds in China;citation_author=Z. G. Zhang;citation_author=B. Li;citation_author=X. Chen;citation_year=1986" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Improved Detection and Characterization of Copy Number Variations Among Diverse Pig Breeds by Array CGH" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
<meta name="twitter:description" content="As a major component of genomic variation, copy number variations (CNVs) are considered as promising markers for some phenotypic and economically important traits in domestic animals. Using a custom-designed 1M array CGH (aCGH), we performed CNV discovery in 12 pig samples from one Asian wild boar population, six Chinese indigenous breeds, and two European commercial breeds. In total, we identified 758 CNV regions (CNVRs), covering 47.43 Mb of the pig genome sequence. Of the total porcine genes, 1295 genes were completely or partially overlapped with the identified CNVRs, which enriched in the terms related to sensory perception of the environment, neurodevelopmental processes, response to external stimuli, and immunity. Further probing the potential functions of these genes, we also found a suite of genes related important traits, which make them a promising resource for exploring the genetic basis of phenotype differences among diverse pig breeds. Compared with previous relevant studies, the current study highlights that different platforms can complement each other, and the combined implementation of different platforms is beneficial to achieve the most comprehensive CNV calls. CNVs detected in diverse populations herein are essentially complementary to the CNV map in the pig genome, which would be helpful for understanding the pig genome variants and investigating the associations between various phenotypes and CNVs." />
<meta name="og-title" property="og:title" content="Improved Detection and Characterization of Copy Number Variations Among Diverse Pig Breeds by Array CGH" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/6/1253" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="As a major component of genomic variation, copy number variations (CNVs) are considered as promising markers for some phenotypic and economically important traits in domestic animals. Using a custom-designed 1M array CGH (aCGH), we performed CNV discovery in 12 pig samples from one Asian wild boar population, six Chinese indigenous breeds, and two European commercial breeds. In total, we identified 758 CNV regions (CNVRs), covering 47.43 Mb of the pig genome sequence. Of the total porcine genes, 1295 genes were completely or partially overlapped with the identified CNVRs, which enriched in the terms related to sensory perception of the environment, neurodevelopmental processes, response to external stimuli, and immunity. Further probing the potential functions of these genes, we also found a suite of genes related important traits, which make them a promising resource for exploring the genetic basis of phenotype differences among diverse pig breeds. Compared with previous relevant studies, the current study highlights that different platforms can complement each other, and the combined implementation of different platforms is beneficial to achieve the most comprehensive CNV calls. CNVs detected in diverse populations herein are essentially complementary to the CNV map in the pig genome, which would be helpful for understanding the pig genome variants and investigating the associations between various phenotypes and CNVs." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
    <title>Improved Detection and Characterization of Copy Number Variations Among Diverse Pig Breeds by Array CGH | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"LKntiqt5OeFFs_EOoV6JwVvSG8dKOkFoD2cUD29CPZY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459919","apath":"\/ggg\/5\/6\/1253.atom","pisa":"ggg;5\/6\/1253","ac":{"\/ggg\/5\/6\/1253.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/6\/1253.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/6\/1253"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/6\/1253"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/6\/1253.full":true,"\/content\/5\/6\/1253":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022JLYFZIth638eYerdkBuCfV\\\\\\\/gYQGPtRyeKALxdHJCVrIGJ7wKqtaNQBCSQ1sqM+jAxitEWvBvFBv2C71qQq6GiT+MasVLQYFvj5+RA3Cw8XHQpoAK+wqgBkNaecoiGUKcjYDKy0FqVD8KX\\\\\\\/A+GMnvWQAJw5UQiR+VTFhAYQ96M2ixtc86UQ\\\\\\\/82rGv5QBHzAhYeOeYwaJT1DbNmH3aSfPvG8Djp7bGNe20w7Zq10OcfRsswxf1PRhVXq3PmGQ0WNelLnVWEnc7EyvcwwZpI+PmMfXexlvlRkjAWcDJbLFhN7nJUOuesrK2WsfTPreXFDycXoXgm3VaJYbqWiQ5KRoEGzou8eM0+mcfKoK0VX6rqLb5O3diH4cLbzd\\\\\\\/dlKSWChMYOVGMqUx2OXW4lTublmJctcqs9E7qEFe8WmxnkAoUDZiaKOeCClij6hIlsUjA00U+7+mUErwvdgmt738k3XSbR8uxUp6LNOSUONJgGbuR1a5vNDCKxU2edyWDp+cGAwE4sIL070dT5Klf1WUtyKiVuD17R+14O9k12Dv9IOYBNEO0RAYul9joJc3biazAJHvWOmo0AyrpyQkWlqnxAT19LiRFTLHC36di17HQ\\\\\\\/2yyy16AsNnTl+3Bfge5xyhwbBLYBvi3GOBgwAyF6AbbXzUrj7iX+xrD\\\\\\\/ZXNd3NRX59u4vYgqK53wI06RBvK2b34C0qls73DcxInM7eHBsCJCLYW2vlt01nq+IbzcRL6bbbaKfIaxSOk7E\\\\\\\/c6bVcnfoiGpJZFpAunSs2gfZ54MppXXFBNMzQoT97Dwul2QW3c9lfSKfPqKnBknyWT+wN3vHgxN1mZMMo4B1MOQx0xbh\\\\\\\/C6gxz8iQEQ3zPMyA1b3+UIYxt0X2A5qVnnzIf2W+UkD79CO07lFEnqgvLbqn7AkE8p7OuMSP2P5kEUoFEw6\\\\\\\/2pFAjijr\\\\\\\/5p0gjEMZkuHfH432MT4OQKDPRn8tp4duLKaSIwCjW\\\\\\\/7KmyO8bEeDUaFiaq0OcNOX+EY7Z4piiWVChzA24tn\\\\\\\/4zH\\\\\\\/A28Noq4re1MgUz3YKi1zTJl3XvJvm0VDa\\\\\\\/71AKAY518NjV7RxXZv419lyH8\\\\\\\/NKI0Ln2RQvKsWLCRDfketxfKGg3X33KxCwYy77CfNtX3VaHgr8Yzhh+NJFcrhaq+CTUggApkXcIX6Zb00dE5ZA1nLbvatWMGfinHyVjhSZlkMXUVSqgJwiWh16jyn6M1YbLMgzqXrvzra5fE6PkLK5AFyE\\\\\\\/zbRiVIV8uI\\\\\\\/eZjXlKELWn8CC8mUrPb6u1r8\\\u0022,\\\u0022iv\\\u0022:\\\u0022429BWgV5SqlHYTUSAkA5dA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222135055a31269d840f5e57c5dbf6ff08ec898ddee6fa15e122f8b079b508f47f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/5YLximrz1SnehNERWWEIkR4f04jg0urJTBcwaOmL7mw563+64OMizdqDHDsX6oZM8dbOW8I0zao7WnjlvOVrrF1\\\\\\\/HR1mx2eZkL+a35GqYsvT2oFeKZaaV6LYqEbayKwY60L+Bt0iMxwXmdFVpD9uJlWoIeG0dkQaTb+4rvgzZsYa1Ya9y70+LMKLBEUI7QKftlKeHSXw\\\\\\\/whDdB7lz9RuPM6ZMw6XbREgOSJ4HNbI4X4BvZdc2G09E9jkhgN5ecad5Uj4i7uYQCptlDnB5JxVQB1cczrF4E2KgDMI3zVxzwH3o0KdMfOMUG09Qn2ucJzdUlks05qlxTR4yIJOQJfIXBphIV8GhCFKY8jS+\\\\\\\/ylByQivp1KMuYed7+bfmJsfKlbiMfU2NvtfvHIOmr8jcqq\\\\\\\/GAV8kFjyTX0vBp5xL1l84lJ3G7NOafEI8tzDtEVZAd9EEB1XsIdKCpF8DwUTRw0HenL5Gq0A29XdcdQCfDnYt9hZkdMEcTeGOmgX5neL1+gHHiNWKYV00O\\\\\\\/99hWvAL2yWX9dz2SYDyedQnt5nthQHQ1kNKAECLXpy5ymnwePKYdxore8mEGNOv9m\\\\\\\/jLv8i6pPgrq5TMVqjUBbaAmJEui4OqS27E51uyHultY9uVdPickCftilR4ZEqrK+cYFiLxgRAOmLXnzfEhcH\\\\\\\/vnzj7tTKskRtVEotSZwqzugejBmLG0NjQHpSD1PVOt7vf49toLa7w\\\\\\\/h8qFHhKWH0IOefFZ0leVyafxIqoAQpv0rmO6uAV44sSh3XGHgp0Y6GNLZ056\\\\\\\/LJc7Q0JW5qi5p\\\\\\\/lNv6QdlkuqWHs6diUHoW5gHsZyO3LDd+T9anmU5BRA8fO+Augv1cLcmUNjUNHdEYKrHlln824m9IncBGImXTT1imIVwb+sXUlc8lOpjP0tpQYpZJz45KKXiPn0IZa0nZrPkQV5EuxH6hE6qZecRH17Wzrk3nIOXFktw3f83MUZIGs+Ya0pY6XXP7Ic0rT3d916WvFBvHSFzSTPTmGLfNmlxcG4cuhjzH3NTGOQ1C9gTyYpykgayCrsLQgKD+2jJMbnit8bHNQABhMx+d8qZgkJB0Kbe08ayzETBsbDY62UctEGIDtB1wwWnK6oXVHbhljc8eQi7WnAwdAlB4s7zWh6vLLkj6LWGbByg000zfO31jOXBiP7UemRaYzlHzE4rKgIPj71vbpMonJ1LlRYTvWZ9TyV\\\\\\\/UXUW7Ahr8bqyMb8wwE6abbz86HiL62CEymeMlUJ78qcqbTV4+l5WEjTwIjbjkrXlzd3GVFPfSvSV6nb8+7JnJ\\\\\\\/4ACP9pDEhQ3R97Up9ZW2FA7XaD5D5Zd5mkLAvsc9OXUqo7qGvTCrZADplo0V1L9QnGEgqiaXMjCX3ATI9FzLrIasaEH0KGqIT9x5QC5loncKGAytEikT+cox2rTrD1Vu+ff2kJQcxNiWYIWO+EoZReFkZp\\\\\\\/sXJBpwo6MAoq48gJCeJ\\\\\\\/CFSdMJ1Q50MqInYICc1Jc4dIvah+YCdNNqgyV2bLEBZUi8acFqy0WaE2+Q2XkyqTL\\\\\\\/SHhfL7WcifQYn5JrSSMY0z16H+bTN\\\\\\\/WIl3o28EdCypsaVDNuQd7kBoELc20g\\\\\\\/46FUf3DlAWbE\\\\\\\/VXoTUQBR7GL1lCz4TWkR\\\\\\\/QZv16BnuBTmXMUmxOjiw7Rw\\\\\\\/6rNtnAXUUEDQkT3PhgLNw14bmFR+e8BLXs7HFHaJomCS5cnkJqvfTVSTIOwvgCUoecSkTPhqUgm45Ri\\\\\\\/tBP8MZJ6z1QmAa2IHsgZZQn3pNiahIMoV1EtMhdd+IuQyN2Ko\\\\\\\/3zvH33QD0N9qq+AD6+VyXGugJcRAPxom91p0rngvF6MbK0wyf4n50KPSU1\\\\\\\/47TyOGyJUeZpTuSvIbljvLIrU5jeoiMobjjcVQ2MmYDk7dnZSloqd5iUoGHO7mbQRFh\\\\\\\/kqjBc1qjvFcy6m3jYj8xwdCXV7\\\\\\\/YLZpE1Z5vZ0FANukV2Hf4XTdpH7R99dQAlc5WrcuOt8rL0l3NXZ4GcwkskDjYwbkRzT3OfCNin0rz0n8a9rKv82Dg8ztZzkovkRCJH1p9pc3U6Enw50cWZPwb8To3e+5YI+kN6HrC0CNJ1YXWHV0IEYKHnjxtLgClRD\\\\\\\/\\\\\\\/\\\\\\\/lCEeW8olREHiu7UCP1EZ9k0F8WVsn1kP13xqRUlw\\\\\\\/dDhjM4d8exa1F3W6KoRoVFGnVcUJXN6c2e4ZqZM8BWuvWLtuu208R3XPEZanT\\\\\\\/9I48exFvq2YYq7DY3KjRZf\\\\\\\/0aACo00tnRG7B6gI73QqT+cOH76+rq+IXojt3I8KlvI3OnUAhUSHswa2pjMkAnz2YKbUALgpvlJeBzgk\\\\\\\/6Ub52iT5dAhllcQWqnyYr3kYvod1x+3+XI5YpvOmFMlBbYF2NMiTE+zNd42sXweCL+AHbQWku+OZR0Tp3bLWOd4dimi6O1iR5MpNGjAHvlKb5D2dE1MjaWvJ61X9VVQFRxV2eCBRNB4wWQJ6NzR+iUOQVYntxHN8xYBGv6YP3BTOgOhzsh1xFDImUbXO3ICQGiVJxPkci\\\\\\\/WOVCcL9sqh10KtrY0+uCOKx0bOLr6VpKzhUGI4IUU2MEpWfJF3lbx3eSgdAjKkH9vNhZodAB6tzhKIZPOxOrmxxUnKailEaMv2sp7JWHQxCMWuukicj5tJgrGMaE5HPPvWhYDogExg8hDEdoruQD2HruJRO\\\\\\\/O9u66DKlsIM7KSHAaO8VmoXP+vPOorjc7Fonemk5YrepV4D0PoEW8SuHYjM166f4uGfJJAqzR\\\\\\\/J1+UiFy9Juad8LJCdTuDdZvADoY2BaFONeAMnk1a\\\\\\\/+yrOlv9mhAZtZPl1ck1qt\\\\\\\/i9cYNDM+3U\\\\\\\/t2pv\\\\\\\/kmMs3zpFMfAsrDGwOgtwUCP3S+Qz4AkCBN3SSYsTsEGpYU0+w9IqDdIUDd2CP559Dd\\\\\\\/baR3wN0C06USZDyeOcm62Yw1c8Y2XNZOdASte4lTrfe5LjQB3NISwuOz2awzwq\\\\\\\/laRRB2Ybc1W8ID7GN7uFoeE0A5I12JsywpWENgCAc6eYsoTuJbCCmZz6p04slEp7JUtt4FRBoyl5Zsh7q6zpTMISgijMMikjmzikOaqPPWRotQ1DLTn936V50ItPWZCbxIpmfFlBfT6XUvtpQhcs6KVyE28w3VdcfCJ4IE8D2+hjc3ItFcyQ213bjJjL75cyQkAAWepXHCbwDEvxumWntybaQCMQ\\\\\\\/n19Z7PGOYS1hHXbV2oggNjFnO9vCDEbaKYw8cXcVrGaRbm1eX8WyLC6M8GahkRk6ozK6Ipo6kfzClMnXngh8fgv03WzEdtl3KQ9BiBj16yLNLyir2XE2gGhBCNx9c\\\\\\\/vesKP11b0rMj4OJEMJAoPUGrdGEahCfgf+3jl8W5KXzAJ8GctDOxow1WDE888yj53w3dPArIWBY6eeQ\\\\\\\/bwcEvShoDT6oxJYWd\\\\\\\/4jiO56JkqQbPXBRW7kkVo\\\\\\\/3i1KOA+bbcjqR6zealVIx6DK5UtcwmjCbDmO8oUy2Wr\\\\\\\/8jUKvPZpKC\\\\\\\/PONA7z05342qO57Cb60JWLjP0rds1bNSmITw\\\\\\\/nA5lj8LFuxTTKv4k2IR2SG5675xPPIJ0lLseACvFN9NSfBa5r80mqd5Sb1VcJ0e9QYYe8U9ecPmMu+BIIou30OCqFNPL2ongLG6RE5LDXMg5eMY5ayTGG5YECW1cSf6vDsLhZOmiqz9QEuvwLRDxHKWbOY40WgBoh9D07bIKHXrYtdLdx1y8P2eqk2zxjtmvCWYeFovnlczRGS6Ni+AGeJcVN2UbsxVcbmuaalcDP49clauRt\\\\\\\/oVIezl4Q2oCTtt+BWzGo7DI\\\\\\\/ek2VXbjxvlc4cB36IpG9t8LnYz5UQjg23\\\\\\\/OytwD+LbKQPJ+zOitf\\\\\\\/Pa9qCQ34ot69dsOiH9zHbu14bC3JKzCDhSqnc54sRcqDcz7W0OIAWdor\\\\\\\/l\\\\\\\/uBKZ7w1aMu50NT8VNRezsxShoTx72YTMjpkWo2aaCiEqeM\\\\\\\/s\\\\\\\/TrWRc6ydP1dCHuTgBeQR+X5kWs68aWO4Xi8ju8dCwN4doEcXG6oAjY2JtH0A6+YkBLQzgTsIZ7TxuBYq1NEhkRYiIL9FwX4Mj\\\\\\\/VvUN+ZeYGg\\\\\\\/XRjhjCAtvswWtZhkTn9HJtnghhXZpvUn9JjsVmY\\\\\\\/PyG63FtQg+BAVjEIFUtx5nZj6UcgNdEWi53\\\\\\\/B\\\\\\\/LULLQodWG6agR4yjUE6RKhByaYM+qW4wQBctTh58Dw5LBaX1ovvhGWX6o4jJbANCi4I\\\\\\\/3nQp+Sj43X41wXFjAhiYaw8TFUexn5sgUSCZBxw70ZT9UeBeYv5JUZ+f0v+cbR8O2Wu3dSnkpC37nHOHbv5L2lIsmA3fFFQ+9GnrLr5pBXo\\\\\\\/6VvKW\\\\\\\/ULay3Pw8+NT\\\\\\\/++XJ+KKY8MhV0aQ+oUNTwA3+wbG4v8aYhVs04JCPJr5G7uzBtVhAnMhnc5O\\\\\\\/A1k2sFoKQ5MORo5F3chuSZ\\\\\\\/5zv\\\\\\\/yGRXLxJ9CUDDGz5lfj1IsjWOiDvzEpN39jyFjpuRbviDfdHXY4kF97N3S9nPvf6783RTMbOMYOZY1Ceq\\\\\\\/kTO13ITReSO3RaczWrqe56LrXJ1GOavhVMOqdnCG5pjI3txC00g0pcyFhIDNDudgGpnjb\\\\\\\/AR3YPwzPCEOvpm16o19OC1d4++BkHgyC4RwpApLmb2I7J18iTAvnkyn6h2denoae1HaRHyufKN\\\\\\\/3EI0pXqGNi4GcJwETzOZeTuFpsF4jJjxqVQZuQn+Z3I\\\\\\\/ohwtn4TMS9bF10n1VGtyS+Qf95tJks8gB8r6eWq8ikHSknD1DHFWgeMh1Dw3BFfF\\\\\\\/OTfhCxwSsyFMvr4PClLuxcweYlh4Jfplv\\\\\\\/1Iiz15jLxpmjdNvPUSy1UtrQp8EkQfF+iMgnOa79G9j0Cqy4LQ2Y9NdPN2+7ib984nhPYHnB5NnDj7XA21SC7AThd7KMrnAIVxFK4k1Sy\\\\\\\/z81zol5gI+FRxS18xK9xCILrf8NaN\\\\\\\/FsD07jq\\\\\\\/GC3bcNsluBMQEGTL2s+KHuB2WFcFszh1oqR4+LFzsV\\\\\\\/AEI86LIPyHiCuHPkz7SQsRJ207bvXCJMX\\\\\\\/lrJU4x2LheDpPI\\\\\\\/zW0sY1saU5l8gEDOPAfX91foDN7fpzQu5b\\\\\\\/w9i3PxVwHkDjMf3rOv5EXnP6vs7l1MHHMo\\\\\\\/oeZaLePuwfIfJ+fbZloY99h170UK3u1jAczUWSZBGLbTzb6OBGfZCibzanQjch9kAUB9cC\\\\\\\/FSTaHBXRuYPqsQhR4rTNN\\\\\\\/asiN0UTxOf503a0MoVDqWSzKjAYo209Jd90Xw1ropT2bdidlDbs5vy39aOtFTJHsKpGuNlotgH11CbfeDdawv5h4WqR9zJUhwhAyBT+BLQwPIIY6ITqaQFIxBOkOjC4zW91xMDIUeS9pqofJ7kWbAHH5gU8Emi9CY2i7Chd9UYTwxeWzLtSJ77REOT27czgqRRItSt3LyH1nLqxq1qDB4FcKPcg7X130M\\\\\\\/wgL4zrkQqYFkY6mdWgFYIynGrJvW760YVv+uca2kjFnycOofx5Ubw9z9FzqzjwdHFd4i5i5BSjesnu4oDGGnaazesazTX9PCWkkHcYYgmDRhb9545bMaDrQBYL4vSpEKWeuFHUXAQH9FETNjrb8MDeIZgZqa4uqlNH49qUn4c0rLsR7UWus3aQK9YOysHcXTj5FAT6HcCrcgRwEcuGCMN2RKl1dg1lgYDNwuqRjje13fa+KupN+DlM+11pCBmOJOlF0TVu4MRZRD98gOxelvmF3y8e5uM3a7vdm79C6AIumo+\\\\\\\/LiggZZfQL9vG9MGTUyNQCPT79uspiu91BvQS1Es7H1A2mw9sVLKFJBGwvSywu+dNaITxT8ki\\\\\\\/Ft2y9JtFUtg8Z1v7Yg6IAneiQpH8SSQIBGmwdgBIlNWyt44BsnX7LDomfMPYuAtrMrvVOWi9kvxH65YVRIb1sLSlx3GpA7xlibhzHYtPvCNC9UBKHL7A3PfgfsYM\\\\\\\/q22qvLhCq0WF0W69fSh7GwT+4ecKgfF+0qDNnAJvH8\\\\\\\/8t7uPS3dE3sQP6lc6yqaodYwCVrCmmYh9XVCu99n6oIOf9TB9Z3m2Uj0pl1uCHnM3G5kMqkwsOnqpZWlC7hWIEUkaPdw7GLaMrFHM4CKMqBXkOqUom04aLkGTE3nmI1qOcMkxo70tbPDW\\\\\\\/hZoq8eGHTUMXTFw+CdJaVuhh8ps4oyZePPiTkvf\\\\\\\/jMySr2V21gOjc5wHuNvcI9kmz39D\\\\\\\/N2KMbokU+T37ta\\\\\\\/loMgKXPiE+6Ykw\\\\\\\/UZfytnM3hfOuVPLxbYEZ8RQGL9rY725mP6VcFQsw2nBv82k68WSedeg0L6qSbSM2gT5ZmvyC97xLsfT\\\\\\\/qHNhzkv9vZNPFPiTqbMwrm4R\\\\\\\/CRVFQeToA0xnffyV9arqibK+nKWFEJZ91zTontuQAS76i6CO5Fl4gaIpxUBUtn60oRzfdv8W9k+FCdRqmKAq+UjbmivxXJhFKmnEIJvZklRawlM+cKJ\\\\\\\/MiSGhAbfFcH+lSBTQHmUUZuw0shB9CZJ59jJXr1J+XNpthc4ksV39xIdpbv4Z4l8dW0NEEyyEHUgn5H9XhPykTxDD1FJlaMESZ+dcUy8AGJFg5lVw\\\\\\\/7Pu4TRHUU2mHINn5ZW2\\\\\\\/Et3ACjcXHhaQrRI+z2Q9lDK\\\\\\\/48bzZDZ7m7SvLF7PFZnOIrSWhRR1Z01iEkjVYTv0HkGbwGgwj8V76UaI6xZ\\\\\\\/k6r2qZQiqGXyf\\\\\\\/IcAh4CwoaB\\\\\\\/6b7U96AZA\\\\\\\/REdq72GEJ62uJbj5X75dk0V23bjsDNeaK9KcT4FbINXBDtojtqvQ84nD99cgm5jubVgeuq+DG\\\\\\\/dlWq0TsCfc0zpEtXuBfeyAXimvR9q4aSTUYeNkbwX78IvPrOO\\\\\\\/cyS3ElA70IRevUpkRTj9NQqftPCDoShHMteSMtpCthQeeSXJMvvGZhSI8JI9IoMPvMDylNamniMHy\\\\\\\/5x1n\\\\\\\/7BTeywoPamjuhSHo\\\\\\\/iGkn5NtKK04a\\\\\\\/DeHjL1YQ2Z4GmnfYBvn9P2niiE0E35zcYZ8OnBYsx8uMqTynhdilhYVXpxcolFALzyoxKoCi2GpaWB9vZ+XV4TZDOSJj+QJUQ2PUN8eap4rXP1Kx5AP0J+bi15tIEGzARbvPY621rIXKRgykQe0tsSs2nAUGot04iH64xF6LWWs8VNlezB4OwUit4CBAW9l7VnW6exnZuixaBeFzXixcOnGIXrG5NVvgq1CH4d1BR3TTAghqid5lzJ0tnwXbKUj9hRp3N\\\\\\\/cVtB8CPTPG4BvfLz+TfmNfVAI7LS0Gb7PocZxJqI5LwF+FNCZC1DdkcQ+6rrXaXoS1nuplxmvJpmfXeYesrazBngvaHkLuAOcNL4kfbUr8jli16kmNLTXIMk+oEaGAaAafXtF2Qx03ZhuM1iKYQ+5rd2gTJV04BCz1JXQV0XhWoQnPedZjd\\\\\\\/\\\\\\\/iEX59N5ku3Qox9Qy7F8X\\\\\\\/SvP+IEQYlmi1ZX1RnRjXrcWfVAoJOTj8SpdQ0cg+67bLp+PVzKiSg7wCqlDDxOo9DrzYqw\\\\\\\/1S3LqtFjFQHKbRCF655KxHbgaLLkYYwYZQxMcpJ3ip2NExachh+qyxEcHRm8pTIwX6Htnn0r2rl0WQ93UknbmGRow7NefccYXxqv0xRmvnddFKsYhCwCl96QnKeu6bun42D4ABPIDikfY\\\\\\\/a+9WRW0\\\\\\\/HUut52VJ0BQSOdk8KMZqqjoKkCwLJluyzq564Q9NXBmWO1NCUCX3hDxfUpbE9\\\\\\\/7+3lpeRnzIEZLZPF4CeNWFh6e4Gomej91GvDdXYBBYSyuU1EPnUt17o5mYRRHPaASha47Ngw8NmQVEy5BQXqG45hiwrQqnDyS+y8jFIc8w9xMCO61mzAvSodJFa2vWI9V12cTNgM1zl8bquGYNBny\\\\\\\/5AuA5g\\\\\\\/lCWNhIh47NHLOUH8\\\\\\\/qMPwTcnjmufQzPuZpAYV3+V5yMYOROGYc2wWFaIGwuN1AsKO3zY1EtVz5M9qdQ4d9Nt985jRu84fJtrQG99fLpBzrDQ4ZHz9AQ1NPrlEdU++gwBX3ixdQoV8dMSNQbbak\\\\\\\/56DF87XWmptN5q0ytSI4zw2Ihb+lsvHXRwbgmLRDmUn7H9RWE\\\\\\\/boT268klINwgONhQSuxclg4TFuJSXHIYF2CxElAMaTDvQji1pfUVpAFYXf4NgDKqnEofirBsjrDPVsLcT\\\\\\\/jrItZyABpYiaF4RMrapTWYTjHVmDJBv0hCG0SMYEcOYaXvCsWGB88Z6uJEEhuUzhAXWAHciGfnPi8iBwK7NrEotCX1OV7EHQK5sc6gz5YvxThU9bwAB7e5sF1JrmDYagESFTtQxj+m4lh0J3Pu65slgW4BFWsxh+avOoGQe+8jUkfYb9ww7QixS705VtfkrddpdLR\\\\\\\/8m8OaUOoIipxCkvx5fKPKvPd93sOMZF7I0YGjFdstcebHiGIjp72i4r7Mr26KDFmWKOOToOWi7Zzd26V6g2Fjllfiql5ZGLtu2w5vlW881It6L4pcSFvjAUjXUwYUAvUZ92bsqMranfDYctZtD\\\\\\\/rXB1lSXYRlKKVNHsz4xL5iuGZ+OOVF\\\\\\\/0AUf+JxMAA4DwVCd0Pd8w6PJXFMn\\\\\\\/71GXjUpHVQA06OoNTeSuCRjhJIHQAm0YFZJ\\\\\\\/5E7us6nqr6ZtkkDhV0D\\\\\\\/7sBZd2h643KV28Ir83Ndd4ilE5gFhFnWyL\\\\\\\/jKa+DemdyDc\\\\\\\/oGxLa8XlfhaRlk5GEqjGfxRW7a\\\\\\\/CmT5KgnS5s5rB4VspXZXfhzPAy7ox8Rxis32CS7\\\\\\\/4wunBB3apOd3ZP2qQG6+iSHPJmeII4J28jkgJKKT0HdCdBUYn8SBtDCWJDZOf\\\\\\\/KH0T4ABcdudwwktfCJcLvMPOuRqYgtsynISdeHUSNmKIvHua0wXkeqStfQN6jTrm7iCHzyVQtb+sbZm8tvGNHuWJLhOjaP5OBmxUl\\\\\\\/DrJVMlUsLzDg9A+By0KLS469kdBlxfFdbBdBQpz0NZKQieD\\\\\\\/Y4MNFCJBMyuGxCURKPncX7AyVRk6rqYjTf4+Od1XHyJLzHsSxUPbOon++\\\\\\\/FY0gNmJKtdOdFysx7ncviCQvCu7QvClWxItZ7tRVk9DQn9iQZEAx\\\\\\\/AulU\\\\\\\/wnImWH9mAvCRni9kwKanu3BmhMOuSeJBEitHyQ3xx7MEwF6KNQ1F8YouLtQyK\\\\\\\/gGB5cVY4cpHa0fQDy\\\\\\\/ZO7PVAzpLmkI9WPc6zmi+aMaUvkzZ38NqRuWc2yPbijoHw58yhmxEnZlsBn7+P3vrvNaxe9WJFJJi6fZ5kgFLwQLn2Wuwj+feHWv82hzuUyCwuv1k0BqGnZK7cxEl2lh1eAAlLFcPbnKE55xuPhVfLs7imQZfAKupzb21rFzTVmk86ZWIEdyt+Mrox0zKCTzVlpbBKJvf+b7d1IxVoAxpiXCbXNDUrH0bWMJGGhRxr8zhWoL9F18tLeN69dkGVYBPzHVHLu8wloyZotvnfG\\\\\\\/cV\\\\\\\/P6DgaNdOGNHvcGeAoKLpyOozbqXnX4w0iBmOXK3ng9th9bh18+WZko4OP2dR9vxaF1i1x81lX\\\\\\\/rNfKsjAwU5uF9JKRmln0ta24VinundUHrzgWAqwYKounvLi8aRQMc4M7rrsK6omvEJ+JIrpQvPnxgEOTs7Gz\\\\\\\/Tl56jUYvQS6+TUC3iLDdmdSR0xZFV7GH0Ib\\\\\\\/KjQA9X0pz6tO3YN\\\\\\\/7XmzmvQ24nT9mXxJwPvZSR7ggD2r+I2YpoaKO9ugNqHGqVmnpICWXN9ndxhYXcDHvHsU\\\\\\\/YuB1lcgvsun8HeMJDbcShiKnu9AtcOGxD0ThJGg+SZa8byGJ8sx5iDFN0o1yclZC0+17wE8NZ4jf1Rb9fHABtoyOBwKOEJdA0Xurd\\\\\\\/sW4sD7LW7xdhUNN8LWDZqngl1aDdf9fUaPTOZOCDVwz8rK56Fkf3v96\\\\\\\/6R1hUmP125s+gcwzTf7WEVLo3Lq0IQExa1BSLorwSXayP4sx1FRvoPnGwDU5Mf3MVvGxDhIIZgE2JSDmITkvDEkfJbWGKSkV8nhMpN0q0NS\\\\\\\/tAu+b\\\\\\\/qPRqMMI9Nr6Kf4gHINXgFOERBLlsI4bb6Ul\\\\\\\/P1f9rmtHnKA3QnjHbILbAz5CPym6rtJ2Xxtoum6UKklWXnKYsTgnB2PSAQQP4GIjyVBsgDy88+tY4skCsIbK3GNcNOCI06AJyOms46h\\\\\\\/D+7X+t2IAVUf1vgvflSGrubyrC2VhDckj\\\\\\\/ZaCW7N+WeseU5Hjljtmk6nDSFJgaNDB0+kcCO3ujgyEaY6K6iqJl1VHsstBKEoLQWSfnEfJsaHmx1V4+oUdAQRHvICC3rkZfR\\\\\\\/yqrVinyBEcDdhH\\\\\\\/11TnbLIpCLYblKbJbD6ekdjeprlChISS1amM6l9vFe8rcPd1C8CsoNR5NCmhRK1NXMhuYySKrE+YzjUXLVcHyskl5mNOf\\\\\\\/KzXTvR\\\\\\\/Ht6\\\\\\\/UKsvujJD1STLfH55CsDcO+7zVWx+A+3qRCmh0hJ0MJazzJPmbt85bXz02BG9cFeOv0IHM+zCn4rwE9mg\\\\\\\/AsBmYaQijOmOgyZ2Ps3nT+A8Gv9IwdGUfYxhu4c46FyCHD\\\\\\\/cB\\\\\\\/MKQSxypaY8z4nXReZ+EWrwXWIOA9QUYUx91KAFZUvMgdsolXKRtMNtqXNzOk9gcyKSRRJ+YConYEoAL2UmsDASzR9g\\\\\\\/KrJVoXbZqPVhU7Tt5fIMqvpaXfS2RH4FVyO+jXVpZtCiuL5BmEl0u\\\\\\\/bfy0LcQWtxZyb\\\\\\\/6wJnZGe8hvfgpAM9nm3551RoVi5aKr4hn8DoF\\\\\\\/Kru5q5kCPgzpHVUZ9pwrS\\\\\\\/AuhWgK4Sers+IE4IXzMOwf6jXbd8lAZSvMpU6PJtj6FR5Rw1Ze1WfNys0R2rmOiO3GTIUe6YtCBeBmd3aa7RwQS5tvrKOHN5OCP35W8805laJlNljkKRaMJoEJ9Vll00nstvoOyXV\\\\\\\/h21SyNy4vaQQF3PqaKplgFJuGus8gC2oBI+J0sN2ULPm3AaA5uKRVBFnvDBBKLgotlamgH1pmTZWwYFajDWVuC9SlYtXnwXB8B6eAH5yaK9raTd\\\\\\\/btFlyHkacdVGjU\\\\\\\/YvAm1gc6MNF4aAH7zvtj8GgZVYK02s3vbGYMo8SzkzhCeSU\\\\\\\/QY2Neo6QOUQrdqDsmYUebJw2l8JM+WXaXdtOwlHun5PqiDMiv7LXjMz\\\\\\\/mcAfAFVnjaYDU8s21Dji3WuTlTrTki3XVhIj822iO7mfdW85VMjaVL\\\\\\\/YoxJePvfeUf9Y26P2ajRrEEa+VCZcFHPhSwMQJUVcJ5t2xnT6PTeCmf254ikvHAOD+CrIy2S+TYTAJ165rukS33BjGZqcxGlPoty1VqOHysMQGn26GE2SC1mlkM4nug37xMeJmy1KF5qzGTrHWySN5lnVhByYTRiSsvQsQLipVUm8nP5fgO8dnRqsUs\\\\\\\/cv0nP5RhyIf6qXsAZs7weV6QKAaDFwxiYUuLO0xxwO7NS8KIWI8ATda4z3dgQ0Yr38KRyiex63LyMtb1ECg0j6A3RdNOx3dMDXiHoYiYYRVthgdsoNCtTHbvYAmtK7l2z440pAWes8FA0ZUhxxI5o2hRTp3E4c7G\\\\\\\/3\\\\\\\/N8Azs2llVSodMa2lXjNWzixiJnaiXs8f7aDIUuTgEsEBkEFjHLmgt6XCRqJlpOu7R4GxOVUADXjzu+WwrIsg8ZXJpKMfDvTxyERYg3x4lkBrmmzuZ3KCM18mypUxVdoGA8j56MwXV1NX1Q+9gFtPtUyv6uYnBt+TvDYLFpQHGc8YXqm+gEJZ0Q5CZUaBCSYXF5eIvo+qiLec\\\\\\\/wAdjPrDtuq8NigRdu+jMajXfcfRuDNd06O7R932NjJK8y03qz3FPyjqcBou3i745Ai42CIWET8ueL+XKZi9f7becULhE7c73BM9382XDVoaGn1V27+8q6IBdK5SrKd5ibIfq1SoNniA7AS8t2CI5ldZ+WE8rkk4QkUvM5BA7ElK2QGvzR4h+WQPetFBhhkeWYMuIoTX0okdIRnuNAmbEhqfmdDr0XGH+LbLSVWE5orgVddtJBBtWmRLNl\\\\\\\/BM\\\\\\\/Mg41LdcHIV0l34b9Iyh3tAAXBmFTcQPsEDTA8ySqSrnD6UI0EUxiACzn5kg0pdAvj69uFTeTh8OgJQaOJ08aO5oqwGhFW48OvnJPAhUti4Trxg9ETwdzLaOJRy27R0P3bgpjnou1n+LfyKKwf\\\\\\\/c6iKK6pBW7bsXwK6eamTpO9EySmrgR+n+Q25dOeR2tcSM\\\\\\\/hFpkUazplvZxue0wsMgrzHtRbA8DbbjhiuCrpgNi8vf0qRvYGEFvHERU7oDFL4PDgR2W\\\\\\\/QZKMz2tHDZIMI9bwNDmz1kyhC5vW5koNzVwqzwCJ8rf0kcn37DSZC0StmhRS0h7XMTyQF53GWMuI8KiDAWbJITQ43Zqf2UUZNLjk9\\\\\\\/lELU8yBu7MvrC284E3n4OUtyLrHHJpVJQBFkyV\\\\\\\/1CX1mqEDrMtfru2Q1WJinpaD9wftcEnU4NGsqEVK\\\\\\\/+YvpLpwaPS+p4mSdncCycPGMzz0YEA8Z\\\\\\\/9scS1hJbXLX44zOC5mGerwXa53MQE66r4u8AgBF8q3SRzXl0hsoZPUm6Yi75jxLquMf2FzRktC3jKJZExiJ8ntO4xCp8YLdIpiqPDOJIBhPa+A4HX8e6bA0O3pmubmlox9R6JZda3+15CY3hXtWjraq2x5GZ2z1wA5\\\\\\\/MUaRn9SjFw2ZiUQcMr\\\\\\\/D1NN+bBymTSS50NeHYPYZjP0UajdUe3n\\\\\\\/fR0\\\\\\\/xugM9G6srMv0SLj9COHcrCa4xNIHdtsamxaZuhNwrjBCcRmsVikfZdk\\\\\\\/ZZ\\\\\\\/9mJdnQ25B8kkI8Zj8NjZzpAPOKWHxpU+Rck3vD6wCy6+K\\\\\\\/J5VzHmKFe+zy\\\\\\\/9p7MnfiUWmJvgRF+YS5a3vDVeE0RQARQQ3bCQrKGUmP06m2ZMvmA8FIJN6MzsE+atcC5yKozTrguX8Ht3Qg5bXRAJ1DyHOLfAu0VoiIDYbv36CyY85YnjTE66Eg1incrd3UPKTRPsLUT6f6NZ1\\\\\\\/eH0GvA0MJXUBahB9D\\\\\\\/0roNEolEwscmq0NRA\\\\\\\/1vfmDBgor7\\\\\\\/8yFw8VXa0Z1s6qRZp775cYdRtodd9H5jkiJ9MlyWcKyK0bpdV\\\\\\\/3Y+jfYtLaIKWScK+jjmybDja7s5nX4v9aljTiMEVlxPk6ot5BrcTg2SLD2WGoeRgno9qrvLHc1glaA7qNdtmcrDcT13zf+GerCPyrcQRDlYRNam1orNP1Li3iu+u4TkTdxHP99d7fPvBdkx5FT9ZSObj4PMJrg1Nwy0X1i7daQbNH0aQJPaX9QarnWoBEchEUYIY22xk\\\\\\\/rim7me0Cbpm43LCbcWS+yNNQNGE6HWmScAEMxN4H7EhScEiNMXyy2s0dBolvwVKIMiiGQvprThMDr7+GZtz+kMPJSC20d+6674+fDrvhDqvZdblVRKrkK6cZCiqQja\\\\\\\/UqX1KveAxGCvR7Q82i6IltzKg7gVtOY5vKN3pZyjwK8I1dqR59d4KUfW3j\\\\\\\/9CHmP7RhjNHLdRGuvQaNE+c7q+fcCSBBl1DCeKuxxTkzfESKOLSGFubf40jR4pcxF4nuRNfpNnM7ZW0jwZBQI87XqlSKtCQ0zjYzgnKOOEmujfXdXzz22FjtIIu8NR9Hb8hSKLpL\\\\\\\/UPuUWV8uTIa4s5H9r\\\\\\\/v70ePpqQVu6c6WzrQ2mQlYCuGMKkvsIUY92z\\\\\\\/VYqopoKzLPgxvc+e8j0bpVLwJ3x593+eHlNoJIUbCQpnvj7eDaAZ8B7zB9ktSaO2SAs4BbE4g1V2ND1exLBKFTfdZr2FaqZTPoVaSmH3Zg1DMKCk\\\\\\\/\\\\\\\/wnnhXcAlWkgYZywZwDt7XeMpFuqlgjUVgOLfkPI10B8QT9YFZAXOjTxtsgj1wvD8zH5cgqD7rkbgwQIZTHjovfvkS6pGGh2aNN1Q74cC13+5XA9rj0cK7sMITUgzQg4kOQP0m8gFWdHQlHIdca365K5HOatdoyNNmZ6dSvhep9g3vBoc04f5ilHIt0W6b5Vz7gefMtSnav+3ddz\\\\\\\/TYUKueJeGOhaPVzMa0Ef2cZeIR\\\\\\\/xhUOKuVnGogRoqIjrH\\\\\\\/3hlzbD\\\\\\\/5YFlCJDUKy3SRgN5wVTKml3ttIONjGnz0ZjgrK9pWFe50C\\\\\\\/HTMHf\\\\\\\/8QU1WYb2GYfoDSUX3mGtsGOeig0dZ8yy8aQdqxP2zCM6XBdWgJ2pvvetxjeR3Dc5Yw6jqxwKMyBxMd35kCAnI+qnqGm3qZnVbjpiCTtxN1uUcei8IbFWTb2tyL\\\\\\\/zh0Pa6Y0o8\\\\\\\/PpI\\\\\\\/blwsJcmRZYS4uVdQVIjt6\\\\\\\/ed9Ek94TtQ6NzPrHz3OuWvuBtzOcL3MQ38jppJnUyXBK6tUDmZe3GqlF4lJspWeBL1Xvg+HibC2w5JLBqZl3Q9aHmZb4AgvB26izMWLmdoisOfXraGrc\\\\\\\/f9XR5DI3swFvUrq1YWCEj21NvRn4teGrRpe3SKQTJ6\\\\\\\/WucoLv6oNy5LMNy3dEwDh2Ok\\\\\\\/Y+uVeiKW2ndeq9CN4w908zvHV0XuAnlUUc0y1LojBXACOLpGZedg\\\\\\\/q+6CXtu+ex9bu\\\\\\\/Jn0rUbqoFhSg4KfLB91hnWsCI9kSNrSy2tCyrFWfSfrT9bTokXQHj+pmfu0jRitIEC8NHL\\\\\\\/He96aZRthVEhLjkRaDpApjPtOLMIT3Se97j2d\\\\\\\/F\\\\\\\/7xWVB3YmxLRlhLjsguGzdGOn1lQqSMoVO1E08Jp9E\\\\\\\/1Enn4wTpF6etxF1zS3EhKn9byp1dFewSCkISW2pjiOPj+rGQUYaMtlsNPVzABMYTWRbObZnKwFzYmRqrMua2gU7Qd6P9MoJ1xJzUDtm8Xfx7x+ueEY74NiB0WbRVP+bOivLybOEv6nlF72TypzKvzDCuakXhct3akkBfpMMtwMTskLPcsw8TdqE\\\\\\\/Hky9ilfRr0E\\\\\\\/FjmaN8pqp5xMgZTIuScFrtO7nl2OEIhxrB3g2tr0Ngt+QEtiD\\\\\\\/xOQsYTl0+43DbVpwznlDPL44hvCOpuwNy\\\\\\\/8+BPKV0P+ukuWVHbjGDOYZvLJMx4HgbE\\\\\\\/TXJ0CrNFfVpMebOTD1psejfMMDeJMD8zovbt4B3zX0m33JMUxAbPyEj6rCfBMDDCNFSRFlZ8I7vYEX2hPMegZRD4LCtyNZP0sdblA3snJi+BY\\\\\\\/Uq\\\\\\\/PKw5RIkRjOvTNAKKIhONSrDg5b6pJLRWi98x7ZRxuclBt+CCaPDZQu9z+y8r\\\\\\\/415j2h8AUDOGI3g8LDHlf0JHf20t0VWHwGMrzwWU14PYxoOOehr18yZAExLOepf82xDafjWFRq0PwzCMYxmwcCkK3cmeBHDL3oukUnHDltcd7cyg9x45tDeqpAMyZfOSi5d41E9DATvp5xs0+PNhQXog5MfmLTK09ZQPALJwejfAelYQPnWUDYFQbyCb0RxKa\\\\\\\/PkUI8QaHShr4m3UGNgwA\\\\\\\/wncyaDtuifjX7HW0e+y1pVkO5eR3cD0C4o066kwu7YFqTNDmaKGFhnqyFlzA0LT47jf84cCWuw22fQnTr7WFTEei+\\\\\\\/BENkxptrPZEgLTN16uhTAk582W7pB9uBPfqB5Mq9hOspH7iRFkBk9pWx78OePELVST9PenwsU8dCmPZU1meqr2MIKbNoQBLf5Dolh7jdde1LvPGjkX+yaDeTBR8Iof8M\\\\\\\/xBbOlkFCBq9HMQ6AktAGXd3g9K6OVCJqYj8Y6NN6DHzPKyjf7HA1Y\\\\\\\/wErQuDjAojGZnwJU8tbH+a85Ifc7ofr8ZiKK0Xmy+yruEj5CCbWkVwanL54fRz+dDJJnEHcFzJf3V6aVRrbx5jufrZS1Wf\\\\\\\/TVyzMa9HaB4Kvmirc0Tx0Tc42Dp7ozAZTGzfvXuOnM\\\\\\\/WcFGD8RUs7nYLW2gNhsU3PHCIDA7P6hCUM0RHm8VfWUqbl1ksMVnDZUkX3UztUgP9R09AmuGNAbK4+RQw740Z9DepCtXCljtTTSHYa3o+tR7h2WWPozztXK1eR00xP8td1MF2ptp7eXulZSTFmf5sT2671GtRR\\\\\\\/EeWwABJpvtIWN1WI3gD29KGDjZOa9qlWtkQwZxguj+LH6rJCwTQnXhu4hnHy5VO9tzoe4xSePZkaxounBOCbC\\\\\\\/wkj3cQOLItF18P+w79qk4jz5nbtJgKy7fqhx9uPgEs3jpJLnGwewxGqWWUBnx7WHttxol1rwG5GyPS9iQ9Zr4aGrRpT9PC+tZFO+LRUs1Q+XpReFp0r0GxGg6BFV6f+eS6sUhfcJkqqbaIRTLK\\\\\\\/l8U1Kj3DUVpC4WxZFMY5o9DvdGY1fwizK5L7+vuctTzY8HzAbVJT2tPj2cwpNOPyFOW6aGEqwUNvstmxX4QFoayDlrBBL2+\\\\\\\/D\\\\\\\/I1vySbC838I3kjiZFWJ5ddqP+ygdYU9DH7mUo0HbtuEgrdi3GbcBettzuJ5PqxkLkHPLVce0z\\\\\\\/\\\\\\\/pdd4FX5ti\\\\\\\/z0elNTH9+lA018SVgPQJhicjeWZ+BBJ1P5+Wd0CN3alx2wLhnMEaey8TzC5Hfr8nbEiCqk\\\\\\\/w4T1wzqqBBZxGh30BW7JnX100bX7Vz4IDY2tSXzkt5f3+6SkgSUYSCeVTNcPburZ00G9nY0hlCKyU\\\\\\\/qQwwhMLHuajFv5DFuKWW0jB+vTXtL80NNhT108aqReFeK6Zbm8AQokN3kUpPtBAWSZOQTb9WPkGUS+nrKy+JiDSBY7e4DIywFCnbavf\\\\\\\/DR+zWunezM3yAdtJtreSmM0z8z6lygt4CIvWTOGZIk86QZ\\\\\\\/Ntt9nNknRFGoGihXU2rjBfePO\\\\\\\/PGkjlvoNgkILpp0+Wc7vprZFMKKaYlxRgQ7cNWarhAbsI9hjVesBcyNxxE\\\\\\\/SHZyZddl7o0qQdFrtG5iDLJ2IbeLGm6v+wMcoQ1UpKrJX7lo8bF0m1k9oMFdRJ\\\\\\\/ZvJdgMSvzyogQkZJyxZ9\\\\\\\/vTZQq0wQxU8DiPj0AJQyyrZDnYcYvKu4kg6DFsVYmy1Dyxc1iCbDH6\\\\\\\/fI\\\\\\\/mb+\\\\\\\/ehEF+IMDSdLAYPW8FF3e6cn+vuNwuPMdjTNHSgQ6v8znegUgmJfvXQApomfYiq\\\\\\\/4ZeKaDx37vpHnM+HRb2eEDbBUxdRQycKWChBnuhQVoaIE57vOQDogUG4tAAfkCaXyMQ+7RISEHdzdcQAUq9r+NkC3TmzdIHBoUTnoKO6tE7VgXnNiloUklGQvZzo6jqFMwub8mmfYAuRtzBAJoPhicTexae1S8PGlSNVAYCOVE+pXdhh0LemY3jfmQoPe7bcI9hLVIhBE6UI85SGhaGMEDen8jebPfFizj5afGz6uCBS2o31mkBcROXHOlrPnZbBB3WojeN0xw\\\\\\\/4kIQcg36nDgyqifh8eJ895YxSXUojF0AsBL5IoAuR7jXTUm9iBPtKOGYvLoBk0X6ZKJRSQ\\\\\\\/GfkYjR1Y3HuOGQhQ0pWv\\\\\\\/Kx3O7DMZk\\\\\\\/rzVkqPNjPjphsRHFUoUMyg3yA9\\\\\\\/IhTHL1gEBHWIYCshJtTveVJQLv1Hhct+0QdMGhjMa9ATKMtH\\\\\\\/h8t7cp389Ot7xL0MnYP4wq8tG8nVUM6VAkpFG9nvboKNSndy253XmQgGQJWieQq6hrhiPwS26CD7Gps\\\\\\\/FObjxjA0f2HFzrOwXXLzE2Mfulps0aCve6yQcA8eQtRCj5nrvm5DP0Ci3uu5DpO+2KE3mpT7CC1Yn7rhCJSVPPDVq4Su+JqyyEpTQyxKEgmUUCzTg5xaQLb0PpHHoNFEVpS2fD9RknqOzkdJbjguw03FNWSnCaxgL\\\\\\\/bnWJOmbobiqW9MXEFB60F6r+yo0I6ylEsHznLmez5Pcg1CKTcpMiyM\\\\\\\/jzFV6KQRH9v6TZf61Z7kjqTOw3VhK4paOwMyFxV5bvESuIHwAIdoaXylJyGcMMkqUFno5jtS9aiiT5W2+HL4\\\\\\\/7QO1aiEbsMwA5\\\\\\\/MJpRAf4nkexPjv8X98WZA8BCkRvCYnQTosFozHPh3z7KOUO\\\\\\\/Q8bPmRV9ZxxiGAEwtE8IZHcPSLcJKPwBo3lAqwoEXkRFFH42YsogpPEQ55ObEEHh134oDinxUrgW8SDXiUnC01xuBV2KpNWPPsVHU0xszS1I6kxd8bl8c94Fw6XdhBGcvnjjNynCMCFEuW3oGTKFIijmA1bOf060GfO892qjLEqi7qzFVBddsy41SoWLVg3RHoAzeWiTizR0xlpzWwx9As0XtuoxB4JJSFVAsL+LzJm44eZ8AWIr2yatp+LrxvIjiDuG2otpYIK4WjsUe3ARc3+UI6mcbHkUEhrSJuFl6o6rhF+fGk4JGgNU3ShVfFxJsCtXvivTn31ahTDKyMOvR1KdzGqgEhiBDg1Jklc4uIFYkAHLIemFdeyTqoIQ7suzuAQJX9paFc7aOiYRmfDJb4flz3z4XfRzqX2iznpzTGFbjPg0dKj8h2NO4+4dytfMXQWEYIXoyDKmS12PSCcziPgv1jCT7ppO9eoD1+gzpAvAHUTWVN98+Hf+ZYiPd9Xr3ohFMn+k8+xn5ENvD1SGg5CfYupewC+SDENeS0vKejIN8Mm4vK5Q4CRznO\\\\\\\/wjpnQ4FVuT7j8PR5q7mEPLPisiy2qpFcjj\\\\\\\/LrTEirlM7AkYGYgDoUqKyQBYukD3vZpX82ZZhFlKLSqMUoLxCe+a4T8dFpUpCkQ\\\\\\\/HhG1E0lcG4G02lO+nwnajXB5qt1zBrlcdQSCNakTjKI\\\\\\\/AQa4k\\\\\\\/x1Qrh5oJevkemlg1WiRoEeuEiOioRdqsHoGr4fdRQ5RhGy4gRHCLifNAMft2pe0mEn3uBeQXBtdxh2FiHzwzFXwHA\\\\\\\/Bh75WjoM42xVL2njsqpsaj0nuNwmy7y0xmfW3b0UbY+1FrZCCCqDmzivubRGPVmLcp96Ufy6WfNmo2YfHxEi4ZOF3MVKnXL4jzDWY9pVjH+iItCQ9Q16vKuiJNlvivXcw6hPfzfWKdOtgHehGZqEL5EzoRieLGIvo6QW3w7Ib3acdfPB+r5zcZ\\\\\\\/mH2phIr2K3Ke0Aaudb2tDKjyrphl5uOKhBUyqvzu+nFzOtqYI9TPdu0K45r9Dy\\\\\\\/XivNfM1dKiGrR9fJ8gjBNFePf0R1A9Kq7Pp8C5xuA0SHA4QcygqMsyVp1MzowuB5iH0L5dgGZGQkEMKloeRA48vKJ+9WIZ+nihsTFT3TQcxPBgQTcWLqK3LXZ7ijMVRysuj8UoJvv35SVD+hWCBBTBoDrRkJzOWhcOCvfDBsxs\\\\\\\/+0huqSnR\\\\\\\/tv7t9XbIYkBgYLy8ghytHobm4src3Uu8YHTn3WDJCgXVgNoptNLqTr\\\\\\\/WN6ROnYqISQKmGRDc0kZgwgsrM6kXtUdr6CxV8BJkvVPBEdD+YMXK\\\\\\\/4dq4MMju375oL3HNlMI6Vp+TXMAi0JGA1ouRsy+ZWT7eBp808Wx3zoQHQrT+i9tDdS81OXuKVSwWyxtxEFDplKZwBGQ5rciydKJ\\\\\\\/0NnkZo4XKMqYLYK\\\\\\\/1ZB79tV+AhYpVbj1feb9092jGdt46M51SSbW4shcYjedPA6Qam5oNsj9FpDP2LTgugmWAt9HvROCPtjBpbUXuXKTQmK5AmEGW29hh4UbPdq8rIWr+41PK+cPODHTrO82jeg3hzoGYUdAF8Jq3xqDtnaL37M9yPKN6P3GImnvuBrduP+5poVeKyE1ZVrZJsgtDIL3gQ5itJDsrh2nT4c9Vp1NO\\\\\\\/pDxJ00SIWpUXqI8uViPQ6iO88TkzowmecQP63bCUVgtXw7evQJn1dyImIqWPNpcluqRxJi+Ywhx1c9h+mdubIWKgoILCNo8P+HpHAgOiaFmoGqZjSjn34GxblRm\\\\\\\/UcdJwHOfvFpWxFpAyYeUaT4lJlNB4iiz9rkI3Cv7T2qYSEJkLZIe5s4BVAS3sOe+jC0oIyqkG0DQ0q0UXgQxdjOU0Pf63P6of\\\\\\\/mNZF7e9Bhdnh61qDMhDi7v0EyCZ+9dZ+5wlCvIZCMxZHqVKWck5INeusFtgJ6j4r1RJaI5vmErFcP20SBn7\\\\\\\/VRGS5meT7vnye3AbMjdUUzBDzL7bHCrqAQoKBbTMDoioLd\\\\\\\/daU6MZ8XrActYWWHEJ584Fybeo9CBUH\\\\\\\/po17t7QJ7CM3szLF1\\\\\\\/lzhi+xDDnOZD08ypTn2WnCOQ3UfY\\\\\\\/zHDJOnhsyhSxxc9nXNGxTevRVDPL4NpAZERHplu3CZELSA8tC+qDODoUh1ZTmfvfx8XgjZUGtI14\\\\\\\/W5SFZZQy9hBIEgnXY5XPPqMSkhgnkYltlmfKMkgw13q4teBfUVtSXENWvSLL9n8nVsn+t6fLZHw96Vwvtq1hoagmTPmH2kaedvPPI0KUeRFXI9fy9Ocv1tmDu8AE\\\\\\\/RjmrNHSz9rtElwDoMMaA5gSd00Qn7rR5Tw7b3s71wL3HWig\\\\\\\/CRkmRjzHqdu11+dkYxQUUuY\\\\\\\/z3141mMejuyD+9olm+JDEVZTas0cz4o0sIrjbm8TZIYBHOPpuZKrCai2EjCfDGdXRN7\\\\\\\/LjX1kigwMpR70oi7TwVGc6Fbojv6a3bf\\\\\\\/y+n8RzawxOtvQM9weQckmTJvIsowQsjACl4980m78atd1ffN6gfYMxWXfSK1P73ziEJ+8e8+n2+yMiwkjuW3VBzvipAqdZK7HHn2Y+iPki3on02oQ4AQaF\\\\\\\/7oDj0LDh8iB6yvy8eeqXyfkYuHCQ4ok+aqMDzZY8KozGqNI\\\\\\\/0KSJ1d+OmB17pceILv5bCSE8z9i2yaD3MxJ74qBI3L1q+YIMLenDDRYvQjc4FGblV3gjV1n6OWxBYOroxQ12\\\\\\\/IKpeqSQ9es7gin71zYg8cvmdFbFpcdbbyqnkCROWI6T6Xy7UIwWMH\\\\\\\/oaQybAyRukZnkWuh\\\\\\\/rz4B+oXhRwpKKvevSNRQchse5rCjCclbKH0QO\\\\\\\/sqBmw+j0Aw8sZgJXGt\\\\\\\/Eqi+jSjFiKwAlGUMh9rnURQ9esVusp\\\\\\\/\\\\\\\/kZevFjGEXmgSKmwBuhm2R3b+3EQNHKPBS2WDfakASvRCOCH6ibVDNiL7XKbdUxebruT65PeBToO4efo5VAlZAE8eBymlpa+9R\\\\\\\/okKNx9wYLRGlVX4\\\\\\\/58miUJLqulmsYKOhpeJ\\\\\\\/l9M98Ss1pBopAj3+A+RhuJCRqdKIoDb9xbf9ZmEaT6+UOFhlIvUumuYJLp7yrewiT675ETadUBlugNkqHP8Smj+2+HuqZOGRCxC5F3esuuGyK7leWpD9c3RAYRk2ckJMLBS8U9wBCpSTAUOPk2esk4NRaf14F8orFcY4mQJTAyNvVsXMSftKZllDeP0o2MVVTA4ma5IXkYMIsefC+DzqYmjHOuNIamR1yJRgHAvzuQ0JkYzIaDKj\\\\\\\/62TUjEqYVonWomoDok3oOoxZd1dQcb+DFwCiNiAyCNvXcTQc9qO0SCQ1XUlW0RsmrBslL6REtkoLaPJjUFZ+qUTg4AJTu1lA0dzFuUmLV1wdR64875nA28jCsCNJ9TI7thakev7Mav0DJi+d9ce\\\\\\\/LwAjr\\\\\\\/nLjd6c+SxH+TLD71KxoGYFYfDXDvwhp3lfBI\\\\\\\/0DyjkTWCWvehA6zxtfnhsS\\\\\\\/Gnia9DcI5ZUtenp5Oy7pG2gJpz0mOz2ILmnNVBOV+ygtTEnRZFx7TNK29ePdXUjsYveirDljJYqmr0ZzyGkgEJ92bGe6bwOI7eWf0ZCyRS3aJYhQuQkih\\\\\\\/EDqHhFkIfm4M+BE\\\\\\\/Wf+PNMFh46UbznBLTIPlsSsQ\\\\\\\/oK7xrVCaY2YAmoXxFYKVCyy+7gBkIWWN5JgK5QH2M2YkDFtYyRNyB4c+TNykdQ1DINb5hJYwlUPYJ6WhD8yMNCLxb4hqmEtqwUNqunE+WWVU3KDUay8hreerMTAr\\\\\\\/s34PZKMtAk1D\\\\\\\/PoZXeWljbFh+umKNz37vV0obaIK0DHpBvIrdw7fmXjP1p0ukvJpOWiy+oiAyM1Ap1QUtZPGUKB41+zY5M0TehlXmGVcLEtojEqMn7Ax7vc2GA+PJgKlrVIC+4aPfKyVOL100+4toTBoc85Ddy0sh35oN+I\\\\\\\/+EsmBMfpQJCGxW9g\\\\\\\/ZwG8XVCDn1xd1gh3gGDmXtRmHqxEKqfjy4+mavgyVSlg1HmegUo47cm5zLbgDUAE6UrvcrcolPFQjxZrBrqZyylWUygODu69wzHN9bL3GLbq9S3rSRnHlMGCUOyyiMK1nTh0AeMbB+3jGe2sXDQH7DLUAsDnTOKMh49tO8HZkQ7kn+ZlgXsTaErrocksjzMwDhAu2yLnn7BlFVIulSdDEs2upVthcpOwuo2nNT97xHt98lYecTGLm+1nvrzWb4n+3aoyrC8rKnz6xnNeuNA0QCCluNwnGNlSUTnLgKjFY+HqNFK\\\\\\\/u+yscGXUO2VR+1iUXhWavvs\\\\\\\/lqACyAL7yLa30A17WQZ+JzDGe+5cs35H6A6Q0ijHLorQp0\\\\\\\/7mqCxwlPWrRLfbgrNUmKWCGwu\\\\\\\/mkO3OZoP0A1+roS01Y2U5ZOQhaJek7iBWADGu+hGdmwPwqR0lsl8jmigH6bdk0oIaQgZj09KGBPZBj133epiS\\\\\\\/aVc3sT+EJxUdJwhXc1915fD2NIJm4kWfH4mrPVx0ouiBulr5AZZmbN3jb6l3Ph8c8RPI6UrSGAwd4DhP9bjVXHX2ghS+u1uhzGKaBgCup0eDxYyNoELsCW8+fsvaG+4q3647nWdktl1uLy+zVC1GwfYhl3ONu3FB55nouvOdNdDpNa9XRz1t02R\\\\\\\/7xPqRny7snrZp9YewnXFyOC2ry85vHkVAvd+eWlpA5ARBL0J6Ph6lx4DxJpKNXGBAFWg7JmrdyJ5qvMZJQv1Jg7RmJUX++xg4g0jgU0chXlexSun1YSNf7gTI86EBCL477kPsW88Le6UrM1JwDLJqhLUmzuasZsrCsSGhnVT03FhsnIFlDoAhqsAnv+vUjy2fLY4Hlvg6Ov33whEr\\\\\\\/ThdjGtwh6ywD1V+WfLsWcK91kamgad1KhjidweQcHBi+MbOw+YnywV0MFI8WhtdH5fRyUfskEKN64WNTSagIPP4++Hw09v4X2Y2B5xFYDFXFUD212+L1KCmtopZ2rbpFHv8+2oVr1zn3jPjjivHq91gxuvM7IAd5t0nn5ZMSmIG0N\\\\\\\/ybUJB\\\\\\\/FD2v7AZ4WC0QCWL6gDdgUb16sYS4qYdZf1IcYRBEFQG0lef465GOJ+4LbpySgtLuRCbhsOXW60Eai4rEFyEP2BdzbYwOFYNf8zTEFvkfQR6OdyX7nmJlr+SsqaplqU3+KTkuuTOa0fCixlfBYwJXPqx7X6Q15v1+YXC1i1rfErqne3SfaLR9H7adfsHTot1GgyRXPIT3LFBOle8depKxrFx4yCAQdKqHxLujIfwZI+oCz\\\\\\\/soOjsVNa7AfNd8SaET7UQs5t+FNI2nuPRrcyCLme3\\\\\\\/3oxhTqeI0GufyZ9Obdh0Q376OwrjSUOIkLZxyGlT3zzNsziFZGUHW3y0\\\\\\\/YoeYxqA9avr2ykc3vM7Sb1aMC7Lz3WziiGFXm3+ZzblD6UBAxlO9P+DlAX2r2OvbWCe5R\\\\\\\/059oBHio0HoVdU21HKamVapzoG6LX0acvKe3sEhP7sMt+GDpe+JgGDi\\\\\\\/vHwe6m2Us52qztltkLdjpozERs3E3sAUGH0oxCIAb9iqw2IskqT0eInhZQs0SVrH4oihtVun61nJfhLGujtpiwMEO4G+u9PnaEkTBzhw6FuR0nFrdEeU\\\\\\\/0jP\\\\\\\/9RHUQRMV\\\\\\\/louUPVY\\\\\\\/RQa1abdCM8YCOJeZDNoRsepFprHLxd81VKDco+ldLZjPVvLa6vz6AjdfMc32mALzr6FBp2uIxFzaJ08PMCAWHnhsMhzeuE2Pn7G6oeMeCoVwhIZnvI19UJS6UVFbx5ANrZEzS2mE3C86W2U0X30zH83p6qZTgr+CQGK42Y4pGb2YpIy3zTyWW3wZKj3rJnJOnV82G8ae5RvFebOqG4ng6dvWD88H8haSmjHlVleAz0C6FvwgWeY54mVUCxRllOE5Cj8\\\\\\\/KyE0NhWmghdUGreYS+\\\\\\\/3SO0GrxhfJOev7+q6IMg045K3wmIEQmVFYEBiws8St83iEHYjVGhK+4AemWrY3pxHOO6KxlSn52ySavdie\\\\\\\/UkIQFZDbOwn0k14Myy7dD6WHzdl\\\\\\\/59hjS+HLCuSOAkC8BSKqKS1WDcbZiiI5OP1GqaCBBqkzpUa8WCD0vSzVMXfXGdgcVBnQPxP21kXyUm0iHfzLtnXOcyvETKhSYxsEOUx+6AyvScXcNmVK4lmLVAJYkIaIc4K1LJW1jHAVH5M9dYlKSZipDTrh9ltY\\\\\\\/JuiPdc7rRA88F62gLWfksGJ6Coto8usrryx\\\\\\\/qZN5AU9ZzSMYebHd2caTC7ZS6SJr9fw31d9T5wQ0aimMrO6vDb3Sgyyf8gMU8KjBZUYPJu+1HucC4JJMUqTenNBQTnu19kwA8vZIVBMuFoNSS2aQIMTEZITWObD\\\\\\\/+PN+U\\\\\\\/gZ2K+S+twWZQUPyzlJA9TkEPUgEHlTwa+X2GcU7SshjB2pBlMy+lg7mWx9zFciqkUZ75S\\\\\\\/6or6hI02GfHh\\\\\\\/d+sfsFqRzTfOxjFNVYsh8Hsv4pJBIUoaFXhN4tXGaRYUa9LIvU7sM6+bFRjMK0eBFB1ho05V7L7k+QH41zt3Pe0NO36xVarmlGnEX9miSHzlNhiL9A3dmKfmiKvqE7i7rp0p7xdF7osnsBLlVLbOWO1ddFvbWl69wuKzB\\\\\\\/DtszYIwp6LcUU4BJgZ7yz8EosMniQhLHF3AsiQgadbQ+Pgdy5C+vYwWriDyZEECyKBooV03Bqb15MqbtIL7G2sJvpoo9picl8Y9HI0+4Mr9BIVTZSiY1kt9xQ5TvnG\\\\\\\/a1khEkChGHnqNDqjVg1aVuT7IvEGKkH+k16fMNa+ixaIAiMvDVgSLj5cxegwdoB6qwEhmoW\\\\\\\/b9\\\\\\\/zyWxvK0UGIYFYF5ql7ughbpXIo+GiyvpA17m9GneUYV1AdyeXEXKtL77TgkmLdzVo8GGRR0zS4MfTJ1V212rNyvDlborCqFcUdJSSUI4Ps8HCeJOf7tQUS5VSIQRxc9eiR5ejn3HWjBiMXnaL2bxg6I0kRQqvQ51BGbtmOycyXzIQeKXPeSSj8yq5u6MbmvK3jr4+O8vK9h9RP\\\\\\\/efRGNUOJlEB6txyY\\\\\\\/YJ3hpzFEM1K1Uu73g3wKORnpNO+u1rm5J0GPMILNOEzoTtT9FRQgfxw7DxpDLZcPImNtAtP2vvNXwn57P2abG\\\\\\\/ZDfSakI8em\\\\\\\/lBv67srYg3m54TxFYATM7VLfTvnQnhVHKU1qRzFZ9nEXgoKpsi+nN8gyXEgk3gGdQsxNjjYG9nm2UHdYVx9\\\\\\\/CYo8EOyd5eXDyCkKsS8WMZqbpRtWcvkC9XMe63YfurB8Zz1fwCOGSnwSJXegMnhavagXXBZLJ6\\\\\\\/JM75CQQJayID53GIHKQ27FNE9RCK6goLr9lGJNZkpP7SWEwpqhJ79dQHXZx3KqvXBYSuVBYQ16JsNwr+SW4nMS1aT3XZYtPkC2im56ZBlAqZHGhVu29DUSy+fHqHWQnXOzOYX3lfiZsz4OBO4EST3R0rD8UhhgBrB6kNXyQmp84XAHE4yNOHM0bF0UQfzhQ62Ir9sH5Iv2sb46sTUl6OAeqJxaao0OrgCS57oVTwx22Bbp7tBbRnax7e0RuVrtaLnT\\\\\\\/tvE3Ul0DPuC5AtaHfFPaMFiUdU\\\\\\\/wGEmI8awt1PLykSDDTBj+8SxxUqsm9+Tavd8mgn+jXgdq6rK4n8eaZkLY20r8SrmzUbxMrccs\\\\\\\/WaY2ovi9U7Xu22HmLZQx7eOam3wQAlSJyazn7pLGFZxHJ0dOOJYYXiG1nneMdB90h\\\\\\\/cDrZn2w7n9CAdkPUbpZHxqkTYcqpbOxbcTkl7ryTn+GY75LV4dA66qTPyZR49XpKyXDLu+W9TSWIlpwRty6ibURNBr7EKRk2r3FzF1EbHyiMcGQ4RbXI5uuUa0c1Nrtn2SOLSbf0SIt1\\\\\\\/wZBrFkSs9vjrWWX4AswRBU4j1yYHP0JLAmbzmeGMm61KYEmQMLgv0cFWK2Bt+M\\\\\\\/wjlSkzG2mRWMcebY7fy0zFnw+VqbN1LqbBmmFWZFp9omXvHh0kF87KBcnpcMou+JFiIBox+f1YHotvSaU3jrs6bQc6xnczoEMjkhWH8vIOQhDpCYpRtltWICyDonn1L6QT\\\\\\\/QYVf7Pgr\\\\\\\/BlWbi5AhteMb+tD\\\\\\\/aocDGW1WTcnIvLeVeSUSGXywQO80CDbGnt41ooRT5Eo9B+Dtpm9ufTLpN\\\\\\\/hpIzgyUjfBItxv8VFK7NXPSe8Bsgysea6WvQI1RgKdZmhh0yBjCJByCYVVlJxryljf28EuXGuOWsLbJaeHsO+mMct\\\\\\\/t\\\\\\\/OBcBzBosjq4uIQX5uXSU5nOJG58Mn8PyzcixORlRNGJJYBN330FjyAAUe7IoGXyXAHNLUcIe5lpQaoNA2skv+iT1eQ\\\\\\\/nGNV3vSWDVv5+rhvj16GM49WvN9Hxr+4NNlmR8kz\\\\\\\/YDwgLrasjMBKnFwctHTv0nu17IlJw9w1hzHe+xth7MZ9fEhQb4tSZhDDZJhqxnr5HiljewSPi8PIESQGxfVIUSjyolH7MF7U\\\\\\\/DGseBezvX30cTvo\\\\\\\/zPnil+xR20Osio\\\\\\\/hPHNvwi17ZzOwu7XgfMQgIazSDTWM9clHQBULVXUZiZVeSec0k0hZbk139i6eMjhKpRlN+QDHUUCviWg14VAApywdOUJEo3\\\\\\\/c6jqiX39Eqp+HDJgIKITZGU5nACqSDbWVehwuPDLi8TfBx+nCDJL5qmOHvwRSsSGx5czA+36drWdn9aTxJCom9ukoOiTb2ygEkYhHyZZyHWxAx5Jxw9Ceei4+twfpa0ekPr6o12TER5VWg7qc8lI6L7GLx3no9cY+0fTKZDwLuRj3s6A0HpYiXgq0JmVMi3WlDEJh+1T0icjXbeqUsWR5t9PZqfGGvXb5WkWOMy4YToxbNSLYSJEAmUbLkumVKCOMhe9qKGXSuCKiLYMqkA9ChDqIrzUX1Eisthxzbx3Ls6rbFLVk8udfBDFMKMY63KYcXq1bF19WpUSNXcOQWBLTj+NpO+QgQe9Cjhjbz3v+OGMvhxvGRLTX85NcZ84vv4kMIBQfj\\\\\\\/jO2bjtjXOkHxC5BgLo1+kKfq\\\\\\\/34UWVa2ZPW9TdeMVFsrahGTecufmhu2lbETMSs0PeE+14Lklcc2NyjipgghH\\\\\\\/0XYh9Hmtj2fGqYnSGmMcZTimVGMeXkVlfziaWMUMoYk6IMok80leeIFTIWXTvvRIK5TTcMEKvLYpjAl6cgv3pq+n8vLyOiejnXCdLjr\\\\\\\/p+uVg4Y6Swnm7IHz4heHonUqgbQS1BQE0OLWHt\\\\\\\/pQFOntXl6Wk5NMVpFy7P74QT6Et019JpNlqrdRvh20+CGtGXaDdZ65NmHEFbAAlKuBm3Ytv0uoGJFk7d2DgOEmMP7wzJTWw6iQlXQ0sro3Ysgf8+4iVUIHRB+q5cDEHYVY2DKG5CMLjdRqxo\\\\\\\/ZquJe1WEpM9muVzaZeRVWfbi+NWNrEgwQRg1S7LiPHjMDx7Lc0+3aCEocUj6FGnb1\\\\\\\/PHmCEjwJEzW2daCed1Em4AS\\\\\\\/5zZWPAx6B0lteeiV\\\\\\\/k2x9ktwgR8BK\\\\\\\/ylk+PN+pO4mJ4tKDpTGZF25B4uO9FYXxrWj0680IFeP0HLFu1rLtPYyMxvxwzVELQggwV5a+F9wAzDfZTwU4aAdJhI1imAufmj3R7geDDDg2lt\\\\\\\/UETxW+zmRs5V1U\\\\\\\/jvD1ob7dgsRUHrRAxeP1o5l+AxHc314n86r+A3Jq7D6ICCzFY0EgmQ9YtCINCN+M7dbRZtlTAeQhKCmp3hKxNVVdARloPExLZkVtWVziqgCROgnW8F+Xcp2q0vE1VfDiZCnLSW2k0WRbh\\\\\\\/Ha\\\\\\\/kj+UMpcl6dXp7ZZDLj7NN\\\\\\\/0i4KUnsN+1DfPaNSjDmFdNFNrCMzB1N9W58i1CqmFd1WMubv0+imD1qDc6V7kUERuxhiBgSwXvCSjY+D0YB+0fAXq9\\\\\\\/UTL7CIB5Axa6lzo32W2YZZVA0lhwjVoG76pOq7uCkkI0HTd4Qq69PVO8nPlE\\\\\\\/aImq4NZkHV4KUOVFiGxj8kD7aK2emufn1x8SuSiJ6YIGhayoVqWuhd12j8PIfx3iwT7GTrn9P5KdroO81jx\\\\\\\/5DJY3S1OUs72TvzogmuyQDHVD+x2Du58IH2f9P5QrHBbymqdPgbaG5ffo\\\\\\\/wAAMfzGQ+6ZGhcc6maAEum+q7pezGUbRpUm+i8LTE4IrvPr\\\\\\\/DXZlizi2Ig1owF\\\\\\\/+j+xn7aZVaeo7y655GHbnOjd5T3jKo0qRJ1+c5e1o7mHm4\\\\\\\/xP06hjN4VPta9vRY1IUMwxsbanJcDCJbAtKq8E4dfit6mjTFIS44kjCpRYRU7qJPbOS4jrKKwnmZ42EPfWgvf\\\u0022,\\\u0022iv\\\u0022:\\\u0022SYSwN4pw+QBSn4okB1JqLA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002218d0cb87eb1e7c9f6595b52c81c20bfef9fbd11d1b17dd4ad287ffc29a6eab61\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022R1gDWixMR2o5NA8ypmnNrbq4LxYHMiCSgD6\\\\\\\/HpDK8fGg8XDYTLZBh7ea36dH6jzg0vZs6Gs35WnWJ8YpbQ\\\\\\\/xoeh5Bu8huNOg0jZ16wLyB6LFmNtjmTLnlHpRFLc5jsPzkKOpFxcZlsiyEFyGb23r\\\\\\\/MZAxA2XgjzKAULwjbSw7HhlfBRt3zKRQccLb1DXsY1K\\\\\\\/3RVAwFB7cCoHYyI2SVbThboXmxzLE9L6wRzbk1gZL0LGqrtNycd1AaAmkfXbrGHhKyBKsY6uGlLNSzRNYM4E\\\\\\\/Y1ZtJiHZ6RliS\\\\\\\/w6zDuUBq0Nv7BhMb2unzPkRgLkRcDMnZXktFJueEwjJotyzDJxh8EP1aD9NiydaqbH59oirIffPsLOlXFJo0LudWpcsqC4VeEJ1fGZZzsrkw\\\\\\\/MTv+ZtZ5yqmp1VliXUFl4EX0ksgvRG\\\\\\\/bFSJKzXBkip34jL2Ek530ur8sJgpO7DOvAS7Bb7r23KRbDfAh0LqjlS33JbQ3ZtNemKepEbBEbaQ6s5VU1u9rWHt4yGaNwMaQg4SpO2QsJ7TrPUBn0BrHXLmzOff1N4NdX\\\\\\\/DyDLP8UzuWEo90OaZSlSffINOQdxOLhPHUQRQWQtCMiEFAG66MWSJgWYwv54SAzdQpO1tfi0R2jyWQRuuZheKkpc6oCJXNgAMwInddB45swSDiidD1\\\\\\\/EfUen6kRs1PRELwjhVJRuPar\\\\\\\/NCKHz0ukqakpe8fHvxaW1VGuvzg923jFrSkC75dqhnVenOGDsfhuL6qlPLt4K2kjtw3owjAIBlqyd8B2IRx7uRMp9cREs0bEIXtOF8El8W\\\\\\\/5Ai6vPKWhHSrCsLNy86QUaEZksWPNR2+kUYy7ND7UPwGyPr90B9E+DLSWmZ6BHoy6+E0eMgKOMgnWGHobNihHCicO4RG9zO0+Vai\\\\\\\/wbglDNvMH4ao1yD+KkO5w4xxi4KpQHNvK7YKGgAfetDOyQFBujZvkd\\\\\\\/pUXRVvU\\\\\\\/YKOVhhsaBTj6YnSlyVkpR1lkg6NOFzYpHqN4VVQUcwlSDOMX+m0AotljW4wagp2ya4T1hh\\\\\\\/H3XBf2+RWi1zM+m8er\\\\\\\/tJM1IoXbMMLKex2N9EMeJ99elmG0D5NJ+STf98IwkLNEtQ3bIANSn9Zq03XQweyR8RHszXz3tiH6paza9putSh7FnVLcauSxCEaCqI7TBiZC7cnMwI6TUnz5apkVO644vMKtFvwhsW9LvIDG9tGE0e5ll448Z3m46e8ElcSmb0Ozy2Wou\\\\\\\/f065FTVn0Wm4dzyx+ytK7JEBDARqLBEXwx7i8UP3GRFFeZ1OA97HleQZe+2njRj6Uq9uF4xCBf9rPZJ\\\\\\\/6jLyhPLuYvn38JU9nQR5YZcjNU9DTi+XG38Sbd9qrBT4RsqhDDzlDvHFTVj2ZzOVrYIiMHyA1lozSU78RqotiHYpuXAashNHISIWT6AZedFG1bjFtOVW12+z3QzqHXa0OmhMhTh0Pvk\\\\\\\/k5z4vvg4y8aKl2SIS4rCmsDBmzH06f+30xK4988whZHLecodysReFjbRMFKENXmGY2J2g\\\\\\\/gRX6bSfkAMVq8OHYIu7lG7mW09f\\\\\\\/2jCXHl8wAGRL0yEictgMBLn651fE\\\\\\\/EtlUGwocqZEI729QS5JEtooPE3zabR3rhi92UwConzTu0hIsAR7UJ3fyM\\\\\\\/rIyWAx9hS1TJKjOjCx9+7w\\\\\\\/QdakLUJQwQcJFqSBHap0M+FpQpwceQdWZFQGCU5LOqw8yxQ8m4wnEZNSm3Q+lrFeNc5BEmVypYMCW8QPcaoMpZc5j04fmWaIYpy0tI0Tu1LU9l1c4EwALPSq2FeR7ACyQCoWn8AjIRkb9oTwUddGU8L+TZyfOn1Pea+E6bxwZTfWJ81uTZd7DxyN5hzHvNxUpVu1iu6KpGGKpf8QyHaY1SZyomD5deVEPGUvCDZD5xbArGqMA6pZUKBgckJCltPFdrtPAxig6bTq3lCJyJZyA9IxqNczO0OJSG0X5d4kDT0fUv54K\\\\\\\/D\\\\\\\/O3QPfEly6pGe9XfWVKqOHwUPrR5rDe8r735du9EMz3xAzbvkQ=\\\u0022,\\\u0022iv\\\u0022:\\\u0022XVNRdBsqeiR+xMi\\\\\\\/Lk0q+g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225595b68bd9fc0f48530f0ba99ec00ca0911cc541485d96c605f9afc6ae0882d4\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459919 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1253" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-M81nSBrb050te9t_qz_2o5ZD6mreL1sF6MViczLwRO4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1253" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0avkgpma1XXpE7FKYVx1Y-DXjrLJ4FAYYiYgsNmrAsI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/6/1247" title="Effects of Genic Base Composition on Growth Rate in G+C-rich Genomes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/6/1263" title="Neurospora Heterokaryons with Complementary Duplications and Deficiencies in Their Constituent Nuclei Provide an Approach to Identify Nucleus-Limited Genes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459919" id="node459919" data-pisa="ggg;5/6/1253" data-pisa-master="ggg;g3.115.018473" data-apath="/ggg/5/6/1253.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Improved Detection and Characterization of Copy Number Variations Among Diverse Pig Breeds by Array CGH</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jiying Wang</span>, <span class="highwire-citation-author" data-delta="1">Jicai Jiang</span>, <span class="highwire-citation-author" data-delta="2">Haifei Wang</span>, <span class="highwire-citation-author" data-delta="3">Huimin Kang</span>, <span class="highwire-citation-author" data-delta="4">Qin Zhang</span> and <span class="highwire-citation-author" data-delta="5">Jian-Feng Liu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1253-1261;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.018473 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459919" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jiying Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Key Laboratory of Animal Genetics, Breeding, and Reproduction, Ministry of Agriculture, National Engineering Laboratory for Animal Breeding, College of Animal Science and Technology, China Agricultural University, Beijing 100193, China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Shandong Provincial Key Laboratory of Animal Disease Control and Breeding, Institute of Animal Science and Veterinary Medicine, Shandong Academy of Agricultural Sciences, Jinan 250100, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiying%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiying%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jicai Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Key Laboratory of Animal Genetics, Breeding, and Reproduction, Ministry of Agriculture, National Engineering Laboratory for Animal Breeding, College of Animal Science and Technology, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jicai%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJicai%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Haifei Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Key Laboratory of Animal Genetics, Breeding, and Reproduction, Ministry of Agriculture, National Engineering Laboratory for Animal Breeding, College of Animal Science and Technology, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haifei%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHaifei%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Huimin Kang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Key Laboratory of Animal Genetics, Breeding, and Reproduction, Ministry of Agriculture, National Engineering Laboratory for Animal Breeding, College of Animal Science and Technology, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Huimin%2BKang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHuimin%2BKang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Qin Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Key Laboratory of Animal Genetics, Breeding, and Reproduction, Ministry of Agriculture, National Engineering Laboratory for Animal Breeding, College of Animal Science and Technology, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qin%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQin%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jian-Feng Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Key Laboratory of Animal Genetics, Breeding, and Reproduction, Ministry of Agriculture, National Engineering Laboratory for Animal Breeding, College of Animal Science and Technology, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jian-Feng%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJian-Feng%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:liujf@cau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">liujf@cau.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/6/1253" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459919" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459919/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1253.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459919" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459919/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1253.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459919" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459919/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1253.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459919" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459919/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/6/1253.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459919" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459919/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">As a major component of genomic variation, copy number variations (CNVs) are considered as promising markers for some phenotypic and economically important traits in domestic animals. Using a custom-designed 1M array CGH (aCGH), we performed CNV discovery in 12 pig samples from one Asian wild boar population, six Chinese indigenous breeds, and two European commercial breeds. In total, we identified 758 CNV regions (CNVRs), covering 47.43 Mb of the pig genome sequence. Of the total porcine genes, 1295 genes were completely or partially overlapped with the identified CNVRs, which enriched in the terms related to sensory perception of the environment, neurodevelopmental processes, response to external stimuli, and immunity. Further probing the potential functions of these genes, we also found a suite of genes related important traits, which make them a promising resource for exploring the genetic basis of phenotype differences among diverse pig breeds. Compared with previous relevant studies, the current study highlights that different platforms can complement each other, and the combined implementation of different platforms is beneficial to achieve the most comprehensive CNV calls. CNVs detected in diverse populations herein are essentially complementary to the CNV map in the pig genome, which would be helpful for understanding the pig genome variants and investigating the associations between various phenotypes and CNVs.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-variation" class="hw-term hw-article-keyword hw-article-keyword-genomic-variation" rel="nofollow">genomic variation</a></li><li class="kwd"><a href="/keyword/copy-number-variations" class="hw-term hw-article-keyword hw-article-keyword-copy-number-variations" rel="nofollow">copy number variations</a></li><li class="kwd"><a href="/keyword/array-cgh" class="hw-term hw-article-keyword hw-article-keyword-array-cgh" rel="nofollow">array CGH</a></li><li class="kwd"><a href="/keyword/pigs" class="hw-term hw-article-keyword hw-article-keyword-pigs" rel="nofollow">pigs</a></li></ul><p id="p-5">Copy number variations (CNVs), defined as gains and losses of genomic sequence greater than 50 bp between two individuals of a species (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Mills <em>et al.</em> 2011</a>) have been considered as a major source of genomic variation. Since the milestone works by <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Iafrate <em>et al.</em> (2004)</a> and <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Sebat <em>et al.</em> (2004)</a>, CNVs have attracted extensive attention in genomic studies of human and other species (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Tuzun <em>et al.</em> 2005</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Redon <em>et al.</em> 2006</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alkan <em>et al.</em> 2009</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Sudmant <em>et al.</em> 2010</a>).</p><p id="p-6">In domestic animals, a suite of genes with copy number alteration were found contributing to variation of either phenotypic variability or disease susceptibility, such as the <em>KIT</em> gene for white coat phenotype in pigs (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Giuffra <em>et al.</em> 2002</a>), <em>SOX5</em> gene for the pea-comb phenotype in chickens (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wright <em>et al.</em> 2009</a>), and <em>STX17</em> gene for hair greying and susceptibility to melanoma in horses (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Rosengren Pielberg <em>et al.</em> 2008</a>). Additionally, the study by <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Seroussi <em>et al.</em> (2010)</a> indicated there were close associations between CNVR#456, located on BTA18, and index of total merit and genetic evaluations for protein production, fat production, and herd life in Holstein cattle. These demonstrated that CNVs can be considered as promising markers for some phenotypic and economically important traits or diseases in domestic animals.</p><p id="p-7">Pig is not only one of the most economically important livestock worldwide, but it represents one of the most important research models for various human diseases (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Meurens <em>et al.</em> 2012</a>). In the past few years, many efforts have been made to detect CNVs in pig genome based on three types of technologies, <em>i.e.</em>, array comparative genomic hybridization (aCGH) (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Fadista <em>et al.</em> 2008</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Li <em>et al.</em> 2012</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Wang <em>et al.</em> 2014</a>), SNP genotyping array (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Ramayo-Caldas <em>et al.</em> 2010</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chen <em>et al.</em> 2012</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Wang <em>et al.</em> 2012</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Wang <em>et al.</em> 2013</a>), and genome re-sequencing (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Rubin <em>et al.</em> 2012</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Paudel <em>et al.</em> 2013</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Jiang <em>et al.</em> 2014</a>). Among these technologies, because of their accuracy and cost-effectiveness, aCGHs have remained the most frequently used methods for CNVs identification and genotyping of personal CNVs (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Pinkel <em>et al.</em> 1998</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Feuk <em>et al.</em> 2006</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Carter 2007</a>). Currently, Roche NimbleGen and Agilent Technologies are the major manufacturers of whole-genome aCGH. Because of their different probe distributions, probe numbers, and workflow, it was found that sensitivity, total number, size range, and breakpoint resolution of CNV calls were different among aCGH platforms (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Haraksingh <em>et al.</em> 2011</a>).</p><p id="p-8">In our previous studies, we have conducted relevant studies using a custom-designed 2.1M aCGH (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Wang <em>et al.</em> 2014</a>) and read depth (RD) based on genome re-sequencing (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Jiang <em>et al.</em> 2014</a>) with samples from diverse pig breeds. To enrich the CNV calling and compare the detection performance between different platforms, we performed an alternative aCGH-based CNV identification by a custom-designed 1M aCGH produced by Agilent Technologies. Identified CNVs herein are essentially complementary to the CNV map in the pig genome, which would be helpful for understanding the pig genome variants and investigating the associations between various phenotypes and CNVs.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Ethics statements</h3><p id="p-9">The whole procedure for collection of the ear tissue samples of all animals was performed in strict accordance with the protocol approved by the Institutional Animal Care and Use Committee (IACUC) of China Agricultural University.</p></div><div id="sec-3" class="subsection"><h3>Selection of pig breeds and animals</h3><p id="p-10">In the present study, one Duroc individual was used as the reference, whereas the other 12 individuals selected from diverse populations were used as the test samples. These 12 individuals included one Asian wild pig, two pigs from Yorkshire and Landrace as the representatives of European commercial breeds, and nine unrelated individuals selected from six Chinese indigenous breeds (two Tibetan pigs, two Diannan small-ear pigs, two Meishan pigs, one Min pig, one Daweizi pig, and one Rongchang pig). The illustration of the features of six Chinese indigenous breeds is detailed elsewhere (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Zhang <em>et al.</em> 1986</a>). Genomic DNA for each of 13 individuals was extracted from the ear tissue using Qiagen DNeasy Tissue kit (Qiagen, Germany).</p></div><div id="sec-4" class="subsection"><h3>Array CGH design, hybridization, and CNV calling</h3><p id="p-11">The 1M aCGH was designed and produced by Agilent (Agilent Technologies, Santa Clara, CA, USA) based on the newest build of porcine genome (Sscrofa 10.2) (<a href="http://www.animalgenome.org/repository/pig/">http://www.animalgenome.org/repository/pig/</a>). It covered 18 autosomes and two sex chromosomes containing 965,080 oligonucleotide probes (60 mers) with a median and average intervals of 2191 bp and 2632 bp, respectively. Genomic DNA labeling, hybridization, and array scanning were performed according to the manufacturer’s instructions.</p><p id="p-12">Raw data were first normalized (algorithm: LOWESS) using Feature Extraction software 10.7 (Agilent technologies). Then, Agilent Genomic Workbench Standard Edition 6.5 software (Agilent Technologies) was used to perform CNV interval detection. The QC metrics motif of Workbench 6.5 ensured adequate quality control of the hybridization data. In present study, an array signal, which met the requirements of intensity value &gt;50 and signal-to-noise ratio &gt;25, was included in the analysis. The Aberration Detection Method 2 algorithm (ADM2), with threshold of 6, bin of 10, and a centralization threshold of 6, was used to identify genomic variation given the log2 ratio of fluorescent signals between test and reference DNA samples. Fuzzy zero correction was carried to prevent inclusion of aberrant segments with low average log2 ratios. Additionally, we applied a relatively stringent postanalysis filter—5 probes, 0.5 log ratio—to ignore small, spurious, or low-quality aberrations. Finally, CNV regions (CNVRs) were determined by aggregating overlapping CNVs identified across all samples according to the criteria previously described (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Redon <em>et al.</em> 2006</a>).</p><p id="p-13">The raw data generated from aCGH and the sequence information of probes involved in the aCGH designed in our study have been deposited into the GenBank GEO database (GSE49299) (<a href="http://www.ncbi.nlm.nih.gov/geo/">http://www.ncbi.nlm.nih.gov/geo/</a>).</p></div><div id="sec-5" class="subsection"><h3>Quantitative real-time PCR confirmation</h3><p id="p-14">Quantitative real-time PCR (qPCR) was used to validate 19 CNVRs identified herein. Primers were designed with the Primer3 web tool (<a href="http://frodo.wi.mit.edu/primer3/">http://frodo.wi.mit.edu/primer3/</a>). The glucagon gene (GCG) is highly conserved between species and has been approved to have a single copy in animals (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ballester <em>et al.</em> 2004</a>), and one segment of it was chosen as the control region. To keep the same amplification efficiencies between target and control primers, the PCR efficiencies for all primers used in the study were required to be 1.95–2.10. All qPCR were performed using LightCycler 480 SYBR Green I Master on Roche LightCycler 480 instrument following the manufacturer’s guidelines and cycling conditions. For each sample, we performed validation in duplicate to improve the accuracy of PCR. The second derivative maximum algorithm included within the instrument software was used to determine cycle threshold (Ct) values for each region. The copy number for each test region was calculated using the 2<sup>−ΔΔCt</sup> method (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Livak and Schmittgen 2001</a>), which compares the ΔCt (Ct of the target region minus Ct of the control region) value of samples with CNV to the ΔCt of the reference sample.</p></div><div id="sec-6" class="subsection"><h3>Gene content and functional analyses</h3><p id="p-15">Pig CNVRs were annotated using NCBI gene information (<a href="ftp://ftp.ncbi.nih.gov/genomes/Sus_scrofa/mapview/seq_gene.md.gz">ftp://ftp.ncbi.nih.gov/genomes/Sus_scrofa/mapview/seq_gene.md.gz</a>; <a href="ftp://ftp.ncbi.nlm.nih.gov/gene/DATA/GENE_INFO/Mammalia/Sus_scrofa.gene_info.gz">ftp://ftp.ncbi.nlm.nih.gov/gene/DATA/GENE_INFO/Mammalia/Sus_scrofa.gene_info.gz</a>). Gene Ontology (GO) terms and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway analyses were performed with the DAVID bioinformatics resources 6.7 (<a href="http://david.abcc.ncifcrf.gov/">http://david.abcc.ncifcrf.gov/</a>). Because the number of annotated genes in the pig genome is limited so far, we first converted the pig Ensembl IDs to orthologous human Ensembl IDs by BioMart (<a href="http://www.biomart.org/">http://www.biomart.org/</a>) ahead of GO and pathway analyses. Statistical significance was assessed by using the <em>P</em> value of a modified Fisher’s exact test and Benjamini correction for multiple testing. Additionally, to test whether genes exhibited a different selective constraint among CNV and non-CNV regions, the dN/dS ratio compared with those human species was computed for each gene, and Wilcoxon rank-sum test was used to test the difference of dN/dS ratios between copy numbers’ varied genes and monomorphic ones.</p><p id="p-16">A clustering analysis for all test samples was performed considering the identified CNVR as genetic makers based on the method reported previously (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gazave <em>et al.</em> 2011</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Tian <em>et al.</em> 2013</a>). Specifically, we first built a scoring matrix of the CNVR data for each individual by encoding a value of “0” or “1” according to the absence or presence of any given CNVR. A hierarchical agglomerative clustering was applied on this matrix of individual vectors using the pvclust function from the pvclust R package (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Suzuki and Shimodaira 2006</a>). The agglomerative method chosen was unweighted pair-group method with arithmetic mean (UPGMA). Additionally, 10,000 bootstraps were used to assess the robustness of branches.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Genome-wide CNVs identified among diverse pig breeds</h3><p id="p-17">In total, we identified 2276 CNVs, with 189.67 CNVs per individual. After merging the overlapping CNVs across different samples, a total of 758 CNVRs (<a href="http://www.g3journal.org/content/suppl/2015/04/22/g3.115.018473.DC1/018473SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/04/22/g3.115.018473.DC1/TableS1.xlsx">Table S1</a>) were finally determined, covering 47.43 Mb of the pig genome and corresponding to 1.69% of the genome sequence. The length of these CNVRs ranged from 7.02 kb to 2635.29 kb, with a mean or a median of 62.58 kb or 20.93 kb, respectively.</p><p id="p-18">The results in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> show that the numbers of CNVR are uneven distributed across the genome of individuals surveyed. The average of CNVs per individual was 189.67, ranging from 65 (Y2) to 291 (Z2). Additionally, out of the 758 CNVRs identified in the study, 55% of them were identified in only one sample, confirming that segregating CNVs exist among these individuals tested. However, there was also a considerable proportion of CNVR (12.01%) with frequency ≥50%. In particular, 11 CNVRs harboring 46 CNVs were found among all test samples (<a href="http://www.g3journal.org/content/suppl/2015/04/22/g3.115.018473.DC1/TableS2.xlsx">Table S2</a>). Compared with our previous CNV study using the read depth (RD) method based on sequencing (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Jiang <em>et al.</em> 2014</a>), these CNVRs with 100% frequency in the test samples were reflected by the inherent copy number alteration in the genome of the reference pig <em>per se</em>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466310/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466310/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Pig sample information and CNVs detected in every individual</span><div class="sb-div caption-clear"></div></div></div><p id="p-19">In our study, stringent criteria with mean |log2 ratio| &gt; 0.5 and five consecutive probes were used to call high-confidence CNVs according to the previous studies (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Liu <em>et al.</em> 2010</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Li <em>et al.</em> 2012</a>). Among 12 test samples, two of them (DN1 and R2) were male and the reference (D4) was female. Based on the two sex-mismatched arrays, we assessed the false-positive rate produced under the criteria using a similar method as reported previously (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Fadista <em>et al.</em> 2010</a>). Theoretically, all segments on chrY should be gains, and all segments on chrX should be loss in the male individuals. Consistent with this theoretic inference, all segments on chrY were actually detected as gains in our analyses. As for chrX, 17 segments (1.05 Mb) were identified with the log-intensity ratio &gt;0, resulting in a false-positive rate of 0.36% [1,045,770/(144,288,218*2)]. This indicates very few false-positive CNVs produced under the current criteria.</p></div><div id="sec-9" class="subsection"><h3>Pattern and size distribution of CNV regions</h3><p id="p-20"><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> illustrates the location and characteristics of all CNVRs identified on the 18 autosomes and chromosome X. <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> further demonstrates that the proportions of CNVR differ greatly across the chromosomes ranging from 0.84% (Chr.4) to 3.49% (Chr.16), with the average of 1.94%. In particular, the proportion of CNVRs in chrX was 1.81%, similar to those in autosomes. Among all 758 CNVRs, the number of loss, gain, and both events (loss and gain within the same region) were 529 (69.79%), 200 (26.39%), and 29 (3.83%), respectively. Loss events were approximately 2.65-fold more common than gain events, but slightly smaller than the size of gain regions on average (45.48 kb <em>vs.</em> 80.00 kb). The size distribution of CNVRs (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>) clearly demonstrated that most CNVRs fell into the length interval between 10 kb and 20 kb, and the frequency of CNVRs tends to decrease with the increase of the length.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1253/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic distribution of CNVRs on 18 chromosomes and chromosome X of pigs. The chromosomal locations of 758 CNVRs are indicated by lines. Y-axis values are chromosome names and X-axis values are chromosome position in Mb, which are proportional to the real size of swine genome sequence assembly (10.2) (http://www.ensembl.org/Sus_scrofa/Info/Index)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-507256513" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genomic distribution of CNVRs on 18 chromosomes and chromosome X of pigs. The chromosomal locations of 758 CNVRs are indicated by lines. Y-axis values are chromosome names and X-axis values are chromosome position in Mb, which are proportional to the real size of swine genome sequence assembly (10.2) (http://www.ensembl.org/Sus_scrofa/Info/Index).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1253/F1.medium.gif" width="440" height="164"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/6/1253/F1.medium.gif" width="440" height="164"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1253/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1253/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466177" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-21" class="first-child">Genomic distribution of CNVRs on 18 chromosomes and chromosome X of pigs. The chromosomal locations of 758 CNVRs are indicated by lines. Y-axis values are chromosome names and X-axis values are chromosome position in Mb, which are proportional to the real size of swine genome sequence assembly (10.2) (<a href="http://www.ensembl.org/Sus_scrofa/Info/Index">http://www.ensembl.org/Sus_scrofa/Info/Index</a>).</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1253/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Number and length percentage of CNVRs on the pig chromosomes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-507256513" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Number and length percentage of CNVRs on the pig chromosomes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1253/F2.medium.gif" width="440" height="199"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/6/1253/F2.medium.gif" width="440" height="199"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1253/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1253/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466212" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-22" class="first-child">Number and length percentage of CNVRs on the pig chromosomes.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1253/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Size distribution of CNVRs identified by aCGH." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-507256513" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Size distribution of CNVRs identified by aCGH.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1253/F3.medium.gif" width="440" height="252"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/6/1253/F3.medium.gif" width="440" height="252"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1253/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1253/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466309" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-23" class="first-child">Size distribution of CNVRs identified by aCGH.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Gene content and functional analyses</h3><p id="p-24">We sifted out a total number of 1295 porcine genes (<a href="http://www.g3journal.org/content/suppl/2015/04/22/g3.115.018473.DC1/TableS3.xlsx">Table S3</a>) that were completely or partially overlapped with CNVRs, including 921 protein-coding genes, 343 pseudo genes, 5 miscRNA genes, and 26 genes with other types. These genes overlapped with 355 out of the 758 total identified CNVRs. We further tested the dN/dS ratios for orthologous genes of pig with those of human (<a href="http://www.g3journal.org/content/suppl/2015/04/22/g3.115.018473.DC1/TableS4.xlsx">Table S4</a>). The results showed that CNVR-related genes have dN/dS ratios significantly higher than those with normal copy numbers by Wilcoxon rank-sum tests (<em>P</em> &lt; 6.2E-18).</p><p id="p-25">We further explored if CNVRs identified herein were within duplicated sequence regions or single copy sequence regions with high evolutionary conservation. It is more likely that a CNV of a single copy sequence maintained among other mammals have a phenotypic effect than a CNV at a duplicated sequence. Specifically, we determined those possible single-copy genes in pigs that were also maintained among other mammals based on the resources of one-to-one orthologs in Ensembl (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Vilella <em>et al.</em> 2009</a>). We sifted out those genes with high orthology confidence and &gt;80% identity in pigs and other mammals, including cow, horse, human, mouse, and sheep. Using these criteria, we finally extracted 1454 of such genes in the pig genome and found that 20 of them were involved in or overlapped with 19 CNVRs (<a href="http://www.g3journal.org/content/suppl/2015/04/22/g3.115.018473.DC1/TableS5.xlsx">Table S5</a>).</p><p id="p-26">We further performed GO and KEGG pathway analyses for the genes in CNVRs. The GO analyses revealed 53 GO terms (<a href="http://www.g3journal.org/content/suppl/2015/04/22/g3.115.018473.DC1/TableS6.xlsx">Table S6</a>), of which 12 were statistically significant after Benjamini correction, whereas the KEGG pathway analyses revealed 12 terms (<a href="http://www.g3journal.org/content/suppl/2015/04/22/g3.115.018473.DC1/TableS7.xlsx">Table S7</a>), of which only one reached statistical significance (Olfactory transduction) after Benjamini correction. The significant GO terms were mainly involved in sensory perception of smell or chemical stimulus, olfactory receptor activity, cognition, G-protein-coupled receptor protein signaling pathway, cell surface receptor–linked signal transduction, antigen processing and presentation, and other basic metabolic processes.</p><p id="p-27">We also performed the clustering analysis based on the status of CNVRs carried by each test individual. The resulting dendrogram, together with the bootstrap values of every branch (most of them ≥50), is given in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. The two pigs of European breeds and all other Chinese-originated pigs were clearly divided into two distinct clusters. Within the cluster of Chinese pigs, the genetic relationships of nine individuals tested were basically consistent with that geographical distribution, and two of the three pairs of samples of the same breeds (MS7 and MS8, Z2 and Z5) grouped together. This cluster analysis implies that CNVs may provide information as useful markers in the cluster analysis.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1253/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dendrogram generated by clustering of all the individuals tested on the basis of their CNVR similarities. Numbers at the upper left and upper right of the node and beneath the nodes indicate P value, bp values, and edge numbers, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-507256513" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Dendrogram generated by clustering of all the individuals tested on the basis of their CNVR similarities. Numbers at the upper left and upper right of the node and beneath the nodes indicate &lt;em&gt;P&lt;/em&gt; value, bp values, and edge numbers, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1253/F4.medium.gif" width="440" height="245"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/6/1253/F4.medium.gif" width="440" height="245"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1253/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1253/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466294" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-28" class="first-child">Dendrogram generated by clustering of all the individuals tested on the basis of their CNVR similarities. Numbers at the upper left and upper right of the node and beneath the nodes indicate <em>P</em> value, bp values, and edge numbers, respectively.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Quantitative real-time PCR confirmation</h3><p id="p-29">From the total 758 CNVRs identified in the study, 19 CNVRs, representing different predicted status of copy numbers (<em>i.e.</em>, loss, gain, and both) and different CNVR frequencies (varying from 8.33 to 100%), were chosen to be validated by qPCR. One or two pairs of primers (<a href="http://www.g3journal.org/content/suppl/2015/04/22/g3.115.018473.DC1/TableS8.xlsx">Table S8</a>) were designed for each CNVR and a total of 31 qPCR assays were performed. Of the 31 qPCR assays, 28 (90.32%) were in agreement with prediction by aCGH. When counting the CNVRs, 17 (89.47%) out of the 19 CNVRs (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>) had positive qPCR confirmations by at least one PCR assay. The detailed information of the confirmed 17 CNVRs is listed in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>. Our confirmed rate was higher than or similar to those of previous studies (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Fadista <em>et al.</em> 2008</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Ramayo-Caldas <em>et al.</em> 2010</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hou <em>et al.</em> 2011</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Chen <em>et al.</em> 2012</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Wang <em>et al.</em> 2012</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466229/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466229/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Results of quantitative real-time PCR analyses of the 17 confirmed CNVRs</span><div class="sb-div caption-clear"></div></div></div><p id="p-31">All 12 test samples and one reference sample in the study were tested in the qPCR assays. Consequentially, we also calculated the positive predictive rates and negative predictive rates for the 17 CNVRs confirmed by qPCR analysis. As showed in <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>, the average positive predictive rate is 86.54%, demonstrating that, for the positive samples, qPCR assays are highly consistent with the aCGH predictions. Contrary to the positive samples, for some of the negative samples qPCR assays do not agree with the aCGH prediction, <em>i.e.</em>, high negative predictive rate (average 42.08%) observed. False-negative identification can be explained by the stringent criteria of CNV calling that minimizes the false-positive but inevitably leads to a high false-negative rate.</p></div><div id="sec-12" class="subsection"><h3>Comparison of CNVRs with those of previous studies</h3><p id="p-32">CNVRs identified herein were compared with those previously reported in other studies (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Fadista <em>et al.</em> 2008</a>; <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Ramayo-Caldas <em>et al.</em> 2010</a>; <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Chen <em>et al.</em> 2012</a>; <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Li <em>et al.</em> 2012</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Rubin <em>et al.</em> 2012</a>; <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Wang <em>et al.</em> 2012</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Paudel Y <em>et al.</em> 2013</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Wang <em>et al.</em> 2013</a>; <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Wang <em>et al.</em> 2014</a>; <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Jiang <em>et al.</em> 2014</a>). As shown in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> and <a href="http://www.g3journal.org/content/suppl/2015/04/22/g3.115.018473.DC1/TableS9.xlsx">Table S9</a>, the CNVR number and length overlapped with the 10 previous studies vary greatly, ranging from 0.13% to 61.61% in count percentage (0.02%–63.06% in length). After merging the comparison results, a total of 526 CNVRs overlap with previously reported ones, indicating 69.39% of CNVRs identified in the study can be validated by previous studies and the other ones were first detected in our study. Furthermore, we performed an exhaustive comparison with our previous studies using 2.1M NimbleGen (<a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Wang <em>et al.</em> 2014</a>) and read depth (RD) method (<a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Jiang <em>et al.</em> 2014</a>), whereby the same samples were collected in CNV detection. As shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, among the three types of platforms, RD detects the most CNVRs in number or the largest polymorphic sequence in total length, whereas NimbleGen detects more CNVRs in number than Agilent but nearly equal total length. There were 253 CNVRs (length of 17.31 Mb) detected by all three platforms, whereas 261 CNVRs (length of 13.65 Mb) were identified only in the present study.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466240/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466240/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Comparison between CNVRs detected in the study with those in the previous reports</span><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1253/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Numbers of CNVs (length) overlapped among different platforms." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-507256513" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Numbers of CNVs (length) overlapped among different platforms.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1253/F5.medium.gif" width="328" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/6/1253/F5.medium.gif" width="328" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1253/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1253/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466338" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-34" class="first-child">Numbers of CNVs (length) overlapped among different platforms.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-35">Using a custom-designed 1M CGH array, we identified 758 CNVRs among the 12 pig samples from diverse breeds. We found the number of identified CNVR differed among the 12 individuals, and more CNVs per individual were identified in the pigs of Chinese indigenous breeds and wild population (207.2) than in those of modern commercial breeds (100.2). These results are consistent with the previous studies indicating that Chinese indigenous breeds have larger genetic diversity and higher average heterozygosity in comparison with European breeds (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Zhang and Plastow 2011</a>), which highlights the importance of using Chinese indigenous breeds in research of porcine genetic architecture.</p><p id="p-36">Concerning copy number status, loss events were approximately 2.65-fold more common than gain events in CNVRs. However, previous studies involving humans have suggested that deletions were more deleterious than duplications, and losses tended to be under stronger purifying selection than gains (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Emerson <em>et al.</em> 2008</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Schrider and Hahn 2010</a>). This observation of more loss than gain events is at least partially related to the technical bias. As also noted by others (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Redon <em>et al.</em> 2006</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Fadista <em>et al.</em> 2010</a>), due to the CNV detection pipeline used, the aCGH approach has more power to detect a loss than a gain.</p><p id="p-37">A large amount of annotated genes (1295 genes) are located in the identified CNVRs. CNVR-related genes have dN/dS ratios significantly higher than those with normal copy numbers by Wilcoxon rank-sum tests (<em>P</em> &lt; 6.2E-18), which is consistent with the previous results involving pigs and other species (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Fadista <em>et al.</em> 2010</a>; <a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Li <em>et al.</em> 2012</a>). This result indicated that, compared to genes in non-CNV regions, these genes in CNVRs might undertake a different selective constraint and be subjected to a relaxation of constraint due to the redundancy expected from the variable number of gene copies. In accordance with previous studies, GO and KEGG analyses have evidenced that CNVRs are particularly enriched in genes related to sensory perception of the environment (<em>e.g.</em>, smell, sight, taste), neurodevelopmental processes, response to external stimuli, and immunity (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">De Smith <em>et al.</em> 2008</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Clop <em>et al.</em> 2012</a>; <a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Paudel Y <em>et al.</em> 2013</a>), suggesting their contribution to adaptation in the wild and behavioral changes during domestication.</p><p id="p-38">Further probing the potential functions of these genes completely or partially overlapped with CNVRs, we also found a suite of promising genes that make them a valuble resource for exploring the genetic basis of phenotype differences among diverse pig breeds. For instance, v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog (<em>KIT</em>) has confirmed that gene duplication and a splice mutation leading the skipping of exon 17 is responsible for the dominant white phenotype (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Johansson <em>et al.</em> 2005</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Pielberg <em>et al.</em> 2002</a>). Consistent with the previous studies, in the present study, <em>KIT</em> genes were detected to be duplication only in individuals of Landrace and Yorkshire with solid white coat color. Additionally, we sifted 20 single-copy genes, which also maintained one copy among other mammals. These genes with copy number variations are more likely to have a phenotypic effect due to the natural feature of conservation and single-copy status.</p><p id="p-39">The genetic relationships of nine individuals tested using CNVR-based clustering analysis were basically consistent with their geographic distribution, as well as dendrograms generated using microsatellite markers (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Fang <em>et al.</em> 2005</a>) and high-density SNP data (<a id="xref-ref-17-7" class="xref-bibr" href="#ref-17">Jiang <em>et al.</em> 2014</a>) in previous studies. Therefore, CNVs may reflect demographic history as <a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Paudel <em>et al.</em> (2013)</a> observed in their study, which could be used as genomic markers to investigate pig genetic diversity and evolution. This can be largely reflected by the statistic supports of AU/BP values of resulting branches. It is notable that some branches, <em>e.g.</em>, branch 8, obtained values of au/bp less than 50, representing low reliability in the resulting dendrogram. This demonstrates that CNVRs identified in the study are not sufficiently informative in dendrogram construction in contrast to SNPs and other genetic markers. A potential reason lies in that it is not easy to accurately determine the genotype of CNVRs identified by the aCGH platform compared to other genomic variants.</p><p id="p-40">We compared the CNVRs identified in the present study with those in previously studies. As pointed out by <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Pinto <em>et al.</em> (2011)</a>, the reproducibility among different platforms varied greatly, and the overlapped rates were affected not only by detecting platforms but also by different samples tested. It is notable herein that both 1M aCGH in the present study and 2.1M aCGH in our previous study (<a id="xref-ref-42-6" class="xref-bibr" href="#ref-42">Wang <em>et al.</em> 2014</a>) were used for CNV detection with the same samples. These two aCGH have the same coverage on the pig genome but have different densities and positions in probes across the pig genome. The reason we used the same samples in different studies is that we could systematically compare performance of various technique platforms with the same experimental conditions. That comparison of findings by various methods can mostly reflect inherent characteristics of them. Although the density of the 1M aCGH used in the current study is less than that of 2.1M aCGH in our previous study (<a id="xref-ref-42-7" class="xref-bibr" href="#ref-42">Wang <em>et. al</em>. 2014</a>), the novel 1M probes successfully detected an extra 60 short CNVRs of &lt;10 kb in length (<a href="http://www.g3journal.org/content/suppl/2015/04/22/g3.115.018473.DC1/TableS1.xlsx">Table S1</a>) that have been previously missed by the 2.1M aCGH as well as RD methods.</p><p id="p-41">Although the present study detected a limited number of novel CNVs compared to our previous studies, the goal of the present study is not only to explore novel CNVs besides our previous findings but also to establish a framework for CNV identification with both high detection power and cost efficiency. Our comparative results between aCGH and RD showed that 1M aCGH by Agilent and 2.1M aCGH by NimbleGen can generate CNV findings complementary to each other, especially in the detection of short CNVs &lt;10 kb. Incorporating findings of present 1M aCGH-based studies and our previous 2.1M aCGH-based studies first proved that ultrahigh-density aCGH can achieve improved performance in CNV calling even more powerful than the sequence-based method in short CNV detection. Our findings herein not only were helpful for constructing a more comprehensive CNV map but also offered a feasible way to design a high-density aCGH with high cost efficiency in CNV studies in contrast to traditional sequence-based methods.</p><p id="p-42">In summary, the present study clearly highlights that different platforms can complement each other, and combined implementation of different platforms is beneficial to achieve the most comprehensive CNV calls. CNVs detected in diverse populations herein are essentially complementary to the CNV map in the pig genome, which would be helpful for understanding the pig genome variants and investigating the associations between various phenotypes and CNVs.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-43">The authors thank the editor and the two anonymous reviewers for their constructive comments and suggestions that greatly improved this manuscript. This work was financially supported by the Program for the National High Technology Research and Development Program of China (863 Program 2011AA100302, 2013AA102503), Changjiang Scholar and Innovation Research Team in University (IRT1191), the National Natural Science Foundations of China (31272419), and the New-Century Training Program Foundation for the Talents by the State Education Commission of China (NETC-10-0783).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-5"><p id="p-44">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.018473/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.018473/-/DC1</a></p></li><li class="fn" id="fn-6"><p id="p-45"><em>Communicating editor: I. M. Hall</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-12-07"><span class="received-label">Received </span>December 7, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-04-14"><span class="accepted-label">Accepted </span>April 14, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Wang <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.1" data-doi="10.1038/ng.437"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alkan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kidd</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marques-Bonet</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aksay</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antonacci</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Personalized copy number and segmental duplication maps using next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">1061</span>–<span class="cit-lpage">1067</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAlkan%26rft.auinit1%253DC.%26rft.volume%253D41%26rft.issue%253D10%26rft.spage%253D1061%26rft.epage%253D1067%26rft.atitle%253DPersonalized%2Bcopy%2Bnumber%2Band%2Bsegmental%2Bduplication%2Bmaps%2Busing%2Bnext-generation%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.437%26rft_id%253Dinfo%253Apmid%252F19718026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.437&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19718026&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270330400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ballester</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Castelló</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ibáñez</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sánchez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Folch</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Real-time quantitative PCR-based system for determining transgene copy number in transgenic animals.</span> <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">610</span>–<span class="cit-lpage">613</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioTechniques%26rft.stitle%253DBiotechniques%26rft.aulast%253DBallester%26rft.auinit1%253DM.%26rft.volume%253D37%26rft.issue%253D4%26rft.spage%253D610%26rft.epage%253D613%26rft.atitle%253DReal-time%2Bquantitative%2BPCR-based%2Bsystem%2Bfor%2Bdetermining%2Btransgene%2Bcopy%2Bnumber%2Bin%2Btransgenic%2Banimals.%26rft_id%253Dinfo%253Apmid%252F15517974%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15517974&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224539400018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.3" data-doi="10.1038/ng2028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">N. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Methods and strategies for analyzing copy number variation using DNA microarrays.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">S16</span>–<span class="cit-lpage">S21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DCarter%26rft.auinit1%253DN.%2BP.%26rft.volume%253D39%26rft.issue%253D7%2BSuppl%26rft.spage%253DS16%26rft.epage%253DS21%26rft.atitle%253DMethods%2Band%2Bstrategies%2Bfor%2Banalyzing%2Bcopy%2Bnumber%2Bvariation%2Busing%2BDNA%2Bmicroarrays.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2028%26rft_id%253Dinfo%253Apmid%252F17597776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17597776&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.4" data-doi="10.1186/1471-2164-13-733"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qiao</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ai</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A comprehensive survey of copy number variation in 18 diverse pig populations and identification of candidate copy number variable genes associated with complex traits.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">733</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D13%26rft.spage%253D733%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-733%26rft_id%253Dinfo%253Apmid%252F23270433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-733&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23270433&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.5" data-doi="10.1111/j.1365-2052.2012.02317.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clop</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vidal</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amills</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Copy number variation in the genomes of domestic animals.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">503</span>–<span class="cit-lpage">517</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DClop%26rft.auinit1%253DA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D503%26rft.epage%253D517%26rft.atitle%253DCopy%2Bnumber%2Bvariation%2Bin%2Bthe%2Bgenomes%2Bof%2Bdomestic%2Banimals.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2052.2012.02317.x%26rft_id%253Dinfo%253Apmid%252F22497594%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2052.2012.02317.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22497594&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.6" data-doi="10.1159/000184688"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Smith</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walters</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Froguel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blakemore</span>  <span class="cit-name-given-names">A. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Human genes involved in copy number variation: Mechanisms of origin, functional effects and implications for disease.</span> <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">17</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetic%2Band%2Bgenome%2Bresearch%26rft.stitle%253DCytogenet%2BGenome%2BRes%26rft.aulast%253Dde%2BSmith%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D123%26rft.issue%253D1-4%26rft.spage%253D17%26rft.epage%253D26%26rft.atitle%253DHuman%2Bgenes%2Binvolved%2Bin%2Bcopy%2Bnumber%2Bvariation%253A%2Bmechanisms%2Bof%2Borigin%252C%2Bfunctional%2Beffects%2Band%2Bimplications%2Bfor%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000184688%26rft_id%253Dinfo%253Apmid%252F19287135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000184688&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19287135&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264868300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.7" data-doi="10.1126/science.1158078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Emerson</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardoso-Moreira</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borevitz</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Natural selection shapes genome-wide patterns of copy-number polymorphism in Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">320</span>: <span class="cit-fpage">1629</span>–<span class="cit-lpage">1631</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1158078%26rft_id%253Dinfo%253Apmid%252F18535209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjAvNTg4My8xNjI5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.8" data-doi="10.1371/journal.pone.0003916"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fadista</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nygaard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holm</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomsen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bendixen</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A snapshot of CNVs in the pig genome.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3916</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DFadista%26rft.auinit1%253DJ.%26rft.volume%253D3%26rft.issue%253D12%26rft.spage%253De3916%26rft.epage%253De3916%26rft.atitle%253DA%2Bsnapshot%2Bof%2BCNVs%2Bin%2Bthe%2Bpig%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003916%26rft_id%253Dinfo%253Apmid%252F19079605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003916&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19079605&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.9" data-doi="10.1186/1471-2164-11-284"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fadista</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomsen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holm</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bendixen</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Copy number variation in the bovine genome.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">284</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DFadista%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.spage%253D284%26rft.epage%253D284%26rft.atitle%253DCopy%2Bnumber%2Bvariation%2Bin%2Bthe%2Bbovine%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-284%26rft_id%253Dinfo%253Apmid%252F20459598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-284&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20459598&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.10" data-doi="10.1111/j.1365-2052.2004.01234.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Braunschweig</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The phylogeny of Chinese indigenous pig breeds inferred from microsatellite markers.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">7</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DFang%26rft.auinit1%253DM.%26rft.volume%253D36%26rft.issue%253D1%26rft.spage%253D7%26rft.epage%253D13%26rft.atitle%253DThe%2Bphylogeny%2Bof%2BChinese%2Bindigenous%2Bpig%2Bbreeds%2Binferred%2Bfrom%2Bmicrosatellite%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2052.2004.01234.x%26rft_id%253Dinfo%253Apmid%252F15670125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2052.2004.01234.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15670125&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226473100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.11" data-doi="10.1038/nrg1767"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feuk</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carson</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scherer</span>  <span class="cit-name-given-names">S. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Structural variation in the human genome.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">97</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DFeuk%26rft.auinit1%253DL.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253D85%26rft.epage%253D97%26rft.atitle%253DStructural%2Bvariation%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1767%26rft_id%253Dinfo%253Apmid%252F16418744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1767&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16418744&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234714000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.12" data-doi="10.1101/gr.117242.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gazave</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Darré</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morcillo-Suarez</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petit-Marty</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carreño</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Copy number variation analysis in the great apes reveals species-specific patterns of structural variation.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1626</span>–<span class="cit-lpage">1639</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.117242.110%26rft_id%253Dinfo%253Apmid%252F21824994%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8xMC8xNjI2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.13" data-doi="10.1007/s00335-002-2184-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giuffra</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Törnsten</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marklund</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bongcam-Rudloff</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chardon</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A large duplication associated with dominant white color in pigs originated by homologous recombination between LINE elements flanking KIT.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">569</span>–<span class="cit-lpage">577</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DGiuffra%26rft.auinit1%253DE.%26rft.volume%253D13%26rft.issue%253D10%26rft.spage%253D569%26rft.epage%253D577%26rft.atitle%253DA%2Blarge%2Bduplication%2Bassociated%2Bwith%2Bdominant%2Bwhite%2Bcolor%2Bin%2Bpigs%2Boriginated%2Bby%2Bhomologous%2Brecombination%2Bbetween%2BLINE%2Belements%2Bflanking%2BKIT.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-002-2184-5%26rft_id%253Dinfo%253Apmid%252F12420135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-002-2184-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12420135&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179056500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.14" data-doi="10.1371/journal.pone.0027859"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haraksingh</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abyzov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerstein</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Urban</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide mapping of copy number variation in humans: Comparative analysis of high resolution array platforms.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e27859</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DHaraksingh%26rft.auinit1%253DR.%2BR.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253De27859%26rft.epage%253De27859%26rft.atitle%253DGenome-wide%2Bmapping%2Bof%2Bcopy%2Bnumber%2Bvariation%2Bin%2Bhumans%253A%2Bcomparative%2Banalysis%2Bof%2Bhigh%2Bresolution%2Barray%2Bplatforms.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0027859%26rft_id%253Dinfo%253Apmid%252F22140474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0027859&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22140474&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.15" data-doi="10.1186/1471-2164-12-127"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bickhart</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardone</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic characteristics of cattle copy number variations.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DHou%26rft.auinit1%253DY.%26rft.volume%253D12%26rft.spage%253D127%26rft.epage%253D127%26rft.atitle%253DGenomic%2Bcharacteristics%2Bof%2Bcattle%2Bcopy%2Bnumber%2Bvariations.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-127%26rft_id%253Dinfo%253Apmid%252F21345189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-127&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21345189&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.16" data-doi="10.1038/ng1416"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iafrate</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feuk</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rivera</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Listewnik</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donahoe</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Detection of large-scale variation in the human genome.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">949</span>–<span class="cit-lpage">951</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DIafrate%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D36%26rft.issue%253D9%26rft.spage%253D949%26rft.epage%253D951%26rft.atitle%253DDetection%2Bof%2Blarge-scale%2Bvariation%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1416%26rft_id%253Dinfo%253Apmid%252F15286789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1416&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15286789&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223658100018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.17" data-doi="10.1186/1471-2164-15-593"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Global copy number analyses by next generation sequencing provide insight into pig genome.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">593</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D593%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-593%26rft_id%253Dinfo%253Apmid%252F25023178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-593&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25023178&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.18" data-doi="10.1111/j.1365-2052.2005.01320.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johansson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pielberg</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edfors-lilja</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Polymorphism at the porcine dominant white/KIT locus influence coat colour and peripheral blood cell measures.</span> <abbr class="cit-jnl-abbrev">Anim. Gent.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">288</span>–<span class="cit-lpage">296</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BGent.%26rft.volume%253D36%26rft.spage%253D288%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2052.2005.01320.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2052.2005.01320.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.19" data-doi="10.1186/1471-2164-13-725"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mei</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Identification of genome-wide copy number variations among diverse pig breeds by array CGH.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">725</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D13%26rft.spage%253D725%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-725%26rft_id%253Dinfo%253Apmid%252F23265576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-725&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23265576&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.20" data-doi="10.1101/gr.105403.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardone</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Analysis of copy number variations among diverse cattle breeds.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">693</span>–<span class="cit-lpage">703</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.105403.110%26rft_id%253Dinfo%253Apmid%252F20212021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzUvNjkzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.21" data-doi="10.1006/meth.2001.1262"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Livak</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmittgen</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Analysis of relative gene expression data using real-time quantitative PCR and the 2-[Delta][Delta] CT method.</span> <abbr class="cit-jnl-abbrev">Methods</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">402</span>–<span class="cit-lpage">408</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2B%2528San%2BDiego%252C%2BCalif.%2529%26rft.stitle%253DMethods%26rft.aulast%253DLivak%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D25%26rft.issue%253D4%26rft.spage%253D402%26rft.epage%253D408%26rft.atitle%253DAnalysis%2Bof%2Brelative%2Bgene%2Bexpression%2Bdata%2Busing%2Breal-time%2Bquantitative%2BPCR%2Band%2Bthe%2B2%2528-Delta%2BDelta%2BC%2528T%2529%2529%2BMethod.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fmeth.2001.1262%26rft_id%253Dinfo%253Apmid%252F11846609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/meth.2001.1262&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11846609&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173949500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.22" data-doi="10.1016/j.tim.2011.11.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meurens</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Summerfield</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nauwynck</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saif</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerdts</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The pig: a model for human infectious diseases.</span> <abbr class="cit-jnl-abbrev">Trends Microbiol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">50</span>–<span class="cit-lpage">57</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bmicrobiology%26rft.stitle%253DTrends%2BMicrobiol%26rft.aulast%253DMeurens%26rft.auinit1%253DF.%26rft.volume%253D20%26rft.issue%253D1%26rft.spage%253D50%26rft.epage%253D57%26rft.atitle%253DThe%2Bpig%253A%2Ba%2Bmodel%2Bfor%2Bhuman%2Binfectious%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2011.11.002%26rft_id%253Dinfo%253Apmid%252F22153753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2011.11.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22153753&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300656000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.23" data-doi="10.1038/nature09708"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mills</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walter</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mapping copy number variation by population-scale genome sequencing.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">470</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMills%26rft.auinit1%253DR.%2BE.%26rft.volume%253D470%26rft.issue%253D7332%26rft.spage%253D59%26rft.epage%253D65%26rft.atitle%253DMapping%2Bcopy%2Bnumber%2Bvariation%2Bby%2Bpopulation-scale%2Bgenome%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09708%26rft_id%253Dinfo%253Apmid%252F21293372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09708&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21293372&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286886400033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.24" data-doi="10.1186/1471-2164-14-449"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paudel</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madsen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Megens</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frantz</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bosse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Evolutionary dynamics of copy number variation in pig genomes in the context of adaptation and domestication.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">449</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D449%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-449%26rft_id%253Dinfo%253Apmid%252F23829399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-449&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23829399&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pielberg</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Syvänen</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Unexpectedly high allelic diversity at the KIT locus causing dominant white color in the domestic pig.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">305</span>–<span class="cit-lpage">311</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPielberg%26rft.auinit1%253DG.%26rft.volume%253D160%26rft.issue%253D1%26rft.spage%253D305%26rft.epage%253D311%26rft.atitle%253DUnexpectedly%2BHigh%2BAllelic%2BDiversity%2Bat%2Bthe%2BKIT%2BLocus%2BCausing%2BDominant%2BWhite%2BColor%2Bin%2Bthe%2BDomestic%2BPig%26rft_id%253Dinfo%253Apmid%252F11805065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYwLzEvMzA1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.26" data-doi="10.1038/2524"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pinkel</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Segraves</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sudar</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poole</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">High resolution analysis of DNA copy number variation using comparative genomic hybridization to microarrays.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DPinkel%26rft.auinit1%253DD.%26rft.volume%253D20%26rft.issue%253D2%26rft.spage%253D207%26rft.epage%253D211%26rft.atitle%253DHigh%2Bresolution%2Banalysis%2Bof%2BDNA%2Bcopy%2Bnumber%2Bvariation%2Busing%2Bcomparative%2Bgenomic%2Bhybridization%2Bto%2Bmicroarrays.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F2524%26rft_id%253Dinfo%253Apmid%252F9771718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/2524&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9771718&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000076231300027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.27" data-doi="10.1038/nbt.1852"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pinto</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Darvishi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rajan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rigler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Comprehensive assessment of array-based platforms and calling algorithms for detection of copy number variants.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">512</span>–<span class="cit-lpage">520</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DPinto%26rft.auinit1%253DD.%26rft.volume%253D29%26rft.issue%253D6%26rft.spage%253D512%26rft.epage%253D520%26rft.atitle%253DComprehensive%2Bassessment%2Bof%2Barray-based%2Bplatforms%2Band%2Bcalling%2Balgorithms%2Bfor%2Bdetection%2Bof%2Bcopy%2Bnumber%2Bvariants.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1852%26rft_id%253Dinfo%253Apmid%252F21552272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1852&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21552272&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291342100017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.28" data-doi="10.1186/1471-2164-11-593"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramayo-Caldas</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Castelló</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pena</span>  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alves</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mercadé</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Copy number variation in the porcine genome inferred from a 60 k SNP BeadChip.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">593</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DRamayo%2BCaldas%26rft.auinit1%253DY.%26rft.volume%253D11%26rft.spage%253D593%26rft.epage%253D593%26rft.atitle%253DCopy%2Bnumber%2Bvariation%2Bin%2Bthe%2Bporcine%2Bgenome%2Binferred%2Bfrom%2Ba%2B60%2Bk%2BSNP%2BBeadChip.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-593%26rft_id%253Dinfo%253Apmid%252F20969757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-593&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20969757&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.29" data-doi="10.1038/nature05329"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Redon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishikawa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fitch</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feuk</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perry</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Global variation in copy number in the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">444</span>: <span class="cit-fpage">444</span>–<span class="cit-lpage">454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRedon%26rft.auinit1%253DR.%26rft.volume%253D444%26rft.issue%253D7118%26rft.spage%253D444%26rft.epage%253D454%26rft.atitle%253DGlobal%2Bvariation%2Bin%2Bcopy%2Bnumber%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05329%26rft_id%253Dinfo%253Apmid%252F17122850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05329&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17122850&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242215700038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.30" data-doi="10.1038/ng.185"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosengren Pielberg</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Golovko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sundström</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curik</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lennartsson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A cis-acting regulatory mutation causes premature hair graying and susceptibility to melanoma in the horse.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1004</span>–<span class="cit-lpage">1009</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DRosengren%2BPielberg%26rft.auinit1%253DG.%26rft.volume%253D40%26rft.issue%253D8%26rft.spage%253D1004%26rft.epage%253D1009%26rft.atitle%253DA%2Bcis-acting%2Bregulatory%2Bmutation%2Bcauses%2Bpremature%2Bhair%2Bgraying%2Band%2Bsusceptibility%2Bto%2Bmelanoma%2Bin%2Bthe%2Bhorse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.185%26rft_id%253Dinfo%253Apmid%252F18641652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.185&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18641652&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258026900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.31" data-doi="10.1073/pnas.1217149109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Megens</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrio</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maqbool</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sayyab</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Strong signatures of selection in the domestic pig genome.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">19529</span>–<span class="cit-lpage">19536</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1217149109%26rft_id%253Dinfo%253Apmid%252F23151514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzQ4LzE5NTI5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.32" data-doi="10.1098/rspb.2010.1180"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Gene copy-number polymorphism in nature.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">277</span>: <span class="cit-fpage">3213</span>–<span class="cit-lpage">3221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2010.1180%26rft_id%253Dinfo%253Apmid%252F20591863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMzoiMjc3LzE2OTgvMzIxMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMjUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.33" data-doi="10.1126/science.1098918"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sebat</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lakshmi</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troge</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Large-scale copy number polymorphism in the human genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">305</span>: <span class="cit-fpage">525</span>–<span class="cit-lpage">528</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSebat%26rft.auinit1%253DJ.%26rft.volume%253D305%26rft.issue%253D5683%26rft.spage%253D525%26rft.epage%253D528%26rft.atitle%253DLarge-Scale%2BCopy%2BNumber%2BPolymorphism%2Bin%2Bthe%2BHuman%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1098918%26rft_id%253Dinfo%253Apmid%252F15273396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDUvNTY4My81MjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTI1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.34" data-doi="10.1186/1471-2164-11-673"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seroussi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glick</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shirak</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yakobson</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weller</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Analysis of copy loss and gain variations in Holstein cattle autosomes using BeadChip SNPs.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">673</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DSeroussi%26rft.auinit1%253DE.%26rft.volume%253D11%26rft.spage%253D673%26rft.epage%253D673%26rft.atitle%253DAnalysis%2Bof%2Bcopy%2Bloss%2Band%2Bgain%2Bvariations%2Bin%2BHolstein%2Bcattle%2Bautosomes%2Busing%2BBeadChip%2BSNPs.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-673%26rft_id%253Dinfo%253Apmid%252F21114805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-673&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21114805&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.35" data-doi="10.1126/science.1197005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sudmant</span>  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kitzman</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antonacci</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alkan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malig</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Diversity of human copy number variation and multicopy genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">330</span>: <span class="cit-fpage">641</span>–<span class="cit-lpage">646</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSudmant%26rft.auinit1%253DP.%2BH.%26rft.volume%253D330%26rft.issue%253D6004%26rft.spage%253D641%26rft.epage%253D646%26rft.atitle%253DDiversity%2Bof%2BHuman%2BCopy%2BNumber%2BVariation%2Band%2BMulticopy%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1197005%26rft_id%253Dinfo%253Apmid%252F21030649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzAvNjAwNC82NDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTI1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.36" data-doi="10.1093/bioinformatics/btl117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimodaira</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Pvclust: an R package for assessing the uncertainty in hierarchical clustering.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1540</span>–<span class="cit-lpage">1542</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl117%26rft_id%253Dinfo%253Apmid%252F16595560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjIvMTIvMTU0MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMjUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.37" data-doi="10.1186/1471-2164-14-262"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Copy number variants in locally raised Chinese chicken genomes determined using array comparative genomic hybridization.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">262</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D262%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-262%26rft_id%253Dinfo%253Apmid%252F23594354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-262&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23594354&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.38" data-doi="10.1038/ng1562"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tuzun</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaul</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morrison</span>  <span class="cit-name-given-names">V. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Fine-scale structural variation of the human genome.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">727</span>–<span class="cit-lpage">732</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTuzun%26rft.auinit1%253DE.%26rft.volume%253D37%26rft.issue%253D7%26rft.spage%253D727%26rft.epage%253D732%26rft.atitle%253DFine-scale%2Bstructural%2Bvariation%2Bof%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1562%26rft_id%253Dinfo%253Apmid%252F15895083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1562&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15895083&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230196400019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.39" data-doi="10.1101/gr.073585.107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vilella</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Severin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ureta-Vidal</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heng</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ensembl Compara GeneTrees: Complete, duplication-aware phylogenetic trees in vertebrates.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">335</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.073585.107%26rft_id%253Dinfo%253Apmid%252F19029536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzIvMzI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.40" data-doi="10.1186/1471-2164-13-273"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A genome-wide detection of copy number variations using SNP genotyping arrays in swine.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">273</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DWang%26rft.auinit1%253DJ.%26rft.volume%253D13%26rft.spage%253D273%26rft.epage%253D273%26rft.atitle%253DA%2Bgenome-wide%2Bdetection%2Bof%2Bcopy%2Bnumber%2Bvariations%2Busing%2BSNP%2Bgenotyping%2Barrays%2Bin%2Bswine.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-273%26rft_id%253Dinfo%253Apmid%252F22726314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-273&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22726314&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.41" data-doi="10.1371/journal.pone.0068683"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Identification of Genome-Wide Copy Number Variations among Diverse Pig Breeds Using SNP Genotyping Arrays.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e68683</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D8%26rft.spage%253De68683%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0068683%26rft_id%253Dinfo%253Apmid%252F23935880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0068683&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23935880&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.42" data-doi="10.1371/journal.pone.0087571"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Enhancing Genome-Wide Copy Number Variation Identification by High Density Array CGH Using Diverse Resources of Pig Breeds.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e87571</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D9%26rft.spage%253De87571%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0087571%26rft_id%253Dinfo%253Apmid%252F24475311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0087571&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24475311&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.43" data-doi="10.1371/journal.pgen.1000512"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boije</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meadows</span>  <span class="cit-name-given-names">J. R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bed’hom</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gourichon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Copy number variation in intron 1 of SOX5 causes the Pea-comb phenotype in chickens.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWright%26rft.auinit1%253DD.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253De1000512%26rft.epage%253De1000512%26rft.atitle%253DCopy%2Bnumber%2Bvariation%2Bin%2Bintron%2B1%2Bof%2BSOX5%2Bcauses%2Bthe%2BPea-comb%2Bphenotype%2Bin%2Bchickens.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000512%26rft_id%253Dinfo%253Apmid%252F19521496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000512&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19521496&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1253.44" data-doi="10.2174/138920211795564386"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plastow</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic Diversity in Pig (Sus scrofa) and its Comparison with Human and other Livestock.</span> <abbr class="cit-jnl-abbrev">Curr. Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">138</span>–<span class="cit-lpage">146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCurr%2BGenomics%26rft.aulast%253DZhang%26rft.auinit1%253DC.%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253D138%26rft.epage%253D146%26rft.atitle%253DGenomic%2BDiversity%2Bin%2BPig%2B%2528Sus%2Bscrofa%2529%2Band%2Bits%2BComparison%2Bwith%2BHuman%2Band%2Bother%2BLivestock.%26rft_id%253Dinfo%253Adoi%252F10.2174%252F138920211795564386%26rft_id%253Dinfo%253Apmid%252F21966252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2174/138920211795564386&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21966252&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000290617200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-book" id="cit-5.6.1253.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-source">Pig Breeds in China</span>, <span class="cit-publ-name">Shanghai Scientific and Technical Publisher</span>, <span class="cit-publ-loc">Shanghai</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/6/1253.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/6/1247" title="Effects of Genic Base Composition on Growth Rate in G+C-rich Genomes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/6/1263" title="Neurospora Heterokaryons with Complementary Duplications and Deficiencies in Their Constituent Nuclei Provide an Approach to Identify Nucleus-Limited Genes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/6"><span>Volume 5 Issue 6, June 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/6.cover-source.jpg?itok=NSNFWJbY" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1253.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-OSsa8btf-0phjDmSC50SvzyZQk_y-gX87fktTPWpZ9M" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/6/1253" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/6/1253" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/6/1253" class="active" data-icon-position="" data-hide-link-title="0">Improved Detection and Characterization of Copy Number Variations Among Diverse Pig Breeds by Array CGH</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459919" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-IgyClrBZWSaWmn-C88G3MuhPdkezE5OF7SSRvkGkFWM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/6/1253.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/6/1253" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-huNpFRnmj4CU-ZX90bOSPA_v83ii47-8R6cg7wGkazU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/6/1253" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/6/1253.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459919" id="node459919--2" data-pisa="ggg;5/6/1253" data-pisa-master="ggg;g3.115.018473" data-apath="/ggg/5/6/1253.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Improved Detection and Characterization of Copy Number Variations Among Diverse Pig Breeds by Array CGH</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jiying Wang</span>, <span class="highwire-citation-author" data-delta="1">Jicai Jiang</span>, <span class="highwire-citation-author" data-delta="2">Haifei Wang</span>, <span class="highwire-citation-author" data-delta="3">Huimin Kang</span>, <span class="highwire-citation-author" data-delta="4">Qin Zhang</span> and <span class="highwire-citation-author" data-delta="5">Jian-Feng Liu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1253-1261;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.018473 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459919--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jiying Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Key Laboratory of Animal Genetics, Breeding, and Reproduction, Ministry of Agriculture, National Engineering Laboratory for Animal Breeding, College of Animal Science and Technology, China Agricultural University, Beijing 100193, China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Shandong Provincial Key Laboratory of Animal Disease Control and Breeding, Institute of Animal Science and Veterinary Medicine, Shandong Academy of Agricultural Sciences, Jinan 250100, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiying%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiying%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jicai Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Key Laboratory of Animal Genetics, Breeding, and Reproduction, Ministry of Agriculture, National Engineering Laboratory for Animal Breeding, College of Animal Science and Technology, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jicai%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJicai%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Haifei Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Key Laboratory of Animal Genetics, Breeding, and Reproduction, Ministry of Agriculture, National Engineering Laboratory for Animal Breeding, College of Animal Science and Technology, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haifei%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHaifei%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Huimin Kang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Key Laboratory of Animal Genetics, Breeding, and Reproduction, Ministry of Agriculture, National Engineering Laboratory for Animal Breeding, College of Animal Science and Technology, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Huimin%2BKang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHuimin%2BKang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Qin Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Key Laboratory of Animal Genetics, Breeding, and Reproduction, Ministry of Agriculture, National Engineering Laboratory for Animal Breeding, College of Animal Science and Technology, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qin%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQin%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jian-Feng Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Key Laboratory of Animal Genetics, Breeding, and Reproduction, Ministry of Agriculture, National Engineering Laboratory for Animal Breeding, College of Animal Science and Technology, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jian-Feng%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJian-Feng%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:liujf@cau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">liujf@cau.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/6/1253" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/6/1253&amp;title=Improved%20Detection%20and%20Characterization%20of%20Copy%20Number%20Variations%20Among%20Diverse%20Pig%20Breeds%20by%20Array%20CGH" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/6/1253&amp;title=Improved%20Detection%20and%20Characterization%20of%20Copy%20Number%20Variations%20Among%20Diverse%20Pig%20Breeds%20by%20Array%20CGH" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/6/1253&amp;title=Improved%20Detection%20and%20Characterization%20of%20Copy%20Number%20Variations%20Among%20Diverse%20Pig%20Breeds%20by%20Array%20CGH" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/6/1253&amp;text=Improved%20Detection%20and%20Characterization%20of%20Copy%20Number%20Variations%20Among%20Diverse%20Pig%20Breeds%20by%20Array%20CGH" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/6/1253&amp;title=Improved%20Detection%20and%20Characterization%20of%20Copy%20Number%20Variations%20Among%20Diverse%20Pig%20Breeds%20by%20Array%20CGH" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/6/1253&amp;t=Improved%20Detection%20and%20Characterization%20of%20Copy%20Number%20Variations%20Among%20Diverse%20Pig%20Breeds%20by%20Array%20CGH" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/6/1253&amp;title=Improved%20Detection%20and%20Characterization%20of%20Copy%20Number%20Variations%20Among%20Diverse%20Pig%20Breeds%20by%20Array%20CGH" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/6/1253&amp;title=Improved%20Detection%20and%20Characterization%20of%20Copy%20Number%20Variations%20Among%20Diverse%20Pig%20Breeds%20by%20Array%20CGH" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459919/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459919" id="node459919--3" data-pisa="ggg;5/6/1253" data-pisa-master="ggg;g3.115.018473" data-apath="/ggg/5/6/1253.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Improved Detection and Characterization of Copy Number Variations Among Diverse Pig Breeds by Array CGH</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jiying Wang</span>, <span class="highwire-citation-author" data-delta="1">Jicai Jiang</span>, <span class="highwire-citation-author" data-delta="2">Haifei Wang</span>, <span class="highwire-citation-author" data-delta="3">Huimin Kang</span>, <span class="highwire-citation-author" data-delta="4">Qin Zhang</span> and <span class="highwire-citation-author" data-delta="5">Jian-Feng Liu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1253-1261;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.018473 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459919--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jiying Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Key Laboratory of Animal Genetics, Breeding, and Reproduction, Ministry of Agriculture, National Engineering Laboratory for Animal Breeding, College of Animal Science and Technology, China Agricultural University, Beijing 100193, China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Shandong Provincial Key Laboratory of Animal Disease Control and Breeding, Institute of Animal Science and Veterinary Medicine, Shandong Academy of Agricultural Sciences, Jinan 250100, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiying%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiying%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jicai Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Key Laboratory of Animal Genetics, Breeding, and Reproduction, Ministry of Agriculture, National Engineering Laboratory for Animal Breeding, College of Animal Science and Technology, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jicai%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJicai%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Haifei Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Key Laboratory of Animal Genetics, Breeding, and Reproduction, Ministry of Agriculture, National Engineering Laboratory for Animal Breeding, College of Animal Science and Technology, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haifei%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHaifei%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Huimin Kang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Key Laboratory of Animal Genetics, Breeding, and Reproduction, Ministry of Agriculture, National Engineering Laboratory for Animal Breeding, College of Animal Science and Technology, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Huimin%2BKang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHuimin%2BKang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Qin Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Key Laboratory of Animal Genetics, Breeding, and Reproduction, Ministry of Agriculture, National Engineering Laboratory for Animal Breeding, College of Animal Science and Technology, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qin%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQin%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jian-Feng Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Key Laboratory of Animal Genetics, Breeding, and Reproduction, Ministry of Agriculture, National Engineering Laboratory for Animal Breeding, College of Animal Science and Technology, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jian-Feng%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJian-Feng%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:liujf@cau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">liujf@cau.edu.cn</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1253&amp;rft.epage=1261&amp;rft.atitle=Improved%20Detection%20and%20Characterization%20of%20Copy%20Number%20Variations%20Among%20Diverse%20Pig%20Breeds%20by%20Array%20CGH&amp;rft.volume=5&amp;rft.issue=6&amp;rft.date=2015-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Wang%2C+Jiying&amp;rft.au=Jiang%2C+Jicai&amp;rft.au=Wang%2C+Haifei&amp;rft.au=Kang%2C+Huimin&amp;rft.au=Zhang%2C+Qin&amp;rft.au=Liu%2C+Jian-Feng"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459919/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459919/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459919/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459919/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459919/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459919/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459919/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459919/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459919/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459919/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459919/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459919/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/6/1253" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/6/1253.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/6/1253.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/6/1253.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/6/1253.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
