<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/6/1081" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/6/1081.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/6/1081.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/6/1081.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/6/1081.atom" />
<meta name="HW.pisa" content="ggg;5/6/1081" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Insights into Ongoing Evolution of the Hexachlorocyclohexane Catabolic Pathway from Comparative Genomics of Ten Sphingomonadaceae Strains" />
<meta name="DC.Identifier" content="10.1534/g3.114.015933" />
<meta name="DC.Date" content="2015-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Pearce et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Hexachlorocyclohexane (HCH), a synthetic organochloride, was first used as a broad-acre insecticide in the 1940s, and many HCH-degrading bacterial strains have been isolated from around the globe during the last 20 years. To date, the same degradation pathway (the lin pathway) has been implicated in all strains characterized, although the pathway has only been characterized intensively in two strains and for only a single HCH isomer. To further elucidate the evolution of the lin pathway, we have biochemically and genetically characterized three HCH-degrading strains from the Czech Republic and compared the genomes of these and seven other HCH-degrading bacterial strains. The three new strains each yielded a distinct set of metabolites during their degradation of HCH isomers. Variable assembly of the pathway is a common feature across the 10 genomes, eight of which (including all three Czech strains) were either missing key lin genes or containing duplicate copies of upstream lin genes ( linA-F ). The analysis also confirmed the important role of horizontal transfer mediated by insertion sequence IS6100 in the acquisition of the pathway, with a stronger association of IS6100 to the lin genes in the new strains. In one strain, a linA variant was identified that likely caused a novel degradation phenotype involving a shift in isomer preference. This study identifies a number of strains that are in the early stages of lin pathway acquisition and shows that the state of the pathway can explain the degradation patterns observed." />
<meta name="DC.Contributor" content="Stephen L. Pearce" />
<meta name="DC.Contributor" content="John G. Oakeshott" />
<meta name="DC.Contributor" content="Gunjan Pandey" />
<meta name="article:published_time" content="2015-06-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Insights into Ongoing Evolution of the Hexachlorocyclohexane Catabolic Pathway from Comparative Genomics of Ten Sphingomonadaceae Strains" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Hexachlorocyclohexane (HCH), a synthetic organochloride, was first used as a broad-acre insecticide in the 1940s, and many HCH-degrading bacterial strains have been isolated from around the globe during the last 20 years. To date, the same degradation pathway (the &lt;i&gt;lin&lt;/i&gt; pathway) has been implicated in all strains characterized, although the pathway has only been characterized intensively in two strains and for only a single HCH isomer. To further elucidate the evolution of the &lt;i&gt;lin&lt;/i&gt; pathway, we have biochemically and genetically characterized three HCH-degrading strains from the Czech Republic and compared the genomes of these and seven other HCH-degrading bacterial strains. The three new strains each yielded a distinct set of metabolites during their degradation of HCH isomers. Variable assembly of the pathway is a common feature across the 10 genomes, eight of which (including all three Czech strains) were either missing key &lt;i&gt;lin&lt;/i&gt; genes or containing duplicate copies of upstream &lt;i&gt;lin&lt;/i&gt; genes (&lt;i&gt;linA-F&lt;/i&gt;). The analysis also confirmed the important role of horizontal transfer mediated by insertion sequence IS6100 in the acquisition of the pathway, with a stronger association of IS6100 to the &lt;i&gt;lin&lt;/i&gt; genes in the new strains. In one strain, a &lt;i&gt;linA&lt;/i&gt; variant was identified that likely caused a novel degradation phenotype involving a shift in isomer preference. This study identifies a number of strains that are in the early stages of &lt;i&gt;lin&lt;/i&gt; pathway acquisition and shows that the state of the pathway can explain the degradation patterns observed.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/06/01" />
<meta name="citation_mjid" content="ggg;5/6/1081" />
<meta name="citation_id" content="5/6/1081" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/6/1081" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/6/1081.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/6/1081.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/6/1081.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.015933" />
<meta name="citation_pmid" content="25850427" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1081" />
<meta name="citation_lastpage" content="1094" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Stephen L. Pearce" />
<meta name="citation_author_institution" content="CSIRO Ecosystem Sciences, Acton, ACT-2601, Australia" />
<meta name="citation_author" content="John G. Oakeshott" />
<meta name="citation_author_institution" content="CSIRO Ecosystem Sciences, Acton, ACT-2601, Australia" />
<meta name="citation_author" content="Gunjan Pandey" />
<meta name="citation_author_institution" content="CSIRO Ecosystem Sciences, Acton, ACT-2601, Australia" />
<meta name="citation_author_email" content="gunjan.pandey@csiro.au" />
<meta name="citation_reference" content="citation_journal_title=Omics :  a journal of integrative biology;citation_journal_abbrev=OMICS;citation_author=S. V. Angiuoli;citation_author=A. Gussman;citation_author=W. Klimke;citation_author=G. Cochrane;citation_author=D. Field;citation_title=Toward an online repository of Standard Operating Procedures (SOPs) for (meta)genomic annotation.;citation_pages=137-141;citation_volume=12;citation_year=2008;citation_issue=2;citation_pmid=18416670;citation_doi=10.1089/omi.2008.0017" />
<meta name="citation_reference" content="citation_journal_title=Environmental Microbiology (Print);citation_journal_abbrev=Environmental Microbiology (Print);citation_author=D. Böltner;citation_author=S. Moreno-Morillas;citation_author=J. L. Ramos;citation_title=16S rDNA phylogeny and distribution of lin genes in novel hexachlorocyclohexane-degrading Sphingomonas strains.;citation_pages=1329-1338;citation_volume=7;citation_year=2005;citation_issue=9;citation_pmid=16104856;citation_doi=10.1111/j.1462-5822.2005.00820.x" />
<meta name="citation_reference" content="citation_journal_title=The Science of the Total Environment;citation_journal_abbrev=The Science of the Total Environment;citation_author=K. Breivik;citation_author=J. M. Pacyna;citation_author=J. Münch;citation_title=Use of alpha-, beta- and gamma-hexachlorocyclohexane in Europe, 1970-1996.;citation_pages=151-163;citation_volume=239;citation_year=1999;citation_issue=1-3;citation_pmid=10636769;citation_doi=10.1016/S0048-9697(99)00291-0" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. Chuzeville;citation_author=A. Puymège;citation_author=J. Y. Madec;citation_author=M. Haenni;citation_author=S. Payot;citation_title=Characterization of a New CAMP Factor Carried by an Integrative and Conjugative Element in Streptococcus agalactiae and Spreading in Streptococci.;citation_pages=e48918-e48918;citation_volume=7;citation_year=2012;citation_issue=11;citation_pmid=23152820;citation_doi=10.1371/journal.pone.0048918" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=S. D. Copley;citation_author=J. Rokicki;citation_author=P. Turner;citation_author=H. Daligault;citation_author=M. Nolan;citation_title=The whole genome sequence of Sphingobium chlorophenolicum L-1: insights into the evolution of the pentachlorophenol degradation pathway.;citation_volume=4;citation_year=2012;citation_pmid=22179583;citation_doi=10.1093/gbe/evr137" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=A. L. Delcher;citation_author=A. Phillippy;citation_author=J. Carlton;citation_author=S. L. Salzberg;citation_title=Fast algorithms for large-scale genome alignment and comparison;citation_pages=2478-2483;citation_volume=30;citation_year=2002;citation_issue=11;citation_pmid=12034836;citation_doi=10.1093/nar/30.11.2478" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=C. Dogra;citation_author=V. Raina;citation_author=R. Pal;citation_author=M. Suar;citation_author=S. Lal;citation_title=Organization of lin Genes and IS6100 among Different Strains of Hexachlorocyclohexane-Degrading Sphingomonas paucimobilis: Evidence for Horizontal Gene Transfer;citation_pages=2225-2235;citation_volume=186;citation_year=2004;citation_issue=8;citation_pmid=15060023;citation_doi=10.1128/JB.186.8.2225-2235.2004" />
<meta name="citation_reference" content="citation_journal_title=Genome Announc.;citation_author=C. Dong;citation_author=X. Bai;citation_author=Q. Lai;citation_author=Y. Xie;citation_author=X. Chen;citation_title=Draft genome sequence of Sphingobium sp. strain C100, a polycyclic aromatic hydrocarbon-degrading bacterium from the deep-sea sediment of the Arctic Ocean.;citation_pages=e01210-e01213;citation_volume=2;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Journal of Agricultural and Food Chemistry;citation_journal_abbrev=Journal of Agricultural and Food Chemistry;citation_author=C. D. Elcey;citation_author=A. A. M. Kunhi;citation_title=Substantially enhanced degradation of hexachlorocyclohexane isomers by a microbial consortium on acclimation.;citation_pages=1046-1054;citation_volume=58;citation_year=2010;citation_issue=2;citation_pmid=20041660;citation_doi=10.1021/jf9038259" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=R. Fani;citation_author=M. Brilli;citation_author=P. Liò;citation_title=The origin and evolution of operons: the piecewise building of the proteobacterial histidine operon.;citation_pages=378-390;citation_volume=60;citation_year=2005;citation_issue=3;citation_pmid=15871048;citation_doi=10.1007/s00239-004-0198-1" />
<meta name="citation_reference" content="citation_journal_title=Research in Microbiology (Paris);citation_journal_abbrev=Research in Microbiology (Paris);citation_author=M. Fondi;citation_author=G. Emiliani;citation_author=R. Fani;citation_title=Origin and evolution of operons and metabolic pathways.;citation_pages=502-512;citation_volume=160;citation_year=2009;citation_issue=7;citation_pmid=19465116;citation_doi=10.1016/j.resmic.2009.05.001" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Z. Gai;citation_author=X. Wang;citation_author=X. Liu;citation_author=C. Tai;citation_author=H. Tang;citation_title=The genes coding for the conversion of carbazole to catechol are flanked by IS6100 elements in Sphingomonas sp. strain XLDN2-5.;citation_pages=e10018-e10018;citation_volume=5;citation_year=2010;citation_issue=4;citation_pmid=20368802;citation_doi=10.1371/journal.pone.0010018" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=M. Gaillard;citation_author=T. Vallaeys;citation_author=F. J. Vorhölter;citation_author=M. Minoia;citation_author=C. Werlen;citation_title=The clc Element of Pseudomonas sp. Strain B13, a Genomic Island with Various Catabolic Properties;citation_pages=1999-2013;citation_volume=188;citation_year=2006;citation_issue=5;citation_pmid=16484212;citation_doi=10.1128/JB.188.5.1999-2013.2006" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=H. M. Gan;citation_author=T. H. Chew;citation_author=A. O. Hudson;citation_author=M. A. Savka;citation_title=Genome Sequence of Novosphingobium sp. Strain Rr 2-17, a Nopaline Crown Gall-Associated Bacterium Isolated from Vitis vinifera L. Grapevine;citation_pages=5137-5138;citation_volume=194;citation_year=2012;citation_issue=18;citation_pmid=22933764;citation_doi=10.1128/JB.01159-12" />
<meta name="citation_reference" content="citation_journal_title=Environ. Microbiol.;citation_author=B. Geueke;citation_author=N. Garg;citation_author=S. Ghosh;citation_author=T. Fleischmann;citation_author=C. Holliger;citation_title=Metabolomics of hexachlorocyclohexane (HCH) transformation: ratio of LinA to LinB determines metabolic fate of HCH isomers.;citation_pages=1040-1049;citation_volume=15;citation_year=2013;citation_doi=10.1111/1462-2920.12009" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. Guglielmini;citation_author=L. Quintais;citation_author=M. P. Garcillán-Barcia;citation_author=F. de la Cruz;citation_author=E. P. C. Rocha;citation_title=The repertoire of ICE in prokaryotes underscores the unity, diversity, and ubiquity of conjugation.;citation_pages=e1002222-e1002222;citation_volume=7;citation_year=2011;citation_issue=8;citation_pmid=21876676;citation_doi=10.1371/journal.pgen.1002222" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=L. Guy;citation_author=J. R. Kultima;citation_author=S. G. E. Andersson;citation_title=genoPlotR: comparative gene and genome visualization in R.;citation_volume=26;citation_year=2010;citation_pmid=20624783;citation_doi=10.1093/bioinformatics/btq413" />
<meta name="citation_reference" content="citation_journal_abbrev=Bioinformation;citation_author=M. S. Hasan;citation_author=Q. Liu;citation_author=H. Wang;citation_author=J. Fazekas;citation_author=B. Chen;citation_title=GIST: Genomic island suite of tools for predicting genomic islands in genomic sequences.;citation_pages=203-205;citation_volume=8;citation_year=2012;citation_issue=4;citation_pmid=22419842;citation_doi=10.6026/97320630008203" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Lett.;citation_author=P. R. Herron;citation_author=M. C. Evans;citation_author=P. J. Dyson;citation_title=Low target site specificity of an IS6100 -based mini-transposon, Tn1792, developed for transposon mutagenesis of antibiotic producing Streptomyces;citation_volume=171;citation_year=1999;citation_pmid=10077847;citation_doi=10.1111/j.1574-6968.1999.tb13435.x" />
<meta name="citation_reference" content="citation_journal_title=Biotechnol. Lett.;citation_author=A. K. Johri;citation_author=M. Dua;citation_author=D. Tuteja;citation_author=R. Saxena;citation_author=D. M. Saxena;citation_title=Degradation of alpha, beta, gamma and delta-hexachlorocyclohexanes by Sphingomonas paucimobilis.;citation_pages=885-887;citation_volume=20;citation_year=1998;citation_doi=10.1023/A:1005323811769" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=K. Kato;citation_author=K. Ohtsuki;citation_author=H. Mitsuda;citation_author=T. Yomo;citation_author=S. Negoro;citation_title=Insertion sequence IS6100 on plasmid pOAD2, which degrades nylon oligomers.;citation_pages=1197-1200;citation_volume=176;citation_year=1994;citation_issue=4;citation_pmid=8106333;citation_doi=10.1128/jb.176.4.1197-1200.1994" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Syst. Evol. Microbiol.;citation_author=J. Kaur;citation_author=H. Moskalikova;citation_author=N. Niharika;citation_author=M. Sedlackova;citation_author=A. Hampl;citation_title=Sphingobium baderi sp. nov., isolated from a hexachlorocyclohexane dump site.;citation_volume=63;citation_year=2013;citation_pmid=22544800;citation_doi=10.1099/ijs.0.039834-0" />
<meta name="citation_reference" content="citation_journal_title=Genome Announc.;citation_author=P. Kohli;citation_author=A. Dua;citation_author=N. Sangwan;citation_author=P. Oldach;citation_author=J. P. Khurana;citation_title=Draft genome sequence of Sphingobium ummariense strain RL-3, a hexachlorocyclohexane-degrading bacterium.;citation_pages=e00956-e13;citation_volume=1;citation_year=2013;citation_pmid=24233594" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. Krzywinski;citation_author=J. Schein;citation_author=I. Birol;citation_author=J. Connors;citation_author=R. Gascoyne;citation_title=Circos: an information aesthetic for comparative genomics.;citation_volume=19;citation_year=2009;citation_pmid=19541911;citation_doi=10.1101/gr.092759.109" />
<meta name="citation_reference" content="citation_journal_title=Environmental Science &amp; Technology, ES &amp; T;citation_journal_abbrev=Environmental Science &amp; Technology, ES &amp; T;citation_author=M. Kumar;citation_author=P. Chaudhary;citation_author=M. Dwivedi;citation_author=R. Kumar;citation_author=D. Paul;citation_title=Enhanced biodegradation of beta- and delta-hexachlorocyclohexane in the presence of alpha- and gamma-isomers in contaminated soils.;citation_pages=4005-4011;citation_volume=39;citation_year=2005;citation_issue=11;citation_pmid=15984776;citation_doi=10.1021/es048497q" />
<meta name="citation_reference" content="citation_journal_title=Genome Announc.;citation_author=A. Kumar Singh;citation_author=N. Sangwan;citation_author=A. Sharma;citation_author=V. Gupta;citation_author=J. P. Khurana;citation_title=Draft genome sequence of Sphingobium quisquiliarum strain P25T, a novel hexachlorocyclohexane (HCH)-degrading bacterium isolated from an HCH dumpsite.;citation_pages=e00717-e13;citation_volume=1;citation_year=2013;citation_pmid=24029763" />
<meta name="citation_reference" content="citation_journal_title=J. Basic Microbiol.;citation_author=D. Lal;citation_author=S. Jindal;citation_author=H. Kumari;citation_author=S. Jit;citation_author=A. Nigam;citation_title=Bacterial diversity and real-time PCR based assessment of linA and linB gene distribution at hexachlorocyclohexane contaminated sites.;citation_pages=1-11;citation_volume=53;citation_year=2013;citation_pmid=22359233;citation_doi=10.1002/jobm.201100335" />
<meta name="citation_reference" content="citation_journal_title=Trends in biotechnology;citation_journal_abbrev=Trends Biotechnol;citation_author=R. Lal;citation_author=C. Dogra;citation_author=S. Malhotra;citation_author=P. Sharma;citation_author=R. Pal;citation_title=Diversity, distribution and divergence of lin genes in hexachlorocyclohexane-degrading sphingomonads.;citation_pages=121-130;citation_volume=24;citation_year=2006;citation_issue=3;citation_pmid=16473421;citation_doi=10.1016/j.tibtech.2006.01.005" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=R. Lal;citation_author=G. Pandey;citation_author=P. Sharma;citation_author=K. Kumari;citation_author=S. Malhotra;citation_title=Biochemistry of Microbial Degradation of Hexachlorocyclohexane and Prospects for Bioremediation;citation_pages=58-80;citation_volume=74;citation_year=2010;citation_issue=1;citation_pmid=20197499;citation_doi=10.1128/MMBR.00029-09" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. G. Lawrence;citation_author=J. R. Roth;citation_title=Selfish Operons: Horizontal Transfer May Drive the Evolution of Gene Clusters;citation_pages=1843-1860;citation_volume=143;citation_year=1996;citation_issue=4;citation_pmid=8844169" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=H. Liu;citation_author=S.-J. Wang;citation_author=J.-J. Zhang;citation_author=H. Dai;citation_author=H. Tang;citation_title=Patchwork assembly of nag-like nitroarene dioxygenase genes and the 3-chlorocatechol degradation cluster for evolution of the 2-chloronitrobenzene catabolism pathway in Pseudomonas stutzeri ZWLR2–1.;citation_volume=77;citation_year=2011;citation_pmid=21602392;citation_doi=10.1128/AEM.02543-10" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=Y. R. Luo;citation_author=S. G. Kang;citation_author=S.-J. Kim;citation_author=M.-R. Kim;citation_author=N. Li;citation_title=Genome Sequence of Benzo(a)pyrene-Degrading Bacterium Novosphingobium pentaromativorans US6-1;citation_pages=907-907;citation_volume=194;citation_year=2012;citation_issue=4;citation_pmid=22275104;citation_doi=10.1128/JB.06476-11" />
<meta name="citation_reference" content="citation_journal_title=Appl. Biochem. Biotechnol;citation_author=A. S. Macwan;citation_author=N. Srivastava;citation_author=S. Javed;citation_author=A. Kumar;citation_title=Role of a repeated hexapeptide motif GIHFAP near C-terminus in assembly, stability, and activity of “HCH dehydrochlorinase LinA”;citation_pages=1397-1404;citation_volume=169;citation_year=2013;citation_pmid=23315208;citation_doi=10.1007/s12010-012-0035-8" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=J. Mahillon;citation_author=M. Chandler;citation_title=Insertion Sequences;citation_pages=725-774;citation_volume=62;citation_year=1998;citation_issue=3;citation_pmid=9729608" />
<meta name="citation_reference" content="citation_journal_title=Journal of Applied Microbiology (Print);citation_journal_abbrev=Journal of Applied Microbiology (Print);citation_author=N. Manickam;citation_author=R. Misra;citation_author=S. Mayilraj;citation_title=A novel pathway for the biodegradation of gamma-hexachlorocyclohexane by a Xanthomonas sp. strain ICH12.;citation_pages=1468-1478;citation_volume=102;citation_year=2007;citation_issue=6;citation_pmid=17578411;citation_doi=10.1111/j.1365-2672.2006.03209.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Agricultural and Food Chemistry;citation_journal_abbrev=Journal of Agricultural and Food Chemistry;citation_author=H. K. Manonmani;citation_author=D. H. Chandrashekaraiah;citation_author=N. Sreedhar Reddy;citation_author=C. D. Elcey;citation_author=A. A. Kunhi;citation_title=Isolation and acclimation of a microbial consortium for improved aerobic degradation of alpha-hexachlorocyclohexane.;citation_pages=4341-4351;citation_volume=48;citation_year=2000;citation_issue=9;citation_pmid=10995361;citation_doi=10.1021/jf990712c" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=E. Masai;citation_author=N. Kamimura;citation_author=D. Kasai;citation_author=A. Oguchi;citation_author=A. Ankai;citation_title=Complete Genome Sequence of Sphingobium sp. Strain SYK-6, a Degrader of Lignin-Derived Biaryls and Monoaryls;citation_pages=534-535;citation_volume=194;citation_year=2012;citation_issue=2;citation_pmid=22207743;citation_doi=10.1128/JB.06254-11" />
<meta name="citation_reference" content="citation_journal_title=Biocat. Biotrans;citation_author=M. Mencía;citation_author=A. I. Martínez-Ferri;citation_author=M. Alcalde;citation_author=V. De Lorenzo;citation_title=Identification of a -hexachlorocyclohexane dehydrochlorinase (LinA) variant with improved expression and solubility properties;citation_pages=223-230;citation_volume=24;citation_year=2006;citation_doi=10.1080/10242420600667809" />
<meta name="citation_reference" content="citation_journal_title=J. Antimicrob. Chemother.;citation_author=G. B. Michael;citation_author=K. Kadlec;citation_author=M. T. Sweeney;citation_author=E. Brzuszkiewicz;citation_author=H. Liesegang;citation_title=ICEPmu1, an integrative conjugative element (ICE) of Pasteurella multocida: analysis of the regions that comprise 12 antimicrobial resistance genes.;citation_volume=67;citation_year=2012;citation_pmid=22001175;citation_doi=10.1093/jac/dkr406" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=K. Miyauchi;citation_author=S. K. Suh;citation_author=Y. Nagata;citation_author=M. Takagi;citation_title=Cloning and Sequencing of a 2,5-Dichlorohydroquinone Reductive Dehalogenase Gene Whose Product Is Involved in Degradation of gamma -Hexachlorocyclohexane by Sphingomonas paucimobilis;citation_pages=1354-1359;citation_volume=180;citation_year=1998;citation_issue=6;citation_pmid=9515900" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=K. Miyauchi;citation_author=Y. Adachi;citation_author=Y. Nagata;citation_author=M. Takagi;citation_title=Cloning and Sequencing of a Novel meta-Cleavage Dioxygenase Gene Whose Product Is Involved in Degradation of gamma -Hexachlorocyclohexane in Sphingomonas paucimobilis;citation_pages=6712-6719;citation_volume=181;citation_year=1999;citation_issue=21;citation_pmid=10542173" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=R. Miyazaki;citation_author=Y. Sato;citation_author=M. Ito;citation_author=Y. Ohtsubo;citation_author=Y. Nagata;citation_title=Complete nucleotide sequence of an exogenously isolated plasmid, pLB1, involved in gamma-hexachlorocyclohexane degradation.;citation_volume=72;citation_year=2006;citation_pmid=16963556;citation_doi=10.1128/AEM.01531-06" />
<meta name="citation_reference" content="citation_journal_title=Genome Announc.;citation_author=U. Mukherjee;citation_author=R. Kumar;citation_author=N. K. Mahato;citation_author=J. P. Khurana;citation_author=R. Lal;citation_title=Draft genome sequence of Sphingobium sp. strain HDIPO4, an avid degrader of hexachlorocyclohexane.;citation_pages=e00749-e13;citation_volume=1;citation_year=2013;citation_pmid=24051321" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=F. Nadalin;citation_author=F. Vezzi;citation_author=A. Policriti;citation_title=GapFiller: a de novo assembly approach to fill the gap within paired reads.;citation_pages=S8;citation_volume=13;citation_year=2012;citation_pmid=23282012;citation_doi=10.1186/1471-2105-13-S14-S8" />
<meta name="citation_reference" content="citation_journal_title=Biosci. Biotechnol. Biochem.;citation_author=Y. Nagata;citation_author=T. Hatta;citation_author=R. Imai;citation_author=K. Kimbara;citation_author=M. Fukuda;citation_title=Purification and characterization of gamma-hexachlorocyclohexane (gamma-HCH) dehydrochlorinase (LinA) from Pseudomonas paucimobilis.;citation_pages=1582-1583;citation_volume=57;citation_year=1993;citation_doi=10.1271/bbb.57.1582" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=Y. Nagata;citation_author=T. Nariya;citation_author=R. Ohtomo;citation_author=M. Fukuda;citation_author=K. Yano;citation_title=Cloning and sequencing of a dehalogenase gene encoding an enzyme with hydrolase activity involved in the degradation of gamma-hexachlorocyclohexane in Pseudomonas paucimobilis.;citation_pages=6403-6410;citation_volume=175;citation_year=1993;citation_issue=20;citation_pmid=7691794;citation_doi=10.1128/jb.175.20.6403-6410.1993" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=Y. Nagata;citation_author=R. Ohtomo;citation_author=K. Miyauchi;citation_author=M. Fukuda;citation_author=K. Yano;citation_title=Cloning and sequencing of a 2,5-dichloro-2,5-cyclohexadiene-1,4-diol dehydrogenase gene involved in the degradation of gamma-hexachlorocyclohexane in Pseudomonas paucimobilis.;citation_pages=3117-3125;citation_volume=176;citation_year=1994;citation_issue=11;citation_pmid=7515041;citation_doi=10.1128/jb.176.11.3117-3125.1994" />
<meta name="citation_reference" content="citation_journal_title=Applied Microbiology and Biotechnology;citation_journal_abbrev=Applied Microbiology and Biotechnology;citation_author=Y. Nagata;citation_author=R. Endo;citation_author=M. Ito;citation_author=Y. Ohtsubo;citation_author=M. Tsuda;citation_title=Aerobic degradation of lindane (gamma-hexachlorocyclohexane) in bacteria and its biochemical and molecular basis.;citation_pages=741-752;citation_volume=76;citation_year=2007;citation_issue=4;citation_pmid=17634937;citation_doi=10.1007/s00253-007-1066-x" />
<meta name="citation_reference" content="citation_journal_title=J. Bacteriol.;citation_author=Y. Nagata;citation_author=Y. Ohtsubo;citation_author=R. Endo;citation_author=N. Ichikawa;citation_author=A. Ankai;citation_title=Complete genome sequence of the representative γ-hexachlorocyclohexane-degrading bacterium Sphingobium japonicum UT26.;citation_volume=192;citation_year=2010;citation_pmid=20817768;citation_doi=10.1128/JB.00961-10" />
<meta name="citation_reference" content="citation_journal_title=Enzyme and Microbial Technology;citation_journal_abbrev=Enzyme and Microbial Technology;citation_author=Y. Nagata;citation_author=S. Natsui;citation_author=R. Endo;citation_author=Y. Ohtsubo;citation_author=N. Ichikawa;citation_title=Genomic organization and genomic structural rearrangements of Sphingobium japonicum UT26, an archetypal Î³-hexachlorocyclohexane-degrading bacterium.;citation_pages=499-508;citation_volume=49;citation_year=2011;citation_issue=6-7;citation_pmid=22142724;citation_doi=10.1016/j.enzmictec.2011.10.005" />
<meta name="citation_reference" content="citation_title=Biodegradative Bacteria: How Bacteria Degrade, Survive, Adapt, and Evolve;citation_title=Appearance and evolution of γ-hexachlorocyclohexane-degrading bacteria;citation_author=Y. Nagata;citation_author=M. Tabata;citation_author=S. Ohhata;citation_author=M. Tsuda;citation_pages=19-41;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Syst. Evol. Microbiol.;citation_author=N. Niharika;citation_author=H. Moskalikova;citation_author=J. Kaur;citation_author=F. Khan;citation_author=M. Sedlackova;citation_title=Sphingobium czechense sp. nov., isolated from a hexachlorocyclohexane dump site.;citation_volume=63;citation_year=2013;citation_pmid=22561590;citation_doi=10.1099/ijs.0.039396-0" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Syst. Evol. Microbiol.;citation_author=N. Niharika;citation_author=H. Moskalikova;citation_author=J. Kaur;citation_author=M. Sedlackova;citation_author=A. Hampl;citation_title=Novosphingobium barchaimii sp. nov., isolated from hexachlorocyclohexane-contaminated soil.;citation_volume=63;citation_year=2013;citation_pmid=22544801;citation_doi=10.1099/ijs.0.039826-0" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=M. Okai;citation_author=K. Kubota;citation_author=M. Fukuda;citation_author=Y. Nagata;citation_author=K. Nagata;citation_title=Crystal structure of Î³-hexachlorocyclohexane Dehydrochlorinase LinA from Sphingobium japonicum UT26.;citation_pages=260-269;citation_volume=403;citation_year=2010;citation_issue=2;citation_pmid=20813114;citation_doi=10.1016/j.jmb.2010.08.043" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=C. Pál;citation_author=L. D. Hurst;citation_title=The impact of very short alternative splicing on protein structures and functions in the human genome.;citation_pages=232-236;citation_volume=20;citation_year=2004;citation_issue=5;citation_pmid=15109776;citation_doi=10.1016/j.tig.2004.04.001" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=J. P. Ramsay;citation_author=J. T. Sullivan;citation_author=G. S. Stuart;citation_author=I. L. Lamont;citation_author=C. W. Ronson;citation_title=Excision and transfer of the Mesorhizobium loti R7A symbiosis island requires an integrase IntS, a novel recombination directionality factor RdfS, and a putative relaxase RlxS.;citation_pages=723-734;citation_volume=62;citation_year=2006;citation_issue=3;citation_pmid=17076666;citation_doi=10.1111/j.1365-2958.2006.05396.x" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Richter;citation_author=R. Rosselló-Móra;citation_title=Shifting the genomic gold standard for the prokaryotic species definition.;citation_volume=106;citation_year=2009;citation_pmid=19855009;citation_doi=10.1073/pnas.0906412106" />
<meta name="citation_reference" content="citation_journal_title=Antimicrob. Agents Chemother.;citation_author=A. Rodríguez-Blanco;citation_author=M. L. Lemos;citation_author=C. R. Osorio;citation_title=Integrating conjugative elements as vectors of antibiotic, mercury, and quaternary ammonium compound resistance in marine aquaculture environments.;citation_volume=56;citation_year=2012;citation_pmid=22314526;citation_doi=10.1128/AAC.05997-11" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. Sangwan;citation_author=P. Lata;citation_author=V. Dwivedi;citation_author=A. Singh;citation_author=N. Niharika;citation_title=Comparative metagenomic analysis of soil microbial communities across three hexachlorocyclohexane contamination levels.;citation_pages=e46219-e46219;citation_volume=7;citation_year=2012;citation_issue=9;citation_pmid=23029440;citation_doi=10.1371/journal.pone.0046219" />
<meta name="citation_reference" content="citation_journal_title=ISME J.;citation_author=N. Sangwan;citation_author=H. Verma;citation_author=R. Kumar;citation_author=V. Negi;citation_author=S. Lax;citation_title=Reconstructing an ancestral genotype of two hexachlorocyclohexane-degrading Sphingobium species using metagenomic sequence data.;citation_pages=398-408;citation_volume=8;citation_year=2014;citation_pmid=24030592;citation_doi=10.1038/ismej.2013.153" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. Schmieder;citation_author=R. Edwards;citation_title=Quality control and preprocessing of metagenomic datasets.;citation_volume=27;citation_year=2011;citation_pmid=21278185;citation_doi=10.1093/bioinformatics/btr026" />
<meta name="citation_reference" content="citation_journal_title=Soil Sci. Plant Nutr.;citation_author=K. Senoo;citation_author=H. Wada;citation_title=Isolation and identification of an aerobic γ-HCH-decomposing bacterium from soil.;citation_pages=79-87;citation_volume=35;citation_year=1989;citation_doi=10.1080/00380768.1989.10434739" />
<meta name="citation_reference" content="citation_journal_title=J. Bacteriol.;citation_author=H. M. B. Seth-Smith;citation_author=M. C. Fookes;citation_author=C. K. Okoro;citation_author=S. Baker;citation_author=S. R. Harris;citation_title=Structure, diversity, and mobility of the Salmonella pathogenicity island 7 family of integrative and conjugative elements within Enterobacteriaceae.;citation_volume=194;citation_year=2012;citation_pmid=22247511;citation_doi=10.1128/JB.06403-11" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=P. Sharma;citation_author=R. Pandey;citation_author=K. Kumari;citation_author=G. Pandey;citation_author=C. J. Jackson;citation_title=Kinetic and sequence-structure-function analysis of known LinA variants with different hexachlorocyclohexane isomers.;citation_pages=e25128-e25128;citation_volume=6;citation_year=2011;citation_issue=9;citation_pmid=21949868;citation_doi=10.1371/journal.pone.0025128" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=M. Shettigar;citation_author=S. Pearce;citation_author=R. Pandey;citation_author=F. Khan;citation_author=S. J. Dorrian;citation_title=Cloning of a novel 6-chloronicotinic acid chlorohydrolase from the newly isolated 6-chloronicotinic acid mineralizing Bradyrhizobiaceae strain SG-6C.;citation_pages=e51162;citation_volume=7;citation_year=2012;citation_pmid=23226482;citation_doi=10.1371/journal.pone.0051162" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. T. Simpson;citation_author=K. Wong;citation_author=S. D. Jackman;citation_author=J. E. Schein;citation_author=S. J. M. Jones;citation_title=ABySS: a parallel assembler for short read sequence data.;citation_volume=19;citation_year=2009;citation_pmid=19251739;citation_doi=10.1101/gr.089532.108" />
<meta name="citation_reference" content="citation_journal_title=Applied Microbiology and Biotechnology;citation_journal_abbrev=Applied Microbiology and Biotechnology;citation_author=A. K. Singh;citation_author=P. Chaudhary;citation_author=A. S. Macwan;citation_author=U. N. Diwedi;citation_author=A. Kumar;citation_title=Selective loss of lin genes from hexachlorocyclohexane-degrading Pseudomonas aeruginosa ITRC-5 under different growth conditions.;citation_pages=895-901;citation_volume=76;citation_year=2007;citation_issue=4;citation_pmid=17602219;citation_doi=10.1007/s00253-007-1056-z" />
<meta name="citation_reference" content="citation_journal_title=Applied and Environmental Microbiology;citation_journal_abbrev=Appl. Environ. Microbiol.;citation_author=M. Suar;citation_author=J. R. van der Meer;citation_author=K. Lawlor;citation_author=C. Holliger;citation_author=R. Lal;citation_title=Dynamics of Multiple lin Gene Expression in Sphingomonas paucimobilis B90A in Response to Different Hexachlorocyclohexane Isomers;citation_pages=6650-6656;citation_volume=70;citation_year=2004;citation_issue=11;citation_pmid=15528530;citation_doi=10.1128/AEM.70.11.6650-6656.2004" />
<meta name="citation_reference" content="citation_journal_title=Applied and Environmental Microbiology;citation_journal_abbrev=Appl. Environ. Microbiol.;citation_author=M. Suar;citation_author=A. Hauser;citation_author=T. Poiger;citation_author=H. R. Buser;citation_author=M. D. Müller;citation_title=Enantioselective Transformation of {alpha}-Hexachlorocyclohexane by the Dehydrochlorinases LinA1 and LinA2 from the Soil Bacterium Sphingomonas paucimobilis B90A;citation_pages=8514-8518;citation_volume=71;citation_year=2005;citation_issue=12;citation_pmid=16332842;citation_doi=10.1128/AEM.71.12.8514-8518.2005" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. T. Sullivan;citation_author=H. N. Patrick;citation_author=W. L. Lowther;citation_author=D. B. Scott;citation_author=C. W. Ronson;citation_title=Nodulating strains of Rhizobium loti arise through chromosomal symbiotic gene transfer in the environment.;citation_pages=8985-8989;citation_volume=92;citation_year=1995;citation_issue=19;citation_pmid=7568057;citation_doi=10.1073/pnas.92.19.8985" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=J. T. Sullivan;citation_author=J. R. Trzebiatowski;citation_author=R. W. Cruickshank;citation_author=J. Gouzy;citation_author=S. D. Brown;citation_title=Comparative Sequence Analysis of the Symbiosis Island of Mesorhizobium loti Strain R7A;citation_pages=3086-3095;citation_volume=184;citation_year=2002;citation_issue=11;citation_pmid=12003951;citation_doi=10.1128/JB.184.11.3086-3095.2002" />
<meta name="citation_reference" content="citation_journal_title=Bioscience, Biotechnology, and Biochemistry;citation_journal_abbrev=Bioscience, Biotechnology, and Biochemistry;citation_author=M. Tabata;citation_author=R. Endo;citation_author=M. Ito;citation_author=Y. Ohtsubo;citation_author=A. Kumar;citation_title=The lin genes for Î³-hexachlorocyclohexane degradation in Sphingomonas sp. MM-1 proved to be dispersed across multiple plasmids.;citation_pages=466-472;citation_volume=75;citation_year=2011;citation_issue=3;citation_pmid=21389627;citation_doi=10.1271/bbb.100652" />
<meta name="citation_reference" content="citation_journal_title=Genome Announc.;citation_author=M. Tabata;citation_author=Y. Ohtsubo;citation_author=S. Ohhata;citation_author=M. Tsuda;citation_author=Y. Nagata;citation_title=Complete genome sequence of the γ-hexachlorocyclohexane-degrading bacterium Sphingomonas sp. strain MM-1.;citation_pages=e00247-e13;citation_volume=1;citation_year=2013;citation_doi=10.1128/genomeA.00247-13" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_author=M. Takeo;citation_author=Y. Maeda;citation_author=J. Maeda;citation_author=N. Nishiyama;citation_author=C. Kitamura;citation_title=Two identical nonylphenol monooxygenase genes linked to IS6100 and some putative insertion sequence elements in Sphingomonas sp. NP5.;citation_volume=158;citation_year=2012;citation_pmid=22504436;citation_doi=10.1099/mic.0.055335-0" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=J. C. Thomas;citation_author=F. Berger;citation_author=M. Jacquier;citation_author=D. Bernillon;citation_author=F. Baud-Grasset;citation_title=Isolation and characterization of a novel gamma-hexachlorocyclohexane-degrading bacterium.;citation_pages=6049-6055;citation_volume=178;citation_year=1996;citation_issue=20;citation_pmid=8830705;citation_doi=10.1128/jb.178.20.6049-6055.1996" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=L. Trantirek;citation_author=K. Hynková;citation_author=Y. Nagata;citation_author=A. Murzin;citation_author=A. Ansorgová;citation_title=Reaction mechanism and stereochemistry of gamma-hexachlorocyclohexane dehydrochlorinase LinA.;citation_volume=276;citation_year=2001;citation_pmid=11099497;citation_doi=10.1074/jbc.M007452200" />
<meta name="citation_reference" content="citation_journal_title=Environmental Science and Pollution Research International;citation_journal_abbrev=Environmental Science and Pollution Research International;citation_author=J. Vijgen;citation_author=P. C. Abhilash;citation_author=Y. F. Li;citation_author=R. Lal;citation_author=M. Forter;citation_title=Hexachlorocyclohexane (HCH) as new Stockholm Convention POPs--a global perspective on the management of Lindane and its waste isomers.;citation_pages=152-162;citation_volume=18;citation_year=2011;citation_issue=2;citation_pmid=21104204;citation_doi=10.1007/s11356-010-0417-9" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_journal_abbrev=Microbiology;citation_author=J. Weaden;citation_author=P. Dyson;citation_title=Transposon mutagenesis with IS6100 in the avermectin-producer Streptomyces avermitilis;citation_pages=1963-1970;citation_volume=144;citation_year=1998;citation_issue=7;citation_pmid=9695929;citation_doi=10.1099/00221287-144-7-1963" />
<meta name="citation_reference" content="citation_journal_title=Environmental Science &amp; Technology, ES &amp; T;citation_journal_abbrev=Environmental Science &amp; Technology, ES &amp; T;citation_author=H. Wöhrnschimmel;citation_author=P. Tay;citation_author=H. von Waldow;citation_author=H. Hung;citation_author=Y. F. Li;citation_title=Comparative assessment of the global fate of Î±- and Î²-hexachlorocyclohexane before and after phase-out.;citation_pages=2047-2054;citation_volume=46;citation_year=2012;citation_issue=4;citation_pmid=22320168;citation_doi=10.1021/es203109q" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Microbiology;citation_journal_abbrev=Nat Rev Microbiol;citation_author=R. A. F. Wozniak;citation_author=M. K. Waldor;citation_title=Integrative and conjugative elements: mosaic mobile genetic elements enabling dynamic lateral gene flow.;citation_pages=552-563;citation_volume=8;citation_year=2010;citation_issue=8;citation_pmid=20601965;citation_doi=10.1038/nrmicro2382" />
<meta name="citation_reference" content="citation_journal_title=Environmental Microbiology (Print);citation_journal_abbrev=Environmental Microbiology (Print);citation_author=J. Wu;citation_author=Q. Hong;citation_author=Y. Sun;citation_author=Y. Hong;citation_author=Q. Yan;citation_title=Analysis of the role of LinA and LinB in biodegradation of delta-hexachlorocyclohexane.;citation_pages=2331-2340;citation_volume=9;citation_year=2007;citation_issue=9;citation_pmid=17686029;citation_doi=10.1111/j.1462-2920.2007.01350.x" />
<meta name="citation_reference" content="citation_journal_abbrev=J Biosci Bioeng;citation_author=K. Yasuhira;citation_author=Y. Uedo;citation_author=M. Takeo;citation_author=D.-I. Kato;citation_author=S. Negoro;citation_title=Genetic organization of nylon-oligomer-degrading enzymes from alkalophilic bacterium, Agromyces sp. KY5R.;citation_pages=521-524;citation_volume=104;citation_year=2007;citation_issue=6;citation_pmid=18215642;citation_doi=10.1263/jbb.104.521" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Insights into Ongoing Evolution of the Hexachlorocyclohexane Catabolic Pathway from Comparative Genomics of Ten Sphingomonadaceae Strains" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
<meta name="twitter:description" content="Hexachlorocyclohexane (HCH), a synthetic organochloride, was first used as a broad-acre insecticide in the 1940s, and many HCH-degrading bacterial strains have been isolated from around the globe during the last 20 years. To date, the same degradation pathway (the lin pathway) has been implicated in all strains characterized, although the pathway has only been characterized intensively in two strains and for only a single HCH isomer. To further elucidate the evolution of the lin pathway, we have biochemically and genetically characterized three HCH-degrading strains from the Czech Republic and compared the genomes of these and seven other HCH-degrading bacterial strains. The three new strains each yielded a distinct set of metabolites during their degradation of HCH isomers. Variable assembly of the pathway is a common feature across the 10 genomes, eight of which (including all three Czech strains) were either missing key lin genes or containing duplicate copies of upstream lin genes ( linA-F ). The analysis also confirmed the important role of horizontal transfer mediated by insertion sequence IS6100 in the acquisition of the pathway, with a stronger association of IS6100 to the lin genes in the new strains. In one strain, a linA variant was identified that likely caused a novel degradation phenotype involving a shift in isomer preference. This study identifies a number of strains that are in the early stages of lin pathway acquisition and shows that the state of the pathway can explain the degradation patterns observed." />
<meta name="og-title" property="og:title" content="Insights into Ongoing Evolution of the Hexachlorocyclohexane Catabolic Pathway from Comparative Genomics of Ten Sphingomonadaceae Strains" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/6/1081" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Hexachlorocyclohexane (HCH), a synthetic organochloride, was first used as a broad-acre insecticide in the 1940s, and many HCH-degrading bacterial strains have been isolated from around the globe during the last 20 years. To date, the same degradation pathway (the lin pathway) has been implicated in all strains characterized, although the pathway has only been characterized intensively in two strains and for only a single HCH isomer. To further elucidate the evolution of the lin pathway, we have biochemically and genetically characterized three HCH-degrading strains from the Czech Republic and compared the genomes of these and seven other HCH-degrading bacterial strains. The three new strains each yielded a distinct set of metabolites during their degradation of HCH isomers. Variable assembly of the pathway is a common feature across the 10 genomes, eight of which (including all three Czech strains) were either missing key lin genes or containing duplicate copies of upstream lin genes ( linA-F ). The analysis also confirmed the important role of horizontal transfer mediated by insertion sequence IS6100 in the acquisition of the pathway, with a stronger association of IS6100 to the lin genes in the new strains. In one strain, a linA variant was identified that likely caused a novel degradation phenotype involving a shift in isomer preference. This study identifies a number of strains that are in the early stages of lin pathway acquisition and shows that the state of the pathway can explain the degradation patterns observed." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
    <title>Insights into Ongoing Evolution of the Hexachlorocyclohexane Catabolic Pathway from Comparative Genomics of Ten Sphingomonadaceae Strains | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"c-PX7NKiW8YEt55SBh0lz5MCDW18Bv3Gk7AejbxxBWw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459917","apath":"\/ggg\/5\/6\/1081.atom","pisa":"ggg;5\/6\/1081","ac":{"\/ggg\/5\/6\/1081.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/6\/1081.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/6\/1081"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/6\/1081"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/6\/1081":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022f9TZeMthO07gUdjQT0wguwgscQk1lFBXf\\\\\\\/lyKb+l0biEHzBA5qHy5FiWwN2FgK0KPH9mIGHwZDTXAC9yMMTKQLM5MMSg93dz3cuzv64fn4r2t1NwraGBC3ZkhTC\\\\\\\/Jg9FhsT439bjRbCxcj4JJ+wqA+jtrVwBES8vA4+CcIq+xX0oyHvB5rhJfI2N4sSu5ucNABDQ4T21rfF6ZQ\\\\\\\/kqpzfnmBUNWnoQsyLCZsuwRUemBF\\\\\\\/G+8PfNcKBVHpTi+PAKpxiKOvEpOOZglJDboyfECI0IVWxbiKBlTY2iPlhZt0d5jiqT3Q1cteQ4bVKSeXleQu26276LUvuNYXslwENK+jQHYr3OuQpF\\\\\\\/Qim+ESg71UYU507ZM+HWhMK3Wbu+H7tM+XhcxvdMA5diXalVqQN35syA77GhI2XTKhqLKe2h8itazk0xUHfC+5N4unq\\\\\\\/Ep+EQzbsAAlnsjGZ3BdDidnMKh75jzNth9sKEAayf+lyR9v93L96kMwOB5D6S87Z4eauV81f\\\\\\\/Akp+QAGLb841M7OUmP0pakPuxhcb6kPzVoHdDSUthizMbhsvZ84qra6NENWeFXKuMi\\\\\\\/L9dcSCSafODIxpKwq90r7Sz2talxG7mUHw4K0bnQdesDtcXhy0a\\\\\\\/Nmh+ivuecd5H0yV\\\\\\\/XJ780O5wCWEd8X5iTumVIv2N8Ud0OC7cNqPP5YbNaovllTRWBozzHYTMAfZQdcTEM\\\\\\\/ecu59uqgsW2gPGnQGYRB8BFEAeTQrSmpcAAMc0B7vST6XcJ0N480QSHgH0\\\\\\\/z\\\\\\\/JIxmL4UTZPRD0JT1Tniqe3eRRvs\\\\\\\/ODlOZX07e2ne6PfhuZV8jIyIuv0WNlZqPsh5TuJckL8CQQTC5S9BdLUgTzsKvnnPXLHT\\\\\\\/KbDrkMa6zBHt+uN1swjwLNO3EwEh7ERkkJ8qrfxERiymF1EfO8FVh9Uaei8mhkP5plv4UmYydc0wBwBShQ4lCT8FOKeupL7JD7iv50s+WOW0uYpWiF95gsKpTiEWz5EOi2lIcwuvAOsQDM1fCIedZqkCR2VljjGpF7gPtE9o8i9sZ1wI2EmUSvcIMZJXLYAnGXbptZqDSWyFF51\\\\\\\/VNAz5SZwt9eUcc0mNsqkjmwfw6t1EJOsvNJ55MOM+gjJV0ZnP7D9+bgVww2GbNd5fzuLS3+tm1v\\\\\\\/Sgbube5nB8fgHQjNobZfy9g8Y7o6ZwRbdpvh\\\\\\\/h0jpE6tvY4vAsiDkM2GAsxQPxDaL9JCzyGD1+n8c\\\\\\\/RFbmpcwj\\\\\\\/iJAM+IWayotik4wZ+Qd59A\\\\\\\/hA+qLM1XhHt\\\u0022,\\\u0022iv\\\u0022:\\\u0022hZU3vl39X+77Dh45MMJdcQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002283b039c69021d006f4e95b88e921cffd0784ca12e97e9480556689decfc6813a\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022dmh9yN8QJzQbHvQAnfnQcg1cYFdTTeQkH\\\\\\\/x\\\\\\\/vwBu0Wm+uVA5D3Z3xqAUtjZbwmYqB2+CzbcgJTlnfcBVLuHfpLwCv2pn2kfRgebCObRCYOxxYP\\\\\\\/Jey+VUg4GsV3zlFjG3o+otwm9t69mHGBYzR4zrIMKZCRuLAmodxYQ\\\\\\\/p\\\\\\\/F0l+vdQepLNGfVN7StpfOjK+Ez+9CvPECBPjpQ2lkjKRgTH7zkwyqnzZb4n6jzGmRVANCTeKxdQ0Y+cXuXsFEzyu5Zlc9e9XoxpOn3LFL4F2ExHmI+tgZsfgualKpDxXP9vNTKiv9c2Pga4RMSE698J3A1gftFAZUMdU8tLAK\\\\\\\/s3pd9lz6Q3pYIaQvi72f0mzfl4lruVW9Hx9FwVPlvjMnadkSMmjXlVJU8E85SN2RZwTpXuBCTbn9rH0qCOMIM+jj5t67sAfbu+h33F4ZzaKoEcP0atlMb+Efs3t9kiEAK6coP7INFtABh5NK6j3OWTGfSVSBZBlv+94jW4nfo6UyKEKcVZu\\\\\\\/uieka9wG3Qy0+cqPpEFKhgV90mmWfCXAMLnP16NdValT2hLAt71G6fguBzgjrICujOZfuiWxh041Lc0t2K3O\\\\\\\/+cN1MMFQUpBPcthMkFt29RSUR0f5TbHZdNaqVxbo+zoZuKUu2ytxPvk2xqrV\\\\\\\/W+O\\\\\\\/LNAEJR4cFxeYXRjxWa\\\\\\\/QFSdfll48+H71XRQRaD9LnrEMXd1k4Klxmpq2XNuf8nroAliIachV9jMmrNAAPttY8Ln16mdn5RIooMwPa3WiK\\\\\\\/f2HCVGDrk1aAze8KjMRpgA2yOo\\\\\\\/UYzm0KHB\\\\\\\/pe4dnwx\\\\\\\/g7NqdYNnyBVmbAL3XIAysp\\\\\\\/av5utm0ImxHfAnourC2+4pmOH6QLSRiNR1biECFvhf5LV25BblR3k0Az6BAB+wX24fVr2YKDhGFWmtNpMb+muDhT7HhViPZ+6vMj4W8ld05kpaBQdjYw5m3C5oR4+P7Qx7aAJrB4C0UUQrw+3yATM8YaTPtvdDx3c62LBHpNM0BGMBMpVVAUIOXgiKGRM5hqP1RpJGY5hHz3w2diiI+QSVA3j4U9ZQzZrysdBq4+RzM8neKT7tDuU0URwUjl04P\\\\\\\/eU2Q\\\\\\\/l1k2uxswIs+aq60CZb8MjyzJM93xq3DqWhKKVnIGS260bn\\\\\\\/Zbv\\\\\\\/2Ks\\\\\\\/z2aSwzpayCj5UyfGaxbw2aWWHNhmZA4draejRYDGSo5p9KdRduXUPTSu3jMsHOhrx7alnUTA0+JQPSyKoJciVBn5E+s05dfSyUKOEAedQCHw+3dQ4sIONyDVyLZIMQsEaf3KP\\\\\\\/IphZAFrdQrKXIR+fKBncRvhckGyiToXOnBT+7QBJ6Wr9Q9AnmW1DSiWSJ+ysUPTnFP1cOuInJaIbScVeVWlFMQeQkbBOpc355fWwdPW6xfpysQ0\\\\\\\/+5DejpJ4SeCK0I4jxHCUmKtqgt1cmEiGGfpOJBMJ36hOfMzmBLSzCg+iMTJ0apJ2Y39yFelYMWmLGjQrgESxxf45yvbQPRn5R+f8TFC+5vrGrRC4oiY7y\\\\\\\/k+edPb0\\\\\\\/fyuQPryWfATzPtUgUw2WZEiMDTUuokgxg4o6vJTyjOCkKwH3JHc4dufw4ehQQD4j8mc+fVopO257\\\\\\\/sPo4aZIZcK1ogjF1DmfvkagSEolmzJuJ0h8I\\\\\\\/rLcPkhLp9VxfOAfk5w0HSIuEQputABtKthVOTUjIzdNMm3p3VLWxG9\\\\\\\/HqEuQKzA4t8PddmFUO\\\\\\\/BIywjZAPIFRb9j3OBJLH1Esunx5l5K6t8fZvzRoo6RAdVxuESwuZLEr2c7bghHjgCZQ2LCl0COpsGtrei82WrooEEBT7GkHqtx22gH3yCFeJYA89LTnx4ygVXKN9ST+QkFcDhhtMAJWAbiE1YTuVjdcWlbtx2FVF6yki0xcU6PMsWNRVKfX5yMQBy6DJ9GE\\\\\\\/wqkM3fnob+7fqcuoHB2bixC41uK+MReAUs4pvcClNiTzLJWT3I3itG\\\\\\\/YimkaJK6yn2f88HeiVWUm5j1Y3wG8j4OAsxeYlWtDCvClL9gOvrzrUj3mJl1xfgVga6a66JlVXXS5bASHUpgOCYHkTkYEw6Gq1TscBon7r0OkPz4g2K19OZd3vg2VI7qRLk0e\\\\\\\/i8NslktxYMAxBGHgPEtC826wHZ2CYVMbOQWhzqQ45mFmH4rkEITpRDb1u\\\\\\\/I4S23XJF+yjv\\\\\\\/J\\\\\\\/gr02Zt8rikCPdTcRQEPpWKG897NCR\\\\\\\/LMKYSbAQroA3Op0N+pXQsbb\\\\\\\/xvD8L91jCY5hy2EwGIJC0ojmf6RRlmZJdAuKOkCfMK74G75WfmbtqxxpNuSPMoN3neEEtyX6\\\\\\\/IB7UJxj+RV\\\\\\\/AZ2OvU\\\\\\\/7mtSc3PFZSJb6eXMLAHYdmmstxMl9DnvrBxwSE4KeipTrI86XC3rMy4RjwnyZX2tAmKb2qTxfPosd8yjWB7yWzAS4KSYXRNmxCtxksBS7ZW+TWfJ47E5X5WwaPA5CC\\\\\\\/wn8emsC1cLOaRwFy4qcHd2K3pyq3WXcGgInrmDUzsXIGsxaG3R0HvRsrVru92o7s\\\\\\\/+IxvIevUxn+QGUg5y86uxacDVuJ7MIiq6QtaBIXQYN5h\\\\\\\/+7EN8Tqs3poEWhMQqET5c7rU5Cq08agGqPJCqwIh5cCqMT1SPCIUz2Zmg6gKN8jH01kfw3D\\\\\\\/Gz8DolyEuI7QC+tG3X0OLjsIs78OcsJO\\\\\\\/UtoFnptg0lboaA3aw5wwyzA+h844F+Hl38yQ\\\\\\\/F1XO5MhM2Rp6+B\\\\\\\/iGJfLussTSmDTAj6M0wYLE8WGWIFeCdSgI4v33OSah\\\\\\\/7eP\\\\\\\/trGPgOaSxeseZjP3Xirbwi9ISuV2Ew6H6RBQu76lQ4WcFmacocFx80CetO\\\\\\\/ctqdsfeElxvMHqn08DZrl8UsEhO01qyOYsELSNDBKmzQR5n2uFQg5GX2bIabEIvzEUbUKEE5iSCR2YoWOI2aZYdBN24ppI7PAnebVRd7cy7tCCyHQFVxZB9w6rnJeLSxX3BG5fbByPMN9YjsH8Qsf+Ej28u7F4WjGLu5arVq664PRDyfuRfqmVLafeAy5gnztd0XapfO8FG4wbEQM4\\\\\\\/Sm6a5dzrkKTjm0V8CVxyEaajvszCynJvwof6px3rgWkpYFq8w3mi3LlFLRQ1iuBhUnoat4Mmyesz1TsxLmZMUR4UvfSyzJQo5OV91vU4IsxiA3jKxGQZJLxlnmQ1CqtR2FNTQK1AX5aIu7PIJ1UT2sTInMep4c6\\\\\\\/z3WRpZ6d9L0e43fyZ7d0\\\\\\\/CynFgtpYz1h3qaNlxtPC24dT4L0z\\\\\\\/tmmA616FnsvzIDiZb+0CySkb+lAK05qxzAQa+dVo4D6eSIlTjq+vEy0BGMyNY0hwOwTWTYnSFSWf3k6CRBvS8HP017OCsPEZM5mqOOrhEN2cb6RiWCCSSWEIE24XVYGZbkAJhGbi4bcTKT\\\\\\\/x52mhBggCaFFobXg470dkqowuxK14JZ8ietGC1a3\\\\\\\/mLohIvRqMNgI1ZX1LHF++HAgXN3lbVcPsxmin5JhBsX4tvsZr1np0v5N7+zQJQ2oCeCgDf22zkTr8vyMJNl3vvxYk0XCec5w5buK\\\\\\\/qyvAOs+\\\\\\\/KJyn2XPTYG6lVjjEnZ\\\\\\\/ckXnljX+qhsnfx8hGaHYgGwIp9+cXNxbim9yAiofOSFodhJjPkk6hCZHuf0f3FVbuC0RRrM4gV2nFSeE+AgFGcBA4DUheyAPkWHKTazxQ7VMmLDUWiXcAW9DEgpyFZ0z3hMvJoMCjiVHwgJqZVYWejMeD+zD2IyWa\\\\\\\/Yy\\\\\\\/PtUNip49Esuo8NG27CstIrPwbPBNMwRlIWVVrKCzBzMG5h5prjiU47onuIDzjdV\\\\\\\/HBzjjlQUIW55cuiKUk9rh46kRJclZpkYj6RARWsn2GpjTDxA2mmlNdxDnD9b1V8n01CNd1hfoGqEeJ1s6df5f7uV1Mw59gM8vLUec8+gfadKLU\\\\\\\/tcgolPIBckEAVeo\\\\\\\/LyLzt5A47Wsvc+b4z43NUDjd+LCP9nrM1Cm7LLn\\\\\\\/HbxJKJEE0P1mISZ7KnGMkMFbtR9viiTtqi9XzTrBhjJnjCCH+UMuf9qS2mgEWN0ZDgltdqqklxgVwGsUK0NQPCgcpPCtfzftME9qfTE1refdMSchbmfPlFrN1hG2aBc9HjzBXJPYSW8stRnE89tAT5ZJgf\\\\\\\/CNqS0q2YurTtXTSsOHPXrcEvENRmZl8pADfd0uarv3VFz1+qySORlg\\\\\\\/OrkUdq26GvM5GUfYDrwGH91M9T4v9bk4ZBzTyOpph7HN\\\\\\\/3rBxRRR8qt35MtiwQEGNTvqc6ptZCE4L0LFUz5KmwiWTH7m9yqjyuw3j5ox8rQNwQSepywmB0QNdt+RrOe4Hg0lusV2fJDfv5M4pfVd4gGUi2KsEz7ZFd+oN7hxCn2KBYWiZvv7QP1QlZ4rHU1M\\\\\\\/5VzQTiql1BrzJJgJEVOOQhcKyXFqhousfwcPtX9sHtstV+loQfR8LwS5vs7cQBcsRBL+V93nwN95PY9qqA8xv1aIOwsloskKN3kUo\\\\\\\/Q\\\\\\\/F2AuTzGiGb4QlqVLG+VOB6mRQqLSbhslw5sJr7Pfi3hNcGX7tC4KWRFxyH0Ht8XQLcmVOGp8VRYSmzIwf6youRGxsnXB17296Yz148N6f81jWH3jYXuu+PFrx52nmNpu321RWAXIn+cyIvynh506iL2+shmuRFm8GzQKkXRfvnhprkvAE7fWnDrpv9wN\\\\\\\/Fi+KQSeyFZP9ZPFN4xXQf\\\\\\\/q8CuG6PdCYYD4qYxba+3DYiVJX8HqORFKsArjd7CZ2bzw1b4TxSwvd6316V2tI6Sk7WYzNpBmG6ZgAOAcgug4GNWAKUiq3KPuT\\\\\\\/xbuQ3amrc4QwY4nCg68sintNOVnFATcIv1ET\\\\\\\/nMXEX+Dz3qc7J+JTlsU8Kplnj8tJBg4A+5Jwv32\\\\\\\/+bzVp0wOCtBAiQqB0GnotyV58dgBf6kBMu9ZAc14m82mZ3+Bby65YqwNgZBM+1GHSAMJCay9\\\\\\\/IvzdwzsjMFXQZsiwU+41SxN59M4mVHLJvcHsIlnphxL8n+ND+z0RvewcoX6LedKUHwSAE4aUhM8z34rCSQzY4cZUBkilA8ToIlmDUe+nuT14oLHgL37a4UCw+lp1snyQrCI5dmv9KGX3jbUSzS7xQqRB+TtV62yTNDjszSXM\\\\\\\/y3CgLDJkhj+SCQgFZ79MQiRJ\\\\\\\/T\\\\\\\/I+bFs3H\\\\\\\/nF77JwgmXXxxYUNQGjO4p2keO7wayGGZTwZEPF9nu6YPayEzndVj6J3I\\\\\\\/1JLgqBrS3TTvCHJy67WAb4cmHUCM2scgSrxGStDY+SMeob8e+lYDWhQ4oRSQIOtZB6nZ\\\\\\\/z5kuKZ+hyyHnxR6oA42JDSGdoSm6mYUFA1KbqvQV+p8dcGy0Uo7mpoZIchNjzOqlFasOTDFkhT6iW3ZJr6ayuUxms\\\\\\\/vlQfFL8\\\\\\\/IY1ubECere2gKiREfcIEj+\\\\\\\/\\\\\\\/yN+jE9W5wmJg\\\\\\\/VIUuflrhfLtAK172Rf7Yc0PbDgvxaL9FAkWE6bG8wEZjAuZUFKt+EHpyQyzLxOwMHIDGN1XD2QSi2pcQX9isu7h5zt+XC4NukTyU6IlWwmm6ZjAoUXgJ+IM+vWP0Pb9CMtNwPfC4acamb7sXQmO+ZBbJdb38V7ZIp1Oco6u\\\\\\\/ccvUW9q5atOoGIm5ChZmTj8u9P8eVaA+fusmEfPeQKNrATCDhHcHVre9WVon\\\\\\\/f0m4D4atht4dog7zVYTtp3oM8U8VV2pmT4vdbDWnB1aUz6+M\\\\\\\/Vx22e\\\\\\\/nvhHWKcWJgSyzI704hSPQyZoJl4fTq4ZMWhSidQsKT1C3ZIp74j+aFJEAVq\\\\\\\/Y+azvrHx+wWl0WgZIyIgcFxZYv1b1eaxwYHBgnCYWDRAzFWs8NaklhsSKJhpV2je\\\\\\\/sLmq8mUZOxuQXqFW3x7nLCz86i0ex0K4lqVM8rPbqm5PMzTz3dxjUMgcQIOU4gnlBdFSPJKYwkn9grGoBQtTQvxz9mqd2EP0cYy4o4p2YtaBXE+z9ULfmqMSIe62nJanqK63SK6oBJxm4jYNT5wS3XCC8P+6I2YMEfixd7TYxwUT8dlzHtoYKOnzTNn43Wj3RVZDf0WP2Z1Dm1pnHQRRMJPH1oUTKcCPuMMqoSbsM\\\\\\\/SnkNwbcvnUtwTlyYs5gyyFkaDGgRK7a\\\\\\\/PXMiyhshpic78mS\\\\\\\/+AKTraBTyv3i76nbGUNSuwiDA6f2\\\\\\\/Z91H4WTkbqXxAkAvCi6SYMPxsC859FEj4IWA+aPd92\\\\\\\/h9l9IJA258v33rrXE\\\\\\\/pyi\\\\\\\/zdvSoJYpiZ8gNoqRIrQAqnUesPaeyVf1U4fcMVQYCxr8WxeHAPoGGQgkaxYrmC1tDKXn7NqO+wWNZRy4AQ5cDOaOGfdyVORe55O2gGXMBORucnL99oOeSAGHbw5x6HFnpMtJqvVRCV9auaUKGtTwbPj7UYjkz3jkPDHKGFVgtyhXxgjuTKIf5wmRNP5NJP\\\\\\\/qRy9Tke32ddBFIVi\\\\\\\/MZPsu5KxGuHFXns\\\\\\\/cfo3jAEqzgNq9rYI9XmS8e9pMZNo1I4alXfX3iNa0rB4Zt59Dm32hmIa++12jROOsCp3yzapP4NVR2ZC2P6OsvnRaMaOknonhST\\\\\\\/UHIJ3aA3f7ZuCwx3NtzrgQgp7xTb24xbk7NW96eSv5qVlGNB64uw+9gc7Hv3D4c2a1G8BySJH+mdLSigaHS6jzcAYm2CZHefFh42PaJfN9kAmm8mxivKFtayybqZ3UmDeBJXpUnMpkEHoahLsBtAIlv+D+AloAW5Hie+IdngCOXbIFDJKyUNBTTyyUNlXqrntqZRSWxkv\\\\\\\/eMGLE3qhaAwRGyjYAo1Fif8vPbaLEu8rfPs5kcqh7uRh3OkVq+d40YKV9RwfNpnyfwlfv1GINnTtRNPL5RZTI\\\\\\\/SGGcFYdphhpJdbe2aDMtQGkZ7PaVCBjapBbPGHBqRDmatun8igUxWJ8pHuYm\\\\\\\/DDZfkmT7yb2HhIUXDqRp81QCcaG4pi0u\\\\\\\/+5t9usFDOaGjVOOh1ZyyyVecSh1bzyghLlG033bep06p+Y4Lme3Rq8lMUBcKjy9deYOXVoDrmp0Z\\\\\\\/UoyGVKyVvp\\\\\\\/vsRPIbtxgfiUHeTNpBu7RTn9\\\\\\\/D6XcA+lR4GDRAsbhXhjFyO5rGTRMXGlMasX2Ib0jJ4XjnHNR6p6McVzoYeWqTt9BkVRX1KJ9bnI1QT36rmqUL1WwWjNu0ng0M+vad26yGMXKEjFBn3s7B4Vgqd5ORtE0tltuR4pHEDLBPotnY0argZj0I0VIazdeCuXT5FS\\\\\\\/ziIJm1xNMhfomywtOxdcDVp4dPReE5q\\\\\\\/5QOQKBJjuCUkw85FGaktKU0R2JwMEK2cGm4IaYJPE0gE3KDTVpZXS\\\\\\\/Ecl6vd9E2jPRKEryhOTFP7j7hufEDF\\\\\\\/kK8uVTr+yOmCSaaYy5K6\\\\\\\/qUFLPBvgmUIg2PbEfYg0HtHX07mGe2KhARDWsvUoW2sgp4mVSEnsFV8y54fFQ1x6jwOxgyNaqAF9G14Ft9lXhZbjh8WqEOvfOfNpbyZbUURwVW1TyZqqfkZyxP8ocJRxhAAh34OxmRqIUNoE8EawM0th6ItmLzk6hPHbWRzunSCgjTFKI3LA8dHyZQLd6F4Q3RyTXMC6LkN+JWXTeZ2OOu0Nv9G6YoUs4naaAM\\\\\\\/KO48HpiwERs4egt1kA8\\\\\\\/2a63mkpHYFldqkM8IH506sWjWhnfdZy+JL31GFIpvRTX4kezdHK8s7nTJXbGUgiHquP+3mU+WiaqeIoDAQn+xX8RINLojLWM9+C2xcXhkSArS3s\\\\\\\/T2MHv+9QB4pDwjtCq13\\\\\\\/lebywspNQAT2pRVEDna73awEwH5TmxHKprEt4UMXwrwRaLSWm9nd8ZKl\\\\\\\/U\\\\\\\/9GCMNz2ZFJJC9uZpVJLJCfi0d6BDuSAbgEFoYzjJlhNTT9yclk\\\\\\\/0MZW8M\\\\\\\/jrfpqWJfk5mvzHZAl+8VJCiUcUoBbGQOrXAqejBpwWOgUu7lJoOJk73FDyli6jpqYqabmtOoQWSi7dihI22hSG2Jom062p3viaOJCklqYW4FyTsaFMq7OM62MNWS56jpEqNrkWC0y1IqxXx8Sy\\\\\\\/SO9DkMXvI1lHH8apgb1MfEfbVgMEgJhBDQcwRPBSfp7ZZMzhzKaBPr9Nl7m9aeC5wCEXOAIAcW\\\\\\\/JAMyyazj6jO44yoaLPxabtD8ojUEb0oB+By2i6Z8ir+pDWpyJTJbjC+Af8H0XK2PXL8zKzZJ8+k3haGBN8WMPaZSrPvHde16+58vp0sVNoSDPNEBvf01GRbtBnteyM8533dujhy79AgRQyQ61J1oCCDktoZmT00gBnQKIqJUZv1HB9XcUlJa8R9+WQ8Yh7P4HPAdKhGHDqj6WrK5QU3xGaXx2SrPHvpBCqACraSCuqasT1nZ2wJIZ4YVGjSkFO+e5PBGGmUUiOym2x9YZdQNTLjxGKLTsKVuFphzLcT+FehfgTZh2AXO6eYUZwv+2wRJ6Gh5ta6L8X8+7EDudXJ+EDfpS4jvZv4SRxKEPKNQMVAN4QC3YSeWBYIV9CnNMckzcN2noNTCZ+oGbK1DipH72XbBdRA8GPEmajVgXyy1zL07f2tRbePbolMprrqUCwlQ372NcBneLf9B3HSbsNcJTKzfQQCHao+Tz39vxqAagS6sFO2kiaUEkiAjLsLKk7nCwygGZX3IWv+nADD7xTiAeBElTDvGJjQP\\\\\\\/+JlY0VSn+rZN+a78qhLISRsJbnC5t2LxXh1ituteBLLnWQwYS18NpehhfQDk86BxAXIaVP7iz1GyG2YbGSOZzfSai8uAESKv4gE1KijCFg17T96tO1A0krEjD+BLUmEH5u5hgYTWq9TrjNH\\\\\\\/6sT2oTKZGkz3rk2wl+wVrXSxVPx8UzGQ0b8JZIJGMgqPklgjkzYKn9OuQASy8c6JfCvCjyZJnOJOioes3jtFwCgC28EpO+NCghoTg9aPVhKhD4yNle0cOijKy0DqEyL1Db16rkfIXNA5Cpp5\\\\\\\/lusaDyk7soXLG\\\\\\\/gXHgQ3Wcj8b0eaQ6jamxhaGDHS6pZNY75R6r3ncwaWSNJOvvG0UXOF95GC1DsILA66EnwWTnBNVideJBIWfRI9nPSIJSStkJFRuZ1ZXMmnQx4Bt9EjubT+TR98Clivs43XdsmrlSIJAcODjMUZm04n6o8gTTSMiLVKxAYLl2HvsjtsynPRJ7vEsVhJP5GplkEsQ7zg4RvQT8m6Fk9Bh+3Z9aEUelW4uiscPJYaYOkwkpg39zHsJkviHQH9CiZUvV+GLGPaW4ezfJ1DvAIshzweHTlPhap133t3bm0kYsyPwBRem6l89+\\\\\\\/nR6I4cgmyYLUSnqOwMt5N3zgHtj6jPeycwhmWe9ZcSJatRmi1cDtF1WsvI41gizkTDKkQIT922CnG\\\\\\\/s37JhzyWTJtwfTUIWhzJmUsbmoRcALcGz9qvuHbt0trc8nBv+cX0CL0UvS1SbD9hid9UsoE32wf3JrjauJ+g6YSAh6BmR6TtJFBp7vP6EMzFnf+zr4W82m2CTznqb\\\\\\\/LZ0SZLVwHVn8hNvr2uehQ81\\\\\\\/hosnbMzJzWtI1eV\\\\\\\/woPSihpFJ4AIPx0yqHUDFk90d9zJmZXsaA8zVw6ME3YfmySSGIMBR51cXpOohsjGOxlROgHpisrXeh6lyxDscxYLRlaNBN1MYG4sEqJvssA3S8hoamovZryxZvHQh18vm6dwb3Uk2N+VKSmMh9q7Et8cmDR7pTRV4RvuqDI6Ipa93KsgZSa1eBPTYeRWk2JFYJ+O3NDMoJpvfenbbyvEpJE\\\\\\\/B3\\\\\\\/Tn97frsflALVAxG7hW309EFVDFIAFndPfbAgDC8w1n8ijBS6D+TzUg9Szmh98DaER0orPfiGMcuC6gcyL05+GduduNqy8gfmxKvaKxHSPBynRocNt9+e8FCvbkyx5whGaGScGhrXBxxEJ0rq9Pixle6py\\\\\\\/Q5ZpEqaOCXzATSpUITDXdSjHF8qrCBRrX+XCKVKBEkZYPZm\\\\\\\/+rOJu3B6grU1B\\\\\\\/eUculcYVFQh70THES1gHDjva8wWAJyAaduG584xt1wLWIWbcr0Ydt6YIQwoSqkCWYbiberwd\\\\\\\/FVhhhM0NXYGS1WrB3f4LY7Gno0haELJV7mLW7FauSyQkWMcC829sQr72uPdK1UbjHTWDDY0CcHdBZAX87rOFKMUTBZQvU74fCgrB0vyXkwz3520kO3CP1JN3QgeXiwEHPh+qon\\\\\\\/aiggoTv0PCHNCz+44\\\\\\\/cerBpLLfV4JKkUrjIMA3w8HJ35mm7fGafCO3+wLSl5QNXa7S3N1noxq2SDBd2SEYlbt4LOLsqPM16vKQNL7DeKBY6LWWK9sBWLgNavyEdTTZKeZU8\\\\\\\/CDbfSUaTnFjWoewFZj+AJFPaacOpD5YSSvYwARDjFRDLGcfucNMAFI7hdrTuR9R8yqMTS93scWPnbRRpTdFfTAn81Bl1NVw2LK6WtQSEUBdpkY4sjzWr9Lm3QHQL+kNqBKWBnjvXz\\\\\\\/bMXqmztdz1LMz1+B\\\\\\\/9JdkpI68L292RPJs12KvonxUb3\\\\\\\/7TdvUe3j7SscXMw+facOTueeCto9R\\\\\\\/4Pbab7MNHFBR0IL0HSZMASaJ82lQZR3pZjEIioEBnHot0OV6owUSMzc3f2+Sma5Jsy\\\\\\\/q62C6HaLScm1lukbqpIr6YE56aKi+3lmVjGV7hHruw66s2Ld2ZAA1KVtdXTt23sAVf6\\\\\\\/8cuRL5WmTaW2afhNEaE+rvN1qUk5GEb8WFxMzDREWKQUbnTFDv1GRv5BJsbC\\\\\\\/AZEY3SSnvXfm7fii\\\\\\\/V9gB687akaxyrc0fE99Ohf4464kgilaCVtw764tWiuBLXlK21m0gSp71T3ORBZVuy7eJWdev\\\\\\\/xsl+qBsCyz8StZyUG2qa0riqW\\\\\\\/MF2TQUohullVbv0Q3Smeer4dD0wFCTCxEJqR4gcTmtjgjo8o7UzL3BNuf1Vp0aKSoo\\\\\\\/sH94F31lkCrm8mSgE0g\\\\\\\/XcBTCGJDbLDvf70PUqjvYpxX2JbVpHxID2GDjNHFQfGHlQ4UEgLt\\\\\\\/yzx9+dfNjbd+\\\\\\\/zmtbeKPY2br38HfL5TauGhioAgsoC0ZPodVZ7dfM3ShJil1aKv7uNt+jLxqp5uPzPMETkrvVNr8ohqc4a8S6B6ip24UPChmkYKGnGz5LHZlHfuddXWU1WmRmotbWLLIEFV1tdUZC3Av1a3QOdO58M9eUpR+V5yR4VptoDpB7yCYy1KxhWt2DnGKLVNp1WRvDSG1\\\\\\\/qFTQkYvm181AEaaS1\\\\\\\/4Bbsno4nlV3Dt+q1G3JCK4nvsK7LMa7QPVK8WG2fEI2hNO66aa065O6mHxZeQuM7uLynIw38nqsJ58N7+DZn9UBVtDcF2Rbg25YkfurTAhtFh7kFQvKUFVrFxnYxtFO4xUNGCw5ql7TKCu\\\\\\\/bLuM5EnwTni0E2juiSdNxTqkrbgbOI12lSylyESZeZboK+QzrttEPTu4471WZvDpYnMi0eqPpwGHOvpX+qrAx7zc2vaDDw9nu3zoumptAgFqYgRLl41WzoWVEzJqgxa+g0ElqSgFYqow2pnPs4J9i0YN5GGdss40s3MBqWengaRdZ9A4X230Oz7A2awoGJMhOxbcOchkdPERpjnKRwYZXsgcS\\\\\\\/SsQF+3Z10Ya58yGJYADVB4NbBkCTSC6HEynYdqcKGrc7zIb\\\\\\\/xT1bYUzoRlk7KIgB9qkLogWi2sgNbqKRh6G6fu5IL++1zBqC28ubqyCJpahlkUXFda4c6hb51sQRBD\\\\\\\/dqcK5gA46VnShyFI7FMaeAbHugFIquAKLD3VyzzROXcGYg77xPkvSpT6CiSjLa30aHLLWHF3CwKgRaLy5334FKeDfqtVcu18LyoGxYjRR0PtX8tvkLb4D7th0uwIe4mI93fFeU9kaMsfX0MVsOTZSdlalZ3xZjDMaAKZvNuX1QbnSld6vr1\\\\\\\/M05iOx4iLGG3E15j8bVP2zfTfQw2xSJLnn2VYxngNHH\\\\\\\/+VtAjeGGjhcAeZhqDaCoO\\\\\\\/LSzDiJl1m0xtZqMy55DtGH8KT4Q9jI2GG2AegSHI6kXr8a8SgGRjVwiEnysL1CUmjuYVqVvrVLD0Q1xjOW75jSjmA0L7xUl3VqkSMnCA4GYmUBnw2pKIwZYEWqo\\\\\\\/BJpPdBX+Yzzb1ssq+GxAFc1W3Ys5ANRa6cXb2+3VXw04cV1\\\\\\\/8+7ofeQELe687\\\\\\\/mD5TLorHrT9FTIF1vAdaedj13\\\\\\\/xoJF68NW8IMDoOo5\\\\\\\/uI9o\\\\\\\/0zJ\\\\\\\/6K7\\\\\\\/JTeuRPjS3CqIOZmAlPTBXChb5WnN2qxSjh7xj6SJlM6Tc1cx+QMU3DTtLfcPLkQN3FOaNwmNyx6PlYVGeNNao+Z2lg93pCH\\\\\\\/QnsLx3j4f8AWJGk0YoOqyLLR4kbWg4XXz1p43pSDgfemvan25kwz98rww5j\\\\\\\/8TsvZvdO7ernd033aoVoIkiLYcZgvAI64cj0m\\\\\\\/5PGPma6CgiDzLsuUH1z\\\\\\\/pbwTGt3m3ADKMkFkYb7hD8\\\\\\\/hNCrSXOpHAdfj+HB0fEFGqIeaQ+OHJibjy6U8WsgSOOpTgGe+ZYFsOBJAmwCUViqbZJxt2wU2jsEYoRnFYctCDN2JBlBBNEFvcetgMXAHKn6woi2kawTn9CkMkZjQLK9eLjDljxO2lwnq7MWFTJXfqxpzBL2\\\\\\\/\\\\\\\/jTLqSYT6JeGUlW6oNuOEvrUTyYtQ\\\\\\\/pJ0LgmlmgEM9e7GQ1aymw1G\\\\\\\/9\\\\\\\/\\\\\\\/NFrEGsB8mqpfKnUWGJTDeImKTuITzgUorpKoSNghko6SCB1M8XaO9m1O7U\\\\\\\/u8INbCo\\\\\\\/BuiELv2s24TKKIdMArGXaV7BSanFvpZ75\\\\\\\/LLO5bvq8teGSS65GFMt4kOahTSF9CP2Tqg58be4pm2hdLbM\\\\\\\/iXtadHsq+nAXFx+z6wSjcQ5+ZarnURGqabTRAavEHlb1swNcUXuul1pZMtkml2oZvoovxG9lWHwMH1+ZuF3P+6V0F9Y\\\\\\\/efRBR6Vj7IBMkMMPuc2S+5T50yAb7jSADDoX2to+PAT6km71HdvNZh7nJ0KxDTO4id7ra1cRuaL\\\\\\\/XHv1T3RbVXSbnvPGj7a2QyevUvb9nbwk5za6yqZvi76I\\\\\\\/UE9IGMWVXxIxT\\\\\\\/98cWuBU0pobyFa7m8P61Tb40Sg+D5mgJxPK6skG\\\\\\\/L\\\\\\\/N0f9XALxo\\\\\\\/LhxoV0Gfpnmutl\\\\\\\/VB+Kf3sSpNZV0rya4r6TRGgrUriPlW8emEgWWXVmnmLlyQJCVd9ZlKuO2NMHLSjSMzCPwwT3gzia5hfPYuU8MnQtGXpo83wjw0Jr81RkiDRX\\\\\\\/UZpvhCgid95tkS\\\\\\\/9I2otMLPjv+0c2NghutN\\\\\\\/2XLJh6iV7FKoDwWYP2HmjYNtBSvnUpdG5ptI8xQUfSACB5v6w2EbVuPDuDVBcT8TRkmLAP4BA09qLcCCWjR6dTuMQkEp1bknRuZ90fPNSaNFDJxGlXhTiQy5xGcSp93GWxws1kGsqaYiYwvg30xUEZp4c6zRKx31jScPOmenq5TKBvpwl9vPD08gyi9dc7EqO9oBALekSx1QobceGhLc6nax1ZkTj4OpluHRIwWWfSxzh5lr65VGgQYJpfH6YG\\\\\\\/auUsKUit+zgg3To47ge9Y648QgZQKHg\\\\\\\/rDR6QvOTylTnznXnZCgdQzpQdgi8bkYLUW9NwV32DTGT\\\\\\\/EGAgsI3nPrO+t8FDD7e+FKQ1YvteNaoJ062P9vgAfnAC+VoAeQg0bmhz72SXmeAM600OaZLObz9\\\\\\\/KNUvfGlczklOqIDfi\\\\\\\/fXUHL2pZfvayoUWi9cSkWhvP1UhzgHfTpivJ5mvFedC1jgaQxw2+SinQ4S6o20Z1abQA9l6W7NsPfV4j2FedRxKDKotBDjkrxvhad1XKGAFoi60R+8F3wscItiyv6In5GfdioxMQbG2Hl63jG6fOca\\\\\\\/2XkVZpg2rRgujer1h2eZwUYw1Ru9S718zJ56G4YuLE5ZJlL989apOjCMVeeo0o6xiUbi2YC9bqw1aO8paFg9\\\\\\\/QXSWJAVM9z9F1hLaJ2ZsaTs8WOSkJJk50cv29qG1x\\\\\\\/GtHTtfeCtIJ+e2xVG3Tzt01KxzL6ADsbjv4adImG+t1a0GFYBRLOVzgq8HZYnzJ2OU+AXogIWlmRLk\\\\\\\/J+S7KH0QcMrgaI4NCvJdEZs5hbbgR1kTJ\\\\\\\/XnPYh5xpmvdHV1XlTLqJurx2KEqzlNWUPTgFKF+Ycyk8Q1WoCWqYsd2zdD\\\\\\\/LZ093iUgVR\\\\\\\/B2Qf0pd+QAaFWPA6kvCUMtuwepmPM0jOR7hamEmBFz+tEaUpwGwrRB47QqwvVcDSq9heBITPx+FjsmNfWivY068W00ZtpYSDCgE3JkXiJnPaqNvR9L3xXazdbZ\\\\\\\/xQqMZUfEnbsCgoR1ZbtZ0i9Z6DsiqlyDLZthbGY3MhclK7YX8y\\\\\\\/6GQuBmVBUIh9CSFrLbxVy1W0I51TEbJac8xyB3k666I8+yALKHWCJl7zq\\\\\\\/YydEBOvNL07\\\\\\\/WutJdXUcslbPUd6QBcb2gu+mbtFivYPVMTBql7ON1o8jRuIZAoqfwUqp0kDOT00PJn19EnhTc69Qval8kXhj9LXWlIK5uKwhQ0JwwiujgaKKbXsSUrmt3WumrRBdsFbNCsymw6jvA645JAfUe3IRKsKghLgWuVXg5e043aDYsZJnCP9glBLZ4m\\\\\\\/Yx5dBoewJKOMvzFYURHMh9ydBRScj0Ik\\\\\\\/eN4R4u2CuyybkOrVfW\\\\\\\/CFheQ05gm1VUsJhPbfwbvjykhCsnLfNLspBhz1EYkpAuyYcHGbs430yL5xUSdZk46m9fd7k1PS02ZgX6f4z+nwzmLmeFauknZYgwsQmgimFML4+\\\\\\\/Aq\\\\\\\/JWL2VynhGZgmH6H\\\\\\\/vJyA4rse3P+KdC+zBeH4HviVhqKN2Um6IAy0qzCAVAQBU7tiGd\\\\\\\/gCCyRxvCO0EbHZaomf1QRw5qYBzqA21l3x8Sr66GJZkJzUkV7wkarGYBzK6Pf0hv9OjbobO5cX+U6ujzgc11hpf4MYzdYoSxsbeZhaR4OJ+KXtDsYAmq\\\\\\\/jSL5MmZcRC\\\\\\\/n9tZ686Lb1DCHj6WV9JPNFsXqHj2VyDfQXXeqHr8hf8l2tJS+D5oVA\\\\\\\/ijmJSHjsgZPeoh2O2vcd89ms47Dfqd\\\\\\\/9i+iNUyiRySgujTrLX8SD\\\\\\\/9G+\\\\\\\/jg2279zBqSjUps\\\\\\\/FjXT6Qi0kkFbP7NVMfiO2nPrg2PtAUG1ksOce3GaPn+PuDB26rwIk5YKRWNk3IsMQ9ewcaAgfMmd9FRUJ46u1EezY1slDN8d96zjpqKmn5fDE2\\\\\\\/a4NR3NJ34JeJ9KP8UnkxFU1MxpsdMsS+UJ807TgRRbUpGg1b2YfJfrfeXEZsmBOMyvqRUXUjT8FHV1FTmF9cnox1k7s5QiY\\\\\\\/Mo7Ju7gUIOxCXuLtrLXOXuu76u1v22LEFpbKkKY2iJVZOkUIt0yAYg7VO634hSBiFKqfTP7rSu+PfZRMUr0l7\\\\\\\/2vSI\\\\\\\/02wYhjYFJsLaxTfhdl4nrU8fPyUibRLTB0xfqJ0HX55p6H0oA59j1vgSzlZI+ttjBc9nrkJBHg2UjmyN7lZ3FigsyTcml\\\\\\\/9GkaI+SJBVMRYThdOul2ZND4ptmoaSeXIat6kWqf0H3sJGEdXp4tnXPEAiGbwv+Eass0IhYZIUQ8tdFiPqNI02SOKDRskcG2ayeUSlVWOIWQKbZMoHbttPvWT\\\\\\\/NC7xxlid8eWvN\\\\\\\/KcgyDeaGxt\\\\\\\/mYKyrxcNTEKznLJy+cRGWmGHOsvHJEy0ajQlkVsqrflq755PAXFhIr72b2UKg2inb+gBdy7PtonpyWZkBa1CfIQsqfMhZnfIs6RoUDLSMqMYNhGU1fZ3EFIDzNh5Req5P1dXiE\\\\\\\/32nX243MU6Fy3CrVPVGfQ4Hu5wdUASarBisHVBr\\\\\\\/L1sT2HBLidmuHUnm\\\\\\\/fM0YPD8zyu\\\\\\\/zxMssafmLFb3n5svE1BfNWmSGtLwEFQk9G1pFnHsWbYkqcRZtdsks0Oa91KWedbZ+vlTcAkLhjIh1Zi+8E9GuPMKQ1iFhAAVTn\\\\\\\/M62ut1jYaZUGB0nehfk1gm4A4+eJu\\\\\\\/2eon9cJUSjhrlVMm58o\\\\\\\/rTRaHGrb6zrNLcTzhuJ\\\\\\\/mDpy6mUGNQqHa43IMTkQH9ongV4sSlMjkyt3++o\\\\\\\/D5nOHfZTL8Ap+6DefeuuXgIhX1K3oLYjxhUUiZcbV4J61+oLfaQ12+M0ExGBWcvkxXqV\\\\\\\/YErte06PA4L2CZLHKWXI7n3AxmPGOAam4x6xNSDL6J8OUhTKkjCFuwhVol1Um8QOz+moJkbKKqEATnm2xZfxYXERJKKplLlclf7u83v576DuR85P4niSdlGmUcNz4Hc\\\\\\\/MSezRFf6UohY2VFYcmOxdRiOtg20aTiQmRcqhs3T7gu\\\\\\\/24pqhhJR4drGgZ7SSJAgQRzqqmIVXv5uGfWWuy4q5oYzORwTVcmVvtSa3cFv\\\\\\\/hsaaJ7CIZmJZ3dtEB0Q8+qA7R44uXlajXPn73kO0Cn4QP\\\\\\\/vQDDkHhUmxqAfonLoopkdMHpeGF0fPIynC+8DgXwclxVLZhc5+O7iIWZVDbdz7dJBhiuRuBoz9lXxW6OOG8WIKhsX5NOehwVIyp8iVAPiOh34y7VS5tDXevE+2leHoY2McEQwLtiZlE\\\\\\\/SQmqaKob8AwIE9E5R0sXnmTVP+YMZJSmSkt5K6Gr4xV9gwOKAJuLZGfzrHDnAJv9Rdi9gbX2PjSSTcoNlA0UEy8EdBtC5gaYRhub6BL\\\\\\\/ArRnWJ7\\\\\\\/JprbqrwEJpqzScPabacmQ9AWvwNN4BjiNPgXPU\\\\\\\/5NkW47iAUZkLjckMA3NCw0xxuQ0A\\\\\\\/DoVcTs3VTl4Qvz2zGfY0j1Dqnh\\\\\\\/MjdV3qyJVpt71vb32YhfVDvf6g55AnV19jFufbm3dMXz4KY+jvFfyTPp\\\\\\\/LcaJ9OWgaH+\\\\\\\/nmUFpr51Gx+OKgZ\\\\\\\/YI1I97w7BOnaEKgU4uzcZQ7y5leJkAMDJJqj9n7cdy5\\\\\\\/JZB0Vy9SEEUMgZZh8cjsF5qwkMtbsHlUnoDm21FAKKN+\\\\\\\/qksI364g\\\\\\\/t8kwiYWZ3hGfh\\\\\\\/pXQyqvA\\\\\\\/QF8JiPRv38RHkxwil0fEul9B94uoixFxnr45\\\\\\\/LXN5VMs1575tQhiDFm\\\\\\\/+0BhE+R7vq3Uz8xX\\\\\\\/wfLQhshTlPA7sd93EjLRI2shAkBFn0jsm6wFTkJba52+AygjQ2iXOzwwcYuJYElqRQiyPlE+WNen4N+vyg+RBvcg4\\\\\\\/7U07Vhl5Q4EZrWbTRYZW8Mk9BYE9Z4zMIifL92US3qX\\\\\\\/8lK0vsOJlkDzyouVOYEWpdywuCv3fwB0iVGBtCrmRAgJo2qLkvcdlpyPheopSaNATDSAGxT\\\\\\\/5uhHnTiyEM7txYa+VBdUVVRRiFkyKx1ofjRZfivKNC5asftZVmqAgZtgsfz+RKvjaJ8PL42fJTTHT17i97LANt5RSBj5VRQP01k\\\\\\\/uSHbg9HT413XQ3Oewk42ODVJaCIsJH8P9aCTcnT8IBIa1Zam4I6TiZ2nK+wy70rlehC372AdlEREf8lrWEQP4J75HzVP0khmlbDm03lF2vDSVSI4WR5SXxA+LSc84JAnf6gQm+qggWBsK4\\\\\\\/XzzoEPGDsJSu4+DPvZq37AmcHysFVolVTblqhVe2K8V8dE0LwI3kGqLfHuhR1dk+A9byej5FB4nrXTRp69Lxh7Tlg8liPv5fbRctdiqFm+w4OuL5qSXL7jGHVbE22TkDRivhlkoglBBLnUkcI6PV9VjtFjogGU8ZQVXDn1Tf524mX0clXI9mp67CWaWq3rytaVPhNVQDfO9MgdqSlRf9BuPelHP8j36LWTTGnPN8BFIPu9osHUcUIP3lgHetIPphcsnU8lUPmELYRyXg\\\\\\\/IKKKaBDvFfMnprQue9RLy6CbQ5+UkN1IjndDDudmLzIoe0Wwt6GSKvNzSD67fAEe158eogCDnicWI5fRy23J90tg+3fwBqdS6yRZU47RNtZ2aW\\\\\\\/vdrDd9XoneQ0buJNSzYyxUdBVXUNy0H7i7onyA4RRtXsKnoxzXGIgBt+JvelgPfKGAXiOuvcWd0vgLu7vEsxwmR4mnm1o8FGiclTOzi59sERTU5ekreqfQ0uQr4NBixeD2UsH4Hd45QyLkjybh1EeLe9JH1zOzBgWD\\\\\\\/MkJNa9e5v5vjClHfFmaNHFLUlekmGP2lecbTS1Rv0vIb\\\\\\\/lfHFNR60juPlxNEZAYahypiPUAcpiMqxuNGE4+pfGMRJ6bIcJng5cbQ6RfDbW5cKep5IciZqyjsWyfu0W55K5vGz9JIMD6uwqeTyKQuJI8QvSVdbUW6LADji9oaWIBpepLqGMc8g+fqdYWGEAbVu0aWNoDH56L+wl7x+CxwPcj4F6SpnoQ1lDJNmRbo9UTpo7nha6gEuEiICWFQOgpkSF7pslnvnRdcGFwdTOJOuo\\\\\\\/Sj1OLQFIqZTCe2XtsOHfR6MlxbrsMSPWM+jfmiCGA13fISMnLkn7UC8sGSVQ\\\\\\\/kULxL4t47Ywl46IqI9qRnWNDUeVUqydHx8H0rT+za1oa2DDa9se9SqrY09YvnS\\\\\\\/Zhpxvn9dGkCCS\\\\\\\/+KVWvtne\\\\\\\/vrlmVWTviAdYxfPH1UL+V+0Y47JmdRS\\\\\\\/woZ7SAZY1vwvK3Wwrx2wXzhAhzFskTQ8OCbUZC5pEI78L7\\\\\\\/G2XUwG3sCCZzL0t5OeBEfzAW1FYRMUvQ9yA1AwuTwkS54KFmFI4tyNoW\\\\\\\/FvP4kK+xAlxuO8m2AA9ufLWNkPp8ayX+jHq9EyiNcvFQWKlCuBwitzh2SX895zfCjI8fJCdp0deuI2+NXc+1+pu5uGHSpMKgw84dh3O2lnww+jmXoasG4XfIzHJOarJpTJ2Ngel4t54Fx2L4JLNuV9EJoopPmsO\\\\\\\/AKifTJZ08NZXl8W58p5aS9lU7smFeGmpdxKYHYsZTNo3u3GirBGbZc1cmf8b+daXBuy0xwSDxp\\\\\\\/1A+n2h1F4UfhybHuZakY21KbPyr6iGFEgLWmCLX2JKOEfrDmkO0CINDZKuQWwmDp8LNL+2x72yu3n3Jx9O4uz5nfLriHDsLJlUi1sCZKK5V3LGNPPohYzs0EsUOXPrOAQFuPvuaIxjSdSqFCTUV1oyPHTdPpZWlKXummDfWcHvFlLPZYFf7WpTDMzvFIqne5bB3qah6MYmYDPO5nmhrU7VRNFkRhPzF8fmKb1hStDt4v2foiwU81DuUN2i+QoGEBUhs2yk9E14g5TciaSExNGOU3oqwx\\\\\\\/PifzrllaqhUj3PVMBSSPMyFL\\\\\\\/Bv38oRuwkd5prrUwuUzRBEtJCqQbjsb5fmZiq21nmmx0FPmKHD4ciQmCykB\\\\\\\/Y\\\\\\\/uOGn5wZGzr2tRHAqi3dbHIX9PDx5WuMeILZZYC8YaXrO4+YK4ABZ58GerZxRBU3cN0330IE4rW1BEpAxg2UjVJSfTrI5akP1wjw3fbvhq5LIvx6URwvqC4dfhGrIknYx1QxFbrkHjhfHcIT6W+Tra2gozkdcqamZbTb1h71e\\\\\\\/\\\\\\\/3sGhWz9\\\\\\\/dQo3e4488HNVlT2mJBjYL6rLzHBx+\\\\\\\/T3IlFompCboL708SKgWeWVViIaNqHefRO7E6x4RfLKhxLopJvz+xV18dPfcUlY4LVo3aCHJDRe5MUoa+E97uj4P6imgVBNLn3RvXqACtrFQPmhjESG66UNwaIEB4JPuCzokGAMwXFqBPbpem\\\\\\\/WL\\\\\\\/HOxZ15lFHeI\\\\\\\/R0nofe9KuyrhI9ZjU\\\\\\\/9rK5u+uqkgmWcBwYQAW2KfHn85bN0tSwrUyPpJIrlrGC3R9qW2xaM1C39\\\\\\\/d1pSBaXTcV3y1AsS9hdcw\\\\\\\/hrO3F34SmAwSPiW\\\\\\\/bzzS1o9z395uKwairZYJttRciwJZ7OFMZVfkCuwWxc16bFMcXm+lSqVtCXM3T3bPzIpQEQXQPvnhJXtnYOoPeBpKfpJYu8RYioe+6FwcV0qYYHtcR3eIKYom3qFQneDBV57XgKxrBnnl+3BJm0lIpDuDJ5FWhKU+wV2axWmaiN\\\\\\\/g79jjTiqMZhNR\\\\\\\/PAWW5YKzCPJ+vgDFo0X4VMFMCNkNIwITsjFTpeN7vVjZmZ+umzjlRUM3Oqhq8ngCWE46cgFKiYZfRV73JMByF6TJQObG\\\\\\\/0POmpHKF9vHzRD2zBpayrlJe9W641Spa9shtYQacSrItKzbp8flABc4Z3BwJr50HdRGDEdBOl6hCXk6lk5UWM9YmJfvQh9CWsTcwataOa+U\\\\\\\/4tQsPJGntDKP7rE0fGeUV3bYtF7reeKH87N97kBITK1tq0iaeqFUZVSn22lqfgwiRx7fRYvjXyp+x4qbNbxnrlxK2Ey+eQCn0Km8Cszip6z8LFAVt80S46q6rbywSWEWM1IBpmhyAD+wDjaZ2jbQXnXVKDUkcnRyh+vOIlz+TS82EtJfX3SI3vcA1J4VkDRtcfSMdfKyFAHK9KVpJGpgQ703sCaP1Isk\\\\\\\/H80UmomKhl8+zMH+LxR\\\\\\\/nnOtZylhFXj8ecnSgvIBFN9OXgY8UKyUHP74xGqWrldHGD9e8RDC83x7f3MeLpJ17Vx3gG\\\\\\\/urEH83pKvW7H7xquYNGg\\\\\\\/ZphzlZHNAX+gfugy4T4PewlG4ghlLRMX3f0Is5CiF4tSDA11Kvp\\\\\\\/X0IOf\\\\\\\/l1u0O+eVdO5\\\\\\\/W785EcoSkc1r+kJDQwqSCvcu6U9or58WLAQ9IWlavL3z+5PIFuHhPslriyuvB6c3zP+XI8eQvCdDHK8ghf5geSOnfr3QOR\\\\\\\/ZttuBMFtIGkLROcDuX2kPZSrVk1l16Pmq0l\\\\\\\/H2Tyna9p82aXPAwshW2DOLursAPYnSGaHqrS874VLZ12VkcBnHUTEKxUHb3CpkEJAhMWZ9dQC+KfY7tf01SdrH+ZIlp0ahnR1GBxQWhVna0o2ewHJHoxp6jwSjjHyI2m7TbsSBWAinYIC66Sp655K5+0vVNC6KaIDUWFoWNFkXWkRBt\\\\\\\/ZlOTMU3HSZnGETc4ZmwxjuBT+D6Yyzs\\\\\\\/8URnmpEm449WnFnSEspUmkiWo6i0RRGz\\\\\\\/UyYGi+6uPbbbzLXGAID5c74d6v90TzjRU3W2EtUNBoIfn1Hs3cg63LFp4bMEzmL5Fo+L2FJgv35Xc32NbW84JD\\\\\\\/Y7NedAbNTkLuc2655AXuFlUd2GwKekEaD74HH6waWSg1eSUd9n7WJ24Be8zIjGz9VnMcqZieZ4goNTArnbOJi0mSlJ0JoThNA0MmKv917bW4pWv4v+hIiW9peWJKShfHyC3No1g8JKhqC5q1XTyTWiHIWXxH1rxeRuvV1d0PfP+HsmqcnpBP5HcNRNQgE7D7QqmQYWV3P9xB+0WhvXz91hSADRdBiRT1pkVrtx7y52vzJu7rBlD58tJx1UZfuZujOa4ESPmLlGpc2FuwHbfCwWD6hdPdAw5qcubpXTVpwlVTBfjjyOXbbnRWPNeoa5P+Zhx8sfB1Xd1S6phxrq+FfxQMgXWSUmCvqfjiO5Eq2fgEMvyGP8cLENnMZiHekonIWeDPG2gCZ45KVXikqc8OiV0Fzub0MwsxxedP6OGNLfidRHQKl567nghf2A+o0BdxOoSIiBjC2oo+DcM9rLnDILVOnQBeKklugrHBcyQPKwqfKcLug\\\\\\\/WfdIM1mr79twEOhdhgorky7dY00X6EglhBb1TqA\\\\\\\/bCY+D6u+lmqSmjPtsU8gZRCFrTV8TdbpFjPAKcY8it+7Fv7wyj9kRBaEp1oZZb6I2anmPTL19VOjX+zYYkt8theX3n1KKAnB9COJvgS4jNqZeczqaTZ074h+PXlQPCxG7DjZe6j6ZzgexaOd\\\\\\\/I41tm4CQzqOt7mLRmTnlu\\\\\\\/jUra13mVb\\\\\\\/1ghkhEG8scS3jgite0sLG03Eyss8P6tSmAY8WYnlHyB0vDQCdEubO4qanP3QU9ZqQKuMmXqqqYQRLzm\\\\\\\/GXgqKsS+8Sx5FiGTvUB+MM5rclqAZmFeOpfAkTSm4bJdTJLrQfAyf1p8vQt3Wu6i3VAQjCg9\\\\\\\/npo2ohC19wNvnKxQy2TIC10JZBwFyb5lAjU7NvV2LZ0C45Gmu9BCJVojbM4EB44IqV7QUMW4IqOtwdrk8scI8TK1faQ9WCi7\\\\\\\/w+xeAnpH5CNvFPSs\\\\\\\/nqpjhcYC7Bnl7MyeO\\\\\\\/xkXVQ5YaEt+tEa2RpgjtIyBQb3fka+3aTUoKM\\\\\\\/h1DumC8IB0IWo90DBZ0WXILNCJO1fXfrNfn5VsSCT8UjbZXT+ZzQ0Ep2xl8xTx0s4vljpii7uKinzfPfatu2R\\\\\\\/Ftvc0bcHI+OC5Yf1DK05TvIJ9zCul6\\\\\\\/gto3h+DJMbJyK2wS1ASPRL6DZdJaAeNM4qPnYf+sLB39r7nWM6dr3WGSEVxEKKOawLaT7cny2gh8CvibC70qF0EEGzyJxmTMVyhYq7cexIQSQD7HOpJdtVMxM0kLnU30QElty6lvJZ47GYaxSX\\\\\\\/w9QeGdlEyFE4+SRWIbD46CY8o3N0FB\\\\\\\/fCTJvGC+AUlKGqzK04CDsb0q6ogWr19YIijxUmXsWdKUocxMijMrbMNGctqW8J4eNNxrq4NWqLshwHL0VnJj9hOWuX4W\\\\\\\/cSSLHAwi4u6YHX4ZgzMaA6fHniEnSHRypQYUCEfPBivbpk\\\\\\\/CCfdyKGifgeK+o\\\\\\\/YIv5PBKOWRCMrH1ZcIGYZCLt+zOTjrKRLwuWMDn5qCza85HrqTnHMxJlI7dMAPmqAojVEs0neuROlT8y36imNX1OWG34BlIuC98j4Hds2OYVcxnce8sUR76KkE9AZVaQoIT+Fcz+QpEgzbUOyMaew\\\\\\\/3vyfE3lI1TwGEQ5fL5JHtZVn4IJgnO1kYsLfjyyMpDWH48Y92hifhCXRa0\\\\\\\/XvHrHSN\\\\\\\/8fuWI4KaorY1sevkTmpMQSKuTUiaHraps9KpuLL2axobPVt5psoDhfciVVjUcxiAu9QnSBXgSHVeQpaGxOp\\\\\\\/oLLnIZtm5bpIB9r9kUgeAEorZ\\\\\\\/WpSY755x6uYf8BkopcI8duny7jof\\\\\\\/3JVbXeK4WH\\\\\\\/3m0TJJZqwvwIfc29FQ2zms8+qZtntrr9A\\\\\\\/ZnwShC96s0ZTG+NKG4hPYfaP2iAZwJOrFUpxJK71bfkmOGh5HvLXGUPGgzFDjhTxNILLhd1WIxGb5EtVoguADBjOMYfUsetBopFwp\\\\\\\/2Gu2ZR2Guz66S2RvwBvQxOOSomcQp1CmV1DyCO3mpwLmKR8p+BLooth1ktamsgDW7DoFK\\\\\\\/O7R7KYc2tKxEEBurFIBXDCppB7pUDrvTOT1\\\\\\\/x\\\\\\\/GkEXbE6k5FOPywMm+Evwn\\\\\\\/zCZrD2sEnxJ+vSgTD4Kw2l6EJ3wSPuh6rx6sUAIXp220yxjTBzkuJSb2QFjp4dSizrDaCuVQE5Y9qZ2x3c9MVGfiQkHL7i4MxeEKC2\\\\\\\/TnXj7Lt4kRAvx\\\\\\\/nkcJC0V6Y0JTkAdWNWeCz+17i8E4GvsvYUCdEgKXLar6gav2rs\\\\\\\/od0lqZT0rkpcieWnI5LLgE6kSWxE5cP6cabtixx6682Dnk2nPYWdLuoe4v2gtFqDC+xzVsL0czOP6rju7Lj4RPbcYn28TZfFIPn2kQ4ra\\\\\\\/y1iAjdMUrn0ogquztG5cOE5Mz6JndNHLSWan3lAejKBkwj3+V3I42XzBx50uDiRwhsBL5k7FgNj1ZPl+ekRj+ucmpBn+pzdDvzucK5Mzq37YxxD\\\\\\\/VUBz\\\\\\\/x5LUAoDMgp5Hy6OUGmvmDxPIBYUEkMzVntfICF4N6u0Z\\\\\\\/zQ6iwKA676xsk4GpPoNJPP\\\\\\\/n+2kRUi1ESbv6iMheEyZvcrNjzSwimnamcUeN\\\\\\\/tPBJElF3Akg6wUcAJEDPoO5LJHPdctZEfy5vFXB7r47Nd22sg8emnsCZiATc898RpeA4qxerEZ\\\\\\\/JSI\\\\\\\/cMykO0bUHdkXmn17DAbRzyLWdWZ\\\\\\\/v3HaoEsV+xn6nbMNWu4Lo0zh7xETHimawwORgbBSZU\\\\\\\/4lB7NlKOj0KRxlo3z48f5dWbC7BWebGr4xag3aDfJPs7GVOrQ+zQ0DKlzQfJLQnYnAVreVAepn8+hRCbPgsN8czkovO\\\\\\\/HawTfiOy\\\\\\\/973PABxb0Qp1EqqWdhOiuwTNoGoOSHqdA27fGZ8\\\\\\\/wXGlyuwrSLAm3i3p5Mnbi+UHifT30fNJdeAAcdQVNwg+mdxZk98QcGEiIYi2YFX6fIt4F3+K1WjqjSGOXuuffWxj4PWj08ac\\\\\\\/IJz0M8NECarmPcgWojelbOXSAjNztxpNhhU4BNyDTec5LL2w0FJP17HvyAvZU2aPx9JHWEgtYqCVPAVRFMrfLBavHbXBDANZrd+KPk\\\\\\\/IRko+Pu8E1zkg\\\\\\\/CTptnQU3nM0Ml5L2jZxxT4HuudAXMi9uam3cKEk3Skix+y0Ev9uxwiT1rop3RESAl2IKlJhiHwO4CjSQBPmR6vt1+RNghBYOiyfAaMV8Te+UO7KTcPB1wJMAuYDjpNPauD6iXYsA1hjZeWOOkWTS0p4LgxZQ6PpbmScA9nNwuQKIAW++BbKGEHGkpwipSyI8+PciMGss1eVDD7TSkzqS8gk4ZsRbUZAOo\\\\\\\/ejNNf5M\\\\\\\/83gZgVPgYhu3S6OsEMScABPu0Rx5l36Wjisk5TWacmJx10PdytARfWtinuAUX1VQwOsblcA289N7KhNPdCncu7GDKkbBYW\\\\\\\/QVpuv3ActPtyJsahl\\\\\\\/d\\\\\\\/KXMtYh\\\\\\\/3heMyNxJF5WzX\\\\\\\/vimhuLjbRzUoElEwRKDhlNSx\\\\\\\/d7LEBANo9vEKRzJ32s0bWh1PC\\\\\\\/+\\\\\\\/rKgEDex+GzzuQrA66kW6Ry518Mz1W5144hPmtAPdaSu+NEUooNr03IF7RIBwxG44diam\\\\\\\/tqT6mGBeu4DaKw7WzR6Tz+ry9Ca+CWBkR4O83yTaS4xRuA6G68ec7ZfclgSeWQS2NQTxK2Pzfj5xhaEMxFTfEXkNSYSnndyOlMsXwEwE9gGZsnuK5si9TVaqJaFgU4PqRX3u2S0JXey6SnjV230rY9Pn+YZHLUBPUN3oVlr5Ql6G2XWCK69Gm5PUTRoOrkXm7XscY2EQ5eGk2XIgSQXupnCztds5y5ELn1BmzgGZ0kNEJByxkjbJCA7Pli8Quf2PgH\\\\\\\/JLEowOjHxjJGV\\\\\\\/Bu7a\\\\\\\/adUAZ6FPnJ9EYDNpjxo0xTkJDx\\\\\\\/1XekmwuoW4WvbxZErdOPZEFXUTuFGMCbvfyKbvGEFwgggv93glm3DwjnGH3GvRB6SGOKid4WietzKOfLag1ngNQOVF7DIytE5yNtMqeHrX9exyKliX90g8ALKpkoSliWGggxt9W74uOTg+1O1Mae83kfLSkGmXcnJH3AYXh\\\\\\\/OZ5PenKGqUaGa8m7Iyk0B1NrodR2MZyw81Hb\\\\\\\/zQnrw4ZAapZvFKOtZLDQWcp9aiPcfqiRZ\\\\\\\/Ctuc5oHLRYaPEz3AE7AUkpjGl+T0z3XpyfgbmFmReBP0Qn\\\\\\\/I07qmUqy1xW2QTFcdMJrAyS5yDdFqGSGTSaIgHcReczHDoUQt5OsqpNj5KEXzfmfVP7h8rt0\\\\\\\/oWASf6KSboNVelBGTM52RrkaLQEQPgOMU\\\\\\\/6tWFxU2GLRv4u4ABpvvoAUHFPAVcl6bFDvh4oYR8K2EtttNOoVCJS2UQVGRhwvYicxsQg2Cjes5MbuVAR3Yaus+ENwvm03aUrMrIVIgiWUikOceDhjOorvp8m\\\\\\\/90BPpYI2tInywgBn5EGVWZXS2QcXajq2eVtzfSoqfBUbqooaGl19fAXcgIh9bMd710WUOHD2Ty439DnQfl302uBPIddRR4m8Uq2d4aT\\\\\\\/hyL8UC51JTXp+H9ZsRt31qCn11DRmPhgznXEH6NCXjcuxABY2Yjd7TaJERNDzA4AVRqk1sUG2nGTpZsddaoQsso\\\\\\\/2vU\\\\\\\/1orv7iDhi05KMuBnoG7b6uMK3FlVfR5wMeNULJF1c4Gnqoth6AvZU+\\\\\\\/W7P8XO9yjSdMruMO5zuzuL0oAWKy5qV3eNBEy5pExEHN56vddG0kEuxIHQ+QZcfIE1gOlNgl+fLfvRuSdv+m+N9C6gqOpurvrY5wjdJlKvlhVFbVsehBRJxk+0irz8GDPo5vYDh8tQWLlgO0+gaIQxcs\\\\\\\/RidQ+mySchZUl+SxpMVaK2ZEie3CrxyjqJagPCeOLadEhMcOlg2vSxY4Rx2fCy1\\\\\\\/GL0C\\\\\\\/1G5bvu0t2IAnfSlzxA8HSA6u19EFingMh3i5kNBshPqCXC5JzDxsru1LLnrUrnH8rKEaeAqYUsSGxf6lHvuDDCCcL\\\\\\\/BSFm7rnyPWI+uLuAHSCPDTtXTSDG3fA+LIVz9PHd0wdgOXTRpd1e9vnCwJXP6JwdIQiOMIkyqomCUCyRw48gnX3HOfLMzggsBj6b5M4XOJnIwtdyGAuHvoZ2sLXJOerwk1V+lCURohHY\\\\\\\/kD9vSg877W7ewadtMXmc0XKQvCNJc9b5isqPY4wp4JjkxvaUMfj\\\\\\\/vBfCOEJoos5tu4vfkJAaJ50cMU3J9juGwFIQp3Z83tiRT0oYICFhQWQfJeyvLI5koim4nY9kDPuY2c9b+4WGvxG+fRB75U5mFAa8kDY\\\\\\\/Aaz9fkbnC\\\\\\\/KC85PIfrIiNM259lUVpgdXq02wgc6ZaGu2WPT+1eTyVJ3vw6AfTTGVN6YJnXwxpu4GTeQV5Na5NXgMzpvQ7kG0lksBHvKbjHhUZ+ZzA1RoTrXn6BJpxjwMmoq3Wo3IEf6cs2v\\\\\\\/yggz6zoXWytlll7NeDuQ9Vlkrpu85OA+tyIocoG5cMKn7bXG1Pvj4JFVkwLO7zEVJexZJ6HkRtr4jYsVg2DSiH4r3EB3zBu8rq0vg+9ApDVgl173pXSrd+FPrIZUG2M1+W9Mx2QrzICUy2xPZWA6Xl2APMvc7ZEIiv++8Imy0omig13Xz6HDB7evX\\\\\\\/hodl2l5\\\\\\\/221RQ5pByBG4gNC8vgID9a5gQ4JrBxaEFOG8O+KPYWgov+henmhSnPRhkkQZ80JgdDY201qI7Kx\\\\\\\/8JDFqrRoLaPVy7ZUemLCVH5oxwt\\\\\\\/2To0yi4i6yXKQWY1LfhOV5beU5QNNytypZQ4wW6Oe4B25RSSXZT+YO4LKQWpPZ9pB\\\\\\\/32JXMCdafmbGC6wy1K8WhVwitukwvXYezfbwUWJxMK1MBHGlW6xZS0n33W3EuxFKFFX6cb1mNPNU9t1ZyHrbpFYjc1H2STyeg8lcmRxghLVzbf8Fg2xMzixZ81r7sTs8o3nc2UDakMk3VnNyZLZURLNsLKQt0+pq9GWow9\\\\\\\/odyBFHx2r13f6qD37GPlbrN8YU4KWWfUI+A1IEHvQVMSrHoQDG35bzpcE\\\\\\\/YQzZULj4dmFCu1ZhVLxKEXMyNtpDF0ODzIyGEzHIc+rkJqIipJTO9\\\\\\\/CXiD7M31+VOtet6120MXX5u7JUwe3y6kPQgM03HSCTN57KmGZ3DiX2ZO4C\\\\\\\/U+A1YEanN6UWADUGZas5H+hL29+2mK\\\\\\\/wMmXo9d68j2dmOg7roBoeLxivvWYMEzCqNEWFLANBi\\\\\\\/mncrI8BMwj0J14aycjfPqZuEjlGL1GnlC0H\\\\\\\/fb8rRAX9DZWDj271Rggmm5i1XCnIcfvQW4eBosL+KvAhu9OMhb9R1neYksl1d+fgVvw7tIG3we95FpF23w9intShGjjIf5JNUVKCYFkwRT7uEFk25rZp7a0YUTJBCeG+MJN7rBzNzWIRVLW+05iWVnZeLd\\\\\\\/vNun6NRctcZZ6FcLCNi3+G9bvOSwYPGk0AwGOMdrXGF4bN+uK6aTgfbICQKXRuOuY+752HdbKRnJPxVqbOW7v0CRvbhzocyOhAiAkG4fH2T42MHbQLDG6cI6Wy9v1L44yrxzcK6AN2IJJpBT+KzL5qk4O\\\\\\\/rPDD4dWAjps3ug\\\\\\\/uXvVhs5pdg7ozakHGk+YSY2GoY+GzUDeVR56CA4c8Wdquqf2xpqE1NWx3HQHVAb9OrVjMaL4pxS+tav6+htQPzhCv9RE5wcVRNmoEEE+qEC8O96qJ40W1EVh4+tcv4kufubO3Kh0SS9Rz8qD7Cl9d8drF0QN8lK3JIivbPQwWRF\\\\\\\/s6Dcq5Wj9ClGrdgAPz5e0RlRwtFP1eONLFFReJJSdM1WL8pnpfuh0EKSrB4VcJ2MzvWqPV6I2WSGxtqC42r5Xm0kxRPK1oRJvg+zKy4x5ZdM6x5DhvdjX9WMaVCdLKbJQAiEUGM8g7+S5k8flN\\\\\\\/SWwx3Us2bh19BkOThPq5sy+VhR5hAbc3X39edA+AFGPxewd7RWDwZiwoXm92NbRqGIUsklR27U88SfExK2CVLrdWHttQ01yw5Ug22ARJekJDicepqhzq2A9l7UzOQAG6DAeGAu4Ct2L0sJ0uuYNbcS8m\\\\\\\/BPROxk1aGob0gfb+optG11DFQ77xkH3Hzla3Imp8FVVwA5dg+9YLrj+jG4gjAh2aJnkJsLzMyohlmhCfC2tU0BXIADnrjZY8kMw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ZTZi0BZWqrNSFp6N8tcGUw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e2b6c3b6a17072c30418dcf3444594427c75fa9a007f4e28f7de698aad767f62\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022xM+OOo+15IJb99CbEwOwlcWRNq\\\\\\\/Q4HTdDCmEXV9SBaaHsEqkLnA37\\\\\\\/sEO2ImOgX2ATKyuHs55egeJkQjKQUykqYS+QP2tIJIJR4uRMGzddf1J1Bujnr5J25z5d\\\\\\\/wohEsndpPHk3TOhFCfljKA\\\\\\\/6n4clqso6BfpgtHxxiNBp8ZZ+j2ExYQLlV47+zTTa8C2dKQeEuBoprw94yvMVueSljHX05V\\\\\\\/SlCgOECPtJSIGzTzW9iRwUGPGxByVaUHOKrhSZlePG5tMWZChzVoXEUoZzOMfpMZTA8FHBP8U+y1oKrGSCwIIUS5I\\\\\\\/2PYdCEu4ybE06LYSRRy1jH+RTyDaCfH0MgEXyOIju9r2KF5ys0j2DGXNW6jO16fJ32iVYlDfEWy+onwwA6vIwisKP+lT9eTRaCh09Qqv9d651eJ5OSfWm\\\\\\\/xQt4f2\\\\\\\/3ES8yy3iJ0\\\\\\\/25S\\\\\\\/qGKpshQrXFTaHOksr9HUkVnte7kmbZv\\\\\\\/XIBzVPNN09bQp\\\\\\\/2b9BZMNj7I25nRJwFdtQQHxLiCk216fKO8I3Ia7HHOKTHiJA+iYd524ZnEfIWXzUkCBZUaoYOoHxQzGBn2e\\\\\\\/+\\\\\\\/pgghxt1i9KJDJLqGmjAUJ1A3q\\\\\\\/knB3Vj2FlASn8T4IiY7t0Qq8UajLZqFLY9O1g3vtcyUkCPYS5NtL3yTHbYfZH64jVgcyuyFemo2rjTriEVWjy2xU4cQEGxMnuYFsEP1uiPRd6w+yBcblToO0XMm0aghJM24DXqpDu7KR8YAAv+pAL8k38knsVBpwceWBcxq5hxLrq2yryDtsda8dt9iQEvHpDh+BPCreSpERjE1afb8q8gUqQYJBNoxXM0ak7hvbRFSaTAPuIsASSc\\\\\\\/nFG5HQocpXBki1u1c0YXPQT0G1OUPKdsJDF9KxidajAwNQEwXc7zSz11Lj\\\\\\\/7TAjC1T36ulbfQXjf+fpI7hV5kiTkDCt6y1LueT7UP6+PrfNhQWIZaks4Gh2IDsr8YBY9uTIQNYCOLLw1Eh9bxCeWSySkckrvCUztm9EJt2yqkWGlIawuGUfSEFAvf0kzknXYruCpMTBVObWmwt8AzC5AxxZu0ZtZmiJBFIIKuXddYTZOhRPlWbkEGDHpZBEgYTRKhgx4VtMM7Ipsd\\\\\\\/vWoGVqNVeB+pL8Adc0Z9dMCSxSwKpbuaRkYw2WVAZpw\\\\\\\/leL80NgowapIu0VwUBiJYMqrtQyi7cmms0r+hbzmncOt2t\\\\\\\/LKiLvSanm\\\\\\\/B6\\\\\\\/A80tEAVmAEqaW11GXoMyF36hWwxAfgqu0szCJFo8i0kQXtsV0y96hi31TCAVOkqAeVNZSlnH2Utw6jcrKf2R8Zw0hGGr5gH3EV6O\\\\\\\/8OQFW7OkEU5qVVi5J2XlXDn3OY8TFrUEm6zq21SB6+8J9+VS6DlJev4YfauJZJQR98iGdR7tSQltwERFP7xavw64ADev5KQ0WiFbinHWND61gDDhxug\\\\\\\/8i8Pdndt7cLtQ4HV8ceYO6DsX0hKEspUFVHfjnxw5fHQZWVwEiGuAO4amLrgDXOGTjNMbeO\\\\\\\/GSUOziPm+jjXZI6uco1NoJ9JKbDTQxnXTRVsNjy+N4JK6KXpvKWEOXqiy+N6D9pqa+EHiV+5orZoYNnVnRT4Y40WaYzL1YC7zAs1+RKJ\\\\\\\/lV3l4PkUi8kMElEDWfOAYHnFGpK3EG8N3NVhlfBu3Ew\\\\\\\/GiceZv2JIRTgA1AFWvV6EGB8OTxDtjqfVAEDWBWYDWV+OssfS\\\\\\\/Q3Z\\\\\\\/TrQyYCVbwkr4UnSN0Pql2kr1rnIHcTs1JUhVgVELAMkVyzFWoJ5bAsRYiynVTMfTA774d7NTI4\\\\\\\/7NtjcYa2taJWIabEsKWrcLN9ubop406\\\\\\\/F9Vc2V43Q2SFRCYIOdAcw3H4cwZB9RIGuoodZd1H7ShpDjGCcmpAx4PeNxOpwe9lRa4JznHFXWriUXQf5H7PNEA7TlsyLG1UoaJV8v3K1CX5LenfPwFvW+PZyagwV19rRxa\\\\\\\/zUAS8ltZ2NGfYlvVKpWSZcOrcDrBDVjR3\\\\\\\/EsnieI0+Whc32A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ukU57XEM4T0UzRDkk2DVsQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002259395b7e1973ad3469fed8dd60ca9f09a0288b9a2228140c3071df477c059ea9\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459917 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1081" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-byPSX0-e1Ik4twpiushsDLoH1WEUsDt45hH3mKJga54" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1081" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-oa71k9xofrUCR0KKyOtHzh0yx16n-hYGZin7CmwJJJE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/6/1075" title="Gene Network Reconstruction by Integration of Prior Biological Knowledge" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/6/1095" title="BAC-End Sequence-Based SNP Mining in Allotetraploid Cotton (Gossypium) Utilizing Resequencing Data, Phylogenetic Inferences, and Perspectives for Genetic Mapping" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459917" id="node459917" data-pisa="ggg;5/6/1081" data-pisa-master="ggg;g3.114.015933" data-apath="/ggg/5/6/1081.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Insights into Ongoing Evolution of the Hexachlorocyclohexane Catabolic Pathway from Comparative Genomics of Ten Sphingomonadaceae Strains</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Stephen L. Pearce</span>, <span class="highwire-citation-author" data-delta="1">John G. Oakeshott</span> and <span class="highwire-citation-author" data-delta="2">Gunjan Pandey</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1081-1094;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015933 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459917" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Stephen L. Pearce </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Ecosystem Sciences, Acton, ACT-2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BL.%2BPearce%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pearce%20SL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BL.%2BPearce%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John G. Oakeshott </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Ecosystem Sciences, Acton, ACT-2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BG.%2BOakeshott%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oakeshott%20JG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BG.%2BOakeshott%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Gunjan Pandey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Ecosystem Sciences, Acton, ACT-2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gunjan%2BPandey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pandey%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGunjan%2BPandey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gunjan.pandey@csiro.au" class="" data-icon-position="" data-hide-link-title="0">gunjan.pandey@csiro.au</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/6/1081" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459917" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459917/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1081.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459917" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459917/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1081.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459917" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459917/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1081.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459917" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459917/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/6/1081.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459917" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459917/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Hexachlorocyclohexane (HCH), a synthetic organochloride, was first used as a broad-acre insecticide in the 1940s, and many HCH-degrading bacterial strains have been isolated from around the globe during the last 20 years. To date, the same degradation pathway (the <em>lin</em> pathway) has been implicated in all strains characterized, although the pathway has only been characterized intensively in two strains and for only a single HCH isomer. To further elucidate the evolution of the <em>lin</em> pathway, we have biochemically and genetically characterized three HCH-degrading strains from the Czech Republic and compared the genomes of these and seven other HCH-degrading bacterial strains. The three new strains each yielded a distinct set of metabolites during their degradation of HCH isomers. Variable assembly of the pathway is a common feature across the 10 genomes, eight of which (including all three Czech strains) were either missing key <em>lin</em> genes or containing duplicate copies of upstream <em>lin</em> genes (<em>linA-F</em>). The analysis also confirmed the important role of horizontal transfer mediated by insertion sequence IS6100 in the acquisition of the pathway, with a stronger association of IS6100 to the <em>lin</em> genes in the new strains. In one strain, a <em>linA</em> variant was identified that likely caused a novel degradation phenotype involving a shift in isomer preference. This study identifies a number of strains that are in the early stages of <em>lin</em> pathway acquisition and shows that the state of the pathway can explain the degradation patterns observed.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/is6100" class="hw-term hw-article-keyword hw-article-keyword-is6100" rel="nofollow">IS6100</a></li><li class="kwd"><a href="/keyword/evolution" class="hw-term hw-article-keyword hw-article-keyword-evolution" rel="nofollow">evolution</a></li><li class="kwd"><a href="/keyword/hexachlorocyclohexane" class="hw-term hw-article-keyword hw-article-keyword-hexachlorocyclohexane" rel="nofollow">hexachlorocyclohexane</a></li><li class="kwd"><a href="/keyword/lindane" class="hw-term hw-article-keyword hw-article-keyword-lindane" rel="nofollow">lindane</a></li><li class="kwd"><a href="/keyword/transposon" class="hw-term hw-article-keyword hw-article-keyword-transposon" rel="nofollow">transposon</a></li></ul><p id="p-3">Hexachlorocyclohexane (HCH) is a synthetic organochloride that saw widespread global use after the discovery of its insecticidal properties in the 1940s. However, the off-target toxicity of HCH has caused it to be phased out of use during the last two decades in many countries and it was added to the Stockholm Convention list of persistent organic pollutants in 2009 (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Vijgen <em>et al.</em> 2011</a>). The manufacture of HCH produces a number of different isomers that differ in the equatorial and axial arrangement of hydrogen and chlorine atoms. Only one isomer (γ-HCH, known commercially as lindane) has useful insecticidal properties, and this isomer makes up only a small proportion (8–15%) of the HCH produced (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Breivik <em>et al.</em> 1999</a>). The remaining isomers (predominantly α-HCH, 55–80%, β-HCH, 5–14%, and δ-HCH, 2–16%), most of which still show high mammalian toxicity (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Breivik <em>et al.</em> 1999</a>), are separated and dumped in large waste stockpiles around the world (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lal <em>et al.</em> 2010</a>). The long half-life (<em>e.g.</em>, 1.6 and 4.8 years for α- and β-HCH, respectively, in soil) of many HCH isomers means that they will remain an environmental hazard for many years (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Wöhrnschimmel <em>et al.</em> 2012</a>).</p><p id="p-4">More than 60 HCH-degrading bacterial strains have been isolated from contaminated fields since the first such strain was reported in 1989 (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Johri <em>et al.</em> 1998</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Senoo and Wada 1989</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Thomas <em>et al.</em> 1996</a>). These strains are predominantly from the family Sphingomonadaceae; however, strains from distant genera such as <em>Microbacterium</em> and <em>Bacillus</em> also are represented (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Lal <em>et al.</em> 2010</a>). The most complete understanding of bacterial HCH degradation to date concerns the study of the insecticidal γ-HCH isomer. Interestingly, only one pathway (mediated by the <em>lin</em> genes) has currently been identified for its degradation, with examples found both inside and outside the Sphingomonadaceae. Many aspects of the pathway are still unclear; however, two recent reviews extensively cover our current understanding (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Lal <em>et al.</em> 2010</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Nagata <em>et al.</em> 2007</a>). The <em>lin</em> pathway was originally identified in <em>Sphingobium japonicum</em> UT26 and is composed of an upstream [conversion of HCH to 2,5-dichlorohydroquinone (2,5-DCHQ)] and downstream component (conversion of 2,5-DCHQ into tricarboxylic acid cycle intermediates). The upstream pathway initially proceeds via two successive dehydrochlorinations (mediated by LinA) followed by two LinB-catalyzed hydrolytic dechlorinations and a dehydrogenation by LinC to produce 2,5-DCHQ (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Nagata <em>et al.</em> 1993a</a>,<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">b</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Nagata <em>et al.</em> 1994</a>). The downstream pathway involves an initial reductive dechlorination by the glutathione <em>S</em>-transferase LinD, followed by ring cleavage and conversion to β-ketoadipate by LinE and LinF, respectively (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Miyauchi <em>et al.</em> 1999</a>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">1998</a>). β-Ketoadipate is a common intermediate in aromatic metabolism and is converted into tricarboxylic acid cycle intermediates by LinGH and LinJ (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Nagata <em>et al.</em> 2007</a>).</p><p id="p-5">The catabolic pathway for the major isomers other than γ-HCH has not been fully elucidated; however, <em>lin</em> genes are known to play a role. The initial steps in the degradation of these other isomers can be catalyzed by either LinA or LinB, depending on the axial and equatorial arrangement of chlorine atoms in each isomer. β-HCH lacks the 1,2-biaxial HCl pair required by the reaction mechanism of LinA, and is instead initially metabolized by LinB (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Trantirek <em>et al.</em> 2001</a>). α- and δ-HCH are each capable of degradation by LinA <em>in vitro</em>, but in bacterial cell cultures δ-HCH appears to be initially degraded by LinB (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Geueke <em>et al.</em> 2013</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kumar <em>et al.</em> 2005</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Suar <em>et al.</em> 2005</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Wu <em>et al.</em> 2007</a>). The expression ratio of LinA to LinB is known to affect the initial metabolism of HCH isomers and is the likely explanation for why δ-HCH is degraded via LinB (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Geueke <em>et al.</em> 2013</a>). The subsequent steps in degradation of these three isomers are as yet unknown. A summary of the degradation pathway for each isomer relevant to this work can be found in <a href="http://www.g3journal.org/content/suppl/2015/04/07/g3.114.015933.DC1/015933SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/04/07/g3.114.015933.DC1/FigureS1.pdf">Figure S1</a>.</p><p id="p-6">As a recently introduced synthetic compound, HCH also provides a model system for examining how bacteria evolve new catabolic pathways. Although they are chromosomally located in <em>S. japonicum</em> UT26, most upstream components of the <em>lin</em> pathway are found in unique regions of the UT26 genome associated with the insertion sequence IS<em>6100</em>, suggesting that they have been acquired through horizontal transfer (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lal <em>et al.</em> 2006</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Nagata <em>et al.</em> 2011</a>). IS<em>6100</em> is a member of the IS6 family, which can mobilize genes between two directly repeated IS<em>6100</em> elements through replicon fusion and subsequent resolution by homologous recombination (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Mahillon and Chandler 1998</a>). This process results in an increase in the number of copies of IS<em>6100</em> in a genome, with a copy of the element left at the original site.</p><p id="p-7">The horizontal transfer of <em>lin</em> genes is observed in other strains, such as the recently sequenced <em>Sphingomonas</em> sp. MM-1, in which all the upstream <em>lin</em> genes are found on plasmids (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Tabata <em>et al.</em> 2011</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Tabata <em>et al.</em> 2013</a>). A number of other genomes of HCH-degrading <em>Sphingobium</em> strains isolated from India and Japan have been sequenced recently, although their genomic organization of <em>lin</em> genes is yet to be fully described (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kohli <em>et al.</em> 2013</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kumar Singh <em>et al.</em> 2013</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Mukherjee <em>et al.</em> 2013</a>). In addition to these isolated strains, the metagenomes of soil samples have been sequenced to assess the population dynamics of <em>lin</em> genes in contaminated sites. These studies revealed that the prevalence of upstream <em>lin</em> genes is correlated with the level of HCH contamination in the soil, suggesting selective pressure drives acquisition of the transmissible <em>lin</em> genes in HCH contaminated soil (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Sangwan <em>et al.</em> 2012</a>).</p><p id="p-8">Although most characterized strains have a full complement of <em>lin</em> genes, there is a growing body of evidence from newly sequenced strains that key components of the pathway frequently are missing (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Dogra <em>et al.</em> 2004</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Kohli <em>et al.</em> 2013</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Kumar Singh <em>et al.</em> 2013</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Mukherjee <em>et al.</em> 2013</a>). Given the apparent nature of pathway acquisition through a series of horizontal transfer events, these missing genes are likely to reflect early stages in the acquisition of the pathway, with some species yet to acquire some genes. With an incomplete pathway, these strains are unlikely to be able to completely mineralize HCH, instead converting it cometabolically. Many HCH-degrading strains have been poorly characterized in this regard, however, leaving open the possibility that there may be alternate steps in HCH degradation yet to be discovered. An additional limitation with the currently sequenced species is that, apart from <em>S. japonicum</em> UT26, the others have all been isolated from Indian soil samples, with many being isolated from the same field. Any insights gained from these strains might therefore be specific to the particular environmental conditions they encounter and restrict our ability to draw broader conclusions about the evolution of the <em>lin</em> pathway.</p><p id="p-9">Here, we present an analysis of three HCH-degrading strains (<em>Sphingobium czechense</em> LL01, <em>Novosphingobium barchaimii</em> LL02, and <em>Sphingobium baderi</em> LL03) isolated from soil samples at a Czech lindane-manufacturing site (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kaur <em>et al.</em> 2013</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Niharika <em>et al.</em> 2013a</a>,<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">b</a>). Strain LL01 has been reported to degrade α-, β-, γ-, and δ-HCH and LL03 to degrade only α-, γ-, and δ-HCH. Strain LL02 has been reported to degrade technical-HCH (a mix of HCH isomers); however, it is unknown which pure isomers it is capable of degrading. No information has yet been reported on how the HCH degradation in these strains compares with other, well-characterized, strains or on the relative rates of isomer degradation in each strain. Both of these features are known to be affected by the <em>lin</em> gene complement of a strain (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Sharma <em>et al.</em> 2011</a>; <a id="xref-ref-81-2" class="xref-bibr" href="#ref-81">Wu <em>et al.</em> 2007</a>)</p><p id="p-10">We first compared the degradation patterns and metabolites of α-, β-, γ-, and δ-HCH in each strain with the well-characterized strain <em>S. japonicum</em> UT26, which degrades all four isomers. We were able to confirm the previously published abilities of LL01 and LL03 and show that <em>N. barchaimii</em> LL02 degrades α-, γ-, and δ-HCH but with a shift in isomer preference, degrading δ-HCH much faster and γ-HCH much slower than the other strains. We then sequenced the genomes of the three Czech strains to compare them with the published genomes of UT26 (isolated in Japan) and six other strains isolated in India. The comparative genomic analysis presented here finds partial <em>lin</em> pathways in half of the studied strains, with variable organization of those genes that are present. It also highlights the key role of IS<em>6100</em> elements in <em>lin</em> gene transmission, showing a variety of insertion sites for IS<em>6100</em> elements around the <em>lin</em> genes, and the first association of IS<em>6100</em> with the <em>linGHIJ</em> cluster. Finally, we identify a LinA variant in <em>N. barchaimii</em> LL02 that is presumed to be responsible for the shift in isomer preference.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Strains and media</h3><p id="p-11">Bacterial strains used in this study were obtained from the DSMZ (<a href="http://www.dsmz.de">www.dsmz.de</a>) under the accession numbers DSM-16413 (<em>S. japonicum</em> UT26), DSM-25410 (<em>S. czechense</em> LL01), DSM-25411 (<em>N. barchaimii</em> LL02), and DSM-25433 (<em>S. baderi</em> LL03). Quarter-strength Luria Bertani medium (QSLB; 2.5 g of tryptone, 1.25 g of yeast extract, 2.5 g of NaCl, 20 g of sodium succinate, and 20 mL of glycerol per liter) was used for the growth of these strains at 30°.</p></div><div id="sec-3" class="subsection"><h3>Degradation assays</h3><p id="p-12">Seed cultures of each strain were prepared by inoculating glycerol stocks into QSLB and incubating at 30° overnight with shaking (180 rpm). The seed cultures were then centrifuged at 8000<em>g</em> for 10 min at 4° and the pellet resuspended in QSLB. After standardizing to an OD<sub>600</sub> of 0.3, the cells were added (1% v/v) into a 500-mL flask with 20 mL of QSLB containing either α-, β-, γ-, or δ-HCH at a concentration of 0.5 ppm. Uninoculated QSLB containing HCH was used as a negative control, and all treatments were tested in triplicate cultures.</p><p id="p-13">Samples of growth media from each flask (500 µL) were taken at 0, 0.5, 1, 4, 8, 24, 32, and 52 hr after inoculation. Because of the slow degradation of γ-HCH by strain LL02, an additional sample was taken at 96 hr after inoculation. Residual HCH and any metabolites were extracted with equal volumes of hexane and analyzed by gas chromatography using an Agilent 7890A gas chromatograph with an electron capture detector and a BPX-50 column (length: 30 m, internal diameter: 0.32 mm, film thickness: 0.25 µm). The temperature conditions for the analysis were held isothermally at 100° for 5 min, followed by an increase at a rate of 10° per minute to 200°, which was held for a further 5 min. Quantitation of HCH isomers was performed with standard curves produced from serial dilutions of the control flask media. A qualitative assessment of the metabolites produced by each strain was made by comparison to the retention time of 1,2,3-TCB, 1,2,4-TCB, and 1,3,5-TCB as authentic standards (Sigma-Aldrich) or to the pentachlorocyclohexane (PCCH) or pentachlorocyclohexanol metabolites produced with purified LinA and LinB respectively from the relevant HCH isomer (<a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Trantirek <em>et al.</em> 2001</a>).</p></div><div id="sec-4" class="subsection"><h3>Genome sequencing and analysis</h3><p id="p-14">Genomic DNA for <em>S. czechense</em> LL01, <em>N. barchaimii</em> LL02 and <em>S. baderi</em> LL03 was prepared using the QIAGEN Genomic Tip kit (QIAGEN) following the manufacturer’s protocol. DNA was sequenced on an Illumina HiSeq2000 by Macrogen (Seoul, Korea). Both a short (500 bp) and long (5 kb) insert library was prepared and 100-bp paired reads were obtained from each sample. The initial reads were quality filtered (minimum average quality = 3) and trimmed (minimum quality = 20) using PrinSeq (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Schmieder and Edwards 2011</a>) and then downsampled to 100−120× coverage based on predicted genome size. FastQC was used to inspect the reads for quality issues before proceeding with the assembly.</p><p id="p-15">Initial assemblies were constructed using the ABySS assembler (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Simpson <em>et al.</em> 2009</a>) with a range of k-mer lengths and the best assembly based on the contig N50 metric for each strain was selected for scaffolding using the matepair reads. Gaps in the final assembly were closed with GapFiller (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Nadalin <em>et al.</em> 2012</a>) and annotation of the final assemblies was performed using PGAAP [National Center for Biotechnology Information (NCBI)].</p><p id="p-16">Concurrent with this work, the genome of <em>S. baderi</em> LL03 has been independently sequenced using the 454 GS FLX Titanium technology. Comparison with the assembly produced in this study from Illumina sequencing indicates that the two have near identical sequences, with 98.2% of the 454 assembly aligning to the Illumina assembly with greater than 99.8% identity. All analysis described has been carried out on the Illumina assembly. The Whole Genome Shotgun sequences for <em>S. czechense</em> LL01, <em>N. barchaimii</em> LL02, and <em>S. baderi</em> LL03 have been deposited at DDBJ/EMBL/GenBank under the accession numbers JACT00000000, JACU00000000, and JACV00000000 respectively. The versions described in this paper are JACT01000000, JACU01000000, and JACV01000000.</p><p id="p-17">In addition to the aforementioned sequences, genome sequences for <em>S. japonicum</em> UT26 (GCF_000091125), <em>S. indicum</em> B90A (AJXQ01000000), <em>Sphingomonas</em> sp. MM-1 (GCF_000347675), <em>Sphingobium</em> sp. HDIP04 (ATDO01000000), <em>S. chinhatense</em> IP26 (AUDA01000000), <em>S. quisquiliarum</em> P25 (ATHO01000000), <em>S. ummariense</em> RL-3 (AUWY01000000), <em>Sphingobium</em> sp. SYK-6 (GCF_000283515), <em>Sphingomonas wittichii</em> RW-1 (GCF_000016765), and <em>Novosphingobium aromaticivorans</em> DSM12444 (GCF_000013325) and the <em>linB</em> containing plasmid pLB1 (NC_008330) were used in our analysis. Average nucleotide identities (ANIs) of all pairwise comparisons were calculated with JSpecies (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Richter and Rosselló-Móra 2009</a>). Whole-genome alignments were performed against UT26, LL01, LL02 and LL03 using MUMmer (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Delcher <em>et al.</em> 2002</a>) and regions of horizontal transfer in each target genome were predicted with the Genomic Island Suite of Tools (GIST), which combines the results of several common horizontal transfer prediction tools to produce a single, improved estimate (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hasan <em>et al.</em> 2012</a>). This information was visualized using Circos (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Krzywinski <em>et al.</em> 2009</a>).</p><p id="p-18">The <em>lin</em> genes of UT26 (<em>linA</em>; SJA_C1-18560, <em>linB</em>; SJA_C1-19590, <em>linC</em>; SJA_C1-00590, <em>linD</em>; SJA_P1-01390; <em>linE</em>; SJA_P1-01430, <em>linR</em>; SJA_P1-01440; <em>linF</em>; SJA_C2-04820, <em>linG</em>; SJA_C2-05210, <em>linH</em>; SJA_C2-05220, <em>linI</em>; SJA_C2-05200, <em>linJ</em>; SJA_C2-05190, <em>linK</em>; SJA_C1-00210, <em>linL</em>; SJA_C1-00220, <em>linM</em>; SJA_C1-00230, <em>linN</em>; SJA_C1-00240,) were used as queries for TBLASTX searches in the newly sequenced strains. The top hits of each search were then aligned to the non-redundant NCBI database to confirm their assignment. The genomic regions containing <em>lin</em> genes were aligned pairwise with MEGABLAST and displayed with genoPlotR (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Guy <em>et al.</em> 2010</a>) to examine the conserved regions between the studied sequences.</p></div></div><div class="section results" id="sec-5"><h2 class="">Results</h2><div id="sec-6" class="subsection"><h3>Degradation of HCH</h3><p id="p-19">Growth assays with four HCH isomers allowed us to compare the abilities of LL01, LL02, and LL03 to degrade the various HCH isomers with those of the well characterized strain UT26 (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). We were able to confirm the ability of LL01 to degrade all four isomers tested (relative rates: α = γ &gt; δ &gt;&gt; β), and of LL03 to degrade α-, γ-, and δ-HCH (α &gt; γ &gt;&gt; δ) (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Kaur <em>et al.</em> 2013</a>; <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Niharika <em>et al.</em> 2013a</a>). LL02 was found to degrade only α-, γ-, and δ-HCH (δ &gt; α &gt;&gt; γ). Compared with UT26, LL01 exhibited a similar degradation pattern for each isomer, whereas LL03 was much slower than UT26 for the three isomers it could degrade. LL02 had a novel degradation pattern, with slower γ-HCH and faster δ-HCH degradation than the other strains. Although LinA variants are known to affect the α-HCH enantiomer specificity or the α/γ isomer preference (<a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Sharma <em>et al.</em> 2011</a>; <a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Suar <em>et al.</em> 2005</a>), this is the first strain reported to degrade δ-HCH faster than all other isomers.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1081/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Degradation assays for HCH isomers. Degradation of &#x3B1;-HCH, &#x3B2;-HCH, &#x3B3;-HCH, and &#x3B4;-HCH (left-hand axis) by UT26, LL01, LL02, and LL03 and relative quantification of metabolites by peak area (right-hand axis). Time postinoculation is shown on the bottom axis and the timescale for the &#x3B3;-HCH assay has been adjusted to observe both the very fast (less than 1 hr) and very slow (96 hr) degradation of &#x3B3;-HCH by different strains. Values are the mean of three biological replicates, with standard deviations. Identity of metabolites was confirmed by comparison to authentic standards or to metabolites produced with purified UT26 LinA or LinB enzymes. HCH, hexachlorocyclohexane; PCCH, pentachlorocyclohexene; PCHL, pentachlorocyclohexanol; TCB, trichlorobenzene." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1945182906" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Degradation assays for HCH isomers. Degradation of &#x3B1;-HCH, &#x3B2;-HCH, &#x3B3;-HCH, and &#x3B4;-HCH (left-hand axis) by UT26, LL01, LL02, and LL03 and relative quantification of metabolites by peak area (right-hand axis). Time postinoculation is shown on the bottom axis and the timescale for the &#x3B3;-HCH assay has been adjusted to observe both the very fast (less than 1 hr) and very slow (96 hr) degradation of &#x3B3;-HCH by different strains. Values are the mean of three biological replicates, with standard deviations. Identity of metabolites was confirmed by comparison to authentic standards or to metabolites produced with purified UT26 LinA or LinB enzymes. HCH, hexachlorocyclohexane; PCCH, pentachlorocyclohexene; PCHL, pentachlorocyclohexanol; TCB, trichlorobenzene.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1081/F1.medium.gif" width="440" height="303"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/6/1081/F1.medium.gif" width="440" height="303"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1081/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1081/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466349" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-20" class="first-child">Degradation assays for HCH isomers. Degradation of α-HCH, β-HCH, γ-HCH, and δ-HCH (left-hand axis) by UT26, LL01, LL02, and LL03 and relative quantification of metabolites by peak area (right-hand axis). Time postinoculation is shown on the bottom axis and the timescale for the γ-HCH assay has been adjusted to observe both the very fast (less than 1 hr) and very slow (96 hr) degradation of γ-HCH by different strains. Values are the mean of three biological replicates, with standard deviations. Identity of metabolites was confirmed by comparison to authentic standards or to metabolites produced with purified UT26 LinA or LinB enzymes. HCH, hexachlorocyclohexane; PCCH, pentachlorocyclohexene; PCHL, pentachlorocyclohexanol; TCB, trichlorobenzene.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">Metabolites produced in the growth assays were identified by comparison of their retention time to known standards. Strain LL01 produced the same metabolite profile for each isomer as UT26, with degradation of β-HCH−producing pentachlorocyclohexanol and the remaining isomers producing pentachlorocyclohexane (PCCH), and trichlorobenzene (TCB). β-PCCH was not directly observed in the degradation of α-HCH by strain LL01 but is inferred from the observed TCB metabolite. As the bulk of the δ-HCH degradation occurred between the 8- and 24-hr samples, no metabolites for δ-HCH degradation were observed in UT26 and LL01.</p><p id="p-22">Strain LL03 produced the same metabolites as UT26 for α-HCH and γ-HCH but produced 1,2,4-trichlorobenzene (1,2,4-TCB) and 1,2,3-trichlorobenzene (1,2,3-TCB) when incubated with δ-HCH and could not degrade β-HCH at all. The δ-HCH metabolites produced by LL03 have been observed when δ-HCH is incubated with purified UT26 LinA enzyme but not in cell cultures of various strains, as δ-HCH is preferentially degraded by LinB (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Geueke <em>et al.</em> 2013</a>). These metabolites, combined with the lack of degradation of β-HCH (a LinB-initiated degradation), are consistent with the claim that LL03 lacks a <em>linB</em> gene, leading to LinA-initiated degradation pathways (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Kaur <em>et al.</em> 2013</a>).</p><p id="p-23">Strain LL02 is similar to LL03 in that degradation of α-, γ-, and δ-HCH all appears to be LinA-initiated, producing their respective PCCHs, and no degradation of β-HCH is observed. The largest difference between LL02 and LL03 comes in the further degradation of γ-PCCH, with LL03 (and UT26) accumulating 1,2,4-TCB as a dead end product, whereas LL02 accumulates 1,2,3-TCB, 1,2,4-TCB, and 1,3,5-TCB. This is the first report of the 1,2,3- and 1,3,5-TCB isomers being observed from γ-HCH degradation. It is unclear why different TCB isomers would be produced in this degradation, as 1,2,4-TCB is proposed to be spontaneously produced from tetrachlorocyclohexadiene, the putative final product of LinA activity (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Nagata <em>et al.</em> 1993b</a>). However, as 1,2,4-TCB is identified in degradation assays with purified LinA enzyme, the altered metabolite profile provides additional evidence for the presence of a novel LinA variant in strain LL02.</p></div><div id="sec-7" class="subsection"><h3>Genome sequences of Czech strains</h3><p id="p-24">For each strain, short and long insert libraries were prepared and sequenced using Illumina HiSequation 2000. After quality control, the short insert sequences were assembled using the ABySS assembler and the best assembly for each strain was selected for scaffolding and gap filling using the matepair reads. The final assemblies produced draft genomes comprising 21, 26, and 82 contigs for LL01, LL02, and LL03, respectively (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). These contigs were assembled into 18, 19, and 11 scaffolds, respectively. The LL01 assembly contains four chromosomal scaffolds (totaling 4.1 Mb in size) and a plasmid scaffold of 186 kb. The LL02 assembly contains 10 chromosomal scaffolds and a plasmid scaffold of 242 kb. In the LL03 assembly, 98% of the genome is contained in four circular scaffolds, comprising two chromosomal scaffolds (3.4 Mb and 641 kb) and two plasmid scaffolds (510 kb and 34 kb). The total genome size and GC% content for each assembly is consistent with previously sequenced genomes from the relevant genus (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Copley <em>et al.</em> 2012</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dong <em>et al.</em> 2014</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gan <em>et al.</em> 2012</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Luo <em>et al.</em> 2012</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Masai <em>et al.</em> 2012</a>). Annotation of each genome was performed using the PGAAP (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Angiuoli <em>et al.</em> 2008</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466298/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466298/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Assembly summary: summary of the final genome assemblies produced for this study</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3><em>lin</em> genes and IS<em>6100</em> elements in LL01, LL02, and LL03</h3><p id="p-25">We identified the <em>lin</em> genes present in LL01, LL02, and LL03 by homology to the protein sequences from UT26 using the top BLAST hits (E-value &lt; 1e-20) (<a href="http://www.g3journal.org/content/suppl/2015/04/07/g3.114.015933.DC1/TableS1.xlsx">Table S1</a>). Each potential hit was then used as a BLAST query against the NCBI nr database to confirm that its closest match was a <em>lin</em> gene. There is consistent representation of the conserved downstream elements of the pathway representing core sphingonomad functional systems, with each strain containing a <em>linKLMN</em> cluster encoding an ABC transporter and most components of the <em>linGHIJ</em> cluster encoding the β-ketoadipate pathway (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Nagata <em>et al.</em> 2010</a>). In strain LL01, however, the <em>linGHIJ</em> cluster is disrupted by an IS<em>6100</em> element, causing the transcriptional regulator <em>linI</em> to be interrupted and <em>linJ</em> (encoding an acetyl-CoA acetyltransferase) to be lost.</p><p id="p-26">Unlike the core functional genes, there is considerable variation in the presence of the remaining <em>lin</em> genes (<em>linA-F</em>) in the three strains. In LL01, we identified a complete upstream pathway, with <em>linA</em>, <em>linB</em> (two copies) and <em>linC</em> all present; however, <em>linRED</em> and <em>linF</em> are missing. In LL02 we only identified <em>linA</em> and an IS<em>6100</em>-truncated <em>linF</em>, whereas in LL03 we only identified <em>linA</em>, <em>linF</em>, and a disrupted <em>linR</em>. These variable genes (<em>linA-F</em>) comprise the HCH-specific component of the pathway, in contrast to the more general role performed by the more consistently represented core functional genes. The presence or absence of <em>linA</em> and <em>linB</em> in each of the three strains is sufficient in these cases to explain which HCH isomers each strain could degrade. Further sequencing of additional isolates would identify the extent of upstream <em>lin</em> gene variation and its association with HCH isomer degradation.</p><p id="p-27">We also assessed each genome for the prevalence of IS<em>6100</em> elements, given their known association with the <em>lin</em> genes (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Dogra <em>et al.</em> 2004</a>). We identified 14, 19, and 24 IS<em>6100</em> elements in LL01, LL02, and LL03, respectively, either confirmed through a match to the IS<em>6100</em> transposase (accession BAI98840) or inferred by a match to two 15 bp IS<em>6100</em> inverted repeats (GGCTCTGTTGCAAA) across contig boundaries (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Mahillon and Chandler 1998</a>). These numbers fall within the range seen among sequenced HCH-degrading strains (13−26) and are much greater than seen in the three non−HCH-degrading strains from the same genera used in our comparative analysis; <em>Novosphingobium aromaticivorans</em> DSM12444 (0), <em>Sphingomonas wittichii</em> RW-1 (1), and <em>Sphingobium</em> sp. SYK-6 (0) (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). IS<em>6100</em> is not a <em>lin</em>-gene−specific transposable element, having also been associated with catabolic genes for nylon (<em>Flavobacterium</em> sp. KI72, <em>Pseudomonas</em> sp. NK87), carbazole (<em>Sphingomonas</em> sp. XLDN2-5), and nonyl-phenols (<em>Sphingomonas</em> sp. NP5), among others, however, only small numbers of IS<em>6100</em> elements (2−5) have been identified in these cases (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gai <em>et al.</em> 2010</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kato <em>et al.</em> 1994</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Takeo <em>et al.</em> 2012</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Yasuhira <em>et al.</em> 2007</a>). The IS<em>6100</em> elements in LL01, LL02, and LL03 appear to be highly clustered around the <em>lin</em> genes, supporting the proposed involvement of IS<em>6100</em> in <em>lin</em> gene recruitment. In LL03 for example, the 19 IS6100 elements found on the 3.4 Mb main chromosome scaffold lie at a median distance of 25.5 kb from the nearest <em>lin</em> gene, with five found near <em>linGHIJ</em>, four near <em>linX</em> and <em>linA</em>, two near <em>linKLMN</em>, and seven near <em>linF</em> (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466289/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466289/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">IS<em>6100</em> elements in the studied genomes</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1081/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Whole-genome alignments of sequenced strains. Whole-genome alignments of the other 12 genomes (see the Materials and Methods section for names and accession numbers) to each of the UT26, LL01, LL02, and LL03 genomes. The color gradient of the inner rings indicates homology to the reference genome sequence. On display starting from the innermost ring are three non&#x2212;HCH-degrading strains, then 10 HCH-degrading strains, with the reference genome the outermost ring. Reference chromosomes or scaffolds predicted to be from chromosomes are colored blue, plasmids are colored red, and small scaffolds that cannot be unambiguously determined to be from either chromosomal or plasmid origin are colored yellow. The positions of IS6100 elements (purple) and lin genes (black) in the reference genome are indicated. Predicted genomic islands are highlighted in green. HCH, hexachlorocyclohexane." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1945182906" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Whole-genome alignments of sequenced strains. Whole-genome alignments of the other 12 genomes (see the &lt;em&gt;Materials and Methods&lt;/em&gt; section for names and accession numbers) to each of the UT26, LL01, LL02, and LL03 genomes. The color gradient of the inner rings indicates homology to the reference genome sequence. On display starting from the innermost ring are three non&#x2212;HCH-degrading strains, then 10 HCH-degrading strains, with the reference genome the outermost ring. Reference chromosomes or scaffolds predicted to be from chromosomes are colored blue, plasmids are colored red, and small scaffolds that cannot be unambiguously determined to be from either chromosomal or plasmid origin are colored yellow. The positions of IS&lt;em&gt;6100&lt;/em&gt; elements (purple) and &lt;em&gt;lin&lt;/em&gt; genes (black) in the reference genome are indicated. Predicted genomic islands are highlighted in green. HCH, hexachlorocyclohexane.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1081/F2.medium.gif" width="377" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/6/1081/F2.medium.gif" width="377" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1081/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1081/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466333" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-30" class="first-child">Whole-genome alignments of sequenced strains. Whole-genome alignments of the other 12 genomes (see the <em>Materials and Methods</em> section for names and accession numbers) to each of the UT26, LL01, LL02, and LL03 genomes. The color gradient of the inner rings indicates homology to the reference genome sequence. On display starting from the innermost ring are three non−HCH-degrading strains, then 10 HCH-degrading strains, with the reference genome the outermost ring. Reference chromosomes or scaffolds predicted to be from chromosomes are colored blue, plasmids are colored red, and small scaffolds that cannot be unambiguously determined to be from either chromosomal or plasmid origin are colored yellow. The positions of IS<em>6100</em> elements (purple) and <em>lin</em> genes (black) in the reference genome are indicated. Predicted genomic islands are highlighted in green. HCH, hexachlorocyclohexane.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Comparison of Czech strains to other HCH-degrading strains</h3><p id="p-31">Previous genomic comparisons between UT26 and B90A have suggested that these two strains may have been derived from a common ancestor containing most of the <em>lin</em> genes (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Sangwan <em>et al.</em> 2014</a>). However, a similar analysis between UT26 and three other HCH-degrading strains (<em>Sphingobium</em> sp. TKS, <em>Sphingobium</em> sp. MI1205 and <em>Sphingomonas</em> sp. MM-1) suggested the independent acquisition of HCH degradation in each of the four strains (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Nagata <em>et al.</em> 2014</a>). The expanding number of genomes available for HCH-degrading strains allows us to test the relative contribution of these two modes of evolution more broadly in <em>lin</em> pathway evolution.</p><p id="p-32">Whole-genome alignments of nine HCH-degrading strains (<em>S. czechense</em> LL01, <em>N. barchaimii</em> LL02, <em>S</em>. <em>baderi</em> LL03, <em>S. indicum</em> B90A, <em>S. chinhatense</em> IP26, <em>Sphingobium</em> sp. HDIP04, <em>S. quisquiliarum</em> P25, <em>S. ummariense</em> RL-3, and <em>Sphingomonas</em> sp. MM-1), and the three non−HCH- degrading strains mentioned previously, to UT26 reveal both modes of evolution noted previously (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Three of the other HCH degraders, B90A, IP26, and HDIP04, all have very high homology (ANI of 97.1–97.7%) to large regions of the UT26 genome, indicating a recent common ancestor from which they may have obtained most <em>lin</em> genes. However, the remaining six strains, including all three Czech strains, show smaller regions of homology, at lower levels of identity to UT26 (ANI 75.3–81.9%), indicating more divergence among the shared set of genes common to all sphingomonads. Compared with this low overall level of homology, the high identity (generally &gt;97%) of <em>lin</em> genes in these strains to UT26 indicates that they have likely been acquired horizontally, rather than through shared ancestry with UT26. The sequence composition of the genomic regions containing <em>lin</em> genes also suggests the acquisition by horizontal transfer; the GIST (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Hasan <em>et al.</em> 2012</a>) identified predicted genomic islands which contain all <em>lin</em> genes in UT26 except <em>linKLMN</em>.</p><p id="p-33">The aforementioned alignments were repeated with each of LL01, LL02, and LL03 as a reference genome to determine whether these strains acquired their <em>lin</em> genes independently of each other (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). Unlike what was observed in the comparisons of UT26, these three showed no close relationship to any of the other genomes studied (LL01 ANI: 73.6–79.9%, LL02 ANI: 73.0–75.1%, LL03 ANI: 75.7–81.8%). Where small areas of strong homology exist between the strains, they are associated with the <em>lin</em> genes, IS<em>6100</em> elements or genomic islands predicted by GIST. This suggests that the distinct <em>lin</em> gene complements of each of these strains may reflect independent horizontal gene transfer events. This whole-genome analysis is consistent with phylogenies based on 16S sequences that place LL01, LL02, and LL03 relatively distant from other HCH-degrading strains (<a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Kaur <em>et al.</em> 2013</a>; <a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Niharika <em>et al.</em> 2013a</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Niharika <em>et al.</em> 2013b</a>).</p><p id="p-34">As indicated previously, there is considerable variation in <em>lin</em> gene complement among the 10 HCH-degrading genomes considered here (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The four very closely related strains (UT26, B90A, IP26, and HDIP04) each have a complete <em>lin</em> pathway, suggesting that it was acquired from their common ancestor. Of the remaining strains, only MM-1 has a complete pathway, whereas LL01, LL02, LL03, RL-3, and P25 have missing components. The selective pressure to acquire a <em>lin</em> pathway in HCH-contaminated soil suggests that these strains with partial <em>lin</em> pathways are likely in the early stages of pathway assembly, having acquired only a few of the required genes, but we cannot discount the possibility that some represent instances of gene loss (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Sangwan <em>et al.</em> 2012</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1081/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="lin pathway composition of studied strains. Presence of lin genes in the 10 HCH-degrading strains studied, along with the percent identity of each encoded protein to the equivalent in UT26. Genes grouped in operons are indicated with braces to the left of the gene names. Multiple values indicate duplicate copies of the gene in a strain. The neighbor-joining phylogeny above the strain names was produced from a clustalw alignment of the 16S rDNA sequences (UT26: AF039168, B90A: AY519129, IP26: EF190507, HDIP04: EF424393, LL01: JN646865, P25: EU781657, LL03: JN695620, RL-3: EF207155, MM-1: CP004036;G432_r19183, LL02: JN695619) of each strain with E. coli (ECK3843) as an outgroup (not shown). HCH, hexachlorocyclohexane." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1945182906" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;lin&lt;/em&gt; pathway composition of studied strains. Presence of &lt;em&gt;lin&lt;/em&gt; genes in the 10 HCH-degrading strains studied, along with the percent identity of each encoded protein to the equivalent in UT26. Genes grouped in operons are indicated with braces to the left of the gene names. Multiple values indicate duplicate copies of the gene in a strain. The neighbor-joining phylogeny above the strain names was produced from a clustalw alignment of the 16S rDNA sequences (UT26: AF039168, B90A: AY519129, IP26: EF190507, HDIP04: EF424393, LL01: JN646865, P25: EU781657, LL03: JN695620, RL-3: EF207155, MM-1: CP004036;G432_r19183, LL02: JN695619) of each strain with &lt;em&gt;E. coli&lt;/em&gt; (ECK3843) as an outgroup (not shown). HCH, hexachlorocyclohexane.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1081/F3.medium.gif" width="373" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/6/1081/F3.medium.gif" width="373" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1081/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1081/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466279" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-35" class="first-child"><em>lin</em> pathway composition of studied strains. Presence of <em>lin</em> genes in the 10 HCH-degrading strains studied, along with the percent identity of each encoded protein to the equivalent in UT26. Genes grouped in operons are indicated with braces to the left of the gene names. Multiple values indicate duplicate copies of the gene in a strain. The neighbor-joining phylogeny above the strain names was produced from a clustalw alignment of the 16S rDNA sequences (UT26: AF039168, B90A: AY519129, IP26: EF190507, HDIP04: EF424393, LL01: JN646865, P25: EU781657, LL03: JN695620, RL-3: EF207155, MM-1: CP004036;G432_r19183, LL02: JN695619) of each strain with <em>E. coli</em> (ECK3843) as an outgroup (not shown). HCH, hexachlorocyclohexane.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">There are also instances of multiple copies of the <em>linA-F</em> genes, with two copies of <em>linA</em> in B90A, two of <em>linB</em> in LL01, two of <em>linC</em> in MM1, two of <em>linRED</em> in RL-3, two of <em>linF</em> in B90A and IP26, and three of <em>linF</em> in MM1. In the case of <em>linF</em>, however, all extra copies are disrupted by IS<em>6100</em> elements, and it is unknown if they are functional. Multiple copies of the upstream genes may relax selective constraints, allowing functional specialization of the duplicated copies as seen in the enantioselective degradation of α-HCH by LinA1/LinA2 in B90A (<a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">Suar <em>et al.</em> 2005</a>), or it may increase the expression of the gene, with <em>linA</em>, <em>linB</em>, and <em>linC</em> all being expressed constitutively (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Suar <em>et al.</em> 2004</a>). Little variation is seen in protein sequences for these genes however, with almost all copies observed having very high identities to the UT26 sequences (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). This paucity of variation is observed in all <em>lin</em> genes except <em>linKLMN</em>, further supporting the prediction that these genes, including core downstream components such as the β-ketoadipate pathway encoding <em>linGHIJ</em>, have been acquired by horizontal transfer. The greater variation among <em>linKLMN</em> sequences suggests that this cluster has not undergone horizontal transfer, but has been acquired from a common ancestor.</p></div><div id="sec-10" class="subsection"><h3>Genomic organization of <em>lin</em> genes in HCH-degrading strains</h3><p id="p-37">We looked at microsynteny at the <em>lin</em> gene loci to investigate the extent of the horizontally transferred regions and their ongoing rearrangement (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Alignments of genomic regions from the above ten genome sequences plus a <em>linB</em> containing plasmid isolated exogenously from HCH-contaminated soil (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Miyazaki <em>et al.</em> 2006</a>) revealed high levels of conservation around the <em>lin</em> genes, generally delimited by IS<em>6100</em> elements. These regions range from many kilobases in length in the case of the downstream <em>lin</em> genes to just a few hundred base pairs for some of the upstream <em>lin</em> genes. Because longer horizontally transferred sequences are more likely to have a higher fitness cost, the size of these conserved regions may be related to the extent of horizontal transfer of each region. The high level of conservation of gene content and intergenic sequence around all <em>lin</em> genes except <em>linKLMN</em> (<a href="http://www.g3journal.org/content/suppl/2015/04/07/g3.114.015933.DC1/015933SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/04/07/g3.114.015933.DC1/FigureS2.pdf">Figure S2</a>) suggests that each has been derived from a single original source.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1081/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic organization of upstream lin genes. Alignments detailing strong conservation and IS6100 involvement immediately surrounding the upstream lin genes in HCH-degrading strains. Alignments of the regions surrounding (A) linA and (B) linB and linC were performed with MEGABLAST and ordered to give maximal pairwise alignment lengths. Conserved regions are indicated by red shading where the matches are in the same orientation and by blue shading where the matches are in reverse orientation. Also found associated with these genes are an IS6100 truncated linF and linX/linX2 (linC-like sequences not necessary for HCH degradation). HCH, hexachlorocyclohexane." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1945182906" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genomic organization of upstream &lt;em&gt;lin&lt;/em&gt; genes. Alignments detailing strong conservation and IS&lt;em&gt;6100&lt;/em&gt; involvement immediately surrounding the upstream &lt;em&gt;lin&lt;/em&gt; genes in HCH-degrading strains. Alignments of the regions surrounding (A) &lt;em&gt;linA&lt;/em&gt; and (B) &lt;em&gt;linB&lt;/em&gt; and &lt;em&gt;linC&lt;/em&gt; were performed with MEGABLAST and ordered to give maximal pairwise alignment lengths. Conserved regions are indicated by red shading where the matches are in the same orientation and by blue shading where the matches are in reverse orientation. Also found associated with these genes are an IS&lt;em&gt;6100&lt;/em&gt; truncated &lt;em&gt;linF&lt;/em&gt; and &lt;em&gt;linX/linX2&lt;/em&gt; (&lt;em&gt;linC&lt;/em&gt;-like sequences not necessary for HCH degradation). HCH, hexachlorocyclohexane.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1081/F4.medium.gif" width="440" height="402"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/6/1081/F4.medium.gif" width="440" height="402"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1081/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1081/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466193" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-38" class="first-child">Genomic organization of upstream <em>lin</em> genes. Alignments detailing strong conservation and IS<em>6100</em> involvement immediately surrounding the upstream <em>lin</em> genes in HCH-degrading strains. Alignments of the regions surrounding (A) <em>linA</em> and (B) <em>linB</em> and <em>linC</em> were performed with MEGABLAST and ordered to give maximal pairwise alignment lengths. Conserved regions are indicated by red shading where the matches are in the same orientation and by blue shading where the matches are in reverse orientation. Also found associated with these genes are an IS<em>6100</em> truncated <em>linF</em> and <em>linX/linX2</em> (<em>linC</em>-like sequences not necessary for HCH degradation). HCH, hexachlorocyclohexane.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1081/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic organization of downstream lin genes. Alignments detailing conservation and rearrangements in the genomic regions of the downstream lin genes. Alignments of the regions surrounding (A) linDER and (B) linFEb and linGHIJ, were performed with MEGABLAST and ordered to give maximal pairwise alignment lengths. Conserved regions are indicated by red shading where the matches are in the same orientation and by blue shading where the matches are in reverse orientation. Note that association of IS6100 elements with linGHIJ is unique to strains LL01, LL02 and LL03." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1945182906" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genomic organization of downstream &lt;em&gt;lin&lt;/em&gt; genes. Alignments detailing conservation and rearrangements in the genomic regions of the downstream &lt;em&gt;lin&lt;/em&gt; genes. Alignments of the regions surrounding (A) &lt;em&gt;linDER&lt;/em&gt; and (B) &lt;em&gt;linFEb&lt;/em&gt; and &lt;em&gt;linGHIJ&lt;/em&gt;, were performed with MEGABLAST and ordered to give maximal pairwise alignment lengths. Conserved regions are indicated by red shading where the matches are in the same orientation and by blue shading where the matches are in reverse orientation. Note that association of IS&lt;em&gt;6100&lt;/em&gt; elements with &lt;em&gt;linGHIJ&lt;/em&gt; is unique to strains LL01, LL02 and LL03.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1081/F5.medium.gif" width="440" height="383"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/6/1081/F5.medium.gif" width="440" height="383"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1081/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1081/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466216" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-39" class="first-child">Genomic organization of downstream <em>lin</em> genes. Alignments detailing conservation and rearrangements in the genomic regions of the downstream <em>lin</em> genes. Alignments of the regions surrounding (A) <em>linDER</em> and (B) <em>linFEb</em> and <em>linGHIJ</em>, were performed with MEGABLAST and ordered to give maximal pairwise alignment lengths. Conserved regions are indicated by red shading where the matches are in the same orientation and by blue shading where the matches are in reverse orientation. Note that association of IS<em>6100</em> elements with <em>linGHIJ</em> is unique to strains LL01, LL02 and LL03.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">One major difference between strains is the location of IS<em>6100</em> insertions around certain <em>lin</em> gene regions, in particular <em>linA</em> and <em>linRED</em>. In the case of <em>linA</em>, we identified five different insertion sites downstream of <em>linA</em>. The UT26 sequence has an insertion 3638 bp downstream of <em>linA</em>, while there are also insertions at 1550 bp (LL01, LL02, LL03, RL-3, MM1, B90A-A2, and P25), 1254 bp (IP26), and 1197 bp (HDIP04) downstream of <em>linA</em>, as well as the B90A<sub>LinA1</sub> insertion 444 bp into the <em>linA</em> gene (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>). Of the eight <em>linRED</em> clusters identified, five have an IS<em>6100</em> element upstream of the cluster (UT26, IP26, HDIP04, and two clusters in RL-3). Each of these IS elements is located in a different position (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>). These examples show the continuing activity of IS<em>6100</em> in HCH-degrading strains and their potential for ongoing involvement in the transfer and evolution of the <em>lin</em> pathway.</p><p id="p-41">Not all <em>lin</em> genes show such variability, however. The location of IS<em>6100</em> around <em>linB</em> is quite consistent in all strains, with insertion sites 307 bp upstream and 105 bp downstream of the gene (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a>). The only exceptions to this are found in IP26, where the insertion site is 387 bp upstream, and in UT26, which lacks any flanking IS elements around <em>linB</em>. As will be discussed below, this lack of variability in IS<em>6100</em> arrangement around <em>linB</em> may be because it is a recent acquisition that has been selectively favored to increase <em>linB</em> transmissibility.</p></div><div id="sec-11" class="subsection"><h3><em>N. barchaimii</em> LL02 contains an uncharacterized LinA variant</h3><p id="p-42">Closer analysis of the <em>linA</em> gene sequence in <em>N. barchaimii</em> revealed that it encodes an uncharacterized LinA enzyme (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). Compared with the two B90A variants LinA1 (accession number AAN64239) and LinA2 (ACM48253), the LinA<sub>LL02</sub> contains 11 and 17 amino acid differences respectively, as well as a seven amino acid deletion at the C-terminal end. Many of the mutations in LinA<sub>LL02</sub> are clustered near the catalytic histidine (H73) and affect amino acids predicted to form the active site (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Okai <em>et al.</em> 2010</a>). In other characterized LinA enzymes, small changes to the amino acid sequence are capable of altering the α-HCH enantiomer preference or the α-/γ-HCH isomer preference. The mutations in LinA<sub>LL02</sub>, therefore, provide candidates for the shift in isomer preference that sees strain LL02 preferentially degrading δ-HCH. Further work characterizing LinA<sub>LL02</sub> would confirm this hypothesis and identify which mutations are necessary for the altered phenotype. A sequence identical to LinA<sub>LL02</sub> has also been recently identified in a metagenomic screen of Indian soil samples, indicating that it is not a geographically restricted variant (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lal <em>et al.</em> 2013</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1081/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of LinA sequence variants. (A) Amino acid differences between major LinA variants and the uncharacterized LinA of LL02. (B) Proposed mechanism for the IS6100-mediated 21-bp deletion in LL02. The initial insertion site of the IS6100 cointegrates (yellow) in a LinAUT26-like sequence is marked with a black triangle. The insertion induces the duplication of 8bp at the insertion site (DR1 and DR1&#xB4; in blue). IS6100 elements are capable of reversion through homologous recombination between the induced direct repeats (DR1&#xD7;DR1&#xB4;). A near-identical sequence (DR2) is slightly further downstream of the insertion, however, and recombination between DR1 and DR2 gives rise to the sequence observed in LL02, with the deletion of one DR and the intervening sequence (colored red). Also shown is the sequence of LinA1 in B90A, containing the IS6100 insertion and a different downstream sequence. This has likely arisen through homologous recombination between IS6100 elements, deleting the intervening region containing the original C-terminus of LinA." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1945182906" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of LinA sequence variants. (A) Amino acid differences between major LinA variants and the uncharacterized LinA of LL02. (B) Proposed mechanism for the IS6100-mediated 21-bp deletion in LL02. The initial insertion site of the IS&lt;em&gt;6100&lt;/em&gt; cointegrates (yellow) in a LinA&lt;sub&gt;UT26&lt;/sub&gt;-like sequence is marked with a black triangle. The insertion induces the duplication of 8bp at the insertion site (DR1 and DR1&#xB4; in blue). IS&lt;em&gt;6100&lt;/em&gt; elements are capable of reversion through homologous recombination between the induced direct repeats (DR1&#xD7;DR1&#xB4;). A near-identical sequence (DR2) is slightly further downstream of the insertion, however, and recombination between DR1 and DR2 gives rise to the sequence observed in LL02, with the deletion of one DR and the intervening sequence (colored red). Also shown is the sequence of LinA1 in B90A, containing the IS&lt;em&gt;6100&lt;/em&gt; insertion and a different downstream sequence. This has likely arisen through homologous recombination between IS&lt;em&gt;6100&lt;/em&gt; elements, deleting the intervening region containing the original C-terminus of LinA.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1081/F6.medium.gif" width="440" height="342"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/5/6/1081/F6.medium.gif" width="440" height="342"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1081/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1081/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466248" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-43" class="first-child">Comparison of LinA sequence variants. (A) Amino acid differences between major LinA variants and the uncharacterized LinA of LL02. (B) Proposed mechanism for the IS6100-mediated 21-bp deletion in LL02. The initial insertion site of the IS<em>6100</em> cointegrates (yellow) in a LinA<sub>UT26</sub>-like sequence is marked with a black triangle. The insertion induces the duplication of 8bp at the insertion site (DR1 and DR1´ in blue). IS<em>6100</em> elements are capable of reversion through homologous recombination between the induced direct repeats (DR1×DR1´). A near-identical sequence (DR2) is slightly further downstream of the insertion, however, and recombination between DR1 and DR2 gives rise to the sequence observed in LL02, with the deletion of one DR and the intervening sequence (colored red). Also shown is the sequence of LinA1 in B90A, containing the IS<em>6100</em> insertion and a different downstream sequence. This has likely arisen through homologous recombination between IS<em>6100</em> elements, deleting the intervening region containing the original C-terminus of LinA.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><div id="sec-13" class="subsection"><h3>Recruitment of <em>lin</em> genes in sequenced genomes</h3><p id="p-44">Until recently, almost all characterized HCH-degrading strains contained a full complement of <em>lin</em> genes [<em>Sphingomonas</em> sp. γ12-7, the only exception, being deficient in <em>linRED</em> (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Böltner <em>et al.</em> 2005</a>)]. However, emerging evidence suggests that there is considerable variability in the recruitment of the pathway, with eight of the ten genomes studied here either missing or containing multiple copies of the <em>linA-F</em> genes. Of these variable genes, only <em>linA</em> is present in all strains, suggesting that it is necessary for a strain to be isolated in an enrichment screen for HCH degradation. <em>linA</em> also appears to be sufficient for isolation in an enrichment screen, as it is the only upstream <em>lin</em> gene found in <em>S. quisquiliarum</em> P25. The remaining genes (<em>linB-F</em>) are each absent in several strains and, given the horizontal transfer involved in pathway acquisition, likely represent early stages of pathway evolution, having acquired some <em>lin</em> genes but not all. These strains may therefore provide a valuable resource for studying the gradual accumulation of the <em>lin</em> genes and what function individual elements may serve in the absence of a full pathway.</p><p id="p-45">We cannot be certain that these strains are yet to acquire some <em>lin</em> genes, however, because they are known to be highly mobile as the result of IS<em>6100</em>-mediated genome rearrangements (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Nagata <em>et al.</em> 2011</a>). Laboratory studies have shown that <em>lin</em> genes are lost rapidly when the strain is no longer exposed to HCH, with <em>Pseudomonas aeruginosa</em> ITRC-5 losing <em>linA-E</em> when grown in rich media (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Singh <em>et al.</em> 2007</a>). Some of the cases of missing <em>lin</em> genes may therefore be due to gene loss events, rather than the strain not yet acquiring the gene. This is the most likely explanation for some genes in LL03, as an IS<em>6100</em>-interrupted <em>linR</em> suggests that an IS<em>6100</em>-mediated rearrangement is the cause of <em>linED</em> missing from the genome. The situation in the Czech strains is further complicated because preliminary data suggested that they were able to grow on HCH as a sole carbon source (<a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">Kaur <em>et al.</em> 2013</a>; <a id="xref-ref-52-4" class="xref-bibr" href="#ref-52">Niharika <em>et al.</em> 2013a</a>,<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">b</a>); however, we have been unable to replicate this phenotype. This finding suggests that these strains may have undergone secondary loss of some <em>lin</em> genes, during either preservation at the stock center or in the process of resuscitation, making it impossible to determine the original <em>lin</em> complement of these lines. The analysis of isolated cultures may therefore provide an incomplete picture of <em>lin</em> pathway evolution and need to be interpreted in light of other approaches, such as metagenome sequencing, that look at pathway organization in soil populations.</p><p id="p-46">Where instances of missing <em>lin</em> genes can be confirmed in natural populations, there are two main proposals for how these strains are able to degrade HCH. It is possible that the functions performed by missing <em>lin</em> genes may be carried out by other unknown enzymes in the organism. As an alternative, the complementing functions may be carried out by other members of the microbial community. Both of these hypotheses have been supported in different strains. <em>Xanthomonas</em> sp. ICH12 provides an example of the former, with the identification of a novel metabolite after PCCH production (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Manickam <em>et al.</em> 2007</a>). The latter hypothesis is supported by studies into microbial consortia isolated from HCH-contaminated Indian soil samples, in which the community was able to degrade HCH far more effectively than any of the individually isolated elements (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Elcey and Kunhi 2010</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Manonmani <em>et al.</em> 2000</a>).</p><p id="p-47">Also informative in respect of the <em>lin</em> pathway’s origins are the genomic regions immediately surrounding the <em>lin</em> genes. In almost all cases, the genomic context of each <em>lin</em> gene (including flanking genes and intergenic regions) is highly conserved (<em>e.g.</em>, 100% match for the 1 kb downstream of <em>linA</em>). The conserved region generally extends out to a mobile genetic element, usually an IS<em>6100</em> element, and then abruptly ceases. Together with the whole-genome analysis suggesting multiple, independent acquisitions of the <em>lin</em> genes, these data suggest that each <em>lin</em> gene has a single evolutionary origin and then is rapidly distributed across species through horizontal transfer events.</p><p id="p-48"><em>linB</em> is an interesting case because there appear to be two potential mechanisms of transfer. In UT26, <em>linB</em> is situated chromosomally in a GIST-predicted genomic island (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). Closer inspection of the predicted region identified it as a putative 73-kb integrative and conjugative element (ICE). ICEs (also known as conjugative transposons) are widely dispersed, horizontally transferred elements that encode their own conjugative transfer and integration/excision from a host genome (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Guglielmini <em>et al.</em> 2011</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Wozniak and Waldor 2010</a>). This 73-kb region on the UT26 first chromosome contains the integrase, relaxase, and recombination directionality factor required for excision and integration of the region (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Ramsay <em>et al.</em> 2006</a>), as well as a type IV secretory system that conjugatively transfers the excised element between cells. These elements are known to carry accessory genes responsible for a range of functions including pathogenicity (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chuzeville <em>et al.</em> 2012</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Seth-Smith <em>et al.</em> 2012</a>), symbiosis (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Sullivan <em>et al.</em> 1995</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Sullivan <em>et al.</em> 2002</a>), antibiotic resistance (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Michael <em>et al.</em> 2012</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Rodríguez-Blanco <em>et al.</em> 2012</a>), and xenobiotic catabolism (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gaillard <em>et al.</em> 2006</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Shettigar <em>et al.</em> 2012</a>). In all other sequenced genomes containing a <em>linB</em>, however, the gene is flanked by two IS<em>6100</em> elements, with intergenic sequence 307 bp upstream and 105 bp downstream identical to the UT26 sequence (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4B</a>). As IS<em>6100</em> elements only very rarely excise themselves without leaving a copy of the element, it is highly likely that the IS<em>6100-linB</em>-IS<em>6100</em> element is derived from the <em>linB</em> containing ICE seen in UT26. Thus, Although <em>linB</em> may initially have been acquired by conjugation and integration of an ICE, it appears that the acquisition of flanking IS<em>6100</em> elements has subsequently been key to the widespread prevalence of <em>linB</em> in isolated strains. With high HCH concentrations appearing to impart a selective pressure for acquisition of the <em>lin</em> genes (<a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Sangwan <em>et al.</em> 2012</a>), an increase in IS<em>6100</em> activity could provide a possible mechanism.</p></div><div id="sec-14" class="subsection"><h3>IS<em>6100</em> involvement in <em>lin</em> gene acquisition</h3><p id="p-49">The involvement of IS<em>6100</em> in <em>lin</em> gene acquisition is still clearly an active process, with a variety of IS<em>6100</em> element insertion sites identified. That these elements tended to define the limits of conserved regions around the <em>lin</em> genes is further evidence for the importance of IS<em>6100</em> in their horizontal transfer, as noted above. In addition to the widespread association of IS<em>6100</em> with the upstream <em>lin</em> genes, the new genomes sequenced for this study reveal the association of IS<em>6100</em> elements with the genes of the β-ketoadipate pathway (<em>linGHIJ</em>). This is the first time that IS<em>6100</em> elements have been found to be associated with these downstream components of the HCH degradation pathway. The fact that this arrangement is found in all three Czech strains suggests that, like the aforementioned <em>linB</em>, the acquisition of flanking IS<em>6100</em> elements may be the key to widespread horizontal transfer.</p><p id="p-50">The clustered arrangement of IS<em>6100</em> with so many of the <em>lin</em> genes adds further evidence for a selectively favored increase in IS<em>6100</em> activity and <em>lin</em> gene transmissibility. In HCH-contaminated soil, an increase in IS<em>6100</em> activity would introduce a number of new elements in the genome. Any new IS<em>6100</em> elements inserted near <em>lin</em> genes will be favored in due course as they increase the chance of <em>lin</em> genes being passed on to other community members. A pressure to acquire the upstream <em>lin</em> genes in HCH contaminated soil is therefore likely to explain the newly identified association with the downstream genes.</p><p id="p-51">The increasing involvement of IS<em>6100</em> with the <em>lin</em> genes may indicate the beginning of a second phase in the evolution of the pathway (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). The first phase of evolution, the initial formation of the pathway, is proposed to have occurred with the recruitment of the upstream genes (through various means) into an ancestral strain containing the downstream genes that make up core sphingomonad functions (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Nagata <em>et al.</em> 2014</a>). The current phase appears to involve a consolidation of the pathway, in which we see an increase in IS<em>6100</em> activity that expands IS<em>6100</em> association to genes that were initially transferred through other means (<em>e.g.</em>, <em>linB</em>) and to downstream genes that were not previously known to be transferrable (<em>linGHIJ</em>). Although this appears to introduce more instability in the pathway than previously seen, it also increases the probability of the pathway being acquired after exposure to HCH. Indeed, the entire catalytic component of the <em>lin</em> pathway is now capable of IS<em>6100</em>-mediated transfer, opening up the possibility of transfer to species that lack the downstream components common to all sphingomonads.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1081/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Hypothesized model of the lin pathway evolution. Proposed model of the processes involved in lin pathway evolution. Note that these steps are likely to be overlapping to some degree." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1945182906" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Hypothesized model of the &lt;em&gt;lin&lt;/em&gt; pathway evolution. Proposed model of the processes involved in &lt;em&gt;lin&lt;/em&gt; pathway evolution. Note that these steps are likely to be overlapping to some degree.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1081/F7.medium.gif" width="440" height="222"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/5/6/1081/F7.medium.gif" width="440" height="222"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1081/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1081/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466252" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-52" class="first-child">Hypothesized model of the <em>lin</em> pathway evolution. Proposed model of the processes involved in <em>lin</em> pathway evolution. Note that these steps are likely to be overlapping to some degree.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>IS6100-mediated C-terminal truncation of LinA<sub>LL02</sub></h3><p id="p-53">In addition to influencing the pathway organization, transposition and recombination of IS6100 elements can affect the sequence of Lin enzymes. In the case of LinA, an IS6100 insertion is seen at nucleotide position 444 of LinA1 found in B90A, which alters the C-terminal sequence of the protein. In LinA<sub>LL02</sub>, the seven amino acid C-terminal truncation is likely caused by imperfect excision of this IS6100 element (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>). Insertion of an IS6100 cointegrate is accompanied by an 8-bp direct repeat duplication of the insertion site (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Herron <em>et al.</em> 1999</a>). Homologous recombination can occur at this direct repeat, allowing reversion of the IS element insertion, although this occurs at very low rates (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Weaden and Dyson 1998</a>). Because this insertion in LinA occurs in a repeated GIHFAP motif, there is a nearly identical copy of the landing site direct repeat just downstream of the insertion (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a>). An imperfect excision of the IS6100 element at this secondary direct repeat would give rise to the sequence observed in the genome of LL02.</p><p id="p-54">The C-terminal extension is involved in stabilizing the LinA trimer and in closing off the active site once the substrate is bound (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Macwan <em>et al.</em> 2013</a>). The length of this extension is important for enzyme function, with two GIHFAP motifs producing enzymes with greater activity than those with one or three (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Macwan <em>et al.</em> 2013</a>). In the case of LinA<sub>LL02</sub>, the predicted loss of activity due to the single repeat may be compensated for by the C132R substitution. This substitution was identified by an <em>in vitro</em> mutagenesis assay as being responsible for a sixfold increase in LinA expression and solubility in <em>Escherichia coli</em> cells (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Mencía <em>et al.</em> 2006</a>). LinA<sub>LL02</sub> is the first natural variant to be identified with this mutation.</p></div><div id="sec-16" class="subsection"><h3>Continuing pathway evolution</h3><p id="p-55">Many different models have been proposed for the evolution of new metabolic pathways (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fondi <em>et al.</em> 2009</a>). The <em>lin</em> pathway provides an excellent model to study the early evolution of new pathways, given how recently it has arisen. Our analysis of 10 sequenced genomes suggests that the <em>lin</em> pathway is following the piecewise assembly model in which components of the pathway are pieced together from preexisting genes or operons (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Fani <em>et al.</em> 2005</a>). This approach is very similar to that proposed for the pentachlorophenol (PCP) degradation pathway of <em>S. chlorophenolicum</em> L1, which shares some downstream components of the <em>lin</em> pathway. <em>S. chlorophenolicum</em> L1 was found to share the downstream pathway (homologs of the <em>linF</em> and <em>linEb</em> genes) with a common ancestor and to then have acquired three upstream genes responsible for the specific degradation of PCP through two separate horizontal transfer events (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Copley <em>et al.</em> 2012</a>). One of the transfer events brought in two catabolic genes in a regulated operon (similar to the case of <em>linRED</em>) and one brought in a single catabolic gene (similar to the proposed acquisition of <em>linA</em>, <em>linB</em>, and <em>linC</em>). In the case of the PCP pathway, however, the lack of a known mechanism for the horizontal transfer precludes a discussion of the future evolutionary trajectory of the system.</p><p id="p-56">In the case of the <em>lin</em> system, however, we can identify some of the mechanisms involved and their continuing effects. Various methods of horizontal transfer have been responsible for the initial recruitment of the upstream <em>lin</em> genes to strains already possessing a downstream component (<a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Nagata <em>et al.</em> 2014</a>) and this work implicates an ICE in the acquisition of <em>linB</em>. An initial association of IS<em>6100</em> with some of the <em>lin</em> genes has spread such that the entire pathway (at least for the strains isolated from the Czech Republic) is now associated with IS<em>6100</em> elements. The <em>lin</em> pathway therefore provides valuable insight into the intermediate steps of IS<em>6100</em> activity, which has not been possible in previous investigations of IS<em>6100</em>-associated catabolic pathways. For example, in both the carbazole degradation pathway of <em>Sphingomonas</em> sp. XLDN2-5 and the 2-chloronitrobenzene degradation pathway of <em>Pseudomonas stutzeri</em> ZWLR2-1, the entire degradation pathway is organized into several large clusters flanked very closely by IS<em>6100</em> elements (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Gai <em>et al.</em> 2010</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Liu <em>et al.</em> 2011</a>). Although the origins of these clusters have been identified, no instances of intermediate pathway assembly have been observed. Here, we show the dynamic nature of IS<em>6100</em>-mediated pathway assembly, with many different insertion sites around the <em>lin</em> genes. This suggests that the <em>lin</em> pathway is still in the process of evolving and that IS<em>6100</em> is playing an active role.</p><p id="p-57">IS<em>6100</em> activity also may be resulting in a clustering of the <em>lin</em> genes. Homologous recombination between directly repeated IS<em>6100</em> elements can cause the deletion of the intervening genomic sequence (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Mahillon and Chandler 1998</a>). Although this is known to be responsible for deletions of <em>lin</em> genes (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Nagata <em>et al.</em> 2011</a>) it could also be responsible for bringing together upstream components of the pathway. In B90A we find a <em>linA1-linF-linC</em> cluster, and the <em>linF-linC</em> component also is found in IP26 and MM1-pISP1. In LL01 <em>linB</em> and <em>linC</em> are colocated, whereas HDIP04 and MM1-pISP4 each contain a <em>linB-linC-linF</em> cluster. This clustering is predicted by the Selfish Operon Theory, which posits that operon formation can be driven by selective pressure to reduce the number of horizontal transfer events required to acquire the pathway (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lawrence and Roth 1996</a>). Although the contribution of this process to the formation of operons is disputed (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Pál and Hurst 2004</a>), the proposal that clustering of genes in the same pathway can result from horizontal transfer is supported by our observations of the <em>lin</em> pathway. Under this model, we expect to see additional clustering of the <em>lin</em> genes as strains continue to be exposed to HCH. Further sequencing of recently isolated strains will help determine if this predicted evolutionary trajectory is accurate.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-58">This research was funded in part by CSIRO's Land and Water Flagship. SP was supported by Australian Postgraduate Award (APA).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-3"><p id="p-59">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.015933/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.015933/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-60"><em>Communicating editor: S. I. Wright</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-11-26"><span class="received-label">Received </span>November 26, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-03-29"><span class="accepted-label">Accepted </span>March 29, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Pearce <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.1" data-doi="10.1089/omi.2008.0017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Angiuoli</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gussman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klimke</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cochrane</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Field</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Toward an online repository of Standard Operating Procedures (SOPs) for (meta)genomic annotation.</span> <abbr class="cit-jnl-abbrev">OMICS</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">137</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOmics%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bintegrative%2Bbiology%26rft.stitle%253DOMICS%26rft.aulast%253DAngiuoli%26rft.auinit1%253DS.%2BV.%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253D137%26rft.epage%253D141%26rft.atitle%253DToward%2Ban%2Bonline%2Brepository%2Bof%2BStandard%2BOperating%2BProcedures%2B%2528SOPs%2529%2Bfor%2B%2528meta%2529genomic%2Bannotation.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fomi.2008.0017%26rft_id%253Dinfo%253Apmid%252F18416670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/omi.2008.0017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18416670&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257319700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.2" data-doi="10.1111/j.1462-5822.2005.00820.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Böltner</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreno-Morillas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramos</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">16S rDNA phylogeny and distribution of <em>lin</em> genes in novel hexachlorocyclohexane-degrading <em>Sphingomonas</em> strains.</span> <abbr class="cit-jnl-abbrev">Environ. Microbiol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1329</span>–<span class="cit-lpage">1338</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnvironmental%2BMicrobiology%2B%2528Print%2529%26rft.stitle%253DEnvironmental%2BMicrobiology%2B%2528Print%2529%26rft.aulast%253DB%25C3%25B6ltner%26rft.auinit1%253DD.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253D1329%26rft.epage%253D1338%26rft.atitle%253D16S%2BrDNA%2Bphylogeny%2Band%2Bdistribution%2Bof%2Blin%2Bgenes%2Bin%2Bnovel%2Bhexachlorocyclohexane-degrading%2BSphingomonas%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1462-5822.2005.00820.x%26rft_id%253Dinfo%253Apmid%252F16104856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1462-5822.2005.00820.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16104856&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231024000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.3" data-doi="10.1016/S0048-9697(99)00291-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breivik</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pacyna</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Münch</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Use of alpha-, beta- and gamma-hexachlorocyclohexane in Europe, 1970–1996.</span> <abbr class="cit-jnl-abbrev">Sci. Total Environ.</abbr> <span class="cit-vol">239</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">163</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BScience%2Bof%2Bthe%2BTotal%2BEnvironment%26rft.stitle%253DThe%2BScience%2Bof%2Bthe%2BTotal%2BEnvironment%26rft.aulast%253DBreivik%26rft.auinit1%253DK.%26rft.volume%253D239%26rft.issue%253D1-3%26rft.spage%253D151%26rft.epage%253D163%26rft.atitle%253DUse%2Bof%2Balpha-%252C%2Bbeta-%2Band%2Bgamma-hexachlorocyclohexane%2Bin%2BEurope%252C%2B1970-1996.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0048-9697%252899%252900291-0%26rft_id%253Dinfo%253Apmid%252F10636769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0048-9697(99)00291-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10636769&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083337200012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.4" data-doi="10.1371/journal.pone.0048918"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chuzeville</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puymège</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madec</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haenni</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payot</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Characterization of a new CAMP factor carried by an integrative and conjugative element in <em>Streptococcus agalactiae</em> and spreading in Streptococci.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e48918</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DChuzeville%26rft.auinit1%253DS.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De48918%26rft.epage%253De48918%26rft.atitle%253DCharacterization%2Bof%2Ba%2BNew%2BCAMP%2BFactor%2BCarried%2Bby%2Ban%2BIntegrative%2Band%2BConjugative%2BElement%2Bin%2BStreptococcus%2Bagalactiae%2Band%2BSpreading%2Bin%2BStreptococci.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0048918%26rft_id%253Dinfo%253Apmid%252F23152820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0048918&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23152820&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.5" data-doi="10.1093/gbe/evr137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Copley</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokicki</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daligault</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The whole genome sequence of <em>Sphingobium chlorophenolicum</em> L-1: insights into the evolution of the pentachlorophenol degradation pathway.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">184</span>–<span class="cit-lpage">198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevr137%26rft_id%253Dinfo%253Apmid%252F22179583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjQvMi8xODQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTA4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.6" data-doi="10.1093/nar/30.11.2478"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Delcher</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillippy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlton</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Fast algorithms for large-scale genome alignment and comparison.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2478</span>–<span class="cit-lpage">2483</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DDelcher%26rft.auinit1%253DA.%2BL.%26rft.volume%253D30%26rft.issue%253D11%26rft.spage%253D2478%26rft.epage%253D2483%26rft.atitle%253DFast%2Balgorithms%2Bfor%2Blarge-scale%2Bgenome%2Balignment%2Band%2Bcomparison%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F30.11.2478%26rft_id%253Dinfo%253Apmid%252F12034836%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMC8xMS8yNDc4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEwODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.7" data-doi="10.1128/JB.186.8.2225-2235.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dogra</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raina</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pal</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suar</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Organization of <em>lin</em> genes and IS<em>6100</em> among different strains of hexachlorocyclohexane-degrading <em>Sphingomonas paucimobilis</em>: evidence for horizontal gene transfer.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">2225</span>–<span class="cit-lpage">2235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DDogra%26rft.auinit1%253DC.%26rft.volume%253D186%26rft.issue%253D8%26rft.spage%253D2225%26rft.epage%253D2235%26rft.atitle%253DOrganization%2Bof%2Blin%2BGenes%2Band%2BIS6100%2Bamong%2BDifferent%2BStrains%2Bof%2BHexachlorocyclohexane-Degrading%2BSphingomonas%2Bpaucimobilis%253A%2BEvidence%2Bfor%2BHorizontal%2BGene%2BTransfer%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.186.8.2225-2235.2004%26rft_id%253Dinfo%253Apmid%252F15060023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE4Ni84LzIyMjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTA4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Draft genome sequence of <em>Sphingobium</em> sp. strain C100, a polycyclic aromatic hydrocarbon-degrading bacterium from the deep-sea sediment of the Arctic Ocean.</span> <abbr class="cit-jnl-abbrev">Genome Announc.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e01210</span>–<span class="cit-lpage">e01213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BAnnounc.%26rft.volume%253D2%26rft.spage%253De01210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.9" data-doi="10.1021/jf9038259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elcey</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunhi</span>  <span class="cit-name-given-names">A. A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Substantially enhanced degradation of hexachlorocyclohexane isomers by a microbial consortium on acclimation.</span> <abbr class="cit-jnl-abbrev">J. Agric. Food Chem.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">1046</span>–<span class="cit-lpage">1054</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAgricultural%2Band%2BFood%2BChemistry%26rft.stitle%253DJournal%2Bof%2BAgricultural%2Band%2BFood%2BChemistry%26rft.aulast%253DElcey%26rft.auinit1%253DC.%2BD.%26rft.volume%253D58%26rft.issue%253D2%26rft.spage%253D1046%26rft.epage%253D1054%26rft.atitle%253DSubstantially%2Benhanced%2Bdegradation%2Bof%2Bhexachlorocyclohexane%2Bisomers%2Bby%2Ba%2Bmicrobial%2Bconsortium%2Bon%2Bacclimation.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fjf9038259%26rft_id%253Dinfo%253Apmid%252F20041660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/jf9038259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20041660&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.10" data-doi="10.1007/s00239-004-0198-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fani</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brilli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liò</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The origin and evolution of operons: the piecewise building of the proteobacterial histidine operon.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">378</span>–<span class="cit-lpage">390</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DFani%26rft.auinit1%253DR.%26rft.volume%253D60%26rft.issue%253D3%26rft.spage%253D378%26rft.epage%253D390%26rft.atitle%253DThe%2Borigin%2Band%2Bevolution%2Bof%2Boperons%253A%2Bthe%2Bpiecewise%2Bbuilding%2Bof%2Bthe%2Bproteobacterial%2Bhistidine%2Boperon.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-004-0198-1%26rft_id%253Dinfo%253Apmid%252F15871048%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-004-0198-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15871048&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228828400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.11" data-doi="10.1016/j.resmic.2009.05.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fondi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emiliani</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Origin and evolution of operons and metabolic pathways.</span> <abbr class="cit-jnl-abbrev">Res. Microbiol.</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">502</span>–<span class="cit-lpage">512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DResearch%2Bin%2BMicrobiology%2B%2528Paris%2529%26rft.stitle%253DResearch%2Bin%2BMicrobiology%2B%2528Paris%2529%26rft.aulast%253DFondi%26rft.auinit1%253DM.%26rft.volume%253D160%26rft.issue%253D7%26rft.spage%253D502%26rft.epage%253D512%26rft.atitle%253DOrigin%2Band%2Bevolution%2Bof%2Boperons%2Band%2Bmetabolic%2Bpathways.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.resmic.2009.05.001%26rft_id%253Dinfo%253Apmid%252F19465116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.resmic.2009.05.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19465116&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.12" data-doi="10.1371/journal.pone.0010018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gai</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tai</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genes coding for the conversion of carbazole to catechol are flanked by IS<em>6100</em> elements in <em>Sphingomonas</em> sp. strain XLDN2–5.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e10018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DGai%26rft.auinit1%253DZ.%26rft.volume%253D5%26rft.issue%253D4%26rft.spage%253De10018%26rft.epage%253De10018%26rft.atitle%253DThe%2Bgenes%2Bcoding%2Bfor%2Bthe%2Bconversion%2Bof%2Bcarbazole%2Bto%2Bcatechol%2Bare%2Bflanked%2Bby%2BIS6100%2Belements%2Bin%2BSphingomonas%2Bsp.%2Bstrain%2BXLDN2-5.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0010018%26rft_id%253Dinfo%253Apmid%252F20368802%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0010018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20368802&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.13" data-doi="10.1128/JB.188.5.1999-2013.2006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaillard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vallaeys</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vorhölter</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minoia</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Werlen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The <em>clc</em> element of <em>Pseudomonas</em> sp. strain B13, a genomic island with various catabolic properties.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">1999</span>–<span class="cit-lpage">2013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DGaillard%26rft.auinit1%253DM.%26rft.volume%253D188%26rft.issue%253D5%26rft.spage%253D1999%26rft.epage%253D2013%26rft.atitle%253DThe%2Bclc%2BElement%2Bof%2BPseudomonas%2Bsp.%2BStrain%2BB13%252C%2Ba%2BGenomic%2BIsland%2Bwith%2BVarious%2BCatabolic%2BProperties%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.188.5.1999-2013.2006%26rft_id%253Dinfo%253Apmid%252F16484212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE4OC81LzE5OTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTA4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.14" data-doi="10.1128/JB.01159-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gan</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chew</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">A. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Savka</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome sequence of <em>Novosphingobium</em> sp. strain Rr 2–17, a nopaline crown gall-associated bacterium isolated from <em>Vitis vinifera</em> L. grapevine.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">5137</span>–<span class="cit-lpage">5138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DGan%26rft.auinit1%253DH.%2BM.%26rft.volume%253D194%26rft.issue%253D18%26rft.spage%253D5137%26rft.epage%253D5138%26rft.atitle%253DGenome%2BSequence%2Bof%2BNovosphingobium%2Bsp.%2BStrain%2BRr%2B2-17%252C%2Ba%2BNopaline%2BCrown%2BGall-Associated%2BBacterium%2BIsolated%2Bfrom%2BVitis%2Bvinifera%2BL.%2BGrapevine%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.01159-12%26rft_id%253Dinfo%253Apmid%252F22933764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE5NC8xOC81MTM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEwODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.15" data-doi="10.1111/1462-2920.12009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Geueke</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garg</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghosh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fleischmann</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holliger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Metabolomics of hexachlorocyclohexane (HCH) transformation: ratio of LinA to LinB determines metabolic fate of HCH isomers.</span> <abbr class="cit-jnl-abbrev">Environ. Microbiol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1040</span>–<span class="cit-lpage">1049</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnviron.%2BMicrobiol.%26rft.volume%253D15%26rft.spage%253D1040%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1462-2920.12009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1462-2920.12009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.16" data-doi="10.1371/journal.pgen.1002222"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guglielmini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quintais</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcillán-Barcia</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de la Cruz</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rocha</span>  <span class="cit-name-given-names">E. P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The repertoire of ICE in prokaryotes underscores the unity, diversity, and ubiquity of conjugation.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGuglielmini%26rft.auinit1%253DJ.%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253De1002222%26rft.epage%253De1002222%26rft.atitle%253DThe%2Brepertoire%2Bof%2BICE%2Bin%2Bprokaryotes%2Bunderscores%2Bthe%2Bunity%252C%2Bdiversity%252C%2Band%2Bubiquity%2Bof%2Bconjugation.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002222%26rft_id%253Dinfo%253Apmid%252F21876676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002222&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21876676&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.17" data-doi="10.1093/bioinformatics/btq413"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guy</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kultima</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">S. G. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">genoPlotR: comparative gene and genome visualization in R.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2334</span>–<span class="cit-lpage">2335</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq413%26rft_id%253Dinfo%253Apmid%252F20624783%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMTgvMjMzNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMDgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.18" data-doi="10.6026/97320630008203"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hasan</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fazekas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">GIST: Genomic island suite of tools for predicting genomic islands in genomic sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformation</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBioinformation%26rft.aulast%253DHasan%26rft.auinit1%253DM.%2BS.%26rft.volume%253D8%26rft.issue%253D4%26rft.spage%253D203%26rft.epage%253D205%26rft.atitle%253DGIST%253A%2BGenomic%2Bisland%2Bsuite%2Bof%2Btools%2Bfor%2Bpredicting%2Bgenomic%2Bislands%2Bin%2Bgenomic%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.6026%252F97320630008203%26rft_id%253Dinfo%253Apmid%252F22419842%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.6026/97320630008203&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22419842&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.19" data-doi="10.1111/j.1574-6968.1999.tb13435.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Herron</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dyson</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Low target site specificity of an IS<em>6100</em> -based mini-transposon, Tn<em>1792</em>, developed for transposon mutagenesis of antibiotic producing <em>Streptomyces</em></span>. <abbr class="cit-jnl-abbrev">FEMS Microbiol. Lett.</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">215</span>–<span class="cit-lpage">221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BLett.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6968.1999.tb13435.x%26rft_id%253Dinfo%253Apmid%252F10077847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc2xlIjtzOjU6InJlc2lkIjtzOjk6IjE3MS8yLzIxNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMDgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.20" data-doi="10.1023/A:1005323811769"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johri</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dua</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuteja</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saxena</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saxena</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Degradation of alpha, beta, gamma and delta-hexachlorocyclohexanes by <em>Sphingomonas paucimobilis</em>.</span> <abbr class="cit-jnl-abbrev">Biotechnol. Lett.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">885</span>–<span class="cit-lpage">887</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechnol.%2BLett.%26rft.volume%253D20%26rft.spage%253D885%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1005323811769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1005323811769&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.21" data-doi="10.1128/jb.176.4.1197-1200.1994"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kato</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohtsuki</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitsuda</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yomo</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Negoro</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Insertion sequence IS<em>6100</em> on plasmid pOAD2, which degrades nylon oligomers.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">1197</span>–<span class="cit-lpage">1200</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DKato%26rft.auinit1%253DK.%26rft.volume%253D176%26rft.issue%253D4%26rft.spage%253D1197%26rft.epage%253D1200%26rft.atitle%253DInsertion%2Bsequence%2BIS6100%2Bon%2Bplasmid%2BpOAD2%252C%2Bwhich%2Bdegrades%2Bnylon%2Boligomers.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.176.4.1197-1200.1994%26rft_id%253Dinfo%253Apmid%252F8106333%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni80LzExOTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTA4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.22" data-doi="10.1099/ijs.0.039834-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaur</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moskalikova</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niharika</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sedlackova</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hampl</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title"><em>Sphingobium baderi</em> sp. nov., isolated from a hexachlorocyclohexane dump site.</span> <abbr class="cit-jnl-abbrev">Int. J. Syst. Evol. Microbiol.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">673</span>–<span class="cit-lpage">678</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BSyst.%2BEvol.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fijs.0.039834-0%26rft_id%253Dinfo%253Apmid%252F22544800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/ijs.0.039834-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22544800&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kohli</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dua</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sangwan</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oldach</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khurana</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Draft genome sequence of <em>Sphingobium ummariense</em> strain RL-3, a hexachlorocyclohexane-degrading bacterium.</span> <abbr class="cit-jnl-abbrev">Genome Announc.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">e00956</span>–<span class="cit-lpage">e13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BAnnounc.%26rft.volume%253D1%26rft.spage%253De00956%26rft_id%253Dinfo%253Apmid%252F24233594%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24233594&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.24" data-doi="10.1101/gr.092759.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krzywinski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schein</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birol</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connors</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gascoyne</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Circos: an information aesthetic for comparative genomics.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1639</span>–<span class="cit-lpage">1645</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.092759.109%26rft_id%253Dinfo%253Apmid%252F19541911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTYzOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMDgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.25" data-doi="10.1021/es048497q"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaudhary</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dwivedi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Enhanced biodegradation of beta- and delta-hexachlorocyclohexane in the presence of alpha- and gamma-isomers in contaminated soils.</span> <abbr class="cit-jnl-abbrev">Environ. Sci. Technol.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">4005</span>–<span class="cit-lpage">4011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnvironmental%2BScience%2B%2526%2BTechnology%252C%2BES%2B%2526%2BT%26rft.stitle%253DEnvironmental%2BScience%2B%2526%2BTechnology%252C%2BES%2B%2526%2BT%26rft.aulast%253DKumar%26rft.auinit1%253DM.%26rft.volume%253D39%26rft.issue%253D11%26rft.spage%253D4005%26rft.epage%253D4011%26rft.atitle%253DEnhanced%2Bbiodegradation%2Bof%2Bbeta-%2Band%2Bdelta-hexachlorocyclohexane%2Bin%2Bthe%2Bpresence%2Bof%2Balpha-%2Band%2Bgamma-isomers%2Bin%2Bcontaminated%2Bsoils.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fes048497q%26rft_id%253Dinfo%253Apmid%252F15984776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/es048497q&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15984776&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar Singh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sangwan</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khurana</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Draft genome sequence of <em>Sphingobium quisquiliarum</em> strain P25T, a novel hexachlorocyclohexane (HCH)-degrading bacterium isolated from an HCH dumpsite.</span> <abbr class="cit-jnl-abbrev">Genome Announc.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">e00717</span>–<span class="cit-lpage">e13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BAnnounc.%26rft.volume%253D1%26rft.spage%253De00717%26rft_id%253Dinfo%253Apmid%252F24029763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24029763&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.27" data-doi="10.1002/jobm.201100335"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lal</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jindal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumari</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jit</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nigam</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Bacterial diversity and real-time PCR based assessment of <em>linA</em> and <em>linB</em> gene distribution at hexachlorocyclohexane contaminated sites.</span> <abbr class="cit-jnl-abbrev">J. Basic Microbiol.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBasic%2BMicrobiol.%26rft.volume%253D53%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjobm.201100335%26rft_id%253Dinfo%253Apmid%252F22359233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jobm.201100335&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22359233&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.28" data-doi="10.1016/j.tibtech.2006.01.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lal</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dogra</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malhotra</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pal</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Diversity, distribution and divergence of <em>lin</em> genes in hexachlorocyclohexane-degrading sphingomonads.</span> <abbr class="cit-jnl-abbrev">Trends Biotechnol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">121</span>–<span class="cit-lpage">130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bbiotechnology%26rft.stitle%253DTrends%2BBiotechnol%26rft.aulast%253DLal%26rft.auinit1%253DR.%26rft.volume%253D24%26rft.issue%253D3%26rft.spage%253D121%26rft.epage%253D130%26rft.atitle%253DDiversity%252C%2Bdistribution%2Band%2Bdivergence%2Bof%2Blin%2Bgenes%2Bin%2Bhexachlorocyclohexane-degrading%2Bsphingomonads.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibtech.2006.01.005%26rft_id%253Dinfo%253Apmid%252F16473421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibtech.2006.01.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16473421&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.29" data-doi="10.1128/MMBR.00029-09"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lal</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumari</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malhotra</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Biochemistry of microbial degradation of hexachlorocyclohexane and prospects for bioremediation.</span> <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">58</span>–<span class="cit-lpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DLal%26rft.auinit1%253DR.%26rft.volume%253D74%26rft.issue%253D1%26rft.spage%253D58%26rft.epage%253D80%26rft.atitle%253DBiochemistry%2Bof%2BMicrobial%2BDegradation%2Bof%2BHexachlorocyclohexane%2Band%2BProspects%2Bfor%2BBioremediation%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMMBR.00029-09%26rft_id%253Dinfo%253Apmid%252F20197499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo3OiI3NC8xLzU4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEwODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lawrence</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roth</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Selfish operons: horizontal transfer may drive the evolution of gene clusters.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">1843</span>–<span class="cit-lpage">1860</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLawrence%26rft.auinit1%253DJ.%2BG.%26rft.volume%253D143%26rft.issue%253D4%26rft.spage%253D1843%26rft.epage%253D1860%26rft.atitle%253DSelfish%2BOperons%253A%2BHorizontal%2BTransfer%2BMay%2BDrive%2Bthe%2BEvolution%2Bof%2BGene%2BClusters%26rft_id%253Dinfo%253Apmid%252F8844169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0My80LzE4NDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTA4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.31" data-doi="10.1128/AEM.02543-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dai</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Patchwork assembly of <em>nag</em>-like nitroarene dioxygenase genes and the 3-chlorocatechol degradation cluster for evolution of the 2-chloronitrobenzene catabolism pathway in <em>Pseudomonas stutzeri</em> ZWLR2–1.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">4547</span>–<span class="cit-lpage">4552</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.02543-10%26rft_id%253Dinfo%253Apmid%252F21602392%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI3Ny8xMy80NTQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEwODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.32" data-doi="10.1128/JB.06476-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">Y. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">M.-R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome sequence of benzo(a)pyrene-degrading bacterium <em>Novosphingobium pentaromativorans</em> US6–1.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">907</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DLuo%26rft.auinit1%253DY.%2BR.%26rft.volume%253D194%26rft.issue%253D4%26rft.spage%253D907%26rft.epage%253D907%26rft.atitle%253DGenome%2BSequence%2Bof%2BBenzo%2528a%2529pyrene-Degrading%2BBacterium%2BNovosphingobium%2Bpentaromativorans%2BUS6-1%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.06476-11%26rft_id%253Dinfo%253Apmid%252F22275104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTk0LzQvOTA3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEwODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.33" data-doi="10.1007/s12010-012-0035-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Macwan</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Javed</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Role of a repeated hexapeptide motif GIHFAP near C-terminus in assembly, stability, and activity of “HCH dehydrochlorinase LinA”</span>. <abbr class="cit-jnl-abbrev">Appl. Biochem. Biotechnol</abbr>. <span class="cit-vol">169</span>: <span class="cit-fpage">1397</span>–<span class="cit-lpage">1404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BBiochem.%2BBiotechnol%26rft.volume%253D169%26rft.spage%253D1397%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12010-012-0035-8%26rft_id%253Dinfo%253Apmid%252F23315208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12010-012-0035-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23315208&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mahillon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chandler</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Insertion sequences.</span> <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">725</span>–<span class="cit-lpage">774</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DMahillon%26rft.auinit1%253DJ.%26rft.volume%253D62%26rft.issue%253D3%26rft.spage%253D725%26rft.epage%253D774%26rft.atitle%253DInsertion%2BSequences%26rft_id%253Dinfo%253Apmid%252F9729608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI2Mi8zLzcyNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMDgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.35" data-doi="10.1111/j.1365-2672.2006.03209.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manickam</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misra</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mayilraj</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A novel pathway for the biodegradation of gamma-hexachlorocyclohexane by a <em>Xanthomonas</em> sp. strain ICH12.</span> <abbr class="cit-jnl-abbrev">J. Appl. Microbiol.</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">1468</span>–<span class="cit-lpage">1478</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BApplied%2BMicrobiology%2B%2528Print%2529%26rft.stitle%253DJournal%2Bof%2BApplied%2BMicrobiology%2B%2528Print%2529%26rft.aulast%253DManickam%26rft.auinit1%253DN.%26rft.volume%253D102%26rft.issue%253D6%26rft.spage%253D1468%26rft.epage%253D1478%26rft.atitle%253DA%2Bnovel%2Bpathway%2Bfor%2Bthe%2Bbiodegradation%2Bof%2Bgamma-hexachlorocyclohexane%2Bby%2Ba%2BXanthomonas%2Bsp.%2Bstrain%2BICH12.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2672.2006.03209.x%26rft_id%253Dinfo%253Apmid%252F17578411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2672.2006.03209.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17578411&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.36" data-doi="10.1021/jf990712c"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manonmani</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chandrashekaraiah</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sreedhar Reddy</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elcey</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunhi</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Isolation and acclimation of a microbial consortium for improved aerobic degradation of alpha-hexachlorocyclohexane.</span> <abbr class="cit-jnl-abbrev">J. Agric. Food Chem.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">4341</span>–<span class="cit-lpage">4351</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAgricultural%2Band%2BFood%2BChemistry%26rft.stitle%253DJournal%2Bof%2BAgricultural%2Band%2BFood%2BChemistry%26rft.aulast%253DManonmani%26rft.auinit1%253DH.%2BK.%26rft.volume%253D48%26rft.issue%253D9%26rft.spage%253D4341%26rft.epage%253D4351%26rft.atitle%253DIsolation%2Band%2Bacclimation%2Bof%2Ba%2Bmicrobial%2Bconsortium%2Bfor%2Bimproved%2Baerobic%2Bdegradation%2Bof%2Balpha-hexachlorocyclohexane.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fjf990712c%26rft_id%253Dinfo%253Apmid%252F10995361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/jf990712c&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10995361&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.37" data-doi="10.1128/JB.06254-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Masai</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kamimura</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasai</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oguchi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ankai</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Complete genome sequence of <em>Sphingobium</em> sp. strain SYK-6, a degrader of lignin-derived biaryls and monoaryls.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">534</span>–<span class="cit-lpage">535</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DMasai%26rft.auinit1%253DE.%26rft.volume%253D194%26rft.issue%253D2%26rft.spage%253D534%26rft.epage%253D535%26rft.atitle%253DComplete%2BGenome%2BSequence%2Bof%2BSphingobium%2Bsp.%2BStrain%2BSYK-6%252C%2Ba%2BDegrader%2Bof%2BLignin-Derived%2BBiaryls%2Band%2BMonoaryls%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.06254-11%26rft_id%253Dinfo%253Apmid%252F22207743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTk0LzIvNTM0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEwODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.38" data-doi="10.1080/10242420600667809"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mencía</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martínez-Ferri</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alcalde</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Lorenzo</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Identification of a -hexachlorocyclohexane dehydrochlorinase (LinA) variant with improved expression and solubility properties</span>. <abbr class="cit-jnl-abbrev">Biocat. Biotrans</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">223</span>–<span class="cit-lpage">230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiocat.%2BBiotrans%26rft.volume%253D24%26rft.spage%253D223%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10242420600667809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10242420600667809&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.39" data-doi="10.1093/jac/dkr406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michael</span>  <span class="cit-name-given-names">G. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kadlec</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sweeney</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brzuszkiewicz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liesegang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">ICEP<em>mu</em>1, an integrative conjugative element (ICE) of <em>Pasteurella multocida</em>: analysis of the regions that comprise 12 antimicrobial resistance genes.</span> <abbr class="cit-jnl-abbrev">J. Antimicrob. Chemother.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">84</span>–<span class="cit-lpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAntimicrob.%2BChemother.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjac%252Fdkr406%26rft_id%253Dinfo%253Apmid%252F22001175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamFjIjtzOjU6InJlc2lkIjtzOjc6IjY3LzEvODQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTA4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miyauchi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suh</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagata</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takagi</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Cloning and sequencing of a 2,5-dichlorohydroquinone reductive dehalogenase gene whose product is involved in degradation of gamma-hexachlorocyclohexane by <em>Sphingomonas paucimobilis</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">1354</span>–<span class="cit-lpage">1359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DMiyauchi%26rft.auinit1%253DK.%26rft.volume%253D180%26rft.issue%253D6%26rft.spage%253D1354%26rft.epage%253D1359%26rft.atitle%253DCloning%2Band%2BSequencing%2Bof%2Ba%2B2%252C5-Dichlorohydroquinone%2BReductive%2BDehalogenase%2BGene%2BWhose%2BProduct%2BIs%2BInvolved%2Bin%2BDegradation%2Bof%2Bgamma%2B-Hexachlorocyclohexane%2Bby%2BSphingomonas%2Bpaucimobilis%26rft_id%253Dinfo%253Apmid%252F9515900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE4MC82LzEzNTQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTA4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miyauchi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adachi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagata</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takagi</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Cloning and sequencing of a novel meta-cleavage dioxygenase gene whose product is involved in degradation of gamma-hexachlorocyclohexane in <em>Sphingomonas paucimobilis</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">181</span>: <span class="cit-fpage">6712</span>–<span class="cit-lpage">6719</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DMiyauchi%26rft.auinit1%253DK.%26rft.volume%253D181%26rft.issue%253D21%26rft.spage%253D6712%26rft.epage%253D6719%26rft.atitle%253DCloning%2Band%2BSequencing%2Bof%2Ba%2BNovel%2Bmeta-Cleavage%2BDioxygenase%2BGene%2BWhose%2BProduct%2BIs%2BInvolved%2Bin%2BDegradation%2Bof%2Bgamma%2B-Hexachlorocyclohexane%2Bin%2BSphingomonas%2Bpaucimobilis%26rft_id%253Dinfo%253Apmid%252F10542173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE4MS8yMS82NzEyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEwODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.42" data-doi="10.1128/AEM.01531-06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miyazaki</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ito</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohtsubo</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagata</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Complete nucleotide sequence of an exogenously isolated plasmid, pLB1, involved in gamma-hexachlorocyclohexane degradation.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">6923</span>–<span class="cit-lpage">6933</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.01531-06%26rft_id%253Dinfo%253Apmid%252F16963556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI3Mi8xMS82OTIzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEwODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mukherjee</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahato</span>  <span class="cit-name-given-names">N. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khurana</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lal</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Draft genome sequence of <em>Sphingobium</em> sp. strain HDIPO4, an avid degrader of hexachlorocyclohexane.</span> <abbr class="cit-jnl-abbrev">Genome Announc.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">e00749</span>–<span class="cit-lpage">e13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BAnnounc.%26rft.volume%253D1%26rft.spage%253De00749%26rft_id%253Dinfo%253Apmid%252F24051321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24051321&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.44" data-doi="10.1186/1471-2105-13-S14-S8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nadalin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vezzi</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Policriti</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">GapFiller: a de novo assembly approach to fill the gap within paired reads.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">S8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D13%26rft.spage%253DS8%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-13-S14-S8%26rft_id%253Dinfo%253Apmid%252F23282012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-13-S14-S8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23282012&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.45" data-doi="10.1271/bbb.57.1582"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagata</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hatta</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imai</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimbara</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fukuda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1993</span><span class="cit-comment">a</span> <span class="cit-article-title">Purification and characterization of gamma-hexachlorocyclohexane (gamma-HCH) dehydrochlorinase (LinA) from <em>Pseudomonas paucimobilis</em>.</span> <abbr class="cit-jnl-abbrev">Biosci. Biotechnol. Biochem.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">1582</span>–<span class="cit-lpage">1583</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiosci.%2BBiotechnol.%2BBiochem.%26rft.volume%253D57%26rft.spage%253D1582%26rft_id%253Dinfo%253Adoi%252F10.1271%252Fbbb.57.1582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1271/bbb.57.1582&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.46" data-doi="10.1128/jb.175.20.6403-6410.1993"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagata</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nariya</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohtomo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fukuda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yano</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1993</span><span class="cit-comment">b</span> <span class="cit-article-title">Cloning and sequencing of a dehalogenase gene encoding an enzyme with hydrolase activity involved in the degradation of gamma-hexachlorocyclohexane in <em>Pseudomonas paucimobilis</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">6403</span>–<span class="cit-lpage">6410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DNagata%26rft.auinit1%253DY.%26rft.volume%253D175%26rft.issue%253D20%26rft.spage%253D6403%26rft.epage%253D6410%26rft.atitle%253DCloning%2Band%2Bsequencing%2Bof%2Ba%2Bdehalogenase%2Bgene%2Bencoding%2Ban%2Benzyme%2Bwith%2Bhydrolase%2Bactivity%2Binvolved%2Bin%2Bthe%2Bdegradation%2Bof%2Bgamma-hexachlorocyclohexane%2Bin%2BPseudomonas%2Bpaucimobilis.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.175.20.6403-6410.1993%26rft_id%253Dinfo%253Apmid%252F7691794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE3NS8yMC82NDAzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEwODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.47" data-doi="10.1128/jb.176.11.3117-3125.1994"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagata</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohtomo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miyauchi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fukuda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yano</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Cloning and sequencing of a 2,5-dichloro-2,5-cyclohexadiene-1,4-diol dehydrogenase gene involved in the degradation of gamma-hexachlorocyclohexane in <em>Pseudomonas paucimobilis</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">3117</span>–<span class="cit-lpage">3125</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DNagata%26rft.auinit1%253DY.%26rft.volume%253D176%26rft.issue%253D11%26rft.spage%253D3117%26rft.epage%253D3125%26rft.atitle%253DCloning%2Band%2Bsequencing%2Bof%2Ba%2B2%252C5-dichloro-2%252C5-cyclohexadiene-1%252C4-diol%2Bdehydrogenase%2Bgene%2Binvolved%2Bin%2Bthe%2Bdegradation%2Bof%2Bgamma-hexachlorocyclohexane%2Bin%2BPseudomonas%2Bpaucimobilis.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.176.11.3117-3125.1994%26rft_id%253Dinfo%253Apmid%252F7515041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE3Ni8xMS8zMTE3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEwODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.48" data-doi="10.1007/s00253-007-1066-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagata</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ito</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohtsubo</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsuda</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Aerobic degradation of lindane (gamma-hexachlorocyclohexane) in bacteria and its biochemical and molecular basis.</span> <abbr class="cit-jnl-abbrev">Appl. Microbiol. Biotechnol.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">741</span>–<span class="cit-lpage">752</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2BMicrobiology%2Band%2BBiotechnology%26rft.stitle%253DApplied%2BMicrobiology%2Band%2BBiotechnology%26rft.aulast%253DNagata%26rft.auinit1%253DY.%26rft.volume%253D76%26rft.issue%253D4%26rft.spage%253D741%26rft.epage%253D752%26rft.atitle%253DAerobic%2Bdegradation%2Bof%2Blindane%2B%2528gamma-hexachlorocyclohexane%2529%2Bin%2Bbacteria%2Band%2Bits%2Bbiochemical%2Band%2Bmolecular%2Bbasis.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00253-007-1066-x%26rft_id%253Dinfo%253Apmid%252F17634937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00253-007-1066-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17634937&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249117500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.49" data-doi="10.1128/JB.00961-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagata</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohtsubo</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ichikawa</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ankai</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Complete genome sequence of the representative γ-hexachlorocyclohexane-degrading bacterium <em>Sphingobium japonicum</em> UT26.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">5852</span>–<span class="cit-lpage">5853</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBacteriol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.00961-10%26rft_id%253Dinfo%253Apmid%252F20817768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE5Mi8yMS81ODUyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEwODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.50" data-doi="10.1016/j.enzmictec.2011.10.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagata</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Natsui</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohtsubo</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ichikawa</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic organization and genomic structural rearrangements of <em>Sphingobium japonicum</em> UT26, an archetypal γ-hexachlorocyclohexane-degrading bacterium.</span> <abbr class="cit-jnl-abbrev">Enzyme Microb. Technol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">508</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnzyme%2Band%2BMicrobial%2BTechnology%26rft.stitle%253DEnzyme%2Band%2BMicrobial%2BTechnology%26rft.aulast%253DNagata%26rft.auinit1%253DY.%26rft.volume%253D49%26rft.issue%253D6-7%26rft.spage%253D499%26rft.epage%253D508%26rft.atitle%253DGenomic%2Borganization%2Band%2Bgenomic%2Bstructural%2Brearrangements%2Bof%2BSphingobium%2Bjaponicum%2BUT26%252C%2Ban%2Barchetypal%2B%25CE%25B3-hexachlorocyclohexane-degrading%2Bbacterium.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.enzmictec.2011.10.005%26rft_id%253Dinfo%253Apmid%252F22142724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.enzmictec.2011.10.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22142724&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298529600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-book" id="cit-5.6.1081.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagata</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tabata</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohhata</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsuda</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Appearance and evolution of γ-hexachlorocyclohexane-degrading bacteria</span>, pp. <span class="cit-fpage">19</span>–<span class="cit-lpage">41</span> in <span class="cit-source"><em>Biodegradative Bacteria: How Bacteria Degrade</em>, <em>Survive</em>, <em>Adapt</em>, <em>and Evolve</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Nojiri</span>  <span class="cit-name-given-names">H.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Tsuda</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Fukuda</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Kamagata</span>  <span class="cit-name-given-names">Y.</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Japan</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.52" data-doi="10.1099/ijs.0.039396-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Niharika</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moskalikova</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaur</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khan</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sedlackova</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">a</span> <span class="cit-article-title"><em>Sphingobium czechense</em> sp. nov., isolated from a hexachlorocyclohexane dump site.</span> <abbr class="cit-jnl-abbrev">Int. J. Syst. Evol. Microbiol.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">723</span>–<span class="cit-lpage">728</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BSyst.%2BEvol.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fijs.0.039396-0%26rft_id%253Dinfo%253Apmid%252F22561590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/ijs.0.039396-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22561590&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.53" data-doi="10.1099/ijs.0.039826-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Niharika</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moskalikova</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaur</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sedlackova</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hampl</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">b</span> <span class="cit-article-title"><em>Novosphingobium barchaimii</em> sp. nov., isolated from hexachlorocyclohexane-contaminated soil.</span> <abbr class="cit-jnl-abbrev">Int. J. Syst. Evol. Microbiol.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">667</span>–<span class="cit-lpage">672</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BSyst.%2BEvol.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fijs.0.039826-0%26rft_id%253Dinfo%253Apmid%252F22544801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/ijs.0.039826-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22544801&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.54" data-doi="10.1016/j.jmb.2010.08.043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Okai</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kubota</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fukuda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagata</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagata</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Crystal structure of γ-hexachlorocyclohexane dehydrochlorinase LinA from <em>Sphingobium japonicum</em> UT26.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">403</span>: <span class="cit-fpage">260</span>–<span class="cit-lpage">269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DOkai%26rft.auinit1%253DM.%26rft.volume%253D403%26rft.issue%253D2%26rft.spage%253D260%26rft.epage%253D269%26rft.atitle%253DCrystal%2Bstructure%2Bof%2B%25CE%25B3-hexachlorocyclohexane%2BDehydrochlorinase%2BLinA%2Bfrom%2BSphingobium%2Bjaponicum%2BUT26.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2010.08.043%26rft_id%253Dinfo%253Apmid%252F20813114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2010.08.043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20813114&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.55" data-doi="10.1016/j.tig.2004.04.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pál</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurst</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Evidence against the selfish operon theory.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">232</span>–<span class="cit-lpage">234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DWen%26rft.auinit1%253DF.%26rft.volume%253D20%26rft.issue%253D5%26rft.spage%253D232%26rft.epage%253D236%26rft.atitle%253DThe%2Bimpact%2Bof%2Bvery%2Bshort%2Balternative%2Bsplicing%2Bon%2Bprotein%2Bstructures%2Band%2Bfunctions%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2004.04.001%26rft_id%253Dinfo%253Apmid%252F15109776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2004.04.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15109776&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221435500005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.56" data-doi="10.1111/j.1365-2958.2006.05396.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramsay</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sullivan</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stuart</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamont</span>  <span class="cit-name-given-names">I. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronson</span>  <span class="cit-name-given-names">C. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Excision and transfer of the Mesorhizobium loti R7A symbiosis island requires an integrase IntS, a novel recombination directionality factor RdfS, and a putative relaxase RlxS.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">723</span>–<span class="cit-lpage">734</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DRamsay%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D62%26rft.issue%253D3%26rft.spage%253D723%26rft.epage%253D734%26rft.atitle%253DExcision%2Band%2Btransfer%2Bof%2Bthe%2BMesorhizobium%2Bloti%2BR7A%2Bsymbiosis%2Bisland%2Brequires%2Ban%2Bintegrase%2BIntS%252C%2Ba%2Bnovel%2Brecombination%2Bdirectionality%2Bfactor%2BRdfS%252C%2Band%2Ba%2Bputative%2Brelaxase%2BRlxS.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2006.05396.x%26rft_id%253Dinfo%253Apmid%252F17076666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2006.05396.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17076666&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.57" data-doi="10.1073/pnas.0906412106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Richter</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosselló-Móra</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Shifting the genomic gold standard for the prokaryotic species definition.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">19126</span>–<span class="cit-lpage">19131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0906412106%26rft_id%253Dinfo%253Apmid%252F19855009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzQ1LzE5MTI2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEwODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.58" data-doi="10.1128/AAC.05997-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rodríguez-Blanco</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemos</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osorio</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Integrating conjugative elements as vectors of antibiotic, mercury, and quaternary ammonium compound resistance in marine aquaculture environments.</span> <abbr class="cit-jnl-abbrev">Antimicrob. Agents Chemother.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">2619</span>–<span class="cit-lpage">2626</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntimicrob.%2BAgents%2BChemother.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAAC.05997-11%26rft_id%253Dinfo%253Apmid%252F22314526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWFjIjtzOjU6InJlc2lkIjtzOjk6IjU2LzUvMjYxOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMDgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.59" data-doi="10.1371/journal.pone.0046219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sangwan</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lata</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dwivedi</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niharika</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Comparative metagenomic analysis of soil microbial communities across three hexachlorocyclohexane contamination levels.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e46219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSangwan%26rft.auinit1%253DN.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253De46219%26rft.epage%253De46219%26rft.atitle%253DComparative%2Bmetagenomic%2Banalysis%2Bof%2Bsoil%2Bmicrobial%2Bcommunities%2Bacross%2Bthree%2Bhexachlorocyclohexane%2Bcontamination%2Blevels.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0046219%26rft_id%253Dinfo%253Apmid%252F23029440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0046219&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23029440&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.60" data-doi="10.1038/ismej.2013.153"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sangwan</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verma</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Negi</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lax</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Reconstructing an ancestral genotype of two hexachlorocyclohexane-degrading <em>Sphingobium</em> species using metagenomic sequence data.</span> <abbr class="cit-jnl-abbrev">ISME J.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">398</span>–<span class="cit-lpage">408</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DISME%2BJ.%26rft.volume%253D8%26rft.spage%253D398%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fismej.2013.153%26rft_id%253Dinfo%253Apmid%252F24030592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ismej.2013.153&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24030592&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.61" data-doi="10.1093/bioinformatics/btr026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmieder</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Quality control and preprocessing of metagenomic datasets.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">863</span>–<span class="cit-lpage">864</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr026%26rft_id%253Dinfo%253Apmid%252F21278185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNy82Lzg2MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMDgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.62" data-doi="10.1080/00380768.1989.10434739"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Senoo</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wada</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Isolation and identification of an aerobic γ-HCH-decomposing bacterium from soil.</span> <abbr class="cit-jnl-abbrev">Soil Sci. Plant Nutr.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSoil%2BSci.%2BPlant%2BNutr.%26rft.volume%253D35%26rft.spage%253D79%26rft_id%253Dinfo%253Adoi%252F10.1080%252F00380768.1989.10434739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/00380768.1989.10434739&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1989T896800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.63" data-doi="10.1128/JB.06403-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seth-Smith</span>  <span class="cit-name-given-names">H. M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fookes</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okoro</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Structure, diversity, and mobility of the <em>Salmonella</em> pathogenicity island 7 family of integrative and conjugative elements within <em>Enterobacteriaceae</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">1494</span>–<span class="cit-lpage">1504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBacteriol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.06403-11%26rft_id%253Dinfo%253Apmid%252F22247511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE5NC82LzE0OTQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTA4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.64" data-doi="10.1371/journal.pone.0025128"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumari</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Kinetic and sequence-structure-function analysis of known LinA variants with different hexachlorocyclohexane isomers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e25128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSharma%26rft.auinit1%253DP.%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253De25128%26rft.epage%253De25128%26rft.atitle%253DKinetic%2Band%2Bsequence-structure-function%2Banalysis%2Bof%2Bknown%2BLinA%2Bvariants%2Bwith%2Bdifferent%2Bhexachlorocyclohexane%2Bisomers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0025128%26rft_id%253Dinfo%253Apmid%252F21949868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0025128&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21949868&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.65" data-doi="10.1371/journal.pone.0051162"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shettigar</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pearce</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khan</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorrian</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Cloning of a novel 6-chloronicotinic acid chlorohydrolase from the newly isolated 6-chloronicotinic acid mineralizing Bradyrhizobiaceae strain SG-6C.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e51162</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D7%26rft.spage%253De51162%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0051162%26rft_id%253Dinfo%253Apmid%252F23226482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0051162&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23226482&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.66" data-doi="10.1101/gr.089532.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackman</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schein</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">S. J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">ABySS: a parallel assembler for short read sequence data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1117</span>–<span class="cit-lpage">1123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.089532.108%26rft_id%253Dinfo%253Apmid%252F19251739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzYvMTExNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMDgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.67" data-doi="10.1007/s00253-007-1056-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaudhary</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macwan</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diwedi</span>  <span class="cit-name-given-names">U. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Selective loss of <em>lin</em> genes from hexachlorocyclohexane-degrading <em>Pseudomonas aeruginosa</em> ITRC-5 under different growth conditions.</span> <abbr class="cit-jnl-abbrev">Appl. Microbiol. Biotechnol.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">895</span>–<span class="cit-lpage">901</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2BMicrobiology%2Band%2BBiotechnology%26rft.stitle%253DApplied%2BMicrobiology%2Band%2BBiotechnology%26rft.aulast%253DSingh%26rft.auinit1%253DA.%2BK.%26rft.volume%253D76%26rft.issue%253D4%26rft.spage%253D895%26rft.epage%253D901%26rft.atitle%253DSelective%2Bloss%2Bof%2Blin%2Bgenes%2Bfrom%2Bhexachlorocyclohexane-degrading%2BPseudomonas%2Baeruginosa%2BITRC-5%2Bunder%2Bdifferent%2Bgrowth%2Bconditions.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00253-007-1056-z%26rft_id%253Dinfo%253Apmid%252F17602219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00253-007-1056-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17602219&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.68" data-doi="10.1128/AEM.70.11.6650-6656.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suar</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Meer</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawlor</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holliger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lal</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Dynamics of multiple <em>lin</em> gene expression in <em>Sphingomonas paucimobilis</em> B90A in response to different hexachlorocyclohexane isomers.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">6650</span>–<span class="cit-lpage">6656</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft.stitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft.aulast%253DSuar%26rft.auinit1%253DM.%26rft.volume%253D70%26rft.issue%253D11%26rft.spage%253D6650%26rft.epage%253D6656%26rft.atitle%253DDynamics%2Bof%2BMultiple%2Blin%2BGene%2BExpression%2Bin%2BSphingomonas%2Bpaucimobilis%2BB90A%2Bin%2BResponse%2Bto%2BDifferent%2BHexachlorocyclohexane%2BIsomers%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.70.11.6650-6656.2004%26rft_id%253Dinfo%253Apmid%252F15528530%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI3MC8xMS82NjUwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEwODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.69" data-doi="10.1128/AEM.71.12.8514-8518.2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suar</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hauser</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poiger</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buser</span>  <span class="cit-name-given-names">H. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Enantioselective transformation of alpha-hexachlorocyclohexane by the dehydrochlorinases LinA1 and LinA2 from the soil bacterium <em>Sphingomonas paucimobilis</em> B90A.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">8514</span>–<span class="cit-lpage">8518</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft.stitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft.aulast%253DSuar%26rft.auinit1%253DM.%26rft.volume%253D71%26rft.issue%253D12%26rft.spage%253D8514%26rft.epage%253D8518%26rft.atitle%253DEnantioselective%2BTransformation%2Bof%2B%257Balpha%257D-Hexachlorocyclohexane%2Bby%2Bthe%2BDehydrochlorinases%2BLinA1%2Band%2BLinA2%2Bfrom%2Bthe%2BSoil%2BBacterium%2BSphingomonas%2Bpaucimobilis%2BB90A%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.71.12.8514-8518.2005%26rft_id%253Dinfo%253Apmid%252F16332842%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI3MS8xMi84NTE0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEwODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.70" data-doi="10.1073/pnas.92.19.8985"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sullivan</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patrick</span>  <span class="cit-name-given-names">H. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowther</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronson</span>  <span class="cit-name-given-names">C. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Nodulating strains of <em>Rhizobium loti</em> arise through chromosomal symbiotic gene transfer in the environment.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">8985</span>–<span class="cit-lpage">8989</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSullivan%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D92%26rft.issue%253D19%26rft.spage%253D8985%26rft.epage%253D8989%26rft.atitle%253DNodulating%2Bstrains%2Bof%2BRhizobium%2Bloti%2Barise%2Bthrough%2Bchromosomal%2Bsymbiotic%2Bgene%2Btransfer%2Bin%2Bthe%2Benvironment.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.92.19.8985%26rft_id%253Dinfo%253Apmid%252F7568057%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTIvMTkvODk4NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMDgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.71" data-doi="10.1128/JB.184.11.3086-3095.2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sullivan</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trzebiatowski</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cruickshank</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gouzy</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Comparative sequence analysis of the symbiosis island of <em>Mesorhizobium loti</em> strain R7A.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">184</span>: <span class="cit-fpage">3086</span>–<span class="cit-lpage">3095</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DSullivan%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D184%26rft.issue%253D11%26rft.spage%253D3086%26rft.epage%253D3095%26rft.atitle%253DComparative%2BSequence%2BAnalysis%2Bof%2Bthe%2BSymbiosis%2BIsland%2Bof%2BMesorhizobium%2Bloti%2BStrain%2BR7A%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.184.11.3086-3095.2002%26rft_id%253Dinfo%253Apmid%252F12003951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE4NC8xMS8zMDg2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEwODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.72" data-doi="10.1271/bbb.100652"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tabata</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ito</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohtsubo</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The <em>lin</em> genes for γ-hexachlorocyclohexane degradation in <em>Sphingomonas sp</em>. MM-1 proved to be dispersed across multiple plasmids.</span> <abbr class="cit-jnl-abbrev">Biosci. Biotechnol. Biochem.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">466</span>–<span class="cit-lpage">472</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioscience%252C%2BBiotechnology%252C%2Band%2BBiochemistry%26rft.stitle%253DBioscience%252C%2BBiotechnology%252C%2Band%2BBiochemistry%26rft.aulast%253DTabata%26rft.auinit1%253DM.%26rft.volume%253D75%26rft.issue%253D3%26rft.spage%253D466%26rft.epage%253D472%26rft.atitle%253DThe%2Blin%2Bgenes%2Bfor%2B%25CE%25B3-hexachlorocyclohexane%2Bdegradation%2Bin%2BSphingomonas%2Bsp.%2BMM-1%2Bproved%2Bto%2Bbe%2Bdispersed%2Bacross%2Bmultiple%2Bplasmids.%26rft_id%253Dinfo%253Adoi%252F10.1271%252Fbbb.100652%26rft_id%253Dinfo%253Apmid%252F21389627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1271/bbb.100652&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21389627&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.73" data-doi="10.1128/genomeA.00247-13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tabata</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohtsubo</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohhata</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsuda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagata</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Complete genome sequence of the γ-hexachlorocyclohexane-degrading bacterium <em>Sphingomonas sp</em>. strain MM-1.</span> <abbr class="cit-jnl-abbrev">Genome Announc.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">e00247</span>–<span class="cit-lpage">e13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BAnnounc.%26rft.volume%253D1%26rft.spage%253De00247%26rft_id%253Dinfo%253Adoi%252F10.1128%252FgenomeA.00247-13%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/genomeA.00247-13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.74" data-doi="10.1099/mic.0.055335-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takeo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maeda</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maeda</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishiyama</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kitamura</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Two identical nonylphenol monooxygenase genes linked to IS<em>6100</em> and some putative insertion sequence elements in <em>Sphingomonas</em> sp. NP5.</span> <abbr class="cit-jnl-abbrev">Microbiology</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">1796</span>–<span class="cit-lpage">1807</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fmic.0.055335-0%26rft_id%253Dinfo%253Apmid%252F22504436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/mic.0.055335-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22504436&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.75" data-doi="10.1128/jb.178.20.6049-6055.1996"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berger</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacquier</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernillon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baud-Grasset</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Isolation and characterization of a novel gamma-hexachlorocyclohexane-degrading bacterium.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">6049</span>–<span class="cit-lpage">6055</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DThomas%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D178%26rft.issue%253D20%26rft.spage%253D6049%26rft.epage%253D6055%26rft.atitle%253DIsolation%2Band%2Bcharacterization%2Bof%2Ba%2Bnovel%2Bgamma-hexachlorocyclohexane-degrading%2Bbacterium.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.178.20.6049-6055.1996%26rft_id%253Dinfo%253Apmid%252F8830705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE3OC8yMC82MDQ5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEwODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.76" data-doi="10.1074/jbc.M007452200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trantirek</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hynková</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagata</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murzin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ansorgová</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Reaction mechanism and stereochemistry of gamma-hexachlorocyclohexane dehydrochlorinase LinA.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">276</span>: <span class="cit-fpage">7734</span>–<span class="cit-lpage">7740</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M007452200%26rft_id%253Dinfo%253Apmid%252F11099497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjExOiIyNzYvMTEvNzczNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMDgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.77" data-doi="10.1007/s11356-010-0417-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vijgen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abhilash</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lal</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forter</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Hexachlorocyclohexane (HCH) as new Stockholm Convention POPs—a global perspective on the management of Lindane and its waste isomers.</span> <abbr class="cit-jnl-abbrev">Environ. Sci. Pollut. Res. Int.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">152</span>–<span class="cit-lpage">162</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnvironmental%2BScience%2Band%2BPollution%2BResearch%2BInternational%26rft.stitle%253DEnvironmental%2BScience%2Band%2BPollution%2BResearch%2BInternational%26rft.aulast%253DVijgen%26rft.auinit1%253DJ.%26rft.volume%253D18%26rft.issue%253D2%26rft.spage%253D152%26rft.epage%253D162%26rft.atitle%253DHexachlorocyclohexane%2B%2528HCH%2529%2Bas%2Bnew%2BStockholm%2BConvention%2BPOPs--a%2Bglobal%2Bperspective%2Bon%2Bthe%2Bmanagement%2Bof%2BLindane%2Band%2Bits%2Bwaste%2Bisomers.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11356-010-0417-9%26rft_id%253Dinfo%253Apmid%252F21104204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11356-010-0417-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21104204&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286762200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.78" data-doi="10.1099/00221287-144-7-1963"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weaden</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dyson</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Transposon mutagenesis with IS<em>6100</em> in the avermectin producer <em>Streptomyces avermitilis</em></span>. <abbr class="cit-jnl-abbrev">Microbiol.</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">1963</span>–<span class="cit-lpage">1970</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft.stitle%253DMicrobiology%26rft.aulast%253DWeaden%26rft.auinit1%253DJ.%26rft.volume%253D144%26rft.issue%253D7%26rft.spage%253D1963%26rft.epage%253D1970%26rft.atitle%253DTransposon%2Bmutagenesis%2Bwith%2BIS6100%2Bin%2Bthe%2Bavermectin-producer%2BStreptomyces%2Bavermitilis%26rft_id%253Dinfo%253Adoi%252F10.1099%252F00221287-144-7-1963%26rft_id%253Dinfo%253Apmid%252F9695929%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/00221287-144-7-1963&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9695929&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000075000100028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.79" data-doi="10.1021/es203109q"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wöhrnschimmel</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tay</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Waldow</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hung</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Comparative assessment of the global fate of α- and β-hexachlorocyclohexane before and after phase-out.</span> <abbr class="cit-jnl-abbrev">Environ. Sci. Technol.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">2047</span>–<span class="cit-lpage">2054</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnvironmental%2BScience%2B%2526%2BTechnology%252C%2BES%2B%2526%2BT%26rft.stitle%253DEnvironmental%2BScience%2B%2526%2BTechnology%252C%2BES%2B%2526%2BT%26rft.aulast%253DWohrnschimmel%26rft.auinit1%253DH.%26rft.volume%253D46%26rft.issue%253D4%26rft.spage%253D2047%26rft.epage%253D2054%26rft.atitle%253DComparative%2Bassessment%2Bof%2Bthe%2Bglobal%2Bfate%2Bof%2B%25CE%25B1-%2Band%2B%25CE%25B2-hexachlorocyclohexane%2Bbefore%2Band%2Bafter%2Bphase-out.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fes203109q%26rft_id%253Dinfo%253Apmid%252F22320168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/es203109q&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22320168&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.80" data-doi="10.1038/nrmicro2382"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wozniak</span>  <span class="cit-name-given-names">R. A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waldor</span>  <span class="cit-name-given-names">M. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Integrative and conjugative elements: mosaic mobile genetic elements enabling dynamic lateral gene flow.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">552</span>–<span class="cit-lpage">563</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DWozniak%26rft.auinit1%253DR.%2BA.%26rft.volume%253D8%26rft.issue%253D8%26rft.spage%253D552%26rft.epage%253D563%26rft.atitle%253DIntegrative%2Band%2Bconjugative%2Belements%253A%2Bmosaic%2Bmobile%2Bgenetic%2Belements%2Benabling%2Bdynamic%2Blateral%2Bgene%2Bflow.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2382%26rft_id%253Dinfo%253Apmid%252F20601965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro2382&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20601965&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279987200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.81" data-doi="10.1111/j.1462-2920.2007.01350.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Analysis of the role of LinA and LinB in biodegradation of delta-hexachlorocyclohexane.</span> <abbr class="cit-jnl-abbrev">Environ. Microbiol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">2331</span>–<span class="cit-lpage">2340</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnvironmental%2BMicrobiology%2B%2528Print%2529%26rft.stitle%253DEnvironmental%2BMicrobiology%2B%2528Print%2529%26rft.aulast%253DWu%26rft.auinit1%253DJ.%26rft.volume%253D9%26rft.issue%253D9%26rft.spage%253D2331%26rft.epage%253D2340%26rft.atitle%253DAnalysis%2Bof%2Bthe%2Brole%2Bof%2BLinA%2Band%2BLinB%2Bin%2Bbiodegradation%2Bof%2Bdelta-hexachlorocyclohexane.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1462-2920.2007.01350.x%26rft_id%253Dinfo%253Apmid%252F17686029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1462-2920.2007.01350.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17686029&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1081.82" data-doi="10.1263/jbb.104.521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yasuhira</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uedo</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takeo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kato</span>  <span class="cit-name-given-names">D.-I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Negoro</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic organization of nylon-oligomer-degrading enzymes from alkalophilic bacterium, <em>Agromyces</em> sp. KY5R.</span> <abbr class="cit-jnl-abbrev">J. Biosci. Bioeng.</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">521</span>–<span class="cit-lpage">524</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BBiosci%2BBioeng%26rft.aulast%253DYasuhira%26rft.auinit1%253DK.%26rft.volume%253D104%26rft.issue%253D6%26rft.spage%253D521%26rft.epage%253D524%26rft.atitle%253DGenetic%2Borganization%2Bof%2Bnylon-oligomer-degrading%2Benzymes%2Bfrom%2Balkalophilic%2Bbacterium%252C%2BAgromyces%2Bsp.%2BKY5R.%26rft_id%253Dinfo%253Adoi%252F10.1263%252Fjbb.104.521%26rft_id%253Dinfo%253Apmid%252F18215642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1263/jbb.104.521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18215642&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1081.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/6/1081.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/6/1075" title="Gene Network Reconstruction by Integration of Prior Biological Knowledge" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/6/1095" title="BAC-End Sequence-Based SNP Mining in Allotetraploid Cotton (Gossypium) Utilizing Resequencing Data, Phylogenetic Inferences, and Perspectives for Genetic Mapping" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/6"><span>Volume 5 Issue 6, June 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/6.cover-source.jpg?itok=NSNFWJbY" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1081" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-geKD9v3bWzP5h-IaZ1y3T4dZunnRrmz2w7KbhL070Kw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/6/1081" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/6/1081" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/6/1081" class="active" data-icon-position="" data-hide-link-title="0">Insights into Ongoing Evolution of the Hexachlorocyclohexane Catabolic Pathway from Comparative Genomics of Ten Sphingomonadaceae Strains</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459917" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-2L1g0d5i9n74YPo3__z6r3hfB1wDTfIx3d1ns7ktIbc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/6/1081.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/6/1081" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-3QpHbsPTEptBLrfsBf-W0JliS_oLxkCxCpJxBxCI4C8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/6/1081" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/6/1081.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459917" id="node459917--2" data-pisa="ggg;5/6/1081" data-pisa-master="ggg;g3.114.015933" data-apath="/ggg/5/6/1081.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Insights into Ongoing Evolution of the Hexachlorocyclohexane Catabolic Pathway from Comparative Genomics of Ten Sphingomonadaceae Strains</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Stephen L. Pearce</span>, <span class="highwire-citation-author" data-delta="1">John G. Oakeshott</span> and <span class="highwire-citation-author" data-delta="2">Gunjan Pandey</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1081-1094;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015933 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459917--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Stephen L. Pearce </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Ecosystem Sciences, Acton, ACT-2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BL.%2BPearce%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pearce%20SL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BL.%2BPearce%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John G. Oakeshott </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Ecosystem Sciences, Acton, ACT-2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BG.%2BOakeshott%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oakeshott%20JG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BG.%2BOakeshott%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Gunjan Pandey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Ecosystem Sciences, Acton, ACT-2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gunjan%2BPandey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pandey%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGunjan%2BPandey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gunjan.pandey@csiro.au" class="" data-icon-position="" data-hide-link-title="0">gunjan.pandey@csiro.au</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/6/1081" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/6/1081&amp;title=Insights%20into%20Ongoing%20Evolution%20of%20the%20Hexachlorocyclohexane%20Catabolic%20Pathway%20from%20Comparative%20Genomics%20of%20Ten%20Sphingomonadaceae%20Strains" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/6/1081&amp;title=Insights%20into%20Ongoing%20Evolution%20of%20the%20Hexachlorocyclohexane%20Catabolic%20Pathway%20from%20Comparative%20Genomics%20of%20Ten%20Sphingomonadaceae%20Strains" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/6/1081&amp;title=Insights%20into%20Ongoing%20Evolution%20of%20the%20Hexachlorocyclohexane%20Catabolic%20Pathway%20from%20Comparative%20Genomics%20of%20Ten%20Sphingomonadaceae%20Strains" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/6/1081&amp;text=Insights%20into%20Ongoing%20Evolution%20of%20the%20Hexachlorocyclohexane%20Catabolic%20Pathway%20from%20Comparative%20Genomics%20of%20Ten%20Sphingomonadaceae%20Strains" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/6/1081&amp;title=Insights%20into%20Ongoing%20Evolution%20of%20the%20Hexachlorocyclohexane%20Catabolic%20Pathway%20from%20Comparative%20Genomics%20of%20Ten%20Sphingomonadaceae%20Strains" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/6/1081&amp;t=Insights%20into%20Ongoing%20Evolution%20of%20the%20Hexachlorocyclohexane%20Catabolic%20Pathway%20from%20Comparative%20Genomics%20of%20Ten%20Sphingomonadaceae%20Strains" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/6/1081&amp;title=Insights%20into%20Ongoing%20Evolution%20of%20the%20Hexachlorocyclohexane%20Catabolic%20Pathway%20from%20Comparative%20Genomics%20of%20Ten%20Sphingomonadaceae%20Strains" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/6/1081&amp;title=Insights%20into%20Ongoing%20Evolution%20of%20the%20Hexachlorocyclohexane%20Catabolic%20Pathway%20from%20Comparative%20Genomics%20of%20Ten%20Sphingomonadaceae%20Strains" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459917/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459917" id="node459917--3" data-pisa="ggg;5/6/1081" data-pisa-master="ggg;g3.114.015933" data-apath="/ggg/5/6/1081.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Insights into Ongoing Evolution of the Hexachlorocyclohexane Catabolic Pathway from Comparative Genomics of Ten Sphingomonadaceae Strains</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Stephen L. Pearce</span>, <span class="highwire-citation-author" data-delta="1">John G. Oakeshott</span> and <span class="highwire-citation-author" data-delta="2">Gunjan Pandey</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1081-1094;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015933 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459917--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Stephen L. Pearce </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Ecosystem Sciences, Acton, ACT-2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BL.%2BPearce%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pearce%20SL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BL.%2BPearce%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John G. Oakeshott </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Ecosystem Sciences, Acton, ACT-2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BG.%2BOakeshott%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oakeshott%20JG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BG.%2BOakeshott%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Gunjan Pandey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Ecosystem Sciences, Acton, ACT-2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gunjan%2BPandey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pandey%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGunjan%2BPandey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gunjan.pandey@csiro.au" class="" data-icon-position="" data-hide-link-title="0">gunjan.pandey@csiro.au</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1081&amp;rft.epage=1094&amp;rft.atitle=Insights%20into%20Ongoing%20Evolution%20of%20the%20Hexachlorocyclohexane%20Catabolic%20Pathway%20from%20Comparative%20Genomics%20of%20Ten%20Sphingomonadaceae%20Strains&amp;rft.volume=5&amp;rft.issue=6&amp;rft.date=2015-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Pearce%2C+Stephen+L.&amp;rft.au=Oakeshott%2C+John+G.&amp;rft.au=Pandey%2C+Gunjan"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459917/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459917/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459917/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459917/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459917/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459917/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459917/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459917/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459917/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459917/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459917/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459917/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/6/1081" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/6/1081.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/6/1081.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/6/1081.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/6/1081.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
