<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/6/1095" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/6/1095.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/6/1095.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/6/1095.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/6/1095.atom" />
<meta name="HW.pisa" content="ggg;5/6/1095" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="BAC-End Sequence-Based SNP Mining in Allotetraploid Cotton (Gossypium) Utilizing Resequencing Data, Phylogenetic Inferences, and Perspectives for Genetic Mapping" />
<meta name="DC.Identifier" content="10.1534/g3.115.017749" />
<meta name="DC.Date" content="2015-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Hulse-Kemp et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="A bacterial artificial chromosome library and BAC-end sequences for cultivated cotton ( Gossypium hirsutum L.) have recently been developed. This report presents genome-wide single nucleotide polymorphism (SNP) mining utilizing resequencing data with BAC-end sequences as a reference by alignment of 12 G. hirsutum L. lines, one G. barbadense L. line, and one G. longicalyx Hutch and Lee line. A total of 132,262 intraspecific SNPs have been developed for G. hirsutum , whereas 223,138 and 470,631 interspecific SNPs have been developed for G. barbadense and G. longicalyx , respectively. Using a set of interspecific SNPs, 11 randomly selected and 77 SNPs that are putatively associated with the homeologous chromosome pair 12 and 26, we mapped 77 SNPs into two linkage groups representing these chromosomes, spanning a total of 236.2 cM in an interspecific F2 population ( G. barbadense 3-79 × G. hirsutum TM-1). The mapping results validated the approach for reliably producing large numbers of both intraspecific and interspecific SNPs aligned to BAC-ends. This will allow for future construction of high-density integrated physical and genetic maps for cotton and other complex polyploid genomes. The methods developed will allow for future Gossypium resequencing data to be automatically genotyped for identified SNPs along the BAC-end sequence reference for anchoring sequence assemblies and comparative studies." />
<meta name="DC.Contributor" content="Amanda M. Hulse-Kemp" />
<meta name="DC.Contributor" content="Hamid Ashrafi" />
<meta name="DC.Contributor" content="Kevin Stoffel" />
<meta name="DC.Contributor" content="Xiuting Zheng" />
<meta name="DC.Contributor" content="Christopher A. Saski" />
<meta name="DC.Contributor" content="Brian E. Scheffler" />
<meta name="DC.Contributor" content="David D. Fang" />
<meta name="DC.Contributor" content="Z. Jeffrey Chen" />
<meta name="DC.Contributor" content="Allen Van Deynze" />
<meta name="DC.Contributor" content="David M. Stelly" />
<meta name="article:published_time" content="2015-06-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="BAC-End Sequence-Based SNP Mining in Allotetraploid Cotton (Gossypium) Utilizing Resequencing Data, Phylogenetic Inferences, and Perspectives for Genetic Mapping" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;A bacterial artificial chromosome library and BAC-end sequences for cultivated cotton (&lt;i&gt;Gossypium hirsutum&lt;/i&gt; L.) have recently been developed. This report presents genome-wide single nucleotide polymorphism (SNP) mining utilizing resequencing data with BAC-end sequences as a reference by alignment of 12 &lt;i&gt;G. hirsutum&lt;/i&gt; L. lines, one &lt;i&gt;G. barbadense&lt;/i&gt; L. line, and one &lt;i&gt;G. longicalyx&lt;/i&gt; Hutch and Lee line. A total of 132,262 intraspecific SNPs have been developed for &lt;i&gt;G. hirsutum&lt;/i&gt;, whereas 223,138 and 470,631 interspecific SNPs have been developed for &lt;i&gt;G. barbadense&lt;/i&gt; and &lt;i&gt;G. longicalyx&lt;/i&gt;, respectively. Using a set of interspecific SNPs, 11 randomly selected and 77 SNPs that are putatively associated with the homeologous chromosome pair 12 and 26, we mapped 77 SNPs into two linkage groups representing these chromosomes, spanning a total of 236.2 cM in an interspecific F2 population (&lt;i&gt;G. barbadense&lt;/i&gt; 3-79 × &lt;i&gt;G. hirsutum&lt;/i&gt; TM-1). The mapping results validated the approach for reliably producing large numbers of both intraspecific and interspecific SNPs aligned to BAC-ends. This will allow for future construction of high-density integrated physical and genetic maps for cotton and other complex polyploid genomes. The methods developed will allow for future &lt;i&gt;Gossypium&lt;/i&gt; resequencing data to be automatically genotyped for identified SNPs along the BAC-end sequence reference for anchoring sequence assemblies and comparative studies.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/06/01" />
<meta name="citation_mjid" content="ggg;5/6/1095" />
<meta name="citation_id" content="5/6/1095" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/6/1095" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/6/1095.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/6/1095.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/6/1095.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.017749" />
<meta name="citation_pmid" content="25858960" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1095" />
<meta name="citation_lastpage" content="1105" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Amanda M. Hulse-Kemp" />
<meta name="citation_author_institution" content="Department of Soil and Crop Sciences, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author_institution" content="Interdisciplinary Genetics Program, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9670-9433" />
<meta name="citation_author" content="Hamid Ashrafi" />
<meta name="citation_author_institution" content="Seed Biotechnology Center, University of California, Davis, California 95616" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2094-2367" />
<meta name="citation_author" content="Kevin Stoffel" />
<meta name="citation_author_institution" content="Seed Biotechnology Center, University of California, Davis, California 95616" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3468-4119" />
<meta name="citation_author" content="Xiuting Zheng" />
<meta name="citation_author_institution" content="Department of Soil and Crop Sciences, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8574-4708" />
<meta name="citation_author" content="Christopher A. Saski" />
<meta name="citation_author_institution" content="Clemson University Genomics Institute, Clemson University, Clemson, South Carolina 29634" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2780-4274" />
<meta name="citation_author" content="Brian E. Scheffler" />
<meta name="citation_author_institution" content="USDA-ARS, Genomics and Bioinformatics Research Unit, Stoneville, Mississippi 38766" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1968-8952" />
<meta name="citation_author" content="David D. Fang" />
<meta name="citation_author_institution" content="USDA-ARS, Cotton Fiber Bioscience Research Unit, New Orleans, Louisiana 70124" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0036-5459" />
<meta name="citation_author" content="Z. Jeffrey Chen" />
<meta name="citation_author_institution" content="Department of Molecular Biosciences, Center for Computational Biology and Bioinformatics, and Institute for Cellular and Molecular Biology, The University of Texas, Austin, Texas 78712" />
<meta name="citation_author" content="Allen Van Deynze" />
<meta name="citation_author_institution" content="Seed Biotechnology Center, University of California, Davis, California 95616" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2093-0577" />
<meta name="citation_author" content="David M. Stelly" />
<meta name="citation_author_institution" content="Department of Soil and Crop Sciences, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author_institution" content="Interdisciplinary Genetics Program, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author_email" content="stelly@tamu.edu" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=The Plant Genome;citation_author=H. Ashrafi;citation_author=A. M. Hulse-Kemp;citation_author=F. Wang;citation_author=S. S. Yang;citation_author=X. Guan;citation_title=A Long-Read Transcriptome Assembly of Cotton (Gossypium hirsutum L.) and Intraspecific Single Nucleotide Polymorphism Discovery.;citation_pages=1-14;citation_volume=8;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. Blenda;citation_author=D. D. Fang;citation_author=J. F. Rami;citation_author=O. Garsmeur;citation_author=F. Luo;citation_title=A high density consensus genetic map of tetraploid cotton that integrates multiple component maps through molecular marker redundancy check.;citation_pages=e45739-e45739;citation_volume=7;citation_year=2012;citation_issue=9;citation_pmid=23029214;citation_doi=10.1371/journal.pone.0045739" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=A. Bohra;citation_author=A. Dubey;citation_author=R. K. Saxena;citation_author=R. V. Penmetsa;citation_author=K. N. Poornima;citation_title=Analysis of BAC-end sequences (BESs) and development of BES-SSR markers for genetic mapping and hybrid purity assessment in pigeonpea (Cajanus spp.).;citation_pages=56-56;citation_volume=11;citation_year=2011;citation_pmid=21447154;citation_doi=10.1186/1471-2229-11-56" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. L. Byers;citation_author=D. B. Harker;citation_author=S. M. Yourstone;citation_author=P. J. Maughan;citation_author=J. A. Udall;citation_title=Development and mapping of SNP assays in allotetraploid cotton.;citation_pages=1201-1214;citation_volume=124;citation_year=2012;citation_issue=7;citation_pmid=22252442;citation_doi=10.1007/s00122-011-1780-8" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=A. Ching;citation_author=K. S. Caldwell;citation_author=M. Jung;citation_author=M. Dolan;citation_author=O. S. Smith;citation_title=SNP frequency, haplotype structure and linkage disequilibrium in elite maize inbred lines.;citation_pages=19-19;citation_volume=3;citation_year=2002;citation_pmid=12366868;citation_doi=10.1186/1471-2156-3-19" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_volume=6;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=D. D. Fang;citation_author=L. L. Hinze;citation_author=R. G. Percy;citation_author=P. Li;citation_author=D. Deng;citation_title=A microsatellite-based genome-wide analysis of genetic diversity and linkage disequilibrium in Upland cotton (Gossypium hirsutum L.) cultivars from major cotton-growing countries.;citation_pages=391-401;citation_volume=191;citation_year=2013;citation_doi=10.1007/s10681-013-0886-2" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=J. E. Frelichowski;citation_author=M. B. Palmer;citation_author=D. Main;citation_author=J. P. Tomkins;citation_author=R. G. Cantrell;citation_title=Cotton genome mapping with new microsatellites from Acala 'Maxxa' BAC-ends.;citation_pages=479-491;citation_volume=275;citation_year=2006;citation_issue=5;citation_pmid=16501995;citation_doi=10.1007/s00438-006-0106-z" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=J. Goecks;citation_author=A. Nekrutenko;citation_author=J. Taylor;citation_title=Galaxy: a comprehensive approach for supporting accessible, reproducible, and transparent computational research in the life sciences.;citation_pages=R86-R86;citation_volume=11;citation_year=2010;citation_issue=8;citation_pmid=20738864;citation_doi=10.1186/gb-2010-11-8-r86" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=D. Gordon;citation_author=C. Abajian;citation_author=P. Green;citation_title=Consed: A Graphical Tool for Sequence Finishing;citation_pages=195-202;citation_volume=8;citation_year=1998;citation_issue=3;citation_pmid=9521923;citation_doi=10.1101/gr.8.3.195" />
<meta name="citation_reference" content="citation_title=Linkage map construction and quantitative trait locus analysis of agronomic and fiber quality traits in cotton;citation_author=M. A. Gore;citation_author=D. D. Fang;citation_author=J. A. Poland;citation_author=J. Zhang;citation_author=R. G. Percy;citation_volume=7;citation_year=2014;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=Y. Han;citation_author=D. Chagne;citation_author=K. Gasic;citation_author=E. H. Rikkerink;citation_author=J. E. Beever;citation_title=BAC-end sequence-based SNPs and Bin mapping for rapid integration of physical and genetic maps in apple.;citation_pages=282-288;citation_volume=93;citation_year=2009;citation_issue=3;citation_pmid=19059473;citation_doi=10.1016/j.ygeno.2008.11.005" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=C. N. Hansey;citation_author=B. Vaillancourt;citation_author=R. S. Sekhon;citation_author=N. De Leon;citation_author=S. M. Kaeppler;citation_title=Maize (Zea mays L.) genome diversity as revealed by RNA-sequencing.;citation_pages=e33071-e33071;citation_volume=7;citation_year=2012;citation_issue=3;citation_pmid=22438891;citation_doi=10.1371/journal.pone.0033071" />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot. (Lond.);citation_author=B. Hendrix;citation_author=J. M. Stewart;citation_title=Estimation of the nuclear DNA content of Gossypium species.;citation_volume=95;citation_year=2005;citation_pmid=15701660;citation_doi=10.1093/aob/mci078" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=A. M. Hulse-Kemp;citation_author=H. Ashrafi;citation_author=X. Zheng;citation_author=F. Wang;citation_author=K. A. Hoegenauer;citation_title=Development and bin mapping of gene-associated interspecific SNPs for cotton (Gossypium hirsutum L.) introgression breeding efforts.;citation_pages=945;citation_volume=15;citation_year=2014;citation_pmid=25359292;citation_doi=10.1186/1471&amp;ndash;2164&amp;ndash;15&amp;ndash;945" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=D. L. Hyten;citation_author=S. B. Cannon;citation_author=Q. J. Song;citation_author=N. Weeks;citation_author=E. W. Fickus;citation_title=High-throughput SNP discovery through deep resequencing of a reduced representation library to anchor and orient scaffolds in the soybean whole genome sequence.;citation_pages=38-38;citation_volume=11;citation_year=2010;citation_pmid=20078886;citation_doi=10.1186/1471-2164-11-38" />
<meta name="citation_reference" content="citation_title=Detection, validation and application of genotyping-by-sequencing based single nucleotide polymorphisms in upland cotton (Gossypium hirsutum L.);citation_author=M. S. Islam;citation_author=G. N. Thyssen;citation_author=J. N. Jenkins;citation_author=D. D. Fang;citation_volume=8;citation_year=2015;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_title=Multipoint maximum likelihood mapping in a full-sib family of an outbreeding species.;citation_pages=343-349;citation_volume=93;citation_year=2011;citation_issue=5;citation_pmid=21878144;citation_doi=10.1017/S0016672311000279" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Kohel;citation_author=T. Richmond;citation_author=C. Lewis;citation_title=Texas Marker-1. Description of a Genetic Standard for Gossypium hirsutum L.;citation_pages=670-671;citation_volume=10;citation_year=1970;citation_issue=6;citation_doi=10.2135/cropsci1970.0011183X001000060019x" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. M. Lam;citation_author=X. Xu;citation_author=X. Liu;citation_author=W. B. Chen;citation_author=G. H. Yang;citation_title=Resequencing of 31 wild and cultivated soybean genomes identifies patterns of genetic diversity and selection.;citation_pages=1053-1059;citation_volume=42;citation_year=2010;citation_issue=12;citation_pmid=21076406;citation_doi=10.1038/ng.715" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=F. G. Li;citation_author=G. Y. Fan;citation_author=K. B. Wang;citation_author=F. M. Sun;citation_author=Y. L. Yuan;citation_title=Genome sequence of the cultivated cotton Gossypium arboreum.;citation_pages=567-572;citation_volume=46;citation_year=2014;citation_pmid=24836287;citation_doi=10.1038/ng.2987" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Plant Genomics;citation_author=J. Mammadov;citation_author=R. Aggarwal;citation_author=R. Buyyarapu;citation_author=S. Kumpatla;citation_title=SNP markers and their impact on plant breeding.;citation_pages=728398;citation_volume=2012;citation_year=2012;citation_pmid=23316221" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=P. Ollitrault;citation_author=J. Terol;citation_author=A. Garcia-Lor;citation_author=A. Berard;citation_author=A. Chauveau;citation_title=SNP mining in C. clementina BAC end sequences; transferability in the Citrus genus (Rutaceae), phylogenetic inferences and perspectives for genetic mapping.;citation_pages=13-13;citation_volume=13;citation_year=2012;citation_pmid=22233093;citation_doi=10.1186/1471-2164-13-13" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=A. H. Paterson;citation_author=J. F. Wendel;citation_author=H. Gundlach;citation_author=H. Guo;citation_author=J. Jenkins;citation_title=Repeated polyploidization of Gossypium genomes and the evolution of spinnable cotton fibres.;citation_pages=423-427;citation_volume=492;citation_year=2012;citation_pmid=23257886;citation_doi=10.1038/nature11798" />
<meta name="citation_reference" content="citation_journal_title=Can. J. Genet. Cytol.;citation_author=L. Phillips;citation_author=M. Strickland;citation_title=The cytology of a hybrid between Gossypium hirsutum and G. longicalyx.;citation_pages=91-95;citation_volume=8;citation_year=1966;citation_doi=10.1139/g66-011" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_title=Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.;citation_author=J. A. Poland;citation_author=P. J. Brown;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_volume=7;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=K. M. Rai;citation_author=S. K. Singh;citation_author=A. Bhardwaj;citation_author=V. Kumar;citation_author=D. Lakhwani;citation_title=Large-scale resource development in Gossypium hirsutum L. by 454 sequencing of genic-enriched libraries from six diverse genotypes.;citation_pages=953-963;citation_volume=11;citation_year=2013;citation_pmid=23782852;citation_doi=10.1111/pbi.12088" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=Y. J. Shen;citation_author=H. Jiang;citation_author=J. P. Jin;citation_author=Z. B. Zhang;citation_author=B. Xi;citation_title=Development of Genome-Wide DNA Polymorphism Database for Map-Based Cloning of Rice Genes;citation_pages=1198-1205;citation_volume=135;citation_year=2004;citation_issue=3;citation_pmid=15266053;citation_doi=10.1104/pp.103.038463" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=A. Van Deynze;citation_author=K. Stoffel;citation_author=M. Lee;citation_author=T. A. Wilkins;citation_author=A. Kozik;citation_title=Sampling nucleotide diversity in cotton.;citation_pages=125-125;citation_volume=9;citation_year=2009;citation_pmid=19840401;citation_doi=10.1186/1471-2229-9-125" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=H. Wang;citation_author=R. V. Penmetsa;citation_author=M. Yuan;citation_author=L. Gong;citation_author=Y. Zhao;citation_title=Development and characterization of BAC-end sequence derived SSRs, and their incorporation into a new higher density genetic map for cultivated peanut (Arachis hypogaea L.).;citation_pages=10-10;citation_volume=12;citation_year=2012;citation_pmid=22260238;citation_doi=10.1186/1471-2229-12-10" />
<meta name="citation_reference" content="citation_title=Genetics and Genomics of Cotton;citation_title=Evolution and natural history of the cotton genus;citation_author=J. F. Wendel;citation_author=C. Brubaker;citation_author=I. Alvarez;citation_author=R. Cronn;citation_author=J. M. Stewart;citation_pages=3-22;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=X. Zheng;citation_author=D. Levine;citation_author=J. Shen;citation_author=S. M. Gogarten;citation_author=C. Laurie;citation_title=A high-performance computing toolset for relatedness and principal component analysis of SNP data.;citation_volume=28;citation_year=2012;citation_pmid=23060615;citation_doi=10.1093/bioinformatics/bts606" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="BAC-End Sequence-Based SNP Mining in Allotetraploid Cotton (Gossypium) Utilizing Resequencing Data, Phylogenetic Inferences, and Perspectives for Genetic Mapping" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
<meta name="twitter:description" content="A bacterial artificial chromosome library and BAC-end sequences for cultivated cotton ( Gossypium hirsutum L.) have recently been developed. This report presents genome-wide single nucleotide polymorphism (SNP) mining utilizing resequencing data with BAC-end sequences as a reference by alignment of 12 G. hirsutum L. lines, one G. barbadense L. line, and one G. longicalyx Hutch and Lee line. A total of 132,262 intraspecific SNPs have been developed for G. hirsutum , whereas 223,138 and 470,631 interspecific SNPs have been developed for G. barbadense and G. longicalyx , respectively. Using a set of interspecific SNPs, 11 randomly selected and 77 SNPs that are putatively associated with the homeologous chromosome pair 12 and 26, we mapped 77 SNPs into two linkage groups representing these chromosomes, spanning a total of 236.2 cM in an interspecific F2 population ( G. barbadense 3-79 × G. hirsutum TM-1). The mapping results validated the approach for reliably producing large numbers of both intraspecific and interspecific SNPs aligned to BAC-ends. This will allow for future construction of high-density integrated physical and genetic maps for cotton and other complex polyploid genomes. The methods developed will allow for future Gossypium resequencing data to be automatically genotyped for identified SNPs along the BAC-end sequence reference for anchoring sequence assemblies and comparative studies." />
<meta name="og-title" property="og:title" content="BAC-End Sequence-Based SNP Mining in Allotetraploid Cotton (Gossypium) Utilizing Resequencing Data, Phylogenetic Inferences, and Perspectives for Genetic Mapping" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/6/1095" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="A bacterial artificial chromosome library and BAC-end sequences for cultivated cotton ( Gossypium hirsutum L.) have recently been developed. This report presents genome-wide single nucleotide polymorphism (SNP) mining utilizing resequencing data with BAC-end sequences as a reference by alignment of 12 G. hirsutum L. lines, one G. barbadense L. line, and one G. longicalyx Hutch and Lee line. A total of 132,262 intraspecific SNPs have been developed for G. hirsutum , whereas 223,138 and 470,631 interspecific SNPs have been developed for G. barbadense and G. longicalyx , respectively. Using a set of interspecific SNPs, 11 randomly selected and 77 SNPs that are putatively associated with the homeologous chromosome pair 12 and 26, we mapped 77 SNPs into two linkage groups representing these chromosomes, spanning a total of 236.2 cM in an interspecific F2 population ( G. barbadense 3-79 × G. hirsutum TM-1). The mapping results validated the approach for reliably producing large numbers of both intraspecific and interspecific SNPs aligned to BAC-ends. This will allow for future construction of high-density integrated physical and genetic maps for cotton and other complex polyploid genomes. The methods developed will allow for future Gossypium resequencing data to be automatically genotyped for identified SNPs along the BAC-end sequence reference for anchoring sequence assemblies and comparative studies." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
    <title>BAC-End Sequence-Based SNP Mining in Allotetraploid Cotton (Gossypium) Utilizing Resequencing Data, Phylogenetic Inferences, and Perspectives for Genetic Mapping | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Bj76jYkr2rJYhqxBGAJrkE-RPpzbvwsae6zCt2aMNk8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459907","apath":"\/ggg\/5\/6\/1095.atom","pisa":"ggg;5\/6\/1095","ac":{"\/ggg\/5\/6\/1095.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/6\/1095.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/6\/1095"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/6\/1095"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/6\/1095.short?cited-by=yes\u0026legid=ggg%3Bg3.115.017749v1":true,"\/content\/5\/6\/1095":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022VNP095ysOwsFg9YvmkEIUXvEjaZ2md0BikMIYjHdQjZduR6R3Vp3jbArJrtrgSm+YhmC2UDJAErGYmoSy9dzH0cwazeWpvUiqjlCDu4HB1QvrtPdGpEcu6gsWgIq1PXYbf\\\\\\\/pzhhRyoEQHIjJ6CTE8gnUy1SiW89B4roo06sNBYVyijZRj7kkzCpjJfj+UtX8Vp1jClYSyKgZl0PGDql+IPJt4v0TZQzGSGAskyyjKDMnFKpfqINjXx5\\\\\\\/\\\\\\\/+JlRUkwqpE2NWmQxkzbUFOZEyd916Z0VqeR8mhnm5QoFrFjbC+201wzPfJK5+AO6b1UaG6u56xhA\\\\\\\/+DL69QsCNJp7NriNwBKPbIhtY4+zktDtHMdsTOO6spokrV5GRbIIeeYJ9SZVYIQeUQjEaQYcksqDT6yksOe+i32TVE0BkQ1GAGb0KqWAz\\\\\\\/S77ioSeUmxX5sBEnMyiblaFAX1kAqjJezWcgKDXmjw9dLO+\\\\\\\/Xe5noweY4dOKPm\\\\\\\/bE4ZwPWmSSjLMXGQdgFm6\\\\\\\/FuhEvMScPHbMZg\\\\\\\/Ky2+2Dn3Kddq6Q0mI+tXNFaJAvzECsPKE9Eo3zhNEtINS9atFLQxw+Tgx6+uosFrbaaML\\\\\\\/K3bfg1ZavqlfR03eQckwtFyNTAGDzo\\\\\\\/dLct2oSrQt6quz9OqNn9xJqkho1OjPzGLSnlskSauD6izkTdQpM5BdLJiIkAxTkkAARd0uWOZPVHQMi8CxejfNAHPhpe1mRxWupjkmqNDtE6MG7ALczZpfzjO8sw4EHYGwOErwl72xmaCTz1nq3gBnL45J5tdDk5vqfZT9QKFO6lOJw5pI0xxA3PLROAwQB3i4qj5kPh72ZAnAuv8A9DlclLCGi0cyOTBA7Ari22htnlzq\\\\\\\/CMbORJAdyFvsgJ+OOHeO3EKOlyMgTFiFYtndGtlPNRMPmnU3yqVsDjIAqCyJTd0W\\\\\\\/KkHj0ZJo6i6uqYFHM94ax0QS8SZmu0QAeFNwR3CVXdLi85FwhmL\\\\\\\/gjqztf0TRxmv8T2TsKFkGu6ltWEwcJV8QAg9s+Kq4PvIzwkPjWI+7p\\\\\\\/9Tljm9LI9I0d0bB+4ZI0MkBuCGyyXs\\\\\\\/xtVwpfHz\\\\\\\/0MYX6EFpBW\\\\\\\/h1tvjvZ3aJW0gbdiWEOeiOsXHITsYj0f6pe1WmH0wIbQh5IAsuVzHqSrYyak2rezPHhnsnSuOxAVqCt8hjxLxH3S+XS5dZ6wUqEV+NTvQc0jtLtcar7bRopbQrl8i1x0S7Cv9e6KWpv2WpLNEZMqpyeSjxLpRVc3VcjZAAQGSnQxP7071j6loRORQ\\\u0022,\\\u0022iv\\\u0022:\\\u0022pX9OSDHpNeumNhv1tuECMg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022836a1736779c51e1686cab91c16c1f260765988913348009ff9d3f654bcd3c20\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022SlbR5EuaF1UG14ytHdI29VAnjmYI8gQIklYsHjrATtTClQjZ3SERj7eLfedyI9Obtio7oSfM7GriGjO4ria+iTiK4tbKfLzp4CQWM74+ScR\\\\\\\/DAV9DCQtXSxOZaFmtNX7Wbh0rCbk81XKNuW9sQmMTMy5XYhno96I06009KciXlaKnSBNOoXZj+OJRLNJs45+IaHtUJ1zkxKWa4YPOKlK6vWnvtAW6eeDN2oEWh9VGe2cV13KFmPE6k3YxpWSTyYSvZvxKSs\\\\\\\/tAqbgvQYYIWQFq9VR2fQoR8ktrM1wuJT7G6KwY9lh102FjsgNxtoZbGPmKCz2I4WC1nKCyC59bSUfmy51tjFfLSXfEO6aTfjICnU4bW9KtuzYbXg+6s8rT9\\\\\\\/00361geb9xWPWGlURuQtFvAWiYrRlEB9B9514Ovs3mAE4CGXQbQ1D2RcKMYaODvhtJpQYJSz0S7Sg9YZleD5vZPq0gI2kMRo5OatiefCLJdyGaCfgVq7z8OpR7MWuobeQFGCmEP+YIKzacqsdmt4V5c9puqqApnTw1PwrZq6VKmBcNrQlJGVtYmkCgFzr5tU5RaokISX13xrJO6gCFsjxRtydMm7vYpf7gsgakk8loQauWztMspL1SYpUcvMxnoa1qgsX7ktot9a+uo3dV+2EvDFqFw1zE123iXMDGiUlzadSZgU9uwbY+b3efrfs3cGhqcw+14XnKjKPlQHXKWg5R4tvBcyeq1+8LpO+jl5V2Hw4oq9Tb1FgC3GSd7ip\\\\\\\/jdExQXLcGKTUDfwDDbaDJOdJXm494ZI6b0lv7LCU\\\\\\\/b8T6P5Sd+VPAPL6EKDZ+B39XT9xChnjCq0qUFQfGyUQLokopnQWXmAjgvUfMY+NkZeyrt6J09SLFqw\\\\\\\/gjlnRsmBXqoxdj8BT+sOjZyVxSqLjCHYapSq6C\\\\\\\/n874+m8jVgkBhXtmzPQfFBKFdxYIxCZWQfEAHtMpDem8zhhS2ebtP\\\\\\\/dSBdj5K3wtsFUU5iW0NPCQreJ45Q7Qb6PHNz89vPRFpiMy3dBBSuvWuysUw1ouXQMHidn21JPEuOG6+aXDe7vA47mdjeWlKMLwCstNYOn8TCj56FL80\\\\\\\/y0ZED7+L9xT4+Dj5N8erd9CbsaJ9IEk89nDshXbhz4wG+qHNVee0lTkRQN9A0i49w35D9FxNR0kJPtdGgXR8ac9pUdHfH\\\\\\\/0Xqe81uyW0LanlHB7KDpNuQJ0CE\\\\\\\/gLeTr91wqNr\\\\\\\/P3mUnoXOjbd9ZwtWxmqxwU1tP8K44ERDV83mFpHmsXWoc75rJCUUa2GcQ++8rjn5An5iuLhR5zV7Gyzdo8v3aJIG4aAeHlpqBi6bEUgNTufqv+wHaIABUKKdDxhA4FhGxu2PmG7L8qQ\\\\\\\/ALGadcBPZgOIM0d3xfR+2Zedt32Tvp5hQv2Gv5Qi54IBbdxajVc+7axWexklJCZ4n+gvzvTUcrnSdan6KaoP2aTkn\\\\\\\/Yo5Y65B+A4d2YbWhWSFb8F67aZ5+Zc0jxekj2D\\\\\\\/Rf3prsLLGEeQNJPydDLqYST2toDhJ9CUnijbBfTO3l0fxxLziPDGTsRpKzXM8yJF645B4S7nN9VdauoP0TL\\\\\\\/CrEpzkUkZwZEdmLwRhuFGJIu6qgZlcMgBbrY0PLAMLx6M8XhwuMtlHReaEpDV\\\\\\\/KwLv56St1r7M3sa02HFlCeql9g0pjJf96Var6iWV+TBBC0BOUBzXnCCuQkECC6\\\\\\\/Gmgh1O75WSDADnWVoZeF+UXuDHin4az+ZxzDKmhWgi735TMz8fT80ZowPCj3DNRJJEh9TZbGJxHClghSx0tLGJYC\\\\\\\/MW4In780mwllvn8rfX8KysW0ObLI0\\\\\\\/eVUrjc\\\\\\\/6l8ebCBJ7FwkxpZA5DSNpAsZ7kSTQEWEigDFF2t1X\\\\\\\/0mVyRvc\\\\\\\/NmEkqfWYuwHgN8MIh3McDGGjhra4exx\\\\\\\/Rw9wfCaG+m4ONcPFwaNapdtnvLQNE922CVBigCwt05YZqJhF1fxUE3WnR8YzsTnIzfOAcnDKcOt+GBnJoeVww+IhIoPMeqsjO7OBa8BkDY6hZcnHzvSOqkrFdA+O2c18GUIsagVoxxpbzvPFBECwnbkI3pKvMXcCqFEfFgi6iYmp3rX6nK+Rh8j473YRm21YTX9GV71NYcQCPzgQnutsSF6unxyLzwIjijq\\\\\\\/c5fB71LlyUGeV+v4t\\\\\\\/C8W5qQkoVhKA9kBLw9ta9EV+wm7Ald0detdyLqKKDiZWQBA9+mBFINk3vfW3c8HSNXhoEyTRwC459qiTg1AMVCTZIwPXph5oQP441R0OLMdjAU0tLCxphvkY3yKZRCBA24olvybiwXsbq0AFh+7wvfZzk769clxcVSfZZua6MG1uLFAuIixfVR7xsn88Z76RlerOyH2INft0ZfUBtoDrzocKBCeO6AJnH55icdbJs6qo8HxlilTZpbkHE+PzuyJhoi77nPrUkiZh1F9aO5kY0GXX9WK9zmKAEUyGiFyJVFpr+mCFtaUK\\\\\\\/qj5ouayprlYmBeEahvfRUf9ueKs6lIIJ69dWGqkqRwgFtHAbJjXLzVC7Rf\\\\\\\/Fg4Y+hlpytpbTCNdjtRhm6BH\\\\\\\/AVhk3EqSidXN4LKCwE5TEeCXG6VNx09RvCdEptG+mVxGEZhwM5jfIYKD4gG\\\\\\\/GOoiCt3PNJf4Req7U+auhLZ3DwDydsPjQOWRZq2kZgcN83fze39GhGDnbhGbKDNY44jX79agmNdyPbij\\\\\\\/JTn7W9ViDSbN3qJZ0XwkeUlyj27\\\\\\\/B0QclsJ71ltJL5ELdjD8uS5ftX1lFk8TaKFClGRSucHdqLySabWCrpUa8ieDurDgmRtRSVsyc73IpQxcpOFtH6cAMr6oCFmnJ+hJaK9f1xxGWpGiZQOwm5dBcVU3g3+8H2Ki29awT6VTuddt1Jm4G3Y\\\\\\\/4YXbPCqOcqGbiQ1zsjuCjBDf5zV93Ls2r5EU4MjK7BOSzGMRj7vvVw2kedSb2Hnmb0\\\\\\\/MnQe4Bkr\\\\\\\/eGwY1kBkjWgB+Eo\\\\\\\/HEBeSDg\\\\\\\/9rNWOw4ESHm47cdV0giBmHgLVmSfo3XtUyVT13L5PkwA67yXf1qkKhlJjEAuBtG2lU\\\\\\\/N89BwuzMW79DqVtaBE96C7ZelJv9183FiARPHtsZExI5q+h\\\\\\\/Qod3p5dTSGCVJqhyTxA+4NNWwzLLEy76KvSaQCoweajeQk2lL\\\\\\\/wQ41XarPICRgHKV\\\\\\\/E8Ht6RPL7rjkxLQz7wB4tNQTDPCSA9y3sHdZPvyYmJCmNGroXx45UU1JAvU\\\\\\\/Gi3gG0y5V9j0JBsRoNQL6aXLUVOjJoBiL2vthXdhgZd6KMPCtUebkG5+9EFbnaiLM2DW1TydGdAZkt6n85rdwsglk98UhzZMzb+IFTv\\\\\\\/mYjB8TVhcuUWQxpJTyTAbyNE4+BI+WdA3YnBzOyiK7Tpnl30qGKgv8HJfbwL7u5ltquyot5dDq+FJvu6TPgN2zUFPkCcD4HxtXn2RmFGpsfJg01AFQ1Tgy8b+l3glR5TrsoZTXmAXweKeyNmLzBqCumeq8ik3SaMQFr5lfjFDOghXridJyowSq4QxQeOLJjUshGTWBBZPKmD0\\\\\\\/qWNbbfa5NeLNUX29z+FWFHbVLB1dDpndqB5chUQrEnwaj5n3OtehGdvaaTJ8RoHORTKijLj6jK1r6+u7JmJzfmogLPfzAlfrHrRCsAkXX\\\\\\\/b0xSYH5OE+4ES55sk5EWke98QGMelJrnrPIfPTyiYUpRrmuLAxt9f09OkhzYLhQ2kSrXKPq1VR+HeM+NP04engNZ1l\\\\\\\/v1oAekoGnPqGN\\\\\\\/GeDLW1BZKLdwT8gkGdxJSIi\\\\\\\/8hclmb+faIaC\\\\\\\/jewfXsm0y+x0VYTvQuh8g3mmHZH7YYfcDd1FmGC\\\\\\\/46Gql9FsZBTYLxTHnToKTA4LC4u8hYEfHfYkLPVB7lMQ72nl+zPYF4zLMHbxs5nfw7UzGIHw+AQZyyTCbYnZKN3NdkdtZkPmnGPFyT9yesd2fV4HiiEnwqC+yWiUDjIffULbGsTfKhcTiqI7QHz\\\\\\\/2MEgnPNqZBqLVqjaCwkWm\\\\\\\/kbI1PUMkw95v\\\\\\\/7O6I2lcYrfVZtFfbvUGY+Lc7IWXBabh7n+MCLTZJpo0QpDjGfl36H6xTeTHEql6jOPu\\\\\\\/3Pil+Axy\\\\\\\/F+cNNabnqn0r4L0MCCgATSKfIQSj2moAb50XKs5a8ow5MSR4TpYDERhq6vEmXw8zlrkEx+eTSRHOnuKXPf9Bq7DOk+bPhdVcY8WB04ncMkD7SDC7kkKTKpjozrpPrxtlkTip+cbebudJkfuUXckStSTyNnNCljQoSCBgqD4tJMnLBHNBhd7Rhio9I6+VjAVWSt5Tdoy4ovTfD6HQYby+Lp7Pbd8fnpFXXb+9PZJx\\\\\\\/8W\\\\\\\/\\\\\\\/DIhxZeD1CYXv\\\\\\\/jHj1r\\\\\\\/3ESEJOApiJ3h8siWHtCdnIoQs0c2iP9nv8v2Cgi+CWIga232Vidsan5BEknEiWhtr\\\\\\\/AZ4c0Goa+nkuf4DOq\\\\\\\/8mBrF\\\\\\\/xpFuwAEhr\\\\\\\/4ANg05K0MRazqbRMk7jEp+QNQ4Jr2+75im630xivGx1LNP8GYS+ZeBIrzYG1JzUEudGq8ZbKdFya+MBIimF8a8UCdwI99kJW4Gfjicjpy2vxzD2gEpduLespvrNdgIv26B\\\\\\\/+dFGZz1YjVhVFH5q3dnzzxp59+tZJdmbC3h+7QUpJHzA4F0DAXf\\\\\\\/JLPXcSRjIWeOJA37\\\\\\\/bIDn\\\\\\\/WKaAzMiTwAkin\\\\\\\/iRgbwEL5oxSylMujNymLQGXkQw82ElBhNtdfrQKxmt81jGC2FUzu2hva0Tt7cduANEfWUs5luegpjaKNG5JGd3BwoOvpmea3yLtRW6+np2edJxy\\\\\\\/lB7PwdZ5BWwB7FupLhoONUYpTR0PwPzRTvkm+Ni+JBM9lHECEnTs4MvRsulrOHIDnBt8sKwXtwhYY+SotNEYGGf\\\\\\\/K4aeT7G1PgjlCTz8RCoN1uZSw+7tgLC7e99xQ7bD2\\\\\\\/auUrNvxZXmAO4UZ3+7bf9l3pdosKCOwzckQT0XNjAfKO4wIJcBrDHtZxJe0\\\\\\\/UkLdbfcAdGfKroTiFd6vw1PpgUpU3qLgqXEU2JiCcBzm5u1RnGUUlauh3WdxV0mR84sf64s9bj+s14jMjpzxQAHfWPx5y9ul+izErH2UgIZC9FpW\\\\\\\/CxWul2PUbKeHpTp05yWPlGItY+Rh9qh5SUUoM0rFh1o+t4BR51eOTM2RgTmkM6Q+Gy4xmv71r2ADYzKkXTt7lx6ZfgrXg67FfUkPaGlJRn6gtTM+gzHr0TrLfbWxrIzDKBIv\\\\\\\/HOyeZ5UTTRcIk4ZNY\\\\\\\/GZEm+xYy599FSxr0GQNBfCQGwIYwE95tjOJ62F8rgzQw5+csdIxST9uX2GrcxhfGFvqwz0C8hciudyMvMYVokmn6IwI2fUS7PyN8XhWMC3fnl1YrdMOeythqHmzdlXfsgI9EzdZ7\\\\\\\/xHD567hiTXCwwoxY9s+nCKzgLvO6guHqXkIr0xnlIzp8\\\\\\\/FYlgeTaz8xiZhHP9yIGntCH5mlnh0GwnNPiKnAC7EPDJdmlA1oy3m3z87qbGrPRhCHXCBHvUwBUCD2l3E0tBBdgqgtg7CxdJa3w6q7ysCp4CqRYMnLdTzHZfyQwhfZHH+IfrfEwB81DNpAb3KZDsBpNcdolI++fTUhvRidnd68WOyn0iofwVsSVkcmk1O4qDrmfn7PtV2WixImLTczeQVWEKtVZizk7x647dVfl9T3A4jzgvbRDTzbxPitPcbQl19dWbm1V7FQB5AxsJdSnp+nb8jkzxdrbZfnXdaLPVRIyAd3FUZczgNbRPgv0q3q\\\\\\\/iOoWYAMDnRZpXbi8R\\\\\\\/DDf0NqILK3Bkf6ZEZH0fdxsiTEbVpYD0T7moCvNovYp+EJxy5eU0onM19h06qTrWeKZGM9jtMPfmQ\\\\\\\/bi1DB0Gg\\\\\\\/eSf\\\\\\\/8bJZcHTJENHO8hxEUX+JYrRBBqtVxR+ou\\\\\\\/Zx9QycvBagfWgqT+XfGyaQe+Sm\\\\\\\/rivc22NqfFOdha+mAZqrsMU3DOUE\\\\\\\/q1ihxvehW5Exd3Ne+6zLnMtIpwAiR+rO3wM+5e7N4yT9SlQPhGVAAkJ6rqeLOwsYIqWxMa0ibUtVNM323JWsWDyy7vqcCgXfG8EN96M5bW\\\\\\\/vc7kt0o3aWQhI\\\\\\\/Qbg4IEF\\\\\\\/qq6zmbGCwVzg\\\\\\\/66krwXWjCVsJXLdRNj3qgzjz3clSNDlYxaAU+hqN+PBXnlSQz37OA0kkihWO7TXLy\\\\\\\/PGoWHXu7y+5lJ0490fXlHs3Gfz8DLLrRRlTNkHjbpD63251KqxNbuj1xtyCcIGu8Et979vDyCD8D02pxq5eKLZ9iZo9uwj9PJfKPXcOEZjzT41bXu\\\\\\\/ONx\\\\\\\/RQXbBC\\\\\\\/aVA5XY1ymNtm8r\\\\\\\/1zp75218lHRQIgoyjjazswRy43LGEV42qw2mAlMx3dgyj53EtnIcM9cQgHBM0NI+p138ogzDfeFgM8oaSC84n+Sb+LehYgEnlIrxT4FVI+xhh\\\\\\\/IgE0atH7o26gb77RZa94zgrHaF4Qj6FCbIS7hffi4UjW1jNmvzMPod2FbgXfB+KpMyj7hzXMUe3glFzeBBs1wO7Teb0oIS6BVTR\\\\\\\/5CZLm5\\\\\\\/kETUKxQPc+ajGKeMes2Zrms4i4OZoDUv\\\\\\\/DQDLwi7OEqwaW6nc6mWnJWev8Rw50Ewm57BSTNaSady172NR97yzNMYiVWm0VddQMfwjz2Fvz5w9vS0vqWlDzWSdwJ5QV0ZkOib+Y0btyOth2Ej8g3NoftWAFMNojRWyqMZcNlC2vo9Cte0kOpvqb45UJV0IG16KXdsZ3a2I8klq1Xfut0fzzyEKQudkXfmlJkVIko35LAyLYX1UserwA\\\\\\\/JtteFRQ\\\\\\\/MIbSSNvsDcfdiG+6DISAaRlBFhUjuVBL38djn34LM0qM8Z6QTns+WjarxX7RoT67CWrhuJDtJBNH94cQ287WpqeqsJ36nP+giiLO6KoMyRbGtcQPLGuBSzM5NR7CRsJ38aOA7zsEjKqWM59ctS6bKfYpW1LVOrsjvnct1PGme8+v+WAxPwXNbW8h6L6H5JcGJHnzqkNUdkwfgCq13bmV2wlBUx4pvaGMUTHeQFp4gVThM1b8NVZHHZY9FbDnoVM31xFVGN+yUv\\\\\\\/Vdw\\\\\\\/wbdt6Uk3QSk4sxcFSPkfHU8VDc20TRczBxyeBqZI3lMCEIPMPuXw1dKS3bpBYBj5+F79EI0JHtWDNd03loTRkDni58ZCuRSp8keKPCyJs51B5MS8HI1R2+RQArLtVvTFCgAoB6ajkyheku\\\\\\\/JIrlejkJOnKql9etUGRmSFL\\\\\\\/7Ky7\\\\\\\/8p0BLEy3NDMwVeESmCOcoeOetL5bPjRWQI2GMagzz1Ol1PjqCZBZ0RU6BKmGApGF3X5HHr\\\\\\\/jM2uSRXUTCWnUHIK4jFPyW5yLVVAhiUNYKso1q5JKH2Op9F1TtfV316Cw9hAtYptM\\\\\\\/vi0qcRakhXf69ussKdf451K27SjM5VOUZtn89wDLOqPpDzQ4PPkyaTnundOtzgCknAWhwjGm8X2LWtwp2k23VM6WDO1ooXFTid1nlRam+dxugmrxW5lNvT3gWMgxB2pgTild6rlc5bk4Bu3SC1C+PRLDB9N1JUADBle77GAV07p\\\\\\\/apQpIY1LfP6Xc4g8HQ9fMVNGP\\\\\\\/9ZRZi2ufD1\\\\\\\/WGOeaQZO10OREwp1ySmrkPKW9Aou13zF75dWIKNik2vRkvCNeRoMazZ9YUBspL35pepfgCVxDkzebdixXfGUNjyiccCvGg1gwBPbZUokVggi3RgPQkgwT2R1QnMgxEKSxk1AYp9dsQRRpgHImLZtvYfnCBcXgt91bU9eyqubbGc5TvBOu6eEC0txAWJrxvM7lE\\\\\\\/gfoqw+k1dTuTC5U0HPzVDQ9ft8ufqPanUBfu68plbVA4gzvKnRtP0MrvRGIaR50eyDdwyUqf3G9eRYi7sLwj1ucWl6KsRmYRRKYrpIvpYvZwMUZCVWU0rq4pCz8kgEkk1Fi\\\\\\\/orePPePUNmVcBL88JYk13txl86aAIXmoVFxViWpPEiQZdKE2f7ybBqp8JSGCBIlmo9TgFtz+mjkg\\\\\\\/TrzuaICdpMdPP7GU0x1o+pRIMINDfmKP0Pk4JMapp\\\\\\\/TkreMVDQX56xdzTlPkf05402ApVSZiym1EfuI2K1JaY8DWjYp0IiRZ6lR9MnhQvID8vGYXmBrVlN+Xtxb\\\\\\\/t0xXa6LmWfhbXvxx\\\\\\\/zWMJH1qFG6y33gJgFuE3H4ZR2H3t1iFOESQ93d2PlWou+kWaHa6s62pOUnVw3GjcoclXM\\\\\\\/0BuSf2j1QSyO1dK3c9xILLn+zTZSrfvnOIvK+79zLQQkjlKCK6idhnDp5ORa\\\\\\\/K3SfowmZFNcwFp4QI8jnNyJFKxXfcTjeNYVD7L2p2osFOqd6sCCYNXNtoiysvsRuTv\\\\\\\/UOAUP8ExtI+FRoecTEehnFxdn2yt0HmYi2dutkHsa1Emjd66xGPQS\\\\\\\/dZNZzPmT4+pvJZFi8LeKZm95hyQsMOvIovTvZnNKY0QJdcmq4bfI3WmaCyrHIunTM+iaCyLrgaoymo1B2K5dr846K6iolDAk0OVhVEkuSb90zhZuBUnEIYG9XgUBC\\\\\\\/q58\\\\\\\/Z3avTQdQf1kThATVvnzvRwMym9t\\\\\\\/RY07Ea9DE5gIXnsnpb7+5OgPSWnO\\\\\\\/FWoCQzoYwz\\\\\\\/x\\\\\\\/XKQ580JV38q26y9uOMqsZkzEH19cawQkk6RaWcYO2m2oYl1xiUnIcmB8UMGYllfNDlR53XzO51\\\\\\\/pSmiGWo0kFfS+MnEnoXVxx1OZqUiYNEoKq+gDCdqhTOI3dlECJ53CDmLCeKDH9vz8cWB56LI37upynHZxNCKfJnetLG0HB7tItpy57Xlru6GfwTc6fFytYM2A\\\\\\\/MlzV1NcH2u6XG1unA3FJ4u0DezV0zYdpook0WgKQif3j4cK1x1AdbCOtAY+gP5WY04DBy8nAKPgzWoWAQY7LJj0Ib1abb6aam1WxdcViPARuDOU4QmSIZ\\\\\\\/KWwsz6cb97XH46e2b9YiIzWVvvpn2O0vmGV35wTxnShRUWAS4QLxxldXivJh+lMvJDIMhcA00bgWcQUzMvuLtR8KINlbywswYYYOme0BoAAkidIfAdyVbKa5mPQM6xpqV6HK6T0DXGANwoSoYu+\\\\\\\/pe+AbKBUhXXD95B7KaMEXfxrL0yaN3NCL0qjmBOgeViaXB\\\\\\\/Xqo7TXrIqiC3BA8r+Ng1SX8a8\\\\\\\/OG6imnCWa8J3pP+krOOZRjbgRa0ZXWkzP2ifn1xwjSyCy6tZDfImj33UlM4J\\\\\\\/DB5sGLNKwM+zgpFVeHR335uB5xFe444NEmfx98HlevdVAI\\\\\\\/drXgjhk04h3oX2VOlkJEjnGQqSOWVdRNyTXkzSTzmrGm2PwJqVfGZJGlI\\\\\\\/xsiPnHPx2JG\\\\\\\/Ba+niiOiuvTy\\\\\\\/7qf6XBRjw3Iew\\\\\\\/YdKiDrKOBIDMfpi9JHPN4ByH6c+DjVhUbWerPnGTHod8UzfRC5bA3ZI004W3J6LDshjkw33ZVfRqm7HT7h0S4z4Aisvo9l1RJ4u6A\\\\\\\/eJt+1cISYvUpCSy85VRTl7bAfVcjGNc0O6hxpxcuzJskPNnrlp2Ii4t4QSfkopJEgGAD2tTOPiCNEdE3X9KHj+\\\\\\\/GAI3OfC3UK+y\\\\\\\/yj3Fali5taPM4AVGVIBYXD6JvGVGe7m6X5bLwOVl0lBApyvTlL4p9uzpDgfLu1sbNkwLgKpl518Rm\\\\\\\/LWy3LcTpc8AZlE8rw0g1ZSbWTg+cMK90pAhRTsgNzrEVucVZw285dlXKcLtyWIcCEE2LTMWbcXZdaDaUI1jjDZv3pbfjA4Oracq3IDCCJ4QHXMEJFPfjgAKwoVQ5EKuNeDz9iCZhZcL4NHElS8vXfCdTWkllnHZd5dVCxQnSRx8kGTinwGpy4tHt4BOX+bjFMTZdeEhEGktAss9R3MfPnJOTW2usw9QCfY8zy1+H8WtuGTHVfQScfnmfxtYCCIDWvco1u7XgBL59hzxIFsGzmY4d4nLSGxDGbqE7bSH4G\\\\\\\/esV7cBxSZg2PMfPODN9eor+aEgafZqk\\\\\\\/nY+wKWQ\\\\\\\/W5UpBzrKIViw\\\\\\\/NRKZC0VmHY9sKWiEK95cUw+L5ZwCWpeqPjeX4NTvb9QYxWKK1YBclvQBrxD25CuBS8j\\\\\\\/182y4+EAA\\\\\\\/1S9CaR4v6tQAF2JGWzqgWNR8FcCh3Eg5C3Mxp7HRsU\\\\\\\/t7581HOPAHrHZtUtE8v\\\\\\\/MCPgUbMeqUqni8rkpbhfkgcyXHmks6GGdUlRQlAdY\\\\\\\/LYlsm8lKoHfVBL2g8Wm3ybETe+0A78CJnbOddXokkA0yyCZ5v81js1lIbBLlj\\\\\\\/1nKXRRax+RVtyFX1M\\\\\\\/yoh\\\\\\\/M8EGSrHwkePK08N4skymRyFnD42V5lNLxqFHgNHJh\\\\\\\/km5UGAbbL+FxxfyRPjfjIHtc7qqho2yzbiRPu6LNUIQoFSFMhyrZRPsxH9gMhZOjZwDlPvV5moeiNqyO8pEae+oN69fAuKW4q8WEO4QttiQANd\\\\\\\/7qPuhfrwozJxCrSp2eJ29WFeO5YJt4F37DP3fNMrCCI5vssag5b62fKEpi99g8wXvBaN2WkIoadj+aOGcyj0m1OrQN1241Ha3rowPlR9KeAixOW09LIsKujP2dCn2McTAsPUrOxLib403dFItGfk0FMgcFUgX9K0n4ezE63sStS+OHd7pT\\\\\\\/+jwCTC68k8ItlYFmd3doTySI+PPetyIv+WFr4vbIlnbdLWFTvKNRf6uRU3jUszGihsUy5LyS\\\\\\\/Mvpu4uOMNT6um54zyUguDq5\\\\\\\/QMiepFia+NuPmQGfuLV6e9zHzc\\\\\\\/q2lBy2Makc7Rt6zPPaV5QqBdX+wOxCzCzMIW\\\\\\\/6e\\\\\\\/h4QgGWIITTZ6EI9e5E8gSwGjmWXS+Qze+gXfywTSFMLJuxVct\\\\\\\/XNjTLiDH5NadFiglrKQUFT66ULLwaVH2A6fWKxN20sD9RjoG81DZOBqlKk4Xn2gTERt+4KLPWumRsDdpbm1xH63TP1pSp+7GH9Y4sV6eHlt64wFYAefv3V4RQaft9Fwrbscd+qXsSG5y9Z85oW\\\\\\\/Drj014ULpMVap4t+QTb7Qqz7lZQ6rSVIaG4p3R7y3KQxcxX6kN8O2Gr8ugq5rQyo0T7MwV512jpjPYb0Tdx5R5YUyu9DsI47tfrr9XxonMZCH+TwTKHkku3K+uc663rQHZYGLtVGb7n4Jyg8riPSwbeaXtM8wzDzq0+XJZbVSzknE1gc3TgQxcjsRRd2aGo9HbHWc7sGAOoZKVlxVwJmz3QSevQr+z6H+sf\\\\\\\/vGt\\\\\\\/u1siRdQ+E+zZOOzY3udBZrRLCq1oyPVUchIrNwodn8oWMOgWhCKl7KLqgAFdYUyih+tLz8to6xFpOpZdgj8Mlf1h1hnTLcWb1koVNzWmJJ0gnL8Sw2cGsxFPIuW4t9WldwBva0xW7DJAquhAco+WB\\\\\\\/KWgMOWBJ1KGe\\\\\\\/A1\\\\\\\/IIbj217mk731WSjgJmuOP3fyfy6HiFvyxbUAv3Jw9w5b3Ri73mPlYh9q0ej6LFV8mF9Y0mXUXJ8gcUNFHXLiM1G7dM7ko5BsL25utnC1HkzLKhRxUwcszAi\\\\\\\/o5\\\\\\\/D6cGtmWfLL+45mMi1yCIUzxCzLkKYVUQsQbDgJAMlBrzZ3qmSNkLLpJgAseleSVn9CUWcYEHo2EkxnCK824SnhLHl5wuWCd8YZqwLEsjCd9SmkD76xALI32Tb3WT6DgdEQ5OLra4PTdkeN5ryta9GEhJVuy2aF3K2l100C4xOKVPAYMnC8XTPeyD5E3bWnpWPcLs84IyvWKmxsE4vol4EAzi8ZNp\\\\\\\/W9KSxvtWT5PRQnJahtvuOO7Ckl\\\\\\\/jATJR0CH1ncIqn3rgsXCG8a0mqvPb\\\\\\\/ILMAl4wig7bXM5qw09G5egqRL01APkSJHW6BZTZVHKLh0EY8NRzvKHzEjMHQZ3A9iwMQuM9AGDoXHB+M37BngFufjpEhYGO5x7Xuzryq8KuiL7jL5QiRBHvkbtKdCknVbtIUF0tf4j7utfot5Pg5qpTmJCDDCqGNk4Qp6xgZsz8xheb5XK8iOWjF3umj1XAXh72o93EkeeNbqYJneXpbaM\\\\\\\/vY+lEQD\\\\\\\/aT7hoAdPgl5lqrB4AJSiEemURMH7Xm3ZIiO78UM1T8V67DVUhDtRZhn\\\\\\\/TJTFUI9AKJkoNRIMOIwXV\\\\\\\/RDXEGBIvLY1Tb2yQ3tfHCOCjp5hacnIw1Yh8xe5UpoxIhchceRgxsD3kTnL+fmRMOiEN+Bc6FZtYB7dvUoo4fguUm5cnzRbWUJG5\\\\\\\/1yL83NoGsfy4TZWhIFWJ35SCdgya+bpctN7mftOMo3+j77Yl95N+lywKges1hjyDqqDNQ6GkmSB\\\\\\\/cuqliXXEOh1598LVxALBdCQ+ieb9esEKXy7O03\\\\\\\/jEGAjNvJZGUqe2s+CAJ3RSaiNZMmMxlzbmh4a+0+FoC5Lw3cSco5cHDIzxCXX2N68RDQDBcAuGcYRw\\\\\\\/aIUbdhoe8nISyllTmHfZezFkspamFgE9ItEAxRk8nkC6TDxNjsZCjSgfKdxoZICojW+u9oXEUsoT\\\\\\\/nDSxoRkm+AkYT9cktVdB\\\\\\\/cTKSbJ6gEal1d3qTockArxD7fdQ+DDYtEm+Vq8ilVy0Zht9iqjpX52NOAGkqm9ZtFnRPcdvcpaXc8Y+s+STeKbZhi1uKTSnwmLCLRvv904JUzg6CELjMGdzF69HFXy+OSlpsH4rletqR48fQpwrzMXZFmXUQ9iwfKx+AuVtx2wTzyOV2vqgcIWPzVyNRcEORsPwo8Pq7WgG5IJe4\\\\\\\/zzMf4mG4qvzsblENi5y5pTIICc3AW1DfScsRBWUtUIV5DDL1sw1vDtNkf3bzO5vYdS1uXgChXYWPF\\\\\\\/+ZdMNnpAHAIBIyARiBjl2RAnoLn0xs7eqI0T7pZUtMfAuqJ37WbEQIOl6v9Kgtgd1+iHWtj\\\\\\\/CPJV6A98HmsHposn1Oa9Z2CgPyDkUNqQMvZlFNG0brMk0tZJXhKdyUN1YwiprTomH6KuaUnk2wWtzvi0TC1C2XV0SyhxUogz6oAfwDsFVnebyWDAF54XMBuI8r\\\\\\\/ufjP0Ws5WJ53\\\\\\\/NZkhLYKs27odOjBboCOzVEq7Ze872Jz6q4FsGcy2b3qJgrDy2ZS73cn8WD9dScZXpl3StfNnTH+BA5XS27npGiLKM\\\\\\\/QZlAhhcaaQ1TMnh4IJFnJ9EwnN+87B3hOLPzK0dgcXaSPbARs3NBwbKGhvEn7tUqskQIlXxOsiv\\\\\\\/p1u1vT8M6lctAEBZxWWxm99UAfXgCDFkYwWI2SSPGsvn6FNKR1OnjpQwkrdEUq\\\\\\\/aP5LZQpDDzsIgWr4RCqpUO+Ig6MCQoOPcnVjI7Do7heSBLErvOw0pq60JPBbD+nSgUCah\\\\\\\/QsGD55Ia7VozcZfxaGM\\\\\\\/VCvRbKXVO2dwx1uluFR+85W9oatK03kFWrGOJrIfaNvBWy98G4WDnFVTTOoPPiVDD+aXYs5WRDA5n+se48LXb7qoObxWi4XjWt8ZmAy9KuX7gpJIgtMkgQWQuG1r+Sn63i3Xi1mFVSn9udOtBYnl6z5EMKLnU6Wwg96K2ts+S1c9NxpmTlOqequBLdYUlH5bxkgRgCDjW+5R7VJWYcSi8BlUMbfYtGSlYzHAiTx5Kfq1GLemUTaCu+\\\\\\\/h06f95FZZCtc5HzuvHYGJx9IIi+IDtMUfLpdPgPAAzXo45lE82WRnvlwWz75DyTHdkkjV7n8GwZs9PGtzfN4WmglTtILVK+gxQFtSgUja+8edNnC2T4NK80W+FVEGdbOGshj7YA5IitDHrrzFuWm8HhlD+7Y+qBexH+E9WyGpKKPirCyUTRlcBDbpU03ASmfCXg6AFrHw+32eyjAC7PI0CE6h1stf1blKaAhIGzCawh9VooKmqKTyORGKn8CJE3l0MiBAyZ5O34QtAezPys87Tcjy9dh5YHIrM6taB9bZsZLGLA7Ms8RsIIi\\\\\\\/Mqc9rCRslpvfPY5M6OkgPqejnmHQXsmGxtvud5AMATHMRlW4qYsqMn5LOJGYGnFKg1WWtJCRm0GX0utF5aQ6I\\\\\\\/wBXbFGHKmdmwq50aybTMjk3ufyOPTTotwZAPfXBYfXWo82col7l9debzvIW+h5qlAaCgShHWLa23hAk4RCHJ42kQZafZ+h55j6quvEU7Qo36UKQGf49QL\\\\\\\/CKeitCTjgRY0RGRWM6TYEtyFht+0GfGCp5gG53ob0r4KkM0YYwQRzDqir7RqaGEitrAPkCszLl2derE7dWWjox58+Q2J7ugngIzGqJlG4EtmM62qaLAJdXCFKCQkhf+qkc8j2AfTHPnpWuvRqatkxA+G6kRu1VyPnm81dAgtw4M0yUWIlJzHvg3S\\\\\\\/jzwplcr1fQJfn\\\\\\\/J3QLKYt88H3Z+BTeM0k\\\\\\\/xMDnwP7ML9i1W8RpNxOJQkROKKoQ2YUI0sgEiRWAuKmV431DKZWqIf7sFaFSy+efxwpAyGCH0QDnQ08FeHKvBG6vZDErxYWirBvYK2lPKgFvGRXGombZt68HDMXy5Yn2q515B6muPQlh5L7QXYi0OQ3jB4LtYLUnlpWjWCC4n2XZh0pUbKV9xDs7dGJ6X6SkydV006fHDqBFpfPI8\\\\\\\/Z21nDTe+oPXTWAds1oqAlH0a9bt\\\\\\\/6UoKualbwAQcpEOpTRLf2xeoeOnP1S9k133xiXkIcHbH5hj\\\\\\\/BIOmNPr8UyNAqLNdikEvq1CFMXILxA11AnCvjyKoGs4bhd28h1YZA5jOEUj+20V3KelZAflaTyfgJpJVOxBg959SDWjtEHLOfYTdrzi428x22RdNfxtKdSUmX72UIS2xNwq1bQdJOkRe6WYyFtpWAKq2diuF2Lm2FVcmcftIaw9xLHbC38PK1yEbS04JU3B01+8jSRZaoqRsaRAozPrDMTB1tYXmkBiB3C5kp\\\\\\\/tB726Y+PP7QP\\\\\\\/pDeoSTZUvTgkTwBqEwbmOJbUI+A470Or3qFRh9HCwPPsoTIR3dR\\\\\\\/JHzhuFQe0iRJAQoPHf8BvYfPF3C4SpoHvfSGhu4c3w8LC9D8WF+yMH\\\\\\\/iF1cwzqLYvzjQbOjC3E9U+Q8wNx\\\\\\\/YVxCmOynqBVSbRYXX+KfaWQHqr9RsdZalp+q7oLFHkm45Vpj9cK4AlkyF0JKV6onoAbuAXOaEXYWjmPZvt0XPvKKU1SLBmaOxd6ni8pbqNSSW7h\\\\\\\/WQg3rOLlmgsepUxP\\\\\\\/sKMMXh1cv883VJCtjaZiUmBoMugqWj50JVxfGyXXj3DKefQ\\\\\\\/1YROIQOZu4V507VuiaM+9b99jUPt2em3HI5+wKVLrqUX+HahHJg6F1cbBjI8YHbNZdkZVWEGhpfcAqT19dV\\\\\\\/2a2s0gj7UI4EZJwLEL6OMfPRKxtZ9Xf\\\\\\\/1GgAMCf9qxC8KzFliKBZaUKcs41s8AQTwwwvUJQ40x\\\\\\\/jI4yKIxny6WHMSTHlssgzNoHuFUBrfKMPiiToBtJxEhEQqc8GTuyYNHq8ssVSoa3Arr1Diw6f2L+3IPQfzEh+QaAfPZqvJzWcs5RnIiZokFGZu0AG0XPobo0tqXUdqYlR3dcXlHq2rwaGCNRo0++yM3GMGe9tdD+UV6Zz+Vz1Q4SWrpzks3LuaEg78tLQMK0F8qfZvkPZIIaAJUi8HrTWvRqwRecwEV8g7OHFQjJLY6JC+FKLcsSK63kF9Y2d0yor8q3Qx\\\\\\\/BREk1ziQjzjR0M3+wcxHs7jIEmx4zBQRz2yA9y5M8JI60ehPBJ9O4Wcp866bmv08cz3DbDSzVxHQ+B7x1Iz1+LCmMn8g8OvC2qjgOBo\\\\\\\/WgNvXYd68ZNTlvNahv7mEICABmXTEhhXvznaxur8G+WTW3EKFk\\\\\\\/GGEGfCUDI2+8NqqaiTj62EQVgXJoTrTgqNIS\\\\\\\/WYKsKd\\\\\\\/0LIIdSK3+dRY3IDALURbx\\\\\\\/sty1Nv2BmKfFd72MYFXLPZ689fkSPwwyjOQUUZsF3Rmjhn6Bchz5XZv3tNDyPaaJV\\\\\\\/ELQtilVrHHZmET0c5QxRmwYCgPY5yajiuqHSZOAQhswdSpC2GZllGzBmZ2wg\\\\\\\/qamM9DQXFy+b5g0k24l5+wux3Slaf4f9A1TyTXzkiddItmgRfykGh5uwgpsrLWkFfGXbxEeP3EGsEWvjMl5YSTus4ikPjTPjed5yjv1nZniFhGzASTaWyQZRRZHzJOyQyt8nt5n1Wn6pwpECd3nlllPvtJg93OFUko3n5PLO7WDBVRV8s\\\\\\\/v122BN9guIBBgb44fjSe4kAGo\\\\\\\/+qOAMMAR4G6sN5GDHm0YqH9mEmdgDWiOI0tE98BH2i7a50CKcgVkXjXcAxz9nbyBJKISZslxEUdiwBnEIeH5XFIBYPl3fCy4\\\\\\\/nxpjYFQJqweiMQyfY9z2BVAw\\\\\\\/N+KMKU9VTpxU2WD4EN0YUDnVsO0B1xCAyVCbZ2gjCx3kz+VM1sO2H8EMH1Przq37gZy5IGCU7J9i0GhxHzMqqC8D+lzGAfsu9vXo3UhOagPVCDbvCEuE\\\\\\\/iswBtNGjCLoKajF4JRtzZ\\\\\\\/\\\\\\\/L+58qrylxMdOlPwWbhQLN7O7A8QNoA+SbYL2TzB\\\\\\\/yTJlkI+unTBJujG6wu7QV0WmxWhogVUSbFNEOqZx86uYgQSOlR4gSNgvZXAlKmzVknp5ql50ltv+tq3K6UU8uLlJpsZjuo67tTDNdEh7olbfqSdddt1jbA3P3tKy5KCa61h4aNOzT8H\\\\\\\/GDYdZwuFt5TEEam5iMmpJ+WOzVSJIPPd8id0ixJfxv0oLC9OLZm3o\\\\\\\/OTItrv0\\\\\\\/+Seyk4NcLLhF6voHYYwnzTEv4L2QzBsfdd81OCjtPP9T5cGZFZ6RDoYV6xnMSIRH8bTCun\\\\\\\/t6FQgvHQ40NOCsnUm65iYCpNqOCyP5rfAeGqy0OWi+yShOdfj9UhloJ3LSkIOCc+lanGCOAdjcrbl5iLn8\\\\\\\/8vGRPot9Or4GaTPISdn0V7GcCt+wxKuKWZE1zilR7pCXd3mRl7Hr2zBwF794CLoGDXNdMJIlarO95es3AXGKYKT69AH8TWuKlp0EnIlSfmZ3ydPlYeWsCSY8PDdgONxjFPV7P3muBNLTbFH4GRqElpI68W0tWRUJzcvoCyMGEiuCg2Qz1p+waXSL9vC\\\\\\\/o3KmTpMElzPJQKSgLcBeq12VMXdhPccnGSXs8lrVTO7VHmKrxzHl4J0zdbi9lV3e3sz\\\\\\\/2SQqQPLD3iT1RjNlCRjrJRWOAnMlQ9pB25wpQ3r5qXNL3wjzUjwPRTR3w6C7AkkKvMF6LTLgqw5jfI3xGODP0MFugUnExFMchrvVIkKNnRU3RUzoXlDa58fRIWSbZE5UES7Ee2IgIZxxwWr+W+DGlZyQHxlcHUS6+mzqCx6+hCpKpR5Y0w\\\\\\\/VN7pnQJahDmRrWPF1bMcMd7HUYMJSbCJxrii6bBh8WHaKnM3w2jFm8MUR4hLYSqmszBXqjNHujx65+6fnmwk3CYlQHImCFYFg4TpM6OQH10pupOTP2uXryXG\\\\\\\/hU\\\\\\\/wUi7qD\\\\\\\/W55shytvtZgJBsEGfKucgj1PTEQN8PQodSPJiFk2u6+ltOmJgBiWp0DFrCP\\\\\\\/d+t4m+2l2XawUhJhIu4bh4hIIFMe\\\\\\\/DkoIfK6\\\\\\\/usDFf8eNhsxoExrV1Cn\\\\\\\/wyYg+fsnTTmD8cJ92nZvPIH5T3i7EC03K2QmTXFVwrY\\\\\\\/ZJmzOF3o7KwFzqYFTihqJvvDMjrWMf0faXUyHcXBE2dNMGBlu48Pj7FOuEPFsM4w7SjHRpGQU+uvEErQjqvjFAsWDZqz2+ZF4QE2X8N3Tq9vdarV9FuWmutwDOeN\\\\\\\/f2YWeFZDAnv2rznGMamc3IBZ7zTN7yYWjNj5\\\\\\\/a8irJrI1jUmUsOsYu1fHwvi1wDw9QP\\\\\\\/FPAOB2OS7G7tBRXnxmYzzogp1Qnbj\\\\\\\/xZLHvDH3xPM3BFvunMhmjv6rm+29du2QjpwZhbPRMutFirhy5sapRDE+ApWVGiqf3TqXKgNcvii3nosen8fOf7l3phqUuDFt1bcElOy+brQoV+BtIyR9Ol6AE7j8UrI2FwiIOQq74GpMfe90E0ZDuZ+iPHuS73v0KlwsbhJ7av3tGEWLsL\\\\\\\/GoiAtXHenzsroI7f\\\\\\\/i4cchEUggZRwTK8uld23EMjisG3GN6qwwAHXXesoHkBU57BrjHEvCuZBHupdo4nx3S2s8LvKskNqGthsNDVEuR4KIKMcto2iDhcwrO6mUHlX\\\\\\\/X3jRINzDSPXJi94BsQr\\\\\\\/COPCicHrFlRTKEfgQlELtOn2fJHbp31bX7AFQNqBrhgmYtmUW2\\\\\\\/+D4HpSILBzGJhN+O9CTl1KOHC9xCU4IV1F05IK7UakgEcJaKcd+oZ1EYIZHn27m6cGTvxL8WFD3hDO3DgUVTEH2HRY\\\\\\\/MAXAZUYILSFFOwJCRzcL2kN\\\\\\\/VtDiV2Jv11jiwNFa1uVpZLJUW1fxIYHY0QMNld6Jw2XGRouwzT9t7pgiYQvMReBXl39FBWY8aCwT8m6h5gnoeX\\\\\\\/83wveOpbnb57C3jR842gpXMDIkydZKd9QvDUykoqWmsz0hQZngA9gFy1DBKSj2cP9\\\\\\\/fTwesG91axG8awRtpSaXqfzS3upA4662wWlRdnhmFoQYZ16KS8Wq5WILrLEN7J5kyMQVFUTqzIeaDGt+FSsLAawXFYXWNzbjuMvqRApq1krnAbEXriRneGQPfnXj1J7hhLPa1qTLED0utA5MPJ+CZEnK8PQT7+wyvduKYQo7TYc98Gm9CYYuPMoMgmrSFDYonU1uAh4uiujS+uzUNy18qgosMoJsh3joyfL0BEZJFiPlwjt\\\\\\\/oHBCWRlE3\\\\\\\/cH2j5OX37\\\\\\\/l\\\\\\\/jbGTw7VxLUx60HdBE2slpbEedAbvRhfJWXxLFP8DxzqAXDX6nhUwohNYvLrNV8mBrQcMXV\\\\\\\/XeEk4LqycSywXiTxBAD9j7WT7dxhTzH2SaRrdez3wm4a6EEHCfpjL1r8GmZOVO+h5f4BejkSZ\\\\\\\/ixU6ARb+Ug1hXhGxGGFvxUXJITf6fZQQhVV++GQCqI2zsn78ilZDRzkY37FOKHrRn2FhS39Q3GUidqjQxBMS1HAy9FeLAlTI\\\\\\\/qL6DxIQFLAXBccI9sWmDcbaprpsTaNmPjxVp+8Ti672YgmsCsYQrkSv2M9gsnwT1qtX8j\\\\\\\/9L3WEl5HqkTqtJD\\\\\\\/2dt1P9JGSaPXWUJo3G8wt46iP5GyybzBVLV9K\\\\\\\/PHbvlQai2TZpv1m+o+w\\\\\\\/ygr5lBi+42+Ai0vv2UxntHm9jI+Q2IecGPkFjJHNozGWBHCV3zlUxC0dETVz6Hi9qAn3W7PkzA29oZElTqzME6Zs4h1RAb6X0+tYnHPoN9YkLv0NqLw9TWqZlRQHHUWXqDczwHgZJ3e7gvmTH9TB9ACKTnvUsqvYCGnNHdbJEbl2ICHu+3alx3hz\\\\\\\/NRAg\\\\\\\/Qw8UKuIO9HA2Umwq8EChefwzM4S9x8N2gDx2wXrZUFbqs\\\\\\\/\\\\\\\/Is9JDtZofKzLOtkC9qnlw8zl8QbQstpTcH8cL9TBOzfst08+ZSBaRJdFUg6A5MuZsoBa1KNOUp7v7fcZIsStgnCRT\\\\\\\/259BzGx1gK7KeOcYmqNFP5nauj7P90e9c5uM305xPpwK5NgFVbIUJO4wu4wMuUAE758Cv\\\\\\\/tM7MxPtZW0nP2bNHuSw7W93wdyW\\\\\\\/205SQgCsLAs8xgEVbD0sGT8FIq0SAEK7fWD12mtozwRdilkG5EJ+ieHrPsTCeKdyv3hQ50G8KDvCPzvd7CVvvhojUF\\\\\\\/98lW8MCPS3NLq1vnyLqSIfHfdaz+1lgu\\\\\\\/d0mbrATHtOZBoFswbpHWu1kQ1868qQpaX3S6d4Q70yxaYGYfOZ6ss6ohe3DzGMra\\\\\\\/s7icd69j2k1uP\\\\\\\/bwVAXsi3jh\\\\\\\/Rb+5y9wgXuGpgDl+5t+O+jvg7Cf7VNIJMi+AXWkyDbLmVSujmgvhN\\\\\\\/r3\\\\\\\/sBgVcDTw7kx46VrAeq3j1JFFYQ+KFNJ3f\\\\\\\/cVnq2JxnawYeRmyCYe6QE2d9OCa9DotC5f0rZoNysJfCy\\\\\\\/Me2j5dv1pIe73E947D3\\\\\\\/wPQq7NHJ06nYobaxCUN9kYPe2uA8YDP+o7Sl8ytUwrQS5jBEGZWzsxB4\\\\\\\/ydrALY3XJUEbxGSRZsORvOO45Qp4T1j9Kawq9XyYJ6zxZ+QWKCy681y3Fup0BA7+NVXbDxmAyzDBuat3+7n2ZzydRs8SI1EsST1wk6hof2Af3N\\\\\\\/nIqTQCTDYQXiMuiSnliC1nqS6VoHXOOhYVlEp5GWPnboGMSf\\\\\\\/98r9BoO20kLrjQ1LnJz6N9NWsL0NODCyyjF2VMOAgM+PCC\\\\\\\/g6JAS35mP\\\\\\\/YfYpuPjEIxhOeR76\\\\\\\/o1472J4RxSWJQKdBp5b+qV+AbaIW5Z4LQ9fuQbcjck2kUH7qJLmyPqVMQpOpdhM8WvDRlTZ0yc5lHpLZuU6KuLLlBdWp5IGwCQIUz9pokFD8Zt4udUfiSmugnb66UGHn2RVNBtoHdWSSmsfR4BH6GHdI\\\\\\\/R38e4BmIJBWzT8oNs5Qr\\\\\\\/eFwGEDYm\\\\\\\/KriZfwJCHDWhAl20\\\\\\\/DCyweFdz0Jf0QI\\\\\\\/iNS0YfYHrPD9amcOtSZDHojPCWr+hp94QK0SkkHk\\\\\\\/4h\\\\\\\/RgiRZJWlWLeJwKMUy3t08VkgSlizK\\\\\\\/WxXJXQDsNrLlrx9urS4dfIxSI9t9apz1IqNmpV\\\\\\\/ti8lWX5cEMKifn2KT8SbDEu7vdmtkSaZOvGnPRSW7QOpa4RB\\\\\\\/KS66Kalr9kKz2pAW\\\\\\\/JuhdrB3XAktfxLqvmeFBUPewpHLhAilxs0gi+mmrgXba7qTaodV+nKCZ4b4X3mUuL2OJsZ5GcFjTDWYbkdRDHflutNdHfSN8eScVNz0gUJjEHkfloXSs9rj1B2a6j+s2o8ctVqS9wUa9ODNrw5jZ466gryw3\\\\\\\/vqd92mYG+Ps2emAv1eTfUmpUZdVz47oHAqz7wswaKpopaOrukxD0Gl4z0kFmm8QnJBwqqQbRsyohfFVUV1xYtNDh7a7FWC4NTLecsoDyWFEgYG0tnbxONgc6gxlUpvQ7V36MdcHzK++TrWx5uMSB6fafQRENeWGwfuIH6XP3dDBnkjbTAFtzNWsX72FduqtpxIZZFnSKyv1gmPvxtbdsledMITgvi7V7DeRbyHk23dVNwxE2GYLMeUoCouu5Gv6M19OBRz3HFWDBZCX0n\\\\\\\/jtyo80kqow5UEPfzD8Vl\\\\\\\/6F3d2PmOvsAiL0ZToHKoOfIhGmp7vleiAG7RYmZiDJBZiwVgfPnSSYgjQpgtU2Jpy0T4FS\\\\\\\/aTYsJBwMILjS3sI4h8ecKynSbKpzqg0BLXm8pFkfLbWazxlDyj1pJsSmt5BCvBXCvrcs7ecBn2sfTXZaiiQlQEHLrFkquoTPE2wNghxrV0IesnFAlo8R8MrU9RtFuhWd4HQ9pOzvn+BonTt4TNYTSGxRHcLJhXRgDvK3+rSFqz0xbtifUzf\\\\\\\/mJu8C14V3SBsuVEiA2d41\\\\\\\/aqH2xWhZZxkcmKXc5VglXb8Wcx+01l2RZ3dawm14WLtgQZXTJaZPbrqEtvFwYK9DGJlgKX5if\\\\\\\/mPzBcJF2vWv0tkP6ZwygGoB0gPU7wP6g5tA741p25CJpoWpifAejfNTJzt0M\\\\\\\/OCUAW6WSc1aqbfqMlxvVKbNGIr0zB4C7OkA8QaEYhZwxhcbb5qcCg2vmrSVWJhcf0tH9cfg1mCi0vXT6FqS5MFIsDLUIXIi8XXz51omONawyKXU4PuqfVmlIhQN8cbZHmVPhxUNwHMTAqbMUm9ZkwNtpxXxvE9uCDtv3fTMcdQgs7NdafskxgJF2Qqj41n\\\\\\\/f4irbRdXIIwskqRp9hpos4MbMZpi5vFmuoGJ7eirgO5qH0CB0qe7P6Ckb\\\\\\\/qO72wv16I+cv41fYa43Zqv1JVk4nsq6LcqiMqrrSvQOIggSiHRX5twvZmq38QJMP+L3NhoQEeXkDNzC79tHp3hBk3MtTJ1Noq0BF+z82tYXarGIRUqctMhY5cgB7kWH4LuLVihVKEKHmOCCaUbny\\\\\\\/rG5vT8l38Wuk5DW5bonSzeHJxWY2K3E1NaMlM6zyIsnFg8F8jFjjNYjMJe9+2v21TA5IoaJbmfEyxVtLUkzF1euuYph332bvT9LI\\\\\\\/FOhDc\\\\\\\/7O4hPFeDcgcQoecp2ti2WMwupyXGUrrtcgXqFC1euP270oLuZz8njut5gOCSOdAMe\\\\\\\/gevWoob79j\\\\\\\/WkbxOobSwi8MCQ1KB\\\\\\\/JucDe1ZwIy0PebkLGIMNU9grsIjcLjgfKkFEHYxI0zc3G3gQnmO1aHrJDFLblZI8sENFo8\\\\\\\/PpNvMG2dEyCk9m9nx1MdnGPTbSdJgdFQBAP2hVVasre3SEnwnKhmxS3F09tlnB8j1Yq908ct6cjqBHi\\\\\\\/S0IDocsyibOO5Y4kCBhJsm76i9WDQ+PYW1jlAXM1YfAUejMtUgON04Irx\\\\\\\/BR9+PBYbhysQLAwdx9Dsj8mGiGOunNsLak\\\\\\\/Uu2t3S0Fr+QOUTGNc8uNPOb3itcjkIGnC5RBnJNM+T38GXj83ZH5PAAL92bEYJudFSM3uR+ELIBbHiP9IXzhpD2U9FH\\\\\\\/W50HfuX5R3FfZgldLigCBms8qRJivnpHY3weslcTspbRMtMjqyAXaZBcZwV5i9xoDyILOnWWsqQbXGwquMSIqpenfnBcXt2BFyoiKm474kZvRmQfsIpT7IeY6TM6eXAxo8mC4uVmaj0czXHGEm65+ipWPeioBw5zT3bEvX78Y88poccPTZukQjGSh6uwoPczhkjQ00CPFJkenJFFHPpvBsZwB30yBBYF5ZTYwa+eNG9ihQPRqaROZ7ZNHwj+TWTuZ+ozYYdXHMQ67etniOtvwGUKI5cHkBkoVAXfezukQBTL8pcEI+V7KaeAok8pl8W92TsvnD0KilpWFSZ8gE6QTHjghLUBb2yejiyxR3qxCFR++Oe0FVKpAq72JEg4tlUBhZneDiOvs3UMJMmcWGLFRqdZAQpII34fqJ1ad3rgvb2uuhLMDuQcIXnt+5\\\\\\\/eqI5as5gmivXebsdoxFFo1+1Gx+76dyQze\\\\\\\/eUNuAXNCiC4\\\\\\\/X1\\\\\\\/nR5MekguqGluIeRcTlp1ymWlcF\\\\\\\/v98iNB3xbQeVTTE8wFm4GNtoYxp9zyv2lmJJwNFEpcZaK9v2i23bc2rbx6m0R4tqwp+Fm2wWKJ06mnNY3HvNNdCJ9ZbR71zRo00xzV7+EBGZuu1XkXQpDdDl5Cu7vLGjdDTSSKzDjW6pD7RsJVhuwvxxRGNb3Ll1KTzBdFDUx5\\\\\\\/YSiZYEGJFcvpkQkVGQDhV8mFSYe6+Oj8iChUDcoIeRD9349KCvvablhfV6PyJvNFi4BaWkOpZ7O5\\\\\\\/UhkveHFUl5iUsIJT0kWuFO0PMd5PUFWlRgjLWxhxxy1YvUpo7ANWQPaSMLuK861gHP0OzRMGmnOYyyLX8zAzd6MZrDsKXnvXcEPCLeAGBNSVEF2JRVn4nOeEhll5YRiJ+eFmIKregGLXPE5f6eCa01i6hKYTe85VLUEkd4P92V8XPpYB4nfxI2MYXUOmPEFfJlr368FCr7DfQ4W+FXchuM8Ylbc\\\\\\\/f+KtiWg4FXcNHLQdHXNO09jFhZioGw8uwbGO7XEP9BHRjpz2lJX91Jf9LEHqd4Vg\\\\\\\/c\\\\\\\/JYr6Z67eja6hzIC5+uRbhctgQFqFnTb1ObcCeDEU2UjogqCXO8B7gB6NmGSmnXP8TsphKJxC+3YA\\\\\\\/jPX+2vJ5yKrbXa7obyjVXLwvjGFAH8C9aUSBNpRdphXZKcs7sO5xzBNaLhe7a\\\\\\\/0oUhOUUh2oO7qhxPfoF\\\\\\\/L5bOi9NjgjXBSD1JVBA7rdleld23ZC2OlIq0XRFvvVblLx\\\\\\\/9MOvwgBHMPLDYeNh7FIDj+t0Btlq+kbuotJ6UHmCeIa9KCHGcPlPhwpve0Boj3h4ouP4QjnLgu3nThYB1GBiw5BMAp8HhUdc09bVL2k7LANzSMwrbjXr7mBwxs+NHlu8lGJVTAp83diPFn72+2VzaE8pjf1kYOCh6Wu51t9Pw6NutbzoE45IDipmViW\\\\\\\/uHwZ6egPWjb\\\\\\\/R2h3JYFZqGtDY+TOs3P8L5RdVe0C+AhM1NzMIrfERLtW5viYNpOytCO\\\\\\\/meDmpMo08lx7ooKM0CvBesWuIJZtwjoTQr5vgSmspnfkqrOeslVC5v6rrxmOiPF8b6GeuGsx7Be7WujCzXU\\\\\\\/I7QrAopQeitzJ1Gj6S+9vwhmhT1r1mKkYa8lKAe3WZo5dpODw8Fn3gMRsWA9qwfHe5XNlSoOqgBQx7UHqY6y+9BjZvwsjl++GzKz44mP7zqOF2ulhKzTXjuw3pFw9q4BvZ2ocqSLzvuT6JDhmqmq54HjLaI+ek7DG6vVa7kiNxxwJCEGm67MXq8dAXHfBqfeYGL3ObeJBHb5dCbmCiVj3uax6BIEJ4PZz8yPrt6KOFRYCb+L+9ahPErRCWpIjAjYS6BNCGuLC8a+11luHD11jJtv5fR1d4AXiWi73+zcmN9pD3NgMwD8JQvFRJoVUI3joj5OXpi5\\\\\\\/hZCVyguSqyKmiNFAaoTk7g+bEoYByncllZPHNcbP4cN4mE1AAHc0z4tFijzHp+O+\\\\\\\/b8GV9qBGUAaBEgUP3SGiZZ1bDbI\\\\\\\/xshz0wge78BdujQHH4wXVASvOfRnzZrCF\\\\\\\/lEnPHIpRcqml\\\\\\\/jsrWEb9tbUMAo1rJ+ccoPUjAUbcKLmCjtSjADXOlWLv5bMKg9FqzBODkc4FYWHGgkH9y8YxpBG714h5vTV5HMU8c6re4UjX7r2QDs1nI1FHQt0W+pW4W5VN2VOzgN3G7SPMlLEEAa7jx5zK\\\\\\\/YaaR6opTU57hj9tBOxzvNkwtZNMYNPsEzqiuVxvz5Ke15r7YmivaaR0XGiDPgGiaYt7nxn9Y1ycW8eW08udIzgv2mvSSqOYlS1gCH+mzMSykNuaaEH3Dv5LuA538Tuuum8TbqRaCz1cKsgIsFT82ZX2sHBDWTbylIt7OOMTCWamXVBtO0kb0DAIa3VRFDXXUfUMNK3e3Ls8Bfl6IjsKObEFjyOpMfw09uoC49lKFNOuc2BUyQGt\\\\\\\/1fQrRrFt5T0qSGMYHZuXZPi\\\\\\\/39ILI6oR6k6BjuVXADqiYlf3Ko4ZoV0PrZqIWI1YV\\\\\\\/NwTfQ0HvYdIrkE1z4mCo7v0T879x\\\\\\\/JyCsMD8lNcgpm1h91ThfHTzMnmqMtoylLWWWBHuytGp\\\\\\\/L914uIVM11jSKwfzbyfrQz20w0xta685YHXTGa465o2+ch7wcSCyrcUhEKHUJsav0\\\\\\\/aBtRiI7fSv6hQzeMYqT+8pI3fAROZ9+Gjz+v110qGiRf3yFGM318msbLXdIHyzKDDbNGZ+P83FJWaZuJsSFFKpL2g0iOknXndkbDNnM8T0VW4CimIv6ntOrwFJ6\\\\\\\/2AF0dfHwO4smi7iexbpFgbPU+Wa97LdRxyHntCJ1\\\\\\\/H27mQlIs0oNv4SDJS465mgIgsf0YOvGH97WClDH3f26dHPyJyRmv0\\\\\\\/3CHByFl\\\\\\\/SvudJKvjJiCpMH4aB7wcTBJT9RZ788n127pgygzsrrpeP0aI9\\\\\\\/BIUpTT7XGaDoPOny6Qn6N6bWDf44P5MWd23YhaLvcZlZ3tK3Qww9FDarXc8zXpHOU2ngocKwgZpVjsx\\\\\\\/3WOyh4sG9lSO6aVC3b9RtWkw7OqMoHUeefvfXSUM4S8RnAUpXOKSnxCPH4+TjsNJVSHRAujXFADTIrJ7jyaUDf7I0AX5fxb9dUfia6lZwGCNzhdOLqAE6nzfpFymVhpg5ygq+gnnaRkwWeFRqw7Yisfc0BftkUeOJKRvNwOPifkVZpZtQH\\\\\\\/9X9CBfbl0U1Gn7B6UxRH30NHxD0Wkznd6Ai\\\\\\\/Vy65xUe5iEuZ3zbDbMWVvzeWNMTCMsWzWegag0EzLRnqnrjVlAvZwhJPaVqcVaOdYP6uLmqxjsMyby9vB5gsmeGVviE0MMzoTkmfpnKU+OBQpv0ElHhixPTG1sK0a8CLRPlqH79Fk7\\\\\\\/pPmNDVHhFPpq1MwFZRd\\\\\\\/P2u66ORoHGQFq7q5rEMu4PollRtnkJ2dsrMmMDutcHHevFFCTLfoRzGQU\\\\\\\/bC4SOeTpCAx0BeNLbYEtupWVOtLv5zSvz8H7ouTy289NgtyR9mNoLfUrOKREAdbu\\\\\\\/p2cLC1SuYWvLZACDHAGb4G+OuDwJHx6i\\\\\\\/rH7TmIbh3n1rYZqNzuZNaVpadzJ4UVbxzT\\\\\\\/iegOMgjU2XOR4BnwutMMKtZU3+3OjpuaamZHgNqzDH6SBdouVRNnOK52Ib8OVTlPPo61slJ7lEbUJuugrBx4CYSToCrkD4vekxPiRu762AlGlz9gd5Dqtx+Gy8xNuE+zxFGjunFwEdvfJIeVJOPVrmU3mM3GBCj7FC\\\\\\\/\\\\\\\/+qUVo7ZAfWt1ObEh3ayCzpxcbafRdnpTMiDt4Znt8vPkWegxtG7v4KG88awZ9W+zkJuDZ9Ql6CdmMDplc1j5f3ArottuU\\\\\\\/eWr5wDbNPj1rGSVkpVcyspv6DB3qf2KUX0QjmL0MkYlc6SN2Fjx3Q0jW+xxf4euWpBMWwczXcyo8ddH50NhuYXir+I0DX4x93UTtraagN9kcY47WReUDaVe0DxDIGe0nxWq0GZSs3XANJQivpAWerOP+2Rh0PsJlmXFwtKqSLFdW\\\\\\\/mC8JA\\\\\\\/NYss9REvjBYduCwgQiSN86ujkn\\\\\\\/xTVVdLfOkrNG7jg5J5FKyB3tEVDxnTJQ9AcbPKtiebijgmcgGq6mM+pJkuo+Rs1lBrSzmWn7xLTz4UVo6rOCOHURBojTbgktmL5nlZYTvj3F1LykTDsh0UOkpt+OoYp3L7\\\\\\\/iUKd9pnhgI19vuxftU6efjn76HH9NSAXou6aHCcPNm9C5St+ziGRgUDgj5Ge6fL\\\\\\\/lv9T9zA0s5qgAJAhRPOcQxznif+mg0I0tJtNYMukr0Cfq\\\\\\\/e4yOxq6eHtjPidAhCvYtlFXUc6cZQ6a9xxw\\\\\\\/c8qnS9Jd8DoDYKvl0u7X6RbCAqrkQjl69Vjn4jxDNPpXSR6unMfG9yf+zeaKtTjkQzbq04+767W\\\\\\\/d8rOnmoO7RjiTkLoxsd7XDrvboONGs9dYGBQA30cgEsxDsk3iLHAc3AKqdgnyz8714SfwCbrj0ec5xjbVKQahZF8mq4DdMWYM6uXZ1V33HNC5EfQgJVVsQ03of2zLFw33vfrVxLWNOXZInCv5JLpKW59Z94hytx5SKOVXqa86i5BcLu0qIlxclqhjDW5P8otkts3gsaI2k08pVw4yMme9nYy8YalT6nG0X0GFCB1alGELA+XaC+cF2FqlFcSZfdfWOP8ascj52CHibtk3akzMakCme4ASapOaHcpsF3h21wN+tSxwXVD9mhXzqjPbT2KzQnaF1jMPWlq7bv3nTkmsu7fB220Y5NxLHbydg8JM2jpXMawW0q8q5UcVQUUbMyvgbPdnRPZpq+7ehumbrtEP4FKggnPzXXEo10ByQ5nZF258CptBDrGvCjQsGbUXUMpfK9K+vu5ma5phG6D7UNZ3djxHtF7vjjLLxUgvhxB8y3l3111K2LpvkwCgIxJuYjGJ7ZAlit9Jw4mnLuyKJrGZZo7\\\\\\\/itTxPhM6JDWOmyivrH22l8\\\\\\\/4cfTLi4Rcy2OsnXVzp8HRUPHoeDPhec894LQhqzonP5AvcV5gz6mS2etih4rYQO1GA4V1eoG9t65ubfRaRVNpgbEpENt+pByzd3ScIv+m+vcit10AYpK2\\\\\\\/a\\\\\\\/XUaYz5zLgoDfU6CSUtjnaICmTwIZpkxfgmh4jNjxJFu\\\\\\\/CzhsUb\\\\\\\/Gjo0lNC0O6fvyggP1w0KJxEYN+AFsqWytvgJYaR\\\\\\\/FJg61x1O8YDhECSJm0W0Sehca8yxpAMCy0s7gvYDm4+zrWxh5TYDQKipJ2cTRKukaD95UvNAq8ERyHsAX2nWXR1ZG1S4heyINB3q25R+xCaOwd632ZGKsuAaPvaDaFz2\\\\\\\/JpP4RKiNWWS7DfTeqXuAYtS3DC7nuG1jwAMLOJSKBJFbJmFWI2xpXegBOa37uU9Yx5I4dbpKyaKch1MKDUnR3JmRwNwdbENJgU\\\\\\\/uwSWj1vCPAdeBOvkQDEIlyaIE7HURdo5noySjPmBrktzlphOtus30PPtaSCnNVtuT1gCG1Q23boUgwdXJWNbAPIlRtWSVx7l9HBFcFPZV+F0iHlF20xTHWmtlMB4d4vv9rcUb9yX1eDPyJdH7LgNIpRNxf8eD3m5mXA79KGJ8brPI\\\\\\\/qY9h2rx6cxhPtFvFWKznVc1u2yp9FhqxKe4LitLPQkQBLyin+NTirtYopLolwS4xxbzAbjXxycRGKfjFLQn0bRJzsHkWN9gB37dWHSHVF6qyWjbgQQeE8xBpY5auSekrZJF9rYBw4h+Uy5RlXZ7gAZCQrP4YBoK61zwYyhktSLywWObvQDbEUBXZ2UrQTNiqWcDQjzk6fcJwWXFE\\\\\\\/YiOw+gLRdocZURpzk1gPEF44Guf4o\\\\\\\/WCdI5dZaH5\\\\\\\/cgOZdQf2rr1dGlvAbUFIHs\\\\\\\/nGLVHnA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ndsUoh0ce+Go+ooJHOMZXg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222ec83e4900180160875301f5fea3ab32d42c188c531f681b4d58dd7fa6ca03dd\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022F88tn9nKpEpoFtonAMxqOhUFOCtDb\\\\\\\/gZWLcCDvb4euDWvkIczBRdQLxvQIvJkgKDF3s8yOxQU1HrwrfdiJGwXDSey4hblVBYJIG4owL42lXKtQ3XxgsjTAgSMrqa\\\\\\\/S+LqmrsbTYGIEpSYvlAeTyvsQ54cihpQpE25pq0sy9e6f0HiCDzF0DKAgjtHJYgouBpHj0dmnb5mnbNoW07d06Ch9WTfWqG26kp65eh2LwGxxThMc+sc74J7zoPUAQUUKVfxxIzofP89LKPR7NjUy+simWLZjQnOdLrfSDWjkS0a1eZNLV5eIXpLN9h3xLYQes6XS6DwybYHm37QW\\\\\\\/2uv7+2h9JWJEUneQ4I8TA4gx3udxOAiBRVvK\\\\\\\/GJOAT+JaxZ+XCSB1a0Ic+8J4Tf23EimoI7dYbYJf\\\\\\\/GhDhENQRB6jBDSso0rRB9ONAM6G++5Bf31pojO07FNdmWo3H0t90fpp1RsOtuNodSV1HlA5NTaCi0NpMcBnoNjpzSq\\\\\\\/yEYN8mCm0Wam03FUEcn3jlj3eKWsky8bh80Nx3UKaIG9o+StNDfjfelJ5pw0t3kG7QkMehFxG8hqRrs8w8EqJL6DlL\\\\\\\/lF0qj2ym8Gt7\\\\\\\/Zh9tWVS34fOFY\\\\\\\/fKQZvhVlo9H2Q7FTWvBvPhbGQxt20bNXxgb9760zqCLaPBfUCnO50Gu+NdHA04STPCxIBg9viFFufSPLAEML+BtmGdPRSP4e5\\\\\\\/TbVg1eLx9l7rZZbjDlxXCU4Aa0iwj1rmy0y1uX5ouf+ab9TrXjMVxP\\\\\\\/hrxnAoRpqyeP0H+ck44QPj0jVC7ds+WwoynCYXMKaQO9gyVsnhfMPJgNqKi2gxMYU+nnd5ZEX93EMb3L+GyDNOA10Zs7KFN1SwYurMREQQKH4o\\\\\\\/p4MKZPro6czXavdjQGQuHAerZfH8e\\\\\\\/sKOhiMWcsz40M4Ru8sEKrt\\\\\\\/eCQpEwz7G+iZ6viahTiO9pPy7MJUgsbhXpPKJWnGg7dGBYtgm40oorogHyQPTNzJKKcinSON4giuqFTTw5oA01Yg8JXYLpKdTq89jlf8j\\\\\\\/CYTP8rTQn\\\\\\\/3nxQ5oqlOyGHOEX927i696ygVvxWVObd1fTUie3kn6tKfFew5MexnJ5YVYHYmMB8NvLcXG0uHymUoukPUWsfi1aactgCswQrqobVk4B4h2KyFco8Tx7nKfPxHgzLwyv0fdLtKUD7bltcqQxI3AjL1KbmlXoML\\\\\\\/hN2fWx9PX+oparCU9uYMqNBdBalc0l5xTc3KQtwj6xILge\\\\\\\/S4+DpuR7gG1Rvun0aeGpv+do\\\\\\\/S6Bh056JQyjSZ92S4za\\\\\\\/jAS3GjFJq0f1O9o1ANLsyIJyBK+Lx1wiispkyoGZtcTsKr45PIxGuPHEF7XT+j1muJeMiFNqFXPi7QofdvWhUJv9EhD\\\\\\\/eQ5NlT5U5rcg4zKncPTWTkimgz7gHMYhCPo7k78WrJbNRvSCg5kBUssu7PaYNLxUHcnnovGx\\\\\\\/14z0ypQRPqcqR+8VXvHjk6DP4Asv6Ccm+5sqkycEU4V8u3L+6\\\\\\\/678hd3srkQbyXifBxkCE6YwKWH94LNqF317s\\\\\\\/v1\\\\\\\/PQs4MhiTfpLMuE5xlvz2pZTSajjrxiskTtaTrE5hGucaSNkzpeJ2buBPARbI\\\\\\\/NH5qmIuDdOQSrOcBkN5D+clDAcPnkhUFBLLrSTxdU7P5qzoLvUhtJFFvdJWn6voK\\\\\\\/8cW4Y4fnOIBscU2bUSIvj5hoSmzTt2xhiIQXq7k3yUWSTEZcnhnhHcPPt7DL23OYZeaqw0EX3yi02aRwt5+5EUc5T6xnsfsi9mO8OODz8eyoLVr6ND6E5y+BwDzrXeMQ6JNzMZL7I39Llpsk4VcW7OAcg20qbq\\\\\\\/4f9dAdPDbl3wkOVOomi7zzAcGvM\\\\\\\/4YiEcz1uArZ+WqI6vNlk6uJzcxgA3TKyqeOnD0nviekN\\\\\\\/XpVvTlGdapVOE0d1K\\\\\\\/N\\\\\\\/HmGzFL8etBAlQ7ms\\\\\\\/1mx0EB33pTVhYJJizpVsF3d8GJFi55bCPYVBohhePSObzahqelDjYGNTq31hS3sRFsPUEfOGY7lrUK6mUqh9AfeVscAEuZ5Es+dH3C\\\\\\\/ZbkYQI1GYTD75m071XKrTSP3ZSYQd5mrtWwzEGvoem8wQFRiwRACHu\\\u0022,\\\u0022iv\\\u0022:\\\u0022xT1gqyqVUZCbqDQxvH3phw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b235dcb78ca2547d57bd535597cbe0840c12c677bc6dc165a51868aa715ec2ac\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459907 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1095" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-v5M7KkLzqSMOg-VSQSFwGMPoeWisgJCrSY8xT5gLbPI" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1095" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-_DOZz8oNAa754dS0Rsdl7-KqsNm64mp0AAZNDbX-1Y4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/6/1081" title="Insights into Ongoing Evolution of the Hexachlorocyclohexane Catabolic Pathway from Comparative Genomics of Ten Sphingomonadaceae Strains" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/6/1107" title="Novel Genes Critical for Hypoxic Preconditioning in Zebrafish Are Regulators of Insulin and Glucose Metabolism" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459907" id="node459907" data-pisa="ggg;5/6/1095" data-pisa-master="ggg;g3.115.017749" data-apath="/ggg/5/6/1095.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">BAC-End Sequence-Based SNP Mining in Allotetraploid Cotton (<em>Gossypium</em>) Utilizing Resequencing Data, Phylogenetic Inferences, and Perspectives for Genetic Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9670-9433" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amanda M. Hulse-Kemp</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2094-2367" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Hamid Ashrafi</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-3468-4119" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kevin Stoffel</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-8574-4708" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xiuting Zheng</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-2780-4274" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christopher A. Saski</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-1968-8952" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Brian E. Scheffler</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-0036-5459" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David D. Fang</span>, <span class="highwire-citation-author" data-delta="7">Z. Jeffrey Chen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-2093-0577" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Allen Van Deynze</span> and <span class="highwire-citation-author" data-delta="9">David M. Stelly</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1095-1105;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017749 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459907" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Amanda M. Hulse-Kemp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Soil and Crop Sciences, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdisciplinary Genetics Program, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amanda%2BM.%2BHulse-Kemp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hulse-Kemp%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmanda%2BM.%2BHulse-Kemp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9670-9433" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amanda M. Hulse-Kemp</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Hamid Ashrafi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Seed Biotechnology Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hamid%2BAshrafi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ashrafi%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHamid%2BAshrafi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2094-2367" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hamid Ashrafi</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kevin Stoffel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Seed Biotechnology Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BStoffel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stoffel%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKevin%2BStoffel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3468-4119" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kevin Stoffel</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Xiuting Zheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Soil and Crop Sciences, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiuting%2BZheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zheng%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXiuting%2BZheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8574-4708" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xiuting Zheng</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Christopher A. Saski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Clemson University Genomics Institute, Clemson University, Clemson, South Carolina 29634</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BA.%2BSaski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Saski%20CA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristopher%2BA.%2BSaski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2780-4274" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christopher A. Saski</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Brian E. Scheffler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>USDA-ARS, Genomics and Bioinformatics Research Unit, Stoneville, Mississippi 38766</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BE.%2BScheffler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scheffler%20BE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrian%2BE.%2BScheffler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1968-8952" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Brian E. Scheffler</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">David D. Fang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>USDA-ARS, Cotton Fiber Bioscience Research Unit, New Orleans, Louisiana 70124</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BD.%2BFang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fang%20DD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BD.%2BFang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0036-5459" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David D. Fang</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Z. Jeffrey Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Molecular Biosciences, Center for Computational Biology and Bioinformatics, and Institute for Cellular and Molecular Biology, The University of Texas, Austin, Texas 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Z.%2BJeffrey%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20ZJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZ.%2BJeffrey%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Allen Van Deynze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Seed Biotechnology Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allen%2BVan%2BDeynze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Deynze%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAllen%2BVan%2BDeynze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2093-0577" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Allen Van Deynze</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">David M. Stelly </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Soil and Crop Sciences, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdisciplinary Genetics Program, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BM.%2BStelly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stelly%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BM.%2BStelly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:stelly@tamu.edu" class="" data-icon-position="" data-hide-link-title="0">stelly@tamu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/6/1095" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459907" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459907/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1095.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459907" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459907/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1095.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459907" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459907/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1095.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459907" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459907/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/6/1095.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459907" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459907/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">A bacterial artificial chromosome library and BAC-end sequences for cultivated cotton (<em>Gossypium hirsutum</em> L.) have recently been developed. This report presents genome-wide single nucleotide polymorphism (SNP) mining utilizing resequencing data with BAC-end sequences as a reference by alignment of 12 <em>G. hirsutum</em> L. lines, one <em>G. barbadense</em> L. line, and one <em>G. longicalyx</em> Hutch and Lee line. A total of 132,262 intraspecific SNPs have been developed for <em>G. hirsutum</em>, whereas 223,138 and 470,631 interspecific SNPs have been developed for <em>G. barbadense</em> and <em>G. longicalyx</em>, respectively. Using a set of interspecific SNPs, 11 randomly selected and 77 SNPs that are putatively associated with the homeologous chromosome pair 12 and 26, we mapped 77 SNPs into two linkage groups representing these chromosomes, spanning a total of 236.2 cM in an interspecific F2 population (<em>G. barbadense</em> 3-79 × <em>G. hirsutum</em> TM-1). The mapping results validated the approach for reliably producing large numbers of both intraspecific and interspecific SNPs aligned to BAC-ends. This will allow for future construction of high-density integrated physical and genetic maps for cotton and other complex polyploid genomes. The methods developed will allow for future <em>Gossypium</em> resequencing data to be automatically genotyped for identified SNPs along the BAC-end sequence reference for anchoring sequence assemblies and comparative studies.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/snp-genotyping" class="hw-term hw-article-keyword hw-article-keyword-snp-genotyping" rel="nofollow">SNP genotyping</a></li><li class="kwd"><a href="/keyword/cotton-genomics" class="hw-term hw-article-keyword hw-article-keyword-cotton-genomics" rel="nofollow">cotton genomics</a></li><li class="kwd"><a href="/keyword/resequencing" class="hw-term hw-article-keyword hw-article-keyword-resequencing" rel="nofollow">resequencing</a></li><li class="kwd"><a href="/keyword/bac-derived-snps" class="hw-term hw-article-keyword hw-article-keyword-bac-derived-snps" rel="nofollow">BAC-derived SNPs</a></li><li class="kwd"><a href="/keyword/intraspecific" class="hw-term hw-article-keyword hw-article-keyword-intraspecific" rel="nofollow">intraspecific</a></li></ul><p id="p-3">Marker development in crop species has been an important aspect to facilitate genomics-based crop improvement. Single nucleotide polymorphisms (SNPs) are the most abundant form of markers in all organisms, because they have a possibility of occurring at every nucleotide position in the genome. This makes them ideal candidates for construction of high-density genetic maps, which can then be used for marker-based crop improvement and genetic analyses. In general, for marker-assisted selection, a large number of molecular markers are required in crop species as the majority of the traits of interest such as yield, drought and heat tolerance, nitrogen and water use efficiency, disease resistance, and fiber quality are complex and controlled by many genomic loci of small effect. Therefore, for marker-assisted breeding to be effective, markers for most of the regions controlling a trait need to be included in selection criteria (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Mammadov <em>et al.</em> 2012</a>). SNPs have been found to occur approximately every 60–120 bp in the maize genome (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Ching <em>et al.</em> 2002</a>), every 268 bp in the rice genome (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Shen <em>et al.</em> 2004</a>), and every 185–266 bp in the soybean genome (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Lam <em>et al.</em> 2010</a>). The markers developed to date in cotton have been limited mostly to amplified fragment length polymorphisms (AFLPs), restriction fragment length polymorphisms (RFLPs), and simple sequence repeats (SSRs). Some recent efforts to identify SNPs in cotton using transcriptome sequencing in intraspecific and interspecific cotton lines (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hulse-Kemp <em>et al.</em> 2014</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Zhu <em>et al.</em> 2014</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ashrafi <em>et al.</em> 2015</a>), single copy sequences between <em>G. hirsutum</em> and <em>G. barbadense</em> (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Van Deynze <em>et al.</em> 2009</a>), and a variety of reduced representation libraries (RRLs) techniques investigating combinations of <em>G. hirsutum</em> lines (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Byers <em>et al.</em> 2012</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Rai <em>et al.</em> 2013</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Zhu <em>et al.</em> 2014</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Islam <em>et al.</em> 2015</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gore <em>et al.</em> 2014</a>) have been reported and have been increasing in efficiency using a diverse range of germplasm. These studies of cotton have generated anywhere from a few hundred SNPs to tens of thousands of SNPs. However, success with cotton has been limited compared to crop species such as maize (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hansey <em>et al.</em> 2012</a>), soybean (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hyten <em>et al.</em> 2010</a>), barley, and wheat (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Poland <em>et al.</em> 2012</a>), which also used a broad range of germplasm and have been able to develop hundreds of thousands of SNPs.</p><p id="p-4">Reasons for reduced success with SNP development within <em>Gossypium</em> compared to efforts in other crops are perhaps due to the evolutionary history of the cotton genome and recent polyploid generation. The most widely cultivated species of cotton, <em>Gossypium hirsutum</em> L., descends from a recently formed allotetraploid hybrid (1–2 MYA), 2n = 4x = 52, genomic equation 2[AD]<sub>1</sub> (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Wendel <em>et al.</em> 2009</a>). Modern cultivated cotton, including Upland types, have undergone at least two independent bottleneck events, domestication, and elite cultivar selection, which further reduced the overall diversity that can be found in elite cultivars. The ancestral A- and D-like genomes are thought to have diverged only 5–10 MYA. Because of limited time for evolutionary divergence, homeologous regions of the A- and D-subgenomes of cotton retain a very high similarity. Although reference sequences have recently become available for the diploid cottons, <em>G. raimondii</em> (extant relative to the allotetraploid D-subgenome) (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Paterson <em>et al.</em> 2012</a>), and <em>G. arboreum</em> (extant relative to the allotetraploid A-subgenome) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Li <em>et al.</em> 2014</a>), a reference is not yet available for allotetraploid <em>G. hirsutum</em>.</p><p id="p-5">Large-scale SNP development in cotton has been hindered by the lack of a high-quality <em>G. hirsutum</em> reference which, in concert with high levels of similarity between subgenomes and low levels of diversity, makes it very difficult to unambiguously map short-read sequences obtained from next-generation sequencing technologies (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Zhu <em>et al.</em> 2014</a>). Recently, we developed three independent bacteria artificial chromosome (BAC) libraries, two generated from restriction-enzyme partial digestion (BstYI/HindIII) and one generated by random-shearing (C. A. Saski, A. M. Hulse-Kemp, J. Schmutz, B. Liu, D. M. Stelly, J. A. Scheffler, D. C. Jones, D. G. Peterson, Z. J. Chen, and B. E. Scheffler). From these, 179,209 high-quality BAC-end sequences (BESs) were generated. Development of markers aligned to BAC-end sequences is desired for several reasons. First, the quality of Sanger-sequencing is currently the highest on a single read basis. Second, the markers can serve as a rigid interface between a BAC-based physical map with genetic maps. Third, taken together, long-range contiguity and marker distribution can begin to be contextualized on a genome-wide scale. Recent reports suggest the utility of BESs in simple sequence repeat (SSR) marker development in pigeon pea (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bohra <em>et al.</em> 2011</a>), cotton (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Frelichowski <em>et al.</em> 2006</a>), and peanut (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Wang <em>et al.</em> 2012</a>), and for SNP development via PCR-directed sequencing methods in apple (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Han <em>et al.</em> 2009</a>) and in <em>Citrus</em> (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Ollitrault <em>et al.</em> 2012</a>) to anchor genomic sequences.</p><p id="p-6">Thus, the primary goals of the present study were: to develop large numbers of genomic-based SNPs for cultivated cotton, <em>G. hirsutum</em>; to compare the levels of diversity among elite cultivars, a wild accession, an additional tetraploid species (<em>G. barbadense</em> L.), and a diploid cotton species (<em>G. longicalyx</em>); to experimentally validate <em>in silico</em>–derived SNPs; and to demonstrate mapping ability and utility of developed SNPs. This article reports on utilizing BESs as a high-quality reference for SNP discovery through resequencing. The new SNPs will be a resource for SNP-based integration of the physical and genetic maps, and the methodology can also serve as a useful model in resolving other complex plant genomes.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Source of BAC clones and BAC-end sequencing</h3><p id="p-7">Two complementary BAC libraries (BstYI and HindIII) and a random sheared BAC library from <em>G. hirsutum</em> genetic standard line Texas Marker-1 (TM-1) were used in this study. BAC DNA was isolated and sequenced by Sanger methods at USDA-ARS (Stoneville, MS). A total of 179,209 BAC-end sequences were retained after quality trimming and filtering (LIBGSS_039228) (Saski <em>et al.</em> unpublished data).</p></div><div id="sec-3" class="subsection"><h3>Plant material and DNA extraction</h3><p id="p-8">The seed of 11 <em>G. hirsutum</em> lines (TM-1, Sealand 542, PD-1, Paymaster HS-26, M-240 RNR, Fibermax 832, Coker 312, SureGrow 747, Stoneville 474, Tamcot Sphinx, and TX0231) and the <em>G. barbadense</em> genetic standard line, 3-79, were planted at Texas A&amp;M University, University of California Davis, or the USDA-ARS facility in New Orleans, Louisiana. Young leaf tissues were sampled from each plant and used to isolate genomic DNA using the Qiagen DNeasy (Qiagen, Valencia, USA) plant extraction kit following manufacturer instructions, including RNase digestion. DNA concentrations and qualitative absorbance values were determined using the Nanodrop Spectrophotometer (Thermo Fisher Scientific, Waltham, MA).</p></div><div id="sec-4" class="subsection"><h3>Library preparation and sequencing</h3><p id="p-9">Double-stranded DNA was quantified with a PicoGreen assay on the Synergy HT plate reader (Bio-Tek, Highland Park, IL). Libraries were prepared using an in-house protocol with individual barcoding. One and a half micrograms of each DNA sample was randomly sheared using a Bioruptor instrument (Diagenode, Denville, NJ), and then size selection was performed using AMPure XP beads to 300–500 bp. Fragments were end-repaired using NEBNext End Repair Module and then purified with AMPure beads. Next, an adenine was ligated at the end of the fragments, adapters were ligated, and the final products were purified with AMPure beads. Enrichment PCR was performed for 14 cycles, and then the PCR product was run on a 1.5% gel to confirm enrichment of product and size range. A final round of purification was performed using AMPure beads. The final libraries were assessed for quality with the Bioanalyzer (Agilent, Santa Clara, CA) to determine final library size and concentration. Each sample was sequenced with paired-end sequencing (2×100 bp) on two Illumina HiSeq2500 lanes. Raw, paired-read sequence files were uploaded to NCBI under BioProject PRJNA257223 and SRA numbers (SRX667500, <em>G. hirsutum</em> TM-1; SRX668168, <em>G. hirsutum</em> Sealand 542; SRX668322, <em>G. hirsutum</em> PD-1; SRX668354, <em>G. hirsutum</em> Paymaster HS-26; SRX669467, <em>G. hirsutum</em> M-240 RNR; SRX669468, <em>G. hirsutum</em> Fibermax 832,; SRX669469, <em>G. hirsutum</em> Coker 312; SRX669470, <em>G. hirsutum</em> SureGrow 747; SRX669471, <em>G. hirsutum</em> Stoneville 474; SRX669472, <em>G. hirsutum</em> Tamcot Sphinx; SRX669473, <em>G. hirsutum</em> TX0231; SRX669474, <em>G. barbadense</em> 3-79). Raw reads for <em>G. hirsutum</em> Acala Maxxa and for <em>G. longicalyx</em> were obtained from the NCBI Small Read Archive under numbers SRR617482 and SRR617704.</p></div><div id="sec-5" class="subsection"><h3>SNP mining from <em>G. hirsutum</em> aligned to BAC-end sequences</h3><p id="p-10">All raw sequence files were assessed for initial quality using FastQC (<a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>), and the first 13 bases from each read were removed due to poor quality. The remaining reads were quality-trimmed, and adapters and any reads fewer than 40 bases were removed using the FastX toolkit (<a href="http://hannonlab.cshl.edu/fastx_toolkit/">http://hannonlab.cshl.edu/fastx_toolkit/</a>). The files were assessed for final quality and concatenated into a single file to be utilized as single-end read data due to the size of the BESs being used as a reference. Once the sequences had been processed, they were imported into CLC Genomics Workbench v 6.0.2 (Valencia, CA). Reads from Sealand 542, PD-1, and 3-79 were aligned to the BAC-end sequence reference using different length and similarity fractions to determine optimal parameters. Iteration-1 utilized length fraction 0.70 and similarity fraction 0.99, whereas Iteration-2 utilized length fraction 0.99 and similarity fraction 0.98. SNPs were called using the Probabilistic variant caller in CLC Genomics Workbench using a minimum sequence depth of six, variant probability of 50.0, required presence in both forward and reverse reads, four maximum expected variants, and one standard genetic code. Theoretical homeo-SNP positions (variants between homeologous regions caused by ambiguous mapping of homeologous reads from different subgenomes in the allotetraploid) were determined by aligning the TM-1 Illumina-based sample, which is the same genotype as the reference, to call SNPs using the same parameters as all other samples. These identified homeo-SNP positions were removed from the SNPs identified in the other samples.</p><p id="p-11">A random set of called 32 SNP positions was manually viewed to assess quality of alignments around the SNP positions, which included markers only found in Iteration-1 and markers only found in Iteration-2; markers found in both Iteration-1 and Iteration-2 from the SNPs discovered using <em>G. barbadense</em> 3-79 were selected for empirical testing. Half of the 32 markers were identified as overlapping transcriptome-derived SNPs as reported in <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Hulse-Kemp <em>et al.</em> (2014)</a>, whereas half of the SNPs did not overlap the Hulse-Kemp <em>et al.</em> dataset and thus they were assumed to not be associated with the transcriptome. Primers were designed for KASP (LGC Genomics) SNP assays using BatchPrimer3 (allele-specific primers and allele-flanking primers; Tm: optimum, 57°; minimum, 55°; maximum, 60°; max difference, 2°; product size: minimum, 50 bp; optimum, 50 bp; maximum, 100 bp). Primers were synthesized and diluted according to KASP developer (LGC Genomics, Hoddesdon, UK) instructions. KASP assays were run on a “<em>G. barbadense</em> screening panel” (<a href="http://www.g3journal.org/content/suppl/2015/04/09/g3.115.017749.DC1/017749SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/04/09/g3.115.017749.DC1/TableS1.xlsx">Table S1</a>) containing 12 samples including TM-1 (Stelly Lab), TM-1 (USDA), 3-79 (×2), F1 – 3-79×TM-1 (×2), RIL01-04 (3-79×TM-1), and water nontemplate control (×2) according to the manufacturer’s suggested PCR conditions. Plates were read using the Pherastar at 38, 44, and 50 cycles, and then they were analyzed using the KlusterCaller program. SNP assays were labeled as “good” if samples produced clean clusters that allowed for differentiation and scoring of the two parents and the F1 genotypes, or as “bad” if no definable clusters were produced or no amplification occurred. (These definitions of “good” and “bad” are used subsequently throughout the rest of the article.) Markers included in the “good” and “bad” categories for Iteration-1 and Iteration-2 were calculated (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) to determine optimal parameter settings.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466274/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466274/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">KASP assay screening results for <em>G. barbadense</em>–derived markers</span><div class="sb-div caption-clear"></div></div></div><p id="p-13">All 12 <em>G. hirsutum</em> samples and the <em>G. barbadense</em> sample were analyzed using Iteration-2 parameters. The <em>G. longicalyx</em> sample was analyzed using the same length fraction but similarity fraction of 0.96. Samples were processed individually and homeo-SNPs identified using TM-1 were removed. Unique SNP positions were collated into a master file and required 100% homozygous identification in at least one <em>G. hirsutum</em> line. Positions where coverage was 1 to 2 SDs above the average coverage in the homozygous sample(s) were removed, because regions with high coverage are likely to be associated with repetitive regions. Additional unique SNP positions identified as being homozygous in <em>G. barbadense</em> or <em>G. longicalyx</em> were added to the master file. Only positions with probability scores &gt;0.98 were retained for <em>G. barbadense</em> and <em>G. longicalyx</em>. A hierarchical method was utilized to determine retention; SNPs were retained in the <em>G. hirsutum</em> file initially, then in <em>G. barbadense</em>, and then in <em>G. longicalyx</em>. Three nonredundant variant call format (VCF) files were obtained from each of the three species groups. These VCF files were uploaded to dbSNP in NCBI (batch 1062135 for <em>G. hirsutum</em> lines, batch 1062133 for <em>G. barbadense</em>, and batch 1062134 for <em>G. longicalyx</em>).</p><p id="p-14">A final VCF file was derived using GATK software (<a href="https://www.broadinstitute.org/gatk/">https://www.broadinstitute.org/gatk/</a>) that contained genotypes at all positions for all samples using the list of nonredundant SNP positions, binary alignment map (BAM) files exported from CLC Workbench for each individual sample, and BES FASTA file. The GATK UnifiedGenotyper command with default settings except for the GENOTYPE_GIVEN_ALLELES, EMIT_ALL_SITES and minimum base quality of 20 options were used to call genotypes for all previously identified SNP positions along the BES reference for the 12 <em>G. hirsutum</em> lines, <em>G. barbadense</em>, and <em>G. longicalyx</em>. Distributions of SNP types, number of missing calls per sample, and number of heterozygous calls per sample were calculated from the final VCF file using VCFtools (<a href="http://vcftools.sourceforge.net/">http://vcftools.sourceforge.net/</a>). The number of homozygous differences between samples was determined using BCFtools command gtcheck.</p></div><div id="sec-6" class="subsection"><h3>BAC-end sequence and BES-derived SNP annotation</h3><p id="p-15">A total of 179,209 BAC-end sequences that were used as the reference were screened for repetitive content with the RepeatMasker software (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Tarailo-Graovac and Chen 2009</a>) version open-2.0.5 (database version 20140131) and cross match version 1.090518 (distributed with the Consed software) (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Gordon <em>et al.</em> 1998</a>), with viridiplantae as the query species. Repeat masked sequences were aligned using blastx (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1990</a>) to the nonredundant protein database (Genbank) using an e-value cutoff of 1e<sup>−5</sup> and the output was filtered for best hit using an in-house .xml parser to annotate BESs. All identified SNPs were analyzed to determine if they were located in BESs associated with coding regions.</p></div><div id="sec-7" class="subsection"><h3>Marker alignment to diploid cotton reference genomes</h3><p id="p-16">Markers identified were exported with 50-bp flanking sequence on both sides. SNPs were coded with IUPAC ambiguity codes and the total 101 bp sequence of each SNP with flanking sequence was mapped using Burrows-Wheeler Aligner (BWA) in Galaxy (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Goecks <em>et al.</em> 2010</a>) using default settings. All SNPs were mapped to both the <em>Gossypium raimondii</em> (D<sub>5</sub>) reference genome (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Paterson <em>et al.</em> 2012</a>) and the <em>Gossypium arboreum</em> (A<sub>2</sub>) reference genome (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Li <em>et al.</em> 2014</a>). Alignments were corrected to report SNP positions.</p></div><div id="sec-8" class="subsection"><h3>SNP screening</h3><p id="p-17">A set of 48 randomly selected intraspecific SNP markers were selected for experimental screening to estimate average validation rate of <em>in silico</em> determined intraspecific SNPs. Primers were designed using previously mentioned parameters in BatchPrimer3, and KASP assays were run per manufacturer instructions. A “<em>G. hirsutum</em> screening panel” that included 32 <em>G. hirsutum</em> lines, <em>G. barbadense</em> line 3-79, and water nontemplate controls was used (<a href="http://www.g3journal.org/content/suppl/2015/04/09/g3.115.017749.DC1/TableS2.xlsx">Table S2</a>). An additional set of 96 markers developed from the 3-79 line were selected based on diploid (D<sub>5</sub> genome) alignment information with inferred positioning on allotetraploid chromosomes 12 and 26 (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Blenda <em>et al.</em> 2012</a>). The markers were designed and screened on the <em>G. barbadense</em> screening panel (<a href="http://www.g3journal.org/content/suppl/2015/04/09/g3.115.017749.DC1/TableS1.xlsx">Table S1</a>).</p><p id="p-18">A random set of 32 markers developed from <em>G. longicalyx</em> were selected and primers were designed. KASP assays were run per manufacturer’s instructions on a “<em>G. longicalyx</em> screening panel” that contained <em>G. longicalyx</em> (×2), <em>G. hirsutum</em> cv. TM-1 (×2), synthetic allotetraploid “FADD” (×2), 2 BC1F1 samples (FADD × TM-1), <em>G. barbadense</em> line 3-79, <em>G. hirsutum</em> accession TX0231, and two nontemplate controls (<a href="http://www.g3journal.org/content/suppl/2015/04/09/g3.115.017749.DC1/TableS3.xlsx">Table S3</a>). Plates were cycled and analyzed as previously mentioned. SNP sequences and primers for all screened markers are listed in <a href="http://www.g3journal.org/content/suppl/2015/04/09/g3.115.017749.DC1/TableS4.xlsx">Table S4</a>.</p></div><div id="sec-9" class="subsection"><h3>Interspecific linkage mapping</h3><p id="p-19">Good markers obtained in screening of the <em>G. barbadense</em> SNPs were used to genotype 118 F2 (3-79×TM-1) individuals, two parents (<em>G. hirsutum</em> line TM-1 and <em>G. barbadense</em> line 3-79), and F1 (3-79×TM-1) as controls. KASP assays were run using the Fluidigm system in 96.96 dynamic array format, utilizing multiple arrays, and read using the Fluidigm BioMark HD (Fluidigm, San Francisco, CA). Clustering for genotyping was performed using Fluidigm SNP Genotyping Analysis software. Genotypes of the 118 F2s for successfully genotyped markers were imported into JoinMap 4.1 (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Van Ooijen 2011</a>), and identical markers were removed (<a href="http://www.g3journal.org/content/suppl/2015/04/09/g3.115.017749.DC1/TableS5.xlsx">Table S5</a>) and linkage mapped using the maximum likelihood algorithm and Haldane’s mapping function with default parameters. Linkage groups were determined using LOD scores of 5.0 or more. Cytogenetic stocks including F1 hypo-aneuploids, each deficient for a known <em>G. hirsutum</em> chromosome, were also genotyped for the same markers.</p></div><div id="sec-10" class="subsection"><h3>Phylogenetic analysis</h3><p id="p-20">The VCF file produced from GATK that included genotypes from all 14 samples was imported into R (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">R Development Core Team 2010</a>). The SNPRelate package (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Zheng <em>et al.</em> 2012</a>) was used to perform a principle component analysis, and eigenvalue1 and eigenvalue2 were used to visualize samples. A distance matrix was determined between samples and then used for hierarchical clustering over 10,000 permutations (z.threshold = 15, outlier.n = 2). The clustering results were used to create a dendrogram tree of the 14 samples.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Candidate SNPs derived from BAC-end sequences</h3><p id="p-21">We identified a total of 132,262 intraspecific SNPs for <em>G. hirsutum</em> that occurred, on average, every 888 base pairs (bp). The distribution of base pairs between adjacent intraspecific SNPs found on the same BES is, on average, 76 bp (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). Interspecific SNPs for <em>G. barbadense</em> and <em>G. longicalyx</em> occurred at a much higher rate than intraspecific SNPs, which was expected due to longer divergence time between the species. A total of 223,138 interspecific SNPs between <em>G. barbadense</em> and <em>G. hirsutum</em> were determined. Although SNPs were similarly spaced (78 bp) as the intraspecific SNPs, a larger number of SNPs was identified compared to intraspecific SNPs because they occurred on more BESs (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). A total of 470,631 interspecific SNPs were identified between <em>G. longicalyx</em> and <em>G. hirsutum</em>. The distance distribution between adjacent SNPs found on the same BES from <em>G. longicalyx</em> is quite different from the <em>G. hirsutum</em> and <em>G. barbadense</em> distributions, and it shows that SNPs are more likely to be in close proximity if found on the same BES due to an overall elevated number of SNPs (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>). This difference is expected and reflects the greater divergence of <em>G. longicalyx</em>. Overlap of SNPs identified in multiple species is shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. Considering all identified polymorphisms across all three species, a SNP was identified, on average, every 152 bases and nucleotide diversity across all polymorphic sites or Nei’s Pi was found to be 0.1789.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1095/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distance between polymorphisms developed in (A) G. hirsutum, (B) G. barbadense, and (C) G. longicalyx relative to the G. hirsutum-derived BAC-end sequences." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-53162546" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distance between polymorphisms developed in (A) &lt;em&gt;G. hirsutum&lt;/em&gt;, (B) &lt;em&gt;G. barbadense&lt;/em&gt;, and (C) &lt;em&gt;G. longicalyx&lt;/em&gt; relative to the &lt;em&gt;G. hirsutum&lt;/em&gt;-derived BAC-end sequences.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1095/F1.medium.gif" width="284" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/6/1095/F1.medium.gif" width="284" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1095/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1095/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466321" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-22" class="first-child">Distance between polymorphisms developed in (A) <em>G. hirsutum</em>, (B) <em>G. barbadense</em>, and (C) <em>G. longicalyx</em> relative to the <em>G. hirsutum</em>-derived BAC-end sequences.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1095/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overlap of SNPs identified using G. hirsutum, G. barbadense, and G. longicalyx samples." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-53162546" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Overlap of SNPs identified using &lt;em&gt;G. hirsutum&lt;/em&gt;, &lt;em&gt;G. barbadense&lt;/em&gt;, and &lt;em&gt;G. longicalyx&lt;/em&gt; samples.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1095/F2.medium.gif" width="440" height="274"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/6/1095/F2.medium.gif" width="440" height="274"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1095/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1095/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466278" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-23" class="first-child">Overlap of SNPs identified using <em>G. hirsutum</em>, <em>G. barbadense</em>, and <em>G. longicalyx</em> samples.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">Types of SNPs were distributed similarly in all three species, with transitions being more abundant than transversions (<a href="http://www.g3journal.org/content/suppl/2015/04/09/g3.115.017749.DC1/FigureS1.pdf">Figure S1</a>). However, <em>G. longicalyx</em> had a lower ratio of transition to transversion (1.63) compared to <em>G. hirsutum</em> (2.19) and <em>G. barbadense</em> (2.21) (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The amounts of missing data and heterozygous loci were also similar between species (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). The percentage of missing loci was related to evolutionary divergence between the samples analyzed and the reference, with <em>G. longicalyx</em>, <em>G. barbadense</em>, and wild <em>G. hirsutum</em> line TX0231 exhibiting the largest percentages of missing data (in decreasing order). High levels of missing data in <em>G. longicalyx</em> were expected because as a diploid, it would not contain loci from both A- and D-subgenomes of the tetraploid species, but rather would contain mostly only A-subgenome-like loci because it is closely related to the A-genome diploids (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Phillips and Strickland 1966</a>). Homozygous differences, which are positions for which two samples are both homozygous for a different base at a single locus, between samples were counted for each pair of samples (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). Overall the number of differences between pairs of cultivated <em>G. hirsutum</em> samples was quite variable, ranging from 5562 (Paymaster HS-26 <em>vs.</em> Fibermax 832) to 29,052 (TM-1 <em>vs.</em> Tamcot Sphinx). Differences between cultivated samples and the uncultivated <em>G. hirsutum</em> TX0231 were, in general, quite similar, as was the case between all <em>G. hirsutum</em> samples with <em>G. barbadense</em> and <em>G. longicalyx</em>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466158/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466158/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Distribution of SNP types identified in <em>G. hirsutum</em>, <em>G. barbadense</em>, and <em>G. longicalyx</em></span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466327/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466327/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Description of missing data and heterozygous loci in the final VCF file for <em>G. hirsutum</em>, <em>G. barbadense</em>, and <em>G. longicalyx</em> samples</span><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466199/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466199/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Pairwise comparison of homozygous different genotypes between 14 resequenced samples using BCFtools command gtcheck</span><div class="sb-div caption-clear"></div></div></div><p id="p-25">To investigate the functional characteristics of the identified SNPs, all 179,209 BESs used as a reference were first repeat masked, which resulted in a total of 13 million masked bases (11% of total BES length). The masked bases were largely composed of interspersed repeats and retroelements. The remaining bases were aligned to the Genbank nonredundant protein database, which resulted in 9.48% of all BESs having a hit to coding sequences. When all determined SNPs were considered, a similar proportion of genic SNPs (∼7%) was identified in <em>G. hirsutum</em> and <em>G. barbadense</em>, whereas a larger proportion was identified in <em>G. longicalyx</em> (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>).</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466211/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466211/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Annotation of BAC-derived SNPs</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Conversion of <em>in silico</em> SNPs to assays</h3><p id="p-26"><em>G. barbadense</em> SNPs developed <em>in silico</em> were randomly and nonrandomly (based on theoretical location on chromosomes 12 and 26 based on alignment to the D<sub>5</sub> sequence) selected for primer design and experimental screening. Initially, a random set of 32 <em>G. barbadense</em> markers was selected to determine optimal parameters for homeo-SNP removal. Screening produced a total of 13 SNPs (40.6%) that were found to have acceptable clustering patterns, of which seven (53.8%) were associated and six (46.2%) were not associated with the transcriptome, respectively (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Thus, no difference was seen for selecting markers based on association with the transcriptome. When attempting to determine the best parameters for <em>in silico</em> SNP calling, results from Iteration-1 and Iteration-2 were compared to determine which iteration resulted in the set of markers with the highest overall validation rate. Taking into account the 13 good markers, Iteration-1 produced a success rate of 61.1% (11/18), whereas Interation-2 produced a success rate of 84.6% (11/13); therefore, Iteration-2 mapping parameters (0.99 length fraction and 0.98 similarity fraction) were utilized for final mapping of all samples, except for <em>G. longicalyx</em>, as noted in the <em>Materials and Methods</em> (<a href="http://www.g3journal.org/content/suppl/2015/04/09/g3.115.017749.DC1/TableS6.xlsx">Table S6</a>).</p><p id="p-27">Utilizing the SNPs identified in <em>G. barbadense</em>, following homeo-SNP removal, an additional 96 interspecific SNPs with successfully designed primers were selected for screening based on inferred positioning on chromosomes 12 and 26. A total of 77 (or 80.2%) markers were categorized as good markers (<a href="http://www.g3journal.org/content/suppl/2015/04/09/g3.115.017749.DC1/TableS7.xlsx">Table S7</a>).</p><p id="p-28">A total of 48 intraspecific SNP markers were screened on 32 <em>G. hirsutum</em> samples and <em>G. barbadense</em> line 3-79, which produced 40 good markers or an 83.3% success rate (<a href="http://www.g3journal.org/content/suppl/2015/04/09/g3.115.017749.DC1/TableS8.xlsx">Table S8</a>). To obtain primers for the 48 markers, 87 SNPs were randomly selected for primer design because primer design was successful in 56% of cases. Sample call rates ranged from 0.175 to 1.000. Excluding the two lowest samples, all samples had greater than 72.5% call rate. Marker call rates ranged from 0.545 to 0.970.</p><p id="p-29">A total of 32 <em>G. longicalyx</em> interspecific SNP markers amenable to successful primer design were selected for screening. Primer design for this diploid species was significantly more difficult compared to the other species due to the elevated number of SNPs and their close proximity within a BES (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>). On screening, a total of 31 (96.9%) SNPs were determined to produce good assays (<a href="http://www.g3journal.org/content/suppl/2015/04/09/g3.115.017749.DC1/TableS9.xlsx">Table S9</a>). Two of the markers listed as good but exhibit very close clusters (GH_TBb029G06r260, GH_TBb048K02r139), the proximity of which may lead to difficulties for cluster separation with additional samples. So, conservatively, 29 would be regarded as good markers, <em>i.e.</em>, a 90.6% success rate.</p></div><div id="sec-14" class="subsection"><h3>Alignment of markers to diploid cotton genomes</h3><p id="p-30">When all markers from all three species identified were aligned to the high-quality <em>G. raimondii</em> (D<sub>5</sub>) reference (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Paterson <em>et al.</em> 2012</a>) genome, 38.8% of markers could be aligned to the genome. This was distinctly different from transcriptome-derived markers reported by <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Hulse-Kemp <em>et al.</em> (2014)</a>, of which 75.9% could be aligned to the D<sub>5</sub> genome. The starkly different percentage of mapped markers is likely due to the higher sequence conservation in genic regions. Markers were also aligned to the <em>G. arboreum</em> (A<sub>2</sub>) draft genome (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Li <em>et al.</em> 2014</a>). The percentages of markers aligning to the A<sub>2</sub> and D<sub>5</sub> reference genomes were very similar for <em>G. hirsutum</em> (70.6%/40.9%) and <em>G. barbadense</em> (69.6%/41.1%), whereas the percentage was very different for <em>G. longicalyx</em> (55%/18.4%). A larger fraction of markers that can be aligned to the A<sub>2</sub> genome is expected for <em>G. longicalyx</em> because it is an F-genome diploid that is closely related to A-genome diploids, including <em>G. arboreum</em>, and it is relatively distant from the D<sub>5</sub> species, <em>G. raimondii</em> (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Wendel <em>et al.</em> 2009</a>).</p><p id="p-31">Utilizing the alignment information to both A- and D-diploid reference genomes, markers can theoretically be localized to either the A-subgenome or the D-subgenome in the tetraploid by considering whether the markers align uniquely to either the A<sub>2</sub> or D<sub>5</sub>. In both <em>G. hirsutum</em> and <em>G. barbadense</em>, ∼52–53% of markers aligned uniquely to A<sub>2</sub>, so they can be putatively localized to the A-subgenome, whereas ∼22–24% of markers aligned uniquely to D<sub>5</sub> and can be putatively localized to the D-subgenome. Because the A<sub>2</sub> is approximately twice the size of the D<sub>5</sub>, it is expected that a larger percentage of markers should be uniquely localized to A<sub>2</sub> (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hendrix and Stewart 2005</a>). <em>G. longicalyx</em> had approximately the same proportion of A-specific markers, but the proportion of D-specific markers is considerably lower (12.6%), which is expected because it is closely related to A-genome diploids.</p></div><div id="sec-15" class="subsection"><h3>Interspecific linkage mapping of candidate SNPs and anchoring of contigs</h3><p id="p-32">From the random and nonrandom (selected for chromosomes 12 and 26 based on alignment to D<sub>5</sub> reference sequence) screened SNPs identified from <em>G. barbadense</em>, genotypes were obtained for 88 markers that represent 67 unique BAC-end sequences. These 88 markers were screened against a population of 118 F2 (3-79 × TM-1) individuals and F1 hypo-aneuploid stocks. On linkage mapping of the F2s with JoinMap 4.1 software, two linkage groups representing a total of 236.2 cM were obtained (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). As expected, due to being randomly selected from the entire data set, all of the markers tested from the randomly selected set (11) were not linked and were listed as singletons (<a href="http://www.g3journal.org/content/suppl/2015/04/09/g3.115.017749.DC1/TableS10.xlsx">Table S10</a>). The resulting two linkage groups were identified as allotetraploid homeologous chromosomes 12 and 26 by loss of heterozygosity in F1 hypo-aneuploid samples and by alignment information to the D<sub>5</sub> reference genome.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1095/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Linkage groups determined utilizing 118 interspecific (G. barbadense line 3-79 &#xD7; G. hirsutum line TM-1) F2 samples for BAC-end&#x2013;derived SNPs in JoinMap." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-53162546" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Linkage groups determined utilizing 118 interspecific (&lt;em&gt;G. barbadense&lt;/em&gt; line 3-79 &#xD7; &lt;em&gt;G. hirsutum&lt;/em&gt; line TM-1) F&lt;sub&gt;2&lt;/sub&gt; samples for BAC-end&#x2013;derived SNPs in JoinMap.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1095/F3.medium.gif" width="440" height="241"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/6/1095/F3.medium.gif" width="440" height="241"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1095/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1095/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466311" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-33" class="first-child">Linkage groups determined utilizing 118 interspecific (<em>G. barbadense</em> line 3-79 × <em>G. hirsutum</em> line TM-1) F<sub>2</sub> samples for BAC-end–derived SNPs in JoinMap.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>PCA and dendrogram analysis</h3><p id="p-34">Principle component analysis with the SNPRelate program was able to successfully separate <em>G. hirsutum</em> samples from the other species, <em>G. barbadense</em> and <em>G. longicalyx</em> (<a href="http://www.g3journal.org/content/suppl/2015/04/09/g3.115.017749.DC1/FigureS2.pdf">Figure S2</a>). The analysis also showed a slight difference with cultivated <em>G. hirsutum</em> lines and the one wild <em>G. hirsutum</em> line TX0231. Similarly a dendrogram compiled with the SNPRelate program (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>) was able to separate the wild species and indicated <em>G. longicalyx</em> as the out-group, as expected. It also showed the kinship coefficient between <em>G. hirsutum</em> samples to be extremely high (near 1) and the individual dissimilarity, or difference between individuals, to be very low (close to 0).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1095/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dendrogram produced by hierarchical clustering utilizing BAC-end sequence-associated SNPs for 12 G. hirsutum samples (TM-1, Sealand 542, PD-1, Paymaster HS-26, M-240 RNR, Fibermax 832, Coker 312, SureGrow 747, Stoneville 474, Tamcot Sphinx, Acala Maxxa, TX0231), G. barbadense (3-79), and G. longicalyx using the SNPRelate package in R." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-53162546" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Dendrogram produced by hierarchical clustering utilizing BAC-end sequence-associated SNPs for 12 &lt;em&gt;G. hirsutum&lt;/em&gt; samples (TM-1, Sealand 542, PD-1, Paymaster HS-26, M-240 RNR, Fibermax 832, Coker 312, SureGrow 747, Stoneville 474, Tamcot Sphinx, Acala Maxxa, TX0231), &lt;em&gt;G. barbadense&lt;/em&gt; (3-79), and &lt;em&gt;G. longicalyx&lt;/em&gt; using the SNPRelate package in R.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1095/F4.medium.gif" width="259" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/6/1095/F4.medium.gif" width="259" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1095/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1095/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466205" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-35" class="first-child">Dendrogram produced by hierarchical clustering utilizing BAC-end sequence-associated SNPs for 12 <em>G. hirsutum</em> samples (TM-1, Sealand 542, PD-1, Paymaster HS-26, M-240 RNR, Fibermax 832, Coker 312, SureGrow 747, Stoneville 474, Tamcot Sphinx, Acala Maxxa, TX0231), <em>G. barbadense</em> (3-79), and <em>G. longicalyx</em> using the SNPRelate package in R.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><div id="sec-18" class="subsection"><h3>Reliability of the SNP-based integration of physical and genetic maps</h3><p id="p-36">The resulting linkage maps demonstrate the feasibility of integrating physical maps by utilizing markers associated with BAC-end sequences for genetic mapping. In this way, a large number of markers can be identified using resequencing data and can be genetically mapped with a moderately sized mapping population to obtain genetic maps associated with BAC resources. We demonstrate that even with an F2 mapping population of 118 individuals, we are able to obtain recombination events within the length of the BAC (average 120 kilobases) and are able to separate markers associated with the forward and reverse reads of a BAC in some instances. Additionally, multiple markers from the same BES were assayed on the population and linkage mapping places of all those markers except for two from GH_TBh104B19r in the same mapping position. The accuracy of mapping BES-associated SNPs makes it possible to utilize SNP mapping for ordering and orienting BACs and BAC contigs when SNPs are identifiable on both forward and reverse end sequences of single BACs, or when multiple SNPs are identified within a given BES.</p></div><div id="sec-19" class="subsection"><h3>Factors that affect development of BES-associated SNPs in cotton</h3><p id="p-37">Development of genomic-based SNPs in cotton has largely been hindered by availability of a high-quality reference, which has led to development efforts largely targeting transcriptome-based SNPs or SNPs identified using reduced representation libraries. In general, genomic enablement for modern cultivated cotton through marker-assisted breeding is constrained by extremely low diversity and large identical regions between the subgenomes. Because of the high similarity between homeologous chromosomes and particularly genic regions, it is extremely difficult to use short next-generation sequencing technology-derived reads to uniquely localize them to a reference sequence. The availability of high-quality BAC-end sequences from a newly developed BAC library resource has provided a high-quality reference of Sanger reads. With a relatively even distribution across the subgenomes, the BESs are a superior reference with very high quality, allowing mapping of short NGS reads obtained from resequencing that have much higher error rates. In this study, this is achieved by using very high stringencies over the entire length of quality-trimmed NGS reads. However, this mapping approach is not feasible with references that are not specific to allotetraploid cotton. When the reference is known to be of much higher quality compared to typical next-generation assemblies, assay development from <em>in silico</em>–derived SNPs using this reference is also greatly enhanced. The flanking sequence based on the BES reference should theoretically be correct for TM-1, which can represent cultivated cotton with expectedly low levels of overall diversity and high levels of synteny; this hypothesis holds up to experimental testing as large number of SNPs are able to be successfully genotyped. This is likely due to the automatic inclusion of correct haplotype information for homeo-SNP alleles, which has been shown to increase percentage of co-dominant markers and overall success rates within polyploid cotton (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Islam <em>et al.</em> 2015</a>).</p><p id="p-38">Although success rates of the BES-associated SNPs are higher than those of previous NGS and RRL approaches (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Byers <em>et al.</em> 2012</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Rai <em>et al.</em> 2013</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Gore <em>et al.</em> 2014</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Van Deynze <em>et al.</em> 2009</a>; <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Hulse-Kemp <em>et al.</em> 2014</a>; <a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Zhu <em>et al.</em> 2014</a>; <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Islam <em>et al.</em> 2015</a>), all <em>in silico</em> SNPs were not verified. This could be due to many different reasons, primarily because the BES reference only represents a small portion of the cotton genome, mostly noncoding intergenic regions. Thus, although reads are required to have unique mapping to the reference, it is still possible that additional instances of the sequence exist in the overall cotton genome. This is particularly likely due to ancient paleo-polyploid events that have been discovered in the cotton evolutionary lineage (<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Paterson <em>et al.</em> 2012</a>). Even in largely unique genic regions, gene families and duplicates are common, which will create falsely identified SNPs that, when experimentally assayed, will produce unidentifiable clusters and result in unreliable or bad markers.</p></div><div id="sec-20" class="subsection"><h3>Utilization of BES-associated markers in <em>G. hirsutum</em> germplasm</h3><p id="p-39">The availability of large numbers of intraspecific SNP markers is essential for marker-assisted breeding in <em>G. hirsutum</em> lines, particularly with elite cultivars, which exhibit very low levels of polymorphism. Within a typical elite-by-elite single cross, only a very small percentage of markers will subsequently segregate (typically &lt;5%), so it is difficult to map large numbers of markers from intraspecific crosses. To date, this has led to use of interspecific crosses for the majority of mapping in allotetraploid cotton, such as the one utilized in this effort of <em>G. hirsutum</em> (line TM-1) by <em>G. barbadense</em> (line 3-79) (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Yu <em>et al.</em> 2012</a>). However, in this study we see that even within the small set of 48 experimentally tested intraspecific markers (<a href="http://www.g3journal.org/content/suppl/2015/04/09/g3.115.017749.DC1/TableS8.xlsx">Table S8</a>), most of the 32 tested <em>G. hirsutum</em> lines can be uniquely identified. Even the three TM-1 samples obtained from different labs had a small number of identified differences. The KASP assays were largely successful for most samples, except for Fibermax 966 and Deltapine 90, which likely had a much lower DNA concentration when measured via nanodrop, so it was not reaching a genotyping end-point consistently with the other samples and thus led to many uncalled genotypes. Call frequencies for individual markers showed greater variability than sample call frequencies, which is likely due to the need for optimization of PCR conditions for different marker sets. However, when using the Fluidigm system, this is difficult because all marker sets are run under the same conditions. It was found that while SNPs were identified in a relatively small number of samples from U.S. lines, polymorphisms were also transferrable to other germplasm sources, such as the three Australian samples (Delta Opal, Sicot 70, and Siokra 1-4) and one Indian line (MCU-5) included on the panel. This indicates that ascertainment bias may not be very large, and that sample lineage is likely more important than the country of germplasm origin.</p><p id="p-40">Although only small differences exist between lines, the PCA and hierarchical clustering analyses were able to distinguish relationships among the samples. In the PCA, the wild <em>G. hirsutum</em> line TX0231 was markedly different from the cultivated samples, and the Tamcot Sphinx (a MAR program–derived sample) was the next most unique sample, which was expected due to the assumed diversity of those samples. Within the cultivated lines, when zoomed-in to a very small section of the plot, the samples appear to occupy three different clusters. Cluster 1 contains TM-1 and Fibermax 832, cluster 2 contains Acala Maxxa, Paymaster HS-26, M-240 RNR, and PD-1, and cluster 3 contains Sealand 542, SureGrow 747, Stoneville 474, and Coker 312. In <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Fang <em>et al.</em> (2013)</a>, three of the included samples, SureGrow 747, Stoneville 474, and Fibermax 832, were all identified as being in the same group (group 6). Although we found two of the samples in the same cluster, Fibermax 832 was found in a separate cluster. <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Fang <em>et al.</em> (2013)</a> also identified Acala Maxxa and Paymaster HS-26 to occur in the same group (group 7), which correlates with our analysis where both are in cluster 2. The PCA results correlate with the hierarchical clustering analysis with the shared samples showing the greatest relationship: Acala Maxxa and Paymaster HS-26 from Fang group 7, and SureGrow 747 and Stoneville 474 from Fang group 6. The TM-1 sample, which was the sample utilized as a reference, was also found to have the largest individual dissimilarity after the MAR sample (Tamcot Sphinx). It may be possible that this information is correct because TM-1 was a line established in the mid 20th century (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kohel <em>et al.</em> 1970</a>); however, it is also likely that due to ascertainment bias from using this sample as the reference, it looks more different than the other cultivated <em>G. hirsutum</em> lines.</p></div><div id="sec-21" class="subsection"><h3>Development of BES-associated markers is helpful for integration of physical and genetic maps</h3><p id="p-41">The abundance of markers and accuracy of localization using SNPs associated with BAC-end sequences will be extremely helpful for integrating an allotetraploid cotton physical map with genetic maps. On finalization of an allotetraploid cotton physical map, BES-associated SNPs can also be utilized to integrate unplaced contigs and singletons to enhance the completion of a quality draft reference sequence. Fine- mapping utilizing BES-associated SNPs with a large population can also be used to correct ordering and localization of contigs. Integrating genetic maps with quantitative trait loci (QTL) mapping will allow for utilization of BAC resources for QTL cloning and fine-scale investigation of important regions in the cotton genome.</p></div></div><div class="section conclusions" id="sec-22"><h2 class="">Conclusions</h2><p id="p-42">The largest set of intraspecific and interspecific SNPs for cultivated cotton to date has been developed. These SNPs are associated with BACs and will serve as an interface between future physical and genetic maps. They were developed using genomic sequences from multiple lines and species aligned to BAC-end sequences generated by Sanger sequencing, which provided a high-quality reference. Experimental validation was highly successful and indicated that the SNPs will allow for future high-density mapping. Furthermore, additional lines can be resequenced and quickly genotyped for the identified SNP positions as shown here using the GATK software. The developed markers complement currently available genic-based SNPs and simple sequence repeat markers, and provide a largely evenly distributed set of markers for mapping the entire cotton genome with high-density. This will promote more extensive genomic-based studies and breeding of cotton.</p></div><div class="section" id="sec-23"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-43">The authors acknowledge the Whole Systems Genomics Initiative (WSGI) for providing computational resources and systems administration support for the WSGI HPC cluster, and Kranti Konganti for assistance with utilizing the WSGI resources. The authors thank Fei Wang for maintaining the AgriGenomics Laboratory where KASP assays were performed, Alan Pepper (TAMU) for providing DNA for TX0231, and Iain Wilson’s group (CSIRO) for providing DNA for Australian and Indian lines included in <em>G. hirsutum</em> screening. This work was supported by funding from Cotton Incorporated grant no. 03–430 and the National Science Foundation, Plant Genome Research Program (NSF-PGRP) grant no. IOS1025947.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-2"><p id="p-44">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017749/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017749/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-45"><em>Communicating editor: E. Akhunov</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-03-02"><span class="received-label">Received </span>March 2, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-03-30"><span class="accepted-label">Accepted </span>March 30, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Hulse-Kemp <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.1" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashrafi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hulse-Kemp</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A Long-Read Transcriptome Assembly of Cotton (<em>Gossypium hirsutum</em> L.) and Intraspecific Single Nucleotide Polymorphism Discovery.</span> <abbr class="cit-jnl-abbrev">The Plant Genome</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BGenome%26rft.volume%253D8%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.3" data-doi="10.1371/journal.pone.0045739"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blenda</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rami</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garsmeur</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A high density consensus genetic map of tetraploid cotton that integrates multiple component maps through molecular marker redundancy check.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e45739</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBlenda%26rft.auinit1%253DA.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253De45739%26rft.epage%253De45739%26rft.atitle%253DA%2Bhigh%2Bdensity%2Bconsensus%2Bgenetic%2Bmap%2Bof%2Btetraploid%2Bcotton%2Bthat%2Bintegrates%2Bmultiple%2Bcomponent%2Bmaps%2Bthrough%2Bmolecular%2Bmarker%2Bredundancy%2Bcheck.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0045739%26rft_id%253Dinfo%253Apmid%252F23029214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0045739&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23029214&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.4" data-doi="10.1186/1471-2229-11-56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bohra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubey</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saxena</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Penmetsa</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poornima</span>  <span class="cit-name-given-names">K. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Analysis of BAC-end sequences (BESs) and development of BES-SSR markers for genetic mapping and hybrid purity assessment in pigeonpea (<em>Cajanus</em> spp.).</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DBohra%26rft.auinit1%253DA.%26rft.volume%253D11%26rft.spage%253D56%26rft.epage%253D56%26rft.atitle%253DAnalysis%2Bof%2BBAC-end%2Bsequences%2B%2528BESs%2529%2Band%2Bdevelopment%2Bof%2BBES-SSR%2Bmarkers%2Bfor%2Bgenetic%2Bmapping%2Band%2Bhybrid%2Bpurity%2Bassessment%2Bin%2Bpigeonpea%2B%2528Cajanus%2Bspp.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-11-56%26rft_id%253Dinfo%253Apmid%252F21447154%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-11-56&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21447154&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.5" data-doi="10.1007/s00122-011-1780-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Byers</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harker</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yourstone</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maughan</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development and mapping of SNP assays in allotetraploid cotton.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">1201</span>–<span class="cit-lpage">1214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DByers%26rft.auinit1%253DR.%2BL.%26rft.volume%253D124%26rft.issue%253D7%26rft.spage%253D1201%26rft.epage%253D1214%26rft.atitle%253DDevelopment%2Band%2Bmapping%2Bof%2BSNP%2Bassays%2Bin%2Ballotetraploid%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1780-8%26rft_id%253Dinfo%253Apmid%252F22252442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1780-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22252442&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.6" data-doi="10.1186/1471-2156-3-19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ching</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caldwell</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dolan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">O. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">SNP frequency, haplotype structure and linkage disequilibrium in elite maize inbred lines.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DChing%26rft.auinit1%253DA.%26rft.volume%253D3%26rft.spage%253D19%26rft.epage%253D19%26rft.atitle%253DSNP%2Bfrequency%252C%2Bhaplotype%2Bstructure%2Band%2Blinkage%2Bdisequilibrium%2Bin%2Belite%2Bmaize%2Binbred%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-3-19%26rft_id%253Dinfo%253Apmid%252F12366868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-3-19&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12366868&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: e19379.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.8" data-doi="10.1007/s10681-013-0886-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinze</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Percy</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A microsatellite-based genome-wide analysis of genetic diversity and linkage disequilibrium in Upland cotton (<em>Gossypium hirsutum</em> L.) cultivars from major cotton-growing countries.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">391</span>–<span class="cit-lpage">401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D191%26rft.spage%253D391%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-013-0886-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-013-0886-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.9" data-doi="10.1007/s00438-006-0106-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frelichowski</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Main</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomkins</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantrell</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Cotton genome mapping with new microsatellites from Acala ’Maxxa’ BAC-ends.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">275</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DFrelichowski%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D275%26rft.issue%253D5%26rft.spage%253D479%26rft.epage%253D491%26rft.atitle%253DCotton%2Bgenome%2Bmapping%2Bwith%2Bnew%2Bmicrosatellites%2Bfrom%2BAcala%2B%2527Maxxa%2527%2BBAC-ends.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-006-0106-z%26rft_id%253Dinfo%253Apmid%252F16501995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-006-0106-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16501995&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237504600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.10" data-doi="10.1186/gb-2010-11-8-r86"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goecks</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nekrutenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Galaxy: a comprehensive approach for supporting accessible, reproducible, and transparent computational research in the life sciences.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DGoecks%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D8%26rft.spage%253DR86%26rft.epage%253DR86%26rft.atitle%253DGalaxy%253A%2Ba%2Bcomprehensive%2Bapproach%2Bfor%2Bsupporting%2Baccessible%252C%2Breproducible%252C%2Band%2Btransparent%2Bcomputational%2Bresearch%2Bin%2Bthe%2Blife%2Bsciences.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-8-r86%26rft_id%253Dinfo%253Apmid%252F20738864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-8-r86&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20738864&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.11" data-doi="10.1101/gr.8.3.195"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abajian</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Consed: a graphical tool for sequence finishing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DGordon%26rft.auinit1%253DD.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D195%26rft.epage%253D202%26rft.atitle%253DConsed%253A%2BA%2BGraphical%2BTool%2Bfor%2BSequence%2BFinishing%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.8.3.195%26rft_id%253Dinfo%253Apmid%252F9521923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjgvMy8xOTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTA5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-book" id="cit-5.6.1095.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gore</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Percy</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source">Linkage map construction and quantitative trait locus analysis of agronomic and fiber quality traits in cotton</span>, <span class="cit-publ-name">The Plant Genome</span> <span class="cit-vol">7</span>(<span class="cit-issue">1</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.13" data-doi="10.1016/j.ygeno.2008.11.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chagne</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gasic</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rikkerink</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beever</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">BAC-end sequence-based SNPs and bin mapping for rapid integration of physical and genetic maps in apple.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">282</span>–<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DHan%26rft.auinit1%253DY.%26rft.volume%253D93%26rft.issue%253D3%26rft.spage%253D282%26rft.epage%253D288%26rft.atitle%253DBAC-end%2Bsequence-based%2BSNPs%2Band%2BBin%2Bmapping%2Bfor%2Brapid%2Bintegration%2Bof%2Bphysical%2Band%2Bgenetic%2Bmaps%2Bin%2Bapple.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2008.11.005%26rft_id%253Dinfo%253Apmid%252F19059473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2008.11.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19059473&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263901000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.14" data-doi="10.1371/journal.pone.0033071"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hansey</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaillancourt</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sekhon</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Leon</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaeppler</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Maize (<em>Zea mays</em> L.) genome diversity as revealed by RNA-sequencing.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e33071</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DHansey%26rft.auinit1%253DC.%2BN.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253De33071%26rft.epage%253De33071%26rft.atitle%253DMaize%2B%2528Zea%2Bmays%2BL.%2529%2Bgenome%2Bdiversity%2Bas%2Brevealed%2Bby%2BRNA-sequencing.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0033071%26rft_id%253Dinfo%253Apmid%252F22438891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0033071&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22438891&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.15" data-doi="10.1093/aob/mci078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hendrix</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Estimation of the nuclear DNA content of <em>Gossypium</em> species.</span> <abbr class="cit-jnl-abbrev">Ann. Bot. (Lond.)</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">797</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%2B%2528Lond.%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmci078%26rft_id%253Dinfo%253Apmid%252F15701660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW5uYm90IjtzOjU6InJlc2lkIjtzOjg6Ijk1LzUvNzg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEwOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.16" data-doi="10.1186/1471&amp;ndash;2164&amp;ndash;15&amp;ndash;945"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hulse-Kemp</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashrafi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoegenauer</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Development and bin mapping of gene-associated interspecific SNPs for cotton (<em>Gossypium hirsutum</em> L.) introgression breeding efforts.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">945</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D945%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471%2526ndash%253B2164%2526ndash%253B15%2526ndash%253B945%26rft_id%253Dinfo%253Apmid%252F25359292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471&amp;ndash;2164&amp;ndash;15&amp;ndash;945&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25359292&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.17" data-doi="10.1186/1471-2164-11-38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hyten</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cannon</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weeks</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fickus</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">High-throughput SNP discovery through deep resequencing of a reduced representation library to anchor and orient scaffolds in the soybean whole genome sequence.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DHyten%26rft.auinit1%253DD.%2BL.%26rft.volume%253D11%26rft.spage%253D38%26rft.epage%253D38%26rft.atitle%253DHigh-throughput%2BSNP%2Bdiscovery%2Bthrough%2Bdeep%2Bresequencing%2Bof%2Ba%2Breduced%2Brepresentation%2Blibrary%2Bto%2Banchor%2Band%2Borient%2Bscaffolds%2Bin%2Bthe%2Bsoybean%2Bwhole%2Bgenome%2Bsequence.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-38%26rft_id%253Dinfo%253Apmid%252F20078886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-38&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20078886&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-book" id="cit-5.6.1095.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Islam</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thyssen</span>  <span class="cit-name-given-names">G. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">D. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-source"><em>Detection</em>, <em>validation and application of genotyping-by-sequencing based single nucleotide polymorphisms in upland cotton (Gossypium hirsutum L.)</em></span>, <span class="cit-publ-name">The Plant Genome</span> <span class="cit-vol">8</span>(<span class="cit-issue">1</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.19" data-doi="10.1017/S0016672311000279"><div class="cit-metadata"><cite>Van Ooijen<span class="cit-auth cit-collab">J.W.</span>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Multipoint maximum likelihood mapping in a full-sib family of an outbreeding species.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DVAN%2BOoijen%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D93%26rft.issue%253D5%26rft.spage%253D343%26rft.epage%253D349%26rft.atitle%253DMultipoint%2Bmaximum%2Blikelihood%2Bmapping%2Bin%2Ba%2Bfull-sib%2Bfamily%2Bof%2Ban%2Boutbreeding%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672311000279%26rft_id%253Dinfo%253Apmid%252F21878144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672311000279&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21878144&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.20" data-doi="10.2135/cropsci1970.0011183X001000060019x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kohel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richmond</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Texas marker-1. Description of a genetic standard for <em>Gossypium hirsutum</em> L.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">670</span>–<span class="cit-lpage">671</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DKohel%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D670%26rft.epage%253D671%26rft.atitle%253DTexas%2BMarker-1.%2BDescription%2Bof%2Ba%2BGenetic%2BStandard%2Bfor%2BGossypium%2Bhirsutum%2BL.%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1970.0011183X001000060019x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1970.0011183X001000060019x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.21" data-doi="10.1038/ng.715"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Resequencing of 31 wild and cultivated soybean genomes identifies patterns of genetic diversity and selection.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">1053</span>–<span class="cit-lpage">1059</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DLam%26rft.auinit1%253DH.%2BM.%26rft.volume%253D42%26rft.issue%253D12%26rft.spage%253D1053%26rft.epage%253D1059%26rft.atitle%253DResequencing%2Bof%2B31%2Bwild%2Band%2Bcultivated%2Bsoybean%2Bgenomes%2Bidentifies%2Bpatterns%2Bof%2Bgenetic%2Bdiversity%2Band%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.715%26rft_id%253Dinfo%253Apmid%252F21076406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.715&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21076406&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.22" data-doi="10.1038/ng.2987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">G. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome sequence of the cultivated cotton <em>Gossypium arboreum</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">567</span>–<span class="cit-lpage">572</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D567%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2987%26rft_id%253Dinfo%253Apmid%252F24836287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24836287&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mammadov</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aggarwal</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buyyarapu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumpatla</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SNP markers and their impact on plant breeding.</span> <abbr class="cit-jnl-abbrev">Int. J. Plant Genomics</abbr> <span class="cit-vol">2012</span>: <span class="cit-fpage">728398</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BPlant%2BGenomics%26rft.volume%253D2012%26rft.spage%253D728398%26rft_id%253Dinfo%253Apmid%252F23316221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23316221&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.24" data-doi="10.1186/1471-2164-13-13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ollitrault</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terol</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Lor</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chauveau</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SNP mining in <em>C. clementina</em> BAC end sequences; transferability in the Citrus genus (Rutaceae), phylogenetic inferences and perspectives for genetic mapping.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DOllitrault%26rft.auinit1%253DP.%26rft.volume%253D13%26rft.spage%253D13%26rft.epage%253D13%26rft.atitle%253DSNP%2Bmining%2Bin%2BC.%2Bclementina%2BBAC%2Bend%2Bsequences%253B%2Btransferability%2Bin%2Bthe%2BCitrus%2Bgenus%2B%2528Rutaceae%2529%252C%2Bphylogenetic%2Binferences%2Band%2Bperspectives%2Bfor%2Bgenetic%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-13%26rft_id%253Dinfo%253Apmid%252F22233093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22233093&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.25" data-doi="10.1038/nature11798"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gundlach</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Repeated polyploidization of <em>Gossypium</em> genomes and the evolution of spinnable cotton fibres.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">492</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D492%26rft.spage%253D423%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11798%26rft_id%253Dinfo%253Apmid%252F23257886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11798&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23257886&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312488200055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.26" data-doi="10.1139/g66-011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strickland</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">The cytology of a hybrid between <em>Gossypium hirsutum</em> and <em>G. longicalyx</em>.</span> <abbr class="cit-jnl-abbrev">Can. J. Genet. Cytol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">95</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BGenet.%2BCytol.%26rft.volume%253D8%26rft.spage%253D91%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg66-011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g66-011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A19667584400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: e32253.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-book" id="cit-5.6.1095.28"><div class="cit-metadata unstructured">R Development Core Team, 2010 <em>R: A language and environment for statistical computing</em>, edited by R Development Core Team. R Foundation for Statistical Computing, Vienna.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.29" data-doi="10.1111/pbi.12088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rai</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhardwaj</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lakhwani</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Large-scale resource development in <em>Gossypium hirsutum</em> L. by 454 sequencing of genic-enriched libraries from six diverse genotypes.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">953</span>–<span class="cit-lpage">963</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D11%26rft.spage%253D953%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12088%26rft_id%253Dinfo%253Apmid%252F23782852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23782852&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.30" data-doi="10.1104/pp.103.038463"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">Y. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xi</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Development of genome-wide DNA polymorphism database for map-based cloning of rice genes.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">1198</span>–<span class="cit-lpage">1205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DShen%26rft.auinit1%253DY.-J.%26rft.volume%253D135%26rft.issue%253D3%26rft.spage%253D1198%26rft.epage%253D1205%26rft.atitle%253DDevelopment%2Bof%2BGenome-Wide%2BDNA%2BPolymorphism%2BDatabase%2Bfor%2BMap-Based%2BCloning%2Bof%2BRice%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.103.038463%26rft_id%253Dinfo%253Apmid%252F15266053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTM1LzMvMTE5OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMDk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-book" id="cit-5.6.1095.31"><div class="cit-metadata unstructured">Tarailo-Graovac, M., and N. Chen, 2009 Using RepeatMasker to identify repetitive elements in genomic sequences. Curr. Protoc. Bioinformatics Chapter 4:Unit 4: 10.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.32" data-doi="10.1186/1471-2229-9-125"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Deynze</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoffel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkins</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Sampling nucleotide diversity in cotton.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">125</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DVan%2BDeynze%26rft.auinit1%253DA.%2BE.%26rft.volume%253D9%26rft.spage%253D125%26rft.epage%253D125%26rft.atitle%253DSampling%2Bnucleotide%2Bdiversity%2Bin%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-9-125%26rft_id%253Dinfo%253Apmid%252F19840401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-9-125&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19840401&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.33" data-doi="10.1186/1471-2229-12-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Penmetsa</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development and characterization of BAC-end sequence derived SSRs, and their incorporation into a new higher density genetic map for cultivated peanut (<em>Arachis hypogaea</em> L.).</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DWang%26rft.auinit1%253DH.%26rft.volume%253D12%26rft.spage%253D10%26rft.epage%253D10%26rft.atitle%253DDevelopment%2Band%2Bcharacterization%2Bof%2BBAC-end%2Bsequence%2Bderived%2BSSRs%252C%2Band%2Btheir%2Bincorporation%2Binto%2Ba%2Bnew%2Bhigher%2Bdensity%2Bgenetic%2Bmap%2Bfor%2Bcultivated%2Bpeanut%2B%2528Arachis%2Bhypogaea%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-12-10%26rft_id%253Dinfo%253Apmid%252F22260238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-12-10&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22260238&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-book" id="cit-5.6.1095.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brubaker</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alvarez</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cronn</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Evolution and natural history of the cotton genus</span>, pp. <span class="cit-fpage">3</span>–<span class="cit-lpage">22</span> in <span class="cit-source">Genetics and Genomics of Cotton</span>, edited by A. H. <span class="cit-ed"><span class="cit-name-surname">Paterson</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-other" id="cit-5.6.1095.35"><div class="cit-metadata unstructured">Yu, J.Z., R.J. Kohel, D.D. Fang, J. Cho, A. Van Deynze <em>et al.</em>, 2012 A high-density simple sequence repeat and single nucleotide polymorphism genetic map of the tetraploid cotton genome. G3 (Bethesda) 2:43–58.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1095.36" data-doi="10.1093/bioinformatics/bts606"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levine</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogarten</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laurie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A high-performance computing toolset for relatedness and principal component analysis of SNP data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">3326</span>–<span class="cit-lpage">3328</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts606%26rft_id%253Dinfo%253Apmid%252F23060615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMjQvMzMyNiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMDk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-other" id="cit-5.6.1095.37"><div class="cit-metadata unstructured">Zhu, Q.H., A. Spriggs, J.M. Taylor, D. Llewellyn, and I. Wilson, 2014 Transcriptome and complexity-reduced, DNA-based identification of intraspecies single-nucleotide polymorphisms in the polyploid <em>Gossypium hirsutum</em> L. G3 (Bethesda) 4:1893–1905.</div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/6/1095.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/6/1081" title="Insights into Ongoing Evolution of the Hexachlorocyclohexane Catabolic Pathway from Comparative Genomics of Ten Sphingomonadaceae Strains" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/6/1107" title="Novel Genes Critical for Hypoxic Preconditioning in Zebrafish Are Regulators of Insulin and Glucose Metabolism" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/6"><span>Volume 5 Issue 6, June 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/6.cover-source.jpg?itok=NSNFWJbY" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1095.short?cited-by=yes&amp;legid=ggg%3Bg3.115.017749v1" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-_g5-o0gVZAPZrN7Yvbi5taX88FaRMczeI7WPD7agGkM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/6/1095" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/6/1095" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/6/1095" class="active" data-icon-position="" data-hide-link-title="0">BAC-End Sequence-Based SNP Mining in Allotetraploid Cotton (Gossypium) Utilizing Resequencing Data, Phylogenetic Inferences, and Perspectives for Genetic Mapping</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459907" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-R6CIONCrzIvqKMUjU4t0rDejd2Fxb4lFsxY9YoNgUJA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/6/1095.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/6/1095" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-6DySVOkJ2qyJ7ySZXod6Vg2KNxDWHYiZTQTN9TnJAT8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/6/1095" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/6/1095.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459907" id="node459907--2" data-pisa="ggg;5/6/1095" data-pisa-master="ggg;g3.115.017749" data-apath="/ggg/5/6/1095.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">BAC-End Sequence-Based SNP Mining in Allotetraploid Cotton (<em>Gossypium</em>) Utilizing Resequencing Data, Phylogenetic Inferences, and Perspectives for Genetic Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9670-9433" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amanda M. Hulse-Kemp</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2094-2367" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Hamid Ashrafi</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-3468-4119" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kevin Stoffel</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-8574-4708" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xiuting Zheng</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-2780-4274" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christopher A. Saski</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-1968-8952" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Brian E. Scheffler</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-0036-5459" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David D. Fang</span>, <span class="highwire-citation-author" data-delta="7">Z. Jeffrey Chen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-2093-0577" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Allen Van Deynze</span> and <span class="highwire-citation-author" data-delta="9">David M. Stelly</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1095-1105;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017749 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459907--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Amanda M. Hulse-Kemp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Soil and Crop Sciences, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdisciplinary Genetics Program, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amanda%2BM.%2BHulse-Kemp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hulse-Kemp%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmanda%2BM.%2BHulse-Kemp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9670-9433" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amanda M. Hulse-Kemp</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Hamid Ashrafi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Seed Biotechnology Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hamid%2BAshrafi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ashrafi%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHamid%2BAshrafi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2094-2367" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hamid Ashrafi</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kevin Stoffel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Seed Biotechnology Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BStoffel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stoffel%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKevin%2BStoffel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3468-4119" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kevin Stoffel</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Xiuting Zheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Soil and Crop Sciences, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiuting%2BZheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zheng%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXiuting%2BZheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8574-4708" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xiuting Zheng</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Christopher A. Saski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Clemson University Genomics Institute, Clemson University, Clemson, South Carolina 29634</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BA.%2BSaski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Saski%20CA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristopher%2BA.%2BSaski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2780-4274" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christopher A. Saski</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Brian E. Scheffler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>USDA-ARS, Genomics and Bioinformatics Research Unit, Stoneville, Mississippi 38766</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BE.%2BScheffler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scheffler%20BE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrian%2BE.%2BScheffler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1968-8952" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Brian E. Scheffler</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">David D. Fang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>USDA-ARS, Cotton Fiber Bioscience Research Unit, New Orleans, Louisiana 70124</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BD.%2BFang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fang%20DD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BD.%2BFang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0036-5459" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David D. Fang</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Z. Jeffrey Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Molecular Biosciences, Center for Computational Biology and Bioinformatics, and Institute for Cellular and Molecular Biology, The University of Texas, Austin, Texas 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Z.%2BJeffrey%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20ZJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZ.%2BJeffrey%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Allen Van Deynze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Seed Biotechnology Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allen%2BVan%2BDeynze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Deynze%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAllen%2BVan%2BDeynze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2093-0577" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Allen Van Deynze</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">David M. Stelly </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Soil and Crop Sciences, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdisciplinary Genetics Program, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BM.%2BStelly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stelly%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BM.%2BStelly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:stelly@tamu.edu" class="" data-icon-position="" data-hide-link-title="0">stelly@tamu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/6/1095" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/6/1095&amp;title=BAC-End%20Sequence-Based%20SNP%20Mining%20in%20Allotetraploid%20Cotton%20%28Gossypium%29%20Utilizing%20Resequencing%20Data%2C%20Phylogenetic%20Inferences%2C%20and%20Perspectives%20for%20Genetic%20Mapping" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/6/1095&amp;title=BAC-End%20Sequence-Based%20SNP%20Mining%20in%20Allotetraploid%20Cotton%20%28Gossypium%29%20Utilizing%20Resequencing%20Data%2C%20Phylogenetic%20Inferences%2C%20and%20Perspectives%20for%20Genetic%20Mapping" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/6/1095&amp;title=BAC-End%20Sequence-Based%20SNP%20Mining%20in%20Allotetraploid%20Cotton%20%28Gossypium%29%20Utilizing%20Resequencing%20Data%2C%20Phylogenetic%20Inferences%2C%20and%20Perspectives%20for%20Genetic%20Mapping" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/6/1095&amp;text=BAC-End%20Sequence-Based%20SNP%20Mining%20in%20Allotetraploid%20Cotton%20%28Gossypium%29%20Utilizing%20Resequencing%20Data%2C%20Phylogenetic%20Inferences%2C%20and%20Perspectives%20for%20Genetic%20Mapping" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/6/1095&amp;title=BAC-End%20Sequence-Based%20SNP%20Mining%20in%20Allotetraploid%20Cotton%20%28Gossypium%29%20Utilizing%20Resequencing%20Data%2C%20Phylogenetic%20Inferences%2C%20and%20Perspectives%20for%20Genetic%20Mapping" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/6/1095&amp;t=BAC-End%20Sequence-Based%20SNP%20Mining%20in%20Allotetraploid%20Cotton%20%28Gossypium%29%20Utilizing%20Resequencing%20Data%2C%20Phylogenetic%20Inferences%2C%20and%20Perspectives%20for%20Genetic%20Mapping" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/6/1095&amp;title=BAC-End%20Sequence-Based%20SNP%20Mining%20in%20Allotetraploid%20Cotton%20%28Gossypium%29%20Utilizing%20Resequencing%20Data%2C%20Phylogenetic%20Inferences%2C%20and%20Perspectives%20for%20Genetic%20Mapping" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/6/1095&amp;title=BAC-End%20Sequence-Based%20SNP%20Mining%20in%20Allotetraploid%20Cotton%20%28Gossypium%29%20Utilizing%20Resequencing%20Data%2C%20Phylogenetic%20Inferences%2C%20and%20Perspectives%20for%20Genetic%20Mapping" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459907/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459907" id="node459907--3" data-pisa="ggg;5/6/1095" data-pisa-master="ggg;g3.115.017749" data-apath="/ggg/5/6/1095.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">BAC-End Sequence-Based SNP Mining in Allotetraploid Cotton (<em>Gossypium</em>) Utilizing Resequencing Data, Phylogenetic Inferences, and Perspectives for Genetic Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9670-9433" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amanda M. Hulse-Kemp</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2094-2367" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Hamid Ashrafi</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-3468-4119" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kevin Stoffel</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-8574-4708" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xiuting Zheng</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-2780-4274" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christopher A. Saski</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-1968-8952" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Brian E. Scheffler</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-0036-5459" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David D. Fang</span>, <span class="highwire-citation-author" data-delta="7">Z. Jeffrey Chen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-2093-0577" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Allen Van Deynze</span> and <span class="highwire-citation-author" data-delta="9">David M. Stelly</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1095-1105;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017749 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459907--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Amanda M. Hulse-Kemp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Soil and Crop Sciences, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdisciplinary Genetics Program, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amanda%2BM.%2BHulse-Kemp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hulse-Kemp%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmanda%2BM.%2BHulse-Kemp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9670-9433" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amanda M. Hulse-Kemp</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Hamid Ashrafi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Seed Biotechnology Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hamid%2BAshrafi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ashrafi%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHamid%2BAshrafi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2094-2367" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hamid Ashrafi</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kevin Stoffel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Seed Biotechnology Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BStoffel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stoffel%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKevin%2BStoffel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3468-4119" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kevin Stoffel</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Xiuting Zheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Soil and Crop Sciences, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiuting%2BZheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zheng%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXiuting%2BZheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8574-4708" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xiuting Zheng</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Christopher A. Saski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Clemson University Genomics Institute, Clemson University, Clemson, South Carolina 29634</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BA.%2BSaski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Saski%20CA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristopher%2BA.%2BSaski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2780-4274" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christopher A. Saski</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Brian E. Scheffler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>USDA-ARS, Genomics and Bioinformatics Research Unit, Stoneville, Mississippi 38766</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BE.%2BScheffler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scheffler%20BE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrian%2BE.%2BScheffler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1968-8952" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Brian E. Scheffler</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">David D. Fang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>USDA-ARS, Cotton Fiber Bioscience Research Unit, New Orleans, Louisiana 70124</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BD.%2BFang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fang%20DD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BD.%2BFang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0036-5459" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David D. Fang</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Z. Jeffrey Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Molecular Biosciences, Center for Computational Biology and Bioinformatics, and Institute for Cellular and Molecular Biology, The University of Texas, Austin, Texas 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Z.%2BJeffrey%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20ZJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZ.%2BJeffrey%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Allen Van Deynze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Seed Biotechnology Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allen%2BVan%2BDeynze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Deynze%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAllen%2BVan%2BDeynze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2093-0577" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Allen Van Deynze</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">David M. Stelly </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Soil and Crop Sciences, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdisciplinary Genetics Program, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BM.%2BStelly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stelly%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BM.%2BStelly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:stelly@tamu.edu" class="" data-icon-position="" data-hide-link-title="0">stelly@tamu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1095&amp;rft.epage=1105&amp;rft.atitle=BAC-End%20Sequence-Based%20SNP%20Mining%20in%20Allotetraploid%20Cotton%20%28Gossypium%29%20Utilizing%20Resequencing%20Data%2C%20Phylogenetic%20Inferences%2C%20and%20Perspectives%20for%20Genetic%20Mapping&amp;rft.volume=5&amp;rft.issue=6&amp;rft.date=2015-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Hulse-Kemp%2C+Amanda+M.&amp;rft.au=Ashrafi%2C+Hamid&amp;rft.au=Stoffel%2C+Kevin&amp;rft.au=Zheng%2C+Xiuting&amp;rft.au=Saski%2C+Christopher+A.&amp;rft.au=Scheffler%2C+Brian+E.&amp;rft.au=Fang%2C+David+D.&amp;rft.au=Chen%2C+Z.+Jeffrey&amp;rft.au=Van+Deynze%2C+Allen&amp;rft.au=Stelly%2C+David+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459907/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459907/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459907/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459907/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459907/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459907/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459907/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459907/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459907/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459907/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459907/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459907/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/6/1095" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-22" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/6/1095.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/6/1095.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/6/1095.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/6/1095.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
