<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/6/1247" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/6/1247.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/6/1247.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/6/1247.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/6/1247/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/6/1247.atom" />
<meta name="HW.pisa" content="ggg;5/6/1247" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Effects of Genic Base Composition on Growth Rate in G+C-rich Genomes" />
<meta name="DC.Identifier" content="10.1534/g3.115.016824" />
<meta name="DC.Date" content="2015-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2015 Kelkar et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="10.1534/g3.115.024042" />
<meta name="DC.Description" content="The source and significance of the wide variation in the genomic base composition of bacteria have been a matter of continued debate. Although the variation was originally attributed to a strictly neutral process, i.e. , species-specific differences in mutational patterns, recent genomic comparisons have shown that bacteria with G+C-rich genomes experience a mutational bias toward A+T. This difference between the mutational input to a genome and its overall base composition suggests the action of natural selection. Here, we examine if selection acts on G+C contents in Caulobacter crescentus and Pseudomonas aeruginosa , which both have very G+C-rich genomes, by testing whether the expression of gene variants that differ only in their base compositions at synonymous sites affects cellular growth rates. In C. crescentus , expression of the more A+T-rich gene variants decelerated growth, indicating that selection on genic base composition is, in part, responsible for the high G+C content of this genome. In contrast, no comparable effect was observed in P. aeruginosa , which has similarly high genome G+C contents. Selection for increased genic G+C-contents in C. crescentus acts independently of the species-specific codon usage pattern and represents an additional selective force operating in bacterial genomes." />
<meta name="DC.Contributor" content="Yogeshwar D. Kelkar" />
<meta name="DC.Contributor" content="Daniel S. Phillips" />
<meta name="DC.Contributor" content="Howard Ochman" />
<meta name="article:published_time" content="2015-06-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Effects of Genic Base Composition on Growth Rate in G+C-rich Genomes" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The source and significance of the wide variation in the genomic base composition of bacteria have been a matter of continued debate. Although the variation was originally attributed to a strictly neutral process, &lt;i&gt;i.e.&lt;/i&gt;, species-specific differences in mutational patterns, recent genomic comparisons have shown that bacteria with G+C-rich genomes experience a mutational bias toward A+T. This difference between the mutational input to a genome and its overall base composition suggests the action of natural selection. Here, we examine if selection acts on G+C contents in &lt;i&gt;Caulobacter crescentus&lt;/i&gt; and &lt;i&gt;Pseudomonas aeruginosa&lt;/i&gt;, which both have very G+C-rich genomes, by testing whether the expression of gene variants that differ only in their base compositions at synonymous sites affects cellular growth rates. In &lt;i&gt;C. crescentus&lt;/i&gt;, expression of the more A+T-rich gene variants decelerated growth, indicating that selection on genic base composition is, in part, responsible for the high G+C content of this genome. In contrast, no comparable effect was observed in &lt;i&gt;P. aeruginosa&lt;/i&gt;, which has similarly high genome G+C contents. Selection for increased genic G+C-contents in &lt;i&gt;C. crescentus&lt;/i&gt; acts independently of the species-specific codon usage pattern and represents an additional selective force operating in bacterial genomes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/06/01" />
<meta name="citation_mjid" content="ggg;5/6/1247" />
<meta name="citation_id" content="5/6/1247" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/6/1247" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/6/1247.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/6/1247.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/6/1247.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.016824" />
<meta name="citation_pmid" content="25897009" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="6" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1247" />
<meta name="citation_lastpage" content="1252" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Yogeshwar D. Kelkar" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, Yale University, New Haven, Connecticut 06520" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4440-0191" />
<meta name="citation_author" content="Daniel S. Phillips" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, Yale University, New Haven, Connecticut 06520" />
<meta name="citation_author" content="Howard Ochman" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, Yale University, New Haven, Connecticut 06520" />
<meta name="citation_author_email" content="howard.ochman@austin.utexas.edu" />
<meta name="citation_reference" content="citation_journal_title=Biochemical and biophysical research communications;citation_journal_abbrev=Biochem Biophys Res Commun;citation_author=S. Basak;citation_author=T. C. Ghosh;citation_title=On the origin of genomic adaptation at high temperature for prokaryotic organisms.;citation_pages=629-632;citation_volume=330;citation_year=2005;citation_issue=3;citation_pmid=15809043;citation_doi=10.1016/j.bbrc.2005.02.134" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=S. D. Bentley;citation_author=J. Parkhill;citation_title=Comparative genomic structure of prokaryotes.;citation_pages=771-792;citation_volume=38;citation_year=2004;citation_pmid=15568993;citation_doi=10.1146/annurev.genet.38.072902.094318" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=J. A. Birdsell;citation_title=Integrating Genomics, Bioinformatics, and Classical Genetics to Study the Effects of Recombination on Genome Evolution;citation_pages=1181-1197;citation_volume=19;citation_year=2002;citation_issue=7;citation_pmid=12082137;citation_doi=10.1093/oxfordjournals.molbev.a004176" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. Bohlin;citation_author=L. Snipen;citation_author=S. P. Hardy;citation_author=A. B. Kristoffersen;citation_author=K. Lagesen;citation_title=Analysis of intra-genomic GC content homogeneity within prokaryotes.;citation_pages=464-464;citation_volume=11;citation_year=2010;citation_pmid=20691090;citation_doi=10.1186/1471-2164-11-464" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=E. C. Cox;citation_author=C. Yanofsky;citation_title=Altered base ratios in the DNA of an Escherichia coli mutator strain.;citation_pages=1895-1902;citation_volume=58;citation_year=1967;citation_issue=5;citation_pmid=4866980;citation_doi=10.1073/pnas.58.5.1895" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=H. Dong;citation_author=L. Nilsson;citation_author=C. G. Kurland;citation_title=Co-variation of tRNA abundance and codon usage in Escherichia coli at different growth rates.;citation_pages=649-663;citation_volume=260;citation_year=1996;citation_issue=5;citation_pmid=8709146;citation_doi=10.1006/jmbi.1996.0428" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genomics and human genetics;citation_journal_abbrev=Annu Rev Genomics Hum Genet;citation_author=L. Duret;citation_author=N. Galtier;citation_title=Biased gene conversion and the evolution of mammalian genomic landscapes.;citation_pages=285-311;citation_volume=10;citation_year=2009;citation_pmid=19630562;citation_doi=10.1146/annurev-genom-082908-150001" />
<meta name="citation_reference" content="citation_journal_title=J. Theor. Biol.;citation_author=E. Freese;citation_title=On the evolution of the base composition of DNA.;citation_pages=82-101;citation_volume=3;citation_year=1962;citation_doi=10.1016/S0022-5193(62)80005-8" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=N. Galtier;citation_author=J. R. Lobry;citation_title=Relationships between genomic G+C content, RNA secondary structures, and optimal growth temperature in prokaryotes.;citation_pages=632-636;citation_volume=44;citation_year=1997;citation_issue=6;citation_pmid=9169555;citation_doi=10.1007/PL00006186" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=R. Hershberg;citation_author=D. A. Petrov;citation_title=Evidence that mutation is universally biased towards AT in bacteria.;citation_pages=e1001115;citation_volume=6;citation_year=2010;citation_pmid=20838599;citation_doi=10.1371/journal.pgen.1001115" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=F. Hildebrand;citation_author=A. Meyer;citation_author=A. Eyre-Walker;citation_title=Evidence of selection upon genomic GC-content in bacteria.;citation_pages=e1001107;citation_volume=6;citation_year=2010;citation_issue=9;citation_pmid=20838593;citation_doi=10.1371/journal.pgen.1001107" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=L. D. Hurst;citation_author=A. R. Merchant;citation_title=High guanine-cytosine content is not an adaptation to high temperature: a comparative analysis amongst prokaryotes;citation_pages=493-497;citation_volume=268;citation_year=2001;citation_issue=1466;citation_pmid=11296861;citation_doi=10.1098/rspb.2000.1397" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=T. Ikemura;citation_title=Codon usage and tRNA content in unicellular and multicellular organisms;citation_pages=13-34;citation_volume=2;citation_year=1985;citation_issue=1;citation_pmid=3916708" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=F. Lassalle;citation_author=S. PÃ©rian;citation_author=T. Bataillon;citation_author=X. Nesme;citation_author=L. Duret;citation_title=GC-Content Evolution in Bacterial Genomes: The Biased Gene Conversion Hypothesis Expands.;citation_pages=e1004941;citation_volume=11;citation_year=2015;citation_pmid=25659072;citation_doi=10.1371/journal.pgen.1004941" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G.-W. Li;citation_author=O. Eugene;citation_author=J. S. Weissman;citation_title=The anti-Shine-Dalgarno sequence drives translational pausing and codon choice in bacteria.;citation_pages=538-541;citation_volume=484;citation_year=2012;citation_issue=7395;citation_pmid=22456704;citation_doi=10.1038/nature10965" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. A. Lind;citation_author=D. I. Andersson;citation_title=Whole-genome mutational biases in bacteria.;citation_volume=105;citation_year=2008;citation_pmid=19001264;citation_doi=10.1073/pnas.0804445105" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=J. P. McCutcheon;citation_author=N. A. Moran;citation_title=Functional convergence in reduced genomes of bacterial symbionts spanning 200 My of evolution.;citation_volume=2;citation_year=2010;citation_pmid=20829280;citation_doi=10.1093/gbe/evq055" />
<meta name="citation_reference" content="citation_journal_title=Hereditas;citation_journal_abbrev=Hereditas;citation_author=C. E. McEwan;citation_author=D. Gatherer;citation_author=N. R. McEwan;citation_title=Nitrogen-fixing aerobic bacteria have higher genomic GC content than non-fixing species within the same genus.;citation_pages=173-178;citation_volume=128;citation_year=1998;citation_issue=2;citation_pmid=9687237;citation_doi=10.1111/j.1601-5223.1998.00173.x" />
<meta name="citation_reference" content="citation_journal_abbrev=Microb Biotechnol;citation_author=A. M. MiroÅczuk;citation_author=Ã. T. KovÃ¡cs;citation_author=O. P. Kuipers;citation_title=Induction of natural competence in Bacillus cereus ATCC14579.;citation_pages=226-235;citation_volume=1;citation_year=2008;citation_issue=3;citation_pmid=21261842;citation_doi=10.1111/j.1751-7915.2008.00023.x" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=A. Muto;citation_author=S. Osawa;citation_title=The guanine and cytosine content of genomic DNA and bacterial evolution.;citation_pages=166-169;citation_volume=84;citation_year=1987;citation_issue=1;citation_pmid=3467347;citation_doi=10.1073/pnas.84.1.166" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=H. Naya;citation_author=H. Romero;citation_author=A. Zavala;citation_author=B. Alvarez;citation_author=H. Musto;citation_title=Aerobiosis increases the genomic guanine plus cytosine content (GC%) in prokaryotes.;citation_pages=260-264;citation_volume=55;citation_year=2002;citation_issue=3;citation_pmid=12187379;citation_doi=10.1007/s00239-002-2323-3" />
<meta name="citation_reference" content="citation_journal_title=Stat. Comput.;citation_title=R: A language and environment for statistical computing. R Found.;citation_pages=409;citation_volume=1;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. Raghavan;citation_author=Y. D. Kelkar;citation_author=H. Ochman;citation_title=A selective force favoring increased G+C content in bacterial genes.;citation_volume=109;citation_year=2012;citation_pmid=22908296;citation_doi=10.1073/pnas.1205683109" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=E. P. C. Rocha;citation_author=A. Danchin;citation_title=Base composition bias might result from competition for metabolic resources.;citation_pages=291-294;citation_volume=18;citation_year=2002;citation_issue=6;citation_pmid=12044357;citation_doi=10.1016/S0168-9525(02)02690-2" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=E. P. C. Rocha;citation_author=E. J. Feil;citation_title=Mutational patterns cannot explain genome composition: Are there any neutral sites in the genomes of bacteria?;citation_pages=e1001104;citation_volume=6;citation_year=2010;citation_pmid=20838590;citation_doi=10.1371/journal.pgen.1001104" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=H. Romero;citation_author=E. Pereira;citation_author=H. Naya;citation_author=H. Musto;citation_title=Oxygen and guanine-cytosine profiles in marine environments.;citation_pages=203-206;citation_volume=69;citation_year=2009;citation_issue=2;citation_pmid=19554248;citation_doi=10.1007/s00239-009-9230-9" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. M. Sharp;citation_author=E. Bailes;citation_author=R. J. Grocock;citation_author=J. F. Peden;citation_author=R. E. Sockett;citation_title=Variation in the strength of selected codon usage bias among bacteria;citation_pages=1141-1153;citation_volume=33;citation_year=2005;citation_issue=4;citation_pmid=15728743;citation_doi=10.1093/nar/gki242" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=P. M. Sharp;citation_author=L. R. Emery;citation_author=K. Zeng;citation_title=Forces that influence the evolution of codon bias;citation_pages=1203-1212;citation_volume=365;citation_year=2010;citation_issue=1544;citation_pmid=20308095;citation_doi=10.1098/rstb.2009.0305" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. M. Sharp;citation_author=W. H. Li;citation_title=The codon adaptation index-a measure of directional synonymous codon usage bias, and its potential applications;citation_pages=1281-1295;citation_volume=15;citation_year=1987;citation_issue=3;citation_pmid=3547335;citation_doi=10.1093/nar/15.3.1281" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=N. Sueoka;citation_title=ON THE GENETIC BASIS OF VARIATION AND HETEROGENEITY OF DNA BASE COMPOSITION;citation_pages=582-592;citation_volume=48;citation_year=1962;citation_issue=4;citation_pmid=13918161;citation_doi=10.1073/pnas.48.4.582" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. H. Thomas;citation_author=R. D. Wagner;citation_author=A. K. Arakaki;citation_author=J. Skolnick;citation_author=J. R. Kirby;citation_title=The mosaic genome of Anaeromyxobacter dehalogenans strain 2CP-C suggests an aerobic common ancestor to the delta-proteobacteria.;citation_pages=e2103-e2103;citation_volume=3;citation_year=2008;citation_issue=5;citation_pmid=18461135;citation_doi=10.1371/journal.pone.0002103" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=M. Touchon;citation_author=C. Hoede;citation_author=O. Tenaillon;citation_author=V. Barve;citation_author=S. Baeriswyl;citation_title=Organised genome dynamics in the Escherichia coli species results in highly diverse adaptive paths.;citation_pages=e1000344-e1000344;citation_volume=5;citation_year=2009;citation_issue=1;citation_pmid=19165319;citation_doi=10.1371/journal.pgen.1000344" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=J. T. Van Leuven;citation_author=J. P. McCutcheon;citation_title=An AT mutational bias in the tiny GC-rich endosymbiont genome of Hodgkinia.;citation_volume=4;citation_year=2012;citation_pmid=22113795;citation_doi=10.1093/gbe/evr125" />
<meta name="citation_reference" content="citation_journal_title=Biochemical and biophysical research communications;citation_journal_abbrev=Biochem Biophys Res Commun;citation_author=H.-C. Wang;citation_author=E. Susko;citation_author=A. J. Roger;citation_title=On the correlation between genomic G+C content and optimal growth temperature in prokaryotes: data quality and confounding factors.;citation_pages=681-684;citation_volume=342;citation_year=2006;citation_issue=3;citation_pmid=16499870;citation_doi=10.1016/j.bbrc.2006.02.037" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Effects of Genic Base Composition on Growth Rate in G+C-rich Genomes" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
<meta name="twitter:description" content="The source and significance of the wide variation in the genomic base composition of bacteria have been a matter of continued debate. Although the variation was originally attributed to a strictly neutral process, i.e. , species-specific differences in mutational patterns, recent genomic comparisons have shown that bacteria with G+C-rich genomes experience a mutational bias toward A+T. This difference between the mutational input to a genome and its overall base composition suggests the action of natural selection. Here, we examine if selection acts on G+C contents in Caulobacter crescentus and Pseudomonas aeruginosa , which both have very G+C-rich genomes, by testing whether the expression of gene variants that differ only in their base compositions at synonymous sites affects cellular growth rates. In C. crescentus , expression of the more A+T-rich gene variants decelerated growth, indicating that selection on genic base composition is, in part, responsible for the high G+C content of this genome. In contrast, no comparable effect was observed in P. aeruginosa , which has similarly high genome G+C contents. Selection for increased genic G+C-contents in C. crescentus acts independently of the species-specific codon usage pattern and represents an additional selective force operating in bacterial genomes." />
<meta name="og-title" property="og:title" content="Effects of Genic Base Composition on Growth Rate in G+C-rich Genomes" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/6/1247" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The source and significance of the wide variation in the genomic base composition of bacteria have been a matter of continued debate. Although the variation was originally attributed to a strictly neutral process, i.e. , species-specific differences in mutational patterns, recent genomic comparisons have shown that bacteria with G+C-rich genomes experience a mutational bias toward A+T. This difference between the mutational input to a genome and its overall base composition suggests the action of natural selection. Here, we examine if selection acts on G+C contents in Caulobacter crescentus and Pseudomonas aeruginosa , which both have very G+C-rich genomes, by testing whether the expression of gene variants that differ only in their base compositions at synonymous sites affects cellular growth rates. In C. crescentus , expression of the more A+T-rich gene variants decelerated growth, indicating that selection on genic base composition is, in part, responsible for the high G+C content of this genome. In contrast, no comparable effect was observed in P. aeruginosa , which has similarly high genome G+C contents. Selection for increased genic G+C-contents in C. crescentus acts independently of the species-specific codon usage pattern and represents an additional selective force operating in bacterial genomes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
    <title>Effects of Genic Base Composition on Growth Rate in G+C-rich Genomes | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"4_uuqDUy3rAwftYIQmwvUe6yiBZOh3TrJJ4_IfmlONE","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459911","apath":"\/ggg\/5\/6\/1247.atom","pisa":"ggg;5\/6\/1247","ac":{"\/ggg\/5\/6\/1247.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/6\/1247.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/6\/1247"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/6\/1247"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/6\/1247.full":true,"\/content\/5\/6\/1247":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022sKZaBdLj6O3Y0rG\\\\\\\/5A9d+fg\\\\\\\/TJUOU3rmR6bl5dv0gkN1P+zs2gnIdqmkV7krYyuCuPfqlf1zWaljPgg0RELE6JNnqaRDNJrcA+TJcCGg+LSjBdgAJHyx9xY1UKAel\\\\\\\/vWLkfukNTX0IYkHjeZdB0GO3FvNV\\\\\\\/8YQpieMzpsvXigLrq9OlxwqxWcEdhKXz4e8+mJD\\\\\\\/226299O0hUvs6FsfqslLtkO69IYrVHVs3aJkcgGtTIUb47a2tQWkPiRxb4Ol1gtLSdnQCgH0O8P3xuxSBlgU6ErT2SBUC0Y+q992og6sA8usH2O742Qm\\\\\\\/NqeCgF1zXFy8G1qUNsSERLJkxsCxyqR\\\\\\\/z9DKkbrZeHQ9s+dblo7LSoliezclDK\\\\\\\/RFNLf\\\\\\\/F52qDk6elYoEaeCbrSxlGCT5iQsOgoxgohrikZhizluLZn32hOWACQnuhQV4N6Po\\\\\\\/+0A3lMprIwTqmcWi0ILPfkMQY2\\\\\\\/rKxxD6TDwhC\\\\\\\/zqDuewwdtOAMbRPeKgJPx4xv5c1iGYQrlokMXgZQlCOEQrZoM9iNx4eMkFz3WDRPbJMnQLAZxQ3K\\\\\\\/\\\\\\\/tvNc5wwsL7wN96WwDGu2JMNMdbvsCgaD6\\\\\\\/SIsiwN502hN0wqbnCS23dYtzOKNjgEwbU\\\\\\\/lCHGWexrR8grQBRTRbOP58tf\\\\\\\/NVT4PT9ABtm\\\\\\\/ROJYdTq7Y9\\\\\\\/tJqKHGOlwkge7rQUsaoGoYsyJlI\\\\\\\/uZ4hMck9egkoE9bnFtSYnkFRh11Ruyz7Z2YG1P8lkAnnMaAHZ4xM6zvE3fsY0DkYy\\\\\\\/EVngMJfxHyKSm8ggT0HZmXGW\\\\\\\/bESdNBfJUunQID1Kd8VOQq9MrwpH2RJFH3nsl6YMT\\\\\\\/hoySvhAVKI51LSEGpWtUqk6QumHpG5IgSa73Q8vcjBHlKHVyFrYgdqROeDIIaZl4oC7SrpejznvMhQ2nXBzXcTjM5ve\\\\\\\/Qaw1alQjk1UXXaHfNC3Xo3t9hVll67mq4excs\\\\\\\/BnCvnH7d2PUva+qkvMYmiTh3XpSBKeY9PQm9C0Bfo1PgWNRIqN95JVfcCRkuhCkrR7PI1gVgBN7ZN76Lnieq3jI9bAPcPnvzEIUfZpQ71FXywgi+nXpL4MNeB8\\\\\\\/28eNURWDFwUZCl1z8OumVhaJt6cjBCF8jUbwXPnm\\\\\\\/U6oiO1zpYhnEZAFIeB8UunMiM5sq77PQbZXdKJ\\\\\\\/AAh7tQ1aiGdltr1gq9QyHCdbgDDHFaEtZlsTSiwC5owvJn2IDNVkiStczQ8xmLYddNiJaN5FtIX7sRvLU1vhffBVCYZ0G\\\\\\\/4XfcDb\\\\\\\/EPe6gG\\\u0022,\\\u0022iv\\\u0022:\\\u0022fKYqp7wL3yOL1SuyFLwZhQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c7fd1689f79d0bf909934c32de78ec778b09f48ec14e0b77ad452e3608c3164a\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00225dvsGJ25Yy0bJboFK5oX\\\\\\\/+uB40sTpypF+xvV1glBZi0Eahf1643aaYRn19qiMscqGGTxN5zrEHb3V5dmsnae+LO2eYD7+NxphLRdOlxosYJkt53\\\\\\\/bTfrLm\\\\\\\/Hqx37IERAUlAASw7nuGa8aS9jB\\\\\\\/rgzWEJeOWdTVYCRWMwueTUC3uO2JF+yKTaKld4mc2RFanLfcBY9rhvAcwTOGezUlgCKNBCNCnd8hnD7DwzQCEwvn0cRoBV\\\\\\\/fCeboC1qoyYW8ugwkQTf9hPuTcXvXnWySzgWQZwb3kLXw+nqDLAopSQK\\\\\\\/5q+PnZWeDXFw8s6HyNhPEdTOm+\\\\\\\/DTZ8zRt\\\\\\\/7ryWjKY5H1j0pw\\\\\\\/FXEIiaasSvQsZQONkr1UgT6HOA5eqqUXvWEDXI5if7jRutblZz\\\\\\\/al+noOPDfm0TO9RSjNG44rIBPm1GbFq8voVjqTwSpbfaNSU4zEW4PJDqFqSuVk5KYpBxfJ4g6M5FIx7mf0y\\\\\\\/Qp2S5LGE+\\\\\\\/ucx+0ivRfetusT06Oj4NI5IcvTu5Cj6GjCtBUtgB4PSomdRipWjLgJZUNAZpL3dwm3EJLX+1LXH1KjJW0aLMNBk63HPjp40MSCUhJjuawMhrqujzN9Pd7U69Qvl6rT2OI5ssJiyCMJhezBy3S223iAe7I1IFEY87rBJ+4ToPQuPhU9+NZFBk5utR2K1C07y\\\\\\\/596UYfwxh1E++tXmH\\\\\\\/EzeuGL6doAlHMJHczkzDKoK9CcfNJFaZ32sPeZLozAjUmr1dsvfnB5WrhOi2E5jMmhtXI81UZCAS2OV2cQQRbt5V32YRduOYApk1H9j4Ea6VceEm5JsMh0N\\\\\\\/SGrAsz4mFzHwrzPCcJIKMHxIlUrg89M7MGwagZVgBvD2A6MAFQgMPUsH7dtQm+u0u4soeN47ItdRO6VvaUnwNW0mxdUzp188nGGw+0h5yI35LxZjJWQDAGGV6T3EgqzeJWoAcLe6xDH5zn29ku2F1rar8KB\\\\\\\/PRSYI4nxRE1RSUgZLRv8\\\\\\\/mN9YhZjaCY5tUMciw5FGCXYhqQZnlf746HCXud\\\\\\\/9Ym6d8Ae+pZ3xnhiN3gkGvz0hM6bi3oVa1unawxS\\\\\\\/P16bfU25kZ56DmD4ozRX5MjY\\\\\\\/MZhd4GmUq4ChV4PKxFKnKGK3h6BSvHT\\\\\\\/7AZgLYMJPWhZg\\\\\\\/KkkF7KgerI8t\\\\\\\/HlALoDYyNk6MnGMOtWwS3i41ScfjSUMLDGD\\\\\\\/w96OFyIVBbVELtG0bu5rOdwryvnQn\\\\\\\/u0aHmAqZMdSiZQ2e43OxHwr7GU3j8o1\\\\\\\/Sg8R47nKnebdXNhneol8Jl2pmy6aX3HPf13o4gM+L2dJxag2zZFalEEb3SOXWY3nibEVbHsaDgdn50bXDRWe+E7XJJ0YxtdhCayb4rQAKqiDx6vn4qU6yxveuIS\\\\\\\/WfIC58xV4Xye85KJzXOTeQoa1eoZl1HixJIlLKYRjPoipegGrSix0w8JQ890aXi\\\\\\\/HJt4h9QAFZJRRMFCmMkeYZ4zIUlT9L1C2eA7jSM6JKp595LK1FU3WZ1G1FS1Vte70ssJIejY0z+yf6oeXZf71zS2OoJ5W7z84otvBGTQ5wCwIqrcHPNLhBz4OV1xdqcmwVCv3LxQrtEBPO92cvgHAAXq0fFc9FFa+ODMEW2PfZrtDEvRQ7gvETbpb4AOWR880NtcttQ5Diz8yjyCLaqEFESC6svVwAhoBBZdkNGMZ8oJymf+WgbxHVGHEilT2VDE\\\\\\\/Q7Y0Mq7HgzCc3G+\\\\\\\/9cJ7tSdsrN3iQBYLJDxDDBpWAUC8U2zR2bdc8DAFX1who1owbDB55FFpWQFs+jY11pHvumagwLExFujg2Y1IyIfq9auVV+gCxHzgZHwILJn53BApPvIhBF5pIM0cNSbtttWXdPpkq9vhQ2a+l5PTIXwuDbNnnppZuPnQUCX4BwoSj\\\\\\\/\\\\\\\/Y90\\\\\\\/v31y6tPPeMcUAyAefSfpkXmPI1QHlE6SIgh7M9yTUs0mOpu6kU3tE0Jd9dr4+qC5AMVId57T\\\\\\\/UHMqA4lxN+z8PObqVTe8CD0f\\\\\\\/1jF9cWSJHoV0\\\\\\\/i9jUJQSbJCiHGYlSwE2E0+5a4BZVdPpP+HnfvYaAaeRDx27SEaRUr+eBrrQsERzCdc7QLqcGypNZvMLtJ4t\\\\\\\/vDQQX0RAjwxppkNB4KfFSbUa2CjDmbav8W9IQ4GWiBSYZqE9cLdBanGutpgOtf6pABCIUcRQ07sms5AFZR\\\\\\\/ayCbn7Ax1OXdOgLr1AGCfHHtaFPCVCVVvDmtfY8hrW6ItNVpK0nvKkp4kwTaztwVUg1AWTJ+3h3gyz\\\\\\\/yZRETJl3+IIe+tkhMA1E0mtfo4pSEfpoxgccr4uCdUPqr7VbSTrprH8CLZxhhLTksz9+KQSNoG8nzBSNnOkLfpP3wunoIefbaOupFZIUFRIK+zGlpQ2np\\\\\\\/pebH\\\\\\\/h2VYQ6vTQDVrUEw0Mo2yXXPKtzCveF12kU1NHa0U3a1A4Q\\\\\\\/HzDFVYad508GDpKngIV3etJlZjHviR\\\\\\\/mqUUKhpYAjevXl0pZYQifyZk04pGAd331wmv8BDkI\\\\\\\/zJfUwfMUmWkESiQfxbW8XsbnvCeRt1YBOd\\\\\\\/w6Q7wmJcWh+ElquO6+spmglYBT2PzT4FSMjnGhkly2LfNIc4okQiSlM8WkmZ9teOaJ4zgk78JaZsV3G1gCYCzco\\\\\\\/dUO9\\\\\\\/552Yt62XfH2fqK89aqdPmyl+6+\\\\\\\/o+uDOSfrgqryBCLjIn8zDg3ZGTCS5vhkqKhkqxmkgLDRaW37+1h0EAu9\\\\\\\/O0vHz2YF\\\\\\\/aYI+KjxzuxAv9e5aBtt7c2YaaUgU6IXrvQxzG0j4SkhsY9zIzIz\\\\\\\/vqtGYzLToX6zCGWCkXHdBFB83ZiRjL4QGX+9VvZBBVZqyAuiMulqH6N+8\\\\\\\/BxN1I5FWd+Y+c1\\\\\\\/nz+lAYsbF2vPERzOBgTqxOlC8u4uTl3IY\\\\\\\/cYJo1LUArCk94\\\\\\\/e5BiLM3eegCo9PKePJAr2rxQZFN2rMnbA4zXLj9iqGqemZC23YqL71YqDrQWlXLjs7aX6FYhjvKHwkuyZpwYHzA35\\\\\\\/H+SLZf3HKKfcGUVc4ZPHy6ZdCPOKnAQV62kWeyzlwdB3x\\\\\\\/AYUftpRsUmyi+yxyqmFPRuo5TUUiaR0iKfASgNmxvSiRQzT0dWraCD9lkK7JdTI\\\\\\\/rsf8bmWwnQ0MniMAfLJMeBlK5K1O4OEggTLQfo2CXT2bYk2LU33sts2daWoBj8Kvmq97cKutK2meR\\\\\\\/PY2AI9lzyUVFvwGJjblrf9P0dsp0iTQEl+XcKJoAnqSUbrv8dlEwkl4uaLJeUhuuIs6RkQFeoFbZhFiXd9etoArhP3X0RPTGxeXF6fjUGT720zbQQ9q\\\\\\\/a9Q5tKasfy2U68Lz0gnmQczpdAMKOtaZZ\\\\\\\/imdv+jJp\\\\\\\/mGjyFvrlac9g7a3SMb3SMgM2s9B\\\\\\\/mh\\\\\\\/QLjaYTolnTZ2Bko2XTXesgXxsWefY9F4Fweyq4vIIlF9kNrCapv6\\\\\\\/HG+hLfW17odpdyPVQB8mqeD9whYI55oHAz06rjkJlCcZy3odhzSVRF4jIiwOmDfX6rtdhxis3Jik4emlK4BQn5d2GtOyfpEzfyBPjH6FoKwsvGzsOAhVRXymP0LsyZZT+oVrwxqz2KODF+wrlpBeNa01MVBMJRvD4Byrd4oaSQhgYNY8jc+lrHVLYUEgO1dggr3A9LLhMwKQ\\\\\\\/u0jnh0Std8l3ZCDlW6+bgpKz2fQaOHor2c1ssAe+\\\\\\\/T\\\\\\\/LF9kyIcLep8eg4Bln4eJwpFukCrAIDtetmA2wFVPc+q7T5GCx4+gOVEVHgwG6O5H9eWug4D9nx5gY+kuzvO6l7OO5ThCdWRomp3MWdlayCjTpWyuIV5KBQx7WiUfjw36++arHSLGyIKp8SZV8dbOEiLxMWQD\\\\\\\/lR39vqzmavgYpL\\\\\\\/pSTL+vY4W6OWr6sSlUI0yQebspWR6x2+R\\\\\\\/rgcAvQG3N2rjohBQM0Cxa3\\\\\\\/RQuTUoV292pYlel3yg4qV4h2F61T0sus+Qu8duRSN9nc2ayeByMfX6TYsfU1TBOThPOPjZcDVgGhPLoYxCrr6cDdLZSSuv0BVrUNmIM863MZ0DtzpwREiDklX\\\\\\\/x0ZiRuFA4W8k0+uiDmUx9z8tdp3Jn4leOfWmoKD2IGZ6YJMUXhZ1DuTnhTsUNH4tkkLFWnRmepgSPC485E6F8GRZnFFW0SztvzKcGvr5rLdViWPgJvjnM5Eme9YOTZIlvcQYvwBC\\\\\\\/AarUUWWkZVyskCvM8CJOLxjFjsQ2raBRa2n3kA9zG5YWM4i6rrnYh5QyfUtRLbQdJdyvGYqLR3ZL2fzNF21ZxV2E9vlR+y0p0\\\\\\\/sOztT4IiD8XDpQNsaR2LjhLNvDGdIZvcwQ4BXcxERqfNzzgZP2wTDh7MkpoQ+pvYP9wtNMyK51IRt43M9IgGlJjJZJiUpgqq7ECre8prFDORFM1ies3\\\\\\\/xhCcYDNP7EtfSYZ8o0N+1Huq1JbwuEjtGCPSyEbhXQxJvF6E+CgBbvBvB4\\\\\\\/y3iY6ey6wo9XdT8p\\\\\\\/3uP8L6p6g+RDEfGk1MNKdYUzv1qsRscxSg1U43\\\\\\\/EAAZnQPswQHuQDNauB\\\\\\\/0nKjXyIeN70pgJusl\\\\\\\/GmtRjyX+xmw6YI3cViHVa46ZCHIjEO3i\\\\\\\/+dQpsG8UEBFd894M6QKma4lfKeTGAqnkXuor0uaNDJg6EfDmNZbV71yDlonq1tce++ew3GtyGtrkeH82U4mpydPcXqd4oC06ATfHc459tzOorwTg7L8CFsR5b7ZSFEZnh3dnChQWAgMHD6YeJ7vOhkrehLGf6iL2oVoP4zArT4ztiEVeXfeUHgXKzn4x9u5WSx6lNzwUE48lkNDQp1Q9f6RzxG2CIXNw5gLNFJk9qay0I0b2veHQEh35cizTZNPAwhinBPmQdxiF7JfQRJSkW8YskeGGdb09SFNDsfASdt5ph3otIKcHtgvOhMo2YWljkuhkJcxHgg+iSDs3QJ6Te0czPcDEjrWgeArn2U1fquL6vidzkXiRU\\\\\\\/PmUcMNdG\\\\\\\/CxBhdqtTbipQ31Y\\\\\\\/34IJpqa3z459MHunj9mQYV3r9dE4PYv7rk9uoumtnhD7YarWRCy0GH1wdY4CWwTeyCZuC6i+wx3mQTkc8ZqSGIgLEGeXYjNtkywA\\\\\\\/eH+Mhaw+yWvZwOTGBfXNbcw+XsozCx05r0YeRG2Qy9b+jZMHUnj59KjDnDkf9AHDAMwfWEQf3O2gbn4E0c2W+tIgKreMJFex0DLenUx\\\\\\\/iu+tb75dp07\\\\\\\/rc54yFPhQ9oAIg\\\\\\\/bYkHU2QwQwi7E5rRv\\\\\\\/NNu+So2E17fe9rCfk1QzNFzmZrWobYtrruChiS4Nxemi+ar6J\\\\\\\/q\\\\\\\/xHqwKsOXhbpKeOMLykDq6nY78x\\\\\\\/5bcNDZ0SmVfgUJUzILaD1Rr5YEpzOlM\\\\\\\/YocILaKwIXiqDa9aWWL09qCv8ndmS6SZgcBxATdCRnQaLbjFiWTirHD+k3TLUih7AUzwolAyy1iStj7MdldQP0AUM56\\\\\\\/ZIiPB6ya1rZD1B52O6JDJBwAd5tfeCERiISakdFjHTUCiihbIjCxQujdaiE23sK6nwtgyQmifWTqKapP012XJRmpbrgXi1aFbOhUNoVerL1ugl0yWc1BLGqotKLUTT1OYXR952xdXBM1y7zz\\\\\\\/hF2g1TN3yVUiV8kKwUIUW1BuatI5XS0UBrUIJSn9VQvKLVaRCia2BXQSxUzbWlYPKIMtMAhdFf5oDAaEdTKCfr8vPb8LbmAlLZqdhQl0U55GeUn+I8uzlt75ePXAuAqWWx33tXlVd5LYxEqGkhabv2Q6xJ8EplfFNvx2MsfBQ7uZKMVTdWaOxi+Zu9CUL6yD0hrgUIQT5UxNnRxdmBtNsiNiKqPU3mxhxWB2HjXoeY8BwshTEI+JrrRHGzY+arhWO9o2XMGWbVLxcFqtYOkZH3mMmn\\\\\\\/fsgpuLEKiJLIdg9Sw\\\\\\\/i+bOdSEfGs9OxeyZYPirZZe+FV5h0qnI85X1Ifsa3f63XnSZp7xXqABGZCL6iUWHV8aObVd\\\\\\\/h7iGm5sVVqSrHauXk7IFXeImyV4fuRazMJrUE9rGobrEcbFpIFUUC\\\\\\\/k6hImgTYVv2PweY35FkNpEihr8cJvlllRPXGyyCZ+3iKBr5wvYPp7yRS3fimooLEE0hVVKyWf2fwF7yBARDzCVfD0VQnOqmOdPiteFhOVgJnbLiDcMQbArDElurlpgEfNsZgnrUz0B0W2KDwegRN4538KlXE8V6ZwFfGSEuqCnyxaLVtsSEUkVZs6TB+mKD+dT5MUCJbxaDZ6LegxFwlRUl8msJlom1xZwCl0ZS4KoTE1liMInQSd8FoHT3HjuP3+Wxvsu3mk1oNLlQhqM4HfWTyz\\\\\\\/oUfzTw0uweNdWwY9ya4T+258M9dedp97C9F8ki7byHkKhUzYLLnGLcEWthKE5gnCooMLUPuYW8uRe6AFXOOHvS8Lt3ovN\\\\\\\/jWnQpHBNSUmvfotM2SVYvtBRNBmeb2DD1LzSJij9p\\\\\\\/OdVcMdQLIvkr0Jxeg4NItYlzQh0Vye40SSOjiuv\\\\\\\/+viT2xQb1mLVosmEXeOfl90d5\\\\\\\/clt85vXn0laaldsgshU31BVVEVgZdZPP8C79LovBW9gHvzWQc\\\\\\\/91ucJwK8hElY\\\\\\\/FiPkTXzAaLD3LePHNjIzh1AUv5FuufvjhhaEwY00NCdtIn2X80UaHRJjyiuKWqUn9lwixOnujM6ZKQs9Hf2lA0jkGkg0G0PG7eckK7SVn9DY9gs\\\\\\\/ID51QrwT1nTwLTVPehMPaqvXzw3toKFVmYbcv5nshD8WBCrjEQTejVbXqJvIZAe\\\\\\\/rYNv54cQZplKVcAkYo0Vt7Au4m0g1kWKCNa7DTeGPUWlPoLgT3NAxHJBog0wBDRpMDiv5ScI4b3OUSk7ygpHkk62BTHZXwlvw8fJ5rzpAiERXU5K1w4vHP4BquO396mOwlDXJJzuTMV6UnHzsv52QVqpZ0UMKuFtlcGzI2+FQczrOpJ7n4xmIZhVEaCK2pXEZQBHZczTHofYs5IMieM9AUkRhQGksS4HXZpy2k+hzXdmPFJcr0vQ+h8O0KGvQabLsNu+W09xyWVZx47VPFpYbH3TpKftjzV96\\\\\\\/DoAHJiQDzl44Km\\\\\\\/2IwlE\\\\\\\/+O5CPl4jK6jsaxJU1\\\\\\\/MycQoXzZ86C9O2ogiYgNrL9z7D2oHqXpfo+yy8tWrPLGfXSF4HVCWfuizPJMOx9J6ZcXIX1Ty6NWSpDMDNjL681GRLfV42F9AibOlVi+89jcGxDDiDyJ3KFAYc3u1CUwjtz0RDN59O6NV9wxOqs4mTkQiMyqzSfccpiG7aOecfCI5\\\\\\\/3HJ4nKHfiFI37Q7HHdvyNGEcxhjBB3c8sDxAN9sXNMIHiSfiLAh+pxwv+suatMiel2B9OyyOXy4kbOcjIWqBSQ9oRfFKK5RXeqR7UX8fZgQc+XzCwWX+6KYnv\\\\\\\/+mKTr65tt0HYcjz8UJxgiBZ0661Ys65t8f2dTxFjDLAiKmuvIpsRR95\\\\\\\/0v0+jOKsVac3KVtF0GJ5e0uLJzLyoLIycmW4J7ZmHs8aQa7JHWhX1chfmSp5SIktlYFBM\\\\\\\/ijRa+aon3Q8ko9ktczaA7gqWAjQl5fXp5pi\\\\\\\/4tlayVcrQ8oHpcL1hvfp3CEr30AA7P8F2mleHGrI+r5eIl3yjGqNymYh3E4ck1vdUK2JcpqRucfDbR0Yv9ZTH6QO0xavYylz2iOBiNXANKOO6nEuvEnj64Sj2rPQKOXV99Fw83hb61kT9koQBxEhpHaCSjlMQrdwfMQdrp7T1jJximC4Ow\\\\\\\/3NXD0TN3Aey1SDyYA0AqYEgkL8gQF7VLslctN1Fh6lwHRzKIa66Js16\\\\\\\/gSYK8jdcAo9J5WsObs+vDn+YDC0\\\\\\\/WxTINcUEllu3x6YTMk5VMo08w5BWolppJN\\\\\\\/YIPrkroOeRWFRAcxYfBlg9lSibEq8Mwne48IeGSUrREyNSAU1vEt2z9I\\\\\\\/kmk9vaYUl\\\\\\\/FY5yho432t4w2SYJDUyr2TwCcOp4g0zV6ddUQOLRTdASnycazSDfJN5nMoPPMIDHu3SXMC2tMHpmPX2MDV58GJ47KiR+8CKwhJ4S1Zsh\\\\\\\/5aYfJCZbHahOS7BTvylCR+wqO9A5fkgM3lYMpsyUvzsgcwYrijfRPimrTUCbNsmxHqn80PhVgRVuDBw6Tq+Z6afOtHr3XvT1f+ZNuqCzEA7R3CunYD8dMyM5qX6Bagb9CrpfZmVhwqZ3+MXAsJ3dHaJbVup2xKGtWABX1eRFHWd\\\\\\\/jw4p8t5u2UiA9KWEaxtdAElbcaQARwZicGEwDT2I5XvqZOVOo7tuzFk2QVHj5aLk2STW2SnsqHyo4aANyQFhUjrRXhP7URPGJry03mfCDXEEln6HZP0dQfp\\\\\\\/WEz9ILIb2zYAkdlZ1f3R06TSNh\\\\\\\/BtTIiQLMRiWUDIGd3i0r626H+Xn7X9VJOque5OjrFMIkwDYqklxHj1IBIQt\\\\\\\/DQ5F0DYsYg5laC0CcFrmyLYD1WYYH1bGgfSRY90rDwc2L3zYgX2310K8FAUABZg97EekECcUvg+I9YO6T+SBse\\\\\\\/0cYHbauAC9l5gVrfihxd+6damhXA90p35wh7c1KN8Cw+PypIgLS+JRJJ62AtomfgNWnB7Nn1QsRQ1sVCCYYmJUMUlKTPY7lcgFL7mxz2Sa3xJXhdeflO8GIoSxzCtSXhnc7tT+tarE55siv+2SoaqGbttuqFBLdAIndJQU7A5Ak13epHybT9X8sh6BwPIQ2k3RFfXs\\\\\\\/8Ccz5Vw2qj+K44jm78rw+wh2p4YFk52CrWZVe1dCgcdjSiEagqlZKmC8DBhw4YT7za7nY5izt5OZ7PRvn0GOr6dQUWqfcEkoHlFbPFCOG4neEPHhBX9n2CghvZcs4P4xjLWoZMVjLedvQzGQNBrjkCSVRjBeRs7K4IsCndeTUE7+CEpqPv2shSnx3vPZnh62W0fQXudsqVV+gJu23fmHNtIfxp3hxIovmemwSjsGvs1gOFmcvCU8AJLF+NTFayu\\\\\\\/Vab7iJu1Bc90+m9dGYWsGO036xKsVlj6qVBVVEZnRh19FTBeoX7P3V+SdnQ+2ySjFYhc3NWk1k6M76tzWe5cDBgGTT0iP1AToC0rFhnb2ruJ+CuQePKDUcWCie2YsMgJR\\\\\\\/P8mi9yyVYezl87ckGp4VbKYR5\\\\\\\/zXOgzfNofTAOWRY5j9EeX0IjPrKnJeBDhQ9tMGAU89yxzZl+eLpf6EPGKkjRVG0AV67kUu55\\\\\\\/\\\\\\\/34EdyWgUzXioz02Apz3trq81hvEP9GKukIK\\\\\\\/TCy6vuUTDBT+oxj4ii1OJG+DdeNC96khr968wJt9M9hmv5Vv6a194LYISDX2k+o2s3Gjuwk2p8RDLtebQSXMj9KjMNBHVvh981LIrQZxeYVS1RZ74V942WeiRNVf5APpwwF2+2WiWFLf65zLJDiVfvn58sW9PQPsVHLDVLwkXeuGoRlofBfHqnoYI5cIAz\\\\\\\/KRA2OiK2tysCJd2USDlwP+ljg91yFlHEtsrZ7mvNFWwHROcYFqIiiDCCmvb+PnLwd2yjmanvctWHoLHptaAMfXywXl2Zk05snfSoiE+x7rGsn8+KlTG9B8njqX6u7VwVsx1T4KX\\\\\\\/k3P6qmSVsUn6tCyzEScziVfhNGcDhe8tcXfU6ArEeu7NlYi4pYCpACpEIxJF0Ew+\\\\\\\/VLfB0OP2EqoD4MmrCO2YDff4VclrCNJKjsKzsjdjIUrfM6tp+Y\\\\\\\/jM4pTkZ6HbNs1K9x3Sp6uBKS5LXvrHrXrJcoWE08k6lIjXwqsxGjU3TwtLneeIpZ2Eo0+5EvrDTxsWvg16IYUwN6Y2XyH6ptvhRa0YW5tjF1prO2KDjBO8q4H6L71+1x9R66UlSUjtS5Jmh0DGugLYpdTyDYsul\\\\\\\/CvuhHeAzmuDAKBRhGb31fIxkBeDfafoGVzE11KWh\\\\\\\/MXj9OYHc9ajU2MxbpFcinqIfAbYSI4gf174bhQMMEYV\\\\\\\/FvGG3yRpR7mQIf+looG5PZWnzWo+zsEiDkj\\\\\\\/26EqrLhjKMpikbSISqmIrntOV8BmR8c6h\\\\\\\/oZInjvVHKT+XSi2YT96AtMZCwEZzTHflbWEh6iCLXafcvWSH\\\\\\\/XvxtX3mpMW9CGXUBg278oaep9ekBra6ysZ5dRhNnMPfbwMjpBWfBpAwwCfaox3Aaozrw0UppRDCJIxqWAybOfft1YSVHRs\\\\\\\/TiXLbpao9Q+ukNxUgLN5xpQYEqKkUaw43Zj8tpdIwuLBo0cg7RkcMpYgARo5VJ9acgaUHKer6VAOM4TnOJHqqh\\\\\\\/Hr5\\\\\\\/5NUffk68qZatS2CtY6xRstkoOFg2lNGymkaEY1xT5qZxIMfMenj5VC7jLd0dzgF6qYtqDxtgvQKIAoaNA2Gx\\\\\\\/cQwFp54EsNXLS6VzcpJKTxcrQpjHbIn1uLYmSuP8pG4gV+WwIwhQsi5pT\\\\\\\/32zSgyqx9AH9Z43D5TMMegk\\\\\\\/4CpgYzq+wbTvC6DxNu8i5CjNpFV\\\\\\\/sOwLIQC1d6D2wsdz\\\\\\\/vxtalwrihw4TxO0j8d8RkE9oy8R95\\\\\\\/8NlA\\\\\\\/F9eNWsM+vJ28X6eP+FpUuE2lp1hhMvhMeJer6FfIv4tijZ\\\\\\\/odTa8e0P0Xsxo4eobUSg4d3V\\\\\\\/S13Z93Ku9cEK9zicmoo99UmImve9icCOxjLvhVlkPLctQGEjoRX0BhKsD+nzJ9zc8Q68PrjdDmUuHDCAw\\\\\\\/ys0fgtHo+yuCFZYaSYSy3DMOHVRCZkHjA3Jw1k3niOCG+YrUmOKbpgI71x4ixCc74IaA7ln9K79I\\\\\\\/p8elxHnBqT2XDCXURiKCoWBJ9GtErnBd1\\\\\\\/NNPa\\\\\\\/0IDcVh43Kkq3938khFxXUj3mOxNE1bl+fy2w142J41MkrP75yelBdBbjC0DmBSZiFzeNfz5v\\\\\\\/DcMtRGoWb7fHN+py\\\\\\\/HDWq0abN5OQRBxpDzolfLVRFYvxFIBxdxHCsGrSdMY7eAl0MydhQ1BA8NeeAyeFJcEbd7rKnQjz5HHhOWPufTwbz8MYBnRhRzeRAvvqmcyAGkREXIraf9vnlKFF4q0jUioJloVRNh3AHI16W7ccz+8NVe8fVzQn7fN1C0ddcve3vwOsCBEOVhFE20Cgmrls5NZAydJsbQQW78sFNiHhgFxAxvBiO5E7rUjH7FJoVFeor64uLkl2STOXgvKF5v6ZUX1dZg5mMFeUjJD3Il3lC7GVuxbNqvZ7BrQ60RkX20UjJFDbmunR2j\\\\\\\/MZCm1hs+baNToZK+5E2g9mHn4ZwyFCUez4kSwsXdkdO0YoJXp2njNHOpY9IDS4mxWzIswhQnk\\\\\\\/5ZJiF3m9L2qmpdrH4WiR2vTtcj1uxwrMxykxO28QzAHdn+e1is7GuuqLCWcIIEYKE\\\\\\\/gYV13V4e682xo1NjW4RwNJ4wQ6nHpNzoqVBN0tlynOlqDgT8H04CPz+LxIGxEE0PDufBETRSzY3Mzr9z28K0AXKnrb78FTxA5T+GjBuQMu6ENNKqY7bYnb+kmRgNTnQ\\\\\\\/r5cp63PDiKb+fT\\\\\\\/IHrdd4nQwvLXbsPr+hJQ9zkEKJRgJD0C\\\\\\\/+1\\\\\\\/+5KddVoz+PV6SifecF0M324OJqvcd4HiM3At+fGliV+SBxms\\\\\\\/uWt54mxLILb0qxFS6lJOvSpt2f4sFHb4VhC4xWwtZfjWVvBGyRKYOA57g\\\\\\\/bYwnS9WjYhuVqHzzuIhA17WlikkhNJjRt6mAKBdRGVGocjtDEXMS3h+YVvN6PwQ7XnH2pNy5FE6RQv2EDLmZGTkp15XLyyvINCiUrXIOTG8Qq0vxQe80Zn9T79TKjJE4mEkWW8Iv8RY2zqvwBF\\\\\\\/9x4tmq05o98XhDQY7d+AXSqztnq29e4wV1+q3MXPg9LHFDjk3joh6plzqr8yM50r601v\\\\\\\/qiggpDYL1W733IEufA1EUfCvI33qqQEbBN9b2Qc17VauA9zT+jJoTDNO\\\\\\\/42ZIKh3WaRn6Nmtxo\\\\\\\/dnweoT3Ss4HWpx6sAym7vuA\\\\\\\/5X0AcbJP8RJWG+VSU4y4xP9WKWe0rK8z3ObS8L+1dLTQiYKOOH6WUBWOIwXeDn3ZupgiOa8IltjbdyxQPJTBplbOddEBP8CX4L4Kk9uwfoESL7A2gFcSjYTmq2SU+VTCMiHb9\\\\\\\/IveVZ1d1O3Td2u7omvI14YyZC1Ziaw\\\\\\\/wQp4d6HdKFMjRXxRdNHZDApteV\\\\\\\/y8TSirbRDM5DOzUvMxp1r3jpqQw1sFsnHn8VYNPE4eYp\\\\\\\/IkM3lf05ALEY2oVtDasExFSUaPyaxr3O5YaeCcTkvPKB7C\\\\\\\/WdkJbydieR7qFfDvZTfVtlnI1fVoCjyTrIEus\\\\\\\/\\\\\\\/kGk1JJkRIgK9Qf8y9Jzp7b1D\\\\\\\/TWoVuVDskGfst0AP0qoymZBL4JtZAqHpvvB2UQIO2dHxKQA3z4cdgyz8TaZwEc5CGH9ydTia0nsH3uB+SgS1ShfeT2k\\\\\\\/9deBzeZ6lwW3P7EB2uxpacLQ1msolJZPJLwcyDP2k6WML4KBBOk9MFcGFYeYMu1i\\\\\\\/OeCaeZzo\\\\\\\/pMKM7ZzZBBgbbfowPB6mdlOpWDEBLadBTfyH72vi9SLc1Mx9Odma1xOtGP9IiPtTQB9sireQk\\\\\\\/2wPzMWnmbSzElPmfmYRrMv9jUdgwE0RwXY+BXggYNK9H8bMppZQydxCoDadVdWm9r+j8tDtKFWmCzUKontLeL8YJw9b5pxGqZeUsWPHxn17+JqPoi1\\\\\\\/A4899NNwXAPAfaVjr4RP05p5XpxEVjhcCxA2CSyG+4w9P46hfEL+xpPgnZNtPbOiMd\\\\\\\/DG4hyQ4Fc+znroyqoFaBy45Vgt\\\\\\\/Q87ZqdZ1y2CHfTb0nrpNHxtWESP02KVL1f9z1CYDnwJokOmSNyYBoaNItAY5ShxBZ7nwxO6yCBsDUGR1kC20n\\\\\\\/MuOjwF4N90ENep\\\\\\\/C9gTyWJS03Sj\\\\\\\/uMHrJScbJOAwltA7pFUFibAH6wyjfwKywMezjyGyt6kXKp8N5+8Io0TLZKNQHaMaQl\\\\\\\/nhNzNhmnhRNUMTI8tdYgpeMuLqZqvl3w8ip3Kwk4vedhN7JZoMh6iBHemH6Gufr+Z4Gdbw\\\\\\\/PIiYSAU9fMFUn+ykiy3vb5LhtiogNhbWsvMBxbv+k7IEuoALo4huzAVWy6Is02t5dtFs\\\\\\\/ziCIYLADEG9IgGc+gM9esXFjvvH7KggVkWMD655dVp6bjrnKn0JGjoo6BXQSScPtdgC0kx6Lqvsedqs5WZE+lMV4M5SgJba3QJY5blCqPFcTRmx0ooLbmjFxwjw8lbbnPK2sPGpPwO5Dr9Yb+d2xM+EeCf\\\\\\\/8piCaqGdJ2WNwC83cJlJnCbprjlTM3P1sSGsAzgK3FBO5SONkuYxP7i86bKOhCdVxnI6XSu3Hw6Lh3pqkzkJtVzGXpSKU1AyesDg3dlC+tU3k6ZfE5WA8Gid8CCfo978sSEUkq3w23GDiZqbvS6s3e4Hosu\\\\\\\/zBnjP58m9tHQmEgFBJ\\\\\\\/qIchRszphpaxn0KDzOBC6JzccOrLOLUblxqJY1h30Y1SLXSW\\\\\\\/TL\\\\\\\/HeFp9M1o\\\\\\\/QhG3HXpS8q6ck8N\\\\\\\/ycOgclrMqqxPlJdlLG0\\\\\\\/uzd3G3P0x6wmbLW8B0LBOXjSJLbxkI7\\\\\\\/Wv4lxzxsozNqePv8DyzFaLgCTz3KmL+W3iygXvAXqPoKPnX9S9opXFqE3ylZ2i16VSAE856S9yYD\\\\\\\/qSTQ6IpsLvMI0Tm3SK9ZkR607S332eev7oTWd2AsHZhBwafEKhvDsq8lNKnP5BWmr9wfYHECbwKeep0S\\\\\\\/QpfHhCZRLbw9Sx0ed6L9PeRjh3quaHIJ\\\\\\\/0Zh5T7i8UkeDhVZ4NiA5zmLsAWKmmWsKnSaLG3mU3X6AhN77+TAUOUQnmsvW4wnNR9u8EZ0TzTICpPUrggZV8MB7ycy93yNOnKU40LnmiCCbxtAhe+UGvNKaYQu3\\\\\\\/AzoaqVxrSIXMqkWatOURZ\\\\\\\/cC3MZHSyVN1kltROGdrrjHRBe7ZhlVY3tPeaJEpl4dbsBHcVnwe152MRDpv4bdN+sZdo7kSXD1SDd00DL5SUKFSAmmPVB9Xeeh468toxTZZLTYo8dW7wjmRIjPv48zKfa71fhTToMQXE7Kv2vai12e5naZjWCaRIlv0E4jCCAxUE1L2FAN7MYcrN++clXnsDGrHMdRyHLT5YlsHCfjk\\\\\\\/8HmmsHzjL38OBJzK\\\\\\\/udN1YG3PyXqMvIoUWODw+GJP1E+TEQGDbUr62OzLmhi2jX7xpJF6LbHT9qELI0t0nlQyAO8BEJgxhOvzxK131K0AtsFn8LaTbuHu4ijD0IdR1Ed\\\\\\\/APf2aSP+RC81PcG0lbzaD6upQ3oRwcGyLoYtd\\\\\\\/uaVzq2jd9o6qaE0WwaRd8Io+zXh0NOPTwCUrw9lXadmuAbgIweUMalbxma75xBukDDcrM0pyaBfIj6z6gwMI9r606e4qE6W62cctjscWBHxp6ShxXnr61w+4D2igFwrusCNqZLFxU77AKgfXrNoOXF4pzNOq+BWmwXeXFaqTA2ze4e46lGig6YeH4OQ+8ehFzc88cvrZgrsK9PaCI1jQTaXlSW3p+iyFnHs7SPkMA\\\\\\\/57fh3r8933XW9AUG\\\\\\\/pqWsggM7okg2mFfE3IOUpdZavBgXvQ5h+st5JDajePjdD+XZ9Zp69W0nGYO3gMAUngT8yprId6lM8fq\\\\\\\/1EFIRtxSSdk8OSLpKKetNhR7RkBMXcG4IH6MMUT1+PVn9piz+RWF8hyzkpM3vh01YToJ4GZ\\\\\\\/CTLaVw12oBaud6XEyHW7Ac+jmZUPCJRBwwmBc1bZ04Zxd2gZTjSP6+VDXJtAlzP0TunpITPAgGVcYwUtpRfVjY2QTlXHOltHKtxDn96qn+XO21Rh+2QhYa9AE9ulymtzw2IX7CSqSzt9H2+TPSEtsdkPGlkDEnTF6wCf7Nq610HMIQhwk71P5t0lYu+ttliT2PXv7kPINX2iAQnR0o5x0\\\\\\\/onzLQRLgrcZUVj9Tno3hCnQQXuhGa0e1YAv+sOlsVAlSbxPvro8jFUWWCiVFzpTw0i4lfTW5HpfxkVq7MBQfI4\\\\\\\/1C\\\\\\\/WE4WBloywLjpXyNoiUFqbsC+2NHUMnsfPW4YaQeVLXItIGH\\\\\\\/JM\\\\\\\/\\\\\\\/a\\\\\\\/HABfLnO624B48C4BNVXcqIaiJLal+C1fpp\\\\\\\/cE4RCHSFoKbWhA4MUi4FjCB7Y8ewNni6nAzttGX86YjxGhWg+YBhdu\\\\\\\/Z8wwpAIOeacRUrcVo5+tj0MOa3KaF13ZeHjVmD7zy\\\\\\\/FFbBe3bWXTQIkwqUmsxaurIv5W8apecsTAbI6eSzaf7R3GQv8VJvDcNKolKqEB\\\\\\\/wPLazVAkUPC29os4P6heQWCB8gEqN05Zv2qkbhZ3mE9Ay7xk5xpe0VLmaHDTqhYecOe\\\\\\\/G5rqMmZsqPuHvg99\\\\\\\/KU+vpRrJbtZP+NLHc9IG0y01Q2F8ESjXuGE73bGHy3XF2mMs2SVgmR1AiMXX1dvuZ\\\\\\\/hRIC41Zki3qEJcKAASbsRrVBpULYVSvZu+CUAGthJJYCh4WhWn6azaRMI+GcVsVMcrt\\\\\\\/USTx5ZQAodqOobFWV50pNLR4v2o35qEHYkPJiURt1gyLGwLbPZmjXhxLbb3CcqNuV+KHjvYqRdGRXSiHO\\\\\\\/TaX6WlyQ7hNtVay+Nc25pbW7em03mp5vgrDEJeQuunTXet96mSt+LJl6PVGA\\\\\\\/Zq0jfZsKnMaRlml3HfN8Qe9K2A2dQ+S+uMg5EyBCXFVk10nBTyMA\\\\\\\/H5JBZbIKasayYvmXHVf7hLnVnnBzvRg7bhF86RUVEVaolHk\\\\\\\/tketf9nuu0dHduOU7GWbQX09pr8zf91WsP\\\\\\\/0oNBjcOVTa5vyh86nF1opDO6Y+d0nHM5hTcSNmVhWE8\\\\\\\/QuGNNsamjxVxEjXgpG3RS2bStIvlOW4lTLLF1AjVwuronkP7nqn5Hd3nZt7+PEfVSi6PHDxKP99EIpZ8S37rJRkeeiiuSi4Laz1su4jRbqs2R1tKRNdkWrEohbuHjsNBIXgEzKocKd0kBLEEfPf1sipssY4eoOLQNJHqe4P\\\\\\\/\\\\\\\/NeIyleR1OtdOFOR7Ww1+3nh92qSLKQ75gGdCRs7irnWuQha2R7zSAH7fz99PJSAFTP+cjRF5uTmkhuLW0NsLwuFRmO4prUhaD664S+Jz6Jk0\\\\\\\/dJN\\\\\\\/m111XzEZUSyxxp0S7w9fQPvZNX\\\\\\\/PyS+S9NzNWW0cCiglP60DsdD2gnegKCV58XN2Hq+aVHansa1Kf5L45hQTcNeiudppXp5gaV+\\\\\\\/N5qszeqZYh1zL14bbBR1XPwO7oEsD4uLf7V4c7kkqfL74cSuhRkKAXJz7bgJOETNEIjb9sfNupczwJJaajJQsmXBr9RFWsLCGCg1Rb7qoR8Fi0DC1pgwjuvfELpl5e4hxFNUMao\\\\\\\/7qkICgSSTqpZ\\\\\\\/wYBBzwP5Q3ItX3rUuAVVQTQFNkBmSz\\\\\\\/LSCCZiesuJrbaHeZhebP64q6QI8qCElSvt7XV9RvA+2cvWY+\\\\\\\/XWnj4VdyetmPVtUICm4NMrvLDfPFkXqk7nuFyGNRi97eo\\\\\\\/iav2JJkvgE+h4vyXuP61yNErLNN5+CaP5jEEh+F7PFHP\\\\\\\/0nkFdGj\\\\\\\/T3ZFgQV3nm2MfHzqwDiiGJxg\\\\\\\/FC2\\\\\\\/ylS8RC+haq\\\\\\\/muGP6TsjU2H0IL9gdgmEHg0DedHR0ZSEZpsn+79bjCZsELyXFALk\\\\\\\/Sn8yRkoTLZPalfxSmuQp8Qbv\\\\\\\/0Rqv+FTyHKNuGt3aE+eg2EifB8FOEyWEk4h+fvu+i2LgatQmtRYNfeeOvdba3wk9LcWncWU7OiIlzhjF9gFhQs4LRAkxh8YUm6VZ23fNvirYep\\\\\\\/KnL0UfhfYXerAfFIx8AAE0PFIGyCCb4HDNHAoCQmSCtA4dbL6MBQzT7pCxQHpFrhZ2E0ne64dBNWAb04pXHW\\\\\\\/oLroVNWeHbMPe+C1c7bR4iTKKoW\\\\\\\/NXhG1amM7BkF8voUWziKqfybfr6juWxPU8hAIeTM\\\\\\\/GrJMwY4axR0s89qCAsZhQ0HsFMdBJXrV1064A1qqXnUfMRniOR9yzZex6Cehd3XBScDA\\\\\\\/ZN6FIBzp7HMtygwR4VJ3FmuLG+kQDIIlwGB9KvYjhbGA5\\\\\\\/Plx5a1PVZeiBiutJkUSvyIw\\\\\\\/XJNNAc7DQDcdsFPzZTFg54psYwtyI2MXatoCYEuoraT+gQCuBYbcSZ9Y6Q989rvh\\\\\\\/k+HIOj6OKy\\\\\\\/oHIhk\\\\\\\/\\\\\\\/9sxOmfy5azTA2Za8Obqrj4T3VTebvXUgngupvGTxKJV78ixX81YASBXr9SC9eIB4sHnbASJepTpg4lyzNpKubnWHZX5pq7ewEpAPS5JJ2bztBV27zjptbKJPzGruRscSshRS27PH3b7YKkGjjd4oGVG4zP2KhubrJ9nBx\\\\\\\/D7AxVQRqrHkhuiCAsN7v8zeC+QUhIzFHO\\\\\\\/a0h0c2y8nUsCJ\\\\\\\/dcH0VCaCSKfpSx2lbdity\\\\\\\/\\\\\\\/PZfYLcrwC9\\\\\\\/+zE2mg30NnbjT1tTJZgdoU5dabupA2U1wGAVkv2CGW9tlW74U1u29HgmQIwyt56n4mJPNnpq1p2\\\\\\\/MK5gxi6bHqFBmcFveht30GuRm2w07CsIkqmWSMX5Gk\\\\\\\/+SpsGMxP8I2EC7kl3fX\\\\\\\/gEZH43tv6MDnJFpkpupUSo10RXfsMQ1TUp0JoGX7irN7qVCivYJvggKdeKdGpdPwWh0MS1S7TOR01Ma8ot2XjGOyqL\\\\\\\/8pCGQ8u7edCfCo7Y29MB1OSXhK04BcBG0khVynlIe7T8BQxVI6F7uETiz7iSVhDmlV9loU52ndmAMVPkKccqryLg+aHHGiJlEcEU2Pb5a3prz25Io3x0pWcyvFIRihYAE7ljxzNj58JwBtZHx2y5fl7bMxOBuD7VRafI0IrEov+PA4UONUxCzNdurs2gA9XbiBP5m6qpGDm9j6SQ\\\\\\\/LUWYdBPoWXjdu1bu2EnpvO1dCyiA6B5LTc1tAS9sO1t7dKX20hJc5LasEgFk6vFSH+c9\\\\\\\/Y9D6aj7L534oSBfqW0RK6BvTpzvJy594f7TZBfPXUWpV8nUO+MJG7VPCKMPbrBnSGVGkLwASvuyk\\\\\\\/98JNi7OrCzitLxor66nCtyJTdbLmVFL9ovpSxv9wahSLVTPpeF83O6XT\\\\\\\/hRFaid5GzhLITjn3NQeeMtHZ9wCZYaxWu9YtHvW9b9+XtG+c9gKGcuMdWq\\\\\\\/XUjDs5KwcB+9MO4dacfFYCCss8E5h0pvKLW9BsroHVb6rpJYWLzM5nBCf3k0q0y+y6WG7yQN6ypp0PWQ27JJvrmokEIsG+mRS1jSgRCGPcS\\\\\\\/LUmkU9BIsr3rRN8m9U2lDDbFjgVY\\\\\\\/VjGPvlXSTZMcpMeEAbnBgmm4ZESRyyZaIxi1tDG0uchXsSUAnT0dkjxAtd2xfTsYGE+22hhNKzRs0vxrMtvAGgkbCVDL5L8Mtbc+gfzD9sF1lPNjEzA4h8D4t0\\\\\\\/ynylJCLcdDuIFYUATjS8JLVerGFglon+z3kk6\\\\\\\/OyZF3w8ZbroPvBfOTPFK\\\\\\\/XGgcxURtkBdAbdN1KSnd2sY+8qElQgsyA2xuOh5nl2wkDt\\\\\\\/KYXLd+It+iRzVvNn1mb3zKhSikLZWkCtF017GgJ0vJJQsR+KoN3zU7LYiEuZaBOGKu0GIUD01HyWd9LSH5um7vxOS+7gY5DHbb\\\\\\\/doTL7nRD6gPTMHW4Qy\\\\\\\/5AfmrZ9CB8K9lsOyzPv9BWg+idUL\\\\\\\/Uzn4ihguGmevGZYeJFyuUDlg+\\\\\\\/cKjg3RZrgJikotcXyirHjmQ5g8vllA7OIOr+WvXSruafBV6jIRqT9w5a51zcQGv6xkxLuc1QQsluz23DW8U2S2AyABl8vf8M5JsYx0jtfRp2WaZN+4IXJ2nq3paSSSCkAPtcnf6cDX4\\\\\\\/\\\\\\\/LGK2LCFiX3X39LkAxst1UdTLn8dSwT24xzM9nFDI3cgTRwKIO0oxYyRmV2uihxQTLNuy838Nnwoy\\\\\\\/6++wj3r+gN0+VmtOiKmW6dRcQMi8cZw5ky5WHFPuOOzA7YZ9QT80T0pxcFwUp7YthgUunPUyVPuJciK7ZTCq69UULOSmgce6mlD8MqPu4fqyS8IY\\\\\\\/Not5LwJ7oFakG3lPo+IYThjT5lEJ5NHrfG1aXs+P+u7wH1Z\\\\\\\/fqoVWoI9usQfikPwkU0uhd1kArDyD7SQf5v6OeJ7hVv+q8nYHhyFYgRpuJu6UfhYwOpadKZGqAPysmdw06Bh8ygZcorhMPG92fT9FRcPkwaqMQubxVZo9z1yXxNbtWNquAUKHjDA7HZFS\\\\\\\/RdS1Qgt+upeF5KoXAJyUBSzIJl1gCf7OXeVE9E95h9hLNGf05USOCTrOmS\\\\\\\/wmkc\\\\\\\/5U7dxRaROh0MIy34W4gQvY8bPLYI0IHD+JIOE\\\\\\\/SsLOAGwTa89sILDpO5CoLUwIj8r8sd46V957RVQ1fI1E4EqC7lZafy\\\\\\\/+WAwwYcbZnKxi2CB4eJiDJKB\\\\\\\/sXuYHyVILm+9TVgcy3zn531LinZWLD0zvnDJAPHMSq8izTBSYdPYi24mjo4dQqqr7ggEpA5Tq2luikZWHzMINo3w2oPj2WcEBdydnDuEqbu1iD2ggZcfWEAns5LUBjxkkzi49kSfuPI1pqfcgmcfeGIrTAvRQnhDidL03DKrp80upOT+M2CCv1sCWnVsdXQ0dm2csIp5uUfXwWzq+1NiiYpqnO0qU4QHwem+2s0NJaDC67lDTtP2pWpwDRAWcI3Ro3sT6r7l4pBQBnOEmtmPd9DfY0xEp7dv6XxUs1PwSrKQLvWOMAS2T5quDJilvqqBht6W9WZ8xA7Ii8MiO6hKjLJzVYj37\\\\\\\/P+8vtv+e6zbpu\\\\\\\/dtIanw+pf0hnELN7k6JLa6eCSHIM08zady7CoIZHQudZ2qfGUq4H+o0La6qTol4t6h2wUmkjQlF6e64xWXtox0fx+nTtVqbGgJyHfNRd2FZzmaAq3X7Cpl7kFow8dFJt5gRaxG9jL\\\\\\\/np1MjVP+p6blvKyZ4zxJAxJcL8RryxuijPyfOCDxFMjogJh9FVyBvX+ifsjSwfVc5xLnukF8Zt4N7mNaV1YavmwjAxDBGdY8GlHpWPmsqIHaYflg\\\\\\\/9gcq86KLn00VGNLKjjXi7XJetPrOm8XpjJmlY7HbSj3GcxhOGInT1B3u9HCwWgOtETqpDZHJL9GrUDo+KPvzGXQ3udIrVdW\\\\\\\/XHLlK+jJH3lT6EFopNkX3x69FMuYh3OmgZSbiUDNFmvxaQt6YlC7T61t56kfqC6+W7fLosng3Q6+TZeFGdrrFBAE2Mp6eNCwFi1+PKbNCSMMkoIxpTUMPuJOznYY68Wasfx4f77yBPfvLgZtwZd85pQ9JTYK4yUvz5VSPe9pqebzQqCbJAMSmWCM95B2+5YmtrORpuEazhtfFx2d7LWEkwUashvVOpqEZn5Y3vu1n2GPmXZaXNVT1e4hPLlEgemDRV99sFCItXCbAISGPFjR+\\\\\\\/065UKbX6BN1axw+Z9xVUYIa5jmjFIHTjPO0ybJGBYKaqjYCOCd5L1SgQxp\\\\\\\/LYgyw3AxvOb0Mm4VFCbCstR43GlX1FIBDoZGsOmotn3igbknNQrMU+P1GZtWUJqVvQoQli+ijWgFke1k0STvaVcvv1VGFJM+TII21TyDE5\\\\\\\/zTu\\\\\\\/vZOaY5SRYoDmoiTvoW\\\\\\\/u2TCzt2gpNZ+U5MRX8XpfiWL\\\\\\\/BfL1P7okX0Lmua3sbVNhOLeeOvw\\\\\\\/XGGO+Cskv9nO7ftcq97y1kK608C+oi0JjVNxZ9Ha5rEHDwmwlyOhy+VqhVZs1XSuXtdQjeb+efur1MpmHCP6M5YT1RdaTx7ORFSDLhrGKW8qfBn34V28dMydITnZRmCLvDZYY8yx9PIAeAlv7PnUhiRdhLzuTOBddDzh3zAnLUHvDjoIko55PieyejL3\\\\\\\/9kSrCwbA1xPIuGSPfE5Vy\\\\\\\/OUQ+qXmXS3AOHcjZbGVxsxQ0CGkC9cO+g5v4HyqKYRsNfCg8Q13ckb\\\\\\\/J7\\\\\\\/gQC9oN6K5gGKRqHJLSqXrubP+E1QjwM7SlhQCYbPa+bAtmVSk96dCzQyhF96Wiaes17gB07G+kAOXCCvrNc\\\\\\\/jqJvzTcF3boxlSAiJNBhT+KbfS26wmC0g4go14eFfT+bZURjrSvtOvn7aA1xQpTwWmAMZmEHXy4\\\\\\\/q2luKNKUAo6mp0+g4z36UNIRPOpZacgiJXcC3AP56m3L1Nzm+tg\\\\\\\/PbBOwkAG4OT1QTMHBi0IHe8JlX1rLLIQX9Nd2ocz0vB+x\\\\\\\/BJsSTtc4uy+BW\\\\\\\/4hnqJ0J\\\\\\\/u0xvxLIH93\\\\\\\/Vw53AL4eydK2WFZqtB11kCreVkdJbJurXFleLAmhYw1xhORo\\\\\\\/l8Xgr8CUAHtbYhj2IdSWDkWT9\\\\\\\/inmMFwkL+sRw7MVxnoz8XLv6ca6tvI6dzdQ2B\\\\\\\/fHX0n+mZi6TSp90C9kFVNLex9A7xzqWgBhuRLSuLdmZM7l6iHT4tE3f5eDSPJZ\\\\\\\/Tu\\\\\\\/4petij7JM5ofXK+xzVGkerkEQbnMbXPhiNe38ZLyUnhYqW2QH5ofk9UgdLm+Fu7qhRqbco72nWkEo6XNlPAHFDcyyZnwBstT8MlFwZDPJZ7fiwcLPGK4V1cQOgNHgvTUY3ak89Rx72GzrEjVSaCiKzv6tL+txOEgPj951rU\\\\\\\/U5zT71mAZCJ7\\\\\\\/oJUAhxtV3q+kFvjfHsIsUNQCz2N0Df2db7RZX2CFKBje1yOm6pbwcTPe4UVqRpHPYMXINNB+sMieGaNddcvDj6uAlNdJOL0GLqNyf1e+DqMyjI4rgIfSCkFaNIjXDwWei8nDgaA0fRIf+lN5LxAJI3OQqVSoPc9nmRueVGJbEc5Bdt2jdaoWJoH71e75vdbVlvnUFur97JwNkQoH3QJFWd2u39hnoihTsw\\\\\\\/4HMae5VYN8jiwdAtnj8w8cukANwL8fIp5UsBOapCJnR9M35aJyga8Pap6io20OhGGTdsUez3re1u0hEZY4hSch3JBg\\\\\\\/V9PSo2AQiVSwYm5OpOdnADdr0bmzFx7KnTABHJOunPPufsglDKTDUmOdKvI1q3heOzPHzOqIelhUUxUzNIWtIDoEjICw+ld8E7dsfE8f3ufmOdNLvernCL+Y9fH2hOpifU9tq3f6UcUJXS+tPX3r1jUXonZtp3UZCt2ioxY\\\\\\\/euFFqF1MkLMSJPKYoNtwyYqOOS+pZbP1cYe18sfBTMAc6lH\\\\\\\/Tj4qRlckflw69TraMMAcDmPkTvQbnNCFL+yhLiYWE360KwFzZHJqqyr7oAfYVp4oikza5wnUHpueXXKtTNbk1AHcqkxE6Yo5UjqYdnVrPUn4hk5futHHNiE9\\\\\\\/cE234cNAtziBNliF49ig2U2+QxGKivvZpZjrkYQ8a1tIcb8zOPpehsmf1o9uIyPSfw7VR4UQOPkmGbr7yjU8TuYi3omn9SfO8ki5DSxftvQzXeAsnCQl7DtCxaBW5E7AV4BfZxnay3TckAAfSND7+f1z9D2ztQCUu6gBip9BaKZBDnXsvraCPl3zb3GoFg2Ri0p6X8Qjjw1lbeD7zCv7NTXduZLES0FqmIVB6CWcUB4NP4ItuT2x+wPHHMdTQj\\\\\\\/K7vw4X3ZPVLsKuE\\\\\\\/ERdW29CO085e9TURdkyBFo5TJaYzwK8c1ZWeIMa7rTk32B53cfQRZoqUD4IxAbPeFm\\\\\\\/bQAvGvDEy2nDANZIIc\\\\\\\/d0izPEFSn9Wm65NV7adbBF7LLZO4BU3fEWuKfhZ+KEn\\\\\\\/dHuBwML6KE2PIddX4Gl2SguW\\\\\\\/Puf8+5nBOtLwWvzDBAtf0LijwwATPimq\\\\\\\/66xPJvXD7vo6d798ind0Pwb0lWTF5f6e7AvBNV2Pv5pSjrG0qO7IsimhZfhUsZ3TS+Ps4G1Wot3TPbytE1o1XUuF3NuJdoFwkiDDRJMbDhM7hZJC5Zy+b+LXmmojE52JmBl3KUr6yDsl\\\\\\\/DgaJx2wsbUGhsJZI6987Blv3dVUxkknfplVZOIzlua6QSufNRXn65B\\\\\\\/uabRXIHsdAuUAnzHVBV9Ve2QotHYi3fTXp3Kdbt2xxfwnhEzJTg7MsJY0WqQS7rIm7Vzy+YdL\\\\\\\/XH1Yxc5kZrrJdchAw+pvrB4+E2wbM1gboas1G2HkZKjPHVa+ot9lU4YjwCgEHOMcZ27AS1yWVmRpuRy9oftCkXP7RFNs4zJfbVluX04DjahSmcR8T8yYuPyaBlvZQhjs3VDim6xZZETnJX+M+aPF1vYgjn17zWNh\\\\\\\/mN8KcIkpZX7ax1ku1Y8j2Khp1Lez\\\\\\\/b6ATihEXpgC11xHOVv5s62BjIyB0MaCQTiMISOJcrKhfzGWwfTDD8khcxDWK+OY80iEsEY2L+fLxcjsX5ZonVCNuptwbWfv6q1TZztk2DPWHo4R984LOljRcQvnlZxOXH8X70Qp7sR86IzZF2FoTskFUUnA90mQ0zs4wGe0y846C\\\\\\\/m3q9LfVX0qxEy3ovuwxrTcHT81xiTDETsdvjftzS52DEX8PO6IXvCgLLulsEButOCRrlpgkug+PAWRVix+04F8EP\\\\\\\/fnDMtO3P2CEsvX\\\\\\\/Au9L3KIJYx6WfAxpQc2QqLuNa8q7gYHjvTF\\\\\\\/h92ASMi20BtpYqq6axXkr2HHgPyuDe60Qd21tDRCaAaijcY6M0JFK\\\\\\\/mK\\\\\\\/Wf4xt3KVQTX5G+aFJOTkdJVdPAVerSE2M+JZKxlgwTZ\\\\\\\/ayuq8sJJRcce0iwQ+DnXYxdLz6hWNEp7MO5yEJ0vSG5LPT3Mq0NRbty8S3U0Wr16WzRgjBCT7EkN5SMLp278hzTFL+Qe6RlGqtpzfzysTMXECS5e0Z0Npdl9bRBA3t+cqT3kK3wywgSvoheuxu0VIvpxNHgbfcMtakODv5AuZEYGvTLGFaBYwVTjB+zM73YttSdGYay3HbsBsUsrIie7YhAZLny3vEykpRd8jbXnGXKa5qfg9RUdFDiNYGNNfTUemHlezByI0E0kpUWXhpLTErU6jEF60BOPPLHJp9MPMDaPPJ7NO1MIKLQVlMMMBOwt7nLwB7zh9UG5B7xuM9RzHc60OfJhOckZvo3vo3c1qSwnXxh6oBnBkOo3+0drjEqAtp4pKXZ8teLxDt9xr6kcem+\\\\\\\/DyACsuO5\\\\\\\/Fp2dQBAGMSIwhK5Fr1M6YGNuAGVUWgmNzrBZBl0Q\\\\\\\/5UDsSKwcGMjkdIeyaLkRU6Pq1SgelQOVyGccEhQzIXtQUxykFCwR694oiU92aF0CCjPZzaWL8zpsy+3zkNyh0Pb7vFmV4DsphDaYt0GzukKvGmWbm5GCN6MafmDBZjd6FGtnJcNuC\\\\\\\/GmRLLQNFsbLOhFkjm7bxHcrqAfjHGRtQUVgzTu0nOIbSujXvmJ8pnuJuydfXmK0vSr3vJO1z+3GgThbUonRTAoLqi4edlKpLqEvwsmPuRC2JukKKojCxVJtv7FAYFvMTlTx3+zSsuhJEg8I3MvLY69HbvE\\\\\\\/+7hpA\\\\\\\/TLTYQFx5pNeHlbIwPWO53CkDsvD8ylJbx+QH1iiSZu1ZIq5D6m9DL3eJmkgxeWerGP2eFTKwuuhI1Y7awRLb2r\\\\\\\/toEiGAY\\\\\\\/TWujVzPJBh7h1w+uLUhGO+RXigrcu5zaz9aYzqdxhwjI3ZsN4WT7pyKaHbd1Qb0s8EpczweKN702LHDAhXkhxTZdwnG4ob3RH6Sfta+236B6SxAMBfzDLagna5dUc\\\\\\\/L+Efv3HZPujw0aXcSpArvHmoEDAMknrmiACKk9xS0uNDogeFMp5+b52JBunCIxTJJqSN7YxNADDkSy0vtnc4pFt3WGr2OusDiAemsDezKCjxVb3FBEPIhPqUpyG9X8ydiYGRBWo3346rSYtJQhLm\\\\\\\/jy89UoDscRMBEbK2md5A2caHvCOm\\\\\\\/vAhzr5XBoINISb6W+4XHilAhwHuu5Z+E3zZiE\\\\\\\/LnjxsE+GoRlbiCCKUKJaAXqqZkC4aDKedDlZ8HuDcOfI0znP9zD8LlVka1KKfmnbuW1gZ6f5Q3TexpU71Wc5gJrIKcV8A2p350kfFUM1Vw2tCAkDGAHGSIYznm7Fn\\\\\\\/RYi3LY4vNWaev7Mvl\\\\\\\/TwDuHrqD7JFZRew90VfuyxIph8t+dyrypCE1lYvRFrE+ffw8Q9w5XGKSvnsAgPQ3gmCiCu8THK45M9iVyNFZ7JYc60DMy47B5kIF4BV5gQHbENyD\\\\\\\/Vm1rY\\\\\\\/J6DY9ahaWVVS4W1TVk0ng3iidts\\\\\\\/ppGEFSsmYbE2SOpxvC9dNo0ox9a1IzNebMHa\\\\\\\/\\\\\\\/JR+oXQO9zzF+hJq+93HLc+yjcNxa13HgJ4zBO6henQPb8zsFauqyRZFZLLYYjn25mKO57YR3vVq9\\\\\\\/YtaXp1qGkHl6AhOFq\\\\\\\/0UW8gnb+yuZ7+1ME+27Di2mnxbPzyrysFrcL5mN1mMr6uw6gkp0b+Fi6zHW9lP8x88H07s+XyyXm1hZlg29F9erXMDrUccYqGoirRlKcLD063eCfgk7My\\\\\\\/Xp78Kpex06AqJtvj7YqYe2otsT3sMApJBVqkODplpYe+D3ezHq493au94mncLhoW2C52Ry2490qti8VLe2CRpKcZglbt4AR7Igi+qHPYq24OXgVvBK1+lmTX\\\\\\\/ARXrai\\\\\\\/pbBA88mnruFDPrO6I\\\\\\\/RheGGV1tKHolNR+ZjgzTqRLoPha7MWPKecA9XGrKNJqZIHrIKOBRC5V9MNi3pxO+7zqAcwQ2yx9QD6m6je\\\\\\\/VPyR1+WzVn7F7cKP68ZJ5F9MPCydH5oyhPpjlR6Zu87slBOvD88vXkbpdnlTJ7DMRyL3fSZcOO65x6MbMfTyss5jCmB8FjnynWX\\\\\\\/IFrfn0pZ9eIAXKDECjQNWVkH\\\\\\\/kxMO58MT0Tp\\\\\\\/FJvpuQaudXfJrzwDjt05k\\\\\\\/5oMVKxuOymXC81sd1gBzk5OdnxKr1\\\\\\\/gJ194VkrXJ\\\\\\\/MJJU6IYRw7ZlgeasPqGsvugrTsVMscuW46NCGLZRE69xZWTAdXQ8PSLXvoj2Y1JFPw+Tm7Nm2i7GzL6W\\\\\\\/FItBSNbGn+l8xB9rQZIuDTUD1FFmp4XkSfLrgGDsJWzwOeb+F8MOl28pi6T3o95nAzv1Kq4VxXw3ROi0zL4N+sm4AUn4mzCvx2WxI6W2KXCCMSaGlDaRQ9sQ9HSqRp5Ks7ergJwz3ZjX2tWyS3FvihNmEb4STxHaclvBNOUQ\\\\\\\/T28tGrT9Kij55Oop1e+f28u4W9PiMsqaUQresdhxFWm4mj03ORsXyPH75qZUgdeseCWkiOGv6gi8mbimKB7C3akZVEX00EocGxI5DFlpprRCdVrqcanP7BJYstb3EcY8peJnDPlV\\\\\\\/R5bjLEvBrMAnic6+p65FRPRtPJQIcdIeLzEgP+yNoOBHo9Td+urZoeMELW9daBwovYY\\\\\\\/uqFv2Ki3MC9Tdp4kCTsKJF9B\\\\\\\/0c1p2XJwN1DStH5XS\\\\\\\/vUjW2liYxjUyCtV9C+HYeOUDAv4kWqr7YDGR6rSD+M7JIQhoktoWLboszaU0o1zrVwKZdV1oEew5zer7Sx0bMnfCCphkhfVrcjj2K6irln+qyeiXT7eGbTpG8ukUNShfS+4GxeKg9MOs1Yf7UJzQ8lvHiH0qsV3I7y8PB\\\\\\\/k9ostFQRT3LvOm2Pri3pRaTd0X+g6Boss8ZQTE4ndmiaRQsYd6UuaYEynAoiCIsYGFdIU2yIz0MfhtZhos3heF7CncfIldbxBqaYmC7\\\\\\\/MNIAv0PJsFAnlgBpyYtFAa86hh8+Rql\\\\\\\/\\\\\\\/kcrGyptwcF+xM4TIz4lbnj6wVEXfBTU5Q5NyTgYX1KMPpaAuVQKOWQCGor\\\\\\\/1pjCIrVdQJXGrFLZkSIrnSLwjlW7YuLDiDh6CzlAwORZvO45uVsR+EwmcbxHaxSfdxl2N1Mt2Gy9e1otl7qSQLeMGqm2W5F6DZJC5cGK0QIqlYFKNxQaaS7R\\\\\\\/I5MTg6p5jvx8ltreXDOFReDTAh20c8jJ7vw9Ydu5W1QvIrMA5TTE\\\\\\\/Cl69o\\\\\\\/cTZkoykz\\\\\\\/AUOpiq5SzPwOoaTq8nIMPk0mrXQ3x2V\\\\\\\/9JfhBf52BzEF2UMvO3WddIvUhj4EwII7U9OsI\\\\\\\/w5w1KgZbbNXn8MZ85j+5olV2SVut0zJ+5M1wePrLmVg9t4lqMSOSzoNL2kJfh3SHnwZZDWifXKGVM2owEpQ67nNp5qCCtLKb27cJJcVDpj\\\\\\\/rkNgnRoDzPmNlCHUR5D\\\\\\\/V6YXcmnxbD6JZP83Id19TF3B2ahLsyjhTEXpQ+G3RAsO8PwxWXq2oxWFFUYJ1Kmg9SWpZb\\\\\\\/yk9BwLGm+HFy5lDJEBUiAMPB8ohKYxlHEdD+zh92QYPch9l4AxoogpWK\\\\\\\/cXJZ8\\\\\\\/heylqS6fLAKkFUpS4oIgAdtJytCQ2KITdX7WqIZQWgIibC0KQyzUUqJowEqeVIMlb7xBG3qKOtskRcj+IQHkpZKcBbJjFh7N+4hW9\\\\\\\/LNNgVCtWpmWp+rPaYJyeR8uyW2cTdvVaHF2syIevQaMCP3Ml\\\\\\\/N22qx1TfCzoHdJ9vOLxuw7rqtRnml5nakHewPYeXTVYBFEFxZwPo3gBbk9pEb8nluFJ4hAM2SzdWavSRhMZBYFVPV5nq0plG3gcJq76Iqp3u88Nbn5\\\\\\\/EiROjJ6qElDFAlf8DbhwCXxXOc9CubwMNOEnvxCxQ9zWS20q327kU8YBoyof5TEmaosx\\\\\\\/PAMfv+q5P11YM+dKhwZ6aSqiPbkrVQTj6wuAO1vpvyb8io23laFpCUUlAJ+c6PA6DIoqy9FnJHMYejl4ZknJ0qOm9oqmtSoheNOOjWmTXAdFw9nBmF9w3nNtf0a5MZHaI5EWzVJsAgXsRACSqc7E\\\\\\\/zqhVrp1Mgf4DAPL1e4HBWDZW\\\\\\\/WqgBYRYBR2yqh+\\\\\\\/cHEkxUFrWwBA9GDLb896HERWeyyyGUdpXHe7Lb7R8Erc6FIh5bBf3CL6fmMfsOw0I64cWPpTeoGjX6bGeXWrv\\\\\\\/7QPE\\\\\\\/g6nJ57DRLIy6kfX0Q7D4Xlf9n4w0HInECf5LupzNbStTMbII6tk1Avjc5ALl2UXTJYLiTtoTRgsNw0i2lfHF9+5wbVxItfNw0y89imnklCvjIFOeDTPZzhQJLuIxSpFmkVvtcVKB94YN+5PtyW5n0OrMbgkLvfkSTIx5O+uVh4sjeesn3obxv+4Os7S\\\\\\\/xv\\\\\\\/4gmEvdzy9onCqgnZCln+gLrBO7uFmdZxZ3kIjmxViB69CXGlZNh\\\\\\\/wly4VmUUpEG8MBixTobLjSQo1uRX2ttUCAeR+uY7R68i3MScR\\\\\\\/ademCe5t1jJJA8xby3vrJyD0wW1iXtjK4P2bU0XwU11k3zqQUpWvIHXUxdczIX3zCpqloF3ohqHKajg8dU8edTsDjP31k0OYcHgS3qs0ZSXMcD1PIcP5UB9sgwT0uQqrMj88mmhNBU4ovc9FPtnL62QHrTjp8fhGrPe85IvdQhgKM26zWcJxfMJeSKqRw6lAqg8GJ5dR+h4ZAQ8thT6WKNN5i8ts\\\u0022,\\\u0022iv\\\u0022:\\\u0022OLp41sh5Pr933+IDliq4Ng==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022548e03160910c7249a2f878a509d6a9f255ddf68ddf515ce2313b48f699e204c\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kvKfXdRLWIMnbwNlZN4kB5ZEy56eaLXfklaF+Nbb82jRv\\\\\\\/d2kKFaDYXmHHSA2TpTqkehjr4s2icAl6mM5P6AxfF3tEhxir+fiUQUpIbfctGxwqdiU43NCOvqNQoT0\\\\\\\/vsmbKwgvQ+Y4FSy3OKl9GpE15z+509nkgeSbx2PsFOeLSJxaYH3W4Zl4ma4stAmo5QuYDDP5NivpSFpVgfZmYdHyRLZ0jVO88KCfmL6\\\\\\\/Ph0yn+zfuglLUfIQgCrHpQW1H\\\\\\\/Rw3KLE1rS5QzYaSt1cBNsbqY6mg9Fx1jCM6S\\\\\\\/AVlpRGtIk5h7+VQrzT8bf33\\\\\\\/PmJnQerCMTirQ2ml0NItqpMQ+GqnSsGJq2uF1M0Bov\\\\\\\/W0wZ4lbwhkait5uTHi6bWpxIDuoAejnPKZXTC6R7BgG55euR31OJMwkBra1TQygmGMInBMm2cUzzvmfblxREPfla7f2Nln\\\\\\\/LQt38Q6HmZvl802Pcd8j1vqVEI29hLsiY5MmVRaCIjNSfQz1v\\\\\\\/LqQ8D\\\\\\\/jfoB8y4PIQWPkiURejDrPLA581LE8hAX5pzxNLL6g4bSIITqY\\\\\\\/lNvv6DHt0sB8kdiILcRjYMT6WW7m12pj7JK+6W3opCcNsZDzkTVUCajR2zQmjWHuSTcBJ\\\\\\\/oKqJhml\\\\\\\/E9Y4XpRSXxyIt+ZafGckmbzg8BRH6otuPinPNK0ShLxuDU3SDQqdkdDiI2DQk8Jd09DTX\\\\\\\/aCMwYURj4po8U7ubNZrhkQWlWkUXYeVb1XUo0VwYI8HLB2RE7NEKBepQD4YzJhJVxolsa3zs9vowSSMCGuizMIbVNO0QqwUGlsy5HNyLDFT2tUuFvtlMIiP7ABntqwlEcqVKyY\\\\\\\/ylsqXOvRILWLHtv904\\\\\\\/1afjHlYX4qFM\\\\\\\/TZEZnuwc\\\\\\\/p64IDjvilnADC+AUFJkMz8Xrv1AqINjDy9QUdmwIYWIU6iSOKmALnmdE6+RlFVN\\\\\\\/7TKeiXeUK+r9JUaMbIn0Z3FTWn3bqitIj5hdLDsu4pN\\\\\\\/rrTmQto\\\\\\\/oaBdU06oOrx1TEBEXSOodVPUWPvpJWm+XeP3hbXKIPi7WEAYViMf7bkzIL+5UM6UKNAAJBlp8ldfbzzBNB0fB6ZgQAOiTExKRKlPH++D8PmtwtzTfm55WiUwiAha30RW2Wbv6aFu0wh7IgfqHfXw38FTpGaFsAnGMIsTOjhoWXaEmWB49rRqAvCARdeB2KK5TVqPRxY0NBstaRktR6hM7U8dxzS9efqDruwoQl+we0YQiGgPzXdaTq+g0NbVBYvSYXml5c3tMtgovSp56v6glG6kUbzHdJ13SkXapVV7RL2+6hpO3uOMAggQHtXEK3NdVtcSLS1ol6xkszL4zWFyMxLZbznD4EWm8+W07Hqj4ek0TLtkTp2a7I\\\\\\\/9pjlAY04CE9zF4xQd4OhU9ZE3EC9o7x8eGrYiZzuu0S1wqQEPTtVE+cA8lqAjpDjYDzWYpJtpBrKxn7pdPkAdEhxk7wcDmysO3p23TWUaF33E2bpeBilis4kcQaCIb7WlZmLvDTZkMN9iob7gwzi462gXTa1LRwcccSiUgTg7dgjdoK0034B5XgmJS2u1fib26CIXbbFqfcKIbLyqO+8YYswMwQLvuh6MEQw3o6EmgpJqt8FLwPsdt3n1BsrWUv3pVfZzWH79bN0kpBoe6EYRxLfjN9c4xMsdBGj8trQjhstUpKxMb7CptzqJmFCEvLR9pditUgf1hndbCLW+uhUdCZNEKvLR82b1nmAQv6rTsBEBre6sq9LbbdfjBsFTfBjYDUKNl1WA9ewuoNzZmYXrszjiICgw5bJB07SLLLIew874sUsT0IF9M+2xeFC+mKveKN\\\\\\\/GHyQTiEZW2xypQojGliapDyj6wOVF4RZ+dWlpxVpu+MJfbGSEgbFsZwsRNlxv5ip5NHFGHOnL8hlICe7xmpqS5ePPMxM7EAdNkl97xgvf3HQwnGlvAb2V3W+KlLAD+6+PDzmEb7kkLIveIAKBOiy0pV48YUBwGpCe6yf+D+phB0sL02hINWXImwe5SVWzICyeitVqO8=\\\u0022,\\\u0022iv\\\u0022:\\\u0022ufusYKfWzFUIJ5eEmmtphg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002236139214c087a35b33d5098bbfcc0af98bd928229f6c04cf6e468ecf4c256faa\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459911 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1247" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-JiTyN103ByMkXuRToK1bpLJ950R3ZM88HyT9Ec52jJM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1247" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-HVFL6AQWh3dK65vCijgkFFJH0AQOR2qDDQPx5vukXLc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/6/1233" title="Functional Analysis of Developmentally Regulated Genes chs7 and sec22 in the Ascomycete Sordaria macrospora" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/6/1253" title="Improved Detection and Characterization of Copy Number Variations Among Diverse Pig Breeds by Array CGH" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459911" id="node459911" data-pisa="ggg;5/6/1247" data-pisa-master="ggg;g3.115.016824" data-apath="/ggg/5/6/1247.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Effects of Genic Base Composition on Growth Rate in G+C-rich Genomes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4440-0191" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yogeshwar D. Kelkar</span>, <span class="highwire-citation-author" data-delta="1">Daniel S. Phillips</span> and <span class="highwire-citation-author" data-delta="2">Howard Ochman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1247-1252;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.016824 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459911" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yogeshwar D. Kelkar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Ecology and Evolutionary Biology, Yale University, New Haven, Connecticut 06520</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yogeshwar%2BD.%2BKelkar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kelkar%20YD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYogeshwar%2BD.%2BKelkar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4440-0191" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yogeshwar D. Kelkar</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel S. Phillips </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Ecology and Evolutionary Biology, Yale University, New Haven, Connecticut 06520</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BS.%2BPhillips%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Phillips%20DS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BS.%2BPhillips%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Howard Ochman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Ecology and Evolutionary Biology, Yale University, New Haven, Connecticut 06520</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Howard%2BOchman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ochman%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHoward%2BOchman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:howard.ochman@austin.utexas.edu" class="" data-icon-position="" data-hide-link-title="0">howard.ochman@austin.utexas.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/6/1247" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459911" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459911/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1247.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459911" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459911/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1247.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459911" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459911/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1247.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459911" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459911/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/6/1247.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459911" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459911/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-corrections" >
  
      
  
  <div class="pane-content">
    <div class="messages highwire-corrections highwire-correction-type-correction"><h3 class="highwire-corrections-title">This article has a correction. Please see:</h3><ul class="highwire-corrections-list"><li class="relation-68037 first last"><a href="/content/5/12/2921" class="" data-icon-position="" data-hide-link-title="0">Corrigendum - December 01, 2015</a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">The source and significance of the wide variation in the genomic base composition of bacteria have been a matter of continued debate. Although the variation was originally attributed to a strictly neutral process, <em>i.e.</em>, species-specific differences in mutational patterns, recent genomic comparisons have shown that bacteria with G+C-rich genomes experience a mutational bias toward A+T. This difference between the mutational input to a genome and its overall base composition suggests the action of natural selection. Here, we examine if selection acts on G+C contents in <em>Caulobacter crescentus</em> and <em>Pseudomonas aeruginosa</em>, which both have very G+C-rich genomes, by testing whether the expression of gene variants that differ only in their base compositions at synonymous sites affects cellular growth rates. In <em>C. crescentus</em>, expression of the more A+T-rich gene variants decelerated growth, indicating that selection on genic base composition is, in part, responsible for the high G+C content of this genome. In contrast, no comparable effect was observed in <em>P. aeruginosa</em>, which has similarly high genome G+C contents. Selection for increased genic G+C-contents in <em>C. crescentus</em> acts independently of the species-specific codon usage pattern and represents an additional selective force operating in bacterial genomes.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/base-composition" class="hw-term hw-article-keyword hw-article-keyword-base-composition" rel="nofollow">base composition</a></li><li class="kwd"><a href="/keyword/natural-selection" class="hw-term hw-article-keyword hw-article-keyword-natural-selection" rel="nofollow">natural selection</a></li><li class="kwd"><a href="/keyword/mutational-patterns" class="hw-term hw-article-keyword hw-article-keyword-mutational-patterns" rel="nofollow">mutational patterns</a></li><li class="kwd"><a href="/keyword/microbial-genomes" class="hw-term hw-article-keyword hw-article-keyword-microbial-genomes" rel="nofollow">microbial genomes</a></li></ul><p id="p-5">Base composition is highly variable among bacterial species, which ranges among sequenced genomes from 13% to 75% G+C (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">McCutcheon and Moran 2010</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Thomas <em>et al.</em> 2008</a>). Several bacterial phyla contain both high and low G+C lineages, suggesting to some that the repeated, independent emergence of this variation results from a selective force that favors certain base compositions under particular environment conditions (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">McEwan <em>et al.</em> 1998</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Naya <em>et al.</em> 2002</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Rocha and Danchin 2002</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Romero <em>et al.</em> 2009</a>). A contrasting interpretation, posited more than 50 years ago by both <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Sueoka (1962)</a> and <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Freese (1962)</a>, explained the diversity in genomic base compositions among bacteria in terms of a neutral model in which the variation was attributed to the inherent differences among species in their patterns of mutations.</p><p id="p-6">This decades-old view that variation in genomic base composition is neutral and driven by differences in the mutational process has been bolstered by: (i) the inability to ascertain the selective agent that acts on base composition (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Basak and Ghosh 2005</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bentley and Parkhill 2004</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bohlin <em>et al.</em> 2010</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Galtier and Lobry 1997</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hurst and Merchant 2001</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Rocha and Feil 2010</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Wang <em>et al.</em> 2006</a>); (ii) the unrealistic requirements about the strength of selection necessary to favor a single base compositional change in a genome (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Rocha and Feil 2010</a>); and (iii) the recognition that certain DNA replication and repair enzymes alter genomic G+C contents (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cox and Yanofsky 1967</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Lind and Andersson 2008</a>). However, recent comparisons of sequenced genes and genomes have shown that the mutational pattern in bacteria is biased toward A and T, especially in those genomes that are G+C-rich (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Hershberg and Petrov 2010</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hildebrand <em>et al.</em> 2010</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Van Leuven and McCutcheon 2012</a>). The discrepancy between the mutational input and the observed base composition counters the neutral view and suggests that natural selection, or another mechanism such as biased gene conversion (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Duret and Galtier 2009</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Touchon <em>et al.</em> 2009</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Birdsell 2002</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Lassalle <em>et al.</em> 2015</a>), shapes the nucleotide compositions of bacterial genomesâspecifically, that there is preference for increased G+C content in the majority of the sequenced bacterial genomes.</p><p id="p-7">To date, virtually every study that has searched for the underlying basis of the compositional variation among bacteria has been retrospective, based chiefly on the relationship between genomic G+C contents and some environmental variable or molecular feature (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Rocha and Feil 2010</a>). Given the evidence from sequence comparisons that selection serves to increase G+C contents in G+C-rich genomes, we adopted an experimental approach to test the effects of altering base compositions by assaying <em>E. coli</em> strains expressing genes of different G+C contents. Those strains expressing G+C-rich versions of genes displayed higher growth rates than those expressing the identical protein from A+T-rich versions (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Raghavan <em>et al.</em> 2012</a>), showing that selection on base composition is occurring at the level of the gene. These results are supported by the observation that four-fold degenerate sites often have more extreme base composition than any intergenic regions, although both classes of sites are traditionally assumed to be evolving neutrally to the same base composition (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Muto and Osawa 1987</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Hershberg and Petrov 2010</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Raghavan <em>et al.</em> 2012</a>).</p><p id="p-8">If selection on the G+C richness of genes is the source of the base compositional variation in bacterial genomes, it should be most evident in species whose genomes have very high G+C contents. Therefore, we examined whether gene-level selection on base composition, similar to that observed in <em>E. coli</em>, is operating in two phylogenetically divergent species, <em>Caulobacter cresentus</em> and <em>Pseudomonas aeruginosa</em>. These species have among the highest G+C contents of any bacterial genome (67% G+C) and are therefore expected to show some of the most pronounced effects of selection for increased G+C contents.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Bacterial strains and plasmids</h3><p id="p-9"><em>Caulobacter crescentus</em> CB15 (NA1000) and <em>Pseudomonas aeruginosa</em> PAO1 were used as hosts to express variants of the green fluorescent protein (GFP) gene that encoded the identical protein but differed in base composition at synonymous sites. For expression in <em>C. crescentus</em>, GFP variants were cloned into pBXMCS-2, which confers kanamycin resistance and carries a xylose inducible promoter (Pxyl). For expression in <em>P. aeruginosa</em>, GFP variants were cloned into pNW33N, which confers chloramphenicol resistance and carries the IPTG-inducible Phyperspank promoter (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">MiroÅczuk <em>et al.</em> 2008</a>).</p></div><div id="sec-3" class="subsection"><h3>Plasmid constructs</h3><p id="p-10">GFP variants were each amplified from their original pET15 expression vectors (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Raghavan <em>et al.</em> 2012</a>) using primers that included restriction sites to enable cloning into the appropriate vector. For cloning into pBXMCS-2, the primers incorporated <em>Apa</em>I and <em>Eco</em>R1 restriction sites, which placed the GFP gene downstream to the Pxyl promoter. For cloning into pNW33N, the primers incorporated restriction sites for <em>Bpu</em>10I and <em>Bzt</em>Z17I. Primer sequences are listed in <a href="http://www.g3journal.org/content/suppl/2015/04/20/g3.115.016824.DC1/016824SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/04/20/g3.115.016824.DC1/TableS1.pdf">Table S1</a>. Digestions and ligations proceeded according to the supplierâs instructions (NEB), and the identity of all constructs was confirmed by sequencing. These plasmids as well as plasmids lacking any GFP insert were transformed into the appropriate bacterial host by electroporation. A total of 12 GFP variants were placed into the <em>C. crescentus</em> background. Nine were placed in the <em>P. aeruginosa</em> background. The GFP variants ranged from 40.4% to 55.7% G+C and differed only in their base compositions at synonymous sites.</p></div><div id="sec-4" class="subsection"><h3>Fitness assays</h3><p id="p-11">To test the effects of GFP expression on the growth rate of <em>C. crescentus</em>, 3 Î¼l of an overnight culture of each strain containing a GFP variant were inoculated into 147 Î¼l of Peptone-Yeast Extract medium supplemented with 5 Î¼g/ml kanamycin. After propagation for 2 hr at 30Â°, xylose was added to a final concentration of 0.03% to induce GFP expression. The optical density (OD<sub>600</sub>) and GFP fluorescence of each culture were measured every hour for 10 hr on a Victor3 Microplate Reader (PerkinElmer). Growth rate experiments in <em>P. aeruginosa</em> proceeded similarly except that: (1) cells were propagated in LB containing 17 Î¼g/ml chloramphenicol at 37Â°; (2) 1 mM IPTG was added to induce GFP expression; and (3) fluorescence and optical density were measured every hour for 4 hr. To gauge GFP expression, fluorescence values of cultures were normalized using the fluorescence values of negative controls, <em>i.e.</em>, cultures harboring no GFP inserts, as<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1247/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1247/embed/mml-math-1.gif"/></noscript></span></span>where <em>F<sub>GFPi</sub></em> and <em>OD600<sub>GFPi</sub></em> are the observed fluorescence and absorbance values, respectively, of cultures with GFP clone <em>i</em>, <em>Fn<sub>GFPi</sub></em>, the normalized fluorescence of the culture, and <em>F<sub>CONTROL</sub></em> and <em>OD600<sub>CONTROL</sub></em> are the observed fluorescence and absorbance values, respectively, of the negative control. Measurements of optical density and fluorescence of the growth cultures are provided in <a href="http://www.g3journal.org/content/suppl/2015/04/20/g3.115.016824.DC1/TableS2.xls">Table S2</a> (<em>C. crescentus</em>) and <a href="http://www.g3journal.org/content/suppl/2015/04/20/g3.115.016824.DC1/TableS3.xlsx">Table S3</a> (<em>P. aeruginosa</em>).</p></div><div id="sec-5" class="subsection"><h3>Computational analysis</h3><p id="p-12">The codon adaptation index (CAI) of each GFP variant was calculated using the method of <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Sharp and Li (1987)</a> using EMBOSS Explorer (<a href="http://embossgui.sourceforge.net">embossgui.sourceforge.net</a>) as follows. For each species, a reference set of genes encoding highly expressed proteins (ribosomal proteins, elongation factors, and chaperonins) was used to compile codon usage table using the âcuspâ tool. These codon usage tables were used by the âcaiâ tool to calculate the species-specific CAI values. Complete genome sequences were obtained from Genbank. Shine-Dalgarno (SD)-like sequences that have high predicted affinity (&gt;6 kcal/mol) to anti-SD sites of bacterial ribosomes were obtained (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Li <em>et al.</em> 2012</a>) and their frequencies in each GFP variant measured. Regression analyses were performed using the lm package in the R statistical environment (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">R Development Core Team 2011</a>).</p></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><p id="p-13">Comparative sequence analysis suggested a role for selection in increasing the G+C contents of protein-coding genes in virtually all G+C-rich bacterial genomes (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Hershberg and Petrov 2010</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Hildebrand <em>et al.</em> 2010</a>). We tested this hypothesis experimentally in <em>Caulobacter crescentus</em> and <em>Pseudomonas aeruginosa</em>, which have among the highest genomic G+C contents of any bacterial species. Specifically, in each species, we assayed the growth rates of isogenic strains that gratuitously expressed variants of a GFP reporter gene that each differed in its G+C content at synonymous sites, with the goal of establishing whether selection is actually acting on the base composition of genes, not genomes <em>per se</em>. Moreover, use of the same GFP variants as those in a previous study (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Raghavan <em>et al.</em> 2012</a>) allowed us to better define the manner in which selection is operating.</p><div id="sec-7" class="subsection"><h3><em>Caulobacter crescentus</em></h3><p id="p-14">The tested strain CB15 (NA1000) has a base composition of 90% at four-fold degenerate sites (GC4) and 62% G+C at noncoding sites. Assaying the growth rates of isogenic strains that each expressed one of 10 GFP variants having different base compositions at synonymous sites, there is a significant relationship between the G+C content of the expressed GFP gene and growth rate (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>) (<em>r<sup>2</sup></em> = 0.30; <em>P</em> = 0.04). When GFP expression is not induced, there is no significant relationship between %G+C and growth rate (<em>P</em> = 0.18), indicating that the fitness effect depends on gene expression and/or protein production. These results are analogous to those previously obtained for <em>E. coli</em> (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1C</a>) (<a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">Raghavan <em>et al.</em> 2012</a>); however, unlike <em>E. coli</em>, <em>C. crescentus</em> shows a weak negative relationship between G+C content and doubling time even in the absence of the inducer (<em>P</em> = 0.1). This difference is likely the consequence of leaky GFP expression in the noninduced cultures of <em>C. crescentus</em>, which is higher than that observed in <em>E. coli</em> (<a href="http://www.g3journal.org/content/suppl/2015/04/20/g3.115.016824.DC1/FigureS1.pdf">Figure S1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1247/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between base composition of GFP genes and doubling times in hosts expressing GFP gene variants having different base composition at synonymous sites: (A) C. crescentus; (B) P. aeruginosa; and (C) E. coli. A significant positive association between G+C content and fitness was observed for C. crescentus (P = 0.04; r2 = 0.30) and E. coli (P = 0.001; r2 = 0.55) on induction of GFP expression (black circles); no such significant relationship was observed in P. aeruginosa (P = 0.90) or in any species when GFP was not expressed (blue circles) (P = 0.18, P = 0.67, and P = 0.94 for un-induced cultures of C. crescentus, P. aeruginosa, and E. coli, respectively). Error bars designate the SEM, which represents how well the sample mean represents the corresponding population mean." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1504881604" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relationship between base composition of GFP genes and doubling times in hosts expressing GFP gene variants having different base composition at synonymous sites: (A) &lt;em&gt;C. crescentus&lt;/em&gt;; (B) &lt;em&gt;P. aeruginosa&lt;/em&gt;; and (C) &lt;em&gt;E. coli&lt;/em&gt;. A significant positive association between G+C content and fitness was observed for &lt;em&gt;C. crescentus&lt;/em&gt; (&lt;em&gt;P&lt;/em&gt; = 0.04; &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; = 0.30) and &lt;em&gt;E. coli&lt;/em&gt; (&lt;em&gt;P&lt;/em&gt; = 0.001; &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; = 0.55) on induction of GFP expression (black circles); no such significant relationship was observed in &lt;em&gt;P. aeruginosa&lt;/em&gt; (&lt;em&gt;P&lt;/em&gt; = 0.90) or in any species when GFP was not expressed (blue circles) (&lt;em&gt;P&lt;/em&gt; = 0.18, &lt;em&gt;P&lt;/em&gt; = 0.67, and &lt;em&gt;P&lt;/em&gt; = 0.94 for un-induced cultures of &lt;em&gt;C. crescentus&lt;/em&gt;, &lt;em&gt;P. aeruginosa&lt;/em&gt;, and &lt;em&gt;E. coli&lt;/em&gt;, respectively). Error bars designate the SEM, which represents how well the sample mean represents the corresponding population mean.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1247/F1.medium.gif" width="440" height="156"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/6/1247/F1.medium.gif" width="440" height="156"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1247/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1247/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466175" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-15" class="first-child">Relationship between base composition of GFP genes and doubling times in hosts expressing GFP gene variants having different base composition at synonymous sites: (A) <em>C. crescentus</em>; (B) <em>P. aeruginosa</em>; and (C) <em>E. coli</em>. A significant positive association between G+C content and fitness was observed for <em>C. crescentus</em> (<em>P</em> = 0.04; <em>r<sup>2</sup></em> = 0.30) and <em>E. coli</em> (<em>P</em> = 0.001; <em>r<sup>2</sup></em> = 0.55) on induction of GFP expression (black circles); no such significant relationship was observed in <em>P. aeruginosa</em> (<em>P</em> = 0.90) or in any species when GFP was not expressed (blue circles) (<em>P</em> = 0.18, <em>P</em> = 0.67, and <em>P</em> = 0.94 for un-induced cultures of <em>C. crescentus</em>, <em>P. aeruginosa</em>, and <em>E. coli</em>, respectively). Error bars designate the SEM, which represents how well the sample mean represents the corresponding population mean.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">Because codon usage in highly expressed genes can be biased for tRNA optimization (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Sharp <em>et al.</em> 2005</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">2010</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Dong <em>et al.</em> 1996</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Ikemura 1985</a>), we also examined whether the relationship between growth rate and G+C content of the GFP variants might be attributable to codon usage preferences, as indicated by the CAI. The relationship between CAI of the GFP gene variant and bacterial fitness is not significant in <em>C. crescentus</em> (<em>P</em> = 0.1) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>), which differs from the situation for <em>E. coli</em>, whereby some of the GFP gene variants yielding higher growth rates show bias toward those codons observed in highly expressed genes (<em>P</em> &lt; 0.05) (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2C</a>) (<a id="xref-ref-23-6" class="xref-bibr" href="#ref-23">Raghavan <em>et al.</em> 2012</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1247/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between codon adaptation indices (CAI) of GFP gene variants and doubling time of (A) C. crescentus, (B) P. aeruginosa, and (C) E. coli. A significant positive association between CAI and fitness was observed in E. coli (P = 0.03; r2 =0.30) on induction of GFP but no significant relationship was observed when GFP was not expressed (P = 0.07). There is no significant relationship between the CAI of GFP genes and doubling times in P. aeruginosa under induced (P = 0.48) and not induced (P = 0.40) conditions. Similarly, in C. crescentus, CAI is not significantly related to doubling times under induced (P = 0.12) and not induced (P = 0.37) conditions. Error bars designate the SEM, which represents how well the sample mean represents the corresponding population mean." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1504881604" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relationship between codon adaptation indices (CAI) of GFP gene variants and doubling time of (A) &lt;em&gt;C. crescentus&lt;/em&gt;, (B) &lt;em&gt;P. aeruginosa&lt;/em&gt;, and (C) &lt;em&gt;E. coli&lt;/em&gt;. A significant positive association between CAI and fitness was observed in &lt;em&gt;E. coli&lt;/em&gt; (&lt;em&gt;P&lt;/em&gt; = 0.03; &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; =0.30) on induction of GFP but no significant relationship was observed when GFP was not expressed (&lt;em&gt;P&lt;/em&gt; = 0.07). There is no significant relationship between the CAI of GFP genes and doubling times in &lt;em&gt;P. aeruginosa&lt;/em&gt; under induced (&lt;em&gt;P&lt;/em&gt; = 0.48) and not induced (&lt;em&gt;P&lt;/em&gt; = 0.40) conditions. Similarly, in &lt;em&gt;C. crescentus&lt;/em&gt;, CAI is not significantly related to doubling times under induced (&lt;em&gt;P&lt;/em&gt; = 0.12) and not induced (&lt;em&gt;P&lt;/em&gt; = 0.37) conditions. Error bars designate the SEM, which represents how well the sample mean represents the corresponding population mean.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1247/F2.medium.gif" width="440" height="154"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/6/1247/F2.medium.gif" width="440" height="154"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1247/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1247/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466267" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-17" class="first-child">Relationship between codon adaptation indices (CAI) of GFP gene variants and doubling time of (A) <em>C. crescentus</em>, (B) <em>P. aeruginosa</em>, and (C) <em>E. coli</em>. A significant positive association between CAI and fitness was observed in <em>E. coli</em> (<em>P</em> = 0.03; <em>r<sup>2</sup></em> =0.30) on induction of GFP but no significant relationship was observed when GFP was not expressed (<em>P</em> = 0.07). There is no significant relationship between the CAI of GFP genes and doubling times in <em>P. aeruginosa</em> under induced (<em>P</em> = 0.48) and not induced (<em>P</em> = 0.40) conditions. Similarly, in <em>C. crescentus</em>, CAI is not significantly related to doubling times under induced (<em>P</em> = 0.12) and not induced (<em>P</em> = 0.37) conditions. Error bars designate the SEM, which represents how well the sample mean represents the corresponding population mean.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">Because the effect of G+C contents at synonymous sites in <em>C. crescentus</em> is not associated with the tRNA optimization, we then examined whether the amount of protein produced by the different GFP variants might be responsible for differences in growth rates. We found that bacterial growth rates are independent of GFP protein production, as measured using GFP fluorescence (<em>P</em> = 0.2) (<a href="http://www.g3journal.org/content/suppl/2015/04/20/g3.115.016824.DC1/FigureS2.pdf">Figure S2</a>).</p><p id="p-19">The presence of SD-like sequences in transcripts is associated with translational pausing in bacteria because such sequences have high affinity to the anti-SD sites of bacterial ribosomes (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Li <em>et al.</em> 2012</a>). As a result, SD-like sequences are disfavored in bacterial coding regions. We examined whether the fitness variation in our bacterial strains could be explained by differences in the frequency of SD-like sequences in the GFP variants. For each GFP variant, we counted the frequency of those SD-like sequences that have high, predicted affinity for the ribosomsal anti-SD sites (&gt;6 kcal/mol) (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Li <em>et al.</em> 2012</a>). We found that the frequency of anti-SD sites was not significantly associated with fitness of <em>C. crescentus</em> strains (<em>P</em> = 0.381). We also noted a similar absence of relationship between frequency of SD-like sites and fitness in the isogenic <em>E. coli</em> strains (<em>P</em> = 0.16) (<a id="xref-ref-23-7" class="xref-bibr" href="#ref-23">Raghavan <em>et al.</em> 2012</a>). This suggests that selection on G+C-content of bacterial genomes is not driven by avoidance of SD-like sequences in coding regions.</p></div><div id="sec-8" class="subsection"><h3><em>Pseudomonas aeruginosa</em></h3><p id="p-20">The tested strain, PA01, has a base composition of 91% G+C at four-fold degenerate sites (GC4) but an expected GC4 of only 71% based on its mutational spectrum (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Hildebrand <em>et al.</em> 2010</a>), again implying a role for selection in increasing the G+C contents of protein-coding genes. In contrast to the situation for <em>C. crescentus</em> and in <em>E. coli</em> (<a id="xref-ref-23-8" class="xref-bibr" href="#ref-23">Raghavan <em>et al.</em> 2012</a>), we detected no significant relationship between the G+C contents of the expressed GFP gene variants and growth rate in this strain (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>) (<em>P</em> = 0.90). Additionally, and again unlike the situation for <em>C. crescentus</em> and in <em>E. coli</em>, reporter gene expression results cause little change in the growth rate of <em>P. aeruginosa</em>. The maximum difference in doubling time between induced and noninduced strains of <em>P. aeruginosa</em> was only 1.08-fold, as opposed to 5.3-fold and 1.4-fold for <em>E. coli</em> and <em>C. crescentus</em>, respectively. Moreover, the maximal difference in doubling times among strains in which different GFP gene variants were induced was only 1.2-fold for <em>P. aeruginosa</em> compared to 3.2-fold and 2.0-fold for <em>E. coli</em> and <em>C. crescentus</em>, respectively. Together, these findings show that the expression and translation of GFP pose little metabolic burden on <em>P. aeruginosa</em> in that this reporter gene system is insensitive to fitness effects that might be associated with changes in base composition in this species.</p></div></div><div class="section discussion" id="sec-9"><h2 class="">Discussion</h2><p id="p-21">The variation in genomic base composition among bacteria has been thought to arise primarily from species-specific differences in the frequencies of each mutation (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Sueoka 1962</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Freese 1962</a>). Recent comparisons of closely related genomes indicate that, for most genomes, the input of new mutations would not produce the observed base compositions. In general, new mutations would almost universally result in genomes that are more A+T-rich. The disparity between the G+C content expected from new mutations to a genome and its current base composition is best explained by the action of natural selection or by another process, such as biased gene conversion. Naturally, missense and nonsense mutations are affected by selection on protein structure and function, which might serve to alter overall base composition of genes (although not necessarily toward higher G+C); however, genome comparisons have shown that selection for increased G+C contents also acts on "silent" sites, <em>i.e.</em>, noncoding DNA and synonymous codon positions, previously assumed to evolve in a neutral manner. We adopted an experimental approach to study the effects of selection for increased G+C contents and to gain some understanding about the substrate on which selection is operating.</p><div id="sec-10" class="subsection"><h3>Two types of codon usage bias operating in bacterial genomes</h3><p id="p-22">We tested whether the expression of gene variants differing only in their base compositions at synonymous sites exerted differential effects on the cellular growth rates of two G+C-rich bacteria, <em>C. cresentus</em> and <em>P. aeruginosa</em>. In <em>C. cresentus</em>, the strains expressing G+C-rich gene variants had higher growth rates than strains producing the identical protein from G+C-poor gene variants, mirroring results previously reported for <em>E. coli</em> (<a id="xref-ref-23-9" class="xref-bibr" href="#ref-23">Raghavan <em>et al.</em> 2012</a>). In both <em>C. cresentus</em> and <em>E. coli</em>, selection favors synonymous codons that were G+C-rich; although there is &gt;30% difference between the G+C contents at four-fold degenerate sites in the two species.</p><p id="p-23">It has long been known that the highly expressed genes in many bacterial genomes utilize a very limited set of the possible synonymous codons, an adaptation thought to optimize the use of available tRNAs, thereby promoting translation speed and/or accuracy (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Sharp <em>et al.</em> 2005</a>, <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">2010</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Dong <em>et al.</em> 1996</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Ikemura 1985</a>). Selection for increased G+C at synonymous sites, as detected in our experiments, also represents a form of bias in codon usage, and we can examine the extent to which these two biasesâone for tRNA optimization and the other for increased G+C contentâoverlap and select for the same codons.</p><p id="p-24">Our experiments used identical GFP gene variants in both <em>E. coli</em> and <em>C. cresentus</em>, but because each species has different codon usage preferences in its highly expressed genes, the association between selection for optimal codons (as reflected by CAI values) and selection for higher G+C codons are not the same in the two species (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). Although synonymous sites of <em>C. cresentus</em> are very G+C-rich, there is no significant association between the CAI of the GFP gene variants and growth rates in this species. In contrast, the relationship between growth rates and the G+C contents of the GFP gene variants in <em>E. coli</em> is confounded to some extent by codon biases for tRNA optimization in that the gene variants conferring the highest growth rates contain synonymous codons preferred by highly expressed genes.</p><p id="p-25">The association between growth rate and the CAI of the expressed GFP gene variant was not originally recognized in <em>E. coli</em> due to differences in the numbers of genes in the reference sets used to derive CAI values (14 <em>vs.</em> 51) (<a id="xref-ref-23-10" class="xref-bibr" href="#ref-23">Raghavan <em>et al.</em> 2012</a>). To distinguish between the roles of the two types of codon usage preferences (tRNA optimization and increased G+C), we first isolated the relationships of GFP genesâ CAI with G+C content and, separately, with bacterial fitness, using two separate linear regressions, and then examined the relationship between the residual variations in fitness and the G+C contents. Using these procedures to examine each factor separately, we found that <em>E. coli</em> growth rates are significantly associated with the overall G+C content of the expressed GFP gene variants (<em>P</em> = 0.012; <em>r<sup>2</sup></em> = 0.37), and that CAI is not significantly related to fitness after factoring out the effect of G+C contents on fitness (<em>P</em> = 0.39).</p><p id="p-26">These results show that the base composition of a single expressed gene can contribute to cellular growth rates and that in two of the species tested, selection favors increases in the G+C contents of expressed genes. Genomes with very different base compositions and codon usage patterns showing a similar selective response to higher genic G+C contents indicate that synonymous codon choice in bacteria is subject to two adaptive forces and that these forces need not operate in the same direction.</p></div><div id="sec-11" class="subsection"><h3>On what process is selection acting?</h3><p id="p-27">Based on several observations, it appears this selective force to increase genic G+C contents is operating during the process of translation. First, differences in growth rates are observed only when the reporter genes are expressed, limiting the action of selection to transcription, translation, or protein efficiency. Second, there is no relationship between genic G+C content and the amount of protein that the corresponding gene produces, implying that the selective differences between G+C-rich and G+C-poor variants are not caused by the overall accumulation of the encoded protein. Third, the relationship between the genic G+C content and growth rates is not apparent after removal of the ribosome-binding site from the gene (<a id="xref-ref-23-11" class="xref-bibr" href="#ref-23">Raghavan <em>et al.</em> 2012</a>), which excludes transcription as a major substrate for this selection force. Thus, the G+C effect on growth rates might be caused by differences in the ways that A+T-rich and G+C-rich mRNAs interact with the ribosome. This suggests that selection is occurring during the process of translation, likely mediated by the thus far unrecognized effects of G+C contents on translational speed or accuracy, or on mRNA secondary structure and stability.</p><p id="p-28">These results lend support to the hypothesis that selection is responsible for the relative G+C richness of coding regions of multiple bacterial genomes, despite a mutational bias toward A/T (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Hershberg and Petrov 2010</a>; <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Hildebrand <em>et al.</em> 2010</a>). These findings also help explain why the G+C content of synonymous sites often differs from that of intergenic regions, particularly in G+C-rich and G+C-moderate genomes, although both are generally considered to evolve neutrally (<a id="xref-ref-23-12" class="xref-bibr" href="#ref-23">Raghavan <em>et al.</em> 2012</a>). It should be noted that the base composition of intergenic regions, like that at synonymous sites, need not be strictly neutral and reflect the underlying pattern of mutation; intergenic regions often contain functional, noncoding elements whose sequences are under selective constraints and their base compositions might also be modified by a nonselective process, such as biased gene conversion (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Duret and Galtier 2009</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Touchon <em>et al.</em> 2009</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Birdsell 2002</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Lassalle <em>et al.</em> 2015</a>). Given the results of experimental and comparative analyses, it currently appears that genomic base composition stems from a variety of sources rather than from a single selective agent acting on the entire genome.</p></div><div id="sec-12" class="subsection"><h3>How widespread among bacteria is selection for increased G+C contents?</h3><p id="p-29">The majority of sequenced bacterial genomes are G+C-rich in their coding regions (<a id="xref-ref-23-13" class="xref-bibr" href="#ref-23">Raghavan <em>et al.</em> 2012</a>), and we propose that translational selection has played a role in the compositional bias in many of these genomes. However, in contrast to the situation for <em>C. crescentus</em> and <em>E. coli</em>, we saw no significant relationship between the genic G+C content and growth rates in <em>Pseudomonas aeruginosa</em>. Although it is tempting to conclude that selection on genic G+C does not occur in this species, and that the high G+C content of its genome results from other factors, the disparity is most likely attributable to aspects of our experiment system. In <em>P. aeruginosa</em>, expression of GFP reporter genes did not impact growth rates, preventing the differentiation of any selective effects caused by the reporter gene variants (whereas in <em>C. crescentus</em> and <em>E. coli</em>, induction of the reporter gene resulted in a reduction in doubling times of up to five-fold). It is possible that the relatively low base composition of the pNW33N plasmid used for reporter gene expression in <em>P. aeruginosa</em> precludes a sufficiently high level of GFP expression to detect a selective effect. It is notable, however, that <em>P. aeruginosa</em>, unlike most other bacteria with high G+C contents and relatively large genome sizes, displays weak codon usage bias in its highly expressed genes relative to its moderately expressed genes (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Sharp <em>et al.</em> 2010</a>), raising the possibility that additional forces contribute to base composition in some of the most G+C-rich bacterial genomes.</p><p id="p-30">We focused primarily on the G+C-rich bacterial genomes, because these are presumably under the strongest selection for increased G+C given the mutational bias toward A+T. However, there also exist bacterial taxa that have large effective population sizes and that consist primarily of A+T-rich genomes (<a id="xref-ref-23-14" class="xref-bibr" href="#ref-23">Raghavan <em>et al.</em> 2012</a>), and we might expect that in such taxa, which include <em>Bacillus</em> and <em>Vibrio</em>, translational selection on G+C content does not operate. As a group, these genomes do not exhibit as much compositional disparity between noncoding and synonymous coding sites as do the G+C-rich genomes (<a id="xref-ref-23-15" class="xref-bibr" href="#ref-23">Raghavan <em>et al.</em> 2012</a>).</p><p id="p-31">In conclusion, we found new evidence of selection on the nucleotide composition of expressed protein-coding genes in the evolutionarily and compositionally divergent genomes of <em>E. coli</em> and <em>C. crescentus</em>: both species display higher growth rates when expressing G+C-rich versions of genes encoding the identical protein. We also observed that our experimental technique cannot effectively test this G+C effect in all bacterial species, and an alternative experimental method may be required for investigating these species. Our results indicate that, in addition to the codon usage preferences that optimize tRNA usage in highly expressed genes, there is selection to increase the G+C contents of protein regions, and that the two forms of selections need not be acting in concert. Our results also suggest that the strength of selection on genic G+C content, like that on codon usage preferences, varies among bacteria and contributes to the wide variation in genomic base compositions.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-32">We thank Rahul Raghavan for supplying GFP constructs, which were originally obtained from Josh Plotkin and Grzegorz Kudla. We also thank Fabienne Wichmann, Jo Handelsman, Akos Kovacs, Christine Jacobs-Wagner, and Whit Schofield for advice, plasmids, and strains. We are grateful to the reviewers for their helpful comments and suggestions. This work is supported by NIH award GM0108657 (to H.O.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-3"><p id="p-33">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.016824/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.016824/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-34"><em>Communicating editor: P. Pfaffelhuber</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-01-29"><span class="received-label">Received </span>January 29, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-04-14"><span class="accepted-label">Accepted </span>April 14, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2015 Kelkar <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.1" data-doi="10.1016/j.bbrc.2005.02.134"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Basak</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghosh</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">On the origin of genomic adaptation at high temperature for prokaryotic organisms.</span> <abbr class="cit-jnl-abbrev">Biochem. Biophys. Res. Commun.</abbr> <span class="cit-vol">330</span>: <span class="cit-fpage">629</span>â<span class="cit-lpage">632</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2Band%2Bbiophysical%2Bresearch%2Bcommunications%26rft.stitle%253DBiochem%2BBiophys%2BRes%2BCommun%26rft.aulast%253DBasak%26rft.auinit1%253DS.%26rft.volume%253D330%26rft.issue%253D3%26rft.spage%253D629%26rft.epage%253D632%26rft.atitle%253DOn%2Bthe%2Borigin%2Bof%2Bgenomic%2Badaptation%2Bat%2Bhigh%2Btemperature%2Bfor%2Bprokaryotic%2Borganisms.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbrc.2005.02.134%26rft_id%253Dinfo%253Apmid%252F15809043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbrc.2005.02.134&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15809043&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1247.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228427200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.2" data-doi="10.1146/annurev.genet.38.072902.094318"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bentley</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parkhill</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Comparative genomic structure of prokaryotes.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">771</span>â<span class="cit-lpage">792</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DBentley%26rft.auinit1%253DS.%2BD.%26rft.volume%253D38%26rft.spage%253D771%26rft.epage%253D792%26rft.atitle%253DComparative%2Bgenomic%2Bstructure%2Bof%2Bprokaryotes.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.38.072902.094318%26rft_id%253Dinfo%253Apmid%252F15568993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.38.072902.094318&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15568993&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1247.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226244600025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.3" data-doi="10.1093/oxfordjournals.molbev.a004176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Birdsell</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Integrating genomics, bioinformatics, and clinical genetics to study the effects of recombination on genome evolution.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1181</span>â<span class="cit-lpage">1197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DBirdsell%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D1181%26rft.epage%253D1197%26rft.atitle%253DIntegrating%2BGenomics%252C%2BBioinformatics%252C%2Band%2BClassical%2BGenetics%2Bto%2BStudy%2Bthe%2BEffects%2Bof%2BRecombination%2Bon%2BGenome%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a004176%26rft_id%253Dinfo%253Apmid%252F12082137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjE5LzcvMTE4MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMjQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.4" data-doi="10.1186/1471-2164-11-464"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bohlin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snipen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hardy</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kristoffersen</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lagesen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Analysis of intra-genomic GC content homogeneity within prokaryotes.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DBohlin%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.spage%253D464%26rft.epage%253D464%26rft.atitle%253DAnalysis%2Bof%2Bintra-genomic%2BGC%2Bcontent%2Bhomogeneity%2Bwithin%2Bprokaryotes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-464%26rft_id%253Dinfo%253Apmid%252F20691090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-464&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20691090&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1247.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.5" data-doi="10.1073/pnas.58.5.1895"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yanofsky</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1967</span>â<span class="cit-article-title">Altered base ratios in the DNA of an <em>Escherichia coli</em> mutator strain.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">1895</span>â<span class="cit-lpage">1902</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DCox%26rft.auinit1%253DE.%2BC.%26rft.volume%253D58%26rft.issue%253D5%26rft.spage%253D1895%26rft.epage%253D1902%26rft.atitle%253DAltered%2Bbase%2Bratios%2Bin%2Bthe%2BDNA%2Bof%2Ban%2BEscherichia%2Bcoli%2Bmutator%2Bstrain.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.58.5.1895%26rft_id%253Dinfo%253Apmid%252F4866980%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI1OC81LzE4OTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTI0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.6" data-doi="10.1006/jmbi.1996.0428"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nilsson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurland</span>  <span class="cit-name-given-names">C. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span>â<span class="cit-article-title">Co-variation of tRNA abundance and codon usage in <em>Escherichia coli</em> at different growth rates.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">260</span>: <span class="cit-fpage">649</span>â<span class="cit-lpage">663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DDong%26rft.auinit1%253DH.%26rft.volume%253D260%26rft.issue%253D5%26rft.spage%253D649%26rft.epage%253D663%26rft.atitle%253DCo-variation%2Bof%2BtRNA%2Babundance%2Band%2Bcodon%2Busage%2Bin%2BEscherichia%2Bcoli%2Bat%2Bdifferent%2Bgrowth%2Brates.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1996.0428%26rft_id%253Dinfo%253Apmid%252F8709146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1996.0428&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8709146&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1247.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VA24800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.7" data-doi="10.1146/annurev-genom-082908-150001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duret</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galtier</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Biased gene conversion and the evolution of mammalian genomic landscape.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">285</span>â<span class="cit-lpage">311</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenomics%2BHum%2BGenet%26rft.aulast%253DDuret%26rft.auinit1%253DL.%26rft.volume%253D10%26rft.spage%253D285%26rft.epage%253D311%26rft.atitle%253DBiased%2Bgene%2Bconversion%2Band%2Bthe%2Bevolution%2Bof%2Bmammalian%2Bgenomic%2Blandscapes.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genom-082908-150001%26rft_id%253Dinfo%253Apmid%252F19630562%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genom-082908-150001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19630562&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1247.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270313200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.8" data-doi="10.1016/S0022-5193(62)80005-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Freese</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1962</span>â<span class="cit-article-title">On the evolution of the base composition of DNA.</span> <abbr class="cit-jnl-abbrev">J. Theor. Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">82</span>â<span class="cit-lpage">101</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BTheor.%2BBiol.%26rft.volume%253D3%26rft.spage%253D82%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-5193%252862%252980005-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0022-5193(62)80005-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A19627196B00007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.9" data-doi="10.1007/PL00006186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galtier</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lobry</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">Relationships between genomic G+C content, RNA secondary structures and optimal growth temperature in prokaryotes.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">632</span>â<span class="cit-lpage">636</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DGaltier%26rft.auinit1%253DN.%26rft.volume%253D44%26rft.issue%253D6%26rft.spage%253D632%26rft.epage%253D636%26rft.atitle%253DRelationships%2Bbetween%2Bgenomic%2BG%252BC%2Bcontent%252C%2BRNA%2Bsecondary%2Bstructures%252C%2Band%2Boptimal%2Bgrowth%2Btemperature%2Bin%2Bprokaryotes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FPL00006186%26rft_id%253Dinfo%253Apmid%252F9169555%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/PL00006186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9169555&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1247.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XB49700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.10" data-doi="10.1371/journal.pgen.1001115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hershberg</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Evidence that mutation is universally biased towards AT in bacteria.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D6%26rft.spage%253De1001115%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001115%26rft_id%253Dinfo%253Apmid%252F20838599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001115&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20838599&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1247.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.11" data-doi="10.1371/journal.pgen.1001107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hildebrand</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Evidence of selection upon genomic GC-content in bacteria.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001107</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHildebrand%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253De1001107%26rft.atitle%253DEvidence%2Bof%2Bselection%2Bupon%2Bgenomic%2BGC-content%2Bin%2Bbacteria.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001107%26rft_id%253Dinfo%253Apmid%252F20838593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001107&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20838593&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1247.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.12" data-doi="10.1098/rspb.2000.1397"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hurst</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merchant</span>  <span class="cit-name-given-names">A. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">High guanine-cytosine content is not an adaptation to high temperature: a comparative analysis amongst prokaryotes.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">268</span>: <span class="cit-fpage">493</span>â<span class="cit-lpage">497</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DHurst%26rft.auinit1%253DL.%2BD.%26rft.volume%253D268%26rft.issue%253D1466%26rft.spage%253D493%26rft.epage%253D497%26rft.atitle%253DHigh%2Bguanine-cytosine%2Bcontent%2Bis%2Bnot%2Ban%2Badaptation%2Bto%2Bhigh%2Btemperature%253A%2Ba%2Bcomparative%2Banalysis%2Bamongst%2Bprokaryotes%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2000.1397%26rft_id%253Dinfo%253Apmid%252F11296861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMjoiMjY4LzE0NjYvNDkzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ikemura</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span>â<span class="cit-article-title">Codon usage and tRNA content in unicellular and multicellular organisms.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">13</span>â<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DIkemura%26rft.auinit1%253DT%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253D13%26rft.epage%253D34%26rft.atitle%253DCodon%2Busage%2Band%2BtRNA%2Bcontent%2Bin%2Bunicellular%2Band%2Bmulticellular%2Borganisms%26rft_id%253Dinfo%253Apmid%252F3916708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjY6IjIvMS8xMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMjQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.14" data-doi="10.1371/journal.pgen.1004941"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lassalle</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">PÃ©rian</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bataillon</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nesme</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duret</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">GC-Content Evolution in Bacterial Genomes: The Biased Gene Conversion Hypothesis Expands.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1004941</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1004941%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004941%26rft_id%253Dinfo%253Apmid%252F25659072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004941&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25659072&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1247.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.15" data-doi="10.1038/nature10965"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eugene</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">The anti-ShineâDalgarno sequence drives translational pausing and codon choice in bacteria.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">484</span>: <span class="cit-fpage">538</span>â<span class="cit-lpage">541</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DG.%2BW.%26rft.volume%253D484%26rft.issue%253D7395%26rft.spage%253D538%26rft.epage%253D541%26rft.atitle%253DThe%2Banti-Shine-Dalgarno%2Bsequence%2Bdrives%2Btranslational%2Bpausing%2Band%2Bcodon%2Bchoice%2Bin%2Bbacteria.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10965%26rft_id%253Dinfo%253Apmid%252F22456704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10965&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22456704&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1247.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303200400056&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.16" data-doi="10.1073/pnas.0804445105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lind</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">D. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Whole-genome mutational biases in bacteria.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">17878</span>â<span class="cit-lpage">17883</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0804445105%26rft_id%253Dinfo%253Apmid%252F19001264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzQ2LzE3ODc4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.17" data-doi="10.1093/gbe/evq055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCutcheon</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moran</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Functional convergence in reduced genomes of bacterial symbionts spanning 200 My of evolution.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">708</span>â<span class="cit-lpage">718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevq055%26rft_id%253Dinfo%253Apmid%252F20829280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjIvMC83MDgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTI0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.18" data-doi="10.1111/j.1601-5223.1998.00173.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McEwan</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatherer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McEwan</span>  <span class="cit-name-given-names">N. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span>â<span class="cit-article-title">Nitrogen-fixing aerobic bacteria have higher genomic GC content than non-fixing species within the same genus.</span> <abbr class="cit-jnl-abbrev">Hereditas</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">173</span>â<span class="cit-lpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHereditas%26rft.stitle%253DHereditas%26rft.aulast%253DMcEwan%26rft.auinit1%253DC.%2BE.%26rft.volume%253D128%26rft.issue%253D2%26rft.spage%253D173%26rft.epage%253D178%26rft.atitle%253DNitrogen-fixing%2Baerobic%2Bbacteria%2Bhave%2Bhigher%2Bgenomic%2BGC%2Bcontent%2Bthan%2Bnon-fixing%2Bspecies%2Bwithin%2Bthe%2Bsame%2Bgenus.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1601-5223.1998.00173.x%26rft_id%253Dinfo%253Apmid%252F9687237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1601-5223.1998.00173.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9687237&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1247.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000074318500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.19" data-doi="10.1111/j.1751-7915.2008.00023.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">MiroÅczuk</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">KovÃ¡cs</span>  <span class="cit-name-given-names">Ã. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuipers</span>  <span class="cit-name-given-names">O. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Induction of natural competence in <em>Bacillus cereus</em> ATCC14579.</span> <abbr class="cit-jnl-abbrev">Microb. Biotechnol.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">226</span>â<span class="cit-lpage">235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMicrob%2BBiotechnol%26rft.aulast%253DMironczuk%26rft.auinit1%253DA.%2BM.%26rft.volume%253D1%26rft.issue%253D3%26rft.spage%253D226%26rft.epage%253D235%26rft.atitle%253DInduction%2Bof%2Bnatural%2Bcompetence%2Bin%2BBacillus%2Bcereus%2BATCC14579.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1751-7915.2008.00023.x%26rft_id%253Dinfo%253Apmid%252F21261842%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1751-7915.2008.00023.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21261842&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1247.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.20" data-doi="10.1073/pnas.84.1.166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muto</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osawa</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span>â<span class="cit-article-title">The guanine and cytosine content of genomic DNA and bacterial evolution.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">166</span>â<span class="cit-lpage">169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DMuto%26rft.auinit1%253DA.%26rft.volume%253D84%26rft.issue%253D1%26rft.spage%253D166%26rft.epage%253D169%26rft.atitle%253DThe%2Bguanine%2Band%2Bcytosine%2Bcontent%2Bof%2Bgenomic%2BDNA%2Band%2Bbacterial%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.84.1.166%26rft_id%253Dinfo%253Apmid%252F3467347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI4NC8xLzE2NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMjQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.21" data-doi="10.1007/s00239-002-2323-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Naya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romero</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zavala</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alvarez</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Musto</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Aerobiosis increases the genomic guanine plus cytosine content (GC%) in prokaryotes.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">260</span>â<span class="cit-lpage">264</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DNaya%26rft.auinit1%253DH.%26rft.volume%253D55%26rft.issue%253D3%26rft.spage%253D260%26rft.epage%253D264%26rft.atitle%253DAerobiosis%2Bincreases%2Bthe%2Bgenomic%2Bguanine%2Bplus%2Bcytosine%2Bcontent%2B%2528GC%2525%2529%2Bin%2Bprokaryotes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-002-2323-3%26rft_id%253Dinfo%253Apmid%252F12187379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-002-2323-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12187379&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1247.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177512700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Development Core Team</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">R: A language and environment for statistical computing. R Found.</span> <abbr class="cit-jnl-abbrev">Stat. Comput.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BComput.%26rft.volume%253D1%26rft.spage%253D409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.23" data-doi="10.1073/pnas.1205683109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raghavan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelkar</span>  <span class="cit-name-given-names">Y. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ochman</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">A selective force favoring increased G+C content in bacterial genes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">14504</span>â<span class="cit-lpage">14507</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1205683109%26rft_id%253Dinfo%253Apmid%252F22908296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzM2LzE0NTA0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.24" data-doi="10.1016/S0168-9525(02)02690-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rocha</span>  <span class="cit-name-given-names">E. P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danchin</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Base composition bias might result from competition for metabolic resources.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">291</span>â<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DRocha%26rft.auinit1%253DE.%2BP.%26rft.volume%253D18%26rft.issue%253D6%26rft.spage%253D291%26rft.epage%253D294%26rft.atitle%253DBase%2Bcomposition%2Bbias%2Bmight%2Bresult%2Bfrom%2Bcompetition%2Bfor%2Bmetabolic%2Bresources.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252802%252902690-2%26rft_id%253Dinfo%253Apmid%252F12044357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(02)02690-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12044357&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1247.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175854700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.25" data-doi="10.1371/journal.pgen.1001104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rocha</span>  <span class="cit-name-given-names">E. P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feil</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Mutational patterns cannot explain genome composition: Are there any neutral sites in the genomes of bacteria?</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D6%26rft.spage%253De1001104%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001104%26rft_id%253Dinfo%253Apmid%252F20838590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20838590&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1247.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.26" data-doi="10.1007/s00239-009-9230-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Romero</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pereira</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Musto</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Oxygen and guanine-cytosine profiles in marine environments.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">203</span>â<span class="cit-lpage">206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DRomero%26rft.auinit1%253DH.%26rft.volume%253D69%26rft.issue%253D2%26rft.spage%253D203%26rft.epage%253D206%26rft.atitle%253DOxygen%2Band%2Bguanine-cytosine%2Bprofiles%2Bin%2Bmarine%2Benvironments.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-009-9230-9%26rft_id%253Dinfo%253Apmid%252F19554248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-009-9230-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19554248&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1247.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.27" data-doi="10.1093/nar/gki242"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailes</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grocock</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peden</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sockett</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Variation in the strength of selected codon usage bias among bacteria.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1141</span>â<span class="cit-lpage">1153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D33%26rft.issue%253D4%26rft.spage%253D1141%26rft.epage%253D1153%26rft.atitle%253DVariation%2Bin%2Bthe%2Bstrength%2Bof%2Bselected%2Bcodon%2Busage%2Bbias%2Bamong%2Bbacteria%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki242%26rft_id%253Dinfo%253Apmid%252F15728743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjMzLzQvMTE0MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMjQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.28" data-doi="10.1098/rstb.2009.0305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emery</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Forces that influence the evolution of codon bias.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">365</span>: <span class="cit-fpage">1203</span>â<span class="cit-lpage">1212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D365%26rft.issue%253D1544%26rft.spage%253D1203%26rft.epage%253D1212%26rft.atitle%253DForces%2Bthat%2Binfluence%2Bthe%2Bevolution%2Bof%2Bcodon%2Bbias%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2009.0305%26rft_id%253Dinfo%253Apmid%252F20308095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEzOiIzNjUvMTU0NC8xMjAzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.29" data-doi="10.1093/nar/15.3.1281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span>â<span class="cit-article-title">The Codon Adaptation Index â a measure of directional synonymous codon usage bias, and its potential applications.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1281</span>â<span class="cit-lpage">1295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D1281%26rft.epage%253D1295%26rft.atitle%253DThe%2Bcodon%2Badaptation%2Bindex-a%2Bmeasure%2Bof%2Bdirectional%2Bsynonymous%2Bcodon%2Busage%2Bbias%252C%2Band%2Bits%2Bpotential%2Bapplications%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F15.3.1281%26rft_id%253Dinfo%253Apmid%252F3547335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjE1LzMvMTI4MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMjQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.30" data-doi="10.1073/pnas.48.4.582"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sueoka</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1962</span>â<span class="cit-article-title">On the genetic basis of variation and heterogeneity of DNA base composition.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">582</span>â<span class="cit-lpage">592</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSueoka%26rft.auinit1%253DN.%26rft.volume%253D48%26rft.issue%253D4%26rft.spage%253D582%26rft.epage%253D592%26rft.atitle%253DON%2BTHE%2BGENETIC%2BBASIS%2BOF%2BVARIATION%2BAND%2BHETEROGENEITY%2BOF%2BDNA%2BBASE%2BCOMPOSITION%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.48.4.582%26rft_id%253Dinfo%253Apmid%252F13918161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI0OC80LzU4MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMjQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.31" data-doi="10.1371/journal.pone.0002103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arakaki</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skolnick</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirby</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">The mosaic genome of <em>Anaeromyxobacter dehalogenans</em> strain 2CP-C suggests an aerobic common ancestor to the delta-proteobacteria.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e2103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DThomas%26rft.auinit1%253DS.%2BH.%26rft.volume%253D3%26rft.issue%253D5%26rft.spage%253De2103%26rft.epage%253De2103%26rft.atitle%253DThe%2Bmosaic%2Bgenome%2Bof%2BAnaeromyxobacter%2Bdehalogenans%2Bstrain%2B2CP-C%2Bsuggests%2Ban%2Baerobic%2Bcommon%2Bancestor%2Bto%2Bthe%2Bdelta-proteobacteria.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0002103%26rft_id%253Dinfo%253Apmid%252F18461135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0002103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18461135&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1247.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.32" data-doi="10.1371/journal.pgen.1000344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Touchon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoede</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tenaillon</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barve</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baeriswyl</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Organized genome dynamics in the <em>Escherichia coli</em> species results in highly diverse adaptive paths.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DTouchon%26rft.auinit1%253DM.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253De1000344%26rft.epage%253De1000344%26rft.atitle%253DOrganised%2Bgenome%2Bdynamics%2Bin%2Bthe%2BEscherichia%2Bcoli%2Bspecies%2Bresults%2Bin%2Bhighly%2Bdiverse%2Badaptive%2Bpaths.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000344%26rft_id%253Dinfo%253Apmid%252F19165319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000344&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19165319&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1247.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.33" data-doi="10.1093/gbe/evr125"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Leuven</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCutcheon</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">An AT mutational bias in the tiny GC-rich endosymbiont genome of <em>Hodgkinia</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">24</span>â<span class="cit-lpage">27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevr125%26rft_id%253Dinfo%253Apmid%252F22113795%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjY6IjQvMS8yNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMjQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.6.1247.34" data-doi="10.1016/j.bbrc.2006.02.037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Susko</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roger</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">On the correlation between genomic G+C content and optimal growth temperature in prokaryotes: data quality and confounding factors.</span> <abbr class="cit-jnl-abbrev">Biochem. Biophys. Res. Commun.</abbr> <span class="cit-vol">342</span>: <span class="cit-fpage">681</span>â<span class="cit-lpage">684</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2Band%2Bbiophysical%2Bresearch%2Bcommunications%26rft.stitle%253DBiochem%2BBiophys%2BRes%2BCommun%26rft.aulast%253DWang%26rft.auinit1%253DH.%2BC.%26rft.volume%253D342%26rft.issue%253D3%26rft.spage%253D681%26rft.epage%253D684%26rft.atitle%253DOn%2Bthe%2Bcorrelation%2Bbetween%2Bgenomic%2BG%252BC%2Bcontent%2Band%2Boptimal%2Bgrowth%2Btemperature%2Bin%2Bprokaryotes%253A%2Bdata%2Bquality%2Band%2Bconfounding%2Bfactors.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbrc.2006.02.037%26rft_id%253Dinfo%253Apmid%252F16499870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbrc.2006.02.037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16499870&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1247.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236168600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/6/1247.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/6/1233" title="Functional Analysis of Developmentally Regulated Genes chs7 and sec22 in the Ascomycete Sordaria macrospora" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/6/1253" title="Improved Detection and Characterization of Copy Number Variations Among Diverse Pig Breeds by Array CGH" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/6"><span>Volume 5 Issue 6, June 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/6.cover-source.jpg?itok=NSNFWJbY" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1247.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-hxZ8JzupJfDQIQBUAil3nD7ysD6EovgVvmQANNh-KkU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/6/1247" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/6/1247" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/6/1247" class="active" data-icon-position="" data-hide-link-title="0">Effects of Genic Base Composition on Growth Rate in G+C-rich Genomes</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459911" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-HJgwSikcgpMaMdSfHBEuD1-aIR7d0R89nOIC-7_Dr64" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/6/1247.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/6/1247" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-FbVwYkrFGEuUXFfkxYD_g6wZ5HPc5jh2Uk2ifl6dPR0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/6/1247" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/6/1247.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459911" id="node459911--2" data-pisa="ggg;5/6/1247" data-pisa-master="ggg;g3.115.016824" data-apath="/ggg/5/6/1247.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Effects of Genic Base Composition on Growth Rate in G+C-rich Genomes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4440-0191" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yogeshwar D. Kelkar</span>, <span class="highwire-citation-author" data-delta="1">Daniel S. Phillips</span> and <span class="highwire-citation-author" data-delta="2">Howard Ochman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1247-1252;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.016824 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459911--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yogeshwar D. Kelkar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Ecology and Evolutionary Biology, Yale University, New Haven, Connecticut 06520</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yogeshwar%2BD.%2BKelkar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kelkar%20YD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYogeshwar%2BD.%2BKelkar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4440-0191" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yogeshwar D. Kelkar</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel S. Phillips </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Ecology and Evolutionary Biology, Yale University, New Haven, Connecticut 06520</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BS.%2BPhillips%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Phillips%20DS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BS.%2BPhillips%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Howard Ochman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Ecology and Evolutionary Biology, Yale University, New Haven, Connecticut 06520</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Howard%2BOchman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ochman%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHoward%2BOchman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:howard.ochman@austin.utexas.edu" class="" data-icon-position="" data-hide-link-title="0">howard.ochman@austin.utexas.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/6/1247" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/6/1247&amp;title=Effects%20of%20Genic%20Base%20Composition%20on%20Growth%20Rate%20in%20G%2BC-rich%20Genomes" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/6/1247&amp;title=Effects%20of%20Genic%20Base%20Composition%20on%20Growth%20Rate%20in%20G%2BC-rich%20Genomes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/6/1247&amp;title=Effects%20of%20Genic%20Base%20Composition%20on%20Growth%20Rate%20in%20G%2BC-rich%20Genomes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/6/1247&amp;text=Effects%20of%20Genic%20Base%20Composition%20on%20Growth%20Rate%20in%20G%2BC-rich%20Genomes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/6/1247&amp;title=Effects%20of%20Genic%20Base%20Composition%20on%20Growth%20Rate%20in%20G%2BC-rich%20Genomes" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/6/1247&amp;t=Effects%20of%20Genic%20Base%20Composition%20on%20Growth%20Rate%20in%20G%2BC-rich%20Genomes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/6/1247&amp;title=Effects%20of%20Genic%20Base%20Composition%20on%20Growth%20Rate%20in%20G%2BC-rich%20Genomes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/6/1247&amp;title=Effects%20of%20Genic%20Base%20Composition%20on%20Growth%20Rate%20in%20G%2BC-rich%20Genomes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459911/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459911" id="node459911--3" data-pisa="ggg;5/6/1247" data-pisa-master="ggg;g3.115.016824" data-apath="/ggg/5/6/1247.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Effects of Genic Base Composition on Growth Rate in G+C-rich Genomes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4440-0191" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yogeshwar D. Kelkar</span>, <span class="highwire-citation-author" data-delta="1">Daniel S. Phillips</span> and <span class="highwire-citation-author" data-delta="2">Howard Ochman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1247-1252;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.016824 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459911--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yogeshwar D. Kelkar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Ecology and Evolutionary Biology, Yale University, New Haven, Connecticut 06520</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yogeshwar%2BD.%2BKelkar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kelkar%20YD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYogeshwar%2BD.%2BKelkar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4440-0191" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yogeshwar D. Kelkar</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel S. Phillips </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Ecology and Evolutionary Biology, Yale University, New Haven, Connecticut 06520</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BS.%2BPhillips%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Phillips%20DS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BS.%2BPhillips%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Howard Ochman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Ecology and Evolutionary Biology, Yale University, New Haven, Connecticut 06520</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Howard%2BOchman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ochman%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHoward%2BOchman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:howard.ochman@austin.utexas.edu" class="" data-icon-position="" data-hide-link-title="0">howard.ochman@austin.utexas.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1247&amp;rft.epage=1252&amp;rft.atitle=Effects%20of%20Genic%20Base%20Composition%20on%20Growth%20Rate%20in%20G%2BC-rich%20Genomes&amp;rft.volume=5&amp;rft.issue=6&amp;rft.date=2015-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Kelkar%2C+Yogeshwar+D.&amp;rft.au=Phillips%2C+Daniel+S.&amp;rft.au=Ochman%2C+Howard"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459911/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459911/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459911/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459911/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459911/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459911/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459911/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459911/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459911/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459911/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459911/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459911/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/6/1247" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/6/1247.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/6/1247.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/6/1247.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/6/1247.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
