<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/6/1301" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/6/1301.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/6/1301.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/6/1301.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/6/1301.atom" />
<meta name="HW.pisa" content="ggg;5/6/1301" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Mutations that Separate the Functions of the Proofreading Subunit of the Escherichia coli Replicase" />
<meta name="DC.Identifier" content="10.1534/g3.115.017285" />
<meta name="DC.Date" content="2015-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Whatley and Kreuzer. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The dnaQ gene of Escherichia coli encodes the ε subunit of DNA polymerase III, which provides the 3′ → 5′ exonuclease proofreading activity of the replicative polymerase. Prior studies have shown that loss of ε leads to high mutation frequency, partially constitutive SOS, and poor growth. In addition, a previous study from our laboratory identified dnaQ knockout mutants in a screen for mutants specifically defective in the SOS response after quinolone (nalidixic acid) treatment. To explain these results, we propose a model whereby, in addition to proofreading, ε plays a distinct role in replisome disassembly and/or processing of stalled replication forks. To explore this model, we generated a pentapeptide insertion mutant library of the dnaQ gene, along with site-directed mutants, and screened for separation of function mutants. We report the identification of separation of function mutants from this screen, showing that proofreading function can be uncoupled from SOS phenotypes (partially constitutive SOS and the nalidixic acid SOS defect). Surprisingly, the two SOS phenotypes also appear to be separable from each other. These findings support the hypothesis that ε has additional roles aside from proofreading. Identification of these mutants, especially those with normal proofreading but SOS phenotype(s), also facilitates the study of the role of ε in SOS processes without the confounding results of high mutator activity associated with dnaQ knockout mutants." />
<meta name="DC.Contributor" content="Zakiya Whatley" />
<meta name="DC.Contributor" content="Kenneth N. Kreuzer" />
<meta name="article:published_time" content="2015-06-01" />
<meta name="article:section" content="Mutant Screen Report" />
<meta name="citation_title" content="Mutations that Separate the Functions of the Proofreading Subunit of the Escherichia coli Replicase" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The &lt;i&gt;dnaQ&lt;/i&gt; gene of &lt;i&gt;Escherichia coli&lt;/i&gt; encodes the ε subunit of DNA polymerase III, which provides the 3′ → 5′ exonuclease proofreading activity of the replicative polymerase. Prior studies have shown that loss of ε leads to high mutation frequency, partially constitutive SOS, and poor growth. In addition, a previous study from our laboratory identified &lt;i&gt;dnaQ&lt;/i&gt; knockout mutants in a screen for mutants specifically defective in the SOS response after quinolone (nalidixic acid) treatment. To explain these results, we propose a model whereby, in addition to proofreading, ε plays a distinct role in replisome disassembly and/or processing of stalled replication forks. To explore this model, we generated a pentapeptide insertion mutant library of the &lt;i&gt;dnaQ&lt;/i&gt; gene, along with site-directed mutants, and screened for separation of function mutants. We report the identification of separation of function mutants from this screen, showing that proofreading function can be uncoupled from SOS phenotypes (partially constitutive SOS and the nalidixic acid SOS defect). Surprisingly, the two SOS phenotypes also appear to be separable from each other. These findings support the hypothesis that ε has additional roles aside from proofreading. Identification of these mutants, especially those with normal proofreading but SOS phenotype(s), also facilitates the study of the role of ε in SOS processes without the confounding results of high mutator activity associated with &lt;i&gt;dnaQ&lt;/i&gt; knockout mutants.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/06/01" />
<meta name="citation_mjid" content="ggg;5/6/1301" />
<meta name="citation_id" content="5/6/1301" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/6/1301" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/6/1301.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/6/1301.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/6/1301.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.017285" />
<meta name="citation_pmid" content="25878065" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Mutant Screen Report" />
<meta name="citation_firstpage" content="1301" />
<meta name="citation_lastpage" content="1311" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Zakiya Whatley" />
<meta name="citation_author_institution" content="University Program in Genetics and Genomics, Duke University, Durham, North Carolina 27705" />
<meta name="citation_author" content="Kenneth N. Kreuzer" />
<meta name="citation_author_institution" content="University Program in Genetics and Genomics, Duke University, Durham, North Carolina 27705" />
<meta name="citation_author_institution" content="Department of Biochemistry, Duke University Medical Center, Durham, North Carolina 27710" />
<meta name="citation_author_email" content="kenneth.kreuzer@duke.edu" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=B. P. Anton;citation_author=E. A. Raleigh;citation_title=Transposon-Mediated Linker Insertion Scanning Mutagenesis of the Escherichia coli McrA Endonuclease;citation_pages=5699-5707;citation_volume=186;citation_year=2004;citation_issue=17;citation_pmid=15317774;citation_doi=10.1128/JB.186.17.5699-5707.2004" />
<meta name="citation_reference" content="citation_journal_title=Mol. Syst. Biol.;citation_author=T. Baba;citation_author=T. Ara;citation_author=M. Hasegawa;citation_author=Y. Takai;citation_author=Y. Okumura;citation_title=Construction of Escherichia coli K-12 in-frame, single-gene knockout mutants: the Keio collection.;citation_pages=1-11;citation_volume=2;citation_year=2006;citation_doi=10.1038/msb4100050" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=M. H. Barnes;citation_author=P. Spacciapoli;citation_author=D. H. Li;citation_author=N. C. Brown;citation_title=The 3'-5' exonuclease site of DNA polymerase III from gram-positive bacteria: definition of a novel motif structure.;citation_pages=45-50;citation_volume=165;citation_year=1995;citation_issue=1;citation_pmid=7489914;citation_doi=10.1016/0378-1119(95)00530-J" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=H. M. Berman;citation_author=J. Westbrook;citation_author=Z. Feng;citation_author=G. Gilliland;citation_author=T. N. Bhat;citation_title=The Protein Data Bank;citation_pages=235-242;citation_volume=28;citation_year=2000;citation_issue=1;citation_pmid=10592235;citation_doi=10.1093/nar/28.1.235" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=A. Bernad;citation_author=L. Blanco;citation_author=J. M. Lazaro;citation_author=G. Martin;citation_author=M. Salas;citation_title=A conserved 3'----5' exonuclease active site in prokaryotic and eukaryotic DNA polymerases.;citation_pages=219-228;citation_volume=59;citation_year=1989;citation_issue=1;citation_pmid=2790959;citation_doi=10.1016/0092-8674(89)90883-0" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=L. Blanco;citation_author=A. Bernad;citation_author=M. Salas;citation_title=Evidence favouring the hypothesis of a conserved 3'-5' exonuclease active site in DNA-dependent DNA polymerases.;citation_pages=139-144;citation_volume=112;citation_year=1992;citation_issue=1;citation_pmid=1551594;citation_doi=10.1016/0378-1119(92)90316-H" />
<meta name="citation_reference" content="citation_journal_title=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_journal_abbrev=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_author=D. Bressanin;citation_author=A. Stefan;citation_author=F. D. Piaz;citation_author=S. Cianchetta;citation_author=L. Reggiani;citation_title=Proteolysis of the proofreading subunit controls the assembly of Escherichia coli DNA polymerase III catalytic core.;citation_pages=1606-1615;citation_volume=1794;citation_year=2009;citation_issue=11;citation_pmid=19635595;citation_doi=10.1016/j.bbapap.2009.07.011" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=Y. Cao;citation_author=B. Hallet;citation_author=D. Sherratt;citation_author=F. Hayes;citation_title=Structure-function correlations in the XerD site-specific recombinase revealed by pentapeptide scanning mutagenesis.;citation_pages=39-53;citation_volume=274;citation_year=1997;citation_issue=1;citation_pmid=9398514;citation_doi=10.1006/jmbi.1997.1380" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=G. E. Degnen;citation_author=E. C. Cox;citation_title=Conditional Mutator Gene in Escherichia coli: Isolation, Mapping, and Effector Studies;citation_pages=477-487;citation_volume=117;citation_year=1974;citation_issue=2;citation_pmid=4590472" />
<meta name="citation_reference" content="citation_journal_title=Biochemistry;citation_author=E. F. DeRose;citation_author=T. Darden;citation_author=S. Harvey;citation_author=S. Gabel;citation_author=F. W. Perrino;citation_title=ELUCIDATION OF THE  -  INTERFACE OF ESCHERICHIA COLI DNA POLYMERASE III BY NMR SPECTROSCOPY;citation_pages=3635-3644;citation_volume=42;citation_year=2003;citation_pmid=12667053;citation_doi=10.1021/bi0205451" />
<meta name="citation_reference" content="citation_journal_title=Antimicrob. Agents Chemother.;citation_author=K. Drlica;citation_author=M. Malik;citation_author=R. J. Kerns;citation_author=X. Zhao;citation_title=Quinolone-mediated bacterial cell death.;citation_volume=52;citation_year=2008;citation_pmid=17724149;citation_doi=10.1128/AAC.01617-06" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=H. Echols;citation_author=C. Lu;citation_author=P. Burgers;citation_title=Mutator strains of Escherichia coli, mutD and dnaQ, with defective exonucleolytic editing by DNA polymerase III holoenzyme.;citation_pages=2189-2192;citation_volume=80;citation_year=1983;citation_issue=8;citation_pmid=6340117;citation_doi=10.1073/pnas.80.8.2189" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=I. J. Fijalkowska;citation_author=R. M. Schaaper;citation_title=Effects of Escherichia coli dnaE antimutator alleles in a proofreading-deficient mutD5 strain.;citation_pages=5979-5986;citation_volume=177;citation_year=1995;citation_issue=20;citation_pmid=7592352;citation_doi=10.1128/jb.177.20.5979-5986.1995" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=I. J. Fijalkowska;citation_author=R. M. Schaaper;citation_title=Mutants in the Exo I motif of Escherichia coli dnaQ: defective proofreading and inviability due to error catastrophe.;citation_pages=2856-2861;citation_volume=93;citation_year=1996;citation_issue=7;citation_pmid=8610131;citation_doi=10.1073/pnas.93.7.2856" />
<meta name="citation_reference" content="citation_journal_title=Methods in enzymology;citation_journal_abbrev=Methods Enzymol;citation_author=P. L. Foster;citation_title=Methods for determining spontaneous mutation rates.;citation_pages=195-213;citation_volume=409;citation_year=2006;citation_pmid=16793403;citation_doi=10.1016/S0076-6879(05)09012-9" />
<meta name="citation_reference" content="citation_journal_title=Structure (London);citation_journal_abbrev=Structure (London);citation_author=S. Hamdan;citation_author=P. D. Carr;citation_author=S. E. Brown;citation_author=D. L. Ollis;citation_author=N. E. Dixon;citation_title=Structural basis for proofreading during replication of the Escherichia coli chromosome.;citation_pages=535-546;citation_volume=10;citation_year=2002;citation_issue=4;citation_pmid=11937058;citation_doi=10.1016/S0969-2126(02)00738-4" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=C. M. Hamilton;citation_author=M. Aldea;citation_author=B. K. Washburn;citation_author=P. Babitzke;citation_author=S. R. Kushner;citation_title=New method for generating deletions and gene replacements in Escherichia coli.;citation_pages=4617-4622;citation_volume=171;citation_year=1989;citation_issue=9;citation_pmid=2548993;citation_doi=10.1128/jb.171.9.4617-4622.1989" />
<meta name="citation_reference" content="citation_journal_title=Molecular &amp; general genetics :  MGG;citation_journal_abbrev=Mol Gen Genet;citation_author=T. Horiuchi;citation_author=H. Maki;citation_author=M. Sekiguchi;citation_title=A new conditional lethal mutator (dnaQ49) in Escherichia coli K12.;citation_pages=277-283;citation_volume=163;citation_year=1978;citation_issue=3;citation_pmid=355854;citation_doi=10.1007/BF00271956" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=S. Jergic;citation_author=N. P. Horan;citation_author=M. M. Elshenawy;citation_author=C. E. Mason;citation_author=T. Urathamakul;citation_title=A direct proofreader-clamp interaction stabilizes the Pol III replicase in the polymerization mode.;citation_volume=32;citation_year=2013;citation_pmid=23435564;citation_doi=10.1038/emboj.2012.347" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=P. Jonczyk;citation_author=A. Nowicka;citation_author=I. J. Fijalkowska;citation_author=R. M. Schaaper;citation_author=Z. Ciesla;citation_title=In Vivo Protein Interactions within the Escherichia coli DNA Polymerase III Core;citation_pages=1563-1566;citation_volume=180;citation_year=1998;citation_issue=6;citation_pmid=9515927" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=M. A. Keniry;citation_author=A. Y. Park;citation_author=E. A. Owen;citation_author=S. M. Hamdan;citation_author=G. Pintacuda;citation_title=Structure of the {theta} Subunit of Escherichia coli DNA Polymerase III in Complex with the {varepsilon} Subunit;citation_pages=4464-4473;citation_volume=188;citation_year=2006;citation_issue=12;citation_pmid=16740953;citation_doi=10.1128/JB.01992-05" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=D. R. Kim;citation_author=C. S. McHenry;citation_title=In Vivo Assembly of Overproduced DNA Polymerase III: OVERPRODUCTION, PURIFICATION, AND CHARACTERIZATION OF THE {alpha}, {alpha}-{varepsilon}, AND {alpha}-{varepsilon}-{theta} SUBUNITS;citation_pages=20681-20689;citation_volume=271;citation_year=1996;citation_issue=34;citation_pmid=8702818;citation_doi=10.1074/jbc.271.34.20681" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=E. D. Lancy;citation_author=M. R. Lifsics;citation_author=D. G. Kehres;citation_author=R. Maurer;citation_title=Isolation and characterization of mutants with deletions in dnaQ, the gene for the editing subunit of DNA polymerase III in Salmonella typhimurium.;citation_pages=5572-5580;citation_volume=171;citation_year=1989;citation_issue=10;citation_pmid=2551891;citation_doi=10.1128/jb.171.10.5572-5580.1989" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=M. R. Lifsics;citation_author=E. D. Lancy;citation_author=R. Maurer;citation_title=DNA replication defect in Salmonella typhimurium mutants lacking the editing (epsilon) subunit of DNA polymerase III.;citation_pages=6965-6973;citation_volume=174;citation_year=1992;citation_issue=21;citation_pmid=1400246;citation_doi=10.1128/jb.174.21.6965-6973.1992" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=H. Maki;citation_author=T. Horiuchi;citation_author=M. Sekiguchi;citation_title=Structure and expression of the dnaQ mutator and the RNase H genes of Escherichia coli: overlap of the promoter regions.;citation_pages=7137-7141;citation_volume=80;citation_year=1983;citation_issue=23;citation_pmid=6316347;citation_doi=10.1073/pnas.80.23.7137" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=L. S. McDaniel;citation_author=L. H. Rogers;citation_author=W. E. Hill;citation_title=Survival of recombination-deficient mutants of Escherichia coli during incubation with nalidixic acid.;citation_pages=1195-1198;citation_volume=134;citation_year=1978;citation_issue=3;citation_pmid=350844" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=A. McPartland;citation_author=L. Green;citation_author=H. Echols;citation_title=Control of recA gene RNA in E. coli: regulatory and signal genes.;citation_pages=731-737;citation_volume=20;citation_year=1980;citation_issue=3;citation_pmid=6998563;citation_doi=10.1016/0092-8674(80)90319-0" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=K. G. Newmark;citation_author=E. K. O’Reilly;citation_author=J. R. Pohlhaus;citation_author=K. N. Kreuzer;citation_title=Genetic analysis of the requirements for SOS induction by nalidixic acid in Escherichia coli.;citation_pages=69-76;citation_volume=356;citation_year=2005;citation_pmid=16005583;citation_doi=10.1016/j.gene.2005.04.029" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. Ozawa;citation_author=N. P. Horan;citation_author=A. Robinson;citation_author=H. Yagi;citation_author=F. R. Hill;citation_title=Proofreading exonuclease on a tether: the complex between the E. coli DNA polymerase III subunits alpha, epsilon, theta and beta reveals a highly flexible arrangement of the proofreading domain.;citation_volume=41;citation_year=2013;citation_pmid=23580545;citation_doi=10.1093/nar/gkt162" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. Ozawa;citation_author=S. Jergic;citation_author=A. Y. Park;citation_author=N. E. Dixon;citation_author=G. Otting;citation_title=The proofreading exonuclease subunit epsilon of Escherichia coli DNA polymerase III is tethered to the polymerase subunit alpha via a flexible linker.;citation_volume=36;citation_year=2008;citation_pmid=18663010;citation_doi=10.1093/nar/gkn489" />
<meta name="citation_reference" content="citation_journal_title=Biochemistry;citation_author=F. W. Perrino;citation_author=S. Harvey;citation_author=S. M. McNeill;citation_title=Two functional domains of the epsilon subunit of DNA polymerase III.;citation_pages=16001-16009;citation_volume=38;citation_year=1999;citation_pmid=10625468;citation_doi=10.1021/bi991429+" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=J. R. Pohlhaus;citation_author=K. N. Kreuzer;citation_title=Norfloxacin-induced DNA gyrase cleavage complexes block Escherichia coli replication forks, causing double-stranded breaks in vivo.;citation_pages=1416-1429;citation_volume=56;citation_year=2005;citation_issue=6;citation_pmid=15916595;citation_doi=10.1111/j.1365-2958.2005.04638.x" />
<meta name="citation_reference" content="citation_journal_title=J. Bacteriol.;citation_author=J. R. Pohlhaus;citation_author=D. T. Long;citation_author=E. O’Reilly;citation_author=K. N. Kreuzer;citation_title=The epsilon subunit of DNA polymerase III is involved in the nalidixic acid-induced SOS response in Escherichia coli.;citation_volume=190;citation_year=2008;citation_pmid=18539731;citation_doi=10.1128/JB.00173-08" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. J. Saveson;citation_author=S. T. Lovett;citation_title=Enhanced Deletion Formation by Aberrant DNA Replication in Escherichia coli;citation_pages=457-470;citation_volume=146;citation_year=1997;citation_issue=2;citation_pmid=9177997" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=R. M. Schaaper;citation_title=Mechanisms of mutagenesis in the Escherichia coli mutator mutD5: role of DNA mismatch repair.;citation_pages=8126-8130;citation_volume=85;citation_year=1988;citation_issue=21;citation_pmid=3054881;citation_doi=10.1073/pnas.85.21.8126" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. M. Schaaper;citation_title=Escherichia coli mutator mutD5 is defective in the mutHLS pathway of DNA mismatch repair.;citation_pages=205-212;citation_volume=121;citation_year=1989;citation_issue=2;citation_pmid=2659431" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=R. M. Schaaper;citation_author=R. Cornacchio;citation_title=An Escherichia coli dnaE mutation with suppressor activity toward mutator mutD5.;citation_pages=1974-1982;citation_volume=174;citation_year=1992;citation_issue=6;citation_pmid=1548237;citation_doi=10.1128/jb.174.6.1974-1982.1992" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=R. M. Schaaper;citation_author=M. Radman;citation_title=The extreme mutator effect of Escherichia coli mutD5 results from saturation of mismatch repair by excessive DNA replication errors.;citation_pages=3511-3516;citation_volume=8;citation_year=1989;citation_pmid=2555167" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=R. Scheuermann;citation_author=S. Tam;citation_author=P. M. Burgers;citation_author=C. Lu;citation_author=H. Echols;citation_title=Identification of the epsilon-subunit of Escherichia coli DNA polymerase III holoenzyme as the dnaQ gene product: a fidelity subunit for DNA replication.;citation_pages=7085-7089;citation_volume=80;citation_year=1983;citation_issue=23;citation_pmid=6359162;citation_doi=10.1073/pnas.80.23.7085" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=S. C. Slater;citation_author=M. R. Lifsics;citation_author=M. O’Donnell;citation_author=R. Maurer;citation_title=holE, the gene coding for the theta subunit of DNA polymerase III of Escherichia coli: characterization of a holE mutant and comparison with a dnaQ (epsilon-subunit) mutant.;citation_pages=815-821;citation_volume=176;citation_year=1994;citation_issue=3;citation_pmid=8300534;citation_doi=10.1128/jb.176.3.815-821.1994" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=T. R. Steck;citation_author=K. Drlica;citation_title=Bacterial chromosome segregation: evidence for DNA gyrase involvement in decatenation.;citation_pages=1081-1088;citation_volume=36;citation_year=1984;citation_issue=4;citation_pmid=6323018;citation_doi=10.1016/0092-8674(84)90058-8" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=P. S. Studwell;citation_author=M. O’Donnell;citation_title=Processive replication is contingent on the exonuclease subunit of DNA polymerase III holoenzyme.;citation_pages=1171-1178;citation_volume=265;citation_year=1990;citation_issue=2;citation_pmid=2153103" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=P. S. Studwell-Vaughan;citation_author=M. O’Donnell;citation_title=DNA polymerase III accessory proteins. V. Theta encoded by holE;citation_pages=11785-11791;citation_volume=268;citation_year=1993;citation_issue=16;citation_pmid=8505306" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. A. Taft-Benz;citation_author=R. M. Schaaper;citation_title=Mutational analysis of the 3′→5′ proofreading exonuclease of Escherichia coli DNA polymerase III.;citation_volume=26;citation_year=1998;citation_pmid=9705512;citation_doi=10.1093/nar/26.17.4005" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=S. A. Taft-Benz;citation_author=R. M. Schaaper;citation_title=The C-Terminal Domain of DnaQ Contains the Polymerase Binding Site;citation_pages=2963-2965;citation_volume=181;citation_year=1999;citation_issue=9;citation_pmid=10217794" />
<meta name="citation_reference" content="citation_journal_title=Molecular &amp; general genetics :  MGG;citation_journal_abbrev=Mol Gen Genet;citation_author=K. Takano;citation_author=Y. Nakabeppu;citation_author=H. Maki;citation_author=T. Horiuchi;citation_author=M. Sekiguchi;citation_title=Structure and function of dnaQ and mutD mutators of Escherichia coli.;citation_pages=9-13;citation_volume=205;citation_year=1986;citation_issue=1;citation_pmid=3540531;citation_doi=10.1007/BF02428026" />
<meta name="citation_reference" content="citation_journal_title=BioTechniques;citation_journal_abbrev=Biotechniques;citation_author=S. A. Thibodeau;citation_author=R. Fang;citation_author=J. K. Joung;citation_title=High-throughput beta-galactosidase assay for bacterial cell-based reporter systems.;citation_pages=410-415;citation_volume=36;citation_year=2004;citation_issue=3;citation_pmid=15038156" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=A. Toste Rego;citation_author=A. N. Holding;citation_author=H. Kent;citation_author=M. H. Lamers;citation_title=Architecture of the Pol III-clamp-exonuclease complex reveals key roles of the exonuclease subunit in processive DNA synthesis and repair.;citation_volume=32;citation_year=2013;citation_pmid=23549287;citation_doi=10.1038/emboj.2013.68" />
<meta name="citation_reference" content="citation_journal_title=Protein Engineering Design and Selection;citation_journal_abbrev=Protein Eng Des Sel;citation_author=J. C. Wootton;citation_author=M. H. Drummond;citation_title=The Q-linker: a class of interdomain sequences found in bacterial multidomain regulatory proteins;citation_pages=535-543;citation_volume=2;citation_year=1989;citation_issue=7;citation_pmid=2664763;citation_doi=10.1093/protein/2.7.535" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Mutations that Separate the Functions of the Proofreading Subunit of the Escherichia coli Replicase" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
<meta name="twitter:description" content="The dnaQ gene of Escherichia coli encodes the ε subunit of DNA polymerase III, which provides the 3′ → 5′ exonuclease proofreading activity of the replicative polymerase. Prior studies have shown that loss of ε leads to high mutation frequency, partially constitutive SOS, and poor growth. In addition, a previous study from our laboratory identified dnaQ knockout mutants in a screen for mutants specifically defective in the SOS response after quinolone (nalidixic acid) treatment. To explain these results, we propose a model whereby, in addition to proofreading, ε plays a distinct role in replisome disassembly and/or processing of stalled replication forks. To explore this model, we generated a pentapeptide insertion mutant library of the dnaQ gene, along with site-directed mutants, and screened for separation of function mutants. We report the identification of separation of function mutants from this screen, showing that proofreading function can be uncoupled from SOS phenotypes (partially constitutive SOS and the nalidixic acid SOS defect). Surprisingly, the two SOS phenotypes also appear to be separable from each other. These findings support the hypothesis that ε has additional roles aside from proofreading. Identification of these mutants, especially those with normal proofreading but SOS phenotype(s), also facilitates the study of the role of ε in SOS processes without the confounding results of high mutator activity associated with dnaQ knockout mutants." />
<meta name="og-title" property="og:title" content="Mutations that Separate the Functions of the Proofreading Subunit of the Escherichia coli Replicase" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/6/1301" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The dnaQ gene of Escherichia coli encodes the ε subunit of DNA polymerase III, which provides the 3′ → 5′ exonuclease proofreading activity of the replicative polymerase. Prior studies have shown that loss of ε leads to high mutation frequency, partially constitutive SOS, and poor growth. In addition, a previous study from our laboratory identified dnaQ knockout mutants in a screen for mutants specifically defective in the SOS response after quinolone (nalidixic acid) treatment. To explain these results, we propose a model whereby, in addition to proofreading, ε plays a distinct role in replisome disassembly and/or processing of stalled replication forks. To explore this model, we generated a pentapeptide insertion mutant library of the dnaQ gene, along with site-directed mutants, and screened for separation of function mutants. We report the identification of separation of function mutants from this screen, showing that proofreading function can be uncoupled from SOS phenotypes (partially constitutive SOS and the nalidixic acid SOS defect). Surprisingly, the two SOS phenotypes also appear to be separable from each other. These findings support the hypothesis that ε has additional roles aside from proofreading. Identification of these mutants, especially those with normal proofreading but SOS phenotype(s), also facilitates the study of the role of ε in SOS processes without the confounding results of high mutator activity associated with dnaQ knockout mutants." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
    <title>Mutations that Separate the Functions of the Proofreading Subunit of the Escherichia coli Replicase | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"VOE2Z72jehjDdzwjtvFE9FPa9OjaqtCNiV2KcZ9yMuU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459910","apath":"\/ggg\/5\/6\/1301.atom","pisa":"ggg;5\/6\/1301","ac":{"\/ggg\/5\/6\/1301.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/6\/1301.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/6\/1301"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/6\/1301"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/6\/1301.full":true,"\/content\/5\/6\/1301":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002229f63SqXnrCfPzPPBFR9lbv+bD6Xyz7EqjxmZYvVv94rHGbB8d7Q4snEtE+C+8FoGTVr8aLsLeOtyaUk8hGrxNVB33w3QICTvUizFstz9zHYudZaYCjOT10S+ET2eTrjglD10DdxWq5lMTjkdOxX05eX+ugA6xm+MBaMgKRAYhO+viVG1cnQuaw3Ou8hbshf8m8\\\\\\\/Pspk1S5bEric9qsryG7yjvfRJ7C\\\\\\\/tj72OdepdH4qBAxTk7TPBRHTNgAxb+R4RmZfwQYAIr+aOQYAKXVwQ07zGjp2EwsJQ73Hpo3hD1ogiJbiAfecFDWTcy4J9VSe8IjzC6lbRNXd8PQMK7zM0UDbINUP9Kr4BfGujih9i+59AKE+zWvAxm1NR+v7kdID3Rhe3NCH3\\\\\\\/8dY280qfsEY2NH543LjXz0Akt8LIfhEUmievxvAzvLUmvl6cTvcCFj+sp7jRQbXoupxpxw1aypXFT6lOHGKJUTF0WOvVnx9NCCkDStFc69RQrV0n8p8KAD0ASy5vp061bkqc5jPLRSZnk0\\\\\\\/hqMdCMW1pkwQo8mBdHpsOhHHPVhxlw26Noq+20dPPjpo1j1jRH2PZHAFouOclQRUM8G\\\\\\\/Kdw5EO2lppTdHgbl94\\\\\\\/DiG87aZTXx4ngLfzbcLAf8s8rdN9PVH2xOcDsrinRa4TV2Ck4u6aPqBjv60XTV7FYBa8NJlDuISjsvElIWsMrBjs28HgfU4O4tDyelIRYCTwk3KkN8eb7Ziso6IPojYxKpaTcvFEPR10OZMISViy1eeUVsP4fqcNpLDwN\\\\\\\/W7PHGY7ZEPjycehZG081thy\\\\\\\/VmUNwJ6MCCzPp62fwButWBZB8ee9KsyP4NTVgYnF8bcSxexMhfDaR8rZAuHP6gqda1A455taKBQQ6T\\\\\\\/sbf+DQ7D6wQb0VdO4gVpqUHDxYCgGaZqAkejs8OZWmWI6V\\\\\\\/oq672pbGslpUkwpXe3APEfBuYojGxLvvVyAp2mjxW+7UG3SDhHAInL+WNWRbbZFzeC+fU+CtOH\\\\\\\/N5OSC4XvZWfVmaJMnLxgD4rdFYiHlbdM90rEErkuQODzxMxnGcf6hXXDZO9tiLWXispzUPYckk+D1rCgRVvvdijKjWskABFj\\\\\\\/MmJoB4LhsuhHxn1ONiqyIuI\\\\\\\/5XxiyD0p8xpoNl\\\\\\\/SAVfecZ0jVjgNu3HFr1AB0\\\\\\\/GPzDEIBRdnEbpz40CFerMktxT+2585zjB5smlf7dHFqPzYc798KxdcpO9AoT\\\\\\\/O196mQF6+Ri5Mx9cYjbQaxkEz9fY88LBeNXBOVoYKi0VC\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/5TS4mTj+\\\\\\\/IAnQlrCLuZzw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022380b15b5592be9516d6e57b3fcae1fd4defa06d06148fb313ec1a8450a7898fa\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022IJoeUo\\\\\\\/afXF2EH9iMcSWINSlT9xcv+V3Yi7M\\\\\\\/DIvH2gyhETno6sYOC8qHcnYgzyZIjl1L4XDUO1oeM7PcYGsGmtd0OyHv5Bcme3ldSvs37oObZiuZwMDYbO19qOtcbDB07tLDtN+cm7QbtZyw9rxjBezTw8dy6E6Ot06ZZe7gz3aaDW6wRcbSz0g0kP60NNuxjkmmjikPCe0ki4niXpm2DNqHasOaWwA3J1kkQf\\\\\\\/2FtUpQeqb++O5iGrzTH8HJy13dk\\\\\\\/gba6gn0QBW\\\\\\\/MTMZtoM0yS8VB0ybR0PeVDZt9IPJhIgoNya\\\\\\\/2qhuWJtUt2+4OB1lc5+sVkKHYn1250U1pU6z6yT\\\\\\\/ShiZ\\\\\\\/GCigpeAnLDlWtgej\\\\\\\/RBFabu88IgOm8HL1GosIOpGLCfQXb+17ASTDJD+NmNp7i\\\\\\\/9EX\\\\\\\/uu3pKCCmmvBaJC1hMfgAPrGBJbRn37BzMBWrYj\\\\\\\/0GmkzbFPYuUMVhfbu1xVmiZ7nwq2wFgPqY8k2VGZPACssNmOaKAdN0bjj0kKVidzxd6DzH\\\\\\\/2uE3nwklLOAQC8233uD84dmPboIh9FH0\\\\\\\/ijptqNOek9mH8mL\\\\\\\/wr7FBH+x6PGAs5DK9dTd0ca1dphRpwfZo5i4UqNIAJsiIDqZVz1ctP2Zn7\\\\\\\/xThd0iXvI6XM3f1O\\\\\\\/CYRVLtmLK3BtQ2bZYZPI+SlGCgn+BPPzSLAHB1cvrgBfHuWxDHXnMGumB\\\\\\\/GAX\\\\\\\/JlfWHW2aTjteze++ZFSKQK8rK3KcvfFMaCHm36XYiPJxN2YiF6pBLAhq8WSpLcsWCG7O8bShrYiPCkVLdxUb7zRvzRv2X+CbXGp5LTxAvVenA6jSBUAOMzM+Zu3nNMnmBGWzMm0LB8HFK+rLCXM8cJx87FR0+VM\\\\\\\/XYN6HnYhTddNNvFNb8PR3jaa6nrfQTMxF\\\\\\\/UJHGvmXwoKc4Cpo3wZeiB1M7jTXsh+DJBd+imCxz9ymLBx1wZ2bKMJV7kXqo8UbsDZRHxJzzuBxo96dURnP3GfmVRzKumwTVbGZL9\\\\\\\/L3yTGIHuJ5Jmx3PHda4ESgH10DkPZc31j96vuHPw8VY0lxNYHUf8MtnZ7B5WKPc8YviF93W5rsVtSthMDjVuMYn07SZAUBFyjzEHxrKh1KaEgTIeoSZGxq6\\\\\\\/qgu\\\\\\\/7fcv70ZggckWOdG8UFEQx8MyK9Z5huFAvAp4BLVepigDZNxtKQ13o8Ds90jhSes7FjFij567vQV1s8a\\\\\\\/nSwwjUk3Fs7uDEwci98ep+KEwzr25viKWGuVMF3+AbxybuMd+CtDook2fo+PTReKEV0jyEJvLRuPL5nY9kSqTEaY0eEftqH4MOY8zjzCDNLU+S9laee0C\\\\\\\/ogr1p+RQ1rURBBmsqzmAB0uqdvj9nw90R6EvXTCeiEf1akKD7T9EujYTxRLp0TCE2qiX6ZLtxwJj95mcq+ZkLaTNO2NzK2MnjR\\\\\\\/MeGRqZGtejLprA4EaxOyo7E2NhhPgpbzgmaK9YpYhnkbIKLgEIE3qLdeA615uLU2JDG1sO\\\\\\\/97Dp1zya9Ko27A5yK+vGTPgUGhwTEvL+hoAZaZq6NCOe72\\\\\\\/aY50LTgz1siz0l4iq9f78GC7IJeeDk3C4a+WrwaKNx+f7E+m0vsuuOQiEvSiWXPG4uNTo1+h7ThkBrIyqCr3dU75Boa0wjGAsMoCtNHikBbXnVWe2t\\\\\\\/y6RE+g1DpLbXKZE5HwugpTnxezB7b8gJwrWaLZEOq0iV6+\\\\\\\/hokgrEGhuSuYHWRsW2ZABCQ5\\\\\\\/Nd1H5MyZ\\\\\\\/FT0WUpshEFw4\\\\\\\/wYC7Oee9vADqVTggy0RUoqBM+jvwy3dc9lEKV1LHlPjP\\\\\\\/ya6sca6iuzbXoc\\\\\\\/u7PqGrVhFqC9mg5FtPY75ZR88VeTj+tv9cLwDUWtLPoTRf+zBlV1tBWp4tguPR7jvtAJ3AYVDjxl5xHy3Nnhyq\\\\\\\/gvaVlIbPfbP\\\\\\\/qyORkQbMk888jqqdk7rm80vT2496oY3OSNUPNyUeoHZCS0v\\\\\\\/mCstWWpxlsmLm2x4T\\\\\\\/vOKMGUcmC2+\\\\\\\/yPrneU4SzczlDHe\\\\\\\/cIz6kCVA\\\\\\\/keeIh77pH1plorKSQdR8qoP58xr3jNx5AEI3prKyHzdr4CskdkIDOk4uQDsy5GM4JryTGzG7ykA3MJS6xUMajwPgt+KA2URYnh+RnT2d6P3lqeVEaEAA0Z6n3mLuGkoh9Z\\\\\\\/yHmKruscmSBbXaXmHsgga0m2D3vWppDFED4coBsO8YHMXVGESyC7PT17HHdSJLQBdDM++A9YhftOpXik24df7\\\\\\\/2rT4BIfu9hwXPpqmXpauN8SXFCEnqcHK8BriwkdeyCkw31rkHTfy7jA8CvQxuLbPEFg+nWOGXNRObJZ2wgnsnZGI30XsuAYoXzLYWp+CzY2G48BcrPKqXG97vq8+1ilZcFe\\\\\\\/ZWVUdsfXBlU+2cHjRmJIop1rxWcF0qlkJxgq+x47ltmzUD8eUQLTibxWJTHJqUglBxEioLfWXR0P8H0oI6Y528ItyflZh4kX8TLNHfZ7WOo7pXSPkh+4KYUoU2N+DidTEdMxOptugKyofKDS7\\\\\\\/5+D5jRDitw0MM1jEDVYuFATkY6g3TPWwJgvlMD2aHQsIS3WoMinBUtJ\\\\\\\/p4SQMWnSO3JCulHz\\\\\\\/cE3nvmpa+Fj5dnw\\\\\\\/4CHQT6Ckm1oPqIbRF8t+LATF4UHpGUKwphddeZC\\\\\\\/AlVXhuAw\\\\\\\/trU8AxyBUReg+a61+ckGdj6Jt3nADWv05pthUhMWx+igkDZto812hoxeNWUFd9sJJz1iASwLjNwBvjloU\\\\\\\/EI4w0V+GzP2EHPGOi9EKxPdADOVll+FrWqi7HyjzPhYTPpInoERFoPeImRr1cJxteN8kAjeXcGPpW5frmVA6Nz+590WzG7MRvLX\\\\\\\/qJmNijw35dR96f+CHqHdFpqQyw4pnLfj7hsjZFLNlypanvvQWskmaog2hGscE\\\\\\\/lckY+PCCeYdhKLarvMnheF6JqoR7\\\\\\\/WvugSqkxW21acCuuit+udjkK8FoUv1ggwF1wvBww8MtzOb6tRqOmf6mSl6975OtOOmQFq0ZEpqrXRcqzvzpctyhfoDeoJ8s8gWe2+KJ57ohKPLE9+yEGsiF\\\\\\\/a82mqbJEJHqZu7598Wg69vXABphgC3K\\\\\\\/TpZsgEyq+4JViRDTf\\\\\\\/4z7xMWgJ9162uxIFLucK9CPE4EZyzQGjzAzKYfMJIQWxNBgnbqCQwmHXkmzse05Af\\\\\\\/zV60GK2EOmYQ4q56nYnq9\\\\\\\/XN7+Y86+GLesUbzaVo5xsWidzc\\\\\\\/wWeRq\\\\\\\/fLM1RztCrWva58PzVjnjawonP\\\\\\\/g7KB2tj7gvcbyCSrL\\\\\\\/328QwsQWEok2kEz\\\\\\\/ZV32jGCX2V\\\\\\\/lDkpsVdFfGOMIsrzoydtfrqe82XQfVOskT9qoJwYmFfxRFBj\\\\\\\/qUI45yT+F7mNHvx1aroPluHb2wxC2q7z4tzU6vHiaxPD3H+1K+bioxgljvQrxYmL94+yce6N25DkhVZvJZOi7lg0\\\\\\\/GbjF8id5oUcpf2kiuVRlHSDv4uDzMvZug9Z1SOLfb+ZmDUZJjjHx1TBfa1KXX+2YqoPc\\\\\\\/Obt2U8FN\\\\\\\/8IyEXgUofA34MjfNazBKzHH++j0hJabMe3DaIkqjOg\\\\\\\/oi4LhDYVFaoD6JnBdMVmknP8VNRPMxFRhhvgDHuZLaD+TGTmEkX4v7wZ5MCGdB6bEov0B75OdAvqWgw3YcyBdusV0I8PBPp5Gb1Wwc\\\\\\\/GzG8CM0uDjVKxNFU6JWoh2PtgwbY2nS+AE7aWmT9TlpmymWoamKaAUX8Pw9p93TwcXEaVT6sqfIBy128F09xJxHemRQ2JJbvBIjsS7Gb9LzW3D8j2IjO351GEoYnxcMKxGrmoFcnPa9qz\\\\\\\/xG83WgzvaLSUXLdIjYUnBx1ZvgJkeJu6GRYxKf+Pzhmzr9fQ8gVvCeUkqlbisb7BL9QsdviHfUa+1msv5fKJPVwVDpPDiUItJtC+LHfE9hflrrojiblVDkGFW0lhrJj1SUvK7vru0f1dJSHvfme+kcTxcnyjQbfeJ322xMUiNXk3sNFAdhVnprqFVjJKBRMutevMFcNJPTK5zzSXK6aXLdk+qm1DSN8XlQdIKkCQhINhovYY52PxmThYHCPK8otO+UZFHT4QZYfHi00+cMXH+KP0lDajNMQpJPzO9VgvqGkujQVDiO4G\\\\\\\/F0KernVru2OfNxuP107i7uGNihVzT6jBT3ZLuWUejPBGQBjJXqhA\\\\\\\/eW0G4bhTgpv5R2q8tkgYsWSOskNQpcPpI8KND9AeGXGaaAGaDz3L7kdE3k6BSEAVa751uIU3e1zA6LIamuV9Qojhi3lx0wPE9S\\\\\\\/JySpbmU0SPMyMLXkMRXwhg9rBCrNRFE4JPobVF1zesbeJ\\\\\\\/1CtrytosxOozfR9dVcFheKmyP9ddfALs7S9MUdQyXlBiha\\\\\\\/NS1SRpPZkEJ22Fus8dLGQv\\\\\\\/\\\\\\\/ybcb\\\\\\\/kDsCseQ5JaI+4EOeOzFRyIvKG2oZFgv\\\\\\\/rHchC9kdSm6qt2W+vESvVAOCEC8VLpyasduSufKMy3wzQRJnaD2ybyk+FPOhM7PC64VI1vxabeaYBPI8XXT4ECRahUBH5XTEZVbCyPVxA79Kf01CrySD80STkMDq9h6IiaxckeBtzXYso0smmTiu27TVedjXA6Rcmu8gL2hgHCBt5keckzmSK2rbiI2Ab2\\\\\\\/LVDcn+WnOWq53QmlhSwXDGILHmwnRRuXkwL48agviMaZvGH18jpk8MqAdCeoMvtAmEBxvvE0o0SlK26iKlUXDAk9X9\\\\\\\/1CQjA4IWBOly79UoMscMwjBkdkuQUwHg02QENRRxcTlnA+3yO\\\\\\\/hZ4dJAmPVM1914aWCMdhkbsZdP\\\\\\\/vKYAqWW9st2luS6bi4XP2VYTZIRZEGyFDprHtfM4fVhnH89urzJUHiVGv0OiQ52\\\\\\\/VJk\\\\\\\/Nkr8CnGT7oDSrZ24UeqVCKKUx+lye9MGmkLROX\\\\\\\/4825m9c6DUwjYz86939iYVlHJB2NtZeXPD2oYILskrnYyZp6dXXS7tnnD0HMVmXhudjIjkRQu4iIGbTa+VFmes5odnihf8yxqOjx1RHgP0391XGe+YTXubVHp6ot6yL\\\\\\\/cuj07bogOMiO3G7\\\\\\\/fXqgH27MUWCkI29mQtPXqhE\\\\\\\/YlAd4Sc1C+IrtR88OQhfME6H0p0hG41hlk91RlMebfDWpCc4lc9KXmZDv0D7s4E+PIx4LnODgnLVHL3EOSSShZ7Keeczldl2ojPCDgwUEGf2EpGCq6j0mNE2DxiwlXmfSbBIWZbbJbaZI\\\\\\\/P3DDoHMjJuaMY0I796EoJIQRfnicnsEyptydWs1Csr+idXdNKS2aD\\\\\\\/d3CUPQJ+Y6hAwjCReg\\\\\\\/M7qsHNFhDWaCFCY05SkUHSFt7EBuG0Xwre+R+o+xe9XCCR2IeLHNP1O4i8t4atu2kbiQmA27L\\\\\\\/gWMXXXF2ttk3CGSuaTVBmPaE6K6xnmn+KQIQ8VJ3\\\\\\\/HcKn7iVAMnALBxyxQE8dHvwWzVvmGGtKqb\\\\\\\/fJzF5u9VdWOOWGT9gFJUQi0bTV0oLVYcyP0LUu2LM4rb0jbmlUgMpsaquOKmN03DmVvtDa0F+iD\\\\\\\/+DACcaanmzKheLUR0YhRrlo9BLbRKyh\\\\\\\/bBGJnlsZiOAedWkH0fm71f5y+5s\\\\\\\/jTuuL6tDMOXhlvXrLpve6XYMLiKRtnV+tCsXaKVegfMoYege\\\\\\\/ab9\\\\\\\/L6Yt1jAPGFpB\\\\\\\/anOrD2WwZhWIIKixKYWo4MUuyQnqZ3rWvdY48PEGqLM2LULa3Jxp\\\\\\\/oIFPSlYByYe81cEH7UOzK\\\\\\\/hLFpUHvFaq0wrxvB0DOq8\\\\\\\/y5hXHe\\\\\\\/GLut3EUEy0gNl++LXrRiglDQkhooAFnzNXudkXqmiS4dtvAZ2BuCuTlx9ovauxAJk8N8FThnRH8rNrSOR8sroGYm2P1vLixWldnZTtoqG43H+PureDB8AkG7QN5xuuj319Qg6i1Bua8KiduMN48RJ8vm10jE\\\\\\\/doqATDzkfnecktE9FTAmbWX6yC2+qvOCpD6raX1O9nIQ01Bu5jjLlYIYuo40amHjX\\\\\\\/GssfWGNyId3tzgPR16cNse3mTZI4Zvx\\\\\\\/aIKsAJkiCc8iSOyeFSou7fruM8g2sToZjpxIoo+CucvQcWxOyQHDh2sZ6bHziYFplEXDP3mmxjEynFBdXbpSVUMvTiJ7H17TmSibwDcvVEmItEMqLki4BfLMg1gxnjYwfFcYAcN0muHURlnmQP3RZVjUx1IReYAIl89iDR0hY5Shyge9FsMg\\\\\\\/KayJ+86jbYeCXxwkU1UY1ToTy4AVd2\\\\\\\/yjKXWUPOPM+RqB6ltTtX2mssvfnMjSPZnkVqph2BzQHYNINWdv\\\\\\\/F5P4KqxrZrUkqq5zFSno9mDpQorZYNd+EQOJlyRLTwX7XqeJ1gh\\\\\\\/y+iUzaKshtBeD\\\\\\\/A3dBtocYAMlb0MwGNp0P8VkHdtqXJVURQcU92OIlE5oV7xl56rk4kQt3Chbrr6HB9JrKGMr0Lvc4BoVR9hnnCi19zxihSMSllVeKJnRosKW8VOmFDDHQUu+dG0fWmwMFjIEzd8ueATnZLwel8ATHUuZ+RGmZtvI92My4gxxvOvfeueN6h1yV1RNwORpnTXHBAZgR6lMWLwUsVAD0D4\\\\\\\/r1KsfJ8ML2QU64zbzYeqmkLOh0cFBnNWBu6til94I3ui4K\\\\\\\/1Eq+OIBHn20kQhgbgvJBIn8sAfTXdi1K8UJebDu\\\\\\\/X3i4KIgdGYDUu+EFw3In\\\\\\\/SAGu78W61\\\\\\\/q8B3E0R38qEOtLIFIXWTeQiDC5r\\\\\\\/f1rqN8igXJaJc\\\\\\\/xQPcT9nW97\\\\\\\/1HpDfSqkG7jSKoWJLzRsgFHnmjJakN7B1naeUeShv15JNBUXl\\\\\\\/Dpm66AT+vbLLPDbuC4BppCjtYwqRs9Kx8vC4B8mOOIj5bYEFbCVxNnUEz6Fn5WqRcq8uRaISHA+GAKoX8kwPVdBUnr7w3jUH5cGiM4r0SZMIT+ju5P6v6auA6OZHRZkLL85Y4cD\\\\\\\/vio0PjUwJrsSKHuhCgH5l3lakaJ2EmnosOQgfYzzet36JchPYJcs3xlsjJRaFhx6l7pxlbXI7fSN2xB3DPzxgqdvOVrGgxpjmteP1omi2P1VuUi3u5Np+5nscLD2AmC+hhlHDI+p7nflojtWcGnKXm+zhu8IQCxuD+wjEkquQCgyvqPcBIPO206gJJBIgWykGwAYI4MzUvzZRc\\\\\\\/9owtwog\\\\\\\/LodSwqAs7cctjSiO0W0d41k1EpfFBdWQG5hHDcUXapP8MHQBgRJDp9dtO0b5uzszBMoYqfU8IHORWCNRCtE25OVBHoKnZl6cKUN\\\\\\\/OhoPR7JmBAugiHNBrXo7GRfCNuJPx8\\\\\\\/7s60B5kd1tMm4uUCWyTXBMCfKb+VqywHtlqa0CINNWDEtd\\\\\\\/y97FGHK+yg0MGE+hiJWt+fjFJwRI4ALCWmIUAvJzZ2IWHRGh\\\\\\\/fyZHCQ6TLY9pycPimV1H443Y36PMxR0ib6wgdoW+qKHPJcoEVEhtBGFogsEEf6eiTnzcAvrLi3brCAX4EPiGxhbBT4eiKidEeZw0fUSyXd5Tm83FRE8\\\\\\\/nG2oV7xvICnXJ2ZR5DBjJemOKNfokXNeVJ3kD9AKbVXvPa1LfgPHIvfRXIi6nMXo9hIXi2FI7NPmUdGtgs4UqB5Y2h1K\\\\\\\/r09w2HR8K4nymktnDv3foiy7DCyhAzULxTb66KPeJIAVgWDMPtazApA2opOsyUOes4\\\\\\\/XPlLiKlTXgEfIAszh8RQzJ5Bps5SliY\\\\\\\/Aycb6STMJNEeTk0HeWwmIa+S+cXcNS7xTyTUxFyM08eCB1ySF6PaHZv9wdNdruAEoXDiCaP0k7SRmOno4Hdup8\\\\\\\/6yRT+bNuKWnbbVR\\\\\\\/8jgmGAYifOgIRPZ\\\\\\\/7KS8AdY7\\\\\\\/R\\\\\\\/Zgv14pxvUZqib3gNDKG4E4kStoQ4t0npHiqjVrIEhumsWHr+6jWkXPe\\\\\\\/GQjYuvkRW5R+41Uq6l7C9PDXSme7JzPUZpYE2FbDZ6UuJfuPWTcXZG9fOMUfgzSX3g7oisPHm+y+ZUOIHGyINt2eMjjWd3TQsYR\\\\\\\/RjpH6X4cM\\\\\\\/Grjp37KmO2LXnj5eIU201TqWiq3ykta9imeVox+67VHLemCLegV8f66T+3qj+PvswAQ4w9V60EgqWw4njgoGp9dzMqRVB9MxhM4Tqfu+5iYic3pbl5uBcYGEKiVwjwnqCtrOWWckKAJ8gvzCv3iXsF+mhMf4xsok\\\\\\\/f7Rp4fgz10gKtyg5+HI+T3Lj1n9pFUE7Y2HqywW6Ddyoa9lNAHZHNj5CyAfKbND4BSwghdV8sNlvYKct5iNXiIDLJcfbHakYT1hNBIsZ1ADXxr7ASp+KwjjaMYdV\\\\\\\/RY2iI7rr9Q4zjtBT7CuIJfPRjNU8uIZmZnQ9Or8pImPT0lxIcVpOClpvO8uKr5wqX\\\\\\\/P2y26JJd4utUbjQ04JWlXeWhgsaQmdn\\\\\\\/qbDzjZLSGwA\\\\\\\/\\\\\\\/TLp8F9mL+fpHYqNuL+iJ9YOiMXYLAYsSt0wlz+BOBZeopdPknYZ1VeDEzMXQgSRHRb0uaWzNv9i6jrVN\\\\\\\/K5nRvgOWO7QgvaAV5vCAIvM95nMNZpyu++U0iC1Sl24BanWYDtWTp7dbKxqQhNXgnZKRy73e2Hg0uhAo8\\\\\\\/jHMGFmOlfFwbA8RVWr6MKTh\\\\\\\/vboB2PWmODuAocCUcFBRbhPMT8bzOFQcmkbdjBIi8ZaVj8EkmaeY4s0IY1hjlbF9V8gDgUa33XKXuFGzh0o8PL7rFuVl58P76LFQ99Po+lQv5H1GnewGUVNGnGVZAP9ICN\\\\\\\/7Uamm4EOUQ30Xxxal\\\\\\\/E\\\\\\\/aCp1LxjbK5AtFlh9fDiQKXAd40k2+vBP5vH69a8BpdGceiK5SeOaQN7ZvlPHYFhfDs1PNZkSxz2c+5MWJOrISHqbNQTRTnmleTAzUwimqp25Pf0ptwVfUs10IoF6KW8qMWjWQjBWI7RdZ99XXcF02pQtLaiSCW3SY6MWdQDtCkrznD6bk00w+y\\\\\\\/BJ2Q7PLlsMiA26TDMnOejQFpjUYwJAbyqXPKsW+rQcdhKc4hXI70GT+9cFW+qydiEaUShEUQwu23talRMn6fbEmJJQSRFwDx1NFApPGzqN3fQcNIaQ4AkZfnM2rGDThwSniGUSYj1DXBxyI574ZZHQz1naKaOlHXCrD0hf8x+HxNoMTbMiY4IxTwkjIRLkedzmwwcdmcRFKiOTPz1eg3TrJeEE5Gkm23weo97TWU8g+MgWUC\\\\\\\/cMN7YhzqFJKwa7x6Gi9j6+TwyTf340FvcVocfUoe7I3kNlKXZ0jG5xWK7SG9X1iE6w0qKtrtoILtHNBCLDEbD+9qx2NazuwA7j+Rq5OLsDi3vytTWWjUE2BL0b4Hl0WfJy5lmui2xe+ZITR32C9nMtoxE9M3g36YNjHfBHRyoTYYTHvp7U+eD+adrh2RtefZkirbEntEEezmVuLvn1x+Xq65v6SbjL1byyjXYzO2p8IVEosmvQkAHnhSrJ73suXkUJ8T5ww1sDYTrgdm6Pvvv0OSC3Fq8kao0ixjpdSdOPfg2lIs08M+gdaWXx4dkc6CQDE1RmxkV36gl3uOf4q62aUPOrJtD2jYL\\\\\\\/08HryOFtVHYdzadQzWsShGA0dgxJEIpY1SlIgk84iXgrUq2B+XdzgRX62QgUa+2kIlSzSBauEEYLGzNvkgFtbtAvocWaYyeW+skKrZS1DYHT2o7121wsl3dwxzHNlUf9HyTQJXn\\\\\\\/dcpt1ND4IDsEPlgh0mvadYGrirn6MnEfXF2gn8BzPXkzl1jPwIl+j3abZmmq7jhP08k\\\\\\\/6i0EvPkmeVdE8jgQh0iEQxU0ZlCG+UjxGxv37GpjrmefWN8FOdHc6nLN+KJoimdq4GqchgpR+wvK2fGy7KJLwoXyIIq2pGj9jbmdoyVW7+WT4L3GbM6Sta1VD6pg5FSbXYKwpP4b2qgitCNg9h8WnyR9V8VJIwoIzeQStRqNVo5+Z0ZEg137AkuuCsh8bmQgfijXPqxnCGSsFloTXtVuo+x5oJ+mV+35Be7qP9vQGLY3WrPEBh1mtjq+1HsXV6tXC3uHO3ki2qfoc9kFUODPJCDZrogQVTvU6w+Vi87Oq0iRu+qf6zFr3Nf0Bj\\\\\\\/BM\\\\\\\/5mR6scRWMhL+ZZ6Ayj5V+ZvGtqDB14TPvOFq3OY1OTttIvBwPaQtNivy1Al3ceDSsMYcMzBcAWSWLZJOKB0CDotjMsK2M8H3+aBd3R23Xim0cF+mI09HADfYQUsHmAzLmWY+Xx22uB6eBubs00JUhb2SEzYcpH33LEl\\\\\\\/CD13v\\\\\\\/TX8XTQqi18U3VdJVU2ZPNRu9a5ThgjKTiTKTkfq4knHaKaSml0a5I5pmSYi8ZLaG5uRCx3ds9LS112piXe9QaxRxrgVL5uLgZ27kyjnO2Dpw\\\\\\\/0WYxvz64K5BdGbtz4Nfdgbg9kmJDRIE4vc4Ho3L1sboS4K5Cc1Q1ce7oO1\\\\\\\/hjAihAhsrXxudipcEu4+4G\\\\\\\/coqXoLKIfWS\\\\\\\/ScTCFcPfCdvKCRoKdVTHHXRjtudw6UbD7iYTBzP2yaeLbqI33nyjw4jBlA0ODth2yJVJ\\\\\\\/YThBRsvoszcmvL7y9uqw9jrO9\\\\\\\/rCkCFuGsvIJbgLGsUsbQa2Z+vR2a0ZPB89ErGwFKbxrfZTXqqUQt6opqoVuHTiMr52tJ0kevveawAOJsczttJjv8M4t0zN2n2PQeT3paSIeeIatMK9VAPd3SD01xcOLEatVkMKlYxftZ9YtmkprrEjPRCmtlmCi0pAGp3OiC7KIRBGdymW2sUspRaXbEtcFPpITelyBzrDEhhS1lUSIZHjF+rm0QsBkE9SlrUSXQk\\\\\\\/TbfQZnSfwnBZ3gvqcKYZWesDvKla9cGXxrOy01H3KVkEGcYEEfoGc\\\\\\\/YsJelaYpz6NHOA1WXzDGELzZ8tV6hm8VZbPmVKfBWVQBSMYM1B+HXbb16+8RArDPZt40elmlIIaKqESlLugUNE2A54mYC0wBtyGHD0Eh0uJu7Z\\\\\\\/nZhFAhECNXuG9XKv9FA4Z5mbniEmY6lZSx\\\\\\\/W6yY1Mowno00u9w84B0i7eaVCnc5+FpsY4p0wLBA0dPfIcydq+YU4WLf9EpWVVqD3A+ZkyDSlerwkvztkn3cZs1W4F7HMFksVJIlLw\\\\\\\/cmSw5nWI\\\\\\\/S306N6+vNOsJ68vF66djtXjyTJlBuF57iIPf1WHHC652gMP3hFphavxfiYJc32kS8vR6VpI2XJ8hfkAFon1IZ5v896vEIc7YlMBM38Uj6EJqggY5WEEIVYxEqYHFa\\\\\\\/PqHlG0vZOfMQm2PA+9w0QbSpehhSRQHfuIeQMevepgxwBSqneiGLzZ4OJZN6F\\\\\\\/I4B+NSi2DCc3KmELOKOvou2Wx4yd0L6iC\\\\\\\/gbj21eXYx75ZYT6CEu+jWR1aSSDdAqgcoShsgQ72UARMcNLBr928TsOkzT\\\\\\\/46lW\\\\\\\/aS8INBLhwLFEQ1ix5mlCM6MNUVECvbYM8TWuZZ+FeOMmA6gnruJw+UjeB1Wfu46DM4TS43jQX8YsvVgCiMrAOgAlmUlES7L+URi1BOtGQbaKH44SDOWNj+\\\\\\\/nSnDUr9YKJ9IjKI2SdVsFFq5\\\\\\\/80IB5\\\\\\\/1pUg981p\\\\\\\/3x9b8lqJ\\\\\\\/gv7OT3JHH8amPQDbk9f9g3aprG49mb8GFwmz\\\\\\\/iinrc3oleKQi5XGBTGU+hMmW0v++9G2jYsDV2\\\\\\\/8loweYbvfyCW0Qy7dQtXMR+3hD9i56tQJnzk7Ozpc0nKPrLRv2Wje8f4a\\\\\\\/j4TfdPwLDtUDVggL+daoo\\\\\\\/lvSbrjv9J0BW9PtUHVAbMwfNn5M\\\\\\\/pfy+4kfco8Oq20wqPOKR2E++rqxhwbA2a7kX7nReYlX9StsNYtqY7n1dgcx8wZdSUOweKiw0oZhr4VyxpGIxvLGa5JpH6CRit7D6GFwzzVlNN\\\\\\\/fMGkCOMlMi81wTe2ixm6YLSHhAe3nPdmwqDpCfPt3tEXRBbmLvWKuWK1hFB0XomwAo7lgp4NqOgN6qnfweOrivmLqJ1MNlJJPY\\\\\\\/b311bcUwQyaZPt7Gn0nIFPws3Q2PgFKcS3m\\\\\\\/BZN7jHhtIf1ZlWEL3TjxtqDFyMl8hYTksFfGXAu7dAaEXJ4kZjKW3HICd\\\\\\\/PeEpUkZo6VZpe9xbmgTTOOEtIoB8KF5WxHcfAjim6b3a0dBg8wOmqByRSjkr0XJJkL7mtQQbeCWJUw+5FBooE32fWN72y1+qK4ceRi0g8wA+vWYSG\\\\\\\/PBoDg\\\\\\\/fSkfMs87qUeWG6TXtzVZS6ti0N6VYfueVMRVhp\\\\\\\/p7XoTxe5CEkAJ5mJN5GJRR\\\\\\\/vMjvr4FO3TCPYGqvPfCbSPehL3MHQkL\\\\\\\/PGA4jA8HqjwpUFaWIbt7zrAwHyWrQY+35sf5zJu5QC1qz4lJrosj1GcPXaJjX2rU0nBwpVom9pjd6HkGJgW3oabIHzZ4eWqV2sNbR99HOYF2tw3IYTc4FCvYD42whOh798sbCw5enjmiEy4ST6nL0VZhOmyJF3aWA8pU6sKDswReQkhM1O4mhsPOHen\\\\\\\/nQ97rJbFJvVaDxqda9S4gMBENkIuGSWLscEgLxunNNcT6j7jxIGagc9052SHOXspXDIo4twv2wt2cdltId1rqBOFcaiiCQ6hO8tMHtTJHH9WGzgQwpz7jfmIt+0dEaPvrpbyo38Nmen2k8dx4BB94KQ31+nGP\\\\\\\/Y4Xsc0Xmi7ueIlpwijw2mQHjh54k+bVeDOtukkJ4KGzaWxKAQMtj7ogwIKh\\\\\\\/nTHlH5TIgcNLLTTYAUDGRY4ojjXnq6fgWFFHBAu3DLc1T6LfQJuMiSnqQ\\\\\\\/RB+h6V2LeitKOfl11q7axEZk+Uz+laS98OttaQlJC+qFn7OLSOvl32uiOwUq\\\\\\\/6dKUPN\\\\\\\/hyBWgxbwJUgqvAZatt9MEhTB\\\\\\\/AP3rE2UQAydVo+sem9b6waTYVPDbWFDhYVpTKoP0oyfIWit9Z3XgeYyspw6MkhE8ELnggNqg+JR5iuQ+jMT12z9x5BqzPt1p+gZGqVvkN87i3d2UDzxukPbnc0LRqLHG\\\\\\\/etl7+l75g6RnmtL\\\\\\\/2I589ZhQSyXibm9HflHKdyCzsyNoMhgQabUfdv988yc5bD2NZ\\\\\\\/DH8ypU2f0QaaQMayxRBHr9YNPx3paq0Hz6kXmF83Gn1vfAjtjW\\\\\\\/KqrVWv6zXaa1AP\\\\\\\/+diireOWhNrKZqSO\\\\\\\/Dal\\\\\\\/yvRztJMqQH6fIseitQmnD1xPxx2m7Huk8mk8VGyrhL2yEGsjgLCCB6vqfdNiGjnNzgfjM8AiIP\\\\\\\/7FNMm2m0Q9k38I77QOvm5WiwhJ9GarOtFE\\\\\\\/rmc4B8Oqv0PApvIsfX1MIV6jz01rtVeWG3an5DPXzSQj1RMR8fZectxEZoGKVdPaT3mhOoa1Bmva0ALk8YdQNya9TLANje8pCBKqrJLeCw4ycT\\\\\\\/NEy7tLciqmWQyQUSB1h44s+mhAsNN6zG6oXQ5U\\\\\\\/Bl0eri0O3tBg8F7RzPca\\\\\\\/LYsRJBF2Yix+Ybg9v3A2oDkrhvR5fqlfXHlNpLUxTOLuztFXnfF0ePPXN0M9GdNnjtdclqu\\\\\\\/DgDsMLtGhJktbnz0LBbe0ntpcS6CMC0BuzmqQxoLwoBYKWgUdf8zxEg\\\\\\\/\\\\\\\/pOXaICkcshYSeLr1bRRmnh8YoPj\\\\\\\/kZHIHKNoEzE0S0rzhcr5Fs1EIhuMJtyADeGNXu8gAPo+3g0FS0u4ANey2RIAs9JkEim3FcJ35EahJvU7t+1Q54kiRVc0OGR5AQFnmQRUgGKmCLehfq1AhklQ8Rjk6RShqgMTchenKHnNmvtVCjWbF88HiParpNashOwxL3a9TGp5liklYvATffTg8PnmEFU28hzerJ+tF7GogcyMUhYY64QhX6+vZCfP8hQG\\\\\\\/+4ydkjRDuV6X46g35PZ3hyJsjhEOIELjbYKkt4JKFq2VBY54ILouMEv0ystBD1uqOi3j5aK1+aJU6YxUQSaVTeEwN6U22QqYH\\\\\\\/NgXYbP\\\\\\\/kHNR61UgJNNVHCwp1NHWVqw1ZtEgHJmNz2Tn+2j2qZlntdnhfTXeNMbQR+Uk5yCrKDeVji5z\\\\\\\/1RriwD7zub8XawV4ib9kqzPwevY6x3kTVXkByh7kFd3SrvvED5qbQKTnJ4pK2jQEAnENJxfv+LWTR3I3vIOrivasB4W7uXPrjzVw24tllYkn3PXmSAADcAcQzuy1Qf90pIJ3Dm46TsFQRru9vTUtDJRFMr2AQIuGNoJQDrI\\\\\\\/RnXnHYNxSaSKLg1GmERbYYP9+ZgOIeGwEkOR6CBvLONszzhdCM0ldWsiXgOTnQORafMhHdVexzq9v0EzW79VE+b0UqgdLicXrIAH3scncG032w2xUpU7rg2eAV\\\\\\\/AWFocAOQ3\\\\\\\/lvYQGXBdqS5GNnmR\\\\\\\/aIV3Rvhdrpsq7vp6o\\\\\\\/yeGBRWxa7PFkOFVwKd4x+9KdVlBrQplHQaPUfPFy41HKFfQT2d4LhnGBSjXDM3vmX\\\\\\\/2cOGQJMMr1U2JjE+sVjNer7M5AC4byPq1Qfa\\\\\\\/zVcieramAE6sIbo\\\\\\\/wVmdTLWmXMBPRgzvEDa7malgeI+wljWyANT8t9Q72QV1s6cLGc59xYvsoccWhyWmc\\\\\\\/aPBHR255pVQHPcYoKxDjB1xWJ1FChxqQwk4ZoJqLXvE3tA+doyteMCET\\\\\\\/qPq7V87oFm\\\\\\\/Rg+0HHLit97go04a9v0OiDOCUIAOcKBg0yPn6+nug85m6lacOWvko2ZHAMfmG5+ECT4HVzeVrNKnW9AVt57lKMoH0nRSwkCzpJrSWJ8q1gOaG8gh\\\\\\\/R7Mv1r2LjezHIz64nhErDHbPnfduSy6\\\\\\\/bK9XxNoIcsjWMCRhm1Vw2kVQnX1Eauft98dspyJxkeCAZPfcREJufDb1I+GjL3tBokkKGeKgWNn1TdlWdjIrb8OGPczNbTl8SGSAp+n+X07yUBDoWWYUuoNFyLxHkv7OIphkImHa1hUVKxDMv29nxDxNNZfk9DemuOpJNUkrP5Z6ei1NASNUrLe84zQyKvmVTAH94ArxioCW3A78IN3q1OmFgOteOGvTPPg857rRgs25uStMULkkR3lqxPJsYFWE9qw\\\\\\\/3xuBqXPfIFuFagwY5cUTKIFu1KEdPbAHg6EDL9+tKd1KSQdowjIEZXqziAuKN05XkPb37UkG+ZG\\\\\\\/xhevOGwlYqe\\\\\\\/KywqIgWQ8kD5l3ipLAWT2OqnkYyvICitOfuGRg8BJwYlc4PI61PWRAv7slFHII\\\\\\\/ALIA1TyA2+fVbChd\\\\\\\/TIkivDqabJuR5NRfc9Y8eQ6e95Akg59PS9LqH0ubJe8fc4v0E2LfMvP9zNEumTPQcGGiXEfUz07tCdFqWVEtCiFxzGzTXCApnJADSO08N5hoqo+KuNJ8kXQi86UIHPrwWsdZq+VLWhUSr3plvzPaQVL\\\\\\\/KBdC4qkfHJL3q2OEGbbXnU9zHQ7gcBTr69uiiVzuLo3647vuMMXpwiSfMrLWhyGcHU\\\\\\\/DxjW+R4t3pkDxTBu3IEKXJZTs72OzdTqxIwTOQVFKwzt6O1kZJRDR78+ayGZHfMbISZ35bpaBeJtSaWzGgoKeurANC01klngQ2FgQq6HM3k\\\\\\\/uyH+wX5O9DARA1Xfc52x2VrjNkITgn3YBCJvgduEkMYXWqkc+06WAvFJA5eV6v6cUS3fHx+TbUvwXfUW\\\\\\\/YIKxobmjdZ0ghi1K03t5KlKlpCXK1Ue+dN6FKNiy4Jc\\\\\\\/SQur+xZxLBIT5NTlM+BwT2zqFjWph8TWrnN5Zi6x6+W9Dx\\\\\\\/UrHtrli1TdhVCmQu3DDiA3NZQtgTKYB8sfbaWTsv8DPd71Xp1S+64fPcA5O0JwFwCmeJqIHjRcFggbe2x2omxPi096aIG3r+b3ACbW7X+qlq6kq3TsllkEZVKr1+upEa+rOI7BUmMtJvx8ztYig5QlSqDa1Uxi\\\\\\\/lIEbw1YHpIzNz9+KwxO14mVlVG6wQ5UWRO8XAL0bwdCu5PDh9ibEMB1T5hj2ToET7+dNAHQ27Hr3tANzACWd24JzQoScSL6F7sKeDkCVIo6BH\\\\\\\/0cBc\\\\\\\/m\\\\\\\/OjQ1P+CKbypMLDB3iv8UkaoD78gYU8jrDOAWGmQ6Zr0bdiDNynplNo9lpBJNUpIeE8kf5iWhbVrgmkkzD0JShAwMB2k72EIKkZ8C2dx6meehcOXYaBRP2UdmeTWEuSR8XTs8Sm6yNbL\\\\\\\/QSVms8XG2Wgi2m\\\\\\\/PNz+D4FDm\\\\\\\/BkQMH\\\\\\\/9Ri2\\\\\\\/DlpqOOLt6fjvIZ1uEnM90ww+lqq3lr\\\\\\\/tSx1qvzKqmBXVFQM0RZJiNykb+uzsyRouzRIcLSO7iqLZxgzUvEvqC1E\\\\\\\/2HUqhxyh4\\\\\\\/yE42+bTlZ5EyNgE8hYUCN+9KMNlvi1X6R71nJRHkXxm1I6gM+LUbDtJbgmysM8lOe0MD+HAcGzafqUTwc+YWArraNUrYDTKG\\\\\\\/nuMtTfQA0VKXHe02wDXZ\\\\\\\/djk8T+qqWLESItqjT\\\\\\\/gdyDzsennX6b5ALSqORbqrl3UABQRe2ACA7pT1E+Uq9XIIVfHc7roYBdz7As4GN7rm0k7hs6b6pmUIr4yDLMlCiJ851bGK5zSC0+iyrHE15NdXYgFTCbTsa45PVt4PXcYwQLn70HkAHZqjGcTIWUs5oBu44yWcRu8w91hcRbViTV\\\\\\\/AR6sctaJ9vsWHsHEd240tUoJUGHlVF7ywtGD4nP9+qLl+A3y3Lan6yH0flAqPAyE3uFryXRB1GSEYoNo6uylnU94YTbFmLlNOvIHokJkn9DQF+3nmehx0z7CX+BnIk7Cd7ywFi1MgssnRrJYpRiajeU0pGPEtlxsNfeA8kXEYEa29eB1Hkh5IBGuaNCHltHyqVD7McXNTXglh2\\\\\\\/rkSEp2gD2YqfZfKGnpHS3t2pKCHrCfSepormQN7xy3AqhwCg\\\\\\\/9zsNurRuXh2\\\\\\\/WxJ3H+Wo668QOhQk9Ugm6AIV+DpLrwdhyxdaC9Yw1E43lxKzFoiTLxpjcJa0cp0orN5wQVCFaosIjuxg+2CClCOLfSqz8+AXKf+6i4qNz2kSwh8vxm5lzwjEovQAi5Gvbge\\\\\\\/uPFXLbw5CiuiHTP09QfsgjH5wy+KTaC7RWFsjE4PWI7zyqMNT4L9wZfF\\\\\\\/PViprkZwU6NYXO3hipoyM+N3D81o1jgI2XVsXpkzlWcwf41DXEAAk\\\\\\\/gNA3hvBNQQFeHG8r\\\\\\\/nHJHaXmxhD3hqsw0DDayD\\\\\\\/Xw24e98cIrVadVSp5IeqVenXa16wufJEcB82kkyf40hVEWXPqFzwW9WEDi24hrWCpe+FiF6uVIhhy81DppaGFiOb4id0xBnw3SUh9+7upRLg5u0RbT6psbciDEzWQd7iTWZHmoyFi0U1EEyNTNix6C7fCP015HMV2FTGZ0IxirTcb6p1gZBo1Tk9ZYsjqMR9Lu1rAgwloSNIgVD4sxAtuMedKegp3ZBdOWY9B56CMY8X2ZNySdETfKBs49P8JdByL6JjLoR7zIaOtX5o+qvB7IJJE3vzvw3Go9\\\\\\\/+RWbNlqGVp1dJBJv+m5+eFBGl9mpK2dbgGpP20lPSX3uXERsixavyoVui9Z\\\\\\\/DE+pR1Scrght7Ulm\\\\\\\/G6QHQ8O0YQWOCIaOAcdX\\\\\\\/TCNcV8gF6aWz8c\\\\\\\/ywYlzCw+f8YFHXmP1KHc9406nlzfkjv5TbcOjDTBEacn5XB\\\\\\\/2Ask8fZaUDuTxwjH341qHCGn1DM7V604dkz2uY86sdEia7VmUG49FKYJocmPLOgKH7G5\\\\\\\/w2JLUIvW++sl7\\\\\\\/rIJeqTnWOVf2QH6BAImQBKFGazCHMSy5nd7TXFH7I9azSuVOuhNY\\\\\\\/9ryqcsk0MK2dj68aYeXIDf5KU\\\\\\\/qDnNxmYvpZ499RLOU5c6lSSmE1XtIgRwnTED9dpoZStz9DgqU7FqJbimMFfuOL8HMZkGGeA8sFMSLhEPhKJgDvKV8KjTQoMccdv91yTOft5FTeO0z47+MV1zRd6RuBC2LKC5NiJdzaTHtZS59tJqpYtVie4g4JXJXI6Bw95neM6jO9UwjJyxVUCRXuYN4WqyCz4pxuPq5xdxBdaP8IGPdKf8hwT\\\\\\\/jxflK\\\\\\\/YG+CP3FN0nJffdFxiesrsOUz0EQpKfZqFGydObEqPQkfGf8gu155QhTsxPNcRflH6HvWdg3ftglX3W7TAvEIJ7cj1mcmyNXpQaBiox60Zpg4+DCrEZaUpRo66bxY+rh87WxKvzKMwo2pkp\\\\\\\/Byk+ScIcdpyTHr9znDQa2ORp1dbdRAuGgBWO5yh7bEZDkZHIxJ1zRMLPlizxWbfX4p4dOJr1\\\\\\\/\\\\\\\/DsEZJOLsst+agmXBvZo5jmBSuzP5y6ihedYYDezpnX0qhpbSTbGFz4Btgrd2f9Nzq0VXM7RwKPQ1LZDBEAbbqgQDYE7R1p73anf4Gsk49w1UAMmUG1nK0LUWkIJjsI6DqvHNq8p2+bpsRDbj1WG9nLSPSTDoDx70+C2xHkGweQ0782aYg3hdi4Y8oxs7n60BDFTPWypjnNlk37\\\\\\\/Aif0UR+yA0VllMqGLaYkUDM+4zjoLKRP6IkMtQ7cJnRtBtLEcZBRPV0bLUj6E1G23h\\\\\\\/Zwj3jHE+OGldA8XjMv8goX2DF5Qm2jNZ8Sc7R0ft19Sde5nSFJDKZLVSu9HAkfmQT80\\\\\\\/kj34No535t43Pfbzef0lhmRSKr0RKNL4NYia2OCpGz6VpUb20YhyQ3HMyj284A4m9HEjyWh5VNEa\\\\\\\/edTO76uWhpRNkEgT5f0l5NjhwUsyn\\\\\\\/I2i7BI3l+FRfUfFVT80lNopD+6BVL6TB27wGah6au7juNctI8F3bnLF5GDkC83dDl1jlLfjOC04N8IHjTFjT8fKakohje2UuxMqJwI0fq5WgD+3MpC\\\\\\\/aaGTAZoWMGNLONimAbRIjhk9Jh1d1sVMIQv5roTCjSZb1STtwg\\\\\\\/N2xR2OHbZH4AHqbbEJvBSSI8XFGfsJTiP14KcHwNcWurqbqUlrndfZkfk3mTmEw\\\\\\\/tG4tWjQrVzH\\\\\\\/PA1N2lBAI9hgYQ76+\\\\\\\/6l1821hsfZCAb7JBU\\\\\\\/vOvirKeRRCbUEcNjJoDCUd5o5Yj+csol4KUqgkFMTj1CN5gfJrc00JvFBJqCEwz4EJiWgb9oghVKQbw\\\\\\\/zDPG0VScE6b5M4vF8LfdHPu7hC6xqLy7AQCMUz3IFJ\\\\\\\/++u73FhLTcciffZT0dr2aBTbcjUH9hdBPzh6X0YrLHA5x8gfgC8SpVBuEvdI3lak+rs581wsKOv9ntVCIhacsR52cgTRShBPf4ZDUyLcxUEbc3Y3ewI2gynqR\\\\\\\/f+b3d6f2pVvUkWYPy1CazWB1Ecjz7VGR1QhGDEQ7f0hNs0H2WWnQVsA8CDyEiMLYAwgrs3lpI8pg0KEqm+qU8R9mp0dI8khY5zReOxhXkARGtkcov37QaRAMQgdXQ6PcIdoe\\\\\\\/SGwnBp+EGYL2qqm37vdYjSaujyoULCSri7J9A5DwwPpko+Q0gbBEfreutsnrptF7VUKTzQLPtres2Mp9cy9Bqm96vP9mAC91WkGIIGNyJzev6Etn7XNC\\\\\\\/JmSVUnRwuHTKqlrkqao9Yh7BcskLdsZ+9v+cEFtOeQd5pU4xuiCaPgn1nk23JdhAF9kGGJU83LRo0TI3h6cKburzkCTP4WmjaaAJ4Wla5fqft5QxyZy9mRbtQPW8za1praX9Kv4i6K+98Wgou6yMei2yJEw2KDxXnpQMeeTFQkJZFpXLhDB0O+CJtRoj+w1FZyK+MvtTFioXgDLZH+uzmMGI7LaRpCQ8duMFZsBz53dtGsbeNzNwIBt5uBChrwCL\\\\\\\/VbTiLaXbbplRcemRP5Gr3dzrpJcN4qdzqlngCedtxazjEVyFMOIfqvx+yfcWpoHuYLLNd8NcIArwHxv\\\\\\\/Ck9wIfk5KBfxGV79ukbUFzxhJVTAFdoPIDMKN5XAtPDi3gZh4Chp2Hyj1Xn3f7xWvdSXm00PkL2b\\\\\\\/EM5uV2Rv4Pa28ncnfW6ZBdfJ++A3I51I7UE252MiOSYm+B5lDhXT5M\\\\\\\/sRmxCl5EQm0IfoVhQoNVpUPNJXnIdK6eUgcJIfXu1E+Ezy4ELYUyiH\\\\\\\/STahKQKZrD0RO7QBmhKh4ZpULrehz3ugDvXtUzuiamgxmr0aD50hQMFwgMYI7QBB7DuHSNXeFtwtK9PzMSbQL2ppcL5IHYfAlC00+ujD3a\\\\\\\/ykHYgXcUEdkD6KjWxfYcll3hurTL8gwbH99smwPVKgQixjzOACux8NbiH61COl2ZwWWcy19cTaXlBVgNC+TaDLZ91WBnYgKUWil0LFCadPBMgUBouMFZz2nNEs47jsxCCgX0wS373rnQ+j6cCpoX3ahTUDWNhyO0PnE2Di+cGPGJRbHjCAD5Nb7zj+sebS00\\\\\\\/uCJS\\\\\\\/rwPGfL4qUnqcrRMO3fqHB2LykT3Q7Wryca6Qkf7ew5YuRwG3vr96zQEzIJ1Yz6RkI6XKg6UC63TnH1kgAvbaf2fLHH8io9M2pqZLCBwUTSXNZB33UWtZ7e\\\\\\\/RyhZVRMuSsDPjTaOL9EZ34FXZO59VPjbXpR9SITpVuUnc5uO0m\\\\\\\/nUnYVqIgjXqi6rGIUSpzsUg0ySSILySPeRSvyBfgxshPO1NoWBGmxjv7EXPLh2dHq\\\\\\\/9sj574bCjdtSEBx8YwrdxOqEhuk80oxPzYYezlxIlprXqb1T+l2psR6x0eNVMJBVVWY1TqowQ5sOx8x3KoQxbaXA6aJyt679zdCKJ3G7\\\\\\\/QaYDPpM3nSKNEqpFA\\\\\\\/VQ3UMIKg+qozO0L8VsPpkKHkxR840RTtvXxgV3Dw\\\\\\\/\\\\\\\/fBTdpiDb+YqZrrw1egA2TBRWtpiS3v\\\\\\\/3XW7eCZiCKYimScdnM0ycxGnuhmpT0n8vz9wqJTlrSa1S9QMBRD4knUJmzjLgi4koF7yfDW1zp3c3g91v9WUlTGIiLABXFHkh7XYu4Yv15RDoTRyvDuzmsWPNn5AMw1rpvU5xWoZgDjMqbFe0rjxiuktKBInxBfQHdNd2CIBfRfK\\\\\\\/jo\\\\\\\/ys6rPHQqnoKxQOITthtQhJ2ssoSpnsXXzrcRYsXjw0lwCGGJFJUKAZ3puGBA2g1cOM5rLLg6Z\\\\\\\/\\\\\\\/05jIdHXy5\\\\\\\/n+TkLOF79+fnwhszo33GMCIZUx92fK2A\\\\\\\/8\\\\\\\/jHy0ojpbLRGM\\\\\\\/wWDAe4g4Us4IuL16KfIb+jFzMQJa5+JyBhiZQwHGPLCB0\\\\\\\/em\\\\\\\/73wCLZx0W0W5ea\\\\\\\/gi9VGlFxuB34PSuihII9gt0UEIgmswNMjhidjaijYnBud56nVAS+gAN3Dcg4vjKSxiJM3VIRekS4LgtzbUYmJpl81UjdJLDcP\\\\\\\/DE2V4P1ZvSiOcJc69VExdzP1b3bX31iHHkCu05Q4NIGRVRbFWgWCD\\\\\\\/NkHvW4kdbe89rKRP9TnCsXBMNJ37toYcqJnUAd2LHYy6jhu6KOCD+cno5Y290XES0VV1ayk6E+wVlTmRaFR50tL0XkJnc+wcOkDWhTfr63kJ+QyUHnb9LveUUTzQws8RuJqIy\\\\\\\/jpF3l\\\\\\\/zEPneJBg40Yg90209pMzG\\\\\\\/PrpEsbMYEw48XzfK+KmmiKUqKaJcB844zuEmUnOBT1DkA+PoORFH+vm675Ayb\\\\\\\/N8gG8Vl+oUKJVR3wCpQ5PyCHflvH433fzXbRwmTAQhcHC3FNqp0tR45WJ5Nnc2ZFQQ3HkFyCGBQk8iSq3eNpj\\\\\\\/6xTIzblds8OrAR92bExSL941werA\\\\\\\/PDG\\\\\\\/NlPeWGfm9fBXlw7\\\\\\\/sGR3kvS1QEX8yczapBnklYwdgPQ0uX1jtJ3WILqwR2\\\\\\\/KE8CAepZLSjJIIKZmRmdIq2ODIEMzdt\\\\\\\/tdEkwKmBj0OOfMCYWmrjp0Tv6yAclk6iAPfMntEcc8s3onNshCdElPB2h13xSYrzqPZd+bEg5im\\\\\\\/wDhL3q\\\\\\\/K5t\\\\\\\/Ivie5IxdI64UXQi8YAcNnY5n8GKf8cYVIESkZ1nn\\\\\\\/yrhU3oIllHsT2tS69NgjcoLdNm0BNHir+eTOWDLGjM1O7ytcW\\\\\\\/h9rHrKmykzC9gkue9fDYD1LGX\\\\\\\/MvHoI\\\\\\\/sdTZVx1ymdJEkD4TRaHU9hRMZ8OZayHeI0ObeVy2ajMqsc4s4l6zk8l100SMf9MTByYSAdTGNUZRVyPkB1HCXstLKNtAcrphzahi7Hpj0j88aljhMXQmoDR\\\\\\\/7PgqlkL2\\\\\\\/zoOjgQLqx7ArUcQseM\\\\\\\/eVCg3ziVaulQRxWeWqGpQ+rMa8t4+9O6D73Bf9Sy5YEH4XwWlVvJiR4UOGsF9KJIZdDclcDT5sP0ixNGAtxLPAMlqiJWXaPWjretoMZmK4IH7HHGuu0Srm3vXifIYwBhZPqFgCfBg\\\\\\\/vyjLM5d3VmgjFP9XpD5RPhtXQIIBvG9mlOQ5jx2mpXvK2e279hgXaJGCDjIIJGj8F5nbHyF529kULoojRpgpCzR1vKpxj0RVf1VH3sJ+2OK+MZh\\\\\\\/X1cRuA66UKCbRllR8Xj\\\\\\\/MR6LGRaTfG1ounfDT3grCuGjPEiPhyAbE9KOYhwavABJbSj4TAKA+R6IYxol8lXb1SahaZygCIVYkH9pJx7Ll4JM3XprnXV1sLWRdjxczw5ILjtmGc3pKwx9s1F22CfwzJoU3VdTzt31xXNtZFXGdF3rbw0cCJV\\\\\\\/9Yi+Nw38IuMwWkgSvNPI7qbykUXt2+DHAXJGb\\\\\\\/SNwm0q9KaEpxv0ekm3x7nEDBitIyylr0+oQX97hRZheetZTc6nW6B21qJiniS32HhL8c0MacMIpjeB\\\\\\\/8ovQHeYxQrCEseS1VtQgnVI5232pj1IoGvzBciAvndnOlWtzMSaP6a6xp6tfZQGBG1t23CFGcYcdDf7JHX+\\\\\\\/D39QWJiuc1MkjDXM3o2OiBXh30o1GIwUz4FBQ+I1f89M3YF\\\\\\\/XoNzurk2fWu7NNr7qxugTpJBwF5a4Mdao4WBpWNMdyqUy3dJ8hlDcF1P2A7reInPJ3ZGqyKIs1t\\\\\\\/IsgaXLcv10uSJEDHKV8j0Wpe8SaMQs0sFMov4x24nZO7xbO6YEqBtgGfwvKeIb7fNWRbX3PyHDHGGymHhLFvs3WCXn1Wgdq+VZDzuUNob3+oLMmN9AuY8Vwe5WX4mi+EM2JVWbBG\\\\\\\/hHBjdI0AKXVFUeM+rD4LpMObXuu1W0S5PB\\\\\\\/1YZJfhj47DXwuZM\\\\\\\/h\\\\\\\/Cb1DaB7eb55LTutQvwKXmBYOugOZsrdAtgaYK9U+wbbUlMMW9LbmBbBfgkvYzUAWHvaO8llXYODPW2hOWFRuUDXi4pRY9uPL+Cs4poHQ5AML93zazQ8AkS\\\\\\\/QNL9xL96HLUE9Ok6cVRiWs56jb8OkIqi453iiHZujsb1nzjOg5ybGXK1X7dpwWSp2\\\\\\\/Xra3IqGKKG0PcAvzP+8eNyYrcbZZoQQFpYuVd4Vigx+KkQbAT8pt8b0b0m2O0FZwUSjcft0nwb\\\\\\\/xyyYpiJRk6S4mzN\\\\\\\/DDoBvKMXw4X0cD7XsC0csDw\\\\\\\/1RmlMpi4o2OT5Lvr7e1f0BuYXU4KXp43s6YpFrWtPS12UCFlvJxJdFcc5YvHTKGbZ66orKE2xw+wEGQg3V3vvKpkqT3X2ZKraNlZdh6ldQeX6Y2Ztmj9nkqofv2olh05552HWj1\\\\\\\/J0VVE2jTtkyJ1p4nssX3Tvynu9Shtxy5IehPt1ZkI+YAC3cEtexoeBDlOZ1oXIP2LADS8BsZuvYAT5v\\\\\\\/5wIWAS+bpIyNI6hFghC4F9Tf3sjDbEo5y+HZ3pRyQbr94VegDLQngwsIcLFw11OCTBer9ECJVjfkDSOp2X2JMWanPelsMG9G6U4J9ctkBAKLw4VefoVw5lTZLIhxYnMdCbYv54bkYxnpH9hGj5PvcB1H1QFM8MMsEQPBqxA64He2ypxZYgYMSWM7fVcrUB2XHwiXeNEYcYiz8CnWx5hYAop5N1StWWM2Ed3XuYd6k9NKlr5gZ4sG2LaPjjPkSxRCVHrByZbusLku1xBk9siHcoSEPouEqP2hbdAqQ0yMtkEYSmnqefaVJL2ioZMv1ynUnOGCNpExOLMNytWr338TARFRKFaJ2P0CXQmTJ1AS2akR7kpB9UV8kCpMQx5K\\\\\\\/ETqgyXB5jMZVFdDlkl3cWgLeXF4VuQdDsKTusnxB4yIYOTlNmcWoq1MPGk7S6ZprTGgLYBKRped6wuAz3XBZyoEhZVSx2jlPAyAonnU5Pvp10LKwA6ALueJcarYDGtAdtzn9Mjr2anOjWFh+gFfYCTmxi8oxjMlqI\\\\\\\/vW5yf5vT4ChnsM\\\\\\\/ER7LqwXnVZW2MC\\\\\\\/hAIgl539JZQ10yM8xTRWsLATwSKeU1c72cGgWU1FvImqq1rlB1ApnqgCDKTvGcPJiHHchO9+9eY1MJfMYXFYyurAOUDU0Qa+WKnHGuxwBQQpuzS4vfanCY5TOQSZvHXxHNZNWdImwaji7\\\\\\\/eFm8FlisXv3B2xSMjxpEW\\\\\\\/mTqvw1tToYlP6APw8f1kLvHrP3J2aq6PoyEJrTGaoKZD\\\\\\\/BuLqmrYxMbPqvwY6\\\\\\\/Zo4NSnKREBkAzpxQnKklU6cCH9g3mCwq4yOrbHF9MA73BFdCcpGgqUU5N95MnKZLJ6Ph\\\\\\\/k4L7qQHQEQjdd0MFsDJJNIIp8x\\\\\\\/GWNBch1rCpHUd2vujqQ2tFZooE\\\\\\\/IRHW71DgTG0WKbTNn5M5fZcey8Fhf2av7v3vbrexA6+cCx\\\\\\\/cgrUnT9stoGaZp8EYqrgsj48QThxf+6qnLEclT6jShaInfqxXthDekzV7yt0kzEwwRu17erGkbdIdduzQoRSlSwxqkFiauiQIiVI0sWtasUp2Ny+p93\\\\\\\/ATrZMeNook2s3B7v+5GwbWaretBBaO82S7TUCnQeUrPU8aMjAED3iJuygCtIsHg8+6WhSBivHLZkHNCh4smLuNgkBInaYfS\\\\\\\/SBXv5\\\\\\\/L6CqQbwKNbMhZxBE2jYKOOMFhyMKfZ02qwlx+AhjggZYaO1IQJ0IftjVSOYE7FsAkmdAGk9QpQQgJS1A2CLC\\\\\\\/U5R13R\\\\\\\/8yrfyjrcIWCh8R9h4ZhR6ZmZkaO4zUMhIq8ynHdKQ\\\\\\\/H+KaK\\\\\\\/PJiJsa9ptvWUxipnfqYUhVFvcXc4+KKtZ+QClWnEml7z66ujwTnDBlfEu2aZw2rh8EayowxFhav\\\\\\\/lUmvHsnlQTgCkvvuVbcjc1KdACgKOa4E7N6niH0T3L9IOj4FKWVa6cS0f\\\\\\\/U8wfjetO5KYQKmfgFaOEDd2uHpPxS9hfTmIZCQsjRrH85OpGAfrdsDp53f+mqQUduURr080ysHMUOsaPoxcKVNNrfkAa1tbLKgMwvB5O9Fo2S7oV4S7geXLlggy5QyYVyFsLnPHO77Kxpj8R9zwSyxOFn3O4kKye+BVPtJGhBJlD1TDXob2oiIvHTX2dSHBm\\\\\\\/nowoMicZ\\\\\\\/Mcp5TsC7XD3YDW18CwNfptb8MHcefeEBUQn0TlbZ3B+udNwMsbNoQCfcsGldi+IZIUpcd\\\\\\\/xDOCNBCl7H2swOYVrMzvZLMmhB4H6CrH8tOcq7JLvZitJxWYVh8+tV95nkd7aHBQDf5w6YQ+3yGQB1WId3uvsccaVtWSGgwB+FA3BDyjP+dFYyBVdMomixJtjQQNieyz0WY\\\\\\\/tAAgyTKCiWjEJXvWbfNY9GHXG0HUFrCfoz2dibQrmRJCzP2Z2uNw59vHgozI++pjxzdm42ZkW73NwPzN1D6FRehoJof5uC3kQCHliGsFzKIeTzMN7oohyQl9KKY6OOulvN2J0p29aVLb1iB4prqgCe3xSiyaup3BOCThSYmUUyPI+eMbGjSAKfO6bDa+roUj6QPrMnTt4ZxHm3vAhWFbcP29nTRRCpJBU1Qb7yPUD+pAiAh5L20wiSgk7z41frSuUOQDDaVgkU1qwetK5AncgBn2FEaGWm7cwJk85MlGMiYunzh8Ypf3q+aHKvhNBkQElZesX\\\\\\\/DdMM3yxTvH2HraboJd5t1JGQDVihmxfxf3Ur5WyrmX7RFwMO\\\\\\\/mygKfWg14cEpMRoPV4d8GGvXyFgdSSsNMXvgmBWP5IbvGaBjRt8I3MlozcrfxYzuZP3otXhRjXidC7YBrKVsZDc2OVYNvF7WY7baVvbzpYZtUfwYf027Z3I8gH1RBFr659RDZSRxaEOH6YmHlj7fnvv3M8V3ssC78QnUuRS1ZDE7UArvjG1Rh43dJZpTFvLKQKyfAy78meSw+zWX16zwyT9KSkYGwJF6UHLCvZNKbjDnwVauSY6obleD5aRk8O6BoijCpsFBCKHN\\\\\\\/MKrTGVystuM5gFaMAIi5JtqJ6jBKsD7gfU48KNULQYcqyhylunyDstrRvVRbcYo6J4r+D9iGvvcDjFV49Vxu+85adqcWwdU5Ttkdkd4gDN8fvvHlpksZH+x0T6VophnUilg\\\\\\\/8XAuuvNniVGZdmg5WRHVvY0HQERhDC9TGAgrciv+SHGCjg+aBV3r01eSrgWWQHC\\\\\\\/T6tZJyUFGvv79oN8mgX8gRQn72K0w2PPj\\\\\\\/Ggm4tp2lvuS+wKF79r+4UMy6PodJ3aA\\\\\\\/RLCyhegQvoB4VPDlRcJBhI7oNRAvVxg7kGw8jeaTR6+cLDnuVTHMgM4c8Lddibu5ex2ZNbXCshh3ce1+rxWuer+COjjt0NedLYbvogSKALvGMMGF43cSiYenrxZfkSQQ4Dz3PtxeFNVbt8ooySe\\\\\\\/ucXzmOEpYYFvMvgDjajwVqwQinAHdgq2kouQc1ep3lfmu56CHy5IxVF6G5pWbC5JZb78K2dmszqTxltZfQ33f+FczWnCisnccIoq9C0EnPBQGu7vbXyQP6ftYc1cQA0oLJE0OUgiEbwctYbaDpIEDkW7ech4Wn+0HmYNHHJBAG8XPm\\\\\\\/JCRXhQ1LLTEmNDa\\\\\\\/0T+aZKh8di8sRhoQf5pwt5kQPtgKHUDJIUCgj3KAMexnbMO+d25qUdnjZDyV1X3Z3r4\\\\\\\/8C10RL+XBNPtQJBi99mpKkrYb9d4WBtnQGW56ox0kY5o6SM0UKcvAzLsQlrYAEsuRHfu4Ftyb5Gn1u42hKiFYuLDpENVEQxVaYrSo00PjyyknetcwvrWnEr6KV0PikVaO9iIkFwEj19JfALnY3l3RBl2usneXpts8VvFxgX6fdvgTgv6\\\\\\\/pWj\\\\\\\/esFejDfNiHf3BWtEYEwwHDDkeF2QOq1KAqyfBSJLlr3P+rJA6k1nAilqDkvoQqTJuCLg\\\\\\\/l+caKIHj5bW6Dw7Ri\\\\\\\/QtC6oe+tugiANsOeoG5X+ZFlS80EUFmzlktR\\\\\\\/xQlh3JbTvajuhKFoXf4UC4l5hdPYk2Wmhb93FMBibEBAdG5jRKx\\\\\\\/5s6FysCbncZiLjANaT0f4VWH73EV6tFtmYj5j8EbOhs6CppnwjtKjz2kIA5L+yAy7GZR4bntHiHTVETKJqq3uYbCjq341IDjHeNjlbHOe1pzUp9fsUvX\\\\\\\/HxqoRGVPErhD0FBQQ1HhJPyiQJ5cS\\\\\\\/JaHLqynY9yK\\\\\\\/3NyKI0xEB1zohchpfgDjvcXlktXkGUw1H+vgIj1xbZj3wlqx4cFkYDVAwbRa7xjF+zIufjfmUH7WHUKK7JNdejSg6d\\\\\\\/nJSG7cZDElH4JJW5cOguCtNFSit+JcrBFOS46ZdraMjc7t7byhrcXdK6DMyGvg8WwbdYikmqipCOUyj2HkL5mj51QL1UPzEfNLFSfeBZ9KZTboOrXa2g97Ykmf8S\\\\\\\/tcU2RgXc+ZNFpTDxVUF3WC89+7r\\\\\\\/T5bX5IOaxwUc3Hjmq1VV26ym1QICBBjkjoWLvfuIXhvel5oqzlIBwxCsn5LPCygFc1kfXBSjZREDigUIwnh6A6yqBrojqYsDUZMullPBUTL49C1feOVlHsNwaBkxcgFVoq9i3pt\\\\\\\/GwUyIbJGGFtS3bHmqCuOOtpCOZUA8QJJq2pd66kf9l+RwUeHP1TNkyqP0bCEaYW9sM8qq4DMQ2YyiOYMh+J+bLqY+rRBoPBSxiPP0HCrFuTrmTDBifovG2ReGd\\\\\\\/ccv\\\\\\\/e4e1K45+pBinmS5UxHJDEmoB\\\\\\\/Q\\\\\\\/T6HGiWCOBzsYDfHiZy5S7dQFwqEyHeyVfpqqCYcMbHeQYYs+7p6dOEq4KIcry8kFXas0ueUSj5EGfJosowlaS4TqRynFO4\\\\\\\/PofG8Ao1R0XzEfaggbFxJUoY\\\\\\\/3QUwBzGWcnU02LsN+7BfFKXWLL4rprYI+OtKqdZL7fTZxe1EpFBKfFU8Zy9x5nytXyTB7E9i7sIcNOEaPimZUujgN4ftstATo+NSK6AGPfNOqQoKsQXpGr4FGUMNfxWp1IEjPEQ7RLpVhaN+0SAGYpnP3bJrszx1qdJi1Q3mh0TBQQyTNrlRJoD3BmvDhhtTFlZH59JezNMs3pvSxRlltcqW72+INuQOBZiD0BHrCpT1mXiZ31sDEl+Z4YSp0QLZNwoUFOXMA8LS7K5Cb4CbU+y4DbULdyumTO1i6tSfw6X3w+S3oyTVJwQfvyFOk42nlgcWAAw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022hvkvkRSYWYEjGbN5sBIWxQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022343e22e115261d53edc0901b3056807a3ffaa3b54a47d958614f1123cb96d3bc\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022QRFtsRWGKV3farZ7ZEtjT8nRcayc0w0bnEM5MCP+vSstYBrwXliFreNx+7foUGralA9HoM5nkF75q9SaW\\\\\\\/jAEY+9SrhWz4zUJPyFssdU16FyLg5MPN16K\\\\\\\/B44rhcSjJuRfHChhdsWCRoAOBM7TPgs8GxyJxCYnBZTY8l5pGhUxeDjOA7jw6Wp7vPiNb6n1IrYB6yWJt1pjY0fszPjvzXmKp1fUeMuZdELXBpNZk3eBnY+mTUfSnQF2IWbUsfnmOgZ3Yfv\\\\\\\/3ICeGeq0BTaIdWJ0bLhP0g0wMS4nhBQYWKL2ApQKIRiJuxnnlw6wpPuTkOkdcyS0sbQyF6CYXzdzIybbUotWmJKfye+RMbtmSCVUSs57hp4waduLf320aBR1HhDUAQ9x+F9FYfDHOmwWBOW1oTkqZ4oIFWM1yr8ZiwX0iIE2W3BO+RaPNoJEbMNnElqJkG5dEtZ2Bb39ZQbkjtdfFd2i1o1Lnu\\\\\\\/\\\\\\\/obad8sDPip1FdGITytssD4DNoQDzUReaA3ygMjldYnpm89hlNB4M18SANNc3nNEdr6pcN1USH08pxZInBeZpbmdko1aARBdp4S1b1EuhnBxY00UjNN6yHAJeq\\\\\\\/npiEU8yzohWAltl3LNPtR36LTNGVWrCCnfCr\\\\\\\/hyN8jP2ExLuhvsXO+\\\\\\\/xbgt\\\\\\\/dL4WlxgF5fNQAoLPT6aGjqcsXnckTp+Tm7lekxuQw4a9dyu5nHWRrB8sq7YsQx5UGyRqi+jdOFIgX1\\\\\\\/0UN\\\\\\\/A6GyO\\\\\\\/9WpMo8V8DbbTOLC9W1uD+7KOdigGWbbI02dxwtPDrMd8FjKaeKkrz5DlPk8IYRKW8aql3upoicYY0sLDrcW6tZufpm7zTgjAsYyDKn1lHqccGJemJ5CDL1LHhoq5JNVQDkCbA5kdQw2MBAnbDKmRQnUhDnOkTFiEiJL8wP8XRw8k3PM5V542OvflkBOdqJxT8BO1LkC5nVwom3gqSMJpiKW38nfvnsmMveS1gwafXYyk1dClOYKCZpyV83+LSbLF3LuNk6RbdnfbvU0NtVRdhHavMjbn3SkLoc0pUfacJYuYSxrcFnKsRbNr1b1ZhVj6qoaoKl6tkZ\\\\\\\/w4I+xrlDp+f4WZdG0HWl33d11I810bIZBq2ZetQ0Kuql7dd1va8+KuJfILhE+Zzqf8JjSe2aU1cxMfuI8ivkdyiJl37TkG1AScEAPU7+Xoouy3VyeomSQq47RaPoq9iURXEfQjvXlkYeTVnsI44EvweV7O0dvRTR3NXyEB2XXV\\\\\\\/poIeht5Bp5IUQuf2vQG\\\\\\\/HNvMIG+ysccte2jAUHHYehTo6BWEMe5tkF\\\\\\\/\\\\\\\/7BewIahhvheNU1w5yH9yj\\\\\\\/b6pvnWPi1rHG\\\\\\\/8EY2Tm147\\\\\\\/olMxaWRe\\\\\\\/LvxzNhvjJhlMbLO5LBBpAnE3gAib4BE4DVpreszkXnroEeZ3IDvBdmiI0QAp3rVBC5ouP+XUHIzAuDkq9kDswNFqbJW71PMeGZL+I8SxCJd0KKgfXczCueLNbyWGAO1zBe6BnLIDYowGCx+dkVTlcMWt4xEGBHYzsYeFW93k7Rgdm\\\\\\\/5yeQ59Yi1KjejJFzjSa+N15gWBiMTYXZ5n9DDAxLstD+TMz6T3XFGGQjbPaQ9ozl+mvkeEj+TPpLpF\\\\\\\/pJTNmlQbZWPvrwnKWQ0xEfeM97LVgIACbp5qKiXlWlVCzs5210L3YFjeBa4eq8jP\\\\\\\/isYkF2pOFlRl1MBlKxoIp9W\\\\\\\/8Tnfp3cQuW4YzFHSxybeZYuyd8DHFz29hiqWpJH5Q\\\\\\\/o7gpJqLwCETItm+2ID8Mu1N4IoRi8c1B9Ua6tF3mTsYosSdVHRuwdTk+2z+wU5\\\\\\\/I4+qxxyXFs1n1rTGgkGcJfi4mgqjPQjz9NlpVL9Oy3KCBifuPr9T2BFK8ucnOr0tCRSxb2qLW9K9K6kxX4RTiJBfI\\\\\\\/mcE81NAGG\\\\\\\/0lfH79NyoNQ3efBOpeimw2f4vf0fa+C1X5FyRtAvWDBHF+NSULEX3BXMmeBdGd4trQRHDlWKe0qU0JJPGLC5A3WwFpcT4w==\\\u0022,\\\u0022iv\\\u0022:\\\u00220ICL0A0Guwlo5sPfkZANUg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022178c9e37cffb6fed95d53b9c26cdd973ce2ed92d1a5061b7feecb571294bd3d9\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459910 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-mutant-screen-report">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1301" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--TbNT0VpCMpIFRAxFDXE8NHBsrCFkiad0fu7yf0u8YE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1301" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9CYGVJbAqofjdQBf05KY12e0OZn2YYSfHrt-zw4j7Qw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/6/1283" title="A Multifunctional Mutagenesis System for Analysis of Gene Function in Zebrafish" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/6/1313" title="Corrigendum" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459910" id="node459910" data-pisa="ggg;5/6/1301" data-pisa-master="ggg;g3.115.017285" data-apath="/ggg/5/6/1301.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mutations that Separate the Functions of the Proofreading Subunit of the <em>Escherichia coli</em> Replicase</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zakiya Whatley</span> and <span class="highwire-citation-author" data-delta="1">Kenneth N. Kreuzer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1301-1311;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017285 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459910" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zakiya Whatley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University Program in Genetics and Genomics, Duke University, Durham, North Carolina 27705</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zakiya%2BWhatley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whatley%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZakiya%2BWhatley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kenneth N. Kreuzer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University Program in Genetics and Genomics, Duke University, Durham, North Carolina 27705</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry, Duke University Medical Center, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kenneth%2BN.%2BKreuzer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kreuzer%20KN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKenneth%2BN.%2BKreuzer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kenneth.kreuzer@duke.edu" class="" data-icon-position="" data-hide-link-title="0">kenneth.kreuzer@duke.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/6/1301" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459910" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459910/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1301.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459910" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459910/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1301.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459910" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459910/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/6/1301.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459910" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459910/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">The <em>dnaQ</em> gene of <em>Escherichia coli</em> encodes the ε subunit of DNA polymerase III, which provides the 3′ → 5′ exonuclease proofreading activity of the replicative polymerase. Prior studies have shown that loss of ε leads to high mutation frequency, partially constitutive SOS, and poor growth. In addition, a previous study from our laboratory identified <em>dnaQ</em> knockout mutants in a screen for mutants specifically defective in the SOS response after quinolone (nalidixic acid) treatment. To explain these results, we propose a model whereby, in addition to proofreading, ε plays a distinct role in replisome disassembly and/or processing of stalled replication forks. To explore this model, we generated a pentapeptide insertion mutant library of the <em>dnaQ</em> gene, along with site-directed mutants, and screened for separation of function mutants. We report the identification of separation of function mutants from this screen, showing that proofreading function can be uncoupled from SOS phenotypes (partially constitutive SOS and the nalidixic acid SOS defect). Surprisingly, the two SOS phenotypes also appear to be separable from each other. These findings support the hypothesis that ε has additional roles aside from proofreading. Identification of these mutants, especially those with normal proofreading but SOS phenotype(s), also facilitates the study of the role of ε in SOS processes without the confounding results of high mutator activity associated with <em>dnaQ</em> knockout mutants.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/dna-polymerase" class="hw-term hw-article-keyword hw-article-keyword-dna-polymerase" rel="nofollow">DNA polymerase</a></li><li class="kwd"><a href="/keyword/epsilon-subunit" class="hw-term hw-article-keyword hw-article-keyword-epsilon-subunit" rel="nofollow">epsilon subunit</a></li><li class="kwd"><a href="/keyword/linker-scanning-mutagenesis" class="hw-term hw-article-keyword hw-article-keyword-linker-scanning-mutagenesis" rel="nofollow">linker-scanning mutagenesis</a></li><li class="kwd"><a href="/keyword/mutation-rate" class="hw-term hw-article-keyword hw-article-keyword-mutation-rate" rel="nofollow">mutation rate</a></li><li class="kwd"><a href="/keyword/sos-response" class="hw-term hw-article-keyword hw-article-keyword-sos-response" rel="nofollow">SOS response</a></li></ul><p id="p-4">The product of <em>dnaQ</em>, the ε subunit of <em>Escherichia coli</em> replicative polymerase, DNA polymerase III (Pol III), provides the 3′ → 5′ exonuclease activity for proofreading function (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Echols <em>et al.</em> 1983</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Scheuermann <em>et al.</em> 1983</a>). The protein physically interacts with the α and θ subunits, gene products of <em>dnaE</em> and <em>holE</em>, respectively; together, these three subunits comprise the Pol III core. The polymerase activity is in α, whereas the function of θ has not been clearly defined beyond binding to ε (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Studwell-Vaughan and O’Donnell 1993</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Slater <em>et al.</em> 1994</a>). In addition to fidelity, ε adds structural integrity, increasing the processivity of Pol III in biochemical studies (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Studwell and O’Donnell 1990</a>).</p><p id="p-5">ε is a 243-amino-acid protein comprising the N-terminal proofreading domain and the C-terminal segment (εCTS) that binds to the α subunit (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). The N-terminal domain (1–186; N186) contains three conserved exo motifs: exo I (8-21), exo II (95-108), and exo IIIε (128-192) (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bernad <em>et al.</em> 1989</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Blanco <em>et al.</em> 1992</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barnes <em>et al.</em> 1995</a>). Mutational analyses have shown that exo I, exo II, and exo IIIε contain conserved residues that are required for exonuclease activity, and the N186 domain is sufficient for proofreading (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Taft-Benz and Schaaper 1998</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Perrino <em>et al.</em> 1999</a>). In addition to proofreading function, N186 also contains residues responsible for binding to θ (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Derose <em>et al.</em> 2003</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Keniry <em>et al.</em> 2006</a>). The proposed Q-linker (190-212) of εCTS contains four consecutive glutamines (194-197) and remains flexible even when bound to α, as determined by NMR spectroscopy (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Wootton and Drummond 1989</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Ozawa <em>et al.</em> 2008</a>). At the junction of the N-terminal domain and the C-terminal segment, there is a β-clamp binding motif (QTSMAF at amino acids 182–187) (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Jergic <em>et al.</em> 2013</a>). The β-clamp binding motif is responsible for the ε–β interaction of the tripartite complex comprising α, ε, and β (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Toste Rego <em>et al.</em> 2013</a>). Additionally, there is a poorly mapped site for chaperone DnaK binding within the εCTS (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bressanin <em>et al.</em> 2009</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1301/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Organization of &#x3B5; domains and motifs. (B) Structural data from RCSB Protein Data Bank accession numbers IJ54 and 4GX8. Residues 182&#x2013;218 do not have structural information available." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-755602320" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Organization of &#x3B5; domains and motifs. (B) Structural data from RCSB Protein Data Bank accession numbers IJ54 and 4GX8. Residues 182&#x2013;218 do not have structural information available.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1301/F1.medium.gif" width="274" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/6/1301/F1.medium.gif" width="274" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1301/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1301/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466168" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-6" class="first-child">(A) Organization of ε domains and motifs. (B) Structural data from RCSB Protein Data Bank accession numbers IJ54 and 4GX8. Residues 182–218 do not have structural information available.</p><div class="sb-div caption-clear"></div></div></div><p id="p-7">Various <em>dnaQ</em> mutant alleles have been discovered or created, many of which, including <em>dnaQ49</em> and <em>mutD5</em>, generate an extremely high mutator phenotype (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Degnen and Cox 1974</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Horiuchi <em>et al.</em> 1978</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Echols <em>et al.</em> 1983</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fijalkowska and Schaaper 1996</a>). <em>MutD5</em> (T15I) is an extreme mutator, with mutation frequencies up to 3000-fold higher than wild-type levels using rifampicin resistance as a measure (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Degnen and Cox 1974</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Schaaper 1988</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Fijalkowska and Schaaper 1996</a>). <em>DnaQ49</em> (V96G) is a recessive, temperature-sensitive mutant with negligible mutator activity at 28–30° but high mutator activity at 37° (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Horiuchi <em>et al.</em> 1978</a>; <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Echols <em>et al.</em> 1983</a>). The ε proteins encoded by these two mutants differ in their ability to associate with α and, hence, the Pol III core (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Takano <em>et al.</em> 1986</a>). Cells carrying extreme mutator alleles of <em>dnaQ</em> are not viable, at least in part because of loss of proofreading, which leads to massive misincorporation and overload of the mismatch repair system (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Schaaper 1989</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Schaaper and Radman 1989</a>; <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Fijalkowska and Schaaper 1996</a>). However, the extreme increase in mutation frequency facilitates the rapid acquisition of suppressor mutations in <em>dnaE</em> (encodes α), which alleviate the growth defect and reduce the mutator phenotype (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lancy <em>et al.</em> 1989</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Schaaper and Cornacchio 1992</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fijalkowska and Schaaper 1995</a>).</p><p id="p-8">In addition to the mutator phenotype, <em>dnaQ</em> mutants also display a partially constitutive SOS phenotype and an increased frequency of direct repeat recombination (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lifsics <em>et al.</em> 1992</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Slater <em>et al.</em> 1994</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Saveson and Lovett 1997</a>). Although these phenotypes can be rationalized as being caused by loss of proofreading function, they may instead (or in addition) relate to the effects of ε on the behavior (<em>e.g.</em>, processivity) of DNA polymerase (for example, see discussion by <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Saveson and Lovett 1997</a>).</p><p id="p-9">Previous studies in our laboratory uncovered an unexpected connection between ε and quinolone-induced DNA damage in <em>E. coli</em>. Quinolones are a broad-spectrum antibiotic class that target type II topoisomerases and lead to inhibition of DNA synthesis. There are numerous quinolone derivatives, including members of the more potent subclass known as fluoroquinolones. There are also variations on the detailed mechanism of drug action, depending on the particular structure of the quinolone derivative and the cellular conditions (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Drlica <em>et al.</em> 2008</a>). Quinolones, including the parent compound nalidixic acid, target DNA gyrase in <em>E. coli</em>, stabilizing the so-called cleavage complex in which gyrase is covalently attached to the two newly cleaved 5′ ends of DNA in the strand-passage reaction intermediate. These stabilized cleavage complexes (SCCs) are reversible and their presence is necessary, but not sufficient, for cell death. Genetic evidence implies that quinolone cytotoxicity depends on formation of chromosomal double-strand breaks; however, the mechanism(s) of break formation is still unclear and the cytotoxic breaks may not be the breaks within the SCC (for review, see <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Drlica <em>et al.</em> 2008</a>). The strongest evidence relates to the importance of RecBCD protein, which requires DNA ends to act. RecBC knockout mutants are hypersensitive to nalidixic acid treatment (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">McDaniel <em>et al.</em> 1978</a>), implying that RecBC-dependent recombination rescues quinolone-induced breaks. Also, SOS induction following nalidixic acid treatment requires RecBCD enzyme, which processes broken ends and facilitates RecA loading for induction of the damage response (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">McPartland <em>et al.</em> 1980</a>). Direct evidence of chromosome fragmentation after quinolone treatment was uncovered using sedimentation analysis and supercoiling assays (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Steck and Drlica 1984</a>).</p><p id="p-10">A transposon insertion screen to identify mutants defective in SOS induction following nalidixic treatment returned 18 mutants, all of which were <em>recB</em> or <em>recC</em> mutants (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Newmark <em>et al.</em> 2005</a>). This study confirmed the RecBC-dependent nature of quinolone-induced SOS response, but failed to uncover any novel mutants. However, this screen would likely have missed mutants that were partially SOS constitutive. A subsequent screen of SOS constitutive transposon mutants revealed <em>dnaQ</em> mutants with a substantial but incomplete defect in SOS induction following quinolone treatment (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Pohlhaus <em>et al.</em> 2008</a>). This was not a general SOS defect, because the mutants were capable of RecFOR-dependent SOS induction (after treatment with mitomycin C). The <em>dnaQ</em> transposon insertion mutants therefore exhibited an SOS profile similar to <em>recBC</em> mutants, implicating ε in the mechanism of double-strand break formation after nalidixic acid treatment. Our hypothesis is that this ε role in double-strand break formation is not related to exonuclease activity, but rather reflects some function of ε in replication fork stability that is critical for how the fork responds to the quinolone-induced cleavage complexes, which have been shown to block replication forks <em>in vivo</em> (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Pohlhaus and Kreuzer 2005</a>).</p><p id="p-11">The hypothesis above predicts that ε performs multiple functions, which should be mutationally separable. Isolation of separation of function mutants would substantiate the hypothesis and could also provide a very valuable reagent—a non-mutator (stable) <em>dnaQ</em> mutant with SOS phenotypes. In this study, we introduced small perturbations throughout the <em>dnaQ</em> sequence using pentapeptide linker-scanning mutagenesis. This procedure introduces variable five-amino acid cassettes, as well as a subset of small insertions with stop codons, randomly throughout the sequence of a gene, and has proven effective in the separation of phenotypes for multi-functional proteins XerD and McrA (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cao <em>et al.</em> 1997</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Anton and Raleigh 2004</a>). Five-amino-acid insertions anywhere in the coding sequence can potentially cause separation of function, whereas stop codon insertions toward the C terminus might be expected to create partially functional truncation proteins. Our results show that the proofreading function of ε can be uncoupled from the SOS phenotypes. We also isolated insertion mutants that separate partially constitutive SOS and defective nalidixic acid–induced SOS phenotypes, as well as a mutationally stable <em>dnaQ</em> allele with SOS phenotypes. Overall, our results implicate ε-dependent replisome behavior in the bacterial response to nalidixic acid.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Bacterial strains and plasmids</h3><p id="p-12"><em>E. coli</em> strain JH39 [F- <em>sfiA11</em>, <em>thr-1</em>, <em>leu-6</em>, <em>hisG4</em>, <em>argE3</em>, <em>ilv(Ts)</em>, <em>galK2</em>, <em>srlD</em>, <em>rpsL31</em>, <em>lacΔU169</em>, <em>dinD1</em>::MudI(Ap<sup>r</sup> lac)] was the parental strain used to generate <em>dnaQ</em> mutants with chromosomal <em>dnaQ</em> mutations. The <em>dnaQ</em> deletion from the Keio collection (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baba <em>et al.</em> 2006</a>) was moved into JH39 by phage P1-mediated transduction, followed by PCR verification. All plasmid constructions were done in DH5α [F<sup>−</sup> <em>endA1 glnV44 thi-1 recA1 relA1 gyrA96 deoR nupG Φ80dlacZΔM15 Δ(lacZYA-argF)U169</em>, <em>hsdR17(r<sub>K</sub><sup>-</sup> m<sub>K</sub><sup>+</sup>)</em>, <em>λ</em><sup>−</sup>]. Plasmids are summarized in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466202/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466202/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Plasmid constructs</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Linker scanning mutagenesis</h3><p id="p-13">Mutagenesis was performed with the New England Biolabs GPS-LS system according to provided instructions. Plasmid pMM5Z was created by digesting pMM5 with <em>Aat</em>II restriction enzyme and annealing oligos <em>Bgl</em>II #1 (5′-TGAGATCTCCGCAACGT-3′) and <em>Bgl</em>II #2 (5′-TGCGAGATCTCAACGT-3′). The transposon reaction of plasmid pMM5Z (see <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) was electroporated into DH5α using a BioRad Gene Pulser according to the manufacturer instructions, selecting for insertion mutants on plates with chloramphenicol and ampicillin. After overnight incubation at 37°, 1017 colonies were collected and pooled into the primary mutant library.</p></div><div id="sec-4" class="subsection"><h3>Generating the <em>dnaQ</em> mutant library</h3><p id="p-14">Plasmid DNA from the primary library of 1017 colonies (see <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>) was digested with <em>Pst</em>I and <em>Dra</em>III. Fragments containing the <em>dnaQ</em> gene along with the 1376-bp transposon insertion were separated by gel electrophoresis and extracted. The extracted fragments were ligated to fresh <em>Pst</em>I/<em>Dra</em>III-cleaved pMM5Z vector and electroporated into DH5α (selecting again with chloramphenicol and ampicillin). The colonies were pooled and a secondary plasmid library was generated and extracted. The secondary plasmid library was digested with PmeI, gel-purified, and re-ligated—these steps remove the majority of the transposon, leaving behind a 15-bp insertion with a single PmeI site. The desired products were isolated by transformation into DH5α. The plasmid library was harvested and then the final <em>dnaQ</em> insertion mutant library was subcloned into the polylinker of pMAK705 using <em>Cla</em>I and <em>Pst</em>I sites that flank the <em>dnaQ</em> gene.</p></div><div id="sec-5" class="subsection"><h3>Site-directed mutagenesis</h3><p id="p-15">The Stratagene QuikChange Mutagenesis kit was used according to manufacturer’s instructions to generate the moderate mutator <em>dnaQ928</em>, which causes the G17S substitution (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Taft-Benz and Schaaper 1998</a>) in the context of plasmid pMM5Z. Similarly, we created strong (QLSLPL) and weak (ATSMAF; Q182A) β-clamp binding mutants in the motif (QTSMAF at residues 182–187) using primers QLSLPL (5′-CTG GCGATGACCGGTGGTCAACTATCGTTGCCTTTAGCGATGGAAGGAGAGAC-3′), QLSLPL_anti (5′-GTCTCTCCTTCCATCGCTAAAGGCAACGATAGTTGACCACCGGTCA TCGCCAG-3′), Q182A (5′-GGCGATGACCGGTGGTGCAACGTCGATGGCTTTT-3′), and Q182A_anti (5′-AAAAGCCATCGACGTTGCACCACCGGTCATCGCC-3′).</p></div><div id="sec-6" class="subsection"><h3>Gene replacement</h3><p id="p-16">Following the method of <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Hamilton <em>et al.</em> (1989)</a>, we incorporated insertion and site-directed mutations into the chromosome of JH39 at the native location of <em>dnaQ</em>. This method uses plasmid pMAK705, with its temperature-sensitive replicon, to facilitate incorporation into the chromosome. Plasmid pMAK705 containing a <em>dnaQ</em> allele can integrate into the <em>dnaQ</em> region of the chromosome by homologous recombination, and integrants are selected on chloramphenicol plates at the nonpermissive temperature (44°). Selection is then relieved by growth at the permissive temperature (30°), which favors segregants that lose the plasmid from the chromosome. Chloramphenicol-sensitive segregants were screened for those with the desired <em>dnaQ</em> in the chromosome via PCR amplification using chromosome-specific primers dnaQ5F (5′- TGCCCCAAAACGAAGGCAGT -3′) and dnaQI3R2 (5′- TGCAGCCACAAAACGCAGTG -3′). PCR products were digested with PmeI and sequenced using primers dnaQseq1 (5′-ACCGCTCCGCGTTGTGTTCC-3′) or dnaQseq2 (5′-CGGTTGTTGGTGGTGCGGGT-3′).</p></div><div id="sec-7" class="subsection"><h3>Quantitative β-galactosidase assay</h3><p id="p-17">The convenient <em>dinD</em>::<em>lacZ</em> reporter allows monitoring of SOS in the JH39 strain. SOS levels were determined using a β-galactosidase protocol modified for 96-well microtiter plates (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Thibodeau <em>et al.</em> 2004</a>). Overnight cultures were diluted 1:100 in LB and grown for 2 hr. They were then treated with nalidixic acid (10 μg/ml) or no drug and incubated for an additional 2 hr. Cell density (A<sub>630</sub>) was measured and 100 μl of each culture was lysed with Novagen PopCulture reagent at room temperature. A 15-μl aliquot of lysed cells was incubated with 135 μl Z-Buffer (40 mM Na<sub>2</sub>HPO<sub>4</sub>, 60 mM NaH<sub>2</sub>PO<sub>4</sub>, 10 mM KCl, 1 mM MgSO<sub>4</sub>), 0.5 μl TCEP [Tris (2-carboxyethyl) phosphine], and 30 μl of 2-Nitrophenyl β-D-galactopyranoside (4 mg/mL). The accumulation of 2-nitrophenol product was measured at A<sub>405</sub> for 60 min. Miller units were calculated using the modified equation (V*1000*CF1*CF2)/(A<sub>595</sub>*CF3*relative volume of cell lysate used). CF1, CF2, and CF3 are conversion factors that account for absence of Na<sub>2</sub>CO<sub>3</sub>, A<sub>415</sub> to A<sub>420</sub>, and A<sub>595</sub> to A<sub>600</sub>, respectively (making this equation comparable to that of Miller). This method assesses the rate of β-galactosidase accumulation over time and is therefore more accurate than simple endpoint measurement β-galactosidase assays.</p></div><div id="sec-8" class="subsection"><h3>Mutation rate measurements</h3><p id="p-18">The rifampicin-resistant colony-forming units of at least eight independent overnight cultures and the total colony-forming units of at least three overnight cultures were measured. The mutation rate of each strain was calculated using the Lea Coulson Method of the Medians Fluctuation Analysis Calculator (FALCOR) program (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Foster 2006</a>).</p></div></div><div class="section results-discussion" id="sec-9"><h2 class="">Results and Discussion</h2><div id="sec-10" class="subsection"><h3>Generation of <em>dnaQ</em> mutants</h3><p id="p-19">To determine if the multiple phenotypes of a <em>dnaQ</em> mutant are separable, we used the GPS-LS Mutagenesis kit (NEB) to create a plasmid library of <em>dnaQ</em> insertion mutants. This system introduces a 1376-bp transposon randomly throughout the target sequence. Subsequent digestion with PmeI excises the majority of the transposon, leaving behind a 15-bp segment that encodes a 5-codon insertion and, in a subset of cases, one of the five codons is a stop codon that generates truncations in the protein.</p><p id="p-20">Our general strategy involved sequencing the sites of insertion in a number of plasmid-borne <em>dnaQ</em> alleles, and then selecting a subset to move into the chromosome and replace the native <em>dnaQ</em> gene (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The subset was selected based on available structural, biochemical, and genetic data in the literature. We included insertions within both the N186 domain and the CTS. Insertions within the N-terminal domain were selected based on structural considerations (Protein Database ID: 2XY8 and 2GUI) (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Berman <em>et al.</em> 2000</a>); we chose locations that seemed less likely to greatly disrupt protein structure and proofreading/exonuclease function. Thus, we avoided highly conserved residues and chose targets on the outer surfaces of the protein rather than buried in the core. We also included two insertion/truncations, one of which (at residue 132) likely generates a complete knockout as a reference for loss of function. The other (at residue 224) creates a truncation in the CTS, which might possibly compromise a function involved in replication fork behavior without affecting proofreading function.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466314/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466314/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">DnaQ insertion mutants</span><div class="sb-div caption-clear"></div></div></div><p id="p-22">In addition to insertion mutants, we generated three site-specific mutants based on prior studies of <em>dnaQ</em>. First, we recreated a moderate mutator allele, <em>dnaQ928</em>. This allele behaves as a dominant mutation and alters the exo I motif (G17S), indicating that it retains the ability to associate with α (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Taft-Benz and Schaaper 1998</a>). We also regenerated two interesting mutations that have been shown to effect binding of β-clamp <em>in vitro</em> (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Jergic <em>et al.</em> 2013</a>). These authors characterized a β-clamp binding motif within ε, QTSMAF, at position 182–187, and generated strong (QLSLPL) and weak (ATSMAF; Q182A) binding alleles. Interestingly, they showed that a stronger β-clamp interaction with ε led to a more stable and processive Pol III core; therefore, these mutants are expected to alter replisome behavior and might affect SOS phenotypes.</p><p id="p-23">The desired subset of insertion mutations (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>) and the site-specific mutations were moved onto the chromosome, replacing the native <em>dnaQ</em> in the correct location and with the natural sequences for gene expression and regulation. We focused on chromosomal constructs because prior experiments revealed conflicting results when <em>dnaQ</em> mutants were expressed from a plasmid in comparison to expression from the chromosome, indicating that aberrant gene expression from the plasmid was a confounding issue (data not shown). Chromosomal gene replacement was achieved using plasmid pMAK705, a temperature-sensitive plasmid with chloramphenicol resistance (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Hamilton <em>et al.</em> 1989</a>). The <em>dnaQ</em> mutant fragment was cloned into the polylinker of pMAK705 and substituted into the chromosome of parental strain JH39 (see <em>Materials and Methods</em>). Chromosomal mutant segregants that completely lack the substitution plasmid were confirmed by lack of chloramphenicol resistance and by sequencing chromosomal PCR products that contained the mutant <em>dnaQ</em> allele. Additionally, the <em>dnaE</em> gene of each insertion and site-directed mutant was sequenced in its entirety to check for possible suppressor mutations. Although the sequencing confirmed each <em>dnaE</em> gene as wild-type, we cannot exclude the possibility of suppressor mutations elsewhere in the genome.</p></div><div id="sec-11" class="subsection"><h3>Phenotypes of <em>dnaQ</em> mutants</h3><p id="p-24">Proofreading activity of <em>dnaQ</em> was assessed using the well-established rifampicin resistance assay as a measure of mutation rate. At least eight independent cultures of each strain were grown overnight and appropriate dilutions/aliquots were plated on LB agar with and without rifampicin, and the frequency of rifampicin resistant over total colony forming units was calculated. Mutation rates were then calculated using the Lea Coulson Method of the Medians (see <em>Materials and Methods</em>) (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Foster 2006</a>). SOS phenotypes were determined using a high-throughput β-galactosidase assay in microtiter plates, measuring both constitutive SOS levels and the ability of the mutants to induce SOS following nalidixic acid treatment. All mutation and SOS measurements are summarized in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466164/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466164/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Summary of SOS values and mutation rates</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">To analyze the relationships between phenotypes of the various mutants and compare them to wild-type and a <em>dnaQ</em> deletion mutant, we plotted phenotypes in pairs. First, we compared mutation rates to basal SOS levels as a measure of proofreading activity (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The mutation rates were also compared to the increase in SOS levels with nalidixic acid treatment (ΔNalidixic acid) (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Finally, the basal SOS levels were compared to the ΔNalidixic acid levels to look for possible separation of SOS functions (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1301/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Basal SOS levels and mutation rate. SOS levels (bars) are indicated by &#x3B2;-galactosidase activity and correspond to the left axis. Mutation rate (&#x25A1;) corresponds with the right axis. Basal SOS error bars indicate SE and mutation rate error bars represent the upper and lower 95% C.I.s. Samples are sorted by decreasing mutation rate. Wild-type (WT) is green and &#x394;DnaQ is red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-755602320" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Basal SOS levels and mutation rate. SOS levels (bars) are indicated by &#x3B2;-galactosidase activity and correspond to the left axis. Mutation rate (&#x25A1;) corresponds with the right axis. Basal SOS error bars indicate SE and mutation rate error bars represent the upper and lower 95% C.I.s. Samples are sorted by decreasing mutation rate. Wild-type (WT) is green and &#x394;DnaQ is red.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1301/F2.medium.gif" width="440" height="300"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/6/1301/F2.medium.gif" width="440" height="300"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1301/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1301/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466227" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-27" class="first-child">Basal SOS levels and mutation rate. SOS levels (bars) are indicated by β-galactosidase activity and correspond to the left axis. Mutation rate (□) corresponds with the right axis. Basal SOS error bars indicate SE and mutation rate error bars represent the upper and lower 95% C.I.s. Samples are sorted by decreasing mutation rate. Wild-type (WT) is green and ΔDnaQ is red.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1301/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The &#x394; nalidixic acid SOS levels and mutation rate. The &#x394; nalidixic acid SOS levels (bars) represent the difference between &#xB1; nalidixic acid samples (note position of &#x394; nalidixic acid = 0 on the Y axis) and are indicated by &#x3B2;-galactosidase activity corresponding to the left axis. Mutation rate (&#x25A1;) corresponds with the right axis. The &#x394; nalidixic acid SOS error bars indicate SE and mutation rate error bars represent the upper and lower 95% C.I.s. Samples are sorted by decreasing mutation rate. Wild-type is green and &#x394;DnaQ is red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-755602320" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The &#x394; nalidixic acid SOS levels and mutation rate. The &#x394; nalidixic acid SOS levels (bars) represent the difference between &#xB1; nalidixic acid samples (note position of &#x394; nalidixic acid = 0 on the Y axis) and are indicated by &#x3B2;-galactosidase activity corresponding to the left axis. Mutation rate (&#x25A1;) corresponds with the right axis. The &#x394; nalidixic acid SOS error bars indicate SE and mutation rate error bars represent the upper and lower 95% C.I.s. Samples are sorted by decreasing mutation rate. Wild-type is green and &#x394;DnaQ is red.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1301/F3.medium.gif" width="440" height="305"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/6/1301/F3.medium.gif" width="440" height="305"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1301/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1301/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466185" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-28" class="first-child">The Δ nalidixic acid SOS levels and mutation rate. The Δ nalidixic acid SOS levels (bars) represent the difference between ± nalidixic acid samples (note position of Δ nalidixic acid = 0 on the Y axis) and are indicated by β-galactosidase activity corresponding to the left axis. Mutation rate (□) corresponds with the right axis. The Δ nalidixic acid SOS error bars indicate SE and mutation rate error bars represent the upper and lower 95% C.I.s. Samples are sorted by decreasing mutation rate. Wild-type is green and ΔDnaQ is red.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1301/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Basal and &#x394; nalidixic acid SOS levels. Basal &#x3B2;-galactosidase activity (bar) graphs to the left axis and &#x394; nalidixic acid values (&#x25CB;) to the right axis (difference between nalidixic acid SOS level and basal SOS level; note the position of 0 on Y axis). Error bars represent SE. Samples are sorted by increasing basal SOS levels. Wild-type is green and &#x394;DnaQ is red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-755602320" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Basal and &#x394; nalidixic acid SOS levels. Basal &#x3B2;-galactosidase activity (bar) graphs to the left axis and &#x394; nalidixic acid values (&#x25CB;) to the right axis (difference between nalidixic acid SOS level and basal SOS level; note the position of 0 on Y axis). Error bars represent SE. Samples are sorted by increasing basal SOS levels. Wild-type is green and &#x394;DnaQ is red.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1301/F4.medium.gif" width="440" height="300"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/6/1301/F4.medium.gif" width="440" height="300"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1301/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1301/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466334" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-29" class="first-child">Basal and Δ nalidixic acid SOS levels. Basal β-galactosidase activity (bar) graphs to the left axis and Δ nalidixic acid values (○) to the right axis (difference between nalidixic acid SOS level and basal SOS level; note the position of 0 on Y axis). Error bars represent SE. Samples are sorted by increasing basal SOS levels. Wild-type is green and ΔDnaQ is red.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">In the first two comparisons, we aligned all mutants according to decreasing mutation rate from left to right (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Although somewhat arbitrary, the mutants roughly fall into three categories (separated by dotted lines left to right): high mutator (like <em>ΔdnaQ</em>; red bar); moderate mutator [like <em>dnaQ928</em> (G17S)]; and non-mutator (like WT; green bar). The lowest median mutation rate of the moderate mutator group is 11-fold higher than wild-type, whereas the lowest median mutation rate in the high mutator group is more than 225-fold higher than wild-type. Three of the mutants in the right “nonmutator” category appeared to have a slight elevation (2-fold to 5-fold) of mutation rate compared to the wild-type; these include the two site-directed mutants in the β-clamp binding motif (Q182A and QLSLPL) as well as insertion mutant <sup>162</sup>LFKQL.</p></div><div id="sec-12" class="subsection"><h3>Insertions that behave like wild-type</h3><p id="p-31">Wild-type strains that are <em>dnaQ</em><sup>+</sup> are nonmutators, are not SOS constitutive, and respond to nalidixic acid treatment with high levels of SOS induction. Insertions <sup>115</sup>KLFKH, <sup>125</sup>LFKHF, and <sup>196</sup>LFKQQ behave in this way (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>), indicating that these insertions are innocuous for all ε functions and for important structural features of the protein. The insertion at residue 115 is at the end of α helix 4 and between motifs exo II and exo IIIε, on the outside surface of the protein. The <sup>125</sup>LFKHF insertion is also between these two motifs at the end of the α helix, but before β sheet 5. Insertion <sup>196</sup>LFKQQ is within the four consecutive glutamines of the Q-linker region (but does retain two runs of three glutamines). NMR studies of εCTS determined that the Q-linker region and especially the stretch of four glutamines are highly flexible; therefore, the flexible nature of this region might possibly make it tolerant to insertions or other alterations.</p></div><div id="sec-13" class="subsection"><h3>Insertions that behave like <em>ΔdnaQ</em></h3><p id="p-32">Complete knockout of ε leads to high mutation rate, partially constitutive SOS, and failure to induce SOS following nalidixic acid treatment. Mutants that behave in this way include <sup>30</sup>IICLN, <sup>79</sup>CLNTF, <sup>97</sup>ILFKQ, <sup>224</sup>AHV*, <sup>232</sup>LFKQV, and <sup>132</sup>V* (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). Although the insertion at 30 is not within the conserved exo I or exo II motifs, it is at the core of the protein on β-strand 2, which seems likely to disrupt protein folding (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hamdan <em>et al.</em> 2002</a>). The truncation <sup>132</sup>V* eliminates much of the exo IIIε motif and the entire C-terminal segment; again, it is not surprising that it abolishes all ε function. Insertion <sup>97</sup>ILFKQ maps very close to the conditionally lethal mutator <em>dnaQ49</em> (V96G). Both mutations are within the ε exo II motif, and <em>dnaQ49</em> has been shown to be recessive and encode an ε that cannot bind to α (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Takano <em>et al.</em> 1986</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Jonczyk <em>et al.</em> 1998</a>). The insertion mutant <sup>79</sup>CLNTF, located between exo I and exo II, may disrupt protein stability as it is located in α helix 3, which comprises the active site along with α helix 7 and the edges of β sheets 1–3 (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Hamdan <em>et al.</em> 2002</a>).</p><p id="p-33">The εCTS associates with the α subunit, and the two proteins can be cross-linked via residue 235 in the εCTS (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Perrino <em>et al.</em> 1999</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Taft-Benz and Schaaper 1999</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Ozawa <em>et al.</em> 2008</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Toste Rego <em>et al.</em> 2013</a>). Residues 218–237 form an α helix (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Ozawa <em>et al.</em> 2013</a>), which is the site of insertion mutant <sup>232</sup>LFKQV. Given the location of this insertion, outside of the proofreading NTD, we surmise that it either disrupts the ability of ε to bind to α or results in an unstable protein. A similar inference seems likely for insertion mutant <sup>224</sup>AHV*, which truncates the εCTS, including the aforementioned α helix.</p></div><div id="sec-14" class="subsection"><h3>Proofreading can be uncoupled from SOS phenotypes</h3><p id="p-34">We next turn to several apparent separations of function mutants that were uncovered. When comparing mutation rates to the basal (no drug) SOS levels, most of the mutants behaved similarly to wild-type or similarly to the ε knockout, or were somewhere in between as partially functional mutants would do (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>). We found no mutants with complete separation of function, namely high mutator/low basal SOS or low mutator/partially constitutive SOS (far left and far right of <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>, respectively). However, a number of mutants did exhibit moderate mutator activity (reduced proofreading) yet showed wild-type levels of basal SOS expression (middle of <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>): G17S (<em>dnaQ928</em>); <sup>76</sup>CLNNK; <sup>143</sup>NSCLN; <sup>228</sup>CLNKA; and <sup>57</sup>LVFKQ. Although G17S and <sup>143</sup>NSCLN are in motifs exo I and exo IIIε, respectively, the other insertions are not within the conserved residues related to proofreading. Insertion <sup>57</sup>LVFKQ lies between motifs exo I and exo II; therefore, the behavior of this mutant aligns with a report that implicates this region for full proofreading function (<a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Taft-Benz and Schaaper 1998</a>). The moderate mutator phenotype of mutant <sup>228</sup>CLNKA, which has an insertion within the εCTS a helix, could reflect altered binding of ε to α (see above).</p><p id="p-35">Mutants with moderate mutator but low basal SOS could be explained by two models: (1) SOS is induced only by extreme levels of misincorporation or (2) the two phenotypes are caused by two different functions of ε. Further experiments are necessary to distinguish between these two models. It is worth noting that the SOS constitutive phenotype of ε knockout mutants is rather modest (<em>dinD</em>::<em>lacZ</em> expression increased by only approximately 50%; see <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>), precluding strong conclusions based on intermediate levels of basal SOS expression.</p><p id="p-36">Considering the relationship between proofreading and the ability to induce SOS after nalidixic acid treatment, many of the mutants again behaved like either wild-type or the ε knockout mutant, or somewhere in between as partially functional mutants might do (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>). However, we found two striking separation of function mutants—insertion mutants <sup>208</sup>VFKHQ and <sup>143</sup>NSCLN (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>).</p><p id="p-37">Insertion mutant <sup>208</sup>VFKHQ displayed fully functional proofreading but no ability to induce SOS after nalidixic acid exposure (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3</a>). This insertion is within the εCTS, outside of the proofreading domain. Because the proofreading function appears normal, the insertion presumably does not abolish the interaction of α with εCTS (although it may alter the consequences of the interaction; see <em>Further discussion</em>). Mutant Q182A has a somewhat similar phenotype to <sup>208</sup>VFKHQ and is discussed below.</p><p id="p-38">Insertion mutant <sup>143</sup>NSCLN shows a different kind of separation of function, with a 22-fold increase in mutation rate but a robust ability to induce SOS following nalidixic acid exposure (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3</a>). This mutant has an insertion within Exo IIIε, consistent with its mutator phenotype. The ability of the <sup>143</sup>NSCLN mutant to induce SOS after nalidixic acid treatment indicates that the insertion does not disrupt whatever interaction is necessary for this function (see <em>Further discussion</em>). <sup>143</sup>NSCLN mutant appeared to induce somewhat higher levels of SOS with nalidixic acid treatment than the wild-type. Likewise, insertion mutant <sup>162</sup>LFKQL, with a nearly wild-type mutation rate, induced SOS after nalidixic acid at a level that was significantly (74%) higher than that of the wild-type (<em>P</em> = 0.0064). These two mutants appear to generate a novel phenotype of <em>dnaQ</em> mutants hyper-responsive to nalidixic acid–induced cleavage complexes (see <em>Further discussion</em>).</p></div><div id="sec-15" class="subsection"><h3>Relationship of constitutive SOS to nalidixic acid–induced SOS</h3><p id="p-39">We next discuss the comparison of basal SOS levels with the increase in SOS following nalidixic acid treatment; in this regard, the mutants scattered widely (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Considering mutants with little or no ability to induce SOS following nalidixic acid treatment, phenotypes ranged from basal SOS similar to wild-type (Q<sup>182</sup>A, <sup>76</sup>CLNNK, and <sup>208</sup>VFKHQ; separation of function for SOS) to significantly higher constitutive levels than the ε knockout (<sup>30</sup>IICLN and <sup>232</sup>LFKQV). We also identified partially constitutive mutants that are able to induce SOS following nalidixic acid treatment (<sup>162</sup>LFKQL and QLSLPL; discussed above and below, respectively).</p></div><div id="sec-16" class="subsection"><h3>Site-directed mutants</h3><p id="p-40">The β-clamp binding mutants appear to be separation of function alleles. QLSLPL, the strong β-clamp binding mutant, retains catalytic proofreading function as expected but showed a basal SOS level significantly higher than wild-type (<em>P</em> = 0.0000134) and closer to the ε knockout (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2</a>; <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). This mutant also demonstrates separation of SOS phenotypes, because it shows a robust induction of SOS with nalidixic acid treatment (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>; <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). This mutant ε was shown to have a stronger ε–β interaction than wild-type (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Jergic <em>et al.</em> 2013</a>), which could provide a barrier to dissociation of the replisome on encounter with endogenous damage, potentially elevating basal SOS (see <em>Further discussion</em>).</p><p id="p-41">The weak β-clamp binding mutant, Q182A, revealed a particularly interesting collection of phenotypes. It demonstrated weak mutator activity (approximately five-fold higher mutation rate than that of wild-type), along with a basal SOS level that was significantly <em>lower</em> than that of wild-type (<em>P</em> = 0.0042) and greatly reduced induction of SOS following nalidixic acid treatment. These results implicate the ε-β clamp interaction in the SOS phenotypes caused by the absence of ε (see <em>Further discussion</em>).</p></div><div id="sec-17" class="subsection"><h3>Further discussion</h3><p id="p-42">The multiple phenotypes of <em>dnaQ</em> knockout mutants are consistent with the possibility that ε has multiple functions. This possibility was raised decades ago, supported by the isolation of suppressor mutations within <em>dnaE</em> that reverse the growth deficiency but not the proofreading defect of a <em>dnaQ</em> mutant (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Lancy <em>et al.</em> 1989</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Lifsics <em>et al.</em> 1992</a>). The isolation of separation of function mutants in this report strongly supports the proposal of multiple ε functions (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). Certain separation of function mutants isolated here also provide a very useful reagent, namely ε variants with defects in functions relating to SOS induction but wild-type mutation rates, <em>i.e.</em>, mutationally stable strains that do not produce confounding results due to spurious global mutations. Our data also identify sites in ε (residues 115, 125, 196) that are tolerant of five amino-acid insertions with no phenotypic consequence.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466224/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466224/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Summary of separation of function mutants</span><div class="sb-div caption-clear"></div></div></div><p id="p-44">These results can be viewed through the lens of recent studies on the clamp binding motif of ε and the interactions between clamp, ε, and α (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Jergic <em>et al.</em> 2013</a>; <a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Toste Rego <em>et al.</em> 2013</a>). <a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Toste Rego <em>et al.</em> (2013)</a> analyzed the interactions of the three proteins, demonstrating that ε enhances the stability of the clamp–α complex and identifying sites where each pair of proteins can be cross-linked to each other. The increase in stability of the complex due to ε binding is presumably related to the stimulation of polymerase processivity by ε (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Studwell and O’Donnell 1990</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kim and Mchenry 1996</a>), and clamp binding by ε, in turn, improves proofreading ability (<a id="xref-ref-48-5" class="xref-bibr" href="#ref-48">Toste Rego <em>et al.</em> 2013</a>). Based on these and other results, <a id="xref-ref-48-6" class="xref-bibr" href="#ref-48">Toste Rego <em>et al.</em> (2013)</a> developed a model in which α and ε simultaneously bind to the canonical binding pocket of the two protomers of the β clamp dimer. Importantly, they also identified cross-links between α and both the proofreading NTD of ε and the extreme ε C-terminus (residue 235). These results led to an elegant model for how the polymerase complex responds to both template damage and misincorporation.</p><p id="p-45">As mentioned above, misincorporated bases are more efficiently proofread by ε in the context of the tripartite complex. Template damage, however, cannot be corrected by ε, but rather is proposed to cause weakening of the clamp–ε interaction, which can allow binding of the translesion polymerase Pol IV. One important feature of the model is that ε remains associated with α via the flexible εCTS even when the ε–clamp association is lost. Several aspects of this model could relate to the phenotypes of ε mutants analyzed here, and we propose that the dynamic behavior of the tripartite complex described above could directly relate to the response of the replisome to endogenous lesions and to blocking gyrase cleavage complexes.</p><p id="p-46">As the tripartite complex encounters endogenous damage, the weak clamp binder (Q<sup>182</sup>A) would be expected to more readily dissociate from the clamp, allowing translesion polymerases to access the free site on the β clamp. Q<sup>182</sup>A has weak mutator activity (five-fold higher than wild-type), and we propose two mechanisms that could lead to this. One scenario is that Q182A has weaker exonuclease activity, and this leads to less efficient proofreading, hence the weak mutator activity observed. The other scenario is that the weak binder allows Pol IV to access the β clamp more often, with or without a lesion present. Pol IV is a low-fidelity polymerase, and this occasional access could be the cause of the weak mutator activity. If the weak clamp binder allows the translesion polymerase to associate more easily, it could synthesize past the lesion and dissociate, then Q<sup>182</sup>A ε (still bound to α) would regain access to the clamp to continue high-fidelity replication without inducing SOS. More efficient TLS across the endogenous template damage is thus proposed to reduce basal SOS levels, which is observed for Q<sup>182</sup>A.</p><p id="p-47">Under these same assumptions, the strong clamp binder, QLSLPL, would remain bound to the β clamp, preventing access by the translesion polymerase. This mutant would not easily allow the translesion polymerase to access the β clamp. This is consistent with the mutation rate similar to that of wild-type observed with this mutant. When facing endogenous damage, the replication fork with ε tightly bound to α would stall, because the translesion polymerase does not have access to the β clamp in QLSLPL and the damage could not be readily bypassed. This prolonged polymerase stalling could result in increased SOS induction, explaining the constitutive SOS phenotype of QLSLPL.</p><p id="p-48">Our results indicate that the ε–β clamp interaction is required for nalidixic acid–induced SOS, such that the weak clamp binder leads to defective SOS induction following nalidixic acid. These results are consistent with a particular version of the replication run-off model. We know that quinolone-induced SCCs cause replication fork stalling, and stalling presumably leads to DnaB helicase unloading. We propose that the tripartite polymerase complex usually cannot proceed to the SCC at this stage, but occasionally moves forward, encountering and disrupting the SCC. The resulting broken arm of the replication fork creates a double-stranded end for RecBC to resect, which in turn is recognized by RecA for SOS induction.</p><p id="p-49">As outlined above, the weak clamp mutant (Q182A) more easily allows translesion polymerases to associate with the tripartite complex. We propose that this weaker, less-processive tripartite complex would not have the ability to move forward and disrupt the SCC. Thus, the weak clamp binder does not create a break or double-strand end for RecBC, leading to the defect in nalidixic acid-induced SOS.</p><p id="p-50">With the strong clamp-binding mutant (QLSLPL), the tripartite complex is tightly associated, showing even higher processivity than the wild-type complex (<a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Jergic <em>et al.</em> 2013</a>). According to this model, this mutant complex moves forward efficiently, readily disrupting the SCC and thereby generating double-strand ends and inducing SOS efficiently (equal to or perhaps slightly higher than wild-type; see <a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3</a>).</p><p id="p-51">In this model, an ε knockout causes a defect in nalidixic acid–induced SOS response because ε is required for sufficient processivity to disrupt the SCC. In several of our SOS-defective insertion mutants (<sup>30</sup>IICLN, <sup>79</sup>CLNTF, <sup>132</sup>V*, <sup>224</sup>AHV*, and <sup>232</sup>LFKQV), the high mutators are likely knockouts due to protein instability, misfolding, and/or inability to associate with α at the tripartite complex, eliminating all functions of ε.</p><p id="p-52">This replication run-off model can explain some of the striking separation of function mutants. For example, <sup>162</sup>LFKQL behaves similar to the strong clamp binder; it is a low mutator (consistent with preventing Pol IV access to the clamp), SOS constitutive, and able to induce SOS after nalidixic acid at levels even higher than that of wild-type. Like the weak β-clamp mutant Q182A, <sup>76</sup>CLNNK has modestly higher mutation rate and lower nalidixic acid–induced SOS levels than the wild-type (with basal SOS levels in between those of Q182A and wild-type). Biochemical characterization of these various <em>dnaQ</em> mutants could directly test whether they affect the tripartite complex as predicted and whether disruption of the SCC occurs in the manner predicted by the replication run-off model.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-53">We would like to thank Meta Kuehn and Sue Jinks-Robertson for insightful discussions and feedback. This work was supported by National Institutes of Health grant RO1-GM72089. Z.W. was supported in part by National Institutes of Health training grant 5T32-GM007184.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-54"><em>Communicating editor: R. A. Sclafani</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-02-06"><span class="received-label">Received </span>February 6, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-04-09"><span class="accepted-label">Accepted </span>April 9, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Whatley and Kreuzer</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.1" data-doi="10.1128/JB.186.17.5699-5707.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anton</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raleigh</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Transposon-mediated linker insertion scanning mutagenesis of the <em>Escherichia coli</em> McrA endonuclease.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">5699</span>–<span class="cit-lpage">5707</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DAnton%26rft.auinit1%253DB.%2BP.%26rft.volume%253D186%26rft.issue%253D17%26rft.spage%253D5699%26rft.epage%253D5707%26rft.atitle%253DTransposon-Mediated%2BLinker%2BInsertion%2BScanning%2BMutagenesis%2Bof%2Bthe%2BEscherichia%2Bcoli%2BMcrA%2BEndonuclease%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.186.17.5699-5707.2004%26rft_id%253Dinfo%253Apmid%252F15317774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE4Ni8xNy81Njk5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.2" data-doi="10.1038/msb4100050"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baba</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ara</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hasegawa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takai</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okumura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Construction of Escherichia coli K-12 in-frame, single-gene knockout mutants: the Keio collection.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BSyst.%2BBiol.%26rft.volume%253D2%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb4100050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/msb4100050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.3" data-doi="10.1016/0378-1119(95)00530-J"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barnes</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spacciapoli</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">N. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">The 3′-5′ exonuclease site of DNA polymerase III from gram-positive bacteria: definition of a novel motif structure.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">45</span>–<span class="cit-lpage">50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DBarnes%26rft.auinit1%253DM.%2BH.%26rft.volume%253D165%26rft.issue%253D1%26rft.spage%253D45%26rft.epage%253D50%26rft.atitle%253DThe%2B3%2527-5%2527%2Bexonuclease%2Bsite%2Bof%2BDNA%2Bpolymerase%2BIII%2Bfrom%2Bgram-positive%2Bbacteria%253A%2Bdefinition%2Bof%2Ba%2Bnovel%2Bmotif%2Bstructure.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252895%252900530-J%26rft_id%253Dinfo%253Apmid%252F7489914%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(95)00530-J&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7489914&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995TF52400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.4" data-doi="10.1093/nar/28.1.235"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berman</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Westbrook</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilliland</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhat</span>  <span class="cit-name-given-names">T. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The Protein Data Bank.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">235</span>–<span class="cit-lpage">242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DBerman%26rft.auinit1%253DH.%2BM.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D235%26rft.epage%253D242%26rft.atitle%253DThe%2BProtein%2BData%2BBank%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.1.235%26rft_id%253Dinfo%253Apmid%252F10592235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjI4LzEvMjM1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.5" data-doi="10.1016/0092-8674(89)90883-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernad</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanco</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lazaro</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salas</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">A conserved 3′—-5′ exonuclease active site in prokaryotic and eukaryotic DNA polymerases.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">219</span>–<span class="cit-lpage">228</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBernad%26rft.auinit1%253DA.%26rft.volume%253D59%26rft.issue%253D1%26rft.spage%253D219%26rft.epage%253D228%26rft.atitle%253DA%2Bconserved%2B3%2527----5%2527%2Bexonuclease%2Bactive%2Bsite%2Bin%2Bprokaryotic%2Band%2Beukaryotic%2BDNA%2Bpolymerases.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252889%252990883-0%26rft_id%253Dinfo%253Apmid%252F2790959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(89)90883-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2790959&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989AU52300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.6" data-doi="10.1016/0378-1119(92)90316-H"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blanco</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernad</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salas</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Evidence favouring the hypothesis of a conserved 3′-5′ exonuclease active site in DNA-dependent DNA polymerases.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DBlanco%26rft.auinit1%253DL.%26rft.volume%253D112%26rft.issue%253D1%26rft.spage%253D139%26rft.epage%253D144%26rft.atitle%253DEvidence%2Bfavouring%2Bthe%2Bhypothesis%2Bof%2Ba%2Bconserved%2B3%2527-5%2527%2Bexonuclease%2Bactive%2Bsite%2Bin%2BDNA-dependent%2BDNA%2Bpolymerases.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252892%252990316-H%26rft_id%253Dinfo%253Apmid%252F1551594%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(92)90316-H&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1551594&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HL48900022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.7" data-doi="10.1016/j.bbapap.2009.07.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bressanin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stefan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piaz</span>  <span class="cit-name-given-names">F. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cianchetta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reggiani</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Proteolysis of the proofreading subunit controls the assembly of <em>Escherichia coli</em> DNA polymerase III catalytic core.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1794</span>: <span class="cit-fpage">1606</span>–<span class="cit-lpage">1615</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DBressanin%26rft.auinit1%253DD.%26rft.volume%253D1794%26rft.issue%253D11%26rft.spage%253D1606%26rft.epage%253D1615%26rft.atitle%253DProteolysis%2Bof%2Bthe%2Bproofreading%2Bsubunit%2Bcontrols%2Bthe%2Bassembly%2Bof%2BEscherichia%2Bcoli%2BDNA%2Bpolymerase%2BIII%2Bcatalytic%2Bcore.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbapap.2009.07.011%26rft_id%253Dinfo%253Apmid%252F19635595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbapap.2009.07.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19635595&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.8" data-doi="10.1006/jmbi.1997.1380"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hallet</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherratt</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Structure-function correlations in the XerD site-specific recombinase revealed by pentapeptide scanning mutagenesis.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">274</span>: <span class="cit-fpage">39</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DCao%26rft.auinit1%253DY.%26rft.volume%253D274%26rft.issue%253D1%26rft.spage%253D39%26rft.epage%253D53%26rft.atitle%253DStructure-function%2Bcorrelations%2Bin%2Bthe%2BXerD%2Bsite-specific%2Brecombinase%2Brevealed%2Bby%2Bpentapeptide%2Bscanning%2Bmutagenesis.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1997.1380%26rft_id%253Dinfo%253Apmid%252F9398514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1997.1380&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9398514&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Degnen</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">E. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">Conditional mutator gene in <em>Escherichia coli</em>: isolation, mapping, and effector studies.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">477</span>–<span class="cit-lpage">487</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DDegnen%26rft.auinit1%253DG.%2BE.%26rft.volume%253D117%26rft.issue%253D2%26rft.spage%253D477%26rft.epage%253D487%26rft.atitle%253DConditional%2BMutator%2BGene%2Bin%2BEscherichia%2Bcoli%253A%2BIsolation%252C%2BMapping%252C%2Band%2BEffector%2BStudies%26rft_id%253Dinfo%253Apmid%252F4590472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTE3LzIvNDc3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.10" data-doi="10.1021/bi0205451"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DeRose</span>  <span class="cit-name-given-names">E. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Darden</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harvey</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perrino</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Elucidation of the epsilon-theta subunit interface of <em>Escherichia coli</em> DNA polymerase III by NMR spectroscopy.</span> <abbr class="cit-jnl-abbrev">Biochemistry</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">3635</span>–<span class="cit-lpage">3644</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D42%26rft.spage%253D3635%26rft.atitle%253DELUCIDATION%2BOF%2BTHE%2B%2B-%2B%2BINTERFACE%2BOF%2BESCHERICHIA%2BCOLI%2BDNA%2BPOLYMERASE%2BIII%2BBY%2BNMR%2BSPECTROSCOPY%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi0205451%26rft_id%253Dinfo%253Apmid%252F12667053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi0205451&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12667053&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.11" data-doi="10.1128/AAC.01617-06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drlica</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malik</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerns</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Quinolone-mediated bacterial cell death.</span> <abbr class="cit-jnl-abbrev">Antimicrob. Agents Chemother.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">385</span>–<span class="cit-lpage">392</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntimicrob.%2BAgents%2BChemother.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAAC.01617-06%26rft_id%253Dinfo%253Apmid%252F17724149%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWFjIjtzOjU6InJlc2lkIjtzOjg6IjUyLzIvMzg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.12" data-doi="10.1073/pnas.80.8.2189"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Echols</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgers</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Mutator strains of <em>Escherichia coli</em>, <em>mutD</em> and <em>dnaQ</em>, with defective exonucleolytic editing by DNA polymerase III holoenzyme.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">2189</span>–<span class="cit-lpage">2192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DEchols%26rft.auinit1%253DH.%26rft.volume%253D80%26rft.issue%253D8%26rft.spage%253D2189%26rft.epage%253D2192%26rft.atitle%253DMutator%2Bstrains%2Bof%2BEscherichia%2Bcoli%252C%2BmutD%2Band%2BdnaQ%252C%2Bwith%2Bdefective%2Bexonucleolytic%2Bediting%2Bby%2BDNA%2Bpolymerase%2BIII%2Bholoenzyme.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.80.8.2189%26rft_id%253Dinfo%253Apmid%252F6340117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI4MC84LzIxODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTMwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.13" data-doi="10.1128/jb.177.20.5979-5986.1995"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fijalkowska</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaaper</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Effects of <em>Escherichia coli dnaE</em> Antimutator Alleles in a Proofreading-Deficient <em>mutD5</em> Strain.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">5979</span>–<span class="cit-lpage">5986</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DFijalkowska%26rft.auinit1%253DI.%2BJ.%26rft.volume%253D177%26rft.issue%253D20%26rft.spage%253D5979%26rft.epage%253D5986%26rft.atitle%253DEffects%2Bof%2BEscherichia%2Bcoli%2BdnaE%2Bantimutator%2Balleles%2Bin%2Ba%2Bproofreading-deficient%2BmutD5%2Bstrain.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.177.20.5979-5986.1995%26rft_id%253Dinfo%253Apmid%252F7592352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE3Ny8yMC81OTc5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.14" data-doi="10.1073/pnas.93.7.2856"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fijalkowska</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaaper</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Mutants in the Exo I motif of <em>Escherichia coli dnaQ</em>: defective proofreading and inviability due to error catastrophe.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">2856</span>–<span class="cit-lpage">2861</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DFijalkowska%26rft.auinit1%253DI.%2BJ.%26rft.volume%253D93%26rft.issue%253D7%26rft.spage%253D2856%26rft.epage%253D2861%26rft.atitle%253DMutants%2Bin%2Bthe%2BExo%2BI%2Bmotif%2Bof%2BEscherichia%2Bcoli%2BdnaQ%253A%2Bdefective%2Bproofreading%2Band%2Binviability%2Bdue%2Bto%2Berror%2Bcatastrophe.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.93.7.2856%26rft_id%253Dinfo%253Apmid%252F8610131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5My83LzI4NTYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTMwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.15" data-doi="10.1016/S0076-6879(05)09012-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foster</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Methods for determining spontaneous mutation rates.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">409</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DFoster%26rft.auinit1%253DP.%2BL.%26rft.volume%253D409%26rft.spage%253D195%26rft.epage%253D213%26rft.atitle%253DMethods%2Bfor%2Bdetermining%2Bspontaneous%2Bmutation%2Brates.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0076-6879%252805%252909012-9%26rft_id%253Dinfo%253Apmid%252F16793403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0076-6879(05)09012-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16793403&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238354800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.16" data-doi="10.1016/S0969-2126(02)00738-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamdan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carr</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ollis</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dixon</span>  <span class="cit-name-given-names">N. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Structural basis for proofreading during replication of the <em>Escherichia coli</em> chromosome.</span> <abbr class="cit-jnl-abbrev">Structure</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">535</span>–<span class="cit-lpage">546</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStructure%2B%2528London%2529%26rft.stitle%253DStructure%2B%2528London%2529%26rft.aulast%253DHamdan%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.issue%253D4%26rft.spage%253D535%26rft.epage%253D546%26rft.atitle%253DStructural%2Bbasis%2Bfor%2Bproofreading%2Bduring%2Breplication%2Bof%2Bthe%2BEscherichia%2Bcoli%2Bchromosome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0969-2126%252802%252900738-4%26rft_id%253Dinfo%253Apmid%252F11937058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0969-2126(02)00738-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11937058&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-2" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.17" data-doi="10.1128/jb.171.9.4617-4622.1989"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aldea</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Washburn</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babitzke</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kushner</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">New method for generating deletions and gene replacements in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">4617</span>–<span class="cit-lpage">4622</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DHamilton%26rft.auinit1%253DC.%2BM.%26rft.volume%253D171%26rft.issue%253D9%26rft.spage%253D4617%26rft.epage%253D4622%26rft.atitle%253DNew%2Bmethod%2Bfor%2Bgenerating%2Bdeletions%2Band%2Bgene%2Breplacements%2Bin%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.171.9.4617-4622.1989%26rft_id%253Dinfo%253Apmid%252F2548993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE3MS85LzQ2MTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTMwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.18" data-doi="10.1007/BF00271956"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Horiuchi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sekiguchi</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">A new conditional lethal mutator (<em>dnaQ49</em>) in <em>Escherichia coli</em> K12.</span> <abbr class="cit-jnl-abbrev">Mol. Gen. Genet.</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">277</span>–<span class="cit-lpage">283</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bgeneral%2Bgenetics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGen%2BGenet%26rft.aulast%253DHoriuchi%26rft.auinit1%253DT.%26rft.volume%253D163%26rft.issue%253D3%26rft.spage%253D277%26rft.epage%253D283%26rft.atitle%253DA%2Bnew%2Bconditional%2Blethal%2Bmutator%2B%2528dnaQ49%2529%2Bin%2BEscherichia%2Bcoli%2BK12.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00271956%26rft_id%253Dinfo%253Apmid%252F355854%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00271956&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=355854&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1978FK99700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.19" data-doi="10.1038/emboj.2012.347"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jergic</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horan</span>  <span class="cit-name-given-names">N. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshenawy</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mason</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Urathamakul</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A direct proofreader-clamp interaction stabilizes the Pol III replicase in the polymerization mode.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1322</span>–<span class="cit-lpage">1333</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Femboj.2012.347%26rft_id%253Dinfo%253Apmid%252F23435564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo5OiIzMi85LzEzMjIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTMwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jonczyk</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nowicka</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fijalkowska</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaaper</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ciesla</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">In vivo protein interactions within the <em>Escherichia coli</em> DNA polymerase III core.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">1563</span>–<span class="cit-lpage">1566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DJonczyk%26rft.auinit1%253DP.%26rft.volume%253D180%26rft.issue%253D6%26rft.spage%253D1563%26rft.epage%253D1566%26rft.atitle%253DIn%2BVivo%2BProtein%2BInteractions%2Bwithin%2Bthe%2BEscherichia%2Bcoli%2BDNA%2BPolymerase%2BIII%2BCore%26rft_id%253Dinfo%253Apmid%252F9515927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE4MC82LzE1NjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTMwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.21" data-doi="10.1128/JB.01992-05"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keniry</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">A. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Owen</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamdan</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pintacuda</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Structure of the theta subunit of <em>Escherichia coli</em> DNA polymerase III in complex with the epsilon subunit.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">4464</span>–<span class="cit-lpage">4473</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DKeniry%26rft.auinit1%253DM.%2BA.%26rft.volume%253D188%26rft.issue%253D12%26rft.spage%253D4464%26rft.epage%253D4473%26rft.atitle%253DStructure%2Bof%2Bthe%2B%257Btheta%257D%2BSubunit%2Bof%2BEscherichia%2Bcoli%2BDNA%2BPolymerase%2BIII%2Bin%2BComplex%2Bwith%2Bthe%2B%257Bvarepsilon%257D%2BSubunit%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.01992-05%26rft_id%253Dinfo%253Apmid%252F16740953%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE4OC8xMi80NDY0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.22" data-doi="10.1074/jbc.271.34.20681"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McHenry</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">In vivo assembly of overproduced DNA polymerase III. Overproduction, purification, and characterization of the alpha, alpha-epsilon, and alpha-epsilon-theta subunits.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">271</span>: <span class="cit-fpage">20681</span>–<span class="cit-lpage">20689</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DKim%26rft.auinit1%253DD.%2BR.%26rft.volume%253D271%26rft.issue%253D34%26rft.spage%253D20681%26rft.epage%253D20689%26rft.atitle%253DIn%2BVivo%2BAssembly%2Bof%2BOverproduced%2BDNA%2BPolymerase%2BIII%253A%2BOVERPRODUCTION%252C%2BPURIFICATION%252C%2BAND%2BCHARACTERIZATION%2BOF%2BTHE%2B%257Balpha%257D%252C%2B%257Balpha%257D-%257Bvarepsilon%257D%252C%2BAND%2B%257Balpha%257D-%257Bvarepsilon%257D-%257Btheta%257D%2BSUBUNITS%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.271.34.20681%26rft_id%253Dinfo%253Apmid%252F8702818%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzEvMzQvMjA2ODEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTMwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.23" data-doi="10.1128/jb.171.10.5572-5580.1989"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lancy</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lifsics</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kehres</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Isolation and characterization of mutants with deletions in <em>dnaQ</em>, the gene for the editing subunit of DNA polymerase III in <em>Salmonella typhimurium</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">5572</span>–<span class="cit-lpage">5580</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DLancy%26rft.auinit1%253DE.%2BD.%26rft.volume%253D171%26rft.issue%253D10%26rft.spage%253D5572%26rft.epage%253D5580%26rft.atitle%253DIsolation%2Band%2Bcharacterization%2Bof%2Bmutants%2Bwith%2Bdeletions%2Bin%2BdnaQ%252C%2Bthe%2Bgene%2Bfor%2Bthe%2Bediting%2Bsubunit%2Bof%2BDNA%2Bpolymerase%2BIII%2Bin%2BSalmonella%2Btyphimurium.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.171.10.5572-5580.1989%26rft_id%253Dinfo%253Apmid%252F2551891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE3MS8xMC81NTcyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.24" data-doi="10.1128/jb.174.21.6965-6973.1992"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lifsics</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lancy</span>  <span class="cit-name-given-names">E. D.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">DNA replication defect in <em>Salmonella typhimurium</em> mutants lacking the editing (epsilon) subunit of DNA polymerase III.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">6965</span>–<span class="cit-lpage">6973</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DLifsics%26rft.auinit1%253DM.%2BR.%26rft.volume%253D174%26rft.issue%253D21%26rft.spage%253D6965%26rft.epage%253D6973%26rft.atitle%253DDNA%2Breplication%2Bdefect%2Bin%2BSalmonella%2Btyphimurium%2Bmutants%2Blacking%2Bthe%2Bediting%2B%2528epsilon%2529%2Bsubunit%2Bof%2BDNA%2Bpolymerase%2BIII.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.174.21.6965-6973.1992%26rft_id%253Dinfo%253Apmid%252F1400246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE3NC8yMS82OTY1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-5.6.1301.25" data-doi="10.1073/pnas.80.23.7137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horiuchi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sekiguchi</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Structure and expression of the <em>dnaQ</em> mutator and the RNase H genes of <em>Escherichia coli</em>: overlap of the promoter regions.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">7137</span>–<span class="cit-lpage">7141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DMaki%26rft.auinit1%253DH.%26rft.volume%253D80%26rft.issue%253D23%26rft.spage%253D7137%26rft.epage%253D7141%26rft.atitle%253DStructure%2Band%2Bexpression%2Bof%2Bthe%2BdnaQ%2Bmutator%2Band%2Bthe%2BRNase%2BH%2Bgenes%2Bof%2BEscherichia%2Bcoli%253A%2Boverlap%2Bof%2Bthe%2Bpromoter%2Bregions.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.80.23.7137%26rft_id%253Dinfo%253Apmid%252F6316347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODAvMjMvNzEzNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMzAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McDaniel</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">L. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">Survival of recombination-deficient mutants of <em>Escherichia coli</em> during incubation with nalidixic acid.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">1195</span>–<span class="cit-lpage">1198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DMcDaniel%26rft.auinit1%253DL%2BS%26rft.volume%253D134%26rft.issue%253D3%26rft.spage%253D1195%26rft.epage%253D1198%26rft.atitle%253DSurvival%2Bof%2Brecombination-deficient%2Bmutants%2Bof%2BEscherichia%2Bcoli%2Bduring%2Bincubation%2Bwith%2Bnalidixic%2Bacid.%26rft_id%253Dinfo%253Apmid%252F350844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjEzNC8zLzExOTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTMwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.27" data-doi="10.1016/0092-8674(80)90319-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McPartland</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Echols</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Control of <em>recA</em> gene RNA in <em>E. coli</em>: regulatory and signal genes.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">731</span>–<span class="cit-lpage">737</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMcPartland%26rft.auinit1%253DA.%26rft.volume%253D20%26rft.issue%253D3%26rft.spage%253D731%26rft.epage%253D737%26rft.atitle%253DControl%2Bof%2BrecA%2Bgene%2BRNA%2Bin%2BE.%2Bcoli%253A%2Bregulatory%2Band%2Bsignal%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252880%252990319-0%26rft_id%253Dinfo%253Apmid%252F6998563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(80)90319-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6998563&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1980JZ69100015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.28" data-doi="10.1016/j.gene.2005.04.029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Newmark</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Reilly</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pohlhaus</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreuzer</span>  <span class="cit-name-given-names">K. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genetic analysis of the requirements for SOS induction by nalidixic acid in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">356</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">76</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DNewmark%26rft.auinit1%253DK.%2BG.%26rft.volume%253D356%26rft.spage%253D69%26rft.epage%253D76%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Bthe%2Brequirements%2Bfor%2BSOS%2Binduction%2Bby%2Bnalidixic%2Bacid%2Bin%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2005.04.029%26rft_id%253Dinfo%253Apmid%252F16005583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2005.04.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16005583&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231750700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.29" data-doi="10.1093/nar/gkt162"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ozawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horan</span>  <span class="cit-name-given-names">N. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yagi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">F. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Proofreading exonuclease on a tether: the complex between the <em>E. coli</em> DNA polymerase III subunits alpha, epsilon, theta and beta reveals a highly flexible arrangement of the proofreading domain.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">5354</span>–<span class="cit-lpage">5367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt162%26rft_id%253Dinfo%253Apmid%252F23580545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MS8xMC81MzU0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.30" data-doi="10.1093/nar/gkn489"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ozawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jergic</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">A. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dixon</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Otting</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The proofreading exonuclease subunit epsilon of <em>Escherichia coli</em> DNA polymerase III is tethered to the polymerase subunit alpha via a flexible linker.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">5074</span>–<span class="cit-lpage">5082</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn489%26rft_id%253Dinfo%253Apmid%252F18663010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzNi8xNS81MDc0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.31" data-doi="10.1021/bi991429+"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perrino</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harvey</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McNeill</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Two functional domains of the epsilon subunit of DNA polymerase III.</span> <abbr class="cit-jnl-abbrev">Biochemistry</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">16001</span>–<span class="cit-lpage">16009</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D38%26rft.spage%253D16001%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi991429%252B%26rft_id%253Dinfo%253Apmid%252F10625468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi991429+&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10625468&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084115400035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.32" data-doi="10.1111/j.1365-2958.2005.04638.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pohlhaus</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreuzer</span>  <span class="cit-name-given-names">K. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Norfloxacin-induced DNA gyrase cleavage complexes block <em>Escherichia coli</em> replication forks, causing double-stranded breaks in vivo.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">1416</span>–<span class="cit-lpage">1429</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DPohlhaus%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D56%26rft.issue%253D6%26rft.spage%253D1416%26rft.epage%253D1429%26rft.atitle%253DNorfloxacin-induced%2BDNA%2Bgyrase%2Bcleavage%2Bcomplexes%2Bblock%2BEscherichia%2Bcoli%2Breplication%2Bforks%252C%2Bcausing%2Bdouble-stranded%2Bbreaks%2Bin%2Bvivo.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2005.04638.x%26rft_id%253Dinfo%253Apmid%252F15916595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2005.04638.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15916595&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229181800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.33" data-doi="10.1128/JB.00173-08"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pohlhaus</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Reilly</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreuzer</span>  <span class="cit-name-given-names">K. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The epsilon subunit of DNA polymerase III is involved in the nalidixic acid-induced SOS response in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">5239</span>–<span class="cit-lpage">5247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBacteriol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.00173-08%26rft_id%253Dinfo%253Apmid%252F18539731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE5MC8xNS81MjM5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saveson</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lovett</span>  <span class="cit-name-given-names">S. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Enhanced deletion formation by aberrant DNA replication in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">457</span>–<span class="cit-lpage">470</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSaveson%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D146%26rft.issue%253D2%26rft.spage%253D457%26rft.epage%253D470%26rft.atitle%253DEnhanced%2BDeletion%2BFormation%2Bby%2BAberrant%2BDNA%2BReplication%2Bin%2BEscherichia%2Bcoli%26rft_id%253Dinfo%253Apmid%252F9177997%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ2LzIvNDU3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.35" data-doi="10.1073/pnas.85.21.8126"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaaper</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Mechanisms of mutagenesis in the <em>Escherichia coli</em> mutator <em>mutD5</em>: role of DNA mismatch repair.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">8126</span>–<span class="cit-lpage">8130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSchaaper%26rft.auinit1%253DR.%2BM.%26rft.volume%253D85%26rft.issue%253D21%26rft.spage%253D8126%26rft.epage%253D8130%26rft.atitle%253DMechanisms%2Bof%2Bmutagenesis%2Bin%2Bthe%2BEscherichia%2Bcoli%2Bmutator%2BmutD5%253A%2Brole%2Bof%2BDNA%2Bmismatch%2Brepair.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.85.21.8126%26rft_id%253Dinfo%253Apmid%252F3054881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODUvMjEvODEyNiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMzAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaaper</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title"><em>Escherichia coli</em> mutator <em>mutD5</em> is defective in the <em>mutHLS</em> pathway of DNA mismatch repair.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">205</span>–<span class="cit-lpage">212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSchaaper%26rft.auinit1%253DR.%2BM.%26rft.volume%253D121%26rft.issue%253D2%26rft.spage%253D205%26rft.epage%253D212%26rft.atitle%253DEscherichia%2Bcoli%2Bmutator%2BmutD5%2Bis%2Bdefective%2Bin%2Bthe%2BmutHLS%2Bpathway%2Bof%2BDNA%2Bmismatch%2Brepair.%26rft_id%253Dinfo%253Apmid%252F2659431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIxLzIvMjA1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.37" data-doi="10.1128/jb.174.6.1974-1982.1992"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaaper</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cornacchio</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">An <em>Escherichia coli dnaE</em> mutation with suppressor activity toward mutator <em>mutD5</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">1974</span>–<span class="cit-lpage">1982</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DSchaaper%26rft.auinit1%253DR.%2BM.%26rft.volume%253D174%26rft.issue%253D6%26rft.spage%253D1974%26rft.epage%253D1982%26rft.atitle%253DAn%2BEscherichia%2Bcoli%2BdnaE%2Bmutation%2Bwith%2Bsuppressor%2Bactivity%2Btoward%2Bmutator%2BmutD5.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.174.6.1974-1982.1992%26rft_id%253Dinfo%253Apmid%252F1548237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE3NC82LzE5NzQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTMwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaaper</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Radman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">The extreme mutator effect of <em>Escherichia coli mutD5</em> results from saturation of mismatch repair by excessive DNA replication errors.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">3511</span>–<span class="cit-lpage">3516</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft.volume%253D8%26rft.spage%253D3511%26rft_id%253Dinfo%253Apmid%252F2555167%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2555167&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989AV30800042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.39" data-doi="10.1073/pnas.80.23.7085"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scheuermann</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tam</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgers</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Echols</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Identification of the epsilon-subunit of <em>Escherichia coli</em> DNA polymerase III holoenzyme as the <em>dnaQ</em> gene product: a fidelity subunit for DNA replication.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">7085</span>–<span class="cit-lpage">7089</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DScheuermann%26rft.auinit1%253DR.%26rft.volume%253D80%26rft.issue%253D23%26rft.spage%253D7085%26rft.epage%253D7089%26rft.atitle%253DIdentification%2Bof%2Bthe%2Bepsilon-subunit%2Bof%2BEscherichia%2Bcoli%2BDNA%2Bpolymerase%2BIII%2Bholoenzyme%2Bas%2Bthe%2BdnaQ%2Bgene%2Bproduct%253A%2Ba%2Bfidelity%2Bsubunit%2Bfor%2BDNA%2Breplication.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.80.23.7085%26rft_id%253Dinfo%253Apmid%252F6359162%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODAvMjMvNzA4NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMzAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.40" data-doi="10.1128/jb.176.3.815-821.1994"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slater</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lifsics</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Donnell</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title"><em>holE</em>, the gene coding for the theta subunit of DNA polymerase III of <em>Escherichia coli</em>: characterization of a <em>holE</em> mutant and comparison with a <em>dnaQ</em> (epsilon-subunit) mutant.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">815</span>–<span class="cit-lpage">821</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DSlater%26rft.auinit1%253DS.%2BC.%26rft.volume%253D176%26rft.issue%253D3%26rft.spage%253D815%26rft.epage%253D821%26rft.atitle%253DholE%252C%2Bthe%2Bgene%2Bcoding%2Bfor%2Bthe%2Btheta%2Bsubunit%2Bof%2BDNA%2Bpolymerase%2BIII%2Bof%2BEscherichia%2Bcoli%253A%2Bcharacterization%2Bof%2Ba%2BholE%2Bmutant%2Band%2Bcomparison%2Bwith%2Ba%2BdnaQ%2B%2528epsilon-subunit%2529%2Bmutant.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.176.3.815-821.1994%26rft_id%253Dinfo%253Apmid%252F8300534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTc2LzMvODE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.41" data-doi="10.1016/0092-8674(84)90058-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steck</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drlica</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Bacterial chromosome segregation: evidence for DNA gyrase involvement in decatenation.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1081</span>–<span class="cit-lpage">1088</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSteck%26rft.auinit1%253DT.%2BR.%26rft.volume%253D36%26rft.issue%253D4%26rft.spage%253D1081%26rft.epage%253D1088%26rft.atitle%253DBacterial%2Bchromosome%2Bsegregation%253A%2Bevidence%2Bfor%2BDNA%2Bgyrase%2Binvolvement%2Bin%2Bdecatenation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252884%252990058-8%26rft_id%253Dinfo%253Apmid%252F6323018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(84)90058-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6323018&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984SP24300036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Studwell</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Donnell</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Processive replication is contingent on the exonuclease subunit of DNA polymerase III holoenzyme.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">265</span>: <span class="cit-fpage">1171</span>–<span class="cit-lpage">1178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DStudwell%26rft.auinit1%253DP.%2BS.%26rft.volume%253D265%26rft.issue%253D2%26rft.spage%253D1171%26rft.epage%253D1178%26rft.atitle%253DProcessive%2Breplication%2Bis%2Bcontingent%2Bon%2Bthe%2Bexonuclease%2Bsubunit%2Bof%2BDNA%2Bpolymerase%2BIII%2Bholoenzyme.%26rft_id%253Dinfo%253Apmid%252F2153103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNjUvMi8xMTcxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Studwell-Vaughan</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Donnell</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">DNA polymerase III accessory proteins. V. Theta encoded by holE.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">268</span>: <span class="cit-fpage">11785</span>–<span class="cit-lpage">11791</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DStudwell-Vaughan%26rft.auinit1%253DP.%2BS.%26rft.volume%253D268%26rft.issue%253D16%26rft.spage%253D11785%26rft.epage%253D11791%26rft.atitle%253DDNA%2Bpolymerase%2BIII%2Baccessory%2Bproteins.%2BV.%2BTheta%2Bencoded%2Bby%2BholE%26rft_id%253Dinfo%253Apmid%252F8505306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNjgvMTYvMTE3ODUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTMwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.44" data-doi="10.1093/nar/26.17.4005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taft-Benz</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaaper</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Mutational analysis of the 3′→5′ proofreading exonuclease of <em>Escherichia coli</em> DNA polymerase III.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">4005</span>–<span class="cit-lpage">4011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F26.17.4005%26rft_id%253Dinfo%253Apmid%252F9705512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8xNy80MDA1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taft-Benz</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaaper</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The C-terminal domain of <em>dnaQ</em> contains the polymerase binding site.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">181</span>: <span class="cit-fpage">2963</span>–<span class="cit-lpage">2965</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DTaft-Benz%26rft.auinit1%253DS.%2BA.%26rft.volume%253D181%26rft.issue%253D9%26rft.spage%253D2963%26rft.epage%253D2965%26rft.atitle%253DThe%2BC-Terminal%2BDomain%2Bof%2BDnaQ%2BContains%2Bthe%2BPolymerase%2BBinding%2BSite%26rft_id%253Dinfo%253Apmid%252F10217794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE4MS85LzI5NjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTMwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.46" data-doi="10.1007/BF02428026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takano</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakabeppu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horiuchi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sekiguchi</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Structure and function of <em>dnaQ</em> and <em>mutD</em> mutators of <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Gen. Genet.</abbr> <span class="cit-vol">205</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bgeneral%2Bgenetics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGen%2BGenet%26rft.aulast%253DTakano%26rft.auinit1%253DK.%26rft.volume%253D205%26rft.issue%253D1%26rft.spage%253D9%26rft.epage%253D13%26rft.atitle%253DStructure%2Band%2Bfunction%2Bof%2BdnaQ%2Band%2BmutD%2Bmutators%2Bof%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02428026%26rft_id%253Dinfo%253Apmid%252F3540531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02428026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3540531&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986E278100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thibodeau</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joung</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">High-throughput beta-galactosidase assay for bacterial cell-based reporter systems.</span> <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">410</span>–<span class="cit-lpage">415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioTechniques%26rft.stitle%253DBiotechniques%26rft.aulast%253DThibodeau%26rft.auinit1%253DS.%2BA.%26rft.volume%253D36%26rft.issue%253D3%26rft.spage%253D410%26rft.epage%253D415%26rft.atitle%253DHigh-throughput%2Bbeta-galactosidase%2Bassay%2Bfor%2Bbacterial%2Bcell-based%2Breporter%2Bsystems.%26rft_id%253Dinfo%253Apmid%252F15038156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15038156&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220143800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.48" data-doi="10.1038/emboj.2013.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Toste Rego</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holding</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamers</span>  <span class="cit-name-given-names">M. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Architecture of the Pol III-clamp-exonuclease complex reveals key roles of the exonuclease subunit in processive DNA synthesis and repair.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1334</span>–<span class="cit-lpage">1343</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Femboj.2013.68%26rft_id%253Dinfo%253Apmid%252F23549287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo5OiIzMi85LzEzMzQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTMwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1301.49" data-doi="10.1093/protein/2.7.535"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wootton</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">M. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">The Q-linker: a class of interdomain sequences found in bacterial multidomain regulatory proteins.</span> <abbr class="cit-jnl-abbrev">Protein Eng.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">535</span>–<span class="cit-lpage">543</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtein%2BEngineering%2BDesign%2Band%2BSelection%26rft.stitle%253DProtein%2BEng%2BDes%2BSel%26rft.aulast%253DWootton%26rft.auinit1%253DJ.C.%26rft.volume%253D2%26rft.issue%253D7%26rft.spage%253D535%26rft.epage%253D543%26rft.atitle%253DThe%2BQ-linker%253A%2Ba%2Bclass%2Bof%2Binterdomain%2Bsequences%2Bfound%2Bin%2Bbacterial%2Bmultidomain%2Bregulatory%2Bproteins%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fprotein%252F2.7.535%26rft_id%253Dinfo%253Apmid%252F2664763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoicHJvZW5nIjtzOjU6InJlc2lkIjtzOjc6IjIvNy81MzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTMwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/6/1301.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/6/1283" title="A Multifunctional Mutagenesis System for Analysis of Gene Function in Zebrafish" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/6/1313" title="Corrigendum" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/6"><span>Volume 5 Issue 6, June 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/6.cover-source.jpg?itok=NSNFWJbY" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1301.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-slg4j-WCwbrfzH5fJiDQ74RYvtiQiOxFfwjc0f-hyLU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/mutant-screen-report" class="" data-icon-position="" data-hide-link-title="0">Mutant Screen Report</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/6/1301" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/6/1301" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/6/1301" class="active" data-icon-position="" data-hide-link-title="0">Mutations that Separate the Functions of the Proofreading Subunit of the Escherichia coli Replicase</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459910" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-_w8a4wAFklgzM2teQRW40P9Ds8nwm2JZBm-XWfrqBH8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/6/1301.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/6/1301" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-V46lLJeXhD2j8KVCWCy9IN2UuVW4rHufG2cYGoyBpaY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/6/1301" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/6/1301.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459910" id="node459910--2" data-pisa="ggg;5/6/1301" data-pisa-master="ggg;g3.115.017285" data-apath="/ggg/5/6/1301.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mutations that Separate the Functions of the Proofreading Subunit of the <em>Escherichia coli</em> Replicase</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zakiya Whatley</span> and <span class="highwire-citation-author" data-delta="1">Kenneth N. Kreuzer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1301-1311;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017285 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459910--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zakiya Whatley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University Program in Genetics and Genomics, Duke University, Durham, North Carolina 27705</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zakiya%2BWhatley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whatley%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZakiya%2BWhatley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kenneth N. Kreuzer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University Program in Genetics and Genomics, Duke University, Durham, North Carolina 27705</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry, Duke University Medical Center, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kenneth%2BN.%2BKreuzer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kreuzer%20KN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKenneth%2BN.%2BKreuzer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kenneth.kreuzer@duke.edu" class="" data-icon-position="" data-hide-link-title="0">kenneth.kreuzer@duke.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/6/1301" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/6/1301&amp;title=Mutations%20that%20Separate%20the%20Functions%20of%20the%20Proofreading%20Subunit%20of%20the%20Escherichia%20coli%20Replicase" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/6/1301&amp;title=Mutations%20that%20Separate%20the%20Functions%20of%20the%20Proofreading%20Subunit%20of%20the%20Escherichia%20coli%20Replicase" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/6/1301&amp;title=Mutations%20that%20Separate%20the%20Functions%20of%20the%20Proofreading%20Subunit%20of%20the%20Escherichia%20coli%20Replicase" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/6/1301&amp;text=Mutations%20that%20Separate%20the%20Functions%20of%20the%20Proofreading%20Subunit%20of%20the%20Escherichia%20coli%20Replicase" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/6/1301&amp;title=Mutations%20that%20Separate%20the%20Functions%20of%20the%20Proofreading%20Subunit%20of%20the%20Escherichia%20coli%20Replicase" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/6/1301&amp;t=Mutations%20that%20Separate%20the%20Functions%20of%20the%20Proofreading%20Subunit%20of%20the%20Escherichia%20coli%20Replicase" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/6/1301&amp;title=Mutations%20that%20Separate%20the%20Functions%20of%20the%20Proofreading%20Subunit%20of%20the%20Escherichia%20coli%20Replicase" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/6/1301&amp;title=Mutations%20that%20Separate%20the%20Functions%20of%20the%20Proofreading%20Subunit%20of%20the%20Escherichia%20coli%20Replicase" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459910/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459910" id="node459910--3" data-pisa="ggg;5/6/1301" data-pisa-master="ggg;g3.115.017285" data-apath="/ggg/5/6/1301.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mutations that Separate the Functions of the Proofreading Subunit of the <em>Escherichia coli</em> Replicase</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zakiya Whatley</span> and <span class="highwire-citation-author" data-delta="1">Kenneth N. Kreuzer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1301-1311;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017285 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459910--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zakiya Whatley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University Program in Genetics and Genomics, Duke University, Durham, North Carolina 27705</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zakiya%2BWhatley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whatley%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZakiya%2BWhatley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kenneth N. Kreuzer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University Program in Genetics and Genomics, Duke University, Durham, North Carolina 27705</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry, Duke University Medical Center, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kenneth%2BN.%2BKreuzer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kreuzer%20KN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKenneth%2BN.%2BKreuzer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kenneth.kreuzer@duke.edu" class="" data-icon-position="" data-hide-link-title="0">kenneth.kreuzer@duke.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1301&amp;rft.epage=1311&amp;rft.atitle=Mutations%20that%20Separate%20the%20Functions%20of%20the%20Proofreading%20Subunit%20of%20the%20Escherichia%20coli%20Replicase&amp;rft.volume=5&amp;rft.issue=6&amp;rft.date=2015-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Whatley%2C+Zakiya&amp;rft.au=Kreuzer%2C+Kenneth+N."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459910/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459910/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459910/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459910/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459910/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459910/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459910/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459910/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459910/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459910/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459910/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459910/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495856" id="node495856" data-pisa="ggg;10/3/917" data-pisa-master="ggg;g3.119.400936" data-apath="/ggg/10/3/917.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/917" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Mutational Analysis of N-Ethyl-N-Nitrosourea (ENU) in the Fission Yeast <em>Schizosaccharomyces pombe</em></span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495835" id="node495835" data-pisa="ggg;g3.119.400927v1" data-pisa-master="ggg;g3.119.400927" data-apath="/ggg/early/2020/02/21/g3.119.400927.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/02/21/g3.119.400927" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Identification of Suppressors of <em>top-2 </em>Embryonic Lethality in <em>Caenorhabditis elegans</em></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495548" id="node495548" data-pisa="ggg;10/2/479" data-pisa-master="ggg;g3.119.400933" data-apath="/ggg/10/2/479.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/479" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Identification of the Genetic Requirements for Zinc Tolerance and Toxicity in <em>Saccharomyces cerevisiae</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Mutant%20Screen%20Report" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Mutant Screen Report</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/6/1301" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/6/1301.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/6/1301.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/5/6/1301.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
