<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/6/1043" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/6/1043.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/6/1043.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/6/1043.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/6/1043/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/6/1043.atom" />
<meta name="HW.pisa" content="ggg;5/6/1043" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Single-Cell Based Quantitative Assay of Chromosome Transmission Fidelity" />
<meta name="DC.Identifier" content="10.1534/g3.115.017913" />
<meta name="DC.Date" content="2015-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Zhu et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Errors in mitosis are a primary cause of chromosome instability (CIN), generating aneuploid progeny cells. Whereas a variety of factors can influence CIN, under most conditions mitotic errors are rare events that have been difficult to measure accurately. Here we report a green fluorescent protein−based quantitative chromosome transmission fidelity (qCTF) assay in budding yeast that allows sensitive and quantitative detection of CIN and can be easily adapted to high-throughput analysis. Using the qCTF assay, we performed genome-wide quantitative profiling of genes that affect CIN in a dosage-dependent manner and identified genes that elevate CIN when either increased (icCIN) or decreased in copy number (dcCIN). Unexpectedly, qCTF screening also revealed genes whose change in copy number quantitatively suppress CIN, suggesting that the basal error rate of the wild-type genome is not minimized, but rather, may have evolved toward an optimal level that balances both stability and low-level karyotype variation for evolutionary adaptation." />
<meta name="DC.Contributor" content="Jin Zhu" />
<meta name="DC.Contributor" content="Dominic Heinecke" />
<meta name="DC.Contributor" content="Wahid A. Mulla" />
<meta name="DC.Contributor" content="William D. Bradford" />
<meta name="DC.Contributor" content="Boris Rubinstein" />
<meta name="DC.Contributor" content="Andrew Box" />
<meta name="DC.Contributor" content="Jeffrey S. Haug" />
<meta name="DC.Contributor" content="Rong Li" />
<meta name="article:published_time" content="2015-06-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Single-Cell Based Quantitative Assay of Chromosome Transmission Fidelity" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Errors in mitosis are a primary cause of chromosome instability (CIN), generating aneuploid progeny cells. Whereas a variety of factors can influence CIN, under most conditions mitotic errors are rare events that have been difficult to measure accurately. Here we report a green fluorescent protein−based quantitative chromosome transmission fidelity (qCTF) assay in budding yeast that allows sensitive and quantitative detection of CIN and can be easily adapted to high-throughput analysis. Using the qCTF assay, we performed genome-wide quantitative profiling of genes that affect CIN in a dosage-dependent manner and identified genes that elevate CIN when either increased (icCIN) or decreased in copy number (dcCIN). Unexpectedly, qCTF screening also revealed genes whose change in copy number quantitatively suppress CIN, suggesting that the basal error rate of the wild-type genome is not minimized, but rather, may have evolved toward an optimal level that balances both stability and low-level karyotype variation for evolutionary adaptation.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/06/01" />
<meta name="citation_mjid" content="ggg;5/6/1043" />
<meta name="citation_id" content="5/6/1043" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/6/1043" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/6/1043.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/6/1043.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/6/1043.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.017913" />
<meta name="citation_pmid" content="25823586" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1043" />
<meta name="citation_lastpage" content="1056" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jin Zhu" />
<meta name="citation_author_institution" content="Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110" />
<meta name="citation_author" content="Dominic Heinecke" />
<meta name="citation_author_institution" content="Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110" />
<meta name="citation_author" content="Wahid A. Mulla" />
<meta name="citation_author_institution" content="Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110" />
<meta name="citation_author_institution" content="Department of Molecular and Integrative Physiology, University of Kansas Medical Center, 3901 Rainbow Boulevard, Kansas City, Kansas 66160" />
<meta name="citation_author" content="William D. Bradford" />
<meta name="citation_author_institution" content="Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110" />
<meta name="citation_author" content="Boris Rubinstein" />
<meta name="citation_author_institution" content="Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110" />
<meta name="citation_author" content="Andrew Box" />
<meta name="citation_author_institution" content="Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110" />
<meta name="citation_author" content="Jeffrey S. Haug" />
<meta name="citation_author_institution" content="Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110" />
<meta name="citation_author" content="Rong Li" />
<meta name="citation_author_institution" content="Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110" />
<meta name="citation_author_institution" content="Department of Molecular and Integrative Physiology, University of Kansas Medical Center, 3901 Rainbow Boulevard, Kansas City, Kansas 66160" />
<meta name="citation_author_email" content="rli@stowers.org" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1185-7283" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=A. Aguilera;citation_author=T. García-Muse;citation_title=R loops: from transcription byproducts to threats to genome stability.;citation_pages=115-124;citation_volume=46;citation_year=2012;citation_issue=2;citation_pmid=22541554;citation_doi=10.1016/j.molcel.2012.04.009" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=B. N. Ames;citation_author=W. E. Durston;citation_author=E. Yamasaki;citation_author=F. D. Lee;citation_title=Carcinogens are Mutagens: A Simple Test System Combining Liver Homogenates for Activation and Bacteria for Detection;citation_pages=2281-2285;citation_volume=70;citation_year=1973;citation_issue=8;citation_pmid=4151811;citation_doi=10.1073/pnas.70.8.2281" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=E. L. Barnhart;citation_author=R. K. Dorer;citation_author=A. W. Murray;citation_author=S. C. Schuyler;citation_title=Reduced Mad2 expression keeps relaxed kinetochores from arresting budding yeast in mitosis.;citation_volume=22;citation_year=2011;citation_pmid=21593209;citation_doi=10.1091/mbc.E09-01-0029" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=R. A. Burrell;citation_author=S. E. McClelland;citation_author=D. Endesfelder;citation_author=P. Groth;citation_author=M.-C. Weller;citation_title=Replication stress links structural and numerical cancer chromosomal instability.;citation_pages=492-496;citation_volume=494;citation_year=2013;citation_pmid=23446422;citation_doi=10.1038/nature11935" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. Chen;citation_author=R. D. Kolodner;citation_title=Gross chromosomal rearrangements in Saccharomyces cerevisiae replication and recombination defective mutants.;citation_pages=81-85;citation_volume=23;citation_year=1999;citation_issue=1;citation_pmid=10471504;citation_doi=10.1038/12687" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Chen;citation_author=W. D. Bradford;citation_author=C. W. Seidel;citation_author=R. Li;citation_title=Hsp90 stress potentiates rapid cellular adaptation through induction of aneuploidy.;citation_pages=246-250;citation_volume=482;citation_year=2012;citation_issue=7384;citation_pmid=22286062;citation_doi=10.1038/nature10795" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=J. S. Choy;citation_author=E. O’Toole;citation_author=B. M. Schuster;citation_author=M. J. Crisp;citation_author=T. S. Karpova;citation_title=Genome-wide haploinsufficiency screen reveals a novel role for γ-TuSC in spindle organization and genome stability.;citation_volume=24;citation_year=2013;citation_pmid=23825022;citation_doi=10.1091/mbc.E12-12-0902" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=T. Davoli;citation_author=W. A.;citation_author=K. E. Xu;citation_author=L. M. Mengwasser;citation_author=J. C. Sack;citation_author=. Yoon;citation_title=Cumulative haploinsufficiency and triplosensitivity drive aneuploidy patterns and shape the cancer genome.;citation_pages=948-962;citation_volume=155;citation_year=2013;citation_pmid=24183448;citation_doi=10.1016/j.cell.2013.10.011" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. M. Deutschbauer;citation_author=D. F. Jaramillo;citation_author=M. Proctor;citation_author=J. Kumm;citation_author=M. E. Hillenmeyer;citation_title=Mechanisms of haploinsufficiency revealed by genome-wide profiling in yeast.;citation_volume=169;citation_year=2005;citation_pmid=15716499;citation_doi=10.1534/genetics.104.036871" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=L. Dirick;citation_author=W. Bendris;citation_author=V. Loubiere;citation_author=T. Gostan;citation_author=E. Gueydon;citation_title=Metabolic and environmental conditions determine nuclear genomic instability in budding yeast lacking mitochondrial DNA.;citation_volume=4;citation_year=2014;citation_pmid=24374640;citation_doi=10.1534/g3.113.010108" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Franceschini;citation_author=D. Szklarczyk;citation_author=S. Frankild;citation_author=M. Kuhn;citation_author=M. Simonovic;citation_title=STRING v9.1: protein-protein interaction networks, with increased coverage and integration.;citation_volume=41;citation_year=2013;citation_pmid=23203871;citation_doi=10.1093/nar/gks1094" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=L. Francisco;citation_author=W. Wang;citation_author=C. S. Chan;citation_title=Type 1 protein phosphatase acts in opposition to IpL1 protein kinase in regulating yeast chromosome segregation.;citation_pages=4731-4740;citation_volume=14;citation_year=1994;citation_issue=7;citation_pmid=8007975;citation_doi=10.1128/MCB.14.7.4731" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=J. B. Geigl;citation_author=A. C. Obenauf;citation_author=T. Schwarzbraun;citation_author=M. R. Speicher;citation_title=Defining 'chromosomal instability'.;citation_pages=64-69;citation_volume=24;citation_year=2008;citation_issue=2;citation_pmid=18192061;citation_doi=10.1016/j.tig.2007.11.006" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Ghaemmaghami;citation_author=W.-K. Huh;citation_author=K. Bower;citation_author=R. W. Howson;citation_author=A. Belle;citation_title=Global analysis of protein expression in yeast.;citation_pages=737-741;citation_volume=425;citation_year=2003;citation_issue=6959;citation_pmid=14562106;citation_doi=10.1038/nature02046" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. E. Haber;citation_title=Mating-Type Genes and MAT Switching in Saccharomyces cerevisiae;citation_pages=33-64;citation_volume=191;citation_year=2012;citation_issue=1;citation_pmid=22555442;citation_doi=10.1534/genetics.111.134577" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=I. M. Henry;citation_author=B. P. Dilkes;citation_author=E. S. Miller;citation_author=D. Burkart-Waco;citation_author=L. Comai;citation_title=Phenotypic consequences of aneuploidy in Arabidopsis thaliana.;citation_volume=186;citation_year=2010;citation_pmid=20876566;citation_doi=10.1534/genetics.110.121079" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=C. H. Ho;citation_author=L. Magtanong;citation_author=S. L. Barker;citation_author=D. Gresham;citation_author=S. Nishimura;citation_title=A molecular barcoded yeast ORF library enables mode-of-action analysis of bioactive compounds.;citation_pages=369-377;citation_volume=27;citation_year=2009;citation_issue=4;citation_pmid=19349972;citation_doi=10.1038/nbt.1534" />
<meta name="citation_reference" content="citation_journal_title=Cancer Sci.;citation_author=G. Itoh;citation_author=S. Sugino;citation_author=M. Ikeda;citation_author=M. Mizuguchi;citation_author=S.-I. Kanno;citation_title=Nucleoporin Nup188 is required for chromosome alignment in mitosis.;citation_pages=871-879;citation_volume=104;citation_year=2013;citation_pmid=23551833;citation_doi=10.1111/cas.12159" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=R. Kumaran;citation_author=S.-Y. Yang;citation_author=J.-Y. Leu;citation_title=Characterization of chromosome stability in diploid, polyploid and hybrid yeast cells.;citation_pages=e68094;citation_volume=8;citation_year=2013;citation_pmid=23874507;citation_doi=10.1371/journal.pone.0068094" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=J. D. Laney;citation_author=E. F. Mobley;citation_author=M. Hochstrasser;citation_title=The Short-Lived Mat{alpha}2 Transcriptional Repressor Is Protected from Degradation In Vivo by Interactions with Its Corepressors Tup1 and Ssn6;citation_pages=371-380;citation_volume=26;citation_year=2006;citation_issue=1;citation_pmid=16354707;citation_doi=10.1128/MCB.26.1.371-380.2006" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=M. S. Lee;citation_author=M. Henry;citation_author=P. A. Silver;citation_title=A protein that shuttles between the nucleus and the cytoplasm is an important mediator of RNA export.;citation_pages=1233-1246;citation_volume=10;citation_year=1996;citation_issue=10;citation_pmid=8675010;citation_doi=10.1101/gad.10.10.1233" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. Z. Li;citation_author=S. J. Elledge;citation_title=MAGIC, an in vivo genetic method for the rapid construction of recombinant DNA molecules.;citation_pages=311-319;citation_volume=37;citation_year=2005;citation_issue=3;citation_pmid=15731760;citation_doi=10.1038/ng1505" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=R. Li;citation_author=A. W. Murray;citation_title=Feedback control of mitosis in budding yeast.;citation_pages=519-531;citation_volume=66;citation_year=1991;citation_issue=3;citation_pmid=1651172;citation_doi=10.1016/0092-8674(81)90015-5" />
<meta name="citation_reference" content="citation_title=EMBO Rep.;citation_author=N. McGranahan;citation_author=R. A. Burrell;citation_author=D. Endesfelder;citation_author=M. R. Novelli;citation_author=C. Swanton;citation_title=Cancer chromosomal instability: therapeutic and diagnostic challenges.;citation_volume=13;citation_year=2012;citation_pmid=22595889;citation_doi=10.1038/embor.2012.61" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Molecular cell biology;citation_journal_abbrev=Nat Rev Mol Cell Biol;citation_author=A. Musacchio;citation_author=E. D. Salmon;citation_title=The spindle-assembly checkpoint in space and time.;citation_pages=379-393;citation_volume=8;citation_year=2007;citation_issue=5;citation_pmid=17426725;citation_doi=10.1038/nrm2163" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=M. Ni;citation_author=M. Feretzaki;citation_author=W. Li;citation_author=A. Floyd-Averette;citation_author=P. Mieczkowski;citation_title=Unisexual and heterosexual meiotic reproduction generate aneuploidy and phenotypic diversity de novo in the yeast Cryptococcus neoformans.;citation_pages=e1001653;citation_volume=11;citation_year=2013;citation_pmid=24058295;citation_doi=10.1371/journal.pbio.1001653" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=I. I. Ouspenski;citation_author=S. J. Elledge;citation_author=B. R. Brinkley;citation_title=New yeast genes important for chromosome integrity and segregation identified by dosage effects on genome stability.;citation_volume=27;citation_year=1999;citation_pmid=10454593;citation_doi=10.1093/nar/27.15.3001" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. Papp;citation_author=C. Pal;citation_author=L. D. Hurst;citation_title=Dosage sensitivity and the evolution of gene families in yeast.;citation_pages=194-197;citation_volume=424;citation_year=2003;citation_issue=6945;citation_pmid=12853957;citation_doi=10.1038/nature01771" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=N. Pavelka;citation_author=G. Rancati;citation_author=J. Zhu;citation_author=W. D. Bradford;citation_author=A. Saraf;citation_title=Aneuploidy confers quantitative proteome changes and phenotypic variation in budding yeast.;citation_pages=321-325;citation_volume=468;citation_year=2010;citation_issue=7321;citation_pmid=20962780;citation_doi=10.1038/nature09529" />
<meta name="citation_reference" content="citation_journal_title=Cancer Metastasis Rev.;citation_author=T. Potapova;citation_author=J. Zhu;citation_author=R. Li;citation_title=Aneuploidy and chromosomal instability: a vicious cycle driving cellular evolution and cancer genome chaos.;citation_pages=377-389;citation_volume=32;citation_year=2013;citation_pmid=23709119;citation_doi=10.1007/s10555-013-9436-6" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=R. M. Ricke;citation_author=K. B. Jeganathan;citation_author=J. M. van Deursen;citation_title=Bub1 overexpression induces aneuploidy and tumor formation through Aurora B kinase hyperactivation.;citation_volume=193;citation_year=2011;citation_pmid=21646403;citation_doi=10.1083/jcb.201012035" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. D. Ryan;citation_author=E. M. C. Britigan;citation_author=L. M. Zasadil;citation_author=K. Witte;citation_author=A. Audhya;citation_title=Up-regulation of the mitotic checkpoint component Mad1 causes chromosomal instability and resistance to microtubule poisons.;citation_volume=109;citation_year=2012;citation_pmid=22778409;citation_doi=10.1073/pnas.1201911109" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Cancer;citation_journal_abbrev=Nat Rev Cancer;citation_author=T. Santarius;citation_author=J. Shipley;citation_author=D. Brewer;citation_author=M. R. Stratton;citation_author=C. S. Cooper;citation_title=A census of amplified and overexpressed human cancer genes.;citation_pages=59-64;citation_volume=10;citation_year=2010;citation_issue=1;citation_pmid=20029424;citation_doi=10.1038/nrc2771" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=J. M. Santos-Pereira;citation_author=A. B. Herrero;citation_author=M. L. García-Rubio;citation_author=A. Marín;citation_author=S. Moreno;citation_title=The Npl3 hnRNP prevents R-loop-mediated transcription-replication conflicts and genome instability;citation_pages=2445-2458;citation_volume=27;citation_year=2013;citation_issue=22;citation_pmid=24240235;citation_doi=10.1101/gad.229880.113" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Cancer;citation_journal_abbrev=Nat Rev Cancer;citation_author=J.-M. Schvartzman;citation_author=R. Sotillo;citation_author=R. Benezra;citation_title=Mitotic chromosomal instability and cancer: mouse modelling of the human disease.;citation_pages=102-115;citation_volume=10;citation_year=2010;citation_issue=2;citation_pmid=20094045;citation_doi=10.1038/nrc2781" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Selmecki;citation_author=A. Forche;citation_author=J. Berman;citation_title=Aneuploidy and Isochromosome Formation in Drug-Resistant Candida albicans;citation_pages=367-370;citation_volume=313;citation_year=2006;citation_issue=5785;citation_pmid=16857942;citation_doi=10.1126/science.1128242" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=P. Shannon;citation_author=A. Markiel;citation_author=O. Ozier;citation_author=N. S. Baliga;citation_author=J. T. Wang;citation_title=Cytoscape: A Software Environment for Integrated Models of Biomolecular Interaction Networks;citation_pages=2498-2504;citation_volume=13;citation_year=2003;citation_issue=11;citation_pmid=14597658;citation_doi=10.1101/gr.1239303" />
<meta name="citation_reference" content="citation_journal_title=Cancer cell;citation_journal_abbrev=Cancer Cell;citation_author=R. Sotillo;citation_author=E. Hernando;citation_author=E. Díaz-Rodríguez;citation_author=J. Teruya-Feldstein;citation_author=C. Cordón-Cardo;citation_title=Mad2 overexpression promotes aneuploidy and tumorigenesis in mice.;citation_pages=9-23;citation_volume=11;citation_year=2007;citation_issue=1;citation_pmid=17189715;citation_doi=10.1016/j.ccr.2006.10.019" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Spencer;citation_author=S. L. Gerring;citation_author=C. Connelly;citation_author=P. Hieter;citation_title=Mitotic chromosome transmission fidelity mutants in Saccharomyces cerevisiae.;citation_pages=237-249;citation_volume=124;citation_year=1990;citation_issue=2;citation_pmid=2407610" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. C. Stirling;citation_author=M. S. Bloom;citation_author=T. Solanki-Patil;citation_author=S. Smith;citation_author=P. Sipahimalani;citation_title=The complete spectrum of yeast chromosome instability genes identifies candidate CIN cancer genes and functional roles for ASTRA complex components.;citation_pages=e1002057-e1002057;citation_volume=7;citation_year=2011;citation_issue=4;citation_pmid=21552543;citation_doi=10.1371/journal.pgen.1002057" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. D. Strome;citation_author=X. Wu;citation_author=M. Kimmel;citation_author=S. E. Plon;citation_title=Heterozygous screen in Saccharomyces cerevisiae identifies dosage-sensitive genes that affect chromosome stability.;citation_volume=178;citation_year=2008;citation_pmid=18245329;citation_doi=10.1534/genetics.107.084103" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. M. Torres;citation_author=T. Sokolsky;citation_author=C. M. Tucker;citation_author=L. Y. Chan;citation_author=M. Boselli;citation_title=Effects of Aneuploidy on Cellular Physiology and Cell Division in Haploid Yeast;citation_pages=916-924;citation_volume=317;citation_year=2007;citation_issue=5840;citation_pmid=17702937;citation_doi=10.1126/science.1142210" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. R. Veatch;citation_author=M. A. McMurray;citation_author=Z. W. Nelson;citation_author=D. E. Gottschling;citation_title=Mitochondrial dysfunction leads to nuclear genome instability via an iron-sulfur cluster defect.;citation_pages=1247-1258;citation_volume=137;citation_year=2009;citation_issue=7;citation_pmid=19563757;citation_doi=10.1016/j.cell.2009.04.014" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=L. Wahba;citation_author=J. D. Amon;citation_author=D. Koshland;citation_author=M. Vuica-Ross;citation_title=RNase H and multiple RNA biogenesis factors cooperate to prevent RNA:DNA hybrids from generating genome instability.;citation_pages=978-988;citation_volume=44;citation_year=2011;citation_issue=6;citation_pmid=22195970;citation_doi=10.1016/j.molcel.2011.10.017" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. Wang;citation_author=D. Duncan;citation_author=Z. Shi;citation_author=B. Zhang;citation_title=WEB-based GEne SeT AnaLysis Toolkit (WebGestalt): update 2013.;citation_volume=41;citation_year=2013;citation_pmid=23703215;citation_doi=10.1093/nar/gkt439" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=C. D. Warren;citation_author=D. M. Brady;citation_author=R. C. Johnston;citation_author=J. S. Hanna;citation_author=K. G. Hardwick;citation_title=Distinct chromosome segregation roles for spindle checkpoint proteins.;citation_volume=13;citation_year=2002;citation_pmid=12221113;citation_doi=10.1091/mbc.E02-04-0203" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. W. Y. Yuen;citation_author=C. D. Warren;citation_author=O. Chen;citation_author=T. Kwok;citation_author=P. Hieter;citation_title=Systematic genome instability screens in yeast and their potential relevance to cancer.;citation_volume=104;citation_year=2007;citation_pmid=17360454;citation_doi=10.1073/pnas.0610642104" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=T. I. Zack;citation_author=S. E. Schumacher;citation_author=S. L. Carter;citation_author=A. D. Cherniack;citation_author=G. Saksena;citation_title=Pan-cancer patterns of somatic copy number alteration.;citation_pages=1134-1140;citation_volume=45;citation_year=2013;citation_pmid=24071852;citation_doi=10.1038/ng.2760" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. Zhu;citation_author=N. Pavelka;citation_author=W. D. Bradford;citation_author=G. Rancati;citation_author=R. Li;citation_title=Karyotypic determinants of chromosome instability in aneuploid budding yeast.;citation_pages=e1002719-e1002719;citation_volume=8;citation_year=2012;citation_issue=5;citation_pmid=22615582;citation_doi=10.1371/journal.pgen.1002719" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Single-Cell Based Quantitative Assay of Chromosome Transmission Fidelity" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
<meta name="twitter:description" content="Errors in mitosis are a primary cause of chromosome instability (CIN), generating aneuploid progeny cells. Whereas a variety of factors can influence CIN, under most conditions mitotic errors are rare events that have been difficult to measure accurately. Here we report a green fluorescent protein−based quantitative chromosome transmission fidelity (qCTF) assay in budding yeast that allows sensitive and quantitative detection of CIN and can be easily adapted to high-throughput analysis. Using the qCTF assay, we performed genome-wide quantitative profiling of genes that affect CIN in a dosage-dependent manner and identified genes that elevate CIN when either increased (icCIN) or decreased in copy number (dcCIN). Unexpectedly, qCTF screening also revealed genes whose change in copy number quantitatively suppress CIN, suggesting that the basal error rate of the wild-type genome is not minimized, but rather, may have evolved toward an optimal level that balances both stability and low-level karyotype variation for evolutionary adaptation." />
<meta name="og-title" property="og:title" content="Single-Cell Based Quantitative Assay of Chromosome Transmission Fidelity" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/6/1043" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Errors in mitosis are a primary cause of chromosome instability (CIN), generating aneuploid progeny cells. Whereas a variety of factors can influence CIN, under most conditions mitotic errors are rare events that have been difficult to measure accurately. Here we report a green fluorescent protein−based quantitative chromosome transmission fidelity (qCTF) assay in budding yeast that allows sensitive and quantitative detection of CIN and can be easily adapted to high-throughput analysis. Using the qCTF assay, we performed genome-wide quantitative profiling of genes that affect CIN in a dosage-dependent manner and identified genes that elevate CIN when either increased (icCIN) or decreased in copy number (dcCIN). Unexpectedly, qCTF screening also revealed genes whose change in copy number quantitatively suppress CIN, suggesting that the basal error rate of the wild-type genome is not minimized, but rather, may have evolved toward an optimal level that balances both stability and low-level karyotype variation for evolutionary adaptation." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
    <title>Single-Cell Based Quantitative Assay of Chromosome Transmission Fidelity | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"VGTscKv_Byv6DVFne82M4JOlL1xxoFjleZMil17M0Qw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459953","apath":"\/ggg\/5\/6\/1043.atom","pisa":"ggg;5\/6\/1043","ac":{"\/ggg\/5\/6\/1043.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/6\/1043.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/6\/1043"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/6\/1043"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/6\/1043":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022gTe\\\\\\\/+c7XMIzGnoBHeCHymjENPKz\\\\\\\/0PN\\\\\\\/I2RjJHgBZ26Fza5MazslXh4dhP2jzK0YqPYR+cgk0b5BAyUH0lfpBHslVe3ndJEVAic6QsayrZ9YhaUTs\\\\\\\/pEfCUSFuSgFCeJcouHgWEqUUhMflhqEZ9kaSBu1PicHBbgrvnSafZBmoumMNTIbXF0qdwwlygyq1t0EIBwuLULj2zTPj6crXs0CJMloJ\\\\\\\/TExRWaKSi83vUg2wOav4G8B7dQtGYU2NjwtAbe2YTerduCJybvbBXsGl5fPo505\\\\\\\/UNuGkdEWkwdKxlOKb7svVgWeSCImueyzfRSMhhj6MuLP46D4KAhXXox1+oaTsd4W121OqCi2kES5nBjxdM4N0IWorau5\\\\\\\/muQrhkaUni5eMMMPdg12BPxDKvcrrTU4uckCvbRCtC20aWIE++W1dskbDIBKqK+cuCzpmy7+Yf2Q0W0bjSG6nCD3UwbrL0y3YXR3qv2WeExtaLPLb3USBGzx4bYaYKQeXdx8uk3ZIdvsXr1B3XGfrEwdvb1zY0LaSY4Df+zgFibydisDr\\\\\\\/pGVeMl1jVSJgK1eX\\\\\\\/F3zSIhhmMkY4XeTtQtwY7\\\\\\\/NXHTJOm2kjdFrbOYa8k41UOW2VofHxTrneZd\\\\\\\/L8SA6jvkdSry7LWetuAsivF1xjYBVVaWPVqLn4OCEerINTx2dPUzhkYX06UQ2TYDNPQjcLt31SMGYuJqYojP4OUcv9QyEMB3JK7gH31FkGeBC4KFI65cZYWCTstUKvVd5aJ73\\\\\\\/\\\\\\\/oLQiQazfywMZI+DGBlRMms6XujDSAKhyZfYgpsGipiPnf30F\\\\\\\/\\\\\\\/bG2k+3pTO73g4r6X1KK\\\\\\\/UfbpJHA8viND7pRqv9Fv9jz5NQUluTT4J5xXvsvj0RFGz8FpysV7QpAKnzNRIC678uL3Ry0tvhio4QDD7pOGWpFWbE7f8Xwb3Ge\\\\\\\/8uNvb3fDHq4IS6y2whdW7rvMASNG5xEh4iMcDM\\\\\\\/5SNzyEOTuqiwk19lCPB64PUxjHSv3rVwBKd82vCQkNWPlg3xdJp4yu0O1IITNXysvzMT+lVqjAdi\\\\\\\/2qrcuMN4oGTOHjfB4TEfzZuW8znjbwJw6aX+p9gxz\\\\\\\/nvAZEhGMTKkQXnNsUEDPA355CDdimGLYqQa8j\\\\\\\/h6fzWfXcA82zx3ZqvOXMJZoTMVAVV8f9nHY1lecLHfAhxfeA8rYzBHXQBVqL\\\\\\\/g\\\\\\\/BgS+arufvq1nCFHT5GLNoTrPSAMivJZUvgydezW7wlR3fvhx36TFZWRGokljMGq91Wu8hA6Tq9BM2Rr0f1lmv\\\\\\\/\\\u0022,\\\u0022iv\\\u0022:\\\u0022enerQUodrJmlilBXLfuOQg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222937dc53d0cb53f5897c795df98ee7bdee9ba5c5661f2808e085636ba6fdd82b\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229VYg8WiMlcZO9zOE5CQ6aLAgpBAhSDiLoRUv7vbj7+BCoXAQz1Y3xPbb9DkdjLyOd4PzU1Gm35sK7BWhhSmoNbOCstmvLVTocz+wCJUkFIsG1II8JyA7za54Pufsg9de3jOrq1j+OhSRgbKTeYR8SWLZunXrRERAFMxMrYqzr\\\\\\\/vtVGxfEP+H2qEuBU421rS5fPGNVXRRDuIdvk+5uf1imTjoE2aKZydDcrMEArgvhXuRhI0SAuxsLaFeDG7EIeE1+TwwH+LC7xJOerVRwmkYYZcwEswvzMYiL0XnCBX3T5HYbWlUqEleFpC7STeBjdVFejqy4VkVtEtIL+1eyP149irvNPAZZJVNTZgE48f2\\\\\\\/OfHUUTcxGtwgJJsXYYI0kly7qI\\\\\\\/XSoB6ZxElsnFXwcSgAiW0pGksP\\\\\\\/QLAvHP+fa1BTkFmnVFd+XNjkHMfahG7ABMbbQ2fjxyw2FRcn3yzU0kINZTIm2FjKtI3VYshMC8\\\\\\\/vkm+8Nue22\\\\\\\/r0u4dcmVgVdXaP4M1AoKTRTriITf8c1sHKIeQhtMKHJW+rhkKu0DJnlYZB\\\\\\\/\\\\\\\/xPuSFGdyNZ\\\\\\\/eYF26IxhnBF6SiTecDKiGba2TESLbf3kjv5FJLiii0rIGrVIgC++SljfGLCB4kOmciIvXAYKGG08FOi2inqDpEvRQL53UJi9KEOqSiM2ddyTcXrjnu8xZ8GwPd0YJ9qgBiajpQxr7FZrhUUPpBrd+M3xmQmVeJq49YkXAqsq+KF8dqsYn0U7jnvZ2SqCGF+iI+wHLGhfmmWe2DPXJ0ei86o+aPYdMgyNj1wxjwi3kszHC3A5qQgi1PFF1BsBIca061zsLw295z7LDGv6h3Nt6+s+DECyHtYhAhiwAELis23kn\\\\\\\/3edpMLI57f9X+ee+5yF3FmoN0i1WqQ3n\\\\\\\/l+zwh55CDH3gak0VIcsxFsCKNBZKEeUtkXwmpPetxKvBSK4TpaxBzgopk+mYb3+YzbarKjxulXp5RX9tp8B5UPiA21hUej9X99sqi6Fe8zdF0LA7iF99igY+e1GKrZl9fQiLLb0NFE0GxDkqiNPL41R6uvSb1NFb8mIvD0ZAT5M2XTNV0Ob\\\\\\\/mETpaOcElpAjhVmTktC6QGxWEOwvTngK0mBiaHg8319aK25aL8ZeuSmxreT6p3qzNH7JsAPtmWhvmlbJjwWcmJ7R5xZrNAxE8\\\\\\\/jSp31XeKx0PmJ9tCReIIZg\\\\\\\/qf2AVl5zX1mmBl85zdqZl2psrpLVTO90FsjDDWzknCfX+Uy91r8gpl3nC+dWdZ0s+XTbBjLtMTzBAQ93vBBNbuLYvbEKZ8HcJLCELrPHuf1+mAKNYQcGCakRb+Br8jHWtxm8+o\\\\\\\/IptcVrXVK5Lag6iQUgH90XyqlWMPaG0ertOmEFHp1WkXUKtEIG3AzIuHcOzsmkEdgF5vnGg0n6\\\\\\\/EulsxHmgGk\\\\\\\/GiS960RXT4zkqsobCVqmSmuww0i7m\\\\\\\/iWHs+KYjvI4QCCbLkftaa\\\\\\\/ZBPxxdnsVuucQWRKucCFRUMjJW7mhiKrY99in2lBMToWFhjaoIrqU1+r9sJFUC5P3sc1KW1m3\\\\\\\/MYO7XMw0fbKK5QoUtriB4HiAdqEYC\\\\\\\/\\\\\\\/YVFDI4Q6duLtdMWB+QiPXmUMxHCPzPP8dVcGAXVR3xto6FfPaDdqV+iXb4AQ27QFWc3iorbQ3Ea2FOzekGEmGDoGA6N7pS3bTthq5by7TXiNkVvoi0LidEVUM1MxMrQAm+TGnTzhOdilbJZ4v\\\\\\\/v8FHp0sWKgJeNtlh\\\\\\\/4taTbumVrY9jS6jvEWte80Uv04yhGPYJ+XzUWaLRlK7UoLzrz2B+uXaJCA1GUY2RFP69gCp5aoyN1naEFg56cdUh19N05aLssVO9l4FKYMOlTcUewvTHjoZLIdlZnUfVcbEob+B5mdl4sczT3mnKO5HnUFEF6NPxZfMkJRAG5s1LBAWahEiLKXNS4AfoTMDsKOrUNTWBLac585VJ1qb9Ey4CrbGC03pRaK2tBP8pKyYsrD4iEa3zOwwX1wzmK+\\\\\\\/kNQLeYmLrigPxJ++JSuJfeCHucig9PA3bRcyd53nBxjhJKz5\\\\\\\/YO9t1Zd7+9FlCVhJGv1qayzQk8jOKiMXVjj+kLx1mxescP3OrBavNWaxeO3mXaSsQzXPklQ4P57NOWYGZRup2bXvF1sDI2pF2y7CrY8ek6SUkljd0aAx\\\\\\\/t868azbjt2Adhy2e4is1F+kqZGDNs2vZ2VzR7eGEbBR6Y5ZJnkkZlvefuwR5z6x\\\\\\\/q0IpoKIAo1BiIZ+wRoMOD1gVP+2t+Dsza2O87nNE68Jljo4o298HgS4VVCnmU462lG+xxa30YnuProbdRZQbOpfhZlHLaFSjLRN\\\\\\\/u7Ea6iAWJOf\\\\\\\/Z1QjWjKM4utZmqCtY9u+HR76zzY3At7nuBtYl4YxJ0Be8Oq8fYRkei9mTJjpwHf3JIbBuP9RsVi2nytTgBgzgL3PwCuwFtILtMfgkBbooQY7WmnctEOgrHO4VAS3aSTSQRCBkXq5fLu1CElTW3J91PrciNjUeAn3y7kNabKFJYGjNCHAf6FGzYur1qa1gziVxAO\\\\\\\/qm9w\\\\\\\/ZFYkD7ky151h4rTrJpaxrisGGY8e1VFYNTcvvtqBu8S2Gu56q+R+7wdfXayuyUu86+8Cnt9jmoAs1RBhhAvUNk+lXlgW91ZLSmuACCh1\\\\\\\/4REbX6f\\\\\\\/uCeW2mg+8GJtxPLeHE4W9DXMhNam7hh+6WHhGRIOTT7Hx0l3bkE7VSUd5jCvy+NuuJsu2G3Vvpefmt3orPFmE2vPpW9djDfHtzLXHRb5PezqXvrgu7f2cmLkqpCTA16q\\\\\\\/sQ45Kp1wtggvCGA2GMo+MCfzCWXZalAxl\\\\\\\/ViF+TrUpA3vYqBQcOvVcIiiB7OowHuELhLkP2Z8VBHwyHkRdxZXWOxHUduBPR9XOI1t4vlSUnagV5QlhltDDFy\\\\\\\/gj3QU0FyUbA\\\\\\\/rwZc3t0ICg3Ax1lf5GUY0fhd05RCbCKArW8ZFu6CWV\\\\\\\/jqHlceRbsPRpLS0qzdXA\\\\\\\/SX2IBjXWJscgAypjpkfmkhS4GivkcqTM9wsU1PJ7QwSjrY4eFxLtkzdNuSZG41uxZ\\\\\\\/czsrCnxxlw1fUXB9M+jNgGjozKiXquc5N97STmY1cGfLChgNH9unCd5s\\\\\\\/fRSM8ob28QEcfWpA+mVqRGKgxdDt8r0PesMa9XHw0dLLGey8zpUQPdtAuj1hj8H54V04igbtTn083uvKxSrIWkYQe2Yxn7hMbFyDpBpZRYVD34OaBOGS93GUnZ2RRJQyVbR5cORIi4VCprBosNnJzFcaq2tN3i\\\\\\\/DvkbqOzqyynuYFhyZUH7aVbK8O\\\\\\\/GOazB60GRxKmvNuENFhmZ8OTy8842eLa9BgYUDy2lkEQOD++lrmT3GNMPD7IDfalqrmPKGCi2kfS+Kls61RKDHR95AckStfBM6MZCt5mz1TPX0ZihrftxySgE3VZPKl4kQoXXzrXFNJxJCNXFJrF9fYOPxDt3HJlj9JDr6\\\\\\\/2jD\\\\\\\/EOUI+v4zPqB0ePcebaPuEesHVzPaEcFi6SwrSwIojrJQz8f9rxdvoOrAhI0LTDmqBAyxPS0zf6gb4z9RL+DDUSgc1XrQF8SnVqCPXenO6SauTJRWUgoC7T7yfMOKCRS2UlbUb+Sqw5IEqM0umR\\\\\\\/vGobuhEN1Fy+X\\\\\\\/u\\\\\\\/mXq7r\\\\\\\/u1Wqfd845nF3lqeeFHepeVCMiuypynRnwBZeO\\\\\\\/0bDxmrMAfWbXB0UVoujiA\\\\\\\/vZoaq1Cx809quVFUh+MUf6sznKCsic+ZOF\\\\\\\/dGLlhmwAMJgiheUke+hbcNmdJRGD80hitdTpyEi\\\\\\\/VCaf5QTmUMvAxmGN1gPdMQjmxguwe6fjL36XXwYzVCn7ZdZju87bidKCCbgptJ+mhi6feztmSnpgmu9ArOyW8oy43H+mLPVCTHWaasn5gKWBPXENdTViIydlCoNc7V\\\\\\\/stTStQKIkaNPT\\\\\\\/mvh7LpNQH7zOZrF6O7UfnQCBeQeKG6EutEVmU7CXKbHCUXVgox0mkn0Wo7C5KCmf04\\\\\\\/GFizfAPQeOckzET\\\\\\\/rjcOty0kUZ7B7KjxsRzItDGkZz6FhwL7TvVVkfcDQX2ScD1MYYPENY3hTD\\\\\\\/bIgagQwXSPTZ9RbHfyBQtSG2XLMQYj31kXkMrCScP\\\\\\\/sU+pE9XZQVqCe5RzqKWTLa0d1nL96Q8upf++Zl4nTfc1cBJMnUCYdclPGJTazQwIwD36G\\\\\\\/SzxgAdaZwQ7SkPJ2PJONMHdFRpnpajUBQAgNltiT54jBYN8pFBxsgQVhERAB1S9r0aWBNEr3P\\\\\\\/MwaHHFA9UsxXXBEs5bZaVT4TeAvDaIJZBOWor3+ekAiUHxcb2ttm8oDwdVN+YvRrd7M6DVFzBPpK+QFxspzGJt7BbWk7hTMeBYLRilrTSt9QrWNWiLqJWqaI6ts0z+8guyHo+XdwhhGtSW98F39SXVPLKz8r8h0Q\\\\\\\/ODw3\\\\\\\/oHUd5IqHodn6Xiz+wopPkBTCb1nA7XQlPSkq5CmzFk\\\\\\\/yQQ\\\\\\\/JVGgLChI7KZBW3DoE5TG850W9Rr4v0c7iMOBjz+eVIlFcm06bPWhtnnLuGDVOIO947zT+HJv2eXWwZw\\\\\\\/TBgNa9d4VLxO0yIGG\\\\\\\/Zn4JJBpOCdMy3CmR24kka+yOEFWmBUBpYFTSfknci+kmvyE+6gKlWYz8MOQ3JSxyzzuo18k+7cLdG6TLBAzSQRxuL1kz45yxufAY30DQ4ruFRAYaQafXGGWAESRW19Idf7vZzOjd3bhDr7esxrTPtftJMVrBKn2H\\\\\\\/k5dH5sJaRtCobOFS3d4qjpbktD89YrZxJIyxoUoYjearlkdwdjW\\\\\\\/CphjEjCs2ABQuj92RpD3636NzAQxv+jas8oYuyoaA1CjPShcvBwC\\\\\\\/12EIkX5\\\\\\\/wdklonhwi9J\\\\\\\/fk5CWwJHffllr6xVhts4\\\\\\\/XoavmqV0k+j+MDJ2JFY0546OYGOllX+jtcDl7y4Ab7CB5E6B5IQe4GR4f54xx84xkOPbZPLiWJXwYJziZnwsTu6jYZT0FSO+5hh9g0y3CDrvOHJ+z4gdgKmtfg8kCEtgO0AsIbEZlzYCip5YuUXy2lrLRGscjrwsPEltYVY5EbbpOuCDZ9OfcZcfcc6T2lGw5Pkr6xTs\\\\\\\/QF1W0C27CMjbUuQOf+HmBO7kIu7ZTN\\\\\\\/oJB903P5Sx3hHgLjh\\\\\\\/lNYpbJosb8S9zG3\\\\\\\/sWu\\\\\\\/6c6S6uS6XAnNR+Ek7QPGO5GyZ+TIM+0M4A6zdWk638momdgFQufxRvRwywsux7zEEfVcV6JTjIw0ymrbzID+2BsoqGj5xBa3aWVb9jLvGcxCT6HdbmdqDpNP0on0wpp\\\\\\\/yBPJXAKzuTj2qgfY4PwCeSCJCiEJ7H4J3pxQuD0J\\\\\\\/1LD0doZ3GTgVTOcOulqvlzQOMplMI21EkO2ZwuQ2dkpuzM4sJPSTJ4LI2jW+6\\\\\\\/FNtm\\\\\\\/ItbcW+9saF97aWAdzoKALgUnc3ZYZwgOecwufu1e\\\\\\\/sMkEopeLnmTWgeTLBQ5Gm0hdLvQQct\\\\\\\/nhjKYhtwnwDq2tcix8qCnvjy7AbQdmcSiPXqsWIGqqabiL966PbMt\\\\\\\/nog9zxUjlW3ORQCT9NHsazCv+GlGNss6z6K+XfEowLXTH3kY5Eai7W6qnwIfz3aQigsGfFkT3i+Hz+8r9i\\\\\\\/q\\\\\\\/YxwSHS6TIc7wzL\\\\\\\/5P9lb+v8\\\\\\\/rdCOpMIgaSaiX8lRrk4SWp\\\\\\\/cDDz8Fd4cJBdDcAjsFc+aCUhiS6Ds6I0IkViZwBt7hNPkDN94LxHMEEQmJPms\\\\\\\/9aHQMpQl6nElC+1hGBCKGCyob7MKIOC\\\\\\\/RkTll\\\\\\\/fC8XWcHb5aP+1y6HP1ulQxweNlU6UoN3IwOlV7lkASA4dvEzh6xWWEB\\\\\\\/DXnSPaGInup9rW4aSOngwt3NpwZ6EV78kkdQGMZSyX86ZYLZcu2w4ywokaWf79wriNwm2hW3vRT9cepSM1dyVwoxYN87fQVvLJiz5TPRWlUkWU8ckSM0YL4Xza2q9xnW1kS9CS\\\\\\\/Sp6LMSnKmJxffLxgBvdt4DNr4rHujLQI0wJxe39Nr0LDQhClxBBNy2i+bCWNVFBG93hfnzrTDndovjxWEjceRoW8wn7lYz56APZRqaD5bafk848\\\\\\\/cxqU1tLmtA6kS49mf0msF2WLfMSJEjHdnlZPFwKAIiUBem6b9i4i5uETUmgFBfiqwUSVTeVzu\\\\\\\/sn9hDDCxVp88Spd9oKSZjd4O8oUvzB\\\\\\\/Mc+vcALvpccV6trrJQ99sKC40UALq4nYWFG9XjO\\\\\\\/4g6jrUfIPY5zfAYE9\\\\\\\/\\\\\\\/OPtPPzEY9UY3\\\\\\\/fQvevAnS0\\\\\\\/CwllTymzx1tgxMEMRYepzKJfIKveSJPCf6OSX8VvrXuJuG3py9edOXRmrO2OOWBzNtXqqAtLBbalwxWCTQryrm4mciS4uxFMVc2gPZsuNdmfcobPn35Rh35\\\\\\\/wUc1I7lWrTH1+RMC+Af8zhPVQ9dx1\\\\\\\/9k2F\\\\\\\/S4p93YZ65sL\\\\\\\/WO67blUim2TGRGfFgHLOZ0cdxRrK2jKO6dgsoVQVKWDKQuHN1A27kyQq10DPhavYK5znI3NwF2ATaRDH3HQ2Uw4nJGsF8RzzHAUbE8y5KJirTFGu7JBcsFxxK9oV+bYHFIXMXiuoVWShAZYrn3Dp4Q8sBVJQxZoDW03MkK\\\\\\\/XJpK+kWKOXOHQsjVnrp7d4tAU\\\\\\\/I9Tpk+FgB2xD1xMzu0kS5zoIaXyd5rZT8ZgzJXlf+mnb641MZV43isX3OcwgenwQlO4DghWbfKKG\\\\\\\/H189YqPwG02b\\\\\\\/rYkL8d3Sq5khkAuoI6NXW1yXhAmFa958pTWxUP\\\\\\\/fe7E7Lep\\\\\\\/knvQagpbIPuA\\\\\\\/OEblFZ9zV+fQDu+Is+DJEW6SXyx0IeVSC+Yyevis81KAKHahTRS51urV2SwFbXpAbINvETBhorprgf3HKSJ+m0SMf+tDApjL2AOB\\\\\\\/F8WiAZts\\\\\\\/bjQIeZOt\\\\\\\/K6nJnA2iSqFYFXI23NnJs62uC+eebFXWuRtnY5tQ5knQUmK1nbPuY0pEGuJwvAezJCTXCSDcDp6AP5WNdPtF9cVBI6xY7Yb6RHsAKZEaOSzb2ecN+9\\\\\\\/6\\\\\\\/n7f9ZHUa91cNjJD7Pjv5F3OeY+ZbiWKGgEAnRedu8j2nJmeR8nH+4ToxY8PKhNL4q0dCj4aTVQpcQ8xsLlthNmUrYIDEaBgZHNV\\\\\\\/4Um4CcKiBhI6FVj6y6MNcr6\\\\\\\/tuNa7Ke+WthtlamHUUB6LQXLUvaenOKhhXljL2OBjL3NjC9FJsbsYmZgCXHRPRqnKBV+fSPtTH+915fOxqrmC\\\\\\\/TSTQID5\\\\\\\/82i\\\\\\\/Y+oZ3cp21HmzyX1WLwfOizIlYhJrMR4LEqV5jtZCRhLxZgXruq3sb0RHcCuCSMm+eekU3uwtnx6UTj0sRAtB7v1QfS5nSINsT4c5V7PbtqhL7vjK+LpM8qEYpI+vPa9AHxPYeANq2rWZWqO5RzBu1+yv40ZCr1YEd4wLes\\\\\\\/BDDhnSbhtkajsI+TXwApWZUzZWxeTGqydVAGm5HmQUtqgi80do\\\\\\\/mnB0ogB+ceilMgKcs0KvphG8qdszVBgFgTPlw1TsskLV+XjCJVkgJgDfnnFObPMCdwHuks1DJIP35o7aEktLPnAvPily03JS8sO4qARZc70Cx96Clpx0FTc3gMex06cUgkk18intnC3hxvaT6qYnYnlPivKVquIDQtVpKj6ChcZ77Jqso3ISR\\\\\\\/XBiP34NE7\\\\\\\/W035x7ihnK5kYGc9Y+1hHFKH0cbxjbp\\\\\\\/22hpbcsz57OjMa3bvauMG84IEqqL9cexqbMd+1uZp8U2yCdZGE2J92WZEIWFCl6LBvw1fw+R9RN9hk+ky+F10hYTdiu+M+UglHxM\\\\\\\/XXi5QrHUjxgY8qFcsteUSoPXR49w1T08oJ63sqxNT1G8JY17LdKiD\\\\\\\/H2Bgme5v9W+sPccY0w21HLPNBUnyMmarDGWT2OrHNF\\\\\\\/yGFSE3IrAy1+Q5j6GSLp1v9pehpUjC\\\\\\\/+HhDGxQgYomE3rV4d8wXuJWXb2AVDJBhlgGVN5uaP5Vjwl8\\\\\\\/p3Ybq6hHm1BjVHqY6N8El7xDCnh0nLIFnpNgh7nFvv2KDUCP3JaGZm1\\\\\\\/q6gePg2LF+oaU8coshsyscf8DCXIjHtDYwdSHZH56FDXPD0Zl60pT4Lu\\\\\\\/k3M5gSUmSC1POjlwYdy5fd\\\\\\\/J8qgLHKiZ0fsUeZUCiDSm8ceIfAn\\\\\\\/kyvqDKUxPxJ9rh\\\\\\\/M+WWqKfxgnxgwFFnOAjCcjnJQw84Z43tu4ksQUa\\\\\\\/mT3HGyTfzJe1qg8a\\\\\\\/jhR\\\\\\\/67wWHXFt+fYUjcRVtqd6H9d1vm9QOfzKR7lEvSseauQX4zp9Cgvs\\\\\\\/vzoQDO6woCrt\\\\\\\/7lEcAUQcsrivkLtEMjmq2+JIzUpmBi8ul\\\\\\\/U4QaYvoVgwBUDG+6NwHu6p\\\\\\\/KdH477JwCFtkdgUH3+EPVYSzq1B5iNoB3QDFcNoVmAaW0p+njknfx7eWYpmBuvFWnQnGsGxq5DRIDGqv\\\\\\\/vGZmViDepooM0HDY3IHBaNUm0kVoSs9aFrJeSx0QZEJ1nIA1mPfO6XksUvdDwfPL7y+cDBCfyHnB4llXrZ0rmvpJpdQ6GgnIxoxbRgJrhl3nJVBV\\\\\\\/MGEVQbKv\\\\\\\/MPYBrIMW7V+jOk08kcQPWMml8MTudEyurM6dO3JkJzQ30BIlRKnFEuAMi499zFcskT6h3nwQqijcVmwk2c6\\\\\\\/WZU2MKagoq\\\\\\\/SQK\\\\\\\/p2DpiUYpPcqabt94BFb2LYTk+Kdg+Sxtq1QZtwIeVVkmdcz2UxkELedmkM7xFM9cs690\\\\\\\/Qp0EiE7RPzfeETgFuk+fiX9V0gBZzPhxMAJHjwWEK8j3CSiN52PcaqJHwza1ASLb6LFCHjKzvHRJfbwnaBGojnuPsguJmWdZNobTPZrycaRQUHAmzrEJcHQqXhLyX9XE9vXv978Ot+bxaJzdB3HLZtrfEbObizzh\\\\\\\/yGswmWIvaGEkXIWq\\\\\\\/J8U4pkbPYDHODl2NVZrXUvCZGm0HUbHiBHddshkEMQYXq8LWU8qXNRif3AoE1cBQYc7brYkscF5Ytb7naOruqYRCOwOreFrm8RwidwVHECL9kkq61IyKvl9+454CQm+RG5Ob0a948swHAohE0cZZZkKNwloHjM9k5bQtSaDZpyyZrv44jVTl7zLQrYud9\\\\\\\/ubcRJ40LRIOq5mMu9XGg72LCVtT+XL6P8+PepORXdEAyvAQ113DNYJcpgz3vxmkfGZ+lhp4MeYjBg9fhkfDg7OgzO8idOVt3X8xAXIiZDY0w+PkvernH5MGvVItki+iPzzw5BC7LVP+ZHrknVKwrkTcebfxhbZ\\\\\\\/6HCKpbnUIq2dCOTRDQqpE0dPsjTkuqBm5LyICVhjAfwMjerydMJ1JL5bU4aCuQ+TXrDfUmOBPC\\\\\\\/XPF1tviXVuq2oPcdrNWcvT810kWYQauRllgDkIz1oNX5dOpSRt\\\\\\\/nIXIPpL1ENnjk7Xp8+ydVKaqzWfO6jNU\\\\\\\/HOUGsCfsFuciGisptsJzbYP7ri7F2eZrMPhN+4o9NCorh9p0qpbgi4F\\\\\\\/5fgJwW87jIwodU77I\\\\\\\/LDRSSfVEB5+Ai\\\\\\\/xMOaMl2nUpCC6LS5G6isT7cmzcBL2qRb5Gbh5OVQmIUyqvK3g5NEFDwlwx\\\\\\\/m\\\\\\\/QQfmzzKcj4CVoHz8RFwNQGIrJ0JqMTA+82D\\\\\\\/058l6qkYgDBl9fKaVhnnNV3z4ycUr11AS\\\\\\\/ML6+8djnoS+2FPk3L1ZjG02OZ+FV9zKhjRc8+LqulBc63OukQpoRfVx566FuhpRHdM7KVz0QZuehRoJHVC7QlX5XZNrYgUlZIia6tyYmwPBORowvEbzhcx5U4N14GltqmWDCv09MFlcHqh6tDOjpVn9Jt3cYH8\\\\\\\/X4qvmeneD4Wjc9xTgAcl5aF1TYCEdcVeZoDMntkctGd9Uro7TpU3Ul2k2P0GtW4FerphO5nznuvT2MiRpRF8icqSKGU2j1uTej4rydz79IN5NWNxdYK7P5PIbKulw\\\\\\\/d\\\\\\\/FA4zMoBybvffYDYd0WHqvAD2EkXWJcLygWG1ueVoFmLu1\\\\\\\/GOec9WDF9ChVf2orw44glhSz6Y34LzoRazqj7dLtmCKmKy76klD0RhMYbQAtpCy1ZHqT1XJjHdS66xKGUMr+KrFH8528XWyr7Oetg9+pSUxPUJYMO5cvwKTXmClVVQNLoXriDHsln8sV6IIXWBFax1MAxiXdGVJ60tKMRpdlTopShAJx+xwVe+aTUjCCPB8VchAoiDvgWxDSohFplSYmExXbLe6Eoxe867EZxxoHWfA7ecnWoYY6sMnBoHFOldR6N3w7+aMejdiIkF8LywfUjsCr1D7UT6PuzSpTj111ixSwfUk+FQxkum6Dk01AHGKMF5PCTlYUPY9LR+\\\\\\\/ZqBXrDBWIXEEQVcwvjfQcV\\\\\\\/e5vFhmkJiAA5JPvQsSj1sVkqK1zrmEqRXYq+kJNNEGpqDkjYgawqxOedXioWUgHRNeb42RpiKoOw\\\\\\\/aqeovpurYd5JSnoE2RfuCfWbMq9LHAprhrSzl\\\\\\\/8Y9IxBz0tbcdXxg5zvSo43k4JKzRLr75mqv7Si1xF9znwIC1g5g7w+56pBuM8eFxI+5He2GOcwZiTXxFB0x5BIHaBN8t8h4ZboWxokaFTgDlFTrG9Ig2y8uadvNXfdn61YZeaJDNhqHOuqim0qBrupa61QB5EtR5HgHc0h8v\\\\\\\/g6tS1gpHPIlivczV8cc0N6KFwcUohR9q3bxZpG0ck68XGBAptC6VGnqWrBGYwJH+BOSqaz6wb\\\\\\\/UbReSYOKiYSWoRzx81aNOt+DnOPGMvIRhJPM96nwfF945nOp0H+c3izvhx3gYBAoW3xKQQx\\\\\\\/OB+ZaGEOJYohPyGqjAtK07EGV7581daoQ9CJn8jGpDdnO7SnqNztO0UGc0uH3xDG2ovyePB72ESVa7\\\\\\\/mw6e3aThgd\\\\\\\/IMsV57WXuMBPBT3cJg8UeNYT0XQUkObRnXcQrVDFdpYrXsBkUZvJ\\\\\\\/a3pSmxgNuspPJmKw6geltbGFdQpj9YpDxW7RapsQLyWPyoJV05Gt9aBP9lsC4+3A8ZmLISzkIuvH\\\\\\\/YRb1ENo4viniLKznZYUFjO5vxTLP6HvAJBVH73MKVOTA17yEHYlTiS9USMAjiSiH4b1Cq6cHCma9ulpE9e4IPkaEcBu68iyoy2Wdi10HpLHeR8JgmVXzG5\\\\\\\/kLvm28R+2HccAXoDZ2BgNOrVXgOPDTAE75yj6xWuZQ3G8ctNkHTpYdaoAUNql5L6HBi29QnzofGJfuJA+4V4f205q39+bYy6GXmAjAyo9W0opjGOEMEWV2w4gvl8w3NHi2cz4nbPm8s4lEJj3vEoubiD0aiHnvc5Ayty2kpmNASfExzLM4djtXSHp0E\\\\\\\/ztA3O3Z6eoGJL\\\\\\\/SHNj9uAiy8nJRllbLowU8ghj4C9+XG3bosD\\\\\\\/baBw9iiebk5wDK4yviLv6khPtAOTWMyEdSJFxYFno9B8CF3KZErQ3l20GzQ+MJyz0GUb98EAjiuEnowAII3nyJBBUtS3d3mzHWREI9t4qphpzpeeKtl7ltg33QdrcJSzyIT1QJ\\\\\\\/wy2ACD9UoWY2d3L26iiTgxoR8qnQVEgiRuMjik2KxGZhbIDXSDNbSaonIWZUC\\\\\\\/bf90bdzzuhXezDXv5s66zgS6IjlRxvSe5IdoIXl6mqYpwkQLpNoWU4cDY+awvMKtEd5i\\\\\\\/TcqdFdgzpIp8nGotHkLyMat6n+4uHs72rrl+LeNIwK9zxVFlpFcHp24KsAABOwaOfiIfW3Tp3yOJb1WwBaXVyO\\\\\\\/xsjsi0Ww\\\\\\\/03L6941l8V9QQNrdcD3kj4v3GGRdayuXqqaj2tEtNlFLrRwcjOSoRKmUTz53BdH8sNS46IbbD0S5DI0XbK+Fj3TI1wZAH2m1z5w4X7tSwzaBAimsIn5SKN99NMv7ySRQ9Y6yuMKkBQoREhPr8eYQc+EyLYzk3eQ+o7rMeroLnKgHeJ1\\\\\\\/+K288EU7EhoWVUp8qFslr5qUO7lCcpmOIc5OGZqAg4y+3FsEDGV6AVJQf6t8e6fU0hIt0FpxoJspqVJKdWldb7187wOaG7lNuCWn+PB9j+aQefNACecynQe31XOyJ4CSCVAGJukA\\\\\\\/f9U99J0LvnS7MTbuoE4esyxRQCoi2DQaMue2RpewE5coBTOq39QmknlpXTsWuox\\\\\\\/Bqd8KIviuJ+1ff7lw+itJ\\\\\\\/kSUW7TjL723fHJKUORo7IxvPCYwg2nRv3KZLsU\\\\\\\/KNUCRmBVj0GPsyb1fS3+KTcJZkY7mWxfDXukG+e29N3+3EY7UFwOOcEEB3jWF4HdCic04kYfb3S+H0LrKtKOKXuWlUJHizNyt3jyVCR\\\\\\\/CUrvPj4bRwLctO+2A6e74MpBHCv+0kIU86OzIyQjpvmGfrVi4sf4\\\\\\\/vZ2f0iB5DqjxFXuY+f\\\\\\\/raFmtz1epOrIkA1y3ZDwwYebyRod7h3zPKq5XLGOBjLO1gs\\\\\\\/kdwqV00lcOT01KDNo0fjxEKAdTl1Z\\\\\\\/UUYfU\\\\\\\/5sfIVD0vj23M5nBKiwTUSZ9iXBSsSyhv0bIHqh6b2dXevhIsfkJGrNJEojygHZQd5Wwfp5EDt2ZSFkNOOBDo25dtP\\\\\\\/GOapiq4lsxiipKt49Nbs96mBBphGS4A0Gmu7dGhchrLOQvszl+q8\\\\\\\/urjQPulni7XmAVhSkTsRB2Fv9ks5Ggv83ZJQQElV3BhkTHA4AgrGXIGv8B4+MxxVldpdOGMgQJ16DCDVns+2esGLvP3XtrTllmqStjinE+ehY5ctpiSh3GoRTVqyXdsKCNQRoZ3NXJIy6zGu99ENMyHFh4lTkEYLOL+\\\\\\\/ApWSXIDz+piBn9SsX5wkfy20zLrwn5sAeFL+OLxpgQr3toFQKFeL9kmpDfadSYKqiC420zbN7XcqYpNyo631A4iJs4kDD6xjRnIUP4Y0dzyHMA9nL2Oo7gU5zJO8jpNEKMCQ8orfON4NDCRid2y4VojrQBafiG2VKEaNZj4KAxHMAQQDT9UfMckgEJFhArF\\\\\\\/GPEPoDi5IpCoHGzmDwWEHULcQP5+DEgEENFNKb51N5lB5Bb4Vcb14+CAFq+tmh6hI7sFHGGBhMFc4fY70s8t39mpPxSZYJum7yNyowg3kmk+hE3lQdDyXCtCmdrsXSKShfPIMaGcyKFjL9Z9\\\\\\\/abGditF3EaRARJd2amfnS2CucvlCfJZJ4cpkO2jhvvzLu7MANyglwFXejSqtJTZnda\\\\\\\/lryH0rCjDDODRszhmaKxkoTakN+wExHEuwNgN9gjsE8eJ8u0CsMy5hXGdFCbidwdNn61nQKw549GxMeLlU4v2QEs1YutdMkMpuALc3zTuB7YSidf6DUbNdXm5HHDaV+clic+4LiSXrLxBuV+ibzoKb2coa1x\\\\\\\/SVKf\\\\\\\/zW23p14ySNl5+qXkqI3czSAGk0zJcdDGo8zrihs6WBHltOrC+LEkWw3m8m\\\\\\\/AEvELcDpMRHGYUfPa3pWV1WOUrMAMev1cLoHQUDdNE\\\\\\\/dgCLH5uGwS2rOc\\\\\\\/ItzSqaxR9gO3vhGJWJ9VcZF5KPFtdEepkHa9kXKfFcynAly3ymkX29nCl0LprhGpirNy8KoX8EJYIaRmi16WGooCwUJOsbBezdJXatkjfySpw1exTNVi3u0TBCHKeN\\\\\\\/LETCjPJm2w\\\\\\\/U4I0ape\\\\\\\/9Z++Bt5Ef5z3DIiqxgQhDS9PIifjjRX8dUIn5ziIcixQ8MjVaaaBNKmFCWnXbv9Dl3pG57\\\\\\\/CO8GSNvc0gUPvqfLo\\\\\\\/ZzhlkxEMkwojVyoBdhGIJnHgb4kVCxp1f6+Tt2R5URDKPHW3LfsidIH0q+8hITTSuZIO8dMxH5fmW9UJXzu5r0i8bYw67VYj7W8VjxIN3ed0XWsaTyLADeT5QMBoVsK54XvNB4EewizcB2RpLkKkC\\\\\\\/Tk6n1cDO9zkGBFWXsnseXHEeZ+5LZa2rJl+i1Bkxz7l5k6sjteuR8wFtc7plPWGfpkFZ9YmZg5JKEOSQaV0wbzA2bPt+wsDgo+pVvagXlpgACcLHrjDB8zsP+kgGYfV4Nn0GGzWxPCUNMVbJPOGfALvrZpsFD4vTNGp2TqSa0OZPxDf2HJClvv0EHnIzpx1PuZWaeqP8IdSV\\\\\\\/iLiHPJ\\\\\\\/JJKiSpa1wxhj2c2Y3IPMjQ4kldT9f0gCDIrrRty\\\\\\\/zYEImqjAjYW7o8vXkRmfy5Z1riyQdwFx7galx1OJajFzq5zg2rD6WjxHV9mDg619MY27zx4rsL0XN4mYMTQGn9vkMPXt74PEhKUskKzaRpBnTkhDDXBSowfsrBEG4EbTki1Ag0vvsBIumXZchrD7s2O6IGqNjr6trhNaFPgnOK1RUpx7V7GuXrfw5G6\\\\\\\/Bc+KHjquJfSQ\\\\\\\/ubdn+J7BccMZ38tYHKwoOjQhDv6BFjIcz+cwu7fv2LaQRjlgkB3jei7r9GqfuXDExOtdn+yY5x2M9eA8PnAZU26MOudF6sJhXygmhh2U2MJGvnD1sSjIGaHm0nMXQADwy+NMAJrAoRwnU0\\\\\\\/hEvPn7XBtaF7FEqEoLxZF5+YBoZfBbqD5vcRewTom374776PZRujq4fSEc+Y8ZM6sOPiKubdImqgDGOK7ybip\\\\\\\/3se34QgddTXjmlTPGrgz2DiP48uqQbjmvXHIG0m0DJ4zU4qkj1NGi\\\\\\\/3tv2TViOwmE+Njd2ppPgS9c5GqaoDgpkdEDCCiCwQVaqCG1CJNYt\\\\\\\/WaSRPgYK0yjV+23CnI2RuVAtWRmNtgSqaJAVqICjh\\\\\\\/euXkjf83CHFtbKYNWfYygCnTkSAt9OsaSjY8N4NJ\\\\\\\/ev8WTHVHvf13BNiqc+4oSRGXoQgaLHcFTzlYIvFoqWsqfQuifi\\\\\\\/IUAHIALSgcRxTisUvH+0d+Ix762UWPPlhS9UgCtLcAXxJUYm6KW\\\\\\\/SmsUK8nc6gXdV+\\\\\\\/Lgh8T\\\\\\\/wbSq60WwEF0NTS7OwM1RMUUoJICX5Eq7xcmA3PF4FRQdWXE572i\\\\\\\/2oYLouYeYOCqTtMqaGvcKda9Iwl75FKpykX4fLzMwVaSI69kiSZLo4rHOX15wqk2Lv+A8UsKWL3Ge0qcDs+V0NGHnsNfEoeiL5bbAJgoalJ10p1DtOVpaEKgMjtsaYD6c4QUtzuDSLwuAIdEkB3hxwpfU8oP4VVDt3VzLsYs10YV36Htj3eOzIa1z6BomZB9A9z1nAnQvxDzPXJ3BGK0r3ac0wg7eL93GAWqrb8gFPNT\\\\\\\/aWr57FTkXIbjBszhqEr5lUWtH7uVarwLRQ9HqWGaUt7Ld6AIiNBWYkEFIc\\\\\\\/AKZysNNx29cWR2I79qmi5raie4BoYda3U\\\\\\\/27lz0yvA9mxHMoYfj38yKYGaOpPe3wfMCgK1XYRc5pyhul1YvmejLNwD7G\\\\\\\/N6bNGmo+7KHy33sBBeDStxHLUXqFRDuQh8PuTfDy+CJJe1xP\\\\\\\/sGekIQ5DbJ0AYijsyHQstPhfsXdc4FU2MkgySdRyhBcduEWHrPTM5Dtc\\\\\\\/Eomi6AsVcb95xiouhnQkIwRxfNij7\\\\\\\/RAMdSdsAO1h\\\\\\\/aW5oNnTVEXb02YIxR4Gdryz+J1Se5E\\\\\\\/PLya7gJGFV\\\\\\\/7MdZTfQtfB08ri+qfrYWnNgtE84Gt6pF43wEUnCDltDvIkTo71GdvqesJUyv7EORWsyt7p+ojTY8Yg8MwFDvcg5\\\\\\\/AsCX4NJUR8oAi\\\\\\\/KV\\\\\\\/VMKCLPV7RnrijuPqJJh8DX0znELpampB+JtbFvrpvWDasJljVZLGpXG5CjBL\\\\\\\/qS9bdB1qAlC9fbQ8pRNsZw9pWJoUuCrk+K7hSbf4t3l4TzpPh8XtUmIkVqfbHBWGaVWaJI8BfXAEghm3N1D7+LePLEtXl3u1O75uFXkKV5+JLGZdugGNpb0WPYbJxhxpSEDmJcfKT\\\\\\\/V0gW\\\\\\\/jKNkV6+lHIPGzip0I25H5EUZIpVp4jE772NPbdRDeIBMyIeljquSmSkxqSP4Fffd78vgztZNvjUprxEDByU5Q+H4ki7Q\\\\\\\/MlSPYQn0wx0brR+L7QJig3s+6HxWl2lGIVYptxpkQkrAo7q9xQ9EhMLswlcFcScdji6CbbecYgW1TIaVYGpGVauTZXA+EQF5ftaleXmtIqBjjHaDwXyZxHJ0TwpMt9GSl5Uxv8cm6zYEz39emkPl7pmiqNVSX5295UB7aHNihWH4iMH8bhLpa33v2v1BPkX5VTgQ6k8bHvHHE6ec8MgrLU5jLgAs3xGVa6dlo6wVWCNgkyxsoVcPtylTK8hyz7CF+T7z3QHVkuZ3G1hOWHE8ukAOBG6jE2JB73iKujSyDQBwsz0qFTgZLU2l0mYLg\\\\\\\/IVvbRrFPS0iicw4A5yXNMYUR9ZuCTrCWyVvcEuMEsq4JERolUTIqtl+XZvMzM+rgxJrGUhxQLGuo97l7JE4uEX\\\\\\\/enKdIN\\\\\\\/vgG5Tf5O6Ol6g1ZMe3lj+aXnHrt1CZNeLdvifGoUzoDEtEIPtxtLZYKpS4W\\\\\\\/UPMd3rUTLZOTpSafbJl51c2Eoto4GoTshdpGOxyfvtx0rHyUFHJF4WzX0JHclQXTdoYgfaT2H39HGyyWQ\\\\\\\/k28JlcZjVYlG1jYnJEwzr7pYLBatSQ2xolvQccS+oi8CrGg2qL22nnJpaq9EeBnzZtVMafbFg0fah17wFbAlErep9xmM1IYFHgD5YFB9p+FGv6hxVWP+a\\\\\\\/9pJM0x1tMSD7de0cP+L6tFz5Rm8ymZXzwnhSauSpSholmIMjbugdyPE154xSNBkvKK+NgFME3Lmq6EthVuXnetEFJe9RHE1A4Gyhg92AuswuDSkB6FpZiIOwi7uuqqmsmmcs4WrtTuIKOxeVTcEwAeVi45+n3PJsCi45wLob+Sck1UuypMzAvpkHImFDmkHzmVd+tYurX5UzVEjPk3wUMqSkfJb3NR4\\\\\\\/FOKqPYr\\\\\\\/8F6io2XesXJnJt+kUFcREuuScJZmrUUgQW02O3ZlJrUDMLVbRL4U9MdCBnBpERGYh4LxueBimw1axx1JNZusXKF\\\\\\\/9+PblvjIrwBVJs2L2crW4HKFhcR0bnVw16Zcdm0R+L3FAgPyoQDXSZ4LjPXDwDZvpaSB+p8eKkT82xcoFbT3skhCDp+BzkrnIvmUERiJ457aobRq\\\\\\\/q7ax6QGWUKfAQTO2nGIHyEOnz88MekojkqnNURdM0my0URtOKGtvtaXUxkHY3r2UeeeRDZ9T7lbwtbwVuJa\\\\\\\/KWAfDabETlTgn9oZMLjkqEcCPw\\\\\\\/IrH95yVbEiTlfx\\\\\\\/2k8yaurr12WjhRE6qkDUIezerZHauZYBMFS0FOx8dWuCDkcYEXDxArLlYWFu4TQ+ueqd+s0tOGkeZ63A3RelhUShV0BzC\\\\\\\/kP4llPJKnBAAkraej7htxjXgwuYatouX+1hO6wAONozVr5zqmIeVrDLUjL74X\\\\\\\/arY\\\\\\\/Gi9EkPzt1E\\\\\\\/EtSR9O6kUZTtM1sb7DWyA24RJS1J1fsQ8Da3g0HHZd5L281ozJaL4xhucde1wbyw5he5wc5KIhm9n3VF2QN3E8hlrzPEioOkzQcNlb6qgmX\\\\\\\/dD8plBbqjKGGISraPS4toOXD6iox1ywIbUg6l1XuoXBlMMdrMvErN1hOAeHY6r+3EPTc2ROG50MzMopuzWWN0D255xSkTbO5TVYURVHsP6SnjaP6T3EX5lyGtmv9kmQC7iC24nYs+hRTxONU5DZrg5hX+p\\\\\\\/YUc42EePH5g4hVOCiokE2RF1JZ+OQO\\\\\\\/RwhfpZNaij8yxtGTwxOGTv\\\\\\\/9\\\\\\\/NJ0dgSKLNIs\\\\\\\/FJxIwIPmI3zQmGaxuLjSukeN6\\\\\\\/eNnhhk+DcqWYpCusyit64JPpg4fArHAAAKkm26vUYA6K5JcKXcReCtmi27CseyRfDxI+Pfw7j8niRWGkyo26+M\\\\\\\/cntMRCL7J+psJDgbHqlvHiODGuKXDbpMS6HkAfApIjjOFxlWs7vxhT0Ku5DIKJ8P98\\\\\\\/ieJMdkEz67vB2aX2\\\\\\\/mqo2MzA9JQDJjsZKvVr+p1yeiMQc6s1o9t26bk9hSEQ6qKIWBcyATvT0N85BIovF0OlJ3TeVi3IWYR37mcREL7B7sfqXcbXupBCl3DgiNeCWtQDm34s9dCcI7L\\\\\\\/hB0GfIX2+EjS81gDOkyi\\\\\\\/B7Q4C15Rl8AmkYwe0sdYAsk\\\\\\\/jlftTCvhScG1xrtT58Xtn\\\\\\\/LvZEqMyJs5u1cK7PGHVn4GqnQAg9Vo3KnrYa9wtjc+NBjEleFyXRHlKzQV+GYQAzhwvjMhKrjf4EwXdu3HCoBIVX6uOKKjwXzO\\\\\\\/+HqiCjMMPv1d0+d+viq5wIctVJ4uJS7\\\\\\\/fvb7EYHcog+6Mw8lqukz670A\\\\\\\/5is1mFOOPdpS7B+YPyi+MR\\\\\\\/HRyFP3OL\\\\\\\/7ItiveTrMXFgUtiZ1anmMSUKQEDubJBnOwpJNBMg9x7HfyU2J97jQgZV4K+zW5di8pQYXbKjbC0ZTzzXaLOxEGWUVmEGbGCR51bIFdoRoX6yynqC9V7epq8ym3OdbJRrXQOXo\\\\\\\/ZPVNyvf5auVHQR0L44UwwaKtnCSKtgAQ8atVL3ZAFyD5CxXPXJOe9yokylSlpr6HrJkQ6SE8Vv5+HK11pVbDFc4AubDUbfVTYu4f3P2RI7qletBRRgppngCkW\\\\\\\/P26k4T0FmKEC1kUjo08V6WyA5xraS9pfdTKX4XYQj6SXiTnP7fCDGkzlnzVUnCY5RKf6iDZp5+GaPb+l8yLodiooXlrZmtIppgn4VsqHMarscTisucADivbNkwv3T+Dmp97pCxxbcs0Gyd7M+JHlJCThvuPnPvTECww3vfjZZuMQQDy\\\\\\\/9r\\\\\\\/v\\\\\\\/tNC6+W5VoB5vDat31rntXBKlvqeuQcYZRl28A2YPGaD8AxmFDRpcFEz4R+pf4DPW13HKKOMobj+xPRmUErev+NzkIF10wDeCQoXggnoB3YiNG+NDrawgVIm+cGjlH7PwW+4SyypuFr3tQYAm4WvYyXh+Xd30Pid+EYadGgOKoqMCRo\\\\\\\/qGRC4mCIPPf3gBTRa4l2x08gJxXcrTZ0pHptD24zR5SaNGmUWz42b0Sjps5tcpen1\\\\\\\/tJYXxc\\\\\\\/jNYABeiMiBwUUFKM69c38cdehpF9F\\\\\\\/rptZInb+vRWKISkv8tWSn5ZIvOXQD63XOIYT+eXAEN6eIDAiopCZYyauaqZMOF0+Un5Z8OziTGbHrGQktQxkU\\\\\\\/mJVlAqhcwa\\\\\\\/HzoQDa57z\\\\\\\/+SFpzanvAIP156P4UfolJv5NsTHx8DGHNWJA\\\\\\\/l3EbEG\\\\\\\/SEdh3R8QM8DswPO7\\\\\\\/pl7ubdbFvQa\\\\\\\/h2EMeOiESh\\\\\\\/Y9qfBaidVGrRfvbUHNQxWGz2hoez1AVr6HyNzA\\\\\\\/TXUnSEphABIXEjkPEsruuQB8tof+TbVRxzq0Yg5NiBwQ4Mg0vxL+H7SKQG2Z\\\\\\\/rfFgVZO9jsxABNQGoH971nG14J+4nBCmbaA9+ulzXDNePPLQNmLlSINB24mAZrUUnVZrXmdDzTjH\\\\\\\/+Tp5+7hGUZZjj1VTtZqfcgicP+WNUAReyyJLSFPpnjE6lTTI1eR\\\\\\\/y5UMzKIWOWFLZKRosGG0OLHPdKtN1nnXuxvOaLnhdrmAviQ3RYJOtVUwaixCD2hIhBA8fB6284ITmCmEPbVui7Vjqdk5JGjzvoY4RSwU\\\\\\\/VaUC+j6\\\\\\\/\\\\\\\/UkDGKEU+BXtzktXM2iGT66dY6e9T3lNos+x02XmCpuyFiPa+J5JBuZ726UMWf\\\\\\\/5DR3byeAJ+snhYoNEQW6RAKwPsPL79jqfQeeXSrlmAyk2hp30rfFtQ0SRtTgjy321xymoVDCSSNJtufr7NUbEnxedPhGHx6S\\\\\\\/3+GZNnJyYlrKc6Ys\\\\\\\/cDCsRz6CO\\\\\\\/UIRvXLTgEHd5TTGEvMfcn\\\\\\\/aVkI41fNORpf02w2cvANyQCcTFy9+bL+f5CDKgipRRmPp+5bcA8D\\\\\\\/zyJQP2RLHnlnre2ihlh0QIjBqTNFJVV0W6PRvElVGQs0aZ3fGt5LrqrxiVBuAZx78LDx911b7ckgRrAuxKpi\\\\\\\/Z2u\\\\\\\/osFVd3wuGXvOQ4ybs\\\\\\\/pB4wa807k4+eV3apWYL5jNZdDA95JgBcF67Oa23WzGgXHaq34pMtQY1Ocj83kiLBo2mEgbUAbwB2gl7uj6\\\\\\\/9k+EwawAtj5uwxv1ZQXIPImsmdlHAQfBFtUurZ34wOlj60iPJ0599YTdRHzQVMgOqhtkY3C5M+fqZRPSWPTpYvyT4xcOZBQWDekKksgtqgInFj5NLFpxw\\\\\\\/rfI\\\\\\\/ZzXT487b4VNazvk8mSXuCWtP99cvN4TuKxAA4i9OAjQrtfSMEZqPW\\\\\\\/fPV1DynwQwyb394uCJ6mu2KbVRf80dWezvq+pB9ehNJIj7AfCGoRZH2vQPfnAtL7HGr3SarpiqZB452wr6QyRQZjMc9WTQFmFhoCJniM9pi\\\\\\\/guzREJq4DwilaQDm6h1PMFYMF1Bp38LXYg1HK1OSbYfEFOiQxIGw6+L+hBfO44pW0U4tuwwjHYZ79WVb0WXGQo0QiHq+YkTZTmqXdmVGzPbfD6dhetWcD6g0Ci6pPH4WN0iyw9ILczYKbawyCjIGIPEvbintmvmx\\\\\\\/YnhBFI1lQ2zHthp2V5Mugt\\\\\\\/9D4o4KZb76gmgbx4s\\\\\\\/DnGsduZLeo3EiojWzZ4sr0ZcFYBmoGC4N\\\\\\\/3kflpJcmhz\\\\\\\/Yxjy9B7+np\\\\\\\/0KZRNUEGpqCI3uyLiUF6\\\\\\\/n6sdwMV\\\\\\\/sqC2C8jO93hzv9NTlDoMsmdzy6DoaLPwjriYJPr1OEW0dBUn9AWjf4viKB7LtUHUrcuUgyOdY0AOjFjV\\\\\\\/ZM\\\\\\\/aXH\\\\\\\/8AKauFWmKmzRsi\\\\\\\/aVYrooQ2O9jegHcZyemDYOa\\\\\\\/TSNX55dRNN2UGE2HrvATGh\\\\\\\/5\\\\\\\/5b8deMk3hku\\\\\\\/+TdvDZ1zDaYzKJyyjoeeXw8pBXOve8W\\\\\\\/+X1okzCCy+k7V7W1kbZPJ7ZCBXzfFx7Z7fhNoGXOn7HA5ymuf8u0Ef8fCzyrng3xe\\\\\\\/WgyDwFrlj2r770ZwPSd2KiC46Yt91nE1P08aSB8vB2zsduEGGxI3snJpWsF6p8KUIGLFiDC+N1gfeAl5G4fK5fE5XLkfv7zxd8rWT5l+fAiOiyOhTS0TUCkoqAc9hK+h6cHFuzDXsaX2GqvwXGaoOG\\\\\\\/nPNoIoGAhFSLcmUtNAunPyTnbe0tGQg\\\\\\\/aH6QX2q7k9zlwOSB+TWyMDnsy6UyPATE8BP\\\\\\\/J3+M2A++i5y0\\\\\\\/IRSTZZ016xOeo6LrHunGvAlBBlKvHJLWHJBSDsGeF9524F+pz9HzQZTvD9JoDApgWXMA9KztAgQfv260sLssLq8o+y\\\\\\\/MkyXUrhhDH3R2P\\\\\\\/iyayuI6VVSBUsp7tdoh8b2up2SFMmD\\\\\\\/NGRcFQGonsSzZa78ueYqv4BGMaeteJ\\\\\\\/aLUQYVb8HT+Ka3ogIubDYZVLE5cmPZs5hF\\\\\\\/Ij3Q7iy4KJgWTzPrOpOK5ctHLRSSs45Tftlvxz7nwnjmVqGOnOsYvXdcg5TrTLxdO6qZEuh5CL36zn7irZSuCascM1smvq\\\\\\\/kl2baxI6OJoO95kzaO5xh7SVdLPGPd66ZUHcCeDARN+qfP+nr4iN1OFRn87e\\\\\\\/BMYxDuDctBw\\\\\\\/pup3zbn4wy4CxEufqordk+yfVgOTgkzKHwdhhU5V5FoxkNZEjMyG95kg5Tpno9+UKJ+AX6XABgS4HI6KS4PlS3g9c9o2PwjqOq2phpDgHaSZpiMOT54jrHT3R0jPIKY3b8Ja15aLhXcwo6zkZ5Q\\\\\\\/aKSwSCkbhna5zupL09HSSfYG1k\\\\\\\/zq2Y5aBj94Ro+TlbiRyJ9ANWChScwqvH9Ff40uk5PK9pEB55TBnd+D7GIwKT8tT5V9s\\\\\\\/7Ws0lfUlZBfTia\\\\\\\/8um4VHMncHQoUSWPNLf\\\\\\\/iPnC2wrqOemu60DSiXcun33\\\\\\\/yE7ttuY+70gWGYd++gayyZBjfdt\\\\\\\/2RhMWoVnhlgUyp\\\\\\\/l5MXivv3\\\\\\\/TcKv+CtPitEHrPrnRcogG5T5ahcSyR0bmXvLdChSozlJY6bCbJ\\\\\\\/zaWwYAFMyUhjVy6uR2vT64VxysmZXFk80L4YcnNxBqJcBeKWLuUJjHfsMScOzANCCRlUjqZB6p1j5TD+RcuT2rNy7qHH7SF2ADAGQZZulAZWHUh5qWj1IBHmQPIQv+GsW8kggDnB4HEh49crYka6Mwmof0IrvDMUuj5KTxn1MWqihIfNXVw7hDxKclYmGoo1yvzQGyNV6eLimjpxi2YH8hp3\\\\\\\/tQE12IjjKu4u8ytT3xg\\\\\\\/9UnVVKE2gvcONMCAOC5yhMXv0kPDTbhU4IrlStd9mDCB7NVGes6aSUbY5qqJ9noz\\\\\\\/FOhtP97QBD\\\\\\\/rw38jlIxK3QVe3fiUoNmAAtdCJzGOFJgHKGmQqA3k91Bx3hiwu+TLU\\\\\\\/DYx6X3gZZrZF41QC0\\\\\\\/uJklPHa9m7eiew3LjPBMM8786hv0+tKwCwSZ92PqZ\\\\\\\/wEHOsjh27cpugqRjLg9R31sEqVd7XieU8JEow+3tA5MZLBrp\\\\\\\/fkJdrAHxa7D1AtBfV+ws+fNjxSr\\\\\\\/IGDREqTgRdeAAmPmWTNwtEikfplPgxbYUxhyu2+0p7RfdA9sX0eTjlS9hOMGnntr190e6HThWDISU42l8YChVCI9yv9sARzfctI2BDCm6RBeMC1KNfNoc9Q3gg6va56ZwIkYVx2p2Z+w1H7iGHNma45Sx0KzYPQMjm\\\\\\\/Wc1pW5b1DtCiAGgWkXMAHzy9JY\\\\\\\/72JwEo\\\\\\\/QyTpFeMjpFuQpgeTS2v3cvWgpIZgjPZv7bOEEAbCL14coTDUdilkjMsfpiILNESImWk+K0iyYOhYf1CU12CBAxR95tz3WrKS0XFVCZlICvEuHD9TvLHdJowXsKK\\\\\\\/pG+QLZtD8JQgM9k2I3JXH8bHeErzVrH0ZVNUZZNnwc\\\\\\\/g+02XEcdX+vp3Rr6j3RuU14J+o0sFcDeg2+gcyriD7PBi4lKd4rZ4rD2L9PTii5P2BLzwqH9aVeFt50A22roklA\\\\\\\/FTDBOrJ+8k66MOORx2HFkk5kBp\\\\\\\/D02FlsMkWGKc6ttuoniNpOEFmKlAYFlpN8awsLVQOIdvYDbYQwUFu24GvRaxtvKhnjixJKlyr5eHuiXm\\\\\\\/4hb1cSNFMJi4MptOOFHy0eX0A1WRp90oiANRk6oW5Kbqeg7Qkg5Xc+C4pxBRpL07xG4T8AUJRs4HMP0z6sy\\\\\\\/66yzcjXtY6idjpw\\\\\\\/fpyZJKUxuRi9l2dza7syXJtkACHiJpgDu9gTKwIfVw8rpbhAGmEjKgGP+CVs6wpzVa5DdVaPVBuIkk\\\\\\\/HiQ\\\\\\\/UcwIfgB\\\\\\\/83la8zuQu728uhQRVlXBPHEOnEGJPKrWgrvZTL2kGmIct3NPGBpTgO8WxEgGlWt0D7QI+974bfIcdH0eIjeBSUdM3R4kxosA16d6QRMqKkeTZf3pFhqfyYY4lDDB0O5r0JFoH1wdLbCgelQ6fujptsMw4m4ABQ2TD2F8YGObn66Pm7NrgUGct5QsXXWCneLqxE+71L5vvuwe4GsbARTRVx9aqTW0n5Gd0vlEHIN2MKq1X6x5uuH69cGqcN0hvURsuEl73GWJuP23cJtCn5xHijwKA6WiVOgNYxvKJEllcoaB8+afILS8HaGr1wUnyHxElGjthCIaR\\\\\\\/1JM96ffF7UJfLl4zMxmFOccY8M1WNLs0ewxm7NHyvg7rnkWlYg3b6O45xoqlkvLNvACMyMru1OFV7C7h2+698XTHN8dpyurkRyKCgZKSz82zj0Rm6lPEPeMxmpNEz5VtTkxoAWymwPZ9neMVzlAu6EzTkwIQTZY6+HBbqWFu6znBAF1yBtTSTegO0ZsBHui70ZUhqr2Ku3fg8BifJTRvkke6HyyLNCPp28DvNfHpUhGpjCIXNc3ef33hZSSJ6HapY3dQL1KAVPoFw3Ts0CdAVHrzthdLE7e82wNYFo\\\\\\\/zfWfi\\\\\\\/euIjf1wZtqBvp+Pyd5LhvliQCy8powWzXQj+O8eddgYHvH4WRbj6lLpjOMr4l0jbtQZkqNwsP2\\\\\\\/fh0Ilc7NLKSKIw1aAXgLjJnsd5RxHbSHDo\\\\\\\/YFdPzmFEV8bG1TcdgbyTW6fnicOxC8TKEVgVxmh3+KG3pu5gg+iNdAXENMpCixZsFu882QTCLHIDF8SbVI07pWfJTGiBeeveg6n7ek0PxPlGKUMB+eWpWhhbL6MDUQSBKSpLRcAc4i9Y0811Yq\\\\\\\/xt0sM+VNnl5ZZ1kOmZVOJF37chVrTuHJk+aFRvuc2eG7vcgt+\\\\\\\/\\\\\\\/BUOD13vxOSCTlYIvkZsOpWFIe0981xL9jIrfV\\\\\\\/GD3INxsv0o\\\\\\\/FqXmRneZBpLeI\\\\\\\/IZUROorxiJqQ2\\\\\\\/aCkQevVWDuxkJJX6kbxl9\\\\\\\/2TCiE7TVwT\\\\\\\/ziHKR\\\\\\\/hijolxtoskGtCJTFrEHxRJByxKPSiEfJGJhB4NVQ+sITydb4A1yfeMoHM2bqJWZiv4Rwo57a1SerdZRhfvGCYRIUsLXRgv74z0\\\\\\\/GBcVTtgQLgpj65I+nPDn8Vb\\\\\\\/h+DzpOl\\\\\\\/MRlU\\\\\\\/GTgzefOCnBa98FgclZaLXfN+60CtfJpSlUj+UUei6JbwPuf0P1Lw5UZd5ucj9IPoFuxwOA0+s\\\\\\\/WPt6i6qE+ZknJO3KbNMYaRb9Ty+3x41b9Prge5BdRGuGsrz36UfjZtAA3ONUHc\\\\\\\/uOFKj7cHPVTBhwYW8YHkeOojLuwJ9+kUaxYZ+xayAsx0i6RgKJ9995hjoFezb+PeJgH0QJ+Ew7znl+MzxIwg7n9CjI59XfyeZwAhVMpWbqB1f+qexY0N2VF6S3kJw8Cp8JYQoaLZ97VuCeU73eAxLy7jrmYgt3MrnjavAxvEDbDglIcnTBYcQ\\\\\\\/z9YemVvMR+rqu8Ny3MV92fOJ7ZA8\\\\\\\/Vmjn8SCY\\\\\\\/ywjFIPakw938IYSTPnAoTbyk0iG94vmI6MCTVmQAu8mwyL+aBX91FbnGSapfWyzplpCeUVtwcfR7GmPRrdDyEjnbnTYlG47dC8Ix6\\\\\\\/KariqQNWtAbeaazmIKBIZpuO9bIjuiLDidCjTNv3gsjWtqW2rS2aOtFYrnLrChZb2okT+kegoIlGO9OhKDVnjeEJYoy819Lvff6TnEKlfyMY9\\\\\\\/q5JMrhjEu2ZzCopX0jX+uYNsJfP0NFKSttKc7t4oyMao2ytGXn3FRnNFFrIY3hSCRWof+xQJHEYFUT1HR1DQVo2N6osEG4a2okS3CnHQgcpTsPidVW+FTCtNAYz9laOq7vizExCdX8S3gpMdO+FWBZ9a\\\\\\\/saNux\\\\\\\/sesMzphNxMAQlcxJu5rZTw+reBO02Bn06\\\\\\\/\\\\\\\/BaKQxzugY1GSZcqx7b57APO0Rn3qyDuvKaUAs3AqOIrySZ+cGSAMpqtSXlZpxCDgAtHdnmILpClLhUryMbrKmLGqxHJ\\\\\\\/OeC1v5KfgYWOlILyL90M0wkaFimwjnYSx\\\\\\\/zbVU\\\\\\\/sHW1iWXxTGk16jTwvcugqqZQhQ4tmXPExJBLvml70nFosRnCPk4CzIoB\\\\\\\/T8eNlew7aqf1+MnJGVtWgTfcH\\\\\\\/hSXqGkO4+i3sYbX\\\\\\\/9SjWeGSx7oY96vRUqwgxu7ZWvOCLN+8CtfGFLrqCUrGlpd5YXMwJ4rjDeXlBsOGpt0TPP38qXUulbLqgD3JKSu1lJvFQy8m4vbfLbx8GTg3NTcXqnrafY0MPfgSijnGkpp53YuJhUcc6rJ+K7b4zhALmkH5T2udsFW6cWjtCA5xhhd6Z5S26lXn3wc+MRDH9mkr5UxnkdLf6aznODiCHHYz\\\\\\\/qqnDQ97r+q+u9tpgNOvvKTPPPWo32n4Kq5e5XJ5ISVnVX0ITNuGTa5lAxXFCCw21+2Pc0flBctKVVKXMBDhnrnqluSSu2VYp06WU1L7dF7at7LF+oRFEVnMDVuqe9xlzcCsaJSnzE685\\\\\\\/RmDpJeTS8D5lNNzTpGP4lE45BbB3HcBGdkr9K9Fgi7uvqkJS2WlnrW0wzeMSlRsCnTvyJB\\\\\\\/lXHuNWOub28hFWn424vHCBai8hWyx5MoYW2cniVLb2HMYHs1IPhKm+s5k5GAoCbFJOAWOXzzt0IVQRxNmPw\\\\\\\/UG9ZbyhmMDp1GfQf6Jip7FcopODsfblUplme0dKGaOO8djOdob5W3i8oXy8jVSiNcOSCs61+8IP8NmDhmQkufDGzlrPXMIJf+WPL2RaJ+9DKxrpGhbLzM7uZowpULl2tTtsZG8+BDbfIHQQvkfVs6bVcxD7i81rKMycAiHIUwp1I618b7VRJCn+Rnuf1\\\\\\\/ktJARue93mzPJtuelvvGGE+4pdh5xQ3qDxAvfzolkemgcgh8EeqHP24YZ9FTBpiTAiQLGgAzKCQZS0nJ801P8F7k+0b0C8u+pGHLJB4Fr5ZFFEbZumnedW4uMwBePGnqj0buuZtw37RyD9ON\\\\\\\/IKbVNImpsJBSV7IIC2o0JVpn4dYI9Y41vxvrsmO0hTZarEPVjjsZszEQgxAyQYijnh\\\\\\\/JQ+iQStKvizy\\\\\\\/i0Swxb2C8iz\\\\\\\/pWrxqGjlKHyjtVMrma7wmWTETaEFKD89ycTGtfkSlfTCZcW75GBc0jDlxzcXsqJ+4TTjbHxmLXcDUFwo\\\\\\\/JvBbZXM4IkW1Fi90nJH+RBiT\\\\\\\/fMid6WT4QwkrhrpKVOEE\\\\\\\/8C434M7NZlPfnSxNR0sPJ8K9VBjyBi02cXSJ259+Qp3f2Wo5XAM3yDI6+S7GAMm3l\\\\\\\/pMOWjx83KkBUn9IjGXs4ohWrY4qNTHv0hyy+8Y7rnsLdHJw0IdB1kprL5VMAM5NSaRlJeJb7RYffqS7rz81WtESP95YJqIQtIlJlWtaigHnAbibeuhX3g6Xc1xKOor\\\\\\\/\\\\\\\/bO39allg6CPnwwEvkXMlYl9kt1qtuMoq\\\\\\\/AIpIlUg8y+gX5h7U9XmF2Gk1cPZR7S6u20vmKnkyH\\\\\\\/BgtbqygyWiYDADjY4mnJAbSaACQ91+oEvJetLFrUhpuSHPpm4aKAmogYyzDW4sbzPzXryxXNccFF0+GVVnbcuodyEW6NX0jjYiT4Enqp12gGzN7gUf6fhSjLPfylSGB+QMc30PDJq\\\\\\\/BJrBdwDvWewNz1kLGVzQ6YBkeMBzDo8zV8+3KztQGyEZGVxX\\\\\\\/XSasB\\\\\\\/6xdxCfcKWNAGBRVLzlAo967J4zJw4ap2p3aEmBmOLHevS0xuyf7\\\\\\\/fVJwlwk0Ms1ncOZTg1t9+KdcE0skhs3YMgpJ5lHvMGhz+b0sHwIij8P+kHiQyEHHKVag9Ow4mHZ4Y+0N274Rgj8a+yB89pfOWC+fuHf5u87G1wV0TSye7lZnp1HH\\\\\\\/Rxuq756DwL0HvJCwEMMrZ9pXuzkNX+d1xtv4G2Yvnr9JW+qrW207A9KA8v5+FUKU+\\\\\\\/tuPTFwHhbI3CnobgURjfIlUDQ0suYeMhixwgU7uqiadv14Y14v7FlVc+Ana9CI38H8oCQk5Uxd07WtO6r6Sy\\\\\\\/\\\\\\\/pQ7DCAogeQjm0TrGp8X2rZ9vY8sRsVsjXzA1PhjUtt1MJYR4eKK58SAitrfM3b5YmVou\\\\\\\/N3JbOsgtlSdgEFb3M8duFymov85Xn3GHw3xB+SihYKs0zsWMQUQ2r6NwyAoF8YRbRUH+guyPLL7YsgsXRP+qfw3idH2t9az6ZW1TiQ+S4cJgYxeHImCOUwbLfRRKU7tiyy+zZCacMqTVwYOZaJ3Wfky9dg5KLECTVmKvIwGUJvo1vki7AToMGTTrtj7ZfUkUtqBrfFHOt1AINd0EHGX52EEKQJz\\\\\\\/lWSICa3rJHY3cmUDxD+D2v0LUEl\\\\\\\/mHqR5KBcqTdckw9109\\\\\\\/k3OF0Wkb3cI9JktDvDbvwRAQ73\\\\\\\/EiufYzZifWi3AbdOrN3mShv6zqPhR2ibVOzK9ZTerPbCTuhW1jV+0mVydcdPDwD+3faOP+cALy1c+IJ7e4M7tg33Uukpqj3AylYM8rinUaKgINYQW\\\\\\\/YYRK+BlPEaz1kqUCbT5D1kOAOfueb3Jt5fP\\\\\\\/GRMUYppVPQ7hFGzr+PSc85+ofA\\\\\\\/gyzHnH7K1Z2N5pBbA9NmMWNtcGBGGWzxlbKiFvG99A4Szh0XY8IBtslD6ssP16CwwvISmLgUAtZeARUr0WY7mEJ+gNtBNEUxkht4wTlyqQKx+GBSZnGz\\\\\\\/h3qGaPM4vP3IkwG9THM1jEyqyNLu5KuAsox232rsZm+BWlkmf45ZmLieQw33+nbb9Z6iXmGQMJAbLvyLLv7TfQK7LA99XsOaDKVtZLbZF1eWG\\\\\\\/760aKYq4qonVwH3qrg4BiaaiDTKEWXdoxJGV0iHzpNycQML9oxhqU7WpQTPPWJf8wfSkmvOREsEN5QxejHR8SIJwY4Xgwe2qEpYYeKBE07lBXobViu1NIjF9qxfTr3Nu43hqMzFG5hMeHOiuYKYtK032XWB4cLuB8p6B8o3l2dG2fyYNZCSJ5kmOICGxwA51j1m37ZfBbOxeooPKjS6pfC2N+AyKSgTbDYsrFoGDqaup3BOh79bQX32Rg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022mqAdx8kQpxY1cU2VDCHzEA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f7e38c54ed72f5c9b7a749f3fa7cafe8b2f1b747638688606b9d60f10b71f3e2\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/9WL38N0Xozd6Ilmmy0PK0YKuF\\\\\\\/QbF2aP7cC4lehd6skeEFfuNn85\\\\\\\/4Rl72v2PHWttLG9Mn9jHQpNHXhykUVtrIWAKzXYjpi6J2dynExK6FmnerYv+R\\\\\\\/JM1at+gc1sSxcLmJxLnAh3+AiBlQ7i85Cp5p1jMR1N1HukJnfDJZ0c6gvxj4hMe6AjYBO+b+M50cR0QzzXWeOYgiXx9HU+qAFdZVRzGX7K9UxA65fOXO5L8x9PQU\\\\\\\/kYk4XOJQt1chUEhb3SoEbXpQWrqG8vDhxKPGbq+dadca65OoPeBmAREzTYx9hQAlqSVFXsQwr1id6Zd7sIc87gINvn\\\\\\\/vEGKUZaRDE3wh7\\\\\\\/Q7BVYlW0ofC31RTaxHmDnYYGHB79YLRPLNMnlI2H\\\\\\\/NZpRoL0+NmiSBwR0xe\\\\\\\/AgXIkYnr0bi2fXY83UXZLMYhwVABRUHa+Bwau7x0ogCqJRo2utcr8g6p76o+vR3neVngZqZfufrPQ\\\\\\\/gHiZWFUbECjW5wOXJ7XvKKe0+n9p5MNEmTzH6YdvIt4zhXUt1TAd8KdgwWAjlMWp4z7RhtmJS0rgc4hGUEKiGZqMntDbqw1XlkTShovpJTBi+bTPXh49+hNH9gbAWX9iB15vcjrWP4dGyhFtFuN9n0CimnhbpwWdW4rGUEne5\\\\\\\/nWLN4iVLsDh9JysiNELELQRvHVRBToRLowA83DQA0TbdohSit4Zg5AgI\\\\\\\/imk6N5rPCH7D9A9liGRTmLvtDCC0DkOHOc0g4v9Ux0HYAzf5lBieVRmP6RPYsFM4h\\\\\\\/r52CYQCGPdEhJfcQRxFsY4iWUaBUUzENw0nmzo3bO5KixPawJL5qWPFMgN8mFzBoxLKmAmQtE26l7bLDkT3MRMq2gJ3kV7OpQ02QL\\\\\\\/XbqnDC8cpQE5n+5oO0bedTcymf1h5rsSghVNQQTicBxh\\\\\\\/pGS2UJb7+3mv05eUOBE5BkrtzHLOFLqC\\\\\\\/56Ms83MERhB+GT7g05J8WFz6AeJkC668bjMggHz\\\\\\\/ANVxt+czEGuqfv\\\\\\\/ItrDvwPNNybQrpbChLBSzGMHccCCbC7tBI+nmoBNrc+6Mf\\\\\\\/NPwhAWs0dnzYvfKA5q1AXFYX7aqXiuAmWdxyftvwxTd8ec1OYxyrU0bl+MLXT9VTHaLD75EdocLvn1oGMF+1nKUuMNCEux4\\\\\\\/iJ+jH5pCuqbFB0zQZ2Oe3QZH9jCphkjTbttP1NzgQzF+ds6KAGuoYQ\\\\\\\/Z9PfcbbGHfFam1FXpPO7AsfgxNyeRTQsdimcTOB1syLKlg53cA0015jP1YRmS9Z7WrGMYhg8G\\\\\\\/1lPXGf\\\\\\\/r7TUVctmqf+5Ie8bZ0gU2fHs46tXdrjwAq2KB94J5mRCltq3yYfHkyj4XONPoVI9lHtq0Tnl1eGu4ayjjwP+5fsSSDqDP7vXq8XeuAUd33mK+xkFgnq+UJ2XqUICl8XrYgCl0TYEdLrVWb2R3+G75hHfdS9U72lUvmVQq+TBNP+pRSFy+P9nFyPw5Lnsl2s32Pdi8Y9SAf2exRIdro6MGMlgukmCigsDoRRByqvwtRSUKRfhBlaqrjFE+D4tqpoLE2Y3bJJA78d\\\\\\\/TcUEb+liJ5MkB+PQsdPp5kfUkQ3Hj+mc9bFsxJcpcK+7CUTA5Hby5f0JtO6+GcblS7df1ZLTIfldpYYO3HVX6Jfzk92AyAga9qyeSeuAjrV3RsaR2E7EAmf6JcZN\\\\\\\/WBxIV7+R771+k0ZSXer4STUiGjC22hBERfQB2TqCA7Vn7nT9XZAUULbWRPdPAXrJ5w11ru78DqCY5EmwuhwZN5+uDmr\\\\\\\/T+x3Kv+lls3ciNSVBD3fcgW\\\\\\\/nKHlE8JD5CqFRXbIi1+fWLQpd90zc162MHN8dQjp1SHKDMS2\\\\\\\/UM5cEiSvspYXi9IPkTfZCBi\\\\\\\/UxphaWS8T032nXhamboqUEbLt8EY6FIufFminWYIElYQaMeOLsLSUdcQyCr7qE0FgSGM7EP7wWUro6Tl1bRxLKzudAUmerQk8R7XsX\\\\\\\/3vYRhL8M\\\\\\\/09SioEGihVl690AwRjFQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022xWFEvbDIvghgXFzOM5uA4Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229c0b007167dfd14ae4e6333b3be852134550cd4f3770a03458be35e452284764\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459953 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1043" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-6VoMeH2YmFmxf3o6_LL5QsMNsdYLZWD1LS1uvoif7zc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1043" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-fAMVT7wVXIrOMSrkxVLI8pF7FIYpgrzoo7JLRRfAoS8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/6/1035" title="Novel Genes Affecting Blood Pressure Detected Via Gene-Based Association Analysis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/6/1057" title="X Chromosome and Autosome Dosage Responses in Drosophila melanogaster Heads" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459953" id="node459953" data-pisa="ggg;5/6/1043" data-pisa-master="ggg;g3.115.017913" data-apath="/ggg/5/6/1043.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Single-Cell Based Quantitative Assay of Chromosome Transmission Fidelity</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jin Zhu</span>, <span class="highwire-citation-author" data-delta="1">Dominic Heinecke</span>, <span class="highwire-citation-author" data-delta="2">Wahid A. Mulla</span>, <span class="highwire-citation-author" data-delta="3">William D. Bradford</span>, <span class="highwire-citation-author" data-delta="4">Boris Rubinstein</span>, <span class="highwire-citation-author" data-delta="5">Andrew Box</span>, <span class="highwire-citation-author" data-delta="6">Jeffrey S. Haug</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-1185-7283" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rong Li</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1043-1056;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017913 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459953" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jin Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jin%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJin%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dominic Heinecke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dominic%2BHeinecke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Heinecke%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADominic%2BHeinecke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Wahid A. Mulla </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular and Integrative Physiology, University of Kansas Medical Center, 3901 Rainbow Boulevard, Kansas City, Kansas 66160</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wahid%2BA.%2BMulla%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mulla%20WA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWahid%2BA.%2BMulla%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">William D. Bradford </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BD.%2BBradford%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bradford%20WD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BD.%2BBradford%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Boris Rubinstein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Boris%2BRubinstein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rubinstein%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABoris%2BRubinstein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Andrew Box </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BBox%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Box%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BBox%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jeffrey S. Haug </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BS.%2BHaug%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haug%20JS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeffrey%2BS.%2BHaug%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Rong Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular and Integrative Physiology, University of Kansas Medical Center, 3901 Rainbow Boulevard, Kansas City, Kansas 66160</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rong%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARong%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1185-7283" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rong Li</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rli@stowers.org" class="" data-icon-position="" data-hide-link-title="0">rli@stowers.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/6/1043" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459953" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459953/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1043.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459953" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459953/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1043.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459953" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459953/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1043.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459953" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459953/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/6/1043.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459953" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459953/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Errors in mitosis are a primary cause of chromosome instability (CIN), generating aneuploid progeny cells. Whereas a variety of factors can influence CIN, under most conditions mitotic errors are rare events that have been difficult to measure accurately. Here we report a green fluorescent protein−based quantitative chromosome transmission fidelity (qCTF) assay in budding yeast that allows sensitive and quantitative detection of CIN and can be easily adapted to high-throughput analysis. Using the qCTF assay, we performed genome-wide quantitative profiling of genes that affect CIN in a dosage-dependent manner and identified genes that elevate CIN when either increased (icCIN) or decreased in copy number (dcCIN). Unexpectedly, qCTF screening also revealed genes whose change in copy number quantitatively suppress CIN, suggesting that the basal error rate of the wild-type genome is not minimized, but rather, may have evolved toward an optimal level that balances both stability and low-level karyotype variation for evolutionary adaptation.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/mitosis" class="hw-term hw-article-keyword hw-article-keyword-mitosis" rel="nofollow">mitosis</a></li><li class="kwd"><a href="/keyword/chromosome-instability" class="hw-term hw-article-keyword hw-article-keyword-chromosome-instability" rel="nofollow">chromosome instability</a></li><li class="kwd"><a href="/keyword/qctf-assay" class="hw-term hw-article-keyword hw-article-keyword-qctf-assay" rel="nofollow">qCTF assay</a></li><li class="kwd"><a href="/keyword/dosage-sensitive-gene" class="hw-term hw-article-keyword hw-article-keyword-dosage-sensitive-gene" rel="nofollow">dosage-sensitive gene</a></li></ul><p id="p-3">A fundamental requirement for mitotic proliferation of eukaryotic cells is to accurately transmit chromosomes such that individual cells of the population inherit identical numbers of each chromosome. In the context of this study, we refer to chromosome instability (CIN) as a measure of the likelihood of mitotic errors occurring in a euploid cell population that produces aneuploid progeny cells exhibiting gain or loss of chromosomes. A broader definition of CIN also includes segmental or structural changes of chromosomes (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Geigl <em>et al.</em> 2008</a>). Recent studies have shown that aneuploidy, due to imbalanced chromosome stoichiometry, alters the relative expression level of many genes and can lead to dramatically modified cellular phenotypes (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Henry <em>et al.</em> 2010</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Pavelka <em>et al.</em> 2010</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Torres <em>et al.</em> 2007</a>). Thus, to maintain genotypic and phenotypic stability, euploid organisms have evolved intricate mechanisms to ensure mitotic fidelity and suppress CIN (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Musacchio and Salmon 2007</a>). In euploid unicellular organisms, including the budding yeast <em>Saccharomyces cerevisiae</em>, CIN was estimated to be exceedingly low, with a mitotic error rate on the order of 1 in 10<sup>6</sup> cell divisions (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kumaran <em>et al.</em> 2013</a>). In most somatic cell types of metazoans, CIN is also expected to be low, although direct measurements have been more difficult. The majority of solid tumor cells, on the other hand, exhibit moderate to drastically elevated CIN and other chromosome abnormalities (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">McGranahan <em>et al.</em> 2012</a>). High CIN in tumors often predicts poor prognosis, and there is increasing evidence pointing to aneuploidy as an important form of driver mutation during tumor evolution (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Davoli <em>et al.</em> 2013</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Potapova <em>et al.</em> 2013</a>).</p><p id="p-4">Understanding the mechanisms that maintain genome stability in normal cells or cause elevated CIN in cancer necessitates the development of reliable and highly quantitative methods for measuring CIN. During the past two decades, several assays in budding yeast that use different principles were developed to measure CIN on a semiquantitative level, enabling fruitful studies of genes and mechanisms that confer accurate chromosome segregation in this model organism. These existing CIN assays are based on changes in growth ability or color of yeast colonies triggered by the loss of certain native or artificial chromosomes (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Stirling <em>et al.</em> 2011</a>). The chromosome transmission fidelity (CTF) assay measures whole chromosome loss by using an artificial chromosome. The a-like faker (ALF) assay uses a readout that can be the consequence of a whole chromosome loss, chromosome rearrangements, or gene conversion. Finally, the gross-chromosomal rearrangements assay only detects terminal chromosomal deletions (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chen and Kolodner 1999</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Spencer <em>et al.</em> 1990</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Yuen <em>et al.</em> 2007</a>). These assays in combination with genome-wide open reading frame (ORF) deletion or conditional mutant libraries led to the identification of 692 CIN genes in yeast (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Stirling <em>et al.</em> 2011</a>). Despite their usefulness, these assays lack quantitative rigor as the result of two main factors. First, the readouts of these assays are several steps downstream of the actual CIN event and may be complicated by defects or noise in intermediary processes such as mating, colony growth, and/or colony color development. Second, even where CIN has been elevated by tens to hundreds of fold, mitotic errors are stochastic rare events that require large population size for accurate rate measurement. However, examining the use of existing assays to score thousands to millions of colonies can be both costly and labor-intensive. The combined effects of these two factors make existing assays semiquantitative at best and prone to fluctuations and false readouts, especially when adapted to genome-scale analysis.</p><p id="p-5">In this study, we developed a single cell−based, quantitative chromosome transmission fidelity (qCTF) assay for the measurement of chromosome transmission fidelity in yeast that can be performed in either low- or high-throughput formats and is based on a principle that can be extended to metazoans and other multicellular organisms. The high precision and reduced labor costs of qCTF allowed us to carry out two genome-wide screens examining quantitative effects of gene copy number changes on CIN that we report here. Our results validate the qCTF assay in yeast and demonstrate its utility in studying the genetics of processes regulating CIN in eukaryotes.</p><div class="section methods-materials" id="sec-1"><h2 class="">Methods and Materials</h2><p id="p-6">Strain and plasmid information can be found in <a href="http://www.g3journal.org/content/suppl/2015/03/30/g3.115.017913.DC1/017913SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/03/30/g3.115.017913.DC1/TableS1.xlsx">Table S1</a>. Polymerase chain reaction (PCR) and quantitative polymerase chain reaction (qPCR) primers can be found in <a href="http://www.g3journal.org/content/suppl/2015/03/30/g3.115.017913.DC1/TableS2.xlsx">Table S2</a>.</p><div id="sec-2" class="subsection"><h3>Construction of the yeast reporter strains RLY8492 and RLY8493</h3><p id="p-7">The MATα locus was inserted into the pRS305 vector, containing a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">LEU2</a></em> marker. Together the <em>MATα</em> locus and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">LEU2</a></em> marker sequence were amplified using PCR and transformed into YPH278 to replace the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006779">SUP11</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> sequence on the present mini-chromosome (MC). The resulting strain was then mated with the RLY2626 strain, which is modified to have the a-specific <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002869">MFA1</a></em> gene tagged with 3×GFP::<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001378">HIS5</a>, resulting a strong fluorescent reporter. After inducing sporulation, the meiotic progeny were screened for His<sup>+</sup>, Leu<sup>+</sup>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005654">ADE2</a></em> growth phenotype, resulting in the desired RLY8492 strain. To generate the RLY8493 strain, the <em>MATa</em> locus of the RLY8492 strain was deleted using the <em>natMX</em> module contained in the pFA6a-natMX plasmid.</p></div><div id="sec-3" class="subsection"><h3>Construction of MoBY-ORF control plasmid RLB884</h3><p id="p-8">The MoBY-ORF control plasmid pJZ013 was constructed as described previously (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Ho <em>et al.</em> 2009</a>). In short, the KanMX4 module was amplified with PCR from the pFA6a-KanMX4 plasmid and subsequently cotransformed into the RLY2626 S288c wild-type (WT) strain together with the <em>XhoI</em>-linearized p5472 plasmid as backbone (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Ho <em>et al.</em> 2009</a>). Yeast colonies growing on SC-Ura+G418 media plates were used to inoculate SC-Ura+G418 liquid medium from which the plasmid was recovered using a Zymoprep Yeast Plasmid Miniprep II Kit (D2004; Zymo Research, Irvine, CA). The isolated plasmid was then transformed into the bacterial host strain BUN20 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Li and Elledge 2005</a>), resulting in the bacterial strain RLB884. For confirmation the RLB884 plasmid was isolated from the bacterial RLB884 strain using the GenEluteTM Plasmid Miniprep kit (PLN350-1KT; Sigma Life Science) and subsequently sequenced to confirm the presence of the UPTAG- (TATTTACGCGGGA-GACTCGT) and DOWNTAG- (ATACACGTCGAAGGAGTGCC) barcode.</p></div><div id="sec-4" class="subsection"><h3>Construction of strains for dcCIN gene screen</h3><p id="p-9">RLY8493 cells were pinned in quadruplicates from 96-well plate glycerol stocks onto SC-Leu + CloNat (100 µg/mL) agar plates using the Singer RoToR HDA Robot (RO7026-100Y; Singer Instruments) with 96 long Pin Repads (RP-MP-2L; Singer Scientific). The MATa haploid yeast Knockout (KO) collection was pinned from 384-well plate glycerol stocks onto YPD with G418 (200 µg/mL) agar plates using the Singer RoToR HDA Robot with 384 long Pin Repads (RP-MP-3L; Singer Scientific). Strains of both mating type were then grown for 2−3 d at 30° in a low-temperature 815 Precision Incubator (3721; Thermo Electronic Corporation). When the colonies reached a sufficient size, RLY8493 cells and MATa haploid KO cells were mixed on YPD agar plates using the Singer RoToR HDA Robot with 384 short Pin Repads (RP-MP-384; Singer Scientific). The cells from the RLY8493 strain were pinned onto the YPD plates first, followed by the cells of MATa KO strain on top. The cells were then incubated at 30° overnight (O/N) in an 815 Precision Incubator. After incubation, diploids were selected-for by pinning from the YPD+ plates onto SC-Leu + CloNat (100 µg/mL) + G418 (200 µg/mL) agar plates using the Singer RoToR HDA Robot with 384 short Pin Repads. The cells were then incubated at 30° for 1−2 d. After incubation, the cells were transferred into an untreated 96-well flat bottom microplate with lid (222-8030-F1K; Evergreen Scientific) containing SC-Leu + CloNat (100 µg/mL) + G418 (200 µg/mL) liquid medium using the Singer RoToR HDA Robot with 96 long Pin Repads. The 96-well microtiter plates were sealed with breathable Airpore Tape Sheets (19571; QIAGEN) and allowed to grow for 1−2 d at 30°/220 rpm in a Multitron Infors Shaking Incubator (80120805BC2; ATR Biotech). From the resulting cell suspension 100 µL were transferred into 100 µL of 50% Glycerol (1:2 dilution) in each well of a new 96-well microtiter plate. Prepared untreated 96-well flat bottom microplates with lids were then sealed using Seal &amp; Sample Aluminum Foil Lids (536619; Beckman Coulter) and stored until use in a Revco Ultima II −80° Freezer (Ult2586-9-A38; Kendro Laboratory Products).</p></div><div id="sec-5" class="subsection"><h3>Construction of strains for the icCIN gene screen</h3><p id="p-10">The MoBY-ORF library collection was transformed in a high-throughput manner into the previously constructed RLY8492 yeast strain in a 96-well format using the BioMek FX Laboratory Automation Workstation/Biomek Software (717013; Beckman Coulter). To summarize, RLY8492 strain was inoculated into 200 mL of YPD liquid medium and incubated O/N at 30°/250 rpm in a Multitron Infors Shaking Incubator (80120805BC2; ATR Biotech). After ∼12 hr OD<sub>600</sub> was measured using an Ultrospec 3100 pro spectrophotometer (80-2112-31; Amersham Biosciences) and diluted to an OD<sub>600</sub> of 0.1 using fresh YPD liquid medium. The cell culture was further incubated at 30°/250 rpm for 3 hr after which the cell culture was transferred into 50-mL centrifuge tubes with screw caps (21008-178; VWR) and spun down at 1734 g for 3 min using an Allegra X-22R centrifuge. Supernatant was aspirated, and the pellet was washed twice with 5 mL of double-distilled water (ddH<sub>2</sub>O), once with 20 mL of 0.1 M LiOAc, and then resuspended in 1.25 mL of 0.1 M LiOAc. The prepared yeast cell suspension was then added to 1.875 mL of 1 M LiOAc, 0.625 mL of ddH<sub>2</sub>O, 2.5 mL of 2 mg/mL denatured/sheared salmon sperm DNA, and 12.5 mL of 50% polyethylene glycol 3350. Then, 100 µL of yeast transformation mix was distributed into each well of a 96-well half skirt PCR plate (MPX-96M2; Phenix Research Products), after which 10 µL of MoBY-ORF plasmid was added and repeatedly mixed using the BioMek FX Laboratory Automation Workstation/Biomek Software. The 96-well half skirt PCR plate containing the transformation mix was then covered with breathable Airpore Tape Sheets (19571; QIAGEN) and heat-shocked at 42°/220 rpm for 1 hr using a Multitron Infors Incubator. After heat shock the PCR plate was spun down at 1180 g for 5 min using an Allegra 25R Centrifuge (425752; Beckman Coulter) and the supernatant was aspirated. Pellets were resuspended in 15 µL of ddH<sub>2</sub>O, resealed with breathable Airpore Tape Sheets and shaken for 5 min at 1500 rpm using a Eppendorf Mixmate (PCB-11; Eppendorf). From the resuspended wells, 7 µL were spotted onto SC-Ura agar plates. SC-Ura agar plates were grown at 30° in a low-temperature 815 Precision Incubator (3721; Thermo Electronic Corporation) for 1 d. Transformants were picked with a 96 Solid Pin Multi-Blot Replicator (VP407; V&amp;P Scientific) and used to inoculate 100 µL of SC-Ura liquid medium in an untreated 96-well flat bottom microplate with lid (222-8030-F1K; Evergreen Scientific). The cell culture was incubated O/N at 30°/220 rpm using a Multitron Infors Incubator and then mixed with 100 µL of 50% glycerol to make a saturated library stock. Prepared untreated 96-well flat bottom microplates with lids were then sealed using seal &amp; sample aluminum foil lids (536619; Beckman Coulter) and stored until use in a Revco Ultima II −80° Freezer (Ult2586-9-A38; Kendro Laboratory Products).</p></div><div id="sec-6" class="subsection"><h3>High-throughput CIN screening procedure in yeast</h3><p id="p-11">For the dcCIN screen, 30 µL of glycerol stock was used to inoculate 1.5 mL of SC-Leu medium (1:50 dilution) in a 2.2-mL deep-well plate, sterile pp wells w/conical bottom (M1810S; PHENIX Research Products) using a Biomek FX Laboratory Automation Workstation/Biomek Software (717013; Beckman Coulter). Subsequently the 2.2-mL Deep Well Plate containing the cell suspension was sealed with AeraSeal Sterile Microporous Sealing Film (LMT-AERAS-EX; PHENIX Research Products) and incubated for 24 hr, attached to a TC-7 Rotor (500484410; New Brunswick Scientific) set to speed level 10, inside a low temperature 815 Precision Incubator (3721; Thermo Electronic Corporation) at 30°. After the first growth phase, the block was mixed for 5 min using an Eppendorf Mixmate (PCB-11; Eppendorf) set to 1400 rpm and 23 µL of cell suspension was diluted in 207 µL of SC-Leu medium inside a 96-well nontissue culture plate with flat-bottom and low-evaporation Lid (351172; BD Falcon). The 96-well plate was shaken for 3 min at 1000 rpm using an Eppendorf Mixmate and the OD<sub>600</sub> of each strain was measured on a SpectraMax M2 Multimode Microplate reader (DE05224; Molecular Devices). After this step, 30 µL of the cell suspension was diluted into 1.5 mL of SC-Complete medium (1:50 dilution) in a 2.2-ml deep well plate using the Biomek FX Laboratory Automation Workstation. The remaining 200 µL of cell suspension inside the 96-well plate were spun down using an Allegra 25R Centrifuge (425752; Beckman Coulter) set to 3000 rpm for 3 min and the supernatant aspirated by the Biomek FX Laboratory Automation Workstation.</p><p id="p-12">The cell pellets were resuspended in 200 µL of 4% paraformaldehyde , shaken for 3 min at 1000 rpm using a Eppendorf Mixmate, and fixed for 15 min at room temperature (RT). The fixed cells were then washed twice by centrifugation using the Allegra 25R Centrifuge set to 3000 rpm for 3 min, the supernatant was aspirated with the Biomek FX Laboratory Automation Workstation, and cell pellets were resuspended in 200 µL of phosphate-buffered saline, pH 7.4. For the last wash, the fixed cells were spun down using the Allegra 25R Centrifuge set to 1698 g for 3 min, resuspended in 250 µL of phosphate-buffered saline, pH 7.4, and shaken on the Eppendorf Mixmate for 3 min at 1000 rpm. The plates containing the fixed and washed cells were then stored at 4° in the dark for flow-cytometry analysis. The new 2.2-mL deep well plate inoculated in SC-Ura was again sealed with AeraSeal Sterile Microporous Sealing Film and incubated for 24 hr attached to a TC-7 Rotor set to speed level 10 inside a low temperature 815 Precision Incubator at 30°. After the second growth phase, the previously described screening process was repeated with the only exception being that no new 2.2-mL deep well plate was inoculated and the current 2.2-mL deep well plate was stored in a Revco Ultima II −80° Freezer (Ult2586-9-A38; Kendro Laboratory Products:) for later confirmation.</p><p id="p-13">For the icCIN screen, the same process was followed except for the fact that the cells were first grown in SD-Ura-Leu liquid medium and then in SC-Ura liquid medium.</p></div><div id="sec-7" class="subsection"><h3>Flow-cytometric analysis of screening samples</h3><p id="p-14">Fixed cells contained inside the 96-well nontissue culture plate with flat-bottom and low-evaporation lid (351172; BD Falcon) were transferred into a 96-well MatriTube Storage Plate with clear tubes (MSP096-A; Matrical Bioscience) and covered with a SonicMan Pin Lid (SL0096-P19-SS; Matrical Bioscience). The 96-well MatriTubeTM Storage Plate was then sonicated for 20 sec at 50% power using a SonicMan HT-sonication instrument (SCM1000; Matrical Bioscience). Sonicated cells were then transferred back into an untreated 96-well flat bottom microplate with lid (222-8030-F1K; Evergreen Scientific) and placed on the MACSQuant Analyzer (130-092-197; Miltenyi Biotec) to count the number of green fluorescent protein (GFP)-positive cells in the cell population. For both screens, 125 µL of mixed cell suspension was run per sample. Then, 0.3 million events were counted for each dcCIN screen sample and 0.2 million each for the icCIN samples. Data acquired on the MACSQuant Analyzer were analyzed with FlowJo software 7.6.5 (Tree Star) to calculate the percentage of GFP-positive cells. The previously recorded OD<sub>600</sub> readings and the differences in frequency of GFP-positive events were entered into Excel2010 (Microsoft Office, version 14.0.7106.5003) to calculate the CIN rate. The loss rate [frequency of cells that lose MCs per generation] was calculated using equation (9) in <a href="http://www.g3journal.org/content/suppl/2015/03/30/g3.115.017913.DC1/FileS1.pdf">File S1</a>.</p></div><div id="sec-8" class="subsection"><h3>Isolation and sequence confirmation of MoBY-ORF plasmids</h3><p id="p-15">Sterile PP 2.2-mL deep well plates with conical bottoms (M1810S; PHENIX Research Products) were thawed at RT. The complete 1.5 mL contained in the well was transferred to a 2-mL Eppendorf Safe-Lock Tubes (0030 120.094; Eppendorf). Plasmid isolation was performed according to the standard protocol for liquid culture of the Zymoprep Yeast Plasmid Miniprep II kit (D2004; Zymo Research) with two important changes. First, the centrifugation of the culture was performed for 3 min at 13,200 rpm in a Eppendorf Centrifuge 5415D (022621408; Eppendorf) and second, instead of the Zylomylase incubation at 37° the cell pellet was resuspended in Solution 1 was bead beaten for 10 min using 50 µL of 0.6-mm acid-washed glass beads (G8772-10G; Sigma-Aldrich) on a Fisher Vortex Genie 2 (12-812; Fisher Scientific). The ORF region of the isolated plasmid was sequenced using the standard MoBY-ORF primers for the 5′ (ACGTTCAGACGTATCAGTACATCACGAGACTACTA) and 3′ (ATGTTACTTACCACATCACGATAGGTCTCACGATC) position.</p></div><div id="sec-9" class="subsection"><h3>Manual transformations of individual plasmids into <em>S. cerevisiae</em></h3><p id="p-16">To transform plasmids into budding yeast, we inoculated 5 mL of YPD liquid medium with a single yeast colony and incubated it O/N at 30°/230 rpm in a Multitron Infors Shaking Incubator (80120805BC2; ATR Biotech). The culture was then diluted up to 50 mL (1:10 dilution) with fresh YPD liquid medium and incubated for 4 hr at 30°/230 rpm. Salmon sperm DNA was boiled for 10 min in boiling water and then put on ice for 5 min. Cells were centrifuged for 3 min/1734 g. The cell pellet was washed with 1 mL of 1M lithium acetate and transferred into a 1.5-mL tube. Cells were centrifuged for 3 min/1734 g. The cell pellet was resuspended in 1 volume of 1 M lithium acetate. Transformation vials were prepared by mixing, (1) 12-μL cell suspension, (2) 5-μL boiled salmon sperm DNA, (3) ∼200- to 500-ng plasmid DNA, and (4) 45 -μL of 50% polyethylene glycol. Samples were incubated for 1 hr at RT, after which 6 μL of 60% glycerol was added followed by 1 hr additional incubation at RT. Cells were heat shocked at 45° for 10 min in a water bath. The cells were then transferred to selective agar plates and incubated at 30° for 24−48 hr.</p></div><div id="sec-10" class="subsection"><h3>Increasing gene copy number by genomic integration with qPCR confirmation</h3><p id="p-17">To integrate additional copies of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002840">NPL3</a></em> or <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002161">MCD1</a></em> into the genome, we PCR-amplified the corresponding gene from RLY2626 genomic DNA (gDNA) and ligated it into <em>XhoI</em>- or <em>SacI</em>-digested pRS306, an integrating plasmid with a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> marker. The resulting construct was transformed into Top10 bacterial cells from which colonies were picked and amplified in 2xYT + Amp liquid medium. Plasmid isolation was performed using the GenElute Plasmid Miniprep kit (PLN350-1KT; Sigma Life Science) followed by sequencing with standard T3 (GCAATTAACCCTCACTAAAGG) and T7 (TAATACGACTCACTATAGGG) primers to determine the presence of the inserted gene. The plasmid was then digested using a restriction enzyme (<em>AseI</em> for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002840">NPL3</a></em>, <em>AflII</em> for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002161">MCD1</a></em>), which cuts once inside the middle of the gene sequence and transformed into the RLY8492 yeast strain, containing the MC. The transformed cells were plated on SD-Ura-Leu plates and grown at 30° in a low temperature 815 Precision Incubator (3721; Thermo Electronic Corporation) for 2 d. After incubation, 18 individual colonies were picked and amplified in SD-Ura-Leu liquid medium, from which gDNA was extracted using the standard protocol from the Masterpure Yeast DNA Purification Kit (MPY80200; Epicenter). The qPCR analysis, to determine gene copy number variation, was done in two steps. First, three different primer pairs for the gene of interest were validated for their specificity using haploid WT gDNA and including <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001855">ACT1</a></em> as endogenous control. Haploid WT gDNA was diluted in series (1:5 dilution) ranging from 100 ng/µL to 0.0064 ng/µL and mixed with 5 µM forward + reverse primer mix inside a Microamp Optical 384-well Reaction plate with barcode (4309849; Applied Biosystems) using a CAS-4200 Liquid Handling System (CAS4200; Corbett Robotics). The prepared Microamp Optical 384-well Reaction plate with barcode was sealed with MicroAmp Optical Adhesive Films (4311971; Applied Biosystems), spun down for 15 sec and analyzed with 7900HT Fast Real-Time PCR system (4329002; Applied Biosystems). Second, the primer pair and gDNA concentration giving the best peak specificity was chosen for the actual copy number determination. gDNA from each of the 18 colonies and one haploid WT was combined with the previously determined most specific primer pair as well as four endogenous controls, namely <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001855">ACT1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004550">TUB1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000364">CDC28</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005185">ZWF1</a></em>. The Microamp Optical 384-well Reaction plate with barcode, containing 4 replicates for gDNA with each primer combination, was prepared with the CAS-4200 Liquid Handling System. The prepared Microamp Optical 384-well Reaction plate with barcode was sealed with MicroAmp Optical Adhesive Films, spun down for 15 sec and analyzed with 7900HT Fast Real-Time PCR system. The analysis of the gene copy number was done using the Biogazelle qbase PLUS 2.6 system.</p></div><div id="sec-11" class="subsection"><h3>Construction of <em>NPL3</em> mutant plasmids, <em>ESP1</em> plasmid, <em>MAD2</em> plasmid</h3><p id="p-18">Two DNA fragments containing promoter, coding region and terminator sequence of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002840">NPL3</a></em> point mutant were PCR amplified respectively from plasmid pPS876 (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Lee <em>et al.</em> 1996</a>). DNA fragments containing <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003330">ESP1</a></em> or <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003567">MAD2</a></em> gene were PCR amplified from RLY2626 genomic DNA. Each fragment was then ligated into the <em>Xho</em>I-linearized pRS316 (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002840">NPL3</a></em>) or pRS314 (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003330">ESP1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003567">MAD2</a></em>) plasmid with Quick Ligation Kit (M2200S; New England Biolabs).</p></div><div id="sec-12" class="subsection"><h3>Yeast doubling time measurement</h3><p id="p-19">Growth curves were obtained from liquid cultures grown in 96-well plates and monitored every 15 min by absorbance at 595 nm using a TECAN Infinite M200 plate reader at 30°. Before the start of the experiment, freshly revived colonies of the indicated strains were grown O/N in selective medium to maintain the MC and plasmids, refreshed in selective medium for 4−5 hr, and then normalized to the starting optical density (OD) of 0.05. The growth rate was defined as a value proportional to the value of the maximal slope of the log-scaled OD curve. The curve was interpolated by splines of the second order, and the first derivative of the interpolated function was found numerically. The maximum of the first derivative was then computed generating an approximate value of the growth rate k, and the doubling time was found as Ln(2)/k. All computations were performed using Mathematica (Wolfram Research, IL).</p></div><div id="sec-13" class="subsection"><h3>Protein interaction network analysis of validated candidate genes</h3><p id="p-20">High confidence (0.700) known and predicted protein−protein interaction data were retrieved from STRING v9.1 (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Franceschini <em>et al.</em> 2013</a>). The interaction network was visualized using Cytoscape 3.1 software (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Shannon <em>et al.</em> 2003</a>) with a circular layout.</p></div><div id="sec-14" class="subsection"><h3>Gene Onology (GO) analysis of validated candidate genes</h3><p id="p-21">dcCIN genes and icCIN genes were classified using GO Slim Mapper tool from SGD (Saccharomyces Genome Database). GO enrichment analysis was performed with WebGestalt(<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Wang <em>et al.</em> 2013</a>) .</p></div><div id="sec-15" class="subsection"><h3>Statistical analysis</h3><p id="p-22">All statistical analyses were performed with GraphPad Prism software (GraphPad Software, Inc., San Diego, CA).</p></div></div><div class="section results" id="sec-16"><h2 class="">Results</h2><div id="sec-17" class="subsection"><h3>A fluorescence-based, single-cell assay for quantitative CIN measurement</h3><p id="p-23">The basic principle for our qCTF assay is to provide a direct readout of a chromosome loss event that can be detected in single cells. To this end, we engineered a system that leads to irreversible gain of GFP fluorescence signal soon after the loss of a tester chromosome, which can be detected and quantified in large populations of cells by the use of flow cytometry. Using the yeast mating type-determination system (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Haber 2012</a>) we first tagged the <em>MATa</em>-specific <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002869">MFA1</a></em> gene with a 3×GFP fusion at the 3′ end of the ORF by homologous recombination, because <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002869">Mfa1p</a> is the greatest-expressed <em>MATa</em> specific protein (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Ghaemmaghami <em>et al.</em> 2003</a>). The <em>MATα</em> locus was then introduced into the tester chromosome (see below) in a haploid <em>MATa</em> strain. The α2 transcriptional repressor produced from the <em>MATα</em> locus strongly repressed MATa-specific genes, such as Mfa1p-3×GFP. Thus, when the tester chromosome is present, Mfa1p-3×GFP expression is strongly repressed; however, if this chromosome is lost, Mfa1p-3×GFP will be switched on and the cell will become highly fluorescent within one cell cycle after the chromosome loss event due to rapid proteasome degradation of the α2 repressor (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Laney <em>et al.</em> 2006</a>) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1043/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Design and validation of the quantitative chromosome transmission fidelity (qCTF) assay. (A) Illustration of the organization of the mini-chromosome (MC) and how the presence of MC represses the expression of a-specific gene MFA1. The yellow bars flanking MAT&#x3B1; and LEU2 loci represent pBR322 DNA. TEL, telomere; CEN, centromere. Chromosomes are not drawn proportionally to actual size. (B) Flow cytometry analysis of qCTF strain grown in nonselective media for MC. Black line outlines the small population of highly fluorescence cells that had lost MC. (C) Representative quantitative polymerase chain reaction karyotyping of colonies from GFP&#x2212; (MC+) and GFP+ (MC&#x2212;) cells. Roman number indicates each yeast chromosome. The green bar represents the left arm and the blue bar represents right arm. (D) Growth curves of cell with or without MC. Box-and-whisker plots summarize the minimum doubling time of cell with or without MC. The center line of the box indicates the median value, and boxes indicate the interquartile range (IQR). The bottom whisker contains data points that are within 1.5 IQR of the lower quartile and the top whisker contains data points that are within 1.5 IQR of the upper quartile. A P-value of 0.503 was calculated from t-test with n = 6. N.S., nonsignificant. (E) The linearized dependence of the loss rate of a given chromosome on the ratio of doubling times of cells with normal copy number of this chromosome (eup) and that of cells that have lost a copy of the chromosome (ane). An example was plotted with data in (D) using the equations (13) and (14) in File S1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1330484461" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Design and validation of the quantitative chromosome transmission fidelity (qCTF) assay. (A) Illustration of the organization of the mini-chromosome (MC) and how the presence of MC represses the expression of a-specific gene &lt;em&gt;MFA1&lt;/em&gt;. The yellow bars flanking &lt;em&gt;MAT&#x3B1;&lt;/em&gt; and &lt;em&gt;LEU2&lt;/em&gt; loci represent pBR322 DNA. TEL, telomere; CEN, centromere. Chromosomes are not drawn proportionally to actual size. (B) Flow cytometry analysis of qCTF strain grown in nonselective media for MC. Black line outlines the small population of highly fluorescence cells that had lost MC. (C) Representative quantitative polymerase chain reaction karyotyping of colonies from GFP&lt;sup&gt;&#x2212;&lt;/sup&gt; (MC&lt;sup&gt;+&lt;/sup&gt;) and GFP&lt;sup&gt;+&lt;/sup&gt; (MC&lt;sup&gt;&#x2212;&lt;/sup&gt;) cells. Roman number indicates each yeast chromosome. The green bar represents the left arm and the blue bar represents right arm. (D) Growth curves of cell with or without MC. Box-and-whisker plots summarize the minimum doubling time of cell with or without MC. The center line of the box indicates the median value, and boxes indicate the interquartile range (IQR). The bottom whisker contains data points that are within 1.5 IQR of the lower quartile and the top whisker contains data points that are within 1.5 IQR of the upper quartile. A &lt;em&gt;P&lt;/em&gt;-value of 0.503 was calculated from &lt;em&gt;t&lt;/em&gt;-test with n = 6. N.S., nonsignificant. (E) The linearized dependence of the loss rate of a given chromosome on the ratio of doubling times of cells with normal copy number of this chromosome (eup) and that of cells that have lost a copy of the chromosome (ane). An example was plotted with data in (D) using the equations (13) and (14) in File S1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1043/F1.medium.gif" width="327" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/6/1043/F1.medium.gif" width="327" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1043/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1043/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466222" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-24" class="first-child">Design and validation of the quantitative chromosome transmission fidelity (qCTF) assay. (A) Illustration of the organization of the mini-chromosome (MC) and how the presence of MC represses the expression of a-specific gene <em>MFA1</em>. The yellow bars flanking <em>MATα</em> and <em>LEU2</em> loci represent pBR322 DNA. TEL, telomere; CEN, centromere. Chromosomes are not drawn proportionally to actual size. (B) Flow cytometry analysis of qCTF strain grown in nonselective media for MC. Black line outlines the small population of highly fluorescence cells that had lost MC. (C) Representative quantitative polymerase chain reaction karyotyping of colonies from GFP<sup>−</sup> (MC<sup>+</sup>) and GFP<sup>+</sup> (MC<sup>−</sup>) cells. Roman number indicates each yeast chromosome. The green bar represents the left arm and the blue bar represents right arm. (D) Growth curves of cell with or without MC. Box-and-whisker plots summarize the minimum doubling time of cell with or without MC. The center line of the box indicates the median value, and boxes indicate the interquartile range (IQR). The bottom whisker contains data points that are within 1.5 IQR of the lower quartile and the top whisker contains data points that are within 1.5 IQR of the upper quartile. A <em>P</em>-value of 0.503 was calculated from <em>t</em>-test with n = 6. N.S., nonsignificant. (E) The linearized dependence of the loss rate of a given chromosome on the ratio of doubling times of cells with normal copy number of this chromosome (eup) and that of cells that have lost a copy of the chromosome (ane). An example was plotted with data in (D) using the equations (13) and (14) in <a href="http://www.g3journal.org/content/suppl/2015/03/30/g3.115.017913.DC1/FileS1.pdf">File S1</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">The tester chromosome can in principle be any chromosome; however, loss of most native chromosomes in haploid or diploid strains is likely to result in lethality or reduced proliferation rate compared to the non-fluorescent euploid cells in the population, leading to an under-estimation of CIN. To solve this issue, we took advantage of the previously developed MC composed of the left arm of yeast Chromosome III (Chr III) and a very short right arm (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Spencer <em>et al.</em> 1990</a>). The MC carries 70 ORFs (<a href="http://www.g3journal.org/content/suppl/2015/03/30/g3.115.017913.DC1/TableS3.xlsx">Table S3</a>) and the selectable nutrient marker (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">LEU2</a></em>), thus minimizing significant fitness consequence of MC loss if cultured in nonselective media, as confirmed below (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1D</a>). We introduced the aforementioned <em>MATα</em> locus to the right arm of this highly telocentric MC (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>). The proximity of <em>MATα</em> locus to the centromere and its flanking by different DNA sequences from the bacteria plasmid pBR322 make it highly unlikely that the locus is lost through mitotic recombination (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Spencer <em>et al.</em> 1990</a>). In addition, by using the classical CTF assay the MC loss rate was estimated to be 7×10<sup>−4</sup> per cell division (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Warren <em>et al.</em> 2002</a>) and whole MC loss was the predominant mechanism observed. As such, our MC-based qCTF assay is designed to report the frequency of whole chromosome loss and minimizes the complication from other genetic or fitness changes.</p><p id="p-26">To validate various assumptions of the assay, we subjected the aforementioned qCTF strain grown in media nonselective for the MC to flow cytometry analysis. As expected, the vast majority of the population was negative for GFP fluorescence. A very small fraction (0.246 ± 0.008%, n = 8 experiments) exhibited a GFP fluorescence level ∼100-fold greater than the GFP<sup>−</sup> population (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>). For negative and positive controls, this highly fluorescent population was not present from a parental strain where the 3×GFP tag was not inserted and was the dominant population in the parent strain containing the 3×GFP tag but without the added <em>MATα</em> locus (<a href="http://www.g3journal.org/content/suppl/2015/03/30/g3.115.017913.DC1/FigureS1.pdf">Figure S1A</a>). We then sorted the GFP<sup>+</sup> population using fluorescence-activated cell sorting and plated cells at single-colony density on plates with complete media (YPD) followed by replicating on leucine-deficient media. 97% of these colonies were Leu<sup>-</sup>, suggesting the loss of the MC (<a href="http://www.g3journal.org/content/suppl/2015/03/30/g3.115.017913.DC1/FigureS1.pdf">Figure S1B</a>). Furthermore, 11 colonies were randomly picked and karyotyped by using the previously developed qPCR karyotyping assay (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Pavelka <em>et al.</em> 2010</a>). Unlike the MC-containing strain, which had two copies of Chr III left arm and 1 copy of the right arm, all 11 colonies contained only one copy of each of these, confirming MC loss (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1C</a>, <a href="http://www.g3journal.org/content/suppl/2015/03/30/g3.115.017913.DC1/FigureS1.pdf">Figure S1C</a>). These analyses confirmed that GPF<sup>+</sup> cells resulted mostly from MC loss.</p></div><div id="sec-18" class="subsection"><h3>Quantitative performance of qCTF assay</h3><p id="p-27">To obtain quantitative measurement of CIN with qCTF assay in high throughput, we developed a mathematical formula for extrapolation of MC loss rate <em>m</em> from a simple experimental design. <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-1.gif"/></noscript></span></span>, where <em>q</em> is found from <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-2.gif"/></noscript></span></span>. This formula, a derivation of which is presented in the extended Materials and Methods, contains 4 independent parameters: <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-3.gif"/></noscript></span></span> ratio of number (<span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-4.gif"/></noscript></span></span>) of MC<sup>−</sup> cells (GPF<sup>+</sup> population) to number (<span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-5.gif"/></noscript></span></span>) of MC<sup>+</sup> cells (GFP<sup>-</sup> population) at <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-6.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-7.gif"/></noscript></span></span> ratio of number <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-8.gif"/></noscript></span></span> of MC<sup>−</sup> cells to number <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-9.gif"/></noscript></span></span> of MC<sup>+</sup> cells at time <em>t</em> after <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-10.gif"/></noscript></span></span> divisions of MC<sup>+</sup> cells with the growth rate <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-11.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-12.gif"/></noscript></span></span> ratio of the doubling time <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-13.gif"/></noscript></span></span> for MC<sup>−</sup> cells to the doubling time <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-14.gif"/></noscript></span></span> for MC<sup>+</sup> cells. Using the sorted GPF<sup>+</sup> and GFP<sup>−</sup> populations, we compared the growth rates with or without MC present and found that MC loss had minimal impact on growth (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1D</a>). In this case <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-15.gif"/></noscript></span></span> and the formula simplifies to <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-16.gif"/></noscript></span></span> Because in most cases, <em>m</em> &lt;&lt; 1 the formula can be simplified even further to <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1043/embed/mml-math-17.gif"/></noscript></span></span> For future adaptation of the qCTF assay to measurements of native chromosome loss rate, the linear plot can be used once relative growth rates under assay conditions are directly assessed with sorted GFP<sup>+</sup> and GFP<sup>−</sup> populations (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1E</a>).</p><p id="p-28">To test the performance of the qCTF assay, we applied it to a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003054">MAD1</a></em> deletion strain. <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003054">MAD1</a></em> encodes a core component of the spindle assembly checkpoint (SAC), and we compared the obtained CIN rate with those reported previously for WT and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003054">mad1</a>Δ</em> by using ALF (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Li and Murray 1991</a>) and CTF assays (<a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Spencer <em>et al.</em> 1990</a>). For WT cells, the CIN rate measured with qCTF assay was within 2-fold of that estimated with CTF assay that scored for half sectors (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>) but ∼20 fold greater than that estimated from ALF. This was reasonable as the CTF assay also was based on loss of MC, whereas ALF assay was based on loss of the native Chr III. The different results between qCTF (or CTF) and ALF assay may be due to a difference in inherent transmission fidelity between MC and Chr III and/or the fact that MC-containing strain has an additional chromosome (MC) compared with the haploid strain used for ALF. Despite the difference in CIN rate obtained for corresponding strains between qCTF and ALF assays, the CIN rate change for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003054">mad1</a>Δ</em> mutant compared with WT as measured with these assays agreed well (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). In this comparative experiment, one advantage of qCTF immediately became apparent: the qCTF assay examines millions of cells in minutes and is easily performed in replicates, enabling rigorous statistical analysis.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1043/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Quantitative performance of the quantitative chromosome transmission fidelity (qCTF) assay. (A) A table comparing the qCTF assay with the a-like faker (ALF) assay and CTF assay in detecting the rate of elevation of chromosome instability (CIN) caused by loss-of-function/deletion of MAD1. For qCTF assay, the data were shown as Mean &#xB1; SEM, n = 8. (B) Box-and-whisker plots showing the effect of different chemicals on CIN. P-values were calculated from the Mann&#x2013;Whitney U test with n = 8. *P &lt; 0.05; ***P &lt; 0.001. DMSO, dimethyl sulfoxide; MMS, methyl methanesulfonate; CAT, carboxyatractyloside." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1330484461" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Quantitative performance of the quantitative chromosome transmission fidelity (qCTF) assay. (A) A table comparing the qCTF assay with the a-like faker (ALF) assay and CTF assay in detecting the rate of elevation of chromosome instability (CIN) caused by loss-of-function/deletion of &lt;em&gt;MAD1&lt;/em&gt;. For qCTF assay, the data were shown as Mean &#xB1; SEM, n = 8. (B) Box-and-whisker plots showing the effect of different chemicals on CIN. &lt;em&gt;P&lt;/em&gt;-values were calculated from the Mann&#x2013;Whitney &lt;em&gt;U&lt;/em&gt; test with n = 8. *P &lt; 0.05; ***&lt;em&gt;P&lt;/em&gt; &lt; 0.001. DMSO, dimethyl sulfoxide; MMS, methyl methanesulfonate; CAT, carboxyatractyloside.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1043/F2.medium.gif" width="361" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/6/1043/F2.medium.gif" width="361" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1043/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1043/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466198" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-29" class="first-child">Quantitative performance of the quantitative chromosome transmission fidelity (qCTF) assay. (A) A table comparing the qCTF assay with the a-like faker (ALF) assay and CTF assay in detecting the rate of elevation of chromosome instability (CIN) caused by loss-of-function/deletion of <em>MAD1</em>. For qCTF assay, the data were shown as Mean ± SEM, n = 8. (B) Box-and-whisker plots showing the effect of different chemicals on CIN. <em>P</em>-values were calculated from the Mann–Whitney <em>U</em> test with n = 8. *P &lt; 0.05; ***<em>P</em> &lt; 0.001. DMSO, dimethyl sulfoxide; MMS, methyl methanesulfonate; CAT, carboxyatractyloside.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">One potential industrial application of qCTF assay is for identification of aneugens, chemicals that trigger CIN and produce aneuploidy. qCTF could augment the widely-used Ames test for assessing carcinogenic potential (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ames <em>et al.</em> 1973</a>). Radicicol, an Hsp90 inhibitor, was identified in a recent study as a potent aneugen for budding yeast by using the laborious colony-counting CTF assay (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chen <em>et al.</em> 2012</a>). The qCTF assay confirmed radicicol’s dramatic ability to induce CIN (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). Several translation inhibitors that also affect proteome homeostasis had milder but nonetheless significant enhancing effects on CIN (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>). Benomyl, a fungicide that destabilizes microtubules, was not found to elevate CIN at 30 µg/mL concentration with CTF assay (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Chen <em>et al.</em> 2012</a>); however, qCTF detected a 3.5-fold statistically significant increase in CIN (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B</a>). We also tested the emerging idea that DNA replication stress causes CIN by examining the effect of hydroxyurea and methyl methanesulfonate by qCTF (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Burrell <em>et al.</em> 2013</a>). The result revealed a mild but significant increase in CIN (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2B</a>) at concentrations of these compounds that did not cause severe growth retardation. Finally, we found several inhibitors of mitochondrial function to cause a significant elevation of CIN (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2B</a>) at concentrations that did not arrest mitotic proliferation, supporting several previous studies implicating mitochondrial function in the fidelity of chromosome segregation (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dirick <em>et al.</em> 2014</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Veatch <em>et al.</em> 2009</a>).</p></div><div id="sec-19" class="subsection"><h3>Quantitative profiling of genes that affect CIN when reduced in copy number</h3><p id="p-31">We envisioned two unique strengths with qCTF assay over existing assays for measuring CIN: enabling quantitative measurements of CIN with unprecedented sensitivity; and easy adaptability to high-throughput format. Taking advantage of both benefits we performed a genome-wide profiling of dosage-sensitive genes affecting CIN. Recent studies have shown that copy number alteration is a frequently occurring genetic change in cancer cells and that dosage imbalance of several highly conserved SAC proteins elevate CIN (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Schvartzman <em>et al.</em> 2010</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Zack <em>et al.</em> 2013</a>). Identification of genes that affect CIN in a dosage-sensitive manner will facilitate the understanding of the molecular basis of aneuploidy-associated genome instability (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Potapova <em>et al.</em> 2013</a>). However, unlike strong loss of function mutations gene dosage effects are likely to be relatively subtle and requiring reliable and sensitive quantitative assays. Two previous studies (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Choy <em>et al.</em> 2013</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Strome <em>et al.</em> 2008</a>) identified sets of genes that cause increased CIN when hemizygous. We refer to these genes as dcCIN genes. dcCIN genes from these two studies showed minimal overlap (<a href="http://www.g3journal.org/content/suppl/2015/03/30/g3.115.017913.DC1/FigureS2.pdf">Figure S2A</a>) and most were not found to cause CIN when tested in haploid yeast with deletion or conditional alleles of these genes (<a href="http://www.g3journal.org/content/suppl/2015/03/30/g3.115.017913.DC1/FigureS2.pdf">Figure S2A</a>). These discrepancies could be due to assay differences or the noise and semiquantitative nature of the previous assays, especially given that heterozygous deletion mostly confers a mild CIN phenotype. We therefore decided to screen for additional dcCIN genes by using qCTF assay in a high-throughput format.</p><p id="p-32">The <em>MATa</em> locus of the WT qCTF yeast strain was deleted so that it would behave as an <em>MATα</em> haploid strain and can be mated to the nonessential <em>MATa</em> ORF deletion strain library, generating a library of diploid strains carrying heterozygous deletion of 4919 ORFs, of 4977 nonessential ORFs and containing the qCTF system in the genetic background (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). We first used qCTF to validate 254 previously identified nonessential dcCIN genes. The assay was performed in 96-well sample format, where ∼0.3 million cells per replicate and eight replicates were analyzed for each strain at 0- and 24-hr time points after switching the cultures to nonselective media (see the section <em>Materials and Methods</em>). OD at each time point also was measured for each strain to estimate cell doubling number. We found 44% (111/254) of previously identified dcCIN genes were verified by qCTF to produce significant increase in CIN (fold change &gt;1.5, <em>P</em> &lt; 0.01) compared with a WT diploid control, and 87% of these caused elevation of CIN by less than twofold (<a href="http://www.g3journal.org/content/suppl/2015/03/30/g3.115.017913.DC1/TableS4.xlsx">Table S4</a>). These results also validated the ability of qCTF to quantify small changes in CIN. We therefore went on to screen the remaining heterozygous deletion strains by qCTF. Because the primary screen of 4914 genes by flow cytometry was performed without replicates, strains with the top 105 highest and the top 101 lowest CIN rates were picked as candidate hits. These candidate hits were then rescreened with eight biological replicates, and 80 of them were validated to cause significant change in CIN (fold change &gt;1.5, <em>P</em> &lt; 0.01) compared with the WT control (<a href="http://www.g3journal.org/content/suppl/2015/03/30/g3.115.017913.DC1/TableS4.xlsx">Table S4</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1043/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide screen of nonessential yeast open-reading frame deletions for decreased in copy number (dcCIN) genes with the quantitative chromosome transmission fidelity (qCTF) assay. (A) A schematic representation of the strain generation and screen procedure for identifying dcCIN genes. In brief, qCTF strains hemizygous for each ORF deletion were obtained by mating as described in detail in the section Materials and Methods. OD and mini-chromosome negative (MC&#x2212;) cell frequency (flow cytometry) were monitored at the beginning and end points for each culture grown in no-selective media for MC. Analysis details are given in the section Materials and Methods. (B) Gene Ontology Slim functional classification of dcCIN genes, showing only major groups with at least 10 genes with some highly similar groups being omitted. (C) A bar plot shows the rate of chromosome instability (CIN) of the top 25 dcCIN genes (orange bars). Data are shown as Mean &#xB1; SEM (n = 8). (D) Protein interaction network among dcCIN genes. Genes that cause different fold changes in CIN rate are differentially color coded as indicated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1330484461" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genome-wide screen of nonessential yeast open-reading frame deletions for decreased in copy number (dcCIN) genes with the quantitative chromosome transmission fidelity (qCTF) assay. (A) A schematic representation of the strain generation and screen procedure for identifying dcCIN genes. In brief, qCTF strains hemizygous for each ORF deletion were obtained by mating as described in detail in the section &lt;em&gt;Materials and Methods&lt;/em&gt;. OD and mini-chromosome negative (MC&lt;sup&gt;&#x2212;&lt;/sup&gt;) cell frequency (flow cytometry) were monitored at the beginning and end points for each culture grown in no-selective media for MC. Analysis details are given in the section &lt;em&gt;Materials and Methods&lt;/em&gt;. (B) Gene Ontology Slim functional classification of dcCIN genes, showing only major groups with at least 10 genes with some highly similar groups being omitted. (C) A bar plot shows the rate of chromosome instability (CIN) of the top 25 dcCIN genes (orange bars). Data are shown as Mean &#xB1; SEM (n = 8). (D) Protein interaction network among dcCIN genes. Genes that cause different fold changes in CIN rate are differentially color coded as indicated.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1043/F3.medium.gif" width="440" height="369"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/6/1043/F3.medium.gif" width="440" height="369"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1043/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1043/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466265" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-33" class="first-child">Genome-wide screen of nonessential yeast open-reading frame deletions for decreased in copy number (dcCIN) genes with the quantitative chromosome transmission fidelity (qCTF) assay. (A) A schematic representation of the strain generation and screen procedure for identifying dcCIN genes. In brief, qCTF strains hemizygous for each ORF deletion were obtained by mating as described in detail in the section <em>Materials and Methods</em>. OD and mini-chromosome negative (MC<sup>−</sup>) cell frequency (flow cytometry) were monitored at the beginning and end points for each culture grown in no-selective media for MC. Analysis details are given in the section <em>Materials and Methods</em>. (B) Gene Ontology Slim functional classification of dcCIN genes, showing only major groups with at least 10 genes with some highly similar groups being omitted. (C) A bar plot shows the rate of chromosome instability (CIN) of the top 25 dcCIN genes (orange bars). Data are shown as Mean ± SEM (n = 8). (D) Protein interaction network among dcCIN genes. Genes that cause different fold changes in CIN rate are differentially color coded as indicated.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">The aforementioned analyses using qCTF produced a list of 186 high-confidence dcCIN genes. GO Slim Classification analysis revealed a subset of dcCIN genes belong to diverse processes such as DNA damage response, transcription, and cytoplasmic translation (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). In addition, GO analysis revealed enrichment for genes regulating cell cycle, M phase, and the spindle pole body (<a href="http://www.g3journal.org/content/suppl/2015/03/30/g3.115.017913.DC1/FigureS3.pdf">Figure S3, A and B</a>), as expected. Cell-cycle regulators among the top 25 hits included <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004069">SIC1</a></em> (a G1/S cyclin-dependent kinase inhibitor), <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000038">CLN3</a></em> (a G1 cyclin), and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003054">MAD1</a></em> (a component of the SAC), <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002214">MBP1</a></em> (a transcription factor involved in G1/S regulation) (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>). Protein interaction network analysis (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Franceschini <em>et al.</em> 2013</a>) revealed that 95 of the dcCIN genes have interaction partners within the group, most notably the presence of a clear ribosome protein cluster (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3D</a>). This finding may be consistent with the effect of protein synthesis inhibitor on CIN (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2B</a>), suggesting a requirement for a balanced proteome in maintaining high chromosome segregation fidelity. Curiously, 162 dcCIN genes (<a href="http://www.g3journal.org/content/suppl/2015/03/30/g3.115.017913.DC1/FigureS2.pdf">Figure S2C</a>) were not identified as CIN genes in haploid deletion screens (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Stirling <em>et al.</em> 2011</a>). For example, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004571">NUP188</a></em> encodes a subunit of the inner ring of the nuclear pore complex and was not previously known to be involved in chromosome segregation; however, its human homolog was recently found to be required for chromosome alignment in mitosis in Hela cells (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Itoh <em>et al.</em> 2013</a>). Thus it may be useful to rescreen of the haploid ORF deletion library by qCTF assay. Interestingly, the nonessential dcCIN genes identified here were not found to be significantly enriched for core protein complexes (<a href="http://www.g3journal.org/content/suppl/2015/03/30/g3.115.017913.DC1/FigureS3.pdf">Figure S3E</a>). This differs from the genes that are haploinsufficient for growth (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Deutschbauer <em>et al.</em> 2005</a>).</p></div><div id="sec-20" class="subsection"><h3>Identification of genes that affect CIN when increased in copy number</h3><p id="p-35">Gene or segmental chromosome amplification often lead to increased expression of the affected genes and is a frequently observed genetic change in evolving genomes (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Santarius <em>et al.</em> 2010</a>). It has been shown that increased expression of certain mitotic regulators disrupt chromosome transmission fidelity (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Ricke <em>et al.</em> 2011</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Ryan <em>et al.</em> 2012</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Sotillo <em>et al.</em> 2007</a>), but a comprehensive analysis of genes that elevates CIN at increased copy number has not be reported. To this end, we used qCTF assay in combination with the MoBY-ORF library containing 4956 yeast ORFs with their natural promoter and terminator (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Ho <em>et al.</em> 2009</a>) to perform a genome-wide profiling of genes for which a moderate increase in copy number affects CIN (referred as icCIN genes). The presence of a centromeric sequence on MoBY-ORF plasmids ensures that the copy number of the gene associated with each ORF is only increased by 1- to 3-fold. The individual centromeric MoBY-ORF plasmids were transformed in a high throughput manner into the qCTF yeast strain with an efficiency of 88%, resulting in 4392 strains (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). The transformants were then subjected to high-throughput qCTF assay by flow cytometry as with the dcCIN screen, with the qCTF strain carrying an empty MoBY plasmid as the control. It is interesting to note that the CIN rate in this control strain is 1.4 fold higher (<em>P</em> &lt; 0.0001) than the parental strain without the centromeric MoBY vector (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>), and additional centromeric plasmids leads to further increase in CIN (<a href="http://www.g3journal.org/content/suppl/2015/03/30/g3.115.017913.DC1/FigureS4.pdf">Figure S4</a>). This finding suggests that extra centromeres in an otherwise-euploid genome elevate CIN.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1043/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide screen for increased in copy number (icCIN) genes with the quantitative chromosome transmission fidelity (qCTF) assay. (A) A schematic representation of the strain generation and screen procedure for a genome-wide screen for icCIN genes using the yeast MoBY open-reading frame (ORF) library. Experimental details are given in the section Materials and Methods. (B) Box-and-whisker plots comparing the CIN rate of the wild-type qCTF strain without (W/O) or with the MoBY empty control plasmid and qCTF strains with the MoBY ORF plasmids, n = 8 for WT qCTF strain; n = 58 for qCTF strain with control plasmid; n = 4932 for all MoBY ORFs in the primary screen. P value was calculated from the Mann-Whitney U test. ****P &lt; 0.0001; N.S., nonsignificant. (C). Bar plots showing the rate of chromosome instability (CIN) of the 25 icCIN genes (orange bars) and two genes (blue bars) that suppressed basal CIN when increased in copy number. Data are shown as mean &#xB1; SEM (n= 8). (D) Protein interaction network among icCIN genes. Genes that cause different fold changes in CIN rate are differentially color coded as indicated. (E) Gene Ontology Slim functional classification of icCIN genes. Only groups with at least two genes are shown with some highly similar groups being omitted. (F) Bar plots comparing the frequency of essential gene and genes involved in core protein complex from icCIN genes or all MoBY plasmids screened (Background). P value was calculated from Fisher&#x2019;s exact test. **P &lt; 0.01; N.S., nonsignificant." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1330484461" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genome-wide screen for increased in copy number (icCIN) genes with the quantitative chromosome transmission fidelity (qCTF) assay. (A) A schematic representation of the strain generation and screen procedure for a genome-wide screen for icCIN genes using the yeast MoBY open-reading frame (ORF) library. Experimental details are given in the section &lt;em&gt;Materials and Methods&lt;/em&gt;. (B) Box-and-whisker plots comparing the CIN rate of the wild-type qCTF strain without (W/O) or with the MoBY empty control plasmid and qCTF strains with the MoBY ORF plasmids, n = 8 for WT qCTF strain; n = 58 for qCTF strain with control plasmid; n = 4932 for all MoBY ORFs in the primary screen. &lt;em&gt;P&lt;/em&gt; value was calculated from the Mann-Whitney &lt;em&gt;U&lt;/em&gt; test. ****&lt;em&gt;P&lt;/em&gt; &lt; 0.0001; N.S., nonsignificant. (C). Bar plots showing the rate of chromosome instability (CIN) of the 25 icCIN genes (orange bars) and two genes (blue bars) that suppressed basal CIN when increased in copy number. Data are shown as mean &#xB1; SEM (n= 8). (D) Protein interaction network among icCIN genes. Genes that cause different fold changes in CIN rate are differentially color coded as indicated. (E) Gene Ontology Slim functional classification of icCIN genes. Only groups with at least two genes are shown with some highly similar groups being omitted. (F) Bar plots comparing the frequency of essential gene and genes involved in core protein complex from icCIN genes or all MoBY plasmids screened (Background). &lt;em&gt;P&lt;/em&gt; value was calculated from Fisher&#x2019;s exact test. **&lt;em&gt;P&lt;/em&gt; &lt; 0.01; N.S., nonsignificant.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1043/F4.medium.gif" width="307" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/6/1043/F4.medium.gif" width="307" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1043/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1043/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466330" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-36" class="first-child">Genome-wide screen for increased in copy number (icCIN) genes with the quantitative chromosome transmission fidelity (qCTF) assay. (A) A schematic representation of the strain generation and screen procedure for a genome-wide screen for icCIN genes using the yeast MoBY open-reading frame (ORF) library. Experimental details are given in the section <em>Materials and Methods</em>. (B) Box-and-whisker plots comparing the CIN rate of the wild-type qCTF strain without (W/O) or with the MoBY empty control plasmid and qCTF strains with the MoBY ORF plasmids, n = 8 for WT qCTF strain; n = 58 for qCTF strain with control plasmid; n = 4932 for all MoBY ORFs in the primary screen. <em>P</em> value was calculated from the Mann-Whitney <em>U</em> test. ****<em>P</em> &lt; 0.0001; N.S., nonsignificant. (C). Bar plots showing the rate of chromosome instability (CIN) of the 25 icCIN genes (orange bars) and two genes (blue bars) that suppressed basal CIN when increased in copy number. Data are shown as mean ± SEM (n= 8). (D) Protein interaction network among icCIN genes. Genes that cause different fold changes in CIN rate are differentially color coded as indicated. (E) Gene Ontology Slim functional classification of icCIN genes. Only groups with at least two genes are shown with some highly similar groups being omitted. (F) Bar plots comparing the frequency of essential gene and genes involved in core protein complex from icCIN genes or all MoBY plasmids screened (Background). <em>P</em> value was calculated from Fisher’s exact test. **<em>P</em> &lt; 0.01; N.S., nonsignificant.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">The CIN rate distribution of all screened genes showed no significant difference when compared with the CIN rate distribution of the strain carrying the empty control plasmid, but outliers could be observed (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>). A total of 207 genes whose CIN rate ranked top 101 and bottom 106 were picked as candidate hits. To ensure the correct identity of the selected ORFs, the MoBY-ORF plasmids were recovered from these strains, amplified, and sequenced, revealing mislabeled or duplicated ORFs. To control for the mutagenic effect of plasmid transformation, 200 plasmids carrying confirmed genes were then retransformed into the qCTF strain and subjected to secondary validation using eight biological replicates, resulting in 25 high-confidence (fold change &gt;1.5, <em>P</em> &lt; 0.01) icCIN genes (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4C</a>). Protein interaction network analysis (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Franceschini <em>et al.</em> 2013</a>) revealed that more than 50% of icCIN genes interact with each other (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4D</a>). The major interaction network is composed mostly of genes involved in cell cycle regulation (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4E</a>), such as <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006323">CLB2</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003341">CLB6</a></em> (encoding two B-type cyclins), <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004639">MIH1</a></em> (encoding the budding yeast homolog of the Wee1 tyrosine kinase regulating <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000364">Cdk1</a> activation), and SAC genes <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003054">MAD1</a></em> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003550"><em>MAD</em>3</a>. A smaller gene cluster included <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005285">ATO2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005284">CIT1</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004351">ADE13</a></em>, all encoding metabolic proteins. Enrichment for cell cycle genes was also indicated by GO analysis (<a href="http://www.g3journal.org/content/suppl/2015/03/30/g3.115.017913.DC1/FigureS3.pdf">Figure S3C</a>). Proteins encoded by icCIN genes were enriched for nuclear and spindle components (<a href="http://www.g3journal.org/content/suppl/2015/03/30/g3.115.017913.DC1/FigureS3.pdf">Figure S3D</a>). icCIN genes were also enriched for essential genes and showed higher though statistically insignificant increase in the fraction of core complex proteins (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4F</a>).</p><p id="p-38"><em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003054">MAD1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006323">CLB2</a></em> represent overlapping hits from the dcCIN and icCIN screens (<a href="http://www.g3journal.org/content/suppl/2015/03/30/g3.115.017913.DC1/FigureS2.pdf">Figure S2D</a>), suggesting the existence of two-way dosage sensitive CIN genes. To identify more of these genes, dcCIN analysis was performed for the 25 icCIN genes, which include 10 essential genes not previously included in the dcCIN screen but now deleted manually in a WT diploid qCTF strain. Altogether, 9 of 25 icCIN genes were found to be also dcCIN genes (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). Interestingly, 6 of these genes encode components of core protein complexes (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). For example, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000935">GLC7</a></em>, encoding the catalytic subunit of protein phosphatase 1, is a stoichiometric subunit of the cleavage and polyadenylation factor complex, and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005430">Nuf2</a>, is a component of the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001406">Ndc80</a> kinetochore complex. The fact that both increase and reduction in copy numbers of these genes enhanced CIN may support the idea that their dosage effect is due to an imbalanced protein stoichiometry rather than insufficiency or over-abundance in the activity of the encoded proteins (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Papp <em>et al.</em> 2003</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1043/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Characterization of chromosome instability (CIN) gene dosage sensitivity. (A) A bar plot shows CIN rate of nine strains each having heterozygous deletion of one icCIN gene (dark gray bar). These are thus two-way dosage-sensitive genes. Data are shown as mean &#xB1; SEM (n = 8). (B) A table displaying core complex involvement of six two-way dosage-sensitive CIN genes. (C) Bar plots showing the CIN rate of haploid strains with different copy number of genetically integrated NPL3. WT control has one native copy of NPL3. Data are shown as mean &#xB1; SEM, n = 8. P value was calculated from t-test. ****P &lt; 0.0001. (D) Bar plots comparing the CIN rate of strains containing a blank centromeric plasmid, NPL3 wild-type centromeric plasmid, and NPL3 point mutant centromeric plasmid. Top panel indicates the position of the point mutation in the second RNA Recognition Motif (RRM2) of Npl3p. N (N terminus); C (C terminus). Mean &#xB1; SEM, n = 8. P value was calculated from t-test. **P &lt; 0.01. (E) Bar plots showing the CIN rate of haploid strains with different copy number of genetically integrated MCD1. Data are shown as mean &#xB1; SEM, n = 8. P value was calculated from t-test. ****P &lt; 0.0001. (F) Bar plots comparing CIN rate of strains containing two blank centromeric plasmids (Empty Plasmids Control), MCD1 centromeric plasmid plus one blank centromeric plasmid (MCD1), ESP1 centromeric plasmid plus one blank centromeric plasmid (ESP1), and MCD1 centromeric plasmid plus ESP1 centromeric plasmid (MCD1 + ESP1). Mean &#xB1; SEM, n = 8. P value was calculated from t-test. ***P &lt; 0.001. (G) Bar plots comparing CIN rate of strains containing two blank centromeric plasmids (Empty Plasmids Control), MAD1 centromeric plasmid plus one blank centromeric plasmid (MAD1), MAD2 centromeric plasmid plus one blank centromeric plasmid (MAD2), and MAD1 centromeric plasmid plus MAD2 centromeric plasmid (MAD1 + MAD2). Mean &#xB1; SEM, n = 8. P value was calculated from t-test. ****P &lt; 0.0001." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1330484461" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Characterization of chromosome instability (CIN) gene dosage sensitivity. (A) A bar plot shows CIN rate of nine strains each having heterozygous deletion of one icCIN gene (dark gray bar). These are thus two-way dosage-sensitive genes. Data are shown as mean &#xB1; SEM (n = 8). (B) A table displaying core complex involvement of six two-way dosage-sensitive CIN genes. (C) Bar plots showing the CIN rate of haploid strains with different copy number of genetically integrated &lt;em&gt;NPL3&lt;/em&gt;. WT control has one native copy of &lt;em&gt;NPL3&lt;/em&gt;. Data are shown as mean &#xB1; SEM, n = 8. &lt;em&gt;P&lt;/em&gt; value was calculated from &lt;em&gt;t&lt;/em&gt;-test. ****&lt;em&gt;P&lt;/em&gt; &lt; 0.0001. (D) Bar plots comparing the CIN rate of strains containing a blank centromeric plasmid, &lt;em&gt;NPL3&lt;/em&gt; wild-type centromeric plasmid, and &lt;em&gt;NPL3&lt;/em&gt; point mutant centromeric plasmid. Top panel indicates the position of the point mutation in the second RNA Recognition Motif (RRM2) of Npl3p. N (N terminus); C (C terminus). Mean &#xB1; SEM, n = 8. &lt;em&gt;P&lt;/em&gt; value was calculated from &lt;em&gt;t&lt;/em&gt;-test. **&lt;em&gt;P&lt;/em&gt; &lt; 0.01. (E) Bar plots showing the CIN rate of haploid strains with different copy number of genetically integrated &lt;em&gt;MCD1&lt;/em&gt;. Data are shown as mean &#xB1; SEM, n = 8. &lt;em&gt;P&lt;/em&gt; value was calculated from &lt;em&gt;t&lt;/em&gt;-test. ****&lt;em&gt;P&lt;/em&gt; &lt; 0.0001. (F) Bar plots comparing CIN rate of strains containing two blank centromeric plasmids (Empty Plasmids Control), &lt;em&gt;MCD1&lt;/em&gt; centromeric plasmid plus one blank centromeric plasmid (MCD1), &lt;em&gt;ESP1&lt;/em&gt; centromeric plasmid plus one blank centromeric plasmid (ESP1), and &lt;em&gt;MCD1&lt;/em&gt; centromeric plasmid plus &lt;em&gt;ESP1&lt;/em&gt; centromeric plasmid (MCD1 + ESP1). Mean &#xB1; SEM, n = 8. &lt;em&gt;P&lt;/em&gt; value was calculated from &lt;em&gt;t&lt;/em&gt;-test. ***&lt;em&gt;P&lt;/em&gt; &lt; 0.001. (G) Bar plots comparing CIN rate of strains containing two blank centromeric plasmids (Empty Plasmids Control), &lt;em&gt;MAD1&lt;/em&gt; centromeric plasmid plus one blank centromeric plasmid (MAD1), &lt;em&gt;MAD2&lt;/em&gt; centromeric plasmid plus one blank centromeric plasmid (MAD2), and &lt;em&gt;MAD1&lt;/em&gt; centromeric plasmid plus &lt;em&gt;MAD2&lt;/em&gt; centromeric plasmid (MAD1 + MAD2). Mean &#xB1; SEM, n = 8. &lt;em&gt;P&lt;/em&gt; value was calculated from &lt;em&gt;t&lt;/em&gt;-test. ****&lt;em&gt;P&lt;/em&gt; &lt; 0.0001.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1043/F5.medium.gif" width="278" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/6/1043/F5.medium.gif" width="278" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1043/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1043/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466231" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-39" class="first-child">Characterization of chromosome instability (CIN) gene dosage sensitivity. (A) A bar plot shows CIN rate of nine strains each having heterozygous deletion of one icCIN gene (dark gray bar). These are thus two-way dosage-sensitive genes. Data are shown as mean ± SEM (n = 8). (B) A table displaying core complex involvement of six two-way dosage-sensitive CIN genes. (C) Bar plots showing the CIN rate of haploid strains with different copy number of genetically integrated <em>NPL3</em>. WT control has one native copy of <em>NPL3</em>. Data are shown as mean ± SEM, n = 8. <em>P</em> value was calculated from <em>t</em>-test. ****<em>P</em> &lt; 0.0001. (D) Bar plots comparing the CIN rate of strains containing a blank centromeric plasmid, <em>NPL3</em> wild-type centromeric plasmid, and <em>NPL3</em> point mutant centromeric plasmid. Top panel indicates the position of the point mutation in the second RNA Recognition Motif (RRM2) of Npl3p. N (N terminus); C (C terminus). Mean ± SEM, n = 8. <em>P</em> value was calculated from <em>t</em>-test. **<em>P</em> &lt; 0.01. (E) Bar plots showing the CIN rate of haploid strains with different copy number of genetically integrated <em>MCD1</em>. Data are shown as mean ± SEM, n = 8. <em>P</em> value was calculated from <em>t</em>-test. ****<em>P</em> &lt; 0.0001. (F) Bar plots comparing CIN rate of strains containing two blank centromeric plasmids (Empty Plasmids Control), <em>MCD1</em> centromeric plasmid plus one blank centromeric plasmid (MCD1), <em>ESP1</em> centromeric plasmid plus one blank centromeric plasmid (ESP1), and <em>MCD1</em> centromeric plasmid plus <em>ESP1</em> centromeric plasmid (MCD1 + ESP1). Mean ± SEM, n = 8. <em>P</em> value was calculated from <em>t</em>-test. ***<em>P</em> &lt; 0.001. (G) Bar plots comparing CIN rate of strains containing two blank centromeric plasmids (Empty Plasmids Control), <em>MAD1</em> centromeric plasmid plus one blank centromeric plasmid (MAD1), <em>MAD2</em> centromeric plasmid plus one blank centromeric plasmid (MAD2), and <em>MAD1</em> centromeric plasmid plus <em>MAD2</em> centromeric plasmid (MAD1 + MAD2). Mean ± SEM, n = 8. <em>P</em> value was calculated from <em>t</em>-test. ****<em>P</em> &lt; 0.0001.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-21" class="subsection"><h3>Two genes suppressing basal CIN when increased in copy number</h3><p id="p-40">A surprising finding of the icCIN screen, likely detected only as a result of the highly quantitative and sensitive nature of qCTF assay, was that two genes, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002161">MCD1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002840">NPL3</a></em>, each significantly reduced basal CIN rate (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4C</a> and <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>) when present at increased copy numbers. <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002840">NPL3</a></em> encodes the yeast hnRNP that regulates mRNA transcription, processing and transport out of the nucleus (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Lee <em>et al.</em> 1996</a>). To more accurately evaluate the dosage effect of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002840">NPL3</a></em> on CIN, we introduced different copies of this gene into the genome through homologous integration and analyzed the resulting strains with qCTF. This analysis showed a continuous decrease in CIN when <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002840">NPL3</a></em> copy number increased from 1 to 3 copies in the genome, but further increases began to reverse the CIN suppression phenotype, demonstrating quantitative dosage effect of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002840">NPL3</a></em> on CIN (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5C</a>). To test whether CIN suppression requires the RNA-binding activity of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002840">Npl3p</a>, we introduced a previously characterized point mutation (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Lee <em>et al.</em> 1996</a>) in the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002840">Npl3p</a> RNA recognition motif into a centromeric plasmid and used qCTF to test its effect on CIN suppression. Interestingly, the <em>npl3-48</em> plasmid harboring point mutation S230P increased CIN in contrast to CIN suppression by WT <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002840">NPL3</a></em> plasmid (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5D</a>).</p><p id="p-41">We also used homologous integration to introduce various copy numbers of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002161">MCD1</a></em>, encoding the cohesion subunit cleaved by separase, into the yeast genome. In contrast to the results for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002840">NPL3</a></em>, the decrease in CIN was roughly constant despite adding up to six copies of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002161">MCD1</a></em> (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5E</a>), suggesting that <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002161"><em>MCD1</em></a> dosage was no longer limiting after one additional copy. We considered the possibility that increased <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002161">MCD1</a></em> copy number could counter against some basal activity of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003330">Esp1</a> that disrupts proper chromosome cohesion. The addition of a centromeric plasmid containing <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003330">ESP1</a></em> caused a fourfold increase in CIN and overrode the effect of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002161">MCD1</a></em> gain (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5F</a>). Finally based on a previous study that <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003054">MAD1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003567">MAD2</a></em> gene dosage needs to be balanced for optimal checkpoint function (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barnhart <em>et al.</em> 2011</a>), we introduced a MoBY plasmid carrying <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003567">MAD2</a></em> into the strain with the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003054">MAD1</a></em> plasmid. Interestingly, whereas increased dosage of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003567">MAD2</a></em> by itself moderately reduced CIN level, the dual presence of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003054">MAD1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003567">MAD2</a></em> plasmid suppressed CIN to significantly below that of the basal level (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5G</a>). We note that none of the aforementioned gene copy number manipulations drastically altered the growth rate (<a href="http://www.g3journal.org/content/suppl/2015/03/30/g3.115.017913.DC1/FigureS5.pdf">Figure S5</a>), and any change in cell cycle time was accounted for in our protocol of CIN rate measurement and calculation.</p></div></div><div class="section discussion" id="sec-22"><h2 class="">Discussion</h2><p id="p-42">In this article, we have presented the development, validation and high-throughput application of a single cell−based and truly quantitative assay for assessment of chromosome transmission fidelity in budding yeast. A key design feature of qCTF is that a strong fluorescence signal from Mfa1p-3×GFP expression is gained in single cells upon removal of a transcriptional repressor carried on a particular chromosome that is lost during mitosis. Such events, although rare, can be captured and quantified in large cell populations by using flow cytometry. Combined with 96-well or possibly even 384-well sample plates and robotic plate loader, qCTF can be adapted for quantitative high-throughput analysis. We demonstrate the usefulness of qCTF by applying it to two genome-wide screens in yeast to identify genes that affect CIN in a dosage-sensitive manner. Our results show that qCTF is sensitive enough to detect the impact a small change in gene copy number can have on CIN, including not only the increase in CIN but also a decrease from the basal level. The assay is also sufficiently sensitive to detect increases in CIN caused by the presence of a centromeric plasmid, supporting the model of a recent study suggesting that CIN can be influenced by over-loading of the mitotic system with extra chromosome or centromeres to be segregated (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Zhu <em>et al.</em> 2012</a>).</p><p id="p-43">The development of the qCTF assay provides a convenient and sensitive way to monitor CIN in budding yeast. However, it has several limitations. First, the current design does not rule out the possibility of mutations other than MC loss that contribute to the appearance of GFP<sup>+</sup> cells. For example, point mutations in the α2 locus could contribute to the GFP<sup>+</sup> population with MC being present. Thus, it is important to confirm MC loss in GFP<sup>+</sup> cells of hits of interest in future studies. We therefore tested some our top hits and hits involved in DNA damage response (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000063">NUP60</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000476">HSM3</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000478">CHK1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">DUN1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000065">RFA1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004681">PDS5</a></em>), as the latter are more likely to be mutagenic, or chromatin organization (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000104">HIR1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000629">SNT1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002633">HTA1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002982">PUF4</a></em>) for the presence of MC after sorting for the GFP<sup>+</sup> cells in selective media. We found that less than 2% of GFP<sup>+</sup> cells from six icCIN hits still had a MC, except <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002872">SPP41</a></em> (15.5%) (<a href="http://www.g3journal.org/content/suppl/2015/03/30/g3.115.017913.DC1/FigureS6.pdf">Figure S6A</a>). All but one (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000104">HIR1</a></em>/<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000104">hir1</a>Δ</em>) of the dcCIN hits tested had a level of GFP<sup>+</sup> cells containing the MC at frequencies similar to or lower than that of the diploid control (<a href="http://www.g3journal.org/content/suppl/2015/03/30/g3.115.017913.DC1/FigureS6.pdf">Figure S6B</a>). These results suggest that although the qCTF assay could be complicated by other genetic changes, MC loss is the predominant genetic change that leads to GFP expression in most strains examined. Second, the competition between MC<sup>+</sup> and MC<sup>−</sup> cells could bias the result of qCTF assay. In our high-throughput screen, we assumed similar growth rate between MC<sup>+</sup> and MC<sup>−</sup> cells to simplify the experimental design. To further validate this assumption, we compared the growth rates between MC<sup>+</sup> and MC<sup>−</sup> cells from several of our top hits. We found that for the five icCIN hits, MC<sup>+</sup> cells grew slightly (5–10%) faster than MC<sup>-</sup> cells, suggesting a slight underestimation of CIN rate (<a href="http://www.g3journal.org/content/suppl/2015/03/30/g3.115.017913.DC1/FigureS6.pdf">Figure S6C</a>). For the three tested dcCIN hits, the differences in growth rates of these pairs were insignificant (<a href="http://www.g3journal.org/content/suppl/2015/03/30/g3.115.017913.DC1/FigureS6.pdf">Figure S6D</a>). Third, the qCTF assay only reports the loss of the short and telocentric MC, but not a natural yeast chromosome. The stability of natural chromosomes could be affected by <em>cis</em>-acting elements such as centromeric sequence (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Kumaran <em>et al.</em> 2013</a>). Indeed, in a previous dcCIN gene screen using a different yeast MC, it was found that 85% of the hits resulted in greater than twofold increase in chromosome V loss rate (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Strome <em>et al.</em> 2008</a>).</p><p id="p-44">Despite the aforementioned caveats, qCTF can still be a quantitative tool for probing molecular pathways controlling mitosis and chromosome segregation. It also can be useful in industrial settings for quantitative evaluating the aneugneic potential of environmental chemicals or pharmaceutical products. Likewise, the influence of natural products with anticancer therapeutic potential on CIN also can be evaluated using qCTF. Although our current qCTF assay is limited to tracking the segregation of an MC in yeast, we envision further expansion of the concept in two directions. First, the assay can be adapted to measurement of loss frequencies for any native chromosomes in a diploid or polyploidy yeast genome by inserting the repressor gene at a centromere-proximal location of the chromosome of interest, and then obtaining the growth rates of the yeast population before and after the loss of a copy of this chromosome (the latter can be sorted with fluorescence-activated cell sorting) to allow chromosome loss rate calculation. Such an adaptation of the assay may provide new insights into the relation between chromosome properties and transmission fidelity. Second, the assay may be adaptable to metazoan cells and organisms using appropriate repressor-reporter systems following similar design principles of the yeast qCTF assay. Such an assay has the potential to be superior in sensitivity, quantitation, and applicability to high throughput analysis compared with existing methods such as fluorescence in-situ hybridization or spectral karyotypin−based detection of chromosome numerical abnormality.</p><p id="p-45">It is reassuring that the hits obtained from both screens performed with qCTF are enriched for cellular processes known to directly affect CIN, such as cell-cycle regulation and mitosis (<a href="http://www.g3journal.org/content/suppl/2015/03/30/g3.115.017913.DC1/FigureS3.pdf">Figure S3, A−D</a>). However, our dcCIN hits overlap only slightly with previously CIN screens that used ORF deletion or temperature-sensitive mutant strains (<a href="http://www.g3journal.org/content/suppl/2015/03/30/g3.115.017913.DC1/FigureS2.pdf">Figure S2C</a>). This may in part be explained by the fact that the previous screens were not aimed at finding dosage-sensitive genes but rather genes whose complete absence or inactivation would elevate CIN. Furthermore, inactivation of an essential gene could lead to cell-cycle arrest or rapid cell death, which would prevent growth or colony-based detection of chromosome loss events. It would thus be worthwhile in the future to use qCTF assay to re-profile yeast gene deletion or conditional loss-of-function libraries. We also noted that qCTF validated fewer than half of the hits from previous screens for genes that elevate CIN upon dosage reduction, whereas 40% (74/186) of the dcCIN hits, including many known to function in mitosis and cell cycle regulation, were missed in previous screens. This is likely to be explained by subtle effects of small gene dosage variation compounded by the difficulty in quantification associated with previous assays.</p><p id="p-46">Our icCIN screen is the first of its kind to search for genes affecting CIN when increased in copy number and is distinct from previous work that uses the Gal-promoter (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Ouspenski <em>et al.</em> 1999</a>), which causes gross and deregulated overexpressions of ORFs. The centromere present on the MoBY plasmid helps to maintain the transgene at a low copy number and the presence of the native promoter in front of each ORF maintain the normal transcriptional regulation of the respective gene, which is often important for cell cycle regulators. The top hit from this screen is <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003054">MAD1</a></em>, encoding a highly conserved SAC kinase that forms stoichiometric complex with another conserved SAC protein <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003567">Mad2</a>. It was shown previously that a balanced <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003054">Mad1</a>/<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003567">Mad2</a> dosage is critical for check point function and proper completion of mitosis. This requirement may also account for elevated CIN observed in aneuploid strains with higher Chr VII to Chr X ratios (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Barnhart <em>et al.</em> 2011</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Zhu <em>et al.</em> 2012</a>). It is intriguing that the next three greatest-impact icCIN genes, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002872">SPP41</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000935">GLC7</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003506">TAF1</a></em>, all encode proteins with known roles in transcription and RNA processing. Further, one of the two genes that suppress CIN when increased in dosage, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002840">NPL3</a></em>, is a major RNA binding protein important for pre-mRNA splicing and transport. <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000935">GLC7</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003506">TAF1</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002840">NPL3</a></em> gene deletion also were shown previously to elevate CIN (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Francisco <em>et al.</em> 1994</a>; <a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Stirling <em>et al.</em> 2011</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wahba <em>et al.</em> 2011</a>). One potential role hypothesized for <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002840">Npl3</a> is to limit the concentration of RNA in the nucleus, thereby preventing the formation of DNA:RNA hybrids (R-loops), which can lead to DNA damage and CIN (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aguilera and García-Muse 2012</a>). A recent study demonstrated a direct involvement of Npl3-containing heterogeneous ribonucleoprotein particle complex that suppresses CIN in preventing R-loop stabilization (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Santos-Pereira <em>et al.</em> 2013</a>), consistent with our result demonstrating that introducing a point mutation in the RNA recognition motif of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002840">Npl3</a> not only reversed the CIN-suppressing effect but led to elevated CIN. In addition to this potential mechanism, these RNA regulatory genes could also exert their effects on CIN by influencing the timing or the level of expression of cell cycle regulators such as G1 or mitotic cyclins.</p><p id="p-47">Our analyses using the qCTF assay revealed three conditions that could suppress basal CIN level through different mechanisms based on the known functions of these genes - increased copy numbers of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002840">NPL3</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002161">MCD1</a></em>, or both <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003054">MAD1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003567">MAD2</a></em> together. This raises the question as to why the basal CIN level for WT euploid cells growing under stress-free condition is not minimized, since the reduction of basal CIN rate can be so easily accomplished through small increase in the expression of different genes. A potential answer to this question is that the basal error rate for mitosis might have been optimized rather than minimized to ensure a high-level genome stability and also permit adaptability by maintaining a minute level of karyotypic variants in the population. Emerging evidence from different organisms have shown that aneuploid karyotypes usually confer decreased fitness under stress-free condition, but in the presence of stress, the significant phenotypic variation conferred by karyotype diversity provides the substrate for evolutionary selection of adaptive variants (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Ni <em>et al.</em> 2013</a>; <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Pavelka <em>et al.</em> 2010</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Selmecki <em>et al.</em> 2006</a>). This dichotomy implies the possibility of an optimal error rate small enough to be well tolerated without stress but large enough to endow the population a sufficient level of inherent adaptability to cope with exposure to acute stress.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-48">We thank F. Spencer, C. Boone, S. Elledge, and P. Silver for kindly sharing reagents; B. Geppert and S. McCroskey for experimental assistance. J. Zhu and D. Heinecke were graduate students registered with the Open University. This work was supported by funding from National Institutes of Health (R01GM059964) to R. Li. </p><p id="p-49">Author contributions: J.Z. and R.L. designed the study. J.Z., D.H., W.M., and D.B. performed all experiments. B.R. developed the formula to calculate CIN rate. A.B. and J.H. set up the high-throughput flow cytometry method and performed cell sorting experiments. R.L. coordinated and supervised the project. J.Z., D.H., B.R., and R.L. prepared figures and wrote the manuscript. All authors read and agreed the paper content.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-1"><p id="p-50">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017913/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017913/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-51"><em>Communicating editor: J. Berman</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-09-08"><span class="received-label">Received </span>September 8, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-03-21"><span class="accepted-label">Accepted </span>March 21, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Zhu <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.1" data-doi="10.1016/j.molcel.2012.04.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aguilera</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">García-Muse</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">R loops: from transcription byproducts to threats to genome stability.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">124</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DAguilera%26rft.auinit1%253DA.%26rft.volume%253D46%26rft.issue%253D2%26rft.spage%253D115%26rft.epage%253D124%26rft.atitle%253DR%2Bloops%253A%2Bfrom%2Btranscription%2Bbyproducts%2Bto%2Bthreats%2Bto%2Bgenome%2Bstability.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2012.04.009%26rft_id%253Dinfo%253Apmid%252F22541554%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2012.04.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22541554&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1043.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303365300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.2" data-doi="10.1073/pnas.70.8.2281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ames</span>  <span class="cit-name-given-names">B. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durston</span>  <span class="cit-name-given-names">W. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamasaki</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">F. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Carcinogens are mutagens: a simple test system combining liver homogenates for activation and bacteria for detection.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">2281</span>–<span class="cit-lpage">2285</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DAmes%26rft.auinit1%253DB.%2BN.%26rft.volume%253D70%26rft.issue%253D8%26rft.spage%253D2281%26rft.epage%253D2285%26rft.atitle%253DCarcinogens%2Bare%2BMutagens%253A%2BA%2BSimple%2BTest%2BSystem%2BCombining%2BLiver%2BHomogenates%2Bfor%2BActivation%2Band%2BBacteria%2Bfor%2BDetection%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.70.8.2281%26rft_id%253Dinfo%253Apmid%252F4151811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI3MC84LzIyODEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTA0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.3" data-doi="10.1091/mbc.E09-01-0029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barnhart</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorer</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schuyler</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Reduced Mad2 expression keeps relaxed kinetochores from arresting budding yeast in mitosis.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2448</span>–<span class="cit-lpage">2457</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E09-01-0029%26rft_id%253Dinfo%253Apmid%252F21593209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8xNC8yNDQ4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEwNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.4" data-doi="10.1038/nature11935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burrell</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McClelland</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endesfelder</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Groth</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weller</span>  <span class="cit-name-given-names">M.-C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Replication stress links structural and numerical cancer chromosomal instability.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">494</span>: <span class="cit-fpage">492</span>–<span class="cit-lpage">496</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D494%26rft.spage%253D492%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11935%26rft_id%253Dinfo%253Apmid%252F23446422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11935&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23446422&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1043.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000315661500042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.5" data-doi="10.1038/12687"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolodner</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Gross chromosomal rearrangements in <em>Saccharomyces cerevisiae</em> replication and recombination defective mutants.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DChen%26rft.auinit1%253DC.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D81%26rft.epage%253D85%26rft.atitle%253DGross%2Bchromosomal%2Brearrangements%2Bin%2BSaccharomyces%2Bcerevisiae%2Breplication%2Band%2Brecombination%2Bdefective%2Bmutants.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F12687%26rft_id%253Dinfo%253Apmid%252F10471504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/12687&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10471504&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1043.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000082337300021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.6" data-doi="10.1038/nature10795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradford</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seidel</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Hsp90 stress potentiates rapid cellular adaptation through induction of aneuploidy.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">482</span>: <span class="cit-fpage">246</span>–<span class="cit-lpage">250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DChen%26rft.auinit1%253DG.%26rft.volume%253D482%26rft.issue%253D7384%26rft.spage%253D246%26rft.epage%253D250%26rft.atitle%253DHsp90%2Bstress%2Bpotentiates%2Brapid%2Bcellular%2Badaptation%2Bthrough%2Binduction%2Bof%2Baneuploidy.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10795%26rft_id%253Dinfo%253Apmid%252F22286062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22286062&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1043.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299994100044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.7" data-doi="10.1091/mbc.E12-12-0902"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Choy</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Toole</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schuster</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crisp</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karpova</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide haploinsufficiency screen reveals a novel role for γ-TuSC in spindle organization and genome stability.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2753</span>–<span class="cit-lpage">2763</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E12-12-0902%26rft_id%253Dinfo%253Apmid%252F23825022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyNC8xNy8yNzUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEwNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.8" data-doi="10.1016/j.cell.2013.10.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davoli</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">A.</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mengwasser</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sack</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoon</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Cumulative haploinsufficiency and triplosensitivity drive aneuploidy patterns and shape the cancer genome.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">948</span>–<span class="cit-lpage">962</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D155%26rft.spage%253D948%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.10.011%26rft_id%253Dinfo%253Apmid%252F24183448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.10.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24183448&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1043.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000326934300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.9" data-doi="10.1534/genetics.104.036871"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deutschbauer</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaramillo</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Proctor</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumm</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hillenmeyer</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Mechanisms of haploinsufficiency revealed by genome-wide profiling in yeast.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">1915</span>–<span class="cit-lpage">1925</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.036871%26rft_id%253Dinfo%253Apmid%252F15716499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS80LzE5MTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTA0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-other" id="cit-5.6.1043.10" data-doi="10.1534/g3.113.010108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dirick</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bendris</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loubiere</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gostan</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gueydon</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Metabolic and environmental conditions determine nuclear genomic instability in budding yeast lacking mitochondrial DNA.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">4</span>: <span class="cit-fpage">411</span>–<span class="cit-lpage">423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.010108%26rft_id%253Dinfo%253Apmid%252F24374640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjQvMy80MTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTA0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.11" data-doi="10.1093/nar/gks1094"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Franceschini</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szklarczyk</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frankild</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simonovic</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">STRING v9.1: protein-protein interaction networks, with increased coverage and integration.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">D808</span>–<span class="cit-lpage">D815</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1094%26rft_id%253Dinfo%253Apmid%252F23203871%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MS9EMS9EODA4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEwNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.12" data-doi="10.1128/MCB.14.7.4731"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Francisco</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Type 1 protein phosphatase acts in opposition to IpL1 protein kinase in regulating yeast chromosome segregation.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">4731</span>–<span class="cit-lpage">4740</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DFrancisco%26rft.auinit1%253DL%26rft.volume%253D14%26rft.issue%253D7%26rft.spage%253D4731%26rft.epage%253D4740%26rft.atitle%253DType%2B1%2Bprotein%2Bphosphatase%2Bacts%2Bin%2Bopposition%2Bto%2BIpL1%2Bprotein%2Bkinase%2Bin%2Bregulating%2Byeast%2Bchromosome%2Bsegregation.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.14.7.4731%26rft_id%253Dinfo%253Apmid%252F8007975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE0LzcvNDczMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMDQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.13" data-doi="10.1016/j.tig.2007.11.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Geigl</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Obenauf</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwarzbraun</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Speicher</span>  <span class="cit-name-given-names">M. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Defining ‘chromosomal instability’.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">64</span>–<span class="cit-lpage">69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DGeigl%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D24%26rft.issue%253D2%26rft.spage%253D64%26rft.epage%253D69%26rft.atitle%253DDefining%2B%2527chromosomal%2Binstability%2527.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2007.11.006%26rft_id%253Dinfo%253Apmid%252F18192061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2007.11.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18192061&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1043.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253390300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.14" data-doi="10.1038/nature02046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ghaemmaghami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huh</span>  <span class="cit-name-given-names">W.-K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bower</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howson</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belle</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Global analysis of protein expression in yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">425</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">741</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGhaemmaghami%26rft.auinit1%253DS.%26rft.volume%253D425%26rft.issue%253D6959%26rft.spage%253D737%26rft.epage%253D741%26rft.atitle%253DGlobal%2Banalysis%2Bof%2Bprotein%2Bexpression%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02046%26rft_id%253Dinfo%253Apmid%252F14562106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14562106&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1043.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185924500046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.15" data-doi="10.1534/genetics.111.134577"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haber</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mating-type genes and MAT switching in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHaber%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D191%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D64%26rft.atitle%253DMating-Type%2BGenes%2Band%2BMAT%2BSwitching%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.134577%26rft_id%253Dinfo%253Apmid%252F22555442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTkxLzEvMzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTA0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.16" data-doi="10.1534/genetics.110.121079"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henry</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dilkes</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burkart-Waco</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Comai</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Phenotypic consequences of aneuploidy in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">1231</span>–<span class="cit-lpage">1245</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.121079%26rft_id%253Dinfo%253Apmid%252F20876566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Ni80LzEyMzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTA0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.17" data-doi="10.1038/nbt.1534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magtanong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gresham</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishimura</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A molecular barcoded yeast ORF library enables mode-of-action analysis of bioactive compounds.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">369</span>–<span class="cit-lpage">377</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DHo%26rft.auinit1%253DC.%2BH.%26rft.volume%253D27%26rft.issue%253D4%26rft.spage%253D369%26rft.epage%253D377%26rft.atitle%253DA%2Bmolecular%2Bbarcoded%2Byeast%2BORF%2Blibrary%2Benables%2Bmode-of-action%2Banalysis%2Bof%2Bbioactive%2Bcompounds.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1534%26rft_id%253Dinfo%253Apmid%252F19349972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19349972&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1043.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264971800023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.18" data-doi="10.1111/cas.12159"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Itoh</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugino</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ikeda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mizuguchi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanno</span>  <span class="cit-name-given-names">S.-I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Nucleoporin Nup188 is required for chromosome alignment in mitosis.</span> <abbr class="cit-jnl-abbrev">Cancer Sci.</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">871</span>–<span class="cit-lpage">879</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BSci.%26rft.volume%253D104%26rft.spage%253D871%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fcas.12159%26rft_id%253Dinfo%253Apmid%252F23551833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/cas.12159&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23551833&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1043.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.19" data-doi="10.1371/journal.pone.0068094"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumaran</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leu</span>  <span class="cit-name-given-names">J.-Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Characterization of chromosome stability in diploid, polyploid and hybrid yeast cells.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e68094</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De68094%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0068094%26rft_id%253Dinfo%253Apmid%252F23874507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0068094&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23874507&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1043.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.20" data-doi="10.1128/MCB.26.1.371-380.2006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laney</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mobley</span>  <span class="cit-name-given-names">E. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochstrasser</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The short-lived Matα2 transcriptional repressor is protected from degradation in vivo by interactions with its corepressors Tup1 and Ssn6.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">380</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DLaney%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D26%26rft.issue%253D1%26rft.spage%253D371%26rft.epage%253D380%26rft.atitle%253DThe%2BShort-Lived%2BMat%257Balpha%257D2%2BTranscriptional%2BRepressor%2BIs%2BProtected%2Bfrom%2BDegradation%2BIn%2BVivo%2Bby%2BInteractions%2Bwith%2BIts%2BCorepressors%2BTup1%2Band%2BSsn6%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.26.1.371-380.2006%26rft_id%253Dinfo%253Apmid%252F16354707%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjI2LzEvMzcxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEwNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.21" data-doi="10.1101/gad.10.10.1233"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henry</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silver</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">A protein that shuttles between the nucleus and the cytoplasm is an important mediator of RNA export.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1233</span>–<span class="cit-lpage">1246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DLee%26rft.auinit1%253DM%2BS%26rft.volume%253D10%26rft.issue%253D10%26rft.spage%253D1233%26rft.epage%253D1246%26rft.atitle%253DA%2Bprotein%2Bthat%2Bshuttles%2Bbetween%2Bthe%2Bnucleus%2Band%2Bthe%2Bcytoplasm%2Bis%2Ban%2Bimportant%2Bmediator%2Bof%2BRNA%2Bexport.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.10.10.1233%26rft_id%253Dinfo%253Apmid%252F8675010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjEwLzEwLzEyMzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTA0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.22" data-doi="10.1038/ng1505"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elledge</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">MAGIC, an in vivo genetic method for the rapid construction of recombinant DNA molecules.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">311</span>–<span class="cit-lpage">319</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DM.%2BZ.%26rft.volume%253D37%26rft.issue%253D3%26rft.spage%253D311%26rft.epage%253D319%26rft.atitle%253DMAGIC%252C%2Ban%2Bin%2Bvivo%2Bgenetic%2Bmethod%2Bfor%2Bthe%2Brapid%2Bconstruction%2Bof%2Brecombinant%2BDNA%2Bmolecules.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1505%26rft_id%253Dinfo%253Apmid%252F15731760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1505&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15731760&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1043.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227297800024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.23" data-doi="10.1016/0092-8674(81)90015-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Feedback control of mitosis in budding yeast.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">519</span>–<span class="cit-lpage">531</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLi%26rft.auinit1%253DR.%26rft.volume%253D66%26rft.issue%253D3%26rft.spage%253D519%26rft.epage%253D531%26rft.atitle%253DFeedback%2Bcontrol%2Bof%2Bmitosis%2Bin%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252881%252990015-5%26rft_id%253Dinfo%253Apmid%252F1651172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(81)90015-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1651172&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1043.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991GA94100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-other" id="cit-5.6.1043.24" data-doi="10.1038/embor.2012.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McGranahan</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burrell</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endesfelder</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novelli</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swanton</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Cancer chromosomal instability: therapeutic and diagnostic challenges.</span> <span class="cit-source">EMBO Rep.</span> <span class="cit-vol">13</span>: <span class="cit-fpage">528</span>–<span class="cit-lpage">538</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BRep.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fembor.2012.61%26rft_id%253Dinfo%253Apmid%252F22595889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZW1ib3IiO3M6NToicmVzaWQiO3M6ODoiMTMvNi81MjgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTA0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.25" data-doi="10.1038/nrm2163"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Musacchio</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salmon</span>  <span class="cit-name-given-names">E. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The spindle-assembly checkpoint in space and time.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">379</span>–<span class="cit-lpage">393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DMusacchio%26rft.auinit1%253DA.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253D379%26rft.epage%253D393%26rft.atitle%253DThe%2Bspindle-assembly%2Bcheckpoint%2Bin%2Bspace%2Band%2Btime.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm2163%26rft_id%253Dinfo%253Apmid%252F17426725%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm2163&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17426725&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1043.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245970300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.26" data-doi="10.1371/journal.pbio.1001653"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ni</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feretzaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Floyd-Averette</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mieczkowski</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Unisexual and heterosexual meiotic reproduction generate aneuploidy and phenotypic diversity de novo in the yeast <em>Cryptococcus neoformans</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1001653</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D11%26rft.spage%253De1001653%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001653%26rft_id%253Dinfo%253Apmid%252F24058295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001653&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24058295&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1043.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.27" data-doi="10.1093/nar/27.15.3001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ouspenski</span>  <span class="cit-name-given-names">I. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elledge</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brinkley</span>  <span class="cit-name-given-names">B. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">New yeast genes important for chromosome integrity and segregation identified by dosage effects on genome stability.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">3001</span>–<span class="cit-lpage">3008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F27.15.3001%26rft_id%253Dinfo%253Apmid%252F10454593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyNy8xNS8zMDAxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEwNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.28" data-doi="10.1038/nature01771"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Papp</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pal</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurst</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Dosage sensitivity and the evolution of gene families in yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">424</span>: <span class="cit-fpage">194</span>–<span class="cit-lpage">197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPapp%26rft.auinit1%253DB.%26rft.volume%253D424%26rft.issue%253D6945%26rft.spage%253D194%26rft.epage%253D197%26rft.atitle%253DDosage%2Bsensitivity%2Band%2Bthe%2Bevolution%2Bof%2Bgene%2Bfamilies%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01771%26rft_id%253Dinfo%253Apmid%252F12853957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01771&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12853957&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1043.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184032700042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.29" data-doi="10.1038/nature09529"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pavelka</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rancati</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradford</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saraf</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Aneuploidy confers quantitative proteome changes and phenotypic variation in budding yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">468</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">325</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPavelka%26rft.auinit1%253DN.%26rft.volume%253D468%26rft.issue%253D7321%26rft.spage%253D321%26rft.epage%253D325%26rft.atitle%253DAneuploidy%2Bconfers%2Bquantitative%2Bproteome%2Bchanges%2Band%2Bphenotypic%2Bvariation%2Bin%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09529%26rft_id%253Dinfo%253Apmid%252F20962780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09529&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20962780&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1043.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284051000054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.30" data-doi="10.1007/s10555-013-9436-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Potapova</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Aneuploidy and chromosomal instability: a vicious cycle driving cellular evolution and cancer genome chaos.</span> <abbr class="cit-jnl-abbrev">Cancer Metastasis Rev.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">377</span>–<span class="cit-lpage">389</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BMetastasis%2BRev.%26rft.volume%253D32%26rft.spage%253D377%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10555-013-9436-6%26rft_id%253Dinfo%253Apmid%252F23709119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10555-013-9436-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23709119&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1043.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.31" data-doi="10.1083/jcb.201012035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ricke</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeganathan</span>  <span class="cit-name-given-names">K. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Deursen</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Bub1 overexpression induces aneuploidy and tumor formation through Aurora B kinase hyperactivation.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">1049</span>–<span class="cit-lpage">1064</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.201012035%26rft_id%253Dinfo%253Apmid%252F21646403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjEwOiIxOTMvNi8xMDQ5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEwNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.32" data-doi="10.1073/pnas.1201911109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ryan</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Britigan</span>  <span class="cit-name-given-names">E. M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zasadil</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witte</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Audhya</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Up-regulation of the mitotic checkpoint component Mad1 causes chromosomal instability and resistance to microtubule poisons.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">E2205</span>–<span class="cit-lpage">E2214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1201911109%26rft_id%253Dinfo%253Apmid%252F22778409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzMzL0UyMjA1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEwNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.33" data-doi="10.1038/nrc2771"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Santarius</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shipley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brewer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stratton</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A census of amplified and overexpressed human cancer genes.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Cancer</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BCancer%26rft.stitle%253DNat%2BRev%2BCancer%26rft.aulast%253DSantarius%26rft.auinit1%253DT.%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253D59%26rft.epage%253D64%26rft.atitle%253DA%2Bcensus%2Bof%2Bamplified%2Band%2Boverexpressed%2Bhuman%2Bcancer%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrc2771%26rft_id%253Dinfo%253Apmid%252F20029424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrc2771&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20029424&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1043.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273809000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.34" data-doi="10.1101/gad.229880.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Santos-Pereira</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herrero</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">García-Rubio</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marín</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreno</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The Npl3 hnRNP prevents R-loop-mediated transcription—replication conflicts and genome instability.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2445</span>–<span class="cit-lpage">2458</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DSantos-Pereira%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D27%26rft.issue%253D22%26rft.spage%253D2445%26rft.epage%253D2458%26rft.atitle%253DThe%2BNpl3%2BhnRNP%2Bprevents%2BR-loop-mediated%2Btranscription-replication%2Bconflicts%2Band%2Bgenome%2Binstability%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.229880.113%26rft_id%253Dinfo%253Apmid%252F24240235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjI3LzIyLzI0NDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTA0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.35" data-doi="10.1038/nrc2781"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schvartzman</span>  <span class="cit-name-given-names">J.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sotillo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benezra</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mitotic chromosomal instability and cancer: mouse modelling of the human disease.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Cancer</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">102</span>–<span class="cit-lpage">115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BCancer%26rft.stitle%253DNat%2BRev%2BCancer%26rft.aulast%253DSchvartzman%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253D102%26rft.epage%253D115%26rft.atitle%253DMitotic%2Bchromosomal%2Binstability%2Band%2Bcancer%253A%2Bmouse%2Bmodelling%2Bof%2Bthe%2Bhuman%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrc2781%26rft_id%253Dinfo%253Apmid%252F20094045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrc2781&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20094045&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1043.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273809900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.36" data-doi="10.1126/science.1128242"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Selmecki</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forche</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Aneuploidy and isochromosome formation in drug-resistant <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">313</span>: <span class="cit-fpage">367</span>–<span class="cit-lpage">370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSelmecki%26rft.auinit1%253DA.%26rft.volume%253D313%26rft.issue%253D5785%26rft.spage%253D367%26rft.epage%253D370%26rft.atitle%253DAneuploidy%2Band%2BIsochromosome%2BFormation%2Bin%2BDrug-Resistant%2BCandida%2Balbicans%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1128242%26rft_id%253Dinfo%253Apmid%252F16857942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTMvNTc4NS8zNjciO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTA0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.37" data-doi="10.1101/gr.1239303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shannon</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Markiel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozier</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baliga</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Cytoscape: a software environment for integrated models of biomolecular interaction networks.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2498</span>–<span class="cit-lpage">2504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DShannon%26rft.auinit1%253DP.%26rft.volume%253D13%26rft.issue%253D11%26rft.spage%253D2498%26rft.epage%253D2504%26rft.atitle%253DCytoscape%253A%2BA%2BSoftware%2BEnvironment%2Bfor%2BIntegrated%2BModels%2Bof%2BBiomolecular%2BInteraction%2BNetworks%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1239303%26rft_id%253Dinfo%253Apmid%252F14597658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMy8xMS8yNDk4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEwNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.38" data-doi="10.1016/j.ccr.2006.10.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sotillo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernando</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Díaz-Rodríguez</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teruya-Feldstein</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cordón-Cardo</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mad2 overexpression promotes aneuploidy and tumorigenesis in mice.</span> <abbr class="cit-jnl-abbrev">Cancer Cell</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2Bcell%26rft.stitle%253DCancer%2BCell%26rft.aulast%253DSotillo%26rft.auinit1%253DR.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D9%26rft.epage%253D23%26rft.atitle%253DMad2%2Boverexpression%2Bpromotes%2Baneuploidy%2Band%2Btumorigenesis%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ccr.2006.10.019%26rft_id%253Dinfo%253Apmid%252F17189715%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ccr.2006.10.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17189715&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1043.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243701000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spencer</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerring</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connelly</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hieter</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Mitotic chromosome transmission fidelity mutants in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSpencer%26rft.auinit1%253DF.%26rft.volume%253D124%26rft.issue%253D2%26rft.spage%253D237%26rft.epage%253D249%26rft.atitle%253DMitotic%2Bchromosome%2Btransmission%2Bfidelity%2Bmutants%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Apmid%252F2407610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI0LzIvMjM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEwNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.40" data-doi="10.1371/journal.pgen.1002057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stirling</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solanki-Patil</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sipahimalani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The complete spectrum of yeast chromosome instability genes identifies candidate CIN cancer genes and functional roles for ASTRA complex components.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002057</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DStirling%26rft.auinit1%253DP.%2BC.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253De1002057%26rft.epage%253De1002057%26rft.atitle%253DThe%2Bcomplete%2Bspectrum%2Bof%2Byeast%2Bchromosome%2Binstability%2Bgenes%2Bidentifies%2Bcandidate%2BCIN%2Bcancer%2Bgenes%2Band%2Bfunctional%2Broles%2Bfor%2BASTRA%2Bcomplex%2Bcomponents.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002057%26rft_id%253Dinfo%253Apmid%252F21552543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21552543&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1043.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.41" data-doi="10.1534/genetics.107.084103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strome</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimmel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plon</span>  <span class="cit-name-given-names">S. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Heterozygous screen in <em>Saccharomyces cerevisiae</em> identifies dosage-sensitive genes that affect chromosome stability.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">1193</span>–<span class="cit-lpage">1207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.084103%26rft_id%253Dinfo%253Apmid%252F18245329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC8zLzExOTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTA0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.42" data-doi="10.1126/science.1142210"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Torres</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sokolsky</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">L. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boselli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Effects of aneuploidy on cellular physiology and cell division in haploid yeast.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">317</span>: <span class="cit-fpage">916</span>–<span class="cit-lpage">924</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTorres%26rft.auinit1%253DE.%2BM.%26rft.volume%253D317%26rft.issue%253D5840%26rft.spage%253D916%26rft.epage%253D924%26rft.atitle%253DEffects%2Bof%2BAneuploidy%2Bon%2BCellular%2BPhysiology%2Band%2BCell%2BDivision%2Bin%2BHaploid%2BYeast%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1142210%26rft_id%253Dinfo%253Apmid%252F17702937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTcvNTg0MC85MTYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTA0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.43" data-doi="10.1016/j.cell.2009.04.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Veatch</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMurray</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">Z. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gottschling</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mitochondrial dysfunction leads to nuclear genome instability via an iron-sulfur cluster defect.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">1247</span>–<span class="cit-lpage">1258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DVeatch%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D137%26rft.issue%253D7%26rft.spage%253D1247%26rft.epage%253D1258%26rft.atitle%253DMitochondrial%2Bdysfunction%2Bleads%2Bto%2Bnuclear%2Bgenome%2Binstability%2Bvia%2Ban%2Biron-sulfur%2Bcluster%2Bdefect.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2009.04.014%26rft_id%253Dinfo%253Apmid%252F19563757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2009.04.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19563757&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1043.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267373400018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.44" data-doi="10.1016/j.molcel.2011.10.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wahba</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amon</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koshland</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vuica-Ross</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">RNase H and multiple RNA biogenesis factors cooperate to prevent RNA:DNA hybrids from generating genome instability.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">978</span>–<span class="cit-lpage">988</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DWahba%26rft.auinit1%253DL.%26rft.volume%253D44%26rft.issue%253D6%26rft.spage%253D978%26rft.epage%253D988%26rft.atitle%253DRNase%2BH%2Band%2Bmultiple%2BRNA%2Bbiogenesis%2Bfactors%2Bcooperate%2Bto%2Bprevent%2BRNA%253ADNA%2Bhybrids%2Bfrom%2Bgenerating%2Bgenome%2Binstability.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2011.10.017%26rft_id%253Dinfo%253Apmid%252F22195970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2011.10.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22195970&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1043.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298827200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.45" data-doi="10.1093/nar/gkt439"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duncan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">WEB-based GEne SeT AnaLysis Toolkit (WebGestalt): update 2013.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">W77</span>–<span class="cit-lpage">W83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt439%26rft_id%253Dinfo%253Apmid%252F23703215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjQxL1cxL1c3NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMDQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.46" data-doi="10.1091/mbc.E02-04-0203"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Warren</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brady</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hardwick</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Distinct chromosome segregation roles for spindle checkpoint proteins.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">3029</span>–<span class="cit-lpage">3041</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E02-04-0203%26rft_id%253Dinfo%253Apmid%252F12221113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjEzLzkvMzAyOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMDQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.47" data-doi="10.1073/pnas.0610642104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yuen</span>  <span class="cit-name-given-names">K. W. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warren</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwok</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hieter</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Systematic genome instability screens in yeast and their potential relevance to cancer.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">3925</span>–<span class="cit-lpage">3930</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0610642104%26rft_id%253Dinfo%253Apmid%252F17360454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA0LzEwLzM5MjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTA0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.48" data-doi="10.1038/ng.2760"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zack</span>  <span class="cit-name-given-names">T. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schumacher</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cherniack</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saksena</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Pan-cancer patterns of somatic copy number alteration.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">1134</span>–<span class="cit-lpage">1140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D1134%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2760%26rft_id%253Dinfo%253Apmid%252F24071852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2760&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24071852&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1043.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1043.49" data-doi="10.1371/journal.pgen.1002719"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavelka</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradford</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rancati</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Karyotypic determinants of chromosome instability in aneuploid budding yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002719</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DZhu%26rft.auinit1%253DJ.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253De1002719%26rft.epage%253De1002719%26rft.atitle%253DKaryotypic%2Bdeterminants%2Bof%2Bchromosome%2Binstability%2Bin%2Baneuploid%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002719%26rft_id%253Dinfo%253Apmid%252F22615582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002719&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22615582&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1043.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/6/1043.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/6/1035" title="Novel Genes Affecting Blood Pressure Detected Via Gene-Based Association Analysis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/6/1057" title="X Chromosome and Autosome Dosage Responses in Drosophila melanogaster Heads" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/6"><span>Volume 5 Issue 6, June 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/6.cover-source.jpg?itok=NSNFWJbY" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1043" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-LHqK8e3qc24vrFyuoXUhYdaI5JcPiLIcINZdC-7yA64" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/6/1043" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/6/1043" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/6/1043" class="active" data-icon-position="" data-hide-link-title="0">Single-Cell Based Quantitative Assay of Chromosome Transmission Fidelity</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459953" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-h1qf0KBk2GXeV2X3wCi9a-k4--F4IpZvTYc7w343aD8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/6/1043.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/6/1043" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Bp3Al_3Kz5pMJ7Vv-Xz-w1tqi0b5l4fp0UQPNeL35ss" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/6/1043" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/6/1043.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459953" id="node459953--2" data-pisa="ggg;5/6/1043" data-pisa-master="ggg;g3.115.017913" data-apath="/ggg/5/6/1043.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Single-Cell Based Quantitative Assay of Chromosome Transmission Fidelity</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jin Zhu</span>, <span class="highwire-citation-author" data-delta="1">Dominic Heinecke</span>, <span class="highwire-citation-author" data-delta="2">Wahid A. Mulla</span>, <span class="highwire-citation-author" data-delta="3">William D. Bradford</span>, <span class="highwire-citation-author" data-delta="4">Boris Rubinstein</span>, <span class="highwire-citation-author" data-delta="5">Andrew Box</span>, <span class="highwire-citation-author" data-delta="6">Jeffrey S. Haug</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-1185-7283" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rong Li</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1043-1056;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017913 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459953--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jin Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jin%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJin%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dominic Heinecke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dominic%2BHeinecke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Heinecke%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADominic%2BHeinecke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Wahid A. Mulla </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular and Integrative Physiology, University of Kansas Medical Center, 3901 Rainbow Boulevard, Kansas City, Kansas 66160</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wahid%2BA.%2BMulla%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mulla%20WA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWahid%2BA.%2BMulla%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">William D. Bradford </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BD.%2BBradford%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bradford%20WD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BD.%2BBradford%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Boris Rubinstein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Boris%2BRubinstein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rubinstein%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABoris%2BRubinstein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Andrew Box </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BBox%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Box%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BBox%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jeffrey S. Haug </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BS.%2BHaug%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haug%20JS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeffrey%2BS.%2BHaug%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Rong Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular and Integrative Physiology, University of Kansas Medical Center, 3901 Rainbow Boulevard, Kansas City, Kansas 66160</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rong%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARong%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1185-7283" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rong Li</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rli@stowers.org" class="" data-icon-position="" data-hide-link-title="0">rli@stowers.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/6/1043" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/6/1043&amp;title=Single-Cell%20Based%20Quantitative%20Assay%20of%20Chromosome%20Transmission%20Fidelity" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/6/1043&amp;title=Single-Cell%20Based%20Quantitative%20Assay%20of%20Chromosome%20Transmission%20Fidelity" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/6/1043&amp;title=Single-Cell%20Based%20Quantitative%20Assay%20of%20Chromosome%20Transmission%20Fidelity" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/6/1043&amp;text=Single-Cell%20Based%20Quantitative%20Assay%20of%20Chromosome%20Transmission%20Fidelity" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/6/1043&amp;title=Single-Cell%20Based%20Quantitative%20Assay%20of%20Chromosome%20Transmission%20Fidelity" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/6/1043&amp;t=Single-Cell%20Based%20Quantitative%20Assay%20of%20Chromosome%20Transmission%20Fidelity" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/6/1043&amp;title=Single-Cell%20Based%20Quantitative%20Assay%20of%20Chromosome%20Transmission%20Fidelity" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/6/1043&amp;title=Single-Cell%20Based%20Quantitative%20Assay%20of%20Chromosome%20Transmission%20Fidelity" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459953/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459953" id="node459953--3" data-pisa="ggg;5/6/1043" data-pisa-master="ggg;g3.115.017913" data-apath="/ggg/5/6/1043.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Single-Cell Based Quantitative Assay of Chromosome Transmission Fidelity</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jin Zhu</span>, <span class="highwire-citation-author" data-delta="1">Dominic Heinecke</span>, <span class="highwire-citation-author" data-delta="2">Wahid A. Mulla</span>, <span class="highwire-citation-author" data-delta="3">William D. Bradford</span>, <span class="highwire-citation-author" data-delta="4">Boris Rubinstein</span>, <span class="highwire-citation-author" data-delta="5">Andrew Box</span>, <span class="highwire-citation-author" data-delta="6">Jeffrey S. Haug</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-1185-7283" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rong Li</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1043-1056;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017913 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459953--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jin Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jin%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJin%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dominic Heinecke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dominic%2BHeinecke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Heinecke%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADominic%2BHeinecke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Wahid A. Mulla </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular and Integrative Physiology, University of Kansas Medical Center, 3901 Rainbow Boulevard, Kansas City, Kansas 66160</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wahid%2BA.%2BMulla%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mulla%20WA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWahid%2BA.%2BMulla%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">William D. Bradford </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BD.%2BBradford%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bradford%20WD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BD.%2BBradford%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Boris Rubinstein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Boris%2BRubinstein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rubinstein%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABoris%2BRubinstein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Andrew Box </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BBox%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Box%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BBox%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jeffrey S. Haug </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BS.%2BHaug%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haug%20JS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeffrey%2BS.%2BHaug%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Rong Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, 1000 East 50th Street, Kansas City, Missouri 64110</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular and Integrative Physiology, University of Kansas Medical Center, 3901 Rainbow Boulevard, Kansas City, Kansas 66160</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rong%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARong%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1185-7283" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rong Li</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rli@stowers.org" class="" data-icon-position="" data-hide-link-title="0">rli@stowers.org</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1043&amp;rft.epage=1056&amp;rft.atitle=Single-Cell%20Based%20Quantitative%20Assay%20of%20Chromosome%20Transmission%20Fidelity&amp;rft.volume=5&amp;rft.issue=6&amp;rft.date=2015-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Zhu%2C+Jin&amp;rft.au=Heinecke%2C+Dominic&amp;rft.au=Mulla%2C+Wahid+A.&amp;rft.au=Bradford%2C+William+D.&amp;rft.au=Rubinstein%2C+Boris&amp;rft.au=Box%2C+Andrew&amp;rft.au=Haug%2C+Jeffrey+S.&amp;rft.au=Li%2C+Rong"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459953/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459953/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459953/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459953/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459953/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459953/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459953/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459953/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459953/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459953/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459953/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459953/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/6/1043" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods and Materials</a></li><li class="odd"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-22" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/6/1043.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/6/1043.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/6/1043.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/6/1043.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
