<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/6/1273" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/6/1273.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/6/1273.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/6/1273.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/6/1273/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/6/1273.atom" />
<meta name="HW.pisa" content="ggg;5/6/1273" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Dynamic Mobile DNA Family in the Yeast Mitochondrial Genome" />
<meta name="DC.Identifier" content="10.1534/g3.115.017822" />
<meta name="DC.Date" content="2015-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Wu and Hao. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Transposable elements (TEs) are an important factor shaping eukaryotic genomes. Although a significant body of research has been conducted on the abundance of TEs in nuclear genomes, TEs in mitochondrial genomes remain elusive. In this study, we successfully assembled 28 complete yeast mitochondrial genomes and took advantage of the power of population genomics to determine mobile DNAs and their propensity. We have observed compelling evidence of GC clusters propagating within the mitochondrial genome and being horizontally transferred between species. These mitochondrial TEs experience rapid diversification by nucleotide substitution and, more importantly, undergo dynamic merger and shuffling to form new TEs. Given the hyper mobile and transformable nature of mitochondrial TEs, our findings open the door to a deeper understanding of eukaryotic mitochondrial genome evolution and the origin of nonautonomous TEs." />
<meta name="DC.Contributor" content="Baojun Wu" />
<meta name="DC.Contributor" content="Weilong Hao" />
<meta name="article:published_time" content="2015-06-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Dynamic Mobile DNA Family in the Yeast Mitochondrial Genome" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Transposable elements (TEs) are an important factor shaping eukaryotic genomes. Although a significant body of research has been conducted on the abundance of TEs in nuclear genomes, TEs in mitochondrial genomes remain elusive. In this study, we successfully assembled 28 complete yeast mitochondrial genomes and took advantage of the power of population genomics to determine mobile DNAs and their propensity. We have observed compelling evidence of GC clusters propagating within the mitochondrial genome and being horizontally transferred between species. These mitochondrial TEs experience rapid diversification by nucleotide substitution and, more importantly, undergo dynamic merger and shuffling to form new TEs. Given the hyper mobile and transformable nature of mitochondrial TEs, our findings open the door to a deeper understanding of eukaryotic mitochondrial genome evolution and the origin of nonautonomous TEs.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/06/01" />
<meta name="citation_mjid" content="ggg;5/6/1273" />
<meta name="citation_id" content="5/6/1273" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/6/1273" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/6/1273.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/6/1273.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/6/1273.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.017822" />
<meta name="citation_pmid" content="25897011" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1273" />
<meta name="citation_lastpage" content="1282" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Baojun Wu" />
<meta name="citation_author_institution" content="Department of Biological Sciences, Wayne State University, Detroit, Michigan 48202" />
<meta name="citation_author" content="Weilong Hao" />
<meta name="citation_author_institution" content="Department of Biological Sciences, Wayne State University, Detroit, Michigan 48202" />
<meta name="citation_author_email" content="haow@wayne.edu" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. F. Altschul;citation_author=T. L. Madden;citation_author=A. A. Schaffer;citation_author=J. Zhang;citation_author=Z. Zhang;citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.;citation_volume=25;citation_year=1997;citation_pmid=9254694;citation_doi=10.1093/nar/25.17.3389" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. J. Alverson;citation_author=X. Wei;citation_author=D. W. Rice;citation_author=D. B. Stern;citation_author=K. Barry;citation_title=Insights into the evolution of mitochondrial genome size from complete sequences of Citrullus lanatus and Cucurbita pepo (Cucurbitaceae).;citation_volume=27;citation_year=2010;citation_pmid=20118192;citation_doi=10.1093/molbev/msq029" />
<meta name="citation_reference" content="citation_journal_title=Journal of computational biology :  a journal of computational molecular cell biology;citation_journal_abbrev=J Comput Biol;citation_author=A. Bankevich;citation_author=S. Nurk;citation_author=D. Antipov;citation_author=A. A. Gurevich;citation_author=M. Dvorkin;citation_title=SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing.;citation_pages=455-477;citation_volume=19;citation_year=2012;citation_issue=5;citation_pmid=22506599;citation_doi=10.1089/cmb.2012.0021" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Bergstrom;citation_author=J. T. Simpson;citation_author=F. Salinas;citation_author=B. Barre;citation_author=L. Parts;citation_title=A high-definition view of functional genetic variation from natural yeast genomes.;citation_volume=31;citation_year=2014;citation_pmid=24425782;citation_doi=10.1093/molbev/msu037" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=G. Bernardi;citation_title=Organization and evolution of the mitochondrial genome of yeast.;citation_pages=25-35;citation_volume=9;citation_year=1976;citation_issue=1;citation_pmid=798040;citation_doi=10.1007/BF01796120" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=C. W. Birky;citation_title=THE INHERITANCE OF GENES IN MITOCHONDRIA AND CHLOROPLASTS: Laws, Mechanisms, and Models;citation_pages=125-148;citation_volume=35;citation_year=2001;citation_issue=1;citation_pmid=11700280;citation_doi=10.1146/annurev.genet.35.102401.090231" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=C. Bleykasten-Grosshans;citation_author=A. Friedrich;citation_author=J. Schacherer;citation_title=Genome-wide analysis of intraspecific transposon diversity in yeast.;citation_pages=399;citation_volume=14;citation_year=2013;citation_pmid=23768249;citation_doi=10.1186/1471-2164-14-399" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Boetzer;citation_author=C. V. Henkel;citation_author=H. J. Jansen;citation_author=D. Butler;citation_author=W. Pirovano;citation_title=Scaffolding pre-assembled contigs using SSPACE.;citation_volume=27;citation_year=2011;citation_pmid=21149342;citation_doi=10.1093/bioinformatics/btq683" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=M. Boetzer;citation_author=W. Pirovano;citation_title=Toward almost closed genomes with GapFiller.;citation_pages=R56-R56;citation_volume=13;citation_year=2012;citation_issue=6;citation_pmid=22731987;citation_doi=10.1186/gb-2012-13-6-r56" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=C. Bouchier;citation_author=L. Ma;citation_author=S. Creno;citation_author=B. Dujon;citation_author=C. Fairhead;citation_title=Complete mitochondrial genome sequences of three Nakaseomyces species reveal invasion by palindromic GC clusters and considerable size expansion.;citation_volume=9;citation_year=2009;citation_pmid=19758332;citation_doi=10.1111/j.1567-1364.2009.00551.x" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Camacho;citation_author=G. Coulouris;citation_author=V. Avagyan;citation_author=N. Ma;citation_author=J. Papadopoulos;citation_title=BLAST+: architecture and applications.;citation_pages=421-421;citation_volume=10;citation_year=2009;citation_pmid=20003500;citation_doi=10.1186/1471-2105-10-421" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=M. Carr;citation_author=D. Bensasson;citation_author=C. M. Bergman;citation_title=Evolutionary genomics of transposable elements in Saccharomyces cerevisiae.;citation_pages=e50978;citation_volume=7;citation_year=2012;citation_pmid=23226439;citation_doi=10.1371/journal.pone.0050978" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. Carver;citation_author=S. R. Harris;citation_author=M. Berriman;citation_author=J. Parkhill;citation_author=J. A. McQuillan;citation_title=Artemis: an integrated platform for visualization and analysis of high-throughput sequence-based experimental data.;citation_volume=28;citation_year=2012;citation_pmid=22199388;citation_doi=10.1093/bioinformatics/btr703" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=A. C. Christensen;citation_title=Plant mitochondrial genome evolution can be explained by DNA repair mechanisms.;citation_volume=5;citation_year=2013;citation_pmid=23645599;citation_doi=10.1093/gbe/evt069" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=L. S. Collier;citation_author=D. A. Largaespada;citation_title=Transposable elements and the dynamic somatic genome.;citation_pages=S5;citation_volume=8;citation_year=2007;citation_pmid=18047697;citation_doi=10.1186/gb-2007-8-s1-s5" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=G. E. Crooks;citation_author=G. Hon;citation_author=J. M. Chandonia;citation_author=S. E. Brenner;citation_title=WebLogo: A Sequence Logo Generator;citation_pages=1188-1190;citation_volume=14;citation_year=2004;citation_issue=6;citation_pmid=15173120;citation_doi=10.1101/gr.849004" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=M. de Zamaroczy;citation_author=G. Bernardi;citation_title=The GC clusters of the mitochondrial genome of yeast and their evolutionary origin.;citation_pages=1-22;citation_volume=41;citation_year=1986;citation_issue=1;citation_pmid=3009268;citation_doi=10.1016/0378-1119(86)90262-3" />
<meta name="citation_reference" content="citation_journal_abbrev=Genome Dyn;citation_author=J. M. Deragon;citation_author=J. M. Casacuberta;citation_author=O. Panaud;citation_title=Plant transposable elements.;citation_pages=69-82;citation_volume=4;citation_year=2008;citation_pmid=18756078" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=C. L. Dieckmann;citation_author=B. Gandy;citation_title=Preferential recombination between GC clusters in yeast mitochondrial DNA.;citation_pages=4197-4203;citation_volume=6;citation_year=1987;citation_pmid=3327690" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=B. Dujon;citation_author=P. P. Slonimski;citation_author=L. Weill;citation_title=MITOCHONDRIAL GENETICS IX: A MODEL FOR RECOMBINATION AND SEGREGATION OF MITOCHONDRIAL GENOMES IN SACCHAROMYCES CEREVISIAE;citation_pages=415-437;citation_volume=78;citation_year=1974;citation_issue=1;citation_pmid=4613610" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. C. Edgar;citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput;citation_pages=1792-1797;citation_volume=32;citation_year=2004;citation_issue=5;citation_pmid=15034147;citation_doi=10.1093/nar/gkh340" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=D. Erpenbeck;citation_author=O. Voigt;citation_author=G. Worheide;citation_author=D. V. Lavrov;citation_title=The mitochondrial genomes of sponges provide evidence for multiple invasions by Repetitive Hairpin-forming Elements (RHE).;citation_pages=591-591;citation_volume=10;citation_year=2009;citation_pmid=20003196;citation_doi=10.1186/1471-2164-10-591" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=R. Eskes;citation_author=L. Liu;citation_author=H. Ma;citation_author=M. Y. Chao;citation_author=L. Dickson;citation_title=Multiple Homing Pathways Used by Yeast Mitochondrial Group II Introns;citation_pages=8432-8446;citation_volume=20;citation_year=2000;citation_issue=22;citation_pmid=11046140;citation_doi=10.1128/MCB.20.22.8432-8446.2000" />
<meta name="citation_reference" content="citation_journal_title=Genome;citation_author=I. Fattash;citation_author=R. Rooke;citation_author=A. Wong;citation_author=C. Hui;citation_author=T. Luu;citation_title=Miniature inverted-repeat transposable elements: discovery, distribution, and activity.;citation_pages=475-486;citation_volume=56;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. S. Fritsch;citation_author=C. D. Chabbert;citation_author=B. Klaus;citation_author=L. M. Steinmetz;citation_title=A genome-wide map of mitochondrial DNA recombination in yeast.;citation_volume=198;citation_year=2014;citation_pmid=25081569;citation_doi=10.1534/genetics.114.166637" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. Gordon;citation_author=P. Green;citation_title=Consed: a graphical editor for next-generation sequencing.;citation_volume=29;citation_year=2013;citation_pmid=23995391;citation_doi=10.1093/bioinformatics/btt515" />
<meta name="citation_reference" content="citation_journal_title=Biochimie;citation_author=S. Gwiazda;citation_author=K. Salomon;citation_author=B. Appel;citation_author=S. Muller;citation_title=RNA self-ligation: from oligonucleotides to full length ribozymes.;citation_pages=1457-1463;citation_volume=94;citation_year=2012;citation_pmid=22465106;citation_doi=10.1016/j.biochi.2012.03.015" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=W. Hao;citation_author=G. B. Golding;citation_title=The fate of laterally transferred genes: Life in the fast lane to adaptation or death.;citation_pages=636-643;citation_volume=16;citation_year=2006;citation_issue=5;citation_pmid=16651664;citation_doi=10.1101/gr.4746406" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=N. J. Hepburn;citation_author=D. W. Schmidt;citation_author=J. P. Mower;citation_title=Loss of two introns from the Magnolia tripetala mitochondrial cox2 gene implicates horizontal gene transfer and gene conversion as a novel mechanism of intron loss.;citation_volume=29;citation_year=2012;citation_pmid=22593225;citation_doi=10.1093/molbev/mss130" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=M. S. Islam;citation_author=B. Studer;citation_author=S. L. Byrne;citation_author=J. D. Farrell;citation_author=F. Panitz;citation_title=The genome and transcriptome of perennial ryegrass mitochondria.;citation_pages=202;citation_volume=14;citation_year=2013;citation_pmid=23521852;citation_doi=10.1186/1471-2164-14-202" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=O. Khakhlova;citation_author=R. Bock;citation_title=Elimination of deleterious mutations in plastid genomes by gene conversion.;citation_pages=85-94;citation_volume=46;citation_year=2006;citation_issue=1;citation_pmid=16553897;citation_doi=10.1111/j.1365-313X.2006.02673.x" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=T. Kim;citation_author=W. Hao;citation_title=DiscML: an R package for estimating evolutionary rates of discrete characters using maximum likelihood.;citation_pages=320;citation_volume=15;citation_year=2014;citation_pmid=25260628;citation_doi=10.1186/1471-2105-15-320" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=V. Knoop;citation_author=M. Unseld;citation_author=J. Marienfeld;citation_author=P. Brandt;citation_author=S. Sunkel;citation_title=copia-, gypsy- and LINE-Like Retrotransposon Fragments in the Mitochondrial Genome of Arabidopsis thaliana;citation_pages=579-585;citation_volume=142;citation_year=1996;citation_issue=2;citation_pmid=8852855" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=F. Koll;citation_author=J. Boulay;citation_author=L. Belcour;citation_author=Y. d’Aubenton-Carafa;citation_title=Contribution of ultra-short invasive elements to the evolution of the mitochondrial genome in the genus Podospora.;citation_volume=24;citation_year=1996;citation_pmid=8649993;citation_doi=10.1093/nar/24.9.1734" />
<meta name="citation_reference" content="citation_journal_title=Annual review of biochemistry;citation_journal_abbrev=Annu Rev Biochem;citation_author=A. M. Lambowitz;citation_author=M. Belfort;citation_title=Introns as Mobile Genetic Elements;citation_pages=587-622;citation_volume=62;citation_year=1993;citation_issue=1;citation_pmid=8352597;citation_doi=10.1146/annurev.bi.62.070193.003103" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. F. Lang;citation_author=M. Jakubkova;citation_author=E. Hegedusova;citation_author=R. Daoud;citation_author=L. Forget;citation_title=Massive programmed translational jumping in mitochondria.;citation_volume=111;citation_year=2014;citation_pmid=24711422;citation_doi=10.1073/pnas.1322190111" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=D. V. Lavrov;citation_title=Rapid proliferation of repetitive palindromic elements in mtDNA of the endemic Baikalian sponge Lubomirskia baicalensis.;citation_volume=27;citation_year=2010;citation_pmid=20026479;citation_doi=10.1093/molbev/msp317" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=D. S. Lawrie;citation_author=P. W. Messer;citation_author=R. Hershberg;citation_author=D. A. Petrov;citation_title=Strong purifying selection at synonymous sites in D. melanogaster.;citation_pages=e1003527;citation_volume=9;citation_year=2013;citation_pmid=23737754;citation_doi=10.1371/journal.pgen.1003527" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=R. Lindner;citation_author=C. C. Friedel;citation_title=A comprehensive evaluation of alignment algorithms in the context of RNA-seq.;citation_pages=e52403;citation_volume=7;citation_year=2012;citation_pmid=23300661;citation_doi=10.1371/journal.pone.0052403" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Liti;citation_author=D. M. Carter;citation_author=A. M. Moses;citation_author=J. Warringer;citation_author=L. Parts;citation_title=Population genomics of domestic and wild yeasts.;citation_pages=337-341;citation_volume=458;citation_year=2009;citation_issue=7236;citation_pmid=19212322;citation_doi=10.1038/nature07743" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=R. Luo;citation_author=B. Liu;citation_author=Y. Xie;citation_author=Z. Li;citation_author=W. Huang;citation_title=SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler.;citation_pages=18;citation_volume=1;citation_year=2012;citation_pmid=23587118;citation_doi=10.1186/2047-217X-1-18" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Lynch;citation_author=J. S. Conery;citation_title=The Origins of Genome Complexity;citation_pages=1401-1404;citation_volume=302;citation_year=2003;citation_issue=5649;citation_pmid=14631042;citation_doi=10.1126/science.1089370" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Lynch;citation_author=B. Koskella;citation_author=S. Schaack;citation_title=Mutation pressure and the evolution of organelle genomic architecture.;citation_pages=1727-1730;citation_volume=311;citation_year=2006;citation_issue=5768;citation_pmid=16556832;citation_doi=10.1126/science.1118884" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=R. E. Mills;citation_author=E. A. Bennett;citation_author=R. C. Iskow;citation_author=S. E. Devine;citation_title=Which transposable elements are active in the human genome?;citation_pages=183-191;citation_volume=23;citation_year=2007;citation_issue=4;citation_pmid=17331616;citation_doi=10.1016/j.tig.2007.02.006" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=J. V. Moran;citation_author=S. Zimmerly;citation_author=R. Eskes;citation_author=J. C. Kennell;citation_author=A. M. Lambowitz;citation_title=Mobile group II introns of yeast mitochondrial DNA are novel site- specific retroelements;citation_pages=2828-2838;citation_volume=15;citation_year=1995;citation_issue=5;citation_pmid=7537853" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=L. Mularoni;citation_author=Y. Zhou;citation_author=T. Bowen;citation_author=S. Gangadharan;citation_author=S. J. Wheelan;citation_title=Retrotransposon Ty1 integration targets specifically positioned asymmetric nucleosomal DNA segments in tRNA hotspots.;citation_volume=22;citation_year=2012;citation_pmid=22219510;citation_doi=10.1101/gr.129460.111" />
<meta name="citation_reference" content="citation_journal_title=IUBMB life;citation_journal_abbrev=IUBMB Life;citation_author=S. Muller;citation_author=B. Appel;citation_author=T. Krellenberg;citation_author=S. Petkovic;citation_title=The many faces of the hairpin ribozyme: structural and functional variants of a small catalytic RNA.;citation_pages=36-47;citation_volume=64;citation_year=2012;citation_issue=1;citation_pmid=22131309;citation_doi=10.1002/iub.575" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=M. Nakazono;citation_author=A. Kanno;citation_author=N. Tsutsumi;citation_author=A. Hirai;citation_title=Palindromic repeated sequences (PRSs) in the mitochondrial genome of rice: evidence for their insertion after divergence of the genus Oryza from the other Gramineae.;citation_pages=273-281;citation_volume=24;citation_year=1994;citation_issue=2;citation_pmid=7509206;citation_doi=10.1007/BF00020167" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=K. Okonechnikov;citation_author=O. Golosova;citation_author=M. Fursov;citation_title=Unipro UGENE: a unified bioinformatics toolkit.;citation_volume=28;citation_year=2012;citation_pmid=22368248;citation_doi=10.1093/bioinformatics/bts091" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=M. Pagel;citation_author=A. Meade;citation_author=D. Barker;citation_title=Bayesian estimation of ancestral character states on phylogenies.;citation_volume=53;citation_year=2004;citation_pmid=15545248;citation_doi=10.1080/10635150490522232" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=B. Paquin;citation_author=M. J. Laforest;citation_author=B. F. Lang;citation_title=Double-Hairpin Elements in the Mitochondrial DNA of Allomyces: Evidence for Mobility;citation_pages=1760-1768;citation_volume=17;citation_year=2000;citation_issue=11;citation_pmid=11070063;citation_doi=10.1093/oxfordjournals.molbev.a026274" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=M. F. Polz;citation_author=E. J. Alm;citation_author=W. P. Hanage;citation_title=Horizontal gene transfer and the evolution of bacterial and archaeal population structure.;citation_pages=170-175;citation_volume=29;citation_year=2013;citation_pmid=23332119;citation_doi=10.1016/j.tig.2012.12.006" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=L. Rodriguez-Moreno;citation_author=V. M. Gonzalez;citation_author=A. Benjak;citation_author=M. C. Marti;citation_author=P. Puigdomenech;citation_title=Determination of the melon chloroplast and mitochondrial genome sequences reveals that the largest reported mitochondrial genome in plants contains a significant amount of DNA having a nuclear origin.;citation_pages=424-424;citation_volume=12;citation_year=2011;citation_pmid=21854637;citation_doi=10.1186/1471-2164-12-424" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. V. Sanchez-Puerta;citation_author=Y. Cho;citation_author=J. P. Mower;citation_author=A. J. Alverson;citation_author=J. D. Palmer;citation_title=Frequent, phylogenetically local horizontal transfer of the cox1 group I Intron in flowering plant mitochondria.;citation_volume=25;citation_year=2008;citation_pmid=18524785;citation_doi=10.1093/molbev/msn129" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=S. Schaack;citation_author=C. Gilbert;citation_author=C. Feschotte;citation_title=Promiscuous DNA: horizontal transfer of transposable elements and why it matters for eukaryotic evolution.;citation_pages=537-546;citation_volume=25;citation_year=2010;citation_issue=9;citation_pmid=20591532;citation_doi=10.1016/j.tree.2010.06.001" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. P. Shannon;citation_author=T. C. Kaufman;citation_author=M. W. Shen;citation_author=B. H. Judd;citation_title=LETHALITY PATTERNS AND MORPHOLOGY OF SELECTED LETHAL AND SEMI-LETHAL MUTATIONS IN THE ZESTE-WHITE REGION OF DROSOPHILA MELANOGASTER;citation_pages=615-638;citation_volume=72;citation_year=1972;citation_issue=4;citation_pmid=4631596" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=D. R. Smith;citation_author=R. W. Lee;citation_title=Mitochondrial genome of the colorless green alga Polytomella capuana: a linear molecule with an unprecedented GC content.;citation_volume=25;citation_year=2008;citation_pmid=18222946;citation_doi=10.1093/molbev/msm245" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=D. R. Smith;citation_author=R. W. Lee;citation_title=The mitochondrial and plastid genomes of Volvox carteri: bloated molecules rich in repetitive DNA.;citation_pages=132-132;citation_volume=10;citation_year=2009;citation_pmid=19323823;citation_doi=10.1186/1471-2164-10-132" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Stamatakis;citation_title=RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models.;citation_volume=22;citation_year=2006;citation_pmid=16928733;citation_doi=10.1093/bioinformatics/btl446" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=E. M. Turk;citation_author=V. Das;citation_author=R. D. Seibert;citation_author=E. D. Andrulis;citation_title=The mitochondrial RNA landscape of Saccharomyces cerevisiae.;citation_pages=e78105;citation_volume=8;citation_year=2013;citation_pmid=24143261;citation_doi=10.1371/journal.pone.0078105" />
<meta name="citation_reference" content="citation_journal_title=Genome Biology and Evolution;citation_journal_abbrev=Genome Biol Evol;citation_author=G. L. Wallau;citation_author=M. F. Ortiz;citation_author=E. L. Loreto;citation_title=Horizontal Transposon Transfer in Eukarya: Detection, Bias, and Perspectives;citation_pages=689-699;citation_volume=4;citation_year=2012;citation_issue=8;citation_pmid=22798449;citation_doi=10.1093/gbe/evs055" />
<meta name="citation_reference" content="citation_journal_title=Molecular &amp; general genetics :  MGG;citation_journal_abbrev=Mol Gen Genet;citation_author=G. Weiller;citation_author=C. M. Schueller;citation_author=R. J. Schweyen;citation_title=Putative target sites for mobile G + C rich clusters in yeast mitochondrial DNA: single elements and tandem arrays.;citation_pages=272-283;citation_volume=218;citation_year=1989;citation_issue=2;citation_pmid=2674655;citation_doi=10.1007/BF00331278" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. Yin;citation_author=J. Heckman;citation_title=Highly conserved GC-rich palindromic DNA sequences flank tRNA genes in Neurospora crassa mitochondria.;citation_pages=325-332;citation_volume=26;citation_year=1981;citation_issue=3 Pt 1;citation_pmid=6276013;citation_doi=10.1016/0092-8674(81)90201-4" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. R. Zerbino;citation_author=E. Birney;citation_title=Velvet: algorithms for de novo short read assembly using de Bruijn graphs.;citation_volume=18;citation_year=2008;citation_pmid=18349386;citation_doi=10.1101/gr.074492.107" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Zuker;citation_title=Mfold web server for nucleic acid folding and hybridization prediction;citation_pages=3406-3415;citation_volume=31;citation_year=2003;citation_issue=13;citation_pmid=12824337;citation_doi=10.1093/nar/gkg595" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Z. Zuo;citation_author=D. Peng;citation_author=X. Yin;citation_author=X. Zhou;citation_author=H. Cheng;citation_title=Genome-wide analysis reveals origin of transfer RNA genes from tRNA halves.;citation_volume=30;citation_year=2013;citation_pmid=23744908;citation_doi=10.1093/molbev/mst107" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Dynamic Mobile DNA Family in the Yeast Mitochondrial Genome" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
<meta name="twitter:description" content="Transposable elements (TEs) are an important factor shaping eukaryotic genomes. Although a significant body of research has been conducted on the abundance of TEs in nuclear genomes, TEs in mitochondrial genomes remain elusive. In this study, we successfully assembled 28 complete yeast mitochondrial genomes and took advantage of the power of population genomics to determine mobile DNAs and their propensity. We have observed compelling evidence of GC clusters propagating within the mitochondrial genome and being horizontally transferred between species. These mitochondrial TEs experience rapid diversification by nucleotide substitution and, more importantly, undergo dynamic merger and shuffling to form new TEs. Given the hyper mobile and transformable nature of mitochondrial TEs, our findings open the door to a deeper understanding of eukaryotic mitochondrial genome evolution and the origin of nonautonomous TEs." />
<meta name="og-title" property="og:title" content="A Dynamic Mobile DNA Family in the Yeast Mitochondrial Genome" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/6/1273" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Transposable elements (TEs) are an important factor shaping eukaryotic genomes. Although a significant body of research has been conducted on the abundance of TEs in nuclear genomes, TEs in mitochondrial genomes remain elusive. In this study, we successfully assembled 28 complete yeast mitochondrial genomes and took advantage of the power of population genomics to determine mobile DNAs and their propensity. We have observed compelling evidence of GC clusters propagating within the mitochondrial genome and being horizontally transferred between species. These mitochondrial TEs experience rapid diversification by nucleotide substitution and, more importantly, undergo dynamic merger and shuffling to form new TEs. Given the hyper mobile and transformable nature of mitochondrial TEs, our findings open the door to a deeper understanding of eukaryotic mitochondrial genome evolution and the origin of nonautonomous TEs." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6.cover-source.jpg" />
    <title>A Dynamic Mobile DNA Family in the Yeast Mitochondrial Genome | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"b4lTfM9mgNCm1K_KyLdzTMr8jYVkTAcDBYUZ9mSaz-A","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459968","apath":"\/ggg\/5\/6\/1273.atom","pisa":"ggg;5\/6\/1273","ac":{"\/ggg\/5\/6\/1273.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/6\/1273.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/6\/1273"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/6\/1273"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/6\/1273":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Tfx6n8RlVMX2WoHAZ8NI1yQCIDJezTFikD0SwxTvjNWkIxKXxzTs77+XtEbpew7KPKfeSxuYXgXle26lPcacICZNgA6MB9vupKCQcZni5wk6RjmO9VOXKJ3rWwbuq+6hPjLrU83p7VIiLl1\\\\\\\/F73k36iwmDz6rQrIaiIZ1V+VbCqglhBLDwzwlFQAdaM5Cqiw3iuYI7I6zfMPao42O4FaAbpFQKYkGBP8sTxJfdDr6zb4p0wqn0RH\\\\\\\/2dHnLzfgKZIwQ7cPi3cMkoaLYb4tQwA2cJK5i9SuLSZOGpvn89SRoTtYmfJ\\\\\\\/\\\\\\\/dUeZG3VPt6QiadInqyqhcg7MKThr8iwPwQ3XIp9ioWiIouBveDsVtjq70llfwJAdiOGZeSTgEq8zzed\\\\\\\/bdmqO878E4dWU4aq6T8wQvfjC9XJug6P7Vxpp8s5F4Zl64wTrfvL+C8f0dpdqFs\\\\\\\/7DRdFPZKFKkzr0Vih1yHrnJ+XzJVAAPeB1J0WJvCOPLvXLGB4rw2p3FKN9v2K+dKzTf1Qqy1Z02QmTVtjea44gNTJ6LzDzhpwp2OdTpJFEYwjjilguBDFAk9zlDRv1yrQNw02r30fhYBnDZhKFbWP\\\\\\\/ID\\\\\\\/e\\\\\\\/e5uHBg96P+prgBjYbqaagYUzRAhek8oZvgjs3Leg6ayJBgml6YwR5+CrhxJaf5iVeW9orez8QQp4\\\\\\\/s4pUwDL6Xd+\\\\\\\/N8RjyEKBN5ubvGD+vlqEzRqEhQIGx\\\\\\\/sqTH2cOcpaB7DFi8v6hqf+kNfK8iHLhSg\\\\\\\/45i+kFWGC7zAu\\\\\\\/7fG97CZvUMcBILcbjnw9qcP\\\\\\\/xnBXOzFH+PkBYMF7Wz11ymDS3cWC84yYs3ULa2hcjJG0uQvE323Kbt4IYEIS4G+m1cl232xFkaoHSGSl2yGVHZjhkUzn8NM85YDgWkdH4R8EJ\\\\\\\/LWqeEkd8nP\\\\\\\/Iu84ZuqanTjyoPMc1sNPXp2MzRjtnMqQsUs7+UrlGIRTbCt9Sn8zr1MdBwgCbD68vc\\\\\\\/7C0NEBtpI82fM6heJjL4sxbpQGuhBaxCw84e2gG\\\\\\\/+juwaMZyaC9X4NDK4DY7hfiZb84mfbHBB32tFPExEIhz0v64IWjUiJYLbnEnLsvCotp\\\\\\\/M+0mE508RlxkbZi6\\\\\\\/wnBx2D9NtbafswfIilcabejTyAbm2fgNhpgUoKrEN1wZi68gGxi1KTglms9M5M6LzWdg4GxukJVVWRvnwdFr3oTi7fiakVK4YV7G9rSwFgs4byUIvWQoSHngufN\\\\\\\/Nfihbjg9PyX++zmrdFLtj15nBftodX0c\\\\\\\/ctvqihsHrXOOmi\\\u0022,\\\u0022iv\\\u0022:\\\u0022m396gsAM9cGdjeyh6Fso9Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022518828f3da280cdc451a0cf0ec8f18e6634ef72e94cb880c6084098de0a7b7f4\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022O1KLupEGSqFBcTv92jr9WOZrS+U9MoDIlRmFyrbyGz2Q+7RFTI0OdwXUBKkM\\\\\\\/wLrMbY\\\\\\\/zJEbbCOJvE7gj3xms9tYVsCIzrVqfJfzR5A+Fd64+pNh03CesIAz2ejXpSbYN1vnB\\\\\\\/JwmsOXgsdtEV5D2hg2n4Qw1L5FoMV0WSiLXGycoPl7OxbdqVHNKN1zJCvTbl6YiSy+\\\\\\\/xWau\\\\\\\/31gfdmPx5hDZZVtsvVVvyzNFWRTrrkEKAcCHSnseg6BNqhBui1vlDkfUizfVSCNcWNVzq\\\\\\\/OXiCn+rOKXs3Y4QqbDplzLHGrXNa01FtJof7Q6S0cOgFP6rE39AaqOnOwDwJHU\\\\\\\/UJULS68fnvEDJav7pb3XDfiD8XHO49JuR7Ziwm9DPMua3SgUJCMHE7MMx\\\\\\\/h\\\\\\\/B\\\\\\\/yhx5FxU9LbD9BrL3n7x+4RGja6ufSJbdKNto5GLpl+v41QuIRTRZ6oNgjfEVvQI4mNVUnzbPVnP37w99CyBteEcc\\\\\\\/9kjxeqFThLWzTYIZn0X5eIAEjezxOhmVi5aUs5dKUIR56OtZDINOD9fEAGnd4bnlSIYwSBvWHfeDfCyquMkj\\\\\\\/6bhuL8T9A4ICg3j3ygiPjCiDJnt7V5w3dEsi2QEx93VK7lLTVM7D37lT02sWzoPseqRH2noCN52\\\\\\\/V7CQahYvevxXHHDLy77K0RSkiqBTpwZa8eSvsXPnsGZONiHQnyR49KoBlmhLoTYAGOpKo9MWmEro3Tx\\\\\\\/wyOqILmxW42yH4Q18bg5basxd\\\\\\\/tAsaU5CiAkv8Vrs9JeMOLvS0h8MBr74V+xPuBoGR19EK3WppQmaKRliHEF5KS1dW9coA6GJYd0T8V9eX\\\\\\\/e2U\\\\\\\/FctKIYxEHtaPrgQYrPjfJ6SoJ206cEamd1PWc2zgiz\\\\\\\/LHuk5BPd3De8ZaPyf9HFvNIEbvKeW+yXlSt960ESdxG8WHWLEyS+0uW7m\\\\\\\/wJU\\\\\\\/4l4ZRZpQW5wKcpyXgGhaH0yL3q+AXaeSuaJQomxVn7BdJfnxOMP\\\\\\\/I0kWLD+osLQmiw2D28PPaGM0IgMzqK8Zc3WYbnR2gRdZNunAWg88rMfi86EbqEdwhvFKd5DVBqkKhCiVNjsbRTZcVCSKxyTpZdVeNZk+bd2S1vs\\\\\\\/Haj+THxpsGKHwQh9dFQHCacC\\\\\\\/IMFHqop54nTnirz58b0J0BVcwrVpfKXdfd4NVODFV21SBJCW\\\\\\\/om\\\\\\\/GnfmqYgpj5WNJ7iaRKenfLWCUb\\\\\\\/5qB7MXiMR4FcEYFAfwG6CcIGAIz660bCpeyMjiMMywgTz1V7OG5+I\\\\\\\/4rRPEiJj7H3QrN9ucatYatDqev9hiJUKsTKJ1CZXXh2Cb2eCURjMqvCvwapBWmROCh1YsUk6ag+fxWWCpZ2q\\\\\\\/Pd210tRtSb9I81s+vjX4s8Sn5XZ4ChpZDarWd\\\\\\\/EgWDkCTTMJlvTCwN\\\\\\\/\\\\\\\/7SF6kbT4cN0K8QHOYoJ3eZV5NaLwq9CcAW22rSjP7T2z5LRuC\\\\\\\/yeIhh6d8reWM\\\\\\\/yQVS7NcBcW+9qGveGdVa+\\\\\\\/WJf6t2rIT+YuSisfgeI14aofPNHTWu9Wa0ODyir42V3hSG5MQiakW5GrBvmLpepzh1c6SPgTiJs822\\\\\\\/iw8pYgVTtujuKL8UHNo2FDZ0kvlhXEtsp1jqadcLHsk720JxWaRD92pGWdQRmUmu0xy4df5ej\\\\\\\/x0mytcPsNhqT7lWngh5yxbJUHP\\\\\\\/q4REVrd0GzSEmqSVrSV3WiIxN71AS3m39m6cCfBzcnh7x4zwPL7Escvi80uKCRwOujWU73DW36NB9qQiaudug4UEBkd5xp2JNDDKalDDQfR8G69effUYpwjYDTY+Dso\\\\\\\/0GwWTPA7jEZ99l+Uba16oZiCfvHAesbCU07up+MKZKqq9XSepdNDvBTtkWgWeB5M5QeZl4BfwfkF\\\\\\\/GvlMum8SGG+rdKCJh7NZ6Z2JrxLGXgmOzccv7mmwC9knORyLxl7tKsofa22l+y0t4F\\\\\\\/STrqmnbi6S3Jo8XP966XMC0BGJ05AFXZ5DW\\\\\\\/IA\\\\\\\/bMOVYXkPwPUYUR8p2PY99YD4jo42M\\\\\\\/DhsaB22q\\\\\\\/OgSVhx6\\\\\\\/RD7kAkPL\\\\\\\/ERVCE+aexYFXmcuZ2O5z8Rh+bLPbDkLnrb2peZrpdSAYlX7mjeiaE4zL3JZHzrpNFod668\\\\\\\/kZuX14h9apjqLL6buf0rV312dT9pb84rbXSHkKMRb707JnEmXrlERiHxDvORWuPZpmfTcvC6p5BCTRdTuaDn+89Y1W1YfaJqvksa0x36\\\\\\\/XtTZ02lDMQJUuM58XI1qpbA0\\\\\\\/QZcAGRmiNzOrzqebLoKIXESN+DRGj49wLAfQcCnUcVcw0nsCBUOHWMy\\\\\\\/u9kToZFz4XtqHLlFQNCvrHFiaGTRxPAXLo0YHzX9rvdiOP5NnbM\\\\\\\/JRLXrTIyuRv3SDhECtpJEHvbFaY0d1zh26qoxoE1WYdneEbX7fk0boks19VLYvjH+OtnHqlTfofAkWEcG4rHVCLS70rVlr6ZU3UbjpWqjkZdo1ZIwvEs7hIsM160r88CwEWWXyCYay0PxKkiiVFoTwNzWBacpp5eCxlaBydfK656UcGoGqDMiS71kjDJYq\\\\\\\/XRbo+u2eCaQ4BK0WfSYS7n9N9vBML7R76AFxFUUQlnkQiWgQ2YaFJHNJMRrc6lWUsTSr2ctrSNp1iksitttchI9qygJK5xJ8fhu382vx7E8+CzOagqyGa1VbvfHAO\\\\\\\/hP8Q4J5lqlWrrKLHHFzSh2j3qUWlcx67J2PvacIwL6TDeZ9cvjY+6nMztItCk7n\\\\\\\/E7itGgvxRK3sJ4+v2uhdS5TR61tZ4KU4rfOKqxvhg+hLjzUZVwPgRX+n2Vm08X3AIrnFMDjGYXZYI4iW+GHn2N5Jq6rVANboKZls5TOdW81swUEXDVPQI0O+jmiVQnEl7HErfP8RIvjyDHeqe9pjkw2sXgEBrynKCyP2Ln\\\\\\\/NW24K8rADHtk6qUoJn6n2oT4BoK6VwkRIoA2rJmhY\\\\\\\/Q1oQAwgR4KYNmyxnluIM1tVvIrQi\\\\\\\/ynAzAhvmjA0jFJXLweJqOXYqczGG0Q2G\\\\\\\/mY0aiTXSPjz\\\\\\\/dwcBl+BoTzaxnAHO6Q4UQNsLEY9mQEJeyqRCezIlc08uP5h1GRN0Gz6fREOiGFmJ+E751K64wBS0jWG2QCu+pRM+iC1ix4865p9Gg7VA2MaYsNfZdCx5SldDD9daZAg4zyYl0TCEtVVRN2e4d1oaaQnOudHua5DiXlGyluzv4uaYTGSuBwMMYFZjCqwb23F0VZ+km2n0bJDkPcimXKZF+DwFahplm9hrsCWY64SntkSMMYH7h37A3rvKjEbww9IT9Et5p3tpfc1VdBxqhL5ng9cnz5I0dkqjtDugDF2X3c7RVA8KvDoh4UULU59Vj96Xo0aMsItOAY\\\\\\\/i8Wym3DriC2Bb5UtZQYrgRdwfjlc4PGadoPBNd2YHQpdneO1dYvaW9TJBQ1DgihbtqGtd5sEowyw+bGW5hnpu0U+J7Ons+6Iu0n8eUq8mXJV3IxXYMg+CnNc7VerBYeM1jKHvrIQuWSSdEWRKOKtzTtgCAVsPUQIsPFuymIYAX13CP7bDs+w7jVPVcTL9WR\\\\\\\/IsG5xW62CYYBRN\\\\\\\/cocUbFYVTkSF4LW8eiEPo1Ut0DMsulBVGi9qmPP9+LzTxySG9A3F8CKCQpKjEJOz0JBBVxRMNsin55ztrgpe5FwwI6PDsFTFaT5l45\\\\\\\/NyIDjgMgY7DmQlwSdOYfXSPAjncMwvHu6cOz\\\\\\\/RK20txwWThMNHcFnezaUI0zCo1CDlhRgLtlb5gbylmeFJ+5haCrhXkQ\\\\\\\/pH3Ur8EXKroTQrPaX452WrjEczMpydsY1eDlWV8sfDSK62SjslOBc+y3lUhbA64AbaqydKohYfWUs7XH1ewVatnNvHqATfAynNII6aBBNywldO35TlVykOktHaQ96kOU1fADryB\\\\\\\/vp8eROipF6sNVeI56hM9Ts371TIVm29H8eih5PDA8jBbAIdDE3zKmwmzOC\\\\\\\/hC3KIua9CDYExH+47HBJU8hOlfg4a6kr4pQVKwni8aHodZvGie1e67MiyWgHwczU22AnddZyfIqZ\\\\\\\/DbfcILVOYt574K25\\\\\\\/hrotLZi1EARpxCILhtGlmZNU54q+0iRKqxttiTc2tNfsVmJBaG\\\\\\\/uKvpjoroWGakmI\\\\\\\/ZpYkmUyA4sAl\\\\\\\/PGC3PqpAAXi4eT4ijDeRWSyLulK4eW750E75GSDzrBAU+KWRnBcTwfvKfhmCtlKxJ71tBTMJQHlnQrez7YFoChtu7qKlZnFX1TNxTu+PwcOfY2726MkhcuyROYs60BrrR\\\\\\\/y5zNZtVaRiAjDUFXhj5uTYxmsa7L9PKqkZSk0iT5vFn3AcTSgR2MMurV7EuMrz6\\\\\\\/KkwQEubinxtxoIL5MEe2\\\\\\\/JS1hzY9NWKuYiokN2BYgRzp01bUD6PnFMr9t1X2rPVLTzML+gcKiFa8ctL\\\\\\\/79inQrMPCJBMl62PTl1q7d9vLgnwxyQ55ZwB8jEdDrEvizOfOL6CRWtp8fsTEK1o0\\\\\\\/bF6xNetXnjepmRacICMdCvYzbMRUS9h0eIFs4SxA5YgeUgGSt+OtpRcQKqjS8qHxuqrPD8hk4EV4XuXFSawh\\\\\\\/mZzn7WEL6pwSpUn3cs5wfhijwYPbqmmcA9cAGcWT7mrrpuOjuGj8y6Vi0K97cnP8tdVdjGLylQJVn9EhFWmQge\\\\\\\/fksv1h90KgSgymNXi9qwpDt2t14ZHesP+lxqQsNUfTHxagrKgTJJFbQLP0iaN1wzbVmq1uAKRRMeJpbV2H4KUJxXFsqZKYaIA2g5IMikI9oJ24nPQ7N4kic\\\\\\\/nUgOgIwrEYh3i74h733FCGtkBnm+BDX2+ZgrHoRI20RvEOF4VXY34ybFY3v0mwK1OHLqNFdr99hzZUvJLWhw7Vg78fR2\\\\\\\/CDozBVo1jl72zdGQGYkd0TnOqlNv4TQDJELPihKnzIITM5WrNF1anFmGh3tbNv0zZxvWo\\\\\\\/5C+hsvsf\\\\\\\/Vawtd8KSQoDsMjwymUNqzHrMQQVwp6ItwOl8ra\\\\\\\/RcJIWUgNvwJvMCUkx7ycRtrnSIbz9tDla2tDd8B98IKHtW4mvyzj5fTNulBXtIS4CRqUmyKF0NjUKdFUxJbdYG0gUd69xBxyxF0+JGAyf+dTYqSM5qdYZONvXFF84NZxMGGWgmUljrW0BnsgBgISt8MLfpER6t6oWIKVsqGQ0nidtwQjEsBQdSLcjU\\\\\\\/x6wkdXzXfITH3bcvomtjs1ojWoY6A0l1VSvq0ZymrnhmwbJLPtUGhZ+YkfshZ8RUkhnvItO7cVE7VkGNMm7txXPIdYY9hS2YRhlRZ2HZZbE38tL8ZGhIyHPgY9ht9IruIlZlMhi5tHG0o9Wq3WtnY0k2E0xCShtqCzVDMHe2CRgxw65WQY1r03eYEYr9dX76fcwDJfKcxkydLcdMXOsxxPGpurnGF6qd8jYMI65xxB2O5Bz8znm\\\\\\\/zQWyfoW2ZwZy2LofdeceK00d9WNvmHNUSVa4uU87hhbmbGGqQoSnv8E4RW6iSb1BYEIxBx8R5Yws+vfl9xC7ltYNRruNKZjEBqMxPG3QSXUtki1pJh5MtiHvenAu9P1UxxBBX\\\\\\\/Ok+eotXexXdBDQ7V1p9+FB8mzDggvG9dBvNA7nXgtWp+z0lz5hJokcLNMccuYM6iDJD8n3ZVWLFFiDdZc1BpnSzocRnRHYK829WwelDFoFvqmD\\\\\\\/bYDAp1PWilK8L7Lp9pXndghXkXpov8dDJCMDgpVLnajxH1xgPlWo6vAGEYLCiPSlanW6hFN4Z9KxzM+kgkDZtJ72FdnDqS7DTmBFuv4kdtQr0AyJyFswDME5O5FSS5rEfpU2E7787t6efgp3rDO4USby4I\\\\\\\/Ad9DbmYo1Cnaibfc0J1jvUMfKZUf7cjum+vDxtaLIRr7BpdREAeNqBxeQUfZwTDWFDcgZ1Iw6gVphJf9X6\\\\\\\/qO5kEF3DeO0IVE0WjZMO55euNEmuYoXAeKhw8eBd2Z42KAMj7nftvAV7nGmXpEHqxgJ3WfId2MMiBudhYyqldJg7ko1hpnjyScclapbvfSwVUfKqRl+Ww8+LBzVw3ynzVMqC+ZTGqmYjViSfpyxFhFGGQ+ha1YQ2dT5WYarMkRMhnOBmCN0D13J6hXf4qpVsur5b3isI8YYoiAw5Xcvwn7\\\\\\\/0mMs6+ra3tYsKx8\\\\\\\/uiBy8J9oNnviGyEsr2K6Bxu8CiViqWc0zEehyZgcaQVViFQWkL8+KOKsiFArtJbVE+FGKki\\\\\\\/mL5TZWP6fMH0TBLAP1F26L6XufwzHyjwu8E+xsomrOxNIKH1\\\\\\\/ReUHecXi7\\\\\\\/GtQAgJOsvLKWhS\\\\\\\/nsSOsPq+PB+h8SjOfQrDLW36Y6t4DruR279LIe5ZfhYakamY9iteukw4HpcBXUtEfsXrZIie0LxyYg2hPVwmftli+Fm2Y2lrziToPH\\\\\\\/e\\\\\\\/nHNeexvodTqfc+7f6V2o3Vnk5eLQnP4JckmFM5TlkNtdQMNMy9b9DMnKnu29vKGUWe5EqQ\\\\\\\/2C\\\\\\\/Y6rH82uKTICWe99pOyHpNaqXPxRzj6j78j7GMeLpbJAbsPA1I0OQzrv8l+jg55wyl2zCfu\\\\\\\/dr1+fjq4DCz9jwZJ3GuUK0RGQDuPwoqxvtdM9BMwbLJ2TgLqMSWD1UOe6z\\\\\\\/VY5rE0L\\\\\\\/myDb\\\\\\\/RwRJ8n\\\\\\\/5N7b2hfPRUDOwEYVEA5sn6LcU735imt5rCG52u\\\\\\\/+t5vXeJyFntR0GGRSIMJErwBXeV7l6Q+TrIIGtwSDA88ISbKsuBim2n2HzkN2tSMisFK9UK+90HoxTjqrDXsayf9v3IWWTHBAxJaTR2FMIhR0auIzK3DLtXcZq9j5DBnuaJI477F\\\\\\\/YS6pl5peudD9egM2llsCGTRjUXq47e9HWKqDUX6TTho2H5qHoybSRavMWg6snuStyKmQxBwMS\\\\\\\/HkcSNkg1VGbp1hN2YOh9oTgyMtZE9ozadSgOSdMYNNkGT7\\\\\\\/ue4A0I9IrwAL0ldLcteKivTLzIHVZYAVv61FBWHeG0TnUi14R9iR8fYB1iBWt6eUmFFTEXUuQqBv2NbV5z8oS\\\\\\\/2wlWcmdFJFdikgiY8ComRLJmowqLbB4VYbFjOwi7ZyNuoCOmB0Vszo2yoBjPiN3ydWc8TcgLesJmlSfZgdzaOcEo2Q5N+r5wLtokHFfu5nteX1Y17rNHXOsLLef7aT20SJQm\\\\\\\/Wr2FBEmFUDFT9XCCq8Vux6BTPeeyMj6BguX3xIqm1Xlwb5Rfh7dHAKaXNHos+ogMJf9TgPh8pTOmQixFOAdBCqyYOrWyZ9jX5vTgeoIVNeG0p6r\\\\\\\/qp9eAl8C9clfEBAzPqH3iB8JGEkdSRssRh0w0dEnQD+XWDeMKetRq96nlYeyUh+agHwiv17sSXXGq2BToWrHDn4iu15KnXvdC6EVxdWs1O5hpBvwsuFDs5\\\\\\\/tQWUQCW21f\\\\\\\/EH0oRhR2YYznTXGS5UmVeQrR+iBjJ39d77lCqnSaCa6DTJCNr8f4VDNgpHBbytPkNACNmZnBfHtbeZLlvYCxybsgPAf0E9rp7bXTEuxMKdhkbVfrf2WIBjRL+JO1FbxbX6plhDMJ8wDbEnOx++U0dJRlKRQGQNPOzpA6jQsdnKtNMaLChrj980VT+VnUbF4jZoj04DJ1mQHxsR7xQ38\\\\\\\/h1z5OA6G3tAdYulRMsvM2GRrRJTM\\\\\\\/1+q1rPWr59G14IiZlsEeqcPhza\\\\\\\/X6lS0IJx3H5o8JtQdVK8EAWagJnIo+ymQRNXOAgbn3uz28NWG+Hv8YAx6HXy7G9aiqtr1lCYDg+IC4lF3BThDvCAgWBH7mPbUzZlafC\\\\\\\/VmV17LE1iiowXHXkykbal5e2a3guaLzL\\\\\\\/ni0m0htO8xp4B4PSzRdXPUa7wdCdJrMYoJEEDpR9sFwnEI7mBLPPR3mStyXsWKZXaU2PsNjWD54zq3PkQCeNtXuGnaE+gwqs1u\\\\\\\/bk8pB\\\\\\\/5BanbeqxmXp2vDWG1Mhkh1xMj1x+z9OTjirk1k6aGsrakJ+vRF5da20uJQgMxWdW\\\\\\\/lPkRTd8ms6kZ1CnhSIHmvIeHeJtzoz1O\\\\\\\/Zate6KG9Zc0ZjrL0XFzWiRO6LyFWwmhG9TIbdwoK9iQi8uEuuAEZhCFLO6N7WlnNMoQ0z1J7WQONk0sfU8CfSkuwWXS3qGTpuUcIjvYG+YI2S5bSQI3Z58RAFHxWmrZ3a2hIn1LPlvGcZ2eVYodkmuqJInoazjim8LOCaIzFIb6196IQ12MVhs3GgAnCxEyb\\\\\\\/gr+mDwtvEFSwEAZ0yweIHz+jvpnyxQ+SdsUo\\\\\\\/sQCT9wkStGUdG8UIhWmFZDNhs3UeP2ZZZSeb0CiUv62PZkmQ3tHTu4wHjrx+A3WZXK8XRldDKQB0wV+Qg2CY\\\\\\\/1JEN65mlCeBaJLbdg7O+DQTixvNzTmSTZSPBhXZkrVCdYOFMRWyY2vUe7iNxrZmHjmPhi8QaGKj0lK3UooBEu+dqq7rsfkHKlMuzxSfJw38aqu0aCnnRhtWIC6utCX7d3dM6uDDwj41l2oQMAwgW0V3JoeDWajzN5WgkWjS6NhCpiNF9egJQXWzsKT\\\\\\\/NMNjH6d\\\\\\\/qjh8xz1pQR30rJbjxZ\\\\\\\/9DE4k4rVgA8PsX26s8Q4aQwOpPal8R559gQWovf4jJkn2CDEWU74Zc6mb4p6d0eKggu0TJvCRka8ct6roEvVFCQ8SENwfLZky5A8xroEaT5\\\\\\\/9\\\\\\\/NqSy6bPu+7ri+G19uylilu\\\\\\\/MPO1nh7r9cigiGZhVQJyLDxH1gKWCJCSBPZ4RD0pV+0pSo3xhjmi1SQmbexmd2+x2z\\\\\\\/2qmhv0rvK2dGAmdvopClxPjkZ0L7530g8WY2xCnRaiGunJDNS3jnZ6fVcUI1g8FV5fcB2ddKEXd3QhE4MRmk3SjquljPiQxQbNZ9MQQ9eAMi7eAGmcZpDZev5BuvmW6XnwOZ8rX5p0KB+ZtSBn5zHMfJ5TElMAMtgZvzj8ReuiEf9jWfmdtct+Y9XHvBAiyK7IDZuVm7ix5cN4UCFVJUr46iXQ\\\\\\\/GKwwM4PMltsSVCjKRBZ+wpEy9iwQpZaVTAHIYgh8rdXrxXCU4JVOHSFmnBA2fgsL4NO6L1IkAbISdJNLCfqU2eKhmQmVBgK7gsbAL8jx+8NHD7kwfy\\\\\\\/\\\\\\\/iuAoiLFIl3rvDTSizR2q4nM6emPXdlcTjfzhjo3xMOmGMMeg\\\\\\\/KAIcFh5xYU10QT\\\\\\\/Owmrl68SWbGKxoHpHUW9SOCBAAbGjdbRrr9yP6gs0LLQx4wlTlBZYE\\\\\\\/sVLOmMgvFE7rmJ5k9tshqG4odY13+dP43w5StQZUT7nxtImQD+X906WexVZSR+ZvndkgA\\\\\\\/7dBZ07HTBY+KokpISfkSrs7VjAubgjpWVJn44goWeCuokFSCjIo5h2JN5BRig7vVZz8JrZNsEV87lX32Sf0GXRh375CkBiVduVczdgXG2ZonoFXxT+qA6qYI1rM\\\\\\\/9ogfPttN8K41Np6c+Efk6qsOqU6bMVtXS7aS2fKwPzzaZNoNxToDtVLq38OQOr6GGIMvhcHIT2VzaHstxRA+S8tcQW2kp+4xClYiWWzm3NEQuQcZ3lMPrzOLB7Wg76TMH\\\\\\\/buti+PrhRgWiYT\\\\\\\/9k+ldcvPnsjVMzgYMYDjoudU1C3wAxrfJezx9Yvz0FlZp\\\\\\\/16lOmJJJGetiRQFKB0P3TbQAILmxnmlPgrIVhRTksalFZqi7p4yTH+YSPhR5RMMEuz7gd65\\\\\\\/7iTE2bMClxYcOQBjOdgPEPWlsS+xa4xLgB4L5Ny9TfnSuVS0xO3w5yjVyXzY3sU276EfCUM2wdSuT4Jomz5R3+WRuSWKV5ZCUVs9MwmejHWZHcja\\\\\\\/Q2NGRTpajxi1dDzVPLXO\\\\\\\/MB7ZhoUxlMU9MdQGeBVPddcAIIMLwm0+IxL2dZP32dAWUnLx2p+VD7DtrKgiJT29sIKXDPyS31O4SewNed+fNF1YxFhlwvv3fpr9L7xzFITcWpz9Gjeojb0HqPMJ4GAGKNMovvUSSdsL+BgHjjCqdeneRleEKgtBTmRvFNISiTBh\\\\\\\/cLNa+nH59IjbZgWpoW84aA5I8oomHxfH2ygpwlco+0iHpcLD16jizlvjRUaVFUF+DUuXC2unnGx56SExfhhj7OC1pFofDFuXeZaSEfHSmUBoZ0fGE\\\\\\\/lzikRdn46sXAbnjj7vJObQGgPEx\\\\\\\/1\\\\\\\/7lbR5rcbXpVeoS1N+0nwR+O5AdTpxkER61EZE93SzmstC7ITpfKmhOJhK22YX0enFjs+k006TwipKqp40j7XGw6xmp7gAdnPtYiPCWYTgeXFcMoSwYVTYeNiJoaZPA8NzV9uQKl+nlD8GnNx8khosyZmaiC6+bgeClkZwANI7yoH8LqHxx+a\\\\\\\/5AL\\\\\\\/9z\\\\\\\/SlRJQmx7qtaC3mh1Ze+yJFqo\\\\\\\/JpRE4n3AbBGQgPCOrJCHPn85Ir8TYhvABNJ\\\\\\\/B33ePqrpy5yHajuaTkkJmbcnS5CU8Nud7nBGh0YEw74Y9+kZRTeVJHwBB57NjUuwnqrEZaBWWwbJ3ouTsVIqXYGMXBbbKi9tUOh1s5iXmF4jGN85B32BAgNc4HR99i2y9TGG1avIVgjsvCHcREotqtRatqvx00m1j14PU0luLZcZe5CvdJV4ebdjvuTrQvwjH0vvTHbMrqLpOW4Q2ZwewqRc\\\\\\\/gsrsINm6Fk8q8rOfHZweTuTGfhG7\\\\\\\/anNvkONtvuf7k1vk4dEfw6uPnjXN3dHeF538Rio8e3Xke+\\\\\\\/5OWZn9iyaRHi+fqjnvPBxebHlpYQzTbrpNIZuUG+UMiKhT5qyzQyUNPxUrSnVH7aTiZnkXKgMG64OvNxgoanKgpagaiIvYvToKXS6vs2fuhV0zvG5Mbgki9H+KZ1LvHK2BpzBLNEkFCbSye0Cxqjch2nSthSN5Zr4wpnw+buVVvO3xuuYAFEwNn9McwmoB8C29JfKD7U\\\\\\\/HR04ejoHobwjQjeWvPoYHVMjaCTSe0nuKV93AXSo2B+1KZjdCZIi3YVUmKobNpsn+byOQ5DIknVbZsT5qY9MTSIy\\\\\\\/ZX2wubxZWruVCCVtj7hi7DjwufIopAEIov8a2Xg6AUFPL9vO28TWa2b9E2bQH\\\\\\\/eqebxvjfI8ojEuwJepho22E8KMm7Ix4kAJhoAuaoiN6AZpDN2vFHCpOyRCS+1EzazjTL8FOlP2drENnNW6ltn+uTfCn+UESorLogKaRsraD1V+cCwLbxGVuJQGNhfBr9VbvP119MhBvB6lZydTpPMgIMqySat\\\\\\\/z0VyQMQwwsJxD0axM6qPmmfHxr8mgo8WGZ1v+xHE8m99ZdbgiF047+lpLU4918NF+4IYtHXvwys5ByoEgulDtwrCdXGNB57ICihfn9Rwo8bBxGsVq46MjN96KkvjYhuq+JuGLFBxvw07HKWAXcj+DR1gGfosTUQQQ95BoL9BNQHMELSQczWQhg0JLKv1Ry0hHY8ssAXIT1u291btcmddxjf65Pw6fnMF+2MmAu2WGPNygbrch2hXEWr9\\\\\\\/+AUT0jJytoYj2LIkcri09x3vd0Oe7yWMSC1DnLKb8cOdoVAuNiwzFvx8QARJQV4yj1BcKZs4L2tWRDEbrcBwdt6EkTtNwSet\\\\\\\/xYort3vnZc65F0dPQeu7mfPm4QhkKCyHiX\\\\\\\/IOE82Y\\\\\\\/Y\\\\\\\/61X0zf3Xv5Vdh50+eqHle79W+jsBZXdGQ3g4unv3GAbO+Anm\\\\\\\/je+COBGHKsUmqb5GU3Y1\\\\\\\/ft1\\\\\\\/AK2y4IoCNJ8eIK4b3WXBpvoe8lweRxOE1rHlpMwie1QmYth72\\\\\\\/Lfo3vPKhCb+F2OT9cFAZSd0oMUVFiD2wx0DlotG1wFGRY6AXuk7bKa1fHjDx\\\\\\\/ZM9pqEHTKZsw7pn428R\\\\\\\/LvjFQsa6vK7kNNtqMtqwyQvLDl2dr8vkfoYEVCl4vRTT\\\\\\\/rbbJmD3ma976rnw3XzCI7E7kR6kH\\\\\\\/H\\\\\\\/3PaWRdLVxsuwEnFmBV8tXDQtpfoXUiY60+Vmkhv8GJVqC4cmyxqba9ofgiTEoWNC+viqBNT03l3Y+hE1zC\\\\\\\/yrVBZje4fVoW5mypx\\\\\\\/DyLODrpJkNb0vK3G2CiOzQiD3gOLQVgLU\\\\\\\/e\\\\\\\/IhOe98BYO4vo9B7FruUjNQNr5QwnFXzN1osjI3nmIndyn0RnpeQIctdmlPQ5U\\\\\\\/nnnkaBHHJV9RYVSmhTk7ZarR1OxQLdzz3ObxNK91VxkbIAfKbtyp6FVGZDGrGzAWPKx4xdwvkXNWNWvwPWIj39eqPE\\\\\\\/vTSbIUExZZddsM\\\\\\\/EliN8eoJq3NZYSBM1NMuGtUNdLJRDV\\\\\\\/Fysjb1ZNYENDH39T2yUE3Yu9sauFzzYPyaoMMqEZKnNeCRZMzfaVMwCvH+QGLrWei6yj+DGJyY1kkqyO9WdKjwvjusEbGX7GefmMsj6vfp\\\\\\\/e\\\\\\\/7o7J6m9YDnks+gGujay0Kw4CbuXf8R86huMSlK5ucFHumMYM6ccSMqkLZD3P9MwrVP5rNex0a13CDNlkJwD88cRXmcyzuxPD4vwdcUowy1gQLjkLdEXQxPvU6YFu3vz0M714z9oYnzadO3q\\\\\\\/UHT5bhv7XCwQE4eDtrlPnAUlPuqk7NcHTAlQdNyOWVI1QZlTpzeVLIv5BjZgcKqHxdLz3uW+zp24YEuFutarZcMQIUOYP9W6e2FxaUHII1jYHU8SS1fvhs6YMEt8aBGOymLBFx0QM0gULRxHVsxYVKk0D6f1W02fLfpbMJBv49cQ1+T7AV57qo6cFqlJCF5fUdesR\\\\\\\/1j0XkzhsXOHkdWBKfzB4Q\\\\\\\/40m0sjeYRArA1XjyZxF13gr\\\\\\\/3ZvkU9Ki2AcaGF6v8kFvbWJC4\\\\\\\/9GICEpwpH+ToTPdB9rv806wM2YQ6eQMZvVnDfTOGSR1WNNdwfzNCz5+GYEjniXl1A7BW7gn2+2Z+TEfS8UCPap41N0ynolhnjNSu3XX9iU5s42bia5gFmBW837f+e+HdblcCxFf8e\\\\\\\/bDq+U1zjAv0s2zLcTnlpL4ZywbtF18XOnoEv9cKcesejLe2bpyPxmthFjjF1vnTD1lP5WpZ7cQ4a9hE1M8Z9FrRwLQoVC58UytDaajmpTPTynawXBe10zk2JN4Ihbifk+FDkATO\\\\\\\/\\\\\\\/kj8po4zGsvou5wpKiRtk5J8\\\\\\\/cPkg\\\\\\\/kWhxlxVPCWVVOD73bXIAYdXk8Xeu3ZfkP0f65wl6HpJZ7XuHzLXzNSZgxnog5w5lZK8J9n6IiYBkjZx0+H5lsnasAmTUnrsC1iIzW1Z3BYlnJDgcxTr2lS4amDbwYWpX7LEUoObFHKntHZWH77ujn459N0jAgcNnjlBiyT3tjA37b+bnK5GFwqIMx5tuKVDHhQasiG3cKMLkFE1MWxC1RDtIUQ1JBtQfHFDfAV37SOku1XhY8WriQpqf9TYctrlnOL+8timnui5gF2f1nt5ZmOb4MS9Kcu1E3n+etPmCM\\\\\\\/JqLRFhpFEOF+7\\\\\\\/ImVJwQPY7J46k1k9WVQxa+UglB40XmssKlw4GSNSzui\\\\\\\/1Za1jBq3n\\\\\\\/WiJLqk59Pe3zMF4F7AGoi+ZkwQ23gfxZ8qU0qps53\\\\\\\/nwz0SDWd38BOXgR8aENwsHT2na17wIJBzPuDdkJMpvogdhIgU7Xx+CRhERLJ0K8YfY+EAJ9h1s4gRbhhDyxFvypBqqgR8uwsGKquG2IRjJ\\\\\\\/bPh3UqJ76DOPly97KzTgH1yQqUF82xgGegBBg\\\\\\\/B0J8Yz5kjgahQNVtM\\\\\\\/qTVpxEYCLgxxxdqheq6RneQhWC6neMLI9a\\\\\\\/rwtYAGPwg1Po9kG10EWKb5\\\\\\\/GbIx1mQbKiyt10qO1ROxTc89qy3XGBq85PC5eRZPxpMeGjk0nZeVk\\\\\\\/Djmus+5lqpYY+BY2HNJMz557b+bOb37+BL0CmvSx1xTeQTTp\\\\\\\/kAKugoKYXHqhOOlCCL0X9qtwq75z5XpHR0qscIN69ByUNIfHgvuY2nWCBovcBoobpcK5YRyIbiR\\\\\\\/qCO79Tg6UAB\\\\\\\/eQ\\\\\\\/XMjAzYV9cduDLyz9ppp9gEg80VnHw1CoEhqs7r8IFghPNDsnHkaXgIs4AE0kwPOvZ+WaZSPBvZqNOz2ZlUeE8JJJX334eFisqv+NVimvglhg8642X0FWqkyxmzPJHGIvlI4oqXs7NB7fl\\\\\\\/XmWGfJuaD3+Jtva2NTPw4t9SPspJl9Ey0oQYa6NHixIkXD1dCeW9y+U5v\\\\\\\/YUYpTo23N8\\\\\\\/UDLRpVT8MT+m\\\\\\\/LAGZGKwfTIpbp1jC7tymunzL3LNkXLS8FFLSbFqGZ5n2MWRmqiHG9bGA\\\\\\\/XyqUbjUiBOwpWj8HZiW\\\\\\\/0VWnlrsAMb3ebLS6MweFUVhUdEJC1H97Glb3T0wApF7Z3i3wQA49twzrynWSIEtdnKiSTClWSk6FZFbPz+SnjCW0ZCtp0jpo0Wk1FDthyi8mdeU9uFCqsleIh81yMFW9j73grgIQJnTCO2eLbf2SjDlry\\\\\\\/kYyduqhNJEuRiG0PGVV8dk3NUuFiXIZVvPMAmEV2u4Y6C+WQu74j+fFCvScTriqfgd8E4eNa091nmS6nzfkUxolh543N+PfM9qFF81fRWk+35uUZOD7LGQNhQGd0GNqKsmwEHMajf+1IrugMnI1hQLma4kCCrJ9OkjDa2b1\\\\\\\/BU8hyN5VcDoTJK7UQvQGvSlrddU4NelNy5XlUxrDTmtyHgHk8Qj27j7KkcfOJKUHfDyYjC1awIXm67VMHUeEVtQ\\\\\\\/dLSeLEHklHaPlK96uXyYq1DXFVSk0VZOrwlQHtcMSAso5hkA1\\\\\\\/HNK7OeYs8L4VODDiqBoUiLtTSSF6zEA1uG+ysQPUdUgC1mGWLc1LsvaWECokhTvUjoN2pIXDZmO79rE5D27WFJ0N+Nu4di0zpFin1UpF7P0ohlvCP2YioXQ3ZourGLj3rV8foRk+FnV7LbEYkGkmKUNMsOxM3l5uo0Hz4eYgTX1sLLAwoD3kYITnVKH9EpoWVLOO6veAkWWo3ZUqJhhAHBCiuYgItKzRohpD5ytD+oJAUU0aJF6mv3ZB7+FiX3KMYL+2i1nO+Tnm5x\\\\\\\/aanqhniGii122VaNkg00Vivil9DMJIAHm7t\\\\\\\/9W+ZEMoldZAt6b6sEkdk64FiSttFHrNsUK8oDrBbp7yN8T0i6y3S\\\\\\\/HILZL9TaZXIv4G9RCyBX+4b26G09qkw3guhvYFZL+n3T0w2IWy1qJY5UdoNFSc155CeGmZXJPzlJlff+qdghzMACuJccRa6MdyJOt4w+DLIdee+dSws+g8aT4r6PKumSd8OdGdoItRXpSMOQpmvOiu1Y6xus9SDWVlDIZQshOFim4hfpv9hFKgCiIl+98SzvvjDNUzThcmVnCdB945DgOhWZ8aRoP43dlau5Qwa2hEpKLWkyZo6oN79Dp289B4WIevPke1DKZdAbKKr9GCqvriIh42GF8xWDo24QhrhxEG0tEuJ+e\\\\\\\/N14fGkvtBLkvDgngV3X90s2sL3yiID+l2yxmld8t3oKkLgxY7GnpoPeGC93e3qqXmpD3cG+dTHd5ytR80j6a5RohTun2Q7\\\\\\\/OeSucCZ7XKMVnPYYUzj2iVNwI5DjDAJMgjQfT2PICSS2YtBNr3Jdr+g\\\\\\\/dvv2Li2UBhVLt9jiM+GBAV\\\\\\\/T\\\\\\\/Z\\\\\\\/\\\\\\\/ODRlKMR92cyOgkg9zxMX0xRcgvY89wCiVva7PKv6uim96KFralk4a2iP+iBPuRZc+UF0N40nov7j8UO+J0wZk6edRXzQCzYOEeVFvJLV3wC4pIorGTWnCW0VHj\\\\\\\/jExl03Yxxp+2Hula7jsi5HhD861\\\\\\\/VZ7a2uyRB6YqfSCSJFFs7GDCWeB+a5tYBQDrpwAWQrUQ0sNM8iCzPOg6pCnD2MsE0bWhsg4e3ccmqbZ8Oa5\\\\\\\/f1X+rl4PZjhUq0ubFw1oHFzVe0loSwtJdU2xN6d6jyOKcJ4FnKogAOvVbkDYnzvx5fXw2R3UNFKWA9IV+cRink5xvXCuyh\\\\\\\/s6rWf+jL0D6lKs0xlSLuTTF1A23wZZyOBfOTCKS1pXpwb+X640BMtZhyodXTYClSeqROORnr\\\\\\\/v3vZ3Ub86rD0Assn5bJzbhsp+L1bD2yJt8ZErqzwab9BM6ykMlly6ifRkxewSxHlpQvtZ9ZMPW06vYmRC1SFo59oRdZmeGHXgUUTM+P2zPUgiQBHy9i4YFdqhLkJy5HI6WxYu7vPoFOLAso1Jw+tOGHN9I0FoK6mrEeZLUXvw2pqLn6w4dHN84gB3qDa5ledxJ5Vujo9\\\\\\\/XiAh5OgCqjkiU07oYlTTcHYbj\\\\\\\/N02yvxzre8+jW9lal3+HE4HshNU1yHLNPYCkmXI3QkU7jlH1I\\\\\\\/ZA2ia8J\\\\\\\/P8hftw+Mv3hUziWZEp7o6zoaQybjNKNVxzibbPi\\\\\\\/MXDICVmIn+0eBrOkOGuklDoSnxINj3ItUXGvfqfkULa5MQOqtL5JGXN9BhVAwg7BedF7eQCLMlucUVRMV6d0dV0nHkFfT\\\\\\\/scBkmUOH93nm4nSR5HuWum4AIINL9iB4kiqF3d4t7DnLDXqzHWQlz4SuSeUEPuVb9mBnHoInrSEwoKhbRMKL5H9p9swDiJuFok\\\\\\\/Sq5WV0OzLI1UDqWWfSna\\\\\\\/\\\\\\\/YugO2nCbfkffegFAkd7NwdHUVUrgmmEuPKRcS+thHCLJuNpN6PN2UA8mq0u2cg\\\\\\\/ZEVSvV72vCGn30l4IpjzKUgabalNDVUbZ6KzCmJXBIC1+4EbEONGiPKnBYHIfAAmoyYdlvwqVszLpzWKNCls8dwqdFMPnOWWKc4lTZ2LynEqN1agbVKx1+CxdDFUHuxIIWPfJ2WyhNP9uXdxUQdhrBzeC\\\\\\\/hkBU\\\\\\\/gOFFhlnCWzzVXHJJXxOJoD60v4GnQoDdzH9CY8pMVx8DkPMgHJGMZSoU3qWExPK9VgYYldy430\\\\\\\/iKea7MeetiMVddJGbwr5g+PR7JkAK1RR4W9u+1k473CnwWf5NC04XN\\\\\\\/Uqy1k5gnY5HyEwt1x8IY3l0WyynOGQt+GiDrW1m2NQiVXedbO5WavvxYEiy3fZyVUuik2XbMmMpcgvJZCE8UHR8Kx4KVZ63o7G4fDB+cYXJCoZeRORKTL8U7s8pgaZS+Og8vGYoK40I9BBLfnUaXoJrpWguR3gn+HOc5\\\\\\\/hglSZaEWgYX8scWojBEEXtaBUcVEqWvK0CK9Nc+iLjG9MyolX6i44OnZ5onmD3XdzJvzsmTNbsAP5kFui4pDw3gji9fv0Akvh0IHutZ5BS\\\\\\\/1rDZ5Cz3Xm+SDK7DMZveUE220GrxhIIUqU05S3DyXJVoa\\\\\\\/MjzeAfzjw7SV1ELEe\\\\\\\/jO64KYt4uGbdFFqY3Q\\\\\\\/8gTSErWMKiar7HEz9ywTjP4osoZxQeGY7KuP3FcgqYmuh0YTZ8bN4LarmOZDsBiHzRVTCowFeK\\\\\\\/EWiVtrXR5Crgu1+d8zRljnFJoBsQ5WJcY2mj9F\\\\\\\/2kRk38T2A2OYIH6pPsS+xFoVsl9IfwENxNfQSv+wN94LACCARg8GiH\\\\\\\/SYdlcQJtW9s29ty7aKDFK\\\\\\\/V5xgImi87CWw9Ygm94p2ON6hNGxosQL7E06ViKyTcNnXRM017yoVtb1HjdekPMHqLEszs4K5xy4CpC6zjC1vZU9Revjm50xScARJgccatCp1t0ZJJkGPo0C0zsWorcV18MwXQIPc+lfYjtqUuXVp\\\\\\\/lnDDXRR\\\\\\\/TkRJItr5axcmTxaRgdcijeZiPtGOV47ROlJ3vldKlCio2yJm4XchoJXTQh9ilwEPkKi\\\\\\\/KuP1QmVjOHSn\\\\\\\/hKXpOKYj9HsCBSXhmQDU2QFVECUXxQBeqazUvN1HtjvYGRF1lVckunJibBFhPA3DOShLFK4qvbG35mlwDuq0isl+my9BFFdJ8nKdjUqDmxug9R\\\\\\\/UFPT5bGroXMIo5Cn15JYScJH3uA27gXiSBiSQiRwHjN8uaMnx+vR+IggWI\\\\\\\/HyT0Dewn0QWi0\\\\\\\/h4MfV6oSTZ+I69XLhR3uOLHUhwpPCgctIVBEEQBdT9cFOZpz48KAQ3zu13OJl3Yxu\\\\\\\/7sW2S2UhNsJulyHbDvp5BVI8tuNFdgSPfkclN7DXU5Q9F5gF0VAS5S8gbwv7bGQUD607VeRgSI4LYY4gvlLvzabKylD\\\\\\\/p5lkpZ0Pbd8\\\\\\\/j8yhX8JXcKhOQGKVnIU+n7NJCc3crCDmc+D3\\\\\\\/41TIbk5+gzk87rl40B8ENFSXxLmiFfmmQCeTHn5wKsQHgBOn38voP66x8SoZwoe8\\\\\\\/ln0FQAmzLmIJpZW4suRkDq8zFdZJy3iC5A0hjuyBftSj8VHIPV733aWq1hoxiilW+0gOnVjcihoCA85vOVNAdEeYvnZrfsuUSNbErTCCgwQap5o+CLuytpCz5CHqzRYST3osJbTU9OXJprZH6WKq8voOqo6YdUencdrC3UOl3yd752HQgofIGkDLJotn1NN3oeSmNbSoN\\\\\\\/xQ5Y\\\\\\\/5fdx3qJvaXlgNlfWIOI1TP9GDsAiltsz\\\\\\\/krzTPNU2cHqMfI6PVJu0nKiUzZNY\\\\\\\/PrkRD2R8j9n9c\\\\\\\/XN9phJzfGw9+VFJs9gSpQ+\\\\\\\/QIjy6vLt\\\\\\\/RV6qvLw\\\\\\\/aCP3ImrvlmLeC0CS9pO8ml5S4Tt6ETNWJjr0t4GMyp6t8NzeXObqggYF\\\\\\\/Dcp+z9rPXfScFEpL0mSs0b4\\\\\\\/1EvNjzn29Fw9A1mzFJotyTC2+BwzKwjVca8AtxaBv13sNH0E9+gJGrUahC\\\\\\\/au9sdKWeKZveM30TkNqKujpDnBLOf7uq3i\\\\\\\/fa9zkmD1kz15ye2D44H0cGg5zQeWnBFLWHORemnEmu\\\\\\\/L57pV6HP6JPSb17\\\\\\\/EQFG4Eos3mwuoOhrI6P9dUrwsiEkazGVr0R7LLlUdaFmPFd2SsdBrx16E3awoWYG0gnPXwtzzct1M4mMr3zxSpD3Ml3aThCoT6eca7f3dapxMQS8dbJwueWxOYXWmxnQnF57mOfCZj3K6FX8qK5RauHvcEiIPoP4e1KmNc96X\\\\\\\/GeMsnESOS5kRwUV1r3QnDn0aMSjgAWa1Zvz7moEvkRUgpq7PPPipwz77u0sGzbW+tDDly+bEPBGWPQg7\\\\\\\/Bw8pRlbhHN2+P7L7Pz31zii26eIQN5LJyuZUvf09sOJMAoDDrJjSmjc8JE9LSFar2wH5na3+vzp2eIeWKa0uO+U5QCPtU0G83+Q57RNoxTQYTcbtMvIszupXgcQcRB7\\\\\\\/qElYrLYd2abiVwQ9hVd+qcoo+lgg+kwEIoO+h82eJvyr0iq4x6EaYgDDi2BIGySbAFZX1H0OUBBEyJ2rCDCchGYBxeq9xt0Wq2x4tdAXG8FoAcM2Rr39iDRo0lmlR0nsHHj73uIQp\\\\\\\/PASzc5Ow6PZ8Xosc8PiDbKT4JcmdosWLkWU4Si6G0oC\\\\\\\/LXuX5H\\\\\\\/GioSPfDLpkXnb\\\\\\\/UauPzRKGqdSyEx4ifGY\\\\\\\/pnitxEmC8WtQmGCTFJnQTuMGeiMWPyzvVFS7Y5S1pi775ATytnVL8CO4OoRwBZRM70vmkTyJ3DOMXA8PUD1oDqIk5DWd5aDWZbxL6\\\\\\\/iLreGmzKeWJQAhUKpmlijpjm0m6imY9m3mEcg4GY7W2rdWUwHeOD3zuPM+ZLfzUf49ClSj9JCH9txTF9O5SkMCTI47+GSc+gM9HEbLOu0ssZMHvVGkgBn78C805fMOEJSQ0nnux9YFgagL3iPPWrDfEQGUGp7Lw9xyz52MUC6X2Ga2Htmu\\\\\\\/q6PuM1nSxNCp83BA6xnrPaDrMUja9+NQ1h4YVNl9t5vMXwcA6R+N6nuhcZX\\\\\\\/u1LNSqulyhgixy3zG2Y65YNetpDdSpUHUAP3cGhxjmS3iyFF7NaAehCNtlE1k5WphvItNTwO1eMm2AOTaM694\\\\\\\/AcQPJTYKWGUsTfprCHUnOozTMnLGT37OQAgMVygHZSorVVPA7luT6x7Rx3URe4XZQ1WWID8GFCu0FiKz\\\\\\\/UabNTIdtKgkE7b61LJp9JGC7UoMXlBKJuRSinYaaLuv8PaxJKnzqpbO\\\\\\\/GZZDXyCJNU0wiRhAQlrN8saxct3Bi5gzZjbZLqmbZRTMl9UkUwjQ4XimUgRzCyRvGwfn1gpz00Dm3KrsGsvw57\\\\\\\/0R2B4QwQ0CYvvJ4Xus1TYR7KPw2N\\\\\\\/fClzhs2cHA66XNqIQyY\\\\\\\/XGJvlAtKesxkLy5Pxb6Jbt7g5xkRHPVTScfmPAgDcsHALL1CmQBj6CMxaFqffSu77pC2C1JaOuDlsEJTMYkjvAJ6\\\\\\\/cDtX12Q7SNxEindxgmqLS91QFV697zS75Pc1P5eo4H3rMvOAPhFgpnYNy0k3E2nm4rKpq8U3fvRUzsTX5eTJzCU+g\\\\\\\/M\\\\\\\/EHfeCOe\\\\\\\/1b3AynnVSW+WWjnXXhmrDwDEvlrf\\\\\\\/34V20LNTHLYmn9ofzs522oCN42epyQ8iNes5T4bqmc45jhFzBVlmiOkUYnH0niR0XoOxYs4bXGhqKEKZSWQhZjpvhEuKeEHjPL1JggMrGHGigcnqsKLtBNe3wYQRalU5x4QVeaMaf6VcglXj9tdUIBAbgxiBFLDjXAp3CWjxB9AM+4UPQ+sJ+JCbrluLJwQTjYQMGnjR39iVpUD\\\\\\\/JD6jtXUmMRWnH3bhZE+GWCubSKA7pVXY3Eh4ta83LxJfj6JO9UUz7WjpWfcyLC0\\\\\\\/JDLs9wjQ2A\\\\\\\/d8v1pQHM5RRglTwjSA\\\\\\\/tl+qCwB\\\\\\\/iWImBdq8utQG\\\\\\\/HJzTrrcQ7qKeKoDDaLoXRPboruvj1FIVnfXQU8sdwF7zyI8ZsyjVKAZSehMDej6EpjQlZ9ImjKnVdBKxpCL3uJ5N7dm7FWMeYEdisJXCFDLmbyVbSZhV4MoW7GcR3JR0ltgyCE7yrELjtuGIy\\\\\\\/Wx9IB5pEoVPBv\\\\\\\/CTNsqpquNbjy9XAatcUB2J+m1HRYoeJISJStIcd9wiTcggcfUjoHgMRkd61LwUokViwyYyrGn0yVzAnENozWLnCpDEop1Vh4rKC1Ktez1v7h+psc5khMUOeqNn0L3Zsdk1kEdQJSiKphyuxGzbzsJa3yAXFe08BKPlZZqn97s0EVqU0vvFc1o6nlxFvhG5Weccky\\\\\\\/XSq7XDT+lLP+aI3WxEf10t+yaACBu1\\\\\\\/mPYdCXc6OqpMd+xM7chSZGSvfNiZMItI3csKHwJRzfR1qmgwbB5tyIPyIfLydbwKXrE1U95MopJ2HAK2lWuflL2nVmB\\\\\\\/ACrLhI5lfF4FMfjkDUsVlj+9EqkIEIrx\\\\\\\/vLQhDxcWY9cwv+76yB4PH+eCH6iHRrvGtvut1OIZIoLipLC+VcrKFieddwG+Zhbp91uQLC+8f3IyZmBuGSVEb6phUrjeAgiTVuawdFnkq1JQNxSRLxNmV81O8ldi7oGUc1XW2WYXjYRd68MZYmO47f4fhL9nGfm\\\\\\\/k1xKItowKCdTr8qSFjVSdR\\\\\\\/7Yedkg8sPzYg4mzipGaurQfQnmVfX254AUA7apyer93e7Xb+nIT2pHiRRQItAIpAGNLF4KyjHgp6wtdbE1N68MJyApmC17fbsz4nQc\\\\\\\/JhXz5Puq7+fEFhQZv6LtsBGgzlmM4VL6MSppqHjTw7yzZhRwIweWUCQl4CGrN2JeYT6SPg1HMsDDaemjNgBQrWQtt8zTau2bdfZ27Xk5lnCBkkg+Ns1aVWpnTpm5tOEjdVW8gQ85OSP281EZonRuezT5xOfHiFw2pyxAvmYg03c37dM3RIGM\\\\\\\/r7mqpYgvWkiq4wu4azoMLdya5TPd7G9cB9UpyTF5IbxanH1OO5eBtE88ibqT+mahgJXuK9YgXJc+Z\\\\\\\/3Um\\\\\\\/3waOJiguuu+2eySoBNVQkngvoGRbLlb\\\\\\\/eRI6Do7oPQ\\\\\\\/nWrJTPyUs0ADxFAk1x\\\\\\\/q9ZFd5trFZOhGI94bzQgfnA12zc45Bpyd82EXKnfPiymmSMBgHoTSHtTp5m+INI9k\\\\\\\/0vN9PI8BLk5Knz+ft2Owb79bxMv6LWPxGlReXryIpPOvEbyCgyvOijMx8gXz7LASZMeai7Msq8W8lVu+RoG3HuL0PEXibW4xmoP06ssigBM6+xKmntJKdoupmxGs7SBYW6cWLCvDLMu14fGDIlAgbH0IGGkrJn1mVCvu7biuLHDHMAcKxzJm6Uh2r\\\\\\\/3\\\\\\\/1WS4ofirHUvfWwC\\\\\\\/XhxPoVNl2pIq3sWrhoDPic4YG\\\\\\\/gTpLZYUK35lt\\\\\\\/\\\\\\\/nTxd7Y4ttAxwFpTVKbtTz1airafeRJSbHTh4pKggC6lSFgHWqM7isGedqmTx17wYPKjxsF951wPd7RwggJB+XBvigu3hQykvqbqwBfiIRT+\\\\\\\/tyQsMQbLFdG+70Q144Nn41lepkRRMAevx1fvA3FReIj7Vwzhmp4ngRAB3PUMnr7fb4mM6DMESYylBzjsQ9mdaGDZdLFH0WRLedewJEjh0dJaT4OqjTxl7XV6x\\\\\\\/8J7P3dvWmUn+7VY3349djccvi3P8PyEumDXvCBSOUniBqVFs8u55FMr1\\\\\\\/yy1FrV79ps6GT72E0LewoBI1\\\\\\\/UVx2MUlUBkAaisqHl9Dwl+rgOIzl1K4QaHlrIMQ3oY4WoVUxUcDP+61PBQlWQCDhNnEx8RiVgBcTFu7OMWbtFD8pVFPeQSGKJL+GRl4os3h76jPoJ2Ce4Y3Lop\\\\\\\/ETc+cLX\\\\\\\/VS1\\\\\\\/5QtcjJ7sIhiruY10De1VzE4QFn5lRKMGex1gxsxd06mfRX\\\\\\\/TwzqyLPGDo47I5oiMC+xejxZxxb10amwJm3bncnQLLnAoqzHLp7ZZ5Yn1sCkYi8Es\\\\\\\/uZ57OUSMHY0RmV8xKKoXd7mmW5QCawTWVXU4PqTibbAf9STVXN6UB7+LSZXCtgj9cbzpPyXNl4cTmiat3sqOMiYxdgYo1SoYjBBfyGbK3xWRGlS3tIyQW1HtrN7hCbatAmJ8Zo92qeUIaB\\\\\\\/mFYhOJLQxuO61ZqgvWk+nRcC4sf+FxY2ACy2MqlABiWXdkv76f4aRTRARkhXCP\\\\\\\/vkZ5Y04bvwMAw8qCsMfTv27xBpNfnc9EfyBit035ydpucN2NVE9cyNT1AQwpPTKHAYZJ496r74ULIRBMCNCkv9tKtPW7vQg6WNZcz9H0OLPwdHMkGYYhwbtyqJ43\\\\\\\/bh5u+KywwRg\\\\\\\/ysprvnp+8kELJIpuqsxKnCbEnIJsGgO3oEMSoDKFbTJCq+89ms3nVHp2\\\\\\\/7yhjzkUUce7OIhVR7MaFbMgBduETZfC4ef62MA8qATCkDfkRlbnu2tw7ZByJRwKKIax14TL5i7uERuphZ5kxCy\\\\\\\/BUQjeFuVU9BTOy0U6UQPHiBQPwRgVZbuMfljuW0\\\\\\\/Sx4+dKPNLyx3OYY61ri0TtuRslPmTY4GN6nWluaEW6oXv6tJm8lA21RnogGdX402pNV0+2jddfmuXGLvwn9qSXRYs0xFurC0Zudo2yqCzBK4NvahGRtrAAH3B2L0CvCWR0k+o2h9kItnhjncUtoK\\\\\\\/JbKABgnuhx7qqaDQM5b9qlWVVIsqVpstAg3yOZu04Eel5lvQBzsIpn5wQZkSA4ir7zWTwU57x66vX0N4GSuLFWFX0EenPE+J5KAJg29Jng9C4\\\\\\\/khVRXKIoOK2MPUk65DjjEu1lxQ\\\\\\\/woaaG5Xge6rpRnbQsrxdOtzr7vUO4uVqDm8C+9v45nKmOpfeqOTQ3pvJAzymE5lcUfKekx2gs+f6PjBNOqTCVY44ROvNVpMRoF1b3Jnhip0Nn+GR1u5ryboI2u3OOmN9lKwHvefHDvhljFt6Rr3NoyT6QSvSjSNKf21YuxcnIOBBxAjwBPxiCyLeIFW1QAzaZg1e+8X6kLkE5J1aT8z+Mlhp9TvrmrkaLSEnIQ3h30eFaxtDVjV4n0JHK1jVxz2b7w9fNEFlFxBFe8Yg8ufoqwl2+GOaf6RB\\\\\\\/WQrmtRBT9VNyjhJTYTn9+yKAgjbQf+phKPB0l+q1H7RNMG\\\\\\\/wFzdTHfBcZmybTt1a45O2m0wMQV4qMAXIeXlmNfcOn\\\\\\\/P+8QM4F9CcmkyK+Er3QaP78WTlAPkGaK+OKBoy1QL6tWOOCoO1G6zRBoKgwD\\\\\\\/qv4iW\\\\\\\/pZrxm80FQnQmXnEhH7xOm4AHBqOfy4P5nAly2fvrLA+TokmSaW1ogYYkVVi5AK5y7HttzLdvgGMAVX8mz5kc+i\\\\\\\/+cOeUEvJYkHPpP7AyMknGAFkmkdvIG8Lus4ADeafkLbjP3av8IzJlwX2YHXc0CavE1L6A6EkuwEpfi0LURvhZevc4m\\\\\\\/btXMXQkZZ3R+FHhyikZwCKXP4Bj6qYMDgyM3ZneAMl9HWX9GWGnRrDhmGGrNuBbIUgpL1q\\\\\\\/7WTGaB0T2OIdC96q5TBTuQBQBuUcOrK8e3M0r17vwvknPaSdtX+wYVb8M6THDR+Nv8r26v\\\\\\\/AfjmT0dCIdXMFLOJDPv\\\\\\\/buYCH9Fsf1fz35ZV11EHfOja+xp7A1nbceYM9SkeVPcO\\\\\\\/nxumdV4HN\\\\\\\/7SKpIkd7MLDpvvxsJlO\\\\\\\/aa80hgCpEMlj29X1w2uzEOdCswKBoChVRRXIsx4fRYlkU2h7STUJDeJy21sQGHIxcPlpryWpIU9kSO+DsuSXCI+pHaphxir+m+PzH3sYbs8tMhoSkabJVqK1Y8Wgb2maMS6ctFNyv3vz2pdi6czJ6vRczVzVoL06q9\\\\\\\/o5xfNRKTuTJwiqRlBN47LhhcORKl\\\\\\\/7o1wy8m\\\\\\\/n2IhmbGQBdilu+VGHgW20epZx9tMLs7TOELSn8AO51nQMWo2PHBmqIMs\\\\\\\/LS6ibVW4inbQcQp+wO65L+RUzsQeitkHYUYYNhPfO+XlwOJRKPU85e29SSX6aPYcnAdYUz\\\\\\\/N9fUOaM8+Z4Q6kY\\\\\\\/Rm3XC8sXIfJHbaT6DvVY3ZDSoTtT0LMVNB+pNzwnkzlsCbDBV0E4UuvNkjY+idEIpBAQSgzQ62WCZzCE2c98xxplYlVN4+Caqa32O\\\\\\\/+V0KxOH5DrtdfsGAYP6BUiNUV\\\\\\\/1UsLGJJJbWg1Cf4jNioVpJs71d3Y6+ngTkA42c2uWM8eDHRwfRjYi8MZ1W8cIERz741ihS0Ppi\\\\\\\/o5Q\\\\\\\/Os3RtbHxHWVh8BmDt6UsBR0Bro\\\\\\\/kEAC9YnRpmq5RZwTEKZU5wJm\\\\\\\/anmUa4\\\\\\\/L6A00MMADzPcvDOhdSTZhbTX6cD+suQOUf7OcLI4oLQTFM2MS3Uv9P9dim\\\\\\\/zbxWQ3TyK5QvhKM1OR3Iv+PJRgLoRStxnn4OZACAI5xIuUexsoJzZGQhahVoapGF\\\\\\\/GJFLLiPSD9kZmU09O5Ut1B6jxQE\\\\\\\/EfHvTyif+0sKFAlJKmPctrqXUQHRjpta\\\\\\\/08oCK03a3mGL+\\\\\\\/s4dS2nRI9AMCG7C0JsKOiCCmL3mOZEa\\\\\\\/njAx9UdlqvajV3vcEcfIvFNgsJl6L8fh\\\\\\\/aMxWffH4ubZ5bWoPozTD\\\\\\\/fUo8Dufh3XjDWFmjZSifX2vljBTtP9fGRM\\\\\\\/c4jkA\\\\\\\/7pqLlxQm87GUyX84rOE\\\\\\\/i2tLjuD4A61DPfLkJHya2py0iJ5IDUnkImsxhS8vKakROerYMHE2wFN6cwuriqRedev2nkwTxB3+ObVZBDoGU3DNIAyBLye\\\\\\\/m\\\\\\\/KCLbtWTxWdSMRdpjUSobEtYa3NGruqBA0gyuqjbdE\\\\\\\/\\\\\\\/q9HMhvytvGkrBtqWVnYkdMARuKy\\\\\\\/AIwIor04eK137fbYc6gFRA7BjpTPOjuhEj7+y+u3kMyyvSOZgFtxMgSMlZVbqZOWMGEa6DBwWYijW6ObfaQTPaXCgWFoFVq3sY9Cw\\\\\\\/B66umc2ZusOP3LsyBCErTWjwNS0RP\\\\\\\/77Nsb8fbVsYJuiWMxPAr02PcMYpEyYWwWJDkRZCXVa7IB2jqXnaluv\\\\\\\/Y+55ZUFbaFvU+\\\\\\\/qvs403ZNV76l\\\\\\\/qdIDkclIPKbvncbNmyduPwmMpzpY5z9hQWCVy9Cz\\\\\\\/CbFCsXJVvuzyQU3qPN4zcs03gJF1cqNPpCLAQM5i8Z42wfdtZXMK9H9y21YGFhqU4cPF0T7i1upCrsbB2GupZMWYl6qWJ88MIVdjrrepN0IzlJ6pHPGw68jwL47jjcqr44tuhHegT+fKgLRYx9oB0P3m1xyR1efpcz0118mHCmUvprBtL0mWKwTxEORT5MKJSGAcqSpEJPMriaJtENsshb3tdtL5ziX03cpkpi8KAFbKbR7xGFixGYM7Jq4zpG0TS54YCRX1u64gQXtoMif8\\\\\\\/wTFgQjhebvqIqeJ\\\\\\\/G6R0tvog6zrdjDCFqLRe0thLOnowUjUJm9vGpGe2vV1HgZI8VU\\\\\\\/MjDpnMcN5ydcVcGnaeQUWZDo6ZDhUjrHbcjJRHucF9EcL6N9wz6o3r0HDvXd1QeGssVEaxgdLQGIFlndbbDxHUflbAfYifMhzSMwlxzVatHheME0EH+QLLRFxqMPl+VzfpuhSTjfsYeC+KdnzVsrN9sXs4ISDyp8\\\\\\\/Nk5kquG0MJPvHMcYjOP3fjFtQOui3TNmhDjHkzQM0pLByXMxwVWgY7IEnr7AF1JCJjzKZlt\\\\\\\/Xh8+XXHjlUr6krTE+ee\\\\\\\/9W8BXYyNuO7arK9jaruzeu9yua1nATWhVLjmglo90u4fK+V0pSHUnrbvOV9G1\\\\\\\/zHdbwJ0qgX6iikBe9k9fLeZ8r8R8WeEY90UGbVEchUD9eaLfh95IJqXx+z3oJUVeMWeYCzrkuVApKxEc6d6X8\\\\\\\/\\\\\\\/K7SK8P0sPf4Ks7cAO4nG0OW7XMvlXTflKWLudX7RUf2YEXl1NikPTTRqIXdyCAnewBBFRTqnnwehyEnLZX4mAiJqZ\\\\\\\/lNbeJX1IOUhPBbywznyfa2P58Pv\\\\\\\/b7pQJI1pXvqtRyZBYX\\\\\\\/XQhGnTIgn1IwDinGZcAqRtoWWB8hoS5mFu67LpAaSfEqRudJ2u3gXm598+Y7vxg0YO6ONXE79rJD\\\\\\\/ZzjE1zW7nFf5l0zhlZ5hTfT+5l6Ij2NJOADK9A7V3FNP7Fck\\\\\\\/ftKRzp+3VLBNveyNy1PQR62yfhTDxJAA1RCpVT0Na7HU4oqZWsgRd575OEkl4ZvvHMIBhAvgQokzBMPcAkUFuFdk3D8H1QdJykWKO1EDPLRbux7kkYoFnCOXOgMVjJOd8GzocaZ6ZYEa8vB+LTCl+z63wdXkku7hrgUL5Jz1Cx2TNo4lDpf\\\\\\\/tOCE726ITIYI8juFzsut27LamokexzBa1xXKW5LLWimUI8z+Rm\\\\\\\/AjgagK+1b4RwBdCwfLETIek8tnbQbC6MKMXlXWXIG2wBdXmxqRYOuHChF1Um9ByZVHYD2PvsniZagEZDtLzgUHqzEQ7GVl5l7O\\\\\\\/6rRq\\\\\\\/tfac3djZ+5uls3TaJ3QnbQ3hWyaPKmgc4AFUH5tqtZH6+AgRJBxQTZpe0Io3eQuAylYd7pk2QFcyA1eZZd9VUM2nT2jXNglw\\\\\\\/VzbD996vt7Ed8p4Sbm5j\\\\\\\/s4DzUf7webbeTC4NcJDLPKHjZ9k4YdIT8XO7tR\\\\\\\/rqQqALbrUFCRmUj+8wQBtRrjGSIEikaJz09H8ilnGlBYXD9XIX1nHO+l1PG1Fh8CkLUi+C58G7zV5Fl5butDm3rWrFB\\\\\\\/bdD5lX2AVDeAkWB3oYYjHB073jXvPnDR1SpNBQDU78PQWhy\\\\\\\/diDfZU0f11VI9QNBae6iBuHOmPzWWTmNV7SkzZjzYrzYtMwmJCmEzD8HjRijceFeInMcA1G1NPCqqd791sXZBbJCFX3zBB4fixAmYgcefiUz2N7iLcGS7CkUewNIA8YXYhsOLe0qJxOK5XQC2232mgpZiqXba1LganjRnK12AE+sg6wOHxag6KRabZZr48gT+nnOfsUtYb+XMN9XuRpZori0UY5dk+xfQ3rCYHUOU8jGYFkQzvORmDcwhfsrD\\\\\\\/wEvBJSBouRCcJhpt2YJM1O6hvybGUDghVwH+QXO8klvzdbL3ydJ9x73Zst75p+dSdAOQnwiRiJlO7PtlKcY7chvAWfROC+io0AlxqeshtaaZlXQmx33ikVpVFOlBlsvfDonIdpGrknCDfnmm7Sw6b6mFc4ye2wpMCzCUn3QAwPIKixETBU8csGowVy3iY8FEdVO4herTjui4h6tPKxN+tdFxygAcU1vXsKLSu0JsQBvR8LfTWp6jQcrd4xCXVJWr8jNstGjRDPSL7v92LWIvLNje9Nt63JixRgHQUHI5RZZ1DyZDY++D9h5QkMeoVQoM2Biyj+LA5ehdLcfYHMHSNtpr9iMvRDHR5wPuc3lQ7LDqK++AgNYG5Zfdn2i+QrjLxzEDT5LgsUKq0IeKim2rGBkOb+CV8w5yVx0EAenFoivJkHhKoBsJTKUigjUo6EBa+S4D6EmqEuxBf9qPSTJk+kWre8V41qL9EgfZ+VTU8Aw+Y9IjLaiWaqW8Vnx3j2dtlMTw3UfpnZUG361z\\\\\\\/EXhYsCOliZFxIWDUpo99I8WG\\\\\\\/l0ka6UuCh2Eha06UxHUXw\\\\\\\/F7\\\\\\\/muydAzM2dQqkBpCAHsUn2negPYjYA8AnMxzek4rp5AidAjLGX8WPQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/hUhdxKIBnSt5bY29vIdCg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b356f71cbbc3cfb01ef31fff940feb2a412a2cd642e8101a83cd34311effaa65\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022d6YkBCqsCmPfL\\\\\\\/qcwV\\\\\\\/qbtf8oF7pV039iwRISaATOQmlyb9mUzmJASMMaDsi843wzWSeKoGsOX1hrC54FDD9YqKei4uaN5XfpdtyuXeeyJfZ40AqvCaljhN38HEglUAZS7xSBrDD7PHs6Gn1xR7rImuQNnSCbck8yzAC9X+wE4vChGpk7HikHwQZx6dqqrQ0Vwrkf0W43wCVjF7faY8azCb4Jted32Ntm8WSYmK244aXek6MqeS75diHUrBUHu+zGshJb123JRMlIvNjdVMdhgK3KBIq6f5uKMHtvrIOj0IeLc6MRKqwSq1pLU9NBOH47fufPKc6JdTO3uF9Lczuq8rcmvFemP0afsw9tyxyIPnlRmvX9JOp9ERlW77UCgLM8KCeRbQYCMO0qwmLjwYfN1XCjWc06Lqk5mJWpqci1hTGzjfJgzm\\\\\\\/0AiTBJ70MSW2BPy7Gcqu9sKMAynMe5aKJGulFV+arIkVlJmaG8pOh5\\\\\\\/066XN9kKlgGYpM5iDFr7\\\\\\\/qq6TJ+rY4qtUJcHU+711ZPrs6IGEV08Chm34iIbn7DwJOt6OU7Be6wpsIjvUN2JYaMtziMa0DwLKD1GkSOEqP8IAtmaf2j0sJDphqXXFuNuEZSmDs9Ab0ZBWLv29cz+9oZsTwzyDDrRp9XCeCh59IHpwiRbpCQ\\\\\\\/VEbHvttJHKysDMA3bnx1+ZL9VxABY1LS0vH7Zt9G+s9eFXaj3ibAxqO6HQ21Z55eIeansxiNoo\\\\\\\/nzqny3h+9R4UZnFC5W0J+Gt7x+ZvJc01sF5Huma5t\\\\\\\/xgdRMNfyTprvyVdCoUUMoBkoXy1S\\\\\\\/SH91A0pkVKcKXb3YOvmQL8D8sJjfecN5FzVoDl16BZzyW2qgQ1azW5Ddh2Q7Q1UYGjZf7UtzuZPedtQM66Kk2DhW7tZueC1ZaMR7WA7fOOpGA3z1PXVaaB13wvndpVV615+AeOErVS\\\\\\\/CNTbpZusiRo1FE1M7AoUBHgtxoisGVR6HI7nk8bQN6pkldOiy84S5cwlk2dbqpBeJ8wt2WYwghIxc4wR4+Uz0fi3qR\\\\\\\/Fv4DRRIrVqY9K2LTymTnBnaWnD7ErTzlaGDPbg1kPRqL+9d7VjCVXXSdNI9LOj8eAuYsAPJLZfJtMjXKcmwUzWf4oIRKkgCL7BgSSHj2GycZLUTfGc44bCG9EWlKOtWScHdU8bNPHQv7t1MQ4zmdPuTTT2ufoVlC0zZHRz4ZTDyjfmM93S61pyv9fD4pHk4Dq+lqkgPbDQZPz0WLF+qW\\\\\\\/Aplg01KSbjl9xndAuO7ch1KMuoFhXZo0Zrc+iiPEX4VOuYAsKYKkWIEMyHJxaLTEqaJudZ\\\\\\\/dXTJ2D\\\\\\\/r+s1wQZYNhZG7DAxwOqdkCqFlLRAOGunua6viZ6Oc\\\\\\\/tfzopiLIgojWNJyxXseqRvx2jrbJxGjAM7ax4XMfY43oIwrqfGiSRVicqzGBVBPfluynEZm0VUZfY5\\\\\\\/Ay2v8Hn\\\\\\\/g0CrvAV1lqtwuxX1Ixyo3Ka8MdN9j5x6AjbO3PRXeyRh7ZK5kbAyCF45cAf04Cydt2kL6+x\\\\\\\/WAgD8TppNOQaSwOprA\\\\\\\/1jzHJAKLOOiBeoN6jD7vpvbsk1c\\\\\\\/2Hi1lfPQwrU5mpS4kSImuobDYZNlOeFEGLlkkST7KN6bkHXT1paIrNTcRyx59YQU3TG0hCafF2k8kfjsb1sAQO9uIWFUipS28Ouzo8NTfeqC6Er9Milmu\\\\\\\/3qnRrMaHRf9uFfhaPmIi7stPKPUz+2\\\\\\\/5QA7yfNW\\\\\\\/x3wY5SKX1iLbWd0x7Hk1f2EviITTR25hUuS347dEcjiqy3ZzePnThVuADwhyjDQXpLQpDqXfn4EwD8yEvwdYCpca7BFOQAnaFFkztgScaF7ayYXMBDvLJiueG7ZdrdUZABUSwy4sX2i9j\\\\\\\/PolXfeQyt1c1F6QW1pksEWzCAddAZIlwuTLrxvobtd80c8RDQoFPxo+smJzpfCbIQ33ebR6yo0BL5GgSGCh4CtJiPD00pXMDq\\\\\\\/maVVoNDB6QUZclxndoRDrg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022F1AEKXW8nfQTFNM8oInBmQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a3e64faed4156335a601f7d7c1175e3392d3bf717e3f013c68ff3c9cffab5b18\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459968 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1273" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-VAkc0vavno_SejDsoSdCYN3RDVTSOmcHgR67uZVvrQg" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1273" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ZXE0N5sitnfZ2o-eK-xa2nvyKS1dGWq1Y9ywFrUDhR4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/6/1263" title="Neurospora Heterokaryons with Complementary Duplications and Deficiencies in Their Constituent Nuclei Provide an Approach to Identify Nucleus-Limited Genes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/6/1283" title="A Multifunctional Mutagenesis System for Analysis of Gene Function in Zebrafish" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459968" id="node459968" data-pisa="ggg;5/6/1273" data-pisa-master="ggg;g3.115.017822" data-apath="/ggg/5/6/1273.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Dynamic Mobile DNA Family in the Yeast Mitochondrial Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Baojun Wu</span> and <span class="highwire-citation-author" data-delta="1">Weilong Hao</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1273-1282;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017822 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459968" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Baojun Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, Wayne State University, Detroit, Michigan 48202</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Baojun%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABaojun%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Weilong Hao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, Wayne State University, Detroit, Michigan 48202</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Weilong%2BHao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hao%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWeilong%2BHao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:haow@wayne.edu" class="" data-icon-position="" data-hide-link-title="0">haow@wayne.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/6/1273" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459968" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459968/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1273.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459968" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459968/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1273.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459968" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459968/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/6/1273.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459968" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459968/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/6/1273.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459968" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459968/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Transposable elements (TEs) are an important factor shaping eukaryotic genomes. Although a significant body of research has been conducted on the abundance of TEs in nuclear genomes, TEs in mitochondrial genomes remain elusive. In this study, we successfully assembled 28 complete yeast mitochondrial genomes and took advantage of the power of population genomics to determine mobile DNAs and their propensity. We have observed compelling evidence of GC clusters propagating within the mitochondrial genome and being horizontally transferred between species. These mitochondrial TEs experience rapid diversification by nucleotide substitution and, more importantly, undergo dynamic merger and shuffling to form new TEs. Given the hyper mobile and transformable nature of mitochondrial TEs, our findings open the door to a deeper understanding of eukaryotic mitochondrial genome evolution and the origin of nonautonomous TEs.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/gc-cluster" class="hw-term hw-article-keyword hw-article-keyword-gc-cluster" rel="nofollow">GC cluster</a></li><li class="kwd"><a href="/keyword/horizontal-transfer" class="hw-term hw-article-keyword hw-article-keyword-horizontal-transfer" rel="nofollow">horizontal transfer</a></li><li class="kwd"><a href="/keyword/mitochondrial-genome" class="hw-term hw-article-keyword hw-article-keyword-mitochondrial-genome" rel="nofollow">mitochondrial genome</a></li><li class="kwd"><a href="/keyword/transposable-element" class="hw-term hw-article-keyword hw-article-keyword-transposable-element" rel="nofollow">transposable element</a></li><li class="kwd"><a href="/keyword/palindromic-sequence" class="hw-term hw-article-keyword hw-article-keyword-palindromic-sequence" rel="nofollow">palindromic sequence</a></li></ul><p id="p-3">Transposable elements (TEs) are widely distributed among eukaryotic nuclear genomes and are a major contributor to genomic variation (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Collier and Largaespada 2007</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Deragon <em>et al.</em> 2008</a>). In contrast to our detailed understanding of nuclear TEs, TEs in the organelle genomes are less well-characterized. Several TE-like sequences have been reported in plant and yeast mitochondrial (mt)DNA; however, all these TE-like sequences are fragments of TEs from the nuclear genome that have arisen from nuclear-derived insertions (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Knoop <em>et al.</em> 1996</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alverson <em>et al.</em> 2010</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Rodriguez-Moreno <em>et al.</em> 2011</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Mularoni <em>et al.</em> 2012</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Islam <em>et al.</em> 2013</a>).</p><p id="p-4">Another kind of mitochondrial TE-like elements are palindromic GC clusters, which are characterized by their relatively high GC content and palindromic structure (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Yin <em>et al.</em> 1981</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">de Zamaroczy and Bernardi 1986</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Weiller <em>et al.</em> 1989</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Paquin <em>et al.</em> 2000</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Smith and Lee 2008</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Erpenbeck <em>et al.</em> 2009</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Smith and Lee 2009</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lavrov 2010</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lang <em>et al.</em> 2014</a>). These GC clusters have been proposed to be TE-like based primarily on their sporadic distribution (<a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Weiller <em>et al.</em> 1989</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Nakazono <em>et al.</em> 1994</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Koll <em>et al.</em> 1996</a>; <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Paquin <em>et al.</em> 2000</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Lavrov 2010</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Lang <em>et al.</em> 2014</a>). Consistent with the hypothesis of GC clusters being TE-like, other marks of TEs including copy number variation and putative target-site duplication have been observed in GC clusters (<a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">Weiller <em>et al.</em> 1989</a>; <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Lang <em>et al.</em> 2014</a>).</p><p id="p-5">Although mobility of GC clusters has been suggested in previous studies, there is a lack of compelling evidence convincingly demonstrating that GC clusters are <em>bona fide</em> mitochondrial TEs . The evolutionary fate of GC clusters is ultimately determined by their intraspecific variation and population genetics processes, and the fast growing population genomics data emerge as excellent resources for a better understanding of the nature of GC clusters. In this study, we assembled 28 complete mitochondrial genomes from <em>Saccharomyces cerevisiae</em> and <em>S</em><em>. paradoxus</em>, and compared their GC clusters together with five other published <em>S</em><em>. cerevisiae</em> and <em>S. paradoxus</em> mitochondrial genomes. Our results reveal that one 42-nucleotide palindromic GC cluster (GC42) is of rapid proliferation in <em>S</em><em>. cerevisiae</em> and is involved in homologous-recombination–mediated genetic exchange between <em>S. cerevisiae</em> and <em>S. paradoxus</em>. GC42 and other GC clusters have highly dynamic evolutionary trajectories featuring rapid nucleotide substitutions, dynamic merger, and shuffling of GC-cluster units. Possible transposition mechanisms and evolutionary/functional consequences of GC clusters are discussed.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strains and mitochondrial genome assembly</h3><p id="p-6">The raw Illumina sequencing reads from <em>S. cerevisiae</em> and <em>S. paradoxus</em> were obtained from the NCBI SRA database (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bergstrom <em>et al.</em> 2014</a>). The reads were assembled using a combination of software: SOAPdenovo (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Luo <em>et al.</em> 2012</a>), SPAdes (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bankevich <em>et al.</em> 2012</a>), Velvet (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Zerbino and Birney 2008</a>), and Consed (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gordon and Green 2013</a>). Five different K-mers (21, 33, 55, 77, and 89) were used during the assembly processes and default settings were chosen for all remaining parameters. The best-assembled contigs were used to fill gaps using SSPACE (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Boetzer <em>et al.</em> 2011</a>) and GapFiller (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Boetzer and Pirovano 2012</a>). The assembled genomes were evaluated by mapping back the raw reads using BWA (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Li and Durbin 2009</a>). We have successfully completed mitochondrial genomes for 14 <em>S. cerevisiae</em> strains and 14 <em>S. paradoxus</em> strains (GenBank accession numbers <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KP712778&amp;atom=%2Fggg%2F5%2F6%2F1273.atom">KP712778</a>–<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KP712805&amp;atom=%2Fggg%2F5%2F6%2F1273.atom">KP712805</a>). The complete mitochondrial genomes of <em>S. cerevisiae</em> S288c, YJM789, YJM993, No7, and <em>S. paradoxus</em> CBS432 were obtained from the GenBank database. All 33 complete mitochondrial genomes are subject to further analysis.</p></div><div id="sec-3" class="subsection"><h3>Detection and analysis of GC cluster repeats</h3><p id="p-7">Dispersed repeats in the <em>S. cerevisiae</em> S288c mitochondrial genome were identified using RepeatFinder implemented in the UGENE package (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Okonechnikov <em>et al.</em> 2012</a>) with a minimum length of 30 bp. Sequences with GC content more than 30% were grouped by <em>BLASTClust</em> (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1997</a>) with 70% length coverage and 100% sequence identity. The flanking sequences were manually inspected to define GC cluster boundaries. The four most abundant GC-rich dispersed repeats in the <em>S. cerevisiae</em> S288c genome are shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. For convenience, the most abundant GC cluster was named after its length, <em>i.e.</em>, GC42 is a GC cluster 42 bp in length. The secondary structures of GC clusters were predicted by Mfold 4.6 with default parameters (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Zuker 2003</a>). The nucleotide variation of GC42 copies was visualized using WebLogo (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Crooks <em>et al.</em> 2004</a>). To identify homologs of GC clusters, BLASTN searches (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Camacho <em>et al.</em> 2009</a>) were performed and significant matches were required to have 90% sequence identity and 95% length coverage.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466162/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466162/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">The four most abundant dispersed repeats and number of identical copies in the <em>Saccharomyces cerevisiae</em> reference genome S288c</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Phylogenetic and phylogenomic analysis</h3><p id="p-9">After mining the nuclear genomic data (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Liti <em>et al.</em> 2009</a>), 630 nuclear-encoded single-copy genes were universally present in all 18 <em>S. cerevisiae</em> and 15 <em>S. paradoxus</em> strains, and all 630 genes were included in further phylogenomic analysis. Each gene was aligned individually using MUSCLE (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Edgar 2004</a>). The concatenated sequences of all gene alignments were used to reconstruct the phylogenetic relationship of these strains. Phylogenetic trees were constructed using the RAxML program (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Stamatakis 2006</a>) under a GTR + Γ substitution model, and 100 bootstrap iterations were performed. The phylogenetic relationship was constructed for homologous flanking regions using 100 nt upstream and 100 nt downstream of the target GC42 homologous position. In <em>S. paradoxus</em>, the same homologous position can have standing-alone GC42 in some strains and merged GC cluster in other strains. We have manually inspected the sequence alignment of the homologous flanking ±100-bp regions.</p></div><div id="sec-5" class="subsection"><h3>Transcriptome data analysis and GC42 expression</h3><p id="p-10">The pair-ended (100PE) raw RNA-seq data of two strains, GCDA5 and GCDA8, isogenic to wild-type <em>S. cerevisiae</em> S288c (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Turk <em>et al.</em> 2013</a>) were obtained from the NCBI SRA database (SRR900186 and SRR900220 for GCDA8; SRR900222 and SRR900223 for GCDA5). This dataset is ideal for the examination of GC42 expression, because the sequencing libraries were prepared with no DSN treatment, no polyA selection, no ribosomal or tRNA subtraction, and no size selection. This study is not concerned about exon/intron junctions that are only present in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007260">cox1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007270">cob</a></em>, and 21S rRNA genes, and BWA (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Li and Durbin 2009</a>) has been shown to be superior to other mapping programs on RNA-seq reads 100 bp in length (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Lindner and Friedel 2012</a>). We chose BWA with default settings to directly map RNA-seq reads onto the reference mitochondrial genome. The expression levels for individual GC42 copies and intron-lacking protein coding genes were calculated as RPKM with Artemis v16.0.0 (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Carver <em>et al.</em> 2012</a>). The RPKMs were normalized to have identical <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007268">atp6</a></em> RPKMs (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1273/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1273/embed/mml-math-1.gif"/></noscript></span></span>, <em>x</em> is the <em>atp6</em> RPKM in GCDA8, <em>y</em> is the <em>atp6</em> RPKM in GCDA5) between the GCDA5 and GCDA8 strains.</p></div><div id="sec-6" class="subsection"><h3>Quantification of the GC42 turnover rates</h3><p id="p-11">The distribution of GC42 was mapped on the phylogenetic tree. Gain and loss at homologous sites were modeled as a two-state continuous-time Markov process, with states 0 (absence) and 1 (presence) on a phylogeny using the tree branch length as a relative time scale in the R package DiscML (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kim and Hao 2014</a>). The turnover rate is expressed as the number of gains/losses per site per nucleotide substitution (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hao and Golding 2006</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Wu and Hao 2014</a>). The GC42 turnover rates were estimated using the simplistic (one-parameter) model by constraining the gain and loss rates to be the same and the two-parameter model separating the gain and loss rates. The ancestral state for each GC42 homologous position was estimated using BayesTraits (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Pagel <em>et al.</em> 2004</a>).</p><p id="p-12">The polymorphic level of GC42 presence/absence was compared against those in five nuclear-encoded transposons (Ty1–Ty5) in <em>S. cerevisiae</em>. The phylogenetic distribution of each Ty transposon was obtained from (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Carr <em>et al.</em> 2012</a>). The pairwise difference was calculated as <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1273/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/6/1273/embed/mml-math-2.gif"/></noscript></span></span>; sites with missing information were excluded from each pairwise comparison. The choice of performing pairwise comparison was made because the Ty presence/absence data, unlike the GC42 distribution data, contained missing information for most homologous positions and were unsuitable for reliable turnover rate estimation.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Rapid turnover of GC42</h3><p id="p-13">TEs are often of a high copy number in the genome. We initiated a search for dispersed repeats with high GC content in the reference <em>S. cerevisiae</em> S288c mitochondrial genome. The four most abundant GC clusters range from 5 to 25 identical copies in the <em>S. cerevisiae</em> S288c mitochondrial genome (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). These identified GC clusters are all flanked by short direct repeats (AG or ACT) in a manner similar to target-site duplication in class II TEs (DNA transposons). To obtain a detailed picture of the mobility of GC clusters, the most redundant 42-nucleotide GC cluster (GC42) was chosen for further comprehensive analysis. Given the fact that TEs are subject to degeneration (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Carr <em>et al.</em> 2012</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bleykasten-Grosshans <em>et al.</em> 2013</a>), the search criterion for GC-cluster homologs was relaxed to 90% sequence identity and 95% match in length to the query (see <em>Materials and Methods</em>).</p><p id="p-14">In <em>S. cerevisiae</em>, we identified 89 GC42 positions, with five present in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007270">cob</a></em> introns, two in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007260">cox1</a></em> introns, and the remaining 82 GC42 positions at intergenic regions. Consistent with previous studies (<a id="xref-ref-63-4" class="xref-bibr" href="#ref-63">Weiller <em>et al.</em> 1989</a>), GC42 is sporadically distributed among the conspecific strains in <em>S. cerevisiae</em> (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). To further demonstrate whether the sporadic GC42 distribution resulted from multiple independent losses or from GC42’s own mobility, we sought a quantitative approach to measure the rates of GC42 gain and loss. If the sporadic GC42 distribution resulted from multiple independent losses, one should expect a negligibly low rate of GC42 gain but a substantially high rate of GC42 loss. If the sporadic GC42 distribution resulted from GC42’s own mobility, both the rates of GC42 gain and loss are expected to be high. The overall rate (± SE) of GC42 gain in <em>S. cerevisiae</em> was estimated to be 135.2 ± 8.0 gains per site per nucleotide substitution (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, see <em>Materials and Methods</em> for detailed explanation). That is, that GC42 gain takes place at a rate approximately two orders of magnitude higher than nucleotide substitution. The rate of GC42 loss was estimated to be 235.2 ± 14.0 (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). The high rates of GC42 turnover support the hypothesis that GC42 is of high mobility. Furthermore, GC42 appears to be more presence/absence polymorphic than all five nuclear-encoded Ty transposons (Ty1–Ty5) in <em>S. cerevisiae</em> (<a href="http://www.g3journal.org/content/suppl/2015/04/20/g3.115.017822.DC1/017822SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/04/20/g3.115.017822.DC1/FigureS1.pdf">Figure S1</a>), which is consistent with high GC42 mobility. Unfortunately, reliable turnover rate estimation could not be performed on the Ty data, because they contain missing information at most identified Ty positions due to the relatively low sequence coverage in the nuclear genomes.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1273/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of GC42 homologs among 18 S. cerevisiae strains. For each homologous position, the likely ancestral state was estimated and shown in a pie chart. The pie charts on the top row favor an ancestral state of absence, whereas the pie charts on the second row favor an ancestral state of presence. Five closely related pairs of genomes were subject to detailed analysis of GC42 density in Figure 2; each pair is shown in a distinct color. The arrow refers to a GC42 homologous position that is shared between S. cerevisiae and S. paradoxus and involved in phylogenetic analysis for horizontal transfer in Figure 4." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2109141736" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of GC42 homologs among 18 &lt;em&gt;S. cerevisiae&lt;/em&gt; strains. For each homologous position, the likely ancestral state was estimated and shown in a pie chart. The pie charts on the top row favor an ancestral state of absence, whereas the pie charts on the second row favor an ancestral state of presence. Five closely related pairs of genomes were subject to detailed analysis of GC42 density in Figure 2; each pair is shown in a distinct color. The arrow refers to a GC42 homologous position that is shared between &lt;em&gt;S. cerevisiae&lt;/em&gt; and &lt;em&gt;S. paradoxus&lt;/em&gt; and involved in phylogenetic analysis for horizontal transfer in Figure 4.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1273/F1.medium.gif" width="440" height="103"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/6/1273/F1.medium.gif" width="440" height="103"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1273/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1273/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466347" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-15" class="first-child">Distribution of GC42 homologs among 18 <em>S. cerevisiae</em> strains. For each homologous position, the likely ancestral state was estimated and shown in a pie chart. The pie charts on the top row favor an ancestral state of absence, whereas the pie charts on the second row favor an ancestral state of presence. Five closely related pairs of genomes were subject to detailed analysis of GC42 density in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>; each pair is shown in a distinct color. The arrow refers to a GC42 homologous position that is shared between <em>S. cerevisiae</em> and <em>S. paradoxus</em> and involved in phylogenetic analysis for horizontal transfer in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466308/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466308/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">GC42 turnover rates (± SE) estimated for different phylogenetic groups in <em>S. cerevisiae</em> and <em>S. paradoxus</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-17">A notable bias toward GC42 loss was observed in <em>S. cerevisiae</em> (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>), which can be explained by the deleterious nature of GC42 as a type of TEs and perhaps transient fate at many mitochondrial genomic locations. Under such circumstances, one would expect higher GC42 turnover rates and less bias toward loss among more closely related genomes due to recent evolutionary separation, which does not yet provide sufficient time to purge deleterious genetic elements. Consistently, when estimation was performed within more closely related clades (<em>i.e.</em>, clade A and clade B in <a href="http://www.g3journal.org/content/suppl/2015/04/20/g3.115.017822.DC1/FigureS2.pdf">Figure S2</a>), the turnover rates are higher than those estimated for the entire species and there is an insignificant trend toward GC42 gain, at minimum, and no more bias toward GC42 loss (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). Furthermore, the ancestral state at each GC42 homologous position was estimated (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Among the 89 GC42 positions, 46 positions favor ancestral “presence” and 43 positions favor ancestral “absence.” The ancestral absence of GC42 in these positions supports a significant number of GC42 gains during mitochondrial genome evolution. Thus, the sporadic distribution of GC42 homologs is likely due to their own dynamic lifecycle as TEs.</p></div><div id="sec-9" class="subsection"><h3>GC42 is under functional constraint</h3><p id="p-18">TEs can move from one genomic location to another faster than the genome can replicate. The reproductive success of TEs will depend on their ability to rapidly proliferate within the genome. To maintain their functional integrity, the TE sequences are expected to be under selection to purge mutations disrupting TE activity. GC clusters are known to form palindromic structures (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>) (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">de Zamaroczy and Bernardi 1986</a>; <a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Lang <em>et al.</em> 2014</a>), and we sought whether their palindromic structures are under functional constraint. If GC42 is under no functional constraint, then it would be subject to random substitutions along the 42 nucleotides. Among all the GC42 homologs in <em>S. cerevisiae</em>, 367 nucleotide changes (by comparing to the consensus sequence in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>) are located at 12 sites in the loops regions, whereas 174 nucleotide changes are at the remaining 30 nucleotide sites (<em>P</em> = 8.04×10<sup>−7</sup>, Fisher’s exact test). The significantly high proportion of nucleotide changes in the loop regions suggests that the secondary hairpin structure of GC42 is of functional importance. The two most abundant GC42 homologs distinct from the consensus (36<sup>G</sup>, 10<sup>C</sup>36<sup>G</sup>) only have nucleotide changes in the loop regions (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>). These substitution-containing homologs of GC42 also show copy number variation among strains following a sporadic distribution (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a>). The constraint on the secondary structure suggests that the palindromic structure is functionally important for GC42.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1273/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Characteristics of GC42 sequences in S. cerevisiae. (A) Predicted secondary structure of GC42 based on the consensus sequence. The nucleotides in loop regions are in red. (B) Sequence logo for all GC42 homologous sequences. The nucleotides in loop regions are in red. (C) Distribution of the nine most abundant GC42 sequence types. For each GC42 sequence type, the total GC42 copy number in all S. cerevisiae strains and the number of strains containing the corresponding sequence type are shown. (D) GC42 density (units/kb) in indel regions compared with that in whole genomes in the five pairs of closely related genomes. The P-value is based on the Mann&#x2013;Whitney U-test." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2109141736" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Characteristics of GC42 sequences in &lt;em&gt;S. cerevisiae&lt;/em&gt;. (A) Predicted secondary structure of GC42 based on the consensus sequence. The nucleotides in loop regions are in red. (B) Sequence logo for all GC42 homologous sequences. The nucleotides in loop regions are in red. (C) Distribution of the nine most abundant GC42 sequence types. For each GC42 sequence type, the total GC42 copy number in all &lt;em&gt;S. cerevisiae&lt;/em&gt; strains and the number of strains containing the corresponding sequence type are shown. (D) GC42 density (units/kb) in indel regions compared with that in whole genomes in the five pairs of closely related genomes. The &lt;em&gt;P&lt;/em&gt;-value is based on the Mann&#x2013;Whitney &lt;em&gt;U&lt;/em&gt;-test.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1273/F2.medium.gif" width="440" height="231"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/6/1273/F2.medium.gif" width="440" height="231"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1273/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1273/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466195" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-19" class="first-child">Characteristics of GC42 sequences in <em>S. cerevisiae</em>. (A) Predicted secondary structure of GC42 based on the consensus sequence. The nucleotides in loop regions are in red. (B) Sequence logo for all GC42 homologous sequences. The nucleotides in loop regions are in red. (C) Distribution of the nine most abundant GC42 sequence types. For each GC42 sequence type, the total GC42 copy number in all <em>S. cerevisiae</em> strains and the number of strains containing the corresponding sequence type are shown. (D) GC42 density (units/kb) in indel regions compared with that in whole genomes in the five pairs of closely related genomes. The <em>P</em>-value is based on the Mann–Whitney <em>U</em>-test.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">The average nucleotide diversity was measured separately for the loop and stem regions of GC42, and was compared against the average pairwise synonymous substitution rates in seven (all but the <em>var</em> gene) protein genes (<a href="http://www.g3journal.org/content/suppl/2015/04/20/g3.115.017822.DC1/FigureS3.pdf">Figure S3</a>). Here, we used the average pairwise synonymous substitution rates as an approximate guide for genome-wide mutation rate due to the difficulty to accurately align the entire sequences of extremely AT-rich mitochondrial genomes. The average nucleotide diversity of the GC42 loop regions is higher than the median pairwise synonymous substitution rates of most protein coding genes. This could be explained by nonallelic homologous recombination among the GC42 sequences, by purifying selection acting on gene synonymous sites (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Lawrie <em>et al.</em> 2013</a>), and/or by targeted copy correction by gene conversion on protein coding genes (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Khakhlova and Bock 2006</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Christensen 2013</a>). Importantly, the diversity of the GC42 stem regions is much lower than the pairwise synonymous substitution rates of most protein coding genes. Conservatively speaking, the substitution rate in the GC42 stem regions has been significantly reduced from the genome-wide mutation rate, thus GC42 is believed to be under functional constraint.</p><p id="p-21">Sequence transposition in one genome leads to insertions/deletions (indels) in a two-genome comparison. Recent TE activity can generate genomic indels, which in turn serve as indicators for recent TE activity (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Mills <em>et al.</em> 2007</a>). In <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>, some closely related genomes show very different GC42 distribution, suggesting that GC42 has recently been, and/or still is, active. In five pairs of closely related genomes (colored in <a href="http://www.g3journal.org/content/suppl/2015/04/20/g3.115.017822.DC1/FigureS2.pdf">Figure S2</a>), we have identified sequences (indels) present in one genome but not the other, and found that the density of GC42 (units/kb) is significantly higher in these indels than in the whole genomes (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2D</a>). The high GC42 density in the indels of closely related genome pairs is likely contributed by the recent TE activity of GC42.</p><p id="p-22">Nucleotide substitutions immediately flanking GC42 were observed (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Importantly, the variation in these nucleotide sites is associated with the presence or absence of GC42, but not necessarily always with the phylogenetic relationship. These findings suggest that the variable nucleotides immediately flanking GC42 are likely the co-conversion tract of GC42 insertion, a common sequence mark of insertion for many mobile sequences, <em>e.g.</em>, group I and group II introns (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lambowitz and Belfort 1993</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Moran <em>et al.</em> 1995</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Sanchez-Puerta <em>et al.</em> 2008</a>). The presence of putative GC42 co-conversion tract suggests active mobility of GC42 at some point of evolution.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1273/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Nucleotide alignment of two GC42 homologous positions and their flanking regions. (A) GC42 presence and absence. (B) GC42 presence, GC42 subunit presence, and GC42 absence. The GC42 unit is overlined. Dots indicate identities relative to the top sequence in each panel, whereas letters represent nucleotide differences. The strain names are colored by their GC42 pattern: GC42 presence in red, GC42 subunit presence in blue, and GC42 absence in green. The putative co-conversion tract nucleotides are colored for GC42 presence or GC42 subunit presence in the same color with the corresponding strain." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2109141736" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Nucleotide alignment of two GC42 homologous positions and their flanking regions. (A) GC42 presence and absence. (B) GC42 presence, GC42 subunit presence, and GC42 absence. The GC42 unit is overlined. Dots indicate identities relative to the &lt;em&gt;top&lt;/em&gt; sequence in each panel, whereas letters represent nucleotide differences. The strain names are colored by their GC42 pattern: GC42 presence in red, GC42 subunit presence in blue, and GC42 absence in green. The putative co-conversion tract nucleotides are colored for GC42 presence or GC42 subunit presence in the same color with the corresponding strain.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1273/F3.medium.gif" width="440" height="280"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/6/1273/F3.medium.gif" width="440" height="280"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1273/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1273/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466283" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-23" class="first-child">Nucleotide alignment of two GC42 homologous positions and their flanking regions. (A) GC42 presence and absence. (B) GC42 presence, GC42 subunit presence, and GC42 absence. The GC42 unit is overlined. Dots indicate identities relative to the <em>top</em> sequence in each panel, whereas letters represent nucleotide differences. The strain names are colored by their GC42 pattern: GC42 presence in red, GC42 subunit presence in blue, and GC42 absence in green. The putative co-conversion tract nucleotides are colored for GC42 presence or GC42 subunit presence in the same color with the corresponding strain.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Exchange of GC42 between species</h3><p id="p-24">Many TE families are known to have horizontal transmission for their long-term maintenance during evolution (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Schaack <em>et al.</em> 2010</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Wallau <em>et al.</em> 2012</a>). To explore whether GC42 is involved in horizontal transfer and proliferation in other species (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a> and <a href="http://www.g3journal.org/content/suppl/2015/04/20/g3.115.017822.DC1/FigureS2.pdf">Figure S2</a>), we searched GC42 homologs in 15 <em>S. paradoxus</em> strains. GC42 homologs in two <em>S. paradoxus</em> strains (N44 and IFO1804) show 100% identity with the GC42 at the homologous position in <em>S. cerevisiae</em> S288c (also the most abundant GC42 type shown in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>), and the flanking regions in these two <em>S. paradoxus</em> strains show much closer relationships with <em>S. cerevisiae</em> than any other <em>S. paradoxus</em> strains (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>). These suggest that the transfer of GC42 into <em>S. paradoxus</em> N44 and IFO1804 from <em>S. cerevisiae</em> was mediated by homologous recombination. Moreover, <em>S. paradoxus</em> N44 and IFO1804 bear a second GC42 homolog of 100% identity with the first GC42 copy. In a BLASTN search using the flanking region of the second GC42 as a query, significant hits were found in only two other <em>S. paradoxus</em> strains, CBS432 and Y7; however, both strains lack GC42 at this homologous position (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4C</a>). Because no other GC42 homologs in <em>S. paradoxus</em> than these two copies in N44 and IFO180 are 100% identical with any GC42 homologs in <em>S. cerevisiae</em>, it is thus most likely that the second GC42 copy in <em>S. paradoxus</em> N44 and IFO1804 was inserted recently and perhaps due to proliferation after the transfer of the first GC42 from <em>S. cerevisiae</em>.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1273/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="GC42 exchange between S. cerevisiae and S. paradoxus. (A) Schematic presentation of horizontal transfer and propagation into new host. (B) Evidence of horizontal transfer. Phylogenetic analysis for the flanking &#xB1;100-bp homologous regions of a GC42 unit (arrow in Figure 1). (C) Evidence for proliferation after horizontal transfer. Sequence alignment of the homologous region for the second GC42 in N44 and IFO1804 strains. The GC42 position is highlighted in gray. Dots indicate identities relative to the N44 GC42 sequence, whereas letters represent nucleotide differences." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2109141736" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;GC42 exchange between &lt;em&gt;S. cerevisiae&lt;/em&gt; and &lt;em&gt;S. paradoxus&lt;/em&gt;. (A) Schematic presentation of horizontal transfer and propagation into new host. (B) Evidence of horizontal transfer. Phylogenetic analysis for the flanking &#xB1;100-bp homologous regions of a GC42 unit (arrow in Figure 1). (C) Evidence for proliferation after horizontal transfer. Sequence alignment of the homologous region for the second GC42 in N44 and IFO1804 strains. The GC42 position is highlighted in gray. Dots indicate identities relative to the &lt;em&gt;N44&lt;/em&gt; GC42 sequence, whereas letters represent nucleotide differences.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1273/F4.medium.gif" width="440" height="231"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/6/1273/F4.medium.gif" width="440" height="231"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1273/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1273/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466329" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-25" class="first-child">GC42 exchange between <em>S. cerevisiae</em> and <em>S. paradoxus</em>. (A) Schematic presentation of horizontal transfer and propagation into new host. (B) Evidence of horizontal transfer. Phylogenetic analysis for the flanking ±100-bp homologous regions of a GC42 unit (arrow in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). (C) Evidence for proliferation after horizontal transfer. Sequence alignment of the homologous region for the second GC42 in N44 and IFO1804 strains. The GC42 position is highlighted in gray. Dots indicate identities relative to the <em>N44</em> GC42 sequence, whereas letters represent nucleotide differences.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">It is also evident that the GC42 flanking regions in two <em>S. cerevisiae</em> strains (L1528 and DBVPG6765) are more closely related to <em>S. paradoxus</em> than any other <em>S. cerevisiae</em> strains (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4B</a>). The GC42 homologs in <em>S. cerevisiae</em> L1528 and DBVPG6765 share higher similarity with the consensus GC42 sequence in <em>S. paradoxus</em> (98% identity) than that in <em>S. cerevisiae</em> (88.1% identity). These results suggest that GC42 exchange also takes place from <em>S. paradoxus</em> to <em>S. cerevisiae</em> via homologous recombination at the flanking regions. Similarly, GC42 is sporadically distributed in <em>S. paradoxus</em> strains, ranging from 2 to 11 copies. GC42 homologs in <em>S. paradoxus</em> also show a higher proportion of nucleotide changes in the loop regions than in the stem regions. Among the 113 GC42 homologs in <em>S. paradoxus</em>, 48 nucleotide changes are located at 12 sites in the loop regions, whereas 2 nucleotide changes are at the remaining 30 nucleotide sites (<em>P</em> = 2.42×10<sup>−12</sup>, Fisher’s exact test). It is worth noting that the ratio of nucleotide changes in the loops over in the stems (48:2) in <em>S. paradoxus</em> is higher than that (367:174) in <em>S. cerevisiae</em> (<em>P</em> = 3.27×10<sup>−06</sup>, Fisher’s exact test). These results could be explained by the greater diversity in loop regions among the <em>S. paradoxus</em> strains than that among the <em>S. cerevisiae</em>, and a higher fraction of nucleotide changes in the stems have been purged in <em>S. paradoxus</em> than in <em>S. cerevisiae</em>. Despite its relative low copy number in <em>S. paradoxus</em>, GC42 should still be considered mobile in <em>S. paradoxus</em>. In fact, the estimated turnover rates in <em>S. paradoxus</em> and in clade B are higher than those in <em>S. cerevisiae</em> and in clade A (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). Among the 15 GC42 homologous positions in <em>S. paradoxus</em>, four positions were inferred to favor an ancestral state “absence” (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Furthermore, GC42 exchange can take place at homologous positions between different genomes, which would not be detectable in the analysis of turnover rates.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1273/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Merger and shuffling of GC42. There are two types of GC42 in S. paradoxus: stand-alone and part of a merged 86-nucleotide GC cluster. (A) Predicted secondary structure of the merged 86-nucleotide GC cluster in S. paradoxus. (B) Distribution of stand-alone GC42 (green) and merged GC42 (red) in S. paradoxus. For each homologous position, the likely ancestral state was estimated and shown in a pie chart. The pie charts on the top row favor an ancestral state of absence, whereas the pie charts on the second row favor an ancestral state of presence. (C) Multiple alignment of GC42 and their precursor GC18 in S. cerevisiae. (D) Schematic presentation of merger and shuffling of GC clusters." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2109141736" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Merger and shuffling of GC42. There are two types of GC42 in &lt;em&gt;S. paradoxus&lt;/em&gt;: stand-alone and part of a merged 86-nucleotide GC cluster. (A) Predicted secondary structure of the merged 86-nucleotide GC cluster in &lt;em&gt;S. paradoxus&lt;/em&gt;. (B) Distribution of stand-alone GC42 (green) and merged GC42 (red) in &lt;em&gt;S. paradoxus&lt;/em&gt;. For each homologous position, the likely ancestral state was estimated and shown in a pie chart. The pie charts on the top row favor an ancestral state of absence, whereas the pie charts on the second row favor an ancestral state of presence. (C) Multiple alignment of GC42 and their precursor GC18 in &lt;em&gt;S. cerevisiae&lt;/em&gt;. (D) Schematic presentation of merger and shuffling of GC clusters.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1273/F5.medium.gif" width="440" height="307"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/6/1273/F5.medium.gif" width="440" height="307"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1273/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1273/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466179" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-27" class="first-child">Merger and shuffling of GC42. There are two types of GC42 in <em>S. paradoxus</em>: stand-alone and part of a merged 86-nucleotide GC cluster. (A) Predicted secondary structure of the merged 86-nucleotide GC cluster in <em>S. paradoxus</em>. (B) Distribution of stand-alone GC42 (green) and merged GC42 (red) in <em>S. paradoxus</em>. For each homologous position, the likely ancestral state was estimated and shown in a pie chart. The pie charts on the top row favor an ancestral state of absence, whereas the pie charts on the second row favor an ancestral state of presence. (C) Multiple alignment of GC42 and their precursor GC18 in <em>S. cerevisiae</em>. (D) Schematic presentation of merger and shuffling of GC clusters.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>GC cluster merger and the birth of new GC clusters</h3><p id="p-28">In <em>S. paradoxus</em>, GC42 can be found in two forms, stand-alone and merged with another GC cluster (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>), both of which show copy number variation among different strains (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>). The stand-alone and merger forms can be found in different strains at the same homologous site, determined by their homologous flanking regions (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a>). We then sought to address whether the merger of GC clusters is a general phenomenon during the evolution of GC clusters. One 18-nucleotide GC cluster immediately flanked by AT-rich sequences was found to be identical with the 3′-terminus of GC42 (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5C</a>). This GC18 sequence was found in 11 of the 18 studied <em>S. cerevisiae</em> strains and of copy number variation, but was absent from any <em>S. paradoxus</em> strains (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5C</a>). Like intact GC42, GC18 also has putative co-conversion tracts. In one GC42 homologous position (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>), all the T nucleotide substitutions are associated with the presence of GC18. These results suggest that GC18 might be a smaller mobile unit than GC42 and raise the possibility that GC42 itself could have resulted from merger of two smaller GC clusters (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5D</a>). We further noticed that the second and third most abundant GC clusters in <em>S. cerevisiae</em> S288c share an identical 5′-terminus (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). Using the 5′-terminus as a query, we have identified 403 GC-rich sequences in the 18 studied <em>S. cerevisiae</em> strains, all of which share the first 13 nucleotides 5-ACTCCTTCGGGGT-3 but might have different downstream adjacent sequences (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). These sequences are of 76 distinct sequence types with various lengths. Thus, it seems to be common that GC clusters undergo active merger and shuffling during yeast mitochondrial genome evolution.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1273/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of GC clusters that share an identical 13-nucleotide sequence at the 5&#x2032;-end. The boxplot refers to GC content for all GC clusters sharing GC13 (conserved region). The a-axis indicates the length of GC-rich sequences adjacent to the identical 13-nucleotide sequence, whereas the y-axis indicates the number of unique sequence types for each sequence length." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2109141736" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of GC clusters that share an identical 13-nucleotide sequence at the 5&#x2032;-end. The boxplot refers to GC content for all GC clusters sharing GC13 (conserved region). The a-axis indicates the length of GC-rich sequences adjacent to the identical 13-nucleotide sequence, whereas the y-axis indicates the number of unique sequence types for each sequence length.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1273/F6.medium.gif" width="440" height="137"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/5/6/1273/F6.medium.gif" width="440" height="137"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1273/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1273/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466256" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-29" class="first-child">Distribution of GC clusters that share an identical 13-nucleotide sequence at the 5′-end. The boxplot refers to GC content for all GC clusters sharing GC13 (conserved region). The a-axis indicates the length of GC-rich sequences adjacent to the identical 13-nucleotide sequence, whereas the y-axis indicates the number of unique sequence types for each sequence length.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Expression of the GC42 sequences</h3><p id="p-30">Many GC42 sequences are transcribed into RNAs, but their express levels vary substantially (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). Twelve GC42 homologs show higher expression levels than the var (rps3) gene, and two GC42 homologs show high expression levels comparable to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007267">atp8</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007268">atp6</a></em>. We further investigated the expression level of the 12 most highly expressed GC42 sequences and their flanking sequences (upstream 40 nucleotides and downstream 40 nucleotides). We have observed that seven GC42 sequences are transcribed at levels similar to their flanking sequences (<em>e.g.</em>, in <a href="http://www.g3journal.org/content/suppl/2015/04/20/g3.115.017822.DC1/FigureS4.pdf">Figure S4, A, D, E, G, H, J, and K</a>) and five GC42 sequences are transcribed at higher levels than their flanking sequences (<em>e.g.</em>, in <a href="http://www.g3journal.org/content/suppl/2015/04/20/g3.115.017822.DC1/FigureS4.pdf">Figure S4 B, C, F, I, and L</a>). These results suggest that at least some GC42 sequences are independently transcribed in the host.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1273/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Expression of GC42 homologs. As a comparison, three protein genes, rps3 (var), atp8, and atp6, are shown. The RPKMs are normalized to have identical atp6 RPKMs between the GCDA5 and GCDA8 strains, both of which are derived from S. cerevisiae BY4741 (isogenic to S288c)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2109141736" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Expression of GC42 homologs. As a comparison, three protein genes, &lt;em&gt;rps3&lt;/em&gt; (&lt;em&gt;var&lt;/em&gt;), &lt;em&gt;atp8&lt;/em&gt;, and &lt;em&gt;atp6&lt;/em&gt;, are shown. The RPKMs are normalized to have identical &lt;em&gt;atp6&lt;/em&gt; RPKMs between the GCDA5 and GCDA8 strains, both of which are derived from &lt;em&gt;S. cerevisiae&lt;/em&gt; BY4741 (isogenic to S288c).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1273/F7.medium.gif" width="440" height="404"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/5/6/1273/F7.medium.gif" width="440" height="404"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1273/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1273/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466316" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-31" class="first-child">Expression of GC42 homologs. As a comparison, three protein genes, <em>rps3</em> (<em>var</em>), <em>atp8</em>, and <em>atp6</em>, are shown. The RPKMs are normalized to have identical <em>atp6</em> RPKMs between the GCDA5 and GCDA8 strains, both of which are derived from <em>S. cerevisiae</em> BY4741 (isogenic to S288c).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><div id="sec-14" class="subsection"><h3>The origin and evolutionary history of GC-rich TEs</h3><p id="p-32">During evolution, the GC-rich TEs both accumulate nucleotide substitutions and undergo unit merging and shuffling. Nucleotide changes in <em>S. cerevisiae</em> and <em>S. paradoxus</em> strains all support a higher proportion of nucleotide changes in the loop regions than in the stem regions. This is also supported by the interspecific difference of GC42 homologs between <em>S. cerevisiae</em> and <em>S. paradoxus</em>. For instance, the GC42 consensus sequence in <em>S. cerevisiae</em> (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2A</a>) differs by four nucleotides (positions 16, 17, 24, 37) from that in <em>S. paradoxus</em> (<a href="http://www.g3journal.org/content/suppl/2015/04/20/g3.115.017822.DC1/FigureS5.pdf">Figure S5</a>) and all four nucleotides are in the loop regions (<em>P</em> = 0.011, Fisher’s exact test). GC42 undergoes dynamic transformation, presumably derived from merger of smaller GC clusters as observed in <em>S. cerevisiae</em> and further merging into bigger GC clusters as observed in <em>S. paradoxus</em> (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5</a>). GC clusters can potentially undergo merger and separation (fusion and fission) in a bidirectional manner, but the abundance of each GC cluster unit depends on its own mobile activity and functional constraint under selection.</p><p id="p-33">Our study observed exchange of GC42 between <em>S. cerevisiae</em> and <em>S. paradoxus</em> via homologous recombination at the conserved flanking regions, which was recently recognized as an often overlooked mechanism mediating horizontal transfer (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Polz <em>et al.</em> 2013</a>). Similarly, mitochondrial introns have been previously documented to be involved in horizontal transfer mediated by homologous recombination at the conserved flanking regions (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hepburn <em>et al.</em> 2012</a>; <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Wu and Hao 2014</a>). Horizontal transfer of another GC cluster has been documented, which is involved in the transfer of a 48-nucleotide GC cluster within the var (rps3) gene between <em>S. cerevisiae</em> and <em>Kluyveromyces lactis</em> (<a id="xref-ref-36-5" class="xref-bibr" href="#ref-36">Lang <em>et al.</em> 2014</a>). Given the similar dynamics of GC clusters observed in <em>S. cerevisiae</em> and <em>S. paradoxus</em>, we propose that after horizontal transfer of GC-rich TEs, these TEs would experience the similar evolutionary dynamics (nucleotide substitutions, merging, and shuffling) as in the previous host (illustrated in <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>). This process shares all the important features documented in animal TEs, <em>e.g.</em>, rapid mutation accumulation, diversification, and proliferation in a host and possible horizontal transfer into a new host (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Schaack <em>et al.</em> 2010</a>).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/6/1273/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Model for the lifecycle of GC-rich TEs. GC-rich TEs undergo rapid substitution, dynamic merger, shuffling, and horizontal transfer, with details discussed in the main text." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2109141736" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Model for the lifecycle of GC-rich TEs. GC-rich TEs undergo rapid substitution, dynamic merger, shuffling, and horizontal transfer, with details discussed in the main text.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/6/1273/F8.medium.gif" width="440" height="131"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/5/6/1273/F8.medium.gif" width="440" height="131"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/6/1273/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/6/1273/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466337" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-34" class="first-child">Model for the lifecycle of GC-rich TEs. GC-rich TEs undergo rapid substitution, dynamic merger, shuffling, and horizontal transfer, with details discussed in the main text.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Amplification mechanism of GC-rich TEs</h3><p id="p-35">GC clusters, including GC42, bear putative target-site duplication (<a id="xref-ref-63-5" class="xref-bibr" href="#ref-63">Weiller <em>et al.</em> 1989</a>; <a id="xref-ref-36-6" class="xref-bibr" href="#ref-36">Lang <em>et al.</em> 2014</a>) (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2B</a> and <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>), a key feature in class II TEs (DNA transposons). It has been proposed that endonucleases encoded in mitochondrial introns may facilitate the mobility of GC clusters as DNA transposons (<a id="xref-ref-36-7" class="xref-bibr" href="#ref-36">Lang <em>et al.</em> 2014</a>). However, target-site duplication in class II TEs is involved in a cut-and-paste transposition mechanism, which does not often lead to a substantial increase in copy numbers. There is one possibility for a cut-and-paste transposition mechanism to increase copy numbers. It would require the combination of stable heteroplasmic mtDNAs containing different GC42 patterns, relatively efficient mtDNA recombination and segregation, and biased retention of high GC42-copy mtDNA genotypes. Among these three requisites, only mtDNA recombination is commonly recognized (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Shannon <em>et al.</em> 1972</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Dujon <em>et al.</em> 1974</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Fritsch <em>et al.</em> 2014</a>). The heteroplasmic state in <em>S. cerevisiae</em> is generally believed to be transient and to last no more than 20 mitotic cell divisions (vegetative segregation) (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Birky 2001</a>), and there has been no evidence supporting GC cluster–rich mtDNA genotypes being preferentially retained from heteroplasmic cells. A similar challenge has been documented in a previous study on the copy number variation of nuclear MITEs (miniature inverted-repeat transposable elements) (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Fattash <em>et al.</em> 2013</a>). To obtain a more sophisticated answer on the transposition and proliferation of GC-rich TEs, we sought to access the possible contribution of the cut-and-paste mechanism in GC-rich TEs. If cut-and-paste were the only transposition mechanism of GC42, then we would expect all GC42-absence positions to have either one single set of target site nucleotides (AG) (presumably never inserted or perfectly cut out) or GC42 fragment with two sets of target site nucleotides (AG) as footprint. Among the 89 homologous positions, GC42-absence sites in 25 homologous positions only have a single set of target site nucleotides, and five positions have footprint. All these findings suggest that cut-and-paste is unlikely the sole mechanism of GC42 transposition.</p><p id="p-36">We then investigated the possibility of GC-rich TEs being involved in RNA-mediated transposition (also known as copy-and-paste). The expression of GC42 would be an excellent precondition for reverse transcription (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>). Furthermore, yeast mitochondrial genomes contain several functional intron ORFs, which encode endonucleases and reverse transcriptases involved in intron mobility (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Eskes <em>et al.</em> 2000</a>; <a id="xref-ref-36-8" class="xref-bibr" href="#ref-36">Lang <em>et al.</em> 2014</a>). The presence of functional reverse transcriptase, in principle, can facilitate RNA-mediated retrotransposition of GC-rich TEs. We suspect that the rapid GC42 proliferation is, in part, due to RNA-mediated retrotransposition (copy-and-paste). However, additional experimental evidence is required to conclusively determine the proliferation mechanisms. Previous studies of mitochondrial intron transposition and retrotransposition have documented the co-conversion of flanking exon sequences as the result of mobile intron insertion mediated by intron-encoded enzymes (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Lambowitz and Belfort 1993</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Moran <em>et al.</em> 1995</a>; <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Sanchez-Puerta <em>et al.</em> 2008</a>). The putative co-conversion tracts are likely the footprints of GC cluster insertions (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>).</p></div><div id="sec-16" class="subsection"><h3>Merger and shuffling of palindromic clusters: a potential source of evolutionary novelty</h3><p id="p-37">Palindromic GC clusters were first discovered approximately 40 years ago (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bernardi 1976</a>), but the origin and evolution of GC clusters are still poorly understood. Our findings show, for the first time, that GC clusters undergo dynamic merger and shuffling (<a id="xref-fig-5-9" class="xref-fig" href="#F5">Figure 5</a>). It is reasonable to believe that GC clusters have variable mobile activities because of their different secondary structures. The highly dynamic merger and shuffling processes will alter the secondary structure and mobile activity of GC-rich TEs and ultimately change their abundance at both the genomic and population levels.</p><p id="p-38">The merger of palindromic sequences has been proposed as an important mechanism to create functional and structurally complex RNAs. For instance, nuclear tRNA halves can form hairpin structure and can be ligated into chimeric tRNA with cloverleaf structure during evolution (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Zuo <em>et al.</em> 2013</a>), that is, merger and shuffling of tRNA fragments created modern tRNAs. Modern tRNAs could have been inserted into the genome via retrotransposition (<a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Zuo <em>et al.</em> 2013</a>), which is likely also crucial for the mobility of GC clusters. Many hairpin-structured RNAs have been previously shown to bear ribozyme activity, which catalyzes self-cleavage and ligation reactions (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Gwiazda <em>et al.</em> 2012</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Muller <em>et al.</em> 2012</a>). The potential ribozyme activity of palindromic sequences could play an important role in initiating and promoting their own merger and shuffling. We tend to believe that the shuffling of hairpin-forming sequences is likely associated with RNA-mediated ligation and retrotransposition.</p></div><div id="sec-17" class="subsection"><h3>Evolution of GC clusters and mitochondrial genome size</h3><p id="p-39">Most GC clusters, including GC42, are located in intergenic regions, whose size is often strongly associated with mitochondrial genome size (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bouchier <em>et al.</em> 2009</a>). GC clusters are short in length, and thus their direct sequence-length contribution to mitochondrial genome size is minimal. However, GC clusters have been suggested as mtDNA recombination hotspots (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Dieckmann and Gandy 1987</a>), and GC cluster–mediated gene conversion can insert or delete large genomic fragments, which ultimately lead to substantial alteration of genome size. GC clusters have also been proposed to play a role in increasing mitochondrial genome size by inducing long AT-rich stretches (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Bouchier <em>et al.</em> 2009</a>). Furthermore, the abundance of TE sequences has been shown as the result of nonadaptive processes such as mutation and genetic drift during the evolution of genome size (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lynch and Conery 2003</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Lynch <em>et al.</em> 2006</a>). The nonadaptive evolutionary theory (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Lynch and Conery 2003</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Lynch <em>et al.</em> 2006</a>) would predict GC clusters, as one type of TE sequence, to be more abundant in mitochondrial genomes under stronger genetic drift. Our findings suggest that the merger and shuffling processes can change the mobile activity of GC clusters, which will determine the abundance of GC clusters and ultimately influence mitochondrial genome size evolution. As more abundant population genomics data become available from a broad spectrum of species, the above hypotheses can be tested in a more rigorous and sophisticated manner.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">The authors thank Dr. Aimée Dudley and three anonymous reviewers for their helpful comments. The authors are grateful for the grid computing service from Computing &amp; Information Technology of Wayne State University. This work was supported by funds from Wayne State University (to W.H.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-3"><p id="p-41">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017822/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017822/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-42">Sequence data from this article have been deposited under GenBank accession nos. KP712778-KP712805.</p></li><li class="fn" id="fn-5"><p id="p-43"><em>Communicating editor: A. M. Dudley</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-03-09"><span class="received-label">Received </span>March 9, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-04-17"><span class="accepted-label">Accepted </span>April 17, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Wu and Hao</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.1" data-doi="10.1093/nar/25.17.3389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madden</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaffer</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3389</span>–<span class="cit-lpage">3402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.17.3389%26rft_id%253Dinfo%253Apmid%252F9254694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xNy8zMzg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.2" data-doi="10.1093/molbev/msq029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alverson</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stern</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barry</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Insights into the evolution of mitochondrial genome size from complete sequences of <em>Citrullus lanatus</em> and <em>Cucurbita pepo</em> (Cucurbitaceae).</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1436</span>–<span class="cit-lpage">1448</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq029%26rft_id%253Dinfo%253Apmid%252F20118192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI3LzYvMTQzNiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMjczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.3" data-doi="10.1089/cmb.2012.0021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bankevich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nurk</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antipov</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gurevich</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dvorkin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing.</span> <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">455</span>–<span class="cit-lpage">477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DBankevich%26rft.auinit1%253DA.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D455%26rft.epage%253D477%26rft.atitle%253DSPAdes%253A%2Ba%2Bnew%2Bgenome%2Bassembly%2Balgorithm%2Band%2Bits%2Bapplications%2Bto%2Bsingle-cell%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2012.0021%26rft_id%253Dinfo%253Apmid%252F22506599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.2012.0021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22506599&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.4" data-doi="10.1093/molbev/msu037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bergstrom</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salinas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barre</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A high-definition view of functional genetic variation from natural yeast genomes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">872</span>–<span class="cit-lpage">888</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu037%26rft_id%253Dinfo%253Apmid%252F24425782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMxLzQvODcyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.5" data-doi="10.1007/BF01796120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardi</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">Organization and evolution of the mitochondrial genome of yeast.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DBernardi%26rft.auinit1%253DG.%26rft.volume%253D9%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D35%26rft.atitle%253DOrganization%2Band%2Bevolution%2Bof%2Bthe%2Bmitochondrial%2Bgenome%2Bof%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01796120%26rft_id%253Dinfo%253Apmid%252F798040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01796120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=798040&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.6" data-doi="10.1146/annurev.genet.35.102401.090231"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Birky</span>  <span class="cit-name-given-names">C. W.</span>  <span class="cit-name-suffix">Jr</span>.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The inheritance of genes in mitochondria and chloroplasts: laws, mechanisms, and models.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">125</span>–<span class="cit-lpage">148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DBirky%26rft.auinit1%253DC.%2BW.%26rft.volume%253D35%26rft.issue%253D1%26rft.spage%253D125%26rft.epage%253D148%26rft.atitle%253DTHE%2BINHERITANCE%2BOF%2BGENES%2BIN%2BMITOCHONDRIA%2BAND%2BCHLOROPLASTS%253A%2BLaws%252C%2BMechanisms%252C%2Band%2BModels%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.35.102401.090231%26rft_id%253Dinfo%253Apmid%252F11700280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.35.102401.090231&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11700280&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172991500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.7" data-doi="10.1186/1471-2164-14-399"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bleykasten-Grosshans</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedrich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schacherer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide analysis of intraspecific transposon diversity in yeast.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D399%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-399%26rft_id%253Dinfo%253Apmid%252F23768249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-399&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23768249&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.8" data-doi="10.1093/bioinformatics/btq683"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boetzer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henkel</span>  <span class="cit-name-given-names">C. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pirovano</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Scaffolding pre-assembled contigs using SSPACE.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">578</span>–<span class="cit-lpage">579</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq683%26rft_id%253Dinfo%253Apmid%252F21149342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNy80LzU3OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMjczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.9" data-doi="10.1186/gb-2012-13-6-r56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boetzer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pirovano</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Toward almost closed genomes with GapFiller.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">R56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DBoetzer%26rft.auinit1%253DM.%26rft.volume%253D13%26rft.issue%253D6%26rft.spage%253DR56%26rft.epage%253DR56%26rft.atitle%253DToward%2Balmost%2Bclosed%2Bgenomes%2Bwith%2BGapFiller.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2012-13-6-r56%26rft_id%253Dinfo%253Apmid%252F22731987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2012-13-6-r56&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22731987&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.10" data-doi="10.1111/j.1567-1364.2009.00551.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bouchier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Creno</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fairhead</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Complete mitochondrial genome sequences of three <em>Nakaseomyces</em> species reveal invasion by palindromic GC clusters and considerable size expansion.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1283</span>–<span class="cit-lpage">1292</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2009.00551.x%26rft_id%253Dinfo%253Apmid%252F19758332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjg6IjkvOC8xMjgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.11" data-doi="10.1186/1471-2105-10-421"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Camacho</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulouris</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Avagyan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papadopoulos</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">BLAST+: architecture and applications.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DCamacho%26rft.auinit1%253DC.%26rft.volume%253D10%26rft.spage%253D421%26rft.epage%253D421%26rft.atitle%253DBLAST%252B%253A%2Barchitecture%2Band%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-421%26rft_id%253Dinfo%253Apmid%252F20003500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-421&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20003500&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.12" data-doi="10.1371/journal.pone.0050978"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carr</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bensasson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergman</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evolutionary genomics of transposable elements in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e50978</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D7%26rft.spage%253De50978%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0050978%26rft_id%253Dinfo%253Apmid%252F23226439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0050978&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23226439&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.13" data-doi="10.1093/bioinformatics/btr703"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carver</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berriman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parkhill</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McQuillan</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Artemis: an integrated platform for visualization and analysis of high-throughput sequence-based experimental data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">464</span>–<span class="cit-lpage">469</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr703%26rft_id%253Dinfo%253Apmid%252F22199388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyOC80LzQ2NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMjczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.14" data-doi="10.1093/gbe/evt069"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">A. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Plant mitochondrial genome evolution can be explained by DNA repair mechanisms.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1079</span>–<span class="cit-lpage">1086</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevt069%26rft_id%253Dinfo%253Apmid%252F23645599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjg6IjUvNi8xMDc5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.15" data-doi="10.1186/gb-2007-8-s1-s5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collier</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Largaespada</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Transposable elements and the dynamic somatic genome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">8</span>(<span class="cit-supplement">Suppl 1</span>): <span class="cit-fpage">S5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D8%26rft.spage%253DS5%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2007-8-s1-s5%26rft_id%253Dinfo%253Apmid%252F18047697%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2007-8-s1-s5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18047697&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.16" data-doi="10.1101/gr.849004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crooks</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hon</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chandonia</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brenner</span>  <span class="cit-name-given-names">S. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">WebLogo: a sequence logo generator.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1188</span>–<span class="cit-lpage">1190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DCrooks%26rft.auinit1%253DG.%2BE.%26rft.volume%253D14%26rft.issue%253D6%26rft.spage%253D1188%26rft.epage%253D1190%26rft.atitle%253DWebLogo%253A%2BA%2BSequence%2BLogo%2BGenerator%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.849004%26rft_id%253Dinfo%253Apmid%252F15173120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE0LzYvMTE4OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMjczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.17" data-doi="10.1016/0378-1119(86)90262-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Zamaroczy</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardi</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">The GC clusters of the mitochondrial genome of yeast and their evolutionary origin.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253Dde%2BZamaroczy%26rft.auinit1%253DM.%26rft.volume%253D41%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D22%26rft.atitle%253DThe%2BGC%2Bclusters%2Bof%2Bthe%2Bmitochondrial%2Bgenome%2Bof%2Byeast%2Band%2Btheir%2Bevolutionary%2Borigin.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252886%252990262-3%26rft_id%253Dinfo%253Apmid%252F3009268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(86)90262-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3009268&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986A488400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deragon</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casacuberta</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panaud</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Plant transposable elements.</span> <abbr class="cit-jnl-abbrev">Genome Dyn.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DGenome%2BDyn%26rft.aulast%253DDeragon%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D4%26rft.spage%253D69%26rft.epage%253D82%26rft.atitle%253DPlant%2Btransposable%2Belements.%26rft_id%253Dinfo%253Apmid%252F18756078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18756078&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dieckmann</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gandy</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Preferential recombination between GC clusters in yeast mitochondrial DNA.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">4197</span>–<span class="cit-lpage">4203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft.volume%253D6%26rft.spage%253D4197%26rft_id%253Dinfo%253Apmid%252F3327690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=3327690&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987L435200041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slonimski</span>  <span class="cit-name-given-names">P. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weill</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">Mitochondrial genetics IX: A model for recombination and segregation of mitochondrial genomes in saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">415</span>–<span class="cit-lpage">437</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDujon%26rft.auinit1%253DB.%26rft.volume%253D78%26rft.issue%253D1%26rft.spage%253D415%26rft.epage%253D437%26rft.atitle%253DMITOCHONDRIAL%2BGENETICS%2BIX%253A%2BA%2BMODEL%2BFOR%2BRECOMBINATION%2BAND%2BSEGREGATION%2BOF%2BMITOCHONDRIAL%2BGENOMES%2BIN%2BSACCHAROMYCES%2BCEREVISIAE%26rft_id%253Dinfo%253Apmid%252F4613610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiI3OC8xLzQxNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMjczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.21" data-doi="10.1093/nar/gkh340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1792</span>–<span class="cit-lpage">1797</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjMyLzUvMTc5MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMjczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.22" data-doi="10.1186/1471-2164-10-591"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Erpenbeck</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voigt</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Worheide</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lavrov</span>  <span class="cit-name-given-names">D. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The mitochondrial genomes of sponges provide evidence for multiple invasions by Repetitive Hairpin-forming Elements (RHE).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DErpenbeck%26rft.auinit1%253DD.%26rft.volume%253D10%26rft.spage%253D591%26rft.epage%253D591%26rft.atitle%253DThe%2Bmitochondrial%2Bgenomes%2Bof%2Bsponges%2Bprovide%2Bevidence%2Bfor%2Bmultiple%2Binvasions%2Bby%2BRepetitive%2BHairpin-forming%2BElements%2B%2528RHE%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-591%26rft_id%253Dinfo%253Apmid%252F20003196%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-591&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20003196&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.23" data-doi="10.1128/MCB.20.22.8432-8446.2000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eskes</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">M. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dickson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Multiple homing pathways used by yeast mitochondrial group II introns.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">8432</span>–<span class="cit-lpage">8446</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DEskes%26rft.auinit1%253DR.%26rft.volume%253D20%26rft.issue%253D22%26rft.spage%253D8432%26rft.epage%253D8446%26rft.atitle%253DMultiple%2BHoming%2BPathways%2BUsed%2Bby%2BYeast%2BMitochondrial%2BGroup%2BII%2BIntrons%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.20.22.8432-8446.2000%26rft_id%253Dinfo%253Apmid%252F11046140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8yMi84NDMyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fattash</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rooke</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hui</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Miniature inverted-repeat transposable elements: discovery, distribution, and activity.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">475</span>–<span class="cit-lpage">486</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%26rft.volume%253D56%26rft.spage%253D475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.25" data-doi="10.1534/genetics.114.166637"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fritsch</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chabbert</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klaus</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A genome-wide map of mitochondrial DNA recombination in yeast.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">755</span>–<span class="cit-lpage">771</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.166637%26rft_id%253Dinfo%253Apmid%252F25081569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNzU1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.26" data-doi="10.1093/bioinformatics/btt515"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Consed: a graphical editor for next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">2936</span>–<span class="cit-lpage">2937</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt515%26rft_id%253Dinfo%253Apmid%252F23995391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjkvMjIvMjkzNiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMjczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.27" data-doi="10.1016/j.biochi.2012.03.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gwiazda</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salomon</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Appel</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">RNA self-ligation: from oligonucleotides to full length ribozymes.</span> <abbr class="cit-jnl-abbrev">Biochimie</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">1457</span>–<span class="cit-lpage">1463</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimie%26rft.volume%253D94%26rft.spage%253D1457%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.biochi.2012.03.015%26rft_id%253Dinfo%253Apmid%252F22465106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.biochi.2012.03.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22465106&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.28" data-doi="10.1101/gr.4746406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Golding</span>  <span class="cit-name-given-names">G. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The fate of laterally transferred genes: life in the fast lane to adaptation or death.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">636</span>–<span class="cit-lpage">643</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DHao%26rft.auinit1%253DW.%26rft.volume%253D16%26rft.issue%253D5%26rft.spage%253D636%26rft.epage%253D643%26rft.atitle%253DThe%2Bfate%2Bof%2Blaterally%2Btransferred%2Bgenes%253A%2BLife%2Bin%2Bthe%2Bfast%2Blane%2Bto%2Badaptation%2Bor%2Bdeath.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4746406%26rft_id%253Dinfo%253Apmid%252F16651664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE2LzUvNjM2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.29" data-doi="10.1093/molbev/mss130"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hepburn</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mower</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Loss of two introns from the <em>Magnolia tripetala</em> mitochondrial <em>cox2</em> gene implicates horizontal gene transfer and gene conversion as a novel mechanism of intron loss.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">3111</span>–<span class="cit-lpage">3120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss130%26rft_id%253Dinfo%253Apmid%252F22593225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyOS8xMC8zMTExIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.30" data-doi="10.1186/1471-2164-14-202"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Islam</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Studer</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farrell</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panitz</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The genome and transcriptome of <em>perennial ryegrass</em> mitochondria.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D202%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-202%26rft_id%253Dinfo%253Apmid%252F23521852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-202&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23521852&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.31" data-doi="10.1111/j.1365-313X.2006.02673.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khakhlova</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bock</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Elimination of deleterious mutations in plastid genomes by gene conversion.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DKhakhlova%26rft.auinit1%253DO.%26rft.volume%253D46%26rft.issue%253D1%26rft.spage%253D85%26rft.epage%253D94%26rft.atitle%253DElimination%2Bof%2Bdeleterious%2Bmutations%2Bin%2Bplastid%2Bgenomes%2Bby%2Bgene%2Bconversion.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2006.02673.x%26rft_id%253Dinfo%253Apmid%252F16553897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2006.02673.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16553897&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236035700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.32" data-doi="10.1186/1471-2105-15-320"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">DiscML: an R package for estimating evolutionary rates of discrete characters using maximum likelihood.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D15%26rft.spage%253D320%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-15-320%26rft_id%253Dinfo%253Apmid%252F25260628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-15-320&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25260628&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knoop</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Unseld</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marienfeld</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brandt</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sunkel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">copia-, gypsy- and LINE-like retrotransposon fragments in the mitochondrial genome of Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">142</span>: <span class="cit-fpage">579</span>–<span class="cit-lpage">585</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKnoop%26rft.auinit1%253DV.%26rft.volume%253D142%26rft.issue%253D2%26rft.spage%253D579%26rft.epage%253D585%26rft.atitle%253Dcopia-%252C%2Bgypsy-%2Band%2BLINE-Like%2BRetrotransposon%2BFragments%2Bin%2Bthe%2BMitochondrial%2BGenome%2Bof%2BArabidopsis%2Bthaliana%26rft_id%253Dinfo%253Apmid%252F8852855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQyLzIvNTc5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.34" data-doi="10.1093/nar/24.9.1734"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koll</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boulay</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belcour</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">d’Aubenton-Carafa</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Contribution of ultra-short invasive elements to the evolution of the mitochondrial genome in the genus <em>Podospora</em>.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1734</span>–<span class="cit-lpage">1741</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F24.9.1734%26rft_id%253Dinfo%253Apmid%252F8649993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjI0LzkvMTczNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMjczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.35" data-doi="10.1146/annurev.bi.62.070193.003103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lambowitz</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belfort</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Introns as mobile genetic elements.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">587</span>–<span class="cit-lpage">622</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.aulast%253DLambowitz%26rft.auinit1%253DA.%26rft.volume%253D62%26rft.issue%253D1%26rft.spage%253D587%26rft.epage%253D622%26rft.atitle%253DIntrons%2Bas%2BMobile%2BGenetic%2BElements%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.bi.62.070193.003103%26rft_id%253Dinfo%253Apmid%252F8352597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.bi.62.070193.003103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8352597&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LK68700020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.36" data-doi="10.1073/pnas.1322190111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lang</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jakubkova</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hegedusova</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daoud</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forget</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Massive programmed translational jumping in mitochondria.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">5926</span>–<span class="cit-lpage">5931</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1322190111%26rft_id%253Dinfo%253Apmid%252F24711422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTExLzE2LzU5MjYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTI3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.37" data-doi="10.1093/molbev/msp317"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lavrov</span>  <span class="cit-name-given-names">D. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Rapid proliferation of repetitive palindromic elements in mtDNA of the endemic Baikalian sponge <em>Lubomirskia baicalensis</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">757</span>–<span class="cit-lpage">760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp317%26rft_id%253Dinfo%253Apmid%252F20026479%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI3LzQvNzU3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.38" data-doi="10.1371/journal.pgen.1003527"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lawrie</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messer</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hershberg</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Strong purifying selection at synonymous sites in <em>D. melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003527%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003527%26rft_id%253Dinfo%253Apmid%252F23737754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003527&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23737754&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.39" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMjczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.40" data-doi="10.1371/journal.pone.0052403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lindner</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedel</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A comprehensive evaluation of alignment algorithms in the context of RNA-seq.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e52403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D7%26rft.spage%253De52403%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0052403%26rft_id%253Dinfo%253Apmid%252F23300661%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0052403&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23300661&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.41" data-doi="10.1038/nature07743"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moses</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Population genomics of domestic and wild yeasts.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLiti%26rft.auinit1%253DG.%26rft.volume%253D458%26rft.issue%253D7236%26rft.spage%253D337%26rft.epage%253D341%26rft.atitle%253DPopulation%2Bgenomics%2Bof%2Bdomestic%2Band%2Bwild%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07743%26rft_id%253Dinfo%253Apmid%252F19212322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07743&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212322&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264285600041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.42" data-doi="10.1186/2047-217X-1-18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D1%26rft.spage%253D18%26rft_id%253Dinfo%253Adoi%252F10.1186%252F2047-217X-1-18%26rft_id%253Dinfo%253Apmid%252F23587118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/2047-217X-1-18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23587118&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.43" data-doi="10.1126/science.1089370"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conery</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The origins of genome complexity.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">302</span>: <span class="cit-fpage">1401</span>–<span class="cit-lpage">1404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D302%26rft.issue%253D5649%26rft.spage%253D1401%26rft.epage%253D1404%26rft.atitle%253DThe%2BOrigins%2Bof%2BGenome%2BComplexity%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1089370%26rft_id%253Dinfo%253Apmid%252F14631042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDIvNTY0OS8xNDAxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.44" data-doi="10.1126/science.1118884"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koskella</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaack</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Mutation pressure and the evolution of organelle genomic architecture.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">311</span>: <span class="cit-fpage">1727</span>–<span class="cit-lpage">1730</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D311%26rft.issue%253D5768%26rft.spage%253D1727%26rft.epage%253D1730%26rft.atitle%253DMutation%2Bpressure%2Band%2Bthe%2Bevolution%2Bof%2Borganelle%2Bgenomic%2Barchitecture.%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1118884%26rft_id%253Dinfo%253Apmid%252F16556832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTEvNTc2OC8xNzI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.45" data-doi="10.1016/j.tig.2007.02.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mills</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iskow</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devine</span>  <span class="cit-name-given-names">S. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Which transposable elements are active in the human genome?</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">183</span>–<span class="cit-lpage">191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DMills%26rft.auinit1%253DR.%2BE.%26rft.volume%253D23%26rft.issue%253D4%26rft.spage%253D183%26rft.epage%253D191%26rft.atitle%253DWhich%2Btransposable%2Belements%2Bare%2Bactive%2Bin%2Bthe%2Bhuman%2Bgenome%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2007.02.006%26rft_id%253Dinfo%253Apmid%252F17331616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2007.02.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17331616&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246002700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moran</span>  <span class="cit-name-given-names">J. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimmerly</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eskes</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kennell</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lambowitz</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Mobile group II introns of yeast mitochondrial DNA are novel site-specific retroelements.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">2828</span>–<span class="cit-lpage">2838</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DMoran%26rft.auinit1%253DJ.%2BV.%26rft.volume%253D15%26rft.issue%253D5%26rft.spage%253D2828%26rft.epage%253D2838%26rft.atitle%253DMobile%2Bgroup%2BII%2Bintrons%2Bof%2Byeast%2Bmitochondrial%2BDNA%2Bare%2Bnovel%2Bsite-%2Bspecific%2Bretroelements%26rft_id%253Dinfo%253Apmid%252F7537853%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE1LzUvMjgyOCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMjczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.47" data-doi="10.1101/gr.129460.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mularoni</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gangadharan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wheelan</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Retrotransposon Ty1 integration targets specifically positioned asymmetric nucleosomal DNA segments in tRNA hotspots.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">693</span>–<span class="cit-lpage">703</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.129460.111%26rft_id%253Dinfo%253Apmid%252F22219510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzQvNjkzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.48" data-doi="10.1002/iub.575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Appel</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krellenberg</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petkovic</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The many faces of the hairpin ribozyme: structural and functional variants of a small catalytic RNA.</span> <abbr class="cit-jnl-abbrev">IUBMB Life</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">36</span>–<span class="cit-lpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIUBMB%2Blife%26rft.stitle%253DIUBMB%2BLife%26rft.aulast%253DMuller%26rft.auinit1%253DS.%26rft.volume%253D64%26rft.issue%253D1%26rft.spage%253D36%26rft.epage%253D47%26rft.atitle%253DThe%2Bmany%2Bfaces%2Bof%2Bthe%2Bhairpin%2Bribozyme%253A%2Bstructural%2Band%2Bfunctional%2Bvariants%2Bof%2Ba%2Bsmall%2Bcatalytic%2BRNA.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fiub.575%26rft_id%253Dinfo%253Apmid%252F22131309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/iub.575&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22131309&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.49" data-doi="10.1007/BF00020167"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakazono</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanno</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsutsumi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirai</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Palindromic repeated sequences (PRSs) in the mitochondrial genome of rice: evidence for their insertion after divergence of the genus <em>Oryza</em> from the other Gramineae.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">281</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DNakazono%26rft.auinit1%253DM.%26rft.volume%253D24%26rft.issue%253D2%26rft.spage%253D273%26rft.epage%253D281%26rft.atitle%253DPalindromic%2Brepeated%2Bsequences%2B%2528PRSs%2529%2Bin%2Bthe%2Bmitochondrial%2Bgenome%2Bof%2Brice%253A%2Bevidence%2Bfor%2Btheir%2Binsertion%2Bafter%2Bdivergence%2Bof%2Bthe%2Bgenus%2BOryza%2Bfrom%2Bthe%2Bother%2BGramineae.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00020167%26rft_id%253Dinfo%253Apmid%252F7509206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00020167&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7509206&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994MX41500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.50" data-doi="10.1093/bioinformatics/bts091"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Okonechnikov</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Golosova</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fursov</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite> and U. team, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Unipro UGENE: a unified bioinformatics toolkit.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1166</span>–<span class="cit-lpage">1167</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts091%26rft_id%253Dinfo%253Apmid%252F22368248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyOC84LzExNjYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTI3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.51" data-doi="10.1080/10635150490522232"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pagel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meade</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Bayesian estimation of ancestral character states on phylogenies.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">673</span>–<span class="cit-lpage">684</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150490522232%26rft_id%253Dinfo%253Apmid%252F15545248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoic3lzYmlvIjtzOjU6InJlc2lkIjtzOjg6IjUzLzUvNjczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.52" data-doi="10.1093/oxfordjournals.molbev.a026274"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paquin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laforest</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lang</span>  <span class="cit-name-given-names">B. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Double-hairpin elements in the mitochondrial DNA of <em>allomyces</em>: evidence for mobility.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1760</span>–<span class="cit-lpage">1768</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DPaquin%26rft.auinit1%253DB.%26rft.volume%253D17%26rft.issue%253D11%26rft.spage%253D1760%26rft.epage%253D1768%26rft.atitle%253DDouble-Hairpin%2BElements%2Bin%2Bthe%2BMitochondrial%2BDNA%2Bof%2BAllomyces%253A%2BEvidence%2Bfor%2BMobility%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a026274%26rft_id%253Dinfo%253Apmid%252F11070063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIxNy8xMS8xNzYwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.53" data-doi="10.1016/j.tig.2012.12.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Polz</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alm</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanage</span>  <span class="cit-name-given-names">W. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Horizontal gene transfer and the evolution of bacterial and archaeal population structure.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">170</span>–<span class="cit-lpage">175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D29%26rft.spage%253D170%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2012.12.006%26rft_id%253Dinfo%253Apmid%252F23332119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2012.12.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23332119&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316243500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.54" data-doi="10.1186/1471-2164-12-424"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez-Moreno</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benjak</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marti</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puigdomenech</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Determination of the melon chloroplast and mitochondrial genome sequences reveals that the largest reported mitochondrial genome in plants contains a significant amount of DNA having a nuclear origin.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">424</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DRodriguez-Moreno%26rft.auinit1%253DL.%26rft.volume%253D12%26rft.spage%253D424%26rft.epage%253D424%26rft.atitle%253DDetermination%2Bof%2Bthe%2Bmelon%2Bchloroplast%2Band%2Bmitochondrial%2Bgenome%2Bsequences%2Breveals%2Bthat%2Bthe%2Blargest%2Breported%2Bmitochondrial%2Bgenome%2Bin%2Bplants%2Bcontains%2Ba%2Bsignificant%2Bamount%2Bof%2BDNA%2Bhaving%2Ba%2Bnuclear%2Borigin.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-424%26rft_id%253Dinfo%253Apmid%252F21854637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-424&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21854637&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.55" data-doi="10.1093/molbev/msn129"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanchez-Puerta</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mower</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alverson</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Frequent, phylogenetically local horizontal transfer of the <em>cox1</em> group I Intron in flowering plant mitochondria.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1762</span>–<span class="cit-lpage">1777</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn129%26rft_id%253Dinfo%253Apmid%252F18524785%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI1LzgvMTc2MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMjczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.56" data-doi="10.1016/j.tree.2010.06.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaack</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feschotte</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Promiscuous DNA: horizontal transfer of transposable elements and why it matters for eukaryotic evolution.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">537</span>–<span class="cit-lpage">546</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DSchaack%26rft.auinit1%253DS.%26rft.volume%253D25%26rft.issue%253D9%26rft.spage%253D537%26rft.epage%253D546%26rft.atitle%253DPromiscuous%2BDNA%253A%2Bhorizontal%2Btransfer%2Bof%2Btransposable%2Belements%2Band%2Bwhy%2Bit%2Bmatters%2Bfor%2Beukaryotic%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2010.06.001%26rft_id%253Dinfo%253Apmid%252F20591532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2010.06.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20591532&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281657500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shannon</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaufman</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Judd</span>  <span class="cit-name-given-names">B. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Lethality patterns and morphology of selected lethal and semi-lethal mutations in the zeste-white region of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">615</span>–<span class="cit-lpage">638</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DShannon%26rft.auinit1%253DM.%2BP.%26rft.volume%253D72%26rft.issue%253D4%26rft.spage%253D615%26rft.epage%253D638%26rft.atitle%253DLETHALITY%2BPATTERNS%2BAND%2BMORPHOLOGY%2BOF%2BSELECTED%2BLETHAL%2BAND%2BSEMI-LETHAL%2BMUTATIONS%2BIN%2BTHE%2BZESTE-WHITE%2BREGION%2BOF%2BDROSOPHILA%2BMELANOGASTER%26rft_id%253Dinfo%253Apmid%252F4631596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzIvNC82MTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzYvMTI3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.58" data-doi="10.1093/molbev/msm245"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mitochondrial genome of the colorless green alga <em>Polytomella capuana</em>: a linear molecule with an unprecedented GC content.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">487</span>–<span class="cit-lpage">496</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm245%26rft_id%253Dinfo%253Apmid%252F18222946%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI1LzMvNDg3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.59" data-doi="10.1186/1471-2164-10-132"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The mitochondrial and plastid genomes of <em>Volvox carteri</em>: bloated molecules rich in repetitive DNA.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DSmith%26rft.auinit1%253DD.%2BR.%26rft.volume%253D10%26rft.spage%253D132%26rft.epage%253D132%26rft.atitle%253DThe%2Bmitochondrial%2Band%2Bplastid%2Bgenomes%2Bof%2BVolvox%2Bcarteri%253A%2Bbloated%2Bmolecules%2Brich%2Bin%2Brepetitive%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-132%26rft_id%253Dinfo%253Apmid%252F19323823%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-132&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19323823&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.60" data-doi="10.1093/bioinformatics/btl446"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2688</span>–<span class="cit-lpage">2690</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl446%26rft_id%253Dinfo%253Apmid%252F16928733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjIvMjEvMjY4OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMjczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.61" data-doi="10.1371/journal.pone.0078105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turk</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Das</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seibert</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andrulis</span>  <span class="cit-name-given-names">E. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The mitochondrial RNA landscape of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e78105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D8%26rft.spage%253De78105%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0078105%26rft_id%253Dinfo%253Apmid%252F24143261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0078105&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24143261&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.62" data-doi="10.1093/gbe/evs055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wallau</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ortiz</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loreto</span>  <span class="cit-name-given-names">E. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Horizontal transposon transfer in eukarya: detection, bias, and perspectives.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">689</span>–<span class="cit-lpage">699</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%2Band%2BEvolution%26rft.stitle%253DGenome%2BBiol%2BEvol%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253D689%26rft.epage%253D699%26rft.atitle%253DHorizontal%2BTransposon%2BTransfer%2Bin%2BEukarya%253A%2BDetection%252C%2BBias%252C%2Band%2BPerspectives%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevs055%26rft_id%253Dinfo%253Apmid%252F22798449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evs055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22798449&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.63" data-doi="10.1007/BF00331278"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weiller</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schueller</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schweyen</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Putative target sites for mobile G + C rich clusters in yeast mitochondrial DNA: single elements and tandem arrays.</span> <abbr class="cit-jnl-abbrev">Mol. Gen. Genet.</abbr> <span class="cit-vol">218</span>: <span class="cit-fpage">272</span>–<span class="cit-lpage">283</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bgeneral%2Bgenetics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGen%2BGenet%26rft.aulast%253DWeiller%26rft.auinit1%253DG.%26rft.volume%253D218%26rft.issue%253D2%26rft.spage%253D272%26rft.epage%253D283%26rft.atitle%253DPutative%2Btarget%2Bsites%2Bfor%2Bmobile%2BG%2B%252B%2BC%2Brich%2Bclusters%2Bin%2Byeast%2Bmitochondrial%2BDNA%253A%2Bsingle%2Belements%2Band%2Btandem%2Barrays.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00331278%26rft_id%253Dinfo%253Apmid%252F2674655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00331278&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2674655&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989AJ18300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-book" id="cit-5.6.1273.64"><div class="cit-metadata unstructured">Wu, B., and W. Hao, 2014 Horizontal transfer and gene conversion as an important driving force in shaping the landscape of mitochondrial introns. G3 (Bethesda) 4: 605–612.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.65" data-doi="10.1016/0092-8674(81)90201-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heckman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite> and U. L. RajBhandary, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Highly conserved GC-rich palindromic DNA sequences flank tRNA genes in <em>Neurospora crassa</em> mitochondria.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">325</span>–<span class="cit-lpage">332</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DYin%26rft.auinit1%253DS.%26rft.volume%253D26%26rft.issue%253D3%2BPt%2B1%26rft.spage%253D325%26rft.epage%253D332%26rft.atitle%253DHighly%2Bconserved%2BGC-rich%2Bpalindromic%2BDNA%2Bsequences%2Bflank%2BtRNA%2Bgenes%2Bin%2BNeurospora%2Bcrassa%2Bmitochondria.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252881%252990201-4%26rft_id%253Dinfo%253Apmid%252F6276013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(81)90201-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6276013&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F6%2F1273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1981MR02500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.66" data-doi="10.1101/gr.074492.107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zerbino</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Velvet: algorithms for de novo short read assembly using de Bruijn graphs.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">821</span>–<span class="cit-lpage">829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.074492.107%26rft_id%253Dinfo%253Apmid%252F18349386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE4LzUvODIxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.67" data-doi="10.1093/nar/gkg595"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zuker</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Mfold web server for nucleic acid folding and hybridization prediction.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3406</span>–<span class="cit-lpage">3415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DZuker%26rft.auinit1%253DM.%26rft.volume%253D31%26rft.issue%253D13%26rft.spage%253D3406%26rft.epage%253D3415%26rft.atitle%253DMfold%2Bweb%2Bserver%2Bfor%2Bnucleic%2Bacid%2Bfolding%2Band%2Bhybridization%2Bprediction%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg595%26rft_id%253Dinfo%253Apmid%252F12824337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMS8xMy8zNDA2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS82LzEyNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-5.6.1273.68" data-doi="10.1093/molbev/mst107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zuo</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide analysis reveals origin of transfer RNA genes from tRNA halves.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2087</span>–<span class="cit-lpage">2098</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst107%26rft_id%253Dinfo%253Apmid%252F23744908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjMwLzkvMjA4NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNi8xMjczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/6/1273.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/6/1263" title="Neurospora Heterokaryons with Complementary Duplications and Deficiencies in Their Constituent Nuclei Provide an Approach to Identify Nucleus-Limited Genes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/6/1283" title="A Multifunctional Mutagenesis System for Analysis of Gene Function in Zebrafish" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/6"><span>Volume 5 Issue 6, June 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/6.cover-source.jpg?itok=NSNFWJbY" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/6/1273" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4Dpl47wt4YEW4PDUzfOMGZ32BR3RQLizwWwn-Qf60V0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/6/1273" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/6/1273" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/6/1273" class="active" data-icon-position="" data-hide-link-title="0">A Dynamic Mobile DNA Family in the Yeast Mitochondrial Genome</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459968" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-nmdCIe8sRVkS_-IzWXlO6rZY0vcoqWCqKsjmmdqqAjw" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/6/1273.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/6/1273" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-0Um6u2RUvF_oQ5JP65Kf5gSgSZ_8QQW2DQvzuqB2Mc8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/6/1273" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/6/1273.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459968" id="node459968--2" data-pisa="ggg;5/6/1273" data-pisa-master="ggg;g3.115.017822" data-apath="/ggg/5/6/1273.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Dynamic Mobile DNA Family in the Yeast Mitochondrial Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Baojun Wu</span> and <span class="highwire-citation-author" data-delta="1">Weilong Hao</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1273-1282;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017822 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459968--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Baojun Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, Wayne State University, Detroit, Michigan 48202</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Baojun%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABaojun%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Weilong Hao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, Wayne State University, Detroit, Michigan 48202</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Weilong%2BHao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hao%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWeilong%2BHao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:haow@wayne.edu" class="" data-icon-position="" data-hide-link-title="0">haow@wayne.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/6/1273" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/6/1273&amp;title=A%20Dynamic%20Mobile%20DNA%20Family%20in%20the%20Yeast%20Mitochondrial%20Genome" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/6/1273&amp;title=A%20Dynamic%20Mobile%20DNA%20Family%20in%20the%20Yeast%20Mitochondrial%20Genome" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/6/1273&amp;title=A%20Dynamic%20Mobile%20DNA%20Family%20in%20the%20Yeast%20Mitochondrial%20Genome" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/6/1273&amp;text=A%20Dynamic%20Mobile%20DNA%20Family%20in%20the%20Yeast%20Mitochondrial%20Genome" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/6/1273&amp;title=A%20Dynamic%20Mobile%20DNA%20Family%20in%20the%20Yeast%20Mitochondrial%20Genome" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/6/1273&amp;t=A%20Dynamic%20Mobile%20DNA%20Family%20in%20the%20Yeast%20Mitochondrial%20Genome" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/6/1273&amp;title=A%20Dynamic%20Mobile%20DNA%20Family%20in%20the%20Yeast%20Mitochondrial%20Genome" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/6/1273&amp;title=A%20Dynamic%20Mobile%20DNA%20Family%20in%20the%20Yeast%20Mitochondrial%20Genome" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459968/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459968" id="node459968--3" data-pisa="ggg;5/6/1273" data-pisa-master="ggg;g3.115.017822" data-apath="/ggg/5/6/1273.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Dynamic Mobile DNA Family in the Yeast Mitochondrial Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Baojun Wu</span> and <span class="highwire-citation-author" data-delta="1">Weilong Hao</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1273-1282;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017822 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459968--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Baojun Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, Wayne State University, Detroit, Michigan 48202</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Baojun%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABaojun%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Weilong Hao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, Wayne State University, Detroit, Michigan 48202</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Weilong%2BHao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hao%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWeilong%2BHao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:haow@wayne.edu" class="" data-icon-position="" data-hide-link-title="0">haow@wayne.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1273&amp;rft.epage=1282&amp;rft.atitle=A%20Dynamic%20Mobile%20DNA%20Family%20in%20the%20Yeast%20Mitochondrial%20Genome&amp;rft.volume=5&amp;rft.issue=6&amp;rft.date=2015-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Wu%2C+Baojun&amp;rft.au=Hao%2C+Weilong"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459968/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459968/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459968/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459968/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459968/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459968/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459968/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459968/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459968/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459968/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459968/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459968/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/6/1273" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/6/1273.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/6/1273.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/6/1273.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/6/1273.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
